
<html lang="en"     class="pb-page"  data-request-id="ac62f63f-1bdb-465b-89e4-f9d04b512ba4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-19;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c00821;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of SK-575 as a Highly Potent and Efficacious Proteolysis-Targeting Chimera Degrader of PARP1 for Treating Cancers" /></meta><meta name="dc.Creator" content="Chaoguo  Cao" /></meta><meta name="dc.Creator" content="Jie  Yang" /></meta><meta name="dc.Creator" content="Yong  Chen" /></meta><meta name="dc.Creator" content="Peiting  Zhou" /></meta><meta name="dc.Creator" content="Yingwei  Wang" /></meta><meta name="dc.Creator" content="Wu  Du" /></meta><meta name="dc.Creator" content="Lifeng  Zhao" /></meta><meta name="dc.Creator" content="Yuanwei  Chen" /></meta><meta name="dc.Description" content="The nuclear protein poly(ADP-ribose) polymerase-1 (PARP1) has a well-established role in the signaling and repair of DNA and is a validated therapeutic target for cancers and other human diseases. ..." /></meta><meta name="Description" content="The nuclear protein poly(ADP-ribose) polymerase-1 (PARP1) has a well-established role in the signaling and repair of DNA and is a validated therapeutic target for cancers and other human diseases. ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 14, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00821" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00821" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00821" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00821" /></link>
        
    
    

<title>Discovery of SK-575 as a Highly Potent and Efficacious Proteolysis-Targeting Chimera Degrader of PARP1 for Treating Cancers | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00821" /></meta><meta property="og:title" content="Discovery of SK-575 as a Highly Potent and Efficacious Proteolysis-Targeting Chimera Degrader of PARP1 for Treating Cancers" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0028.jpeg" /></meta><meta property="og:description" content="The nuclear protein poly(ADP-ribose) polymerase-1 (PARP1) has a well-established role in the signaling and repair of DNA and is a validated therapeutic target for cancers and other human diseases. Here, we have designed, synthesized, and evaluated a series of small-molecule PARP1 degraders based on the proteolysis-targeting chimera (PROTAC) concept. Our efforts have led to the discovery of highly potent PARP1 degraders, as exemplified by compound 18 (SK-575). SK-575 potently inhibits the growth of cancer cells bearing BRCA1/2 mutations and induces potent and specific degradation of PARP1 in various human cancer cells even at low picomolar concentrations. SK-575 achieves durable tumor growth inhibition in mice when used as a single agent or in combination with cytotoxic agents, such as temozolomide and cisplatin. These data demonstrate that SK-575 is a highly potent and efficacious PARP1 degrader." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00821"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00821">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00821&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00821&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00821&amp;href=/doi/10.1021/acs.jmedchem.0c00821" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 19</span><span class="cit-fg-pageRange">, 11012-11033</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/19" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00916" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00956" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of SK-575 as a Highly Potent and Efficacious Proteolysis-Targeting Chimera Degrader of PARP1 for Treating Cancers</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Chaoguo Cao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chaoguo Cao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy and Cancer Center, West China Hospital of Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chaoguo++Cao">Chaoguo Cao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jie Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jie Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy and Cancer Center, West China Hospital of Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jie++Yang">Jie Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yong Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yong Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy and Cancer Center, West China Hospital of Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yong++Chen">Yong Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peiting Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peiting Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy and Cancer Center, West China Hospital of Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peiting++Zhou">Peiting Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yingwei Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yingwei Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy and Cancer Center, West China Hospital of Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yingwei++Wang">Yingwei Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wu Du</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wu Du</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hinova Pharmaceuticals Inc., 4th Floor, Building RongYao A, No. 5, Keyuan South Road, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wu++Du">Wu Du</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lifeng Zhao</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lifeng Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sichuan Industrial Institute of Antibiotics, Chengdu University, Chengdu 610052, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#cca0a5aaa9a2abb6a4ada38cbfafb9e2a9a8b9e2afa2"><span class="__cf_email__" data-cfemail="2d41444b48434a57454c426d5e4e5803484958034e43">[email protected]</span></a>. Phone: +86-28-85164063.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lifeng++Zhao">Lifeng Zhao</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Yuanwei Chen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuanwei Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy and Cancer Center, West China Hospital of Sichuan University, Chengdu 610041, China</div><div class="loa-info-affiliations-info">Hinova Pharmaceuticals Inc., 4th Floor, Building RongYao A, No. 5, Keyuan South Road, Chengdu 610041, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#f188869299949fb1829284df949584df929f"><span class="__cf_email__" data-cfemail="2f56584c474a416f5c4c5a014a4b5a014c41">[email protected]</span></a>. Phone: +86-28-85503817.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuanwei++Chen">Yuanwei Chen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4248-3613" title="Orcid link">http://orcid.org/0000-0003-4248-3613</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00821&amp;href=/doi/10.1021%2Facs.jmedchem.0c00821" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 19</span><span class="cit-pageRange">, 11012–11033</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 14, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 June 2020</li><li><span class="item_label"><b>Published</b> online</span>14 September 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 October 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00821" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00821</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D11012%26pageCount%3D22%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DChaoguo%2BCao%252C%2BJie%2BYang%252C%2BYong%2BChen%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D19%26contentID%3Dacs.jmedchem.0c00821%26title%3DDiscovery%2Bof%2BSK-575%2Bas%2Ba%2BHighly%2BPotent%2Band%2BEfficacious%2BProteolysis-Targeting%2BChimera%2BDegrader%2Bof%2BPARP1%2Bfor%2BTreating%2BCancers%26numPages%3D22%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D11033%26publicationDate%3DOctober%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00821"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3419</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00821" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of SK-575 as a Highly Potent and Efficacious Proteolysis-Targeting Chimera Degrader of PARP1 for Treating Cancers&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Chaoguo&quot;,&quot;last_name&quot;:&quot;Cao&quot;},{&quot;first_name&quot;:&quot;Jie&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Yong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Peiting&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Yingwei&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Wu&quot;,&quot;last_name&quot;:&quot;Du&quot;},{&quot;first_name&quot;:&quot;Lifeng&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Yuanwei&quot;,&quot;last_name&quot;:&quot;Chen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;19&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;11012-11033&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00821&quot;},&quot;abstract&quot;:&quot;The nuclear protein poly(ADP-ribose) polymerase-1 (PARP1) has a well-established role in the signaling and repair of DNA and is a validated therapeutic target for cancers and other human diseases. Here, we have designed, synthesized, and evaluated a series of small-molecule PARP1 degraders based on the proteolysis-targeting chimera (PROTAC) concept. Our efforts have led to the discovery of highly potent PARP1 degraders, as exemplified by compound 18 (SK-575). SK-575 potently inhibits the growth of cancer cells bearing BRCA1/2 mutations and induces potent and specific degradation of PARP1 in various human cancer cells even at low picomolar concentrations. SK-575 achieves durable tumor growth inhibition in mice when used as a single agent or in combination with cytotoxic agents, such as temozolomide and cisplatin. These data demonstrate that SK-575 is a highly potent and efficacious PARP1 degrader.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00821&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00821" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00821&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00821" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00821&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00821" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00821&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00821&amp;href=/doi/10.1021/acs.jmedchem.0c00821" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00821" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00821" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00821%26sid%3Dliteratum%253Aachs%26pmid%3D32924477%26genre%3Darticle%26aulast%3DCao%26date%3D2020%26atitle%3DDiscovery%2Bof%2BSK-575%2Bas%2Ba%2BHighly%2BPotent%2Band%2BEfficacious%2BProteolysis-Targeting%2BChimera%2BDegrader%2Bof%2BPARP1%2Bfor%2BTreating%2BCancers%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D19%26spage%3D11012%26epage%3D11033%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/19" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/jmcmar.2020.63.issue-19/20201008/jmcmar.2020.63.issue-19.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0028.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The nuclear protein poly(ADP-ribose) polymerase-1 (PARP1) has a well-established role in the signaling and repair of DNA and is a validated therapeutic target for cancers and other human diseases. Here, we have designed, synthesized, and evaluated a series of small-molecule PARP1 degraders based on the proteolysis-targeting chimera (PROTAC) concept. Our efforts have led to the discovery of highly potent PARP1 degraders, as exemplified by compound <b>18</b> (SK-575). SK-575 potently inhibits the growth of cancer cells bearing BRCA1/2 mutations and induces potent and specific degradation of PARP1 in various human cancer cells even at low picomolar concentrations. SK-575 achieves durable tumor growth inhibition in mice when used as a single agent or in combination with cytotoxic agents, such as temozolomide and cisplatin. These data demonstrate that SK-575 is a highly potent and efficacious PARP1 degrader.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73355" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73355" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Poly(ADP-ribose) polymerase-1 (PARP1) is the most abundant and well-characterized member of poly(ADP-ribose) polymerase (PARP) family of enzymes.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> As a DNA-dependent nuclear enzyme, PARP1 plays a pivotal role in the signaling and repair of DNA damage.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Following DNA damage caused by the cell’s own metabolic activity, chemical poisons, or ionizing radiation,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> PARP1 is rapidly activated, which then recognizes and binds to DNA single-strand breaks (SSBs), where it catalyzes the break of nicotinamide adenine dinucleotide (NAD<sup>+</sup>) to ADP-ribose and nicotinamide and then transfers the ADP-ribose units to synthesize poly(ADP-ribose) (PAR) chains either on itself or on several acceptor proteins.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5−7)</a> These PAR chains with more negative charges and a larger steric hindrance can make PARP1 dissociate from the DNA breaks by reducing the affinity between PARP1 and DNA and then guide DNA repair factors to combine with the DNA gap to repair the damaged part.<a onclick="showRef(event, 'ref2 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref2 ref8 ref9">(2,8,9)</a> This is the key process known as the base excision repair pathway for DNA repair.<a onclick="showRef(event, 'ref2 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref5">(2,5)</a></div><div class="NLM_p">The PARP1 knockout cancer cells and animals exhibit a high degree of sensitivity to radiation or cytotoxic agents.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> Initially, PARP1 inhibitors (PARPi) were developed as promising coadjuvant components of standard chemo- and radiotherapy for tumor therapy.<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> Cancer cells bearing mutations in BRCA-1 or -2 are exquisitely sensitive to PARPi, which is called “synthetic lethality”.<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15−17)</a> In this regard, many small-molecule PARPi, such as olaparib,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> rucaparib,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> niraparib,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and talazoparib,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> have been developed in different stages of clinical trials.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> For instance, olaparib, as the first oral potent PARP1/2 inhibitor, has been approved for the treatment of BRCA-mutation cancer in the clinic, especially for ovarian and breast cancers.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> However, there are still challenges in terms of PARPi usage that limit their therapeutic utility, including the acquisition of drug resistance and the low proportion of BRCA-1 or -2 mutations in cancer cells.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In addition to the inhibition of PARP1 enzyme activity, the mechanism of action of PARPi as an antitumor agent is via PARP1 trapping.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> In short, PARPi prevents the PARP1 protein from dissociating from DNA breaks to form PARP1–PARPi–DNA complexes, which could block and damage DNA replication, and is therefore more toxic than merely unrepaired SSBs due to the inactivation of PARP1. Thus, PARPi can kill tumor cells and are also toxic to normal cells.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> However, we are unable to assess the relative contribution of these two pathways (PARP1 catalytic inhibition and trapping) to PARP inhibitor-mediated cytotoxicity.</div><div class="NLM_p">Besides the role in cancer cells, the overactivation of PARP1 has been associated with several pathophysiological conditions, such as neurological diseases (e.g., stroke, neurodegeneration, and so forth),<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> ischemia-reperfusion injury (e.g., myocardial infarction),<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> severe acute critical illness (e.g., shock, acute lung injury, acute liver failure, and so forth),<a onclick="showRef(event, 'ref28 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref28 ref30 ref31">(28,30,31)</a> and inflammatory diseases.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> In these nononcological diseases, the overstimulation of PARP1 with the catalytic activity results in the cellular depletion of NAD<sup>+</sup> and forces the activation of the salvage synthesis pathway. NAD<sup>+</sup> depletion leads to ATP depletion, resulting in cellular energetic deficit and cell necrotic death.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Moreover, the overactivation of PARP1 leads to the accumulation of free PAR polymers to reach the cytotoxic level, which induces mitochondria release of apoptosis-inducing factors rapidly, leading to cell death termed parthanatos.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Several kinds of PARPi have been proved to be effective in nononcological indications. However, their usage is still cautious, mainly because of their PARP1-trapping activity and mechanism-independent cytotoxicity.<a onclick="showRef(event, 'ref24 ref35'); return false;" href="javascript:void(0);" class="ref ref24 ref35">(24,35)</a> Consequently, developing a practical strategy, by blocking both the catalytic and scaffolding functions of PARP1 without causing PARP1 trapping, is urgent and meaningful for PARP1-related diseases.</div><div class="NLM_p">The proteolysis-targeting chimera (PROTAC) concept can be dated back to 2001.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> PROTAC molecules are heterobifunctional small molecules containing two ligands, which specifically bind to the target protein and E3 ubiquitin ligase, respectively, tethered together by a linker. These molecules are capable of driving the target proteins to E3 ubiquitin ligase, which results in the formation of a ternary complex and leads to ubiquitination of the target proteins and consequent proteasome-mediated degradation.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> In the past decade, the PROTAC strategy has gained great progress in inducing target protein degradation in vitro and in vivo, such as BET,<a onclick="showRef(event, 'ref38 ref39 ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref38 ref39 ref40 ref41 ref42">(38−42)</a> AR,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> ER,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> ALK,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and CDK9,<a onclick="showRef(event, 'ref46 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref46 ref47 ref48">(46−48)</a> which are attributed to the discovery of several highly potent small-molecule ligands for E3 ubiquitin ligase. Several PARP1/2 degraders have been reported, including compounds <b>1</b>,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a><b>2</b> (iRucaparib-AP5),<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a><b>3</b> (iRucaparib-AP6),<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and <b>4</b> (iVeliparib-AP6)<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Compound <b>1</b>, based on the niraparib derivative and the nutlin-3 derivative, specifically induced the PARP1 cleavage and cell apoptosis in MDA-MB-231 cells. In comparison, two of the compounds, <b>2</b> and <b>3</b>, prepared using different lengths of all polyethylene glycol (PEG) linkers, were reported as potent and efficacious PARP1 degraders with half-maximal degrading concentrations (DC<sub>50</sub>) of 36 and 82 nM in primary rat neonatal cardiomyocytes, respectively. However, these compounds could not completely degrade the intracellular PARP1 protein, and the remaining part can still play a catalytic role, thus limiting their therapeutic efficacy. Besides, compound <b>4</b> emerged as a potent PARP2 degrader, which was designed using veliparib as the PARP2 ligand and thalidomide as a cereblon (CRBN) ligand.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structure of three previously reported PARP1 degraders <b>1</b>, <b>2</b>, and <b>3</b> and one previously reported PARP2 degrader <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In this study, we describe the design, synthesis, and evaluation of PARP1-PROTAC degraders based upon a PAPR1 inhibitor olaparib and thalidomide/lenalidomide as ligands for cereblon/cullin 4A. Our study has resulted in the discovery of highly potent PARP1-PROTAC degraders (hereafter, called PARP1 degraders), exemplified by compound <b>18</b> (SK-575). SK-575 is capable of effectively inducing PARP1 protein degradation with DC<sub>50</sub> at the scale of picomolar (pM) and maximum degradation (<i>D</i><sub>max</sub>) > 95% in PARP1-positive cancer cell lines, and achieving durable tumor growth inhibition in mice at well-tolerated dose schedules. SK-575 is a highly potent and specific PARP1 degrader and warrants extensive evaluation for the treatment of cancers and other diseases.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05158" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05158" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design of PARP1 PROTAC Based on Olaparib</h3><div class="NLM_p">To explore potential PARP1 degraders, we chose olaparib as the PARP1 ligand in our design because of its excellent activity against PARP1/2 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> According to the cocrystal structure of olaparib in complex with PARP1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), the amide moiety on olaparib is essential for binding to PARP1.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The cyclopropyl ring is located close to the opening of the ligand-binding pocket and thus may represent a suitable position to link with CRBN ligands without losing the binding affinity dramatically. Based on the abovementioned design principles, we designed and synthesized a series of PARP1-targeting PROTACs using olaparib as the PARP1 ligand, diverse linkers, and thalidomide/lenalidomide as CRBN ligands.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of PROTACs for PARP1. (A) Chemical structures of PARP1/2 inhibitor <b>5</b> (olaparib). (B) Crystal structure of <b>5</b> bound to the PARP1 catalytic domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DS3">5DS3</a>). The possible linker-tethering positions are shown as red arrows.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Determination of the Optimal Linker Length and Composition</h3><div class="NLM_p">It has been confirmed that a series of PARP1 degraders that conjugate olaparib to thalidomide with the PEG unit failed to induce PARP1 degradation (see Figure S1 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Supporting Information</a>).<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49,50)</a> We speculated that the length and the composition of the linker may be underexplored, and an optimized linker may induce a favorable interaction between PARP1 and CRBN E3 ligase. Thus, we have performed extensive optimization of the chemical composition of the linker with different lengths. A series of PARP1 degraders, compounds <b>6–11</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), with different amino acid linkers (e.g., 4-aminobutyric acid and/or 6-aminocaproic acid) were synthesized by a condensation reaction. Their growth inhibitory effect against BRCA1-deficient MDA-MB-436 and BRCA2-deficient Capan-1 cell lines and the IC<sub>50</sub> values (half-maximal growth inhibitory concentration) were then evaluated by a cell growth assay. Compound <b>11</b> has IC<sub>50</sub> values of 0.95 μM in MDA-MB-436 and 1.5 μM in Capan-1, which were 52 and 56 times less potent than olaparib with IC<sub>50</sub> values of 18 and 56 nM in these two cell lines, respectively. Next, on changing the NH group in <b>10</b> to an oxygen atom yielded <b>12</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), which showed IC<sub>50</sub> values of 2.2 μM in MDA-MB-436 and 7.1 μM in Capan-1, and is less potent than <b>10</b>. The similar conversion of the NH group in <b>11</b> to an oxygen atom generated <b>13</b>, which has IC<sub>50</sub> values of 1.7 and 2.2 μM in the inhibition of cell growth in MDA-MB-436 and Capan-1 cells, respectively, and is less potent than <b>11</b>. These results demonstrate that the amino acid linkers have an adverse influence on the cellular potency of the resulting PROTAC molecules. Thus, we identified that PROTAC molecules conjugating olaparib and CRBN ligands with amino acid linkers could not be beneficial for protein–protein interactions (PPIs) between CRBN E3 ligase and PARP1, cell penetration, and so on.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Optimization of the Linker Length and Composition of PARP1 Degraders</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0020.gif" alt="" id="GRAPHIC-d7e553-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0021.gif" alt="" id="gr19" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were obtained from three independent experiments (mean ± SD).</p></div></div><div></div></div><div class="NLM_p">We next sought to investigate the optimal linkers and synthesized compounds <b>14–16</b>, <b>18</b>, and <b>20</b> that incorporated alkyl dicarboxylic acid of different lengths and ethylenediamine (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Surprisingly, compound <b>18</b>, which contained dodecarboxylic acid, displayed similar potency to olaparib in cell growth inhibition and achieved IC<sub>50</sub> values of 19 nM in MDA-MB-436 cells and 56 nM in Capan-1 cells. Furthermore, changing the linker in <b>18</b> by decreasing or increasing one additional methylene group resulted in <b>17</b> or <b>19</b>, respectively, which are less potent than <b>18</b> in MDA-MB-436 and Capan-1 cells.</div><div class="NLM_p">Encouraged by the high potency of compound <b>18</b> in the inhibition of cell growth, we next synthesized compounds <b>21</b> and <b>22</b> converting the −NH(CH<sub>2</sub>)<sub>2</sub>NH– group in <b>18</b> to a −NH(CH<sub>2</sub>)<sub>4</sub>NH– or −NH(CH<sub>2</sub>)<sub>6</sub>NH– group to further improve the potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). However, compounds <b>21</b> and <b>22</b>, which were ∼2 times less potent than compound <b>18</b> in cell growth inhibition, achieved IC<sub>50</sub> values of 39 and 42 nM in MDA-MB-436 cells and 131 and 110 nM in Capan-1 cells, respectively. Furthermore, considering that the linker length and chemical composition were important for PARP1 degraders to effectively inhibit cell growth, we replaced the −NH(CH<sub>2</sub>)<sub>2</sub>NH– group in <b>18</b> with a different length linker containing an oxygen atom and generated compounds <b>23–25</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). These compounds turned out to be less potent than compound <b>18</b> in cell growth inhibition. Consequently, these data thus established that a linker length of 15–16 atoms, as in compound <b>18</b> or <b>19</b>, is optimal for these PAPR1 degraders for the inhibition of cell growth.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Further Optimization of the Linker Length and Composition</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0022.gif" alt="" id="GRAPHIC-d7e680-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0023.gif" alt="" id="gr20" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were obtained from three independent experiments (mean ± SD).</p></div></div><div></div></div><div class="NLM_p">We further investigated the protein degradation activity of the resulting PARP1 degraders with IC<sub>50</sub> < 200 nM in MDA-MD-436 cells, and the protein level of PARP1 was analyzed through the western blotting assay at four concentrations (1000, 100, 10, and 1 nM) in MDA-MB-436 and Capan-1 cell lines with 24 h treatment. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, compound <b>18</b> showed the best degradation activity at four concentrations in these two cancer cell lines, which is consistent with its cell growth inhibitory activity. In comparison, the corresponding PARP1 inhibitor, olaparib, failed to reduce the protein level of PARP1 at all tested concentrations (1–1000 nM). Compounds <b>20</b> and <b>21</b> with the same length linker showed similar IC<sub>50</sub> values in inhibition of cell growth, but they revealed different degradation activities. It may be that changes in the position of the amide bond in the linker influenced the formation of special conformations. It is noteworthy that compound <b>21</b> effectively reduced the PARP1 protein by 70% at 10 nM and by 90.5% at 100 nM in MDA-MB-436 cell lines. Interestingly, compound <b>21</b> reduced the PARP1 protein by 45% at 1 μM, less than that at 100 nM, probably because of the “hook” effect that has been observed previously for PROTAC degraders.<a onclick="showRef(event, 'ref43 ref44 ref52'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref52">(43,44,52)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Western blotting analysis of the PARP1 protein in MDA-MB-436 (A) and Capan-1 (B) cells treated with the PARP1 inhibitor OLA (olaparib), PARP1 degraders <b>16–27</b>, and a control compound <b>28</b>. Cells were treated with each compound at 1, 10, 100, and 1000 nM for 24 h, and β-tubulin was used as the loading control. The fold under the strip represents the PARP1/β-tubulin ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Examination of the CRBN Ligand Portion in Compound <b>18</b></h3><div class="NLM_p">On the basis of <b>18</b>, we designed and synthesized compound <b>26</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) that tethered the phenyl ring of thalidomide from different positions.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Compound <b>26</b> has IC<sub>50</sub> values of 35 and 159 nM in inhibition of cell growth in MDA-MB-436 and Capan-1 cell lines, respectively, which is 2–3 times less potent than <b>18</b>. In addition, converting the 3-carbonyl group of the thalidomide moiety in <b>18</b> to a methylene group generated <b>27</b>, which inhibited cell growth in MDA-MB-436 and Capan-1 cells with IC<sub>50</sub> values of 92 nM and 194 nM, respectively, and is 4–5 times less potent than compound <b>18</b>. Our western blotting analysis showed that <b>26</b> and <b>27</b> effectively induced PARP1 degradation, but their activities are weaker than <b>18</b> in the abovementioned two cell lines.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Further Optimization of the CRBN Ligand Portion in Compound <b>18</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0024.gif" alt="" id="GRAPHIC-d7e797-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0025.gif" alt="" id="gr21" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were obtained from three independent experiments (mean ± SD).</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Design and Synthesis of Control Compound for <b>18</b></h3><div class="NLM_p">As revealed by the cocrystal structures, the amino group of piperidine-2,6-dione in thalidomide forms a strong hydrogen bond with CRBN<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54,55)</a> and methylation of the amino group of piperidine-2,6-dione in thalidomide completely abrogates the binding to CRBN.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> We, therefore, synthesized compound <b>28</b> by methylation of the amino group of piperidine-2,6-dione in compound <b>18</b> as an additional control compound (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Compound <b>28</b> was completely ineffective in inducing PARP1 degradation in MDA-MB-436 and Capan-1 cells at concentrations up to 1 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). These data clearly suggest that the induced degradation of PARP1 by compound <b>18</b> was cereblon/cullin-4A ligase-dependent, consistent with the PROTAC design.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design of <b>28</b> as a control compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Examination of the VHL Ligand Portion in Compound <b>18</b></h3><div class="NLM_p">Like CRBN ligands (such as thalidomide and its analogues), VHL ligands have been successfully used for the design of PROTAC degraders.<a onclick="showRef(event, 'ref39 ref41 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref39 ref41 ref43 ref44">(39,41,43,44)</a> Therefore, on the basis of <b>18</b>, we designed and synthesized compounds <b>29–32</b> using VHL ligands (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Compound <b>29</b> has IC<sub>50</sub> values of 54 and 87 nM in inhibition of cell growth in MDA-MB-436 and Capan-1 cell lines, respectively, which are 2–3 times less potent than <b>18</b>. After changing the linker length, the cell proliferation inhibitory activity of these compounds decreased when compared with <b>29</b>. Our western blotting analysis showed that compounds <b>29–32</b> had only a moderate effect in reducing the level of PARP1 protein at 1 μM concentrations in the MDA-MB-436 cell line (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Figure S2</a>). Taken together, these data indicate that these compounds mainly work as inhibitors, especially compound <b>29</b>, but not as PARP1 degraders. More studies are needed to determine if the VHL/cullin 2 E3 ligase system can be used for the design of highly potent and effective PARP1 degraders.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Further Optimization of the VHL Ligand Portion in Compound <b>18</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0026.gif" alt="" id="GRAPHIC-d7e909-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0027.gif" alt="" id="gr22" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were obtained from three independent experiments (mean ± SD).</p></div></div><div></div></div><div class="NLM_p">On direct comparison, the previously reported PARP1 degrader, compound <b>3</b> (iRucaparib-AP6), has IC<sub>50</sub> values of 0.71 and 0.95 μM in inhibition of cell growth in MDA-MB-436 and Capan-1 cell lines, respectively, which are 36 and 17 times less potent than <b>18</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Western blotting data showed that compound <b>18</b> was highly potent and effective in reducing PARP1 protein at 0.01–10 μM concentrations with 12 h treatment in MDA-MB-436 and SW620 cell lines (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). However, compound <b>3</b> exhibited only a moderate effect in reducing the level of PARP1 protein under the same conditions.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Western blotting analysis of PARP1 protein in MDA-MB-436 (A) and SW620 (B) cells treated with compounds <b>3</b> and <b>18</b>. Cells were treated with each compound at 1, 10, 100, 1000 nM, and 10 μM for 12 h, and β-tubulin was used as the loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Although compound <b>18</b> effectively induced degradation of the PARP1 protein at concentrations as low as 1–10 nM with 24 h treatment in MDA-MB-436 and Capan-1 cell lines, it had little or no effect on inducing degradation of the PARP2 protein in these two cell lines at concentrations up to 1 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Figure S3</a>). Similarly, olaparib and compound <b>28</b> had no effect on the level of PARP2 protein. Hence, our data demonstrate that compound <b>18</b> is an efficient and specific PARP1 degrader. Based on the data of cell growth inhibition activity and degradation activity of PARP1, compound <b>18</b> appears to be the most potent PARP1 degrader among these resulting compounds. Consequently, we pursued an extensive investigation of compound <b>18</b> (SK-575).<named-content content-type="anchor" rid="fig5" type="simple"></named-content></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Evaluation of Degradation Activity of SK-575 in Cancer Cells</h3><div class="NLM_p">We further evaluated SK-575 in MDA-MB-436, Capan-1, and SW620 cells for its potency in inducing PARP1 protein degradation with a 24 h treatment period in a wide range of concentrations (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">S4</a>). Quantification of our western blotting data showed that SK-575 achieved DC<sub>50</sub> values of 1.26 and 0.509 nM in MDA-MB-436 and SW620 cells, respectively, and >99% PARP1 degradation at 10 nM in these cell lines. SK-575, with a DC<sub>50</sub> value of 6.72 nM, achieved near-complete degradation at 100 nM in Capan-1 cells.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Further characterization of SK-575. (A) PARP1 degradation at the indicated dose of SK-575 with a 24 h treatment period in MDA-MB-436, Capan-1, and SW620 cells (left). PARP1 protein was examined by western blotting, and the PARP1 protein level was quantified and normalized to the corresponding density of β-tubulin protein (right). DC<sub>50</sub> values are calculated by two biologically independent experiments. (B) Time course of PARP1 degradation by SK-575 in MDA-MB-436 (30 nM), Capan-1 (100 nM), and SW620 (30 nM) cells. (C) Time course of PARP1 degradation by SK-575 in MDA-MB-436 (100 nM) cells. (D) Cellular PARP1 recovery levels after washing SK-575. Cells were pre-treated with SK-575 for 2 h and washing the compound for the indicated times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next evaluated the kinetics of SK-575 in the induction of PARP1 degradation at 30 nM in MDA-MB-436 and SW620 cell lines. Our data showed that SK-575 effectively reduced the PARP1 protein level within 1 h and achieved near-complete PARP1 depletion at the 2 h timepoint, indicating fast kinetics (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). The degradation activity of SK-575 in Capan-1 cell lines is weaker than that in MDA-MB-436 and SW620 cell lines, and it can reach the maximum degradation in 6 h at 100 nM.</div><div class="NLM_p">To assess the durability of PARP1 protein degradation, we treated MDA-MB-436 cells with SK-575 at 100 nM concentration for 5 days, illustrating that it induced sustained depletion of PARP1(<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). In another experiment, we treated MDA-MB-436 cells with SK-575 for 2 h at 100 nM concentration and collected cells at various timepoints after washing the compound. Our data showed that the level of PARP1 protein degrader-treated cells remained below the level in vehicle-treated cells, even after 24 h of washing (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D).</div><div class="NLM_p last">To further assess the efficiency and broad applicability of SK-575, six different cancer cell lines were used to evaluate its potency in inducing PARP1 protein degradation with a 24 h treatment period. Surprisingly, besides the LNCaP cells achieving the DC<sub>50</sub> value at the nanomolar level (DC<sub>50</sub> = 1.08 nM), the DC<sub>50</sub> of all the tested cells were on the scale of picomolar after 24 h treatment (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A and S4 in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Supporting Information</a>). The DC<sub>50</sub> values were 384 pM in HCC1937, 512 pM in 22Rv1, 263 pM in MDA-MB-468, 578 pM in PC-3, and 228 pM in MDA-MB-231. It is noteworthy that SK-575 can reduce the PARP1 protein level by >99% with a 24 h treatment period. At the same time, we also observed the rapid degradation of PARP1 protein in all of these cells (Figure S5 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Supporting Information</a>). Taken together, our data demonstrated that SK-575 not only displays the fast and efficient degradation of PARP1 but also has a broad application.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Study on the Mechanism of SK-575</h3><div class="NLM_p">We investigated the mechanism of PARP1 degradation induced by SK-575 in MDA-MB-436 and SW620 cell lines, obtaining the results shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>. The data showed that the degradation of PARP1 protein induced by SK-575 could be effectively blocked by pretreatment with a PARP1 inhibitor (olaparib) or a CRBN ligand (pomalidomide), in a dose-dependent manner for 24 h treatment. These results supported that the high potency of SK-575 as a PARP1 degrader is dependent on binding to PARP1 and CRBN. Furthermore, a combination of a proteasome inhibitor (MG132 or carfilzomib) or NEDD8-activating enzyme (NAE) inhibitor (MLN4924) and SK-575 treatment completely blocked PARP1 degradation induced by SK-575, indicating that the fast and efficient PARP1 protein degradation by SK-575 was based on a NAE-mediated and proteasome-dependent mechanism. Taken together, these mechanistic data provided clear evidence that SK-575 is a bona fide PROTAC PARP1 degrader.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Western blotting analysis of PARP1 protein in MDA-MB-436 (A) and SW620 (B) cells after a 2 h pretreatment with PARP1 inhibitor OLA (olaparib), CRBN ligand POM (pomalidomide), proteasome inhibitor (MG132 or carfilzomib), or NEDD8-activating E1 enzyme inhibitor (MLN4924). Cells were treated with the individual compound at indicated concentrations, followed by a 24 h treatment with SK-575 at 30 nM, and β-tubulin was used as the loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Evaluation Activity of SK-575 in Vitro</h3><div class="NLM_p">Using a PARP1 activity assay, we confirmed that SK-575 and its methylation counterpart <b>28</b> are still strongly bound to PARP1, with IC<sub>50</sub> values of 2.30 and 2.64 nM, respectively, which are equivalent to their parental inhibitor olaparib with an IC<sub>50</sub> value of 3.35 nM (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B). Next, we further examined the ability of SK-575 and olaparib to suppress the cell growth inhibition of HR-deficient cell lines containing BRCA1/2 or PTEN, and HR-proficient cancer cell lines (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C and Table S1 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Supporting Information</a>).<a onclick="showRef(event, 'ref21 ref56'); return false;" href="javascript:void(0);" class="ref ref21 ref56">(21,56)</a> Both SK-575 and olaparib potently inhibit HR-deficient and HR-proficient cell growth with IC<sub>50</sub> values at the micromolar level. However, SK-575 is more potent than olaparib, indicating that the potency of cell growth inhibition is due in part to their ability to degrade PARP1. In addition, we evaluated the ability to potentiate the cytotoxic effect of SK-575 by the alkylating agent methyl methanesulfonate (MMS). After a growth-inhibition curve for SW620 cells was established, the effectiveness of MMS was potentiated in a dose-dependent manner by adding PARP1 degrader at the concentration of nanomolar (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D). Moreover, we evaluated the preliminary safety of SK-575 with six normal cell lines (L-O2, HEK-293, NCM460, HUV-EC, MCF-10A, and H9c2), and the results are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Figure S6</a>. Our data showed that the toxicity of SK-575 to these cells is acceptable, similar to olaparib.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Highly potent and broad applicability of SK-575. (A) DC<sub>50</sub> of SK-575 for PARP1 in different cancer cells. Data are the average of two biologically independent experiments. (B) PARP1 activity assay. (C) IC<sub>50</sub> values of SK-575 and olaparib. Cells were treated with indicated doses for 7 days. Cell viability was determined by the CCK-8 assay, and the data were calculated using GraphPad Prism 7 software. (D) Potentiation of MMS cell killing of cultured SW620 cells in combination with SK-575. Increasing concentrations of MMS were coincubated with or without the PARP1 degrader at single concentrations ranging from 3 to 300 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Because DNA double-strand breaks induce the phosphorylation of histone H2AX (γH2AX) in the chromatin flanking the break site, the DNA damage levels can be monitored by estimating the formation of γH2AX.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> We evaluated the ability of SK-575 and olaparib to induce the formation of γH2AX foci in the MDA-MB-436 (BRCA1<sup>–/–</sup>) and Capan-1 (BRCA2<sup>–/–</sup>) cell lines by the western blotting assay. We found that SK-575 effectively induced the formation of γH2AX in these two cells in a dose-dependent manner. In contrast, olaparib exhibited a weaker ability than SK-575 at the same concentration (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Western blotting analysis of γH2AX proteins in MDA-MB-436 (A) and Capan-1 (B) cells, with β-tubulin used as the loading control. Cells were treated with PARP1 inhibitor (olaparib) or PARP1 degrader (SK-575) at indicated concentrations, followed by a 24 h treatment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> PK Evaluation of SK-575 in Mice</h3><div class="NLM_p">To assess the antitumor activity of SK-575 in vivo, we first performed a pharmacokinetic (PK) analysis in mice and obtained the data shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A. In this experiment, a single dose of SK-575 at 25 mg/kg intraperitoneally administered, followed by plasma level quantitation revealed a maximum concentration of over 2.1 μM at 0.25 h postinjection, indicating that SK-575 is absorbed rapidly. In addition, our data showed that SK-575 with a half-life of 3.08 h (<i>n</i> = 3) had concentrations of 1.83 μM, 182 nM, and 95 nM in plasma at 1, 4, and 8 h timepoints, respectively. Importantly, the plasma concentration remained above 10 nM, a concentration needed for effective PARP1 degradation shown in our in vitro experiments at 24 h postinjection. Taken together, our PK data indicate that a single dose of SK-575 at 25 mg/kg achieved sufficient exposure in plasma for more than 24 h.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. In vivo evaluation of the PARP1 degrader SK-575. (A) PK study of SK-575 in mice (<i>n</i> = 3 per group). (B) PD study of SK-575 in the SW620 tumor tissue in mice. The treatment group mice-bearing xenograft SW620 tumors were treated with a single dose of SK-575 (25 mg/kg, ip). Tumor tissues were harvested at the indicated timepoints for the western blotting assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> PD Evaluation of SK-575 in SW620 Xenograft Tumor Tissues in Mice</h3><div class="NLM_p last">Based on these favorable PK properties, we evaluated the ability to degrade the PARP1 protein of SK-575 in the SW620 xenograft tumor tissue in mice. Once the tumors had reached 200–300 mm<sup>3</sup>, the mice were randomly assigned to the vehicle control group and the PROTAC group. The mice were administered a single dose of SK-575 at 25 mg/kg intraperitoneal injection and were sacrificed at 1, 3, 6, and 24 h timepoints after treatment. The mice treated with vehicle control were sacrificed at the 3 h timepoint. Our western blotting data (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>B) showed that a single dose of SK-575 at 25 mg/kg is very effective in reducing the level of PARP1 protein in the xenograft tumor tissue. The degradation of PARP1 protein started at 1 h after treatment and reached the maximum effect at 6 h, indicating a rapid degradation of PARP1 in vivo when compared to the vehicle. Our pharmacodynamic (PD) data thus demonstrate that a single dose of SK-575 at 25 mg/kg is highly effective in inducing near-complete elimination of PARP1 protein in tumor tissues in mice with the effect persisting for 24 h.</div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Antitumor Effect of SK-575 Administration in Xenograft Tumor Models</h3><div class="NLM_p">On the basis of its impressive PK and PD data, we evaluated the efficacy of SK-575 in three xenograft models in mice when used as a single agent or in combination with cytotoxic agents at different doses and schedules.</div><div class="NLM_p">In the beginning, to assess the in vivo antitumor effects of SK-575 used as a single agent, we treated nude mice bearing established subcutaneous Capan-1 tumor xenograft with SK-575 and obtained the results shown in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A. Capan-1 is a human pancreatic cancer cell line with BRCA2 deficiency.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Olaparib effectively retarded tumor growth over the control, but SK-575 can achieve much similar antitumor activity to olaparib at a lower dosage and with fewer times of administration. At the lower dose of 25 mg/kg once daily, the administration of SK-575 five times a week for 3 weeks intraperitoneally had a similar effect on tumor growth with olaparib (100 mg/kg, oral, daily dosing). SK-575 at these doses (25 and 50 mg/kg) were well tolerated, with no mice lethality or significant weight loss observed during the treatment time (Figure S7 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Supporting Information</a>).</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. (A) Antitumor efficacy of SK-575 in BRCA2-mutated Capan-1 xenografts (<i>n</i> = 6). (B) Antitumor efficacy of SK-575 in combination with TMZ in SW620 xenografts (<i>n</i> = 6). Nude mice were dosed once daily for 5 consecutive days (days 0–4 are indicated by a short line). (C) Antitumor efficacy of SK-575 in combination with cisplatin in BRCA2-mutated Capan-1 xenografts (<i>n</i> = 6). Nude mice were dosed with SK-575 once daily for 5 consecutive days (days 0–4 are indicated by a short line) and cisplatin was dosed intraperitoneally at 6 mg/kg on the 2 day timepoint. The efficacy of the treatment is expressed as mean ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We observed a clear additive effect of combining the PARP1 degrader SK-575 with MMS in SW620 cell lines in vitro (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D). Thus, to assess the antitumor activity of SK-575 in combination with temozolomide (TMZ) in vivo, we used a nude mice colorectal SW620 xenograft model to compare the antitumor activity of TMZ (at 50 mg/kg for 5 days, po, once daily) either alone or in combination with SK-575 (at 5 and 10 mg/kg for 5 days, ip, once daily) or olaparib (at 10 mg/kg for 5 days, po, once daily) as the reference standard (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>B). This dose and schedule of olaparib had previously demonstrated potentiation of TMZ antitumor activity.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Our data showed that TMZ alone can suppress tumor growth and its efficacy improved when combined with SK-575 in a dose-dependent manner. At a high dose of 10 mg/kg combination with TMZ for 5 consecutive days, SK-575 significantly inhibited tumor regrowth compared with TMZ alone (<i>P</i> < 0.01), similar to the results of olaparib (<i>P</i> < 0.01). We examined combinatorial toxicity through body weight loss and the results are shown in Figure S7 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Supporting Information</a>. Our data indicated that the TMZ-alone group was well tolerated with a maximum mean body weight loss of 9% at the 6 day timepoint and recovered quickly after treatment. However, the body weight loss was greater in the two combination groups of 10 mg/kg of SK-575 and olaparib, with a maximum mean body weight loss of 15% at the 9 day timepoint in these two groups and full recovery within a week. There was no animal lethality during the whole experiment. Overall, our combination therapy of PARP1 degrader was well-tolerated at these doses and schedules. It is worth noting that this is the first preclinical example of PROTAC compounds combined with other drugs for tumor treatment in vivo, which would provide favorable evidence for the clinical application of PROTAC compounds.</div><div class="NLM_p last">We next examined the additive effect of SK-575 when combined with cisplatin in vivo. Nude mice bearing Capan-1 tumor xenografts were treated with SK-575 (at 25 and 50 mg/kg for 5 days, ip, once daily) or olaparib (at 10 mg/kg for 5 days, po, once daily) in combination with a 6 mg/kg intraperitoneal injection of cisplatin at the 2 day timepoint (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>C). Nude mice treated with cisplatin alone had antitumor activity, and the efficacy improved when combined with SK-575 in a dose-dependent manner. Significant potentiation was observed as early as day 16 (<i>P</i> = 0.027) for the combination groups of SK-575 at 50 mg/kg and continued to differentiate from the cisplatin-alone group to day 32 (<i>P</i> < 0.01). However, we did note that the combination regimens involving cisplatin resulted in a maximum average body weight loss of 9 and 12% at the 10 day timepoint (Figure S7 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Supporting Information</a>), which were observed for groups that contained SK-575 doses of 25 and 50 mg/kg, respectively. In the same experiment, the nude mice treated with olaparib and cisplatin also showed combinatorial toxicity, with a maximum body weight loss of 11%. Mice recovered their body weight after a week of maximum body weight loss, and no mice lethality was observed.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64048" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64048" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, by linking the known PARP1 inhibitor olaparib to the E3 ligase ligand (e.g., thalidomide and lenalidomide), we have designed and synthesized a series of PARP1 degraders. Through extensive optimization of the length and composition of the linker, we identified compound <b>18</b> (SK-575) as a promising PARP1 degrader that exhibits rapid, highly potent, and specific degradation of PARP1 protein in various cancer cells with DC<sub>50</sub> value at concentrations as low as picomolar with a 24 h treatment time. Furthermore, SK-575 can cause persistent depletion of PARP1 protein in MDA-MB-436 cells at 100 nM for 5 days treatment, and the degraded PARP1 protein induced by the pretreatment of SK-575 is rarely recovered after washing it for 24 h. SK-575 displays good potency against PARP1. Our data show that SK-575 can not only effectively inhibit the growth of a variety of tumor cells as a single agent, as exemplified by the MDA-MB-436 and Capan-1 cells, but also potentiate the cell killing by the alkylating agent MMS. Significantly, our PK and PD data indicate that a single dose of SK-575 achieved sufficient exposure in plasma for over 24 h and effectively induced PARP1 degradation in the SW620 xenograft tumor tissue with the effect persisting for >24 h. Furthermore, SK-575 significantly inhibited the tumor growth in vivo as a single-agent in HR-deficient xenograft models, and with synergistic sensitization in combination with cytotoxic agents (TMZ or cisplatin) at well-tolerated dose schedules. Collectively, SK-575 is the most potent and efficacious PARP1 degrader reported to date and warrants further evaluation as a potential new therapy for the treatment of diseases caused by PARP1 hyperactivation.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45784" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45784" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compounds <b>6–11</b> were synthesized according to the route shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Compound <b>35</b> was obtained by condensation of 3-hydroxyphthalic anhydride (<b>33</b>) and 3-aminopiperidine-2,6-dione hydrochloride (<b>34</b>). Compound <b>35</b> was alkylated with <i>tert</i>-butyl bromate to obtain <b>36</b>, which was hydrolyzed in trifluoroacetic acid (TFA) to result in intermediate <b>37</b>. Condensation of <b>38</b> obtained from a commercial source with <i>N</i>-Boc-piperazine in the presence of 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]-pyridinium 3-oxide hexafluorophosphate (HATU) and triethylamine in dimethylformamide (DMF) afforded compound <b>39</b>. Compounds <b>40a,b</b> were prepared with <b>39</b> and <i>N</i>-Boc-butylamine or <i>N</i>-Boc-hexamine using a similar procedure described above. After Boc-deprotection of <b>40a,b</b> with HCl in DCM, Compounds <b>6</b> and <b>7</b> were obtained by condensation of <b>40a,b</b> with <b>37</b> using a similar method as used for <b>39</b>. Compounds <b>8–11</b> were prepared from <b>40a,b</b> by the same procedure as used for the synthesis of <b>6</b>.</div><figure id="sch1" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>6–11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) TEA, toluene, reflux; (b) <i>tert</i>-butyl bromoacetate, KI, KHCO<sub>3</sub>, DMF, 60 °C; (c) TFA/DCM, rt; (d) <i>N</i>-Boc-piperazine, HATU, TEA, DMF, rt; (e) HCl, EtOH, rt; (f) HATU, DIPEA, DMF, rt; (g) HCl, DCM/MeOH, rt.</p></p></figure><div class="NLM_p">Synthesis of compounds <b>12</b> and <b>13</b> is shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Condensation of 3-fluorophthalic anhydride (<b>42a</b>) with <b>34</b> afforded intermediate <b>43a</b>. Compound <b>44</b> was synthesized from <b>43a</b> by the substitution reaction. <i>tert</i>-Butyl in <b>44</b> was removed in TFA/DCM to obtain compound <b>45</b>. Compounds <b>12</b> and <b>13</b> were prepared from <b>41a,b</b> by the same procedure as used for the synthesis of <b>6</b>.</div><figure id="sch2" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>12–13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) NaOAc, AcOH, reflux; (b) <i>tert</i>-butyl glycinate, DIPEA, NMP, 90 °C; (c) TFA, DCM, rt; (d) HCl, DCM/MeOH, rt; (e) HATU, DIPEA, DMF, rt.</p></p></figure><div class="NLM_p">Synthesis of PARP1 degraders <b>14–25</b> is shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Compounds <b>46a–f</b> were prepared by a substitution reaction of <b>43a</b> with different lengths of mono-Boc-protected alkyl diamines. Intermediates <b>47a–g</b> were synthesized by an amide condensation reaction with <b>39</b> and different lengths of alkyl dicarboxylic acid. Condensation of <b>46a–f</b> with <b>47a–g</b> using previously described methods generated PARP1 degraders <b>14–25</b>.</div><figure id="sch3" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>14–25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) DIPEA, NMP, 90 °C; (b) HBTU, DIPEA, DMF, rt; (c) HCl, DCM, MeOH, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, intermediate <b>49</b> was obtained from <b>42b</b> through two steps by a process similar to that used for the synthesis of <b>44</b> from <b>42a</b>. Compound <b>26</b> was prepared from <b>47e</b> using the method used in the preparation of <b>14–25</b>.</div><figure id="sch4" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0016.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compound <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) NaOAc, AcOH, reflux; (b) <i>N</i>-Boc-ethylenediamine, DIPEA, NMP, 90 °C; (c) HCl, DCM, MeOH, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><div class="NLM_p">The synthesis of compound <b>27</b> is shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Alkylation of lenalidomide with <i>N</i>-Boc-2-aminoacetaldehyde (<b>49</b>) afforded intermediate <b>50</b>. Condensation of <b>47e</b> with <b>50</b> using the previously described procedure resulted in compound <b>27</b>.</div><figure id="sch5" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0017.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) NaBH<sub>3</sub>CN, MeOH, 50 °C; (b) HCl, DCM, MeOH, rt; (c) HATU, DIPEA, DMF, rt.</p></p></figure><div class="NLM_p">The synthesis of compound <b>28</b> is shown in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>. Compound <b>43a</b> was methylated with MeI to obtain <b>51</b>. Compound <b>52</b> was obtained by a substitution reaction of <b>51</b> with <i>N</i>-Boc-ethylenediamine. Compound <b>28</b> was prepared to start from <b>47e</b> in the same method as used for <b>14–25</b>.</div><figure id="sch6" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0018.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>28</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C; (b) DIPEA, NMP, 90 °C; (c) HCl, DCM, MeOH, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><div class="NLM_p">Finally, compounds <b>29–32</b> were synthesized according to the route shown in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>. It follows the published procedure for the synthesis of intermediate compounds <b>53–56</b>, shown in Scheme S1 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">Supporting Information</a>. Condensation of <b>47e</b> with <b>53–56</b> performed using previously described methods generated PARP1 degraders <b>29–32</b>.</div><figure id="sch7" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0019.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compounds <b>29–32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) HCl, DCM, MeOH, rt; (b) HATU, DIPEA, DMF, rt.</p></p></figure></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11817" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11817" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Chemistry: General Information</h3><div class="NLM_p last">Unless otherwise specified, all reactions were carried out under the atmosphere, and common reagents or materials were obtained from commercial sources and used without further purification. Compound <b>3</b> (iRucaparib-AP6) was purchased from the commercial source MedChemExpress. Olaparib, TMZ, and cisplatin were purchased from the commercial source Adamas-beta. Flash column chromatography was performed using silica gel from Qingdao Haiyang, and its specification was 300–400 mesh. NMR spectra were recorded on a Bruker AMX 400 spectrometer and calibrated using TMS or residual deuterated solvent as an internal reference (CDCl<sub>3</sub>: 7.26 ppm for <sup>1</sup>H NMR and 77.16 ppm for <sup>13</sup>C NMR; DMSO-<i>d</i><sub>6</sub>: 2.50 ppm for <sup>1</sup>H NMR and 39.50 ppm for <sup>13</sup>C NMR; CD<sub>3</sub>OD: 3.31 ppm for <sup>1</sup>H NMR and 49.00 ppm for <sup>13</sup>C NMR). In the reported spectral data, all NMR spectra are reported in ppm, and the format (δ) chemical shift (multiplicity, J values in Hz, integration) is described by the following abbreviations: s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, and br = broadening of peaks, which was often observed because of the slow rotation of the amide bonds. HRMS spectra were recorded on a Waters Q-TOF Premier. The final compounds were all purified using a C18 reverse-phase preparative high-performance liquid chromatography (HPLC) column with solvent A [0.1% TFA in water] and solvent B (0.1% TFA in MeCN) as eluents. The purity of all the final compounds was confirmed to be >95% by HPLC.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)-<i>N</i>-(4-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-4-oxobutyl)acetamide (<b>6</b>)</h3><div class="NLM_p">Compound <b>6</b> was prepared following the published procedure for the synthesis of compound <b>37</b>.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> In a round-bottom flask, 3-aminopiperidine-2,6-dione hydrochloride <b>33</b> (1.64 g, 10.0 mmol, 1.0 equiv), 3-hydroxyphthalic anhydride <b>34</b> (1.64 g, 10.0 mmol, 1.0 equiv), and triethylamine (1.56 mL, 12.0 mmol, 1.2 equiv) were dissolved in toluene (60 mL). The resulting reaction mixture was heated to reflux in an oil bath for 12 h. After cooling to ambient temperature, evaporation of most of the solvent afforded a crude product, which was purified by column chromatography with the DCM/EtOAc system to obtain <b>35</b> as a white solid (2.41 g, 88% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 11.16 (s, 1H), 11.08 (s, 1H), 7.65 (t, <i>J</i> = 7.6 Hz, 1H), 7.32 (d, <i>J</i> = 7.2 Hz, 1H), 7.25 (d, <i>J</i> = 8.4 Hz, 1H), 5.07 (dd, <i>J</i> = 12.8 Hz, <i>J</i> = 5.2 Hz, 1H), 2.93–2.84 (m, 1H), 2.61–2.46 (m, 3H), 2.05–2.01 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 172.75, 169.96, 166.98, 165.78, 155.43, 136.35, 133.12, 123.52, 114.34, 114.26, 48.62, 30.93, 22.01.</div><div class="NLM_p">In a round-bottom flask, compound <b>35</b> (1.9 g, 7.0 mmol, 1.0 equiv) was dissolved in DMF (20 mL). KI (116 mg, 0.7 mmol, 0.1 equiv) and KHCO<sub>3</sub> (1.05 g, 10.05 mmol, 1.5 equiv) were added to the stirred solution. Then, <i>tert</i>-butyl bromoacetate (1.24 ml, 8.4 mmol, 1.2 equiv) was added dropwise and the mixture was stirred at 60 °C overnight. EtOAc (200 mL) was added to the reaction mixture, washed with saturated brine, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtering and decompressing the solvent, the residue was purified by DCM/EtOAc to obtain <b>36</b> as a white solid (2.16 g, 85% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 8.19 (s, 1H), 7.66 (dd, <i>J</i> = 8.4, <i>J</i> = 7.4 Hz, 1H), 7.50 (d, <i>J</i> = 7.3 Hz, 1H), 7.10 (d, <i>J</i> = 8.4 Hz, 1H), 5.01–4.92 (m, 1H), 4.78 (s, 2H), 2.93–2.67 (m, 3H), 2.16–2.07 (m, 1H), 1.47 (s, 9H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ (ppm) 171.12, 168.09, 167.00, 166.96, 165.57, 155.65, 136.40, 134.04, 119.94, 117.71, 117.03, 83.24, 66.69, 49.31, 31.51, 28.16, 22.70.</div><div class="NLM_p">In a round-bottom flask, <b>36</b> (1.7 g, 4.4 mmol) was dissolved in 20 mL of TFA/DCM (1:1) and stirred at room temperature for 2 h. After concentration of the solvent, residue <b>37</b> obtained as a white solid was used for the following steps without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.22 (s, 1H), 11.11 (s, 1H), 7.80 (dd, <i>J</i> = 8.6, 7.3 Hz, 1H), 7.48 (d, <i>J</i> = 7.2 Hz, 1H), 7.40 (d, <i>J</i> = 8.5 Hz, 1H), 5.11 (dd, <i>J</i> = 12.9, <i>J</i> = 5.4 Hz, 1H), 4.99 (s, 2H), 2.90 (m, 1H), 2.69–2.53 (m, 2H), 2.05 (m, 1H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 173.24, 170.37, 169.96, 167.20, 165.63, 155.60, 137.22, 133.72, 120.35, 116.79, 116.22, 65.48, 49.26, 31.42, 22.44. HRMS (DART-TOF): calcd for C<sub>15</sub>H<sub>12</sub>N<sub>2</sub>NaO<sub>7</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 355.0542; found, 355.0541.</div><div class="NLM_p">In a round-bottom flask, compound <b>38</b> (4.45 g, 15 mmol, 1.0 equiv) and <i>N</i>-Boc-piperazine (3.35 g, 18 mmol, 1.2 equiv) were dissolved in DMF (100 mL) and the mixture was cooled to 0 °C. Triethylamine (4.2 mL, 30 mmol, 2.0 equiv) and HATU (6.80 g, 18 mmol, 1.2 equiv) were added sequentially to a stirred solution. The resulting mixture was stirred at room temperature for 5 h. Water (500 mL) was added to the reaction solution and stirred at 0 °C for 1 h. A large amount of white solid was obtained by suction filtration, and the filter cake was washed with a small amount of ice water and cold EtOAc to obtain the intermediate crude product (6.43 g, 92% yield).</div><div class="NLM_p">In a round-bottom flask, the intermediate obtained from the previous step (6.43 g, 13.8 mmol) was taken and dissolved in ethanol (50 mL) and then HCl (6 N, 6.9 mL) was added dropwise to the quickly stirred solution. The resulting mixture was stirred at room temperature for 3 h after dropping and then concentrated in vacuo. The pH of the solution was adjusted to >10 with aqueous ammonium hydroxide (4 N). The resulting mixture was extracted with DCM (3 × 100 mL), then the combined organic layers were washed with brine, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and evaporation, the crude residue was purified by flash column chromatography with hexane/EtOAc to obtain <b>39</b> as a white solid (4.45 g, 88% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 12.58 (s, 1H), 8.26 (d, <i>J</i> = 7.7 Hz, 1H), 7.96 (d, <i>J</i> = 7.9 Hz, 1H), 7.88 (t, <i>J</i> = 7.2 Hz, 1H), 7.82 (t, <i>J</i> = 7.4 Hz, 1H), 7.44–7.37 (m, 1H), 7.31 (dd, <i>J</i> = 6.4, <i>J</i> = 1.7 Hz, 1H), 7.20 (t, <i>J</i> = 9.0 Hz, 1H), 5.75 (s, 1H), 4.32 (s, 2H), 3.53 (s, 2H), 3.06 (s, 2H), 2.70 (t, <i>J</i> = 4.8 Hz, 2H), 2.56 (t, <i>J</i> = 4.7 Hz, 2H). HRMS (DART-TOF): calcd for C<sub>20</sub>H<sub>20</sub>FN<sub>4</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i>, 367.1570; found, 367.1565.</div><div class="NLM_p">In a round-bottom flask, compound <b>39</b> (732 mg, 2.0 mmol, 1.0 equiv) and <i>N</i>-Boc-butylamine (406 mg, 2.0 mmol, 1.0 equiv) were dissolved in DMF (10 mL) and the mixture was cooled to 0 °C. DIPEA (0.66 mL, 4.0 mmol, 2.0 equiv) and HATU (836 mg, 2.2 mmol, 1.1 equiv) were added sequentially. The resulting mixture was stirred at room temperature for 3 h, then brine (100 mL) was added, and extracted with EtOAc (3 × 100 mL). The combined organic layers were washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and evaporation, the crude residue was purified by flash column chromatography with DCM/MeOH (10:1) to obtain <b>40a</b> as a white solid (904 mg, 82% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.66 (br, 1H), 8.49–8.37 (m, 1H), 7.71 (dd, <i>J</i> = 10.5, <i>J</i> = 4.0 Hz, 3H), 7.40–7.22 (m, 2H), 7.00 (dd, <i>J</i> = 12.3, <i>J</i> = 5.9 Hz, 1H), 4.90 (s, 1H), 4.27 (s, 2H), 3.87–3.21 (m, 8H), 3.14 (m, 2H), 2.39 (t, <i>J</i> = 7.1 Hz, 1H), 2.32 (t, <i>J</i> = 6.9 Hz, 1H), 1.88–1.74 (m, 2H), 1.39 (m, 9H). HRMS (DART-TOF): calcd for C<sub>29</sub>H<sub>34</sub>FN<sub>5</sub>NaO<sub>5</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 574.2442; found, 574.2437.</div><div class="NLM_p last">In a round-bottom flask, compound <b>40a</b> (121.2 mg, 0.22 mmol, 1.1 equiv) was dissolved in DCM (10 mL) and MeOH (2 mL) and then HCl (1 mL, 4 M in 1,4-dioxane) was added. After stirring for 2 h, the solvent was evaporated and the residue was used in the next step without further purification. The intermediate obtained from the previous step was dissolved in DMF (10 mL). After cooling to 0 °C, compound <b>37</b> (66.4 mg, 0.2 mmol, 1.0 equiv), DIPEA (99 μL, 0.6 mmol, 3.0 equiv), and HATU (76 mg, 0.2 mmol, 1.0 equiv) were added sequentially to the stirred solution. The resulting mixture was stirred at room temperature for 1 h, then brine (50 mL) was added, and extracted with EtOAc (3 × 50 mL). The combined organic layers were washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and evaporation, the crude residue was purified by flash column chromatography with DCM/MeOH (10:1) to obtain <b>6</b> as a white powder (108.6 mg, 71% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.26 (br, 1H), 9.66 (br, 1H), 8.41 (dd, <i>J</i> = 6.2, <i>J</i> = 1.8 Hz, 1H), 7.78–7.65 (m, 4H), 7.65–7.55 (m, 1H), 7.54–7.46 (m, 1H), 7.37–7.23 (m, 2H), 7.19 (d, <i>J</i> = 8.4 Hz, 1H), 7.01 (t, <i>J</i> = 8.5 Hz, 1H), 4.98 (m, 1H), 4.64 (s, 2H), 4.26 (s, 2H), 3.85–3.15 (m, 10H), 2.92–2.70 (m, 3H), 2.49–2.31 (m, 2H), 2.13 (s, 1H), 1.98–1.84 (m, 2H). HRMS (DART-TOF): calcd for C<sub>39</sub>H<sub>36</sub>FN<sub>7</sub>NaO<sub>9</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 788.2456; found, 788.2458. Purity, 97.2%.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)-<i>N</i>-(6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexyl)acetamide (<b>7</b>)</h3><div class="NLM_p last">Compound <b>7</b> (55.7 mg, 70% yield) was obtained as a white powder using the procedure for the synthesis of compound <b>6</b> with <b>40b</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.13 (br, 1H), 9.85 (br, 1H), 8.48–8.40 (m, 1H), 7.81–7.65 (m, 4H), 7.62–7.48 (m, 2H), 7.35–7.26 (m, 2H), 7.20 (d, <i>J</i> = 8.4 Hz, 1H), 7.02 (t, <i>J</i> = 8.7 Hz, 1H), 5.08–4.87 (m, 1H), 4.64 (q, <i>J</i> = 14.2 Hz, 2H), 4.27 (s, 2H), 3.85–3.15 (m, 10H), 2.93–2.67 (m, 3H), 2.43–2.25 (m, 2H), 2.19–2.09 (m, 1H), 1.72–1.52 (m, 4H), 1.45–1.31 (m, 2H). HRMS (DART-TOF): calcd for C<sub>41</sub>H<sub>40</sub>FN<sub>7</sub>NaO<sub>9</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 816.2769; found, 816.2766. Purity, 96.3%.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 4-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)-<i>N</i>-(4-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-4-oxobutyl)butanamide (<b>8</b>)</h3><div class="NLM_p last">Compound <b>8</b> (62.9 mg, 74% yield) was obtained as a white powder using the procedure employed for the synthesis of compound <b>6</b> with <b>41a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 12.58 (s, 1H), 11.11 (s, 1H), 8.26 (dd, <i>J</i> = 7.8, <i>J</i> = 1.1 Hz, 1H), 7.98 (m, 2H), 7.93–7.85 (m, 1H), 7.86–7.74 (m, 3H), 7.49 (d, <i>J</i> = 7.1 Hz, 1H), 7.46–7.33 (m, 3H), 7.23 (t, <i>J</i> = 9.0 Hz, 1H), 5.12 (dd, <i>J</i> = 12.9, <i>J</i> = 5.4 Hz, 1H), 4.76 (d, <i>J</i> = 8.6 Hz, 2H), 4.33 (s, 2H), 3.69–2.98 (m, 12H), 2.95–2.82 (m, 1H), 2.66–2.47 (m, 2H), 2.34 (t, <i>J</i> = 6.7 Hz, 1H), 2.28 (t, <i>J</i> = 7.1 Hz, 1H), 2.14–1.95 (m, 3H), 1.72–1.54 (m, 4H). HRMS (DART-TOF): calcd for C<sub>43</sub>H<sub>43</sub>FN<sub>8</sub>NaO<sub>10</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 873.2984; found, 873.2984. Purity, 98.1%.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 6-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)-<i>N</i>-(4-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-4-oxobutyl)hexanamide (<b>9</b>)</h3><div class="NLM_p last">Compound <b>9</b> (71.9 mg, 82% yield) was obtained as a white powder using the procedure employed for the synthesis of compound <b>6</b> with <b>41b</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 10.92 (br, 1H), 10.30 (br, 1H), 8.48–8.39 (m, 1H), 7.82–7.66 (m, 4H), 7.61–7.49 (m, 2H), 7.36–7.24 (m, 3H), 7.20 (d, <i>J</i> = 8.4 Hz, 1H), 7.03 (t, <i>J</i> = 8.8 Hz, 1H), 6.44–6.33 (m, 1H), 4.99 (dd, <i>J</i> = 11.9, <i>J</i> = 5.6 Hz, 1H), 4.72–4.56 (m, 2H), 4.27 (s, 2H), 3.84–3.13 (m, 12H), 2.92–2.69 (m, 3H), 2.43 (t, <i>J</i> = 6.2 Hz, 1H), 2.35 (t, <i>J</i> = 6.3 Hz, 1H), 2.26–2.05 (m, 3H), 1.67–1.50 (m, 4H), 1.45–1.32 (m, 3H). HRMS (DART-TOF): calcd for C<sub>45</sub>H<sub>47</sub>FN<sub>8</sub>NaO<sub>10</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 901.3297; found, 901.3294. Purity, 96.2%.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 4-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)-<i>N</i>-(6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexyl)butanamide (<b>10</b>)</h3><div class="NLM_p last">Compound <b>10</b> (68.5 mg, 78% yield) was obtained as a white powder using the procedure employed for the synthesis of compound <b>6</b> with <b>41c</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.23 (br, 1H), 9.75 (s, 1H), 8.42 (dd, <i>J</i> = 6.6, <i>J</i> = 2.5 Hz, 1H), 7.80–7.62 (m, 5H), 7.51 (d, <i>J</i> = 7.3 Hz, 1H), 7.35–7.24 (m, 2H), 7.21 (d, <i>J</i> = 8.4 Hz, 1H), 7.02 (t, <i>J</i> = 8.7 Hz, 1H), 6.61–6.42 (m, 1H), 5.03–4.89 (m, 1H), 4.65 (s, 2H), 4.27 (s, 2H), 3.82–3.11 (m, 12H), 2.91–2.71 (m, 3H), 2.34 (t, <i>J</i> = 7.0 Hz, 1H), 2.26 (t, <i>J</i> = 7.1 Hz, 1H), 2.21 (t, <i>J</i> = 7.2 Hz, 2H), 2.17–2.14 (m, 1H), 1.93–1.78 (m, 2H), 1.67–1.54 (m, 2H), 1.53–1.41 (m, 2H), 1.39–1.26 (m, 3H). HRMS (DART-TOF): calcd for C<sub>45</sub>H<sub>47</sub>FN<sub>8</sub>NaO<sub>10</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 901.3297; found, 901.3296. Purity, 95.5%.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 6-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)-<i>N</i>-(6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexyl)hexanamide (<b>11</b>)</h3><div class="NLM_p last">Compound <b>11</b> (68.8 mg, 76% yield) was obtained as a white powder using the procedure employed for the synthesis of compound <b>6</b> with <b>41d</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.15 (br, 1H), 10.08 (s, 1H), 8.50–8.38 (m, 1H), 7.81–7.66 (m, 4H), 7.61–7.47 (m, 2H), 7.37–7.26 (m, 3H), 7.20 (d, <i>J</i> = 8.4 Hz, 1H), 7.03 (t, <i>J</i> = 8.8 Hz, 1H), 6.05 (s, 1H), 5.00 (dd, <i>J</i> = 11.7, <i>J</i> = 5.9 Hz, 1H), 4.64 (q, <i>J</i> = 14.2 Hz, 2H), 4.28 (s, 2H), 3.86–3.11 (m, 13H), 2.95–2.66 (m, 3H), 2.35 (t, <i>J</i> = 6.7 Hz, 1H), 2.35 (t, <i>J</i> = 7.1 Hz, 1H), 2.24–2.09 (m, 3H), 1.70–1.53 (m, 6H), 1.54–1.42 (m, 2H), 1.43–1.26 (m, 4H). HRMS (DART-TOF): calcd for C<sub>47</sub>H<sub>51</sub>FN<sub>8</sub>NaO<sub>10</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 929.3610; found, 929.3614. Purity, 98.1%.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 4-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)acetamido)-<i>N</i>-(6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexyl)butanamide (<b>12</b>)</h3><div class="NLM_p">In a round-bottom flask, 3-fluorophthalic anhydride <b>42a</b> (3.32 g, 20 mmol, 1.0 equiv), 3-aminopiperidine-2,6-dione hydrochloride <b>34</b> (3.29 g, 20 mmol, 1.0 equiv), and sodium acetate (1.97 g, 24 mmol, 1.2 equiv) were dissolved in AcOH (100 mL). The resulting mixture was heated to reflux at 120 °C overnight. After cooling to room temperature, the solvent was removed in vacuo and the residue was poured into water (200 mL) and extracted with EtOAc (5 × 100 mL). The combined organic layers were washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and evaporation, the crude residue was pulped with DCM and then filtered to obtain a white powder <b>43a</b> (4.52 g, 82% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 11.15 (s, 1H), 7.98–7.93 (m, 1H), 7.80–7.72 (m, 2H), 5.17 (dd, <i>J</i> = 13.2 Hz, <i>J</i> = 5.2 Hz, 1H), 2.95–2.86 (m, 1H), 2.64–2.47 (m, 2H), 2.10–2.06 (m, 1H).</div><div class="NLM_p">In a round-bottom flask, compound <b>43a</b> (552 mg, 2 mmol, 1.0 equiv), <i>tert</i>-butyl glycinate (324 mg, 2.4 mmol, 1.2 equiv), and DIPEA (660 μL, 4 mmol, 2.0 equiv) were dissolved in NMP (5 mL). The resulting mixture was heated to reflux at 90 °C overnight. After cooling to room temperature, the resulting mixture was poured into water (50 mL) and extracted with EtOAc (3 × 100 mL). The combined organic layers were washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and evaporation, the crude residue was purified by flash column chromatography with petroleum ether/EtOAc to obtain <b>44</b> as a yellow powder (503 mg, 65% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 7.57 (dd, <i>J</i> = 8.5, <i>J</i> = 7.1 Hz, 1H), 7.12 (d, <i>J</i> = 7.1 Hz, 1H), 6.94 (d, <i>J</i> = 8.5 Hz, 1H), 5.51 (s, 2H), 5.09 (m, 1H), 2.94–2.66 (m, 3H), 2.14 (m, 1H), 1.51 (s, 9H). HRMS (DART-TOF): calcd for C<sub>19</sub>H<sub>21</sub>N<sub>3</sub>NaO<sub>6</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 410.1323; found, 410.1326.</div><div class="NLM_p">In a round-bottom flask, <b>44</b> (503 mg, 1.3 mmol) was dissolved in 20 ml TFA/DCM (1:1) and stirred at room temperature for 2 h. After concentration of the solvent, residue <b>45</b> obtained as a yellow solid was used for the following steps without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 12.75 (s, 1H), 11.01 (s, 1H), 7.56 (t, <i>J</i> = 8.2 Hz, 1H), 7.18 (d, <i>J</i> = 6.80 Hz, 1H), 6.89 (d, <i>J</i> = 8.4 Hz, 1H), 6.86 (t, J = 5.6 Hz, 1H), 5.04 (dd, <i>J</i> = 13.2 Hz, <i>J</i> = 5.8 Hz, 1H), 4.12 (d, <i>J</i> = 5.2 Hz, 2H), 2.84–2.75 (m, 1H), 2.73–2.59 (m, 2H), 2.09–2.07 (m, 1H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 172.3, 170.2, 169.8, 169.3, 167.1, 145.5, 137.6, 132.1, 119.2, 110.8, 110.1, 61.0, 49.9, 31.7, 21.5. HRMS (DART-TOF): calcd for C<sub>15</sub>H<sub>13</sub>N<sub>3</sub>NaO<sub>6</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 354.0702; found, 354.0768.</div><div class="NLM_p last">Compound <b>12</b> (64.0 mg, 73% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>6</b> with <b>41c</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 10.90 (s, 1H), 9.31 (s, 1H), 8.48–8.36 (m, 1H), 7.84–7.66 (m, 3H), 7.49 (t, <i>J</i> = 7.5 Hz, 1H), 7.38–7.28 (m, 2H), 7.14 (d, <i>J</i> = 6.8 Hz, 1H), 7.05 (t, <i>J</i> = 8.7 Hz, 1H), 6.85–6.69 (m, 2H), 6.36–6.24 (m, 1H), 5.02–4.86 (m, 1H), 4.28 (s, 2H), 3.94 (d, <i>J</i> = 5.0 Hz, 2H), 3.80–3.11 (m, 12H), 2.95–2.69 (m, 3H), 2.31 (dt, <i>J</i> = 12.6, <i>J</i> = 6.6 Hz, 2H), 2.19–2.04 (m, 3H), 1.74–1.86 (m, 2H), 1.66–1.56 (m, 2H), 1.43–1.53 (m, 2H), 1.38–1.29 (m, 2H). HRMS (DART-TOF): calcd for C<sub>45</sub>H<sub>48</sub>FN<sub>9</sub>NaO<sub>9</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 900.3451; found, 900.3461. Purity, 98.7%.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 6-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)acetamido)-<i>N</i>-(6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexyl)hexanamide (<b>13</b>)</h3><div class="NLM_p last">Compound <b>13</b> (73.3 mg, 81% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>6</b> with <b>41d</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 10.96 (s, 1H), 9.39 (s, 1H), 8.49–8.39 (m, 1H), 7.83–7.67 (m, 3H), 7.50 (t, <i>J</i> = 7.8 Hz, 1H), 7.38–7.27 (m, 2H), 7.15 (d, <i>J</i> = 7.1 Hz, 1H), 7.04 (t, <i>J</i> = 8.6 Hz, 1H), 6.97–6.86 (m, 1H), 6.80 (d, <i>J</i> = 8.5 Hz, 1H), 6.74 (t, <i>J</i> = 5.9 Hz, 1H), 6.04–5.91 (m, 1H), 5.05–4.88 (m, 1H), 4.28 (s, 2H), 3.94 (d, <i>J</i> = 5.9 Hz, 2H), 3.82–3.14 (m, 12H), 2.93–2.67 (m, 3H), 2.36 (t, <i>J</i> = 6.7 Hz, 1H), 2.28 (t, <i>J</i> = 7.1 Hz, 1H), 2.18–2.01 (m, 3H), 1.68–1.40 (m, 8H), 1.37–1.17 (m, 4H). HRMS (DART-TOF): calcd for C<sub>47</sub>H<sub>52</sub>FN<sub>9</sub>NaO<sub>9</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 928.3764; found, 928.3767. Purity, 95.9%.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i>-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)-6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexanamide (<b>14</b>)</h3><div class="NLM_p last">Compound <b>14</b> (39.6 mg, 50% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46a</b> and <b>47a</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.08 (br, 1H), 9.49 (br, 1H), 8.42 (d, <i>J</i> = 6.1 Hz, 1H), 7.86–7.62 (m, 3H), 7.44 (t, <i>J</i> = 7.7 Hz, 1H), 7.37–7.26 (m, 2H), 7.13 (s, 1H), 7.07–6.98 (m, 2H), 6.93 (d, <i>J</i> = 8.5 Hz, 1H), 6.47 (br, 1H), 4.92 (d, <i>J</i> = 5.8 Hz, 1H), 4.27 (s, 2H), 3.85–3.12 (m, 12H), 2.91–2.64 (m, 3H), 2.38 (s, 1H), 2.23 (s, 1H), 2.09 (m, 1H), 1.71–1.49 (m, 4H). HRMS (DART-TOF): calcd for C<sub>41</sub>H<sub>41</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 815.2924; found, 815.2933. Purity, 96.2%.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i>-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)-8-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-8-oxooctanamide (<b>15</b>)</h3><div class="NLM_p last">Compound <b>15</b> (46.9 mg, 57% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46a</b> and <b>47b</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.07 (br, 1H), 9.42 (s, 1H), 8.51–8.33 (m, 1H), 7.84–7.62 (m, 3H), 7.44 (t, <i>J</i> = 7.4 Hz, 1H), 7.37–7.26 (m, 2H), 7.04 (dd, <i>J</i> = 12.5, <i>J</i> = 7.8 Hz, 2H), 6.94 (d, <i>J</i> = 8.6 Hz, 1H), 6.51 (br, 1H), 6.40 (s, 1H), 5.01–4.84 (m, 1H), 4.27 (s, 2H), 3.85–3.16 (m, 12H), 2.91–2.67 (m, 3H), 2.33 (t, <i>J</i> = 7.1 Hz, 1H), 2.25 (t, <i>J</i> = 6.8 Hz, 1H), 2.18–2.04 (m, 3H), 1.67–1.46 (m, 4H), 1.35–1.19 (m, 6H). HRMS (DART-TOF): calcd for C<sub>43</sub>H<sub>45</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 843.3237; found, 843.3234. Purity, 97.3%.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> <i>N</i>-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)-10-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-10-oxodecanamide (<b>16</b>)</h3><div class="NLM_p last">Compound <b>16</b> (46.6 mg, 55% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46a</b> and <b>47c</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.03 (br, 1H), 9.27 (br, 1H), 8.44 (d, <i>J</i> = 6.4 Hz, 1H), 7.83–7.62 (m, 3H), 7.46 (t, <i>J</i> = 7.7 Hz, 1H), 7.38–7.27 (m, 2H), 7.04 (m, 2H), 6.96 (d, <i>J</i> = 8.5 Hz, 1H), 6.40 (s, 2H), 5.00–4.84 (m, 1H), 4.27 (s, 2H), 3.88–3.09 (m, 12H), 2.90–2.67 (m, 3H), 2.29 (m, 2H), 2.18–2.04 (m, 3H), 1.67–1.46 (m, 4H), 1.36–1.15 (m, 8H). HRMS (DART-TOF): calcd for C<sub>45</sub>H<sub>49</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 871.3550; found, 871.3554. Purity, 98.5%.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> <i>N</i>-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)-11-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-11-oxoundecanamide (<b>17</b>)</h3><div class="NLM_p last">Compound <b>17</b> (46.6 mg, 55% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46a</b> and <b>47d</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.18 (br, 1H), 9.32 (br, 1H), 8.49–8.29 (m, 1H), 7.84–7.61 (m, 3H), 7.44 (t, <i>J</i> = 7.8 Hz, 1H), 7.38–7.26 (m, 2H), 7.03 (dd, <i>J</i> = 12.5, <i>J</i> = 7.8 Hz, 2H), 6.95 (t, <i>J</i> = 7.3 Hz, 1H), 6.50 (br, 1H), 6.39 (s, 1H), 4.93 (dd, <i>J</i> = 12.1, <i>J</i> = 5.4 Hz, 1H), 4.26 (s, 2H), 3.89–3.10 (m, 12H), 2.91–2.64 (m, 3H), 2.32 (t, <i>J</i> = 7.1 Hz, 1H), 2.25 (t, <i>J</i> = 6.9 Hz, 1H), 2.19–2.07 (m, 3H), 1.67–1.44 (m, 4H), 1.31–1.15 (m, 10H). HRMS (DART-TOF): calcd for C<sub>46</sub>H<sub>51</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 885.3706; found, 885.3706. Purity, 96.7%.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <i>N</i>-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)-12-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamide (<b>18</b>)</h3><div class="NLM_p">In a round-bottom flask, compound <b>43a</b> (2.76 g, 10 mmol, 1.0 equiv), <i>N</i>-Boc-ethylenediamine (1.76 g, 10.1 mmol, 1.1 equiv), and DIPEA (3.3 mL, 20 mmol, 2.0 equiv) were dissolved in NMP (50 mL). The resulting mixture was heated to reflux at 90 °C overnight. After cooling to room temperature, the resulting mixture was poured into water (200 mL) and extracted with EtOAc (3 × 200 mL). The combined organic layers were washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and evaporation, the crude residue was purified by flash column chromatography with petroleum ether/EtOAc to obtain intermediate <b>46a</b> as a yellow powder (2.74 g, 66% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 8.04 (s, 1H), 7.50 (dd, <i>J</i> = 8.5, <i>J</i> = 7.1 Hz, 1H), 7.12 (d, <i>J</i> = 7.1 Hz, 1H), 6.98 (d, <i>J</i> = 8.6 Hz, 1H), 6.39 (t, <i>J</i> = 6.0 Hz, 1H), 4.92 (dd, <i>J</i> = 12.1, <i>J</i> = 5.3 Hz, 1H), 4.82 (s, 1H), 3.45 (q, <i>J</i> = 5.9 Hz, 2H), 3.36 (q, <i>J</i> = 5.7 Hz, 2H), 2.92–2.68 (m, 3H), 2.13 (m, 1H), 1.45 (s, 9H). HRMS (DART-TOF): calcd for C<sub>20</sub>H<sub>24</sub>N<sub>4</sub>NaO<sub>6</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 439.1588; found, 439.1592.</div><div class="NLM_p">In a round-bottom flask, HBTU (418 mg, 1.1 mmol, 1.1 equiv), dodecanedioic acid (690 mg, 3.0 mmol, 3.0 equiv), and DIPEA (990 μL, 6 mmol, 6.0 equiv) were dissolved in DMF (10 mL) and then the mixture was stirred at room temperature for 5 min. Compound <b>39</b> (366 mg, 1.0 mmol, 1.0 equiv) was added to the mixture and the resulting mixture was stirred at room temperature for 16 h. The resulting mixture was poured into water (50 mL) and extracted with EtOAc (3 × 100 mL). The combined organic layers were washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and evaporation, the crude residue was purified by flash column chromatography with DCM/MeOH to obtain intermediate <b>47e</b> as a white solid (502.9 mg, 87% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.83 (br, 1H), 8.44 (d, <i>J</i> = 6.5 Hz, 1H), 7.74 (m, 3H), 7.31 (m, 2H), 7.02 (t, <i>J</i> = 8.8 Hz, 1H), 4.28 (s, 2H), 3.91–3.14 (m, 8H), 2.33 (m, 4H), 1.69–1.52 (m, 4H), 1.37–1.18 (m, 12H). HRMS (DART-TOF): calcd for C<sub>32</sub>H<sub>39</sub>FN<sub>4</sub>NaO<sub>5</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 601.2802; found, 601.2802.</div><div class="NLM_p last">In a round-bottom flask, compound <b>46a</b> (45.8 mg, 0.11 mmol, 1.1 equiv) was dissolved in DCM (10 mL) and MeOH (2 mL) and then HCl (1 mL, 4 M in 1,4-dioxane) was added. After stirring for 2 h, the solvent was evaporated and the residue was used in the next step without further purification. The intermediate obtained from the previous step, compound <b>47e</b> (57.8 mg, 0.1 mmol, 1.0 equiv) and DIPEA (33 μL, 0.2 mmol, 2.0 equiv) were dissolved in DMF (10 mL). After cooling to 0 °C, HATU (38 mg, 0.1 mmol, 1.0 equiv) was added sequentially to a stirred solution. The resulting mixture was stirred at room temperature for 1 h, then brine (50 mL) was added, and extracted with EtOAc (3 × 50 mL). The combined organic layers were washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and evaporation, the crude residue was purified by flash column chromatography with DCM/MeOH (10:1) to obtain <b>18</b> as a yellow powder (50.8 mg, 58% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.17 (br, 1H), 9.29 (s, 1H), 8.42 (d, <i>J</i> = 6.4 Hz, 1H), 7.85–7.61 (m, 3H), 7.44 (t, <i>J</i> = 7.7 Hz, 1H), 7.37–7.25 (m, 2H), 7.14–6.99 (m, 2H), 6.96 (d, <i>J</i> = 8.5 Hz, 1H), 6.47 (br, 1H), 6.39 (s, 1H), 4.96–4.82 (m, 1H), 4.26 (s, 2H), 3.86–3.10 (m, 12H), 2.90–2.65 (m, 3H), 2.33 (t, <i>J</i> = 6.7 Hz, 1H), 2.26 (t, <i>J</i> = 6.4 Hz, 1H), 2.17–2.06 (m, 3H), 1.67–1.45 (m, 4H), 1.33–1.09 (m, 12H). HRMS (DART-TOF): calcd for C<sub>47</sub>H<sub>53</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 899.3863; found, 899.3861. Purity, 96.3%.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i>-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)-13-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-13-oxotridecanamide (<b>19</b>)</h3><div class="NLM_p last">Compound <b>19</b> (56.9 mg, 64% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46a</b> and <b>47f</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.05 (br, 1H), 9.15 (br, 1H), 8.48–8.34 (m, 1H), 7.82–7.63 (m, 3H), 7.51–7.40 (m, 1H), 7.38–7.27 (m, 2H), 7.04 (m, 2H), 6.97 (d, <i>J</i> = 8.6 Hz, 1H), 6.38 (m, 2H), 4.93 (m, 1H), 4.27 (s, 2H), 3.88–3.17 (m, 12H), 2.92–2.67 (m, 3H), 2.34 (t, <i>J</i> = 7.3 Hz, 1H), 2.26 (t, <i>J</i> = 7.3 Hz, 1H), 2.20–2.04 (m, 3H), 1.66–1.46 (m, 4H), 1.33–1.15 (m, 14H). HRMS (DART-TOF): calcd for C<sub>48</sub>H<sub>55</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 913.4019; found, 913.4028. Purity, 97.2%.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> <i>N</i>-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)-14-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-14-oxotetradecanamide (<b>20</b>)</h3><div class="NLM_p last">Compound <b>20</b> (60.6 mg, 67% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46a</b> and <b>47g</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 10.90 (d, <i>J</i> = 31.1 Hz, 1H), 9.03 (br, 1H), 8.51–8.36 (m, 1H), 7.84–7.62 (m, 3H), 7.50–7.41 (m, 1H), 7.37–7.27 (m, 2H), 7.11–6.91 (m, 3H), 6.47–6.36 (m, 1H), 6.28 (s, 1H), 4.93 (dd, <i>J</i> = 12.0, <i>J</i> = 5.5 Hz, 1H), 4.27 (s, 2H), 3.92–3.10 (m, 12H), 2.93–2.65 (m, 3H), 2.34 (t, <i>J</i> = 7.4 Hz, 1H), 2.27 (t, <i>J</i> = 7.1 Hz, 1H), 2.19–2.06 (m, 3H), 1.68–1.50 (m, 4H), 1.31–1.13 (m, 16H). HRMS (DART-TOF): calcd for C<sub>49</sub>H<sub>57</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 927.4176; found, 927.4189. Purity, 98.5%.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> <i>N</i>-(4-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)butyl)-12-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamide (<b>21</b>)</h3><div class="NLM_p last">Compound <b>21</b> (56.0 mg, 62% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46b</b> and <b>47e</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.33 (br, 1H), 9.26 (s, 1H), 8.43 (d, <i>J</i> = 5.9 Hz, 1H), 7.78–7.64 (m, 3H), 7.44 (t, <i>J</i> = 7.7 Hz, 1H), 7.37–7.27 (m, 2H), 7.03 (d, <i>J</i> = 6.8 Hz, 2H), 6.84 (d, <i>J</i> = 8.5 Hz, 1H), 6.20 (s, 1H), 5.93 (s, 1H), 4.97–4.84 (m, 1H), 4.26 (s, 2H), 3.87–3.11 (m, 12H), 2.90–2.66 (m, 3H), 2.32 (s, 1H), 2.25 (s, 1H), 2.15–2.04 (m, 3H), 1.74–1.46 (m, 8H), 1.36–1.11 (m, 14H). HRMS (DART-TOF): calcd for C<sub>49</sub>H<sub>57</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 927.4181; found, 927.4186. Purity, 98.4%.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> <i>N</i>-(6-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)hexyl)-12-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamide (<b>22</b>)</h3><div class="NLM_p last">Compound <b>22</b> (60.6 mg, 65% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46c</b> and <b>47e</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.02 (br, 1H), 8.96 (br, 1H), 8.53–8.36 (m, 1H), 7.88–7.62 (m, 3H), 7.46 (t, <i>J</i> = 7.8 Hz, 1H), 7.39–7.27 (m, 2H), 7.13–6.96 (m, 2H), 6.85 (d, <i>J</i> = 8.5 Hz, 1H), 6.21 (t, <i>J</i> = 5.4 Hz, 1H), 5.75–5.54 (m, 1H), 4.92 (dd, <i>J</i> = 11.7, <i>J</i> = 5.3 Hz, 1H), 4.27 (s, 2H), 3.89–3.12 (m, 12H), 2.91–2.66 (m, 3H), 2.44–2.30 (t, <i>J</i> = 7.2 Hz, 1H), 2.23–2.29 (t, <i>J</i> = 6.8 Hz, 1H), 2.19–2.04 (m, 3H), 1.71–1.54 (m, 6H), 1.53–1.45 (m, 2H), 1.44–1.18 (m, 16H). HRMS (DART-TOF): calcd for C<sub>51</sub>H<sub>61</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 955.4489; found, 955.4496. Purity, 97.7%.</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> <i>N</i>-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)-12-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamide (<b>23</b>)</h3><div class="NLM_p last">Compound <b>23</b> (39.5 mg, 49% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46d</b> and <b>47e</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 10.77 (br, 1H), 8.97 (br, 1H), 8.52–8.41 (m, 1H), 7.86–7.63 (m, 3H), 7.56–7.44 (m, 1H), 7.37–7.27 (m, 2H), 7.10 (d, <i>J</i> = 7.1 Hz, 1H), 7.03 (t, <i>J</i> = 8.7 Hz, 1H), 6.91 (d, <i>J</i> = 8.5 Hz, 1H), 6.56 (t, <i>J</i> = 5.3 Hz, 1H), 6.17 (s, 1H), 4.91 (dd, <i>J</i> = 12.0, <i>J</i> = 5.3 Hz, 1H), 4.28 (s, 2H), 3.94–3.15 (m, 16H), 2.97–2.65 (m, 3H), 2.34 (t, <i>J</i> = 7.3 Hz, 1H), 2.27 (t, <i>J</i> = 7.0 Hz, 1H), 2.21–2.05 (m, 3H), 1.65–1.49 (m, 4H), 1.32–1.12 (m, 12H). HRMS (DART-TOF): calcd for C<sub>49</sub>H<sub>57</sub>FN<sub>8</sub>NaO<sub>9</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 943.4125; found, 943.4136. Purity, 98.1%.</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> <i>N</i>-(2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethoxy)ethyl)-12-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamide (<b>24</b>)</h3><div class="NLM_p last">Compound <b>24</b> (55.9 mg, 58% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46e</b> and <b>47e</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 10.66 (br, 1H), 9.05 (br, 1H), 8.51–8.37 (m, 1H), 7.83–7.66 (m, 3H), 7.55–7.42 (m, 1H), 7.38–7.27 (m, 2H), 7.09 (d, <i>J</i> = 7.1 Hz, 1H), 7.04 (t, <i>J</i> = 8.6 Hz, 1H), 6.89 (d, <i>J</i> = 8.5 Hz, 1H), 6.51 (t, <i>J</i> = 5.4 Hz, 1H), 6.08 (s, 1H), 4.97–4.84 (m, 1H), 4.28 (s, 2H), 3.95–3.07 (m, 20H), 2.92–2.64 (m, 3H), 2.34 (t, <i>J</i> = 7.4 Hz, 1H), 2.27 (t, <i>J</i> = 7.1 Hz, 1H), 2.19–2.06 (m, 3H), 1.69–1.48 (m, 4H), 1.33–1.16 (m, 12H). HRMS (DART-TOF): calcd for C<sub>51</sub>H<sub>61</sub>FN<sub>8</sub>NaO<sub>10</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 987.4387; found, 987.4396. Purity, 96.6%.</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> <i>N</i>-(3-(2-(2-(3-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)propoxy)ethoxy)ethoxy)propyl)-12-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamide (<b>25</b>)</h3><div class="NLM_p last">Compound <b>25</b> (55.9 mg, 54% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>46f</b> and <b>47e</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 10.82 (br, 1H), 8.95 (br, 1H), 8.53–8.35 (m, 1H), 7.83–7.61 (m, 3H), 7.47 (t, <i>J</i> = 7.8 Hz, 1H), 7.38–7.27 (m, 2H), 7.15–6.97 (m, 2H), 6.91 (d, <i>J</i> = 8.6 Hz, 1H), 6.44 (t, <i>J</i> = 5.5 Hz, 1H), 6.22 (s, 1H), 4.96–4.84 (m, 1H), 4.27 (s, 2H), 3.97–3.08 (m, 24H), 2.91–2.65 (m, 3H), 2.34 (t, <i>J</i> = 7.3 Hz, 1H), 2.26 (t, <i>J</i> = 6.9 Hz, 1H), 2.19–2.04 (m, 3H), 1.96–1.88 (m, 2H), 1.79–1.70 (m, 2H), 1.66–1.49 (m, 4H), 1.35–1.17 (m, 12H). HRMS (DART-TOF): calcd for C<sub>55</sub>H<sub>69</sub>FN<sub>8</sub>NaO<sub>11</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 1059.4962; found, 1059.4962. Purity, 95.8%.</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> <i>N</i>-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)ethyl)-12-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamide (<b>26</b>)</h3><div class="NLM_p last">Compound <b>26</b> (58.7 mg, 67% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>48</b> and <b>47e</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 10.95 (br, 1H), 9.14 (br, 1H), 8.50–8.35 (m, 1H), 7.85–7.64 (m, 3H), 7.51 (d, <i>J</i> = 8.2 Hz, 1H), 7.37–7.27 (m, 2H), 7.04 (t, <i>J</i> = 8.5 Hz, 1H), 6.86 (d, <i>J</i> = 1.8 Hz, 1H), 6.68 (dd, <i>J</i> = 8.4, <i>J</i> = 1.9 Hz, 1H), 6.41 (t, <i>J</i> = 5.9 Hz, 1H), 5.74 (t, <i>J</i> = 4.6 Hz, 1H), 4.94 (dd, <i>J</i> = 11.9, <i>J</i> = 5.4 Hz, 1H), 4.28 (s, 2H), 3.93–3.13 (m, 12H), 2.93–2.63 (m, 3H), 2.34 (t, <i>J</i> = 7.1 Hz, 1H), 2.27 (t, <i>J</i> = 6.8 Hz, 1H), 2.22–2.07 (m, 3H), 1.68–1.47 (m, 4H), 1.32–1.15 (m, 12H). HRMS (DART-TOF): calcd for C<sub>47</sub>H<sub>53</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 899.3863; found, 899.3866. Purity, 96.6%.</div></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> <i>N</i>-(2-((2-(2,6-Dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)amino)ethyl)-12-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamide (<b>27</b>)</h3><div class="NLM_p last">Compound <b>27</b> (52.6 mg, 61% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>50</b> and <b>47e</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ (ppm) 8.41–8.33 (m, 1H), 7.94 (d, <i>J</i> = 7.8 Hz, 1H), 7.90–7.77 (m, 2H), 7.53–7.45 (m, 1H), 7.39 (br, 1H), 7.32 (dd, <i>J</i> = 10.4, <i>J</i> = 5.0 Hz, 1H), 7.16 (t, <i>J</i> = 9.0 Hz, 1H), 7.08 (dd, <i>J</i> = 7.2, <i>J</i> = 4.3 Hz, 1H), 6.86 (d, <i>J</i> = 8.0 Hz, 1H), 5.27–5.08 (d, <i>J</i> = 11.6 Hz, 1H), 4.38 (s, 2H), 4.27 (s, 2H), 3.85–3.40 (m, 8H), 3.39–3.25 (m, 4H), 3.00–2.84 (m, 1H), 2.84–2.71 (m, 1H), 2.56–2.30 (m, 3H), 2.19 (t, <i>J</i> = 7.3 Hz, 3H), 1.72–1.47 (m, 4H), 1.35–1.12 (m, 12H). HRMS (DART-TOF): calcd for C<sub>47</sub>H<sub>55</sub>FN<sub>8</sub>NaO<sub>7</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 885.4070; found, 885.4074. Purity, 98.0%.</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 12-(4-(2-Fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-<i>N</i>-(2-((2-(1-methyl-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)-12-oxododecanamide (<b>28</b>)</h3><div class="NLM_p last">Compound <b>28</b> (50.7 mg, 57% yield) was obtained as a yellow powder using the procedure employed for the synthesis of compound <b>18</b> with <b>52</b> and <b>47e</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 10.73 (br, 1H), 8.53–8.33 (m, 1H), 7.86–7.62 (m, 3H), 7.48 (t, <i>J</i> = 7.8 Hz, 1H), 7.38–7.27 (m, 2H), 7.14–6.94 (m, 3H), 6.39 (s, 1H), 6.07 (br, 1H), 4.92 (dd, <i>J</i> = 12.1, <i>J</i> = 5.5 Hz, 1H), 4.27 (s, 2H), 3.88–3.22 (m, 12H), 3.19 (s, 3H), 3.01–2.89 (m, 1H), 2.82–2.68 (m, 2H), 2.34 (t, <i>J</i> = 7.0 Hz, 1H), 2.27 (t, <i>J</i> = 6.7 Hz, 1H), 2.16 (t, <i>J</i> = 7.6 Hz, 2H), 2.11–2.02 (m, 1H), 1.67–1.51 (m, 4H), 1.35–1.17 (m, 12H). HRMS (DART-TOF): calcd for C<sub>48</sub>H<sub>55</sub>FN<sub>8</sub>NaO<sub>8</sub><sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 913.4019; found, 913.4026. Purity, 96.5%</div></div><div id="sec5_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(12-(4-(2-Fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>29</b>)</h3><div class="NLM_p last">Compound <b>29</b> (61.7 mg, 76% yield) was obtained as a white powder from <b>47e</b> and <b>53</b> using the procedure employed for the synthesis of compound <b>22</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.68 (s, 1H), 8.64 (s, 1H), 8.46–8.35 (m, 1H), 7.78–7.63 (m, 3H), 7.57 (s, 1H), 7.30 (q, <i>J</i> = 8.3 Hz, 6H), 6.98 (t, <i>J</i> = 8.7 Hz, 1H), 6.68 (s, 1H), 4.71 (t, <i>J</i> = 7.9 Hz, 1H), 4.64–4.47 (m, 3H), 4.30 (dd, <i>J</i> = 15.1, 5.2 Hz, 1H), 4.26–4.12 (m, 3H), 4.07 (d, <i>J</i> = 11.2 Hz, 1H), 3.87–3.08 (m, 9H), 2.67–2.36 (m, 5H), 2.35–2.04 (m, 5H), 1.65–1.45 (m, 4H), 1.31–1.11 (m, 12H), 0.94 (s, 9H). HRMS (DART-TOF): calcd for C<sub>54</sub>H<sub>67</sub>FN<sub>8</sub>NaO<sub>7</sub>S<sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 1013.4735; found, 1013.4732. Purity, 96.4%.</div></div><div id="sec5_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-(12-(4-(2-Fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamido)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>30</b>)</h3><div class="NLM_p last">Compound <b>30</b> (41.2 mg, 55% yield) was obtained as a white powder from <b>47e</b> and <b>54</b> using the procedure employed for the synthesis of compound <b>22</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.60 (s, 1H), 8.64 (s, 1H), 8.46–8.34 (m, 1H), 8.07 (s, 1H), 7.79–7.60 (m, 4H), 7.38–7.22 (m, 6H), 7.06–6.88 (m, 2H), 4.77 (t, <i>J</i> = 8.0 Hz, 1H), 4.71 (d, <i>J</i> = 9.1 Hz, 1H), 4.55 (dd, <i>J</i> = 15.5, 6.6 Hz, 2H), 4.36–4.21 (m, 3H), 4.16 (s, 1H), 4.04–3.93 (m, 3H), 3.81–3.13 (m, 9H), 2.46 (s, 3H), 2.36–2.20 (m, 5H), 2.21–2.09 (m, 2H), 1.65–1.45 (m, 4H), 1.28–1.13 (m, 12H), 0.98 (s, 9H). HRMS (DART-TOF): calcd for C<sub>56</sub>H<sub>70</sub>FN<sub>9</sub>NaO<sub>8</sub>S<sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 1070.4950; found, 1070.4953. Purity, 98.3%.</div></div><div id="sec5_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(4-(12-(4-(2-Fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamido)butanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>31</b>)</h3><div class="NLM_p last">Compound <b>31</b> (55.6 mg, 57% yield) was obtained as a white powder from <b>47e</b> and <b>55</b> using the procedure employed for the synthesis of compound <b>22</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ (ppm) 11.59 (br, 1H), 8.58 (s, 1H), 8.40–8.24 (m, 1H), 7.77–7.60 (m, 3H), 7.58 (t, <i>J</i> = 5.7 Hz, 1H), 7.24 (q, <i>J</i> = 8.4 Hz, 6H), 6.95 (d, <i>J</i> = 8.4 Hz, 1H), 6.42 (t, <i>J</i> = 5.2 Hz, 1H), 4.67 (t, <i>J</i> = 8.0 Hz, 1H), 4.53–4.40 (m, 3H), 4.33 (s, 1H), 4.25 (dd, <i>J</i> = 15.2, 5.2 Hz, 1H), 4.18 (s, 2H), 4.02 (d, <i>J</i> = 11.1 Hz, 1H), 3.78–3.01 (m, 12H), 2.38 (s, 3H), 2.34–2.23 (m, 2H), 2.23–2.07 (m, 4H), 2.02 (t, <i>J</i> = 6.0 Hz, 2H), 1.73–1.61 (m, 2H), 1.59–1.35 (m, 3H), 1.31 (d, <i>J</i> = 6.6 Hz, 2H), 1.19–1.07 (m, 10H), 0.90 (s, 9H). HRMS (DART-TOF): calcd for C<sub>58</sub>H<sub>74</sub>FN<sub>9</sub>NaO<sub>8</sub>S<sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 1098.5263; found, 1098.5257. Purity, 97.6%.</div></div><div id="sec5_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(6-(12-(4-(2-Fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-12-oxododecanamido)hexanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>32</b>)</h3><div class="NLM_p last">Compound <b>32</b> (63.1 mg, 69% yield) was obtained as a white powder from <b>47e</b> and <b>56</b> using the procedure employed for the synthesis of compound <b>22</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.70 (s, 1H), 8.63 (s, 1H), 8.40 (d, <i>J</i> = 6.9 Hz, 1H), 7.81–7.58 (m, 4H), 7.30 (dd, <i>J</i> = 17.7, 9.2 Hz, 6H), 6.99 (d, <i>J</i> = 8.1 Hz, 1H), 6.86 (s, 1H), 6.10 (s, 1H), 4.71 (t, <i>J</i> = 7.9 Hz, 1H), 4.66–4.48 (m, 3H), 4.48–4.36 (m, 1H), 4.31 (dd, <i>J</i> = 15.4, 5.3 Hz, 1H), 4.24 (s, 2H), 4.07 (d, <i>J</i> = 10.9 Hz, 1H), 3.83–3.17 (m, 10H), 3.12 (d, <i>J</i> = 6.1 Hz, 2H), 2.53 (s, 2H), 2.44 (s, 3H), 2.42–2.12 (m, 5H), 2.07 (t, <i>J</i> = 7.3 Hz, 2H), 1.65–1.45 (m, 6H), 1.45–1.34 (m, 2H), 1.32–1.12 (m, 14H), 0.94 (s, 9H). HRMS (DART-TOF): calcd for C<sub>60</sub>H<sub>78</sub>FN<sub>9</sub>NaO<sub>8</sub>S<sup>+</sup> [M + Na]<sup>+</sup><i>m</i>/<i>z</i>, 1126.5576; found, 1126.5576. Purity, 98.8%.</div></div><div id="sec5_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Cell Lines and Culture Methods</h3><div class="NLM_p last">MDA-MB-436, Capan-1, HCC1937, SW620, MDA-MB-468, MDA-MB-231, PC-3, 22Rv1, LNCaP, L-O2, HEK-293, NCM460, HUV-EC, MCF-10A, and H9c2 cell lines were obtained from the American Type Culture Collection (ATCC) or Cell bank of Chinese Academy of Sciences (CCAS, China). MDA-MB-436, HCC1937, SW620, 22Rv1, and LNCaP cell lines were cultured in RPMI-1640 (HyClone, SH30809.01B) medium supplemented with 10% (v/v) fetal bovine serum (FBS, Gemini, 900108) and 1% (v/v) penicillin–streptomycin (HyClone, SV30010). MDA-MB-468, MDA-MB-231, L-O2, HEK-293, NCM460, HUV-EC, MCF-10A, and H9c2 cells were cultured in DMEM (HyClone, SH30022.01B) supplemented with 10% (v/v) FBS and 1% (v/v) penicillin–streptomycin. Capan-1 cells were cultured in Iscove’s modified Dulbecco’s medium (HyClone, SH32559.01B) supplemented with 20% (v/v) FBS and 1% (v/v) penicillin–streptomycin. PC-3 cells were cultured in DME/F12 (1:1) (HyClone, SH30026.01B) supplemented with 10% (v/v) FBS and 1% (v/v) penicillin–streptomycin. All cells were grown in a humidified incubator at 37 °C and 5% CO<sub>2</sub>.</div></div><div id="sec5_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Cell Growth Inhibition Assays</h3><div class="NLM_p last">Cells were grown at 37 °C, under 95% air and 5% CO<sub>2</sub> until about reaching 70% confluencet, and subcultured at least twice before the experiment. Cells were seeded in 96-well plates at the individual density in 80 μL of culture medium for 24 h. The cell seeding numbers for individual cell lines were as follows: MDA-MB-436 (700/well), Capan-1 (700/well), HCC1937 (1000/well), SW620 (1000/well), MDA-MB-468 (1500/well), MDA-MB-231 (1500/well), PC-3 (700/well), 22Rv1 (1000/well), and LNCaP (1200/well). Compounds were prepared as a 10 mM stock solution in 100% DMSO and tested in 10-dose IC<sub>50</sub> mode with a threefold serial dilution with culture medium (20 μL) starting at 10–20 μM. After the addition of the tested compounds, the plates were incubated for approximately 7 days (Capan-1 for 13 days) at 37 °C under 95% air and 5% CO<sub>2</sub> after dosing. The 10 μL Cell Counting Kit-8 (CCK-8, SAB, CP002) reagent was added to the well, incubated for at least 1–3 h, and measured at 450 nm via spectrophotometry (Thermo Multiskan MK3). Data were calculated from two wells as the mean percentage of cell growth relative to DMSO-only wells, and IC<sub>50</sub> was calculated by nonlinear regression analysis using GraphPad Prism v6.0 software.</div></div><div id="sec5_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Potentiation of MMS Cytotoxicity by SK-575 Determined by the Use of SW620 Cell Growth Assays</h3><div class="NLM_p last">SW620 cells (1000/well) were seeded in 96-well plates in 80 μL of growth medium and left to attach for 24 h. Cells were preincubated with vehicle control (DMSO) or with a single concentration of SK-575 (3, 10, 100, or 300 nM, 10 μL) for 1 h prior to the addition of MMS (0, 1.5 3, 5, 7, 10, 12, and 15 μg/mL, 10 μL), which were diluted with culture medium. Cells were incubated in the presence of each drug combination for 5 days before cell growth was quantified by the use of the CCK-8 assay by measuring the medium’s absorbance at 450 nm via spectrophotometry (Thermo Multiskan MK3). The data were calculated from three wells as the mean percentage of cell growth relative to DMSO-only wells.</div></div><div id="sec5_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Western Blotting Analysis</h3><div class="NLM_p last">Cells (5 × 10<sup>5</sup> to 9 × 10<sup>5</sup>) were seeded in 6-well plates at an individual density for 24 h and then treated with DMSO or compounds at the indicated concentrations for various times, or isolated tumor samples were immediately frozen and ground with a mortar in liquid nitrogen. Cells were washed with cold PBS and lysed in RIPA buffer containing protease inhibitors (MCE, HY-K0010). The lysate was centrifuged (13,000 rpm, 4 °C, 15 min); the protein concentrations were determined by the BCA Assay Kit (Beyotime, p0012s). A total of 20–30 μg proteins was loaded onto 7.5–12.5% SDS-PAGE gel and then transferred onto polyvinylidene fluoride (Millipore) membranes. The membranes were blocked for 1 h, incubated with primary antibodies overnight at 4 °C, and then washed three times with Tris-buffered saline with Tween 20 (TBST) for 10 min. After incubating with the secondary HRP antibody for 2 h at room temperature, the membranes were washed three times with TBST for 10 min and then exposed on autoradiograph films using enhanced chemiluminescence. Software <a href="http://GelQuant.NET" class="ext-link">GelQuant.NET</a> (provided by <a href="http://biochemlabsolutions.com" class="ext-link">BiochemLabSolutions.com</a>) was used to quantify the percentage of PARP1 degradation. The primary antibodies used were PARP1 (1/1000 dilution, CST#9532), PARP2 (1/1000 dilution, ABS#140704), γH2AX (1/1000 dilution, CST#9718), and β-tubulin (1/1000 dilution, CST#5536).</div></div><div id="sec5_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> In Vitro PARP1 Activity Assays</h3><div class="NLM_p last">Kinase activity assays were performed at Reaction Biology Corp. as previously described.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> In brief, compounds were tested in consecutive threefold dilution starting from 10 μM, 10 points, in duplicate. Enzyme assay was conducted in a reaction buffer containing 50 mM Tris–HCl (pH 8.0), 50 mM NaCl, 10 mM MgCl<sub>2</sub>, 1 mM DTT, 1% DMSO, and 20 g/mL activated DNA. PARP1 (20 μM) was added to the reaction buffer and mixed gently. Various compounds were added to the reaction mixture using Acoustic Technology (Echo 550, LabCyte Inc. Sunnyvale, CA) and incubated for 20 min at 25 °C. Reactions were initiated by adding <sup>32</sup>P–NAD<sup>+</sup> (10 μM) into the system, which were incubated for 2 h at 25 °C. After transferring the reaction mixture to a filter paper and washing out free NAD<sup>+</sup> with 0.75% phosphoric acid, the signal was immediately measured by chemiluminescence on a Synergy microplate reader (Bio-Tek). The data were analyzed using Excel and GraphPad Prism v6.0 software for IC<sub>50</sub> curve fits.</div></div><div id="sec5_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> PK Determination</h3><div class="NLM_p last">The PK properties were determined at XPiscoric Inc., China. We carried out PK determinations using female ICR mice (20–23 g, 6–7 week old, <i>N</i> = 3 per group, purchased from Beijing HFK Bioscience Co. Ltd., Beijing, China). SK-575 was formulated in 5% (v/v) DMSO, 15% Solutol HS15, and 80% (v/v) D5W and administered intraperitoneally at a single dose of 25 mg/kg. Plasma samples were collected at 0.5, 1, 2, 4, 6, 8, and 24 h after drug treatment (<i>n</i> = 3). Methanol was added to precipitate the protein and the samples were vortexed for 120 s, centrifuged (12,000 rpm, 10 min, 4 °C), the supernatant was absorbed, and the concentrations of the compound were determined by liquid chromatography with tandem mass spectrometry according to a standard curve.</div></div><div id="sec5_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Mouse Xenograft Experiments</h3><div class="NLM_p last">Animal studies were conducted under the approval of the Experimental Animal Management Committee of Sichuan University (IACUC number: 20100318). The model was established by the transplantation of Capan-1 (5 × 10<sup>6</sup>) or SW620 (6 × 10<sup>6</sup>) cells in 0.1 mL medium that were subcutaneously injected into the right armpit of each of the 5–6 week old male BALB/c nude mice, obtained from Beijing HFK Bioscience Co. Ltd., Beijing, China. When the average tumor volume reached approximately 150 mm<sup>3</sup>, the mice were selected on the basis of tumor volume and body weight and randomly assigned to treatment groups and vehicle control groups with six animals in each group. Tumor volumes and body weights were measured 2–3 times per week, and tumor volumes were calculated as per the formula [length/2] × [width<sup>2</sup>].</div></div><div id="sec5_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Tumor Proliferation Inhibition Test</h3><div class="NLM_p last">SK-575, olaparib, and TMZ were dissolved in the vehicle containing 5% (v/v) DMSO, 20% (v/v) PEG400, 10% (v/v) Cremophor EL, and 65% (v/v) normal saline (NS). Cisplatin was dissolved in NS. In the single-agent studies for Capan-1 xenograft (6 mice per group), olaparib (100 mg/kg) was administered by oral gavage once daily for 21 consecutive days. SK-575 (25 and 50 mg/kg) and vehicle were administered intraperitoneally, once daily five times per week for 3 weeks. In the TMZ combination studies for SW620 xenograft (6 mice per group), olaparib (10 mg/kg) by oral gavage and SK-575 (5 and 10 mg/kg) and vehicle by intraperitoneal injection were administered once daily for 5 days starting on day 1. TMZ (50 mg/kg) or its vehicle was administered orally once daily for 5 consecutive days starting on day 0, 45 min after olaparib or SK-575 was administered. In the cisplatin combination studies for Capan-1 xenograft (6 mice per group), olaparib (100 mg/kg) by oral gavage and SK-575 (25 and 50 mg/kg) and vehicle by intraperitoneal injection were administered once daily for 5 days starting on day 0. Cisplatin (6 mg/kg) or its vehicle was administered by intraperitoneal injection on day 2, 45 min after olaparib or SK-575 was administered.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00821" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06538" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06538" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00821?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00821</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Chemical structure of the five previously reported PARP1/2 PAOTAC; western blotting analysis of PARP1 and PARP2 in cancer cell lines treated with indicated compounds; cell growth inhibition activity of SK-575 and olaparib in cancer cell lines and normal cell lines; body weight changes in mice; <sup>1</sup>H NMR spectrum for SK-575; and HPLC results for SK-575 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Recommended compound characterization checklist (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_002.xls" class="ext-link">XLS</a>)</p></li><li><p class="inline">Molecular formula strings and IC<sub>50</sub> and DC<sub>50</sub> data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf">jm0c00821_si_001.pdf (1.57 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_002.xls">jm0c00821_si_002.xls (148.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_003.csv">jm0c00821_si_003.csv (5.29 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00821" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15052" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15052" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lifeng Zhao</span> - <span class="hlFld-Affiliation affiliation">Sichuan
Industrial Institute of Antibiotics, Chengdu
University, Chengdu 610052, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#325e5b54575c55485a535d724151471c5756471c515c"><span class="__cf_email__" data-cfemail="9af6f3fcfff4fde0f2fbf5dae9f9efb4fffeefb4f9f4">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuanwei Chen</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy
and Cancer Center, West China Hospital of
Sichuan University, Chengdu 610041, China</span>; 
    <span class="hlFld-Affiliation affiliation">Hinova
Pharmaceuticals Inc., 4th Floor, Building RongYao A, No. 5, Keyuan South Road, Chengdu 610041, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4248-3613" title="Orcid link">http://orcid.org/0000-0003-4248-3613</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#88f1ffebe0ede6c8fbebfda6edecfda6ebe6"><span class="__cf_email__" data-cfemail="a9d0decac1ccc7e9dacadc87cccddc87cac7">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chaoguo Cao</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy
and Cancer Center, West China Hospital of
Sichuan University, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jie Yang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy
and Cancer Center, West China Hospital of
Sichuan University, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yong Chen</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy
and Cancer Center, West China Hospital of
Sichuan University, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peiting Zhou</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy
and Cancer Center, West China Hospital of
Sichuan University, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yingwei Wang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy, Collaborative Innovation Center of Biotherapy
and Cancer Center, West China Hospital of
Sichuan University, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wu Du</span> - <span class="hlFld-Affiliation affiliation">Hinova
Pharmaceuticals Inc., 4th Floor, Building RongYao A, No. 5, Keyuan South Road, Chengdu 610041, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.C., J.Y., and Y.C. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32624" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32624" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Wu Du, Dr. Jiang Yu, Dr. Ying-Wei Wang, and Dr. Li-Feng Zhao for helpful discussions and suggestions. This work was supported by the National Natural Science Foundation of China (no. 81672951) and Chengdu Industrial Cluster Collaborative Innovation Project (2016-XT00-00024-GX).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BRCA</td><td class="NLM_def"><p class="first last">breast cancer susceptibility genes</p></td></tr><tr><td class="NLM_term">BRE</td><td class="NLM_def"><p class="first last">base excision repair</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">D5W</td><td class="NLM_def"><p class="first last">dextrose (5%) in water</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylacetamide</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">hexafluorophosphate azabenzotriazole tetramethyl uronium</p></td></tr><tr><td class="NLM_term">HR</td><td class="NLM_def"><p class="first last">homologous recombination</p></td></tr><tr><td class="NLM_term">MMS</td><td class="NLM_def"><p class="first last">methyl methanesulfonate</p></td></tr><tr><td class="NLM_term">NAD<sup>+</sup></td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide</p></td></tr><tr><td class="NLM_term">NAE</td><td class="NLM_def"><p class="first last">NEDD8-activating enzyme</p></td></tr><tr><td class="NLM_term">NS</td><td class="NLM_def"><p class="first last">normal saline</p></td></tr><tr><td class="NLM_term">PAR</td><td class="NLM_def"><p class="first last">poly(ADP-ribose)</p></td></tr><tr><td class="NLM_term">PARP</td><td class="NLM_def"><p class="first last">Poly(ADP ribose) polymerase</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate-buffered saline</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">pharmacodynamics</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">polyethylene glycol</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PPIs</td><td class="NLM_def"><p class="first last">protein–protein interactions</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">PTEN</td><td class="NLM_def"><p class="first last">phosphate and tension homolog deleted on chromosome ten</p></td></tr><tr><td class="NLM_term">SSBs</td><td class="NLM_def"><p class="first last">single-strand breaks</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">TMZ</td><td class="NLM_def"><p class="first last">temozolomide</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 58 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kleine, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesniewicz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassa, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hottiger, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litchfield, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shilton, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lüscher, B.</span></span> <span> </span><span class="NLM_article-title">Substrate-assisted catalysis by PARP10 limits its activity to mono-ADP-ribosylation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.molcel.2008.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=18851833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1GksLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=57-69&author=H.+Kleineauthor=E.+Porebaauthor=K.+Lesniewiczauthor=P.+O.+Hassaauthor=M.+O.+Hottigerauthor=D.+W.+Litchfieldauthor=B.+H.+Shiltonauthor=B.+L%C3%BCscher&title=Substrate-assisted+catalysis+by+PARP10+limits+its+activity+to+mono-ADP-ribosylation&doi=10.1016%2Fj.molcel.2008.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate-assisted catalysis by PARP10 limits its activity to mono-ADP-ribosylation</span></div><div class="casAuthors">Kleine, Henning; Poreba, Elzbieta; Lesniewicz, Krzysztof; Hassa, Paul O.; Hottiger, Michael O.; Litchfield, David W.; Shilton, Brian H.; Luescher, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">ADP-ribosylation controls many processes, including transcription, DNA repair, and bacterial toxicity.  ADP-ribosyltransferases and poly-ADP-ribose polymerases (PARPs) catalyze mono- and poly-ADP-ribosylation, resp., and depend on a highly conserved glutamate residue in the active center for catalysis.  However, there is an apparent absence of this glutamate for the recently described PARP6-PARP16, raising questions about how these enzymes function.  We find that PARP10, in contrast to PARP1, lacks the catalytic glutamate and has transferase rather than polymerase activity.  Despite this fundamental difference, PARP10 also modifies acidic residues.  Consequently, we propose an alternative catalytic mechanism for PARP10 compared to PARP1 in which the acidic target residue of the substrate functionally substitutes for the catalytic glutamate by using substrate-assisted catalysis to transfer ADP-ribose.  This mechanism explains why the novel PARPs are unable to function as polymerases.  This discovery will help to illuminate the different biol. functions of mono- vs. poly-ADP-ribosylation in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopRTvaqa_wIrVg90H21EOLACvtfcHk0ljdC0dTraOaLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1GksLzO&md5=a7d3d0b015773d18f92a434f6a8e0232</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DKleine%26aufirst%3DH.%26aulast%3DPoreba%26aufirst%3DE.%26aulast%3DLesniewicz%26aufirst%3DK.%26aulast%3DHassa%26aufirst%3DP.%2BO.%26aulast%3DHottiger%26aufirst%3DM.%2BO.%26aulast%3DLitchfield%26aufirst%3DD.%2BW.%26aulast%3DShilton%26aufirst%3DB.%2BH.%26aulast%3DL%25C3%25BCscher%26aufirst%3DB.%26atitle%3DSubstrate-assisted%2520catalysis%2520by%2520PARP10%2520limits%2520its%2520activity%2520to%2520mono-ADP-ribosylation%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D32%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.molcel.2008.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouleau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendzel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, G. G.</span></span> <span> </span><span class="NLM_article-title">PARP inhibition: PARP1 and beyond</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1038/nrc2812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnrc2812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=20200537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1yms7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=293-301&author=M.+Rouleauauthor=A.+Patelauthor=M.+J.+Hendzelauthor=S.+H.+Kaufmannauthor=G.+G.+Poirier&title=PARP+inhibition%3A+PARP1+and+beyond&doi=10.1038%2Fnrc2812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PARP inhibition: PARP1 and beyond</span></div><div class="casAuthors">Rouleau, Michele; Patel, Anand; Hendzel, Michael J.; Kaufmann, Scott H.; Poirier, Guy G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-301</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent findings have thrust poly(ADP-ribose) polymerases (PARPs) into the limelight as potential chemotherapeutic targets.  To provide a framework for understanding these recent observations, the authors have reviewed what is known about the structures and functions of the family of PARP enzymes, and then outlined a series of questions that should be addressed to guide the rational development of PARP inhibitors as anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOiY_n1IClmbVg90H21EOLACvtfcHk0ljdC0dTraOaLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXis1yms7w%253D&md5=fa3e3964e84229317dadbd505f11ad86</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc2812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2812%26sid%3Dliteratum%253Aachs%26aulast%3DRouleau%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DHendzel%26aufirst%3DM.%2BJ.%26aulast%3DKaufmann%26aufirst%3DS.%2BH.%26aulast%3DPoirier%26aufirst%3DG.%2BG.%26atitle%3DPARP%2520inhibition%253A%2520PARP1%2520and%2520beyond%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D293%26epage%3D301%26doi%3D10.1038%2Fnrc2812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amé, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spenlehauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Murcia, G.</span></span> <span> </span><span class="NLM_article-title">The PARP superfamily</span>. <i>Bioessays</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">882</span>– <span class="NLM_lpage">893</span>, <span class="refDoi"> DOI: 10.1002/bies.20085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1002%2Fbies.20085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15273990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnt1CqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2004&pages=882-893&author=J.-C.+Am%C3%A9author=C.+Spenlehauerauthor=G.+de+Murcia&title=The+PARP+superfamily&doi=10.1002%2Fbies.20085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The PARP superfamily</span></div><div class="casAuthors">Ame, Jean-Christophe; Spenlehauer, Catherine; de Murcia, Gilbert</div><div class="citationInfo"><span class="NLM_cas:title">BioEssays</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">882-893</span>CODEN:
                <span class="NLM_cas:coden">BIOEEJ</span>;
        ISSN:<span class="NLM_cas:issn">0265-9247</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribosyl)ation is an immediate DNA-damage-dependent post-translational modification of histones and other nuclear proteins that contributes to the survival of injured proliferating cells.  Poly(ADP-ribose) polymerases (PARPs) now constitute a large family of 18 proteins, encoded by different genes and displaying a conserved catalytic domain in which PARP-1 (113 kDa), the founding member, and PARP-2 (62 kDa) are so far the sole enzymes whose catalytic activity has been shown to be immediately stimulated by DNA strand breaks.  A large repertoire of sequences encoding novel PARPs now extends considerably the field of poly(ADP-ribosyl)ation reactions to various aspects of the cell biol. including cell proliferation and cell death.  Some of these new members interact with each other, share common partners and common subcellular localizations suggesting possible fine tuning in the regulation of this post-translational modification of proteins.  This review summarizes the present knowledge of this emerging superfamily, which might ultimately improve pharmacol. strategies to enhance both antitumor efficacy and the treatment of a no. of inflammatory and neurodegenerative disorders.  A provisional nomenclature is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojkP1P6SYh67Vg90H21EOLACvtfcHk0ljdC0dTraOaLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnt1CqtLk%253D&md5=892151993f54e3c1a03019001a365fbf</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fbies.20085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbies.20085%26sid%3Dliteratum%253Aachs%26aulast%3DAm%25C3%25A9%26aufirst%3DJ.-C.%26aulast%3DSpenlehauer%26aufirst%3DC.%26aulast%3Dde%2BMurcia%26aufirst%3DG.%26atitle%3DThe%2520PARP%2520superfamily%26jtitle%3DBioessays%26date%3D2004%26volume%3D26%26spage%3D882%26epage%3D893%26doi%3D10.1002%2Fbies.20085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frostholm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, A.</span></span> <span> </span><span class="NLM_article-title">Embryonic and postnatal expression of four gamma-aminobutyric acid transporter mRNAs in the mouse brain and leptomeninges</span>. <i>J. Comp. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>376</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1002/(sici)1096-9861(19961216)376:3<431::aid-cne6>3.0.co;2-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1002%2F%28SICI%291096-9861%2819961216%29376%3A3%3C431%3A%3AAID-CNE6%3E3.0.CO%3B2-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=8956109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK2sXivFanug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=376&publication_year=1996&pages=431-446&author=J.+E.+Evansauthor=A.+Frostholmauthor=A.+Rotter&title=Embryonic+and+postnatal+expression+of+four+gamma-aminobutyric+acid+transporter+mRNAs+in+the+mouse+brain+and+leptomeninges&doi=10.1002%2F%28sici%291096-9861%2819961216%29376%3A3%3C431%3A%3Aaid-cne6%3E3.0.co%3B2-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Embryonic and postnatal expression of four gamma-aminobutyric acid transporter mRNAs in the mouse brain and leptomeninges</span></div><div class="casAuthors">Evans, J. E.; Frostholm, A.; Rotter, A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Comparative Neurology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">376</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">431-446</span>CODEN:
                <span class="NLM_cas:coden">JCNEAM</span>;
        ISSN:<span class="NLM_cas:issn">0021-9967</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss</span>)
        </div><div class="casAbstract">The distribution of gamma-aminobutyric acid (GABA) transporter mRNAs (mGATs) was studied in mouse brain during embryonic and postnatal development using in situ hybridization with radiolabeled oligonucleotide probes.  Mouse GATs 1 and 4 were present in the ventricular and subventricular zones of the lateral ventricle from gestational day 13.  During postnatal development, mGAT1 mRNA was distributed diffusely throughout the brain and spinal cord, with the highest expression present in the olfactory bulbs, hippocampus, and cerebellar cortex.  The mGAT4 message was densely distributed throughout the central nervous system during postnatal week 1; however, the hybridization signal in the cerebral cortex and hippocampus decreased during postnatal weeks 2 and 3, and in adults, mGAT4 labeling was restricted largely to the olfactory bulbs, midbrain, deep cerebellar nuclei, medulla, and spinal cord.  Mouse GAT2 mRNA was expressed only in proliferating and migrating cerebellar granule cells, whereas mGAT3 mRNA was absent from the brain and spinal cord throughout development.  Each of the four mGATs was present to some degree in the leptomeninges.  The expression of mGATs 2 and 3 was almost entirely restricted to the pia-arachnoid, whereas mGATs 1 and 4 were present only in specific regions of the membrane.  Although mGATs 1 and 4 may subserve the classical purpose of terminating inhibitory GABAergic transmission through neuronal and glial uptake mechanisms, GABA transporters in the pia-arachnoid may help to regulate the amt. of GABA available to proliferating and migrating neurons at the sub-pial surface during perinatal development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA2cnk2eidm7Vg90H21EOLACvtfcHk0lheLQUkRAJjJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFanug%253D%253D&md5=98dfbfc1ba6bccb7bf99dee1a9e655a4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-9861%2819961216%29376%3A3%3C431%3A%3AAID-CNE6%3E3.0.CO%3B2-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-9861%252819961216%2529376%253A3%253C431%253A%253AAID-CNE6%253E3.0.CO%253B2-3%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DJ.%2BE.%26aulast%3DFrostholm%26aufirst%3DA.%26aulast%3DRotter%26aufirst%3DA.%26atitle%3DEmbryonic%2520and%2520postnatal%2520expression%2520of%2520four%2520gamma-aminobutyric%2520acid%2520transporter%2520mRNAs%2520in%2520the%2520mouse%2520brain%2520and%2520leptomeninges%26jtitle%3DJ.%2520Comp.%2520Neurol.%26date%3D1996%26volume%3D376%26spage%3D431%26epage%3D446%26doi%3D10.1002%2F%28sici%291096-9861%2819961216%29376%3A3%3C431%3A%3Aaid-cne6%3E3.0.co%3B2-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jagtap, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabó, C.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose) polymerase and the therapeutic effects of its inhibitors</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1038/nrd1718</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnrd1718" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15864271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjs1OktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=421-440&author=P.+Jagtapauthor=C.+Szab%C3%B3&title=Poly%28ADP-ribose%29+polymerase+and+the+therapeutic+effects+of+its+inhibitors&doi=10.1038%2Fnrd1718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribose) polymerase and the therapeutic effects of its inhibitors</span></div><div class="casAuthors">Jagtap, Prakash; Szabo, Csaba</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">421-440</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerases (PARPs) are involved in the regulation of many cellular functions.  Three consequences of the activation of PARP1, which is the main isoform of the PARP family, are particularly important for drug development: first, its role in DNA repair; second, its capacity to deplete cellular energetic pools, which culminates in cell dysfunction and necrosis; and third, its capacity to promote the transcription of pro-inflammatory genes.  Consequently, pharmacol. inhibitors of PARP have the potential to enhance the cytotoxicity of certain DNA-damaging anticancer drugs, reduce parenchymal cell necrosis (for example, in stroke or myocardial infarction) and downregulate multiple simultaneous pathways of inflammation and tissue injury (for example, in circulatory shock, colitis or diabetic complications).  The first ultrapotent novel PARP inhibitors have now entered human clin. trials.  This article presents an overview of the principal pathophysiol. pathways and mechanisms that are governed by PARP, followed by the main structures and therapeutic actions of various classes of novel PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDsrfd_xv8g7Vg90H21EOLACvtfcHk0lheLQUkRAJjJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjs1OktL8%253D&md5=1dfd177b8064b017efb5dd7779655d55</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd1718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1718%26sid%3Dliteratum%253Aachs%26aulast%3DJagtap%26aufirst%3DP.%26aulast%3DSzab%25C3%25B3%26aufirst%3DC.%26atitle%3DPoly%2528ADP-ribose%2529%2520polymerase%2520and%2520the%2520therapeutic%2520effects%2520of%2520its%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2005%26volume%3D4%26spage%3D421%26epage%3D440%26doi%3D10.1038%2Fnrd1718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dantzer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ame, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Murcia, G.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose): novel functions for an old molecule</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">517</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1038/nrm1963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnrm1963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=16829982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1Ohsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=517-528&author=V.+Schreiberauthor=F.+Dantzerauthor=J.-C.+Ameauthor=G.+de+Murcia&title=Poly%28ADP-ribose%29%3A+novel+functions+for+an+old+molecule&doi=10.1038%2Fnrm1963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribose): novel functions for an old molecule</span></div><div class="casAuthors">Schreiber, Valerie; Dantzer, Francoise; Ame, Jean-Christophe; de Murcia, Gilbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">517-528</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The addn. to proteins of the neg. charged polymer of ADP-ribose, which is synthesized by poly(ADP-ribose) polymerase (PARPs) from NAD, is a unique post-translational modification.  It regulates not only cell survival and cell-death programs, but also an increasing no. of other biol. functions with which novel members of the PARP family have been assocd.  These functions include transcriptional regulation, telomere cohesion and mitotic spindle formation during cell division, intracellular trafficking, and energy metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLsI4_BKfjdrVg90H21EOLACvtfcHk0lheLQUkRAJjJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1Ohsb4%253D&md5=676e83755491500f3bf2137f348dfa1b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm1963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1963%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiber%26aufirst%3DV.%26aulast%3DDantzer%26aufirst%3DF.%26aulast%3DAme%26aufirst%3DJ.-C.%26aulast%3Dde%2BMurcia%26aufirst%3DG.%26atitle%3DPoly%2528ADP-ribose%2529%253A%2520novel%2520functions%2520for%2520an%2520old%2520molecule%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D7%26spage%3D517%26epage%3D528%26doi%3D10.1038%2Fnrm1963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durkacz, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omidiji, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shall, S.</span></span> <span> </span><span class="NLM_article-title">(ADP-ribose)n participates in DNA excision repair</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">593</span>– <span class="NLM_lpage">596</span>, <span class="refDoi"> DOI: 10.1038/283593a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2F283593a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=6243744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaL3cXktVagtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=1980&pages=593-596&author=B.+W.+Durkaczauthor=O.+Omidijiauthor=D.+A.+Grayauthor=S.+Shall&title=%28ADP-ribose%29n+participates+in+DNA+excision+repair&doi=10.1038%2F283593a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">(ADP-ribose)n participates in DNA excision repair</span></div><div class="casAuthors">Durkacz, Barbara W.; Omidiji, Olusesan; Gray, Douglas A.; Shall, Sydney</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">5747</span>),
    <span class="NLM_cas:pages">593-6</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The drop in cellular NAD content and the rejoining of DNA strand breaks, i.e. elevation of poly(ADP-ribose) polymerase (I) activity, after treatment of L1210 leukemia lymphoblast cells with alkylating agent, 10.5-210.6 μM Me2SO4, were inhibited by inhibitors of I, e.g. 3-aminobenzamide (Ki 4.4 μM at 26°).  The inhibitors all potentiated the cytotoxicity of Me2SO4; 2 mM aminobenzamide increased the cytotoxicity 4-fold.  In addn., the rejoining of strand breaks was prevented by nutritionally depleting cells of NAD (10-20% of normal).  Poly(ADP-ribose) may thus be involved in DNA excision repair, and possible mechanisms are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSidm4P44EPbVg90H21EOLACvtfcHk0lirb0R1UmIUbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXktVagtLc%253D&md5=93bfba1a7a3d3a2ea3705741d9f63734</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2F283593a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F283593a0%26sid%3Dliteratum%253Aachs%26aulast%3DDurkacz%26aufirst%3DB.%2BW.%26aulast%3DOmidiji%26aufirst%3DO.%26aulast%3DGray%26aufirst%3DD.%2BA.%26aulast%3DShall%26aufirst%3DS.%26atitle%3D%2528ADP-ribose%2529n%2520participates%2520in%2520DNA%2520excision%2520repair%26jtitle%3DNature%26date%3D1980%26volume%3D283%26spage%3D593%26epage%3D596%26doi%3D10.1038%2F283593a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bürke, A.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose). The most elaborate metabolite of NAD+</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>272</i></span>,  <span class="NLM_fpage">4576</span>– <span class="NLM_lpage">4589</span>, <span class="refDoi"> DOI: 10.1111/j.1742-4658.2005.04864.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1111%2Fj.1742-4658.2005.04864.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=16156780" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=2005&pages=4576-4589&author=A.+B%C3%BCrke&title=Poly%28ADP-ribose%29.+The+most+elaborate+metabolite+of+NAD%2B&doi=10.1111%2Fj.1742-4658.2005.04864.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2005.04864.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2005.04864.x%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25BCrke%26aufirst%3DA.%26atitle%3DPoly%2528ADP-ribose%2529.%2520The%2520most%2520elaborate%2520metabolite%2520of%2520NAD%252B%26jtitle%3DFEBS%2520J.%26date%3D2005%26volume%3D272%26spage%3D4576%26epage%3D4589%26doi%3D10.1111%2Fj.1742-4658.2005.04864.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, W. L.</span></span> <span> </span><span class="NLM_article-title">New insights into the molecular and cellular functions of poly(ADP-ribose) and PARPs</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">411</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1038/nrm3376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnrm3376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=22713970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC38XovVaht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=411-424&author=B.+A.+Gibsonauthor=W.+L.+Kraus&title=New+insights+into+the+molecular+and+cellular+functions+of+poly%28ADP-ribose%29+and+PARPs&doi=10.1038%2Fnrm3376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">New insights into the molecular and cellular functions of poly(ADP-ribose) and PARPs</span></div><div class="casAuthors">Gibson, Bryan A.; Kraus, W. Lee</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">411-424</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerases (PARPs) are enzymes that transfer ADP-ribose groups to target proteins and thereby affect various nuclear and cytoplasmic processes.  The activity of PARP family members, such as PARP1 and PARP2, is tied to cellular signaling pathways, and through poly(ADP-ribosyl)ation (PARylation) they ultimately promote changes in gene expression, RNA and protein abundance, and the location and activity of proteins that mediate signaling responses.  PARPs act in a complex response network that is driven by the cellular, mol. and chem. biol. of poly(ADP-ribose) (PAR).  This PAR-dependent response network is crucial for a broad array of physiol. and pathol. responses and thus is a good target for chem. therapeutics for several diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIm3s5z-XFELVg90H21EOLACvtfcHk0lirb0R1UmIUbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovVaht70%253D&md5=b72f18e950b804fed13258a3930fe004</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrm3376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3376%26sid%3Dliteratum%253Aachs%26aulast%3DGibson%26aufirst%3DB.%2BA.%26aulast%3DKraus%26aufirst%3DW.%2BL.%26atitle%3DNew%2520insights%2520into%2520the%2520molecular%2520and%2520cellular%2520functions%2520of%2520poly%2528ADP-ribose%2529%2520and%2520PARPs%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D411%26epage%3D424%26doi%3D10.1038%2Fnrm3376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stingl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghammer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haidacher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweiger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, E. F.</span></span> <span> </span><span class="NLM_article-title">Mice lacking ADPRT and poly(ADP-ribosyl)ation develop normally but are susceptible to skin disease</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">520</span>, <span class="refDoi"> DOI: 10.1101/gad.9.5.509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1101%2Fgad.9.5.509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=7698643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK2MXksFOksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=509-520&author=Z.+Q.+Wangauthor=B.+Auerauthor=L.+Stinglauthor=H.+Berghammerauthor=D.+Haidacherauthor=M.+Schweigerauthor=E.+F.+Wagner&title=Mice+lacking+ADPRT+and+poly%28ADP-ribosyl%29ation+develop+normally+but+are+susceptible+to+skin+disease&doi=10.1101%2Fgad.9.5.509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Mice lacking ADPRT and poly(ADP-ribosyl)ation develop normally but are susceptible to skin disease</span></div><div class="casAuthors">Wang, Zhaoqi; Auer, Bernhard; Stingl, Laura; Berghammer, Heinrich; Haidacher, Daniela; Schweiger, Manfred; Wagner, Erwin F.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">509-20</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Poly(ADP-ribosyl)ation is catalyzed by NAD+:protein(ADP-ribosyl)transferase (ADPRT), a chromatin-assocd. enzyme which, in the presence of DNA breaks, transfers ADP-ribose from NAD+ to nuclear proteins.  This post-translational modification has been implicated in many fundamental processes, like DNA repair, chromatin stability, cell proliferation, and cell death.  To elucidate the biol. function of ADPRT and poly(ADP-ribosyl)ation in vivo the gene was inactivated in the mouse germ line.  Mice homozygous for the ADPRT mutation are healthy and fertile.  Anal. of mutant tissues and fibroblasts isolated from mutant fetuses revealed the absence of ADPRT enzymic activity and poly(ADP-ribose), implying that no poly(ADP-ribosyl)ated proteins are present.  Mutant embryonic fibroblasts were able to efficiently repair DNA damaged by UV and alkylating agents.  However, proliferation of mutant primary fibroblasts as well as thymocytes following γ-radiation in vivo was impaired.  Moreover, mutant mice are susceptible to the spontaneous development of skin disease as ∼30% of older mice develop epidermal hyperplasia.  The generation of viable ADPRT-/-mice negates an essential role for this enzyme in normal chromatin function, but the impaired proliferation and the onset of skin lesions in older mice suggest a function for ADPRT in response to environmental stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5NqGG4AsF7rVg90H21EOLACvtfcHk0lirb0R1UmIUbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXksFOksLg%253D&md5=e7418b08b067e19b10e75e8f4c83b259</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1101%2Fgad.9.5.509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.9.5.509%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%2BQ.%26aulast%3DAuer%26aufirst%3DB.%26aulast%3DStingl%26aufirst%3DL.%26aulast%3DBerghammer%26aufirst%3DH.%26aulast%3DHaidacher%26aufirst%3DD.%26aulast%3DSchweiger%26aufirst%3DM.%26aulast%3DWagner%26aufirst%3DE.%2BF.%26atitle%3DMice%2520lacking%2520ADPRT%2520and%2520poly%2528ADP-ribosyl%2529ation%2520develop%2520normally%2520but%2520are%2520susceptible%2520to%2520skin%2520disease%26jtitle%3DGenes%2520Dev.%26date%3D1995%26volume%3D9%26spage%3D509%26epage%3D520%26doi%3D10.1101%2Fgad.9.5.509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Murcia, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niedergang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trucco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricoul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutrillaux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeMeur, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walztinger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Murcia, G.</span></span> <span> </span><span class="NLM_article-title">Requirement of poly(ADP-ribose) polymerase in recovery from DNA damage in mice and in cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">7303</span>– <span class="NLM_lpage">7307</span>, <span class="refDoi"> DOI: 10.1073/pnas.94.14.7303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1073%2Fpnas.94.14.7303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=9207086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK2sXksFOntb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=7303-7307&author=J.+M.+de%0AMurciaauthor=C.+Niedergangauthor=C.+Truccoauthor=M.+Ricoulauthor=B.+Dutrillauxauthor=M.+Markauthor=F.+J.+Oliverauthor=M.+Massonauthor=A.+Dierichauthor=M.+LeMeurauthor=C.+Walztingerauthor=P.+Chambonauthor=G.+de+Murcia&title=Requirement+of+poly%28ADP-ribose%29+polymerase+in+recovery+from+DNA+damage+in+mice+and+in+cells&doi=10.1073%2Fpnas.94.14.7303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Requirement of poly(ADP-ribose) polymerase in recovery from DNA damage in mice and in cells</span></div><div class="casAuthors">Menissier de Murcia, Josiane; Niedergang, Claude; Trucco, Carlotta; Ricoul, Michele; Dutrillaux, Bernard; Mark, Manuel; Oliver, F. Javier; Masson, Murielle; Dierich, Andree; LeMeur, Marianne; Walztinger, Caroline; Chambon, Pierre; de Murcia, Gilbert</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7303-7307</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase [PARP; NAD+ ADP-ribosyltransferase; NAD+:poly(adenosine-diphosphate-D-ribosyl)-acceptor ADP-D-ribosyltransferase, EC 2.4.2.30] is a zinc-finger DNA-binding protein that detects specifically DNA strand breaks generated by genotoxic agents.  To det. its biol. function, the authors have inactivated both alleles by gene targeting in mice.  Treatment of PARP-/- mice either by the alkylating agent N-methyl-N-nitrosourea (MNU) or by γ-irradn. revealed an extreme sensitivity and a high genomic instability to both agents.  Following whole body γ-irradn. (8 Gy) mutant mice died rapidly from acute radiation toxicity to the small intestine.  Mice-derived PARP-/- cells displayed a high sensitivity to MNU exposure: a G2/M arrest in mouse embryonic fibroblasts and a rapid apoptotic response and a p53 accumulation were obsd. in splenocytes.  Altogether these results demonstrate that PARP is a survival factor playing an essential and pos. role during DNA damage recovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlwmTBk-nHc7Vg90H21EOLACvtfcHk0lgsh9lQuTBzOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksFOntb4%253D&md5=a652d7d877fa0c29ff1aeb9769fbd86f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.94.14.7303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.94.14.7303%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BMurcia%26aufirst%3DJ.%2BM.%26aulast%3DNiedergang%26aufirst%3DC.%26aulast%3DTrucco%26aufirst%3DC.%26aulast%3DRicoul%26aufirst%3DM.%26aulast%3DDutrillaux%26aufirst%3DB.%26aulast%3DMark%26aufirst%3DM.%26aulast%3DOliver%26aufirst%3DF.%2BJ.%26aulast%3DMasson%26aufirst%3DM.%26aulast%3DDierich%26aufirst%3DA.%26aulast%3DLeMeur%26aufirst%3DM.%26aulast%3DWalztinger%26aufirst%3DC.%26aulast%3DChambon%26aufirst%3DP.%26aulast%3Dde%2BMurcia%26aufirst%3DG.%26atitle%3DRequirement%2520of%2520poly%2528ADP-ribose%2529%2520polymerase%2520in%2520recovery%2520from%2520DNA%2520damage%2520in%2520mice%2520and%2520in%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1997%26volume%3D94%26spage%3D7303%26epage%3D7307%26doi%3D10.1073%2Fpnas.94.14.7303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tentori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graziani, G.</span></span> <span> </span><span class="NLM_article-title">Chemopotentiation by PARP inhibitors in cancer therapy</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2005.02.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.phrs.2005.02.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15911331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1Wltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2005&pages=25-33&author=L.+Tentoriauthor=G.+Graziani&title=Chemopotentiation+by+PARP+inhibitors+in+cancer+therapy&doi=10.1016%2Fj.phrs.2005.02.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Chemopotentiation by PARP inhibitors in cancer therapy</span></div><div class="casAuthors">Tentori, Lucio; Graziani, Grazia</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-33</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerases (PARP) constitute a family of enzymes involved in the regulation of many cellular processes such as DNA repair, gene transcription, cell cycle progression, cell death, chromatin functions and genomic stability.  Among the 18 members identified so far, PARP-1 and PARP-2 are the only proteins stimulated by DNA strand breaks and implicated in the repair of DNA injury.  Therefore, these mols. have been exploited as potential targets for the development of pharmacol. strategies to increase the antitumor efficacy of chemotherapeutic agents, which induce DNA damage.  PARP inhibitors have been shown to restore sensitivity of resistant tumors to methylating agents or topoisomerase I inhibitors, drugs presently used for the treatment of primary and secondary brain tumors or malignancies refractory to std. chemotherapy.  Interestingly, PARP inhibitors may also provide protection from the untoward effects exerted by certain anticancer drugs, which cause oxidative stress and consequent PARP overactivation.  The aim of this article is to provide a brief overview of the recent literature on preclin. studies with the specific and potent inhibitors newly synthesized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoImZCOb5elR7Vg90H21EOLACvtfcHk0lgsh9lQuTBzOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1Wltr8%253D&md5=2a2f1893efeefc3f1d140979beae6f4d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2005.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2005.02.010%26sid%3Dliteratum%253Aachs%26aulast%3DTentori%26aufirst%3DL.%26aulast%3DGraziani%26aufirst%3DG.%26atitle%3DChemopotentiation%2520by%2520PARP%2520inhibitors%2520in%2520cancer%2520therapy%26jtitle%3DPharmacol.%2520Res.%26date%3D2005%26volume%3D52%26spage%3D25%26epage%3D33%26doi%3D10.1016%2Fj.phrs.2005.02.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iniesta, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadopoulos, K.</span></span> <span> </span><span class="NLM_article-title">Development of PARP inhibitors in oncology</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1517/13543780802525324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1517%2F13543780802525324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=19053880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVGhtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2009&pages=31-43&author=J.+Rodonauthor=M.+D.+Iniestaauthor=K.+Papadopoulos&title=Development+of+PARP+inhibitors+in+oncology&doi=10.1517%2F13543780802525324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Development of PARP inhibitors in oncology</span></div><div class="casAuthors">Rodon, Jordi; Iniesta, Maria D.; Papadopoulos, Kyriakos</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-43</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Poly (ADP-ribose) polymerase (PARP) plays a key role in DNA repair mechanisms by detecting and initiating repair after DNA strand breaks.  Inhibition of PARP in DNA repair-defective tumors (like those with BRCA1 or BRCA2 mutations) can lead to gross genomic instability and cell death.  Likewise, combining PARP inhibition with cytotoxic agents such as chemotherapy or radiation therapy is synergistic in many preclin. models.  Several drugs designed to inhibit PARP are currently in clin. development, many following a development path different from that of typical anticancer agents.  In this review we will focus on the early clin. data from PARP inhibitors that are entering clin. trials, the potential tumors they might target, their combination with other drugs and the different biomarkers that are being explored.  Concepts such as 'BRCAness', synthetic lethality, Phase 0 trials and pharmacodynamic markers will be discussed in the context of the development of PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNslMhf0aSo7Vg90H21EOLACvtfcHk0lgsh9lQuTBzOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVGhtbvJ&md5=e031c4e938fd49b8cbf674dbc3ca8410</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1517%2F13543780802525324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543780802525324%26sid%3Dliteratum%253Aachs%26aulast%3DRodon%26aufirst%3DJ.%26aulast%3DIniesta%26aufirst%3DM.%2BD.%26aulast%3DPapadopoulos%26aufirst%3DK.%26atitle%3DDevelopment%2520of%2520PARP%2520inhibitors%2520in%2520oncology%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2009%26volume%3D18%26spage%3D31%26epage%3D43%26doi%3D10.1517%2F13543780802525324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ratnam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. A.</span></span> <span> </span><span class="NLM_article-title">Current development of clinical inhibitors of poly(ADP-ribose) polymerase in oncology</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1383</span>– <span class="NLM_lpage">1388</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-06-2260</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1158%2F1078-0432.CCR-06-2260" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=17332279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitlSjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=1383-1388&author=K.+Ratnamauthor=J.+A.+Low&title=Current+development+of+clinical+inhibitors+of+poly%28ADP-ribose%29+polymerase+in+oncology&doi=10.1158%2F1078-0432.ccr-06-2260"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Current Development of Clinical Inhibitors of Poly(ADP-Ribose) Polymerase in Oncology</span></div><div class="casAuthors">Ratnam, Kapila; Low, Jennifer A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1383-1388</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerase (PARP) is a nuclear enzyme that signals the presence of DNA damage by catalyzing the addn. of ADP-ribose units to DNA, histones, and various DNA repair enzymes and by facilitating DNA repair.  PARP has been gaining increasing interest as a therapeutic target for many diseases and esp. for cancer.  Inhibition of PARP potentiates the activity of DNA-damaging agents, such as alkylators, platinums, topoisomerase inhibitors, and radiation in in vitro and in vivo models.  In addn., tumors with DNA repair defects, such as those arising from patients with BRCA mutations, may be more sensitive to PARP inhibition.  At least five different companies have now initiated oncol. clin. trials with PARP inhibitors, ranging in stage from phase 0 to phase 2.  This review summarizes the preclin. and clin. data currently available for these agents and some of the challenges facing the clin. development of these agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1Muf0l4FhVLVg90H21EOLACvtfcHk0lgsh9lQuTBzOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitlSjtr8%253D&md5=9676a8bf09a0ee2bbb19e3e26b060ba7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-06-2260&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-06-2260%26sid%3Dliteratum%253Aachs%26aulast%3DRatnam%26aufirst%3DK.%26aulast%3DLow%26aufirst%3DJ.%2BA.%26atitle%3DCurrent%2520development%2520of%2520clinical%2520inhibitors%2520of%2520poly%2528ADP-ribose%2529%2520polymerase%2520in%2520oncology%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2007%26volume%3D13%26spage%3D1383%26epage%3D1388%26doi%3D10.1158%2F1078-0432.ccr-06-2260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Narod, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foulkes, W. D.</span></span> <span> </span><span class="NLM_article-title">BRCA1 and BRCA2: 1994 and beyond</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">665</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1038/nrc1431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnrc1431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15343273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFCmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=665-676&author=S.+A.+Narodauthor=W.+D.+Foulkes&title=BRCA1+and+BRCA2%3A+1994+and+beyond&doi=10.1038%2Fnrc1431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">BRCA1 and BRCA2: 1994 and beyond</span></div><div class="casAuthors">Narod, Steven A.; Foulkes, William D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">665-676</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The discovery of the first gene assocd. with hereditary breast cancer, BRCA1, was anticipated to greatly increase our understanding of both hereditary and sporadic forms of breast cancer, and to lead to therapeutic and preventive breakthroughs.  Much has been learned during the past decade about the genetic epidemiol. of breast cancer, the ethnic distribution and clin. consequences of BRCA1 and BRCA2 mutations, and the central role of DNA repair in breast cancer susceptibility.  The ability to translate this knowledge into novel treatments, however, remains elusive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1a5S77C977bVg90H21EOLACvtfcHk0lhNLex-QSjSLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFCmsbk%253D&md5=200600041061619ba39dfbd8dfc91850</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc1431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1431%26sid%3Dliteratum%253Aachs%26aulast%3DNarod%26aufirst%3DS.%2BA.%26aulast%3DFoulkes%26aufirst%3DW.%2BD.%26atitle%3DBRCA1%2520and%2520BRCA2%253A%25201994%2520and%2520beyond%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D665%26epage%3D676%26doi%3D10.1038%2Fnrc1431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tutt, A. N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santarosa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knights, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>434</i></span>,  <span class="NLM_fpage">917</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1038/nature03445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnature03445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15829967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=434&publication_year=2005&pages=917-921&author=H.+Farmerauthor=N.+McCabeauthor=C.+J.+Lordauthor=A.+N.+J.+Tuttauthor=D.+A.+Johnsonauthor=T.+B.+Richardsonauthor=M.+Santarosaauthor=K.+J.+Dillonauthor=I.+Hicksonauthor=C.+Knightsauthor=N.+M.+B.+Martinauthor=S.+P.+Jacksonauthor=G.+C.+M.+Smithauthor=A.+Ashworth&title=Targeting+the+DNA+repair+defect+in+BRCA+mutant+cells+as+a+therapeutic+strategy&doi=10.1038%2Fnature03445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy</span></div><div class="casAuthors">Farmer, Hannah; McCabe, Nuala; Lord, Christopher J.; Tutt, Andrew N. J.; Johnson, Damian A.; Richardson, Tobias B.; Santarosa, Manuela; Dillon, Krystyna J.; Hickson, Ian; Knights, Charlotte; Martin, Niall M. B.; Jackson, Stephen P.; Smith, Graeme C. M.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">434</span>
        (<span class="NLM_cas:issue">7035</span>),
    <span class="NLM_cas:pages">917-921</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">BRCA1 and BRCA2 are important for DNA double-strand break repair by homologous recombination, and mutations in these genes predispose to breast and other cancers.  Poly(ADP-ribose) polymerase (PARP) is an enzyme involved in base excision repair, a key pathway in the repair of DNA single-strand breaks.  The authors show here that BRCA1 or BRCA2 dysfunction unexpectedly and profoundly sensitizes cells to the inhibition of PARP enzymic activity, resulting in chromosomal instability, cell cycle arrest and subsequent apoptosis.  This seems to be because the inhibition of PARP leads to the persistence of DNA lesions normally repaired by homologous recombination.  These results illustrate how different pathways cooperate to repair damage, and suggest that the targeted inhibition of particular DNA repair pathways may allow the design of specific and less toxic therapies for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomVTKE4mFwfrVg90H21EOLACvtfcHk0lhNLex-QSjSLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrc%253D&md5=03b5ba0adc21a5627fe17ae535f45078</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature03445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03445%26sid%3Dliteratum%253Aachs%26aulast%3DFarmer%26aufirst%3DH.%26aulast%3DMcCabe%26aufirst%3DN.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DTutt%26aufirst%3DA.%2BN.%2BJ.%26aulast%3DJohnson%26aufirst%3DD.%2BA.%26aulast%3DRichardson%26aufirst%3DT.%2BB.%26aulast%3DSantarosa%26aufirst%3DM.%26aulast%3DDillon%26aufirst%3DK.%2BJ.%26aulast%3DHickson%26aufirst%3DI.%26aulast%3DKnights%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DN.%2BM.%2BB.%26aulast%3DJackson%26aufirst%3DS.%2BP.%26aulast%3DSmith%26aufirst%3DG.%2BC.%2BM.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DTargeting%2520the%2520DNA%2520repair%2520defect%2520in%2520BRCA%2520mutant%2520cells%2520as%2520a%2520therapeutic%2520strategy%26jtitle%3DNature%26date%3D2005%26volume%3D434%26spage%3D917%26epage%3D921%26doi%3D10.1038%2Fnature03445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flower, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span> <span> </span><span class="NLM_article-title">Specific killing of BRCA2-deficient tumours with inhibitors of poly(ADP-ribose) polymerase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>434</i></span>,  <span class="NLM_fpage">913</span>– <span class="NLM_lpage">917</span>, <span class="refDoi"> DOI: 10.1038/nature03443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnature03443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15829966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=434&publication_year=2005&pages=913-917&author=H.+E.+Bryantauthor=N.+Schultzauthor=H.+D.+Thomasauthor=K.+M.+Parkerauthor=D.+Flowerauthor=E.+Lopezauthor=S.+Kyleauthor=M.+Meuthauthor=N.+J.+Curtinauthor=T.+Helleday&title=Specific+killing+of+BRCA2-deficient+tumours+with+inhibitors+of+poly%28ADP-ribose%29+polymerase&doi=10.1038%2Fnature03443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Specific killing of BRCA2-deficient tumors with inhibitors of poly(ADP-ribose) polymerase</span></div><div class="casAuthors">Bryant, Helen E.; Schultz, Niklas; Thomas, Huw D.; Parker, Kayan M.; Flower, Dan; Lopez, Elena; Kyle, Suzanne; Meuth, Mark; Curtin, Nicola J.; Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">434</span>
        (<span class="NLM_cas:issue">7035</span>),
    <span class="NLM_cas:pages">913-917</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase (PARP1) facilitates DNA repair by binding to DNA breaks and attracting DNA repair proteins to the site of damage.  Nevertheless, PARP1-/- mice are viable, fertile and do not develop early onset tumors.  Here, the authors show that PARP inhibitors trigger γ-H2AX and RAD51 foci formation.  The authors propose that, in the absence of PARP1, spontaneous single-strand breaks collapse replication forks and trigger homologous recombination for repair.  Furthermore, the authors show that BRCA2-deficient cells, as a result of their deficiency in homologous recombination, are acutely sensitive to PARP inhibitors, presumably because resultant collapsed replication forks are no longer repaired.  Thus, PARP1 activity is essential in homologous recombination-deficient BRCA2 mutant cells.  The authors exploit this requirement in order to kill BRCA2-deficient tumors by PARP inhibition alone.  Treatment with PARP inhibitors is likely to be highly tumor specific, because only the tumors (which are BRCA2-/-) in BRCA2+/- patients are defective in homologous recombination.  The use of an inhibitor of a DNA repair enzyme alone to selectively kill a tumor, in the absence of an exogenous DNA-damaging agent, represents a new concept in cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZJ6B7d7dB77Vg90H21EOLACvtfcHk0lhNLex-QSjSLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrY%253D&md5=44631eba667cf75ef99c31d011d6db44</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature03443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03443%26sid%3Dliteratum%253Aachs%26aulast%3DBryant%26aufirst%3DH.%2BE.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3DThomas%26aufirst%3DH.%2BD.%26aulast%3DParker%26aufirst%3DK.%2BM.%26aulast%3DFlower%26aufirst%3DD.%26aulast%3DLopez%26aufirst%3DE.%26aulast%3DKyle%26aufirst%3DS.%26aulast%3DMeuth%26aufirst%3DM.%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DSpecific%2520killing%2520of%2520BRCA2-deficient%2520tumours%2520with%2520inhibitors%2520of%2520poly%2528ADP-ribose%2529%2520polymerase%26jtitle%3DNature%26date%3D2005%26volume%3D434%26spage%3D913%26epage%3D917%26doi%3D10.1038%2Fnature03443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menear, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adcock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockcroft, X.-l.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copsey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cranston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drzewiecki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javaid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knights, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, V. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, I. T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. M. B.</span></span> <span> </span><span class="NLM_article-title">4-[3-(4-Cyclopropanecarbonylpiperazine-1-carbonyl)-4-fluorobenzyl]-2<i>H</i>-phthalazin- 1-one: a novel bioavailable inhibitor of poly(ADP-ribose) polymerase-1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">6581</span>– <span class="NLM_lpage">6591</span>, <span class="refDoi"> DOI: 10.1021/jm8001263</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8001263" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFeqs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=6581-6591&author=K.+A.+Menearauthor=C.+Adcockauthor=R.+Boulterauthor=X.-l.+Cockcroftauthor=L.+Copseyauthor=A.+Cranstonauthor=K.+J.+Dillonauthor=J.+Drzewieckiauthor=S.+Garmanauthor=S.+Gomezauthor=H.+Javaidauthor=F.+Kerriganauthor=C.+Knightsauthor=A.+Lauauthor=V.+M.+Lohauthor=I.+T.+W.+Matthewsauthor=S.+Mooreauthor=M.+J.+O%E2%80%99Connorauthor=G.+C.+M.+Smithauthor=N.+M.+B.+Martin&title=4-%5B3-%284-Cyclopropanecarbonylpiperazine-1-carbonyl%29-4-fluorobenzyl%5D-2H-phthalazin-+1-one%3A+a+novel+bioavailable+inhibitor+of+poly%28ADP-ribose%29+polymerase-1&doi=10.1021%2Fjm8001263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">4-[3-(4-Cyclopropanecarbonylpiperazine-1-carbonyl)-4-fluorobenzyl]-2H-phthalazin-1-one: A Novel Bioavailable Inhibitor of Poly(ADP-ribose) Polymerase-1</span></div><div class="casAuthors">Menear, Keith A.; Adcock, Claire; Boulter, Robert; Cockcroft, Xiao-ling; Copsey, Louise; Cranston, Aaron; Dillon, Krystyna J.; Drzewiecki, Jan; Garman, Sheila; Gomez, Sylvie; Javaid, Hashim; Kerrigan, Frank; Knights, Charlotte; Lau, Alan; Loh, Vincent M., Jr.; Matthews, Ian T. W.; Moore, Stephen; O'Connor, Mark J.; Smith, Graeme C. M.; Martin, Niall M. B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6581-6591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase activation is an immediate cellular response to metabolic-, chem.-, or ionizing radiation-induced DNA damage and represents a new target for cancer therapy.  In this article, we disclose a novel series of substituted 4-benzyl-2H-phthalazin-1-ones that possess high inhibitory enzyme and cellular potency for both PARP-1 and PARP-2.  Optimized compds. from the series also demonstrate good pharmacokinetic profiles, oral bioavailability, and activity in vivo in an SW620 colorectal cancer xenograft model. 4-[3-(4-Cyclopropanecarbonylpiperazine-1-carbonyl)-4-fluorobenzyl]-2H-phthalazin-1-one (KU-0059436, AZD2281) 47 is a single digit nanomolar inhibitor of both PARP-1 and PARP-2 that shows standalone activity against BRCA1-deficient breast cancer cell lines.  Compd. 47 is currently undergoing clin. development for the treatment of BRCA1- and BRCA2-defective cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3IKtdcfeWCbVg90H21EOLACvtfcHk0lg6DGVRGmDYaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFeqs7vE&md5=45aff9bb506360ec26c1c0039388240e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm8001263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8001263%26sid%3Dliteratum%253Aachs%26aulast%3DMenear%26aufirst%3DK.%2BA.%26aulast%3DAdcock%26aufirst%3DC.%26aulast%3DBoulter%26aufirst%3DR.%26aulast%3DCockcroft%26aufirst%3DX.-l.%26aulast%3DCopsey%26aufirst%3DL.%26aulast%3DCranston%26aufirst%3DA.%26aulast%3DDillon%26aufirst%3DK.%2BJ.%26aulast%3DDrzewiecki%26aufirst%3DJ.%26aulast%3DGarman%26aufirst%3DS.%26aulast%3DGomez%26aufirst%3DS.%26aulast%3DJavaid%26aufirst%3DH.%26aulast%3DKerrigan%26aufirst%3DF.%26aulast%3DKnights%26aufirst%3DC.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DLoh%26aufirst%3DV.%2BM.%26aulast%3DMatthews%26aufirst%3DI.%2BT.%2BW.%26aulast%3DMoore%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%2BJ.%26aulast%3DSmith%26aufirst%3DG.%2BC.%2BM.%26aulast%3DMartin%26aufirst%3DN.%2BM.%2BB.%26atitle%3D4-%255B3-%25284-Cyclopropanecarbonylpiperazine-1-carbonyl%2529-4-fluorobenzyl%255D-2H-phthalazin-%25201-one%253A%2520a%2520novel%2520bioavailable%2520inhibitor%2520of%2520poly%2528ADP-ribose%2529%2520polymerase-1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D6581%26epage%3D6591%26doi%3D10.1021%2Fjm8001263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span> <span> </span><span class="NLM_article-title">INPP4B overexpression enhances the antitumor efficacy of PARP inhibitor AG014699 in MDA-MB-231 triple-negative breast cancer cells</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">4469</span>– <span class="NLM_lpage">4477</span>, <span class="refDoi"> DOI: 10.1007/s13277-013-1589-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1007%2Fs13277-013-1589-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntlOkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=4469-4477&author=Y.+Sunauthor=H.+Dingauthor=X.+Liuauthor=X.+Liauthor=L.+Li&title=INPP4B+overexpression+enhances+the+antitumor+efficacy+of+PARP+inhibitor+AG014699+in+MDA-MB-231+triple-negative+breast+cancer+cells&doi=10.1007%2Fs13277-013-1589-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">INPP4B overexpression enhances the antitumor efficacy of PARP inhibitor AG014699 in MDA-MB-231 triple-negative breast cancer cells</span></div><div class="casAuthors">Sun, Ying; Ding, Huan; Liu, Xinguang; Li, Xiaoqing; Li, Li</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">4469-4477</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Although preclin. and clin. studies on poly-(ADP ribose) polymerase (PARP) inhibitor alone or in combination with DNA-damaging agents have shown promising results, further research to improve and broaden the application scope of this therapeutic approach is needed.  The main aim of this study was to evaluate whether overexpressing inositol polyphosphate 4-phosphatase type II (INPP4B) gene, a novel tumor suppressor gene neg. regulating the phosphatidylinositol 3-kinase (PI3K)/AKT signaling pathway, could enhance the antitumor efficacy of PARP inhibitor AG014699 used in the treatment of triple-neg. breast cancer (TNBC).  Here in this report, we used a TNBC cell line MDA-MB-231 without expression of INPP4B as the study model and a lentiviral system to stably overexpress INPP4B gene in MDA-MB-231 cells.  We detected that the overexpression of INPP4B could significantly suppress cell proliferation and block cell cycle progression in G1 phase via decreasing the protein level of phosphorylated AKT.  It is further revealed that PARP inhibitor AG014699 induced DNA damage conferring a G2/M arrest and decreased cell viability, which is paralleled by the induction of apoptosis.  However, PARP inhibitor AG014699 could activate the PI3K/AKT signaling pathway activity and partially offset its therapeutic efficacy.  In our study, a significant enhancement of proliferation inhibition was obsd. when INPP4B overexpression was combined with PARP inhibitor AG014699 in comparison with either single treatment.  The suppression of PI3K/AKT pathway caused by the overexpression of INPP4B contributed to the enhanced antitumor efficacy of the combined therapy.  Our in vitro results indicated that this exptl. therapeutic strategy combining INPP4B overexpression and PARP inhibitor AG014699 might be of potential therapeutic value as a new strategy for the treatment of patients with TNBC and is worthy of further study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrck78_MFHw9bVg90H21EOLACvtfcHk0lhFDJgU4_Hrkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntlOkurY%253D&md5=ac96dcb8191167fdafae194bd5ce39d7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs13277-013-1589-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-013-1589-y%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26atitle%3DINPP4B%2520overexpression%2520enhances%2520the%2520antitumor%2520efficacy%2520of%2520PARP%2520inhibitor%2520AG014699%2520in%2520MDA-MB-231%2520triple-negative%2520breast%2520cancer%2520cells%26jtitle%3DTumor%2520Biol.%26date%3D2014%26volume%3D35%26spage%3D4469%26epage%3D4477%26doi%3D10.1007%2Fs13277-013-1589-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcoxen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toniatti, C.</span></span> <span> </span><span class="NLM_article-title">Niraparib: a poly(ADP-ribose) polymerase (PARP) inhibitor for the treatment of tumors with defective homologous recombination</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3302</span>– <span class="NLM_lpage">3314</span>, <span class="refDoi"> DOI: 10.1021/jm5018237</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5018237" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFams7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3302-3314&author=P.+Jonesauthor=K.+Wilcoxenauthor=M.+Rowleyauthor=C.+Toniatti&title=Niraparib%3A+a+poly%28ADP-ribose%29+polymerase+%28PARP%29+inhibitor+for+the+treatment+of+tumors+with+defective+homologous+recombination&doi=10.1021%2Fjm5018237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Niraparib: A Poly(ADP-ribose) Polymerase (PARP) Inhibitor for the Treatment of Tumors with Defective Homologous Recombination</span></div><div class="casAuthors">Jones, Philip; Wilcoxen, Keith; Rowley, Michael; Toniatti, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3302-3314</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerases (PARPs) are involved in DNA repair following damage by endogenous or exogenous processes.  It has become clear over the past decade that inhibition of PARP in the context of defects in other DNA repair mechanisms provide a tumor specific way to kill cancer cells.  We describe the rationale for this approach and the design and discovery of niraparib, a potent PARP-1/2 inhibitor with good cell based activity, selectivity for cancer over normal cells, and oral bioavailability.  Niraparib was characterized in a no. of preclin. models before moving to phase I clin. trials, where it showed excellent human pharmacokinetics suitable for once a day oral dosing, achieved its pharmacodynamic target for PARP inhibition, and had promising activity in cancer patients.  It is currently being tested in phase 3 clin. trials as maintenance therapy in ovarian cancer and as a treatment for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsaJDnhFDPhbVg90H21EOLACvtfcHk0lhFDJgU4_Hrkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFams7s%253D&md5=ffb7cb2503acde0a1e3452448d98693f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm5018237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5018237%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%26aulast%3DWilcoxen%26aufirst%3DK.%26aulast%3DRowley%26aufirst%3DM.%26aulast%3DToniatti%26aufirst%3DC.%26atitle%3DNiraparib%253A%2520a%2520poly%2528ADP-ribose%2529%2520polymerase%2520%2528PARP%2529%2520inhibitor%2520for%2520the%2520treatment%2520of%2520tumors%2520with%2520defective%2520homologous%2520recombination%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3302%26epage%3D3314%26doi%3D10.1021%2Fjm5018237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boshuizen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajrami, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Post, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">BMN 673, a novel and highly potent PARP1/2 inhibitor for the treatment of human cancers with DNA repair deficiency</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">5003</span>– <span class="NLM_lpage">5015</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-13-1391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1158%2F1078-0432.CCR-13-1391" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=23881923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVyns77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=5003-5015&author=Y.+Shenauthor=F.+L.+Rehmanauthor=Y.+Fengauthor=J.+Boshuizenauthor=I.+Bajramiauthor=R.+Elliottauthor=B.+Wangauthor=C.+J.+Lordauthor=L.+E.+Postauthor=A.+Ashworth&title=BMN+673%2C+a+novel+and+highly+potent+PARP1%2F2+inhibitor+for+the+treatment+of+human+cancers+with+DNA+repair+deficiency&doi=10.1158%2F1078-0432.ccr-13-1391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">BMN 673, a Novel and Highly Potent PARP1/2 Inhibitor for the Treatment of Human Cancers with DNA Repair Deficiency</span></div><div class="casAuthors">Shen, Yuqiao; Rehman, Farah L.; Feng, Ying; Boshuizen, Julia; Bajrami, Ilirjana; Elliott, Richard; Wang, Bing; Lord, Christopher J.; Post, Leonard E.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5003-5015</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: PARP1/2 inhibitors are a class of anticancer agents that target tumor-specific defects in DNA repair.  Here, we describe BMN 673, a novel, highly potent PARP1/2 inhibitor with favorable metabolic stability, oral bioavailability, and pharmacokinetic properties.  Exptl. Design: Potency and selectivity of BMN 673 was detd. by biochem. assays.  Anticancer activity either as a single-agent or in combination with other antitumor agents was evaluated both in vitro and in xenograft cancer models.  Results: BMN 673 is a potent PARP1/2 inhibitor (PARP1 IC50 = 0.57 nmol/L), but it does not inhibit other enzymes that we have tested.  BMN 673 exhibits selective antitumor cytotoxicity and elicits DNA repair biomarkers at much lower concns. than earlier generation PARP1/2 inhibitors (such as olaparib, rucaparib, and veliparib).  In vitro, BMN 673 selectively targeted tumor cells with BRCA1, BRCA2, or PTEN gene defects with 20- to more than 200-fold greater potency than existing PARP1/2 inhibitors.  BMN 673 is readily orally bioavailable, with more than 40% abs. oral bioavailability in rats when dosed in carboxylmethyl cellulose.  Oral administration of BMN 673 elicited remarkable antitumor activity in vivo; xenografted tumors that carry defects in DNA repair due to BRCA mutations or PTEN deficiency were profoundly sensitive to oral BMN 673 treatment at well-tolerated doses in mice.  Synergistic or additive antitumor effects were also found when BMN 673 was combined with temozolomide, SN38, or platinum drugs.  Conclusion: BMN 673 is currently in early-phase clin. development and represents a promising PARP1/2 inhibitor with potentially advantageous features in its drug class.  Clin Cancer Res; 19(18); 5003-15. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWdmHIVEuoz7Vg90H21EOLACvtfcHk0lhrQsiEMp6rgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVyns77J&md5=f58b67f14784c0a0fa1197caffd8d96d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-13-1391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-13-1391%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DRehman%26aufirst%3DF.%2BL.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DBoshuizen%26aufirst%3DJ.%26aulast%3DBajrami%26aufirst%3DI.%26aulast%3DElliott%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DPost%26aufirst%3DL.%2BE.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DBMN%2520673%252C%2520a%2520novel%2520and%2520highly%2520potent%2520PARP1%252F2%2520inhibitor%2520for%2520the%2520treatment%2520of%2520human%2520cancers%2520with%2520DNA%2520repair%2520deficiency%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26spage%3D5003%26epage%3D5015%26doi%3D10.1158%2F1078-0432.ccr-13-1391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, C.</span></span> <span> </span><span class="NLM_article-title">Therapeutic applications of PARP inhibitors: anticancer therapy and beyond</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1217</span>– <span class="NLM_lpage">1256</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2013.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.mam.2013.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=23370117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislyqtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=1217-1256&author=N.+J.+Curtinauthor=C.+Szabo&title=Therapeutic+applications+of+PARP+inhibitors%3A+anticancer+therapy+and+beyond&doi=10.1016%2Fj.mam.2013.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic applications of PARP inhibitors: Anticancer therapy and beyond</span></div><div class="casAuthors">Curtin, Nicola J.; Szabo, Csaba</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1217-1256</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The aim of this article is to describe the current and potential clin. translation of pharmacol. inhibitors of poly(ADP-ribose) polymerase (PARP) for the therapy of various diseases.  The first section of the present review summarizes the available preclin. and clin. data with PARP inhibitors in various forms of cancer.  In this context, the role of PARP in single-strand DNA break repair is relevant, leading to replication-assocd. lesions that cannot be repaired if homologous recombination repair (HRR) is defective, and the synthetic lethality of PARP inhibitors in HRR-defective cancer.  HRR defects are classically assocd. with BRCA1 and 2 mutations assocd. with familial breast and ovarian cancer, but there may be many other causes of HRR defects.  Thus, PARP inhibitors may be the drugs of choice for BRCA mutant breast and ovarian cancers, and extend beyond these tumors if appropriate biomarkers can be developed to identify HRR defects.  Multiple lines of preclin. data demonstrate that PARP inhibition increases cytotoxicity and tumor growth delay in combination with temozolomide, topoisomerase inhibitors and ionizing radiation.  Both single agent and combination clin. trials are underway.  The final part of the first section of the present review summarizes the current status of the various PARP inhibitors that are in various stages of clin. development.  The second section of the present review summarizes the role of PARP in selected non-oncol. indications.  In a no. of severe, acute diseases (such as stroke, neurotrauma, circulatory shock and acute myocardial infarction) the clin. translatability of PARP inhibition is supported by multiple lines of preclin. data, as well as observational data demonstrating PARP activation in human tissue samples.  In these disease indications, PARP overactivation due to oxidative and nitrative stress drives cell necrosis and pro-inflammatory gene expression, which contributes to disease pathol.  Accordingly, multiple lines of preclin. data indicate the efficacy of PARP inhibitors to preserve viable tissue and to down-regulate inflammatory responses.  As the clin. trials with PARP inhibitors in various forms of cancer progress, it is hoped that a second line of clin. investigations, aimed at testing of PARP inhibitors for various non-oncol. indications, will be initiated, as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP4INVXcZ0zbVg90H21EOLACvtfcHk0lhrQsiEMp6rgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislyqtrc%253D&md5=6a514e160666b5dde38e1365211cd41e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2013.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2013.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26aulast%3DSzabo%26aufirst%3DC.%26atitle%3DTherapeutic%2520applications%2520of%2520PARP%2520inhibitors%253A%2520anticancer%2520therapy%2520and%2520beyond%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2013%26volume%3D34%26spage%3D1217%26epage%3D1256%26doi%3D10.1016%2Fj.mam.2013.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Im, S.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senkus, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domchek, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaloge, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goessl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Runswick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conte, P.</span></span> <span> </span><span class="NLM_article-title">Olaparib for metastatic breast cancer in patients with a germline BRCA mutation</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>377</i></span>,  <span class="NLM_fpage">523</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1056/nejmoa1706450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1056%2FNEJMoa1706450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=28578601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlyksrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=377&publication_year=2017&pages=523-533&author=M.+Robsonauthor=S.-A.+Imauthor=E.+Senkusauthor=B.+Xuauthor=S.+M.+Domchekauthor=N.+Masudaauthor=S.+Delalogeauthor=W.+Liauthor=N.+Tungauthor=A.+Armstrongauthor=W.+Wuauthor=C.+Goesslauthor=S.+Runswickauthor=P.+Conte&title=Olaparib+for+metastatic+breast+cancer+in+patients+with+a+germline+BRCA+mutation&doi=10.1056%2Fnejmoa1706450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Olaparib for metastatic breast cancer in patients with a germline BRCA mutation</span></div><div class="casAuthors">Robson, Mark; Im, Seock-Ah; Senkus, Elzbieta; Xu, Binghe; Domchek, Susan M.; Masuda, Norikazu; Delaloge, Suzette; Li, Wei; Tung, Nadine; Armstrong, Anne; Wu, Wenting; Goessl, Carsten; Runswick, Sarah; Conte, Pierfranco</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">377</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">523-533</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Olaparib is an oral poly(ADP-ribose) polymerase inhibitor that has promising antitumor activity in patients with metastatic breast cancer and a germline BRCA mutation.  We conducted a randomized, open-label, phase 3 trial in which olaparib monotherapy was compared with std. therapy in patients with a germline BRCA mutation and human epidermal growth factor receptor type 2 (HER2)-neg. metastatic breast cancer who had received no more than two previous chemotherapy regimens for metastatic disease.  Patients were randomly assigned, in a 2:1 ratio, to receive olaparib tablets (300 mg twice daily) or std. therapy with single-agent chemotherapy of the physician's choice (capecitabine, eribulin, or vinorelbine in 21-day cycles).  The primary end point was progression-free survival, which was assessed by blinded independent central review and was analyzed on an intention-to-treat basis. results Of the 302 patients who underwent randomization, 205 were assigned to receive olaparib and 97 were assigned to receive std. therapy.  Median progression-free survival was significantly longer in the olaparib group than in the std.-therapy group (7.0 mo vs. 4.2 mo; hazard ratio for disease progression or death, 0.58; 95% confidence interval, 0.43 to 0.80; P<0.001).  The response rate was 59.9% in the olaparib group and 28.8% in the std.-therapy group.  The rate of grade 3 or higher adverse events was 36.6% in the olaparib group and 50.5% in the std.-therapy group, and the rate of treatment discontinuation due to toxic effects was 4.9% and 7.7%, resp.  Among patients with HER2-neg. metastatic breast cancer and a germline BRCA mutation, olaparib monotherapy provided a significant benefit over std. therapy; median progression-free survival was 2.8 mo longer and the risk of disease progression or death was 42% lower with olaparib monotherapy than with std. therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMXxR-ZN4IGrVg90H21EOLACvtfcHk0lhrQsiEMp6rgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlyksrbN&md5=7c8c62182a717c16c5762443fb3f6d5d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1706450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1706450%26sid%3Dliteratum%253Aachs%26aulast%3DRobson%26aufirst%3DM.%26aulast%3DIm%26aufirst%3DS.-A.%26aulast%3DSenkus%26aufirst%3DE.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DDomchek%26aufirst%3DS.%2BM.%26aulast%3DMasuda%26aufirst%3DN.%26aulast%3DDelaloge%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DTung%26aufirst%3DN.%26aulast%3DArmstrong%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DGoessl%26aufirst%3DC.%26aulast%3DRunswick%26aufirst%3DS.%26aulast%3DConte%26aufirst%3DP.%26atitle%3DOlaparib%2520for%2520metastatic%2520breast%2520cancer%2520in%2520patients%2520with%2520a%2520germline%2520BRCA%2520mutation%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2017%26volume%3D377%26spage%3D523%26epage%3D533%26doi%3D10.1056%2Fnejmoa1706450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J.</span></span> <span> </span><span class="NLM_article-title">Laying a trap to kill cancer cells: PARP inhibitors and their mechanisms of action</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">362ps17</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aaf9246</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1126%2Fscitranslmed.aaf9246" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=27797957" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=362ps17&author=Y.+Pommierauthor=M.+J.+O%E2%80%99Connorauthor=J.+de+Bono&title=Laying+a+trap+to+kill+cancer+cells%3A+PARP+inhibitors+and+their+mechanisms+of+action&doi=10.1126%2Fscitranslmed.aaf9246"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aaf9246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aaf9246%26sid%3Dliteratum%253Aachs%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%2BJ.%26aulast%3Dde%2BBono%26aufirst%3DJ.%26atitle%3DLaying%2520a%2520trap%2520to%2520kill%2520cancer%2520cells%253A%2520PARP%2520inhibitors%2520and%2520their%2520mechanisms%2520of%2520action%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2016%26volume%3D8%26spage%3D362ps17%26doi%3D10.1126%2Fscitranslmed.aaf9246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">PARP trapping beyond homologous recombination and platinum sensitivity in cancers</span>. <i>Ann. Rev. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1146/annurev-cancerbio-030518-055914</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1146%2Fannurev-cancerbio-030518-055914" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2019&pages=131-150&author=J.+Muraiauthor=Y.+Pommier&title=PARP+trapping+beyond+homologous+recombination+and+platinum+sensitivity+in+cancers&doi=10.1146%2Fannurev-cancerbio-030518-055914"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1146%2Fannurev-cancerbio-030518-055914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-cancerbio-030518-055914%26sid%3Dliteratum%253Aachs%26aulast%3DMurai%26aufirst%3DJ.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DPARP%2520trapping%2520beyond%2520homologous%2520recombination%2520and%2520platinum%2520sensitivity%2520in%2520cancers%26jtitle%3DAnn.%2520Rev.%2520Cancer%2520Biol.%26date%3D2019%26volume%3D3%26spage%3D131%26epage%3D150%26doi%3D10.1146%2Fannurev-cancerbio-030518-055914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">PARP inhibitors: synthetic lethality in the clinic</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1152</span>– <span class="NLM_lpage">1158</span>, <span class="refDoi"> DOI: 10.1126/science.aam7344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1126%2Fscience.aam7344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=28302823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1152-1158&author=C.+J.+Lordauthor=A.+Ashworth&title=PARP+inhibitors%3A+synthetic+lethality+in+the+clinic&doi=10.1126%2Fscience.aam7344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">PARP inhibitors: Synthetic lethality in the clinic</span></div><div class="casAuthors">Lord, Christopher J.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6330</span>),
    <span class="NLM_cas:pages">1152-1158</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  PARP inhibitors (PARPi), a cancer therapy targeting poly(ADP-ribose) polymerase, are the first clin. approved drugs designed to exploit synthetic lethality, a genetic concept proposed nearly a century ago.  Tumors arising in patients who carry germline mutations in either BRCA1 or BRCA2 are sensitive to PARPi because they have a specific type of DNA repair defect.  PARPi also show promising activity in more common cancers that share this repair defect.  However, as with other targeted therapies, resistance to PARPi arises in advanced disease.  In addn., detg. the optimal use of PARPi within drug combination approaches has been challenging.  Nevertheless, the preclin. discovery of PARPi synthetic lethality and the route to clin. approval provide interesting lessons for the development of other therapies.  Here, the authors discuss current knowledge of PARP inhibitors and potential ways to maximize their clin. effectiveness.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAdgOVt6r40rVg90H21EOLACvtfcHk0lin1FvnQ4rjgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmtrg%253D&md5=6f3c5c37b2d5062cb7acbf3a78727f46</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1126%2Fscience.aam7344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aam7344%26sid%3Dliteratum%253Aachs%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DPARP%2520inhibitors%253A%2520synthetic%2520lethality%2520in%2520the%2520clinic%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1152%26epage%3D1158%26doi%3D10.1126%2Fscience.aam7344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Endres, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waeber, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moskowitz, M. A.</span></span> <span> </span><span class="NLM_article-title">Ischemic brain injury is mediated by the activation of poly(ADP-ribose)polymerase</span>. <i>J. Cereb. Blood Flow Metab.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1143</span>– <span class="NLM_lpage">1151</span>, <span class="refDoi"> DOI: 10.1097/00004647-199711000-00002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1097%2F00004647-199711000-00002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=9390645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK2sXotVShtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=1143-1151&author=M.+Endresauthor=Z.-Q.+Wangauthor=S.+Namuraauthor=C.+Waeberauthor=M.+A.+Moskowitz&title=Ischemic+brain+injury+is+mediated+by+the+activation+of+poly%28ADP-ribose%29polymerase&doi=10.1097%2F00004647-199711000-00002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Ischemic brain injury is mediated by the activation of poly(ADP-ribose)polymerase</span></div><div class="casAuthors">Endres, Matthias; Wang, Zhao-Qi; Namura, Shobu; Waeber, Christian; Moskowitz, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cerebral Blood Flow and Metabolism</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1143-1151</span>CODEN:
                <span class="NLM_cas:coden">JCBMDN</span>;
        ISSN:<span class="NLM_cas:issn">0271-678X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott-Raven Publishers</span>)
        </div><div class="casAbstract">Poly(ADP-ribose)polymerase (PARP, EC 2.4.2.30), an abundant nuclear protein activated by DNA nicks, mediates cell death in vitro by NAD (NAD) depletion after exposure to nitric oxide.  The authors examd. whether genetic deletion of PARP (PARP null mice) or its pharmacol. inhibition by 3-aminobenzamide (3-AB) attenuates tissue injury after transient cerebral ischemia.  Twenty-two hours after reperfusion following 2 h of filamentous middle cerebral artery occlusion, ischemic injury was decreased in PARP-/- and PARP+/- mice compared with PARP+/+ litter mates, and also was attenuated in 129/SV wild-type mice after 3-AB treatment compared with controls.  Infarct sparing was accompanied by functional recovery in PARP-/- and 3-AB-treated mice.  Increased poly(ADP-ribose) immunostaining obsd. in ischemic cell nuclei 5 min after reperfusion was reduced by 3-AB treatment.  Levels of NAD-the substrate of PARP-were reduced 2 h after reperfusion and were 35% of contralateral levels at 24 h.  The decreases were attenuated in PARP-/- mice and in 3-AB-treated animals.  Poly(ADP-ribose)polymerase cleavage by caspase-3 (CPP-32) has been proposed as an important step in apoptotic cell death.  Markers of apoptosis, such as oligonucleosomal DNA damage, total DNA fragmentation, and the d. of terminal deoxy-nucleotidyl transferase dUTP nick-end-labeled (TUNEL +) cells, however, did not differ in ischemic brain tissue of PARP-/- mice or in 3-AB-treated animals vs. controls, although there were differences in the no. of TUNEL-stained cells reflecting the decrease in infarct size.  Thus, ischemic brain injury activates PARP and contributes to cell death most likely by NAD depletion and energy failure, although the authors have not excluded a role for PARP in apoptotic cell death at earlier or later stages in ischemic cell death.  Inhibitors of PARP activation could provide a potential therapy in acute stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZgfj6WehsELVg90H21EOLACvtfcHk0lin1FvnQ4rjgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXotVShtbo%253D&md5=38ad29424df8e6f3613cab8c38b0dfcb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1097%2F00004647-199711000-00002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00004647-199711000-00002%26sid%3Dliteratum%253Aachs%26aulast%3DEndres%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DZ.-Q.%26aulast%3DNamura%26aufirst%3DS.%26aulast%3DWaeber%26aufirst%3DC.%26aulast%3DMoskowitz%26aufirst%3DM.%2BA.%26atitle%3DIschemic%2520brain%2520injury%2520is%2520mediated%2520by%2520the%2520activation%2520of%2520poly%2528ADP-ribose%2529polymerase%26jtitle%3DJ.%2520Cereb.%2520Blood%2520Flow%2520Metab.%26date%3D1997%26volume%3D17%26spage%3D1143%26epage%3D1151%26doi%3D10.1097%2F00004647-199711000-00002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eliasson, M. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampei, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandir, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurn, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traystman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose) polymerase gene disruption renders mice resistant to cerebral ischemia</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1095</span>, <span class="refDoi"> DOI: 10.1038/nm1097-1089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnm1097-1089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=9334719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK2sXmsFyqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1997&pages=1089-1095&author=M.+J.+L.+Eliassonauthor=K.+Sampeiauthor=A.+S.+Mandirauthor=P.+D.+Hurnauthor=R.+J.+Traystmanauthor=J.+Baoauthor=A.+Pieperauthor=Z.-Q.+Wangauthor=T.+M.+Dawsonauthor=S.+H.+Snyderauthor=V.+L.+Dawson&title=Poly%28ADP-ribose%29+polymerase+gene+disruption+renders+mice+resistant+to+cerebral+ischemia&doi=10.1038%2Fnm1097-1089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribose) polymerase gene disruption renders mice resistant to cerebral ischemia</span></div><div class="casAuthors">Eliasson, Mikael J. L.; Sampei, Kenji; Mandir, Allen S.; Hurn, Patricia D.; Traystman, Richard J.; Bao, Jun; Pieper, Andrew; Wang, Zhao-Qi; Dawson, Ted M.; Snyder, Solomon H.; Dawson, Valina L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1089-1095</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Nitric oxide (NO) and peroxynitrite, formed from NO and superoxide anion, have been implicated as mediators of neuronal damage following focal ischemia, but their mol. targets have not been defined.  One candidate pathway is DNA damage leading to activation of the nuclear enzyme, poly(ADP-ribose) polymerase (PARP), which catalyzes attachment of ADP ribose units from NAD to nuclear proteins following DNA damage.  Excessive activation of PARP can deplete NAD and ATP, which is consumed in regeneration of NAD, leading to cell death by energy depletion.  The authors show that genetic disruption of PARP provides profound protection against glutamate-NO-mediated ischemic insults in vitro and major decreases in infarct vol. after reversible middle cerebral artery occlusion.  These results provide compelling evidence for a primary involvement of PARP activation in neuronal damage following focal ischemia and suggest that therapies designed towards inhibiting PARP may provide benefit in the treatment of cerebrovascular disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLSq-7dMGKE7Vg90H21EOLACvtfcHk0lhF276UzqTGbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsFyqtb8%253D&md5=d83a2bb75e066d20b415119fb4452e21</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnm1097-1089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1097-1089%26sid%3Dliteratum%253Aachs%26aulast%3DEliasson%26aufirst%3DM.%2BJ.%2BL.%26aulast%3DSampei%26aufirst%3DK.%26aulast%3DMandir%26aufirst%3DA.%2BS.%26aulast%3DHurn%26aufirst%3DP.%2BD.%26aulast%3DTraystman%26aufirst%3DR.%2BJ.%26aulast%3DBao%26aufirst%3DJ.%26aulast%3DPieper%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DZ.-Q.%26aulast%3DDawson%26aufirst%3DT.%2BM.%26aulast%3DSnyder%26aufirst%3DS.%2BH.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26atitle%3DPoly%2528ADP-ribose%2529%2520polymerase%2520gene%2520disruption%2520renders%2520mice%2520resistant%2520to%2520cerebral%2520ischemia%26jtitle%3DNat.%2520Med.%26date%3D1997%26volume%3D3%26spage%3D1089%26epage%3D1095%26doi%3D10.1038%2Fnm1097-1089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pieper, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walles, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clements, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweier, J. L.</span></span> <span> </span><span class="NLM_article-title">Myocardial postischemic injury is reduced by polyADPripose polymerase-1 gene disruption</span>. <i>Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1007/bf03401936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1007%2FBF03401936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10949908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlsFWjsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=271-282&author=A.+A.+Pieperauthor=T.+Wallesauthor=G.+Weiauthor=E.+E.+Clementsauthor=A.+Vermaauthor=S.+H.+Snyderauthor=J.+L.+Zweier&title=Myocardial+postischemic+injury+is+reduced+by+polyADPripose+polymerase-1+gene+disruption&doi=10.1007%2Fbf03401936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Myocardial postischemic injury is reduced by polyADPribose polymerase-1 gene disruption</span></div><div class="casAuthors">Pieper, Andrew A.; Walles, Thorsten; Wei, Guo; Clements, Emily E.; Verma, Ajay; Snyder, Solomon H.; Zweier, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine (New York)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-282</span>CODEN:
                <span class="NLM_cas:coden">MOMEF3</span>;
        ISSN:<span class="NLM_cas:issn">1076-1551</span>.
    
            (<span class="NLM_cas:orgname">Johns Hopkins University Press</span>)
        </div><div class="casAbstract">PolyADPribose polymerase (PARP) is activated by DNA strand breaks to catalyze the addn. of ADPribose groups to nuclear proteins, esp. PARP-1.  Excessive polyADPribosylation leads to cell death through depletion of NAD+ and ATP.  In vivo PARP activation in heart tissue slices was assayed through conversion of [33P]NAD+ into polyADPribose (PAR) following ischemia-reperfusion (I/R) and also monitored by immunohistochem. staining for PAR.  Cardiac contractility, nitric oxide (NO), reactive oxygen species (ROS), NAD+ and ATP levels were examd. in wild type (WT) and in PARP-1 gene-deleted (PARP-1-/-) isolated, perfused mouse hearts.  Myocardial infarct size was assessed following coronary artery occlusion in rats treated with PARP inhibitors.  Ischemia-reperfusion (I/R) augmented formation of nitric oxide, oxygen free radicals and PARP activity.  I/R induced decreases in cardiac contractility and NAD+ levels were attenuated in PARP-1-/- mouse hearts.  PARP inhibitors reduced myocardial infarct size in rats.  Residual polyADPribosylation in PARP-1-/- hearts may reflect alternative forms of PARP.  PolyADPribosylation from PARP-1 and other sources of enzymic PAR synthesis is assocd. with cardiac damage following myocardial ischemia.  PARP inhibitors may have therapeutic utility in myocardial disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrarESEjtYqqrVg90H21EOLACvtfcHk0lhF276UzqTGbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlsFWjsrs%253D&md5=7c656ce837c86eca120242f39d94a2ab</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2FBF03401936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF03401936%26sid%3Dliteratum%253Aachs%26aulast%3DPieper%26aufirst%3DA.%2BA.%26aulast%3DWalles%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DClements%26aufirst%3DE.%2BE.%26aulast%3DVerma%26aufirst%3DA.%26aulast%3DSnyder%26aufirst%3DS.%2BH.%26aulast%3DZweier%26aufirst%3DJ.%2BL.%26atitle%3DMyocardial%2520postischemic%2520injury%2520is%2520reduced%2520by%2520polyADPripose%2520polymerase-1%2520gene%2520disruption%26jtitle%3DMol.%2520Med.%26date%3D2000%26volume%3D6%26spage%3D271%26epage%3D282%26doi%3D10.1007%2Fbf03401936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devalaraja-Narashimha, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singaravelu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padanilam, B. J.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose) polymerase-1 gene ablation protects mice from ischemic renal injury</span>. <i>Am. J. Physiol.: Renal, Fluid Electrolyte Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">F387</span>– <span class="NLM_lpage">F398</span>, <span class="refDoi"> DOI: 10.1152/ajprenal.00436.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1152%2Fajprenal.00436.2003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2005&pages=F387-F398&author=J.+Zhengauthor=K.+Devalaraja-Narashimhaauthor=K.+Singaraveluauthor=B.+J.+Padanilam&title=Poly%28ADP-ribose%29+polymerase-1+gene+ablation+protects+mice+from+ischemic+renal+injury&doi=10.1152%2Fajprenal.00436.2003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1152%2Fajprenal.00436.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajprenal.00436.2003%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDevalaraja-Narashimha%26aufirst%3DK.%26aulast%3DSingaravelu%26aufirst%3DK.%26aulast%3DPadanilam%26aufirst%3DB.%2BJ.%26atitle%3DPoly%2528ADP-ribose%2529%2520polymerase-1%2520gene%2520ablation%2520protects%2520mice%2520from%2520ischemic%2520renal%2520injury%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Renal%252C%2520Fluid%2520Electrolyte%2520Physiol.%26date%3D2005%26volume%3D288%26spage%3DF387%26epage%3DF398%26doi%3D10.1152%2Fajprenal.00436.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zingarelli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabó, C.</span></span> <span> </span><span class="NLM_article-title">Effect of genetic disruption of poly(ADP-ribose) synthetase on delayed production of inflammatory mediators and delayed necrosis during myocardial ischemia-reperfusion injury</span>. <i>Shock</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">60</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1097/00024382-200013010-00011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1097%2F00024382-200013010-00011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10638671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1ajuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2000&pages=60-66&author=Z.+Yangauthor=B.+Zingarelliauthor=C.+Szab%C3%B3&title=Effect+of+genetic+disruption+of+poly%28ADP-ribose%29+synthetase+on+delayed+production+of+inflammatory+mediators+and+delayed+necrosis+during+myocardial+ischemia-reperfusion+injury&doi=10.1097%2F00024382-200013010-00011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of genetic disruption of poly (ADP-ribose) synthetase on delayed production of inflammatory mediators and delayed necrosis during myocardial ischemia-reperfusion injury</span></div><div class="casAuthors">Yang Z; Zingarelli B; Szabo C</div><div class="citationInfo"><span class="NLM_cas:title">Shock (Augusta, Ga.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">60-6</span>
        ISSN:<span class="NLM_cas:issn">1073-2322</span>.
    </div><div class="casAbstract">The nuclear enzyme poly (ADP ribose) synthetase (PARS) has been shown to play an important role in the pathogenesis of various forms of ischemia or reperfusion injury and circulatory shock.  Recent studies demonstrated that inhibition or genetic inactivation of PARS is beneficial in the early phase of myocardial reperfusion injury.  The aim of the present study was to investigate whether inactivation of PARS influences the delayed myocardial necrosis and the production of the proinflammatory cytokine tumor necrosis factor alpha (TNFalpha), the anti-inflammatory cytokine interleukin-10 (IL-10), and the free radical nitric oxide in the late stage of myocardial reperfusion injury.  The results demonstrate that genetic disruption of PARS provides marked protection against the delayed myocardial ischemia and reperfusion injury.  In addition, in the absence of functional PARS, a suppression of TNFalpha, IL-10, and nitric oxide production was found.  These findings provide direct evidence that PARS activation participates in the development of delayed cell injury and delayed mediator production in myocardial reperfusion injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7mQt-ICR3qNfW6udTcc2ebpFciHALyKa7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1ajuw%253D%253D&md5=755aeae7fcb37384e374d064f00c9e04</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1097%2F00024382-200013010-00011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00024382-200013010-00011%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DZingarelli%26aufirst%3DB.%26aulast%3DSzab%25C3%25B3%26aufirst%3DC.%26atitle%3DEffect%2520of%2520genetic%2520disruption%2520of%2520poly%2528ADP-ribose%2529%2520synthetase%2520on%2520delayed%2520production%2520of%2520inflammatory%2520mediators%2520and%2520delayed%2520necrosis%2520during%2520myocardial%2520ischemia-reperfusion%2520injury%26jtitle%3DShock%26date%3D2000%26volume%3D13%26spage%3D60%26epage%3D66%26doi%3D10.1097%2F00024382-200013010-00011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szabó, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span> <span> </span><span class="NLM_article-title">Role of poly(ADP-ribose)synthetase in inflammation</span>. <i>Trends Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">298</span>, <span class="refDoi"> DOI: 10.1016/s0165-6147(98)01193-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2FS0165-6147%2898%2901193-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=9703762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK1cXltFSjsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1998&pages=287-298&author=C.+Szab%C3%B3author=V.+L.+Dawson&title=Role+of+poly%28ADP-ribose%29synthetase+in+inflammation&doi=10.1016%2Fs0165-6147%2898%2901193-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Role of poly(ADP-ribose) synthetase in inflammation and ischemia-reperfusion</span></div><div class="casAuthors">Szabo, Csaba; Dawson, Valina L.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">287-298</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 111 refs.  Oxidative and nitrosative stress can trigger DNA strand breakage, which then activates the nuclear enzyme poly(ADP-ribose) synthetase (PARS).  This enzyme has also been termed poly(ADP-ribose) polymerase (PARP) or poly(ADP-ribose) transferase (pADPRT).  Rapid activation of the enzyme depletes the intracellular concn. of its substrate, NAD, thus slowing the rate of glycolysis, electron transport and subsequently ATP formation.  This process can result in cell dysfunction and cell death.  In this article, the authors overview the impact of pharmacol. inhibition or genetic inactivation of PARS on the course of oxidant-induced cell death in vitro, and in inflammation and reperfusion injury in vivo.  A major trigger for DNA damage in pathophysiol. condition is peroxynitrite, a cytotoxic oxidant formed by the reaction between the free radicals nitric oxide and superoxide.  The pharmacol. inhibition of PARS is a novel approach for the exptl. therapy of various forms of inflammation and shock, stroke, myocardial and intestinal ischemia-reperfusion, and diabetes mellitus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0C66lRIi1hrVg90H21EOLACvtfcHk0liixYbxBBvRCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltFSjsLo%253D&md5=18dac83ea54b17395997e772e2630252</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0165-6147%2898%2901193-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0165-6147%252898%252901193-6%26sid%3Dliteratum%253Aachs%26aulast%3DSzab%25C3%25B3%26aufirst%3DC.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26atitle%3DRole%2520of%2520poly%2528ADP-ribose%2529synthetase%2520in%2520inflammation%26jtitle%3DTrends%2520Pharm.%2520Sci.%26date%3D1998%26volume%3D19%26spage%3D287%26epage%3D298%26doi%3D10.1016%2Fs0165-6147%2898%2901193-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">D’Amours, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desnoyers, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Silva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, G. G.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribosyl)ation reactions in the regulation of nuclear functions</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>342</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">268</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1042%2Fbj3420249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10455009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK1MXmtFChtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=1999&pages=249-268&author=D.+D%E2%80%99Amoursauthor=S.+Desnoyersauthor=I.+D%E2%80%99Silvaauthor=G.+G.+Poirier&title=Poly%28ADP-ribosyl%29ation+reactions+in+the+regulation+of+nuclear+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribosyl)ation reactions in the regulation of nuclear functions</span></div><div class="casAuthors">D'Amours, Damien; Desnoyers, Serge; D'Silva, Icy; Poirier, Guy G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-268</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review with 351 refs.  Poly(ADP-ribosyl)ation is a post-translational modification of proteins.  During this process, mols. of ADP-ribose are added successively on to acceptor proteins to form branched polymers.  This modification is transient but very extensive in vivo, as polymer chains can reach more than 200 units on protein acceptors.  The existence of the poly(ADP-ribose) polymer was first reported nearly 40 yr ago.  Since then, the importance of poly(ADP-ribose) synthesis has been established in many cellular processes.  However, a clear and unified picture of the physiol. role of poly(ADP-ribosyl)ation still remains to be established.  The total dependence of poly(ADP-ribose) synthesis on DNA strand breaks strongly suggests that this post-translational modification is involved in the metab. of nucleic acids.  This view is also supported by the identification of direct protein-protein interactions involving poly(ADP-ribose) polymerase (113 kDa PARP), an enzyme catalyzing the formation of poly(ADP-ribose), and key effectors of DNA repair, replication and transcription reactions.  The presence of PARP in these multiprotein complexes, in addn. to the actual poly(ADP-ribosyl)ation of some components of these complexes, clearly supports an important role for poly(ADP-ribosyl)ation reactions in DNA transactions.  Accordingly, inhibition of poly(ADP-ribose) synthesis by any of several approaches and the anal. of PARP-deficient cells has revealed that the absence of poly(ADP-ribosyl)ation strongly affects DNA metab., most notably DNA repair.  The recent identification of new poly(ADP-ribosyl)ating enzymes with distinct (non-std.) structures in eukaryotes and archaea has revealed a novel level of complexity in the regulation of poly(ADP-ribose) metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh1dXVUbI2MbVg90H21EOLACvtfcHk0liixYbxBBvRCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmtFChtLo%253D&md5=9a9e9c4a9c492cf6841fe146a22f21c5</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1042%2Fbj3420249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj3420249%26sid%3Dliteratum%253Aachs%26aulast%3DD%25E2%2580%2599Amours%26aufirst%3DD.%26aulast%3DDesnoyers%26aufirst%3DS.%26aulast%3DD%25E2%2580%2599Silva%26aufirst%3DI.%26aulast%3DPoirier%26aufirst%3DG.%2BG.%26atitle%3DPoly%2528ADP-ribosyl%2529ation%2520reactions%2520in%2520the%2520regulation%2520of%2520nuclear%2520functions%26jtitle%3DBiochem.%2520J.%26date%3D1999%26volume%3D342%26spage%3D249%26epage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fatokun, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, T. M.</span></span> <span> </span><span class="NLM_article-title">Parthanatos: mitochondrial-linked mechanisms and therapeutic opportunities</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">2000</span>– <span class="NLM_lpage">2016</span>, <span class="refDoi"> DOI: 10.1111/bph.12416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1111%2Fbph.12416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=24684389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFejs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2014&pages=2000-2016&author=A.+A.+Fatokunauthor=V.+L.+Dawsonauthor=T.+M.+Dawson&title=Parthanatos%3A+mitochondrial-linked+mechanisms+and+therapeutic+opportunities&doi=10.1111%2Fbph.12416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Parthanatos: mitochondrial-linked mechanisms and therapeutic opportunities</span></div><div class="casAuthors">Fatokun, Amos A.; Dawson, Valina L.; Dawson, Ted M.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2000-2016</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Cells die by a variety of mechanisms.  Terminally differentiated cells such as neurons die in a variety of disorders, in part, via parthanatos, a process dependent on the activity of poly (ADP-ribose)-polymerase (PARP).  Parthanatos does not require the mediation of caspases for its execution, but is clearly mechanistically dependent on the nuclear translocation of the mitochondrial-assocd. apoptosis-inducing factor (AIF).  The nuclear translocation of this otherwise beneficial mitochondrial protein, occasioned by poly (ADP-ribose) (PAR) produced through PARP overactivation, causes large-scale DNA fragmentation and chromatin condensation, leading to cell death.  This review describes the multistep course of parthanatos and its dependence on PAR signalling and nuclear AIF translocation.  The review also discusses potential targets in the parthanatos cascade as promising avenues for the development of novel, disease-modifying, therapeutic agents.  Linked Articles : This article is part of a themed issue on Mitochondrial Pharmacol.: Energy, Injury & Beyond.  To view the other articles in this issue visit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa2yn_v2ypBLVg90H21EOLACvtfcHk0liixYbxBBvRCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFejs78%253D&md5=e5a7843194a12539d988121c6d9e859c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2Fbph.12416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.12416%26sid%3Dliteratum%253Aachs%26aulast%3DFatokun%26aufirst%3DA.%2BA.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26aulast%3DDawson%26aufirst%3DT.%2BM.%26atitle%3DParthanatos%253A%2520mitochondrial-linked%2520mechanisms%2520and%2520therapeutic%2520opportunities%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2014%26volume%3D171%26spage%3D2000%26epage%3D2016%26doi%3D10.1111%2Fbph.12416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berger, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besson, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulares, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bürkle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarugi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuzzocrea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haskó, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liaudet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacher, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radermacher, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strosznajder, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sümegi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, C.</span></span> <span> </span><span class="NLM_article-title">Opportunities for the repurposing of PARP inhibitors for the therapy of non-oncological diseases</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">192</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1111/bph.13748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1111%2Fbph.13748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=28213892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvV2gt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=192-222&author=N.+A.+Bergerauthor=V.+C.+Bessonauthor=A.+H.+Boularesauthor=A.+B%C3%BCrkleauthor=A.+Chiarugiauthor=R.+S.+Clarkauthor=N.+J.+Curtinauthor=S.+Cuzzocreaauthor=T.+M.+Dawsonauthor=V.+L.+Dawsonauthor=G.+Hask%C3%B3author=L.+Liaudetauthor=F.+Moroniauthor=P.+Pacherauthor=P.+Radermacherauthor=A.+L.+Salzmanauthor=S.+H.+Snyderauthor=F.+G.+Sorianoauthor=R.+P.+Strosznajderauthor=B.+S%C3%BCmegiauthor=R.+A.+Swansonauthor=C.+Szabo&title=Opportunities+for+the+repurposing+of+PARP+inhibitors+for+the+therapy+of+non-oncological+diseases&doi=10.1111%2Fbph.13748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities for the repurposing of PARP inhibitors for the therapy of non-oncological diseases</span></div><div class="casAuthors">Berger, Nathan A.; Besson, Valerie C.; Boulares, A. Hamid; Buerkle, Alexander; Chiarugi, Alberto; Clark, Robert S.; Curtin, Nicola J.; Cuzzocrea, Salvatore; Dawson, Ted M.; Dawson, Valina L.; Hasko, Gyoergy; Liaudet, Lucas; Moroni, Flavio; Pacher, Pal; Radermacher, Peter; Salzman, Andrew L.; Snyder, Solomon H.; Soriano, Francisco Garcia; Strosznajder, Robert P.; Suemegi, Balazs; Swanson, Raymond A.; Szabo, Csaba</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">192-222</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The recent clin. availability of the PARP inhibitor olaparib (Lynparza) opens the door for potential therapeutic repurposing for non-oncol. indications.  Considering (a) the preclin. efficacy data with PARP inhibitors in non-oncol. diseases and (b) the risk-benefit ratio of treating patients with a compd. that inhibits an enzyme that has physiol. roles in the regulation of DNA repair, we have selected indications, where (a) the severity of the disease is high, (b) the available therapeutic options are limited, and (c) the duration of PARP inhibitor administration could be short, to provide first-line options for therapeutic repurposing.  These indications are as follows: acute ischemic stroke; traumatic brain injury; septic shock; acute pancreatitis; and severe asthma and severe acute lung injury.  In addn., chronic, devastating diseases, where alternative therapeutic options cannot halt disease development (e.g. Parkinson's disease, progressive multiple sclerosis or severe fibrotic diseases), should also be considered.  We present a preclin. and clin. action plan for the repurposing of PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2DK53xgBjdLVg90H21EOLACvtfcHk0ljkQTQcK7N1yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvV2gt7Y%253D&md5=213c71ccba975e61c8ff24ad8c28640f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1111%2Fbph.13748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13748%26sid%3Dliteratum%253Aachs%26aulast%3DBerger%26aufirst%3DN.%2BA.%26aulast%3DBesson%26aufirst%3DV.%2BC.%26aulast%3DBoulares%26aufirst%3DA.%2BH.%26aulast%3DB%25C3%25BCrkle%26aufirst%3DA.%26aulast%3DChiarugi%26aufirst%3DA.%26aulast%3DClark%26aufirst%3DR.%2BS.%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26aulast%3DCuzzocrea%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DT.%2BM.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26aulast%3DHask%25C3%25B3%26aufirst%3DG.%26aulast%3DLiaudet%26aufirst%3DL.%26aulast%3DMoroni%26aufirst%3DF.%26aulast%3DPacher%26aufirst%3DP.%26aulast%3DRadermacher%26aufirst%3DP.%26aulast%3DSalzman%26aufirst%3DA.%2BL.%26aulast%3DSnyder%26aufirst%3DS.%2BH.%26aulast%3DSoriano%26aufirst%3DF.%2BG.%26aulast%3DStrosznajder%26aufirst%3DR.%2BP.%26aulast%3DS%25C3%25BCmegi%26aufirst%3DB.%26aulast%3DSwanson%26aufirst%3DR.%2BA.%26aulast%3DSzabo%26aufirst%3DC.%26atitle%3DOpportunities%2520for%2520the%2520repurposing%2520of%2520PARP%2520inhibitors%2520for%2520the%2520therapy%2520of%2520non-oncological%2520diseases%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2018%26volume%3D175%26spage%3D192%26epage%3D222%26doi%3D10.1111%2Fbph.13748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0ljkQTQcK7N1yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-Molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0ljkQTQcK7N1yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0licB2dhXNCjvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0licB2dhXNCjvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.-Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0licB2dhXNCjvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lhl-aKkCO8JQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET) proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=L.+Liuauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+Yangauthor=D.+N.+Ziazadehauthor=F.+Xuauthor=J.+Huauthor=W.+Xiangauthor=L.+Huangauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29+proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0lhl-aKkCO8JQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=941-964&author=X.+Hanauthor=C.+Wangauthor=C.+Qinauthor=W.+Xiangauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=M.+Wangauthor=L.+Zhaoauthor=T.+Xuauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ARD-69+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+androgen+receptor+%28AR%29+for+the+treatment+of+prostate+cancer&doi=10.1021%2Facs.jmedchem.8b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span></div><div class="casAuthors">Han, Xin; Wang, Chao; Qin, Chong; Xiang, Weiguo; Fernandez-Salas, Ester; Yang, Chao-Yie; Wang, Mi; Zhao, Lijie; Xu, Tianfeng; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">941-964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the discovery of highly potent PROTAC degraders of androgen receptor (AR), as exemplified by IARD-69.  I induces degrdn. of AR protein in AR-pos. prostate cancer cell lines in a dose- and time-dependent manner.  I achieves DC50 values of 0.86, 0.76, and 10.4 nM in LNCaP, VCaP, and 22Rv1 AR+ prostate cancer cell lines, resp.  I is capable of reducing the AR protein level by >95% in these prostate cancer cell lines and effectively suppressing AR-regulated gene expression.  I potently inhibits cell growth in these AR-pos. prostate cancer cell lines and is >100 times more potent than AR antagonists.  A single dose of I effectively reduces the level of AR protein in xenograft tumor tissue in mice.  Further optimization of I may ultimately lead to a new therapy for AR+, castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTlOD69tfhtbVg90H21EOLACvtfcHk0lhl-aKkCO8JQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D&md5=2376573fb1a1e19c517d64c70199700a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01631%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ARD-69%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520androgen%2520receptor%2520%2528AR%2529%2520for%2520the%2520treatment%2520of%2520prostate%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D941%26epage%3D964%26doi%3D10.1021%2Facs.jmedchem.8b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.-Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+estrogen+receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0lhyLG73xfY0Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520estrogen%2520receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.-R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lhyLG73xfY0Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.-R.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukhali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnchembio.2538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=29251720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=163-170&author=C.+M.+Olsonauthor=B.+Jiangauthor=M.+A.+Erbauthor=Y.+Liangauthor=Z.+M.+Doctorauthor=Z.+Zhangauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=M.+Boukhaliauthor=J.+L.+Greenauthor=W.+Haasauthor=T.+Nomanbhoyauthor=E.+S.+Fischerauthor=R.+A.+Youngauthor=J.+E.+Bradnerauthor=G.+E.+Winterauthor=N.+S.+Gray&title=Pharmacological+perturbation+of+CDK9+using+selective+CDK9+inhibition+or+degradation&doi=10.1038%2Fnchembio.2538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span></div><div class="casAuthors">Olson, Calla M.; Jiang, Baishan; Erb, Michael A.; Liang, Yanke; Doctor, Zainab M.; Zhang, Zinan; Zhang, Tinghu; Kwiatkowski, Nicholas; Boukhali, Myriam; Green, Jennifer L.; Haas, Wilhelm; Nomanbhoy, Tyzoon; Fischer, Eric S.; Young, Richard A.; Bradner, James E.; Winter, Georg E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">163-170</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), an important regulator of transcriptional elongation, is a promising target for cancer therapy, particularly for cancers driven by transcriptional dysregulation.  The authors characterized NVP-2, a selective ATP-competitive CDK9 inhibitor, and THAL-SNS-032, a selective CDK9 degrader consisting of a CDK-binding SNS-032 ligand linked to a thalidomide deriv. that binds the E3 ubiquitin ligase Cereblon (CRBN).  To the authors' surprise, THAL-SNS-032 induced rapid degrdn. of CDK9 without affecting the levels of other SNS-032 targets.  Moreover, the transcriptional changes elicited by THAL-SNS-032 were more like those caused by NVP-2 than those induced by SNS-032.  Notably, compd. washout did not significantly reduce levels of THAL-SNS-032-induced apoptosis, suggesting that CDK9 degrdn. had prolonged cytotoxic effects compared with CDK9 inhibition.  Thus, the authors' findings suggest that thalidomide conjugation represents a promising strategy for converting multi-targeted inhibitors into selective degraders and reveal that kinase degrdn. can induce distinct pharmacol. effects compared with inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWnT9r0GIlmbVg90H21EOLACvtfcHk0lhyLG73xfY0Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O&md5=88aabb209b4cb7516a652d6c75a35667</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2538%26sid%3Dliteratum%253Aachs%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DBoukhali%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DJ.%2BL.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DNomanbhoy%26aufirst%3DT.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPharmacological%2520perturbation%2520of%2520CDK9%2520using%2520selective%2520CDK9%2520inhibition%2520or%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D163%26epage%3D170%26doi%3D10.1038%2Fnchembio.2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robb, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Contreras, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonawane, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murry, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">7577</span>– <span class="NLM_lpage">7580</span>, <span class="refDoi"> DOI: 10.1039/c7cc03879h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1039%2FC7CC03879H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=28636052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=7577-7580&author=C.+M.+Robbauthor=J.+I.+Contrerasauthor=S.+Kourauthor=M.+A.+Taylorauthor=M.+Abidauthor=Y.+A.+Sonawaneauthor=M.+Zahidauthor=D.+J.+Murryauthor=A.+Natarajanauthor=S.+Rana&title=Chemically+induced+degradation+of+CDK9+by+a+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1039%2Fc7cc03879h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Robb, Caroline M.; Contreras, Jacob I.; Kour, Smit; Taylor, Margaret A.; Abid, Mohammad; Sonawane, Yogesh A.; Zahid, Muhammad; Murry, Daryl J.; Natarajan, Amarnath; Rana, Sandeep</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">54</span>),
    <span class="NLM_cas:pages">7577-7580</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), a member of the cyclin-dependent protein kinase (CDK) family, is involved in transcriptional elongation of several target genes.  CDK9 is ubiquitously expressed and has been shown to contribute to a variety of malignancies such as pancreatic, prostate and breast cancers.  Here we report the development of a heterobifunctional small mol. proteolysis targeting chimera (PROTAC) capable of cereblon (CRBN)-mediated proteasomal degrdn. of CDK9.  In HCT116 cells, it selectively degrades CDK9 while sparing other CDK family members.  This is the first example of a PROTAC that selectively degrades CDK9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXgLtOTwvlnbVg90H21EOLACvtfcHk0ljB3vclFz_yuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D&md5=a9aa8ebf8109f11206740efd1b5e09a2</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1039%2FC7CC03879H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC03879H%26sid%3Dliteratum%253Aachs%26aulast%3DRobb%26aufirst%3DC.%2BM.%26aulast%3DContreras%26aufirst%3DJ.%2BI.%26aulast%3DKour%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DM.%2BA.%26aulast%3DAbid%26aufirst%3DM.%26aulast%3DSonawane%26aufirst%3DY.%2BA.%26aulast%3DZahid%26aufirst%3DM.%26aulast%3DMurry%26aufirst%3DD.%2BJ.%26aulast%3DNatarajan%26aufirst%3DA.%26aulast%3DRana%26aufirst%3DS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520CDK9%2520by%2520a%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D7577%26epage%3D7580%26doi%3D10.1039%2Fc7cc03879h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Discovery of Wogonin-based PROTACs against CDK9 and capable of achieving antitumor activity</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">381</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.08.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.bioorg.2018.08.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=30196207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Kkt7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=373-381&author=J.+Bianauthor=J.+Renauthor=Y.+Liauthor=J.+Wangauthor=X.+Xuauthor=Y.+Fengauthor=H.+Tangauthor=Y.+Wangauthor=Z.+Li&title=Discovery+of+Wogonin-based+PROTACs+against+CDK9+and+capable+of+achieving+antitumor+activity&doi=10.1016%2Fj.bioorg.2018.08.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Wogonin-based PROTACs against CDK9 and capable of achieving antitumor activity</span></div><div class="casAuthors">Bian, Jinlei; Ren, Jie; Li, Yongren; Wang, Jubo; Xu, Xi; Feng, Yifan; Tang, Hui; Wang, Yajing; Li, Zhiyu</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">373-381</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Wogonin is a natural product isolated from the Scutellaria baicalensis and has been proved to be a potent and selective inhibitor of CDK9.  Using this scaffold, we designed and synthesized a series of proteolysis targeting chimeras (PROTACs) targeting CDK9 by recruiting ubiquitin E3 ligase cereblon (CRBN).  For constructing diverse Wogonin-based PROTACs, a "click chem." approach was employed for the synthesis of CDK9-targeting PROTACs.  The results of western blotting assays showed that compds. contg. triazole group in the linker could selectively downregulate the intracellular CDK9 level.  Among these compds., 11c could selectively degrade CDK9 in a concn.-dependent manner.  In addn., the application of the proteasome inhibitor MG132 and CRBN siRNA silencing confirmed that 11c could promote the proteasome-dependent and CRBN-dependent degrdn.  Consistent with the degrdn. of the CDK9 protein, 11c selectively inhibits proliferation of CDK9-overexpressed cancer cells.  Thus, our Wogonin-based PROTAC would be an efficient probe that induces the degrdn. of CDK9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod4sLAWq5vMrVg90H21EOLACvtfcHk0ljB3vclFz_yuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Kkt7bE&md5=6a7158be1fce50296d6d0ec9160db0fc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.08.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.08.028%26sid%3Dliteratum%253Aachs%26aulast%3DBian%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520Wogonin-based%2520PROTACs%2520against%2520CDK9%2520and%2520capable%2520of%2520achieving%2520antitumor%2520activity%26jtitle%3DBioorg.%2520Chem.%26date%3D2018%26volume%3D81%26spage%3D373%26epage%3D381%26doi%3D10.1016%2Fj.bioorg.2018.08.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">Induction of apoptosis in MDA-MB-231 breast cancer cells by a PARP1-targeting PROTAC small molecule</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1039/c8cc07813k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1039%2FC8CC07813K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=30540295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVemsrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2019&pages=369-372&author=Q.+Zhaoauthor=T.+Lanauthor=S.+Suauthor=Y.+Rao&title=Induction+of+apoptosis+in+MDA-MB-231+breast+cancer+cells+by+a+PARP1-targeting+PROTAC+small+molecule&doi=10.1039%2Fc8cc07813k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis in MDA-MB-231 breast cancer cells by a PARP1-targeting PROTAC small molecule</span></div><div class="casAuthors">Zhao, Qiuye; Lan, Tianlong; Su, Shang; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">369-372</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Poly (ADP-ribose) polymerase-1 (PARP1) is a major member of the PARP superfamily that is involved in DNA damage signalling and other important cellular processes.  Here we report the development of a small mol. targeting PARP1 based on the PROTAC strategy.  In the MDA-MB-231 cell line, the representative compd. 3 can induce significant PARP1 cleavage and programmed cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJMa7i0ZDIH7Vg90H21EOLACvtfcHk0ljB3vclFz_yuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVemsrfN&md5=4bf5ac2b63635c348b415d5015c8c0c8</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1039%2FC8CC07813K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8CC07813K%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DLan%26aufirst%3DT.%26aulast%3DSu%26aufirst%3DS.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DInduction%2520of%2520apoptosis%2520in%2520MDA-MB-231%2520breast%2520cancer%2520cells%2520by%2520a%2520PARP1-targeting%2520PROTAC%2520small%2520molecule%26jtitle%3DChem.%2520Commun.%26date%3D2019%26volume%3D55%26spage%3D369%26epage%3D372%26doi%3D10.1039%2Fc8cc07813k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span> <span> </span><span class="NLM_article-title">Uncoupling of PARP1 trapping and inhibition using selective PARP1 degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1223</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0379-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fs41589-019-0379-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=31659317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVCksLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=1223-1231&author=S.+Wangauthor=L.+Hanauthor=J.+Hanauthor=P.+Liauthor=Q.+Dingauthor=Q.-J.+Zhangauthor=Z.-P.+Liuauthor=C.+Chenauthor=Y.+Yu&title=Uncoupling+of+PARP1+trapping+and+inhibition+using+selective+PARP1+degradation&doi=10.1038%2Fs41589-019-0379-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Uncoupling of PARP1 trapping and inhibition using selective PARP1 degradation</span></div><div class="casAuthors">Wang, Shuai; Han, Lei; Han, Jungsoo; Li, Peng; Ding, Qing; Zhang, Qing-Jun; Liu, Zhi-Ping; Chen, Chuo; Yu, Yonghao</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1223-1231</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">PARP1 inhibitors (PARPi) are known to kill tumor cells via two mechanisms (PARP1 catalytic inhibition and PARP1 trapping).  The relative contribution of these two pathways in mediating the cytotoxicity of PARPi, however, is not well understood.  Here we designed a series of small mol. PARP degraders.  Treatment with one such compd. iRucaparib-AP6 results in highly efficient and specific PARP1 degrdn. iRucaparib-AP6 blocks the enzymic activity of PARP1 in vitro, and PARP1-mediated poly-ADP-ribosylation signaling in intact cells.  This strategy mimics PARP1 genetic depletion, which enables the pharmacol. decoupling of PARP1 inhibition from PARP1 trapping.  Finally, by depleting PARP1, iRucaparib-AP6 protects muscle cells and primary cardiomyocytes from DNA-damage-induced energy crisis and cell death.  In summary, these compds. represent 'non-trapping' PARP1 degraders that block both the catalytic activity and scaffolding effects of PARP1, providing an ideal approach for the amelioration of the various pathol. conditions caused by PARP1 hyperactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfhC10Bxxzm7Vg90H21EOLACvtfcHk0ljekYhHbctHlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVCksLjF&md5=2f53f8fbbb41cf55b94ddb70a652df55</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0379-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0379-2%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DQ.-J.%26aulast%3DLiu%26aufirst%3DZ.-P.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DY.%26atitle%3DUncoupling%2520of%2520PARP1%2520trapping%2520and%2520inhibition%2520using%2520selective%2520PARP1%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D1223%26epage%3D1231%26doi%3D10.1038%2Fs41589-019-0379-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawicki-McKenna, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langelier, M.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeNizio, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccio, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karch, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCauley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffen, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascal, J. M.</span></span> <span> </span><span class="NLM_article-title">PARP-1 activation requires local unfolding of an autoinhibitory domain</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">768</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.molcel.2015.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=26626480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFagurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2015&pages=755-768&author=J.+M.+Dawicki-McKennaauthor=M.-F.+Langelierauthor=J.+E.+DeNizioauthor=A.+A.+Riccioauthor=C.+D.+Caoauthor=K.+R.+Karchauthor=M.+McCauleyauthor=J.+D.+Steffenauthor=B.+E.+Blackauthor=J.+M.+Pascal&title=PARP-1+activation+requires+local+unfolding+of+an+autoinhibitory+domain&doi=10.1016%2Fj.molcel.2015.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">PARP-1 Activation Requires Local Unfolding of an Autoinhibitory Domain</span></div><div class="casAuthors">Dawicki-McKenna, Jennine M.; Langelier, Marie-France; DeNizio, Jamie E.; Riccio, Amanda A.; Cao, Connie D.; Karch, Kelly R.; McCauley, Michael; Steffen, Jamin D.; Black, Ben E.; Pascal, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">755-768</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase-1 (PARP-1) creates the posttranslational modification PAR from substrate NAD+ to regulate multiple cellular processes.  DNA breaks sharply elevate PARP-1 catalytic activity to mount a cell survival repair response, whereas persistent PARP-1 hyperactivation during severe genotoxic stress is assocd. with cell death.  The mechanism for tight control of the robust catalytic potential of PARP-1 remains unclear.  By monitoring PARP-1 dynamics using hydrogen/deuterium exchange-mass spectrometry (HXMS), we unexpectedly find that a specific portion of the helical subdomain (HD) of the catalytic domain rapidly unfolds when PARP-1 encounters a DNA break.  Together with biochem. and crystallog. anal. of HD deletion mutants, we show that the HD is an autoinhibitory domain that blocks productive NAD+ binding.  Our mol. model explains how PARP-1 DNA damage detection leads to local unfolding of the HD that relieves autoinhibition, and has important implications for the design of PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJje_1KBikFbVg90H21EOLACvtfcHk0ljekYhHbctHlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFagurvK&md5=cf24c86d7d942be40929b8aee2d797c7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DDawicki-McKenna%26aufirst%3DJ.%2BM.%26aulast%3DLangelier%26aufirst%3DM.-F.%26aulast%3DDeNizio%26aufirst%3DJ.%2BE.%26aulast%3DRiccio%26aufirst%3DA.%2BA.%26aulast%3DCao%26aufirst%3DC.%2BD.%26aulast%3DKarch%26aufirst%3DK.%2BR.%26aulast%3DMcCauley%26aufirst%3DM.%26aulast%3DSteffen%26aufirst%3DJ.%2BD.%26aulast%3DBlack%26aufirst%3DB.%2BE.%26aulast%3DPascal%26aufirst%3DJ.%2BM.%26atitle%3DPARP-1%2520activation%2520requires%2520local%2520unfolding%2520of%2520an%2520autoinhibitory%2520domain%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D60%26spage%3D755%26epage%3D768%26doi%3D10.1016%2Fj.molcel.2015.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.-H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0ljekYhHbctHlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of MD-224 as a first-in-class, highly potent, and efficacious proteolysis targeting chimera murine double minute 2 degrader capable of achieving complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00909</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00909" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2018&pages=448-466&author=Y.+Liauthor=J.+Yangauthor=A.+Aguilarauthor=D.+McEachernauthor=S.+Przybranowskiauthor=L.+Liuauthor=C.-Y.+Yangauthor=M.+Wangauthor=X.+Hanauthor=S.+Wang&title=Discovery+of+MD-224+as+a+first-in-class%2C+highly+potent%2C+and+efficacious+proteolysis+targeting+chimera+murine+double+minute+2+degrader+capable+of+achieving+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00909"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00909%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520MD-224%2520as%2520a%2520first-in-class%252C%2520highly%2520potent%252C%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520murine%2520double%2520minute%25202%2520degrader%2520capable%2520of%2520achieving%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D62%26spage%3D448%26epage%3D466%26doi%3D10.1021%2Facs.jmedchem.8b00909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tinworth, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lithgow, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Small molecule-mediated protein knockdown as a new approach to drug discovery</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2206</span>– <span class="NLM_lpage">2216</span>, <span class="refDoi"> DOI: 10.1039/c6md00347h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1039%2FC6MD00347H" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2206-2216&author=C.+P.+Tinworthauthor=H.+Lithgowauthor=I.+Churcher&title=Small+molecule-mediated+protein+knockdown+as+a+new+approach+to+drug+discovery&doi=10.1039%2Fc6md00347h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1039%2FC6MD00347H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00347H%26sid%3Dliteratum%253Aachs%26aulast%3DTinworth%26aufirst%3DC.%2BP.%26aulast%3DLithgow%26aufirst%3DH.%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DSmall%2520molecule-mediated%2520protein%2520knockdown%2520as%2520a%2520new%2520approach%2520to%2520drug%2520discovery%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D2206%26epage%3D2216%26doi%3D10.1039%2Fc6md00347h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0liHa7z6kifv7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, J.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huan, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.-H.</span></span> <span> </span><span class="NLM_article-title">Novel PARP1/2 inhibitor mefuparib hydrochloride elicits potent in vitro and in vivo anticancer activity, characteristic of high tissue distribution</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4156</span>– <span class="NLM_lpage">4168</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.18632%2Foncotarget.13749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=27926532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A280%3ADC%252BC2sjksVSltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=4156-4168&author=J.-X.+Heauthor=M.+Wangauthor=X.-J.+Huanauthor=C.-H.+Chenauthor=S.-S.+Songauthor=Y.-Q.+Wangauthor=X.-M.+Liaoauthor=C.+Tanauthor=Q.+Heauthor=L.-J.+Tongauthor=Y.-T.+Wangauthor=X.-H.+Liauthor=Y.+Suauthor=Y.-Y.+Shenauthor=Y.-M.+Sunauthor=X.-Y.+Yangauthor=Y.+Chenauthor=Z.-W.+Gaoauthor=X.-Y.+Chenauthor=B.+Xiongauthor=X.-L.+Luauthor=J.+Dingauthor=C.-H.+Yangauthor=Z.-H.+Miao&title=Novel+PARP1%2F2+inhibitor+mefuparib+hydrochloride+elicits+potent+in+vitro+and+in+vivo+anticancer+activity%2C+characteristic+of+high+tissue+distribution&doi=10.18632%2Foncotarget.13749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Novel PARP1/2 inhibitor mefuparib hydrochloride elicits potent in vitro and in vivo anticancer activity, characteristic of high tissue distribution</span></div><div class="casAuthors">He Jin-Xue; Huan Xia-Juan; Chen Chuan-Huizi; Song Shan-Shan; Wang Ying-Qing; Liao Xue-Mei; Tong Lin-Jiang; Wang Yu-Ting; Li Xiao-Hua; Su Yi; Shen Yan-Yan; Sun Yi-Ming; Yang Xin-Ying; Chen Yi; Ding Jian; Miao Ze-Hong; Wang Meng; Tan Cun; He Qian; Xiong Bing; Yang Chun-Hao; Gao Zhi-Wei; Chen Xiao-Yan; Lu Xiu-Lian</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">4156-4168</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The approval of poly(ADP-ribose) polymerase (PARP) inhibitor AZD2281 in 2014 marked the successful establishment of the therapeutic strategy targeting homologous recombination repair defects of cancers in the clinic.  However, AZD2281 has poor water solubility, low tissue distribution and relatively weak in vivo anticancer activity, which appears to become limiting factors for its clinical use.  In this study, we found that mefuparib hydrochloride (MPH) was a potent PARP inhibitor, possessing prominent in vitro and in vivo anticancer activity.  Notably, MPH displayed high water solubility (> 35 mg/ml) and potent PARP1/2 inhibition in a substrate-competitive manner.  It reduced poly(ADP-ribose) (PAR) formation, enhanced γH2AX levels, induced G2/M arrest and subsequent apoptosis in homologous recombination repair (HR)-deficient cells.  Proof-of-concept studies confirmed the MPH-caused synthetic lethality.  MPH showed potent in vitro and in vivo proliferation and growth inhibition against HR-deficient cancer cells and synergistic sensitization of HR-proficient xenografts to the anticancer drug temozolomide.  A good relationship between the anticancer activity and the PARP inhibition of MPH suggested that PAR formation and γH2AX accumulation could serve as its pharmacodynamic biomarkers.  Its high bioavailability (40%~100%) and high tissue distribution in both monkeys and rats were its most important pharmacokinetic features.  Its average concentrations were 33-fold higher in the tissues than in the plasma in rats.  Our work supports the further clinical development of MPH as a novel PARP1/2 inhibitor for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFLMaYGUFFID69zv1FsJFMfW6udTcc2eboo8zqoo8Hvbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjksVSltQ%253D%253D&md5=bf548fd046eb6c1330597a0a75a432ea</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13749%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DJ.-X.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHuan%26aufirst%3DX.-J.%26aulast%3DChen%26aufirst%3DC.-H.%26aulast%3DSong%26aufirst%3DS.-S.%26aulast%3DWang%26aufirst%3DY.-Q.%26aulast%3DLiao%26aufirst%3DX.-M.%26aulast%3DTan%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DQ.%26aulast%3DTong%26aufirst%3DL.-J.%26aulast%3DWang%26aufirst%3DY.-T.%26aulast%3DLi%26aufirst%3DX.-H.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DY.-Y.%26aulast%3DSun%26aufirst%3DY.-M.%26aulast%3DYang%26aufirst%3DX.-Y.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DZ.-W.%26aulast%3DChen%26aufirst%3DX.-Y.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DX.-L.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.-H.%26aulast%3DMiao%26aufirst%3DZ.-H.%26atitle%3DNovel%2520PARP1%252F2%2520inhibitor%2520mefuparib%2520hydrochloride%2520elicits%2520potent%2520in%2520vitro%2520and%2520in%2520vivo%2520anticancer%2520activity%252C%2520characteristic%2520of%2520high%2520tissue%2520distribution%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D4156%26epage%3D4168%26doi%3D10.18632%2Foncotarget.13749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Redon, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonner, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinders, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parchment, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">Histone γH2AX and poly(ADP-ribose) as clinical pharmacodynamic biomarkers</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4532</span>– <span class="NLM_lpage">4542</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-10-0523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1158%2F1078-0432.CCR-10-0523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=20823146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGqs7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=4532-4542&author=C.+E.+Redonauthor=A.+J.+Nakamuraauthor=Y.-W.+Zhangauthor=J.+Jiauthor=W.+M.+Bonnerauthor=R.+J.+Kindersauthor=R.+E.+Parchmentauthor=J.+H.+Doroshowauthor=Y.+Pommier&title=Histone+%CE%B3H2AX+and+poly%28ADP-ribose%29+as+clinical+pharmacodynamic+biomarkers&doi=10.1158%2F1078-0432.ccr-10-0523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Histone γH2AX and Poly(ADP-Ribose) as Clinical Pharmacodynamic Biomarkers</span></div><div class="casAuthors">Redon, Christophe E.; Nakamura, Asako J.; Zhang, Yong-Wei; Ji, Jiuping; Bonner, William M.; Kinders, Robert J.; Parchment, Ralph E.; Doroshow, James H.; Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4532-4542</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Tumor cells are often deficient in DNA damage response (DDR) pathways, and anticancer therapies are commonly based on genotoxic treatments using radiation and/or drugs that damage DNA directly or interfere with DNA metab., leading to the formation of DNA double-strand breaks (DSB), and ultimately to cell death.  Because DSBs induce the phosphorylation of histone H2AX (γH2AX) in the chromatin flanking the break site, an antibody directed against γH2AX can be employed to measure DNA damage levels before and after patient treatment.  Poly(ADP-ribose) polymerases (PARP1 and PARP2) are also activated by DNA damage, and PARP inhibitors show promising activity in cancers with defective homologous recombination (HR) pathways for DSB repair.  Ongoing clin. trials are testing combinations of PARP inhibitors with DNA damaging agents.  Poly(ADP-ribosylation), abbreviated as PAR, can be measured in clin. samples and used to det. the efficiency of PARP inhibitors.  This review summarizes the roles of γH2AX and PAR in the DDR, and their use as biomarkers to monitor drug response and guide clin. trials, esp. phase 0 clin. trials.  We also discuss the choices of relevant samples for γH2AX and PAR analyses.  Clin Cancer Res; 16(18); 4532-42.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocLdGwTAJVALVg90H21EOLACvtfcHk0liz-AIh8VSbIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGqs7bI&md5=4a2bc5d90d93f3de000ccb3f8ffd9c74</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-10-0523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-10-0523%26sid%3Dliteratum%253Aachs%26aulast%3DRedon%26aufirst%3DC.%2BE.%26aulast%3DNakamura%26aufirst%3DA.%2BJ.%26aulast%3DZhang%26aufirst%3DY.-W.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DBonner%26aufirst%3DW.%2BM.%26aulast%3DKinders%26aufirst%3DR.%2BJ.%26aulast%3DParchment%26aufirst%3DR.%2BE.%26aulast%3DDoroshow%26aufirst%3DJ.%2BH.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DHistone%2520%25CE%25B3H2AX%2520and%2520poly%2528ADP-ribose%2529%2520as%2520clinical%2520pharmacodynamic%2520biomarkers%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2010%26volume%3D16%26spage%3D4532%26epage%3D4542%26doi%3D10.1158%2F1078-0432.ccr-10-0523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papeo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posteri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borghi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busel, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caprera, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciomei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Anello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fasolini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isacchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khvat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krasavin, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orsini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perego, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pesenti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pezzetta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rainoldi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccardi-Sirtori, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolaro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sola, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuccotto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donati, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montagnoli, A.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2-[1-(4,4-difluorocyclohexyl)piperidin-4-yl]-6-fluoro-3-oxo-2,3-dihydro-1<i>H</i>-isoind ole-4-carboxamide (NMS-P118): a potent, orally available, and highly selective PARP-1 inhibitor for cancer therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6875</span>– <span class="NLM_lpage">6898</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00680</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00680" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1CitbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6875-6898&author=G.+Papeoauthor=H.+Posteriauthor=D.+Borghiauthor=A.+A.+Buselauthor=F.+Capreraauthor=E.+Casaleauthor=M.+Ciomeiauthor=A.+Cirlaauthor=E.+Cortiauthor=M.+D%E2%80%99Anelloauthor=M.+Fasoliniauthor=B.+Forteauthor=A.+Galvaniauthor=A.+Isacchiauthor=A.+Khvatauthor=M.+Y.+Krasavinauthor=R.+Lupiauthor=P.+Orsiniauthor=R.+Peregoauthor=E.+Pesentiauthor=D.+Pezzettaauthor=S.+Rainoldiauthor=F.+Riccardi-Sirtoriauthor=A.+Scolaroauthor=F.+Solaauthor=F.+Zuccottoauthor=E.+R.+Felderauthor=D.+Donatiauthor=A.+Montagnoli&title=Discovery+of+2-%5B1-%284%2C4-difluorocyclohexyl%29piperidin-4-yl%5D-6-fluoro-3-oxo-2%2C3-dihydro-1H-isoind+ole-4-carboxamide+%28NMS-P118%29%3A+a+potent%2C+orally+available%2C+and+highly+selective+PARP-1+inhibitor+for+cancer+therapy&doi=10.1021%2Facs.jmedchem.5b00680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-[1-(4,4-Difluorocyclohexyl)piperidin-4-yl]-6-fluoro-3-oxo-2,3-dihydro-1H-isoindole-4-carboxamide (NMS-P118): A Potent, Orally Available, and Highly Selective PARP-1 Inhibitor for Cancer Therapy</span></div><div class="casAuthors">Papeo, Gianluca; Posteri, Helena; Borghi, Daniela; Busel, Alina A.; Caprera, Francesco; Casale, Elena; Ciomei, Marina; Cirla, Alessandra; Corti, Emiliana; D'Anello, Matteo; Fasolini, Marina; Forte, Barbara; Galvani, Arturo; Isacchi, Antonella; Khvat, Alexander; Krasavin, Mikhail Y.; Lupi, Rosita; Orsini, Paolo; Perego, Rita; Pesenti, Enrico; Pezzetta, Daniele; Rainoldi, Sonia; Riccardi-Sirtori, Federico; Scolaro, Alessandra; Sola, Francesco; Zuccotto, Fabio; Felder, Eduard R.; Donati, Daniele; Montagnoli, Alessia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6875-6898</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The nuclear protein poly(ADP-ribose) polymerase-1 (PARP-1) has a well-established role in the signaling and repair of DNA and is a prominent target in oncol., as testified by the no. of candidates in clin. testing that unselectively target both PARP-1 and its closest isoform PARP-2.  The goal of the program was to find a PARP-1 selective inhibitor that would potentially mitigate toxicities arising from cross-inhibition of PARP-2.  Thus, an HTS campaign on the proprietary Nerviano Medical Sciences (NMS) chem. collection, followed by SAR optimization, allowed us to discover I.  NMS-P118 proved to be a potent, orally available, and highly selective PARP-1 inhibitor endowed with excellent ADME and pharmacokinetic profiles and high efficacy in vivo both as a single agent and in combination with Temozolomide in MDA-MB-436 and Capan-1 xenograft models, resp.  Cocrystal structures of I with both PARP-1 and PARP-2 catalytic domain proteins allowed rationalization of the obsd. selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM_i5haE561bVg90H21EOLACvtfcHk0liz-AIh8VSbIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1CitbzO&md5=600a505d975aa18639885b179170e684</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00680%26sid%3Dliteratum%253Aachs%26aulast%3DPapeo%26aufirst%3DG.%26aulast%3DPosteri%26aufirst%3DH.%26aulast%3DBorghi%26aufirst%3DD.%26aulast%3DBusel%26aufirst%3DA.%2BA.%26aulast%3DCaprera%26aufirst%3DF.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DCiomei%26aufirst%3DM.%26aulast%3DCirla%26aufirst%3DA.%26aulast%3DCorti%26aufirst%3DE.%26aulast%3DD%25E2%2580%2599Anello%26aufirst%3DM.%26aulast%3DFasolini%26aufirst%3DM.%26aulast%3DForte%26aufirst%3DB.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DIsacchi%26aufirst%3DA.%26aulast%3DKhvat%26aufirst%3DA.%26aulast%3DKrasavin%26aufirst%3DM.%2BY.%26aulast%3DLupi%26aufirst%3DR.%26aulast%3DOrsini%26aufirst%3DP.%26aulast%3DPerego%26aufirst%3DR.%26aulast%3DPesenti%26aufirst%3DE.%26aulast%3DPezzetta%26aufirst%3DD.%26aulast%3DRainoldi%26aufirst%3DS.%26aulast%3DRiccardi-Sirtori%26aufirst%3DF.%26aulast%3DScolaro%26aufirst%3DA.%26aulast%3DSola%26aufirst%3DF.%26aulast%3DZuccotto%26aufirst%3DF.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DMontagnoli%26aufirst%3DA.%26atitle%3DDiscovery%2520of%25202-%255B1-%25284%252C4-difluorocyclohexyl%2529piperidin-4-yl%255D-6-fluoro-3-oxo-2%252C3-dihydro-1H-isoind%2520ole-4-carboxamide%2520%2528NMS-P118%2529%253A%2520a%2520potent%252C%2520orally%2520available%252C%2520and%2520highly%2520selective%2520PARP-1%2520inhibitor%2520for%2520cancer%2520therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6875%26epage%3D6898%26doi%3D10.1021%2Facs.jmedchem.5b00680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nan  Wang</span>, <span class="hlFld-ContribAuthor ">Yuanting  Gu</span>, <span class="hlFld-ContribAuthor ">Jiangrui  Chi</span>, <span class="hlFld-ContribAuthor ">Xinwei  Liu</span>, <span class="hlFld-ContribAuthor ">Youyi  Xiong</span>, <span class="hlFld-ContribAuthor ">Chaochao  Zhong</span>, <span class="hlFld-ContribAuthor ">Fang  Wang</span>, <span class="hlFld-ContribAuthor ">Xinxing  Wang</span>, <span class="hlFld-ContribAuthor ">Lin  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Screening of DNA Damage Repair Genes Involved in the Prognosis of Triple-Negative Breast Cancer Patients Based on Bioinformatics. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Genetics</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fgene.2021.721873" title="DOI URL">https://doi.org/10.3389/fgene.2021.721873</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fgene.2021.721873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffgene.2021.721873%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Genetics%26atitle%3DScreening%252Bof%252BDNA%252BDamage%252BRepair%252BGenes%252BInvolved%252Bin%252Bthe%252BPrognosis%252Bof%252BTriple-Negative%252BBreast%252BCancer%252BPatients%252BBased%252Bon%252BBioinformatics%26aulast%3DWang%26aufirst%3DNan%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Uday Kiran  Velagapudi</span>, <span class="hlFld-ContribAuthor ">Bhargav A.  Patel</span>, <span class="hlFld-ContribAuthor ">Xuwei  Shao</span>, <span class="hlFld-ContribAuthor ">Sanjai Kumar  Pathak</span>, <span class="hlFld-ContribAuthor ">Dana V.  Ferraris</span>, <span class="hlFld-ContribAuthor ">Tanaji T.  Talele</span>. </span><span class="cited-content_cbyCitation_article-title">Recent development in the discovery of PARP inhibitors as anticancer agents: a patent update (2016-2020). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2021,</strong> <em>31 </em>
                                    (7)
                                     , 609-623. <a href="https://doi.org/10.1080/13543776.2021.1886275" title="DOI URL">https://doi.org/10.1080/13543776.2021.1886275</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2021.1886275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2021.1886275%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DRecent%252Bdevelopment%252Bin%252Bthe%252Bdiscovery%252Bof%252BPARP%252Binhibitors%252Bas%252Banticancer%252Bagents%25253A%252Ba%252Bpatent%252Bupdate%252B%2525282016-2020%252529%26aulast%3DVelagapudi%26aufirst%3DUday%2BKiran%26date%3D2021%26date%3D2021%26volume%3D31%26issue%3D7%26spage%3D609%26epage%3D623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tim J.  Wigle</span>, <span class="hlFld-ContribAuthor ">Yue  Ren</span>, <span class="hlFld-ContribAuthor ">Jennifer R.  Molina</span>, <span class="hlFld-ContribAuthor ">Danielle J.  Blackwell</span>, <span class="hlFld-ContribAuthor ">Laurie B.  Schenkel</span>, <span class="hlFld-ContribAuthor ">Kerren K.  Swinger</span>, <span class="hlFld-ContribAuthor ">Kristy  Kuplast‐Barr</span>, <span class="hlFld-ContribAuthor ">Christina R.  Majer</span>, <span class="hlFld-ContribAuthor ">W. David  Church</span>, <span class="hlFld-ContribAuthor ">Alvin Z.  Lu</span>, <span class="hlFld-ContribAuthor ">Jason  Mo</span>, <span class="hlFld-ContribAuthor ">Ryan  Abo</span>, <span class="hlFld-ContribAuthor ">Anne  Cheung</span>, <span class="hlFld-ContribAuthor ">Bryan W.  Dorsey</span>, <span class="hlFld-ContribAuthor ">Mario  Niepel</span>, <span class="hlFld-ContribAuthor ">Nicholas R.  Perl</span>, <span class="hlFld-ContribAuthor ">Melissa M.  Vasbinder</span>, <span class="hlFld-ContribAuthor ">Heike  Keilhack</span>, <span class="hlFld-ContribAuthor ">Kevin W.  Kuntz</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted Degradation of PARP14 Using a Heterobifunctional Small Molecule. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2021,</strong> <em>22 </em>
                                    (12)
                                     , 2107-2110. <a href="https://doi.org/10.1002/cbic.202100047" title="DOI URL">https://doi.org/10.1002/cbic.202100047</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202100047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202100047%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DTargeted%252BDegradation%252Bof%252BPARP14%252BUsing%252Ba%252BHeterobifunctional%252BSmall%252BMolecule%26aulast%3DWigle%26aufirst%3DTim%2BJ.%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D12%26spage%3D2107%26epage%3D2110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiho  Kim</span>, <span class="hlFld-ContribAuthor ">Chuo  Chen</span>, <span class="hlFld-ContribAuthor ">Yonghao  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Avoid the trap: Targeting PARP1 beyond human malignancy. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (4)
                                     , 456-462. <a href="https://doi.org/10.1016/j.chembiol.2021.02.004" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.02.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.02.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DAvoid%252Bthe%252Btrap%25253A%252BTargeting%252BPARP1%252Bbeyond%252Bhuman%252Bmalignancy%26aulast%3DKim%26aufirst%3DChiho%26date%3D2021%26volume%3D28%26issue%3D4%26spage%3D456%26epage%3D462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0028.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structure of three previously reported PARP1 degraders <b>1</b>, <b>2</b>, and <b>3</b> and one previously reported PARP2 degrader <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of PROTACs for PARP1. (A) Chemical structures of PARP1/2 inhibitor <b>5</b> (olaparib). (B) Crystal structure of <b>5</b> bound to the PARP1 catalytic domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DS3">5DS3</a>). The possible linker-tethering positions are shown as red arrows.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Western blotting analysis of the PARP1 protein in MDA-MB-436 (A) and Capan-1 (B) cells treated with the PARP1 inhibitor OLA (olaparib), PARP1 degraders <b>16–27</b>, and a control compound <b>28</b>. Cells were treated with each compound at 1, 10, 100, and 1000 nM for 24 h, and β-tubulin was used as the loading control. The fold under the strip represents the PARP1/β-tubulin ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design of <b>28</b> as a control compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Western blotting analysis of PARP1 protein in MDA-MB-436 (A) and SW620 (B) cells treated with compounds <b>3</b> and <b>18</b>. Cells were treated with each compound at 1, 10, 100, 1000 nM, and 10 μM for 12 h, and β-tubulin was used as the loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Further characterization of SK-575. (A) PARP1 degradation at the indicated dose of SK-575 with a 24 h treatment period in MDA-MB-436, Capan-1, and SW620 cells (left). PARP1 protein was examined by western blotting, and the PARP1 protein level was quantified and normalized to the corresponding density of β-tubulin protein (right). DC<sub>50</sub> values are calculated by two biologically independent experiments. (B) Time course of PARP1 degradation by SK-575 in MDA-MB-436 (30 nM), Capan-1 (100 nM), and SW620 (30 nM) cells. (C) Time course of PARP1 degradation by SK-575 in MDA-MB-436 (100 nM) cells. (D) Cellular PARP1 recovery levels after washing SK-575. Cells were pre-treated with SK-575 for 2 h and washing the compound for the indicated times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Western blotting analysis of PARP1 protein in MDA-MB-436 (A) and SW620 (B) cells after a 2 h pretreatment with PARP1 inhibitor OLA (olaparib), CRBN ligand POM (pomalidomide), proteasome inhibitor (MG132 or carfilzomib), or NEDD8-activating E1 enzyme inhibitor (MLN4924). Cells were treated with the individual compound at indicated concentrations, followed by a 24 h treatment with SK-575 at 30 nM, and β-tubulin was used as the loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Highly potent and broad applicability of SK-575. (A) DC<sub>50</sub> of SK-575 for PARP1 in different cancer cells. Data are the average of two biologically independent experiments. (B) PARP1 activity assay. (C) IC<sub>50</sub> values of SK-575 and olaparib. Cells were treated with indicated doses for 7 days. Cell viability was determined by the CCK-8 assay, and the data were calculated using GraphPad Prism 7 software. (D) Potentiation of MMS cell killing of cultured SW620 cells in combination with SK-575. Increasing concentrations of MMS were coincubated with or without the PARP1 degrader at single concentrations ranging from 3 to 300 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Western blotting analysis of γH2AX proteins in MDA-MB-436 (A) and Capan-1 (B) cells, with β-tubulin used as the loading control. Cells were treated with PARP1 inhibitor (olaparib) or PARP1 degrader (SK-575) at indicated concentrations, followed by a 24 h treatment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. In vivo evaluation of the PARP1 degrader SK-575. (A) PK study of SK-575 in mice (<i>n</i> = 3 per group). (B) PD study of SK-575 in the SW620 tumor tissue in mice. The treatment group mice-bearing xenograft SW620 tumors were treated with a single dose of SK-575 (25 mg/kg, ip). Tumor tissues were harvested at the indicated timepoints for the western blotting assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. (A) Antitumor efficacy of SK-575 in BRCA2-mutated Capan-1 xenografts (<i>n</i> = 6). (B) Antitumor efficacy of SK-575 in combination with TMZ in SW620 xenografts (<i>n</i> = 6). Nude mice were dosed once daily for 5 consecutive days (days 0–4 are indicated by a short line). (C) Antitumor efficacy of SK-575 in combination with cisplatin in BRCA2-mutated Capan-1 xenografts (<i>n</i> = 6). Nude mice were dosed with SK-575 once daily for 5 consecutive days (days 0–4 are indicated by a short line) and cisplatin was dosed intraperitoneally at 6 mg/kg on the 2 day timepoint. The efficacy of the treatment is expressed as mean ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>6–11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) TEA, toluene, reflux; (b) <i>tert</i>-butyl bromoacetate, KI, KHCO<sub>3</sub>, DMF, 60 °C; (c) TFA/DCM, rt; (d) <i>N</i>-Boc-piperazine, HATU, TEA, DMF, rt; (e) HCl, EtOH, rt; (f) HATU, DIPEA, DMF, rt; (g) HCl, DCM/MeOH, rt.</p></p></figure><figure data-id="sch2" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>12–13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) NaOAc, AcOH, reflux; (b) <i>tert</i>-butyl glycinate, DIPEA, NMP, 90 °C; (c) TFA, DCM, rt; (d) HCl, DCM/MeOH, rt; (e) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch3" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>14–25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) DIPEA, NMP, 90 °C; (b) HBTU, DIPEA, DMF, rt; (c) HCl, DCM, MeOH, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch4" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0016.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compound <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) NaOAc, AcOH, reflux; (b) <i>N</i>-Boc-ethylenediamine, DIPEA, NMP, 90 °C; (c) HCl, DCM, MeOH, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch5" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0017.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) NaBH<sub>3</sub>CN, MeOH, 50 °C; (b) HCl, DCM, MeOH, rt; (c) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch6" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0018.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>28</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C; (b) DIPEA, NMP, 90 °C; (c) HCl, DCM, MeOH, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch7" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/medium/jm0c00821_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0019.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compounds <b>29–32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00821/20201001/images/large/jm0c00821_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00821&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) HCl, DCM, MeOH, rt; (b) HATU, DIPEA, DMF, rt.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74717" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74717" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 58 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kleine, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesniewicz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassa, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hottiger, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litchfield, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shilton, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lüscher, B.</span></span> <span> </span><span class="NLM_article-title">Substrate-assisted catalysis by PARP10 limits its activity to mono-ADP-ribosylation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.molcel.2008.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=18851833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1GksLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=57-69&author=H.+Kleineauthor=E.+Porebaauthor=K.+Lesniewiczauthor=P.+O.+Hassaauthor=M.+O.+Hottigerauthor=D.+W.+Litchfieldauthor=B.+H.+Shiltonauthor=B.+L%C3%BCscher&title=Substrate-assisted+catalysis+by+PARP10+limits+its+activity+to+mono-ADP-ribosylation&doi=10.1016%2Fj.molcel.2008.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate-assisted catalysis by PARP10 limits its activity to mono-ADP-ribosylation</span></div><div class="casAuthors">Kleine, Henning; Poreba, Elzbieta; Lesniewicz, Krzysztof; Hassa, Paul O.; Hottiger, Michael O.; Litchfield, David W.; Shilton, Brian H.; Luescher, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">ADP-ribosylation controls many processes, including transcription, DNA repair, and bacterial toxicity.  ADP-ribosyltransferases and poly-ADP-ribose polymerases (PARPs) catalyze mono- and poly-ADP-ribosylation, resp., and depend on a highly conserved glutamate residue in the active center for catalysis.  However, there is an apparent absence of this glutamate for the recently described PARP6-PARP16, raising questions about how these enzymes function.  We find that PARP10, in contrast to PARP1, lacks the catalytic glutamate and has transferase rather than polymerase activity.  Despite this fundamental difference, PARP10 also modifies acidic residues.  Consequently, we propose an alternative catalytic mechanism for PARP10 compared to PARP1 in which the acidic target residue of the substrate functionally substitutes for the catalytic glutamate by using substrate-assisted catalysis to transfer ADP-ribose.  This mechanism explains why the novel PARPs are unable to function as polymerases.  This discovery will help to illuminate the different biol. functions of mono- vs. poly-ADP-ribosylation in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopRTvaqa_wIrVg90H21EOLACvtfcHk0lhH0-LU4tDdTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1GksLzO&md5=a7d3d0b015773d18f92a434f6a8e0232</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DKleine%26aufirst%3DH.%26aulast%3DPoreba%26aufirst%3DE.%26aulast%3DLesniewicz%26aufirst%3DK.%26aulast%3DHassa%26aufirst%3DP.%2BO.%26aulast%3DHottiger%26aufirst%3DM.%2BO.%26aulast%3DLitchfield%26aufirst%3DD.%2BW.%26aulast%3DShilton%26aufirst%3DB.%2BH.%26aulast%3DL%25C3%25BCscher%26aufirst%3DB.%26atitle%3DSubstrate-assisted%2520catalysis%2520by%2520PARP10%2520limits%2520its%2520activity%2520to%2520mono-ADP-ribosylation%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D32%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.molcel.2008.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouleau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendzel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, G. G.</span></span> <span> </span><span class="NLM_article-title">PARP inhibition: PARP1 and beyond</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1038/nrc2812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnrc2812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=20200537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1yms7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=293-301&author=M.+Rouleauauthor=A.+Patelauthor=M.+J.+Hendzelauthor=S.+H.+Kaufmannauthor=G.+G.+Poirier&title=PARP+inhibition%3A+PARP1+and+beyond&doi=10.1038%2Fnrc2812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PARP inhibition: PARP1 and beyond</span></div><div class="casAuthors">Rouleau, Michele; Patel, Anand; Hendzel, Michael J.; Kaufmann, Scott H.; Poirier, Guy G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-301</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent findings have thrust poly(ADP-ribose) polymerases (PARPs) into the limelight as potential chemotherapeutic targets.  To provide a framework for understanding these recent observations, the authors have reviewed what is known about the structures and functions of the family of PARP enzymes, and then outlined a series of questions that should be addressed to guide the rational development of PARP inhibitors as anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOiY_n1IClmbVg90H21EOLACvtfcHk0lh07gOQ6hFpbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXis1yms7w%253D&md5=fa3e3964e84229317dadbd505f11ad86</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc2812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2812%26sid%3Dliteratum%253Aachs%26aulast%3DRouleau%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DHendzel%26aufirst%3DM.%2BJ.%26aulast%3DKaufmann%26aufirst%3DS.%2BH.%26aulast%3DPoirier%26aufirst%3DG.%2BG.%26atitle%3DPARP%2520inhibition%253A%2520PARP1%2520and%2520beyond%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D293%26epage%3D301%26doi%3D10.1038%2Fnrc2812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amé, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spenlehauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Murcia, G.</span></span> <span> </span><span class="NLM_article-title">The PARP superfamily</span>. <i>Bioessays</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">882</span>– <span class="NLM_lpage">893</span>, <span class="refDoi"> DOI: 10.1002/bies.20085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1002%2Fbies.20085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15273990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnt1CqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2004&pages=882-893&author=J.-C.+Am%C3%A9author=C.+Spenlehauerauthor=G.+de+Murcia&title=The+PARP+superfamily&doi=10.1002%2Fbies.20085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The PARP superfamily</span></div><div class="casAuthors">Ame, Jean-Christophe; Spenlehauer, Catherine; de Murcia, Gilbert</div><div class="citationInfo"><span class="NLM_cas:title">BioEssays</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">882-893</span>CODEN:
                <span class="NLM_cas:coden">BIOEEJ</span>;
        ISSN:<span class="NLM_cas:issn">0265-9247</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribosyl)ation is an immediate DNA-damage-dependent post-translational modification of histones and other nuclear proteins that contributes to the survival of injured proliferating cells.  Poly(ADP-ribose) polymerases (PARPs) now constitute a large family of 18 proteins, encoded by different genes and displaying a conserved catalytic domain in which PARP-1 (113 kDa), the founding member, and PARP-2 (62 kDa) are so far the sole enzymes whose catalytic activity has been shown to be immediately stimulated by DNA strand breaks.  A large repertoire of sequences encoding novel PARPs now extends considerably the field of poly(ADP-ribosyl)ation reactions to various aspects of the cell biol. including cell proliferation and cell death.  Some of these new members interact with each other, share common partners and common subcellular localizations suggesting possible fine tuning in the regulation of this post-translational modification of proteins.  This review summarizes the present knowledge of this emerging superfamily, which might ultimately improve pharmacol. strategies to enhance both antitumor efficacy and the treatment of a no. of inflammatory and neurodegenerative disorders.  A provisional nomenclature is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojkP1P6SYh67Vg90H21EOLACvtfcHk0lh07gOQ6hFpbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnt1CqtLk%253D&md5=892151993f54e3c1a03019001a365fbf</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fbies.20085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbies.20085%26sid%3Dliteratum%253Aachs%26aulast%3DAm%25C3%25A9%26aufirst%3DJ.-C.%26aulast%3DSpenlehauer%26aufirst%3DC.%26aulast%3Dde%2BMurcia%26aufirst%3DG.%26atitle%3DThe%2520PARP%2520superfamily%26jtitle%3DBioessays%26date%3D2004%26volume%3D26%26spage%3D882%26epage%3D893%26doi%3D10.1002%2Fbies.20085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frostholm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, A.</span></span> <span> </span><span class="NLM_article-title">Embryonic and postnatal expression of four gamma-aminobutyric acid transporter mRNAs in the mouse brain and leptomeninges</span>. <i>J. Comp. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>376</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1002/(sici)1096-9861(19961216)376:3<431::aid-cne6>3.0.co;2-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1002%2F%28SICI%291096-9861%2819961216%29376%3A3%3C431%3A%3AAID-CNE6%3E3.0.CO%3B2-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=8956109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK2sXivFanug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=376&publication_year=1996&pages=431-446&author=J.+E.+Evansauthor=A.+Frostholmauthor=A.+Rotter&title=Embryonic+and+postnatal+expression+of+four+gamma-aminobutyric+acid+transporter+mRNAs+in+the+mouse+brain+and+leptomeninges&doi=10.1002%2F%28sici%291096-9861%2819961216%29376%3A3%3C431%3A%3Aaid-cne6%3E3.0.co%3B2-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Embryonic and postnatal expression of four gamma-aminobutyric acid transporter mRNAs in the mouse brain and leptomeninges</span></div><div class="casAuthors">Evans, J. E.; Frostholm, A.; Rotter, A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Comparative Neurology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">376</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">431-446</span>CODEN:
                <span class="NLM_cas:coden">JCNEAM</span>;
        ISSN:<span class="NLM_cas:issn">0021-9967</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss</span>)
        </div><div class="casAbstract">The distribution of gamma-aminobutyric acid (GABA) transporter mRNAs (mGATs) was studied in mouse brain during embryonic and postnatal development using in situ hybridization with radiolabeled oligonucleotide probes.  Mouse GATs 1 and 4 were present in the ventricular and subventricular zones of the lateral ventricle from gestational day 13.  During postnatal development, mGAT1 mRNA was distributed diffusely throughout the brain and spinal cord, with the highest expression present in the olfactory bulbs, hippocampus, and cerebellar cortex.  The mGAT4 message was densely distributed throughout the central nervous system during postnatal week 1; however, the hybridization signal in the cerebral cortex and hippocampus decreased during postnatal weeks 2 and 3, and in adults, mGAT4 labeling was restricted largely to the olfactory bulbs, midbrain, deep cerebellar nuclei, medulla, and spinal cord.  Mouse GAT2 mRNA was expressed only in proliferating and migrating cerebellar granule cells, whereas mGAT3 mRNA was absent from the brain and spinal cord throughout development.  Each of the four mGATs was present to some degree in the leptomeninges.  The expression of mGATs 2 and 3 was almost entirely restricted to the pia-arachnoid, whereas mGATs 1 and 4 were present only in specific regions of the membrane.  Although mGATs 1 and 4 may subserve the classical purpose of terminating inhibitory GABAergic transmission through neuronal and glial uptake mechanisms, GABA transporters in the pia-arachnoid may help to regulate the amt. of GABA available to proliferating and migrating neurons at the sub-pial surface during perinatal development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA2cnk2eidm7Vg90H21EOLACvtfcHk0likmR8Nu_PBkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFanug%253D%253D&md5=98dfbfc1ba6bccb7bf99dee1a9e655a4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-9861%2819961216%29376%3A3%3C431%3A%3AAID-CNE6%3E3.0.CO%3B2-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-9861%252819961216%2529376%253A3%253C431%253A%253AAID-CNE6%253E3.0.CO%253B2-3%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DJ.%2BE.%26aulast%3DFrostholm%26aufirst%3DA.%26aulast%3DRotter%26aufirst%3DA.%26atitle%3DEmbryonic%2520and%2520postnatal%2520expression%2520of%2520four%2520gamma-aminobutyric%2520acid%2520transporter%2520mRNAs%2520in%2520the%2520mouse%2520brain%2520and%2520leptomeninges%26jtitle%3DJ.%2520Comp.%2520Neurol.%26date%3D1996%26volume%3D376%26spage%3D431%26epage%3D446%26doi%3D10.1002%2F%28sici%291096-9861%2819961216%29376%3A3%3C431%3A%3Aaid-cne6%3E3.0.co%3B2-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jagtap, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabó, C.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose) polymerase and the therapeutic effects of its inhibitors</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1038/nrd1718</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnrd1718" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15864271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjs1OktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=421-440&author=P.+Jagtapauthor=C.+Szab%C3%B3&title=Poly%28ADP-ribose%29+polymerase+and+the+therapeutic+effects+of+its+inhibitors&doi=10.1038%2Fnrd1718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribose) polymerase and the therapeutic effects of its inhibitors</span></div><div class="casAuthors">Jagtap, Prakash; Szabo, Csaba</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">421-440</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerases (PARPs) are involved in the regulation of many cellular functions.  Three consequences of the activation of PARP1, which is the main isoform of the PARP family, are particularly important for drug development: first, its role in DNA repair; second, its capacity to deplete cellular energetic pools, which culminates in cell dysfunction and necrosis; and third, its capacity to promote the transcription of pro-inflammatory genes.  Consequently, pharmacol. inhibitors of PARP have the potential to enhance the cytotoxicity of certain DNA-damaging anticancer drugs, reduce parenchymal cell necrosis (for example, in stroke or myocardial infarction) and downregulate multiple simultaneous pathways of inflammation and tissue injury (for example, in circulatory shock, colitis or diabetic complications).  The first ultrapotent novel PARP inhibitors have now entered human clin. trials.  This article presents an overview of the principal pathophysiol. pathways and mechanisms that are governed by PARP, followed by the main structures and therapeutic actions of various classes of novel PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDsrfd_xv8g7Vg90H21EOLACvtfcHk0likmR8Nu_PBkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjs1OktL8%253D&md5=1dfd177b8064b017efb5dd7779655d55</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd1718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1718%26sid%3Dliteratum%253Aachs%26aulast%3DJagtap%26aufirst%3DP.%26aulast%3DSzab%25C3%25B3%26aufirst%3DC.%26atitle%3DPoly%2528ADP-ribose%2529%2520polymerase%2520and%2520the%2520therapeutic%2520effects%2520of%2520its%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2005%26volume%3D4%26spage%3D421%26epage%3D440%26doi%3D10.1038%2Fnrd1718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dantzer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ame, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Murcia, G.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose): novel functions for an old molecule</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">517</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1038/nrm1963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnrm1963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=16829982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1Ohsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=517-528&author=V.+Schreiberauthor=F.+Dantzerauthor=J.-C.+Ameauthor=G.+de+Murcia&title=Poly%28ADP-ribose%29%3A+novel+functions+for+an+old+molecule&doi=10.1038%2Fnrm1963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribose): novel functions for an old molecule</span></div><div class="casAuthors">Schreiber, Valerie; Dantzer, Francoise; Ame, Jean-Christophe; de Murcia, Gilbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">517-528</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The addn. to proteins of the neg. charged polymer of ADP-ribose, which is synthesized by poly(ADP-ribose) polymerase (PARPs) from NAD, is a unique post-translational modification.  It regulates not only cell survival and cell-death programs, but also an increasing no. of other biol. functions with which novel members of the PARP family have been assocd.  These functions include transcriptional regulation, telomere cohesion and mitotic spindle formation during cell division, intracellular trafficking, and energy metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLsI4_BKfjdrVg90H21EOLACvtfcHk0likmR8Nu_PBkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1Ohsb4%253D&md5=676e83755491500f3bf2137f348dfa1b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm1963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1963%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiber%26aufirst%3DV.%26aulast%3DDantzer%26aufirst%3DF.%26aulast%3DAme%26aufirst%3DJ.-C.%26aulast%3Dde%2BMurcia%26aufirst%3DG.%26atitle%3DPoly%2528ADP-ribose%2529%253A%2520novel%2520functions%2520for%2520an%2520old%2520molecule%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D7%26spage%3D517%26epage%3D528%26doi%3D10.1038%2Fnrm1963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durkacz, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omidiji, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shall, S.</span></span> <span> </span><span class="NLM_article-title">(ADP-ribose)n participates in DNA excision repair</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">593</span>– <span class="NLM_lpage">596</span>, <span class="refDoi"> DOI: 10.1038/283593a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2F283593a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=6243744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaL3cXktVagtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=1980&pages=593-596&author=B.+W.+Durkaczauthor=O.+Omidijiauthor=D.+A.+Grayauthor=S.+Shall&title=%28ADP-ribose%29n+participates+in+DNA+excision+repair&doi=10.1038%2F283593a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">(ADP-ribose)n participates in DNA excision repair</span></div><div class="casAuthors">Durkacz, Barbara W.; Omidiji, Olusesan; Gray, Douglas A.; Shall, Sydney</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">5747</span>),
    <span class="NLM_cas:pages">593-6</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The drop in cellular NAD content and the rejoining of DNA strand breaks, i.e. elevation of poly(ADP-ribose) polymerase (I) activity, after treatment of L1210 leukemia lymphoblast cells with alkylating agent, 10.5-210.6 μM Me2SO4, were inhibited by inhibitors of I, e.g. 3-aminobenzamide (Ki 4.4 μM at 26°).  The inhibitors all potentiated the cytotoxicity of Me2SO4; 2 mM aminobenzamide increased the cytotoxicity 4-fold.  In addn., the rejoining of strand breaks was prevented by nutritionally depleting cells of NAD (10-20% of normal).  Poly(ADP-ribose) may thus be involved in DNA excision repair, and possible mechanisms are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSidm4P44EPbVg90H21EOLACvtfcHk0liEzfKeTeFgTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXktVagtLc%253D&md5=93bfba1a7a3d3a2ea3705741d9f63734</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2F283593a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F283593a0%26sid%3Dliteratum%253Aachs%26aulast%3DDurkacz%26aufirst%3DB.%2BW.%26aulast%3DOmidiji%26aufirst%3DO.%26aulast%3DGray%26aufirst%3DD.%2BA.%26aulast%3DShall%26aufirst%3DS.%26atitle%3D%2528ADP-ribose%2529n%2520participates%2520in%2520DNA%2520excision%2520repair%26jtitle%3DNature%26date%3D1980%26volume%3D283%26spage%3D593%26epage%3D596%26doi%3D10.1038%2F283593a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bürke, A.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose). The most elaborate metabolite of NAD+</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>272</i></span>,  <span class="NLM_fpage">4576</span>– <span class="NLM_lpage">4589</span>, <span class="refDoi"> DOI: 10.1111/j.1742-4658.2005.04864.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1111%2Fj.1742-4658.2005.04864.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=16156780" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=2005&pages=4576-4589&author=A.+B%C3%BCrke&title=Poly%28ADP-ribose%29.+The+most+elaborate+metabolite+of+NAD%2B&doi=10.1111%2Fj.1742-4658.2005.04864.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2005.04864.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2005.04864.x%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25BCrke%26aufirst%3DA.%26atitle%3DPoly%2528ADP-ribose%2529.%2520The%2520most%2520elaborate%2520metabolite%2520of%2520NAD%252B%26jtitle%3DFEBS%2520J.%26date%3D2005%26volume%3D272%26spage%3D4576%26epage%3D4589%26doi%3D10.1111%2Fj.1742-4658.2005.04864.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, W. L.</span></span> <span> </span><span class="NLM_article-title">New insights into the molecular and cellular functions of poly(ADP-ribose) and PARPs</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">411</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1038/nrm3376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnrm3376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=22713970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC38XovVaht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=411-424&author=B.+A.+Gibsonauthor=W.+L.+Kraus&title=New+insights+into+the+molecular+and+cellular+functions+of+poly%28ADP-ribose%29+and+PARPs&doi=10.1038%2Fnrm3376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">New insights into the molecular and cellular functions of poly(ADP-ribose) and PARPs</span></div><div class="casAuthors">Gibson, Bryan A.; Kraus, W. Lee</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">411-424</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerases (PARPs) are enzymes that transfer ADP-ribose groups to target proteins and thereby affect various nuclear and cytoplasmic processes.  The activity of PARP family members, such as PARP1 and PARP2, is tied to cellular signaling pathways, and through poly(ADP-ribosyl)ation (PARylation) they ultimately promote changes in gene expression, RNA and protein abundance, and the location and activity of proteins that mediate signaling responses.  PARPs act in a complex response network that is driven by the cellular, mol. and chem. biol. of poly(ADP-ribose) (PAR).  This PAR-dependent response network is crucial for a broad array of physiol. and pathol. responses and thus is a good target for chem. therapeutics for several diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIm3s5z-XFELVg90H21EOLACvtfcHk0liEzfKeTeFgTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovVaht70%253D&md5=b72f18e950b804fed13258a3930fe004</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrm3376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3376%26sid%3Dliteratum%253Aachs%26aulast%3DGibson%26aufirst%3DB.%2BA.%26aulast%3DKraus%26aufirst%3DW.%2BL.%26atitle%3DNew%2520insights%2520into%2520the%2520molecular%2520and%2520cellular%2520functions%2520of%2520poly%2528ADP-ribose%2529%2520and%2520PARPs%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D411%26epage%3D424%26doi%3D10.1038%2Fnrm3376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stingl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghammer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haidacher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweiger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, E. F.</span></span> <span> </span><span class="NLM_article-title">Mice lacking ADPRT and poly(ADP-ribosyl)ation develop normally but are susceptible to skin disease</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">520</span>, <span class="refDoi"> DOI: 10.1101/gad.9.5.509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1101%2Fgad.9.5.509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=7698643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK2MXksFOksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=509-520&author=Z.+Q.+Wangauthor=B.+Auerauthor=L.+Stinglauthor=H.+Berghammerauthor=D.+Haidacherauthor=M.+Schweigerauthor=E.+F.+Wagner&title=Mice+lacking+ADPRT+and+poly%28ADP-ribosyl%29ation+develop+normally+but+are+susceptible+to+skin+disease&doi=10.1101%2Fgad.9.5.509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Mice lacking ADPRT and poly(ADP-ribosyl)ation develop normally but are susceptible to skin disease</span></div><div class="casAuthors">Wang, Zhaoqi; Auer, Bernhard; Stingl, Laura; Berghammer, Heinrich; Haidacher, Daniela; Schweiger, Manfred; Wagner, Erwin F.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">509-20</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Poly(ADP-ribosyl)ation is catalyzed by NAD+:protein(ADP-ribosyl)transferase (ADPRT), a chromatin-assocd. enzyme which, in the presence of DNA breaks, transfers ADP-ribose from NAD+ to nuclear proteins.  This post-translational modification has been implicated in many fundamental processes, like DNA repair, chromatin stability, cell proliferation, and cell death.  To elucidate the biol. function of ADPRT and poly(ADP-ribosyl)ation in vivo the gene was inactivated in the mouse germ line.  Mice homozygous for the ADPRT mutation are healthy and fertile.  Anal. of mutant tissues and fibroblasts isolated from mutant fetuses revealed the absence of ADPRT enzymic activity and poly(ADP-ribose), implying that no poly(ADP-ribosyl)ated proteins are present.  Mutant embryonic fibroblasts were able to efficiently repair DNA damaged by UV and alkylating agents.  However, proliferation of mutant primary fibroblasts as well as thymocytes following γ-radiation in vivo was impaired.  Moreover, mutant mice are susceptible to the spontaneous development of skin disease as ∼30% of older mice develop epidermal hyperplasia.  The generation of viable ADPRT-/-mice negates an essential role for this enzyme in normal chromatin function, but the impaired proliferation and the onset of skin lesions in older mice suggest a function for ADPRT in response to environmental stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5NqGG4AsF7rVg90H21EOLACvtfcHk0liEzfKeTeFgTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXksFOksLg%253D&md5=e7418b08b067e19b10e75e8f4c83b259</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1101%2Fgad.9.5.509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.9.5.509%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%2BQ.%26aulast%3DAuer%26aufirst%3DB.%26aulast%3DStingl%26aufirst%3DL.%26aulast%3DBerghammer%26aufirst%3DH.%26aulast%3DHaidacher%26aufirst%3DD.%26aulast%3DSchweiger%26aufirst%3DM.%26aulast%3DWagner%26aufirst%3DE.%2BF.%26atitle%3DMice%2520lacking%2520ADPRT%2520and%2520poly%2528ADP-ribosyl%2529ation%2520develop%2520normally%2520but%2520are%2520susceptible%2520to%2520skin%2520disease%26jtitle%3DGenes%2520Dev.%26date%3D1995%26volume%3D9%26spage%3D509%26epage%3D520%26doi%3D10.1101%2Fgad.9.5.509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Murcia, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niedergang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trucco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricoul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutrillaux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeMeur, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walztinger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Murcia, G.</span></span> <span> </span><span class="NLM_article-title">Requirement of poly(ADP-ribose) polymerase in recovery from DNA damage in mice and in cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">7303</span>– <span class="NLM_lpage">7307</span>, <span class="refDoi"> DOI: 10.1073/pnas.94.14.7303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1073%2Fpnas.94.14.7303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=9207086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK2sXksFOntb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=7303-7307&author=J.+M.+de%0AMurciaauthor=C.+Niedergangauthor=C.+Truccoauthor=M.+Ricoulauthor=B.+Dutrillauxauthor=M.+Markauthor=F.+J.+Oliverauthor=M.+Massonauthor=A.+Dierichauthor=M.+LeMeurauthor=C.+Walztingerauthor=P.+Chambonauthor=G.+de+Murcia&title=Requirement+of+poly%28ADP-ribose%29+polymerase+in+recovery+from+DNA+damage+in+mice+and+in+cells&doi=10.1073%2Fpnas.94.14.7303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Requirement of poly(ADP-ribose) polymerase in recovery from DNA damage in mice and in cells</span></div><div class="casAuthors">Menissier de Murcia, Josiane; Niedergang, Claude; Trucco, Carlotta; Ricoul, Michele; Dutrillaux, Bernard; Mark, Manuel; Oliver, F. Javier; Masson, Murielle; Dierich, Andree; LeMeur, Marianne; Walztinger, Caroline; Chambon, Pierre; de Murcia, Gilbert</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7303-7307</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase [PARP; NAD+ ADP-ribosyltransferase; NAD+:poly(adenosine-diphosphate-D-ribosyl)-acceptor ADP-D-ribosyltransferase, EC 2.4.2.30] is a zinc-finger DNA-binding protein that detects specifically DNA strand breaks generated by genotoxic agents.  To det. its biol. function, the authors have inactivated both alleles by gene targeting in mice.  Treatment of PARP-/- mice either by the alkylating agent N-methyl-N-nitrosourea (MNU) or by γ-irradn. revealed an extreme sensitivity and a high genomic instability to both agents.  Following whole body γ-irradn. (8 Gy) mutant mice died rapidly from acute radiation toxicity to the small intestine.  Mice-derived PARP-/- cells displayed a high sensitivity to MNU exposure: a G2/M arrest in mouse embryonic fibroblasts and a rapid apoptotic response and a p53 accumulation were obsd. in splenocytes.  Altogether these results demonstrate that PARP is a survival factor playing an essential and pos. role during DNA damage recovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlwmTBk-nHc7Vg90H21EOLACvtfcHk0ljYiumFsgz5cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksFOntb4%253D&md5=a652d7d877fa0c29ff1aeb9769fbd86f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.94.14.7303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.94.14.7303%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BMurcia%26aufirst%3DJ.%2BM.%26aulast%3DNiedergang%26aufirst%3DC.%26aulast%3DTrucco%26aufirst%3DC.%26aulast%3DRicoul%26aufirst%3DM.%26aulast%3DDutrillaux%26aufirst%3DB.%26aulast%3DMark%26aufirst%3DM.%26aulast%3DOliver%26aufirst%3DF.%2BJ.%26aulast%3DMasson%26aufirst%3DM.%26aulast%3DDierich%26aufirst%3DA.%26aulast%3DLeMeur%26aufirst%3DM.%26aulast%3DWalztinger%26aufirst%3DC.%26aulast%3DChambon%26aufirst%3DP.%26aulast%3Dde%2BMurcia%26aufirst%3DG.%26atitle%3DRequirement%2520of%2520poly%2528ADP-ribose%2529%2520polymerase%2520in%2520recovery%2520from%2520DNA%2520damage%2520in%2520mice%2520and%2520in%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1997%26volume%3D94%26spage%3D7303%26epage%3D7307%26doi%3D10.1073%2Fpnas.94.14.7303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tentori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graziani, G.</span></span> <span> </span><span class="NLM_article-title">Chemopotentiation by PARP inhibitors in cancer therapy</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2005.02.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.phrs.2005.02.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15911331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1Wltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2005&pages=25-33&author=L.+Tentoriauthor=G.+Graziani&title=Chemopotentiation+by+PARP+inhibitors+in+cancer+therapy&doi=10.1016%2Fj.phrs.2005.02.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Chemopotentiation by PARP inhibitors in cancer therapy</span></div><div class="casAuthors">Tentori, Lucio; Graziani, Grazia</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-33</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerases (PARP) constitute a family of enzymes involved in the regulation of many cellular processes such as DNA repair, gene transcription, cell cycle progression, cell death, chromatin functions and genomic stability.  Among the 18 members identified so far, PARP-1 and PARP-2 are the only proteins stimulated by DNA strand breaks and implicated in the repair of DNA injury.  Therefore, these mols. have been exploited as potential targets for the development of pharmacol. strategies to increase the antitumor efficacy of chemotherapeutic agents, which induce DNA damage.  PARP inhibitors have been shown to restore sensitivity of resistant tumors to methylating agents or topoisomerase I inhibitors, drugs presently used for the treatment of primary and secondary brain tumors or malignancies refractory to std. chemotherapy.  Interestingly, PARP inhibitors may also provide protection from the untoward effects exerted by certain anticancer drugs, which cause oxidative stress and consequent PARP overactivation.  The aim of this article is to provide a brief overview of the recent literature on preclin. studies with the specific and potent inhibitors newly synthesized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoImZCOb5elR7Vg90H21EOLACvtfcHk0ljYiumFsgz5cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1Wltr8%253D&md5=2a2f1893efeefc3f1d140979beae6f4d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2005.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2005.02.010%26sid%3Dliteratum%253Aachs%26aulast%3DTentori%26aufirst%3DL.%26aulast%3DGraziani%26aufirst%3DG.%26atitle%3DChemopotentiation%2520by%2520PARP%2520inhibitors%2520in%2520cancer%2520therapy%26jtitle%3DPharmacol.%2520Res.%26date%3D2005%26volume%3D52%26spage%3D25%26epage%3D33%26doi%3D10.1016%2Fj.phrs.2005.02.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iniesta, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadopoulos, K.</span></span> <span> </span><span class="NLM_article-title">Development of PARP inhibitors in oncology</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1517/13543780802525324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1517%2F13543780802525324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=19053880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVGhtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2009&pages=31-43&author=J.+Rodonauthor=M.+D.+Iniestaauthor=K.+Papadopoulos&title=Development+of+PARP+inhibitors+in+oncology&doi=10.1517%2F13543780802525324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Development of PARP inhibitors in oncology</span></div><div class="casAuthors">Rodon, Jordi; Iniesta, Maria D.; Papadopoulos, Kyriakos</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-43</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Poly (ADP-ribose) polymerase (PARP) plays a key role in DNA repair mechanisms by detecting and initiating repair after DNA strand breaks.  Inhibition of PARP in DNA repair-defective tumors (like those with BRCA1 or BRCA2 mutations) can lead to gross genomic instability and cell death.  Likewise, combining PARP inhibition with cytotoxic agents such as chemotherapy or radiation therapy is synergistic in many preclin. models.  Several drugs designed to inhibit PARP are currently in clin. development, many following a development path different from that of typical anticancer agents.  In this review we will focus on the early clin. data from PARP inhibitors that are entering clin. trials, the potential tumors they might target, their combination with other drugs and the different biomarkers that are being explored.  Concepts such as 'BRCAness', synthetic lethality, Phase 0 trials and pharmacodynamic markers will be discussed in the context of the development of PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNslMhf0aSo7Vg90H21EOLACvtfcHk0ljYiumFsgz5cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVGhtbvJ&md5=e031c4e938fd49b8cbf674dbc3ca8410</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1517%2F13543780802525324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543780802525324%26sid%3Dliteratum%253Aachs%26aulast%3DRodon%26aufirst%3DJ.%26aulast%3DIniesta%26aufirst%3DM.%2BD.%26aulast%3DPapadopoulos%26aufirst%3DK.%26atitle%3DDevelopment%2520of%2520PARP%2520inhibitors%2520in%2520oncology%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2009%26volume%3D18%26spage%3D31%26epage%3D43%26doi%3D10.1517%2F13543780802525324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ratnam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. A.</span></span> <span> </span><span class="NLM_article-title">Current development of clinical inhibitors of poly(ADP-ribose) polymerase in oncology</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1383</span>– <span class="NLM_lpage">1388</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-06-2260</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1158%2F1078-0432.CCR-06-2260" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=17332279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitlSjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=1383-1388&author=K.+Ratnamauthor=J.+A.+Low&title=Current+development+of+clinical+inhibitors+of+poly%28ADP-ribose%29+polymerase+in+oncology&doi=10.1158%2F1078-0432.ccr-06-2260"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Current Development of Clinical Inhibitors of Poly(ADP-Ribose) Polymerase in Oncology</span></div><div class="casAuthors">Ratnam, Kapila; Low, Jennifer A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1383-1388</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerase (PARP) is a nuclear enzyme that signals the presence of DNA damage by catalyzing the addn. of ADP-ribose units to DNA, histones, and various DNA repair enzymes and by facilitating DNA repair.  PARP has been gaining increasing interest as a therapeutic target for many diseases and esp. for cancer.  Inhibition of PARP potentiates the activity of DNA-damaging agents, such as alkylators, platinums, topoisomerase inhibitors, and radiation in in vitro and in vivo models.  In addn., tumors with DNA repair defects, such as those arising from patients with BRCA mutations, may be more sensitive to PARP inhibition.  At least five different companies have now initiated oncol. clin. trials with PARP inhibitors, ranging in stage from phase 0 to phase 2.  This review summarizes the preclin. and clin. data currently available for these agents and some of the challenges facing the clin. development of these agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1Muf0l4FhVLVg90H21EOLACvtfcHk0li0EV9r8ikwnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitlSjtr8%253D&md5=9676a8bf09a0ee2bbb19e3e26b060ba7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-06-2260&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-06-2260%26sid%3Dliteratum%253Aachs%26aulast%3DRatnam%26aufirst%3DK.%26aulast%3DLow%26aufirst%3DJ.%2BA.%26atitle%3DCurrent%2520development%2520of%2520clinical%2520inhibitors%2520of%2520poly%2528ADP-ribose%2529%2520polymerase%2520in%2520oncology%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2007%26volume%3D13%26spage%3D1383%26epage%3D1388%26doi%3D10.1158%2F1078-0432.ccr-06-2260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Narod, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foulkes, W. D.</span></span> <span> </span><span class="NLM_article-title">BRCA1 and BRCA2: 1994 and beyond</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">665</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1038/nrc1431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnrc1431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15343273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFCmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=665-676&author=S.+A.+Narodauthor=W.+D.+Foulkes&title=BRCA1+and+BRCA2%3A+1994+and+beyond&doi=10.1038%2Fnrc1431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">BRCA1 and BRCA2: 1994 and beyond</span></div><div class="casAuthors">Narod, Steven A.; Foulkes, William D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">665-676</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The discovery of the first gene assocd. with hereditary breast cancer, BRCA1, was anticipated to greatly increase our understanding of both hereditary and sporadic forms of breast cancer, and to lead to therapeutic and preventive breakthroughs.  Much has been learned during the past decade about the genetic epidemiol. of breast cancer, the ethnic distribution and clin. consequences of BRCA1 and BRCA2 mutations, and the central role of DNA repair in breast cancer susceptibility.  The ability to translate this knowledge into novel treatments, however, remains elusive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1a5S77C977bVg90H21EOLACvtfcHk0li0EV9r8ikwnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFCmsbk%253D&md5=200600041061619ba39dfbd8dfc91850</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc1431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1431%26sid%3Dliteratum%253Aachs%26aulast%3DNarod%26aufirst%3DS.%2BA.%26aulast%3DFoulkes%26aufirst%3DW.%2BD.%26atitle%3DBRCA1%2520and%2520BRCA2%253A%25201994%2520and%2520beyond%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D665%26epage%3D676%26doi%3D10.1038%2Fnrc1431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tutt, A. N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santarosa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knights, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>434</i></span>,  <span class="NLM_fpage">917</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1038/nature03445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnature03445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15829967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=434&publication_year=2005&pages=917-921&author=H.+Farmerauthor=N.+McCabeauthor=C.+J.+Lordauthor=A.+N.+J.+Tuttauthor=D.+A.+Johnsonauthor=T.+B.+Richardsonauthor=M.+Santarosaauthor=K.+J.+Dillonauthor=I.+Hicksonauthor=C.+Knightsauthor=N.+M.+B.+Martinauthor=S.+P.+Jacksonauthor=G.+C.+M.+Smithauthor=A.+Ashworth&title=Targeting+the+DNA+repair+defect+in+BRCA+mutant+cells+as+a+therapeutic+strategy&doi=10.1038%2Fnature03445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy</span></div><div class="casAuthors">Farmer, Hannah; McCabe, Nuala; Lord, Christopher J.; Tutt, Andrew N. J.; Johnson, Damian A.; Richardson, Tobias B.; Santarosa, Manuela; Dillon, Krystyna J.; Hickson, Ian; Knights, Charlotte; Martin, Niall M. B.; Jackson, Stephen P.; Smith, Graeme C. M.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">434</span>
        (<span class="NLM_cas:issue">7035</span>),
    <span class="NLM_cas:pages">917-921</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">BRCA1 and BRCA2 are important for DNA double-strand break repair by homologous recombination, and mutations in these genes predispose to breast and other cancers.  Poly(ADP-ribose) polymerase (PARP) is an enzyme involved in base excision repair, a key pathway in the repair of DNA single-strand breaks.  The authors show here that BRCA1 or BRCA2 dysfunction unexpectedly and profoundly sensitizes cells to the inhibition of PARP enzymic activity, resulting in chromosomal instability, cell cycle arrest and subsequent apoptosis.  This seems to be because the inhibition of PARP leads to the persistence of DNA lesions normally repaired by homologous recombination.  These results illustrate how different pathways cooperate to repair damage, and suggest that the targeted inhibition of particular DNA repair pathways may allow the design of specific and less toxic therapies for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomVTKE4mFwfrVg90H21EOLACvtfcHk0li0EV9r8ikwnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrc%253D&md5=03b5ba0adc21a5627fe17ae535f45078</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature03445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03445%26sid%3Dliteratum%253Aachs%26aulast%3DFarmer%26aufirst%3DH.%26aulast%3DMcCabe%26aufirst%3DN.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DTutt%26aufirst%3DA.%2BN.%2BJ.%26aulast%3DJohnson%26aufirst%3DD.%2BA.%26aulast%3DRichardson%26aufirst%3DT.%2BB.%26aulast%3DSantarosa%26aufirst%3DM.%26aulast%3DDillon%26aufirst%3DK.%2BJ.%26aulast%3DHickson%26aufirst%3DI.%26aulast%3DKnights%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DN.%2BM.%2BB.%26aulast%3DJackson%26aufirst%3DS.%2BP.%26aulast%3DSmith%26aufirst%3DG.%2BC.%2BM.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DTargeting%2520the%2520DNA%2520repair%2520defect%2520in%2520BRCA%2520mutant%2520cells%2520as%2520a%2520therapeutic%2520strategy%26jtitle%3DNature%26date%3D2005%26volume%3D434%26spage%3D917%26epage%3D921%26doi%3D10.1038%2Fnature03445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flower, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span> <span> </span><span class="NLM_article-title">Specific killing of BRCA2-deficient tumours with inhibitors of poly(ADP-ribose) polymerase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>434</i></span>,  <span class="NLM_fpage">913</span>– <span class="NLM_lpage">917</span>, <span class="refDoi"> DOI: 10.1038/nature03443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnature03443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=15829966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=434&publication_year=2005&pages=913-917&author=H.+E.+Bryantauthor=N.+Schultzauthor=H.+D.+Thomasauthor=K.+M.+Parkerauthor=D.+Flowerauthor=E.+Lopezauthor=S.+Kyleauthor=M.+Meuthauthor=N.+J.+Curtinauthor=T.+Helleday&title=Specific+killing+of+BRCA2-deficient+tumours+with+inhibitors+of+poly%28ADP-ribose%29+polymerase&doi=10.1038%2Fnature03443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Specific killing of BRCA2-deficient tumors with inhibitors of poly(ADP-ribose) polymerase</span></div><div class="casAuthors">Bryant, Helen E.; Schultz, Niklas; Thomas, Huw D.; Parker, Kayan M.; Flower, Dan; Lopez, Elena; Kyle, Suzanne; Meuth, Mark; Curtin, Nicola J.; Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">434</span>
        (<span class="NLM_cas:issue">7035</span>),
    <span class="NLM_cas:pages">913-917</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase (PARP1) facilitates DNA repair by binding to DNA breaks and attracting DNA repair proteins to the site of damage.  Nevertheless, PARP1-/- mice are viable, fertile and do not develop early onset tumors.  Here, the authors show that PARP inhibitors trigger γ-H2AX and RAD51 foci formation.  The authors propose that, in the absence of PARP1, spontaneous single-strand breaks collapse replication forks and trigger homologous recombination for repair.  Furthermore, the authors show that BRCA2-deficient cells, as a result of their deficiency in homologous recombination, are acutely sensitive to PARP inhibitors, presumably because resultant collapsed replication forks are no longer repaired.  Thus, PARP1 activity is essential in homologous recombination-deficient BRCA2 mutant cells.  The authors exploit this requirement in order to kill BRCA2-deficient tumors by PARP inhibition alone.  Treatment with PARP inhibitors is likely to be highly tumor specific, because only the tumors (which are BRCA2-/-) in BRCA2+/- patients are defective in homologous recombination.  The use of an inhibitor of a DNA repair enzyme alone to selectively kill a tumor, in the absence of an exogenous DNA-damaging agent, represents a new concept in cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZJ6B7d7dB77Vg90H21EOLACvtfcHk0lhkNDcMdJlSxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrY%253D&md5=44631eba667cf75ef99c31d011d6db44</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature03443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03443%26sid%3Dliteratum%253Aachs%26aulast%3DBryant%26aufirst%3DH.%2BE.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3DThomas%26aufirst%3DH.%2BD.%26aulast%3DParker%26aufirst%3DK.%2BM.%26aulast%3DFlower%26aufirst%3DD.%26aulast%3DLopez%26aufirst%3DE.%26aulast%3DKyle%26aufirst%3DS.%26aulast%3DMeuth%26aufirst%3DM.%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DSpecific%2520killing%2520of%2520BRCA2-deficient%2520tumours%2520with%2520inhibitors%2520of%2520poly%2528ADP-ribose%2529%2520polymerase%26jtitle%3DNature%26date%3D2005%26volume%3D434%26spage%3D913%26epage%3D917%26doi%3D10.1038%2Fnature03443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menear, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adcock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockcroft, X.-l.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copsey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cranston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drzewiecki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javaid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knights, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, V. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, I. T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. M. B.</span></span> <span> </span><span class="NLM_article-title">4-[3-(4-Cyclopropanecarbonylpiperazine-1-carbonyl)-4-fluorobenzyl]-2<i>H</i>-phthalazin- 1-one: a novel bioavailable inhibitor of poly(ADP-ribose) polymerase-1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">6581</span>– <span class="NLM_lpage">6591</span>, <span class="refDoi"> DOI: 10.1021/jm8001263</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8001263" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFeqs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=6581-6591&author=K.+A.+Menearauthor=C.+Adcockauthor=R.+Boulterauthor=X.-l.+Cockcroftauthor=L.+Copseyauthor=A.+Cranstonauthor=K.+J.+Dillonauthor=J.+Drzewieckiauthor=S.+Garmanauthor=S.+Gomezauthor=H.+Javaidauthor=F.+Kerriganauthor=C.+Knightsauthor=A.+Lauauthor=V.+M.+Lohauthor=I.+T.+W.+Matthewsauthor=S.+Mooreauthor=M.+J.+O%E2%80%99Connorauthor=G.+C.+M.+Smithauthor=N.+M.+B.+Martin&title=4-%5B3-%284-Cyclopropanecarbonylpiperazine-1-carbonyl%29-4-fluorobenzyl%5D-2H-phthalazin-+1-one%3A+a+novel+bioavailable+inhibitor+of+poly%28ADP-ribose%29+polymerase-1&doi=10.1021%2Fjm8001263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">4-[3-(4-Cyclopropanecarbonylpiperazine-1-carbonyl)-4-fluorobenzyl]-2H-phthalazin-1-one: A Novel Bioavailable Inhibitor of Poly(ADP-ribose) Polymerase-1</span></div><div class="casAuthors">Menear, Keith A.; Adcock, Claire; Boulter, Robert; Cockcroft, Xiao-ling; Copsey, Louise; Cranston, Aaron; Dillon, Krystyna J.; Drzewiecki, Jan; Garman, Sheila; Gomez, Sylvie; Javaid, Hashim; Kerrigan, Frank; Knights, Charlotte; Lau, Alan; Loh, Vincent M., Jr.; Matthews, Ian T. W.; Moore, Stephen; O'Connor, Mark J.; Smith, Graeme C. M.; Martin, Niall M. B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6581-6591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase activation is an immediate cellular response to metabolic-, chem.-, or ionizing radiation-induced DNA damage and represents a new target for cancer therapy.  In this article, we disclose a novel series of substituted 4-benzyl-2H-phthalazin-1-ones that possess high inhibitory enzyme and cellular potency for both PARP-1 and PARP-2.  Optimized compds. from the series also demonstrate good pharmacokinetic profiles, oral bioavailability, and activity in vivo in an SW620 colorectal cancer xenograft model. 4-[3-(4-Cyclopropanecarbonylpiperazine-1-carbonyl)-4-fluorobenzyl]-2H-phthalazin-1-one (KU-0059436, AZD2281) 47 is a single digit nanomolar inhibitor of both PARP-1 and PARP-2 that shows standalone activity against BRCA1-deficient breast cancer cell lines.  Compd. 47 is currently undergoing clin. development for the treatment of BRCA1- and BRCA2-defective cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3IKtdcfeWCbVg90H21EOLACvtfcHk0lhkNDcMdJlSxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFeqs7vE&md5=45aff9bb506360ec26c1c0039388240e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm8001263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8001263%26sid%3Dliteratum%253Aachs%26aulast%3DMenear%26aufirst%3DK.%2BA.%26aulast%3DAdcock%26aufirst%3DC.%26aulast%3DBoulter%26aufirst%3DR.%26aulast%3DCockcroft%26aufirst%3DX.-l.%26aulast%3DCopsey%26aufirst%3DL.%26aulast%3DCranston%26aufirst%3DA.%26aulast%3DDillon%26aufirst%3DK.%2BJ.%26aulast%3DDrzewiecki%26aufirst%3DJ.%26aulast%3DGarman%26aufirst%3DS.%26aulast%3DGomez%26aufirst%3DS.%26aulast%3DJavaid%26aufirst%3DH.%26aulast%3DKerrigan%26aufirst%3DF.%26aulast%3DKnights%26aufirst%3DC.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DLoh%26aufirst%3DV.%2BM.%26aulast%3DMatthews%26aufirst%3DI.%2BT.%2BW.%26aulast%3DMoore%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%2BJ.%26aulast%3DSmith%26aufirst%3DG.%2BC.%2BM.%26aulast%3DMartin%26aufirst%3DN.%2BM.%2BB.%26atitle%3D4-%255B3-%25284-Cyclopropanecarbonylpiperazine-1-carbonyl%2529-4-fluorobenzyl%255D-2H-phthalazin-%25201-one%253A%2520a%2520novel%2520bioavailable%2520inhibitor%2520of%2520poly%2528ADP-ribose%2529%2520polymerase-1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D6581%26epage%3D6591%26doi%3D10.1021%2Fjm8001263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span> <span> </span><span class="NLM_article-title">INPP4B overexpression enhances the antitumor efficacy of PARP inhibitor AG014699 in MDA-MB-231 triple-negative breast cancer cells</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">4469</span>– <span class="NLM_lpage">4477</span>, <span class="refDoi"> DOI: 10.1007/s13277-013-1589-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1007%2Fs13277-013-1589-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntlOkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=4469-4477&author=Y.+Sunauthor=H.+Dingauthor=X.+Liuauthor=X.+Liauthor=L.+Li&title=INPP4B+overexpression+enhances+the+antitumor+efficacy+of+PARP+inhibitor+AG014699+in+MDA-MB-231+triple-negative+breast+cancer+cells&doi=10.1007%2Fs13277-013-1589-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">INPP4B overexpression enhances the antitumor efficacy of PARP inhibitor AG014699 in MDA-MB-231 triple-negative breast cancer cells</span></div><div class="casAuthors">Sun, Ying; Ding, Huan; Liu, Xinguang; Li, Xiaoqing; Li, Li</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">4469-4477</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Although preclin. and clin. studies on poly-(ADP ribose) polymerase (PARP) inhibitor alone or in combination with DNA-damaging agents have shown promising results, further research to improve and broaden the application scope of this therapeutic approach is needed.  The main aim of this study was to evaluate whether overexpressing inositol polyphosphate 4-phosphatase type II (INPP4B) gene, a novel tumor suppressor gene neg. regulating the phosphatidylinositol 3-kinase (PI3K)/AKT signaling pathway, could enhance the antitumor efficacy of PARP inhibitor AG014699 used in the treatment of triple-neg. breast cancer (TNBC).  Here in this report, we used a TNBC cell line MDA-MB-231 without expression of INPP4B as the study model and a lentiviral system to stably overexpress INPP4B gene in MDA-MB-231 cells.  We detected that the overexpression of INPP4B could significantly suppress cell proliferation and block cell cycle progression in G1 phase via decreasing the protein level of phosphorylated AKT.  It is further revealed that PARP inhibitor AG014699 induced DNA damage conferring a G2/M arrest and decreased cell viability, which is paralleled by the induction of apoptosis.  However, PARP inhibitor AG014699 could activate the PI3K/AKT signaling pathway activity and partially offset its therapeutic efficacy.  In our study, a significant enhancement of proliferation inhibition was obsd. when INPP4B overexpression was combined with PARP inhibitor AG014699 in comparison with either single treatment.  The suppression of PI3K/AKT pathway caused by the overexpression of INPP4B contributed to the enhanced antitumor efficacy of the combined therapy.  Our in vitro results indicated that this exptl. therapeutic strategy combining INPP4B overexpression and PARP inhibitor AG014699 might be of potential therapeutic value as a new strategy for the treatment of patients with TNBC and is worthy of further study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrck78_MFHw9bVg90H21EOLACvtfcHk0lhkNDcMdJlSxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntlOkurY%253D&md5=ac96dcb8191167fdafae194bd5ce39d7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs13277-013-1589-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-013-1589-y%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26atitle%3DINPP4B%2520overexpression%2520enhances%2520the%2520antitumor%2520efficacy%2520of%2520PARP%2520inhibitor%2520AG014699%2520in%2520MDA-MB-231%2520triple-negative%2520breast%2520cancer%2520cells%26jtitle%3DTumor%2520Biol.%26date%3D2014%26volume%3D35%26spage%3D4469%26epage%3D4477%26doi%3D10.1007%2Fs13277-013-1589-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcoxen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toniatti, C.</span></span> <span> </span><span class="NLM_article-title">Niraparib: a poly(ADP-ribose) polymerase (PARP) inhibitor for the treatment of tumors with defective homologous recombination</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3302</span>– <span class="NLM_lpage">3314</span>, <span class="refDoi"> DOI: 10.1021/jm5018237</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5018237" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFams7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3302-3314&author=P.+Jonesauthor=K.+Wilcoxenauthor=M.+Rowleyauthor=C.+Toniatti&title=Niraparib%3A+a+poly%28ADP-ribose%29+polymerase+%28PARP%29+inhibitor+for+the+treatment+of+tumors+with+defective+homologous+recombination&doi=10.1021%2Fjm5018237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Niraparib: A Poly(ADP-ribose) Polymerase (PARP) Inhibitor for the Treatment of Tumors with Defective Homologous Recombination</span></div><div class="casAuthors">Jones, Philip; Wilcoxen, Keith; Rowley, Michael; Toniatti, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3302-3314</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerases (PARPs) are involved in DNA repair following damage by endogenous or exogenous processes.  It has become clear over the past decade that inhibition of PARP in the context of defects in other DNA repair mechanisms provide a tumor specific way to kill cancer cells.  We describe the rationale for this approach and the design and discovery of niraparib, a potent PARP-1/2 inhibitor with good cell based activity, selectivity for cancer over normal cells, and oral bioavailability.  Niraparib was characterized in a no. of preclin. models before moving to phase I clin. trials, where it showed excellent human pharmacokinetics suitable for once a day oral dosing, achieved its pharmacodynamic target for PARP inhibition, and had promising activity in cancer patients.  It is currently being tested in phase 3 clin. trials as maintenance therapy in ovarian cancer and as a treatment for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsaJDnhFDPhbVg90H21EOLACvtfcHk0lhzyaxqc7Zuiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFams7s%253D&md5=ffb7cb2503acde0a1e3452448d98693f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm5018237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5018237%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%26aulast%3DWilcoxen%26aufirst%3DK.%26aulast%3DRowley%26aufirst%3DM.%26aulast%3DToniatti%26aufirst%3DC.%26atitle%3DNiraparib%253A%2520a%2520poly%2528ADP-ribose%2529%2520polymerase%2520%2528PARP%2529%2520inhibitor%2520for%2520the%2520treatment%2520of%2520tumors%2520with%2520defective%2520homologous%2520recombination%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3302%26epage%3D3314%26doi%3D10.1021%2Fjm5018237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boshuizen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajrami, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Post, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">BMN 673, a novel and highly potent PARP1/2 inhibitor for the treatment of human cancers with DNA repair deficiency</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">5003</span>– <span class="NLM_lpage">5015</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-13-1391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1158%2F1078-0432.CCR-13-1391" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=23881923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVyns77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=5003-5015&author=Y.+Shenauthor=F.+L.+Rehmanauthor=Y.+Fengauthor=J.+Boshuizenauthor=I.+Bajramiauthor=R.+Elliottauthor=B.+Wangauthor=C.+J.+Lordauthor=L.+E.+Postauthor=A.+Ashworth&title=BMN+673%2C+a+novel+and+highly+potent+PARP1%2F2+inhibitor+for+the+treatment+of+human+cancers+with+DNA+repair+deficiency&doi=10.1158%2F1078-0432.ccr-13-1391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">BMN 673, a Novel and Highly Potent PARP1/2 Inhibitor for the Treatment of Human Cancers with DNA Repair Deficiency</span></div><div class="casAuthors">Shen, Yuqiao; Rehman, Farah L.; Feng, Ying; Boshuizen, Julia; Bajrami, Ilirjana; Elliott, Richard; Wang, Bing; Lord, Christopher J.; Post, Leonard E.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5003-5015</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: PARP1/2 inhibitors are a class of anticancer agents that target tumor-specific defects in DNA repair.  Here, we describe BMN 673, a novel, highly potent PARP1/2 inhibitor with favorable metabolic stability, oral bioavailability, and pharmacokinetic properties.  Exptl. Design: Potency and selectivity of BMN 673 was detd. by biochem. assays.  Anticancer activity either as a single-agent or in combination with other antitumor agents was evaluated both in vitro and in xenograft cancer models.  Results: BMN 673 is a potent PARP1/2 inhibitor (PARP1 IC50 = 0.57 nmol/L), but it does not inhibit other enzymes that we have tested.  BMN 673 exhibits selective antitumor cytotoxicity and elicits DNA repair biomarkers at much lower concns. than earlier generation PARP1/2 inhibitors (such as olaparib, rucaparib, and veliparib).  In vitro, BMN 673 selectively targeted tumor cells with BRCA1, BRCA2, or PTEN gene defects with 20- to more than 200-fold greater potency than existing PARP1/2 inhibitors.  BMN 673 is readily orally bioavailable, with more than 40% abs. oral bioavailability in rats when dosed in carboxylmethyl cellulose.  Oral administration of BMN 673 elicited remarkable antitumor activity in vivo; xenografted tumors that carry defects in DNA repair due to BRCA mutations or PTEN deficiency were profoundly sensitive to oral BMN 673 treatment at well-tolerated doses in mice.  Synergistic or additive antitumor effects were also found when BMN 673 was combined with temozolomide, SN38, or platinum drugs.  Conclusion: BMN 673 is currently in early-phase clin. development and represents a promising PARP1/2 inhibitor with potentially advantageous features in its drug class.  Clin Cancer Res; 19(18); 5003-15. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWdmHIVEuoz7Vg90H21EOLACvtfcHk0lhzyaxqc7Zuiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVyns77J&md5=f58b67f14784c0a0fa1197caffd8d96d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-13-1391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-13-1391%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DRehman%26aufirst%3DF.%2BL.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DBoshuizen%26aufirst%3DJ.%26aulast%3DBajrami%26aufirst%3DI.%26aulast%3DElliott%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DPost%26aufirst%3DL.%2BE.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DBMN%2520673%252C%2520a%2520novel%2520and%2520highly%2520potent%2520PARP1%252F2%2520inhibitor%2520for%2520the%2520treatment%2520of%2520human%2520cancers%2520with%2520DNA%2520repair%2520deficiency%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26spage%3D5003%26epage%3D5015%26doi%3D10.1158%2F1078-0432.ccr-13-1391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, C.</span></span> <span> </span><span class="NLM_article-title">Therapeutic applications of PARP inhibitors: anticancer therapy and beyond</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1217</span>– <span class="NLM_lpage">1256</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2013.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.mam.2013.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=23370117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislyqtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=1217-1256&author=N.+J.+Curtinauthor=C.+Szabo&title=Therapeutic+applications+of+PARP+inhibitors%3A+anticancer+therapy+and+beyond&doi=10.1016%2Fj.mam.2013.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic applications of PARP inhibitors: Anticancer therapy and beyond</span></div><div class="casAuthors">Curtin, Nicola J.; Szabo, Csaba</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1217-1256</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The aim of this article is to describe the current and potential clin. translation of pharmacol. inhibitors of poly(ADP-ribose) polymerase (PARP) for the therapy of various diseases.  The first section of the present review summarizes the available preclin. and clin. data with PARP inhibitors in various forms of cancer.  In this context, the role of PARP in single-strand DNA break repair is relevant, leading to replication-assocd. lesions that cannot be repaired if homologous recombination repair (HRR) is defective, and the synthetic lethality of PARP inhibitors in HRR-defective cancer.  HRR defects are classically assocd. with BRCA1 and 2 mutations assocd. with familial breast and ovarian cancer, but there may be many other causes of HRR defects.  Thus, PARP inhibitors may be the drugs of choice for BRCA mutant breast and ovarian cancers, and extend beyond these tumors if appropriate biomarkers can be developed to identify HRR defects.  Multiple lines of preclin. data demonstrate that PARP inhibition increases cytotoxicity and tumor growth delay in combination with temozolomide, topoisomerase inhibitors and ionizing radiation.  Both single agent and combination clin. trials are underway.  The final part of the first section of the present review summarizes the current status of the various PARP inhibitors that are in various stages of clin. development.  The second section of the present review summarizes the role of PARP in selected non-oncol. indications.  In a no. of severe, acute diseases (such as stroke, neurotrauma, circulatory shock and acute myocardial infarction) the clin. translatability of PARP inhibition is supported by multiple lines of preclin. data, as well as observational data demonstrating PARP activation in human tissue samples.  In these disease indications, PARP overactivation due to oxidative and nitrative stress drives cell necrosis and pro-inflammatory gene expression, which contributes to disease pathol.  Accordingly, multiple lines of preclin. data indicate the efficacy of PARP inhibitors to preserve viable tissue and to down-regulate inflammatory responses.  As the clin. trials with PARP inhibitors in various forms of cancer progress, it is hoped that a second line of clin. investigations, aimed at testing of PARP inhibitors for various non-oncol. indications, will be initiated, as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP4INVXcZ0zbVg90H21EOLACvtfcHk0lhzyaxqc7Zuiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislyqtrc%253D&md5=6a514e160666b5dde38e1365211cd41e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2013.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2013.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26aulast%3DSzabo%26aufirst%3DC.%26atitle%3DTherapeutic%2520applications%2520of%2520PARP%2520inhibitors%253A%2520anticancer%2520therapy%2520and%2520beyond%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2013%26volume%3D34%26spage%3D1217%26epage%3D1256%26doi%3D10.1016%2Fj.mam.2013.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Im, S.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senkus, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domchek, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaloge, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goessl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Runswick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conte, P.</span></span> <span> </span><span class="NLM_article-title">Olaparib for metastatic breast cancer in patients with a germline BRCA mutation</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>377</i></span>,  <span class="NLM_fpage">523</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1056/nejmoa1706450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1056%2FNEJMoa1706450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=28578601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlyksrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=377&publication_year=2017&pages=523-533&author=M.+Robsonauthor=S.-A.+Imauthor=E.+Senkusauthor=B.+Xuauthor=S.+M.+Domchekauthor=N.+Masudaauthor=S.+Delalogeauthor=W.+Liauthor=N.+Tungauthor=A.+Armstrongauthor=W.+Wuauthor=C.+Goesslauthor=S.+Runswickauthor=P.+Conte&title=Olaparib+for+metastatic+breast+cancer+in+patients+with+a+germline+BRCA+mutation&doi=10.1056%2Fnejmoa1706450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Olaparib for metastatic breast cancer in patients with a germline BRCA mutation</span></div><div class="casAuthors">Robson, Mark; Im, Seock-Ah; Senkus, Elzbieta; Xu, Binghe; Domchek, Susan M.; Masuda, Norikazu; Delaloge, Suzette; Li, Wei; Tung, Nadine; Armstrong, Anne; Wu, Wenting; Goessl, Carsten; Runswick, Sarah; Conte, Pierfranco</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">377</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">523-533</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Olaparib is an oral poly(ADP-ribose) polymerase inhibitor that has promising antitumor activity in patients with metastatic breast cancer and a germline BRCA mutation.  We conducted a randomized, open-label, phase 3 trial in which olaparib monotherapy was compared with std. therapy in patients with a germline BRCA mutation and human epidermal growth factor receptor type 2 (HER2)-neg. metastatic breast cancer who had received no more than two previous chemotherapy regimens for metastatic disease.  Patients were randomly assigned, in a 2:1 ratio, to receive olaparib tablets (300 mg twice daily) or std. therapy with single-agent chemotherapy of the physician's choice (capecitabine, eribulin, or vinorelbine in 21-day cycles).  The primary end point was progression-free survival, which was assessed by blinded independent central review and was analyzed on an intention-to-treat basis. results Of the 302 patients who underwent randomization, 205 were assigned to receive olaparib and 97 were assigned to receive std. therapy.  Median progression-free survival was significantly longer in the olaparib group than in the std.-therapy group (7.0 mo vs. 4.2 mo; hazard ratio for disease progression or death, 0.58; 95% confidence interval, 0.43 to 0.80; P<0.001).  The response rate was 59.9% in the olaparib group and 28.8% in the std.-therapy group.  The rate of grade 3 or higher adverse events was 36.6% in the olaparib group and 50.5% in the std.-therapy group, and the rate of treatment discontinuation due to toxic effects was 4.9% and 7.7%, resp.  Among patients with HER2-neg. metastatic breast cancer and a germline BRCA mutation, olaparib monotherapy provided a significant benefit over std. therapy; median progression-free survival was 2.8 mo longer and the risk of disease progression or death was 42% lower with olaparib monotherapy than with std. therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMXxR-ZN4IGrVg90H21EOLACvtfcHk0lhSa06LaD1iKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlyksrbN&md5=7c8c62182a717c16c5762443fb3f6d5d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1706450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1706450%26sid%3Dliteratum%253Aachs%26aulast%3DRobson%26aufirst%3DM.%26aulast%3DIm%26aufirst%3DS.-A.%26aulast%3DSenkus%26aufirst%3DE.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DDomchek%26aufirst%3DS.%2BM.%26aulast%3DMasuda%26aufirst%3DN.%26aulast%3DDelaloge%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DTung%26aufirst%3DN.%26aulast%3DArmstrong%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DGoessl%26aufirst%3DC.%26aulast%3DRunswick%26aufirst%3DS.%26aulast%3DConte%26aufirst%3DP.%26atitle%3DOlaparib%2520for%2520metastatic%2520breast%2520cancer%2520in%2520patients%2520with%2520a%2520germline%2520BRCA%2520mutation%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2017%26volume%3D377%26spage%3D523%26epage%3D533%26doi%3D10.1056%2Fnejmoa1706450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J.</span></span> <span> </span><span class="NLM_article-title">Laying a trap to kill cancer cells: PARP inhibitors and their mechanisms of action</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">362ps17</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aaf9246</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1126%2Fscitranslmed.aaf9246" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=27797957" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=362ps17&author=Y.+Pommierauthor=M.+J.+O%E2%80%99Connorauthor=J.+de+Bono&title=Laying+a+trap+to+kill+cancer+cells%3A+PARP+inhibitors+and+their+mechanisms+of+action&doi=10.1126%2Fscitranslmed.aaf9246"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aaf9246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aaf9246%26sid%3Dliteratum%253Aachs%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%2BJ.%26aulast%3Dde%2BBono%26aufirst%3DJ.%26atitle%3DLaying%2520a%2520trap%2520to%2520kill%2520cancer%2520cells%253A%2520PARP%2520inhibitors%2520and%2520their%2520mechanisms%2520of%2520action%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2016%26volume%3D8%26spage%3D362ps17%26doi%3D10.1126%2Fscitranslmed.aaf9246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">PARP trapping beyond homologous recombination and platinum sensitivity in cancers</span>. <i>Ann. Rev. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1146/annurev-cancerbio-030518-055914</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1146%2Fannurev-cancerbio-030518-055914" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2019&pages=131-150&author=J.+Muraiauthor=Y.+Pommier&title=PARP+trapping+beyond+homologous+recombination+and+platinum+sensitivity+in+cancers&doi=10.1146%2Fannurev-cancerbio-030518-055914"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1146%2Fannurev-cancerbio-030518-055914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-cancerbio-030518-055914%26sid%3Dliteratum%253Aachs%26aulast%3DMurai%26aufirst%3DJ.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DPARP%2520trapping%2520beyond%2520homologous%2520recombination%2520and%2520platinum%2520sensitivity%2520in%2520cancers%26jtitle%3DAnn.%2520Rev.%2520Cancer%2520Biol.%26date%3D2019%26volume%3D3%26spage%3D131%26epage%3D150%26doi%3D10.1146%2Fannurev-cancerbio-030518-055914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">PARP inhibitors: synthetic lethality in the clinic</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1152</span>– <span class="NLM_lpage">1158</span>, <span class="refDoi"> DOI: 10.1126/science.aam7344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1126%2Fscience.aam7344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=28302823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1152-1158&author=C.+J.+Lordauthor=A.+Ashworth&title=PARP+inhibitors%3A+synthetic+lethality+in+the+clinic&doi=10.1126%2Fscience.aam7344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">PARP inhibitors: Synthetic lethality in the clinic</span></div><div class="casAuthors">Lord, Christopher J.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6330</span>),
    <span class="NLM_cas:pages">1152-1158</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  PARP inhibitors (PARPi), a cancer therapy targeting poly(ADP-ribose) polymerase, are the first clin. approved drugs designed to exploit synthetic lethality, a genetic concept proposed nearly a century ago.  Tumors arising in patients who carry germline mutations in either BRCA1 or BRCA2 are sensitive to PARPi because they have a specific type of DNA repair defect.  PARPi also show promising activity in more common cancers that share this repair defect.  However, as with other targeted therapies, resistance to PARPi arises in advanced disease.  In addn., detg. the optimal use of PARPi within drug combination approaches has been challenging.  Nevertheless, the preclin. discovery of PARPi synthetic lethality and the route to clin. approval provide interesting lessons for the development of other therapies.  Here, the authors discuss current knowledge of PARP inhibitors and potential ways to maximize their clin. effectiveness.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAdgOVt6r40rVg90H21EOLACvtfcHk0lhSa06LaD1iKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmtrg%253D&md5=6f3c5c37b2d5062cb7acbf3a78727f46</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1126%2Fscience.aam7344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aam7344%26sid%3Dliteratum%253Aachs%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DPARP%2520inhibitors%253A%2520synthetic%2520lethality%2520in%2520the%2520clinic%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1152%26epage%3D1158%26doi%3D10.1126%2Fscience.aam7344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Endres, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waeber, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moskowitz, M. A.</span></span> <span> </span><span class="NLM_article-title">Ischemic brain injury is mediated by the activation of poly(ADP-ribose)polymerase</span>. <i>J. Cereb. Blood Flow Metab.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1143</span>– <span class="NLM_lpage">1151</span>, <span class="refDoi"> DOI: 10.1097/00004647-199711000-00002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1097%2F00004647-199711000-00002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=9390645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK2sXotVShtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=1143-1151&author=M.+Endresauthor=Z.-Q.+Wangauthor=S.+Namuraauthor=C.+Waeberauthor=M.+A.+Moskowitz&title=Ischemic+brain+injury+is+mediated+by+the+activation+of+poly%28ADP-ribose%29polymerase&doi=10.1097%2F00004647-199711000-00002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Ischemic brain injury is mediated by the activation of poly(ADP-ribose)polymerase</span></div><div class="casAuthors">Endres, Matthias; Wang, Zhao-Qi; Namura, Shobu; Waeber, Christian; Moskowitz, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cerebral Blood Flow and Metabolism</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1143-1151</span>CODEN:
                <span class="NLM_cas:coden">JCBMDN</span>;
        ISSN:<span class="NLM_cas:issn">0271-678X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott-Raven Publishers</span>)
        </div><div class="casAbstract">Poly(ADP-ribose)polymerase (PARP, EC 2.4.2.30), an abundant nuclear protein activated by DNA nicks, mediates cell death in vitro by NAD (NAD) depletion after exposure to nitric oxide.  The authors examd. whether genetic deletion of PARP (PARP null mice) or its pharmacol. inhibition by 3-aminobenzamide (3-AB) attenuates tissue injury after transient cerebral ischemia.  Twenty-two hours after reperfusion following 2 h of filamentous middle cerebral artery occlusion, ischemic injury was decreased in PARP-/- and PARP+/- mice compared with PARP+/+ litter mates, and also was attenuated in 129/SV wild-type mice after 3-AB treatment compared with controls.  Infarct sparing was accompanied by functional recovery in PARP-/- and 3-AB-treated mice.  Increased poly(ADP-ribose) immunostaining obsd. in ischemic cell nuclei 5 min after reperfusion was reduced by 3-AB treatment.  Levels of NAD-the substrate of PARP-were reduced 2 h after reperfusion and were 35% of contralateral levels at 24 h.  The decreases were attenuated in PARP-/- mice and in 3-AB-treated animals.  Poly(ADP-ribose)polymerase cleavage by caspase-3 (CPP-32) has been proposed as an important step in apoptotic cell death.  Markers of apoptosis, such as oligonucleosomal DNA damage, total DNA fragmentation, and the d. of terminal deoxy-nucleotidyl transferase dUTP nick-end-labeled (TUNEL +) cells, however, did not differ in ischemic brain tissue of PARP-/- mice or in 3-AB-treated animals vs. controls, although there were differences in the no. of TUNEL-stained cells reflecting the decrease in infarct size.  Thus, ischemic brain injury activates PARP and contributes to cell death most likely by NAD depletion and energy failure, although the authors have not excluded a role for PARP in apoptotic cell death at earlier or later stages in ischemic cell death.  Inhibitors of PARP activation could provide a potential therapy in acute stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZgfj6WehsELVg90H21EOLACvtfcHk0lhSa06LaD1iKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXotVShtbo%253D&md5=38ad29424df8e6f3613cab8c38b0dfcb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1097%2F00004647-199711000-00002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00004647-199711000-00002%26sid%3Dliteratum%253Aachs%26aulast%3DEndres%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DZ.-Q.%26aulast%3DNamura%26aufirst%3DS.%26aulast%3DWaeber%26aufirst%3DC.%26aulast%3DMoskowitz%26aufirst%3DM.%2BA.%26atitle%3DIschemic%2520brain%2520injury%2520is%2520mediated%2520by%2520the%2520activation%2520of%2520poly%2528ADP-ribose%2529polymerase%26jtitle%3DJ.%2520Cereb.%2520Blood%2520Flow%2520Metab.%26date%3D1997%26volume%3D17%26spage%3D1143%26epage%3D1151%26doi%3D10.1097%2F00004647-199711000-00002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eliasson, M. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampei, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandir, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurn, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traystman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose) polymerase gene disruption renders mice resistant to cerebral ischemia</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1095</span>, <span class="refDoi"> DOI: 10.1038/nm1097-1089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnm1097-1089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=9334719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK2sXmsFyqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1997&pages=1089-1095&author=M.+J.+L.+Eliassonauthor=K.+Sampeiauthor=A.+S.+Mandirauthor=P.+D.+Hurnauthor=R.+J.+Traystmanauthor=J.+Baoauthor=A.+Pieperauthor=Z.-Q.+Wangauthor=T.+M.+Dawsonauthor=S.+H.+Snyderauthor=V.+L.+Dawson&title=Poly%28ADP-ribose%29+polymerase+gene+disruption+renders+mice+resistant+to+cerebral+ischemia&doi=10.1038%2Fnm1097-1089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribose) polymerase gene disruption renders mice resistant to cerebral ischemia</span></div><div class="casAuthors">Eliasson, Mikael J. L.; Sampei, Kenji; Mandir, Allen S.; Hurn, Patricia D.; Traystman, Richard J.; Bao, Jun; Pieper, Andrew; Wang, Zhao-Qi; Dawson, Ted M.; Snyder, Solomon H.; Dawson, Valina L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1089-1095</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Nitric oxide (NO) and peroxynitrite, formed from NO and superoxide anion, have been implicated as mediators of neuronal damage following focal ischemia, but their mol. targets have not been defined.  One candidate pathway is DNA damage leading to activation of the nuclear enzyme, poly(ADP-ribose) polymerase (PARP), which catalyzes attachment of ADP ribose units from NAD to nuclear proteins following DNA damage.  Excessive activation of PARP can deplete NAD and ATP, which is consumed in regeneration of NAD, leading to cell death by energy depletion.  The authors show that genetic disruption of PARP provides profound protection against glutamate-NO-mediated ischemic insults in vitro and major decreases in infarct vol. after reversible middle cerebral artery occlusion.  These results provide compelling evidence for a primary involvement of PARP activation in neuronal damage following focal ischemia and suggest that therapies designed towards inhibiting PARP may provide benefit in the treatment of cerebrovascular disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLSq-7dMGKE7Vg90H21EOLACvtfcHk0lgN8Lkdp8Dh1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsFyqtb8%253D&md5=d83a2bb75e066d20b415119fb4452e21</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnm1097-1089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1097-1089%26sid%3Dliteratum%253Aachs%26aulast%3DEliasson%26aufirst%3DM.%2BJ.%2BL.%26aulast%3DSampei%26aufirst%3DK.%26aulast%3DMandir%26aufirst%3DA.%2BS.%26aulast%3DHurn%26aufirst%3DP.%2BD.%26aulast%3DTraystman%26aufirst%3DR.%2BJ.%26aulast%3DBao%26aufirst%3DJ.%26aulast%3DPieper%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DZ.-Q.%26aulast%3DDawson%26aufirst%3DT.%2BM.%26aulast%3DSnyder%26aufirst%3DS.%2BH.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26atitle%3DPoly%2528ADP-ribose%2529%2520polymerase%2520gene%2520disruption%2520renders%2520mice%2520resistant%2520to%2520cerebral%2520ischemia%26jtitle%3DNat.%2520Med.%26date%3D1997%26volume%3D3%26spage%3D1089%26epage%3D1095%26doi%3D10.1038%2Fnm1097-1089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pieper, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walles, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clements, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweier, J. L.</span></span> <span> </span><span class="NLM_article-title">Myocardial postischemic injury is reduced by polyADPripose polymerase-1 gene disruption</span>. <i>Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1007/bf03401936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1007%2FBF03401936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10949908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlsFWjsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=271-282&author=A.+A.+Pieperauthor=T.+Wallesauthor=G.+Weiauthor=E.+E.+Clementsauthor=A.+Vermaauthor=S.+H.+Snyderauthor=J.+L.+Zweier&title=Myocardial+postischemic+injury+is+reduced+by+polyADPripose+polymerase-1+gene+disruption&doi=10.1007%2Fbf03401936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Myocardial postischemic injury is reduced by polyADPribose polymerase-1 gene disruption</span></div><div class="casAuthors">Pieper, Andrew A.; Walles, Thorsten; Wei, Guo; Clements, Emily E.; Verma, Ajay; Snyder, Solomon H.; Zweier, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine (New York)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-282</span>CODEN:
                <span class="NLM_cas:coden">MOMEF3</span>;
        ISSN:<span class="NLM_cas:issn">1076-1551</span>.
    
            (<span class="NLM_cas:orgname">Johns Hopkins University Press</span>)
        </div><div class="casAbstract">PolyADPribose polymerase (PARP) is activated by DNA strand breaks to catalyze the addn. of ADPribose groups to nuclear proteins, esp. PARP-1.  Excessive polyADPribosylation leads to cell death through depletion of NAD+ and ATP.  In vivo PARP activation in heart tissue slices was assayed through conversion of [33P]NAD+ into polyADPribose (PAR) following ischemia-reperfusion (I/R) and also monitored by immunohistochem. staining for PAR.  Cardiac contractility, nitric oxide (NO), reactive oxygen species (ROS), NAD+ and ATP levels were examd. in wild type (WT) and in PARP-1 gene-deleted (PARP-1-/-) isolated, perfused mouse hearts.  Myocardial infarct size was assessed following coronary artery occlusion in rats treated with PARP inhibitors.  Ischemia-reperfusion (I/R) augmented formation of nitric oxide, oxygen free radicals and PARP activity.  I/R induced decreases in cardiac contractility and NAD+ levels were attenuated in PARP-1-/- mouse hearts.  PARP inhibitors reduced myocardial infarct size in rats.  Residual polyADPribosylation in PARP-1-/- hearts may reflect alternative forms of PARP.  PolyADPribosylation from PARP-1 and other sources of enzymic PAR synthesis is assocd. with cardiac damage following myocardial ischemia.  PARP inhibitors may have therapeutic utility in myocardial disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrarESEjtYqqrVg90H21EOLACvtfcHk0lgN8Lkdp8Dh1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlsFWjsrs%253D&md5=7c656ce837c86eca120242f39d94a2ab</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2FBF03401936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF03401936%26sid%3Dliteratum%253Aachs%26aulast%3DPieper%26aufirst%3DA.%2BA.%26aulast%3DWalles%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DClements%26aufirst%3DE.%2BE.%26aulast%3DVerma%26aufirst%3DA.%26aulast%3DSnyder%26aufirst%3DS.%2BH.%26aulast%3DZweier%26aufirst%3DJ.%2BL.%26atitle%3DMyocardial%2520postischemic%2520injury%2520is%2520reduced%2520by%2520polyADPripose%2520polymerase-1%2520gene%2520disruption%26jtitle%3DMol.%2520Med.%26date%3D2000%26volume%3D6%26spage%3D271%26epage%3D282%26doi%3D10.1007%2Fbf03401936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devalaraja-Narashimha, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singaravelu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padanilam, B. J.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose) polymerase-1 gene ablation protects mice from ischemic renal injury</span>. <i>Am. J. Physiol.: Renal, Fluid Electrolyte Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">F387</span>– <span class="NLM_lpage">F398</span>, <span class="refDoi"> DOI: 10.1152/ajprenal.00436.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1152%2Fajprenal.00436.2003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2005&pages=F387-F398&author=J.+Zhengauthor=K.+Devalaraja-Narashimhaauthor=K.+Singaraveluauthor=B.+J.+Padanilam&title=Poly%28ADP-ribose%29+polymerase-1+gene+ablation+protects+mice+from+ischemic+renal+injury&doi=10.1152%2Fajprenal.00436.2003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1152%2Fajprenal.00436.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajprenal.00436.2003%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDevalaraja-Narashimha%26aufirst%3DK.%26aulast%3DSingaravelu%26aufirst%3DK.%26aulast%3DPadanilam%26aufirst%3DB.%2BJ.%26atitle%3DPoly%2528ADP-ribose%2529%2520polymerase-1%2520gene%2520ablation%2520protects%2520mice%2520from%2520ischemic%2520renal%2520injury%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Renal%252C%2520Fluid%2520Electrolyte%2520Physiol.%26date%3D2005%26volume%3D288%26spage%3DF387%26epage%3DF398%26doi%3D10.1152%2Fajprenal.00436.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zingarelli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabó, C.</span></span> <span> </span><span class="NLM_article-title">Effect of genetic disruption of poly(ADP-ribose) synthetase on delayed production of inflammatory mediators and delayed necrosis during myocardial ischemia-reperfusion injury</span>. <i>Shock</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">60</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1097/00024382-200013010-00011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1097%2F00024382-200013010-00011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10638671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1ajuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2000&pages=60-66&author=Z.+Yangauthor=B.+Zingarelliauthor=C.+Szab%C3%B3&title=Effect+of+genetic+disruption+of+poly%28ADP-ribose%29+synthetase+on+delayed+production+of+inflammatory+mediators+and+delayed+necrosis+during+myocardial+ischemia-reperfusion+injury&doi=10.1097%2F00024382-200013010-00011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of genetic disruption of poly (ADP-ribose) synthetase on delayed production of inflammatory mediators and delayed necrosis during myocardial ischemia-reperfusion injury</span></div><div class="casAuthors">Yang Z; Zingarelli B; Szabo C</div><div class="citationInfo"><span class="NLM_cas:title">Shock (Augusta, Ga.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">60-6</span>
        ISSN:<span class="NLM_cas:issn">1073-2322</span>.
    </div><div class="casAbstract">The nuclear enzyme poly (ADP ribose) synthetase (PARS) has been shown to play an important role in the pathogenesis of various forms of ischemia or reperfusion injury and circulatory shock.  Recent studies demonstrated that inhibition or genetic inactivation of PARS is beneficial in the early phase of myocardial reperfusion injury.  The aim of the present study was to investigate whether inactivation of PARS influences the delayed myocardial necrosis and the production of the proinflammatory cytokine tumor necrosis factor alpha (TNFalpha), the anti-inflammatory cytokine interleukin-10 (IL-10), and the free radical nitric oxide in the late stage of myocardial reperfusion injury.  The results demonstrate that genetic disruption of PARS provides marked protection against the delayed myocardial ischemia and reperfusion injury.  In addition, in the absence of functional PARS, a suppression of TNFalpha, IL-10, and nitric oxide production was found.  These findings provide direct evidence that PARS activation participates in the development of delayed cell injury and delayed mediator production in myocardial reperfusion injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7mQt-ICR3qNfW6udTcc2ea8w5Rf91hl97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1ajuw%253D%253D&md5=755aeae7fcb37384e374d064f00c9e04</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1097%2F00024382-200013010-00011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00024382-200013010-00011%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DZingarelli%26aufirst%3DB.%26aulast%3DSzab%25C3%25B3%26aufirst%3DC.%26atitle%3DEffect%2520of%2520genetic%2520disruption%2520of%2520poly%2528ADP-ribose%2529%2520synthetase%2520on%2520delayed%2520production%2520of%2520inflammatory%2520mediators%2520and%2520delayed%2520necrosis%2520during%2520myocardial%2520ischemia-reperfusion%2520injury%26jtitle%3DShock%26date%3D2000%26volume%3D13%26spage%3D60%26epage%3D66%26doi%3D10.1097%2F00024382-200013010-00011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szabó, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span> <span> </span><span class="NLM_article-title">Role of poly(ADP-ribose)synthetase in inflammation</span>. <i>Trends Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">298</span>, <span class="refDoi"> DOI: 10.1016/s0165-6147(98)01193-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2FS0165-6147%2898%2901193-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=9703762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK1cXltFSjsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1998&pages=287-298&author=C.+Szab%C3%B3author=V.+L.+Dawson&title=Role+of+poly%28ADP-ribose%29synthetase+in+inflammation&doi=10.1016%2Fs0165-6147%2898%2901193-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Role of poly(ADP-ribose) synthetase in inflammation and ischemia-reperfusion</span></div><div class="casAuthors">Szabo, Csaba; Dawson, Valina L.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">287-298</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 111 refs.  Oxidative and nitrosative stress can trigger DNA strand breakage, which then activates the nuclear enzyme poly(ADP-ribose) synthetase (PARS).  This enzyme has also been termed poly(ADP-ribose) polymerase (PARP) or poly(ADP-ribose) transferase (pADPRT).  Rapid activation of the enzyme depletes the intracellular concn. of its substrate, NAD, thus slowing the rate of glycolysis, electron transport and subsequently ATP formation.  This process can result in cell dysfunction and cell death.  In this article, the authors overview the impact of pharmacol. inhibition or genetic inactivation of PARS on the course of oxidant-induced cell death in vitro, and in inflammation and reperfusion injury in vivo.  A major trigger for DNA damage in pathophysiol. condition is peroxynitrite, a cytotoxic oxidant formed by the reaction between the free radicals nitric oxide and superoxide.  The pharmacol. inhibition of PARS is a novel approach for the exptl. therapy of various forms of inflammation and shock, stroke, myocardial and intestinal ischemia-reperfusion, and diabetes mellitus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0C66lRIi1hrVg90H21EOLACvtfcHk0ljaEnWjjb5-rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltFSjsLo%253D&md5=18dac83ea54b17395997e772e2630252</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0165-6147%2898%2901193-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0165-6147%252898%252901193-6%26sid%3Dliteratum%253Aachs%26aulast%3DSzab%25C3%25B3%26aufirst%3DC.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26atitle%3DRole%2520of%2520poly%2528ADP-ribose%2529synthetase%2520in%2520inflammation%26jtitle%3DTrends%2520Pharm.%2520Sci.%26date%3D1998%26volume%3D19%26spage%3D287%26epage%3D298%26doi%3D10.1016%2Fs0165-6147%2898%2901193-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">D’Amours, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desnoyers, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Silva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, G. G.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribosyl)ation reactions in the regulation of nuclear functions</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>342</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">268</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1042%2Fbj3420249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10455009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADyaK1MXmtFChtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=1999&pages=249-268&author=D.+D%E2%80%99Amoursauthor=S.+Desnoyersauthor=I.+D%E2%80%99Silvaauthor=G.+G.+Poirier&title=Poly%28ADP-ribosyl%29ation+reactions+in+the+regulation+of+nuclear+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribosyl)ation reactions in the regulation of nuclear functions</span></div><div class="casAuthors">D'Amours, Damien; Desnoyers, Serge; D'Silva, Icy; Poirier, Guy G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-268</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review with 351 refs.  Poly(ADP-ribosyl)ation is a post-translational modification of proteins.  During this process, mols. of ADP-ribose are added successively on to acceptor proteins to form branched polymers.  This modification is transient but very extensive in vivo, as polymer chains can reach more than 200 units on protein acceptors.  The existence of the poly(ADP-ribose) polymer was first reported nearly 40 yr ago.  Since then, the importance of poly(ADP-ribose) synthesis has been established in many cellular processes.  However, a clear and unified picture of the physiol. role of poly(ADP-ribosyl)ation still remains to be established.  The total dependence of poly(ADP-ribose) synthesis on DNA strand breaks strongly suggests that this post-translational modification is involved in the metab. of nucleic acids.  This view is also supported by the identification of direct protein-protein interactions involving poly(ADP-ribose) polymerase (113 kDa PARP), an enzyme catalyzing the formation of poly(ADP-ribose), and key effectors of DNA repair, replication and transcription reactions.  The presence of PARP in these multiprotein complexes, in addn. to the actual poly(ADP-ribosyl)ation of some components of these complexes, clearly supports an important role for poly(ADP-ribosyl)ation reactions in DNA transactions.  Accordingly, inhibition of poly(ADP-ribose) synthesis by any of several approaches and the anal. of PARP-deficient cells has revealed that the absence of poly(ADP-ribosyl)ation strongly affects DNA metab., most notably DNA repair.  The recent identification of new poly(ADP-ribosyl)ating enzymes with distinct (non-std.) structures in eukaryotes and archaea has revealed a novel level of complexity in the regulation of poly(ADP-ribose) metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh1dXVUbI2MbVg90H21EOLACvtfcHk0ljaEnWjjb5-rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmtFChtLo%253D&md5=9a9e9c4a9c492cf6841fe146a22f21c5</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1042%2Fbj3420249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj3420249%26sid%3Dliteratum%253Aachs%26aulast%3DD%25E2%2580%2599Amours%26aufirst%3DD.%26aulast%3DDesnoyers%26aufirst%3DS.%26aulast%3DD%25E2%2580%2599Silva%26aufirst%3DI.%26aulast%3DPoirier%26aufirst%3DG.%2BG.%26atitle%3DPoly%2528ADP-ribosyl%2529ation%2520reactions%2520in%2520the%2520regulation%2520of%2520nuclear%2520functions%26jtitle%3DBiochem.%2520J.%26date%3D1999%26volume%3D342%26spage%3D249%26epage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fatokun, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, T. M.</span></span> <span> </span><span class="NLM_article-title">Parthanatos: mitochondrial-linked mechanisms and therapeutic opportunities</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">2000</span>– <span class="NLM_lpage">2016</span>, <span class="refDoi"> DOI: 10.1111/bph.12416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1111%2Fbph.12416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=24684389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFejs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2014&pages=2000-2016&author=A.+A.+Fatokunauthor=V.+L.+Dawsonauthor=T.+M.+Dawson&title=Parthanatos%3A+mitochondrial-linked+mechanisms+and+therapeutic+opportunities&doi=10.1111%2Fbph.12416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Parthanatos: mitochondrial-linked mechanisms and therapeutic opportunities</span></div><div class="casAuthors">Fatokun, Amos A.; Dawson, Valina L.; Dawson, Ted M.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2000-2016</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Cells die by a variety of mechanisms.  Terminally differentiated cells such as neurons die in a variety of disorders, in part, via parthanatos, a process dependent on the activity of poly (ADP-ribose)-polymerase (PARP).  Parthanatos does not require the mediation of caspases for its execution, but is clearly mechanistically dependent on the nuclear translocation of the mitochondrial-assocd. apoptosis-inducing factor (AIF).  The nuclear translocation of this otherwise beneficial mitochondrial protein, occasioned by poly (ADP-ribose) (PAR) produced through PARP overactivation, causes large-scale DNA fragmentation and chromatin condensation, leading to cell death.  This review describes the multistep course of parthanatos and its dependence on PAR signalling and nuclear AIF translocation.  The review also discusses potential targets in the parthanatos cascade as promising avenues for the development of novel, disease-modifying, therapeutic agents.  Linked Articles : This article is part of a themed issue on Mitochondrial Pharmacol.: Energy, Injury & Beyond.  To view the other articles in this issue visit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa2yn_v2ypBLVg90H21EOLACvtfcHk0ljaEnWjjb5-rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFejs78%253D&md5=e5a7843194a12539d988121c6d9e859c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2Fbph.12416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.12416%26sid%3Dliteratum%253Aachs%26aulast%3DFatokun%26aufirst%3DA.%2BA.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26aulast%3DDawson%26aufirst%3DT.%2BM.%26atitle%3DParthanatos%253A%2520mitochondrial-linked%2520mechanisms%2520and%2520therapeutic%2520opportunities%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2014%26volume%3D171%26spage%3D2000%26epage%3D2016%26doi%3D10.1111%2Fbph.12416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berger, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besson, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulares, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bürkle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarugi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuzzocrea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haskó, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liaudet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacher, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radermacher, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strosznajder, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sümegi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, C.</span></span> <span> </span><span class="NLM_article-title">Opportunities for the repurposing of PARP inhibitors for the therapy of non-oncological diseases</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">192</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1111/bph.13748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1111%2Fbph.13748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=28213892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvV2gt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=192-222&author=N.+A.+Bergerauthor=V.+C.+Bessonauthor=A.+H.+Boularesauthor=A.+B%C3%BCrkleauthor=A.+Chiarugiauthor=R.+S.+Clarkauthor=N.+J.+Curtinauthor=S.+Cuzzocreaauthor=T.+M.+Dawsonauthor=V.+L.+Dawsonauthor=G.+Hask%C3%B3author=L.+Liaudetauthor=F.+Moroniauthor=P.+Pacherauthor=P.+Radermacherauthor=A.+L.+Salzmanauthor=S.+H.+Snyderauthor=F.+G.+Sorianoauthor=R.+P.+Strosznajderauthor=B.+S%C3%BCmegiauthor=R.+A.+Swansonauthor=C.+Szabo&title=Opportunities+for+the+repurposing+of+PARP+inhibitors+for+the+therapy+of+non-oncological+diseases&doi=10.1111%2Fbph.13748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities for the repurposing of PARP inhibitors for the therapy of non-oncological diseases</span></div><div class="casAuthors">Berger, Nathan A.; Besson, Valerie C.; Boulares, A. Hamid; Buerkle, Alexander; Chiarugi, Alberto; Clark, Robert S.; Curtin, Nicola J.; Cuzzocrea, Salvatore; Dawson, Ted M.; Dawson, Valina L.; Hasko, Gyoergy; Liaudet, Lucas; Moroni, Flavio; Pacher, Pal; Radermacher, Peter; Salzman, Andrew L.; Snyder, Solomon H.; Soriano, Francisco Garcia; Strosznajder, Robert P.; Suemegi, Balazs; Swanson, Raymond A.; Szabo, Csaba</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">192-222</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The recent clin. availability of the PARP inhibitor olaparib (Lynparza) opens the door for potential therapeutic repurposing for non-oncol. indications.  Considering (a) the preclin. efficacy data with PARP inhibitors in non-oncol. diseases and (b) the risk-benefit ratio of treating patients with a compd. that inhibits an enzyme that has physiol. roles in the regulation of DNA repair, we have selected indications, where (a) the severity of the disease is high, (b) the available therapeutic options are limited, and (c) the duration of PARP inhibitor administration could be short, to provide first-line options for therapeutic repurposing.  These indications are as follows: acute ischemic stroke; traumatic brain injury; septic shock; acute pancreatitis; and severe asthma and severe acute lung injury.  In addn., chronic, devastating diseases, where alternative therapeutic options cannot halt disease development (e.g. Parkinson's disease, progressive multiple sclerosis or severe fibrotic diseases), should also be considered.  We present a preclin. and clin. action plan for the repurposing of PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2DK53xgBjdLVg90H21EOLACvtfcHk0lg7gMsal0ZYDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvV2gt7Y%253D&md5=213c71ccba975e61c8ff24ad8c28640f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1111%2Fbph.13748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13748%26sid%3Dliteratum%253Aachs%26aulast%3DBerger%26aufirst%3DN.%2BA.%26aulast%3DBesson%26aufirst%3DV.%2BC.%26aulast%3DBoulares%26aufirst%3DA.%2BH.%26aulast%3DB%25C3%25BCrkle%26aufirst%3DA.%26aulast%3DChiarugi%26aufirst%3DA.%26aulast%3DClark%26aufirst%3DR.%2BS.%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26aulast%3DCuzzocrea%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DT.%2BM.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26aulast%3DHask%25C3%25B3%26aufirst%3DG.%26aulast%3DLiaudet%26aufirst%3DL.%26aulast%3DMoroni%26aufirst%3DF.%26aulast%3DPacher%26aufirst%3DP.%26aulast%3DRadermacher%26aufirst%3DP.%26aulast%3DSalzman%26aufirst%3DA.%2BL.%26aulast%3DSnyder%26aufirst%3DS.%2BH.%26aulast%3DSoriano%26aufirst%3DF.%2BG.%26aulast%3DStrosznajder%26aufirst%3DR.%2BP.%26aulast%3DS%25C3%25BCmegi%26aufirst%3DB.%26aulast%3DSwanson%26aufirst%3DR.%2BA.%26aulast%3DSzabo%26aufirst%3DC.%26atitle%3DOpportunities%2520for%2520the%2520repurposing%2520of%2520PARP%2520inhibitors%2520for%2520the%2520therapy%2520of%2520non-oncological%2520diseases%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2018%26volume%3D175%26spage%3D192%26epage%3D222%26doi%3D10.1111%2Fbph.13748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lg7gMsal0ZYDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-Molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lg7gMsal0ZYDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lg7gMsal0ZYDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lgi0ia1BnT3Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.-Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0lgi0ia1BnT3Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lgi0ia1BnT3Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET) proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=L.+Liuauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+Yangauthor=D.+N.+Ziazadehauthor=F.+Xuauthor=J.+Huauthor=W.+Xiangauthor=L.+Huangauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29+proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0liyQ1Qq1m85aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=941-964&author=X.+Hanauthor=C.+Wangauthor=C.+Qinauthor=W.+Xiangauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=M.+Wangauthor=L.+Zhaoauthor=T.+Xuauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ARD-69+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+androgen+receptor+%28AR%29+for+the+treatment+of+prostate+cancer&doi=10.1021%2Facs.jmedchem.8b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span></div><div class="casAuthors">Han, Xin; Wang, Chao; Qin, Chong; Xiang, Weiguo; Fernandez-Salas, Ester; Yang, Chao-Yie; Wang, Mi; Zhao, Lijie; Xu, Tianfeng; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">941-964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the discovery of highly potent PROTAC degraders of androgen receptor (AR), as exemplified by IARD-69.  I induces degrdn. of AR protein in AR-pos. prostate cancer cell lines in a dose- and time-dependent manner.  I achieves DC50 values of 0.86, 0.76, and 10.4 nM in LNCaP, VCaP, and 22Rv1 AR+ prostate cancer cell lines, resp.  I is capable of reducing the AR protein level by >95% in these prostate cancer cell lines and effectively suppressing AR-regulated gene expression.  I potently inhibits cell growth in these AR-pos. prostate cancer cell lines and is >100 times more potent than AR antagonists.  A single dose of I effectively reduces the level of AR protein in xenograft tumor tissue in mice.  Further optimization of I may ultimately lead to a new therapy for AR+, castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTlOD69tfhtbVg90H21EOLACvtfcHk0liyQ1Qq1m85aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D&md5=2376573fb1a1e19c517d64c70199700a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01631%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ARD-69%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520androgen%2520receptor%2520%2528AR%2529%2520for%2520the%2520treatment%2520of%2520prostate%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D941%26epage%3D964%26doi%3D10.1021%2Facs.jmedchem.8b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.-Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+estrogen+receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0lj5o-UMUEo7aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520estrogen%2520receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.-R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lj5o-UMUEo7aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.-R.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukhali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnchembio.2538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=29251720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=163-170&author=C.+M.+Olsonauthor=B.+Jiangauthor=M.+A.+Erbauthor=Y.+Liangauthor=Z.+M.+Doctorauthor=Z.+Zhangauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=M.+Boukhaliauthor=J.+L.+Greenauthor=W.+Haasauthor=T.+Nomanbhoyauthor=E.+S.+Fischerauthor=R.+A.+Youngauthor=J.+E.+Bradnerauthor=G.+E.+Winterauthor=N.+S.+Gray&title=Pharmacological+perturbation+of+CDK9+using+selective+CDK9+inhibition+or+degradation&doi=10.1038%2Fnchembio.2538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span></div><div class="casAuthors">Olson, Calla M.; Jiang, Baishan; Erb, Michael A.; Liang, Yanke; Doctor, Zainab M.; Zhang, Zinan; Zhang, Tinghu; Kwiatkowski, Nicholas; Boukhali, Myriam; Green, Jennifer L.; Haas, Wilhelm; Nomanbhoy, Tyzoon; Fischer, Eric S.; Young, Richard A.; Bradner, James E.; Winter, Georg E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">163-170</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), an important regulator of transcriptional elongation, is a promising target for cancer therapy, particularly for cancers driven by transcriptional dysregulation.  The authors characterized NVP-2, a selective ATP-competitive CDK9 inhibitor, and THAL-SNS-032, a selective CDK9 degrader consisting of a CDK-binding SNS-032 ligand linked to a thalidomide deriv. that binds the E3 ubiquitin ligase Cereblon (CRBN).  To the authors' surprise, THAL-SNS-032 induced rapid degrdn. of CDK9 without affecting the levels of other SNS-032 targets.  Moreover, the transcriptional changes elicited by THAL-SNS-032 were more like those caused by NVP-2 than those induced by SNS-032.  Notably, compd. washout did not significantly reduce levels of THAL-SNS-032-induced apoptosis, suggesting that CDK9 degrdn. had prolonged cytotoxic effects compared with CDK9 inhibition.  Thus, the authors' findings suggest that thalidomide conjugation represents a promising strategy for converting multi-targeted inhibitors into selective degraders and reveal that kinase degrdn. can induce distinct pharmacol. effects compared with inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWnT9r0GIlmbVg90H21EOLACvtfcHk0lj5o-UMUEo7aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O&md5=88aabb209b4cb7516a652d6c75a35667</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2538%26sid%3Dliteratum%253Aachs%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DBoukhali%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DJ.%2BL.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DNomanbhoy%26aufirst%3DT.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPharmacological%2520perturbation%2520of%2520CDK9%2520using%2520selective%2520CDK9%2520inhibition%2520or%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D163%26epage%3D170%26doi%3D10.1038%2Fnchembio.2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robb, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Contreras, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonawane, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murry, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">7577</span>– <span class="NLM_lpage">7580</span>, <span class="refDoi"> DOI: 10.1039/c7cc03879h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1039%2FC7CC03879H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=28636052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=7577-7580&author=C.+M.+Robbauthor=J.+I.+Contrerasauthor=S.+Kourauthor=M.+A.+Taylorauthor=M.+Abidauthor=Y.+A.+Sonawaneauthor=M.+Zahidauthor=D.+J.+Murryauthor=A.+Natarajanauthor=S.+Rana&title=Chemically+induced+degradation+of+CDK9+by+a+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1039%2Fc7cc03879h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Robb, Caroline M.; Contreras, Jacob I.; Kour, Smit; Taylor, Margaret A.; Abid, Mohammad; Sonawane, Yogesh A.; Zahid, Muhammad; Murry, Daryl J.; Natarajan, Amarnath; Rana, Sandeep</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">54</span>),
    <span class="NLM_cas:pages">7577-7580</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), a member of the cyclin-dependent protein kinase (CDK) family, is involved in transcriptional elongation of several target genes.  CDK9 is ubiquitously expressed and has been shown to contribute to a variety of malignancies such as pancreatic, prostate and breast cancers.  Here we report the development of a heterobifunctional small mol. proteolysis targeting chimera (PROTAC) capable of cereblon (CRBN)-mediated proteasomal degrdn. of CDK9.  In HCT116 cells, it selectively degrades CDK9 while sparing other CDK family members.  This is the first example of a PROTAC that selectively degrades CDK9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXgLtOTwvlnbVg90H21EOLACvtfcHk0lhJ4gzDA-stfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D&md5=a9aa8ebf8109f11206740efd1b5e09a2</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1039%2FC7CC03879H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC03879H%26sid%3Dliteratum%253Aachs%26aulast%3DRobb%26aufirst%3DC.%2BM.%26aulast%3DContreras%26aufirst%3DJ.%2BI.%26aulast%3DKour%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DM.%2BA.%26aulast%3DAbid%26aufirst%3DM.%26aulast%3DSonawane%26aufirst%3DY.%2BA.%26aulast%3DZahid%26aufirst%3DM.%26aulast%3DMurry%26aufirst%3DD.%2BJ.%26aulast%3DNatarajan%26aufirst%3DA.%26aulast%3DRana%26aufirst%3DS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520CDK9%2520by%2520a%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D7577%26epage%3D7580%26doi%3D10.1039%2Fc7cc03879h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Discovery of Wogonin-based PROTACs against CDK9 and capable of achieving antitumor activity</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">381</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.08.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.bioorg.2018.08.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=30196207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Kkt7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=373-381&author=J.+Bianauthor=J.+Renauthor=Y.+Liauthor=J.+Wangauthor=X.+Xuauthor=Y.+Fengauthor=H.+Tangauthor=Y.+Wangauthor=Z.+Li&title=Discovery+of+Wogonin-based+PROTACs+against+CDK9+and+capable+of+achieving+antitumor+activity&doi=10.1016%2Fj.bioorg.2018.08.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Wogonin-based PROTACs against CDK9 and capable of achieving antitumor activity</span></div><div class="casAuthors">Bian, Jinlei; Ren, Jie; Li, Yongren; Wang, Jubo; Xu, Xi; Feng, Yifan; Tang, Hui; Wang, Yajing; Li, Zhiyu</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">373-381</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Wogonin is a natural product isolated from the Scutellaria baicalensis and has been proved to be a potent and selective inhibitor of CDK9.  Using this scaffold, we designed and synthesized a series of proteolysis targeting chimeras (PROTACs) targeting CDK9 by recruiting ubiquitin E3 ligase cereblon (CRBN).  For constructing diverse Wogonin-based PROTACs, a "click chem." approach was employed for the synthesis of CDK9-targeting PROTACs.  The results of western blotting assays showed that compds. contg. triazole group in the linker could selectively downregulate the intracellular CDK9 level.  Among these compds., 11c could selectively degrade CDK9 in a concn.-dependent manner.  In addn., the application of the proteasome inhibitor MG132 and CRBN siRNA silencing confirmed that 11c could promote the proteasome-dependent and CRBN-dependent degrdn.  Consistent with the degrdn. of the CDK9 protein, 11c selectively inhibits proliferation of CDK9-overexpressed cancer cells.  Thus, our Wogonin-based PROTAC would be an efficient probe that induces the degrdn. of CDK9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod4sLAWq5vMrVg90H21EOLACvtfcHk0lhJ4gzDA-stfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Kkt7bE&md5=6a7158be1fce50296d6d0ec9160db0fc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.08.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.08.028%26sid%3Dliteratum%253Aachs%26aulast%3DBian%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520Wogonin-based%2520PROTACs%2520against%2520CDK9%2520and%2520capable%2520of%2520achieving%2520antitumor%2520activity%26jtitle%3DBioorg.%2520Chem.%26date%3D2018%26volume%3D81%26spage%3D373%26epage%3D381%26doi%3D10.1016%2Fj.bioorg.2018.08.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">Induction of apoptosis in MDA-MB-231 breast cancer cells by a PARP1-targeting PROTAC small molecule</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1039/c8cc07813k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1039%2FC8CC07813K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=30540295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVemsrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2019&pages=369-372&author=Q.+Zhaoauthor=T.+Lanauthor=S.+Suauthor=Y.+Rao&title=Induction+of+apoptosis+in+MDA-MB-231+breast+cancer+cells+by+a+PARP1-targeting+PROTAC+small+molecule&doi=10.1039%2Fc8cc07813k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis in MDA-MB-231 breast cancer cells by a PARP1-targeting PROTAC small molecule</span></div><div class="casAuthors">Zhao, Qiuye; Lan, Tianlong; Su, Shang; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">369-372</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Poly (ADP-ribose) polymerase-1 (PARP1) is a major member of the PARP superfamily that is involved in DNA damage signalling and other important cellular processes.  Here we report the development of a small mol. targeting PARP1 based on the PROTAC strategy.  In the MDA-MB-231 cell line, the representative compd. 3 can induce significant PARP1 cleavage and programmed cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJMa7i0ZDIH7Vg90H21EOLACvtfcHk0lhJ4gzDA-stfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVemsrfN&md5=4bf5ac2b63635c348b415d5015c8c0c8</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1039%2FC8CC07813K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8CC07813K%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DLan%26aufirst%3DT.%26aulast%3DSu%26aufirst%3DS.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DInduction%2520of%2520apoptosis%2520in%2520MDA-MB-231%2520breast%2520cancer%2520cells%2520by%2520a%2520PARP1-targeting%2520PROTAC%2520small%2520molecule%26jtitle%3DChem.%2520Commun.%26date%3D2019%26volume%3D55%26spage%3D369%26epage%3D372%26doi%3D10.1039%2Fc8cc07813k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span> <span> </span><span class="NLM_article-title">Uncoupling of PARP1 trapping and inhibition using selective PARP1 degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1223</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0379-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fs41589-019-0379-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=31659317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVCksLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=1223-1231&author=S.+Wangauthor=L.+Hanauthor=J.+Hanauthor=P.+Liauthor=Q.+Dingauthor=Q.-J.+Zhangauthor=Z.-P.+Liuauthor=C.+Chenauthor=Y.+Yu&title=Uncoupling+of+PARP1+trapping+and+inhibition+using+selective+PARP1+degradation&doi=10.1038%2Fs41589-019-0379-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Uncoupling of PARP1 trapping and inhibition using selective PARP1 degradation</span></div><div class="casAuthors">Wang, Shuai; Han, Lei; Han, Jungsoo; Li, Peng; Ding, Qing; Zhang, Qing-Jun; Liu, Zhi-Ping; Chen, Chuo; Yu, Yonghao</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1223-1231</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">PARP1 inhibitors (PARPi) are known to kill tumor cells via two mechanisms (PARP1 catalytic inhibition and PARP1 trapping).  The relative contribution of these two pathways in mediating the cytotoxicity of PARPi, however, is not well understood.  Here we designed a series of small mol. PARP degraders.  Treatment with one such compd. iRucaparib-AP6 results in highly efficient and specific PARP1 degrdn. iRucaparib-AP6 blocks the enzymic activity of PARP1 in vitro, and PARP1-mediated poly-ADP-ribosylation signaling in intact cells.  This strategy mimics PARP1 genetic depletion, which enables the pharmacol. decoupling of PARP1 inhibition from PARP1 trapping.  Finally, by depleting PARP1, iRucaparib-AP6 protects muscle cells and primary cardiomyocytes from DNA-damage-induced energy crisis and cell death.  In summary, these compds. represent 'non-trapping' PARP1 degraders that block both the catalytic activity and scaffolding effects of PARP1, providing an ideal approach for the amelioration of the various pathol. conditions caused by PARP1 hyperactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfhC10Bxxzm7Vg90H21EOLACvtfcHk0lioCu-IESOCYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVCksLjF&md5=2f53f8fbbb41cf55b94ddb70a652df55</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0379-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0379-2%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DQ.-J.%26aulast%3DLiu%26aufirst%3DZ.-P.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DY.%26atitle%3DUncoupling%2520of%2520PARP1%2520trapping%2520and%2520inhibition%2520using%2520selective%2520PARP1%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D1223%26epage%3D1231%26doi%3D10.1038%2Fs41589-019-0379-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawicki-McKenna, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langelier, M.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeNizio, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccio, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karch, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCauley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffen, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascal, J. M.</span></span> <span> </span><span class="NLM_article-title">PARP-1 activation requires local unfolding of an autoinhibitory domain</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">768</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1016%2Fj.molcel.2015.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=26626480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFagurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2015&pages=755-768&author=J.+M.+Dawicki-McKennaauthor=M.-F.+Langelierauthor=J.+E.+DeNizioauthor=A.+A.+Riccioauthor=C.+D.+Caoauthor=K.+R.+Karchauthor=M.+McCauleyauthor=J.+D.+Steffenauthor=B.+E.+Blackauthor=J.+M.+Pascal&title=PARP-1+activation+requires+local+unfolding+of+an+autoinhibitory+domain&doi=10.1016%2Fj.molcel.2015.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">PARP-1 Activation Requires Local Unfolding of an Autoinhibitory Domain</span></div><div class="casAuthors">Dawicki-McKenna, Jennine M.; Langelier, Marie-France; DeNizio, Jamie E.; Riccio, Amanda A.; Cao, Connie D.; Karch, Kelly R.; McCauley, Michael; Steffen, Jamin D.; Black, Ben E.; Pascal, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">755-768</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase-1 (PARP-1) creates the posttranslational modification PAR from substrate NAD+ to regulate multiple cellular processes.  DNA breaks sharply elevate PARP-1 catalytic activity to mount a cell survival repair response, whereas persistent PARP-1 hyperactivation during severe genotoxic stress is assocd. with cell death.  The mechanism for tight control of the robust catalytic potential of PARP-1 remains unclear.  By monitoring PARP-1 dynamics using hydrogen/deuterium exchange-mass spectrometry (HXMS), we unexpectedly find that a specific portion of the helical subdomain (HD) of the catalytic domain rapidly unfolds when PARP-1 encounters a DNA break.  Together with biochem. and crystallog. anal. of HD deletion mutants, we show that the HD is an autoinhibitory domain that blocks productive NAD+ binding.  Our mol. model explains how PARP-1 DNA damage detection leads to local unfolding of the HD that relieves autoinhibition, and has important implications for the design of PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJje_1KBikFbVg90H21EOLACvtfcHk0lioCu-IESOCYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFagurvK&md5=cf24c86d7d942be40929b8aee2d797c7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DDawicki-McKenna%26aufirst%3DJ.%2BM.%26aulast%3DLangelier%26aufirst%3DM.-F.%26aulast%3DDeNizio%26aufirst%3DJ.%2BE.%26aulast%3DRiccio%26aufirst%3DA.%2BA.%26aulast%3DCao%26aufirst%3DC.%2BD.%26aulast%3DKarch%26aufirst%3DK.%2BR.%26aulast%3DMcCauley%26aufirst%3DM.%26aulast%3DSteffen%26aufirst%3DJ.%2BD.%26aulast%3DBlack%26aufirst%3DB.%2BE.%26aulast%3DPascal%26aufirst%3DJ.%2BM.%26atitle%3DPARP-1%2520activation%2520requires%2520local%2520unfolding%2520of%2520an%2520autoinhibitory%2520domain%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D60%26spage%3D755%26epage%3D768%26doi%3D10.1016%2Fj.molcel.2015.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.-H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lioCu-IESOCYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of MD-224 as a first-in-class, highly potent, and efficacious proteolysis targeting chimera murine double minute 2 degrader capable of achieving complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00909</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00909" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2018&pages=448-466&author=Y.+Liauthor=J.+Yangauthor=A.+Aguilarauthor=D.+McEachernauthor=S.+Przybranowskiauthor=L.+Liuauthor=C.-Y.+Yangauthor=M.+Wangauthor=X.+Hanauthor=S.+Wang&title=Discovery+of+MD-224+as+a+first-in-class%2C+highly+potent%2C+and+efficacious+proteolysis+targeting+chimera+murine+double+minute+2+degrader+capable+of+achieving+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00909"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00909%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520MD-224%2520as%2520a%2520first-in-class%252C%2520highly%2520potent%252C%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520murine%2520double%2520minute%25202%2520degrader%2520capable%2520of%2520achieving%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D62%26spage%3D448%26epage%3D466%26doi%3D10.1021%2Facs.jmedchem.8b00909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tinworth, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lithgow, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Small molecule-mediated protein knockdown as a new approach to drug discovery</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2206</span>– <span class="NLM_lpage">2216</span>, <span class="refDoi"> DOI: 10.1039/c6md00347h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1039%2FC6MD00347H" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2206-2216&author=C.+P.+Tinworthauthor=H.+Lithgowauthor=I.+Churcher&title=Small+molecule-mediated+protein+knockdown+as+a+new+approach+to+drug+discovery&doi=10.1039%2Fc6md00347h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1039%2FC6MD00347H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00347H%26sid%3Dliteratum%253Aachs%26aulast%3DTinworth%26aufirst%3DC.%2BP.%26aulast%3DLithgow%26aufirst%3DH.%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DSmall%2520molecule-mediated%2520protein%2520knockdown%2520as%2520a%2520new%2520approach%2520to%2520drug%2520discovery%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D2206%26epage%3D2216%26doi%3D10.1039%2Fc6md00347h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lhaSEZY4nrXMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, J.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huan, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.-H.</span></span> <span> </span><span class="NLM_article-title">Novel PARP1/2 inhibitor mefuparib hydrochloride elicits potent in vitro and in vivo anticancer activity, characteristic of high tissue distribution</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4156</span>– <span class="NLM_lpage">4168</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.18632%2Foncotarget.13749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=27926532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A280%3ADC%252BC2sjksVSltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=4156-4168&author=J.-X.+Heauthor=M.+Wangauthor=X.-J.+Huanauthor=C.-H.+Chenauthor=S.-S.+Songauthor=Y.-Q.+Wangauthor=X.-M.+Liaoauthor=C.+Tanauthor=Q.+Heauthor=L.-J.+Tongauthor=Y.-T.+Wangauthor=X.-H.+Liauthor=Y.+Suauthor=Y.-Y.+Shenauthor=Y.-M.+Sunauthor=X.-Y.+Yangauthor=Y.+Chenauthor=Z.-W.+Gaoauthor=X.-Y.+Chenauthor=B.+Xiongauthor=X.-L.+Luauthor=J.+Dingauthor=C.-H.+Yangauthor=Z.-H.+Miao&title=Novel+PARP1%2F2+inhibitor+mefuparib+hydrochloride+elicits+potent+in+vitro+and+in+vivo+anticancer+activity%2C+characteristic+of+high+tissue+distribution&doi=10.18632%2Foncotarget.13749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Novel PARP1/2 inhibitor mefuparib hydrochloride elicits potent in vitro and in vivo anticancer activity, characteristic of high tissue distribution</span></div><div class="casAuthors">He Jin-Xue; Huan Xia-Juan; Chen Chuan-Huizi; Song Shan-Shan; Wang Ying-Qing; Liao Xue-Mei; Tong Lin-Jiang; Wang Yu-Ting; Li Xiao-Hua; Su Yi; Shen Yan-Yan; Sun Yi-Ming; Yang Xin-Ying; Chen Yi; Ding Jian; Miao Ze-Hong; Wang Meng; Tan Cun; He Qian; Xiong Bing; Yang Chun-Hao; Gao Zhi-Wei; Chen Xiao-Yan; Lu Xiu-Lian</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">4156-4168</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The approval of poly(ADP-ribose) polymerase (PARP) inhibitor AZD2281 in 2014 marked the successful establishment of the therapeutic strategy targeting homologous recombination repair defects of cancers in the clinic.  However, AZD2281 has poor water solubility, low tissue distribution and relatively weak in vivo anticancer activity, which appears to become limiting factors for its clinical use.  In this study, we found that mefuparib hydrochloride (MPH) was a potent PARP inhibitor, possessing prominent in vitro and in vivo anticancer activity.  Notably, MPH displayed high water solubility (> 35 mg/ml) and potent PARP1/2 inhibition in a substrate-competitive manner.  It reduced poly(ADP-ribose) (PAR) formation, enhanced γH2AX levels, induced G2/M arrest and subsequent apoptosis in homologous recombination repair (HR)-deficient cells.  Proof-of-concept studies confirmed the MPH-caused synthetic lethality.  MPH showed potent in vitro and in vivo proliferation and growth inhibition against HR-deficient cancer cells and synergistic sensitization of HR-proficient xenografts to the anticancer drug temozolomide.  A good relationship between the anticancer activity and the PARP inhibition of MPH suggested that PAR formation and γH2AX accumulation could serve as its pharmacodynamic biomarkers.  Its high bioavailability (40%~100%) and high tissue distribution in both monkeys and rats were its most important pharmacokinetic features.  Its average concentrations were 33-fold higher in the tissues than in the plasma in rats.  Our work supports the further clinical development of MPH as a novel PARP1/2 inhibitor for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFLMaYGUFFID69zv1FsJFMfW6udTcc2eYhcwYPgjBy1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjksVSltQ%253D%253D&md5=bf548fd046eb6c1330597a0a75a432ea</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13749%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DJ.-X.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHuan%26aufirst%3DX.-J.%26aulast%3DChen%26aufirst%3DC.-H.%26aulast%3DSong%26aufirst%3DS.-S.%26aulast%3DWang%26aufirst%3DY.-Q.%26aulast%3DLiao%26aufirst%3DX.-M.%26aulast%3DTan%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DQ.%26aulast%3DTong%26aufirst%3DL.-J.%26aulast%3DWang%26aufirst%3DY.-T.%26aulast%3DLi%26aufirst%3DX.-H.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DY.-Y.%26aulast%3DSun%26aufirst%3DY.-M.%26aulast%3DYang%26aufirst%3DX.-Y.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DZ.-W.%26aulast%3DChen%26aufirst%3DX.-Y.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DX.-L.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.-H.%26aulast%3DMiao%26aufirst%3DZ.-H.%26atitle%3DNovel%2520PARP1%252F2%2520inhibitor%2520mefuparib%2520hydrochloride%2520elicits%2520potent%2520in%2520vitro%2520and%2520in%2520vivo%2520anticancer%2520activity%252C%2520characteristic%2520of%2520high%2520tissue%2520distribution%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D4156%26epage%3D4168%26doi%3D10.18632%2Foncotarget.13749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Redon, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonner, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinders, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parchment, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">Histone γH2AX and poly(ADP-ribose) as clinical pharmacodynamic biomarkers</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4532</span>– <span class="NLM_lpage">4542</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-10-0523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=10.1158%2F1078-0432.CCR-10-0523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=20823146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGqs7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=4532-4542&author=C.+E.+Redonauthor=A.+J.+Nakamuraauthor=Y.-W.+Zhangauthor=J.+Jiauthor=W.+M.+Bonnerauthor=R.+J.+Kindersauthor=R.+E.+Parchmentauthor=J.+H.+Doroshowauthor=Y.+Pommier&title=Histone+%CE%B3H2AX+and+poly%28ADP-ribose%29+as+clinical+pharmacodynamic+biomarkers&doi=10.1158%2F1078-0432.ccr-10-0523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Histone γH2AX and Poly(ADP-Ribose) as Clinical Pharmacodynamic Biomarkers</span></div><div class="casAuthors">Redon, Christophe E.; Nakamura, Asako J.; Zhang, Yong-Wei; Ji, Jiuping; Bonner, William M.; Kinders, Robert J.; Parchment, Ralph E.; Doroshow, James H.; Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4532-4542</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Tumor cells are often deficient in DNA damage response (DDR) pathways, and anticancer therapies are commonly based on genotoxic treatments using radiation and/or drugs that damage DNA directly or interfere with DNA metab., leading to the formation of DNA double-strand breaks (DSB), and ultimately to cell death.  Because DSBs induce the phosphorylation of histone H2AX (γH2AX) in the chromatin flanking the break site, an antibody directed against γH2AX can be employed to measure DNA damage levels before and after patient treatment.  Poly(ADP-ribose) polymerases (PARP1 and PARP2) are also activated by DNA damage, and PARP inhibitors show promising activity in cancers with defective homologous recombination (HR) pathways for DSB repair.  Ongoing clin. trials are testing combinations of PARP inhibitors with DNA damaging agents.  Poly(ADP-ribosylation), abbreviated as PAR, can be measured in clin. samples and used to det. the efficiency of PARP inhibitors.  This review summarizes the roles of γH2AX and PAR in the DDR, and their use as biomarkers to monitor drug response and guide clin. trials, esp. phase 0 clin. trials.  We also discuss the choices of relevant samples for γH2AX and PAR analyses.  Clin Cancer Res; 16(18); 4532-42.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocLdGwTAJVALVg90H21EOLACvtfcHk0lgb18GvJXM4eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGqs7bI&md5=4a2bc5d90d93f3de000ccb3f8ffd9c74</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-10-0523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-10-0523%26sid%3Dliteratum%253Aachs%26aulast%3DRedon%26aufirst%3DC.%2BE.%26aulast%3DNakamura%26aufirst%3DA.%2BJ.%26aulast%3DZhang%26aufirst%3DY.-W.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DBonner%26aufirst%3DW.%2BM.%26aulast%3DKinders%26aufirst%3DR.%2BJ.%26aulast%3DParchment%26aufirst%3DR.%2BE.%26aulast%3DDoroshow%26aufirst%3DJ.%2BH.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DHistone%2520%25CE%25B3H2AX%2520and%2520poly%2528ADP-ribose%2529%2520as%2520clinical%2520pharmacodynamic%2520biomarkers%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2010%26volume%3D16%26spage%3D4532%26epage%3D4542%26doi%3D10.1158%2F1078-0432.ccr-10-0523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papeo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posteri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borghi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busel, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caprera, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciomei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Anello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fasolini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isacchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khvat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krasavin, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orsini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perego, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pesenti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pezzetta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rainoldi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccardi-Sirtori, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolaro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sola, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuccotto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donati, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montagnoli, A.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2-[1-(4,4-difluorocyclohexyl)piperidin-4-yl]-6-fluoro-3-oxo-2,3-dihydro-1<i>H</i>-isoind ole-4-carboxamide (NMS-P118): a potent, orally available, and highly selective PARP-1 inhibitor for cancer therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6875</span>– <span class="NLM_lpage">6898</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00680</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00680" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1CitbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6875-6898&author=G.+Papeoauthor=H.+Posteriauthor=D.+Borghiauthor=A.+A.+Buselauthor=F.+Capreraauthor=E.+Casaleauthor=M.+Ciomeiauthor=A.+Cirlaauthor=E.+Cortiauthor=M.+D%E2%80%99Anelloauthor=M.+Fasoliniauthor=B.+Forteauthor=A.+Galvaniauthor=A.+Isacchiauthor=A.+Khvatauthor=M.+Y.+Krasavinauthor=R.+Lupiauthor=P.+Orsiniauthor=R.+Peregoauthor=E.+Pesentiauthor=D.+Pezzettaauthor=S.+Rainoldiauthor=F.+Riccardi-Sirtoriauthor=A.+Scolaroauthor=F.+Solaauthor=F.+Zuccottoauthor=E.+R.+Felderauthor=D.+Donatiauthor=A.+Montagnoli&title=Discovery+of+2-%5B1-%284%2C4-difluorocyclohexyl%29piperidin-4-yl%5D-6-fluoro-3-oxo-2%2C3-dihydro-1H-isoind+ole-4-carboxamide+%28NMS-P118%29%3A+a+potent%2C+orally+available%2C+and+highly+selective+PARP-1+inhibitor+for+cancer+therapy&doi=10.1021%2Facs.jmedchem.5b00680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-[1-(4,4-Difluorocyclohexyl)piperidin-4-yl]-6-fluoro-3-oxo-2,3-dihydro-1H-isoindole-4-carboxamide (NMS-P118): A Potent, Orally Available, and Highly Selective PARP-1 Inhibitor for Cancer Therapy</span></div><div class="casAuthors">Papeo, Gianluca; Posteri, Helena; Borghi, Daniela; Busel, Alina A.; Caprera, Francesco; Casale, Elena; Ciomei, Marina; Cirla, Alessandra; Corti, Emiliana; D'Anello, Matteo; Fasolini, Marina; Forte, Barbara; Galvani, Arturo; Isacchi, Antonella; Khvat, Alexander; Krasavin, Mikhail Y.; Lupi, Rosita; Orsini, Paolo; Perego, Rita; Pesenti, Enrico; Pezzetta, Daniele; Rainoldi, Sonia; Riccardi-Sirtori, Federico; Scolaro, Alessandra; Sola, Francesco; Zuccotto, Fabio; Felder, Eduard R.; Donati, Daniele; Montagnoli, Alessia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6875-6898</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The nuclear protein poly(ADP-ribose) polymerase-1 (PARP-1) has a well-established role in the signaling and repair of DNA and is a prominent target in oncol., as testified by the no. of candidates in clin. testing that unselectively target both PARP-1 and its closest isoform PARP-2.  The goal of the program was to find a PARP-1 selective inhibitor that would potentially mitigate toxicities arising from cross-inhibition of PARP-2.  Thus, an HTS campaign on the proprietary Nerviano Medical Sciences (NMS) chem. collection, followed by SAR optimization, allowed us to discover I.  NMS-P118 proved to be a potent, orally available, and highly selective PARP-1 inhibitor endowed with excellent ADME and pharmacokinetic profiles and high efficacy in vivo both as a single agent and in combination with Temozolomide in MDA-MB-436 and Capan-1 xenograft models, resp.  Cocrystal structures of I with both PARP-1 and PARP-2 catalytic domain proteins allowed rationalization of the obsd. selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM_i5haE561bVg90H21EOLACvtfcHk0lgb18GvJXM4eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1CitbzO&md5=600a505d975aa18639885b179170e684</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00680%26sid%3Dliteratum%253Aachs%26aulast%3DPapeo%26aufirst%3DG.%26aulast%3DPosteri%26aufirst%3DH.%26aulast%3DBorghi%26aufirst%3DD.%26aulast%3DBusel%26aufirst%3DA.%2BA.%26aulast%3DCaprera%26aufirst%3DF.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DCiomei%26aufirst%3DM.%26aulast%3DCirla%26aufirst%3DA.%26aulast%3DCorti%26aufirst%3DE.%26aulast%3DD%25E2%2580%2599Anello%26aufirst%3DM.%26aulast%3DFasolini%26aufirst%3DM.%26aulast%3DForte%26aufirst%3DB.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DIsacchi%26aufirst%3DA.%26aulast%3DKhvat%26aufirst%3DA.%26aulast%3DKrasavin%26aufirst%3DM.%2BY.%26aulast%3DLupi%26aufirst%3DR.%26aulast%3DOrsini%26aufirst%3DP.%26aulast%3DPerego%26aufirst%3DR.%26aulast%3DPesenti%26aufirst%3DE.%26aulast%3DPezzetta%26aufirst%3DD.%26aulast%3DRainoldi%26aufirst%3DS.%26aulast%3DRiccardi-Sirtori%26aufirst%3DF.%26aulast%3DScolaro%26aufirst%3DA.%26aulast%3DSola%26aufirst%3DF.%26aulast%3DZuccotto%26aufirst%3DF.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DMontagnoli%26aufirst%3DA.%26atitle%3DDiscovery%2520of%25202-%255B1-%25284%252C4-difluorocyclohexyl%2529piperidin-4-yl%255D-6-fluoro-3-oxo-2%252C3-dihydro-1H-isoind%2520ole-4-carboxamide%2520%2528NMS-P118%2529%253A%2520a%2520potent%252C%2520orally%2520available%252C%2520and%2520highly%2520selective%2520PARP-1%2520inhibitor%2520for%2520cancer%2520therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6875%26epage%3D6898%26doi%3D10.1021%2Facs.jmedchem.5b00680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DS3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DS3','PDB','5DS3'); return false;">PDB: 5DS3</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i65"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00821">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_21674"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00821?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00821</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Chemical structure of the five previously reported PARP1/2 PAOTAC; western blotting analysis of PARP1 and PARP2 in cancer cell lines treated with indicated compounds; cell growth inhibition activity of SK-575 and olaparib in cancer cell lines and normal cell lines; body weight changes in mice; <sup>1</sup>H NMR spectrum for SK-575; and HPLC results for SK-575 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Recommended compound characterization checklist (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_002.xls">XLS</a>)</p></li><li><p class="inline">Molecular formula strings and IC<sub>50</sub> and DC<sub>50</sub> data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_001.pdf">jm0c00821_si_001.pdf (1.57 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_002.xls">jm0c00821_si_002.xls (148.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00821/suppl_file/jm0c00821_si_003.csv">jm0c00821_si_003.csv (5.29 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00821&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-19%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00821%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00821" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67994aa10d581975","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
