
<html lang="en"     class="pb-page"  data-request-id="ac46735c-ac59-47d6-b1fa-d43a87ff1523"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm400045y;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2013.56.issue-6;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="A New Target for an Old Drug: Identifying Mitoxantrone as a Nanomolar Inhibitor of PIM1 Kinase via Kinome-Wide Selectivity Modeling" /></meta><meta name="dc.Creator" content="Xiaobo  Wan" /></meta><meta name="dc.Creator" content="Wei  Zhang" /></meta><meta name="dc.Creator" content="Li  Li" /></meta><meta name="dc.Creator" content="Yuting  Xie" /></meta><meta name="dc.Creator" content="Wei  Li" /></meta><meta name="dc.Creator" content="Niu  Huang" /></meta><meta name="dc.Description" content="The rational design of selective kinase inhibitors remains a great challenge. Here we describe a physics-based approach to computationally modeling the kinase inhibitor selectivity profile. We retr..." /></meta><meta name="Description" content="The rational design of selective kinase inhibitors remains a great challenge. Here we describe a physics-based approach to computationally modeling the kinase inhibitor selectivity profile. We retr..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 15, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm400045y" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm400045y" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm400045y" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm400045y" /></link>
        
    
    

<title>A New Target for an Old Drug: Identifying Mitoxantrone as a Nanomolar Inhibitor of PIM1 Kinase via Kinome-Wide Selectivity Modeling | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm400045y" /></meta><meta property="og:title" content="A New Target for an Old Drug: Identifying Mitoxantrone as a Nanomolar Inhibitor of PIM1 Kinase via Kinome-Wide Selectivity Modeling" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0008.jpeg" /></meta><meta property="og:description" content="The rational design of selective kinase inhibitors remains a great challenge. Here we describe a physics-based approach to computationally modeling the kinase inhibitor selectivity profile. We retrospectively assessed this protocol by computing the binding profiles of 17 well-known kinase inhibitors against 143 kinases. Next, we predicted the binding profile of the chemotherapy drug mitoxantrone, and chose the predicted top five kinase targets for in vitro kinase assays. Remarkably, mitoxantrone was shown to possess low nanomolar inhibitory activity against PIM1 kinase and to inhibit the PIM1-mediated phosphorylation in cancer cells. We further determined the crystal complex structure of PIM1 bound with mitoxantrone, which reveals the structural and mechanistic basis for a novel mode of PIM1 inhibition. Although mitoxantrone’s mechanism of action had been originally thought to act through DNA intercalation and type II topoisomerase inhibition, we hypothesize that PIM1 kinase inhibition might also contribute to mitoxantrone’s therapeutic efficacy and specificity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm400045y"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm400045y">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm400045y&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm400045y&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm400045y&amp;href=/doi/10.1021/jm400045y" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 2619-2629</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm400043d" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm400058j" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A New Target for an Old Drug: Identifying Mitoxantrone as a Nanomolar Inhibitor of PIM1 Kinase via Kinome-Wide Selectivity Modeling</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaobo++Wan">Xiaobo Wan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Zhang">Wei Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Li++Li">Li Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yuting++Xie">Yuting Xie</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Li">Wei Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Niu++Huang">Niu Huang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Graduate School of Peking Union Medical College and Chinese Academy of Medical Sciences, Beijing, China</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">National Institute of Biological Sciences, Beijing, No. 7 Science Park Road, Zhongguancun Life Science Park, Changping District, Beijing 102206, China</span></div><div class="corresp-info"><strong>*</strong>Phone: 86-10-80720645. Fax: 86-10-80720813. E-mail: <a href="/cdn-cgi/l/email-protection#452d30242b222b2c30052b2c27366b24266b262b"><span class="__cf_email__" data-cfemail="b6dec3d7d8d1d8dfc3f6d8dfd4c598d7d598d5d8">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm400045y&amp;href=/doi/10.1021%2Fjm400045y" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 2619–2629</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 26, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 January 2013</li><li><span class="item_label"><b>Published</b> online</span>15 March 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 March 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm400045y" title="DOI URL">https://doi.org/10.1021/jm400045y</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2619%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DXiaobo%2BWan%252C%2BWei%2BZhang%252C%2BLi%2BLi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D6%26contentID%3Djm400045y%26title%3DA%2BNew%2BTarget%2Bfor%2Ban%2BOld%2BDrug%253A%2BIdentifying%2BMitoxantrone%2Bas%2Ba%2BNanomolar%2BInhibitor%2Bof%2BPIM1%2BKinase%2Bvia%2BKinome-Wide%2BSelectivity%2BModeling%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2629%26publicationDate%3DMarch%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm400045y"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2188</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">32</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm400045y" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A New Target for an Old Drug: Identifying Mitoxantrone as a Nanomolar Inhibitor of PIM1 Kinase via Kinome-Wide Selectivity Modeling&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xiaobo&quot;,&quot;last_name&quot;:&quot;Wan&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Yuting&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Niu&quot;,&quot;last_name&quot;:&quot;Huang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;2619-2629&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm400045y&quot;},&quot;abstract&quot;:&quot;The rational design of selective kinase inhibitors remains a great challenge. Here we describe a physics-based approach to computationally modeling the kinase inhibitor selectivity profile. We retrospectively assessed this protocol by computing the binding profiles of 17 well-known kinase inhibitors against 143 kinases. Next, we predicted the binding profile of the chemotherapy drug mitoxantrone, and chose the predicted top five kinase targets for in vitro kinase assays. Remarkably, mitoxantrone was shown to possess low nanomolar inhibitory activity against PIM1 kinase and to inhibit the PIM1-mediated phosphorylation in cancer cells. We further determined the crystal complex structure of PIM1 bound with mitoxantrone, which reveals the structural and mechanistic basis for a novel mode of PIM1 inhibition. Although mitoxantrone’s mechanism of action had been originally thought to act through DNA intercalation and type II topoisomerase inhibition, we hypothesize that PIM1 kinase inhibition might also contribute t&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400045y&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400045y" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400045y&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400045y" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400045y&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400045y" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm400045y&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400045y&amp;href=/doi/10.1021/jm400045y" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm400045y" /></input><a href="/doi/pdf/10.1021/jm400045y" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm400045y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm400045y%26sid%3Dliteratum%253Aachs%26pmid%3D23442188%26genre%3Darticle%26aulast%3DWan%26date%3D2013%26atitle%3DA%2BNew%2BTarget%2Bfor%2Ban%2BOld%2BDrug%253A%2BIdentifying%2BMitoxantrone%2Bas%2Ba%2BNanomolar%2BInhibitor%2Bof%2BPIM1%2BKinase%2Bvia%2BKinome-Wide%2BSelectivity%2BModeling%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D6%26spage%3D2619%26epage%3D2629%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291264" title="Kinase inhibitors">Kinase inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jmcmar.2013.56.issue-6/production/jmcmar.2013.56.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The rational design of selective kinase inhibitors remains a great challenge. Here we describe a physics-based approach to computationally modeling the kinase inhibitor selectivity profile. We retrospectively assessed this protocol by computing the binding profiles of 17 well-known kinase inhibitors against 143 kinases. Next, we predicted the binding profile of the chemotherapy drug mitoxantrone, and chose the predicted top five kinase targets for in vitro kinase assays. Remarkably, mitoxantrone was shown to possess low nanomolar inhibitory activity against PIM1 kinase and to inhibit the PIM1-mediated phosphorylation in cancer cells. We further determined the crystal complex structure of PIM1 bound with mitoxantrone, which reveals the structural and mechanistic basis for a novel mode of PIM1 inhibition. Although mitoxantrone’s mechanism of action had been originally thought to act through DNA intercalation and type II topoisomerase inhibition, we hypothesize that PIM1 kinase inhibition might also contribute to mitoxantrone’s therapeutic efficacy and specificity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44807" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44807" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein kinases are one of the most important drug target families; however, the rational design of selective kinase inhibitors remains challenging due to the sequence and structural conservation across the protein kinase family, especially at the conserved ATP-binding sites where most kinase inhibitors bind.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> To date, 15 small molecule kinase inhibitors targeting ATP sites have been approved for marketing and many more are in different stages of clinical trials. All of these kinase drugs are multikinase inhibitors to a certain extent.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> For example, a well-known kinase inhibitor, sorafenib, inhibits several kinases with IC<sub>50</sub> < 100 nM, including Braf, VEGFR2, and PDGFRβ kinases.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Generally, the kinase inhibitor selectivity studies have been limited to within the kinase family members. One exception is our recent investigation that showed that sorafenib binds to a 5-hydroxytryptamine receptor (5-HT<sub>2B</sub>) at low nanomolar affinity, which revealed the unexpected ligand cross-reactivity between GPCR orthosteric ligands and kinase inhibitors.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">The target binding profile is essential for understanding the mode of action of a drug molecule. Experimentally profiling all possible drug leads against all possible kinase targets would be cost prohibitive; therefore, computational prediction of promiscuous kinase binding propensities would be a great advance. Several computational approaches have been reported to predict the kinase inhibitor selectivity profile.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10">(5-10)</a> The “binding site signature” method identifies the hot spot binding-site residues critical for the inhibitor binding based on the available crystal structures, and then relates the inhibitor binding specificity with the conservation of the hot spot residues from multiple sequence alignment analysis.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> This method was applied in a retrospective study using a data set containing 15 kinase inhibitors and about 280 kinase targets, and satisfactory results were obtained.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The X-React<sup>KIN</sup> method was developed to estimate the potential for cross-activity by combining sequence, structure, and inhibitor binding similarities of the ATP-binding sites using a machine learning approach, which was trained using available inhibitor selectivity data.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> Another approach, implemented in the program SCR, models kinase–inhibitor interactions in full atomic detail and computes the relative binding energies using an empirical scoring function treating protein flexibility through side chain rotamer sampling.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The KinDOCK server was set up to offer rapid and straightforward modeling of kinase–inhibitor complexes by comparative docking.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">Nevertheless, the fundamental process of receptor–ligand binding is governed by thermodynamics principles. Accurately calculating the absolute binding free energies of a given ligand against different biomolecular targets is critical for the ligand binding specificity prediction. Fortunately, the ligand binding specificity prediction against similar targets may be less complicated due to the cancellation of certain free energy components.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> A molecular mechanics generalized Born/surface area (MM-GB/SA) scoring method was developed to be an intermediary between high throughput docking and molecular dynamics-based free energy calculations, and has been demonstrated to be capable of capturing inhibitor selectivity for structurally related proteins.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a> Therefore, we expect that this approach may be practically applicable to model the kinase inhibitor selectivity on a kinome-wide scale, and the derived structural and energetic information may be directly applied to “design in” (i.e., engineering desirable binding spectrum) and “design out” (i.e., eliminating the unwanted off-target interaction) specific kinase binding activities.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><div class="NLM_p last">Here we combine comparative docking and a physics-based sampling and scoring method to model the kinase inhibitor selectivity. We assessed our computational protocol in reproducing the binding profiles of 17 kinase inhibitors against 143 protein kinases. These kinases, ligands, and binding affinity data may be a useful benchmarking set for kinase selectivity modeling (available online at <a href="http://www.huanglab.org.cn/kinome/kinome.xls" class="extLink">http://www.huanglab.org.cn/kinome/kinome.xls</a>). We then asked whether we could discover novel kinase targets for an FDA-approved chemotherapy drug mitoxantrone (MX), originally known to act through DNA intercalation and type II topoisomerase (Topo II) inhibition.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46820" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46820" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Assessment of Kinome-Wide Selectivity Modeling</h3><div class="NLM_p">Briefly, our computational modeling procedure (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>) consists of four steps: 1. Selecting the representative kinase crystal structure in an active conformation. 2. Predicting the ligand binding pose using comparative docking. 3. Sampling the binding site conformation with the presence of the docked ligand. 4. Scoring the resulting complex structure using the MM-GB/SA scoring function. This automatic workflow allows rapid evaluation of compounds of interest against the entire structural kinome. Although the total number of kinase targets is limited by the availability of the crystal structures (covering about 30% of the kinome), the gap could be bridged by protein structure prediction.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. The Flowchart of Our Computational Modeling Procedure</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To reduce the complexity of kinase conformational changes, we focused only on the crystal structures in an active conformation in the present study (detailed in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). This led to a high-quality data set containing 17 well-known kinase inhibitors (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) and 143 kinase targets.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Thus, a total of 2,431 binding interactions were modeled (<a href="http://www.huanglab.org.cn/kinome/kinome-ligand.tgz" class="extLink">http://www.huanglab.org.cn/kinome/kinome-ligand.tgz</a>), and the key comparison results are summarized in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>. The enrichment factor (EF<sub>20</sub>) reflects the ability of the calculations to correctly retrieve the true targets among the top 20% ranking targets, while the predictive index (PI) is a more semiquantitative measurement of the power of the target ranking order.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Note that PI value (ranging from 1 to −1) of 1 indicates the perfect prediction, and 0 indicates prediction to be completely random.</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of 17 kinase inhibitors investigated in the present work.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Retrospective Study on Computational Prediction of Kinase Inhibitor Selectivity</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">inhibitor (HET ID)</th><th class="colsep0 rowsep0" align="center">template PDB ID (chain)</th><th class="colsep0 rowsep0" align="center">target (X-ray)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub> X-ray (nM)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub> lowest (nM)</th><th class="colsep0 rowsep0" align="center">primary target (lowest <i>K</i><sub>d</sub>)</th><th class="colsep0 rowsep0" align="center">no. of targets<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">no. of predicted targets in top 20%<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">EF<sub>20</sub><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">PI<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">VX-745 (52P)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FC1">3FC1</a> (X)</td><td class="colsep0 rowsep0" align="left">p38a</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">p38a</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">5.00</td><td class="colsep0 rowsep0" align="left">0.60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">gefitinib (IRE)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ITY">2ITY</a> (A)</td><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">5.00</td><td class="colsep0 rowsep0" align="left">0.57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BI-2536 (R78)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU">2RKU</a> (A)</td><td class="colsep0 rowsep0" align="left">PLK1</td><td class="colsep0 rowsep0" align="left">0.19</td><td class="colsep0 rowsep0" align="left">0.19</td><td class="colsep0 rowsep0" align="left">PLK1</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">5.00</td><td class="colsep0 rowsep0" align="left">0.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LY-333531 (LY4)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J2I">2J2I</a> (B)</td><td class="colsep0 rowsep0" align="left">PIM1</td><td class="colsep0 rowsep0" align="left">270</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">PRKCQ</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">5.00</td><td class="colsep0 rowsep0" align="left">0.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CP-690550 (MI1)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LXK">3LXK</a> (A)</td><td class="colsep0 rowsep0" align="left">JAK3</td><td class="colsep0 rowsep0" align="left">0.16</td><td class="colsep0 rowsep0" align="left">0.16</td><td class="colsep0 rowsep0" align="left">JAK3</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">3.80</td><td class="colsep0 rowsep0" align="left">0.59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">erlotinib (AQ4)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17">1M17</a> (A)</td><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="left">0.67</td><td class="colsep0 rowsep0" align="left">0.67</td><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">3.04</td><td class="colsep0 rowsep0" align="left">0.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dasatinib (1N1)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GQG">2GQG</a> (B)</td><td class="colsep0 rowsep0" align="left">ABL1</td><td class="colsep0 rowsep0" align="left">0.046</td><td class="colsep0 rowsep0" align="left">0.046</td><td class="colsep0 rowsep0" align="left">ABL1</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">2.75</td><td class="colsep0 rowsep0" align="left">0.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">vandetanib (ZD6)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IVU">2IVU</a> (A)</td><td class="colsep0 rowsep0" align="left">RET</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">4.6</td><td class="colsep0 rowsep0" align="left">RIPK2</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">2.32</td><td class="colsep0 rowsep0" align="left">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A-674563 (SS3)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2UZT">2UZT</a> (A)</td><td class="colsep0 rowsep0" align="left">PKACa</td><td class="colsep0 rowsep0" align="left">56</td><td class="colsep0 rowsep0" align="left">0.51</td><td class="colsep0 rowsep0" align="left">CLK2</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">2.27</td><td class="colsep0 rowsep0" align="left">0.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">flavopiridol (CPB)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BLR">3BLR</a> (A)</td><td class="colsep0 rowsep0" align="left">CDK9</td><td class="colsep0 rowsep0" align="left">6.4</td><td class="colsep0 rowsep0" align="left">0.69</td><td class="colsep0 rowsep0" align="left">ICK</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1.70</td><td class="colsep0 rowsep0" align="left">0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">VX-680 (VX6)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2F4J">2F4J</a> (A)</td><td class="colsep0 rowsep0" align="left">ABL1</td><td class="colsep0 rowsep0" align="left">7.5</td><td class="colsep0 rowsep0" align="left">3.9</td><td class="colsep0 rowsep0" align="left">AurA</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1.69</td><td class="colsep0 rowsep0" align="left">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sunitinib (B49)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y7J">2Y7J</a> (A)</td><td class="colsep0 rowsep0" align="left">PHKg2</td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">0.075</td><td class="colsep0 rowsep0" align="left">PDGFRb</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">1.68</td><td class="colsep0 rowsep0" align="left">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TAE-684 (GUI)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XB7">2XB7</a> (A)</td><td class="colsep0 rowsep0" align="left">ALK</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">0.49</td><td class="colsep0 rowsep0" align="left">ROS1</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">1.61</td><td class="colsep0 rowsep0" align="left">0.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">staurosporine (STU)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3A62">3A62</a> (A)</td><td class="colsep0 rowsep0" align="left">S6K1</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">0.024</td><td class="colsep0 rowsep0" align="left">SLK</td><td class="colsep0 rowsep0" align="left">82</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">1.32</td><td class="colsep0 rowsep0" align="left">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">crizotinib (VGH)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YFX">2YFX</a> (A)</td><td class="colsep0 rowsep0" align="left">ALK</td><td class="colsep0 rowsep0" align="left">3.3</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">MET</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1.28</td><td class="colsep0 rowsep0" align="left">0.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SB-203580 (SB2)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MPA">3MPA</a> (A)</td><td class="colsep0 rowsep0" align="left">p38a</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">p38a</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1.25</td><td class="colsep0 rowsep0" align="left">0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">R406 (585)</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FQS">3FQS</a> (A)</td><td class="colsep0 rowsep0" align="left">SYK</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">0.71</td><td class="colsep0 rowsep0" align="left">FLT3</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">1.21</td><td class="colsep0 rowsep0" align="left">0.30</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Target (X-ray) is the inhibitor bound kinase in the X-ray template structure.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Number of targets is the number of the kinases with the experimental <i>K</i><sub>d</sub> values less than 100 nM.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Number of predicted targets in top 20% denotes the number of true positives in top 20% of ranked kinase targets, which corresponds to the enrichment factors (EF<sub>20</sub>) compared to random selection.</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">PI is semiquantitative measurement of the correlation of predicted binding affinity with experimental values.</p></div></div></div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. The two-dimensional plot for the kinome-wide selectivity modeling of 17 kinase inhibitors. The higher EF<sub>20</sub> value and the PI value represent the higher prediction accuracy for a given inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Overall, the averaged EF<sub>20</sub> value of 2.81 and the averaged PI value of 0.35 indicate a reasonable prediction performance of our computational approach in ranking order of the kinase targets for a given inhibitor. Among them, the p38 mitogen-activated protein kinase (MAPK) inhibitor VX745 (EF<sub>20</sub> = 5, PI = 0.6) and epidermal growth factor receptor (EGFR) kinase inhibitor gefitinib (EF<sub>20</sub> = 5, PI = 0.57) achieve the most significant enrichment and the highest correlation (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Interestingly, the inhibitor selectivity profile largely affects its prediction performance, where the more selective an inhibitor is, the better the prediction accuracy becomes (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). For example, p38α shares 66% sequence identity with p38γ, however, the highly selective p38 MAPK inhibitor VX-745 was reported to be more than 1000-fold selective for p38α (<i>K</i><sub>d</sub> = 2.8 nM) over p38γ (<i>K</i><sub>d</sub> > 10,000 nM). Based on the visual examination of modeled complex structures (Figure S3 in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>), one hydrogen bond forms between carbonyl group on VX-745 and hinge backbone amide group of Met109 in the p38α-VX-745 complex model (3.1 Å) while the corresponding interaction is lost in the p38γ-VX-745 complex model (4.0 Å). The structural characteristics agree with the calculated binding energies of −51.2 and −48.5 kcal/mol, respectively. Clearly, our approach is adept at distinguishing the high affinity binding interactions from the weak ones, but performs poorly in differentiating the different targets with binding affinities in the same range.</div><div class="NLM_p">Our key assumptions are that an inhibitor binds different kinases in the same binding mode, and the same kinase binds different inhibitors in the same conformational state, presumably an active conformation. These assumptions are partially supported by the crystal structure analysis, where the averaged root-mean-square deviation (RMSD) values of each inhibitor in different crystal complex structures are generally smaller than 2.5 Å (Table S1 in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). However, exceptional cases do exist. For example, sunitinib, staurosporine, and SB-203580 are clearly showing significant structural deviations in different complex structures, with RMSDs as large as 3.47 Å, 4.24 Å, and 4.39 Å, individually.</div><div class="NLM_p">Although the majority of the available kinase cocrystal structures are in active conformational states,<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20-22)</a> and the active conformational states for different kinases generally have higher structural conservation, it is not always true that the inhibitors shall only bind to the active conformational states; not to mention the single representative active conformation. The complicated interaction of a given ligand with a subset of protein conformations may account for several failure cases, including sunitinib<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> and SB-203580.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> For example, sunitinib was cocrystallized with cKIT kinase in its inactive conformation<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> while it binds to ITK kinase in the active conformation.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> A simple alternative is to choose several different crystal structures as the template for modeling. Therefore, we modeled the inhibitor selectivity using different cocrystal structures in failure cases (e.g., sunitinib, SB-203580), and selected the best score for final evaluation as suggested in a previous study.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Unfortunately, no improvement was observed (Table S2 in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). Our further approach of applying molecular dynamics simulations did not improve the results either (Table S3 in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). This is consistent with a previous study that the rigid kinase backbone treatment had better performance than allowing more sampling.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p last">Physics-based approaches model the protein–ligand interactions in a physically realistic manner, and have the potential to be more accurate with systematic improvements.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> However, rigorous ligand binding free energy calculations are computationally expensive, are complicated to apply, and cannot satisfy the practical requirements of computational efficiency for large data sets. With the consideration of the approximations made in our approach, it is likely that the cancellation of certain free energy components is critical for our approach to succeed. Ultimately, we can also envision a strategy to integrate physics-based methods and informatics (e.g., sequence and structural analysis) to improve the overall prediction performance. While it is beyond the scope of our current study, it is critical to develop a rigorous method to estimate the conformational free energy landscape as a response to the binding of different types of inhibitors, especially inhibitors bound to inactive conformational states like imatinib.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Prospective Target Profiling of MX</h3><div class="NLM_p">We next investigated the predictive power of our approach. The chemotherapy drug MX was chosen based on three criteria: the availability of a cocrystal structure, lack of kinome-wide profiling data, and the pharmaceutical importance. MX is approved by the FDA for the treatment of certain types of cancers, mostly in acute myeloid leukemia (AML), acute lymphoblastic leukemia (ALL), and metastatic hormone-refractory prostate cancer.<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28-30)</a> An important therapeutic feature of MX is its selective induction of apoptosis in myeloid leukemia cell<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and B-chronic lymphocytic leukemia cells.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> MX was also conferred to benefit in progression-free and overall survival in children with relapsed ALL.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> However, the exact mechanism of action is not clear, and its known targets of DNA and Topo II can hardly interpret its specificity and efficacy against a narrower range of tumors than other anthracycline drugs used in the clinic.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><div class="NLM_p">Based on the cocrystal structure of MX bound with protein kinase B (PknB, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FUM">2FUM</a>, IC<sub>50</sub> ∼ 0.8 μM),<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> a Ser/Thr protein kinase in <i>Mycobacterium tuberculosis</i>, we predicted the potential mammalian kinase targets of MX. Five protein kinases thereby emerged as top scoring hits and were submitted to primary screening using the radiometric kinase assay (Reaction Biology Corp., Malvern, PA)<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> at a compound concentration of 1 μM (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Encouragingly, PIM1 kinase was completely inhibited by 1 μM MX. In addition, we also tested five kinases randomly chosen; none of them were active (Table S4 in <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. The Predicted Top 5 Kinase Targets of MX and Experimental Validation Using Radiolabeled Kinase Assay<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">predicted</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">rank</th><th class="colsep0 rowsep0" align="center">kinase target</th><th class="colsep0 rowsep0" align="center" char=".">energy score (kcal/mol)</th><th class="colsep0 rowsep0" align="center">kinase act. (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">CK2a2</td><td class="colsep0 rowsep0" align="char" char=".">–97.8</td><td class="colsep0 rowsep0" align="left">81.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">PIM1</td><td class="colsep0 rowsep0" align="char" char=".">–95.8</td><td class="colsep0 rowsep0" align="left">0.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">CHK2</td><td class="colsep0 rowsep0" align="char" char=".">–94.7</td><td class="colsep0 rowsep0" align="left">59.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">BTK</td><td class="colsep0 rowsep0" align="char" char=".">–93.3</td><td class="colsep0 rowsep0" align="left">45.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">AKT1</td><td class="colsep0 rowsep0" align="char" char=".">–92.0</td><td class="colsep0 rowsep0" align="left">108.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last">All 5 kinase were tested at a MX concentration of 1 μM in the presence of 10 μM ATP in duplicate.</p></div></div></div><div class="NLM_p">We subsequently determined the dose response curve for PIM1 inhibition using the homogeneous time-resolved fluorescence (HTRF)-based technology, and the IC<sub>50</sub> value of 50.9 ± 7.4 nM was determined from three independent tests (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Note that we would not predict PIM1 to be a target of MX based on the low sequence identity between PIM1 and PknB in ATP site (Table S5 in <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). We also validated the inhibitory activity of MX against Bruton’s tyrosine kinase (IC<sub>50</sub> ∼ 1 μM). In addition, MX was reported to inhibit casein kinase 2 with IC<sub>50</sub> value of 0.66 μM.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Therefore, MX inhibits three out of the predicted top five kinases from low micromolar to nanomolar range. Considering that our computational approach performs well for treating selective kinase inhibitors, we hypothesize that MX may only inhibit very few kinases at low nanomolar activities.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. The inhibition of MX on PIM1 measured by HTRF kinase assay. The half-maximum inhibitory concentration (IC<sub>50</sub>) was measured at 7 concentration points by repeating three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> X-ray Crystal Complex Structure of PIM1–MX</h3><div class="NLM_p">To further validate the predicted binding pose, the crystal complex structure of PIM1–MX was determined at 2.7 Å resolution. Coordinates and detailed methods for the solved crystal structure have been deposited to the PDB database with the accession ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I41">4I41</a>. The detailed crystallization information is included in Table S6 in <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>. The overall structure of the PIM1 kinase resembles other published crystal structures.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> PIM1 kinase adopts a typical kinase fold consisting of N-terminal and C-terminal domains linked by a hinge region. However, the presence of residue Pro123 at the hinge region disrupts the canonical hydrogen bonding pattern with the adenine moiety of ATP or bound inhibitors due to the lack of the hydrogen bond donor. One MX molecule interacts with PIM1 in a non-ATP mimetic binding mode without forming the classic hydrogen bond with the hinge region (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A). The RMSD value of MX between predicted and crystal structures is 2.27 Å while the critical interactions between MX and PIM1 ATP site are well conserved. This suggests our comparative docking and refinement strategy is reliable in structural prediction of kinase–MX complexes. The planar dihydroxyanthraquinone moiety occupies the adenine binding pocket, enclosed by hydrophobic residues Leu44, Phe49, Val52, Ala65, Ile104, Leu120, Val126, Leu174, and Ile185. Extensive charged and polar interactions are formed between its flexible hydroxyethylamino moieties and surrounding residues, one interacting with Asp186, Asn172, Asp167, and Lys169 and the other with Asp131 and Asp128 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structures of the PIM1–MX and the Tau SRE RNA–MX. (A) The detailed view of the ATP-binding site of the PIM1 kinase domain with the MX (yellow in sticks) compared with the predicted binding pose (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FUM">2FUM</a>) of MX (orange in lines). The protein is colored in slate cartoon, and charged residues forming hydrogen bonds with MX are highlighted. (B) Focused view of the tau SRE RNA–MX interaction (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2KGP">2KGP</a>). The MX is shown yellow in sticks, and RNA is shown in slate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although it is a completely different scenario, MX interacts with the splicing regulatory element (SRE) of microtubule-associated protein tau RNA (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B)<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> in similar binding characteristics to PIM1, where the hydrophobic stacking interactions between anthraquinone moiety and neighboring RNA bases and the extensive hydrogen bonding interactions between hydroxyethylamino groups and RNA backbone phosphates are the common features. To our knowledge, MX is the first small-molecule DNA intercalation drug identified to inhibit protein kinase at low nanomolar activity. We expect that an understanding of the structural basis of promiscuity of MX may facilitate the design of more selective MX analogues.</div><div class="NLM_p">Strikingly, our solved crystal complex structure also reveals a novel mode of PIM1 inhibition, where another MX molecule binds to an additional binding site adjacent to the one that binds to the ATP site (Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A), and contributes to PIM1 inhibition by substrate-competitive fashion (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B). This additional binding site overlaps with a high-affinity PIM1 peptide substrate (designated as pimtide) binding site (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The pimtide-binding site is directly involved in recognition of phosphorylation site in Bcl-2-associated death promoter (BAD), eukaryotic translation initiation factor 4B (eIF4B), and other PIM1 substrate proteins containing the consensus motif (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>D).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> MX forms strong polar interactions with several charged residues in this site, including Lys169, Asp170, Asp234, Asp239, Glu243, and Glu247. It also interacts favorably with several hydrophobic residues, including Val206, Tyr207, Ile230, and Ile240 (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A). It is well-known that the substrate-binding interface is less conserved than the ATP site, and inhibitors targeting this region may hold promise for improving the specificity while reducing side effects.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MX competitively binds to the PIM1 substrate binding site. (A) The atomistic interactions between the substrate binding site (slate in cartoon) and MX (yellow in sticks). (B) Kinetic study of the interaction of substrate with PIM1. Kinetic studies (<i>n</i> = 2) were performed in the presence of MX at concentrations of 0, 50, and 100 nM, respectively. (C) The overlay of cocrystal structure PIM1–pimtide (PDB code: 2BIL) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=PIM1">PIM1</a>–MX in the substrate binding site. (D) The sequence comparison of pimtide, Pim1 phosphorylation sites of BAD and eIF4B.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To further define the inhibitory mechanism of MX, we determined the catalytic velocities of PIM1 with respect to either ATP or substrate with the presence of different concentrations of MX. The measured <i>K</i><sub>m</sub> values with respect to substrate are 0.84, 1.01, and 1.68 μM in the presence of 0, 50, and 100 nM MX, respectively; which clearly demonstrates a substrate-competitive inhibitory mechanism (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B). Similarly, MX also competes with ATP with increased <i>K</i><sub>m</sub> values in responding to increased concentrations of MX (Figure S4 in <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). These biochemical results strongly support our structural observation that MX acts as a dual ATP-competitive and substrate-competitive inhibitor by binding to both sites simultaneously. Unfortunately, we cannot directly decompose the binding contribution of each individual site based on our biochemical studies. However, the poorer electron density and higher <i>B</i>-factors of the MX at the substrate binding site might suggest the less thermodynamically stable binding interactions with less significant contribution to the PIM1 inhibition. We expect that our study may provide the structural and mechanistic basis to inspire the novel PIM1 kinase inhibitor development by targeting the substrate binding site.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Novel Mechanism of Action Implicated through Targeting PIM1 Kinase</h3><div class="NLM_p">To gain deeper insights into the mechanism of action of MX, we examined the changes of phosphorylation level of Ser112 on BAD protein by MX in two representative human leukemia cell lines, K562 and MV4-11. BAD phosphorylation by PIM1 results in inhibition of its proapoptotic activity.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> MX, acting exactly like other potent PIM1 kinase inhibitors, can dose-dependently inhibit the BAD phosphorylation in both K562 cells and MV4-11 cells (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). The K562 cell line was reported to be less sensitive to PIM1 inhibitors.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Correspondingly, MX inhibited the BAD phosphorylation in K562 cells at micromolar concentrations (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A). The MV4-11 cell line was more sensitive in responding of PIM1 kinase inhibitors,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> thus, a significant reduction in BAD phosphorylation was observed in MV4-11 cells treated by MX at concentrations as low as 100 nM (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B). Meanwhile, MX does not significantly affect the protein levels of total BAD and PIM1 under these effective concentrations. Thus, our cellular phosphorylation assays indicate that MX directly blocks PIM1 enzymatic activity in intact cells.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. MX inhibits the phosphorylation of Bad at Ser112 in K562 and MV4-11 cell lines. (A) Immunoblot analysis of PIM1 in K562 cell line treated with 0.1% vehicle DMSO alone, 0.1, 0.3, 1, 3, 10, 30 μM MX as well as 10 μM SGI-1776 as positive control. Cell were harvested after 12 h and lysed. (B) Immunoblot analysis of PIM1 in MV4-11 cell line treated with 0.1% vehicle DMSO alone, 10, 30, 100 nM MX as well as 100 nM SGI-1776 as positive control. Cell were harvested after 12 h and lysed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Surprisingly but interestingly, the low nanomolar PIM1 inhibition activity of MX might contribute to its anticancer efficacy and specificity in vivo. First, elevated levels of PIM1 kinase were discovered in several types of hematologic malignancies and solid cancers, especially in AML and prostate cancer.<a onclick="showRef(event, 'ref44 ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46 ref47">(44-47)</a> Second, PIM1 kinase inhibitors have been extensively studied as aggressive prostate cancer and AML treatment.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48, 49)</a> Third, the intracellular concentrations of MX were reported to be 200–300 times higher than in plasma in AML patients.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Therefore, the anticancer activity of MX might lie in its capacity of simultaneously targeting multiple cellular targets. In addition, MX was approved for the treatment of relapsing–remitting multiple sclerosis (MS), secondary progressive MS, and progressive-relapsing MS. Recently, PIM1 kinase inhibitor has demonstrated the efficacy in murine models of MS.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Thus, we hypothesize that the PIM1 inhibition might also contribute to MX’s therapeutic effects in the treatment of MS.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16340" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16340" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The rational design of selective kinase inhibitors remains a great challenge. Here we have developed a physics-based computational procedure to efficiently model the kinase selectivity at the kinome-wide scale. We evaluated 17 therapeutically important kinase inhibitors against 143 protein kinases, with the assumptions that an inhibitor binds to different kinases in the same binding mode, and the same kinase binds with different inhibitors in the same active conformational state. Thus, the relative binding free energy calculation via MM-GB/SA scoring method may capture kinase inhibitor selectivity due to the cancellation of certain free energy components. Encouragingly, our approach is adept in predicting the binding profiles for selective kinase inhibitors. Nevertheless, a more accurate approach is desirable for distinguishing the binding complexes with similar binding affinities. Currently, our approach is limited to treating kinase inhibitors with available cocrystal structure, however, we expect that our approach can be easily adopted to study structural analogues of cocrystal ligands, which may provide practical value in designing new selective kinase inhibitors.</div><div class="NLM_p">To predict the potential novel target for the FDA-approved chemotherapy drug MX, originally known to act through DNA intercalation and Topo II inhibition, we computationally profiled MX across our data set containing 143 kinases in active conformation. By only testing the predicted top five kinases, we successfully identified MX as low nanomolar inhibitor against PIM1 kinase and further validated its PIM1 inhibitory activity in cancer cells. Thus we hypothesize that PIM1 kinase inhibition might also contribute to MX’s therapeutic efficacy and specificity, and it represents a verifiable working hypothesis to stimulate further experimental studies to elucidate molecular mechanisms of MX. As the pharmacologist and Nobel laureate Sir James Black once said, “The most fruitful basis for the discovery of a new drug is to start with an old drug.”<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Our findings might shed light on the mechanism of action of MX in clinical applications. For example, it might be logical to combine MX with other targeted anticancer agents whose antitumor activity is limited by PIM1 expression.</div><div class="NLM_p last">Moreover, the structural characteristics of MX are distinct from those of other PIM1 inhibitors, and may provide a novel scaffold for designing PIM1 inhibitors. In particular, the novel binding mode of MX revealed by our crystallographic study may suggest a new direction for developing specific PIM1 inhibitors targeting the substrate binding site.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71424" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71424" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Kinase Structure Analysis and Inhibitor Selection</h3><div class="NLM_p last">The kinase sequences were retrieved from the Protein Data Bank (PDB), and subsequently aligned with the human kinome sequences (<a href="http://www.kinase.com/" class="extLink">http://www.kinase.com/</a>) using BLAST (version 2.2.17).<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> If sequence identity of a PDB structure was larger than 80% compared to any human kinase sequence, and the sequence length of the structure was larger than 150 residues, the kinase was selected. This led to a total of 195 unique kinases with 2,646 single-chain structures. Three key regulatory elements in kinase domain have been commonly used to distinguish the active conformation from the inactive state, including DFG-motif flipping, αC-helix displacement, and conformational change of the activation loop (Figure S1 in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>).<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Accordingly, approximately 70% of single-chain kinase structures were classified in active conformation by satisfying the structural characteristics of at least two regulatory elements; the rest were intermediate and inactive structures (Figure S2 in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). Next, the redundant single-chain structures of single kinase were clustered by structural variations of 13 conserved ATP binding-site residues using the NMRCLUSTER program,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> and the most representative one from the largest cluster was selected as the target structure (Figure S2C in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). Finally, we compiled a total of 159 unique kinases with a single target structure in the active conformation. Among them, 143 kinases were included in a most recent kinase inhibitor profiling campaign, where we identified 17 well-known kinase inhibitors with at least one available cocrystal structure in active conformation (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Thus, we assessed our computational protocol in reproducing the experimental binding profiles of these 17 kinase inhibitors against 143 protein kinases in active conformation, for a total of 2,431 binding interactions.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Modeling Kinase–Inhibitor Complex Structure</h3><div class="NLM_p last">We modeled the initial kinase–inhibitor complex structures by comparative docking strategy as previously reported.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The cocrystal structure with the best binding affinity was chosen as the template structure for each inhibitor. The inhibitor-bound template structure was superimposed on each target structure using the UCSF Chimera program.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> The coordinates of the inhibitor were transferred from the template structure to the target kinase, and the steric clashes between inhibitor and binding-site residues were identified using Chimera. Next, the generated complex structure was submitted to the MM-GB/SA refinement and rescoring procedure. The binding-site residues within 12 Å of the inhibitor were subjected to side chain predictions and energy minimizations using the Protein Local Optimization Program (PLOP) with an all-atom molecular-mechanics (MM) force field and a generalized Born surface area (GB/SA) implicit solvent model with variable internal dielectric constant.<a onclick="showRef(event, 'ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref57 ref58">(57, 58)</a> The binding energy was computed by subtracting the energies of the optimized free ligand in solution and the free protein in solution from the optimized ligand–protein complex’s energy in solution, accounting for protein–ligand interaction energy, desolvation energies of ligand and protein, and ligand strain energy.<a onclick="showRef(event, 'ref11 ref14 ref59'); return false;" href="javascript:void(0);" class="ref ref11 ref14 ref59">(11, 14, 59)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Quantifying Agreement between Computation and Experiment</h3><div class="NLM_p last">In order to quantify the agreement between the <i>K</i><sub>d</sub> values and our prediction, two parameters were calculated: the enrichment factor of top 20% retrieved targets as a function of the percentage of targets in the database selection (EF<sub>20</sub>) and the predictive index (PI), a sum over all pairwise ligand comparisons introduced by Pearlman.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The predictive index (PI) is defined as<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/jm-2013-00045y_m001.gif" alt="" /></img></span>where the <i>P</i>(<i>i</i>) is the calculated binding score and <i>E</i>(<i>i</i>) is the log of experimental p<i>K</i><sub>d</sub>. This index reflects the consistency between the experiment and prediction, and it ranges from +1 to −1. 1 indicates perfect prediction, and 0 indicates completely random prediction. Similar to the definition of enrichment in ligand docking method, the enrichment factor reflects the ability of our computational method to find true positives throughout the background database compared with random selection.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The EF<sub>20</sub> is calculated as = (number of true targets in subset/number of targets in subset)/(number of true targets/number of kinases). The experimental binding <i>K</i><sub>d</sub> value of 100 nM was chosen as cutoff to discriminate target or nontarget.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Molecular Dynamics-Based MM-PB/SA Calculations</h3><div class="NLM_p last">The kinase–inhibitor complex structure generated after MM-GB/SA refinement was used as starting structure for the MD simulations. MD simulations and energy calculations were performed using AMBER10.0 with the AMBER99SB protein force field.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61, 62)</a> The ligand parameters were assigned using the general atom force field (GAFF)<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> and the AM1-BCC charge scheme.<a onclick="showRef(event, 'ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref64 ref65">(64, 65)</a> All system setups were performed in the TLEAP module, and MD simulations were carried out in the SANDER module. For each complex system, three stages of minimization were performed in the gas phase, followed by the addition of a 30 Å water cap based on the geometric center of the binding site. After 200 ps of equilibration of solvents, a production run of 5 ns was performed at 300 K with a time step of 2.0 fs. All residues including solvents beyond 12 Å of the ligand were fully frozen. 100 snapshots from the last 1 ns of the trajectory were extracted for energy calculation. The interaction energy between receptor and ligand as well as internal energy was calculated with the SANDER module. For the solvation energy, the polar contribution was calculated using PB/SA<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> with PARSE radii while the nonpolar part was estimated proportional to the solvent-accessible surface area.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Primary Kinase Profiling</h3><div class="NLM_p last">The inhibition potential of MX on the predicted top five kinase targets (i.e., CK2a2, PIM1, CHK2, BTK, and AKT1) was tested using P33 radiolabeled kinase assay (Reaction Biology Corp., Malvern, PA).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Each kinase activity assay was performed in duplicate with a MX concentration of 1 μM and an ATP concentration of 10 μM. The testing compound MX was obtained from Sigma-Aldrich, with guaranteed purity >98%.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Recombinant PIM1 Kinase Expression and Purification</h3><div class="NLM_p last">The cDNA of PIM1 kinase (29–313) was amplified by transfer-PCR (TPCR)<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> using Ultimate ORF clone IOH14704 (Invitrogen Corp., Carlsbad, CA) as a template. The forward and reverse primers are 5′-GAAGTGCTGTTCCAGGGCCCACATATGGCTAAGGAGAAGGAGCCCCTG-3′ and 5′-ATCTCAGTGGTGGTGGTGGTGGTGCTCGAGCTATTTGCTGGGCCCCGG-3′. The gene was cloned into modified pET28a vector and verified by sequencing. The PIM1 construct was expressed in <i>Escherichia coli</i> strain BL21 Star (DE3) pLysS in LB media with 0.5 mM IPTG induction for 15 h at 18 °C. Then the cells were lysed and purified with Ni affinity column (GE Lifescience). After the protease cleavages His-tag on the overnight at 4 °C, the digested PIM1 was loaded directly onto a HiTrap Q column (GE Lifescience) for anion-exchange chromatography. The fractions containing PIM1 kinase were further loaded onto a pre-equilibrated Superdex-200 column (GE Lifescience) followed by elution using buffer (20 mM Tris pH 8.0, 200 mM NaCl, 2 mM CaCl<sub>2</sub>, 2 mM MgCl<sub>2</sub>, 5 mM DTT).<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> The purity of PIM1 kinase was characterized using SDS–PAGE.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> HTRF Kinase Assay</h3><div class="NLM_p last">The Ser/Thr KinEase assay kit for Ser/Thr kinases 3 (CisBio) was used to measure the half-maximum inhibitory concentration (IC<sub>50</sub>) of PIM1 kinase according to the manuscript. The HTRF kinase assay was conducted after optimizing the ATP, enzyme, and substrate concentrations. The assay was carried out in 10 μL of kinase reaction buffer containing 50 mM HEPES (pH 7.0), 0.02% NaN3, 0.01% BSA, 0.1 mM orthovanadate, 5 mM MgCl<sub>2</sub>, 1 mM DTT, 5.8 nM PIM1 protein, 100 μM ATP, and 1 μM biotin labeled substrate at RT for 30 min in the presence or absence of inhibitor. The reactions were allowed to stand at room temperature for 1 h and then read in a time-resolved fluorescence detector (Envision, Perkin-Elmer) at 615 and 665 nm simultaneously. The ratio between the signals of 615 and 665 nm was used in the calculation of the IC<sub>50</sub> value. To determine the kinetics binding model of MX on PIM1 on the substrate binding site, initial rates were obtained by keeping a constant PIM1 concentration of 0.2 μM and varying the substrate concentration ranging from 0.15 to 10 μM in the presence of MX at a concentration of 0, 50, and 100 nM, respectively. In a similar way for the ATP kinetic study of PIM1, the initial rates were measured by changing the ATP concentration from 50 to 1000 μM in the presence of MX at a concentration of 0, 25, and 50 nM, respectively. The <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> values were determined with the Michaelis–Menten equation fits and Lineweaver–Burk plot in GraphPad Prism 5.0.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> X-ray Cocrystal Structure Determination</h3><div class="NLM_p last">The crystal of MX bound PIM1 kinase was obtained by soaking. Hanging drop vapor diffusion method was applied to crystallize the apo-PIM1 by mixing 1 μL of protein (10 mg/mL) solution and 1 μL of well buffer at 291 K. PIM1 crystallized in well buffer containing 0.4 M potassium sodium tartrate tetrahydrate after 7 days. For inhibitor soaking, apo-crystal was transferred to a 2 μL drop of well buffer plus 0.2 mM MX, soaked for 1 h. Then the MX concentration was further increased to 0.5 mM for soaking overnight. The soaked crystal was cyroprotected by the well buffer plus about 25% (v/v) glycerol. A data set was collected using the Rigaku X-ray generator (Cu Kα, 007HF) with the RAXIS IV++ detector. Diffractions were integrated and scaled by Denzo and Scalepack programs. The structure was solved by molecular replacement in Phaser with the apo PIM1 structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YWV">1YWV</a><a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>) as search model. Ligand fitting and model adjustment were carried out in Coot, and the model was refined in Refmac5.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> All molecular structure graphic figures were made with PyMOL.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Cellular Phosphorylation Assays</h3><div class="NLM_p last">The human leukemia cell lines K562 and MV4-11 were obtained from the American Type Culture Collection (Manassas, VA) and cultured in the recommended medium. K562 cells were cultured in RPM1640 plus 10% fetal bovine serum (Invitrogen, Carlsbad, CA). MV4-11 cells were cultured under the same conditions in Iscove’s modified Dulbecco’s medium (IMDM) plus 10% fetal bovine serum. Cells were seeded at 2–10 × 10<sup>5</sup> cells/5 mL/dish. After treatment with compounds for 12 h, the cells were scraped into cold PBS, pelleted, and resuspended in 100 μL of ice cold cell lysis buffer supplemented with protease inhibitors. The lysates were normalized by total protein content and analyzed by immunoblotting with the following antibodies: monoclonal anti-PIM1 (Epitomics), monoclonal anti-h-actin antibody (Abcam), polyclonal anti-BAD (Cell Signaling Technology), and monoclonal anti-phospho-BAD S112 (Cell Signaling Technology).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21"><a href="/doi/suppl/10.1021/jm400045y">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30102" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30102" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The details for kinase structure classification, and the criteria to determinate the kinase active conformation. The RMSDs of kinase inhibitors from different cocrystal structures, and the comparisons of experimental binding profiles and prediction results in several failure cases. The kinetic study of the interaction of ATP with PIM1 in the presence of MX. The sequence alignment of ATP site for 5 representative kinases against PknB. The activity profile of five randomly chosen kinases using radiolabeled kinase assay. The PIM1–MX complex crystallization, data collection, and structure determination. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400045y/suppl_file/jm400045y_si_001.pdf">jm400045y_si_001.pdf (1.26 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm400045y" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36568" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36568" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niu Huang</span> - <span class="hlFld-Affiliation affiliation">Graduate School
of Peking Union
Medical College and Chinese Academy of Medical Sciences, Beijing,
China</span>; 
    <span class="hlFld-Affiliation affiliation">National Institute
of Biological
Sciences, Beijing, No. 7 Science Park Road, Zhongguancun Life Science
Park, Changping District, Beijing 102206, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a0c8d5c1cec7cec9d5e0cec9c2d38ec1c38ec3ce"><span class="__cf_email__" data-cfemail="cca4b9ada2aba2a5b98ca2a5aebfe2adafe2afa2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaobo Wan</span> - <span class="hlFld-Affiliation affiliation">Graduate School
of Peking Union
Medical College and Chinese Academy of Medical Sciences, Beijing,
China</span>; 
    <span class="hlFld-Affiliation affiliation">National Institute
of Biological
Sciences, Beijing, No. 7 Science Park Road, Zhongguancun Life Science
Park, Changping District, Beijing 102206, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Zhang</span> - <span class="hlFld-Affiliation affiliation">National Institute
of Biological
Sciences, Beijing, No. 7 Science Park Road, Zhongguancun Life Science
Park, Changping District, Beijing 102206, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Li</span> - <span class="hlFld-Affiliation affiliation">National Institute
of Biological
Sciences, Beijing, No. 7 Science Park Road, Zhongguancun Life Science
Park, Changping District, Beijing 102206, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuting Xie</span> - <span class="hlFld-Affiliation affiliation">National Institute
of Biological
Sciences, Beijing, No. 7 Science Park Road, Zhongguancun Life Science
Park, Changping District, Beijing 102206, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Li</span> - <span class="hlFld-Affiliation affiliation">Graduate School
of Peking Union
Medical College and Chinese Academy of Medical Sciences, Beijing,
China</span>; 
    <span class="hlFld-Affiliation affiliation">National Institute
of Biological
Sciences, Beijing, No. 7 Science Park Road, Zhongguancun Life Science
Park, Changping District, Beijing 102206, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>X.W. and W.Z. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d223e1654-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13217" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13217" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the anonymous reviewers for their constructive comments for improving the manuscript. Computational support was provided by the Supercomputing Center of Chinese Academy of Sciences (SCCAS). We gratefully thank Luyao Ma for computer support. We thank Pan Chen at NIBS for advice on Western blot experiment. We also thank Dr. Andrew Christofferson for proofreading. We acknowledge financial support from the Chinese Ministry of Science and Technology “973” Grant 2011CB812402 (to N.H.).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i24" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i24"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i25" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i25"> Abbreviations Used</h2><tr><td class="NLM_term">PIM</td><td class="NLM_def"><p class="first last">proviral integration site in Moloney murine leukemia virus</p></td></tr><tr><td class="NLM_term">MX</td><td class="NLM_def"><p class="first last">mitoxantrone</p></td></tr><tr><td class="NLM_term">Topo II</td><td class="NLM_def"><p class="first last">type II topoisomerase</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">MAPK</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase</p></td></tr><tr><td class="NLM_term">PknB</td><td class="NLM_def"><p class="first last">protein kinase B</p></td></tr><tr><td class="NLM_term">SRE</td><td class="NLM_def"><p class="first last">splicing regulatory element</p></td></tr><tr><td class="NLM_term">BAD</td><td class="NLM_def"><p class="first last">Bcl-2-associated death promoter</p></td></tr><tr><td class="NLM_term">eIF4B</td><td class="NLM_def"><p class="first last">eukaryotic translation initiation factor 4B</p></td></tr><tr><td class="NLM_term">PLOP</td><td class="NLM_def"><p class="first last">Protein Local Optimization Program</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">PME</td><td class="NLM_def"><p class="first last">particle mesh Ewald</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">MM-GB/SA</td><td class="NLM_def"><p class="first last">molecular mechanics-generalized Born/surface area</p></td></tr><tr><td class="NLM_term">MM-PB/SA</td><td class="NLM_def"><p class="first last">molecular mechanics-Poisson–Boltzmann/surface area</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time-resolved fluorescence</p></td></tr><tr><td class="NLM_term">TPCR</td><td class="NLM_def"><p class="first last">transfer-PCR</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">multiple sclerosis</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45873" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45873" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 71 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Thaimattam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banerjee, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miglani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iqbal, J.</span><span> </span><span class="NLM_article-title">Protein Kinase Inhibitors: Structural Insights Into Selectivity</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">2751</span><span class="NLM_x">–</span> <span class="NLM_lpage">2765</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.2174%2F138161207781757042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=17897021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGmt7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=2751-2765&author=R.+Thaimattamauthor=R.+Banerjeeauthor=R.+Miglaniauthor=J.+Iqbal&title=Protein+Kinase+Inhibitors%3A+Structural+Insights+Into+Selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase inhibitors: structural insights into selectivity</span></div><div class="casAuthors">Thaimattam, Ram; Banerjee, Rahul; Miglani, Rajni; Iqbal, Javed</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">2751-2765</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are involved in many diseases like cancer, inflammation, cardiovascular disease, and diabetes.  They have become attractive target classes for drug development, making kinase inhibitors as important class of therapeutics.  The success of small-mol. ATP-competitive kinase inhibitors such as Gleevec, Iressa, and Tarceva has attracted much attention in the recent past.  Kinases make use of ATP for phosphorylation of a specific residue(s) on their protein substrates.  More than 400 x-ray structures of about 70 different kinases are publicly available.  These structures provide insights into selectivity and mechanisms of inhibition.  However, prediction of binding specificity of kinase inhibitors based on structural information alone appears to be insufficient.  Here, we will review these observations to gain insights into the rules that govern protein kinase inhibitor selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTBCSBUk5vhbVg90H21EOLACvtfcHk0ljCdj8jnOBdNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGmt7jN&md5=b4bffab06eddc7d04f7b0f5a4ad3fad4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F138161207781757042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161207781757042%26sid%3Dliteratum%253Aachs%26aulast%3DThaimattam%26aufirst%3DR.%26aulast%3DBanerjee%26aufirst%3DR.%26aulast%3DMiglani%26aufirst%3DR.%26aulast%3DIqbal%26aufirst%3DJ.%26atitle%3DProtein%2520Kinase%2520Inhibitors%253A%2520Structural%2520Insights%2520Into%2520Selectivity%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2007%26volume%3D13%26spage%3D2751%26epage%3D2765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Morphy, R.</span><span> </span><span class="NLM_article-title">Selectively Nonselective Kinase Inhibition: Striking the Right Balance</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1413</span><span class="NLM_x">–</span> <span class="NLM_lpage">1437</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2009&pages=1413-1437&author=R.+Morphy&title=Selectively+Nonselective+Kinase+Inhibition%3A+Striking+the+Right+Balance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26atitle%3DSelectively%2520Nonselective%2520Kinase%2520Inhibition%253A%2520Striking%2520the%2520Right%2520Balance%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D53%26spage%3D1413%26epage%3D1437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Wilhelm, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNabola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shujath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gawlak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eveleigh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowley, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adnane, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auclair, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gedrich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voznesensky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riedl, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Post, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trail, P. A.</span><span> </span><span class="NLM_article-title">BAY 43-9006 exhibits broad spectrum oral antitumor activity and targets the RAF/MEK/ERK pathway and receptor tyrosine kinases involved in tumor progression and angiogenesis</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">7099</span><span class="NLM_x">–</span> <span class="NLM_lpage">7109</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=7099-7109&author=S.+M.+Wilhelmauthor=C.+Carterauthor=L.+Tangauthor=D.+Wilkieauthor=A.+McNabolaauthor=H.+Rongauthor=C.+Chenauthor=X.+Zhangauthor=P.+Vincentauthor=M.+McHughauthor=Y.+Caoauthor=J.+Shujathauthor=S.+Gawlakauthor=D.+Eveleighauthor=B.+Rowleyauthor=L.+Liuauthor=L.+Adnaneauthor=M.+Lynchauthor=D.+Auclairauthor=I.+Taylorauthor=R.+Gedrichauthor=A.+Voznesenskyauthor=B.+Riedlauthor=L.+E.+Postauthor=G.+Bollagauthor=P.+A.+Trail&title=BAY+43-9006+exhibits+broad+spectrum+oral+antitumor+activity+and+targets+the+RAF%2FMEK%2FERK+pathway+and+receptor+tyrosine+kinases+involved+in+tumor+progression+and+angiogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DS.%2BM.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DWilkie%26aufirst%3DD.%26aulast%3DMcNabola%26aufirst%3DA.%26aulast%3DRong%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DVincent%26aufirst%3DP.%26aulast%3DMcHugh%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DShujath%26aufirst%3DJ.%26aulast%3DGawlak%26aufirst%3DS.%26aulast%3DEveleigh%26aufirst%3DD.%26aulast%3DRowley%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DAdnane%26aufirst%3DL.%26aulast%3DLynch%26aufirst%3DM.%26aulast%3DAuclair%26aufirst%3DD.%26aulast%3DTaylor%26aufirst%3DI.%26aulast%3DGedrich%26aufirst%3DR.%26aulast%3DVoznesensky%26aufirst%3DA.%26aulast%3DRiedl%26aufirst%3DB.%26aulast%3DPost%26aufirst%3DL.%2BE.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DTrail%26aufirst%3DP.%2BA.%26atitle%3DBAY%252043-9006%2520exhibits%2520broad%2520spectrum%2520oral%2520antitumor%2520activity%2520and%2520targets%2520the%2520RAF%252FMEK%252FERK%2520pathway%2520and%2520receptor%2520tyrosine%2520kinases%2520involved%2520in%2520tumor%2520progression%2520and%2520angiogenesis%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D7099%26epage%3D7109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Lin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Life Beyond Kinases: Structure-Based Discovery of Sorafenib as Nanomolar Antagonist of 5-HT Receptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">5749</span><span class="NLM_x">–</span> <span class="NLM_lpage">5759</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300338m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=5749-5759&author=X.+Linauthor=X.-P.+Huangauthor=G.+Chenauthor=R.+Whaleyauthor=S.+Pengauthor=Y.+Wangauthor=G.+Zhangauthor=S.+X.+Wangauthor=S.+Wangauthor=B.+L.+Rothauthor=N.+Huang&title=Life+Beyond+Kinases%3A+Structure-Based+Discovery+of+Sorafenib+as+Nanomolar+Antagonist+of+5-HT+Receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm300338m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300338m%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DS.%2BX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DLife%2520Beyond%2520Kinases%253A%2520Structure-Based%2520Discovery%2520of%2520Sorafenib%2520as%2520Nanomolar%2520Antagonist%2520of%25205-HT%2520Receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D5749%26epage%3D5759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Sheinerman, F. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giraud, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laoui, A.</span><span> </span><span class="NLM_article-title">High Affinity Targets of Protein Kinase Inhibitors Have Similar Residues at the Positions Energetically Important for Binding</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">1134</span><span class="NLM_x">–</span> <span class="NLM_lpage">1156</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=1134-1156&author=F.+B.+Sheinermanauthor=E.+Giraudauthor=A.+Laoui&title=High+Affinity+Targets+of+Protein+Kinase+Inhibitors+Have+Similar+Residues+at+the+Positions+Energetically+Important+for+Binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSheinerman%26aufirst%3DF.%2BB.%26aulast%3DGiraud%26aufirst%3DE.%26aulast%3DLaoui%26aufirst%3DA.%26atitle%3DHigh%2520Affinity%2520Targets%2520of%2520Protein%2520Kinase%2520Inhibitors%2520Have%2520Similar%2520Residues%2520at%2520the%2520Positions%2520Energetically%2520Important%2520for%2520Binding%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D352%26spage%3D1134%26epage%3D1156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Subramanian, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sud, M.</span><span> </span><span class="NLM_article-title">Computational Modeling of Kinase Inhibitor Selectivity</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">395</span><span class="NLM_x">–</span> <span class="NLM_lpage">399</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml1001097" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=395-399&author=G.+Subramanianauthor=M.+Sud&title=Computational+Modeling+of+Kinase+Inhibitor+Selectivity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fml1001097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml1001097%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DG.%26aulast%3DSud%26aufirst%3DM.%26atitle%3DComputational%2520Modeling%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D395%26epage%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Brylinski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skolnick, J.</span><span> </span><span class="NLM_article-title">Comprehensive Structural and Functional Characterization of the Human Kinome by Protein Structure Modeling and Ligand Virtual Screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1839</span><span class="NLM_x">–</span> <span class="NLM_lpage">1854</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100235n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFOnsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1839-1854&author=M.+Brylinskiauthor=J.+Skolnick&title=Comprehensive+Structural+and+Functional+Characterization+of+the+Human+Kinome+by+Protein+Structure+Modeling+and+Ligand+Virtual+Screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Structural and Functional Characterization of the Human Kinome by Protein Structure Modeling and Ligand Virtual Screening</span></div><div class="casAuthors">Brylinski, Michal; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1839-1854</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The growing interest in the identification of kinase inhibitors, promising therapeutics in the treatment of many diseases, has created a demand for the structural characterization of the entire human kinome.  At the outset of the drug development process, the lead-finding stage, approaches that enrich the screening library with bioactive compds. are needed.  Here, protein structure based methods can play an important role, but despite structural genomics efforts, it is unlikely that the three-dimensional structures of the entire kinome will be available soon.  Therefore, at the proteome level, structure-based approaches must rely on predicted models, with a key issue being their utility in virtual ligand screening.  In this study, we employ the recently developed FINDSITE/Q-Dock ligand homol. modeling approach, which is well-suited for proteome-scale applications using predicted structures, to provide extensive structural and functional characterization of the human kinome.  Specifically, we construct structure models for the human kinome; these are subsequently subject to virtual screening against a library of more than 2 million compds.  To rank the compds., we employ a hierarchical approach that combines ligand- and structure-based filters.  Modeling accuracy is carefully validated using available exptl. data with particularly encouraging results found for the ability to identify, without prior knowledge, specific kinase inhibitors.  More generally, the modeling procedure results in a large no. of predicted mol. interactions between kinases and small ligands that should be of practical use in the development of novel inhibitors.  The data set is freely available to the academic community via a user-friendly Web interface at http://cssb.biol.gatech.edu/kinomelhm/ as well as at the ZINC Web site (http://zinc.docking.org/applications/2010Apr/Brylinski-2010.tar.gz).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsNAiYM1dJuLVg90H21EOLACvtfcHk0lg_4a1knGppdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFOnsLbL&md5=4dd283d41804d7f05138f5acf02ad404</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fci100235n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100235n%26sid%3Dliteratum%253Aachs%26aulast%3DBrylinski%26aufirst%3DM.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DComprehensive%2520Structural%2520and%2520Functional%2520Characterization%2520of%2520the%2520Human%2520Kinome%2520by%2520Protein%2520Structure%2520Modeling%2520and%2520Ligand%2520Virtual%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1839%26epage%3D1854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Brylinski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skolnick, J.</span><span> </span><span class="NLM_article-title">Cross-Reactivity Virtual Profiling of the Human Kinome by X-React<sup>KIN</sup>: A Chemical Systems Biology Approach</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">2324</span><span class="NLM_x">–</span> <span class="NLM_lpage">2333</span></span><div class="citationLinks">[<a href="/doi/10.1021/mp1002976" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlyks7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=2324-2333&author=M.+Brylinskiauthor=J.+Skolnick&title=Cross-Reactivity+Virtual+Profiling+of+the+Human+Kinome+by+X-ReactKIN%3A+A+Chemical+Systems+Biology+Approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Cross-Reactivity Virtual Profiling of the Human Kinome by X-ReactKIN: A Chemical Systems Biology Approach</span></div><div class="casAuthors">Brylinski, Michal; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2324-2333</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many drug candidates fail in clin. development due to their insufficient selectivity that may cause undesired side effects.  Therefore, modern drug discovery is routinely supported by computational techniques, which can identify alternate mol. targets with a significant potential for cross-reactivity.  In particular, the development of highly selective kinase inhibitors is complicated by the strong conservation of the ATP-binding site across the kinase family.  In this paper, we describe X-ReactKIN, a new machine learning approach that extends the modeling and virtual screening of individual protein kinases to a system level in order to construct a cross-reactivity virtual profile for the human kinome.  To maximize the coverage of the kinome, X-ReactKIN relies solely on the predicted target structures and employs state-of-the-art modeling techniques.  Benchmark tests carried out against available selectivity data from high-throughput kinase profiling expts. demonstrate that, for almost 70% of the inhibitors, their alternate mol. targets can be effectively identified in the human kinome with a high (>0.5) sensitivity at the expense of a relatively low false pos. rate (<0.5).  Furthermore, in a case study, we demonstrate how X-ReactKIN can support the development of selective inhibitors by optimizing the selection of kinase targets for small-scale counter-screen expts.  The constructed cross-reactivity profiles for the human kinome are freely available to the academic community at http://cssb.biol.gatech.edu/kinomelhm/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6UZWPjvChV7Vg90H21EOLACvtfcHk0lj5uTC4XnWZYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlyks7nK&md5=679887a7ec72771664f4bd420e3f9f3a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fmp1002976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp1002976%26sid%3Dliteratum%253Aachs%26aulast%3DBrylinski%26aufirst%3DM.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DCross-Reactivity%2520Virtual%2520Profiling%2520of%2520the%2520Human%2520Kinome%2520by%2520X-ReactKIN%253A%2520A%2520Chemical%2520Systems%2520Biology%2520Approach%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2010%26volume%3D7%26spage%3D2324%26epage%3D2333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Rockey, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elcock, A. H.</span><span> </span><span class="NLM_article-title">Rapid computational identification of the targets of protein kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">4138</span><span class="NLM_x">–</span> <span class="NLM_lpage">4152</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049461b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4138-4152&author=W.+M.+Rockeyauthor=A.+H.+Elcock&title=Rapid+computational+identification+of+the+targets+of+protein+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm049461b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049461b%26sid%3Dliteratum%253Aachs%26aulast%3DRockey%26aufirst%3DW.%2BM.%26aulast%3DElcock%26aufirst%3DA.%2BH.%26atitle%3DRapid%2520computational%2520identification%2520of%2520the%2520targets%2520of%2520protein%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4138%26epage%3D4152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catherinot, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labesse, G.</span><span> </span><span class="NLM_article-title">kinDOCK: a tool for comparative docking of protein kinase ligands</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">W325</span><span class="NLM_x">–</span> <span class="NLM_lpage">W329</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=W325-W329&author=L.+Martinauthor=V.+Catherinotauthor=G.+Labesse&title=kinDOCK%3A+a+tool+for+comparative+docking+of+protein+kinase+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DCatherinot%26aufirst%3DV.%26aulast%3DLabesse%26aufirst%3DG.%26atitle%3DkinDOCK%253A%2520a%2520tool%2520for%2520comparative%2520docking%2520of%2520protein%2520kinase%2520ligands%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DW325%26epage%3DW329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernacki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Molecular mechanics methods for predicting protein-ligand binding</span> <span class="citation_source-journal">Phys. Chem. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">5166</span><span class="NLM_x">–</span> <span class="NLM_lpage">5177</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1039%2FB608269F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=17203140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFyit7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=5166-5177&author=N.+Huangauthor=C.+Kalyanaramanauthor=K.+Bernackiauthor=M.+P.+Jacobson&title=Molecular+mechanics+methods+for+predicting+protein-ligand+binding"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanics methods for predicting protein-ligand binding</span></div><div class="casAuthors">Huang, Niu; Kalyanaraman, Chakrapani; Bernacki, Katarzyna; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">5166-5177</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review with some new data.  Ligand binding affinity prediction is one of the most important applications of computational chem.  However, accurately ranking compds. with respect to their estd. binding affinities to a biomol. target remains highly challenging.  The authors provide an overview of recent work using mol. mechanics energy functions to address this challenge.  The authors briefly review methods that use mol. dynamics and Monte Carlo simulations to predict abs. and relative ligand binding free energies, as well as the authors' own work in which they have developed a physics-based scoring method that can be applied to hundreds of thousands of compds. by invoking a no. of simplifying approxns.  In the authors' previous studies, they have demonstrated that their scoring method is a promising approach for improving the discrimination between ligands that are known to bind and those that are presumed not to, in virtual screening of large compd. databases.  In new results presented here, the authors explore several improvements to their computational method including modifying the dielec. const. used for the protein and ligand interiors, and empirically scaling energy terms to compensate for deficiencies in the energy model.  Future directions for further improving the authors' physics-based scoring method are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrog6pXfJmCo7Vg90H21EOLACvtfcHk0lj5uTC4XnWZYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFyit7nP&md5=427d48383280229c26a55a8c31c4cc5e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1039%2FB608269F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB608269F%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DBernacki%26aufirst%3DK.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DMolecular%2520mechanics%2520methods%2520for%2520predicting%2520protein-ligand%2520binding%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2006%26volume%3D8%26spage%3D5166%26epage%3D5177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Physics-based methods for studying protein-ligand interactions</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">331</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=17554859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVWls7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=325-331&author=N.+Huangauthor=M.+P.+Jacobson&title=Physics-based+methods+for+studying+protein-ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Physics-based methods for studying protein-ligand interactions</span></div><div class="casAuthors">Huang, Niu; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">325-331</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  The accurate prediction of relative or abs. ligand-binding affinities is challenging in both theor. and practical aspects.  However, with the aid of advanced computing power and sophisticated methodol. development, physics-based free-energy calcns., which can be used to predict ligand-binding affinities, have become increasingly utilized in the field of structure-based drug design.  This review summarizes recent progress made in developing and applying physics-based methods to studying protein-ligand interactions, focusing on methods using mol. mechanics force fields.  Applications of these methods include predicting ligand binding poses, enriching binders in virtual screening, and calcg. relative and abs. binding free energies.  Future directions for further improving current physics-based methods are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH-FuHDbblC7Vg90H21EOLACvtfcHk0ljivlcstJVP3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVWls7o%253D&md5=3aaf948edd0254cc81a44645cbe05a6b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DPhysics-based%2520methods%2520for%2520studying%2520protein-ligand%2520interactions%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2007%26volume%3D10%26spage%3D325%26epage%3D331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernacki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Virtual Screening against Highly Charged Active Sites: Identifying Substrates of Alpha–Beta Barrel Enzymes</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">2059</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi0481186" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=2059-2071&author=C.+Kalyanaramanauthor=K.+Bernackiauthor=M.+P.+Jacobson&title=Virtual+Screening+against+Highly+Charged+Active+Sites%3A+Identifying+Substrates+of+Alpha%E2%80%93Beta+Barrel+Enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi0481186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0481186%26sid%3Dliteratum%253Aachs%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DBernacki%26aufirst%3DK.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DVirtual%2520Screening%2520against%2520Highly%2520Charged%2520Active%2520Sites%253A%2520Identifying%2520Substrates%2520of%2520Alpha%25E2%2580%2593Beta%2520Barrel%2520Enzymes%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26spage%3D2059%26epage%3D2071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Physics-based scoring of protein-ligand complexes: enrichment of known inhibitors in large-scale virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">243</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0502855" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=243-253&author=N.+Huangauthor=C.+Kalyanaramanauthor=J.+J.+Irwinauthor=M.+P.+Jacobson&title=Physics-based+scoring+of+protein-ligand+complexes%3A+enrichment+of+known+inhibitors+in+large-scale+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Physics-Based Scoring of Protein-Ligand Complexes: Enrichment of Known Inhibitors in Large-Scale Virtual Screening</span></div><div class="casAuthors">Huang, Niu; Kalyanaraman, Chakrapani; Irwin, John J.; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">243-253</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors demonstrate that using an all-atom mol. mechanics force field combined with an implicit solvent model for scoring protein-ligand complexes is a promising approach for improving inhibitor enrichment in the virtual screening of large compd. databases.  The rescoring method is evaluated by the extent to which known binders for nine diverse, therapeutically relevant enzymes are enriched against a background of ∼100 000 drug-like decoys.  The improvement in enrichment is most robust and dramatic within the top 1% of the ranked database, i.e., the first thousand compds.; below the first few percent of the ranked database, there is little overall improvement.  The improved early enrichment is likely due to the more realistic treatment of ligand and receptor desolvation in the rescoring procedure.  We also present anecdotal but encouraging results assessing the ability of the rescoring method to predict specificity of inhibitors for structurally related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUqkz9N6IErLVg90H21EOLACvtfcHk0ljivlcstJVP3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N&md5=cf636695aca21a87e050e0f39286ea1d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci0502855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0502855%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DPhysics-based%2520scoring%2520of%2520protein-ligand%2520complexes%253A%2520enrichment%2520of%2520known%2520inhibitors%2520in%2520large-scale%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D243%26epage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Peng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Identifying Multiple-target Ligands via Computational Chemogenomics Approaches</span> <span class="citation_source-journal">Curr. Trends Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1363</span><span class="NLM_x">–</span> <span class="NLM_lpage">1375</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.2174%2F156802612801319016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=1363-1375&author=S.+Pengauthor=X.+Linauthor=Z.+Guoauthor=N.+Huang&title=Identifying+Multiple-target+Ligands+via+Computational+Chemogenomics+Approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.2174%2F156802612801319016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802612801319016%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DIdentifying%2520Multiple-target%2520Ligands%2520via%2520Computational%2520Chemogenomics%2520Approaches%26jtitle%3DCurr.%2520Trends%2520Med.%2520Chem.%26date%3D2012%26volume%3D12%26spage%3D1363%26epage%3D1375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Fry, D. W.</span><span> </span><span class="NLM_article-title">Biochemical pharmacology of anthracenediones and anthrapyrazoles</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">109</span><span class="NLM_x">–</span> <span class="NLM_lpage">125</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2F0163-7258%2891%2990089-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=1666677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK38Xhs1Glt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1991&pages=109-125&author=D.+W.+Fry&title=Biochemical+pharmacology+of+anthracenediones+and+anthrapyrazoles"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical pharmacology of anthracenediones and anthrapyrazoles</span></div><div class="casAuthors">Fry, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-25</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    </div><div class="casAbstract">A review with ∼200 refs.  DNA intercalating antitumor agents represent one of the more widely used classes of therapeutic agents in clin. oncol.  Although a decisive mechanism to explain their ability to kill tumor cells has not been fully defined, the past decade has shown vast progress toward identifying key possibilities.  The anthracenediones and anthrapyrazoles represent two latter generation classes of DNA intercalators that show great clin. promise as antitumor drugs.  The currently known biochem. pharmacol. for these agents are discussed and these data are integrated into a broader picture touching on mechanisms of tumor cell kill.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuqDSbZBlcIbVg90H21EOLACvtfcHk0ljivlcstJVP3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xhs1Glt7k%253D&md5=bd6855d5db49270fac8a9254084b0b5a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2F0163-7258%2891%2990089-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0163-7258%252891%252990089-5%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DD.%2BW.%26atitle%3DBiochemical%2520pharmacology%2520of%2520anthracenediones%2520and%2520anthrapyrazoles%26jtitle%3DPharmacol.%2520Ther.%26date%3D1991%26volume%3D52%26spage%3D109%26epage%3D125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Capranico, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Isabella, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tinelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bigioni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zunino, F.</span><span> </span><span class="NLM_article-title">Similar sequence specificity of mitoxantrone and VM-26 stimulation of in vitro DNA cleavage by mammalian DNA topoisomerase II</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">3038</span><span class="NLM_x">–</span> <span class="NLM_lpage">3046</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi00063a015" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1993&pages=3038-3046&author=G.+Capranicoauthor=P.+De+Isabellaauthor=S.+Tinelliauthor=M.+Bigioniauthor=F.+Zunino&title=Similar+sequence+specificity+of+mitoxantrone+and+VM-26+stimulation+of+in+vitro+DNA+cleavage+by+mammalian+DNA+topoisomerase+II"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fbi00063a015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00063a015%26sid%3Dliteratum%253Aachs%26aulast%3DCapranico%26aufirst%3DG.%26aulast%3DDe%2BIsabella%26aufirst%3DP.%26aulast%3DTinelli%26aufirst%3DS.%26aulast%3DBigioni%26aufirst%3DM.%26aulast%3DZunino%26aufirst%3DF.%26atitle%3DSimilar%2520sequence%2520specificity%2520of%2520mitoxantrone%2520and%2520VM-26%2520stimulation%2520of%2520in%2520vitro%2520DNA%2520cleavage%2520by%2520mammalian%2520DNA%2520topoisomerase%2520II%26jtitle%3DBiochemistry%26date%3D1993%26volume%3D32%26spage%3D3038%26epage%3D3046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lh8twHam2oZeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Pearlman, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charifson, P. S.</span><span> </span><span class="NLM_article-title">Are Free Energy Calculations Useful in Practice? A Comparison with Rapid Scoring Functions for the p38 MAP Kinase Protein System</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">3417</span><span class="NLM_x">–</span> <span class="NLM_lpage">3423</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0100279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD3MXms1Ogu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3417-3423&author=D.+A.+Pearlmanauthor=P.+S.+Charifson&title=Are+Free+Energy+Calculations+Useful+in+Practice%3F+A+Comparison+with+Rapid+Scoring+Functions+for+the+p38+MAP+Kinase+Protein+System"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Are Free Energy Calculations Useful in Practice? A Comparison with Rapid Scoring Functions for the p38 MAP Kinase Protein System</span></div><div class="casAuthors">Pearlman, David A.; Charifson, Paul S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3417-3423</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Precise thermodn. integration free energy simulations have been applied to a congeneric series of 16 inhibitors to the p38 MAP kinase protein for which the exptl. binding data (IC50) is known.  The relative free energy of binding for each compd. has been detd.  For comparison, the same series of compds. have also scored using the best rapid scoring functions used in database screening.  From the results of these calcns., we find (1) that precise free energy simulations allow predictions that are reliable and in good agreement with expt.; (2) that predictions of lower reliability, but still in good qual. agreement with expt., can be obtained using the OWFEG free energy grid method, at a much lower computational cost; (3) and that other methods, not based on free energy simulations yield results in much poorer agreement with expt.  A new predictive index, which measures the reliability of a prediction method in the context of normal use, is defined and calcd. for each scoring method.  Predictive indexes of 0.84, 0.56, 0.04, -0.05, and 0.25 are calcd. for thermodn. integration, OWFEG, ChemScore, PLPScore, and Dock Energy Score, resp., where +1.0 is perfect correct prediction, -1.0 is perfect incorrect prediction, and 0.0 is random.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_vBFabfEdk7Vg90H21EOLACvtfcHk0lh8twHam2oZeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXms1Ogu7w%253D&md5=04b1606cf7a885640133977dc6a1ce63</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm0100279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0100279%26sid%3Dliteratum%253Aachs%26aulast%3DPearlman%26aufirst%3DD.%2BA.%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26atitle%3DAre%2520Free%2520Energy%2520Calculations%2520Useful%2520in%2520Practice%253F%2520A%2520Comparison%2520with%2520Rapid%2520Scoring%2520Functions%2520for%2520the%2520p38%2520MAP%2520Kinase%2520Protein%2520System%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3417%26epage%3D3423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Kufareva, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Type-II kinase inhibitor docking, screening, and profiling using modified structures of active kinase states</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7921</span><span class="NLM_x">–</span> <span class="NLM_lpage">7932</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8010299" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWisb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7921-7932&author=I.+Kufarevaauthor=R.+Abagyan&title=Type-II+kinase+inhibitor+docking%2C+screening%2C+and+profiling+using+modified+structures+of+active+kinase+states"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Type-II Kinase Inhibitor Docking, Screening, and Profiling Using Modified Structures of Active Kinase States</span></div><div class="casAuthors">Kufareva, Irina; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7921-7932</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Type-II kinase inhibitors represent a class of chems. that trap their target kinases in an inactive, so-called DFG-out state, occupying a hydrophobic pocket adjacent to the ATP binding site.  These compds. are often more specific than those that target active DFG-in kinase conformations.  Unfortunately, the discovery of novel type-II scaffolds presents a considerable challenge, partially because the lack of compatible kinase structures makes structure-based methods inapplicable.  We present a computational protocol for converting multiple available DFG-in structures of various kinases (∼70% of mammalian structural kinome) into accurate and specific models of their type-II bound state.  The models, described as deletion-of-loop Asp-Phe-Gly-in (DOLPHIN) kinase models, demonstrate exceptional performance in various inhibitor discovery applications, including compd. pose prediction, screening, and in silico activity profiling.  Given the abundance of the DFG-in structures, the presented approach opens possibilities for kinome-wide discovery of specific mols. targeting inactive kinase states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp699dzZf5YRLVg90H21EOLACvtfcHk0lh8twHam2oZeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWisb%252FK&md5=64fb7b9f789940dd61c68015341af4a6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm8010299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8010299%26sid%3Dliteratum%253Aachs%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DType-II%2520kinase%2520inhibitor%2520docking%252C%2520screening%252C%2520and%2520profiling%2520using%2520modified%2520structures%2520of%2520active%2520kinase%2520states%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7921%26epage%3D7932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Jacobs, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caron, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hare, B. J.</span><span> </span><span class="NLM_article-title">Classifying protein kinase structures guides use of ligand-selectivity profiles to predict inactive conformations: structure of lck/imatinib complex</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">1451</span><span class="NLM_x">–</span> <span class="NLM_lpage">1460</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2008&pages=1451-1460&author=M.+D.+Jacobsauthor=P.+R.+Caronauthor=B.+J.+Hare&title=Classifying+protein+kinase+structures+guides+use+of+ligand-selectivity+profiles+to+predict+inactive+conformations%3A+structure+of+lck%2Fimatinib+complex"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJacobs%26aufirst%3DM.%2BD.%26aulast%3DCaron%26aufirst%3DP.%2BR.%26aulast%3DHare%26aufirst%3DB.%2BJ.%26atitle%3DClassifying%2520protein%2520kinase%2520structures%2520guides%2520use%2520of%2520ligand-selectivity%2520profiles%2520to%2520predict%2520inactive%2520conformations%253A%2520structure%2520of%2520lck%252Fimatinib%2520complex%26jtitle%3DProteins%26date%3D2008%26volume%3D70%26spage%3D1451%26epage%3D1460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Brooijmans, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobilio, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denny, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humblet, C.</span><span> </span><span class="NLM_article-title">An enriched structural kinase database to enable kinome-wide structure-based analyses and drug discovery</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">–</span> <span class="NLM_lpage">774</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fpro.355" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=20135687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktFyrtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2010&pages=763-774&author=N.+Brooijmansauthor=Y.+W.+Changauthor=D.+Mobilioauthor=R.+A.+Dennyauthor=C.+Humblet&title=An+enriched+structural+kinase+database+to+enable+kinome-wide+structure-based+analyses+and+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An enriched structural kinase database to enable kinome-wide structure-based analyses and drug discovery</span></div><div class="casAuthors">Brooijmans, Natasja; Chang, Yu-Wei; Mobilio, Dominick; Denny, Rajiah A.; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">763-774</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The development of a kinase structural database, the kinase knowledge base (KKB), is described.  It covers all human kinase domain structures that have been deposited in the Protein Data Bank.  All structures are renumbered using a common scheme, which enables efficient cross-comparisons and multiple queries of interest to the kinase field.  The common numbering scheme is also used to automatically annotate conserved residues and motifs, and conformationally classify the structures based on the DFG-loop and Helix C. Analyses of residue conservation in the ATP binding site using the full human-kinome-sequence alignment lead to the identification of a conserved hydrogen bond between the hinge region backbone and a glycine in the specificity surface.  Furthermore, 90% of kinases are found to have at least one stabilizing interaction for the hinge region, which has not been described before.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxGHmLofxIH7Vg90H21EOLACvtfcHk0lgD79t8DXaP-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktFyrtb8%253D&md5=d6b66fb4114b942060e6dba7d1da9c96</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fpro.355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.355%26sid%3Dliteratum%253Aachs%26aulast%3DBrooijmans%26aufirst%3DN.%26aulast%3DChang%26aufirst%3DY.%2BW.%26aulast%3DMobilio%26aufirst%3DD.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DAn%2520enriched%2520structural%2520kinase%2520database%2520to%2520enable%2520kinome-wide%2520structure-based%2520analyses%2520and%2520drug%2520discovery%26jtitle%3DProtein%2520Sci.%26date%3D2010%26volume%3D19%26spage%3D763%26epage%3D774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Gajiwala, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshmukh, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diehl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiNitto, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">English, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greig, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lunney, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McTigue, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molina, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quenzer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emmett, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demetri, G. D.</span><span> </span><span class="NLM_article-title">KIT kinase mutants show unique mechanisms of drug resistance to imatinib and sunitinib in gastrointestinal stromal tumor patients</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">1542</span><span class="NLM_x">–</span> <span class="NLM_lpage">1547</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1073%2Fpnas.0812413106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=19164557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVOms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=1542-1547&author=K.+S.+Gajiwalaauthor=J.+C.+Wuauthor=J.+Christensenauthor=G.+D.+Deshmukhauthor=W.+Diehlauthor=J.+P.+DiNittoauthor=J.+M.+Englishauthor=M.+J.+Greigauthor=Y.+A.+Heauthor=S.+L.+Jacquesauthor=E.+A.+Lunneyauthor=M.+McTigueauthor=D.+Molinaauthor=T.+Quenzerauthor=P.+A.+Wellsauthor=X.+Yuauthor=Y.+Zhangauthor=A.+Zouauthor=M.+R.+Emmettauthor=A.+G.+Marshallauthor=H.+M.+Zhangauthor=G.+D.+Demetri&title=KIT+kinase+mutants+show+unique+mechanisms+of+drug+resistance+to+imatinib+and+sunitinib+in+gastrointestinal+stromal+tumor+patients"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">KIT kinase mutants show unique mechanisms of drug resistance to imatinib and sunitinib in gastrointestinal stromal tumor patients</span></div><div class="casAuthors">Gajiwala, Ketan S.; Wu, Joe C.; Christensen, James; Deshmukh, Gayatri D.; Diehl, Wade; DiNitto, Jonathan P.; English, Jessie M.; Greig, Michael J.; He, You-Ai; Jacques, Suzanne L.; Lunney, Elizabeth A.; McTigue, Michele; Molina, Dave; Quenzer, Terri; Wells, Peter A.; Yu, Xiu; Zhang, Yan; Zou, Aihua; Emmett, Mark R.; Marshall, Alan G.; Zhang, Hui-Min; Demetri, George D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1542-1547</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Most gastrointestinal stromal tumors (GISTs) exhibit aberrant activation of the receptor tyrosine kinase (RTK) KIT.  The efficacy of the inhibitors imatinib mesylate and sunitinib malate in GIST patients has been linked to their inhibition of these mutant KIT proteins.  However, patients on imatinib can acquire secondary KIT mutations that render the protein insensitive to the inhibitor.  Sunitinib has shown efficacy against certain imatinib-resistant mutants, although a subset that resides in the activation loop, including D816H/V, remains resistant.  Biochem. and structural studies were undertaken to det. the mol. basis of sunitinib resistance.  Our results show that sunitinib targets the autoinhibited conformation of WT KIT and that the D816H mutant undergoes a shift in conformational equil. toward the active state.  These findings provide a structural and enzymol. explanation for the resistance profile obsd. with the KIT inhibitors.  Prospectively, they have implications for understanding oncogenic kinase mutants and for circumventing drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkwFxa-p-iWbVg90H21EOLACvtfcHk0lgD79t8DXaP-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVOms78%253D&md5=dd226c914c6f9a258426c0f4984a5258</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812413106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812413106%26sid%3Dliteratum%253Aachs%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DWu%26aufirst%3DJ.%2BC.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DDeshmukh%26aufirst%3DG.%2BD.%26aulast%3DDiehl%26aufirst%3DW.%26aulast%3DDiNitto%26aufirst%3DJ.%2BP.%26aulast%3DEnglish%26aufirst%3DJ.%2BM.%26aulast%3DGreig%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DY.%2BA.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLunney%26aufirst%3DE.%2BA.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DMolina%26aufirst%3DD.%26aulast%3DQuenzer%26aufirst%3DT.%26aulast%3DWells%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DEmmett%26aufirst%3DM.%2BR.%26aulast%3DMarshall%26aufirst%3DA.%2BG.%26aulast%3DZhang%26aufirst%3DH.%2BM.%26aulast%3DDemetri%26aufirst%3DG.%2BD.%26atitle%3DKIT%2520kinase%2520mutants%2520show%2520unique%2520mechanisms%2520of%2520drug%2520resistance%2520to%2520imatinib%2520and%2520sunitinib%2520in%2520gastrointestinal%2520stromal%2520tumor%2520patients%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D1542%26epage%3D1547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Kutach, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villaseñor, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belunis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lok, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, L.-N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deval, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novak, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnett, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuglstatter, A.</span><span> </span><span class="NLM_article-title">Crystal Structures of IL-2-inducible T cell Kinase Complexed with Inhibitors: Insights into Rational Drug Design and Activity Regulation</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">154</span><span class="NLM_x">–</span> <span class="NLM_lpage">163</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1111%2Fj.1747-0285.2010.00993.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=20545945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptleqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2010&pages=154-163&author=A.+K.+Kutachauthor=A.+G.+Villase%C3%B1orauthor=D.+Lamauthor=C.+Belunisauthor=C.+Jansonauthor=S.+Lokauthor=L.-N.+Hongauthor=C.-M.+Liuauthor=J.+Devalauthor=T.+J.+Novakauthor=J.+W.+Barnettauthor=W.+Chuauthor=D.+Shawauthor=A.+Kuglstatter&title=Crystal+Structures+of+IL-2-inducible+T+cell+Kinase+Complexed+with+Inhibitors%3A+Insights+into+Rational+Drug+Design+and+Activity+Regulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of IL-2-inducible T cell kinase complexed with inhibitors: insights into rational drug design and activity regulation</span></div><div class="casAuthors">Kutach, Alan K.; Villasenor, Armando G.; Lam, Diana; Belunis, Charles; Janson, Cheryl; Lok, Stephen; Hong, Li-Na; Liu, Chao-Min; Deval, Jerome; Novak, Thomas J.; Barnett, Jim W.; Chu, Wei; Shaw, David; Kuglstatter, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">154-163</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">IL-2-inducible T cell kinase plays an essential role in T cell receptor signaling and is considered a drug target for the treatment of Th2-mediated inflammatory diseases.  By applying high-throughput protein engineering and crystn., we have detd. the x-ray crystal structures of IL-2-inducible T cell kinase in complex with its selective inhibitor BMS-509744 and the broad-spectrum kinase inhibitors sunitinib and RO5191614.  Sunitinib uniquely stabilizes IL-2-inducible T cell kinase in the helix C-in conformation by inducing side chain conformational changes in the ATP-binding site.  This preference of sunitinib to bind to an active kinase conformation is reflective of its broad-spectrum kinase activity.  BMS-509744 uniquely stabilizes the activation loop in a substrate-blocking inactive conformation, indicating that structural changes described for Src family kinases are also involved in the regulation of IL-2-inducible T cell kinase activity.  The obsd. BMS-509744 binding mode allows rationalization of structure-activity relationships reported for this inhibitor class and facilitates further structure-based drug design.  Sequence-based anal. of this binding mode provides guidance for the rational design of inhibitor selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXxFkkRUV7B7Vg90H21EOLACvtfcHk0lgD79t8DXaP-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptleqs7Y%253D&md5=24b9225c52eff5cca4e295dad1c554c3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2010.00993.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2010.00993.x%26sid%3Dliteratum%253Aachs%26aulast%3DKutach%26aufirst%3DA.%2BK.%26aulast%3DVillase%25C3%25B1or%26aufirst%3DA.%2BG.%26aulast%3DLam%26aufirst%3DD.%26aulast%3DBelunis%26aufirst%3DC.%26aulast%3DJanson%26aufirst%3DC.%26aulast%3DLok%26aufirst%3DS.%26aulast%3DHong%26aufirst%3DL.-N.%26aulast%3DLiu%26aufirst%3DC.-M.%26aulast%3DDeval%26aufirst%3DJ.%26aulast%3DNovak%26aufirst%3DT.%2BJ.%26aulast%3DBarnett%26aufirst%3DJ.%2BW.%26aulast%3DChu%26aufirst%3DW.%26aulast%3DShaw%26aufirst%3DD.%26aulast%3DKuglstatter%26aufirst%3DA.%26atitle%3DCrystal%2520Structures%2520of%2520IL-2-inducible%2520T%2520cell%2520Kinase%2520Complexed%2520with%2520Inhibitors%253A%2520Insights%2520into%2520Rational%2520Drug%2520Design%2520and%2520Activity%2520Regulation%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2010%26volume%3D76%26spage%3D154%26epage%3D163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Vogtherr, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saxena, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimme, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schieborr, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pescatore, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delarbre, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wendt, K. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwalbe, H.</span><span> </span><span class="NLM_article-title">NMR characterization of kinase p38 dynamics in free and ligand-bound forms</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">993</span><span class="NLM_x">–</span> <span class="NLM_lpage">997</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fanie.200502770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsFKmtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=993-997&author=M.+Vogtherrauthor=K.+Saxenaauthor=S.+Hoelderauthor=S.+Grimmeauthor=M.+Betzauthor=U.+Schieborrauthor=B.+Pescatoreauthor=M.+Robinauthor=L.+Delarbreauthor=T.+Langerauthor=K.+U.+Wendtauthor=H.+Schwalbe&title=NMR+characterization+of+kinase+p38+dynamics+in+free+and+ligand-bound+forms"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">NMR characterization of kinase p38 dynamics in free and ligand-bound forms</span></div><div class="casAuthors">Vogtherr, Martin; Saxena, Krishna; Hoelder, Swen; Grimme, Susanne; Betz, Marco; Schieborr, Ulrich; Pescatore, Barbara; Robin, Michel; Delarbre, Laure; Langer, Thomas; Wendt, K. Ulrich; Schwalbe, Harald</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">993-997</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In its apo state kinase p38 effects slow motions that can be detected in the NMR spectrum.  One of the affected parts is the pharmacol. interesting DFG motif.  Diarylurea inhibitors that bind to the DFG-out conformation lock this motif in a defined state, whereas DFG-in inhibitors that bind to the adjacent hinge region leave the flexibility of the DFG motif unaffected as seen in crystal structure of the complex of p38 with the inhibitor SB203580.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbgqVXMWgLZLVg90H21EOLACvtfcHk0lhlPXuQPSRsuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsFKmtr0%253D&md5=2194ed9b88b133426dc6cb52ad243d46</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.200502770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200502770%26sid%3Dliteratum%253Aachs%26aulast%3DVogtherr%26aufirst%3DM.%26aulast%3DSaxena%26aufirst%3DK.%26aulast%3DHoelder%26aufirst%3DS.%26aulast%3DGrimme%26aufirst%3DS.%26aulast%3DBetz%26aufirst%3DM.%26aulast%3DSchieborr%26aufirst%3DU.%26aulast%3DPescatore%26aufirst%3DB.%26aulast%3DRobin%26aufirst%3DM.%26aulast%3DDelarbre%26aufirst%3DL.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DWendt%26aufirst%3DK.%2BU.%26aulast%3DSchwalbe%26aufirst%3DH.%26atitle%3DNMR%2520characterization%2520of%2520kinase%2520p38%2520dynamics%2520in%2520free%2520and%2520ligand-bound%2520forms%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26spage%3D993%26epage%3D997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Totrov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Flexible ligand docking to multiple receptor conformations: a practical alternative</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">178</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2Fj.sbi.2008.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=18302984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFClsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=178-184&author=M.+Totrovauthor=R.+Abagyan&title=Flexible+ligand+docking+to+multiple+receptor+conformations%3A+a+practical+alternative"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Flexible ligand docking to multiple receptor conformations: a practical alternative</span></div><div class="casAuthors">Totrov, Maxim; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">178-184</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  State of the art docking algorithms predict an incorrect binding pose for about 50-70% of all ligands when only a single fixed receptor conformation is considered.  In many more cases, lack of receptor flexibility results in meaningless ligand binding scores, even when the correct pose is obtained.  Incorporating conformational rearrangements of the receptor binding pocket into predictions of both ligand binding pose and binding score is crucial for improving structure-based drug design and virtual ligand screening methodologies.  However, direct modeling of protein binding site flexibility remains challenging because of the large conformational space that must be sampled, and difficulties remain in constructing a suitably accurate energy function.  Here the authors show that using multiple fixed receptor conformations, either exptl. detd. by crystallog. or NMR, or computationally generated, is a practical shortcut that may improve docking calcns.  In several cases, such an approach has led to exptl. validated predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppaNNJ60eASbVg90H21EOLACvtfcHk0lhlPXuQPSRsuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFClsrY%253D&md5=61b273c1621cfd64832bdda28898b439</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2008.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2008.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DFlexible%2520ligand%2520docking%2520to%2520multiple%2520receptor%2520conformations%253A%2520a%2520practical%2520alternative%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2008%26volume%3D18%26spage%3D178%26epage%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Schindler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bornmann, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellicena, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarkson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">Structural Mechanism for STI-571 Inhibition of Abelson Tyrosine Kinase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">1938</span><span class="NLM_x">–</span> <span class="NLM_lpage">1942</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1126%2Fscience.289.5486.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=10988075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD3cXms1altLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2000&pages=1938-1942&author=T.+Schindlerauthor=W.+Bornmannauthor=P.+Pellicenaauthor=W.+T.+Millerauthor=B.+Clarksonauthor=J.+Kuriyan&title=Structural+Mechanism+for+STI-571+Inhibition+of+Abelson+Tyrosine+Kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism for STI-571 inhibition of Abelson tyrosine kinase</span></div><div class="casAuthors">Schindler, Thomas; Bornmann, William; Pellicena, Patricia; Miller, W. Todd; Clarkson, Bayard; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">5486</span>),
    <span class="NLM_cas:pages">1938-1942</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The inadvertent activation of the Abelson tyrosine kinase (Abl) causes chronic myelogenous leukemia (CML).  A small-mol. inhibitor of Abl (STI-571) is effective in the treatment of CML.  We report the crystal structure of the catalytic domain of Abl, complexed to a variant of STI-571.  Crit. to the binding of STI-571 is the adoption by the kinase of an inactive conformation, in which a centrally located "activation loop" is not phosphorylated.  The conformation of this loop is distinct from that in active protein kinases, as well as in the inactive form of the closely related Src kinases.  These results suggest that compds. that exploit the distinctive inactivation mechanisms of individual protein kinases can achieve both high affinity and high specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9eAZqIQ1aebVg90H21EOLACvtfcHk0lhlPXuQPSRsuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXms1altLc%253D&md5=56bb233f010f099d0073020ac6b9db59</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.289.5486.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.289.5486.1938%26sid%3Dliteratum%253Aachs%26aulast%3DSchindler%26aufirst%3DT.%26aulast%3DBornmann%26aufirst%3DW.%26aulast%3DPellicena%26aufirst%3DP.%26aulast%3DMiller%26aufirst%3DW.%2BT.%26aulast%3DClarkson%26aufirst%3DB.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DStructural%2520Mechanism%2520for%2520STI-571%2520Inhibition%2520of%2520Abelson%2520Tyrosine%2520Kinase%26jtitle%3DScience%26date%3D2000%26volume%3D289%26spage%3D1938%26epage%3D1942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Powles, T. J.</span><span> </span><span class="NLM_article-title">Evolving clinical strategies: innovative approaches to the use of mitoxantrone—introduction</span> <span class="citation_source-journal">Eur. J. Cancer Care</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1111%2Fj.1365-2354.1997.tb00317.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=1-3&author=T.+J.+Powles&title=Evolving+clinical+strategies%3A+innovative+approaches+to+the+use+of+mitoxantrone%E2%80%94introduction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2354.1997.tb00317.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2354.1997.tb00317.x%26sid%3Dliteratum%253Aachs%26aulast%3DPowles%26aufirst%3DT.%2BJ.%26atitle%3DEvolving%2520clinical%2520strategies%253A%2520innovative%2520approaches%2520to%2520the%2520use%2520of%2520mitoxantrone%25E2%2580%2594introduction%26jtitle%3DEur.%2520J.%2520Cancer%2520Care%26date%3D2008%26volume%3D6%26spage%3D1%26epage%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Harris, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reese, D. M.</span><span> </span><span class="NLM_article-title">Treatment options in hormone-refractory prostate cancer: current and future approaches</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">2177</span><span class="NLM_x">–</span> <span class="NLM_lpage">2192</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=2177-2192&author=K.+A.+Harrisauthor=D.+M.+Reese&title=Treatment+options+in+hormone-refractory+prostate+cancer%3A+current+and+future+approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DK.%2BA.%26aulast%3DReese%26aufirst%3DD.%2BM.%26atitle%3DTreatment%2520options%2520in%2520hormone-refractory%2520prostate%2520cancer%253A%2520current%2520and%2520future%2520approaches%26jtitle%3DDrugs%26date%3D2001%26volume%3D61%26spage%3D2177%26epage%3D2192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Rigacci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpaneto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alterini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrai, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernardi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellesi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi Ferrini, P.</span><span> </span><span class="NLM_article-title">Treatment of large cell lymphoma in elderly patients with a mitoxantrone, cyclophosphamide, etoposide, and prednisone regimen: long-term follow-up results</span> <span class="citation_source-journal">Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">97</span><span class="NLM_x">–</span> <span class="NLM_lpage">104</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2003&pages=97-104&author=L.+Rigacciauthor=A.+Carpanetoauthor=R.+Alteriniauthor=V.+Carraiauthor=F.+Bernardiauthor=G.+Bellesiauthor=G.+Longoauthor=A.+Bosiauthor=P.+Rossi+Ferrini&title=Treatment+of+large+cell+lymphoma+in+elderly+patients+with+a+mitoxantrone%2C+cyclophosphamide%2C+etoposide%2C+and+prednisone+regimen%3A+long-term+follow-up+results"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRigacci%26aufirst%3DL.%26aulast%3DCarpaneto%26aufirst%3DA.%26aulast%3DAlterini%26aufirst%3DR.%26aulast%3DCarrai%26aufirst%3DV.%26aulast%3DBernardi%26aufirst%3DF.%26aulast%3DBellesi%26aufirst%3DG.%26aulast%3DLongo%26aufirst%3DG.%26aulast%3DBosi%26aufirst%3DA.%26aulast%3DRossi%2BFerrini%26aufirst%3DP.%26atitle%3DTreatment%2520of%2520large%2520cell%2520lymphoma%2520in%2520elderly%2520patients%2520with%2520a%2520mitoxantrone%252C%2520cyclophosphamide%252C%2520etoposide%252C%2520and%2520prednisone%2520regimen%253A%2520long-term%2520follow-up%2520results%26jtitle%3DCancer%26date%3D2003%26volume%3D97%26spage%3D97%26epage%3D104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Bhalla, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrado, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tourkina, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullock, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponnathpur, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahoney, M. E.</span><span> </span><span class="NLM_article-title">High-dose mitoxantrone induces programmed cell death or apoptosis in human myeloid leukemia cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">3133</span><span class="NLM_x">–</span> <span class="NLM_lpage">3140</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=1993&pages=3133-3140&author=K.+Bhallaauthor=A.+M.+Ibradoauthor=E.+Tourkinaauthor=C.+Tangauthor=S.+Grantauthor=G.+Bullockauthor=Y.+Huangauthor=V.+Ponnathpurauthor=M.+E.+Mahoney&title=High-dose+mitoxantrone+induces+programmed+cell+death+or+apoptosis+in+human+myeloid+leukemia+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBhalla%26aufirst%3DK.%26aulast%3DIbrado%26aufirst%3DA.%2BM.%26aulast%3DTourkina%26aufirst%3DE.%26aulast%3DTang%26aufirst%3DC.%26aulast%3DGrant%26aufirst%3DS.%26aulast%3DBullock%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DPonnathpur%26aufirst%3DV.%26aulast%3DMahoney%26aufirst%3DM.%2BE.%26atitle%3DHigh-dose%2520mitoxantrone%2520induces%2520programmed%2520cell%2520death%2520or%2520apoptosis%2520in%2520human%2520myeloid%2520leukemia%2520cells%26jtitle%3DBlood%26date%3D1993%26volume%3D82%26spage%3D3133%26epage%3D3140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Bellosillo, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colomer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pons, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gil, J.</span><span> </span><span class="NLM_article-title">Mitoxantrone, a topoisomerase II inhibitor, induces apoptosis of B-chronic lymphocytic leukaemia cells</span> <span class="citation_source-journal">Br. J. Hamaetol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">142</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1046%2Fj.1365-2141.1998.00520.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=9450803" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1998&pages=142-146&author=B.+Bellosilloauthor=D.+Colomerauthor=G.+Ponsauthor=J.+Gil&title=Mitoxantrone%2C+a+topoisomerase+II+inhibitor%2C+induces+apoptosis+of+B-chronic+lymphocytic+leukaemia+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1046%2Fj.1365-2141.1998.00520.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1365-2141.1998.00520.x%26sid%3Dliteratum%253Aachs%26aulast%3DBellosillo%26aufirst%3DB.%26aulast%3DColomer%26aufirst%3DD.%26aulast%3DPons%26aufirst%3DG.%26aulast%3DGil%26aufirst%3DJ.%26atitle%3DMitoxantrone%252C%2520a%2520topoisomerase%2520II%2520inhibitor%252C%2520induces%2520apoptosis%2520of%2520B-chronic%2520lymphocytic%2520leukaemia%2520cells%26jtitle%3DBr.%2520J.%2520Hamaetol.%26date%3D1998%26volume%3D100%26spage%3D142%26epage%3D146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Parker, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leighton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hancock, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutton, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moorman, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ancliff, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masurekar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goulden, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Révész, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darbyshire, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Love, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saha, V.</span><span> </span><span class="NLM_article-title">Effect of mitoxantrone on outcome of children with first relapse of acute lymphoblastic leukaemia (ALL R3): an open-label randomised trial</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">376</span><span class="NLM_x">, </span> <span class="NLM_fpage">2009</span><span class="NLM_x">–</span> <span class="NLM_lpage">2017</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=376&publication_year=2010&pages=2009-2017&author=C.+Parkerauthor=R.+Watersauthor=C.+Leightonauthor=J.+Hancockauthor=R.+Suttonauthor=A.+V.+Moormanauthor=P.+Ancliffauthor=M.+Morganauthor=A.+Masurekarauthor=N.+Gouldenauthor=N.+Greenauthor=T.+R%C3%A9v%C3%A9szauthor=P.+Darbyshireauthor=S.+Loveauthor=V.+Saha&title=Effect+of+mitoxantrone+on+outcome+of+children+with+first+relapse+of+acute+lymphoblastic+leukaemia+%28ALL+R3%29%3A+an+open-label+randomised+trial"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DC.%26aulast%3DWaters%26aufirst%3DR.%26aulast%3DLeighton%26aufirst%3DC.%26aulast%3DHancock%26aufirst%3DJ.%26aulast%3DSutton%26aufirst%3DR.%26aulast%3DMoorman%26aufirst%3DA.%2BV.%26aulast%3DAncliff%26aufirst%3DP.%26aulast%3DMorgan%26aufirst%3DM.%26aulast%3DMasurekar%26aufirst%3DA.%26aulast%3DGoulden%26aufirst%3DN.%26aulast%3DGreen%26aufirst%3DN.%26aulast%3DR%25C3%25A9v%25C3%25A9sz%26aufirst%3DT.%26aulast%3DDarbyshire%26aufirst%3DP.%26aulast%3DLove%26aufirst%3DS.%26aulast%3DSaha%26aufirst%3DV.%26atitle%3DEffect%2520of%2520mitoxantrone%2520on%2520outcome%2520of%2520children%2520with%2520first%2520relapse%2520of%2520acute%2520lymphoblastic%2520leukaemia%2520%2528ALL%2520R3%2529%253A%2520an%2520open-label%2520randomised%2520trial%26jtitle%3DLancet%26date%3D2010%26volume%3D376%26spage%3D2009%26epage%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Coufal, N.; Farnaes, L.</span><span> </span><span class="NLM_article-title">Anthracyclines and Anthracenediones</span>. In  <span class="citation_source-book">Cancer Management in Man: Chemotherapy, Biological Therapy, Hyperthermia and Supporting Measures</span>; <span class="NLM_contrib-group">Minev, B. R.</span>, Ed.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">The Netherlands</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">13</span>, pp  <span class="NLM_fpage">87</span><span class="NLM_x">–</span> <span class="NLM_lpage">102</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1007%2F978-90-481-9704-0_5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=87-102&author=N.+Coufal&author=L.+Farnaesauthor=B.+R.+Minev&title=Cancer+Management+in+Man%3A+Chemotherapy%2C+Biological+Therapy%2C+Hyperthermia+and+Supporting+Measures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1007%2F978-90-481-9704-0_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-90-481-9704-0_5%26sid%3Dliteratum%253Aachs%26aulast%3DCoufal%26aufirst%3DN.%26atitle%3DAnthracyclines%2520and%2520Anthracenediones%26btitle%3DCancer%2520Management%2520in%2520Man%253A%2520Chemotherapy%252C%2520Biological%2520Therapy%252C%2520Hyperthermia%2520and%2520Supporting%2520Measures%26aulast%3DMinev%26aufirst%3DB.%2BR.%26pub%3DSpringer%26date%3D2011%26volume%3D13%26spage%3D87%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Wehenkel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellinzoni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catherinot, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barilone, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labesse, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alzari, P. M.</span><span> </span><span class="NLM_article-title">The structure of PknB in complex with mitoxantrone, an ATP-competitive inhibitor, suggests a mode of protein kinase regulation in mycobacteria</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">580</span><span class="NLM_x">, </span> <span class="NLM_fpage">3018</span><span class="NLM_x">–</span> <span class="NLM_lpage">3022</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2006&pages=3018-3022&author=A.+Wehenkelauthor=P.+Fernandezauthor=M.+Bellinzoniauthor=V.+Catherinotauthor=N.+Bariloneauthor=G.+Labesseauthor=M.+Jacksonauthor=P.+M.+Alzari&title=The+structure+of+PknB+in+complex+with+mitoxantrone%2C+an+ATP-competitive+inhibitor%2C+suggests+a+mode+of+protein+kinase+regulation+in+mycobacteria"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWehenkel%26aufirst%3DA.%26aulast%3DFernandez%26aufirst%3DP.%26aulast%3DBellinzoni%26aufirst%3DM.%26aulast%3DCatherinot%26aufirst%3DV.%26aulast%3DBarilone%26aufirst%3DN.%26aulast%3DLabesse%26aufirst%3DG.%26aulast%3DJackson%26aufirst%3DM.%26aulast%3DAlzari%26aufirst%3DP.%2BM.%26atitle%3DThe%2520structure%2520of%2520PknB%2520in%2520complex%2520with%2520mitoxantrone%252C%2520an%2520ATP-competitive%2520inhibitor%252C%2520suggests%2520a%2520mode%2520of%2520protein%2520kinase%2520regulation%2520in%2520mycobacteria%26jtitle%3DFEBS%2520Lett.%26date%3D2006%26volume%3D580%26spage%3D3018%26epage%3D3022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Anastassiadis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devarajan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, J. R.</span><span> </span><span class="NLM_article-title">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1039</span><span class="NLM_x">–</span> <span class="NLM_lpage">1045</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1038%2Fnbt.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=22037377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1039-1045&author=T.+Anastassiadisauthor=S.+W.+Deaconauthor=K.+Devarajanauthor=H.+Maauthor=J.+R.+Peterson&title=Comprehensive+assay+of+kinase+catalytic+activity+reveals+features+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span></div><div class="casAuthors">Anastassiadis, Theonie; Deacon, Sean W.; Devarajan, Karthik; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. protein kinase inhibitors are widely used to elucidate cellular signaling pathways and are promising therapeutic agents.  Owing to evolutionary conservation of the ATP-binding site, most kinase inhibitors that target this site promiscuously inhibit multiple kinases.  Interpretation of expts. that use these compds. is confounded by a lack of data on the comprehensive kinase selectivity of most inhibitors.  Here we used functional assays to profile the activity of 178 com. available kinase inhibitors against a panel of 300 recombinant protein kinases.  Quant. anal. revealed complex and often unexpected interactions between protein kinases and kinase inhibitors, with a wide spectrum of promiscuity.  Many off-target interactions occur with seemingly unrelated kinases, revealing how large-scale profiling can identify multitargeted inhibitors of specific, diverse kinases.  The results have implications for drug development and provide a resource for selecting compds. to elucidate kinase function and for interpreting the results of expts. involving kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUljWIhZGPkrVg90H21EOLACvtfcHk0lgR-CDH1hq-zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E&md5=b2f705509b0e09afa46b01d4669f0515</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2017%26sid%3Dliteratum%253Aachs%26aulast%3DAnastassiadis%26aufirst%3DT.%26aulast%3DDeacon%26aufirst%3DS.%2BW.%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26atitle%3DComprehensive%2520assay%2520of%2520kinase%2520catalytic%2520activity%2520reveals%2520features%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1039%26epage%3D1045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Li, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, X. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, N. C.</span><span> </span><span class="NLM_article-title">Effects of mitoxantrone on the activity of human protein kinase CK2 holoenzyme</span> <span class="citation_source-journal">Aizheng</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">809</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotVait7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=809-815&author=C.+M.+Liauthor=X.+G.+Liuauthor=X.+C.+Linauthor=X.+W.+Chenauthor=N.+C.+Liang&title=Effects+of+mitoxantrone+on+the+activity+of+human+protein+kinase+CK2+holoenzyme"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of mitoxantrone on activity of human protein kinase CK2 holoenzyme</span></div><div class="casAuthors">Li, Chunmei; Liu, Xinguang; Lin, Xiaocong; Chen, Xiaowen; Liang, Nianci</div><div class="citationInfo"><span class="NLM_cas:title">Aizheng</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">809-815</span>CODEN:
                <span class="NLM_cas:coden">AIZHE4</span>;
        ISSN:<span class="NLM_cas:issn">1000-467X</span>.
    
            (<span class="NLM_cas:orgname">Sun Yat-sen Daxue, Aizheng Zhongxin</span>)
        </div><div class="casAbstract">Protein kinase CK2, a highly conserved protein serine/threonine kinase that is ubiquitously distributed in eukaryotes, has a close relationship with human leukemia.  Mitoxantrone is an effective drug used for acute leukemias.  The effects of mitoxantrone on the activity of recombinant holoenzyme of human protein kinase CK2 and proliferation in human leukemia cell line HL-60 were obsd.  The CK2 holoenzyme composed of α' and β subunits was recombinanted in vitro.  Subsequently, CK2 was treated with mitoxantrone at various concns., followed by addn. of reacting liq. contg. [γ-32P] ATP.  CK2 activity was measured by detecting the radioactivity of 32P transferred onto the substrates of CK2.  The effect of mitoxantrone on the proliferation of HL-60 cells was detected by the trypan blue dye exclusion assay; the changes in the cell cycle were analyzed by flow cytometry (FCM); apoptosis was analyzed by FCM and DNA agarose gel electrophoresis; effects of the drugs on the intrinsic CK2 activity were measured by a specific CK2 peptide substrate.  Mitoxantrone strongly inhibited the holoenzyme activity of recombinant human protein kinase CK2 (IC50=0.66 μmol/L).  The inhibition of CKZ by mitoxantrone was competitive with respect to ATP (Ki 0.25 μmol/L) and mostly non-competitive with respect to casein (Ki 0.66 μmol/L).  Mitoxantrone exerted strong cytotoxicity to HL-60 cells.  When treated with 0.1 μmol/L mitoxantrone for 12 h, the apoptotic rate of HL-60 cells was 9.3%.  Mitoxantrone did not affect intracellular CK2 activity.  Mitoxantrone is a strong inhibitor of recombinant human protein kinase CK2 in vitro.  Apoptosis induced by mitoxantrone in HL-60 cells has no correlation to intracellular CK2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbNRZY5mneZLVg90H21EOLACvtfcHk0lgR-CDH1hq-zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotVait7g%253D&md5=68b230a5a36b6ad93f8c9e8b2a59cfec</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%2BM.%26aulast%3DLiu%26aufirst%3DX.%2BG.%26aulast%3DLin%26aufirst%3DX.%2BC.%26aulast%3DChen%26aufirst%3DX.%2BW.%26aulast%3DLiang%26aufirst%3DN.%2BC.%26atitle%3DEffects%2520of%2520mitoxantrone%2520on%2520the%2520activity%2520of%2520human%2520protein%2520kinase%2520CK2%2520holoenzyme%26jtitle%3DAizheng%26date%3D2008%26volume%3D27%26spage%3D809%26epage%3D815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandiyan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artis, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bremer, R.</span><span> </span><span class="NLM_article-title">Crystal Structures of Proto-oncogene Kinase Pim1: A Target of Aberrant Somatic Hypermutations in Diffuse Large Cell Lymphoma</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">348</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2005&pages=183-193&author=A.+Kumarauthor=V.+Mandiyanauthor=Y.+Suzukiauthor=C.+Zhangauthor=J.+Riceauthor=J.+Tsaiauthor=D.+R.+Artisauthor=P.+Ibrahimauthor=R.+Bremer&title=Crystal+Structures+of+Proto-oncogene+Kinase+Pim1%3A+A+Target+of+Aberrant+Somatic+Hypermutations+in+Diffuse+Large+Cell+Lymphoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DMandiyan%26aufirst%3DV.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DRice%26aufirst%3DJ.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DBremer%26aufirst%3DR.%26atitle%3DCrystal%2520Structures%2520of%2520Proto-oncogene%2520Kinase%2520Pim1%253A%2520A%2520Target%2520of%2520Aberrant%2520Somatic%2520Hypermutations%2520in%2520Diffuse%2520Large%2520Cell%2520Lymphoma%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D348%26spage%3D183%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Zheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donahue, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varani, G.</span><span> </span><span class="NLM_article-title">Structural Basis for Stabilization of the Tau Pre-mRNA Splicing Regulatory Element by Novantrone (Mitoxantrone)</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">557</span><span class="NLM_x">–</span> <span class="NLM_lpage">566</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2Fj.chembiol.2009.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=19477420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1Ogtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=557-566&author=S.+Zhengauthor=Y.+Chenauthor=C.+P.+Donahueauthor=M.+S.+Wolfeauthor=G.+Varani&title=Structural+Basis+for+Stabilization+of+the+Tau+Pre-mRNA+Splicing+Regulatory+Element+by+Novantrone+%28Mitoxantrone%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Stabilization of the Tau Pre-mRNA Splicing Regulatory Element by Novantrone (Mitoxantrone)</span></div><div class="casAuthors">Zheng, Suxin; Chen, Yu; Donahue, Christine P.; Wolfe, Michael S.; Varani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">557-566</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Some familial neurodegenerative diseases are assocd. with mutations that destabilize a putative stem-loop structure within an intronic region of the tau pre-mRNA and alter the prodn. of tau protein isoforms by alternative splicing.  Because stabilization of the stem loop reverses the splicing pattern assocd. with neurodegeneration, small mols. that stabilize this stem loop would provide new ways to dissect the mechanism of neurodegeneration and treat tauopathies.  The anticancer drug mitoxantrone was recently identified in a high throughput screen to stabilize the tau pre-mRNA stem loop.  Here the authors report the soln. structure of the tau mRNA-mitoxantrone complex, validated by the structure-activity relationship of existing mitoxantrone analogs.  The structure describes the mol. basis for their interaction with RNA and provides a rational basis to optimize the activity of this new class of RNA-binding mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNxlGdrUiUcrVg90H21EOLACvtfcHk0lgR-CDH1hq-zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1Ogtb4%253D&md5=3d38884c150535d160fcecc5d1f03534</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2009.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2009.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DDonahue%26aufirst%3DC.%2BP.%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26aulast%3DVarani%26aufirst%3DG.%26atitle%3DStructural%2520Basis%2520for%2520Stabilization%2520of%2520the%2520Tau%2520Pre-mRNA%2520Splicing%2520Regulatory%2520Element%2520by%2520Novantrone%2520%2528Mitoxantrone%2529%26jtitle%3DChem.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D557%26epage%3D566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Bullock, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debreczeni, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amos, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, B. E.</span><span> </span><span class="NLM_article-title">Structure and substrate specificity of the Pim-1 kinase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">41675</span><span class="NLM_x">–</span> <span class="NLM_lpage">41682</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=41675-41682&author=A.+N.+Bullockauthor=J.+Debreczeniauthor=A.+L.+Amosauthor=S.+Knappauthor=B.+E.+Turk&title=Structure+and+substrate+specificity+of+the+Pim-1+kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DDebreczeni%26aufirst%3DJ.%26aulast%3DAmos%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26atitle%3DStructure%2520and%2520substrate%2520specificity%2520of%2520the%2520Pim-1%2520kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D41675%26epage%3D41682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knebel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrice, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barringer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werneburg, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span> </span><span class="NLM_article-title">Pim kinase substrate identification and specificity</span> <span class="citation_source-journal">J. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">141</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">362</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1093%2Fjb%2Fmvm040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=17234686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsVagtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2007&pages=353-362&author=C.+Pengauthor=A.+Knebelauthor=N.+A.+Morriceauthor=X.+Liauthor=K.+Barringerauthor=J.+Liauthor=S.+Jakesauthor=B.+Werneburgauthor=L.+Wang&title=Pim+kinase+substrate+identification+and+specificity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Pim kinase substrate identification and specificity</span></div><div class="casAuthors">Peng, Charline; Knebel, Axel; Morrice, Nick A.; Li, Xiang; Barringer, Kevin; Li, Jun; Jakes, Scott; Werneburg, Brian; Wang, Lian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">353-362</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Japanese Biochemical Society</span>)
        </div><div class="casAbstract">The Pim family of Ser/Thr kinases has been implicated in the process of lymphomagenesis and cell survival.  Known substrates of Pim kinases are few and poorly characterized.  In this study we set out to identify novel Pim-2 substrates using the Kinase Substrate Tracking and Elucidation (KESTREL) approach.  Two potential substrates, eukaryotic initiation factor 4B (eIF4B) and apoptosis inhibitor 5 (API-5), were identified from rat thymus exts.  Sequence comparison of the Pim-2 kinase phosphorylation sites of eIF4B and mouse BAD, the only other known Pim-2 substrate, revealed conserved amino acids preceding the phosphorylated serine residue.  Stepwise replacement of the conserved residues produced a consensus sequence for Pim kinase recognition: RXRHXS.  Pim-1 and Pim-2 catalyzed the phosphorylation of this recognition sequence 20-fold more efficiently than the original (K/R-K/R-R-K/R-L-S/T-a; a = small chain amino acid) Pim-1 phosphorylation site.  The identification of the novel Pim kinase consensus sequence provides a more sensitive and versatile peptide based assay for screening modulators of Pim kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJNFceD0VpZbVg90H21EOLACvtfcHk0lh0O178pHowqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsVagtbs%253D&md5=c303661dce136a9cd826c82d7f7a9943</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fjb%2Fmvm040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjb%252Fmvm040%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DKnebel%26aufirst%3DA.%26aulast%3DMorrice%26aufirst%3DN.%2BA.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DBarringer%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJakes%26aufirst%3DS.%26aulast%3DWerneburg%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DPim%2520kinase%2520substrate%2520identification%2520and%2520specificity%26jtitle%3DJ.%2520Biochem.%26date%3D2007%26volume%3D141%26spage%3D353%26epage%3D362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Aho, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandholm, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peltola, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mankonen, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lilly, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koskinen, P. J.</span><span> </span><span class="NLM_article-title">Pim-1 kinase promotes inactivation of the pro-apoptotic Bad protein by phosphorylating it on the Ser112 gatekeeper site</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">571</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=571&publication_year=2004&pages=43-49&author=T.+L.+Ahoauthor=J.+Sandholmauthor=K.+J.+Peltolaauthor=H.+P.+Mankonenauthor=M.+Lillyauthor=P.+J.+Koskinen&title=Pim-1+kinase+promotes+inactivation+of+the+pro-apoptotic+Bad+protein+by+phosphorylating+it+on+the+Ser112+gatekeeper+site"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAho%26aufirst%3DT.%2BL.%26aulast%3DSandholm%26aufirst%3DJ.%26aulast%3DPeltola%26aufirst%3DK.%2BJ.%26aulast%3DMankonen%26aufirst%3DH.%2BP.%26aulast%3DLilly%26aufirst%3DM.%26aulast%3DKoskinen%26aufirst%3DP.%2BJ.%26atitle%3DPim-1%2520kinase%2520promotes%2520inactivation%2520of%2520the%2520pro-apoptotic%2520Bad%2520protein%2520by%2520phosphorylating%2520it%2520on%2520the%2520Ser112%2520gatekeeper%2520site%26jtitle%3DFEBS%2520Lett.%26date%3D2004%26volume%3D571%26spage%3D43%26epage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Pogacic, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullock, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gasser, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer-Monard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwaller, J.</span><span> </span><span class="NLM_article-title">Structural Analysis Identifies Imidazo[1,2-b]Pyridazines as PIM Kinase Inhibitors with In vitro Antileukemic Activity</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">6916</span><span class="NLM_x">–</span> <span class="NLM_lpage">6924</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=6916-6924&author=V.+Pogacicauthor=A.+N.+Bullockauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=C.+Gasserauthor=A.+Biondiauthor=S.+Meyer-Monardauthor=S.+Knappauthor=J.+Schwaller&title=Structural+Analysis+Identifies+Imidazo%5B1%2C2-b%5DPyridazines+as+PIM+Kinase+Inhibitors+with+In+vitro+Antileukemic+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPogacic%26aufirst%3DV.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DGasser%26aufirst%3DC.%26aulast%3DBiondi%26aufirst%3DA.%26aulast%3DMeyer-Monard%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26atitle%3DStructural%2520Analysis%2520Identifies%2520Imidazo%255B1%252C2-b%255DPyridazines%2520as%2520PIM%2520Kinase%2520Inhibitors%2520with%2520In%2520vitro%2520Antileukemic%2520Activity%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D6916%26epage%3D6924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Amson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigaux, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Przedborski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flandrin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Givol, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telerman, A.</span><span> </span><span class="NLM_article-title">The human protooncogene product p33pim is expressed during fetal hematopoiesis and in diverse leukemias</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">8857</span><span class="NLM_x">–</span> <span class="NLM_lpage">8861</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1989&pages=8857-8861&author=R.+Amsonauthor=F.+Sigauxauthor=S.+Przedborskiauthor=G.+Flandrinauthor=D.+Givolauthor=A.+Telerman&title=The+human+protooncogene+product+p33pim+is+expressed+during+fetal+hematopoiesis+and+in+diverse+leukemias"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmson%26aufirst%3DR.%26aulast%3DSigaux%26aufirst%3DF.%26aulast%3DPrzedborski%26aufirst%3DS.%26aulast%3DFlandrin%26aufirst%3DG.%26aulast%3DGivol%26aufirst%3DD.%26aulast%3DTelerman%26aufirst%3DA.%26atitle%3DThe%2520human%2520protooncogene%2520product%2520p33pim%2520is%2520expressed%2520during%2520fetal%2520hematopoiesis%2520and%2520in%2520diverse%2520leukemias%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1989%26volume%3D86%26spage%3D8857%26epage%3D8861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Cibull, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eble, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ann Baldridge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malott, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, L.</span><span> </span><span class="NLM_article-title">Overexpression of Pim-1 during progression of prostatic adenocarcinoma</span> <span class="citation_source-journal">J. Clin. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">288</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1136%2Fjcp.2005.027672" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=16505280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A280%3ADC%252BD287hslyrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2006&pages=285-288&author=T.+L.+Cibullauthor=T.+D.+Jonesauthor=L.+Liauthor=J.+N.+Ebleauthor=L.+Ann+Baldridgeauthor=S.+R.+Malottauthor=Y.+Luoauthor=L.+Cheng&title=Overexpression+of+Pim-1+during+progression+of+prostatic+adenocarcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of Pim-1 during progression of prostatic adenocarcinoma</span></div><div class="casAuthors">Cibull T L; Jones T D; Li L; Eble J N; Ann Baldridge L; Malott S R; Luo Y; Cheng L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of clinical pathology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">285-8</span>
        ISSN:<span class="NLM_cas:issn">0021-9746</span>.
    </div><div class="casAbstract">AIMS:  Pim-1 is a serine/threonine kinase that has been shown to play an integral role in the development of a number of human cancers, such as haematolymphoid malignancies.  Recently, evidence has shown Pim-1 to be important in prostatic carcinogenesis.  In order to further our understanding of its role in prostate cancer, we investigated Pim-1 expression in normal, premalignant, and malignant prostate tissue.  METHODS:  Using immunohistochemistry, Pim-1 expression was analysed in prostate tissue from 120 radical prostatectomy specimens.  In each case, Pim-1 staining was evaluated in benign prostatic epithelium, high grade prostatic intraepithelial neoplasia (PIN), and prostatic adenocarcinoma.  The number of positively staining cells was estimated, and the intensity of staining was scored on a scale of 0 to 3+.  RESULTS:  Pim-1 immunoreactivity was identified in 120 cases (100%) of adenocarcinoma, 120 cases (100%) of high grade PIN, and 62 cases (52%) of benign glands.  The number of cells staining in benign epithelium (mean 34%) was much lower than that in high grade PIN (mean 80%; p<0.0001) or adenocarcinoma (mean, 84%; p<0.0001).  There was no significant difference between high grade PIN and adenocarcinoma in the percentage of cells staining positively for Pim-1 (p = 0.34).  The staining intensity for Pim-1 was significantly lower in benign prostatic epithelium than in PIN and adenocarcinoma (p<0.001).  There was no statistically significant correlation between the level of Pim-1 expression and Gleason score, patient age, tumour stage, lymph node metastasis, perineural invasion, vascular invasion, surgical margin status, extraprostatic extension, or seminal vesicle invasion.  CONCLUSIONS:  Pim-1 expression is elevated in PIN and prostatic adenocarcinoma compared with benign prostatic epithelium.  This finding suggests that upregulation of Pim-1 may play a role in prostatic neoplasia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRz4dmNESksJSXXDYhVZBdlfW6udTcc2eaR-fAssnPIC7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD287hslyrsw%253D%253D&md5=358e748eac699dc9b6b2d4509553c1e4</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1136%2Fjcp.2005.027672&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjcp.2005.027672%26sid%3Dliteratum%253Aachs%26aulast%3DCibull%26aufirst%3DT.%2BL.%26aulast%3DJones%26aufirst%3DT.%2BD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DEble%26aufirst%3DJ.%2BN.%26aulast%3DAnn%2BBaldridge%26aufirst%3DL.%26aulast%3DMalott%26aufirst%3DS.%2BR.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DCheng%26aufirst%3DL.%26atitle%3DOverexpression%2520of%2520Pim-1%2520during%2520progression%2520of%2520prostatic%2520adenocarcinoma%26jtitle%3DJ.%2520Clin.%2520Pathol.%26date%3D2006%26volume%3D59%26spage%3D285%26epage%3D288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Chen, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redkar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wierda, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandhi, V.</span><span> </span><span class="NLM_article-title">Pim kinase inhibitor, SGI-1776, induces apoptosis in chronic lymphocytic leukemia cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">4150</span><span class="NLM_x">–</span> <span class="NLM_lpage">4157</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2009&pages=4150-4157&author=L.+S.+Chenauthor=S.+Redkarauthor=D.+Bearssauthor=W.+G.+Wierdaauthor=V.+Gandhi&title=Pim+kinase+inhibitor%2C+SGI-1776%2C+induces+apoptosis+in+chronic+lymphocytic+leukemia+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%2BS.%26aulast%3DRedkar%26aufirst%3DS.%26aulast%3DBearss%26aufirst%3DD.%26aulast%3DWierda%26aufirst%3DW.%2BG.%26aulast%3DGandhi%26aufirst%3DV.%26atitle%3DPim%2520kinase%2520inhibitor%252C%2520SGI-1776%252C%2520induces%2520apoptosis%2520in%2520chronic%2520lymphocytic%2520leukemia%2520cells%26jtitle%3DBlood%26date%3D2009%26volume%3D114%26spage%3D4150%26epage%3D4157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Nawijn, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alendar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berns, A.</span><span> </span><span class="NLM_article-title">For better or for worse: the role of Pim oncogenes in tumorigenesis</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1038%2Fnrc2986" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=21150935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFCjsrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=23-34&author=M.+C.+Nawijnauthor=A.+Alendarauthor=A.+Berns&title=For+better+or+for+worse%3A+the+role+of+Pim+oncogenes+in+tumorigenesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">For better or for worse: the role of Pim oncogenes in tumorigenesis</span></div><div class="casAuthors">Nawijn, Martijn C.; Alendar, Andrej; Berns, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-34</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Pim oncogenes are overexpressed in a wide range of tumors from a hematol. and epithelial origin.  Pim genes encode serine/threonine kinases that have been shown to counteract the increased sensitivity to apoptosis induction that is assocd. with MYC-driven tumorigenesis.  Recently, considerable progress has been made in characterizing the pathways of PIM-mediated survival signalling.  Given the unique structure of their active site and the minimal phenotype of mice mutant for all Pim family members, these oncogenes might be promising targets for highly specific and selective drugs with favorable toxicity profiles.  In this Review, we discuss the physiol. functions and oncogenic activities of Pim kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu23UDfObMZrVg90H21EOLACvtfcHk0ljkKEJ05v2NGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFCjsrjE&md5=fe08e70454ca8c4d808715db87d195e1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnrc2986&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2986%26sid%3Dliteratum%253Aachs%26aulast%3DNawijn%26aufirst%3DM.%2BC.%26aulast%3DAlendar%26aufirst%3DA.%26aulast%3DBerns%26aufirst%3DA.%26atitle%3DFor%2520better%2520or%2520for%2520worse%253A%2520the%2520role%2520of%2520Pim%2520oncogenes%2520in%2520tumorigenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2011%26volume%3D11%26spage%3D23%26epage%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Chen, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redkar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taverna, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandhi, V.</span><span> </span><span class="NLM_article-title">Mechanisms of cytotoxicity to Pim kinase inhibitor, SGI-1776, in acute myeloid leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">702</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2011&pages=693-702&author=L.+S.+Chenauthor=S.+Redkarauthor=P.+Tavernaauthor=J.+E.+Cortesauthor=V.+Gandhi&title=Mechanisms+of+cytotoxicity+to+Pim+kinase+inhibitor%2C+SGI-1776%2C+in+acute+myeloid+leukemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%2BS.%26aulast%3DRedkar%26aufirst%3DS.%26aulast%3DTaverna%26aufirst%3DP.%26aulast%3DCortes%26aufirst%3DJ.%2BE.%26aulast%3DGandhi%26aufirst%3DV.%26atitle%3DMechanisms%2520of%2520cytotoxicity%2520to%2520Pim%2520kinase%2520inhibitor%252C%2520SGI-1776%252C%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DBlood%26date%3D2011%26volume%3D118%26spage%3D693%26epage%3D702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Fathi, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arowojolu, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swinnen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajkhowa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Small, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marmsater, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gross, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallan, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M.</span><span> </span><span class="NLM_article-title">A potential therapeutic target for FLT3-ITD AML: PIM1 kinase</span> <span class="citation_source-journal">Leuk. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2Fj.leukres.2011.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=21802138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVemtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2012&pages=224-231&author=A.+T.+Fathiauthor=O.+Arowojoluauthor=I.+Swinnenauthor=T.+Satoauthor=T.+Rajkhowaauthor=D.+Smallauthor=F.+Marmsaterauthor=J.+E.+Robinsonauthor=S.+D.+Grossauthor=M.+Martinsonauthor=S.+Allenauthor=N.+C.+Kallanauthor=M.+Levis&title=A+potential+therapeutic+target+for+FLT3-ITD+AML%3A+PIM1+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A potential therapeutic target for FLT3-ITD AML: PIM1 kinase</span></div><div class="casAuthors">Fathi, Amir T.; Arowojolu, Omotayo; Swinnen, Ian; Sato, Takashi; Rajkhowa, Trivikram; Small, Donald; Marmsater, Fredrik; Robinson, John E.; Gross, Stefan David; Martinson, Matthew; Allen, Shelley; Kallan, Nicholas C.; Levis, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">224-231</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Patients with acute myeloid leukemia (AML) and a FLT3 internal tandem duplication (ITD) mutation have a poor prognosis, and FLT3 inhibitors are now under clin. investigation.  PIM1, a serine/threonine kinase, is up-regulated in FLT3-ITD AML and may be involved in FLT3-mediated leukemogenesis.  We employed a PIM1 inhibitor, AR00459339 (Array Biopharma Inc.), to investigate the effect of PIM1 inhibition in FLT3-mutant AML.  Like FLT3 inhibitors, AR00459339 was preferentially cytotoxic to FLT3-ITD cells, as demonstrated in the MV4-11, Molm-14, and TF/ITD cell lines, as well as 12 FLT3-ITD primary samples.  Unlike FLT3 inhibitors, AR00459339 did not suppress phosphorylation of FLT3, but did promote the de-phosphorylation of downstream FLT3 targets, STAT5, AKT, and BAD.  Combining AR00459339 with a FLT3 inhibitor resulted in additive to mildly synergistic cytotoxic effects.  AR00459339 was cytotoxic to FLT3-ITD samples from patients with secondary resistance to FLT3 inhibitors, suggesting a novel benefit to combining these agents.  We conclude that PIM1 appears to be closely assocd. with FLT3 signaling, and that inhibition of PIM1 may hold therapeutic promise, either as monotherapy, or by overcoming resistance to FLT3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBBVHeB4yqFbVg90H21EOLACvtfcHk0lhklPgzQAJbiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVemtQ%253D%253D&md5=e7430db23fe996d45fe6a0a55ec040b3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2011.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2011.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DFathi%26aufirst%3DA.%2BT.%26aulast%3DArowojolu%26aufirst%3DO.%26aulast%3DSwinnen%26aufirst%3DI.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DRajkhowa%26aufirst%3DT.%26aulast%3DSmall%26aufirst%3DD.%26aulast%3DMarmsater%26aufirst%3DF.%26aulast%3DRobinson%26aufirst%3DJ.%2BE.%26aulast%3DGross%26aufirst%3DS.%2BD.%26aulast%3DMartinson%26aufirst%3DM.%26aulast%3DAllen%26aufirst%3DS.%26aulast%3DKallan%26aufirst%3DN.%2BC.%26aulast%3DLevis%26aufirst%3DM.%26atitle%3DA%2520potential%2520therapeutic%2520target%2520for%2520FLT3-ITD%2520AML%253A%2520PIM1%2520kinase%26jtitle%3DLeuk.%2520Res.%26date%3D2012%26volume%3D36%26spage%3D224%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Sundman-Engberg, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tidefelt, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paul, C.</span><span> </span><span class="NLM_article-title">Intracellular concentrations of mitoxantrone in leukemic cells in vitro vs in vivo</span> <span class="citation_source-journal">Leuk. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">347</span><span class="NLM_x">–</span> <span class="NLM_lpage">352</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2F0145-2126%2893%2990022-D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=8487583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK3sXltVygt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1993&pages=347-352&author=B.+Sundman-Engbergauthor=U.+Tidefeltauthor=A.+Gruberauthor=C.+Paul&title=Intracellular+concentrations+of+mitoxantrone+in+leukemic+cells+in+vitro+vs+in+vivo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular concentrations of mitoxantrone in leukemic cells in vitro vs in vivo</span></div><div class="casAuthors">Sundman-Engberg, Britt; Tidefelt, Ulf; Gruber, Astrid; Paul, Christer</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">347-52</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    </div><div class="casAbstract">The aim of this study was to det. the intracellular pharmacokinetics of mitoxantrone in vivo and to use these results to establish how leukemic cells should be incubated to perform clin. relevant in vitro studies of this drug.  Blood samples were obtained from 11 patients with acute nonlymphoblastic leukemia at certain intervals up to 20 h after the infusion of mitoxantrone 12 mg/m2.  Plasma and leukemic cells were sepd. and the drug concns. were detd. with HPLC.  Before treatment, leukemic cells from 12 patients were incubated with 0.02, 0.05, 0.1, 0.2 and 1.0 μM mitoxantrone for 1-4 h and thereafter cultured in suspension culture for 20 h; during this time cell samples were taken at certain intervals for drug detn.  In cells incubated with 0.05 and 0.2 μM mitoxantrone the cytotoxic effect was measured with the Disk assay after cultivation for 4-5 days.  In vivo, the intracellular levels exceeded the plasma concns. already at the end of infusion and after 2 h the intracellular concns. were 200-300 times higher than in plasma.  In vitro, the intracellular steady state level of mitoxantrone was reached after 1-2 h and there was a pronounced intracellular retention even after 20 h culture in drug-free medium.  Incubation with 0.05 μM during 1 h gave intracellular concns. of mitoxantrone similar to those achieved in vivo.  This incubation concn. gave a mean cytotoxic effect of 53% living cells measured with the Disk assay, which gives good possibilities to discriminate between mitoxantrone-sensitive and unsensitive cells.  Thus, exposing leukemic cells in vitro for in vivo mimicking mitoxantrone concns. could increase the clin. relevance of predictive assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgFiG859g_s7Vg90H21EOLACvtfcHk0lhklPgzQAJbiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltVygt78%253D&md5=8b8f2eaebfa6d4c6186a5b0cb816d0a3</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2F0145-2126%2893%2990022-D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0145-2126%252893%252990022-D%26sid%3Dliteratum%253Aachs%26aulast%3DSundman-Engberg%26aufirst%3DB.%26aulast%3DTidefelt%26aufirst%3DU.%26aulast%3DGruber%26aufirst%3DA.%26aulast%3DPaul%26aufirst%3DC.%26atitle%3DIntracellular%2520concentrations%2520of%2520mitoxantrone%2520in%2520leukemic%2520cells%2520in%2520vitro%2520vs%2520in%2520vivo%26jtitle%3DLeuk.%2520Res.%26date%3D1993%26volume%3D17%26spage%3D347%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Robinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pheneger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patrice, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, A. G.</span><span> </span><span class="NLM_article-title">A Dual PIM 1/3 Kinase Inhibitor Demonstrates Efficacy in Murine Models of Lupus and Multiple Sclerosis</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">188</span><span class="NLM_x">, </span> <span class="NLM_fpage">199.9</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=188&publication_year=2012&pages=199.9&author=J.+Robinsonauthor=J.+Phenegerauthor=L.+Patriceauthor=D.+Wrightauthor=S.+Brownauthor=F.+Sullivanauthor=A.+G.+Davis&title=A+Dual+PIM+1%2F3+Kinase+Inhibitor+Demonstrates+Efficacy+in+Murine+Models+of+Lupus+and+Multiple+Sclerosis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DPheneger%26aufirst%3DJ.%26aulast%3DPatrice%26aufirst%3DL.%26aulast%3DWright%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DSullivan%26aufirst%3DF.%26aulast%3DDavis%26aufirst%3DA.%2BG.%26atitle%3DA%2520Dual%2520PIM%25201%252F3%2520Kinase%2520Inhibitor%2520Demonstrates%2520Efficacy%2520in%2520Murine%2520Models%2520of%2520Lupus%2520and%2520Multiple%2520Sclerosis%26jtitle%3DJ.%2520Immunol.%26date%3D2012%26volume%3D188%26spage%3D199.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Raju, T. N.</span><span> </span><span class="NLM_article-title">The Nobel chronicles. 1988: James Whyte Black, (b 1924), Gertrude Elion (1918–99), and George H Hitchings (1905–98)</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">355</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2000&pages=1022&author=T.+N.+Raju&title=The+Nobel+chronicles.+1988%3A+James+Whyte+Black%2C+%28b+1924%29%2C+Gertrude+Elion+%281918%E2%80%9399%29%2C+and+George+H+Hitchings+%281905%E2%80%9398%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRaju%26aufirst%3DT.%2BN.%26atitle%3DThe%2520Nobel%2520chronicles.%25201988%253A%2520James%2520Whyte%2520Black%252C%2520%2528b%25201924%2529%252C%2520Gertrude%2520Elion%2520%25281918%25E2%2580%259399%2529%252C%2520and%2520George%2520H%2520Hitchings%2520%25281905%25E2%2580%259398%2529%26jtitle%3DLancet%26date%3D2000%26volume%3D355%26spage%3D1022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Altschul, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gish, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipman, D. J.</span><span> </span><span class="NLM_article-title">Basic local alignment search tool</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">215</span><span class="NLM_x">, </span> <span class="NLM_fpage">403</span><span class="NLM_x">–</span> <span class="NLM_lpage">410</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2FS0022-2836%2805%2980360-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=2231712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK3MXitVGmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=1990&pages=403-410&author=S.+F.+Altschulauthor=W.+Gishauthor=W.+Millerauthor=E.+W.+Myersauthor=D.+J.+Lipman&title=Basic+local+alignment+search+tool"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Basic local alignment search tool</span></div><div class="casAuthors">Altschul, Stephen F.; Gish, Warren; Miller, Webb; Myers, Eugene W.; Lipman, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">403-10</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A new approach to rapid sequence comparison, basic local alignment search tool (BLAST), directly approximates alignments that optimize a measure of local similarity, the maximal segment pair (MSP) score.  Recent math. results on the stochastic properties of MSP scores allow an anal. of the performance of this method as well as the statistical significance of alignments it generates.  The basic algorithm is simple and robust; it can be implemented in a no. of ways and applied in a variety of contexts including straightforward DNA and protein sequence database searches, motif searches, gene identification searches, and in the anal. of multiple regions of similarity in long DNA sequences.  In addn. to its flexibility and tractability to math. anal., BLAST is an order of magnitude faster than existing sequence comparison tools of comparable sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGEHw-q53MfLVg90H21EOLACvtfcHk0lgX2v3sqn4KXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXitVGmsA%253D%253D&md5=009d2323eb82f0549356880e1101db16</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2805%2980360-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252805%252980360-2%26sid%3Dliteratum%253Aachs%26aulast%3DAltschul%26aufirst%3DS.%2BF.%26aulast%3DGish%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DW.%26aulast%3DMyers%26aufirst%3DE.%2BW.%26aulast%3DLipman%26aufirst%3DD.%2BJ.%26atitle%3DBasic%2520local%2520alignment%2520search%2520tool%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1990%26volume%3D215%26spage%3D403%26epage%3D410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Huse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">The conformational plasticity of protein kinases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">275</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2FS0092-8674%2802%2900741-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=12015977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2002&pages=275-282&author=M.+Huseauthor=J.+Kuriyan&title=The+conformational+plasticity+of+protein+kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The conformational plasticity of protein kinases</span></div><div class="casAuthors">Huse, Morgan; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">275-282</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with 48 refs.  Protein kinases operate in a large no. of distinct signaling pathways, where the tight regulation of their catalytic activity is crucial to the development and maintenance of eukaryotic organisms.  The catalytic domains of different kinases adopt strikingly similar structures when they are active.  By contrast, crystal structures of inactive kinases have revealed a marked plasticity in the kinase domain that allows the adoption of distinct conformations in response to interactions with specific regulatory domains or proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJrn7F3h4ej7Vg90H21EOLACvtfcHk0lgX2v3sqn4KXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D&md5=cd56edae94ad239f2ce7a5d6c8930cbb</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900741-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900741-9%26sid%3Dliteratum%253Aachs%26aulast%3DHuse%26aufirst%3DM.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DThe%2520conformational%2520plasticity%2520of%2520protein%2520kinases%26jtitle%3DCell%26date%3D2002%26volume%3D109%26spage%3D275%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Kelley, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutcliffe, M. J.</span><span> </span><span class="NLM_article-title">An automated approach for clustering an ensemble of NMR-derived protein structures into conformationally related subfamilies</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1063</span><span class="NLM_x">–</span> <span class="NLM_lpage">1065</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1093%2Fprotein%2F9.11.1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=8961360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK28XnsFamt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=1063-1065&author=L.+A.+Kelleyauthor=S.+P.+Gardnerauthor=M.+J.+Sutcliffe&title=An+automated+approach+for+clustering+an+ensemble+of+NMR-derived+protein+structures+into+conformationally+related+subfamilies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">An automated approach for clustering an ensemble of NMR-derived protein structures into conformationally related subfamilies</span></div><div class="casAuthors">Kelley, Lawrence A.; Gardner, Stephen P.; Sutcliffe, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1063-1065</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A computer program has been developed that automatically, systematically, and rapidly clusters an ensemble of structures into a set of conformationally related subfamilies, and selects a representative structure from each cluster.  The program uses the method of av. linkage to define how clusters are built up, followed by the application of a penalty function that seeks to minimize simultaneously the no. of clusters and the spread across each cluster.  This program, known as NMRCLUST, is available via the World Wide Web (URL:http://neon.chem.le.ac.uk/) and by anonymous ftp from ftp.oxmol.co.uk.  Although developed for the anal. of NMR-derived structures, the program can be used to automatically cluster any data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1EjchxnvGGrVg90H21EOLACvtfcHk0lgX2v3sqn4KXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XnsFamt74%253D&md5=6940b36a9dcbba4dd8352df595737cb2</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F9.11.1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F9.11.1063%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DL.%2BA.%26aulast%3DGardner%26aufirst%3DS.%2BP.%26aulast%3DSutcliffe%26aufirst%3DM.%2BJ.%26atitle%3DAn%2520automated%2520approach%2520for%2520clustering%2520an%2520ensemble%2520of%2520NMR-derived%2520protein%2520structures%2520into%2520conformationally%2520related%2520subfamilies%26jtitle%3DProtein%2520Eng.%26date%3D1996%26volume%3D9%26spage%3D1063%26epage%3D1065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Pettersen, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">UCSF Chimera—a visualization system for exploratory research and analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">–</span> <span class="NLM_lpage">1612</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94a+visualization+system+for+exploratory+research+and+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594a%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaminski, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapp, C. S.</span><span> </span><span class="NLM_article-title">Force field validation using protein side chain prediction</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">11673</span><span class="NLM_x">–</span> <span class="NLM_lpage">11680</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp021564n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2002&pages=11673-11680&author=M.+P.+Jacobsonauthor=G.+A.+Kaminskiauthor=R.+A.+Friesnerauthor=C.+S.+Rapp&title=Force+field+validation+using+protein+side+chain+prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjp021564n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp021564n%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DKaminski%26aufirst%3DG.%2BA.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26atitle%3DForce%2520field%2520validation%2520using%2520protein%2520side%2520chain%2520prediction%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D2002%26volume%3D106%26spage%3D11673%26epage%3D11680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Improved Methods for Side Chain and Loop Predictions via the Protein Local Optimization Program:Variable Dielectric Model for Implicitly Improving the Treatment of Polarization Effects</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">2108</span><span class="NLM_x">–</span> <span class="NLM_lpage">2119</span></span><div class="citationLinks">[<a href="/doi/10.1021/ct700166f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKqtr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=2108-2119&author=K.+Zhuauthor=M.+R.+Shirtsauthor=R.+A.+Friesner&title=Improved+Methods+for+Side+Chain+and+Loop+Predictions+via+the+Protein+Local+Optimization+Program%3AVariable+Dielectric+Model+for+Implicitly+Improving+the+Treatment+of+Polarization+Effects"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Methods for Side Chain and Loop Predictions via the Protein Local Optimization Program: Variable Dielectric Model for Implicitly Improving the Treatment of Polarization Effects</span></div><div class="casAuthors">Zhu, Kai; Shirts, Michael R.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2108-2119</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper presents significant improvements in both accuracy and computational efficiency of protein side chain and loop predictions using the Protein Local Optimization Program (PLOP).  The authors introduce a novel energy model in which the internal dielec. const. of the protein is allowed to vary as a function of the interacting residues and present a phys. rationale for this model.  Using this model, the authors achieve qual. improvements in the accuracy of side chain predictions with respect to exptl. crystal structure and substantially reduce the RMSDs for loop predictions, particularly those predictions involving charged side chains.  For the single side chain prediction of lysine, 40% of the errors are eliminated, and the accuracy increases from 62.6% to 76.8%.  The errors in glutamate and aspartate predictions are reduced by 19% and 24%, resp.  When applied to a set of 240 loop predictions with 6, 8, 10, and 13 residue of loop length, this new model yields unprecedented accuracies with av. backbone root-mean-square deviations of 0.39 Å, 0.68 Å, 0.80 Å, and 1.00 Å for 6, 8, 10, and 13 residue loops, resp.  The authors also describe a series of tech. improvements in the PLOP simulation algorithms, which lead to a speedup of a factor of 2-4 in loop predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWwiam8R8KjrVg90H21EOLACvtfcHk0liYp6jTbhASqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKqtr%252FO&md5=e591b5093032f15199883ca45544853f</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fct700166f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700166f%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DImproved%2520Methods%2520for%2520Side%2520Chain%2520and%2520Loop%2520Predictions%2520via%2520the%2520Protein%2520Local%2520Optimization%2520Program%253AVariable%2520Dielectric%2520Model%2520for%2520Implicitly%2520Improving%2520the%2520Treatment%2520of%2520Polarization%2520Effects%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2007%26volume%3D3%26spage%3D2108%26epage%3D2119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Huang, N.; Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Binding-site assessment by virtual fragment screening</span>.  <span class="citation_source-journal">PLoS One</span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e10109</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Huang%2C+N.%3B+Jacobson%2C+M.+P.+Binding-site+assessment+by+virtual+fragment+screening.+PLoS+One5%2C+e10109."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DBinding-site%2520assessment%2520by%2520virtual%2520fragment%2520screening%26jtitle%3DPLoS%2520One%26volume%3D5%26spage%3De10109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Benchmarking sets for molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6789</span><span class="NLM_x">–</span> <span class="NLM_lpage">6801</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&author=N.+Huangauthor=B.+K.+Shoichetauthor=J.+J.+Irwin&title=Benchmarking+sets+for+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0liYp6jTbhASqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DBenchmarking%2520sets%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6789%26epage%3D6801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheatham, T. E.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K. M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onufriev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, R. J.</span><span> </span><span class="NLM_article-title">The Amber biomolecular simulation programs</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1668</span><span class="NLM_x">–</span> <span class="NLM_lpage">1688</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fjcc.20290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=16200636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1668-1688&author=D.+A.+Caseauthor=T.+E.+Cheathamauthor=T.+Dardenauthor=H.+Gohlkeauthor=R.+Luoauthor=K.+M.+Merzauthor=A.+Onufrievauthor=C.+Simmerlingauthor=B.+Wangauthor=R.+J.+Woods&title=The+Amber+biomolecular+simulation+programs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The amber biomolecular simulation programs</span></div><div class="casAuthors">Case, David A.; Cheatham, Thomas E., III; Darden, Tom; Gohlke, Holger; Luo, Ray; Merz, Kenneth M., Jr.; Onufriev, Alexey; Simmerling, Carlos; Wang, Bing; Woods, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1668-1688</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development, current features, and some directions for future development of the Amber package of computer programs.  This package evolved from a program that was constructed in the late 1970s to do Assisted Model Building with Energy Refinement, and now contains a group of programs embodying a no. of powerful tools of modern computational chem., focused on mol. dynamics and free energy calcns. of proteins, nucleic acids, and carbohydrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0tZP_Umb7rVg90H21EOLACvtfcHk0liYp6jTbhASqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM&md5=93be29ff894bab96c783d24e9886c7d0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20290%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DOnufriev%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWoods%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520Amber%2520biomolecular%2520simulation%2520programs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1668%26epage%3D1688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Hornak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strockbine, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span> </span><span class="NLM_article-title">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">712</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fprot.21123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=16981200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=712-725&author=V.+Hornakauthor=R.+Abelauthor=A.+Okurauthor=B.+Strockbineauthor=A.+Roitbergauthor=C.+Simmerling&title=Comparison+of+multiple+Amber+force+fields+and+development+of+improved+protein+backbone+parameters"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span></div><div class="casAuthors">Hornak, Viktor; Abel, Robert; Okur, Asim; Strockbine, Bentley; Roitberg, Adrian; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">712-725</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The ff94 force field that is commonly assocd. with the Amber simulation package is one of the most widely used parameter sets for biomol. simulation.  After a decade of extensive use and testing, limitations in this force field, such as over-stabilization of α-helixes, were reported by the authors and other researchers.  This led to a no. of attempts to improve these parameters, resulting in a variety of "Amber" force fields and significant difficulty in detg. which should be used for a particular application.  The authors show that several of these continue to suffer from inadequate balance between different secondary structure elements.  In addn., the approach used in most of these studies neglected to account for the existence in Amber of two sets of backbone .vphi./ψ dihedral terms.  This led to parameter sets that provide unreasonable conformational preferences for glycine.  The authors report here an effort to improve the .vphi./ψ dihedral terms in the ff99 energy function.  Dihedral term parameters are based on fitting the energies of multiple conformations of glycine and alanine tetrapeptides from high level ab initio quantum mech. calcns.  The new parameters for backbone dihedrals replace those in the existing ff99 force field.  This parameter set, which the authors denote ff99SB, achieves a better balance of secondary structure elements as judged by improved distribution of backbone dihedrals for glycine and alanine with respect to PDB survey data.  It also accomplishes improved agreement with published exptl. data for conformational preferences of short alanine peptides and better accord with exptl. NMR relaxation data of test protein systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmoASzza-Gd7Vg90H21EOLACvtfcHk0lixBQsdX8oLeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM&md5=de683a26eca9e83ae524726e97ac22fa</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fprot.21123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21123%26sid%3Dliteratum%253Aachs%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DOkur%26aufirst%3DA.%26aulast%3DStrockbine%26aufirst%3DB.%26aulast%3DRoitberg%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3DComparison%2520of%2520multiple%2520Amber%2520force%2520fields%2520and%2520development%2520of%2520improved%2520protein%2520backbone%2520parameters%26jtitle%3DProteins%26date%3D2006%26volume%3D65%26spage%3D712%26epage%3D725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lixBQsdX8oLeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Jakalian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jack, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">132</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2000&pages=132-146&author=A.+Jakalianauthor=B.+L.+Bushauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+I.+Method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span></div><div class="casAuthors">Jakalian, Araz; Bush, Bruce L.; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">132-146</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The AM1-BCC method quickly and efficiently generates high-quality at. charges for use in condensed-phase simulations.  The underlying features of the electron distribution including formal charge and delocalization are first captured by AM1 at. charges for the individual mol.  Bond charge corrections (BCCs), which have been parameterized against the HF/6-31G* electrostatic potential (ESP) of a training set of compds. contg. relevant functional groups, are then added using a formalism identical to the consensus BCI (bond charge increment) approach.  As a proof of the concept, we fit BCCs simultaneously to 45 compds. including O-, N-, and S-contg. functionalities, aroms., and heteroaroms., using only 41 BCC parameters.  AM1-BCC yields charge sets of comparable quality to HF/6-31G* ESP-derived charges in a fraction of the time while reducing instabilities in the at. charges compared to direct ESP-fit methods.  We then apply the BCC parameters to a small "test set" consisting of aspirin, D-glucose, and eryodictyol; the AM1-BCC model again provides at. charges of quality comparable with HF/6-31G* RESP charges, as judged by an increase of only 0.01 to 0.02 at. units in the root-mean-square (RMS) error in ESP.  Based on these encouraging results, we intend to parameterize the AM1-BCC model to provide a consistent charge model for any org. or biol. mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruxZWGx7IgLbVg90H21EOLACvtfcHk0lixBQsdX8oLeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D&md5=aa4774739d3491dbc8dd95ed1948d856</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252820000130%252921%253A2%253C132%253A%253AAID-JCC5%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520I.%2520Method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2000%26volume%3D21%26spage%3D132%26epage%3D146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Jakalian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jack, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1623</span><span class="NLM_x">–</span> <span class="NLM_lpage">1641</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+II.+Parameterization+and+validation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0lh4G81AVsRhfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520II.%2520Parameterization%2520and%2520validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1623%26epage%3D1641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Luo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">David, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Accelerated Poisson-Boltzmann calculations for static and dynamic systems</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1244</span><span class="NLM_x">–</span> <span class="NLM_lpage">1253</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fjcc.10120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=12210150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD38XmsVKgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1244-1253&author=R.+Luoauthor=L.+Davidauthor=M.+K.+Gilson&title=Accelerated+Poisson-Boltzmann+calculations+for+static+and+dynamic+systems"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Accelerated Poisson-Boltzmann calculations for static and dynamic systems</span></div><div class="casAuthors">Luo, Ray; David, Laurent; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1244-1253</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We report an efficient implementation of the finite-difference Poisson-Boltzmann solvent model based on the Modified Incomplete Cholsky Conjugate Gradient algorithm, which gives rather impressive performance for both static and dynamic systems.  This is achieved by implementing the algorithm with Eisenstat's two optimizations, utilizing the electrostatic update in simulations, and applying prudent approxns., including: relaxing the convergence criterion, not updating Poisson-Boltzmann-related forces every step, and using electrostatic focusing.  It is also possible to markedly accelerate the supporting routines that are used to set up the calcns. and to obtain energies and forces.  The resulting finite difference Poisson-Boltzmann method delivers efficiency comparable to the distance-dependent dielec. model for a system tested, HIV Protease, making it a strong candidate for soln.-phase mol. dynamics simulations.  Further, the finite difference method includes all intra-solute electrostatic interactions, whereas the distance dependent dielec. calcns. use a 15-Å cutoff.  The speed of our numerical finite difference method is comparable to that of the pair-wise Generalized Born approxn. to the Poisson-Boltzmann method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopCqAOEfmOgbVg90H21EOLACvtfcHk0lh4G81AVsRhfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmsVKgsLo%253D&md5=7ce67345061bdd6fcfae91be1650a578</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10120%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DDavid%26aufirst%3DL.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DAccelerated%2520Poisson-Boltzmann%2520calculations%2520for%2520static%2520and%2520dynamic%2520systems%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1244%26epage%3D1253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Sitkoff, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">Accurate Calculation of Hydration Free Energies Using Macroscopic Solvent Models</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">1978</span><span class="NLM_x">–</span> <span class="NLM_lpage">1988</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100058a043" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK2cXhsVGrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1994&pages=1978-1988&author=D.+Sitkoffauthor=K.+A.+Sharpauthor=B.+Honig&title=Accurate+Calculation+of+Hydration+Free+Energies+Using+Macroscopic+Solvent+Models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate Calculation of Hydration Free Energies Using Macroscopic Solvent Models</span></div><div class="casAuthors">Sitkoff, Doree; Sharp, Kim A.; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1978-88</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">A method and parametrization scheme which allow fast and accurate calcns. of hydration free energies are described.  The solute is treated as a polarizable cavity of a shape defined by the mol. surface, contg. point charges at the location of at. nuclei.  Electrostatic contributions to solvation are derived from finite difference solns. of the Poisson equation (FDPB method).  Nonpolar (cavity/van der Waals) energies are added as a surface area dependent term, with a single surface tension coeff. (γ) derived from hydrocarbon soly. in water.  At. charges and radii are obtained by modifying existing force-field- or quantum-mech.-derived values, by fitting to exptl. solvation energies of small org. mols.  A new, simple parameter set (parameters for solvation energy, PARSE) is developed specifically for the FDPB/γ method, by choosing at. charges and radii which reproduce the estd. contributions to solvation of simple functional groups.  The PARSE parameters reproduce hydration free energies for a test set of 67 mols. with an av. error of 0.4 kcal/mol.  For amino acid side chain and peptide backbone analogs the av. error is only 0.1 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruCKnU2H8vArVg90H21EOLACvtfcHk0lh4G81AVsRhfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsVGrsrs%253D&md5=78ad4721dced796d92eb27c15f98aba5</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fj100058a043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100058a043%26sid%3Dliteratum%253Aachs%26aulast%3DSitkoff%26aufirst%3DD.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DAccurate%2520Calculation%2520of%2520Hydration%2520Free%2520Energies%2520Using%2520Macroscopic%2520Solvent%2520Models%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1994%26volume%3D98%26spage%3D1978%26epage%3D1988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Erijman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dantes, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernheim, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shifman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peleg, Y.</span><span> </span><span class="NLM_article-title">Transfer-PCR (TPCR): a highway for DNA cloning and protein engineering</span> <span class="citation_source-journal">J. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">175</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2Fj.jsb.2011.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=21515384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslWnu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2011&pages=171-177&author=A.+Erijmanauthor=A.+Dantesauthor=R.+Bernheimauthor=J.+M.+Shifmanauthor=Y.+Peleg&title=Transfer-PCR+%28TPCR%29%3A+a+highway+for+DNA+cloning+and+protein+engineering"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Transfer-PCR (TPCR): A highway for DNA cloning and protein engineering</span></div><div class="casAuthors">Erijman, Ariel; Dantes, Ada; Bernheim, Reut; Shifman, Julia M.; Peleg, Yoav</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">171-177</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">DNA cloning and protein engineering are basic methodologies employed for various applications in all life-science disciplines.  Manipulations of DNA however, could be a lengthy process that slows down subsequent expts.  To facilitate both DNA cloning and protein engineering, Transfer-PCR (TPCR) is presented, a novel approach that integrates in a single tube, PCR amplification of the target DNA from an origin vector and its subsequent integration into the destination vector.  TPCR can be applied for incorporation of DNA fragments into any desired position within a circular plasmid without the need for purifn. of the intermediate PCR product and without the use of any com. kit.  Several examples demonstrate the applicability of the TPCR platform for both DNA cloning and for multiple-site targeted mutagenesis.  In both cases, the TPCR reaction was most efficient within a narrow range of primer concns.  In mutagenesis, TPCR was primarily advantageous for generation of combinatorial libraries of targeted mutants but could be also applied to generation of variants with specific multiple mutations throughout the target gene.  Adaptation of the TPCR platform should facilitate, simplify and significantly reduce time and costs for diverse protein structure and functional studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojVuamZPNVCrVg90H21EOLACvtfcHk0lh4G81AVsRhfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslWnu70%253D&md5=8235dc3a1b08a4c5eb2e5d6f1054aab1</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2011.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2011.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DErijman%26aufirst%3DA.%26aulast%3DDantes%26aufirst%3DA.%26aulast%3DBernheim%26aufirst%3DR.%26aulast%3DShifman%26aufirst%3DJ.%2BM.%26aulast%3DPeleg%26aufirst%3DY.%26atitle%3DTransfer-PCR%2520%2528TPCR%2529%253A%2520a%2520highway%2520for%2520DNA%2520cloning%2520and%2520protein%2520engineering%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D175%26spage%3D171%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Qian, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Studts, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barringer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kronkaitis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baptiste, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mische, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farmer, B.</span><span> </span><span class="NLM_article-title">Expression, purification, crystallization and preliminary crystallographic analysis of human Pim-1 kinase</span> <span class="citation_source-journal">Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">96</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1107%2FS1744309104029963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=16508102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhvVGnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=96-99&author=K.+C.+Qianauthor=J.+Studtsauthor=L.+Wangauthor=K.+Barringerauthor=A.+Kronkaitisauthor=C.+Pengauthor=A.+Baptisteauthor=R.+LaFranceauthor=S.+Mischeauthor=B.+Farmer&title=Expression%2C+purification%2C+crystallization+and+preliminary+crystallographic+analysis+of+human+Pim-1+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification, crystallization and preliminary crystallographic analysis of human Pim-1 kinase</span></div><div class="casAuthors">Qian, Kevin C.; Studts, Joey; Wang, Lian; Barringer, Kevin; Kronkaitis, Anthony; Peng, Charline; Baptiste, Alistair; LaFrance, Roger; Mische, Sheenah; Farmer, Bennett</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology and Crystallization Communications</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">F61</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">96-99</span>CODEN:
                <span class="NLM_cas:coden">ACSFCL</span>;
        ISSN:<span class="NLM_cas:issn">1744-3091</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Pim kinases, including Pim-1, Pim-2, and Pim-3, belong to a distinctive serine/threonine protein kinase family.  They are involved in cytokine-induced signal transduction and the development of lymphoid malignancies.  Their kinase domains are highly homologous to one another, but share low sequence identity to other kinases.  Specifically, there are 2 Pro residues in the conserved hinge-region sequence ERPXPX sepd. by a residue that is non-conserved among Pim kinases.  Here, full-length human Pim-1 kinase (1-313) was cloned and expressed in Escherichia coli as a GST-fusion protein and truncated to Pim-1 (14-313) by thrombin digestion during purifn.  The Pim-1 (14-313) protein was purified to high homogeneity and monodispersity.  This protein prepn. yielded small crystals in the initial screening and large crystals after optimization.  The large crystals of apo Pim-1 kinase diffracted to 2.1 Å resoln. and belonged to space group P65, with unit-cell parameters a = b = 95.9, c = 80.0 Å, β = 120° and one mol. per asym. unit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoLbLQgcrPUrVg90H21EOLACvtfcHk0lge_fzJoEqlmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhvVGnsg%253D%253D&md5=4cfe5a9f248a6699287f9a96c627f4ac</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2FS1744309104029963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1744309104029963%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DK.%2BC.%26aulast%3DStudts%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBarringer%26aufirst%3DK.%26aulast%3DKronkaitis%26aufirst%3DA.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DBaptiste%26aufirst%3DA.%26aulast%3DLaFrance%26aufirst%3DR.%26aulast%3DMische%26aufirst%3DS.%26aulast%3DFarmer%26aufirst%3DB.%26atitle%3DExpression%252C%2520purification%252C%2520crystallization%2520and%2520preliminary%2520crystallographic%2520analysis%2520of%2520human%2520Pim-1%2520kinase%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2005%26volume%3D61%26spage%3D96%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lge_fzJoEqlmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">DeLano, W. L.</span> <span class="citation_source-book">The PyMOL Molecular Graphics System</span>; <span class="NLM_publisher-name">DeLano Scientific</span>: <span class="NLM_publisher-loc">San Carlos, CA, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2002</span><span class="NLM_x">; </span>computer program.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=W.+L.+DeLano&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26btitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DDeLano%2520Scientific%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 32 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Christine Yueh, Justin Rettenmaier, Bing Xia, David R. Hall, Andrey Alekseenko, Kathryn A. Porter, Krister Barkovich, Gyorgy Keseru, Adrian Whitty, James A. Wells, Sandor Vajda, <span class="NLM_string-name hlFld-ContribAuthor">Dima Kozakov</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase Atlas: Druggability Analysis of Potential Allosteric Sites in Kinases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (14)
                                     , 6512-6524. <a href="https://doi.org/10.1021/acs.jmedchem.9b00089" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00089</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00089%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DKinase%252BAtlas%25253A%252BDruggability%252BAnalysis%252Bof%252BPotential%252BAllosteric%252BSites%252Bin%252BKinases%26aulast%3DYueh%26aufirst%3DChristine%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D14012019%26date%3D05072019%26date%3D04062019%26volume%3D62%26issue%3D14%26spage%3D6512%26epage%3D6524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiaojing Wang, Wesley Blackaby, Vivienne Allen, Grace Ka Yan Chan, Jae H. Chang, Po-Chang Chiang, Coura Diène, Jason Drummond, Steven Do, Eric Fan, Eric B. Harstad, Alastair Hodges, Huiyong Hu, Wei Jia, William Kofie, Aleksandr Kolesnikov, Joseph P. Lyssikatos, Justin Ly, Mizio Matteucci, John G. Moffat, Veerendra Munugalavadla, Jeremy Murray, David Nash, Cameron L. Noland, Geoff Del Rosario, Leanne Ross, Craig Rouse, Andrew Sharpe, Dionysos Slaga, Minghua Sun, Vickie Tsui, Heidi Wallweber, Shang-Fan Yu, <span class="NLM_string-name hlFld-ContribAuthor">Allen J. Ebens</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of Pan-Pim Kinase Activity and Oral Bioavailability Leading to Diaminopyrazole (GDC-0339) for the Treatment of Multiple Myeloma. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (4)
                                     , 2140-2153. <a href="https://doi.org/10.1021/acs.jmedchem.8b01857" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01857</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01857%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252BPan-Pim%252BKinase%252BActivity%252Band%252BOral%252BBioavailability%252BLeading%252Bto%252BDiaminopyrazole%252B%252528GDC-0339%252529%252Bfor%252Bthe%252BTreatment%252Bof%252BMultiple%252BMyeloma%26aulast%3DWang%26aufirst%3DXiaojing%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D27112018%26date%3D20022019%26date%3D04022019%26volume%3D62%26issue%3D4%26spage%3D2140%26epage%3D2153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takeshi  Hasegawa</span>, <span class="hlFld-ContribAuthor ">Masatake  Sugita</span>, <span class="hlFld-ContribAuthor ">Takeshi  Kikuchi</span>, and <span class="hlFld-ContribAuthor ">Fumio  Hirata</span>  . </span><span class="cited-content_cbyCitation_article-title">A Systematic Analysis of the Binding Affinity between the Pim-1 Kinase and Its Inhibitors Based on the MM/3D-RISM/KH Method. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (11)
                                     , 2789-2798. <a href="https://doi.org/10.1021/acs.jcim.7b00158" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00158</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00158%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DA%252BSystematic%252BAnalysis%252Bof%252Bthe%252BBinding%252BAffinity%252Bbetween%252Bthe%252BPim-1%252BKinase%252Band%252BIts%252BInhibitors%252BBased%252Bon%252Bthe%252BMM%25252F3D-RISM%25252FKH%252BMethod%26aulast%3DHasegawa%26aufirst%3DTakeshi%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D22032017%26date%3D01112017%26date%3D27112017%26date%3D11102017%26volume%3D57%26issue%3D11%26spage%3D2789%26epage%3D2798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu  Zhou</span>, <span class="hlFld-ContribAuthor ">Jing  Ma</span>, <span class="hlFld-ContribAuthor ">Xingyu  Lin</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Kaichun  Wu</span>, and <span class="hlFld-ContribAuthor ">Niu  Huang</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Discovery of Novel and Selective 5-Hydroxytryptamine 2B Receptor Antagonists for the Treatment of Irritable Bowel Syndrome. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (2)
                                     , 707-720. <a href="https://doi.org/10.1021/acs.jmedchem.5b01631" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01631</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01631%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDiscovery%252Bof%252BNovel%252Band%252BSelective%252B5-Hydroxytryptamine%252B2B%252BReceptor%252BAntagonists%252Bfor%252Bthe%252BTreatment%252Bof%252BIrritable%252BBowel%252BSyndrome%26aulast%3DZhou%26aufirst%3DYu%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D16102015%26date%3D07012016%26date%3D28012016%26date%3D23122015%26volume%3D59%26issue%3D2%26spage%3D707%26epage%3D720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ran  Cao</span>, <span class="hlFld-ContribAuthor ">Yanli  Wang</span>, and <span class="hlFld-ContribAuthor ">Niu  Huang</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of 2-Acylaminothiophene-3-Carboxamides as Multitarget Inhibitors for BCR-ABL Kinase and Microtubules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (11)
                                     , 2435-2442. <a href="https://doi.org/10.1021/acs.jcim.5b00540" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00540</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00540%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDiscovery%252Bof%252B2-Acylaminothiophene-3-Carboxamides%252Bas%252BMultitarget%252BInhibitors%252Bfor%252BBCR-ABL%252BKinase%252Band%252BMicrotubules%26aulast%3DCao%26aufirst%3DRan%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D14052015%26date%3D04112015%26date%3D23112015%26date%3D26102015%26volume%3D55%26issue%3D11%26spage%3D2435%26epage%3D2442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ran  Cao</span>, <span class="hlFld-ContribAuthor ">Niu  Huang</span>, and <span class="hlFld-ContribAuthor ">Yanli  Wang</span>  . </span><span class="cited-content_cbyCitation_article-title">Evaluation and Application of MD-PB/SA in Structure-Based Hierarchical Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (7)
                                     , 1987-1996. <a href="https://doi.org/10.1021/ci5003203" title="DOI URL">https://doi.org/10.1021/ci5003203</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci5003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci5003203%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DEvaluation%252Band%252BApplication%252Bof%252BMD-PB%25252FSA%252Bin%252BStructure-Based%252BHierarchical%252BVirtual%252BScreening%26aulast%3DCao%26aufirst%3DRan%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D01042014%26date%3D08072014%26date%3D28072014%26date%3D30062014%26volume%3D54%26issue%3D7%26spage%3D1987%26epage%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">William J.  Pitts</span>, <span class="hlFld-ContribAuthor ">Alaric J.  Dyckman</span>. </span><span class="cited-content_cbyCitation_article-title">Multiple Sclerosis. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-43. <a href="https://doi.org/10.1002/0471266949.bmc203.pub2" title="DOI URL">https://doi.org/10.1002/0471266949.bmc203.pub2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc203.pub2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc203.pub2%26sid%3Dliteratum%253Aachs%26atitle%3DMultiple%252BSclerosis%26aulast%3DPitts%26aufirst%3DWilliam%2BJ.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D43%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mona  Al-Mugotir</span>, <span class="hlFld-ContribAuthor ">Jeffrey J.  Lovelace</span>, <span class="hlFld-ContribAuthor ">Joseph  George</span>, <span class="hlFld-ContribAuthor ">Mika  Bessho</span>, <span class="hlFld-ContribAuthor ">Dhananjaya  Pal</span>, <span class="hlFld-ContribAuthor ">Lucas  Struble</span>, <span class="hlFld-ContribAuthor ">Carol  Kolar</span>, <span class="hlFld-ContribAuthor ">Sandeep  Rana</span>, <span class="hlFld-ContribAuthor ">Amarnath  Natarajan</span>, <span class="hlFld-ContribAuthor ">Tadayoshi  Bessho</span>, <span class="hlFld-ContribAuthor ">Gloria E. O.  Borgstahl</span>, . </span><span class="cited-content_cbyCitation_article-title">Selective killing of homologous recombination-deficient cancer cell lines by inhibitors of the RPA:RAD52 protein-protein interaction. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2021,</strong> <em>16 </em>
                                    (3)
                                     , e0248941. <a href="https://doi.org/10.1371/journal.pone.0248941" title="DOI URL">https://doi.org/10.1371/journal.pone.0248941</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0248941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0248941%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DSelective%252Bkilling%252Bof%252Bhomologous%252Brecombination-deficient%252Bcancer%252Bcell%252Blines%252Bby%252Binhibitors%252Bof%252Bthe%252BRPA%25253ARAD52%252Bprotein-protein%252Binteraction%26aulast%3DAl-Mugotir%26aufirst%3DMona%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D3%26spage%3De0248941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hwangseo  Park</span>, <span class="hlFld-ContribAuthor ">Jinwon  Jeon</span>, <span class="hlFld-ContribAuthor ">Kewon  Kim</span>, <span class="hlFld-ContribAuthor ">Soyeon  Choi</span>, <span class="hlFld-ContribAuthor ">Sungwoo  Hong</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Virtual Screening and De Novo Design of PIM1 Inhibitors with Anticancer Activity from Natural Products. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2021,</strong> <em>14 </em>
                                    (3)
                                     , 275. <a href="https://doi.org/10.3390/ph14030275" title="DOI URL">https://doi.org/10.3390/ph14030275</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph14030275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph14030275%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3DStructure-Based%252BVirtual%252BScreening%252Band%252BDe%252BNovo%252BDesign%252Bof%252BPIM1%252BInhibitors%252Bwith%252BAnticancer%252BActivity%252Bfrom%252BNatural%252BProducts%26aulast%3DPark%26aufirst%3DHwangseo%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D3%26spage%3D275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Masatake  Sugita</span>, <span class="hlFld-ContribAuthor ">Itaru  Onishi</span>, <span class="hlFld-ContribAuthor ">Masayuki  Irisa</span>, <span class="hlFld-ContribAuthor ">Norio  Yoshida</span>, <span class="hlFld-ContribAuthor ">Fumio  Hirata</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Recognition and Self-Organization in Life Phenomena Studied by a Statistical Mechanics of Molecular Liquids, the RISM/3D-RISM Theory. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (2)
                                     , 271. <a href="https://doi.org/10.3390/molecules26020271" title="DOI URL">https://doi.org/10.3390/molecules26020271</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26020271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26020271%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMolecular%252BRecognition%252Band%252BSelf-Organization%252Bin%252BLife%252BPhenomena%252BStudied%252Bby%252Ba%252BStatistical%252BMechanics%252Bof%252BMolecular%252BLiquids%25252C%252Bthe%252BRISM%25252F3D-RISM%252BTheory%26aulast%3DSugita%26aufirst%3DMasatake%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D2%26spage%3D271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jincai  Yang</span>, <span class="hlFld-ContribAuthor ">Cheng  Shen</span>, <span class="hlFld-ContribAuthor ">Niu  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting or Pretending: Artificial Intelligence for Protein-Ligand Interactions Lack of Sufficiently Large and Unbiased Datasets. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphar.2020.00069" title="DOI URL">https://doi.org/10.3389/fphar.2020.00069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2020.00069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2020.00069%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DPredicting%252Bor%252BPretending%25253A%252BArtificial%252BIntelligence%252Bfor%252BProtein-Ligand%252BInteractions%252BLack%252Bof%252BSufficiently%252BLarge%252Band%252BUnbiased%252BDatasets%26aulast%3DYang%26aufirst%3DJincai%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Claudia  Busonero</span>, <span class="hlFld-ContribAuthor ">Stefano  Leone</span>, <span class="hlFld-ContribAuthor ">Stefania  Bartoloni</span>, <span class="hlFld-ContribAuthor ">Filippo  Acconcia</span>. </span><span class="cited-content_cbyCitation_article-title">Strategies to degrade estrogen receptor α in primary and ESR1 mutant-expressing metastatic breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Molecular and Cellular Endocrinology</span><span> <strong>2019,</strong> <em>480 </em>, 107-121. <a href="https://doi.org/10.1016/j.mce.2018.10.020" title="DOI URL">https://doi.org/10.1016/j.mce.2018.10.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.mce.2018.10.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.mce.2018.10.020%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520and%2520Cellular%2520Endocrinology%26atitle%3DStrategies%252Bto%252Bdegrade%252Bestrogen%252Breceptor%252B%2525CE%2525B1%252Bin%252Bprimary%252Band%252BESR1%252Bmutant-expressing%252Bmetastatic%252Bbreast%252Bcancer%26aulast%3DBusonero%26aufirst%3DClaudia%26date%3D2019%26volume%3D480%26spage%3D107%26epage%3D121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Claudia  Busonero</span>, <span class="hlFld-ContribAuthor ">Stefano  Leone</span>, <span class="hlFld-ContribAuthor ">Fabrizio  Bianchi</span>, <span class="hlFld-ContribAuthor ">Filippo  Acconcia</span>. </span><span class="cited-content_cbyCitation_article-title">In silico screening for ERα down modulators identifies thioridazine as an anti-proliferative agent in primary, 4OH-tamoxifen-resistant and Y537S ERα-expressing breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Cellular Oncology</span><span> <strong>2018,</strong> <em>41 </em>
                                    (6)
                                     , 677-686. <a href="https://doi.org/10.1007/s13402-018-0400-x" title="DOI URL">https://doi.org/10.1007/s13402-018-0400-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s13402-018-0400-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs13402-018-0400-x%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520Oncology%26atitle%3DIn%252Bsilico%252Bscreening%252Bfor%252BER%2525CE%2525B1%252Bdown%252Bmodulators%252Bidentifies%252Bthioridazine%252Bas%252Ban%252Banti-proliferative%252Bagent%252Bin%252Bprimary%25252C%252B4OH-tamoxifen-resistant%252Band%252BY537S%252BER%2525CE%2525B1-expressing%252Bbreast%252Bcancer%252Bcells%26aulast%3DBusonero%26aufirst%3DClaudia%26date%3D2018%26date%3D2018%26volume%3D41%26issue%3D6%26spage%3D677%26epage%3D686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephanie J.  Ward</span>, <span class="hlFld-ContribAuthor ">Hayley E.  Gratton</span>, <span class="hlFld-ContribAuthor ">Peni  Indrayudha</span>, <span class="hlFld-ContribAuthor ">Camille  Michavila</span>, <span class="hlFld-ContribAuthor ">Rishov  Mukhopadhyay</span>, <span class="hlFld-ContribAuthor ">Sigrun K.  Maurer</span>, <span class="hlFld-ContribAuthor ">Simon G.  Caulton</span>, <span class="hlFld-ContribAuthor ">Jonas  Emsley</span>, <span class="hlFld-ContribAuthor ">Ingrid  Dreveny</span>. </span><span class="cited-content_cbyCitation_article-title">The structure of the deubiquitinase USP15 reveals a misaligned catalytic triad and an open ubiquitin-binding channel. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2018,</strong> <em>293 </em>
                                    (45)
                                     , 17362-17374. <a href="https://doi.org/10.1074/jbc.RA118.003857" title="DOI URL">https://doi.org/10.1074/jbc.RA118.003857</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA118.003857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA118.003857%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DThe%252Bstructure%252Bof%252Bthe%252Bdeubiquitinase%252BUSP15%252Breveals%252Ba%252Bmisaligned%252Bcatalytic%252Btriad%252Band%252Ban%252Bopen%252Bubiquitin-binding%252Bchannel%26aulast%3DWard%26aufirst%3DStephanie%2BJ.%26date%3D2018%26volume%3D293%26issue%3D45%26spage%3D17362%26epage%3D17374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Megan H.  Cox</span>, <span class="hlFld-ContribAuthor ">Parth  Kapoor</span>, <span class="hlFld-ContribAuthor ">Deborah A.  Briggs</span>, <span class="hlFld-ContribAuthor ">Ian D.  Kerr</span>. </span><span class="cited-content_cbyCitation_article-title">Residues contributing to drug transport by ABCG2 are localised to multiple drug-binding pockets. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2018,</strong> <em>475 </em>
                                    (9)
                                     , 1553-1567. <a href="https://doi.org/10.1042/BCJ20170923" title="DOI URL">https://doi.org/10.1042/BCJ20170923</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20170923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20170923%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DResidues%252Bcontributing%252Bto%252Bdrug%252Btransport%252Bby%252BABCG2%252Bare%252Blocalised%252Bto%252Bmultiple%252Bdrug-binding%252Bpockets%26aulast%3DCox%26aufirst%3DMegan%2BH.%26date%3D2018%26date%3D2018%26volume%3D475%26issue%3D9%26spage%3D1553%26epage%3D1567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joonhyeok  Choi</span>, <span class="hlFld-ContribAuthor ">You-Mie  Lee</span>, <span class="hlFld-ContribAuthor ">Jun-Goo  Jee</span>. </span><span class="cited-content_cbyCitation_article-title">Thiopurine Drugs Repositioned as Tyrosinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2018,</strong> <em>19 </em>
                                    (1)
                                     , 77. <a href="https://doi.org/10.3390/ijms19010077" title="DOI URL">https://doi.org/10.3390/ijms19010077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms19010077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms19010077%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DThiopurine%252BDrugs%252BRepositioned%252Bas%252BTyrosinase%252BInhibitors%26aulast%3DChoi%26aufirst%3DJoonhyeok%26date%3D2018%26date%3D2017%26volume%3D19%26issue%3D1%26spage%3D77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">HeggoduG.  Rohit Kumar</span>, <span class="hlFld-ContribAuthor ">Chethan S.  Kumar</span>, <span class="hlFld-ContribAuthor ">Hulihalli N.  Kiran Kumar</span>, <span class="hlFld-ContribAuthor ">Gopal M.  Advi Rao</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of protein kinases by anticancer DNA intercalator, 4-butylaminopyrimido[4′,5′:4,5]thieno(2,3- b )quinoline. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2017,</strong> <em>7 </em>
                                    (3)
                                     , 303-310. <a href="https://doi.org/10.1016/j.apsb.2017.01.001" title="DOI URL">https://doi.org/10.1016/j.apsb.2017.01.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2017.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2017.01.001%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DInhibition%252Bof%252Bprotein%252Bkinases%252Bby%252Banticancer%252BDNA%252Bintercalator%25252C%252B4-butylaminopyrimido%25255B4%2525E2%252580%2525B2%25252C5%2525E2%252580%2525B2%25253A4%25252C5%25255Dthieno%2525282%25252C3-%252Bb%252B%252529quinoline%26aulast%3DRohit%2BKumar%26aufirst%3DHeggoduG.%26date%3D2017%26volume%3D7%26issue%3D3%26spage%3D303%26epage%3D310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lian-Xiang  Luo</span>, <span class="hlFld-ContribAuthor ">Xing-Xing  Fan</span>, <span class="hlFld-ContribAuthor ">Ying  Li</span>, <span class="hlFld-ContribAuthor ">Xia  Peng</span>, <span class="hlFld-ContribAuthor ">Yin-Chun  Ji</span>, <span class="hlFld-ContribAuthor ">Wendy Wen-Luan  Hsiao</span>, <span class="hlFld-ContribAuthor ">Liang  Liu</span>, <span class="hlFld-ContribAuthor ">Elaine Lai-Han  Leung</span>, <span class="hlFld-ContribAuthor ">Xiao-Jun  Yao</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of mitoxantrone as a new inhibitor of ROS1 fusion protein in non-small cell lung cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Med. Chem. Commun.</span><span> <strong>2017,</strong> <em>8 </em>
                                    (3)
                                     , 621-624. <a href="https://doi.org/10.1039/C6MD00643D" title="DOI URL">https://doi.org/10.1039/C6MD00643D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6MD00643D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6MD00643D%26sid%3Dliteratum%253Aachs%26jtitle%3DMed.%2520Chem.%2520Commun.%26atitle%3DIdentification%252Bof%252Bmitoxantrone%252Bas%252Ba%252Bnew%252Binhibitor%252Bof%252BROS1%252Bfusion%252Bprotein%252Bin%252Bnon-small%252Bcell%252Blung%252Bcancer%252Bcells%26aulast%3DLuo%26aufirst%3DLian-Xiang%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D3%26spage%3D621%26epage%3D624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ran  Cao</span>, <span class="hlFld-ContribAuthor ">Yanli  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Molecular Targets for Small-Molecule Drugs with a Ligand-Based Interaction Fingerprint Approach. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (12)
                                     , 1352-1361. <a href="https://doi.org/10.1002/cmdc.201500228" title="DOI URL">https://doi.org/10.1002/cmdc.201500228</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201500228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201500228%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DPredicting%252BMolecular%252BTargets%252Bfor%252BSmall-Molecule%252BDrugs%252Bwith%252Ba%252BLigand-Based%252BInteraction%252BFingerprint%252BApproach%26aulast%3DCao%26aufirst%3DRan%26date%3D2016%26date%3D2015%26volume%3D11%26issue%3D12%26spage%3D1352%26epage%3D1361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benny J.  Evison</span>, <span class="hlFld-ContribAuthor ">Brad E.  Sleebs</span>, <span class="hlFld-ContribAuthor ">Keith G.  Watson</span>, <span class="hlFld-ContribAuthor ">Don R.  Phillips</span>, <span class="hlFld-ContribAuthor ">Suzanne M.  Cutts</span>. </span><span class="cited-content_cbyCitation_article-title">Mitoxantrone, More than Just Another Topoisomerase II Poison. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2016,</strong> <em>36 </em>
                                    (2)
                                     , 248-299. <a href="https://doi.org/10.1002/med.21364" title="DOI URL">https://doi.org/10.1002/med.21364</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21364%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DMitoxantrone%25252C%252BMore%252Bthan%252BJust%252BAnother%252BTopoisomerase%252BII%252BPoison%26aulast%3DEvison%26aufirst%3DBenny%2BJ.%26date%3D2016%26date%3D2015%26volume%3D36%26issue%3D2%26spage%3D248%26epage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pazit  Shaul</span>, <span class="hlFld-ContribAuthor ">Kfir B.  Steinbuch</span>, <span class="hlFld-ContribAuthor ">Eran  Blacher</span>, <span class="hlFld-ContribAuthor ">Reuven  Stein</span>, <span class="hlFld-ContribAuthor ">Micha  Fridman</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring the Effects of Glycosylation and Etherification of the Side Chains of the Anticancer Drug Mitoxantrone. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2015,</strong> <em>10 </em>
                                    (9)
                                     , 1528-1538. <a href="https://doi.org/10.1002/cmdc.201500274" title="DOI URL">https://doi.org/10.1002/cmdc.201500274</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201500274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201500274%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DExploring%252Bthe%252BEffects%252Bof%252BGlycosylation%252Band%252BEtherification%252Bof%252Bthe%252BSide%252BChains%252Bof%252Bthe%252BAnticancer%252BDrug%252BMitoxantrone%26aulast%3DShaul%26aufirst%3DPazit%26date%3D2015%26date%3D2015%26volume%3D10%26issue%3D9%26spage%3D1528%26epage%3D1538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">. </span><span class="cited-content_cbyCitation_article-title">Overview: General Principles. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 15-28. <a href="https://doi.org/10.1201/b18824-3" title="DOI URL">https://doi.org/10.1201/b18824-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1201/b18824-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1201%2Fb18824-3%26sid%3Dliteratum%253Aachs%26atitle%3DOverview%25253A%252BGeneral%252BPrinciples%26date%3D2015%26date%3D2015%26spage%3D15%26epage%3D28%26pub%3DCRC%2520Press%26atitle%3DActive%252BPhytochemicals%252Bfrom%252BChinese%252BHerbal%252BMedicines%26aulast%3DHo%26aufirst%3DWing%26date%3D2015%26date%3D2015%26volume%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mo  Zhu</span>, <span class="hlFld-ContribAuthor ">Likai  Zhou</span>, <span class="hlFld-ContribAuthor ">Yuchao  Yao</span>, <span class="hlFld-ContribAuthor ">Shuai  Li</span>, <span class="hlFld-ContribAuthor ">Mengjiao  Lv</span>, <span class="hlFld-ContribAuthor ">Kerang  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoliu  Li</span>, <span class="hlFld-ContribAuthor ">Hua  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Anticancer activity and DNA binding property of the dimers of triphenylethylene–coumarin hybrid with two amino side chains. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2015,</strong> <em>24 </em>
                                    (6)
                                     , 2314-2324. <a href="https://doi.org/10.1007/s00044-014-1296-2" title="DOI URL">https://doi.org/10.1007/s00044-014-1296-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-014-1296-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-014-1296-2%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DAnticancer%252Bactivity%252Band%252BDNA%252Bbinding%252Bproperty%252Bof%252Bthe%252Bdimers%252Bof%252Btriphenylethylene%2525E2%252580%252593coumarin%252Bhybrid%252Bwith%252Btwo%252Bamino%252Bside%252Bchains%26aulast%3DZhu%26aufirst%3DMo%26date%3D2015%26date%3D2014%26volume%3D24%26issue%3D6%26spage%3D2314%26epage%3D2324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian  Bailly</span>. </span><span class="cited-content_cbyCitation_article-title">Anticancer Properties of Lamellarins. </span><span class="cited-content_cbyCitation_journal-name">Marine Drugs</span><span> <strong>2015,</strong> <em>13 </em>
                                    (3)
                                     , 1105-1123. <a href="https://doi.org/10.3390/md13031105" title="DOI URL">https://doi.org/10.3390/md13031105</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/md13031105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmd13031105%26sid%3Dliteratum%253Aachs%26jtitle%3DMarine%2520Drugs%26atitle%3DAnticancer%252BProperties%252Bof%252BLamellarins%26aulast%3DBailly%26aufirst%3DChristian%26date%3D2015%26date%3D2015%26volume%3D13%26issue%3D3%26spage%3D1105%26epage%3D1123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carmen  Avendaño</span>, <span class="hlFld-ContribAuthor ">J. Carlos  Menéndez</span>. </span><span class="cited-content_cbyCitation_article-title">Anticancer Drugs Acting via Radical Species. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 133-195. <a href="https://doi.org/10.1016/B978-0-444-62649-3.00004-1" title="DOI URL">https://doi.org/10.1016/B978-0-444-62649-3.00004-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-444-62649-3.00004-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-444-62649-3.00004-1%26sid%3Dliteratum%253Aachs%26atitle%3DAnticancer%252BDrugs%252BActing%252Bvia%252BRadical%252BSpecies%26aulast%3DAvenda%25C3%25B1o%26aufirst%3DCarmen%26date%3D2015%26spage%3D133%26epage%3D195%26pub%3DElsevier%26atitle%3DMedicinal%252BChemistry%252Bof%252BAnticancer%252BDrugs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan-Fen  Dai</span>, <span class="hlFld-ContribAuthor ">Xing-Ming  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">A Survey on the Computational Approaches to Identify Drug Targets in the Postgenomic Era. </span><span class="cited-content_cbyCitation_journal-name">BioMed Research International</span><span> <strong>2015,</strong> <em>2015 </em>, 1-9. <a href="https://doi.org/10.1155/2015/239654" title="DOI URL">https://doi.org/10.1155/2015/239654</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2015/239654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2015%2F239654%26sid%3Dliteratum%253Aachs%26jtitle%3DBioMed%2520Research%2520International%26atitle%3DA%252BSurvey%252Bon%252Bthe%252BComputational%252BApproaches%252Bto%252BIdentify%252BDrug%252BTargets%252Bin%252Bthe%252BPostgenomic%252BEra%26aulast%3DDai%26aufirst%3DYan-Fen%26date%3D2015%26volume%3D2015%26spage%3D1%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>, <span class="hlFld-ContribAuthor ">Valeria A.  Glazunova</span>, <span class="hlFld-ContribAuthor ">Lyubov G.  Dezhenkova</span>, <span class="hlFld-ContribAuthor ">Yuri N.  Luzikov</span>, <span class="hlFld-ContribAuthor ">Vladimir N.  Buyanov</span>, <span class="hlFld-ContribAuthor ">Helena M.  Treshalina</span>, <span class="hlFld-ContribAuthor ">Nina A.  Lesnaya</span>, <span class="hlFld-ContribAuthor ">Vladimir I.  Romanenko</span>, <span class="hlFld-ContribAuthor ">Dmitry N.  Kaluzhny</span>, <span class="hlFld-ContribAuthor ">Jan  Balzarini</span>, <span class="hlFld-ContribAuthor ">Keli  Agama</span>, <span class="hlFld-ContribAuthor ">Yves  Pommier</span>, <span class="hlFld-ContribAuthor ">Alexander A.  Shtil</span>, <span class="hlFld-ContribAuthor ">Maria N.  Preobrazhenskaya</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and evaluation of new antitumor 3-aminomethyl-4,11-dihydroxynaphtho[2,3-f]indole-5,10-diones. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>86 </em>, 797-805. <a href="https://doi.org/10.1016/j.ejmech.2014.09.021" title="DOI URL">https://doi.org/10.1016/j.ejmech.2014.09.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2014.09.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2014.09.021%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bevaluation%252Bof%252Bnew%252Bantitumor%252B3-aminomethyl-4%25252C11-dihydroxynaphtho%25255B2%25252C3-f%25255Dindole-5%25252C10-diones%26aulast%3DShchekotikhin%26aufirst%3DAndrey%2BE.%26date%3D2014%26volume%3D86%26spage%3D797%26epage%3D805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jinho  Lee</span>, <span class="hlFld-ContribAuthor ">Kunal N.  More</span>, <span class="hlFld-ContribAuthor ">Seun-Ah  Yang</span>, <span class="hlFld-ContribAuthor ">Victor S.  Hong</span>. </span><span class="cited-content_cbyCitation_article-title">3,5-Bis(aminopyrimidinyl)indole Derivatives: Synthesis and Evaluation of Pim Kinase Inhibitory Activities. </span><span class="cited-content_cbyCitation_journal-name">Bulletin of the Korean Chemical Society</span><span> <strong>2014,</strong> <em>35 </em>
                                    (7)
                                     , 2123-2129. <a href="https://doi.org/10.5012/bkcs.2014.35.7.2123" title="DOI URL">https://doi.org/10.5012/bkcs.2014.35.7.2123</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5012/bkcs.2014.35.7.2123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5012%2Fbkcs.2014.35.7.2123%26sid%3Dliteratum%253Aachs%26jtitle%3DBulletin%2520of%2520the%2520Korean%2520Chemical%2520Society%26atitle%3D3%25252C5-Bis%252528aminopyrimidinyl%252529indole%252BDerivatives%25253A%252BSynthesis%252Band%252BEvaluation%252Bof%252BPim%252BKinase%252BInhibitory%252BActivities%26aulast%3DLee%26aufirst%3DJinho%26date%3D2014%26volume%3D35%26issue%3D7%26spage%3D2123%26epage%3D2129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guanhai  Tan</span>, <span class="hlFld-ContribAuthor ">Yuchao  Yao</span>, <span class="hlFld-ContribAuthor ">Yunjing  Gu</span>, <span class="hlFld-ContribAuthor ">Shuai  Li</span>, <span class="hlFld-ContribAuthor ">Mengjiao  Lv</span>, <span class="hlFld-ContribAuthor ">Kerang  Wang</span>, <span class="hlFld-ContribAuthor ">Hua  Chen</span>, <span class="hlFld-ContribAuthor ">Xiaoliu  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Cytotoxicity and DNA binding property of the dimers of triphenylethylene–coumarin hybrid with one amino side chain. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>24 </em>
                                    (13)
                                     , 2825-2830. <a href="https://doi.org/10.1016/j.bmcl.2014.04.106" title="DOI URL">https://doi.org/10.1016/j.bmcl.2014.04.106</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2014.04.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2014.04.106%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCytotoxicity%252Band%252BDNA%252Bbinding%252Bproperty%252Bof%252Bthe%252Bdimers%252Bof%252Btriphenylethylene%2525E2%252580%252593coumarin%252Bhybrid%252Bwith%252Bone%252Bamino%252Bside%252Bchain%26aulast%3DTan%26aufirst%3DGuanhai%26date%3D2014%26volume%3D24%26issue%3D13%26spage%3D2825%26epage%3D2830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jennifer  FitzGerald</span>, <span class="hlFld-ContribAuthor ">Laura S.  Murillo</span>, <span class="hlFld-ContribAuthor ">Gemma  O'Brien</span>, <span class="hlFld-ContribAuthor ">Enda  O'Connell</span>, <span class="hlFld-ContribAuthor ">Aisling  O'Connor</span>, <span class="hlFld-ContribAuthor ">Kevin  Wu</span>, <span class="hlFld-ContribAuthor ">Guan-Nan  Wang</span>, <span class="hlFld-ContribAuthor ">Michael D.  Rainey</span>, <span class="hlFld-ContribAuthor ">Alessandro  Natoni</span>, <span class="hlFld-ContribAuthor ">Sandra  Healy</span>, <span class="hlFld-ContribAuthor ">Michael  O'Dwyer</span>, <span class="hlFld-ContribAuthor ">Corrado  Santocanale</span>, . </span><span class="cited-content_cbyCitation_article-title">A High Through-Put Screen for Small Molecules Modulating MCM2 Phosphorylation Identifies Ryuvidine as an Inducer of the DNA Damage Response. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2014,</strong> <em>9 </em>
                                    (6)
                                     , e98891. <a href="https://doi.org/10.1371/journal.pone.0098891" title="DOI URL">https://doi.org/10.1371/journal.pone.0098891</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0098891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0098891%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DA%252BHigh%252BThrough-Put%252BScreen%252Bfor%252BSmall%252BMolecules%252BModulating%252BMCM2%252BPhosphorylation%252BIdentifies%252BRyuvidine%252Bas%252Ban%252BInducer%252Bof%252Bthe%252BDNA%252BDamage%252BResponse%26aulast%3DFitzGerald%26aufirst%3DJennifer%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D6%26spage%3De98891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lian  Zhao</span>, <span class="hlFld-ContribAuthor ">Yuchao  Yao</span>, <span class="hlFld-ContribAuthor ">Shuai  Li</span>, <span class="hlFld-ContribAuthor ">Mengjiao  Lv</span>, <span class="hlFld-ContribAuthor ">Hua  Chen</span>, <span class="hlFld-ContribAuthor ">Xiaoliu  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Cytotoxicity and DNA binding property of triphenylethylene–coumarin hybrids with two amino side chains. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>24 </em>
                                    (3)
                                     , 900-904. <a href="https://doi.org/10.1016/j.bmcl.2013.12.084" title="DOI URL">https://doi.org/10.1016/j.bmcl.2013.12.084</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2013.12.084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2013.12.084%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCytotoxicity%252Band%252BDNA%252Bbinding%252Bproperty%252Bof%252Btriphenylethylene%2525E2%252580%252593coumarin%252Bhybrids%252Bwith%252Btwo%252Bamino%252Bside%252Bchains%26aulast%3DZhao%26aufirst%3DLian%26date%3D2014%26volume%3D24%26issue%3D3%26spage%3D900%26epage%3D904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Li</span>, <span class="hlFld-ContribAuthor ">Xiaobo  Wan</span>, <span class="hlFld-ContribAuthor ">Fanqi  Zeng</span>, <span class="hlFld-ContribAuthor ">Yuting  Xie</span>, <span class="hlFld-ContribAuthor ">Yanli  Wang</span>, <span class="hlFld-ContribAuthor ">Wei  Zhang</span>, <span class="hlFld-ContribAuthor ">Li  Li</span>, <span class="hlFld-ContribAuthor ">Niu  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">More than just a GPCR ligand: structure-based discovery of thioridazine derivatives as Pim-1 kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Med. Chem. Commun.</span><span> <strong>2014,</strong> <em>5 </em>
                                    (4)
                                     , 507-511. <a href="https://doi.org/10.1039/C4MD00030G" title="DOI URL">https://doi.org/10.1039/C4MD00030G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00030G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00030G%26sid%3Dliteratum%253Aachs%26jtitle%3DMed.%2520Chem.%2520Commun.%26atitle%3DMore%252Bthan%252Bjust%252Ba%252BGPCR%252Bligand%25253A%252Bstructure-based%252Bdiscovery%252Bof%252Bthioridazine%252Bderivatives%252Bas%252BPim-1%252Bkinase%252Binhibitors%26aulast%3DLi%26aufirst%3DWei%26date%3D2014%26date%3D2014%26volume%3D5%26issue%3D4%26spage%3D507%26epage%3D511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. The Flowchart of Our Computational Modeling Procedure</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of 17 kinase inhibitors investigated in the present work.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. The two-dimensional plot for the kinome-wide selectivity modeling of 17 kinase inhibitors. The higher EF<sub>20</sub> value and the PI value represent the higher prediction accuracy for a given inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. The inhibition of MX on PIM1 measured by HTRF kinase assay. The half-maximum inhibitory concentration (IC<sub>50</sub>) was measured at 7 concentration points by repeating three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structures of the PIM1–MX and the Tau SRE RNA–MX. (A) The detailed view of the ATP-binding site of the PIM1 kinase domain with the MX (yellow in sticks) compared with the predicted binding pose (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FUM">2FUM</a>) of MX (orange in lines). The protein is colored in slate cartoon, and charged residues forming hydrogen bonds with MX are highlighted. (B) Focused view of the tau SRE RNA–MX interaction (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2KGP">2KGP</a>). The MX is shown yellow in sticks, and RNA is shown in slate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MX competitively binds to the PIM1 substrate binding site. (A) The atomistic interactions between the substrate binding site (slate in cartoon) and MX (yellow in sticks). (B) Kinetic study of the interaction of substrate with PIM1. Kinetic studies (<i>n</i> = 2) were performed in the presence of MX at concentrations of 0, 50, and 100 nM, respectively. (C) The overlay of cocrystal structure PIM1–pimtide (PDB code: 2BIL) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=PIM1">PIM1</a>–MX in the substrate binding site. (D) The sequence comparison of pimtide, Pim1 phosphorylation sites of BAD and eIF4B.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/medium/jm-2013-00045y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. MX inhibits the phosphorylation of Bad at Ser112 in K562 and MV4-11 cell lines. (A) Immunoblot analysis of PIM1 in K562 cell line treated with 0.1% vehicle DMSO alone, 0.1, 0.3, 1, 3, 10, 30 μM MX as well as 10 μM SGI-1776 as positive control. Cell were harvested after 12 h and lysed. (B) Immunoblot analysis of PIM1 in MV4-11 cell line treated with 0.1% vehicle DMSO alone, 10, 30, 100 nM MX as well as 100 nM SGI-1776 as positive control. Cell were harvested after 12 h and lysed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-6/jm400045y/production/images/large/jm-2013-00045y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400045y&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i26">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63262" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63262" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 71 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Thaimattam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banerjee, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miglani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iqbal, J.</span><span> </span><span class="NLM_article-title">Protein Kinase Inhibitors: Structural Insights Into Selectivity</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">2751</span><span class="NLM_x">–</span> <span class="NLM_lpage">2765</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.2174%2F138161207781757042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=17897021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGmt7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=2751-2765&author=R.+Thaimattamauthor=R.+Banerjeeauthor=R.+Miglaniauthor=J.+Iqbal&title=Protein+Kinase+Inhibitors%3A+Structural+Insights+Into+Selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase inhibitors: structural insights into selectivity</span></div><div class="casAuthors">Thaimattam, Ram; Banerjee, Rahul; Miglani, Rajni; Iqbal, Javed</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">2751-2765</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are involved in many diseases like cancer, inflammation, cardiovascular disease, and diabetes.  They have become attractive target classes for drug development, making kinase inhibitors as important class of therapeutics.  The success of small-mol. ATP-competitive kinase inhibitors such as Gleevec, Iressa, and Tarceva has attracted much attention in the recent past.  Kinases make use of ATP for phosphorylation of a specific residue(s) on their protein substrates.  More than 400 x-ray structures of about 70 different kinases are publicly available.  These structures provide insights into selectivity and mechanisms of inhibition.  However, prediction of binding specificity of kinase inhibitors based on structural information alone appears to be insufficient.  Here, we will review these observations to gain insights into the rules that govern protein kinase inhibitor selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTBCSBUk5vhbVg90H21EOLACvtfcHk0lh10RQTI1ubBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGmt7jN&md5=b4bffab06eddc7d04f7b0f5a4ad3fad4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F138161207781757042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161207781757042%26sid%3Dliteratum%253Aachs%26aulast%3DThaimattam%26aufirst%3DR.%26aulast%3DBanerjee%26aufirst%3DR.%26aulast%3DMiglani%26aufirst%3DR.%26aulast%3DIqbal%26aufirst%3DJ.%26atitle%3DProtein%2520Kinase%2520Inhibitors%253A%2520Structural%2520Insights%2520Into%2520Selectivity%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2007%26volume%3D13%26spage%3D2751%26epage%3D2765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Morphy, R.</span><span> </span><span class="NLM_article-title">Selectively Nonselective Kinase Inhibition: Striking the Right Balance</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1413</span><span class="NLM_x">–</span> <span class="NLM_lpage">1437</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2009&pages=1413-1437&author=R.+Morphy&title=Selectively+Nonselective+Kinase+Inhibition%3A+Striking+the+Right+Balance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26atitle%3DSelectively%2520Nonselective%2520Kinase%2520Inhibition%253A%2520Striking%2520the%2520Right%2520Balance%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D53%26spage%3D1413%26epage%3D1437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Wilhelm, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNabola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shujath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gawlak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eveleigh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowley, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adnane, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auclair, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gedrich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voznesensky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riedl, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Post, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trail, P. A.</span><span> </span><span class="NLM_article-title">BAY 43-9006 exhibits broad spectrum oral antitumor activity and targets the RAF/MEK/ERK pathway and receptor tyrosine kinases involved in tumor progression and angiogenesis</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">7099</span><span class="NLM_x">–</span> <span class="NLM_lpage">7109</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=7099-7109&author=S.+M.+Wilhelmauthor=C.+Carterauthor=L.+Tangauthor=D.+Wilkieauthor=A.+McNabolaauthor=H.+Rongauthor=C.+Chenauthor=X.+Zhangauthor=P.+Vincentauthor=M.+McHughauthor=Y.+Caoauthor=J.+Shujathauthor=S.+Gawlakauthor=D.+Eveleighauthor=B.+Rowleyauthor=L.+Liuauthor=L.+Adnaneauthor=M.+Lynchauthor=D.+Auclairauthor=I.+Taylorauthor=R.+Gedrichauthor=A.+Voznesenskyauthor=B.+Riedlauthor=L.+E.+Postauthor=G.+Bollagauthor=P.+A.+Trail&title=BAY+43-9006+exhibits+broad+spectrum+oral+antitumor+activity+and+targets+the+RAF%2FMEK%2FERK+pathway+and+receptor+tyrosine+kinases+involved+in+tumor+progression+and+angiogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DS.%2BM.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DWilkie%26aufirst%3DD.%26aulast%3DMcNabola%26aufirst%3DA.%26aulast%3DRong%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DVincent%26aufirst%3DP.%26aulast%3DMcHugh%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DShujath%26aufirst%3DJ.%26aulast%3DGawlak%26aufirst%3DS.%26aulast%3DEveleigh%26aufirst%3DD.%26aulast%3DRowley%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DAdnane%26aufirst%3DL.%26aulast%3DLynch%26aufirst%3DM.%26aulast%3DAuclair%26aufirst%3DD.%26aulast%3DTaylor%26aufirst%3DI.%26aulast%3DGedrich%26aufirst%3DR.%26aulast%3DVoznesensky%26aufirst%3DA.%26aulast%3DRiedl%26aufirst%3DB.%26aulast%3DPost%26aufirst%3DL.%2BE.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DTrail%26aufirst%3DP.%2BA.%26atitle%3DBAY%252043-9006%2520exhibits%2520broad%2520spectrum%2520oral%2520antitumor%2520activity%2520and%2520targets%2520the%2520RAF%252FMEK%252FERK%2520pathway%2520and%2520receptor%2520tyrosine%2520kinases%2520involved%2520in%2520tumor%2520progression%2520and%2520angiogenesis%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D7099%26epage%3D7109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Lin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Life Beyond Kinases: Structure-Based Discovery of Sorafenib as Nanomolar Antagonist of 5-HT Receptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">5749</span><span class="NLM_x">–</span> <span class="NLM_lpage">5759</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300338m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=5749-5759&author=X.+Linauthor=X.-P.+Huangauthor=G.+Chenauthor=R.+Whaleyauthor=S.+Pengauthor=Y.+Wangauthor=G.+Zhangauthor=S.+X.+Wangauthor=S.+Wangauthor=B.+L.+Rothauthor=N.+Huang&title=Life+Beyond+Kinases%3A+Structure-Based+Discovery+of+Sorafenib+as+Nanomolar+Antagonist+of+5-HT+Receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm300338m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300338m%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DS.%2BX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DLife%2520Beyond%2520Kinases%253A%2520Structure-Based%2520Discovery%2520of%2520Sorafenib%2520as%2520Nanomolar%2520Antagonist%2520of%25205-HT%2520Receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D5749%26epage%3D5759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Sheinerman, F. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giraud, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laoui, A.</span><span> </span><span class="NLM_article-title">High Affinity Targets of Protein Kinase Inhibitors Have Similar Residues at the Positions Energetically Important for Binding</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">1134</span><span class="NLM_x">–</span> <span class="NLM_lpage">1156</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=1134-1156&author=F.+B.+Sheinermanauthor=E.+Giraudauthor=A.+Laoui&title=High+Affinity+Targets+of+Protein+Kinase+Inhibitors+Have+Similar+Residues+at+the+Positions+Energetically+Important+for+Binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSheinerman%26aufirst%3DF.%2BB.%26aulast%3DGiraud%26aufirst%3DE.%26aulast%3DLaoui%26aufirst%3DA.%26atitle%3DHigh%2520Affinity%2520Targets%2520of%2520Protein%2520Kinase%2520Inhibitors%2520Have%2520Similar%2520Residues%2520at%2520the%2520Positions%2520Energetically%2520Important%2520for%2520Binding%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D352%26spage%3D1134%26epage%3D1156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Subramanian, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sud, M.</span><span> </span><span class="NLM_article-title">Computational Modeling of Kinase Inhibitor Selectivity</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">395</span><span class="NLM_x">–</span> <span class="NLM_lpage">399</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml1001097" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=395-399&author=G.+Subramanianauthor=M.+Sud&title=Computational+Modeling+of+Kinase+Inhibitor+Selectivity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fml1001097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml1001097%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DG.%26aulast%3DSud%26aufirst%3DM.%26atitle%3DComputational%2520Modeling%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D395%26epage%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Brylinski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skolnick, J.</span><span> </span><span class="NLM_article-title">Comprehensive Structural and Functional Characterization of the Human Kinome by Protein Structure Modeling and Ligand Virtual Screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1839</span><span class="NLM_x">–</span> <span class="NLM_lpage">1854</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100235n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFOnsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1839-1854&author=M.+Brylinskiauthor=J.+Skolnick&title=Comprehensive+Structural+and+Functional+Characterization+of+the+Human+Kinome+by+Protein+Structure+Modeling+and+Ligand+Virtual+Screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Structural and Functional Characterization of the Human Kinome by Protein Structure Modeling and Ligand Virtual Screening</span></div><div class="casAuthors">Brylinski, Michal; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1839-1854</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The growing interest in the identification of kinase inhibitors, promising therapeutics in the treatment of many diseases, has created a demand for the structural characterization of the entire human kinome.  At the outset of the drug development process, the lead-finding stage, approaches that enrich the screening library with bioactive compds. are needed.  Here, protein structure based methods can play an important role, but despite structural genomics efforts, it is unlikely that the three-dimensional structures of the entire kinome will be available soon.  Therefore, at the proteome level, structure-based approaches must rely on predicted models, with a key issue being their utility in virtual ligand screening.  In this study, we employ the recently developed FINDSITE/Q-Dock ligand homol. modeling approach, which is well-suited for proteome-scale applications using predicted structures, to provide extensive structural and functional characterization of the human kinome.  Specifically, we construct structure models for the human kinome; these are subsequently subject to virtual screening against a library of more than 2 million compds.  To rank the compds., we employ a hierarchical approach that combines ligand- and structure-based filters.  Modeling accuracy is carefully validated using available exptl. data with particularly encouraging results found for the ability to identify, without prior knowledge, specific kinase inhibitors.  More generally, the modeling procedure results in a large no. of predicted mol. interactions between kinases and small ligands that should be of practical use in the development of novel inhibitors.  The data set is freely available to the academic community via a user-friendly Web interface at http://cssb.biol.gatech.edu/kinomelhm/ as well as at the ZINC Web site (http://zinc.docking.org/applications/2010Apr/Brylinski-2010.tar.gz).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsNAiYM1dJuLVg90H21EOLACvtfcHk0li5uw0VnenOww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFOnsLbL&md5=4dd283d41804d7f05138f5acf02ad404</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fci100235n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100235n%26sid%3Dliteratum%253Aachs%26aulast%3DBrylinski%26aufirst%3DM.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DComprehensive%2520Structural%2520and%2520Functional%2520Characterization%2520of%2520the%2520Human%2520Kinome%2520by%2520Protein%2520Structure%2520Modeling%2520and%2520Ligand%2520Virtual%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1839%26epage%3D1854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Brylinski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skolnick, J.</span><span> </span><span class="NLM_article-title">Cross-Reactivity Virtual Profiling of the Human Kinome by X-React<sup>KIN</sup>: A Chemical Systems Biology Approach</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">2324</span><span class="NLM_x">–</span> <span class="NLM_lpage">2333</span></span><div class="citationLinks">[<a href="/doi/10.1021/mp1002976" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlyks7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=2324-2333&author=M.+Brylinskiauthor=J.+Skolnick&title=Cross-Reactivity+Virtual+Profiling+of+the+Human+Kinome+by+X-ReactKIN%3A+A+Chemical+Systems+Biology+Approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Cross-Reactivity Virtual Profiling of the Human Kinome by X-ReactKIN: A Chemical Systems Biology Approach</span></div><div class="casAuthors">Brylinski, Michal; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2324-2333</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many drug candidates fail in clin. development due to their insufficient selectivity that may cause undesired side effects.  Therefore, modern drug discovery is routinely supported by computational techniques, which can identify alternate mol. targets with a significant potential for cross-reactivity.  In particular, the development of highly selective kinase inhibitors is complicated by the strong conservation of the ATP-binding site across the kinase family.  In this paper, we describe X-ReactKIN, a new machine learning approach that extends the modeling and virtual screening of individual protein kinases to a system level in order to construct a cross-reactivity virtual profile for the human kinome.  To maximize the coverage of the kinome, X-ReactKIN relies solely on the predicted target structures and employs state-of-the-art modeling techniques.  Benchmark tests carried out against available selectivity data from high-throughput kinase profiling expts. demonstrate that, for almost 70% of the inhibitors, their alternate mol. targets can be effectively identified in the human kinome with a high (>0.5) sensitivity at the expense of a relatively low false pos. rate (<0.5).  Furthermore, in a case study, we demonstrate how X-ReactKIN can support the development of selective inhibitors by optimizing the selection of kinase targets for small-scale counter-screen expts.  The constructed cross-reactivity profiles for the human kinome are freely available to the academic community at http://cssb.biol.gatech.edu/kinomelhm/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6UZWPjvChV7Vg90H21EOLACvtfcHk0li5uw0VnenOww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlyks7nK&md5=679887a7ec72771664f4bd420e3f9f3a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fmp1002976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp1002976%26sid%3Dliteratum%253Aachs%26aulast%3DBrylinski%26aufirst%3DM.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DCross-Reactivity%2520Virtual%2520Profiling%2520of%2520the%2520Human%2520Kinome%2520by%2520X-ReactKIN%253A%2520A%2520Chemical%2520Systems%2520Biology%2520Approach%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2010%26volume%3D7%26spage%3D2324%26epage%3D2333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Rockey, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elcock, A. H.</span><span> </span><span class="NLM_article-title">Rapid computational identification of the targets of protein kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">4138</span><span class="NLM_x">–</span> <span class="NLM_lpage">4152</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049461b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4138-4152&author=W.+M.+Rockeyauthor=A.+H.+Elcock&title=Rapid+computational+identification+of+the+targets+of+protein+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm049461b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049461b%26sid%3Dliteratum%253Aachs%26aulast%3DRockey%26aufirst%3DW.%2BM.%26aulast%3DElcock%26aufirst%3DA.%2BH.%26atitle%3DRapid%2520computational%2520identification%2520of%2520the%2520targets%2520of%2520protein%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4138%26epage%3D4152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catherinot, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labesse, G.</span><span> </span><span class="NLM_article-title">kinDOCK: a tool for comparative docking of protein kinase ligands</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">W325</span><span class="NLM_x">–</span> <span class="NLM_lpage">W329</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=W325-W329&author=L.+Martinauthor=V.+Catherinotauthor=G.+Labesse&title=kinDOCK%3A+a+tool+for+comparative+docking+of+protein+kinase+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DCatherinot%26aufirst%3DV.%26aulast%3DLabesse%26aufirst%3DG.%26atitle%3DkinDOCK%253A%2520a%2520tool%2520for%2520comparative%2520docking%2520of%2520protein%2520kinase%2520ligands%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DW325%26epage%3DW329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernacki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Molecular mechanics methods for predicting protein-ligand binding</span> <span class="citation_source-journal">Phys. Chem. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">5166</span><span class="NLM_x">–</span> <span class="NLM_lpage">5177</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1039%2FB608269F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=17203140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFyit7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=5166-5177&author=N.+Huangauthor=C.+Kalyanaramanauthor=K.+Bernackiauthor=M.+P.+Jacobson&title=Molecular+mechanics+methods+for+predicting+protein-ligand+binding"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanics methods for predicting protein-ligand binding</span></div><div class="casAuthors">Huang, Niu; Kalyanaraman, Chakrapani; Bernacki, Katarzyna; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">5166-5177</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review with some new data.  Ligand binding affinity prediction is one of the most important applications of computational chem.  However, accurately ranking compds. with respect to their estd. binding affinities to a biomol. target remains highly challenging.  The authors provide an overview of recent work using mol. mechanics energy functions to address this challenge.  The authors briefly review methods that use mol. dynamics and Monte Carlo simulations to predict abs. and relative ligand binding free energies, as well as the authors' own work in which they have developed a physics-based scoring method that can be applied to hundreds of thousands of compds. by invoking a no. of simplifying approxns.  In the authors' previous studies, they have demonstrated that their scoring method is a promising approach for improving the discrimination between ligands that are known to bind and those that are presumed not to, in virtual screening of large compd. databases.  In new results presented here, the authors explore several improvements to their computational method including modifying the dielec. const. used for the protein and ligand interiors, and empirically scaling energy terms to compensate for deficiencies in the energy model.  Future directions for further improving the authors' physics-based scoring method are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrog6pXfJmCo7Vg90H21EOLACvtfcHk0liNz_Tur-qpMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFyit7nP&md5=427d48383280229c26a55a8c31c4cc5e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1039%2FB608269F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB608269F%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DBernacki%26aufirst%3DK.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DMolecular%2520mechanics%2520methods%2520for%2520predicting%2520protein-ligand%2520binding%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2006%26volume%3D8%26spage%3D5166%26epage%3D5177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Physics-based methods for studying protein-ligand interactions</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">331</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=17554859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVWls7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=325-331&author=N.+Huangauthor=M.+P.+Jacobson&title=Physics-based+methods+for+studying+protein-ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Physics-based methods for studying protein-ligand interactions</span></div><div class="casAuthors">Huang, Niu; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">325-331</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  The accurate prediction of relative or abs. ligand-binding affinities is challenging in both theor. and practical aspects.  However, with the aid of advanced computing power and sophisticated methodol. development, physics-based free-energy calcns., which can be used to predict ligand-binding affinities, have become increasingly utilized in the field of structure-based drug design.  This review summarizes recent progress made in developing and applying physics-based methods to studying protein-ligand interactions, focusing on methods using mol. mechanics force fields.  Applications of these methods include predicting ligand binding poses, enriching binders in virtual screening, and calcg. relative and abs. binding free energies.  Future directions for further improving current physics-based methods are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH-FuHDbblC7Vg90H21EOLACvtfcHk0ljCf5qyXtpv5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVWls7o%253D&md5=3aaf948edd0254cc81a44645cbe05a6b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DPhysics-based%2520methods%2520for%2520studying%2520protein-ligand%2520interactions%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2007%26volume%3D10%26spage%3D325%26epage%3D331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernacki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Virtual Screening against Highly Charged Active Sites: Identifying Substrates of Alpha–Beta Barrel Enzymes</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">2059</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi0481186" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=2059-2071&author=C.+Kalyanaramanauthor=K.+Bernackiauthor=M.+P.+Jacobson&title=Virtual+Screening+against+Highly+Charged+Active+Sites%3A+Identifying+Substrates+of+Alpha%E2%80%93Beta+Barrel+Enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi0481186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0481186%26sid%3Dliteratum%253Aachs%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DBernacki%26aufirst%3DK.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DVirtual%2520Screening%2520against%2520Highly%2520Charged%2520Active%2520Sites%253A%2520Identifying%2520Substrates%2520of%2520Alpha%25E2%2580%2593Beta%2520Barrel%2520Enzymes%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26spage%3D2059%26epage%3D2071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Physics-based scoring of protein-ligand complexes: enrichment of known inhibitors in large-scale virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">243</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0502855" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=243-253&author=N.+Huangauthor=C.+Kalyanaramanauthor=J.+J.+Irwinauthor=M.+P.+Jacobson&title=Physics-based+scoring+of+protein-ligand+complexes%3A+enrichment+of+known+inhibitors+in+large-scale+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Physics-Based Scoring of Protein-Ligand Complexes: Enrichment of Known Inhibitors in Large-Scale Virtual Screening</span></div><div class="casAuthors">Huang, Niu; Kalyanaraman, Chakrapani; Irwin, John J.; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">243-253</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors demonstrate that using an all-atom mol. mechanics force field combined with an implicit solvent model for scoring protein-ligand complexes is a promising approach for improving inhibitor enrichment in the virtual screening of large compd. databases.  The rescoring method is evaluated by the extent to which known binders for nine diverse, therapeutically relevant enzymes are enriched against a background of ∼100 000 drug-like decoys.  The improvement in enrichment is most robust and dramatic within the top 1% of the ranked database, i.e., the first thousand compds.; below the first few percent of the ranked database, there is little overall improvement.  The improved early enrichment is likely due to the more realistic treatment of ligand and receptor desolvation in the rescoring procedure.  We also present anecdotal but encouraging results assessing the ability of the rescoring method to predict specificity of inhibitors for structurally related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUqkz9N6IErLVg90H21EOLACvtfcHk0lhp361U7-XxGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N&md5=cf636695aca21a87e050e0f39286ea1d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci0502855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0502855%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DPhysics-based%2520scoring%2520of%2520protein-ligand%2520complexes%253A%2520enrichment%2520of%2520known%2520inhibitors%2520in%2520large-scale%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D243%26epage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Peng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Identifying Multiple-target Ligands via Computational Chemogenomics Approaches</span> <span class="citation_source-journal">Curr. Trends Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1363</span><span class="NLM_x">–</span> <span class="NLM_lpage">1375</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.2174%2F156802612801319016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=1363-1375&author=S.+Pengauthor=X.+Linauthor=Z.+Guoauthor=N.+Huang&title=Identifying+Multiple-target+Ligands+via+Computational+Chemogenomics+Approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.2174%2F156802612801319016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802612801319016%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DIdentifying%2520Multiple-target%2520Ligands%2520via%2520Computational%2520Chemogenomics%2520Approaches%26jtitle%3DCurr.%2520Trends%2520Med.%2520Chem.%26date%3D2012%26volume%3D12%26spage%3D1363%26epage%3D1375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Fry, D. W.</span><span> </span><span class="NLM_article-title">Biochemical pharmacology of anthracenediones and anthrapyrazoles</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">109</span><span class="NLM_x">–</span> <span class="NLM_lpage">125</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2F0163-7258%2891%2990089-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=1666677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK38Xhs1Glt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1991&pages=109-125&author=D.+W.+Fry&title=Biochemical+pharmacology+of+anthracenediones+and+anthrapyrazoles"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical pharmacology of anthracenediones and anthrapyrazoles</span></div><div class="casAuthors">Fry, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-25</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    </div><div class="casAbstract">A review with ∼200 refs.  DNA intercalating antitumor agents represent one of the more widely used classes of therapeutic agents in clin. oncol.  Although a decisive mechanism to explain their ability to kill tumor cells has not been fully defined, the past decade has shown vast progress toward identifying key possibilities.  The anthracenediones and anthrapyrazoles represent two latter generation classes of DNA intercalators that show great clin. promise as antitumor drugs.  The currently known biochem. pharmacol. for these agents are discussed and these data are integrated into a broader picture touching on mechanisms of tumor cell kill.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuqDSbZBlcIbVg90H21EOLACvtfcHk0lhp361U7-XxGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xhs1Glt7k%253D&md5=bd6855d5db49270fac8a9254084b0b5a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2F0163-7258%2891%2990089-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0163-7258%252891%252990089-5%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DD.%2BW.%26atitle%3DBiochemical%2520pharmacology%2520of%2520anthracenediones%2520and%2520anthrapyrazoles%26jtitle%3DPharmacol.%2520Ther.%26date%3D1991%26volume%3D52%26spage%3D109%26epage%3D125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Capranico, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Isabella, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tinelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bigioni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zunino, F.</span><span> </span><span class="NLM_article-title">Similar sequence specificity of mitoxantrone and VM-26 stimulation of in vitro DNA cleavage by mammalian DNA topoisomerase II</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">3038</span><span class="NLM_x">–</span> <span class="NLM_lpage">3046</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi00063a015" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1993&pages=3038-3046&author=G.+Capranicoauthor=P.+De+Isabellaauthor=S.+Tinelliauthor=M.+Bigioniauthor=F.+Zunino&title=Similar+sequence+specificity+of+mitoxantrone+and+VM-26+stimulation+of+in+vitro+DNA+cleavage+by+mammalian+DNA+topoisomerase+II"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fbi00063a015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00063a015%26sid%3Dliteratum%253Aachs%26aulast%3DCapranico%26aufirst%3DG.%26aulast%3DDe%2BIsabella%26aufirst%3DP.%26aulast%3DTinelli%26aufirst%3DS.%26aulast%3DBigioni%26aufirst%3DM.%26aulast%3DZunino%26aufirst%3DF.%26atitle%3DSimilar%2520sequence%2520specificity%2520of%2520mitoxantrone%2520and%2520VM-26%2520stimulation%2520of%2520in%2520vitro%2520DNA%2520cleavage%2520by%2520mammalian%2520DNA%2520topoisomerase%2520II%26jtitle%3DBiochemistry%26date%3D1993%26volume%3D32%26spage%3D3038%26epage%3D3046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lhp361U7-XxGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Pearlman, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charifson, P. S.</span><span> </span><span class="NLM_article-title">Are Free Energy Calculations Useful in Practice? A Comparison with Rapid Scoring Functions for the p38 MAP Kinase Protein System</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">3417</span><span class="NLM_x">–</span> <span class="NLM_lpage">3423</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0100279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD3MXms1Ogu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3417-3423&author=D.+A.+Pearlmanauthor=P.+S.+Charifson&title=Are+Free+Energy+Calculations+Useful+in+Practice%3F+A+Comparison+with+Rapid+Scoring+Functions+for+the+p38+MAP+Kinase+Protein+System"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Are Free Energy Calculations Useful in Practice? A Comparison with Rapid Scoring Functions for the p38 MAP Kinase Protein System</span></div><div class="casAuthors">Pearlman, David A.; Charifson, Paul S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3417-3423</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Precise thermodn. integration free energy simulations have been applied to a congeneric series of 16 inhibitors to the p38 MAP kinase protein for which the exptl. binding data (IC50) is known.  The relative free energy of binding for each compd. has been detd.  For comparison, the same series of compds. have also scored using the best rapid scoring functions used in database screening.  From the results of these calcns., we find (1) that precise free energy simulations allow predictions that are reliable and in good agreement with expt.; (2) that predictions of lower reliability, but still in good qual. agreement with expt., can be obtained using the OWFEG free energy grid method, at a much lower computational cost; (3) and that other methods, not based on free energy simulations yield results in much poorer agreement with expt.  A new predictive index, which measures the reliability of a prediction method in the context of normal use, is defined and calcd. for each scoring method.  Predictive indexes of 0.84, 0.56, 0.04, -0.05, and 0.25 are calcd. for thermodn. integration, OWFEG, ChemScore, PLPScore, and Dock Energy Score, resp., where +1.0 is perfect correct prediction, -1.0 is perfect incorrect prediction, and 0.0 is random.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_vBFabfEdk7Vg90H21EOLACvtfcHk0liAhVjuYTHmMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXms1Ogu7w%253D&md5=04b1606cf7a885640133977dc6a1ce63</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm0100279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0100279%26sid%3Dliteratum%253Aachs%26aulast%3DPearlman%26aufirst%3DD.%2BA.%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26atitle%3DAre%2520Free%2520Energy%2520Calculations%2520Useful%2520in%2520Practice%253F%2520A%2520Comparison%2520with%2520Rapid%2520Scoring%2520Functions%2520for%2520the%2520p38%2520MAP%2520Kinase%2520Protein%2520System%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3417%26epage%3D3423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Kufareva, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Type-II kinase inhibitor docking, screening, and profiling using modified structures of active kinase states</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7921</span><span class="NLM_x">–</span> <span class="NLM_lpage">7932</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8010299" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWisb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7921-7932&author=I.+Kufarevaauthor=R.+Abagyan&title=Type-II+kinase+inhibitor+docking%2C+screening%2C+and+profiling+using+modified+structures+of+active+kinase+states"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Type-II Kinase Inhibitor Docking, Screening, and Profiling Using Modified Structures of Active Kinase States</span></div><div class="casAuthors">Kufareva, Irina; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7921-7932</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Type-II kinase inhibitors represent a class of chems. that trap their target kinases in an inactive, so-called DFG-out state, occupying a hydrophobic pocket adjacent to the ATP binding site.  These compds. are often more specific than those that target active DFG-in kinase conformations.  Unfortunately, the discovery of novel type-II scaffolds presents a considerable challenge, partially because the lack of compatible kinase structures makes structure-based methods inapplicable.  We present a computational protocol for converting multiple available DFG-in structures of various kinases (∼70% of mammalian structural kinome) into accurate and specific models of their type-II bound state.  The models, described as deletion-of-loop Asp-Phe-Gly-in (DOLPHIN) kinase models, demonstrate exceptional performance in various inhibitor discovery applications, including compd. pose prediction, screening, and in silico activity profiling.  Given the abundance of the DFG-in structures, the presented approach opens possibilities for kinome-wide discovery of specific mols. targeting inactive kinase states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp699dzZf5YRLVg90H21EOLACvtfcHk0liAhVjuYTHmMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWisb%252FK&md5=64fb7b9f789940dd61c68015341af4a6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm8010299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8010299%26sid%3Dliteratum%253Aachs%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DType-II%2520kinase%2520inhibitor%2520docking%252C%2520screening%252C%2520and%2520profiling%2520using%2520modified%2520structures%2520of%2520active%2520kinase%2520states%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7921%26epage%3D7932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Jacobs, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caron, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hare, B. J.</span><span> </span><span class="NLM_article-title">Classifying protein kinase structures guides use of ligand-selectivity profiles to predict inactive conformations: structure of lck/imatinib complex</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">1451</span><span class="NLM_x">–</span> <span class="NLM_lpage">1460</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2008&pages=1451-1460&author=M.+D.+Jacobsauthor=P.+R.+Caronauthor=B.+J.+Hare&title=Classifying+protein+kinase+structures+guides+use+of+ligand-selectivity+profiles+to+predict+inactive+conformations%3A+structure+of+lck%2Fimatinib+complex"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJacobs%26aufirst%3DM.%2BD.%26aulast%3DCaron%26aufirst%3DP.%2BR.%26aulast%3DHare%26aufirst%3DB.%2BJ.%26atitle%3DClassifying%2520protein%2520kinase%2520structures%2520guides%2520use%2520of%2520ligand-selectivity%2520profiles%2520to%2520predict%2520inactive%2520conformations%253A%2520structure%2520of%2520lck%252Fimatinib%2520complex%26jtitle%3DProteins%26date%3D2008%26volume%3D70%26spage%3D1451%26epage%3D1460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Brooijmans, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobilio, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denny, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humblet, C.</span><span> </span><span class="NLM_article-title">An enriched structural kinase database to enable kinome-wide structure-based analyses and drug discovery</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">–</span> <span class="NLM_lpage">774</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fpro.355" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=20135687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktFyrtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2010&pages=763-774&author=N.+Brooijmansauthor=Y.+W.+Changauthor=D.+Mobilioauthor=R.+A.+Dennyauthor=C.+Humblet&title=An+enriched+structural+kinase+database+to+enable+kinome-wide+structure-based+analyses+and+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An enriched structural kinase database to enable kinome-wide structure-based analyses and drug discovery</span></div><div class="casAuthors">Brooijmans, Natasja; Chang, Yu-Wei; Mobilio, Dominick; Denny, Rajiah A.; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">763-774</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The development of a kinase structural database, the kinase knowledge base (KKB), is described.  It covers all human kinase domain structures that have been deposited in the Protein Data Bank.  All structures are renumbered using a common scheme, which enables efficient cross-comparisons and multiple queries of interest to the kinase field.  The common numbering scheme is also used to automatically annotate conserved residues and motifs, and conformationally classify the structures based on the DFG-loop and Helix C. Analyses of residue conservation in the ATP binding site using the full human-kinome-sequence alignment lead to the identification of a conserved hydrogen bond between the hinge region backbone and a glycine in the specificity surface.  Furthermore, 90% of kinases are found to have at least one stabilizing interaction for the hinge region, which has not been described before.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxGHmLofxIH7Vg90H21EOLACvtfcHk0lhHnbTZgT19SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktFyrtb8%253D&md5=d6b66fb4114b942060e6dba7d1da9c96</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fpro.355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.355%26sid%3Dliteratum%253Aachs%26aulast%3DBrooijmans%26aufirst%3DN.%26aulast%3DChang%26aufirst%3DY.%2BW.%26aulast%3DMobilio%26aufirst%3DD.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DAn%2520enriched%2520structural%2520kinase%2520database%2520to%2520enable%2520kinome-wide%2520structure-based%2520analyses%2520and%2520drug%2520discovery%26jtitle%3DProtein%2520Sci.%26date%3D2010%26volume%3D19%26spage%3D763%26epage%3D774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Gajiwala, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshmukh, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diehl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiNitto, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">English, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greig, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lunney, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McTigue, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molina, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quenzer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emmett, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demetri, G. D.</span><span> </span><span class="NLM_article-title">KIT kinase mutants show unique mechanisms of drug resistance to imatinib and sunitinib in gastrointestinal stromal tumor patients</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">1542</span><span class="NLM_x">–</span> <span class="NLM_lpage">1547</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1073%2Fpnas.0812413106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=19164557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVOms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=1542-1547&author=K.+S.+Gajiwalaauthor=J.+C.+Wuauthor=J.+Christensenauthor=G.+D.+Deshmukhauthor=W.+Diehlauthor=J.+P.+DiNittoauthor=J.+M.+Englishauthor=M.+J.+Greigauthor=Y.+A.+Heauthor=S.+L.+Jacquesauthor=E.+A.+Lunneyauthor=M.+McTigueauthor=D.+Molinaauthor=T.+Quenzerauthor=P.+A.+Wellsauthor=X.+Yuauthor=Y.+Zhangauthor=A.+Zouauthor=M.+R.+Emmettauthor=A.+G.+Marshallauthor=H.+M.+Zhangauthor=G.+D.+Demetri&title=KIT+kinase+mutants+show+unique+mechanisms+of+drug+resistance+to+imatinib+and+sunitinib+in+gastrointestinal+stromal+tumor+patients"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">KIT kinase mutants show unique mechanisms of drug resistance to imatinib and sunitinib in gastrointestinal stromal tumor patients</span></div><div class="casAuthors">Gajiwala, Ketan S.; Wu, Joe C.; Christensen, James; Deshmukh, Gayatri D.; Diehl, Wade; DiNitto, Jonathan P.; English, Jessie M.; Greig, Michael J.; He, You-Ai; Jacques, Suzanne L.; Lunney, Elizabeth A.; McTigue, Michele; Molina, Dave; Quenzer, Terri; Wells, Peter A.; Yu, Xiu; Zhang, Yan; Zou, Aihua; Emmett, Mark R.; Marshall, Alan G.; Zhang, Hui-Min; Demetri, George D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1542-1547</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Most gastrointestinal stromal tumors (GISTs) exhibit aberrant activation of the receptor tyrosine kinase (RTK) KIT.  The efficacy of the inhibitors imatinib mesylate and sunitinib malate in GIST patients has been linked to their inhibition of these mutant KIT proteins.  However, patients on imatinib can acquire secondary KIT mutations that render the protein insensitive to the inhibitor.  Sunitinib has shown efficacy against certain imatinib-resistant mutants, although a subset that resides in the activation loop, including D816H/V, remains resistant.  Biochem. and structural studies were undertaken to det. the mol. basis of sunitinib resistance.  Our results show that sunitinib targets the autoinhibited conformation of WT KIT and that the D816H mutant undergoes a shift in conformational equil. toward the active state.  These findings provide a structural and enzymol. explanation for the resistance profile obsd. with the KIT inhibitors.  Prospectively, they have implications for understanding oncogenic kinase mutants and for circumventing drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkwFxa-p-iWbVg90H21EOLACvtfcHk0lgsiQNn9uL_IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVOms78%253D&md5=dd226c914c6f9a258426c0f4984a5258</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812413106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812413106%26sid%3Dliteratum%253Aachs%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DWu%26aufirst%3DJ.%2BC.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DDeshmukh%26aufirst%3DG.%2BD.%26aulast%3DDiehl%26aufirst%3DW.%26aulast%3DDiNitto%26aufirst%3DJ.%2BP.%26aulast%3DEnglish%26aufirst%3DJ.%2BM.%26aulast%3DGreig%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DY.%2BA.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLunney%26aufirst%3DE.%2BA.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DMolina%26aufirst%3DD.%26aulast%3DQuenzer%26aufirst%3DT.%26aulast%3DWells%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DEmmett%26aufirst%3DM.%2BR.%26aulast%3DMarshall%26aufirst%3DA.%2BG.%26aulast%3DZhang%26aufirst%3DH.%2BM.%26aulast%3DDemetri%26aufirst%3DG.%2BD.%26atitle%3DKIT%2520kinase%2520mutants%2520show%2520unique%2520mechanisms%2520of%2520drug%2520resistance%2520to%2520imatinib%2520and%2520sunitinib%2520in%2520gastrointestinal%2520stromal%2520tumor%2520patients%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D1542%26epage%3D1547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Kutach, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villaseñor, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belunis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lok, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, L.-N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deval, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novak, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnett, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuglstatter, A.</span><span> </span><span class="NLM_article-title">Crystal Structures of IL-2-inducible T cell Kinase Complexed with Inhibitors: Insights into Rational Drug Design and Activity Regulation</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">154</span><span class="NLM_x">–</span> <span class="NLM_lpage">163</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1111%2Fj.1747-0285.2010.00993.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=20545945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptleqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2010&pages=154-163&author=A.+K.+Kutachauthor=A.+G.+Villase%C3%B1orauthor=D.+Lamauthor=C.+Belunisauthor=C.+Jansonauthor=S.+Lokauthor=L.-N.+Hongauthor=C.-M.+Liuauthor=J.+Devalauthor=T.+J.+Novakauthor=J.+W.+Barnettauthor=W.+Chuauthor=D.+Shawauthor=A.+Kuglstatter&title=Crystal+Structures+of+IL-2-inducible+T+cell+Kinase+Complexed+with+Inhibitors%3A+Insights+into+Rational+Drug+Design+and+Activity+Regulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of IL-2-inducible T cell kinase complexed with inhibitors: insights into rational drug design and activity regulation</span></div><div class="casAuthors">Kutach, Alan K.; Villasenor, Armando G.; Lam, Diana; Belunis, Charles; Janson, Cheryl; Lok, Stephen; Hong, Li-Na; Liu, Chao-Min; Deval, Jerome; Novak, Thomas J.; Barnett, Jim W.; Chu, Wei; Shaw, David; Kuglstatter, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">154-163</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">IL-2-inducible T cell kinase plays an essential role in T cell receptor signaling and is considered a drug target for the treatment of Th2-mediated inflammatory diseases.  By applying high-throughput protein engineering and crystn., we have detd. the x-ray crystal structures of IL-2-inducible T cell kinase in complex with its selective inhibitor BMS-509744 and the broad-spectrum kinase inhibitors sunitinib and RO5191614.  Sunitinib uniquely stabilizes IL-2-inducible T cell kinase in the helix C-in conformation by inducing side chain conformational changes in the ATP-binding site.  This preference of sunitinib to bind to an active kinase conformation is reflective of its broad-spectrum kinase activity.  BMS-509744 uniquely stabilizes the activation loop in a substrate-blocking inactive conformation, indicating that structural changes described for Src family kinases are also involved in the regulation of IL-2-inducible T cell kinase activity.  The obsd. BMS-509744 binding mode allows rationalization of structure-activity relationships reported for this inhibitor class and facilitates further structure-based drug design.  Sequence-based anal. of this binding mode provides guidance for the rational design of inhibitor selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXxFkkRUV7B7Vg90H21EOLACvtfcHk0lgsiQNn9uL_IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptleqs7Y%253D&md5=24b9225c52eff5cca4e295dad1c554c3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2010.00993.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2010.00993.x%26sid%3Dliteratum%253Aachs%26aulast%3DKutach%26aufirst%3DA.%2BK.%26aulast%3DVillase%25C3%25B1or%26aufirst%3DA.%2BG.%26aulast%3DLam%26aufirst%3DD.%26aulast%3DBelunis%26aufirst%3DC.%26aulast%3DJanson%26aufirst%3DC.%26aulast%3DLok%26aufirst%3DS.%26aulast%3DHong%26aufirst%3DL.-N.%26aulast%3DLiu%26aufirst%3DC.-M.%26aulast%3DDeval%26aufirst%3DJ.%26aulast%3DNovak%26aufirst%3DT.%2BJ.%26aulast%3DBarnett%26aufirst%3DJ.%2BW.%26aulast%3DChu%26aufirst%3DW.%26aulast%3DShaw%26aufirst%3DD.%26aulast%3DKuglstatter%26aufirst%3DA.%26atitle%3DCrystal%2520Structures%2520of%2520IL-2-inducible%2520T%2520cell%2520Kinase%2520Complexed%2520with%2520Inhibitors%253A%2520Insights%2520into%2520Rational%2520Drug%2520Design%2520and%2520Activity%2520Regulation%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2010%26volume%3D76%26spage%3D154%26epage%3D163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Vogtherr, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saxena, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimme, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schieborr, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pescatore, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delarbre, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wendt, K. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwalbe, H.</span><span> </span><span class="NLM_article-title">NMR characterization of kinase p38 dynamics in free and ligand-bound forms</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">993</span><span class="NLM_x">–</span> <span class="NLM_lpage">997</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fanie.200502770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsFKmtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=993-997&author=M.+Vogtherrauthor=K.+Saxenaauthor=S.+Hoelderauthor=S.+Grimmeauthor=M.+Betzauthor=U.+Schieborrauthor=B.+Pescatoreauthor=M.+Robinauthor=L.+Delarbreauthor=T.+Langerauthor=K.+U.+Wendtauthor=H.+Schwalbe&title=NMR+characterization+of+kinase+p38+dynamics+in+free+and+ligand-bound+forms"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">NMR characterization of kinase p38 dynamics in free and ligand-bound forms</span></div><div class="casAuthors">Vogtherr, Martin; Saxena, Krishna; Hoelder, Swen; Grimme, Susanne; Betz, Marco; Schieborr, Ulrich; Pescatore, Barbara; Robin, Michel; Delarbre, Laure; Langer, Thomas; Wendt, K. Ulrich; Schwalbe, Harald</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">993-997</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In its apo state kinase p38 effects slow motions that can be detected in the NMR spectrum.  One of the affected parts is the pharmacol. interesting DFG motif.  Diarylurea inhibitors that bind to the DFG-out conformation lock this motif in a defined state, whereas DFG-in inhibitors that bind to the adjacent hinge region leave the flexibility of the DFG motif unaffected as seen in crystal structure of the complex of p38 with the inhibitor SB203580.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbgqVXMWgLZLVg90H21EOLACvtfcHk0lgsiQNn9uL_IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsFKmtr0%253D&md5=2194ed9b88b133426dc6cb52ad243d46</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.200502770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200502770%26sid%3Dliteratum%253Aachs%26aulast%3DVogtherr%26aufirst%3DM.%26aulast%3DSaxena%26aufirst%3DK.%26aulast%3DHoelder%26aufirst%3DS.%26aulast%3DGrimme%26aufirst%3DS.%26aulast%3DBetz%26aufirst%3DM.%26aulast%3DSchieborr%26aufirst%3DU.%26aulast%3DPescatore%26aufirst%3DB.%26aulast%3DRobin%26aufirst%3DM.%26aulast%3DDelarbre%26aufirst%3DL.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DWendt%26aufirst%3DK.%2BU.%26aulast%3DSchwalbe%26aufirst%3DH.%26atitle%3DNMR%2520characterization%2520of%2520kinase%2520p38%2520dynamics%2520in%2520free%2520and%2520ligand-bound%2520forms%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26spage%3D993%26epage%3D997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Totrov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Flexible ligand docking to multiple receptor conformations: a practical alternative</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">178</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2Fj.sbi.2008.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=18302984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFClsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=178-184&author=M.+Totrovauthor=R.+Abagyan&title=Flexible+ligand+docking+to+multiple+receptor+conformations%3A+a+practical+alternative"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Flexible ligand docking to multiple receptor conformations: a practical alternative</span></div><div class="casAuthors">Totrov, Maxim; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">178-184</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  State of the art docking algorithms predict an incorrect binding pose for about 50-70% of all ligands when only a single fixed receptor conformation is considered.  In many more cases, lack of receptor flexibility results in meaningless ligand binding scores, even when the correct pose is obtained.  Incorporating conformational rearrangements of the receptor binding pocket into predictions of both ligand binding pose and binding score is crucial for improving structure-based drug design and virtual ligand screening methodologies.  However, direct modeling of protein binding site flexibility remains challenging because of the large conformational space that must be sampled, and difficulties remain in constructing a suitably accurate energy function.  Here the authors show that using multiple fixed receptor conformations, either exptl. detd. by crystallog. or NMR, or computationally generated, is a practical shortcut that may improve docking calcns.  In several cases, such an approach has led to exptl. validated predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppaNNJ60eASbVg90H21EOLACvtfcHk0lgkt_Lt___fUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFClsrY%253D&md5=61b273c1621cfd64832bdda28898b439</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2008.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2008.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DFlexible%2520ligand%2520docking%2520to%2520multiple%2520receptor%2520conformations%253A%2520a%2520practical%2520alternative%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2008%26volume%3D18%26spage%3D178%26epage%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Schindler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bornmann, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellicena, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarkson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">Structural Mechanism for STI-571 Inhibition of Abelson Tyrosine Kinase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">1938</span><span class="NLM_x">–</span> <span class="NLM_lpage">1942</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1126%2Fscience.289.5486.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=10988075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD3cXms1altLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2000&pages=1938-1942&author=T.+Schindlerauthor=W.+Bornmannauthor=P.+Pellicenaauthor=W.+T.+Millerauthor=B.+Clarksonauthor=J.+Kuriyan&title=Structural+Mechanism+for+STI-571+Inhibition+of+Abelson+Tyrosine+Kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism for STI-571 inhibition of Abelson tyrosine kinase</span></div><div class="casAuthors">Schindler, Thomas; Bornmann, William; Pellicena, Patricia; Miller, W. Todd; Clarkson, Bayard; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">5486</span>),
    <span class="NLM_cas:pages">1938-1942</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The inadvertent activation of the Abelson tyrosine kinase (Abl) causes chronic myelogenous leukemia (CML).  A small-mol. inhibitor of Abl (STI-571) is effective in the treatment of CML.  We report the crystal structure of the catalytic domain of Abl, complexed to a variant of STI-571.  Crit. to the binding of STI-571 is the adoption by the kinase of an inactive conformation, in which a centrally located "activation loop" is not phosphorylated.  The conformation of this loop is distinct from that in active protein kinases, as well as in the inactive form of the closely related Src kinases.  These results suggest that compds. that exploit the distinctive inactivation mechanisms of individual protein kinases can achieve both high affinity and high specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9eAZqIQ1aebVg90H21EOLACvtfcHk0lgkt_Lt___fUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXms1altLc%253D&md5=56bb233f010f099d0073020ac6b9db59</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.289.5486.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.289.5486.1938%26sid%3Dliteratum%253Aachs%26aulast%3DSchindler%26aufirst%3DT.%26aulast%3DBornmann%26aufirst%3DW.%26aulast%3DPellicena%26aufirst%3DP.%26aulast%3DMiller%26aufirst%3DW.%2BT.%26aulast%3DClarkson%26aufirst%3DB.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DStructural%2520Mechanism%2520for%2520STI-571%2520Inhibition%2520of%2520Abelson%2520Tyrosine%2520Kinase%26jtitle%3DScience%26date%3D2000%26volume%3D289%26spage%3D1938%26epage%3D1942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Powles, T. J.</span><span> </span><span class="NLM_article-title">Evolving clinical strategies: innovative approaches to the use of mitoxantrone—introduction</span> <span class="citation_source-journal">Eur. J. Cancer Care</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1111%2Fj.1365-2354.1997.tb00317.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=1-3&author=T.+J.+Powles&title=Evolving+clinical+strategies%3A+innovative+approaches+to+the+use+of+mitoxantrone%E2%80%94introduction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2354.1997.tb00317.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2354.1997.tb00317.x%26sid%3Dliteratum%253Aachs%26aulast%3DPowles%26aufirst%3DT.%2BJ.%26atitle%3DEvolving%2520clinical%2520strategies%253A%2520innovative%2520approaches%2520to%2520the%2520use%2520of%2520mitoxantrone%25E2%2580%2594introduction%26jtitle%3DEur.%2520J.%2520Cancer%2520Care%26date%3D2008%26volume%3D6%26spage%3D1%26epage%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Harris, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reese, D. M.</span><span> </span><span class="NLM_article-title">Treatment options in hormone-refractory prostate cancer: current and future approaches</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">2177</span><span class="NLM_x">–</span> <span class="NLM_lpage">2192</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=2177-2192&author=K.+A.+Harrisauthor=D.+M.+Reese&title=Treatment+options+in+hormone-refractory+prostate+cancer%3A+current+and+future+approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DK.%2BA.%26aulast%3DReese%26aufirst%3DD.%2BM.%26atitle%3DTreatment%2520options%2520in%2520hormone-refractory%2520prostate%2520cancer%253A%2520current%2520and%2520future%2520approaches%26jtitle%3DDrugs%26date%3D2001%26volume%3D61%26spage%3D2177%26epage%3D2192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Rigacci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpaneto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alterini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrai, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernardi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellesi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi Ferrini, P.</span><span> </span><span class="NLM_article-title">Treatment of large cell lymphoma in elderly patients with a mitoxantrone, cyclophosphamide, etoposide, and prednisone regimen: long-term follow-up results</span> <span class="citation_source-journal">Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">97</span><span class="NLM_x">–</span> <span class="NLM_lpage">104</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2003&pages=97-104&author=L.+Rigacciauthor=A.+Carpanetoauthor=R.+Alteriniauthor=V.+Carraiauthor=F.+Bernardiauthor=G.+Bellesiauthor=G.+Longoauthor=A.+Bosiauthor=P.+Rossi+Ferrini&title=Treatment+of+large+cell+lymphoma+in+elderly+patients+with+a+mitoxantrone%2C+cyclophosphamide%2C+etoposide%2C+and+prednisone+regimen%3A+long-term+follow-up+results"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRigacci%26aufirst%3DL.%26aulast%3DCarpaneto%26aufirst%3DA.%26aulast%3DAlterini%26aufirst%3DR.%26aulast%3DCarrai%26aufirst%3DV.%26aulast%3DBernardi%26aufirst%3DF.%26aulast%3DBellesi%26aufirst%3DG.%26aulast%3DLongo%26aufirst%3DG.%26aulast%3DBosi%26aufirst%3DA.%26aulast%3DRossi%2BFerrini%26aufirst%3DP.%26atitle%3DTreatment%2520of%2520large%2520cell%2520lymphoma%2520in%2520elderly%2520patients%2520with%2520a%2520mitoxantrone%252C%2520cyclophosphamide%252C%2520etoposide%252C%2520and%2520prednisone%2520regimen%253A%2520long-term%2520follow-up%2520results%26jtitle%3DCancer%26date%3D2003%26volume%3D97%26spage%3D97%26epage%3D104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Bhalla, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrado, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tourkina, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullock, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponnathpur, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahoney, M. E.</span><span> </span><span class="NLM_article-title">High-dose mitoxantrone induces programmed cell death or apoptosis in human myeloid leukemia cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">3133</span><span class="NLM_x">–</span> <span class="NLM_lpage">3140</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=1993&pages=3133-3140&author=K.+Bhallaauthor=A.+M.+Ibradoauthor=E.+Tourkinaauthor=C.+Tangauthor=S.+Grantauthor=G.+Bullockauthor=Y.+Huangauthor=V.+Ponnathpurauthor=M.+E.+Mahoney&title=High-dose+mitoxantrone+induces+programmed+cell+death+or+apoptosis+in+human+myeloid+leukemia+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBhalla%26aufirst%3DK.%26aulast%3DIbrado%26aufirst%3DA.%2BM.%26aulast%3DTourkina%26aufirst%3DE.%26aulast%3DTang%26aufirst%3DC.%26aulast%3DGrant%26aufirst%3DS.%26aulast%3DBullock%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DPonnathpur%26aufirst%3DV.%26aulast%3DMahoney%26aufirst%3DM.%2BE.%26atitle%3DHigh-dose%2520mitoxantrone%2520induces%2520programmed%2520cell%2520death%2520or%2520apoptosis%2520in%2520human%2520myeloid%2520leukemia%2520cells%26jtitle%3DBlood%26date%3D1993%26volume%3D82%26spage%3D3133%26epage%3D3140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Bellosillo, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colomer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pons, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gil, J.</span><span> </span><span class="NLM_article-title">Mitoxantrone, a topoisomerase II inhibitor, induces apoptosis of B-chronic lymphocytic leukaemia cells</span> <span class="citation_source-journal">Br. J. Hamaetol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">142</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1046%2Fj.1365-2141.1998.00520.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=9450803" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1998&pages=142-146&author=B.+Bellosilloauthor=D.+Colomerauthor=G.+Ponsauthor=J.+Gil&title=Mitoxantrone%2C+a+topoisomerase+II+inhibitor%2C+induces+apoptosis+of+B-chronic+lymphocytic+leukaemia+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1046%2Fj.1365-2141.1998.00520.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1365-2141.1998.00520.x%26sid%3Dliteratum%253Aachs%26aulast%3DBellosillo%26aufirst%3DB.%26aulast%3DColomer%26aufirst%3DD.%26aulast%3DPons%26aufirst%3DG.%26aulast%3DGil%26aufirst%3DJ.%26atitle%3DMitoxantrone%252C%2520a%2520topoisomerase%2520II%2520inhibitor%252C%2520induces%2520apoptosis%2520of%2520B-chronic%2520lymphocytic%2520leukaemia%2520cells%26jtitle%3DBr.%2520J.%2520Hamaetol.%26date%3D1998%26volume%3D100%26spage%3D142%26epage%3D146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Parker, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leighton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hancock, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutton, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moorman, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ancliff, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masurekar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goulden, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Révész, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darbyshire, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Love, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saha, V.</span><span> </span><span class="NLM_article-title">Effect of mitoxantrone on outcome of children with first relapse of acute lymphoblastic leukaemia (ALL R3): an open-label randomised trial</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">376</span><span class="NLM_x">, </span> <span class="NLM_fpage">2009</span><span class="NLM_x">–</span> <span class="NLM_lpage">2017</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=376&publication_year=2010&pages=2009-2017&author=C.+Parkerauthor=R.+Watersauthor=C.+Leightonauthor=J.+Hancockauthor=R.+Suttonauthor=A.+V.+Moormanauthor=P.+Ancliffauthor=M.+Morganauthor=A.+Masurekarauthor=N.+Gouldenauthor=N.+Greenauthor=T.+R%C3%A9v%C3%A9szauthor=P.+Darbyshireauthor=S.+Loveauthor=V.+Saha&title=Effect+of+mitoxantrone+on+outcome+of+children+with+first+relapse+of+acute+lymphoblastic+leukaemia+%28ALL+R3%29%3A+an+open-label+randomised+trial"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DC.%26aulast%3DWaters%26aufirst%3DR.%26aulast%3DLeighton%26aufirst%3DC.%26aulast%3DHancock%26aufirst%3DJ.%26aulast%3DSutton%26aufirst%3DR.%26aulast%3DMoorman%26aufirst%3DA.%2BV.%26aulast%3DAncliff%26aufirst%3DP.%26aulast%3DMorgan%26aufirst%3DM.%26aulast%3DMasurekar%26aufirst%3DA.%26aulast%3DGoulden%26aufirst%3DN.%26aulast%3DGreen%26aufirst%3DN.%26aulast%3DR%25C3%25A9v%25C3%25A9sz%26aufirst%3DT.%26aulast%3DDarbyshire%26aufirst%3DP.%26aulast%3DLove%26aufirst%3DS.%26aulast%3DSaha%26aufirst%3DV.%26atitle%3DEffect%2520of%2520mitoxantrone%2520on%2520outcome%2520of%2520children%2520with%2520first%2520relapse%2520of%2520acute%2520lymphoblastic%2520leukaemia%2520%2528ALL%2520R3%2529%253A%2520an%2520open-label%2520randomised%2520trial%26jtitle%3DLancet%26date%3D2010%26volume%3D376%26spage%3D2009%26epage%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Coufal, N.; Farnaes, L.</span><span> </span><span class="NLM_article-title">Anthracyclines and Anthracenediones</span>. In  <span class="citation_source-book">Cancer Management in Man: Chemotherapy, Biological Therapy, Hyperthermia and Supporting Measures</span>; <span class="NLM_contrib-group">Minev, B. R.</span>, Ed.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">The Netherlands</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">13</span>, pp  <span class="NLM_fpage">87</span><span class="NLM_x">–</span> <span class="NLM_lpage">102</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1007%2F978-90-481-9704-0_5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=87-102&author=N.+Coufal&author=L.+Farnaesauthor=B.+R.+Minev&title=Cancer+Management+in+Man%3A+Chemotherapy%2C+Biological+Therapy%2C+Hyperthermia+and+Supporting+Measures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1007%2F978-90-481-9704-0_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-90-481-9704-0_5%26sid%3Dliteratum%253Aachs%26aulast%3DCoufal%26aufirst%3DN.%26atitle%3DAnthracyclines%2520and%2520Anthracenediones%26btitle%3DCancer%2520Management%2520in%2520Man%253A%2520Chemotherapy%252C%2520Biological%2520Therapy%252C%2520Hyperthermia%2520and%2520Supporting%2520Measures%26aulast%3DMinev%26aufirst%3DB.%2BR.%26pub%3DSpringer%26date%3D2011%26volume%3D13%26spage%3D87%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Wehenkel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellinzoni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catherinot, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barilone, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labesse, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alzari, P. M.</span><span> </span><span class="NLM_article-title">The structure of PknB in complex with mitoxantrone, an ATP-competitive inhibitor, suggests a mode of protein kinase regulation in mycobacteria</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">580</span><span class="NLM_x">, </span> <span class="NLM_fpage">3018</span><span class="NLM_x">–</span> <span class="NLM_lpage">3022</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2006&pages=3018-3022&author=A.+Wehenkelauthor=P.+Fernandezauthor=M.+Bellinzoniauthor=V.+Catherinotauthor=N.+Bariloneauthor=G.+Labesseauthor=M.+Jacksonauthor=P.+M.+Alzari&title=The+structure+of+PknB+in+complex+with+mitoxantrone%2C+an+ATP-competitive+inhibitor%2C+suggests+a+mode+of+protein+kinase+regulation+in+mycobacteria"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWehenkel%26aufirst%3DA.%26aulast%3DFernandez%26aufirst%3DP.%26aulast%3DBellinzoni%26aufirst%3DM.%26aulast%3DCatherinot%26aufirst%3DV.%26aulast%3DBarilone%26aufirst%3DN.%26aulast%3DLabesse%26aufirst%3DG.%26aulast%3DJackson%26aufirst%3DM.%26aulast%3DAlzari%26aufirst%3DP.%2BM.%26atitle%3DThe%2520structure%2520of%2520PknB%2520in%2520complex%2520with%2520mitoxantrone%252C%2520an%2520ATP-competitive%2520inhibitor%252C%2520suggests%2520a%2520mode%2520of%2520protein%2520kinase%2520regulation%2520in%2520mycobacteria%26jtitle%3DFEBS%2520Lett.%26date%3D2006%26volume%3D580%26spage%3D3018%26epage%3D3022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Anastassiadis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devarajan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, J. R.</span><span> </span><span class="NLM_article-title">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1039</span><span class="NLM_x">–</span> <span class="NLM_lpage">1045</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1038%2Fnbt.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=22037377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1039-1045&author=T.+Anastassiadisauthor=S.+W.+Deaconauthor=K.+Devarajanauthor=H.+Maauthor=J.+R.+Peterson&title=Comprehensive+assay+of+kinase+catalytic+activity+reveals+features+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span></div><div class="casAuthors">Anastassiadis, Theonie; Deacon, Sean W.; Devarajan, Karthik; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. protein kinase inhibitors are widely used to elucidate cellular signaling pathways and are promising therapeutic agents.  Owing to evolutionary conservation of the ATP-binding site, most kinase inhibitors that target this site promiscuously inhibit multiple kinases.  Interpretation of expts. that use these compds. is confounded by a lack of data on the comprehensive kinase selectivity of most inhibitors.  Here we used functional assays to profile the activity of 178 com. available kinase inhibitors against a panel of 300 recombinant protein kinases.  Quant. anal. revealed complex and often unexpected interactions between protein kinases and kinase inhibitors, with a wide spectrum of promiscuity.  Many off-target interactions occur with seemingly unrelated kinases, revealing how large-scale profiling can identify multitargeted inhibitors of specific, diverse kinases.  The results have implications for drug development and provide a resource for selecting compds. to elucidate kinase function and for interpreting the results of expts. involving kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUljWIhZGPkrVg90H21EOLACvtfcHk0lgATSvLeXkCZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E&md5=b2f705509b0e09afa46b01d4669f0515</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2017%26sid%3Dliteratum%253Aachs%26aulast%3DAnastassiadis%26aufirst%3DT.%26aulast%3DDeacon%26aufirst%3DS.%2BW.%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26atitle%3DComprehensive%2520assay%2520of%2520kinase%2520catalytic%2520activity%2520reveals%2520features%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1039%26epage%3D1045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Li, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, X. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, N. C.</span><span> </span><span class="NLM_article-title">Effects of mitoxantrone on the activity of human protein kinase CK2 holoenzyme</span> <span class="citation_source-journal">Aizheng</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">809</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotVait7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=809-815&author=C.+M.+Liauthor=X.+G.+Liuauthor=X.+C.+Linauthor=X.+W.+Chenauthor=N.+C.+Liang&title=Effects+of+mitoxantrone+on+the+activity+of+human+protein+kinase+CK2+holoenzyme"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of mitoxantrone on activity of human protein kinase CK2 holoenzyme</span></div><div class="casAuthors">Li, Chunmei; Liu, Xinguang; Lin, Xiaocong; Chen, Xiaowen; Liang, Nianci</div><div class="citationInfo"><span class="NLM_cas:title">Aizheng</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">809-815</span>CODEN:
                <span class="NLM_cas:coden">AIZHE4</span>;
        ISSN:<span class="NLM_cas:issn">1000-467X</span>.
    
            (<span class="NLM_cas:orgname">Sun Yat-sen Daxue, Aizheng Zhongxin</span>)
        </div><div class="casAbstract">Protein kinase CK2, a highly conserved protein serine/threonine kinase that is ubiquitously distributed in eukaryotes, has a close relationship with human leukemia.  Mitoxantrone is an effective drug used for acute leukemias.  The effects of mitoxantrone on the activity of recombinant holoenzyme of human protein kinase CK2 and proliferation in human leukemia cell line HL-60 were obsd.  The CK2 holoenzyme composed of α' and β subunits was recombinanted in vitro.  Subsequently, CK2 was treated with mitoxantrone at various concns., followed by addn. of reacting liq. contg. [γ-32P] ATP.  CK2 activity was measured by detecting the radioactivity of 32P transferred onto the substrates of CK2.  The effect of mitoxantrone on the proliferation of HL-60 cells was detected by the trypan blue dye exclusion assay; the changes in the cell cycle were analyzed by flow cytometry (FCM); apoptosis was analyzed by FCM and DNA agarose gel electrophoresis; effects of the drugs on the intrinsic CK2 activity were measured by a specific CK2 peptide substrate.  Mitoxantrone strongly inhibited the holoenzyme activity of recombinant human protein kinase CK2 (IC50=0.66 μmol/L).  The inhibition of CKZ by mitoxantrone was competitive with respect to ATP (Ki 0.25 μmol/L) and mostly non-competitive with respect to casein (Ki 0.66 μmol/L).  Mitoxantrone exerted strong cytotoxicity to HL-60 cells.  When treated with 0.1 μmol/L mitoxantrone for 12 h, the apoptotic rate of HL-60 cells was 9.3%.  Mitoxantrone did not affect intracellular CK2 activity.  Mitoxantrone is a strong inhibitor of recombinant human protein kinase CK2 in vitro.  Apoptosis induced by mitoxantrone in HL-60 cells has no correlation to intracellular CK2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbNRZY5mneZLVg90H21EOLACvtfcHk0lgh8qka4DZL4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotVait7g%253D&md5=68b230a5a36b6ad93f8c9e8b2a59cfec</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%2BM.%26aulast%3DLiu%26aufirst%3DX.%2BG.%26aulast%3DLin%26aufirst%3DX.%2BC.%26aulast%3DChen%26aufirst%3DX.%2BW.%26aulast%3DLiang%26aufirst%3DN.%2BC.%26atitle%3DEffects%2520of%2520mitoxantrone%2520on%2520the%2520activity%2520of%2520human%2520protein%2520kinase%2520CK2%2520holoenzyme%26jtitle%3DAizheng%26date%3D2008%26volume%3D27%26spage%3D809%26epage%3D815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandiyan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artis, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bremer, R.</span><span> </span><span class="NLM_article-title">Crystal Structures of Proto-oncogene Kinase Pim1: A Target of Aberrant Somatic Hypermutations in Diffuse Large Cell Lymphoma</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">348</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2005&pages=183-193&author=A.+Kumarauthor=V.+Mandiyanauthor=Y.+Suzukiauthor=C.+Zhangauthor=J.+Riceauthor=J.+Tsaiauthor=D.+R.+Artisauthor=P.+Ibrahimauthor=R.+Bremer&title=Crystal+Structures+of+Proto-oncogene+Kinase+Pim1%3A+A+Target+of+Aberrant+Somatic+Hypermutations+in+Diffuse+Large+Cell+Lymphoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DMandiyan%26aufirst%3DV.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DRice%26aufirst%3DJ.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DBremer%26aufirst%3DR.%26atitle%3DCrystal%2520Structures%2520of%2520Proto-oncogene%2520Kinase%2520Pim1%253A%2520A%2520Target%2520of%2520Aberrant%2520Somatic%2520Hypermutations%2520in%2520Diffuse%2520Large%2520Cell%2520Lymphoma%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D348%26spage%3D183%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Zheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donahue, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varani, G.</span><span> </span><span class="NLM_article-title">Structural Basis for Stabilization of the Tau Pre-mRNA Splicing Regulatory Element by Novantrone (Mitoxantrone)</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">557</span><span class="NLM_x">–</span> <span class="NLM_lpage">566</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2Fj.chembiol.2009.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=19477420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1Ogtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=557-566&author=S.+Zhengauthor=Y.+Chenauthor=C.+P.+Donahueauthor=M.+S.+Wolfeauthor=G.+Varani&title=Structural+Basis+for+Stabilization+of+the+Tau+Pre-mRNA+Splicing+Regulatory+Element+by+Novantrone+%28Mitoxantrone%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Stabilization of the Tau Pre-mRNA Splicing Regulatory Element by Novantrone (Mitoxantrone)</span></div><div class="casAuthors">Zheng, Suxin; Chen, Yu; Donahue, Christine P.; Wolfe, Michael S.; Varani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">557-566</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Some familial neurodegenerative diseases are assocd. with mutations that destabilize a putative stem-loop structure within an intronic region of the tau pre-mRNA and alter the prodn. of tau protein isoforms by alternative splicing.  Because stabilization of the stem loop reverses the splicing pattern assocd. with neurodegeneration, small mols. that stabilize this stem loop would provide new ways to dissect the mechanism of neurodegeneration and treat tauopathies.  The anticancer drug mitoxantrone was recently identified in a high throughput screen to stabilize the tau pre-mRNA stem loop.  Here the authors report the soln. structure of the tau mRNA-mitoxantrone complex, validated by the structure-activity relationship of existing mitoxantrone analogs.  The structure describes the mol. basis for their interaction with RNA and provides a rational basis to optimize the activity of this new class of RNA-binding mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNxlGdrUiUcrVg90H21EOLACvtfcHk0lgh8qka4DZL4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1Ogtb4%253D&md5=3d38884c150535d160fcecc5d1f03534</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2009.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2009.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DDonahue%26aufirst%3DC.%2BP.%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26aulast%3DVarani%26aufirst%3DG.%26atitle%3DStructural%2520Basis%2520for%2520Stabilization%2520of%2520the%2520Tau%2520Pre-mRNA%2520Splicing%2520Regulatory%2520Element%2520by%2520Novantrone%2520%2528Mitoxantrone%2529%26jtitle%3DChem.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D557%26epage%3D566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Bullock, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debreczeni, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amos, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, B. E.</span><span> </span><span class="NLM_article-title">Structure and substrate specificity of the Pim-1 kinase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">41675</span><span class="NLM_x">–</span> <span class="NLM_lpage">41682</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=41675-41682&author=A.+N.+Bullockauthor=J.+Debreczeniauthor=A.+L.+Amosauthor=S.+Knappauthor=B.+E.+Turk&title=Structure+and+substrate+specificity+of+the+Pim-1+kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DDebreczeni%26aufirst%3DJ.%26aulast%3DAmos%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26atitle%3DStructure%2520and%2520substrate%2520specificity%2520of%2520the%2520Pim-1%2520kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D41675%26epage%3D41682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knebel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrice, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barringer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werneburg, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span> </span><span class="NLM_article-title">Pim kinase substrate identification and specificity</span> <span class="citation_source-journal">J. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">141</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">362</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1093%2Fjb%2Fmvm040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=17234686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsVagtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2007&pages=353-362&author=C.+Pengauthor=A.+Knebelauthor=N.+A.+Morriceauthor=X.+Liauthor=K.+Barringerauthor=J.+Liauthor=S.+Jakesauthor=B.+Werneburgauthor=L.+Wang&title=Pim+kinase+substrate+identification+and+specificity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Pim kinase substrate identification and specificity</span></div><div class="casAuthors">Peng, Charline; Knebel, Axel; Morrice, Nick A.; Li, Xiang; Barringer, Kevin; Li, Jun; Jakes, Scott; Werneburg, Brian; Wang, Lian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">353-362</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Japanese Biochemical Society</span>)
        </div><div class="casAbstract">The Pim family of Ser/Thr kinases has been implicated in the process of lymphomagenesis and cell survival.  Known substrates of Pim kinases are few and poorly characterized.  In this study we set out to identify novel Pim-2 substrates using the Kinase Substrate Tracking and Elucidation (KESTREL) approach.  Two potential substrates, eukaryotic initiation factor 4B (eIF4B) and apoptosis inhibitor 5 (API-5), were identified from rat thymus exts.  Sequence comparison of the Pim-2 kinase phosphorylation sites of eIF4B and mouse BAD, the only other known Pim-2 substrate, revealed conserved amino acids preceding the phosphorylated serine residue.  Stepwise replacement of the conserved residues produced a consensus sequence for Pim kinase recognition: RXRHXS.  Pim-1 and Pim-2 catalyzed the phosphorylation of this recognition sequence 20-fold more efficiently than the original (K/R-K/R-R-K/R-L-S/T-a; a = small chain amino acid) Pim-1 phosphorylation site.  The identification of the novel Pim kinase consensus sequence provides a more sensitive and versatile peptide based assay for screening modulators of Pim kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJNFceD0VpZbVg90H21EOLACvtfcHk0lgh8qka4DZL4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsVagtbs%253D&md5=c303661dce136a9cd826c82d7f7a9943</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fjb%2Fmvm040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjb%252Fmvm040%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DKnebel%26aufirst%3DA.%26aulast%3DMorrice%26aufirst%3DN.%2BA.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DBarringer%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJakes%26aufirst%3DS.%26aulast%3DWerneburg%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DPim%2520kinase%2520substrate%2520identification%2520and%2520specificity%26jtitle%3DJ.%2520Biochem.%26date%3D2007%26volume%3D141%26spage%3D353%26epage%3D362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Aho, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandholm, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peltola, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mankonen, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lilly, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koskinen, P. J.</span><span> </span><span class="NLM_article-title">Pim-1 kinase promotes inactivation of the pro-apoptotic Bad protein by phosphorylating it on the Ser112 gatekeeper site</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">571</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=571&publication_year=2004&pages=43-49&author=T.+L.+Ahoauthor=J.+Sandholmauthor=K.+J.+Peltolaauthor=H.+P.+Mankonenauthor=M.+Lillyauthor=P.+J.+Koskinen&title=Pim-1+kinase+promotes+inactivation+of+the+pro-apoptotic+Bad+protein+by+phosphorylating+it+on+the+Ser112+gatekeeper+site"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAho%26aufirst%3DT.%2BL.%26aulast%3DSandholm%26aufirst%3DJ.%26aulast%3DPeltola%26aufirst%3DK.%2BJ.%26aulast%3DMankonen%26aufirst%3DH.%2BP.%26aulast%3DLilly%26aufirst%3DM.%26aulast%3DKoskinen%26aufirst%3DP.%2BJ.%26atitle%3DPim-1%2520kinase%2520promotes%2520inactivation%2520of%2520the%2520pro-apoptotic%2520Bad%2520protein%2520by%2520phosphorylating%2520it%2520on%2520the%2520Ser112%2520gatekeeper%2520site%26jtitle%3DFEBS%2520Lett.%26date%3D2004%26volume%3D571%26spage%3D43%26epage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Pogacic, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullock, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gasser, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer-Monard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwaller, J.</span><span> </span><span class="NLM_article-title">Structural Analysis Identifies Imidazo[1,2-b]Pyridazines as PIM Kinase Inhibitors with In vitro Antileukemic Activity</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">6916</span><span class="NLM_x">–</span> <span class="NLM_lpage">6924</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=6916-6924&author=V.+Pogacicauthor=A.+N.+Bullockauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=C.+Gasserauthor=A.+Biondiauthor=S.+Meyer-Monardauthor=S.+Knappauthor=J.+Schwaller&title=Structural+Analysis+Identifies+Imidazo%5B1%2C2-b%5DPyridazines+as+PIM+Kinase+Inhibitors+with+In+vitro+Antileukemic+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPogacic%26aufirst%3DV.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DGasser%26aufirst%3DC.%26aulast%3DBiondi%26aufirst%3DA.%26aulast%3DMeyer-Monard%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26atitle%3DStructural%2520Analysis%2520Identifies%2520Imidazo%255B1%252C2-b%255DPyridazines%2520as%2520PIM%2520Kinase%2520Inhibitors%2520with%2520In%2520vitro%2520Antileukemic%2520Activity%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D6916%26epage%3D6924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Amson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigaux, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Przedborski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flandrin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Givol, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telerman, A.</span><span> </span><span class="NLM_article-title">The human protooncogene product p33pim is expressed during fetal hematopoiesis and in diverse leukemias</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">8857</span><span class="NLM_x">–</span> <span class="NLM_lpage">8861</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1989&pages=8857-8861&author=R.+Amsonauthor=F.+Sigauxauthor=S.+Przedborskiauthor=G.+Flandrinauthor=D.+Givolauthor=A.+Telerman&title=The+human+protooncogene+product+p33pim+is+expressed+during+fetal+hematopoiesis+and+in+diverse+leukemias"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmson%26aufirst%3DR.%26aulast%3DSigaux%26aufirst%3DF.%26aulast%3DPrzedborski%26aufirst%3DS.%26aulast%3DFlandrin%26aufirst%3DG.%26aulast%3DGivol%26aufirst%3DD.%26aulast%3DTelerman%26aufirst%3DA.%26atitle%3DThe%2520human%2520protooncogene%2520product%2520p33pim%2520is%2520expressed%2520during%2520fetal%2520hematopoiesis%2520and%2520in%2520diverse%2520leukemias%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1989%26volume%3D86%26spage%3D8857%26epage%3D8861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Cibull, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eble, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ann Baldridge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malott, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, L.</span><span> </span><span class="NLM_article-title">Overexpression of Pim-1 during progression of prostatic adenocarcinoma</span> <span class="citation_source-journal">J. Clin. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">288</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1136%2Fjcp.2005.027672" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=16505280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A280%3ADC%252BD287hslyrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2006&pages=285-288&author=T.+L.+Cibullauthor=T.+D.+Jonesauthor=L.+Liauthor=J.+N.+Ebleauthor=L.+Ann+Baldridgeauthor=S.+R.+Malottauthor=Y.+Luoauthor=L.+Cheng&title=Overexpression+of+Pim-1+during+progression+of+prostatic+adenocarcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of Pim-1 during progression of prostatic adenocarcinoma</span></div><div class="casAuthors">Cibull T L; Jones T D; Li L; Eble J N; Ann Baldridge L; Malott S R; Luo Y; Cheng L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of clinical pathology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">285-8</span>
        ISSN:<span class="NLM_cas:issn">0021-9746</span>.
    </div><div class="casAbstract">AIMS:  Pim-1 is a serine/threonine kinase that has been shown to play an integral role in the development of a number of human cancers, such as haematolymphoid malignancies.  Recently, evidence has shown Pim-1 to be important in prostatic carcinogenesis.  In order to further our understanding of its role in prostate cancer, we investigated Pim-1 expression in normal, premalignant, and malignant prostate tissue.  METHODS:  Using immunohistochemistry, Pim-1 expression was analysed in prostate tissue from 120 radical prostatectomy specimens.  In each case, Pim-1 staining was evaluated in benign prostatic epithelium, high grade prostatic intraepithelial neoplasia (PIN), and prostatic adenocarcinoma.  The number of positively staining cells was estimated, and the intensity of staining was scored on a scale of 0 to 3+.  RESULTS:  Pim-1 immunoreactivity was identified in 120 cases (100%) of adenocarcinoma, 120 cases (100%) of high grade PIN, and 62 cases (52%) of benign glands.  The number of cells staining in benign epithelium (mean 34%) was much lower than that in high grade PIN (mean 80%; p<0.0001) or adenocarcinoma (mean, 84%; p<0.0001).  There was no significant difference between high grade PIN and adenocarcinoma in the percentage of cells staining positively for Pim-1 (p = 0.34).  The staining intensity for Pim-1 was significantly lower in benign prostatic epithelium than in PIN and adenocarcinoma (p<0.001).  There was no statistically significant correlation between the level of Pim-1 expression and Gleason score, patient age, tumour stage, lymph node metastasis, perineural invasion, vascular invasion, surgical margin status, extraprostatic extension, or seminal vesicle invasion.  CONCLUSIONS:  Pim-1 expression is elevated in PIN and prostatic adenocarcinoma compared with benign prostatic epithelium.  This finding suggests that upregulation of Pim-1 may play a role in prostatic neoplasia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRz4dmNESksJSXXDYhVZBdlfW6udTcc2eYtX8osZd_XT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD287hslyrsw%253D%253D&md5=358e748eac699dc9b6b2d4509553c1e4</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1136%2Fjcp.2005.027672&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjcp.2005.027672%26sid%3Dliteratum%253Aachs%26aulast%3DCibull%26aufirst%3DT.%2BL.%26aulast%3DJones%26aufirst%3DT.%2BD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DEble%26aufirst%3DJ.%2BN.%26aulast%3DAnn%2BBaldridge%26aufirst%3DL.%26aulast%3DMalott%26aufirst%3DS.%2BR.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DCheng%26aufirst%3DL.%26atitle%3DOverexpression%2520of%2520Pim-1%2520during%2520progression%2520of%2520prostatic%2520adenocarcinoma%26jtitle%3DJ.%2520Clin.%2520Pathol.%26date%3D2006%26volume%3D59%26spage%3D285%26epage%3D288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Chen, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redkar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wierda, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandhi, V.</span><span> </span><span class="NLM_article-title">Pim kinase inhibitor, SGI-1776, induces apoptosis in chronic lymphocytic leukemia cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">4150</span><span class="NLM_x">–</span> <span class="NLM_lpage">4157</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2009&pages=4150-4157&author=L.+S.+Chenauthor=S.+Redkarauthor=D.+Bearssauthor=W.+G.+Wierdaauthor=V.+Gandhi&title=Pim+kinase+inhibitor%2C+SGI-1776%2C+induces+apoptosis+in+chronic+lymphocytic+leukemia+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%2BS.%26aulast%3DRedkar%26aufirst%3DS.%26aulast%3DBearss%26aufirst%3DD.%26aulast%3DWierda%26aufirst%3DW.%2BG.%26aulast%3DGandhi%26aufirst%3DV.%26atitle%3DPim%2520kinase%2520inhibitor%252C%2520SGI-1776%252C%2520induces%2520apoptosis%2520in%2520chronic%2520lymphocytic%2520leukemia%2520cells%26jtitle%3DBlood%26date%3D2009%26volume%3D114%26spage%3D4150%26epage%3D4157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Nawijn, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alendar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berns, A.</span><span> </span><span class="NLM_article-title">For better or for worse: the role of Pim oncogenes in tumorigenesis</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1038%2Fnrc2986" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=21150935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFCjsrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=23-34&author=M.+C.+Nawijnauthor=A.+Alendarauthor=A.+Berns&title=For+better+or+for+worse%3A+the+role+of+Pim+oncogenes+in+tumorigenesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">For better or for worse: the role of Pim oncogenes in tumorigenesis</span></div><div class="casAuthors">Nawijn, Martijn C.; Alendar, Andrej; Berns, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-34</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Pim oncogenes are overexpressed in a wide range of tumors from a hematol. and epithelial origin.  Pim genes encode serine/threonine kinases that have been shown to counteract the increased sensitivity to apoptosis induction that is assocd. with MYC-driven tumorigenesis.  Recently, considerable progress has been made in characterizing the pathways of PIM-mediated survival signalling.  Given the unique structure of their active site and the minimal phenotype of mice mutant for all Pim family members, these oncogenes might be promising targets for highly specific and selective drugs with favorable toxicity profiles.  In this Review, we discuss the physiol. functions and oncogenic activities of Pim kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu23UDfObMZrVg90H21EOLACvtfcHk0lhFeuLtxKbjmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFCjsrjE&md5=fe08e70454ca8c4d808715db87d195e1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnrc2986&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2986%26sid%3Dliteratum%253Aachs%26aulast%3DNawijn%26aufirst%3DM.%2BC.%26aulast%3DAlendar%26aufirst%3DA.%26aulast%3DBerns%26aufirst%3DA.%26atitle%3DFor%2520better%2520or%2520for%2520worse%253A%2520the%2520role%2520of%2520Pim%2520oncogenes%2520in%2520tumorigenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2011%26volume%3D11%26spage%3D23%26epage%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Chen, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redkar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taverna, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandhi, V.</span><span> </span><span class="NLM_article-title">Mechanisms of cytotoxicity to Pim kinase inhibitor, SGI-1776, in acute myeloid leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">702</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2011&pages=693-702&author=L.+S.+Chenauthor=S.+Redkarauthor=P.+Tavernaauthor=J.+E.+Cortesauthor=V.+Gandhi&title=Mechanisms+of+cytotoxicity+to+Pim+kinase+inhibitor%2C+SGI-1776%2C+in+acute+myeloid+leukemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%2BS.%26aulast%3DRedkar%26aufirst%3DS.%26aulast%3DTaverna%26aufirst%3DP.%26aulast%3DCortes%26aufirst%3DJ.%2BE.%26aulast%3DGandhi%26aufirst%3DV.%26atitle%3DMechanisms%2520of%2520cytotoxicity%2520to%2520Pim%2520kinase%2520inhibitor%252C%2520SGI-1776%252C%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DBlood%26date%3D2011%26volume%3D118%26spage%3D693%26epage%3D702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Fathi, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arowojolu, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swinnen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajkhowa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Small, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marmsater, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gross, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallan, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M.</span><span> </span><span class="NLM_article-title">A potential therapeutic target for FLT3-ITD AML: PIM1 kinase</span> <span class="citation_source-journal">Leuk. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2Fj.leukres.2011.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=21802138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVemtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2012&pages=224-231&author=A.+T.+Fathiauthor=O.+Arowojoluauthor=I.+Swinnenauthor=T.+Satoauthor=T.+Rajkhowaauthor=D.+Smallauthor=F.+Marmsaterauthor=J.+E.+Robinsonauthor=S.+D.+Grossauthor=M.+Martinsonauthor=S.+Allenauthor=N.+C.+Kallanauthor=M.+Levis&title=A+potential+therapeutic+target+for+FLT3-ITD+AML%3A+PIM1+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A potential therapeutic target for FLT3-ITD AML: PIM1 kinase</span></div><div class="casAuthors">Fathi, Amir T.; Arowojolu, Omotayo; Swinnen, Ian; Sato, Takashi; Rajkhowa, Trivikram; Small, Donald; Marmsater, Fredrik; Robinson, John E.; Gross, Stefan David; Martinson, Matthew; Allen, Shelley; Kallan, Nicholas C.; Levis, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">224-231</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Patients with acute myeloid leukemia (AML) and a FLT3 internal tandem duplication (ITD) mutation have a poor prognosis, and FLT3 inhibitors are now under clin. investigation.  PIM1, a serine/threonine kinase, is up-regulated in FLT3-ITD AML and may be involved in FLT3-mediated leukemogenesis.  We employed a PIM1 inhibitor, AR00459339 (Array Biopharma Inc.), to investigate the effect of PIM1 inhibition in FLT3-mutant AML.  Like FLT3 inhibitors, AR00459339 was preferentially cytotoxic to FLT3-ITD cells, as demonstrated in the MV4-11, Molm-14, and TF/ITD cell lines, as well as 12 FLT3-ITD primary samples.  Unlike FLT3 inhibitors, AR00459339 did not suppress phosphorylation of FLT3, but did promote the de-phosphorylation of downstream FLT3 targets, STAT5, AKT, and BAD.  Combining AR00459339 with a FLT3 inhibitor resulted in additive to mildly synergistic cytotoxic effects.  AR00459339 was cytotoxic to FLT3-ITD samples from patients with secondary resistance to FLT3 inhibitors, suggesting a novel benefit to combining these agents.  We conclude that PIM1 appears to be closely assocd. with FLT3 signaling, and that inhibition of PIM1 may hold therapeutic promise, either as monotherapy, or by overcoming resistance to FLT3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBBVHeB4yqFbVg90H21EOLACvtfcHk0lhFeuLtxKbjmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVemtQ%253D%253D&md5=e7430db23fe996d45fe6a0a55ec040b3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2011.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2011.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DFathi%26aufirst%3DA.%2BT.%26aulast%3DArowojolu%26aufirst%3DO.%26aulast%3DSwinnen%26aufirst%3DI.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DRajkhowa%26aufirst%3DT.%26aulast%3DSmall%26aufirst%3DD.%26aulast%3DMarmsater%26aufirst%3DF.%26aulast%3DRobinson%26aufirst%3DJ.%2BE.%26aulast%3DGross%26aufirst%3DS.%2BD.%26aulast%3DMartinson%26aufirst%3DM.%26aulast%3DAllen%26aufirst%3DS.%26aulast%3DKallan%26aufirst%3DN.%2BC.%26aulast%3DLevis%26aufirst%3DM.%26atitle%3DA%2520potential%2520therapeutic%2520target%2520for%2520FLT3-ITD%2520AML%253A%2520PIM1%2520kinase%26jtitle%3DLeuk.%2520Res.%26date%3D2012%26volume%3D36%26spage%3D224%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Sundman-Engberg, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tidefelt, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paul, C.</span><span> </span><span class="NLM_article-title">Intracellular concentrations of mitoxantrone in leukemic cells in vitro vs in vivo</span> <span class="citation_source-journal">Leuk. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">347</span><span class="NLM_x">–</span> <span class="NLM_lpage">352</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2F0145-2126%2893%2990022-D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=8487583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK3sXltVygt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1993&pages=347-352&author=B.+Sundman-Engbergauthor=U.+Tidefeltauthor=A.+Gruberauthor=C.+Paul&title=Intracellular+concentrations+of+mitoxantrone+in+leukemic+cells+in+vitro+vs+in+vivo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular concentrations of mitoxantrone in leukemic cells in vitro vs in vivo</span></div><div class="casAuthors">Sundman-Engberg, Britt; Tidefelt, Ulf; Gruber, Astrid; Paul, Christer</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">347-52</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    </div><div class="casAbstract">The aim of this study was to det. the intracellular pharmacokinetics of mitoxantrone in vivo and to use these results to establish how leukemic cells should be incubated to perform clin. relevant in vitro studies of this drug.  Blood samples were obtained from 11 patients with acute nonlymphoblastic leukemia at certain intervals up to 20 h after the infusion of mitoxantrone 12 mg/m2.  Plasma and leukemic cells were sepd. and the drug concns. were detd. with HPLC.  Before treatment, leukemic cells from 12 patients were incubated with 0.02, 0.05, 0.1, 0.2 and 1.0 μM mitoxantrone for 1-4 h and thereafter cultured in suspension culture for 20 h; during this time cell samples were taken at certain intervals for drug detn.  In cells incubated with 0.05 and 0.2 μM mitoxantrone the cytotoxic effect was measured with the Disk assay after cultivation for 4-5 days.  In vivo, the intracellular levels exceeded the plasma concns. already at the end of infusion and after 2 h the intracellular concns. were 200-300 times higher than in plasma.  In vitro, the intracellular steady state level of mitoxantrone was reached after 1-2 h and there was a pronounced intracellular retention even after 20 h culture in drug-free medium.  Incubation with 0.05 μM during 1 h gave intracellular concns. of mitoxantrone similar to those achieved in vivo.  This incubation concn. gave a mean cytotoxic effect of 53% living cells measured with the Disk assay, which gives good possibilities to discriminate between mitoxantrone-sensitive and unsensitive cells.  Thus, exposing leukemic cells in vitro for in vivo mimicking mitoxantrone concns. could increase the clin. relevance of predictive assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgFiG859g_s7Vg90H21EOLACvtfcHk0lhFeuLtxKbjmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltVygt78%253D&md5=8b8f2eaebfa6d4c6186a5b0cb816d0a3</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2F0145-2126%2893%2990022-D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0145-2126%252893%252990022-D%26sid%3Dliteratum%253Aachs%26aulast%3DSundman-Engberg%26aufirst%3DB.%26aulast%3DTidefelt%26aufirst%3DU.%26aulast%3DGruber%26aufirst%3DA.%26aulast%3DPaul%26aufirst%3DC.%26atitle%3DIntracellular%2520concentrations%2520of%2520mitoxantrone%2520in%2520leukemic%2520cells%2520in%2520vitro%2520vs%2520in%2520vivo%26jtitle%3DLeuk.%2520Res.%26date%3D1993%26volume%3D17%26spage%3D347%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Robinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pheneger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patrice, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, A. G.</span><span> </span><span class="NLM_article-title">A Dual PIM 1/3 Kinase Inhibitor Demonstrates Efficacy in Murine Models of Lupus and Multiple Sclerosis</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">188</span><span class="NLM_x">, </span> <span class="NLM_fpage">199.9</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=188&publication_year=2012&pages=199.9&author=J.+Robinsonauthor=J.+Phenegerauthor=L.+Patriceauthor=D.+Wrightauthor=S.+Brownauthor=F.+Sullivanauthor=A.+G.+Davis&title=A+Dual+PIM+1%2F3+Kinase+Inhibitor+Demonstrates+Efficacy+in+Murine+Models+of+Lupus+and+Multiple+Sclerosis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DPheneger%26aufirst%3DJ.%26aulast%3DPatrice%26aufirst%3DL.%26aulast%3DWright%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DSullivan%26aufirst%3DF.%26aulast%3DDavis%26aufirst%3DA.%2BG.%26atitle%3DA%2520Dual%2520PIM%25201%252F3%2520Kinase%2520Inhibitor%2520Demonstrates%2520Efficacy%2520in%2520Murine%2520Models%2520of%2520Lupus%2520and%2520Multiple%2520Sclerosis%26jtitle%3DJ.%2520Immunol.%26date%3D2012%26volume%3D188%26spage%3D199.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Raju, T. N.</span><span> </span><span class="NLM_article-title">The Nobel chronicles. 1988: James Whyte Black, (b 1924), Gertrude Elion (1918–99), and George H Hitchings (1905–98)</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">355</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2000&pages=1022&author=T.+N.+Raju&title=The+Nobel+chronicles.+1988%3A+James+Whyte+Black%2C+%28b+1924%29%2C+Gertrude+Elion+%281918%E2%80%9399%29%2C+and+George+H+Hitchings+%281905%E2%80%9398%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRaju%26aufirst%3DT.%2BN.%26atitle%3DThe%2520Nobel%2520chronicles.%25201988%253A%2520James%2520Whyte%2520Black%252C%2520%2528b%25201924%2529%252C%2520Gertrude%2520Elion%2520%25281918%25E2%2580%259399%2529%252C%2520and%2520George%2520H%2520Hitchings%2520%25281905%25E2%2580%259398%2529%26jtitle%3DLancet%26date%3D2000%26volume%3D355%26spage%3D1022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Altschul, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gish, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipman, D. J.</span><span> </span><span class="NLM_article-title">Basic local alignment search tool</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">215</span><span class="NLM_x">, </span> <span class="NLM_fpage">403</span><span class="NLM_x">–</span> <span class="NLM_lpage">410</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2FS0022-2836%2805%2980360-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=2231712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK3MXitVGmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=1990&pages=403-410&author=S.+F.+Altschulauthor=W.+Gishauthor=W.+Millerauthor=E.+W.+Myersauthor=D.+J.+Lipman&title=Basic+local+alignment+search+tool"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Basic local alignment search tool</span></div><div class="casAuthors">Altschul, Stephen F.; Gish, Warren; Miller, Webb; Myers, Eugene W.; Lipman, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">403-10</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A new approach to rapid sequence comparison, basic local alignment search tool (BLAST), directly approximates alignments that optimize a measure of local similarity, the maximal segment pair (MSP) score.  Recent math. results on the stochastic properties of MSP scores allow an anal. of the performance of this method as well as the statistical significance of alignments it generates.  The basic algorithm is simple and robust; it can be implemented in a no. of ways and applied in a variety of contexts including straightforward DNA and protein sequence database searches, motif searches, gene identification searches, and in the anal. of multiple regions of similarity in long DNA sequences.  In addn. to its flexibility and tractability to math. anal., BLAST is an order of magnitude faster than existing sequence comparison tools of comparable sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGEHw-q53MfLVg90H21EOLACvtfcHk0lhtYTkNVF_czQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXitVGmsA%253D%253D&md5=009d2323eb82f0549356880e1101db16</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2805%2980360-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252805%252980360-2%26sid%3Dliteratum%253Aachs%26aulast%3DAltschul%26aufirst%3DS.%2BF.%26aulast%3DGish%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DW.%26aulast%3DMyers%26aufirst%3DE.%2BW.%26aulast%3DLipman%26aufirst%3DD.%2BJ.%26atitle%3DBasic%2520local%2520alignment%2520search%2520tool%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1990%26volume%3D215%26spage%3D403%26epage%3D410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Huse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">The conformational plasticity of protein kinases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">275</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2FS0092-8674%2802%2900741-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=12015977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2002&pages=275-282&author=M.+Huseauthor=J.+Kuriyan&title=The+conformational+plasticity+of+protein+kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The conformational plasticity of protein kinases</span></div><div class="casAuthors">Huse, Morgan; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">275-282</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with 48 refs.  Protein kinases operate in a large no. of distinct signaling pathways, where the tight regulation of their catalytic activity is crucial to the development and maintenance of eukaryotic organisms.  The catalytic domains of different kinases adopt strikingly similar structures when they are active.  By contrast, crystal structures of inactive kinases have revealed a marked plasticity in the kinase domain that allows the adoption of distinct conformations in response to interactions with specific regulatory domains or proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJrn7F3h4ej7Vg90H21EOLACvtfcHk0lhtYTkNVF_czQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D&md5=cd56edae94ad239f2ce7a5d6c8930cbb</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900741-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900741-9%26sid%3Dliteratum%253Aachs%26aulast%3DHuse%26aufirst%3DM.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DThe%2520conformational%2520plasticity%2520of%2520protein%2520kinases%26jtitle%3DCell%26date%3D2002%26volume%3D109%26spage%3D275%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Kelley, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutcliffe, M. J.</span><span> </span><span class="NLM_article-title">An automated approach for clustering an ensemble of NMR-derived protein structures into conformationally related subfamilies</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1063</span><span class="NLM_x">–</span> <span class="NLM_lpage">1065</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1093%2Fprotein%2F9.11.1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=8961360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK28XnsFamt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=1063-1065&author=L.+A.+Kelleyauthor=S.+P.+Gardnerauthor=M.+J.+Sutcliffe&title=An+automated+approach+for+clustering+an+ensemble+of+NMR-derived+protein+structures+into+conformationally+related+subfamilies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">An automated approach for clustering an ensemble of NMR-derived protein structures into conformationally related subfamilies</span></div><div class="casAuthors">Kelley, Lawrence A.; Gardner, Stephen P.; Sutcliffe, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1063-1065</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A computer program has been developed that automatically, systematically, and rapidly clusters an ensemble of structures into a set of conformationally related subfamilies, and selects a representative structure from each cluster.  The program uses the method of av. linkage to define how clusters are built up, followed by the application of a penalty function that seeks to minimize simultaneously the no. of clusters and the spread across each cluster.  This program, known as NMRCLUST, is available via the World Wide Web (URL:http://neon.chem.le.ac.uk/) and by anonymous ftp from ftp.oxmol.co.uk.  Although developed for the anal. of NMR-derived structures, the program can be used to automatically cluster any data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1EjchxnvGGrVg90H21EOLACvtfcHk0lj3wPEY91NG1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XnsFamt74%253D&md5=6940b36a9dcbba4dd8352df595737cb2</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F9.11.1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F9.11.1063%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DL.%2BA.%26aulast%3DGardner%26aufirst%3DS.%2BP.%26aulast%3DSutcliffe%26aufirst%3DM.%2BJ.%26atitle%3DAn%2520automated%2520approach%2520for%2520clustering%2520an%2520ensemble%2520of%2520NMR-derived%2520protein%2520structures%2520into%2520conformationally%2520related%2520subfamilies%26jtitle%3DProtein%2520Eng.%26date%3D1996%26volume%3D9%26spage%3D1063%26epage%3D1065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Pettersen, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">UCSF Chimera—a visualization system for exploratory research and analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">–</span> <span class="NLM_lpage">1612</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94a+visualization+system+for+exploratory+research+and+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594a%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaminski, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapp, C. S.</span><span> </span><span class="NLM_article-title">Force field validation using protein side chain prediction</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">11673</span><span class="NLM_x">–</span> <span class="NLM_lpage">11680</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp021564n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2002&pages=11673-11680&author=M.+P.+Jacobsonauthor=G.+A.+Kaminskiauthor=R.+A.+Friesnerauthor=C.+S.+Rapp&title=Force+field+validation+using+protein+side+chain+prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjp021564n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp021564n%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DKaminski%26aufirst%3DG.%2BA.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26atitle%3DForce%2520field%2520validation%2520using%2520protein%2520side%2520chain%2520prediction%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D2002%26volume%3D106%26spage%3D11673%26epage%3D11680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Improved Methods for Side Chain and Loop Predictions via the Protein Local Optimization Program:Variable Dielectric Model for Implicitly Improving the Treatment of Polarization Effects</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">2108</span><span class="NLM_x">–</span> <span class="NLM_lpage">2119</span></span><div class="citationLinks">[<a href="/doi/10.1021/ct700166f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKqtr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=2108-2119&author=K.+Zhuauthor=M.+R.+Shirtsauthor=R.+A.+Friesner&title=Improved+Methods+for+Side+Chain+and+Loop+Predictions+via+the+Protein+Local+Optimization+Program%3AVariable+Dielectric+Model+for+Implicitly+Improving+the+Treatment+of+Polarization+Effects"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Methods for Side Chain and Loop Predictions via the Protein Local Optimization Program: Variable Dielectric Model for Implicitly Improving the Treatment of Polarization Effects</span></div><div class="casAuthors">Zhu, Kai; Shirts, Michael R.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2108-2119</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper presents significant improvements in both accuracy and computational efficiency of protein side chain and loop predictions using the Protein Local Optimization Program (PLOP).  The authors introduce a novel energy model in which the internal dielec. const. of the protein is allowed to vary as a function of the interacting residues and present a phys. rationale for this model.  Using this model, the authors achieve qual. improvements in the accuracy of side chain predictions with respect to exptl. crystal structure and substantially reduce the RMSDs for loop predictions, particularly those predictions involving charged side chains.  For the single side chain prediction of lysine, 40% of the errors are eliminated, and the accuracy increases from 62.6% to 76.8%.  The errors in glutamate and aspartate predictions are reduced by 19% and 24%, resp.  When applied to a set of 240 loop predictions with 6, 8, 10, and 13 residue of loop length, this new model yields unprecedented accuracies with av. backbone root-mean-square deviations of 0.39 Å, 0.68 Å, 0.80 Å, and 1.00 Å for 6, 8, 10, and 13 residue loops, resp.  The authors also describe a series of tech. improvements in the PLOP simulation algorithms, which lead to a speedup of a factor of 2-4 in loop predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWwiam8R8KjrVg90H21EOLACvtfcHk0lj3wPEY91NG1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKqtr%252FO&md5=e591b5093032f15199883ca45544853f</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fct700166f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700166f%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DImproved%2520Methods%2520for%2520Side%2520Chain%2520and%2520Loop%2520Predictions%2520via%2520the%2520Protein%2520Local%2520Optimization%2520Program%253AVariable%2520Dielectric%2520Model%2520for%2520Implicitly%2520Improving%2520the%2520Treatment%2520of%2520Polarization%2520Effects%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2007%26volume%3D3%26spage%3D2108%26epage%3D2119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Huang, N.; Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Binding-site assessment by virtual fragment screening</span>.  <span class="citation_source-journal">PLoS One</span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e10109</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Huang%2C+N.%3B+Jacobson%2C+M.+P.+Binding-site+assessment+by+virtual+fragment+screening.+PLoS+One5%2C+e10109."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DBinding-site%2520assessment%2520by%2520virtual%2520fragment%2520screening%26jtitle%3DPLoS%2520One%26volume%3D5%26spage%3De10109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Benchmarking sets for molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6789</span><span class="NLM_x">–</span> <span class="NLM_lpage">6801</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&author=N.+Huangauthor=B.+K.+Shoichetauthor=J.+J.+Irwin&title=Benchmarking+sets+for+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0lj3wPEY91NG1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DBenchmarking%2520sets%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6789%26epage%3D6801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheatham, T. E.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merz, K. M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onufriev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, R. J.</span><span> </span><span class="NLM_article-title">The Amber biomolecular simulation programs</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1668</span><span class="NLM_x">–</span> <span class="NLM_lpage">1688</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fjcc.20290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=16200636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1668-1688&author=D.+A.+Caseauthor=T.+E.+Cheathamauthor=T.+Dardenauthor=H.+Gohlkeauthor=R.+Luoauthor=K.+M.+Merzauthor=A.+Onufrievauthor=C.+Simmerlingauthor=B.+Wangauthor=R.+J.+Woods&title=The+Amber+biomolecular+simulation+programs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The amber biomolecular simulation programs</span></div><div class="casAuthors">Case, David A.; Cheatham, Thomas E., III; Darden, Tom; Gohlke, Holger; Luo, Ray; Merz, Kenneth M., Jr.; Onufriev, Alexey; Simmerling, Carlos; Wang, Bing; Woods, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1668-1688</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development, current features, and some directions for future development of the Amber package of computer programs.  This package evolved from a program that was constructed in the late 1970s to do Assisted Model Building with Energy Refinement, and now contains a group of programs embodying a no. of powerful tools of modern computational chem., focused on mol. dynamics and free energy calcns. of proteins, nucleic acids, and carbohydrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0tZP_Umb7rVg90H21EOLACvtfcHk0lgEtCv0NvYOeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM&md5=93be29ff894bab96c783d24e9886c7d0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20290%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DOnufriev%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWoods%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520Amber%2520biomolecular%2520simulation%2520programs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1668%26epage%3D1688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Hornak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strockbine, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span> </span><span class="NLM_article-title">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">712</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fprot.21123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=16981200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=712-725&author=V.+Hornakauthor=R.+Abelauthor=A.+Okurauthor=B.+Strockbineauthor=A.+Roitbergauthor=C.+Simmerling&title=Comparison+of+multiple+Amber+force+fields+and+development+of+improved+protein+backbone+parameters"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span></div><div class="casAuthors">Hornak, Viktor; Abel, Robert; Okur, Asim; Strockbine, Bentley; Roitberg, Adrian; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">712-725</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The ff94 force field that is commonly assocd. with the Amber simulation package is one of the most widely used parameter sets for biomol. simulation.  After a decade of extensive use and testing, limitations in this force field, such as over-stabilization of α-helixes, were reported by the authors and other researchers.  This led to a no. of attempts to improve these parameters, resulting in a variety of "Amber" force fields and significant difficulty in detg. which should be used for a particular application.  The authors show that several of these continue to suffer from inadequate balance between different secondary structure elements.  In addn., the approach used in most of these studies neglected to account for the existence in Amber of two sets of backbone .vphi./ψ dihedral terms.  This led to parameter sets that provide unreasonable conformational preferences for glycine.  The authors report here an effort to improve the .vphi./ψ dihedral terms in the ff99 energy function.  Dihedral term parameters are based on fitting the energies of multiple conformations of glycine and alanine tetrapeptides from high level ab initio quantum mech. calcns.  The new parameters for backbone dihedrals replace those in the existing ff99 force field.  This parameter set, which the authors denote ff99SB, achieves a better balance of secondary structure elements as judged by improved distribution of backbone dihedrals for glycine and alanine with respect to PDB survey data.  It also accomplishes improved agreement with published exptl. data for conformational preferences of short alanine peptides and better accord with exptl. NMR relaxation data of test protein systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmoASzza-Gd7Vg90H21EOLACvtfcHk0lgEtCv0NvYOeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM&md5=de683a26eca9e83ae524726e97ac22fa</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fprot.21123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21123%26sid%3Dliteratum%253Aachs%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DOkur%26aufirst%3DA.%26aulast%3DStrockbine%26aufirst%3DB.%26aulast%3DRoitberg%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3DComparison%2520of%2520multiple%2520Amber%2520force%2520fields%2520and%2520development%2520of%2520improved%2520protein%2520backbone%2520parameters%26jtitle%3DProteins%26date%3D2006%26volume%3D65%26spage%3D712%26epage%3D725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lgEtCv0NvYOeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Jakalian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jack, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">132</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2000&pages=132-146&author=A.+Jakalianauthor=B.+L.+Bushauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+I.+Method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span></div><div class="casAuthors">Jakalian, Araz; Bush, Bruce L.; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">132-146</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The AM1-BCC method quickly and efficiently generates high-quality at. charges for use in condensed-phase simulations.  The underlying features of the electron distribution including formal charge and delocalization are first captured by AM1 at. charges for the individual mol.  Bond charge corrections (BCCs), which have been parameterized against the HF/6-31G* electrostatic potential (ESP) of a training set of compds. contg. relevant functional groups, are then added using a formalism identical to the consensus BCI (bond charge increment) approach.  As a proof of the concept, we fit BCCs simultaneously to 45 compds. including O-, N-, and S-contg. functionalities, aroms., and heteroaroms., using only 41 BCC parameters.  AM1-BCC yields charge sets of comparable quality to HF/6-31G* ESP-derived charges in a fraction of the time while reducing instabilities in the at. charges compared to direct ESP-fit methods.  We then apply the BCC parameters to a small "test set" consisting of aspirin, D-glucose, and eryodictyol; the AM1-BCC model again provides at. charges of quality comparable with HF/6-31G* RESP charges, as judged by an increase of only 0.01 to 0.02 at. units in the root-mean-square (RMS) error in ESP.  Based on these encouraging results, we intend to parameterize the AM1-BCC model to provide a consistent charge model for any org. or biol. mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruxZWGx7IgLbVg90H21EOLACvtfcHk0lgrBsKnQmkzkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D&md5=aa4774739d3491dbc8dd95ed1948d856</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252820000130%252921%253A2%253C132%253A%253AAID-JCC5%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520I.%2520Method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2000%26volume%3D21%26spage%3D132%26epage%3D146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Jakalian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jack, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1623</span><span class="NLM_x">–</span> <span class="NLM_lpage">1641</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+II.+Parameterization+and+validation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0lgrBsKnQmkzkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520II.%2520Parameterization%2520and%2520validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1623%26epage%3D1641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Luo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">David, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Accelerated Poisson-Boltzmann calculations for static and dynamic systems</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1244</span><span class="NLM_x">–</span> <span class="NLM_lpage">1253</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1002%2Fjcc.10120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=12210150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD38XmsVKgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1244-1253&author=R.+Luoauthor=L.+Davidauthor=M.+K.+Gilson&title=Accelerated+Poisson-Boltzmann+calculations+for+static+and+dynamic+systems"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Accelerated Poisson-Boltzmann calculations for static and dynamic systems</span></div><div class="casAuthors">Luo, Ray; David, Laurent; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1244-1253</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We report an efficient implementation of the finite-difference Poisson-Boltzmann solvent model based on the Modified Incomplete Cholsky Conjugate Gradient algorithm, which gives rather impressive performance for both static and dynamic systems.  This is achieved by implementing the algorithm with Eisenstat's two optimizations, utilizing the electrostatic update in simulations, and applying prudent approxns., including: relaxing the convergence criterion, not updating Poisson-Boltzmann-related forces every step, and using electrostatic focusing.  It is also possible to markedly accelerate the supporting routines that are used to set up the calcns. and to obtain energies and forces.  The resulting finite difference Poisson-Boltzmann method delivers efficiency comparable to the distance-dependent dielec. model for a system tested, HIV Protease, making it a strong candidate for soln.-phase mol. dynamics simulations.  Further, the finite difference method includes all intra-solute electrostatic interactions, whereas the distance dependent dielec. calcns. use a 15-Å cutoff.  The speed of our numerical finite difference method is comparable to that of the pair-wise Generalized Born approxn. to the Poisson-Boltzmann method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopCqAOEfmOgbVg90H21EOLACvtfcHk0lgrBsKnQmkzkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmsVKgsLo%253D&md5=7ce67345061bdd6fcfae91be1650a578</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10120%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DDavid%26aufirst%3DL.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DAccelerated%2520Poisson-Boltzmann%2520calculations%2520for%2520static%2520and%2520dynamic%2520systems%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1244%26epage%3D1253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Sitkoff, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">Accurate Calculation of Hydration Free Energies Using Macroscopic Solvent Models</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">1978</span><span class="NLM_x">–</span> <span class="NLM_lpage">1988</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100058a043" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK2cXhsVGrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1994&pages=1978-1988&author=D.+Sitkoffauthor=K.+A.+Sharpauthor=B.+Honig&title=Accurate+Calculation+of+Hydration+Free+Energies+Using+Macroscopic+Solvent+Models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate Calculation of Hydration Free Energies Using Macroscopic Solvent Models</span></div><div class="casAuthors">Sitkoff, Doree; Sharp, Kim A.; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1978-88</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">A method and parametrization scheme which allow fast and accurate calcns. of hydration free energies are described.  The solute is treated as a polarizable cavity of a shape defined by the mol. surface, contg. point charges at the location of at. nuclei.  Electrostatic contributions to solvation are derived from finite difference solns. of the Poisson equation (FDPB method).  Nonpolar (cavity/van der Waals) energies are added as a surface area dependent term, with a single surface tension coeff. (γ) derived from hydrocarbon soly. in water.  At. charges and radii are obtained by modifying existing force-field- or quantum-mech.-derived values, by fitting to exptl. solvation energies of small org. mols.  A new, simple parameter set (parameters for solvation energy, PARSE) is developed specifically for the FDPB/γ method, by choosing at. charges and radii which reproduce the estd. contributions to solvation of simple functional groups.  The PARSE parameters reproduce hydration free energies for a test set of 67 mols. with an av. error of 0.4 kcal/mol.  For amino acid side chain and peptide backbone analogs the av. error is only 0.1 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruCKnU2H8vArVg90H21EOLACvtfcHk0lgrBsKnQmkzkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsVGrsrs%253D&md5=78ad4721dced796d92eb27c15f98aba5</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fj100058a043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100058a043%26sid%3Dliteratum%253Aachs%26aulast%3DSitkoff%26aufirst%3DD.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DAccurate%2520Calculation%2520of%2520Hydration%2520Free%2520Energies%2520Using%2520Macroscopic%2520Solvent%2520Models%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1994%26volume%3D98%26spage%3D1978%26epage%3D1988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Erijman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dantes, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernheim, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shifman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peleg, Y.</span><span> </span><span class="NLM_article-title">Transfer-PCR (TPCR): a highway for DNA cloning and protein engineering</span> <span class="citation_source-journal">J. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">175</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1016%2Fj.jsb.2011.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=21515384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslWnu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2011&pages=171-177&author=A.+Erijmanauthor=A.+Dantesauthor=R.+Bernheimauthor=J.+M.+Shifmanauthor=Y.+Peleg&title=Transfer-PCR+%28TPCR%29%3A+a+highway+for+DNA+cloning+and+protein+engineering"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Transfer-PCR (TPCR): A highway for DNA cloning and protein engineering</span></div><div class="casAuthors">Erijman, Ariel; Dantes, Ada; Bernheim, Reut; Shifman, Julia M.; Peleg, Yoav</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">171-177</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">DNA cloning and protein engineering are basic methodologies employed for various applications in all life-science disciplines.  Manipulations of DNA however, could be a lengthy process that slows down subsequent expts.  To facilitate both DNA cloning and protein engineering, Transfer-PCR (TPCR) is presented, a novel approach that integrates in a single tube, PCR amplification of the target DNA from an origin vector and its subsequent integration into the destination vector.  TPCR can be applied for incorporation of DNA fragments into any desired position within a circular plasmid without the need for purifn. of the intermediate PCR product and without the use of any com. kit.  Several examples demonstrate the applicability of the TPCR platform for both DNA cloning and for multiple-site targeted mutagenesis.  In both cases, the TPCR reaction was most efficient within a narrow range of primer concns.  In mutagenesis, TPCR was primarily advantageous for generation of combinatorial libraries of targeted mutants but could be also applied to generation of variants with specific multiple mutations throughout the target gene.  Adaptation of the TPCR platform should facilitate, simplify and significantly reduce time and costs for diverse protein structure and functional studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojVuamZPNVCrVg90H21EOLACvtfcHk0lj-chTxZDl_vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslWnu70%253D&md5=8235dc3a1b08a4c5eb2e5d6f1054aab1</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2011.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2011.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DErijman%26aufirst%3DA.%26aulast%3DDantes%26aufirst%3DA.%26aulast%3DBernheim%26aufirst%3DR.%26aulast%3DShifman%26aufirst%3DJ.%2BM.%26aulast%3DPeleg%26aufirst%3DY.%26atitle%3DTransfer-PCR%2520%2528TPCR%2529%253A%2520a%2520highway%2520for%2520DNA%2520cloning%2520and%2520protein%2520engineering%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D175%26spage%3D171%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Qian, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Studts, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barringer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kronkaitis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baptiste, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mische, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farmer, B.</span><span> </span><span class="NLM_article-title">Expression, purification, crystallization and preliminary crystallographic analysis of human Pim-1 kinase</span> <span class="citation_source-journal">Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">96</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1107%2FS1744309104029963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=16508102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhvVGnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=96-99&author=K.+C.+Qianauthor=J.+Studtsauthor=L.+Wangauthor=K.+Barringerauthor=A.+Kronkaitisauthor=C.+Pengauthor=A.+Baptisteauthor=R.+LaFranceauthor=S.+Mischeauthor=B.+Farmer&title=Expression%2C+purification%2C+crystallization+and+preliminary+crystallographic+analysis+of+human+Pim-1+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification, crystallization and preliminary crystallographic analysis of human Pim-1 kinase</span></div><div class="casAuthors">Qian, Kevin C.; Studts, Joey; Wang, Lian; Barringer, Kevin; Kronkaitis, Anthony; Peng, Charline; Baptiste, Alistair; LaFrance, Roger; Mische, Sheenah; Farmer, Bennett</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology and Crystallization Communications</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">F61</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">96-99</span>CODEN:
                <span class="NLM_cas:coden">ACSFCL</span>;
        ISSN:<span class="NLM_cas:issn">1744-3091</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Pim kinases, including Pim-1, Pim-2, and Pim-3, belong to a distinctive serine/threonine protein kinase family.  They are involved in cytokine-induced signal transduction and the development of lymphoid malignancies.  Their kinase domains are highly homologous to one another, but share low sequence identity to other kinases.  Specifically, there are 2 Pro residues in the conserved hinge-region sequence ERPXPX sepd. by a residue that is non-conserved among Pim kinases.  Here, full-length human Pim-1 kinase (1-313) was cloned and expressed in Escherichia coli as a GST-fusion protein and truncated to Pim-1 (14-313) by thrombin digestion during purifn.  The Pim-1 (14-313) protein was purified to high homogeneity and monodispersity.  This protein prepn. yielded small crystals in the initial screening and large crystals after optimization.  The large crystals of apo Pim-1 kinase diffracted to 2.1 Å resoln. and belonged to space group P65, with unit-cell parameters a = b = 95.9, c = 80.0 Å, β = 120° and one mol. per asym. unit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoLbLQgcrPUrVg90H21EOLACvtfcHk0lj-chTxZDl_vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhvVGnsg%253D%253D&md5=4cfe5a9f248a6699287f9a96c627f4ac</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2FS1744309104029963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1744309104029963%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DK.%2BC.%26aulast%3DStudts%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBarringer%26aufirst%3DK.%26aulast%3DKronkaitis%26aufirst%3DA.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DBaptiste%26aufirst%3DA.%26aulast%3DLaFrance%26aufirst%3DR.%26aulast%3DMische%26aufirst%3DS.%26aulast%3DFarmer%26aufirst%3DB.%26atitle%3DExpression%252C%2520purification%252C%2520crystallization%2520and%2520preliminary%2520crystallographic%2520analysis%2520of%2520human%2520Pim-1%2520kinase%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2005%26volume%3D61%26spage%3D96%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Fjm400045y&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Fjm400045y&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Fjm400045y&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lj-chTxZDl_vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">DeLano, W. L.</span> <span class="citation_source-book">The PyMOL Molecular Graphics System</span>; <span class="NLM_publisher-name">DeLano Scientific</span>: <span class="NLM_publisher-loc">San Carlos, CA, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2002</span><span class="NLM_x">; </span>computer program.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=W.+L.+DeLano&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26btitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DDeLano%2520Scientific%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FC1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FC1','PDB','3FC1'); return false;">PDB: 3FC1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ITY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ITY','PDB','2ITY'); return false;">PDB: 2ITY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU','PDB','2RKU'); return false;">PDB: 2RKU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J2I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J2I','PDB','2J2I'); return false;">PDB: 2J2I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LXK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LXK','PDB','3LXK'); return false;">PDB: 3LXK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17','PDB','1M17'); return false;">PDB: 1M17</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GQG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GQG','PDB','2GQG'); return false;">PDB: 2GQG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IVU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IVU','PDB','2IVU'); return false;">PDB: 2IVU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2UZT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2UZT','PDB','2UZT'); return false;">PDB: 2UZT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BLR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BLR','PDB','3BLR'); return false;">PDB: 3BLR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2F4J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2F4J','PDB','2F4J'); return false;">PDB: 2F4J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y7J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y7J','PDB','2Y7J'); return false;">PDB: 2Y7J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XB7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XB7','PDB','2XB7'); return false;">PDB: 2XB7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3A62" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3A62','PDB','3A62'); return false;">PDB: 3A62</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YFX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YFX','PDB','2YFX'); return false;">PDB: 2YFX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MPA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MPA','PDB','3MPA'); return false;">PDB: 3MPA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FQS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FQS','PDB','3FQS'); return false;">PDB: 3FQS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FUM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FUM','PDB','2FUM'); return false;">PDB: 2FUM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I41" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I41','PDB','4I41'); return false;">PDB: 4I41</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2KGP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2KGP','PDB','2KGP'); return false;">PDB: 2KGP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=PIM1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=PIM1','PDB','PIM1'); return false;">PDB: PIM1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YWV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YWV','PDB','1YWV'); return false;">PDB: 1YWV</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i21"><a href="/doi/suppl/10.1021/jm400045y">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10540"></div></div></div></div></div><hr /></hr><p class="last">The details for kinase structure classification, and the criteria to determinate the kinase active conformation. The RMSDs of kinase inhibitors from different cocrystal structures, and the comparisons of experimental binding profiles and prediction results in several failure cases. The kinetic study of the interaction of ATP with PIM1 in the presence of MX. The sequence alignment of ATP site for 5 representative kinases against PknB. The activity profile of five randomly chosen kinases using radiolabeled kinase assay. The PIM1–MX complex crystallization, data collection, and structure determination. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400045y/suppl_file/jm400045y_si_001.pdf">jm400045y_si_001.pdf (1.26 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm400045y&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm400045y%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-6%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm400045y" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799230a3ce9d1c3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
