
<html lang="en"     class="pb-page"  data-request-id="cfb74728-17fc-43b6-b9d5-a60c24736584"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2011.54.issue-1;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/jm101195a"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Trends in Kinase Selectivity: Insights for Target Class-Focused Library Screening" /></meta><meta name="dc.Creator" content="Shana L.  Posy" /></meta><meta name="dc.Creator" content="Mark A.  Hermsmeier" /></meta><meta name="dc.Creator" content="Wayne  Vaccaro" /></meta><meta name="dc.Creator" content="Karl-Heinz  Ott" /></meta><meta name="dc.Creator" content="Gordon  Todderud" /></meta><meta name="dc.Creator" content="Jonathan S.  Lippy" /></meta><meta name="dc.Creator" content="George L.  Trainor" /></meta><meta name="dc.Creator" content="Deborah A.  Loughney" /></meta><meta name="dc.Creator" content="Stephen R.  Johnson" /></meta><meta name="dc.Description" content="A kinome-wide selectivity screen of &gt;20000 compounds with a rich representation of many structural classes has been completed. Analysis of the selectivity patterns for each class shows that a broad..." /></meta><meta name="Description" content="A kinome-wide selectivity screen of &gt;20000 compounds with a rich representation of many structural classes has been completed. Analysis of the selectivity patterns for each class shows that a broad..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 3, 2010" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm101195a" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2010 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm101195a" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm101195a" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm101195a" /></link>
        
    
    

<title>Trends in Kinase Selectivity: Insights for Target Class-Focused Library Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm101195a" /></meta><meta property="og:title" content="Trends in Kinase Selectivity: Insights for Target Class-Focused Library Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0010.jpeg" /></meta><meta property="og:description" content="A kinome-wide selectivity screen of &gt;20000 compounds with a rich representation of many structural classes has been completed. Analysis of the selectivity patterns for each class shows that a broad spectrum of structural scaffolds can achieve specificity for many kinase families. Kinase selectivity and potency are inversely correlated, a trend that is also found in a large set of kinase functional data. Although selective and nonselective compounds are mostly similar in their physicochemical characteristics, we identify specific features that are present more frequently in compounds that bind to many kinases. Our results support a scaffold-oriented approach for building compound collections to screen kinase targets." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm101195a"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm101195a">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm101195a&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm101195a&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm101195a&amp;href=/doi/10.1021/jm101195a" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2011</span><span class="cit-fg-volume">, 54</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 54-66</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/54/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm100064d" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm100978n" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Trends in Kinase Selectivity: Insights for Target Class-Focused Library Screening</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shana+L.++Posy">Shana L. Posy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mark+A.++Hermsmeier">Mark A. Hermsmeier</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wayne++Vaccaro">Wayne Vaccaro</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Karl-Heinz++Ott">Karl-Heinz Ott</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gordon++Todderud">Gordon Todderud</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jonathan+S.++Lippy">Jonathan S. Lippy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=George+L.++Trainor">George L. Trainor</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Deborah+A.++Loughney">Deborah A. Loughney</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+R.++Johnson">Stephen R. Johnson</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">â </span> <span class="aff-text">Computer-Assisted Drug Design, Applied Biotechnology, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">â¡</span> <span class="aff-text">Chemistry Informatics, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">Â§</span> <span class="aff-text">Discovery Chemistry, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">â¥</span> <span class="aff-text">Bioinformatics, Applied Biotechnology, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">â¥</span> <span class="aff-text">Lead Evaluation, Applied Biotechnology, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div><div class="corresp-info"><strong>*</strong>To whom correspondence should be addressed. E-mail: <a href="/cdn-cgi/l/email-protection#acdfd8c9dcc4c9c282c6c3c4c2dfc3c2eccec1df82cfc3c1"><span class="__cf_email__" data-cfemail="7c0f08190c14191252161314120f13123c1e110f521f1311">[emailÂ protected]</span></a>. Telephone: (609) 252-3003. Fax: (609) 252-6030.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm101195a&amp;href=/doi/10.1021%2Fjm101195a" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2011</span></span><span class="cit-volume">, 54</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 54â66</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 3, 2010</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 March 2010</li><li><span class="item_label"><b>Published</b> online</span>3 December 2010</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 January 2011</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm101195a" title="DOI URL">https://doi.org/10.1021/jm101195a</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2010 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D54%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DShana%2BL.%2BPosy%252C%2BMark%2BA.%2BHermsmeier%252C%2BWayne%2BVaccaro%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D54%26issueNum%3D1%26contentID%3Djm101195a%26title%3DTrends%2Bin%2BKinase%2BSelectivity%253A%2BInsights%2Bfor%2BTarget%2BClass-Focused%2BLibrary%2BScreening%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D66%26publicationDate%3DJanuary%2B2011">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm101195a"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">5520</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">58</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm101195a" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Trends in Kinase Selectivity: Insights for Target Class-Focused Library Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Shana&quot;,&quot;last_name&quot;:&quot;L. Posy&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;A. Hermsmeier&quot;},{&quot;first_name&quot;:&quot;Wayne&quot;,&quot;last_name&quot;:&quot;Vaccaro&quot;},{&quot;first_name&quot;:&quot;Karl-Heinz&quot;,&quot;last_name&quot;:&quot;Ott&quot;},{&quot;first_name&quot;:&quot;Gordon&quot;,&quot;last_name&quot;:&quot;Todderud&quot;},{&quot;first_name&quot;:&quot;Jonathan&quot;,&quot;last_name&quot;:&quot;S. Lippy&quot;},{&quot;first_name&quot;:&quot;George&quot;,&quot;last_name&quot;:&quot;L. Trainor&quot;},{&quot;first_name&quot;:&quot;Deborah&quot;,&quot;last_name&quot;:&quot;A. Loughney&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;R. Johnson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2010&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;54&quot;,&quot;pages&quot;:&quot;54-66&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm101195a&quot;},&quot;abstract&quot;:&quot;A kinome-wide selectivity screen of &gt;20000 compounds with a rich representation of many structural classes has been completed. Analysis of the selectivity patterns for each class shows that a broad spectrum of structural scaffolds can achieve specificity for many kinase families. Kinase selectivity and potency are inversely correlated, a trend that is also found in a large set of kinase functional data. Although selective and nonselective compounds are mostly similar in their physicochemical characteristics, we identify specific features that are present more frequently in compounds that bind to many kinases. Our results support a scaffold-oriented approach for building compound collections to screen kinase targets.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm101195a&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm101195a" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm101195a&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm101195a" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm101195a&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm101195a" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm101195a&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm101195a&amp;href=/doi/10.1021/jm101195a" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm101195a" /></input><a href="/doi/pdf/10.1021/jm101195a" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm101195a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm101195a%26sid%3Dliteratum%253Aachs%26pmid%3D21128601%26genre%3Darticle%26aulast%3DPosy%26date%3D2011%26atitle%3DTrends%2Bin%2BKinase%2BSelectivity%253A%2BInsights%2Bfor%2BTarget%2BClass-Focused%2BLibrary%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D54%26issue%3D1%26spage%3D54%26epage%3D66%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291654" title="Homology">Homology</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/54/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jmcmar.2011.54.issue-1/production/jmcmar.2011.54.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A kinome-wide selectivity screen of >20000 compounds with a rich representation of many structural classes has been completed. Analysis of the selectivity patterns for each class shows that a broad spectrum of structural scaffolds can achieve specificity for many kinase families. Kinase selectivity and potency are inversely correlated, a trend that is also found in a large set of kinase functional data. Although selective and nonselective compounds are mostly similar in their physicochemical characteristics, we identify specific features that are present more frequently in compounds that bind to many kinases. Our results support a scaffold-oriented approach for building compound collections to screen kinase targets.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13160" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13160" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The human genome contains more than 500 protein kinases, which transfer the Î³-phosphate of ATP to the hydroxyl group of substrate tyrosine, serine, or threonine residues.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Exogenous kinase inhibitors compete with ATP to occupy the ATP binding site, a cleft between the two lobes of the kinase catalytic domain that is highly conserved across the kinome.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Kinase inhibitors have been developed as therapeutic agents that selectively target one or more kinases, and marketed kinase drugs exhibit a wide range of selectivity. More selective inhibitors include gefitinib<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> and fasudil. Examples of less selective inhibitors are sorafenib, which targets stem cell factor receptor (KIT),<a class="ref internalNav" href="#fn1" aria-label="a">a</a> platelet-derived growth factor receptor (PDGFR), B-RAF, and vascular endothelial growth factor receptor-2 (VEGFR2);<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and dasatinib, which inhibits Src family kinases, ABL1, KIT, PDGFR-Î², focal adhesion kinase (FAK), and EphA2.<a onclick="showRef(event, 'ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9">(6-9)</a><div class="NLM_fn" id="fn1">a<p class="last">Abbreviations: AAK1, AP2-associated kinase 1; Abl, Abelson murine leukemia viral oncogene; ABL1, Abelson murine leukemia viral oncogene homologue 1; Ack, activated Cdc42-associated kinase; Alk, anaplastic lymphoma kinase; ARG, Abelson murine leukemia viral oncogene homologue 2; ASK2, apoptosis signal-regulating kinase 2; Axl, AXL receptor tyrosine kinase; BLK, B lymphoid tyrosine kinase; B-RAF, v-raf murine sarcoma viral oncogene homologue B1; CDF, cumulative distribution function; CDK2, cyclin-dependent kinase 2; CK1D/CSNK1D, casein kinase I isoform Î´; CK1E/CSNK1E, kinase I isoform Îµ; Csk, c-src tyrosine kinase; CTK, Csk-type protein tyrosine kinase; DDR, discoidin domain receptor tyrosine kinase; EGFR, epidermal growth factor receptor; Eph, ephrin receptor; FAK, focal adhesion kinase; Fer, (fps/fes-related) tyrosine kinase; FGFR, fibroblast growth factor receptor; FGR, Gardner-Rasheed feline sarcoma viral (v-fgr) oncogene homologue; FLT3, fms-related tyrosine kinase 3; GPCR, G-protein-coupled receptor; GSK, glycogen synthase kinase; HCK, hemopoietic cell kinase; HTS, high-throughput screen; IKKÎ², inhibitor of the Îº light polypeptide gene enhancer in B cells, kinase Î²; InsR, insulin receptor; JakA, Janus kinase; Jnk3, c-Jun N-terminal kinase 3; KIT, Hardy-Zuckerman 4 feline sarcoma viral oncogene homologue; LCK, lymphocyte-specific protein tyrosine kinase; LKB1, serine/threonine kinase 11; MAPK, mitogen-activated protein kinase; MAP3K4, mitogen-activated protein kinase kinase kinase 4; MAP3K6, mitogen-activated protein kinase kinase kinase 6; MEKK, MAP/ERK kinase kinase; Met, met proto-oncogene (hepatocyte growth factor receptor); MST3, mammalian STE20-like protein kinase 3; Musk, muscle, skeletal, receptor tyrosine kinase; MW, molecular weight; NAK, NF-ÎºB-activating kinase; PDGFR, platelet-derived growth factor receptor; Ret, ret proto-oncogene; Sev, v-ros UR2 sarcoma virus oncogene homologue; SMARTS, Smiles (simplified molecular input line entry) ARbitrary Target Specification; Src, v-src sarcoma (Schmidt-Ruppin A-2) viral oncogene homologue; Syk, spleen tyrosine kinase; Tc, Tanimoto coefficient; Tek, TEK tyrosine kinase, endothelial; Tie, tyrosine kinase with immunoglobulin-like and EGF-like domains; TK, tyrosine kinase; Trk, neurotrophic tyrosine kinase; VEGFR2, vascular endothelial growth factor receptor-2; YSK1, yeast Sps1/Ste20-related kinase 1.</p></div></div><div class="NLM_p">Lead compounds in a kinase drug discovery project are screened against multiple kinases to monitor their selectivity for the target (or targets) of interest. Selectivity screening typically involves close homologues of the intended target, which have ATP binding sites highly similar to those of the target and thus are likely to bind similar compounds. Recent large-scale kinase selectivity screens have confirmed that selective inhibitors tend to bind kinases that are grouped together on the kinome phylogenetic tree, and less commonly to distantly related kinases.<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10-12)</a> While these studies have characterized the targets of select inhibitors, they have not systematically explored the molecular characteristics that influence whether compounds are selective or nonselective in kinome-wide screens. Moreover, the previous screening sets maximized structural diversity and therefore included few representatives of each class of chemical structure. Fully characterizing the selectivity patterns of specific structural classes would require many examples of each class.</div><div class="NLM_p last">Here, we report the results of a nearly kinome-wide selectivity screen of >20000 compounds with a rich representation of many structural classes. We describe the selectivity patterns for each class and show that a broad spectrum of structural scaffolds can achieve specificity for most kinases. We also map a trade-off between kinase potency and selectivity that is consistent across multiple assay platforms. Although we find that selective and nonselective compounds are mostly similar in their physicochemical characteristics, we identify specific features that are present more frequently in compounds that bind to many kinases. Finally, we discuss the implications of our results for building compound collections to screen kinase targets.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21655" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21655" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Summary of the Data Set</h3><div class="NLM_p last">A total of 21851 compounds were screened at Ambit BioSciences at 1 Î¼M in an assay panel of 317â402 kinases; 12960 compounds were screened against the full 402-kinase panel. Compounds were selected for screening by two mechanisms. Approximately 13000 compounds were culled from preexisting sets of kinase inhibitors that had exhibited activity in one or more kinase functional assays. The remainder were recently synthesized for ongoing kinase projects and were submitted to Ambit BioSciences for screening regardless of their activity in in-house assays. Because of the two-track selection process, the collection includes both a diverse set of compounds and deep sampling of select chemotypes.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Activity Cutoff Selections</h3><div class="NLM_p">The binding assays performed at Ambit BioSciences were compared to functional data generated internally for a smaller set of compounds to determine reasonable activity thresholds. The two formats were compared for 40 targets for which substantial data were available. Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> shows the comparison between the IC<sub>50</sub> values determined by functional assays and the Ambit BioSciences binding data (% control) for CDK2 (19678 compounds), FLT3 (6880 compounds), and EGFR (3155 compounds). A theoretical IC<sub>50</sub> can be computed from a single-concentration binding result using the following equation (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for its derivation):<span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/jm-2010-01195a_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span>The red curves in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> indicate the theoretical IC<sub>50</sub> values that correspond to the single-point binding values on the <i>y</i>-axes as calculated with eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a>. If the single-point binding data perfectly corresponded to the functional data, all of the data points would lie along the calculated curves. On the basis of visual inspection of the relationship between the binding data and functional data, it was determined that for most targets a clear correspondence was apparent. The three assays shown are representative of the range of comparisons observed, with FLT3 being among the best, CDK2 being among the worst, and EGFR being typical of the majority of cases. The relatively poor correspondence for CDK2 is likely due to the presence of the cofactor cyclin E in the functional assay and not in the binding assay (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Relationship between the Ambit BioSciences binding assays and functional IC<sub>50</sub> determinations performed in house for three representative assays: (A) FLT3, (B) EGFR, and (C) CDK2. Functional IC<sub>50</sub> values are plotted on a log scale. Plots were generated with Excel 2003.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Given the reasonable relationship between the red lines calculated from the single-concentration binding data in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> and the IC<sub>50</sub> values from functional assays, we used eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a> to select cutoffs of 5% control (very potent activity), 13% control (potent activity), and 33% control (moderate activity), corresponding to estimated IC<sub>50</sub> values of roughly 50, 150, and 500 nM, respectively. Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> illustrates the agreement between the binding assay cutoffs and the IC<sub>50</sub> values from the functional assays for the three representative kinases. For example, 3155 compounds were tested in both the EGFR Ambit BioSciences binding assay and the in-house EGFR functional assay; 83% of the compounds with binding activity below the cutoff for very potent activity (â¤5% control) had IC<sub>50</sub> values in the functional assay of â¤50 nM, and 76% of the compounds with binding activity below the cutoff for potent activity (â¤13% control) had IC<sub>50</sub> values of â¤150 nM. For CDK2, 64% of the compounds with binding activity that was â¤5% control had IC<sub>50</sub> values in the CDK2 functional assay of â¤50 nM and 69% of the compounds with binding activity that was â¤13% control had IC<sub>50</sub> values of â¤150 nM.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Measure of Correspondence between the Ambit BioSciences Binding Assays and Functional Assay IC<sub>50</sub> Determinations</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" colspan="3" align="center" char=".">log(estimated IC<sub>50</sub>)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> vs functional log(IC<sub>50</sub>)</th><th class="colsep0 rowsep0" colspan="3" align="center" char=".">activity threshold correspondence</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">target</th><th class="colsep0 rowsep0" align="center" char=".">slope</th><th class="colsep0 rowsep0" align="center" char=".">intercept</th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sup>2</sup></th><th class="colsep0 rowsep0" align="center" char=".">5%/50 nM<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">13%/150 nM<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">33%/500 nM<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CDK2</td><td class="colsep0 rowsep0" align="char" char=".">0.42</td><td class="colsep0 rowsep0" align="char" char=".">1.18</td><td class="colsep0 rowsep0" align="char" char=".">0.17</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="char" char=".">69</td><td class="colsep0 rowsep0" align="char" char=".">76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FLT3</td><td class="colsep0 rowsep0" align="char" char=".">0.88</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td><td class="colsep0 rowsep0" align="char" char=".">0.67</td><td class="colsep0 rowsep0" align="char" char=".">84</td><td class="colsep0 rowsep0" align="char" char=".">88</td><td class="colsep0 rowsep0" align="char" char=".">93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="char" char=".">0.81</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td><td class="colsep0 rowsep0" align="char" char=".">0.47</td><td class="colsep0 rowsep0" align="char" char=".">83</td><td class="colsep0 rowsep0" align="char" char=".">76</td><td class="colsep0 rowsep0" align="char" char=".">69</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Log(estimated IC<sub>50</sub>) is calculated from the estimated IC<sub>50</sub> for the Ambit BioSciences binding assay (eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a>).</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Percentage of all compounds with Ambit BioSciences binding values that are â¤5% control that are also more potent than 50 nM in the functional assay.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Percentage of all compounds with Ambit BioSciences binding values that are â¤13% control that are also more potent than 150 nM in the functional assay.</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">Percentage of all compounds with Ambit BioSciences binding values that are â¤33% control that are also more potent than 500 nM in the functional assay.</p></div></div></div><div class="NLM_p">Of the 21851 compounds screened, 4187 (19.1%) compounds bound none of the assayed kinases at even a moderate level (â¤33% control). Of the active compounds, 7790 demonstrated both selectivity and extreme potency [the âSP setâ (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>)]. It is important to note that throughout this work, selectivity is defined by the number of kinase targets bound below a certain cutoff. Thus, selective compounds bind few targets, and nonselective compounds bind many targets. The selectivity index SI(<i>x</i>) quantifies selectivity for a given activity threshold <i>x</i>. For example, an SI(33) value of 0.2 indicates that a compound bound 20% of the targets assayed at â¤33% control. This definition of selectivity does not take into account the relative affinities of the compound for its multiple targets. A compound that was assayed in 100 binding assays and bound two targets is considered selective, regardless of whether it bound the two targets with equal affinity or whether it bound target A 10 times more strongly than target B.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Heat map of binding data for potent, selective compounds. Each row represents a single kinase and each column a single compound. Kinases are grouped by family and compounds by structural similarity. Heat map values range from 0% (red, maximum binding activity) to 100% (blue, minimum binding) of control. (A) Binding data for all potent, selective compounds (the SP set). (B) Detailed view of binding to Tec, Syk, and Src family kinases for two clusters of structurally similar compounds. The heat map was generated with Spotfire DecisionSite version 9.1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A shows a heat map of the Ambit BioSciences activities for the SP set. Each row represents a single kinase and each column a single compound. Kinases are grouped by family/subfamily classification, and compounds are clustered by structural similarity. Bright regions of the heat map indicate clusters of similar compounds that bound to particular families of kinases. Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B presents a detailed view of one heat map segment. The two boxed regions highlight two clusters of compounds: compounds in the first cluster bound Tec family kinases as well as select Src family kinases (primarily BLK and HCK); compounds in the second cluster did not bind Tec kinases and bound to almost all Src family kinases.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Kinase Hit Rates</h3><div class="NLM_p">Each kinase was potently bound by at least one compound, but the hit rates for the kinases varied from 0.03 to 18% (<13% control). Hit rates for all targets for all 21851 compounds, and for only the compounds in the SP set, are provided in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>. Of all 402 kinases, the kinases with the lowest hit rates were the tyrosine kinase CTK (4 hits) and two serine/threonine kinases that function in the MAPK pathway, MAP3K6/ASK2 (6 hits) and MAP3K4 (13 hits). Their low hit rates may indicate that it would be difficult to develop inhibitors to target these kinases.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Average Hit Rates for Members of Each Tyrosine Kinase Family</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">family</th><th class="colsep0 rowsep0" align="center" char=".">hit rate (all)</th><th class="colsep0 rowsep0" align="center" char=".">hit rate (SP set)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDGFR</td><td class="colsep0 rowsep0" align="char" char=".">13.09</td><td class="colsep0 rowsep0" align="char" char=".">10.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ret</td><td class="colsep0 rowsep0" align="char" char=".">7.97</td><td class="colsep0 rowsep0" align="char" char=".">2.73</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DDR</td><td class="colsep0 rowsep0" align="char" char=".">7.61</td><td class="colsep0 rowsep0" align="char" char=".">7.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Src</td><td class="colsep0 rowsep0" align="char" char=".">7.41</td><td class="colsep0 rowsep0" align="char" char=".">4.51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">InsR</td><td class="colsep0 rowsep0" align="char" char=".">5.95</td><td class="colsep0 rowsep0" align="char" char=".">4.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JakA</td><td class="colsep0 rowsep0" align="char" char=".">5.91</td><td class="colsep0 rowsep0" align="char" char=".">6.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Trk</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="char" char=".">2.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ack</td><td class="colsep0 rowsep0" align="char" char=".">4.94</td><td class="colsep0 rowsep0" align="char" char=".">1.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Abl</td><td class="colsep0 rowsep0" align="char" char=".">4.76</td><td class="colsep0 rowsep0" align="char" char=".">2.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Tec</td><td class="colsep0 rowsep0" align="char" char=".">4.42</td><td class="colsep0 rowsep0" align="char" char=".">3.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">VEGFR</td><td class="colsep0 rowsep0" align="char" char=".">4.25</td><td class="colsep0 rowsep0" align="char" char=".">3.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Axl</td><td class="colsep0 rowsep0" align="char" char=".">4.19</td><td class="colsep0 rowsep0" align="char" char=".">1.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Tie</td><td class="colsep0 rowsep0" align="char" char=".">3.98</td><td class="colsep0 rowsep0" align="char" char=".">1.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="char" char=".">3.93</td><td class="colsep0 rowsep0" align="char" char=".">4.72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Fak</td><td class="colsep0 rowsep0" align="char" char=".">3.73</td><td class="colsep0 rowsep0" align="char" char=".">0.93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Sev</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">1.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Eph</td><td class="colsep0 rowsep0" align="char" char=".">3.66</td><td class="colsep0 rowsep0" align="char" char=".">1.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Alk</td><td class="colsep0 rowsep0" align="char" char=".">3.59</td><td class="colsep0 rowsep0" align="char" char=".">0.96</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Musk</td><td class="colsep0 rowsep0" align="char" char=".">3.46</td><td class="colsep0 rowsep0" align="char" char=".">1.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Csk</td><td class="colsep0 rowsep0" align="char" char=".">2.65</td><td class="colsep0 rowsep0" align="char" char=".">2.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Met</td><td class="colsep0 rowsep0" align="char" char=".">2.44</td><td class="colsep0 rowsep0" align="char" char=".">2.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FGFR</td><td class="colsep0 rowsep0" align="char" char=".">2.26</td><td class="colsep0 rowsep0" align="char" char=".">0.74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Fer</td><td class="colsep0 rowsep0" align="char" char=".">1.09</td><td class="colsep0 rowsep0" align="char" char=".">0.28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Syk</td><td class="colsep0 rowsep0" align="char" char=".">0.89</td><td class="colsep0 rowsep0" align="char" char=".">0.42</td></tr></tbody></table></div></div><div class="NLM_p last">Because most kinases targeted for drug discovery efforts belong to the tyrosine kinase (TK) branch of the kinome, we examined the hit rates for TKs in more detail. Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> lists for each TK family the average hit rates for all compounds and for the SP set only. The family of type III receptor tyrosine kinases (family PDGFR in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) had the highest average hit rate for all compounds (13%). This family includes KIT and Flt3, which are known to bind many structurally diverse compounds.<a onclick="showRef(event, 'ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16 ref17">(13-17)</a> When only the SP set was considered, the average hit rate for the PDGFR family (10.4%) was roughly the same as for all compounds, showing that both selective and nonselective compounds bound to PDGFR family kinases. The RET and DDR families had similar average hit rates for all compounds of 8 and 7.6%, respectively. However, they had very different average hit rates when only the SP set was considered. For DDR kinases, the SP set average hit rate was comparable to the hit rate for all compounds (7.3%), while for RET kinases, the average hit rate dropped to 2.7%. The reduction in the hit rate for the SP set indicates that most of the compounds that bound to RET kinases are nonselective.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Activity Homology and Sequence Identity</h3><div class="NLM_p">The compound activity profiles that were generated for each kinase can be used to calculate activity-based measures of kinase homology. We define activity homology as the prior probability that a compound will be active for kinase B given that it is active for kinase A (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> shows the cumulative distribution function of activity homology (CDF<sub>H</sub>) for pairs of kinases which were binned by the sequence identity of their kinase catalytic domains. In general, kinase pairs with higher sequence identity had greater activity homology; half the kinase pairs with 80â90% sequence identity had >60% activity homology, while only 20% of kinase pairs with 50â60% sequence identity had activity homologies in the same range.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. CDF of activity homology. Pairs of kinases were binned by sequence identity. For the kinase pairs in each sequence identity bin, the cumulative distribution function of activity homology is plotted as a separate function. Activity homology for kinases A and B is the fraction of compounds active for kinase A that are also active for kinase B.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Exceptions to the general trend of sequence homology correlating with activity homology exist. For example, the kinase domains of YSK1 and MST3 share 88% sequence identity, with only one residue difference (Ile-78 vs Val-94) in their active sites. We would therefore predict that most compounds that bind YSK1 would also bind MST3 (and vice versa) and that these two kinases would have similar hit rates. Both kinases are very selective, with hit rates of 1.1 and 1.2%, respectively (corresponding to 244 and 262 hits, respectively, out of 21851 compounds tested). Surprisingly, however, only 53.3% of compounds that were active in YSK1 were active in MST3, and only 49.6% of MST3 actives bound YSK1. Despite their high sequence similarity, activity in the YSK1 assay is not necessarily predictive of activity in the MST3 assay, and vice versa. The observed lack of overlap between compounds active for YSK1 and MST3 may be caused in part by the low sensitivity of the YSK1 and MST3 binding assays and the limitations of data obtained by screening at a single compound concentration. Nonetheless, the activity homology of these two kinases is unusually low as compared to those of kinase pairs with similar sequence homology (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).</div><div class="NLM_p">Cases of low sequence identity and high activity homology can also be found. Bamborough et al.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> found that LKB1 and AAK1, which are distantly related in sequence, had unexpectedly similar profiles of inhibitor activity. Our data confirm this activity homology: 87% of compounds that bound LKB1 also bound AAK1. Similarly, the kinase domains of the EphA and EphB kinases share low overall sequence homology with PDGFRB (19â26% identity). When only the 26 residues of the ATP binding site are considered, EphA and EphB kinases are more similar to PDGFRB (â¼50% identity). Despite their dissimilarity in sequence, some EphA and EphB kinases had high activity homology with PDGFRB. For example, EphB1 has low sequence similarity to PDGFRB but high activity homology; 75% of EphB1 actives were also active in PDGFRB (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). EphA6, in contrast, has the same level of sequence similarity to PDGFRB as EphB1 but much lower activity homology: only 46% of EphA6 actives showed activity in PDGFRB. One explanation for this disparity is the identity of the so-called gatekeeper, a residue in the active site that can affect kinase selectivity.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Both EphB1 and PDGFRB have a threonine in the gatekeeper position, while EphA6 has a valine (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). This single position may account for the greater activity homology of EphB1 to PDGFRB than EphA6.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Sequence Similarity and Activity Homology of EphB1 and EphA6 to PDGFR</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center" char=".">PDGFRB sequence identity (%)</th><th class="colsep0 rowsep0" align="center" char=".">PDGFRB binding site sequence identity (%)</th><th class="colsep0 rowsep0" align="center" char=".">PDGFRB activity homology (%)</th><th class="colsep0 rowsep0" align="center">gatekeeper</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDGFR</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="left">T</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EphB1</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">75</td><td class="colsep0 rowsep0" align="left">T</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EphA6</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="left">V</td></tr></tbody></table></div></div><div class="NLM_p last">Activity homology can be used to identify kinases that may present selectivity issues when targeting a specific kinase. For example, 20 kinases had >90% activity homology with EphB2 (i.e., they bound to 90% or more of the 372 compounds that bound EphB2). These 20 activity homologues include three EphA and three EphB kinases, which as close sequence homologues would be expected to have high activity similarity to EphB2, but they also include more distantly related tyrosine kinases such as LCK (98%), ARG (95%), and FGR (91%).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Potency and Selectivity</h3><div class="NLM_p">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A plots maximum binding activity versus selectivity [SI(13) (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for the definition)] for very potent compounds. Because binding is measured as percent control, a compound that is more potent has a lower value of maximum activity. As seen in the figure, compounds that are slightly less potent bind very potently to fewer kinases. Compounds with maximum activity of 4â5% control all have SI(13) values of <0.04 (i.e., they bind potently to 16 or fewer kinases) and a mean SI(13) value of 0.008 (i.e., on average they bind potently to three kinases). In contrast, compounds that are slightly more potent can bind with high affinity to more kinases. As potency increases in the plot (maximum activity decreases), the range of SI(13) values increases as well. Compounds with maximum activity of 0â1% control have a mean SI(13) value of 0.077, indicating potent binding to 31 kinases on average, and 25% of compounds in this bin have SI(13) values of â¥0.10. Compounds that are more potent generally acquire activity for multiple kinases,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> probably because of the conservation of ATP binding site features among kinases.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Plots of potency vs selectivity. (A) Maximum binding activity vs selectivity index [SI(13)] for potent compounds. More potent compounds have lower values of maximum binding activity, and more selective compounds have lower selectivity indices. The maximum binding values were binned using a bin width of 1.0, and the SI(13) values were plotted as a box plot. For each bin, the top of the corresponding box indicates the 75th percentile of SI(13), the bottom of the box indicates the 25th percentile, and the white line inside the box indicates the mean value. The number of compounds in each bin and the mean SI(13) value are shown in the table below the plot. (B) Minimum IC<sub>50</sub> vs selectivity index [SI(100 nM)] for compounds tested in at least 10 in-house kinase functional assays. The minimum IC<sub>50</sub> values were binned using a bin width of 20 nM, and the SI(100 nM) values for each bin were plotted as a box plot as in panel A. (C) Minimum IC<sub>50</sub> vs log(IC<sub>50</sub> ratio) for compounds tested in at least 10 in-house kinase functional assays. See the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for calculation of the log(IC<sub>50</sub> ratio) values. Plots were generated with TIBCO Spotfire version 2.2. The scatter plots corresponding to panels A and B are included in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A plots the trend of increased potency correlating with decreased selectivity for the data set as a whole, for which the more potent and less potent compounds being compared include structurally diverse molecules. The same trend holds true for pairs of structurally similar compounds that vary in potency; 20003 pairs of structurally similar compounds in which both compounds are very potent but one compound is more potent than the other were assembled. In 14298 cases (71.5%), the more potent compound is also less selective. Compounds <b>1</b> and <b>2</b> are one such pair: both are imidazo[1,2-<i>a</i>]thieno[3,2-<i>e</i>]pyrazines and were designed to inhibit IKKÎ².<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> They differ only in the substitution pattern of the pendant phenyl (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). Compound <b>1</b> shows potent activity in five Ambit BioSciences assays and is most potent in the IKKÎ² assay, with an activity of 0.2% control. Compound <b>2</b>, although structurally similar to <b>1</b>, is both more selective and less potent and bound only IKKÎ² at 4.8% control.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures and kinome binding profiles of a representative pair of compounds. The structures of compounds <b>1</b> and <b>2</b> are shown at the left and their binding activities at the right. Binding data are shown for all targets that were bound by compounds <b>1</b> and <b>2</b> (â¤13% control).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Importantly, the selectivityâpotency trend outlined above is also observed with a panel of kinase enzyme assays. Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B shows the correlation of greater potency with decreased selectivity [SI(100 nM)] for this data set. Both panels A and B of Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> highlight the difficulty of achieving potency and selectivity for a single target. It is critical to note that in these plots, selectivity is calculated as the fraction of assays for which a compound is active below a given threshold. The relative activity of the compound for different targets that meet the activity threshold is ignored. This trend is particularly relevant in the hit identification stage of drug discovery programs.</div><div class="NLM_p last">An alternative measure of kinase selectivity commonly used with functional data is a compoundâs fold selectivity for its primary target over other, secondary targets. Even if a compound potently inhibits a number of kinases, it may be considered selective if its activity for the target of interest is 1 order of magnitude greater than its activity for any other kinase. To quantify this relative selectivity, we compute the log of the ratio of the compoundâs second-most potent IC<sub>50</sub> (for kinase 2) to its most potent IC<sub>50</sub> (for kinase 1). A higher log(IC<sub>50</sub> ratio) indicates greater selectivity; a compound with a log(IC<sub>50</sub> ratio) value of 1 exhibits 10-fold selectivity for its primary target (kinase 1). When the log(IC<sub>50</sub> ratio) is plotted versus potency, only a weak correlation is seen between selectivity and potency (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C), with the average log(IC<sub>50</sub> ratio) exhibiting a slight decrease as potency decreases. This confirms that within a background of activity against a number of kinases, compounds can nonetheless be made to be extremely potent and relatively selective for a single target during the course of lead optimization.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Structural Features of Nonselective and Selective Compounds</h3><div class="NLM_p">The data set includes compounds that span the selectivity spectrum from 0.002 â¤ SI(33) â¤ 0.76. A set of 286 compounds are particularly nonselective, with moderate or potent activity for 40% or more of assayed kinases. To elucidate what structural features may confer such nonselective kinase activity, the physicochemical properties of nonselective compounds were compared to the properties of the SP set. Both the SP set and the nonselective compounds were filtered by structural similarity to avoid any bias introduced by including multiple similar structures. Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a> lists the structural descriptors used and the coefficient for each descriptor, which indicates how well-separated the descriptor distributions are for the selective and nonselective classes.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Physicochemical Descriptors and Their Separation Coefficients for Selective and Nonselective Compounds<a class="ref internalNav" href="#tbl4-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">descriptor</th><th class="colsep0 rowsep0" align="center">coefficient</th><th class="colsep0 rowsep0" align="center">descriptor</th><th class="colsep0 rowsep0" align="center">coefficient</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of polar hydrogens</td><td class="colsep0 rowsep0" align="left">0.39</td><td class="colsep0 rowsep0" align="left">no. of heavy atoms</td><td class="colsep0 rowsep0" align="left">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of rings</td><td class="colsep0 rowsep0" align="left">0.36</td><td class="colsep0 rowsep0" align="left">monopole</td><td class="colsep0 rowsep0" align="left">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">quadrupole <i>Y</i></td><td class="colsep0 rowsep0" align="left">0.35</td><td class="colsep0 rowsep0" align="left">calculated LogP (VolSurf)</td><td class="colsep0 rowsep0" align="left">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of NH and OH groups</td><td class="colsep0 rowsep0" align="left">0.31</td><td class="colsep0 rowsep0" align="left">quadrupole <i>Z</i></td><td class="colsep0 rowsep0" align="left">0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of N and O atoms</td><td class="colsep0 rowsep0" align="left">0.28</td><td class="colsep0 rowsep0" align="left">molecular volume (VolSurf)</td><td class="colsep0 rowsep0" align="left">0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of amides</td><td class="colsep0 rowsep0" align="left">0.10</td><td class="colsep0 rowsep0" align="left">molecular surface area (VolSurf)</td><td class="colsep0 rowsep0" align="left">0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of halogens</td><td class="colsep0 rowsep0" align="left">0.08</td><td class="colsep0 rowsep0" align="left">MW</td><td class="colsep0 rowsep0" align="left">0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">quadrupole <i>X</i></td><td class="colsep0 rowsep0" align="left">0.05</td><td class="colsep0 rowsep0" align="left">no. of rotors</td><td class="colsep0 rowsep0" align="left">0.00</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl4-fn1"><div class="footnote" id="tbl4-fn1"><sup>a</sup><p class="last">Additional descriptors and coefficients are given in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</p></div></div></div><div class="NLM_p">As shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, no descriptor completely separates the two classes; for each structural feature, there is significant overlap between the distributions for selective and nonselective compounds. Nonetheless, the results do highlight features that are more prominent in nonselective compounds. One of the descriptors that most significantly discriminates between the two classes is the number of rings: only 9.6% of selective compounds have at least 6 rings, while 33% of nonselective compounds do (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A). Another feature that tends to differ between the two classes is a shape descriptor that corresponds to the âwidthâ of the molecules. Steric quadroples describe the extents of a molecule, with the <i>X</i>, <i>Y</i>, and <i>Z</i> quadrupoles mapping to the length, width, and height, respectively. Nonselective compounds have larger values of the <i>Y</i> steric quadrupole, indicating that they may span the width of the ATP binding site more fully than selective compounds and thereby have the potential to interact with more residues. In addition, nonselective compounds are more likely to have many polar hydrogens: 23% of selective compounds have at least four polar hydrogens, while 59% of nonselective compounds have at least four polar hydrogens (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B). One possible explanation is that the greater number of hydrogen bond donors in nonselective compounds increases the likelihood of forming hydrogen bonds with the residues lining the ATP binding site in a greater variety of protein conformations. Many, though not all, of the additional polar hydrogens in nonselective compounds are due to the more frequent presence of primary amines, which have two polar hydrogens each. Positively charged amines can potentially interact with the catalytic aspartate and with the negatively charged residues that coordinate Mg<sup>2+</sup> ions when ATP is bound in the active site.</div><div class="NLM_p">Interestingly, no significant difference is seen in the molecular weight (MW) distributions of the two compound sets. It might be expected that larger compounds would tend to be more specific, because functional groups are typically added to a molecular core to achieve greater specificity. In this data set, however, no such trend is observed. The MW distributions for both selective and nonselective compounds are skewed toward higher molecular weights, with a slightly increased skew for nonselective compounds. 68% of nonselective compounds have molecular weights of >400, as do 59% of selective compounds. As one can see in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>C, the MW distribution of the nonselective compounds does not differ substantially from that of the selective compounds.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structural characteristics of selective and nonselective compounds: (A) number of rings, (B) number of polar hydrogens, and (C) molecular weight. The distributions are shown for the filtered nonselective and SP sets. Plots were prepared with Excel 2003.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Chemotypes and Specificity</h3><div class="NLM_p">Using the chemotype assignment scheme described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>, 34 chemotypes are represented in the SP set by at least 50 examples. Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a> plots the frequency with which each chemotype is active (â¤13% control) against each kinase family/subfamily. While there are gaps in the family coverage, many chemotypes have at least a few examples that are active against every family. An equivalent plot with less selective compounds included has even fewer gaps (data not shown). For many of the chemotypes, the defining substructure is a heteroatomic ring system that can in principle bind to the hinge region of the ATP binding site. For these chemotypes, the common structural motif can be viewed as a core scaffold, and the members of the chemotype differ in the identity of the substituents appended to the core. Because most chemotypes have representatives that can bind to most kinase families, the results suggest that for a specific compound, the identity of the substituents may drive its activity pattern as much or more than the identity of the core. Some chemotypes, such as the pyridines and 1<i>H</i>-pyrazoles, have relatively evenly distributed activities across many different families, while other chemotypes are skewed toward a particular family. For instance, 74% of the indolin-2-ones are active against NAK kinases (283 of 318), and 83% (318 of 383) exhibit GSK activity. This does not necessarily reflect an intrinsic affinity of this particular core for the NAK and GSK families, because the screening collection contains many structurally similar analogues and does not constitute an unbiased set. Despite its skewed target distribution, the indolin-2-one core is capable of binding many different kinase families (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). Our data suggest that many different core scaffolds can potentially be modified to produce selective compounds for most kinase targets.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Kinase families and chemotype specificity. Thirteen chemotypes with â¥50 representatives in the SP set are represented along the <i>X</i>-axis, and kinase families are listed on the <i>Y</i>-axis. The color of each rectangle indicates the number of compounds in the given chemotype that potently inhibit one or more kinases in the specified family. Plots were prepared with TIBCO Spotfire version 2.2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Structural Similarity and Profile Similarity</h3><div class="NLM_p">Chemically similar compounds would be expected to inhibit the same targets and thus to have similar activity profiles. Bamborough et al.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> observed that structural similarity correlated with activity profile similarity in their data set, and our larger compound set follows the same trend. Profileâprofile similarities were calculated as a continuous Tanimoto score as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. A higher Tanimoto coefficient indicates greater profileâprofile similarity. Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a> plots the CDF of the activity profile similarity (CDF<sub>P</sub>) for pairs of compounds that were binned by their structural similarity [Tc (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>)]. As shown in the figure, pairs of compounds that have more similar structures tend to have more similar activity profiles, and therefore, the CDFs are shifted to the right as structural similarity increases. For example, approximately half (52%) of compound pairs with moderate structural similarity (0.5â0.6) have an activity profile similarity of >0.73. In contrast, 83% of highly similar compound pairs (structural similarity of >0.9) have activity profile similarities in that range.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. CDF of profile similarity (CDF<sub>P</sub>) by structural similarity bin. Pairs of compounds were binned by structural similarity (Tc). For the compound pairs in each structural similarity bin, the cumulative distribution function of profile similarity is plotted as a separate function.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although structurally similar compounds tend to have similar activity profiles across the entire panel (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>), the likelihood that structural neighbors will have similar activities for a particular kinase differs from target to target. For each target, we identified compounds that are structurally similar to compounds that bind the target and computed the fraction of these structural neighbors that themselves bind the target. The distribution of the neighbor hit rates by target, shown in Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>A, is centered around 22%. On average, â¼22% of structural analogues of a known active compound can be expected to be active. As the number of active compounds for a target increases, the neighbor hit rate also increases (Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>B); if a target is inhibited by relatively few compounds, a smaller fraction of structural analogues of the active compounds are themselves active.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Neighbor hit rates. (A) Distribution of neighbor hit rates over the set of kinase targets. The neighbor hit rate is defined as the percent of structural analogues of a targetâs actives that are themselves active for the same target. (B) For each target, the number of hits is plotted vs the neighbor hit rate. The targets are colored by kinase class.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18843" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18843" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Kinome Profiling and Target âDruggabilityâ</h3><div class="NLM_p">Recent studies have described Ambit KinomeScan screening results for small sets of 200â500 structurally diverse compounds that were tested against panels of 119â317 kinases.<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10-12)</a> Our compound set is much larger and, while it includes a structurally diverse set, was selected to include multiple representatives of many structural clusters. While the distribution of hit rates found here largely agrees with that found by Bamborough et al., there are significant differences in hit rates for a number of kinases that are likely due in large part to the differences in the compound sets. GAK, JNK3, CK1D (CSNK1D), CK1E (CSNK1E), and JNK3 had exceptionally high hit rates in the earlier study, while in our data set, GAK had a moderately high hit rate (8.1% of all compounds) and JNK3, CK1D, and CK1E all had low hit rates (1.4, 2.6, and 3.6%, respectively). In a more typical case in which our hit rates confirm previous findings, kinases of the MEKK/MAP3K family had very few hits in our collection, in agreement with the earlier studyâs low hit rates for MAP3K5 (ASK1) and MAP3K4. The lack of hits in two such different compound sets indicates that these kinases may be difficult targets for drug discovery efforts.</div><div class="NLM_p">In contrast, the kinases of the PDGFR family had high hit rates in both screens. This led us to examine whether PDGFR kinases have any unusual binding site features that may contribute to their binding versatility. Alignment of the PDGFR kinases reveals a conserved Cys-x-Tyr-Gly-Asp motif in the extended hinge region at the outer edge of the binding site. While the Gly and Asp of the motif are commonly found in other kinase families, the Cys-x-Tyr sequence is unique to the PDGFR family. The large tyrosine is in a site commonly occupied by Gly. It is not obvious why this unusual combination of residues would allow PDGFR kinases to bind to structurally diverse ligands. One possibility is that the Cys-x-Tyr motif may allow sampling of alternative conformations of the extended hinge in PDGFR kinases as compared to other kinases.</div><div class="NLM_p last">The neighbor hit rate, coupled with the hit rate for all compounds, provides a measure of a kinaseâs potential tractability as a drug target. The correlation between the two rates suggests that the more hits that are found in an initial kinase screen, the more likely it is that analogues of the first hits will themselves be active. Interestingly, our finding that on average 22% of analogues of active compounds tend to be actives themselves agrees with an earlier study showing that 30% of close analogues (Tanimoto coefficient of â¥0.85) of hits in high-throughput assays are themselves active.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, the range we find is 0â54%, suggesting that this metric should be computed for each target.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Implications for Kinase-Focused Screening Collections</h3><div class="NLM_p">Many chemical substructures have been identified as kinase-specific privileged structures; i.e., compounds containing these structure fragments are enriched for kinase activity as compared to other target classes.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24">(21-24)</a> These fragments have flat aromatic ring systems containing hydrogen bond acceptors and donors that can interact with the hinge residues of the ATP binding site. The chemotypes in our compound set represent many such substructures, including well-known kinase-binding scaffolds such as pyrrolo[2,1-<i>f</i>][1,2,4]triazines<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and 1<i>H</i>-pyrazoles.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> While these core substructures may indeed bind preferentially to kinases as compared to other target classes such as GPCRs or ion channels, within the kinome the concept of privileged substructures may be less applicable. Our data show that, in general, most chemotypes can bind selectively to a broad range of kinases; we do not find many chemotypes that are capable of binding to only a few specific kinase families. This indicates that kinase family specificity depends less on the identity of the core scaffold and more on the specific substituents appended to the core.</div><div class="NLM_p">Various procedures for selecting compound sets for high-throughput screening (HTS) of kinase targets have been described.<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31 ref32 ref33">(27-33)</a> By screening a small, diverse collection of compounds that are enriched for compounds likely to bind kinases, one can find leads for novel kinase targets without conducting a more extensive HTS campaign of more than 1 million compounds. To assemble such kinase-focused screening libraries, candidate compounds are assigned a score reflecting their predicted kinase binding potential. Such scores may incorporate various similarity metrics based on comparisons to known kinase inhibitors,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> docking scores or interaction fingerprints for one or more kinase structures,<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30-32)</a> descriptor-based activity predictions,<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> and/or activities from previous kinase screens. High-scoring compounds are typically identified and then passed through a diversity filter. This step ensures that multiple diverse scaffolds are included and that no single scaffold is overrepresented in the focused library.</div><div class="NLM_p">Our results support a potentially complementary paradigm. In this approach, a small number of known kinase scaffolds would be selected on the basis of their synthetic accessibility and potential for conferring intellectual property. The library would comprise as many analogues containing these scaffolds as possible. If a diversity pick were required to select a subset from many analogues, it would be based only on comparison of the structural fragments appended to the core scaffold. Such an approach would take advantage of the proven kinase binding potential of known cores and allow exploration of more diverse fragments in the noncore regions than may be found in standard kinase-focused collections. Scaffold-oriented kinase libraries in line with this paradigm have been synthesized and successfully screened for active kinase inhibitors.<a onclick="showRef(event, 'ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36">(34-36)</a> Of particular note is a recent effort to systematically identify and build libraries around novel scaffolds with potential hinge-binding elements that were not represented in the kinase literature.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div><div class="NLM_p last">A scaffold-oriented screening strategy may be especially appropriate for kinases with low neighbor hit rates. Harper et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> have shown that a targetâs neighbor hit rate has a large impact on the number of compounds that need to be screened to find an expected number of leads. For kinases with high neighbor hit rates, hits will likely be found by screening a compound collection of many diverse chemotypes (where each is represented by a small number of compounds). Because analogues of active compounds have a high probability of being active, active chemotypes can be identified even if they are sparsely represented in the screening collection. For a kinase with a low neighbor hit rate, however, even if a particular scaffold can bind the target of interest, many analogues containing the scaffold may need to be sampled to identify a lead compound. Because most chemotypes have the potential to bind most kinase families, screening a large number of analogues of a single chemotype may be more productive for more difficult kinase targets than screening fewer examples of more chemotypes.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Molecular Determinants of Selectivity</h3><div class="NLM_p last">In general, we find that selective and nonselective compounds have overlapping profiles of physicochemical features. We did not see a significant difference in the molecular weights of nonselective compounds as compared to selective compounds, in contrast to an earlier work that found that molecular weight greatly affected selectivity.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> We did find that extremely nonselective compounds tend to have more rings and more polar hydrogens than selective compounds, suggesting that these features should be avoided when feasible in designing new kinase inhibitors. Initial attempts to select feature combinations that would predict compound selectivity have shown some promise (data not shown). Future refinement of the models is expected to improve their predictive capabilities.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64270" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64270" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The kinome profiling data described in this work can be used to evaluate the potential druggability of kinase targets and to identify potential off-target liabilities. Because of the observed variation in target tractability and our finding that most chemotypes can bind to most kinase families, we propose a scaffold-oriented screening strategy that complements other methods of designing kinase-focused libraries. The observed variability in hit rates and neighbor hit rates for the highly conserved kinases suggests that targets in less conserved classes may have similar screening variability. The considerations that we outline for kinase screening strategies may therefore apply to target classes such as GPCRs and ion channels as well.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26868" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26868" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Binding Assays</h3><div class="NLM_p last">The binding assays were conducted by Ambit BioSciences as described previously.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> Compounds were screened at a single concentration of 1 Î¼M. The assays measure a compoundâs ability to inhibit binding of a bait ligand, with 0% control corresponding to full inhibition and 100% control to no inhibition. Three different activity cutoffs were used in this work to define moderate, potent, and extremely potent binding. On the basis of comparisons between Ambit BioSciences binding data and in-house functional assay data (see the following section), moderate activity was defined as â¤33% control, potent activity as â¤13% control, and extremely potent activity as â¤5% control. The hit rates listed in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> were calculated with the potent activity threshold (â¤13% control). For each compound, a selectivity index at an activity threshold <i>x</i>, SI(<i>x</i>), is calculated as the number of assays for which the compound was active â¤ <i>x</i>% control divided by the total number of kinases for which the compound was assayed. The range of potential SI values is 0â1; an SI of 0 indicates that no kinases were bound at the given threshold, and an SI of 1 means that all kinases were bound.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Functional CDK2, FLT3, and EGFR Assays</h3><div class="NLM_p last">The kinase functional assays were performed in 384-well plates using a 30 Î¼L reaction volume. The reactions were run for 1 h in reaction buffer [100 mM HEPES (pH 7.4), 10 mM MgCl<sub>2,</sub> 0.015% Brij35, and 4 mM DTT]. The reaction was initiated by combination of kinase CDK2/cyclin E (Proqinase), FLT3 (Invitrogen), or HER1/EGFR (Invitrogen) and 1.5 Î¼M peptide substrate fluorescein-QSPKKG-NH<sub>2</sub> (for CDK2/cyclin E) or fluorescein-EAIYAAPFAKKK-NH<sub>2</sub> (for FLT3 and HER1/EGFR) in the presence of ATP in wells containing the compound to be tested. The ATP concentrations used in these assays were 30 Î¼M for CDK2, 200 Î¼M for FLT3, and 6 Î¼M for HER1/EGFR. The final concentration of DMSO in these reaction mixtures was 1.6%. The reaction mixture was analyzed on the Caliper LabChip 3000 (Caliper, Hopkinton, MA) by the electrophoretic mobility shift of the fluorescent substrate and phosphorylated product. Inhibition data were calculated by comparison to no enzyme control reactions for 100% inhibition and DMSO-only reactions for 0% inhibition. Doseâresponse curves were generated to determine the concentration required to inhibit 50% of kinase activity (IC<sub>50</sub>). Compounds were dissolved at 10 mM in dimethyl sulfoxide (DMSO) and evaluated at 11 concentrations, each in duplicate. IC<sub>50</sub> values were derived by nonlinear regression analysis.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Selection of Activity Cutoffs</h3><div class="NLM_p last">The Logit transformation is related mathematically to the Hill equation with the assumption that the Hill slope is 1:<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/jm-2010-01195a_m002.gif" alt="" /></img></span>which can be rearranged to<span class="NLM_disp-formula" id="ueq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/jm-2010-01195a_m003.gif" alt="" /></img></span>which is mathematically equivalent to<span class="NLM_disp-formula" id="ueq3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/jm-2010-01195a_m004.gif" alt="" /></img></span>where Maxeffect and Mineffect are the maximum and minimum effects possible in the assay, respectively. In the case of the Ambit BioSciences binding assays, the maximum effect is 0% control, the minimum effect is 100% control, and the dose is 1 Î¼M. The following equation was utilized to calculate the estimated IC<sub>50</sub> values (red lines shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>).<span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/jm-2010-01195a_m005.gif" alt="" /></img><span class="labelSpan">(2)</span></span></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Structural Similarity</h3><div class="NLM_p last">To compare compounds by their chemical structures, we retrieved the connection table for each compound from the corporate database. An atom-pair fingerprint<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> was generated for each structure. The Tanimoto coefficient (Tc) was used as a measure of structural similarity.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Two compounds are considered structurally similar if their fingerprints have a Tc of â¥0.7. The all-against-all matrix of compound pairwise similarities was calculated.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Neighbor Hit Rates</h3><div class="NLM_p last">Structural neighbors of a compound make up the set of compounds with Tc values of â¥0.7 to the query structure. A compound is not considered its own neighbor even though its Tanimoto similarity to itself is equal to 1. For each kinase, all structural neighbors of all potently active compounds (â¤13% control) were identified, and the combined neighbor list was filtered to remove duplicates. The neighbor hit rate is defined as the number of structural neighbors that were active for the target of interest (â¤13% control) divided by the total number of structural neighbors.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> SP Set</h3><div class="NLM_p last">A total of 7790 compounds were identified on the basis of their selectivity and extreme potency. These compounds are collectively labeled the SP set. Selectivity denotes moderate or potent activity for at most 10% of the panel [i.e., SI(33) â¤ 0.1], and extreme potency is defined as activity for at least one kinase of â¤5% control and >0% control. Values of 0% control have been shown to have a higher false positive rate upon retesting than values greater than zero, especially in cases where a compound has 0% control activity for a single kinase and no non-zero activity for any other kinase (data not shown). Compounds were therefore required to have activity greater than zero and less than or equal to 5% control to be considered very potent and included in this set.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Potency and Selectivity</h3><div class="NLM_p last">To compare potency and selectivity, we found 11935 compounds that met the criterion for extreme potency (>0 and â¤5% control for at least one kinase), regardless of their selectivity. For these compounds, the SI(13) was calculated to measure selectivity using a potent activity threshold; 20003 pairs of structurally similar compounds were identified as described above (both compounds in each pair were extremely potent).</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Selectivity in Functional Assays</h3><div class="NLM_p last">A total of 12455 compounds that were tested in at least 10 in-house assays of kinase catalytic activity and had an IC<sub>50</sub> of â¤100 nM for at least one assay were identified. For each compound, a selectivity index, SI(100 nM), was calculated as the number of assays that gave IC<sub>50</sub> values of â¤100 nM divided by the total number of kinases for which the compound was assayed. Selectivity was also calculated for each compound as the log of the ratio of the second-most potent IC<sub>50</sub> to the most potent IC<sub>50</sub>. This measure describes the relative selectivity of the compound for its highest-affinity target and does not involve an activity cutoff.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Kinase Sequences and Alignments</h3><div class="NLM_p last">Ambit BioSciences targets were assigned to groups and families on the basis of the Manning classification scheme.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The DNA insert sequence of each target was obtained from Ambit BioSciences and translated into its corresponding protein sequence. Each pair of protein sequences was aligned with T-coffee version 6,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and the sequence identity was calculated as the number of identical aligned residues divided by the number of aligned positions. To compare the ATP binding sites of kinases in the PDGFR, EphA, and EphB families, we identified 26 binding site residues by visual inspection of a representative kinase crystal structure and mapped them to the corresponding positions in the pairwise sequence alignments.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Chemotype Assignment</h3><div class="NLM_p">ClassPharmer version 4.6<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> was used to systematically identify the common ring systems in a large database of known kinase inhibitors (data not shown). Two hundred ninety-seven ring systems that contain the hydrogen bonding characteristics of a kinase inhibitor were manually selected and converted to SMARTS patterns. The SMARTS generated from this procedure were used for substructure searching in Pipeline Pilot version 7.5.2<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> to assign compounds to âchemotypeâ classes. Compounds were annotated with all matching substructures and thus may belong to multiple chemotypes. The substructure motifs are not true chemotypes in the sense that they are not âcoresâ, and many matching structural fragments may be noncore substituents. However, most of the substructures used to define the chemotype classes are known to function as kinase-binding cores and can form the critical hydrogen bonds to the hinge residues of the ATP binding site.</div><div class="NLM_p last">Thirty-four chemotypes contained at least 50 representatives in the SP set. For each of these 34 chemotypes, for each kinase family, the number of compounds in the SP set that bound potently (â¤13% control) to at least one family member was tallied. These raw counts were converted to frequencies for each chemotype by dividing the count by the total number of SP compounds in each chemotype.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Activity Homology</h3><div class="NLM_p">The activity homology for each pair of kinases A and B was calculated as the prior probability of activity for kinase B given activity for kinase A, i.e., the percent of active compounds (â¤13% control) for kinase B that were also active for kinase A. An activity homology of 65% means that 65% of the compounds that were active for kinase B also had activity for kinase A.</div><div class="NLM_p last">Kinase pairs were binned by pairwise sequence identity (as described above), and the cumulative distribution function of activity homology (CDF<sub>H</sub>) was calculated separately for the kinase pairs in each sequence identity bin. CDF<sub>H<i>i</i></sub>(<i>x</i>) is the percent of kinase pairs in sequence bin <i>i</i> that have activity homology of â¤<i>x</i>.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Physicochemical Features of Selective and Nonselective Compounds</h3><div class="NLM_p">Two hundred eighty-six compounds were identified with an SI(33) of â¥0.4. For each of the 286 nonselective compounds and the 7790 compounds in the SP set, a single three-dimensional conformation was generated from the connection table using Omega version 2.3.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Simple derived properties such as molecular weight, the number of polar hydrogens, etc., were calculated with molProps, an in-house tool based on the OpenEye OEChem toolkit.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Additional physicochemical features were calculated with VolSurf version 4.1.4.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44, 45)</a> For each of the 90 total descriptors, the separation of the nonselective and selective classes was calculated using the mean-differences test. The separation coefficient for descriptor <i>i</i>, <i>S</i><sub><i>i</i></sub>, equals (avg<sub>s,<i>i</i></sub> â avg<sub>n,<i>i</i></sub>)<sup>2</sup>/(stddev<sub>s,<i>i</i></sub><sup>2</sup> + stddev<sub>n,<i>i</i></sub><sup>2</sup>), where avg<sub>s,<i>i</i></sub> and stddev<sub>s,<i>i</i></sub> are the average and standard deviation of the values of descriptor <i>i</i> for selective compounds, respectively, and avg<sub>n,<i>i</i></sub> and stddev<sub>n,<i>i</i></sub> are the average and standard deviation of the values of descriptor <i>i</i> for nonselective compounds, respectively.</div><div class="NLM_p last">The analysis was repeated with the selective and nonselective compounds clustered by structure to reduce the bias introduced by having large clusters of very similar compounds in the compound sets. Using the structural similarity matrix described above, group-average linkage clustering was applied to both the SP set and the set of nonselective compounds. Two similarity cutoffs were tried. When a tight similarity cutoff of 0.7 was employed, 3945 selective clusters (including singletons) and 152 nonselective clusters were found. A looser similarity cutoff of 0.5 yielded 1302 selective clusters and 61 nonselective clusters. For both cutoffs, a single representative was chosen randomly for each cluster and the separation coefficients were recalculated. The calculations were also conducted with all nonselective compounds included and only the SP set clustered by structure. While the magnitude of the separation coefficients differed depending on how compounds were clustered, the observed trends are the same in all cases and the same features emerge as important. The values shown were derived from the analysis with the tighter cutoff and both selective and nonselective compounds clustered.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Activity Profile Similarity</h3><div class="NLM_p">To compare the profile similarity of two compounds, we developed an algorithm based on the Tanimoto coefficient. The normal Tanimoto algorithm uses a fingerprint in which each bit is a binary response. For a fingerprint based on activity, an arbitrary activity cutoff must be set to separate the actives from the inactives. This introduces a large discontinuity when activities fall close to the cutoff value and can make the similarity results rather dependent on that value. Additionally, it was found that the large amount of negative activity data (i.e., lack of activity) in the results for these assays often overwhelms the positive data upon calculation of the Tanimoto coefficient, especially for more selective compounds. The Continuous Tanimoto algorithm was designed to address both of these issues.</div><div class="NLM_p">The Tanimoto algorithm was modified to remove this discontinuity at the cutoff threshold by making the contribution of one bit a linear function of the activities of one assay for the two molecules.</div><div class="NLM_p">An activity fingerprint is comprised of the values for each assay (% control). A continuous version of the Tanimoto index that uses the raw activity values rather than binning values into discrete active and inactive categories was developed. The Tanimoto profile similarity for compounds <b>1</b> and <b>2</b> is<span class="NLM_disp-formula" id="ueq4"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/jm-2010-01195a_m006.gif" alt="" /></img></span>where <i>N</i> is the total number of assays, <i>A</i><sub><b>1</b>,<i>i</i></sub> is the activity of compound <b>1</b> in assay <i>i</i>, and <i>A</i><sub><b>2</b>,<i>i</i></sub> is the activity of compound <b>2</b> in assay <i>i</i>. The constants <i>a</i> and <i>b</i> were empirically set to 10 and 1, respectively. The contribution for each assay is 10 if both compounds are 0% control, 1 if they are both 100% control, and 0 if one is 100% and the other 0%. For intermediate values, the contribution is a linear extrapolation. There is a normalization factor to put the results between 0 and 100% similarity.</div><div class="NLM_p">The ratio of <i>a</i> to <i>b</i> is the weight of the contribution given to active results versus inactive. A ratio of 1 generates similarity results that are determined primarily by the inactive data. This makes all highly selective compounds very similar, regardless of the few targets against which they are potent. A very high ratio would consider only the active data that are more appropriate for selective compounds. A low ratio works better for nonselective compounds. A ratio of 10 produced similarity results for most of our compounds that make intuitive sense upon comparison of histograms of the raw profile data.</div><div class="NLM_p last">Compound pairs were binned by pairwise structural similarity (Tc), and the cumulative distribution function of activity profile similarity (CDF<sub>P</sub>) was calculated separately for the compound pairs in each structural similarity bin. CDF<sub>P<i>i</i></sub>(<i>x</i>) is the percent of compound pairs in structural similarity bin <i>i</i> that have activity profile similarity of â¤<i>x</i>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35"><a href="/doi/suppl/10.1021/jm101195a">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04207" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04207" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Scatter plots of the data shown in panels A and B of Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>, the list of Ambit BioSciences targets and their group/family classification, hit rates for all compounds and for the SP set for all targets, separation coefficients for all physicochemical properties, neighbor hit rates for all targets, correspondence between Ambit BioSciences binding values and functional data for 37 additional assays, and pairwise activity homologies. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm101195a/suppl_file/jm101195a_si_001.pdf">jm101195a_si_001.pdf (324.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm101195a/suppl_file/jm101195a_si_002.pdf">jm101195a_si_002.pdf (2.61 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm101195a" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56536" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56536" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen R. Johnson</span> - <span class="hlFld-Affiliation affiliation">Computer-Assisted Drug Design, Applied Biotechnology, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a2d1d6c7d2cac7cc8cc8cdcaccd1cdcce2c0cfd18cc1cdcf"><span class="__cf_email__" data-cfemail="1c6f68796c74797232767374726f73725c7e716f327f7371">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shana L. Posy</span> - <span class="hlFld-Affiliation affiliation">Computer-Assisted Drug Design, Applied Biotechnology, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark A. Hermsmeier</span> - <span class="hlFld-Affiliation affiliation">Chemistry Informatics, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wayne Vaccaro</span> - <span class="hlFld-Affiliation affiliation">Discovery Chemistry, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Karl-Heinz Ott</span> - <span class="hlFld-Affiliation affiliation">Bioinformatics, Applied Biotechnology, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gordon Todderud</span> - <span class="hlFld-Affiliation affiliation">Lead Evaluation, Applied Biotechnology, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonathan S. Lippy</span> - <span class="hlFld-Affiliation affiliation">Lead Evaluation, Applied Biotechnology, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">George L. Trainor</span> - <span class="hlFld-Affiliation affiliation">Discovery Chemistry, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Deborah A. Loughney</span> - <span class="hlFld-Affiliation affiliation">Computer-Assisted Drug Design, Applied Biotechnology, Bristol-Myers Squibb Research and Development, P.O. Box 4000, Princeton, New Jersey 08543, United States</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70083" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70083" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Lawrence Williams for his critical reading of the manuscript. John Tokarski and David Langley contributed to the selection of compounds for the screening set. Tai Wong was instrumental in facilitating the collaboration with Ambit BioSciences. Songmei Xu, Randi Brown, Liang Schweizer, and Litao Zhang contributed to the collection of the functional data plotted in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21334" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21334" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 45 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">â</span> <span class="NLM_lpage">1934</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lj760kCU62rBA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Sheinerman, F. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giraud, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laoui, A.</span><span> </span><span class="NLM_article-title">High affinity targets of protein kinase inhibitors have similar residues at the positions energetically important for binding</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">1134</span><span class="NLM_x">â</span> <span class="NLM_lpage">1156</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=1134-1156&author=F.+B.+Sheinermanauthor=E.+Giraudauthor=A.+Laoui&title=High+affinity+targets+of+protein+kinase+inhibitors+have+similar+residues+at+the+positions+energetically+important+for+binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSheinerman%26aufirst%3DF.%2BB.%26aulast%3DGiraud%26aufirst%3DE.%26aulast%3DLaoui%26aufirst%3DA.%26atitle%3DHigh%2520affinity%2520targets%2520of%2520protein%2520kinase%2520inhibitors%2520have%2520similar%2520residues%2520at%2520the%2520positions%2520energetically%2520important%2520for%2520binding%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D352%26spage%3D1134%26epage%3D1156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Barker, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grundy, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godfrey, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlow, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Healy, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodburn, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curry, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarlett, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henthorn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, L.</span><span> </span><span class="NLM_article-title">Studies leading to the identification of ZD1839 (IRESSA): An orally active, selective epidermal growth factor receptor tyrosine kinase inhibitor targeted to the treatment of cancer</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1911</span><span class="NLM_x">â</span> <span class="NLM_lpage">1914</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1016%2FS0960-894X%2801%2900344-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=11459659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFylsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1911-1914&author=A.+J.+Barkerauthor=K.+H.+Gibsonauthor=W.+Grundyauthor=A.+A.+Godfreyauthor=J.+J.+Barlowauthor=M.+P.+Healyauthor=J.+R.+Woodburnauthor=S.+E.+Ashtonauthor=B.+J.+Curryauthor=L.+Scarlettauthor=L.+Henthornauthor=L.+Richards&title=Studies+leading+to+the+identification+of+ZD1839+%28IRESSA%29%3A+An+orally+active%2C+selective+epidermal+growth+factor+receptor+tyrosine+kinase+inhibitor+targeted+to+the+treatment+of+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Studies leading to the identification of ZD1839 (iressa): an orally active, selective epidermal growth factor receptor tyrosine kinase inhibitor targeted to the treatment of cancer</span></div><div class="casAuthors">Barker, A. J.; Gibson, K. H.; Grundy, W.; Godfrey, A. A.; Barlow, J. J.; Healy, M. P.; Woodburn, J. R.; Ashton, S. E.; Curry, B. J.; Scarlett, L.; Henthorn, L.; Richards, L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1911-1914</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">This paper describes the development of the epidermal growth factor receptor tyrosine kinase inhibitor ZD1839 from a lead series of 4-anilinoquinazoline compds.  ZD1839 has suitable properties for use as a clin. effective drug and shows activity against human tumors.  In particular, the use of pharmacokinetic data in the development of ZD1839 is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvsJ1FTqEwvbVg90H21EOLACvtfcHk0lj760kCU62rBA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltFylsLo%253D&md5=ebb210453ade246e43de9fd07ae1d581</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2801%2900344-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252801%252900344-4%26sid%3Dliteratum%253Aachs%26aulast%3DBarker%26aufirst%3DA.%2BJ.%26aulast%3DGibson%26aufirst%3DK.%2BH.%26aulast%3DGrundy%26aufirst%3DW.%26aulast%3DGodfrey%26aufirst%3DA.%2BA.%26aulast%3DBarlow%26aufirst%3DJ.%2BJ.%26aulast%3DHealy%26aufirst%3DM.%2BP.%26aulast%3DWoodburn%26aufirst%3DJ.%2BR.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DCurry%26aufirst%3DB.%2BJ.%26aulast%3DScarlett%26aufirst%3DL.%26aulast%3DHenthorn%26aufirst%3DL.%26aulast%3DRichards%26aufirst%3DL.%26atitle%3DStudies%2520leading%2520to%2520the%2520identification%2520of%2520ZD1839%2520%2528IRESSA%2529%253A%2520An%2520orally%2520active%252C%2520selective%2520epidermal%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520inhibitor%2520targeted%2520to%2520the%2520treatment%2520of%2520cancer%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2001%26volume%3D11%26spage%3D1911%26epage%3D1914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Wakeling, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guy, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodburn, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curry, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, K. H.</span><span> </span><span class="NLM_article-title">ZD1839 (Iressa): An orally active inhibitor of epidermal growth factor signaling with potential for cancer therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">5749</span><span class="NLM_x">â</span> <span class="NLM_lpage">5754</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=12384534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD38XotFyktr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=5749-5754&author=A.+E.+Wakelingauthor=S.+P.+Guyauthor=J.+R.+Woodburnauthor=S.+E.+Ashtonauthor=B.+J.+Curryauthor=A.+J.+Barkerauthor=K.+H.+Gibson&title=ZD1839+%28Iressa%29%3A+An+orally+active+inhibitor+of+epidermal+growth+factor+signaling+with+potential+for+cancer+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">ZD1839 (Iressa): An orally active inhibitor of epidermal growth factor signaling with potential for cancer therapy</span></div><div class="casAuthors">Wakeling, Alan E.; Guy, Simon P.; Woodburn, Jim R.; Ashton, Susan E.; Curry, Brenda J.; Barker, Andrew J.; Gibson, Keith H.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5749-5754</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) is a promising target for anticancer therapy because of its role in tumor growth, metastasis and angiogenesis, and tumor resistance to chemotherapy and radiotherapy.  We have developed a low-mol.-wt. EGFR tyrosine kinase inhibitor (EGFR-TKI), ZD1839 (Iressa).  ZD1839, a substituted anilinoquinazoline, is a potent EGFR-TKI (IC50 = 0.033 Î¼M) that selectively inhibits EGF-stimulated tumor cell growth (IC50 = 0.054 Î¼M) and that blocks EGF-stimulated EGFR autophosphorylation in tumor cells.  In studies with mice bearing a range of human tumor-derived xenografts, ZD1839 given p.o. once a day inhibited tumor growth in a dose-dependent manner.  The level of expression of EGFR did not det. xenograft tumor sensitivity to ZD1839.  Long-term ZD1839 (>3 mo) treatment of mice bearing A431 xenografts was well tolerated, and ZD1839 completely inhibited tumor growth and induced regression of established tumors.  No drug-resistant tumors appeared during ZD1839 treatment, but some tumors regrew after drug withdrawal.  These studies indicate the potential utility of ZD1839 in the treatment of many human tumors and indicate that continuous once-a-day p.o. dosing might be a suitable therapeutic regimen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkoHZ9zL06abVg90H21EOLACvtfcHk0lgAqUmmNBCrNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotFyktr8%253D&md5=f5f86c652786db476a2d23b5bbd1a9e9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWakeling%26aufirst%3DA.%2BE.%26aulast%3DGuy%26aufirst%3DS.%2BP.%26aulast%3DWoodburn%26aufirst%3DJ.%2BR.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DCurry%26aufirst%3DB.%2BJ.%26aulast%3DBarker%26aufirst%3DA.%2BJ.%26aulast%3DGibson%26aufirst%3DK.%2BH.%26atitle%3DZD1839%2520%2528Iressa%2529%253A%2520An%2520orally%2520active%2520inhibitor%2520of%2520epidermal%2520growth%2520factor%2520signaling%2520with%2520potential%2520for%2520cancer%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2002%26volume%3D62%26spage%3D5749%26epage%3D5754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Wilhelm, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNabola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shujath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gawlak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eveleigh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowley, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adnane, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auclair, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gedrich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voznesensky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riedl, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Post, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trail, P. A.</span><span> </span><span class="NLM_article-title">BAY 43-9006 exhibits broad spectrum oral antitumor activity and targets the RAF/MEK/ERK pathway and receptor tyrosine kinases involved in tumor progression and angiogenesis</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">7099</span><span class="NLM_x">â</span> <span class="NLM_lpage">7109</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=7099-7109&author=S.+M.+Wilhelmauthor=C.+Carterauthor=L.+Tangauthor=D.+Wilkieauthor=A.+McNabolaauthor=H.+Rongauthor=C.+Chenauthor=X.+Zhangauthor=P.+Vincentauthor=M.+McHughauthor=Y.+Caoauthor=J.+Shujathauthor=S.+Gawlakauthor=D.+Eveleighauthor=B.+Rowleyauthor=L.+Liuauthor=L.+Adnaneauthor=M.+Lynchauthor=D.+Auclairauthor=I.+Taylorauthor=R.+Gedrichauthor=A.+Voznesenskyauthor=B.+Riedlauthor=L.+E.+Postauthor=G.+Bollagauthor=P.+A.+Trail&title=BAY+43-9006+exhibits+broad+spectrum+oral+antitumor+activity+and+targets+the+RAF%2FMEK%2FERK+pathway+and+receptor+tyrosine+kinases+involved+in+tumor+progression+and+angiogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DS.%2BM.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DWilkie%26aufirst%3DD.%26aulast%3DMcNabola%26aufirst%3DA.%26aulast%3DRong%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DVincent%26aufirst%3DP.%26aulast%3DMcHugh%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DShujath%26aufirst%3DJ.%26aulast%3DGawlak%26aufirst%3DS.%26aulast%3DEveleigh%26aufirst%3DD.%26aulast%3DRowley%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DAdnane%26aufirst%3DL.%26aulast%3DLynch%26aufirst%3DM.%26aulast%3DAuclair%26aufirst%3DD.%26aulast%3DTaylor%26aufirst%3DI.%26aulast%3DGedrich%26aufirst%3DR.%26aulast%3DVoznesensky%26aufirst%3DA.%26aulast%3DRiedl%26aufirst%3DB.%26aulast%3DPost%26aufirst%3DL.%2BE.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DTrail%26aufirst%3DP.%2BA.%26atitle%3DBAY%252043-9006%2520exhibits%2520broad%2520spectrum%2520oral%2520antitumor%2520activity%2520and%2520targets%2520the%2520RAF%252FMEK%252FERK%2520pathway%2520and%2520receptor%2520tyrosine%2520kinases%2520involved%2520in%2520tumor%2520progression%2520and%2520angiogenesis%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D7099%26epage%3D7109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Lombardo, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrish, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Behnia, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castaneda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornelius, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doweyko, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairchild, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inigo, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamath, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marathe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schieven, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tokarski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wityak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borzilleri, R. M.</span><span> </span><span class="NLM_article-title">Discovery of N-(2-chloro-6-methyl- phenyl)-2-(6-(4-(2-hydroxyethyl)- piperazin-1-yl)-2-methylpyrimidin-4- ylamino)thiazole-5-carboxamide (BMS-354825), a dual Src/Abl kinase inhibitor with potent antitumor activity in preclinical assays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">6658</span><span class="NLM_x">â</span> <span class="NLM_lpage">6661</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049486a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=6658-6661&author=L.+J.+Lombardoauthor=F.+Y.+Leeauthor=P.+Chenauthor=D.+Norrisauthor=J.+C.+Barrishauthor=K.+Behniaauthor=S.+Castanedaauthor=L.+A.+Corneliusauthor=J.+Dasauthor=A.+M.+Doweykoauthor=C.+Fairchildauthor=J.+T.+Huntauthor=I.+Inigoauthor=K.+Johnstonauthor=A.+Kamathauthor=D.+Kanauthor=H.+Kleiauthor=P.+Maratheauthor=S.+Pangauthor=R.+Petersonauthor=S.+Pittauthor=G.+L.+Schievenauthor=R.+J.+Schmidtauthor=J.+Tokarskiauthor=M.+L.+Wenauthor=J.+Wityakauthor=R.+M.+Borzilleri&title=Discovery+of+N-%282-chloro-6-methyl-+phenyl%29-2-%286-%284-%282-hydroxyethyl%29-+piperazin-1-yl%29-2-methylpyrimidin-4-+ylamino%29thiazole-5-carboxamide+%28BMS-354825%29%2C+a+dual+Src%2FAbl+kinase+inhibitor+with+potent+antitumor+activity+in+preclinical+assays"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm049486a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049486a%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DL.%2BJ.%26aulast%3DLee%26aufirst%3DF.%2BY.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DBarrish%26aufirst%3DJ.%2BC.%26aulast%3DBehnia%26aufirst%3DK.%26aulast%3DCastaneda%26aufirst%3DS.%26aulast%3DCornelius%26aufirst%3DL.%2BA.%26aulast%3DDas%26aufirst%3DJ.%26aulast%3DDoweyko%26aufirst%3DA.%2BM.%26aulast%3DFairchild%26aufirst%3DC.%26aulast%3DHunt%26aufirst%3DJ.%2BT.%26aulast%3DInigo%26aufirst%3DI.%26aulast%3DJohnston%26aufirst%3DK.%26aulast%3DKamath%26aufirst%3DA.%26aulast%3DKan%26aufirst%3DD.%26aulast%3DKlei%26aufirst%3DH.%26aulast%3DMarathe%26aufirst%3DP.%26aulast%3DPang%26aufirst%3DS.%26aulast%3DPeterson%26aufirst%3DR.%26aulast%3DPitt%26aufirst%3DS.%26aulast%3DSchieven%26aufirst%3DG.%2BL.%26aulast%3DSchmidt%26aufirst%3DR.%2BJ.%26aulast%3DTokarski%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DM.%2BL.%26aulast%3DWityak%26aufirst%3DJ.%26aulast%3DBorzilleri%26aufirst%3DR.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25282-chloro-6-methyl-%2520phenyl%2529-2-%25286-%25284-%25282-hydroxyethyl%2529-%2520piperazin-1-yl%2529-2-methylpyrimidin-4-%2520ylamino%2529thiazole-5-carboxamide%2520%2528BMS-354825%2529%252C%2520a%2520dual%2520Src%252FAbl%2520kinase%2520inhibitor%2520with%2520potent%2520antitumor%2520activity%2520in%2520preclinical%2520assays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D6658%26epage%3D6661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastuck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boesche, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reader, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweetman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouwmeester, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubauer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rick, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span> </span><span class="NLM_article-title">Quantitative chemical proteomics reveals mechanisms of action of clinical ABL kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1035</span><span class="NLM_x">â</span> <span class="NLM_lpage">1044</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1038%2Fnbt1328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=17721511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWksLvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=1035-1044&author=M.+Bantscheffauthor=D.+Eberhardauthor=Y.+Abrahamauthor=S.+Bastuckauthor=M.+Boescheauthor=S.+Hobsonauthor=T.+Mathiesonauthor=J.+Perrinauthor=M.+Raidaauthor=C.+Rauauthor=V.+Readerauthor=G.+Sweetmanauthor=A.+Bauerauthor=T.+Bouwmeesterauthor=C.+Hopfauthor=U.+Kruseauthor=G.+Neubauerauthor=N.+Ramsdenauthor=J.+Rickauthor=B.+Kusterauthor=G.+Drewes&title=Quantitative+chemical+proteomics+reveals+mechanisms+of+action+of+clinical+ABL+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative chemical proteomics reveals mechanisms of action of clinical ABL kinase inhibitors</span></div><div class="casAuthors">Bantscheff, Marcus; Eberhard, Dirk; Abraham, Yann; Bastuck, Sonja; Boesche, Markus; Hobson, Scott; Mathieson, Toby; Perrin, Jessica; Raida, Manfred; Rau, Christina; Reader, Valerie; Sweetman, Gavain; Bauer, Andreas; Bouwmeester, Tewis; Hopf, Carsten; Kruse, Ulrich; Neubauer, Gitte; Ramsden, Nigel; Rick, Jens; Kuster, Bernhard; Drewes, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1035-1044</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors describe a chem. proteomics approach to profile the interaction of small mols. with hundreds of endogenously expressed protein kinases and purine-binding proteins.  This subproteome is captured by immobilized nonselective kinase inhibitors (kinobeads), and the bound proteins are quantified in parallel by mass spectrometry using isobaric tags for relative and abs. quantification (iTRAQ).  By measuring the competition with the affinity matrix, the authors assess the binding of drugs to their targets in cell lysates and in cells.  By mapping drug-induced changes in the phosphorylation state of the captured proteome, the authors also analyze signaling pathways downstream of target kinases.  Quant. profiling of the drugs imatinib (Gleevec), dasatinib (Sprycel) and bosutinib in K562 cells confirms known targets including ABL and SRC family kinases and identifies the receptor tyrosine kinase DDR1 and the oxidoreductase NQO2 as novel targets of imatinib.  The data suggest that the approach is a valuable tool for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsC3OpIuhl8LVg90H21EOLACvtfcHk0lgrYHno6eSLNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWksLvF&md5=ed20d4684a8a6733aba24ca64602bae8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnbt1328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1328%26sid%3Dliteratum%253Aachs%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DAbraham%26aufirst%3DY.%26aulast%3DBastuck%26aufirst%3DS.%26aulast%3DBoesche%26aufirst%3DM.%26aulast%3DHobson%26aufirst%3DS.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DPerrin%26aufirst%3DJ.%26aulast%3DRaida%26aufirst%3DM.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DReader%26aufirst%3DV.%26aulast%3DSweetman%26aufirst%3DG.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DBouwmeester%26aufirst%3DT.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DKruse%26aufirst%3DU.%26aulast%3DNeubauer%26aufirst%3DG.%26aulast%3DRamsden%26aufirst%3DN.%26aulast%3DRick%26aufirst%3DJ.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DQuantitative%2520chemical%2520proteomics%2520reveals%2520mechanisms%2520of%2520action%2520of%2520clinical%2520ABL%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D1035%26epage%3D1044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Huang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reeves, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairchild, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platero, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, E.</span><span> </span><span class="NLM_article-title">Identification of candidate molecular markers predicting sensitivity in solid tumors to dasatinib: Rationale for patient selection</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">2226</span><span class="NLM_x">â</span> <span class="NLM_lpage">2238</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=2226-2238&author=F.+Huangauthor=K.+Reevesauthor=X.+Hanauthor=C.+Fairchildauthor=S.+Plateroauthor=T.+W.+Wongauthor=F.+Leeauthor=P.+Shawauthor=E.+Clark&title=Identification+of+candidate+molecular+markers+predicting+sensitivity+in+solid+tumors+to+dasatinib%3A+Rationale+for+patient+selection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DF.%26aulast%3DReeves%26aufirst%3DK.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DFairchild%26aufirst%3DC.%26aulast%3DPlatero%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DT.%2BW.%26aulast%3DLee%26aufirst%3DF.%26aulast%3DShaw%26aufirst%3DP.%26aulast%3DClark%26aufirst%3DE.%26atitle%3DIdentification%2520of%2520candidate%2520molecular%2520markers%2520predicting%2520sensitivity%2520in%2520solid%2520tumors%2520to%2520dasatinib%253A%2520Rationale%2520for%2520patient%2520selection%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D2226%26epage%3D2238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">â</span> <span class="NLM_lpage">39</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0lgrYHno6eSLNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, K.</span><span> </span><span class="NLM_article-title">Assessment of chemical coverage of kinome space and its implications for kinase drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7898</span><span class="NLM_x">â</span> <span class="NLM_lpage">7914</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8011036" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7898-7914&author=P.+Bamboroughauthor=D.+Drewryauthor=G.+Harperauthor=G.+K.+Smithauthor=K.+Schneider&title=Assessment+of+chemical+coverage+of+kinome+space+and+its+implications+for+kinase+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Chemical Coverage of Kinome Space and Its Implications for Kinase Drug Discovery</span></div><div class="casAuthors">Bamborough, Paul; Drewry, David; Harper, Gavin; Smith, Gary K.; Schneider, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7914</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">More than 500 compds. chosen to represent kinase inhibitor space have been screened against a panel of over 200 protein kinases.  Significant results include the identification of hits against new kinases including PIM1 and MPSK1, and the expansion of the inhibition profiles of several literature compds.  A detailed anal. of the data through the use of affinity fingerprints has produced findings with implications for biol. target selection, the choice of tool compds. for target validation, and lead discovery and optimization.  In a detailed examn. of the tyrosine kinases, interesting relationships have been found between targets and compds.  Taken together, these results show how broad cross-profiling can provide important insights to assist kinase drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBRI-arQk3t7Vg90H21EOLACvtfcHk0lifv3Z3oyZPbg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI&md5=8d98c80fb157b987a19e32a1efb53103</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm8011036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8011036%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDrewry%26aufirst%3DD.%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DG.%2BK.%26aulast%3DSchneider%26aufirst%3DK.%26atitle%3DAssessment%2520of%2520chemical%2520coverage%2520of%2520kinome%2520space%2520and%2520its%2520implications%2520for%2520kinase%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7898%26epage%3D7914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">â</span> <span class="NLM_lpage">336</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lifv3Z3oyZPbg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">â</span> <span class="NLM_lpage">132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lifv3Z3oyZPbg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Schmidt-Arras, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwable, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohmer, F. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serve, H.</span><span> </span><span class="NLM_article-title">Flt3 receptor tyrosine kinase as a drug target in leukemia</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1867</span><span class="NLM_x">â</span> <span class="NLM_lpage">1883</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.2174%2F1381612043384394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=15180525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktV2gt78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=1867-1883&author=D.+Schmidt-Arrasauthor=J.+Schwableauthor=F.+D.+Bohmerauthor=H.+Serve&title=Flt3+receptor+tyrosine+kinase+as+a+drug+target+in+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Flt3 receptor tyrosine kinase as a drug target in Leukemia</span></div><div class="casAuthors">Schmidt-Arras, Dirk; Schwaeble, Joachim; Boehmer, Frank-D.; Serve, Hubert</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1867-1883</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The hematopoietic class III receptor tyrosine kinase (RTK) Flt3 (Flk2, STK1) has recently received much attention as a potential drug target.  Activation of Flt3 by different types of mutations plays an important role for proliferation, resistance to apoptosis, and prevention of differentiation of leukemic blasts in acute myeloid leukemia (AML).  At least one type of such mutations - an internal tandem duplication in the Flt3 juxtamembrane domain (Flt3-ITD) - has been assocd. with an unfavorable prognosis.  Signal transduction of Flt3 involves activation of several conserved pathways, including the RAS/MAP-Kinase and the phosphoinositide-3-kinase/Akt signaling cascades.  Transforming versions of Flt3 exhibit altered signaling, for example a very pronounced activation of STAT5, ultimately resulting in alternate profiles of gene expression and cell transformation.  Selective inhibitors of Flt3 tyrosine kinase activity have the potential to suppress aberrant Flt3 signaling.  Although highly homologous to other class III RTKs, Flt3 is resistant to the phenylaminopyrimidine STI571 (Gleevec, Imatinib), a potent inhibitor of other RTKs in the family, such as the PDGFÎ²-receptor or c-Kit.  STI571 binding to Flt3 is prevented by the phenylalanine 691 side-chain in the ATP binding center and mutating this site to threonine renders the corresponding Flt3 mutant sensitive to STI571.  Compds. of several other structural families, including the quinoxaline AGI296, the bis(1H-2-indolyl)-l-methanone D-65476, the indolinones SU5416 and SU11248, the indolocarbazoles PKC412 and CEP-701, and the piperazonyl quinazoline CT53518, are potent inhibitors of Flt3 kinase.  They exhibit different selectivity profiles, both with respect to other kinases and among wildtype Flt3 and its activated versions.  These compds. hold promise as novel drugs against AML and as probes for understanding activation mechanisms and signaling pathways in the class III RTK family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM-Zx05aCHZ7Vg90H21EOLACvtfcHk0lj6L6xplDb5qg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktV2gt78%253D&md5=48bf03b9447756d8ac59223b515c1391</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2174%2F1381612043384394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612043384394%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt-Arras%26aufirst%3DD.%26aulast%3DSchwable%26aufirst%3DJ.%26aulast%3DBohmer%26aufirst%3DF.%2BD.%26aulast%3DServe%26aufirst%3DH.%26atitle%3DFlt3%2520receptor%2520tyrosine%2520kinase%2520as%2520a%2520drug%2520target%2520in%2520leukemia%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2004%26volume%3D10%26spage%3D1867%26epage%3D1883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Sawyers, C. L.</span><span> </span><span class="NLM_article-title">Finding the next Gleevec: FLT3 targeted kinase inhibitor therapy for acute myeloid leukemia</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">413</span><span class="NLM_x">â</span> <span class="NLM_lpage">415</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1016%2FS1535-6108%2802%2900080-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=12124170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKjsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=413-415&author=C.+L.+Sawyers&title=Finding+the+next+Gleevec%3A+FLT3+targeted+kinase+inhibitor+therapy+for+acute+myeloid+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Finding the next Gleevec: FLT3 targeted kinase inhibitor therapy for acute myeloid leukemia</span></div><div class="casAuthors">Sawyers, Charles L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">413-415</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Activating mutations in the FLT3 receptor tyrosine kinase occur in 30% of patients with acute myeloid leukemia.  Small mol. FLT3 kinase inhibitors show selective antitumor activity in preclin. models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJQQJ_qew_HbVg90H21EOLACvtfcHk0lj6L6xplDb5qg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKjsLk%253D&md5=30686782dc9bc4e953158a07dbe4a187</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS1535-6108%2802%2900080-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1535-6108%252802%252900080-6%26sid%3Dliteratum%253Aachs%26aulast%3DSawyers%26aufirst%3DC.%2BL.%26atitle%3DFinding%2520the%2520next%2520Gleevec%253A%2520FLT3%2520targeted%2520kinase%2520inhibitor%2520therapy%2520for%2520acute%2520myeloid%2520leukemia%26jtitle%3DCancer%2520Cell%26date%3D2002%26volume%3D1%26spage%3D413%26epage%3D415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Lefevre, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glotin, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calipel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouriaux, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kherrouche, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurage, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auclair, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascarelli, F.</span><span> </span><span class="NLM_article-title">Roles of stem cell factor/c-Kit and effects of Glivec/STI571 in human uveal melanoma cell tumorigenesis</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">31769</span><span class="NLM_x">â</span> <span class="NLM_lpage">31779</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=31769-31779&author=G.+Lefevreauthor=A.+L.+Glotinauthor=A.+Calipelauthor=F.+Mouriauxauthor=T.+Tranauthor=Z.+Kherroucheauthor=C.+A.+Maurageauthor=C.+Auclairauthor=F.+Mascarelli&title=Roles+of+stem+cell+factor%2Fc-Kit+and+effects+of+Glivec%2FSTI571+in+human+uveal+melanoma+cell+tumorigenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLefevre%26aufirst%3DG.%26aulast%3DGlotin%26aufirst%3DA.%2BL.%26aulast%3DCalipel%26aufirst%3DA.%26aulast%3DMouriaux%26aufirst%3DF.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DKherrouche%26aufirst%3DZ.%26aulast%3DMaurage%26aufirst%3DC.%2BA.%26aulast%3DAuclair%26aufirst%3DC.%26aulast%3DMascarelli%26aufirst%3DF.%26atitle%3DRoles%2520of%2520stem%2520cell%2520factor%252Fc-Kit%2520and%2520effects%2520of%2520Glivec%252FSTI571%2520in%2520human%2520uveal%2520melanoma%2520cell%2520tumorigenesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D31769%26epage%3D31779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Dai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frey, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtin, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albert, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arries, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlozzari, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauch, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouska, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bousquet, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cunha, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glaser, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcotte, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsh, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moskey, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pease, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stoll, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tapang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wishart, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidsen, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaelides, M. R.</span><span> </span><span class="NLM_article-title">Thienopyrimidine ureas as novel and potent multitargeted receptor tyrosine kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6066</span><span class="NLM_x">â</span> <span class="NLM_lpage">6083</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050458h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpt1Wht7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6066-6083&author=Y.+Daiauthor=Y.+Guoauthor=R.+R.+Freyauthor=Z.+Jiauthor=M.+L.+Curtinauthor=A.+A.+Ahmedauthor=D.+H.+Albertauthor=L.+Arnoldauthor=S.+S.+Arriesauthor=T.+Barlozzariauthor=J.+L.+Bauchauthor=J.+J.+Bouskaauthor=P.+F.+Bousquetauthor=G.+A.+Cunhaauthor=K.+B.+Glaserauthor=J.+Guoauthor=J.+Liauthor=P.+A.+Marcotteauthor=K.+C.+Marshauthor=M.+D.+Moskeyauthor=L.+J.+Peaseauthor=K.+D.+Stewartauthor=V.+S.+Stollauthor=P.+Tapangauthor=N.+Wishartauthor=S.+K.+Davidsenauthor=M.+R.+Michaelides&title=Thienopyrimidine+ureas+as+novel+and+potent+multitargeted+receptor+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Thienopyrimidine Ureas as Novel and Potent Multitargeted Receptor Tyrosine Kinase Inhibitors</span></div><div class="casAuthors">Dai, Yujia; Guo, Yan; Frey, Robin R.; Ji, Zhiqin; Curtin, Michael L.; Ahmed, Asma A.; Albert, Daniel H.; Arnold, Lee; Arries, Shannon S.; Barlozzari, Teresa; Bauch, Joy L.; Bouska, Jennifer J.; Bousquet, Peter F.; Cunha, George A.; Glaser, Keith B.; Guo, Jun; Li, Junling; Marcotte, Patrick A.; Marsh, Kennan C.; Moskey, Maria D.; Pease, Lori J.; Stewart, Kent D.; Stoll, Vincent S.; Tapang, Paul; Wishart, Neil; Davidsen, Steven K.; Michaelides, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6066-6083</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of novel thienopyrimidine-based receptor tyrosine kinase inhibitors has been discovered.  Investigation of structure-activity relationships at the 5- and 6-positions of the thienopyrimidine nucleus led to a series of N,N'-diaryl ureas that potently inhibit all of the vascular endothelial growth factor (VEGF) and platelet-derived growth factor (PDGF) receptor tyrosine kinases.  A kinase insert domain-contg. receptor (KDR) homol. model suggests that these compds. bind to the "inactive conformation" of the enzyme with the urea portion extending into the back hydrophobic pocket adjacent to the ATP (ATP) binding site.  A no. of compds. have been identified as displaying excellent in vivo potency.  In particular, compds. 28 and 76 possess favorable pharmacokinetic (PK) profiles and demonstrate potent antitumor efficacy against the HT1080 human fibrosarcoma xenograft tumor growth model (tumor growth inhibition (TGI) = 75% at 25 mg/kgÂ·day, per os (po)).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYifOiC_HYHrVg90H21EOLACvtfcHk0limoSAJlKDTNw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpt1Wht7Y%253D&md5=ef245a26e40d18cfaf7f98260827f274</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm050458h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050458h%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DFrey%26aufirst%3DR.%2BR.%26aulast%3DJi%26aufirst%3DZ.%26aulast%3DCurtin%26aufirst%3DM.%2BL.%26aulast%3DAhmed%26aufirst%3DA.%2BA.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DArnold%26aufirst%3DL.%26aulast%3DArries%26aufirst%3DS.%2BS.%26aulast%3DBarlozzari%26aufirst%3DT.%26aulast%3DBauch%26aufirst%3DJ.%2BL.%26aulast%3DBouska%26aufirst%3DJ.%2BJ.%26aulast%3DBousquet%26aufirst%3DP.%2BF.%26aulast%3DCunha%26aufirst%3DG.%2BA.%26aulast%3DGlaser%26aufirst%3DK.%2BB.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMarcotte%26aufirst%3DP.%2BA.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DMoskey%26aufirst%3DM.%2BD.%26aulast%3DPease%26aufirst%3DL.%2BJ.%26aulast%3DStewart%26aufirst%3DK.%2BD.%26aulast%3DStoll%26aufirst%3DV.%2BS.%26aulast%3DTapang%26aufirst%3DP.%26aulast%3DWishart%26aufirst%3DN.%26aulast%3DDavidsen%26aufirst%3DS.%2BK.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26atitle%3DThienopyrimidine%2520ureas%2520as%2520novel%2520and%2520potent%2520multitargeted%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6066%26epage%3D6083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Manley, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drueckes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fendrich, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liebetanz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martiny-Baron, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trappe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wartmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span> </span><span class="NLM_article-title">Extended kinase profile and properties of the protein kinase inhibitor nilotinib</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1804</span><span class="NLM_x">, </span> <span class="NLM_fpage">445</span><span class="NLM_x">â</span> <span class="NLM_lpage">453</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1804&publication_year=2010&pages=445-453&author=P.+W.+Manleyauthor=P.+Drueckesauthor=G.+Fendrichauthor=P.+Furetauthor=J.+Liebetanzauthor=G.+Martiny-Baronauthor=J.+Mestanauthor=J.+Trappeauthor=M.+Wartmannauthor=D.+Fabbro&title=Extended+kinase+profile+and+properties+of+the+protein+kinase+inhibitor+nilotinib"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DFendrich%26aufirst%3DG.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DLiebetanz%26aufirst%3DJ.%26aulast%3DMartiny-Baron%26aufirst%3DG.%26aulast%3DMestan%26aufirst%3DJ.%26aulast%3DTrappe%26aufirst%3DJ.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DFabbro%26aufirst%3DD.%26atitle%3DExtended%2520kinase%2520profile%2520and%2520properties%2520of%2520the%2520protein%2520kinase%2520inhibitor%2520nilotinib%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2010%26volume%3D1804%26spage%3D445%26epage%3D453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Goldstein, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">High-throughput kinase profiling as a platform for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">391</span><span class="NLM_x">â</span> <span class="NLM_lpage">397</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1038%2Fnrd2541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=18404149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=391-397&author=D.+M.+Goldsteinauthor=N.+S.+Grayauthor=P.+P.+Zarrinkar&title=High-throughput+kinase+profiling+as+a+platform+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput kinase profiling as a platform for drug discovery</span></div><div class="casAuthors">Goldstein, David M.; Gray, Nathanael S.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">391-397</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput profiling of compd. libraries against large panels of kinases is becoming tech. feasible.  In contrast to the traditional linear, target-centric approach to discovery, this approach may provide a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone, and could thereby significantly improve the productivity of kinase inhibitor discovery.  To fully exploit the potential of kinases as drug targets, novel strategies for the efficient discovery of inhibitors are required.  In contrast to the traditional, linear process of inhibitor discovery, high-throughput kinase profiling enables a parallel approach by interrogating compds. against hundreds of targets in a single screen.  Compd. potency and selectivity are detd. simultaneously, providing a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwnJ59cVxbJbVg90H21EOLACvtfcHk0limoSAJlKDTNw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D&md5=eaa61fd2958ff04fecf86a993f91833d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd2541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2541%26sid%3Dliteratum%253Aachs%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DHigh-throughput%2520kinase%2520profiling%2520as%2520a%2520platform%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D391%26epage%3D397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Belema, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaulieu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouellet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McIntyre, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattoli, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daloisio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillooly, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brassil, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zusi, F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vyas, D. M.</span><span> </span><span class="NLM_article-title">Synthesis and structure-activity relationship of imidazo(1,2-a)thieno(3,2-e)pyrazines as IKK-Î² inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">4284</span><span class="NLM_x">â</span> <span class="NLM_lpage">4289</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=4284-4289&author=M.+Belemaauthor=A.+Bunkerauthor=V.+N.+Nguyenauthor=F.+Beaulieuauthor=C.+Ouelletauthor=Y.+Qiuauthor=Y.+Zhangauthor=A.+Martelauthor=J.+R.+Burkeauthor=K.+W.+McIntyreauthor=M.+A.+Pattoliauthor=C.+Daloisioauthor=K.+M.+Gilloolyauthor=W.+J.+Clarkeauthor=P.+J.+Brassilauthor=F.+C.+Zusiauthor=D.+M.+Vyas&title=Synthesis+and+structure-activity+relationship+of+imidazo%281%2C2-a%29thieno%283%2C2-e%29pyrazines+as+IKK-%CE%B2+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBelema%26aufirst%3DM.%26aulast%3DBunker%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DV.%2BN.%26aulast%3DBeaulieu%26aufirst%3DF.%26aulast%3DOuellet%26aufirst%3DC.%26aulast%3DQiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMartel%26aufirst%3DA.%26aulast%3DBurke%26aufirst%3DJ.%2BR.%26aulast%3DMcIntyre%26aufirst%3DK.%2BW.%26aulast%3DPattoli%26aufirst%3DM.%2BA.%26aulast%3DDaloisio%26aufirst%3DC.%26aulast%3DGillooly%26aufirst%3DK.%2BM.%26aulast%3DClarke%26aufirst%3DW.%2BJ.%26aulast%3DBrassil%26aufirst%3DP.%2BJ.%26aulast%3DZusi%26aufirst%3DF.%2BC.%26aulast%3DVyas%26aufirst%3DD.%2BM.%26atitle%3DSynthesis%2520and%2520structure-activity%2520relationship%2520of%2520imidazo%25281%252C2-a%2529thieno%25283%252C2-e%2529pyrazines%2520as%2520IKK-%25CE%25B2%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D4284%26epage%3D4289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Martin, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kofron, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Traphagen, L. M.</span><span> </span><span class="NLM_article-title">Do structurally similar molecules have similar biological activity?</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">4350</span><span class="NLM_x">â</span> <span class="NLM_lpage">4358</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020155c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=4350-4358&author=Y.+C.+Martinauthor=J.+L.+Kofronauthor=L.+M.+Traphagen&title=Do+structurally+similar+molecules+have+similar+biological+activity%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm020155c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020155c%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DY.%2BC.%26aulast%3DKofron%26aufirst%3DJ.%2BL.%26aulast%3DTraphagen%26aufirst%3DL.%2BM.%26atitle%3DDo%2520structurally%2520similar%2520molecules%2520have%2520similar%2520biological%2520activity%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D4350%26epage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Aronov, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClain, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moody, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">Kinase-likeness and kinase-privileged fragments: Toward virtual polypharmacology</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1214</span><span class="NLM_x">â</span> <span class="NLM_lpage">1222</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701021b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1214-1222&author=A.+M.+Aronovauthor=B.+McClainauthor=C.+S.+Moodyauthor=M.+A.+Murcko&title=Kinase-likeness+and+kinase-privileged+fragments%3A+Toward+virtual+polypharmacology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm701021b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701021b%26sid%3Dliteratum%253Aachs%26aulast%3DAronov%26aufirst%3DA.%2BM.%26aulast%3DMcClain%26aufirst%3DB.%26aulast%3DMoody%26aufirst%3DC.%2BS.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DKinase-likeness%2520and%2520kinase-privileged%2520fragments%253A%2520Toward%2520virtual%2520polypharmacology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1214%26epage%3D1222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Briem, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, J.</span><span> </span><span class="NLM_article-title">Classifying âkinase inhibitor-likenessâ by using machine-learning methods</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">558</span><span class="NLM_x">â</span> <span class="NLM_lpage">566</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=558-566&author=H.+Briemauthor=J.+Gunther&title=Classifying+%E2%80%9Ckinase+inhibitor-likeness%E2%80%9D+by+using+machine-learning+methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DGunther%26aufirst%3DJ.%26atitle%3DClassifying%2520%25E2%2580%259Ckinase%2520inhibitor-likeness%25E2%2580%259D%2520by%2520using%2520machine-learning%2520methods%26jtitle%3DChemBioChem%26date%3D2005%26volume%3D6%26spage%3D558%26epage%3D566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Garcia-Echeverria, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Traxler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, D. B.</span><span> </span><span class="NLM_article-title">ATP site-directed competitive and irreversible inhibitors of protein kinases</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">â</span> <span class="NLM_lpage">57</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=28-57&author=C.+Garcia-Echeverriaauthor=P.+Traxlerauthor=D.+B.+Evans&title=ATP+site-directed+competitive+and+irreversible+inhibitors+of+protein+kinases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia-Echeverria%26aufirst%3DC.%26aulast%3DTraxler%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DD.%2BB.%26atitle%3DATP%2520site-directed%2520competitive%2520and%2520irreversible%2520inhibitors%2520of%2520protein%2520kinases%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2000%26volume%3D20%26spage%3D28%26epage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Dumas, J.</span><span> </span><span class="NLM_article-title">Protein kinase inhibitors: Emerging pharmacophores 1997â2000</span> <span class="citation_source-journal">Expert Opin. Ther. Pat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">â</span> <span class="NLM_lpage">429</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1517%2F13543776.11.3.405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslSisrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=405-429&author=J.+Dumas&title=Protein+kinase+inhibitors%3A+Emerging+pharmacophores+1997%E2%88%922000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase inhibitors: emerging pharmacophores 1997 - 2000</span></div><div class="casAuthors">Dumas, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-429</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review with 292 refs. is given.  Kinase inhibition is a major area for therapeutic intervention against a variety of diseases including cancer, inflammatory disorders and diabetes.  While this mode of action remains very promising at this time, it faces crit. challenges, such as drug safety, and biopharmaceutical profile.  This review provides an update on kinase inhibitor pharmacophores from a structural standpoint.  Advances in older inhibitor classes as well as new emerging lead structures will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozUM0oS-Vu8rVg90H21EOLACvtfcHk0lj84SoIdlW-kQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhslSisrc%253D&md5=4f6dc306b741963d931d6abfe22e40a2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1517%2F13543776.11.3.405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.11.3.405%26sid%3Dliteratum%253Aachs%26aulast%3DDumas%26aufirst%3DJ.%26atitle%3DProtein%2520kinase%2520inhibitors%253A%2520Emerging%2520pharmacophores%25201997%25E2%2588%25922000%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2001%26volume%3D11%26spage%3D405%26epage%3D429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Hunt, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitt, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borzilleri, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullo-Brown, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fargnoli, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fink, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortillo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vite, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wautlet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhide, R.</span><span> </span><span class="NLM_article-title">Discovery of the pyrrolo[2,1-f][1,2,4]triazine nucleus as a new kinase inhibitor template</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4054</span><span class="NLM_x">â</span> <span class="NLM_lpage">4059</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049892u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4054-4059&author=J.+T.+Huntauthor=T.+Mittauthor=R.+Borzilleriauthor=J.+Gullo-Brownauthor=J.+Fargnoliauthor=B.+Finkauthor=W.+C.+Hanauthor=S.+Mortilloauthor=G.+Viteauthor=B.+Wautletauthor=T.+Wongauthor=C.+Yuauthor=X.+Zhengauthor=R.+Bhide&title=Discovery+of+the+pyrrolo%5B2%2C1-f%5D%5B1%2C2%2C4%5Dtriazine+nucleus+as+a+new+kinase+inhibitor+template"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm049892u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049892u%26sid%3Dliteratum%253Aachs%26aulast%3DHunt%26aufirst%3DJ.%2BT.%26aulast%3DMitt%26aufirst%3DT.%26aulast%3DBorzilleri%26aufirst%3DR.%26aulast%3DGullo-Brown%26aufirst%3DJ.%26aulast%3DFargnoli%26aufirst%3DJ.%26aulast%3DFink%26aufirst%3DB.%26aulast%3DHan%26aufirst%3DW.%2BC.%26aulast%3DMortillo%26aufirst%3DS.%26aulast%3DVite%26aufirst%3DG.%26aulast%3DWautlet%26aufirst%3DB.%26aulast%3DWong%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DBhide%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520the%2520pyrrolo%255B2%252C1-f%255D%255B1%252C2%252C4%255Dtriazine%2520nucleus%2520as%2520a%2520new%2520kinase%2520inhibitor%2520template%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4054%26epage%3D4059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Saxty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyle, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barford, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downham, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, R. A.</span><span> </span><span class="NLM_article-title">Identification of inhibitors of protein kinase B using fragment-based lead discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2293</span><span class="NLM_x">â</span> <span class="NLM_lpage">2296</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm070091b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVKjur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2293-2296&author=G.+Saxtyauthor=S.+J.+Woodheadauthor=V.+Berdiniauthor=T.+G.+Daviesauthor=M.+L.+Verdonkauthor=P.+G.+Wyattauthor=R.+G.+Boyleauthor=D.+Barfordauthor=R.+Downhamauthor=M.+D.+Garrettauthor=R.+A.+Carr&title=Identification+of+inhibitors+of+protein+kinase+B+using+fragment-based+lead+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Inhibitors of Protein Kinase B Using Fragment-Based Lead Discovery</span></div><div class="casAuthors">Saxty, Gordon; Woodhead, Steven J.; Berdini, Valerio; Davies, Thomas G.; Verdonk, Marcel L.; Wyatt, Paul G.; Boyle, Robert G.; Barford, David; Downham, Robert; Garrett, Michelle D.; Carr, Robin A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2293-2296</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using fragment-based screening techniques, 5-methyl-4-phenyl-1H-pyrazole (IC50 80 Î¼M) was identified as a novel, low mol. wt. inhibitor of protein kinase B (PKB).  Herein we describe the rapid elaboration of highly potent and ligand efficient analogs using a fragment growing approach.  Iterative structure-based design was supported by protein-ligand structure detns. using a PKA-PKB "chimera" and a final protein-ligand structure of a lead compd. in PKBÎ² itself.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlCYaIT8Xn87Vg90H21EOLACvtfcHk0ljYRxUjfNN11A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVKjur8%253D&md5=2a09e4bbaca1493be8e215927e7e72e1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm070091b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070091b%26sid%3Dliteratum%253Aachs%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWoodhead%26aufirst%3DS.%2BJ.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DBoyle%26aufirst%3DR.%2BG.%26aulast%3DBarford%26aufirst%3DD.%26aulast%3DDownham%26aufirst%3DR.%26aulast%3DGarrett%26aufirst%3DM.%2BD.%26aulast%3DCarr%26aufirst%3DR.%2BA.%26atitle%3DIdentification%2520of%2520inhibitors%2520of%2520protein%2520kinase%2520B%2520using%2520fragment-based%2520lead%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2293%26epage%3D2296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Deanda, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reno, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D. H.</span><span> </span><span class="NLM_article-title">Kinase-targeted library design through the application of the PharmPrint methodology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2395</span><span class="NLM_x">â</span> <span class="NLM_lpage">2403</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800276t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWgt7%252FF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=2395-2403&author=F.+Deandaauthor=E.+L.+Stewartauthor=M.+J.+Renoauthor=D.+H.+Drewry&title=Kinase-targeted+library+design+through+the+application+of+the+PharmPrint+methodology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase-Targeted Library Design through the Application of the PharmPrint Methodology</span></div><div class="casAuthors">Deanda, Felix; Stewart, Eugene L.; Reno, Michael J.; Drewry, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2395-2403</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The PharmPrint methodol., as modified and implemented by Deanda and Stewart, was prospectively evaluated for use as a virtual high-throughput screening tool by applying it to the design of target-focused arrays.  To this end, PharmPrint quant. structure-activity relationship (QSAR) models for the prediction of AKT1, Aurora-A, and ROCK1 inhibition were constructed and used to virtually screen two large combinatorial libraries.  Based on predicted activities, an Aurora-A targeted array and a ROCK1 targeted array were designed and synthesized.  One control group per designed array was also synthesized to assess the enrichment levels achieved by the QSAR models.  For the Aurora-A targeted array, the hit rate, against the intended target, was 42.9%, whereas that of the control group was 0%.  Thus, the enrichment level achieved by the Aurora-A QSAR model was incalculable.  For the ROCK1 targeted array, the hit rate against the intended target was 30.6%, whereas that of the control group was 5.10%, making the enrichment level achieved by the ROCK1 QSAR model 6-fold above control.  Clearly, these results support the use of the PharmPrint methodol. as a virtual screening tool for the design of kinase-targeted arrays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXVi-syw96s7Vg90H21EOLACvtfcHk0ljYRxUjfNN11A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWgt7%252FF&md5=0a7c65ed90b10dc8de5016af052304f6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci800276t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800276t%26sid%3Dliteratum%253Aachs%26aulast%3DDeanda%26aufirst%3DF.%26aulast%3DStewart%26aufirst%3DE.%2BL.%26aulast%3DReno%26aufirst%3DM.%2BJ.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26atitle%3DKinase-targeted%2520library%2520design%2520through%2520the%2520application%2520of%2520the%2520PharmPrint%2520methodology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D2395%26epage%3D2403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Manallack, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gancia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montana, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Livingstone, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitley, D. C.</span><span> </span><span class="NLM_article-title">Selecting screening candidates for kinase and G protein-coupled receptor targets using neural networks</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">1256</span><span class="NLM_x">â</span> <span class="NLM_lpage">1262</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci020267c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1256-1262&author=D.+T.+Manallackauthor=W.+R.+Pittauthor=E.+Ganciaauthor=J.+G.+Montanaauthor=D.+J.+Livingstoneauthor=M.+G.+Fordauthor=D.+C.+Whitley&title=Selecting+screening+candidates+for+kinase+and+G+protein-coupled+receptor+targets+using+neural+networks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Selecting Screening Candidates for Kinase and G Protein-Coupled Receptor Targets Using Neural Networks</span></div><div class="casAuthors">Manallack, David T.; Pitt, Will R.; Gancia, Emanuela; Montana, John G.; Livingstone, David J.; Ford, Martyn G.; Whitley, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1256-1262</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of neural networks has been trained, using consensus methods, to recognize compds. that act at biol. targets belonging to specific gene families.  The MDDR database was used to provide compds. targeted against gene families and sets of randomly selected mols.  BCUT parameters were employed as input descriptors that encode structural properties and information relevant to ligand-receptor interactions.  In each case, the networks identified over 80% of the compds. targeting a gene family.  The technique was applied to purchasing compds. from external suppliers, and results from screening against one gene family demonstrated impressive abilities to predict the activity of the majority of known hit compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3B0ertK2pfbVg90H21EOLACvtfcHk0ljYRxUjfNN11A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrg%253D&md5=264cc5d5bc3657018ca0bd232954765c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci020267c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci020267c%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DGancia%26aufirst%3DE.%26aulast%3DMontana%26aufirst%3DJ.%2BG.%26aulast%3DLivingstone%26aufirst%3DD.%2BJ.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DWhitley%26aufirst%3DD.%2BC.%26atitle%3DSelecting%2520screening%2520candidates%2520for%2520kinase%2520and%2520G%2520protein-coupled%2520receptor%2520targets%2520using%2520neural%2520networks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D1256%26epage%3D1262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Akritopoulou-Zanze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Kinase-targeted libraries: The design and synthesis of novel, potent, and selective kinase inhibitors</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">â</span> <span class="NLM_lpage">297</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=291-297&author=I.+Akritopoulou-Zanzeauthor=P.+J.+Hajduk&title=Kinase-targeted+libraries%3A+The+design+and+synthesis+of+novel%2C+potent%2C+and+selective+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAkritopoulou-Zanze%26aufirst%3DI.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DKinase-targeted%2520libraries%253A%2520The%2520design%2520and%2520synthesis%2520of%2520novel%252C%2520potent%252C%2520and%2520selective%2520kinase%2520inhibitors%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D291%26epage%3D297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Martin, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, D. C.</span><span> </span><span class="NLM_article-title">Surrogate AutoShim: Predocking into a universal ensemble kinase receptor for three dimensional activity prediction, very quickly, without a crystal structure</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">â</span> <span class="NLM_lpage">881</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700455u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktVOkurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=873-881&author=E.+J.+Martinauthor=D.+C.+Sullivan&title=Surrogate+AutoShim%3A+Predocking+into+a+universal+ensemble+kinase+receptor+for+three+dimensional+activity+prediction%2C+very+quickly%2C+without+a+crystal+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Surrogate AutoShim: Predocking into a Universal Ensemble Kinase Receptor for Three Dimensional Activity Prediction, Very Quickly, without a Crystal Structure</span></div><div class="casAuthors">Martin, Eric J.; Sullivan, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">873-881</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">"Ensemble surrogate AutoShim" is a kinase specific extension of the AutoShim docking method that solves the three traditional limitations of conventional docking: (1) it gives good correlations with affinity, (2) does not require a target protein structure, and (3) for a preprocessed company archive of 1.5 million compds., is as fast as traditional 2D QSAR.  It does require several hundred exptl. IC50 values for each new target.  Original AutoShim adds pharmacophore "shims" to a crystal structure binding site.  An iterative partial least squares (PLS) procedure selects the best pose, while adjusting the shim wts. to reproduce IC50 data.  Surrogate AutoShim adjusts shims in one crystal structure to reproduce IC50 data for a different kinase target.  Ensemble surrogate AutoShim uses 16 structurally diverse kinase crystal structures as a "universal ensemble kinase receptor", suitable for any kinase target.  The 1.5 million member Novartis screening collection has been predocked into the shimmed ensemble, so new kinase models can be built, and the entire corporate archive virtually screened, in hours rather than weeks.  A kinase-biased set of 10 000 compds., that samples the entire corporate archive, has been designed for lead discovery by iterative kinase screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5HSUkdEqTcLVg90H21EOLACvtfcHk0ljn7osi7_w8hw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktVOkurg%253D&md5=0fe14c274192ec38b65224cae779cf35</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci700455u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700455u%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%2BJ.%26aulast%3DSullivan%26aufirst%3DD.%2BC.%26atitle%3DSurrogate%2520AutoShim%253A%2520Predocking%2520into%2520a%2520universal%2520ensemble%2520kinase%2520receptor%2520for%2520three%2520dimensional%2520activity%2520prediction%252C%2520very%2520quickly%252C%2520without%2520a%2520crystal%2520structure%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D873%26epage%3D881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Gozalbes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Froloff, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartori, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteils, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baudelle, R.</span><span> </span><span class="NLM_article-title">Development and experimental validation of a docking strategy for the generation of kinase-targeted libraries</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3124</span><span class="NLM_x">â</span> <span class="NLM_lpage">3132</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701367r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3124-3132&author=R.+Gozalbesauthor=L.+Simonauthor=N.+Froloffauthor=E.+Sartoriauthor=C.+Monteilsauthor=R.+Baudelle&title=Development+and+experimental+validation+of+a+docking+strategy+for+the+generation+of+kinase-targeted+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm701367r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701367r%26sid%3Dliteratum%253Aachs%26aulast%3DGozalbes%26aufirst%3DR.%26aulast%3DSimon%26aufirst%3DL.%26aulast%3DFroloff%26aufirst%3DN.%26aulast%3DSartori%26aufirst%3DE.%26aulast%3DMonteils%26aufirst%3DC.%26aulast%3DBaudelle%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520experimental%2520validation%2520of%2520a%2520docking%2520strategy%2520for%2520the%2520generation%2520of%2520kinase-targeted%2520libraries%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3124%26epage%3D3132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Dongyu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claudio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donovan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juswinder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patrick, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gretchen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen-Cherng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jason, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jessica, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serene, J.</span><span> </span><span class="NLM_article-title">A kinase-focused compound collection: Compilation and screening strategy</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">385</span><span class="NLM_x">â</span> <span class="NLM_lpage">394</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1111%2Fj.1747-0285.2006.00396.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=16882313" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2006&pages=385-394&author=S.+Dongyuauthor=C.+Claudioauthor=D.+Zhanauthor=B.+Scottauthor=C.+Donovanauthor=S.+Juswinderauthor=C.+Patrickauthor=H.+Gretchenauthor=L.+Wen-Cherngauthor=D.+Jasonauthor=F.+Jessicaauthor=J.+Serene&title=A+kinase-focused+compound+collection%3A+Compilation+and+screening+strategy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2006.00396.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2006.00396.x%26sid%3Dliteratum%253Aachs%26aulast%3DDongyu%26aufirst%3DS.%26aulast%3DClaudio%26aufirst%3DC.%26aulast%3DZhan%26aufirst%3DD.%26aulast%3DScott%26aufirst%3DB.%26aulast%3DDonovan%26aufirst%3DC.%26aulast%3DJuswinder%26aufirst%3DS.%26aulast%3DPatrick%26aufirst%3DC.%26aulast%3DGretchen%26aufirst%3DH.%26aulast%3DWen-Cherng%26aufirst%3DL.%26aulast%3DJason%26aufirst%3DD.%26aulast%3DJessica%26aufirst%3DF.%26aulast%3DSerene%26aufirst%3DJ.%26atitle%3DA%2520kinase-focused%2520compound%2520collection%253A%2520Compilation%2520and%2520screening%2520strategy%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2006%26volume%3D67%26spage%3D385%26epage%3D394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erickson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I.</span><span> </span><span class="NLM_article-title">Kinase inhibitor data modeling and de novo inhibitor design with fragment approaches</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6456</span><span class="NLM_x">â</span> <span class="NLM_lpage">6466</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901147e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6456-6466&author=M.+Viethauthor=J.+Ericksonauthor=J.+Wangauthor=Y.+Websterauthor=M.+Maderauthor=R.+Higgsauthor=I.+Watson&title=Kinase+inhibitor+data+modeling+and+de+novo+inhibitor+design+with+fragment+approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm901147e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901147e%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DErickson%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWebster%26aufirst%3DY.%26aulast%3DMader%26aufirst%3DM.%26aulast%3DHiggs%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DI.%26atitle%3DKinase%2520inhibitor%2520data%2520modeling%2520and%2520de%2520novo%2520inhibitor%2520design%2520with%2520fragment%2520approaches%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6456%26epage%3D6466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Akritopoulou-Zanze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darczak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarris, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phelan, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djuric, S. W.</span><span> </span><span class="NLM_article-title">Scaffold oriented synthesis. Part 1: Design, preparation, and biological evaluation of thienopyrazoles as kinase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">96</span><span class="NLM_x">â</span> <span class="NLM_lpage">99</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=96-99&author=I.+Akritopoulou-Zanzeauthor=D.+Darczakauthor=K.+Sarrisauthor=K.+M.+Phelanauthor=J.+R.+Huthauthor=D.+Songauthor=E.+F.+Johnsonauthor=Y.+Jiaauthor=S.+W.+Djuric&title=Scaffold+oriented+synthesis.+Part+1%3A+Design%2C+preparation%2C+and+biological+evaluation+of+thienopyrazoles+as+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAkritopoulou-Zanze%26aufirst%3DI.%26aulast%3DDarczak%26aufirst%3DD.%26aulast%3DSarris%26aufirst%3DK.%26aulast%3DPhelan%26aufirst%3DK.%2BM.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DDjuric%26aufirst%3DS.%2BW.%26atitle%3DScaffold%2520oriented%2520synthesis.%2520Part%25201%253A%2520Design%252C%2520preparation%252C%2520and%2520biological%2520evaluation%2520of%2520thienopyrazoles%2520as%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D96%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Gracias, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akritopoulou-Zanze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abad-Zapatero, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glaser, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcotte, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pease, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidsen, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaelides, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djuric, S. W.</span><span> </span><span class="NLM_article-title">Scaffold oriented synthesis. Part 2: Design, synthesis and biological evaluation of pyrimido-diazepines as receptor tyrosine kinase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">2691</span><span class="NLM_x">â</span> <span class="NLM_lpage">2695</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=2691-2695&author=V.+Graciasauthor=Z.+Jiauthor=I.+Akritopoulou-Zanzeauthor=C.+Abad-Zapateroauthor=J.+R.+Huthauthor=D.+Songauthor=P.+J.+Hajdukauthor=E.+F.+Johnsonauthor=K.+B.+Glaserauthor=P.+A.+Marcotteauthor=L.+Peaseauthor=N.+B.+Soniauthor=K.+D.+Stewartauthor=S.+K.+Davidsenauthor=M.+R.+Michaelidesauthor=S.+W.+Djuric&title=Scaffold+oriented+synthesis.+Part+2%3A+Design%2C+synthesis+and+biological+evaluation+of+pyrimido-diazepines+as+receptor+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGracias%26aufirst%3DV.%26aulast%3DJi%26aufirst%3DZ.%26aulast%3DAkritopoulou-Zanze%26aufirst%3DI.%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DGlaser%26aufirst%3DK.%2BB.%26aulast%3DMarcotte%26aufirst%3DP.%2BA.%26aulast%3DPease%26aufirst%3DL.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DStewart%26aufirst%3DK.%2BD.%26aulast%3DDavidsen%26aufirst%3DS.%2BK.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DDjuric%26aufirst%3DS.%2BW.%26atitle%3DScaffold%2520oriented%2520synthesis.%2520Part%25202%253A%2520Design%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520pyrimido-diazepines%2520as%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D2691%26epage%3D2695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Kettle, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, R. A.</span><span> </span><span class="NLM_article-title">Toward the comprehensive systematic enumeration and synthesis of novel kinase inhibitors based on a 4-anilinoquinazoline binding mode</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">â</span> <span class="NLM_lpage">533</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci9004767" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1GmsrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=525-533&author=J.+G.+Kettleauthor=R.+A.+Ward&title=Toward+the+comprehensive+systematic+enumeration+and+synthesis+of+novel+kinase+inhibitors+based+on+a+4-anilinoquinazoline+binding+mode"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the Comprehensive Systematic Enumeration and Synthesis of Novel Kinase Inhibitors Based on a 4-Anilinoquinazoline Binding Mode</span></div><div class="casAuthors">Kettle, Jason G.; Ward, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">525-533</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There are currently eight small-mol. kinase inhibitors approved as cancer treatments, and a significantly larger no. of compds. are in the earlier stages of clin. development.  Although kinase inhibitors are most commonly developed in a cancer setting, other disease areas have been targeted.  The vast majority of reported kinase small-mol. inhibitors contain functionalities that interact with the ATP binding site of the kinase.  The 4-anilinoquinazolines have previously been reported as potent epidermal growth factor receptor (EGFR) inhibitors, binding at the hinge' region of the ATP site.  Subsequently, this chem. series has been optimized against a no. of different kinases including Src and Aurora B.  Here, we detail the computational enumeration of ring systems that have the ability to make comparable interactions to the 4-anilinoquinazoline core.  These were prioritized by computational, medicinal, and synthetic chem. input, and a no. of libraries were subsequently synthesized (no prepn. shown).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoThVN2M1nTkLVg90H21EOLACvtfcHk0ljUZvlzDPlyAQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1GmsrY%253D&md5=eb70799255add46dec8828d65b861377</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci9004767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9004767%26sid%3Dliteratum%253Aachs%26aulast%3DKettle%26aufirst%3DJ.%2BG.%26aulast%3DWard%26aufirst%3DR.%2BA.%26atitle%3DToward%2520the%2520comprehensive%2520systematic%2520enumeration%2520and%2520synthesis%2520of%2520novel%2520kinase%2520inhibitors%2520based%2520on%2520a%25204-anilinoquinazoline%2520binding%2520mode%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D525%26epage%3D533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Harper, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickett, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, D. V.</span><span> </span><span class="NLM_article-title">Design of a compound screening collection for use in high throughput screening</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">63</span><span class="NLM_x">â</span> <span class="NLM_lpage">70</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.2174%2F138620704772884832" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=14965262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2004&pages=63-70&author=G.+Harperauthor=S.+D.+Pickettauthor=D.+V.+Green&title=Design+of+a+compound+screening+collection+for+use+in+high+throughput+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a compound screening collection for use in high throughput screening</span></div><div class="casAuthors">Harper, G.; Pickett, S. D.; Green, D. V. S.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-71</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">In this paper we introduce a quant. model that relates chem. structural similarity to biol. activity, and in particular to the activity of lead series of compds. in high-throughput assays.  From this model we derive the optimal screening collection make up for a given fixed size of screening collection, and identify the conditions under which a diverse collection of compds. or a collection focusing on particular regions of chem. space are appropriate strategies.  We derive from the model a diversity function that may be used to assess compds. for acquisition or libraries for combinatorial synthesis by their ability to complement an existing screening collection.  The diversity function is linked directly through the model to the goal of more frequent discovery of lead series from high-throughput screening.  We show how the model may also be used to derive relationships between collection size and probabilities of lead discovery in high-throughput screening, and to guide the judicious application of structural filters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpThXGuh69P87Vg90H21EOLACvtfcHk0lhrBHKnCaif3w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D&md5=b29b45b082fe5fc9d6400e1c8fd83cc0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2174%2F138620704772884832&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620704772884832%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DGreen%26aufirst%3DD.%2BV.%26atitle%3DDesign%2520of%2520a%2520compound%2520screening%2520collection%2520for%2520use%2520in%2520high%2520throughput%2520screening%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2004%26volume%3D7%26spage%3D63%26epage%3D70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Carhart, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkataraghavan, R.</span><span> </span><span class="NLM_article-title">Atom pairs as molecular features in structure-activity studies: Definition and applications</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">64</span><span class="NLM_x">â</span> <span class="NLM_lpage">73</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci00046a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADyaL2MXitVehu7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1985&pages=64-73&author=R.+E.+Carhartauthor=D.+H.+Smithauthor=R.+Venkataraghavan&title=Atom+pairs+as+molecular+features+in+structure-activity+studies%3A+Definition+and+applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Atom pairs as molecular features in structure-activity studies:  definition and applications</span></div><div class="casAuthors">Carhart, Raymond E.; Smith, Dennis H.; Venkataraghavan, R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">64-73</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">A simple type of substructure called an atom pair is defined in terms of the at. environments of, and shortest path sepns. between, all pairs of atoms in the topol. representation of a chem. structure.  An algorithm is presented for computing atom pairs from such a representation.  Two applications of atom pairs to structure-activity problems are described.  In the 1st, a measure of similarity between compds. is defined, and the use of this measure in probing large data bases of structures is discussed.  In the 2nd, an heuristic technique called trend vector anal. is described.  The trend vector summarizes the correlation, within a set of structures, of the occurrence of atom pairs of different types with measured biol. activity.  These correlations can be used to est. the biol. activity of new compds.  A comparison of trend vector anal. with discriminant plane anal. is presented for 1 series of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV9lxLct8p0rVg90H21EOLACvtfcHk0lhrBHKnCaif3w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXitVehu7o%253D&md5=9e2cc9e9fda27db87328b738cd6878e2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci00046a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci00046a002%26sid%3Dliteratum%253Aachs%26aulast%3DCarhart%26aufirst%3DR.%2BE.%26aulast%3DSmith%26aufirst%3DD.%2BH.%26aulast%3DVenkataraghavan%26aufirst%3DR.%26atitle%3DAtom%2520pairs%2520as%2520molecular%2520features%2520in%2520structure-activity%2520studies%253A%2520Definition%2520and%2520applications%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1985%26volume%3D25%26spage%3D64%26epage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Similarity-based virtual screening using 2D fingerprints</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">â</span> <span class="NLM_lpage">1053</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1016%2Fj.drudis.2006.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=17129822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1eqtb%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=1046-1053&author=P.+Willett&title=Similarity-based+virtual+screening+using+2D+fingerprints"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity-based virtual screening using 2D fingerprints</span></div><div class="casAuthors">Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">23 & 24</span>),
    <span class="NLM_cas:pages">1046-1053</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  This paper summarizes recent work at the University of Sheffield on virtual screening methods that use 2D fingerprint measures of structural similarity.  A detailed comparison of a large no. of similarity coeffs. demonstrates that the well-known Tanimoto coeff. remains the method of choice for the computation of fingerprint-based similarity, despite possessing some inherent biases related to the sizes of the mols. that are being sought.  Group fusion involves combining the results of similarity searches based on multiple ref. structures and a single similarity measure.  We demonstrate the effectiveness of this approach to screening, and also describe an approx. form of group fusion, turbo similarity searching, that can be used when just a single ref. structure is available.  This paper discusses the use of binary-encoded fragment substructures to scan databases to find mols. that are structurally similar to a bioactive query compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3TRt6CMU5VrVg90H21EOLACvtfcHk0lhrBHKnCaif3w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1eqtb%252FN&md5=095fe152b1ae1abbea69f02123651874</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DSimilarity-based%2520virtual%2520screening%2520using%25202D%2520fingerprints%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D1046%26epage%3D1053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Notredame, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heringa, J.</span><span> </span><span class="NLM_article-title">T-coffee: A novel method for fast and accurate multiple sequence alignment</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">302</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">â</span> <span class="NLM_lpage">217</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1006%2Fjmbi.2000.4042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=10964570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtVGntr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=302&publication_year=2000&pages=205-217&author=C.+Notredameauthor=D.+G.+Higginsauthor=J.+Heringa&title=T-coffee%3A+A+novel+method+for+fast+and+accurate+multiple+sequence+alignment"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">T-Coffee: A Novel Method for Fast and Accurate Multiple Sequence Alignment</span></div><div class="casAuthors">Notredame, Cedric; Higgins, Desmond G.; Heringa, Jaap</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">302</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">205-217</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">We describe a new method (T-Coffee) for multiple sequence alignment that provides a dramatic improvement in accuracy with a modest sacrifice in speed as compared to the most commonly used alternatives.  The method is broadly based on the popular progressive approach to multiple alignment but avoids the most serious pitfalls caused by the greedy nature of this algorithm.  With T-Coffee we pre-process a data set of all pair-wise alignments between the sequences.  This provides us with a library of alignment information that can be used to guide the progressive alignment.  Intermediate alignments are then based not only on the sequences to be aligned next but also on how all of the sequences align with each other.  This alignment information can be derived from heterogeneous sources such as a mixt. of alignment programs and/or structure superposition.  Here, we illustrate the power of the approach by using a combination of local and global pair-wise alignments to generate the library.  The resulting alignments are significantly more reliable, as detd. by comparison with a set of 141 test cases, than any of the popular alternatives that we tried.  The improvement, esp. clear with the more difficult test cases, is always visible, regardless of the phylogenetic spread of the sequences in the tests.  (c) 2000 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraooGHJVZRYrVg90H21EOLACvtfcHk0lhrBHKnCaif3w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtVGntr8%253D&md5=62c7b0c4e224acf1b172810571d5755d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2000.4042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2000.4042%26sid%3Dliteratum%253Aachs%26aulast%3DNotredame%26aufirst%3DC.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26aulast%3DHeringa%26aufirst%3DJ.%26atitle%3DT-coffee%253A%2520A%2520novel%2520method%2520for%2520fast%2520and%2520accurate%2520multiple%2520sequence%2520alignment%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2000%26volume%3D302%26spage%3D205%26epage%3D217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span> <span class="citation_source-book">ClassPharmer</span>, version 4.5; <span class="NLM_publisher-name">simulationsplus, Inc.</span>: <span class="NLM_publisher-loc">Lancaster, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ClassPharmer%2C+version+4.5%3B+simulationsplus%2C+Inc.%3A+Lancaster%2C+CA%2C+2008."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DClassPharmer%26pub%3Dsimulationsplus%252C%2520Inc%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span> <span class="citation_source-book">Pipeline Pilot</span>, version 7.5; <span class="NLM_publisher-name">Accelrys</span>: <span class="NLM_publisher-loc">San Diego</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Pipeline+Pilot%2C+version+7.5%3B+Accelrys%3A+San+Diego%2C+2008."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DPipeline%2520Pilot%26pub%3DAccelrys%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span> <span class="citation_source-book">OEChem</span>, version 1.7.0; <span class="NLM_publisher-name">OpenEye Scientific Software, Inc.</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=OEChem%2C+version+1.7.0%3B+OpenEye+Scientific+Software%2C+Inc.%3A+Santa+Fe%2C+NM%2C+2008."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DOEChem%26pub%3DOpenEye%2520Scientific%2520Software%252C%2520Inc%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Crivori, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrupt, P.-A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, B.</span><span> </span><span class="NLM_article-title">Predicting blood-brain barrier permeation from three-dimensional molecular structure</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">2204</span><span class="NLM_x">â</span> <span class="NLM_lpage">2216</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm990968%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjtFKhtLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=2204-2216&author=P.+Crivoriauthor=G.+Crucianiauthor=P.-A.+Carruptauthor=B.+Testa&title=Predicting+blood-brain+barrier+permeation+from+three-dimensional+molecular+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting blood-brain barrier permeation from three-dimensional molecular structure</span></div><div class="casAuthors">Crivori, Patrizia; Cruciani, Gabriele; Carrupt, Pierre-Alain; Testa, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2204-2216</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Predicting blood-brain barrier (BBB) permeation remains a challenge in drug design.  Since it is impossible to det. exptl. the BBB partitioning of large nos. of preclin. candidates, alternative evaluation methods based on computerized models are desirable.  The present study was conducted to demonstrate the value of descriptors derived from 3D mol. fields in estg. the BBB permeation of a large set of compds. and to produce a simple math. model suitable for external prediction.  The method used (VolSurf) transforms 3D fields into descriptors and correlates them to the exptl. permeation by a discriminant partial least squares procedure.  The model obtained here correctly predicts more than 90% of the BBB permeation data.  By quantifying the favorable and unfavorable contributions of physicochem. and structural properties, it also offers valuable insights for drug design, pharmacol. profiling, and screening.  The computational procedure is fully automated and quite fast.  The method thus appears as a valuable new tool in virtual screening where selection or prioritization of candidates is required from large collections of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoURsaBTpmsObVg90H21EOLACvtfcHk0ljOPIFdRFzNZg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjtFKhtLk%253D&md5=569f3500268859660bb9e5206da270fe</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm990968%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990968%252B%26sid%3Dliteratum%253Aachs%26aulast%3DCrivori%26aufirst%3DP.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DCarrupt%26aufirst%3DP.-A.%26aulast%3DTesta%26aufirst%3DB.%26atitle%3DPredicting%2520blood-brain%2520barrier%2520permeation%2520from%2520three-dimensional%2520molecular%2520structure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D2204%26epage%3D2216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guba, W.</span><span> </span><span class="NLM_article-title">VolSurf: A new tool for the pharmacokinetic optimization of lead compounds</span> <span class="citation_source-journal">Eur. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">S29</span><span class="NLM_x">â</span> <span class="NLM_lpage">S39</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1016%2FS0928-0987%2800%2900162-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=11033425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntF2gtbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2000&pages=S29-S39&author=G.+Crucianiauthor=M.+Pastorauthor=W.+Guba&title=VolSurf%3A+A+new+tool+for+the+pharmacokinetic+optimization+of+lead+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">VolSurf: a new tool for the pharmacokinetic optimization of lead compounds</span></div><div class="casAuthors">Cruciani, G.; Pastor, M.; Guba, W.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">S29-S39</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A method for the modeling and prediction of pharmacokinetic properties based on computed mol. interaction fields and multivariate statistics has been investigated in different exptl. datasets.  The program VolSurf was used to correlate 3D mol. structures with physico-chem. and pharmacokinetic properties.  In membrane partitioning, VolSurf produced a two-component model explaining 94% of the total variation with a predictive q2 of 0.90.  This result was achieved without conformational sampling and without any quantum-chem. calcn.  For the prediction of blood-brain barrier penetration the VolSurf model was able to predict the BBB profile for most of the drugs in the external prediction set.  In Caco-2 and MDCK permeation expts., VolSurf was used with success to establish statistical models and to predict the behavior of new compds.  The method thus appears as a valuable new property filter in virtual screening and as a novel tool in optimizing the pharmacokinetic profile of pharmaceutically relevant compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCi7-OnhbhmbVg90H21EOLACvtfcHk0ljOPIFdRFzNZg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntF2gtbo%253D&md5=c1ed35a3b3180548485ef1faa833f6c0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2FS0928-0987%2800%2900162-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0928-0987%252800%252900162-7%26sid%3Dliteratum%253Aachs%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DPastor%26aufirst%3DM.%26aulast%3DGuba%26aufirst%3DW.%26atitle%3DVolSurf%253A%2520A%2520new%2520tool%2520for%2520the%2520pharmacokinetic%2520optimization%2520of%2520lead%2520compounds%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2000%26volume%3D11%26spage%3DS29%26epage%3DS39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 58 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xutong Li, Zhaojun Li, Xiaolong Wu, Zhaoping Xiong, Tianbiao Yang, Zunyun Fu, Xiaohong Liu, Xiaoqin Tan, Feisheng Zhong, Xiaozhe Wan, Dingyan Wang, Xiaoyu Ding, Ruirui Yang, Hui Hou, Chunpu Li, Hong Liu, Kaixian Chen, Hualiang Jiang, <span class="NLM_string-name hlFld-ContribAuthor">Mingyue Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Deep Learning Enhancing Kinome-Wide Polypharmacology Profiling: Model Construction and Experiment Validation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (16)
                                     , 8723-8737. <a href="https://doi.org/10.1021/acs.jmedchem.9b00855" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00855</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00855%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDeep%252BLearning%252BEnhancing%252BKinome-Wide%252BPolypharmacology%252BProfiling%25253A%252BModel%252BConstruction%252Band%252BExperiment%252BValidation%26aulast%3DLi%26aufirst%3DXutong%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D26052019%26date%3D15082019%26date%3D31072019%26volume%3D63%26issue%3D16%26spage%3D8723%26epage%3D8737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrei Ursu, Jessica L. Childs-Disney, Alicia J. Angelbello, Matthew G. Costales, Samantha M. Meyer, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Disney</span>. </span><span class="cited-content_cbyCitation_article-title">Gini Coefficients as a Single Value Metric to Define Chemical Probe Selectivity. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (8)
                                     , 2031-2040. <a href="https://doi.org/10.1021/acschembio.0c00486" title="DOI URL">https://doi.org/10.1021/acschembio.0c00486</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00486%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DGini%252BCoefficients%252Bas%252Ba%252BSingle%252BValue%252BMetric%252Bto%252BDefine%252BChemical%252BProbe%252BSelectivity%26aulast%3DUrsu%26aufirst%3DAndrei%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15062020%26date%3D22062020%26date%3D09072020%26date%3D22062020%26volume%3D15%26issue%3D8%26spage%3D2031%26epage%3D2040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Scott D. Bembenek, Gavin Hirst, <span class="NLM_string-name hlFld-ContribAuthor">Taraneh Mirzadegan</span>. </span><span class="cited-content_cbyCitation_article-title">Determination of a Focused Mini Kinase Panel for Early Identification of Selective Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (7)
                                     , 1434-1440. <a href="https://doi.org/10.1021/acs.jcim.8b00222" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00222</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00222%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDetermination%252Bof%252Ba%252BFocused%252BMini%252BKinase%252BPanel%252Bfor%252BEarly%252BIdentification%252Bof%252BSelective%252BKinase%252BInhibitors%26aulast%3DBembenek%26aufirst%3DScott%2BD.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D12042018%26date%3D22062018%26date%3D24052018%26volume%3D58%26issue%3D7%26spage%3D1434%26epage%3D1440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephanie  Heinzlmeir</span>, <span class="hlFld-ContribAuthor ">Denis  Kudlinzki</span>, <span class="hlFld-ContribAuthor ">Sridhar  Sreeramulu</span>, <span class="hlFld-ContribAuthor ">Susan  Klaeger</span>, <span class="hlFld-ContribAuthor ">Santosh Lakshmi  Gande</span>, <span class="hlFld-ContribAuthor ">Verena  Linhard</span>, <span class="hlFld-ContribAuthor ">Mathias  Wilhelm</span>, <span class="hlFld-ContribAuthor ">Huichao  Qiao</span>, <span class="hlFld-ContribAuthor ">Dominic  Helm</span>, <span class="hlFld-ContribAuthor ">Benjamin  Ruprecht</span>, <span class="hlFld-ContribAuthor ">Krishna  Saxena</span>, <span class="hlFld-ContribAuthor ">Guillaume  MÃ©dard</span>, <span class="hlFld-ContribAuthor ">Harald  Schwalbe</span>, and <span class="hlFld-ContribAuthor ">Bernhard  Kuster</span>  . </span><span class="cited-content_cbyCitation_article-title">Chemical Proteomics and Structural Biology Define EPHA2 Inhibition by Clinical Kinase Drugs. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2016,</strong> <em>11 </em>
                                    (12)
                                     , 3400-3411. <a href="https://doi.org/10.1021/acschembio.6b00709" title="DOI URL">https://doi.org/10.1021/acschembio.6b00709</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.6b00709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.6b00709%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DChemical%252BProteomics%252Band%252BStructural%252BBiology%252BDefine%252BEPHA2%252BInhibition%252Bby%252BClinical%252BKinase%252BDrugs%26aulast%3DHeinzlmeir%26aufirst%3DStephanie%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D16082016%26date%3D21102016%26date%3D07112016%26date%3D16122016%26date%3D21102016%26volume%3D11%26issue%3D12%26spage%3D3400%26epage%3D3411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Serge  Christmann-Franck</span>, <span class="hlFld-ContribAuthor ">Gerard J. P.  van Westen</span>, <span class="hlFld-ContribAuthor ">George  Papadatos</span>, <span class="hlFld-ContribAuthor ">Fanny  Beltran Escudie</span>, <span class="hlFld-ContribAuthor ">Alexander  Roberts</span>, <span class="hlFld-ContribAuthor ">John P.  Overington</span>, and <span class="hlFld-ContribAuthor ">Daniel  Domine</span>  . </span><span class="cited-content_cbyCitation_article-title">Unprecedently Large-Scale Kinase Inhibitor Set Enabling the Accurate Prediction of CompoundâKinase Activities: A Way toward Selective Promiscuity by Design?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (9)
                                     , 1654-1675. <a href="https://doi.org/10.1021/acs.jcim.6b00122" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00122</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00122%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DUnprecedently%252BLarge-Scale%252BKinase%252BInhibitor%252BSet%252BEnabling%252Bthe%252BAccurate%252BPrediction%252Bof%252BCompound%2525E2%252580%252593Kinase%252BActivities%25253A%252BA%252BWay%252Btoward%252BSelective%252BPromiscuity%252Bby%252BDesign%25253F%26aulast%3DChristmann-Franck%26aufirst%3DSerge%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D03032016%26date%3D11082016%26date%3D26092016%26date%3D02082016%26volume%3D56%26issue%3D9%26spage%3D1654%26epage%3D1675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alina  Bora</span>, <span class="hlFld-ContribAuthor ">Sorin  Avram</span>, <span class="hlFld-ContribAuthor ">Ionel  Ciucanu</span>, <span class="hlFld-ContribAuthor ">Marius  Raica</span>, and <span class="hlFld-ContribAuthor ">Stefana  Avram</span>  . </span><span class="cited-content_cbyCitation_article-title">Predictive Models for Fast and Effective Profiling of Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (5)
                                     , 895-905. <a href="https://doi.org/10.1021/acs.jcim.5b00646" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00646</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00646%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPredictive%252BModels%252Bfor%252BFast%252Band%252BEffective%252BProfiling%252Bof%252BKinase%252BInhibitors%26aulast%3DBora%26aufirst%3DAlina%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D24102015%26date%3D26042016%26date%3D23052016%26date%3D11042016%26volume%3D56%26issue%3D5%26spage%3D895%26epage%3D905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ling-Jie  Gao</span>, <span class="hlFld-ContribAuthor ">Sona  Kovackova</span>, <span class="hlFld-ContribAuthor ">Michal  Å Ã¡la</span>, <span class="hlFld-ContribAuthor ">Anna Teresa  Ramadori</span>, <span class="hlFld-ContribAuthor ">Steven  De Jonghe</span>, and <span class="hlFld-ContribAuthor ">Piet  Herdewijn</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Dual Death-Associated Protein Related Apoptosis Inducing Protein Kinase 1 and 2 Inhibitors by a Scaffold Hopping Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (18)
                                     , 7624-7643. <a href="https://doi.org/10.1021/jm5007929" title="DOI URL">https://doi.org/10.1021/jm5007929</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm5007929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm5007929%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BDual%252BDeath-Associated%252BProtein%252BRelated%252BApoptosis%252BInducing%252BProtein%252BKinase%252B1%252Band%252B2%252BInhibitors%252Bby%252Ba%252BScaffold%252BHopping%252BApproach%26aulast%3DGao%26aufirst%3DLing-Jie%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D22052014%26date%3D11092014%26date%3D25092014%26date%3D02092014%26volume%3D57%26issue%3D18%26spage%3D7624%26epage%3D7643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Atli  Thorarensen</span>, <span class="hlFld-ContribAuthor ">Mary Ellen  Banker</span>, <span class="hlFld-ContribAuthor ">Andrew  Fensome</span>, <span class="hlFld-ContribAuthor ">Jean-Baptiste  Telliez</span>, <span class="hlFld-ContribAuthor ">Brian  Juba</span>, <span class="hlFld-ContribAuthor ">Fabien  Vincent</span>, <span class="hlFld-ContribAuthor ">Robert M.  Czerwinski</span>, and <span class="hlFld-ContribAuthor ">Agustin  Casimiro-Garcia</span>  . </span><span class="cited-content_cbyCitation_article-title">ATP-Mediated Kinome Selectivity: The Missing Link in Understanding the Contribution of Individual JAK Kinase Isoforms to Cellular Signaling. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2014,</strong> <em>9 </em>
                                    (7)
                                     , 1552-1558. <a href="https://doi.org/10.1021/cb5002125" title="DOI URL">https://doi.org/10.1021/cb5002125</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb5002125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb5002125%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DATP-Mediated%252BKinome%252BSelectivity%25253A%252BThe%252BMissing%252BLink%252Bin%252BUnderstanding%252Bthe%252BContribution%252Bof%252BIndividual%252BJAK%252BKinase%252BIsoforms%252Bto%252BCellular%252BSignaling%26aulast%3DThorarensen%26aufirst%3DAtli%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D19032014%26date%3D12052014%26date%3D28052014%26date%3D18072014%26date%3D12052014%26volume%3D9%26issue%3D7%26spage%3D1552%26epage%3D1558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shana L.  Posy</span>, <span class="hlFld-ContribAuthor ">Brian L.  Claus</span>, <span class="hlFld-ContribAuthor ">Matt E.  Pokross</span>, and <span class="hlFld-ContribAuthor ">Stephen R.  Johnson</span>  . </span><span class="cited-content_cbyCitation_article-title">3D Matched Pairs: Integrating Ligand- and Structure-Based Knowledge for Ligand Design and Receptor Annotation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (7)
                                     , 1576-1588. <a href="https://doi.org/10.1021/ci400201k" title="DOI URL">https://doi.org/10.1021/ci400201k</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400201k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400201k%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3D3D%252BMatched%252BPairs%25253A%252BIntegrating%252BLigand-%252Band%252BStructure-Based%252BKnowledge%252Bfor%252BLigand%252BDesign%252Band%252BReceptor%252BAnnotation%26aulast%3DPosy%26aufirst%3DShana%2BL.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D04042013%26date%3D10072013%26date%3D22072013%26date%3D30062013%26volume%3D53%26issue%3D7%26spage%3D1576%26epage%3D1588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert  Urich</span>, <span class="hlFld-ContribAuthor ">Grant  Wishart</span>, <span class="hlFld-ContribAuthor ">Michael  Kiczun</span>, <span class="hlFld-ContribAuthor ">AndrÃ©  Richters</span>, <span class="hlFld-ContribAuthor ">Naomi  Tidten-Luksch</span>, <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>, <span class="hlFld-ContribAuthor ">Brad  Sherborne</span>, <span class="hlFld-ContribAuthor ">Paul G.  Wyatt</span>, and <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>  . </span><span class="cited-content_cbyCitation_article-title">De Novo Design of Protein Kinase Inhibitors by in Silico Identification of Hinge Region-Binding Fragments. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2013,</strong> <em>8 </em>
                                    (5)
                                     , 1044-1052. <a href="https://doi.org/10.1021/cb300729y" title="DOI URL">https://doi.org/10.1021/cb300729y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb300729y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb300729y%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDe%252BNovo%252BDesign%252Bof%252BProtein%252BKinase%252BInhibitors%252Bby%252Bin%252BSilico%252BIdentification%252Bof%252BHinge%252BRegion-Binding%252BFragments%26aulast%3DUrich%26aufirst%3DRobert%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D28122012%26date%3D12032013%26date%3D27032013%26date%3D17052013%26volume%3D8%26issue%3D5%26spage%3D1044%26epage%3D1052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shunye  Zhou</span>, <span class="hlFld-ContribAuthor ">Youyong  Li</span>, and <span class="hlFld-ContribAuthor ">Tingjun  Hou</span>  . </span><span class="cited-content_cbyCitation_article-title">Feasibility of Using Molecular Docking-Based Virtual Screening for Searching Dual Target Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (4)
                                     , 982-996. <a href="https://doi.org/10.1021/ci400065e" title="DOI URL">https://doi.org/10.1021/ci400065e</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400065e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400065e%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFeasibility%252Bof%252BUsing%252BMolecular%252BDocking-Based%252BVirtual%252BScreening%252Bfor%252BSearching%252BDual%252BTarget%252BKinase%252BInhibitors%26aulast%3DZhou%26aufirst%3DShunye%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D28012013%26date%3D03042013%26date%3D22042013%26date%3D18032013%26volume%3D53%26issue%3D4%26spage%3D982%26epage%3D996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">N. Yi  Mok</span>, <span class="hlFld-ContribAuthor ">Sara  Maxe</span>, and <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>  . </span><span class="cited-content_cbyCitation_article-title">Locating Sweet Spots for Screening Hits and Evaluating Pan-Assay Interference Filters from the Performance Analysis of Two Lead-like Libraries. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (3)
                                     , 534-544. <a href="https://doi.org/10.1021/ci300382f" title="DOI URL">https://doi.org/10.1021/ci300382f</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci300382f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci300382f%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLocating%252BSweet%252BSpots%252Bfor%252BScreening%252BHits%252Band%252BEvaluating%252BPan-Assay%252BInterference%252BFilters%252Bfrom%252Bthe%252BPerformance%252BAnalysis%252Bof%252BTwo%252BLead-like%252BLibraries%26aulast%3DMok%26aufirst%3DN.%2BYi%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D15082012%26date%3D04032013%26date%3D25032013%26volume%3D53%26issue%3D3%26spage%3D534%26epage%3D544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephan C.  SchÃ¼rer</span> and <span class="hlFld-ContribAuthor ">Steven M.  Muskal</span>  . </span><span class="cited-content_cbyCitation_article-title">Kinome-wide Activity Modeling from Diverse Public High-Quality Data Sets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (1)
                                     , 27-38. <a href="https://doi.org/10.1021/ci300403k" title="DOI URL">https://doi.org/10.1021/ci300403k</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci300403k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci300403k%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DKinome-wide%252BActivity%252BModeling%252Bfrom%252BDiverse%252BPublic%252BHigh-Quality%252BData%252BSets%26aulast%3DSch%25C3%25BCrer%26aufirst%3DStephan%2BC.%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D24082012%26date%3D09012013%26date%3D28012013%26date%3D21122012%26volume%3D53%26issue%3D1%26spage%3D27%26epage%3D38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Richard A.  Norman</span>, <span class="hlFld-ContribAuthor ">Anne-Kathrin  Schott</span>, <span class="hlFld-ContribAuthor ">David M.  Andrews</span>, <span class="hlFld-ContribAuthor ">Jason  Breed</span>, <span class="hlFld-ContribAuthor ">Kevin M.  Foote</span>, <span class="hlFld-ContribAuthor ">Andrew P.  Garner</span>, <span class="hlFld-ContribAuthor ">Derek  Ogg</span>, <span class="hlFld-ContribAuthor ">Jonathon P.  Orme</span>, <span class="hlFld-ContribAuthor ">Jennifer H.  Pink</span>, <span class="hlFld-ContribAuthor ">Karen  Roberts</span>, <span class="hlFld-ContribAuthor ">David A.  Rudge</span>, <span class="hlFld-ContribAuthor ">Andrew P.  Thomas</span>, and <span class="hlFld-ContribAuthor ">Andrew G.  Leach</span>  . </span><span class="cited-content_cbyCitation_article-title">ProteinâLigand Crystal Structures Can Guide the Design of Selective Inhibitors of the FGFR Tyrosine Kinase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (11)
                                     , 5003-5012. <a href="https://doi.org/10.1021/jm3004043" title="DOI URL">https://doi.org/10.1021/jm3004043</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm3004043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm3004043%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProtein%2525E2%252580%252593Ligand%252BCrystal%252BStructures%252BCan%252BGuide%252Bthe%252BDesign%252Bof%252BSelective%252BInhibitors%252Bof%252Bthe%252BFGFR%252BTyrosine%252BKinase%26aulast%3DNorman%26aufirst%3DRichard%2BA.%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D15112011%26date%3D21052012%26date%3D14062012%26volume%3D55%26issue%3D11%26spage%3D5003%26epage%3D5012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huifen  Chen</span>, <span class="hlFld-ContribAuthor ">Bryan K.  Chan</span>, <span class="hlFld-ContribAuthor ">Jason  Drummond</span>, <span class="hlFld-ContribAuthor ">Anthony A.  Estrada</span>, <span class="hlFld-ContribAuthor ">Janet  Gunzner-Toste</span>, <span class="hlFld-ContribAuthor ">Xingrong  Liu</span>, <span class="hlFld-ContribAuthor ">Yichin  Liu</span>, <span class="hlFld-ContribAuthor ">John  Moffat</span>, <span class="hlFld-ContribAuthor ">Daniel  Shore</span>, <span class="hlFld-ContribAuthor ">Zachary K.  Sweeney</span>, <span class="hlFld-ContribAuthor ">Thuy  Tran</span>, <span class="hlFld-ContribAuthor ">Shumei  Wang</span>, <span class="hlFld-ContribAuthor ">Guiling  Zhao</span>, <span class="hlFld-ContribAuthor ">Haitao  Zhu</span>, and <span class="hlFld-ContribAuthor ">Daniel J.  Burdick</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Selective LRRK2 Inhibitors Guided by Computational Analysis and Molecular Modeling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (11)
                                     , 5536-5545. <a href="https://doi.org/10.1021/jm300452p" title="DOI URL">https://doi.org/10.1021/jm300452p</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm300452p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm300452p%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BSelective%252BLRRK2%252BInhibitors%252BGuided%252Bby%252BComputational%252BAnalysis%252Band%252BMolecular%252BModeling%26aulast%3DChen%26aufirst%3DHuifen%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D31032012%26date%3D01062012%26date%3D14062012%26date%3D16052012%26volume%3D55%26issue%3D11%26spage%3D5536%26epage%3D5545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Satoshi  Niijima</span>, <span class="hlFld-ContribAuthor ">Akira  Shiraishi</span>, and <span class="hlFld-ContribAuthor ">Yasushi  Okuno</span>  . </span><span class="cited-content_cbyCitation_article-title">Dissecting Kinase Profiling Data to Predict Activity and Understand Cross-Reactivity of Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2012,</strong> <em>52 </em>
                                    (4)
                                     , 901-912. <a href="https://doi.org/10.1021/ci200607f" title="DOI URL">https://doi.org/10.1021/ci200607f</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200607f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200607f%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDissecting%252BKinase%252BProfiling%252BData%252Bto%252BPredict%252BActivity%252Band%252BUnderstand%252BCross-Reactivity%252Bof%252BKinase%252BInhibitors%26aulast%3DNiijima%26aufirst%3DSatoshi%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D17122011%26date%3D26032012%26date%3D23042012%26date%3D13032012%26volume%3D52%26issue%3D4%26spage%3D901%26epage%3D912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin W.  Rowbottom</span>, <span class="hlFld-ContribAuthor ">Raffaella  Faraoni</span>, <span class="hlFld-ContribAuthor ">Qi  Chao</span>, <span class="hlFld-ContribAuthor ">Brian T.  Campbell</span>, <span class="hlFld-ContribAuthor ">Andiliy G.  Lai</span>, <span class="hlFld-ContribAuthor ">Eduardo  Setti</span>, <span class="hlFld-ContribAuthor ">Maiko  Ezawa</span>, <span class="hlFld-ContribAuthor ">Kelly G.  Sprankle</span>, <span class="hlFld-ContribAuthor ">Sunny  Abraham</span>, <span class="hlFld-ContribAuthor ">Lan  Tran</span>, <span class="hlFld-ContribAuthor ">Brian  Struss</span>, <span class="hlFld-ContribAuthor ">Michael  Gibney</span>, <span class="hlFld-ContribAuthor ">Robert C.  Armstrong</span>, <span class="hlFld-ContribAuthor ">Ruwanthi N.  Gunawardane</span>, <span class="hlFld-ContribAuthor ">Ronald R.  Nepomuceno</span>, <span class="hlFld-ContribAuthor ">Ianina  Valenta</span>, <span class="hlFld-ContribAuthor ">Helen  Hua</span>, <span class="hlFld-ContribAuthor ">Michael F.  Gardner</span>, <span class="hlFld-ContribAuthor ">Merryl D.  Cramer</span>, <span class="hlFld-ContribAuthor ">Dana  Gitnick</span>, <span class="hlFld-ContribAuthor ">Darren E.  Insko</span>, <span class="hlFld-ContribAuthor ">Julius L.  Apuy</span>, <span class="hlFld-ContribAuthor ">Susan  Jones-Bolin</span>, <span class="hlFld-ContribAuthor ">Arup K.  Ghose</span>, <span class="hlFld-ContribAuthor ">Torsten  Herbertz</span>, <span class="hlFld-ContribAuthor ">Mark A.  Ator</span>, <span class="hlFld-ContribAuthor ">Bruce D.  Dorsey</span>, <span class="hlFld-ContribAuthor ">Bruce  Ruggeri</span>, <span class="hlFld-ContribAuthor ">Michael  Williams</span>, <span class="hlFld-ContribAuthor ">Shripad  Bhagwat</span>, <span class="hlFld-ContribAuthor ">Joyce  James</span>, and <span class="hlFld-ContribAuthor ">Mark W.  Holladay</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of 1-(3-(6,7-Dimethoxyquinazolin-4-yloxy)phenyl)-3-(5-(1,1,1-trifluoro-2-methylpropan-2-yl)isoxazol-3-yl)urea Hydrochloride (CEP-32496), a Highly Potent and Orally Efficacious Inhibitor of V-RAF Murine Sarcoma Viral Oncogene Homologue B1 (BRAF) V600E. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (3)
                                     , 1082-1105. <a href="https://doi.org/10.1021/jm2009925" title="DOI URL">https://doi.org/10.1021/jm2009925</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm2009925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm2009925%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252B1-%2525283-%2525286%25252C7-Dimethoxyquinazolin-4-yloxy%252529phenyl%252529-3-%2525285-%2525281%25252C1%25252C1-trifluoro-2-methylpropan-2-yl%252529isoxazol-3-yl%252529urea%252BHydrochloride%252B%252528CEP-32496%252529%25252C%252Ba%252BHighly%252BPotent%252Band%252BOrally%252BEfficacious%252BInhibitor%252Bof%252BV-RAF%252BMurine%252BSarcoma%252BViral%252BOncogene%252BHomologue%252BB1%252B%252528BRAF%252529%252BV600E%26aulast%3DRowbottom%26aufirst%3DMartin%2BW.%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D25072011%26date%3D23012012%26date%3D09022012%26date%3D14122011%26volume%3D55%26issue%3D3%26spage%3D1082%26epage%3D1105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven W.  Millward</span>, <span class="hlFld-ContribAuthor ">Ryan K.  Henning</span>, <span class="hlFld-ContribAuthor ">Gabriel A.  Kwong</span>, <span class="hlFld-ContribAuthor ">Suresh  Pitram</span>, <span class="hlFld-ContribAuthor ">Heather D.  Agnew</span>, <span class="hlFld-ContribAuthor ">Kaycie M.  Deyle</span>, <span class="hlFld-ContribAuthor ">Arundhati  Nag</span>, <span class="hlFld-ContribAuthor ">Jason  Hein</span>, <span class="hlFld-ContribAuthor ">Su Seong  Lee</span>, <span class="hlFld-ContribAuthor ">Jaehong  Lim</span>, <span class="hlFld-ContribAuthor ">Jessica A.  Pfeilsticker</span>, <span class="hlFld-ContribAuthor ">K. Barry  Sharpless</span>, and <span class="hlFld-ContribAuthor ">James R.  Heath</span>  . </span><span class="cited-content_cbyCitation_article-title">Iterative in Situ Click Chemistry Assembles a Branched Capture Agent and Allosteric Inhibitor for Akt1. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2011,</strong> <em>133 </em>
                                    (45)
                                     , 18280-18288. <a href="https://doi.org/10.1021/ja2064389" title="DOI URL">https://doi.org/10.1021/ja2064389</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ja2064389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fja2064389%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DIterative%252Bin%252BSitu%252BClick%252BChemistry%252BAssembles%252Ba%252BBranched%252BCapture%252BAgent%252Band%252BAllosteric%252BInhibitor%252Bfor%252BAkt1%26aulast%3DMillward%26aufirst%3DSteven%2BW.%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D11072011%26date%3D24102011%26date%3D16112011%26date%3D30092011%26volume%3D133%26issue%3D45%26spage%3D18280%26epage%3D18288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">W. Patrick  Walters</span>, <span class="hlFld-ContribAuthor ">Jeremy  Green</span>, <span class="hlFld-ContribAuthor ">Jonathan R.  Weiss</span>, and <span class="hlFld-ContribAuthor ">Mark A.  Murcko</span>  . </span><span class="cited-content_cbyCitation_article-title">What Do Medicinal Chemists Actually Make? A 50-Year Retrospective. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>54 </em>
                                    (19)
                                     , 6405-6416. <a href="https://doi.org/10.1021/jm200504p" title="DOI URL">https://doi.org/10.1021/jm200504p</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm200504p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm200504p%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DWhat%252BDo%252BMedicinal%252BChemists%252BActually%252BMake%25253F%252BA%252B50-Year%252BRetrospective%26aulast%3DWalters%26aufirst%3DW.%2BPatrick%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D25042011%26date%3D31082011%26date%3D13102011%26date%3D14072011%26volume%3D54%26issue%3D19%26spage%3D6405%26epage%3D6416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Martin</span>, <span class="hlFld-ContribAuthor ">Prasenjit  Mukherjee</span>, <span class="hlFld-ContribAuthor ">David  Sullivan</span>, and <span class="hlFld-ContribAuthor ">Johanna  Jansen</span>  . </span><span class="cited-content_cbyCitation_article-title">Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (8)
                                     , 1942-1956. <a href="https://doi.org/10.1021/ci1005004" title="DOI URL">https://doi.org/10.1021/ci1005004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci1005004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci1005004%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProfile-QSAR%25253A%252BA%252BNovel%252Bmeta-QSAR%252BMethod%252Bthat%252BCombines%252BActivities%252Bacross%252Bthe%252BKinase%252BFamily%252BTo%252BAccurately%252BPredict%252BAffinity%25252C%252BSelectivity%25252C%252Band%252BCellular%252BActivity%26aulast%3DMartin%26aufirst%3DEric%26date%3D2011%26date%3D2011%26date%3D2010%26date%3D20122010%26date%3D19072011%26date%3D22082011%26date%3D13062011%26volume%3D51%26issue%3D8%26spage%3D1942%26epage%3D1956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul  Bamborough</span>, <span class="hlFld-ContribAuthor ">Murray J.  Brown</span>, <span class="hlFld-ContribAuthor ">John A.  Christopher</span>, <span class="hlFld-ContribAuthor ">Chun-wa  Chung</span>, and <span class="hlFld-ContribAuthor ">Geoff W.  Mellor</span>  . </span><span class="cited-content_cbyCitation_article-title">Selectivity of Kinase Inhibitor Fragments. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>54 </em>
                                    (14)
                                     , 5131-5143. <a href="https://doi.org/10.1021/jm200349b" title="DOI URL">https://doi.org/10.1021/jm200349b</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm200349b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm200349b%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelectivity%252Bof%252BKinase%252BInhibitor%252BFragments%26aulast%3DBamborough%26aufirst%3DPaul%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D25032011%26date%3D23062011%26date%3D28072011%26volume%3D54%26issue%3D14%26spage%3D5131%26epage%3D5143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniele  Carettoni</span>, <span class="hlFld-ContribAuthor ">Benjamin  Bader</span>. </span><span class="cited-content_cbyCitation_article-title">Assay Development and HighâThroughput Screening. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-42. <a href="https://doi.org/10.1002/0471266949.bmc165.pub2" title="DOI URL">https://doi.org/10.1002/0471266949.bmc165.pub2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc165.pub2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc165.pub2%26sid%3Dliteratum%253Aachs%26atitle%3DAssay%252BDevelopment%252Band%252BHigh%2525E2%252580%252590Throughput%252BScreening%26aulast%3DCarettoni%26aufirst%3DDaniele%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D42%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carrow I.  Wells</span>, <span class="hlFld-ContribAuthor ">Hassan  Al-Ali</span>, <span class="hlFld-ContribAuthor ">David M.  Andrews</span>, <span class="hlFld-ContribAuthor ">Christopher R. M.  Asquith</span>, <span class="hlFld-ContribAuthor ">Alison D.  Axtman</span>, <span class="hlFld-ContribAuthor ">Ivan  Dikic</span>, <span class="hlFld-ContribAuthor ">Daniel  Ebner</span>, <span class="hlFld-ContribAuthor ">Peter  Ettmayer</span>, <span class="hlFld-ContribAuthor ">Christian  Fischer</span>, <span class="hlFld-ContribAuthor ">Mathias  Frederiksen</span>, <span class="hlFld-ContribAuthor ">Robert E.  Futrell</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>, <span class="hlFld-ContribAuthor ">Stephanie B.  Hatch</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Ulrich  LÃ¼cking</span>, <span class="hlFld-ContribAuthor ">Michael  Michaelides</span>, <span class="hlFld-ContribAuthor ">Caitlin E.  Mills</span>, <span class="hlFld-ContribAuthor ">Susanne  MÃ¼ller</span>, <span class="hlFld-ContribAuthor ">Dafydd  Owen</span>, <span class="hlFld-ContribAuthor ">Alfredo  Picado</span>, <span class="hlFld-ContribAuthor ">Kumar S.  Saikatendu</span>, <span class="hlFld-ContribAuthor ">Martin  SchrÃ¶der</span>, <span class="hlFld-ContribAuthor ">Alexandra  Stolz</span>, <span class="hlFld-ContribAuthor ">Mariana  Tellechea</span>, <span class="hlFld-ContribAuthor ">Brandon J.  Turunen</span>, <span class="hlFld-ContribAuthor ">Santiago  Vilar</span>, <span class="hlFld-ContribAuthor ">Jinhua  Wang</span>, <span class="hlFld-ContribAuthor ">William J.  Zuercher</span>, <span class="hlFld-ContribAuthor ">Timothy M.  Willson</span>, <span class="hlFld-ContribAuthor ">David H.  Drewry</span>. </span><span class="cited-content_cbyCitation_article-title">The Kinase Chemogenomic Set (KCGS): An Open Science Resource for Kinase Vulnerability Identification. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (2)
                                     , 566. <a href="https://doi.org/10.3390/ijms22020566" title="DOI URL">https://doi.org/10.3390/ijms22020566</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22020566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22020566%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DThe%252BKinase%252BChemogenomic%252BSet%252B%252528KCGS%252529%25253A%252BAn%252BOpen%252BScience%252BResource%252Bfor%252BKinase%252BVulnerability%252BIdentification%26aulast%3DWells%26aufirst%3DCarrow%2BI.%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D2%26spage%3D566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arramshetti  Venkanna</span>, <span class="hlFld-ContribAuthor ">Lalita  Subedi</span>, <span class="hlFld-ContribAuthor ">Mahesh K.  Teli</span>, <span class="hlFld-ContribAuthor ">Prema Dhorma  Lama</span>, <span class="hlFld-ContribAuthor ">Bhargav Gupta  Nangunuri</span>, <span class="hlFld-ContribAuthor ">Sang-Yoon  Lee</span>, <span class="hlFld-ContribAuthor ">Sun Yeou  Kim</span>, <span class="hlFld-ContribAuthor ">Mi-hyun  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">Positioning of an unprecedented spiro[5.5]undeca ring system into kinase inhibitor space. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-78158-9" title="DOI URL">https://doi.org/10.1038/s41598-020-78158-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-78158-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-78158-9%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DPositioning%252Bof%252Ban%252Bunprecedented%252Bspiro%25255B5.5%25255Dundeca%252Bring%252Bsystem%252Binto%252Bkinase%252Binhibitor%252Bspace%26aulast%3DVenkanna%26aufirst%3DArramshetti%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moira  Rachman</span>, <span class="hlFld-ContribAuthor ">DÃ¡vid  Bajusz</span>, <span class="hlFld-ContribAuthor ">AnasztÃ¡zia  HetÃ©nyi</span>, <span class="hlFld-ContribAuthor ">Andrea  Scarpino</span>, <span class="hlFld-ContribAuthor ">BalÃ¡zs  MerÅ</span>, <span class="hlFld-ContribAuthor ">Attila  Egyed</span>, <span class="hlFld-ContribAuthor ">LÃ¡szlÃ³  Buday</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>, <span class="hlFld-ContribAuthor ">GyÃ¶rgy M.  KeserÅ±</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a novel kinase hinge binder fragment by dynamic undocking. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 552-558. <a href="https://doi.org/10.1039/C9MD00519F" title="DOI URL">https://doi.org/10.1039/C9MD00519F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00519F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00519F%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bnovel%252Bkinase%252Bhinge%252Bbinder%252Bfragment%252Bby%252Bdynamic%252Bundocking%26aulast%3DRachman%26aufirst%3DMoira%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D5%26spage%3D552%26epage%3D558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mariana  GonzÃ¡lez-Medina</span>, <span class="hlFld-ContribAuthor ">Filip  MiljkoviÄ</span>, <span class="hlFld-ContribAuthor ">Gernot S.  Haase</span>, <span class="hlFld-ContribAuthor ">Peter  Drueckes</span>, <span class="hlFld-ContribAuthor ">Joerg  Trappe</span>, <span class="hlFld-ContribAuthor ">Stefan  Laufer</span>, <span class="hlFld-ContribAuthor ">JÃ¼rgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Promiscuity analysis of a kinase panel screen with designated p38 alpha inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>187 </em>, 112004. <a href="https://doi.org/10.1016/j.ejmech.2019.112004" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.112004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.112004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.112004%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPromiscuity%252Banalysis%252Bof%252Ba%252Bkinase%252Bpanel%252Bscreen%252Bwith%252Bdesignated%252Bp38%252Balpha%252Binhibitors%26aulast%3DGonz%25C3%25A1lez-Medina%26aufirst%3DMariana%26date%3D2020%26volume%3D187%26spage%3D112004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David H.  Drewry</span>, <span class="hlFld-ContribAuthor ">Carrow I.  Wells</span>, <span class="hlFld-ContribAuthor ">William J.  Zuercher</span>, <span class="hlFld-ContribAuthor ">Timothy M.  Willson</span>. </span><span class="cited-content_cbyCitation_article-title">A Perspective on Extreme Open Science: Companies Sharing Compounds without Restriction. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2019,</strong> <em>24 </em>
                                    (5)
                                     , 505-514. <a href="https://doi.org/10.1177/2472555219838210" title="DOI URL">https://doi.org/10.1177/2472555219838210</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555219838210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555219838210%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DA%252BPerspective%252Bon%252BExtreme%252BOpen%252BScience%25253A%252BCompanies%252BSharing%252BCompounds%252Bwithout%252BRestriction%26aulast%3DDrewry%26aufirst%3DDavid%2BH.%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D5%26spage%3D505%26epage%3D514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daren  Fearon</span>, <span class="hlFld-ContribAuthor ">Isaac M.  Westwood</span>, <span class="hlFld-ContribAuthor ">Rob L.M.  van Montfort</span>, <span class="hlFld-ContribAuthor ">Richard  Bayliss</span>, <span class="hlFld-ContribAuthor ">Keith  Jones</span>, <span class="hlFld-ContribAuthor ">Vassilios  Bavetsias</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and profiling of a 3-aminopyridin-2-one-based kinase targeted fragment library: Identification of 3-amino-5-(pyridin-4-yl)pyridin-2(1H)-one scaffold for monopolar spindle 1 (MPS1) and Aurora kinases inhibition. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (11)
                                     , 3021-3029. <a href="https://doi.org/10.1016/j.bmc.2018.04.033" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.04.033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.04.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.04.033%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bprofiling%252Bof%252Ba%252B3-aminopyridin-2-one-based%252Bkinase%252Btargeted%252Bfragment%252Blibrary%25253A%252BIdentification%252Bof%252B3-amino-5-%252528pyridin-4-yl%252529pyridin-2%2525281H%252529-one%252Bscaffold%252Bfor%252Bmonopolar%252Bspindle%252B1%252B%252528MPS1%252529%252Band%252BAurora%252Bkinases%252Binhibition%26aulast%3DFearon%26aufirst%3DDaren%26date%3D2018%26volume%3D26%26issue%3D11%26spage%3D3021%26epage%3D3029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">You-Lu  Pan</span>, <span class="hlFld-ContribAuthor ">Yan-Ling  Liu</span>, <span class="hlFld-ContribAuthor ">Jian-Zhong  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Simulation Studies on the Binding Selectivity of 1-(1H-Benzimidazol-5-yl)-5-aminopyrazoles in Complexes with FGFR1 and FGFR4. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (4)
                                     , 767. <a href="https://doi.org/10.3390/molecules23040767" title="DOI URL">https://doi.org/10.3390/molecules23040767</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23040767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23040767%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DComputational%252BSimulation%252BStudies%252Bon%252Bthe%252BBinding%252BSelectivity%252Bof%252B1-%2525281H-Benzimidazol-5-yl%252529-5-aminopyrazoles%252Bin%252BComplexes%252Bwith%252BFGFR1%252Band%252BFGFR4%26aulast%3DPan%26aufirst%3DYou-Lu%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D4%26spage%3D767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">C. I.  Wells</span>, <span class="hlFld-ContribAuthor ">N. R.  Kapadia</span>, <span class="hlFld-ContribAuthor ">R. M.  CouÃ±ago</span>, <span class="hlFld-ContribAuthor ">D. H.  Drewry</span>. </span><span class="cited-content_cbyCitation_article-title">In depth analysis of kinase cross screening data to identify chemical starting points for inhibition of the Nek family of kinases. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2018,</strong> <em>9 </em>
                                    (1)
                                     , 44-66. <a href="https://doi.org/10.1039/C7MD00510E" title="DOI URL">https://doi.org/10.1039/C7MD00510E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7MD00510E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7MD00510E%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DIn%252Bdepth%252Banalysis%252Bof%252Bkinase%252Bcross%252Bscreening%252Bdata%252Bto%252Bidentify%252Bchemical%252Bstarting%252Bpoints%252Bfor%252Binhibition%252Bof%252Bthe%252BNek%252Bfamily%252Bof%252Bkinases%26aulast%3DWells%26aufirst%3DC.%2BI.%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D1%26spage%3D44%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dilip  Narayanan</span>, <span class="hlFld-ContribAuthor ">Osman A. B. S. M.  Gani</span>, <span class="hlFld-ContribAuthor ">Franz X. E.  Gruber</span>, <span class="hlFld-ContribAuthor ">Richard A.  Engh</span>. </span><span class="cited-content_cbyCitation_article-title">Data driven polypharmacological drug design for lung cancer: analyses for targeting ALK, MET, and EGFR. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2017,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-017-0229-8" title="DOI URL">https://doi.org/10.1186/s13321-017-0229-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-017-0229-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-017-0229-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DData%252Bdriven%252Bpolypharmacological%252Bdrug%252Bdesign%252Bfor%252Blung%252Bcancer%25253A%252Banalyses%252Bfor%252Btargeting%252BALK%25252C%252BMET%25252C%252Band%252BEGFR%26aulast%3DNarayanan%26aufirst%3DDilip%26date%3D2017%26date%3D2017%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">E. V.  Koroleva</span>, <span class="hlFld-ContribAuthor ">Zh. I.  Ignatovich</span>, <span class="hlFld-ContribAuthor ">Yu. V.  Sinyutich</span>, <span class="hlFld-ContribAuthor ">K. N.  Gusak</span>. </span><span class="cited-content_cbyCitation_article-title">Aminopyrimidine derivatives as protein kinases inhibitors. Molecular design, synthesis, and biologic activity. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2016,</strong> <em>52 </em>
                                    (2)
                                     , 139-177. <a href="https://doi.org/10.1134/S1070428016020019" title="DOI URL">https://doi.org/10.1134/S1070428016020019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428016020019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428016020019%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DAminopyrimidine%252Bderivatives%252Bas%252Bprotein%252Bkinases%252Binhibitors.%252BMolecular%252Bdesign%25252C%252Bsynthesis%25252C%252Band%252Bbiologic%252Bactivity%26aulast%3DKoroleva%26aufirst%3DE.%2BV.%26date%3D2016%26date%3D2016%26volume%3D52%26issue%3D2%26spage%3D139%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonathan M  Elkins</span>, <span class="hlFld-ContribAuthor ">Vita  Fedele</span>, <span class="hlFld-ContribAuthor ">Marta  Szklarz</span>, <span class="hlFld-ContribAuthor ">Kamal R  Abdul Azeez</span>, <span class="hlFld-ContribAuthor ">Eidarus  Salah</span>, <span class="hlFld-ContribAuthor ">Jowita  Mikolajczyk</span>, <span class="hlFld-ContribAuthor ">Sergei  Romanov</span>, <span class="hlFld-ContribAuthor ">Nikolai  Sepetov</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Bryan L  Roth</span>, <span class="hlFld-ContribAuthor ">Ayman  Al Haj Zen</span>, <span class="hlFld-ContribAuthor ">Denis  Fourches</span>, <span class="hlFld-ContribAuthor ">Eugene  Muratov</span>, <span class="hlFld-ContribAuthor ">Alex  Tropsha</span>, <span class="hlFld-ContribAuthor ">Joel  Morris</span>, <span class="hlFld-ContribAuthor ">Beverly A  Teicher</span>, <span class="hlFld-ContribAuthor ">Mark  Kunkel</span>, <span class="hlFld-ContribAuthor ">Eric  Polley</span>, <span class="hlFld-ContribAuthor ">Karen E  Lackey</span>, <span class="hlFld-ContribAuthor ">Francis L  Atkinson</span>, <span class="hlFld-ContribAuthor ">John P  Overington</span>, <span class="hlFld-ContribAuthor ">Paul  Bamborough</span>, <span class="hlFld-ContribAuthor ">Susanne  MÃ¼ller</span>, <span class="hlFld-ContribAuthor ">Daniel J  Price</span>, <span class="hlFld-ContribAuthor ">Timothy M  Willson</span>, <span class="hlFld-ContribAuthor ">David H  Drewry</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">William J  Zuercher</span>. </span><span class="cited-content_cbyCitation_article-title">Comprehensive characterization of the Published Kinase Inhibitor Set. </span><span class="cited-content_cbyCitation_journal-name">Nature Biotechnology</span><span> <strong>2016,</strong> <em>34 </em>
                                    (1)
                                     , 95-103. <a href="https://doi.org/10.1038/nbt.3374" title="DOI URL">https://doi.org/10.1038/nbt.3374</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nbt.3374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnbt.3374%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Biotechnology%26atitle%3DComprehensive%252Bcharacterization%252Bof%252Bthe%252BPublished%252BKinase%252BInhibitor%252BSet%26aulast%3DElkins%26aufirst%3DJonathan%2BM%26date%3D2016%26date%3D2016%26volume%3D34%26issue%3D1%26spage%3D95%26epage%3D103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olivier  Defert</span>, <span class="hlFld-ContribAuthor ">Sandro  Boland</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase profiling in early stage drug discovery: sorting things out. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2015,</strong> <em>18 </em>, 52-61. <a href="https://doi.org/10.1016/j.ddtec.2015.10.002" title="DOI URL">https://doi.org/10.1016/j.ddtec.2015.10.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2015.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2015.10.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DKinase%252Bprofiling%252Bin%252Bearly%252Bstage%252Bdrug%252Bdiscovery%25253A%252Bsorting%252Bthings%252Bout%26aulast%3DDefert%26aufirst%3DOlivier%26date%3D2015%26volume%3D18%26spage%3D52%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Osman A.  Gani</span>, <span class="hlFld-ContribAuthor ">Balmukund  Thakkar</span>, <span class="hlFld-ContribAuthor ">Dilip  Narayanan</span>, <span class="hlFld-ContribAuthor ">Kazi A.  Alam</span>, <span class="hlFld-ContribAuthor ">Peter  Kyomuhendo</span>, <span class="hlFld-ContribAuthor ">Ulli  Rothweiler</span>, <span class="hlFld-ContribAuthor ">Veronica  Tello-Franco</span>, <span class="hlFld-ContribAuthor ">Richard A.  Engh</span>. </span><span class="cited-content_cbyCitation_article-title">Assessing protein kinase target similarity: Comparing sequence, structure, and cheminformatics approaches. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics</span><span> <strong>2015,</strong> <em>1854 </em>
                                    (10)
                                     , 1605-1616. <a href="https://doi.org/10.1016/j.bbapap.2015.05.004" title="DOI URL">https://doi.org/10.1016/j.bbapap.2015.05.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbapap.2015.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbapap.2015.05.004%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Proteins%2520and%2520Proteomics%26atitle%3DAssessing%252Bprotein%252Bkinase%252Btarget%252Bsimilarity%25253A%252BComparing%252Bsequence%25252C%252Bstructure%25252C%252Band%252Bcheminformatics%252Bapproaches%26aulast%3DGani%26aufirst%3DOsman%2BA.%26date%3D2015%26volume%3D1854%26issue%3D10%26spage%3D1605%26epage%3D1616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Said A.  Said</span>, <span class="hlFld-ContribAuthor ">Hassan A.  El-Sayed</span>, <span class="hlFld-ContribAuthor ">Abd El-Galil E.  Amr</span>, <span class="hlFld-ContribAuthor ">Mohamed M.  Abdalla</span>. </span><span class="cited-content_cbyCitation_article-title">Selective and Orally Bioavailable CHK1 Inhibitors of Some Synthesized Substituted Thieno[2,3-b]pyridine Candidates. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Pharmacology</span><span> <strong>2015,</strong> <em>11 </em>
                                    (7)
                                     , 659-671. <a href="https://doi.org/10.3923/ijp.2015.659.671" title="DOI URL">https://doi.org/10.3923/ijp.2015.659.671</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3923/ijp.2015.659.671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3923%2Fijp.2015.659.671%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Pharmacology%26atitle%3DSelective%252Band%252BOrally%252BBioavailable%252BCHK1%252BInhibitors%252Bof%252BSome%252BSynthesized%252BSubstituted%252BThieno%25255B2%25252C3-b%25255Dpyridine%252BCandidates%26aulast%3DSaid%26aufirst%3DSaid%2BA.%26date%3D2015%26volume%3D11%26issue%3D7%26spage%3D659%26epage%3D671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edgar  Jacoby</span>, <span class="hlFld-ContribAuthor ">Gary  Tresadern</span>, <span class="hlFld-ContribAuthor ">Scott  Bembenek</span>, <span class="hlFld-ContribAuthor ">Berthold  Wroblowski</span>, <span class="hlFld-ContribAuthor ">Christophe  Buyck</span>, <span class="hlFld-ContribAuthor ">Jean-Marc  Neefs</span>, <span class="hlFld-ContribAuthor ">Dmitrii  Rassokhin</span>, <span class="hlFld-ContribAuthor ">Alain  Poncelet</span>, <span class="hlFld-ContribAuthor ">Jeremy  Hunt</span>, <span class="hlFld-ContribAuthor ">Herman  van Vlijmen</span>. </span><span class="cited-content_cbyCitation_article-title">Extending kinome coverage by analysis of kinase inhibitor broad profiling data. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2015,</strong> <em>20 </em>
                                    (6)
                                     , 652-658. <a href="https://doi.org/10.1016/j.drudis.2015.01.002" title="DOI URL">https://doi.org/10.1016/j.drudis.2015.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2015.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2015.01.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DExtending%252Bkinome%252Bcoverage%252Bby%252Banalysis%252Bof%252Bkinase%252Binhibitor%252Bbroad%252Bprofiling%252Bdata%26aulast%3DJacoby%26aufirst%3DEdgar%26date%3D2015%26volume%3D20%26issue%3D6%26spage%3D652%26epage%3D658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isao  Nakanishi</span>, <span class="hlFld-ContribAuthor ">Katsumi  Murata</span>, <span class="hlFld-ContribAuthor ">Naoya  Nagata</span>, <span class="hlFld-ContribAuthor ">Masakuni  Kurono</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Kinoshita</span>, <span class="hlFld-ContribAuthor ">Misato  Yasue</span>, <span class="hlFld-ContribAuthor ">Takako  Miyazaki</span>, <span class="hlFld-ContribAuthor ">Yoshinori  Takei</span>, <span class="hlFld-ContribAuthor ">Shinya  Nakamura</span>, <span class="hlFld-ContribAuthor ">Atsushi  Sakurai</span>, <span class="hlFld-ContribAuthor ">Nobuko  Iwamoto</span>, <span class="hlFld-ContribAuthor ">Keiji  Nishiwaki</span>, <span class="hlFld-ContribAuthor ">Tetsuko  Nakaniwa</span>, <span class="hlFld-ContribAuthor ">Yusuke  Sekiguchi</span>, <span class="hlFld-ContribAuthor ">Akira  Hirasawa</span>, <span class="hlFld-ContribAuthor ">Gozoh  Tsujimoto</span>, <span class="hlFld-ContribAuthor ">Kazuo  Kitaura</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of protein kinase CK2 inhibitors using solvent dipole ordering virtual screening. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>96 </em>, 396-404. <a href="https://doi.org/10.1016/j.ejmech.2015.04.032" title="DOI URL">https://doi.org/10.1016/j.ejmech.2015.04.032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2015.04.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2015.04.032%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Bprotein%252Bkinase%252BCK2%252Binhibitors%252Busing%252Bsolvent%252Bdipole%252Bordering%252Bvirtual%252Bscreening%26aulast%3DNakanishi%26aufirst%3DIsao%26date%3D2015%26volume%3D96%26spage%3D396%26epage%3D404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John S.  Tokarski</span>, <span class="hlFld-ContribAuthor ">Adriana  Zupa-Fernandez</span>, <span class="hlFld-ContribAuthor ">Jeffrey A.  Tredup</span>, <span class="hlFld-ContribAuthor ">Kristen  Pike</span>, <span class="hlFld-ContribAuthor ">ChiehYing  Chang</span>, <span class="hlFld-ContribAuthor ">Dianlin  Xie</span>, <span class="hlFld-ContribAuthor ">Lihong  Cheng</span>, <span class="hlFld-ContribAuthor ">Donna  Pedicord</span>, <span class="hlFld-ContribAuthor ">Jodi  Muckelbauer</span>, <span class="hlFld-ContribAuthor ">Stephen R.  Johnson</span>, <span class="hlFld-ContribAuthor ">Sophie  Wu</span>, <span class="hlFld-ContribAuthor ">Suzanne C.  Edavettal</span>, <span class="hlFld-ContribAuthor ">Yang  Hong</span>, <span class="hlFld-ContribAuthor ">Mark R.  Witmer</span>, <span class="hlFld-ContribAuthor ">Lisa L.  Elkin</span>, <span class="hlFld-ContribAuthor ">Yuval  Blat</span>, <span class="hlFld-ContribAuthor ">William J.  Pitts</span>, <span class="hlFld-ContribAuthor ">David S.  Weinstein</span>, <span class="hlFld-ContribAuthor ">James R.  Burke</span>. </span><span class="cited-content_cbyCitation_article-title">Tyrosine Kinase 2-mediated Signal Transduction in T Lymphocytes Is Blocked by Pharmacological Stabilization of Its Pseudokinase Domain. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2015,</strong> <em>290 </em>
                                    (17)
                                     , 11061-11074. <a href="https://doi.org/10.1074/jbc.M114.619502" title="DOI URL">https://doi.org/10.1074/jbc.M114.619502</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.M114.619502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.M114.619502%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DTyrosine%252BKinase%252B2-mediated%252BSignal%252BTransduction%252Bin%252BT%252BLymphocytes%252BIs%252BBlocked%252Bby%252BPharmacological%252BStabilization%252Bof%252BIts%252BPseudokinase%252BDomain%26aulast%3DTokarski%26aufirst%3DJohn%2BS.%26date%3D2015%26volume%3D290%26issue%3D17%26spage%3D11061%26epage%3D11074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Taha M. A.  Eldebss</span>, <span class="hlFld-ContribAuthor ">Sobhi M.  Gomha</span>, <span class="hlFld-ContribAuthor ">Mohamed M.  Abdulla</span>, <span class="hlFld-ContribAuthor ">Reem K.  Arafa</span>. </span><span class="cited-content_cbyCitation_article-title">Novel pyrrole derivatives as selective CHK1 inhibitors: design, regioselective synthesis and molecular modeling. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2015,</strong> <em>6 </em>
                                    (5)
                                     , 852-859. <a href="https://doi.org/10.1039/C4MD00560K" title="DOI URL">https://doi.org/10.1039/C4MD00560K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00560K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00560K%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DNovel%252Bpyrrole%252Bderivatives%252Bas%252Bselective%252BCHK1%252Binhibitors%25253A%252Bdesign%25252C%252Bregioselective%252Bsynthesis%252Band%252Bmolecular%252Bmodeling%26aulast%3DEldebss%26aufirst%3DTaha%2BM.%2BA.%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D5%26spage%3D852%26epage%3D859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabrizio  FerrÃÂ¨</span>, <span class="hlFld-ContribAuthor ">Antonio  Palmeri</span>, <span class="hlFld-ContribAuthor ">Manuela  Helmer-Citterich</span>. </span><span class="cited-content_cbyCitation_article-title">Computational methods for analysis and inference of kinase/inhibitor relationships. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Genetics</span><span> <strong>2014,</strong> <em>5 </em><a href="https://doi.org/10.3389/fgene.2014.00196" title="DOI URL">https://doi.org/10.3389/fgene.2014.00196</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fgene.2014.00196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffgene.2014.00196%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Genetics%26atitle%3DComputational%252Bmethods%252Bfor%252Banalysis%252Band%252Binference%252Bof%252Bkinase%25252Finhibitor%252Brelationships%26aulast%3DFerr%25C3%2583%25C2%25A8%26aufirst%3DFabrizio%26date%3D2014%26date%3D2014%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amalie Frederikke  Rudolf</span>, <span class="hlFld-ContribAuthor ">Tine  Skovgaard</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Lars Juhl  Jensen</span>, <span class="hlFld-ContribAuthor ">Jens  Berthelsen</span>, . </span><span class="cited-content_cbyCitation_article-title">A Comparison of Protein Kinases Inhibitor Screening Methods Using Both Enzymatic Activity and Binding Affinity Determination. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2014,</strong> <em>9 </em>
                                    (6)
                                     , e98800. <a href="https://doi.org/10.1371/journal.pone.0098800" title="DOI URL">https://doi.org/10.1371/journal.pone.0098800</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0098800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0098800%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DA%252BComparison%252Bof%252BProtein%252BKinases%252BInhibitor%252BScreening%252BMethods%252BUsing%252BBoth%252BEnzymatic%252BActivity%252Band%252BBinding%252BAffinity%252BDetermination%26aulast%3DRudolf%26aufirst%3DAmalie%2BFrederikke%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D6%26spage%3De98800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shanthi  Kanthala</span>, <span class="hlFld-ContribAuthor ">Ted  Gauthier</span>, <span class="hlFld-ContribAuthor ">Seetharama  Satyanarayanajois</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-activity relationships of peptidomimetics that inhibit PPI of HER2-HER3. </span><span class="cited-content_cbyCitation_journal-name">Biopolymers</span><span> <strong>2014,</strong> <em>101 </em>
                                    (6)
                                     , 693-702. <a href="https://doi.org/10.1002/bip.22441" title="DOI URL">https://doi.org/10.1002/bip.22441</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/bip.22441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbip.22441%26sid%3Dliteratum%253Aachs%26jtitle%3DBiopolymers%26atitle%3DStructure-activity%252Brelationships%252Bof%252Bpeptidomimetics%252Bthat%252Binhibit%252BPPI%252Bof%252BHER2-HER3%26aulast%3DKanthala%26aufirst%3DShanthi%26date%3D2014%26date%3D2014%26volume%3D101%26issue%3D6%26spage%3D693%26epage%3D702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Osman A. B. S. M.  Gani</span>, <span class="hlFld-ContribAuthor ">Dilip  Narayanan</span>, <span class="hlFld-ContribAuthor ">Richard A.  Engh</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluating the Predictivity of Virtual Screening for
              A
              bl Kinase Inhibitors to Hinder Drug Resistance. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2013,</strong> <em>82 </em>
                                    (5)
                                     , 506-519. <a href="https://doi.org/10.1111/cbdd.12170" title="DOI URL">https://doi.org/10.1111/cbdd.12170</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.12170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.12170%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DEvaluating%252Bthe%252BPredictivity%252Bof%252BVirtual%252BScreening%252Bfor%252BA%252Bbl%252BKinase%252BInhibitors%252Bto%252BHinder%252BDrug%252BResistance%26aulast%3DGani%26aufirst%3DOsman%2BA.%2BB.%2BS.%2BM.%26date%3D2013%26date%3D2013%26volume%3D82%26issue%3D5%26spage%3D506%26epage%3D519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James R.  Brown</span>, <span class="hlFld-ContribAuthor ">David  Drewry</span>, <span class="hlFld-ContribAuthor ">Francisco-Javier  Gamo</span>, <span class="hlFld-ContribAuthor ">Jose F.  Garcia-Bustos</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase Inhibitors Among Hits from Malaria Cellular Screens. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2013,</strong>,, 261-291. <a href="https://doi.org/10.1002/9783527675401.ch13" title="DOI URL">https://doi.org/10.1002/9783527675401.ch13</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527675401.ch13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527675401.ch13%26sid%3Dliteratum%253Aachs%26atitle%3DKinase%252BInhibitors%252BAmong%252BHits%252Bfrom%252BMalaria%252BCellular%252BScreens%26aulast%3DBrown%26aufirst%3DJames%2BR.%26date%3D2013%26date%3D2013%26spage%3D261%26epage%3D291%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DProtein%252BPhosphorylation%252Bin%252BParasites%26aulast%3DDoerig%26aufirst%3DChristian%26date%3D2013%26volume%3D465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Charlotte E.  Allen</span>, <span class="hlFld-ContribAuthor ">Chiau L.  Chow</span>, <span class="hlFld-ContribAuthor ">John J.  Caldwell</span>, <span class="hlFld-ContribAuthor ">Isaac M.  Westwood</span>, <span class="hlFld-ContribAuthor ">Rob L.  M. van Montfort</span>, <span class="hlFld-ContribAuthor ">Ian  Collins</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and evaluation of heteroaryl substituted diazaspirocycles as scaffolds to probe the ATP-binding site of protein kinases. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2013,</strong> <em>21 </em>
                                    (18)
                                     , 5707-5724. <a href="https://doi.org/10.1016/j.bmc.2013.07.021" title="DOI URL">https://doi.org/10.1016/j.bmc.2013.07.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2013.07.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2013.07.021%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bevaluation%252Bof%252Bheteroaryl%252Bsubstituted%252Bdiazaspirocycles%252Bas%252Bscaffolds%252Bto%252Bprobe%252Bthe%252BATP-binding%252Bsite%252Bof%252Bprotein%252Bkinases%26aulast%3DAllen%26aufirst%3DCharlotte%2BE.%26date%3D2013%26volume%3D21%26issue%3D18%26spage%3D5707%26epage%3D5724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrei  LeitÃ£o</span>, <span class="hlFld-ContribAuthor ">Alexander  Schramm</span>, <span class="hlFld-ContribAuthor ">Angelika  Eggert</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a New Bioactive Molecule for Neuroblastoma. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2013,</strong> <em>82 </em>
                                    (2)
                                     , 233-241. <a href="https://doi.org/10.1111/cbdd.12148" title="DOI URL">https://doi.org/10.1111/cbdd.12148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.12148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.12148%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DDiscovery%252Bof%252Ba%252BNew%252BBioactive%252BMolecule%252Bfor%252BNeuroblastoma%26aulast%3DLeit%25C3%25A3o%26aufirst%3DAndrei%26date%3D2013%26date%3D2013%26volume%3D82%26issue%3D2%26spage%3D233%26epage%3D241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mikko  Taipale</span>, <span class="hlFld-ContribAuthor ">Irina  Krykbaeva</span>, <span class="hlFld-ContribAuthor ">Luke  Whitesell</span>, <span class="hlFld-ContribAuthor ">Sandro  Santagata</span>, <span class="hlFld-ContribAuthor ">Jianming  Zhang</span>, <span class="hlFld-ContribAuthor ">Qingsong  Liu</span>, <span class="hlFld-ContribAuthor ">Nathanael S  Gray</span>, <span class="hlFld-ContribAuthor ">Susan  Lindquist</span>. </span><span class="cited-content_cbyCitation_article-title">Chaperones as thermodynamic sensors of drug-target interactions reveal kinase inhibitor specificities in living cells. </span><span class="cited-content_cbyCitation_journal-name">Nature Biotechnology</span><span> <strong>2013,</strong> <em>31 </em>
                                    (7)
                                     , 630-637. <a href="https://doi.org/10.1038/nbt.2620" title="DOI URL">https://doi.org/10.1038/nbt.2620</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nbt.2620&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnbt.2620%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Biotechnology%26atitle%3DChaperones%252Bas%252Bthermodynamic%252Bsensors%252Bof%252Bdrug-target%252Binteractions%252Breveal%252Bkinase%252Binhibitor%252Bspecificities%252Bin%252Bliving%252Bcells%26aulast%3DTaipale%26aufirst%3DMikko%26date%3D2013%26date%3D2013%26volume%3D31%26issue%3D7%26spage%3D630%26epage%3D637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iva  DoleÄkovÃ¡</span>, <span class="hlFld-ContribAuthor ">Michal  Äesnek</span>, <span class="hlFld-ContribAuthor ">Martin  DraÄinskÃ½</span>, <span class="hlFld-ContribAuthor ">JiÅÃ­  Brynda</span>, <span class="hlFld-ContribAuthor ">JiÅÃ­  Voller</span>, <span class="hlFld-ContribAuthor ">Zlatko  Janeba</span>, <span class="hlFld-ContribAuthor ">VladimÃ­r  KryÅ¡tof</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological evaluation of guanidino analogues of roscovitine. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>62 </em>, 443-452. <a href="https://doi.org/10.1016/j.ejmech.2013.01.021" title="DOI URL">https://doi.org/10.1016/j.ejmech.2013.01.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2013.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2013.01.021%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bguanidino%252Banalogues%252Bof%252Broscovitine%26aulast%3DDole%25C4%258Dkov%25C3%25A1%26aufirst%3DIva%26date%3D2013%26volume%3D62%26spage%3D443%26epage%3D452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul  Bamborough</span>. </span><span class="cited-content_cbyCitation_article-title">System-based drug discovery within the human kinome. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2012,</strong> <em>7 </em>
                                    (11)
                                     , 1053-1070. <a href="https://doi.org/10.1517/17460441.2012.724056" title="DOI URL">https://doi.org/10.1517/17460441.2012.724056</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2012.724056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2012.724056%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DSystem-based%252Bdrug%252Bdiscovery%252Bwithin%252Bthe%252Bhuman%252Bkinome%26aulast%3DBamborough%26aufirst%3DPaul%26date%3D2012%26date%3D2012%26volume%3D7%26issue%3D11%26spage%3D1053%26epage%3D1070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steve  Wenglowsky</span>, <span class="hlFld-ContribAuthor ">David  Moreno</span>, <span class="hlFld-ContribAuthor ">Ellen R.  Laird</span>, <span class="hlFld-ContribAuthor ">Susan L.  Gloor</span>, <span class="hlFld-ContribAuthor ">Li  Ren</span>, <span class="hlFld-ContribAuthor ">Tyler  Risom</span>, <span class="hlFld-ContribAuthor ">Joachim  Rudolph</span>, <span class="hlFld-ContribAuthor ">Hillary L.  Sturgis</span>, <span class="hlFld-ContribAuthor ">Walter C.  Voegtli</span>. </span><span class="cited-content_cbyCitation_article-title">Pyrazolopyridine inhibitors of B-RafV600E. Part 4: Rational design and kinase selectivity profile of cell potent type II inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2012,</strong> <em>22 </em>
                                    (19)
                                     , 6237-6241. <a href="https://doi.org/10.1016/j.bmcl.2012.08.007" title="DOI URL">https://doi.org/10.1016/j.bmcl.2012.08.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2012.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2012.08.007%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPyrazolopyridine%252Binhibitors%252Bof%252BB-RafV600E.%252BPart%252B4%25253A%252BRational%252Bdesign%252Band%252Bkinase%252Bselectivity%252Bprofile%252Bof%252Bcell%252Bpotent%252Btype%252BII%252Binhibitors%26aulast%3DWenglowsky%26aufirst%3DSteve%26date%3D2012%26volume%3D22%26issue%3D19%26spage%3D6237%26epage%3D6241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joost CM  Uitdehaag</span>, <span class="hlFld-ContribAuthor ">Folkert  Verkaar</span>, <span class="hlFld-ContribAuthor ">Husam  Alwan</span>, <span class="hlFld-ContribAuthor ">Jos  de Man</span>, <span class="hlFld-ContribAuthor ">Rogier C  Buijsman</span>, <span class="hlFld-ContribAuthor ">Guido JR  Zaman</span>. </span><span class="cited-content_cbyCitation_article-title">A guide to picking the most selective kinase inhibitor tool compounds for pharmacological validation of drug targets. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Pharmacology</span><span> <strong>2012,</strong> <em>166 </em>
                                    (3)
                                     , 858-876. <a href="https://doi.org/10.1111/j.1476-5381.2012.01859.x" title="DOI URL">https://doi.org/10.1111/j.1476-5381.2012.01859.x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/j.1476-5381.2012.01859.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fj.1476-5381.2012.01859.x%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Pharmacology%26atitle%3DA%252Bguide%252Bto%252Bpicking%252Bthe%252Bmost%252Bselective%252Bkinase%252Binhibitor%252Btool%252Bcompounds%252Bfor%252Bpharmacological%252Bvalidation%252Bof%252Bdrug%252Btargets%26aulast%3DUitdehaag%26aufirst%3DJoost%2BCM%26date%3D2012%26date%3D2012%26volume%3D166%26issue%3D3%26spage%3D858%26epage%3D876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Igor A.  Schepetkin</span>, <span class="hlFld-ContribAuthor ">Liliya N.  Kirpotina</span>, <span class="hlFld-ContribAuthor ">Andrei I.  Khlebnikov</span>, <span class="hlFld-ContribAuthor ">Tracey S.  Hanks</span>, <span class="hlFld-ContribAuthor ">Irina  Kochetkova</span>, <span class="hlFld-ContribAuthor ">David W.  Pascual</span>, <span class="hlFld-ContribAuthor ">Mark A.  Jutila</span>, <span class="hlFld-ContribAuthor ">Mark T.  Quinn</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Characterization of a Novel Class of c-Jun N-terminal Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmacology</span><span> <strong>2012,</strong> <em>81 </em>
                                    (6)
                                     , 832-845. <a href="https://doi.org/10.1124/mol.111.077446" title="DOI URL">https://doi.org/10.1124/mol.111.077446</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/mol.111.077446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fmol.111.077446%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmacology%26atitle%3DIdentification%252Band%252BCharacterization%252Bof%252Ba%252BNovel%252BClass%252Bof%252Bc-Jun%252BN-terminal%252BKinase%252BInhibitors%26aulast%3DSchepetkin%26aufirst%3DIgor%2BA.%26date%3D2012%26date%3D2012%26volume%3D81%26issue%3D6%26spage%3D832%26epage%3D845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Federica  Amigoni</span>, <span class="hlFld-ContribAuthor ">Elena  Legnaghi</span>, <span class="hlFld-ContribAuthor ">Paolo  Pevarello</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase inhibitors for CNS diseases: an analysis of the recent patent literature. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutical Patent Analyst</span><span> <strong>2012,</strong> <em>1 </em>
                                    (2)
                                     , 177-192. <a href="https://doi.org/10.4155/ppa.12.19" title="DOI URL">https://doi.org/10.4155/ppa.12.19</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/ppa.12.19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Fppa.12.19%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutical%2520Patent%2520Analyst%26atitle%3DKinase%252Binhibitors%252Bfor%252BCNS%252Bdiseases%25253A%252Ban%252Banalysis%252Bof%252Bthe%252Brecent%252Bpatent%252Bliterature%26aulast%3DAmigoni%26aufirst%3DFederica%26date%3D2012%26volume%3D1%26issue%3D2%26spage%3D177%26epage%3D192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Philip B.  Cox</span>, <span class="hlFld-ContribAuthor ">Anil  Vasudevan</span>. </span><span class="cited-content_cbyCitation_article-title">Generating a High-Quality Compound Collection. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2012,</strong>,, 9-21. <a href="https://doi.org/10.1002/9783527645251.ch2" title="DOI URL">https://doi.org/10.1002/9783527645251.ch2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527645251.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527645251.ch2%26sid%3Dliteratum%253Aachs%26atitle%3DGenerating%252Ba%252BHigh-Quality%252BCompound%252BCollection%26aulast%3DCox%26aufirst%3DPhilip%2BB.%26date%3D2012%26date%3D2012%26spage%3D9%26epage%3D21%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DManagement%252Bof%252BChemical%252Band%252BBiological%252BSamples%252Bfor%252BScreening%252BApplications%26aulast%3DWigglesworth%26aufirst%3DMark%26date%3D2012%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eduard R.  Felder</span>, <span class="hlFld-ContribAuthor ">Alessandra  Badari</span>, <span class="hlFld-ContribAuthor ">Teresa  Disingrini</span>, <span class="hlFld-ContribAuthor ">Sergio  Mantegani</span>, <span class="hlFld-ContribAuthor ">Christian  Orrenius</span>, <span class="hlFld-ContribAuthor ">Nilla  Avanzi</span>, <span class="hlFld-ContribAuthor ">Antonella  Isacchi</span>, <span class="hlFld-ContribAuthor ">Barbara  Salom</span>. </span><span class="cited-content_cbyCitation_article-title">The generation of purinome-targeted libraries as a means to diversify ATP-mimetic chemical classes for lead finding. </span><span class="cited-content_cbyCitation_journal-name">Molecular Diversity</span><span> <strong>2012,</strong> <em>16 </em>
                                    (1)
                                     , 27-51. <a href="https://doi.org/10.1007/s11030-012-9361-6" title="DOI URL">https://doi.org/10.1007/s11030-012-9361-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11030-012-9361-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11030-012-9361-6%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Diversity%26atitle%3DThe%252Bgeneration%252Bof%252Bpurinome-targeted%252Blibraries%252Bas%252Ba%252Bmeans%252Bto%252Bdiversify%252BATP-mimetic%252Bchemical%252Bclasses%252Bfor%252Blead%252Bfinding%26aulast%3DFelder%26aufirst%3DEduard%2BR.%26date%3D2012%26date%3D2012%26volume%3D16%26issue%3D1%26spage%3D27%26epage%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mindy I  Davis</span>, <span class="hlFld-ContribAuthor ">Jeremy P  Hunt</span>, <span class="hlFld-ContribAuthor ">Sanna  Herrgard</span>, <span class="hlFld-ContribAuthor ">Pietro  Ciceri</span>, <span class="hlFld-ContribAuthor ">Lisa M  Wodicka</span>, <span class="hlFld-ContribAuthor ">Gabriel  Pallares</span>, <span class="hlFld-ContribAuthor ">Michael  Hocker</span>, <span class="hlFld-ContribAuthor ">Daniel K  Treiber</span>, <span class="hlFld-ContribAuthor ">Patrick P  Zarrinkar</span>. </span><span class="cited-content_cbyCitation_article-title">Comprehensive analysis of kinase inhibitor selectivity. </span><span class="cited-content_cbyCitation_journal-name">Nature Biotechnology</span><span> <strong>2011,</strong> <em>29 </em>
                                    (11)
                                     , 1046-1051. <a href="https://doi.org/10.1038/nbt.1990" title="DOI URL">https://doi.org/10.1038/nbt.1990</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnbt.1990%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Biotechnology%26atitle%3DComprehensive%252Banalysis%252Bof%252Bkinase%252Binhibitor%252Bselectivity%26aulast%3DDavis%26aufirst%3DMindy%2BI%26date%3D2011%26date%3D2011%26volume%3D29%26issue%3D11%26spage%3D1046%26epage%3D1051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">ChandrasekharÂ V.  Miduturu</span>, <span class="hlFld-ContribAuthor ">Xianming  Deng</span>, <span class="hlFld-ContribAuthor ">Nicholas  Kwiatkowski</span>, <span class="hlFld-ContribAuthor ">Wannian  Yang</span>, <span class="hlFld-ContribAuthor ">Laurent  Brault</span>, <span class="hlFld-ContribAuthor ">Panagis  Filippakopoulos</span>, <span class="hlFld-ContribAuthor ">Eunah  Chung</span>, <span class="hlFld-ContribAuthor ">Qingkai  Yang</span>, <span class="hlFld-ContribAuthor ">Juerg  Schwaller</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">RandallÂ W.  King</span>, <span class="hlFld-ContribAuthor ">Jiing-Dwan  Lee</span>, <span class="hlFld-ContribAuthor ">Sanna  Herrgard</span>, <span class="hlFld-ContribAuthor ">Patrick  Zarrinkar</span>, <span class="hlFld-ContribAuthor ">NathanaelÂ S.  Gray</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Kinase Profiling: A More Efficient Approach toward the Discovery of New Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chemistry & Biology</span><span> <strong>2011,</strong> <em>18 </em>
                                    (7)
                                     , 868-879. <a href="https://doi.org/10.1016/j.chembiol.2011.05.010" title="DOI URL">https://doi.org/10.1016/j.chembiol.2011.05.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2011.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2011.05.010%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%2526%2520Biology%26atitle%3DHigh-Throughput%252BKinase%252BProfiling%25253A%252BA%252BMore%252BEfficient%252BApproach%252Btoward%252Bthe%252BDiscovery%252Bof%252BNew%252BKinase%252BInhibitors%26aulast%3DMiduturu%26aufirst%3DChandrasekhar%25C2%25A0V.%26date%3D2011%26volume%3D18%26issue%3D7%26spage%3D868%26epage%3D879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Relationship between the Ambit BioSciences binding assays and functional IC<sub>50</sub> determinations performed in house for three representative assays: (A) FLT3, (B) EGFR, and (C) CDK2. Functional IC<sub>50</sub> values are plotted on a log scale. Plots were generated with Excel 2003.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Heat map of binding data for potent, selective compounds. Each row represents a single kinase and each column a single compound. Kinases are grouped by family and compounds by structural similarity. Heat map values range from 0% (red, maximum binding activity) to 100% (blue, minimum binding) of control. (A) Binding data for all potent, selective compounds (the SP set). (B) Detailed view of binding to Tec, Syk, and Src family kinases for two clusters of structurally similar compounds. The heat map was generated with Spotfire DecisionSite version 9.1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. CDF of activity homology. Pairs of kinases were binned by sequence identity. For the kinase pairs in each sequence identity bin, the cumulative distribution function of activity homology is plotted as a separate function. Activity homology for kinases A and B is the fraction of compounds active for kinase A that are also active for kinase B.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Plots of potency vs selectivity. (A) Maximum binding activity vs selectivity index [SI(13)] for potent compounds. More potent compounds have lower values of maximum binding activity, and more selective compounds have lower selectivity indices. The maximum binding values were binned using a bin width of 1.0, and the SI(13) values were plotted as a box plot. For each bin, the top of the corresponding box indicates the 75th percentile of SI(13), the bottom of the box indicates the 25th percentile, and the white line inside the box indicates the mean value. The number of compounds in each bin and the mean SI(13) value are shown in the table below the plot. (B) Minimum IC<sub>50</sub> vs selectivity index [SI(100 nM)] for compounds tested in at least 10 in-house kinase functional assays. The minimum IC<sub>50</sub> values were binned using a bin width of 20 nM, and the SI(100 nM) values for each bin were plotted as a box plot as in panel A. (C) Minimum IC<sub>50</sub> vs log(IC<sub>50</sub> ratio) for compounds tested in at least 10 in-house kinase functional assays. See the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for calculation of the log(IC<sub>50</sub> ratio) values. Plots were generated with TIBCO Spotfire version 2.2. The scatter plots corresponding to panels A and B are included in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures and kinome binding profiles of a representative pair of compounds. The structures of compounds <b>1</b> and <b>2</b> are shown at the left and their binding activities at the right. Binding data are shown for all targets that were bound by compounds <b>1</b> and <b>2</b> (â¤13% control).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structural characteristics of selective and nonselective compounds: (A) number of rings, (B) number of polar hydrogens, and (C) molecular weight. The distributions are shown for the filtered nonselective and SP sets. Plots were prepared with Excel 2003.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Kinase families and chemotype specificity. Thirteen chemotypes with â¥50 representatives in the SP set are represented along the <i>X</i>-axis, and kinase families are listed on the <i>Y</i>-axis. The color of each rectangle indicates the number of compounds in the given chemotype that potently inhibit one or more kinases in the specified family. Plots were prepared with TIBCO Spotfire version 2.2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. CDF of profile similarity (CDF<sub>P</sub>) by structural similarity bin. Pairs of compounds were binned by structural similarity (Tc). For the compound pairs in each structural similarity bin, the cumulative distribution function of profile similarity is plotted as a separate function.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/medium/jm-2010-01195a_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Neighbor hit rates. (A) Distribution of neighbor hit rates over the set of kinase targets. The neighbor hit rate is defined as the percent of structural analogues of a targetâs actives that are themselves active for the same target. (B) For each target, the number of hits is plotted vs the neighbor hit rate. The targets are colored by kinase class.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-1/jm101195a/production/images/large/jm-2010-01195a_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm101195a&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i36">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77276" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77276" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 45 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">â</span> <span class="NLM_lpage">1934</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lgdoVbLZzbsww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Sheinerman, F. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giraud, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laoui, A.</span><span> </span><span class="NLM_article-title">High affinity targets of protein kinase inhibitors have similar residues at the positions energetically important for binding</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">1134</span><span class="NLM_x">â</span> <span class="NLM_lpage">1156</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=1134-1156&author=F.+B.+Sheinermanauthor=E.+Giraudauthor=A.+Laoui&title=High+affinity+targets+of+protein+kinase+inhibitors+have+similar+residues+at+the+positions+energetically+important+for+binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSheinerman%26aufirst%3DF.%2BB.%26aulast%3DGiraud%26aufirst%3DE.%26aulast%3DLaoui%26aufirst%3DA.%26atitle%3DHigh%2520affinity%2520targets%2520of%2520protein%2520kinase%2520inhibitors%2520have%2520similar%2520residues%2520at%2520the%2520positions%2520energetically%2520important%2520for%2520binding%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D352%26spage%3D1134%26epage%3D1156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Barker, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grundy, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godfrey, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlow, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Healy, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodburn, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curry, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarlett, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henthorn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, L.</span><span> </span><span class="NLM_article-title">Studies leading to the identification of ZD1839 (IRESSA): An orally active, selective epidermal growth factor receptor tyrosine kinase inhibitor targeted to the treatment of cancer</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1911</span><span class="NLM_x">â</span> <span class="NLM_lpage">1914</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1016%2FS0960-894X%2801%2900344-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=11459659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFylsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1911-1914&author=A.+J.+Barkerauthor=K.+H.+Gibsonauthor=W.+Grundyauthor=A.+A.+Godfreyauthor=J.+J.+Barlowauthor=M.+P.+Healyauthor=J.+R.+Woodburnauthor=S.+E.+Ashtonauthor=B.+J.+Curryauthor=L.+Scarlettauthor=L.+Henthornauthor=L.+Richards&title=Studies+leading+to+the+identification+of+ZD1839+%28IRESSA%29%3A+An+orally+active%2C+selective+epidermal+growth+factor+receptor+tyrosine+kinase+inhibitor+targeted+to+the+treatment+of+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Studies leading to the identification of ZD1839 (iressa): an orally active, selective epidermal growth factor receptor tyrosine kinase inhibitor targeted to the treatment of cancer</span></div><div class="casAuthors">Barker, A. J.; Gibson, K. H.; Grundy, W.; Godfrey, A. A.; Barlow, J. J.; Healy, M. P.; Woodburn, J. R.; Ashton, S. E.; Curry, B. J.; Scarlett, L.; Henthorn, L.; Richards, L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1911-1914</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">This paper describes the development of the epidermal growth factor receptor tyrosine kinase inhibitor ZD1839 from a lead series of 4-anilinoquinazoline compds.  ZD1839 has suitable properties for use as a clin. effective drug and shows activity against human tumors.  In particular, the use of pharmacokinetic data in the development of ZD1839 is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvsJ1FTqEwvbVg90H21EOLACvtfcHk0lgdoVbLZzbsww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltFylsLo%253D&md5=ebb210453ade246e43de9fd07ae1d581</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2801%2900344-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252801%252900344-4%26sid%3Dliteratum%253Aachs%26aulast%3DBarker%26aufirst%3DA.%2BJ.%26aulast%3DGibson%26aufirst%3DK.%2BH.%26aulast%3DGrundy%26aufirst%3DW.%26aulast%3DGodfrey%26aufirst%3DA.%2BA.%26aulast%3DBarlow%26aufirst%3DJ.%2BJ.%26aulast%3DHealy%26aufirst%3DM.%2BP.%26aulast%3DWoodburn%26aufirst%3DJ.%2BR.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DCurry%26aufirst%3DB.%2BJ.%26aulast%3DScarlett%26aufirst%3DL.%26aulast%3DHenthorn%26aufirst%3DL.%26aulast%3DRichards%26aufirst%3DL.%26atitle%3DStudies%2520leading%2520to%2520the%2520identification%2520of%2520ZD1839%2520%2528IRESSA%2529%253A%2520An%2520orally%2520active%252C%2520selective%2520epidermal%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520inhibitor%2520targeted%2520to%2520the%2520treatment%2520of%2520cancer%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2001%26volume%3D11%26spage%3D1911%26epage%3D1914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Wakeling, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guy, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodburn, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curry, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, K. H.</span><span> </span><span class="NLM_article-title">ZD1839 (Iressa): An orally active inhibitor of epidermal growth factor signaling with potential for cancer therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">5749</span><span class="NLM_x">â</span> <span class="NLM_lpage">5754</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=12384534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD38XotFyktr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=5749-5754&author=A.+E.+Wakelingauthor=S.+P.+Guyauthor=J.+R.+Woodburnauthor=S.+E.+Ashtonauthor=B.+J.+Curryauthor=A.+J.+Barkerauthor=K.+H.+Gibson&title=ZD1839+%28Iressa%29%3A+An+orally+active+inhibitor+of+epidermal+growth+factor+signaling+with+potential+for+cancer+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">ZD1839 (Iressa): An orally active inhibitor of epidermal growth factor signaling with potential for cancer therapy</span></div><div class="casAuthors">Wakeling, Alan E.; Guy, Simon P.; Woodburn, Jim R.; Ashton, Susan E.; Curry, Brenda J.; Barker, Andrew J.; Gibson, Keith H.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5749-5754</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) is a promising target for anticancer therapy because of its role in tumor growth, metastasis and angiogenesis, and tumor resistance to chemotherapy and radiotherapy.  We have developed a low-mol.-wt. EGFR tyrosine kinase inhibitor (EGFR-TKI), ZD1839 (Iressa).  ZD1839, a substituted anilinoquinazoline, is a potent EGFR-TKI (IC50 = 0.033 Î¼M) that selectively inhibits EGF-stimulated tumor cell growth (IC50 = 0.054 Î¼M) and that blocks EGF-stimulated EGFR autophosphorylation in tumor cells.  In studies with mice bearing a range of human tumor-derived xenografts, ZD1839 given p.o. once a day inhibited tumor growth in a dose-dependent manner.  The level of expression of EGFR did not det. xenograft tumor sensitivity to ZD1839.  Long-term ZD1839 (>3 mo) treatment of mice bearing A431 xenografts was well tolerated, and ZD1839 completely inhibited tumor growth and induced regression of established tumors.  No drug-resistant tumors appeared during ZD1839 treatment, but some tumors regrew after drug withdrawal.  These studies indicate the potential utility of ZD1839 in the treatment of many human tumors and indicate that continuous once-a-day p.o. dosing might be a suitable therapeutic regimen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkoHZ9zL06abVg90H21EOLACvtfcHk0li41u8GMxIV_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotFyktr8%253D&md5=f5f86c652786db476a2d23b5bbd1a9e9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWakeling%26aufirst%3DA.%2BE.%26aulast%3DGuy%26aufirst%3DS.%2BP.%26aulast%3DWoodburn%26aufirst%3DJ.%2BR.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DCurry%26aufirst%3DB.%2BJ.%26aulast%3DBarker%26aufirst%3DA.%2BJ.%26aulast%3DGibson%26aufirst%3DK.%2BH.%26atitle%3DZD1839%2520%2528Iressa%2529%253A%2520An%2520orally%2520active%2520inhibitor%2520of%2520epidermal%2520growth%2520factor%2520signaling%2520with%2520potential%2520for%2520cancer%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2002%26volume%3D62%26spage%3D5749%26epage%3D5754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Wilhelm, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNabola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shujath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gawlak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eveleigh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowley, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adnane, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auclair, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gedrich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voznesensky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riedl, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Post, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trail, P. A.</span><span> </span><span class="NLM_article-title">BAY 43-9006 exhibits broad spectrum oral antitumor activity and targets the RAF/MEK/ERK pathway and receptor tyrosine kinases involved in tumor progression and angiogenesis</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">7099</span><span class="NLM_x">â</span> <span class="NLM_lpage">7109</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=7099-7109&author=S.+M.+Wilhelmauthor=C.+Carterauthor=L.+Tangauthor=D.+Wilkieauthor=A.+McNabolaauthor=H.+Rongauthor=C.+Chenauthor=X.+Zhangauthor=P.+Vincentauthor=M.+McHughauthor=Y.+Caoauthor=J.+Shujathauthor=S.+Gawlakauthor=D.+Eveleighauthor=B.+Rowleyauthor=L.+Liuauthor=L.+Adnaneauthor=M.+Lynchauthor=D.+Auclairauthor=I.+Taylorauthor=R.+Gedrichauthor=A.+Voznesenskyauthor=B.+Riedlauthor=L.+E.+Postauthor=G.+Bollagauthor=P.+A.+Trail&title=BAY+43-9006+exhibits+broad+spectrum+oral+antitumor+activity+and+targets+the+RAF%2FMEK%2FERK+pathway+and+receptor+tyrosine+kinases+involved+in+tumor+progression+and+angiogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DS.%2BM.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DWilkie%26aufirst%3DD.%26aulast%3DMcNabola%26aufirst%3DA.%26aulast%3DRong%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DVincent%26aufirst%3DP.%26aulast%3DMcHugh%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DShujath%26aufirst%3DJ.%26aulast%3DGawlak%26aufirst%3DS.%26aulast%3DEveleigh%26aufirst%3DD.%26aulast%3DRowley%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DAdnane%26aufirst%3DL.%26aulast%3DLynch%26aufirst%3DM.%26aulast%3DAuclair%26aufirst%3DD.%26aulast%3DTaylor%26aufirst%3DI.%26aulast%3DGedrich%26aufirst%3DR.%26aulast%3DVoznesensky%26aufirst%3DA.%26aulast%3DRiedl%26aufirst%3DB.%26aulast%3DPost%26aufirst%3DL.%2BE.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DTrail%26aufirst%3DP.%2BA.%26atitle%3DBAY%252043-9006%2520exhibits%2520broad%2520spectrum%2520oral%2520antitumor%2520activity%2520and%2520targets%2520the%2520RAF%252FMEK%252FERK%2520pathway%2520and%2520receptor%2520tyrosine%2520kinases%2520involved%2520in%2520tumor%2520progression%2520and%2520angiogenesis%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D7099%26epage%3D7109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Lombardo, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrish, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Behnia, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castaneda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornelius, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doweyko, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairchild, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inigo, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamath, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marathe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schieven, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tokarski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wityak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borzilleri, R. M.</span><span> </span><span class="NLM_article-title">Discovery of N-(2-chloro-6-methyl- phenyl)-2-(6-(4-(2-hydroxyethyl)- piperazin-1-yl)-2-methylpyrimidin-4- ylamino)thiazole-5-carboxamide (BMS-354825), a dual Src/Abl kinase inhibitor with potent antitumor activity in preclinical assays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">6658</span><span class="NLM_x">â</span> <span class="NLM_lpage">6661</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049486a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=6658-6661&author=L.+J.+Lombardoauthor=F.+Y.+Leeauthor=P.+Chenauthor=D.+Norrisauthor=J.+C.+Barrishauthor=K.+Behniaauthor=S.+Castanedaauthor=L.+A.+Corneliusauthor=J.+Dasauthor=A.+M.+Doweykoauthor=C.+Fairchildauthor=J.+T.+Huntauthor=I.+Inigoauthor=K.+Johnstonauthor=A.+Kamathauthor=D.+Kanauthor=H.+Kleiauthor=P.+Maratheauthor=S.+Pangauthor=R.+Petersonauthor=S.+Pittauthor=G.+L.+Schievenauthor=R.+J.+Schmidtauthor=J.+Tokarskiauthor=M.+L.+Wenauthor=J.+Wityakauthor=R.+M.+Borzilleri&title=Discovery+of+N-%282-chloro-6-methyl-+phenyl%29-2-%286-%284-%282-hydroxyethyl%29-+piperazin-1-yl%29-2-methylpyrimidin-4-+ylamino%29thiazole-5-carboxamide+%28BMS-354825%29%2C+a+dual+Src%2FAbl+kinase+inhibitor+with+potent+antitumor+activity+in+preclinical+assays"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm049486a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049486a%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DL.%2BJ.%26aulast%3DLee%26aufirst%3DF.%2BY.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DBarrish%26aufirst%3DJ.%2BC.%26aulast%3DBehnia%26aufirst%3DK.%26aulast%3DCastaneda%26aufirst%3DS.%26aulast%3DCornelius%26aufirst%3DL.%2BA.%26aulast%3DDas%26aufirst%3DJ.%26aulast%3DDoweyko%26aufirst%3DA.%2BM.%26aulast%3DFairchild%26aufirst%3DC.%26aulast%3DHunt%26aufirst%3DJ.%2BT.%26aulast%3DInigo%26aufirst%3DI.%26aulast%3DJohnston%26aufirst%3DK.%26aulast%3DKamath%26aufirst%3DA.%26aulast%3DKan%26aufirst%3DD.%26aulast%3DKlei%26aufirst%3DH.%26aulast%3DMarathe%26aufirst%3DP.%26aulast%3DPang%26aufirst%3DS.%26aulast%3DPeterson%26aufirst%3DR.%26aulast%3DPitt%26aufirst%3DS.%26aulast%3DSchieven%26aufirst%3DG.%2BL.%26aulast%3DSchmidt%26aufirst%3DR.%2BJ.%26aulast%3DTokarski%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DM.%2BL.%26aulast%3DWityak%26aufirst%3DJ.%26aulast%3DBorzilleri%26aufirst%3DR.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25282-chloro-6-methyl-%2520phenyl%2529-2-%25286-%25284-%25282-hydroxyethyl%2529-%2520piperazin-1-yl%2529-2-methylpyrimidin-4-%2520ylamino%2529thiazole-5-carboxamide%2520%2528BMS-354825%2529%252C%2520a%2520dual%2520Src%252FAbl%2520kinase%2520inhibitor%2520with%2520potent%2520antitumor%2520activity%2520in%2520preclinical%2520assays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D6658%26epage%3D6661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastuck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boesche, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reader, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweetman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouwmeester, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubauer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rick, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span> </span><span class="NLM_article-title">Quantitative chemical proteomics reveals mechanisms of action of clinical ABL kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1035</span><span class="NLM_x">â</span> <span class="NLM_lpage">1044</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1038%2Fnbt1328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=17721511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWksLvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=1035-1044&author=M.+Bantscheffauthor=D.+Eberhardauthor=Y.+Abrahamauthor=S.+Bastuckauthor=M.+Boescheauthor=S.+Hobsonauthor=T.+Mathiesonauthor=J.+Perrinauthor=M.+Raidaauthor=C.+Rauauthor=V.+Readerauthor=G.+Sweetmanauthor=A.+Bauerauthor=T.+Bouwmeesterauthor=C.+Hopfauthor=U.+Kruseauthor=G.+Neubauerauthor=N.+Ramsdenauthor=J.+Rickauthor=B.+Kusterauthor=G.+Drewes&title=Quantitative+chemical+proteomics+reveals+mechanisms+of+action+of+clinical+ABL+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative chemical proteomics reveals mechanisms of action of clinical ABL kinase inhibitors</span></div><div class="casAuthors">Bantscheff, Marcus; Eberhard, Dirk; Abraham, Yann; Bastuck, Sonja; Boesche, Markus; Hobson, Scott; Mathieson, Toby; Perrin, Jessica; Raida, Manfred; Rau, Christina; Reader, Valerie; Sweetman, Gavain; Bauer, Andreas; Bouwmeester, Tewis; Hopf, Carsten; Kruse, Ulrich; Neubauer, Gitte; Ramsden, Nigel; Rick, Jens; Kuster, Bernhard; Drewes, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1035-1044</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors describe a chem. proteomics approach to profile the interaction of small mols. with hundreds of endogenously expressed protein kinases and purine-binding proteins.  This subproteome is captured by immobilized nonselective kinase inhibitors (kinobeads), and the bound proteins are quantified in parallel by mass spectrometry using isobaric tags for relative and abs. quantification (iTRAQ).  By measuring the competition with the affinity matrix, the authors assess the binding of drugs to their targets in cell lysates and in cells.  By mapping drug-induced changes in the phosphorylation state of the captured proteome, the authors also analyze signaling pathways downstream of target kinases.  Quant. profiling of the drugs imatinib (Gleevec), dasatinib (Sprycel) and bosutinib in K562 cells confirms known targets including ABL and SRC family kinases and identifies the receptor tyrosine kinase DDR1 and the oxidoreductase NQO2 as novel targets of imatinib.  The data suggest that the approach is a valuable tool for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsC3OpIuhl8LVg90H21EOLACvtfcHk0li41u8GMxIV_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWksLvF&md5=ed20d4684a8a6733aba24ca64602bae8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnbt1328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1328%26sid%3Dliteratum%253Aachs%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DAbraham%26aufirst%3DY.%26aulast%3DBastuck%26aufirst%3DS.%26aulast%3DBoesche%26aufirst%3DM.%26aulast%3DHobson%26aufirst%3DS.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DPerrin%26aufirst%3DJ.%26aulast%3DRaida%26aufirst%3DM.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DReader%26aufirst%3DV.%26aulast%3DSweetman%26aufirst%3DG.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DBouwmeester%26aufirst%3DT.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DKruse%26aufirst%3DU.%26aulast%3DNeubauer%26aufirst%3DG.%26aulast%3DRamsden%26aufirst%3DN.%26aulast%3DRick%26aufirst%3DJ.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DQuantitative%2520chemical%2520proteomics%2520reveals%2520mechanisms%2520of%2520action%2520of%2520clinical%2520ABL%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D1035%26epage%3D1044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Huang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reeves, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairchild, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platero, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, E.</span><span> </span><span class="NLM_article-title">Identification of candidate molecular markers predicting sensitivity in solid tumors to dasatinib: Rationale for patient selection</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">2226</span><span class="NLM_x">â</span> <span class="NLM_lpage">2238</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=2226-2238&author=F.+Huangauthor=K.+Reevesauthor=X.+Hanauthor=C.+Fairchildauthor=S.+Plateroauthor=T.+W.+Wongauthor=F.+Leeauthor=P.+Shawauthor=E.+Clark&title=Identification+of+candidate+molecular+markers+predicting+sensitivity+in+solid+tumors+to+dasatinib%3A+Rationale+for+patient+selection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DF.%26aulast%3DReeves%26aufirst%3DK.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DFairchild%26aufirst%3DC.%26aulast%3DPlatero%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DT.%2BW.%26aulast%3DLee%26aufirst%3DF.%26aulast%3DShaw%26aufirst%3DP.%26aulast%3DClark%26aufirst%3DE.%26atitle%3DIdentification%2520of%2520candidate%2520molecular%2520markers%2520predicting%2520sensitivity%2520in%2520solid%2520tumors%2520to%2520dasatinib%253A%2520Rationale%2520for%2520patient%2520selection%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D2226%26epage%3D2238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">â</span> <span class="NLM_lpage">39</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0lg1CcMMZkwY_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, K.</span><span> </span><span class="NLM_article-title">Assessment of chemical coverage of kinome space and its implications for kinase drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7898</span><span class="NLM_x">â</span> <span class="NLM_lpage">7914</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8011036" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7898-7914&author=P.+Bamboroughauthor=D.+Drewryauthor=G.+Harperauthor=G.+K.+Smithauthor=K.+Schneider&title=Assessment+of+chemical+coverage+of+kinome+space+and+its+implications+for+kinase+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Chemical Coverage of Kinome Space and Its Implications for Kinase Drug Discovery</span></div><div class="casAuthors">Bamborough, Paul; Drewry, David; Harper, Gavin; Smith, Gary K.; Schneider, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7914</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">More than 500 compds. chosen to represent kinase inhibitor space have been screened against a panel of over 200 protein kinases.  Significant results include the identification of hits against new kinases including PIM1 and MPSK1, and the expansion of the inhibition profiles of several literature compds.  A detailed anal. of the data through the use of affinity fingerprints has produced findings with implications for biol. target selection, the choice of tool compds. for target validation, and lead discovery and optimization.  In a detailed examn. of the tyrosine kinases, interesting relationships have been found between targets and compds.  Taken together, these results show how broad cross-profiling can provide important insights to assist kinase drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBRI-arQk3t7Vg90H21EOLACvtfcHk0lg1CcMMZkwY_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI&md5=8d98c80fb157b987a19e32a1efb53103</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm8011036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8011036%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDrewry%26aufirst%3DD.%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DG.%2BK.%26aulast%3DSchneider%26aufirst%3DK.%26atitle%3DAssessment%2520of%2520chemical%2520coverage%2520of%2520kinome%2520space%2520and%2520its%2520implications%2520for%2520kinase%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7898%26epage%3D7914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">â</span> <span class="NLM_lpage">336</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lhGYq_wpokCeA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">â</span> <span class="NLM_lpage">132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lhGYq_wpokCeA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Schmidt-Arras, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwable, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohmer, F. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serve, H.</span><span> </span><span class="NLM_article-title">Flt3 receptor tyrosine kinase as a drug target in leukemia</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1867</span><span class="NLM_x">â</span> <span class="NLM_lpage">1883</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.2174%2F1381612043384394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=15180525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktV2gt78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=1867-1883&author=D.+Schmidt-Arrasauthor=J.+Schwableauthor=F.+D.+Bohmerauthor=H.+Serve&title=Flt3+receptor+tyrosine+kinase+as+a+drug+target+in+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Flt3 receptor tyrosine kinase as a drug target in Leukemia</span></div><div class="casAuthors">Schmidt-Arras, Dirk; Schwaeble, Joachim; Boehmer, Frank-D.; Serve, Hubert</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1867-1883</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The hematopoietic class III receptor tyrosine kinase (RTK) Flt3 (Flk2, STK1) has recently received much attention as a potential drug target.  Activation of Flt3 by different types of mutations plays an important role for proliferation, resistance to apoptosis, and prevention of differentiation of leukemic blasts in acute myeloid leukemia (AML).  At least one type of such mutations - an internal tandem duplication in the Flt3 juxtamembrane domain (Flt3-ITD) - has been assocd. with an unfavorable prognosis.  Signal transduction of Flt3 involves activation of several conserved pathways, including the RAS/MAP-Kinase and the phosphoinositide-3-kinase/Akt signaling cascades.  Transforming versions of Flt3 exhibit altered signaling, for example a very pronounced activation of STAT5, ultimately resulting in alternate profiles of gene expression and cell transformation.  Selective inhibitors of Flt3 tyrosine kinase activity have the potential to suppress aberrant Flt3 signaling.  Although highly homologous to other class III RTKs, Flt3 is resistant to the phenylaminopyrimidine STI571 (Gleevec, Imatinib), a potent inhibitor of other RTKs in the family, such as the PDGFÎ²-receptor or c-Kit.  STI571 binding to Flt3 is prevented by the phenylalanine 691 side-chain in the ATP binding center and mutating this site to threonine renders the corresponding Flt3 mutant sensitive to STI571.  Compds. of several other structural families, including the quinoxaline AGI296, the bis(1H-2-indolyl)-l-methanone D-65476, the indolinones SU5416 and SU11248, the indolocarbazoles PKC412 and CEP-701, and the piperazonyl quinazoline CT53518, are potent inhibitors of Flt3 kinase.  They exhibit different selectivity profiles, both with respect to other kinases and among wildtype Flt3 and its activated versions.  These compds. hold promise as novel drugs against AML and as probes for understanding activation mechanisms and signaling pathways in the class III RTK family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM-Zx05aCHZ7Vg90H21EOLACvtfcHk0lhlx7wg35nrWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktV2gt78%253D&md5=48bf03b9447756d8ac59223b515c1391</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2174%2F1381612043384394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612043384394%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt-Arras%26aufirst%3DD.%26aulast%3DSchwable%26aufirst%3DJ.%26aulast%3DBohmer%26aufirst%3DF.%2BD.%26aulast%3DServe%26aufirst%3DH.%26atitle%3DFlt3%2520receptor%2520tyrosine%2520kinase%2520as%2520a%2520drug%2520target%2520in%2520leukemia%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2004%26volume%3D10%26spage%3D1867%26epage%3D1883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Sawyers, C. L.</span><span> </span><span class="NLM_article-title">Finding the next Gleevec: FLT3 targeted kinase inhibitor therapy for acute myeloid leukemia</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">413</span><span class="NLM_x">â</span> <span class="NLM_lpage">415</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1016%2FS1535-6108%2802%2900080-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=12124170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKjsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=413-415&author=C.+L.+Sawyers&title=Finding+the+next+Gleevec%3A+FLT3+targeted+kinase+inhibitor+therapy+for+acute+myeloid+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Finding the next Gleevec: FLT3 targeted kinase inhibitor therapy for acute myeloid leukemia</span></div><div class="casAuthors">Sawyers, Charles L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">413-415</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Activating mutations in the FLT3 receptor tyrosine kinase occur in 30% of patients with acute myeloid leukemia.  Small mol. FLT3 kinase inhibitors show selective antitumor activity in preclin. models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJQQJ_qew_HbVg90H21EOLACvtfcHk0lhlx7wg35nrWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKjsLk%253D&md5=30686782dc9bc4e953158a07dbe4a187</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS1535-6108%2802%2900080-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1535-6108%252802%252900080-6%26sid%3Dliteratum%253Aachs%26aulast%3DSawyers%26aufirst%3DC.%2BL.%26atitle%3DFinding%2520the%2520next%2520Gleevec%253A%2520FLT3%2520targeted%2520kinase%2520inhibitor%2520therapy%2520for%2520acute%2520myeloid%2520leukemia%26jtitle%3DCancer%2520Cell%26date%3D2002%26volume%3D1%26spage%3D413%26epage%3D415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Lefevre, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glotin, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calipel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouriaux, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kherrouche, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurage, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auclair, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascarelli, F.</span><span> </span><span class="NLM_article-title">Roles of stem cell factor/c-Kit and effects of Glivec/STI571 in human uveal melanoma cell tumorigenesis</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">31769</span><span class="NLM_x">â</span> <span class="NLM_lpage">31779</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=31769-31779&author=G.+Lefevreauthor=A.+L.+Glotinauthor=A.+Calipelauthor=F.+Mouriauxauthor=T.+Tranauthor=Z.+Kherroucheauthor=C.+A.+Maurageauthor=C.+Auclairauthor=F.+Mascarelli&title=Roles+of+stem+cell+factor%2Fc-Kit+and+effects+of+Glivec%2FSTI571+in+human+uveal+melanoma+cell+tumorigenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLefevre%26aufirst%3DG.%26aulast%3DGlotin%26aufirst%3DA.%2BL.%26aulast%3DCalipel%26aufirst%3DA.%26aulast%3DMouriaux%26aufirst%3DF.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DKherrouche%26aufirst%3DZ.%26aulast%3DMaurage%26aufirst%3DC.%2BA.%26aulast%3DAuclair%26aufirst%3DC.%26aulast%3DMascarelli%26aufirst%3DF.%26atitle%3DRoles%2520of%2520stem%2520cell%2520factor%252Fc-Kit%2520and%2520effects%2520of%2520Glivec%252FSTI571%2520in%2520human%2520uveal%2520melanoma%2520cell%2520tumorigenesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D31769%26epage%3D31779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Dai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frey, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtin, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albert, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arries, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlozzari, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauch, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouska, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bousquet, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cunha, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glaser, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcotte, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsh, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moskey, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pease, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stoll, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tapang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wishart, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidsen, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaelides, M. R.</span><span> </span><span class="NLM_article-title">Thienopyrimidine ureas as novel and potent multitargeted receptor tyrosine kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6066</span><span class="NLM_x">â</span> <span class="NLM_lpage">6083</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050458h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpt1Wht7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6066-6083&author=Y.+Daiauthor=Y.+Guoauthor=R.+R.+Freyauthor=Z.+Jiauthor=M.+L.+Curtinauthor=A.+A.+Ahmedauthor=D.+H.+Albertauthor=L.+Arnoldauthor=S.+S.+Arriesauthor=T.+Barlozzariauthor=J.+L.+Bauchauthor=J.+J.+Bouskaauthor=P.+F.+Bousquetauthor=G.+A.+Cunhaauthor=K.+B.+Glaserauthor=J.+Guoauthor=J.+Liauthor=P.+A.+Marcotteauthor=K.+C.+Marshauthor=M.+D.+Moskeyauthor=L.+J.+Peaseauthor=K.+D.+Stewartauthor=V.+S.+Stollauthor=P.+Tapangauthor=N.+Wishartauthor=S.+K.+Davidsenauthor=M.+R.+Michaelides&title=Thienopyrimidine+ureas+as+novel+and+potent+multitargeted+receptor+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Thienopyrimidine Ureas as Novel and Potent Multitargeted Receptor Tyrosine Kinase Inhibitors</span></div><div class="casAuthors">Dai, Yujia; Guo, Yan; Frey, Robin R.; Ji, Zhiqin; Curtin, Michael L.; Ahmed, Asma A.; Albert, Daniel H.; Arnold, Lee; Arries, Shannon S.; Barlozzari, Teresa; Bauch, Joy L.; Bouska, Jennifer J.; Bousquet, Peter F.; Cunha, George A.; Glaser, Keith B.; Guo, Jun; Li, Junling; Marcotte, Patrick A.; Marsh, Kennan C.; Moskey, Maria D.; Pease, Lori J.; Stewart, Kent D.; Stoll, Vincent S.; Tapang, Paul; Wishart, Neil; Davidsen, Steven K.; Michaelides, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6066-6083</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of novel thienopyrimidine-based receptor tyrosine kinase inhibitors has been discovered.  Investigation of structure-activity relationships at the 5- and 6-positions of the thienopyrimidine nucleus led to a series of N,N'-diaryl ureas that potently inhibit all of the vascular endothelial growth factor (VEGF) and platelet-derived growth factor (PDGF) receptor tyrosine kinases.  A kinase insert domain-contg. receptor (KDR) homol. model suggests that these compds. bind to the "inactive conformation" of the enzyme with the urea portion extending into the back hydrophobic pocket adjacent to the ATP (ATP) binding site.  A no. of compds. have been identified as displaying excellent in vivo potency.  In particular, compds. 28 and 76 possess favorable pharmacokinetic (PK) profiles and demonstrate potent antitumor efficacy against the HT1080 human fibrosarcoma xenograft tumor growth model (tumor growth inhibition (TGI) = 75% at 25 mg/kgÂ·day, per os (po)).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYifOiC_HYHrVg90H21EOLACvtfcHk0lh2f3lpi4tnYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpt1Wht7Y%253D&md5=ef245a26e40d18cfaf7f98260827f274</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm050458h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050458h%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DFrey%26aufirst%3DR.%2BR.%26aulast%3DJi%26aufirst%3DZ.%26aulast%3DCurtin%26aufirst%3DM.%2BL.%26aulast%3DAhmed%26aufirst%3DA.%2BA.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DArnold%26aufirst%3DL.%26aulast%3DArries%26aufirst%3DS.%2BS.%26aulast%3DBarlozzari%26aufirst%3DT.%26aulast%3DBauch%26aufirst%3DJ.%2BL.%26aulast%3DBouska%26aufirst%3DJ.%2BJ.%26aulast%3DBousquet%26aufirst%3DP.%2BF.%26aulast%3DCunha%26aufirst%3DG.%2BA.%26aulast%3DGlaser%26aufirst%3DK.%2BB.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMarcotte%26aufirst%3DP.%2BA.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DMoskey%26aufirst%3DM.%2BD.%26aulast%3DPease%26aufirst%3DL.%2BJ.%26aulast%3DStewart%26aufirst%3DK.%2BD.%26aulast%3DStoll%26aufirst%3DV.%2BS.%26aulast%3DTapang%26aufirst%3DP.%26aulast%3DWishart%26aufirst%3DN.%26aulast%3DDavidsen%26aufirst%3DS.%2BK.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26atitle%3DThienopyrimidine%2520ureas%2520as%2520novel%2520and%2520potent%2520multitargeted%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6066%26epage%3D6083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Manley, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drueckes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fendrich, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liebetanz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martiny-Baron, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trappe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wartmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span> </span><span class="NLM_article-title">Extended kinase profile and properties of the protein kinase inhibitor nilotinib</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1804</span><span class="NLM_x">, </span> <span class="NLM_fpage">445</span><span class="NLM_x">â</span> <span class="NLM_lpage">453</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1804&publication_year=2010&pages=445-453&author=P.+W.+Manleyauthor=P.+Drueckesauthor=G.+Fendrichauthor=P.+Furetauthor=J.+Liebetanzauthor=G.+Martiny-Baronauthor=J.+Mestanauthor=J.+Trappeauthor=M.+Wartmannauthor=D.+Fabbro&title=Extended+kinase+profile+and+properties+of+the+protein+kinase+inhibitor+nilotinib"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DFendrich%26aufirst%3DG.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DLiebetanz%26aufirst%3DJ.%26aulast%3DMartiny-Baron%26aufirst%3DG.%26aulast%3DMestan%26aufirst%3DJ.%26aulast%3DTrappe%26aufirst%3DJ.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DFabbro%26aufirst%3DD.%26atitle%3DExtended%2520kinase%2520profile%2520and%2520properties%2520of%2520the%2520protein%2520kinase%2520inhibitor%2520nilotinib%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2010%26volume%3D1804%26spage%3D445%26epage%3D453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Goldstein, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">High-throughput kinase profiling as a platform for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">391</span><span class="NLM_x">â</span> <span class="NLM_lpage">397</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1038%2Fnrd2541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=18404149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=391-397&author=D.+M.+Goldsteinauthor=N.+S.+Grayauthor=P.+P.+Zarrinkar&title=High-throughput+kinase+profiling+as+a+platform+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput kinase profiling as a platform for drug discovery</span></div><div class="casAuthors">Goldstein, David M.; Gray, Nathanael S.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">391-397</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput profiling of compd. libraries against large panels of kinases is becoming tech. feasible.  In contrast to the traditional linear, target-centric approach to discovery, this approach may provide a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone, and could thereby significantly improve the productivity of kinase inhibitor discovery.  To fully exploit the potential of kinases as drug targets, novel strategies for the efficient discovery of inhibitors are required.  In contrast to the traditional, linear process of inhibitor discovery, high-throughput kinase profiling enables a parallel approach by interrogating compds. against hundreds of targets in a single screen.  Compd. potency and selectivity are detd. simultaneously, providing a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwnJ59cVxbJbVg90H21EOLACvtfcHk0lh2f3lpi4tnYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D&md5=eaa61fd2958ff04fecf86a993f91833d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd2541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2541%26sid%3Dliteratum%253Aachs%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DHigh-throughput%2520kinase%2520profiling%2520as%2520a%2520platform%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D391%26epage%3D397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Belema, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaulieu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouellet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McIntyre, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattoli, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daloisio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillooly, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brassil, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zusi, F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vyas, D. M.</span><span> </span><span class="NLM_article-title">Synthesis and structure-activity relationship of imidazo(1,2-a)thieno(3,2-e)pyrazines as IKK-Î² inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">4284</span><span class="NLM_x">â</span> <span class="NLM_lpage">4289</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=4284-4289&author=M.+Belemaauthor=A.+Bunkerauthor=V.+N.+Nguyenauthor=F.+Beaulieuauthor=C.+Ouelletauthor=Y.+Qiuauthor=Y.+Zhangauthor=A.+Martelauthor=J.+R.+Burkeauthor=K.+W.+McIntyreauthor=M.+A.+Pattoliauthor=C.+Daloisioauthor=K.+M.+Gilloolyauthor=W.+J.+Clarkeauthor=P.+J.+Brassilauthor=F.+C.+Zusiauthor=D.+M.+Vyas&title=Synthesis+and+structure-activity+relationship+of+imidazo%281%2C2-a%29thieno%283%2C2-e%29pyrazines+as+IKK-%CE%B2+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBelema%26aufirst%3DM.%26aulast%3DBunker%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DV.%2BN.%26aulast%3DBeaulieu%26aufirst%3DF.%26aulast%3DOuellet%26aufirst%3DC.%26aulast%3DQiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMartel%26aufirst%3DA.%26aulast%3DBurke%26aufirst%3DJ.%2BR.%26aulast%3DMcIntyre%26aufirst%3DK.%2BW.%26aulast%3DPattoli%26aufirst%3DM.%2BA.%26aulast%3DDaloisio%26aufirst%3DC.%26aulast%3DGillooly%26aufirst%3DK.%2BM.%26aulast%3DClarke%26aufirst%3DW.%2BJ.%26aulast%3DBrassil%26aufirst%3DP.%2BJ.%26aulast%3DZusi%26aufirst%3DF.%2BC.%26aulast%3DVyas%26aufirst%3DD.%2BM.%26atitle%3DSynthesis%2520and%2520structure-activity%2520relationship%2520of%2520imidazo%25281%252C2-a%2529thieno%25283%252C2-e%2529pyrazines%2520as%2520IKK-%25CE%25B2%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D4284%26epage%3D4289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Martin, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kofron, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Traphagen, L. M.</span><span> </span><span class="NLM_article-title">Do structurally similar molecules have similar biological activity?</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">4350</span><span class="NLM_x">â</span> <span class="NLM_lpage">4358</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020155c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=4350-4358&author=Y.+C.+Martinauthor=J.+L.+Kofronauthor=L.+M.+Traphagen&title=Do+structurally+similar+molecules+have+similar+biological+activity%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm020155c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020155c%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DY.%2BC.%26aulast%3DKofron%26aufirst%3DJ.%2BL.%26aulast%3DTraphagen%26aufirst%3DL.%2BM.%26atitle%3DDo%2520structurally%2520similar%2520molecules%2520have%2520similar%2520biological%2520activity%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D4350%26epage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Aronov, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClain, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moody, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">Kinase-likeness and kinase-privileged fragments: Toward virtual polypharmacology</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1214</span><span class="NLM_x">â</span> <span class="NLM_lpage">1222</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701021b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1214-1222&author=A.+M.+Aronovauthor=B.+McClainauthor=C.+S.+Moodyauthor=M.+A.+Murcko&title=Kinase-likeness+and+kinase-privileged+fragments%3A+Toward+virtual+polypharmacology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm701021b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701021b%26sid%3Dliteratum%253Aachs%26aulast%3DAronov%26aufirst%3DA.%2BM.%26aulast%3DMcClain%26aufirst%3DB.%26aulast%3DMoody%26aufirst%3DC.%2BS.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DKinase-likeness%2520and%2520kinase-privileged%2520fragments%253A%2520Toward%2520virtual%2520polypharmacology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1214%26epage%3D1222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Briem, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, J.</span><span> </span><span class="NLM_article-title">Classifying âkinase inhibitor-likenessâ by using machine-learning methods</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">558</span><span class="NLM_x">â</span> <span class="NLM_lpage">566</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=558-566&author=H.+Briemauthor=J.+Gunther&title=Classifying+%E2%80%9Ckinase+inhibitor-likeness%E2%80%9D+by+using+machine-learning+methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DGunther%26aufirst%3DJ.%26atitle%3DClassifying%2520%25E2%2580%259Ckinase%2520inhibitor-likeness%25E2%2580%259D%2520by%2520using%2520machine-learning%2520methods%26jtitle%3DChemBioChem%26date%3D2005%26volume%3D6%26spage%3D558%26epage%3D566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Garcia-Echeverria, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Traxler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, D. B.</span><span> </span><span class="NLM_article-title">ATP site-directed competitive and irreversible inhibitors of protein kinases</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">â</span> <span class="NLM_lpage">57</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=28-57&author=C.+Garcia-Echeverriaauthor=P.+Traxlerauthor=D.+B.+Evans&title=ATP+site-directed+competitive+and+irreversible+inhibitors+of+protein+kinases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia-Echeverria%26aufirst%3DC.%26aulast%3DTraxler%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DD.%2BB.%26atitle%3DATP%2520site-directed%2520competitive%2520and%2520irreversible%2520inhibitors%2520of%2520protein%2520kinases%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2000%26volume%3D20%26spage%3D28%26epage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Dumas, J.</span><span> </span><span class="NLM_article-title">Protein kinase inhibitors: Emerging pharmacophores 1997â2000</span> <span class="citation_source-journal">Expert Opin. Ther. Pat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">â</span> <span class="NLM_lpage">429</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1517%2F13543776.11.3.405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslSisrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=405-429&author=J.+Dumas&title=Protein+kinase+inhibitors%3A+Emerging+pharmacophores+1997%E2%88%922000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase inhibitors: emerging pharmacophores 1997 - 2000</span></div><div class="casAuthors">Dumas, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-429</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review with 292 refs. is given.  Kinase inhibition is a major area for therapeutic intervention against a variety of diseases including cancer, inflammatory disorders and diabetes.  While this mode of action remains very promising at this time, it faces crit. challenges, such as drug safety, and biopharmaceutical profile.  This review provides an update on kinase inhibitor pharmacophores from a structural standpoint.  Advances in older inhibitor classes as well as new emerging lead structures will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozUM0oS-Vu8rVg90H21EOLACvtfcHk0liqx8bo--nGbg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhslSisrc%253D&md5=4f6dc306b741963d931d6abfe22e40a2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1517%2F13543776.11.3.405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.11.3.405%26sid%3Dliteratum%253Aachs%26aulast%3DDumas%26aufirst%3DJ.%26atitle%3DProtein%2520kinase%2520inhibitors%253A%2520Emerging%2520pharmacophores%25201997%25E2%2588%25922000%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2001%26volume%3D11%26spage%3D405%26epage%3D429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Hunt, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitt, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borzilleri, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullo-Brown, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fargnoli, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fink, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortillo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vite, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wautlet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhide, R.</span><span> </span><span class="NLM_article-title">Discovery of the pyrrolo[2,1-f][1,2,4]triazine nucleus as a new kinase inhibitor template</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4054</span><span class="NLM_x">â</span> <span class="NLM_lpage">4059</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049892u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4054-4059&author=J.+T.+Huntauthor=T.+Mittauthor=R.+Borzilleriauthor=J.+Gullo-Brownauthor=J.+Fargnoliauthor=B.+Finkauthor=W.+C.+Hanauthor=S.+Mortilloauthor=G.+Viteauthor=B.+Wautletauthor=T.+Wongauthor=C.+Yuauthor=X.+Zhengauthor=R.+Bhide&title=Discovery+of+the+pyrrolo%5B2%2C1-f%5D%5B1%2C2%2C4%5Dtriazine+nucleus+as+a+new+kinase+inhibitor+template"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm049892u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049892u%26sid%3Dliteratum%253Aachs%26aulast%3DHunt%26aufirst%3DJ.%2BT.%26aulast%3DMitt%26aufirst%3DT.%26aulast%3DBorzilleri%26aufirst%3DR.%26aulast%3DGullo-Brown%26aufirst%3DJ.%26aulast%3DFargnoli%26aufirst%3DJ.%26aulast%3DFink%26aufirst%3DB.%26aulast%3DHan%26aufirst%3DW.%2BC.%26aulast%3DMortillo%26aufirst%3DS.%26aulast%3DVite%26aufirst%3DG.%26aulast%3DWautlet%26aufirst%3DB.%26aulast%3DWong%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DBhide%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520the%2520pyrrolo%255B2%252C1-f%255D%255B1%252C2%252C4%255Dtriazine%2520nucleus%2520as%2520a%2520new%2520kinase%2520inhibitor%2520template%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4054%26epage%3D4059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Saxty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyle, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barford, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downham, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, R. A.</span><span> </span><span class="NLM_article-title">Identification of inhibitors of protein kinase B using fragment-based lead discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2293</span><span class="NLM_x">â</span> <span class="NLM_lpage">2296</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm070091b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVKjur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2293-2296&author=G.+Saxtyauthor=S.+J.+Woodheadauthor=V.+Berdiniauthor=T.+G.+Daviesauthor=M.+L.+Verdonkauthor=P.+G.+Wyattauthor=R.+G.+Boyleauthor=D.+Barfordauthor=R.+Downhamauthor=M.+D.+Garrettauthor=R.+A.+Carr&title=Identification+of+inhibitors+of+protein+kinase+B+using+fragment-based+lead+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Inhibitors of Protein Kinase B Using Fragment-Based Lead Discovery</span></div><div class="casAuthors">Saxty, Gordon; Woodhead, Steven J.; Berdini, Valerio; Davies, Thomas G.; Verdonk, Marcel L.; Wyatt, Paul G.; Boyle, Robert G.; Barford, David; Downham, Robert; Garrett, Michelle D.; Carr, Robin A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2293-2296</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using fragment-based screening techniques, 5-methyl-4-phenyl-1H-pyrazole (IC50 80 Î¼M) was identified as a novel, low mol. wt. inhibitor of protein kinase B (PKB).  Herein we describe the rapid elaboration of highly potent and ligand efficient analogs using a fragment growing approach.  Iterative structure-based design was supported by protein-ligand structure detns. using a PKA-PKB "chimera" and a final protein-ligand structure of a lead compd. in PKBÎ² itself.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlCYaIT8Xn87Vg90H21EOLACvtfcHk0lhagS_Nn-Cr4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVKjur8%253D&md5=2a09e4bbaca1493be8e215927e7e72e1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm070091b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070091b%26sid%3Dliteratum%253Aachs%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWoodhead%26aufirst%3DS.%2BJ.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DBoyle%26aufirst%3DR.%2BG.%26aulast%3DBarford%26aufirst%3DD.%26aulast%3DDownham%26aufirst%3DR.%26aulast%3DGarrett%26aufirst%3DM.%2BD.%26aulast%3DCarr%26aufirst%3DR.%2BA.%26atitle%3DIdentification%2520of%2520inhibitors%2520of%2520protein%2520kinase%2520B%2520using%2520fragment-based%2520lead%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2293%26epage%3D2296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Deanda, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reno, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D. H.</span><span> </span><span class="NLM_article-title">Kinase-targeted library design through the application of the PharmPrint methodology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2395</span><span class="NLM_x">â</span> <span class="NLM_lpage">2403</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800276t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWgt7%252FF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=2395-2403&author=F.+Deandaauthor=E.+L.+Stewartauthor=M.+J.+Renoauthor=D.+H.+Drewry&title=Kinase-targeted+library+design+through+the+application+of+the+PharmPrint+methodology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase-Targeted Library Design through the Application of the PharmPrint Methodology</span></div><div class="casAuthors">Deanda, Felix; Stewart, Eugene L.; Reno, Michael J.; Drewry, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2395-2403</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The PharmPrint methodol., as modified and implemented by Deanda and Stewart, was prospectively evaluated for use as a virtual high-throughput screening tool by applying it to the design of target-focused arrays.  To this end, PharmPrint quant. structure-activity relationship (QSAR) models for the prediction of AKT1, Aurora-A, and ROCK1 inhibition were constructed and used to virtually screen two large combinatorial libraries.  Based on predicted activities, an Aurora-A targeted array and a ROCK1 targeted array were designed and synthesized.  One control group per designed array was also synthesized to assess the enrichment levels achieved by the QSAR models.  For the Aurora-A targeted array, the hit rate, against the intended target, was 42.9%, whereas that of the control group was 0%.  Thus, the enrichment level achieved by the Aurora-A QSAR model was incalculable.  For the ROCK1 targeted array, the hit rate against the intended target was 30.6%, whereas that of the control group was 5.10%, making the enrichment level achieved by the ROCK1 QSAR model 6-fold above control.  Clearly, these results support the use of the PharmPrint methodol. as a virtual screening tool for the design of kinase-targeted arrays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXVi-syw96s7Vg90H21EOLACvtfcHk0lhagS_Nn-Cr4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWgt7%252FF&md5=0a7c65ed90b10dc8de5016af052304f6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci800276t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800276t%26sid%3Dliteratum%253Aachs%26aulast%3DDeanda%26aufirst%3DF.%26aulast%3DStewart%26aufirst%3DE.%2BL.%26aulast%3DReno%26aufirst%3DM.%2BJ.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26atitle%3DKinase-targeted%2520library%2520design%2520through%2520the%2520application%2520of%2520the%2520PharmPrint%2520methodology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D2395%26epage%3D2403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Manallack, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gancia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montana, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Livingstone, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitley, D. C.</span><span> </span><span class="NLM_article-title">Selecting screening candidates for kinase and G protein-coupled receptor targets using neural networks</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">1256</span><span class="NLM_x">â</span> <span class="NLM_lpage">1262</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci020267c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1256-1262&author=D.+T.+Manallackauthor=W.+R.+Pittauthor=E.+Ganciaauthor=J.+G.+Montanaauthor=D.+J.+Livingstoneauthor=M.+G.+Fordauthor=D.+C.+Whitley&title=Selecting+screening+candidates+for+kinase+and+G+protein-coupled+receptor+targets+using+neural+networks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Selecting Screening Candidates for Kinase and G Protein-Coupled Receptor Targets Using Neural Networks</span></div><div class="casAuthors">Manallack, David T.; Pitt, Will R.; Gancia, Emanuela; Montana, John G.; Livingstone, David J.; Ford, Martyn G.; Whitley, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1256-1262</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of neural networks has been trained, using consensus methods, to recognize compds. that act at biol. targets belonging to specific gene families.  The MDDR database was used to provide compds. targeted against gene families and sets of randomly selected mols.  BCUT parameters were employed as input descriptors that encode structural properties and information relevant to ligand-receptor interactions.  In each case, the networks identified over 80% of the compds. targeting a gene family.  The technique was applied to purchasing compds. from external suppliers, and results from screening against one gene family demonstrated impressive abilities to predict the activity of the majority of known hit compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3B0ertK2pfbVg90H21EOLACvtfcHk0lhagS_Nn-Cr4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrg%253D&md5=264cc5d5bc3657018ca0bd232954765c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci020267c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci020267c%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DGancia%26aufirst%3DE.%26aulast%3DMontana%26aufirst%3DJ.%2BG.%26aulast%3DLivingstone%26aufirst%3DD.%2BJ.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DWhitley%26aufirst%3DD.%2BC.%26atitle%3DSelecting%2520screening%2520candidates%2520for%2520kinase%2520and%2520G%2520protein-coupled%2520receptor%2520targets%2520using%2520neural%2520networks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D1256%26epage%3D1262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Akritopoulou-Zanze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Kinase-targeted libraries: The design and synthesis of novel, potent, and selective kinase inhibitors</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">â</span> <span class="NLM_lpage">297</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=291-297&author=I.+Akritopoulou-Zanzeauthor=P.+J.+Hajduk&title=Kinase-targeted+libraries%3A+The+design+and+synthesis+of+novel%2C+potent%2C+and+selective+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAkritopoulou-Zanze%26aufirst%3DI.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DKinase-targeted%2520libraries%253A%2520The%2520design%2520and%2520synthesis%2520of%2520novel%252C%2520potent%252C%2520and%2520selective%2520kinase%2520inhibitors%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D291%26epage%3D297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Martin, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, D. C.</span><span> </span><span class="NLM_article-title">Surrogate AutoShim: Predocking into a universal ensemble kinase receptor for three dimensional activity prediction, very quickly, without a crystal structure</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">â</span> <span class="NLM_lpage">881</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700455u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktVOkurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=873-881&author=E.+J.+Martinauthor=D.+C.+Sullivan&title=Surrogate+AutoShim%3A+Predocking+into+a+universal+ensemble+kinase+receptor+for+three+dimensional+activity+prediction%2C+very+quickly%2C+without+a+crystal+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Surrogate AutoShim: Predocking into a Universal Ensemble Kinase Receptor for Three Dimensional Activity Prediction, Very Quickly, without a Crystal Structure</span></div><div class="casAuthors">Martin, Eric J.; Sullivan, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">873-881</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">"Ensemble surrogate AutoShim" is a kinase specific extension of the AutoShim docking method that solves the three traditional limitations of conventional docking: (1) it gives good correlations with affinity, (2) does not require a target protein structure, and (3) for a preprocessed company archive of 1.5 million compds., is as fast as traditional 2D QSAR.  It does require several hundred exptl. IC50 values for each new target.  Original AutoShim adds pharmacophore "shims" to a crystal structure binding site.  An iterative partial least squares (PLS) procedure selects the best pose, while adjusting the shim wts. to reproduce IC50 data.  Surrogate AutoShim adjusts shims in one crystal structure to reproduce IC50 data for a different kinase target.  Ensemble surrogate AutoShim uses 16 structurally diverse kinase crystal structures as a "universal ensemble kinase receptor", suitable for any kinase target.  The 1.5 million member Novartis screening collection has been predocked into the shimmed ensemble, so new kinase models can be built, and the entire corporate archive virtually screened, in hours rather than weeks.  A kinase-biased set of 10 000 compds., that samples the entire corporate archive, has been designed for lead discovery by iterative kinase screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5HSUkdEqTcLVg90H21EOLACvtfcHk0lgFLuNsKQN9ow"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktVOkurg%253D&md5=0fe14c274192ec38b65224cae779cf35</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci700455u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700455u%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%2BJ.%26aulast%3DSullivan%26aufirst%3DD.%2BC.%26atitle%3DSurrogate%2520AutoShim%253A%2520Predocking%2520into%2520a%2520universal%2520ensemble%2520kinase%2520receptor%2520for%2520three%2520dimensional%2520activity%2520prediction%252C%2520very%2520quickly%252C%2520without%2520a%2520crystal%2520structure%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D873%26epage%3D881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Gozalbes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Froloff, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartori, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteils, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baudelle, R.</span><span> </span><span class="NLM_article-title">Development and experimental validation of a docking strategy for the generation of kinase-targeted libraries</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3124</span><span class="NLM_x">â</span> <span class="NLM_lpage">3132</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701367r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3124-3132&author=R.+Gozalbesauthor=L.+Simonauthor=N.+Froloffauthor=E.+Sartoriauthor=C.+Monteilsauthor=R.+Baudelle&title=Development+and+experimental+validation+of+a+docking+strategy+for+the+generation+of+kinase-targeted+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm701367r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701367r%26sid%3Dliteratum%253Aachs%26aulast%3DGozalbes%26aufirst%3DR.%26aulast%3DSimon%26aufirst%3DL.%26aulast%3DFroloff%26aufirst%3DN.%26aulast%3DSartori%26aufirst%3DE.%26aulast%3DMonteils%26aufirst%3DC.%26aulast%3DBaudelle%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520experimental%2520validation%2520of%2520a%2520docking%2520strategy%2520for%2520the%2520generation%2520of%2520kinase-targeted%2520libraries%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3124%26epage%3D3132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Dongyu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claudio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donovan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juswinder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patrick, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gretchen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen-Cherng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jason, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jessica, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serene, J.</span><span> </span><span class="NLM_article-title">A kinase-focused compound collection: Compilation and screening strategy</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">385</span><span class="NLM_x">â</span> <span class="NLM_lpage">394</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1111%2Fj.1747-0285.2006.00396.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=16882313" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2006&pages=385-394&author=S.+Dongyuauthor=C.+Claudioauthor=D.+Zhanauthor=B.+Scottauthor=C.+Donovanauthor=S.+Juswinderauthor=C.+Patrickauthor=H.+Gretchenauthor=L.+Wen-Cherngauthor=D.+Jasonauthor=F.+Jessicaauthor=J.+Serene&title=A+kinase-focused+compound+collection%3A+Compilation+and+screening+strategy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2006.00396.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2006.00396.x%26sid%3Dliteratum%253Aachs%26aulast%3DDongyu%26aufirst%3DS.%26aulast%3DClaudio%26aufirst%3DC.%26aulast%3DZhan%26aufirst%3DD.%26aulast%3DScott%26aufirst%3DB.%26aulast%3DDonovan%26aufirst%3DC.%26aulast%3DJuswinder%26aufirst%3DS.%26aulast%3DPatrick%26aufirst%3DC.%26aulast%3DGretchen%26aufirst%3DH.%26aulast%3DWen-Cherng%26aufirst%3DL.%26aulast%3DJason%26aufirst%3DD.%26aulast%3DJessica%26aufirst%3DF.%26aulast%3DSerene%26aufirst%3DJ.%26atitle%3DA%2520kinase-focused%2520compound%2520collection%253A%2520Compilation%2520and%2520screening%2520strategy%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2006%26volume%3D67%26spage%3D385%26epage%3D394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erickson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I.</span><span> </span><span class="NLM_article-title">Kinase inhibitor data modeling and de novo inhibitor design with fragment approaches</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6456</span><span class="NLM_x">â</span> <span class="NLM_lpage">6466</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901147e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6456-6466&author=M.+Viethauthor=J.+Ericksonauthor=J.+Wangauthor=Y.+Websterauthor=M.+Maderauthor=R.+Higgsauthor=I.+Watson&title=Kinase+inhibitor+data+modeling+and+de+novo+inhibitor+design+with+fragment+approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm901147e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901147e%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DErickson%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWebster%26aufirst%3DY.%26aulast%3DMader%26aufirst%3DM.%26aulast%3DHiggs%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DI.%26atitle%3DKinase%2520inhibitor%2520data%2520modeling%2520and%2520de%2520novo%2520inhibitor%2520design%2520with%2520fragment%2520approaches%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6456%26epage%3D6466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Akritopoulou-Zanze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darczak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarris, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phelan, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djuric, S. W.</span><span> </span><span class="NLM_article-title">Scaffold oriented synthesis. Part 1: Design, preparation, and biological evaluation of thienopyrazoles as kinase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">96</span><span class="NLM_x">â</span> <span class="NLM_lpage">99</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=96-99&author=I.+Akritopoulou-Zanzeauthor=D.+Darczakauthor=K.+Sarrisauthor=K.+M.+Phelanauthor=J.+R.+Huthauthor=D.+Songauthor=E.+F.+Johnsonauthor=Y.+Jiaauthor=S.+W.+Djuric&title=Scaffold+oriented+synthesis.+Part+1%3A+Design%2C+preparation%2C+and+biological+evaluation+of+thienopyrazoles+as+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAkritopoulou-Zanze%26aufirst%3DI.%26aulast%3DDarczak%26aufirst%3DD.%26aulast%3DSarris%26aufirst%3DK.%26aulast%3DPhelan%26aufirst%3DK.%2BM.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DDjuric%26aufirst%3DS.%2BW.%26atitle%3DScaffold%2520oriented%2520synthesis.%2520Part%25201%253A%2520Design%252C%2520preparation%252C%2520and%2520biological%2520evaluation%2520of%2520thienopyrazoles%2520as%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D96%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Gracias, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akritopoulou-Zanze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abad-Zapatero, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glaser, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcotte, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pease, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidsen, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaelides, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djuric, S. W.</span><span> </span><span class="NLM_article-title">Scaffold oriented synthesis. Part 2: Design, synthesis and biological evaluation of pyrimido-diazepines as receptor tyrosine kinase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">2691</span><span class="NLM_x">â</span> <span class="NLM_lpage">2695</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=2691-2695&author=V.+Graciasauthor=Z.+Jiauthor=I.+Akritopoulou-Zanzeauthor=C.+Abad-Zapateroauthor=J.+R.+Huthauthor=D.+Songauthor=P.+J.+Hajdukauthor=E.+F.+Johnsonauthor=K.+B.+Glaserauthor=P.+A.+Marcotteauthor=L.+Peaseauthor=N.+B.+Soniauthor=K.+D.+Stewartauthor=S.+K.+Davidsenauthor=M.+R.+Michaelidesauthor=S.+W.+Djuric&title=Scaffold+oriented+synthesis.+Part+2%3A+Design%2C+synthesis+and+biological+evaluation+of+pyrimido-diazepines+as+receptor+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGracias%26aufirst%3DV.%26aulast%3DJi%26aufirst%3DZ.%26aulast%3DAkritopoulou-Zanze%26aufirst%3DI.%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DGlaser%26aufirst%3DK.%2BB.%26aulast%3DMarcotte%26aufirst%3DP.%2BA.%26aulast%3DPease%26aufirst%3DL.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DStewart%26aufirst%3DK.%2BD.%26aulast%3DDavidsen%26aufirst%3DS.%2BK.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DDjuric%26aufirst%3DS.%2BW.%26atitle%3DScaffold%2520oriented%2520synthesis.%2520Part%25202%253A%2520Design%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520pyrimido-diazepines%2520as%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D2691%26epage%3D2695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Kettle, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, R. A.</span><span> </span><span class="NLM_article-title">Toward the comprehensive systematic enumeration and synthesis of novel kinase inhibitors based on a 4-anilinoquinazoline binding mode</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">â</span> <span class="NLM_lpage">533</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci9004767" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1GmsrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=525-533&author=J.+G.+Kettleauthor=R.+A.+Ward&title=Toward+the+comprehensive+systematic+enumeration+and+synthesis+of+novel+kinase+inhibitors+based+on+a+4-anilinoquinazoline+binding+mode"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the Comprehensive Systematic Enumeration and Synthesis of Novel Kinase Inhibitors Based on a 4-Anilinoquinazoline Binding Mode</span></div><div class="casAuthors">Kettle, Jason G.; Ward, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">525-533</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There are currently eight small-mol. kinase inhibitors approved as cancer treatments, and a significantly larger no. of compds. are in the earlier stages of clin. development.  Although kinase inhibitors are most commonly developed in a cancer setting, other disease areas have been targeted.  The vast majority of reported kinase small-mol. inhibitors contain functionalities that interact with the ATP binding site of the kinase.  The 4-anilinoquinazolines have previously been reported as potent epidermal growth factor receptor (EGFR) inhibitors, binding at the hinge' region of the ATP site.  Subsequently, this chem. series has been optimized against a no. of different kinases including Src and Aurora B.  Here, we detail the computational enumeration of ring systems that have the ability to make comparable interactions to the 4-anilinoquinazoline core.  These were prioritized by computational, medicinal, and synthetic chem. input, and a no. of libraries were subsequently synthesized (no prepn. shown).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoThVN2M1nTkLVg90H21EOLACvtfcHk0liBFsb9GnPJOw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1GmsrY%253D&md5=eb70799255add46dec8828d65b861377</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci9004767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9004767%26sid%3Dliteratum%253Aachs%26aulast%3DKettle%26aufirst%3DJ.%2BG.%26aulast%3DWard%26aufirst%3DR.%2BA.%26atitle%3DToward%2520the%2520comprehensive%2520systematic%2520enumeration%2520and%2520synthesis%2520of%2520novel%2520kinase%2520inhibitors%2520based%2520on%2520a%25204-anilinoquinazoline%2520binding%2520mode%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D525%26epage%3D533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Harper, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickett, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, D. V.</span><span> </span><span class="NLM_article-title">Design of a compound screening collection for use in high throughput screening</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">63</span><span class="NLM_x">â</span> <span class="NLM_lpage">70</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.2174%2F138620704772884832" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=14965262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2004&pages=63-70&author=G.+Harperauthor=S.+D.+Pickettauthor=D.+V.+Green&title=Design+of+a+compound+screening+collection+for+use+in+high+throughput+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a compound screening collection for use in high throughput screening</span></div><div class="casAuthors">Harper, G.; Pickett, S. D.; Green, D. V. S.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-71</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">In this paper we introduce a quant. model that relates chem. structural similarity to biol. activity, and in particular to the activity of lead series of compds. in high-throughput assays.  From this model we derive the optimal screening collection make up for a given fixed size of screening collection, and identify the conditions under which a diverse collection of compds. or a collection focusing on particular regions of chem. space are appropriate strategies.  We derive from the model a diversity function that may be used to assess compds. for acquisition or libraries for combinatorial synthesis by their ability to complement an existing screening collection.  The diversity function is linked directly through the model to the goal of more frequent discovery of lead series from high-throughput screening.  We show how the model may also be used to derive relationships between collection size and probabilities of lead discovery in high-throughput screening, and to guide the judicious application of structural filters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpThXGuh69P87Vg90H21EOLACvtfcHk0liBFsb9GnPJOw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D&md5=b29b45b082fe5fc9d6400e1c8fd83cc0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2174%2F138620704772884832&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620704772884832%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DGreen%26aufirst%3DD.%2BV.%26atitle%3DDesign%2520of%2520a%2520compound%2520screening%2520collection%2520for%2520use%2520in%2520high%2520throughput%2520screening%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2004%26volume%3D7%26spage%3D63%26epage%3D70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Carhart, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkataraghavan, R.</span><span> </span><span class="NLM_article-title">Atom pairs as molecular features in structure-activity studies: Definition and applications</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">64</span><span class="NLM_x">â</span> <span class="NLM_lpage">73</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci00046a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADyaL2MXitVehu7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1985&pages=64-73&author=R.+E.+Carhartauthor=D.+H.+Smithauthor=R.+Venkataraghavan&title=Atom+pairs+as+molecular+features+in+structure-activity+studies%3A+Definition+and+applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Atom pairs as molecular features in structure-activity studies:  definition and applications</span></div><div class="casAuthors">Carhart, Raymond E.; Smith, Dennis H.; Venkataraghavan, R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">64-73</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">A simple type of substructure called an atom pair is defined in terms of the at. environments of, and shortest path sepns. between, all pairs of atoms in the topol. representation of a chem. structure.  An algorithm is presented for computing atom pairs from such a representation.  Two applications of atom pairs to structure-activity problems are described.  In the 1st, a measure of similarity between compds. is defined, and the use of this measure in probing large data bases of structures is discussed.  In the 2nd, an heuristic technique called trend vector anal. is described.  The trend vector summarizes the correlation, within a set of structures, of the occurrence of atom pairs of different types with measured biol. activity.  These correlations can be used to est. the biol. activity of new compds.  A comparison of trend vector anal. with discriminant plane anal. is presented for 1 series of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV9lxLct8p0rVg90H21EOLACvtfcHk0liZ67yM0ucNew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXitVehu7o%253D&md5=9e2cc9e9fda27db87328b738cd6878e2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci00046a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci00046a002%26sid%3Dliteratum%253Aachs%26aulast%3DCarhart%26aufirst%3DR.%2BE.%26aulast%3DSmith%26aufirst%3DD.%2BH.%26aulast%3DVenkataraghavan%26aufirst%3DR.%26atitle%3DAtom%2520pairs%2520as%2520molecular%2520features%2520in%2520structure-activity%2520studies%253A%2520Definition%2520and%2520applications%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1985%26volume%3D25%26spage%3D64%26epage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Similarity-based virtual screening using 2D fingerprints</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">â</span> <span class="NLM_lpage">1053</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1016%2Fj.drudis.2006.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=17129822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1eqtb%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=1046-1053&author=P.+Willett&title=Similarity-based+virtual+screening+using+2D+fingerprints"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity-based virtual screening using 2D fingerprints</span></div><div class="casAuthors">Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">23 & 24</span>),
    <span class="NLM_cas:pages">1046-1053</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  This paper summarizes recent work at the University of Sheffield on virtual screening methods that use 2D fingerprint measures of structural similarity.  A detailed comparison of a large no. of similarity coeffs. demonstrates that the well-known Tanimoto coeff. remains the method of choice for the computation of fingerprint-based similarity, despite possessing some inherent biases related to the sizes of the mols. that are being sought.  Group fusion involves combining the results of similarity searches based on multiple ref. structures and a single similarity measure.  We demonstrate the effectiveness of this approach to screening, and also describe an approx. form of group fusion, turbo similarity searching, that can be used when just a single ref. structure is available.  This paper discusses the use of binary-encoded fragment substructures to scan databases to find mols. that are structurally similar to a bioactive query compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3TRt6CMU5VrVg90H21EOLACvtfcHk0liZ67yM0ucNew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1eqtb%252FN&md5=095fe152b1ae1abbea69f02123651874</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DSimilarity-based%2520virtual%2520screening%2520using%25202D%2520fingerprints%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D1046%26epage%3D1053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Notredame, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heringa, J.</span><span> </span><span class="NLM_article-title">T-coffee: A novel method for fast and accurate multiple sequence alignment</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">302</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">â</span> <span class="NLM_lpage">217</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1006%2Fjmbi.2000.4042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=10964570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtVGntr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=302&publication_year=2000&pages=205-217&author=C.+Notredameauthor=D.+G.+Higginsauthor=J.+Heringa&title=T-coffee%3A+A+novel+method+for+fast+and+accurate+multiple+sequence+alignment"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">T-Coffee: A Novel Method for Fast and Accurate Multiple Sequence Alignment</span></div><div class="casAuthors">Notredame, Cedric; Higgins, Desmond G.; Heringa, Jaap</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">302</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">205-217</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">We describe a new method (T-Coffee) for multiple sequence alignment that provides a dramatic improvement in accuracy with a modest sacrifice in speed as compared to the most commonly used alternatives.  The method is broadly based on the popular progressive approach to multiple alignment but avoids the most serious pitfalls caused by the greedy nature of this algorithm.  With T-Coffee we pre-process a data set of all pair-wise alignments between the sequences.  This provides us with a library of alignment information that can be used to guide the progressive alignment.  Intermediate alignments are then based not only on the sequences to be aligned next but also on how all of the sequences align with each other.  This alignment information can be derived from heterogeneous sources such as a mixt. of alignment programs and/or structure superposition.  Here, we illustrate the power of the approach by using a combination of local and global pair-wise alignments to generate the library.  The resulting alignments are significantly more reliable, as detd. by comparison with a set of 141 test cases, than any of the popular alternatives that we tried.  The improvement, esp. clear with the more difficult test cases, is always visible, regardless of the phylogenetic spread of the sequences in the tests.  (c) 2000 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraooGHJVZRYrVg90H21EOLACvtfcHk0liZ67yM0ucNew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtVGntr8%253D&md5=62c7b0c4e224acf1b172810571d5755d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2000.4042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2000.4042%26sid%3Dliteratum%253Aachs%26aulast%3DNotredame%26aufirst%3DC.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26aulast%3DHeringa%26aufirst%3DJ.%26atitle%3DT-coffee%253A%2520A%2520novel%2520method%2520for%2520fast%2520and%2520accurate%2520multiple%2520sequence%2520alignment%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2000%26volume%3D302%26spage%3D205%26epage%3D217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span> <span class="citation_source-book">ClassPharmer</span>, version 4.5; <span class="NLM_publisher-name">simulationsplus, Inc.</span>: <span class="NLM_publisher-loc">Lancaster, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ClassPharmer%2C+version+4.5%3B+simulationsplus%2C+Inc.%3A+Lancaster%2C+CA%2C+2008."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DClassPharmer%26pub%3Dsimulationsplus%252C%2520Inc%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span> <span class="citation_source-book">Pipeline Pilot</span>, version 7.5; <span class="NLM_publisher-name">Accelrys</span>: <span class="NLM_publisher-loc">San Diego</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Pipeline+Pilot%2C+version+7.5%3B+Accelrys%3A+San+Diego%2C+2008."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DPipeline%2520Pilot%26pub%3DAccelrys%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span> <span class="citation_source-book">OEChem</span>, version 1.7.0; <span class="NLM_publisher-name">OpenEye Scientific Software, Inc.</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=OEChem%2C+version+1.7.0%3B+OpenEye+Scientific+Software%2C+Inc.%3A+Santa+Fe%2C+NM%2C+2008."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DOEChem%26pub%3DOpenEye%2520Scientific%2520Software%252C%2520Inc%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Crivori, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrupt, P.-A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, B.</span><span> </span><span class="NLM_article-title">Predicting blood-brain barrier permeation from three-dimensional molecular structure</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">2204</span><span class="NLM_x">â</span> <span class="NLM_lpage">2216</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm990968%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjtFKhtLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=2204-2216&author=P.+Crivoriauthor=G.+Crucianiauthor=P.-A.+Carruptauthor=B.+Testa&title=Predicting+blood-brain+barrier+permeation+from+three-dimensional+molecular+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting blood-brain barrier permeation from three-dimensional molecular structure</span></div><div class="casAuthors">Crivori, Patrizia; Cruciani, Gabriele; Carrupt, Pierre-Alain; Testa, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2204-2216</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Predicting blood-brain barrier (BBB) permeation remains a challenge in drug design.  Since it is impossible to det. exptl. the BBB partitioning of large nos. of preclin. candidates, alternative evaluation methods based on computerized models are desirable.  The present study was conducted to demonstrate the value of descriptors derived from 3D mol. fields in estg. the BBB permeation of a large set of compds. and to produce a simple math. model suitable for external prediction.  The method used (VolSurf) transforms 3D fields into descriptors and correlates them to the exptl. permeation by a discriminant partial least squares procedure.  The model obtained here correctly predicts more than 90% of the BBB permeation data.  By quantifying the favorable and unfavorable contributions of physicochem. and structural properties, it also offers valuable insights for drug design, pharmacol. profiling, and screening.  The computational procedure is fully automated and quite fast.  The method thus appears as a valuable new tool in virtual screening where selection or prioritization of candidates is required from large collections of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoURsaBTpmsObVg90H21EOLACvtfcHk0lgko78fs9_leg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjtFKhtLk%253D&md5=569f3500268859660bb9e5206da270fe</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm990968%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990968%252B%26sid%3Dliteratum%253Aachs%26aulast%3DCrivori%26aufirst%3DP.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DCarrupt%26aufirst%3DP.-A.%26aulast%3DTesta%26aufirst%3DB.%26atitle%3DPredicting%2520blood-brain%2520barrier%2520permeation%2520from%2520three-dimensional%2520molecular%2520structure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D2204%26epage%3D2216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guba, W.</span><span> </span><span class="NLM_article-title">VolSurf: A new tool for the pharmacokinetic optimization of lead compounds</span> <span class="citation_source-journal">Eur. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">S29</span><span class="NLM_x">â</span> <span class="NLM_lpage">S39</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm101195a&amp;key=10.1016%2FS0928-0987%2800%2900162-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm101195a&amp;key=11033425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm101195a&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntF2gtbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2000&pages=S29-S39&author=G.+Crucianiauthor=M.+Pastorauthor=W.+Guba&title=VolSurf%3A+A+new+tool+for+the+pharmacokinetic+optimization+of+lead+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">VolSurf: a new tool for the pharmacokinetic optimization of lead compounds</span></div><div class="casAuthors">Cruciani, G.; Pastor, M.; Guba, W.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">S29-S39</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A method for the modeling and prediction of pharmacokinetic properties based on computed mol. interaction fields and multivariate statistics has been investigated in different exptl. datasets.  The program VolSurf was used to correlate 3D mol. structures with physico-chem. and pharmacokinetic properties.  In membrane partitioning, VolSurf produced a two-component model explaining 94% of the total variation with a predictive q2 of 0.90.  This result was achieved without conformational sampling and without any quantum-chem. calcn.  For the prediction of blood-brain barrier penetration the VolSurf model was able to predict the BBB profile for most of the drugs in the external prediction set.  In Caco-2 and MDCK permeation expts., VolSurf was used with success to establish statistical models and to predict the behavior of new compds.  The method thus appears as a valuable new property filter in virtual screening and as a novel tool in optimizing the pharmacokinetic profile of pharmaceutically relevant compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCi7-OnhbhmbVg90H21EOLACvtfcHk0lgko78fs9_leg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntF2gtbo%253D&md5=c1ed35a3b3180548485ef1faa833f6c0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2FS0928-0987%2800%2900162-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0928-0987%252800%252900162-7%26sid%3Dliteratum%253Aachs%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DPastor%26aufirst%3DM.%26aulast%3DGuba%26aufirst%3DW.%26atitle%3DVolSurf%253A%2520A%2520new%2520tool%2520for%2520the%2520pharmacokinetic%2520optimization%2520of%2520lead%2520compounds%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2000%26volume%3D11%26spage%3DS29%26epage%3DS39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i35"><a href="/doi/suppl/10.1021/jm101195a">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_03314"></div></div></div></div></div><hr /></hr><p class="last">Scatter plots of the data shown in panels A and B of Figure <a id="rightTab-" class=" internalNav" href="#fig4">4</a>, the list of Ambit BioSciences targets and their group/family classification, hit rates for all compounds and for the SP set for all targets, separation coefficients for all physicochemical properties, neighbor hit rates for all targets, correspondence between Ambit BioSciences binding values and functional data for 37 additional assays, and pairwise activity homologies. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm101195a/suppl_file/jm101195a_si_001.pdf">jm101195a_si_001.pdf (324.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm101195a/suppl_file/jm101195a_si_002.pdf">jm101195a_si_002.pdf (2.61 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm101195a&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2011.54.issue-1%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Fjm101195a" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm101195a" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679928b779ec3bf2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
