
<html lang="en"     class="pb-page"  data-request-id="c5a9ebcc-80d9-4401-b094-1fd171d34874"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.5b02041;issue:issue:10.1021/jmcmar.2016.59.issue-9;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Delineation of Polypharmacology across the Human Structural Kinome Using a Functional Site Interaction Fingerprint Approach" /></meta><meta name="dc.Creator" content="Zheng  Zhao" /></meta><meta name="dc.Creator" content="Li  Xie" /></meta><meta name="dc.Creator" content="Lei  Xie" /></meta><meta name="dc.Creator" content="Philip E.  Bourne" /></meta><meta name="dc.Description" content="Targeted polypharmacology of kinases has emerged as a promising strategy to design efficient and safe therapeutics. Here, we perform a systematic study of kinase–ligand binding modes for the human ..." /></meta><meta name="Description" content="Targeted polypharmacology of kinases has emerged as a promising strategy to design efficient and safe therapeutics. Here, we perform a systematic study of kinase–ligand binding modes for the human ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 17, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b02041" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b02041" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b02041" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b02041" /></link>
        
    
    

<title>Delineation of Polypharmacology across the Human Structural Kinome Using a Functional Site Interaction Fingerprint Approach | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b02041" /></meta><meta property="og:title" content="Delineation of Polypharmacology across the Human Structural Kinome Using a Functional Site Interaction Fingerprint Approach" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0012.jpeg" /></meta><meta property="og:description" content="Targeted polypharmacology of kinases has emerged as a promising strategy to design efficient and safe therapeutics. Here, we perform a systematic study of kinase–ligand binding modes for the human structural kinome at scale (208 kinases, 1777 unique ligands, and their complexes) by integrating chemical genomics and structural genomics data and by introducing a functional site interaction fingerprint (Fs-IFP) method. New insights into kinase–ligand binding modes were obtained. We establish relationships between the features of binding modes, the ligands, and the binding pockets, respectively. We also drive the intrinsic binding specificity and which correlation with amino acid conservation. Third, we explore the landscape of the binding modes and highlight the regions of “selectivity pocket” and “selectivity entrance”. Finally, we demonstrate that Fs-IFP similarity is directly correlated to the experimentally determined profile. These improve our understanding of kinase–ligand interactions and contribute to the design of novel polypharmacological therapies targeting kinases." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b02041"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b02041">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b02041&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b02041&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b02041&amp;href=/doi/10.1021/acs.jmedchem.5b02041" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4326-4341</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01980" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b02042" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Delineation of Polypharmacology across the Human Structural Kinome Using a Functional Site Interaction Fingerprint Approach</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zheng++Zhao">Zheng Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Li++Xie">Li Xie</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lei++Xie">Lei Xie</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Philip+E.++Bourne">Philip E. Bourne</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20894, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Scripps Ranch, San Diego, California 92131, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Computer Science, Hunter College, The City University of New York, New York, New York 10065, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">The Graduate Center, The City University of New York, New York, New York 10016, United States</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Office of the Director, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div><div class="corresp-info"><strong>*</strong>L.X.: phone, +1-212-396-6550; email, <a href="/cdn-cgi/l/email-protection#402c25296e3829250028352e3425326e23352e396e252435"><span class="__cf_email__" data-cfemail="c2aea7abecbaaba782aab7acb6a7b0eca1b7acbbeca7a6b7">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>P.E.B.: phone, +1-301-402-9818; e-mail, <a href="/cdn-cgi/l/email-protection#403028292c29306e222f35322e25002e29286e272f36"><span class="__cf_email__" data-cfemail="68180001040118460a071d1a060d28060100460f071e">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b02041&amp;href=/doi/10.1021%2Facs.jmedchem.5b02041" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4326–4341</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 1, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 December 2015</li><li><span class="item_label"><b>Published</b> online</span>17 March 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 May 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b02041" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b02041</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4326%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DZheng%2BZhao%252C%2BLi%2BXie%252C%2BLei%2BXie%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D9%26contentID%3Dacs.jmedchem.5b02041%26title%3DDelineation%2Bof%2BPolypharmacology%2Bacross%2Bthe%2BHuman%2BStructural%2BKinome%2BUsing%2Ba%2BFunctional%2BSite%2BInteraction%2BFingerprint%2BApproach%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4341%26publicationDate%3DMay%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b02041"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1709</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">25</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b02041" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Delineation of Polypharmacology across the Human Structural Kinome Using a Functional Site Interaction Fingerprint Approach&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Zheng&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Lei&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Philip&quot;,&quot;last_name&quot;:&quot;E. Bourne&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;4326-4341&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b02041&quot;},&quot;abstract&quot;:&quot;Targeted polypharmacology of kinases has emerged as a promising strategy to design efficient and safe therapeutics. Here, we perform a systematic study of kinase–ligand binding modes for the human structural kinome at scale (208 kinases, 1777 unique ligands, and their complexes) by integrating chemical genomics and structural genomics data and by introducing a functional site interaction fingerprint (Fs-IFP) method. New insights into kinase–ligand binding modes were obtained. We establish relationships between the features of binding modes, the ligands, and the binding pockets, respectively. We also drive the intrinsic binding specificity and which correlation with amino acid conservation. Third, we explore the landscape of the binding modes and highlight the regions of “selectivity pocket” and “selectivity entrance”. Finally, we demonstrate that Fs-IFP similarity is directly correlated to the experimentally determined profile. These improve our understanding of kinase–ligand interactions and contribute to th&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b02041&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b02041" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b02041&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b02041" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b02041&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b02041" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b02041&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b02041&amp;href=/doi/10.1021/acs.jmedchem.5b02041" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b02041" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b02041" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (10 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b02041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b02041%26sid%3Dliteratum%253Aachs%26pmid%3D26929980%26genre%3Darticle%26aulast%3DZhao%26date%3D2016%26atitle%3DDelineation%2Bof%2BPolypharmacology%2Bacross%2Bthe%2BHuman%2BStructural%2BKinome%2BUsing%2Ba%2BFunctional%2BSite%2BInteraction%2BFingerprint%2BApproach%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D9%26spage%3D4326%26epage%3D4341%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292580" title="Binding modes">Binding modes</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291967" title="Cluster chemistry">Cluster chemistry</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/jmcmar.2016.59.issue-9/20160512/jmcmar.2016.59.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Targeted polypharmacology of kinases has emerged as a promising strategy to design efficient and safe therapeutics. Here, we perform a systematic study of kinase–ligand binding modes for the human structural kinome at scale (208 kinases, 1777 unique ligands, and their complexes) by integrating chemical genomics and structural genomics data and by introducing a functional site interaction fingerprint (Fs-IFP) method. New insights into kinase–ligand binding modes were obtained. We establish relationships between the features of binding modes, the ligands, and the binding pockets, respectively. We also drive the intrinsic binding specificity and which correlation with amino acid conservation. Third, we explore the landscape of the binding modes and highlight the regions of “selectivity pocket” and “selectivity entrance”. Finally, we demonstrate that Fs-IFP similarity is directly correlated to the experimentally determined profile. These improve our understanding of kinase–ligand interactions and contribute to the design of novel polypharmacological therapies targeting kinases.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/59/9"><issue-title>Computational Methods for Medicinal Chemistry</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57446" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57446" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein kinases are enzymes that phosphorylate other proteins (typical) and indeed other molecules (atypical) and hence play a vital role in signal transduction and cell differentiation.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Clinical evidence from anticancer studies have suggested that many cancers involve mutations harbored in the kinase leading to signal disruption.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> The same is true of a variety of other disease states, and hence the design of safe and efficient kinase inhibitors is an extensive area of pharmaceutical research. Since the first kinase-targeted drug, imatinib, was approved in 2001,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> 30 kinase inhibitors have been approved (through Nov 2015) by the U.S. Food and Drug Administration (FDA).<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6-8)</a> Notwithstanding this success, developing a kinase-targeted inhibitor with a desired selectivity profile across the human kinome remains a daunting task, as all kinases share a common catalytic domain and folding scaffold that binds ATP.<a onclick="showRef(event, 'ref4 ref9'); return false;" href="javascript:void(0);" class="ref ref4 ref9">(4, 9)</a> Thus, it is vital to uncover the basic principles that govern the kinase–ligand binding specificity across the kinome so that we can rationally design safe and efficient kinase-targeted (poly)pharmcological therapeutics.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">With the availability of an increasing number of diverse kinase–ligand complexes structures released by the Protein Data Bank (PDB),<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> many efforts have been devoted to establishing subtle binding modes by utilizing solved kinase–ligand complex structures. Kinase–ligand interaction patterns have been analyzed by directly comparing the binding similarity and the structural–activity relationship of kinase ligands.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16 ref17 ref18">(12-18)</a> These studies have facilitated rational drug design and uncovered the inhibition mechanisms for certain drug molecules.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27">(19-27)</a> However, many studies are only limited to the study of the binding mode of one kinase or a family of closely related kinases when a specific kinase is the focused target for drug design and discovery. A typical example of such structure-based drug design is crizotinib<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> for advanced non-small-cell lung cancer (NSCLC) with <i>ALK</i> rearrangement. Such work circumvents binding-pocket comparison among distantly related kinases and hence ignores selectivity across the kinome. Recently, several studies have analyzed the binding modes on a whole kinome scale using available X-ray structures of the human kinome.<a onclick="showRef(event, 'ref12 ref13 ref19 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref19 ref27 ref28">(12, 13, 19, 27, 28)</a> McGrego<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> has used a set of 220 X-ray kinase–ligand structures to training a common pharmacophore for the ATP binding sit, and showed that common similar local structures exist for diverse kinases. Similarly, Kinnings  et al.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> have used a geometric hashing method to undertake the structural comparison of ATP binding sites and validated drug cross-reactivity. Huang  et al.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> have provided a network analysis scheme of kinase selectivity potential at the ATP binding site by mapping the sequences of 518 human kinases onto the structural alignment of 116 kinases of known three-dimensional structures and established a systematic guideline to develop ATP-competitive inhibitors. However, global sequence alignment of kinases may not reflect their relationships at the ligand binding site.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Moreover, an increasing body of evidence suggests that the binding pocket of human kinases is not just in the region of the ATP binding site. A hydrophobic pocket adjacent to the ATP binding site and an allosteric pocket distal to the ATP binding site can also be used to achieve drug selectivity.<a onclick="showRef(event, 'ref17 ref29'); return false;" href="javascript:void(0);" class="ref ref17 ref29">(17, 29)</a> Thus, the detailed analysis of all regions of the binding pocket may shed new light on the binding selectivity of kinases. Recently Bryan  et al.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> established that the structural features of the kinase binding site are correlated with bound inhibitors by using a combinatorial clustering of kinase residue position subsets method. The residues of the binding site are defined using a type II inhibitor imatinib-bound kinase complex as a template. Then all other kinases are mapped to the template structure using a Pfam kinase family multiple alignment.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> van Linden  et al.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> have presented another systematic analysis of kinase–ligand interactions in all regions of the catalytic cleft of 1252 human kinase–ligand cocrystal structures that comprise only 190 kinases. The binding pocket of each human kinase was divided into three regions: front cleft, gate cleft, and back cleft. The ligand binding modes have been identified for each region. This divide-and-conquer analysis improves our understanding of kinase binding modes for several families of kinases. In spite of these advances, a whole kinome comparison and analysis of diverse kinases in all ligand-bound regions is needed to comprehensively study kinase inhibitor selectivity across the human kinome if kinase-targeted polypharmacology is to be successful.</div><div class="NLM_p">Until June 2015, 2383 kinase complex structures belonging to 208 distinct human kinases had been released by the PDB. The fast-growing numbers of 3D complex structures combined with abundant kinase bioassay data not only provide us with new opportunities to study the binding modes across the human structural kinome but also present new challenges in fully utilizing these diverse data sets. In short, novel computational methods are needed to support kinome-wide comparison and analysis. Previously, for one or more ligands to be screened against a single target, a docking method<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> would be applied to obtain the ligand bound complex and hence determine the binding mode. Another class of methods involves creating a pharmacophore model,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> in which the pharmacophore for the desired target protein is first trained. Then the trained pharmacophore is used to represent the structure–activity relationship (SAR) or to screen a compound library to select potential inhibitors. A promising method is to compare the binding modes of several kinases using a protein–ligand interaction fingerprint (IFP),<a onclick="showRef(event, 'ref25 ref27 ref33'); return false;" href="javascript:void(0);" class="ref ref25 ref27 ref33">(25, 27, 33)</a> which encodes the protein–ligand interfacial interaction as 1D fingerprints which discriminate the protein–ligand binding modes. The IFP method has been widely applied to docking pose analysis,<a onclick="showRef(event, 'ref25 ref34'); return false;" href="javascript:void(0);" class="ref ref25 ref34">(25, 34)</a> scoring function refinement,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> virtual screening,<a onclick="showRef(event, 'ref22 ref26 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref22 ref26 ref36 ref37">(22, 26, 36, 37)</a> binding site comparison,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> protein–ligand interaction mining,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> binding mode prediction,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and other applications.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> However, these studies have been limited to only one kinase or a family of high-homology kinases analyzed against multiple compounds. In these cases, the proteins can be aligned easily and the list of binding site residues, which constitute the binding pocket, are consistent for different protein structures. Thus, the encoded fingerprints for different compounds conserve the consensus sequences and can be compared easily. Here we extend the IFP method to determine the selectivity of kinase inhibitors across the complete human structural kinome, which includes many diverse families. For IFPs to be comparable across the complete human kinome requires the accurate alignment of diverse binding pockets for all kinases so that the consensus sequences of binding site residues can be retrieved for further comparison.</div><div class="NLM_p">It is not a trivial task to align a large number of diverse binding pockets using conventional global sequence or structure alignment methods.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40, 41)</a> An alternative method is to use ligand-binding site (aka functional site) comparison that can detect remote structural, functional, and evolutionary relationships across fold space.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Many methods have been developed to superimpose or detect binding sites (see recent reviews<a onclick="showRef(event, 'ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45">(43-45)</a> and herein). We have developed a sequence-order-independent profile–profile alignment (SOIPPA) method for ligand binding site comparison and implemented it in the software SMAP.<a onclick="showRef(event, 'ref40 ref42 ref46'); return false;" href="javascript:void(0);" class="ref ref40 ref42 ref46">(40, 42, 46)</a> SMAP has been successfully applied to side effect prediction,<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47, 48)</a> drug repositioning,<a onclick="showRef(event, 'ref49 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref49 ref50 ref51">(49-51)</a> polypharmacology drug design,<a onclick="showRef(event, 'ref52 ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref52 ref53 ref54 ref55">(52-55)</a> and more.<a onclick="showRef(event, 'ref40 ref42 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref40 ref42 ref56 ref57">(40, 42, 56, 57)</a> SMAP extracts aligned consensus sequence motifs, which consist of a list of binding site residues essential to derive the IFP. Moreover, the binding site residues are not only sequence-order-independent but also spatially matched providing pairwise aligned binding pockets. In this study, we further improve our SMAP method through combination with the protein–ligand IFP method. In our improved scheme, first, a comparable list of binding site residues is retrieved by superposing kinase complex structures using SMAP. Then the IFP is obtained by encoding the superposed function-site sequence motif for every kinase–ligand structure. Thus, the sequence order independent aligned function-site IFP strategy (termed Fs-IFP) provides a new solution to the comparison and analysis of diverse ligand binding modes (for details see the <a class="ref internalNav" href="#sec4" aria-label="Method">Method</a> section).</div><div class="NLM_p last">In this paper, we use Fs-IFP to carry out the global analysis of ligand binding modes across the human kinome at an unprecedented scale and complexity (208 kinases, 1777 unique ligands, and their complexes) by integrating chemical genomics and structural genomics data. With what follows we first introduce the details of the function-site interaction fingerprint (Fs-IFP) approach. Second, we provide a detailed analysis of the relationship of ligand similarity, protein similarity, and Fs-IFP similarity. Third, we apply Fs-IFP to uncover the binding modes across the human structural kinome. Finally, we reveal the correlation of the binding mode and experimentally determined kinase binding profile. These findings provide new insights that can be applied to the in silico design of polypharmacology compounds targeting kinases.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Fs-IFP Data Set of Human Protein Kinase Structural Database (HPK-SD)</h3><div class="NLM_p">We obtained an Fs-IFP data set derived from a three-step protocol (see <a class="ref internalNav" href="#sec4" aria-label="Method">Method</a> Section) as shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>d, and the complete data set is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf" class="ext-link">Supporting Information Table S1</a>. In the Fs-IFPs data set, 2383 superimposed binding pockets are shown using the encoded bit strings. Each row represents the binding mode of one kinase–ligand complex structure using the bit strings. Each column represents the specific protein–ligand interactions of the same spatial position among the aligned kinase–ligand complex structures.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Flowchart describing the Fs-IFP method. (a) The protein kinases that have released apo and holo structures of catalytic domains are marked with blue and red filled circles, respectively, in the evolutionary tree.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> (b) PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYU">3BYU</a>, which is used as a reference structure for all kinases. (c). Superimposed binding site. Each row represents a sequence-order-independent motif comprising the binding pocket for a given ligand-bound structure. Each column represents the corresponding amino acid residues at the same spatial position of the binding pocket for all ligand-bound structures. The D-NULL mark means there is no corresponding residue at the specific position of the given ligand-bound structures. (d) Encoded binding site using the Fs-IFP approach. Each row represents the Fs-IFP bit-strings for one given ligand-bound structure. Each column including 7 bits represents an Fs-IFP code of one residue of the binding site by using seven types of interactions. The same column means that the residue matches the same position in 2383 superposed binding pockets. A bar is used if there is no typical interaction between the specific amino acid and the ligand in the given ligand-bound structure. The bit is marked as “1” if the specific type of interaction exists, otherwise marked as “0” if the type of interaction does not exist between the specific amino acid and the ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Distribution of Ligand-FP, Kinase-FP, and Fs-IFP Similarity</h3><div class="NLM_p">In HPK-SD, there are 2383 ligands, which contain 1777 unique small molecules. The top 20 most common ligands are shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a. It is not surprising that the top 3 most common molecules are ATP, ANP, and ADP because ATP is the natural ligand, and ANP and ADP are similar to ATP. The distribution of molecular weights for all ligands was shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b. The molecular weights of more than 80% of the ligands are between 300 and 550 Da, in keeping with Lipinski’s rules.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Distribution of the ligand characteristics in 2383 cocrystallized kinase structures: (a) distribution of the top 20 frequently occurring ligands; (b) distribution of the molecular weight of the ligands in all cocrystallized kinase structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">On the basis of the ECFP fingerprint, the distribution of chemical structural similarity of ligands (ligand-FP) is shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> in red. We also compared the similarity of the binding sites of 2383 kinase structures in the HPK-SD (kinase-FP) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>, blue). In order to compare the distribution of ligand-FP similarity, kinase-FP similarity, and Fs-IFP similarity with each other, the distribution of the Fs-IFP similarity is shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> in green.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Distribution of Tanimoto coefficient (TC) value for the fingerprints similarity of the ligands (ligand-FP) found in HPK-SD (red), kinase pockets (kinase-FP) of the HPK-SD (blue), and the kinase–ligand interaction fingerprints (Fs-IFP) of the kinase–ligand complexes (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The distribution of the ligand-FP similarity has a mean TC of 0.12, a standard derivation of 0.085, and a long tail indicating there are some highly similar ligands in the HPK-SD, which is consistent with the results shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a. About 10 000 pairwise comparisons of ligand-FPs have TC values of 1.0 implying identical ligands, as our statistics show in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a. The top 5 redundant ligands are ANP, ADP, ATP, ACP, and STU. It will be interesting to subsequently analyze the binding modes of the Fs-IFPs based on the same ligands. However, the distribution of the ligand-FP similarity has a peak value when TC = 0.1, and the peak value is lower than that of the binding site or that of the kinase–ligand interaction as described in the following analysis.</div><div class="NLM_p last">The distribution of the similarity of the kinase binding sites as shown by kinase-FP (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> in blue) is close to a normal distribution with a mean of 0.18 and a standard derivation of 0.066. Given that the kinase structure has a highly similar scaffold,<a onclick="showRef(event, 'ref18 ref27'); return false;" href="javascript:void(0);" class="ref ref18 ref27">(18, 27)</a> including one N-lobe consisting of five β sheets (β1−β5) and one helix (C-helix), and a C-lobe consisting of four β sheets (β6−β9) and additional helixes (see labels in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>b) and connected with a hinge-loop, the discriminatory power of kinase-FP is significant. Although the kinase scaffold is similar, the TC value of binding site fingerprints takes the difference of both amino acids and the direction of side chains into account. As a result, the similarity of the binding site is not high. The distinct property of the binding sites as described here by kinase-FP makes designing more selective kinase inhibitors possible. Different from the distribution of ligand similarity and binding-site similarity of kinases, the distribution of Fs-IFP similarity has a high mean value of 0.41 and a large standard deviation of 0.32. Comparatively, the average value of the similarity of the Fs-IFP is higher than that of ligand FPs or kinase pocket FPs across the human kinome. This implies that chemicals with diverse scaffolds achieve similar binding modes. It is noted that when TC = 0, there are about 12 000 pairs of Fs-IFPs. This suggests that pairs of protein–ligand interactions are in different regions of the kinase binding pocket, which do not overlap each other, or the pairs of protein–ligand interactions have different binding modes. For example, the ligand in PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H8F">1H8F</a> (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>d) and the ligand in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q4C">3Q4C</a> (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>h) bind to different locations of the binding pocket as shown in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>i. The ligand in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q4C">3Q4C</a> is at the entrance of ATP binding pocket clustered as cluster 5d in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>i. The ligand in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H8F">1H8F</a> binds to the allosteric site clustered as cluster 4 in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>i and interacts with the activation loop and the head of the C-helix. In addition, we noted that when the TC is larger than 0.75, the mean of the Fs-IFP similarity distribution is relatively lower than that of the ligand-FP similarity distribution. This implies that the binding mode is not highly similar even if there are many highly similar ligands. Thus, the same ligand may potentially accommodate a different pocket with different binding modes.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Correlation of Ligand-FP, Kinase-FP, and Fs-IFP</h3><div class="NLM_p"><a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> shows the correlation between the ligand structure, kinase binding pocket, and kinase–ligand binding mode by analyzing the relationship of pairwise similarity of ligand-FPs, kinase-FPs, and Fs-IFPs.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Correlation of the Tanimoto coefficient (TC) for ligand FP, kinase-pocket FP, and kinase–ligand IFP similarity using a violin plot: (a, b) correlation of the TC of the ligand FP similarity and the kinase pocket FP similarity; (c, e) correlation of the TC of the ligand FP similarity and the kinase–ligand IFP similarity; (d, f) correlation of the TC of the kinase pocket FP similarity and the kinase–ligand IFP similarity. In all panels, the width of the violin box is proportional to the number of TCs. The black bars range from 25% to 75%, and the white points on the bars represent the medians. Distributions (%) of ligand-FP, kinase-FP, and Fs-IFP in the corresponding bin of similarity are shown at the upper portion of every panel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As expected, similar pockets bind similar ligands. The correlation of the kinase pocket and the ligand characteristics is shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a and <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b, respectively. From the point of view of the ligand (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b), when the TC value of ligand similarity changes from 0.0 to 1.0, the distribution of the TC values of the kinase pocket similarity shows no significant change, with all medians approximately 0.19. This suggests that the ligand similarity cannot directly infer the binding pocket similarity. In contrast, there is some correspondence between kinase pocket similarity and ligand similarity for the most similar pockets as shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a. For the 0.12% of pockets with similarity larger than TC = 0.5, the median of the TC for ligand similarity increases from 0.3 to 1.0 with the increase in the TC of pocket similarity. The correlation coefficient for pairs within this range is R2 = 0.28. This result is consistent with the conventional view that similar pockets bind similar ligands.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a></div><div class="NLM_p last">We further characterize how the protein–ligand binding modes are correlated with the characteristics of ligands and kinase pockets. <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c and <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>d show the correlation of the ligand-FPs and the Fs-IFPs and that of the kinase pocket-FPs and the Fs-IFPs, respectively. The same basic features are observed in the two panels. The medians of both ligand-FP similarity and kinase pocket-FP similarity increase slowly with the increase of Fs-IFP similarity. This suggests that Fs-IFP is to some degree correlated not only with the ligand characteristics but also the kinase pocket properties. In <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c, the medians of the TC value of ligand similarity slowly increase along with the TC value of Fs-IFP similarity. For the 1.04% of Fs-IFP’s with similarity larger than TC = 0.7, the median of the TC for ligand similarity increases from 0.2 to 0.6 with the increase in the TC of Fs-IFP similarity. The correlation coefficient for pairs within this range is R2 = 0.24. In addition, at each range of the TC value of Fs-IFP similarity, the TC value of ligand similarity has an extensive distribution from 0.0 to 1.0 as shown in the direction of <i>Y</i>-axis. This suggests that significantly different ligands (low ligand-FP value) may bind to the kinase pocket with similar binding modes (high Fs-IFP value). Alternatively, ligands may be promiscuous and adaptively bind to similar binding pockets. Moreover, in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>e, where the ligand-similarity TC is 1.0, the median of the TC value for Fs-IFP is approximately 0.5. This implies that the same ligand can bind to proteins with different binding modes. Our data set includes several redundant ligands, such as ANP, ADP, ATP, and ACP, as aforementioned. They are highly similar to each other but have dissimilar binding modes. We can infer that the different binding modes enable these ligands to better accommodate different pockets. The correlation coefficient for pairs within this range is R2 = 0.23. It follows, and is confirmed by <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>f, that similar kinase pockets have similar Fs-IFPs. There is some correspondence between kinase pocket similarity and Fs-IFP similarity for similar pockets, as shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>f. The correlation coefficient for pairs within this range of 0–1.0 is R2 = 0.23, and the violin boxes of different bins have the larger overlap. We also noted that when TC = 1.0 of kinase-FP, there is 0.002% of Fs-IFP binding modes with the different TC from 0.4 to 1.0 for Fs-IFP, which implies that different ligands can bind the highly similar pockets with different binding modes.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Structural and Evolutionary Characteristics of Kinase–Ligand Interactions</h3><div class="NLM_p">In the process of encoding the Fs-IFP, each 7 bits represent 7 different contacts for one residue (see <a class="ref internalNav" href="#sec4" aria-label="Method">Method</a> section). For HPK-SD, there are 37 954 contacts between the ligand and the kinase in the kinase–ligand binding interfaces, and these can be characterized as follows. They include 83.2% apolar (van de Waals) contacts, 8.82% contacts where the H-bond of the protein is a donor, 5.91% where the H-bond of the protein is an acceptor, 1.07% contacts of aromatic edge to face, 1.0% contacts of aromatic face to edge, and 0.01% electrostatic contacts, respectively, as shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a. On average, there are 16 contacts for each kinase–ligand interaction. The 16 contacts forming the kinase–ligand interaction typically consist of 13 apolar, 1 aromatic, and 2 H-bond contacts, respectively. We counted the number of interactions that occur on each residue of the superimposed protein–ligand pocket for different types of interactions. As shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>b, the top 5 apolar interactions are located at positions β2, β3, hinge (two), and β7. The top 3 aromatic interactions are located at the hinge (two) and the DFG peptide. The top 4 H-bond contacts are located at the positions β3, hinge (two), and the DFG peptide. These observations suggest that β2, β3, β7, hinge, and DFG provide the majority of the protein–ligand interactions.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Distribution of interaction types at the kinase–ligand binding interface: (a) distribution of the contribution of six interaction types at the kinase–ligand binding interface; (b) distribution of occurrences of the six interaction types for each residue location at the binding interface. The <i>y</i>-axis is the number of interaction occurrences for the corresponding amino acids given along the <i>x</i>-axis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Correlation between interactions and the amino acid conservation at each amino acid position constituting the binding interface: (a) distribution of interaction occurrence for each amino acid position; (b) the corresponding distribution in (a) mapped to the template structure; (c) conservation of amino acids at the binding site; (d) correlation between residue conservation and the distribution of interactions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next we inspect which amino acids contribute to specific contacts. At each location the number of interactions is shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>a. The amino acids involved in the top 6 most common interactions are located at positions β2, β3, hinge (two), β7, and the gatekeeper (the first residue of hinge region). Their locations are marked using the top 6 largest balls, as shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>b, where the size of the purple spheres is proportional to the number of interactions. The 6 amino acids that form the top 6 most conserved interactions are located at the binding site of adenosine from ATP and form the core of the binding pocket. The residues at the regions of β2, β3, hinge, and DFG also have higher hit frequencies than other locations and thus make major contributions to the binding. Typically there are one to three hydrogen bond contacts between the adenosine and the main-chain atoms of the hinge residues. The aromatic ring of adenosine forms aromatic interaction with an amino acid on the sheet of β3 and an amino acid around DFG<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and is in keeping with the observation that most of the current kinase-targeted drugs are ATP-competitive.<a onclick="showRef(event, 'ref17 ref29'); return false;" href="javascript:void(0);" class="ref ref17 ref29">(17, 29)</a> Notwithstanding these similarities, other amino acids contribute to different binding mode in different structures and may play important roles in establishing selectivity for targeted polypharmacology.</div><div class="NLM_p last">We further delineated what kinds of residues contribute to the binding contacts at specific locations in order to reveal the role of residue conservation in binding specificity. At every residue location of the superimposed binding sites, we inspected the constituent amino acid residues and calculated their ratios. The amino acid with the maximum ratio is defined as the conserved residue at that specific position. The results are shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>c. The top 3 most conserved residues are Val, Ala, and Leu, located at β2, β3, and β7, respectively. They have the highest interaction frequencies marked as ①, ②, and ④ in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>a. Residue conservation is linearly correlated with the frequency of interactions on each residue as shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>d (correlation coefficient 0.89). The location corresponding to these three most conserved amino acids forms the mouth of the core binding pocket, which traps the ligand, as shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>b. Beyond that the conservation in the hinge region is also relatively high. Hinge interactions mainly take place through the main-chain atoms of the hinge residues.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Mutation of hinge residues rarely affects ATP binding<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> but frequently changes the conformation of the kinase hydrophobic pocket resulting in drug resistance,<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> such as the T315I mutation in Abl kinase and T670I mutation in KIT kinase.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> The hinge region may be responsible for subtle selectivity resulting from different side chain conformations and types of kinases. The C-helix region and the catalytic region have lower conservation as shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>c and may be responsible for the specific selectivity of a ligand. The catalytic region is distal to the ATP site, and the C-helix contributes to forming the hydrophobic pocket and the allosteric site. Recently these regions have been increasingly utilized to design allosteric inhibitors and type II inhibitors to achieve improved selectivity.<a onclick="showRef(event, 'ref15 ref63'); return false;" href="javascript:void(0);" class="ref ref15 ref63">(15, 63)</a></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Clustering the Fs-IFP Data Set</h3><div class="NLM_p">To further explore the kinase binding site, we cluster the Fs-IFPs of complexes (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a>) and obtain a hierarchical clustering tree as shown in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>. Setting the Fs-IFP similarity as the distance for clustering reveals five clusters, marked 1–5. All structures in each cluster are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf" class="ext-link">Supporting Information Table S2</a>. Among the five clusters, the binding modes are completely different. Such information may provide critical clues to targeted polypharmacology by utilizing binding pocket promiscuity and selectivity. To further show the spatial differences between binding pockets, we analyzed the binding modes for the five clusters individually. Approximately 90% of the structures in cluster 5 have a common character, i.e., have at least one hydrogen-bond interaction with hinge region. To further reveal the interaction characters, we divide cluster 5 into 4 subclusters: cluster 5a–d. We randomly choose eight PDB entries one from each cluster/subcluster to illustrate the protein–ligand interactions: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ITJ">4ITJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A5U">2A5U</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PXF">3PXF</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H8F">1H8F</a> are chosen to represent clusters 1–4, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PRZ">3PRZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BYG">1BYG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ZZ2">1ZZ2</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q4C">3Q4C</a> are chosen to represent clusters 5a–d. Their 3D structures are shown in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Rectangular tree layout of the Fs-IFP data set for HPK-SD. Five distinct clusters were found. Cluster5 was further divided into 5 subclusters: cluster 5a–d.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Binding modes for each cluster 1–4 and 5a–d based on complete linkage clustering: (a–h) for cluster 1–4 and 5a-d using PDB entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ITJ">4ITJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A5U">2A5U</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PXF">3PXF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H8F">1H8F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PRZ">3PRZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BYG">1BYG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ZZ2">1ZZ2</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q4C">3Q4C</a> as the representative structure, respectively; (i) site locations for the binding regions for the eight clusters.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>a shows the binding mode for cluster 1 where the ligand is located within the hydrophobic cavity at the back of the ATP binding site, also shown in red in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>i. Gatekeeper residues control the size of the cavity and the interaction with the ligand. The gatekeeper residue is different in different kinases, for example, threonine in EGFR but phenylalanine in CDK-2, providing the opportunity for high selectivity when the cavity is utilized. The success of the first generation inhibitors, including erlotinib, gefitinib, and lapatinib<a onclick="showRef(event, 'ref8 ref64'); return false;" href="javascript:void(0);" class="ref ref8 ref64">(8, 64)</a> for non-small-cell lung cancer targeted EGFR,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> is thought to make use of the interactions with the hydrophobic cavity.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Because of drug resistance, especially through mutation of the gatekeeper T790M,<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> the new generation of EGFR inhibitors, such as <i>N</i>-[2-[2-(dimethylamino)ethylmethylamino]-4-methoxy-5-[[4-(1-methlindol-3-yl)pyrimidin-2-yl]amino]phenyl]prop-2-enamide (AZD9291),<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a><i>N</i>-[3-[[2-[4-(4-acetylpiperazin-1-yl)-2-methoxyanilino]-5-(trifleoromethyl)pyrimidin-4-yl]amino]phenyl]prop-2-enamide (CO-1686),<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> and <i>N</i>-[3-[5-chloro-2-[2-methoxy-4-(4-methylpiperazin-1-yl)anilino]pyrimidin-4-yl]oxyphenyl]prop-2-enamide (WZ4002),<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> are designed to overcome the T790M mutation. Existing examples of the key gatekeeper pocket can be utilized to achieve the desired selectivity of kinase targeted inhibitors.</div><div class="NLM_p"><a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>b shows the location of cluster 2 in the center of the kinase pocket as shown in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>i. The ligand has three direct interactions, with the Asp of the DFG tripeptide, Lys on the “roof” of β3, and Glu on the C-helix. The location of the three interactions is just proximal to the ATP site and is at the catalytic center of the kinase, where the ATP is catalyzed to ADP.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> The occupation of this space inhibits the catalytic activity of the kinase as utilized by the FDA approved drugs, vemurafenib<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> and ceritinib.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> As a BRAF mutant (V600E) inhibitor to treat the late-stage melanoma, vemurafenib was designed using fragment-based lead discovery. Its selectivity was achieved by adding the sulfonamido group at the location of cluster 2. Similarly, ceritinib, a drug to treat ALK-positive metastatic non-small-cell lung cancer, also applied the same strategy of using a sulfonyl group.</div><div class="NLM_p"><a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>c shows the location of cluster 3, which accommodates the space at the back of the ATP binding site, which is between β5 and the C-helix. There are H-bond interactions with the residue on β3 and the residue on the C-helix. Between the β5 and C-helix, typically, there is not too much space available. However, when the kinase is in the C-helix-out conformation such as in PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK">1XKK</a> or <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BBT">3BBT</a>, the location of cluster 3 can be exposed and has been exploited.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> The allosteric inhibitors 5-bromo-<i>N</i>-(2,3-dihydroxypropoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (PD318088)<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> and <i>N</i>-[5-[3,4-difluoro-2-[(2-fluoro-4-iodophenyl)amino]phenyl]-1,3,4-oxadiazol-2-yl]-4-morpholineethanamine (PD334581) (type III inhibitors)<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> have been reported to explore this location on cluster 3.</div><div class="NLM_p"><a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>d shows a binding mode between the head of the C-helix and the activation loop. The location of the subpocket is away from the ATP binding site. The tail of the FDA approved drug ponatinib<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> occupies this part of the subpocket. Presumably this subpocket can be further explored to improve the selectivity of the primary target of ponatinib, BCR-Abl, as well as other kinase targets.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a></div><div class="NLM_p">For the aforementioned four clusters, a common pattern is that there is no interaction with the hinge region. These subpockets are located at the back of the ATP binding site and adjacent to each other. These regions are often used to improve selectivity, and the regions including cluster 1–3 are often called as the “selectivity pocket” to differentiate them from the ATP binding pocket. Typically, kinase inhibitors use these regions to achieve selectivity. Type I inhibitors often utilize the hydrophobic subpocket at the site of cluster 1 to improve the selectivity; types II and III occupy more space at the regions of cluster 1–4 to achieve the specific selectivity. Conversely, it is very challenging to design desired inhibitor using the regions corresponding to cluster 1–4 because these regions are more flexible than the ATP binding site, depending on the conformational change of the inactive kinase state (DFG out or C-helix out).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Currently, most kinase inhibitors are type I inhibitors, which are ATP-competitive and occupy the ATP binding site as shown in cluster 5 in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> and <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>e–h. Besides the common hinge interactions, they have distinguishing characteristics and thus divided into four subclusters.</div><div class="NLM_p"><a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>e shows the binding mode of cluster 5a, which interacts with the hinge region through two H-bonds. Moreover, the ligand interacts with the Lys and Glu of the C-helix. This interaction pattern is similar to that of cluster 2. In fact, cluster 5a is close to cluster 2 as described for the cluster 2 (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>b). The important difference is the hinge interaction or not. Thus, to achieve the selectivity of cluster 2, fragments that occupy the location of the cluster 5 can be added to the inhibitor of cluster 2.</div><div class="NLM_p"><a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>f shows the typical binding mode of the ATP-competitive site of cluster 5b. Besides the interactions of the hinge region, there is an interaction with a residue on the activation loop. To achieve selectivity, the ligand at this location should be extended to other subpockets, such as the adjacent subpockets: cluster 2, cluster 5a, cluster 5c, and cluster 5d.</div><div class="NLM_p"><a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>g shows the binding site of cluster 5c, which has interactions with the activation loop and the DFG tripeptide. Similar to cluster 5b, the ligand has a longer tail and extensively utilizes the distal space of the ATP binding site. It will affect the phosphorylation of the substrate because the tail is close to the binding site of the substrate. For example, the approved drug afatinib is selective in this way (see the cocrystal structure PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G5J">4G5J</a>).<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a></div><div class="NLM_p"><a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>h illustrates the binding mode of cluster 5d at the entrance of the binding pocket adjacent to the adenosine binding site of the ATP binding pocket, as shown in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>i. Besides the hinge interaction, the ligand has interactions with the loop following the hinge residues. Because the entrance is located between the loop following the hinge region and the P-loop, different kinases present diverse characteristics. This “selectivity entrance” can be utilized to achieve improved selectivity compared to the ATP binding site.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Approximately 57% of the approved kinase-targeted drugs take advantage of this selectivity entrance.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> In 2013 two irreversible kinase inhibitors ibrutinib<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> and afatinib<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> were approved that utilize the selectivity entrance.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> These inhibitors covalently inhibit their targets and compete with ATP. The covalent bond is formed by a Michael addition with a Cys residue of the kinase.<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> The Cys residues involved in the two approved drugs are both located at the selectivity entrance.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The third generation of EGFR inhibitors for NSCLC, osimertinib<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> and rociletinib,<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> are also irreversible, and both involved the Cys residue at the entrance. The Cys residue has been a hot-point amino acid for the design of irreversible kinase inhibitors and significantly improves the binding affinity.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a></div><div class="NLM_p">From <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>a to <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>h, every cluster has a different binding site as shown in the cartoon panel of <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>i, which provides an overview of the relative locations in the kinase pocket. The promise is to design kinase inhibitors by choosing a type of binding mode or combining binding modes. Indeed, potential inhibitors have been developed by targeting the different position from the typical ATP site.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> For example, there are 30 kinase-targeted drugs approved by FDA through Nov 2015, and one drug apatanib<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> approved by the Chinese FDA in Dec 2014. Apatanib has a typical binding mode for a type I inhibitor and occupies clusters 2, 5a, 5b, and 5d. The binding modes of 30 drugs approved by the FDA are summarized in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> based on their binding positions. The 3D binding mode can be retrieved and checked based on the corresponding PDB<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> entries in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> or from the reviews of FDA approved drugs.<a onclick="showRef(event, 'ref64 ref81'); return false;" href="javascript:void(0);" class="ref ref64 ref81">(64, 81)</a> The types of drugs were also identified in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> based on the binding position and the conformational state of the kinase.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Distributions of the Binding Modes for FDA-Approved Kinase-Targeted Drugs<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="8" align="center">involved clusters for the binding modes</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">approved inhibitor</th><th class="colsep0 rowsep0" align="center">approved year</th><th class="colsep0 rowsep0" align="center">PDB entry</th><th class="colsep0 rowsep0" align="center">1</th><th class="colsep0 rowsep0" align="center">2</th><th class="colsep0 rowsep0" align="center">3</th><th class="colsep0 rowsep0" align="center">4</th><th class="colsep0 rowsep0" align="center">5a</th><th class="colsep0 rowsep0" align="center">5b</th><th class="colsep0 rowsep0" align="center">5c</th><th class="colsep0 rowsep0" align="center">5d</th><th class="colsep0 rowsep0" align="center">inhibitor of type</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">IMATINIB</td><td class="colsep0 rowsep0">2001</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BKJ">4BKJ</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">II</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">GEFITINIB</td><td class="colsep0 rowsep0">2003</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I22">4I22</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">ERLOTINIB</td><td class="colsep0 rowsep0">2004</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HJO">4HJO</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">SORAFENIB</td><td class="colsep0 rowsep0">2005</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ASD">4ASD</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">II</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">DASATINIB</td><td class="colsep0 rowsep0">2006</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QLG">3QLG</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">SUNITINIB</td><td class="colsep0 rowsep0">2006</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y7J">2Y7J</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">NILOTINIB</td><td class="colsep0 rowsep0">2007</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GP0">3GP0</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">II</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">LAPATINIB</td><td class="colsep0 rowsep0">2007</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK">1XKK</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">PAZOPANIB</td><td class="colsep0 rowsep0">2009</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X9F">2X9F</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">VANDETANIB</td><td class="colsep0 rowsep0">2011</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IVU">2IVU</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">RUXOLITINIB</td><td class="colsep0 rowsep0">2011</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VS7">3VS7</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">CRIZOTINIB</td><td class="colsep0 rowsep0">2011</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZBF">3ZBF</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">VEMURAFENIB</td><td class="colsep0 rowsep0">2011</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OG7">3OG7</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">TOFACITINIB</td><td class="colsep0 rowsep0">2012</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LXN">3LXN</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">BOSUTINIB</td><td class="colsep0 rowsep0">2012</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OTW">4OTW</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">CABOZANTINIB</td><td class="colsep0 rowsep0">2012</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U6J">3U6J</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">II</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">PONATINIB</td><td class="colsep0 rowsep0">2012</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C8B">4C8B</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">II</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">REGORAFENIB</td><td class="colsep0 rowsep0">2012</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WZE">3WZE</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">II</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">AXITINIB</td><td class="colsep0 rowsep0">2012</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGC">4AGC</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">IBRUTINIB</td><td class="colsep0 rowsep0">2013</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IFG">4IFG</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">AFATINIB</td><td class="colsep0 rowsep0">2013</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G5J">4G5J</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">TRAMETINIB</td><td class="colsep0 rowsep0">2013</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP1">3PP1</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">III</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">DABRAFENIB</td><td class="colsep0 rowsep0">2013</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CQE">4CQE</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">NINTEDANIB</td><td class="colsep0 rowsep0">2014</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C7Q">3C7Q</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">IDELALISIB</td><td class="colsep0 rowsep0">2014</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XE0">4XE0</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">CERITINIB</td><td class="colsep0 rowsep0">2014</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MKC">4MKC</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">PALBOCICLIB</td><td class="colsep0 rowsep0">2015</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2EUF">2EUF</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">LENVATINIB</td><td class="colsep0 rowsep0">2015</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WZD">3WZD</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">II</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">COBIMETINIB</td><td class="colsep0 rowsep0">2015</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AN2">4AN2</a></td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">III</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">OSIMERTINIB</td><td class="colsep0 rowsep0">2015</td><td class="colsep0 rowsep0"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZAU">4ZAU</a></td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">√</td><td class="colsep0 rowsep0">I</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">The “√” means the interaction exists in the corresponding regions of the binding pocket (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>i).</p></div></div></div><div class="NLM_p">Many drugs targeting kinases do so via the ATP binding site. Also the “selectivity pocket” and the “selectivity entrance” are targeted frequently. In the previous section we have shown that the amino acids involved in the binding of the adenosine of ATP are conserved, as shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>b,c. Thus, it is possible to take advantage of a combination of different binding modes on different subpockets to achieve the desired selectivity profile using a fragment-based strategy.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> All existing drugs utilized more than one binding region to achieve their specificity (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). It is noted that trametinib and cobinetinib,<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> novel allosteric inhibitors, do not use the ATP binding interaction with the hinge but use the cluster 1–3 and cluster 5c regions. This type of approved drug may become more prevalent in the future. Overall, the five different spatial locations not only reflect pocket promiscuity but ligand adaptability. There is no distinct boundary between clusters, which implies that the whole kinase pocket should be used to refine the design of the inhibitor in order to obtain the optimal specificity.</div><div class="NLM_p last">The flexibility of protein–ligand binding is another important factor in designing a desired inhibitor. Typically, type II inhibitor induces the dramatic change of active loop to rearrange the inhibitor-stabilized conformation.<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> The induced conformational change is not unique for type II inhibitors, and an example is axitinib, which induced different binding modes in different kinase structures (e.g., in PDB entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGC">4AGC</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TWP">4TWP</a>).<a onclick="showRef(event, 'ref88 ref89'); return false;" href="javascript:void(0);" class="ref ref88 ref89">(88, 89)</a> In wild-type VEGFR, axitinib inhibits the target with a type II binding mode and DFG-out state as shown in the crystal structure (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGC">4AGC</a>). In the wide-type VEGFR, the gatekeeper is a small amino acid Val916; moreover the tail of axitinib consists of a phenyl fragment, which has a close interaction with Val916. Recently, a study showes axitinib can inhibit the T315I mutant bcr-abl1 with a distinct binding conformation.<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> Here the T315I mutation blocked axitinib from entering the region of the selectivity pocket; thus axitinib induced a new type I binding mode. The mutation prevents the phenyl fragment from entering the hydrophobic subpocket, resulting in a flip of the phenyl group to accommodate the mutation, as shown in the X-ray structure (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TWP">4TWP</a>). This poses another challenging task in designing the desired selectivity.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Analysis of the Correlation between the Fs-IFP Data Set and the Experimentally Determined Kinase Binding Profile</h3><div class="NLM_p">We further evaluate the correlation of the Fs-IFP and the experimentally determined kinase binding profile from KINOMEscan<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> data taken from the HMS LINCS database.<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> We use the common small molecules in the KINOMEscan data and the ligands from HPK-SD to build relationships between the experimental activity values and the binding mode of Fs-IFP (see detailed procedures in the <a class="ref internalNav" href="#sec4" aria-label="Method">Method</a> section). <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>a shows the pairwise correlation of the thermodynamic constant <i>K</i><sub>d</sub> and Fs-IFP. The pairwise similarity of <i>K</i><sub>d</sub>, which is measured by Euclidean distance, is to a certain degree correlated with the binding mode of Fs-IFP as indicated from the trend in the curve fitting first-order exponential decay. The correlation coefficient is 0.40. When the pairwise similarity of Fs-IFP is higher than 0.5, the binding profile determined by the experimental <i>K</i><sub>d</sub> tends to be similar and vice versa. <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>b shows the relationship between percent of control (PoC) and Fs-IFP and trends in a similar way to <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>a. The correlation coefficient is 0.28. The two trend curves both show that the more similar are the binding modes, the more consistent the experimental values are with each other. Thus, Fs-IFP could be a useful in silico tool for virtual drug development, since Fs-IFP is related to the experimental profile. For a given drug as a reference, if a newly designed inhibitor has a high similar Fs-IFP, such as more than 0.5, it may imply the inhibitor has similar activity to the reference drug. In <a href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>, we also show that there is no significant correlation between the kinase pocket IFP and the experimental profile, further validating that Fs-IFP could be a better measure than pocket similarity to drive drug design.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Correlation between Fs-IFP and experimentally determined kinase binding profiles. The curves were obtained by using fit first-order exponential decay. (a) Correlation of the pairwise difference of the <i>K</i><sub>d</sub> values and pairwise similarity of Fs-IFPs. (b) Correlation of the pairwise difference of the PoC values and pairwise similarity of Fs-IFPs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">It is worth noting that the encoded Fs-IFP interaction excludes interactions from any water molecules and metal ions in the binding site. Water molecules, metal ions, and molecular dynamics are important aspects of the ligand interaction. In future work the intent is that Fs-IFP will better mimic the realistic kinase–ligand interaction by including water molecules, metal ions, and molecular dynamics. In so doing the Fs-IFP method will better discriminate binding modes, subtle interactions, and conserved interactions.</div><div class="NLM_p last">In summary, understanding the structural basis of ligand binding selectivity and promiscuity across the human kinome provides critical information on drug repurposing and discovering targeted polypharmacology. We have demonstrated that Fs-IFP is a potentially powerful tool to study ligand binding profiles across the kinome. Currently Fs-IFP is encoded using the residues that cover all subpockets. Since a ligand may only occupy one or more, but not all, subpockets, an Fs-IFP that is based on subpockets may offer greater predictive power in studying protein–ligand interactions on a genome-wide scale. By integrating Fs-IFP with other computational tools such as historeceptomic fingerprints for druglike compounds<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> and multiple omics data, in the future it should be possible for us to develop more powerful predictive models for polypharmacology prediction to support drug repositioning and target identification.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27148" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27148" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this paper, we have presented an Fs-IFP method, which combines sequence-order-independent binding site alignment with a protein–ligand IFP technique to identify all interaction patterns for human kinases. We characterized the binding modes across the human structural kinome using sets of comparable Fs-IFPs and gained new insights into the structural discriminants of binding specificity and promiscuity. Moreover, we have analyzed the relationships between Fs-IFPs, ligand-FPs, and kinase-FPs. Our analysis supports the notion that similar binding pockets bind similar ligands with similar interaction patterns. However, similar ligands may bind to diverse binding pockets with different interaction patterns. The binding pocket clustering based on Fs-IFP similarity reveals several distinct subpockets. The residue conservation in these subpockets is highly correlated with specific protein–ligand interactions. The selectivity of inhibitors can be achieved by combining the different subpockets. We should pay more attention to the “selectivity pocket” and “selectivity entrance” in order to obtain the desirable selectivity of a specific target or targeted polypharmacology. Furthermore, the Fs-IFP is correlated with kinome binding profiles from experimental data. Thus, the comparison of interaction patterns using the Fs-IFP across the kinome will facilitate the detailed understanding of the kinase binding mode and rational design of kinase inhibitors with desirable selectivity profiles and SAR. In principle, the Fs-IFP method can be applied to recognize protein–ligand interaction pattern across all of fold space, thereby providing a potentially powerful tool for revealing polypharmacology profiles of proteins beyond kinases.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Method</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13423" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13423" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Fs-IFP Method</h3><div class="NLM_p">Our Fs-IFP method follows three steps as shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Step 1: To Prepare the Data Set of Human Protein Kinase Ligand-Bound Structures</h4><div class="NLM_p last">We first downloaded all UniProt entries of human protein kinases (HPK) from the Universal Protein Resource (UniProt) based on the document “Human protein kinases: classification and index” released by UniProt in June, 2015.<a onclick="showRef(event, 'ref92 ref93'); return false;" href="javascript:void(0);" class="ref ref92 ref93">(92, 93)</a> Then for every kinase UniProt entry, we retrieved the PDB entires of all kinases and downloaded all kinase structures from the Protein Data Bank (PDB).<a onclick="showRef(event, 'ref11 ref85'); return false;" href="javascript:void(0);" class="ref ref11 ref85">(11, 85)</a> Third, we excluded the PDB structures built by homology modeling and without the kinase catalytic domain; thus we obtained all kinase structures comprising 2786 structures from 235 human kinases. Finally, by exclusion of the nonligand bound kinase structures, as many as 2383 kinase–ligand complex structures, which we refer to as the HPK structural data set (HPK-SD), were obtained.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Step 2: To Compare Kinase Binding Sites Using SMAP</h4><div class="NLM_p">We superimposed the binding sites of all structures in the HPK-SD set by an all-against-all comparison using SMAP.<a onclick="showRef(event, 'ref40 ref42'); return false;" href="javascript:void(0);" class="ref ref40 ref42">(40, 42)</a> Here the SMAP parameter of “Ligand Contact Distance Cutoff” was set as 6.5 Å, which is an interaction distance threshold used frequently in the literatures.<a onclick="showRef(event, 'ref94 ref95'); return false;" href="javascript:void(0);" class="ref ref94 ref95">(94, 95)</a> Other SMAP parameters were set as default values. By use of SMAP, the function-site sequence motif, which constitutes the binding pocket, was obtained. Moreover, the function-site sequence motif is pairwise aligned. In addition, SMAP is a fast method for proteome-wide ligand-binding site comparison.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div><div class="NLM_p">Then we chose a function-site sequence motif as the representative template. We noted the binding pocket of the structure of a lymphocyte-specific protein tyrosine kinase (LCK; PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYU">3BYU</a>) consisted of the highest number of amino acid residues (80) as shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>b and covered all other aligned function-site sequence motifs. Thus, the function-site sequence motif in PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYU">3BYU</a> was selected as the reference template for the binding pocket.</div><div class="NLM_p last">For a given ligand-bound structure, we aligned it with the template <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYU">3BYU</a> and retrieved the aligned amino acid residues to comprise the binding site. To present all binding pockets using the same number of amino acids, we marked the position as “D-NULL” if the position corresponding to the template 3BYU did not have amino acid that contributes to the binding pocket. Thus, we could superimpose every binding pocket for all structures in HPK-SD and retrieve and align all binding site residues that constitute the binding pocket for these structures. We represented the binding pocket using a feature vector with the same number of elements (amino acid residues or “D-NULL”).</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Step 3: To Encode Fs-IFP for HPK-SD</h4><div class="NLM_p last">To encode the Fs-IFP, the polar hydrogen atoms were added for every function-site sequence motif and the corresponding ligand. Then we generated Fs-IFPs for the human kinase data set using PyPlif software by Radifar et al.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> PyPlif is a python implementation of IFP.<a onclick="showRef(event, 'ref25 ref33 ref34 ref36 ref97'); return false;" href="javascript:void(0);" class="ref ref25 ref33 ref34 ref36 ref97">(25, 33, 34, 36, 97)</a> For every residue in the binding site, a 7-bit array was encoded. The seven bits represented seven types of interactions using the interactional geometric rules:<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> (1) apolar (van der Waals); (2) aromatic face to face; (3) aromatic edge to face; (4) hydrogen bond (protein as hydrogen bond donor); (5) hydrogen bond (protein as hydrogen bond acceptor); (6) electrostatic interaction (protein positively charged); (7) electrostatic interaction (protein negatively charged). Then an Fs-IFP with a length of 560 bits (7 bits × 80 residues) was encoded according to the list of binding site residues for every kinase–ligand complex structure. Thus, we obtained an Fs-IFP data set based on our HPK-SD. Here PyPlif software is used to encode and analyze IFP, which is very fast and does not need significant computer resource.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Fs-IFP Similarity and Clustering</h3><div class="NLM_p last">For the HPK’s Fs-IFP data set, we carried out an all-against-all comparison by calculating the pairwise Tanimoto coefficient (TC) between any two Fs-IFPs.<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> We further clustered all ligand-bound structures using the TC values of Fs-IFPs as similarity measure. The R software was used for clustering and drawing the hierarchical tree; the complete linkage clustering method was used using the R-function hclust with the A2R library.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Ligand Similarity</h3><div class="NLM_p last">To determine the ligand characteristics in HPK-SD, we collected the information for all ligands extracted from all structures in HPK-SD. We described every ligand as a 1024-bit extended-connectivity fingerprint (ECFP), which is a circular topological fingerprint designed for molecular characterization.<a onclick="showRef(event, 'ref100 ref101 ref102'); return false;" href="javascript:void(0);" class="ref ref100 ref101 ref102">(100-102)</a> The ECFP was calculated using the Screenmd function Addin in Chemaxom.<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> Then we compared the pairwise similarity of all ligands by calculating the pairwise TC.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Kinase Binding Site Similarity</h3><div class="NLM_p last">Besides the attributes of protein–ligand binding modes and ligands, the features of all binding sites in HPK-SD were compared with each other to inspect the characteristics of the binding pockets, such as similarity and relationship with the attributes of ligand binding modes and ligands. First we defined the binding site for every ligand-bound structure using all amino acids that present one heavy atom closer than 6.5 Å to any heavy atom of the ligand.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> Then the binding site was encoded into a fingerprint and compared using the FuzCav software,<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> which is a novel alignment-free high-throughput algorithm to compute pairwise similarities between the binding sites by calculating the TC.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> KINOMEScan Data Analysis</h3><div class="NLM_p">We collected the compound-kinase profiling assay data from KINOMEscan,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> a large commercial kinase assay panels. KINOMEscan measures kinase-compound binding profiling shown by PoC (100 PoC means “no hit”; 0 PoC means “strong hit”) and the thermodynamic inhibitor constant <i>K</i><sub>d</sub>. Through Sep 22, 2015, the Harvard Medical School Library of Integrated Network-Based Cellular Signatures (HMS LINCS) contained 159 sets of small molecules profiling reports for use with KINOMEscan.<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> There are 87 compounds with the PoC profiling results in the presence of 10 μM assay compound concentration, 16 compounds with the PoC profiling results in the presence of 1 μM assay compound concentration, and 56 compounds with the profiling results of binding constants (<i>K</i><sub>d</sub>). Thus, we first evaluated the small molecule overlap between KINOMEsan data and our HPK-SD data set. We obtained 39 compounds with both X-ray complex structures and the compound-kinase PoC value at the screening concentration of 10 μM and 15 compounds with both X-ray complex structures and the compound-kinase PoC value at the screening concentration of 1 μM (shown in <a href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf" class="ext-link">Supporting Information Table S3</a>). There are 68 compounds with both X-ray complex structures and corresponding <i>K</i><sub>d</sub> values. Among them, 58 compounds have <i>K</i><sub>d</sub> less than 100 nM (<a href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf" class="ext-link">Supporting Information Table S4</a>).</div><div class="NLM_p last">We further studied the correlation of the experimental data and the binding mode derived from the Fs-IFP. Here we chose all PoC results from <a href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf" class="ext-link">Supporting Information Table S3</a> at the screening concentration of 10 μM to obtain the correlation with the binding mode using Fs-IFP. The PoC results at the screening concentration of 1 μM were not included because the number of data points was too small. Meanwhile, we also counted all <i>K</i><sub>d</sub> results from <a href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf" class="ext-link">Supporting Information Table S4</a>. Then we obtained the pairwise similarity by calculating the Euclidean distance of any two experimental values in the PoC data (or in <i>K</i><sub>d</sub> data) corresponding to the pairwise similarity of Fs-IFP using the TC.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20"><a href="/doi/suppl/10.1021/acs.jmedchem.5b02041">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10852" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10852" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b02041" class="ext-link">10.1021/acs.jmedchem.5b02041</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Correlation between the kinase pocket IFP and the experimental profile; the structures in every cluster; compounds with both cocrystallized kinase structures and PoC value; compounds with both cocrystallized kinase structures and <i>K</i><sub>d</sub> value (<a href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Complete Fs-IFP database table (<a href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_002.xls" class="ext-link">XLS</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf">jm5b02041_si_001.pdf (132.42 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_002.xls">jm5b02041_si_002.xls (2.62 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b02041" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48051" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48051" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lei Xie</span> - <span class="hlFld-Affiliation affiliation">Department
of Computer Science, Hunter College, The
City University of New York, New
York, New York 10065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">The
Graduate Center, The City University of New York, New York, New York 10016, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#49252c206731202c09213c273d2c3b672a3c2730672c2d3c"><span class="__cf_email__" data-cfemail="2b474e420553424e6b435e455f4e5905485e4552054e4f5e">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philip E. Bourne</span> - <span class="hlFld-Affiliation affiliation">Office of the
Director, National Institutes of Health, Bethesda, Maryland 20892, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c3b3abaaafaab3eda1acb6b1ada683adaaabeda4acb5"><span class="__cf_email__" data-cfemail="81f1e9e8ede8f1afe3eef4f3efe4c1efe8e9afe6eef7">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zheng Zhao</span> - <span class="hlFld-Affiliation affiliation">National
Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20894, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Xie</span> - <span class="hlFld-Affiliation affiliation">Scripps
Ranch, San Diego, California 92131, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d459e2391-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74684" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74684" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was supported by the Intramural Research Program of the National Library of Medicine, National Institutes of Health (Z.Z. and P.E.B.), the National Library of Medicine, National Institutes of Health under Award R01LM011986 (L.X.), and the National Institute on Minority Health and Health Disparities, National Institutes of Health, under Award G12MD007599 (L.X.). We also appreciate Dr. Enade Perdana Istyastono et al. for providing the IFP software, Pyplif and Dr. T. Exner et al. for providing the docking program PLANTS.</p></div><div class="postAsapNotesSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="postAsapNotesSection">Note Added After ASAP Publication</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66276" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66276" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="first last">After this paper was published ASAP on March 17, 2016, the number of kinases was changed from 85 to 190 in line 8 of the left column on the third page. The corrected version was reposted April 5, 2016.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i22" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i22"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i23" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i23"> Abbreviations Used</h2><tr><td class="NLM_term">IFP</td><td class="NLM_def"><p class="first last">interaction fingerprint</p></td></tr><tr><td class="NLM_term">Fs-IFP</td><td class="NLM_def"><p class="first last">functional site interaction fingerprint</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">U.S. Food and Drug Administration</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">non-small-cell lung cancer</p></td></tr><tr><td class="NLM_term">ALK</td><td class="NLM_def"><p class="first last">anaplastic lymphoma kinase</p></td></tr><tr><td class="NLM_term">PoC</td><td class="NLM_def"><p class="first last">percent of control</p></td></tr><tr><td class="NLM_term">HPK-SD</td><td class="NLM_def"><p class="first last">human protein kinase structural data set</p></td></tr><tr><td class="NLM_term">TC</td><td class="NLM_def"><p class="first last">Tanimoto coefficient</p></td></tr><tr><td class="NLM_term">ECFP</td><td class="NLM_def"><p class="first last">extended-connectivity fingerprint</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33114" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33114" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 103 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">–</span> <span class="NLM_lpage">1934</span><span class="refDoi"> DOI: 10.1126/science.1075762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome&doi=10.1126%2Fscience.1075762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lhhz2T6p66_PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934%26doi%3D10.1126%2Fscience.1075762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span><span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lg_9HtwuzizeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Ward, R. A.; Goldberg, F.</span> <span class="citation_source-book">Kinase Drug Discovery</span>; <span class="NLM_publisher-name">RSC Publishing</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=R.+A.+Ward&author=F.+Goldberg&title=Kinase+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26btitle%3DKinase%2520Drug%2520Discovery%26pub%3DRSC%2520Publishing%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Klebl, B.; Müller, G.; Hamacher, M.</span> <span class="citation_source-book">Protein Kinase as Drug Targets</span>; <span class="NLM_publisher-name">Wiley-VCH Verlag GambH & Co. KGaA</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">49</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1002%2F9783527633470" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=B.+Klebl&author=G.+M%C3%BCller&author=M.+Hamacher&title=Protein+Kinase+as+Drug+Targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2F9783527633470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527633470%26sid%3Dliteratum%253Aachs%26aulast%3DKlebl%26aufirst%3DB.%26btitle%3DProtein%2520Kinase%2520as%2520Drug%2520Targets%26pub%3DWiley-VCH%2520Verlag%2520GambH%2520%2526%2520Co.%2520KGaA%26date%3D2011%26volume%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Goldman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melo, J. V.</span><span> </span><span class="NLM_article-title">Chronic myeloid leukemia--advances in biology and new approaches to treatment</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">349</span><span class="NLM_x">, </span> <span class="NLM_fpage">1451</span><span class="NLM_x">–</span> <span class="NLM_lpage">1464</span><span class="refDoi"> DOI: 10.1056/NEJMra020777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1056%2FNEJMra020777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=14534339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotVGgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=349&publication_year=2003&pages=1451-1464&author=J.+M.+Goldmanauthor=J.+V.+Melo&title=Chronic+myeloid+leukemia%2D%2Dadvances+in+biology+and+new+approaches+to+treatment&doi=10.1056%2FNEJMra020777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chronic myeloid leukemia - Advances in biology and new approaches to treatment</span></div><div class="casAuthors">Goldman, John M.; Melo, Junia V.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">349</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1451-1464</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review presents recent advances in knowledge related to the pathogenesis of chronic myeloid leukemia (CML) and indicates where these may have important therapeutic implications.  It discusses the cytokinetics, cytogenetics, mol. events, graft-vs.-leukemia effect and treatment options of CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXmKTzKKnN9bVg90H21EOLACvtfcHk0lg_9HtwuzizeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotVGgsLc%253D&md5=88ae3d2434d003dca02f0e4af5bfff8d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1056%2FNEJMra020777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra020777%26sid%3Dliteratum%253Aachs%26aulast%3DGoldman%26aufirst%3DJ.%2BM.%26aulast%3DMelo%26aufirst%3DJ.%2BV.%26atitle%3DChronic%2520myeloid%2520leukemia--advances%2520in%2520biology%2520and%2520new%2520approaches%2520to%2520treatment%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2003%26volume%3D349%26spage%3D1451%26epage%3D1464%26doi%3D10.1056%2FNEJMra020777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The (un)targeted cancer kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">166</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span><span class="refDoi"> DOI: 10.1038/nchembio.297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnchembio.297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=20154661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=166-169&author=O.+Fedorovauthor=S.+Mullerauthor=S.+Knapp&title=The+%28un%29targeted+cancer+kinome&doi=10.1038%2Fnchembio.297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The (un)targeted cancer kinome</span></div><div class="casAuthors">Fedorov, Oleg; Mueller, Susanne; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">166-169</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review and discussion.  The complexity of cancer signaling and the resulting difficulties in target selection have strongly biased kinase drug discovery towards clin. validated targets.  Recently, novel kinase targets that are uncharacterized have emerged from genome sequencing and RNAi studies.  Chem. probes are urgently needed to functionally annotate these kinases and to stimulate new drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwcnq0-qjPirVg90H21EOLACvtfcHk0lg_9HtwuzizeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D&md5=198ae8d39d40247e17aef5e9de0f2a99</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.297%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520%2528un%2529targeted%2520cancer%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D166%26epage%3D169%26doi%3D10.1038%2Fnchembio.297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Sarnpitak, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mujumdar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coster, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorse, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krasavin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofmann, A.</span><span> </span><span class="NLM_article-title">Panel docking of small-molecule libraries - Prospects to improve efficiency of lead compound discovery</span> <span class="citation_source-journal">Biotechnol. Adv.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">947</span><span class="refDoi"> DOI: 10.1016/j.biotechadv.2015.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.biotechadv.2015.05.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=941-947&author=P.+Sarnpitakauthor=P.+Mujumdarauthor=P.+Taylorauthor=M.+Crossauthor=M.+J.+Costerauthor=A.+D.+Gorseauthor=M.+Krasavinauthor=A.+Hofmann&title=Panel+docking+of+small-molecule+libraries+-+Prospects+to+improve+efficiency+of+lead+compound+discovery&doi=10.1016%2Fj.biotechadv.2015.05.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2015.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2015.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DSarnpitak%26aufirst%3DP.%26aulast%3DMujumdar%26aufirst%3DP.%26aulast%3DTaylor%26aufirst%3DP.%26aulast%3DCross%26aufirst%3DM.%26aulast%3DCoster%26aufirst%3DM.%2BJ.%26aulast%3DGorse%26aufirst%3DA.%2BD.%26aulast%3DKrasavin%26aufirst%3DM.%26aulast%3DHofmann%26aufirst%3DA.%26atitle%3DPanel%2520docking%2520of%2520small-molecule%2520libraries%2520-%2520Prospects%2520to%2520improve%2520efficiency%2520of%2520lead%2520compound%2520discovery%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2015%26volume%3D33%26spage%3D941%26epage%3D947%26doi%3D10.1016%2Fj.biotechadv.2015.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausen, M. H.</span><span> </span><span class="NLM_article-title">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1016/j.drudis.2015.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.drudis.2015.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=26210956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=5-10&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=Small-molecule+kinase+inhibitors%3A+an+analysis+of+FDA-approved+drugs&doi=10.1016%2Fj.drudis.2015.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors (SMKIs), 28 of which are approved by the US Food and Drug Administration (FDA), have been actively pursued as promising targeted therapeutics.  Here, we assess the key structural and physicochem. properties, target selectivity and mechanism of function, and therapeutic indications of these approved inhibitors.  Our anal. showed that >30% of approved SMKIs have a mol. wt. (MW) exceeding 500 and all have a total ring count of between three and five.  The assumption that type II inhibitors tend to be more selective than type I inhibitors has been proved to be unreliable.  Although previous SMKI research was concd. on tyrosine kinase inhibitors for cancer treatment, recent progress indicates diversification of SMKI research in terms of new targets, mechanistic types, and therapeutic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZrY68ZOI2xLVg90H21EOLACvtfcHk0ljIdx__oRyXVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P&md5=228524fa464393053ecbfac19e994929</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DSmall-molecule%2520kinase%2520inhibitors%253A%2520an%2520analysis%2520of%2520FDA-approved%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.drudis.2015.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The ins and outs of selective kinase inhibitor development</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">–</span> <span class="NLM_lpage">821</span><span class="refDoi"> DOI: 10.1038/nchembio.1938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnchembio.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=26485069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=818-821&author=S.+Mullerauthor=A.+Chaikuadauthor=N.+S.+Grayauthor=S.+Knapp&title=The+ins+and+outs+of+selective+kinase+inhibitor+development&doi=10.1038%2Fnchembio.1938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The ins and outs of selective kinase inhibitor development</span></div><div class="casAuthors">Muller, Susanne; Chaikuad, Apirat; Gray, Nathanael S.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">818-821</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein kinases have emerged as one of the most successful families of drug targets.  To date, most selective kinase inhibitors have been discovered serendipitously either through broad selectivity screening or through the discovery of unique binding modes.  Here the authors discuss design strategies that could lead to a broader coverage of the kinome with selective inhibitors and to a more rational approach for developing them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQzaMrxSYx_rVg90H21EOLACvtfcHk0ljIdx__oRyXVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF&md5=f5adf0bbaa388c87b26bae23cfad0bae</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1938%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520ins%2520and%2520outs%2520of%2520selective%2520kinase%2520inhibitor%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D818%26epage%3D821%26doi%3D10.1038%2Fnchembio.1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Jacoby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tresadern, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bembenek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wroblowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buyck, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neefs, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rassokhin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poncelet, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Vlijmen, H.</span><span> </span><span class="NLM_article-title">Extending kinome coverage by analysis of kinase inhibitor broad profiling data</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">652</span><span class="NLM_x">–</span> <span class="NLM_lpage">658</span><span class="refDoi"> DOI: 10.1016/j.drudis.2015.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.drudis.2015.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25596550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFKgsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=652-658&author=E.+Jacobyauthor=G.+Tresadernauthor=S.+Bembenekauthor=B.+Wroblowskiauthor=C.+Buyckauthor=J.+M.+Neefsauthor=D.+Rassokhinauthor=A.+Ponceletauthor=J.+Huntauthor=H.+van+Vlijmen&title=Extending+kinome+coverage+by+analysis+of+kinase+inhibitor+broad+profiling+data&doi=10.1016%2Fj.drudis.2015.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Extending kinome coverage by analysis of kinase inhibitor broad profiling data</span></div><div class="casAuthors">Jacoby, Edgar; Tresadern, Gary; Bembenek, Scott; Wroblowski, Berthold; Buyck, Christophe; Neefs, Jean-Marc; Rassokhin, Dmitrii; Poncelet, Alain; Hunt, Jeremy; van Vlijmen, Herman</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">652-658</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The explored kinome was extended with broad profiling using the DiscoveRx and Millipore assay panels.  The anal. of the profiling of 3368 selected inhibitors on 456 kinases in the DiscoveRx format delivered several insights.  First, the coverage depended on the threshold of the selectivity parameter.  Second, the relation between hit confirmation rates and inhibitor selectivity showed unexpectedly that higher selectivity can increase the likelihood of false positives.  Third, comparing the coverage of a focused to that of a random library showed that the design based on a max. no. of scaffolds was superior to a limited no. of scaffolds.  Therefore, selective compds. can be used in target validation, enable the jumpstarting of new kinase drug discovery projects, and chart new biol. space via phenotypic screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3cxoYeXbpxbVg90H21EOLACvtfcHk0ljIdx__oRyXVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKgsL8%253D&md5=b72bec6134c3eb716615f68ece233575</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DBembenek%26aufirst%3DS.%26aulast%3DWroblowski%26aufirst%3DB.%26aulast%3DBuyck%26aufirst%3DC.%26aulast%3DNeefs%26aufirst%3DJ.%2BM.%26aulast%3DRassokhin%26aufirst%3DD.%26aulast%3DPoncelet%26aufirst%3DA.%26aulast%3DHunt%26aufirst%3DJ.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%26atitle%3DExtending%2520kinome%2520coverage%2520by%2520analysis%2520of%2520kinase%2520inhibitor%2520broad%2520profiling%2520data%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D652%26epage%3D658%26doi%3D10.1016%2Fj.drudis.2015.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span> </span><span class="NLM_article-title">The Protein Data Bank and the challenge of structural genomics</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl</span><span class="NLM_x">) </span> <span class="NLM_fpage">957</span><span class="NLM_x">–</span> <span class="NLM_lpage">959</span><span class="refDoi"> DOI: 10.1038/80734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2F80734" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2000&pages=957-959&issue=Suppl&author=H.+M.+Bermanauthor=T.+N.+Bhatauthor=P.+E.+Bourneauthor=Z.+Fengauthor=G.+Gillilandauthor=H.+Weissigauthor=J.+Westbrook&title=The+Protein+Data+Bank+and+the+challenge+of+structural+genomics&doi=10.1038%2F80734"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2F80734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F80734%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DWestbrook%26aufirst%3DJ.%26atitle%3DThe%2520Protein%2520Data%2520Bank%2520and%2520the%2520challenge%2520of%2520structural%2520genomics%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2000%26volume%3D7%26issue%3DSuppl%26spage%3D957%26epage%3D959%26doi%3D10.1038%2F80734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span> </span><span class="NLM_article-title">Binding site similarity analysis for the functional classification of the protein kinase family</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">318</span><span class="NLM_x">–</span> <span class="NLM_lpage">329</span><span class="refDoi"> DOI: 10.1021/ci800289y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800289y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=318-329&author=S.+L.+Kinningsauthor=R.+M.+Jackson&title=Binding+site+similarity+analysis+for+the+functional+classification+of+the+protein+kinase+family&doi=10.1021%2Fci800289y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Similarity Analysis for the Functional Classification of the Protein Kinase Family</span></div><div class="casAuthors">Kinnings, Sarah L.; Jackson, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">318-329</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methods for analyzing complete gene families are becoming of increasing importance to the drug discovery process, because similarities and differences within a family are often the key to understanding functional differences that can be exploited in drug design.  We undertake a large-scale structural comparison of protein kinase ATP-binding sites using a geometric hashing method.  Subsequently, we propose a relevant classification of the protein kinase family based on the structural similarity of its binding sites.  Our classification is not only able to reveal the great diversity of different protein kinases and therefore their different potential for inhibitor selectivity, but it is also able to distinguish subtle differences within binding site conformation reflecting the protein activation state.  Furthermore, using exptl. inhibition profiling, we demonstrate that our classification can be used to identify protein kinase binding sites that are known exptl. to bind the same drug, demonstrating that it has potential as an inverse (protein) virtual screening tool, by identifying which other sites have the potential to bind a given drug.  In this way the cross-reactivities of the anticancer drugs Tarceva and Gleevec are rationalized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6kpf80r8s5LVg90H21EOLACvtfcHk0lhRc0O9BBqhFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltF2qsg%253D%253D&md5=742343601ca1e3c0549fa4f383c2566b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci800289y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800289y%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DBinding%2520site%2520similarity%2520analysis%2520for%2520the%2520functional%2520classification%2520of%2520the%2520protein%2520kinase%2520family%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D318%26epage%3D329%26doi%3D10.1021%2Fci800289y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">McGregor, M. J.</span><span> </span><span class="NLM_article-title">A pharmacophore map of small molecule protein kinase inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2374</span><span class="NLM_x">–</span> <span class="NLM_lpage">2382</span><span class="refDoi"> DOI: 10.1021/ci700244t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700244t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKksL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=2374-2382&author=M.+J.+McGregor&title=A+pharmacophore+map+of+small+molecule+protein+kinase+inhibitors&doi=10.1021%2Fci700244t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A Pharmacophore Map of Small Molecule Protein Kinase Inhibitors</span></div><div class="casAuthors">McGregor, Malcolm J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2374-2382</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases have emerged as one of the major drug target classes that are amenable to the development of small mol. inhibitors.  They share a conserved structural similarity in the region of the ATP binding site, where most inhibitors interact.  Using a pharmacophore approach, we have explored the features of protein-ligand interactions for a set of 220 kinase crystal structures from the Protein Data Bank (PDB).  The resulting "pharmacophore map" shows the interactions made by all ligands with their receptors simultaneously.  This gives insight that has been applied in the design of kinase screening sets and combinatorial libraries.  An algorithm is described that scores small mol. structures for goodness of fit to the resulting binding site description obtained from the anal.  The algorithm identifies a pose that is close to the crystal structure pose for the majority of ligands, with only the 2D chem. structure as input and no knowledge of the crystal structure from which it was derived.  Application of the algorithm to a test screening set gave a useful enrichment of active compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-PlOGvAO_Z7Vg90H21EOLACvtfcHk0lhRc0O9BBqhFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKksL%252FO&md5=38d2b55c3fbb1232fd7d091f95414633</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fci700244t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700244t%26sid%3Dliteratum%253Aachs%26aulast%3DMcGregor%26aufirst%3DM.%2BJ.%26atitle%3DA%2520pharmacophore%2520map%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D2374%26epage%3D2382%26doi%3D10.1021%2Fci700244t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Huse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">The conformational plasticity of protein kinases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">275</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span><span class="refDoi"> DOI: 10.1016/S0092-8674(02)00741-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2FS0092-8674%2802%2900741-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=12015977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2002&pages=275-282&author=M.+Huseauthor=J.+Kuriyan&title=The+conformational+plasticity+of+protein+kinases&doi=10.1016%2FS0092-8674%2802%2900741-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The conformational plasticity of protein kinases</span></div><div class="casAuthors">Huse, Morgan; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">275-282</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with 48 refs.  Protein kinases operate in a large no. of distinct signaling pathways, where the tight regulation of their catalytic activity is crucial to the development and maintenance of eukaryotic organisms.  The catalytic domains of different kinases adopt strikingly similar structures when they are active.  By contrast, crystal structures of inactive kinases have revealed a marked plasticity in the kinase domain that allows the adoption of distinct conformations in response to interactions with specific regulatory domains or proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJrn7F3h4ej7Vg90H21EOLACvtfcHk0lhRc0O9BBqhFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D&md5=cd56edae94ad239f2ce7a5d6c8930cbb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900741-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900741-9%26sid%3Dliteratum%253Aachs%26aulast%3DHuse%26aufirst%3DM.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DThe%2520conformational%2520plasticity%2520of%2520protein%2520kinases%26jtitle%3DCell%26date%3D2002%26volume%3D109%26spage%3D275%26epage%3D282%26doi%3D10.1016%2FS0092-8674%2802%2900741-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Exploration of type II binding mode: a privileged approach for kinase inhibitor focused drug discovery?</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1230</span><span class="NLM_x">–</span> <span class="NLM_lpage">1241</span><span class="refDoi"> DOI: 10.1021/cb500129t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+a+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F&doi=10.1021%2Fcb500129t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Type II Binding Mode: A Privileged Approach for Kinase Inhibitor Focused Drug Discovery?</span></div><div class="casAuthors">Zhao, Zheng; Wu, Hong; Wang, Li; Liu, Yi; Knapp, Stefan; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1230-1241</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ATP site of kinases displays remarkable conformational flexibility when accommodating chem. diverse small mol. inhibitors.  The so-called activation segment, whose conformation controls catalytic activity and access to the substrate binding pocket, can undergo a large conformational change with the active state assuming a 'DFG-in' and an inactive state assuming a 'DFG-out' conformation.  Compds. that preferentially bind to the DFG-out conformation are typically called 'type II' inhibitors in contrast to 'type I' inhibitors that bind to the DFG-in conformation.  This review surveys the large no. of type II inhibitors that have been developed and provides an anal. of their crystallog. detd. binding modes.  Using a small library of type II inhibitors, we demonstrate that more than 200 kinases can be targeted, suggesting that type II inhibitors may not be intrinsically more selective than type I inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4jYt6Db3Ap7Vg90H21EOLACvtfcHk0libsDPrFyl3AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D&md5=95bf185ff7f973eda5a65f4096bfaa82</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520a%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241%26doi%3D10.1021%2Fcb500129t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Furtmann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of three-dimensional activity cliffs formed by kinase inhibitors with different binding modes and cliff mapping of structural analogues</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">252</span><span class="NLM_x">–</span> <span class="NLM_lpage">264</span><span class="refDoi"> DOI: 10.1021/jm5009264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5009264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=252-264&author=N.+Furtmannauthor=Y.+Huauthor=J.+Bajorath&title=Comprehensive+analysis+of+three-dimensional+activity+cliffs+formed+by+kinase+inhibitors+with+different+binding+modes+and+cliff+mapping+of+structural+analogues&doi=10.1021%2Fjm5009264"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm5009264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5009264%26sid%3Dliteratum%253Aachs%26aulast%3DFurtmann%26aufirst%3DN.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComprehensive%2520analysis%2520of%2520three-dimensional%2520activity%2520cliffs%2520formed%2520by%2520kinase%2520inhibitors%2520with%2520different%2520binding%2520modes%2520and%2520cliff%2520mapping%2520of%2520structural%2520analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D252%26epage%3D264%26doi%3D10.1021%2Fjm5009264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0libsDPrFyl3AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Liao, J. J.</span><span> </span><span class="NLM_article-title">Molecular recognition of protein kinase binding pockets for design of potent and selective kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">409</span><span class="NLM_x">–</span> <span class="NLM_lpage">424</span><span class="refDoi"> DOI: 10.1021/jm0608107</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0608107" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1Cmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=409-424&author=J.+J.+Liao&title=Molecular+recognition+of+protein+kinase+binding+pockets+for+design+of+potent+and+selective+kinase+inhibitors&doi=10.1021%2Fjm0608107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Recognition of Protein Kinase Binding Pockets for Design of Potent and Selective Kinase Inhibitors</span></div><div class="casAuthors">Liao, Jeffrey Jie-Lou</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">409-424</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Many kinase inhibitors have failed in preclin. or clin. development because of the lack of such selectivity that induces intolerable side effects, mostly because the catalytic cleft is highly conserved in sequence and conformation.  Systematic anal. of the crystal structures of protein kinases can offer insight into the design of highly selective kinase inhibitors to overcome these effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroDTl2SBUfbbVg90H21EOLACvtfcHk0liojWkwQUf9bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1Cmsg%253D%253D&md5=77542d3ba589bd834a0d2b97a24eea05</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm0608107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608107%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DJ.%2BJ.%26atitle%3DMolecular%2520recognition%2520of%2520protein%2520kinase%2520binding%2520pockets%2520for%2520design%2520of%2520potent%2520and%2520selective%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D409%26epage%3D424%26doi%3D10.1021%2Fjm0608107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafleur, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevado, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Kinase selectivity potential for inhibitors targeting the ATP binding site: a network analysis</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">198</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btp650</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1093%2Fbioinformatics%2Fbtp650" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19942586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlGkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=198-204&author=D.+Huangauthor=T.+Zhouauthor=K.+Lafleurauthor=C.+Nevadoauthor=A.+Caflisch&title=Kinase+selectivity+potential+for+inhibitors+targeting+the+ATP+binding+site%3A+a+network+analysis&doi=10.1093%2Fbioinformatics%2Fbtp650"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase selectivity potential for inhibitors targeting the ATP binding site: a network analysis</span></div><div class="casAuthors">Huang, Danzhi; Zhou, Ting; Lafleur, Karine; Nevado, Cristina; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">198-204</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Small-mol. inhibitors targeting the ATP binding pocket of the catalytic domain of protein kinases have potential to become drugs devoid of (major) side effects, particularly if they bind selectively.  Here, the sequences of the 518 human kinases are first mapped onto the structural alignment of 116 kinases of known three-dimensional structure.  The multiple structure alignment is then used to encode the known strategies for developing selective inhibitors into a fingerprint.  Finally, a network anal. is used to partition the kinases into clusters according to similarity of their fingerprints, i.e., physico-chem. characteristics of the residues responsible for selective binding.  For each kinase the network anal. reveals the likelihood to find selective inhibitors targeting the ATP binding site.  Systematic guidelines are proposed to develop selective inhibitors.  Importantly, the network anal. suggests that the tyrosine kinase EphB4 has high selectivity potential, which is consistent with the selectivity profile of two novel EphB4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQVQsrpqB_ILVg90H21EOLACvtfcHk0liojWkwQUf9bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlGkug%253D%253D&md5=b6c63ccd4654219a56752f9cfc1356a3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp650&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp650%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DLafleur%26aufirst%3DK.%26aulast%3DNevado%26aufirst%3DC.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DKinase%2520selectivity%2520potential%2520for%2520inhibitors%2520targeting%2520the%2520ATP%2520binding%2520site%253A%2520a%2520network%2520analysis%26jtitle%3DBioinformatics%26date%3D2010%26volume%3D26%26spage%3D198%26epage%3D204%26doi%3D10.1093%2Fbioinformatics%2Fbtp650" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Cui, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran-Dube, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nambu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pairish, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Funk, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrous, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McTigue, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grodsky, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padrique, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alton, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timofeevski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mroczkowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kania, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, M. P.</span><span> </span><span class="NLM_article-title">Structure based drug design of crizotinib (PF-02341066), a potent and selective dual inhibitor of mesenchymal-epithelial transition factor (c-MET) kinase and anaplastic lymphoma kinase (ALK)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6342</span><span class="NLM_x">–</span> <span class="NLM_lpage">6363</span><span class="refDoi"> DOI: 10.1021/jm2007613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2007613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVegtLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6342-6363&author=J.+J.+Cuiauthor=M.+Tran-Dubeauthor=H.+Shenauthor=M.+Nambuauthor=P.+P.+Kungauthor=M.+Pairishauthor=L.+Jiaauthor=J.+Mengauthor=L.+Funkauthor=I.+Botrousauthor=M.+McTigueauthor=N.+Grodskyauthor=K.+Ryanauthor=E.+Padriqueauthor=G.+Altonauthor=S.+Timofeevskiauthor=S.+Yamazakiauthor=Q.+Liauthor=H.+Zouauthor=J.+Christensenauthor=B.+Mroczkowskiauthor=S.+Benderauthor=R.+S.+Kaniaauthor=M.+P.+Edwards&title=Structure+based+drug+design+of+crizotinib+%28PF-02341066%29%2C+a+potent+and+selective+dual+inhibitor+of+mesenchymal-epithelial+transition+factor+%28c-MET%29+kinase+and+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1021%2Fjm2007613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Based Drug Design of Crizotinib (PF-02341066), a Potent and Selective Dual Inhibitor of Mesenchymal-Epithelial Transition Factor (c-MET) Kinase and Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Cui, J. Jean; Tran-Dube, Michelle; Shen, Hong; Nambu, Mitchell; Kung, Pei-Pei; Pairish, Mason; Jia, Lei; Meng, Jerry; Funk, Lee; Botrous, Iriny; McTigue, Michele; Grodsky, Neil; Ryan, Kevin; Padrique, Ellen; Alton, Gordon; Timofeevski, Sergei; Yamazaki, Shinji; Li, Qiuhua; Zou, Helen; Christensen, James; Mroczkowski, Barbara; Bender, Steve; Kania, Robert S.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6342-6363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Because of the crit. roles of aberrant signaling in cancer, both c-MET and ALK receptor tyrosine kinases are attractive oncol. targets for therapeutic intervention.  The cocrystal structure of 3 (PHA-665752), bound to c-MET kinase domain, revealed a novel ATP site environment, which served as the target to guide parallel, multiattribute drug design.  A novel 2-amino-5-aryl-3-benzyloxypyridine series was created to more effectively make the key interactions achieved with 3.  In the novel series, the 2-aminopyridine core allowed a 3-benzyloxy group to reach into the same pocket as the 2,6-dichlorophenyl group of 3 via a more direct vector and thus with a better ligand efficiency (LE).  Further optimization of the lead series generated the clin. candidate crizotinib (PF-02341066), which demonstrated potent in vitro and in vivo c-MET kinase and ALK inhibition, effective tumor growth inhibition, and good pharmaceutical properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHs_cKJ-U3t7Vg90H21EOLACvtfcHk0liojWkwQUf9bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVegtLrN&md5=dbab4c1b3e7c76180dbad04be2b6a474</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm2007613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2007613%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26aulast%3DTran-Dube%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DNambu%26aufirst%3DM.%26aulast%3DKung%26aufirst%3DP.%2BP.%26aulast%3DPairish%26aufirst%3DM.%26aulast%3DJia%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DJ.%26aulast%3DFunk%26aufirst%3DL.%26aulast%3DBotrous%26aufirst%3DI.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DGrodsky%26aufirst%3DN.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DPadrique%26aufirst%3DE.%26aulast%3DAlton%26aufirst%3DG.%26aulast%3DTimofeevski%26aufirst%3DS.%26aulast%3DYamazaki%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DMroczkowski%26aufirst%3DB.%26aulast%3DBender%26aufirst%3DS.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DStructure%2520based%2520drug%2520design%2520of%2520crizotinib%2520%2528PF-02341066%2529%252C%2520a%2520potent%2520and%2520selective%2520dual%2520inhibitor%2520of%2520mesenchymal-epithelial%2520transition%2520factor%2520%2528c-MET%2529%2520kinase%2520and%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6342%26epage%3D6363%26doi%3D10.1021%2Fjm2007613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lgs80YGZfOLmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Chuaqui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span> </span><span class="NLM_article-title">Interaction profiles of protein kinase-inhibitor complexes and their application to virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">121</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1021/jm049312t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049312t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=121-133&author=C.+Chuaquiauthor=Z.+Dengauthor=J.+Singh&title=Interaction+profiles+of+protein+kinase-inhibitor+complexes+and+their+application+to+virtual+screening&doi=10.1021%2Fjm049312t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm049312t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049312t%26sid%3Dliteratum%253Aachs%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DInteraction%2520profiles%2520of%2520protein%2520kinase-inhibitor%2520complexes%2520and%2520their%2520application%2520to%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D121%26epage%3D133%26doi%3D10.1021%2Fjm049312t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Patricelli, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nomanbhoy, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagannathan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aban, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okerberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herring, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozarich, J. W.</span><span> </span><span class="NLM_article-title">In situ kinase profiling reveals functionally relevant properties of native kinases</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">699</span><span class="NLM_x">–</span> <span class="NLM_lpage">710</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2011.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.chembiol.2011.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=21700206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotVejtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=699-710&author=M.+P.+Patricelliauthor=T.+K.+Nomanbhoyauthor=J.+Wuauthor=H.+Brownauthor=D.+Zhouauthor=J.+Zhangauthor=S.+Jagannathanauthor=A.+Abanauthor=E.+Okerbergauthor=C.+Herringauthor=B.+Nordinauthor=H.+Weissigauthor=Q.+Yangauthor=J.+D.+Leeauthor=N.+S.+Grayauthor=J.+W.+Kozarich&title=In+situ+kinase+profiling+reveals+functionally+relevant+properties+of+native+kinases&doi=10.1016%2Fj.chembiol.2011.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">In Situ Kinase Profiling Reveals Functionally Relevant Properties of Native Kinases</span></div><div class="casAuthors">Patricelli, Matthew P.; Nomanbhoy, Tyzoon K.; Wu, Jiangyue; Brown, Heidi; Zhou, David; Zhang, Jianming; Jagannathan, Subadhra; Aban, Arwin; Okerberg, Eric; Herring, Chris; Nordin, Brian; Weissig, Helge; Yang, Qingkai; Lee, Jiing-Dwan; Gray, Nathanael S.; Kozarich, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">699-710</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Protein kinases are intensely studied mediators of cellular signaling, yet important questions remain regarding their regulation and in vivo properties.  Here, we use a probe-based chemoprotemics platform to profile several well studied kinase inhibitors against >200 kinases in native cell proteomes and reveal biol. targets for some of these inhibitors.  Several striking differences were identified between native and recombinant kinase inhibitory profiles, in particular, for the Raf kinases.  The native kinase binding profiles presented here closely mirror the cellular activity of these inhibitors, even when the inhibition profiles differ dramatically from recombinant assay results.  Addnl., Raf activation events could be detected on live cell treatment with inhibitors.  These studies highlight the complexities of protein kinase behavior in the cellular context and demonstrate that profiling with only recombinant/purified enzymes can be misleading.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopb21npuaC_rVg90H21EOLACvtfcHk0lgs80YGZfOLmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotVejtbo%253D&md5=dda091b895850c95131f80c3aba9e8c7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DNomanbhoy%26aufirst%3DT.%2BK.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DJagannathan%26aufirst%3DS.%26aulast%3DAban%26aufirst%3DA.%26aulast%3DOkerberg%26aufirst%3DE.%26aulast%3DHerring%26aufirst%3DC.%26aulast%3DNordin%26aufirst%3DB.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DLee%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKozarich%26aufirst%3DJ.%2BW.%26atitle%3DIn%2520situ%2520kinase%2520profiling%2520reveals%2520functionally%2520relevant%2520properties%2520of%2520native%2520kinases%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D699%26epage%3D710%26doi%3D10.1016%2Fj.chembiol.2011.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Shemon, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heil, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granovsky, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McElheny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chimon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosner, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koide, S.</span><span> </span><span class="NLM_article-title">Characterization of the Raf kinase inhibitory protein (RKIP) binding pocket: NMR-based screening identifies small-molecule ligands</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e10479</span><span class="refDoi"> DOI: 10.1371/journal.pone.0010479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pone.0010479" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e10479&author=A.+N.+Shemonauthor=G.+L.+Heilauthor=A.+E.+Granovskyauthor=M.+M.+Clarkauthor=D.+McElhenyauthor=A.+Chimonauthor=M.+R.+Rosnerauthor=S.+Koide&title=Characterization+of+the+Raf+kinase+inhibitory+protein+%28RKIP%29+binding+pocket%3A+NMR-based+screening+identifies+small-molecule+ligands&doi=10.1371%2Fjournal.pone.0010479"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0010479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0010479%26sid%3Dliteratum%253Aachs%26aulast%3DShemon%26aufirst%3DA.%2BN.%26aulast%3DHeil%26aufirst%3DG.%2BL.%26aulast%3DGranovsky%26aufirst%3DA.%2BE.%26aulast%3DClark%26aufirst%3DM.%2BM.%26aulast%3DMcElheny%26aufirst%3DD.%26aulast%3DChimon%26aufirst%3DA.%26aulast%3DRosner%26aufirst%3DM.%2BR.%26aulast%3DKoide%26aufirst%3DS.%26atitle%3DCharacterization%2520of%2520the%2520Raf%2520kinase%2520inhibitory%2520protein%2520%2528RKIP%2529%2520binding%2520pocket%253A%2520NMR-based%2520screening%2520identifies%2520small-molecule%2520ligands%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26spage%3De10479%26doi%3D10.1371%2Fjournal.pone.0010479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuaqui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span> </span><span class="NLM_article-title">Structural interaction fingerprint (SIFt): a novel method for analyzing three-dimensional protein-ligand binding interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">344</span><span class="refDoi"> DOI: 10.1021/jm030331x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030331x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=337-344&author=Z.+Dengauthor=C.+Chuaquiauthor=J.+Singh&title=Structural+interaction+fingerprint+%28SIFt%29%3A+a+novel+method+for+analyzing+three-dimensional+protein-ligand+binding+interactions&doi=10.1021%2Fjm030331x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Interaction Fingerprint (SIFt): A Novel Method for Analyzing Three-Dimensional Protein-Ligand Binding Interactions</span></div><div class="casAuthors">Deng, Zhan; Chuaqui, Claudio; Singh, Juswinder</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-344</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Representing and understanding the three-dimensional (3D) structural information of protein-ligand complexes is a crit. step in the rational drug discovery process.  Traditional anal. methods are proving inadequate and inefficient in dealing with the massive amt. of structural information being generated from x-ray crystallog., NMR, and in silico approaches such as structure-based docking expts.  Here, we present SIFt (structural interaction fingerprint), a novel method for representing and analyzing 3D protein-ligand binding interactions.  Key to this approach is the generation of an interaction fingerprint that translates 3D structural binding information from a protein-ligand complex into a one-dimensional binary string.  Each fingerprint represents the "structural interaction profile" of the complex that can be used to organize, analyze, and visualize the rich amt. of information encoded in ligand-receptor complexes and also to assist database mining.  We have applied SIFt to tackle three common tasks in structure-based drug design.  The first involved the anal. and organization of a typical set of results generated from a docking study.  Using SIFt, docking poses with similar binding modes were identified, clustered, and subsequently compared with conventional scoring function information.  A second application of SIFt was to analyze ∼90 known x-ray crystal structures of protein kinase-inhibitor complexes obtained from the Protein Databank.  Using SIFt, we were able to organize the structures and reveal striking similarities and diversity between their small mol. binding interactions.  Finally, we have shown how SIFt can be used as an effective mol. filter during the virtual chem. library screening process to select mols. with desirable binding mode(s) and/or desirable interaction patterns with the protein target.  In summary, SIFt shows promise to fully leverage the wealth of information being generated in rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkjji1ni0HHbVg90H21EOLACvtfcHk0lgPtJ1QD1rZBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D&md5=a652b70184155c107202cbe3bc74596a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm030331x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030331x%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DStructural%2520interaction%2520fingerprint%2520%2528SIFt%2529%253A%2520a%2520novel%2520method%2520for%2520analyzing%2520three-dimensional%2520protein-ligand%2520binding%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D337%26epage%3D344%26doi%3D10.1021%2Fjm030331x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Witek, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smusz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rataj, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mordalski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bojarski, A. J.</span><span> </span><span class="NLM_article-title">An application of machine learning methods to structural interaction fingerprints--a case study of kinase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">585</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.12.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.bmcl.2013.12.017" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=580-585&author=J.+Witekauthor=S.+Smuszauthor=K.+Ratajauthor=S.+Mordalskiauthor=A.+J.+Bojarski&title=An+application+of+machine+learning+methods+to+structural+interaction+fingerprints%2D%2Da+case+study+of+kinase+inhibitors&doi=10.1016%2Fj.bmcl.2013.12.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.12.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.12.017%26sid%3Dliteratum%253Aachs%26aulast%3DWitek%26aufirst%3DJ.%26aulast%3DSmusz%26aufirst%3DS.%26aulast%3DRataj%26aufirst%3DK.%26aulast%3DMordalski%26aufirst%3DS.%26aulast%3DBojarski%26aufirst%3DA.%2BJ.%26atitle%3DAn%2520application%2520of%2520machine%2520learning%2520methods%2520to%2520structural%2520interaction%2520fingerprints--a%2520case%2520study%2520of%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D580%26epage%3D585%26doi%3D10.1016%2Fj.bmcl.2013.12.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">KLIFS: a knowledge-based structural database to navigate kinase-ligand interaction space</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="NLM_x">–</span> <span class="NLM_lpage">277</span><span class="refDoi"> DOI: 10.1021/jm400378w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400378w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=249-277&author=O.+P.+van+Lindenauthor=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+a+knowledge-based+structural+database+to+navigate+kinase-ligand+interaction+space&doi=10.1021%2Fjm400378w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: A Knowledge-Based Structural Database To Navigate Kinase-Ligand Interaction Space</span></div><div class="casAuthors">van Linden, Oscar P. J.; Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinases regulate the majority of signal transduction pathways in cells and have become important targets for the development of designer drugs.  We present a systematic anal. of kinase-ligand interactions in all regions of the catalytic cleft of all 1252 human kinase-ligand cocrystal structures present in the Protein Data Bank (PDB).  The kinase-ligand interaction fingerprints and structure database (KLIFS) contains a consistent alignment of 85 kinase ligand binding site residues that enables the identification of family specific interaction features and classification of ligands according to their binding modes.  We illustrate how systematic mining of kinase-ligand interaction space gives new insights into how conserved and selective kinase interaction hot spots can accommodate the large diversity of chem. scaffolds in kinase ligands.  These analyses lead to an improved understanding of the structural requirements of kinase binding that will be useful in ligand discovery and design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4tMmT6S-vRrVg90H21EOLACvtfcHk0lgPtJ1QD1rZBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P&md5=8519942e1703ba04f64797e07df4b712</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm400378w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400378w%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520a%2520knowledge-based%2520structural%2520database%2520to%2520navigate%2520kinase-ligand%2520interaction%2520space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D249%26epage%3D277%26doi%3D10.1021%2Fjm400378w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Bryant, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moll, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavraki, L. E.</span><span> </span><span class="NLM_article-title">Combinatorial clustering of residue position subsets predicts inhibitor affinity across the human kinome</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003087</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1003087" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003087&author=D.+H.+Bryantauthor=M.+Mollauthor=P.+W.+Finnauthor=L.+E.+Kavraki&title=Combinatorial+clustering+of+residue+position+subsets+predicts+inhibitor+affinity+across+the+human+kinome&doi=10.1371%2Fjournal.pcbi.1003087"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003087%26sid%3Dliteratum%253Aachs%26aulast%3DBryant%26aufirst%3DD.%2BH.%26aulast%3DMoll%26aufirst%3DM.%26aulast%3DFinn%26aufirst%3DP.%2BW.%26aulast%3DKavraki%26aufirst%3DL.%2BE.%26atitle%3DCombinatorial%2520clustering%2520of%2520residue%2520position%2520subsets%2520predicts%2520inhibitor%2520affinity%2520across%2520the%2520human%2520kinome%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3De1003087%26doi%3D10.1371%2Fjournal.pcbi.1003087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Cui, J. J.</span><span> </span><span class="NLM_article-title">Targeting receptor tyrosine kinase MET in cancer: small molecule inhibitors and clinical progress</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4427</span><span class="NLM_x">–</span> <span class="NLM_lpage">4453</span><span class="refDoi"> DOI: 10.1021/jm401427c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401427c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWltbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4427-4453&author=J.+J.+Cui&title=Targeting+receptor+tyrosine+kinase+MET+in+cancer%3A+small+molecule+inhibitors+and+clinical+progress&doi=10.1021%2Fjm401427c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Receptor Tyrosine Kinase MET in Cancer: Small Molecule Inhibitors and Clinical Progress</span></div><div class="casAuthors">Cui, J. Jean</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4427-4453</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The HGF/MET signaling pathway is crit. in mediating a wide range of normal physiol. functions including embryol. development, wound healing, and tissue regeneration.  Aberrant activation of the pathway has frequently been found in human cancers via protein overexpression, mutation, gene amplification, and also paracrine or autocrine up-regulation.  In addn., the activation of HGF/MET signaling confers resistance to the effects of cancer treatments.  Therefore, inhibition of the HGF/MET signaling pathway has great potential for therapeutic intervention in cancer.  Currently, there are three approaches toward modulating HGF/MET signaling in human clin. studies of cancer: anti-HGF monoclonal antibodies, MET monoclonal antibodies, and small mol. MET inhibitors.  Preliminary clin. benefit from inhibition of HGF or MET has been reported.  This Perspective will provide an overview of the HGF/MET signaling pathway in cancer and then will review the development of small mol. MET inhibitors and their progress in clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoARAsEnH4X5rVg90H21EOLACvtfcHk0lgtP8ij49UCpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWltbvE&md5=534fd1c6fd8b83be41251d0d556903d1</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm401427c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401427c%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26atitle%3DTargeting%2520receptor%2520tyrosine%2520kinase%2520MET%2520in%2520cancer%253A%2520small%2520molecule%2520inhibitors%2520and%2520clinical%2520progress%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4427%26epage%3D4453%26doi%3D10.1021%2Fjm401427c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Punta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coggill, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhardt, R. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mistry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tate, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boursnell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forslund, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceric, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clements, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonnhammer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eddy, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bateman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, R. D.</span><span> </span><span class="NLM_article-title">The Pfam protein families database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D290</span><span class="NLM_x">–</span> <span class="NLM_lpage">301</span><span class="refDoi"> DOI: 10.1093/nar/gkr1065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1093%2Fnar%2Fgkr1065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=22127870" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12hurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D290-301&author=M.+Puntaauthor=P.+C.+Coggillauthor=R.+Y.+Eberhardtauthor=J.+Mistryauthor=J.+Tateauthor=C.+Boursnellauthor=N.+Pangauthor=K.+Forslundauthor=G.+Cericauthor=J.+Clementsauthor=A.+Hegerauthor=L.+Holmauthor=E.+L.+Sonnhammerauthor=S.+R.+Eddyauthor=A.+Batemanauthor=R.+D.+Finn&title=The+Pfam+protein+families+database&doi=10.1093%2Fnar%2Fgkr1065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Pfam protein families database</span></div><div class="casAuthors">Punta, Marco; Coggill, Penny C.; Eberhardt, Ruth Y.; Mistry, Jaina; Tate, John; Boursnell, Chris; Pang, Ningze; Forslund, Kristoffer; Ceric, Goran; Clements, Jody; Heger, Andreas; Holm, Liisa; Sonnhammer, Erik L. L.; Eddy, Sean R.; Bateman, Alex; Finn, Robert D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D290-D301</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Pfam is a widely used database of protein families, currently contg. more than 13 000 manually curated protein families as of release 26.0.  Pfam is available via servers in the UK (http://pfam.sanger.ac.uk/), the USA (http://pfam.janelia.org/) and Sweden (http://pfam.sbc.su.se/).  Here, we report on changes that have occurred since our 2010 NAR paper (release 24.0).  Over the last 2 years, we have generated 1840 new families and increased coverage of the UniProt Knowledgebase (UniProtKB) to nearly 80%.  Notably, we have taken the step of opening up the annotation of our families to the Wikipedia community, by linking Pfam families to relevant Wikipedia pages and encouraging the Pfam and Wikipedia communities to improve and expand those pages.  We continue to improve the Pfam web-site and add new visualizations, such as the sunburst' representation of taxonomic distribution of families.  In this work we addnl. address two topics that will be of particular interest to the Pfam community.  First, we explain the definition and use of family-specific, manually curated gathering thresholds.  Second, we discuss some of the features of domains of unknown function (also known as DUFs), which constitute a rapidly growing class of families within Pfam.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonKjUY4YgVhLVg90H21EOLACvtfcHk0lgtP8ij49UCpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12hurvK&md5=1acf93ff21d746e21cd96b6626efa0a7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1065%26sid%3Dliteratum%253Aachs%26aulast%3DPunta%26aufirst%3DM.%26aulast%3DCoggill%26aufirst%3DP.%2BC.%26aulast%3DEberhardt%26aufirst%3DR.%2BY.%26aulast%3DMistry%26aufirst%3DJ.%26aulast%3DTate%26aufirst%3DJ.%26aulast%3DBoursnell%26aufirst%3DC.%26aulast%3DPang%26aufirst%3DN.%26aulast%3DForslund%26aufirst%3DK.%26aulast%3DCeric%26aufirst%3DG.%26aulast%3DClements%26aufirst%3DJ.%26aulast%3DHeger%26aufirst%3DA.%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DBateman%26aufirst%3DA.%26aulast%3DFinn%26aufirst%3DR.%2BD.%26atitle%3DThe%2520Pfam%2520protein%2520families%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD290%26epage%3D301%26doi%3D10.1093%2Fnar%2Fgkr1065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaLonde, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindvall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semus, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peishoff, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Head, M. S.</span><span> </span><span class="NLM_article-title">A critical assessment of docking programs and scoring functions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5912</span><span class="NLM_x">–</span> <span class="NLM_lpage">5931</span><span class="refDoi"> DOI: 10.1021/jm050362n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050362n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5912-5931&author=G.+L.+Warrenauthor=C.+W.+Andrewsauthor=A.+M.+Capelliauthor=B.+Clarkeauthor=J.+LaLondeauthor=M.+H.+Lambertauthor=M.+Lindvallauthor=N.+Nevinsauthor=S.+F.+Semusauthor=S.+Sengerauthor=G.+Tedescoauthor=I.+D.+Wallauthor=J.+M.+Woolvenauthor=C.+E.+Peishoffauthor=M.+S.+Head&title=A+critical+assessment+of+docking+programs+and+scoring+functions&doi=10.1021%2Fjm050362n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Assessment of Docking Programs and Scoring Functions</span></div><div class="casAuthors">Warren, Gregory L.; Andrews, C. Webster; Capelli, Anna-Maria; Clarke, Brian; LaLonde, Judith; Lambert, Millard H.; Lindvall, Mika; Nevins, Neysa; Semus, Simon F.; Senger, Stefan; Tedesco, Giovanna; Wall, Ian D.; Woolven, James M.; Peishoff, Catherine E.; Head, Martha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5912-5931</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking is a computational technique that samples conformations of small mols. in protein binding sites; scoring functions are used to assess which of these conformations best complements the protein binding site.  An evaluation of 10 docking programs and 37 scoring functions was conducted against eight proteins of seven protein types for three tasks: binding mode prediction, virtual screening for lead identification, and rank-ordering by affinity for lead optimization.  All of the docking programs were able to generate ligand conformations similar to crystallog. detd. protein/ligand complex structures for at least one of the targets.  However, scoring functions were less successful at distinguishing the crystallog. conformation from the set of docked poses.  Docking programs identified active compds. from a pharmaceutically relevant pool of decoy compds.; however, no single program performed well for all of the targets.  For prediction of compd. affinity, none of the docking programs or scoring functions made a useful prediction of ligand binding affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJn8_WHSjurVg90H21EOLACvtfcHk0lgDgwyQqJjOoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D&md5=463ab40c3d23e67b62df408fc5e7f08d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm050362n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050362n%26sid%3Dliteratum%253Aachs%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DCapelli%26aufirst%3DA.%2BM.%26aulast%3DClarke%26aufirst%3DB.%26aulast%3DLaLonde%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DSemus%26aufirst%3DS.%2BF.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26aulast%3DHead%26aufirst%3DM.%2BS.%26atitle%3DA%2520critical%2520assessment%2520of%2520docking%2520programs%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5912%26epage%3D5931%26doi%3D10.1021%2Fjm050362n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Koes, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camacho, C. J.</span><span> </span><span class="NLM_article-title">Pharmer: efficient and exact pharmacophore search</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1307</span><span class="NLM_x">–</span> <span class="NLM_lpage">1314</span><span class="refDoi"> DOI: 10.1021/ci200097m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200097m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFSltL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1307-1314&author=D.+R.+Koesauthor=C.+J.+Camacho&title=Pharmer%3A+efficient+and+exact+pharmacophore+search&doi=10.1021%2Fci200097m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmer: Efficient and Exact Pharmacophore Search</span></div><div class="casAuthors">Koes, David Ryan; Camacho, Carlos J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1307-1314</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pharmacophore search is a key component of many drug discovery efforts.  Pharmer is a new computational approach to pharmacophore search that scales with the breadth and complexity of the query, not the size of the compd. library being screened.  Two novel methods for organizing pharmacophore data, the Pharmer KDB-tree and Bloom fingerprints, enable Pharmer to perform an exact pharmacophore search of almost two million structures in less than a minute.  In general, Pharmer is more than an order of magnitude faster than existing technologies.  The complete source code is available under an open-source license at http://pharmer.sourceforge.net.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVdJ0tI_jotrVg90H21EOLACvtfcHk0lgDgwyQqJjOoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFSltL4%253D&md5=cd9dc10f5ebf14f931829849395cde39</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci200097m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200097m%26sid%3Dliteratum%253Aachs%26aulast%3DKoes%26aufirst%3DD.%2BR.%26aulast%3DCamacho%26aufirst%3DC.%2BJ.%26atitle%3DPharmer%253A%2520efficient%2520and%2520exact%2520pharmacophore%2520search%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1307%26epage%3D1314%26doi%3D10.1021%2Fci200097m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Mordalski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosciolek, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristiansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sylte, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bojarski, A. J.</span><span> </span><span class="NLM_article-title">Protein binding site analysis by means of structural interaction fingerprint patterns</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">6816</span><span class="NLM_x">–</span> <span class="NLM_lpage">6819</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.09.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.bmcl.2011.09.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=21974955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKktr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=6816-6819&author=S.+Mordalskiauthor=T.+Kosciolekauthor=K.+Kristiansenauthor=I.+Sylteauthor=A.+J.+Bojarski&title=Protein+binding+site+analysis+by+means+of+structural+interaction+fingerprint+patterns&doi=10.1016%2Fj.bmcl.2011.09.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Protein binding site analysis by means of structural interaction fingerprint patterns</span></div><div class="casAuthors">Mordalski, Stefan; Kosciolek, Tomasz; Kristiansen, Kurt; Sylte, Ingebrigt; Bojarski, Andrzej J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6816-6819</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We introduce a new approach to the known concept of interaction profiles, based on Structural Interaction Fingerprints (SIFt), for precise and rapid binding site description.  A set of scripts for batch generation and anal. of SIFt were prepd., and the implementation is computationally efficient and supports parallelization.  It is based on a 9-digit binary interaction pattern that describes phys. ligand-protein interactions in structures and models of ligand-protein complexes.  The tool performs anal. and identifies binding site residues (crucial and auxiliary) and classifies interactions according to type (hydrophobic, arom., charge, polar, side chain, and backbone).  It is convenient and easy to use, and gives manageable output data for both, interpretation and further processing.  In the presented Letter, SIFts are applied to analyze binding sites in models of antagonist-5-HT7 receptor complexes and structures of cyclin dependent kinase 2-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryunTDaocwe7Vg90H21EOLACvtfcHk0lgDgwyQqJjOoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKktr3E&md5=cddbdf237f0c8ae19e308ad47bbb1919</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.09.027%26sid%3Dliteratum%253Aachs%26aulast%3DMordalski%26aufirst%3DS.%26aulast%3DKosciolek%26aufirst%3DT.%26aulast%3DKristiansen%26aufirst%3DK.%26aulast%3DSylte%26aufirst%3DI.%26aulast%3DBojarski%26aufirst%3DA.%2BJ.%26atitle%3DProtein%2520binding%2520site%2520analysis%2520by%2520means%2520of%2520structural%2520interaction%2520fingerprint%2520patterns%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D6816%26epage%3D6819%26doi%3D10.1016%2Fj.bmcl.2011.09.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Optimizing fragment and scaffold docking by use of molecular interaction fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">207</span><span class="refDoi"> DOI: 10.1021/ci600342e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+fragment+and+scaffold+docking+by+use+of+molecular+interaction+fingerprints&doi=10.1021%2Fci600342e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0lgcigYU2uwWUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520fragment%2520and%2520scaffold%2520docking%2520by%2520use%2520of%2520molecular%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D195%26epage%3D207%26doi%3D10.1021%2Fci600342e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Mpamhanga, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLay, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Knowledge-based interaction fingerprint scoring: a simple method for improving the effectiveness of fast scoring functions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">686</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1021/ci050420d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050420d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsVGjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=686-698&author=C.+P.+Mpamhangaauthor=B.+Chenauthor=I.+M.+McLayauthor=P.+Willett&title=Knowledge-based+interaction+fingerprint+scoring%3A+a+simple+method+for+improving+the+effectiveness+of+fast+scoring+functions&doi=10.1021%2Fci050420d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge-Based Interaction Fingerprint Scoring: A Simple Method for Improving the Effectiveness of Fast Scoring Functions</span></div><div class="casAuthors">Mpamhanga, Chidochangu P.; Chen, Beining; McLay, Iain M.; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">686-698</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new method for the postprocessing of docking outputs has been developed, based on encoding putative 3D binding modes (docking solns.) as ligand-protein interactions into simple bit strings, a method analogous to the structural interaction fingerprint.  Instead of employing traditional scoring functions, the method uses a series of new, knowledge-based scores derived from the similarity of the bit strings for each docking soln. to that of a known ref. binding mode.  A GOLD docking study was carried out using the Bissantz estrogen receptor antagonist set along with the new scoring method.  Superior recovery rates, with up to 2-fold enrichments, were obsd. when the new knowledge-based scoring was compared to the GOLD fitness score.  In addn., top ranking sets of mols. (actives and potential actives or decoys) were structurally diverse with low mol. wts. and structural complexities.  Principal component anal. and clustering of the fingerprints permits the easy sepn. of active from inactive binding modes and the visualization of diverse binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZfTfk88wvHLVg90H21EOLACvtfcHk0lgcigYU2uwWUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsVGjuw%253D%253D&md5=0fecc7a1894b8bf0e53f077deefc2a85</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci050420d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050420d%26sid%3Dliteratum%253Aachs%26aulast%3DMpamhanga%26aufirst%3DC.%2BP.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DMcLay%26aufirst%3DI.%2BM.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DKnowledge-based%2520interaction%2520fingerprint%2520scoring%253A%2520a%2520simple%2520method%2520for%2520improving%2520the%2520effectiveness%2520of%2520fast%2520scoring%2520functions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D686%26epage%3D698%26doi%3D10.1021%2Fci050420d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Perez-Nueno, V. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabal, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borrell, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teixido, J.</span><span> </span><span class="NLM_article-title">APIF: a new interaction fingerprint based on atom pairs and its application to virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1245</span><span class="NLM_x">–</span> <span class="NLM_lpage">1260</span><span class="refDoi"> DOI: 10.1021/ci900043r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900043r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVSltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=1245-1260&author=V.+I.+Perez-Nuenoauthor=O.+Rabalauthor=J.+I.+Borrellauthor=J.+Teixido&title=APIF%3A+a+new+interaction+fingerprint+based+on+atom+pairs+and+its+application+to+virtual+screening&doi=10.1021%2Fci900043r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">APIF: A New Interaction Fingerprint Based on Atom Pairs and Its Application to Virtual Screening</span></div><div class="casAuthors">Perez-Nueno, Violeta I.; Rabal, Obdulia; Borrell, Jose I.; Teixido, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1245-1260</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new interaction fingerprint (IF) called APIF (atom-pairs-based interaction fingerprint) has been developed for postprocessing protein-ligand docking results.  Unlike other existing fingerprints which employ abs. locations of individual interactions, APIF considers the relative positions of pairs of interacting atoms.  Docking-based virtual screening was performed with GOLD using the crystal structures of trypsin, rhinovirus, HIV protease, carboxypeptidase, and estrogen receptor-α as targets.  A score derived from the similarity of the bit strings for each docking soln. to that of a known ref. binding mode was obtained.  Comparisons between APIF, GoldScore function, and std. interaction fingerprint (CHIF) scores were performed using enrichment plots.  Superior recovery rates were obsd. in the IF score cases.  Comparable results were achieved by using either of the two interaction fingerprints, substantially improving GoldScore function enrichment factors.  Binding mode analyses were also carried out in order to study the best method for selecting conformations with a binding mode similar to that of the ref. crystd. complex.  These showed that the first conformations retrieved by interaction fingerprint scores had a more similar binding mode to the ref. complex than those retrieved by the GoldScore function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxejfqiHaBwrVg90H21EOLACvtfcHk0lgcigYU2uwWUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVSltb4%253D&md5=ae8a172d6de2aba35748b02ec38e3bfe</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci900043r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900043r%26sid%3Dliteratum%253Aachs%26aulast%3DPerez-Nueno%26aufirst%3DV.%2BI.%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DBorrell%26aufirst%3DJ.%2BI.%26aulast%3DTeixido%26aufirst%3DJ.%26atitle%3DAPIF%253A%2520a%2520new%2520interaction%2520fingerprint%2520based%2520on%2520atom%2520pairs%2520and%2520its%2520application%2520to%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D1245%26epage%3D1260%26doi%3D10.1021%2Fci900043r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geppert, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Searching for target-selective compounds using different combinations of multiclass support vector machine ranking methods, kernel functions, and fingerprint descriptors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">582</span><span class="NLM_x">–</span> <span class="NLM_lpage">592</span><span class="refDoi"> DOI: 10.1021/ci800441c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800441c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVKkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=582-592&author=A.+M.+Wassermannauthor=H.+Geppertauthor=J.+Bajorath&title=Searching+for+target-selective+compounds+using+different+combinations+of+multiclass+support+vector+machine+ranking+methods%2C+kernel+functions%2C+and+fingerprint+descriptors&doi=10.1021%2Fci800441c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Searching for Target-Selective Compounds Using Different Combinations of Multiclass Support Vector Machine Ranking Methods, Kernel Functions, and Fingerprint Descriptors</span></div><div class="casAuthors">Wassermann, Anne Mai; Geppert, Hanna; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">582-592</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of small chem. compds. that are selective for a target protein over one or more closely related members of the same family is of high relevance for applications in chem. biol.  Conventional 2D similarity searching using known selective mols. as templates has recently been found to preferentially detect selective over non-selective and inactive database compds.  To improve the initially obsd. search performance, we have attempted to use 2D fingerprints as descriptors for support vector machine (SVM)-based selectivity searching.  Different from typically applied binary SVM compd. classification, SVM anal. has been adapted here for multiclass predictions and compd. ranking to distinguish between selective, active but non-selective, and inactive compds.  In systematic database search calcns., we tested combinations of four alternative SVM ranking schemes, four different kernel functions, and four fingerprints and were able to further improve selectivity search performance by effectively removing non-selective mols. from high ranking positions while retaining high recall of selective compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplfCnoxqEkrrVg90H21EOLACvtfcHk0lgcigYU2uwWUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVKkt7w%253D&md5=6b946bdf10fe4cd6c9fd19f8d2b70eb3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci800441c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800441c%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DGeppert%26aufirst%3DH.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSearching%2520for%2520target-selective%2520compounds%2520using%2520different%2520combinations%2520of%2520multiclass%2520support%2520vector%2520machine%2520ranking%2520methods%252C%2520kernel%2520functions%252C%2520and%2520fingerprint%2520descriptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D582%26epage%3D592%26doi%3D10.1021%2Fci800441c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuaqui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span> </span><span class="NLM_article-title">Knowledge-based design of target-focused libraries using protein-ligand interaction constraints</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">490</span><span class="NLM_x">–</span> <span class="NLM_lpage">500</span><span class="refDoi"> DOI: 10.1021/jm050381x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050381x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=490-500&author=Z.+Dengauthor=C.+Chuaquiauthor=J.+Singh&title=Knowledge-based+design+of+target-focused+libraries+using+protein-ligand+interaction+constraints&doi=10.1021%2Fjm050381x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm050381x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050381x%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DKnowledge-based%2520design%2520of%2520target-focused%2520libraries%2520using%2520protein-ligand%2520interaction%2520constraints%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D490%26epage%3D500%26doi%3D10.1021%2Fjm050381x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Chupakhin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baskin, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varnek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Predicting ligand binding modes from neural networks trained on protein-ligand interaction fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">–</span> <span class="NLM_lpage">772</span><span class="refDoi"> DOI: 10.1021/ci300200r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300200r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslOgs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=763-772&author=V.+Chupakhinauthor=G.+Marcouauthor=I.+Baskinauthor=A.+Varnekauthor=D.+Rognan&title=Predicting+ligand+binding+modes+from+neural+networks+trained+on+protein-ligand+interaction+fingerprints&doi=10.1021%2Fci300200r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Ligand Binding Modes from Neural Networks Trained on Protein-Ligand Interaction Fingerprints</span></div><div class="casAuthors">Chupakhin, Vladimir; Marcou, Gilles; Baskin, Igor; Varnek, Alexandre; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">763-772</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors herewith present a novel approach to predict protein-ligand binding modes from the single two-dimensional structure of the ligand.  Known protein-ligand x-ray structures were converted into binary bit strings encoding protein-ligand interactions.  An artificial neural network was then set up to first learn and then predict protein-ligand interaction fingerprints from simple ligand descriptors.  Specific models were constructed for three targets (CDK2, p38-α, HSP90-α) and 146 ligands for which protein-ligand x-ray structures are available.  These models were able to predict protein-ligand interaction fingerprints and to discriminate important features from minor interactions.  Predicted interaction fingerprints were successfully used as descriptors to discriminate true ligands from decoys by virtual screening.  In some but not all cases, the predicted interaction fingerprints furthermore enable to efficiently rerank cross-docking poses and prioritize the best possible docking solns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-ZJ0Zoe63l7Vg90H21EOLACvtfcHk0lgtwCLit3l6KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslOgs78%253D&md5=85e39717c824c274fbb62e39f1873dcb</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci300200r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300200r%26sid%3Dliteratum%253Aachs%26aulast%3DChupakhin%26aufirst%3DV.%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DBaskin%26aufirst%3DI.%26aulast%3DVarnek%26aufirst%3DA.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DPredicting%2520ligand%2520binding%2520modes%2520from%2520neural%2520networks%2520trained%2520on%2520protein-ligand%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D763%26epage%3D772%26doi%3D10.1021%2Fci300200r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A unified statistical model to support local sequence order independent similarity searching for ligand-binding sites and its application to genome-based drug discovery</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">i305</span><span class="NLM_x">–</span> <span class="NLM_lpage">312</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btp220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1093%2Fbioinformatics%2Fbtp220" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=i305-312&author=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourne&title=A+unified+statistical+model+to+support+local+sequence+order+independent+similarity+searching+for+ligand-binding+sites+and+its+application+to+genome-based+drug+discovery&doi=10.1093%2Fbioinformatics%2Fbtp220"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp220%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520unified%2520statistical%2520model%2520to%2520support%2520local%2520sequence%2520order%2520independent%2520similarity%2520searching%2520for%2520ligand-binding%2520sites%2520and%2520its%2520application%2520to%2520genome-based%2520drug%2520discovery%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3Di305%26epage%3D312%26doi%3D10.1093%2Fbioinformatics%2Fbtp220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Briedis, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starr, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Analysis of the human kinome using methods including fold recognition reveals two novel kinases</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1597</span><span class="refDoi"> DOI: 10.1371/journal.pone.0001597</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pone.0001597" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e1597&author=K.+M.+Briedisauthor=A.+Starrauthor=P.+E.+Bourne&title=Analysis+of+the+human+kinome+using+methods+including+fold+recognition+reveals+two+novel+kinases&doi=10.1371%2Fjournal.pone.0001597"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0001597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0001597%26sid%3Dliteratum%253Aachs%26aulast%3DBriedis%26aufirst%3DK.%2BM.%26aulast%3DStarr%26aufirst%3DA.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DAnalysis%2520of%2520the%2520human%2520kinome%2520using%2520methods%2520including%2520fold%2520recognition%2520reveals%2520two%2520novel%2520kinases%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26spage%3De1597%26doi%3D10.1371%2Fjournal.pone.0001597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Detecting evolutionary relationships across existing fold space, using sequence order-independent profile-profile alignments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">5441</span><span class="NLM_x">–</span> <span class="NLM_lpage">5446</span><span class="refDoi"> DOI: 10.1073/pnas.0704422105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1073%2Fpnas.0704422105" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=5441-5446&author=L.+Xieauthor=P.+E.+Bourne&title=Detecting+evolutionary+relationships+across+existing+fold+space%2C+using+sequence+order-independent+profile-profile+alignments&doi=10.1073%2Fpnas.0704422105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0704422105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0704422105%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDetecting%2520evolutionary%2520relationships%2520across%2520existing%2520fold%2520space%252C%2520using%2520sequence%2520order-independent%2520profile-profile%2520alignments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D5441%26epage%3D5446%26doi%3D10.1073%2Fpnas.0704422105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Konc, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janezic, D.</span><span> </span><span class="NLM_article-title">Binding site comparison for function prediction and pharmaceutical discovery</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25C</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1016/j.sbi.2013.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.sbi.2013.11.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25C&publication_year=2014&pages=34-39&author=J.+Koncauthor=D.+Janezic&title=Binding+site+comparison+for+function+prediction+and+pharmaceutical+discovery&doi=10.1016%2Fj.sbi.2013.11.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2013.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2013.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJanezic%26aufirst%3DD.%26atitle%3DBinding%2520site%2520comparison%2520for%2520function%2520prediction%2520and%2520pharmaceutical%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2014%26volume%3D25C%26spage%3D34%26epage%3D39%26doi%3D10.1016%2Fj.sbi.2013.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Towards structural systems pharmacology to study complex diseases and personalized medicine</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003554</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1003554" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003554&author=L.+Xieauthor=X.+Geauthor=H.+Tanauthor=L.+Xieauthor=Y.+Zhangauthor=T.+Hartauthor=X.+Yangauthor=P.+E.+Bourne&title=Towards+structural+systems+pharmacology+to+study+complex+diseases+and+personalized+medicine&doi=10.1371%2Fjournal.pcbi.1003554"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003554%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DGe%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHart%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DTowards%2520structural%2520systems%2520pharmacology%2520to%2520study%2520complex%2520diseases%2520and%2520personalized%2520medicine%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003554%26doi%3D10.1371%2Fjournal.pcbi.1003554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Nisius, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sha, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span> </span><span class="NLM_article-title">Structure-based computational analysis of protein binding sites for function and druggability prediction</span> <span class="citation_source-journal">J. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">159</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1016/j.jbiotec.2011.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.jbiotec.2011.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=22197384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2012&pages=123-134&author=B.+Nisiusauthor=F.+Shaauthor=H.+Gohlke&title=Structure-based+computational+analysis+of+protein+binding+sites+for+function+and+druggability+prediction&doi=10.1016%2Fj.jbiotec.2011.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based computational analysis of protein binding sites for function and druggability prediction</span></div><div class="casAuthors">Nisius, Britta; Sha, Fan; Gohlke, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">123-134</span>CODEN:
                <span class="NLM_cas:coden">JBITD4</span>;
        ISSN:<span class="NLM_cas:issn">0168-1656</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein binding sites are the places where mol. interactions occur.  Thus, the anal. of protein binding sites is of crucial importance to understand the biol. processes proteins are involved in.  Herein, we focus on the computational anal. of protein binding sites and present structure-based methods that enable function prediction for orphan proteins and prediction of target druggability.  We present the general ideas behind these methods, with a special emphasis on the scopes and limitations of these methods and their validation.  Addnl., we present some successful applications of computational binding site anal. to emphasize the practical importance of these methods for biotechnol./bioeconomy and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXzgcXRf62cbVg90H21EOLACvtfcHk0lgNWGuzFrKeLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahurs%253D&md5=9d8c5a3a41abd1ce0444ed23c3b3f7a9</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.jbiotec.2011.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbiotec.2011.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DNisius%26aufirst%3DB.%26aulast%3DSha%26aufirst%3DF.%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DStructure-based%2520computational%2520analysis%2520of%2520protein%2520binding%2520sites%2520for%2520function%2520and%2520druggability%2520prediction%26jtitle%3DJ.%2520Biotechnol.%26date%3D2012%26volume%3D159%26spage%3D123%26epage%3D134%26doi%3D10.1016%2Fj.jbiotec.2011.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A robust and efficient algorithm for the shape description of protein structures and its application in predicting ligand binding sites</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 4</span><span class="NLM_x">) </span> <span class="NLM_fpage">S9</span><span class="refDoi"> DOI: 10.1186/1471-2105-8-S4-S9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1186%2F1471-2105-8-S4-S9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=17570152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BD2szotFyitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=S9&issue=Suppl.+4&author=L.+Xieauthor=P.+E.+Bourne&title=A+robust+and+efficient+algorithm+for+the+shape+description+of+protein+structures+and+its+application+in+predicting+ligand+binding+sites&doi=10.1186%2F1471-2105-8-S4-S9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A robust and efficient algorithm for the shape description of protein structures and its application in predicting ligand binding sites</span></div><div class="casAuthors">Xie Lei; Bourne Philip E</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8 Suppl 4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">S9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  An accurate description of protein shape derived from protein structure is necessary to establish an understanding of protein-ligand interactions, which in turn will lead to improved methods for protein-ligand docking and binding site analysis.  Most current shape descriptors characterize only the local properties of protein structure using an all-atom representation and are slow to compute.  We need new shape descriptors that have the ability to capture both local and global structural information, are robust for application to models and low quality structures and are computationally efficient to permit high throughput analysis of protein structures.  RESULTS:  We introduce a new shape description that requires only the Calpha atoms to represent the protein structure, thus making it both fast and suitable for use on models and low quality structures.  The notion of a geometric potential is introduced to quantitatively describe the shape of the structure.  This geometric potential is dependent on both the global shape of the protein structure as well as the surrounding environment of each residue.  When applying the geometric potential for binding site prediction, approximately 85% of known binding sites can be accurately identified with above 50% residue coverage and 80% specificity.  Moreover, the algorithm is fast enough for proteome-scale applications.  Proteins with fewer than 500 amino acids can be scanned in less than two seconds.  CONCLUSION:  The reduced representation of the protein structure combined with the geometric potential provides a fast, quantitative description of protein-ligand binding sites with potential for use in large-scale predictions, comparisons and analysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3yUgGb2q8-gpJitSxOdIlfW6udTcc2eZsI3n0UDaYnrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2szotFyitQ%253D%253D&md5=bd06c6a317d572d275df0b4d7d317af7</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-8-S4-S9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-8-S4-S9%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520robust%2520and%2520efficient%2520algorithm%2520for%2520the%2520shape%2520description%2520of%2520protein%2520structures%2520and%2520its%2520application%2520in%2520predicting%2520ligand%2520binding%2520sites%26jtitle%3DBMC%2520Bioinf.%26date%3D2007%26volume%3D8%26issue%3DSuppl.%25204%26spage%3DS9%26doi%3D10.1186%2F1471-2105-8-S4-S9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: identification of the protein-ligand binding network to explain the side effects of CETP inhibitors</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000387</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1000387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19436720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BD1MzktVKnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000387&author=L.+Xieauthor=J.+Liauthor=L.+Xieauthor=P.+E.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+identification+of+the+protein-ligand+binding+network+to+explain+the+side+effects+of+CETP+inhibitors&doi=10.1371%2Fjournal.pcbi.1000387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery using chemical systems biology: identification of the protein-ligand binding network to explain the side effects of CETP inhibitors</span></div><div class="casAuthors">Xie Li; Li Jerry; Xie Lei; Bourne Philip E</div><div class="citationInfo"><span class="NLM_cas:title">PLoS computational biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e1000387</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Systematic identification of protein-drug interaction networks is crucial to correlate complex modes of drug action to clinical indications.  We introduce a novel computational strategy to identify protein-ligand binding profiles on a genome-wide scale and apply it to elucidating the molecular mechanisms associated with the adverse drug effects of Cholesteryl Ester Transfer Protein (CETP) inhibitors.  CETP inhibitors are a new class of preventive therapies for the treatment of cardiovascular disease.  However, clinical studies indicated that one CETP inhibitor, Torcetrapib, has deadly off-target effects as a result of hypertension, and hence it has been withdrawn from phase III clinical trials.  We have identified a panel of off-targets for Torcetrapib and other CETP inhibitors from the human structural genome and map those targets to biological pathways via the literature.  The predicted protein-ligand network is consistent with experimental results from multiple sources and reveals that the side-effect of CETP inhibitors is modulated through the combinatorial control of multiple interconnected pathways.  Given that combinatorial control is a common phenomenon observed in many biological processes, our findings suggest that adverse drug effects might be minimized by fine-tuning multiple off-target interactions using single or multiple therapies.  This work extends the scope of chemogenomics approaches and exemplifies the role that systems biology has in the future of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSfisyXvTOXkCnGiHuhFGv8fW6udTcc2eZsI3n0UDaYnrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzktVKnsA%253D%253D&md5=9ed4e6023419132e66f8800d29676cb0</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000387%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520identification%2520of%2520the%2520protein-ligand%2520binding%2520network%2520to%2520explain%2520the%2520side%2520effects%2520of%2520CETP%2520inhibitors%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3De1000387%26doi%3D10.1371%2Fjournal.pcbi.1000387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Chang, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palsson, B. O.</span><span> </span><span class="NLM_article-title">Drug off-target effects predicted using structural analysis in the context of a metabolic network model</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000938</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1000938" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=e1000938&author=R.+L.+Changauthor=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourneauthor=B.+O.+Palsson&title=Drug+off-target+effects+predicted+using+structural+analysis+in+the+context+of+a+metabolic+network+model&doi=10.1371%2Fjournal.pcbi.1000938"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000938%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DR.%2BL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DPalsson%26aufirst%3DB.%2BO.%26atitle%3DDrug%2520off-target%2520effects%2520predicted%2520using%2520structural%2520analysis%2520in%2520the%2520context%2520of%2520a%2520metabolic%2520network%2520model%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3De1000938%26doi%3D10.1371%2Fjournal.pcbi.1000938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchmeier, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: repositioning the safe medicine Comtan to treat multi-drug and extensively drug resistant tuberculosis</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000423</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1000423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19578428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BD1MvmsFGntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000423&author=S.+L.+Kinningsauthor=N.+Liuauthor=N.+Buchmeierauthor=P.+J.+Tongeauthor=L.+Xieauthor=P.+E.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+repositioning+the+safe+medicine+Comtan+to+treat+multi-drug+and+extensively+drug+resistant+tuberculosis&doi=10.1371%2Fjournal.pcbi.1000423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery using chemical systems biology: repositioning the safe medicine Comtan to treat multi-drug and extensively drug resistant tuberculosis</span></div><div class="casAuthors">Kinnings Sarah L; Liu Nina; Buchmeier Nancy; Tonge Peter J; Xie Lei; Bourne Philip E</div><div class="citationInfo"><span class="NLM_cas:title">PLoS computational biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1000423</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The rise of multi-drug resistant (MDR) and extensively drug resistant (XDR) tuberculosis around the world, including in industrialized nations, poses a great threat to human health and defines a need to develop new, effective and inexpensive anti-tubercular agents.  Previously we developed a chemical systems biology approach to identify off-targets of major pharmaceuticals on a proteome-wide scale.  In this paper we further demonstrate the value of this approach through the discovery that existing commercially available drugs, prescribed for the treatment of Parkinson's disease, have the potential to treat MDR and XDR tuberculosis.  These drugs, entacapone and tolcapone, are predicted to bind to the enzyme InhA and directly inhibit substrate binding.  The prediction is validated by in vitro and InhA kinetic assays using tablets of Comtan, whose active component is entacapone.  The minimal inhibition concentration (MIC(99)) of entacapone for Mycobacterium tuberculosis (M.tuberculosis) is approximately 260.0 microM, well below the toxicity concentration determined by an in vitro cytotoxicity model using a human neuroblastoma cell line.  Moreover, kinetic assays indicate that Comtan inhibits InhA activity by 47.0% at an entacapone concentration of approximately 80 microM.  Thus the active component in Comtan represents a promising lead compound for developing a new class of anti-tubercular therapeutics with excellent safety profiles.  More generally, the protocol described in this paper can be included in a drug discovery pipeline in an effort to discover novel drug leads with desired safety profiles, and therefore accelerate the development of new drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSY95JLzAotpaNc3HjJRoFAfW6udTcc2eZUW28XaXaTR7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvmsFGntg%253D%253D&md5=9b5e736b9cf022446f4c0530bbe666af</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000423%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DBuchmeier%26aufirst%3DN.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520repositioning%2520the%2520safe%2520medicine%2520Comtan%2520to%2520treat%2520multi-drug%2520and%2520extensively%2520drug%2520resistant%2520tuberculosis%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3De1000423%26doi%3D10.1371%2Fjournal.pcbi.1000423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Structure-based systems biology for analyzing off-target binding</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">189</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.1016/j.sbi.2011.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.sbi.2011.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=21292475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1Kis7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=189-199&author=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourne&title=Structure-based+systems+biology+for+analyzing+off-target+binding&doi=10.1016%2Fj.sbi.2011.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based systems biology for analyzing off-target binding</span></div><div class="casAuthors">Xie, Lei; Xie, Li; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">189-199</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Here off-target binding implies the binding of a small mol. of therapeutic interest to a protein target other than the primary target for which it was intended.  Increasingly such off-targeting appears to be the norm rather than the exception, rational drug design notwithstanding, and can lead to detrimental side-effects, or opportunities to reposition a therapeutic agent to treat a different condition.  Not surprisingly, there is significant interest in detg. a priori what off-targets exist on a proteome-wide scale.  Beyond detg. putative off-targets is the need to understand the impact of such binding on the complete biol. system, with the ultimate goal of being able to predict the phenotypic outcome.  While a very ambitious goal, some progress is being made.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrElfKnuS2XpbVg90H21EOLACvtfcHk0lhUZ02jlDKBNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1Kis7w%253D&md5=76036b3b2a7bd319f437d5c875bccbde</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2011.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2011.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DStructure-based%2520systems%2520biology%2520for%2520analyzing%2520off-target%2520binding%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D189%26epage%3D199%26doi%3D10.1016%2Fj.sbi.2011.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Garijo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinnings, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gil, Y.</span><span> </span><span class="NLM_article-title">Quantifying reproducibility in computational biology: the case of the tuberculosis drugome</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e80278</span><span class="refDoi"> DOI: 10.1371/journal.pone.0080278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pone.0080278" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e80278&author=D.+Garijoauthor=S.+Kinningsauthor=L.+Xieauthor=L.+Xieauthor=Y.+Zhangauthor=P.+E.+Bourneauthor=Y.+Gil&title=Quantifying+reproducibility+in+computational+biology%3A+the+case+of+the+tuberculosis+drugome&doi=10.1371%2Fjournal.pone.0080278"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0080278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0080278%26sid%3Dliteratum%253Aachs%26aulast%3DGarijo%26aufirst%3DD.%26aulast%3DKinnings%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DGil%26aufirst%3DY.%26atitle%3DQuantifying%2520reproducibility%2520in%2520computational%2520biology%253A%2520the%2520case%2520of%2520the%2520tuberculosis%2520drugome%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De80278%26doi%3D10.1371%2Fjournal.pone.0080278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Durrant, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amaro, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urbaniak, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haapalainen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Guilmi, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wunder, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">A multidimensional strategy to detect polypharmacological targets in the absence of structural and sequence homology</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000648</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1000648" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=e1000648&author=J.+D.+Durrantauthor=R.+E.+Amaroauthor=L.+Xieauthor=M.+D.+Urbaniakauthor=M.+A.+Fergusonauthor=A.+Haapalainenauthor=Z.+Chenauthor=A.+M.+Di+Guilmiauthor=F.+Wunderauthor=P.+E.+Bourneauthor=J.+A.+McCammon&title=A+multidimensional+strategy+to+detect+polypharmacological+targets+in+the+absence+of+structural+and+sequence+homology&doi=10.1371%2Fjournal.pcbi.1000648"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000648%26sid%3Dliteratum%253Aachs%26aulast%3DDurrant%26aufirst%3DJ.%2BD.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DUrbaniak%26aufirst%3DM.%2BD.%26aulast%3DFerguson%26aufirst%3DM.%2BA.%26aulast%3DHaapalainen%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DDi%2BGuilmi%26aufirst%3DA.%2BM.%26aulast%3DWunder%26aufirst%3DF.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DA%2520multidimensional%2520strategy%2520to%2520detect%2520polypharmacological%2520targets%2520in%2520the%2520absence%2520of%2520structural%2520and%2520sequence%2520homology%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3De1000648%26doi%3D10.1371%2Fjournal.pcbi.1000648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Mycobacterium tuberculosis drugome and its polypharmacological implications</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000976</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1000976" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=e1000976&author=S.+L.+Kinningsauthor=L.+Xieauthor=K.+H.+Fungauthor=R.+M.+Jacksonauthor=L.+Xieauthor=P.+E.+Bourne&title=The+Mycobacterium+tuberculosis+drugome+and+its+polypharmacological+implications&doi=10.1371%2Fjournal.pcbi.1000976"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000976%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DFung%26aufirst%3DK.%2BH.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Mycobacterium%2520tuberculosis%2520drugome%2520and%2520its%2520polypharmacological%2520implications%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3De1000976%26doi%3D10.1371%2Fjournal.pcbi.1000976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evangelidis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: weak inhibition of multiple kinases may contribute to the anti-cancer effect of nelfinavir</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1002037</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1002037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1002037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=21552547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsl2nsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002037&author=L.+Xieauthor=T.+Evangelidisauthor=L.+Xieauthor=P.+E.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+weak+inhibition+of+multiple+kinases+may+contribute+to+the+anti-cancer+effect+of+nelfinavir&doi=10.1371%2Fjournal.pcbi.1002037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery using chemical systems biology: weak inhibition of multiple kinases may contribute to the anti-cancer effect of Nelfinavir</span></div><div class="casAuthors">Xie, Li; Evangelidis, Thomas; Xie, Lei; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e1002037</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Nelfinavir is a potent HIV-protease inhibitor with pleiotropic effects in cancer cells.  Exptl. studies connect its anti-cancer effects to the suppression of the Akt signaling pathway, but the actual mol. targets remain unknown.  Using a structural proteome-wide off-target pipeline, which integrates mol. dynamics simulation and MM/GBSA free energy calcns. with ligand binding site comparison and biol. network anal., we identified putative human off-targets of Nelfinavir and analyzed the impact on the assocd. biol. processes.  Our results suggest that Nelfinavir is able to inhibit multiple members of the protein kinase-like superfamily, which are involved in the regulation of cellular processes vital for carcinogenesis and metastasis.  The computational predictions are supported by kinase activity assays and are consistent with existing exptl. and clin. evidence.  This finding provides a mol. basis to explain the broad-spectrum anti-cancer effect of Nelfinavir and presents opportunities to optimize the drug as a targeted polypharmacol. agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGookAtcKasU9bVg90H21EOLACvtfcHk0libsIvwLsmbFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsl2nsrw%253D&md5=021c009e7d74cc1085577a5e11bfb23b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002037%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DEvangelidis%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520weak%2520inhibition%2520of%2520multiple%2520kinases%2520may%2520contribute%2520to%2520the%2520anti-cancer%2520effect%2520of%2520nelfinavir%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3De1002037%26doi%3D10.1371%2Fjournal.pcbi.1002037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Novel computational approaches to polypharmacology as a means to define responses to individual drugs</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">379</span><span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010611-134630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1146%2Fannurev-pharmtox-010611-134630" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=361-379&author=L.+Xieauthor=L.+Xieauthor=S.+L.+Kinningsauthor=P.+E.+Bourne&title=Novel+computational+approaches+to+polypharmacology+as+a+means+to+define+responses+to+individual+drugs&doi=10.1146%2Fannurev-pharmtox-010611-134630"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010611-134630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010611-134630%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DNovel%2520computational%2520approaches%2520to%2520polypharmacology%2520as%2520a%2520means%2520to%2520define%2520responses%2520to%2520individual%2520drugs%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2012%26volume%3D52%26spage%3D361%26epage%3D379%26doi%3D10.1146%2Fannurev-pharmtox-010611-134630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A machine learning-based method to improve docking scoring functions and its application to drug repurposing</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">408</span><span class="NLM_x">–</span> <span class="NLM_lpage">419</span><span class="refDoi"> DOI: 10.1021/ci100369f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100369f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=408-419&author=S.+L.+Kinningsauthor=N.+Liuauthor=P.+J.+Tongeauthor=R.+M.+Jacksonauthor=L.+Xieauthor=P.+E.+Bourne&title=A+machine+learning-based+method+to+improve+docking+scoring+functions+and+its+application+to+drug+repurposing&doi=10.1021%2Fci100369f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A Machine learning-based method to improve docking scoring functions and its application to drug repurposing</span></div><div class="casAuthors">Kinnings, Sarah L.; Liu, Nina; Tonge, Peter J.; Jackson, Richard M.; Xie, Lei; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">408-419</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking scoring functions are notoriously weak predictors of binding affinity.  They typically assign a common set of wts. to the individual energy terms that contribute to the overall energy score; however, these wts. should be gene family dependent.  In addn., they incorrectly assume that individual interactions contribute toward the total binding affinity in an additive manner.  In reality, noncovalent interactions often depend on one another in a nonlinear manner.  In this paper, we show how the use of support vector machines (SVMs), trained by assocg. sets of individual energy terms retrieved from mol. docking with the known binding affinity of each compd. from high-throughput screening expts., can be used to improve the correlation between known binding affinities and those predicted by the docking program eHiTS.  We construct two prediction models: a regression model trained using IC50 values from BindingDB, and a classification model trained using active and decoy compds. from the Directory of Useful Decoys (DUD).  Moreover, to address the issue of overrepresentation of neg. data in high-throughput screening data sets, we have designed a multiple-planar SVM training procedure for the classification model.  The increased performance that both SVMs give when compared with the original eHiTS scoring function highlights the potential for using nonlinear methods when deriving overall energy scores from their individual components.  We apply the above methodol. to train a new scoring function for direct inhibitors of Mycobacterium tuberculosis (M.tb) InhA.  By combining ligand binding site comparison with the new scoring function, we propose that phosphodiesterase inhibitors can potentially be repurposed to target M.tb InhA.  Our methodol. may be applied to other gene families for which target structures and activity data are available, as demonstrated in the work presented here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohmsJPycuBRrVg90H21EOLACvtfcHk0libsIvwLsmbFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSitbs%253D&md5=4f008a6145fc0f6d2bf4e14030555dec</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fci100369f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100369f%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520machine%2520learning-based%2520method%2520to%2520improve%2520docking%2520scoring%2520functions%2520and%2520its%2520application%2520to%2520drug%2520repurposing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D408%26epage%3D419%26doi%3D10.1021%2Fci100369f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Chang, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palsson, B. O.</span><span> </span><span class="NLM_article-title">Antibacterial mechanisms identified through structural systems pharmacology</span> <span class="citation_source-journal">BMC Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">102</span><span class="refDoi"> DOI: 10.1186/1752-0509-7-102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1186%2F1752-0509-7-102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=24112686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltlGntLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2013&pages=102&author=R.+L.+Changauthor=L.+Xieauthor=P.+E.+Bourneauthor=B.+O.+Palsson&title=Antibacterial+mechanisms+identified+through+structural+systems+pharmacology&doi=10.1186%2F1752-0509-7-102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial mechanisms identified through structural systems pharmacology</span></div><div class="casAuthors">Chang, Roger L.; Xie, Lei; Bourne, Philip E.; Palsson, Bernhard O.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Systems Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">102/1-102/12</span>CODEN:
                <span class="NLM_cas:coden">BSBMCC</span>;
        ISSN:<span class="NLM_cas:issn">1752-0509</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">The growing discipline of structural systems pharmacol. is applied prospectively in this study to predict pharmacol. outcomes of antibacterial compds. in Escherichia coli K12.  This work builds upon previously established methods for structural prediction of ligand binding pockets on protein mols. and uses and expands upon the previously developed genome scale model of metab. integrated with protein structures (GEM-PRO) for E. coli, structurally accounting for protein complexes.  Carefully selected case studies display the potential for this structural systems pharmacol. framework in discovery and development of antibacterial compds.  The prediction framework for antibacterial activity of compds. was validated for a control set of well-studied compds., recapitulating exptl.-detd. protein binding interactions and deleterious growth phenotypes resulting from these interactions.  The antibacterial activity of fosfomycin, sulfathiazole, and trimethoprim were accurately predicted, and as a neg. control glucose has no predicted antibacterial activity.  Previously uncharacterized mechanisms of action were predicted for compds. with known antibacterial properties, including (1-hydroxyheptane-1,1-diyl)bis(phosphonic acid) and cholesteryl oleate.  Five candidate inhibitors were predicted for a desirable target protein without any known inhibitors, tryptophan synthase β subunit (TrpB).  In addn. to the predictions presented, this effort also included significant expansion of the previously developed GEM-PRO to account for physiol. assemblies of protein complex structures with activities included in the E. coli K12 metabolic network.  Thus, the structural systems pharmacol. framework presented in this study is effective in the prediction of mol. mechanisms of antibacterial compds.  The study provides a promising proof of principle for such an approach to antibacterial development and raises specific mol. and systemic hypotheses about antibacterials that are amenable to exptl. testing.  This framework, and perhaps also the specific predictions of antibacterials, is extensible to developing antibacterial treatments for pathogenic E. coli and other bacterial pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTSJ0x0vSPZLVg90H21EOLACvtfcHk0lhXR2WEdbh56g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltlGntLY%253D&md5=f00b0087ecf99cb55af0977dbebf5164</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1186%2F1752-0509-7-102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1752-0509-7-102%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DR.%2BL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DPalsson%26aufirst%3DB.%2BO.%26atitle%3DAntibacterial%2520mechanisms%2520identified%2520through%2520structural%2520systems%2520pharmacology%26jtitle%3DBMC%2520Syst.%2520Biol.%26date%3D2013%26volume%3D7%26spage%3D102%26doi%3D10.1186%2F1752-0509-7-102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span> </span><span class="NLM_article-title">Lead- and drug-like compounds: the rule-of-five revolution</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span><span class="refDoi"> DOI: 10.1016/j.ddtec.2004.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.ddtec.2004.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=24981612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=337-341&author=C.+A.+Lipinski&title=Lead-+and+drug-like+compounds%3A+the+rule-of-five+revolution&doi=10.1016%2Fj.ddtec.2004.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Lead- and drug-like compounds: the rule-of-five revolution</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-341</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Citations in CAS SciFinder to the rule-of-five (RO5) publication will exceed 1000 by year-end 2004.  Trends in the RO5 literature explosion that can be discerned are the further definitions of drug-like.  This topic is explored in terms of drug-like physicochem. features, drug-like structural features, a comparison of drug-like and non-drug-like in drug discovery and a discussion of how drug-like features relate to clin. success.  Physicochem. features of CNS drugs and features related to CNS blood-brain transporter affinity are briefly reviewed.  Recent literature on features of non-oral drugs is reviewed and how features of lead-like compds. differ from those of drug-like compds. is discussed.  Most recently, partly driven by NIH roadmap initiatives, considerations have arisen as to what tool-like means in the search for chem. tools to probe biol. space.  All these topics frame the scope of this short review/perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnjiyP3CY7lLVg90H21EOLACvtfcHk0lhXR2WEdbh56g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D&md5=952c29b47003b884116d790ae9e4721f</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DLead-%2520and%2520drug-like%2520compounds%253A%2520the%2520rule-of-five%2520revolution%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D337%26epage%3D341%26doi%3D10.1016%2Fj.ddtec.2004.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Applying thermodynamic profiling in lead finding and optimization</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">95</span><span class="NLM_x">–</span> <span class="NLM_lpage">110</span><span class="refDoi"> DOI: 10.1038/nrd4486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnrd4486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25614222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVemtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=95-110&author=G.+Klebe&title=Applying+thermodynamic+profiling+in+lead+finding+and+optimization&doi=10.1038%2Fnrd4486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Applying thermodynamic profiling in lead finding and optimization</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">95-110</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery involves the optimization of various physicochem. properties of a ligand, particularly its binding affinity for its target receptor (or receptors).  In recent years, there has been growing interest in using thermodn. profiling of ligand-receptor interactions in order to select and optimize those ligands that might be most likely to become drug candidates with desirable physicochem. properties.  The thermodn. of binding is influenced by multiple factors, including hydrogen bonding and hydrophobic interactions, desolvation, residual mobility, dynamics and the local water structure.  This article discusses key issues in understanding the effects of these factors and applying this knowledge in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqtHj7SsL6rrVg90H21EOLACvtfcHk0lhXR2WEdbh56g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVemtr4%253D&md5=c2e12760623de81335ce252cb804c602</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnrd4486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4486%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DApplying%2520thermodynamic%2520profiling%2520in%2520lead%2520finding%2520and%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D95%26epage%3D110%26doi%3D10.1038%2Fnrd4486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Constantine, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzler, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonnell, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todderud, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldfarb, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newitt, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiefer, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tortolani, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tokarski, J.</span><span> </span><span class="NLM_article-title">Multiple and single binding modes of fragment-like kinase inhibitors revealed by molecular modeling, residue type-selective protonation, and nuclear overhauser effects</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">6225</span><span class="NLM_x">–</span> <span class="NLM_lpage">6229</span><span class="refDoi"> DOI: 10.1021/jm800747w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800747w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=6225-6229&author=K.+L.+Constantineauthor=L.+Muellerauthor=W.+J.+Metzlerauthor=P.+A.+McDonnellauthor=G.+Todderudauthor=V.+Goldfarbauthor=Y.+Fanauthor=J.+A.+Newittauthor=S.+E.+Kieferauthor=M.+Gaoauthor=D.+Tortolaniauthor=W.+Vaccaroauthor=J.+Tokarski&title=Multiple+and+single+binding+modes+of+fragment-like+kinase+inhibitors+revealed+by+molecular+modeling%2C+residue+type-selective+protonation%2C+and+nuclear+overhauser+effects&doi=10.1021%2Fjm800747w"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm800747w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800747w%26sid%3Dliteratum%253Aachs%26aulast%3DConstantine%26aufirst%3DK.%2BL.%26aulast%3DMueller%26aufirst%3DL.%26aulast%3DMetzler%26aufirst%3DW.%2BJ.%26aulast%3DMcDonnell%26aufirst%3DP.%2BA.%26aulast%3DTodderud%26aufirst%3DG.%26aulast%3DGoldfarb%26aufirst%3DV.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DNewitt%26aufirst%3DJ.%2BA.%26aulast%3DKiefer%26aufirst%3DS.%2BE.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DTortolani%26aufirst%3DD.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DTokarski%26aufirst%3DJ.%26atitle%3DMultiple%2520and%2520single%2520binding%2520modes%2520of%2520fragment-like%2520kinase%2520inhibitors%2520revealed%2520by%2520molecular%2520modeling%252C%2520residue%2520type-selective%2520protonation%252C%2520and%2520nuclear%2520overhauser%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D6225%26epage%3D6229%26doi%3D10.1021%2Fjm800747w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eliseenkova, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubert, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohammadi, M.</span><span> </span><span class="NLM_article-title">A molecular brake in the kinase hinge region regulates the activity of receptor tyrosine kinases</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">717</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.molcel.2007.06.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=17803937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=717-730&author=H.+Chenauthor=J.+Maauthor=W.+Liauthor=A.+V.+Eliseenkovaauthor=C.+Xuauthor=T.+A.+Neubertauthor=W.+T.+Millerauthor=M.+Mohammadi&title=A+molecular+brake+in+the+kinase+hinge+region+regulates+the+activity+of+receptor+tyrosine+kinases&doi=10.1016%2Fj.molcel.2007.06.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">A molecular brake in the kinase hinge region regulates the activity of receptor tyrosine kinases</span></div><div class="casAuthors">Chen, Huaibin; Ma, Jinghong; Li, Wanqing; Eliseenkova, Anna V.; Xu, Chongfeng; Neubert, Thomas A.; Miller, W. Todd; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">717-730</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Activating mutations in the tyrosine kinase domain of receptor tyrosine kinases (RTKs) cause cancer and skeletal disorders.  Comparison of the crystal structures of unphosphorylated and phosphorylated wild-type FGFR2 kinase domains (FGFR2Ks) with those of seven unphosphorylated pathogenic mutants reveals an autoinhibitory "mol. brake" mediated by a triad of residues in the kinase hinge region of all FGFRs.  Structural anal. shows that many other RTKs, including PDGFRs, VEGFRs, KIT, CSF1R, FLT3, TEK, and TIE, are also subject to regulation by this brake.  Pathogenic mutations activate FGFRs and other RTKs by disengaging the brake either directly or indirectly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobYTkvrN7lYrVg90H21EOLACvtfcHk0lhN4_W-wfvX0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I&md5=dba8a6b97b9d1826b2223146be137963</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.028%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DEliseenkova%26aufirst%3DA.%2BV.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DMiller%26aufirst%3DW.%2BT.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DA%2520molecular%2520brake%2520in%2520the%2520kinase%2520hinge%2520region%2520regulates%2520the%2520activity%2520of%2520receptor%2520tyrosine%2520kinases%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D717%26epage%3D730%26doi%3D10.1016%2Fj.molcel.2007.06.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawyers, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varmus, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">Inhibition of drug-resistant mutants of ABL, KIT, and EGF receptor kinases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">11011</span><span class="NLM_x">–</span> <span class="NLM_lpage">11016</span><span class="refDoi"> DOI: 10.1073/pnas.0504952102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1073%2Fpnas.0504952102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=16046538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnvVWjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=11011-11016&author=T.+A.+Carterauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=A.+M.+Velascoauthor=M.+A.+Fabianauthor=D.+K.+Treiberauthor=Z.+V.+Milanovauthor=C.+E.+Atteridgeauthor=W.+H.+Biggsauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=S.+A.+Mehtaauthor=H.+K.+Patelauthor=W.+Paoauthor=C.+L.+Sawyersauthor=H.+Varmusauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=Inhibition+of+drug-resistant+mutants+of+ABL%2C+KIT%2C+and+EGF+receptor+kinases&doi=10.1073%2Fpnas.0504952102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of drug-resistant mutants of ABL, KIT, and EGF receptor kinases</span></div><div class="casAuthors">Carter, Todd A.; Wodicka, Lisa M.; Shah, Neil P.; Velasco, Anne Marie; Fabian, Miles A.; Treiber, Daniel K.; Milanov, Zdravko V.; Atteridge, Corey E.; Biggs, William H., III; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Mehta, Shamal A.; Patel, Hitesh K.; Pao, William; Sawyers, Charles L.; Varmus, Harold; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">11011-11016</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">To realize the full potential of targeted protein kinase inhibitors for the treatment of cancer, it is important to address the emergence of drug resistance in treated patients.  Mutant forms of BCR-ABL, KIT, and the EGF receptor (EGFR) have been found that confer resistance to the drugs imatinib, gefitinib, and erlotinib.  The mutations weaken or prevent drug binding, and interestingly, one of the most common sites of mutation in all three kinases is a highly conserved "gatekeeper" threonine residue near the kinase active site.  We have identified existing clin. compds. that bind and inhibit drug-resistant mutant variants of ABL, KIT, and EGFR.  We found that the Aurora kinase inhibitor VX-680 and the p38 inhibitor BIRB-796 inhibit the imatinib- and BMS-354825-resistant ABL(T315I) kinase.  The KIT/FLT3 inhibitor SU-11248 potently inhibits the imatinib-resistant KIT(V559D/T670I) kinase, consistent with the clin. efficacy of SU-11248 against imatinib-resistant gastrointestinal tumors, and the EGFR inhibitors EKB-569 and Cl-1033, but not GW-572016 and ZD-6474, potently inhibit the gefitinib- and erlotinib-resistant EGFR(L858R/T790M) kinase.  EKB-569 and Cl-1033 are already in clin. trials, and our results suggest that they should be considered for testing in the treatment of gefitinib/erlotinib-resistant non-small cell lung cancer.  The results highlight the strategy of screening existing clin. compds. against newly identified drug-resistant mutant variants to find compds. that may serve as starting points for the development of next-generation drugs, or that could be used directly to treat patients that have acquired resistance to first-generation targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdgFgcC03tybVg90H21EOLACvtfcHk0lhN4_W-wfvX0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnvVWjurg%253D&md5=448c95c51b0cf9b5035df4065cd8f62c</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0504952102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0504952102%26sid%3Dliteratum%253Aachs%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPao%26aufirst%3DW.%26aulast%3DSawyers%26aufirst%3DC.%2BL.%26aulast%3DVarmus%26aufirst%3DH.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DInhibition%2520of%2520drug-resistant%2520mutants%2520of%2520ABL%252C%2520KIT%252C%2520and%2520EGF%2520receptor%2520kinases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D11011%26epage%3D11016%26doi%3D10.1073%2Fpnas.0504952102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Foda, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeliger, M. A.</span><span> </span><span class="NLM_article-title">Kinase inhibitors: an allosteric add-on</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">796</span><span class="NLM_x">–</span> <span class="NLM_lpage">797</span><span class="refDoi"> DOI: 10.1038/nchembio.1630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnchembio.1630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25195010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKms7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=796-797&author=Z.+H.+Fodaauthor=M.+A.+Seeliger&title=Kinase+inhibitors%3A+an+allosteric+add-on&doi=10.1038%2Fnchembio.1630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors - An allosteric add-on</span></div><div class="casAuthors">Foda, Zachariah H.; Seeliger, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">796-797</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Development of highly specific kinase inhibitors has been a long-standing challenge in chem. biol.  The structural and mechanistic characterization of an Erk1/2 kinase inhibitor provides new strategies to develop specific kinase inhibitors by targeting a binding pocket adjacent to the ATP binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoPgAa-_d1sLVg90H21EOLACvtfcHk0lhCjLCYh9lnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKms7vM&md5=40ce7d0360a5c59d73c8dd5cda0af412</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1630%26sid%3Dliteratum%253Aachs%26aulast%3DFoda%26aufirst%3DZ.%2BH.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26atitle%3DKinase%2520inhibitors%253A%2520an%2520allosteric%2520add-on%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D796%26epage%3D797%26doi%3D10.1038%2Fnchembio.1630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausen, M. H.</span><span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">422</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0lhCjLCYh9lnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Cadranel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruppert, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beau-Faller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wislez, M.</span><span> </span><span class="NLM_article-title">Therapeutic strategy for advanced EGFR mutant non-small-cell lung carcinoma</span> <span class="citation_source-journal">Crit. Rev. Oncol. Hematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">477</span><span class="NLM_x">–</span> <span class="NLM_lpage">493</span><span class="refDoi"> DOI: 10.1016/j.critrevonc.2013.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.critrevonc.2013.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=23911281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BC3sfmt12ksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2013&pages=477-493&author=J.+Cadranelauthor=A.+M.+Ruppertauthor=M.+Beau-Fallerauthor=M.+Wislez&title=Therapeutic+strategy+for+advanced+EGFR+mutant+non-small-cell+lung+carcinoma&doi=10.1016%2Fj.critrevonc.2013.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic strategy for advanced EGFR mutant non-small-cell lung carcinoma</span></div><div class="casAuthors">Cadranel Jacques; Ruppert Anne-Marie; Beau-Faller Michele; Wislez Marie</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncology/hematology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">477-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Activating mutation in exons 19 or 21 of epidermal growth factor receptor (EGFR) in non-small-cell lung cancers (NSCLC) are associated with increased sensitivity to EGFR tyrosine kinase inhibitors (EGFR-TKIs), such as gefitinib and erlotinib.  Cancer patients harboring activating EGFR mutations benefit from first-line TKI therapy.  Yet 10% of patients present a primary TKI resistance, while 50% of the others develop a secondary resistance within 9-12 months after starting TKI.  The RECIST's definition of progression appears flawed when applied to EGFR-mutated NSCLC patients.  Most often, tumor volume shrinking widely exceeds 30% during TKI response and kinetics of growth is low during relapse.  At present, secondary resistance mechanisms associated with progression are better known: clonal selection of EGFR resistance mutation (T790M mutation in exon 20), amplification of transmembrane receptors for other growth factors (c-met, HER family, IGF1R, or AXL), downstream molecular alterations in EGFR signaling pathway (PI3K or PTEN), and epithelial-mesenchymal transition or transdifferentiation to small-cell cancer.  The best strategy for secondary resistance is not well-defined: maintaining TKI therapy, switching to chemotherapy, combining both treatments, or using new therapies targeting other signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT9fVHNxWBIVa91pGlMMoT5fW6udTcc2eb3s8pfETv13bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfmt12ksQ%253D%253D&md5=202252954321dde8fa5aa196492a10e7</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.critrevonc.2013.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.critrevonc.2013.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DCadranel%26aufirst%3DJ.%26aulast%3DRuppert%26aufirst%3DA.%2BM.%26aulast%3DBeau-Faller%26aufirst%3DM.%26aulast%3DWislez%26aufirst%3DM.%26atitle%3DTherapeutic%2520strategy%2520for%2520advanced%2520EGFR%2520mutant%2520non-small-cell%2520lung%2520carcinoma%26jtitle%3DCrit.%2520Rev.%2520Oncol.%2520Hematol.%26date%3D2013%26volume%3D88%26spage%3D477%26epage%3D493%26doi%3D10.1016%2Fj.critrevonc.2013.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Engel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richters, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomassi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keul, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Termathe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lategahn, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayer-Wrangowski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grutter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uhlenbrock, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krull, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaumann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eppmann, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kibies, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffgaard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menninger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz-Cuaran, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heuckmann, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tinnefeld, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahedi, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sos, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz-Fademrecht, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kast, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span> </span><span class="NLM_article-title">Targeting drug resistance in EGFR with covalent inhibitors: a structure-based design approach</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">6844</span><span class="NLM_x">–</span> <span class="NLM_lpage">6863</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01082</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01082" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCjsrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6844-6863&author=J.+Engelauthor=A.+Richtersauthor=M.+Getlikauthor=S.+Tomassiauthor=M.+Keulauthor=M.+Termatheauthor=J.+Lategahnauthor=C.+Beckerauthor=S.+Mayer-Wrangowskiauthor=C.+Grutterauthor=N.+Uhlenbrockauthor=J.+Krullauthor=N.+Schaumannauthor=S.+Eppmannauthor=P.+Kibiesauthor=F.+Hoffgaardauthor=J.+Heilauthor=S.+Menningerauthor=S.+Ortiz-Cuaranauthor=J.+M.+Heuckmannauthor=V.+Tinnefeldauthor=R.+P.+Zahediauthor=M.+L.+Sosauthor=C.+Schultz-Fademrechtauthor=R.+K.+Thomasauthor=S.+M.+Kastauthor=D.+Rauh&title=Targeting+drug+resistance+in+EGFR+with+covalent+inhibitors%3A+a+structure-based+design+approach&doi=10.1021%2Facs.jmedchem.5b01082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Drug Resistance in EGFR with Covalent Inhibitors: A Structure-Based Design Approach</span></div><div class="casAuthors">Engel, Julian; Richters, Andre; Getlik, Matthaeus; Tomassi, Stefano; Keul, Marina; Termathe, Martin; Lategahn, Jonas; Becker, Christian; Mayer-Wrangowski, Svenja; Gruetter, Christian; Uhlenbrock, Niklas; Kruell, Jasmin; Schaumann, Niklas; Eppmann, Simone; Kibies, Patrick; Hoffgaard, Franziska; Heil, Jochen; Menninger, Sascha; Ortiz-Cuaran, Sandra; Heuckmann, Johannes M.; Tinnefeld, Verena; Zahedi, Rene P.; Sos, Martin L.; Schultz-Fademrecht, Carsten; Thomas, Roman K.; Kast, Stefan M.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6844-6863</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Receptor tyrosine kinases represent one of the prime targets in cancer therapy, as the dysregulation of these elementary transducers of extracellular signals, like the epidermal growth factor receptor (EGFR), contributes to the onset of cancer, such as non-small cell lung cancer (NSCLC).  Strong efforts were directed to the development of irreversible inhibitors and led to compd. CO-1686, which takes advantage of increased residence time at EGFR by alkylating Cys797 and thereby preventing toxic effects.  Here, the authors present a structure-based approach, rationalized by subsequent computational anal. of conformational ligand ensembles in soln., to design novel and irreversible EGFR inhibitors based on a screening hit that was identified in a phenotype screen of 80 NSCLC cell lines against approx. 1500 compds.  Using protein x-ray crystallog., the authors deciphered the binding mode in engineered cSrc (T338M/S345C), a validated model system for EGFR-T790M, which constituted the basis for further rational design approaches.  Chem. synthesis led to further compd. collections that revealed increased biochem. potency and, in part, selectivity toward mutated (L858R and L858R/T790M) vs. nonmutated EGFR.  Further cell-based and kinetic studies were performed to substantiate the authors initial findings.  Utilizing proteolytic digestion and nano-LC-MS/MS anal., the authors confirmed the alkylation of Cys797.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNGJPIuqoVObVg90H21EOLACvtfcHk0ljEU4oxb_-m1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCjsrvM&md5=d5264a2a99d7752af0e587f23aeb7723</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01082%26sid%3Dliteratum%253Aachs%26aulast%3DEngel%26aufirst%3DJ.%26aulast%3DRichters%26aufirst%3DA.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DTomassi%26aufirst%3DS.%26aulast%3DKeul%26aufirst%3DM.%26aulast%3DTermathe%26aufirst%3DM.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DBecker%26aufirst%3DC.%26aulast%3DMayer-Wrangowski%26aufirst%3DS.%26aulast%3DGrutter%26aufirst%3DC.%26aulast%3DUhlenbrock%26aufirst%3DN.%26aulast%3DKrull%26aufirst%3DJ.%26aulast%3DSchaumann%26aufirst%3DN.%26aulast%3DEppmann%26aufirst%3DS.%26aulast%3DKibies%26aufirst%3DP.%26aulast%3DHoffgaard%26aufirst%3DF.%26aulast%3DHeil%26aufirst%3DJ.%26aulast%3DMenninger%26aufirst%3DS.%26aulast%3DOrtiz-Cuaran%26aufirst%3DS.%26aulast%3DHeuckmann%26aufirst%3DJ.%2BM.%26aulast%3DTinnefeld%26aufirst%3DV.%26aulast%3DZahedi%26aufirst%3DR.%2BP.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DThomas%26aufirst%3DR.%2BK.%26aulast%3DKast%26aufirst%3DS.%2BM.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DTargeting%2520drug%2520resistance%2520in%2520EGFR%2520with%2520covalent%2520inhibitors%253A%2520a%2520structure-based%2520design%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6844%26epage%3D6863%26doi%3D10.1021%2Facs.jmedchem.5b01082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Siegelin, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borczuk, A. C.</span><span> </span><span class="NLM_article-title">Epidermal growth factor receptor mutations in lung adenocarcinoma</span> <span class="citation_source-journal">Lab. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span><span class="refDoi"> DOI: 10.1038/labinvest.2013.147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Flabinvest.2013.147" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2014&pages=129-137&author=M.+D.+Siegelinauthor=A.+C.+Borczuk&title=Epidermal+growth+factor+receptor+mutations+in+lung+adenocarcinoma&doi=10.1038%2Flabinvest.2013.147"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Flabinvest.2013.147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Flabinvest.2013.147%26sid%3Dliteratum%253Aachs%26aulast%3DSiegelin%26aufirst%3DM.%2BD.%26aulast%3DBorczuk%26aufirst%3DA.%2BC.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520mutations%2520in%2520lung%2520adenocarcinoma%26jtitle%3DLab.%2520Invest.%26date%3D2014%26volume%3D94%26spage%3D129%26epage%3D137%26doi%3D10.1038%2Flabinvest.2013.147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Greig, S. L.</span><span> </span><span class="NLM_article-title">Osimertinib: First Global Approval</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">273</span><span class="refDoi"> DOI: 10.1007/s40265-015-0533-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1007%2Fs40265-015-0533-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=26729184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC28XlvFymtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=263-273&author=S.+L.+Greig&title=Osimertinib%3A+First+Global+Approval&doi=10.1007%2Fs40265-015-0533-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Osimertinib: First Global Approval</span></div><div class="casAuthors">Greig, Sarah L.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-273</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">Osimertinib (Tagrisso, AZD9291) is an oral, third-generation epidermal growth factor receptor tyrosine kinase inhibitor (EGFR TKI) that is being developed by AstraZeneca for the treatment of advanced non-small cell lung cancer (NSCLC).  Osimertinib has been designed to target the EGFR T790M mutation that is often present in NSCLC patients with acquired EGFR TKI resistance, while sparing wild-type EGFR.  In Nov. 2015, the tablet formulation of osimertinib was granted accelerated approval in the USA for the treatment of patients with metastatic EGFR T790M mutation-pos. NSCLC (as detected by an FDA-approved test) who have progressed on or after EGFR TKI therapy.  Osimertinib has also been granted accelerated assessment status for this indication in the EU, and is in phase III development for first- and second-line and adjuvant treatment of advanced EGFR mutation-pos. NSCLC in several countries.  Phase I trials in patients with advanced solid tumors are also being conducted.  This article summarizes the milestones in the development of osimertinib leading to this first approval for NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD-OV6l6JC5rVg90H21EOLACvtfcHk0ljEU4oxb_-m1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlvFymtA%253D%253D&md5=fb46898918bdfe76320c26177710fb11</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1007%2Fs40265-015-0533-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-015-0533-4%26sid%3Dliteratum%253Aachs%26aulast%3DGreig%26aufirst%3DS.%2BL.%26atitle%3DOsimertinib%253A%2520First%2520Global%2520Approval%26jtitle%3DDrugs%26date%3D2016%26volume%3D76%26spage%3D263%26epage%3D273%26doi%3D10.1007%2Fs40265-015-0533-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Walter, A. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjin, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haringsma, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubrovskiy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labenski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheets, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St Martin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karp, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kalken, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaturvedi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nacht, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westlin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaw-Tsai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raponi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Dyke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Etter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmons, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harding, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, A.</span><span> </span><span class="NLM_article-title">Discovery of a mutant-selective covalent inhibitor of EGFR that overcomes T790M-mediated resistance in NSCLC</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1404</span><span class="NLM_x">–</span> <span class="NLM_lpage">1415</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-13-0314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1158%2F2159-8290.CD-13-0314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=24065731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2rtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=1404-1415&author=A.+O.+Walterauthor=R.+T.+Sjinauthor=H.+J.+Haringsmaauthor=K.+Ohashiauthor=J.+Sunauthor=K.+Leeauthor=A.+Dubrovskiyauthor=M.+Labenskiauthor=Z.+Zhuauthor=Z.+Wangauthor=M.+Sheetsauthor=T.+St+Martinauthor=R.+Karpauthor=D.+van+Kalkenauthor=P.+Chaturvediauthor=D.+Niuauthor=M.+Nachtauthor=R.+C.+Petterauthor=W.+Westlinauthor=K.+Linauthor=S.+Jaw-Tsaiauthor=M.+Raponiauthor=T.+Van+Dykeauthor=J.+Etterauthor=Z.+Weaverauthor=W.+Paoauthor=J.+Singhauthor=A.+D.+Simmonsauthor=T.+C.+Hardingauthor=A.+Allen&title=Discovery+of+a+mutant-selective+covalent+inhibitor+of+EGFR+that+overcomes+T790M-mediated+resistance+in+NSCLC&doi=10.1158%2F2159-8290.CD-13-0314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Mutant-Selective Covalent Inhibitor of EGFR that Overcomes T790M-Mediated Resistance in NSCLC</span></div><div class="casAuthors">Walter, Annette O.; Sjin, Robert Tjin Tham; Haringsma, Henry J.; Ohashi, Kadoaki; Sun, Jing; Lee, Kwangho; Dubrovskiy, Aleksandr; Labenski, Matthew; Zhu, Zhendong; Wang, Zhigang; Sheets, Michael; St. Martin, Thia; Karp, Russell; van Kalken, Dan; Chaturvedi, Prasoon; Niu, Deqiang; Nacht, Mariana; Petter, Russell C.; Westlin, William; Lin, Kevin; Jaw-Tsai, Sarah; Raponi, Mitch; Van Dyke, Terry; Etter, Jeff; Weaver, Zoe; Pao, William; Singh, Juswinder; Simmons, Andrew D.; Harding, Thomas C.; Allen, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1404-1415</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Patients with non-small cell lung cancer (NSCLC) with activating EGF receptor (EGFR) mutations initially respond to first-generation reversible EGFR tyrosine kinase inhibitors.  However, clin. efficacy is limited by acquired resistance, frequently driven by the EGFRT790M mutation.  CO-1686 is a novel, irreversible, and orally delivered kinase inhibitor that specifically targets the mutant forms of EGFR, including T790M, while exhibiting minimal activity toward the wild-type (WT) receptor.  Oral administration of CO-1686 as single agent induces tumor regression in EGFR-mutated NSCLC tumor xenograft and transgenic models.  Minimal activity of CO-1686 against the WT EGFR receptor was obsd.  In NSCLC cells with acquired resistance to CO-1686 in vitro, there was no evidence of addnl. mutations or amplification of the EGFR gene, but resistant cells exhibited signs of epithelial-mesenchymal transition and demonstrated increased sensitivity to AKT inhibitors.  These results suggest that CO-1686 may offer a novel therapeutic option for patients with mutant EGFR NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR0CDWQZ5wn7Vg90H21EOLACvtfcHk0ljEU4oxb_-m1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2rtLvN&md5=3d111f4bf958f2fda6890fbc0e1d99f5</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-13-0314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-13-0314%26sid%3Dliteratum%253Aachs%26aulast%3DWalter%26aufirst%3DA.%2BO.%26aulast%3DSjin%26aufirst%3DR.%2BT.%26aulast%3DHaringsma%26aufirst%3DH.%2BJ.%26aulast%3DOhashi%26aufirst%3DK.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DDubrovskiy%26aufirst%3DA.%26aulast%3DLabenski%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DSheets%26aufirst%3DM.%26aulast%3DSt%2BMartin%26aufirst%3DT.%26aulast%3DKarp%26aufirst%3DR.%26aulast%3Dvan%2BKalken%26aufirst%3DD.%26aulast%3DChaturvedi%26aufirst%3DP.%26aulast%3DNiu%26aufirst%3DD.%26aulast%3DNacht%26aufirst%3DM.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DWestlin%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DJaw-Tsai%26aufirst%3DS.%26aulast%3DRaponi%26aufirst%3DM.%26aulast%3DVan%2BDyke%26aufirst%3DT.%26aulast%3DEtter%26aufirst%3DJ.%26aulast%3DWeaver%26aufirst%3DZ.%26aulast%3DPao%26aufirst%3DW.%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DSimmons%26aufirst%3DA.%2BD.%26aulast%3DHarding%26aufirst%3DT.%2BC.%26aulast%3DAllen%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520a%2520mutant-selective%2520covalent%2520inhibitor%2520of%2520EGFR%2520that%2520overcomes%2520T790M-mediated%2520resistance%2520in%2520NSCLC%26jtitle%3DCancer%2520Discovery%26date%3D2013%26volume%3D3%26spage%3D1404%26epage%3D1415%26doi%3D10.1158%2F2159-8290.CD-13-0314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelletti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortot, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirieac, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacob, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padera, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engen, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span> </span><span class="NLM_article-title">Novel mutant-selective EGFR kinase inhibitors against EGFR T790M</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">1070</span><span class="NLM_x">–</span> <span class="NLM_lpage">1074</span><span class="refDoi"> DOI: 10.1038/nature08622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnature08622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=20033049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SktbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=1070-1074&author=W.+Zhouauthor=D.+Ercanauthor=L.+Chenauthor=C.+H.+Yunauthor=D.+Liauthor=M.+Capellettiauthor=A.+B.+Cortotauthor=L.+Chirieacauthor=R.+E.+Iacobauthor=R.+Paderaauthor=J.+R.+Engenauthor=K.+K.+Wongauthor=M.+J.+Eckauthor=N.+S.+Grayauthor=P.+A.+Janne&title=Novel+mutant-selective+EGFR+kinase+inhibitors+against+EGFR+T790M&doi=10.1038%2Fnature08622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Novel mutant-selective EGFR kinase inhibitors against EGFR T790M</span></div><div class="casAuthors">Zhou, Wenjun; Ercan, Dalia; Chen, Liang; Yun, Cai-Hong; Li, Danan; Capelletti, Marzia; Cortot, Alexis B.; Chirieac, Lucian; Iacob, Roxana E.; Padera, Robert; Engen, John R.; Wong, Kwok-Kin; Eck, Michael J.; Gray, Nathanael S.; Janne, Pasi A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7276</span>),
    <span class="NLM_cas:pages">1070-1074</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy of epidermal growth factor receptor (EGFR) kinase inhibitors in EGFR-mutant non-small-cell lung cancer (NSCLC) is limited by the development of drug-resistance mutations, including the gatekeeper T790M mutation.  Strategies targeting EGFR T790M with irreversible inhibitors have had limited success and are assocd. with toxicity due to concurrent inhibition of wild-type EGFR.  All current EGFR inhibitors possess a structurally related quinazoline-based core scaffold and were identified as ATP-competitive inhibitors of wild-type EGFR.  Here we identify a covalent pyrimidine EGFR inhibitor by screening an irreversible kinase inhibitor library specifically against EGFR T790M.  These agents are 30- to 100-fold more potent against EGFR T790M, and up to 100-fold less potent against wild-type EGFR, than quinazoline-based EGFR inhibitors in vitro.  They are also effective in murine models of lung cancer driven by EGFR T790M.  Co-crystn. studies reveal a structural basis for the increased potency and mutant selectivity of these agents.  These mutant-selective irreversible EGFR kinase inhibitors may be clin. more effective and better tolerated than quinazoline-based inhibitors.  Our findings demonstrate that functional pharmacol. screens against clin. important mutant kinases represent a powerful strategy to identify new classes of mutant-selective kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcksZlIe-1bbVg90H21EOLACvtfcHk0ljDPP5EEL4vZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1SktbvI&md5=9b5d41a2be401c67a4b9f04a41cf1725</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fnature08622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08622%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DCortot%26aufirst%3DA.%2BB.%26aulast%3DChirieac%26aufirst%3DL.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DPadera%26aufirst%3DR.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26atitle%3DNovel%2520mutant-selective%2520EGFR%2520kinase%2520inhibitors%2520against%2520EGFR%2520T790M%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D1070%26epage%3D1074%26doi%3D10.1038%2Fnature08622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Adams, J. A.</span><span> </span><span class="NLM_article-title">Kinetic and catalytic mechanisms of protein kinases</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">2271</span><span class="NLM_x">–</span> <span class="NLM_lpage">2290</span><span class="refDoi"> DOI: 10.1021/cr000230w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000230w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvV2qtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2001&pages=2271-2290&author=J.+A.+Adams&title=Kinetic+and+catalytic+mechanisms+of+protein+kinases&doi=10.1021%2Fcr000230w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and catalytic mechanisms of protein kinases</span></div><div class="casAuthors">Adams, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2271-2290</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 178 refs.  The vertebrate genome is estd. to contain as many as 2000 protein kinases to carry out a variety of biol. functions.  Although the phosphoryl transfer mechanism is chem. simple, the phosphorylation of protein substrates in the active site of a protein kinase is complex and involves structural changes.  A conserved catalytic core region of approx. 200-250 amino acids is shared within the protein kinase family, but this core structure frequently interacts with regulatory proteins or domains that either enhance or repress catalytic function.  This structural hierarchy offers an extraordinary level of complexity to the enzyme family and makes functional studies of catalysis and regulation intricate.  While the noncatalytic domains and subunits affect kinase function, in the interest of brevity the review focuses largely on the core structure.  Although data are reviewed from many different enzymes within the protein kinase family, emphasis is placed on cAMP-dependent protein kinase A (PKA), which is by far the best understood protein kinase from a mechanistic viewpoint.  PKA consists of a heterotetramer composed of 2 catalytic (C) subunits and a regulatory dimer.  Owing to the highly conserved nature of the catalytic core structure, a description of PKA encompasses the main features of all members of the protein kinase family.  The C subunit of PKA is comprised of 2 domains, including a small ATP-binding domain composed primarily of α-helixes and a larger substrate-binding domain composed mostly of β-strands.  The 2 domains are connected by a small linker region to generate the binding pocket for ATP and the substrate.  Whereas the ATP is nestled deep between the domains, the 20-residue inhibitory peptide (PKI) resides on the periphery of the pocket, and the activation peptide with its essential PKA phosphorylation site, Thr-197, is also nearby.  The review focuses on approaches taken to assess the catalytic and kinetic mechanisms of protein phosphorylation catalyzed by protein kinases, and concs. its attention on the roles of active site residues, metal cations, conformational changes, and regulatory phosphorylation in the processing of peptide and protein substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooLsgkSVlhcLVg90H21EOLACvtfcHk0ljDPP5EEL4vZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvV2qtLo%253D&md5=8e3a97ed93e74df275a166b003f1757a</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fcr000230w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000230w%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DJ.%2BA.%26atitle%3DKinetic%2520and%2520catalytic%2520mechanisms%2520of%2520protein%2520kinases%26jtitle%3DChem.%2520Rev.%26date%3D2001%26volume%3D101%26spage%3D2271%26epage%3D2290%26doi%3D10.1021%2Fcr000230w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nolop, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, P.</span><span> </span><span class="NLM_article-title">Vemurafenib: the first drug approved for BRAF-mutant cancer</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">886</span><span class="refDoi"> DOI: 10.1038/nrd3847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnrd3847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=23060265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=873-886&author=G.+Bollagauthor=J.+Tsaiauthor=J.+Zhangauthor=C.+Zhangauthor=P.+Ibrahimauthor=K.+Nolopauthor=P.+Hirth&title=Vemurafenib%3A+the+first+drug+approved+for+BRAF-mutant+cancer&doi=10.1038%2Fnrd3847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Vemurafenib: the first drug approved for BRAF-mutant cancer</span></div><div class="casAuthors">Bollag, Gideon; Tsai, James; Zhang, Jiazhong; Zhang, Chao; Ibrahim, Prabha; Nolop, Keith; Hirth, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">873-886</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The identification of driver oncogenes has provided important targets for drugs that can change the landscape of cancer therapies.  One such example is the BRAF oncogene, which is found in about half of all melanomas as well as several other cancers.  As a druggable kinase, oncogenic BRAF has become a crucial target of small-mol. drug discovery efforts.  Following a rapid clin. development path, vemurafenib (Zelboraf; Plexxikon/Roche) was approved for the treatment of BRAF-mutated metastatic melanoma in the United States in August 2011 and the European Union in Feb. 2012.  This Review describes the underlying biol. of BRAF, the technol. used to identify vemurafenib and its clin. development milestones, along with future prospects based on lessons learned during its development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT6Fbb43oRTbVg90H21EOLACvtfcHk0ljDPP5EEL4vZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL&md5=b1b38cad542be9b3a30060b77c584bc1</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1038%2Fnrd3847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3847%26sid%3Dliteratum%253Aachs%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DNolop%26aufirst%3DK.%26aulast%3DHirth%26aufirst%3DP.%26atitle%3DVemurafenib%253A%2520the%2520first%2520drug%2520approved%2520for%2520BRAF-mutant%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D873%26epage%3D886%26doi%3D10.1038%2Fnrd3847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Shaw, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felip, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, L. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camidge, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vansteenkiste, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Pas, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riely, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santoro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldwasser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boral, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, J. A.</span><span> </span><span class="NLM_article-title">Ceritinib in ALK-rearranged non-small-cell lung cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">370</span><span class="NLM_x">, </span> <span class="NLM_fpage">1189</span><span class="NLM_x">–</span> <span class="NLM_lpage">1197</span><span class="refDoi"> DOI: 10.1056/NEJMoa1311107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1056%2FNEJMoa1311107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=24670165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvVCitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2014&pages=1189-1197&author=A.+T.+Shawauthor=D.+W.+Kimauthor=R.+Mehraauthor=D.+S.+Tanauthor=E.+Felipauthor=L.+Q.+Chowauthor=D.+R.+Camidgeauthor=J.+Vansteenkisteauthor=S.+Sharmaauthor=T.+De+Pasauthor=G.+J.+Rielyauthor=B.+J.+Solomonauthor=J.+Wolfauthor=M.+Thomasauthor=M.+Schulerauthor=G.+Liuauthor=A.+Santoroauthor=Y.+Y.+Lauauthor=M.+Goldwasserauthor=A.+L.+Boralauthor=J.+A.+Engelman&title=Ceritinib+in+ALK-rearranged+non-small-cell+lung+cancer&doi=10.1056%2FNEJMoa1311107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Ceritinib in ALK-rearranged non-small-cell lung cancer</span></div><div class="casAuthors">Shaw, Alice T.; Kim, Dong-Wan; Mehra, Ranee; Tan, Daniel S. W.; Felip, Enriqueta; Chow, Laura Q. M.; Camidge, Ross; Vansteenkiste, Johan; Sharma, Sunil; De Pas, Tommaso; Riely, Gregory J.; Solomon, Benjamin J.; Wolf, Juergen; Thomas, Michael; Schuler, Martin; Liu, Geoffrey; Santoro, Armando; Lau, Yvonne Y.; Goldwasser, Meredith; Boral, Anthony L.; Engelman, Jeffrey A.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">370</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1189-1197</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background: Non-small-cell lung cancer (NSCLC) harboring the anaplastic lymphoma kinase gene (ALK) rearrangement is sensitive to the ALK inhibitor crizotinib, but resistance invariably develops.  Ceritinib (LDK378) is a new ALK inhibitor that has shown greater antitumor potency than crizotinib in preclin. studies.  Methods: In this phase 1 study, we administered oral ceritinib in doses of 50 to 750 mg once daily to patients with advanced cancers harboring genetic alterations in ALK.  In an expansion phase of the study, patients received the max. tolerated dose.  Patients were assessed to det. the safety, pharmacokinetic properties, and antitumor activity of ceritinib.  Tumor biopsies were performed before ceritinib treatment to identify resistance mutations in ALK in a group of patients with NSCLC who had had disease progression during treatment with crizotinib.  Results: A total of 59 patients were enrolled in the dose-escalation phase.  The max. tolerated dose of ceritinib was 750 mg once daily; dose-limiting toxic events included diarrhea, vomiting, dehydration, elevated aminotransferase levels, and hypophosphatemia.  This phase was followed by an expansion phase, in which an addnl. 71 patients were treated, for a total of 130 patients overall.  Among 114 patients with NSCLC who received at least 400 mg of ceritinib per day, the overall response rate was 58% (95% confidence interval [CI], 48 to 67).  Among 80 patients who had received crizotinib previously, the response rate was 56% (95% CI, 45 to 67).  Responses were obsd. in patients with various resistance mutations in ALK and in patients without detectable mutations.  Among patients with NSCLC who received at least 400 mg of ceritinib per day, the median progression-free survival was 7.0 mo (95% CI, 5.6 to 9.5).  Conclusions: Ceritinib was highly active in patients with advanced, ALK-rearranged NSCLC, including those who had had disease progression during crizotinib treatment, regardless of the presence of resistance mutations in ALK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxuct3wv2aF7Vg90H21EOLACvtfcHk0lg0BcuGCFpwbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvVCitL0%253D&md5=dff3054469948ceb26a00e3eccac56c1</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1311107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1311107%26sid%3Dliteratum%253Aachs%26aulast%3DShaw%26aufirst%3DA.%2BT.%26aulast%3DKim%26aufirst%3DD.%2BW.%26aulast%3DMehra%26aufirst%3DR.%26aulast%3DTan%26aufirst%3DD.%2BS.%26aulast%3DFelip%26aufirst%3DE.%26aulast%3DChow%26aufirst%3DL.%2BQ.%26aulast%3DCamidge%26aufirst%3DD.%2BR.%26aulast%3DVansteenkiste%26aufirst%3DJ.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DDe%2BPas%26aufirst%3DT.%26aulast%3DRiely%26aufirst%3DG.%2BJ.%26aulast%3DSolomon%26aufirst%3DB.%2BJ.%26aulast%3DWolf%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DSchuler%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DSantoro%26aufirst%3DA.%26aulast%3DLau%26aufirst%3DY.%2BY.%26aulast%3DGoldwasser%26aufirst%3DM.%26aulast%3DBoral%26aufirst%3DA.%2BL.%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26atitle%3DCeritinib%2520in%2520ALK-rearranged%2520non-small-cell%2520lung%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2014%26volume%3D370%26spage%3D1189%26epage%3D1197%26doi%3D10.1056%2FNEJMoa1311107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Fabbro, D.</span><span> </span><span class="NLM_article-title">25 years of small molecular weight kinase inhibitors: potentials and limitations</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">766</span><span class="NLM_x">–</span> <span class="NLM_lpage">775</span><span class="refDoi"> DOI: 10.1124/mol.114.095489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1124%2Fmol.114.095489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25549667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVKltLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2015&pages=766-775&author=D.+Fabbro&title=25+years+of+small+molecular+weight+kinase+inhibitors%3A+potentials+and+limitations&doi=10.1124%2Fmol.114.095489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">25 years of small molecular weight kinase inhibitors: potentials and limitations</span></div><div class="casAuthors">Fabbro, Doriano</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">766-775</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Deregulation of protein and lipid kinase activities leads to a variety of pathologies, ranging from cancer inflammatory diseases, diabetes, infectious diseases, and cardiovascular disorders.  Protein kinases and lipid kinases represent, therefore, an important target for the pharmaceutical industry.  In fact, approx. one-third of all protein targets under investigation in the pharmaceutical industry are protein or lipid kinases.  To date, 30 kinase inhibitors have been approved, which, with few exceptions, are mainly for oncol. indications and directed against only a handful of protein and lipid kinases, leaving 70% of the kinome untapped.  Despite these successes in kinase drug discovery, the development of kinase inhibitors with outstanding selectivity, identification and validation of driver kinase(s) in diseases, and the emerging problem of resistance to the inhibition of key target kinases remain major challenges.  This mini review provides an insight into protein and lipid kinase drug discovery with respect to achievements, binding modes of inhibitors, and novel avenues for the generation of second-generation kinase inhibitors to treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZhF-9NQJx7bVg90H21EOLACvtfcHk0lg0BcuGCFpwbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVKltLk%253D&md5=6422b036e4e447379ac6c608662954c2</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.095489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.095489%26sid%3Dliteratum%253Aachs%26aulast%3DFabbro%26aufirst%3DD.%26atitle%3D25%2520years%2520of%2520small%2520molecular%2520weight%2520kinase%2520inhibitors%253A%2520potentials%2520and%2520limitations%26jtitle%3DMol.%2520Pharmacol.%26date%3D2015%26volume%3D87%26spage%3D766%26epage%3D775%26doi%3D10.1124%2Fmol.114.095489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Ohren, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pavlovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitehead, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuffa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McConnell, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spessard, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banotai, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delaney, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Omer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sebolt-Leopold, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dudley, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flamme, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warmus, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrett, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tecle, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasemann, C. A.</span><span> </span><span class="NLM_article-title">Structures of human MAP kinase kinase 1 (MEK1) and MEK2 describe novel noncompetitive kinase inhibition</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1192</span><span class="NLM_x">–</span> <span class="NLM_lpage">1197</span><span class="refDoi"> DOI: 10.1038/nsmb859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnsmb859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=15543157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1192-1197&author=J.+F.+Ohrenauthor=H.+Chenauthor=A.+Pavlovskyauthor=C.+Whiteheadauthor=E.+Zhangauthor=P.+Kuffaauthor=C.+Yanauthor=P.+McConnellauthor=C.+Spessardauthor=C.+Banotaiauthor=W.+T.+Muellerauthor=A.+Delaneyauthor=C.+Omerauthor=J.+Sebolt-Leopoldauthor=D.+T.+Dudleyauthor=I.+K.+Leungauthor=C.+Flammeauthor=J.+Warmusauthor=M.+Kaufmanauthor=S.+Barrettauthor=H.+Tecleauthor=C.+A.+Hasemann&title=Structures+of+human+MAP+kinase+kinase+1+%28MEK1%29+and+MEK2+describe+novel+noncompetitive+kinase+inhibition&doi=10.1038%2Fnsmb859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of human MAP kinase kinase 1 (MEK1) and MEK2 describe novel noncompetitive kinase inhibition</span></div><div class="casAuthors">Ohren, Jeffrey F.; Chen, Huifen; Pavlovsky, Alexander; Whitehead, Christopher; Zhang, Erli; Kuffa, Peter; Yan, Chunhong; McConnell, Patrick; Spessard, Cindy; Banotai, Craig; Mueller, W. Thomas; Delaney, Amy; Omer, Charles; Sebolt-Leopold, Judith; Dudley, David T.; Leung, Iris K.; Flamme, Cathlin; Warmus, Joseph; Kaufman, Michael; Barrett, Stephen; Tecle, Haile; Hasemann, Charles A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1192-1197</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MEK1 and MEK2 kinases are closely related, dual-specificity tyrosine/threonine protein kinases found in the Ras/Raf/MEK/ERK MAP kinase (MAPK) signaling pathway.  Approx. 30% of all human cancers have a constitutively activated MAPK pathway, and constitutive activation of MEK1 results in cellular transformation.  Here, the authors present the x-ray crystal structures of human MEK1 and MEK2 kinases, each detd. as a ternary complex with MgATP and an inhibitor to a resoln. of 2.4 and 3.2 Å, resp.  The structures revealed that MEK1 and MEK2 each had a unique inhibitor-binding pocket adjacent to the MgATP-binding site.  The presence of the potent inhibitors induced several conformational changes in the unphosphorylated MEK1 and MEK2 kinases that locked them into a closed but catalytically inactive species.  Thus, the structures reported here reveal a novel, noncompetitive mechanism for protein kinase inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIY9begVVfW7Vg90H21EOLACvtfcHk0lg0BcuGCFpwbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM&md5=79ebbc2e3a9cdd3fbecb8bb35ea9dba3</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1038%2Fnsmb859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb859%26sid%3Dliteratum%253Aachs%26aulast%3DOhren%26aufirst%3DJ.%2BF.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DPavlovsky%26aufirst%3DA.%26aulast%3DWhitehead%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DE.%26aulast%3DKuffa%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DMcConnell%26aufirst%3DP.%26aulast%3DSpessard%26aufirst%3DC.%26aulast%3DBanotai%26aufirst%3DC.%26aulast%3DMueller%26aufirst%3DW.%2BT.%26aulast%3DDelaney%26aufirst%3DA.%26aulast%3DOmer%26aufirst%3DC.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DDudley%26aufirst%3DD.%2BT.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DFlamme%26aufirst%3DC.%26aulast%3DWarmus%26aufirst%3DJ.%26aulast%3DKaufman%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DS.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DHasemann%26aufirst%3DC.%2BA.%26atitle%3DStructures%2520of%2520human%2520MAP%2520kinase%2520kinase%25201%2520%2528MEK1%2529%2520and%2520MEK2%2520describe%2520novel%2520noncompetitive%2520kinase%2520inhibition%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1192%26epage%3D1197%26doi%3D10.1038%2Fnsmb859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Huang, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metcalf, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundaramoorthi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lentini, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keats, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wardwell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ning, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snodgrass, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broudy, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russian, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Commodore, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narasimhan, N. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohemmad, Q. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iuliucci, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivera, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalgarno, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawyer, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clackson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakespeare, W. C.</span><span> </span><span class="NLM_article-title">Discovery of 3-[2-(imidazo[1,2-b]pyridazin-3-yl)ethynyl]-4-methyl-N-{4-[(4-methylpiperazin-1-y l)methyl]-3-(trifluoromethyl)phenyl}benzamide (AP24534), a potent, orally active pan-inhibitor of breakpoint cluster region-abelson (BCR-ABL) kinase including the T315I gatekeeper mutant</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">4701</span><span class="NLM_x">–</span> <span class="NLM_lpage">4719</span><span class="refDoi"> DOI: 10.1021/jm100395q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100395q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslels7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4701-4719&author=W.+S.+Huangauthor=C.+A.+Metcalfauthor=R.+Sundaramoorthiauthor=Y.+Wangauthor=D.+Zouauthor=R.+M.+Thomasauthor=X.+Zhuauthor=L.+Caiauthor=D.+Wenauthor=S.+Liuauthor=J.+Romeroauthor=J.+Qiauthor=I.+Chenauthor=G.+Bandaauthor=S.+P.+Lentiniauthor=S.+Dasauthor=Q.+Xuauthor=J.+Keatsauthor=F.+Wangauthor=S.+Wardwellauthor=Y.+Ningauthor=J.+T.+Snodgrassauthor=M.+I.+Broudyauthor=K.+Russianauthor=T.+Zhouauthor=L.+Commodoreauthor=N.+I.+Narasimhanauthor=Q.+K.+Mohemmadauthor=J.+Iuliucciauthor=V.+M.+Riveraauthor=D.+C.+Dalgarnoauthor=T.+K.+Sawyerauthor=T.+Clacksonauthor=W.+C.+Shakespeare&title=Discovery+of+3-%5B2-%28imidazo%5B1%2C2-b%5Dpyridazin-3-yl%29ethynyl%5D-4-methyl-N-%7B4-%5B%284-methylpiperazin-1-y+l%29methyl%5D-3-%28trifluoromethyl%29phenyl%7Dbenzamide+%28AP24534%29%2C+a+potent%2C+orally+active+pan-inhibitor+of+breakpoint+cluster+region-abelson+%28BCR-ABL%29+kinase+including+the+T315I+gatekeeper+mutant&doi=10.1021%2Fjm100395q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-[2-(Imidazo[1,2-b]pyridazin-3-yl)ethynyl]-4-methyl-N-[4-((4-methylpiperazin-1-yl)methyl)-3-(trifluoromethyl)phenyl]benzamide (AP24534), a Potent, Orally Active Pan-Inhibitor of Breakpoint Cluster Region-Abelson (BCR-ABL) Kinase Including the T315I Gatekeeper Mutant</span></div><div class="casAuthors">Huang, Wei-Sheng; Metcalf, Chester A.; Sundaramoorthi, Raji; Wang, Yihan; Zou, Dong; Thomas, R. Mathew; Zhu, Xiaotian; Cai, Lisi; Wen, David; Liu, Shuangying; Romero, Jan; Qi, Jiwei; Chen, Ingrid; Banda, Geetha; Lentini, Scott P.; Das, Sasmita; Xu, Qihong; Keats, Jeff; Wang, Frank; Wardwell, Scott; Ning, Yaoyu; Snodgrass, Joseph T.; Broudy, Marc I.; Russian, Karin; Zhou, Tianjun; Commodore, Lois; Narasimhan, Narayana I.; Mohemmad, Qurish K.; Iuliucci, John; Rivera, Victor M.; Dalgarno, David C.; Sawyer, Tomi K.; Clackson, Tim; Shakespeare, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4701-4719</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the treatment of chronic myeloid leukemia (CML) with BCR-ABL kinase inhibitors, the T315I gatekeeper mutant has emerged as resistant to all currently approved agents.  This report describes the structure-guided design of a novel series of potent pan-inhibitors of BCR-ABL, including the T315I mutation.  A key structural feature is the carbon-carbon triple bond linker which skirts the increased bulk of Ile315 side chain.  Extensive SAR studies led to the discovery of development candidate benzamide I (AP24534), which inhibited the kinase activity of both native BCR-ABL and the T315I mutant with low nM IC50s, and potently inhibited proliferation of corresponding Ba/F3-derived cell lines.  Daily oral administration of I significantly prolonged survival of mice injected i.v. with BCR-ABLT315I expressing Ba/F3 cells.  These data, coupled with a favorable ADME profile, support the potential of I to be an effective treatment for CML, including patients refractory to all currently approved therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoci2zVFUazgLVg90H21EOLACvtfcHk0ljeVU9d4gjUUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslels7s%253D&md5=0dcd7b4ff31164b1d9883e897b7058ff</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fjm100395q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100395q%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%2BS.%26aulast%3DMetcalf%26aufirst%3DC.%2BA.%26aulast%3DSundaramoorthi%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DThomas%26aufirst%3DR.%2BM.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DL.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DRomero%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DBanda%26aufirst%3DG.%26aulast%3DLentini%26aufirst%3DS.%2BP.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DKeats%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWardwell%26aufirst%3DS.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DSnodgrass%26aufirst%3DJ.%2BT.%26aulast%3DBroudy%26aufirst%3DM.%2BI.%26aulast%3DRussian%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DNarasimhan%26aufirst%3DN.%2BI.%26aulast%3DMohemmad%26aufirst%3DQ.%2BK.%26aulast%3DIuliucci%26aufirst%3DJ.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26atitle%3DDiscovery%2520of%25203-%255B2-%2528imidazo%255B1%252C2-b%255Dpyridazin-3-yl%2529ethynyl%255D-4-methyl-N-%257B4-%255B%25284-methylpiperazin-1-y%2520l%2529methyl%255D-3-%2528trifluoromethyl%2529phenyl%257Dbenzamide%2520%2528AP24534%2529%252C%2520a%2520potent%252C%2520orally%2520active%2520pan-inhibitor%2520of%2520breakpoint%2520cluster%2520region-abelson%2520%2528BCR-ABL%2529%2520kinase%2520including%2520the%2520T315I%2520gatekeeper%2520mutant%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4701%26epage%3D4719%26doi%3D10.1021%2Fjm100395q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">O’Hare, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakespeare, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eide, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivera, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adrian, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metcalf, C. A.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyner, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loriaux, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corbin, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wardwell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ning, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keats, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundaramoorthi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snodgrass, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Commodore, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawyer, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalgarno, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deininger, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Druker, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clackson, T.</span><span> </span><span class="NLM_article-title">AP24534, a pan-BCR-ABL inhibitor for chronic myeloid leukemia, potently inhibits the T315I mutant and overcomes mutation-based resistance</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">412</span><span class="refDoi"> DOI: 10.1016/j.ccr.2009.09.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.ccr.2009.09.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19878872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFKltb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=401-412&author=T.+O%E2%80%99Hareauthor=W.+C.+Shakespeareauthor=X.+Zhuauthor=C.+A.+Eideauthor=V.+M.+Riveraauthor=F.+Wangauthor=L.+T.+Adrianauthor=T.+Zhouauthor=W.+S.+Huangauthor=Q.+Xuauthor=C.+A.+Metcalfauthor=J.+W.+Tynerauthor=M.+M.+Loriauxauthor=A.+S.+Corbinauthor=S.+Wardwellauthor=Y.+Ningauthor=J.+A.+Keatsauthor=Y.+Wangauthor=R.+Sundaramoorthiauthor=M.+Thomasauthor=D.+Zhouauthor=J.+Snodgrassauthor=L.+Commodoreauthor=T.+K.+Sawyerauthor=D.+C.+Dalgarnoauthor=M.+W.+Deiningerauthor=B.+J.+Drukerauthor=T.+Clackson&title=AP24534%2C+a+pan-BCR-ABL+inhibitor+for+chronic+myeloid+leukemia%2C+potently+inhibits+the+T315I+mutant+and+overcomes+mutation-based+resistance&doi=10.1016%2Fj.ccr.2009.09.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">AP24534, a pan-BCR-ABL inhibitor for chronic myeloid leukemia, potently inhibits the T315I mutant and overcomes mutation-based resistance</span></div><div class="casAuthors">O'Hare, Thomas; Shakespeare, William C.; Zhu, Xiaotian; Eide, Christopher A.; Rivera, Victor M.; Wang, Frank; Adrian, Lauren T.; Zhou, Tianjun; Huang, Wei-Sheng; Xu, Qihong; Metcalf, Chester A., III; Tyner, Jeffrey W.; Loriaux, Marc M.; Corbin, Amie S.; Wardwell, Scott; Ning, Yaoyu; Keats, Jeffrey A.; Wang, Yihan; Sundaramoorthi, Raji; Thomas, Mathew; Zhou, Dong; Snodgrass, Joseph; Commodore, Lois; Sawyer, Tomi K.; Dalgarno, David C.; Deininger, Michael W. N.; Druker, Brian J.; Clackson, Tim</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">401-412</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibition of BCR-ABL by imatinib induces durable responses in many patients with chronic myeloid leukemia (CML), but resistance attributable to kinase domain mutations can lead to relapse and a switch to second-line therapy with nilotinib or dasatinib.  Despite three approved therapeutic options, the cross-resistant BCR-ABLT315I mutation and compd. mutants selected on sequential inhibitor therapy remain major clin. challenges.  We report design and preclin. evaluation of AP24534, a potent, orally available multitargeted kinase inhibitor active against T315I and other BCR-ABL mutants.  AP24534 inhibited all tested BCR-ABL mutants in cellular and biochem. assays, suppressed BCR-ABLT315I-driven tumor growth in mice, and completely abrogated resistance in cell-based mutagenesis screens.  Our work supports clin. evaluation of AP24534 as a pan-BCR-ABL inhibitor for treatment of CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox3g7ZYNXwLbVg90H21EOLACvtfcHk0ljeVU9d4gjUUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFKltb3N&md5=62a383734fc345b97647ba71aa7aee04</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.09.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.09.028%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Hare%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DEide%26aufirst%3DC.%2BA.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DAdrian%26aufirst%3DL.%2BT.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DW.%2BS.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMetcalf%26aufirst%3DC.%2BA.%26aulast%3DTyner%26aufirst%3DJ.%2BW.%26aulast%3DLoriaux%26aufirst%3DM.%2BM.%26aulast%3DCorbin%26aufirst%3DA.%2BS.%26aulast%3DWardwell%26aufirst%3DS.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DKeats%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DSundaramoorthi%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DSnodgrass%26aufirst%3DJ.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%26aulast%3DDruker%26aufirst%3DB.%2BJ.%26aulast%3DClackson%26aufirst%3DT.%26atitle%3DAP24534%252C%2520a%2520pan-BCR-ABL%2520inhibitor%2520for%2520chronic%2520myeloid%2520leukemia%252C%2520potently%2520inhibits%2520the%2520T315I%2520mutant%2520and%2520overcomes%2520mutation-based%2520resistance%26jtitle%3DCancer%2520Cell%26date%3D2009%26volume%3D16%26spage%3D401%26epage%3D412%26doi%3D10.1016%2Fj.ccr.2009.09.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Solca, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahl, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zoephel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bader, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanderson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraemer, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Himmelsbach, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haaksma, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adolf, G. R.</span><span> </span><span class="NLM_article-title">Target binding properties and cellular activity of afatinib (BIBW 2992), an irreversible ErbB family blocker</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">343</span><span class="NLM_x">, </span> <span class="NLM_fpage">342</span><span class="NLM_x">–</span> <span class="NLM_lpage">350</span><span class="refDoi"> DOI: 10.1124/jpet.112.197756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1124%2Fjpet.112.197756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=22888144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2012&pages=342-350&author=F.+Solcaauthor=G.+Dahlauthor=A.+Zoephelauthor=G.+Baderauthor=M.+Sandersonauthor=C.+Kleinauthor=O.+Kraemerauthor=F.+Himmelsbachauthor=E.+Haaksmaauthor=G.+R.+Adolf&title=Target+binding+properties+and+cellular+activity+of+afatinib+%28BIBW+2992%29%2C+an+irreversible+ErbB+family+blocker&doi=10.1124%2Fjpet.112.197756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Target binding properties and cellular activity of afatinib (BIBW 2992), an irreversible ErbB family blocker</span></div><div class="casAuthors">Solca, Flavio; Dahl, Goeran; Zoephel, Andreas; Bader, Gerd; Sanderson, Michael; Klein, Christian; Kraemer, Oliver; Himmelsbach, Frank; Haaksma, Eric; Adolf, Guenther R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-350</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Deregulation of the ErbB (proto-oncogene B of the avian erythroblastosis virus AEV-H strain) receptor network is well recognized as an oncogenic driver in epithelial cancers.  Several targeted drugs have been developed, including antibodies and small-mol. kinase inhibitors, each of them characterized by distinct patterns of ErbB receptor interactions.  Understanding the precise pharmacol. properties of these compds. is important for optimal use in clin. practice.  Afatinib [BIBW 2992; N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butenamide] is an ATP-competitive anilinoquinazoline deriv. harboring a reactive acrylamide group.  It was designed to covalently bind and irreversibly block enzymically active ErbB receptor family members.  Here, we show by X-ray crystallog. the covalent binding of afatinib to wild-type epidermal growth factor receptor (EGFR) and by mass spectrometry the covalent interaction with EGFR, EGFRL858R/T790M, human epidermal growth factor receptor 2 (HER2), and ErbB-4.  Afatinib potently inhibits the enzymatic activity of ErbB-4 (EC50 = 1 nM) and the proliferation of cancer cell lines driven by multiple ErbB receptor aberrations at concns. below 100 nM.  N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butanamide (BI 37781), a close analog of afatinib lacking the acrylamide group and thus incapable of covalent bond formation, had similar potency on cells driven by EGFR or EGFRL858R, but less or no detectable activity on cells expressing EGFRL858R/T790M HER2 or ErbB-4.  These results stress the importance of the acrylamide group and show that afatinib differs from approved ErbB targeting agents by irreversibly inhibiting the kinase activity of all ErbB family members.  They provide a mechanistic rationale for the distinct pharmacol. features of this compd. and explain the clin. activity seen in some patients who are resistant to antibody or kinase inhibitor therapy because of secondary mutations or ErbB receptor "reprogramming.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofXgQqaKXv-bVg90H21EOLACvtfcHk0lg9q3VuPfxQHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE&md5=4a57df32abe8101f287fc6251f63f92d</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1124%2Fjpet.112.197756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.112.197756%26sid%3Dliteratum%253Aachs%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DSanderson%26aufirst%3DM.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DKraemer%26aufirst%3DO.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DHaaksma%26aufirst%3DE.%26aulast%3DAdolf%26aufirst%3DG.%2BR.%26atitle%3DTarget%2520binding%2520properties%2520and%2520cellular%2520activity%2520of%2520afatinib%2520%2528BIBW%25202992%2529%252C%2520an%2520irreversible%2520ErbB%2520family%2520blocker%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2012%26volume%3D343%26spage%3D342%26epage%3D350%26doi%3D10.1124%2Fjpet.112.197756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Pan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheerens, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprengeler, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrill, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendonca, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweeney, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grothaus, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffery, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spoerke, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honigberg, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalrymple, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, J. T.</span><span> </span><span class="NLM_article-title">Discovery of selective irreversible inhibitors for Bruton’s tyrosine kinase</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">58</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1002/cmdc.200600221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1002%2Fcmdc.200600221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=17154430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=58-61&author=Z.+Panauthor=H.+Scheerensauthor=S.+J.+Liauthor=B.+E.+Schultzauthor=P.+A.+Sprengelerauthor=L.+C.+Burrillauthor=R.+V.+Mendoncaauthor=M.+D.+Sweeneyauthor=K.+C.+Scottauthor=P.+G.+Grothausauthor=D.+A.+Jefferyauthor=J.+M.+Spoerkeauthor=L.+A.+Honigbergauthor=P.+R.+Youngauthor=S.+A.+Dalrympleauthor=J.+T.+Palmer&title=Discovery+of+selective+irreversible+inhibitors+for+Bruton%E2%80%99s+tyrosine+kinase&doi=10.1002%2Fcmdc.200600221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective irreversible inhibitors for bruton's tyrosine kinase</span></div><div class="casAuthors">Pan, Zhengying; Scheerens, Heleen; Li, Shyr-Jiann; Schultz, Brian E.; Sprengeler, Paul A.; Burrill, L. Chuck; Mendonca, Rohan V.; Sweeney, Michael D.; Scott, Keana C. K.; Grothaus, Paul G.; Jeffery, Douglas A.; Spoerke, Jill M.; Honigberg, Lee A.; Young, Peter R.; Dalrymple, Stacie A.; Palmer, James T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-61</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Synthesis and pharmacol. evaluation of a series of peptidomimetic quinoline derivs. was undertaken to evaluate their efficacy in acting as selective irreversible inhibitors of Bruton's tyrosine kinase (Btk).  With the relative scarcity of knowledge on the inhibition of Btk it is crucial to discover a potent and selective tool compd. for this kinase.  Herein is described the discovery of selective irreversible Btk inhibitors and their efficacy in a mouse RA model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovT3kBCYECdbVg90H21EOLACvtfcHk0lg9q3VuPfxQHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D&md5=023ba8bca9942ad0127d7cf32293e9d2</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600221%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DScheerens%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.%2BJ.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26aulast%3DSprengeler%26aufirst%3DP.%2BA.%26aulast%3DBurrill%26aufirst%3DL.%2BC.%26aulast%3DMendonca%26aufirst%3DR.%2BV.%26aulast%3DSweeney%26aufirst%3DM.%2BD.%26aulast%3DScott%26aufirst%3DK.%2BC.%26aulast%3DGrothaus%26aufirst%3DP.%2BG.%26aulast%3DJeffery%26aufirst%3DD.%2BA.%26aulast%3DSpoerke%26aufirst%3DJ.%2BM.%26aulast%3DHonigberg%26aufirst%3DL.%2BA.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DDalrymple%26aufirst%3DS.%2BA.%26aulast%3DPalmer%26aufirst%3DJ.%2BT.%26atitle%3DDiscovery%2520of%2520selective%2520irreversible%2520inhibitors%2520for%2520Bruton%25E2%2580%2599s%2520tyrosine%2520kinase%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26spage%3D58%26epage%3D61%26doi%3D10.1002%2Fcmdc.200600221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambrogio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirieac, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padera, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baum, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Himmelsbach, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rettig, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solca, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span> </span><span class="NLM_article-title">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">4702</span><span class="NLM_x">–</span> <span class="NLM_lpage">4711</span><span class="refDoi"> DOI: 10.1038/onc.2008.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fonc.2008.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=18408761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=4702-4711&author=D.+Liauthor=L.+Ambrogioauthor=T.+Shimamuraauthor=S.+Kuboauthor=M.+Takahashiauthor=L.+R.+Chirieacauthor=R.+F.+Paderaauthor=G.+I.+Shapiroauthor=A.+Baumauthor=F.+Himmelsbachauthor=W.+J.+Rettigauthor=M.+Meyersonauthor=F.+Solcaauthor=H.+Greulichauthor=K.+K.+Wong&title=BIBW2992%2C+an+irreversible+EGFR%2FHER2+inhibitor+highly+effective+in+preclinical+lung+cancer+models&doi=10.1038%2Fonc.2008.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span></div><div class="casAuthors">Li, D.; Ambrogio, L.; Shimamura, T.; Kubo, S.; Takahashi, M.; Chirieac, L. R.; Padera, R. F.; Shapiro, G. I.; Baum, A.; Himmelsbach, F.; Rettig, W. J.; Meyerson, M.; Solca, F.; Greulich, H.; Wong, K-K.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4702-4711</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genetic alterations in the kinase domain of the epidermal growth factor receptor (EGFR) in non-small cell lung cancer (NSCLC) patients are assocd. with sensitivity to treatment with small mol. tyrosine kinase inhibitors.  Although first-generation reversible, ATP-competitive inhibitors showed encouraging clin. responses in lung adenocarcinoma tumors harboring such EGFR mutations, almost all patients developed resistance to these inhibitors over time.  Such resistance to first-generation EGFR inhibitors was frequently linked to an acquired T790M point mutation in the kinase domain of EGFR, or upregulation of signaling pathways downstream of HER3.  Overcoming these mechanisms of resistance, as well as primary resistance to reversible EGFR inhibitors driven by a subset of EGFR mutations, will be necessary for development of an effective targeted therapy regimen.  Here, we show that BIBW2992, an anilino-quinazoline designed to irreversibly bind EGFR and HER2, potently suppresses the kinase activity of wild-type and activated EGFR and HER2 mutants, including erlotinib-resistant isoforms.  Consistent with this activity, BIBW2992 suppresses transformation in isogenic cell-based assays, inhibits survival of cancer cell lines and induces tumor regression in xenograft and transgenic lung cancer models, with superior activity over erlotinib.  These findings encourage further testing of BIBW2992 in lung cancer patients harboring EGFR or HER2 oncogenes.  Oncogene (2008) 27, 4702-4711; doi:10.1038/onc.2008.109; published online 14 Apr. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5PQdKep-Sp7Vg90H21EOLACvtfcHk0li1i-Reg0sC7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D&md5=24945893e83b4e3d01c267100fdaf57c</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.109%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DAmbrogio%26aufirst%3DL.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DKubo%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DChirieac%26aufirst%3DL.%2BR.%26aulast%3DPadera%26aufirst%3DR.%2BF.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DRettig%26aufirst%3DW.%2BJ.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.%2BK.%26atitle%3DBIBW2992%252C%2520an%2520irreversible%2520EGFR%252FHER2%2520inhibitor%2520highly%2520effective%2520in%2520preclinical%2520lung%2520cancer%2520models%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D4702%26epage%3D4711%26doi%3D10.1038%2Fonc.2008.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Food and Drug Administration</span><span class="NLM_x"> (FDA) approved drug products. </span><a href="https://www.accessdata.fda.gov/scripts/cder/drugsatfda/" class="extLink">https://www.accessdata.fda.gov/scripts/cder/drugsatfda/</a> (accessed Dec 21,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Food+and+Drug+Administration+%28FDA%29+approved+drug+products.+https%3A%2F%2Fwww.accessdata.fda.gov%2Fscripts%2Fcder%2Fdrugsatfda%2F+%28accessed+Dec+21%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Garuti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">2981</span><span class="NLM_x">–</span> <span class="NLM_lpage">2994</span><span class="refDoi"> DOI: 10.2174/092986711796391705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.2174%2F092986711796391705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=21651479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsl2ru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=2981-2994&author=L.+Garutiauthor=M.+Robertiauthor=G.+Bottegoni&title=Irreversible+protein+kinase+inhibitors&doi=10.2174%2F092986711796391705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible protein kinase inhibitors</span></div><div class="casAuthors">Garuti, L.; Roberti, M.; Bottegoni, G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2981-2994</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Targeting cancer with small mol. irreversible inhibitors of kinases represents an emerging challenge in drug discovery.  Irreversible inhibitors bind to kinase active site in a covalent and irreversible form, most frequently by reacting with a nucleophilic cysteine residue, located near the ATP binding pocket.  The most common mechanism is the Michael reaction, that refers to the addn. of a nucleophile, such as cysteine, to an α,β-unsatd. carbonyl.  The nucleophile reacts at the electrophilic β-position to form an adduct; as a result the inhibitor irreversibly blocks binding of ATP to the kinase, rendering the kinase inactive.  Different cysteine-reactive groups have been evaluated, an acrylamide or a substituted acrylamide moiety are the Michael acceptors of choice.  There are some advantages for the irreversible kinase inhibition.  These compds. are highly selective because they target a specific cysteine and only a limited no. of kinases has a cysteine at the corresponding position.  Another advantage is that covalent bond formation can overcome competition with the high endogenous concn. of ATP.  A further motivation for designing irreversible inhibitors is their longer duration of action respect to conventional inhibitors.  In fact, once bound to enzyme, these compds. do not readily dissoc. and the inhibition continues even after the inhibitor leaves the circulation.  Moreover, these inhibitors have the potential to overcome and prevent the emergence of acquired resistance conferred by mutations.  In this review examples of irreversible inhibitors are reported, focusing on chem. structures, SAR and biol. activities.  The great potential of these compds. could open new and promising perspectives for a broader application of this approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLFECsvPg2LVg90H21EOLACvtfcHk0li1i-Reg0sC7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsl2ru70%253D&md5=8c732cb9de810f9fac6484c245e41867</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.2174%2F092986711796391705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711796391705%26sid%3Dliteratum%253Aachs%26aulast%3DGaruti%26aufirst%3DL.%26aulast%3DRoberti%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D2981%26epage%3D2994%26doi%3D10.2174%2F092986711796391705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabnis, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhrlage, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Developing irreversible inhibitors of the protein kinase cysteinome</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.chembiol.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=23438744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=146-159&author=Q.+Liuauthor=Y.+Sabnisauthor=Z.+Zhaoauthor=T.+Zhangauthor=S.+J.+Buhrlageauthor=L.+H.+Jonesauthor=N.+S.+Gray&title=Developing+irreversible+inhibitors+of+the+protein+kinase+cysteinome&doi=10.1016%2Fj.chembiol.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span></div><div class="casAuthors">Liu, Qingsong; Sabnis, Yogesh; Zhao, Zheng; Zhang, Tinghu; Buhrlage, Sara J.; Jones, Lyn H.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-159</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are a large family of approx. 530 highly conserved enzymes that transfer a γ-phosphate group from ATP to a variety of amino acid residues, such as tyrosine, serine, and threonine, that serves as a ubiquitous mechanism for cellular signal transduction.  The clin. success of a no. of kinase-directed drugs and the frequent observation of disease causing mutations in protein kinases suggest that a large no. of kinases may represent therapeutically relevant targets.  To date, the majority of clin. and preclin. kinase inhibitors are ATP competitive, noncovalent inhibitors that achieve selectivity through recognition of unique features of particular protein kinases.  Recently, there has been renewed interest in the development of irreversible inhibitors that form covalent bonds with cysteine or other nucleophilic residues in the ATP-binding pocket.  Irreversible kinase inhibitors have a no. of potential advantages including prolonged pharmacodynamics, suitability for rational design, high potency, and ability to validate pharmacol. specificity through mutation of the reactive cysteine residue.  Here, we review recent efforts to develop cysteine-targeted irreversible protein kinase inhibitors and discuss their modes of recognizing the ATP-binding pocket and their biol. activity profiles.  In addn., we provided an informatics assessment of the potential "kinase cysteinome" and discuss strategies for the efficient development of new covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVGD7TX3oIeLVg90H21EOLACvtfcHk0lhMaAAoSuzWJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D&md5=cef80823123bae88362e816a0a638133</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDeveloping%2520irreversible%2520inhibitors%2520of%2520the%2520protein%2520kinase%2520cysteinome%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D146%26epage%3D159%26doi%3D10.1016%2Fj.chembiol.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Mi, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiwari, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, X. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">To, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambudkar, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, L. W.</span><span> </span><span class="NLM_article-title">Apatinib (YN968D1) reverses multidrug resistance by inhibiting the efflux function of multiple ATP-binding cassette transporters</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">7981</span><span class="NLM_x">–</span> <span class="NLM_lpage">7991</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-0111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1158%2F0008-5472.CAN-10-0111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=20876799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yqs7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=7981-7991&author=Y.+J.+Miauthor=Y.+J.+Liangauthor=H.+B.+Huangauthor=H.+Y.+Zhaoauthor=C.+P.+Wuauthor=F.+Wangauthor=L.+Y.+Taoauthor=C.+Z.+Zhangauthor=C.+L.+Daiauthor=A.+K.+Tiwariauthor=X.+X.+Maauthor=K.+K.+Toauthor=S.+V.+Ambudkarauthor=Z.+S.+Chenauthor=L.+W.+Fu&title=Apatinib+%28YN968D1%29+reverses+multidrug+resistance+by+inhibiting+the+efflux+function+of+multiple+ATP-binding+cassette+transporters&doi=10.1158%2F0008-5472.CAN-10-0111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Apatinib (YN968D1) Reverses Multidrug Resistance by Inhibiting the Efflux Function of Multiple ATP-Binding Cassette Transporters</span></div><div class="casAuthors">Mi, Yan-jun; Liang, Yong-ju; Huang, Hong-bing; Zhao, Hong-yun; Wu, Chung-Pu; Wang, Fang; Tao, Li-yang; Zhang, Chuan-zhao; Dai, Chun-Ling; Tiwari, Amit K.; Ma, Xiao-xu; To, Kenneth Kin Wah; Ambudkar, Suresh V.; Chen, Zhe-Sheng; Fu, Li-wu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7981-7991</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Apatinib, a small-mol. multitargeted tyrosine kinase inhibitor, is in phase III clin. trial for the treatment of patients with non-small-cell lung cancer and gastric cancer in China.  In this study, we detd. the effect of apatinib on the interaction of specific antineoplastic compds. with P-glycoprotein (ABCB1), multidrug resistance protein 1 (MRP1, ABCC1), and breast cancer resistance protein (BCRP, ABCG2).  Our results showed that apatinib significantly enhanced the cytotoxicity of ABCB1 or ABCG2 substrate drugs in KBv200, MCF-7/adr, and HEK293/ABCB1 cells overexpressing ABCB1 and in S1-M1-80, MCF-7/FLV1000, and HEK293/ABCG2-R2 cells overexpressing ABCG2 (wild-type).  In contrast, apatinib did not alter the cytotoxicity of specific substrates in the parental cells and cells overexpressing ABCC1.  Apatinib significantly increased the intracellular accumulation of rhodamine 123 and doxorubicin in the multidrug resistance (MDR) cells.  Furthermore, apatinib significantly inhibited the photoaffinity labeling of both ABCB1 and ABCG2 with [125I]iodoarylazidoprazosin in a concn.-dependent manner.  The ATPase activity of both ABCB1 and ABCG2 was significantly increased by apatinib.  However, apatinib, at a concn. that produced a reversal of MDR, did not significantly alter the ABCB1 or ABCG2 protein or mRNA expression levels or the phosphorylation of AKT and extracellular signal-regulated kinase 1/2 (ERK1/2).  Importantly, apatinib significantly enhanced the effect of paclitaxel against the ABCB1-resistant KBv200 cancer cell xenografts in nude mice.  In conclusion, apatinib reverses ABCB1- and ABCG2-mediated MDR by inhibiting their transport function, but not by blocking the AKT or ERK1/2 pathway or downregulating ABCB1 or ABCG2 expression.  Apatinib may be useful in circumventing MDR to other conventional antineoplastic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7UdIyUL3T57Vg90H21EOLACvtfcHk0lhMaAAoSuzWJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yqs7nF&md5=ecf314e6eda8940a8a739398c959eb43</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-0111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-0111%26sid%3Dliteratum%253Aachs%26aulast%3DMi%26aufirst%3DY.%2BJ.%26aulast%3DLiang%26aufirst%3DY.%2BJ.%26aulast%3DHuang%26aufirst%3DH.%2BB.%26aulast%3DZhao%26aufirst%3DH.%2BY.%26aulast%3DWu%26aufirst%3DC.%2BP.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DTao%26aufirst%3DL.%2BY.%26aulast%3DZhang%26aufirst%3DC.%2BZ.%26aulast%3DDai%26aufirst%3DC.%2BL.%26aulast%3DTiwari%26aufirst%3DA.%2BK.%26aulast%3DMa%26aufirst%3DX.%2BX.%26aulast%3DTo%26aufirst%3DK.%2BK.%26aulast%3DAmbudkar%26aufirst%3DS.%2BV.%26aulast%3DChen%26aufirst%3DZ.%2BS.%26aulast%3DFu%26aufirst%3DL.%2BW.%26atitle%3DApatinib%2520%2528YN968D1%2529%2520reverses%2520multidrug%2520resistance%2520by%2520inhibiting%2520the%2520efflux%2520function%2520of%2520multiple%2520ATP-binding%2520cassette%2520transporters%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D7981%26epage%3D7991%26doi%3D10.1158%2F0008-5472.CAN-10-0111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0li9DfkEXZacAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">Salama, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. B.</span><span> </span><span class="NLM_article-title">Trametinib (GSK1120212) in the treatment of melanoma</span> <span class="citation_source-journal">Expert Opin. Pharmacother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">619</span><span class="NLM_x">–</span> <span class="NLM_lpage">627</span><span class="refDoi"> DOI: 10.1517/14656566.2013.770475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1517%2F14656566.2013.770475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=23432625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFSlu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=619-627&author=A.+K.+Salamaauthor=K.+B.+Kim&title=Trametinib+%28GSK1120212%29+in+the+treatment+of+melanoma&doi=10.1517%2F14656566.2013.770475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">(GSK1120212) in the treatment of melanoma</span></div><div class="casAuthors">Salama, April K. S.; Kim, Kevin B.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Pharmacotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">619-627</span>CODEN:
                <span class="NLM_cas:coden">EOPHF7</span>;
        ISSN:<span class="NLM_cas:issn">1465-6566</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The discovery of somatic mutations in melanoma has advanced our knowledge of the biol. of the disease.  The mutations, such as those in NRAS, BRAF, GNAQ and GNA11, promote the growth of melanoma cells in most part through the mitogen-activated protein kinase (MAPK) pathway.  Understanding the mol. pathways of some of these mutations has resulted in the successful development of selective BRAF inhibitors.  Yet, a cure for advanced melanoma is far from reality.  Targeting MAPK/ERK kinase (MEK), an essential intermediary kinase protein within the MAPK pathway, may be a promising way to treat patients with BRAF or other genomic mutation.Areas covered: The authors discuss the MAPK pathway in melanoma and review the preclin. and clin. studies of the MEK inhibitor, , in melanoma.  They also discuss the potential of using trametinib in the targeted therapy of advanced melanoma.Expert opinion: Studies have demonstrated the activity of trametinib in BRAF-mutant melanoma, suggesting that it could be a very reasonable alternative to BRAF inhibitors for these patients.  Current clin. investigations have shown great promise with the combination of trametinib and dabrafenib in patients with BRAF-mutant melanoma; a no. of clin. trials of trametinib in combination with other targeted drugs are underway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQSJOlEgHVLbVg90H21EOLACvtfcHk0li9DfkEXZacAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFSlu70%253D&md5=8cdadb7c9472b8a0fca57d1db37cff6f</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1517%2F14656566.2013.770475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14656566.2013.770475%26sid%3Dliteratum%253Aachs%26aulast%3DSalama%26aufirst%3DA.%2BK.%26aulast%3DKim%26aufirst%3DK.%2BB.%26atitle%3DTrametinib%2520%2528GSK1120212%2529%2520in%2520the%2520treatment%2520of%2520melanoma%26jtitle%3DExpert%2520Opin.%2520Pharmacother.%26date%3D2013%26volume%3D14%26spage%3D619%26epage%3D627%26doi%3D10.1517%2F14656566.2013.770475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Rational design of inhibitors that bind to inactive kinase conformations</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/nchembio799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnchembio799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=16783341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=358-364&author=Y.+Liuauthor=N.+S.+Gray&title=Rational+design+of+inhibitors+that+bind+to+inactive+kinase+conformations&doi=10.1038%2Fnchembio799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design of inhibitors that bind to inactive kinase conformations</span></div><div class="casAuthors">Liu, Yi; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The majority of kinase inhibitors, known as type I inhibitors that have been developed so far target the ATP binding site of the kinase in its active conformation, in which the activation loop is phosphorylated.  Recently, crystal structures of inhibitors such as imatinib, BIRB796, and sorafenib (BAY43-9006), known as type II inhibitors, have revealed a new binding mode that exploits an addnl. binding site immediately adjacent to the region occupied by ATP.  This pocket is made accessible by an activation-loop rearrangement that is characteristic of kinases in an inactive conformation.  The examn. of binding features of known type II inhibitors is discussed and a pharmacophore model currently used in the design of a new generation of structurally diverse type II inhibitors is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxahsMPlhgDbVg90H21EOLACvtfcHk0li9DfkEXZacAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D&md5=4a9347a44a2cf4a412c9741fdcc5696b</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1038%2Fnchembio799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio799%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DRational%2520design%2520of%2520inhibitors%2520that%2520bind%2520to%2520inactive%2520kinase%2520conformations%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group">McTigue, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solowiej, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kania, R. S.</span><span> </span><span class="NLM_article-title">Molecular conformations, interactions, and properties associated with drug efficiency and clinical performance among VEGFR TK inhibitors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">18281</span><span class="NLM_x">–</span> <span class="NLM_lpage">18289</span><span class="refDoi"> DOI: 10.1073/pnas.1207759109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1073%2Fpnas.1207759109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=22988103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslymtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=18281-18289&author=M.+McTigueauthor=B.+W.+Murrayauthor=J.+H.+Chenauthor=Y.+L.+Dengauthor=J.+Solowiejauthor=R.+S.+Kania&title=Molecular+conformations%2C+interactions%2C+and+properties+associated+with+drug+efficiency+and+clinical+performance+among+VEGFR+TK+inhibitors&doi=10.1073%2Fpnas.1207759109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular conformations, interactions, and properties associated with drug efficiency and clinical performance among VEGFR TK inhibitors</span></div><div class="casAuthors">McTigue, Michele; Murray, Brion William; Chen, Jeffrey H.; Deng, Ya-Li; Solowiej, James; Kania, Robert S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">18281-18289, S18281/1-S18281/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Analyses of compds. in clin. development have shown that ligand efficient-mols. with privileged phys. properties and low dose are less likely to fail in the various stages of clin. testing, have fewer postapproval withdrawals, and are less likely to receive black box safety warnings.  However, detailed side-by-side examn. of mol. interactions and properties within single drug classes are lacking.  As a class, VEGF receptor tyrosine kinase inhibitors (VEGFR TKIs) have changed the landscape of how cancer is treated, particularly in clear cell renal cell carcinoma, which is molecularly linked to the VEGF signaling axis.  Despite the clear role of the mol. target, member mols. of this validated drug class exhibit distinct clin. efficacy and safety profiles in comparable renal cell carcinoma clin. studies.  The first head-to-head randomized phase III comparative study between active VEGFR TKIs has confirmed significant differences in clin. performance.  To elucidate how fundamental drug potency-efficiency is achieved and impacts differentiation within the VEGFR TKI class, we detd. potencies, time dependence, selectivities, and X-ray structures of the drug-kinase complexes using a VEGFR2 TK construct inclusive of the important juxtamembrane domain.  Collectively, the studies elucidate unique drug-kinase interactions that are dependent on distinct juxtamembrane domain conformations, resulting in significant potency and ligand efficiency differences.  The identified structural trends are consistent with in vitro measurements, which translate well to clin. performance, underscoring a principle that may be broadly applicable to prospective drug design for optimal in vivo performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsIUWiD4sXtbVg90H21EOLACvtfcHk0li9DfkEXZacAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslymtb3I&md5=4a5ec677298a8c924c0f9557ca688a79</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1207759109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1207759109%26sid%3Dliteratum%253Aachs%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DDeng%26aufirst%3DY.%2BL.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DKania%26aufirst%3DR.%2BS.%26atitle%3DMolecular%2520conformations%252C%2520interactions%252C%2520and%2520properties%2520associated%2520with%2520drug%2520efficiency%2520and%2520clinical%2520performance%2520among%2520VEGFR%2520TK%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D18281%26epage%3D18289%26doi%3D10.1073%2Fpnas.1207759109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group">Pemovska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McTigue, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallioniemi, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porkka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wennerberg, K.</span><span> </span><span class="NLM_article-title">Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">519</span><span class="NLM_x">, </span> <span class="NLM_fpage">102</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1038/nature14119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnature14119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25686603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=519&publication_year=2015&pages=102-105&author=T.+Pemovskaauthor=E.+Johnsonauthor=M.+Kontroauthor=G.+A.+Repaskyauthor=J.+Chenauthor=P.+Wellsauthor=C.+N.+Croninauthor=M.+McTigueauthor=O.+Kallioniemiauthor=K.+Porkkaauthor=B.+W.+Murrayauthor=K.+Wennerberg&title=Axitinib+effectively+inhibits+BCR-ABL1%28T315I%29+with+a+distinct+binding+conformation&doi=10.1038%2Fnature14119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation</span></div><div class="casAuthors">Pemovska, Tea; Johnson, Eric; Kontro, Mika; Repasky, Gretchen A.; Chen, Jeffrey; Wells, Peter; Cronin, Ciaran N.; McTigue, Michele; Kallioniemi, Olli; Porkka, Kimmo; Murray, Brion W.; Wennerberg, Krister</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">519</span>
        (<span class="NLM_cas:issue">7541</span>),
    <span class="NLM_cas:pages">102-105</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The BCR-ABL1 fusion gene is a driver oncogene in chronic myeloid leukemia and 30-50% of cases of adult acute lymphoblastic leukemia.  Introduction of ABL1 kinase inhibitors (for example, imatinib) has markedly improved patient survival, but acquired drug resistance remains a challenge.  Point mutations in the ABL1 kinase domain weaken inhibitor binding and represent the most common clin. resistance mechanism.  The BCR-ABL1 kinase domain gatekeeper mutation Thr315Ile (T315I) confers resistance to all approved ABL1 inhibitors except ponatinib, which has toxicity limitations.  Here we combine comprehensive drug sensitivity and resistance profiling of patient cells ex vivo with structural anal. to establish the VEGFR tyrosine kinase inhibitor axitinib as a selective and effective inhibitor for T315I-mutant BCR-ABL1-driven leukemia.  Axitinib potently inhibited BCR-ABL1(T315I), at both biochem. and cellular levels, by binding to the active form of ABL1(T315I) in a mutation-selective binding mode.  These findings suggest that the T315I mutation shifts the conformational equil. of the kinase in favor of an active (DFG-in) A-loop conformation, which has more optimal binding interactions with axitinib.  Treatment of a T315I chronic myeloid leukemia patient with axitinib resulted in a rapid redn. of T315I-pos. cells from bone marrow.  Taken together, our findings demonstrate an unexpected opportunity to repurpose axitinib, an anti-angiogenic drug approved for renal cancer, as an inhibitor for ABL1 gatekeeper mutant drug-resistant leukemia patients.  This study shows that wild-type proteins do not always sample the conformations available to disease-relevant mutant proteins and that comprehensive drug testing of patient-derived cells can identify unpredictable, clin. significant drug-repositioning opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreXgtLS5A1lLVg90H21EOLACvtfcHk0ljAs9dceppzwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntbs%253D&md5=af8549ae0527bfcbda6d6e05b548f928</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1038%2Fnature14119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14119%26sid%3Dliteratum%253Aachs%26aulast%3DPemovska%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DE.%26aulast%3DKontro%26aufirst%3DM.%26aulast%3DRepasky%26aufirst%3DG.%2BA.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWells%26aufirst%3DP.%26aulast%3DCronin%26aufirst%3DC.%2BN.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DKallioniemi%26aufirst%3DO.%26aulast%3DPorkka%26aufirst%3DK.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DWennerberg%26aufirst%3DK.%26atitle%3DAxitinib%2520effectively%2520inhibits%2520BCR-ABL1%2528T315I%2529%2520with%2520a%2520distinct%2520binding%2520conformation%26jtitle%3DNature%26date%3D2015%26volume%3D519%26spage%3D102%26epage%3D105%26doi%3D10.1038%2Fnature14119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span>Harvard Medical School (HMS) LINCS Database. <a href="http://lincs.hms.harvard.edu/explore/" class="extLink">http://lincs.hms.harvard.edu/explore/</a> (accessed Sep 22,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Harvard+Medical+School+%28HMS%29+LINCS+Database.+http%3A%2F%2Flincs.hms.harvard.edu%2Fexplore%2F+%28accessed+Sep+22%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group">Shmelkov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grigoryan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swetnam, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tivon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shmelkov, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardozo, T.</span><span> </span><span class="NLM_article-title">Historeceptomic fingerprints for drug-like compounds</span> <span class="citation_source-journal">Front. Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">371</span><span class="refDoi"> DOI: 10.3389/fphys.2015.00371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.3389%2Ffphys.2015.00371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=26733872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BC28rnsFGktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=371&author=E.+Shmelkovauthor=A.+Grigoryanauthor=J.+Swetnamauthor=J.+Xinauthor=D.+Tivonauthor=S.+V.+Shmelkovauthor=T.+Cardozo&title=Historeceptomic+fingerprints+for+drug-like+compounds&doi=10.3389%2Ffphys.2015.00371"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Historeceptomic Fingerprints for Drug-Like Compounds</span></div><div class="casAuthors">Shmelkov Evgeny; Grigoryan Arsen; Xin Junyang; Cardozo Timothy; Swetnam James; Tivon Doreen; Shmelkov Sergey V</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in physiology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">371</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Most drugs exert their beneficial and adverse effects through their combined action on several different molecular targets (polypharmacology).  The true molecular fingerprint of the direct action of a drug has two components: the ensemble of all the receptors upon which a drug acts and their level of expression in organs/tissues.  Conversely, the fingerprint of the adverse effects of a drug may derive from its action in bystander tissues.  The ensemble of targets is almost always only partially known.  Here we describe an approach improving upon and integrating both components: in silico identification of a more comprehensive ensemble of targets for any drug weighted by the expression of those receptors in relevant tissues.  Our system combines more than 300,000 experimentally determined bioactivity values from the ChEMBL database and 4.2 billion molecular docking scores.  We integrated these scores with gene expression data for human receptors across a panel of human tissues to produce drug-specific tissue-receptor (historeceptomics) scores.  A statistical model was designed to identify significant scores, which define an improved fingerprint representing the unique activity of any drug.  These multi-dimensional historeceptomic fingerprints describe, in a novel, intuitive, and easy to interpret style, the holistic, in vivo picture of the mechanism of any drug's action.  Valuable applications in drug discovery and personalized medicine, including the identification of molecular signatures for drugs with polypharmacologic modes of action, detection of tissue-specific adverse effects of drugs, matching molecular signatures of a disease to drugs, target identification for bioactive compounds with unknown receptors, and hypothesis generation for drug/compound phenotypes may be enabled by this approach.  The system has been deployed at drugable.org for access through a user-friendly web site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSGtyA2XMAYRySm0e6ZOaemfW6udTcc2ebFhjVJd-jq87ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rnsFGktQ%253D%253D&md5=172619889fcb56477c076851c7219bed</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.3389%2Ffphys.2015.00371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphys.2015.00371%26sid%3Dliteratum%253Aachs%26aulast%3DShmelkov%26aufirst%3DE.%26aulast%3DGrigoryan%26aufirst%3DA.%26aulast%3DSwetnam%26aufirst%3DJ.%26aulast%3DXin%26aufirst%3DJ.%26aulast%3DTivon%26aufirst%3DD.%26aulast%3DShmelkov%26aufirst%3DS.%2BV.%26aulast%3DCardozo%26aufirst%3DT.%26atitle%3DHistoreceptomic%2520fingerprints%2520for%2520drug-like%2520compounds%26jtitle%3DFront.%2520Physiol.%26date%3D2015%26volume%3D6%26spage%3D371%26doi%3D10.3389%2Ffphys.2015.00371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group">UniProt Consortium</span><span> </span><span class="NLM_article-title">Update on activities at the Universal Protein Resource (UniProt) in 2013</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">D43</span><span class="NLM_x">–</span> <span class="NLM_lpage">D47</span><span class="refDoi"> DOI: 10.1093/nar/gks1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1093%2Fnar%2Fgks1068" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=D43-D47&author=UniProt+Consortium&title=Update+on+activities+at+the+Universal+Protein+Resource+%28UniProt%29+in+2013&doi=10.1093%2Fnar%2Fgks1068"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks1068%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUpdate%2520on%2520activities%2520at%2520the%2520Universal%2520Protein%2520Resource%2520%2528UniProt%2529%2520in%25202013%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3DD43%26epage%3DD47%26doi%3D10.1093%2Fnar%2Fgks1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span>UniProt Consortium. Human and mouse protein kinases: classification and index. <a href="http://www.uniprot.org/docs/pkinfam.txt" class="extLink">http://www.uniprot.org/docs/pkinfam.txt</a> (accessed May 15,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=UniProt+Consortium.+Human+and+mouse+protein+kinases%3A+classification+and+index.+http%3A%2F%2Fwww.uniprot.org%2Fdocs%2Fpkinfam.txt+%28accessed+May+15%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group">Rarey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">A fast flexible docking method using an incremental construction algorithm</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">261</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">489</span><span class="refDoi"> DOI: 10.1006/jmbi.1996.0477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1006%2Fjmbi.1996.0477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=8780787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADyaK28XltlKisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1996&pages=470-489&author=M.+Rareyauthor=B.+Kramerauthor=T.+Lengauerauthor=G.+Klebe&title=A+fast+flexible+docking+method+using+an+incremental+construction+algorithm&doi=10.1006%2Fjmbi.1996.0477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">A fast flexible docking method using an incremental construction algorithm</span></div><div class="casAuthors">Rarey, Matthias; Kramer, Bernd; Lengauer, Thomas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">470-489</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">We present an automatic method for docking org. ligands into protein binding sites.  The method can be used in the design process of specific protein ligands.  It combines an appropriate model of the physico-chem. properties of the docked mols. with efficient methods for sampling the conformational space of the ligand.  If the ligand is flexible, it can adopt a large variety of different conformations.  Each such min. in conformational space presents a potential candidate for the conformation of the ligand in the complexed state.  Our docking method samples the conformation space of the ligand on the basis of a discrete model and uses a tree-search technique for placing the ligand incrementally into the active site.  For placing the first fragment of the ligand into the protein, we use hashing techniques adapted from computer vision.  The incremental construction algorithm is based on a greedy strategy combined with efficient methods for overlap detection and for the search of new interactions.  We present results on 19 complexes of which the binding geometry has been crystallog. detd.  All considered ligands are docked in at most three minutes on a current workstation.  The exptl. obsd. binding mode of the ligand is reproduced with 0.5 to 1.2 Å rms deviation.  It is almost always found among the highest-ranking conformations computed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx9N2sFXkOfLVg90H21EOLACvtfcHk0li1Ap7nRvQt-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XltlKisLo%253D&md5=8cd8f384906ed90e894d118399e4dc4a</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0477%26sid%3Dliteratum%253Aachs%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DKramer%26aufirst%3DB.%26aulast%3DLengauer%26aufirst%3DT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520fast%2520flexible%2520docking%2520method%2520using%2520an%2520incremental%2520construction%2520algorithm%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1996%26volume%3D261%26spage%3D470%26epage%3D489%26doi%3D10.1006%2Fjmbi.1996.0477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schalon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bret, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">sc-PDB: an annotated database of druggable binding sites from the Protein Data Bank</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">717</span><span class="NLM_x">–</span> <span class="NLM_lpage">727</span><span class="refDoi"> DOI: 10.1021/ci050372x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050372x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFCgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=717-727&author=E.+Kellenbergerauthor=P.+Mullerauthor=C.+Schalonauthor=G.+Bretauthor=N.+Foataauthor=D.+Rognan&title=sc-PDB%3A+an+annotated+database+of+druggable+binding+sites+from+the+Protein+Data+Bank&doi=10.1021%2Fci050372x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">sc-PDB: An Annotated Database of Druggable Binding Sites from the Protein Data Bank</span></div><div class="casAuthors">Kellenberger, Esther; Muller, Pascal; Schalon, Claire; Bret, Guillaume; Foata, Nicolas; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">717-727</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The s.c.-PDB is a collection of 6,415 three-dimensional structures of binding sites found in the Protein Data Bank (PDB).  Binding sites were extd. from all high-resoln. crystal structures in which a complex between a protein cavity and a small-mol.-wt. ligand could be identified.  Importantly, ligands are considered from a pharmacol. and not a structural point of view.  Therefore, solvents, detergents, and most metal ions are not stored in the s.c.-PDB.  Ligands are classified into four main categories: nucleotides (< 4-mer), peptides (< 9-mer), cofactors, and org. compds.  The corresponding binding site is formed by all protein residues (including amino acids, cofactors, and important metal ions) with at least one atom within 6.5 Å of any ligand atom.  The database was carefully annotated by browsing several protein databases (PDB, UniProt, and GO) and storing, for every s.c.-PDB entry, the following features: protein name, function, source, domain and mutations, ligand name, and structure.  The repository of ligands has also been archived by diversity anal. of mol. scaffolds, and several chemoinformatics descriptors were computed to better understand the chem. space covered by stored ligands.  The s.c.-PDB may be used for several purposes: (i) screening a collection of binding sites for predicting the most likely target(s) of any ligand, (ii) analyzing the mol. similarity between different cavities, and (iii) deriving rules that describe the relationship between ligand pharmacophoric points and active-site properties.  The database is periodically updated and accessible on the web at http://bioinfo-pharma.u-strasbg.fr/scPDB/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNlZ_WkpBUQrVg90H21EOLACvtfcHk0li1Ap7nRvQt-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFCgtQ%253D%253D&md5=d035f998192ff55026c619b38f4b82ab</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fci050372x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050372x%26sid%3Dliteratum%253Aachs%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DMuller%26aufirst%3DP.%26aulast%3DSchalon%26aufirst%3DC.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DFoata%26aufirst%3DN.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3Dsc-PDB%253A%2520an%2520annotated%2520database%2520of%2520druggable%2520binding%2520sites%2520from%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D717%26epage%3D727%26doi%3D10.1021%2Fci050372x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group">Radifar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuniarti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Istyastono, E. P.</span><span> </span><span class="NLM_article-title">PyPLIF: python-based protein-ligand fnteraction fingerprinting</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span><span class="refDoi"> DOI: 10.6026/97320630009325</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.6026%2F97320630009325" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=325-328&author=M.+Radifarauthor=N.+Yuniartiauthor=E.+P.+Istyastono&title=PyPLIF%3A+python-based+protein-ligand+fnteraction+fingerprinting&doi=10.6026%2F97320630009325"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.6026%2F97320630009325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6026%252F97320630009325%26sid%3Dliteratum%253Aachs%26aulast%3DRadifar%26aufirst%3DM.%26aulast%3DYuniarti%26aufirst%3DN.%26aulast%3DIstyastono%26aufirst%3DE.%2BP.%26atitle%3DPyPLIF%253A%2520python-based%2520protein-ligand%2520fnteraction%2520fingerprinting%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D9%26spage%3D325%26epage%3D328%26doi%3D10.6026%2F97320630009325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group">Uchikoga, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirokawa, T.</span><span> </span><span class="NLM_article-title">Analysis of protein-protein docking decoys using interaction fingerprints: application to the reconstruction of CaM-ligand complexes</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">236</span><span class="refDoi"> DOI: 10.1186/1471-2105-11-236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1186%2F1471-2105-11-236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=20459766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BC3czltVCqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=236&author=N.+Uchikogaauthor=T.+Hirokawa&title=Analysis+of+protein-protein+docking+decoys+using+interaction+fingerprints%3A+application+to+the+reconstruction+of+CaM-ligand+complexes&doi=10.1186%2F1471-2105-11-236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of protein-protein docking decoys using interaction fingerprints: application to the reconstruction of CaM-ligand complexes</span></div><div class="casAuthors">Uchikoga Nobuyuki; Hirokawa Takatsugu</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">236</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Protein-protein docking for proteins with large conformational changes was analyzed by using interaction fingerprints, one of the scales for measuring similarities among complex structures, utilized especially for searching near-native protein-ligand or protein-protein complex structures.  Here, we have proposed a combined method for analyzing protein-protein docking by taking large conformational changes into consideration.  This combined method consists of ensemble soft docking with multiple protein structures, refinement of complexes, and cluster analysis using interaction fingerprints and energy profiles.  RESULTS:  To test for the applicability of this combined method, various CaM-ligand complexes were reconstructed from the NMR structures of unbound CaM.  For the purpose of reconstruction, we used three known CaM-ligands, namely, the CaM-binding peptides of cyclic nucleotide gateway (CNG), CaM kinase kinase (CaMKK) and the plasma membrane Ca2+ ATPase pump (PMCA), and thirty-one structurally diverse CaM conformations.  For each ligand, 62000 CaM-ligand complexes were generated in the docking step and the relationship between their energy profiles and structural similarities to the native complex were analyzed using interaction fingerprint and RMSD.  Near-native clusters were obtained in the case of CNG and CaMKK.  CONCLUSIONS:  The interaction fingerprint method discriminated near-native structures better than the RMSD method in cluster analysis.  We showed that a combined method that includes the interaction fingerprint is very useful for protein-protein docking analysis of certain cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAfzPCyZCcdP6AExegM5LLfW6udTcc2eaba3HDrsD5VLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3czltVCqsw%253D%253D&md5=8ef588ac0002c004b473ab2b7b287c81</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-236%26sid%3Dliteratum%253Aachs%26aulast%3DUchikoga%26aufirst%3DN.%26aulast%3DHirokawa%26aufirst%3DT.%26atitle%3DAnalysis%2520of%2520protein-protein%2520docking%2520decoys%2520using%2520interaction%2520fingerprints%253A%2520application%2520to%2520the%2520reconstruction%2520of%2520CaM-ligand%2520complexes%26jtitle%3DBMC%2520Bioinf.%26date%3D2010%26volume%3D11%26spage%3D236%26doi%3D10.1186%2F1471-2105-11-236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group">Jaccard, P.</span><span> </span><span class="NLM_article-title">The distribution of the flora in the alpine zone</span> <span class="citation_source-journal">New Phytol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1912</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1111/j.1469-8137.1912.tb05611.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1111%2Fj.1469-8137.1912.tb05611.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1912&pages=37-50&author=P.+Jaccard&title=The+distribution+of+the+flora+in+the+alpine+zone&doi=10.1111%2Fj.1469-8137.1912.tb05611.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1111%2Fj.1469-8137.1912.tb05611.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1469-8137.1912.tb05611.x%26sid%3Dliteratum%253Aachs%26aulast%3DJaccard%26aufirst%3DP.%26atitle%3DThe%2520distribution%2520of%2520the%2520flora%2520in%2520the%2520alpine%2520zone%26jtitle%3DNew%2520Phytol.%26date%3D1912%26volume%3D11%26spage%3D37%26epage%3D50%26doi%3D10.1111%2Fj.1469-8137.1912.tb05611.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group">Hartigan, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, M. A.</span><span> </span><span class="NLM_article-title">Algorithm AS 136: A K-means clustering algorithm</span> <span class="citation_source-journal">Appl. Stat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1979</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">100</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span><span class="refDoi"> DOI: 10.2307/2346830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.2307%2F2346830" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1979&pages=100-108&author=J.+A.+Hartiganauthor=M.+A.+Wong&title=Algorithm+AS+136%3A+A+K-means+clustering+algorithm&doi=10.2307%2F2346830"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.2307%2F2346830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2307%252F2346830%26sid%3Dliteratum%253Aachs%26aulast%3DHartigan%26aufirst%3DJ.%2BA.%26aulast%3DWong%26aufirst%3DM.%2BA.%26atitle%3DAlgorithm%2520AS%2520136%253A%2520A%2520K-means%2520clustering%2520algorithm%26jtitle%3DAppl.%2520Stat.%26date%3D1979%26volume%3D28%26spage%3D100%26epage%3D108%26doi%3D10.2307%2F2346830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group">Rogers, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, M.</span><span> </span><span class="NLM_article-title">Extended-connectivity fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">–</span> <span class="NLM_lpage">754</span><span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-connectivity+fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0lhvuowxfAngKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-connectivity%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Improving the search performance of extended connectivity fingerprints through activity-oriented feature filtering and application of a bit-density-dependent similarity function</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">540</span><span class="NLM_x">–</span> <span class="NLM_lpage">548</span><span class="refDoi"> DOI: 10.1002/cmdc.200800408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1002%2Fcmdc.200800408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19263458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltlCgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=540-548&author=Y.+Huauthor=E.+Lounkineauthor=J.+Bajorath&title=Improving+the+search+performance+of+extended+connectivity+fingerprints+through+activity-oriented+feature+filtering+and+application+of+a+bit-density-dependent+similarity+function&doi=10.1002%2Fcmdc.200800408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the Search Performance of Extended Connectivity Fingerprints through Activity-Oriented Feature Filtering and Application of a Bit-Density-Dependent Similarity Function</span></div><div class="casAuthors">Hu, Ye; Lounkine, Eugen; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">540-548</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Improving fingerprint search performance: An activity-oriented feature-filtering procedure and a corresponding similarity function were developed for mol.-specific fingerprints, recording ensembles of structural patterns such as the popular extended connectivity fingerprints.  Shown are comparisons of search calcns. for cyclooxygenase inhibitors based on k nearest neighbor (1NN, 10NN) and Tanimoto coeff. (Tc) calcns., and the ACF BDM approach introduced herein.  The Pipeline Pilot extended connectivity fingerprints (ECFPs) are currently among the most popular similarity search tools in drug discovery settings.  ECFPs do not have a fixed bit string format but generate variable nos. of structural features for individual test mols.  This variable string design makes ECFP representations amenable to compd.-class-directed modification.  An intuitive feature-filtering technique was devised that focuses ECFP search calcns. on feature string ensembles of given compd. activity classes.  In combination with a simple bit-d.-dependent similarity function, feature filtering consistently improved the search performance of ECFP calcns. based on Tanimoto similarity and state-of-the-art data fusion techniques on a diverse array of activity classes.  Feature filtering and the bit d. similarity metric are easily implemented in the Pipeline Pilot environment.  The approach provides a viable alternative to conventional similarity searching and should be of general interest to further improve the success rate of practical ECFP applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-keIL6bpoJ7Vg90H21EOLACvtfcHk0lhvuowxfAngKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltlCgtrg%253D&md5=b1e5043a5ca7806d9b6accfcaf606e3b</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800408%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DImproving%2520the%2520search%2520performance%2520of%2520extended%2520connectivity%2520fingerprints%2520through%2520activity-oriented%2520feature%2520filtering%2520and%2520application%2520of%2520a%2520bit-density-dependent%2520similarity%2520function%26jtitle%3DChemMedChem%26date%3D2009%26volume%3D4%26spage%3D540%26epage%3D548%26doi%3D10.1002%2Fcmdc.200800408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span>JChem Base was used for structure searching and chemical database access and management.  <span class="citation_source-book">JChem 6.2.0</span>; <span class="NLM_publisher-name">ChemAxon</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">; </span><a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=JChem+Base+was+used+for+structure+searching+and+chemical+database+access+and+management.+JChem+6.2.0%3B+ChemAxon%2C+2014%3B+http%3A%2F%2Fwww.chemaxon.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DJChem%25206.2.0%26pub%3DChemAxon%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group">Weill, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Alignment-free ultra-high-throughput comparison of druggable protein-ligand binding sites</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span><span class="refDoi"> DOI: 10.1021/ci900349y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900349y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1yjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=123-135&author=N.+Weillauthor=D.+Rognan&title=Alignment-free+ultra-high-throughput+comparison+of+druggable+protein-ligand+binding+sites&doi=10.1021%2Fci900349y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Alignment-Free Ultra-High-Throughput Comparison of Druggable Protein-Ligand Binding Sites</span></div><div class="casAuthors">Weill, Nathanael; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">123-135</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inferring the biol. function of a protein from its three-dimensional structure as well as explaining why a drug may bind to various targets is of crucial importance to modern drug discovery.  Here we present a generic 4833-integer vector describing druggable protein-ligand binding sites that can be applied to any protein and any binding cavity.  The fingerprint registers counts of pharmacophoric triplets from the Cα at. coordinates of binding-site-lining residues.  Starting from a customized data set of diverse protein-ligand binding site pairs, the most appropriate metric and a similarity threshold could be defined for similar binding sites.  The method (FuzCav) has been used in various scenarios: (i) screening a collection of 6000 binding sites for similarity to different queries; (ii) classifying protein families (serine endopeptidases, protein kinases) by binding site diversity; (iii) discriminating adenine-binding cavities from decoys.  The fingerprint generation and comparison supports ultra-high throughput (ca. 1000 measures/s), does not require prior alignment of protein binding sites, and is able to detect local similarity among subpockets.  It is thus particularly well suited to the functional annotation of novel genomic structures with low sequence identity to known X-ray templates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7OYklk3JFxLVg90H21EOLACvtfcHk0lgpHRfOoT_XwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1yjsg%253D%253D&md5=541a9800a3aa0891f179316412f1df24</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Fci900349y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900349y%26sid%3Dliteratum%253Aachs%26aulast%3DWeill%26aufirst%3DN.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DAlignment-free%2520ultra-high-throughput%2520comparison%2520of%2520druggable%2520protein-ligand%2520binding%2520sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D123%26epage%3D135%26doi%3D10.1021%2Fci900349y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 25 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Dominique Sydow, Paula Schmiel, Jérémie Mortier, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Volkamer</span>. </span><span class="cited-content_cbyCitation_article-title">KinFragLib: Exploring the Kinase Inhibitor Space Using Subpocket-Focused Fragmentation and Recombination. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6081-6094. <a href="https://doi.org/10.1021/acs.jcim.0c00839" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00839</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00839%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DKinFragLib%25253A%252BExploring%252Bthe%252BKinase%252BInhibitor%252BSpace%252BUsing%252BSubpocket-Focused%252BFragmentation%252Band%252BRecombination%26aulast%3DSydow%26aufirst%3DDominique%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D23072020%26date%3D06112020%26volume%3D60%26issue%3D12%26spage%3D6081%26epage%3D6094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zheng Zhao, <span class="NLM_string-name hlFld-ContribAuthor">Philip E. Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Insights into the Binding Modes of Viral RNA-Dependent RNA Polymerases Using a Function-Site Interaction Fingerprint Method for RNA Virus Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Proteome Research</span><span> <strong>2020,</strong> <em>19 </em>
                                    (11)
                                     , 4698-4705. <a href="https://doi.org/10.1021/acs.jproteome.0c00623" title="DOI URL">https://doi.org/10.1021/acs.jproteome.0c00623</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jproteome.0c00623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jproteome.0c00623%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Proteome%2520Research%26atitle%3DStructural%252BInsights%252Binto%252Bthe%252BBinding%252BModes%252Bof%252BViral%252BRNA-Dependent%252BRNA%252BPolymerases%252BUsing%252Ba%252BFunction-Site%252BInteraction%252BFingerprint%252BMethod%252Bfor%252BRNA%252BVirus%252BDrug%252BDiscovery%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11082020%26date%3D29092020%26date%3D18092020%26volume%3D19%26issue%3D11%26spage%3D4698%26epage%3D4705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Enade P. Istyastono, Muhammad Radifar, Nunung Yuniarti, Vivitri D. Prasasty, <span class="NLM_string-name hlFld-ContribAuthor">Sudi Mungkasi</span>. </span><span class="cited-content_cbyCitation_article-title">PyPLIF HIPPOS: A Molecular Interaction Fingerprinting Tool for Docking Results of AutoDock Vina and PLANTS. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (8)
                                     , 3697-3702. <a href="https://doi.org/10.1021/acs.jcim.0c00305" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00305</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00305%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPyPLIF%252BHIPPOS%25253A%252BA%252BMolecular%252BInteraction%252BFingerprinting%252BTool%252Bfor%252BDocking%252BResults%252Bof%252BAutoDock%252BVina%252Band%252BPLANTS%26aulast%3DIstyastono%26aufirst%3DEnade%2BP.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D27032020%26date%3D04082020%26date%3D20072020%26volume%3D60%26issue%3D8%26spage%3D3697%26epage%3D3702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zheng Zhao, <span class="NLM_string-name hlFld-ContribAuthor">Philip E. Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Revealing Acquired Resistance Mechanisms of Kinase-Targeted Drugs Using an on-the-Fly, Function-Site Interaction Fingerprint Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2020,</strong> <em>16 </em>
                                    (5)
                                     , 3152-3161. <a href="https://doi.org/10.1021/acs.jctc.9b01134" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b01134</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b01134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b01134%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DRevealing%252BAcquired%252BResistance%252BMechanisms%252Bof%252BKinase-Targeted%252BDrugs%252BUsing%252Ban%252Bon-the-Fly%25252C%252BFunction-Site%252BInteraction%252BFingerprint%252BApproach%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D13112019%26date%3D24042020%26date%3D13042020%26volume%3D16%26issue%3D5%26spage%3D3152%26epage%3D3161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zheng Zhao, Lei Xie, <span class="NLM_string-name hlFld-ContribAuthor">Philip E. Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Insights into Characterizing Binding Sites in Epidermal Growth Factor Receptor Kinase Mutants. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (1)
                                     , 453-462. <a href="https://doi.org/10.1021/acs.jcim.8b00458" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00458</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00458%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStructural%252BInsights%252Binto%252BCharacterizing%252BBinding%252BSites%252Bin%252BEpidermal%252BGrowth%252BFactor%252BReceptor%252BKinase%252BMutants%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D10072018%26date%3D11012019%26date%3D24122018%26volume%3D59%26issue%3D1%26spage%3D453%26epage%3D462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Qingsong  Liu</span>, <span class="hlFld-ContribAuthor ">Spencer  Bliven</span>, <span class="hlFld-ContribAuthor ">Lei  Xie</span>, and <span class="hlFld-ContribAuthor ">Philip E.  Bourne</span>  . </span><span class="cited-content_cbyCitation_article-title">Determining Cysteines Available for Covalent Inhibition Across the Human Kinome. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (7)
                                     , 2879-2889. <a href="https://doi.org/10.1021/acs.jmedchem.6b01815" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01815</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01815%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDetermining%252BCysteines%252BAvailable%252Bfor%252BCovalent%252BInhibition%252BAcross%252Bthe%252BHuman%252BKinome%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D13122016%26date%3D04042017%26date%3D13042017%26date%3D22032017%26volume%3D60%26issue%3D7%26spage%3D2879%26epage%3D2889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shiliang  Li</span>, <span class="hlFld-ContribAuthor ">Chaoqian  Cai</span>, <span class="hlFld-ContribAuthor ">Jiayu  Gong</span>, <span class="hlFld-ContribAuthor ">Xiaofeng  Liu</span>, <span class="hlFld-ContribAuthor ">Honglin  Li</span>. </span><span class="cited-content_cbyCitation_article-title">A fast protein binding site comparison algorithm for proteome‐wide protein function prediction and drug repurposing. </span><span class="cited-content_cbyCitation_journal-name">Proteins: Structure, Function, and Bioinformatics</span><span> <strong>2021,</strong> <em>44 </em><a href="https://doi.org/10.1002/prot.26176" title="DOI URL">https://doi.org/10.1002/prot.26176</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/prot.26176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fprot.26176%26sid%3Dliteratum%253Aachs%26jtitle%3DProteins%253A%2520Structure%252C%2520Function%252C%2520and%2520Bioinformatics%26atitle%3DA%252Bfast%252Bprotein%252Bbinding%252Bsite%252Bcomparison%252Balgorithm%252Bfor%252Bproteome%2525E2%252580%252590wide%252Bprotein%252Bfunction%252Bprediction%252Band%252Bdrug%252Brepurposing%26aulast%3DLi%26aufirst%3DShiliang%26date%3D2021%26date%3D2021%26volume%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hwangseo  Park</span>, <span class="hlFld-ContribAuthor ">Hoi-Yun  Jung</span>, <span class="hlFld-ContribAuthor ">Shinmee  Mah</span>, <span class="hlFld-ContribAuthor ">Kewon  Kim</span>, <span class="hlFld-ContribAuthor ">Sungwoo  Hong</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase and GPCR polypharmacological approach for the identification of efficient anticancer medicines. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2020,</strong> <em>18 </em>
                                    (41)
                                     , 8402-8413. <a href="https://doi.org/10.1039/D0OB01917H" title="DOI URL">https://doi.org/10.1039/D0OB01917H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0OB01917H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0OB01917H%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DKinase%252Band%252BGPCR%252Bpolypharmacological%252Bapproach%252Bfor%252Bthe%252Bidentification%252Bof%252Befficient%252Banticancer%252Bmedicines%26aulast%3DPark%26aufirst%3DHwangseo%26date%3D2020%26date%3D2020%26volume%3D18%26issue%3D41%26spage%3D8402%26epage%3D8413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Philip E.  Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Current Type I/II Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 13-28. <a href="https://doi.org/10.1007/978-3-030-48283-1_2" title="DOI URL">https://doi.org/10.1007/978-3-030-48283-1_2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-48283-1_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-48283-1_2%26sid%3Dliteratum%253Aachs%26atitle%3DOverview%252Bof%252BCurrent%252BType%252BI%25252FII%252BKinase%252BInhibitors%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2020%26date%3D2020%26spage%3D13%26epage%3D28%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DNext%252BGeneration%252BKinase%252BInhibitors%26aulast%3DShapiro%26aufirst%3DPaul%26date%3D2020%26volume%3D101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Janez  Konc</span>. </span><span class="cited-content_cbyCitation_article-title">Binding site comparisons for target-centered drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (5)
                                     , 445-454. <a href="https://doi.org/10.1080/17460441.2019.1588883" title="DOI URL">https://doi.org/10.1080/17460441.2019.1588883</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1588883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1588883%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DBinding%252Bsite%252Bcomparisons%252Bfor%252Btarget-centered%252Bdrug%252Bdiscovery%26aulast%3DKonc%26aufirst%3DJanez%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D5%26spage%3D445%26epage%3D454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-Yves  Trosset</span>, <span class="hlFld-ContribAuthor ">Christian  Cavé</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Drug–Target Profiling. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 89-103. <a href="https://doi.org/10.1007/978-1-4939-9145-7_6" title="DOI URL">https://doi.org/10.1007/978-1-4939-9145-7_6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-9145-7_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-9145-7_6%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BDrug%2525E2%252580%252593Target%252BProfiling%26aulast%3DTrosset%26aufirst%3DJean-Yves%26date%3D2019%26date%3D2019%26spage%3D89%26epage%3D103%26pub%3DSpringer%2520New%2520York%26atitle%3DTarget%252BIdentification%252Band%252BValidation%252Bin%252BDrug%252BDiscovery%26aulast%3DMoll%26aufirst%3DJ%25C3%25BCrgen%26date%3D2019%26volume%3D1953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Obdulia  Rabal</span>, <span class="hlFld-ContribAuthor ">Andrea  Castellar</span>, <span class="hlFld-ContribAuthor ">Julen  Oyarzabal</span>. </span><span class="cited-content_cbyCitation_article-title">Novel pharmacological maps of protein lysine methyltransferases: key for target deorphanization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2018,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-018-0288-5" title="DOI URL">https://doi.org/10.1186/s13321-018-0288-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-018-0288-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-018-0288-5%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DNovel%252Bpharmacological%252Bmaps%252Bof%252Bprotein%252Blysine%252Bmethyltransferases%25253A%252Bkey%252Bfor%252Btarget%252Bdeorphanization%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2018%26date%3D2018%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christiane  Ehrt</span>, <span class="hlFld-ContribAuthor ">Tobias  Brinkjost</span>, <span class="hlFld-ContribAuthor ">Oliver  Koch</span>, . </span><span class="cited-content_cbyCitation_article-title">A benchmark driven guide to binding site comparison: An exhaustive evaluation using tailor-made data sets (ProSPECCTs). </span><span class="cited-content_cbyCitation_journal-name">PLOS Computational Biology</span><span> <strong>2018,</strong> <em>14 </em>
                                    (11)
                                     , e1006483. <a href="https://doi.org/10.1371/journal.pcbi.1006483" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1006483</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1006483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1006483%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Computational%2520Biology%26atitle%3DA%252Bbenchmark%252Bdriven%252Bguide%252Bto%252Bbinding%252Bsite%252Bcomparison%25253A%252BAn%252Bexhaustive%252Bevaluation%252Busing%252Btailor-made%252Bdata%252Bsets%252B%252528ProSPECCTs%252529%26aulast%3DEhrt%26aufirst%3DChristiane%26date%3D2018%26date%3D2018%26volume%3D14%26issue%3D11%26spage%3De1006483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cameron  Mura</span>, <span class="hlFld-ContribAuthor ">Eli J  Draizen</span>, <span class="hlFld-ContribAuthor ">Philip E  Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Structural biology meets data science: does anything change?. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2018,</strong> <em>52 </em>, 95-102. <a href="https://doi.org/10.1016/j.sbi.2018.09.003" title="DOI URL">https://doi.org/10.1016/j.sbi.2018.09.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2018.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2018.09.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DStructural%252Bbiology%252Bmeets%252Bdata%252Bscience%25253A%252Bdoes%252Banything%252Bchange%25253F%26aulast%3DMura%26aufirst%3DCameron%26date%3D2018%26volume%3D52%26spage%3D95%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Philip E.  Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Progress with covalent small-molecule kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2018,</strong> <em>23 </em>
                                    (3)
                                     , 727-735. <a href="https://doi.org/10.1016/j.drudis.2018.01.035" title="DOI URL">https://doi.org/10.1016/j.drudis.2018.01.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2018.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2018.01.035%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DProgress%252Bwith%252Bcovalent%252Bsmall-molecule%252Bkinase%252Binhibitors%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2018%26volume%3D23%26issue%3D3%26spage%3D727%26epage%3D735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elizabeth  Brunk</span>, <span class="hlFld-ContribAuthor ">Swagatika  Sahoo</span>, <span class="hlFld-ContribAuthor ">Daniel C  Zielinski</span>, <span class="hlFld-ContribAuthor ">Ali  Altunkaya</span>, <span class="hlFld-ContribAuthor ">Andreas  Dräger</span>, <span class="hlFld-ContribAuthor ">Nathan  Mih</span>, <span class="hlFld-ContribAuthor ">Francesco  Gatto</span>, <span class="hlFld-ContribAuthor ">Avlant  Nilsson</span>, <span class="hlFld-ContribAuthor ">German Andres  Preciat Gonzalez</span>, <span class="hlFld-ContribAuthor ">Maike Kathrin  Aurich</span>, <span class="hlFld-ContribAuthor ">Andreas  Prlić</span>, <span class="hlFld-ContribAuthor ">Anand  Sastry</span>, <span class="hlFld-ContribAuthor ">Anna D  Danielsdottir</span>, <span class="hlFld-ContribAuthor ">Almut  Heinken</span>, <span class="hlFld-ContribAuthor ">Alberto  Noronha</span>, <span class="hlFld-ContribAuthor ">Peter W  Rose</span>, <span class="hlFld-ContribAuthor ">Stephen K  Burley</span>, <span class="hlFld-ContribAuthor ">Ronan M T  Fleming</span>, <span class="hlFld-ContribAuthor ">Jens  Nielsen</span>, <span class="hlFld-ContribAuthor ">Ines  Thiele</span>, <span class="hlFld-ContribAuthor ">Bernhard O  Palsson</span>. </span><span class="cited-content_cbyCitation_article-title">Recon3D enables a three-dimensional view of gene variation in human metabolism. </span><span class="cited-content_cbyCitation_journal-name">Nature Biotechnology</span><span> <strong>2018,</strong> <em>36 </em>
                                    (3)
                                     , 272-281. <a href="https://doi.org/10.1038/nbt.4072" title="DOI URL">https://doi.org/10.1038/nbt.4072</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nbt.4072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnbt.4072%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Biotechnology%26atitle%3DRecon3D%252Benables%252Ba%252Bthree-dimensional%252Bview%252Bof%252Bgene%252Bvariation%252Bin%252Bhuman%252Bmetabolism%26aulast%3DBrunk%26aufirst%3DElizabeth%26date%3D2018%26date%3D2018%26volume%3D36%26issue%3D3%26spage%3D272%26epage%3D281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert  Roskoski</span>, <span class="hlFld-ContribAuthor ">Abdollah  Sadeghi-Nejad</span>. </span><span class="cited-content_cbyCitation_article-title">Role of RET protein-tyrosine kinase inhibitors in the treatment RET-driven thyroid and lung cancers. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Research</span><span> <strong>2018,</strong> <em>128 </em>, 1-17. <a href="https://doi.org/10.1016/j.phrs.2017.12.021" title="DOI URL">https://doi.org/10.1016/j.phrs.2017.12.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.phrs.2017.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.phrs.2017.12.021%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Research%26atitle%3DRole%252Bof%252BRET%252Bprotein-tyrosine%252Bkinase%252Binhibitors%252Bin%252Bthe%252Btreatment%252BRET-driven%252Bthyroid%252Band%252Blung%252Bcancers%26aulast%3DRoskoski%26aufirst%3DRobert%26date%3D2018%26volume%3D128%26spage%3D1%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ernest  Awoonor-Williams</span>, <span class="hlFld-ContribAuthor ">Andrew G.  Walsh</span>, <span class="hlFld-ContribAuthor ">Christopher N.  Rowley</span>. </span><span class="cited-content_cbyCitation_article-title">Modeling covalent-modifier drugs. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics</span><span> <strong>2017,</strong> <em>1865 </em>
                                    (11)
                                     , 1664-1675. <a href="https://doi.org/10.1016/j.bbapap.2017.05.009" title="DOI URL">https://doi.org/10.1016/j.bbapap.2017.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbapap.2017.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbapap.2017.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Proteins%2520and%2520Proteomics%26atitle%3DModeling%252Bcovalent-modifier%252Bdrugs%26aulast%3DAwoonor-Williams%26aufirst%3DErnest%26date%3D2017%26volume%3D1865%26issue%3D11%26spage%3D1664%26epage%3D1675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiao  Liu</span>, <span class="hlFld-ContribAuthor ">Chen  Chen</span>, <span class="hlFld-ContribAuthor ">Annie  Gao</span>, <span class="hlFld-ContribAuthor ">Hang Hang  Tong</span>, <span class="hlFld-ContribAuthor ">Lei  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">VariFunNet, an integrated multiscale modeling framework to study the effects of rare non-coding variants in genome-wide association studies: Applied to Alzheimer's disease. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 2177-2182. <a href="https://doi.org/10.1109/BIBM.2017.8217995" title="DOI URL">https://doi.org/10.1109/BIBM.2017.8217995</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1109/BIBM.2017.8217995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1109%2FBIBM.2017.8217995%26sid%3Dliteratum%253Aachs%26atitle%3DVariFunNet%25252C%252Ban%252Bintegrated%252Bmultiscale%252Bmodeling%252Bframework%252Bto%252Bstudy%252Bthe%252Beffects%252Bof%252Brare%252Bnon-coding%252Bvariants%252Bin%252Bgenome-wide%252Bassociation%252Bstudies%25253A%252BApplied%252Bto%252BAlzheimer%252527s%252Bdisease%26aulast%3DLiu%26aufirst%3DQiao%26date%3D2017%26spage%3D2177%26epage%3D2182%26pub%3DIEEE%26atitle%3D2017%252BIEEE%252BInternational%252BConference%252Bon%252BBioinformatics%252Band%252BBiomedicine%252B%252528BIBM%252529%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert  Roskoski</span>. </span><span class="cited-content_cbyCitation_article-title">ROS1 protein-tyrosine kinase inhibitors in the treatment of ROS1 fusion protein-driven non-small cell lung cancers. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Research</span><span> <strong>2017,</strong> <em>121 </em>, 202-212. <a href="https://doi.org/10.1016/j.phrs.2017.04.022" title="DOI URL">https://doi.org/10.1016/j.phrs.2017.04.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.phrs.2017.04.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.phrs.2017.04.022%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Research%26atitle%3DROS1%252Bprotein-tyrosine%252Bkinase%252Binhibitors%252Bin%252Bthe%252Btreatment%252Bof%252BROS1%252Bfusion%252Bprotein-driven%252Bnon-small%252Bcell%252Blung%252Bcancers%26aulast%3DRoskoski%26aufirst%3DRobert%26date%3D2017%26volume%3D121%26spage%3D202%26epage%3D212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Lei  Xie</span>, <span class="hlFld-ContribAuthor ">Philip E.  Bourne</span>, . </span><span class="cited-content_cbyCitation_article-title">Insights into the binding mode of MEK type-III inhibitors. A step towards discovering and designing allosteric kinase inhibitors across the human kinome. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2017,</strong> <em>12 </em>
                                    (6)
                                     , e0179936. <a href="https://doi.org/10.1371/journal.pone.0179936" title="DOI URL">https://doi.org/10.1371/journal.pone.0179936</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0179936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0179936%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DInsights%252Binto%252Bthe%252Bbinding%252Bmode%252Bof%252BMEK%252Btype-III%252Binhibitors.%252BA%252Bstep%252Btowards%252Bdiscovering%252Band%252Bdesigning%252Ballosteric%252Bkinase%252Binhibitors%252Bacross%252Bthe%252Bhuman%252Bkinome%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D6%26spage%3De0179936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sam  Tonddast-Navaei</span>, <span class="hlFld-ContribAuthor ">Bharath  Srinivasan</span>, <span class="hlFld-ContribAuthor ">Jeffrey  Skolnick</span>. </span><span class="cited-content_cbyCitation_article-title">On the importance of composite protein multiple ligand interactions in protein pockets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Chemistry</span><span> <strong>2017,</strong> <em>38 </em>
                                    (15)
                                     , 1252-1259. <a href="https://doi.org/10.1002/jcc.24523" title="DOI URL">https://doi.org/10.1002/jcc.24523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcc.24523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcc.24523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Chemistry%26atitle%3DOn%252Bthe%252Bimportance%252Bof%252Bcomposite%252Bprotein%252Bmultiple%252Bligand%252Binteractions%252Bin%252Bprotein%252Bpockets%26aulast%3DTonddast-Navaei%26aufirst%3DSam%26date%3D2017%26date%3D2016%26volume%3D38%26issue%3D15%26spage%3D1252%26epage%3D1259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoko  Koyama</span>, <span class="hlFld-ContribAuthor ">Noriyuki  Yamaotsu</span>, <span class="hlFld-ContribAuthor ">Izumi  Nakagome</span>, <span class="hlFld-ContribAuthor ">Shin-ichiro  Ozawa</span>, <span class="hlFld-ContribAuthor ">Tomoki  Yoshida</span>, <span class="hlFld-ContribAuthor ">Daichi  Hayakawa</span>, <span class="hlFld-ContribAuthor ">Shuichi  Hirono</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-step virtual screening to develop selective DYRK1A inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2017,</strong> <em>72 </em>, 229-239. <a href="https://doi.org/10.1016/j.jmgm.2017.01.014" title="DOI URL">https://doi.org/10.1016/j.jmgm.2017.01.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2017.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2017.01.014%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DMulti-step%252Bvirtual%252Bscreening%252Bto%252Bdevelop%252Bselective%252BDYRK1A%252Binhibitors%26aulast%3DKoyama%26aufirst%3DTomoko%26date%3D2017%26volume%3D72%26spage%3D229%26epage%3D239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">T  Cardozo</span>, <span class="hlFld-ContribAuthor ">E  Shmelkov</span>, <span class="hlFld-ContribAuthor ">K  Felsovalyi</span>, <span class="hlFld-ContribAuthor ">J  Swetnam</span>, <span class="hlFld-ContribAuthor ">T  Butler</span>, <span class="hlFld-ContribAuthor ">D  Malaspina</span>, <span class="hlFld-ContribAuthor ">S V  Shmelkov</span>. </span><span class="cited-content_cbyCitation_article-title">Chemistry-based molecular signature underlying the atypia of clozapine. </span><span class="cited-content_cbyCitation_journal-name">Translational Psychiatry</span><span> <strong>2017,</strong> <em>7 </em>
                                    (2)
                                     , e1036-e1036. <a href="https://doi.org/10.1038/tp.2017.6" title="DOI URL">https://doi.org/10.1038/tp.2017.6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/tp.2017.6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Ftp.2017.6%26sid%3Dliteratum%253Aachs%26jtitle%3DTranslational%2520Psychiatry%26atitle%3DChemistry-based%252Bmolecular%252Bsignature%252Bunderlying%252Bthe%252Batypia%252Bof%252Bclozapine%26aulast%3DCardozo%26aufirst%3DT%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D2%26spage%3De1036%26epage%3De1036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alpesh Ramanlal  Patel</span>, <span class="hlFld-ContribAuthor ">Ari  Hardianto</span>, <span class="hlFld-ContribAuthor ">Shoba  Ranganathan</span>, <span class="hlFld-ContribAuthor ">Fei  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Divergent response of homologous ATP sites to stereospecific ligand fluorination for selectivity enhancement. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2017,</strong> <em>15 </em>
                                    (7)
                                     , 1570-1574. <a href="https://doi.org/10.1039/C7OB00129K" title="DOI URL">https://doi.org/10.1039/C7OB00129K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7OB00129K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7OB00129K%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DDivergent%252Bresponse%252Bof%252Bhomologous%252BATP%252Bsites%252Bto%252Bstereospecific%252Bligand%252Bfluorination%252Bfor%252Bselectivity%252Benhancement%26aulast%3DPatel%26aufirst%3DAlpesh%2BRamanlal%26date%3D2017%26date%3D2017%26volume%3D15%26issue%3D7%26spage%3D1570%26epage%3D1574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Flowchart describing the Fs-IFP method. (a) The protein kinases that have released apo and holo structures of catalytic domains are marked with blue and red filled circles, respectively, in the evolutionary tree.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> (b) PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYU">3BYU</a>, which is used as a reference structure for all kinases. (c). Superimposed binding site. Each row represents a sequence-order-independent motif comprising the binding pocket for a given ligand-bound structure. Each column represents the corresponding amino acid residues at the same spatial position of the binding pocket for all ligand-bound structures. The D-NULL mark means there is no corresponding residue at the specific position of the given ligand-bound structures. (d) Encoded binding site using the Fs-IFP approach. Each row represents the Fs-IFP bit-strings for one given ligand-bound structure. Each column including 7 bits represents an Fs-IFP code of one residue of the binding site by using seven types of interactions. The same column means that the residue matches the same position in 2383 superposed binding pockets. A bar is used if there is no typical interaction between the specific amino acid and the ligand in the given ligand-bound structure. The bit is marked as “1” if the specific type of interaction exists, otherwise marked as “0” if the type of interaction does not exist between the specific amino acid and the ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Distribution of the ligand characteristics in 2383 cocrystallized kinase structures: (a) distribution of the top 20 frequently occurring ligands; (b) distribution of the molecular weight of the ligands in all cocrystallized kinase structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Distribution of Tanimoto coefficient (TC) value for the fingerprints similarity of the ligands (ligand-FP) found in HPK-SD (red), kinase pockets (kinase-FP) of the HPK-SD (blue), and the kinase–ligand interaction fingerprints (Fs-IFP) of the kinase–ligand complexes (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Correlation of the Tanimoto coefficient (TC) for ligand FP, kinase-pocket FP, and kinase–ligand IFP similarity using a violin plot: (a, b) correlation of the TC of the ligand FP similarity and the kinase pocket FP similarity; (c, e) correlation of the TC of the ligand FP similarity and the kinase–ligand IFP similarity; (d, f) correlation of the TC of the kinase pocket FP similarity and the kinase–ligand IFP similarity. In all panels, the width of the violin box is proportional to the number of TCs. The black bars range from 25% to 75%, and the white points on the bars represent the medians. Distributions (%) of ligand-FP, kinase-FP, and Fs-IFP in the corresponding bin of similarity are shown at the upper portion of every panel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Distribution of interaction types at the kinase–ligand binding interface: (a) distribution of the contribution of six interaction types at the kinase–ligand binding interface; (b) distribution of occurrences of the six interaction types for each residue location at the binding interface. The <i>y</i>-axis is the number of interaction occurrences for the corresponding amino acids given along the <i>x</i>-axis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Correlation between interactions and the amino acid conservation at each amino acid position constituting the binding interface: (a) distribution of interaction occurrence for each amino acid position; (b) the corresponding distribution in (a) mapped to the template structure; (c) conservation of amino acids at the binding site; (d) correlation between residue conservation and the distribution of interactions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Rectangular tree layout of the Fs-IFP data set for HPK-SD. Five distinct clusters were found. Cluster5 was further divided into 5 subclusters: cluster 5a–d.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Binding modes for each cluster 1–4 and 5a–d based on complete linkage clustering: (a–h) for cluster 1–4 and 5a-d using PDB entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ITJ">4ITJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A5U">2A5U</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PXF">3PXF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H8F">1H8F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PRZ">3PRZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BYG">1BYG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ZZ2">1ZZ2</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q4C">3Q4C</a> as the representative structure, respectively; (i) site locations for the binding regions for the eight clusters.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/medium/jm-2015-02041k_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Correlation between Fs-IFP and experimentally determined kinase binding profiles. The curves were obtained by using fit first-order exponential decay. (a) Correlation of the pairwise difference of the <i>K</i><sub>d</sub> values and pairwise similarity of Fs-IFPs. (b) Correlation of the pairwise difference of the PoC values and pairwise similarity of Fs-IFPs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02041/20160506/images/large/jm-2015-02041k_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02041&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71617" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71617" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 103 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">–</span> <span class="NLM_lpage">1934</span><span class="refDoi"> DOI: 10.1126/science.1075762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome&doi=10.1126%2Fscience.1075762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lh9T_l8wi0V9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934%26doi%3D10.1126%2Fscience.1075762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span><span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lgcKxmU2WwQ0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Ward, R. A.; Goldberg, F.</span> <span class="citation_source-book">Kinase Drug Discovery</span>; <span class="NLM_publisher-name">RSC Publishing</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=R.+A.+Ward&author=F.+Goldberg&title=Kinase+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26btitle%3DKinase%2520Drug%2520Discovery%26pub%3DRSC%2520Publishing%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Klebl, B.; Müller, G.; Hamacher, M.</span> <span class="citation_source-book">Protein Kinase as Drug Targets</span>; <span class="NLM_publisher-name">Wiley-VCH Verlag GambH & Co. KGaA</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">49</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1002%2F9783527633470" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=B.+Klebl&author=G.+M%C3%BCller&author=M.+Hamacher&title=Protein+Kinase+as+Drug+Targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2F9783527633470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527633470%26sid%3Dliteratum%253Aachs%26aulast%3DKlebl%26aufirst%3DB.%26btitle%3DProtein%2520Kinase%2520as%2520Drug%2520Targets%26pub%3DWiley-VCH%2520Verlag%2520GambH%2520%2526%2520Co.%2520KGaA%26date%3D2011%26volume%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Goldman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melo, J. V.</span><span> </span><span class="NLM_article-title">Chronic myeloid leukemia--advances in biology and new approaches to treatment</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">349</span><span class="NLM_x">, </span> <span class="NLM_fpage">1451</span><span class="NLM_x">–</span> <span class="NLM_lpage">1464</span><span class="refDoi"> DOI: 10.1056/NEJMra020777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1056%2FNEJMra020777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=14534339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotVGgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=349&publication_year=2003&pages=1451-1464&author=J.+M.+Goldmanauthor=J.+V.+Melo&title=Chronic+myeloid+leukemia%2D%2Dadvances+in+biology+and+new+approaches+to+treatment&doi=10.1056%2FNEJMra020777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chronic myeloid leukemia - Advances in biology and new approaches to treatment</span></div><div class="casAuthors">Goldman, John M.; Melo, Junia V.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">349</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1451-1464</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review presents recent advances in knowledge related to the pathogenesis of chronic myeloid leukemia (CML) and indicates where these may have important therapeutic implications.  It discusses the cytokinetics, cytogenetics, mol. events, graft-vs.-leukemia effect and treatment options of CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXmKTzKKnN9bVg90H21EOLACvtfcHk0lgcKxmU2WwQ0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotVGgsLc%253D&md5=88ae3d2434d003dca02f0e4af5bfff8d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1056%2FNEJMra020777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra020777%26sid%3Dliteratum%253Aachs%26aulast%3DGoldman%26aufirst%3DJ.%2BM.%26aulast%3DMelo%26aufirst%3DJ.%2BV.%26atitle%3DChronic%2520myeloid%2520leukemia--advances%2520in%2520biology%2520and%2520new%2520approaches%2520to%2520treatment%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2003%26volume%3D349%26spage%3D1451%26epage%3D1464%26doi%3D10.1056%2FNEJMra020777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The (un)targeted cancer kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">166</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span><span class="refDoi"> DOI: 10.1038/nchembio.297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnchembio.297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=20154661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=166-169&author=O.+Fedorovauthor=S.+Mullerauthor=S.+Knapp&title=The+%28un%29targeted+cancer+kinome&doi=10.1038%2Fnchembio.297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The (un)targeted cancer kinome</span></div><div class="casAuthors">Fedorov, Oleg; Mueller, Susanne; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">166-169</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review and discussion.  The complexity of cancer signaling and the resulting difficulties in target selection have strongly biased kinase drug discovery towards clin. validated targets.  Recently, novel kinase targets that are uncharacterized have emerged from genome sequencing and RNAi studies.  Chem. probes are urgently needed to functionally annotate these kinases and to stimulate new drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwcnq0-qjPirVg90H21EOLACvtfcHk0lhxEWKzAh8Whg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D&md5=198ae8d39d40247e17aef5e9de0f2a99</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.297%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520%2528un%2529targeted%2520cancer%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D166%26epage%3D169%26doi%3D10.1038%2Fnchembio.297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Sarnpitak, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mujumdar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coster, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorse, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krasavin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofmann, A.</span><span> </span><span class="NLM_article-title">Panel docking of small-molecule libraries - Prospects to improve efficiency of lead compound discovery</span> <span class="citation_source-journal">Biotechnol. Adv.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">947</span><span class="refDoi"> DOI: 10.1016/j.biotechadv.2015.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.biotechadv.2015.05.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=941-947&author=P.+Sarnpitakauthor=P.+Mujumdarauthor=P.+Taylorauthor=M.+Crossauthor=M.+J.+Costerauthor=A.+D.+Gorseauthor=M.+Krasavinauthor=A.+Hofmann&title=Panel+docking+of+small-molecule+libraries+-+Prospects+to+improve+efficiency+of+lead+compound+discovery&doi=10.1016%2Fj.biotechadv.2015.05.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2015.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2015.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DSarnpitak%26aufirst%3DP.%26aulast%3DMujumdar%26aufirst%3DP.%26aulast%3DTaylor%26aufirst%3DP.%26aulast%3DCross%26aufirst%3DM.%26aulast%3DCoster%26aufirst%3DM.%2BJ.%26aulast%3DGorse%26aufirst%3DA.%2BD.%26aulast%3DKrasavin%26aufirst%3DM.%26aulast%3DHofmann%26aufirst%3DA.%26atitle%3DPanel%2520docking%2520of%2520small-molecule%2520libraries%2520-%2520Prospects%2520to%2520improve%2520efficiency%2520of%2520lead%2520compound%2520discovery%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2015%26volume%3D33%26spage%3D941%26epage%3D947%26doi%3D10.1016%2Fj.biotechadv.2015.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausen, M. H.</span><span> </span><span class="NLM_article-title">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1016/j.drudis.2015.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.drudis.2015.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=26210956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=5-10&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=Small-molecule+kinase+inhibitors%3A+an+analysis+of+FDA-approved+drugs&doi=10.1016%2Fj.drudis.2015.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors (SMKIs), 28 of which are approved by the US Food and Drug Administration (FDA), have been actively pursued as promising targeted therapeutics.  Here, we assess the key structural and physicochem. properties, target selectivity and mechanism of function, and therapeutic indications of these approved inhibitors.  Our anal. showed that >30% of approved SMKIs have a mol. wt. (MW) exceeding 500 and all have a total ring count of between three and five.  The assumption that type II inhibitors tend to be more selective than type I inhibitors has been proved to be unreliable.  Although previous SMKI research was concd. on tyrosine kinase inhibitors for cancer treatment, recent progress indicates diversification of SMKI research in terms of new targets, mechanistic types, and therapeutic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZrY68ZOI2xLVg90H21EOLACvtfcHk0lhxEWKzAh8Whg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P&md5=228524fa464393053ecbfac19e994929</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DSmall-molecule%2520kinase%2520inhibitors%253A%2520an%2520analysis%2520of%2520FDA-approved%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.drudis.2015.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The ins and outs of selective kinase inhibitor development</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">–</span> <span class="NLM_lpage">821</span><span class="refDoi"> DOI: 10.1038/nchembio.1938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnchembio.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=26485069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=818-821&author=S.+Mullerauthor=A.+Chaikuadauthor=N.+S.+Grayauthor=S.+Knapp&title=The+ins+and+outs+of+selective+kinase+inhibitor+development&doi=10.1038%2Fnchembio.1938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The ins and outs of selective kinase inhibitor development</span></div><div class="casAuthors">Muller, Susanne; Chaikuad, Apirat; Gray, Nathanael S.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">818-821</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein kinases have emerged as one of the most successful families of drug targets.  To date, most selective kinase inhibitors have been discovered serendipitously either through broad selectivity screening or through the discovery of unique binding modes.  Here the authors discuss design strategies that could lead to a broader coverage of the kinome with selective inhibitors and to a more rational approach for developing them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQzaMrxSYx_rVg90H21EOLACvtfcHk0lhxEWKzAh8Whg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF&md5=f5adf0bbaa388c87b26bae23cfad0bae</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1938%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520ins%2520and%2520outs%2520of%2520selective%2520kinase%2520inhibitor%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D818%26epage%3D821%26doi%3D10.1038%2Fnchembio.1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Jacoby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tresadern, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bembenek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wroblowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buyck, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neefs, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rassokhin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poncelet, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Vlijmen, H.</span><span> </span><span class="NLM_article-title">Extending kinome coverage by analysis of kinase inhibitor broad profiling data</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">652</span><span class="NLM_x">–</span> <span class="NLM_lpage">658</span><span class="refDoi"> DOI: 10.1016/j.drudis.2015.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.drudis.2015.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25596550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFKgsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=652-658&author=E.+Jacobyauthor=G.+Tresadernauthor=S.+Bembenekauthor=B.+Wroblowskiauthor=C.+Buyckauthor=J.+M.+Neefsauthor=D.+Rassokhinauthor=A.+Ponceletauthor=J.+Huntauthor=H.+van+Vlijmen&title=Extending+kinome+coverage+by+analysis+of+kinase+inhibitor+broad+profiling+data&doi=10.1016%2Fj.drudis.2015.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Extending kinome coverage by analysis of kinase inhibitor broad profiling data</span></div><div class="casAuthors">Jacoby, Edgar; Tresadern, Gary; Bembenek, Scott; Wroblowski, Berthold; Buyck, Christophe; Neefs, Jean-Marc; Rassokhin, Dmitrii; Poncelet, Alain; Hunt, Jeremy; van Vlijmen, Herman</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">652-658</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The explored kinome was extended with broad profiling using the DiscoveRx and Millipore assay panels.  The anal. of the profiling of 3368 selected inhibitors on 456 kinases in the DiscoveRx format delivered several insights.  First, the coverage depended on the threshold of the selectivity parameter.  Second, the relation between hit confirmation rates and inhibitor selectivity showed unexpectedly that higher selectivity can increase the likelihood of false positives.  Third, comparing the coverage of a focused to that of a random library showed that the design based on a max. no. of scaffolds was superior to a limited no. of scaffolds.  Therefore, selective compds. can be used in target validation, enable the jumpstarting of new kinase drug discovery projects, and chart new biol. space via phenotypic screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3cxoYeXbpxbVg90H21EOLACvtfcHk0liszQ_n18-aCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKgsL8%253D&md5=b72bec6134c3eb716615f68ece233575</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DBembenek%26aufirst%3DS.%26aulast%3DWroblowski%26aufirst%3DB.%26aulast%3DBuyck%26aufirst%3DC.%26aulast%3DNeefs%26aufirst%3DJ.%2BM.%26aulast%3DRassokhin%26aufirst%3DD.%26aulast%3DPoncelet%26aufirst%3DA.%26aulast%3DHunt%26aufirst%3DJ.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%26atitle%3DExtending%2520kinome%2520coverage%2520by%2520analysis%2520of%2520kinase%2520inhibitor%2520broad%2520profiling%2520data%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D652%26epage%3D658%26doi%3D10.1016%2Fj.drudis.2015.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span> </span><span class="NLM_article-title">The Protein Data Bank and the challenge of structural genomics</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl</span><span class="NLM_x">) </span> <span class="NLM_fpage">957</span><span class="NLM_x">–</span> <span class="NLM_lpage">959</span><span class="refDoi"> DOI: 10.1038/80734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2F80734" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2000&pages=957-959&issue=Suppl&author=H.+M.+Bermanauthor=T.+N.+Bhatauthor=P.+E.+Bourneauthor=Z.+Fengauthor=G.+Gillilandauthor=H.+Weissigauthor=J.+Westbrook&title=The+Protein+Data+Bank+and+the+challenge+of+structural+genomics&doi=10.1038%2F80734"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2F80734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F80734%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DWestbrook%26aufirst%3DJ.%26atitle%3DThe%2520Protein%2520Data%2520Bank%2520and%2520the%2520challenge%2520of%2520structural%2520genomics%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2000%26volume%3D7%26issue%3DSuppl%26spage%3D957%26epage%3D959%26doi%3D10.1038%2F80734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span> </span><span class="NLM_article-title">Binding site similarity analysis for the functional classification of the protein kinase family</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">318</span><span class="NLM_x">–</span> <span class="NLM_lpage">329</span><span class="refDoi"> DOI: 10.1021/ci800289y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800289y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=318-329&author=S.+L.+Kinningsauthor=R.+M.+Jackson&title=Binding+site+similarity+analysis+for+the+functional+classification+of+the+protein+kinase+family&doi=10.1021%2Fci800289y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Similarity Analysis for the Functional Classification of the Protein Kinase Family</span></div><div class="casAuthors">Kinnings, Sarah L.; Jackson, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">318-329</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methods for analyzing complete gene families are becoming of increasing importance to the drug discovery process, because similarities and differences within a family are often the key to understanding functional differences that can be exploited in drug design.  We undertake a large-scale structural comparison of protein kinase ATP-binding sites using a geometric hashing method.  Subsequently, we propose a relevant classification of the protein kinase family based on the structural similarity of its binding sites.  Our classification is not only able to reveal the great diversity of different protein kinases and therefore their different potential for inhibitor selectivity, but it is also able to distinguish subtle differences within binding site conformation reflecting the protein activation state.  Furthermore, using exptl. inhibition profiling, we demonstrate that our classification can be used to identify protein kinase binding sites that are known exptl. to bind the same drug, demonstrating that it has potential as an inverse (protein) virtual screening tool, by identifying which other sites have the potential to bind a given drug.  In this way the cross-reactivities of the anticancer drugs Tarceva and Gleevec are rationalized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6kpf80r8s5LVg90H21EOLACvtfcHk0liszQ_n18-aCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltF2qsg%253D%253D&md5=742343601ca1e3c0549fa4f383c2566b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci800289y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800289y%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DBinding%2520site%2520similarity%2520analysis%2520for%2520the%2520functional%2520classification%2520of%2520the%2520protein%2520kinase%2520family%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D318%26epage%3D329%26doi%3D10.1021%2Fci800289y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">McGregor, M. J.</span><span> </span><span class="NLM_article-title">A pharmacophore map of small molecule protein kinase inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2374</span><span class="NLM_x">–</span> <span class="NLM_lpage">2382</span><span class="refDoi"> DOI: 10.1021/ci700244t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700244t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKksL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=2374-2382&author=M.+J.+McGregor&title=A+pharmacophore+map+of+small+molecule+protein+kinase+inhibitors&doi=10.1021%2Fci700244t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A Pharmacophore Map of Small Molecule Protein Kinase Inhibitors</span></div><div class="casAuthors">McGregor, Malcolm J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2374-2382</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases have emerged as one of the major drug target classes that are amenable to the development of small mol. inhibitors.  They share a conserved structural similarity in the region of the ATP binding site, where most inhibitors interact.  Using a pharmacophore approach, we have explored the features of protein-ligand interactions for a set of 220 kinase crystal structures from the Protein Data Bank (PDB).  The resulting "pharmacophore map" shows the interactions made by all ligands with their receptors simultaneously.  This gives insight that has been applied in the design of kinase screening sets and combinatorial libraries.  An algorithm is described that scores small mol. structures for goodness of fit to the resulting binding site description obtained from the anal.  The algorithm identifies a pose that is close to the crystal structure pose for the majority of ligands, with only the 2D chem. structure as input and no knowledge of the crystal structure from which it was derived.  Application of the algorithm to a test screening set gave a useful enrichment of active compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-PlOGvAO_Z7Vg90H21EOLACvtfcHk0liszQ_n18-aCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKksL%252FO&md5=38d2b55c3fbb1232fd7d091f95414633</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fci700244t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700244t%26sid%3Dliteratum%253Aachs%26aulast%3DMcGregor%26aufirst%3DM.%2BJ.%26atitle%3DA%2520pharmacophore%2520map%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D2374%26epage%3D2382%26doi%3D10.1021%2Fci700244t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Huse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">The conformational plasticity of protein kinases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">275</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span><span class="refDoi"> DOI: 10.1016/S0092-8674(02)00741-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2FS0092-8674%2802%2900741-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=12015977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2002&pages=275-282&author=M.+Huseauthor=J.+Kuriyan&title=The+conformational+plasticity+of+protein+kinases&doi=10.1016%2FS0092-8674%2802%2900741-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The conformational plasticity of protein kinases</span></div><div class="casAuthors">Huse, Morgan; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">275-282</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with 48 refs.  Protein kinases operate in a large no. of distinct signaling pathways, where the tight regulation of their catalytic activity is crucial to the development and maintenance of eukaryotic organisms.  The catalytic domains of different kinases adopt strikingly similar structures when they are active.  By contrast, crystal structures of inactive kinases have revealed a marked plasticity in the kinase domain that allows the adoption of distinct conformations in response to interactions with specific regulatory domains or proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJrn7F3h4ej7Vg90H21EOLACvtfcHk0lj7TdM2SeziSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvV2lsLw%253D&md5=cd56edae94ad239f2ce7a5d6c8930cbb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900741-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900741-9%26sid%3Dliteratum%253Aachs%26aulast%3DHuse%26aufirst%3DM.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DThe%2520conformational%2520plasticity%2520of%2520protein%2520kinases%26jtitle%3DCell%26date%3D2002%26volume%3D109%26spage%3D275%26epage%3D282%26doi%3D10.1016%2FS0092-8674%2802%2900741-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Exploration of type II binding mode: a privileged approach for kinase inhibitor focused drug discovery?</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1230</span><span class="NLM_x">–</span> <span class="NLM_lpage">1241</span><span class="refDoi"> DOI: 10.1021/cb500129t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+a+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F&doi=10.1021%2Fcb500129t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Type II Binding Mode: A Privileged Approach for Kinase Inhibitor Focused Drug Discovery?</span></div><div class="casAuthors">Zhao, Zheng; Wu, Hong; Wang, Li; Liu, Yi; Knapp, Stefan; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1230-1241</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ATP site of kinases displays remarkable conformational flexibility when accommodating chem. diverse small mol. inhibitors.  The so-called activation segment, whose conformation controls catalytic activity and access to the substrate binding pocket, can undergo a large conformational change with the active state assuming a 'DFG-in' and an inactive state assuming a 'DFG-out' conformation.  Compds. that preferentially bind to the DFG-out conformation are typically called 'type II' inhibitors in contrast to 'type I' inhibitors that bind to the DFG-in conformation.  This review surveys the large no. of type II inhibitors that have been developed and provides an anal. of their crystallog. detd. binding modes.  Using a small library of type II inhibitors, we demonstrate that more than 200 kinases can be targeted, suggesting that type II inhibitors may not be intrinsically more selective than type I inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4jYt6Db3Ap7Vg90H21EOLACvtfcHk0lj7TdM2SeziSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D&md5=95bf185ff7f973eda5a65f4096bfaa82</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520a%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241%26doi%3D10.1021%2Fcb500129t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Furtmann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of three-dimensional activity cliffs formed by kinase inhibitors with different binding modes and cliff mapping of structural analogues</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">252</span><span class="NLM_x">–</span> <span class="NLM_lpage">264</span><span class="refDoi"> DOI: 10.1021/jm5009264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5009264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=252-264&author=N.+Furtmannauthor=Y.+Huauthor=J.+Bajorath&title=Comprehensive+analysis+of+three-dimensional+activity+cliffs+formed+by+kinase+inhibitors+with+different+binding+modes+and+cliff+mapping+of+structural+analogues&doi=10.1021%2Fjm5009264"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm5009264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5009264%26sid%3Dliteratum%253Aachs%26aulast%3DFurtmann%26aufirst%3DN.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComprehensive%2520analysis%2520of%2520three-dimensional%2520activity%2520cliffs%2520formed%2520by%2520kinase%2520inhibitors%2520with%2520different%2520binding%2520modes%2520and%2520cliff%2520mapping%2520of%2520structural%2520analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D252%26epage%3D264%26doi%3D10.1021%2Fjm5009264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0lj7TdM2SeziSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Liao, J. J.</span><span> </span><span class="NLM_article-title">Molecular recognition of protein kinase binding pockets for design of potent and selective kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">409</span><span class="NLM_x">–</span> <span class="NLM_lpage">424</span><span class="refDoi"> DOI: 10.1021/jm0608107</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0608107" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1Cmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=409-424&author=J.+J.+Liao&title=Molecular+recognition+of+protein+kinase+binding+pockets+for+design+of+potent+and+selective+kinase+inhibitors&doi=10.1021%2Fjm0608107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Recognition of Protein Kinase Binding Pockets for Design of Potent and Selective Kinase Inhibitors</span></div><div class="casAuthors">Liao, Jeffrey Jie-Lou</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">409-424</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Many kinase inhibitors have failed in preclin. or clin. development because of the lack of such selectivity that induces intolerable side effects, mostly because the catalytic cleft is highly conserved in sequence and conformation.  Systematic anal. of the crystal structures of protein kinases can offer insight into the design of highly selective kinase inhibitors to overcome these effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroDTl2SBUfbbVg90H21EOLACvtfcHk0lgPlSdGCPjCZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1Cmsg%253D%253D&md5=77542d3ba589bd834a0d2b97a24eea05</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm0608107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608107%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DJ.%2BJ.%26atitle%3DMolecular%2520recognition%2520of%2520protein%2520kinase%2520binding%2520pockets%2520for%2520design%2520of%2520potent%2520and%2520selective%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D409%26epage%3D424%26doi%3D10.1021%2Fjm0608107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafleur, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevado, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Kinase selectivity potential for inhibitors targeting the ATP binding site: a network analysis</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">198</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btp650</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1093%2Fbioinformatics%2Fbtp650" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19942586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlGkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=198-204&author=D.+Huangauthor=T.+Zhouauthor=K.+Lafleurauthor=C.+Nevadoauthor=A.+Caflisch&title=Kinase+selectivity+potential+for+inhibitors+targeting+the+ATP+binding+site%3A+a+network+analysis&doi=10.1093%2Fbioinformatics%2Fbtp650"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase selectivity potential for inhibitors targeting the ATP binding site: a network analysis</span></div><div class="casAuthors">Huang, Danzhi; Zhou, Ting; Lafleur, Karine; Nevado, Cristina; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">198-204</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Small-mol. inhibitors targeting the ATP binding pocket of the catalytic domain of protein kinases have potential to become drugs devoid of (major) side effects, particularly if they bind selectively.  Here, the sequences of the 518 human kinases are first mapped onto the structural alignment of 116 kinases of known three-dimensional structure.  The multiple structure alignment is then used to encode the known strategies for developing selective inhibitors into a fingerprint.  Finally, a network anal. is used to partition the kinases into clusters according to similarity of their fingerprints, i.e., physico-chem. characteristics of the residues responsible for selective binding.  For each kinase the network anal. reveals the likelihood to find selective inhibitors targeting the ATP binding site.  Systematic guidelines are proposed to develop selective inhibitors.  Importantly, the network anal. suggests that the tyrosine kinase EphB4 has high selectivity potential, which is consistent with the selectivity profile of two novel EphB4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQVQsrpqB_ILVg90H21EOLACvtfcHk0lgPlSdGCPjCZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlGkug%253D%253D&md5=b6c63ccd4654219a56752f9cfc1356a3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp650&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp650%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DLafleur%26aufirst%3DK.%26aulast%3DNevado%26aufirst%3DC.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DKinase%2520selectivity%2520potential%2520for%2520inhibitors%2520targeting%2520the%2520ATP%2520binding%2520site%253A%2520a%2520network%2520analysis%26jtitle%3DBioinformatics%26date%3D2010%26volume%3D26%26spage%3D198%26epage%3D204%26doi%3D10.1093%2Fbioinformatics%2Fbtp650" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Cui, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran-Dube, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nambu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pairish, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Funk, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrous, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McTigue, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grodsky, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padrique, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alton, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timofeevski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mroczkowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kania, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, M. P.</span><span> </span><span class="NLM_article-title">Structure based drug design of crizotinib (PF-02341066), a potent and selective dual inhibitor of mesenchymal-epithelial transition factor (c-MET) kinase and anaplastic lymphoma kinase (ALK)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6342</span><span class="NLM_x">–</span> <span class="NLM_lpage">6363</span><span class="refDoi"> DOI: 10.1021/jm2007613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2007613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVegtLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6342-6363&author=J.+J.+Cuiauthor=M.+Tran-Dubeauthor=H.+Shenauthor=M.+Nambuauthor=P.+P.+Kungauthor=M.+Pairishauthor=L.+Jiaauthor=J.+Mengauthor=L.+Funkauthor=I.+Botrousauthor=M.+McTigueauthor=N.+Grodskyauthor=K.+Ryanauthor=E.+Padriqueauthor=G.+Altonauthor=S.+Timofeevskiauthor=S.+Yamazakiauthor=Q.+Liauthor=H.+Zouauthor=J.+Christensenauthor=B.+Mroczkowskiauthor=S.+Benderauthor=R.+S.+Kaniaauthor=M.+P.+Edwards&title=Structure+based+drug+design+of+crizotinib+%28PF-02341066%29%2C+a+potent+and+selective+dual+inhibitor+of+mesenchymal-epithelial+transition+factor+%28c-MET%29+kinase+and+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1021%2Fjm2007613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Based Drug Design of Crizotinib (PF-02341066), a Potent and Selective Dual Inhibitor of Mesenchymal-Epithelial Transition Factor (c-MET) Kinase and Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Cui, J. Jean; Tran-Dube, Michelle; Shen, Hong; Nambu, Mitchell; Kung, Pei-Pei; Pairish, Mason; Jia, Lei; Meng, Jerry; Funk, Lee; Botrous, Iriny; McTigue, Michele; Grodsky, Neil; Ryan, Kevin; Padrique, Ellen; Alton, Gordon; Timofeevski, Sergei; Yamazaki, Shinji; Li, Qiuhua; Zou, Helen; Christensen, James; Mroczkowski, Barbara; Bender, Steve; Kania, Robert S.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6342-6363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Because of the crit. roles of aberrant signaling in cancer, both c-MET and ALK receptor tyrosine kinases are attractive oncol. targets for therapeutic intervention.  The cocrystal structure of 3 (PHA-665752), bound to c-MET kinase domain, revealed a novel ATP site environment, which served as the target to guide parallel, multiattribute drug design.  A novel 2-amino-5-aryl-3-benzyloxypyridine series was created to more effectively make the key interactions achieved with 3.  In the novel series, the 2-aminopyridine core allowed a 3-benzyloxy group to reach into the same pocket as the 2,6-dichlorophenyl group of 3 via a more direct vector and thus with a better ligand efficiency (LE).  Further optimization of the lead series generated the clin. candidate crizotinib (PF-02341066), which demonstrated potent in vitro and in vivo c-MET kinase and ALK inhibition, effective tumor growth inhibition, and good pharmaceutical properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHs_cKJ-U3t7Vg90H21EOLACvtfcHk0lgPlSdGCPjCZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVegtLrN&md5=dbab4c1b3e7c76180dbad04be2b6a474</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm2007613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2007613%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26aulast%3DTran-Dube%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DNambu%26aufirst%3DM.%26aulast%3DKung%26aufirst%3DP.%2BP.%26aulast%3DPairish%26aufirst%3DM.%26aulast%3DJia%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DJ.%26aulast%3DFunk%26aufirst%3DL.%26aulast%3DBotrous%26aufirst%3DI.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DGrodsky%26aufirst%3DN.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DPadrique%26aufirst%3DE.%26aulast%3DAlton%26aufirst%3DG.%26aulast%3DTimofeevski%26aufirst%3DS.%26aulast%3DYamazaki%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DMroczkowski%26aufirst%3DB.%26aulast%3DBender%26aufirst%3DS.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DStructure%2520based%2520drug%2520design%2520of%2520crizotinib%2520%2528PF-02341066%2529%252C%2520a%2520potent%2520and%2520selective%2520dual%2520inhibitor%2520of%2520mesenchymal-epithelial%2520transition%2520factor%2520%2528c-MET%2529%2520kinase%2520and%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6342%26epage%3D6363%26doi%3D10.1021%2Fjm2007613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lg7PUgdsj2xzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Chuaqui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span> </span><span class="NLM_article-title">Interaction profiles of protein kinase-inhibitor complexes and their application to virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">121</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1021/jm049312t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049312t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=121-133&author=C.+Chuaquiauthor=Z.+Dengauthor=J.+Singh&title=Interaction+profiles+of+protein+kinase-inhibitor+complexes+and+their+application+to+virtual+screening&doi=10.1021%2Fjm049312t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm049312t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049312t%26sid%3Dliteratum%253Aachs%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DInteraction%2520profiles%2520of%2520protein%2520kinase-inhibitor%2520complexes%2520and%2520their%2520application%2520to%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D121%26epage%3D133%26doi%3D10.1021%2Fjm049312t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Patricelli, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nomanbhoy, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagannathan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aban, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okerberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herring, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozarich, J. W.</span><span> </span><span class="NLM_article-title">In situ kinase profiling reveals functionally relevant properties of native kinases</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">699</span><span class="NLM_x">–</span> <span class="NLM_lpage">710</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2011.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.chembiol.2011.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=21700206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotVejtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=699-710&author=M.+P.+Patricelliauthor=T.+K.+Nomanbhoyauthor=J.+Wuauthor=H.+Brownauthor=D.+Zhouauthor=J.+Zhangauthor=S.+Jagannathanauthor=A.+Abanauthor=E.+Okerbergauthor=C.+Herringauthor=B.+Nordinauthor=H.+Weissigauthor=Q.+Yangauthor=J.+D.+Leeauthor=N.+S.+Grayauthor=J.+W.+Kozarich&title=In+situ+kinase+profiling+reveals+functionally+relevant+properties+of+native+kinases&doi=10.1016%2Fj.chembiol.2011.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">In Situ Kinase Profiling Reveals Functionally Relevant Properties of Native Kinases</span></div><div class="casAuthors">Patricelli, Matthew P.; Nomanbhoy, Tyzoon K.; Wu, Jiangyue; Brown, Heidi; Zhou, David; Zhang, Jianming; Jagannathan, Subadhra; Aban, Arwin; Okerberg, Eric; Herring, Chris; Nordin, Brian; Weissig, Helge; Yang, Qingkai; Lee, Jiing-Dwan; Gray, Nathanael S.; Kozarich, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">699-710</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Protein kinases are intensely studied mediators of cellular signaling, yet important questions remain regarding their regulation and in vivo properties.  Here, we use a probe-based chemoprotemics platform to profile several well studied kinase inhibitors against >200 kinases in native cell proteomes and reveal biol. targets for some of these inhibitors.  Several striking differences were identified between native and recombinant kinase inhibitory profiles, in particular, for the Raf kinases.  The native kinase binding profiles presented here closely mirror the cellular activity of these inhibitors, even when the inhibition profiles differ dramatically from recombinant assay results.  Addnl., Raf activation events could be detected on live cell treatment with inhibitors.  These studies highlight the complexities of protein kinase behavior in the cellular context and demonstrate that profiling with only recombinant/purified enzymes can be misleading.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopb21npuaC_rVg90H21EOLACvtfcHk0lg7PUgdsj2xzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotVejtbo%253D&md5=dda091b895850c95131f80c3aba9e8c7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DNomanbhoy%26aufirst%3DT.%2BK.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DJagannathan%26aufirst%3DS.%26aulast%3DAban%26aufirst%3DA.%26aulast%3DOkerberg%26aufirst%3DE.%26aulast%3DHerring%26aufirst%3DC.%26aulast%3DNordin%26aufirst%3DB.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DLee%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKozarich%26aufirst%3DJ.%2BW.%26atitle%3DIn%2520situ%2520kinase%2520profiling%2520reveals%2520functionally%2520relevant%2520properties%2520of%2520native%2520kinases%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D699%26epage%3D710%26doi%3D10.1016%2Fj.chembiol.2011.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Shemon, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heil, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granovsky, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McElheny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chimon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosner, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koide, S.</span><span> </span><span class="NLM_article-title">Characterization of the Raf kinase inhibitory protein (RKIP) binding pocket: NMR-based screening identifies small-molecule ligands</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e10479</span><span class="refDoi"> DOI: 10.1371/journal.pone.0010479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pone.0010479" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e10479&author=A.+N.+Shemonauthor=G.+L.+Heilauthor=A.+E.+Granovskyauthor=M.+M.+Clarkauthor=D.+McElhenyauthor=A.+Chimonauthor=M.+R.+Rosnerauthor=S.+Koide&title=Characterization+of+the+Raf+kinase+inhibitory+protein+%28RKIP%29+binding+pocket%3A+NMR-based+screening+identifies+small-molecule+ligands&doi=10.1371%2Fjournal.pone.0010479"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0010479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0010479%26sid%3Dliteratum%253Aachs%26aulast%3DShemon%26aufirst%3DA.%2BN.%26aulast%3DHeil%26aufirst%3DG.%2BL.%26aulast%3DGranovsky%26aufirst%3DA.%2BE.%26aulast%3DClark%26aufirst%3DM.%2BM.%26aulast%3DMcElheny%26aufirst%3DD.%26aulast%3DChimon%26aufirst%3DA.%26aulast%3DRosner%26aufirst%3DM.%2BR.%26aulast%3DKoide%26aufirst%3DS.%26atitle%3DCharacterization%2520of%2520the%2520Raf%2520kinase%2520inhibitory%2520protein%2520%2528RKIP%2529%2520binding%2520pocket%253A%2520NMR-based%2520screening%2520identifies%2520small-molecule%2520ligands%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26spage%3De10479%26doi%3D10.1371%2Fjournal.pone.0010479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuaqui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span> </span><span class="NLM_article-title">Structural interaction fingerprint (SIFt): a novel method for analyzing three-dimensional protein-ligand binding interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">344</span><span class="refDoi"> DOI: 10.1021/jm030331x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030331x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=337-344&author=Z.+Dengauthor=C.+Chuaquiauthor=J.+Singh&title=Structural+interaction+fingerprint+%28SIFt%29%3A+a+novel+method+for+analyzing+three-dimensional+protein-ligand+binding+interactions&doi=10.1021%2Fjm030331x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Interaction Fingerprint (SIFt): A Novel Method for Analyzing Three-Dimensional Protein-Ligand Binding Interactions</span></div><div class="casAuthors">Deng, Zhan; Chuaqui, Claudio; Singh, Juswinder</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-344</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Representing and understanding the three-dimensional (3D) structural information of protein-ligand complexes is a crit. step in the rational drug discovery process.  Traditional anal. methods are proving inadequate and inefficient in dealing with the massive amt. of structural information being generated from x-ray crystallog., NMR, and in silico approaches such as structure-based docking expts.  Here, we present SIFt (structural interaction fingerprint), a novel method for representing and analyzing 3D protein-ligand binding interactions.  Key to this approach is the generation of an interaction fingerprint that translates 3D structural binding information from a protein-ligand complex into a one-dimensional binary string.  Each fingerprint represents the "structural interaction profile" of the complex that can be used to organize, analyze, and visualize the rich amt. of information encoded in ligand-receptor complexes and also to assist database mining.  We have applied SIFt to tackle three common tasks in structure-based drug design.  The first involved the anal. and organization of a typical set of results generated from a docking study.  Using SIFt, docking poses with similar binding modes were identified, clustered, and subsequently compared with conventional scoring function information.  A second application of SIFt was to analyze ∼90 known x-ray crystal structures of protein kinase-inhibitor complexes obtained from the Protein Databank.  Using SIFt, we were able to organize the structures and reveal striking similarities and diversity between their small mol. binding interactions.  Finally, we have shown how SIFt can be used as an effective mol. filter during the virtual chem. library screening process to select mols. with desirable binding mode(s) and/or desirable interaction patterns with the protein target.  In summary, SIFt shows promise to fully leverage the wealth of information being generated in rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkjji1ni0HHbVg90H21EOLACvtfcHk0ljMCadz839nPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D&md5=a652b70184155c107202cbe3bc74596a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm030331x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030331x%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DStructural%2520interaction%2520fingerprint%2520%2528SIFt%2529%253A%2520a%2520novel%2520method%2520for%2520analyzing%2520three-dimensional%2520protein-ligand%2520binding%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D337%26epage%3D344%26doi%3D10.1021%2Fjm030331x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Witek, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smusz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rataj, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mordalski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bojarski, A. J.</span><span> </span><span class="NLM_article-title">An application of machine learning methods to structural interaction fingerprints--a case study of kinase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">585</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.12.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.bmcl.2013.12.017" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=580-585&author=J.+Witekauthor=S.+Smuszauthor=K.+Ratajauthor=S.+Mordalskiauthor=A.+J.+Bojarski&title=An+application+of+machine+learning+methods+to+structural+interaction+fingerprints%2D%2Da+case+study+of+kinase+inhibitors&doi=10.1016%2Fj.bmcl.2013.12.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.12.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.12.017%26sid%3Dliteratum%253Aachs%26aulast%3DWitek%26aufirst%3DJ.%26aulast%3DSmusz%26aufirst%3DS.%26aulast%3DRataj%26aufirst%3DK.%26aulast%3DMordalski%26aufirst%3DS.%26aulast%3DBojarski%26aufirst%3DA.%2BJ.%26atitle%3DAn%2520application%2520of%2520machine%2520learning%2520methods%2520to%2520structural%2520interaction%2520fingerprints--a%2520case%2520study%2520of%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D580%26epage%3D585%26doi%3D10.1016%2Fj.bmcl.2013.12.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">KLIFS: a knowledge-based structural database to navigate kinase-ligand interaction space</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="NLM_x">–</span> <span class="NLM_lpage">277</span><span class="refDoi"> DOI: 10.1021/jm400378w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400378w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=249-277&author=O.+P.+van+Lindenauthor=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+a+knowledge-based+structural+database+to+navigate+kinase-ligand+interaction+space&doi=10.1021%2Fjm400378w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: A Knowledge-Based Structural Database To Navigate Kinase-Ligand Interaction Space</span></div><div class="casAuthors">van Linden, Oscar P. J.; Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinases regulate the majority of signal transduction pathways in cells and have become important targets for the development of designer drugs.  We present a systematic anal. of kinase-ligand interactions in all regions of the catalytic cleft of all 1252 human kinase-ligand cocrystal structures present in the Protein Data Bank (PDB).  The kinase-ligand interaction fingerprints and structure database (KLIFS) contains a consistent alignment of 85 kinase ligand binding site residues that enables the identification of family specific interaction features and classification of ligands according to their binding modes.  We illustrate how systematic mining of kinase-ligand interaction space gives new insights into how conserved and selective kinase interaction hot spots can accommodate the large diversity of chem. scaffolds in kinase ligands.  These analyses lead to an improved understanding of the structural requirements of kinase binding that will be useful in ligand discovery and design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4tMmT6S-vRrVg90H21EOLACvtfcHk0ljMCadz839nPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P&md5=8519942e1703ba04f64797e07df4b712</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm400378w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400378w%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520a%2520knowledge-based%2520structural%2520database%2520to%2520navigate%2520kinase-ligand%2520interaction%2520space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D249%26epage%3D277%26doi%3D10.1021%2Fjm400378w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Bryant, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moll, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavraki, L. E.</span><span> </span><span class="NLM_article-title">Combinatorial clustering of residue position subsets predicts inhibitor affinity across the human kinome</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003087</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1003087" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003087&author=D.+H.+Bryantauthor=M.+Mollauthor=P.+W.+Finnauthor=L.+E.+Kavraki&title=Combinatorial+clustering+of+residue+position+subsets+predicts+inhibitor+affinity+across+the+human+kinome&doi=10.1371%2Fjournal.pcbi.1003087"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003087%26sid%3Dliteratum%253Aachs%26aulast%3DBryant%26aufirst%3DD.%2BH.%26aulast%3DMoll%26aufirst%3DM.%26aulast%3DFinn%26aufirst%3DP.%2BW.%26aulast%3DKavraki%26aufirst%3DL.%2BE.%26atitle%3DCombinatorial%2520clustering%2520of%2520residue%2520position%2520subsets%2520predicts%2520inhibitor%2520affinity%2520across%2520the%2520human%2520kinome%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3De1003087%26doi%3D10.1371%2Fjournal.pcbi.1003087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Cui, J. J.</span><span> </span><span class="NLM_article-title">Targeting receptor tyrosine kinase MET in cancer: small molecule inhibitors and clinical progress</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4427</span><span class="NLM_x">–</span> <span class="NLM_lpage">4453</span><span class="refDoi"> DOI: 10.1021/jm401427c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401427c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWltbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4427-4453&author=J.+J.+Cui&title=Targeting+receptor+tyrosine+kinase+MET+in+cancer%3A+small+molecule+inhibitors+and+clinical+progress&doi=10.1021%2Fjm401427c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Receptor Tyrosine Kinase MET in Cancer: Small Molecule Inhibitors and Clinical Progress</span></div><div class="casAuthors">Cui, J. Jean</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4427-4453</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The HGF/MET signaling pathway is crit. in mediating a wide range of normal physiol. functions including embryol. development, wound healing, and tissue regeneration.  Aberrant activation of the pathway has frequently been found in human cancers via protein overexpression, mutation, gene amplification, and also paracrine or autocrine up-regulation.  In addn., the activation of HGF/MET signaling confers resistance to the effects of cancer treatments.  Therefore, inhibition of the HGF/MET signaling pathway has great potential for therapeutic intervention in cancer.  Currently, there are three approaches toward modulating HGF/MET signaling in human clin. studies of cancer: anti-HGF monoclonal antibodies, MET monoclonal antibodies, and small mol. MET inhibitors.  Preliminary clin. benefit from inhibition of HGF or MET has been reported.  This Perspective will provide an overview of the HGF/MET signaling pathway in cancer and then will review the development of small mol. MET inhibitors and their progress in clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoARAsEnH4X5rVg90H21EOLACvtfcHk0lhyRjoTIxCO4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWltbvE&md5=534fd1c6fd8b83be41251d0d556903d1</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm401427c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401427c%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26atitle%3DTargeting%2520receptor%2520tyrosine%2520kinase%2520MET%2520in%2520cancer%253A%2520small%2520molecule%2520inhibitors%2520and%2520clinical%2520progress%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4427%26epage%3D4453%26doi%3D10.1021%2Fjm401427c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Punta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coggill, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhardt, R. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mistry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tate, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boursnell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forslund, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceric, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clements, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonnhammer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eddy, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bateman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, R. D.</span><span> </span><span class="NLM_article-title">The Pfam protein families database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D290</span><span class="NLM_x">–</span> <span class="NLM_lpage">301</span><span class="refDoi"> DOI: 10.1093/nar/gkr1065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1093%2Fnar%2Fgkr1065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=22127870" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12hurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D290-301&author=M.+Puntaauthor=P.+C.+Coggillauthor=R.+Y.+Eberhardtauthor=J.+Mistryauthor=J.+Tateauthor=C.+Boursnellauthor=N.+Pangauthor=K.+Forslundauthor=G.+Cericauthor=J.+Clementsauthor=A.+Hegerauthor=L.+Holmauthor=E.+L.+Sonnhammerauthor=S.+R.+Eddyauthor=A.+Batemanauthor=R.+D.+Finn&title=The+Pfam+protein+families+database&doi=10.1093%2Fnar%2Fgkr1065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Pfam protein families database</span></div><div class="casAuthors">Punta, Marco; Coggill, Penny C.; Eberhardt, Ruth Y.; Mistry, Jaina; Tate, John; Boursnell, Chris; Pang, Ningze; Forslund, Kristoffer; Ceric, Goran; Clements, Jody; Heger, Andreas; Holm, Liisa; Sonnhammer, Erik L. L.; Eddy, Sean R.; Bateman, Alex; Finn, Robert D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D290-D301</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Pfam is a widely used database of protein families, currently contg. more than 13 000 manually curated protein families as of release 26.0.  Pfam is available via servers in the UK (http://pfam.sanger.ac.uk/), the USA (http://pfam.janelia.org/) and Sweden (http://pfam.sbc.su.se/).  Here, we report on changes that have occurred since our 2010 NAR paper (release 24.0).  Over the last 2 years, we have generated 1840 new families and increased coverage of the UniProt Knowledgebase (UniProtKB) to nearly 80%.  Notably, we have taken the step of opening up the annotation of our families to the Wikipedia community, by linking Pfam families to relevant Wikipedia pages and encouraging the Pfam and Wikipedia communities to improve and expand those pages.  We continue to improve the Pfam web-site and add new visualizations, such as the sunburst' representation of taxonomic distribution of families.  In this work we addnl. address two topics that will be of particular interest to the Pfam community.  First, we explain the definition and use of family-specific, manually curated gathering thresholds.  Second, we discuss some of the features of domains of unknown function (also known as DUFs), which constitute a rapidly growing class of families within Pfam.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonKjUY4YgVhLVg90H21EOLACvtfcHk0lhyRjoTIxCO4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12hurvK&md5=1acf93ff21d746e21cd96b6626efa0a7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1065%26sid%3Dliteratum%253Aachs%26aulast%3DPunta%26aufirst%3DM.%26aulast%3DCoggill%26aufirst%3DP.%2BC.%26aulast%3DEberhardt%26aufirst%3DR.%2BY.%26aulast%3DMistry%26aufirst%3DJ.%26aulast%3DTate%26aufirst%3DJ.%26aulast%3DBoursnell%26aufirst%3DC.%26aulast%3DPang%26aufirst%3DN.%26aulast%3DForslund%26aufirst%3DK.%26aulast%3DCeric%26aufirst%3DG.%26aulast%3DClements%26aufirst%3DJ.%26aulast%3DHeger%26aufirst%3DA.%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DBateman%26aufirst%3DA.%26aulast%3DFinn%26aufirst%3DR.%2BD.%26atitle%3DThe%2520Pfam%2520protein%2520families%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD290%26epage%3D301%26doi%3D10.1093%2Fnar%2Fgkr1065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaLonde, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindvall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semus, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peishoff, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Head, M. S.</span><span> </span><span class="NLM_article-title">A critical assessment of docking programs and scoring functions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5912</span><span class="NLM_x">–</span> <span class="NLM_lpage">5931</span><span class="refDoi"> DOI: 10.1021/jm050362n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050362n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5912-5931&author=G.+L.+Warrenauthor=C.+W.+Andrewsauthor=A.+M.+Capelliauthor=B.+Clarkeauthor=J.+LaLondeauthor=M.+H.+Lambertauthor=M.+Lindvallauthor=N.+Nevinsauthor=S.+F.+Semusauthor=S.+Sengerauthor=G.+Tedescoauthor=I.+D.+Wallauthor=J.+M.+Woolvenauthor=C.+E.+Peishoffauthor=M.+S.+Head&title=A+critical+assessment+of+docking+programs+and+scoring+functions&doi=10.1021%2Fjm050362n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Assessment of Docking Programs and Scoring Functions</span></div><div class="casAuthors">Warren, Gregory L.; Andrews, C. Webster; Capelli, Anna-Maria; Clarke, Brian; LaLonde, Judith; Lambert, Millard H.; Lindvall, Mika; Nevins, Neysa; Semus, Simon F.; Senger, Stefan; Tedesco, Giovanna; Wall, Ian D.; Woolven, James M.; Peishoff, Catherine E.; Head, Martha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5912-5931</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking is a computational technique that samples conformations of small mols. in protein binding sites; scoring functions are used to assess which of these conformations best complements the protein binding site.  An evaluation of 10 docking programs and 37 scoring functions was conducted against eight proteins of seven protein types for three tasks: binding mode prediction, virtual screening for lead identification, and rank-ordering by affinity for lead optimization.  All of the docking programs were able to generate ligand conformations similar to crystallog. detd. protein/ligand complex structures for at least one of the targets.  However, scoring functions were less successful at distinguishing the crystallog. conformation from the set of docked poses.  Docking programs identified active compds. from a pharmaceutically relevant pool of decoy compds.; however, no single program performed well for all of the targets.  For prediction of compd. affinity, none of the docking programs or scoring functions made a useful prediction of ligand binding affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJn8_WHSjurVg90H21EOLACvtfcHk0lhyRjoTIxCO4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D&md5=463ab40c3d23e67b62df408fc5e7f08d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm050362n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050362n%26sid%3Dliteratum%253Aachs%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DCapelli%26aufirst%3DA.%2BM.%26aulast%3DClarke%26aufirst%3DB.%26aulast%3DLaLonde%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DSemus%26aufirst%3DS.%2BF.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26aulast%3DHead%26aufirst%3DM.%2BS.%26atitle%3DA%2520critical%2520assessment%2520of%2520docking%2520programs%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5912%26epage%3D5931%26doi%3D10.1021%2Fjm050362n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Koes, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camacho, C. J.</span><span> </span><span class="NLM_article-title">Pharmer: efficient and exact pharmacophore search</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1307</span><span class="NLM_x">–</span> <span class="NLM_lpage">1314</span><span class="refDoi"> DOI: 10.1021/ci200097m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200097m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFSltL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1307-1314&author=D.+R.+Koesauthor=C.+J.+Camacho&title=Pharmer%3A+efficient+and+exact+pharmacophore+search&doi=10.1021%2Fci200097m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmer: Efficient and Exact Pharmacophore Search</span></div><div class="casAuthors">Koes, David Ryan; Camacho, Carlos J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1307-1314</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pharmacophore search is a key component of many drug discovery efforts.  Pharmer is a new computational approach to pharmacophore search that scales with the breadth and complexity of the query, not the size of the compd. library being screened.  Two novel methods for organizing pharmacophore data, the Pharmer KDB-tree and Bloom fingerprints, enable Pharmer to perform an exact pharmacophore search of almost two million structures in less than a minute.  In general, Pharmer is more than an order of magnitude faster than existing technologies.  The complete source code is available under an open-source license at http://pharmer.sourceforge.net.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVdJ0tI_jotrVg90H21EOLACvtfcHk0lg9s9j6OY6jrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFSltL4%253D&md5=cd9dc10f5ebf14f931829849395cde39</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci200097m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200097m%26sid%3Dliteratum%253Aachs%26aulast%3DKoes%26aufirst%3DD.%2BR.%26aulast%3DCamacho%26aufirst%3DC.%2BJ.%26atitle%3DPharmer%253A%2520efficient%2520and%2520exact%2520pharmacophore%2520search%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1307%26epage%3D1314%26doi%3D10.1021%2Fci200097m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Mordalski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosciolek, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristiansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sylte, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bojarski, A. J.</span><span> </span><span class="NLM_article-title">Protein binding site analysis by means of structural interaction fingerprint patterns</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">6816</span><span class="NLM_x">–</span> <span class="NLM_lpage">6819</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.09.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.bmcl.2011.09.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=21974955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKktr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=6816-6819&author=S.+Mordalskiauthor=T.+Kosciolekauthor=K.+Kristiansenauthor=I.+Sylteauthor=A.+J.+Bojarski&title=Protein+binding+site+analysis+by+means+of+structural+interaction+fingerprint+patterns&doi=10.1016%2Fj.bmcl.2011.09.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Protein binding site analysis by means of structural interaction fingerprint patterns</span></div><div class="casAuthors">Mordalski, Stefan; Kosciolek, Tomasz; Kristiansen, Kurt; Sylte, Ingebrigt; Bojarski, Andrzej J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6816-6819</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We introduce a new approach to the known concept of interaction profiles, based on Structural Interaction Fingerprints (SIFt), for precise and rapid binding site description.  A set of scripts for batch generation and anal. of SIFt were prepd., and the implementation is computationally efficient and supports parallelization.  It is based on a 9-digit binary interaction pattern that describes phys. ligand-protein interactions in structures and models of ligand-protein complexes.  The tool performs anal. and identifies binding site residues (crucial and auxiliary) and classifies interactions according to type (hydrophobic, arom., charge, polar, side chain, and backbone).  It is convenient and easy to use, and gives manageable output data for both, interpretation and further processing.  In the presented Letter, SIFts are applied to analyze binding sites in models of antagonist-5-HT7 receptor complexes and structures of cyclin dependent kinase 2-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryunTDaocwe7Vg90H21EOLACvtfcHk0lg9s9j6OY6jrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKktr3E&md5=cddbdf237f0c8ae19e308ad47bbb1919</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.09.027%26sid%3Dliteratum%253Aachs%26aulast%3DMordalski%26aufirst%3DS.%26aulast%3DKosciolek%26aufirst%3DT.%26aulast%3DKristiansen%26aufirst%3DK.%26aulast%3DSylte%26aufirst%3DI.%26aulast%3DBojarski%26aufirst%3DA.%2BJ.%26atitle%3DProtein%2520binding%2520site%2520analysis%2520by%2520means%2520of%2520structural%2520interaction%2520fingerprint%2520patterns%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D6816%26epage%3D6819%26doi%3D10.1016%2Fj.bmcl.2011.09.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Optimizing fragment and scaffold docking by use of molecular interaction fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">207</span><span class="refDoi"> DOI: 10.1021/ci600342e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+fragment+and+scaffold+docking+by+use+of+molecular+interaction+fingerprints&doi=10.1021%2Fci600342e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0ljziQYz3LhaUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520fragment%2520and%2520scaffold%2520docking%2520by%2520use%2520of%2520molecular%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D195%26epage%3D207%26doi%3D10.1021%2Fci600342e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Mpamhanga, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLay, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Knowledge-based interaction fingerprint scoring: a simple method for improving the effectiveness of fast scoring functions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">686</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1021/ci050420d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050420d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsVGjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=686-698&author=C.+P.+Mpamhangaauthor=B.+Chenauthor=I.+M.+McLayauthor=P.+Willett&title=Knowledge-based+interaction+fingerprint+scoring%3A+a+simple+method+for+improving+the+effectiveness+of+fast+scoring+functions&doi=10.1021%2Fci050420d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge-Based Interaction Fingerprint Scoring: A Simple Method for Improving the Effectiveness of Fast Scoring Functions</span></div><div class="casAuthors">Mpamhanga, Chidochangu P.; Chen, Beining; McLay, Iain M.; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">686-698</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new method for the postprocessing of docking outputs has been developed, based on encoding putative 3D binding modes (docking solns.) as ligand-protein interactions into simple bit strings, a method analogous to the structural interaction fingerprint.  Instead of employing traditional scoring functions, the method uses a series of new, knowledge-based scores derived from the similarity of the bit strings for each docking soln. to that of a known ref. binding mode.  A GOLD docking study was carried out using the Bissantz estrogen receptor antagonist set along with the new scoring method.  Superior recovery rates, with up to 2-fold enrichments, were obsd. when the new knowledge-based scoring was compared to the GOLD fitness score.  In addn., top ranking sets of mols. (actives and potential actives or decoys) were structurally diverse with low mol. wts. and structural complexities.  Principal component anal. and clustering of the fingerprints permits the easy sepn. of active from inactive binding modes and the visualization of diverse binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZfTfk88wvHLVg90H21EOLACvtfcHk0ljziQYz3LhaUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsVGjuw%253D%253D&md5=0fecc7a1894b8bf0e53f077deefc2a85</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci050420d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050420d%26sid%3Dliteratum%253Aachs%26aulast%3DMpamhanga%26aufirst%3DC.%2BP.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DMcLay%26aufirst%3DI.%2BM.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DKnowledge-based%2520interaction%2520fingerprint%2520scoring%253A%2520a%2520simple%2520method%2520for%2520improving%2520the%2520effectiveness%2520of%2520fast%2520scoring%2520functions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D686%26epage%3D698%26doi%3D10.1021%2Fci050420d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Perez-Nueno, V. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabal, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borrell, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teixido, J.</span><span> </span><span class="NLM_article-title">APIF: a new interaction fingerprint based on atom pairs and its application to virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1245</span><span class="NLM_x">–</span> <span class="NLM_lpage">1260</span><span class="refDoi"> DOI: 10.1021/ci900043r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900043r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVSltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=1245-1260&author=V.+I.+Perez-Nuenoauthor=O.+Rabalauthor=J.+I.+Borrellauthor=J.+Teixido&title=APIF%3A+a+new+interaction+fingerprint+based+on+atom+pairs+and+its+application+to+virtual+screening&doi=10.1021%2Fci900043r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">APIF: A New Interaction Fingerprint Based on Atom Pairs and Its Application to Virtual Screening</span></div><div class="casAuthors">Perez-Nueno, Violeta I.; Rabal, Obdulia; Borrell, Jose I.; Teixido, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1245-1260</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new interaction fingerprint (IF) called APIF (atom-pairs-based interaction fingerprint) has been developed for postprocessing protein-ligand docking results.  Unlike other existing fingerprints which employ abs. locations of individual interactions, APIF considers the relative positions of pairs of interacting atoms.  Docking-based virtual screening was performed with GOLD using the crystal structures of trypsin, rhinovirus, HIV protease, carboxypeptidase, and estrogen receptor-α as targets.  A score derived from the similarity of the bit strings for each docking soln. to that of a known ref. binding mode was obtained.  Comparisons between APIF, GoldScore function, and std. interaction fingerprint (CHIF) scores were performed using enrichment plots.  Superior recovery rates were obsd. in the IF score cases.  Comparable results were achieved by using either of the two interaction fingerprints, substantially improving GoldScore function enrichment factors.  Binding mode analyses were also carried out in order to study the best method for selecting conformations with a binding mode similar to that of the ref. crystd. complex.  These showed that the first conformations retrieved by interaction fingerprint scores had a more similar binding mode to the ref. complex than those retrieved by the GoldScore function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxejfqiHaBwrVg90H21EOLACvtfcHk0ljziQYz3LhaUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVSltb4%253D&md5=ae8a172d6de2aba35748b02ec38e3bfe</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci900043r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900043r%26sid%3Dliteratum%253Aachs%26aulast%3DPerez-Nueno%26aufirst%3DV.%2BI.%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DBorrell%26aufirst%3DJ.%2BI.%26aulast%3DTeixido%26aufirst%3DJ.%26atitle%3DAPIF%253A%2520a%2520new%2520interaction%2520fingerprint%2520based%2520on%2520atom%2520pairs%2520and%2520its%2520application%2520to%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D1245%26epage%3D1260%26doi%3D10.1021%2Fci900043r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geppert, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Searching for target-selective compounds using different combinations of multiclass support vector machine ranking methods, kernel functions, and fingerprint descriptors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">582</span><span class="NLM_x">–</span> <span class="NLM_lpage">592</span><span class="refDoi"> DOI: 10.1021/ci800441c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800441c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVKkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=582-592&author=A.+M.+Wassermannauthor=H.+Geppertauthor=J.+Bajorath&title=Searching+for+target-selective+compounds+using+different+combinations+of+multiclass+support+vector+machine+ranking+methods%2C+kernel+functions%2C+and+fingerprint+descriptors&doi=10.1021%2Fci800441c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Searching for Target-Selective Compounds Using Different Combinations of Multiclass Support Vector Machine Ranking Methods, Kernel Functions, and Fingerprint Descriptors</span></div><div class="casAuthors">Wassermann, Anne Mai; Geppert, Hanna; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">582-592</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of small chem. compds. that are selective for a target protein over one or more closely related members of the same family is of high relevance for applications in chem. biol.  Conventional 2D similarity searching using known selective mols. as templates has recently been found to preferentially detect selective over non-selective and inactive database compds.  To improve the initially obsd. search performance, we have attempted to use 2D fingerprints as descriptors for support vector machine (SVM)-based selectivity searching.  Different from typically applied binary SVM compd. classification, SVM anal. has been adapted here for multiclass predictions and compd. ranking to distinguish between selective, active but non-selective, and inactive compds.  In systematic database search calcns., we tested combinations of four alternative SVM ranking schemes, four different kernel functions, and four fingerprints and were able to further improve selectivity search performance by effectively removing non-selective mols. from high ranking positions while retaining high recall of selective compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplfCnoxqEkrrVg90H21EOLACvtfcHk0ljziQYz3LhaUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVKkt7w%253D&md5=6b946bdf10fe4cd6c9fd19f8d2b70eb3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci800441c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800441c%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DGeppert%26aufirst%3DH.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSearching%2520for%2520target-selective%2520compounds%2520using%2520different%2520combinations%2520of%2520multiclass%2520support%2520vector%2520machine%2520ranking%2520methods%252C%2520kernel%2520functions%252C%2520and%2520fingerprint%2520descriptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D582%26epage%3D592%26doi%3D10.1021%2Fci800441c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuaqui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span> </span><span class="NLM_article-title">Knowledge-based design of target-focused libraries using protein-ligand interaction constraints</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">490</span><span class="NLM_x">–</span> <span class="NLM_lpage">500</span><span class="refDoi"> DOI: 10.1021/jm050381x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050381x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=490-500&author=Z.+Dengauthor=C.+Chuaquiauthor=J.+Singh&title=Knowledge-based+design+of+target-focused+libraries+using+protein-ligand+interaction+constraints&doi=10.1021%2Fjm050381x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm050381x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050381x%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DKnowledge-based%2520design%2520of%2520target-focused%2520libraries%2520using%2520protein-ligand%2520interaction%2520constraints%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D490%26epage%3D500%26doi%3D10.1021%2Fjm050381x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Chupakhin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baskin, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varnek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Predicting ligand binding modes from neural networks trained on protein-ligand interaction fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">–</span> <span class="NLM_lpage">772</span><span class="refDoi"> DOI: 10.1021/ci300200r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300200r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslOgs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=763-772&author=V.+Chupakhinauthor=G.+Marcouauthor=I.+Baskinauthor=A.+Varnekauthor=D.+Rognan&title=Predicting+ligand+binding+modes+from+neural+networks+trained+on+protein-ligand+interaction+fingerprints&doi=10.1021%2Fci300200r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Ligand Binding Modes from Neural Networks Trained on Protein-Ligand Interaction Fingerprints</span></div><div class="casAuthors">Chupakhin, Vladimir; Marcou, Gilles; Baskin, Igor; Varnek, Alexandre; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">763-772</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors herewith present a novel approach to predict protein-ligand binding modes from the single two-dimensional structure of the ligand.  Known protein-ligand x-ray structures were converted into binary bit strings encoding protein-ligand interactions.  An artificial neural network was then set up to first learn and then predict protein-ligand interaction fingerprints from simple ligand descriptors.  Specific models were constructed for three targets (CDK2, p38-α, HSP90-α) and 146 ligands for which protein-ligand x-ray structures are available.  These models were able to predict protein-ligand interaction fingerprints and to discriminate important features from minor interactions.  Predicted interaction fingerprints were successfully used as descriptors to discriminate true ligands from decoys by virtual screening.  In some but not all cases, the predicted interaction fingerprints furthermore enable to efficiently rerank cross-docking poses and prioritize the best possible docking solns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-ZJ0Zoe63l7Vg90H21EOLACvtfcHk0lixEEZNHa0pJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslOgs78%253D&md5=85e39717c824c274fbb62e39f1873dcb</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci300200r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300200r%26sid%3Dliteratum%253Aachs%26aulast%3DChupakhin%26aufirst%3DV.%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DBaskin%26aufirst%3DI.%26aulast%3DVarnek%26aufirst%3DA.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DPredicting%2520ligand%2520binding%2520modes%2520from%2520neural%2520networks%2520trained%2520on%2520protein-ligand%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D763%26epage%3D772%26doi%3D10.1021%2Fci300200r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A unified statistical model to support local sequence order independent similarity searching for ligand-binding sites and its application to genome-based drug discovery</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">i305</span><span class="NLM_x">–</span> <span class="NLM_lpage">312</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btp220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1093%2Fbioinformatics%2Fbtp220" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=i305-312&author=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourne&title=A+unified+statistical+model+to+support+local+sequence+order+independent+similarity+searching+for+ligand-binding+sites+and+its+application+to+genome-based+drug+discovery&doi=10.1093%2Fbioinformatics%2Fbtp220"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp220%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520unified%2520statistical%2520model%2520to%2520support%2520local%2520sequence%2520order%2520independent%2520similarity%2520searching%2520for%2520ligand-binding%2520sites%2520and%2520its%2520application%2520to%2520genome-based%2520drug%2520discovery%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3Di305%26epage%3D312%26doi%3D10.1093%2Fbioinformatics%2Fbtp220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Briedis, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starr, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Analysis of the human kinome using methods including fold recognition reveals two novel kinases</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1597</span><span class="refDoi"> DOI: 10.1371/journal.pone.0001597</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pone.0001597" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e1597&author=K.+M.+Briedisauthor=A.+Starrauthor=P.+E.+Bourne&title=Analysis+of+the+human+kinome+using+methods+including+fold+recognition+reveals+two+novel+kinases&doi=10.1371%2Fjournal.pone.0001597"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0001597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0001597%26sid%3Dliteratum%253Aachs%26aulast%3DBriedis%26aufirst%3DK.%2BM.%26aulast%3DStarr%26aufirst%3DA.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DAnalysis%2520of%2520the%2520human%2520kinome%2520using%2520methods%2520including%2520fold%2520recognition%2520reveals%2520two%2520novel%2520kinases%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26spage%3De1597%26doi%3D10.1371%2Fjournal.pone.0001597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Detecting evolutionary relationships across existing fold space, using sequence order-independent profile-profile alignments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">5441</span><span class="NLM_x">–</span> <span class="NLM_lpage">5446</span><span class="refDoi"> DOI: 10.1073/pnas.0704422105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1073%2Fpnas.0704422105" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=5441-5446&author=L.+Xieauthor=P.+E.+Bourne&title=Detecting+evolutionary+relationships+across+existing+fold+space%2C+using+sequence+order-independent+profile-profile+alignments&doi=10.1073%2Fpnas.0704422105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0704422105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0704422105%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDetecting%2520evolutionary%2520relationships%2520across%2520existing%2520fold%2520space%252C%2520using%2520sequence%2520order-independent%2520profile-profile%2520alignments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D5441%26epage%3D5446%26doi%3D10.1073%2Fpnas.0704422105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Konc, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janezic, D.</span><span> </span><span class="NLM_article-title">Binding site comparison for function prediction and pharmaceutical discovery</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25C</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1016/j.sbi.2013.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.sbi.2013.11.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25C&publication_year=2014&pages=34-39&author=J.+Koncauthor=D.+Janezic&title=Binding+site+comparison+for+function+prediction+and+pharmaceutical+discovery&doi=10.1016%2Fj.sbi.2013.11.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2013.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2013.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJanezic%26aufirst%3DD.%26atitle%3DBinding%2520site%2520comparison%2520for%2520function%2520prediction%2520and%2520pharmaceutical%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2014%26volume%3D25C%26spage%3D34%26epage%3D39%26doi%3D10.1016%2Fj.sbi.2013.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Towards structural systems pharmacology to study complex diseases and personalized medicine</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003554</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1003554" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003554&author=L.+Xieauthor=X.+Geauthor=H.+Tanauthor=L.+Xieauthor=Y.+Zhangauthor=T.+Hartauthor=X.+Yangauthor=P.+E.+Bourne&title=Towards+structural+systems+pharmacology+to+study+complex+diseases+and+personalized+medicine&doi=10.1371%2Fjournal.pcbi.1003554"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003554%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DGe%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHart%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DTowards%2520structural%2520systems%2520pharmacology%2520to%2520study%2520complex%2520diseases%2520and%2520personalized%2520medicine%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003554%26doi%3D10.1371%2Fjournal.pcbi.1003554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Nisius, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sha, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span> </span><span class="NLM_article-title">Structure-based computational analysis of protein binding sites for function and druggability prediction</span> <span class="citation_source-journal">J. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">159</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1016/j.jbiotec.2011.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.jbiotec.2011.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=22197384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2012&pages=123-134&author=B.+Nisiusauthor=F.+Shaauthor=H.+Gohlke&title=Structure-based+computational+analysis+of+protein+binding+sites+for+function+and+druggability+prediction&doi=10.1016%2Fj.jbiotec.2011.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based computational analysis of protein binding sites for function and druggability prediction</span></div><div class="casAuthors">Nisius, Britta; Sha, Fan; Gohlke, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">123-134</span>CODEN:
                <span class="NLM_cas:coden">JBITD4</span>;
        ISSN:<span class="NLM_cas:issn">0168-1656</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein binding sites are the places where mol. interactions occur.  Thus, the anal. of protein binding sites is of crucial importance to understand the biol. processes proteins are involved in.  Herein, we focus on the computational anal. of protein binding sites and present structure-based methods that enable function prediction for orphan proteins and prediction of target druggability.  We present the general ideas behind these methods, with a special emphasis on the scopes and limitations of these methods and their validation.  Addnl., we present some successful applications of computational binding site anal. to emphasize the practical importance of these methods for biotechnol./bioeconomy and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXzgcXRf62cbVg90H21EOLACvtfcHk0liYi-FkeO3vHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahurs%253D&md5=9d8c5a3a41abd1ce0444ed23c3b3f7a9</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.jbiotec.2011.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbiotec.2011.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DNisius%26aufirst%3DB.%26aulast%3DSha%26aufirst%3DF.%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DStructure-based%2520computational%2520analysis%2520of%2520protein%2520binding%2520sites%2520for%2520function%2520and%2520druggability%2520prediction%26jtitle%3DJ.%2520Biotechnol.%26date%3D2012%26volume%3D159%26spage%3D123%26epage%3D134%26doi%3D10.1016%2Fj.jbiotec.2011.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A robust and efficient algorithm for the shape description of protein structures and its application in predicting ligand binding sites</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 4</span><span class="NLM_x">) </span> <span class="NLM_fpage">S9</span><span class="refDoi"> DOI: 10.1186/1471-2105-8-S4-S9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1186%2F1471-2105-8-S4-S9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=17570152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BD2szotFyitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=S9&issue=Suppl.+4&author=L.+Xieauthor=P.+E.+Bourne&title=A+robust+and+efficient+algorithm+for+the+shape+description+of+protein+structures+and+its+application+in+predicting+ligand+binding+sites&doi=10.1186%2F1471-2105-8-S4-S9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A robust and efficient algorithm for the shape description of protein structures and its application in predicting ligand binding sites</span></div><div class="casAuthors">Xie Lei; Bourne Philip E</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8 Suppl 4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">S9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  An accurate description of protein shape derived from protein structure is necessary to establish an understanding of protein-ligand interactions, which in turn will lead to improved methods for protein-ligand docking and binding site analysis.  Most current shape descriptors characterize only the local properties of protein structure using an all-atom representation and are slow to compute.  We need new shape descriptors that have the ability to capture both local and global structural information, are robust for application to models and low quality structures and are computationally efficient to permit high throughput analysis of protein structures.  RESULTS:  We introduce a new shape description that requires only the Calpha atoms to represent the protein structure, thus making it both fast and suitable for use on models and low quality structures.  The notion of a geometric potential is introduced to quantitatively describe the shape of the structure.  This geometric potential is dependent on both the global shape of the protein structure as well as the surrounding environment of each residue.  When applying the geometric potential for binding site prediction, approximately 85% of known binding sites can be accurately identified with above 50% residue coverage and 80% specificity.  Moreover, the algorithm is fast enough for proteome-scale applications.  Proteins with fewer than 500 amino acids can be scanned in less than two seconds.  CONCLUSION:  The reduced representation of the protein structure combined with the geometric potential provides a fast, quantitative description of protein-ligand binding sites with potential for use in large-scale predictions, comparisons and analysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3yUgGb2q8-gpJitSxOdIlfW6udTcc2eYMwNlGYkrSIrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2szotFyitQ%253D%253D&md5=bd06c6a317d572d275df0b4d7d317af7</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-8-S4-S9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-8-S4-S9%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520robust%2520and%2520efficient%2520algorithm%2520for%2520the%2520shape%2520description%2520of%2520protein%2520structures%2520and%2520its%2520application%2520in%2520predicting%2520ligand%2520binding%2520sites%26jtitle%3DBMC%2520Bioinf.%26date%3D2007%26volume%3D8%26issue%3DSuppl.%25204%26spage%3DS9%26doi%3D10.1186%2F1471-2105-8-S4-S9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: identification of the protein-ligand binding network to explain the side effects of CETP inhibitors</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000387</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1000387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19436720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BD1MzktVKnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000387&author=L.+Xieauthor=J.+Liauthor=L.+Xieauthor=P.+E.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+identification+of+the+protein-ligand+binding+network+to+explain+the+side+effects+of+CETP+inhibitors&doi=10.1371%2Fjournal.pcbi.1000387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery using chemical systems biology: identification of the protein-ligand binding network to explain the side effects of CETP inhibitors</span></div><div class="casAuthors">Xie Li; Li Jerry; Xie Lei; Bourne Philip E</div><div class="citationInfo"><span class="NLM_cas:title">PLoS computational biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e1000387</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Systematic identification of protein-drug interaction networks is crucial to correlate complex modes of drug action to clinical indications.  We introduce a novel computational strategy to identify protein-ligand binding profiles on a genome-wide scale and apply it to elucidating the molecular mechanisms associated with the adverse drug effects of Cholesteryl Ester Transfer Protein (CETP) inhibitors.  CETP inhibitors are a new class of preventive therapies for the treatment of cardiovascular disease.  However, clinical studies indicated that one CETP inhibitor, Torcetrapib, has deadly off-target effects as a result of hypertension, and hence it has been withdrawn from phase III clinical trials.  We have identified a panel of off-targets for Torcetrapib and other CETP inhibitors from the human structural genome and map those targets to biological pathways via the literature.  The predicted protein-ligand network is consistent with experimental results from multiple sources and reveals that the side-effect of CETP inhibitors is modulated through the combinatorial control of multiple interconnected pathways.  Given that combinatorial control is a common phenomenon observed in many biological processes, our findings suggest that adverse drug effects might be minimized by fine-tuning multiple off-target interactions using single or multiple therapies.  This work extends the scope of chemogenomics approaches and exemplifies the role that systems biology has in the future of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSfisyXvTOXkCnGiHuhFGv8fW6udTcc2eYMwNlGYkrSIrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzktVKnsA%253D%253D&md5=9ed4e6023419132e66f8800d29676cb0</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000387%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520identification%2520of%2520the%2520protein-ligand%2520binding%2520network%2520to%2520explain%2520the%2520side%2520effects%2520of%2520CETP%2520inhibitors%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3De1000387%26doi%3D10.1371%2Fjournal.pcbi.1000387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Chang, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palsson, B. O.</span><span> </span><span class="NLM_article-title">Drug off-target effects predicted using structural analysis in the context of a metabolic network model</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000938</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1000938" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=e1000938&author=R.+L.+Changauthor=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourneauthor=B.+O.+Palsson&title=Drug+off-target+effects+predicted+using+structural+analysis+in+the+context+of+a+metabolic+network+model&doi=10.1371%2Fjournal.pcbi.1000938"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000938%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DR.%2BL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DPalsson%26aufirst%3DB.%2BO.%26atitle%3DDrug%2520off-target%2520effects%2520predicted%2520using%2520structural%2520analysis%2520in%2520the%2520context%2520of%2520a%2520metabolic%2520network%2520model%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3De1000938%26doi%3D10.1371%2Fjournal.pcbi.1000938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchmeier, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: repositioning the safe medicine Comtan to treat multi-drug and extensively drug resistant tuberculosis</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000423</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1000423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19578428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BD1MvmsFGntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000423&author=S.+L.+Kinningsauthor=N.+Liuauthor=N.+Buchmeierauthor=P.+J.+Tongeauthor=L.+Xieauthor=P.+E.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+repositioning+the+safe+medicine+Comtan+to+treat+multi-drug+and+extensively+drug+resistant+tuberculosis&doi=10.1371%2Fjournal.pcbi.1000423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery using chemical systems biology: repositioning the safe medicine Comtan to treat multi-drug and extensively drug resistant tuberculosis</span></div><div class="casAuthors">Kinnings Sarah L; Liu Nina; Buchmeier Nancy; Tonge Peter J; Xie Lei; Bourne Philip E</div><div class="citationInfo"><span class="NLM_cas:title">PLoS computational biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1000423</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The rise of multi-drug resistant (MDR) and extensively drug resistant (XDR) tuberculosis around the world, including in industrialized nations, poses a great threat to human health and defines a need to develop new, effective and inexpensive anti-tubercular agents.  Previously we developed a chemical systems biology approach to identify off-targets of major pharmaceuticals on a proteome-wide scale.  In this paper we further demonstrate the value of this approach through the discovery that existing commercially available drugs, prescribed for the treatment of Parkinson's disease, have the potential to treat MDR and XDR tuberculosis.  These drugs, entacapone and tolcapone, are predicted to bind to the enzyme InhA and directly inhibit substrate binding.  The prediction is validated by in vitro and InhA kinetic assays using tablets of Comtan, whose active component is entacapone.  The minimal inhibition concentration (MIC(99)) of entacapone for Mycobacterium tuberculosis (M.tuberculosis) is approximately 260.0 microM, well below the toxicity concentration determined by an in vitro cytotoxicity model using a human neuroblastoma cell line.  Moreover, kinetic assays indicate that Comtan inhibits InhA activity by 47.0% at an entacapone concentration of approximately 80 microM.  Thus the active component in Comtan represents a promising lead compound for developing a new class of anti-tubercular therapeutics with excellent safety profiles.  More generally, the protocol described in this paper can be included in a drug discovery pipeline in an effort to discover novel drug leads with desired safety profiles, and therefore accelerate the development of new drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSY95JLzAotpaNc3HjJRoFAfW6udTcc2easqYswPcMd9rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvmsFGntg%253D%253D&md5=9b5e736b9cf022446f4c0530bbe666af</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000423%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DBuchmeier%26aufirst%3DN.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520repositioning%2520the%2520safe%2520medicine%2520Comtan%2520to%2520treat%2520multi-drug%2520and%2520extensively%2520drug%2520resistant%2520tuberculosis%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3De1000423%26doi%3D10.1371%2Fjournal.pcbi.1000423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Structure-based systems biology for analyzing off-target binding</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">189</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.1016/j.sbi.2011.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.sbi.2011.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=21292475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1Kis7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=189-199&author=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourne&title=Structure-based+systems+biology+for+analyzing+off-target+binding&doi=10.1016%2Fj.sbi.2011.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based systems biology for analyzing off-target binding</span></div><div class="casAuthors">Xie, Lei; Xie, Li; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">189-199</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Here off-target binding implies the binding of a small mol. of therapeutic interest to a protein target other than the primary target for which it was intended.  Increasingly such off-targeting appears to be the norm rather than the exception, rational drug design notwithstanding, and can lead to detrimental side-effects, or opportunities to reposition a therapeutic agent to treat a different condition.  Not surprisingly, there is significant interest in detg. a priori what off-targets exist on a proteome-wide scale.  Beyond detg. putative off-targets is the need to understand the impact of such binding on the complete biol. system, with the ultimate goal of being able to predict the phenotypic outcome.  While a very ambitious goal, some progress is being made.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrElfKnuS2XpbVg90H21EOLACvtfcHk0lhYlF-qTy713Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1Kis7w%253D&md5=76036b3b2a7bd319f437d5c875bccbde</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2011.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2011.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DStructure-based%2520systems%2520biology%2520for%2520analyzing%2520off-target%2520binding%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D189%26epage%3D199%26doi%3D10.1016%2Fj.sbi.2011.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Garijo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinnings, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gil, Y.</span><span> </span><span class="NLM_article-title">Quantifying reproducibility in computational biology: the case of the tuberculosis drugome</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e80278</span><span class="refDoi"> DOI: 10.1371/journal.pone.0080278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pone.0080278" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e80278&author=D.+Garijoauthor=S.+Kinningsauthor=L.+Xieauthor=L.+Xieauthor=Y.+Zhangauthor=P.+E.+Bourneauthor=Y.+Gil&title=Quantifying+reproducibility+in+computational+biology%3A+the+case+of+the+tuberculosis+drugome&doi=10.1371%2Fjournal.pone.0080278"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0080278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0080278%26sid%3Dliteratum%253Aachs%26aulast%3DGarijo%26aufirst%3DD.%26aulast%3DKinnings%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DGil%26aufirst%3DY.%26atitle%3DQuantifying%2520reproducibility%2520in%2520computational%2520biology%253A%2520the%2520case%2520of%2520the%2520tuberculosis%2520drugome%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De80278%26doi%3D10.1371%2Fjournal.pone.0080278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Durrant, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amaro, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urbaniak, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haapalainen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Guilmi, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wunder, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">A multidimensional strategy to detect polypharmacological targets in the absence of structural and sequence homology</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000648</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1000648" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=e1000648&author=J.+D.+Durrantauthor=R.+E.+Amaroauthor=L.+Xieauthor=M.+D.+Urbaniakauthor=M.+A.+Fergusonauthor=A.+Haapalainenauthor=Z.+Chenauthor=A.+M.+Di+Guilmiauthor=F.+Wunderauthor=P.+E.+Bourneauthor=J.+A.+McCammon&title=A+multidimensional+strategy+to+detect+polypharmacological+targets+in+the+absence+of+structural+and+sequence+homology&doi=10.1371%2Fjournal.pcbi.1000648"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000648%26sid%3Dliteratum%253Aachs%26aulast%3DDurrant%26aufirst%3DJ.%2BD.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DUrbaniak%26aufirst%3DM.%2BD.%26aulast%3DFerguson%26aufirst%3DM.%2BA.%26aulast%3DHaapalainen%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DDi%2BGuilmi%26aufirst%3DA.%2BM.%26aulast%3DWunder%26aufirst%3DF.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DA%2520multidimensional%2520strategy%2520to%2520detect%2520polypharmacological%2520targets%2520in%2520the%2520absence%2520of%2520structural%2520and%2520sequence%2520homology%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3De1000648%26doi%3D10.1371%2Fjournal.pcbi.1000648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Mycobacterium tuberculosis drugome and its polypharmacological implications</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000976</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1000976" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=e1000976&author=S.+L.+Kinningsauthor=L.+Xieauthor=K.+H.+Fungauthor=R.+M.+Jacksonauthor=L.+Xieauthor=P.+E.+Bourne&title=The+Mycobacterium+tuberculosis+drugome+and+its+polypharmacological+implications&doi=10.1371%2Fjournal.pcbi.1000976"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000976%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DFung%26aufirst%3DK.%2BH.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Mycobacterium%2520tuberculosis%2520drugome%2520and%2520its%2520polypharmacological%2520implications%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3De1000976%26doi%3D10.1371%2Fjournal.pcbi.1000976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evangelidis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: weak inhibition of multiple kinases may contribute to the anti-cancer effect of nelfinavir</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1002037</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1002037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1371%2Fjournal.pcbi.1002037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=21552547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsl2nsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002037&author=L.+Xieauthor=T.+Evangelidisauthor=L.+Xieauthor=P.+E.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+weak+inhibition+of+multiple+kinases+may+contribute+to+the+anti-cancer+effect+of+nelfinavir&doi=10.1371%2Fjournal.pcbi.1002037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery using chemical systems biology: weak inhibition of multiple kinases may contribute to the anti-cancer effect of Nelfinavir</span></div><div class="casAuthors">Xie, Li; Evangelidis, Thomas; Xie, Lei; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e1002037</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Nelfinavir is a potent HIV-protease inhibitor with pleiotropic effects in cancer cells.  Exptl. studies connect its anti-cancer effects to the suppression of the Akt signaling pathway, but the actual mol. targets remain unknown.  Using a structural proteome-wide off-target pipeline, which integrates mol. dynamics simulation and MM/GBSA free energy calcns. with ligand binding site comparison and biol. network anal., we identified putative human off-targets of Nelfinavir and analyzed the impact on the assocd. biol. processes.  Our results suggest that Nelfinavir is able to inhibit multiple members of the protein kinase-like superfamily, which are involved in the regulation of cellular processes vital for carcinogenesis and metastasis.  The computational predictions are supported by kinase activity assays and are consistent with existing exptl. and clin. evidence.  This finding provides a mol. basis to explain the broad-spectrum anti-cancer effect of Nelfinavir and presents opportunities to optimize the drug as a targeted polypharmacol. agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGookAtcKasU9bVg90H21EOLACvtfcHk0liSxgnogE9ujw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsl2nsrw%253D&md5=021c009e7d74cc1085577a5e11bfb23b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002037%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DEvangelidis%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520weak%2520inhibition%2520of%2520multiple%2520kinases%2520may%2520contribute%2520to%2520the%2520anti-cancer%2520effect%2520of%2520nelfinavir%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3De1002037%26doi%3D10.1371%2Fjournal.pcbi.1002037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Novel computational approaches to polypharmacology as a means to define responses to individual drugs</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">379</span><span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010611-134630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1146%2Fannurev-pharmtox-010611-134630" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=361-379&author=L.+Xieauthor=L.+Xieauthor=S.+L.+Kinningsauthor=P.+E.+Bourne&title=Novel+computational+approaches+to+polypharmacology+as+a+means+to+define+responses+to+individual+drugs&doi=10.1146%2Fannurev-pharmtox-010611-134630"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010611-134630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010611-134630%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DNovel%2520computational%2520approaches%2520to%2520polypharmacology%2520as%2520a%2520means%2520to%2520define%2520responses%2520to%2520individual%2520drugs%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2012%26volume%3D52%26spage%3D361%26epage%3D379%26doi%3D10.1146%2Fannurev-pharmtox-010611-134630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A machine learning-based method to improve docking scoring functions and its application to drug repurposing</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">408</span><span class="NLM_x">–</span> <span class="NLM_lpage">419</span><span class="refDoi"> DOI: 10.1021/ci100369f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100369f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=408-419&author=S.+L.+Kinningsauthor=N.+Liuauthor=P.+J.+Tongeauthor=R.+M.+Jacksonauthor=L.+Xieauthor=P.+E.+Bourne&title=A+machine+learning-based+method+to+improve+docking+scoring+functions+and+its+application+to+drug+repurposing&doi=10.1021%2Fci100369f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A Machine learning-based method to improve docking scoring functions and its application to drug repurposing</span></div><div class="casAuthors">Kinnings, Sarah L.; Liu, Nina; Tonge, Peter J.; Jackson, Richard M.; Xie, Lei; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">408-419</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking scoring functions are notoriously weak predictors of binding affinity.  They typically assign a common set of wts. to the individual energy terms that contribute to the overall energy score; however, these wts. should be gene family dependent.  In addn., they incorrectly assume that individual interactions contribute toward the total binding affinity in an additive manner.  In reality, noncovalent interactions often depend on one another in a nonlinear manner.  In this paper, we show how the use of support vector machines (SVMs), trained by assocg. sets of individual energy terms retrieved from mol. docking with the known binding affinity of each compd. from high-throughput screening expts., can be used to improve the correlation between known binding affinities and those predicted by the docking program eHiTS.  We construct two prediction models: a regression model trained using IC50 values from BindingDB, and a classification model trained using active and decoy compds. from the Directory of Useful Decoys (DUD).  Moreover, to address the issue of overrepresentation of neg. data in high-throughput screening data sets, we have designed a multiple-planar SVM training procedure for the classification model.  The increased performance that both SVMs give when compared with the original eHiTS scoring function highlights the potential for using nonlinear methods when deriving overall energy scores from their individual components.  We apply the above methodol. to train a new scoring function for direct inhibitors of Mycobacterium tuberculosis (M.tb) InhA.  By combining ligand binding site comparison with the new scoring function, we propose that phosphodiesterase inhibitors can potentially be repurposed to target M.tb InhA.  Our methodol. may be applied to other gene families for which target structures and activity data are available, as demonstrated in the work presented here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohmsJPycuBRrVg90H21EOLACvtfcHk0liSxgnogE9ujw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSitbs%253D&md5=4f008a6145fc0f6d2bf4e14030555dec</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fci100369f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100369f%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520machine%2520learning-based%2520method%2520to%2520improve%2520docking%2520scoring%2520functions%2520and%2520its%2520application%2520to%2520drug%2520repurposing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D408%26epage%3D419%26doi%3D10.1021%2Fci100369f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Chang, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palsson, B. O.</span><span> </span><span class="NLM_article-title">Antibacterial mechanisms identified through structural systems pharmacology</span> <span class="citation_source-journal">BMC Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">102</span><span class="refDoi"> DOI: 10.1186/1752-0509-7-102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1186%2F1752-0509-7-102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=24112686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltlGntLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2013&pages=102&author=R.+L.+Changauthor=L.+Xieauthor=P.+E.+Bourneauthor=B.+O.+Palsson&title=Antibacterial+mechanisms+identified+through+structural+systems+pharmacology&doi=10.1186%2F1752-0509-7-102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial mechanisms identified through structural systems pharmacology</span></div><div class="casAuthors">Chang, Roger L.; Xie, Lei; Bourne, Philip E.; Palsson, Bernhard O.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Systems Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">102/1-102/12</span>CODEN:
                <span class="NLM_cas:coden">BSBMCC</span>;
        ISSN:<span class="NLM_cas:issn">1752-0509</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">The growing discipline of structural systems pharmacol. is applied prospectively in this study to predict pharmacol. outcomes of antibacterial compds. in Escherichia coli K12.  This work builds upon previously established methods for structural prediction of ligand binding pockets on protein mols. and uses and expands upon the previously developed genome scale model of metab. integrated with protein structures (GEM-PRO) for E. coli, structurally accounting for protein complexes.  Carefully selected case studies display the potential for this structural systems pharmacol. framework in discovery and development of antibacterial compds.  The prediction framework for antibacterial activity of compds. was validated for a control set of well-studied compds., recapitulating exptl.-detd. protein binding interactions and deleterious growth phenotypes resulting from these interactions.  The antibacterial activity of fosfomycin, sulfathiazole, and trimethoprim were accurately predicted, and as a neg. control glucose has no predicted antibacterial activity.  Previously uncharacterized mechanisms of action were predicted for compds. with known antibacterial properties, including (1-hydroxyheptane-1,1-diyl)bis(phosphonic acid) and cholesteryl oleate.  Five candidate inhibitors were predicted for a desirable target protein without any known inhibitors, tryptophan synthase β subunit (TrpB).  In addn. to the predictions presented, this effort also included significant expansion of the previously developed GEM-PRO to account for physiol. assemblies of protein complex structures with activities included in the E. coli K12 metabolic network.  Thus, the structural systems pharmacol. framework presented in this study is effective in the prediction of mol. mechanisms of antibacterial compds.  The study provides a promising proof of principle for such an approach to antibacterial development and raises specific mol. and systemic hypotheses about antibacterials that are amenable to exptl. testing.  This framework, and perhaps also the specific predictions of antibacterials, is extensible to developing antibacterial treatments for pathogenic E. coli and other bacterial pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTSJ0x0vSPZLVg90H21EOLACvtfcHk0lgD4nkIRilFTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltlGntLY%253D&md5=f00b0087ecf99cb55af0977dbebf5164</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1186%2F1752-0509-7-102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1752-0509-7-102%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DR.%2BL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DPalsson%26aufirst%3DB.%2BO.%26atitle%3DAntibacterial%2520mechanisms%2520identified%2520through%2520structural%2520systems%2520pharmacology%26jtitle%3DBMC%2520Syst.%2520Biol.%26date%3D2013%26volume%3D7%26spage%3D102%26doi%3D10.1186%2F1752-0509-7-102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span> </span><span class="NLM_article-title">Lead- and drug-like compounds: the rule-of-five revolution</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span><span class="refDoi"> DOI: 10.1016/j.ddtec.2004.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.ddtec.2004.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=24981612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=337-341&author=C.+A.+Lipinski&title=Lead-+and+drug-like+compounds%3A+the+rule-of-five+revolution&doi=10.1016%2Fj.ddtec.2004.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Lead- and drug-like compounds: the rule-of-five revolution</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-341</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Citations in CAS SciFinder to the rule-of-five (RO5) publication will exceed 1000 by year-end 2004.  Trends in the RO5 literature explosion that can be discerned are the further definitions of drug-like.  This topic is explored in terms of drug-like physicochem. features, drug-like structural features, a comparison of drug-like and non-drug-like in drug discovery and a discussion of how drug-like features relate to clin. success.  Physicochem. features of CNS drugs and features related to CNS blood-brain transporter affinity are briefly reviewed.  Recent literature on features of non-oral drugs is reviewed and how features of lead-like compds. differ from those of drug-like compds. is discussed.  Most recently, partly driven by NIH roadmap initiatives, considerations have arisen as to what tool-like means in the search for chem. tools to probe biol. space.  All these topics frame the scope of this short review/perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnjiyP3CY7lLVg90H21EOLACvtfcHk0lgD4nkIRilFTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D&md5=952c29b47003b884116d790ae9e4721f</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DLead-%2520and%2520drug-like%2520compounds%253A%2520the%2520rule-of-five%2520revolution%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D337%26epage%3D341%26doi%3D10.1016%2Fj.ddtec.2004.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Applying thermodynamic profiling in lead finding and optimization</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">95</span><span class="NLM_x">–</span> <span class="NLM_lpage">110</span><span class="refDoi"> DOI: 10.1038/nrd4486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnrd4486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25614222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVemtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=95-110&author=G.+Klebe&title=Applying+thermodynamic+profiling+in+lead+finding+and+optimization&doi=10.1038%2Fnrd4486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Applying thermodynamic profiling in lead finding and optimization</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">95-110</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery involves the optimization of various physicochem. properties of a ligand, particularly its binding affinity for its target receptor (or receptors).  In recent years, there has been growing interest in using thermodn. profiling of ligand-receptor interactions in order to select and optimize those ligands that might be most likely to become drug candidates with desirable physicochem. properties.  The thermodn. of binding is influenced by multiple factors, including hydrogen bonding and hydrophobic interactions, desolvation, residual mobility, dynamics and the local water structure.  This article discusses key issues in understanding the effects of these factors and applying this knowledge in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqtHj7SsL6rrVg90H21EOLACvtfcHk0lha9FX--T6log"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVemtr4%253D&md5=c2e12760623de81335ce252cb804c602</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnrd4486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4486%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DApplying%2520thermodynamic%2520profiling%2520in%2520lead%2520finding%2520and%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D95%26epage%3D110%26doi%3D10.1038%2Fnrd4486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Constantine, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzler, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonnell, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todderud, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldfarb, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newitt, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiefer, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tortolani, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tokarski, J.</span><span> </span><span class="NLM_article-title">Multiple and single binding modes of fragment-like kinase inhibitors revealed by molecular modeling, residue type-selective protonation, and nuclear overhauser effects</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">6225</span><span class="NLM_x">–</span> <span class="NLM_lpage">6229</span><span class="refDoi"> DOI: 10.1021/jm800747w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800747w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=6225-6229&author=K.+L.+Constantineauthor=L.+Muellerauthor=W.+J.+Metzlerauthor=P.+A.+McDonnellauthor=G.+Todderudauthor=V.+Goldfarbauthor=Y.+Fanauthor=J.+A.+Newittauthor=S.+E.+Kieferauthor=M.+Gaoauthor=D.+Tortolaniauthor=W.+Vaccaroauthor=J.+Tokarski&title=Multiple+and+single+binding+modes+of+fragment-like+kinase+inhibitors+revealed+by+molecular+modeling%2C+residue+type-selective+protonation%2C+and+nuclear+overhauser+effects&doi=10.1021%2Fjm800747w"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm800747w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800747w%26sid%3Dliteratum%253Aachs%26aulast%3DConstantine%26aufirst%3DK.%2BL.%26aulast%3DMueller%26aufirst%3DL.%26aulast%3DMetzler%26aufirst%3DW.%2BJ.%26aulast%3DMcDonnell%26aufirst%3DP.%2BA.%26aulast%3DTodderud%26aufirst%3DG.%26aulast%3DGoldfarb%26aufirst%3DV.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DNewitt%26aufirst%3DJ.%2BA.%26aulast%3DKiefer%26aufirst%3DS.%2BE.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DTortolani%26aufirst%3DD.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DTokarski%26aufirst%3DJ.%26atitle%3DMultiple%2520and%2520single%2520binding%2520modes%2520of%2520fragment-like%2520kinase%2520inhibitors%2520revealed%2520by%2520molecular%2520modeling%252C%2520residue%2520type-selective%2520protonation%252C%2520and%2520nuclear%2520overhauser%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D6225%26epage%3D6229%26doi%3D10.1021%2Fjm800747w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eliseenkova, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubert, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohammadi, M.</span><span> </span><span class="NLM_article-title">A molecular brake in the kinase hinge region regulates the activity of receptor tyrosine kinases</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">717</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.molcel.2007.06.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=17803937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=717-730&author=H.+Chenauthor=J.+Maauthor=W.+Liauthor=A.+V.+Eliseenkovaauthor=C.+Xuauthor=T.+A.+Neubertauthor=W.+T.+Millerauthor=M.+Mohammadi&title=A+molecular+brake+in+the+kinase+hinge+region+regulates+the+activity+of+receptor+tyrosine+kinases&doi=10.1016%2Fj.molcel.2007.06.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">A molecular brake in the kinase hinge region regulates the activity of receptor tyrosine kinases</span></div><div class="casAuthors">Chen, Huaibin; Ma, Jinghong; Li, Wanqing; Eliseenkova, Anna V.; Xu, Chongfeng; Neubert, Thomas A.; Miller, W. Todd; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">717-730</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Activating mutations in the tyrosine kinase domain of receptor tyrosine kinases (RTKs) cause cancer and skeletal disorders.  Comparison of the crystal structures of unphosphorylated and phosphorylated wild-type FGFR2 kinase domains (FGFR2Ks) with those of seven unphosphorylated pathogenic mutants reveals an autoinhibitory "mol. brake" mediated by a triad of residues in the kinase hinge region of all FGFRs.  Structural anal. shows that many other RTKs, including PDGFRs, VEGFRs, KIT, CSF1R, FLT3, TEK, and TIE, are also subject to regulation by this brake.  Pathogenic mutations activate FGFRs and other RTKs by disengaging the brake either directly or indirectly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobYTkvrN7lYrVg90H21EOLACvtfcHk0lha9FX--T6log"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I&md5=dba8a6b97b9d1826b2223146be137963</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.028%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DEliseenkova%26aufirst%3DA.%2BV.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DMiller%26aufirst%3DW.%2BT.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DA%2520molecular%2520brake%2520in%2520the%2520kinase%2520hinge%2520region%2520regulates%2520the%2520activity%2520of%2520receptor%2520tyrosine%2520kinases%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D717%26epage%3D730%26doi%3D10.1016%2Fj.molcel.2007.06.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawyers, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varmus, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">Inhibition of drug-resistant mutants of ABL, KIT, and EGF receptor kinases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">11011</span><span class="NLM_x">–</span> <span class="NLM_lpage">11016</span><span class="refDoi"> DOI: 10.1073/pnas.0504952102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1073%2Fpnas.0504952102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=16046538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnvVWjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=11011-11016&author=T.+A.+Carterauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=A.+M.+Velascoauthor=M.+A.+Fabianauthor=D.+K.+Treiberauthor=Z.+V.+Milanovauthor=C.+E.+Atteridgeauthor=W.+H.+Biggsauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=S.+A.+Mehtaauthor=H.+K.+Patelauthor=W.+Paoauthor=C.+L.+Sawyersauthor=H.+Varmusauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=Inhibition+of+drug-resistant+mutants+of+ABL%2C+KIT%2C+and+EGF+receptor+kinases&doi=10.1073%2Fpnas.0504952102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of drug-resistant mutants of ABL, KIT, and EGF receptor kinases</span></div><div class="casAuthors">Carter, Todd A.; Wodicka, Lisa M.; Shah, Neil P.; Velasco, Anne Marie; Fabian, Miles A.; Treiber, Daniel K.; Milanov, Zdravko V.; Atteridge, Corey E.; Biggs, William H., III; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Mehta, Shamal A.; Patel, Hitesh K.; Pao, William; Sawyers, Charles L.; Varmus, Harold; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">11011-11016</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">To realize the full potential of targeted protein kinase inhibitors for the treatment of cancer, it is important to address the emergence of drug resistance in treated patients.  Mutant forms of BCR-ABL, KIT, and the EGF receptor (EGFR) have been found that confer resistance to the drugs imatinib, gefitinib, and erlotinib.  The mutations weaken or prevent drug binding, and interestingly, one of the most common sites of mutation in all three kinases is a highly conserved "gatekeeper" threonine residue near the kinase active site.  We have identified existing clin. compds. that bind and inhibit drug-resistant mutant variants of ABL, KIT, and EGFR.  We found that the Aurora kinase inhibitor VX-680 and the p38 inhibitor BIRB-796 inhibit the imatinib- and BMS-354825-resistant ABL(T315I) kinase.  The KIT/FLT3 inhibitor SU-11248 potently inhibits the imatinib-resistant KIT(V559D/T670I) kinase, consistent with the clin. efficacy of SU-11248 against imatinib-resistant gastrointestinal tumors, and the EGFR inhibitors EKB-569 and Cl-1033, but not GW-572016 and ZD-6474, potently inhibit the gefitinib- and erlotinib-resistant EGFR(L858R/T790M) kinase.  EKB-569 and Cl-1033 are already in clin. trials, and our results suggest that they should be considered for testing in the treatment of gefitinib/erlotinib-resistant non-small cell lung cancer.  The results highlight the strategy of screening existing clin. compds. against newly identified drug-resistant mutant variants to find compds. that may serve as starting points for the development of next-generation drugs, or that could be used directly to treat patients that have acquired resistance to first-generation targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdgFgcC03tybVg90H21EOLACvtfcHk0lha9FX--T6log"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnvVWjurg%253D&md5=448c95c51b0cf9b5035df4065cd8f62c</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0504952102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0504952102%26sid%3Dliteratum%253Aachs%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPao%26aufirst%3DW.%26aulast%3DSawyers%26aufirst%3DC.%2BL.%26aulast%3DVarmus%26aufirst%3DH.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DInhibition%2520of%2520drug-resistant%2520mutants%2520of%2520ABL%252C%2520KIT%252C%2520and%2520EGF%2520receptor%2520kinases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D11011%26epage%3D11016%26doi%3D10.1073%2Fpnas.0504952102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Foda, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeliger, M. A.</span><span> </span><span class="NLM_article-title">Kinase inhibitors: an allosteric add-on</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">796</span><span class="NLM_x">–</span> <span class="NLM_lpage">797</span><span class="refDoi"> DOI: 10.1038/nchembio.1630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnchembio.1630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25195010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKms7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=796-797&author=Z.+H.+Fodaauthor=M.+A.+Seeliger&title=Kinase+inhibitors%3A+an+allosteric+add-on&doi=10.1038%2Fnchembio.1630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors - An allosteric add-on</span></div><div class="casAuthors">Foda, Zachariah H.; Seeliger, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">796-797</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Development of highly specific kinase inhibitors has been a long-standing challenge in chem. biol.  The structural and mechanistic characterization of an Erk1/2 kinase inhibitor provides new strategies to develop specific kinase inhibitors by targeting a binding pocket adjacent to the ATP binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoPgAa-_d1sLVg90H21EOLACvtfcHk0lgZDV9bC1Xq-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKms7vM&md5=40ce7d0360a5c59d73c8dd5cda0af412</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1630%26sid%3Dliteratum%253Aachs%26aulast%3DFoda%26aufirst%3DZ.%2BH.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26atitle%3DKinase%2520inhibitors%253A%2520an%2520allosteric%2520add-on%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D796%26epage%3D797%26doi%3D10.1038%2Fnchembio.1630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausen, M. H.</span><span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">422</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0lgZDV9bC1Xq-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Cadranel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruppert, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beau-Faller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wislez, M.</span><span> </span><span class="NLM_article-title">Therapeutic strategy for advanced EGFR mutant non-small-cell lung carcinoma</span> <span class="citation_source-journal">Crit. Rev. Oncol. Hematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">477</span><span class="NLM_x">–</span> <span class="NLM_lpage">493</span><span class="refDoi"> DOI: 10.1016/j.critrevonc.2013.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.critrevonc.2013.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=23911281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BC3sfmt12ksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2013&pages=477-493&author=J.+Cadranelauthor=A.+M.+Ruppertauthor=M.+Beau-Fallerauthor=M.+Wislez&title=Therapeutic+strategy+for+advanced+EGFR+mutant+non-small-cell+lung+carcinoma&doi=10.1016%2Fj.critrevonc.2013.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic strategy for advanced EGFR mutant non-small-cell lung carcinoma</span></div><div class="casAuthors">Cadranel Jacques; Ruppert Anne-Marie; Beau-Faller Michele; Wislez Marie</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncology/hematology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">477-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Activating mutation in exons 19 or 21 of epidermal growth factor receptor (EGFR) in non-small-cell lung cancers (NSCLC) are associated with increased sensitivity to EGFR tyrosine kinase inhibitors (EGFR-TKIs), such as gefitinib and erlotinib.  Cancer patients harboring activating EGFR mutations benefit from first-line TKI therapy.  Yet 10% of patients present a primary TKI resistance, while 50% of the others develop a secondary resistance within 9-12 months after starting TKI.  The RECIST's definition of progression appears flawed when applied to EGFR-mutated NSCLC patients.  Most often, tumor volume shrinking widely exceeds 30% during TKI response and kinetics of growth is low during relapse.  At present, secondary resistance mechanisms associated with progression are better known: clonal selection of EGFR resistance mutation (T790M mutation in exon 20), amplification of transmembrane receptors for other growth factors (c-met, HER family, IGF1R, or AXL), downstream molecular alterations in EGFR signaling pathway (PI3K or PTEN), and epithelial-mesenchymal transition or transdifferentiation to small-cell cancer.  The best strategy for secondary resistance is not well-defined: maintaining TKI therapy, switching to chemotherapy, combining both treatments, or using new therapies targeting other signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT9fVHNxWBIVa91pGlMMoT5fW6udTcc2ebn3DxwZGSX97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfmt12ksQ%253D%253D&md5=202252954321dde8fa5aa196492a10e7</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.critrevonc.2013.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.critrevonc.2013.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DCadranel%26aufirst%3DJ.%26aulast%3DRuppert%26aufirst%3DA.%2BM.%26aulast%3DBeau-Faller%26aufirst%3DM.%26aulast%3DWislez%26aufirst%3DM.%26atitle%3DTherapeutic%2520strategy%2520for%2520advanced%2520EGFR%2520mutant%2520non-small-cell%2520lung%2520carcinoma%26jtitle%3DCrit.%2520Rev.%2520Oncol.%2520Hematol.%26date%3D2013%26volume%3D88%26spage%3D477%26epage%3D493%26doi%3D10.1016%2Fj.critrevonc.2013.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Engel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richters, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomassi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keul, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Termathe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lategahn, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayer-Wrangowski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grutter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uhlenbrock, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krull, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaumann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eppmann, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kibies, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffgaard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menninger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz-Cuaran, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heuckmann, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tinnefeld, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahedi, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sos, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz-Fademrecht, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kast, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span> </span><span class="NLM_article-title">Targeting drug resistance in EGFR with covalent inhibitors: a structure-based design approach</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">6844</span><span class="NLM_x">–</span> <span class="NLM_lpage">6863</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01082</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01082" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCjsrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6844-6863&author=J.+Engelauthor=A.+Richtersauthor=M.+Getlikauthor=S.+Tomassiauthor=M.+Keulauthor=M.+Termatheauthor=J.+Lategahnauthor=C.+Beckerauthor=S.+Mayer-Wrangowskiauthor=C.+Grutterauthor=N.+Uhlenbrockauthor=J.+Krullauthor=N.+Schaumannauthor=S.+Eppmannauthor=P.+Kibiesauthor=F.+Hoffgaardauthor=J.+Heilauthor=S.+Menningerauthor=S.+Ortiz-Cuaranauthor=J.+M.+Heuckmannauthor=V.+Tinnefeldauthor=R.+P.+Zahediauthor=M.+L.+Sosauthor=C.+Schultz-Fademrechtauthor=R.+K.+Thomasauthor=S.+M.+Kastauthor=D.+Rauh&title=Targeting+drug+resistance+in+EGFR+with+covalent+inhibitors%3A+a+structure-based+design+approach&doi=10.1021%2Facs.jmedchem.5b01082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Drug Resistance in EGFR with Covalent Inhibitors: A Structure-Based Design Approach</span></div><div class="casAuthors">Engel, Julian; Richters, Andre; Getlik, Matthaeus; Tomassi, Stefano; Keul, Marina; Termathe, Martin; Lategahn, Jonas; Becker, Christian; Mayer-Wrangowski, Svenja; Gruetter, Christian; Uhlenbrock, Niklas; Kruell, Jasmin; Schaumann, Niklas; Eppmann, Simone; Kibies, Patrick; Hoffgaard, Franziska; Heil, Jochen; Menninger, Sascha; Ortiz-Cuaran, Sandra; Heuckmann, Johannes M.; Tinnefeld, Verena; Zahedi, Rene P.; Sos, Martin L.; Schultz-Fademrecht, Carsten; Thomas, Roman K.; Kast, Stefan M.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6844-6863</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Receptor tyrosine kinases represent one of the prime targets in cancer therapy, as the dysregulation of these elementary transducers of extracellular signals, like the epidermal growth factor receptor (EGFR), contributes to the onset of cancer, such as non-small cell lung cancer (NSCLC).  Strong efforts were directed to the development of irreversible inhibitors and led to compd. CO-1686, which takes advantage of increased residence time at EGFR by alkylating Cys797 and thereby preventing toxic effects.  Here, the authors present a structure-based approach, rationalized by subsequent computational anal. of conformational ligand ensembles in soln., to design novel and irreversible EGFR inhibitors based on a screening hit that was identified in a phenotype screen of 80 NSCLC cell lines against approx. 1500 compds.  Using protein x-ray crystallog., the authors deciphered the binding mode in engineered cSrc (T338M/S345C), a validated model system for EGFR-T790M, which constituted the basis for further rational design approaches.  Chem. synthesis led to further compd. collections that revealed increased biochem. potency and, in part, selectivity toward mutated (L858R and L858R/T790M) vs. nonmutated EGFR.  Further cell-based and kinetic studies were performed to substantiate the authors initial findings.  Utilizing proteolytic digestion and nano-LC-MS/MS anal., the authors confirmed the alkylation of Cys797.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNGJPIuqoVObVg90H21EOLACvtfcHk0lj60DE_9-36ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCjsrvM&md5=d5264a2a99d7752af0e587f23aeb7723</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01082%26sid%3Dliteratum%253Aachs%26aulast%3DEngel%26aufirst%3DJ.%26aulast%3DRichters%26aufirst%3DA.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DTomassi%26aufirst%3DS.%26aulast%3DKeul%26aufirst%3DM.%26aulast%3DTermathe%26aufirst%3DM.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DBecker%26aufirst%3DC.%26aulast%3DMayer-Wrangowski%26aufirst%3DS.%26aulast%3DGrutter%26aufirst%3DC.%26aulast%3DUhlenbrock%26aufirst%3DN.%26aulast%3DKrull%26aufirst%3DJ.%26aulast%3DSchaumann%26aufirst%3DN.%26aulast%3DEppmann%26aufirst%3DS.%26aulast%3DKibies%26aufirst%3DP.%26aulast%3DHoffgaard%26aufirst%3DF.%26aulast%3DHeil%26aufirst%3DJ.%26aulast%3DMenninger%26aufirst%3DS.%26aulast%3DOrtiz-Cuaran%26aufirst%3DS.%26aulast%3DHeuckmann%26aufirst%3DJ.%2BM.%26aulast%3DTinnefeld%26aufirst%3DV.%26aulast%3DZahedi%26aufirst%3DR.%2BP.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DThomas%26aufirst%3DR.%2BK.%26aulast%3DKast%26aufirst%3DS.%2BM.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DTargeting%2520drug%2520resistance%2520in%2520EGFR%2520with%2520covalent%2520inhibitors%253A%2520a%2520structure-based%2520design%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6844%26epage%3D6863%26doi%3D10.1021%2Facs.jmedchem.5b01082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Siegelin, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borczuk, A. C.</span><span> </span><span class="NLM_article-title">Epidermal growth factor receptor mutations in lung adenocarcinoma</span> <span class="citation_source-journal">Lab. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span><span class="refDoi"> DOI: 10.1038/labinvest.2013.147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Flabinvest.2013.147" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2014&pages=129-137&author=M.+D.+Siegelinauthor=A.+C.+Borczuk&title=Epidermal+growth+factor+receptor+mutations+in+lung+adenocarcinoma&doi=10.1038%2Flabinvest.2013.147"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Flabinvest.2013.147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Flabinvest.2013.147%26sid%3Dliteratum%253Aachs%26aulast%3DSiegelin%26aufirst%3DM.%2BD.%26aulast%3DBorczuk%26aufirst%3DA.%2BC.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520mutations%2520in%2520lung%2520adenocarcinoma%26jtitle%3DLab.%2520Invest.%26date%3D2014%26volume%3D94%26spage%3D129%26epage%3D137%26doi%3D10.1038%2Flabinvest.2013.147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Greig, S. L.</span><span> </span><span class="NLM_article-title">Osimertinib: First Global Approval</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">273</span><span class="refDoi"> DOI: 10.1007/s40265-015-0533-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1007%2Fs40265-015-0533-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=26729184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC28XlvFymtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=263-273&author=S.+L.+Greig&title=Osimertinib%3A+First+Global+Approval&doi=10.1007%2Fs40265-015-0533-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Osimertinib: First Global Approval</span></div><div class="casAuthors">Greig, Sarah L.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-273</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">Osimertinib (Tagrisso, AZD9291) is an oral, third-generation epidermal growth factor receptor tyrosine kinase inhibitor (EGFR TKI) that is being developed by AstraZeneca for the treatment of advanced non-small cell lung cancer (NSCLC).  Osimertinib has been designed to target the EGFR T790M mutation that is often present in NSCLC patients with acquired EGFR TKI resistance, while sparing wild-type EGFR.  In Nov. 2015, the tablet formulation of osimertinib was granted accelerated approval in the USA for the treatment of patients with metastatic EGFR T790M mutation-pos. NSCLC (as detected by an FDA-approved test) who have progressed on or after EGFR TKI therapy.  Osimertinib has also been granted accelerated assessment status for this indication in the EU, and is in phase III development for first- and second-line and adjuvant treatment of advanced EGFR mutation-pos. NSCLC in several countries.  Phase I trials in patients with advanced solid tumors are also being conducted.  This article summarizes the milestones in the development of osimertinib leading to this first approval for NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD-OV6l6JC5rVg90H21EOLACvtfcHk0lj60DE_9-36ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlvFymtA%253D%253D&md5=fb46898918bdfe76320c26177710fb11</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1007%2Fs40265-015-0533-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-015-0533-4%26sid%3Dliteratum%253Aachs%26aulast%3DGreig%26aufirst%3DS.%2BL.%26atitle%3DOsimertinib%253A%2520First%2520Global%2520Approval%26jtitle%3DDrugs%26date%3D2016%26volume%3D76%26spage%3D263%26epage%3D273%26doi%3D10.1007%2Fs40265-015-0533-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Walter, A. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjin, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haringsma, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubrovskiy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labenski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheets, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St Martin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karp, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kalken, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaturvedi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nacht, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westlin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaw-Tsai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raponi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Dyke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Etter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmons, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harding, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, A.</span><span> </span><span class="NLM_article-title">Discovery of a mutant-selective covalent inhibitor of EGFR that overcomes T790M-mediated resistance in NSCLC</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1404</span><span class="NLM_x">–</span> <span class="NLM_lpage">1415</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-13-0314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1158%2F2159-8290.CD-13-0314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=24065731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2rtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=1404-1415&author=A.+O.+Walterauthor=R.+T.+Sjinauthor=H.+J.+Haringsmaauthor=K.+Ohashiauthor=J.+Sunauthor=K.+Leeauthor=A.+Dubrovskiyauthor=M.+Labenskiauthor=Z.+Zhuauthor=Z.+Wangauthor=M.+Sheetsauthor=T.+St+Martinauthor=R.+Karpauthor=D.+van+Kalkenauthor=P.+Chaturvediauthor=D.+Niuauthor=M.+Nachtauthor=R.+C.+Petterauthor=W.+Westlinauthor=K.+Linauthor=S.+Jaw-Tsaiauthor=M.+Raponiauthor=T.+Van+Dykeauthor=J.+Etterauthor=Z.+Weaverauthor=W.+Paoauthor=J.+Singhauthor=A.+D.+Simmonsauthor=T.+C.+Hardingauthor=A.+Allen&title=Discovery+of+a+mutant-selective+covalent+inhibitor+of+EGFR+that+overcomes+T790M-mediated+resistance+in+NSCLC&doi=10.1158%2F2159-8290.CD-13-0314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Mutant-Selective Covalent Inhibitor of EGFR that Overcomes T790M-Mediated Resistance in NSCLC</span></div><div class="casAuthors">Walter, Annette O.; Sjin, Robert Tjin Tham; Haringsma, Henry J.; Ohashi, Kadoaki; Sun, Jing; Lee, Kwangho; Dubrovskiy, Aleksandr; Labenski, Matthew; Zhu, Zhendong; Wang, Zhigang; Sheets, Michael; St. Martin, Thia; Karp, Russell; van Kalken, Dan; Chaturvedi, Prasoon; Niu, Deqiang; Nacht, Mariana; Petter, Russell C.; Westlin, William; Lin, Kevin; Jaw-Tsai, Sarah; Raponi, Mitch; Van Dyke, Terry; Etter, Jeff; Weaver, Zoe; Pao, William; Singh, Juswinder; Simmons, Andrew D.; Harding, Thomas C.; Allen, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1404-1415</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Patients with non-small cell lung cancer (NSCLC) with activating EGF receptor (EGFR) mutations initially respond to first-generation reversible EGFR tyrosine kinase inhibitors.  However, clin. efficacy is limited by acquired resistance, frequently driven by the EGFRT790M mutation.  CO-1686 is a novel, irreversible, and orally delivered kinase inhibitor that specifically targets the mutant forms of EGFR, including T790M, while exhibiting minimal activity toward the wild-type (WT) receptor.  Oral administration of CO-1686 as single agent induces tumor regression in EGFR-mutated NSCLC tumor xenograft and transgenic models.  Minimal activity of CO-1686 against the WT EGFR receptor was obsd.  In NSCLC cells with acquired resistance to CO-1686 in vitro, there was no evidence of addnl. mutations or amplification of the EGFR gene, but resistant cells exhibited signs of epithelial-mesenchymal transition and demonstrated increased sensitivity to AKT inhibitors.  These results suggest that CO-1686 may offer a novel therapeutic option for patients with mutant EGFR NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR0CDWQZ5wn7Vg90H21EOLACvtfcHk0lgU-xh8PgCuZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2rtLvN&md5=3d111f4bf958f2fda6890fbc0e1d99f5</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-13-0314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-13-0314%26sid%3Dliteratum%253Aachs%26aulast%3DWalter%26aufirst%3DA.%2BO.%26aulast%3DSjin%26aufirst%3DR.%2BT.%26aulast%3DHaringsma%26aufirst%3DH.%2BJ.%26aulast%3DOhashi%26aufirst%3DK.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DDubrovskiy%26aufirst%3DA.%26aulast%3DLabenski%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DSheets%26aufirst%3DM.%26aulast%3DSt%2BMartin%26aufirst%3DT.%26aulast%3DKarp%26aufirst%3DR.%26aulast%3Dvan%2BKalken%26aufirst%3DD.%26aulast%3DChaturvedi%26aufirst%3DP.%26aulast%3DNiu%26aufirst%3DD.%26aulast%3DNacht%26aufirst%3DM.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DWestlin%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DJaw-Tsai%26aufirst%3DS.%26aulast%3DRaponi%26aufirst%3DM.%26aulast%3DVan%2BDyke%26aufirst%3DT.%26aulast%3DEtter%26aufirst%3DJ.%26aulast%3DWeaver%26aufirst%3DZ.%26aulast%3DPao%26aufirst%3DW.%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DSimmons%26aufirst%3DA.%2BD.%26aulast%3DHarding%26aufirst%3DT.%2BC.%26aulast%3DAllen%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520a%2520mutant-selective%2520covalent%2520inhibitor%2520of%2520EGFR%2520that%2520overcomes%2520T790M-mediated%2520resistance%2520in%2520NSCLC%26jtitle%3DCancer%2520Discovery%26date%3D2013%26volume%3D3%26spage%3D1404%26epage%3D1415%26doi%3D10.1158%2F2159-8290.CD-13-0314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelletti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortot, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirieac, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacob, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padera, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engen, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span> </span><span class="NLM_article-title">Novel mutant-selective EGFR kinase inhibitors against EGFR T790M</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">1070</span><span class="NLM_x">–</span> <span class="NLM_lpage">1074</span><span class="refDoi"> DOI: 10.1038/nature08622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnature08622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=20033049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SktbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=1070-1074&author=W.+Zhouauthor=D.+Ercanauthor=L.+Chenauthor=C.+H.+Yunauthor=D.+Liauthor=M.+Capellettiauthor=A.+B.+Cortotauthor=L.+Chirieacauthor=R.+E.+Iacobauthor=R.+Paderaauthor=J.+R.+Engenauthor=K.+K.+Wongauthor=M.+J.+Eckauthor=N.+S.+Grayauthor=P.+A.+Janne&title=Novel+mutant-selective+EGFR+kinase+inhibitors+against+EGFR+T790M&doi=10.1038%2Fnature08622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Novel mutant-selective EGFR kinase inhibitors against EGFR T790M</span></div><div class="casAuthors">Zhou, Wenjun; Ercan, Dalia; Chen, Liang; Yun, Cai-Hong; Li, Danan; Capelletti, Marzia; Cortot, Alexis B.; Chirieac, Lucian; Iacob, Roxana E.; Padera, Robert; Engen, John R.; Wong, Kwok-Kin; Eck, Michael J.; Gray, Nathanael S.; Janne, Pasi A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7276</span>),
    <span class="NLM_cas:pages">1070-1074</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy of epidermal growth factor receptor (EGFR) kinase inhibitors in EGFR-mutant non-small-cell lung cancer (NSCLC) is limited by the development of drug-resistance mutations, including the gatekeeper T790M mutation.  Strategies targeting EGFR T790M with irreversible inhibitors have had limited success and are assocd. with toxicity due to concurrent inhibition of wild-type EGFR.  All current EGFR inhibitors possess a structurally related quinazoline-based core scaffold and were identified as ATP-competitive inhibitors of wild-type EGFR.  Here we identify a covalent pyrimidine EGFR inhibitor by screening an irreversible kinase inhibitor library specifically against EGFR T790M.  These agents are 30- to 100-fold more potent against EGFR T790M, and up to 100-fold less potent against wild-type EGFR, than quinazoline-based EGFR inhibitors in vitro.  They are also effective in murine models of lung cancer driven by EGFR T790M.  Co-crystn. studies reveal a structural basis for the increased potency and mutant selectivity of these agents.  These mutant-selective irreversible EGFR kinase inhibitors may be clin. more effective and better tolerated than quinazoline-based inhibitors.  Our findings demonstrate that functional pharmacol. screens against clin. important mutant kinases represent a powerful strategy to identify new classes of mutant-selective kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcksZlIe-1bbVg90H21EOLACvtfcHk0lgU-xh8PgCuZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1SktbvI&md5=9b5d41a2be401c67a4b9f04a41cf1725</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fnature08622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08622%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DCortot%26aufirst%3DA.%2BB.%26aulast%3DChirieac%26aufirst%3DL.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DPadera%26aufirst%3DR.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26atitle%3DNovel%2520mutant-selective%2520EGFR%2520kinase%2520inhibitors%2520against%2520EGFR%2520T790M%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D1070%26epage%3D1074%26doi%3D10.1038%2Fnature08622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Adams, J. A.</span><span> </span><span class="NLM_article-title">Kinetic and catalytic mechanisms of protein kinases</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">2271</span><span class="NLM_x">–</span> <span class="NLM_lpage">2290</span><span class="refDoi"> DOI: 10.1021/cr000230w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000230w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvV2qtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2001&pages=2271-2290&author=J.+A.+Adams&title=Kinetic+and+catalytic+mechanisms+of+protein+kinases&doi=10.1021%2Fcr000230w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and catalytic mechanisms of protein kinases</span></div><div class="casAuthors">Adams, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2271-2290</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 178 refs.  The vertebrate genome is estd. to contain as many as 2000 protein kinases to carry out a variety of biol. functions.  Although the phosphoryl transfer mechanism is chem. simple, the phosphorylation of protein substrates in the active site of a protein kinase is complex and involves structural changes.  A conserved catalytic core region of approx. 200-250 amino acids is shared within the protein kinase family, but this core structure frequently interacts with regulatory proteins or domains that either enhance or repress catalytic function.  This structural hierarchy offers an extraordinary level of complexity to the enzyme family and makes functional studies of catalysis and regulation intricate.  While the noncatalytic domains and subunits affect kinase function, in the interest of brevity the review focuses largely on the core structure.  Although data are reviewed from many different enzymes within the protein kinase family, emphasis is placed on cAMP-dependent protein kinase A (PKA), which is by far the best understood protein kinase from a mechanistic viewpoint.  PKA consists of a heterotetramer composed of 2 catalytic (C) subunits and a regulatory dimer.  Owing to the highly conserved nature of the catalytic core structure, a description of PKA encompasses the main features of all members of the protein kinase family.  The C subunit of PKA is comprised of 2 domains, including a small ATP-binding domain composed primarily of α-helixes and a larger substrate-binding domain composed mostly of β-strands.  The 2 domains are connected by a small linker region to generate the binding pocket for ATP and the substrate.  Whereas the ATP is nestled deep between the domains, the 20-residue inhibitory peptide (PKI) resides on the periphery of the pocket, and the activation peptide with its essential PKA phosphorylation site, Thr-197, is also nearby.  The review focuses on approaches taken to assess the catalytic and kinetic mechanisms of protein phosphorylation catalyzed by protein kinases, and concs. its attention on the roles of active site residues, metal cations, conformational changes, and regulatory phosphorylation in the processing of peptide and protein substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooLsgkSVlhcLVg90H21EOLACvtfcHk0lgU-xh8PgCuZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvV2qtLo%253D&md5=8e3a97ed93e74df275a166b003f1757a</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fcr000230w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000230w%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DJ.%2BA.%26atitle%3DKinetic%2520and%2520catalytic%2520mechanisms%2520of%2520protein%2520kinases%26jtitle%3DChem.%2520Rev.%26date%3D2001%26volume%3D101%26spage%3D2271%26epage%3D2290%26doi%3D10.1021%2Fcr000230w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nolop, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, P.</span><span> </span><span class="NLM_article-title">Vemurafenib: the first drug approved for BRAF-mutant cancer</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">886</span><span class="refDoi"> DOI: 10.1038/nrd3847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnrd3847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=23060265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=873-886&author=G.+Bollagauthor=J.+Tsaiauthor=J.+Zhangauthor=C.+Zhangauthor=P.+Ibrahimauthor=K.+Nolopauthor=P.+Hirth&title=Vemurafenib%3A+the+first+drug+approved+for+BRAF-mutant+cancer&doi=10.1038%2Fnrd3847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Vemurafenib: the first drug approved for BRAF-mutant cancer</span></div><div class="casAuthors">Bollag, Gideon; Tsai, James; Zhang, Jiazhong; Zhang, Chao; Ibrahim, Prabha; Nolop, Keith; Hirth, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">873-886</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The identification of driver oncogenes has provided important targets for drugs that can change the landscape of cancer therapies.  One such example is the BRAF oncogene, which is found in about half of all melanomas as well as several other cancers.  As a druggable kinase, oncogenic BRAF has become a crucial target of small-mol. drug discovery efforts.  Following a rapid clin. development path, vemurafenib (Zelboraf; Plexxikon/Roche) was approved for the treatment of BRAF-mutated metastatic melanoma in the United States in August 2011 and the European Union in Feb. 2012.  This Review describes the underlying biol. of BRAF, the technol. used to identify vemurafenib and its clin. development milestones, along with future prospects based on lessons learned during its development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT6Fbb43oRTbVg90H21EOLACvtfcHk0ljcuodWt_DXHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL&md5=b1b38cad542be9b3a30060b77c584bc1</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1038%2Fnrd3847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3847%26sid%3Dliteratum%253Aachs%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DNolop%26aufirst%3DK.%26aulast%3DHirth%26aufirst%3DP.%26atitle%3DVemurafenib%253A%2520the%2520first%2520drug%2520approved%2520for%2520BRAF-mutant%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D873%26epage%3D886%26doi%3D10.1038%2Fnrd3847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Shaw, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felip, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, L. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camidge, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vansteenkiste, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Pas, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riely, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santoro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldwasser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boral, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, J. A.</span><span> </span><span class="NLM_article-title">Ceritinib in ALK-rearranged non-small-cell lung cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">370</span><span class="NLM_x">, </span> <span class="NLM_fpage">1189</span><span class="NLM_x">–</span> <span class="NLM_lpage">1197</span><span class="refDoi"> DOI: 10.1056/NEJMoa1311107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1056%2FNEJMoa1311107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=24670165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvVCitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2014&pages=1189-1197&author=A.+T.+Shawauthor=D.+W.+Kimauthor=R.+Mehraauthor=D.+S.+Tanauthor=E.+Felipauthor=L.+Q.+Chowauthor=D.+R.+Camidgeauthor=J.+Vansteenkisteauthor=S.+Sharmaauthor=T.+De+Pasauthor=G.+J.+Rielyauthor=B.+J.+Solomonauthor=J.+Wolfauthor=M.+Thomasauthor=M.+Schulerauthor=G.+Liuauthor=A.+Santoroauthor=Y.+Y.+Lauauthor=M.+Goldwasserauthor=A.+L.+Boralauthor=J.+A.+Engelman&title=Ceritinib+in+ALK-rearranged+non-small-cell+lung+cancer&doi=10.1056%2FNEJMoa1311107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Ceritinib in ALK-rearranged non-small-cell lung cancer</span></div><div class="casAuthors">Shaw, Alice T.; Kim, Dong-Wan; Mehra, Ranee; Tan, Daniel S. W.; Felip, Enriqueta; Chow, Laura Q. M.; Camidge, Ross; Vansteenkiste, Johan; Sharma, Sunil; De Pas, Tommaso; Riely, Gregory J.; Solomon, Benjamin J.; Wolf, Juergen; Thomas, Michael; Schuler, Martin; Liu, Geoffrey; Santoro, Armando; Lau, Yvonne Y.; Goldwasser, Meredith; Boral, Anthony L.; Engelman, Jeffrey A.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">370</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1189-1197</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background: Non-small-cell lung cancer (NSCLC) harboring the anaplastic lymphoma kinase gene (ALK) rearrangement is sensitive to the ALK inhibitor crizotinib, but resistance invariably develops.  Ceritinib (LDK378) is a new ALK inhibitor that has shown greater antitumor potency than crizotinib in preclin. studies.  Methods: In this phase 1 study, we administered oral ceritinib in doses of 50 to 750 mg once daily to patients with advanced cancers harboring genetic alterations in ALK.  In an expansion phase of the study, patients received the max. tolerated dose.  Patients were assessed to det. the safety, pharmacokinetic properties, and antitumor activity of ceritinib.  Tumor biopsies were performed before ceritinib treatment to identify resistance mutations in ALK in a group of patients with NSCLC who had had disease progression during treatment with crizotinib.  Results: A total of 59 patients were enrolled in the dose-escalation phase.  The max. tolerated dose of ceritinib was 750 mg once daily; dose-limiting toxic events included diarrhea, vomiting, dehydration, elevated aminotransferase levels, and hypophosphatemia.  This phase was followed by an expansion phase, in which an addnl. 71 patients were treated, for a total of 130 patients overall.  Among 114 patients with NSCLC who received at least 400 mg of ceritinib per day, the overall response rate was 58% (95% confidence interval [CI], 48 to 67).  Among 80 patients who had received crizotinib previously, the response rate was 56% (95% CI, 45 to 67).  Responses were obsd. in patients with various resistance mutations in ALK and in patients without detectable mutations.  Among patients with NSCLC who received at least 400 mg of ceritinib per day, the median progression-free survival was 7.0 mo (95% CI, 5.6 to 9.5).  Conclusions: Ceritinib was highly active in patients with advanced, ALK-rearranged NSCLC, including those who had had disease progression during crizotinib treatment, regardless of the presence of resistance mutations in ALK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxuct3wv2aF7Vg90H21EOLACvtfcHk0ljcuodWt_DXHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvVCitL0%253D&md5=dff3054469948ceb26a00e3eccac56c1</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1311107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1311107%26sid%3Dliteratum%253Aachs%26aulast%3DShaw%26aufirst%3DA.%2BT.%26aulast%3DKim%26aufirst%3DD.%2BW.%26aulast%3DMehra%26aufirst%3DR.%26aulast%3DTan%26aufirst%3DD.%2BS.%26aulast%3DFelip%26aufirst%3DE.%26aulast%3DChow%26aufirst%3DL.%2BQ.%26aulast%3DCamidge%26aufirst%3DD.%2BR.%26aulast%3DVansteenkiste%26aufirst%3DJ.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DDe%2BPas%26aufirst%3DT.%26aulast%3DRiely%26aufirst%3DG.%2BJ.%26aulast%3DSolomon%26aufirst%3DB.%2BJ.%26aulast%3DWolf%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DSchuler%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DSantoro%26aufirst%3DA.%26aulast%3DLau%26aufirst%3DY.%2BY.%26aulast%3DGoldwasser%26aufirst%3DM.%26aulast%3DBoral%26aufirst%3DA.%2BL.%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26atitle%3DCeritinib%2520in%2520ALK-rearranged%2520non-small-cell%2520lung%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2014%26volume%3D370%26spage%3D1189%26epage%3D1197%26doi%3D10.1056%2FNEJMoa1311107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Fabbro, D.</span><span> </span><span class="NLM_article-title">25 years of small molecular weight kinase inhibitors: potentials and limitations</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">766</span><span class="NLM_x">–</span> <span class="NLM_lpage">775</span><span class="refDoi"> DOI: 10.1124/mol.114.095489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1124%2Fmol.114.095489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25549667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVKltLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2015&pages=766-775&author=D.+Fabbro&title=25+years+of+small+molecular+weight+kinase+inhibitors%3A+potentials+and+limitations&doi=10.1124%2Fmol.114.095489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">25 years of small molecular weight kinase inhibitors: potentials and limitations</span></div><div class="casAuthors">Fabbro, Doriano</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">766-775</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Deregulation of protein and lipid kinase activities leads to a variety of pathologies, ranging from cancer inflammatory diseases, diabetes, infectious diseases, and cardiovascular disorders.  Protein kinases and lipid kinases represent, therefore, an important target for the pharmaceutical industry.  In fact, approx. one-third of all protein targets under investigation in the pharmaceutical industry are protein or lipid kinases.  To date, 30 kinase inhibitors have been approved, which, with few exceptions, are mainly for oncol. indications and directed against only a handful of protein and lipid kinases, leaving 70% of the kinome untapped.  Despite these successes in kinase drug discovery, the development of kinase inhibitors with outstanding selectivity, identification and validation of driver kinase(s) in diseases, and the emerging problem of resistance to the inhibition of key target kinases remain major challenges.  This mini review provides an insight into protein and lipid kinase drug discovery with respect to achievements, binding modes of inhibitors, and novel avenues for the generation of second-generation kinase inhibitors to treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZhF-9NQJx7bVg90H21EOLACvtfcHk0ljcuodWt_DXHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVKltLk%253D&md5=6422b036e4e447379ac6c608662954c2</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.095489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.095489%26sid%3Dliteratum%253Aachs%26aulast%3DFabbro%26aufirst%3DD.%26atitle%3D25%2520years%2520of%2520small%2520molecular%2520weight%2520kinase%2520inhibitors%253A%2520potentials%2520and%2520limitations%26jtitle%3DMol.%2520Pharmacol.%26date%3D2015%26volume%3D87%26spage%3D766%26epage%3D775%26doi%3D10.1124%2Fmol.114.095489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Ohren, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pavlovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitehead, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuffa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McConnell, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spessard, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banotai, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delaney, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Omer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sebolt-Leopold, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dudley, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flamme, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warmus, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrett, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tecle, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasemann, C. A.</span><span> </span><span class="NLM_article-title">Structures of human MAP kinase kinase 1 (MEK1) and MEK2 describe novel noncompetitive kinase inhibition</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1192</span><span class="NLM_x">–</span> <span class="NLM_lpage">1197</span><span class="refDoi"> DOI: 10.1038/nsmb859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnsmb859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=15543157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1192-1197&author=J.+F.+Ohrenauthor=H.+Chenauthor=A.+Pavlovskyauthor=C.+Whiteheadauthor=E.+Zhangauthor=P.+Kuffaauthor=C.+Yanauthor=P.+McConnellauthor=C.+Spessardauthor=C.+Banotaiauthor=W.+T.+Muellerauthor=A.+Delaneyauthor=C.+Omerauthor=J.+Sebolt-Leopoldauthor=D.+T.+Dudleyauthor=I.+K.+Leungauthor=C.+Flammeauthor=J.+Warmusauthor=M.+Kaufmanauthor=S.+Barrettauthor=H.+Tecleauthor=C.+A.+Hasemann&title=Structures+of+human+MAP+kinase+kinase+1+%28MEK1%29+and+MEK2+describe+novel+noncompetitive+kinase+inhibition&doi=10.1038%2Fnsmb859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of human MAP kinase kinase 1 (MEK1) and MEK2 describe novel noncompetitive kinase inhibition</span></div><div class="casAuthors">Ohren, Jeffrey F.; Chen, Huifen; Pavlovsky, Alexander; Whitehead, Christopher; Zhang, Erli; Kuffa, Peter; Yan, Chunhong; McConnell, Patrick; Spessard, Cindy; Banotai, Craig; Mueller, W. Thomas; Delaney, Amy; Omer, Charles; Sebolt-Leopold, Judith; Dudley, David T.; Leung, Iris K.; Flamme, Cathlin; Warmus, Joseph; Kaufman, Michael; Barrett, Stephen; Tecle, Haile; Hasemann, Charles A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1192-1197</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MEK1 and MEK2 kinases are closely related, dual-specificity tyrosine/threonine protein kinases found in the Ras/Raf/MEK/ERK MAP kinase (MAPK) signaling pathway.  Approx. 30% of all human cancers have a constitutively activated MAPK pathway, and constitutive activation of MEK1 results in cellular transformation.  Here, the authors present the x-ray crystal structures of human MEK1 and MEK2 kinases, each detd. as a ternary complex with MgATP and an inhibitor to a resoln. of 2.4 and 3.2 Å, resp.  The structures revealed that MEK1 and MEK2 each had a unique inhibitor-binding pocket adjacent to the MgATP-binding site.  The presence of the potent inhibitors induced several conformational changes in the unphosphorylated MEK1 and MEK2 kinases that locked them into a closed but catalytically inactive species.  Thus, the structures reported here reveal a novel, noncompetitive mechanism for protein kinase inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIY9begVVfW7Vg90H21EOLACvtfcHk0lhCWKgc47zPOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM&md5=79ebbc2e3a9cdd3fbecb8bb35ea9dba3</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1038%2Fnsmb859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb859%26sid%3Dliteratum%253Aachs%26aulast%3DOhren%26aufirst%3DJ.%2BF.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DPavlovsky%26aufirst%3DA.%26aulast%3DWhitehead%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DE.%26aulast%3DKuffa%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DMcConnell%26aufirst%3DP.%26aulast%3DSpessard%26aufirst%3DC.%26aulast%3DBanotai%26aufirst%3DC.%26aulast%3DMueller%26aufirst%3DW.%2BT.%26aulast%3DDelaney%26aufirst%3DA.%26aulast%3DOmer%26aufirst%3DC.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DDudley%26aufirst%3DD.%2BT.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DFlamme%26aufirst%3DC.%26aulast%3DWarmus%26aufirst%3DJ.%26aulast%3DKaufman%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DS.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DHasemann%26aufirst%3DC.%2BA.%26atitle%3DStructures%2520of%2520human%2520MAP%2520kinase%2520kinase%25201%2520%2528MEK1%2529%2520and%2520MEK2%2520describe%2520novel%2520noncompetitive%2520kinase%2520inhibition%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1192%26epage%3D1197%26doi%3D10.1038%2Fnsmb859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Huang, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metcalf, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundaramoorthi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lentini, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keats, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wardwell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ning, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snodgrass, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broudy, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russian, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Commodore, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narasimhan, N. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohemmad, Q. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iuliucci, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivera, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalgarno, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawyer, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clackson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakespeare, W. C.</span><span> </span><span class="NLM_article-title">Discovery of 3-[2-(imidazo[1,2-b]pyridazin-3-yl)ethynyl]-4-methyl-N-{4-[(4-methylpiperazin-1-y l)methyl]-3-(trifluoromethyl)phenyl}benzamide (AP24534), a potent, orally active pan-inhibitor of breakpoint cluster region-abelson (BCR-ABL) kinase including the T315I gatekeeper mutant</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">4701</span><span class="NLM_x">–</span> <span class="NLM_lpage">4719</span><span class="refDoi"> DOI: 10.1021/jm100395q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100395q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslels7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4701-4719&author=W.+S.+Huangauthor=C.+A.+Metcalfauthor=R.+Sundaramoorthiauthor=Y.+Wangauthor=D.+Zouauthor=R.+M.+Thomasauthor=X.+Zhuauthor=L.+Caiauthor=D.+Wenauthor=S.+Liuauthor=J.+Romeroauthor=J.+Qiauthor=I.+Chenauthor=G.+Bandaauthor=S.+P.+Lentiniauthor=S.+Dasauthor=Q.+Xuauthor=J.+Keatsauthor=F.+Wangauthor=S.+Wardwellauthor=Y.+Ningauthor=J.+T.+Snodgrassauthor=M.+I.+Broudyauthor=K.+Russianauthor=T.+Zhouauthor=L.+Commodoreauthor=N.+I.+Narasimhanauthor=Q.+K.+Mohemmadauthor=J.+Iuliucciauthor=V.+M.+Riveraauthor=D.+C.+Dalgarnoauthor=T.+K.+Sawyerauthor=T.+Clacksonauthor=W.+C.+Shakespeare&title=Discovery+of+3-%5B2-%28imidazo%5B1%2C2-b%5Dpyridazin-3-yl%29ethynyl%5D-4-methyl-N-%7B4-%5B%284-methylpiperazin-1-y+l%29methyl%5D-3-%28trifluoromethyl%29phenyl%7Dbenzamide+%28AP24534%29%2C+a+potent%2C+orally+active+pan-inhibitor+of+breakpoint+cluster+region-abelson+%28BCR-ABL%29+kinase+including+the+T315I+gatekeeper+mutant&doi=10.1021%2Fjm100395q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-[2-(Imidazo[1,2-b]pyridazin-3-yl)ethynyl]-4-methyl-N-[4-((4-methylpiperazin-1-yl)methyl)-3-(trifluoromethyl)phenyl]benzamide (AP24534), a Potent, Orally Active Pan-Inhibitor of Breakpoint Cluster Region-Abelson (BCR-ABL) Kinase Including the T315I Gatekeeper Mutant</span></div><div class="casAuthors">Huang, Wei-Sheng; Metcalf, Chester A.; Sundaramoorthi, Raji; Wang, Yihan; Zou, Dong; Thomas, R. Mathew; Zhu, Xiaotian; Cai, Lisi; Wen, David; Liu, Shuangying; Romero, Jan; Qi, Jiwei; Chen, Ingrid; Banda, Geetha; Lentini, Scott P.; Das, Sasmita; Xu, Qihong; Keats, Jeff; Wang, Frank; Wardwell, Scott; Ning, Yaoyu; Snodgrass, Joseph T.; Broudy, Marc I.; Russian, Karin; Zhou, Tianjun; Commodore, Lois; Narasimhan, Narayana I.; Mohemmad, Qurish K.; Iuliucci, John; Rivera, Victor M.; Dalgarno, David C.; Sawyer, Tomi K.; Clackson, Tim; Shakespeare, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4701-4719</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the treatment of chronic myeloid leukemia (CML) with BCR-ABL kinase inhibitors, the T315I gatekeeper mutant has emerged as resistant to all currently approved agents.  This report describes the structure-guided design of a novel series of potent pan-inhibitors of BCR-ABL, including the T315I mutation.  A key structural feature is the carbon-carbon triple bond linker which skirts the increased bulk of Ile315 side chain.  Extensive SAR studies led to the discovery of development candidate benzamide I (AP24534), which inhibited the kinase activity of both native BCR-ABL and the T315I mutant with low nM IC50s, and potently inhibited proliferation of corresponding Ba/F3-derived cell lines.  Daily oral administration of I significantly prolonged survival of mice injected i.v. with BCR-ABLT315I expressing Ba/F3 cells.  These data, coupled with a favorable ADME profile, support the potential of I to be an effective treatment for CML, including patients refractory to all currently approved therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoci2zVFUazgLVg90H21EOLACvtfcHk0lhCWKgc47zPOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslels7s%253D&md5=0dcd7b4ff31164b1d9883e897b7058ff</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fjm100395q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100395q%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%2BS.%26aulast%3DMetcalf%26aufirst%3DC.%2BA.%26aulast%3DSundaramoorthi%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DThomas%26aufirst%3DR.%2BM.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DL.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DRomero%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DBanda%26aufirst%3DG.%26aulast%3DLentini%26aufirst%3DS.%2BP.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DKeats%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWardwell%26aufirst%3DS.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DSnodgrass%26aufirst%3DJ.%2BT.%26aulast%3DBroudy%26aufirst%3DM.%2BI.%26aulast%3DRussian%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DNarasimhan%26aufirst%3DN.%2BI.%26aulast%3DMohemmad%26aufirst%3DQ.%2BK.%26aulast%3DIuliucci%26aufirst%3DJ.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26atitle%3DDiscovery%2520of%25203-%255B2-%2528imidazo%255B1%252C2-b%255Dpyridazin-3-yl%2529ethynyl%255D-4-methyl-N-%257B4-%255B%25284-methylpiperazin-1-y%2520l%2529methyl%255D-3-%2528trifluoromethyl%2529phenyl%257Dbenzamide%2520%2528AP24534%2529%252C%2520a%2520potent%252C%2520orally%2520active%2520pan-inhibitor%2520of%2520breakpoint%2520cluster%2520region-abelson%2520%2528BCR-ABL%2529%2520kinase%2520including%2520the%2520T315I%2520gatekeeper%2520mutant%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4701%26epage%3D4719%26doi%3D10.1021%2Fjm100395q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">O’Hare, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakespeare, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eide, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivera, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adrian, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metcalf, C. A.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyner, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loriaux, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corbin, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wardwell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ning, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keats, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundaramoorthi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snodgrass, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Commodore, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawyer, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalgarno, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deininger, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Druker, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clackson, T.</span><span> </span><span class="NLM_article-title">AP24534, a pan-BCR-ABL inhibitor for chronic myeloid leukemia, potently inhibits the T315I mutant and overcomes mutation-based resistance</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">412</span><span class="refDoi"> DOI: 10.1016/j.ccr.2009.09.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.ccr.2009.09.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19878872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFKltb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=401-412&author=T.+O%E2%80%99Hareauthor=W.+C.+Shakespeareauthor=X.+Zhuauthor=C.+A.+Eideauthor=V.+M.+Riveraauthor=F.+Wangauthor=L.+T.+Adrianauthor=T.+Zhouauthor=W.+S.+Huangauthor=Q.+Xuauthor=C.+A.+Metcalfauthor=J.+W.+Tynerauthor=M.+M.+Loriauxauthor=A.+S.+Corbinauthor=S.+Wardwellauthor=Y.+Ningauthor=J.+A.+Keatsauthor=Y.+Wangauthor=R.+Sundaramoorthiauthor=M.+Thomasauthor=D.+Zhouauthor=J.+Snodgrassauthor=L.+Commodoreauthor=T.+K.+Sawyerauthor=D.+C.+Dalgarnoauthor=M.+W.+Deiningerauthor=B.+J.+Drukerauthor=T.+Clackson&title=AP24534%2C+a+pan-BCR-ABL+inhibitor+for+chronic+myeloid+leukemia%2C+potently+inhibits+the+T315I+mutant+and+overcomes+mutation-based+resistance&doi=10.1016%2Fj.ccr.2009.09.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">AP24534, a pan-BCR-ABL inhibitor for chronic myeloid leukemia, potently inhibits the T315I mutant and overcomes mutation-based resistance</span></div><div class="casAuthors">O'Hare, Thomas; Shakespeare, William C.; Zhu, Xiaotian; Eide, Christopher A.; Rivera, Victor M.; Wang, Frank; Adrian, Lauren T.; Zhou, Tianjun; Huang, Wei-Sheng; Xu, Qihong; Metcalf, Chester A., III; Tyner, Jeffrey W.; Loriaux, Marc M.; Corbin, Amie S.; Wardwell, Scott; Ning, Yaoyu; Keats, Jeffrey A.; Wang, Yihan; Sundaramoorthi, Raji; Thomas, Mathew; Zhou, Dong; Snodgrass, Joseph; Commodore, Lois; Sawyer, Tomi K.; Dalgarno, David C.; Deininger, Michael W. N.; Druker, Brian J.; Clackson, Tim</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">401-412</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibition of BCR-ABL by imatinib induces durable responses in many patients with chronic myeloid leukemia (CML), but resistance attributable to kinase domain mutations can lead to relapse and a switch to second-line therapy with nilotinib or dasatinib.  Despite three approved therapeutic options, the cross-resistant BCR-ABLT315I mutation and compd. mutants selected on sequential inhibitor therapy remain major clin. challenges.  We report design and preclin. evaluation of AP24534, a potent, orally available multitargeted kinase inhibitor active against T315I and other BCR-ABL mutants.  AP24534 inhibited all tested BCR-ABL mutants in cellular and biochem. assays, suppressed BCR-ABLT315I-driven tumor growth in mice, and completely abrogated resistance in cell-based mutagenesis screens.  Our work supports clin. evaluation of AP24534 as a pan-BCR-ABL inhibitor for treatment of CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox3g7ZYNXwLbVg90H21EOLACvtfcHk0liBb4JBQrtqFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFKltb3N&md5=62a383734fc345b97647ba71aa7aee04</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.09.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.09.028%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Hare%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DEide%26aufirst%3DC.%2BA.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DAdrian%26aufirst%3DL.%2BT.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DW.%2BS.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMetcalf%26aufirst%3DC.%2BA.%26aulast%3DTyner%26aufirst%3DJ.%2BW.%26aulast%3DLoriaux%26aufirst%3DM.%2BM.%26aulast%3DCorbin%26aufirst%3DA.%2BS.%26aulast%3DWardwell%26aufirst%3DS.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DKeats%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DSundaramoorthi%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DSnodgrass%26aufirst%3DJ.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%26aulast%3DDruker%26aufirst%3DB.%2BJ.%26aulast%3DClackson%26aufirst%3DT.%26atitle%3DAP24534%252C%2520a%2520pan-BCR-ABL%2520inhibitor%2520for%2520chronic%2520myeloid%2520leukemia%252C%2520potently%2520inhibits%2520the%2520T315I%2520mutant%2520and%2520overcomes%2520mutation-based%2520resistance%26jtitle%3DCancer%2520Cell%26date%3D2009%26volume%3D16%26spage%3D401%26epage%3D412%26doi%3D10.1016%2Fj.ccr.2009.09.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Solca, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahl, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zoephel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bader, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanderson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraemer, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Himmelsbach, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haaksma, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adolf, G. R.</span><span> </span><span class="NLM_article-title">Target binding properties and cellular activity of afatinib (BIBW 2992), an irreversible ErbB family blocker</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">343</span><span class="NLM_x">, </span> <span class="NLM_fpage">342</span><span class="NLM_x">–</span> <span class="NLM_lpage">350</span><span class="refDoi"> DOI: 10.1124/jpet.112.197756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1124%2Fjpet.112.197756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=22888144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2012&pages=342-350&author=F.+Solcaauthor=G.+Dahlauthor=A.+Zoephelauthor=G.+Baderauthor=M.+Sandersonauthor=C.+Kleinauthor=O.+Kraemerauthor=F.+Himmelsbachauthor=E.+Haaksmaauthor=G.+R.+Adolf&title=Target+binding+properties+and+cellular+activity+of+afatinib+%28BIBW+2992%29%2C+an+irreversible+ErbB+family+blocker&doi=10.1124%2Fjpet.112.197756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Target binding properties and cellular activity of afatinib (BIBW 2992), an irreversible ErbB family blocker</span></div><div class="casAuthors">Solca, Flavio; Dahl, Goeran; Zoephel, Andreas; Bader, Gerd; Sanderson, Michael; Klein, Christian; Kraemer, Oliver; Himmelsbach, Frank; Haaksma, Eric; Adolf, Guenther R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-350</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Deregulation of the ErbB (proto-oncogene B of the avian erythroblastosis virus AEV-H strain) receptor network is well recognized as an oncogenic driver in epithelial cancers.  Several targeted drugs have been developed, including antibodies and small-mol. kinase inhibitors, each of them characterized by distinct patterns of ErbB receptor interactions.  Understanding the precise pharmacol. properties of these compds. is important for optimal use in clin. practice.  Afatinib [BIBW 2992; N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butenamide] is an ATP-competitive anilinoquinazoline deriv. harboring a reactive acrylamide group.  It was designed to covalently bind and irreversibly block enzymically active ErbB receptor family members.  Here, we show by X-ray crystallog. the covalent binding of afatinib to wild-type epidermal growth factor receptor (EGFR) and by mass spectrometry the covalent interaction with EGFR, EGFRL858R/T790M, human epidermal growth factor receptor 2 (HER2), and ErbB-4.  Afatinib potently inhibits the enzymatic activity of ErbB-4 (EC50 = 1 nM) and the proliferation of cancer cell lines driven by multiple ErbB receptor aberrations at concns. below 100 nM.  N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butanamide (BI 37781), a close analog of afatinib lacking the acrylamide group and thus incapable of covalent bond formation, had similar potency on cells driven by EGFR or EGFRL858R, but less or no detectable activity on cells expressing EGFRL858R/T790M HER2 or ErbB-4.  These results stress the importance of the acrylamide group and show that afatinib differs from approved ErbB targeting agents by irreversibly inhibiting the kinase activity of all ErbB family members.  They provide a mechanistic rationale for the distinct pharmacol. features of this compd. and explain the clin. activity seen in some patients who are resistant to antibody or kinase inhibitor therapy because of secondary mutations or ErbB receptor "reprogramming.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofXgQqaKXv-bVg90H21EOLACvtfcHk0liBb4JBQrtqFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE&md5=4a57df32abe8101f287fc6251f63f92d</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1124%2Fjpet.112.197756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.112.197756%26sid%3Dliteratum%253Aachs%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DSanderson%26aufirst%3DM.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DKraemer%26aufirst%3DO.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DHaaksma%26aufirst%3DE.%26aulast%3DAdolf%26aufirst%3DG.%2BR.%26atitle%3DTarget%2520binding%2520properties%2520and%2520cellular%2520activity%2520of%2520afatinib%2520%2528BIBW%25202992%2529%252C%2520an%2520irreversible%2520ErbB%2520family%2520blocker%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2012%26volume%3D343%26spage%3D342%26epage%3D350%26doi%3D10.1124%2Fjpet.112.197756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Pan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheerens, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprengeler, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrill, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendonca, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweeney, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grothaus, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffery, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spoerke, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honigberg, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalrymple, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, J. T.</span><span> </span><span class="NLM_article-title">Discovery of selective irreversible inhibitors for Bruton’s tyrosine kinase</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">58</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1002/cmdc.200600221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1002%2Fcmdc.200600221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=17154430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=58-61&author=Z.+Panauthor=H.+Scheerensauthor=S.+J.+Liauthor=B.+E.+Schultzauthor=P.+A.+Sprengelerauthor=L.+C.+Burrillauthor=R.+V.+Mendoncaauthor=M.+D.+Sweeneyauthor=K.+C.+Scottauthor=P.+G.+Grothausauthor=D.+A.+Jefferyauthor=J.+M.+Spoerkeauthor=L.+A.+Honigbergauthor=P.+R.+Youngauthor=S.+A.+Dalrympleauthor=J.+T.+Palmer&title=Discovery+of+selective+irreversible+inhibitors+for+Bruton%E2%80%99s+tyrosine+kinase&doi=10.1002%2Fcmdc.200600221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective irreversible inhibitors for bruton's tyrosine kinase</span></div><div class="casAuthors">Pan, Zhengying; Scheerens, Heleen; Li, Shyr-Jiann; Schultz, Brian E.; Sprengeler, Paul A.; Burrill, L. Chuck; Mendonca, Rohan V.; Sweeney, Michael D.; Scott, Keana C. K.; Grothaus, Paul G.; Jeffery, Douglas A.; Spoerke, Jill M.; Honigberg, Lee A.; Young, Peter R.; Dalrymple, Stacie A.; Palmer, James T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-61</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Synthesis and pharmacol. evaluation of a series of peptidomimetic quinoline derivs. was undertaken to evaluate their efficacy in acting as selective irreversible inhibitors of Bruton's tyrosine kinase (Btk).  With the relative scarcity of knowledge on the inhibition of Btk it is crucial to discover a potent and selective tool compd. for this kinase.  Herein is described the discovery of selective irreversible Btk inhibitors and their efficacy in a mouse RA model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovT3kBCYECdbVg90H21EOLACvtfcHk0liBb4JBQrtqFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D&md5=023ba8bca9942ad0127d7cf32293e9d2</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600221%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DScheerens%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.%2BJ.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26aulast%3DSprengeler%26aufirst%3DP.%2BA.%26aulast%3DBurrill%26aufirst%3DL.%2BC.%26aulast%3DMendonca%26aufirst%3DR.%2BV.%26aulast%3DSweeney%26aufirst%3DM.%2BD.%26aulast%3DScott%26aufirst%3DK.%2BC.%26aulast%3DGrothaus%26aufirst%3DP.%2BG.%26aulast%3DJeffery%26aufirst%3DD.%2BA.%26aulast%3DSpoerke%26aufirst%3DJ.%2BM.%26aulast%3DHonigberg%26aufirst%3DL.%2BA.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DDalrymple%26aufirst%3DS.%2BA.%26aulast%3DPalmer%26aufirst%3DJ.%2BT.%26atitle%3DDiscovery%2520of%2520selective%2520irreversible%2520inhibitors%2520for%2520Bruton%25E2%2580%2599s%2520tyrosine%2520kinase%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26spage%3D58%26epage%3D61%26doi%3D10.1002%2Fcmdc.200600221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambrogio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirieac, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padera, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baum, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Himmelsbach, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rettig, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solca, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span> </span><span class="NLM_article-title">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">4702</span><span class="NLM_x">–</span> <span class="NLM_lpage">4711</span><span class="refDoi"> DOI: 10.1038/onc.2008.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fonc.2008.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=18408761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=4702-4711&author=D.+Liauthor=L.+Ambrogioauthor=T.+Shimamuraauthor=S.+Kuboauthor=M.+Takahashiauthor=L.+R.+Chirieacauthor=R.+F.+Paderaauthor=G.+I.+Shapiroauthor=A.+Baumauthor=F.+Himmelsbachauthor=W.+J.+Rettigauthor=M.+Meyersonauthor=F.+Solcaauthor=H.+Greulichauthor=K.+K.+Wong&title=BIBW2992%2C+an+irreversible+EGFR%2FHER2+inhibitor+highly+effective+in+preclinical+lung+cancer+models&doi=10.1038%2Fonc.2008.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span></div><div class="casAuthors">Li, D.; Ambrogio, L.; Shimamura, T.; Kubo, S.; Takahashi, M.; Chirieac, L. R.; Padera, R. F.; Shapiro, G. I.; Baum, A.; Himmelsbach, F.; Rettig, W. J.; Meyerson, M.; Solca, F.; Greulich, H.; Wong, K-K.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4702-4711</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genetic alterations in the kinase domain of the epidermal growth factor receptor (EGFR) in non-small cell lung cancer (NSCLC) patients are assocd. with sensitivity to treatment with small mol. tyrosine kinase inhibitors.  Although first-generation reversible, ATP-competitive inhibitors showed encouraging clin. responses in lung adenocarcinoma tumors harboring such EGFR mutations, almost all patients developed resistance to these inhibitors over time.  Such resistance to first-generation EGFR inhibitors was frequently linked to an acquired T790M point mutation in the kinase domain of EGFR, or upregulation of signaling pathways downstream of HER3.  Overcoming these mechanisms of resistance, as well as primary resistance to reversible EGFR inhibitors driven by a subset of EGFR mutations, will be necessary for development of an effective targeted therapy regimen.  Here, we show that BIBW2992, an anilino-quinazoline designed to irreversibly bind EGFR and HER2, potently suppresses the kinase activity of wild-type and activated EGFR and HER2 mutants, including erlotinib-resistant isoforms.  Consistent with this activity, BIBW2992 suppresses transformation in isogenic cell-based assays, inhibits survival of cancer cell lines and induces tumor regression in xenograft and transgenic lung cancer models, with superior activity over erlotinib.  These findings encourage further testing of BIBW2992 in lung cancer patients harboring EGFR or HER2 oncogenes.  Oncogene (2008) 27, 4702-4711; doi:10.1038/onc.2008.109; published online 14 Apr. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5PQdKep-Sp7Vg90H21EOLACvtfcHk0liT27fgy8tmXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D&md5=24945893e83b4e3d01c267100fdaf57c</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.109%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DAmbrogio%26aufirst%3DL.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DKubo%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DChirieac%26aufirst%3DL.%2BR.%26aulast%3DPadera%26aufirst%3DR.%2BF.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DRettig%26aufirst%3DW.%2BJ.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.%2BK.%26atitle%3DBIBW2992%252C%2520an%2520irreversible%2520EGFR%252FHER2%2520inhibitor%2520highly%2520effective%2520in%2520preclinical%2520lung%2520cancer%2520models%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D4702%26epage%3D4711%26doi%3D10.1038%2Fonc.2008.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Food and Drug Administration</span><span class="NLM_x"> (FDA) approved drug products. </span><a href="https://www.accessdata.fda.gov/scripts/cder/drugsatfda/" class="extLink">https://www.accessdata.fda.gov/scripts/cder/drugsatfda/</a> (accessed Dec 21,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Food+and+Drug+Administration+%28FDA%29+approved+drug+products.+https%3A%2F%2Fwww.accessdata.fda.gov%2Fscripts%2Fcder%2Fdrugsatfda%2F+%28accessed+Dec+21%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Garuti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">2981</span><span class="NLM_x">–</span> <span class="NLM_lpage">2994</span><span class="refDoi"> DOI: 10.2174/092986711796391705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.2174%2F092986711796391705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=21651479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsl2ru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=2981-2994&author=L.+Garutiauthor=M.+Robertiauthor=G.+Bottegoni&title=Irreversible+protein+kinase+inhibitors&doi=10.2174%2F092986711796391705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible protein kinase inhibitors</span></div><div class="casAuthors">Garuti, L.; Roberti, M.; Bottegoni, G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2981-2994</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Targeting cancer with small mol. irreversible inhibitors of kinases represents an emerging challenge in drug discovery.  Irreversible inhibitors bind to kinase active site in a covalent and irreversible form, most frequently by reacting with a nucleophilic cysteine residue, located near the ATP binding pocket.  The most common mechanism is the Michael reaction, that refers to the addn. of a nucleophile, such as cysteine, to an α,β-unsatd. carbonyl.  The nucleophile reacts at the electrophilic β-position to form an adduct; as a result the inhibitor irreversibly blocks binding of ATP to the kinase, rendering the kinase inactive.  Different cysteine-reactive groups have been evaluated, an acrylamide or a substituted acrylamide moiety are the Michael acceptors of choice.  There are some advantages for the irreversible kinase inhibition.  These compds. are highly selective because they target a specific cysteine and only a limited no. of kinases has a cysteine at the corresponding position.  Another advantage is that covalent bond formation can overcome competition with the high endogenous concn. of ATP.  A further motivation for designing irreversible inhibitors is their longer duration of action respect to conventional inhibitors.  In fact, once bound to enzyme, these compds. do not readily dissoc. and the inhibition continues even after the inhibitor leaves the circulation.  Moreover, these inhibitors have the potential to overcome and prevent the emergence of acquired resistance conferred by mutations.  In this review examples of irreversible inhibitors are reported, focusing on chem. structures, SAR and biol. activities.  The great potential of these compds. could open new and promising perspectives for a broader application of this approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLFECsvPg2LVg90H21EOLACvtfcHk0lhUCGcIFr2ClQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsl2ru70%253D&md5=8c732cb9de810f9fac6484c245e41867</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.2174%2F092986711796391705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711796391705%26sid%3Dliteratum%253Aachs%26aulast%3DGaruti%26aufirst%3DL.%26aulast%3DRoberti%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D2981%26epage%3D2994%26doi%3D10.2174%2F092986711796391705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabnis, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhrlage, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Developing irreversible inhibitors of the protein kinase cysteinome</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1016%2Fj.chembiol.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=23438744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=146-159&author=Q.+Liuauthor=Y.+Sabnisauthor=Z.+Zhaoauthor=T.+Zhangauthor=S.+J.+Buhrlageauthor=L.+H.+Jonesauthor=N.+S.+Gray&title=Developing+irreversible+inhibitors+of+the+protein+kinase+cysteinome&doi=10.1016%2Fj.chembiol.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span></div><div class="casAuthors">Liu, Qingsong; Sabnis, Yogesh; Zhao, Zheng; Zhang, Tinghu; Buhrlage, Sara J.; Jones, Lyn H.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-159</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are a large family of approx. 530 highly conserved enzymes that transfer a γ-phosphate group from ATP to a variety of amino acid residues, such as tyrosine, serine, and threonine, that serves as a ubiquitous mechanism for cellular signal transduction.  The clin. success of a no. of kinase-directed drugs and the frequent observation of disease causing mutations in protein kinases suggest that a large no. of kinases may represent therapeutically relevant targets.  To date, the majority of clin. and preclin. kinase inhibitors are ATP competitive, noncovalent inhibitors that achieve selectivity through recognition of unique features of particular protein kinases.  Recently, there has been renewed interest in the development of irreversible inhibitors that form covalent bonds with cysteine or other nucleophilic residues in the ATP-binding pocket.  Irreversible kinase inhibitors have a no. of potential advantages including prolonged pharmacodynamics, suitability for rational design, high potency, and ability to validate pharmacol. specificity through mutation of the reactive cysteine residue.  Here, we review recent efforts to develop cysteine-targeted irreversible protein kinase inhibitors and discuss their modes of recognizing the ATP-binding pocket and their biol. activity profiles.  In addn., we provided an informatics assessment of the potential "kinase cysteinome" and discuss strategies for the efficient development of new covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVGD7TX3oIeLVg90H21EOLACvtfcHk0ljUIdZuntBzMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D&md5=cef80823123bae88362e816a0a638133</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDeveloping%2520irreversible%2520inhibitors%2520of%2520the%2520protein%2520kinase%2520cysteinome%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D146%26epage%3D159%26doi%3D10.1016%2Fj.chembiol.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Mi, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiwari, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, X. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">To, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambudkar, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, L. W.</span><span> </span><span class="NLM_article-title">Apatinib (YN968D1) reverses multidrug resistance by inhibiting the efflux function of multiple ATP-binding cassette transporters</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">7981</span><span class="NLM_x">–</span> <span class="NLM_lpage">7991</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-0111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1158%2F0008-5472.CAN-10-0111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=20876799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yqs7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=7981-7991&author=Y.+J.+Miauthor=Y.+J.+Liangauthor=H.+B.+Huangauthor=H.+Y.+Zhaoauthor=C.+P.+Wuauthor=F.+Wangauthor=L.+Y.+Taoauthor=C.+Z.+Zhangauthor=C.+L.+Daiauthor=A.+K.+Tiwariauthor=X.+X.+Maauthor=K.+K.+Toauthor=S.+V.+Ambudkarauthor=Z.+S.+Chenauthor=L.+W.+Fu&title=Apatinib+%28YN968D1%29+reverses+multidrug+resistance+by+inhibiting+the+efflux+function+of+multiple+ATP-binding+cassette+transporters&doi=10.1158%2F0008-5472.CAN-10-0111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Apatinib (YN968D1) Reverses Multidrug Resistance by Inhibiting the Efflux Function of Multiple ATP-Binding Cassette Transporters</span></div><div class="casAuthors">Mi, Yan-jun; Liang, Yong-ju; Huang, Hong-bing; Zhao, Hong-yun; Wu, Chung-Pu; Wang, Fang; Tao, Li-yang; Zhang, Chuan-zhao; Dai, Chun-Ling; Tiwari, Amit K.; Ma, Xiao-xu; To, Kenneth Kin Wah; Ambudkar, Suresh V.; Chen, Zhe-Sheng; Fu, Li-wu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7981-7991</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Apatinib, a small-mol. multitargeted tyrosine kinase inhibitor, is in phase III clin. trial for the treatment of patients with non-small-cell lung cancer and gastric cancer in China.  In this study, we detd. the effect of apatinib on the interaction of specific antineoplastic compds. with P-glycoprotein (ABCB1), multidrug resistance protein 1 (MRP1, ABCC1), and breast cancer resistance protein (BCRP, ABCG2).  Our results showed that apatinib significantly enhanced the cytotoxicity of ABCB1 or ABCG2 substrate drugs in KBv200, MCF-7/adr, and HEK293/ABCB1 cells overexpressing ABCB1 and in S1-M1-80, MCF-7/FLV1000, and HEK293/ABCG2-R2 cells overexpressing ABCG2 (wild-type).  In contrast, apatinib did not alter the cytotoxicity of specific substrates in the parental cells and cells overexpressing ABCC1.  Apatinib significantly increased the intracellular accumulation of rhodamine 123 and doxorubicin in the multidrug resistance (MDR) cells.  Furthermore, apatinib significantly inhibited the photoaffinity labeling of both ABCB1 and ABCG2 with [125I]iodoarylazidoprazosin in a concn.-dependent manner.  The ATPase activity of both ABCB1 and ABCG2 was significantly increased by apatinib.  However, apatinib, at a concn. that produced a reversal of MDR, did not significantly alter the ABCB1 or ABCG2 protein or mRNA expression levels or the phosphorylation of AKT and extracellular signal-regulated kinase 1/2 (ERK1/2).  Importantly, apatinib significantly enhanced the effect of paclitaxel against the ABCB1-resistant KBv200 cancer cell xenografts in nude mice.  In conclusion, apatinib reverses ABCB1- and ABCG2-mediated MDR by inhibiting their transport function, but not by blocking the AKT or ERK1/2 pathway or downregulating ABCB1 or ABCG2 expression.  Apatinib may be useful in circumventing MDR to other conventional antineoplastic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7UdIyUL3T57Vg90H21EOLACvtfcHk0ljUIdZuntBzMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yqs7nF&md5=ecf314e6eda8940a8a739398c959eb43</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-0111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-0111%26sid%3Dliteratum%253Aachs%26aulast%3DMi%26aufirst%3DY.%2BJ.%26aulast%3DLiang%26aufirst%3DY.%2BJ.%26aulast%3DHuang%26aufirst%3DH.%2BB.%26aulast%3DZhao%26aufirst%3DH.%2BY.%26aulast%3DWu%26aufirst%3DC.%2BP.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DTao%26aufirst%3DL.%2BY.%26aulast%3DZhang%26aufirst%3DC.%2BZ.%26aulast%3DDai%26aufirst%3DC.%2BL.%26aulast%3DTiwari%26aufirst%3DA.%2BK.%26aulast%3DMa%26aufirst%3DX.%2BX.%26aulast%3DTo%26aufirst%3DK.%2BK.%26aulast%3DAmbudkar%26aufirst%3DS.%2BV.%26aulast%3DChen%26aufirst%3DZ.%2BS.%26aulast%3DFu%26aufirst%3DL.%2BW.%26atitle%3DApatinib%2520%2528YN968D1%2529%2520reverses%2520multidrug%2520resistance%2520by%2520inhibiting%2520the%2520efflux%2520function%2520of%2520multiple%2520ATP-binding%2520cassette%2520transporters%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D7981%26epage%3D7991%26doi%3D10.1158%2F0008-5472.CAN-10-0111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0ljUIdZuntBzMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">Salama, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. B.</span><span> </span><span class="NLM_article-title">Trametinib (GSK1120212) in the treatment of melanoma</span> <span class="citation_source-journal">Expert Opin. Pharmacother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">619</span><span class="NLM_x">–</span> <span class="NLM_lpage">627</span><span class="refDoi"> DOI: 10.1517/14656566.2013.770475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1517%2F14656566.2013.770475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=23432625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFSlu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=619-627&author=A.+K.+Salamaauthor=K.+B.+Kim&title=Trametinib+%28GSK1120212%29+in+the+treatment+of+melanoma&doi=10.1517%2F14656566.2013.770475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">(GSK1120212) in the treatment of melanoma</span></div><div class="casAuthors">Salama, April K. S.; Kim, Kevin B.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Pharmacotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">619-627</span>CODEN:
                <span class="NLM_cas:coden">EOPHF7</span>;
        ISSN:<span class="NLM_cas:issn">1465-6566</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The discovery of somatic mutations in melanoma has advanced our knowledge of the biol. of the disease.  The mutations, such as those in NRAS, BRAF, GNAQ and GNA11, promote the growth of melanoma cells in most part through the mitogen-activated protein kinase (MAPK) pathway.  Understanding the mol. pathways of some of these mutations has resulted in the successful development of selective BRAF inhibitors.  Yet, a cure for advanced melanoma is far from reality.  Targeting MAPK/ERK kinase (MEK), an essential intermediary kinase protein within the MAPK pathway, may be a promising way to treat patients with BRAF or other genomic mutation.Areas covered: The authors discuss the MAPK pathway in melanoma and review the preclin. and clin. studies of the MEK inhibitor, , in melanoma.  They also discuss the potential of using trametinib in the targeted therapy of advanced melanoma.Expert opinion: Studies have demonstrated the activity of trametinib in BRAF-mutant melanoma, suggesting that it could be a very reasonable alternative to BRAF inhibitors for these patients.  Current clin. investigations have shown great promise with the combination of trametinib and dabrafenib in patients with BRAF-mutant melanoma; a no. of clin. trials of trametinib in combination with other targeted drugs are underway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQSJOlEgHVLbVg90H21EOLACvtfcHk0lgspTMSWa3k4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFSlu70%253D&md5=8cdadb7c9472b8a0fca57d1db37cff6f</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1517%2F14656566.2013.770475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14656566.2013.770475%26sid%3Dliteratum%253Aachs%26aulast%3DSalama%26aufirst%3DA.%2BK.%26aulast%3DKim%26aufirst%3DK.%2BB.%26atitle%3DTrametinib%2520%2528GSK1120212%2529%2520in%2520the%2520treatment%2520of%2520melanoma%26jtitle%3DExpert%2520Opin.%2520Pharmacother.%26date%3D2013%26volume%3D14%26spage%3D619%26epage%3D627%26doi%3D10.1517%2F14656566.2013.770475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Rational design of inhibitors that bind to inactive kinase conformations</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/nchembio799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnchembio799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=16783341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=358-364&author=Y.+Liuauthor=N.+S.+Gray&title=Rational+design+of+inhibitors+that+bind+to+inactive+kinase+conformations&doi=10.1038%2Fnchembio799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design of inhibitors that bind to inactive kinase conformations</span></div><div class="casAuthors">Liu, Yi; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The majority of kinase inhibitors, known as type I inhibitors that have been developed so far target the ATP binding site of the kinase in its active conformation, in which the activation loop is phosphorylated.  Recently, crystal structures of inhibitors such as imatinib, BIRB796, and sorafenib (BAY43-9006), known as type II inhibitors, have revealed a new binding mode that exploits an addnl. binding site immediately adjacent to the region occupied by ATP.  This pocket is made accessible by an activation-loop rearrangement that is characteristic of kinases in an inactive conformation.  The examn. of binding features of known type II inhibitors is discussed and a pharmacophore model currently used in the design of a new generation of structurally diverse type II inhibitors is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxahsMPlhgDbVg90H21EOLACvtfcHk0lgspTMSWa3k4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D&md5=4a9347a44a2cf4a412c9741fdcc5696b</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1038%2Fnchembio799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio799%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DRational%2520design%2520of%2520inhibitors%2520that%2520bind%2520to%2520inactive%2520kinase%2520conformations%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group">McTigue, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solowiej, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kania, R. S.</span><span> </span><span class="NLM_article-title">Molecular conformations, interactions, and properties associated with drug efficiency and clinical performance among VEGFR TK inhibitors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">18281</span><span class="NLM_x">–</span> <span class="NLM_lpage">18289</span><span class="refDoi"> DOI: 10.1073/pnas.1207759109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1073%2Fpnas.1207759109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=22988103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslymtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=18281-18289&author=M.+McTigueauthor=B.+W.+Murrayauthor=J.+H.+Chenauthor=Y.+L.+Dengauthor=J.+Solowiejauthor=R.+S.+Kania&title=Molecular+conformations%2C+interactions%2C+and+properties+associated+with+drug+efficiency+and+clinical+performance+among+VEGFR+TK+inhibitors&doi=10.1073%2Fpnas.1207759109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular conformations, interactions, and properties associated with drug efficiency and clinical performance among VEGFR TK inhibitors</span></div><div class="casAuthors">McTigue, Michele; Murray, Brion William; Chen, Jeffrey H.; Deng, Ya-Li; Solowiej, James; Kania, Robert S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">18281-18289, S18281/1-S18281/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Analyses of compds. in clin. development have shown that ligand efficient-mols. with privileged phys. properties and low dose are less likely to fail in the various stages of clin. testing, have fewer postapproval withdrawals, and are less likely to receive black box safety warnings.  However, detailed side-by-side examn. of mol. interactions and properties within single drug classes are lacking.  As a class, VEGF receptor tyrosine kinase inhibitors (VEGFR TKIs) have changed the landscape of how cancer is treated, particularly in clear cell renal cell carcinoma, which is molecularly linked to the VEGF signaling axis.  Despite the clear role of the mol. target, member mols. of this validated drug class exhibit distinct clin. efficacy and safety profiles in comparable renal cell carcinoma clin. studies.  The first head-to-head randomized phase III comparative study between active VEGFR TKIs has confirmed significant differences in clin. performance.  To elucidate how fundamental drug potency-efficiency is achieved and impacts differentiation within the VEGFR TKI class, we detd. potencies, time dependence, selectivities, and X-ray structures of the drug-kinase complexes using a VEGFR2 TK construct inclusive of the important juxtamembrane domain.  Collectively, the studies elucidate unique drug-kinase interactions that are dependent on distinct juxtamembrane domain conformations, resulting in significant potency and ligand efficiency differences.  The identified structural trends are consistent with in vitro measurements, which translate well to clin. performance, underscoring a principle that may be broadly applicable to prospective drug design for optimal in vivo performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsIUWiD4sXtbVg90H21EOLACvtfcHk0lgspTMSWa3k4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslymtb3I&md5=4a5ec677298a8c924c0f9557ca688a79</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1207759109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1207759109%26sid%3Dliteratum%253Aachs%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DDeng%26aufirst%3DY.%2BL.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DKania%26aufirst%3DR.%2BS.%26atitle%3DMolecular%2520conformations%252C%2520interactions%252C%2520and%2520properties%2520associated%2520with%2520drug%2520efficiency%2520and%2520clinical%2520performance%2520among%2520VEGFR%2520TK%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D18281%26epage%3D18289%26doi%3D10.1073%2Fpnas.1207759109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group">Pemovska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McTigue, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallioniemi, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porkka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wennerberg, K.</span><span> </span><span class="NLM_article-title">Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">519</span><span class="NLM_x">, </span> <span class="NLM_fpage">102</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1038/nature14119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1038%2Fnature14119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=25686603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=519&publication_year=2015&pages=102-105&author=T.+Pemovskaauthor=E.+Johnsonauthor=M.+Kontroauthor=G.+A.+Repaskyauthor=J.+Chenauthor=P.+Wellsauthor=C.+N.+Croninauthor=M.+McTigueauthor=O.+Kallioniemiauthor=K.+Porkkaauthor=B.+W.+Murrayauthor=K.+Wennerberg&title=Axitinib+effectively+inhibits+BCR-ABL1%28T315I%29+with+a+distinct+binding+conformation&doi=10.1038%2Fnature14119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation</span></div><div class="casAuthors">Pemovska, Tea; Johnson, Eric; Kontro, Mika; Repasky, Gretchen A.; Chen, Jeffrey; Wells, Peter; Cronin, Ciaran N.; McTigue, Michele; Kallioniemi, Olli; Porkka, Kimmo; Murray, Brion W.; Wennerberg, Krister</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">519</span>
        (<span class="NLM_cas:issue">7541</span>),
    <span class="NLM_cas:pages">102-105</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The BCR-ABL1 fusion gene is a driver oncogene in chronic myeloid leukemia and 30-50% of cases of adult acute lymphoblastic leukemia.  Introduction of ABL1 kinase inhibitors (for example, imatinib) has markedly improved patient survival, but acquired drug resistance remains a challenge.  Point mutations in the ABL1 kinase domain weaken inhibitor binding and represent the most common clin. resistance mechanism.  The BCR-ABL1 kinase domain gatekeeper mutation Thr315Ile (T315I) confers resistance to all approved ABL1 inhibitors except ponatinib, which has toxicity limitations.  Here we combine comprehensive drug sensitivity and resistance profiling of patient cells ex vivo with structural anal. to establish the VEGFR tyrosine kinase inhibitor axitinib as a selective and effective inhibitor for T315I-mutant BCR-ABL1-driven leukemia.  Axitinib potently inhibited BCR-ABL1(T315I), at both biochem. and cellular levels, by binding to the active form of ABL1(T315I) in a mutation-selective binding mode.  These findings suggest that the T315I mutation shifts the conformational equil. of the kinase in favor of an active (DFG-in) A-loop conformation, which has more optimal binding interactions with axitinib.  Treatment of a T315I chronic myeloid leukemia patient with axitinib resulted in a rapid redn. of T315I-pos. cells from bone marrow.  Taken together, our findings demonstrate an unexpected opportunity to repurpose axitinib, an anti-angiogenic drug approved for renal cancer, as an inhibitor for ABL1 gatekeeper mutant drug-resistant leukemia patients.  This study shows that wild-type proteins do not always sample the conformations available to disease-relevant mutant proteins and that comprehensive drug testing of patient-derived cells can identify unpredictable, clin. significant drug-repositioning opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreXgtLS5A1lLVg90H21EOLACvtfcHk0lgMxF5KHQG7ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntbs%253D&md5=af8549ae0527bfcbda6d6e05b548f928</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1038%2Fnature14119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14119%26sid%3Dliteratum%253Aachs%26aulast%3DPemovska%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DE.%26aulast%3DKontro%26aufirst%3DM.%26aulast%3DRepasky%26aufirst%3DG.%2BA.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWells%26aufirst%3DP.%26aulast%3DCronin%26aufirst%3DC.%2BN.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DKallioniemi%26aufirst%3DO.%26aulast%3DPorkka%26aufirst%3DK.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DWennerberg%26aufirst%3DK.%26atitle%3DAxitinib%2520effectively%2520inhibits%2520BCR-ABL1%2528T315I%2529%2520with%2520a%2520distinct%2520binding%2520conformation%26jtitle%3DNature%26date%3D2015%26volume%3D519%26spage%3D102%26epage%3D105%26doi%3D10.1038%2Fnature14119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span>Harvard Medical School (HMS) LINCS Database. <a href="http://lincs.hms.harvard.edu/explore/" class="extLink">http://lincs.hms.harvard.edu/explore/</a> (accessed Sep 22,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Harvard+Medical+School+%28HMS%29+LINCS+Database.+http%3A%2F%2Flincs.hms.harvard.edu%2Fexplore%2F+%28accessed+Sep+22%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group">Shmelkov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grigoryan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swetnam, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tivon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shmelkov, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardozo, T.</span><span> </span><span class="NLM_article-title">Historeceptomic fingerprints for drug-like compounds</span> <span class="citation_source-journal">Front. Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">371</span><span class="refDoi"> DOI: 10.3389/fphys.2015.00371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.3389%2Ffphys.2015.00371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=26733872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BC28rnsFGktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=371&author=E.+Shmelkovauthor=A.+Grigoryanauthor=J.+Swetnamauthor=J.+Xinauthor=D.+Tivonauthor=S.+V.+Shmelkovauthor=T.+Cardozo&title=Historeceptomic+fingerprints+for+drug-like+compounds&doi=10.3389%2Ffphys.2015.00371"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Historeceptomic Fingerprints for Drug-Like Compounds</span></div><div class="casAuthors">Shmelkov Evgeny; Grigoryan Arsen; Xin Junyang; Cardozo Timothy; Swetnam James; Tivon Doreen; Shmelkov Sergey V</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in physiology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">371</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Most drugs exert their beneficial and adverse effects through their combined action on several different molecular targets (polypharmacology).  The true molecular fingerprint of the direct action of a drug has two components: the ensemble of all the receptors upon which a drug acts and their level of expression in organs/tissues.  Conversely, the fingerprint of the adverse effects of a drug may derive from its action in bystander tissues.  The ensemble of targets is almost always only partially known.  Here we describe an approach improving upon and integrating both components: in silico identification of a more comprehensive ensemble of targets for any drug weighted by the expression of those receptors in relevant tissues.  Our system combines more than 300,000 experimentally determined bioactivity values from the ChEMBL database and 4.2 billion molecular docking scores.  We integrated these scores with gene expression data for human receptors across a panel of human tissues to produce drug-specific tissue-receptor (historeceptomics) scores.  A statistical model was designed to identify significant scores, which define an improved fingerprint representing the unique activity of any drug.  These multi-dimensional historeceptomic fingerprints describe, in a novel, intuitive, and easy to interpret style, the holistic, in vivo picture of the mechanism of any drug's action.  Valuable applications in drug discovery and personalized medicine, including the identification of molecular signatures for drugs with polypharmacologic modes of action, detection of tissue-specific adverse effects of drugs, matching molecular signatures of a disease to drugs, target identification for bioactive compounds with unknown receptors, and hypothesis generation for drug/compound phenotypes may be enabled by this approach.  The system has been deployed at drugable.org for access through a user-friendly web site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSGtyA2XMAYRySm0e6ZOaemfW6udTcc2eY4eDwvhQ3fwbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rnsFGktQ%253D%253D&md5=172619889fcb56477c076851c7219bed</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.3389%2Ffphys.2015.00371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphys.2015.00371%26sid%3Dliteratum%253Aachs%26aulast%3DShmelkov%26aufirst%3DE.%26aulast%3DGrigoryan%26aufirst%3DA.%26aulast%3DSwetnam%26aufirst%3DJ.%26aulast%3DXin%26aufirst%3DJ.%26aulast%3DTivon%26aufirst%3DD.%26aulast%3DShmelkov%26aufirst%3DS.%2BV.%26aulast%3DCardozo%26aufirst%3DT.%26atitle%3DHistoreceptomic%2520fingerprints%2520for%2520drug-like%2520compounds%26jtitle%3DFront.%2520Physiol.%26date%3D2015%26volume%3D6%26spage%3D371%26doi%3D10.3389%2Ffphys.2015.00371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group">UniProt Consortium</span><span> </span><span class="NLM_article-title">Update on activities at the Universal Protein Resource (UniProt) in 2013</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">D43</span><span class="NLM_x">–</span> <span class="NLM_lpage">D47</span><span class="refDoi"> DOI: 10.1093/nar/gks1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1093%2Fnar%2Fgks1068" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=D43-D47&author=UniProt+Consortium&title=Update+on+activities+at+the+Universal+Protein+Resource+%28UniProt%29+in+2013&doi=10.1093%2Fnar%2Fgks1068"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks1068%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUpdate%2520on%2520activities%2520at%2520the%2520Universal%2520Protein%2520Resource%2520%2528UniProt%2529%2520in%25202013%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3DD43%26epage%3DD47%26doi%3D10.1093%2Fnar%2Fgks1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span>UniProt Consortium. Human and mouse protein kinases: classification and index. <a href="http://www.uniprot.org/docs/pkinfam.txt" class="extLink">http://www.uniprot.org/docs/pkinfam.txt</a> (accessed May 15,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=UniProt+Consortium.+Human+and+mouse+protein+kinases%3A+classification+and+index.+http%3A%2F%2Fwww.uniprot.org%2Fdocs%2Fpkinfam.txt+%28accessed+May+15%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group">Rarey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">A fast flexible docking method using an incremental construction algorithm</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">261</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">489</span><span class="refDoi"> DOI: 10.1006/jmbi.1996.0477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1006%2Fjmbi.1996.0477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=8780787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADyaK28XltlKisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1996&pages=470-489&author=M.+Rareyauthor=B.+Kramerauthor=T.+Lengauerauthor=G.+Klebe&title=A+fast+flexible+docking+method+using+an+incremental+construction+algorithm&doi=10.1006%2Fjmbi.1996.0477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">A fast flexible docking method using an incremental construction algorithm</span></div><div class="casAuthors">Rarey, Matthias; Kramer, Bernd; Lengauer, Thomas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">470-489</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">We present an automatic method for docking org. ligands into protein binding sites.  The method can be used in the design process of specific protein ligands.  It combines an appropriate model of the physico-chem. properties of the docked mols. with efficient methods for sampling the conformational space of the ligand.  If the ligand is flexible, it can adopt a large variety of different conformations.  Each such min. in conformational space presents a potential candidate for the conformation of the ligand in the complexed state.  Our docking method samples the conformation space of the ligand on the basis of a discrete model and uses a tree-search technique for placing the ligand incrementally into the active site.  For placing the first fragment of the ligand into the protein, we use hashing techniques adapted from computer vision.  The incremental construction algorithm is based on a greedy strategy combined with efficient methods for overlap detection and for the search of new interactions.  We present results on 19 complexes of which the binding geometry has been crystallog. detd.  All considered ligands are docked in at most three minutes on a current workstation.  The exptl. obsd. binding mode of the ligand is reproduced with 0.5 to 1.2 Å rms deviation.  It is almost always found among the highest-ranking conformations computed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx9N2sFXkOfLVg90H21EOLACvtfcHk0lgW6wBoPbGVkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XltlKisLo%253D&md5=8cd8f384906ed90e894d118399e4dc4a</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0477%26sid%3Dliteratum%253Aachs%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DKramer%26aufirst%3DB.%26aulast%3DLengauer%26aufirst%3DT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520fast%2520flexible%2520docking%2520method%2520using%2520an%2520incremental%2520construction%2520algorithm%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1996%26volume%3D261%26spage%3D470%26epage%3D489%26doi%3D10.1006%2Fjmbi.1996.0477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schalon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bret, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">sc-PDB: an annotated database of druggable binding sites from the Protein Data Bank</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">717</span><span class="NLM_x">–</span> <span class="NLM_lpage">727</span><span class="refDoi"> DOI: 10.1021/ci050372x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050372x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFCgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=717-727&author=E.+Kellenbergerauthor=P.+Mullerauthor=C.+Schalonauthor=G.+Bretauthor=N.+Foataauthor=D.+Rognan&title=sc-PDB%3A+an+annotated+database+of+druggable+binding+sites+from+the+Protein+Data+Bank&doi=10.1021%2Fci050372x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">sc-PDB: An Annotated Database of Druggable Binding Sites from the Protein Data Bank</span></div><div class="casAuthors">Kellenberger, Esther; Muller, Pascal; Schalon, Claire; Bret, Guillaume; Foata, Nicolas; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">717-727</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The s.c.-PDB is a collection of 6,415 three-dimensional structures of binding sites found in the Protein Data Bank (PDB).  Binding sites were extd. from all high-resoln. crystal structures in which a complex between a protein cavity and a small-mol.-wt. ligand could be identified.  Importantly, ligands are considered from a pharmacol. and not a structural point of view.  Therefore, solvents, detergents, and most metal ions are not stored in the s.c.-PDB.  Ligands are classified into four main categories: nucleotides (< 4-mer), peptides (< 9-mer), cofactors, and org. compds.  The corresponding binding site is formed by all protein residues (including amino acids, cofactors, and important metal ions) with at least one atom within 6.5 Å of any ligand atom.  The database was carefully annotated by browsing several protein databases (PDB, UniProt, and GO) and storing, for every s.c.-PDB entry, the following features: protein name, function, source, domain and mutations, ligand name, and structure.  The repository of ligands has also been archived by diversity anal. of mol. scaffolds, and several chemoinformatics descriptors were computed to better understand the chem. space covered by stored ligands.  The s.c.-PDB may be used for several purposes: (i) screening a collection of binding sites for predicting the most likely target(s) of any ligand, (ii) analyzing the mol. similarity between different cavities, and (iii) deriving rules that describe the relationship between ligand pharmacophoric points and active-site properties.  The database is periodically updated and accessible on the web at http://bioinfo-pharma.u-strasbg.fr/scPDB/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNlZ_WkpBUQrVg90H21EOLACvtfcHk0lgW6wBoPbGVkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFCgtQ%253D%253D&md5=d035f998192ff55026c619b38f4b82ab</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fci050372x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050372x%26sid%3Dliteratum%253Aachs%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DMuller%26aufirst%3DP.%26aulast%3DSchalon%26aufirst%3DC.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DFoata%26aufirst%3DN.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3Dsc-PDB%253A%2520an%2520annotated%2520database%2520of%2520druggable%2520binding%2520sites%2520from%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D717%26epage%3D727%26doi%3D10.1021%2Fci050372x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group">Radifar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuniarti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Istyastono, E. P.</span><span> </span><span class="NLM_article-title">PyPLIF: python-based protein-ligand fnteraction fingerprinting</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span><span class="refDoi"> DOI: 10.6026/97320630009325</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.6026%2F97320630009325" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=325-328&author=M.+Radifarauthor=N.+Yuniartiauthor=E.+P.+Istyastono&title=PyPLIF%3A+python-based+protein-ligand+fnteraction+fingerprinting&doi=10.6026%2F97320630009325"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.6026%2F97320630009325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6026%252F97320630009325%26sid%3Dliteratum%253Aachs%26aulast%3DRadifar%26aufirst%3DM.%26aulast%3DYuniarti%26aufirst%3DN.%26aulast%3DIstyastono%26aufirst%3DE.%2BP.%26atitle%3DPyPLIF%253A%2520python-based%2520protein-ligand%2520fnteraction%2520fingerprinting%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D9%26spage%3D325%26epage%3D328%26doi%3D10.6026%2F97320630009325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group">Uchikoga, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirokawa, T.</span><span> </span><span class="NLM_article-title">Analysis of protein-protein docking decoys using interaction fingerprints: application to the reconstruction of CaM-ligand complexes</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">236</span><span class="refDoi"> DOI: 10.1186/1471-2105-11-236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1186%2F1471-2105-11-236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=20459766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A280%3ADC%252BC3czltVCqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=236&author=N.+Uchikogaauthor=T.+Hirokawa&title=Analysis+of+protein-protein+docking+decoys+using+interaction+fingerprints%3A+application+to+the+reconstruction+of+CaM-ligand+complexes&doi=10.1186%2F1471-2105-11-236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of protein-protein docking decoys using interaction fingerprints: application to the reconstruction of CaM-ligand complexes</span></div><div class="casAuthors">Uchikoga Nobuyuki; Hirokawa Takatsugu</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">236</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Protein-protein docking for proteins with large conformational changes was analyzed by using interaction fingerprints, one of the scales for measuring similarities among complex structures, utilized especially for searching near-native protein-ligand or protein-protein complex structures.  Here, we have proposed a combined method for analyzing protein-protein docking by taking large conformational changes into consideration.  This combined method consists of ensemble soft docking with multiple protein structures, refinement of complexes, and cluster analysis using interaction fingerprints and energy profiles.  RESULTS:  To test for the applicability of this combined method, various CaM-ligand complexes were reconstructed from the NMR structures of unbound CaM.  For the purpose of reconstruction, we used three known CaM-ligands, namely, the CaM-binding peptides of cyclic nucleotide gateway (CNG), CaM kinase kinase (CaMKK) and the plasma membrane Ca2+ ATPase pump (PMCA), and thirty-one structurally diverse CaM conformations.  For each ligand, 62000 CaM-ligand complexes were generated in the docking step and the relationship between their energy profiles and structural similarities to the native complex were analyzed using interaction fingerprint and RMSD.  Near-native clusters were obtained in the case of CNG and CaMKK.  CONCLUSIONS:  The interaction fingerprint method discriminated near-native structures better than the RMSD method in cluster analysis.  We showed that a combined method that includes the interaction fingerprint is very useful for protein-protein docking analysis of certain cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAfzPCyZCcdP6AExegM5LLfW6udTcc2eaDHw8Wa0uN2Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3czltVCqsw%253D%253D&md5=8ef588ac0002c004b473ab2b7b287c81</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-236%26sid%3Dliteratum%253Aachs%26aulast%3DUchikoga%26aufirst%3DN.%26aulast%3DHirokawa%26aufirst%3DT.%26atitle%3DAnalysis%2520of%2520protein-protein%2520docking%2520decoys%2520using%2520interaction%2520fingerprints%253A%2520application%2520to%2520the%2520reconstruction%2520of%2520CaM-ligand%2520complexes%26jtitle%3DBMC%2520Bioinf.%26date%3D2010%26volume%3D11%26spage%3D236%26doi%3D10.1186%2F1471-2105-11-236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group">Jaccard, P.</span><span> </span><span class="NLM_article-title">The distribution of the flora in the alpine zone</span> <span class="citation_source-journal">New Phytol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1912</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1111/j.1469-8137.1912.tb05611.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1111%2Fj.1469-8137.1912.tb05611.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1912&pages=37-50&author=P.+Jaccard&title=The+distribution+of+the+flora+in+the+alpine+zone&doi=10.1111%2Fj.1469-8137.1912.tb05611.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1111%2Fj.1469-8137.1912.tb05611.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1469-8137.1912.tb05611.x%26sid%3Dliteratum%253Aachs%26aulast%3DJaccard%26aufirst%3DP.%26atitle%3DThe%2520distribution%2520of%2520the%2520flora%2520in%2520the%2520alpine%2520zone%26jtitle%3DNew%2520Phytol.%26date%3D1912%26volume%3D11%26spage%3D37%26epage%3D50%26doi%3D10.1111%2Fj.1469-8137.1912.tb05611.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group">Hartigan, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, M. A.</span><span> </span><span class="NLM_article-title">Algorithm AS 136: A K-means clustering algorithm</span> <span class="citation_source-journal">Appl. Stat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1979</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">100</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span><span class="refDoi"> DOI: 10.2307/2346830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.2307%2F2346830" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1979&pages=100-108&author=J.+A.+Hartiganauthor=M.+A.+Wong&title=Algorithm+AS+136%3A+A+K-means+clustering+algorithm&doi=10.2307%2F2346830"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.2307%2F2346830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2307%252F2346830%26sid%3Dliteratum%253Aachs%26aulast%3DHartigan%26aufirst%3DJ.%2BA.%26aulast%3DWong%26aufirst%3DM.%2BA.%26atitle%3DAlgorithm%2520AS%2520136%253A%2520A%2520K-means%2520clustering%2520algorithm%26jtitle%3DAppl.%2520Stat.%26date%3D1979%26volume%3D28%26spage%3D100%26epage%3D108%26doi%3D10.2307%2F2346830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group">Rogers, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, M.</span><span> </span><span class="NLM_article-title">Extended-connectivity fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">–</span> <span class="NLM_lpage">754</span><span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-connectivity+fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0lgQs_x1XPx05Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-connectivity%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Improving the search performance of extended connectivity fingerprints through activity-oriented feature filtering and application of a bit-density-dependent similarity function</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">540</span><span class="NLM_x">–</span> <span class="NLM_lpage">548</span><span class="refDoi"> DOI: 10.1002/cmdc.200800408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=10.1002%2Fcmdc.200800408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=19263458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltlCgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=540-548&author=Y.+Huauthor=E.+Lounkineauthor=J.+Bajorath&title=Improving+the+search+performance+of+extended+connectivity+fingerprints+through+activity-oriented+feature+filtering+and+application+of+a+bit-density-dependent+similarity+function&doi=10.1002%2Fcmdc.200800408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the Search Performance of Extended Connectivity Fingerprints through Activity-Oriented Feature Filtering and Application of a Bit-Density-Dependent Similarity Function</span></div><div class="casAuthors">Hu, Ye; Lounkine, Eugen; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">540-548</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Improving fingerprint search performance: An activity-oriented feature-filtering procedure and a corresponding similarity function were developed for mol.-specific fingerprints, recording ensembles of structural patterns such as the popular extended connectivity fingerprints.  Shown are comparisons of search calcns. for cyclooxygenase inhibitors based on k nearest neighbor (1NN, 10NN) and Tanimoto coeff. (Tc) calcns., and the ACF BDM approach introduced herein.  The Pipeline Pilot extended connectivity fingerprints (ECFPs) are currently among the most popular similarity search tools in drug discovery settings.  ECFPs do not have a fixed bit string format but generate variable nos. of structural features for individual test mols.  This variable string design makes ECFP representations amenable to compd.-class-directed modification.  An intuitive feature-filtering technique was devised that focuses ECFP search calcns. on feature string ensembles of given compd. activity classes.  In combination with a simple bit-d.-dependent similarity function, feature filtering consistently improved the search performance of ECFP calcns. based on Tanimoto similarity and state-of-the-art data fusion techniques on a diverse array of activity classes.  Feature filtering and the bit d. similarity metric are easily implemented in the Pipeline Pilot environment.  The approach provides a viable alternative to conventional similarity searching and should be of general interest to further improve the success rate of practical ECFP applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-keIL6bpoJ7Vg90H21EOLACvtfcHk0lgQs_x1XPx05Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltlCgtrg%253D&md5=b1e5043a5ca7806d9b6accfcaf606e3b</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800408%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DImproving%2520the%2520search%2520performance%2520of%2520extended%2520connectivity%2520fingerprints%2520through%2520activity-oriented%2520feature%2520filtering%2520and%2520application%2520of%2520a%2520bit-density-dependent%2520similarity%2520function%26jtitle%3DChemMedChem%26date%3D2009%26volume%3D4%26spage%3D540%26epage%3D548%26doi%3D10.1002%2Fcmdc.200800408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span>JChem Base was used for structure searching and chemical database access and management.  <span class="citation_source-book">JChem 6.2.0</span>; <span class="NLM_publisher-name">ChemAxon</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">; </span><a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=JChem+Base+was+used+for+structure+searching+and+chemical+database+access+and+management.+JChem+6.2.0%3B+ChemAxon%2C+2014%3B+http%3A%2F%2Fwww.chemaxon.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DJChem%25206.2.0%26pub%3DChemAxon%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group">Weill, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Alignment-free ultra-high-throughput comparison of druggable protein-ligand binding sites</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span><span class="refDoi"> DOI: 10.1021/ci900349y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900349y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1yjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=123-135&author=N.+Weillauthor=D.+Rognan&title=Alignment-free+ultra-high-throughput+comparison+of+druggable+protein-ligand+binding+sites&doi=10.1021%2Fci900349y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Alignment-Free Ultra-High-Throughput Comparison of Druggable Protein-Ligand Binding Sites</span></div><div class="casAuthors">Weill, Nathanael; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">123-135</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inferring the biol. function of a protein from its three-dimensional structure as well as explaining why a drug may bind to various targets is of crucial importance to modern drug discovery.  Here we present a generic 4833-integer vector describing druggable protein-ligand binding sites that can be applied to any protein and any binding cavity.  The fingerprint registers counts of pharmacophoric triplets from the Cα at. coordinates of binding-site-lining residues.  Starting from a customized data set of diverse protein-ligand binding site pairs, the most appropriate metric and a similarity threshold could be defined for similar binding sites.  The method (FuzCav) has been used in various scenarios: (i) screening a collection of 6000 binding sites for similarity to different queries; (ii) classifying protein families (serine endopeptidases, protein kinases) by binding site diversity; (iii) discriminating adenine-binding cavities from decoys.  The fingerprint generation and comparison supports ultra-high throughput (ca. 1000 measures/s), does not require prior alignment of protein binding sites, and is able to detect local similarity among subpockets.  It is thus particularly well suited to the functional annotation of novel genomic structures with low sequence identity to known X-ray templates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7OYklk3JFxLVg90H21EOLACvtfcHk0lgQs_x1XPx05Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1yjsg%253D%253D&md5=541a9800a3aa0891f179316412f1df24</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Fci900349y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900349y%26sid%3Dliteratum%253Aachs%26aulast%3DWeill%26aufirst%3DN.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DAlignment-free%2520ultra-high-throughput%2520comparison%2520of%2520druggable%2520protein-ligand%2520binding%2520sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D123%26epage%3D135%26doi%3D10.1021%2Fci900349y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BYU','PDB','3BYU'); return false;">PDB: 3BYU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H8F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H8F','PDB','1H8F'); return false;">PDB: 1H8F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q4C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q4C','PDB','3Q4C'); return false;">PDB: 3Q4C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ITJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ITJ','PDB','4ITJ'); return false;">PDB: 4ITJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A5U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A5U','PDB','2A5U'); return false;">PDB: 2A5U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PXF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PXF','PDB','3PXF'); return false;">PDB: 3PXF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PRZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PRZ','PDB','3PRZ'); return false;">PDB: 3PRZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BYG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BYG','PDB','1BYG'); return false;">PDB: 1BYG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ZZ2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ZZ2','PDB','1ZZ2'); return false;">PDB: 1ZZ2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK','PDB','1XKK'); return false;">PDB: 1XKK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BBT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BBT','PDB','3BBT'); return false;">PDB: 3BBT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G5J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G5J','PDB','4G5J'); return false;">PDB: 4G5J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BKJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BKJ','PDB','4BKJ'); return false;">PDB: 4BKJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I22" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I22','PDB','4I22'); return false;">PDB: 4I22</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HJO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HJO','PDB','4HJO'); return false;">PDB: 4HJO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ASD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ASD','PDB','4ASD'); return false;">PDB: 4ASD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QLG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QLG','PDB','3QLG'); return false;">PDB: 3QLG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y7J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y7J','PDB','2Y7J'); return false;">PDB: 2Y7J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GP0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GP0','PDB','3GP0'); return false;">PDB: 3GP0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X9F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X9F','PDB','2X9F'); return false;">PDB: 2X9F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IVU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IVU','PDB','2IVU'); return false;">PDB: 2IVU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VS7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VS7','PDB','3VS7'); return false;">PDB: 3VS7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZBF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZBF','PDB','3ZBF'); return false;">PDB: 3ZBF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OG7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OG7','PDB','3OG7'); return false;">PDB: 3OG7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LXN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LXN','PDB','3LXN'); return false;">PDB: 3LXN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OTW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OTW','PDB','4OTW'); return false;">PDB: 4OTW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U6J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U6J','PDB','3U6J'); return false;">PDB: 3U6J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C8B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C8B','PDB','4C8B'); return false;">PDB: 4C8B</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WZE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WZE','PDB','3WZE'); return false;">PDB: 3WZE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGC','PDB','4AGC'); return false;">PDB: 4AGC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IFG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IFG','PDB','4IFG'); return false;">PDB: 4IFG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP1','PDB','3PP1'); return false;">PDB: 3PP1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CQE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CQE','PDB','4CQE'); return false;">PDB: 4CQE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C7Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C7Q','PDB','3C7Q'); return false;">PDB: 3C7Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XE0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XE0','PDB','4XE0'); return false;">PDB: 4XE0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MKC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MKC','PDB','4MKC'); return false;">PDB: 4MKC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2EUF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2EUF','PDB','2EUF'); return false;">PDB: 2EUF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WZD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WZD','PDB','3WZD'); return false;">PDB: 3WZD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AN2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AN2','PDB','4AN2'); return false;">PDB: 4AN2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZAU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZAU','PDB','4ZAU'); return false;">PDB: 4ZAU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TWP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TWP','PDB','4TWP'); return false;">PDB: 4TWP</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i20"><a href="/doi/suppl/10.1021/acs.jmedchem.5b02041">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_73571"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b02041">10.1021/acs.jmedchem.5b02041</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Correlation between the kinase pocket IFP and the experimental profile; the structures in every cluster; compounds with both cocrystallized kinase structures and PoC value; compounds with both cocrystallized kinase structures and <i>K</i><sub>d</sub> value (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Complete Fs-IFP database table (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_002.xls">XLS</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_001.pdf">jm5b02041_si_001.pdf (132.42 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b02041/suppl_file/jm5b02041_si_002.xls">jm5b02041_si_002.xls (2.62 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b02041%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-9%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b02041" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                10MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679929263ece3d82","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
