
<html lang="en"     class="pb-page"  data-request-id="396abd77-1c78-4a0a-8e0c-4f52e57e73f0"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-17;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c00962"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Rational Design and Evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as Polypharmacological Inhibitors of BET and Kinases" /></meta><meta name="dc.Creator" content="Kaikai  Lv" /></meta><meta name="dc.Creator" content="Weicong  Chen" /></meta><meta name="dc.Creator" content="Danqi  Chen" /></meta><meta name="dc.Creator" content="Jie  Mou" /></meta><meta name="dc.Creator" content="Huijie  Zhang" /></meta><meta name="dc.Creator" content="Tiantian  Fan" /></meta><meta name="dc.Creator" content="Yanlian  Li" /></meta><meta name="dc.Creator" content="Danyan  Cao" /></meta><meta name="dc.Creator" content="Xin  Wang" /></meta><meta name="dc.Creator" content="Lin  Chen" /></meta><meta name="dc.Creator" content="Jingkang  Shen" /></meta><meta name="dc.Creator" content="Dongsheng  Pei" /></meta><meta name="dc.Creator" content="Bing  Xiong" /></meta><meta name="dc.Description" content="Cancer exhibits diverse heterogeneity with a complicated molecular basis that usually harbors genetic and epigenetic abnormality, which poses a big challenge for single-target agents. In the curren..." /></meta><meta name="Description" content="Cancer exhibits diverse heterogeneity with a complicated molecular basis that usually harbors genetic and epigenetic abnormality, which poses a big challenge for single-target agents. In the curren..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 11, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00962" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00962" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00962" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00962" /></link>
        
    
    

<title>Rational Design and Evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as Polypharmacological Inhibitors of BET and Kinases | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00962" /></meta><meta property="og:title" content="Rational Design and Evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as Polypharmacological Inhibitors of BET and Kinases" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0018.jpeg" /></meta><meta property="og:description" content="Cancer exhibits diverse heterogeneity with a complicated molecular basis that usually harbors genetic and epigenetic abnormality, which poses a big challenge for single-target agents. In the current work, we proposed a hybrid strategy by incorporating pharmacophores that bind to the acetylated lysine binding pocket of BET proteins with a typical kinase hinge binder to generate novel polypharmacological inhibitors of BET and kinases. Through elaborating the core structure of 6-(pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-one, we demonstrated that this rational design can produce high potent inhibitors of CDK9 and BET proteins. In this series, compound 40 was identified as the potential lead compound with balanced activities of BRD4 (IC50 = 12.7 nM) and CDK9 (IC50 = 22.4 nM), as well as good antiproliferative activities on a small cancer cell panel. Together, the current study provided a new method for the discovery of bromodomain and kinase dual inhibitors rather than only being discovered by serendipity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00962"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00962">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00962&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00962&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00962&amp;href=/doi/10.1021/acs.jmedchem.0c00962" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 9787-9802</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00899" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00973" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Rational Design and Evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-ones as Polypharmacological Inhibitors of BET and Kinases</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Kaikai Lv</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kaikai Lv</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kaikai++Lv">Kaikai Lv</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Weicong Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weicong Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, Xuzhou Medical University, 209 Tongshan Road, Xuzhou 221006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weicong++Chen">Weicong Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Danqi Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Danqi Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Danqi++Chen">Danqi Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jie Mou</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jie Mou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Jiangsu Key Laboratory of New Drug and Clinical Pharmacy, School of Pharmacy, Xuzhou Medical University, 209 Tongshan Road, Xuzhou 221006, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#2110111111111311111816101161595b494c540f4445540f424f"><span class="__cf_email__" data-cfemail="5160616161616361616866606111292b393c247f3435247f323f">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jie++Mou">Jie Mou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Huijie Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huijie Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huijie++Zhang">Huijie Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tiantian Fan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tiantian Fan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tiantian++Fan">Tiantian Fan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanlian Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanlian Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanlian++Li">Yanlian Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Danyan Cao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Danyan Cao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Danyan++Cao">Danyan Cao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xin Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Wang">Xin Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lin Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lin Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lin++Chen">Lin Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jingkang Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jingkang Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jingkang++Shen">Jingkang Shen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dongsheng Pei</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dongsheng Pei</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, Xuzhou Medical University, 209 Tongshan Road, Xuzhou 221006, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ed899e9d8884ad9597858098c3888998c38e83"><span class="__cf_email__" data-cfemail="8eeafdfeebe7cef6f4e6e3fba0ebeafba0ede0">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dongsheng++Pei">Dongsheng Pei</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Bing Xiong</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bing Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#6f0d17060001082f1c060202410e0c410c01"><span class="__cf_email__" data-cfemail="a9cbd1c0c6c7cee9dac0c4c487c8ca87cac7">[email protected]</span></a>. Fax: +86 21 50807088.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bing++Xiong">Bing Xiong</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9776-8136" title="Orcid link">http://orcid.org/0000-0001-9776-8136</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00962&amp;href=/doi/10.1021%2Facs.jmedchem.0c00962" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 9787–9802</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 11, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 June 2020</li><li><span class="item_label"><b>Published</b> online</span>11 August 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00962" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00962</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9787%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DKaikai%2BLv%252C%2BWeicong%2BChen%252C%2BDanqi%2BChen%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D17%26contentID%3Dacs.jmedchem.0c00962%26title%3DRational%2BDesign%2Band%2BEvaluation%2Bof%2B6-%2528Pyrimidin-2-ylamino%2529-3%252C4-dihydroquinoxalin-2%25281H%2529-ones%2Bas%2BPolypharmacological%2BInhibitors%2Bof%2BBET%2Band%2BKinases%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9802%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00962"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1251</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00962" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Rational Design and Evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as Polypharmacological Inhibitors of BET and Kinases&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Kaikai&quot;,&quot;last_name&quot;:&quot;Lv&quot;},{&quot;first_name&quot;:&quot;Weicong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Danqi&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jie&quot;,&quot;last_name&quot;:&quot;Mou&quot;},{&quot;first_name&quot;:&quot;Huijie&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Tiantian&quot;,&quot;last_name&quot;:&quot;Fan&quot;},{&quot;first_name&quot;:&quot;Yanlian&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Danyan&quot;,&quot;last_name&quot;:&quot;Cao&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Lin&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jingkang&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Dongsheng&quot;,&quot;last_name&quot;:&quot;Pei&quot;},{&quot;first_name&quot;:&quot;Bing&quot;,&quot;last_name&quot;:&quot;Xiong&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;9787-9802&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00962&quot;},&quot;abstract&quot;:&quot;Cancer exhibits diverse heterogeneity with a complicated molecular basis that usually harbors genetic and epigenetic abnormality, which poses a big challenge for single-target agents. In the current work, we proposed a hybrid strategy by incorporating pharmacophores that bind to the acetylated lysine binding pocket of BET proteins with a typical kinase hinge binder to generate novel polypharmacological inhibitors of BET and kinases. Through elaborating the core structure of 6-(pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-one, we demonstrated that this rational design can produce high potent inhibitors of CDK9 and BET proteins. In this series, compound 40 was identified as the potential lead compound with balanced activities of BRD4 (IC50 = 12.7 nM) and CDK9 (IC50 = 22.4 nM), as well as good antiproliferative activities on a small cancer cell panel. Together, the current study provided a new method for the discovery of bromodomain and kinase dual inhibitors rather than only being discovered by serendipity.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00962&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00962" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00962&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00962" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00962&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00962" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00962&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00962&amp;href=/doi/10.1021/acs.jmedchem.0c00962" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00962" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00962" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00962%26sid%3Dliteratum%253Aachs%26pmid%3D32787081%26genre%3Darticle%26aulast%3DLv%26date%3D2020%26atitle%3DRational%2BDesign%2Band%2BEvaluation%2Bof%2B6-%2528Pyrimidin-2-ylamino%2529-3%252C4-dihydroquinoxalin-2%25281H%2529-ones%2Bas%2BPolypharmacological%2BInhibitors%2Bof%2BBET%2Band%2BKinases%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D9787%26epage%3D9802%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Cancer exhibits diverse heterogeneity with a complicated molecular basis that usually harbors genetic and epigenetic abnormality, which poses a big challenge for single-target agents. In the current work, we proposed a hybrid strategy by incorporating pharmacophores that bind to the acetylated lysine binding pocket of BET proteins with a typical kinase hinge binder to generate novel polypharmacological inhibitors of BET and kinases. Through elaborating the core structure of 6-(pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one, we demonstrated that this rational design can produce high potent inhibitors of CDK9 and BET proteins. In this series, compound <b>40</b> was identified as the potential lead compound with balanced activities of BRD4 (IC<sub>50</sub> = 12.7 nM) and CDK9 (IC<sub>50</sub> = 22.4 nM), as well as good antiproliferative activities on a small cancer cell panel. Together, the current study provided a new method for the discovery of bromodomain and kinase dual inhibitors rather than only being discovered by serendipity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33007" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33007" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Complex diseases, such as cancer, can be attributed to the dysregulation of multiple signal pathways and physiological processes, which are hardly cured by the specific modulation of a single target. A polypharmacological inhibitor as an intriguing therapeutic option is gaining a lot of attention in current anticancer drug development.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> It has several advantages with respect to single-target drugs and combination therapies,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and the most important aspects can be summarized as follows: (1) a multitarget drug could avoid the drug–drug interaction issues; (2) it may demonstrate a more predictable pharmacokinetic profile compared with combination therapies; (3) it guarantees the simultaneous presence in the tissues to reduce the acute and delayed toxicities in comparison with a combination therapy; and (4) it usually has synergistic effects to reduce the administration dosage and therefore may avoid the associated toxicity issues of a single-target agent.</div><div class="NLM_p">Started with imatinib, a Bcr-Abl inhibitor for cancer treatment, developing kinase inhibitors for cancer therapies have stimulated tremendous efforts and resulted in magnificent successes with more than 50 kinase inhibitors into the market. However, application of these targeted kinases drugs is impeded by acquired resistance which usually involves drug-induced mutations and epigenetic reprogramming. Based on the recent advance of epigenetic drug development, especially the bromodomain inhibitors, there are a lot of clinical trials ongoing and the investigations indicated that these epigenetic drugs can rewire the gene expression and regulation network to execute its anticancer effects. Besides, there are many studies indicating that combining bromodomain and extra-terminal domain (BET) inhibitors with kinase inhibitors can enhance the antitumor effects through simultaneous targeting of the abnormal signaling kinases pathways and epigenetics-related cellular reprogramming.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> Moreover, the combined use of the BET inhibitor could reduce the probability of acquired resistance of kinase drugs. Giving the distinct benefits of combining BET inhibitors with kinase inhibitors, considerable efforts are dedicated to developing the multitargeted molecules of BET and kinases.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a> However, to the best of our knowledge, all of them are discovered by serendipity (compounds <b>1–3</b> of <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>),<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> as exemplified by the PLK1 and BET dual inhibitor BI-2536,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> or optimized from these known inhibitors (<b>4</b>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative dual bromodomain-kinase inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To overcome this limitation, in this work, we presented a rational strategy by hybridizing<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> the acetylated lysine (KAc) mimic of BET inhibitors with the hinge binder motif extracted from typical kinase inhibitors. We demonstrated that this design can provide a new opportunity to elegantly develop bromodomain-kinase multitargeted agents for antitumor treatment.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58222" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58222" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Inhibitor Design Strategy</h3><div class="NLM_p">In the past two decades, more than 50 kinase inhibitors have been approved by FDA to enter into drug market<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and most of them are ATP-competitive inhibitors with a conserved hydrogen-bond acceptor motif, usually pyridine or pyrimidine, to interact with the backbone of hinge segment in kinase catalytic domains.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Since the ATP binding site of kinases is located at the cleft formed by the N-lobe and the C-lobe, it also has a large solvent-accessible area that can accommodate diverse and large chemical groups. Based on the binding mode of kinase inhibitors and the biological importance of drug combination of BET and kinase inhibitors, we assumed that hybridize kinase essential hinge binder with the well-characterized KAc mimic from bromodomain inhibitors may provide new polypharmacological inhibitors with unique biological effects as anticancer drugs. Previously, the dihydropteridinone in BI-2536 was identified as an essential component for developing dual BET-kinase inhibitors. However, this moiety is both a kinase hinge binder and BET protein’s pharmacophore, which makes the optimization highly interlocked. Therefore, it limits the target scope, and currently, only PLK1 and ALK were found to be the targets of this chemotype.<a onclick="showRef(event, 'ref11 ref12 ref15'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref15">(11,12,15)</a> We have discovered a series of dihydroquinoxalin-2(1<i>H</i>)-one derivatives as potent BET inhibitors and found that the scaffold dihydroquinoxalin-2(1<i>H</i>)-one is a selective and potent KAc mimic, which can form several direct and water-bridged hydrogen bonds with BRD4. Also, the 3,5-dimethylisoxazole motif first reported by Hewings et al.,<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> is considered a general pharmacophore for the KAc binding pocket and has been incorporated into many series of BET and non-BET inhibitors. We proposed to use the hybridization strategy, which will incorporate the KAc binding moiety with a typical kinase hinge binder to discover novel dual BET-kinase inhibitors. This strategy could enable us to select more biological relevant kinases as BET-kinase polypharmacological targets. To test the hypothesis of the hybrid strategy, we designed four prototype compounds (<b>5–8</b> in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) by combining the pyrimidin-2-amine group with two KAc mimic and initialized a docking study to check the binding feasibility of them in the first bromodomain of BRD4 (BRD4-BD1). The docking result indicated that the compounds can bind to the KAc binding pocket of BRD4 and maintain the essential hydrogen-bond interactions between the residue Asn140 and the selected dihydroquinoxalin-2(1<i>H</i>)-one<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18−20)</a> or 3,5-dimethylisoxazole motif<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25">(21−25)</a> (see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Figure S1</a> for the calculated binding conformation). Based on this feasibility to maintain the key interactions in the KAc binding site of BRD4-BD1 and the existence of large solvent-accessible subpocket in the kinase ATP binding site able to accommodate the BET pharmacophore, we synthesized these four compounds to subject to the biological tests of BRD4 and several kinase to verify this approach.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Proof-of-Concept of Molecular Design<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0011.gif" alt="" id="GRAPHIC-d7e447-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0012.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">All BRD4-BD1 IC<sub>50</sub> values and kinase inhibition values are mean values from at least two independent experiments.</p></div></div><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Structure–Activity Relationship</h3><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, two compounds (<b>5</b> and <b>6</b>) containing 3,5-dimethylisoxazole did not show more than 50% inhibition ratio at a concentration of 1 μM in the BRD4-BD1 fluorescence polarization (FP) assay. However, the compounds (<b>7</b> and <b>8</b>) with the dihydroquinoxalin-2(1<i>H</i>)-one scaffold showed encouraging binding activities toward BRD4, with the IC<sub>50</sub> values below 200 nM. Based on this initial results, we prepared two derivatives (<b>9</b> and <b>10</b>) to check the possibility of optimization. Clearly, they demonstrated that the modification at the benzene group could remain a good binding activity to BRD4, especially at the meta-position of the benzene group (<b>10</b> with IC<sub>50</sub> = 160.5 nM). Previous studies have identified the synergistic effect of BRD4 with kinases such as CDKs,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> PI3Ks,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and Aurora kinases.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> Therefore, we tested the inhibition effects of compounds <b>7–10</b> on these selected kinases. As listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compounds <b>7–10</b> showed differential activities to four selected kinases. In detail, compounds <b>7–9</b> have good activities toward CDK9 but with no activity on CDK6, indicating they have the potential to be developed into selective inhibitors. Besides the promising activity on CDK9, compound <b>8</b> also showed good activity on PI3Kα, and compound <b>9</b> showed good activity on Aurora kinase A. For the comparison, we also synthesized the compounds <b>11–14</b> by substituting the pyrimidine core with the benzene motif, which eliminate the hinge region binding capability and therefore should not inhibit the kinase activity. As expected, compounds <b>11–14</b> did not show any activity on these four kinases, further demonstrating that the pyrimidine, a common hinge binder acting as hydrogen-bond acceptor, is very important for the binding to kinases. Contrarily, because the pyrimidine in compounds <b>7–10</b> is not essential for binding to BRD4, the substituents (<b>11–14</b>) still have good binding activities on BRD4, as indicated by the similar IC<sub>50</sub> values from BRD4-BD1 FP assay relative to the corresponding compounds <b>7–10</b>.</div><div class="NLM_p">To assist the further optimization, we solved the cocrystal structure of compound <b>7</b> bound to the BD1 domain of BRD4. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, the dihydroquinoxalin-2(1<i>H</i>)-one scaffold is situated at the same position in the KAc binding site with respect to our previously reported crystal structure and makes essential hydrogen-bond interactions with the conserved residue Asn140. The binding conformation of compound <b>7</b> in the solved structure adopted a U shape by rotating the 4-phenylpyrimidin-2-amine group to the residue Asp145. Comparing with our previous solved crystal structure, the residue Asp145 slightly receded to eliminate the steric clash with the ligand (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cocrystal structure of compound <b>7</b> (cyans) and compound <b>40</b> (blue) in complex with BRD4-BD1 (PDB entry number <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIH">6LIH</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIM">6LIM</a>, respectively). (A) Binding mode of compound <b>7</b> in the KAc binding pocket. Asn140, Tyr97, Trp81, and Asp145 are represented as the stick model, and hydrogen bonds are represented by dashed lines. (B) Superimposing the structure of compound <b>7</b> to our previous crystal structure (PDB entry number: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI5">6JI5</a>) (green). (C) Superimposing the structure of compound <b>7</b> and compound <b>40</b>. (D) Interactions between compound <b>40</b> and WPF pocket that is represented as a surface diagram.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Based on the cocrystal structure, we found that adding a small group at the 2- or 3-position of the benzene group could extend the ligand into the WPF subpocket, which may enhance the binding affinity with BRD4. Because compounds <b>7–10</b> all showed activities on CDK9, as a proof-of-concept project, we decided to simultaneously optimize the compounds as dual inhibitors of BET and CDK9. To probe the binding mode of <b>7</b> with CDK9, we docked the compound into the ATP binding site of CDK9 by utilizing the crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH">6GZH</a> as the template.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> From the docking model (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A), compound <b>7</b> adopted similar binding interactions to the ligand A86 ([4-[[4-[5-(cyclopropylmethyl)-1-methyl-pyrazol-4-yl]-5-fluoranyl-pyrimidin-2-yl]amino]cyclohexyl]azanium) in the original crystal structure, forming the essential hydrogen-bond interactions with the hinge residue Cys106 and stretching the benzene group upward (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). The dihydroquinoxalin-2(1<i>H</i>)-one ring extends to the solvent-accessible area with the isopropyl group reverting to the vicinity of the benzene group. According this predicted binding mode, substitutions on the benzene group also are feasible to enhance the binding activity on CDK9, which is consistent with the idea proposed for the BRD4 optimization.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding conformation of compounds <b>7</b>, <b>21</b>, and <b>40</b> in the CDK9 ATP binding site from the docking study. The protein is shown in the cartoon mode, and the residues of the binding site are shown in the stick mode. (A) Predicted conformation of compound <b>7</b> (cyans) bound to the ATP binding site of CDK9. It can form two hydrogen bonds with hinge residue Cys106. (B) Superimposition of compound <b>7</b> with original ligand A86 (green) in the crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH">6GZH</a>. (C) Predicted conformation of compound <b>21</b> (orange) bound to the ATP binding site of CDK9. (D) Predicted conformation of compound <b>40</b> (blue) bound to the ATP binding site of CDK9.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, various small functional groups were added to different positions of the benzene group. According to the analysis of the cocrystal structure of compound <b>7</b> with BRD4-BD1, the 4-position is at the vicinity of the protein residue Asp145. Therefore, substitutions at the 4-position of the benzene may cause a steric crash to reduce the binding activities to BRD4, which is consistent with the low activities of compounds <b>16</b>, <b>19</b>, and <b>22</b>, especially of compound <b>19</b> (IC<sub>50</sub> = 5850.0 nM). Compounds with substitutions at 3-position of the benzene group also showed decreased activities except compound <b>21</b> with a NH<sub>2</sub> group that may form electrostatic interaction with the nearby negative-charge residue Asp145. Because the 2-position faces the WPF subpocket, the substitutions at 2-position generally have a less detriment effect on the binding with BRD4. Compounds <b>15–24</b> were tested in CDK9 enzymatic inhibition assay, and all of them except <b>17</b> showed moderate to high inhibitory activities. Among them, compound <b>21</b> showed very good activity toward CDK9, with an IC<sub>50</sub> value of 5.9 nM. We docked compound <b>21</b> into the ATP binding site of CDK9 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C) and found that its binding conformation is very similar to the previous predicted binding mode of compound <b>7</b>, only the amino group entering into a hydrophilic subpocket formed by residues Lys48, Glu66, and Asp167, which may create favorable electrostatic interactions.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Exploration of Different Substituents on Benzene Group<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0013.gif" alt="" id="GRAPHIC-d7e725-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd.</th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center">BRD4 BD1 IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">CDK9 IC<sub>50</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">3-OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">965.0 ± 135.0</td><td class="colsep0 rowsep0" align="left">126.4 ± 20.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">4-OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">534.5 ± 9.5</td><td class="colsep0 rowsep0" align="left">31.7 ± 0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">2-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">271.0 ± 12.0</td><td class="colsep0 rowsep0" align="left">1.0%@1 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1.5%@100 nM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">3-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">759.5 ± 42.5</td><td class="colsep0 rowsep0" align="left">195.8 ± 5.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">4-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">5850.0 ± 150.0</td><td class="colsep0 rowsep0" align="left">488.5 ± 34.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">2-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">251.5 ± 73.5</td><td class="colsep0 rowsep0" align="left">39.2 ± 2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">3-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">119.5 ± 30.5</td><td class="colsep0 rowsep0" align="left">5.9 ± 0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">4-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">480.0 ± 48.0</td><td class="colsep0 rowsep0" align="left">48.9 ± 8.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">2-OCH<sub>3</sub>-4-F</td><td class="colsep0 rowsep0" align="left">496.5 ± 82.5</td><td class="colsep0 rowsep0" align="left">46.9 ± 6.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">2-CH<sub>3</sub>-4-F</td><td class="colsep0 rowsep0" align="left">156.2 ± 34.3</td><td class="colsep0 rowsep0" align="left">578.2 ± 97.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">All BRD4-BD1 IC<sub>50</sub> values and CDK9 IC<sub>50</sub> values are mean values from at least two independent experiments. The BET selective inhibitor JQ-1 and CDK9 selective inhibitor Dinaciclib were used as the positive control in our enzymatic assay, respectively, exhibiting an IC<sub>50</sub> value of 41.5 and 3.0 nM.</p></div></div></div><div class="NLM_p">Considering the promising inhibitory activities of compound <b>21</b> on BRD4 and CDK9, we further explore the structure–activity relationship (SAR) around the 3-position of the benzene group in order to obtain potent dual-inhibitors.</div><div class="NLM_p">On the basis of compound <b>21</b>, we synthesized compounds <b>25–30</b> by substituted various groups on the 3-NH<sub>2</sub> of the benzene group (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Compound <b>25</b> with the added methyl group maintained the binding activity on BRD4 relative to <b>21</b>. Compounds with slightly large groups such as ethyl or isopropyl (<b>26</b> and <b>27</b>) decreased the potency about two folds. The substituents larger than isopropyl (<b>28–30</b>) are not tolerable at this position as they showed a dramatic loss in BRD4 binding activities. Because compounds with 2-methyl group on the benzene (<b>17</b> and <b>24</b>) also showed good binding activities on BRD4, we prepared compounds <b>31</b> and compound <b>32</b> to further explore the binding motif around the WPF subpocket (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Comparison of compounds <b>31</b> and <b>32</b>, it was found the methyl at the ortho-position of amino group in compound <b>31</b> may extend into the WPF pocket and make favorable van der Waals (vdW) interactions with the residues lining this subpocket, which slightly increased the binding affinity with BRD4-BD1. Compound <b>33</b> with the tetrahydroquinoline motif at this part exhibited high potency to BRD4 (IC<sub>50</sub> = 27.0 nM), indicating that this added piperidine ring indeed entered into the WPF subpocket. We also investigated the activity of these compounds on CDK9. The result showed that adding the methyl group on the amine resulted in the decrease of the potency of compound <b>25</b> on CDK9 about 4-fold relative to compound <b>21</b>. Substituted with larger groups further reduced the inhibitory activity on CDK9, which generally follows the similar trends in the BRD4 binding assay. Interestingly, although compound <b>33</b> coupled a large bicyclic group on the pyrimidine, it still has a good activity to CDK9 (IC<sub>50</sub> = 67.0 nM), prompting us that the bicycle motifs could be used to further probe the dual inhibitors.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Exploration of 3-NH<sub>2</sub> on Benzene Group<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0014.gif" alt="" id="GRAPHIC-d7e1064-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0015.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">All BRD4-BD1 IC<sub>50</sub> values and CDK9 IC<sub>50</sub> values are mean values from at least two independent experiments.</p></div></div><div></div></div><div class="NLM_p">From the abovementioned SAR study, compounds <b>34–40</b> with bicycles coupled on the pyrimidine group were synthesized. From the BRD4 binding assay, it was revealed that compounds containing the [6,6]-bicyclic ring system (<b>37–40</b>) have better activities than [6,5]-bicycle compounds (<b>34–36</b>), especially compounds <b>38</b> and <b>40</b> with IC<sub>50</sub> values lower than 20 nM. We checked the CDK9 enzymatic inhibition of these compounds and found that compound <b>35</b> with NH group at 3-position of the benzene group has superior inhibitory activity on CDK9 (IC<sub>50</sub> = 13.8 nM). Fortunately, the BRD4-potent compound <b>40</b> also showed high activity to CDK9, with an IC<sub>50</sub> value of 81.6 nM. Comparing compounds <b>38</b> with <b>40</b>, clearly, the nitrogen atom in the bicycle played an important role for the binding interactions. We utilized the molecular docking method to examine the possible binding mechanism. Based on the docking model (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D), the nitrogen atom in the isoquinoline ring of compound <b>40</b> located at the same position as the 2-nitrogen atom in parazole of the original ligand is at the vicinity of the residue Lys48 that participates in the salt bridge interaction with DFG motif. It was noticed that, when compared with the predicted binding conformation of compounds <b>7</b> and <b>21</b>, the dihydroquinoxalin-2(1<i>H</i>)-one ring in compound <b>40</b> adopted a different orientation to alleviate the possible clash with the larger isoquinoline group.</div><div class="NLM_p">Because the R<sub>4</sub> substitutions on pyrimidine are usually utilized to enhance the selectivity on CDK9 (as exemplified in crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH">6GZH</a>, the ligand has a fluorine atom at this position, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B); then, we prepared compounds <b>41–43</b> and examined them in this two assays. From <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, we can find that although <b>41</b> and <b>42</b> retained the activity on CDK9, they showed about 5-fold reduction in BRD4 binding activities. While compound <b>43</b> retained similar high binding activity on BRD4-BD1 as compound <b>40</b>, it did not maintain the inhibitory activity on CDK9. Taking together, compound <b>40</b> showed high potency on both BRD4-BD1 and CDK9 proteins and were selected as a promising dual inhibitor for further profiling the binding mode, selectivity, and cellular activity.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Exploration of Bicycles on the WPF Subpocket<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0016.gif" alt="" id="GRAPHIC-d7e1181-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0017.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">All BRD4-BD1 IC<sub>50</sub> values and CDK9 IC<sub>50</sub> values are mean values from at least two independent experiments.</p></div></div><div></div></div><div class="NLM_p last">The cocrystal structure of compound <b>40</b> bound to BRD4-BD1 was solved with X-ray crystallography. By superimposing on the cocrystal structure of compound <b>7</b>, we can realize that the bicycle ring on the pyrimidine extended into the WPF subpocket, making favorable vdW interactions with this hydrophobic area as expected. The essential dihydroquinoxalin-2(1<i>H</i>)-one scaffold established the same hydrogen bond interactions with the residues of the KAc binding pocket: one hydrogen bond between the oxygen atom of amide of compound <b>40</b> and the side chain of Asn140 and one water-bridged hydrogen bond between the same oxygen and the phenol group of residue Tyr97. The notable difference between the binding mode of compound <b>40</b> and of compound <b>7</b> lies in the pyrimidine ring, which is slightly backward to eliminate the steric clash with the helix around the residue Asp145 and Ile146 and therefore make a new hydrogen bond with the residue Lys91.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Selectivity Profile of Compound <b>40</b></h3><div class="NLM_p">Although compound <b>40</b> was optimized as a dual inhibitor of BRD4 and CDK9, we examined its selectivity in a diverse kinase panel provided by Eurofins Scientific and in-house purified bromodomains. From the result of DiscoveRx diversity-kinase-panel assay that contains 54 kinases, at 1 μM concentration, compound <b>40</b> showed the highest percent inhibition of CDK9, although five others also displayed more than 80% inhibition (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Two other CDK kinases (CDK1 and CDK2) were also strongly inhibited by compound <b>40</b> with the inhibition ratio around 90% at 1 μM concentration (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Table S1</a>). Therefore, we obtained the detailed IC<sub>50</sub> value (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Table S2</a>) of CDK2 (168.9 nM) and showed that compound <b>40</b> has about 8-fold selectivity toward CDK9 over CDK2. However, for CDK6, compound <b>40</b> only showed about 22% inhibition at tested concentration. We also tested the compound on seven in-house prepared bromodomains, including BRD4-BD1, BRD4-BD2, BRD2-BD1, BRD2-BD2, and non-BET bromodomain proteins EP300, CECR2, and BRD9.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> As demonstrated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Table S4</a>, it is a potent BET inhibitor and showed less than 12% inhibition activity to three other bromodomains at 1 μM concentration. Overall, the selectivity measurement of compound <b>40</b> revealed that the design strategy by hybridizing KAc mimic with hinge binder of the kinases is feasible to provide new options for developing polypharmacological inhibitors of bromodomain and kinases. With the detailed analysis of cocrystal structure of <b>40</b>, we could find that although the benzene motif of the bicycle ring extends into the WPF pocket, the other side of bicycle points to the solvent-accessible part. Therefore, we think that it is possible to obtain more selective inhibitors by modifying the bicycle ring motif, as it is the determinant for kinase selectivity.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Selectivity profile of compound <b>40</b> on selected bromodomains and DiscoveRx diversity-kinase-panel. All inhibition values are mean values from at least two independent experiments at the concentration of 1 μM, and for kinase selectivity assay, only kinases with an inhibition ratio of more than 50% were depicted in the <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and whole data were listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Supporting Information</a> (Table S1).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Cellular Activity and Western Blot</h3><div class="NLM_p">To further examine the cellular activity of compound <b>40</b>, total 22 cancer cell lines were screened to identify the sensitive cancer cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Table S5</a>, compound <b>40</b> showed differential effects on the tested cell lines. After 96 h of treating compound <b>40</b>, several cancer cell lines were remarkably sensitive to compound <b>40</b>, but others were not. This discrepancy may be because of the heterogeneity of tumor cells. Tögel et al.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> detected molecular subgroups of colon cancer, which included microsatellite instability (MSI)/microsatellite stable (MSS) and CpG island methylator phenotype (CIMP), mutation status of commonly altered oncogenes, and tumor suppressor genes in colorectal cell lines; they found that colorectal cell lines with MSI were significantly more sensitive to JQ1 compared to MSS cell lines. However, neither an association between JQ1 response and cell line wild type or mutant for RAS/BRAF, PIK3CA/PTEN, or TP53 was observed in their study nor was there a correlation between JQ1 response and CIMP. The positive control compounds of BRD4 inhibitor JQ-1 and CDK9 inhibitor Atuveciclib were applied to these cell lines, and the results (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Supporting Information</a> Table S5) indicated the dual inhibitor <b>40</b> could enhance the antiproliferative activities, suggesting a synergy effect in the cellular context. Hence, the relatively sensitive solid tumor cell line HCT-116 was selected to detect the cellular activity of drug targets.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proliferation inhibition of 22 cancer cell lines by Compound <b>40</b>.The cutoff line (red dotted line) represents IC<sub>50</sub> values that were larger than 500 nM. Precise values are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Supporting Information</a>, Table S5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To test the effects of the drug on activities of CDK9 and BRD4, the HCT-116 cell line was pretreated with compound JQ1, Atuveciclib, or compound <b>40</b> (0 nM, 50 nM, 100 nM, 250 nM, or 500 nM for 24 h, respectively) and then immunoblotted. JQ1 is the positive control of BRD4, and Atuveciclib is the positive control of CDK9. CDK9 and BRD4 facilitate the transition from abortive to productive elongation by phosphorylating the CTD (C-terminal domain) of the large subunit of RNA polymerase II.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Regulation of c-Myc gene transcription appears to be the one function of BRD4 targeted by identifying cancer therapeutic compound activate-related downstream target.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> CDK9 also specifically phosphorylates the AR on Ser81 and regulates AR transcriptional activity.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Western blot analysis showed that compounds JQ1 and compound <b>40</b> caused a dose-dependent reduction in c-Myc protein levels in the HCT-116 cell line. Meanwhile, Atuveciclib caused a dose-dependent decrease in p-AR and p-RNAP II CTD, but there was no significant trending effect on other proteins (The normalized quantified WB data were shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S4). As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, compound <b>40</b> demonstrated good inhibitory activities on CDK9 and BRD4 similar to JQ1 in vitro and large doses could block CDK9 and BRD4 activities for a short period of time.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. HCT-116 cells were treated with the indicated concentration of JQ1, Atuveciclib, or compound <b>40</b> and examined BRD4, c-Myc, CDK9, p-RNA II CTD, P-AR, and GAPDH protein levels by western blotting. The molecular weights of different isoforms of BRD4, c-Myc, and CDK9 are shown at the right of the bands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Pharmacokinetic Parameters of Compound <b>40</b> in ICR Mice<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">route</td><td class="colsep0 rowsep0" align="left">dose</td><td class="colsep0 rowsep0" align="left"><i>K</i><sub>el</sub> (h<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left"><i>T</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (ng·mL<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">AUC<sub>0–<i>t</i></sub> (h·ng·mL<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">AUC<sub>0–inf</sub> (h·ng·mL<sup>–1</sup>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IP</td><td class="colsep0 rowsep0" align="left">3 mg/kg</td><td class="colsep0 rowsep0" align="left">1.45</td><td class="colsep0 rowsep0" align="left">0.480</td><td class="colsep0 rowsep0" align="left">0.194</td><td class="colsep0 rowsep0" align="left">1366</td><td class="colsep0 rowsep0" align="left">757</td><td class="colsep0 rowsep0" align="left">759</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Mice (<i>n</i> = 3); parameters were calculated from composite mean plasma concentration–time data.</p></div></div></div><div class="NLM_p last">To complete the valuation of the lead compound <b>40</b>, we also investigated its in vivo pharmacokinetics (PK) parameters in the mice model (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). From the in vivo PK test, compound <b>40</b> dosed at 3 mg/kg showed a moderate plasma exposure with the AUC<sub>0–<i>t</i></sub> value of 757 h·ng·mL<sup>–1</sup>, indicating that the compound could be considered as a potential lead structure for further optimization.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Chemistry</h3><div class="NLM_p">Compound <b>7</b> is synthesized according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a><b>45</b> is obtained from nucleophilic substitution of 4-bromo-2-fluoro-1-nitrobenzene with <span class="smallcaps smallerCapital">d</span>-alanine, followed by cyclization to afford <b>46</b>. <b>48</b> was prepared by reductive amination with acetone and then alkylation with iodomethane. Then, <a href="http://cn.bing.com/dict/clientsearch?mkt=zh-CN&amp;setLang=zh&amp;form=BDVEHC&amp;ClientVer=BDDTV3.5.0.4311&amp;q=%E9%92%AF" class="ext-link">palladium</a>-catalyzed C–N coupling reaction was utilized to obtain <b>49</b>. After removing the <i>tert</i>-butoxycarbonyl (Boc) protecting group, a nucleophilic substitution reaction was adopted to give the intermediate <b>51a–51d</b>. Sequential acid hydrolysis and <a href="http://cn.bing.com/dict/clientsearch?mkt=zh-CN&amp;setLang=zh&amp;form=BDVEHC&amp;ClientVer=BDDTV3.5.0.4311&amp;q=%E6%B0%AF%E4%BB%A3" class="ext-link">chlorination</a> produced the compounds <b>53a–53d</b>. A <a href="http://cn.bing.com/dict/clientsearch?mkt=zh-CN&amp;setLang=zh&amp;form=BDVEHC&amp;ClientVer=BDDTV3.5.0.4311&amp;q=%E9%92%AF" class="ext-link">palladium</a>-catalyzed Suzuki coupling reaction between aryl chloride (<b>53a</b>) and phenylboronic acid yields end product <b>7</b>.</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <span class="smallcaps smallerCapital">d</span>-alanine, K<sub>2</sub>CO<sub>3</sub>, EtOH/H<sub>2</sub>O = 3:1, 80 °C, 8 h; (b) K<sub>2</sub>CO<sub>3</sub>, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, H<sub>2</sub>O, 60 °C, overnight; (c) acetone, phenylsilane, dibutyltindichloride, THF, rt, 10 h; (d) NaH, 0 °C–rt, 30 min and then iodomethane, DMF, rt, 2 h; (e) <i>tert</i>-butyl carbamate, CS<sub>2</sub>CO<sub>3</sub>, Pa(OAc)<sub>2</sub>, Xphos, 1.4-dioxane, 85 °C, 2 d; (f) HCl in 1.4-dioxane, rt, 24 h; (g) AcOH, 1.4-dioxane, 100 °C, 8 h; (h) HCl in 1.4-dioxane, 100 °C, 8 h; (i) POCl<sub>3</sub>, 60 °C, 8 h; and (j) K<sub>2</sub>CO<sub>3,</sub> Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, 1.4-dioxane: H<sub>2</sub>O = 4:1, 80 °C, 4 h.</p></p></figure><div class="NLM_p">Compounds <b>11-14</b>, as depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, were obtained through palladium-catalyzed Suzuki and Buchwald–Hartwig coupling reactions.</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>11–14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions:(a) phenylboronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, 1.4-dioxane/H<sub>2</sub>O = 4:1, 80 °C, 8 h; (b) <b>48</b>, <i>t</i>-BuONa, Pa<sub>2</sub>(dba)<sub>3</sub>, Xphos, 1.4-dioxane, 100 °C, overnight.</p></p></figure><div class="NLM_p">The synthetic routes of 3,5-dimethyl-isoxazole derivatives <b>5</b> and <b>6</b> are outlined in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Like the preparation of compounds <b>11–14</b>, compound <b>58</b> was obtained through two successive palladium-catalyzed coupling reactions. Acid hydrolysis, which could remove the OMe group, and <a href="http://cn.bing.com/dict/clientsearch?mkt=zh-CN&amp;setLang=zh&amp;form=BDVEHC&amp;ClientVer=BDDTV3.5.0.4311&amp;q=%E6%B0%AF%E4%BB%A3" class="ext-link">chlorination</a> afford intermediate <b>60</b>. As mentioned above, the final step also was Suzuki coupling reaction between <b>60</b> and the corresponding boronic acid derivatives.</div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>5</b> and <b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaHCO<sub>3,</sub> Pa(PPh<sub>3</sub>)<sub>4</sub>, 1.4-dioxane/H<sub>2</sub>O = 4:1, 70 °C, 2 h; (b) 2-amino-4-methoxypyrimidine, <i>t</i>-BuONa, Pa<sub>2</sub>(dba)<sub>3</sub>, Xphos, 1.4-dioxane, 100 °C, 8 h; (c) HCl in 1.4-dioxane, 100 °C, 8 h; (d) POCl<sub>3</sub>, 60 °C, 8 h; (e) K<sub>2</sub>CO<sub>3</sub>, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, 1.4-dioxane/H<sub>2</sub>O = 4:1, 80 °C, 4 h.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78676" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78676" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Over the past decades, many kinase inhibitors have entered into the market for the treatment of cancers. However, cancer is notorious for its high heterogeneity and usually harboring genetic and epigenetic abnormality; therefore, there is an urgent need of polypharmacological agents to simultaneously targeting multiple dysfunctional pathways. BET inhibitors have entered into clinical trials and demonstrated its antitumor effects and promising synergistic effects by combining with current kinase inhibitors. Although several dual inhibitors of BET and kinases have been discovered by serendipity, it remains a challenge for the researchers to rational design dual inhibitors of certain kinases and BET. In the present work, we adopted the hybrid strategy by incorporating typical kinase hinge binder with pharmacophores that bind to the KAc pocket of BET proteins and successfully obtained a series of 6-(pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-ones. Through structure-based drug design, we investigated the SAR and demonstrated that it is feasible to discover potent inhibitors of dual BET and kinases, providing not only a lead compound of CDK9 and BET, but also the promising approach for the community to design specific kinase and BET dual inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34411" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34411" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Chemistry</h3><div class="NLM_p"><sup>1</sup>H NMR spectra were recorded by using a Varian Mercury-400 high performance digital FT-NMR spectrometer with tetramethylsilane as an internal standard. Low-resolution mass spectra were obtained with a Finnigan LCQ Deca XP mass spectrometer using a CAPCELL PAK C18 (50 mm × 2.0 mm, 5 μm) or an Agilent ZORBAX Eclipse XDB C18 (50 mm × 2.1 mm, 5 μm) in positive or negative electrospray mode. The purity of compounds was determined by high-performance liquid chromatography (HPLC) and confirmed to be more than 95%, monitored by UV absorption at 214 and 254 nm. TLC analysis was carried out with glass precoated silica gel GF254 plates. TLC spots were visualized under UV light. All solvents and reagents were used directly as obtained commercially unless otherwise noted.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>N</i>-(3-(3,5-Dimethylisoxazol-4-yl)phenyl)-4-phenylpyrimidin-2-amine (<b>5</b>)</h4><div id="sec4_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Step 1: 4-(3-Bromophenyl)-3,5-dimethylisoxazole (<b>57</b>)</h5><div class="NLM_p last">To a solution of compound <b>56</b> (4 g, 14.2 mmol) and 3,5-dimethylisoxazole-4-boronic acid (1 g, 7.1 mmol) in 1.4-dioxane and H<sub>2</sub>O (4:1 50 mL) was added NaHCO<sub>3</sub> (1.2 g, 14.2 mmol); the mixture was bubbled with N<sub>2</sub> for 5 min; then, Pa(PPh<sub>3</sub>)<sub>4</sub> (820 mg, 0.71 mmol) was added; the mixture was further bubbled with N<sub>2</sub> for 5 min and was heated to 70 °C for 2 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with DCM (3 × 100 mL); the combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0–10% EtOAc/60–90 °C petroleum ether) to afford compound <b>57</b> (1.237 g, 4.906 mmol, 69.1% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.52 (ddt, <i>J</i> = 8.4, 3.6, 1.5 Hz, 1H), 7.43 (dt, <i>J</i> = 3.5, 1.8 Hz, 1H), 7.34 (td, <i>J</i> = 7.8, 3.1 Hz, 1H), 7.24–7.15 (m, 1H), 2.43 (d, <i>J</i> = 3.1 Hz, 3H), 2.29 (d, <i>J</i> = 3.1 Hz, 3H).</div></div><div id="sec4_1_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> Step 2: <i>N</i>-(3-(3,5-Dimethylisoxazol-4-yl)phenyl)-4-methoxypyrimidin-2-amine (<b>58</b>)</h5><div class="NLM_p last">To a solution of compound <b>57</b> (1.6 g, 6.4 mmol) and 2-amine-4-methoxypyrimidine (600 mg, 4.8 mmol) in anhydrous 1.4-dioxane (20 mL) was added <i>t</i>-BuONa (921 mg, 9.6 mmol); the mixture was bubbled with N<sub>2</sub> for 5 min; then, Pa<sub>2</sub>(dba)<sub>3</sub> (446 mg, 0.48 mmol) and Xphos (458 mg, 0.96 mmol) were added; the mixture was further bubbled with N<sub>2</sub> for 5 min and the mixture was heated to 100 °C for 8 h. The reaction was monitored by TLC. Upon completion, the resulting solution was filtered through Celite and concentrated; the residue was purified by silica gel column chromatography (gradient elution, gradient 10–33% EtOAc/60–90 °C petroleum ether) to afford compound <b>58</b> (510 mg, 4.32 mmol, 90% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.17 (d, <i>J</i> = 5.7 Hz, 1H), 7.64 (d, <i>J</i> = 1.8 Hz, 1H), 7.63–7.58 (m, 1H), 7.41 (t, <i>J</i> = 7.9 Hz, 1H), 7.16 (s, 1H), 6.98–6.88 (m, 1H), 6.29–6.18 (m, 1H), 3.96 (s, 3H), 2.46 (s, 3H), 2.33 (s, 3H).</div></div><div id="sec4_1_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> Step 3: 2-((3-(3,5-Dimethylisoxazol-4-yl)phenyl)amino)pyrimidin-4-ol (<b>59</b>)</h5><div class="NLM_p last">Compound <b>58</b> (510 mg, 1.12 mmol) was dissolved in 1.4-dioxane (10 mL) followed by the addition of 4 M HCl in 1.4-dioxane (20 mL), and the mixture was stirred at 100 °C for 24 h. The reaction was monitored by TLC. Upon completion, the resulting solution was poured into ice water slowly, the pH of this solution was adjusted to 7 with aqueous NaOH, and extracted with DCM (3 × 50 mL). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 2–10% MeOH/DCM) to afford compound <b>59</b> (341 mg, 0.784 mmol, 70% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.02 (d, <i>J</i> = 5.7 Hz, 1H), 7.91 (d, <i>J</i> = 5.7 Hz, 1H), 7.73 (d, <i>J</i> = 1.8 Hz, 1H), 7.67–7.63 (m, 1H), 7.41 (t, <i>J</i> = 7.9 Hz, 1H), 7.30 (m, 1H), 6.98 (m, 1H), 2.46 (s, 3H), 2.37 (s, 3H).</div></div><div id="sec4_1_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Step 4: 4-Chloro-<i>N</i>-(3-(3,5-dimethylisoxazol-4-yl)phenyl)pyrimidin-2-amine (<b>60</b>)</h5><div class="NLM_p last">Compound <b>59</b> (341 mg, 1.2 mmol) was dissolved in POCl<sub>3</sub> (10 mL) and then stirred at 60 °C for 8 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was poured into ice water slowly, the pH of this solution was adjusted to 7 with aqueous NaOH, and extracted with DCM (3 × 50 mL). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 20–33% EtOAc/60–90 °C petroleum ether) to afford compound <b>60</b> (254 mg, 1.008 mmol, 84% yield) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.99 (d, <i>J</i> = 5.9 Hz, 1H), 7.80 (d, <i>J</i> = 2.0 Hz, 1H), 7.71 (m, 1H), 7.45 (m, 1H), 7.29 (m, 1H), 7.01 (m, 1H), 2.40 (s, 3H), 2.33 (s, 3H).</div></div><div id="sec4_1_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> Step 5: <i>N</i>-(3-(3,5-Dimethylisoxazol-4-yl)phenyl)-4-phenylpyrimidin-2-amine (<b>5</b>)</h5><div class="NLM_p last">To a solution of compound <b>60</b> (80 mg, 0.266 mmol) and phenylboronic acid (65 mg, 0.532 mmol) in 1.4-dioxane and H<sub>2</sub>O (4:1 10 mL) was added K<sub>2</sub>CO<sub>3</sub> (74 mg, 0.532 mmol); the mixture was bubbled with N<sub>2</sub> for 5 min; then, Pa(dppf)<sub>2</sub>Cl<sub>2</sub> (22 mg, 0.053 mmol) was added; the mixture was further bubbled with N<sub>2</sub> for 5 min and was heated to 80 °C for 8 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with DCM (3 × 50 mL). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 10–33% EtOAc/60–90 °C petroleum ether) to afford compound <b>5</b> (30 mg, 0.088 mmol, 33% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.83 (s, 1H), 8.58 (d, <i>J</i> = 5.2 Hz, 1H), 8.21–8.07 (m, 2H), 7.94–7.87 (m, 2H), 7.55 (dd, <i>J</i> = 5.2, 2.1 Hz, 3H), 7.46–7.38 (m, 2H), 6.99 (dt, <i>J</i> = 7.7, 1.3 Hz, 1H), 2.44 (s, 3H), 2.26 (s, 3H). LC–MS (ESI) calcd for C<sub>21</sub>H<sub>18</sub>N<sub>4</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 343.15; found, 343.16 retention time 3.211 min, HPLC purity = 100%.</div></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-(3-(3,5-Dimethylisoxazol-4-yl)phenyl)-4-(pyridin-4-yl)pyrimidin-2-amine (<b>6</b>)</h4><div class="NLM_p last">Compound <b>6</b> was prepared as a white solid using a method similar to that described for compound <b>5</b> in 30% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.97 (s, 1H), 8.84–8.74 (m, 2H), 8.69 (d, <i>J</i> = 5.1 Hz, 1H), 8.11–8.02 (m, 2H), 7.97–7.88 (m, 1H), 7.81 (t, <i>J</i> = 1.9 Hz, 1H), 7.56 (d, <i>J</i> = 5.1 Hz, 1H), 7.44 (t, <i>J</i> = 7.9 Hz, 1H), 7.01 (dt, <i>J</i> = 7.6, 1.3 Hz, 1H), 2.44 (s, 3H), 2.26 (s, 3H). LC–MS (ESI) calcd for C<sub>20</sub>H<sub>17</sub>N<sub>5</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 344.14; found, 344.12. Retention time 3.230 min, HPLC purity = 99.98%.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-phenylpyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>7</b>)</h4><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> Step 1: (5-Bromo-2-nitrophenyl)-<span class="smallcaps smallerCapital">d</span>-alanine (<b>45</b>)</h5><div class="NLM_p last">To a solution of 4-bromo-2-fluoro-1-nitrobenzene (25.0 g, 113.6 mmol) in EtOH and H<sub>2</sub>O (3:1, 500 mL) were added <span class="smallcaps smallerCapital">d</span>-alanine (11.1 g, 125.0 mmol) and K<sub>2</sub>CO<sub>3</sub> (17.3 g, 125.0 mmol); the reaction mixture was heated to reflux for 8 h. The reaction was monitored by TLC. Upon completion, the resulting solution was evaporated under reduced pressure, and the crude product was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.35 (d, <i>J</i> = 6.9 Hz, 1H), 8.06 (d, <i>J</i> = 9.1 Hz, 1H), 6.90 (s, 1H), 6.85 (d, <i>J</i> = 9.2 Hz, 1H), 4.33 (p, <i>J</i> = 7.0 Hz, 1H), 1.67 (d, <i>J</i> = 7.0 Hz, 3H).</div></div><div id="sec4_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> Step 2: (<i>R</i>)-6-Bromo-3-methyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>46</b>)</h5><div class="NLM_p last">To a solution of compound <b>45</b> (28.7 g, 99.3 mmol) and K<sub>2</sub>CO<sub>3</sub> (27.4 g, 198.6 mmol) in H<sub>2</sub>O (500 mL) was added Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (86.5 g, 496.6 mmol) slowly, and the reaction mixture was heated to 60 °C overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with DCM (3 × 250 mL). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure to afford compound <b>46</b> (9 g, 373 mmol, 38% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.42 (s, 1H), 6.89–6.83 (dd, <i>J</i> = 8.2, 2.0 Hz, 1H), 6.81 (s, 1H), 6.68–6.59 (dd, <i>J</i> = 13.1, 7.6 Hz, 1H), 4.02 (q, <i>J</i> = 6.7 Hz, 1H), 3.92 (s, 1H), 1.45 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Step 3: (<i>R</i>)-6-Bromo-4-isopropyl-3-methyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>47</b>)</h5><div class="NLM_p last">A solution of compound <b>46</b> (9.0 g, 37.3 mmol), phenylsilane (11.9 g, 113.1 mmol), acetone (8.3 mL, 112.0 mmol), and dibutyltindichloride (17.0 g, 56.0 mmol) in THF (100 mL) were stirred at room temperature for 10 h. The reaction was monitored by TLC. Upon completion, the resulting solution evaporated; the residue was purified by silica gel column chromatography (gradient elution, gradient 10–25% EtOAc/60–90 °C petroleum ether) to afford compound <b>47</b> (8.8 g, 33.1 mmol, 87% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.03 (s, 1H), 6.96 (d, <i>J</i> = 1.9 Hz, 1H), 6.88 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 6.73 (d, <i>J</i> = 8.3 Hz, 1H), 4.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.95–3.80 (m, 1H), 1.33 (d, <i>J</i> = 6.7 Hz, 3H), 1.26 (d, <i>J</i> = 6.1 Hz, 3H), 1.20 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> Step 4: (<i>R</i>)-6-Bromo-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>48</b>)</h5><div class="NLM_p last">To a solution of compound <b>47</b> (8.8 g, 31.1 mmol) in anhydrous DMF (50 mL) was added NaH (1.3 g, 60% in mineral oil, 46.6 mmol) portion wise at 0 °C; the reaction mixture was stirred at 0 °C for 2 h, and then, iodomethane (2.9 mL, 46.6 mmol) was added at the same temperature, stirred at room temperature for additional 4 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with DCM (3 × 100 mL), the combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 10–25% EtOAc/60–90 °C petroleum ether) to afford compound <b>48</b> (8.8 g, 29.7 mmol, 95% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.04–6.93 (m, 3H), 4.15 (q, <i>J</i> = 6.7 Hz, 1H), 3.90 (hept, <i>J</i> = 6.5 Hz, 1H), 3.25 (s, 3H), 1.24 (d, <i>J</i> = 6.6 Hz, 3H), 1.17 (d, <i>J</i> = 6.6 Hz, 3H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Step 5: <i>tert</i>-Butyl (<i>R</i>)-(4-isopropyl-1,3-dimethyl-2-oxo-1,2,3,4-tetrahydroquinoxalin-6-yl) Carbamate (<b>49</b>)</h5><div class="NLM_p last"><i>tert</i>-Butyl carbamate (80.0 mg, 0.673 mmol) was added to a solution of compound <b>48</b> (50.0 mg, 0.168 mmol) and Cs<sub>2</sub>CO<sub>3</sub> (110 mg, 0.336 mmol) in 1.4-dioxane (5 mL); the mixture was bubbled with N<sub>2</sub> for 5 min; then, Pd(OAc)<sub>2</sub> (4.0 mg, 0.017 mmol) and Xphos (17.0 mg, 0.034 mmol) were added and the mixture was further bubbled with N<sub>2</sub> for 5 min. The reaction mixture was heated to 85 °C for 2 d. The reaction was monitored by TLC. Upon completion, the resulting solution was filtered through Celite and concentrated, and the residue was purified by silica gel column chromatography (gradient elution, gradient 20–50% EtOAc/60–90 °C petroleum ether) to afford compound <b>49</b> (40 mg, 0.120 mmol, 71% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.09 (s, 1H), 6.84 (d, <i>J</i> = 8.6 Hz, 1H), 6.76 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 6.69 (s, 1H), 4.15 (q, <i>J</i> = 6.8 Hz, 1H), 3.88 (h, <i>J</i> = 6.6 Hz, 1H), 3.34 (s, 3H), 1.52 (s, 9H), 1.33 (d, <i>J</i> = 6.7 Hz, 3H), 1.20 (d, <i>J</i> = 6.6 Hz, 3H), 1.08 (d, <i>J</i> = 6.8 Hz, 3H).</div></div><div id="sec4_1_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Step 6: (<i>R</i>)-6-Amino-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>50</b>)</h5><div class="NLM_p last">Compound <b>49</b> (40.0 mg, 0.120 mmol) was dissolved in 4 M HCl in 1.4-dioxane (5 mL) and then stirred at room temperature for 24 h. The reaction was monitored by TLC. Upon completion, the pH of this solution was adjusted to 7 with aqueous NaOH and extracted with DCM (3 × 200 mL). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 20–50% EtOAc/60–90 °C petroleum ether) to afford compound <b>50</b> (22 mg, 0.094 mmol, 80% yield) as a brown solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 6.66 (d, <i>J</i> = 8.4 Hz, 1H), 6.21–6.10 (m, 2H), 4.05 (q, <i>J</i> = 6.8 Hz, 1H), 3.74 (hept, <i>J</i> = 6.7 Hz, 1H), 3.57 (s, 2H), 3.24 (s, 3H), 1.19 (d, <i>J</i> = 6.7 Hz, 3H), 1.13 (d, <i>J</i> = 6.6 Hz, 3H), 1.01 (d, <i>J</i> = 6.8 Hz, 3H).</div></div><div id="sec4_1_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> Step 7: (<i>R</i>)-4-Isopropyl-6-((4-methoxypyrimidin-2-yl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>51a</b>)</h5><div class="NLM_p last">To a solution of compound <b>50</b> (2.0 g, 8.7 mmol) in 1.4-dioxane (20 mL) was added 2-chloro-4-methoxypyrimidine (1.6 g, 11.3 mmol) followed by acetic acid (2 mL); the reaction mixture was heated to 100 °C for 8 h. The reaction was monitored by TLC. Upon completion, the resulting solution was poured into ice water slowly, the pH of this solution was adjusted to 7 with aqueous NaOH, and extracted with DCM (3 × 100 mL). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 20–50% EtOAc/60–90 °C petroleum ether) to afford compound <b>51a</b> (1.1 g, 3.3 mmol, 92% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.40 (s, 1H), 8.18 (d, <i>J</i> = 5.6 Hz, 1H), 7.45 (d, <i>J</i> = 2.2 Hz, 1H), 7.28 (dd, <i>J</i> = 8.7, 2.3 Hz, 1H), 6.97 (d, <i>J</i> = 8.7 Hz, 1H), 6.25 (d, <i>J</i> = 5.6 Hz, 1H), 4.08 (q, <i>J</i> = 6.6 Hz, 1H), 3.92 (s, 3H), 3.83 (p, <i>J</i> = 6.7 Hz, 1H), 3.24 (s, 3H), 1.26 (d, <i>J</i> = 6.6 Hz, 3H), 1.19 (d, <i>J</i> = 6.6 Hz, 3H), 0.98 (d, <i>J</i> = 6.8 Hz, 3H).</div></div><div id="sec4_1_3_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Step 8: (<i>R</i>)-6-((4-Hydroxypyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>52a</b>)</h5><div class="NLM_p last">Compound <b>51a</b> (1.1 g, 3.3 mmol) was dissolved in 1.4-dioxane (5 mL) followed by the addition of 4 M HCl in 1.4-dioxane (15 mL); the mixture was stirred at 100 °C for 24 h. The reaction was monitored by TLC. Upon completion, the resulting solution was poured into ice water slowly and the pH of this solution was adjusted to 7 with aqueous NaOH and extracted with DCM (3 × 100 mL). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 2–10% MeOH/DCM) to afford compound <b>52a</b> (700 mg, 2.1 mmol, 65.3% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.77 (s, 1H), 8.80 (s, 1H), 7.72 (d, <i>J</i> = 6.5 Hz, 1H), 7.21 (d, <i>J</i> = 2.1 Hz, 1H), 7.13–6.95 (m, 2H), 5.80–5.73 (m, 1H), 4.11 (q, <i>J</i> = 6.7 Hz, 1H), 3.81 (p, <i>J</i> = 6.6 Hz, 1H), 3.25 (s, 3H), 1.26 (d, <i>J</i> = 6.6 Hz, 3H), 1.19 (d, <i>J</i> = 6.6 Hz, 3H), 0.98 (d, <i>J</i> = 6.8 Hz, 3H).</div></div><div id="sec4_1_3_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Step 9: (<i>R</i>)-6-((4-Chloropyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>53a</b>)</h5><div class="NLM_p last">Compound <b>52a</b> (700.0 mg, 2.1 mmol) was dissolved in POCl<sub>3</sub> (10 mL) and then stirred at 60 °C for 8 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was poured into ice water slowly, and the pH of this solution was adjusted to 7 with aqueous NaOH and extracted with DCM (3 × 50 mL). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 20–33% EtOAc/60–90 °C petroleum ether) to afford compound <b>53a</b> (145 mg, 0.419 mmol, 20% yield) as a red solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.89 (s, 1H), 8.40 (d, <i>J</i> = 5.1 Hz, 1H), 7.41 (d, <i>J</i> = 2.2 Hz, 1H), 7.17 (dd, <i>J</i> = 8.6, 2.2 Hz, 1H), 6.99 (d, <i>J</i> = 8.7 Hz, 1H), 6.91 (d, <i>J</i> = 5.2 Hz, 1H), 4.09 (q, <i>J</i> = 6.5 Hz, 1H), 3.79 (p, <i>J</i> = 6.6 Hz, 1H), 3.24 (s, 3H), 1.28 (d, <i>J</i> = 6.6 Hz, 3H), 1.21 (d, <i>J</i> = 6.8 Hz, 3H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_3_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Step 10: (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-phenylpyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>7</b>)</h5><div class="NLM_p last">To a solution of compound <b>53a</b> (50 mg, 0.145 mmol) and phenylboronic acid (27 mg, 0.217 mmol) in 1.4-dioxane and H<sub>2</sub>O (4:1 10 mL) was added K<sub>2</sub>CO<sub>3</sub> (40 mg, 0.289 mmol); the mixture was bubbled with N<sub>2</sub> for 5 min; then, Pa(dppf)<sub>2</sub>Cl<sub>2</sub> (12 mg, 0.0145 mmol) was added; the mixture was further bubbled with N<sub>2</sub> for 5 min and was heated to 80 °C for 8 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with DCM (3 × 50 mL), the combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 1–2% MeOH/DCM) to afford compound <b>7</b> (34 mg, 0.087 mmol, 60% yield) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.56 (s, 1H), 8.54 (d, <i>J</i> = 5.1 Hz, 1H), 8.18 (dd, <i>J</i> = 6.7, 3.0 Hz, 2H), 7.66 (s, 1H), 7.56 (dd, <i>J</i> = 5.1, 1.9 Hz, 3H), 7.38 (d, <i>J</i> = 5.1 Hz, 1H), 7.28 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 7.02 (d, <i>J</i> = 8.7 Hz, 1H), 4.10 (q, <i>J</i> = 6.7 Hz, 1H), 3.91 (p, <i>J</i> = 6.6 Hz, 1H), 3.26 (s, 3H), 1.28 (d, <i>J</i> = 6.6 Hz, 3H), 1.21 (d, <i>J</i> = 6.8 Hz, 3H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>23</sub>H<sub>25</sub>N<sub>5</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 388.21; found, 388.19. Retention time 3.208 min, HPLC purity = 97.779%.</div></div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-(pyridin-4-yl)pyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>8</b>)</h4><div class="NLM_p last">Compound <b>8</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 27% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.56 (s, 1H), 8.54 (d, <i>J</i> = 5.1 Hz, 1H), 8.18 (dd, <i>J</i> = 6.7, 3.0 Hz, 2H), 7.66 (s, 1H), 7.56 (dd, <i>J</i> = 5.1, 1.9 Hz, 3H), 7.38 (d, <i>J</i> = 5.1 Hz, 1H), 7.28 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 7.02 (d, <i>J</i> = 8.7 Hz, 1H), 4.10 (q, <i>J</i> = 6.7 Hz, 1H), 3.91 (p, <i>J</i> = 6.6 Hz, 1H), 3.26 (s, 3H), 1.28 (d, <i>J</i> = 6.6 Hz, 3H), 1.21 (d, <i>J</i> = 6.8 Hz, 3H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>22</sub>H<sub>24</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 389.20; found, 389.12. Retention time 2.508 min, HPLC purity = 98.415%.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (<i>R</i>)-4-Isopropyl-6-((4-(2-methoxyphenyl) pyrimidin-2-yl) amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>9</b>)</h4><div class="NLM_p last">Compound <b>9</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 29% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.52 (s, 1H), 8.46 (d, <i>J</i> = 5.2 Hz, 1H), 7.93 (dd, <i>J</i> = 7.6, 1.8 Hz, 1H), 7.70–7.60 (m, 1H), 7.50 (td, <i>J</i> = 7.9, 7.3, 1.8 Hz, 1H), 7.31 (d, <i>J</i> = 5.2 Hz, 1H), 7.21 (dd, <i>J</i> = 11.1, 7.7 Hz, 2H), 7.10 (t, <i>J</i> = 7.5 Hz, 1H), 6.98 (d, <i>J</i> = 8.7 Hz, 1H), 4.08 (q, <i>J</i> = 6.7 Hz, 1H), 3.88 (s, 3H), 3.87–3.78 (m, 1H), 3.25 (s, 3H), 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.19 (d, <i>J</i> = 6.5 Hz, 3H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub><i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 418.22; found, 418.24. LC–MS (ESI) calcd for C<sub>26</sub>H<sub>26</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 439.22; found, 439.14. Retention time 3.059 min, HPLC purity = 99.532%.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>R</i>)-3-(2-((4-Isopropyl-1,3-dimethyl-2-oxo-1,2,3,4-tetrahydroquinoxalin-6-yl)amino)pyrimidin-4-yl)benzonitrile (<b>10</b>)</h4><div class="NLM_p last">Compound <b>10</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 34% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.68 (s, 1H), 8.64–8.55 (m, 2H), 8.49 (dt, <i>J</i> = 8.0, 1.4 Hz, 1H), 8.04 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.77 (t, <i>J</i> = 7.9 Hz, 1H), 7.59 (s, 1H), 7.49 (d, <i>J</i> = 5.2 Hz, 1H), 7.24 (dd, <i>J</i> = 8.6, 2.2 Hz, 1H), 7.02 (d, <i>J</i> = 8.7 Hz, 1H), 4.11 (q, <i>J</i> = 6.7 Hz, 1H), 3.88 (p, <i>J</i> = 6.6 Hz, 1H), 3.26 (s, 3H), 1.26 (d, <i>J</i> = 6.6 Hz, 3H), 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 0.99 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>24</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 413.20; found, 413.06. Retention time 3.284 min, HPLC purity = 99.103%.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (<i>R</i>)-4-Isopropyl-6-((4-(3-methoxyphenyl)pyrimidin-2-yl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>15</b>)</h4><div class="NLM_p last">Compound <b>15</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 25.3% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.52 (s, 1H), 8.46 (d, <i>J</i> = 5.2 Hz, 1H), 7.93 (dd, <i>J</i> = 7.6, 1.8 Hz, 1H), 7.70–7.60 (m, 1H), 7.50 (td, <i>J</i> = 7.9, 7.3, 1.8 Hz, 1H), 7.31 (d, <i>J</i> = 5.2 Hz, 1H), 7.21 (dd, <i>J</i> = 11.1, 7.7 Hz, 2H), 7.10 (t, <i>J</i> = 7.5 Hz, 1H), 6.98 (d, <i>J</i> = 8.7 Hz, 1H), 4.08 (q, <i>J</i> = 6.7 Hz, 1H), 3.88 (s, 3H), 3.87–3.78 (m, 1H), 3.25 (s, 3H), 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.19 (d, <i>J</i> = 6.5 Hz, 3H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub><i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 418.22; found, 418.21. Retention time 3.211 min, HPLC purity = 99.65%.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (<i>R</i>)-4-Isopropyl-6-((4-(4-methoxyphenyl)pyrimidin-2-yl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>16</b>)</h4><div class="NLM_p last">Compound <b>16</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 25.7% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.52 (s, 1H), 8.46 (d, <i>J</i> = 5.2 Hz, 1H), 7.93 (dd, <i>J</i> = 7.6, 1.8 Hz, 1H), 7.70–7.60 (m, 1H), 7.50 (td, <i>J</i> = 7.9, 7.3, 1.8 Hz, 1H), 7.31 (d, <i>J</i> = 5.2 Hz, 1H), 7.21 (dd, <i>J</i> = 11.1, 7.7 Hz, 2H), 7.10 (t, <i>J</i> = 7.5 Hz, 1H), 6.98 (d, <i>J</i> = 8.7 Hz, 1H), 4.08 (q, <i>J</i> = 6.7 Hz, 1H), 3.88 (s, 3H), 3.87–3.78 (m, 1H), 3.25 (s, 3H), 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.19 (d, <i>J</i> = 6.5 Hz, 3H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub><i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 418.22; found, 418.24. Retention time 3.089 min, HPLC purity = 99.369%.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-(<i>o</i>-tolyl)pyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>17</b>)</h4><div class="NLM_p last">Compound <b>17</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 44% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.51 (s, 1H), 7.48 (d, <i>J</i> = 3.9 Hz, 2H), 7.38 (t, <i>J</i> = 7.4 Hz, 1H), 7.34–7.30 (m, 2H), 7.28 (s, 1H), 7.03 (d, <i>J</i> = 8.6 Hz, 1H), 6.91 (d, <i>J</i> = 8.5 Hz, 1H), 6.89–6.83 (m, 1H), 4.24–4.09 (m, 1H), 3.90 (p, <i>J</i> = 6.6 Hz, 1H), 3.38 (s, 3H), 2.47 (s, 3H), 1.27 (d, <i>J</i> = 6.7 Hz, 3H), 1.24 (d, <i>J</i> = 6.5 Hz, 3H), 1.13 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>27</sub>N<sub>5</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 402.22; found, 402.26. Retention time 3.507 min, HPLC purity = 95.934%.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-(<i>m</i>-tolyl)pyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>18</b>)</h4><div class="NLM_p last">Compound 1<b>8</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 33% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.56 (s, 1H), 8.55–8.49 (m, 1H), 7.96 (d, <i>J</i> = 9.6 Hz, 2H), 7.65–7.59 (m, 1H), 7.43 (t, <i>J</i> = 7.5 Hz, 1H), 7.36 (t, <i>J</i> = 6.2 Hz, 2H), 7.31–7.18 (m, 1H), 7.02 (d, <i>J</i> = 8.8 Hz, 1H), 4.10 (q, <i>J</i> = 6.7 Hz, 1H), 3.89 (p, <i>J</i> = 6.7 Hz, 1H), 3.26 (s, 3H), 2.41 (s, 3H), 1.26 (d, <i>J</i> = 6.6 Hz, 3H), 1.21 (d, <i>J</i> = 6.5 Hz, 3H), 0.99 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>27</sub>N<sub>5</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 402.22; found, 402.23. Retention time 3.288 min, HPLC purity = 97.466%.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-(<i>p</i>-tolyl)pyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>19</b>)</h4><div class="NLM_p last">Compound <b>19</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 29.6% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.53 (s, 1H), 8.50 (d, <i>J</i> = 5.2 Hz, 1H), 8.08 (d, <i>J</i> = 7.9 Hz, 2H), 7.65 (s, 1H), 7.39–7.31 (m, 3H), 7.27 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 7.01 (d, <i>J</i> = 8.7 Hz, 1H), 4.10 (q, <i>J</i> = 6.7 Hz, 1H), 3.91 (p, <i>J</i> = 6.7 Hz, 1H), 3.26 (s, 3H), 2.40 (s, 3H), 1.28 (d, <i>J</i> = 6.7 Hz, 3H), 1.22 (d, <i>J</i> = 6.7 Hz, 3H), 0.99 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>27</sub>N<sub>5</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 402.22; found, 402.23. Retention time 3.232 min, HPLC purity = 99.648%.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>R</i>)-6-((4-(2-Aminophenyl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>20</b>)</h4><div class="NLM_p last">Compound <b>20</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 15% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.53 (s, 1H), 8.42 (dd, <i>J</i> = 5.5, 0.9 Hz, 1H), 7.69 (d, <i>J</i> = 8.0 Hz, 1H), 7.34 (d, <i>J</i> = 2.2 Hz, 1H), 7.31–7.24 (m, 1H), 7.24–7.11 (m, 2H), 7.00 (d, <i>J</i> = 8.8 Hz, 3H), 6.80 (d, <i>J</i> = 8.2 Hz, 1H), 6.59 (td, <i>J</i> = 7.4, 1.3 Hz, 1H), 4.09 (p, <i>J</i> = 7.9, 7.3 Hz, 1H), 3.86 (p, <i>J</i> = 6.7 Hz, 1H), 3.26 (s, 3H), 1.27 (d, <i>J</i> = 6.6 Hz, 3H), 1.21 (d, <i>J</i> = 6.5 Hz, 3H), 0.99 (d, <i>J</i> = 6.6 Hz, 3H); LC–MS (ESI) calcd for C<sub>23</sub>H<sub>26</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 403.22; found, 403.15. Retention time 2.76 min, HPLC purity = 97.821%.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (<i>R</i>)-6-((4-(3-Aminophenyl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>21</b>)</h4><div class="NLM_p last">Compound <b>21</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 16.5% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.51 (s, 1H), 8.48 (d, <i>J</i> = 5.1 Hz, 1H), 7.63 (d, <i>J</i> = 2.3 Hz, 1H), 7.37–7.25 (m, 3H), 7.25–7.13 (m, 2H), 7.02 (d, <i>J</i> = 8.7 Hz, 1H), 6.73 (dt, <i>J</i> = 8.1, 1.6 Hz, 1H), 5.29 (s, 2H), 4.10 (q, <i>J</i> = 6.7 Hz, 1H), 3.90 (p, <i>J</i> = 6.6 Hz, 1H), 3.26 (s, 3H), 1.25 (d, <i>J</i> = 6.6 Hz, 3H), 1.21 (d, <i>J</i> = 6.5 Hz, 3H), 0.98 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>23</sub>H<sub>26</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 403.22; found, 403.21. Retention time 2.485 min, HPLC purity = 99.658%.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (<i>R</i>)-6-((4-(4-Aminophenyl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>22</b>)</h4><div class="NLM_p last">Compound <b>22</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 14% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.33 (s, 1H), 8.34 (d, <i>J</i> = 5.3 Hz, 1H), 7.92 (d, <i>J</i> = 8.3 Hz, 2H), 7.71 (s, 1H), 7.19 (dd, <i>J</i> = 21.1, 7.3 Hz, 2H), 7.00 (d, <i>J</i> = 8.7 Hz, 1H), 6.65 (d, <i>J</i> = 8.3 Hz, 2H), 5.77 (s, 2H), 4.10 (q, <i>J</i> = 6.7 Hz, 1H), 3.94 (p, <i>J</i> = 6.7 Hz, 1H), 3.26 (s, 3H), 1.29 (d, <i>J</i> = 6.5 Hz, 3H), 1.22 (t, <i>J</i> = 5.3 Hz, 3H), 0.99 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>23</sub>H<sub>26</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 403.22; found, 403.25. Retention time 2.651 min, HPLC purity = 97.661%.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (<i>R</i>)-6-((4-(4-Fluoro-2-methoxyphenyl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>23</b>)</h4><div class="NLM_p last">Compound <b>23</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 10% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.49 (s, 1H), 8.44 (d, <i>J</i> = 5.2 Hz, 1H), 7.97 (dd, <i>J</i> = 8.7, 7.1 Hz, 1H), 7.59 (d, <i>J</i> = 2.3 Hz, 1H), 7.29–7.18 (m, 2H), 7.10 (dd, <i>J</i> = 11.5, 2.4 Hz, 1H), 6.99–6.87 (m, 2H), 4.06 (q, <i>J</i> = 6.7 Hz, 1H), 3.88 (s, 3H), 3.80 (q, <i>J</i> = 6.6 Hz, 1H), 3.23 (s, 3H), 1.18 (dd, <i>J</i> = 13.5, 6.6 Hz, 6H), 0.95 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>26</sub>FN<sub>6</sub>O<sub>2</sub><i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 436.21; found, 436.23. Retention time 3.261 min, HPLC purity = 98.539%.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>R</i>)-6-((4-(4-Fluoro-2-methylphenyl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-on (<b>24</b>)</h4><div class="NLM_p last">Compound <b>24</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 17% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.56 (s, 1H), 8.49 (d, <i>J</i> = 5.0 Hz, 1H), 7.57–7.47 (m, 2H), 7.21 (dd, <i>J</i> = 9.7, 6.0, 2.4 Hz, 2H), 7.14 (td, <i>J</i> = 8.6, 2.7 Hz, 1H), 6.98–6.89 (m, 2H), 4.05 (q, <i>J</i> = 6.7 Hz, 1H), 3.74 (p, <i>J</i> = 6.6 Hz, 1H), 3.21 (s, 3H), 2.40 (s, 3H), 1.15 (t, <i>J</i> = 6.8 Hz, 6H), 0.94 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>26</sub>FN<sub>5</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 420.21; found, 420.23. Retention time 3.765 min, HPLC purity = 99.447%.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-(3-(methylamino)phenyl)pyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>25</b>)</h4><div class="NLM_p last">Compound <b>25</b> was prepared as a white solid using a method similar to that described for compound <b>7</b> in 9% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.50 (s, 1H), 8.49 (d, <i>J</i> = 5.2 Hz, 1H), 7.58 (d, <i>J</i> = 2.2 Hz, 1H), 7.37 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 7.33 (dt, <i>J</i> = 7.6, 1.3 Hz, 1H), 7.30–7.20 (m, 3H), 7.01 (d, <i>J</i> = 8.7 Hz, 1H), 6.72 (dd, <i>J</i> = 8.0, 2.4, 1.0 Hz, 1H), 4.09 (q, <i>J</i> = 6.7 Hz, 1H), 3.88 (p, <i>J</i> = 6.7 Hz, 1H), 3.26 (s, 3H), 2.76 (s, 3H), 1.25 (d, <i>J</i> = 6.6 Hz, 3H), 1.20 (d, <i>J</i> = 6.5 Hz, 3H), 0.99 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>28</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 417.23; found, 417.25. Retention time 2.647 min, HPLC purity = 99.204%.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (<i>R</i>)-6-((4-(3-(Ethylamino)phenyl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>26</b>)</h4><div class="NLM_p last">Compound <b>26</b> was prepared as a white solid using a method similar to that described for compound <b>7</b> in 7% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.50 (s, 1H), 8.49 (d, <i>J</i> = 5.2 Hz, 1H), 7.59 (d, <i>J</i> = 2.2 Hz, 1H), 7.45–7.28 (m, 3H), 7.23 (dd, <i>J</i> = 8.8, 6.4 Hz, 2H), 7.00 (d, <i>J</i> = 8.7 Hz, 1H), 6.78–6.66 (m, 1H), 4.10 (q, <i>J</i> = 6.7 Hz, 1H), 3.89 (p, <i>J</i> = 6.6 Hz, 1H), 3.26 (s, 3H), 3.13 (q, <i>J</i> = 7.1 Hz, 2H), 1.28–1.14 (m, 9H), 0.99 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>25</sub>H<sub>30</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 431.25; found, 431.26. Retention time 2.683 min, HPLC purity = 97.197%.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (<i>R</i>)-4-Isopropyl-6-((4-(3-(isopropylamino)phenyl)pyrimidin-2-yl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>27</b>)</h4><div class="NLM_p last">Compound <b>27</b> was prepared as a white solid using a method similar to that described for compound <b>7</b> in 12% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.49 (s, 1H), 8.49 (d, <i>J</i> = 5.2 Hz, 1H), 7.57 (d, <i>J</i> = 2.2 Hz, 1H), 7.38 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 7.32–7.26 (m, 2H), 7.21 (dd, <i>J</i> = 9.3, 6.4 Hz, 2H), 6.99 (d, <i>J</i> = 8.7 Hz, 1H), 6.73 (dt, <i>J</i> = 7.9, 1.5 Hz, 1H), 5.61 (d, <i>J</i> = 8.0 Hz, 1H), 4.10 (q, <i>J</i> = 6.7 Hz, 1H), 3.89 (p, <i>J</i> = 6.6 Hz, 1H), 3.65 (q, <i>J</i> = 6.6 Hz, 1H), 3.26 (s, 3H), 1.26 (d, <i>J</i> = 6.6 Hz, 3H), 1.21 (d, <i>J</i> = 6.6 Hz, 3H), 1.19–1.14 (m, 6H), 0.99 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>26</sub>H<sub>32</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 445.26; found, 445.24. Retention time 2.697 min, HPLC purity = 100%.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (<i>R</i>)-6-((4-(3-(Cyclopentylamino)phenyl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>28</b>)</h4><div class="NLM_p last">Compound 2<b>8</b> was prepared as a white solid using a method similar to that described for compound <b>7</b> in 7% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.45 (d, <i>J</i> = 5.2 Hz, 1H), 7.46–7.33 (m, 2H), 7.31 (q, <i>J</i> = 3.8, 3.0 Hz, 2H), 7.29 (m, 1H), 7.19–7.16 (m, 1H), 7.14 (d, <i>J</i> = 2.3 Hz, 1H), 7.12 (d, <i>J</i> = 5.2 Hz, 1H), 6.94 (d, <i>J</i> = 8.6 Hz, 1H), 6.75 (dd, <i>J</i> = 7.9, 2.4, 1.0 Hz, 1H), 4.20 (q, <i>J</i> = 6.8 Hz, 1H), 3.98 (p, <i>J</i> = 6.7 Hz, 1H), 3.90 (p, <i>J</i> = 6.2 Hz, 1H), 3.40 (s, 3H), 2.08 (td, <i>J</i> = 14.7, 13.4, 6.8 Hz, 2H), 1.78 (q, <i>J</i> = 6.0, 4.6 Hz, 2H), 1.72–1.62 (m, 2H), 1.52 (dt, <i>J</i> = 13.4, 6.7 Hz, 2H), 1.34 (d, <i>J</i> = 6.7 Hz, 3H), 1.28 (d, <i>J</i> = 6.5 Hz, 3H), 1.15 (d, <i>J</i> = 6.8 Hz, 3H); LC–MS (ESI) calcd for C<sub>28</sub>H<sub>34</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 471.28; found, 471.20. Retention time 3.032 min, HPLC purity = 98.164%.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (<i>R</i>)-6-((4-(3-(Cyclohexylamino)phenyl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>29</b>)</h4><div class="NLM_p last">Compound <b>29</b> was prepared as a white solid using a method similar to that described for compound <b>7</b> in 6.6% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.49 (s, 1H), 8.49 (d, <i>J</i> = 5.2 Hz, 1H), 7.53 (d, <i>J</i> = 2.2 Hz, 1H), 7.40 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 7.32 (s, 1H), 7.27 (d, <i>J</i> = 7.5 Hz, 1H), 7.24–7.15 (m, 2H), 7.00 (d, <i>J</i> = 8.7 Hz, 1H), 6.75 (d, <i>J</i> = 8.0 Hz, 1H), 5.63 (m, 1H), 4.10 (q, <i>J</i> = 6.7 Hz, 1H), 3.88 (p, <i>J</i> = 6.5 Hz, 1H), 3.26 (s, 3H), 1.96 (d, <i>J</i> = 12.6 Hz, 2H), 1.73 (d, <i>J</i> = 12.7 Hz, 2H), 1.61 (d, <i>J</i> = 12.6 Hz, 2H), 1.28–1.22 (m, 7H), 1.21 (d, <i>J</i> = 6.6 Hz, 3H), 0.99 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>29</sub>H<sub>36</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 485.30: found, 485.16. Retention time 3.055 min, HPLC purity = 96.265%.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-(3-(phenylamino)phenyl)pyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>30</b>)</h4><div class="NLM_p last">Compound <b>30</b> was prepared as a white solid using a method similar to that described for compound <b>7</b> in 29% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.53 (s, 1H), 8.52 (d, <i>J</i> = 5.1 Hz, 1H), 8.39 (s, 1H), 7.91 (d, <i>J</i> = 2.0 Hz, 1H), 7.57 (dt, <i>J</i> = 7.7, 1.2 Hz, 1H), 7.52 (d, <i>J</i> = 2.2 Hz, 1H), 7.42–7.33 (m, 2H), 7.30–7.19 (m, 4H), 7.14 (dq, <i>J</i> = 6.9, 1.4 Hz, 2H), 6.93–6.81 (m, 2H), 4.08 (q, <i>J</i> = 6.7 Hz, 1H), 3.86 (p, <i>J</i> = 6.6 Hz, 1H), 3.24 (s, 3H), 1.23 (d, <i>J</i> = 6.6 Hz, 3H), 1.16 (d, <i>J</i> = 6.6 Hz, 3H), 0.96 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>29</sub>H<sub>30</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 479.25; found, 479.19. Retention time 3.565 min, HPLC purity = 97.132%.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (<i>R</i>)-6-((4-(3-Amino-2-methylphenyl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>31</b>)</h4><div class="NLM_p last">Compound <b>31</b> was prepared as a white solid using a method similar to that described for compound <b>7</b> in 22% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.49 (s, 1H), 8.45 (d, <i>J</i> = 5.0 Hz, 1H), 7.62–7.56 (m, 1H), 7.24 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 6.94 (dd, <i>J</i> = 8.5, 3.6 Hz, 2H), 6.82 (d, <i>J</i> = 5.0 Hz, 1H), 6.68 (d, <i>J</i> = 2.4 Hz, 1H), 6.56 (dd, <i>J</i> = 8.1, 2.5 Hz, 1H), 5.00 (s, 2H), 4.04 (q, <i>J</i> = 6.6 Hz, 1H), 3.77 (p, <i>J</i> = 6.6 Hz, 1H), 3.21 (s, 3H), 2.19 (s, 3H), 1.15 (t, <i>J</i> = 6.0 Hz, 6H), 0.94 (d, <i>J</i> = 6.8 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>28</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 417.23; found, 417.19. Retention time 2.519 min, HPLC purity = 95.896%.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (<i>R</i>)-6-((4-(5-Amino-2-methylphenyl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>32</b>)</h4><div class="NLM_p last">Compound <b>32</b> was prepared as a white solid using a method similar to that described for compound <b>7</b> in 25.4% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.50 (s, 1H), 8.44 (d, <i>J</i> = 5.0 Hz, 1H), 7.62–7.56 (m, 1H), 7.23 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 7.02–6.90 (m, 2H), 6.80 (d, <i>J</i> = 5.0 Hz, 1H), 6.71 (dd, <i>J</i> = 8.0, 1.3 Hz, 1H), 6.61 (dd, <i>J</i> = 7.6, 1.3 Hz, 1H), 5.00 (s, 2H), 4.04 (q, <i>J</i> = 6.7 Hz, 1H), 3.75 (p, <i>J</i> = 6.6 Hz, 1H), 3.21 (s, 3H), 2.05 (s, 3H), 1.15 (dd, <i>J</i> = 9.5, 6.6 Hz, 6H), 0.94 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>24</sub>H<sub>28</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 417.23; found, 417.20. Retention time 2.537 min, HPLC purity = 95.749%.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-(1,2,3,4-tetrahydroquinolin-5-yl)pyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>33</b>)</h4><div class="NLM_p last">Compound <b>33</b> was prepared as a white solid using a method similar to that described for compound <b>7</b> in 24.6% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.42 (d, <i>J</i> = 5.1 Hz, 1H), 7.46 (d, <i>J</i> = 2.3 Hz, 1H), 7.09–6.97 (m, 2H), 6.88 (d, <i>J</i> = 8.5 Hz, 1H), 6.81 (d, <i>J</i> = 5.1 Hz, 1H), 6.74 (d, <i>J</i> = 7.4 Hz, 1H), 6.55 (d, <i>J</i> = 8.0 Hz, 1H), 4.14–4.07 (m, 1H), 3.89 (p, <i>J</i> = 6.6 Hz, 1H), 3.35 (s, 3H), 3.32 (t, <i>J</i> = 5.6 Hz, 2H), 2.81 (dt, <i>J</i> = 6.2, 3.1 Hz, 2H), 1.87 (q, <i>J</i> = 5.9 Hz, 2H), 1.26 (d, <i>J</i> = 6.9 Hz, 3H), 1.22 (d, <i>J</i> = 6.6 Hz, 3H), 1.10 (d, <i>J</i> = 6.8 Hz, 3H); LC–MS (ESI) calcd for C<sub>26</sub>H<sub>30</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 443.25; found, 443.24. Retention time 2.579 min, HPLC purity = 99.864%.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (<i>R</i>)-6-((4-(1<i>H</i>-Indol-3-yl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>34</b>)</h4><div class="NLM_p last">Compound <b>34</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 26.3% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.80 (d, <i>J</i> = 2.9 Hz, 1H), 9.27 (s, 1H), 8.60 (d, <i>J</i> = 7.8 Hz, 1H), 8.31 (dd, <i>J</i> = 8.5, 4.1 Hz, 2H), 7.51–7.45 (m, 2H), 7.40 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 7.27–7.11 (m, 3H), 7.01 (d, <i>J</i> = 8.7 Hz, 1H), 4.09 (q, <i>J</i> = 6.7 Hz, 1H), 3.85 (p, <i>J</i> = 6.6 Hz, 1H), 3.28 (s, 3H), 1.23 (d, <i>J</i> = 6.2 Hz, 3H), 1.18 (d, <i>J</i> = 6.5 Hz, 3H), 1.00 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>25</sub>H<sub>26</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 427.22; found, 427.27. Retention time 2.779 min, HPLC purity = 99.724%.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (<i>R</i>)-6-((4-(1<i>H</i>-Indol-4-yl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>35</b>)</h4><div class="NLM_p last">Compound <b>35</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 29.8% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.40 (s, 1H), 9.52 (s, 1H), 8.52 (d, <i>J</i> = 5.2 Hz, 1H), 7.73–7.64 (m, 2H), 7.59 (dd, <i>J</i> = 8.1, 1.0 Hz, 1H), 7.51 (t, <i>J</i> = 2.8 Hz, 1H), 7.36–7.28 (m, 2H), 7.24 (t, <i>J</i> = 7.7 Hz, 1H), 7.07 (s, 1H), 6.99 (d, <i>J</i> = 8.7 Hz, 1H), 4.07 (q, <i>J</i> = 6.7 Hz, 1H), 3.85 (p, <i>J</i> = 6.6 Hz, 1H), 3.26 (s, 3H), 1.16 (dd, <i>J</i> = 6.6, 2.6 Hz, 6H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>25</sub>H<sub>26</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 427.22; found, 427.28. Retention time 2.875 min, HPLC purity = 98.975%.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-(2-methyl-2<i>H</i>-indazol-4-yl)pyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>36</b>)</h4><div class="NLM_p last">Compound <b>36</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 36% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.50 (s, 1H), 9.00 (s, 1H), 8.52 (d, <i>J</i> = 5.3 Hz, 1H), 7.86 (d, <i>J</i> = 7.0 Hz, 1H), 7.78 (d, <i>J</i> = 8.5 Hz, 1H), 7.49–7.40 (m, 2H), 7.41–7.27 (m, 2H), 7.03 (d, <i>J</i> = 8.7 Hz, 1H), 4.20 (s, 3H), 4.08 (q, <i>J</i> = 6.7 Hz, 1H), 3.80 (p, <i>J</i> = 6.7 Hz, 1H), 3.26 (s, 3H), 1.18 (dd, <i>J</i> = 11.0, 6.8 Hz, 6H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>25</sub>H<sub>27</sub>N<sub>7</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 442.23; found, 442.25. Retention time 2.836 min, HPLC purity = 98.257%.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (<i>R</i>)-4-Isopropyl-6-((4-(isoquinolin-5-yl)pyrimidin-2-yl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>37</b>)</h4><div class="NLM_p last">Compound <b>37</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 44% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.72 (s, 1H), 9.45–9.39 (m, 1H), 8.63 (d, <i>J</i> = 5.0 Hz, 1H), 8.52 (d, <i>J</i> = 6.0 Hz, 1H), 8.28 (d, <i>J</i> = 8.2 Hz, 1H), 8.19 (d, <i>J</i> = 6.0 Hz, 1H), 8.02 (dd, <i>J</i> = 7.2, 1.2 Hz, 1H), 7.81 (dd, <i>J</i> = 8.2, 7.2 Hz, 1H), 7.70 (s, 1H), 7.13 (d, <i>J</i> = 5.0 Hz, 2H), 6.93 (d, <i>J</i> = 8.7 Hz, 1H), 3.98 (q, <i>J</i> = 6.5 Hz, 1H), 3.55 (m, 1H), 3.20 (s, 3H), 0.99 (d, <i>J</i> = 6.5 Hz, 3H), 0.88 (d, <i>J</i> = 6.7 Hz, 3H), 0.85–0.72 (m, 3H); LC–MS (ESI) calcd for C<sub>26</sub>H<sub>26</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 439.22; found, 439.12. Retention time 2.598 min, HPLC purity = 100%.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-(quinolin-5-yl)pyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>38</b>)</h4><div class="NLM_p last">Compound <b>38</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 47% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.74 (s, 1H), 8.99 (dd, <i>J</i> = 4.1, 1.7 Hz, 1H), 8.77–8.70 (m, 1H), 8.64 (d, <i>J</i> = 5.0 Hz, 1H), 8.18 (dt, <i>J</i> = 8.2, 1.2 Hz, 1H), 7.94–7.80 (m, 2H), 7.71 (s, 1H), 7.58 (dd, <i>J</i> = 8.7, 4.1 Hz, 1H), 7.15 (t, <i>J</i> = 6.8 Hz, 2H), 6.94 (d, <i>J</i> = 8.7 Hz, 1H), 4.00 (q, <i>J</i> = 6.6 Hz, 1H), 3.59–3.39 (m, 1H), 3.22 (s, 3H), 1.00 (d, <i>J</i> = 6.5 Hz, 3H), 0.90 (d, <i>J</i> = 6.7 Hz, 3H), 0.84 (d, <i>J</i> = 8.9 Hz, 3H); LC–MS (ESI) calcd for C<sub>26</sub>H<sub>26</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 439.22; found, 439.20. Retention time 2.637 min, HPLC purity = 98.598%.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((4-(quinolin-4-yl)pyrimidin-2-yl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>39</b>)</h4><div class="NLM_p last">Compound <b>39</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 49% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.05 (d, <i>J</i> = 4.4 Hz, 1H), 8.62 (d, <i>J</i> = 4.9 Hz, 1H), 8.33–8.20 (m, 2H), 7.80 (ddd, <i>J</i> = 8.4, 6.8, 1.4 Hz, 1H), 7.62–7.54 (m, 3H), 7.45 (s, 1H), 7.05 (d, <i>J</i> = 4.9 Hz, 1H), 6.96 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 6.89 (d, <i>J</i> = 8.6 Hz, 1H), 4.14 (q, <i>J</i> = 6.8 Hz, 1H), 3.75–3.64 (m, 1H), 3.36 (s, 3H), 1.12 (d, <i>J</i> = 6.5 Hz, 3H), 1.07 (d, <i>J</i> = 6.8 Hz, 3H), 0.99 (d, <i>J</i> = 6.6 Hz, 3H); LC–MS (ESI) calcd for C<sub>26</sub>H<sub>26</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 439.22; found, 439.19. Retention time 2.931 min, HPLC purity = 95.3%.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (<i>R</i>)-4-Isopropyl-6-((4-(isoquinolin-4-yl)pyrimidin-2-yl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>40</b>)</h4><div class="NLM_p last">Compound <b>40</b> was prepared as a yellow solid using a method similar to that described for compound <b>7</b> in 60% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.73 (s, 1H), 9.44 (s, 1H), 8.70 (s, 1H), 8.64 (d, <i>J</i> = 4.9 Hz, 1H), 8.35 (d, <i>J</i> = 8.4 Hz, 1H), 8.25 (d, <i>J</i> = 8.0 Hz, 1H), 7.84 (m,, 1H), 7.77 (m, 1H), 7.72 (s, 1H), 7.18 (d, <i>J</i> = 4.9 Hz, 1H), 7.14 (d, <i>J</i> = 8.8 Hz, 1H), 6.93 (d, <i>J</i> = 8.7 Hz, 1H), 3.98 (q, <i>J</i> = 6.6 Hz, 1H), 3.46 (m, 1H), 3.20 (s, 3H), 0.98 (d, <i>J</i> = 6.5 Hz, 3H), 0.88 (d, <i>J</i> = 6.7 Hz, 3H), 0.82 (m, 3H). LC–MS (ESI) calcd for C<sub>26</sub>H<sub>26</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 439.22; found, 439.14. Retention time 2.668 min, HPLC purity = 99.351%.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (<i>R</i>)-6-((5-Fluoro-4-(isoquinolin-4-yl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>41</b>)</h4><div class="NLM_p last">Following the procedure for the synthesis of compound <b>7</b> from compound <b>53a</b>, compound <b>41</b> was obtained with <b>53b</b> instead of <b>53a</b> as a yellow solid with 23% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.82 (s, 1H), 9.49 (s, 1H), 8.73 (dd, <i>J</i> = 16.2, 1.7 Hz, 2H), 8.28 (d, <i>J</i> = 7.9 Hz, 1H), 8.01 (d, <i>J</i> = 8.4 Hz, 1H), 7.81 (dddd, <i>J</i> = 23.5, 8.0, 6.8, 1.3 Hz, 2H), 7.64 (s, 1H), 7.11–7.04 (m, 1H), 6.92 (d, <i>J</i> = 8.7 Hz, 1H), 3.97 (q, <i>J</i> = 6.7 Hz, 1H), 3.55–3.47 (m, 1H), 3.19 (s, 3H), 0.98 (d, <i>J</i> = 6.5 Hz, 3H), 0.87 (d, <i>J</i> = 6.7 Hz, 3H), 0.82 (d, <i>J</i> = 5.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>26</sub>H<sub>25</sub>FN<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 457.21; found, 457.18. Retention time 2.789 min, HPLC purity = 96.046%.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (<i>R</i>)-6-((5-Chloro-4-(isoquinolin-4-yl)pyrimidin-2-yl)amino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>42</b>)</h4><div class="NLM_p last">Following the procedure for the synthesis of compound <b>7</b> from compound <b>53a</b>, compound <b>42</b> was obtained with <b>53c</b> instead of <b>53a</b> as a yellow solid with 22% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.96 (s, 1H), 9.46 (s, 1H), 8.73 (s, 1H), 8.61 (s, 1H), 8.26 (dd, <i>J</i> = 7.5, 2.0 Hz, 1H), 7.86–7.72 (m, 3H), 7.59 (s, 1H), 7.03 (s, 1H), 6.92 (d, <i>J</i> = 8.6 Hz, 1H), 3.96 (d, <i>J</i> = 7.1 Hz, 1H), 3.48 (m, 1H), 3.18 (s, 3H), 0.94 (m, 3H), 0.85 (d, <i>J</i> = 6.7 Hz, 3H), 0.78 (m, 3H); LC–MS (ESI) calcd for C<sub>26</sub>H<sub>25</sub>ClN<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 473.18; found, 473.14. Retention time 2.868 min, HPLC purity = 97.252%.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (<i>R</i>)-4-Isopropyl-6-((4-(isoquinolin-4-yl)-5-methylpyrimidin-2-yl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>43</b>)</h4><div class="NLM_p last">Following the procedure for the synthesis of compound <b>7</b> from compound <b>53a</b>, compound <b>43</b> was obtained with <b>53d</b> instead of <b>53a</b> as a yellow solid with 26% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.58 (s, 1H), 9.43 (s, 1H), 8.53 (s, 2H), 8.25 (dd, <i>J</i> = 7.3, 1.9 Hz, 1H), 7.83–7.72 (m, 2H), 7.69 (s, 2H), 7.05–6.98 (m, 1H), 6.88 (d, <i>J</i> = 8.7 Hz, 1H), 3.94 (q, <i>J</i> = 6.7 Hz, 1H), 3.39 (m, 1H), 3.17 (s, 3H), 1.96 (s, 3H), 0.92 (d, <i>J</i> = 6.5 Hz, 3H), 0.85 (d, <i>J</i> = 6.7 Hz, 3H), 0.75 (m 3H); LC–MS (ESI) calcd for C<sub>27</sub>H<sub>28</sub>N<sub>6</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 453.23; found, 453.19. Retention time 2.679 min, HPLC purity = 98.367%.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (<i>R</i>)-6-([1,1′-Biphenyl]-3-ylamino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>11</b>)</h4><div id="sec4_1_36_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> Step 1: [1,1′-Biphenyl]-3-amine (<b>55a</b>)</h5><div class="NLM_p last">To a solution of 3-bromoaniline (212 mg, 1.2 mmol) and K<sub>2</sub>CO<sub>3</sub> (227 mg, 1.6 mmol) in 1.4-dioxane/H<sub>2</sub>O = 4:1(10 mL) was added phenylboronic acid (100 mg, 0.821 mmol); the mixture was bubbled with N<sub>2</sub> for 5 min; then, Pd(dppf)<sub>2</sub>Cl<sub>2</sub> (67 mg, 0.082 mmol) was added and the mixture was further bubbled with N<sub>2</sub> for other 5 min. The mixture was heated to 80 °C for 8 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with DCM (3 × 50 mL). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 10–50% EtOAc/60–90 °C petroleum ether) to afford compound <b>55a</b> (87 mg, 0.515 mmol, 62.3% yield) as a brown solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.62–7.56 (m, 2H), 7.49–7.41 (m, 2H), 7.40–7.35 (m, 1H), 7.26 (d, <i>J</i> = 7.8 Hz, 1H), 7.05–7.00 (m, 1H), 6.95 (d, <i>J</i> = 2.1 Hz, 1H), 6.75–6.68 (m, 1H), 3.69 (s, 2H).</div></div><div id="sec4_1_36_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> Step 2: (<i>R</i>)-6-([1,1′-Biphenyl]-3-ylamino)-4-isopropyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>11</b>)</h5><div class="NLM_p last">To a solution of compound <b>55a</b> (50 mg, 0.295 mmol) and compound <b>48</b> (106 mg, 0.355 mmol) in anhydrous 1.4-dioxane (10 mL) was added <i>t</i>-BuONa (57 mg, 0.570 mmol); the mixture was bubbled with N<sub>2</sub> for 5 min; then, Pa<sub>2</sub>(dba)<sub>3</sub> (28 mg, 0.036 mmol) and Xphos (29 mg, 0.072 mmol) were added, the mixture was further bubbled with N<sub>2</sub> for 5 min and was heated to 100 °C for 8 h. The reaction was monitored by TLC. Upon completion, the resulting solution was filtered through Celite and concentrated; the residue was purified by silica gel column chromatography (gradient elution, gradient 1–5% MeOH/DCM) to afford compound <b>11</b> (34 mg, 0.088 mmol, 29.8% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.12 (s, 1H), 7.60–7.52 (m, 2H), 7.43 (t, <i>J</i> = 7.7 Hz, 2H), 7.38–7.29 (m, 1H), 7.32–7.22 (m, 2H), 7.05–6.92 (m, 3H), 6.65 (dd, <i>J</i> = 6.5, 2.3 Hz, 2H), 4.07 (q, <i>J</i> = 6.7 Hz, 1H), 3.79 (p, <i>J</i> = 6.6 Hz, 1H), 3.22 (s, 3H), 1.22 (d, <i>J</i> = 6.7 Hz, 3H), 1.18 (d, <i>J</i> = 6.5 Hz, 3H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H); LC–MS (ESI) calcd for C<sub>25</sub>H<sub>27</sub>N<sub>3</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 386.22; found, 386.07. Retention time 3.98 min, HPLC purity = 95.337%.</div></div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> (<i>R</i>)-4-Isopropyl-1,3-dimethyl-6-((3-(pyridin-4-yl)phenyl)amino)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>12</b>)</h4><div class="NLM_p last">Compound <b>12</b> was prepared as a white solid using a method similar to that described for compound <b>11</b> in 22% yield. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.66 (d, <i>J</i> = 5.1 Hz, 2H), 7.53–7.47 (m, 2H), 7.38 (t, <i>J</i> = 7.8 Hz, 1H), 7.32–7.26 (m, 1H), 7.13 (dd, <i>J</i> = 22.4, 7.8, 1.9 Hz, 2H), 6.91 (d, <i>J</i> = 8.3 Hz, 1H), 6.69 (d, <i>J</i> = 9.7 Hz, 2H), 5.88 (s, 1H), 4.20 (q, <i>J</i> = 6.8 Hz, 1H), 3.79 (dq, <i>J</i> = 36.6, 6.8 Hz, 1H), 3.39 (s, 3H), 1.30 (d, <i>J</i> = 6.7 Hz, 3H), 1.28–1.25 (m, 3H), 1.15 (d, <i>J</i> = 6.8 Hz, 3H). LC–MS (ESI) calcd for C<sub>24</sub>H<sub>26</sub>N<sub>4</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 387.21; found, 387.22. Retention time 2.687 min, HPLC purity = 98.815%.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> (<i>R</i>)-4-Isopropyl-6-((2′-methoxy-[1,1′-biphenyl]-3-yl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>13</b>)</h4><div class="NLM_p last">Compound <b>13</b> was prepared as a white solid using a method similar to that described for compound <b>11</b> in 15% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.07 (s, 1H), 7.37–7.29 (m, 1H), 7.28–7.20 (m, 2H), 7.14 (d, <i>J</i> = 1.9 Hz, 1H), 7.10 (d, <i>J</i> = 8.3 Hz, 1H), 7.03–6.91 (m, 3H), 6.85 (d, <i>J</i> = 7.6 Hz, 1H), 6.70–6.63 (m, 2H), 4.08 (q, <i>J</i> = 6.6 Hz, 1H), 3.78 (m, 1H), 3.77 (s, 3H), 3.23 (s, 3H), 1.24 (d, <i>J</i> = 6.7 Hz, 3H), 1.18 (d, <i>J</i> = 6.6 Hz, 3H), 0.98 (d, <i>J</i> = 6.6 Hz, 3H). LC–MS (ESI) calcd for C<sub>26</sub>H<sub>29</sub>N<sub>3</sub>O<sub>2</sub><i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 415.23; found, 416.08. Retention time 3.899 min, HPLC purity = 95.415%.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> (<i>R</i>)-3′-((4-Isopropyl-1,3-dimethyl-2-oxo-1,2,3,4-tetrahydroquinoxalin-6-yl)amino)-[1,1′-biphenyl]-3-carbonitrile (<b>14</b>)</h4><div class="NLM_p last">Compound <b>14</b> was prepared as a white solid using a method similar to that described for compound <b>11</b> in 19% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.17 (s, 1H), 8.02 (t, <i>J</i> = 1.8 Hz, 1H), 7.96–7.88 (m, 1H), 7.81 (dt, <i>J</i> = 7.8, 1.3 Hz, 1H), 7.64 (t, <i>J</i> = 7.8 Hz, 1H), 7.35–7.26 (m, 2H), 7.12–7.01 (m, 2H), 6.96 (d, <i>J</i> = 9.0 Hz, 1H), 6.69–6.62 (m, 2H), 4.08 (q, <i>J</i> = 6.7 Hz, 1H), 3.79 (p, <i>J</i> = 6.6 Hz, 1H), 3.22 (s, 3H), 1.23 (d, <i>J</i> = 6.7 Hz, 3H), 1.18 (d, <i>J</i> = 6.5 Hz, 3H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H). LC–MS (ESI) calcd for C<sub>26</sub>H<sub>26</sub>N<sub>4</sub>O <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: 411.21; found, 411.08. Retention time 3.836 min, HPLC purity = 95.697%.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> BRD4-BD1 Protein Expression and Crystallization</h3><div class="NLM_p">The BRD4-BD1 protein expression followed our previous method.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Briefly, colonies from freshly transformed plasmid DNA in <i>Escherichia coli</i> BL21 (DE3)-codon plus-RIL cells were grown overnight at 37 °C in 50 mL of Terrific Broth medium; then, cells were grown at 37 °C to an optical density of ∼0.8 at OD600 before the temperature was decreased to 16 °C. The system was equilibrated at 16 °C until the optical density was ∼1.2 at OD600. Then, protein expression was induced overnight at 16 °C with 0.2 mmol isopropyl-β-<span class="smallcaps smallerCapital">d</span>-thiogalactopyranoside (IPTG). The bacteria were harvested, and cells expressing His6-tagged proteins were purified with the previous protocol.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Finally, the eluted protein was treated overnight at 4 °C with tobacco etch virus (TEV) protease to remove the His6 tag. The protein was concentrated and further purified with size exclusion chromatography on a Superdex 75 16/60 HiLoad gel filtration column. Samples were monitored by SDS-polyacrylamide gel electrophoresis and concentrated to 8–10 mg/mL for protein binding assay and crystallization.</div><div class="NLM_p">The vapor diffusion method was used for crystallization of aliquots of the purified proteins. The complex structure of BRD4-BD1 and the ligands were grown at 18 °C in 1 μL of protein with an equal volume of reservoir solution containing the sodium formate and 10–20% glycerol. Crystals grew to diffracting quality within 1 week in all cases.</div><div class="NLM_p last">Data were collected at 100 K on beam line BL17U at the Shanghai Synchrotron Radiation Facility (SSRF) (Shanghai, China) for the co-crystallized structures.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The data were processed with the HKL2000<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> software packages, and the structures were then solved by molecular replacement using the CCP4 program MOLREP.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The search model used for the crystals was the BRD4-JQ1 complex structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a>). The structures were refined using the CCP4 program REFMAC5 combined with the simulated annealing protocol implemented in the program PHENIX.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> With the aid of the program Coot,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> compound, water molecules, and others were fitted into the initial <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> maps.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Docking Study</h3><div class="NLM_p last">The BRD4-BD1 structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI5">6JI5</a>) and CDK9 structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH">6GZH</a>) were processed with the protein preparation module in Schrödinger software package, and KAc binding site of BRD4-BD1 and ATP binding site of CDK9 were selected for making the grid file with Glide module.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Then, the ligands were prepared to add the partial charge, minimize to obtain the low-energy conformation. Finally, ligands were docked into the grid files with default parameters at the standard precision mode, and the predicted binding modes were depicted with PyMOL software.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> BRD4-BD1 FP Assay</h3><div class="NLM_p last">The binding of compounds to BRD4 was assessed using a fluorescence anisotropy binding assay as reported before.<a onclick="showRef(event, 'ref18 ref19 ref20 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref42 ref43">(18−20,42,43)</a> Generally, the method involves incubating the bromodomain protein BRD4 and the fluorescent ligand to reach thermodynamic equilibrium. The test compound or DMSO vehicle (2‰ final) in a Corning 384 well Black low volume plate (CLS3575) was added to this reaction mixture and equilibrated in the dark for 4 h at room temperature. The anisotropy of the unbound fluorescent ligand is measurably different from the bound value. Fluorescence anisotropy was read on a BioTek Synergy2 multimode microplate reader (λ<sub>ex</sub> = 485 nm, λ<sub>em</sub> = 530 nm; dichroism, −505 nM).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> CDK9 Enzymatic Activity Assay</h3><div class="NLM_p last">The mobility shift assay method was used for testing CDK9 activity. The CDK9 protein was purchased from Carna (batch no. 14CBS-0084G), and the Caliper substrate CTD3 was purchased from GL (batch no. P171207-MJ346885). Generally, the compounds were incubated with the CDK9 protein and substrate CTD3 to reach thermodynamic equilibrium. Then, the activity was measured with the mobility shift method. Dinaciclib was used as the positive control.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Bromodomain Selectivity</h3><div class="NLM_p last">Bromodomains of BRD4-BD2, BRD2-BD1, BRD2-BD2, and EP300 were expressed similar to BRD4-BD1. The FP assay protocol was used to test the binding activities of compound <b>40</b> toward these bromodomains.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Kinase Profiling Study</h3><div class="NLM_p last">A diversity kinase panel from Eurofin Scientific company (<a href="http://www.eurofins.com" class="extLink">www.eurofins.com</a>) were chosen to assess the kinase selectivity, which included about 60 kinases from all kinases subfamilies. The result was listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Table S1</a>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Cellular Antiproliferation Assay</h3><div class="NLM_p last">Different cells were cultured in the corresponding medium conditions recommended by the suppliers. Different cells were seeded onto 96-well plates at a suitable density in a volume of 100 μL medium. After incubation overnight, compounds dissolved in DMSO stock solutions were thawed at room temperature and diluted to the desired concentrations with normal saline. The compounds were added to the assay plate and after 96 h of incubation, the IC<sub>50</sub> was measured with the Cell Counting Kit-8 (CCK-8) assay (Dojindo, Japan) for suspended cells or sulforhodamine B (SRB) (Sigma; MO, USA) for adherent cells.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Western Blot Test</h3><div class="NLM_p last">HCT116 cells were plated into 6-well plates and incubated for 24 h. Then, cells were harvested after 24 h of treatment with compounds at the indicated doses. Western blotting was performed using a previously described method.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> In Vivo Pharmacokinetic Study</h3><div class="NLM_p last">Compound <b>40</b> dissolved in 1% DMSO/0.5% HPMC (5/95, v/v) to a concentration of 1 mg/mL was given to ICR mice (male, 20–22 g, <i>n</i> = 3) by intraperitoneal injection administration. Animal procedures were performed according to institutional ethical guidelines of animal care. Blood samples were collected at 0.25, 0.5, 1, 2, 4, 8, and 24 h after administration (anticoagulant: EDTA–Na<sub>2</sub>); 100 μL of solvent of methanol/acetonitrile (1:1, v/v) with internal standard was added to 10 μL of plasma and vortexed thoroughly. It was centrifuged for 5 min; then, 20 μL of the supernatant was mixed with 20 μL of water for analysis. Samples were analyzed by Xevo TQ-S triple quadrupole mass spectrometer (Waters, USA). The ACQUITY UPLC BEH C18 (1.7 μm, 2.0 mm × 50 mm, Waters, USA) was used for the analysis. Gradient elution was applied consisting of 5 mM ammonium acetate aqueous solution containing 0.1% formic acid and acetonitrile containing 0.1% formic acid. After analyzing the concentrations of these compounds, the value of AUClast, AUCINF_obs and MRTINF_obs was calculated from time–concentration curves in each animal using Phoenix WinNonlin (CERTARA, USA). Cmax was determined as the maximum plasma concentration, and <i>T</i><sub>max</sub> was the time to reach the maximum concentration.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i85"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00962" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87044" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87044" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00962?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00962</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Kinase selectivity of compound <b>40</b>, CDK2 selectivity result of partial compounds, BET BDs selectivity of compound <b>40</b>, non-BET BDs selectivity of compound <b>40</b>, proliferation inhibition of 22 cancer cell lines of compound <b>40</b>, statistical information of crystallography, details of the docking study, superimposition of crystal structures of BI-2536 with compound 7, triple experiments of the western blot study of compounds <b>40</b>, and the spectra of typical synthesized compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Table of molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">PDB files used in the docking study (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_003.zip" class="ext-link">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf">jm0c00962_si_001.pdf (1.07 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_002.csv">jm0c00962_si_002.csv (3.71 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_003.zip">jm0c00962_si_003.zip (300.25 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Cocrystal structures of compounds <b>7</b> and <b>40</b> bound to BRD4-BD1 were deposited into the PDB database with the entry code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIH">6LIH</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIM">6LIM</a>, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00962" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73058" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73058" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jie Mou</span> - <span class="hlFld-Affiliation affiliation">Jiangsu
Key Laboratory of New Drug and Clinical Pharmacy, School of Pharmacy, Xuzhou Medical University, 209 Tongshan Road, Xuzhou 221006, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5b6a6b6b6b6b696b6b626c6a6b1b232133362e753e3f2e753835"><span class="__cf_email__" data-cfemail="9baaababababa9ababa2acaaabdbe3e1f3f6eeb5feffeeb5f8f5">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dongsheng Pei</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, Xuzhou Medical University, 209 Tongshan Road, Xuzhou 221006, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8febfcffeae6cff7f5e7e2faa1eaebfaa1ece1"><span class="__cf_email__" data-cfemail="1e7a6d6e7b775e666476736b307b7a6b307d70">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bing Xiong</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9776-8136" title="Orcid link">http://orcid.org/0000-0001-9776-8136</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a9cbd1c0c6c7cee9dac0c4c487c8ca87cac7"><span class="__cf_email__" data-cfemail="6c0e140503020b2c1f050101420d0f420f02">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaikai Lv</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weicong Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, Xuzhou Medical University, 209 Tongshan Road, Xuzhou 221006, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danqi Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huijie Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tiantian Fan</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanlian Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danyan Cao</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lin Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jingkang Shen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>K.L. and W.C. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i88">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66605" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66605" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful for financial supports from the National Science & Technology Major Project “Key New Drug Creation and Manufacturing Program” of China (Grant nos. 2018ZX09711002-011-018 and 2018ZX09711002-004) and the National Natural Science Foundation of China (Grant no. 81330076). For solving the X-ray crystal structures, we thank the help from the staff of BL17U of Shanghai Synchrotron Radiation Facility, China.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal domain</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetylated lysine</p></td></tr><tr><td class="NLM_term">vdW</td><td class="NLM_def"><p class="first last">van der Waals</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last"><i>tert</i>-butoxycarbonyl</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i90">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60148" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60148" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 44 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anighoro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology: challenges and opportunities in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">7874</span>– <span class="NLM_lpage">7887</span>, <span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+challenges+and+opportunities+in+drug+discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0ljvWgg3fSoedA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520challenges%2520and%2520opportunities%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raghavendra, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pingili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadasi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mettu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, S. V. U. M.</span></span> <span> </span><span class="NLM_article-title">Dual or multi-targeting inhibitors: The next generation anticancer agents</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">1277</span>– <span class="NLM_lpage">1300</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.ejmech.2017.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=29126724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslyrt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2018&pages=1277-1300&author=N.+M.+Raghavendraauthor=D.+Pingiliauthor=S.+Kadasiauthor=A.+Mettuauthor=S.+V.+U.+M.+Prasad&title=Dual+or+multi-targeting+inhibitors%3A+The+next+generation+anticancer+agents&doi=10.1016%2Fj.ejmech.2017.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Dual or multi-targeting inhibitors: The next generation anticancer agents</span></div><div class="casAuthors">Raghavendra, Nulgumnalli Manjunathaiah; Pingili, Divya; Kadasi, Sundeep; Mettu, Akhila; Prasad, S. V. U. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1277-1300</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Dual-targeting/Multi-targeting of oncoproteins by a single drug mol. represents an efficient, logical and alternative approach to drug combinations.  An increasing interest in this approach is indicated by a steady upsurge in the no. of articles on targeting dual/multi proteins published in the last 5 years.  Combining different inhibitors that destiny specific single target is the std. treatment for cancer.  A new generation of dual or multi-targeting drugs is emerging, where a single chem. entity can act on multiple mol. targets.  Dual/Multi-targeting agents are beneficial for solving limited efficiencies, poor safety and resistant profiles of an individual target.  Designing dual/multi-target inhibitors with predefined biol. profiles present a challenge.  The latest advances in bioinformatic tools and the availability of detailed structural information of target proteins have shown a way of discovering multi-targeting mols.  This neoteric artifice that amalgamates the mol. docking of small mols. with protein-based common pharmacophore to design multi-targeting inhibitors is gaining great importance in anticancer drug discovery.  Current review focus on the discoveries of dual targeting agents in cancer therapy using rational, computational, proteomic, bioinformatics and polypharmacol. approach that enables the discovery and rational design of effective and safe multi-target anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVpBUpxW1okLVg90H21EOLACvtfcHk0ljvWgg3fSoedA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslyrt7jO&md5=a14783194c23f7dc5f200cf6eabf9269</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRaghavendra%26aufirst%3DN.%2BM.%26aulast%3DPingili%26aufirst%3DD.%26aulast%3DKadasi%26aufirst%3DS.%26aulast%3DMettu%26aufirst%3DA.%26aulast%3DPrasad%26aufirst%3DS.%2BV.%2BU.%2BM.%26atitle%3DDual%2520or%2520multi-targeting%2520inhibitors%253A%2520The%2520next%2520generation%2520anticancer%2520agents%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D143%26spage%3D1277%26epage%3D1300%26doi%3D10.1016%2Fj.ejmech.2017.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Lera, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic polypharmacology: from combination therapy to multitargeted drugs</span>. <i>Clin. Epigenet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1186/s13148-016-0271-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1186%2Fs13148-016-0271-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=27752293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslOgsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=105-126&author=A.+R.+de%0ALeraauthor=A.+Ganesan&title=Epigenetic+polypharmacology%3A+from+combination+therapy+to+multitargeted+drugs&doi=10.1186%2Fs13148-016-0271-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic polypharmacology: from combination therapy to multitargeted drugs</span></div><div class="casAuthors">de Lera, Angel R.; Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Epigenetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105/1-105/21</span>CODEN:
                <span class="NLM_cas:coden">CELPCI</span>;
        ISSN:<span class="NLM_cas:issn">1868-7083</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  The modern drug discovery process has largely focused its attention in the so-called magic bullets, single chem. entities that exhibit high selectivity and potency for a particular target.  This approach was based on the assumption that the deregulation of a protein was causally linked to a disease state, and the pharmacol. intervention through inhibition of the deregulated target was able to restore normal cell function.  However, the use of cocktails or multicomponent drugs to address several targets simultaneously is also popular to treat multifactorial diseases such as cancer and neurol. disorders.  We review the state of the art with such combinations that have an epigenetic target as one of their mechanisms of action.  Epigenetic drug discovery is a rapidly advancing field, and drugs targeting epigenetic enzymes are in the clinic for the treatment of hematol. cancers.  Approved and exptl. epigenetic drugs are undergoing clin. trials in combination with other therapeutic agents via fused or linked pharmacophores in order to benefit from synergistic effects of polypharmacol.  In addn., ligands are being discovered which, as single chem. entities, are able to modulate multiple epigenetic targets simultaneously (multitarget epigenetic drugs).  These multiple ligands should in principle have a lower risk of drug-drug interactions and drug resistance compared to cocktails or multicomponent drugs.  This new generation may rival the so-called magic bullets in the treatment of diseases that arise as a consequence of the deregulation of multiple signaling pathways provided the challenge of optimization of the activities shown by the pharmacophores with the different targets is addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphm-KIEGyYKrVg90H21EOLACvtfcHk0lg2LMSWwPLP_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslOgsbY%253D&md5=7cd89a93a22121cc9db2c7118e2e23f1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1186%2Fs13148-016-0271-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13148-016-0271-9%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BLera%26aufirst%3DA.%2BR.%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DEpigenetic%2520polypharmacology%253A%2520from%2520combination%2520therapy%2520to%2520multitargeted%2520drugs%26jtitle%3DClin.%2520Epigenet.%26date%3D2016%26volume%3D8%26spage%3D105%26epage%3D126%26doi%3D10.1186%2Fs13148-016-0271-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramadoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahadevan, V.</span></span> <span> </span><span class="NLM_article-title">Targeting the cancer epigenome: synergistic therapy with bromodomain inhibitors</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.drudis.2017.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=28943305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOgu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=76-89&author=M.+Ramadossauthor=V.+Mahadevan&title=Targeting+the+cancer+epigenome%3A+synergistic+therapy+with+bromodomain+inhibitors&doi=10.1016%2Fj.drudis.2017.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the cancer epigenome: synergistic therapy with bromodomain inhibitors</span></div><div class="casAuthors">Ramadoss, Mahalakshmi; Mahadevan, Vijayalakshmi</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-89</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Epigenetic and genomic alterations regulate the transcriptional landscape of cells during cancer onset and progression.  Recent clin. studies targeting the epigenetic 'readers' (bromodomains) for cancer therapy have established the effectiveness of bromodomain (BRD) and extraterminal (BET) inhibitors in treating several types of cancer.  In this review, we discuss key mechanisms of BET inhibition and synergistic combinations of BET inhibitors with histone deacetylase inhibitors (HDACi), histone methyltransferase inhibitors (HMTi), DNA methyltransferase inhibitors (DNMTi), kinase, B-cell lymphoma 2 (Bcl-2) and proteosome inhibitors, and immunomodulatory drugs for cancer therapy.  We also highlight the potential of such combinations to overcome drug resistance, and the evolving approaches to developing novel BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXLHGMo7cO6LVg90H21EOLACvtfcHk0lg2LMSWwPLP_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOgu7bJ&md5=490a32952a2b907f71a88c1e6292db77</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DRamadoss%26aufirst%3DM.%26aulast%3DMahadevan%26aufirst%3DV.%26atitle%3DTargeting%2520the%2520cancer%2520epigenome%253A%2520synergistic%2520therapy%2520with%2520bromodomain%2520inhibitors%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D76%26epage%3D89%26doi%3D10.1016%2Fj.drudis.2017.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donawho, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlhapp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Preclinical characterization of bet family bromodomain inhibitor abbv-075 suggests combination therapeutic strategies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2976</span>– <span class="NLM_lpage">2989</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-16-1793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1158%2F0008-5472.CAN-16-1793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=28416490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Siurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2976-2989&author=M.+H.+Buiauthor=X.+Linauthor=D.+H.+Albertauthor=L.+Liauthor=L.+T.+Lamauthor=E.+J.+Faivreauthor=S.+E.+Warderauthor=X.+Huangauthor=D.+Wilcoxauthor=C.+K.+Donawhoauthor=G.+S.+Sheppardauthor=L.+Wangauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+Hasvoldauthor=T.+Uzielauthor=X.+Luauthor=F.+Kohlhappauthor=G.+Fangauthor=S.+W.+Elmoreauthor=S.+H.+Rosenbergauthor=K.+F.+McDanielauthor=W.+M.+Katiauthor=Y.+Shen&title=Preclinical+characterization+of+bet+family+bromodomain+inhibitor+abbv-075+suggests+combination+therapeutic+strategies&doi=10.1158%2F0008-5472.can-16-1793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical Characterization of BET Family Bromodomain Inhibitor ABBV-075 Suggests Combination Therapeutic Strategies</span></div><div class="casAuthors">Bui, Mai H.; Lin, Xiaoyu; Albert, Daniel H.; Li, Leiming; Lam, Lloyd T.; Faivre, Emily J.; Warder, Scott E.; Huang, Xiaoli; Wilcox, Denise; Donawho, Cherrie K.; Sheppard, George S.; Wang, Le; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa; Uziel, Tamar; Lu, Xin; Kohlhapp, Fred; Fang, Guowei; Elmore, Steven W.; Rosenberg, Saul H.; McDaniel, Keith F.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2976-2989</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">ABBV-075 is a potent and selective BET family bromodomain inhibitor that recently entered phase I clin. trials.  Comprehensive preclin. characterization of ABBV-075 demonstrated broad activity across cell lines and tumor models, representing a variety of hematol. malignancies and solid tumor indications.  In most cancer cell lines derived from solid tumors, ABBV-075 triggers prominent G1 cell-cycle arrest without extensive apoptosis.  In this study, we show that ABBV-075 efficiently triggers apoptosis in acute myeloid leukemia (AML), non-Hodgkin lymphoma, and multiple myeloma cells.  Apoptosis induced by ABBV-075 was mediated in part by modulation of the intrinsic apoptotic pathway, exhibiting synergy with the BCL-2 inhibitor venetoclax in preclin. models of AML.  In germinal center diffuse large B-cell lymphoma, BCL-2 levels or venetoclax sensitivity predicted the apoptotic response to ABBV-075 treatment.  In vivo combination studies uncovered surprising benefits of low doses of ABBV-075 coupled with bortezomib and azacitidine treatment, despite the lack of in vitro synergy between ABBV-075 and these agents.  The in vitro/in vivo activities of ABBV-075 described here may serve as a useful ref. to guide the development of ABBV-075 and other BET family inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6P1blFwkIvrVg90H21EOLACvtfcHk0lg2LMSWwPLP_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Siurw%253D&md5=d9cc4abc44cfd9c0e07383f908592c2e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-1793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-1793%26sid%3Dliteratum%253Aachs%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DDonawho%26aufirst%3DC.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DKohlhapp%26aufirst%3DF.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DPreclinical%2520characterization%2520of%2520bet%2520family%2520bromodomain%2520inhibitor%2520abbv-075%2520suggests%2520combination%2520therapeutic%2520strategies%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2976%26epage%3D2989%26doi%3D10.1158%2F0008-5472.can-16-1793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Astorgues-Xerri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odore, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erba, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltrame, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frapolli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Incalci, M.</span></span> <span> </span><span class="NLM_article-title">The bromodomain inhibitor OTX015 (MK-8628) exerts antitumor activity in triple-negative breast cancer models as single agent and in combination with everolimus</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">7598</span>– <span class="NLM_lpage">7613</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.18632%2Foncotarget.13814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=27935867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252Fis1Cntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=7598-7613&author=R.+V%C3%A1zquezauthor=M.+E.+Riveiroauthor=L.+Astorgues-Xerriauthor=E.+Odoreauthor=K.+Rezaiauthor=E.+Erbaauthor=N.+Paniniauthor=A.+Rinaldiauthor=I.+Kweeauthor=L.+Beltrameauthor=M.+Bekraddaauthor=E.+Cvitkovicauthor=F.+Bertoniauthor=R.+Frapolliauthor=M.+D%E2%80%99Incalci&title=The+bromodomain+inhibitor+OTX015+%28MK-8628%29+exerts+antitumor+activity+in+triple-negative+breast+cancer+models+as+single+agent+and+in+combination+with+everolimus&doi=10.18632%2Foncotarget.13814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain inhibitor OTX015 (MK-8628) exerts anti-tumor activity in triple-negative breast cancer models as single agent and in combination with everolimus</span></div><div class="casAuthors">Vazquez Ramiro; Erba Eugenio; Panini Nicolo; Beltrame Luca; Frapolli Roberta; D'Incalci Maurizio; Riveiro Maria E; Astorgues-Xerri Lucile; Odore Elodie; Bekradda Mohamed; Cvitkovic Esteban; Odore Elodie; Rezai Keyvan; Rinaldi Andrea; Kwee Ivo; Bertoni Francesco; Kwee Ivo; Kwee Ivo; Cvitkovic Esteban; Bertoni Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">7598-7613</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Triple-negative breast cancer (TNBC) is an aggressive and heterogeneous subgroup of breast tumors clinically defined by the lack of estrogen, progesterone and HER2 receptors, limiting the use of the targeted therapies employed in other breast malignancies.  Recent evidence indicates that c-MYC is a key driver of TNBC.  The BET-bromodomain inhibitor OTX015 (MK-8628) has potent antiproliferative activity accompanied by c-MYC down-regulation in several tumor types, and has demonstrated synergism with the mTOR inhibitor everolimus in different models.  The aim of this study was to evaluate the anti-tumor activity of OTX015 as single agent and in combination with everolimus in TNBC models.  OTX015 was assayed in three human TNBC-derived cell lines, HCC1937, MDA-MB-231 and MDA-MB-468, all showing antiproliferative activity after 72 h (GI50 = 75-650 nM).  This was accompanied by cell cycle arrest and decreased expression of cancer stem cells markers.  However, c-MYC protein and mRNA levels were only down-regulated in MDA-MB-468 cells.  Gene set enrichment analysis showed up-regulation of genes involved in epigenetic control of transcription, chromatin and the cell cycle, and down-regulation of stemness-related genes.  In vitro, combination with everolimus was additive in HCC1937 and MDA-MB-231 cells, but antagonistic in MDA-MB-468 cells.  In MDA-MB-231 murine xenografts, tumor mass was significantly (p < 0.05) reduced by OTX015 with respect to vehicle-treated animals (best T/C = 40.7%).  Although everolimus alone was not active, the combination was more effective than OTX015 alone (best T/C = 20.7%).  This work supports current clinical trials with OTX015 in TNBC (NCT02259114).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSpC5NvWs-BQRJ5E29YNGvQfW6udTcc2eYietsWsK9NSbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252Fis1Cntw%253D%253D&md5=ac5f6998cb398ab9dbb9c9493880a5c7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13814%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A1zquez%26aufirst%3DR.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DAstorgues-Xerri%26aufirst%3DL.%26aulast%3DOdore%26aufirst%3DE.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DErba%26aufirst%3DE.%26aulast%3DPanini%26aufirst%3DN.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBeltrame%26aufirst%3DL.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26aulast%3DFrapolli%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Incalci%26aufirst%3DM.%26atitle%3DThe%2520bromodomain%2520inhibitor%2520OTX015%2520%2528MK-8628%2529%2520exerts%2520antitumor%2520activity%2520in%2520triple-negative%2520breast%2520cancer%2520models%2520as%2520single%2520agent%2520and%2520in%2520combination%2520with%2520everolimus%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D7598%26epage%3D7613%26doi%3D10.18632%2Foncotarget.13814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors in anticancer drug discovery: a structural and pharmacological perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9305</span>– <span class="NLM_lpage">9320</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00438</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00438" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9305-9320&author=L.+Carlinoauthor=G.+Rastelli&title=Dual+kinase-bromodomain+inhibitors+in+anticancer+drug+discovery%3A+a+structural+and+pharmacological+perspective&doi=10.1021%2Facs.jmedchem.6b00438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span></div><div class="casAuthors">Carlino, Luca; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9305-9320</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases play crucial roles in several cell transformation processes and are validated drug targets for many human diseases, including cancer.  Nevertheless, most tumors have eluded the effects of inhibition of a single kinase by activating resistance mechanisms and/or alternative pathways and escape mechanisms.  In recent years, multitarget approaches directed toward inhibition of kinases and targets of different families have received increasing attention.  In particular, co-targeting kinases and bromodomain epigenetic reader proteins has rapidly emerged as a promising approach to cancer drug development.  In this manuscript, we will review the recent discoveries that led to the identification and optimization of dual kinase/bromodomain inhibitors.  We will analyze and compare the structural features required for dual inhibition and comment on the potential of this approach in anticancer drug discovery.  Moreover, we will introduce computational approaches useful for the identification of dual kinase/bromodomain inhibitors and generate ad hoc pharmacophore and docking models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkHDSBWCGdbVg90H21EOLACvtfcHk0lhIn-Z2p6NI7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP&md5=1351a235bba09782160a86ae3f904833</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00438%26sid%3Dliteratum%253Aachs%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520in%2520anticancer%2520drug%2520discovery%253A%2520a%2520structural%2520and%2520pharmacological%2520perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9305%26epage%3D9320%26doi%3D10.1021%2Facs.jmedchem.6b00438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Acetyl-lysine binding site of bromodomain-containing protein 4 (BRD4) interacts with diverse kinase inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&author=S.+W.+J.+Emberauthor=J.-Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Sch%C3%B6nbrunn&title=Acetyl-lysine+binding+site+of+bromodomain-containing+protein+4+%28BRD4%29+interacts+with+diverse+kinase+inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span></div><div class="casAuthors">Ember, Stuart W. J.; Zhu, Jin-Yi; Olesen, Sanne H.; Martin, Mathew P.; Becker, Andreas; Berndt, Norbert; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1160-1171</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the bromodomain and extra terminal (BET) family of proteins are essential for the recognition of acetylated lysine (KAc) residues in histones and have emerged as promising drug targets in cancer, inflammation, and contraception research.  In co-crystn. screening campaigns using the first bromodomain of BRD4 (BRD4-1) against kinase inhibitor libraries, we identified and characterized 14 kinase inhibitors (10 distinct chem. scaffolds) as ligands of the KAc binding site.  Among these, the PLK1 inhibitor BI2536 and the JAK2 inhibitor TG101209 displayed strongest inhibitory potential against BRD4 (IC50 = 25 nM and 130 nM, resp.) and high selectivity for BET bromodomains.  Comparative structural anal. revealed markedly different binding modes of kinase hinge-binding scaffolds in the KAc binding site, suggesting that BET proteins are potential off-targets of diverse kinase inhibitors.  Combined, these findings provide a new structural framework for the rational design of next-generation BET-selective and dual-activity BET-kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0X1hjt7cBbVg90H21EOLACvtfcHk0lhIn-Z2p6NI7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D&md5=5ca37df00cdaa0ba81c0cb4a0f70ba5d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DAcetyl-lysine%2520binding%2520site%2520of%2520bromodomain-containing%2520protein%25204%2520%2528BRD4%2529%2520interacts%2520with%2520diverse%2520kinase%2520inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talluri, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topczewski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for the n-terminal bromodomain-and-extra-terminal-family selectivity of a dual kinase-bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9316</span>– <span class="NLM_lpage">9334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01248</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01248" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9316-9334&author=A.+Divakaranauthor=S.+K.+Talluriauthor=A.+M.+Ayoubauthor=N.+K.+Mishraauthor=H.+Cuiauthor=J.+C.+Widenauthor=N.+Berndtauthor=J.-Y.+Zhuauthor=A.+S.+Carlsonauthor=J.+J.+Topczewskiauthor=E.+K.+Schonbrunnauthor=D.+A.+Harkiauthor=W.+C.+K.+Pomerantz&title=Molecular+basis+for+the+n-terminal+bromodomain-and-extra-terminal-family+selectivity+of+a+dual+kinase-bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.8b01248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase-Bromodomain Inhibitor</span></div><div class="casAuthors">Divakaran, Anand; Talluri, Siva K.; Ayoub, Alex M.; Mishra, Neeraj K.; Cui, Huarui; Widen, John C.; Berndt, Norbert; Zhu, Jin-Yi; Carlson, Angela S.; Topczewski, Joseph J.; Schonbrunn, Ernst K.; Harki, Daniel A.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9316-9334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis.  When dysregulated, "reader" proteins become drivers of disease.  In the case of bromodomains, which recognize N-ε-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging.  We describe the >55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase-bromodomain inhibitors.  Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1).  Cellular efficacy was demonstrated via redn. of c-Myc expression, inhibition of NF-κB signaling, and suppression of IL-8 prodn. through potential synergistic inhibition of BRD4(1) and p38α.  These dual inhibitors provide a new scaffold for domain-selective inhibition of BRD4, the aberrant function of which plays a key role in cancer and inflammatory signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZhnL9cPdDi7Vg90H21EOLACvtfcHk0ljB40wCCJ-FXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM&md5=5176ef754a898b3e03fc56e2662ab399</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01248%26sid%3Dliteratum%253Aachs%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DTalluri%26aufirst%3DS.%2BK.%26aulast%3DAyoub%26aufirst%3DA.%2BM.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DCarlson%26aufirst%3DA.%2BS.%26aulast%3DTopczewski%26aufirst%3DJ.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%2BK.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DMolecular%2520basis%2520for%2520the%2520n-terminal%2520bromodomain-and-extra-terminal-family%2520selectivity%2520of%2520a%2520dual%2520kinase-bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9316%26epage%3D9334%26doi%3D10.1021%2Facs.jmedchem.8b01248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erazo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muñoz-Guardiola, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diéguez-Martínez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massefski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offei-Addo, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiBona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becht, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lizcano, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacklow, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Structural and atropisomeric factors governing the selectivity of pyrimido-benzodiazipinones as inhibitors of kinases and bromodomains</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2438</span>– <span class="NLM_lpage">2448</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00638</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00638" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVylsrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2438-2448&author=J.+Wangauthor=T.+Erazoauthor=F.+M.+Fergusonauthor=D.+L.+Buckleyauthor=N.+Gomezauthor=P.+Mu%C3%B1oz-Guardiolaauthor=N.+Di%C3%A9guez-Mart%C3%ADnezauthor=X.+Dengauthor=M.+Haoauthor=W.+Massefskiauthor=O.+Fedorovauthor=N.+K.+Offei-Addoauthor=P.+M.+Parkauthor=L.+Daiauthor=A.+DiBonaauthor=K.+Bechtauthor=N.+D.+Kimauthor=M.+R.+McKeownauthor=J.+M.+Robertsauthor=J.+Zhangauthor=T.+Simauthor=D.+R.+Alessiauthor=J.+E.+Bradnerauthor=J.+M.+Lizcanoauthor=S.+C.+Blacklowauthor=J.+Qiauthor=X.+Xuauthor=N.+S.+Gray&title=Structural+and+atropisomeric+factors+governing+the+selectivity+of+pyrimido-benzodiazipinones+as+inhibitors+of+kinases+and+bromodomains&doi=10.1021%2Facschembio.7b00638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Atropisomeric Factors Governing the Selectivity of Pyrimido-benzodiazipinones as Inhibitors of Kinases and Bromodomains</span></div><div class="casAuthors">Wang, Jinhua; Erazo, Tatiana; Ferguson, Fleur M.; Buckley, Dennis L.; Gomez, Nestor; Munoz-Guardiola, Pau; Dieguez-Martinez, Nora; Deng, Xianming; Hao, Mingfeng; Massefski, Walter; Fedorov, Oleg; Offei-Addo, Nana Kwaku; Park, Paul M.; Dai, Lingling; DiBona, Amy; Becht, Kelly; Kim, Nam Doo; McKeown, Michael R.; Roberts, Justin M.; Zhang, Jinwei; Sim, Taebo; Alessi, Dario R.; Bradner, James E.; Lizcano, Jose M.; Blacklow, Stephen C.; Qi, Jun; Xu, Xiang; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2438-2448</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains have been pursued intensively over the past several years as emerging targets for the development of anticancer and anti-inflammatory agents.  It has recently been shown that some kinase inhibitors are able to potently inhibit the bromodomains of BRD4.  The clin. activities of PLK inhibitor BI-2536 and JAK2-FLT3 inhibitor TG101348 have been attributed to this unexpected polypharmacol., indicating that dual-kinase/bromodomain activity may be advantageous in a therapeutic context.  However, for target validation and biol. investigation, a more selective target profile is desired.  Here, we report that benzo[e]pyrimido-[5,4-b]diazepine-6(11H)-ones, versatile ATP-site directed kinase pharmacophores utilized in the development of inhibitors of multiple kinases, including several previously reported kinase chem. probes, are also capable of exhibiting potent BRD4-dependent pharmacol.  Using a dual kinase-bromodomain inhibitor of the kinase domains of ERK5 and LRRK2, and the bromodomain of BRD4 as a case study, we define the structure-activity relationships required to achieve dual kinase/BRD4 activity, as well as how to direct selectivity toward inhibition of either ERK5 or BRD4.  This effort resulted in identification of one of the first reported kinase-selective chem. probes for ERK5 (JWG-071), a BET selective inhibitor with 1 μM BRD4 IC50 (JWG-115), and addnl. inhibitors with rationally designed polypharmacol. (JWG-047, JWG-069).  Co-crystallog. of seven representative inhibitors with the first bromodomain of BRD4 demonstrate that distinct atropisomeric conformers recognize the kinase ATP-site and the BRD4 acetyl lysine binding site, conformational preferences supported by rigid docking studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcdvUrcfGdq7Vg90H21EOLACvtfcHk0ljB40wCCJ-FXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVylsrvN&md5=bebe87da2f2808ad5aec5b3098304425</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00638%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DErazo%26aufirst%3DT.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGomez%26aufirst%3DN.%26aulast%3DMu%25C3%25B1oz-Guardiola%26aufirst%3DP.%26aulast%3DDi%25C3%25A9guez-Mart%25C3%25ADnez%26aufirst%3DN.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DHao%26aufirst%3DM.%26aulast%3DMassefski%26aufirst%3DW.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DOffei-Addo%26aufirst%3DN.%2BK.%26aulast%3DPark%26aufirst%3DP.%2BM.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DDiBona%26aufirst%3DA.%26aulast%3DBecht%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLizcano%26aufirst%3DJ.%2BM.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DStructural%2520and%2520atropisomeric%2520factors%2520governing%2520the%2520selectivity%2520of%2520pyrimido-benzodiazipinones%2520as%2520inhibitors%2520of%2520kinases%2520and%2520bromodomains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D2438%26epage%3D2448%26doi%3D10.1021%2Facschembio.7b00638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watts, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strebhardt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellenie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Designing dual inhibitors of anaplastic lymphoma kinase (alk) and bromodomain-4 (brd4) by tuning kinase selectivity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2618</span>– <span class="NLM_lpage">2637</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01947</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01947" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2618-2637&author=E.+Wattsauthor=D.+Heidenreichauthor=E.+Tuckerauthor=M.+Raabauthor=K.+Strebhardtauthor=L.+Cheslerauthor=S.+Knappauthor=B.+Bellenieauthor=S.+Hoelder&title=Designing+dual+inhibitors+of+anaplastic+lymphoma+kinase+%28alk%29+and+bromodomain-4+%28brd4%29+by+tuning+kinase+selectivity&doi=10.1021%2Facs.jmedchem.8b01947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Designing Dual Inhibitors of Anaplastic Lymphoma Kinase (ALK) and Bromodomain-4 (BRD4) by Tuning Kinase Selectivity</span></div><div class="casAuthors">Watts, Ellen; Heidenreich, David; Tucker, Elizabeth; Raab, Monika; Strebhardt, Klaus; Chesler, Louis; Knapp, Stefan; Bellenie, Benjamin; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2618-2637</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Concomitant inhibition of anaplastic lymphoma kinase (ALK) and bromodomain-4 (BRD4) is a potential therapeutic strategy for targeting two key oncogenic drivers that co-segregate in a significant fraction of high-risk neuroblastoma patients, mutation of ALK and amplification of MYCN.  Starting from known dual polo-like kinase (PLK)-1-BRD4 inhibitor BI-2536, we employed structure-based design to redesign this series toward compds. with a dual ALK-BRD4 profile.  These efforts led to compd. (R)-2-((2-ethoxy-4-(1-methylpiperidin-4-yl)phenyl)amino)-7-ethyl-5-methyl-8-((4-methylthiophen-2-yl)methyl)-7,8-dihydropteridin-6(5H)-one (16k) demonstrating improved ALK activity and significantly reduced PLK-1 activity, while maintaining BRD4 activity and overall kinome selectivity.  We demonstrate the compds.' on-target engagement with ALK and BRD4 in cells as well as favorable broad kinase and bromodomain selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop6qx6PSj2KLVg90H21EOLACvtfcHk0ljB40wCCJ-FXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D&md5=e00fdf8eb20acc2ef1c8777b2afb750d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01947%26sid%3Dliteratum%253Aachs%26aulast%3DWatts%26aufirst%3DE.%26aulast%3DHeidenreich%26aufirst%3DD.%26aulast%3DTucker%26aufirst%3DE.%26aulast%3DRaab%26aufirst%3DM.%26aulast%3DStrebhardt%26aufirst%3DK.%26aulast%3DChesler%26aufirst%3DL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBellenie%26aufirst%3DB.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDesigning%2520dual%2520inhibitors%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528alk%2529%2520and%2520bromodomain-4%2520%2528brd4%2529%2520by%2520tuning%2520kinase%2520selectivity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2618%26epage%3D2637%26doi%3D10.1021%2Facs.jmedchem.8b01947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshioka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fountain, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheenstra, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strovel, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span> <span> </span><span class="NLM_article-title">BRD4 structure-activity relationships of dual plk1 kinase/brd4 bromodomain inhibitor bi-2536</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">764</span>– <span class="NLM_lpage">769</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00084</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00084" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=764-769&author=L.+Chenauthor=J.+L.+Yapauthor=M.+Yoshiokaauthor=M.+E.+Lanningauthor=R.+N.+Fountainauthor=M.+Rajeauthor=J.+A.+Scheenstraauthor=J.+W.+Strovelauthor=S.+Fletcher&title=BRD4+structure-activity+relationships+of+dual+plk1+kinase%2Fbrd4+bromodomain+inhibitor+bi-2536&doi=10.1021%2Facsmedchemlett.5b00084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Structure-Activity Relationships of Dual PLK1 Kinase/BRD4 Bromodomain Inhibitor BI-2536</span></div><div class="casAuthors">Chen, Lijia; Yap, Jeremy L.; Yoshioka, Makoto; Lanning, Maryanna E.; Fountain, Rachel N.; Raje, Mithun; Scheenstra, Jacob A.; Strovel, Jeffrey W.; Fletcher, Steven</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">764-769</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused library of analogs of the dual PLK1 kinase/BRD4 bromodomain inhibitor BI-2536 was prepd. and then analyzed for BRD4 and PLK1 inhibitory activities.  Particularly, replacement of the cyclopentyl group with a 3-bromobenzyl moiety afforded the most potent BRD4 inhibitor of the series (39j) with a Ki = 8.7 nM, which was equipotent against PLK1.  The superior affinity of 39j over the parental compd. to BRD4 possibly derives from improved interactions with the WPF shelf.  Meanwhile, substitution of the pyrimidine NH with an oxygen atom reversed the PLK1/BRD4 selectivity to convert BI-2536 into a BRD4-selective inhibitor, likely owing to the loss of a crit. hydrogen bond in PLK1.  We believe further fine-tuning will furnish a BRD4 "magic bullet" or an even more potent PLK1/BRD4 dual inhibitor toward the expansion and improved efficacy of the chemotherapy arsenal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5UnNSmf1L17Vg90H21EOLACvtfcHk0ljPrDG5YknH_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D&md5=f39070e652bca20c03a3bc5fe070a7fc</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00084%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYap%26aufirst%3DJ.%2BL.%26aulast%3DYoshioka%26aufirst%3DM.%26aulast%3DLanning%26aufirst%3DM.%2BE.%26aulast%3DFountain%26aufirst%3DR.%2BN.%26aulast%3DRaje%26aufirst%3DM.%26aulast%3DScheenstra%26aufirst%3DJ.%2BA.%26aulast%3DStrovel%26aufirst%3DJ.%2BW.%26aulast%3DFletcher%26aufirst%3DS.%26atitle%3DBRD4%2520structure-activity%2520relationships%2520of%2520dual%2520plk1%2520kinase%252Fbrd4%2520bromodomain%2520inhibitor%2520bi-2536%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D764%26epage%3D769%26doi%3D10.1021%2Facsmedchemlett.5b00084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0ljPrDG5YknH_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klug-Mcleod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunney, E. A.</span></span> <span> </span><span class="NLM_article-title">Kinase hinge binding scaffolds and their hydrogen bond patterns</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6520</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.bmc.2015.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=26358279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVentr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=6520-6527&author=L.+Xingauthor=J.+Klug-Mcleodauthor=B.+Raiauthor=E.+A.+Lunney&title=Kinase+hinge+binding+scaffolds+and+their+hydrogen+bond+patterns&doi=10.1016%2Fj.bmc.2015.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase hinge binding scaffolds and their hydrogen bond patterns</span></div><div class="casAuthors">Xing, Li; Klug-Mcleod, Jacquelyn; Rai, Brajesh; Lunney, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6520-6527</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein kinases constitute a major class of intracellular signaling mols., and describe some of the most prominent drug targets.  Kinase inhibitors commonly employ small chem. scaffolds that form hydrogen bonds with the kinase hinge residues connecting the N- and C-terminal lobes of the catalytic domain.  In general the satisfied hydrogen bonds are required for potent inhibition, therefore constituting a conserved feature in the majority of inhibitor-kinase interactions.  From systematically analyzing the kinase scaffolds extd. from Pfizer crystal structure database (CSDb) the authors recognize that large no. of kinase inhibitors of diverse chem. structures are derived from a relatively small no. of common scaffolds.  Depending on specific substitution patterns, scaffolds may demonstrate versatile binding capacities to interact with kinase hinge.  Afforded by thousands of ligand-protein binary complexes, the hinge hydrogen bond patterns were analyzed with a focus on their three-dimensional configurations.  Most of the compds. engage H6 NH for hinge recognition.  Dual hydrogen bonds are commonly obsd. with addnl. recruitment of H4 CO upstream and/or H6 CO downstream.  Triple hydrogen bonds accounts for small no. of binary complexes.  An unusual hydrogen bond with a non-canonical H5 conformation is obsd., requiring a peptide bond flip by a glycine residue at the H6 position.  Addnl. hydrogen bonds to kinase hinge do not necessarily correlate with an increase in potency; conversely they appear to compromise kinase selectivity.  Such learnings could enhance the prospect of successful therapy design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRC-xiTiNvELVg90H21EOLACvtfcHk0ljPrDG5YknH_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVentr3L&md5=048b4b99628c6e1ae273b00a47ab2377</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DXing%26aufirst%3DL.%26aulast%3DKlug-Mcleod%26aufirst%3DJ.%26aulast%3DRai%26aufirst%3DB.%26aulast%3DLunney%26aufirst%3DE.%2BA.%26atitle%3DKinase%2520hinge%2520binding%2520scaffolds%2520and%2520their%2520hydrogen%2520bond%2520patterns%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D6520%26epage%3D6527%26doi%3D10.1016%2Fj.bmc.2015.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosief, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhawan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthengi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and development of potent and selective dual bromodomain 4 (brd4)/polo-like kinase 1 (plk1) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7785</span>– <span class="NLM_lpage">7795</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00765</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00765" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7785-7795&author=S.+Liuauthor=H.+O.+Yosiefauthor=L.+Daiauthor=H.+Huangauthor=G.+Dhawanauthor=X.+Zhangauthor=A.+M.+Muthengiauthor=J.+Robertsauthor=D.+L.+Buckleyauthor=J.+A.+Perryauthor=L.+Wuauthor=J.+E.+Bradnerauthor=J.+Qiauthor=W.+Zhang&title=Structure-guided+design+and+development+of+potent+and+selective+dual+bromodomain+4+%28brd4%29%2Fpolo-like+kinase+1+%28plk1%29+inhibitors&doi=10.1021%2Facs.jmedchem.8b00765"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors</span></div><div class="casAuthors">Liu, Shuai; Yosief, Hailemichael O.; Dai, Lingling; Huang, He; Dhawan, Gagan; Zhang, Xiaofeng; Muthengi, Alex M.; Roberts, Justin; Buckley, Dennis L.; Perry, Jennifer A.; Wu, Lei; Bradner, James E.; Qi, Jun; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7785-7795</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single mol. could lead to the development of an effective therapeutic strategy for a variety of diseases in which PLK1 and BRD4 are implicated.  Compd. 23 has been found to be a potent dual kinase-bromodomain inhibitor (BRD4-BD1 IC50 = 28 nM, PLK1 IC50 = 40 nM).  Compd. 6 was found to be the most selective PLK1 inhibitor over BRD4 in our series (BRD4-BD1 IC50 = 2579 nM, PLK1 IC50 = 9.9 nM).  Mol. docking studies with 23 and BRD4-BD1/PLK1 as well as with 6 corroborate the biochem. assay results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUL2G-DnFzU7Vg90H21EOLACvtfcHk0lhuOymYZDuBWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L&md5=e34a58694f0827d33d31516200e2c0d8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00765%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYosief%26aufirst%3DH.%2BO.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DDhawan%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMuthengi%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DJ.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DStructure-guided%2520design%2520and%2520development%2520of%2520potent%2520and%2520selective%2520dual%2520bromodomain%25204%2520%2528brd4%2529%252Fpolo-like%2520kinase%25201%2520%2528plk1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7785%26epage%3D7795%26doi%3D10.1021%2Facs.jmedchem.8b00765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olcina, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Optimization of 3,5-dimethylisoxazole derivatives as potent bromodomain ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3217</span>– <span class="NLM_lpage">3227</span>, <span class="refDoi"> DOI: 10.1021/jm301588r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301588r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlekuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3217-3227&author=D.+S.+Hewingsauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=S.+Martinauthor=S.+Picaudauthor=A.+Tumberauthor=C.+Wellsauthor=M.+M.+Olcinaauthor=K.+Freemanauthor=A.+Gillauthor=A.+J.+Ritchieauthor=D.+W.+Sheppardauthor=A.+J.+Russellauthor=E.+M.+Hammondauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=Optimization+of+3%2C5-dimethylisoxazole+derivatives+as+potent+bromodomain+ligands&doi=10.1021%2Fjm301588r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 3,5-Dimethylisoxazole Derivatives as Potent Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Fedorov, Oleg; Filippakopoulos, Panagis; Martin, Sarah; Picaud, Sarah; Tumber, Anthony; Wells, Christopher; Olcina, Monica M.; Freeman, Katherine; Gill, Andrew; Ritchie, Alison J.; Sheppard, David W.; Russell, Angela J.; Hammond, Ester M.; Knapp, Stefan; Brennan, Paul E.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3217-3227</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">(Isoxazolylphenyl)arylmethanols such as I (R = H; R1 = HO; R2 = H; R3 = Ph), I (R = HO; R1 = H; R2 = H; R3 = Ph), and I (R = H, HO; R1 = HO, H; R2 = Et; R3 = Ph, 3-FC6H4, 4-FC6H4, 3-ClC6H4, 4-ClC6H4) were prepd. as inhibitors of the bromodomain and extra terminal domain (BET) bromodomain family member BRD4(1) for potential use as antitumor agents.  The structures of I (R = H; R1 = HO; R2 = H; R3 = Ph) and I (R = HO; R1 = H; R2 = H; R3 = Ph) bound to human BRD4(1) were detd. by X-ray crystallog. to det. features useful for selective binding to BRD4(1).  I (R = H, HO; R1 = HO, H; R2 = H, MeCO; R3 = Ph) and I (R = H, HO; R1 = HO, H; R2 = Et; R3 = 3-ClC6H4) inhibited a human acute myeloid leukemia cell line in vitro with 3-4 fold reduced inhibition and reduced toxicity in other human cells compared to a known BRD4(1) inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVEQ5OOJGC8bVg90H21EOLACvtfcHk0lhuOymYZDuBWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlekuro%253D&md5=bb4890291db8c3e8d404a75dd9dcbb87</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm301588r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301588r%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DOlcina%26aufirst%3DM.%2BM.%26aulast%3DFreeman%26aufirst%3DK.%26aulast%3DGill%26aufirst%3DA.%26aulast%3DRitchie%26aufirst%3DA.%2BJ.%26aulast%3DSheppard%26aufirst%3DD.%2BW.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DHammond%26aufirst%3DE.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DOptimization%2520of%25203%252C5-dimethylisoxazole%2520derivatives%2520as%2520potent%2520bromodomain%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3217%26epage%3D3227%26doi%3D10.1021%2Fjm301588r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttarkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuppusetty, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">3,5-dimethylisoxazoles act as acetyl-lysine-mimetic bromodomain ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6761</span>– <span class="NLM_lpage">6770</span>, <span class="refDoi"> DOI: 10.1021/jm200640v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200640v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6761-6770&author=D.+S.+Hewingsauthor=M.+Wangauthor=M.+Philpottauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=C.+Vuppusettyauthor=B.+Marsdenauthor=S.+Knappauthor=S.+J.+Conwayauthor=T.+D.+Heightman&title=3%2C5-dimethylisoxazoles+act+as+acetyl-lysine-mimetic+bromodomain+ligands&doi=10.1021%2Fjm200640v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Dimethylisoxazoles Act As Acetyl-lysine-mimetic Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Wang, Minghua; Philpott, Martin; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Vuppusetty, Chaitanya; Marsden, Brian; Knapp, Stefan; Conway, Stuart J.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6761-6770</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone-lysine acetylation is a vital chromatin post-translational modification involved in the epigenetic regulation of gene transcription.  Bromodomains bind acetylated lysines, acting as readers of the histone-acetylation code.  Competitive inhibitors of this interaction have antiproliferative and anti-inflammatory properties.  With 57 distinct bromodomains known, the discovery of subtype-selective inhibitors of the histone-bromodomain interaction is of great importance.  We have identified the 3,5-dimethylisoxazole moiety as a novel acetyl-lysine bioisostere, which displaces acetylated histone-mimicking peptides from bromodomains.  Using X-ray crystallog. anal., we have detd. the interactions responsible for the activity and selectivity of 4-substituted 3,5-dimethylisoxazoles against a selection of phylogenetically diverse bromodomains.  By exploiting these interactions, we have developed compd. 4d, which has IC50 values of <5 μM for the bromodomain-contg. proteins BRD2(1) and BRD4(1).  These compds. are promising leads for the further development of selective probes for the bromodomain and extra C-terminal domain (BET) family and CREBBP bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEzK4xMvJhXrVg90H21EOLACvtfcHk0lgeAShXM9mLQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO&md5=7d6f9eff2735c8a7189dec4fb5c5d13d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm200640v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200640v%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DVuppusetty%26aufirst%3DC.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3D3%252C5-dimethylisoxazoles%2520act%2520as%2520acetyl-lysine-mimetic%2520bromodomain%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6761%26epage%3D6770%26doi%3D10.1021%2Fjm200640v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, C.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damaneh, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huan, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery and development of a series of potent and selective bromodomain and extra-terminal protein inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8642</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01094</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01094" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslegsbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8642-8663&author=J.+Huauthor=C.-Q.+Tianauthor=M.+S.+Damanehauthor=Y.+Liauthor=D.+Caoauthor=K.+Lvauthor=T.+Yuauthor=T.+Mengauthor=D.+Chenauthor=X.+Wangauthor=L.+Chenauthor=J.+Liauthor=S.-S.+Songauthor=X.-J.+Huanauthor=L.+Qinauthor=J.+Shenauthor=Y.-Q.+Wangauthor=Z.-H.+Miaoauthor=B.+Xiong&title=Structure-based+discovery+and+development+of+a+series+of+potent+and+selective+bromodomain+and+extra-terminal+protein+inhibitors&doi=10.1021%2Facs.jmedchem.9b01094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery and Development of a Series of Potent and Selective Bromodomain and Extra-Terminal Protein Inhibitors</span></div><div class="casAuthors">Hu, Jianping; Tian, Chang-Qing; Damaneh, Mohammadali Soleimani; Li, Yanlian; Cao, Danyan; Lv, Kaikai; Yu, Ting; Meng, Tao; Chen, Danqi; Wang, Xin; Chen, Lin; Li, Jian; Song, Shan-Shan; Huan, Xia-Juan; Qin, Lihuai; Shen, Jingkang; Wang, Ying-Qing; Miao, Ze-Hong; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8642-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4 has recently emerged as a promising drug target.  Therefore, identifying novel inhibitors with distinct properties could enrich their use in anticancer treatment.  Guided by the cocrystal structure of hit compd. 4 harboring a five-membered-ring linker motif, we quickly identified lead compd. 7, which exhibited good antitumor effects in an MM.1S xenograft model by oral administration.  Encouraged by its high potency and interesting scaffold, we performed further lead optimization to generate a novel potent series of bromodomain and extra-terminal (BET) inhibitors with a (1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1H)-one structure.  Among them, compd. 19 was found to have the best balance of activity, stability, and antitumor efficacy.  After confirming its low brain penetration, we conducted comprehensive preclin. studies, including a multiple-species pharmacokinetics profile, extensive cellular mechanism studies, hERG assay, and in vivo antitumor growth effect testing, and we found that compd. 19 is a potential BET protein drug candidate for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGormQku3ZAy97Vg90H21EOLACvtfcHk0lgeAShXM9mLQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslegsbvK&md5=af9fa23078f8280cc36d3c097d6fbf34</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01094%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DC.-Q.%26aulast%3DDamaneh%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DLv%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DT.%26aulast%3DMeng%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DS.-S.%26aulast%3DHuan%26aufirst%3DX.-J.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.-Q.%26aulast%3DMiao%26aufirst%3DZ.-H.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DStructure-based%2520discovery%2520and%2520development%2520of%2520a%2520series%2520of%2520potent%2520and%2520selective%2520bromodomain%2520and%2520extra-terminal%2520protein%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8642%26epage%3D8663%26doi%3D10.1021%2Facs.jmedchem.9b01094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damaneh, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Structure-based optimization of a series of selective BET inhibitors containing aniline or indoline groups</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.02.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.ejmech.2018.02.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=29525435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktlylsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2018&pages=156-175&author=J.+Huauthor=Y.+Wangauthor=Y.+Liauthor=D.+Caoauthor=L.+Xuauthor=S.+Songauthor=M.+S.+Damanehauthor=J.+Liauthor=Y.+Chenauthor=X.+Wangauthor=L.+Chenauthor=J.+Shenauthor=Z.+Miaoauthor=B.+Xiong&title=Structure-based+optimization+of+a+series+of+selective+BET+inhibitors+containing+aniline+or+indoline+groups&doi=10.1016%2Fj.ejmech.2018.02.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based optimization of a series of selective BET inhibitors containing aniline or indoline groups</span></div><div class="casAuthors">Hu, Jianping; Wang, Yingqing; Li, Yanlian; Cao, Danyan; Xu, Lin; Song, ShanShan; Damaneh, Mohammadali Soleimani; Li, Jian; Chen, Yuelei; Wang, Xin; Chen, Lin; Shen, Jingkang; Miao, Zehong; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">156-175</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recently, several kinase inhibitors were found to also inhibit bromodomains, providing a new strategy for the discovery of bromodomain inhibitors.  Along this line, starting from PLK1-BRD4 dual inhibitor BI-2536, we discovered a new series of dihydroquinoxalin-2(1H)-one with aniline and indoline WPF binders as selective BRD4 inhibitors.  They showed better BRD4-BD1 potency and negligible PLK1 kinase activity comparing with BI-2536.  Addnl., dihydroquinoxalin-2(1H)-ones contg. indoline group showed profound activities in mol. and cellular based assays.  Throughout the study, compds. 9, 28 and 37 showed significant inhibitory activity for c-Myc or PD-L1 protein expression and mRNA transcription both at concn. of 0.2 and 1 μM.  Compd. 9 was found possessing the best balance of binding affinity, in vitro metabolic stability and in vivo pharmacokinetic properties.  Therefore, it was selected for in vivo pharmacol. study.  By using MM.1S cell derived xenograft model, we confirmed compd. 9 showed comparable in vivo tumor inhibition to phase II investigation drug I-BET762, which, together with the novel WPF binder, further indicated the utility of this series of BRD4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovKjmtiqOa9rVg90H21EOLACvtfcHk0lhJot9WHi8WOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktlylsLw%253D&md5=5d3ad7e741a140306d42ae2463dd667e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.02.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.02.070%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DDamaneh%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DStructure-based%2520optimization%2520of%2520a%2520series%2520of%2520selective%2520BET%2520inhibitors%2520containing%2520aniline%2520or%2520indoline%2520groups%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D150%26spage%3D156%26epage%3D175%26doi%3D10.1016%2Fj.ejmech.2018.02.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damaneh, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a series of dihydroquinoxalin-2(1H)-ones as selective BET inhibitors from a dual PLK1-BRD4 inhibitor</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">176</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.05.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.ejmech.2017.05.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=28586718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWmt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2017&pages=176-195&author=J.+Huauthor=Y.+Wangauthor=Y.+Liauthor=L.+Xuauthor=D.+Caoauthor=S.+Songauthor=M.+S.+Damanehauthor=X.+Wangauthor=T.+Mengauthor=Y.-L.+Chenauthor=J.+Shenauthor=Z.+Miaoauthor=B.+Xiong&title=Discovery+of+a+series+of+dihydroquinoxalin-2%281H%29-ones+as+selective+BET+inhibitors+from+a+dual+PLK1-BRD4+inhibitor&doi=10.1016%2Fj.ejmech.2017.05.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a series of dihydroquinoxalin-2(1H)-ones as selective BET inhibitors from a dual PLK1-BRD4 inhibitor</span></div><div class="casAuthors">Hu, Jianping; Wang, Yingqing; Li, Yanlian; Xu, Lin; Cao, Danyan; Song, Shan Shan; Damaneh, Mohammadali Soleimani; Wang, Xin; Meng, Tao; Chen, Yue-Lei; Shen, Jingkang; Miao, Zehong; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">176-195</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recent years have seen much effort to discover new chemotypes of BRD4 inhibitors.  Some kinase inhibitors are potent bromodomain inhibitors, esp. the PLK1 inhibitor BI-2536 and the JAK2 inhibitor TG101209, which can bind to BRD4 with IC50 values of 0.025 μM and 0.13 μM, resp.  Although the concept of dual inhibition is intriguing, selective BRD4 inhibitors are preferred as they may diminish off-target effects and provide more flexibility in anticancer drug combination therapy.  Inspired by BI-2536, the authors designed and prepd. a series of dihydroquinoxalin-2(1H)-one derivs. as selective bromodomain inhibitors.  The authors found compd. (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one had slightly higher activity than (+)-JQ1 in the fluorescence anisotropy assay and potent antiproliferative cellular activity in the MM.1S cell line.  The authors have successfully solved the cocrystal structure of (R)-4-cyclopropyl-1,3-dimethyl-6-(((S)-1-(p-tolyl)ethyl) amino)-3,4-dihydroquinoxalin-2(1H)-one in complex with BRD4-BD1, providing a solid structural basis for the binding mode of compds. of this series.  Compd. (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one exhibited high selectivity over most non-BET subfamily members and did not show bioactivity towards the PLK1 kinase at 10 or 1 μM.  From in vivo studies, compd. (R)-4-Cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one demonstrated a good PK profile, and the results from in vivo pharmacol. studies clearly showed the efficacy of (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one in the mouse MM.1S xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8GQjGV-VP7LVg90H21EOLACvtfcHk0lhJot9WHi8WOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWmt73I&md5=af59e5bd57adeede512b022e3641f4f2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.05.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.05.049%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DDamaneh%26aufirst%3DM.%2BS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMeng%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.-L.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520series%2520of%2520dihydroquinoxalin-2%25281H%2529-ones%2520as%2520selective%2520BET%2520inhibitors%2520from%2520a%2520dual%2520PLK1-BRD4%2520inhibitor%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D137%26spage%3D176%26epage%3D195%26doi%3D10.1016%2Fj.ejmech.2017.05.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4927</span>– <span class="NLM_lpage">4939</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4927-4939&author=X.+Ranauthor=Y.+Zhaoauthor=L.+Liuauthor=L.+Baiauthor=C.-Y.+Yangauthor=B.+Zhouauthor=J.+L.+Meagherauthor=K.+Chinnaswamyauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-Based+Design+of+%CE%B3-Carboline+Analogues+as+Potent+and+Specific+BET+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span></div><div class="casAuthors">Ran, Xu; Zhao, Yujun; Liu, Liu; Bai, Longchuan; Yang, Chao-Yie; Zhou, Bing; Meagher, Jennifer L.; Chinnaswamy, Krishnapriya; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4927-4939</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors of bromodomain and extra terminal proteins (BET), including BRD2, BRD3, and BRD4 proteins have therapeutic potential for the treatment of human cancers and other diseases and conditions.  In this paper, the authors report the design, synthesis, and evaluation of γ-carboline-contg. compds. as a new class of small-mol. BET inhibitors.  The most potent inhibitor I (RX-37) obtained from this study binds to BET bromodomain proteins (BRD2, BRD3, and BRD4) with Ki values of 3.2-24.7 nM and demonstrates high selectivity over other non-BET bromodomain-contg. proteins.  Compd. I potently and selectively inhibits cell growth in human acute leukemia cell lines harboring the rearranged mixed lineage leukemia 1 gene.  The authors have detd. a cocrystal structure of I in complex with BRD4 BD2 at 1.4 Å resoln., which provides a solid structural basis for the compd.'s high binding affinity and for its further structure-based optimization.  Compd. I represents a promising lead compd. for the development of a new class of therapeutics for the treatment of human cancer and other conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv-VCfWLBWYLVg90H21EOLACvtfcHk0lhJot9WHi8WOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM&md5=e6e150169812e439c1d7d91416eb3ee9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00613%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-Based%2520Design%2520of%2520%25CE%25B3-Carboline%2520Analogues%2520as%2520Potent%2520and%2520Specific%2520BET%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4927%26epage%3D4939%26doi%3D10.1021%2Facs.jmedchem.5b00613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singleton, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of small-molecule ligands for the CBP/p300 bromodomains</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">9308</span>– <span class="NLM_lpage">9319</span>, <span class="refDoi"> DOI: 10.1021/ja412434f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja412434f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=9308-9319&author=D.+A.+Hayauthor=O.+Fedorovauthor=S.+Martinauthor=D.+C.+Singletonauthor=C.+Tallantauthor=C.+Wellsauthor=S.+Picaudauthor=M.+Philpottauthor=O.+P.+Monteiroauthor=C.+M.+Rogersauthor=S.+J.+Conwayauthor=T.+P.+C.+Rooneyauthor=A.+Tumberauthor=C.+Yappauthor=P.+Filippakopoulosauthor=M.+E.+Bunnageauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Discovery+and+optimization+of+small-molecule+ligands+for+the+CBP%2Fp300+bromodomains&doi=10.1021%2Fja412434f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Small-Molecule Ligands for the CBP/p300 Bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Fedorov, Oleg; Martin, Sarah; Singleton, Dean C.; Tallant, Cynthia; Wells, Christopher; Picaud, Sarah; Philpott, Martin; Monteiro, Octovia P.; Rogers, Catherine M.; Conway, Stuart J.; Rooney, Timothy P. C.; Tumber, Anthony; Yapp, Clarence; Filippakopoulos, Panagis; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">9308-9319</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors that target bromodomains outside of the bromodomain and extra-terminal (BET) sub-family are lacking.  Here, the authors describe highly potent and selective ligands for the bromodomain module of the human lysine acetyl transferase CBP/p300, developed from a series of 5-isoxazolyl-benzimidazoles.  The starting point was a fragment hit, which was optimized into a more potent and selective lead using parallel synthesis employing Suzuki couplings, benzimidazole-forming reactions, and reductive aminations.  The selectivity of the lead compd. against other bromodomain family members was investigated using a thermal stability assay, which revealed some inhibition of the structurally related BET family members.  To address the BET selectivity issue, x-ray crystal structures of the lead compd. bound to the CREB binding protein (CBP) and the first bromodomain of BRD4 (BRD4(1)) were used to guide the design of more selective compds.  The crystal structures obtained revealed two distinct binding modes.  By varying the aryl substitution pattern and developing conformationally constrained analogs, selectivity for CBP over BRD4(1) was increased.  The optimized compd. is highly potent (Kd = 21 nM) and selective, displaying 40-fold selectivity over BRD4(1).  Cellular activity was demonstrated using fluorescence recovery after photo-bleaching (FRAP) and a p53 reporter assay.  The optimized compds. are cell-active and have nanomolar affinity for CBP/p300; therefore, they should be useful in studies investigating the biol. roles of CBP and p300 and to validate the CBP and p300 bromodomains as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFzgs1DD4QZLVg90H21EOLACvtfcHk0lhJot9WHi8WOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP&md5=e5da3e96a4d234453a481c7ee5e788f2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja412434f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja412434f%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520small-molecule%2520ligands%2520for%2520the%2520CBP%252Fp300%2520bromodomains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D9308%26epage%3D9319%26doi%3D10.1021%2Fja412434f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flick, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabudhe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pletcher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of highly selective inhibitors of the creb binding protein bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5349</span>– <span class="NLM_lpage">5363</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5349-5363&author=R.+A.+Dennyauthor=A.+C.+Flickauthor=J.+Coeauthor=J.+Langilleauthor=A.+Basakauthor=S.+Liuauthor=I.+Stockauthor=P.+Sahasrabudheauthor=P.+Boninauthor=D.+A.+Hayauthor=P.+E.+Brennanauthor=M.+Pletcherauthor=L.+H.+Jonesauthor=E.+L.+P.+Chekler&title=Structure-based+design+of+highly+selective+inhibitors+of+the+creb+binding+protein+bromodomain&doi=10.1021%2Facs.jmedchem.6b01839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Highly Selective Inhibitors of the CREB Binding Protein Bromodomain</span></div><div class="casAuthors">Denny, R. Aldrin; Flick, Andrew C.; Coe, Jotham; Langille, Jonathan; Basak, Arindrajit; Liu, Shenping; Stock, Ingrid; Sahasrabudhe, Parag; Bonin, Paul; Hay, Duncan A.; Brennan, Paul E.; Pletcher, Mathew; Jones, Lyn H.; Chekler, Eugene L. Piatnitski</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5349-5363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. probes are required for preclin. target validation to interrogate novel biol. targets and pathways.  Selective inhibitors of the CREB binding protein (CREBBP)/EP300 bromodomains are required to facilitate the elucidation of biol. assocd. with these important epigenetic targets.  Medicinal chem. optimization that paid particular attention to physiochem. properties delivered chem. probes with desirable potency, selectivity, and permeability attributes.  An important feature of the optimization process was the successful application of rational structure-based drug design to address bromodomain selectivity issues (particularly against the structurally related BRD4 protein).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLWkqF-qAmdbVg90H21EOLACvtfcHk0ljIYKaphqvL1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D&md5=23676a973abba59088dedd5502877105</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01839%26sid%3Dliteratum%253Aachs%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DFlick%26aufirst%3DA.%2BC.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DBasak%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DStock%26aufirst%3DI.%26aulast%3DSahasrabudhe%26aufirst%3DP.%26aulast%3DBonin%26aufirst%3DP.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DPletcher%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26atitle%3DStructure-based%2520design%2520of%2520highly%2520selective%2520inhibitors%2520of%2520the%2520creb%2520binding%2520protein%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5349%26epage%3D5363%26doi%3D10.1021%2Facs.jmedchem.6b01839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampf, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of cf53 as a potent and orally bioavailable bromodomain and extra-terminal (bet) bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6110</span>– <span class="NLM_lpage">6120</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlahu73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6110-6120&author=Y.+Zhaoauthor=B.+Zhouauthor=L.+Baiauthor=L.+Liuauthor=C.-Y.+Yangauthor=J.+L.+Meagherauthor=J.+A.+Stuckeyauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=X.+Ranauthor=A.+Aguilarauthor=Y.+Huauthor=J.+W.+Kampfauthor=X.+Liauthor=T.+Zhaoauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Structure-based+discovery+of+cf53+as+a+potent+and+orally+bioavailable+bromodomain+and+extra-terminal+%28bet%29+bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.8b00483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of CF53 as a Potent and Orally Bioavailable Bromodomain and Extra-Terminal (BET) Bromodomain Inhibitor</span></div><div class="casAuthors">Zhao, Yujun; Zhou, Bing; Bai, Longchuan; Liu, Liu; Yang, Chao-Yie; Meagher, Jennifer L.; Stuckey, Jeanne A.; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Ran, Xu; Aguilar, Angelo; Hu, Yang; Kampf, Jeff W.; Li, Xiaoqin; Zhao, Ting; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6110-6120</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the structure-based discovery of CF53 (28) as a highly potent and orally active inhibitor of bromodomain and extra-terminal (BET) proteins.  By the incorporation of a NH-pyrazole group into the 9H-pyrimido[4,5-b]indole core, we identified a series of compds. that bind to BRD4 BD1 protein with Ki values of <1 nM and achieve low nanomolar potencies in the cell growth inhibition of leukemia and breast cancer cells.  The most-promising compd., CF53, possesses excellent oral pharmacokinetic properties and achieves significant antitumor activity in both triple-neg. breast cancer and acute leukemia xenograft models in mice.  Detn. of the co-crystal structure of CF53 with the BRD4 BD1 protein provides a structural basis for its high binding affinity to BET proteins.  CF53 is very selective over non-BET bromodomain-contg. proteins.  These data establish CF53 as a potent, selective, and orally active BET inhibitor, which warrants further evaluation for advanced preclin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprmt-PPWmeNrVg90H21EOLACvtfcHk0ljIYKaphqvL1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlahu73J&md5=86ca0c87da4cf2dc218122582f65ba7a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00483%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DRan%26aufirst%3DX.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DKampf%26aufirst%3DJ.%2BW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520discovery%2520of%2520cf53%2520as%2520a%2520potent%2520and%2520orally%2520bioavailable%2520bromodomain%2520and%2520extra-terminal%2520%2528bet%2529%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6110%26epage%3D6120%26doi%3D10.1021%2Facs.jmedchem.8b00483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of 4-(6-methoxy-2-methyl-4-(quinolin-4-yl)-9h-pyrimido[4,5-b]indol-7-yl)-3,5-dimethylisoxazole (cd161) as a potent and orally bioavailable bet bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3887</span>– <span class="NLM_lpage">3901</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00193</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00193" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvVKru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3887-3901&author=Y.+Zhaoauthor=L.+Baiauthor=L.+Liuauthor=D.+McEachernauthor=J.+A.+Stuckeyauthor=J.+L.+Meagherauthor=C.-Y.+Yangauthor=X.+Ranauthor=B.+Zhouauthor=Y.+Huauthor=X.+Liauthor=B.+Wenauthor=T.+Zhaoauthor=S.+Liauthor=D.+Sunauthor=S.+Wang&title=Structure-based+discovery+of+4-%286-methoxy-2-methyl-4-%28quinolin-4-yl%29-9h-pyrimido%5B4%2C5-b%5Dindol-7-yl%29-3%2C5-dimethylisoxazole+%28cd161%29+as+a+potent+and+orally+bioavailable+bet+bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.7b00193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of 4-(6-Methoxy-2-methyl-4-(quinolin-4-yl)-9H-pyrimido[4,5-b]indol-7-yl)-3,5-dimethylisoxazole (CD161) as a Potent and Orally Bioavailable BET Bromodomain Inhibitor</span></div><div class="casAuthors">Zhao, Yujun; Bai, Longchuan; Liu, Liu; McEachern, Donna; Stuckey, Jeanne A.; Meagher, Jennifer L.; Yang, Chao-Yie; Ran, Xu; Zhou, Bing; Hu, Yang; Li, Xiaoqin; Wen, Bo; Zhao, Ting; Li, Siwei; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3887-3901</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 9H-pyrimido[4,5-b]indole-contg. compds. was designed and synthesized to obtain potent and orally bioavailable BET inhibitors.  By incorporation of an indole or a quinoline moiety to the 9H-pyrimido[4,5-b]indole core, we identified a series of small mols. showing high binding affinities to BET proteins and low nanomolar potencies in inhibition of cell growth in acute leukemia cell lines.  One such compd., 4-(6-methoxy-2-methyl-4-(quinolin-4-yl)-9H-pyrimido[4,5-b]indol-7-yl)-3,5-dimethylisoxazole (I) has excellent microsomal stability and good oral pharmacokinetics in rats and mice.  Orally administered, I achieves significant antitumor activity in the MV4;11 leukemia and MDA-MB-231 triple-neg. breast cancer xenograft models in mice.  Detn. of the cocrystal structure of I with BRD4 BD2 provides a structural basis for its high binding affinity to BET proteins.  Testing its binding affinities against other bromodomain-contg. proteins shows that I is a highly selective inhibitor of BET proteins.  These data show that I is a potent, selective, and orally active BET inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQdUzlAHU5v7Vg90H21EOLACvtfcHk0lgElSNUcXbDrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvVKru7s%253D&md5=915019e36ab5f5b83b69b1bbc291abb0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00193%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DRan%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520discovery%2520of%25204-%25286-methoxy-2-methyl-4-%2528quinolin-4-yl%2529-9h-pyrimido%255B4%252C5-b%255Dindol-7-yl%2529-3%252C5-dimethylisoxazole%2520%2528cd161%2529%2520as%2520a%2520potent%2520and%2520orally%2520bioavailable%2520bet%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3887%26epage%3D3901%26doi%3D10.1021%2Facs.jmedchem.7b00193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baker, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zannettino, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalk, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkley, C. R.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors induce apoptosis through a MYC independent mechanism and synergise with CDK inhibitors to kill osteosarcoma cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10120</span>– <span class="NLM_lpage">10133</span>, <span class="refDoi"> DOI: 10.1038/srep10120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1038%2Fsrep10120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=25944566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOrsL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=10120-10133&author=E.+K.+Bakerauthor=S.+Taylorauthor=A.+Gupteauthor=P.+P.+Sharpauthor=M.+Waliaauthor=N.+C.+Walshauthor=A.+C.+Zannettinoauthor=A.+M.+Chalkauthor=C.+J.+Burnsauthor=C.+R.+Walkley&title=BET+inhibitors+induce+apoptosis+through+a+MYC+independent+mechanism+and+synergise+with+CDK+inhibitors+to+kill+osteosarcoma+cells&doi=10.1038%2Fsrep10120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors induce apoptosis through a MYC independent mechanism and synergise with CDK inhibitors to kill osteosarcoma cells</span></div><div class="casAuthors">Baker, Emma K.; Taylor, Scott; Gupte, Ankita; Sharp, Phillip P.; Walia, Mannu; Walsh, Nicole C.; Zannettino, Andrew C. W.; Chalk, Alistair M.; Burns, Christopher J.; Walkley, Carl R.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10120</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Osteosarcoma (OS) survival rates have plateaued in part due to a lack of new therapeutic options.  Here we demonstrate that bromodomain inhibitors (BETi), JQ1, I-BET151, I-BET762, exert potent anti-tumor activity against primary and established OS cell lines, mediated by inhibition of BRD4.  Strikingly, unlike previous observations in long-term established human OS cell lines, the antiproliferative activity of JQ1 in primary OS cells was driven by the induction of apoptosis, not cell cycle arrest.  In further contrast, JQ1 activity in OS was mediated independently of MYC downregulation.  We identified that JQ1 suppresses the transcription factor FOSL1 by displacement of BRD4 from its locus.  Loss of FOSL1 phenocopied the antiproliferative effects of JQ1, identifying FOSL1 suppression as a potential novel therapeutic approach for OS.  As a monotherapy JQ1 demonstrated significant anti-tumor activity in vivo in an OS graft model.  Further, combinatorial treatment approaches showed that JQ1 increased the sensitivity of OS cells to doxorubicin and induced potent synergistic activity when rationally combined with CDK inhibitors.  The greater level of activity achieved with the combination of BETi with CDK inhibitors demonstrates the efficacy of this combination therapy.  Taken together, our studies show that BET inhibitors are a promising new therapeutic for OS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiy0tg8DWClLVg90H21EOLACvtfcHk0lgElSNUcXbDrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOrsL7I&md5=c95d487ed91b342bd426413430ba36ee</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fsrep10120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep10120%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DE.%2BK.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DGupte%26aufirst%3DA.%26aulast%3DSharp%26aufirst%3DP.%2BP.%26aulast%3DWalia%26aufirst%3DM.%26aulast%3DWalsh%26aufirst%3DN.%2BC.%26aulast%3DZannettino%26aufirst%3DA.%2BC.%26aulast%3DChalk%26aufirst%3DA.%2BM.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26aulast%3DWalkley%26aufirst%3DC.%2BR.%26atitle%3DBET%2520inhibitors%2520induce%2520apoptosis%2520through%2520a%2520MYC%2520independent%2520mechanism%2520and%2520synergise%2520with%2520CDK%2520inhibitors%2520to%2520kill%2520osteosarcoma%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D10120%26epage%3D10133%26doi%3D10.1038%2Fsrep10120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stratikopoulos, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dendy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabolcs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaykin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefebvre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span> <span> </span><span class="NLM_article-title">Kinase and bet inhibitors together clamp inhibition of pi3k signaling and overcome resistance to therapy</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">837</span>– <span class="NLM_lpage">851</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.ccell.2015.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=26058079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVejurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=837-851&author=E.+E.+Stratikopoulosauthor=M.+Dendyauthor=M.+Szabolcsauthor=A.+J.+Khaykinauthor=C.+Lefebvreauthor=M.-M.+Zhouauthor=R.+Parsons&title=Kinase+and+bet+inhibitors+together+clamp+inhibition+of+pi3k+signaling+and+overcome+resistance+to+therapy&doi=10.1016%2Fj.ccell.2015.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase and BET Inhibitors Together Clamp Inhibition of PI3K Signaling and Overcome Resistance to Therapy</span></div><div class="casAuthors">Stratikopoulos, Elias E.; Dendy, Meaghan; Szabolcs, Matthias; Khaykin, Alan J.; Lefebvre, Celine; Zhou, Ming-Ming; Parsons, Ramon</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">837-851</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Unsustained enzyme inhibition is a barrier to targeted therapy for cancer.  Here, resistance to a class I PI3K inhibitor in a model of metastatic breast cancer driven by PI3K and MYC was assocd. with feedback activation of tyrosine kinase receptors (RTKs), AKT, mTOR, and MYC.  Inhibitors of bromodomain and extra terminal domain (BET) proteins also failed to affect tumor growth.  Interestingly, BET inhibitors lowered PI3K signaling and dissocd. BRD4 from chromatin at regulatory regions of insulin receptor and EGFR family RTKs to reduce their expression.  Combined PI3K and BET inhibition induced cell death, tumor regression, and clamped inhibition of PI3K signaling in a broad range of tumor cell lines to provide a strategy to overcome resistance to kinase inhibitor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmdX2xaXkHjbVg90H21EOLACvtfcHk0lgElSNUcXbDrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVejurbK&md5=91cca562c9bff9e11e94535916583867</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DStratikopoulos%26aufirst%3DE.%2BE.%26aulast%3DDendy%26aufirst%3DM.%26aulast%3DSzabolcs%26aufirst%3DM.%26aulast%3DKhaykin%26aufirst%3DA.%2BJ.%26aulast%3DLefebvre%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DParsons%26aufirst%3DR.%26atitle%3DKinase%2520and%2520bet%2520inhibitors%2520together%2520clamp%2520inhibition%2520of%2520pi3k%2520signaling%2520and%2520overcome%2520resistance%2520to%2520therapy%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D837%26epage%3D851%26doi%3D10.1016%2Fj.ccell.2015.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Čančer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drews, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bengtsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westermark, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelander, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsberg-Nilsson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhrbom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weishaupt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swartling, F. J.</span></span> <span> </span><span class="NLM_article-title">BET and aurora kinase a inhibitors synergize against mycn-positive human glioblastoma cells</span>. <i>Cell Death Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">881</span>– <span class="NLM_lpage">885</span>, <span class="refDoi"> DOI: 10.1038/s41419-019-2120-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1038%2Fs41419-019-2120-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=31754113" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=881-885&author=M.+%C4%8Can%C4%8Derauthor=L.+F.+Drewsauthor=J.+Bengtssonauthor=S.+Bolinauthor=G.+Rosenauthor=B.+Westermarkauthor=S.+Nelanderauthor=K.+Forsberg-Nilssonauthor=L.+Uhrbomauthor=H.+Weishauptauthor=F.+J.+Swartling&title=BET+and+aurora+kinase+a+inhibitors+synergize+against+mycn-positive+human+glioblastoma+cells&doi=10.1038%2Fs41419-019-2120-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41419-019-2120-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-019-2120-1%26sid%3Dliteratum%253Aachs%26aulast%3D%25C4%258Can%25C4%258Der%26aufirst%3DM.%26aulast%3DDrews%26aufirst%3DL.%2BF.%26aulast%3DBengtsson%26aufirst%3DJ.%26aulast%3DBolin%26aufirst%3DS.%26aulast%3DRosen%26aufirst%3DG.%26aulast%3DWestermark%26aufirst%3DB.%26aulast%3DNelander%26aufirst%3DS.%26aulast%3DForsberg-Nilsson%26aufirst%3DK.%26aulast%3DUhrbom%26aufirst%3DL.%26aulast%3DWeishaupt%26aufirst%3DH.%26aulast%3DSwartling%26aufirst%3DF.%2BJ.%26atitle%3DBET%2520and%2520aurora%2520kinase%2520a%2520inhibitors%2520synergize%2520against%2520mycn-positive%2520human%2520glioblastoma%2520cells%26jtitle%3DCell%2520Death%2520Discovery%26date%3D2019%26volume%3D10%26spage%3D881%26epage%3D885%26doi%3D10.1038%2Fs41419-019-2120-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sahni, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gayle, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonk, K. L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vite, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yori, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seachrist, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landis, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keri, R. A.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and extraterminal protein inhibition blocks growth of triple-negative breast cancers through the suppression of aurora kinases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">23756</span>– <span class="NLM_lpage">23768</span>, <span class="refDoi"> DOI: 10.1074/jbc.m116.738666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1074%2Fjbc.M116.738666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=27650498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2gsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=23756-23768&author=J.+M.+Sahniauthor=S.+S.+Gayleauthor=K.+L.+W.+Bonkauthor=L.+C.+Viteauthor=J.+L.+Yoriauthor=B.+Webbauthor=E.+K.+Ramosauthor=D.+D.+Seachristauthor=M.+D.+Landisauthor=J.+C.+Changauthor=J.+E.+Bradnerauthor=R.+A.+Keri&title=Bromodomain+and+extraterminal+protein+inhibition+blocks+growth+of+triple-negative+breast+cancers+through+the+suppression+of+aurora+kinases&doi=10.1074%2Fjbc.m116.738666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain and Extraterminal Protein Inhibition Blocks Growth of Triple-negative Breast Cancers through the Suppression of Aurora Kinases</span></div><div class="casAuthors">Sahni, Jennifer M.; Gayle, Sylvia S.; Weber Bonk, Kristen L.; Vite, Leslie Cuellar; Yori, Jennifer L.; Webb, Bryan; Ramos, Erika K.; Seachrist, Darcie D.; Landis, Melissa D.; Chang, Jenny C.; Bradner, James E.; Keri, Ruth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">23756-23768</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Bromodomain and extraterminal (BET) proteins are epigenetic "readers" that recognize acetylated histones and mark areas of the genome for transcription.  BRD4, a BET family member protein, has been implicated in a no. of types of cancer, and BET protein inhibitors (BETi) are efficacious in many preclin. cancer models.  However, the drivers of response to BETi vary depending on tumor type, and little is known regarding the target genes conveying BETi activity in triple-neg. breast cancer (TNBC).  Here, we show that BETi repress growth of multiple in vitro and in vivo models of TNBC by inducing two terminal responses: apoptosis and senescence.  Unlike in other cancers, response to BETi in TNBC is not dependent upon suppression of MYC.  Instead, both end points are preceded by the appearance of polyploid cells caused by the suppression of Aurora kinases A and B (AURKA/B), which are crit. mediators of mitosis.  In addn., AURKA/B inhibitors phenocopy the effects of BETi.  These results indicate that Aurora kinases play an important role in the growth suppressive activity of BETi in TNBC.  Elucidating the mechanism of response to BETi in TNBC should (1) facilitate the prediction of how distinct TNBC tumors will respond to BETi and (2) inform the rational design of drug combination therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-oIjctyeyMLVg90H21EOLACvtfcHk0lh2M_ju-1y2sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2gsLrI&md5=c39bc177e22af9219f7234460d1c5524</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.738666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.738666%26sid%3Dliteratum%253Aachs%26aulast%3DSahni%26aufirst%3DJ.%2BM.%26aulast%3DGayle%26aufirst%3DS.%2BS.%26aulast%3DBonk%26aufirst%3DK.%2BL.%2BW.%26aulast%3DVite%26aufirst%3DL.%2BC.%26aulast%3DYori%26aufirst%3DJ.%2BL.%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DRamos%26aufirst%3DE.%2BK.%26aulast%3DSeachrist%26aufirst%3DD.%2BD.%26aulast%3DLandis%26aufirst%3DM.%2BD.%26aulast%3DChang%26aufirst%3DJ.%2BC.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DKeri%26aufirst%3DR.%2BA.%26atitle%3DBromodomain%2520and%2520extraterminal%2520protein%2520inhibition%2520blocks%2520growth%2520of%2520triple-negative%2520breast%2520cancers%2520through%2520the%2520suppression%2520of%2520aurora%2520kinases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D23756%26epage%3D23768%26doi%3D10.1074%2Fjbc.m116.738666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minzel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatachalam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fink, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachya, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burstain, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdman, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushansky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shlush, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pikarsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snir-Alkalay, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Neriah, Y.</span></span> <span> </span><span class="NLM_article-title">Small molecules co-targeting ckialpha and the transcriptional kinases cdk7/9 control aml in preclinical models</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.07.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.cell.2018.07.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=30146162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFyktbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=171-185&author=W.+Minzelauthor=A.+Venkatachalamauthor=A.+Finkauthor=E.+Hungauthor=G.+Brachyaauthor=I.+Burstainauthor=M.+Shahamauthor=A.+Rivlinauthor=I.+Omerauthor=A.+Zingerauthor=S.+Eliasauthor=E.+Winterauthor=P.+E.+Erdmanauthor=R.+W.+Sullivanauthor=L.+Fungauthor=F.+Mercurioauthor=D.+Liauthor=J.+Vaccaauthor=N.+Kaushanskyauthor=L.+Shlushauthor=M.+Orenauthor=R.+Levineauthor=E.+Pikarskyauthor=I.+Snir-Alkalayauthor=Y.+Ben-Neriah&title=Small+molecules+co-targeting+ckialpha+and+the+transcriptional+kinases+cdk7%2F9+control+aml+in+preclinical+models&doi=10.1016%2Fj.cell.2018.07.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecules Co-targeting CKIα and the Transcriptional Kinases CDK7/9 Control AML in Preclinical Models</span></div><div class="casAuthors">Minzel, Waleed; Venkatachalam, Avanthika; Fink, Avner; Hung, Eric; Brachya, Guy; Burstain, Ido; Shaham, Maya; Rivlin, Amitai; Omer, Itay; Zinger, Adar; Elias, Shlomo; Winter, Eitan; Erdman, Paul E.; Sullivan, Robert W.; Fung, Leah; Mercurio, Frank; Li, Dansu; Vacca, Joseph; Kaushansky, Nathali; Shlush, Liran; Oren, Moshe; Levine, Ross; Pikarsky, Eli; Snir-Alkalay, Irit; Ben-Neriah, Yinon</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">171-185.e25</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">CKIα ablation induces p53 activation, and CKIα degrdn. underlies the therapeutic effect of lenalidomide in a pre-leukemia syndrome.  Here we describe the development of CKIα inhibitors, which co-target the transcriptional kinases CDK7 and CDK9, thereby augmenting CKIα-induced p53 activation and its anti-leukemic activity.  Oncogene-driving super-enhancers (SEs) are highly sensitive to CDK7/9 inhibition.  We identified multiple newly gained SEs in primary mouse acute myeloid leukemia (AML) cells and demonstrate that the inhibitors abolish many SEs and preferentially suppress the transcription elongation of SE-driven oncogenes.  We show that blocking CKIα together with CDK7 and/or CDK9 synergistically stabilize p53, deprive leukemia cells of survival and proliferation-maintaining SE-driven oncogenes, and induce apoptosis.  Leukemia progenitors are selectively eliminated by the inhibitors, explaining their therapeutic efficacy with preserved hematopoiesis and leukemia cure potential; they eradicate leukemia in MLL-AF9 and Tet2-/-;Flt3ITD AML mouse models and in several patient-derived AML xenograft models, supporting their potential efficacy in curing human leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4QEH6R_ll7Vg90H21EOLACvtfcHk0lh2M_ju-1y2sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFyktbvF&md5=b3e088a6b262dc8998ff334867102aa5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.07.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.07.045%26sid%3Dliteratum%253Aachs%26aulast%3DMinzel%26aufirst%3DW.%26aulast%3DVenkatachalam%26aufirst%3DA.%26aulast%3DFink%26aufirst%3DA.%26aulast%3DHung%26aufirst%3DE.%26aulast%3DBrachya%26aufirst%3DG.%26aulast%3DBurstain%26aufirst%3DI.%26aulast%3DShaham%26aufirst%3DM.%26aulast%3DRivlin%26aufirst%3DA.%26aulast%3DOmer%26aufirst%3DI.%26aulast%3DZinger%26aufirst%3DA.%26aulast%3DElias%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DE.%26aulast%3DErdman%26aufirst%3DP.%2BE.%26aulast%3DSullivan%26aufirst%3DR.%2BW.%26aulast%3DFung%26aufirst%3DL.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DVacca%26aufirst%3DJ.%26aulast%3DKaushansky%26aufirst%3DN.%26aulast%3DShlush%26aufirst%3DL.%26aulast%3DOren%26aufirst%3DM.%26aulast%3DLevine%26aufirst%3DR.%26aulast%3DPikarsky%26aufirst%3DE.%26aulast%3DSnir-Alkalay%26aufirst%3DI.%26aulast%3DBen-Neriah%26aufirst%3DY.%26atitle%3DSmall%2520molecules%2520co-targeting%2520ckialpha%2520and%2520the%2520transcriptional%2520kinases%2520cdk7%252F9%2520control%2520aml%2520in%2520preclinical%2520models%26jtitle%3DCell%26date%3D2018%26volume%3D175%26spage%3D171%26epage%3D185%26doi%3D10.1016%2Fj.cell.2018.07.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7346</span>– <span class="NLM_lpage">7359</span>, <span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0lh2M_ju-1y2sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tögel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nightingale, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chueh, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayachandran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phesse, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arango, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhillon, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gahman, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiau, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mariadason, J. M.</span></span> <span> </span><span class="NLM_article-title">Dual targeting of bromodomain and extraterminal domain proteins, and wnt or mapk signaling, inhibits c-myc expression and proliferation of colorectal cancer cells</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1217</span>– <span class="NLM_lpage">1226</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-15-0724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1158%2F1535-7163.MCT-15-0724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=26983878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC28XptVeqs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=1217-1226&author=L.+T%C3%B6gelauthor=R.+Nightingaleauthor=A.+C.+Chuehauthor=A.+Jayachandranauthor=H.+Tranauthor=T.+Phesseauthor=R.+Wuauthor=O.+M.+Sieberauthor=D.+Arangoauthor=A.+S.+Dhillonauthor=M.+A.+Dawsonauthor=B.+Diez-Dacalauthor=T.+C.+Gahmanauthor=P.+Filippakopoulosauthor=A.+K.+Shiauauthor=J.+M.+Mariadason&title=Dual+targeting+of+bromodomain+and+extraterminal+domain+proteins%2C+and+wnt+or+mapk+signaling%2C+inhibits+c-myc+expression+and+proliferation+of+colorectal+cancer+cells&doi=10.1158%2F1535-7163.mct-15-0724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Targeting of Bromodomain and Extraterminal Domain Proteins, and WNT or MAPK Signaling, Inhibits c-MYC Expression and Proliferation of Colorectal Cancer Cells</span></div><div class="casAuthors">Togel, Lars; Nightingale, Rebecca; Chueh, Anderly C.; Jayachandran, Aparna; Tran, Hoanh; Phesse, Toby; Wu, Rui; Sieber, Oliver M.; Arango, Diego; Dhillon, Amardeep S.; Dawson, Mark A.; Diez-Dacal, Beatriz; Gahman, Timothy C.; Filippakopoulos, Panagis; Shiau, Andrew K.; Mariadason, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1217-1226</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Inhibitors of the bromodomain and extraterminal domain (BET) protein family attenuate the proliferation of several tumor cell lines.  These effects are mediated, at least in part, through repression of c-MYC.  In colorectal cancer, overexpression of c-MYC due to hyperactive WNT/β-catenin/TCF signaling is a key driver of tumor progression; however, effective strategies to target this oncogene remain elusive.  Here, we investigated the effect of BET inhibitors (BETi) on colorectal cancer cell proliferation and c-MYC expression.  Treatment of 20 colorectal cancer cell lines with the BETi JQ1 identified a subset of highly sensitive lines.  JQ1 sensitivity was higher in cell lines with microsatellite instability but was not assocd. with the CpG island methylator phenotype, c-MYC expression or amplification status, BET protein expression, or mutation status of TP53, KRAS/BRAF, or PIK3CA/PTEN.  Conversely, JQ1 sensitivity correlated significantly with the magnitude of c-MYC mRNA and protein repression.  JQ1-mediated c-MYC repression was not due to generalized attenuation of β-catenin/TCF-mediated transcription, as JQ1 had minimal effects on other β-catenin/TCF target genes or β-catenin/TCF reporter activity.  BETi preferentially target super-enhancer-regulated genes, and a super-enhancer in c-MYC was recently identified in HCT116 cells to which BRD4 and effector transcription factors of the WNT/β-catenin/TCF and MEK/ERK pathways are recruited.  Combined targeting of c-MYC with JQ1 and inhibitors of these pathways additively repressed c-MYC and proliferation of HCT116 cells.  These findings demonstrate that BETi downregulate c-MYC expression and inhibit colorectal cancer cell proliferation and identify strategies for enhancing the effects of BETi on c-MYC repression by combinatorial targeting the c-MYC super-enhancer.  Mol Cancer Ther; 15(6); 1217-26. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptJtvXsYFDz7Vg90H21EOLACvtfcHk0lgATEm7PZ19Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptVeqs74%253D&md5=01f9e3ab8f181ca354bb890b58ee6adb</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-15-0724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-15-0724%26sid%3Dliteratum%253Aachs%26aulast%3DT%25C3%25B6gel%26aufirst%3DL.%26aulast%3DNightingale%26aufirst%3DR.%26aulast%3DChueh%26aufirst%3DA.%2BC.%26aulast%3DJayachandran%26aufirst%3DA.%26aulast%3DTran%26aufirst%3DH.%26aulast%3DPhesse%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DR.%26aulast%3DSieber%26aufirst%3DO.%2BM.%26aulast%3DArango%26aufirst%3DD.%26aulast%3DDhillon%26aufirst%3DA.%2BS.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DGahman%26aufirst%3DT.%2BC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DShiau%26aufirst%3DA.%2BK.%26aulast%3DMariadason%26aufirst%3DJ.%2BM.%26atitle%3DDual%2520targeting%2520of%2520bromodomain%2520and%2520extraterminal%2520domain%2520proteins%252C%2520and%2520wnt%2520or%2520mapk%2520signaling%252C%2520inhibits%2520c-myc%2520expression%2520and%2520proliferation%2520of%2520colorectal%2520cancer%2520cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D1217%26epage%3D1226%26doi%3D10.1158%2F1535-7163.mct-15-0724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhadel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollah, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xenarios, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conaway, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carey, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gioeli, D.</span></span> <span> </span><span class="NLM_article-title">CDK9 regulates AR promoter selectivity and cell growth through serine 81 phosphorylation</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2267</span>– <span class="NLM_lpage">2280</span>, <span class="refDoi"> DOI: 10.1210/me.2010-0238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1210%2Fme.2010-0238" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=20980437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVenurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=2267-2280&author=V.+Gordonauthor=S.+Bhadelauthor=W.+Wunderlichauthor=J.+Zhangauthor=S.+B.+Ficarroauthor=S.+A.+Mollahauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=I.+Xenariosauthor=W.+C.+Hahnauthor=M.+Conawayauthor=M.+F.+Careyauthor=D.+Gioeli&title=CDK9+regulates+AR+promoter+selectivity+and+cell+growth+through+serine+81+phosphorylation&doi=10.1210%2Fme.2010-0238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">CDK9 regulates AR promoter selectivity and cell growth through serine 81 phosphorylation</span></div><div class="casAuthors">Gordon, Vicki; Bhadel, Shriti; Wunderlich, Winfried; Zhang, JoAnn; Ficarro, Scott B.; Mollah, Sahana A.; Shabanowitz, Jeffrey; Hunt, Donald F.; Xenarios, Ioannis; Hahn, William C.; Conaway, Mark; Carey, Michael F.; Gioeli, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2267-2280</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Previously we detd. that S81 is the highest stoichiometric phosphorylation on the androgen receptor (AR) in response to hormone.  To explore the role of this phosphorylation on growth, we stably expressed wild-type and S81A mutant AR in LHS and LAPC4 cells.  The cells with increased wild-type AR expression grow faster compared with parental cells and S81A mutant-expressing cells, indicating that loss of S81 phosphorylation limits cell growth.  To explore how S81 regulates cell growth, we tested whether S81 phosphorylation regulates AR transcriptional activity.  LHS cells stably expressing wild-type and S81A mutant AR showed differences in the regulation of endogenous AR target genes, suggesting that S81 phosphorylation regulates promoter selectivity.  We next sought to identify the S81 kinase using ion trap mass spectrometry to analyze AR-assocd. proteins in immunoppts. from cells.  We obsd. cyclin-dependent kinase (CDK)9 assocn. with the AR.  CDK9 phosphorylates the AR on S81 in vitro.  Phosphorylation is specific to S81 because CDK9 did not phosphorylate the AR on other serine phosphorylation sites.  Overexpression of CDK9 with its cognate cyclin, Cyclin T, increased S81 phosphorylation levels in cells.  Small interfering RNA knockdown of CDK9 protein levels decreased hormone-induced S81 phosphorylation.  Addnl., treatment of LNCaP cells with the CDK9 inhibitors, 5,6-dichloro-1-β-D-ribofuranosylbenzimidazole and Flavopiridol, reduced S81 phosphorylation further, suggesting that CDK9 regulates S81 phosphorylation.  Pharmacol. inhibition of CDK9 also resulted in decreased AR transcription in LNCaP cells.  Collectively these results suggest that CDK9 phosphorylation of AR S81 is an important step in regulating AR transcriptional activity and prostate cancer cell growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWRr14oBHWlrVg90H21EOLACvtfcHk0lgATEm7PZ19Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVenurY%253D&md5=577d202cad2fa850526dde99e69cf49d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1210%2Fme.2010-0238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2010-0238%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DV.%26aulast%3DBhadel%26aufirst%3DS.%26aulast%3DWunderlich%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DMollah%26aufirst%3DS.%2BA.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DXenarios%26aufirst%3DI.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DConaway%26aufirst%3DM.%26aulast%3DCarey%26aufirst%3DM.%2BF.%26aulast%3DGioeli%26aufirst%3DD.%26atitle%3DCDK9%2520regulates%2520AR%2520promoter%2520selectivity%2520and%2520cell%2520growth%2520through%2520serine%252081%2520phosphorylation%26jtitle%3DMol.%2520Endocrinol.%26date%3D2010%26volume%3D24%26spage%3D2267%26epage%3D2280%26doi%3D10.1210%2Fme.2010-0238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsten, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhogbi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kool, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisterernst, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerl, K.</span></span> <span> </span><span class="NLM_article-title">Combined BRD4 and CDK9 inhibition as a new therapeutic approach in malignant rhabdoid tumors</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">84986</span>– <span class="NLM_lpage">84995</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.18583</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.18632%2Foncotarget.18583" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=29156698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A280%3ADC%252BC1M3jtFSitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=84986-84995&author=N.+Morenoauthor=T.+Holstenauthor=J.+Mertinsauthor=A.+Zhogbiauthor=P.+Johannauthor=M.+Koolauthor=M.+Meisterernstauthor=K.+Kerl&title=Combined+BRD4+and+CDK9+inhibition+as+a+new+therapeutic+approach+in+malignant+rhabdoid+tumors&doi=10.18632%2Foncotarget.18583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Combined BRD4 and CDK9 inhibition as a new therapeutic approach in malignant rhabdoid tumors</span></div><div class="casAuthors">Moreno Natalia; Holsten Till; Mertins Julius; Zhogbi Annabelle; Meisterernst Michael; Kerl Kornelius; Moreno Natalia; Kerl Kornelius; Johann Pascal; Kool Marcel</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">84986-84995</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Rhabdoid tumors are caused by the deletion of SMARCB1, whose protein encodes the SMARCB1 subunit of the chromatin remodeling complex SWI/SNF that is involved in global chromatin organization and gene expression control.  Simultaneously inhibiting the main players involved in the deregulated transcription machinery is a promising option for preventing exaggerated tumor cell proliferation and survival as it may bypass compensatory mechanisms.  In support of this hypothesis, we report efficient impairment of cellular proliferation and strong induction of cell death elicited by inhibition of bromodomain protein BRD4 and transcription kinase CDK9 using small molecular compounds.  Combination of both compounds efficiently represses antiapoptotic genes and the oncogene MYC.  Our results provide a novel approach for the treatment of RT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxaJ-Xi0XM-zdblLE_E9PWfW6udTcc2ea5XJkGXyg2-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3jtFSitQ%253D%253D&md5=8cf6e23c68ef1a2ba66c5cb9fac43656</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.18583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.18583%26sid%3Dliteratum%253Aachs%26aulast%3DMoreno%26aufirst%3DN.%26aulast%3DHolsten%26aufirst%3DT.%26aulast%3DMertins%26aufirst%3DJ.%26aulast%3DZhogbi%26aufirst%3DA.%26aulast%3DJohann%26aufirst%3DP.%26aulast%3DKool%26aufirst%3DM.%26aulast%3DMeisterernst%26aufirst%3DM.%26aulast%3DKerl%26aufirst%3DK.%26atitle%3DCombined%2520BRD4%2520and%2520CDK9%2520inhibition%2520as%2520a%2520new%2520therapeutic%2520approach%2520in%2520malignant%2520rhabdoid%2520tumors%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D84986%26epage%3D84995%26doi%3D10.18632%2Foncotarget.18583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brägelmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dammert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietlein, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuckmann, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choidas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tischler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habenberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz-Cuaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leenders, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickhoff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Termathe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sallouh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greff, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varga, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balke-Want, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peifer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhardt, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Örfi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kéri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansén, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heukamp, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Büttner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebl, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sos, M. L.</span></span> <span> </span><span class="NLM_article-title">Systematic kinase inhibitor profiling identifies cdk9 as a synthetic lethal target in nut midline carcinoma</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2833</span>– <span class="NLM_lpage">2845</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.08.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.celrep.2017.08.082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=28930680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FgsFygtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=2833-2845&author=J.+Br%C3%A4gelmannauthor=M.+A.+Dammertauthor=F.+Dietleinauthor=J.+M.+Heuckmannauthor=A.+Choidasauthor=S.+B%C3%B6hmauthor=A.+Richtersauthor=D.+Basuauthor=V.+Tischlerauthor=C.+Lorenzauthor=P.+Habenbergerauthor=Z.+Fangauthor=S.+Ortiz-Cuaranauthor=F.+Leendersauthor=J.+Eickhoffauthor=U.+Kochauthor=M.+Getlikauthor=M.+Termatheauthor=M.+Sallouhauthor=Z.+Greffauthor=Z.+Vargaauthor=H.+Balke-Wantauthor=C.+A.+Frenchauthor=M.+Peiferauthor=H.+C.+Reinhardtauthor=L.+%C3%96rfiauthor=G.+K%C3%A9riauthor=S.+Ans%C3%A9nauthor=L.+C.+Heukampauthor=R.+B%C3%BCttnerauthor=D.+Rauhauthor=B.+M.+Kleblauthor=R.+K.+Thomasauthor=M.+L.+Sos&title=Systematic+kinase+inhibitor+profiling+identifies+cdk9+as+a+synthetic+lethal+target+in+nut+midline+carcinoma&doi=10.1016%2Fj.celrep.2017.08.082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Kinase Inhibitor Profiling Identifies CDK9 as a Synthetic Lethal Target in NUT Midline Carcinoma</span></div><div class="casAuthors">Bragelmann Johannes; Dammert Marcel A; Bohm Stefanie; Lorenz Carina; Dietlein Felix; Ansen Sascha; Heuckmann Johannes M; Choidas Axel; Habenberger Peter; Eickhoff Jan; Koch Uwe; Klebl Bert M; Richters Andre; Basu Debjit; Fang Zhizhou; Getlik Matthaus; Termathe Martin; Sallouh Muhammad; Rauh Daniel; Tischler Verena; Ortiz-Cuaran Sandra; Leenders Frauke; Peifer Martin; Greff Zoltan; Varga Zoltan; Keri Gyorgy; Balke-Want Hyatt; French Christopher A; Reinhardt H Christian; Orfi Laszlo; Heukamp Lukas C; Buttner Reinhard; Thomas Roman K; Sos Martin L</div><div class="citationInfo"><span class="NLM_cas:title">Cell reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2833-2845</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Kinase inhibitors represent the backbone of targeted cancer therapy, yet only a limited number of oncogenic drivers are directly druggable.  By interrogating the activity of 1,505 kinase inhibitors, we found that BRD4-NUT-rearranged NUT midline carcinoma (NMC) cells are specifically killed by CDK9 inhibition (CDK9i) and depend on CDK9 and Cyclin-T1 expression.  We show that CDK9i leads to robust induction of apoptosis and of markers of DNA damage response in NMC cells.  While both CDK9i and bromodomain inhibition over time result in reduced Myc protein expression, only bromodomain inhibition induces cell differentiation and a p21-induced cell-cycle arrest in these cells.  Finally, RNA-seq and ChIP-based analyses reveal a BRD4-NUT-specific CDK9i-induced perturbation of transcriptional elongation.  Thus, our data provide a mechanistic basis for the genotype-dependent vulnerability of NMC cells to CDK9i that may be of relevance for the development of targeted therapies for NMC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRoGX5-pfeoO-qrn-c4Of3KfW6udTcc2ea5XJkGXyg2-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FgsFygtA%253D%253D&md5=8f8e1405907c5132a15c875f84ac94ba</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.08.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.08.082%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25A4gelmann%26aufirst%3DJ.%26aulast%3DDammert%26aufirst%3DM.%2BA.%26aulast%3DDietlein%26aufirst%3DF.%26aulast%3DHeuckmann%26aufirst%3DJ.%2BM.%26aulast%3DChoidas%26aufirst%3DA.%26aulast%3DB%25C3%25B6hm%26aufirst%3DS.%26aulast%3DRichters%26aufirst%3DA.%26aulast%3DBasu%26aufirst%3DD.%26aulast%3DTischler%26aufirst%3DV.%26aulast%3DLorenz%26aufirst%3DC.%26aulast%3DHabenberger%26aufirst%3DP.%26aulast%3DFang%26aufirst%3DZ.%26aulast%3DOrtiz-Cuaran%26aufirst%3DS.%26aulast%3DLeenders%26aufirst%3DF.%26aulast%3DEickhoff%26aufirst%3DJ.%26aulast%3DKoch%26aufirst%3DU.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DTermathe%26aufirst%3DM.%26aulast%3DSallouh%26aufirst%3DM.%26aulast%3DGreff%26aufirst%3DZ.%26aulast%3DVarga%26aufirst%3DZ.%26aulast%3DBalke-Want%26aufirst%3DH.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DPeifer%26aufirst%3DM.%26aulast%3DReinhardt%26aufirst%3DH.%2BC.%26aulast%3D%25C3%2596rfi%26aufirst%3DL.%26aulast%3DK%25C3%25A9ri%26aufirst%3DG.%26aulast%3DAns%25C3%25A9n%26aufirst%3DS.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DB%25C3%25BCttner%26aufirst%3DR.%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DKlebl%26aufirst%3DB.%2BM.%26aulast%3DThomas%26aufirst%3DR.%2BK.%26aulast%3DSos%26aufirst%3DM.%2BL.%26atitle%3DSystematic%2520kinase%2520inhibitor%2520profiling%2520identifies%2520cdk9%2520as%2520a%2520synthetic%2520lethal%2520target%2520in%2520nut%2520midline%2520carcinoma%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D20%26spage%3D2833%26epage%3D2845%26doi%3D10.1016%2Fj.celrep.2017.08.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Q.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.-H.</span></span> <span> </span><span class="NLM_article-title">Upgrade of macromolecular crystallography beamline BL17U1 at SSRF</span>. <i>Nucl. Sci. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1007/s41365-018-0398-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1007%2Fs41365-018-0398-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=68-74&author=Q.-S.+Wangauthor=K.-H.+Zhangauthor=Y.+Cuauthor=Z.-J.+Wangauthor=Q.-Y.+Panauthor=K.+Liuauthor=M.-J.+Liauthor=Q.+Xuauthor=C.-Y.+Xuauthor=F.+Yuauthor=J.-H.+He&title=Upgrade+of+macromolecular+crystallography+beamline+BL17U1+at+SSRF&doi=10.1007%2Fs41365-018-0398-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fs41365-018-0398-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs41365-018-0398-9%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.-S.%26aulast%3DZhang%26aufirst%3DK.-H.%26aulast%3DCu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.-J.%26aulast%3DPan%26aufirst%3DQ.-Y.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DM.-J.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DC.-Y.%26aulast%3DYu%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DJ.-H.%26atitle%3DUpgrade%2520of%2520macromolecular%2520crystallography%2520beamline%2520BL17U1%2520at%2520SSRF%26jtitle%3DNucl.%2520Sci.%2520Technol.%26date%3D2018%26volume%3D29%26spage%3D68%26epage%3D74%26doi%3D10.1007%2Fs41365-018-0398-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span>; <span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-ray diffraction data collected in oscillation mode</span>.  <i>Methods in Enzymology</i>, <span class="NLM_year">1997</span>; Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minor&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26btitle%3DMethods%2520in%2520Enzymology%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-Likelihood Method</span>. <i>Acta Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1107%2Fs0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-Likelihood+Method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgVvMfxEgrHbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2Fs0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-Likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <i>Acta Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/s0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1107%2Fs0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.-W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2Fs0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0lgVvMfxEgrHbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2Fs0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2Fs0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgVvMfxEgrHbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0ljKkkBuNi0tnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span> <span> </span><span class="NLM_article-title">Fragment-based drug discovery of 2-thiazolidinones as inhibitors of the histone reader BRD4 bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3833</span>– <span class="NLM_lpage">3851</span>, <span class="refDoi"> DOI: 10.1021/jm301793a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301793a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3833-3851&author=L.+Zhaoauthor=D.+Caoauthor=T.+Chenauthor=Y.+Wangauthor=Z.+Miaoauthor=Y.+Xuauthor=W.+Chenauthor=X.+Wangauthor=Y.+Liauthor=Z.+Duauthor=B.+Xiongauthor=J.+Liauthor=C.+Xuauthor=N.+Zhangauthor=J.+Heauthor=J.+Shen&title=Fragment-based+drug+discovery+of+2-thiazolidinones+as+inhibitors+of+the+histone+reader+BRD4+bromodomain&doi=10.1021%2Fjm301793a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery of 2-Thiazolidinones as Inhibitors of the Histone Reader BRD4 Bromodomain</span></div><div class="casAuthors">Zhao, Lele; Cao, Danyan; Chen, Tiantian; Wang, Yingqing; Miao, Zehong; Xu, Yechun; Chen, Wuyan; Wang, Xin; Li, Yanlian; Du, Zhiyan; Xiong, Bing; Li, Jian; Xu, Chunyan; Zhang, Naixia; He, Jianhua; Shen, Jingkang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3833-3851</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recognizing acetyllysine of histone is a vital process of epigenetic regulation that is mediated by a protein module called bromodomain.  To contribute novel scaffolds for developing into bromodomain inhibitors, the authors utilize a fragment-based drug discovery approach.  By successively applying docking and x-ray crystallog., they were able to identify 9 fragment hits from diffracting more than 60 crystals.  In the present work, they described four of them and carried out the integrated lead optimization for fragment I, which bears a 2-thiazolidinone core.  After several rounds of structure guided modifications, they assessed the druggability of 2-thiazolidinone by modulating in vitro pharmacokinetic studies and cellular activity assay.  The results showed that two potent of 2-thiazolidinone compds. II and III have good metabolic stability.  Also, the cellular assay confirmed the activities of 2-thiazolidinones.  Together, they hope the identified 2-thiazolidinone chemotype and other fragment hits described herein can stimulate researchers to develop more diversified bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfi2pyYq6c4LVg90H21EOLACvtfcHk0ljKkkBuNi0tnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D&md5=8308db5d83cb72b29f261cc134514ece</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm301793a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301793a%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DFragment-based%2520drug%2520discovery%2520of%25202-thiazolidinones%2520as%2520inhibitors%2520of%2520the%2520histone%2520reader%2520BRD4%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3833%26epage%3D3851%26doi%3D10.1021%2Fjm301793a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Fragment-based drug discovery of 2-thiazolidinones as BRD4 inhibitors: 2. Structure-based optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1281</span>– <span class="NLM_lpage">1297</span>, <span class="refDoi"> DOI: 10.1021/jm501504k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501504k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFSqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1281-1297&author=L.+Zhaoauthor=Y.+Wangauthor=D.+Caoauthor=T.+Chenauthor=Q.+Wangauthor=Y.+Liauthor=Y.+Xuauthor=N.+Zhangauthor=X.+Wangauthor=D.+Chenauthor=L.+Chenauthor=Y.-L.+Chenauthor=G.+Xiaauthor=Z.+Shiauthor=Y.-C.+Liuauthor=Y.+Linauthor=Z.+Miaoauthor=J.+Shenauthor=B.+Xiong&title=Fragment-based+drug+discovery+of+2-thiazolidinones+as+BRD4+inhibitors%3A+2.+Structure-based+optimization&doi=10.1021%2Fjm501504k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery of 2-Thiazolidinones as BRD4 Inhibitors: Structure-Based Optimization</span></div><div class="casAuthors">Zhao, Lele; Wang, Yingqing; Cao, Danyan; Chen, Tiantian; Wang, Qi; Li, Yanlian; Xu, Yechun; Zhang, Naixia; Wang, Xin; Chen, Danqi; Chen, Lin; Chen, Yue-Lei; Xia, Guangxin; Shi, Zhe; Liu, Yu-Chih; Lin, Yijyun; Miao, Zehong; Shen, Jingkang; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1281-1297</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The signal transduction of acetylated histone can be processed through a recognition module, bromodomain.  Several inhibitors targeting BRD4, one of the bromodomain members, are in clin. trials as anticancer drugs.  Hereby, the authors report the efforts on discovery and optimization of a new series of 2-thiazolidinones as BRD4 inhibitors along the previous study.  In this work, guided by crystal structure anal., the authors reversed the sulfonamide group and identified a new binding mode.  A structure-activity relationship study on this new series led to several potent BRD4 inhibitors with IC50 of about 0.05-0.1 μM in FP binding assay and GI50 of 0.1-0.3 μM in cell based assays.  To complete the lead-like assessment of this series, the authors further checked its effects on BRD4 downstream protein c-Myc, investigated its selectivity among five different bromodomain proteins, as well as the metabolic stability test, and reinforced the utility of 2-thiazolidinone scaffold as BET bromodomain inhibitors in novel anticancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4x-5lE2iPzbVg90H21EOLACvtfcHk0ljKkkBuNi0tnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFSqtw%253D%253D&md5=4056ac1a6d6598dc61be7cc7ef82a3da</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm501504k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501504k%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.-L.%26aulast%3DXia%26aufirst%3DG.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.-C.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DFragment-based%2520drug%2520discovery%2520of%25202-thiazolidinones%2520as%2520BRD4%2520inhibitors%253A%25202.%2520Structure-based%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1281%26epage%3D1297%26doi%3D10.1021%2Fjm501504k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huo, F.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.-S.</span></span> <span> </span><span class="NLM_article-title">PAK5 promotes the migration and invasion of cervical cancer cells by phosphorylating SATB1</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">994</span>– <span class="NLM_lpage">1006</span>, <span class="refDoi"> DOI: 10.1038/s41418-018-0178-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1038%2Fs41418-018-0178-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=30082769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGrs7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=994-1006&author=F.-C.+Huoauthor=Y.-J.+Panauthor=T.-T.+Liauthor=J.+Mouauthor=D.-S.+Pei&title=PAK5+promotes+the+migration+and+invasion+of+cervical+cancer+cells+by+phosphorylating+SATB1&doi=10.1038%2Fs41418-018-0178-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">PAK5 promotes the migration and invasion of cervical cancer cells by phosphorylating SATB1</span></div><div class="casAuthors">Huo, Fu-Chun; Pan, Yao-Jie; Li, Tong-Tong; Mou, Jie; Pei, Dong-Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Differentiation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">994-1006</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">P21-activated kinase 5 (PAK5) is involved in several oncogenic signaling pathways and its amplification or overexpression has been found in various types of cancer; however, the pathophysiol. role of PAK5 in cervical cancer (CC) remains elusive.  This study aims to elucidate the effects of PAK5 on CC metastasis and its specific regulation mechanism.  We performed western blotting and immunohistochem. (IHC) anal. and found that the expression levels of PAK5 were significantly upregulated in CC cells and tissues.  In addn., statistical anal. via IHC showed that increased PAK5 significantly correlated with CC progression.  Mn2+-Phos-tag SDS-PAGE, western blotting, immunofluorescence and dual luciferase reporter assays were utilized to det. the involvement of SATB1 in PAK5-mediated epithelial-mesenchymal transition (EMT).  We found that PAK5-mediated special AT-rich binding protein-1 (SATB1) phosphorylation on Ser47 initiated EMT cascade and promoted migration and invasion of CC cells.  Furthermore, overexpression of PAK5 induced lung metastasis of CC cells in xenograft modes.  Taken together, we conclude that PAK5 is a novel prognostic indicator and plays an important role in the CC metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo24tsedMNhbVg90H21EOLACvtfcHk0ljXi1fClOpqVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGrs7fO&md5=52c3beae667f631c03592729b7785d60</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fs41418-018-0178-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41418-018-0178-4%26sid%3Dliteratum%253Aachs%26aulast%3DHuo%26aufirst%3DF.-C.%26aulast%3DPan%26aufirst%3DY.-J.%26aulast%3DLi%26aufirst%3DT.-T.%26aulast%3DMou%26aufirst%3DJ.%26aulast%3DPei%26aufirst%3DD.-S.%26atitle%3DPAK5%2520promotes%2520the%2520migration%2520and%2520invasion%2520of%2520cervical%2520cancer%2520cells%2520by%2520phosphorylating%2520SATB1%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2019%26volume%3D26%26spage%3D994%26epage%3D1006%26doi%3D10.1038%2Fs41418-018-0178-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Pan Tang, Jifa Zhang, Jie Liu, Cheng-Ming Chiang, <span class="NLM_string-name hlFld-ContribAuthor">Liang Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2419-2435. <a href="https://doi.org/10.1021/acs.jmedchem.0c01487" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01487%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BBromodomain%252Band%252BExtraterminal%252BProteins%252Bfor%252BDrug%252BDiscovery%25253A%252BFrom%252BCurrent%252BProgress%252Bto%252BTechnological%252BDevelopment%26aulast%3DTang%26aufirst%3DPan%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26082020%26date%3D22022021%26volume%3D64%26issue%3D5%26spage%3D2419%26epage%3D2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhangxu  He</span>, <span class="hlFld-ContribAuthor ">Haomiao  Jiao</span>, <span class="hlFld-ContribAuthor ">Qi  An</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Dan  Zengyangzong</span>, <span class="hlFld-ContribAuthor ">Jiale  Xu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>, <span class="hlFld-ContribAuthor ">Liying  Ma</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel 4-phenylquinazoline-based BRD4 inhibitors for cardiac fibrosis. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>399 </em><a href="https://doi.org/10.1016/j.apsb.2021.07.018" title="DOI URL">https://doi.org/10.1016/j.apsb.2021.07.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2021.07.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2021.07.018%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Bnovel%252B4-phenylquinazoline-based%252BBRD4%252Binhibitors%252Bfor%252Bcardiac%252Bfibrosis%26aulast%3DHe%26aufirst%3DZhangxu%26date%3D2021%26volume%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiawen  Cao</span>, <span class="hlFld-ContribAuthor ">Tiantian  Fan</span>, <span class="hlFld-ContribAuthor ">Yanlian  Li</span>, <span class="hlFld-ContribAuthor ">Zhiyan  Du</span>, <span class="hlFld-ContribAuthor ">Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Xin  Wang</span>, <span class="hlFld-ContribAuthor ">Jingkang  Shen</span>, <span class="hlFld-ContribAuthor ">Xun  Huang</span>, <span class="hlFld-ContribAuthor ">Bing  Xiong</span>, <span class="hlFld-ContribAuthor ">Danyan  Cao</span>. </span><span class="cited-content_cbyCitation_article-title">Phage-Display Based Discovery and Characterization of Peptide Ligands against WDR5. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (5)
                                     , 1225. <a href="https://doi.org/10.3390/molecules26051225" title="DOI URL">https://doi.org/10.3390/molecules26051225</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26051225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26051225%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DPhage-Display%252BBased%252BDiscovery%252Band%252BCharacterization%252Bof%252BPeptide%252BLigands%252Bagainst%252BWDR5%26aulast%3DCao%26aufirst%3DJiawen%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D5%26spage%3D1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative dual bromodomain-kinase inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cocrystal structure of compound <b>7</b> (cyans) and compound <b>40</b> (blue) in complex with BRD4-BD1 (PDB entry number <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIH">6LIH</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIM">6LIM</a>, respectively). (A) Binding mode of compound <b>7</b> in the KAc binding pocket. Asn140, Tyr97, Trp81, and Asp145 are represented as the stick model, and hydrogen bonds are represented by dashed lines. (B) Superimposing the structure of compound <b>7</b> to our previous crystal structure (PDB entry number: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI5">6JI5</a>) (green). (C) Superimposing the structure of compound <b>7</b> and compound <b>40</b>. (D) Interactions between compound <b>40</b> and WPF pocket that is represented as a surface diagram.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding conformation of compounds <b>7</b>, <b>21</b>, and <b>40</b> in the CDK9 ATP binding site from the docking study. The protein is shown in the cartoon mode, and the residues of the binding site are shown in the stick mode. (A) Predicted conformation of compound <b>7</b> (cyans) bound to the ATP binding site of CDK9. It can form two hydrogen bonds with hinge residue Cys106. (B) Superimposition of compound <b>7</b> with original ligand A86 (green) in the crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH">6GZH</a>. (C) Predicted conformation of compound <b>21</b> (orange) bound to the ATP binding site of CDK9. (D) Predicted conformation of compound <b>40</b> (blue) bound to the ATP binding site of CDK9.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Selectivity profile of compound <b>40</b> on selected bromodomains and DiscoveRx diversity-kinase-panel. All inhibition values are mean values from at least two independent experiments at the concentration of 1 μM, and for kinase selectivity assay, only kinases with an inhibition ratio of more than 50% were depicted in the <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and whole data were listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Supporting Information</a> (Table S1).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proliferation inhibition of 22 cancer cell lines by Compound <b>40</b>.The cutoff line (red dotted line) represents IC<sub>50</sub> values that were larger than 500 nM. Precise values are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf" class="ext-link">Supporting Information</a>, Table S5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. HCT-116 cells were treated with the indicated concentration of JQ1, Atuveciclib, or compound <b>40</b> and examined BRD4, c-Myc, CDK9, p-RNA II CTD, P-AR, and GAPDH protein levels by western blotting. The molecular weights of different isoforms of BRD4, c-Myc, and CDK9 are shown at the right of the bands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <span class="smallcaps smallerCapital">d</span>-alanine, K<sub>2</sub>CO<sub>3</sub>, EtOH/H<sub>2</sub>O = 3:1, 80 °C, 8 h; (b) K<sub>2</sub>CO<sub>3</sub>, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, H<sub>2</sub>O, 60 °C, overnight; (c) acetone, phenylsilane, dibutyltindichloride, THF, rt, 10 h; (d) NaH, 0 °C–rt, 30 min and then iodomethane, DMF, rt, 2 h; (e) <i>tert</i>-butyl carbamate, CS<sub>2</sub>CO<sub>3</sub>, Pa(OAc)<sub>2</sub>, Xphos, 1.4-dioxane, 85 °C, 2 d; (f) HCl in 1.4-dioxane, rt, 24 h; (g) AcOH, 1.4-dioxane, 100 °C, 8 h; (h) HCl in 1.4-dioxane, 100 °C, 8 h; (i) POCl<sub>3</sub>, 60 °C, 8 h; and (j) K<sub>2</sub>CO<sub>3,</sub> Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, 1.4-dioxane: H<sub>2</sub>O = 4:1, 80 °C, 4 h.</p></p></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>11–14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions:(a) phenylboronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, 1.4-dioxane/H<sub>2</sub>O = 4:1, 80 °C, 8 h; (b) <b>48</b>, <i>t</i>-BuONa, Pa<sub>2</sub>(dba)<sub>3</sub>, Xphos, 1.4-dioxane, 100 °C, overnight.</p></p></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/medium/jm0c00962_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>5</b> and <b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00962/20200903/images/large/jm0c00962_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00962&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaHCO<sub>3,</sub> Pa(PPh<sub>3</sub>)<sub>4</sub>, 1.4-dioxane/H<sub>2</sub>O = 4:1, 70 °C, 2 h; (b) 2-amino-4-methoxypyrimidine, <i>t</i>-BuONa, Pa<sub>2</sub>(dba)<sub>3</sub>, Xphos, 1.4-dioxane, 100 °C, 8 h; (c) HCl in 1.4-dioxane, 100 °C, 8 h; (d) POCl<sub>3</sub>, 60 °C, 8 h; (e) K<sub>2</sub>CO<sub>3</sub>, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, 1.4-dioxane/H<sub>2</sub>O = 4:1, 80 °C, 4 h.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i90">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34532" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34532" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 44 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anighoro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology: challenges and opportunities in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">7874</span>– <span class="NLM_lpage">7887</span>, <span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+challenges+and+opportunities+in+drug+discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0ljZDk4MSJ_bmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520challenges%2520and%2520opportunities%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raghavendra, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pingili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadasi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mettu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, S. V. U. M.</span></span> <span> </span><span class="NLM_article-title">Dual or multi-targeting inhibitors: The next generation anticancer agents</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">1277</span>– <span class="NLM_lpage">1300</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.ejmech.2017.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=29126724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslyrt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2018&pages=1277-1300&author=N.+M.+Raghavendraauthor=D.+Pingiliauthor=S.+Kadasiauthor=A.+Mettuauthor=S.+V.+U.+M.+Prasad&title=Dual+or+multi-targeting+inhibitors%3A+The+next+generation+anticancer+agents&doi=10.1016%2Fj.ejmech.2017.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Dual or multi-targeting inhibitors: The next generation anticancer agents</span></div><div class="casAuthors">Raghavendra, Nulgumnalli Manjunathaiah; Pingili, Divya; Kadasi, Sundeep; Mettu, Akhila; Prasad, S. V. U. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1277-1300</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Dual-targeting/Multi-targeting of oncoproteins by a single drug mol. represents an efficient, logical and alternative approach to drug combinations.  An increasing interest in this approach is indicated by a steady upsurge in the no. of articles on targeting dual/multi proteins published in the last 5 years.  Combining different inhibitors that destiny specific single target is the std. treatment for cancer.  A new generation of dual or multi-targeting drugs is emerging, where a single chem. entity can act on multiple mol. targets.  Dual/Multi-targeting agents are beneficial for solving limited efficiencies, poor safety and resistant profiles of an individual target.  Designing dual/multi-target inhibitors with predefined biol. profiles present a challenge.  The latest advances in bioinformatic tools and the availability of detailed structural information of target proteins have shown a way of discovering multi-targeting mols.  This neoteric artifice that amalgamates the mol. docking of small mols. with protein-based common pharmacophore to design multi-targeting inhibitors is gaining great importance in anticancer drug discovery.  Current review focus on the discoveries of dual targeting agents in cancer therapy using rational, computational, proteomic, bioinformatics and polypharmacol. approach that enables the discovery and rational design of effective and safe multi-target anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVpBUpxW1okLVg90H21EOLACvtfcHk0ljZDk4MSJ_bmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslyrt7jO&md5=a14783194c23f7dc5f200cf6eabf9269</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRaghavendra%26aufirst%3DN.%2BM.%26aulast%3DPingili%26aufirst%3DD.%26aulast%3DKadasi%26aufirst%3DS.%26aulast%3DMettu%26aufirst%3DA.%26aulast%3DPrasad%26aufirst%3DS.%2BV.%2BU.%2BM.%26atitle%3DDual%2520or%2520multi-targeting%2520inhibitors%253A%2520The%2520next%2520generation%2520anticancer%2520agents%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D143%26spage%3D1277%26epage%3D1300%26doi%3D10.1016%2Fj.ejmech.2017.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Lera, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic polypharmacology: from combination therapy to multitargeted drugs</span>. <i>Clin. Epigenet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1186/s13148-016-0271-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1186%2Fs13148-016-0271-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=27752293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslOgsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=105-126&author=A.+R.+de%0ALeraauthor=A.+Ganesan&title=Epigenetic+polypharmacology%3A+from+combination+therapy+to+multitargeted+drugs&doi=10.1186%2Fs13148-016-0271-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic polypharmacology: from combination therapy to multitargeted drugs</span></div><div class="casAuthors">de Lera, Angel R.; Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Epigenetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105/1-105/21</span>CODEN:
                <span class="NLM_cas:coden">CELPCI</span>;
        ISSN:<span class="NLM_cas:issn">1868-7083</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  The modern drug discovery process has largely focused its attention in the so-called magic bullets, single chem. entities that exhibit high selectivity and potency for a particular target.  This approach was based on the assumption that the deregulation of a protein was causally linked to a disease state, and the pharmacol. intervention through inhibition of the deregulated target was able to restore normal cell function.  However, the use of cocktails or multicomponent drugs to address several targets simultaneously is also popular to treat multifactorial diseases such as cancer and neurol. disorders.  We review the state of the art with such combinations that have an epigenetic target as one of their mechanisms of action.  Epigenetic drug discovery is a rapidly advancing field, and drugs targeting epigenetic enzymes are in the clinic for the treatment of hematol. cancers.  Approved and exptl. epigenetic drugs are undergoing clin. trials in combination with other therapeutic agents via fused or linked pharmacophores in order to benefit from synergistic effects of polypharmacol.  In addn., ligands are being discovered which, as single chem. entities, are able to modulate multiple epigenetic targets simultaneously (multitarget epigenetic drugs).  These multiple ligands should in principle have a lower risk of drug-drug interactions and drug resistance compared to cocktails or multicomponent drugs.  This new generation may rival the so-called magic bullets in the treatment of diseases that arise as a consequence of the deregulation of multiple signaling pathways provided the challenge of optimization of the activities shown by the pharmacophores with the different targets is addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphm-KIEGyYKrVg90H21EOLACvtfcHk0ljZDk4MSJ_bmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslOgsbY%253D&md5=7cd89a93a22121cc9db2c7118e2e23f1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1186%2Fs13148-016-0271-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13148-016-0271-9%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BLera%26aufirst%3DA.%2BR.%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DEpigenetic%2520polypharmacology%253A%2520from%2520combination%2520therapy%2520to%2520multitargeted%2520drugs%26jtitle%3DClin.%2520Epigenet.%26date%3D2016%26volume%3D8%26spage%3D105%26epage%3D126%26doi%3D10.1186%2Fs13148-016-0271-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramadoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahadevan, V.</span></span> <span> </span><span class="NLM_article-title">Targeting the cancer epigenome: synergistic therapy with bromodomain inhibitors</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.drudis.2017.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=28943305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOgu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=76-89&author=M.+Ramadossauthor=V.+Mahadevan&title=Targeting+the+cancer+epigenome%3A+synergistic+therapy+with+bromodomain+inhibitors&doi=10.1016%2Fj.drudis.2017.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the cancer epigenome: synergistic therapy with bromodomain inhibitors</span></div><div class="casAuthors">Ramadoss, Mahalakshmi; Mahadevan, Vijayalakshmi</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-89</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Epigenetic and genomic alterations regulate the transcriptional landscape of cells during cancer onset and progression.  Recent clin. studies targeting the epigenetic 'readers' (bromodomains) for cancer therapy have established the effectiveness of bromodomain (BRD) and extraterminal (BET) inhibitors in treating several types of cancer.  In this review, we discuss key mechanisms of BET inhibition and synergistic combinations of BET inhibitors with histone deacetylase inhibitors (HDACi), histone methyltransferase inhibitors (HMTi), DNA methyltransferase inhibitors (DNMTi), kinase, B-cell lymphoma 2 (Bcl-2) and proteosome inhibitors, and immunomodulatory drugs for cancer therapy.  We also highlight the potential of such combinations to overcome drug resistance, and the evolving approaches to developing novel BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXLHGMo7cO6LVg90H21EOLACvtfcHk0limo6eseUecZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOgu7bJ&md5=490a32952a2b907f71a88c1e6292db77</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DRamadoss%26aufirst%3DM.%26aulast%3DMahadevan%26aufirst%3DV.%26atitle%3DTargeting%2520the%2520cancer%2520epigenome%253A%2520synergistic%2520therapy%2520with%2520bromodomain%2520inhibitors%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D76%26epage%3D89%26doi%3D10.1016%2Fj.drudis.2017.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donawho, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlhapp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Preclinical characterization of bet family bromodomain inhibitor abbv-075 suggests combination therapeutic strategies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2976</span>– <span class="NLM_lpage">2989</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-16-1793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1158%2F0008-5472.CAN-16-1793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=28416490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Siurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2976-2989&author=M.+H.+Buiauthor=X.+Linauthor=D.+H.+Albertauthor=L.+Liauthor=L.+T.+Lamauthor=E.+J.+Faivreauthor=S.+E.+Warderauthor=X.+Huangauthor=D.+Wilcoxauthor=C.+K.+Donawhoauthor=G.+S.+Sheppardauthor=L.+Wangauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+Hasvoldauthor=T.+Uzielauthor=X.+Luauthor=F.+Kohlhappauthor=G.+Fangauthor=S.+W.+Elmoreauthor=S.+H.+Rosenbergauthor=K.+F.+McDanielauthor=W.+M.+Katiauthor=Y.+Shen&title=Preclinical+characterization+of+bet+family+bromodomain+inhibitor+abbv-075+suggests+combination+therapeutic+strategies&doi=10.1158%2F0008-5472.can-16-1793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical Characterization of BET Family Bromodomain Inhibitor ABBV-075 Suggests Combination Therapeutic Strategies</span></div><div class="casAuthors">Bui, Mai H.; Lin, Xiaoyu; Albert, Daniel H.; Li, Leiming; Lam, Lloyd T.; Faivre, Emily J.; Warder, Scott E.; Huang, Xiaoli; Wilcox, Denise; Donawho, Cherrie K.; Sheppard, George S.; Wang, Le; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa; Uziel, Tamar; Lu, Xin; Kohlhapp, Fred; Fang, Guowei; Elmore, Steven W.; Rosenberg, Saul H.; McDaniel, Keith F.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2976-2989</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">ABBV-075 is a potent and selective BET family bromodomain inhibitor that recently entered phase I clin. trials.  Comprehensive preclin. characterization of ABBV-075 demonstrated broad activity across cell lines and tumor models, representing a variety of hematol. malignancies and solid tumor indications.  In most cancer cell lines derived from solid tumors, ABBV-075 triggers prominent G1 cell-cycle arrest without extensive apoptosis.  In this study, we show that ABBV-075 efficiently triggers apoptosis in acute myeloid leukemia (AML), non-Hodgkin lymphoma, and multiple myeloma cells.  Apoptosis induced by ABBV-075 was mediated in part by modulation of the intrinsic apoptotic pathway, exhibiting synergy with the BCL-2 inhibitor venetoclax in preclin. models of AML.  In germinal center diffuse large B-cell lymphoma, BCL-2 levels or venetoclax sensitivity predicted the apoptotic response to ABBV-075 treatment.  In vivo combination studies uncovered surprising benefits of low doses of ABBV-075 coupled with bortezomib and azacitidine treatment, despite the lack of in vitro synergy between ABBV-075 and these agents.  The in vitro/in vivo activities of ABBV-075 described here may serve as a useful ref. to guide the development of ABBV-075 and other BET family inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6P1blFwkIvrVg90H21EOLACvtfcHk0limo6eseUecZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Siurw%253D&md5=d9cc4abc44cfd9c0e07383f908592c2e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-1793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-1793%26sid%3Dliteratum%253Aachs%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DDonawho%26aufirst%3DC.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DKohlhapp%26aufirst%3DF.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DPreclinical%2520characterization%2520of%2520bet%2520family%2520bromodomain%2520inhibitor%2520abbv-075%2520suggests%2520combination%2520therapeutic%2520strategies%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2976%26epage%3D2989%26doi%3D10.1158%2F0008-5472.can-16-1793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Astorgues-Xerri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odore, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erba, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltrame, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frapolli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Incalci, M.</span></span> <span> </span><span class="NLM_article-title">The bromodomain inhibitor OTX015 (MK-8628) exerts antitumor activity in triple-negative breast cancer models as single agent and in combination with everolimus</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">7598</span>– <span class="NLM_lpage">7613</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.18632%2Foncotarget.13814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=27935867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252Fis1Cntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=7598-7613&author=R.+V%C3%A1zquezauthor=M.+E.+Riveiroauthor=L.+Astorgues-Xerriauthor=E.+Odoreauthor=K.+Rezaiauthor=E.+Erbaauthor=N.+Paniniauthor=A.+Rinaldiauthor=I.+Kweeauthor=L.+Beltrameauthor=M.+Bekraddaauthor=E.+Cvitkovicauthor=F.+Bertoniauthor=R.+Frapolliauthor=M.+D%E2%80%99Incalci&title=The+bromodomain+inhibitor+OTX015+%28MK-8628%29+exerts+antitumor+activity+in+triple-negative+breast+cancer+models+as+single+agent+and+in+combination+with+everolimus&doi=10.18632%2Foncotarget.13814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain inhibitor OTX015 (MK-8628) exerts anti-tumor activity in triple-negative breast cancer models as single agent and in combination with everolimus</span></div><div class="casAuthors">Vazquez Ramiro; Erba Eugenio; Panini Nicolo; Beltrame Luca; Frapolli Roberta; D'Incalci Maurizio; Riveiro Maria E; Astorgues-Xerri Lucile; Odore Elodie; Bekradda Mohamed; Cvitkovic Esteban; Odore Elodie; Rezai Keyvan; Rinaldi Andrea; Kwee Ivo; Bertoni Francesco; Kwee Ivo; Kwee Ivo; Cvitkovic Esteban; Bertoni Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">7598-7613</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Triple-negative breast cancer (TNBC) is an aggressive and heterogeneous subgroup of breast tumors clinically defined by the lack of estrogen, progesterone and HER2 receptors, limiting the use of the targeted therapies employed in other breast malignancies.  Recent evidence indicates that c-MYC is a key driver of TNBC.  The BET-bromodomain inhibitor OTX015 (MK-8628) has potent antiproliferative activity accompanied by c-MYC down-regulation in several tumor types, and has demonstrated synergism with the mTOR inhibitor everolimus in different models.  The aim of this study was to evaluate the anti-tumor activity of OTX015 as single agent and in combination with everolimus in TNBC models.  OTX015 was assayed in three human TNBC-derived cell lines, HCC1937, MDA-MB-231 and MDA-MB-468, all showing antiproliferative activity after 72 h (GI50 = 75-650 nM).  This was accompanied by cell cycle arrest and decreased expression of cancer stem cells markers.  However, c-MYC protein and mRNA levels were only down-regulated in MDA-MB-468 cells.  Gene set enrichment analysis showed up-regulation of genes involved in epigenetic control of transcription, chromatin and the cell cycle, and down-regulation of stemness-related genes.  In vitro, combination with everolimus was additive in HCC1937 and MDA-MB-231 cells, but antagonistic in MDA-MB-468 cells.  In MDA-MB-231 murine xenografts, tumor mass was significantly (p < 0.05) reduced by OTX015 with respect to vehicle-treated animals (best T/C = 40.7%).  Although everolimus alone was not active, the combination was more effective than OTX015 alone (best T/C = 20.7%).  This work supports current clinical trials with OTX015 in TNBC (NCT02259114).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSpC5NvWs-BQRJ5E29YNGvQfW6udTcc2eaQ--bl4E9WiLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252Fis1Cntw%253D%253D&md5=ac5f6998cb398ab9dbb9c9493880a5c7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13814%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A1zquez%26aufirst%3DR.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DAstorgues-Xerri%26aufirst%3DL.%26aulast%3DOdore%26aufirst%3DE.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DErba%26aufirst%3DE.%26aulast%3DPanini%26aufirst%3DN.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBeltrame%26aufirst%3DL.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26aulast%3DFrapolli%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Incalci%26aufirst%3DM.%26atitle%3DThe%2520bromodomain%2520inhibitor%2520OTX015%2520%2528MK-8628%2529%2520exerts%2520antitumor%2520activity%2520in%2520triple-negative%2520breast%2520cancer%2520models%2520as%2520single%2520agent%2520and%2520in%2520combination%2520with%2520everolimus%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D7598%26epage%3D7613%26doi%3D10.18632%2Foncotarget.13814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors in anticancer drug discovery: a structural and pharmacological perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9305</span>– <span class="NLM_lpage">9320</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00438</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00438" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9305-9320&author=L.+Carlinoauthor=G.+Rastelli&title=Dual+kinase-bromodomain+inhibitors+in+anticancer+drug+discovery%3A+a+structural+and+pharmacological+perspective&doi=10.1021%2Facs.jmedchem.6b00438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span></div><div class="casAuthors">Carlino, Luca; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9305-9320</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases play crucial roles in several cell transformation processes and are validated drug targets for many human diseases, including cancer.  Nevertheless, most tumors have eluded the effects of inhibition of a single kinase by activating resistance mechanisms and/or alternative pathways and escape mechanisms.  In recent years, multitarget approaches directed toward inhibition of kinases and targets of different families have received increasing attention.  In particular, co-targeting kinases and bromodomain epigenetic reader proteins has rapidly emerged as a promising approach to cancer drug development.  In this manuscript, we will review the recent discoveries that led to the identification and optimization of dual kinase/bromodomain inhibitors.  We will analyze and compare the structural features required for dual inhibition and comment on the potential of this approach in anticancer drug discovery.  Moreover, we will introduce computational approaches useful for the identification of dual kinase/bromodomain inhibitors and generate ad hoc pharmacophore and docking models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkHDSBWCGdbVg90H21EOLACvtfcHk0ljxNZTL8Zi9nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP&md5=1351a235bba09782160a86ae3f904833</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00438%26sid%3Dliteratum%253Aachs%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520in%2520anticancer%2520drug%2520discovery%253A%2520a%2520structural%2520and%2520pharmacological%2520perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9305%26epage%3D9320%26doi%3D10.1021%2Facs.jmedchem.6b00438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Acetyl-lysine binding site of bromodomain-containing protein 4 (BRD4) interacts with diverse kinase inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&author=S.+W.+J.+Emberauthor=J.-Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Sch%C3%B6nbrunn&title=Acetyl-lysine+binding+site+of+bromodomain-containing+protein+4+%28BRD4%29+interacts+with+diverse+kinase+inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span></div><div class="casAuthors">Ember, Stuart W. J.; Zhu, Jin-Yi; Olesen, Sanne H.; Martin, Mathew P.; Becker, Andreas; Berndt, Norbert; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1160-1171</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the bromodomain and extra terminal (BET) family of proteins are essential for the recognition of acetylated lysine (KAc) residues in histones and have emerged as promising drug targets in cancer, inflammation, and contraception research.  In co-crystn. screening campaigns using the first bromodomain of BRD4 (BRD4-1) against kinase inhibitor libraries, we identified and characterized 14 kinase inhibitors (10 distinct chem. scaffolds) as ligands of the KAc binding site.  Among these, the PLK1 inhibitor BI2536 and the JAK2 inhibitor TG101209 displayed strongest inhibitory potential against BRD4 (IC50 = 25 nM and 130 nM, resp.) and high selectivity for BET bromodomains.  Comparative structural anal. revealed markedly different binding modes of kinase hinge-binding scaffolds in the KAc binding site, suggesting that BET proteins are potential off-targets of diverse kinase inhibitors.  Combined, these findings provide a new structural framework for the rational design of next-generation BET-selective and dual-activity BET-kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0X1hjt7cBbVg90H21EOLACvtfcHk0ljxNZTL8Zi9nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D&md5=5ca37df00cdaa0ba81c0cb4a0f70ba5d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DAcetyl-lysine%2520binding%2520site%2520of%2520bromodomain-containing%2520protein%25204%2520%2528BRD4%2529%2520interacts%2520with%2520diverse%2520kinase%2520inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talluri, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topczewski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for the n-terminal bromodomain-and-extra-terminal-family selectivity of a dual kinase-bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9316</span>– <span class="NLM_lpage">9334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01248</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01248" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9316-9334&author=A.+Divakaranauthor=S.+K.+Talluriauthor=A.+M.+Ayoubauthor=N.+K.+Mishraauthor=H.+Cuiauthor=J.+C.+Widenauthor=N.+Berndtauthor=J.-Y.+Zhuauthor=A.+S.+Carlsonauthor=J.+J.+Topczewskiauthor=E.+K.+Schonbrunnauthor=D.+A.+Harkiauthor=W.+C.+K.+Pomerantz&title=Molecular+basis+for+the+n-terminal+bromodomain-and-extra-terminal-family+selectivity+of+a+dual+kinase-bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.8b01248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase-Bromodomain Inhibitor</span></div><div class="casAuthors">Divakaran, Anand; Talluri, Siva K.; Ayoub, Alex M.; Mishra, Neeraj K.; Cui, Huarui; Widen, John C.; Berndt, Norbert; Zhu, Jin-Yi; Carlson, Angela S.; Topczewski, Joseph J.; Schonbrunn, Ernst K.; Harki, Daniel A.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9316-9334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis.  When dysregulated, "reader" proteins become drivers of disease.  In the case of bromodomains, which recognize N-ε-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging.  We describe the >55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase-bromodomain inhibitors.  Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1).  Cellular efficacy was demonstrated via redn. of c-Myc expression, inhibition of NF-κB signaling, and suppression of IL-8 prodn. through potential synergistic inhibition of BRD4(1) and p38α.  These dual inhibitors provide a new scaffold for domain-selective inhibition of BRD4, the aberrant function of which plays a key role in cancer and inflammatory signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZhnL9cPdDi7Vg90H21EOLACvtfcHk0lhgCDOzkoO_lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM&md5=5176ef754a898b3e03fc56e2662ab399</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01248%26sid%3Dliteratum%253Aachs%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DTalluri%26aufirst%3DS.%2BK.%26aulast%3DAyoub%26aufirst%3DA.%2BM.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DCarlson%26aufirst%3DA.%2BS.%26aulast%3DTopczewski%26aufirst%3DJ.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%2BK.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DMolecular%2520basis%2520for%2520the%2520n-terminal%2520bromodomain-and-extra-terminal-family%2520selectivity%2520of%2520a%2520dual%2520kinase-bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9316%26epage%3D9334%26doi%3D10.1021%2Facs.jmedchem.8b01248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erazo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muñoz-Guardiola, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diéguez-Martínez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massefski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offei-Addo, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiBona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becht, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lizcano, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacklow, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Structural and atropisomeric factors governing the selectivity of pyrimido-benzodiazipinones as inhibitors of kinases and bromodomains</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2438</span>– <span class="NLM_lpage">2448</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00638</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00638" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVylsrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2438-2448&author=J.+Wangauthor=T.+Erazoauthor=F.+M.+Fergusonauthor=D.+L.+Buckleyauthor=N.+Gomezauthor=P.+Mu%C3%B1oz-Guardiolaauthor=N.+Di%C3%A9guez-Mart%C3%ADnezauthor=X.+Dengauthor=M.+Haoauthor=W.+Massefskiauthor=O.+Fedorovauthor=N.+K.+Offei-Addoauthor=P.+M.+Parkauthor=L.+Daiauthor=A.+DiBonaauthor=K.+Bechtauthor=N.+D.+Kimauthor=M.+R.+McKeownauthor=J.+M.+Robertsauthor=J.+Zhangauthor=T.+Simauthor=D.+R.+Alessiauthor=J.+E.+Bradnerauthor=J.+M.+Lizcanoauthor=S.+C.+Blacklowauthor=J.+Qiauthor=X.+Xuauthor=N.+S.+Gray&title=Structural+and+atropisomeric+factors+governing+the+selectivity+of+pyrimido-benzodiazipinones+as+inhibitors+of+kinases+and+bromodomains&doi=10.1021%2Facschembio.7b00638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Atropisomeric Factors Governing the Selectivity of Pyrimido-benzodiazipinones as Inhibitors of Kinases and Bromodomains</span></div><div class="casAuthors">Wang, Jinhua; Erazo, Tatiana; Ferguson, Fleur M.; Buckley, Dennis L.; Gomez, Nestor; Munoz-Guardiola, Pau; Dieguez-Martinez, Nora; Deng, Xianming; Hao, Mingfeng; Massefski, Walter; Fedorov, Oleg; Offei-Addo, Nana Kwaku; Park, Paul M.; Dai, Lingling; DiBona, Amy; Becht, Kelly; Kim, Nam Doo; McKeown, Michael R.; Roberts, Justin M.; Zhang, Jinwei; Sim, Taebo; Alessi, Dario R.; Bradner, James E.; Lizcano, Jose M.; Blacklow, Stephen C.; Qi, Jun; Xu, Xiang; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2438-2448</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains have been pursued intensively over the past several years as emerging targets for the development of anticancer and anti-inflammatory agents.  It has recently been shown that some kinase inhibitors are able to potently inhibit the bromodomains of BRD4.  The clin. activities of PLK inhibitor BI-2536 and JAK2-FLT3 inhibitor TG101348 have been attributed to this unexpected polypharmacol., indicating that dual-kinase/bromodomain activity may be advantageous in a therapeutic context.  However, for target validation and biol. investigation, a more selective target profile is desired.  Here, we report that benzo[e]pyrimido-[5,4-b]diazepine-6(11H)-ones, versatile ATP-site directed kinase pharmacophores utilized in the development of inhibitors of multiple kinases, including several previously reported kinase chem. probes, are also capable of exhibiting potent BRD4-dependent pharmacol.  Using a dual kinase-bromodomain inhibitor of the kinase domains of ERK5 and LRRK2, and the bromodomain of BRD4 as a case study, we define the structure-activity relationships required to achieve dual kinase/BRD4 activity, as well as how to direct selectivity toward inhibition of either ERK5 or BRD4.  This effort resulted in identification of one of the first reported kinase-selective chem. probes for ERK5 (JWG-071), a BET selective inhibitor with 1 μM BRD4 IC50 (JWG-115), and addnl. inhibitors with rationally designed polypharmacol. (JWG-047, JWG-069).  Co-crystallog. of seven representative inhibitors with the first bromodomain of BRD4 demonstrate that distinct atropisomeric conformers recognize the kinase ATP-site and the BRD4 acetyl lysine binding site, conformational preferences supported by rigid docking studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcdvUrcfGdq7Vg90H21EOLACvtfcHk0lhgCDOzkoO_lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVylsrvN&md5=bebe87da2f2808ad5aec5b3098304425</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00638%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DErazo%26aufirst%3DT.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGomez%26aufirst%3DN.%26aulast%3DMu%25C3%25B1oz-Guardiola%26aufirst%3DP.%26aulast%3DDi%25C3%25A9guez-Mart%25C3%25ADnez%26aufirst%3DN.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DHao%26aufirst%3DM.%26aulast%3DMassefski%26aufirst%3DW.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DOffei-Addo%26aufirst%3DN.%2BK.%26aulast%3DPark%26aufirst%3DP.%2BM.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DDiBona%26aufirst%3DA.%26aulast%3DBecht%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLizcano%26aufirst%3DJ.%2BM.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DStructural%2520and%2520atropisomeric%2520factors%2520governing%2520the%2520selectivity%2520of%2520pyrimido-benzodiazipinones%2520as%2520inhibitors%2520of%2520kinases%2520and%2520bromodomains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D2438%26epage%3D2448%26doi%3D10.1021%2Facschembio.7b00638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watts, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strebhardt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellenie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Designing dual inhibitors of anaplastic lymphoma kinase (alk) and bromodomain-4 (brd4) by tuning kinase selectivity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2618</span>– <span class="NLM_lpage">2637</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01947</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01947" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2618-2637&author=E.+Wattsauthor=D.+Heidenreichauthor=E.+Tuckerauthor=M.+Raabauthor=K.+Strebhardtauthor=L.+Cheslerauthor=S.+Knappauthor=B.+Bellenieauthor=S.+Hoelder&title=Designing+dual+inhibitors+of+anaplastic+lymphoma+kinase+%28alk%29+and+bromodomain-4+%28brd4%29+by+tuning+kinase+selectivity&doi=10.1021%2Facs.jmedchem.8b01947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Designing Dual Inhibitors of Anaplastic Lymphoma Kinase (ALK) and Bromodomain-4 (BRD4) by Tuning Kinase Selectivity</span></div><div class="casAuthors">Watts, Ellen; Heidenreich, David; Tucker, Elizabeth; Raab, Monika; Strebhardt, Klaus; Chesler, Louis; Knapp, Stefan; Bellenie, Benjamin; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2618-2637</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Concomitant inhibition of anaplastic lymphoma kinase (ALK) and bromodomain-4 (BRD4) is a potential therapeutic strategy for targeting two key oncogenic drivers that co-segregate in a significant fraction of high-risk neuroblastoma patients, mutation of ALK and amplification of MYCN.  Starting from known dual polo-like kinase (PLK)-1-BRD4 inhibitor BI-2536, we employed structure-based design to redesign this series toward compds. with a dual ALK-BRD4 profile.  These efforts led to compd. (R)-2-((2-ethoxy-4-(1-methylpiperidin-4-yl)phenyl)amino)-7-ethyl-5-methyl-8-((4-methylthiophen-2-yl)methyl)-7,8-dihydropteridin-6(5H)-one (16k) demonstrating improved ALK activity and significantly reduced PLK-1 activity, while maintaining BRD4 activity and overall kinome selectivity.  We demonstrate the compds.' on-target engagement with ALK and BRD4 in cells as well as favorable broad kinase and bromodomain selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop6qx6PSj2KLVg90H21EOLACvtfcHk0lhgCDOzkoO_lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D&md5=e00fdf8eb20acc2ef1c8777b2afb750d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01947%26sid%3Dliteratum%253Aachs%26aulast%3DWatts%26aufirst%3DE.%26aulast%3DHeidenreich%26aufirst%3DD.%26aulast%3DTucker%26aufirst%3DE.%26aulast%3DRaab%26aufirst%3DM.%26aulast%3DStrebhardt%26aufirst%3DK.%26aulast%3DChesler%26aufirst%3DL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBellenie%26aufirst%3DB.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDesigning%2520dual%2520inhibitors%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528alk%2529%2520and%2520bromodomain-4%2520%2528brd4%2529%2520by%2520tuning%2520kinase%2520selectivity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2618%26epage%3D2637%26doi%3D10.1021%2Facs.jmedchem.8b01947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshioka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fountain, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheenstra, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strovel, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span> <span> </span><span class="NLM_article-title">BRD4 structure-activity relationships of dual plk1 kinase/brd4 bromodomain inhibitor bi-2536</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">764</span>– <span class="NLM_lpage">769</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00084</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00084" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=764-769&author=L.+Chenauthor=J.+L.+Yapauthor=M.+Yoshiokaauthor=M.+E.+Lanningauthor=R.+N.+Fountainauthor=M.+Rajeauthor=J.+A.+Scheenstraauthor=J.+W.+Strovelauthor=S.+Fletcher&title=BRD4+structure-activity+relationships+of+dual+plk1+kinase%2Fbrd4+bromodomain+inhibitor+bi-2536&doi=10.1021%2Facsmedchemlett.5b00084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Structure-Activity Relationships of Dual PLK1 Kinase/BRD4 Bromodomain Inhibitor BI-2536</span></div><div class="casAuthors">Chen, Lijia; Yap, Jeremy L.; Yoshioka, Makoto; Lanning, Maryanna E.; Fountain, Rachel N.; Raje, Mithun; Scheenstra, Jacob A.; Strovel, Jeffrey W.; Fletcher, Steven</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">764-769</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused library of analogs of the dual PLK1 kinase/BRD4 bromodomain inhibitor BI-2536 was prepd. and then analyzed for BRD4 and PLK1 inhibitory activities.  Particularly, replacement of the cyclopentyl group with a 3-bromobenzyl moiety afforded the most potent BRD4 inhibitor of the series (39j) with a Ki = 8.7 nM, which was equipotent against PLK1.  The superior affinity of 39j over the parental compd. to BRD4 possibly derives from improved interactions with the WPF shelf.  Meanwhile, substitution of the pyrimidine NH with an oxygen atom reversed the PLK1/BRD4 selectivity to convert BI-2536 into a BRD4-selective inhibitor, likely owing to the loss of a crit. hydrogen bond in PLK1.  We believe further fine-tuning will furnish a BRD4 "magic bullet" or an even more potent PLK1/BRD4 dual inhibitor toward the expansion and improved efficacy of the chemotherapy arsenal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5UnNSmf1L17Vg90H21EOLACvtfcHk0lgo5vTckcMbVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D&md5=f39070e652bca20c03a3bc5fe070a7fc</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00084%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYap%26aufirst%3DJ.%2BL.%26aulast%3DYoshioka%26aufirst%3DM.%26aulast%3DLanning%26aufirst%3DM.%2BE.%26aulast%3DFountain%26aufirst%3DR.%2BN.%26aulast%3DRaje%26aufirst%3DM.%26aulast%3DScheenstra%26aufirst%3DJ.%2BA.%26aulast%3DStrovel%26aufirst%3DJ.%2BW.%26aulast%3DFletcher%26aufirst%3DS.%26atitle%3DBRD4%2520structure-activity%2520relationships%2520of%2520dual%2520plk1%2520kinase%252Fbrd4%2520bromodomain%2520inhibitor%2520bi-2536%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D764%26epage%3D769%26doi%3D10.1021%2Facsmedchemlett.5b00084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0lgo5vTckcMbVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klug-Mcleod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunney, E. A.</span></span> <span> </span><span class="NLM_article-title">Kinase hinge binding scaffolds and their hydrogen bond patterns</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6520</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.bmc.2015.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=26358279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVentr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=6520-6527&author=L.+Xingauthor=J.+Klug-Mcleodauthor=B.+Raiauthor=E.+A.+Lunney&title=Kinase+hinge+binding+scaffolds+and+their+hydrogen+bond+patterns&doi=10.1016%2Fj.bmc.2015.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase hinge binding scaffolds and their hydrogen bond patterns</span></div><div class="casAuthors">Xing, Li; Klug-Mcleod, Jacquelyn; Rai, Brajesh; Lunney, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6520-6527</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein kinases constitute a major class of intracellular signaling mols., and describe some of the most prominent drug targets.  Kinase inhibitors commonly employ small chem. scaffolds that form hydrogen bonds with the kinase hinge residues connecting the N- and C-terminal lobes of the catalytic domain.  In general the satisfied hydrogen bonds are required for potent inhibition, therefore constituting a conserved feature in the majority of inhibitor-kinase interactions.  From systematically analyzing the kinase scaffolds extd. from Pfizer crystal structure database (CSDb) the authors recognize that large no. of kinase inhibitors of diverse chem. structures are derived from a relatively small no. of common scaffolds.  Depending on specific substitution patterns, scaffolds may demonstrate versatile binding capacities to interact with kinase hinge.  Afforded by thousands of ligand-protein binary complexes, the hinge hydrogen bond patterns were analyzed with a focus on their three-dimensional configurations.  Most of the compds. engage H6 NH for hinge recognition.  Dual hydrogen bonds are commonly obsd. with addnl. recruitment of H4 CO upstream and/or H6 CO downstream.  Triple hydrogen bonds accounts for small no. of binary complexes.  An unusual hydrogen bond with a non-canonical H5 conformation is obsd., requiring a peptide bond flip by a glycine residue at the H6 position.  Addnl. hydrogen bonds to kinase hinge do not necessarily correlate with an increase in potency; conversely they appear to compromise kinase selectivity.  Such learnings could enhance the prospect of successful therapy design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRC-xiTiNvELVg90H21EOLACvtfcHk0liHMon-M4e2Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVentr3L&md5=048b4b99628c6e1ae273b00a47ab2377</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DXing%26aufirst%3DL.%26aulast%3DKlug-Mcleod%26aufirst%3DJ.%26aulast%3DRai%26aufirst%3DB.%26aulast%3DLunney%26aufirst%3DE.%2BA.%26atitle%3DKinase%2520hinge%2520binding%2520scaffolds%2520and%2520their%2520hydrogen%2520bond%2520patterns%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D6520%26epage%3D6527%26doi%3D10.1016%2Fj.bmc.2015.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosief, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhawan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthengi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and development of potent and selective dual bromodomain 4 (brd4)/polo-like kinase 1 (plk1) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7785</span>– <span class="NLM_lpage">7795</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00765</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00765" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7785-7795&author=S.+Liuauthor=H.+O.+Yosiefauthor=L.+Daiauthor=H.+Huangauthor=G.+Dhawanauthor=X.+Zhangauthor=A.+M.+Muthengiauthor=J.+Robertsauthor=D.+L.+Buckleyauthor=J.+A.+Perryauthor=L.+Wuauthor=J.+E.+Bradnerauthor=J.+Qiauthor=W.+Zhang&title=Structure-guided+design+and+development+of+potent+and+selective+dual+bromodomain+4+%28brd4%29%2Fpolo-like+kinase+1+%28plk1%29+inhibitors&doi=10.1021%2Facs.jmedchem.8b00765"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors</span></div><div class="casAuthors">Liu, Shuai; Yosief, Hailemichael O.; Dai, Lingling; Huang, He; Dhawan, Gagan; Zhang, Xiaofeng; Muthengi, Alex M.; Roberts, Justin; Buckley, Dennis L.; Perry, Jennifer A.; Wu, Lei; Bradner, James E.; Qi, Jun; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7785-7795</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single mol. could lead to the development of an effective therapeutic strategy for a variety of diseases in which PLK1 and BRD4 are implicated.  Compd. 23 has been found to be a potent dual kinase-bromodomain inhibitor (BRD4-BD1 IC50 = 28 nM, PLK1 IC50 = 40 nM).  Compd. 6 was found to be the most selective PLK1 inhibitor over BRD4 in our series (BRD4-BD1 IC50 = 2579 nM, PLK1 IC50 = 9.9 nM).  Mol. docking studies with 23 and BRD4-BD1/PLK1 as well as with 6 corroborate the biochem. assay results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUL2G-DnFzU7Vg90H21EOLACvtfcHk0liHMon-M4e2Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L&md5=e34a58694f0827d33d31516200e2c0d8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00765%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYosief%26aufirst%3DH.%2BO.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DDhawan%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMuthengi%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DJ.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DStructure-guided%2520design%2520and%2520development%2520of%2520potent%2520and%2520selective%2520dual%2520bromodomain%25204%2520%2528brd4%2529%252Fpolo-like%2520kinase%25201%2520%2528plk1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7785%26epage%3D7795%26doi%3D10.1021%2Facs.jmedchem.8b00765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olcina, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Optimization of 3,5-dimethylisoxazole derivatives as potent bromodomain ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3217</span>– <span class="NLM_lpage">3227</span>, <span class="refDoi"> DOI: 10.1021/jm301588r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301588r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlekuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3217-3227&author=D.+S.+Hewingsauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=S.+Martinauthor=S.+Picaudauthor=A.+Tumberauthor=C.+Wellsauthor=M.+M.+Olcinaauthor=K.+Freemanauthor=A.+Gillauthor=A.+J.+Ritchieauthor=D.+W.+Sheppardauthor=A.+J.+Russellauthor=E.+M.+Hammondauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=Optimization+of+3%2C5-dimethylisoxazole+derivatives+as+potent+bromodomain+ligands&doi=10.1021%2Fjm301588r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 3,5-Dimethylisoxazole Derivatives as Potent Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Fedorov, Oleg; Filippakopoulos, Panagis; Martin, Sarah; Picaud, Sarah; Tumber, Anthony; Wells, Christopher; Olcina, Monica M.; Freeman, Katherine; Gill, Andrew; Ritchie, Alison J.; Sheppard, David W.; Russell, Angela J.; Hammond, Ester M.; Knapp, Stefan; Brennan, Paul E.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3217-3227</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">(Isoxazolylphenyl)arylmethanols such as I (R = H; R1 = HO; R2 = H; R3 = Ph), I (R = HO; R1 = H; R2 = H; R3 = Ph), and I (R = H, HO; R1 = HO, H; R2 = Et; R3 = Ph, 3-FC6H4, 4-FC6H4, 3-ClC6H4, 4-ClC6H4) were prepd. as inhibitors of the bromodomain and extra terminal domain (BET) bromodomain family member BRD4(1) for potential use as antitumor agents.  The structures of I (R = H; R1 = HO; R2 = H; R3 = Ph) and I (R = HO; R1 = H; R2 = H; R3 = Ph) bound to human BRD4(1) were detd. by X-ray crystallog. to det. features useful for selective binding to BRD4(1).  I (R = H, HO; R1 = HO, H; R2 = H, MeCO; R3 = Ph) and I (R = H, HO; R1 = HO, H; R2 = Et; R3 = 3-ClC6H4) inhibited a human acute myeloid leukemia cell line in vitro with 3-4 fold reduced inhibition and reduced toxicity in other human cells compared to a known BRD4(1) inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVEQ5OOJGC8bVg90H21EOLACvtfcHk0li_H2obwm-QZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlekuro%253D&md5=bb4890291db8c3e8d404a75dd9dcbb87</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm301588r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301588r%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DOlcina%26aufirst%3DM.%2BM.%26aulast%3DFreeman%26aufirst%3DK.%26aulast%3DGill%26aufirst%3DA.%26aulast%3DRitchie%26aufirst%3DA.%2BJ.%26aulast%3DSheppard%26aufirst%3DD.%2BW.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DHammond%26aufirst%3DE.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DOptimization%2520of%25203%252C5-dimethylisoxazole%2520derivatives%2520as%2520potent%2520bromodomain%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3217%26epage%3D3227%26doi%3D10.1021%2Fjm301588r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttarkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuppusetty, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">3,5-dimethylisoxazoles act as acetyl-lysine-mimetic bromodomain ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6761</span>– <span class="NLM_lpage">6770</span>, <span class="refDoi"> DOI: 10.1021/jm200640v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200640v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6761-6770&author=D.+S.+Hewingsauthor=M.+Wangauthor=M.+Philpottauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=C.+Vuppusettyauthor=B.+Marsdenauthor=S.+Knappauthor=S.+J.+Conwayauthor=T.+D.+Heightman&title=3%2C5-dimethylisoxazoles+act+as+acetyl-lysine-mimetic+bromodomain+ligands&doi=10.1021%2Fjm200640v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Dimethylisoxazoles Act As Acetyl-lysine-mimetic Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Wang, Minghua; Philpott, Martin; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Vuppusetty, Chaitanya; Marsden, Brian; Knapp, Stefan; Conway, Stuart J.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6761-6770</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone-lysine acetylation is a vital chromatin post-translational modification involved in the epigenetic regulation of gene transcription.  Bromodomains bind acetylated lysines, acting as readers of the histone-acetylation code.  Competitive inhibitors of this interaction have antiproliferative and anti-inflammatory properties.  With 57 distinct bromodomains known, the discovery of subtype-selective inhibitors of the histone-bromodomain interaction is of great importance.  We have identified the 3,5-dimethylisoxazole moiety as a novel acetyl-lysine bioisostere, which displaces acetylated histone-mimicking peptides from bromodomains.  Using X-ray crystallog. anal., we have detd. the interactions responsible for the activity and selectivity of 4-substituted 3,5-dimethylisoxazoles against a selection of phylogenetically diverse bromodomains.  By exploiting these interactions, we have developed compd. 4d, which has IC50 values of <5 μM for the bromodomain-contg. proteins BRD2(1) and BRD4(1).  These compds. are promising leads for the further development of selective probes for the bromodomain and extra C-terminal domain (BET) family and CREBBP bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEzK4xMvJhXrVg90H21EOLACvtfcHk0li_H2obwm-QZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO&md5=7d6f9eff2735c8a7189dec4fb5c5d13d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm200640v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200640v%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DVuppusetty%26aufirst%3DC.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3D3%252C5-dimethylisoxazoles%2520act%2520as%2520acetyl-lysine-mimetic%2520bromodomain%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6761%26epage%3D6770%26doi%3D10.1021%2Fjm200640v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, C.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damaneh, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huan, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery and development of a series of potent and selective bromodomain and extra-terminal protein inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8642</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01094</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01094" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslegsbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8642-8663&author=J.+Huauthor=C.-Q.+Tianauthor=M.+S.+Damanehauthor=Y.+Liauthor=D.+Caoauthor=K.+Lvauthor=T.+Yuauthor=T.+Mengauthor=D.+Chenauthor=X.+Wangauthor=L.+Chenauthor=J.+Liauthor=S.-S.+Songauthor=X.-J.+Huanauthor=L.+Qinauthor=J.+Shenauthor=Y.-Q.+Wangauthor=Z.-H.+Miaoauthor=B.+Xiong&title=Structure-based+discovery+and+development+of+a+series+of+potent+and+selective+bromodomain+and+extra-terminal+protein+inhibitors&doi=10.1021%2Facs.jmedchem.9b01094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery and Development of a Series of Potent and Selective Bromodomain and Extra-Terminal Protein Inhibitors</span></div><div class="casAuthors">Hu, Jianping; Tian, Chang-Qing; Damaneh, Mohammadali Soleimani; Li, Yanlian; Cao, Danyan; Lv, Kaikai; Yu, Ting; Meng, Tao; Chen, Danqi; Wang, Xin; Chen, Lin; Li, Jian; Song, Shan-Shan; Huan, Xia-Juan; Qin, Lihuai; Shen, Jingkang; Wang, Ying-Qing; Miao, Ze-Hong; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8642-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4 has recently emerged as a promising drug target.  Therefore, identifying novel inhibitors with distinct properties could enrich their use in anticancer treatment.  Guided by the cocrystal structure of hit compd. 4 harboring a five-membered-ring linker motif, we quickly identified lead compd. 7, which exhibited good antitumor effects in an MM.1S xenograft model by oral administration.  Encouraged by its high potency and interesting scaffold, we performed further lead optimization to generate a novel potent series of bromodomain and extra-terminal (BET) inhibitors with a (1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1H)-one structure.  Among them, compd. 19 was found to have the best balance of activity, stability, and antitumor efficacy.  After confirming its low brain penetration, we conducted comprehensive preclin. studies, including a multiple-species pharmacokinetics profile, extensive cellular mechanism studies, hERG assay, and in vivo antitumor growth effect testing, and we found that compd. 19 is a potential BET protein drug candidate for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGormQku3ZAy97Vg90H21EOLACvtfcHk0li_H2obwm-QZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslegsbvK&md5=af9fa23078f8280cc36d3c097d6fbf34</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01094%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DC.-Q.%26aulast%3DDamaneh%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DLv%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DT.%26aulast%3DMeng%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DS.-S.%26aulast%3DHuan%26aufirst%3DX.-J.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.-Q.%26aulast%3DMiao%26aufirst%3DZ.-H.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DStructure-based%2520discovery%2520and%2520development%2520of%2520a%2520series%2520of%2520potent%2520and%2520selective%2520bromodomain%2520and%2520extra-terminal%2520protein%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8642%26epage%3D8663%26doi%3D10.1021%2Facs.jmedchem.9b01094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damaneh, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Structure-based optimization of a series of selective BET inhibitors containing aniline or indoline groups</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.02.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.ejmech.2018.02.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=29525435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktlylsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2018&pages=156-175&author=J.+Huauthor=Y.+Wangauthor=Y.+Liauthor=D.+Caoauthor=L.+Xuauthor=S.+Songauthor=M.+S.+Damanehauthor=J.+Liauthor=Y.+Chenauthor=X.+Wangauthor=L.+Chenauthor=J.+Shenauthor=Z.+Miaoauthor=B.+Xiong&title=Structure-based+optimization+of+a+series+of+selective+BET+inhibitors+containing+aniline+or+indoline+groups&doi=10.1016%2Fj.ejmech.2018.02.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based optimization of a series of selective BET inhibitors containing aniline or indoline groups</span></div><div class="casAuthors">Hu, Jianping; Wang, Yingqing; Li, Yanlian; Cao, Danyan; Xu, Lin; Song, ShanShan; Damaneh, Mohammadali Soleimani; Li, Jian; Chen, Yuelei; Wang, Xin; Chen, Lin; Shen, Jingkang; Miao, Zehong; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">156-175</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recently, several kinase inhibitors were found to also inhibit bromodomains, providing a new strategy for the discovery of bromodomain inhibitors.  Along this line, starting from PLK1-BRD4 dual inhibitor BI-2536, we discovered a new series of dihydroquinoxalin-2(1H)-one with aniline and indoline WPF binders as selective BRD4 inhibitors.  They showed better BRD4-BD1 potency and negligible PLK1 kinase activity comparing with BI-2536.  Addnl., dihydroquinoxalin-2(1H)-ones contg. indoline group showed profound activities in mol. and cellular based assays.  Throughout the study, compds. 9, 28 and 37 showed significant inhibitory activity for c-Myc or PD-L1 protein expression and mRNA transcription both at concn. of 0.2 and 1 μM.  Compd. 9 was found possessing the best balance of binding affinity, in vitro metabolic stability and in vivo pharmacokinetic properties.  Therefore, it was selected for in vivo pharmacol. study.  By using MM.1S cell derived xenograft model, we confirmed compd. 9 showed comparable in vivo tumor inhibition to phase II investigation drug I-BET762, which, together with the novel WPF binder, further indicated the utility of this series of BRD4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovKjmtiqOa9rVg90H21EOLACvtfcHk0li36nsPDz_4Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktlylsLw%253D&md5=5d3ad7e741a140306d42ae2463dd667e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.02.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.02.070%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DDamaneh%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DStructure-based%2520optimization%2520of%2520a%2520series%2520of%2520selective%2520BET%2520inhibitors%2520containing%2520aniline%2520or%2520indoline%2520groups%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D150%26spage%3D156%26epage%3D175%26doi%3D10.1016%2Fj.ejmech.2018.02.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damaneh, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a series of dihydroquinoxalin-2(1H)-ones as selective BET inhibitors from a dual PLK1-BRD4 inhibitor</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">176</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.05.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.ejmech.2017.05.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=28586718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWmt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2017&pages=176-195&author=J.+Huauthor=Y.+Wangauthor=Y.+Liauthor=L.+Xuauthor=D.+Caoauthor=S.+Songauthor=M.+S.+Damanehauthor=X.+Wangauthor=T.+Mengauthor=Y.-L.+Chenauthor=J.+Shenauthor=Z.+Miaoauthor=B.+Xiong&title=Discovery+of+a+series+of+dihydroquinoxalin-2%281H%29-ones+as+selective+BET+inhibitors+from+a+dual+PLK1-BRD4+inhibitor&doi=10.1016%2Fj.ejmech.2017.05.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a series of dihydroquinoxalin-2(1H)-ones as selective BET inhibitors from a dual PLK1-BRD4 inhibitor</span></div><div class="casAuthors">Hu, Jianping; Wang, Yingqing; Li, Yanlian; Xu, Lin; Cao, Danyan; Song, Shan Shan; Damaneh, Mohammadali Soleimani; Wang, Xin; Meng, Tao; Chen, Yue-Lei; Shen, Jingkang; Miao, Zehong; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">176-195</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recent years have seen much effort to discover new chemotypes of BRD4 inhibitors.  Some kinase inhibitors are potent bromodomain inhibitors, esp. the PLK1 inhibitor BI-2536 and the JAK2 inhibitor TG101209, which can bind to BRD4 with IC50 values of 0.025 μM and 0.13 μM, resp.  Although the concept of dual inhibition is intriguing, selective BRD4 inhibitors are preferred as they may diminish off-target effects and provide more flexibility in anticancer drug combination therapy.  Inspired by BI-2536, the authors designed and prepd. a series of dihydroquinoxalin-2(1H)-one derivs. as selective bromodomain inhibitors.  The authors found compd. (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one had slightly higher activity than (+)-JQ1 in the fluorescence anisotropy assay and potent antiproliferative cellular activity in the MM.1S cell line.  The authors have successfully solved the cocrystal structure of (R)-4-cyclopropyl-1,3-dimethyl-6-(((S)-1-(p-tolyl)ethyl) amino)-3,4-dihydroquinoxalin-2(1H)-one in complex with BRD4-BD1, providing a solid structural basis for the binding mode of compds. of this series.  Compd. (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one exhibited high selectivity over most non-BET subfamily members and did not show bioactivity towards the PLK1 kinase at 10 or 1 μM.  From in vivo studies, compd. (R)-4-Cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one demonstrated a good PK profile, and the results from in vivo pharmacol. studies clearly showed the efficacy of (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one in the mouse MM.1S xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8GQjGV-VP7LVg90H21EOLACvtfcHk0li36nsPDz_4Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWmt73I&md5=af59e5bd57adeede512b022e3641f4f2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.05.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.05.049%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DDamaneh%26aufirst%3DM.%2BS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMeng%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.-L.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520series%2520of%2520dihydroquinoxalin-2%25281H%2529-ones%2520as%2520selective%2520BET%2520inhibitors%2520from%2520a%2520dual%2520PLK1-BRD4%2520inhibitor%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D137%26spage%3D176%26epage%3D195%26doi%3D10.1016%2Fj.ejmech.2017.05.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4927</span>– <span class="NLM_lpage">4939</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4927-4939&author=X.+Ranauthor=Y.+Zhaoauthor=L.+Liuauthor=L.+Baiauthor=C.-Y.+Yangauthor=B.+Zhouauthor=J.+L.+Meagherauthor=K.+Chinnaswamyauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-Based+Design+of+%CE%B3-Carboline+Analogues+as+Potent+and+Specific+BET+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span></div><div class="casAuthors">Ran, Xu; Zhao, Yujun; Liu, Liu; Bai, Longchuan; Yang, Chao-Yie; Zhou, Bing; Meagher, Jennifer L.; Chinnaswamy, Krishnapriya; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4927-4939</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors of bromodomain and extra terminal proteins (BET), including BRD2, BRD3, and BRD4 proteins have therapeutic potential for the treatment of human cancers and other diseases and conditions.  In this paper, the authors report the design, synthesis, and evaluation of γ-carboline-contg. compds. as a new class of small-mol. BET inhibitors.  The most potent inhibitor I (RX-37) obtained from this study binds to BET bromodomain proteins (BRD2, BRD3, and BRD4) with Ki values of 3.2-24.7 nM and demonstrates high selectivity over other non-BET bromodomain-contg. proteins.  Compd. I potently and selectively inhibits cell growth in human acute leukemia cell lines harboring the rearranged mixed lineage leukemia 1 gene.  The authors have detd. a cocrystal structure of I in complex with BRD4 BD2 at 1.4 Å resoln., which provides a solid structural basis for the compd.'s high binding affinity and for its further structure-based optimization.  Compd. I represents a promising lead compd. for the development of a new class of therapeutics for the treatment of human cancer and other conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv-VCfWLBWYLVg90H21EOLACvtfcHk0lh7PZe_jts5sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM&md5=e6e150169812e439c1d7d91416eb3ee9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00613%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-Based%2520Design%2520of%2520%25CE%25B3-Carboline%2520Analogues%2520as%2520Potent%2520and%2520Specific%2520BET%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4927%26epage%3D4939%26doi%3D10.1021%2Facs.jmedchem.5b00613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singleton, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of small-molecule ligands for the CBP/p300 bromodomains</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">9308</span>– <span class="NLM_lpage">9319</span>, <span class="refDoi"> DOI: 10.1021/ja412434f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja412434f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=9308-9319&author=D.+A.+Hayauthor=O.+Fedorovauthor=S.+Martinauthor=D.+C.+Singletonauthor=C.+Tallantauthor=C.+Wellsauthor=S.+Picaudauthor=M.+Philpottauthor=O.+P.+Monteiroauthor=C.+M.+Rogersauthor=S.+J.+Conwayauthor=T.+P.+C.+Rooneyauthor=A.+Tumberauthor=C.+Yappauthor=P.+Filippakopoulosauthor=M.+E.+Bunnageauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Discovery+and+optimization+of+small-molecule+ligands+for+the+CBP%2Fp300+bromodomains&doi=10.1021%2Fja412434f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Small-Molecule Ligands for the CBP/p300 Bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Fedorov, Oleg; Martin, Sarah; Singleton, Dean C.; Tallant, Cynthia; Wells, Christopher; Picaud, Sarah; Philpott, Martin; Monteiro, Octovia P.; Rogers, Catherine M.; Conway, Stuart J.; Rooney, Timothy P. C.; Tumber, Anthony; Yapp, Clarence; Filippakopoulos, Panagis; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">9308-9319</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors that target bromodomains outside of the bromodomain and extra-terminal (BET) sub-family are lacking.  Here, the authors describe highly potent and selective ligands for the bromodomain module of the human lysine acetyl transferase CBP/p300, developed from a series of 5-isoxazolyl-benzimidazoles.  The starting point was a fragment hit, which was optimized into a more potent and selective lead using parallel synthesis employing Suzuki couplings, benzimidazole-forming reactions, and reductive aminations.  The selectivity of the lead compd. against other bromodomain family members was investigated using a thermal stability assay, which revealed some inhibition of the structurally related BET family members.  To address the BET selectivity issue, x-ray crystal structures of the lead compd. bound to the CREB binding protein (CBP) and the first bromodomain of BRD4 (BRD4(1)) were used to guide the design of more selective compds.  The crystal structures obtained revealed two distinct binding modes.  By varying the aryl substitution pattern and developing conformationally constrained analogs, selectivity for CBP over BRD4(1) was increased.  The optimized compd. is highly potent (Kd = 21 nM) and selective, displaying 40-fold selectivity over BRD4(1).  Cellular activity was demonstrated using fluorescence recovery after photo-bleaching (FRAP) and a p53 reporter assay.  The optimized compds. are cell-active and have nanomolar affinity for CBP/p300; therefore, they should be useful in studies investigating the biol. roles of CBP and p300 and to validate the CBP and p300 bromodomains as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFzgs1DD4QZLVg90H21EOLACvtfcHk0lh7PZe_jts5sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP&md5=e5da3e96a4d234453a481c7ee5e788f2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja412434f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja412434f%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520small-molecule%2520ligands%2520for%2520the%2520CBP%252Fp300%2520bromodomains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D9308%26epage%3D9319%26doi%3D10.1021%2Fja412434f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flick, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabudhe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pletcher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of highly selective inhibitors of the creb binding protein bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5349</span>– <span class="NLM_lpage">5363</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5349-5363&author=R.+A.+Dennyauthor=A.+C.+Flickauthor=J.+Coeauthor=J.+Langilleauthor=A.+Basakauthor=S.+Liuauthor=I.+Stockauthor=P.+Sahasrabudheauthor=P.+Boninauthor=D.+A.+Hayauthor=P.+E.+Brennanauthor=M.+Pletcherauthor=L.+H.+Jonesauthor=E.+L.+P.+Chekler&title=Structure-based+design+of+highly+selective+inhibitors+of+the+creb+binding+protein+bromodomain&doi=10.1021%2Facs.jmedchem.6b01839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Highly Selective Inhibitors of the CREB Binding Protein Bromodomain</span></div><div class="casAuthors">Denny, R. Aldrin; Flick, Andrew C.; Coe, Jotham; Langille, Jonathan; Basak, Arindrajit; Liu, Shenping; Stock, Ingrid; Sahasrabudhe, Parag; Bonin, Paul; Hay, Duncan A.; Brennan, Paul E.; Pletcher, Mathew; Jones, Lyn H.; Chekler, Eugene L. Piatnitski</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5349-5363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. probes are required for preclin. target validation to interrogate novel biol. targets and pathways.  Selective inhibitors of the CREB binding protein (CREBBP)/EP300 bromodomains are required to facilitate the elucidation of biol. assocd. with these important epigenetic targets.  Medicinal chem. optimization that paid particular attention to physiochem. properties delivered chem. probes with desirable potency, selectivity, and permeability attributes.  An important feature of the optimization process was the successful application of rational structure-based drug design to address bromodomain selectivity issues (particularly against the structurally related BRD4 protein).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLWkqF-qAmdbVg90H21EOLACvtfcHk0lh7PZe_jts5sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D&md5=23676a973abba59088dedd5502877105</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01839%26sid%3Dliteratum%253Aachs%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DFlick%26aufirst%3DA.%2BC.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DBasak%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DStock%26aufirst%3DI.%26aulast%3DSahasrabudhe%26aufirst%3DP.%26aulast%3DBonin%26aufirst%3DP.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DPletcher%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26atitle%3DStructure-based%2520design%2520of%2520highly%2520selective%2520inhibitors%2520of%2520the%2520creb%2520binding%2520protein%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5349%26epage%3D5363%26doi%3D10.1021%2Facs.jmedchem.6b01839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampf, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of cf53 as a potent and orally bioavailable bromodomain and extra-terminal (bet) bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6110</span>– <span class="NLM_lpage">6120</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlahu73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6110-6120&author=Y.+Zhaoauthor=B.+Zhouauthor=L.+Baiauthor=L.+Liuauthor=C.-Y.+Yangauthor=J.+L.+Meagherauthor=J.+A.+Stuckeyauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=X.+Ranauthor=A.+Aguilarauthor=Y.+Huauthor=J.+W.+Kampfauthor=X.+Liauthor=T.+Zhaoauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Structure-based+discovery+of+cf53+as+a+potent+and+orally+bioavailable+bromodomain+and+extra-terminal+%28bet%29+bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.8b00483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of CF53 as a Potent and Orally Bioavailable Bromodomain and Extra-Terminal (BET) Bromodomain Inhibitor</span></div><div class="casAuthors">Zhao, Yujun; Zhou, Bing; Bai, Longchuan; Liu, Liu; Yang, Chao-Yie; Meagher, Jennifer L.; Stuckey, Jeanne A.; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Ran, Xu; Aguilar, Angelo; Hu, Yang; Kampf, Jeff W.; Li, Xiaoqin; Zhao, Ting; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6110-6120</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the structure-based discovery of CF53 (28) as a highly potent and orally active inhibitor of bromodomain and extra-terminal (BET) proteins.  By the incorporation of a NH-pyrazole group into the 9H-pyrimido[4,5-b]indole core, we identified a series of compds. that bind to BRD4 BD1 protein with Ki values of <1 nM and achieve low nanomolar potencies in the cell growth inhibition of leukemia and breast cancer cells.  The most-promising compd., CF53, possesses excellent oral pharmacokinetic properties and achieves significant antitumor activity in both triple-neg. breast cancer and acute leukemia xenograft models in mice.  Detn. of the co-crystal structure of CF53 with the BRD4 BD1 protein provides a structural basis for its high binding affinity to BET proteins.  CF53 is very selective over non-BET bromodomain-contg. proteins.  These data establish CF53 as a potent, selective, and orally active BET inhibitor, which warrants further evaluation for advanced preclin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprmt-PPWmeNrVg90H21EOLACvtfcHk0lijuKGn0NPKAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlahu73J&md5=86ca0c87da4cf2dc218122582f65ba7a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00483%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DRan%26aufirst%3DX.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DKampf%26aufirst%3DJ.%2BW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520discovery%2520of%2520cf53%2520as%2520a%2520potent%2520and%2520orally%2520bioavailable%2520bromodomain%2520and%2520extra-terminal%2520%2528bet%2529%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6110%26epage%3D6120%26doi%3D10.1021%2Facs.jmedchem.8b00483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of 4-(6-methoxy-2-methyl-4-(quinolin-4-yl)-9h-pyrimido[4,5-b]indol-7-yl)-3,5-dimethylisoxazole (cd161) as a potent and orally bioavailable bet bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3887</span>– <span class="NLM_lpage">3901</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00193</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00193" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvVKru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3887-3901&author=Y.+Zhaoauthor=L.+Baiauthor=L.+Liuauthor=D.+McEachernauthor=J.+A.+Stuckeyauthor=J.+L.+Meagherauthor=C.-Y.+Yangauthor=X.+Ranauthor=B.+Zhouauthor=Y.+Huauthor=X.+Liauthor=B.+Wenauthor=T.+Zhaoauthor=S.+Liauthor=D.+Sunauthor=S.+Wang&title=Structure-based+discovery+of+4-%286-methoxy-2-methyl-4-%28quinolin-4-yl%29-9h-pyrimido%5B4%2C5-b%5Dindol-7-yl%29-3%2C5-dimethylisoxazole+%28cd161%29+as+a+potent+and+orally+bioavailable+bet+bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.7b00193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of 4-(6-Methoxy-2-methyl-4-(quinolin-4-yl)-9H-pyrimido[4,5-b]indol-7-yl)-3,5-dimethylisoxazole (CD161) as a Potent and Orally Bioavailable BET Bromodomain Inhibitor</span></div><div class="casAuthors">Zhao, Yujun; Bai, Longchuan; Liu, Liu; McEachern, Donna; Stuckey, Jeanne A.; Meagher, Jennifer L.; Yang, Chao-Yie; Ran, Xu; Zhou, Bing; Hu, Yang; Li, Xiaoqin; Wen, Bo; Zhao, Ting; Li, Siwei; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3887-3901</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 9H-pyrimido[4,5-b]indole-contg. compds. was designed and synthesized to obtain potent and orally bioavailable BET inhibitors.  By incorporation of an indole or a quinoline moiety to the 9H-pyrimido[4,5-b]indole core, we identified a series of small mols. showing high binding affinities to BET proteins and low nanomolar potencies in inhibition of cell growth in acute leukemia cell lines.  One such compd., 4-(6-methoxy-2-methyl-4-(quinolin-4-yl)-9H-pyrimido[4,5-b]indol-7-yl)-3,5-dimethylisoxazole (I) has excellent microsomal stability and good oral pharmacokinetics in rats and mice.  Orally administered, I achieves significant antitumor activity in the MV4;11 leukemia and MDA-MB-231 triple-neg. breast cancer xenograft models in mice.  Detn. of the cocrystal structure of I with BRD4 BD2 provides a structural basis for its high binding affinity to BET proteins.  Testing its binding affinities against other bromodomain-contg. proteins shows that I is a highly selective inhibitor of BET proteins.  These data show that I is a potent, selective, and orally active BET inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQdUzlAHU5v7Vg90H21EOLACvtfcHk0lijuKGn0NPKAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvVKru7s%253D&md5=915019e36ab5f5b83b69b1bbc291abb0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00193%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DRan%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520discovery%2520of%25204-%25286-methoxy-2-methyl-4-%2528quinolin-4-yl%2529-9h-pyrimido%255B4%252C5-b%255Dindol-7-yl%2529-3%252C5-dimethylisoxazole%2520%2528cd161%2529%2520as%2520a%2520potent%2520and%2520orally%2520bioavailable%2520bet%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3887%26epage%3D3901%26doi%3D10.1021%2Facs.jmedchem.7b00193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baker, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zannettino, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalk, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkley, C. R.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors induce apoptosis through a MYC independent mechanism and synergise with CDK inhibitors to kill osteosarcoma cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10120</span>– <span class="NLM_lpage">10133</span>, <span class="refDoi"> DOI: 10.1038/srep10120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1038%2Fsrep10120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=25944566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOrsL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=10120-10133&author=E.+K.+Bakerauthor=S.+Taylorauthor=A.+Gupteauthor=P.+P.+Sharpauthor=M.+Waliaauthor=N.+C.+Walshauthor=A.+C.+Zannettinoauthor=A.+M.+Chalkauthor=C.+J.+Burnsauthor=C.+R.+Walkley&title=BET+inhibitors+induce+apoptosis+through+a+MYC+independent+mechanism+and+synergise+with+CDK+inhibitors+to+kill+osteosarcoma+cells&doi=10.1038%2Fsrep10120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors induce apoptosis through a MYC independent mechanism and synergise with CDK inhibitors to kill osteosarcoma cells</span></div><div class="casAuthors">Baker, Emma K.; Taylor, Scott; Gupte, Ankita; Sharp, Phillip P.; Walia, Mannu; Walsh, Nicole C.; Zannettino, Andrew C. W.; Chalk, Alistair M.; Burns, Christopher J.; Walkley, Carl R.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10120</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Osteosarcoma (OS) survival rates have plateaued in part due to a lack of new therapeutic options.  Here we demonstrate that bromodomain inhibitors (BETi), JQ1, I-BET151, I-BET762, exert potent anti-tumor activity against primary and established OS cell lines, mediated by inhibition of BRD4.  Strikingly, unlike previous observations in long-term established human OS cell lines, the antiproliferative activity of JQ1 in primary OS cells was driven by the induction of apoptosis, not cell cycle arrest.  In further contrast, JQ1 activity in OS was mediated independently of MYC downregulation.  We identified that JQ1 suppresses the transcription factor FOSL1 by displacement of BRD4 from its locus.  Loss of FOSL1 phenocopied the antiproliferative effects of JQ1, identifying FOSL1 suppression as a potential novel therapeutic approach for OS.  As a monotherapy JQ1 demonstrated significant anti-tumor activity in vivo in an OS graft model.  Further, combinatorial treatment approaches showed that JQ1 increased the sensitivity of OS cells to doxorubicin and induced potent synergistic activity when rationally combined with CDK inhibitors.  The greater level of activity achieved with the combination of BETi with CDK inhibitors demonstrates the efficacy of this combination therapy.  Taken together, our studies show that BET inhibitors are a promising new therapeutic for OS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiy0tg8DWClLVg90H21EOLACvtfcHk0ljBz_K_n1pI8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOrsL7I&md5=c95d487ed91b342bd426413430ba36ee</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fsrep10120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep10120%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DE.%2BK.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DGupte%26aufirst%3DA.%26aulast%3DSharp%26aufirst%3DP.%2BP.%26aulast%3DWalia%26aufirst%3DM.%26aulast%3DWalsh%26aufirst%3DN.%2BC.%26aulast%3DZannettino%26aufirst%3DA.%2BC.%26aulast%3DChalk%26aufirst%3DA.%2BM.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26aulast%3DWalkley%26aufirst%3DC.%2BR.%26atitle%3DBET%2520inhibitors%2520induce%2520apoptosis%2520through%2520a%2520MYC%2520independent%2520mechanism%2520and%2520synergise%2520with%2520CDK%2520inhibitors%2520to%2520kill%2520osteosarcoma%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D10120%26epage%3D10133%26doi%3D10.1038%2Fsrep10120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stratikopoulos, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dendy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabolcs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaykin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefebvre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span> <span> </span><span class="NLM_article-title">Kinase and bet inhibitors together clamp inhibition of pi3k signaling and overcome resistance to therapy</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">837</span>– <span class="NLM_lpage">851</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.ccell.2015.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=26058079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVejurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=837-851&author=E.+E.+Stratikopoulosauthor=M.+Dendyauthor=M.+Szabolcsauthor=A.+J.+Khaykinauthor=C.+Lefebvreauthor=M.-M.+Zhouauthor=R.+Parsons&title=Kinase+and+bet+inhibitors+together+clamp+inhibition+of+pi3k+signaling+and+overcome+resistance+to+therapy&doi=10.1016%2Fj.ccell.2015.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase and BET Inhibitors Together Clamp Inhibition of PI3K Signaling and Overcome Resistance to Therapy</span></div><div class="casAuthors">Stratikopoulos, Elias E.; Dendy, Meaghan; Szabolcs, Matthias; Khaykin, Alan J.; Lefebvre, Celine; Zhou, Ming-Ming; Parsons, Ramon</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">837-851</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Unsustained enzyme inhibition is a barrier to targeted therapy for cancer.  Here, resistance to a class I PI3K inhibitor in a model of metastatic breast cancer driven by PI3K and MYC was assocd. with feedback activation of tyrosine kinase receptors (RTKs), AKT, mTOR, and MYC.  Inhibitors of bromodomain and extra terminal domain (BET) proteins also failed to affect tumor growth.  Interestingly, BET inhibitors lowered PI3K signaling and dissocd. BRD4 from chromatin at regulatory regions of insulin receptor and EGFR family RTKs to reduce their expression.  Combined PI3K and BET inhibition induced cell death, tumor regression, and clamped inhibition of PI3K signaling in a broad range of tumor cell lines to provide a strategy to overcome resistance to kinase inhibitor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmdX2xaXkHjbVg90H21EOLACvtfcHk0ljBz_K_n1pI8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVejurbK&md5=91cca562c9bff9e11e94535916583867</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DStratikopoulos%26aufirst%3DE.%2BE.%26aulast%3DDendy%26aufirst%3DM.%26aulast%3DSzabolcs%26aufirst%3DM.%26aulast%3DKhaykin%26aufirst%3DA.%2BJ.%26aulast%3DLefebvre%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DParsons%26aufirst%3DR.%26atitle%3DKinase%2520and%2520bet%2520inhibitors%2520together%2520clamp%2520inhibition%2520of%2520pi3k%2520signaling%2520and%2520overcome%2520resistance%2520to%2520therapy%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D837%26epage%3D851%26doi%3D10.1016%2Fj.ccell.2015.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Čančer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drews, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bengtsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westermark, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelander, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsberg-Nilsson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhrbom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weishaupt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swartling, F. J.</span></span> <span> </span><span class="NLM_article-title">BET and aurora kinase a inhibitors synergize against mycn-positive human glioblastoma cells</span>. <i>Cell Death Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">881</span>– <span class="NLM_lpage">885</span>, <span class="refDoi"> DOI: 10.1038/s41419-019-2120-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1038%2Fs41419-019-2120-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=31754113" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=881-885&author=M.+%C4%8Can%C4%8Derauthor=L.+F.+Drewsauthor=J.+Bengtssonauthor=S.+Bolinauthor=G.+Rosenauthor=B.+Westermarkauthor=S.+Nelanderauthor=K.+Forsberg-Nilssonauthor=L.+Uhrbomauthor=H.+Weishauptauthor=F.+J.+Swartling&title=BET+and+aurora+kinase+a+inhibitors+synergize+against+mycn-positive+human+glioblastoma+cells&doi=10.1038%2Fs41419-019-2120-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41419-019-2120-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-019-2120-1%26sid%3Dliteratum%253Aachs%26aulast%3D%25C4%258Can%25C4%258Der%26aufirst%3DM.%26aulast%3DDrews%26aufirst%3DL.%2BF.%26aulast%3DBengtsson%26aufirst%3DJ.%26aulast%3DBolin%26aufirst%3DS.%26aulast%3DRosen%26aufirst%3DG.%26aulast%3DWestermark%26aufirst%3DB.%26aulast%3DNelander%26aufirst%3DS.%26aulast%3DForsberg-Nilsson%26aufirst%3DK.%26aulast%3DUhrbom%26aufirst%3DL.%26aulast%3DWeishaupt%26aufirst%3DH.%26aulast%3DSwartling%26aufirst%3DF.%2BJ.%26atitle%3DBET%2520and%2520aurora%2520kinase%2520a%2520inhibitors%2520synergize%2520against%2520mycn-positive%2520human%2520glioblastoma%2520cells%26jtitle%3DCell%2520Death%2520Discovery%26date%3D2019%26volume%3D10%26spage%3D881%26epage%3D885%26doi%3D10.1038%2Fs41419-019-2120-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sahni, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gayle, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonk, K. L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vite, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yori, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seachrist, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landis, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keri, R. A.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and extraterminal protein inhibition blocks growth of triple-negative breast cancers through the suppression of aurora kinases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">23756</span>– <span class="NLM_lpage">23768</span>, <span class="refDoi"> DOI: 10.1074/jbc.m116.738666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1074%2Fjbc.M116.738666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=27650498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2gsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=23756-23768&author=J.+M.+Sahniauthor=S.+S.+Gayleauthor=K.+L.+W.+Bonkauthor=L.+C.+Viteauthor=J.+L.+Yoriauthor=B.+Webbauthor=E.+K.+Ramosauthor=D.+D.+Seachristauthor=M.+D.+Landisauthor=J.+C.+Changauthor=J.+E.+Bradnerauthor=R.+A.+Keri&title=Bromodomain+and+extraterminal+protein+inhibition+blocks+growth+of+triple-negative+breast+cancers+through+the+suppression+of+aurora+kinases&doi=10.1074%2Fjbc.m116.738666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain and Extraterminal Protein Inhibition Blocks Growth of Triple-negative Breast Cancers through the Suppression of Aurora Kinases</span></div><div class="casAuthors">Sahni, Jennifer M.; Gayle, Sylvia S.; Weber Bonk, Kristen L.; Vite, Leslie Cuellar; Yori, Jennifer L.; Webb, Bryan; Ramos, Erika K.; Seachrist, Darcie D.; Landis, Melissa D.; Chang, Jenny C.; Bradner, James E.; Keri, Ruth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">23756-23768</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Bromodomain and extraterminal (BET) proteins are epigenetic "readers" that recognize acetylated histones and mark areas of the genome for transcription.  BRD4, a BET family member protein, has been implicated in a no. of types of cancer, and BET protein inhibitors (BETi) are efficacious in many preclin. cancer models.  However, the drivers of response to BETi vary depending on tumor type, and little is known regarding the target genes conveying BETi activity in triple-neg. breast cancer (TNBC).  Here, we show that BETi repress growth of multiple in vitro and in vivo models of TNBC by inducing two terminal responses: apoptosis and senescence.  Unlike in other cancers, response to BETi in TNBC is not dependent upon suppression of MYC.  Instead, both end points are preceded by the appearance of polyploid cells caused by the suppression of Aurora kinases A and B (AURKA/B), which are crit. mediators of mitosis.  In addn., AURKA/B inhibitors phenocopy the effects of BETi.  These results indicate that Aurora kinases play an important role in the growth suppressive activity of BETi in TNBC.  Elucidating the mechanism of response to BETi in TNBC should (1) facilitate the prediction of how distinct TNBC tumors will respond to BETi and (2) inform the rational design of drug combination therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-oIjctyeyMLVg90H21EOLACvtfcHk0ljBz_K_n1pI8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2gsLrI&md5=c39bc177e22af9219f7234460d1c5524</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.738666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.738666%26sid%3Dliteratum%253Aachs%26aulast%3DSahni%26aufirst%3DJ.%2BM.%26aulast%3DGayle%26aufirst%3DS.%2BS.%26aulast%3DBonk%26aufirst%3DK.%2BL.%2BW.%26aulast%3DVite%26aufirst%3DL.%2BC.%26aulast%3DYori%26aufirst%3DJ.%2BL.%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DRamos%26aufirst%3DE.%2BK.%26aulast%3DSeachrist%26aufirst%3DD.%2BD.%26aulast%3DLandis%26aufirst%3DM.%2BD.%26aulast%3DChang%26aufirst%3DJ.%2BC.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DKeri%26aufirst%3DR.%2BA.%26atitle%3DBromodomain%2520and%2520extraterminal%2520protein%2520inhibition%2520blocks%2520growth%2520of%2520triple-negative%2520breast%2520cancers%2520through%2520the%2520suppression%2520of%2520aurora%2520kinases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D23756%26epage%3D23768%26doi%3D10.1074%2Fjbc.m116.738666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minzel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatachalam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fink, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachya, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burstain, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdman, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushansky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shlush, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pikarsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snir-Alkalay, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Neriah, Y.</span></span> <span> </span><span class="NLM_article-title">Small molecules co-targeting ckialpha and the transcriptional kinases cdk7/9 control aml in preclinical models</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.07.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.cell.2018.07.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=30146162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFyktbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=171-185&author=W.+Minzelauthor=A.+Venkatachalamauthor=A.+Finkauthor=E.+Hungauthor=G.+Brachyaauthor=I.+Burstainauthor=M.+Shahamauthor=A.+Rivlinauthor=I.+Omerauthor=A.+Zingerauthor=S.+Eliasauthor=E.+Winterauthor=P.+E.+Erdmanauthor=R.+W.+Sullivanauthor=L.+Fungauthor=F.+Mercurioauthor=D.+Liauthor=J.+Vaccaauthor=N.+Kaushanskyauthor=L.+Shlushauthor=M.+Orenauthor=R.+Levineauthor=E.+Pikarskyauthor=I.+Snir-Alkalayauthor=Y.+Ben-Neriah&title=Small+molecules+co-targeting+ckialpha+and+the+transcriptional+kinases+cdk7%2F9+control+aml+in+preclinical+models&doi=10.1016%2Fj.cell.2018.07.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecules Co-targeting CKIα and the Transcriptional Kinases CDK7/9 Control AML in Preclinical Models</span></div><div class="casAuthors">Minzel, Waleed; Venkatachalam, Avanthika; Fink, Avner; Hung, Eric; Brachya, Guy; Burstain, Ido; Shaham, Maya; Rivlin, Amitai; Omer, Itay; Zinger, Adar; Elias, Shlomo; Winter, Eitan; Erdman, Paul E.; Sullivan, Robert W.; Fung, Leah; Mercurio, Frank; Li, Dansu; Vacca, Joseph; Kaushansky, Nathali; Shlush, Liran; Oren, Moshe; Levine, Ross; Pikarsky, Eli; Snir-Alkalay, Irit; Ben-Neriah, Yinon</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">171-185.e25</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">CKIα ablation induces p53 activation, and CKIα degrdn. underlies the therapeutic effect of lenalidomide in a pre-leukemia syndrome.  Here we describe the development of CKIα inhibitors, which co-target the transcriptional kinases CDK7 and CDK9, thereby augmenting CKIα-induced p53 activation and its anti-leukemic activity.  Oncogene-driving super-enhancers (SEs) are highly sensitive to CDK7/9 inhibition.  We identified multiple newly gained SEs in primary mouse acute myeloid leukemia (AML) cells and demonstrate that the inhibitors abolish many SEs and preferentially suppress the transcription elongation of SE-driven oncogenes.  We show that blocking CKIα together with CDK7 and/or CDK9 synergistically stabilize p53, deprive leukemia cells of survival and proliferation-maintaining SE-driven oncogenes, and induce apoptosis.  Leukemia progenitors are selectively eliminated by the inhibitors, explaining their therapeutic efficacy with preserved hematopoiesis and leukemia cure potential; they eradicate leukemia in MLL-AF9 and Tet2-/-;Flt3ITD AML mouse models and in several patient-derived AML xenograft models, supporting their potential efficacy in curing human leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4QEH6R_ll7Vg90H21EOLACvtfcHk0ljggJPluFtf1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFyktbvF&md5=b3e088a6b262dc8998ff334867102aa5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.07.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.07.045%26sid%3Dliteratum%253Aachs%26aulast%3DMinzel%26aufirst%3DW.%26aulast%3DVenkatachalam%26aufirst%3DA.%26aulast%3DFink%26aufirst%3DA.%26aulast%3DHung%26aufirst%3DE.%26aulast%3DBrachya%26aufirst%3DG.%26aulast%3DBurstain%26aufirst%3DI.%26aulast%3DShaham%26aufirst%3DM.%26aulast%3DRivlin%26aufirst%3DA.%26aulast%3DOmer%26aufirst%3DI.%26aulast%3DZinger%26aufirst%3DA.%26aulast%3DElias%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DE.%26aulast%3DErdman%26aufirst%3DP.%2BE.%26aulast%3DSullivan%26aufirst%3DR.%2BW.%26aulast%3DFung%26aufirst%3DL.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DVacca%26aufirst%3DJ.%26aulast%3DKaushansky%26aufirst%3DN.%26aulast%3DShlush%26aufirst%3DL.%26aulast%3DOren%26aufirst%3DM.%26aulast%3DLevine%26aufirst%3DR.%26aulast%3DPikarsky%26aufirst%3DE.%26aulast%3DSnir-Alkalay%26aufirst%3DI.%26aulast%3DBen-Neriah%26aufirst%3DY.%26atitle%3DSmall%2520molecules%2520co-targeting%2520ckialpha%2520and%2520the%2520transcriptional%2520kinases%2520cdk7%252F9%2520control%2520aml%2520in%2520preclinical%2520models%26jtitle%3DCell%26date%3D2018%26volume%3D175%26spage%3D171%26epage%3D185%26doi%3D10.1016%2Fj.cell.2018.07.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7346</span>– <span class="NLM_lpage">7359</span>, <span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0ljggJPluFtf1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tögel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nightingale, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chueh, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayachandran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phesse, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arango, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhillon, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gahman, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiau, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mariadason, J. M.</span></span> <span> </span><span class="NLM_article-title">Dual targeting of bromodomain and extraterminal domain proteins, and wnt or mapk signaling, inhibits c-myc expression and proliferation of colorectal cancer cells</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1217</span>– <span class="NLM_lpage">1226</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-15-0724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1158%2F1535-7163.MCT-15-0724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=26983878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC28XptVeqs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=1217-1226&author=L.+T%C3%B6gelauthor=R.+Nightingaleauthor=A.+C.+Chuehauthor=A.+Jayachandranauthor=H.+Tranauthor=T.+Phesseauthor=R.+Wuauthor=O.+M.+Sieberauthor=D.+Arangoauthor=A.+S.+Dhillonauthor=M.+A.+Dawsonauthor=B.+Diez-Dacalauthor=T.+C.+Gahmanauthor=P.+Filippakopoulosauthor=A.+K.+Shiauauthor=J.+M.+Mariadason&title=Dual+targeting+of+bromodomain+and+extraterminal+domain+proteins%2C+and+wnt+or+mapk+signaling%2C+inhibits+c-myc+expression+and+proliferation+of+colorectal+cancer+cells&doi=10.1158%2F1535-7163.mct-15-0724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Targeting of Bromodomain and Extraterminal Domain Proteins, and WNT or MAPK Signaling, Inhibits c-MYC Expression and Proliferation of Colorectal Cancer Cells</span></div><div class="casAuthors">Togel, Lars; Nightingale, Rebecca; Chueh, Anderly C.; Jayachandran, Aparna; Tran, Hoanh; Phesse, Toby; Wu, Rui; Sieber, Oliver M.; Arango, Diego; Dhillon, Amardeep S.; Dawson, Mark A.; Diez-Dacal, Beatriz; Gahman, Timothy C.; Filippakopoulos, Panagis; Shiau, Andrew K.; Mariadason, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1217-1226</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Inhibitors of the bromodomain and extraterminal domain (BET) protein family attenuate the proliferation of several tumor cell lines.  These effects are mediated, at least in part, through repression of c-MYC.  In colorectal cancer, overexpression of c-MYC due to hyperactive WNT/β-catenin/TCF signaling is a key driver of tumor progression; however, effective strategies to target this oncogene remain elusive.  Here, we investigated the effect of BET inhibitors (BETi) on colorectal cancer cell proliferation and c-MYC expression.  Treatment of 20 colorectal cancer cell lines with the BETi JQ1 identified a subset of highly sensitive lines.  JQ1 sensitivity was higher in cell lines with microsatellite instability but was not assocd. with the CpG island methylator phenotype, c-MYC expression or amplification status, BET protein expression, or mutation status of TP53, KRAS/BRAF, or PIK3CA/PTEN.  Conversely, JQ1 sensitivity correlated significantly with the magnitude of c-MYC mRNA and protein repression.  JQ1-mediated c-MYC repression was not due to generalized attenuation of β-catenin/TCF-mediated transcription, as JQ1 had minimal effects on other β-catenin/TCF target genes or β-catenin/TCF reporter activity.  BETi preferentially target super-enhancer-regulated genes, and a super-enhancer in c-MYC was recently identified in HCT116 cells to which BRD4 and effector transcription factors of the WNT/β-catenin/TCF and MEK/ERK pathways are recruited.  Combined targeting of c-MYC with JQ1 and inhibitors of these pathways additively repressed c-MYC and proliferation of HCT116 cells.  These findings demonstrate that BETi downregulate c-MYC expression and inhibit colorectal cancer cell proliferation and identify strategies for enhancing the effects of BETi on c-MYC repression by combinatorial targeting the c-MYC super-enhancer.  Mol Cancer Ther; 15(6); 1217-26. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptJtvXsYFDz7Vg90H21EOLACvtfcHk0lhbownGK8rUDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptVeqs74%253D&md5=01f9e3ab8f181ca354bb890b58ee6adb</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-15-0724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-15-0724%26sid%3Dliteratum%253Aachs%26aulast%3DT%25C3%25B6gel%26aufirst%3DL.%26aulast%3DNightingale%26aufirst%3DR.%26aulast%3DChueh%26aufirst%3DA.%2BC.%26aulast%3DJayachandran%26aufirst%3DA.%26aulast%3DTran%26aufirst%3DH.%26aulast%3DPhesse%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DR.%26aulast%3DSieber%26aufirst%3DO.%2BM.%26aulast%3DArango%26aufirst%3DD.%26aulast%3DDhillon%26aufirst%3DA.%2BS.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DGahman%26aufirst%3DT.%2BC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DShiau%26aufirst%3DA.%2BK.%26aulast%3DMariadason%26aufirst%3DJ.%2BM.%26atitle%3DDual%2520targeting%2520of%2520bromodomain%2520and%2520extraterminal%2520domain%2520proteins%252C%2520and%2520wnt%2520or%2520mapk%2520signaling%252C%2520inhibits%2520c-myc%2520expression%2520and%2520proliferation%2520of%2520colorectal%2520cancer%2520cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D1217%26epage%3D1226%26doi%3D10.1158%2F1535-7163.mct-15-0724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhadel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollah, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xenarios, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conaway, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carey, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gioeli, D.</span></span> <span> </span><span class="NLM_article-title">CDK9 regulates AR promoter selectivity and cell growth through serine 81 phosphorylation</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2267</span>– <span class="NLM_lpage">2280</span>, <span class="refDoi"> DOI: 10.1210/me.2010-0238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1210%2Fme.2010-0238" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=20980437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVenurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=2267-2280&author=V.+Gordonauthor=S.+Bhadelauthor=W.+Wunderlichauthor=J.+Zhangauthor=S.+B.+Ficarroauthor=S.+A.+Mollahauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=I.+Xenariosauthor=W.+C.+Hahnauthor=M.+Conawayauthor=M.+F.+Careyauthor=D.+Gioeli&title=CDK9+regulates+AR+promoter+selectivity+and+cell+growth+through+serine+81+phosphorylation&doi=10.1210%2Fme.2010-0238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">CDK9 regulates AR promoter selectivity and cell growth through serine 81 phosphorylation</span></div><div class="casAuthors">Gordon, Vicki; Bhadel, Shriti; Wunderlich, Winfried; Zhang, JoAnn; Ficarro, Scott B.; Mollah, Sahana A.; Shabanowitz, Jeffrey; Hunt, Donald F.; Xenarios, Ioannis; Hahn, William C.; Conaway, Mark; Carey, Michael F.; Gioeli, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2267-2280</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Previously we detd. that S81 is the highest stoichiometric phosphorylation on the androgen receptor (AR) in response to hormone.  To explore the role of this phosphorylation on growth, we stably expressed wild-type and S81A mutant AR in LHS and LAPC4 cells.  The cells with increased wild-type AR expression grow faster compared with parental cells and S81A mutant-expressing cells, indicating that loss of S81 phosphorylation limits cell growth.  To explore how S81 regulates cell growth, we tested whether S81 phosphorylation regulates AR transcriptional activity.  LHS cells stably expressing wild-type and S81A mutant AR showed differences in the regulation of endogenous AR target genes, suggesting that S81 phosphorylation regulates promoter selectivity.  We next sought to identify the S81 kinase using ion trap mass spectrometry to analyze AR-assocd. proteins in immunoppts. from cells.  We obsd. cyclin-dependent kinase (CDK)9 assocn. with the AR.  CDK9 phosphorylates the AR on S81 in vitro.  Phosphorylation is specific to S81 because CDK9 did not phosphorylate the AR on other serine phosphorylation sites.  Overexpression of CDK9 with its cognate cyclin, Cyclin T, increased S81 phosphorylation levels in cells.  Small interfering RNA knockdown of CDK9 protein levels decreased hormone-induced S81 phosphorylation.  Addnl., treatment of LNCaP cells with the CDK9 inhibitors, 5,6-dichloro-1-β-D-ribofuranosylbenzimidazole and Flavopiridol, reduced S81 phosphorylation further, suggesting that CDK9 regulates S81 phosphorylation.  Pharmacol. inhibition of CDK9 also resulted in decreased AR transcription in LNCaP cells.  Collectively these results suggest that CDK9 phosphorylation of AR S81 is an important step in regulating AR transcriptional activity and prostate cancer cell growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWRr14oBHWlrVg90H21EOLACvtfcHk0lhbownGK8rUDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVenurY%253D&md5=577d202cad2fa850526dde99e69cf49d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1210%2Fme.2010-0238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2010-0238%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DV.%26aulast%3DBhadel%26aufirst%3DS.%26aulast%3DWunderlich%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DMollah%26aufirst%3DS.%2BA.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DXenarios%26aufirst%3DI.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DConaway%26aufirst%3DM.%26aulast%3DCarey%26aufirst%3DM.%2BF.%26aulast%3DGioeli%26aufirst%3DD.%26atitle%3DCDK9%2520regulates%2520AR%2520promoter%2520selectivity%2520and%2520cell%2520growth%2520through%2520serine%252081%2520phosphorylation%26jtitle%3DMol.%2520Endocrinol.%26date%3D2010%26volume%3D24%26spage%3D2267%26epage%3D2280%26doi%3D10.1210%2Fme.2010-0238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsten, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhogbi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kool, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisterernst, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerl, K.</span></span> <span> </span><span class="NLM_article-title">Combined BRD4 and CDK9 inhibition as a new therapeutic approach in malignant rhabdoid tumors</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">84986</span>– <span class="NLM_lpage">84995</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.18583</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.18632%2Foncotarget.18583" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=29156698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A280%3ADC%252BC1M3jtFSitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=84986-84995&author=N.+Morenoauthor=T.+Holstenauthor=J.+Mertinsauthor=A.+Zhogbiauthor=P.+Johannauthor=M.+Koolauthor=M.+Meisterernstauthor=K.+Kerl&title=Combined+BRD4+and+CDK9+inhibition+as+a+new+therapeutic+approach+in+malignant+rhabdoid+tumors&doi=10.18632%2Foncotarget.18583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Combined BRD4 and CDK9 inhibition as a new therapeutic approach in malignant rhabdoid tumors</span></div><div class="casAuthors">Moreno Natalia; Holsten Till; Mertins Julius; Zhogbi Annabelle; Meisterernst Michael; Kerl Kornelius; Moreno Natalia; Kerl Kornelius; Johann Pascal; Kool Marcel</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">84986-84995</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Rhabdoid tumors are caused by the deletion of SMARCB1, whose protein encodes the SMARCB1 subunit of the chromatin remodeling complex SWI/SNF that is involved in global chromatin organization and gene expression control.  Simultaneously inhibiting the main players involved in the deregulated transcription machinery is a promising option for preventing exaggerated tumor cell proliferation and survival as it may bypass compensatory mechanisms.  In support of this hypothesis, we report efficient impairment of cellular proliferation and strong induction of cell death elicited by inhibition of bromodomain protein BRD4 and transcription kinase CDK9 using small molecular compounds.  Combination of both compounds efficiently represses antiapoptotic genes and the oncogene MYC.  Our results provide a novel approach for the treatment of RT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxaJ-Xi0XM-zdblLE_E9PWfW6udTcc2eYOhs4PBtZ3B7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3jtFSitQ%253D%253D&md5=8cf6e23c68ef1a2ba66c5cb9fac43656</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.18583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.18583%26sid%3Dliteratum%253Aachs%26aulast%3DMoreno%26aufirst%3DN.%26aulast%3DHolsten%26aufirst%3DT.%26aulast%3DMertins%26aufirst%3DJ.%26aulast%3DZhogbi%26aufirst%3DA.%26aulast%3DJohann%26aufirst%3DP.%26aulast%3DKool%26aufirst%3DM.%26aulast%3DMeisterernst%26aufirst%3DM.%26aulast%3DKerl%26aufirst%3DK.%26atitle%3DCombined%2520BRD4%2520and%2520CDK9%2520inhibition%2520as%2520a%2520new%2520therapeutic%2520approach%2520in%2520malignant%2520rhabdoid%2520tumors%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D84986%26epage%3D84995%26doi%3D10.18632%2Foncotarget.18583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brägelmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dammert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietlein, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuckmann, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choidas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tischler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habenberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz-Cuaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leenders, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickhoff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Termathe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sallouh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greff, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varga, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balke-Want, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peifer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhardt, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Örfi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kéri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansén, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heukamp, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Büttner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebl, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sos, M. L.</span></span> <span> </span><span class="NLM_article-title">Systematic kinase inhibitor profiling identifies cdk9 as a synthetic lethal target in nut midline carcinoma</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2833</span>– <span class="NLM_lpage">2845</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.08.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2Fj.celrep.2017.08.082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=28930680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FgsFygtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=2833-2845&author=J.+Br%C3%A4gelmannauthor=M.+A.+Dammertauthor=F.+Dietleinauthor=J.+M.+Heuckmannauthor=A.+Choidasauthor=S.+B%C3%B6hmauthor=A.+Richtersauthor=D.+Basuauthor=V.+Tischlerauthor=C.+Lorenzauthor=P.+Habenbergerauthor=Z.+Fangauthor=S.+Ortiz-Cuaranauthor=F.+Leendersauthor=J.+Eickhoffauthor=U.+Kochauthor=M.+Getlikauthor=M.+Termatheauthor=M.+Sallouhauthor=Z.+Greffauthor=Z.+Vargaauthor=H.+Balke-Wantauthor=C.+A.+Frenchauthor=M.+Peiferauthor=H.+C.+Reinhardtauthor=L.+%C3%96rfiauthor=G.+K%C3%A9riauthor=S.+Ans%C3%A9nauthor=L.+C.+Heukampauthor=R.+B%C3%BCttnerauthor=D.+Rauhauthor=B.+M.+Kleblauthor=R.+K.+Thomasauthor=M.+L.+Sos&title=Systematic+kinase+inhibitor+profiling+identifies+cdk9+as+a+synthetic+lethal+target+in+nut+midline+carcinoma&doi=10.1016%2Fj.celrep.2017.08.082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Kinase Inhibitor Profiling Identifies CDK9 as a Synthetic Lethal Target in NUT Midline Carcinoma</span></div><div class="casAuthors">Bragelmann Johannes; Dammert Marcel A; Bohm Stefanie; Lorenz Carina; Dietlein Felix; Ansen Sascha; Heuckmann Johannes M; Choidas Axel; Habenberger Peter; Eickhoff Jan; Koch Uwe; Klebl Bert M; Richters Andre; Basu Debjit; Fang Zhizhou; Getlik Matthaus; Termathe Martin; Sallouh Muhammad; Rauh Daniel; Tischler Verena; Ortiz-Cuaran Sandra; Leenders Frauke; Peifer Martin; Greff Zoltan; Varga Zoltan; Keri Gyorgy; Balke-Want Hyatt; French Christopher A; Reinhardt H Christian; Orfi Laszlo; Heukamp Lukas C; Buttner Reinhard; Thomas Roman K; Sos Martin L</div><div class="citationInfo"><span class="NLM_cas:title">Cell reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2833-2845</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Kinase inhibitors represent the backbone of targeted cancer therapy, yet only a limited number of oncogenic drivers are directly druggable.  By interrogating the activity of 1,505 kinase inhibitors, we found that BRD4-NUT-rearranged NUT midline carcinoma (NMC) cells are specifically killed by CDK9 inhibition (CDK9i) and depend on CDK9 and Cyclin-T1 expression.  We show that CDK9i leads to robust induction of apoptosis and of markers of DNA damage response in NMC cells.  While both CDK9i and bromodomain inhibition over time result in reduced Myc protein expression, only bromodomain inhibition induces cell differentiation and a p21-induced cell-cycle arrest in these cells.  Finally, RNA-seq and ChIP-based analyses reveal a BRD4-NUT-specific CDK9i-induced perturbation of transcriptional elongation.  Thus, our data provide a mechanistic basis for the genotype-dependent vulnerability of NMC cells to CDK9i that may be of relevance for the development of targeted therapies for NMC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRoGX5-pfeoO-qrn-c4Of3KfW6udTcc2ebXIAC2O6ixvLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FgsFygtA%253D%253D&md5=8f8e1405907c5132a15c875f84ac94ba</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.08.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.08.082%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25A4gelmann%26aufirst%3DJ.%26aulast%3DDammert%26aufirst%3DM.%2BA.%26aulast%3DDietlein%26aufirst%3DF.%26aulast%3DHeuckmann%26aufirst%3DJ.%2BM.%26aulast%3DChoidas%26aufirst%3DA.%26aulast%3DB%25C3%25B6hm%26aufirst%3DS.%26aulast%3DRichters%26aufirst%3DA.%26aulast%3DBasu%26aufirst%3DD.%26aulast%3DTischler%26aufirst%3DV.%26aulast%3DLorenz%26aufirst%3DC.%26aulast%3DHabenberger%26aufirst%3DP.%26aulast%3DFang%26aufirst%3DZ.%26aulast%3DOrtiz-Cuaran%26aufirst%3DS.%26aulast%3DLeenders%26aufirst%3DF.%26aulast%3DEickhoff%26aufirst%3DJ.%26aulast%3DKoch%26aufirst%3DU.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DTermathe%26aufirst%3DM.%26aulast%3DSallouh%26aufirst%3DM.%26aulast%3DGreff%26aufirst%3DZ.%26aulast%3DVarga%26aufirst%3DZ.%26aulast%3DBalke-Want%26aufirst%3DH.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DPeifer%26aufirst%3DM.%26aulast%3DReinhardt%26aufirst%3DH.%2BC.%26aulast%3D%25C3%2596rfi%26aufirst%3DL.%26aulast%3DK%25C3%25A9ri%26aufirst%3DG.%26aulast%3DAns%25C3%25A9n%26aufirst%3DS.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DB%25C3%25BCttner%26aufirst%3DR.%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DKlebl%26aufirst%3DB.%2BM.%26aulast%3DThomas%26aufirst%3DR.%2BK.%26aulast%3DSos%26aufirst%3DM.%2BL.%26atitle%3DSystematic%2520kinase%2520inhibitor%2520profiling%2520identifies%2520cdk9%2520as%2520a%2520synthetic%2520lethal%2520target%2520in%2520nut%2520midline%2520carcinoma%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D20%26spage%3D2833%26epage%3D2845%26doi%3D10.1016%2Fj.celrep.2017.08.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Q.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.-H.</span></span> <span> </span><span class="NLM_article-title">Upgrade of macromolecular crystallography beamline BL17U1 at SSRF</span>. <i>Nucl. Sci. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1007/s41365-018-0398-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1007%2Fs41365-018-0398-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=68-74&author=Q.-S.+Wangauthor=K.-H.+Zhangauthor=Y.+Cuauthor=Z.-J.+Wangauthor=Q.-Y.+Panauthor=K.+Liuauthor=M.-J.+Liauthor=Q.+Xuauthor=C.-Y.+Xuauthor=F.+Yuauthor=J.-H.+He&title=Upgrade+of+macromolecular+crystallography+beamline+BL17U1+at+SSRF&doi=10.1007%2Fs41365-018-0398-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fs41365-018-0398-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs41365-018-0398-9%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.-S.%26aulast%3DZhang%26aufirst%3DK.-H.%26aulast%3DCu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.-J.%26aulast%3DPan%26aufirst%3DQ.-Y.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DM.-J.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DC.-Y.%26aulast%3DYu%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DJ.-H.%26atitle%3DUpgrade%2520of%2520macromolecular%2520crystallography%2520beamline%2520BL17U1%2520at%2520SSRF%26jtitle%3DNucl.%2520Sci.%2520Technol.%26date%3D2018%26volume%3D29%26spage%3D68%26epage%3D74%26doi%3D10.1007%2Fs41365-018-0398-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span>; <span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-ray diffraction data collected in oscillation mode</span>.  <i>Methods in Enzymology</i>, <span class="NLM_year">1997</span>; Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minor&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26btitle%3DMethods%2520in%2520Enzymology%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-Likelihood Method</span>. <i>Acta Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1107%2Fs0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-Likelihood+Method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljC3gOrCnZ8bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2Fs0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-Likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <i>Acta Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/s0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1107%2Fs0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.-W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2Fs0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0liCzgFSoafg4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2Fs0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2Fs0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liCzgFSoafg4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0liCzgFSoafg4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span> <span> </span><span class="NLM_article-title">Fragment-based drug discovery of 2-thiazolidinones as inhibitors of the histone reader BRD4 bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3833</span>– <span class="NLM_lpage">3851</span>, <span class="refDoi"> DOI: 10.1021/jm301793a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301793a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3833-3851&author=L.+Zhaoauthor=D.+Caoauthor=T.+Chenauthor=Y.+Wangauthor=Z.+Miaoauthor=Y.+Xuauthor=W.+Chenauthor=X.+Wangauthor=Y.+Liauthor=Z.+Duauthor=B.+Xiongauthor=J.+Liauthor=C.+Xuauthor=N.+Zhangauthor=J.+Heauthor=J.+Shen&title=Fragment-based+drug+discovery+of+2-thiazolidinones+as+inhibitors+of+the+histone+reader+BRD4+bromodomain&doi=10.1021%2Fjm301793a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery of 2-Thiazolidinones as Inhibitors of the Histone Reader BRD4 Bromodomain</span></div><div class="casAuthors">Zhao, Lele; Cao, Danyan; Chen, Tiantian; Wang, Yingqing; Miao, Zehong; Xu, Yechun; Chen, Wuyan; Wang, Xin; Li, Yanlian; Du, Zhiyan; Xiong, Bing; Li, Jian; Xu, Chunyan; Zhang, Naixia; He, Jianhua; Shen, Jingkang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3833-3851</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recognizing acetyllysine of histone is a vital process of epigenetic regulation that is mediated by a protein module called bromodomain.  To contribute novel scaffolds for developing into bromodomain inhibitors, the authors utilize a fragment-based drug discovery approach.  By successively applying docking and x-ray crystallog., they were able to identify 9 fragment hits from diffracting more than 60 crystals.  In the present work, they described four of them and carried out the integrated lead optimization for fragment I, which bears a 2-thiazolidinone core.  After several rounds of structure guided modifications, they assessed the druggability of 2-thiazolidinone by modulating in vitro pharmacokinetic studies and cellular activity assay.  The results showed that two potent of 2-thiazolidinone compds. II and III have good metabolic stability.  Also, the cellular assay confirmed the activities of 2-thiazolidinones.  Together, they hope the identified 2-thiazolidinone chemotype and other fragment hits described herein can stimulate researchers to develop more diversified bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfi2pyYq6c4LVg90H21EOLACvtfcHk0lhLF1nGOE-WKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D&md5=8308db5d83cb72b29f261cc134514ece</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm301793a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301793a%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DFragment-based%2520drug%2520discovery%2520of%25202-thiazolidinones%2520as%2520inhibitors%2520of%2520the%2520histone%2520reader%2520BRD4%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3833%26epage%3D3851%26doi%3D10.1021%2Fjm301793a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Fragment-based drug discovery of 2-thiazolidinones as BRD4 inhibitors: 2. Structure-based optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1281</span>– <span class="NLM_lpage">1297</span>, <span class="refDoi"> DOI: 10.1021/jm501504k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501504k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFSqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1281-1297&author=L.+Zhaoauthor=Y.+Wangauthor=D.+Caoauthor=T.+Chenauthor=Q.+Wangauthor=Y.+Liauthor=Y.+Xuauthor=N.+Zhangauthor=X.+Wangauthor=D.+Chenauthor=L.+Chenauthor=Y.-L.+Chenauthor=G.+Xiaauthor=Z.+Shiauthor=Y.-C.+Liuauthor=Y.+Linauthor=Z.+Miaoauthor=J.+Shenauthor=B.+Xiong&title=Fragment-based+drug+discovery+of+2-thiazolidinones+as+BRD4+inhibitors%3A+2.+Structure-based+optimization&doi=10.1021%2Fjm501504k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery of 2-Thiazolidinones as BRD4 Inhibitors: Structure-Based Optimization</span></div><div class="casAuthors">Zhao, Lele; Wang, Yingqing; Cao, Danyan; Chen, Tiantian; Wang, Qi; Li, Yanlian; Xu, Yechun; Zhang, Naixia; Wang, Xin; Chen, Danqi; Chen, Lin; Chen, Yue-Lei; Xia, Guangxin; Shi, Zhe; Liu, Yu-Chih; Lin, Yijyun; Miao, Zehong; Shen, Jingkang; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1281-1297</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The signal transduction of acetylated histone can be processed through a recognition module, bromodomain.  Several inhibitors targeting BRD4, one of the bromodomain members, are in clin. trials as anticancer drugs.  Hereby, the authors report the efforts on discovery and optimization of a new series of 2-thiazolidinones as BRD4 inhibitors along the previous study.  In this work, guided by crystal structure anal., the authors reversed the sulfonamide group and identified a new binding mode.  A structure-activity relationship study on this new series led to several potent BRD4 inhibitors with IC50 of about 0.05-0.1 μM in FP binding assay and GI50 of 0.1-0.3 μM in cell based assays.  To complete the lead-like assessment of this series, the authors further checked its effects on BRD4 downstream protein c-Myc, investigated its selectivity among five different bromodomain proteins, as well as the metabolic stability test, and reinforced the utility of 2-thiazolidinone scaffold as BET bromodomain inhibitors in novel anticancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4x-5lE2iPzbVg90H21EOLACvtfcHk0lhLF1nGOE-WKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFSqtw%253D%253D&md5=4056ac1a6d6598dc61be7cc7ef82a3da</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm501504k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501504k%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.-L.%26aulast%3DXia%26aufirst%3DG.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.-C.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DFragment-based%2520drug%2520discovery%2520of%25202-thiazolidinones%2520as%2520BRD4%2520inhibitors%253A%25202.%2520Structure-based%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1281%26epage%3D1297%26doi%3D10.1021%2Fjm501504k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huo, F.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.-S.</span></span> <span> </span><span class="NLM_article-title">PAK5 promotes the migration and invasion of cervical cancer cells by phosphorylating SATB1</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">994</span>– <span class="NLM_lpage">1006</span>, <span class="refDoi"> DOI: 10.1038/s41418-018-0178-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=10.1038%2Fs41418-018-0178-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=30082769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGrs7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=994-1006&author=F.-C.+Huoauthor=Y.-J.+Panauthor=T.-T.+Liauthor=J.+Mouauthor=D.-S.+Pei&title=PAK5+promotes+the+migration+and+invasion+of+cervical+cancer+cells+by+phosphorylating+SATB1&doi=10.1038%2Fs41418-018-0178-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">PAK5 promotes the migration and invasion of cervical cancer cells by phosphorylating SATB1</span></div><div class="casAuthors">Huo, Fu-Chun; Pan, Yao-Jie; Li, Tong-Tong; Mou, Jie; Pei, Dong-Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Differentiation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">994-1006</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">P21-activated kinase 5 (PAK5) is involved in several oncogenic signaling pathways and its amplification or overexpression has been found in various types of cancer; however, the pathophysiol. role of PAK5 in cervical cancer (CC) remains elusive.  This study aims to elucidate the effects of PAK5 on CC metastasis and its specific regulation mechanism.  We performed western blotting and immunohistochem. (IHC) anal. and found that the expression levels of PAK5 were significantly upregulated in CC cells and tissues.  In addn., statistical anal. via IHC showed that increased PAK5 significantly correlated with CC progression.  Mn2+-Phos-tag SDS-PAGE, western blotting, immunofluorescence and dual luciferase reporter assays were utilized to det. the involvement of SATB1 in PAK5-mediated epithelial-mesenchymal transition (EMT).  We found that PAK5-mediated special AT-rich binding protein-1 (SATB1) phosphorylation on Ser47 initiated EMT cascade and promoted migration and invasion of CC cells.  Furthermore, overexpression of PAK5 induced lung metastasis of CC cells in xenograft modes.  Taken together, we conclude that PAK5 is a novel prognostic indicator and plays an important role in the CC metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo24tsedMNhbVg90H21EOLACvtfcHk0lhLF1nGOE-WKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGrs7fO&md5=52c3beae667f631c03592729b7785d60</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fs41418-018-0178-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41418-018-0178-4%26sid%3Dliteratum%253Aachs%26aulast%3DHuo%26aufirst%3DF.-C.%26aulast%3DPan%26aufirst%3DY.-J.%26aulast%3DLi%26aufirst%3DT.-T.%26aulast%3DMou%26aufirst%3DJ.%26aulast%3DPei%26aufirst%3DD.-S.%26atitle%3DPAK5%2520promotes%2520the%2520migration%2520and%2520invasion%2520of%2520cervical%2520cancer%2520cells%2520by%2520phosphorylating%2520SATB1%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2019%26volume%3D26%26spage%3D994%26epage%3D1006%26doi%3D10.1038%2Fs41418-018-0178-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIH','PDB','6LIH'); return false;">PDB: 6LIH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIM','PDB','6LIM'); return false;">PDB: 6LIM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI5','PDB','6JI5'); return false;">PDB: 6JI5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH','PDB','6GZH'); return false;">PDB: 6GZH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF','PDB','3MXF'); return false;">PDB: 3MXF</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i85"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00962">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_75541"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00962?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00962</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Kinase selectivity of compound <b>40</b>, CDK2 selectivity result of partial compounds, BET BDs selectivity of compound <b>40</b>, non-BET BDs selectivity of compound <b>40</b>, proliferation inhibition of 22 cancer cell lines of compound <b>40</b>, statistical information of crystallography, details of the docking study, superimposition of crystal structures of BI-2536 with compound 7, triple experiments of the western blot study of compounds <b>40</b>, and the spectra of typical synthesized compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Table of molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_002.csv">CSV</a>)</p></li><li><p class="inline">PDB files used in the docking study (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_003.zip">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_001.pdf">jm0c00962_si_001.pdf (1.07 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_002.csv">jm0c00962_si_002.csv (3.71 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00962/suppl_file/jm0c00962_si_003.zip">jm0c00962_si_003.zip (300.25 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Cocrystal structures of compounds <b>7</b> and <b>40</b> bound to BRD4-BD1 were deposited into the PDB database with the entry code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIH">6LIH</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LIM">6LIM</a>, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00962" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00962" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67994a40bf9d3de3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
