
<html lang="en"     class="pb-page"  data-request-id="7392f890-04f9-46a5-9c31-f2f410fe35c5"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.8b01732;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-22"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Why Some Targets Benefit from beyond Rule of Five Drugs" /></meta><meta name="dc.Creator" content="Megan  Egbert" /></meta><meta name="dc.Creator" content="Adrian  Whitty" /></meta><meta name="dc.Creator" content="György M.  Keserű" /></meta><meta name="dc.Creator" content="Sandor  Vajda" /></meta><meta name="dc.Description" content="Beyond rule-of-five (bRo5) compounds are increasingly used in drug discovery. Here we analyze 37 target proteins that have bRo5 drugs or clinical candidates. Targets can benefit from bRo5 drugs if ..." /></meta><meta name="Description" content="Beyond rule-of-five (bRo5) compounds are increasingly used in drug discovery. Here we analyze 37 target proteins that have bRo5 drugs or clinical candidates. Targets can benefit from bRo5 drugs if ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 12, 2019" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01732" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01732" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01732" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01732" /></link>
        
    
    

<title>Why Some Targets Benefit from beyond Rule of Five Drugs | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01732" /></meta><meta property="og:title" content="Why Some Targets Benefit from beyond Rule of Five Drugs" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0009.jpeg" /></meta><meta property="og:description" content="Beyond rule-of-five (bRo5) compounds are increasingly used in drug discovery. Here we analyze 37 target proteins that have bRo5 drugs or clinical candidates. Targets can benefit from bRo5 drugs if they have “complex” hot spot structure with four or more hots spots, including some strong ones. Complex I targets show positive correlation between binding affinity and molecular weight. These targets are conventionally druggable, but reaching additional hot spots enables improved pharmaceutical properties. Complex II targets, mostly protein kinases, also have strong hot spots but show no correlation between affinity and ligand molecular weight, and the primary motivation for creating larger drugs is to increase selectivity. Each target considered as complex III has some specific reason for requiring bRo5 drugs. Finally, targets with “simple” hot spot structure, i.e., three or fewer weak hot spots, must use larger compounds that interact with surfaces beyond the hot spot region to achieve acceptable affinity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01732"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01732">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01732&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01732&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01732&amp;href=/doi/10.1021/acs.jmedchem.8b01732" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 10005-10025</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.9b00004" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Why Some Targets Benefit from beyond Rule of Five Drugs</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Megan Egbert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Megan Egbert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Megan++Egbert">Megan Egbert</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Adrian Whitty</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adrian Whitty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Boston University, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>A.W.: phone, 1 617 353 2488; e-mail, <a href="/cdn-cgi/l/email-protection#44332c2d30303d0426316a212031"><span class="__cf_email__" data-cfemail="23544b4a57575a6341560d464756">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adrian++Whitty">Adrian Whitty</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0331-4369" title="Orcid link">http://orcid.org/0000-0002-0331-4369</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">György M. Keserű</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">György M. Keserű</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry Research Group, Research Center for Natural Sciences, Magyar Tudósok krt. 2, H-1117 Budapest, Hungary</div></div><span class="conrtib-corresp"><strong>*</strong>G.M.K: e-mail, <a href="/cdn-cgi/l/email-protection#9afde3b4f1ffe9ffe8efdaeeeef1b4f7eefbb4f2ef"><span class="__cf_email__" data-cfemail="24435d0a4f41574156516450504f0a4950450a4c51">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gy%C3%B6rgy+M.++Keser%C5%B1">György M. Keserű</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Sandor Vajda</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sandor Vajda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Chemistry, Boston University, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>S.V.: phone, 1 617 353 4757; e-mail, <a href="/cdn-cgi/l/email-protection#6e180f040a0f2e0c1b400b0a1b"><span class="__cf_email__" data-cfemail="0f796e656b6e4f6d7a216a6b7a">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sandor++Vajda">Sandor Vajda</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1540-8220" title="Orcid link">http://orcid.org/0000-0003-1540-8220</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01732&amp;href=/doi/10.1021%2Facs.jmedchem.8b01732" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 10005–10025</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 12, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 November 2018</li><li><span class="item_label"><b>Published</b> online</span>12 June 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01732" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01732</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10005%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMegan%2BEgbert%252C%2BAdrian%2BWhitty%252C%2BGy%25C3%25B6rgy%2BM.%2BKeser%25C5%25B1%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D22%26contentID%3Dacs.jmedchem.8b01732%26title%3DWhy%2BSome%2BTargets%2BBenefit%2Bfrom%2Bbeyond%2BRule%2Bof%2BFive%2BDrugs%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10025%26publicationDate%3DNovember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01732"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">6859</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">28</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01732" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Why Some Targets Benefit from beyond Rule of Five Drugs&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Megan&quot;,&quot;last_name&quot;:&quot;Egbert&quot;},{&quot;first_name&quot;:&quot;Adrian&quot;,&quot;last_name&quot;:&quot;Whitty&quot;},{&quot;first_name&quot;:&quot;György&quot;,&quot;last_name&quot;:&quot;M. Keserű&quot;},{&quot;first_name&quot;:&quot;Sandor&quot;,&quot;last_name&quot;:&quot;Vajda&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;10005-10025&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01732&quot;},&quot;abstract&quot;:&quot;Beyond rule-of-five (bRo5) compounds are increasingly used in drug discovery. Here we analyze 37 target proteins that have bRo5 drugs or clinical candidates. Targets can benefit from bRo5 drugs if they have “complex” hot spot structure with four or more hots spots, including some strong ones. Complex I targets show positive correlation between binding affinity and molecular weight. These targets are conventionally druggable, but reaching additional hot spots enables improved pharmaceutical properties. Complex II targets, mostly protein kinases, also have strong hot spots but show no correlation between affinity and ligand molecular weight, and the primary motivation for creating larger drugs is to increase selectivity. Each target considered as complex III has some specific reason for requiring bRo5 drugs. Finally, targets with “simple” hot spot structure, i.e., three or fewer weak hot spots, must use larger compounds that interact with surfaces beyond the hot spot region to achieve acceptable affinity.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01732&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01732" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01732&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01732" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01732&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01732" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01732&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01732&amp;href=/doi/10.1021/acs.jmedchem.8b01732" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01732" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01732" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01732%26sid%3Dliteratum%253Aachs%26pmid%3D31188592%26genre%3Darticle%26aulast%3DEgbert%26date%3D2019%26atitle%3DWhy%2BSome%2BTargets%2BBenefit%2Bfrom%2Bbeyond%2BRule%2Bof%2BFive%2BDrugs%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D22%26spage%3D10005%26epage%3D10025%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290901" title="Protein structure">Protein structure</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292601" title="Probes">Probes</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/jmcmar.2019.62.issue-22/20191127/jmcmar.2019.62.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0009.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Beyond rule-of-five (bRo5) compounds are increasingly used in drug discovery. Here we analyze 37 target proteins that have bRo5 drugs or clinical candidates. Targets can benefit from bRo5 drugs if they have “complex” hot spot structure with four or more hots spots, including some strong ones. Complex I targets show positive correlation between binding affinity and molecular weight. These targets are conventionally druggable, but reaching additional hot spots enables improved pharmaceutical properties. Complex II targets, mostly protein kinases, also have strong hot spots but show no correlation between affinity and ligand molecular weight, and the primary motivation for creating larger drugs is to increase selectivity. Each target considered as complex III has some specific reason for requiring bRo5 drugs. Finally, targets with “simple” hot spot structure, i.e., three or fewer weak hot spots, must use larger compounds that interact with surfaces beyond the hot spot region to achieve acceptable affinity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64170" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64170" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Lipinski’s rule of five (Ro5) provides simple conditions to determine if a chemical compound with a certain pharmacological or biological activity has properties consistent with being an orally active drug.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The aim of this rule has been to rationalize compound design (in particular compound library design) so as not to make molecules that are too apolar, floppy, and large and thus have a lower chance of exhibiting oral bioavailability and other desirable pharmaceutical properties. While the Ro5 has served as a very useful guideline for developing orally bioavailable small-molecule drugs and its importance is widely recognized, many compounds that violate the Ro5 are good drug candidates, and it is increasingly accepted that drug development efforts should not be restricted to the Ro5 chemical space.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> In particular, Doak et al.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> provided a comprehensive overview and discussion of close to 500 drugs and clinical candidates with MW ranging from 500 to 2000 Da, including discussions of structure–property trends. More recently, DeGoey et al.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> discussed learnings from AbbVie drug projects in beyond the Ro5 space. The increased interest in “beyond rule of five” (bRo5) compounds is driven by at least four observations. First, some natural products lie outside the Ro5 space but have fairly good oral availability, encouraging the development of bRo5 drug candidates, especially synthetic macrocycles.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9">(5−9)</a> Second, an increasing number of bRo5 compounds are in clinical trials and are becoming approved by the FDA.<a onclick="showRef(event, 'ref4 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref4 ref10 ref11 ref12">(4,10−12)</a> For example, over 30% of approved kinase inhibitors are bRo5 compounds.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Third, increasing attention is being given to small molecules that disrupt protein–protein interactions (PPIs),<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and around 50% of such compounds discussed in the scientific literature are bRo5.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17 ref18">(14−18)</a> Fourth, although oral bioavailability is an important criterion for most successful drugs, many therapeutics used in today’s medical practice are administered parenterally.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Thus, discovery efforts toward parenteral drugs may be useful to provide treatment options for indications with high unmet medical need or to validate a potential “first-in-class” target before intensive effort on oral therapeutics is put in place.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> In fact, many proteins have large binding sites, and the use of larger and more complex compounds can increase selectivity.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p">Recognizing the need for improved understanding of drugs and clinical candidates in the bRo5 space, Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> analyzed the interactions between a set of such compounds and their targets. The underlying assumption of their analysis was that these targets have special characteristics, such as possessing binding sites that are “large, highly lipophilic, highly polar, flexible, flat, or featureless”, and thus contain few opportunities for the binding of conventional Ro5-compliant ligands.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Accordingly, they selected 48 bRo5 drugs and their targets and analyzed the interactions within each specific ligand–receptor pair, focusing on pocket size and shape, buried ligand surface area, ligand interface, ligand and target interface nonpolar atom ratios, hydrogen bond donor and acceptor interactions, affinity, and ligand efficiencies.</div><div class="NLM_p last">While the results by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> already provide important guidance for the design of bioactive drugs for difficult targets, here we describe a substantially expanded analysis of these bRo5 targets and their available ligands. We studied only globular proteins, and after removing the DNA/RNA binding and membrane proteins from their set, we focused on 37 targets. First, we identified the binding hot spots of each protein target using FTMap.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Next, to study how ligands interact with these proteins, we extended the analysis beyond the specific protein–ligand pair Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> selected, to include all compounds, whether Ro5 or bRo5, that bind the target proteins and have structures available in the Protein Data Bank<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (PDB) and a known ligand binding affinity.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> Finally, we analyzed the binding site hot spots and ligand binding profiles of each target and classified them into four groups with different mechanisms explaining how bRo5 drugs increase the likelihood of success. In addition to the structure-based study of ligand binding to hot spots we also show results from a ligand-based analysis of the chemical tractability of the targets and the physicochemical properties of all their ligands collected from ChEMBL,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> to eliminate bias from our structure-based ligand data set.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Binding Site Analysis of bRo5 Protein Targets</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73500" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73500" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Analysis of the 37 target proteins was focused on the binding hot spots of the protein–ligand structure. Hot spots are small regions on the protein surface that contribute a disproportionate amount to the ligand-binding free energy, and hence ligands generally overlap with one or more hot spots.<a onclick="showRef(event, 'ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31">(28−31)</a> For determining the hot spots we used the protein mapping program FTMap.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> FTMap (<a href="http://ftmap.bu.edu/" class="extLink">http://ftmap.bu.edu/</a>) distributes small organic probe molecules of different sizes, shapes, and polarities on the surface of the protein to be studied, finds the most favorable positions for each probe type, clusters the probes, and ranks the clusters on the basis of their average energy. Regions that bind several different probe clusters are called consensus sites (CSs) and are the predicted binding hot spots. The hot spots predicted by FTMap have been shown to agree with the hot spots determined by a variety of experimental methods.<a onclick="showRef(event, 'ref32 ref33 ref34 ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34 ref35 ref36 ref37 ref38">(32−38)</a> Moreover, the strength of the hot spot, defined as the number of probe clusters included, indicates a relative energetic importance of the hot spot in the context of binding small molecules.<a onclick="showRef(event, 'ref32 ref33 ref34 ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34 ref35 ref36 ref37 ref38">(32−38)</a> Thus, the hot spots on each protein are ranked on the basis of the number of probe clusters they contain. Following the notation established in the FTMap server (<a href="http://ftmap.bu.edu" class="extLink">http://ftmap.bu.edu</a>), the hot spots are numbered starting from 0 for the strongest; the number of probe clusters in each hot spot is shown in parentheses after the hot spot rank. For example, a hot spot labeled 1(20) indicates that it is the second strongest hot spot (second to hot spot 0) and that it has a strength of 20 probe clusters. Moreover, a binding site of 0(20), 1(10), and 4(5) indicates that it includes three hot spots: 0, 1, and 4, each with 20, 10, and 5 probe clusters, respectively. Hot spots with fewer than five probe clusters were disregarded. We note that in all figures showing FTMap results, each probe cluster is represented by a single probe at the center of cluster.</div><div class="NLM_p">We have previously studied the hot spot properties on proteins that bind Ro5 drugs and demonstrated that a hot spot of at least 16 probe clusters is required for druggability.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Furthermore, traditional Ro5 druggable targets have one or more additional “secondary” hot spots (with three or more probe clusters) in close proximity to the strong “primary” hot spot.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> FTMap studies have also been performed for targets that bind macrocycles<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and inhibitors of protein–protein interactions.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> On the basis of these latter studies, we expected that bRo5 ligands would always be needed for targets with hot spots that are too far from each other or are too weak, in both cases necessitating larger ligands.<a onclick="showRef(event, 'ref5 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref5 ref39 ref40">(5,39,40)</a> As will be shown, some of the targets studied by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> indeed exhibit these properties. However, we found that the majority (22 out of 37) of the selected proteins that are targets for bRo5 drugs and clinical candidates can also bind small compounds (MW < 500 Da) with high affinity. In some cases, the smaller compounds can have even higher affinity than the larger ones. This observation raises the following questions: Which proteins require or benefit from bRo5 ligands, and can such targets be prospectively identified on the basis of the ligand-free protein structure?</div><div class="NLM_p">Among the 37 protein targets mapped here, we identified two distinct structures of hot spot ensembles in the binding sites. We will refer to these hot spot structures as “complex” and “simple” (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). A complex hot spot structure defines a binding site that consists of four or more hot spots, whereas the simple hot spot ensemble has three or fewer hot spots. On the basis of the number of hot spots alone, we have classified the 37 protein targets as complex or simple (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Twenty-four targets have a complex hot spot structure (mean number of hot spots = 5.63), and the remaining 13 have simple hot spot structure (mean number of hot spots = 2.15), resulting in significant difference (<i>p</i> < 0.001 by the Wilcoxon–Mann–Whitney test, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S1 in Supporting Information</a>). Furthermore, the total number of probe clusters in all hot spots in the binding site can also be used to help determine whether a given hot spot ensemble should be classified as complex or simple. The mean number of probe clusters in complex hot spot structures is 68.88, whereas in simple hot spot structures it is 29.88, and this difference is also significant (<i>p</i> < 0.001 by the Wilcoxon–Mann–Whitney test, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S1</a>). The consequence of complex versus simple hot spot structures will be discussed later in the context of the protein–ligand binding profiles.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures Mapped and Additional Ligand-Bound Structures Considered</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">MW < 500</th><th class="rowsep1 colsep0" colspan="2" align="center">MW > 500</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">target name</th><th class="colsep0 rowsep0" align="center">PDB code</th><th class="colsep0 rowsep0" align="center">chains</th><th class="colsep0 rowsep0" align="center">drug class</th><th class="colsep0 rowsep0" align="center">hot spot structure</th><th class="colsep0 rowsep0" align="center"><i>N</i><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>N</i><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><i>N</i><sub>HA</sub><a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center"><i>N</i><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><i>N</i><sub>HA</sub><a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">hot spots utilized by ligands</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HIV-1 protease/saquinavir</td><td class="colsep0 rowsep0" align="left">3OXC</td><td class="colsep0 rowsep0" align="left">A B</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">327</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">315</td><td class="colsep0 rowsep0" align="left">272</td><td class="colsep0 rowsep0" align="left">0 (21), 1 (18), 2 (18), 3 (13), 4 (11), 5 (8), 6 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">heat shock protein 90/PU-1471</td><td class="colsep0 rowsep0" align="left">2FWZ</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">155</td><td class="colsep0 rowsep0" align="left">147</td><td class="colsep0 rowsep0" align="left">58</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">0 (25), 1 (21), 2 (16), 4 (6), 5 (6), 6 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">thrombin/argatroban</td><td class="colsep0 rowsep0" align="left">1DWC</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">150</td><td class="colsep0 rowsep0" align="left">109</td><td class="colsep0 rowsep0" align="left">65</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">0 (26), 2 (14), 5 (6), 6 (6), 7 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MAP kinase P38/PF03715455</td><td class="colsep0 rowsep0" align="left">2YIS</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">144</td><td class="colsep0 rowsep0" align="left">117</td><td class="colsep0 rowsep0" align="left">66</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">0 (26), 1 (21), 2 (10), 3 (7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bromodomain BRD4/fedratinib</td><td class="colsep0 rowsep0" align="left">4OGJ</td><td class="colsep0 rowsep0" align="left">A B</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">113</td><td class="colsep0 rowsep0" align="left">103</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">0 (21), 1 (18), 2 (11), 3 (8), 4 (8), 5 (7), 6 (5), 7 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">renin/aliskiren</td><td class="colsep0 rowsep0" align="left">2V0Z</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">62</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">0 (24), 1 (15), 2 (12), 3 (8), 4 (8), 5 (7), 6 (7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PPAR-γ/AMG-131</td><td class="colsep0 rowsep0" align="left">3FUR</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">2 (9), 4 (8), 5 (8), 6 (7), 8 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MAP kinase kinase/cobimetinib</td><td class="colsep0 rowsep0" align="left">4AN2</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">0 (18), 1 (17), 2 (14), 3 (13), 4 (13)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">E3 ubiquitin-protein ligase XIAP/birinapant</td><td class="colsep0 rowsep0" align="left">4KMP</td><td class="colsep0 rowsep0" align="left">A B</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">0 (18), 1 (16), 2 (16), 3 (11), 4 (11), 5 (7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">epidermal growth factor receptor/lapatinib</td><td class="colsep0 rowsep0" align="left">1XKK</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">59</td><td class="colsep0 rowsep0" align="left">47</td><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">0 (16), 1 (15), 2 (12), 3 (9), 5 (7), 6 (7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hepatocyte growth factor receptor/BMS-777607</td><td class="colsep0 rowsep0" align="left">3F82</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">54</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">0 (14), 1 (12), 2 (11), 3 (10), 4 (9), 5 (8), 7 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">anaplastic lymphoma kinase/ceritinib</td><td class="colsep0 rowsep0" align="left">4MKC</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">0 (24), 3 (8), 5 (6), 6 (6), 7 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tyrosine protein kinase ABL1/nilotinib</td><td class="colsep0 rowsep0" align="left">3CS9</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">0 (20), 2 (15), 3 (14), 4 (13)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">VEGFR- 2/nintedanib</td><td class="colsep0 rowsep0" align="left">3C7Q</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">0 (20), 1 (17), 2 (10), 3 (9), 4 (8), 6 (8), 7 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">polo-like kinase 1/volasertib</td><td class="colsep0 rowsep0" align="left">3FC2</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">0 (28), 1 (19), 2 (12), 3 (7), 7 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">glucocorticoid receptor/mometasone furoate</td><td class="colsep0 rowsep0" align="left">4P6W</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">0 (25), 1 (21), 2 (13), 3 (7), 4 (7), 5 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cyclophilin A/cyclosporine A</td><td class="colsep0 rowsep0" align="left">1CWA</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">0 (27), 1 (16), 4 (11), 5 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Bcl-2/navitoclax</td><td class="colsep0 rowsep0" align="left">4LVT</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">0 (22), 2 (17), 4 (10), 5 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DOT1-like histone H3 methyltransferase/EPZ-5676</td><td class="colsep0 rowsep0" align="left">4HRA</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">1 (17), 2 (12), 3 (11), 4 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lanosterol 14-α demethylase/itraconazole</td><td class="colsep0 rowsep0" align="left">5EQB</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0 (23), 1 (17), 2 (10), 4 (7), 6 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">isoleucyl-tRNA synthetase/pseudomonic acid A</td><td class="colsep0 rowsep0" align="left">1QU2<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0 (14), 1 (14), 2 (14), 3 (11), 4 (8), 7 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FK506 binding protein/rapamycin</td><td class="colsep0 rowsep0" align="left">4DRI</td><td class="colsep0 rowsep0" align="left">A B</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">0 (27), 1 (20), 2 (16), 3 (13), 4 (6), 5 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Toll-like receptor 4/eritoran</td><td class="colsep0 rowsep0" align="left">2Z65</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0 (22), 1 (20), 2 (13), 3 (12), 4 (10), 5 (8), 6 (5), 7 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FGFR 1/BGJ-398</td><td class="colsep0 rowsep0" align="left">3TT0</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0 (18), 1 (17), 3 (12), 5 (6), 6 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">phosphoinositide-3 kinase/pictilisib (GDC-0941)</td><td class="colsep0 rowsep0" align="left">3DBS</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">65</td><td class="colsep0 rowsep0" align="left">53</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">4 (8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">soluble acetylcholine receptor/tubocurarine</td><td class="colsep0 rowsep0" align="left">3PMZ</td><td class="colsep0 rowsep0" align="left">D E</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">0 (19), 3 (13), 5 (9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein farnesyltransferase/lonafanib</td><td class="colsep0 rowsep0" align="left">1O5M</td><td class="colsep0 rowsep0" align="left">A B</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">0 (19), 2 (15)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">kinesin Eg5/ispinesib</td><td class="colsep0 rowsep0" align="left">4A5Y</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">0 (17), 2 (17), 3 (12)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HMG-CoA reductase/atorvastatin</td><td class="colsep0 rowsep0" align="left">1HWK</td><td class="colsep0 rowsep0" align="left">A B</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">0 (13), 1 (13), 6 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hepatitis C virus NS5b subunit/beclabuvir</td><td class="colsep0 rowsep0" align="left">4NLD</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">2 (10)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hepatitis C virus NS34A protease/simeprevir</td><td class="colsep0 rowsep0" align="left">3KEE</td><td class="colsep0 rowsep0" align="left">A E</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">1 (16), 3 (10)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DNA-directed RNA-polymerase/rifampicin</td><td class="colsep0 rowsep0" align="left">4KMU<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">0 (17)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tubulin-α chain/paclitaxel</td><td class="colsep0 rowsep0" align="left">1JFF</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">2 (16), 5 (8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">integrin α-IIB/eptifibatide</td><td class="colsep0 rowsep0" align="left">2VDN<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">2 (17)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Smoothened homolog/taladegib</td><td class="colsep0 rowsep0" align="left">4JKV<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">eRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0 (20), 1 (19), 3 (12)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Na–K ATPase/ouabain</td><td class="colsep0 rowsep0" align="left">3A3Y</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1 (17), 2 (12), 3 (8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α-amylase/β-acarbose</td><td class="colsep0 rowsep0" align="left">1PPI</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">bRo5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0 (18), 1 (16), 3 (11)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Indicates domain splitting was performed prior to FTMap (see <a class="ref internalNav" href="#sec9" aria-label="Methods">Methods</a>).</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Total number of ligand-bound structures with known binding affinity.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Number of ligand-bound structures with pBA < 7.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Number of high affinity structures with bound ligand for the <500 Da and >500 Da cases.</p></div></div></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Complex versus simple binding site hot spot structure, determined by FTMap. The hot spots are labeled according to rank (starting at 0) and colored according to the standard FTMap output (<a href="http://ftmap.bu.edu" class="extLink">http://ftmap.bu.edu</a>). The color of each hot spot ranked 0–7 is shown explicitly at the bottom of the figure. (a) A complex hot spot structure consists of four or more hot spots. Heat shock protein 90 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FWZ">2FWZ</a>) has the hot spots 0(25), 1(21), 2(16), 4(6), 5(6), and 6(5). The hot spots are labeled on the left, and the compound PU-H71 is overlaid on the right. b) A simple hot spot structure has three or fewer hot spots. HMG-CoA reductase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1HWK">1HWK</a>) has the hot spots 0(13), 1(13), and 6(6). The hot spots are labeled on the left, and atorvastatin is overlaid on the right.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Extended Ligand Data Set for bRo5 Protein Targets</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05352" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05352" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">While for each target Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> described only one particular bound drug or clinical candidate, to study how the protein can bind different ligands we analyzed all ligand-bound structures with at least 90% sequence identity that were available in the PDB<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and had binding affinity information.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> The number of additional ligand-bound structures varied from 0 to 327 per protein, resulting in an extended data set that includes 1499 ligand–protein complexes with structural data (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). All ligands with MW > 500 Da were labeled either “extended Ro5” (eRo5) or “beyond Ro5” (bRo5) as defined by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> (see <a class="ref internalNav" href="#sec9" aria-label="Methods">Methods</a>). The eRo5 category describes drugs that violate the Ro5 MW threshold of 500 Da but maintain MW < 700 Da and ClogP < 7.5 and otherwise satisfy all the Ro5 criteria. The bRo5 class describes ligands that have very large MW > 700 Da or violate any of the Ro5 criteria beyond the eRo5 limits. The original set selected by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> included 26 targets with a bound eRo5 ligand and 22 targets with a bRo5 ligand. However, we restricted consideration to globular proteins, and hence our set of targets contains 23 targets with eRo5 and 14 targets with bRo5 ligands. While this set is skewed toward eRo5 compounds, we think that it is still important to explore why drugs with MW > 500 Da are frequently more successful in clinical trials than smaller candidates that are also available for most of these targets.</div><div class="NLM_p last">All ligands have experimentally determined binding affinity (BA) values in the form of <i>K</i><sub>D</sub>, <i>K</i><sub>I</sub>, or IC<sub>50</sub>. These were converted to pBA values defined as pBA = −log <i>K</i><sub>D</sub>, −log <i>K</i><sub>I</sub>, or −log IC<sub>50</sub>, depending on which measure was reported. Ligands with pBA > 7 were considered high affinity. We note that the decision to restrict consideration to compounds with known binding affinity was adopted to exclude cofactors, fragments, and crystallization additives that frequently occur in X-ray crystal structures. The compounds in our extended set are restricted to those for which experimental crystal structures have been reported bound to the target protein, and so they clearly do not represent all ligands that bind to the 37 targets. Information gained from considering other known ligands with reported affinity values but no crystal structures is examined in the ligand-based analysis at the end of our study.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Targets with Complex Hot Spot Structure</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76766" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76766" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As discussed, the hot spot structure of the ligand binding site of a target is defined as complex if it includes four or more hot spots. Among the 37 proteins considered, 24 satisfy this condition. In addition, in most such targets the hot spot ensemble is strong; i.e., the ensemble includes at least one hot spot with 20 or more probe clusters (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). These complex targets were further classified based on the ligand-binding profile, where data are available, and primarily on the correlation between binding affinity and MW (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S1, Table S1</a>). For targets classified as complex I we observe a positive correlation between binding affinity and molecular weight of the ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S2</a>), whereas no such correlation exists for complex II targets (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S4</a>). This classification scheme is outlined in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and is further discussed in detail below. However, 8 of the 24 targets have too few ligands with structure and binding data (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). For each target in this group, defined as complex III, the motivation for developing eRo5/bRo5 drugs or drug candidates will be individually discussed.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Examples of characteristic structure-based ligand profiles for complex I, complex II, and simple protein targets that bind eRo5/bRo5 drugs and clinical candidates. Examples of complex I targets shown are the heat shock protein 90, thrombin, renin, and PPAR-γ. Examples of complex II targets shown are EGFR, HGFR, ABL1, and ALK. Examples of targets with simple hot spot structure are kinesin Eg5, soluble acetylcholine receptor, HMG-CoA reductase, and HCV NS5b. Only ligands that bind to the above proteins with known structure and binding affinity are shown (see <a class="ref internalNav" href="#sec9" aria-label="Methods">Methods</a>). Plots with ligands in blue (left) show the pBA [pBA = −log(<i>K</i><sub>D</sub> or <i>K</i><sub>I</sub> or IC<sub>50</sub>)] versus molecular weight, and plots with ligands in green (right) show pBA versus total number of probe clusters.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Complex I Targets</h3><div class="NLM_p">The nine targets in this group have multiple, strong hot spots at the binding site, and individual ligands interact with different combinations of the hot spots, ranging from a subset to all hot spots. These complex binding sites are very capable of binding small ligands (MW < 500 Da) with high affinity (pBA > 7) (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). However, the successful drugs and clinical candidates for these targets are large and classified as eRo5/bRo5. To understand why these larger candidates are more successful, we consulted the ligand binding profiles for each complex I target and observed that these targets exhibit a positive correlation between pBA and molecular weight and between pBA and number of probe clusters reached (<i>p</i> < 0.05, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1, Figure S2</a>). Thus, as ligands increase in molecular weight, they reach more hot spots and gain binding affinity (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S2</a>). Furthermore, as will be discussed, we notice that the approved drugs or promising clinical candidates frequently tend to lie at the higher end of the molecular weight spectrum and occupy all or almost all hot spots, in spite of the potential disadvantages in terms of compound developability of moving into the bRo5 range. In spite of these general characteristics, each target may have some specific properties motivating development of eRo5/bRo5 drugs as discussed below.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Thrombin</h4><div class="NLM_p">According to FTMap, thrombin has five binding hot spots 0(26), 2(14), 5(6), 6(6), and 7(5) in the inhibitor binding site, totaling 57 probe clusters, and exhibits the characteristic complex I positive correlations between pBA and MW (<i>p</i> < 0.001) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1 and Figure S2</a>). Overall, 150 ligands were identified to bind thrombin with known structure and affinity. Of the 95 that bound with high affinity, 65 have MW < 500 Da and 30 have MW > 500 Da. Argatroban, the bRo5 ligand discussed by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> is the only FDA approved nonpeptidic direct thrombin inhibitor. It reaches all five hot spots and has a molecular weight just above 500 Da (1DWC:MIT, MW = 509.64 Da, <i>K</i><sub>I</sub> = 39 nM, pBA = 7.41). Ligands exist that are much smaller, such as GR157368 (1QHR:157, MW = 226.31 Da, IC<sub>50</sub> = 130 nM, pBA = 6.89), but these bind only a subset of the hot spots and display somewhat lower affinity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). Two inhibitors that are quite large, but with MW slightly less than argatroban, inogatran (1K21:IGN, MW = 438.6 Da, <i>K</i><sub>I</sub> = 4.2 nM, pBA = 8.38) and ximelagatran (4BAH:MEL, MW = 473.6 Da, <i>K</i><sub>I</sub> = 2.01 nM, pBA = 8.7), bind to four of the five available hot spots, 0(26), 2(14), 5(6), and 6(6). While these two inhibitors had sufficiently high affinity, they were withdrawn from the clinic due to liver toxicity.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> In fact, many low MW thrombin inhibitors have problems, including poor selectivity, inherent toxicity, high-plasma protein binding, poor metabolic stability, rapid elimination from the blood, low anticoagulant activity, and poor oral bioavailability.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Thus, it appears that inhibitors that reach all hot spots of thrombin can have superior pharmaceutical properties in spite of extending into the bRo5 range. This is possibly because the large amount of binding energy that is potentially made available by exploiting all hot spots gives latitude to compromise the binding complementarity in certain locations on the compound, allowing stronger binding groups that bring pharmaceutical liabilities to be replaced with slightly weaker binding but more structurally benign functionality while still achieving sufficient overall binding affinity. In addition, the more complex structure of the drugs is likely to reduce the chances of off-target binding.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. FTMap determined hot spot structure of complex I targets with example ligands bound. Blue ligands represent the eRo5/bRo5 drug/clinical candidate identified by Doak et al., whereas pink ligands represent smaller compounds. FTMap hot spots are colored according to standard output, shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. (a) Hot spot structure of thrombin (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DWC">1DWC</a>). Argatroban (1DWC:MIT, MW = 509.64 Da, <i>K</i><sub>I</sub> = 39 nM, pBA = 7.41) and GR157368 (1QHR:157, MW = 226.31 Da, IC<sub>50</sub> = 130 nM, pBA = 6.89) are shown in blue and pink, respectively. (b) Hot spot structure of renin (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V0Z">2V0Z</a>). Aliskiren (2V0Z:C41, MW = 551.76 Da, <i>K</i><sub>I</sub> = 0.6 nM, pBA = 9.22) and <i>N</i>-piperidin-3-ylpyrimidine-5-carboxamide (5SZ9:74Y, MW = 313.39 Da, IC<sub>50</sub> = 38 μM, pBA = 4.42) are shown in blue and pink, respectively. (c) Hot spot structure of E3 ubiquitin-protein ligase XIAP (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KMP">4KMP</a>). Birinapant (4KMP:GT6, MW = 806.94 Da, <i>K</i><sub>D</sub> = 45 nM, pBA = 7.35) and (2JK7:BI6, MW = 486.61 Da, <i>K</i><sub>I</sub> = 67 nM, pBA = 7.17) are shown in blue and pink, respectively. (d) Hot spot structure of MAP kinase p38 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YIS">2YIS</a>). PF-03715455 (2YIS:YIS MW = 700.27 Da, IC<sub>50</sub> = 1.7 nM, pBA = 8.77) and 3P7B:P7B (MW = 464.58 Da, IC<sub>50</sub> = 18 nM, pBA = 7.74) are shown in blue and pink, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Renin</h4><div class="NLM_p">Renin is another typical complex I protein. 62 ligands were identified to bind renin with known structure and affinity, including high affinity ligands both with MW below and above 500 Da. Overall, there is a positive correlation between pBA and MW (<i>p</i> < 0.001 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1, Figure S2</a>). The bRo5 inhibitor aliskiren (2V0Z:C41, MW = 551.76 Da, <i>K</i><sub>I</sub> = 0.6 nM, pBA = 9.22), discussed by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> reaches six of the seven hot spots, 0(24), 1(15), 2(12), 3(8), 4(8), and 5(7), overlapping with a total of 74 probe clusters (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). Many peptidomimetic inhibitors achieve high affinity by binding to various subsets of these hot spots. For example, remikiren (3D91:REM, MW = 630.84, <i>K</i><sub>D</sub> = 131 pM, pBA = 9.88) binds to one fewer hot spot than aliskiren, as it does not reach hot spot 4(8). However, peptide-like compounds such as remikiren, enalkiren, and zanikiren are poorly absorbed and rapidly metabolized and hence were not successful in clinical development.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> In fact, aliskiren is the only FDA approved direct renin inhibitor available in the United States, and while it does not have the highest affinity, it does extend into a deep and narrow pocket to reach hot spot 1(15) and has somewhat better bioavailability than the peptidomimetics.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> Reaching all hot spots, including 1(15), in the inhibitor binding pocket seems to be one of the keys to the success of aliskiren, perhaps again because the extra binding energy thereby achieved provides scope to optimize regions of the inhibitor for improved pharmaceutical properties.</div><div class="NLM_p"><i>E3 ubiquitin-protein ligase XIAP</i> is a member of the “inhibitor of apoptosis protein” (IAP) family and is endogenously regulated by second mitochondrial activator of caspases (SMAC) via its binding to the BIR domain of XIAP. The bRo5 compound birinapant (4KMP:GT6, MW = 806.94 Da, <i>K</i><sub>D</sub> = 45 nM, pBA = 7.35), considered by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> is a bivalent SMAC-mimetic compound currently in clinical trials for the treatment of cancer (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). In order to understand the binding of bivalent inhibitors, we have mapped the XIAP dimer (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KMP">4KMP</a>). Birinapant reaches all FTMap identified hot spots, 0 (18), 1 (16), 2 (16), 3 (11), 4 (11), and 5 (7), thus stabilizing the dimer. However, most of the 20 compounds that bind to XIAP with known structure and affinity are monovalent SMAC-mimetic inhibitors and only reach the hot spots on one of the XIAP proteins. One such example is BI6 (2JK7:BI6, MW = 486.61 Da, <i>K</i><sub>I</sub> = 67 nM, pBA = 7.17), which binds to hot spots 0(18), 3(11), and 5(7) with relatively high affinity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). Overall, there is a positive correlation between MW and pBA for the 20 XIAP inhibitors identified, but high affinity is achieved with both small (MW < 500 Da) and eRo5/bRo5 compounds. All XIAP inhibitors, including birinapant, exhibit moderate selectivity and also bind to other members of the IAP family of proteins.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> It may not be necessary to bind all hot spots in XIAP to inhibit its activity; however, birinapant appears to be the most successful clinical candidate at this time. More generally, the monovalent inhibitors are approximately 100–1000 times less potent than the corresponding bivalent compounds at the cellular level.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div><div class="NLM_p last"><i>P38 mitogen activated protein kinase (p38 MAPK)</i> has a complex binding site consisting of four hot spots in the DFG-out conformation (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YIS">2YIS</a>) and shows a strong positive correlation between ligand pBA and MW (<i>p</i> < 0.001) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S1</a>). As will be discussed, this property makes the two MAP kinases (p38 MAPK and MEK1) considered here unique among the other kinases in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, since the latter exhibit no correlation between pBA and MW. In fact, the inhibitors of p38 MAPK and MEK1 are type III kinase inhibitors that bind to an allosteric site that is adjacent to the ATP-binding pocket, and the mode of binding is very different from those of the type I and type II inhibitors that bind to the ATP binding site in other kinases.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The bRo5 inhibitor selected by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> PF-03715455 (2YIS:YIS MW = 700.27 Da, IC<sub>50</sub> = 1.7 nM, pBA = 8.77) is a type III kinase inhibitor that binds to hot spots 1(21), 2(10), and 3(7) in the allosteric site and also reaches 0(26) near the ATP site (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d). Another type III p38 MAPK inhibitor, BIRB-796 (1KV2:B96 MW = 527.66 Da, <i>K</i><sub>D</sub> = 0.1 nM, pBA = 10) also binds to all four hot spots and achieves even higher affinity. While large inhibitors that bind to all four hot spots generally achieve the highest affinity, smaller inhibitors can bind a subset of the hot spots and still achieve fairly high affinity. For example, 3P7B:P7B (MW = 464.58 Da, IC<sub>50</sub> = 18 nM, pBA = 7.74) is a type III inhibitor that binds only the allosteric site in the DFG-out conformation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d). Other kinase inhibitors that only bind near the ATP binding site can also achieve a range of affinities, such as 3HVC:GG5 (MW = 239.25, <i>K</i><sub>I</sub> = 600 nM, pBA = 6.22) and neflamapimod (3HP5:52P MW = 436.26 Da, <i>K</i><sub>I</sub> = 0.8 nM, pBA = 9.09). While some high affinity inhibitors mentioned here (BIRB-796, neflamapimod) have advanced to clinical trials for inflammatory diseases, their progress has been hampered by adverse findings such as skin disorders, infection, and a lack of sustained efficacy.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The development of the bRo5 inhibitor PF-03715455, considered by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> was also discontinued<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> but for business reasons rather than for safety and/or efficacy concerns.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Thus, while p38 MAPK remains a challenging drug target, it appears that eRo5/bRo5 ligands which bind all hot spots, including those in the hydrophobic allosteric binding pocket, achieve the highest affinity and likely have improved selectivity as well.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> HIV-1 Aspartic Protease</h4><div class="NLM_p">This enzyme represents a special case of complex I targets. We found 327 compounds that bind HIV-1 protease with known structure and affinity. Of the compounds that bind with high affinity, 97.8% of ligands are large, having MW > 500 Da, and bind to all seven hot spots in the complex binding site. This is explained by recognizing that the binding site resides at a very flexible region of the dimer (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Specifically, FTMap of the protease (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OXC">3OXC</a>) reveals seven hot spots, 0(21), 1(18), 2(18), 3(13), 4(11), 5(8), and 6(5), and it appears that reaching all hot spots is beneficial for the stability of the flexible dimer.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> We identified only six ligands that bind with high affinity that have MW < 500 Da. One example is XK216 (1HWR:216, MW = 406.52, <i>K</i><sub>I</sub> = 4.6 nM, pBA = 8.34), an experimental cyclic urea inhibitor that binds all hot spots incredibly efficiently (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). Larger ligands, such as the FDA-approved saquinavir (3EL4:ROC, MW = 670.84 Da, <i>K</i><sub>D</sub> = 67.4 nM, pBA = 7.17), do not necessarily have higher affinity but do expand further into the binding site (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). On the basis of its hot spot structure, we consider HIV-1 protease to be in the complex I group of targets. In fact, due to the flexibility of the dimer, small inhibitors that do not reach all hot spots would certainly have low binding affinity, resulting in positive correlation between affinity and MW. However, because all biomedically relevant HIV-1 protease inhibitors are large and bind to all hot spots, such a correlation cannot be directly observed (<i>p</i> > 0.05; see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1</a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. HIV-1 protease binding hot spots. (a) FTMap hot spots of HIV-1 protease (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OXC">3OXC</a>): 0(21), 1(18), 2(18), 3(13), 2(111) 5(8), 6(5). (b) Hot spot structure of HIV-1 protease (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OXC">3OXC</a>). Saquinavir (3EL4:ROC, MW = 670.84 Da, <i>K</i><sub>D</sub> = 67.4 nM, pBA = 7.17) and XK216 (1HWR:216, MW = 406.52, <i>K</i><sub>I</sub> = 4.6 nM, pBA = 8.34) are shown in blue and pink, respectively. FTMap hot spots are colored according to standard output, shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. (c) Three plots showing data on ligands that bind HIV-1 protease with known structure and binding affinity. Ligands are distinguished as Ro5, eRo5, and bRo5 by the colors red, blue, and green, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Heat Shock Protein 90 (Hsp90)</h4><div class="NLM_p last">Current inhibitors of Hsp90 target the ATP-binding pocket in the N-terminal domain,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> which includes six hot spots: 0(25), 1(21), 2(16), 4(6), 5(6), and 6(5). The ligands that bind this pocket show a positive correlation between pBA and MW (<i>p</i> < 0.001, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1</a>), characteristic of complex I targets. Most inhibitors have been developed from geldanamycin (1YET:GDM, MW = 560.64 Da, <i>K</i><sub>D</sub> = 1.2 μM, pBA = 5.92), a macrocyclic natural product that binds to most of the hot spots (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S3</a>). Geldanamycin is an effective Hsp90 inhibitor but cannot be used in vivo due to hepatotoxicity.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The inhibitor PU-H71 (2FWZ:H71, MW = 512.37 Da, IC<sub>50</sub> = 50 nM, pBA = 7.30), considered by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> is a purine based analog, which is slightly smaller than geldanamycin, and reaches five hot spots: 0(25), 1(21), 2(16), 4(6), and 6(5) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S3</a>).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> PU-H71 has demonstrated marked specificity for malignant cells, displaying minimal toxicity to normal tissue. Although PU-H71 and many other inhibitors are in active oncology trials, there are currently no approved Hsp90 targeted drugs.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Mitogen-Activated Protein Kinase Kinase (MEK1)</h4><div class="NLM_p last">MEK1 has an allosteric binding site adjacent to the ATP binding pocket, similar to p38 MAPK, but can be accessed in the DFG-in conformation. All 25 ligands identified to bind MEK1 with known structure and affinity bind the DFG-in conformation, and among them 17 bind the allosteric pocket, which has three hot spots, 0(18), 1(17), and 2(14), while eight compounds primarily bind the ATP binding site, 3(13) and 4(13). Cobimetinib (4AN2:EUI, MW = 531.31 Da, IC<sub>50</sub> = 0.9 nM, pBA = 9.05), the inhibitor considered by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> is one of only three FDA approved type III kinase inhibitors, and it binds to all three hot spots in the allosteric binding site. Some smaller inhibitors such as 4ANB:YQY (MW = 477.22 Da, IC<sub>50</sub> = 6.6 nM, pBA = 8.8) bind with less but still high affinity via two hot spots in the allosteric site, 1(17) and 2(14), but have not been approved for clinical use. Two other inhibitors, trametinib (MW = 615.39 Da) and binimetinib (MW = 441.23 Da), have been approved by the FDA for the treatment of certain cancers, but there is no cocrystallized structure to determine how the drugs associate with the binding hot spots of MEK1. Notably, MEK1 and p38 MAPK differ from the other protein kinases, discussed as complex II targets, because they have an allosteric site adjacent to the ATP binding site, which enables inhibitors to bind with very high affinity, shown by the strong positive correlation between the pBA and MW of ligands (<i>p</i> < 0.001, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1</a>).</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Bromodomain-Containing Protein 4 (BRD4)</h4><div class="NLM_p last">The bromodomain family has emerged as a target for cancer therapy. BRD4 is able to bind traditional Ro5 compounds, dual kinase-bromodomain inhibitors, and bivalent inhibitors. Due to the presence of bivalent inhibitors, the hot spots in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> were obtained by considering BRD4 as a dimer. FTMap of the complex reveals three hot spots on each monomer, chain A with 0(21), 3(8), 6(5) and chain B with 1(18), 2(11), 5(7), and two hot spots in the interface of the dimer, 4(8) and 7(5). Taken together, the hot spot structure of BRD4 is incredibly complex. However, the binding cavity of a single chain in BRD4 on its own provides sufficient hot spots for Ro5 drugs such as the cell-permeable small molecule JQ1 (3MXF:JQ1, MW = 458.00 Da, <i>K</i><sub>D</sub> = 49 nM, pBA = 7.31), which exhibits high potency and specificity toward BRD4 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S3</a>).<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> A number of eRo5 compounds have been identified to bind BRD4 as well, including the dual BRD4-kinase inhibitor fedratinib (4OGJ:2TA, MW = 524.68 Da, <i>K</i><sub>D</sub> = 164 nM, pBA = 6.79) considered by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S3</a>). Most eRo5 bromodomain inhibitors are clinical kinase inhibitors that also inhibit bromodomains with therapeutically relevant potencies.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> An example of bivalent inhibitors is (5KHM:XNH, MW = 479.57 Da, <i>K</i><sub>I</sub> = 5 nM, pBA = 8.3), which binds two hot spots on each monomer (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S3</a>). Overall, there is a positive correlation between affinity and MW of compounds that bind BRD4 (<i>p</i> < 0.001), as with all targets classified as complex I. Bivalent inhibitors are able to achieve higher potency and specificity than both the dual BRD4-kinase inhibitors and traditional Ro5 compounds such as JQ1.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a></div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Peroxisome Proliferator-Activated Receptor γ (PPARγ)</h4><div class="NLM_p">The activity of PPAR-γ can be modulated with agonists, partial agonists, and antagonists. The protein is flexible, and the binding of the ligands affects the position of the activating helix (H12), which regulates the binding of cofactor proteins.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The impact of the ligand on the shape of the binding site is shown by the difference between mapping a ligand-bound structure and a ligand-free structure. FTMap of the structure with bound INT131 (3FUR:Z12, MW = 514.21 Da, <i>K</i><sub>I</sub> = 10 nM, pBA = 8), a selective partial agonist considered by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> identifies a very large binding site with five weak hot spots 2(9), 4(8), 5(8), 6(7), and 8(6), while the two strongest hot spots, 0(15) and 1(12), are located at H12, in the cofactor binding site (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S3</a>). In contrast, FTMap of the ligand-free PPAR-γ structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HWR">2HWR</a>) finds 10 hot spots: 0(23), 1(9), 2(9), 3(9), 4(9), 5(7), 6(6), 7(6), 8(6), and 9(6) in the binding site, emphasizing that the protein has a complex hot spot structure (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S3</a>). The best known PPAR-γ ligands are the thiazolidinedione (TZD) class of full agonists that include rosiglitazone (1ZGY:BRL, MW = 357.43 Da, <i>K</i><sub>I</sub> = 1.0 nM, pBA = 9) and pioglitazone (2XKW:P1B, MW = 356.44 Da, <i>K</i><sub>I</sub> = 1320 nM, pBA = 5.88).<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The TZDs have been clinically validated as antidiabetic agents, but their use has been associated with serious side effects, including weight gain, peripheral edema, hepatotoxicity, increased risk of congestive heart failure, and bone fracture. Although the maximal efficacies of the partial agonist, INT131, and full agonist, rosiglitazone, were similar with respect to improvements in glucose tolerance, INT131 had less effect on heart and lung weights, weight gain, hemodilution, and plasma volume. Thus, INT131 appears to selectively modulate PPAR-γ responses, showing antidiabetic efficacy while exhibiting an improved hemodynamic and cardiovascular adverse effect profile compared to the full agonists.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Hots Spots Based on the Mapping of Bound and Unbound Target Structures</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">bound</th><th class="rowsep1 colsep0" colspan="3" align="center">unbound</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">target name</th><th class="colsep0 rowsep0" align="center">hot spot structure</th><th class="colsep0 rowsep0" align="center">PDB code</th><th class="colsep0 rowsep0" align="center">chain</th><th class="colsep0 rowsep0" align="center">FTMap associated hot spots</th><th class="colsep0 rowsep0" align="center">PDB code</th><th class="colsep0 rowsep0" align="center">chain</th><th class="colsep0 rowsep0" align="center">FTMap associated hot spots</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">thrombin</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">1DWC</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0 (26), 2 (14), 5 (6), 6 (6), 7 (5)</td><td class="colsep0 rowsep0" align="left">1SGI</td><td class="colsep0 rowsep0" align="left">E</td><td class="colsep0 rowsep0" align="left">0 (28), 1 (16), 2 (15), 4 (11)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MAP kinase P38</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">2YIS</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (26), 1 (21), 2 (10), 3 (7)</td><td class="colsep0 rowsep0" align="left">4E5B</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (15), 1 (13), 2 (12), 5 (10)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">renin</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">2V0Z</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">0 (24), 1 (15), 2 (12), 3 (8), 4 (8), 5 (7), 6 (7)</td><td class="colsep0 rowsep0" align="left">1BBS</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (23), 1 (19), 2 (15), 3 (9), 4 (8), 5 (8), 6 (7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PPAR-γ</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">3FUR</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2 (9), 4 (8), 5 (8), 6 (7), 8 (6)</td><td class="colsep0 rowsep0" align="left">2HWR</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">0 (23), 1 (9), 2 (9), 3 (9), 4 (9), 5 (7), 6 (6), 7 (6), 8 (6), 9 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">epidermal growth factor receptor</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">1XKK</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (16), 1 (15), 2 (12), 3 (9), 5 (7), 6 (7)</td><td class="colsep0 rowsep0" align="left">4TKS</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (20), 1 (20), 2 (17), 3 (14), 4 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hepatocyte growth factor receptor</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">3F82</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (14), 1 (12), 2 (11), 3 (10), 4 (9), 5 (8), 7 (6)</td><td class="colsep0 rowsep0" align="left">1R1W</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (16), 2 (13), 3 (10), 5 (8), 6 (7), 7 (5), 9 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">anaplastic lymphoma kinase</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">4MKC</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (24), 3 (8), 5 (6), 6 (6), 7 (5)</td><td class="colsep0 rowsep0" align="left">4FNX</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (23), 1 (20)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tyrosine protein kinase ABL1</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">3CS9</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (20), 2 (15), 3 (14), 4 (13)</td><td class="colsep0 rowsep0" align="left">2HZ4</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">0 (26), 1 (19), 2 (11)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">VEGFR- 2</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">3C7Q</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (20), 1 (17), 2 (10), 3 (9), 4 (8), 6 (8), 7 (6)</td><td class="colsep0 rowsep0" align="left">1VR2</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (26), 1 (14), 2 (13), 3 (12), 4 (12), 5 (7), 6 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">polo-like kinase 1</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">3FC2</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (28), 1 (19), 2 (12), 3 (7), 7 (5)</td><td class="colsep0 rowsep0" align="left">2V5Q</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (25), 1 (15), 3 (11), 4 (9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cyclophilin A</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">1CWA</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (27), 1 (16), 4 (11), 5 (6)</td><td class="colsep0 rowsep0" align="left">5KV0</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (26), 2 (11), 3 (11), 6 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Bcl-2</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">4LVT</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (22), 2 (17), 4 (10), 5 (5)</td><td class="colsep0 rowsep0" align="left">1GJH</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (20), 1 (14), 4 (12)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">phosphoinositide-3 kinase</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">3DBS</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left">1E8Y</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (16)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein farnesyltransferase</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">1O5M</td><td class="colsep0 rowsep0" align="left">A B</td><td class="colsep0 rowsep0" align="left">0 (19), 2 (15)</td><td class="colsep0 rowsep0" align="left">1FT1</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">1 (17), 3 (16)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">kinesin Eg5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">4A5Y</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (17), 2 (17), 3 (12)</td><td class="colsep0 rowsep0" align="left">4A28</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">1 (19)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">jepatitis C virus NS5b subunit</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">4NLD</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2 (10)</td><td class="colsep0 rowsep0" align="left">2ZKU</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hepatitis C virus NS34A protease</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">3KEE</td><td class="colsep0 rowsep0" align="left">A E</td><td class="colsep0 rowsep0" align="left">1 (16), 3 (10)</td><td class="colsep0 rowsep0" align="left">1DXP</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tubulin-α chain</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">1JFF</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">2 (16), 5 (8)</td><td class="colsep0 rowsep0" align="left">2XRP</td><td class="colsep0 rowsep0" align="left">G</td><td class="colsep0 rowsep0" align="left">2 (17), 4 (10)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">integrin α-11B</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">2VDN<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">2 (17)</td><td class="colsep0 rowsep0" align="left">2VDL</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">3 (13)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Na–K ATPase</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">3A3Y</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">1 (17), 2 (12), 3 (8)</td><td class="colsep0 rowsep0" align="left">5AW8</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (10), 6 (5), 11 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α-amylase</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">1PPI</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">0 (18), 1 (16), 3 (11)</td><td class="colsep0 rowsep0" align="left">1KXV</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">1 (15), 2 (14), 3 (12)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Indicates domain splitting was performed prior to FTMap (see <a class="ref internalNav" href="#sec9" aria-label="Methods">Methods</a>).</p></div></div></div><div class="NLM_p last">The ligand binding profiles for all complex I targets are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S2</a>. Overall, complex I targets can bind small ligands with high affinity, but additional affinity can be gained by adopting eRo5/bRo5 ligands that reach additional hot spots. While the motivation for employing larger compounds is sometimes to improve potency, in many cases it appears instead to be for the purpose of gaining additional binding energy that can be traded off for improved pharmaceutical properties.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Complex II Targets</h3><div class="NLM_p">The complex II group comprises seven proteins that have complex hot spot structures and show no correlation between affinity and MW (<i>p</i> > 0.05), which distinguishes their ligand binding profiles from complex I targets (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1, Figure S4</a>). For these targets, high affinity binding is usually achieved at low ligand molecular weights (i.e., MW < 400 Da); however larger ligands have nonetheless been explored. Six of the seven complex II targets are kinases. As mentioned previously, over 30% of FDA approved kinase inhibitors have MW > 500 Da, placing them in the eRo5/bRo5 category.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Typically, small kinase inhibitors bind at the ATP binding site, which is present in the DFG-in conformation of the protein, whereas larger inhibitors often reach into the so-called “back pocket” present only in the DFG-out conformation. The lack of increase in binding affinity with increased molecular weight of the ligands for complex II targets indicates that the reason for using bRo5 ligands is not for improved affinity. Indeed, it is well recognized that the reason for developing eRo5/bRo5 inhibitors for kinases is most often to improve selectivity.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Since this statement applies to all kinases considered here in the complex II group, we discuss only three examples in detail, as well as the only non-kinase member of this group, the glucocorticoid receptor.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Tyrosine Protein Kinase ABL1</h4><div class="NLM_p">Inhibitors of ABL1 variants represent first-line therapy for most patients with chronic myelogenous leukemia (CML). We identified numerous ABL1 inhibitors, with MW values ranging from 372 to 595 Da, that bind either the DFG-in or DFG-out conformation. FTMap analysis of the DFG-out conformation (3CS9) shows four hot spots, two near the ATP binding site, 0(20) and 4(13), and two in the DFG back pocket, 2(15) and 3(14). In contrast, mapping the DFG-in conformation (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TWP">4TWP</a>) reveals five hot spots: 0(19), 2(11), 3(9), 4(7), and 6(5), all located near the ATP binding site. Thus, in the DFG-in conformation the kinase has a higher density of hot spots than in the DFG-out conformation (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a). Small ligands, such as the experimental inhibitor PPY-A (2Z60:P3Y, MW = 372.42, <i>K</i><sub>I</sub> = 20.0 nM, pBA = 7.70), can bind the DFG-in conformation with high affinity (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a), whereas larger drugs such as nilotinib (3CS9:NIL MW = 529.52 Da, <i>K</i><sub>D</sub> = 4.9 nM, pBA = 8.31), usually bind the DFG-out conformation (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a). Nilotinib utilizes all four hot spots in the DFG-out conformation by extending into the DFG-out pocket, but it achieves only slightly higher binding affinity than smaller inhibitors. Nilotinib is structurally related to imatinib, the first FDA approved type II kinase inhibitor. The ABL1 inhibitors dasatinib, bosutinib, and ponatinib, which were approved for the treatment of imatinib resistant or intolerant CML, also bind to all four hot spots in the DFG-out conformation and are bRo5 compounds. Thus, for ABL1, bRo5 compounds appear to be most successful, yet all have comparable affinity to the smaller inhibitors (pBA versus MW p-value >0.05) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1</a>). This may be attributed to the overall shift in hot spots from the ATP site in the DFG-in conformation to the back pocket in the DFG-out conformation (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. FTMap hot spot structure of kinase DFG-in versus DFG-out conformations. Hot spots in pink represent the FTMap results for the DFG-in conformation, whereas blue represents DFG-out hot spots. The protein cartoon shows the DFG-out conformation. (a) Hot spots of tyrosine protein kinase ABL1 in DFG-in (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TWP">4TWP</a>) and DFG-out (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS9">3CS9</a>) conformations. (b) Hot spots of anaplastic lymphoma kinase in DFG-in (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MKC">4MKC</a>) and DFG-out (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IUH">5IUH</a>) conformations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. FTMap determined hot spot structure of complex II targets with example ligands bound. Pink ligands represent smaller compounds, whereas blue ligands represent the eRo5/bRo5 drug/clinical candidate identified by Doak et al. White ligands (where applicable) show alternative compounds. FTMap hot spots are colored according to the standard output, as shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. (a) Hot spot structure of tyrosine protein kinase ABL1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS9">3CS9</a>). Nilotinib (3CS9:NIL MW = 529.52 Da, <i>K</i><sub>D</sub> = 4.9 nM, pBA = 8.31) and PPY-A (2Z60:P3Y, MW = 372.42, <i>K</i><sub>I</sub> = 20.0 nM, pBA = 7.70) are shown in blue and pink, respectively. (b) Hot spot structure of epidermal growth factor receptor (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK">1XKK</a>). Lapatinib (1XKK:FMM MW = 581.06 Da, <i>K</i><sub>I</sub> = 3 nM, pBA = 8.52) and 5EDR:5N4 (MW = 361.4 Da, <i>K</i><sub>I</sub> = 34.3 nM, pBA = 7.46) are shown in blue and pink, respectively. (c) Hot spot structure of anaplastic lymphoma kinase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MKC">4MKC</a>). Certinib (4MKC:4MK, MW = 558.13 Da, <i>K</i><sub>I</sub> = 3.7 μM, pBA = 5.43) and 7-azaindole based inhibitor (4JOA:3DK, MW = 390.39 Da, IC<sub>50</sub> = 29 nM, pBA = 7.54) are shown in blue and pink, respectively. (d) Hot spot structure of glucocorticoid receptor (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P6W">4P6W</a>). Mometasone furoate (4P6W:MOF, MW = 521.43 Da, <i>K</i><sub>I</sub> = 0.7 nM, pBA = 9.15), 1P93:DEX (MW = 392.46 Da, <i>K</i><sub>D</sub> = 19 nM, pBA = 7.72), and 3K23:JZN (MW = 655.68 Da, IC<sub>50</sub> = 6.3 nM, pBA = 8.2) are shown in blue, pink, and white, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Epidermal Growth Factor Receptor (EGFR)</h4><div class="NLM_p last">In the active, or DFG-in, conformation, the binding site of EGFR is complex and includes six hot spots, 0(16), 1(15), 2(12), 3(9), 5(7), and 6(7). Of the 59 compounds identified for EGFR, 55 bind the DFG-in conformation and 4 bind the DFG-out conformation. All structurally characterized ligands that bind in the DFG-in conformation associate with the same trio of hot spots 0(16), 1(15), 5(7) in the ATP site, and most extend in different directions to also engage with various combinations of the remaining hot spots or to regions between hot spots. For example, the smallest ligand identified (5EDR:5N4, MW = 361.4 Da, <i>K</i><sub>I</sub> = 34.3 nM, pBA = 7.46) binds only the core trio of hot spots (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b). The FDA approved drug lapatinib (1XKK:FMM MW = 581.06 Da, <i>K</i><sub>I</sub> = 3 nM, pBA = 8.52) also binds this core trio in the DFG-in conformation, but additionally extends to hot spot 3(9) and toward Arg 803 on the other side of the ATP binding site (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b). The four compounds that bind the DFG-out conformation (FTMap: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HG5">5HG5</a>) only bind the strong hot spot 0(25) in the ATP site and extend outward toward to the surface of the kinase. Thus, no additional hot spots are reached in these DFG-out binding compounds and overall, there is no correlation between affinity and MW (<i>p</i> > 0.05; see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1</a>). Because high affinity can be accomplished with very small ligands, we propose that the motivation for creating larger compounds for EGFR is to improve selectivity rather than affinity.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Anaplastic Lymphoma Kinase (ALK)</h4><div class="NLM_p">The clinical candidate certinib (4MKC:4MK, MW = 558.13 Da, <i>K</i><sub>I</sub> = 3.7 μM, pBA = 5.43) binds to the hot spots 0(24), 3(8), and 7(5) in the DFG-in conformation of ALK (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c). A smaller compound, a 7-azaindole based inhibitor (4JOA:3DK, MW = 390.39 Da, IC<sub>50</sub> = 29 nM, pBA = 7.54), also binds to the DFG-in conformation, but a slight conformational change enables the compound to extend deeper into the binding crevice toward hot spots 5(6) and 6(6), achieving fairly high affinity with a low molecular weight (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c). Larger inhibitors such as the pyrazolylamine derivative (5IUH:34Y, MW = 556.66 Da, IC<sub>50</sub> = 402 nM, pBA = 6.4) bind ALK in the DFG-out conformation, which opens a large back pocket for binding. FTMap analysis of ALK in the DFG-out conformation (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IUH">5IUH</a>) reveals five new hot spots, 0(19), 1(11), 2(11), 3(10), and 5(7) concentrated around the DFG-out back pocket, and retains one hot spot 4(8) in the ATP binding site (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b). Overall, the ligands that bind ALK do not show a positive correlation between pBA and MW (<i>p</i> > 0.05), consistent with other kinases (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1</a>). The shift in the locations and strengths of the binding hot spots on ALK between the DFG-in and DFG-out conformations is similar, but more exaggerated, than seen for ABL1 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Thus, a potential reason why a ligand that reaches the hot spots in the DFG-out pocket yields similar affinity to smaller ligands that only are concentrated around ATP binding site may be the weakened ATP binding site in the DFG-out conformation.</div><div class="NLM_p last">The kinases described above show that larger inhibitors generally do not result in improved binding affinity. Similar patterns exist for the additional kinases in our study, namely, the hepatocyte growth factor receptor (HGFR), the vascular endothelial growth factor receptor 2 (VEGFR-2), and polo-like kinase 1 (PLK1). The ligand binding profiles are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S4</a>, and corresponding statistics are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1</a>. On the basis of the observation that kinases can typically bind small ligands with high affinity and do not gain substantial additional affinity by adopting bRo5 ligands, we conclude that the reason for employing larger compounds is likely not for the purpose of improving potency. Instead, the motivation is to achieve greater selectivity, which is accomplished by occupying regions of the binding site that are unique to the protein of interest. Although the fibroblast growth factor receptor (FGFR) is a typical kinase, it has only one ligand with known structure and affinity and hence will be discussed in the complex III group. As mentioned previously, the inhibitors of MAP kinase P38 and MAP kinase kinase considered by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> are type III kinase inhibitors and bind very differently from the type I and type II inhibitors discussed for the other kinases here. In addition to kinases, we discuss the glucocorticoid receptor here because based on its binding profile, it belongs to the complex II group.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Glucocorticoid Receptor</h4><div class="NLM_p last">Glucocorticoid receptor (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P6W">4P6W</a>) has a complex hot spot structure with six hot spots, 0(25), 1(21), 2(13), 3(7), 4(7), and 5(6). We do not observe a correlation between pBA and MW (<i>p</i> > 0.05), and thus the binding profile of the glucocorticoid receptor is similar to those of kinases (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S4</a>). All ligands identified bind three core hot spots 0(25), 1(21), and 3(7), while larger ligands also extend to the more distant secondary hot spots 4(7) and 5(6) in one direction and 2(13) in another direction. For example, the small FDA-approved agonist of the glucocorticoid receptor, dexamethasone (1P93:DEX MW = 392.46 Da, <i>K</i><sub>D</sub> = 19 nM, pBA = 7.72), binds only to the three core hot spots (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>d). Dexamethasone is a typical corticosteroid with many side effects, including fluid retention, increased appetite, mood changes, skin rash, bruising or discoloration. The larger FDA-approved drug mometasone furoate (4P6W:MOF, MW = 521.43 Da, <i>K</i><sub>I</sub> = 0.7 nM, pBA = 9.15), which is used as a topical corticosteroid in ointments and nasal spray, extends beyond the three hot spots to overlap with hot spot 2(13) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>d). It has a glucocorticoid receptor binding affinity 22 times stronger than dexamethasone and higher than many other corticosteroids as well.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> A much larger compound (3K23:JZN, MW = 655.68 Da, IC<sub>50</sub> = 6.3 nM, pBA = 8.2), which also extends toward hot spot 2(13), similar to mometasone furoate, and toward hot spots 4(7) and 5(6), however, does not gain more affinity (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>d).</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Targets in the Complex III Group</h3><div class="NLM_p">Eight proteins with complex hot spot structures have been cocrystallized with fewer than nine ligands of known binding affinity, and hence we were unable to confidently construct ligand-binding profiles. However, the small ligand-binding profiles are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S5</a>, where possible. Although we believe that some of these targets behave similarly to complex I targets while others to complex II, without substantial binding profiles they cannot be added to these groups and hence are defined to form the complex III group. In addition, some of the targets can benefit from eRo5/bRo5 drugs or drug candidates for very specific reasons, and hence each of the nine targets is individually discussed here, without any attempt of trying to find an overarching motivation.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Cyclophilin A</h4><div class="NLM_p">Cyclophilin A has a complex hot spot structure, including four strong hot spots: 0(27), 1(16), 4(11), and 5(6). The drug considered by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> is cyclosporine A (1CWA:PRD_000142, MW = 1,202.61 Da, <i>K</i><sub>D</sub> = 36.8 nM, pBA = 7.43) which is a macrocycle that reaches all four hot spots and extends outward; the complex has been shown to inhibit calcineurin, acting as an immunosuppressant (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a).<a onclick="showRef(event, 'ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref63 ref64">(63,64)</a> Smaller compounds, such as (3RDD:EA4, MW = 251.28 Da, <i>K</i><sub>I</sub> = 16800 nM, pBA = 4.77), bind only three of the hot spots on cyclophilin A and have very weak affinity. In addition, such compounds do not competitively inhibit calcineurin and hence have no biological significance (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a). Reaching all four hot spots creates a span of 19 Å (between 0(27) and 5(6)), and requires a very long molecule. However, the main reason for the high MW is that the cyclophilin-bound cyclosporine directly contributes to the binding of calcineurin, and this function requires additional atoms expanding the macrocycle.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. FTMap determined hot spot structure of complex III targets with example ligands bound. Pink ligands represent smaller compounds, whereas blue ligands represent the eRo5/bRo5 drug/clinical candidateidentified by Doak et al. FTMap hot spots are colored according to standard output as shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. (a) Hot spot structure of cyclophilin A (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CWA">1CWA</a>). Cyclosporine A (1CWA:PRD_000142, MW = 1202.61 Da, <i>K</i><sub>D</sub> = 36.8 nM, pBA = 7.43) and 3RDD:EA4 (MW = 251.28 Da, <i>K</i><sub>I</sub> = 16800 nM, pBA = 4.77) are shown in blue and pink, respectively. (b) Hot spot structure of DOT1-like histone H3 methyltransferase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HRA">4HRA</a>). EPZ-5676 (4HRA:EP6, MW = 562.71 Da, <i>K</i><sub>I</sub> = 0.08 nM, pBA = 10.1) and 4EK9:EP4 (MW = 294.31 Da, <i>K</i><sub>I</sub> = 9 μM, pBA = 5.04) are shown in blue and pink, respectively. (c) Hot spot structure of lanosterol 14a-demethylase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EQB">5EQB</a>). Itraconazole (5EQB:1YN, MW = 705.63 Da, IC<sub>50</sub> = 19.4 nM, pBA = 7.71) and voriconazole (5HS1:VOR, MW = 349.31 Da, <i>K</i><sub>I</sub> = 174 μM, pBA = 3.76) are shown in blue and pink, respectively. (d) Hot spot structure of Bcl-2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LVT">4LVT</a>). Navitoclax (4LVT:1XJ, MW = 974.61 Da, <i>K</i><sub>I</sub> = 0.044 nM, pBA = 10.36) and 2W3L:DRO (MW = 576.09 Da, IC<sub>50</sub> = 30 nM, pBA = 7.52) are shown in blue and pink, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> DOT1-like Histone H3 Methyltransferase</h4><div class="NLM_p last">This protein binds a clinical candidate, EPZ-5676 (4HRA:EP6, MW = 562.71 Da, <i>K</i><sub>I</sub> = 0.08 nM, pBA = 10.1), with very high affinity at hot spots 1(17), 2(12), 3(11), and 4(6) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). While the primary hot spot of 17 probe clusters is slightly weaker than in typical complex targets, the hot spot structure is still relatively complex, with four hot spots. The smallest ligand, EPZ000004 (4EK9:EP4, MW = 294.31 Da, <i>K</i><sub>I</sub> = 9 μM, pBA = 5.04), only binds hot spots 2(12) and 4(6) and has very low affinity (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). A slightly larger ligand, <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine (3QOX:SAH, MW = 384.4 Da, <i>K</i><sub>I</sub> = 0.27 μM, pBA = 6.57), binds to three hot spots, 2(12), 3(11), and 4(6), with higher affinity. However, the three highest affinity inhibitors, including EPZ-5676 which is currently in clinical trials for advanced leukemia,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> bind to all four hot spots (46 total probe clusters), and all have MW > 500 Da. EPZ-5676 has both the highest MW and affinity. Thus, for this protein it appears that a bRo5 compound that binds all hot spots is required to achieve high affinity binding.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Lanosterol 14α-Demethylase</h4><div class="NLM_p last">Lanosterol 14α-demethylase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EQB">5EQB</a>) has a large, complex binding site consisting of five hot spots 0(23), 1(17) 2(10), 4(7), and 6(5) located inside a large cavity, with an additional three hot spots buried even deeper inside the cavity. The bRo5 drug identified by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> is itraconazole (5EQB:1YN, MW = 705.63 Da, IC<sub>50</sub> = 19.4 nM, pBA = 7.71), a high affinity antifungal medication<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> that binds to all five hot spots in the binding site (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>c). The distance between 4(7) and 6(5) is over 18.5 Å, too large to bridge by a Ro5 ligand. Of the three ligands found to bind lanosterol 14α-demethylase with known structure and affinity, itraconazole is the largest, protruding out of the cavity, and has the highest binding affinity. However, there exist smaller inhibitors such as fluconazole (4WMZ:TPF, MW = 306.27 Da, <i>K</i><sub>I</sub> = 30 μM, pBA = 4.52) that binds only to 2(10) and 4(7) and voriconazole (5HS1:VOR, MW = 349.31 Da, <i>K</i><sub>I</sub> = 174 μM, pBA = 3.76) that binds to 0(23), 2(10), and 6(5) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>c). All three are azole compounds, with the heterocyclic nitrogen atom binding to the heme iron atom and a halogenated phenyl ring interacting with a hydrophobic pocket of the enzyme binding site. Itraconazole has a broader spectrum of activity than fluconazole and is better for antifungal prophylaxis<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> but is less well tolerated.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> These differences are limited, and it is not clear whether itraconazole has advantages over the smaller compounds or simply was the first lanosterol 14α-demethylase inhibitor developed.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Bcl-2</h4><div class="NLM_p last">The unique hot spot structure of Bcl-2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LVT">4LVT</a>) has two groups of hot spots 0(22), 4(10) and 2(17), 5(5) separated from one another on the surface of Bcl-2 by approximately 16 Å, requiring a relatively large ligand in order to bind all hot spots. The FDA-approved navitoclax (4LVT:1XJ, MW = 974.61 Da, <i>K</i><sub>I</sub> = 0.044 nM, pBA = 10.36) binds all hot spots and achieves very high affinity (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>d). All but one of the compounds identified to bind Bcl-2 are very large, bridge the gap between the two sets of hot spots, and have high affinity (pBA > 7). We only identified one smaller inhibitor (2W3L:DRO, MW = 576.09 Da, IC<sub>50</sub> = 30 nM, pBA = 7.52) which binds half of the hot spots, 2(17) and 5(5), but MW > 500 Da and affinity is reduced relative to navitoclax (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>d).</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Isoleucyl-tRNA Synthetase</h4><div class="NLM_p last">Aminoacyl-tRNA synthetase is an enzyme that catalyzes the binding of an amino acid onto tRNA via ATP. The binding site of the isoleucyl-tRNA synthetase is a narrow crevice that accommodates both isoleucine and ATP<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> and contains six relatively weak hot spots 0(14), 1(14), 2(14), 3(11), 4(8), and 7(5). The inhibitor pseudomonic acid A (1QU2:MRC, MW = 500.6 Da, KD = 0.14 nM, pBA = 9.85), considered by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> binds to all six hot spots (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S5</a>). The maximum distance between hot spots is over 18 Å; thus a relatively long ligand is required to bind to all hot spots. In addition, the 9-hydroxynonanoic acid moiety of the pseudomonic acid A extends beyond the hot spot ensemble to a hydrophobic surface region, resulting in a molecule that is over 20 Å long. Nevertheless, it is almost within the 500 Da range.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> FK506 Binding Protein</h4><div class="NLM_p last">The FK506-binding protein 12 (FKBP12) binds the immunosuppressant drug rapamycin (4DRI:RAP, MW = 914.17 Da, <i>K</i><sub>I</sub> = 1.0 nM, pBA = 9), which works by inducing inhibitory protein complexes with the kinase mTOR.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> The hot spots between FKBP12 and mTOR are 0(27), 1(20), 2(16), 3(13), 4(6), and 5(5), and rapamycin interacts with all of them (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S5</a>). In principle, 0(27) and 4(6) alone may support a Ro5 ligand of FKBP12 with moderate binding affinity. However, the main biological function of rapamycin is to interact with both FKBP12 and the FRB fragment of mTOR, so additional moieties are needed for interaction with both proteins, and rapamycin is able to achieve that as a fairly large macrocyclic compound.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Toll-like Receptor 4</h4><div class="NLM_p last">Toll-like receptor 4 (TLR4) detects lipopolysaccharides found in most Gram-negative bacteria and hence plays a fundamental role in pathogen recognition and activation of innate immunity. Eritoran is a TLR4 antagonist (2Z65:E55, MW = 1313.7 Da, IC<sub>50</sub> = 1.5 nM, pBA = 8.82).<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> The natural substrates of TLR4 are large lipopolysaccharides that bind to four major hot spots 0(22), 1(20), 2(13), 3(12) in a deep cavity. Eritoran, a lipid derivative, interacts with these hots spots (and additional four hot spots) to compete with the substrate, resulting in a very large molecule with four long “arms” reaching into the hot spots (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S5</a>). Thus, in this case, it appears that a large compound may be needed to compete with the natural substrate.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Fibroblast Growth Factor Receptor 1 (FGFR1)</h4><div class="NLM_p last">Small molecular inhibitors target the ATP site of the kinase domain of FGFR1, which includes the hot spots 0(18), 1(17), 3(12), 5(6), and 6(5). A number of Ro5 kinase inhibitors such as dovitinib (5A46:38O, MW = 392.43 Da, no binding affinity data) strongly overlap with the hot spot 0(18) and lightly overlap with the hot spots 1(17), 3(12), and 5(6) located deeper in the ATP binding site. All such compounds showed toxicities related to VEGFR inhibition, such as hypertension, cardiovascular events, and proteinuria.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> In contrast, the compound studied by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> BGJ398 (3TT0:07J, MW = 560.5 Da, <i>K</i><sub>I</sub> = 13 nM, pBA = 7.89), associates with all hot spots, including a strong overlap with 1(17), 3(12), and 5(6) deep in the pocket, and is a selective fibroblast growth factor receptor (FGFR) inhibitor that does not bind to VEGFR.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Similar to other kinases, the use of an eRo5 compound substantially improves selectivity, but there are not enough compounds with structure and affinity data to generate a ligand binding profile.</div></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">Targets with Simple Hot Spot Structure</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72312" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72312" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Targets with “simple” hot spot structures have three or fewer hot spots in the ligand binding site. The total number of probes in the binding site is generally less than 50 probe clusters (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), and the primary hot spot is weak, with less than 20 probe clusters, which places the targets at or below the threshold for druggability, based on our previous study.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> These weak binding sites are often located in a deep cavity or narrow canyon or are otherwise isolated from additional hot spots. Generally, all ligands that bind these proteins must contact all hot spots in the ensemble and then reach outward for increased binding affinity and possibly increased selectivity by interacting with the surrounding surface regions, which do not have additional hot spots. As a result, some proteins with simple hot spot structure show a positive correlation between pBA versus MW while others do not (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S6, Table S1</a>).</div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Kinesin Eg5</h3><div class="NLM_p">The binding site of kinesin Eg5 consists of three hot spots, 0(17), 2(17), and 3(12), that lie close together in a fairly deep pocket. The primary hot spot has sufficient strength, at 17 probe clusters, to render the binding site weak but druggable.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> As a result of kinesin Eg5’s weak and compact hot spot ensemble, every ligand with known structure and affinity binds to all three hot spots in the binding site. Among these compounds, MW ranges from 274 to 517 Da, and a positive correlation exists between pBA and MW (<i>p</i> < 0.01; see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1</a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The smallest compound, (<i>S</i>)-enastron (2X7C:KZ9, MW = 274.34 Da, IC<sub>50</sub> = 2 μM, pBA = 5.7) lies within 3 Å of all three hot spots in the cavity but does not strongly overlap with the hot spots and has low affinity (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>a). However, the clinical candidate ispinesib (4A5Y:G7X, MW = 517.06 Da, <i>K</i><sub>I</sub> = 2.3 nM, pBA = 8.64), which has the largest molecular weight of all drugs identified for kinesin Eg5, has stronger overlap with the hot spots, extends deeper into the binding pocket, and interacts with the walls of the cavity, resulting in a much higher binding affinity (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>a). While we found only 28 kinesin Eg5 inhibitors with known structure and affinity, over 100 different inhibitors have been described in the scientific literature and there have been at least 35 phase 1 or phase 2 clinical trials initiated.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> Some of the compounds brought to clinical trials have MW < 500 Da, but ispinesib shows promise as being one of the most advanced drug candidates.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a></div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. FTMap determined hot spot structure of type III targets with example ligands bound. Pink ligands represent smaller compounds, whereas blue ligands represent the eRo5/bRo5 drug/clinical candidate identified by Doak et al. White ligands (where applicable) show compounds larger than the Doak et al. identified ligand. FTMap hot spots are colored according to standard output as shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. (a) Hot spot structure of kinesin Eg5 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4A5Y">4A5Y</a>). Ispinesib (4A5Y:G7X, MW = 517.06 Da, <i>K</i><sub>I</sub> = 2.3 nM, pBA = 8.64) and (<i>S</i>)-enastron (2X7C:KZ9, MW = 274.34 Da, IC<sub>50</sub> = 2 μM, pBA = 5.7) are shown in blue and pink, respectively. (b) Hot spot structure of soluble acetylcholine receptor (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PMZ">3PMZ</a>). Tubocurarine (3PMZ:TUB MW = 609.73 Da, <i>K</i><sub>I</sub> = 3481.0 nM, pBA = 5.46), 2WNL:AN5 (MW = 178.23 Da, <i>K</i><sub>I</sub> = 120 μM, pBA = 6), and pinnatoxin A (4XHE:40P, MW = 711.92 Da, <i>K</i><sub>D</sub> = 0.05 nM, pBA = 10.3) are shown in blue, pink, and white, respectively. (c) Hot spot structure of HMG-CoA reductase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1HWK">1HWK</a>). Atorvastatin (1HWK:117, MW = 558.64 Da, <i>K</i><sub>I</sub> = 6.2 nM, pBA = 8.21) and mevastatin (1HW8:114, MW = 408.53 Da, IC<sub>50</sub> = 23 nM, pBA = 7.64) are shown in blue and pink, respectively. (d) Hot spot structure of hepatitis C virus NS5b subunit (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NLD">4NLD</a>). Beclabuvir (4NLD:2N7, MW = 659.84 Da, IC<sub>50</sub> = 0.02 μM, pBA = 7.7) and 2WCX:VGC (MW = 339.45 Da, IC<sub>50</sub> = 0.025 μM, pBA = 7.6) are shown in blue and pink, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Soluble Acetylcholine Receptor</h3><div class="NLM_p last">Similar to kinesin Eg5, most ligands that bind the soluble acetylcholine receptor bind to all three hots spots in the site, 0(19), 3(13), and 5(9), totaling 41 probe clusters (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The site is located in a deep interchain crevice, and the larger ligands form significant interactions with the sides of the crevice, increasing the binding affinity, as shown by the positive correlation between pBA versus MW (<i>p</i> < 0.01; see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1</a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). One of the smallest compounds to bind (2WNL:AN5, MW = 178.23 Da, <i>K</i><sub>I</sub> = 120 μM, pBA = 6) only barely touches the three hot spots, whereas one of the largest compounds, the macrocyclic neurotoxin pinnatoxin A (4XHE:40P, MW = 711.92 Da, <i>K</i><sub>D</sub> < 0.05 nM, pBA = 10.3), extends deeper into the crevice and interacts significantly with the walls of the protein, resulting in a much higher binding affinity (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b). The FDA-approved inhibitor tubocurarine (3PMZ:TUB MW = 609.73 Da, <i>K</i><sub>I</sub> = 3.48 μM, pBA = 5.46) is a macrocycle that also extends into the crevice and interacts with the walls of the protein but with much weaker affinity (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b).</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> HMG-CoA Reductase</h3><div class="NLM_p last">As was the case with kinesin Eg5 and soluble acetylcholine receptor, all ligands of HMG-CoA reductase, the target of statins,<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> bind to the same three hot spots, 0(13), 1(13), and 6(6). The MW of the 19 HMG-CoA reductase ligands in the PDB ranges from 408 to 655 Da, and there is no correlation between pBA and MW (<i>p</i> > 0.05; see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1</a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The best known FDA-approved statin, atorvastatin (1HWK:117, MW = 558.64 Da, <i>K</i><sub>I</sub> = 6.2 nM, pBA = 8.21), interacts with the cavity walls and has acceptable binding affinity (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>c), similar to the other approved statins that have MWs between 420 and 580 Da. The smallest known statin, an experimental compound called mevastatin (1HW8:114, MW = 408.53 Da, IC<sub>50</sub> = 23 nM, pBA = 7.64), also binds to the three hot spots and extends up the cavity wall, resulting in sufficient binding affinity (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>c). However, the side effects of mevastatin include those of other statins, such as myalgias, abdominal pain, and nausea, and this compound additionally has a higher chance of giving more severe side effects related to myotoxicity and hepatotoxicity.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> The increased toxicity is most likely associated with reduced selectivity, and mevastatin was never marketed as a drug. We note that the inhibitor binding site of HMG-CoA reductase is very large and includes two additional hot spots, 4(9) and 5(7), that are far from the three main hot spots and are not utilized by any of the ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S7</a>). It is plausible, based on a structural analysis, that a very large ligand might extend to reach the additional hot spots at the other end of the cavity. However, since the additional hot spots are weak, doing so would likely result in at best a modest improvement in affinity, leaving little scope to trade excess affinity for improved pharmaceutical properties.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Hepatitis C Virus Subunits</h3><div class="NLM_p last">The hepatitis C virus (HCV) NS5b RNA polymerase has only one very weak hot spot, 2(10), located in a shallow cavity on the surface of the protein. There are no other hot spots nearby, and all eight ligands identified that bind to HCV NS5b with known structure and affinity overlap with this one hot spot (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S6</a>). The ligands’ MW ranges from 339 to 659 Da and extend around the hot spot in multiple directions, but overall there is no increase in pBA with MW (<i>p</i> > 0.05; <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1, Figure S6</a>). Small compounds, such as 2WCX:VGC (MW = 339.45 Da, IC<sub>50</sub> = 25 nM, pBA = 7.6), do not reach into the surrounding area to the same extent as larger ligands such as beclabuvir (4NLD:2N7, MW = 659.84 Da, IC<sub>50</sub> = 20 nM, pBA = 7.7), which is currently in phase II clinical trials (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>d). Sofosbuvir (MW = 529.45 Da) has been approved as a drug, but there is no structure of the sofosbuvir-HCV NS5B complex in the PDB. Similar to HCV NS5b RNA polymerase, HCV NS3/4a protease also presents a simple hot spot structure of 1(16) and 3(10), and drugs extend significantly outside the shallow binding pocket, yet there is no correlation between pBA and MW (<i>p</i> > 0.05; <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Table S1, Figure S6</a>). Notably, all five drugs identified to bind the NS3/4a subunit have MW > 740 Da and high affinity (pBA > 8.4; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S6</a>). In the case of these two targets, therefore, the motivation for using bRo5 ligands is not for greater affinity but likely to achieve greater selectivity by interacting with regions of the protein beyond the very simple hot spot ensemble.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Protein Farnesyltransferase</h3><div class="NLM_p last">Farnesyltransferase post-translationally modifies proteins by adding an isoprenoid lipid called a farnesyl group to the -SH of the cysteine near the end of target proteins to form a thioether linkage. This process causes farnesylated proteins to become membrane-associated due to the hydrophobic nature of the farnesyl group. Farnesyl transferase inhibitors inhibit farnesylation of a wide range of target proteins, including Ras. It is thought that these agents block Ras activation through inhibition of the enzyme farnesyl transferase, ultimately resulting in cell growth arrest. Lonafarnib (1O5M:336, MW = 638.8 Da, <i>K</i><sub>I</sub> = 8.3 nM, pBA = 8.08),<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> considered by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> is a farnesyltransferase inhibitor that has been investigated in a human clinical trial as a treatment for progeria. The hot spots in the binding site are 0(19) and 2(15), but 0(19) overlaps with the location of farnesyl diphosphate. The latter also interacts with most ligands, including lonafarnib. The current version of FTMap does not enable mapping in the presence of farnesyl diphosphate and hence may not provide valid information on the hot spots. However, we note that Ro5 farnesyltransferase inhibitors have also been developed. For example, tipifarnib (4LNG:JAN, MW = 489.4 Da, no binding affinity data) was submitted to FDA. Although the application was rejected, the drug is still under development.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Phosphoinositide-3 Kinase</h3><div class="NLM_p">Phosphoinositide-3 kinases (PI3Ks) are a family of enzymes involved in cellular functions such as cell growth, proliferation, differentiation, motility, survival, and intracellular trafficking, which in turn are involved in cancer. Several inhibitors of PI3Ks have been reported in the literature, such as wortmannin (1E7U:KWT, MW = 428.4 Da, <i>K</i><sub>I</sub> = 4.2 nM, pBA = 8.38).<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> While these compounds have been widely used to elucidate the functional role of PI3K, their toxicity and lack of selectivity have limited their therapeutic potential. A number of pharmaceutical companies have thus developed PI3K isoform-specific inhibitors that are FDA approved, including the PIK3CD inhibitor, idelalisib (4XE0:40L, MW = 415.42 Da, IC<sub>50</sub> = 19 nM, pBA = 7.72), and the dual PIK3CA and PIK3CD inhibitor, copanlisib (5G2N:6E2, MW = 480.52 Da, no binding affinity data). In contrast, pictilisib, a pan PI3K inhibitor considered by Doak et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> is still in clinical trials. All inhibitors have substantial toxicity issues, and it is not yet clear whether pictilisib (3DBS:GD9, MW = 513.63 Da, <i>K</i><sub>I</sub> = 3.0 nM, pBA = 8.52) will provide advantages.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> PI3K is a large protein that substantially differs from protein kinases. Mapping finds only the hot spot 4(8) at the ligand binding site, while the stronger hot spots are placed deeper in the protein. However, mapping the unbound structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1E8Y">1E8Y</a>) places 0(16) at the inhibitor binding site, but overall, the site is weak and needs fairly large molecules for strong binding.</div><div class="NLM_p last"><a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> lists six additional simple targets that have three or fewer ligands with known structure and affinity, and thus do not provide data for the construction of meaningful ligand binding profiles. In summary, the motivation for adopting bRo5 compounds for targets with simple hot spot structure can be to achieve improved affinity, selectivity, or perhaps other pharmaceutical properties by contacting additional regions of the protein around the relatively small and weak hot spot ensemble. Thus, while for complex I and complex II targets the use of bRo5 compounds was driven mainly by the need to improve selectivity or other pharmaceutical properties, for targets with simple hot spot structure bRo5 ligands may be a means to compensate for borderline or poor druggability. Importantly, simple and complex targets can easily be distinguished based on their binding site hot spot structure, as identified using FTMap.</div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">Conservation of Hot Spots: Bound versus Unbound Protein Structures</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53863" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53863" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The structural analysis performed above was completed by mapping the proteins from the protein–ligand complexes selected by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> However, the number, strength, and arrangement of binding hot spots are properties of the target protein itself and hence can be determined by the analysis of ligand-free protein structure. Here, the FTMap hot spot calculations were repeated with the unbound structures corresponding to the drug–target complexes selected by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> whenever such were available. Using a threshold of 95% sequence similarity, we identified unbound structures for 21 of the 37 protein targets and applied FTMap. Since ligand binding may promote pocket formation by induced fit or conformational selection, we expected to find weaker hot spots in the ligand-free structures, and a few cases confirmed this expectation (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). However, apart from a slight decrease in the number of probe clusters and minor shifts in hot spot locations, our results show remarkable conservation of major hot spots in ligand binding sites across the unbound and bound structures (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">Unbound structures were found for 12 proteins that have complex hot spot structures in the bound form. The “complex” classification was conserved in the unbound form for nine of these proteins: MAPK p38, thrombin, PPAR-γ, renin, cyclophilin A, EGFR, HGFR, VEGFR-2, and polo-like kinase 1. The “complex” classification of the hot spot structure was lost in three of the 12 targets, however, due to a drop in the number of hot spots below four. For ABL1 and ALK the number of hot spots in the binding site dropped from four to three and two hot spots, respectively, yet the primary hot spot remained strong, with a strength of 26 and 23 probe clusters, respectively. As a result, these targets are still distinct from those designated as having a “simple” hot spot structure because the primary hot spot is so strong, and most simple hot spot structures have only a weak (<20 probe clusters) primary hot spot (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In the case of Bcl-2, the number of hot spots dropped from 4 in the bound structure to 2 in the unbound structure, and the strength of the primary hot spot dropped from 22 probe clusters to 20 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). However, the geometry of the hot spots for Bcl-2 was conserved across the unbound and bound structures, as both show two distinct hot spot regions separated from each other by about 11 Å.</div><div class="NLM_p last">The nine targets with a “simple” hot spot structure that had also crystallized in the unbound form also showed good conservation of hot spots between the bound and unbound structures. Specifically, seven of these targets, PI3K, protein farnesyltransferase, kinesin Eg5, tubulin-α chain, integrin α-11-B, Na–K ATPase, and α-amylase, show very close conservation of hot spots, keeping one to three hot spots in total with the strongest hot spot below 20 probe clusters (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). However, the already weak hot spots for the two hepatitis C virus proteins, NS3/4a and NS5b, were lost in the unbound structure. Thus, the weak sites in these proteins remain weak or become even weaker in the unbound conformation, and their “simple” binding site classification is retained. Overall, while the agreement between FTMap results for unbound and bound structures is not perfect, in most cases the binding site structure is sufficiently conserved and is useful in determining if and how a target may benefit from bRo5 ligands.</div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">Ligand-Based Analysis</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16840" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16840" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The structure-based analysis suggests that most of the targets studied from Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> can bind both Ro5 and eRo5/bRo5 compounds with high affinity; however some of the larger compounds tend to be more successful in clinical trials for the reasons outlined above. A number of targets (11 of 37) have very limited structure-based ligand profiles, however, with fewer than five ligands identified, and thus analysis of their ligand binding profiles was not possible. To check whether these targets generally follow the same pattern of binding both Ro5 and eRo5/bRo5 ligands and are considered druggable, we pulled all ligands for each target from ChEMBL,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> including those that do not have a structure deposited in the PDB. The results of our ChEMBL ligand-based analysis are shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>; only targets with more than 40 high affinity ligands were considered. As the result of this condition, 8 of the 37 targets were excluded. Generally, the number of ligands in each ChEMBL profile greatly increased relative to the structure based profile, especially for the targets that had fewer than five ligands in the structure-based analysis, namely, FK506 binding protein and FGFR1 (complex III) and hepatitis C virus NS34A protease, integrin α-IIB, Smoothened homolog, and Na–K ATPase (simple).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Results of the ChEMBL-Based Ligand Analysis</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" colspan="2" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">Ro5 compliance</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">ligandable,<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a> LIGexp</th><th class="rowsep1 colsep0" colspan="2" align="center">tractable,<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a> LE/LLE</th><th class="rowsep1 colsep0" colspan="2" align="center">Ro5_3<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="rowsep1 colsep0" colspan="2" align="center">Ro5_4<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></th><th class="rowsep1 colsep0" colspan="2" align="center">MW > 500</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">target name</th><th class="colsep0 rowsep0" align="center">hot spot structure</th><th class="colsep0 rowsep0" align="center">total no. of ChEMBL ligands</th><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">%</th><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">%</th><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">%</th><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">%</th><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">%</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HIV-1 protease</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">5638</td><td class="colsep0 rowsep0" align="left">3561</td><td class="colsep0 rowsep0" align="left">63.16</td><td class="colsep0 rowsep0" align="left">75</td><td class="colsep0 rowsep0" align="left">2.11</td><td class="colsep0 rowsep0" align="left">1922</td><td class="colsep0 rowsep0" align="left">53.97</td><td class="colsep0 rowsep0" align="left">183</td><td class="colsep0 rowsep0" align="left">5.14</td><td class="colsep0 rowsep0" align="left">3166</td><td class="colsep0 rowsep0" align="left">88.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">heat shock protein 90</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">841</td><td class="colsep0 rowsep0" align="left">335</td><td class="colsep0 rowsep0" align="left">39.83</td><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="left">27.46</td><td class="colsep0 rowsep0" align="left">326</td><td class="colsep0 rowsep0" align="left">97.31</td><td class="colsep0 rowsep0" align="left">247</td><td class="colsep0 rowsep0" align="left">73.73</td><td class="colsep0 rowsep0" align="left">69</td><td class="colsep0 rowsep0" align="left">20.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">thrombin</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">5921</td><td class="colsep0 rowsep0" align="left">2297</td><td class="colsep0 rowsep0" align="left">38.79</td><td class="colsep0 rowsep0" align="left">1230</td><td class="colsep0 rowsep0" align="left">53.55</td><td class="colsep0 rowsep0" align="left">2051</td><td class="colsep0 rowsep0" align="left">89.29</td><td class="colsep0 rowsep0" align="left">1057</td><td class="colsep0 rowsep0" align="left">46.02</td><td class="colsep0 rowsep0" align="left">1012</td><td class="colsep0 rowsep0" align="left">44.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MAP kinase P38</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">4772</td><td class="colsep0 rowsep0" align="left">2548</td><td class="colsep0 rowsep0" align="left">53.39</td><td class="colsep0 rowsep0" align="left">408</td><td class="colsep0 rowsep0" align="left">16.01</td><td class="colsep0 rowsep0" align="left">2186</td><td class="colsep0 rowsep0" align="left">85.79</td><td class="colsep0 rowsep0" align="left">1509</td><td class="colsep0 rowsep0" align="left">59.22</td><td class="colsep0 rowsep0" align="left">506</td><td class="colsep0 rowsep0" align="left">19.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bromodomain BRD4</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">1088</td><td class="colsep0 rowsep0" align="left">222</td><td class="colsep0 rowsep0" align="left">20.40</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">7.66</td><td class="colsep0 rowsep0" align="left">220</td><td class="colsep0 rowsep0" align="left">99.1</td><td class="colsep0 rowsep0" align="left">149</td><td class="colsep0 rowsep0" align="left">67.12</td><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left">13.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">renin</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">4038</td><td class="colsep0 rowsep0" align="left">2813</td><td class="colsep0 rowsep0" align="left">69.66</td><td class="colsep0 rowsep0" align="left">392</td><td class="colsep0 rowsep0" align="left">13.94</td><td class="colsep0 rowsep0" align="left">1665</td><td class="colsep0 rowsep0" align="left">59.19</td><td class="colsep0 rowsep0" align="left">276</td><td class="colsep0 rowsep0" align="left">9.81</td><td class="colsep0 rowsep0" align="left">2413</td><td class="colsep0 rowsep0" align="left">85.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PPAR-γ</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">2134</td><td class="colsep0 rowsep0" align="left">917</td><td class="colsep0 rowsep0" align="left">42.97</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.11</td><td class="colsep0 rowsep0" align="left">405</td><td class="colsep0 rowsep0" align="left">44.17</td><td class="colsep0 rowsep0" align="left">108</td><td class="colsep0 rowsep0" align="left">11.78</td><td class="colsep0 rowsep0" align="left">531</td><td class="colsep0 rowsep0" align="left">57.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MAP kinase kinase</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">867</td><td class="colsep0 rowsep0" align="left">461</td><td class="colsep0 rowsep0" align="left">53.17</td><td class="colsep0 rowsep0" align="left">150</td><td class="colsep0 rowsep0" align="left">32.54</td><td class="colsep0 rowsep0" align="left">416</td><td class="colsep0 rowsep0" align="left">90.24</td><td class="colsep0 rowsep0" align="left">263</td><td class="colsep0 rowsep0" align="left">57.05</td><td class="colsep0 rowsep0" align="left">176</td><td class="colsep0 rowsep0" align="left">38.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">E3 ubiquitin-protein ligase XIAP</td><td class="colsep0 rowsep0" align="left">complex I</td><td class="colsep0 rowsep0" align="left">1168</td><td class="colsep0 rowsep0" align="left">308</td><td class="colsep0 rowsep0" align="left">26.37</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">4.55</td><td class="colsep0 rowsep0" align="left">174</td><td class="colsep0 rowsep0" align="left">56.49</td><td class="colsep0 rowsep0" align="left">63</td><td class="colsep0 rowsep0" align="left">20.45</td><td class="colsep0 rowsep0" align="left">242</td><td class="colsep0 rowsep0" align="left">78.57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">epidermal growth factor receptor</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">7047</td><td class="colsep0 rowsep0" align="left">2959</td><td class="colsep0 rowsep0" align="left">41.99</td><td class="colsep0 rowsep0" align="left">504</td><td class="colsep0 rowsep0" align="left">17.03</td><td class="colsep0 rowsep0" align="left">2673</td><td class="colsep0 rowsep0" align="left">90.33</td><td class="colsep0 rowsep0" align="left">1963</td><td class="colsep0 rowsep0" align="left">66.34</td><td class="colsep0 rowsep0" align="left">801</td><td class="colsep0 rowsep0" align="left">27.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hepatocyte growth factor receptor</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">3864</td><td class="colsep0 rowsep0" align="left">2382</td><td class="colsep0 rowsep0" align="left">61.65</td><td class="colsep0 rowsep0" align="left">755</td><td class="colsep0 rowsep0" align="left">31.7</td><td class="colsep0 rowsep0" align="left">2042</td><td class="colsep0 rowsep0" align="left">85.73</td><td class="colsep0 rowsep0" align="left">1316</td><td class="colsep0 rowsep0" align="left">55.25</td><td class="colsep0 rowsep0" align="left">967</td><td class="colsep0 rowsep0" align="left">40.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">anaplastic lymphoma kinase</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">1688</td><td class="colsep0 rowsep0" align="left">1020</td><td class="colsep0 rowsep0" align="left">60.43</td><td class="colsep0 rowsep0" align="left">151</td><td class="colsep0 rowsep0" align="left">14.8</td><td class="colsep0 rowsep0" align="left">835</td><td class="colsep0 rowsep0" align="left">81.86</td><td class="colsep0 rowsep0" align="left">392</td><td class="colsep0 rowsep0" align="left">38.43</td><td class="colsep0 rowsep0" align="left">584</td><td class="colsep0 rowsep0" align="left">57.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tyrosine protein kinase ABL1</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">2032</td><td class="colsep0 rowsep0" align="left">1084</td><td class="colsep0 rowsep0" align="left">53.35</td><td class="colsep0 rowsep0" align="left">401</td><td class="colsep0 rowsep0" align="left">36.99</td><td class="colsep0 rowsep0" align="left">963</td><td class="colsep0 rowsep0" align="left">88.84</td><td class="colsep0 rowsep0" align="left">678</td><td class="colsep0 rowsep0" align="left">62.55</td><td class="colsep0 rowsep0" align="left">296</td><td class="colsep0 rowsep0" align="left">27.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">VEGFR- 2</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">8031</td><td class="colsep0 rowsep0" align="left">3894</td><td class="colsep0 rowsep0" align="left">48.49</td><td class="colsep0 rowsep0" align="left">688</td><td class="colsep0 rowsep0" align="left">17.67</td><td class="colsep0 rowsep0" align="left">3585</td><td class="colsep0 rowsep0" align="left">92.06</td><td class="colsep0 rowsep0" align="left">2597</td><td class="colsep0 rowsep0" align="left">66.69</td><td class="colsep0 rowsep0" align="left">839</td><td class="colsep0 rowsep0" align="left">21.55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">polo-like kinase 1</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">737</td><td class="colsep0 rowsep0" align="left">281</td><td class="colsep0 rowsep0" align="left">38.13</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">12.1</td><td class="colsep0 rowsep0" align="left">254</td><td class="colsep0 rowsep0" align="left">90.39</td><td class="colsep0 rowsep0" align="left">161</td><td class="colsep0 rowsep0" align="left">57.3</td><td class="colsep0 rowsep0" align="left">105</td><td class="colsep0 rowsep0" align="left">37.37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">glucocorticoid receptor</td><td class="colsep0 rowsep0" align="left">complex II</td><td class="colsep0 rowsep0" align="left">3510</td><td class="colsep0 rowsep0" align="left">2584</td><td class="colsep0 rowsep0" align="left">73.62</td><td class="colsep0 rowsep0" align="left">221</td><td class="colsep0 rowsep0" align="left">8.55</td><td class="colsep0 rowsep0" align="left">2146</td><td class="colsep0 rowsep0" align="left">83.05</td><td class="colsep0 rowsep0" align="left">930</td><td class="colsep0 rowsep0" align="left">35.99</td><td class="colsep0 rowsep0" align="left">752</td><td class="colsep0 rowsep0" align="left">29.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cyclophilin A</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">231</td><td class="colsep0 rowsep0" align="left">74</td><td class="colsep0 rowsep0" align="left">32.03</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1.35</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">16.22</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">13.51</td><td class="colsep0 rowsep0" align="left">59</td><td class="colsep0 rowsep0" align="left">79.73</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Bcl-2</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">865</td><td class="colsep0 rowsep0" align="left">427</td><td class="colsep0 rowsep0" align="left">49.36</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">109</td><td class="colsep0 rowsep0" align="left">25.53</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">9.6</td><td class="colsep0 rowsep0" align="left">361</td><td class="colsep0 rowsep0" align="left">84.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DOT1-like histone H3 Me-transferase</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">42</td><td class="colsep0 rowsep0" align="left">42.00</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">66.67</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">95.24</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">11.9</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">80.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FK506 binding protein</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">491</td><td class="colsep0 rowsep0" align="left">156</td><td class="colsep0 rowsep0" align="left">31.77</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">2.56</td><td class="colsep0 rowsep0" align="left">62</td><td class="colsep0 rowsep0" align="left">39.74</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">20.51</td><td class="colsep0 rowsep0" align="left">110</td><td class="colsep0 rowsep0" align="left">70.51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FGFR 1</td><td class="colsep0 rowsep0" align="left">complex III</td><td class="colsep0 rowsep0" align="left">1605</td><td class="colsep0 rowsep0" align="left">661</td><td class="colsep0 rowsep0" align="left">41.18</td><td class="colsep0 rowsep0" align="left">87</td><td class="colsep0 rowsep0" align="left">13.16</td><td class="colsep0 rowsep0" align="left">624</td><td class="colsep0 rowsep0" align="left">94.4</td><td class="colsep0 rowsep0" align="left">369</td><td class="colsep0 rowsep0" align="left">55.82</td><td class="colsep0 rowsep0" align="left">270</td><td class="colsep0 rowsep0" align="left">40.85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">phosphoinositide-3 kinase</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">1912</td><td class="colsep0 rowsep0" align="left">704</td><td class="colsep0 rowsep0" align="left">36.82</td><td class="colsep0 rowsep0" align="left">315</td><td class="colsep0 rowsep0" align="left">44.74</td><td class="colsep0 rowsep0" align="left">686</td><td class="colsep0 rowsep0" align="left">97.44</td><td class="colsep0 rowsep0" align="left">515</td><td class="colsep0 rowsep0" align="left">73.15</td><td class="colsep0 rowsep0" align="left">161</td><td class="colsep0 rowsep0" align="left">22.87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein farnesyltransferase</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">2411</td><td class="colsep0 rowsep0" align="left">1204</td><td class="colsep0 rowsep0" align="left">49.94</td><td class="colsep0 rowsep0" align="left">373</td><td class="colsep0 rowsep0" align="left">30.98</td><td class="colsep0 rowsep0" align="left">1070</td><td class="colsep0 rowsep0" align="left">88.87</td><td class="colsep0 rowsep0" align="left">570</td><td class="colsep0 rowsep0" align="left">47.34</td><td class="colsep0 rowsep0" align="left">543</td><td class="colsep0 rowsep0" align="left">45.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">kinesin Eg5</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">1060</td><td class="colsep0 rowsep0" align="left">396</td><td class="colsep0 rowsep0" align="left">37.36</td><td class="colsep0 rowsep0" align="left">105</td><td class="colsep0 rowsep0" align="left">26.52</td><td class="colsep0 rowsep0" align="left">362</td><td class="colsep0 rowsep0" align="left">91.41</td><td class="colsep0 rowsep0" align="left">287</td><td class="colsep0 rowsep0" align="left">72.47</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="left">13.89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HMG-CoA reductase</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">287</td><td class="colsep0 rowsep0" align="left">199</td><td class="colsep0 rowsep0" align="left">69.34</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">13.07</td><td class="colsep0 rowsep0" align="left">154</td><td class="colsep0 rowsep0" align="left">77.39</td><td class="colsep0 rowsep0" align="left">91</td><td class="colsep0 rowsep0" align="left">45.73</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">32.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hepatitis C virus NS5b subunit</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">1267</td><td class="colsep0 rowsep0" align="left">302</td><td class="colsep0 rowsep0" align="left">23.84</td><td class="colsep0 rowsep0" align="left">57</td><td class="colsep0 rowsep0" align="left">18.87</td><td class="colsep0 rowsep0" align="left">227</td><td class="colsep0 rowsep0" align="left">75.17</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">42.38</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">42.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hepatitis C virus NS34A protease</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">498</td><td class="colsep0 rowsep0" align="left">254</td><td class="colsep0 rowsep0" align="left">51.00</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.39</td><td class="colsep0 rowsep0" align="left">212</td><td class="colsep0 rowsep0" align="left">83.46</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.39</td><td class="colsep0 rowsep0" align="left">249</td><td class="colsep0 rowsep0" align="left">98.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">integrin α-IIB</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">2436</td><td class="colsep0 rowsep0" align="left">1027</td><td class="colsep0 rowsep0" align="left">42.16</td><td class="colsep0 rowsep0" align="left">613</td><td class="colsep0 rowsep0" align="left">59.69</td><td class="colsep0 rowsep0" align="left">779</td><td class="colsep0 rowsep0" align="left">75.85</td><td class="colsep0 rowsep0" align="left">545</td><td class="colsep0 rowsep0" align="left">53.07</td><td class="colsep0 rowsep0" align="left">415</td><td class="colsep0 rowsep0" align="left">40.41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Smoothened homolog</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">710</td><td class="colsep0 rowsep0" align="left">340</td><td class="colsep0 rowsep0" align="left">47.89</td><td class="colsep0 rowsep0" align="left">59</td><td class="colsep0 rowsep0" align="left">17.35</td><td class="colsep0 rowsep0" align="left">316</td><td class="colsep0 rowsep0" align="left">92.94</td><td class="colsep0 rowsep0" align="left">261</td><td class="colsep0 rowsep0" align="left">76.76</td><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left">11.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Na–K ATPase</td><td class="colsep0 rowsep0" align="left">simple</td><td class="colsep0 rowsep0" align="left">223</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="left">24.66</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">47</td><td class="colsep0 rowsep0" align="left">85.45</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">43.64</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Ligandable: number, percent of compounds with p<i>K</i><sub>I</sub> > 7.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Tractable: number, percent of compounds with LE > 0.3 and LLE > 5, percent based on total ligandable.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Ro5_3: number of ligands meeting 3 of the 4 Ro5 criteria.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Ro5_4: number of ligands meeting 4 of the 4 Ro5 criteria, percents based on total ligandable.</p></div></div></div><div class="NLM_p">Overall, the ChEMBL ligand analysis confirms that the targets selected by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> bind both Ro5 and eRo5/bRo5 compliant ligands. Across all targets, ligands were composed of an average of ∼43% Ro5 compliant ligands and ∼46% eRo5/bRo5 ligands (i.e., MW > 500 Da); see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Furthermore, we assessed the ligandability of each target with the recently defined LIGexp3 index,<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> which is defined as the proportion of compounds tested against a target with p<i>K</i><sub>I</sub> > 7, our “high affinity” threshold discussed previously. Our results confirm that on average, about half (∼45%) of ligands bind the target with high affinity. Lastly, we assessed the chemical tractability of each target by calculating the percentage of each target’s compounds that have a ligand efficiency greater than 0.3 and lipophilic ligand efficiency greater than 5, where LE = pBA/number of heavy atoms and LLE = pBA – ClogP (i.e., LE > 0.3 and LLE > 5).<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> We note that this concept is used to assess the tractability of a target for drug discovery and is not related to synthetic tractability, i.e., the ease of synthesis of particular chemotypes.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> While some targets show high ligand efficiency, the majority actually have fairly low ligand efficiency, which is not surprising since these targets favor eRo5/bRo5 ligands.</div><div class="NLM_p">There are a few targets in our set with ligands that are especially skewed toward eRo5/bRo5. For example, Bcl-2 has only seven ligands in the structure-based ligand profiles and all have MW > 500 Da and high affinity. The ChEMBL-based ligand profile confirms that Bcl-2 most likely requires a bRo5 drug, as the percentage of Ro5 compliant ligands is very low (9.6%), the percentage of eRo5/bRo5 ligands is very high (84.5% of ligands has MW > 500 Da), and tractability is very low (LE > 0.3 and LLE > 5 for 0.7% of ligands); see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Very similar profiles exist for HIV-1 protease and E3 ubiquitin-protein ligase XIAP (complex I), cyclophilin A (complex III), and hepatitis C virus NS34a (simple), thus confirming that the limited structure-based ligand profiles are representative of the greater ChEMBL-based ligand profile. We note that PPAR-γ presents somewhat of an exception as the chemical tractability appears very low (0.1%) because ligands generally satisfy the LE > 0.3 criterion but notoriously fail the LLE > 5 criterion because the binding site is highly lipophilic.</div><div class="NLM_p last">It was shown that complex II targets, comprising mostly kinases, have strong hot spots resulting in a highly druggable active site, with the motivation for using bRo5 ligands being improved selectivity rather than potency or druglikeness. For the complex II targets that are kinases, the results of the ligand based analysis are in full agreement with these findings. More than 10% of the ligands of each kinase fulfill the LE/LLE criteria (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). At least 85% and 55% of the ligands of these kinases (except for those targeting ALK) fulfill the Ro5_3 and Ro5_4 criteria. ALK ligands, however, are typically larger and more lipophilic than other kinase ligands in this set. The only non-kinase target in this class considered here is glucocorticoid receptor, which has the lowest LE/LLE compliance at 8.5%. Most of the ligands of the glucocorticoid receptor fulfill the LE criterion; however, they notoriously fail the LLE criterion. This is in line with our previous study classifying nuclear hormone receptors as challenging targets for druglike molecules.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Notably, glucocorticoid receptor has the highest ligandability of all type II targets (0.74).</div></div><div id="sec8" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12667" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12667" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The goal of this study has been to understand why some targets benefit from drugs that violate the traditional Ro5 criteria. To investigate this question, we started with a set of 37 proteins that have eRo5/bRo5 drugs or clinical candidates from Doak et. al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Each of the drugs and clinical candidates in this set has MW > 500 Da; thus we focused on the MW criteria of the Ro5 but also calculated all other Ro5 properties. Initial analysis of these targets and all ligands that bind to them revealed that, in addition to eRo5/bRo5 compounds, 22 targets can also bind multiple small compounds (MW < 500 Da) with high affinity. Thus, in these cases the need for ligands outside the Ro5 is not driven by poor druggability per se. Structural analysis of the targets revealed that the targets of eRo5/bRo5 drugs have one of two distinct hot spot structures, referred to as “complex” and “simple”, and both differ from the hot spots structures seen in most proteins that bind FDA approved Ro5 drugs.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Analyses of how small and large ligands engage with the hot spot ensemble at the binding site shed light on why eRo5/bRo5 ligands are often more successful than their smaller alternatives even when these also have high affinity. Specifically, from the 37 proteins studied here, different mechanisms by which expanding to bRo5 improved drug properties emerged. Among the targets with complex hot spot structure, the proteins within the complex I group have similar motivation for eRo5/bRo5 ligand development, as supported by the binding profiles. We also find a common motivation among the targets classified as complex II but not for the targets in the complex III group that are discussed individually. However, improving binding is a common motivation for targets with simple hot spot structure.</div><div class="NLM_p">Complex I targets have a “complex” hot spot structure, and most bind both small compounds (MW < 500 Da) and bRo5 compounds with high affinity, yet the bRo5 compounds are generally more successful. In these targets we observe that there is an increase in binding affinity with increased molecular weight. However, small ligands with fairly high affinity generally exist, and so it does not seem that enhanced binding affinity is the main driver for why bRo5 ligands have been more successful in clinical trials. As shown for thrombin, many smaller drug candidates had problems due to inherent toxicity, high-plasma protein binding, poor metabolic stability, rapid elimination from the blood, and low in vivo activity, while the bRo5 compound argatroban has improved properties. These improved pharmaceutical properties are presumably not inherent in the use of larger compounds; indeed, the opposite might be expected. But good properties were evidently not achieved with Ro5-compliant compounds. A potential explanation is that in moving to the bRo5 range, the extra binding energy gained by engaging more hot spots provides scope for pharmaceutically problematic functionality on a ligand to be substituted with more benign structures that individually bind less well while retaining sufficient overall binding affinity for the compound as a whole. In addition, the higher complexity of the compounds may reduce the likelihood of off-target binding. We emphasize that these observations are based on only a small set of targets, and it is possible that other targets with complex hot spot structures do have pharmaceutically optimized Ro5 ligands. However, in previous work we have mapped a substantial number of proteins that have only Ro5 drugs, and in most cases we have found at most three hot spots, albeit strong ones, in the ligand binding site.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> This finding supports the notion that binding sites with more than three strong enough hot spots can provide means to overcome poor pharmaceutical behavior by trading off extra binding energy for property improvements in the context of a bRo5 compound.</div><div class="NLM_p">Complex II targets also have complex hot spot structure, but they benefit from bRo5 compounds for a different reason than complex I targets, since binding affinity does not improve with ligands of increased molecular weight. All but one complex II targets are protein kinases, with small ligands (i.e., MW < 400 Da) that bind with high affinity, and we conclude that the primary motivation for expanding to eRo5/bRo5 ligands for complex II targets is to obtain improved selectivity rather than higher binding affinity. In addition, we have discussed eight targets that have a complex hot spot structure but too few ligands to see whether or not a correlation exists between MW and binding affinity. In spite of only five ligands, a positive correlation suggests that DOT1-like histone H3 methyltransferase could be classified as complex I, whereas FGFR1 is a typical kinase and behaves like a complex II target. Nevertheless, in view of the limited data, we placed these proteins in a separate group called complex III, together with six more targets that benefit from eRo5/bRo5 drugs for various reasons.</div><div class="NLM_p">In contrast to the targets that have complex binding hot spot structures with many strong hot spots, targets with simple hot spot structures only contain three or fewer relatively weak hot spots, in many cases rendering the target only barely druggable.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Almost all ligands that bind such targets utilize all hot spots and then expand beyond to interact with additional surface regions of the protein, likely to increase binding affinity and/or selectivity. The binding sites of proteins with simple hot spot structure are frequently located in deep pockets or narrow crevices, and hence modest increases in MW can significantly increase the protein–ligand interface area and improve binding affinity, particularly if the interacting surfaces are hydrophobic. As a result, for some proteins with simple hot spot structure the binding affinity of the ligands increases with increasing molecular weight, while in other cases the binding affinity is not affected.</div><div class="NLM_p">Since only a small fraction of ligands of the target proteins considered here have been cocrystallized with the proteins, we also studied all ligands available in the ChEMBL<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> database. Results confirmed general agreement between the ligand profiles, indicating that the limited structure-based ligand profiles were representatives of the true ligand profiles based on all ligands without cocrystallized structures. Specifically, for targets that appeared druggable with both small and large ligands, we often identified a substantial number of small (MW < 500 Da) ligands with high affinity (pBA > 7) in the general ligand data set. In contrast, for many targets with no cocrystallized high affinity Ro5 ligands, the fraction of large ligands (MW > 500 Da) was very high in the general ligand data set.</div><div class="NLM_p last">Importantly, the hot spots of most proteins can be found, without major differences, by mapping either unbound or bound protein structures. Thus, once the X-ray structure of a protein is available, one can map the unbound structure or a structure cocrystallized with any ligand to determine the hot spot structure of the binding site. The hot spots structures will then indicate if it may be useful to consider bRo5 ligands for the target in question. In particular, our results reveal the somewhat counterintuitive finding that bRo5 ligands can be beneficial for targeting complex binding sites on highly druggable proteins, as well as proteins with simple binding sites that are only borderline druggable. Moreover, we highlight that bRo5 drugs can show benefits through a number of different mechanisms and not just increased binding affinity, including improved selectivity along with other improved pharmaceutical properties such as reduced toxicity. Thus, the data encourage the consideration of bRo5 ligands in drug discovery against chemically tractable targets, in cases where achieving selectivity or good pharmaceutical properties is problematic, in addition to being an approach for low druggability targets.</div></div><div id="sec9" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82316" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82316" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec9_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> FTMap of Protein Structures</h3><div class="NLM_p last">The FTMap server (<a href="http://ftmap.bu.edu" class="extLink">http://ftmap.bu.edu</a>)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> was used to identify the binding hot spots for the 37 holo structures listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and the 21 apo structures in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Domain splitting was performed by Protein Domain Parser<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> on four of the holo structures prior to FTMap hot spot identification: DNA-directed RNA-polymerase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KMU">4KMU</a>), integrin α-IIB (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VDN">2VDN</a>), Smoothened homolog (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JKV">4JKV</a>), isoleucyl-tRNA synthetase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QU2">1QU2</a>).</div></div><div id="sec9_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Data Set for Structure-Based Ligand Analysis</h3><div class="NLM_p last">The set of ligands that bind each target with known structure and binding affinity were identified by the following procedure. For any given target, all structures in the PDB that have 90% or greater sequence similarity were identified. These structures were aligned to the protein target of interest using the PyMOL<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> align function, and the het atoms in the PDB file were isolated and analyzed to extract only the ligands that bind within 15 Å of the drug/clinical candidate of interest. ADP and fragments with three or fewer atoms were removed. The ligands were further filtered by eliminating any ligands that do not have known binding affinity in either the PDB Bind Database,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Binding MOAD Database,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> or The Binding Database.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> We accepted binding affinity data in the form of <i>K</i><sub>D</sub>, <i>K</i><sub>I</sub>, and IC<sub>50</sub>; however, where multiple binding affinity data are available for one ligand-protein complex, we set priority as <i>K</i><sub>D</sub> > <i>K</i><sub>I</sub> > IC<sub>50</sub>. Moreover, if multiple affinities of the preferable measurement exist, the highest affinity was selected. Furthermore, when two or more PDB codes presented the same ligand (i.e., same ligand name in the PDB) and had the same binding affinity, we arbitrarily selected one representative and discarded the duplicate ligands. Notably, all but five protein–ligand complexes were solved with X-ray crystallography (resolution: mean = 2.08 Å and standard deviation = 0.45 Å); the distribution is plotted in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">Figure S8</a>.</div></div><div id="sec9_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Binding Site Identification by FTMap</h3><div class="NLM_p last">The binding hot spots of each target represent the available hot spots for drug development in the binding region of the drug/clinical candidate of interest. The most intuitive definition of the binding site is simply a collection of all hot spots utilized by the ligands identified in the PDB. This definition is superb for targets with many similar ligands identified. However, for targets where very few ligands are identified, a more robust rule for the binding site is necessary. On the basis of the targets with many ligands, the ligand-based binding site generally corresponds to all hot spots within ∼8 A of the drug/clinical candidate of interest. Manual filtering is necessary to ensure all hot spots are in the same cavity (i.e., a hot spot around the other side of the protein is not actually within 8 Å). Therefore, in the absence of ligands, the binding site for any given target can be estimated as all hot spots within 8 Å of any atom of the drug/clinical candidate of interest. The binding site listed for each target in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> consists of all the hot spots utilized by the ligands identified with known structure and affinity via the procedure above.</div></div><div id="sec9_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Hot Spot–Ligand Proximity Analysis</h3><div class="NLM_p last">For each target protein, the aligned ligands were assessed for their proximity to each of the hot spots identified by FTMap. If any atom in the ligand was within 3 Å of the high density probe center of the hot spot, it was considered an associated hot spot and the number of probe clusters in the hot spot was added to the total number of probe clusters associated. Only FTMap hot spots with five or more probe clusters were considered, for simplicity.</div></div><div id="sec9_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> ERo5/bRo5 Property Calculation</h3><div class="NLM_p last">Physiochemical descriptors for the extended and beyond Ro5 classification were calculated using the open-source cheminformatics software RDKit (<a href="https://www.rdkit.org/" class="extLink">https://www.rdkit.org/</a>). Specifically, the molecular weight (MW), number of hydrogen bond donors (HBD), number of hydrogen bond acceptors (HBA), calculated octanol–water partition coefficient (CLogP), topological polar surface area (TPSA), and the number of rotatable bonds (NRotB) were calculated from the SMILES string given for each ligand in the PDB. The eRo5 and bRo5 definitions by Doak et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> were used to classify the ligands. For reference, ligands that satisfy all of the following were classified eRo5: MW 500–700 Da, ClogP ≤ 7.5, HBD ≤ 5, HBA ≤ 10, TPSA ≤ 200, NRotB ≤ 20. Ligands that have MW > 500 Da and satisfy one of the following were classified as bRo5: MW > 700 Da, ClogP > 7.5 HBD > 5, HBA > 10, PSA > 200, NRotB > 20, in agreement with the classification used by Doak and colleagues.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div></div><div id="sec9_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Data Set for Ligand-Based Analysis</h3><div class="NLM_p last">As the final step of our study, we examined the properties of all ligands with pBA > 7, which bind each of the 37 proteins. Ligands of the selected 37 protein targets were collected from ChEMBL.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The database has been searched by the target and gene name, and ligands with definite target activity were downloaded. Here we considered both <i>K</i><sub>I</sub> and IC<sub>50</sub> values for this analysis and converted the affinities to pBA values. According to the activity threshold used in the target based analysis, we considered only ligands with pBA > 7; however, the total number of measured ligands is also shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. For statistically meaningful results the ligand based analysis was restricted to targets having at least 40 qualified ligands in ChEMBL. Seven targets (soluble acetylcholine receptor, lanosterol 14-α demethylase, DNA-directed RNA-polymerase, tubulin-α chain, Toll-like receptor 4, α-amylase, and isoleucyl-tRNA synthetase) have less than 40 qualified ligands and therefore were not considered.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01732" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60733" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60733" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01732" class="ext-link">10.1021/acs.jmedchem.8b01732</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Graphical representation and statistical analysis of structure-based ligand binding profiles for all targets with more than five ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf">jm8b01732_si_001.pdf (1.43 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01732" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34382" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34382" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adrian Whitty</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0331-4369" title="Orcid link">http://orcid.org/0000-0002-0331-4369</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#97e0fffee3e3eed7f5e2b9f2f3e2"><span class="__cf_email__" data-cfemail="8cfbe4e5f8f8f5cceef9a2e9e8f9">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">György M. Keserű</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry Research Group, Research Center
for Natural Sciences, Magyar Tudósok krt. 2, H-1117 Budapest, Hungary</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#56312f783d33253324231622223d783b2237783e23"><span class="__cf_email__" data-cfemail="e18698cf8a8492849394a195958acf8c9580cf8994">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sandor Vajda</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Boston University, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1540-8220" title="Orcid link">http://orcid.org/0000-0003-1540-8220</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#087e69626c69486a7d266d6c7d"><span class="__cf_email__" data-cfemail="f187909b9590b19384df949584">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Megan Egbert</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, Boston University, Boston, Massachusetts 02215, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.E. performed the structure-based analysis of the targets. G.M.K. performed the ligand-based analysis. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Acpharis Inc. offers commercial licenses to ATLAS, a software product in function similar to FTMap. Sandor Vajda owns stock in the company. However, the FTMap program and the use of the FTMap server are free for academic and governmental use.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87783" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87783" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Megan Egbert</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=BIO-d7e4452-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Megan Egbert</b> is a graduate student pursuing a Ph.D. in Biomedical Engineering at Boston University. Megan obtained a Bachelor of Science in Engineering in Chemical Engineering and a minor in Biochemistry at the University of Michigan, Ann Arbor. Following her Bachelor’s degree, Megan worked at Intel Corporation as a process engineer. Her current research is focused on extracting information from the binding site of proteins and modeling the binding of protein–small molecule interactions, specifically focusing on beyond rule of five drug–protein interactions.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Adrian Whitty</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=BIO-d7e4457-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Adrian Whitty</b> received a B.Sc. in Chemistry from King’s College London in 1985, followed by M.S. and Ph.D. degrees in Chemistry from the University of Illinois at Chicago. After completing postdoctoral work with William P. Jencks FRS at Brandeis University, he joined the biopharmaceutical company Biogen, where he worked in the Department of Drug Discovery, rising to the position of Director of Physical Biochemistry. In 2008 he moved to Boston University, where he is an Associate Professor in the Department of Chemistry and in the Department of Pharmacology and Experimental Therapeutics. His research interests include protein–protein and protein–ligand binding, especially as related to the discovery of small molecule inhibitors of protein–protein interactions, and to the quantitative, mechanistic analysis of growth factor receptor activation and signaling.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">György M. Keserű</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=BIO-d7e4462-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>György M. Keserű</b> obtained his Ph.D. in Budapest, Hungary, and joined Sanofi heading a chemistry research lab. He moved to Gedeon Richter in 1999 as the Head of Computer-Aided Drug Discovery. He earned a D.Sc. from the Hungarian Academy of Science in 2003, and he was invited for a research professorship at the Budapest University of Technology and Economics. Since 2007 he was appointed as the Head of Discovery Chemistry at Gedeon Richter. He contributed to the discovery of the antipsychotic Vraylar (cariprazine). He served as Director General of the Research Centre for Natural Sciences (RCNS) at the Hungarian Academy of Sciences. Since 2015 he has headed the Medicinal Chemistry Research Group at RCNS. His research interests include medicinal chemistry and drug design.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Sandor Vajda</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=BIO-d7e4467-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Sandor Vajda</b> received an M.S. in Applied Mathematics and a Ph.D. in Chemistry in Budapest, Hungary. He held research positions in the Department of Engineering, University of Warwick, England, and the Department of Chemistry, Princeton University. He was a faculty member in Hungary and at the Mount Sinai School of Medicine, New York, NY. Currently he is Professor of Biomedical Engineering and Chemistry at Boston University. He also directs the Biomolecular Engineering Research Center at Boston University. Dr. Vajda has been active in method development for modeling of biological macromolecules, with emphasis on molecular interactions and drug design. In 2004 he and his graduate students founded SolMap Pharmaceuticals, a start-up company focusing on fragment-based drug design. SolMap was acquired by Forma Therapeutics, Cambridge, MA, in 2008.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i55">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67345" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67345" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This investigation was supported by Grant R35-GM118078 from the National Institute of General Medical Sciences. G.M.K. is supported by the National Brain Research Program (Grant 2017-1.2.1-NKP-2017-00002) of the National Research, Development and Innovation Office, Hungary.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">Ro5</td><td class="NLM_def"><p class="first last">rule of five</p></td></tr><tr><td class="NLM_term">eRo5</td><td class="NLM_def"><p class="first last">extended rule of five</p></td></tr><tr><td class="NLM_term">bRo5</td><td class="NLM_def"><p class="first last">beyond rule of five</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">molecular weight</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">Food and Drug Administration</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">CS</td><td class="NLM_def"><p class="first last">consensus site</p></td></tr><tr><td class="NLM_term">BA</td><td class="NLM_def"><p class="first last">binding affinity</p></td></tr><tr><td class="NLM_term">pBA</td><td class="NLM_def"><p class="first last">negative logarithm of binding affinity in form of <i>K</i><sub>D</sub>, <i>K</i><sub>I</sub>, or IC<sub>50</sub></p></td></tr><tr><td class="NLM_term">NRotB</td><td class="NLM_def"><p class="first last">number of rotatable bonds</p></td></tr><tr><td class="NLM_term">TPSA</td><td class="NLM_def"><p class="first last">topological polar surface area.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i57">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46780" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46780" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 85 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lg1t8Ue1YbDIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Two decades under the influence of the rule of five and the changing properties of approved oral drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslait73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1701-1714&author=M.+D.+Shultz&title=Two+decades+under+the+influence+of+the+rule+of+five+and+the+changing+properties+of+approved+oral+drugs&doi=10.1021%2Facs.jmedchem.8b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1701-1714</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two decades have passed since the rule of five ushered in the concept of "drug-like" properties.  Attempts to quantify, correlate, and categorize mols. based on Ro5 parameters evolved into the introduction of efficiency metrics with far reaching consequences in decision making by industry leaders and scientists seeking to discover new medicines.  Examn. of oral drug parameters approved before and after the original Ro5 anal. demonstrates that some parameters such as clogP and HBD remained const. while the cutoffs for parameters such as mol. wt. and HBA have increased substantially over the past 20 years.  The time dependent increase in the mol. wt. of oral drugs during the past 20 years provides compelling evidence to disprove the hypothesis that mol. wt. is a "drug-like" property.  This anal. does not validate parameters that have not changed as being "drug-like" but instead calls into question the entire hypothesis that "drug-like" properties exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnoxpQsv2kbVg90H21EOLACvtfcHk0liOSGAezvLJcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslait73K&md5=d3c8b1107133dc0b7b277d22fa029597</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DTwo%2520decades%2520under%2520the%2520influence%2520of%2520the%2520rule%2520of%2520five%2520and%2520the%2520changing%2520properties%2520of%2520approved%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1701%26epage%3D1714%26doi%3D10.1021%2Facs.jmedchem.8b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Oral druggable space beyond the rule of 5: Insights from drugs and clinical candidates</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1115</span>– <span class="NLM_lpage">1142</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.chembiol.2014.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25237858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1115-1142&author=B.+C.+Doakauthor=B.+Overauthor=F.+Giordanettoauthor=J.+Kihlberg&title=Oral+druggable+space+beyond+the+rule+of+5%3A+Insights+from+drugs+and+clinical+candidates&doi=10.1016%2Fj.chembiol.2014.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Druggable Space beyond the Rule of 5: Insights from Drugs and Clinical Candidates</span></div><div class="casAuthors">Doak, Bradley Croy; Over, Bjorn; Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1115-1142</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The rule of 5 (Ro5) is a set of in silico guidelines applied to drug discovery to prioritize compds. with an increased likelihood of high oral absorption.  It has been influential in reducing attrition due to poor pharmacokinetics over the last 15 years.  However, strict reliance on the Ro5 may have resulted in lost opportunities, particularly for difficult targets.  To identify opportunities for oral drug discovery beyond the Ro5 (bRo5), we have comprehensively analyzed drugs and clin. candidates with mol. wt. (MW) > 500 Da.  We conclude that oral drugs are found far bRo5 and properties such as intramol. hydrogen bonding, macrocyclization, dosage, and formulations can be used to improve bRo5 bioavailability.  Natural products and structure-based design, often from peptidic leads, are key sources for oral bRo5 drugs.  These insights should help guide the design of oral drugs in bRo5 space, which is of particular interest for difficult targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvYqpeA0dh6rVg90H21EOLACvtfcHk0lg9eIOogRdvew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K&md5=b8a88c793572ad48de3809807a4699d8</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DOral%2520druggable%2520space%2520beyond%2520the%2520rule%2520of%25205%253A%2520Insights%2520from%2520drugs%2520and%2520clinical%2520candidates%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1115%26epage%3D1142%26doi%3D10.1016%2Fj.chembiol.2014.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeGoey, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span> <span> </span><span class="NLM_article-title">Beyond the rule of 5: Lessons learned from AbbVie’s drugs and compound collection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00717</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00717" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2636-2651&author=D.+A.+DeGoeyauthor=H.+J.+Chenauthor=P.+B.+Coxauthor=M.+D.+Wendt&title=Beyond+the+rule+of+5%3A+Lessons+learned+from+AbbVie%E2%80%99s+drugs+and+compound+collection&doi=10.1021%2Facs.jmedchem.7b00717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the Rule of 5: Lessons Learned from AbbVie's Drugs and Compound Collection</span></div><div class="casAuthors">DeGoey, David A.; Chen, Hui-Ju; Cox, Philip B.; Wendt, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2636-2651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing focus on the pursuit of targets considered to be less druggable that offer potential for development of promising new therapeutic agents for the treatment of diseases with large unmet medical need, particularly in the areas of oncol. and virol.  However, conducting drug discovery campaigns in "beyond rule of 5" (bRo5) chem. space presents a significant drug design and development challenge to medicinal chemists to achieve acceptable oral pharmacokinetics.  Retrospective anal. of past successes and failures in drug discovery bRo5 may shed light on the key principles that contribute to the oral bioavailability of successful bRo5 compds. and improve the efficiency of drug design for future projects.  We present here highlights and case studies of lessons learned from discovery of bRo5 compds.  A simple multiparametric scoring function (AB-MPS) was devised that correlated preclin. PK results with cLogD, no. of rotatable bonds, and no. of arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHuaXbtjlALVg90H21EOLACvtfcHk0lg9eIOogRdvew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ&md5=48984dc65410d83d4ac8ddaca030b024</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26aulast%3DDeGoey%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DH.%2BJ.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26atitle%3DBeyond%2520the%2520rule%2520of%25205%253A%2520Lessons%2520learned%2520from%2520AbbVie%25E2%2580%2599s%2520drugs%2520and%2520compound%2520collection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2636%26epage%3D2651%26doi%3D10.1021%2Facs.jmedchem.7b00717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lg9eIOogRdvew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viarengo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Quantifying the chameleonic properties of macrocycles and other high-molecular-weight drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">712</span>– <span class="NLM_lpage">717</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.drudis.2016.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=26891978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=712-717&author=A.+Whittyauthor=M.+Zhongauthor=L.+Viarengoauthor=D.+Beglovauthor=D.+R.+Hallauthor=S.+Vajda&title=Quantifying+the+chameleonic+properties+of+macrocycles+and+other+high-molecular-weight+drugs&doi=10.1016%2Fj.drudis.2016.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying the chameleonic properties of macrocycles and other high-molecular-weight drugs</span></div><div class="casAuthors">Whitty, Adrian; Zhong, Mengqi; Viarengo, Lauren; Beglov, Dmitri; Hall, David R.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">712-717</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Key to the pharmaceutical utility of certain macrocyclic drugs is a 'chameleonic' ability to change their conformation to expose polar groups in aq. soln., but bury them when traversing lipid membranes.  Based on anal. of the structures of 20 macrocyclic compds. that are approved oral drugs, we propose that good soly. requires a topol. polar surface area (TPSA, in Å2) of ≥0.2 × mol. wt. (MW).  Meanwhile, good passive membrane permeability requires a mol. (i.e., 3D) PSA in nonpolar environments of ≤140 Å2.  We show that one or other of these limits is almost invariably violated for compds. with MW > 600 Da, suggesting that some degree of chameleonic behavior is required for most high MW oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlN7rCqts5wLVg90H21EOLACvtfcHk0ljrNdYdRhbkuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D&md5=024476298f7b72efa3f0a9303bed5900</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DZhong%26aufirst%3DM.%26aulast%3DViarengo%26aufirst%3DL.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DQuantifying%2520the%2520chameleonic%2520properties%2520of%2520macrocycles%2520and%2520other%2520high-molecular-weight%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D712%26epage%3D717%26doi%3D10.1016%2Fj.drudis.2016.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The exploration of macrocycles for drug discovery--an underexploited structural class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+exploration+of+macrocycles+for+drug+discovery%2D%2Dan+underexploited+structural+class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0lhKhtQQ9frnXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520exploration%2520of%2520macrocycles%2520for%2520drug%2520discovery--an%2520underexploited%2520structural%2520class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lhKhtQQ9frnXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles are great cycles: Applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+Applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lhKhtQQ9frnXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520Applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery beyond the rule of 5 - opportunities and challenges</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1080/17460441.2017.1264385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1080%2F17460441.2017.1264385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=27883294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BC2sjgtVejtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=115-119&author=B.+C.+Doakauthor=J.+Kihlberg&title=Drug+discovery+beyond+the+rule+of+5+-+opportunities+and+challenges&doi=10.1080%2F17460441.2017.1264385"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery beyond the rule of 5 - Opportunities and challenges</span></div><div class="casAuthors">Doak Bradley C; Kihlberg Jan</div><div class="citationInfo"><span class="NLM_cas:title">Expert opinion on drug discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-119</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRw3tmClaGOihOk4JA3soTqfW6udTcc2eZFZxZCvYQJwLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjgtVejtw%253D%253D&md5=bccc3e6cb938479044d18a2b25df9c1f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1080%2F17460441.2017.1264385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2017.1264385%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520beyond%2520the%2520rule%2520of%25205%2520-%2520opportunities%2520and%2520challenges%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2017%26volume%3D12%26spage%3D115%26epage%3D119%26doi%3D10.1080%2F17460441.2017.1264385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santos, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emery, F. S.</span></span> <span> </span><span class="NLM_article-title">Oral administration of peptide-based drugs: Beyond Lipinski’s rule</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2245</span>– <span class="NLM_lpage">2251</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1002%2Fcmdc.201600288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=27596610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVGgu7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2245-2251&author=G.+B.+Santosauthor=A.+Ganesanauthor=F.+S.+Emery&title=Oral+administration+of+peptide-based+drugs%3A+Beyond+Lipinski%E2%80%99s+rule&doi=10.1002%2Fcmdc.201600288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Administration of Peptide-Based Drugs: Beyond Lipinski's Rule</span></div><div class="casAuthors">Santos, Gabriela B.; Ganesan, A.; Emery, Flavio S.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2245-2251</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The use of peptides in therapy presents several limitations, from physicochem. characteristics to inadequate pharmacokinetic profiles for oral absorption.  As peptides are gaining importance in the therapeutic arsenal, there is an increasing need to rationalize the main characteristics of this compd. class in the market.  Therefore, we performed an extensive anal. of all known peptide drugs and clin. candidates based on their peptide features, physicochem. and structural properties, and correlated these with their administration route and therapeutic classes.  Peptide drugs are widely distributed across drug and pharmacol. space, covering several therapeutic areas with structural diversity and complexity, distributed between groups of cyclic and linear compds.  Although structural and physicochem. properties are clear within these groups, we counter the consensus that cyclic peptides have better oral availability than linear peptides, as most of the orally administrated peptides have linear structures.  This study and review furnishes information that could support peptide drug design, with a new cutoff of known descriptors that go beyond the Rule of Five.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr88nw1qqAJuLVg90H21EOLACvtfcHk0lgnoDPkWD0qRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVGgu7rI&md5=b2e5b6a1dff54c2c1f85671ab7314031</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600288%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DG.%2BB.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DEmery%26aufirst%3DF.%2BS.%26atitle%3DOral%2520administration%2520of%2520peptide-based%2520drugs%253A%2520Beyond%2520Lipinski%25E2%2580%2599s%2520rule%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D2245%26epage%3D2251%26doi%3D10.1002%2Fcmdc.201600288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">Small-molecule kinase inhibitors: An analysis of FDA-approved drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.drudis.2015.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=26210956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=5-10&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=Small-molecule+kinase+inhibitors%3A+An+analysis+of+FDA-approved+drugs&doi=10.1016%2Fj.drudis.2015.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors (SMKIs), 28 of which are approved by the US Food and Drug Administration (FDA), have been actively pursued as promising targeted therapeutics.  Here, we assess the key structural and physicochem. properties, target selectivity and mechanism of function, and therapeutic indications of these approved inhibitors.  Our anal. showed that >30% of approved SMKIs have a mol. wt. (MW) exceeding 500 and all have a total ring count of between three and five.  The assumption that type II inhibitors tend to be more selective than type I inhibitors has been proved to be unreliable.  Although previous SMKI research was concd. on tyrosine kinase inhibitors for cancer treatment, recent progress indicates diversification of SMKI research in terms of new targets, mechanistic types, and therapeutic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZrY68ZOI2xLVg90H21EOLACvtfcHk0lgnoDPkWD0qRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P&md5=228524fa464393053ecbfac19e994929</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DSmall-molecule%2520kinase%2520inhibitors%253A%2520An%2520analysis%2520of%2520FDA-approved%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.drudis.2015.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of protein-protein interactions: Progressing toward the reality</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1102</span>– <span class="NLM_lpage">1114</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.chembiol.2014.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25237857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1102-1114&author=M.+R.+Arkinauthor=Y.+Tangauthor=J.+A.+Wells&title=Small-molecule+inhibitors+of+protein-protein+interactions%3A+Progressing+toward+the+reality&doi=10.1016%2Fj.chembiol.2014.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing toward the Reality</span></div><div class="casAuthors">Arkin, Michelle R.; Tang, Yinyan; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1102-1114</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The past 20 years have seen many advances in our understanding of protein-protein interactions (PPIs) and how to target them with small-mol. therapeutics.  In 2004, we reviewed some early successes; since then, potent inhibitors have been developed for diverse protein complexes, and compds. are now in clin. trials for six targets.  Surprisingly, many of these PPI clin. candidates have efficiency metrics typical of "lead-like" or "drug-like" mols. and are orally available.  Successful discovery efforts have integrated multiple disciplines and make use of all the modern tools of target-based discovery-structure, computation, screening, and biomarkers.  PPIs become progressively more challenging as the interfaces become more complex, i.e., as binding epitopes are displayed on primary, secondary, or tertiary structures.  Here, we review the last 10 years of progress, focusing on the properties of PPI inhibitors that have advanced to clin. trials and prospects for the future of PPI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP01wt-4lfE7Vg90H21EOLACvtfcHk0lgPJGozIdvIHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J&md5=aec3897dc9bc66656723bc56c8d69155</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520Progressing%2520toward%2520the%2520reality%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1102%26epage%3D1114%26doi%3D10.1016%2Fj.chembiol.2014.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Opinion - how many drug targets are there?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">993</span>– <span class="NLM_lpage">996</span>, <span class="refDoi"> DOI: 10.1038/nrd2199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrd2199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=17139284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=993-996&author=J.+P.+Overingtonauthor=B.+Al-Lazikaniauthor=A.+L.+Hopkins&title=Opinion+-+how+many+drug+targets+are+there%3F&doi=10.1038%2Fnrd2199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">How many drug targets are there?</span></div><div class="casAuthors">Overington, John P.; Al-Lazikani, Bissan; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">993-996</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  For the past decade, the no. of mol. targets for approved drugs has been debated.  Here, we reconcile apparently contradictory previous reports into a comprehensive survey, and propose a consensus no. of current drug targets for all classes of approved therapeutic drugs.  One striking feature is the relatively const. historical rate of target innovation (the rate at which drugs against new targets are launched); however, the rate of developing drugs against new families is significantly lower.  The recent approval of drugs that target protein kinases highlights two addnl. trends: an emerging realization of the importance of polypharmacol., and also the power of a gene-family-led approach in generating novel and important therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDoZu-pkUsAbVg90H21EOLACvtfcHk0lgPJGozIdvIHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM&md5=14b5716f4e10b57ae71bf67e52a21929</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd2199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2199%26sid%3Dliteratum%253Aachs%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DOpinion%2520-%2520how%2520many%2520drug%2520targets%2520are%2520there%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D993%26epage%3D996%26doi%3D10.1038%2Fnrd2199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basse, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morelli, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, P.</span></span> <span> </span><span class="NLM_article-title">2P2IDB V2: Update of a structural database dedicated to orthosteric modulation of protein-protein interactions</span>. <i>Database</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2016</i></span>,  <span class="NLM_fpage">baw007</span>, <span class="refDoi"> DOI: 10.1093/database/baw007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fdatabase%2Fbaw007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=26980515" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2016&publication_year=2016&pages=baw007&author=M.+J.+Basseauthor=S.+Betziauthor=X.+Morelliauthor=P.+Roche&title=2P2IDB+V2%3A+Update+of+a+structural+database+dedicated+to+orthosteric+modulation+of+protein-protein+interactions&doi=10.1093%2Fdatabase%2Fbaw007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fdatabase%2Fbaw007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fdatabase%252Fbaw007%26sid%3Dliteratum%253Aachs%26aulast%3DBasse%26aufirst%3DM.%2BJ.%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DMorelli%26aufirst%3DX.%26aulast%3DRoche%26aufirst%3DP.%26atitle%3D2P2IDB%2520V2%253A%2520Update%2520of%2520a%2520structural%2520database%2520dedicated%2520to%2520orthosteric%2520modulation%2520of%2520protein-protein%2520interactions%26jtitle%3DDatabase%26date%3D2016%26volume%3D2016%26spage%3Dbaw007%26doi%3D10.1093%2Fdatabase%2Fbaw007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. C.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of protein-protein interactions: How to mimic a protein partner</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4679</span>– <span class="NLM_lpage">4684</span>, <span class="refDoi"> DOI: 10.2174/138161212802651634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.2174%2F138161212802651634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=22650256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2gu7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=4679-4684&author=D.+C.+Fry&title=Small-molecule+inhibitors+of+protein-protein+interactions%3A+How+to+mimic+a+protein+partner&doi=10.2174%2F138161212802651634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of protein-protein interactions: how to mimic a protein partner</span></div><div class="casAuthors">Fry, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">4679-4684</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This systematic review describes successful examples of small-mol. inhibitors of protein-protein interactions, and compares their binding strategies to those employed by the natural protein partners.  It extends and updates an earlier survey of this type.  From anal. of these systems, common themes and lessons are presented that may assist future drug discovery efforts involving targets in this class.  One encouraging finding is that a wide scope appears to be allowed at these sites in terms of binding strategies and chemotypes, which suggests that the outlook for finding small-mol. protein-protein inhibitors is favorable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTu1bO0sottbVg90H21EOLACvtfcHk0lh57RuA0dY4YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2gu7fL&md5=1d093305caf318924fdf72b239326671</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.2174%2F138161212802651634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212802651634%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DD.%2BC.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520How%2520to%2520mimic%2520a%2520protein%2520partner%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D4679%26epage%3D4684%26doi%3D10.2174%2F138161212802651634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClendon, C. L.</span></span> <span> </span><span class="NLM_article-title">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">1001</span>– <span class="NLM_lpage">1009</span>, <span class="refDoi"> DOI: 10.1038/nature06526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnature06526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=18075579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=1001-1009&author=J.+A.+Wellsauthor=C.+L.+McClendon&title=Reaching+for+high-hanging+fruit+in+drug+discovery+at+protein-protein+interfaces&doi=10.1038%2Fnature06526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span></div><div class="casAuthors">Wells, James A.; McClendon, Christopher L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">1001-1009</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting the interfaces between proteins has huge therapeutic potential, but discovering small-mol. drugs that disrupt protein-protein interactions is an enormous challenge.  Several recent success stories, however, indicate that protein-protein interfaces might be more tractable than has been thought.  These studies discovered small mols. that bind with drug-like potencies to 'hotspots' on the contact surfaces involved in protein-protein interactions.  Remarkably, these small mols. bind deeper within the contact surface of the target protein, and bind with much higher efficiencies, than do the contact atoms of the natural protein partner.  Some of these small mols. are now making their way through clin. trials, so this high-hanging fruit might not be far out of reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-nGR3BqqkrVg90H21EOLACvtfcHk0lh57RuA0dY4YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F&md5=e7ed69fd8c362a71c4b99f029c16a6fc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature06526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06526%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26atitle%3DReaching%2520for%2520high-hanging%2520fruit%2520in%2520drug%2520discovery%2520at%2520protein-protein%2520interfaces%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D1001%26epage%3D1009%26doi%3D10.1038%2Fnature06526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of protein-protein interactions: Progressing towards the dream</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd1343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrd1343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=15060526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1GktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=301-317&author=M.+R.+Arkinauthor=J.+A.+Wells&title=Small-molecule+inhibitors+of+protein-protein+interactions%3A+Progressing+towards+the+dream&doi=10.1038%2Fnrd1343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of protein-protein interactions: progressing towards the dream</span></div><div class="casAuthors">Arkin, Michelle R.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions have a key role in most biol. processes, and offer attractive opportunities for therapeutic intervention.  Developing small mols. that modulate protein-protein interactions is difficult, owing to issues such as the lack of well-defined binding pockets.  Nevertheless, there was important progress in this endeavor in recent years.  Here, the authors use illustrative examples to discuss general strategies for addressing the challenges inherent in the discovery and characterization of small-mol. inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7MiFDdLJu27Vg90H21EOLACvtfcHk0lh57RuA0dY4YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1GktrY%253D&md5=3e102c6207be420b4232056b4acdea12</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd1343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1343%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520Progressing%2520towards%2520the%2520dream%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D301%26epage%3D317%26doi%3D10.1038%2Fnrd1343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, B.</span></span> <span> </span><span class="NLM_article-title">Drug discovery beyond the ’rule-of-five’</span>. <i>Curr. Opin. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">478</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1016/j.copbio.2007.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.copbio.2007.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=18035532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislOkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2007&pages=478-488&author=M.+Q.+Zhangauthor=B.+Wilkinson&title=Drug+discovery+beyond+the+%E2%80%99rule-of-five%E2%80%99&doi=10.1016%2Fj.copbio.2007.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery beyond the 'rule-of-five'</span></div><div class="casAuthors">Zhang, Ming-Qiang; Wilkinson, Barrie</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">478-488</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although a very useful guideline for orally bioavailable small-mol. drug design, the 'rule-of-five' (also known as 'Lipinski's rule of drug-likeness') has to some extent been overemphasized.  Firstly, only 51% of all FDA-approved small-mol. drugs are both used orally and comply with the 'rule-of-five'.  This does not even include the increasing no. of biologicals of which several have reached blockbuster' status.  Secondly, it does not cover natural product and semisynthetic natural product drugs, which constitute over one-third of all marketed small-mol. drugs.  A more balanced and programmatic approach to drug discovery should be more productive than to rely on an overemphasis of 'rule-of-five' compliance.  Rather it should consider proactively the development of parenteral drugs in parallel to oral drugs and to consider the development of therapeutic antibodies in parallel to small-mol. drugs.  These are particularly relevant for efforts against 'first-in-class' and/or particularly challenging targets such as proteases and those involving protein-protein interactions.  In addn., more effort should be invested in natural product research.  Emerging novel technologies such as synthetic biol. (genetic engineering of living organisms to produce small-mol. therapeutics) may address several challenging issues of natural product-based drug discovery including synthetic feasibility and ligand efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGollnOuY9kGrbVg90H21EOLACvtfcHk0ljG-w0QnkOJ4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislOkug%253D%253D&md5=4121024557295c4f241255e43b43ea44</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2007.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2007.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%2BQ.%26aulast%3DWilkinson%26aufirst%3DB.%26atitle%3DDrug%2520discovery%2520beyond%2520the%2520%25E2%2580%2599rule-of-five%25E2%2580%2599%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2007%26volume%3D18%26spage%3D478%26epage%3D488%26doi%3D10.1016%2Fj.copbio.2007.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tonddast-Navaei, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skolnick, J.</span></span> <span> </span><span class="NLM_article-title">On the importance of composite protein multiple ligand interactions in protein pockets</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1252</span>– <span class="NLM_lpage">1259</span>, <span class="refDoi"> DOI: 10.1002/jcc.24523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1002%2Fjcc.24523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=27864975" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVygt7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=1252-1259&author=S.+Tonddast-Navaeiauthor=B.+Srinivasanauthor=J.+Skolnick&title=On+the+importance+of+composite+protein+multiple+ligand+interactions+in+protein+pockets&doi=10.1002%2Fjcc.24523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">On the importance of composite protein multiple ligand interactions in protein pockets</span></div><div class="casAuthors">Tonddast-Navaei, Sam; Srinivasan, Bharath; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1252-1259</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Conventional small mol. drug-discovery approaches target protein pockets.  However, the limited no. of geometrically distinct pockets leads to widespread promiscuity and deleterious side-effects.  Here, the idea of COmposite protein LIGands (COLIG) that interact with each other as well as the protein within a single ligand binding pocket is examd.  As a practical illustration, exptl. evidence that E. coli Dihydrofolate reductase inhibitors are COLIGs is presented.  Then, anal. of a nonredundant set of all holo PDB structures indicates that almost 47-76% of proteins (based on different sequence identity thresholds) can simultaneously bind multiple, interacting ligands in the same pocket.  Moreover, most ligands that are either Singletons and COLIGs bind at the bottom of ligand binding pocket and occupy 30% and 43% of the vol. of the bottom of the pocket.  This suggests the use of COLIGs as a potential new class of small mol. drugs.  © 2016 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjL7YEfq49pLVg90H21EOLACvtfcHk0ljG-w0QnkOJ4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVygt7vL&md5=0b3bc3ceb75d89e6314b1f5fec740061</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fjcc.24523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.24523%26sid%3Dliteratum%253Aachs%26aulast%3DTonddast-Navaei%26aufirst%3DS.%26aulast%3DSrinivasan%26aufirst%3DB.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DOn%2520the%2520importance%2520of%2520composite%2520protein%2520multiple%2520ligand%2520interactions%2520in%2520protein%2520pockets%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2017%26volume%3D38%26spage%3D1252%26epage%3D1259%26doi%3D10.1002%2Fjcc.24523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">How beyond rule of 5 drugs and clinical candidates bind to their targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2312-2327&author=B.+C.+Doakauthor=J.+Zhengauthor=D.+Dobritzschauthor=J.+Kihlberg&title=How+beyond+rule+of+5+drugs+and+clinical+candidates+bind+to+their+targets&doi=10.1021%2Facs.jmedchem.5b01286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></div><div class="casAuthors">Doak, Bradley C.; Zheng, Jie; Dobritzsch, Doreen; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2327</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve discovery of drugs for difficult targets, the opportunities of chem. space beyond the rule of 5 (bRo5) were examd. by retrospective anal. of a comprehensive set of structures for complexes between drugs and clin. candidates and their targets.  The anal. illustrates the potential of compds. far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites.  However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required.  Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility.  However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites.  These insights should contribute to altering perceptions of what targets are considered "druggable" and provide support for drug design in beyond rule of 5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZD-h9VodiPLVg90H21EOLACvtfcHk0ljG-w0QnkOJ4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F&md5=a417b605d1a3ffbc7ee84706401f01f4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520beyond%2520rule%2520of%25205%2520drugs%2520and%2520clinical%2520candidates%2520bind%2520to%2520their%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2312%26epage%3D2327%26doi%3D10.1021%2Facs.jmedchem.5b01286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohnuud, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mottarella, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">755</span>, <span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-755&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+family+of+web+servers+for+determining+and+characterizing+ligand-binding+hot+spots+of+proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0lgntsdryBKUrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520family%2520of%2520web%2520servers%2520for%2520determining%2520and%2520characterizing%2520ligand-binding%2520hot%2520spots%2520of%2520proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D733%26epage%3D755%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lgntsdryBKUrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">The PDBbind database: Collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2977</span>– <span class="NLM_lpage">2980</span>, <span class="refDoi"> DOI: 10.1021/jm030580l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030580l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2977-2980&author=R.+Wangauthor=X.+Fangauthor=Y.+Luauthor=S.+Wang&title=The+PDBbind+database%3A+Collection+of+binding+affinities+for+protein-ligand+complexes+with+known+three-dimensional+structures&doi=10.1021%2Fjm030580l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The PDBbind database: Collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span></div><div class="casAuthors">Wang, Renxiao; Fang, Xueliang; Lu, Yipin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2977-2980</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have screened the entire Protein Data Bank (Release No. 103, Jan. 2003) and identified 5671 protein-ligand complexes out of 19 621 exptl. structures.  A systematic examn. of the primary refs. of these entries has led to a collection of binding affinity data (Kd, Ki, and IC50) for a total of 1359 complexes.  The outcomes of this project have been organized into a Web-accessible database named the PDBbind database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1eUdeWioYBbVg90H21EOLACvtfcHk0lgntsdryBKUrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D&md5=86e609172307402d8b0d4589b1270a2f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm030580l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030580l%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520PDBbind%2520database%253A%2520Collection%2520of%2520binding%2520affinities%2520for%2520protein-ligand%2520complexes%2520with%2520known%2520three-dimensional%2520structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2977%26epage%3D2980%26doi%3D10.1021%2Fjm030580l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span> <span> </span><span class="NLM_article-title">Binding MOAD (mother of all databases)</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">340</span>, <span class="refDoi"> DOI: 10.1002/prot.20512</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1002%2Fprot.20512" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=15971202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVyrtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2005&pages=333-340&author=L.+Huauthor=M.+L.+Bensonauthor=R.+D.+Smithauthor=M.+G.+Lernerauthor=H.+A.+Carlson&title=Binding+MOAD+%28mother+of+all+databases%29&doi=10.1002%2Fprot.20512"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Binding MOAD (Mother of All Databases)</span></div><div class="casAuthors">Hu, Liegi; Benson, Mark L.; Smith, Richard D.; Lerner, Michael G.; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">333-340</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Binding MOAD (Mother of All Databases) is the largest collection of high-quality, protein-ligand complexes available from the Protein Data Bank.  At this time, Binding MOAD contains 5331 protein-ligand complexes comprised of 1780 unique protein families and 2630 unique ligands.  We have searched the crystallog. papers for all 5000 + structures and compiled binding data for 1375 (26%) of the protein-ligand complexes.  The binding-affinity data ranges 13 orders of magnitude.  This is the largest collection of binding data reported to date in the literature.  We have also addressed the issue of redundancy in the data.  To create a nonredundant dataset, one protein from each of the 1780 protein families was chosen as a representative.  Representatives were chosen by tightest binding, best resoln., etc.  For the 1780 "best" complexes that comprise the nonredundant version of Binding MOAD, 475 (27%) have binding data.  This significant collection of protein-ligand complexes will be very useful in elucidating the biophys. patterns of mol. recognition and enzymic regulation.  The complexes with binding-affinity data will help in the development of improved scoring functions and structure-based drug discovery techniques.  The dataset can be accessed at http://www.BindingMOAD.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGronvDiEFLeLbVg90H21EOLACvtfcHk0lim8l_pCVoVEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVyrtr4%253D&md5=146632a2f30fda98cb987e984beb7506</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fprot.20512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20512%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%26aulast%3DBenson%26aufirst%3DM.%2BL.%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DLerner%26aufirst%3DM.%2BG.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DBinding%2520MOAD%2520%2528mother%2520of%2520all%2520databases%2529%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D60%26spage%3D333%26epage%3D340%26doi%3D10.1002%2Fprot.20512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorissen, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">BindingDB: A web-accessible database of experimentally determined protein–ligand binding affinities</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">D198</span>– <span class="NLM_lpage">D201</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fnar%2Fgkl999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=17145705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=D198-D201&author=T.+Liuauthor=Y.+Linauthor=X.+Wenauthor=R.+N.+Jorissenauthor=M.+K.+Gilson&title=BindingDB%3A+A+web-accessible+database+of+experimentally+determined+protein%E2%80%93ligand+binding+affinities&doi=10.1093%2Fnar%2Fgkl999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB: a web-accessible database of experimentally determined protein-ligand binding affinities</span></div><div class="casAuthors">Liu, Tiqing; Lin, Yuhmei; Wen, Xin; Jorissen, Robert N.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D198-D201</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">BindingDB is a publicly accessible database currently contg. ∼20 000 exptl. detd. binding affinities of protein-ligand complexes, for 110 protein targets including isoforms and mutational variants, and ∼11 000 small mol. ligands.  The data are extd. from the scientific literature, data collection focusing on proteins that are drug-targets or candidate drug-targets and for which structural data are present in the Protein Data Bank.  The BindingDB website supports a range of query types, including searches by chem. structure, substructure and similarity; protein sequence; ligand and protein names; affinity ranges and mol. wt.  Data sets generated by BindingDB queries can be downloaded in the form of annotated SDfiles for further anal., or used as the basis for virtual screening of a compd. database uploaded by the user.  The data in BindingDB are linked both to structural data in the PDB via PDB IDs and chem. and sequence searches, and to the literature in PubMed via PubMed IDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoymy3G_LuG-rVg90H21EOLACvtfcHk0lim8l_pCVoVEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D&md5=0ccb20d9b9178a624d4829b5909e7ff8</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl999%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DJorissen%26aufirst%3DR.%2BN.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520A%2520web-accessible%2520database%2520of%2520experimentally%2520determined%2520protein%25E2%2580%2593ligand%2520binding%2520affinities%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DD198%26epage%3DD201%26doi%3D10.1093%2Fnar%2Fgkl999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalovich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: A large-scale bioactivity database for drug discovery</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">D1100</span>– <span class="NLM_lpage">1107</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+A+large-scale+bioactivity+database+for+drug+discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0lim8l_pCVoVEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520A%2520large-scale%2520bioactivity%2520database%2520for%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3D1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span> <span> </span><span class="NLM_article-title">Unraveling hot spots in binding interfaces: Progress and challenges</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">14</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1016/S0959-440X(02)00283-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2FS0959-440X%2802%2900283-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11839484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=14-20&author=W.+L.+DeLano&title=Unraveling+hot+spots+in+binding+interfaces%3A+Progress+and+challenges&doi=10.1016%2FS0959-440X%2802%2900283-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Unraveling hot spots in binding interfaces: progress and challenges</span></div><div class="casAuthors">DeLano, Warren L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14-20</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 49 refs.  Protein interface hot spots, as revealed by alanine scanning mutagenesis, continue to stimulate interest in the biophys. basis of mol. recognition.  Although these regions apparently constitute fertile grounds for intermol. interactions, no general algorithm has yet been developed that can predict hot spots based solely on their shape or compn.  The discovery of structural plasticity in hot spot regions indicates that dynamic simulation techniques may be essential for achieving a predictive understanding of binding interface energetics.  Future progress will depend as much on the application of new computational approaches for dissecting protein interfaces as on expanding the empirical databank of mutagenic substitutions and their effects.  Despite current theor. shortcomings, recent methodol. advances provide efficient exptl. means of probing hot spots and enable immediate applications for hot spots in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk1HNI-4QfrrVg90H21EOLACvtfcHk0lgBZEXJKyyp1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D&md5=a297ae443e69c94b7cf2e342337f29ea</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2802%2900283-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252802%252900283-X%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26atitle%3DUnraveling%2520hot%2520spots%2520in%2520binding%2520interfaces%253A%2520Progress%2520and%2520challenges%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D12%26spage%3D14%26epage%3D20%26doi%3D10.1016%2FS0959-440X%2802%2900283-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Probing hot spots at protein-ligand binding sites: A fragment-based approach using biophysical methods</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">4992</span>– <span class="NLM_lpage">5000</span>, <span class="refDoi"> DOI: 10.1021/jm060490r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060490r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4992-5000&author=A.+Ciulliauthor=G.+Williamsauthor=A.+G.+Smithauthor=T.+L.+Blundellauthor=C.+Abell&title=Probing+hot+spots+at+protein-ligand+binding+sites%3A+A+fragment-based+approach+using+biophysical+methods&doi=10.1021%2Fjm060490r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Probing Hot Spots at Protein-Ligand Binding Sites: A Fragment-Based Approach Using Biophysical Methods</span></div><div class="casAuthors">Ciulli, Alessio; Williams, Glyn; Smith, Alison G.; Blundell, Tom L.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4992-5000</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mapping interactions at protein-ligand binding sites is an important aspect of understanding many biol. reactions and a key part of drug design.  In this paper, we have used a fragment-based approach to probe "hot spots" at the cofactor-binding site of a model dehydrogenase, Escherichia coli ketopantoate reductase.  Our strategy involved the breaking down of NADPH (Kd = 300 nM) into smaller fragments and the biophys. characterization of their binding using WaterLOGSY NMR spectroscopy, isothermal titrn. calorimetry (ITC), and inhibition studies.  The weak binding affinities of fragments were measured by direct ITC titrns. under low c value conditions.  The 2'-phosphate and the reduced nicotinamide groups were found to contribute a large part of the binding energy.  A combination of ITC and site-directed mutagenesis enabled us to locate the fragments at sep. hot spots on opposite ends of the cofactor-binding site.  This study has identified structural determinants for cofactor recognition that represent a blueprint for future inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAWWHBHKxvxbVg90H21EOLACvtfcHk0lgBZEXJKyyp1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D&md5=b6788ad16d6009d7cacd3e429b13523b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm060490r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060490r%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%2BG.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DProbing%2520hot%2520spots%2520at%2520protein-ligand%2520binding%2520sites%253A%2520A%2520fragment-based%2520approach%2520using%2520biophysical%2520methods%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D4992%26epage%3D5000%26doi%3D10.1021%2Fjm060490r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfleger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopitz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeiffer-Marek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baringhaus, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span> <span> </span><span class="NLM_article-title">Hot spots and transient pockets: Predicting the determinants of small-molecule binding to a protein-protein interface</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.1021/ci200322s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200322s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=120-133&author=A.+Metzauthor=C.+Pflegerauthor=H.+Kopitzauthor=S.+Pfeiffer-Marekauthor=K.+H.+Baringhausauthor=H.+Gohlke&title=Hot+spots+and+transient+pockets%3A+Predicting+the+determinants+of+small-molecule+binding+to+a+protein-protein+interface&doi=10.1021%2Fci200322s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Hot Spots and Transient Pockets: Predicting the Determinants of Small-Molecule Binding to a Protein-Protein Interface</span></div><div class="casAuthors">Metz, Alexander; Pfleger, Christopher; Kopitz, Hannes; Pfeiffer-Marek, Stefania; Baringhaus, Karl-Heinz; Gohlke, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">120-133</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interfaces are considered difficult targets for small-mol. protein-protein interaction modulators (PPIMs).  Here, the authors present for the first time a computational strategy that simultaneously considers aspects of energetics and plasticity in the context of PPIM binding to a protein interface.  The strategy aims at identifying the determinants of small-mol. binding, hot spots, and transient pockets, in a protein-protein interface to make use of this knowledge for predicting binding modes of and ranking PPIMs with respect to their affinity.  When applied to interleukin-2 (IL-2), the computationally inexpensive constrained geometric simulation method FRODA outperforms mol. dynamics simulations in sampling hydrophobic transient pockets.  The authors introduce the PPIAnalyzer approach for identifying transient pockets on the basis of geometrical criteria only.  A sequence of docking to identified transient pockets, starting structure selection based on hot spot information, RMSD clustering and intermol. docking energies, and MM-PBSA calcns. allows one to enrich IL-2 PPIMs from a set of decoys and to discriminate between subgroups of IL-2 PPIMs with low and high affinity.  The authors' strategy will be applicable in a prospective manner where nothing else than a protein-protein complex structure is known; hence, it can well be the first step in a structure-based endeavor to identify PPIMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol37VBT1jptbVg90H21EOLACvtfcHk0lgBZEXJKyyp1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt73I&md5=27aa41d22851e357d7edd40c7b7cbf28</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci200322s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200322s%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DA.%26aulast%3DPfleger%26aufirst%3DC.%26aulast%3DKopitz%26aufirst%3DH.%26aulast%3DPfeiffer-Marek%26aufirst%3DS.%26aulast%3DBaringhaus%26aufirst%3DK.%2BH.%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DHot%2520spots%2520and%2520transient%2520pockets%253A%2520Predicting%2520the%2520determinants%2520of%2520small-molecule%2520binding%2520to%2520a%2520protein-protein%2520interface%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D120%26epage%3D133%26doi%3D10.1021%2Fci200322s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Lessons from hot spot analysis for fragment-based drug discovery</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.tips.2015.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=26538314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=724-736&author=D.+R.+Hallauthor=D.+Kozakovauthor=A.+Whittyauthor=S.+Vajda&title=Lessons+from+hot+spot+analysis+for+fragment-based+drug+discovery&doi=10.1016%2Fj.tips.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons from hot spot analysis for fragment-based drug discovery</span></div><div class="casAuthors">Hall, David R.; Kozakov, Dima; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">724-736</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Anal. of binding energy hot spots at protein surfaces can provide crucial insights into the prospects for successful application of fragment-based drug discovery (FBDD), and whether a fragment hit can be advanced into a high-affinity, drug-like ligand.  The key factor is the strength of the top ranking hot spot, and how well a given fragment complements it.  We show that published data are sufficient to provide a sophisticated and quant. understanding of how hot spots derive from a protein 3D structure, and how their strength, no., and spatial arrangement govern the potential for a surface site to bind to fragment-sized and larger ligands.  This improved understanding provides important guidance for the effective application of FBDD in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdQmPulmhPx7Vg90H21EOLACvtfcHk0lj40RnrqMnVeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF&md5=3e682f28482d162590826f8f2e321b96</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLessons%2520from%2520hot%2520spot%2520analysis%2520for%2520fragment-based%2520drug%2520discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D724%26epage%3D736%26doi%3D10.1016%2Fj.tips.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kortvelyesi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Computational mapping identifies the binding sites of organic solvents on proteins</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">4290</span>– <span class="NLM_lpage">4295</span>, <span class="refDoi"> DOI: 10.1073/pnas.062398499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1073%2Fpnas.062398499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11904374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=4290-4295&author=S.+Dennisauthor=T.+Kortvelyesiauthor=S.+Vajda&title=Computational+mapping+identifies+the+binding+sites+of+organic+solvents+on+proteins&doi=10.1073%2Fpnas.062398499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Computational mapping identifies the binding sites of organic solvents on proteins</span></div><div class="casAuthors">Dennis, Sheldon; Kortvelyesi, Tamas; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4290-4295</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Computational mapping places mol. probes-small mols. or functional groups-on a protein surface to identify the most favorable binding positions.  Although x-ray crystallog. and NMR show that org. solvents bind to a limited no. of sites on a protein, current mapping methods result in hundreds of energy min. and do not reveal why some sites bind mols. with different sizes and polarities.  We describe a mapping algorithm that explains the origin of this phenomenon.  The algorithm has been applied to hen egg-white lysozyme and to thermolysin, interacting with eight and four different ligands, resp.  In both cases the search finds the consensus site to which all mols. bind, whereas other positions that bind only certain ligands are not necessarily found.  The consensus sites are pockets of the active site, lined with partially exposed hydrophobic residues and with a no. of polar residues toward the edge.  These sites can accommodate each ligand in a no. of rotational states, some with a hydrogen bond to one of the nearby donor/acceptor groups.  Specific substrates and/or inhibitors of hen egg-white lysozyme and thermolysin interact with the same side chains identified by the mapping, but form several hydrogen bonds and bind in unique orientations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4sEo2VxwF7Vg90H21EOLACvtfcHk0lj40RnrqMnVeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D&md5=90d71ffcc8860e87aefec3c5809b1127</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.062398499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.062398499%26sid%3Dliteratum%253Aachs%26aulast%3DDennis%26aufirst%3DS.%26aulast%3DKortvelyesi%26aufirst%3DT.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DComputational%2520mapping%2520identifies%2520the%2520binding%2520sites%2520of%2520organic%2520solvents%2520on%2520proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D4290%26epage%3D4295%26doi%3D10.1073%2Fpnas.062398499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landon, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancia, D. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Identification of hot spots within druggable binding regions by computational solvent mapping of proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1231</span>– <span class="NLM_lpage">1240</span>, <span class="refDoi"> DOI: 10.1021/jm061134b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061134b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1231-1240&author=M.+R.+Landonauthor=D.+R.+Lanciaauthor=J.+Yuauthor=S.+C.+Thielauthor=S.+Vajda&title=Identification+of+hot+spots+within+druggable+binding+regions+by+computational+solvent+mapping+of+proteins&doi=10.1021%2Fjm061134b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Hot Spots within Druggable Binding Regions by Computational Solvent Mapping of Proteins</span></div><div class="casAuthors">Landon, Melissa R.; Lancia, David R., Jr.; Yu, Jessamin; Thiel, Spencer C.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1231-1240</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here the authors apply the computational solvent mapping (CS-Map) algorithm toward the in silico identification of hot spots, i.e., regions of protein binding sites that are major contributors to the binding energy and, hence, are prime targets in drug design.  The CS-Map algorithm, developed for binding site characterization, moves small org. functional groups around the protein surface and dets. their most energetically favorable binding positions.  The utility of CS-Map algorithm toward the prediction of hot spot regions in druggable binding pockets is illustrated by three test systems: (1) renin aspartic protease, (2) a set of previously characterized druggable proteins, and (3) E. coli ketopantoate reductase.  In each of the three studies, existing literature was used to verify the results.  Based on the authors analyses, the authors conclude that the information provided by CS-Map can contribute substantially to the identification of hot spots, a necessary predecessor of fragment-based drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTVim9shdtBbVg90H21EOLACvtfcHk0lj40RnrqMnVeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWisrw%253D&md5=2d6a0678acdb2dd45fed73b05229bd03</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm061134b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061134b%26sid%3Dliteratum%253Aachs%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DLancia%26aufirst%3DD.%2BR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DThiel%26aufirst%3DS.%2BC.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520hot%2520spots%2520within%2520druggable%2520binding%2520regions%2520by%2520computational%2520solvent%2520mapping%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1231%26epage%3D1240%26doi%3D10.1021%2Fjm061134b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landon, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, Q. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caaveiro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orwig, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petsko, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringe, D.</span></span> <span> </span><span class="NLM_article-title">Detection of ligand binding hot spots on protein surfaces via fragment-based methods: Application to DJ-1 and glucocerebrosidase</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1007/s10822-009-9283-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1007%2Fs10822-009-9283-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=19521672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=491-500&author=M.+R.+Landonauthor=R.+L.+Liebermanauthor=Q.+Q.+Hoangauthor=S.+Juauthor=J.+M.+Caaveiroauthor=S.+D.+Orwigauthor=D.+Kozakovauthor=R.+Brenkeauthor=G.+Y.+Chuangauthor=D.+Beglovauthor=S.+Vajdaauthor=G.+A.+Petskoauthor=D.+Ringe&title=Detection+of+ligand+binding+hot+spots+on+protein+surfaces+via+fragment-based+methods%3A+Application+to+DJ-1+and+glucocerebrosidase&doi=10.1007%2Fs10822-009-9283-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of ligand binding hot spots on protein surfaces via fragment-based methods: application to DJ-1 and glucocerebrosidase</span></div><div class="casAuthors">Landon, Melissa R.; Lieberman, Raquel L.; Hoang, Quyen Q.; Ju, Shulin; Caaveiro, Jose M. M.; Orwig, Susan D.; Kozakov, Dima; Brenke, Ryan; Chuang, Gwo-Yu; Beglov, Dmitry; Vajda, Sandor; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">491-500</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The identification of hot spots, i.e., binding regions that contribute substantially to the free energy of ligand binding, is a crit. step for structure-based drug design.  Here we present the application of two fragment-based methods to the detection of hot spots for DJ-1 and glucocerebrosidase (GCase), targets for the development of therapeutics for Parkinson's and Gaucher's diseases, resp.  While the structures of these two proteins are known, binding information is lacking.  In this study we employ the exptl. multiple solvent crystal structures (MSCS) method and computational fragment mapping (FTMap) to identify regions suitable for the development of pharmacol. chaperones for DJ-1 and GCase.  Comparison of data derived via MSCS and FTMap also shows that FTMap, a computational method for the identification of fragment binding hot spots, is an accurate and robust alternative to the performance of expensive and difficult crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVrKBMCEfe6LVg90H21EOLACvtfcHk0lhb6-LB9iHVHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtb8%253D&md5=00a35c8ecaaa3b9f3f9aeff7c7e61d27</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9283-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9283-2%26sid%3Dliteratum%253Aachs%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DLieberman%26aufirst%3DR.%2BL.%26aulast%3DHoang%26aufirst%3DQ.%2BQ.%26aulast%3DJu%26aufirst%3DS.%26aulast%3DCaaveiro%26aufirst%3DJ.%2BM.%26aulast%3DOrwig%26aufirst%3DS.%2BD.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DDetection%2520of%2520ligand%2520binding%2520hot%2520spots%2520on%2520protein%2520surfaces%2520via%2520fragment-based%2520methods%253A%2520Application%2520to%2520DJ-1%2520and%2520glucocerebrosidase%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D491%26epage%3D500%26doi%3D10.1007%2Fs10822-009-9283-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarnieri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Binding hot spots and amantadine orientation in the influenza a virus M2 proton channel</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">2846</span>– <span class="NLM_lpage">2853</span>, <span class="refDoi"> DOI: 10.1016/j.bpj.2009.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.bpj.2009.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=19917240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhslartbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2009&pages=2846-2853&author=G.+Y.+Chuangauthor=D.+Kozakovauthor=R.+Brenkeauthor=D.+Beglovauthor=F.+Guarnieriauthor=S.+Vajda&title=Binding+hot+spots+and+amantadine+orientation+in+the+influenza+a+virus+M2+proton+channel&doi=10.1016%2Fj.bpj.2009.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Binding hot spots and amantadine orientation in the influenza A virus M2 proton channel</span></div><div class="casAuthors">Chuang, Gwo-Yu; Kozakov, Dima; Brenke, Ryan; Beglov, Dmitri; Guarnieri, Frank; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2846-2853</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Structures of truncated versions of the influenza A virus M2 proton channel have been detd. recently by x-ray crystallog. in the open conformation of the channel, and by NMR in the closed state.  The structures differ in the position of the bound inhibitors.  The x-ray structure shows a single amantadine mol. in the middle of the channel, whereas in the NMR structure four drug mols. bind at the channel's outer surface.  To study this controversy we applied computational solvent mapping, a technique developed for the identification of the most favorable drug binding hot spots of proteins.  The method moves mol. probes - small org. mols. contg. various functional groups - around the protein surface, finds favorable positions using empirical free energy functions, clusters the conformations, and ranks the clusters on the basis of the av. free energy.  The results of the mapping show that in both structures the primary hot spot is an internal cavity overlapping the amantadine binding site seen in the x-ray structure.  However, both structures also have weaker hot spots at the exterior locations that bind rimantadine in the NMR structure, although these sites are partially due to the favorable interactions with the interfacial region of the lipid bilayer.  As confirmed by docking calcns., the open channel binds amantadine at the more favorable internal site, in good agreement with the x-ray structure.  In contrast, the NMR structure is based on a peptide/micelle construct that is able to accommodate the small mol. probes used for the mapping, but has a too narrow pore for the rimantadine to access the internal hot spot, and hence the drug can bind only at the exterior sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC0NckQ50aUbVg90H21EOLACvtfcHk0lhb6-LB9iHVHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhslartbs%253D&md5=004c2022981283d2d9f85218e49d898a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2009.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2009.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DGuarnieri%26aufirst%3DF.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DBinding%2520hot%2520spots%2520and%2520amantadine%2520orientation%2520in%2520the%2520influenza%2520a%2520virus%2520M2%2520proton%2520channel%26jtitle%3DBiophys.%2520J.%26date%3D2009%26volume%3D97%26spage%3D2846%26epage%3D2853%26doi%3D10.1016%2Fj.bpj.2009.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buhrman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kearney, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoleon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovrigina, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovrigin, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattos, C.</span></span> <span> </span><span class="NLM_article-title">Analysis of binding site hot spots on the surface of Ras GTPase</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>413</i></span>,  <span class="NLM_fpage">773</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2011.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.jmb.2011.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=21945529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKisbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=413&publication_year=2011&pages=773-789&author=G.+Buhrmanauthor=C.+O%E2%80%99Connorauthor=B.+Zerbeauthor=B.+M.+Kearneyauthor=R.+Napoleonauthor=E.+A.+Kovriginaauthor=S.+Vajdaauthor=D.+Kozakovauthor=E.+L.+Kovriginauthor=C.+Mattos&title=Analysis+of+binding+site+hot+spots+on+the+surface+of+Ras+GTPase&doi=10.1016%2Fj.jmb.2011.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Binding Site Hot Spots on the Surface of Ras GTPase</span></div><div class="casAuthors">Buhrman, Greg; O'Connor, Casey; Zerbe, Brandon; Kearney, Bradley M.; Napoleon, Raeanne; Kovrigina, Elizaveta A.; Vajda, Sandor; Kozakov, Dima; Kovrigin, Evgenii L.; Mattos, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">413</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">773-789</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We have recently discovered an allosteric switch in Ras, bringing an addnl. level of complexity to this GTPase whose mutants are involved in nearly 30% of cancers.  Upon activation of the allosteric switch, there is a shift in helix 3/loop 7 assocd. with a disorder to order transition in the active site.  Here, we use a combination of multiple solvent crystal structures and computational solvent mapping (FTMap) to det. binding site hot spots in the "off" and "on" allosteric states of the GTP-bound form of H-Ras.  Thirteen sites are revealed, expanding possible target sites for ligand binding well beyond the active site.  Comparison of FTMaps for the H and K isoforms reveals essentially identical hot spots.  Furthermore, using NMR measurements of spin relaxation, we detd. that K-Ras exhibits global conformational dynamics very similar to those we previously reported for H-Ras.  We thus hypothesize that the global conformational rearrangement serves as a mechanism for allosteric coupling between the effector interface and remote hot spots in all Ras isoforms.  At least with respect to the binding sites involving the G domain, H-Ras is an excellent model for K-Ras and probably N-Ras as well.  Ras has so far been elusive as a target for drug design.  The present work identifies various unexplored hot spots throughout the entire surface of Ras, extending the focus from the disordered active site to well-ordered locations that should be easier to target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAmYA3G3UBirVg90H21EOLACvtfcHk0liILUEoq5H0Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKisbrN&md5=d54dee9a6ac58d27f87d85a6d25415a3</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DBuhrman%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DC.%26aulast%3DZerbe%26aufirst%3DB.%26aulast%3DKearney%26aufirst%3DB.%2BM.%26aulast%3DNapoleon%26aufirst%3DR.%26aulast%3DKovrigina%26aufirst%3DE.%2BA.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DKovrigin%26aufirst%3DE.%2BL.%26aulast%3DMattos%26aufirst%3DC.%26atitle%3DAnalysis%2520of%2520binding%2520site%2520hot%2520spots%2520on%2520the%2520surface%2520of%2520Ras%2520GTPase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D413%26spage%3D773%26epage%3D789%26doi%3D10.1016%2Fj.jmb.2011.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span> <span> </span><span class="NLM_article-title">Relationship between hot spot residues and ligand binding hot spots in protein-protein interfaces</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2236</span>– <span class="NLM_lpage">2244</span>, <span class="refDoi"> DOI: 10.1021/ci300175u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300175u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpsl2js7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2236-2244&author=B.+S.+Zerbeauthor=D.+R.+Hallauthor=S.+Vajdaauthor=A.+Whittyauthor=D.+Kozakov&title=Relationship+between+hot+spot+residues+and+ligand+binding+hot+spots+in+protein-protein+interfaces&doi=10.1021%2Fci300175u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Relationship between Hot Spot Residues and Ligand Binding Hot Spots in Protein-Protein Interfaces</span></div><div class="casAuthors">Zerbe, Brandon S.; Hall, David R.; Vajda, Sandor; Whitty, Adrian; Kozakov, Dima</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2236-2244</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the context of protein-protein interactions, the term "hot spot" refers to a residue or cluster of residues that makes a major contribution to the binding free energy, as detd. by alanine scanning mutagenesis.  In contrast, in pharmaceutical research, a hot spot is a site on a target protein that has high propensity for ligand binding and hence is potentially important for drug discovery.  Here we examine the relationship between these two hot spot concepts by comparing alanine scanning data for a set of 15 proteins with results from mapping the protein surfaces for sites that can bind fragment-sized small mols.  We find the two types of hot spots are largely complementary; the residues protruding into hot spot regions identified by computational mapping or exptl. fragment screening are almost always themselves hot spot residues as defined by alanine scanning expts.  Conversely, a residue that is found by alanine scanning to contribute little to binding rarely interacts with hot spot regions on the partner protein identified by fragment mapping.  In spite of the strong correlation between the two hot spot concepts, they fundamentally differ, however.  In particular, while identification of a hot spot by alanine scanning establishes the potential to generate substantial interaction energy with a binding partner, there are addnl. topol. requirements to be a hot spot for small mol. binding.  Hence, only a minority of hot spots identified by alanine scanning represent sites that are potentially useful for small inhibitor binding, and it is this subset that is identified by exptl. or computational fragment screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEczZ5jWRflrVg90H21EOLACvtfcHk0liILUEoq5H0Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpsl2js7o%253D&md5=4b5e10c1dd8e2db2ed4ffda8a175de32</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci300175u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300175u%26sid%3Dliteratum%253Aachs%26aulast%3DZerbe%26aufirst%3DB.%2BS.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DKozakov%26aufirst%3DD.%26atitle%3DRelationship%2520between%2520hot%2520spot%2520residues%2520and%2520ligand%2520binding%2520hot%2520spots%2520in%2520protein-protein%2520interfaces%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2236%26epage%3D2244%26doi%3D10.1021%2Fci300175u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Golden, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayeg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sazinsky, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">Comprehensive experimental and computational analysis of binding energy hot spots at the NF-κB essential modulator/IKKβ protein-protein interface</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">6242</span>– <span class="NLM_lpage">6256</span>, <span class="refDoi"> DOI: 10.1021/ja400914z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja400914z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFKnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6242-6256&author=M.+S.+Goldenauthor=S.+M.+Coteauthor=M.+Sayegauthor=B.+S.+Zerbeauthor=E.+A.+Villarauthor=D.+Beglovauthor=S.+L.+Sazinskyauthor=R.+M.+Georgiadisauthor=S.+Vajdaauthor=D.+Kozakovauthor=A.+Whitty&title=Comprehensive+experimental+and+computational+analysis+of+binding+energy+hot+spots+at+the+NF-%CE%BAB+essential+modulator%2FIKK%CE%B2+protein-protein+interface&doi=10.1021%2Fja400914z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Experimental and Computational Analysis of Binding Energy Hot Spots at the NF-κB Essential Modulator/IKKβ Protein-Protein Interface</span></div><div class="casAuthors">Golden, Mary S.; Cote, Shaun M.; Sayeg, Marianna; Zerbe, Brandon S.; Villar, Elizabeth A.; Beglov, Dmitri; Sazinsky, Stephen L.; Georgiadis, Rosina M.; Vajda, Sandor; Kozakov, Dima; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6242-6256</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a comprehensive anal. of binding energy hot spots at the protein-protein interaction (PPI) interface between nuclear factor kappa B (NF-κB) essential modulator (NEMO) and IκB kinase subunit β (IKKβ), an interaction that is crit. for NF-κB pathway signaling, using exptl. alanine scanning mutagenesis and also the FTMap method for computational fragment screening.  The exptl. results confirm that the previously identified NEMO binding domain (NBD) region of IKKβ contains the highest concn. of hot-spot residues, the strongest of which are W739, W741, and L742 (ΔΔG = 4.3, 3.5, and 3.2 kcal/mol, resp.).  The region occupied by these residues defines a potentially druggable binding site on NEMO that extends for ∼16 Å to addnl. include the regions that bind IKKβ L737 and F734.  NBD residues D738 and S740 are also important for binding but do not make direct contact with NEMO, instead likely acting to stabilize the active conformation of surrounding residues.  We addnl. found two previously unknown hot-spot regions centered on IKKβ residues L708/V709 and L719/I723.  The computational approach successfully identified all three hot-spot regions on IKKβ.  Moreover, the method was able to accurately quantify the energetic importance of all hot-spot residues involving direct contact with NEMO.  Our results provide new information to guide the discovery of small-mol. inhibitors that target the NEMO/IKKβ interaction.  They addnl. clarify the structural and energetic complementarity between "pocket-forming" and "pocket-occupying" hot-spot residues, and further validate computational fragment mapping as a method for identifying hot spots at PPI interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLvGilMwRJGrVg90H21EOLACvtfcHk0liILUEoq5H0Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFKnsL0%253D&md5=d996a1aef7668ff03fe9bf3445c7d700</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja400914z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja400914z%26sid%3Dliteratum%253Aachs%26aulast%3DGolden%26aufirst%3DM.%2BS.%26aulast%3DCote%26aufirst%3DS.%2BM.%26aulast%3DSayeg%26aufirst%3DM.%26aulast%3DZerbe%26aufirst%3DB.%2BS.%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DSazinsky%26aufirst%3DS.%2BL.%26aulast%3DGeorgiadis%26aufirst%3DR.%2BM.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DComprehensive%2520experimental%2520and%2520computational%2520analysis%2520of%2520binding%2520energy%2520hot%2520spots%2520at%2520the%2520NF-%25CE%25BAB%2520essential%2520modulator%252FIKK%25CE%25B2%2520protein-protein%2520interface%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6242%26epage%3D6256%26doi%3D10.1021%2Fja400914z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoleon, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yueh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">New frontiers in druggability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9063</span>– <span class="NLM_lpage">9088</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9063-9088&author=D.+Kozakovauthor=D.+R.+Hallauthor=R.+L.+Napoleonauthor=C.+Yuehauthor=A.+Whittyauthor=S.+Vajda&title=New+frontiers+in+druggability&doi=10.1021%2Facs.jmedchem.5b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">New Frontiers in Druggability</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Napoleon, Raeanne L.; Yueh, Christine; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9063-9088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A powerful early approach to evaluating the druggability of proteins involved detg. the hit rate in NMR-based screening of a library of small compds.  Here, we show that a computational analog of this method, based on mapping proteins using small mols. as probes, can reliably reproduce druggability results from NMR-based screening and can provide a more meaningful assessment in cases where the two approaches disagree.  We apply the method to a large set of proteins.  The results show that, because the method is based on the biophysics of binding rather than on empirical parametrization, meaningful information can be gained about classes of proteins and classes of compds. beyond those resembling validated targets and conventionally druglike ligands.  In particular, the method identifies targets that, while not druggable by druglike compds., may become druggable using compd. classes such as macrocycles or other large mols. beyond the rule-of-five limit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCWoMwMpuM2bVg90H21EOLACvtfcHk0lisymCqOq_w_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF&md5=77fd532a1f0129d096eb7a1fbfb1d422</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00586%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DNapoleon%26aufirst%3DR.%2BL.%26aulast%3DYueh%26aufirst%3DC.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DNew%2520frontiers%2520in%2520druggability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9063%26epage%3D9088%26doi%3D10.1021%2Facs.jmedchem.5b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cencic, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelletier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Structural conservation of druggable hot spots in protein-protein interfaces</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">13528</span>– <span class="NLM_lpage">13533</span>, <span class="refDoi"> DOI: 10.1073/pnas.1101835108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1073%2Fpnas.1101835108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=21808046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2ms7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13528-13533&author=D.+Kozakovauthor=D.+R.+Hallauthor=G.+Y.+Chuangauthor=R.+Cencicauthor=R.+Brenkeauthor=L.+E.+Groveauthor=D.+Beglovauthor=J.+Pelletierauthor=A.+Whittyauthor=S.+Vajda&title=Structural+conservation+of+druggable+hot+spots+in+protein-protein+interfaces&doi=10.1073%2Fpnas.1101835108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structural conservation of druggable hot spots in protein-protein interfaces</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Chuang, Gwo-Yu; Cencic, Regina; Brenke, Ryan; Grove, Laurie E.; Beglov, Dmitri; Pelletier, Jerry; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">13528-13533, S13528/1-S13528/26</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Despite the growing no. of examples of small-mol. inhibitors that disrupt protein-protein interactions (PPIs), the origin of druggability of such targets is poorly understood.  To identify druggable sites in protein-protein interfaces we combine computational solvent mapping, which explores the protein surface using a variety of small "probe" mols., with a conformer generator to account for side-chain flexibility.  Applications to unliganded structures of 15 PPI target proteins show that the druggable sites comprise a cluster of binding hot spots, distinguishable from other regions of the protein due to their concave topol. combined with a pattern of hydrophobic and polar functionality.  This combination of properties confers on the hot spots a tendency to bind org. species possessing some polar groups decorating largely hydrophobic scaffolds.  Thus, druggable sites at PPI are not simply sites that are complementary to particular org. functionality, but rather possess a general tendency to bind org. compds. with a variety of structures, including key side chains of the partner protein.  Results also highlight the importance of conformational adaptivity at the binding site to allow the hot spots to expand to accommodate a ligand of drug-like dimensions.  The crit. components of this adaptivity are largely local, involving primarily low energy side-chain motions within 6 Å of a hot spot.  The structural and physicochem. signature of druggable sites at PPI interfaces is sufficiently robust to be detectable from the structure of the unliganded protein, even when substantial conformational adaptation is required for optimal ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpibu1osEQcZrVg90H21EOLACvtfcHk0lisymCqOq_w_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2ms7rL&md5=fc9b8a30f1b30c97bba5a843fb9dc03b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1101835108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1101835108%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DCencic%26aufirst%3DR.%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DPelletier%26aufirst%3DJ.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DStructural%2520conservation%2520of%2520druggable%2520hot%2520spots%2520in%2520protein-protein%2520interfaces%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D13528%26epage%3D13533%26doi%3D10.1073%2Fpnas.1101835108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cully, M.</span></span> <span> </span><span class="NLM_article-title">Milestone 9: Ximelagatran sets the stage for NOACs</span>. <i>Nat. Rev. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>, <span class="refDoi"> DOI: 10.1038/nrcardio.2017.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrcardio.2017.179" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=M.+Cully&title=Milestone+9%3A+Ximelagatran+sets+the+stage+for+NOACs&doi=10.1038%2Fnrcardio.2017.179"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnrcardio.2017.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrcardio.2017.179%26sid%3Dliteratum%253Aachs%26aulast%3DCully%26aufirst%3DM.%26atitle%3DMilestone%25209%253A%2520Ximelagatran%2520sets%2520the%2520stage%2520for%2520NOACs%26jtitle%3DNat.%2520Rev.%2520Cardiol.%26date%3D2017%26doi%3D10.1038%2Fnrcardio.2017.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keisu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, T. B.</span></span> <span> </span><span class="NLM_article-title">Drug-induced liver injury in humans: The case of ximelagatran</span>. <i>Handb. Exp. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>196</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1007/978-3-642-00663-0_13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1007%2F978-3-642-00663-0_13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvVymsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=196&publication_year=2010&pages=407-418&author=M.+Keisuauthor=T.+B.+Andersson&title=Drug-induced+liver+injury+in+humans%3A+The+case+of+ximelagatran&doi=10.1007%2F978-3-642-00663-0_13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-induced liver injury in humans: the case of ximelagatran</span></div><div class="casAuthors">Keisu, M.; Andersson, T. B.</div><div class="citationInfo"><span class="NLM_cas:title">Handbook of Experimental Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">196</span>
        (<span class="NLM_cas:issue">Adverse Drug Reactions</span>),
    <span class="NLM_cas:pages">407-418</span>CODEN:
                <span class="NLM_cas:coden">HEPHD2</span>;
        ISSN:<span class="NLM_cas:issn">0171-2004</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Ximelagatran was the first orally available direct thrombin inhibitor under clin. development that also reached the market.  Ximelagatran was tested in an extensive clin. program.  Short-term use (< 12 days) in humans including the phase III clin. trials did not indicate any hepatotoxic potential.  Increased hepatic enzyme levels were first obsd. at a higher frequency when evaluating the long-term (>35 days) use of ximelagatran (incidence of >3 × upper limit of normal (ULN) plasma ALT was 7.9%).  The frequency of elevated total bilirubin levels was similar in the ximelagatran and the comparator groups.  However, the combination of ALT > 3 × ULN and total bilirubin > 2 × ULN was 0.5% among patients treated with ximelagatran and 0.1% among patients in the comparator group.  Symptoms such as fever and rash potentially indicating hypersensitivity (immunol. type of reaction) were low and did not differ between ximelagatran and the comparators.  The withdrawal of ximelagatran from the market and termination of the ximelagatran development program was triggered by safety data from a 35-day study, indicating that severe hepatic injury in a patient could develop after exposure to the drug has been completed and that regular liver function monitoring may not mitigate the possible risk of severe hepatic injury.  As for many drugs causing liver injury, the std. preclin. toxicol. studies provided no indication that ximelagatran affected hepatic functions.  In addn., extensive investigations using human-based in vitro models have not been able to define mechanisms explaining the pattern of hepatic injury obsd. in long-term clin. trials.  A pharmacogenomic study provided evidence that the ALT increases were assocd. with major histocompatibility complex (MHC) alleles DRB1'07 and DQA1*02 suggesting a possible immunogenic pathogenesis.  This example provides important clues to the mechanism of idiosyncratic drug-induced liver toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8MnjfyasU1bVg90H21EOLACvtfcHk0li40N2G4iFBYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvVymsr8%253D&md5=8713782d6f002abd9d8f8ca6aa41d7e0</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-00663-0_13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-3-642-00663-0_13%26sid%3Dliteratum%253Aachs%26aulast%3DKeisu%26aufirst%3DM.%26aulast%3DAndersson%26aufirst%3DT.%2BB.%26atitle%3DDrug-induced%2520liver%2520injury%2520in%2520humans%253A%2520The%2520case%2520of%2520ximelagatran%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2010%26volume%3D196%26spage%3D407%26epage%3D418%26doi%3D10.1007%2F978-3-642-00663-0_13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riester, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirsching, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salinas, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebinoga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamphausen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkwirth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesenfeldt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturzebecher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedrich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thurk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwienhorst, A.</span></span> <span> </span><span class="NLM_article-title">Thrombin inhibitors identified by computer-assisted multiparameter design</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">8597</span>– <span class="NLM_lpage">8602</span>, <span class="refDoi"> DOI: 10.1073/pnas.0501983102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1073%2Fpnas.0501983102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=15937115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsFCgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=8597-8602&author=D.+Riesterauthor=F.+Wirschingauthor=G.+Salinasauthor=M.+Kellerauthor=M.+Gebinogaauthor=S.+Kamphausenauthor=C.+Merkwirthauthor=R.+Goetzauthor=M.+Wiesenfeldtauthor=J.+Sturzebecherauthor=W.+Bodeauthor=R.+Friedrichauthor=M.+Thurkauthor=A.+Schwienhorst&title=Thrombin+inhibitors+identified+by+computer-assisted+multiparameter+design&doi=10.1073%2Fpnas.0501983102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Thrombin inhibitors identified by computer-assisted multiparameter design</span></div><div class="casAuthors">Riester, Daniel; Wirsching, Frank; Salinas, Gabriela; Keller, Martina; Gebinoga, Michael; Kamphausen, Stefan; Merkwirth, Christian; Goetz, Ruediger; Wiesenfeldt, Martin; Stuerzebecher, Joerg; Bode, Wolfram; Friedrich, Rainer; Thuerk, Marcel; Schwienhorst, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8597-8602</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Here, the authors present a series of thrombin inhibitors that were generated by using powerful computer-assisted multiparameter optimization process.  The process was organized in design cycles, starting with a set of randomly chosen mols.  Each cycle combined combinatorial synthesis, multiparameter characterization of compds. in a variety of bioassays, and algorithmic processing of the data to devise a set of compds. to be synthesized in the next cycle.  The identified lead compds. exhibited thrombin inhibitory consts. in the lower nanomolar range.  They are by far the most selective synthetic thrombin inhibitors, with selectivities of >100,000-fold toward other proteases such as Factor Xa, Factor XIIa, urokinase, plasmin, and Plasma kallikrein.  Furthermore, these compds. exhibit a favorable profile, comprising nontoxicity, high metabolic stability, low serum protein binding, good soly., high anticoagulant activity, and a slow and exclusively renal elimination from the circulation in a rat model.  Finally, x-ray crystallog. anal. of a thrombin-inhibitor complex revealed a binding mode with a neutral moiety in the S1 pocket of thrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomQF-Oe6iPbbVg90H21EOLACvtfcHk0li40N2G4iFBYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsFCgtrc%253D&md5=f9f43256de6ddd302b4fe4591572a255</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0501983102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0501983102%26sid%3Dliteratum%253Aachs%26aulast%3DRiester%26aufirst%3DD.%26aulast%3DWirsching%26aufirst%3DF.%26aulast%3DSalinas%26aufirst%3DG.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DGebinoga%26aufirst%3DM.%26aulast%3DKamphausen%26aufirst%3DS.%26aulast%3DMerkwirth%26aufirst%3DC.%26aulast%3DGoetz%26aufirst%3DR.%26aulast%3DWiesenfeldt%26aufirst%3DM.%26aulast%3DSturzebecher%26aufirst%3DJ.%26aulast%3DBode%26aufirst%3DW.%26aulast%3DFriedrich%26aufirst%3DR.%26aulast%3DThurk%26aufirst%3DM.%26aulast%3DSchwienhorst%26aufirst%3DA.%26atitle%3DThrombin%2520inhibitors%2520identified%2520by%2520computer-assisted%2520multiparameter%2520design%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D8597%26epage%3D8602%26doi%3D10.1073%2Fpnas.0501983102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pool, J. L.</span></span> <span> </span><span class="NLM_article-title">Direct renin inhibition: Focus on aliskiren</span>. <i>J. Manag. Care Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.18553/jmcp.2007.13.s8-b.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.18553%2Fjmcp.2007.13.s8-b.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=17970614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BD2snptFKqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=21-33&author=J.+L.+Pool&title=Direct+renin+inhibition%3A+Focus+on+aliskiren&doi=10.18553%2Fjmcp.2007.13.s8-b.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Direct renin inhibition: focus on aliskiren</span></div><div class="casAuthors">Pool James L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of managed care pharmacy : JMCP</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8 Suppl B</span>),
    <span class="NLM_cas:pages">21-33</span>
        ISSN:<span class="NLM_cas:issn">1083-4087</span>.
    </div><div class="casAbstract">BACKGROUND:  Despite the availability of many effective, well-tolerated drugs, a significant proportion of treated hypertensive patients still have uncontrolled high blood pressure (BP) and thus face serious morbidity and mortality.  The renin-angiotensin aldosterone system (RAAS) is a key target for BP control and for cardiovascular and renal protection.  Renin controls the rate-limiting step in the RAAS cascade and hence is the optimal target for RAAS suppression.  Aliskiren is the first direct renin inhibitor (DRI) to be approved by the U.S.  Food and Drug Administration and the European Medicines Agency for treating hypertension.  OBJECTIVE:  To provide an overview of the pharmacology, pharmacokinetics, preclinical, and clinical efficacy and safety data on the DRI aliskiren.  RESULTS:  Approximately 70% of essential hypertension is associated with elevated renin levels.  Aliskiren is a potent and highly specific inhibitor of renin, with oral bioavailability of 2.6% and an elimination half-life of 40 hours, making it suitable for once-daily oral administration.  Aliskiren dose-dependently reduced BP, inhibited plasma renin activity (PRA), attenuated renal damage in animal models, and showed efficient and longer- lasting blockade of the RAAS in normotensive human subjects compared with other RAAS inhibitors.  The clinical efficacy and safety of aliskiren have been evaluated both as monotherapy and in combination with other antihypertensive agents in phase II and phase III trials of patients with mild to severe hypertension.  When used as monotherapy, aliskiren led to significant dose-dependent reductions in BP from baseline that were greater than those obtained with placebo and comparable with those achieved with an angiotensin II receptor blocker (ARB).  The combination of aliskiren with a diuretic, a calcium channel blocker (CCB), an angiotensin-converting enzyme inhibitor (ACEI), or an ARB generally had greater and longer-lasting BP-lowering efficacy than did single agents alone.  Aliskiren also countered the reactive increase in PRA caused by diuretic, CC B, ACEI, and ARB therapy.  Once-daily treatment with aliskiren was well tolerated.  CONCLUSIONS:  As a DRI, aliskiren blocks the RAAS more completely than do other current downstream RAAS inhibitors.  When used once daily, aliskiren is a safe and effective antihypertensive agent that can be used as monotherapy or in combination with other agents to provide additional options to improve BP control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS71W2Wu8cAkEg6wdEZmT2rfW6udTcc2ebdELIeDKgVc7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2snptFKqtA%253D%253D&md5=7f59662d64fbd7bbe4b5be1acdd40a5e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.18553%2Fjmcp.2007.13.s8-b.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18553%252Fjmcp.2007.13.s8-b.21%26sid%3Dliteratum%253Aachs%26aulast%3DPool%26aufirst%3DJ.%2BL.%26atitle%3DDirect%2520renin%2520inhibition%253A%2520Focus%2520on%2520aliskiren%26jtitle%3DJ.%2520Manag.%2520Care%2520Pharm.%26date%3D2007%26volume%3D13%26spage%3D21%26epage%3D33%26doi%3D10.18553%2Fjmcp.2007.13.s8-b.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sepehrdad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frishman, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stier, C. T.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sica, D. A.</span></span> <span> </span><span class="NLM_article-title">Direct inhibition of renin as a cardiovascular pharmacotherapy: Focus on aliskiren</span>. <i>Cardiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">242</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1097/CRD.0b013e318093e43a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1097%2FCRD.0b013e318093e43a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=17700383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BD2svovFensg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=242-256&author=R.+Sepehrdadauthor=W.+H.+Frishmanauthor=C.+T.+Stierauthor=D.+A.+Sica&title=Direct+inhibition+of+renin+as+a+cardiovascular+pharmacotherapy%3A+Focus+on+aliskiren&doi=10.1097%2FCRD.0b013e318093e43a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Direct inhibition of renin as a cardiovascular pharmacotherapy: focus on aliskiren</span></div><div class="casAuthors">Sepehrdad Reza; Frishman William H; Stier Charles T Jr; Sica Domenic A</div><div class="citationInfo"><span class="NLM_cas:title">Cardiology in review</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">242-56</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The important role of renin-angiotensin-aldosterone system blockade in the treatment of systemic hypertension, heart failure, diabetic kidney disease, and atherogenesis has been clearly established.  The theoretical therapeutic advantages for inhibiting the detrimental effects of the renin-angiotensin system at its most upstream point have served as the impetus for the development of renin inhibitors.  The advent of aliskiren, the first in a novel class of orally active, nonpeptide, highly specific, human renin inhibitors, provides a new modality in the armamentarium of renin-angiotensin system antagonists.  Studies in marmosets and rats demonstrated that aliskiren reduced blood pressure in a dose-dependent manner and is highly efficacious in blocking plasma renin activity with parallel reductions in the levels of the other downstream constituents of the renin-angiotensin system.  Clinical trials in hypertensive patients have confirmed these benefits with aliskiren whose blood pressure-lowering efficacy is similar to or better than those of standard therapeutic doses of enalapril, losartan, irbesartan, and hydrochlorothiazide.  Aliskiren is well tolerated, with few reported adverse effects even at the highest doses tested.  Given the established beneficial effects of angiotensin-converting enzyme inhibitors and angiotensin receptor blockers in the treatment of cardiovascular and renovascular diseases, future studies may further elucidate a similar protective role for aliskiren both as a monotherapy and as part of a combination therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQE6VB-Pf2cTkBuFdjgVNKSfW6udTcc2eY1dLv5wdvK9bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svovFensg%253D%253D&md5=add28099fe5ccc184b21059b89a4f42a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1097%2FCRD.0b013e318093e43a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCRD.0b013e318093e43a%26sid%3Dliteratum%253Aachs%26aulast%3DSepehrdad%26aufirst%3DR.%26aulast%3DFrishman%26aufirst%3DW.%2BH.%26aulast%3DStier%26aufirst%3DC.%2BT.%26aulast%3DSica%26aufirst%3DD.%2BA.%26atitle%3DDirect%2520inhibition%2520of%2520renin%2520as%2520a%2520cardiovascular%2520pharmacotherapy%253A%2520Focus%2520on%2520aliskiren%26jtitle%3DCardiol.%2520Rev.%26date%3D2007%26volume%3D15%26spage%3D242%26epage%3D256%26doi%3D10.1097%2FCRD.0b013e318093e43a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belunis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowther, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danho, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goggin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remiszewski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuglstatter, A.</span></span> <span> </span><span class="NLM_article-title">The structure of XIAP BIR2: Understanding the selectivity of the BIR domains</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1717</span>– <span class="NLM_lpage">1725</span>, <span class="refDoi"> DOI: 10.1107/S0907444913016284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1107%2FS0907444913016284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23999295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlKrsLnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1717-1725&author=C.+Lukacsauthor=C.+Belunisauthor=R.+Crowtherauthor=W.+Danhoauthor=L.+Gaoauthor=B.+Gogginauthor=C.+A.+Jansonauthor=S.+Liauthor=S.+Remiszewskiauthor=A.+Schuttauthor=M.+K.+Thakurauthor=S.+K.+Singhauthor=S.+Swaminathanauthor=R.+Pandeyauthor=R.+Tyagiauthor=R.+Gosuauthor=A.+V.+Kamathauthor=A.+Kuglstatter&title=The+structure+of+XIAP+BIR2%3A+Understanding+the+selectivity+of+the+BIR+domains&doi=10.1107%2FS0907444913016284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of XIAP BIR2: understanding the selectivity of the BIR domains</span></div><div class="casAuthors">Lukacs, Christine; Belunis, Charles; Crowther, Robert; Danho, Waleed; Gao, Lin; Goggin, Barry; Janson, Cheryl A.; Li, Shirley; Remiszewski, Stacy; Schutt, Andrew; Thakur, Manish K.; Singh, Saroj K.; Swaminathan, Srinivasan; Pandey, Rajat; Tyagi, Rajiv; Gosu, Ramachandraiah; Kamath, Ajith V.; Kuglstatter, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1717-1725</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">XIAP, a member of the inhibitor of apoptosis family of proteins, is a crit. regulator of apoptosis.  Inhibition of the BIR domain-caspase interaction is a promising approach towards treating cancer.  Previous work has been directed towards inhibiting the BIR3-caspase-9 interaction, which blocks the intrinsic apoptotic pathway; selectively inhibiting the BIR2-caspase-3 interaction would also block the extrinsic pathway.  The BIR2 domain of XIAP has successfully been crystd.; peptides and small-mol. inhibitors can be soaked into these crystals, which diffract to high resoln.  Here, the BIR2 apo crystal structure and the structures of five BIR2-tetrapeptide complexes are described.  The structural flexibility obsd. on comparing these structures, along with a comparison with XIAP BIR3, affords an understanding of the structural elements that drive selectivity between BIR2 and BIR3 and which can be used to design BIR2-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6-upk5-drzbVg90H21EOLACvtfcHk0lg_9X0oWW6TJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlKrsLnK&md5=deebf6694f2e53929d7c6ec4e763cf14</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444913016284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913016284%26sid%3Dliteratum%253Aachs%26aulast%3DLukacs%26aufirst%3DC.%26aulast%3DBelunis%26aufirst%3DC.%26aulast%3DCrowther%26aufirst%3DR.%26aulast%3DDanho%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DGoggin%26aufirst%3DB.%26aulast%3DJanson%26aufirst%3DC.%2BA.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DRemiszewski%26aufirst%3DS.%26aulast%3DSchutt%26aufirst%3DA.%26aulast%3DThakur%26aufirst%3DM.%2BK.%26aulast%3DSingh%26aufirst%3DS.%2BK.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DPandey%26aufirst%3DR.%26aulast%3DTyagi%26aufirst%3DR.%26aulast%3DGosu%26aufirst%3DR.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DKuglstatter%26aufirst%3DA.%26atitle%3DThe%2520structure%2520of%2520XIAP%2520BIR2%253A%2520Understanding%2520the%2520selectivity%2520of%2520the%2520BIR%2520domains%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1717%26epage%3D1725%26doi%3D10.1107%2FS0907444913016284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span> <span> </span><span class="NLM_article-title">Inhibitor of apoptosis protein (IAP) antagonists in anticancer agent discovery: Current status and perspectives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01668</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01668" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=H.+Congauthor=L.+Xuauthor=Y.+Wuauthor=Z.+Quauthor=T.+Bianauthor=W.+Zhangauthor=C.+Xingauthor=C.+Zhuang&title=Inhibitor+of+apoptosis+protein+%28IAP%29+antagonists+in+anticancer+agent+discovery%3A+Current+status+and+perspectives&doi=10.1021%2Facs.jmedchem.8b01668"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01668%26sid%3Dliteratum%253Aachs%26aulast%3DCong%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DQu%26aufirst%3DZ.%26aulast%3DBian%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXing%26aufirst%3DC.%26aulast%3DZhuang%26aufirst%3DC.%26atitle%3DInhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529%2520antagonists%2520in%2520anticancer%2520agent%2520discovery%253A%2520Current%2520status%2520and%2520perspectives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26doi%3D10.1021%2Facs.jmedchem.8b01668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pargellis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churchill, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirillo, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grob, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickey, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pav, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regan, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of p38 map kinase by utilizing a novel allosteric binding site</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1038/nsb770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnsb770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11896401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD38Xis1Kisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=268-272&author=C.+Pargellisauthor=L.+Tongauthor=L.+Churchillauthor=P.+F.+Cirilloauthor=T.+Gilmoreauthor=A.+G.+Grahamauthor=P.+M.+Grobauthor=E.+R.+Hickeyauthor=N.+Mossauthor=S.+Pavauthor=J.+Regan&title=Inhibition+of+p38+map+kinase+by+utilizing+a+novel+allosteric+binding+site&doi=10.1038%2Fnsb770"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of p38 MAP kinase by utilizing a novel allosteric binding site</span></div><div class="casAuthors">Pargellis, Christopher; Tong, Liang; Churchill, Laurie; Cirillo, Pier F.; Gilmore, Thomas; Graham, Anne G.; Grob, Peter M.; Hickey, Eugene R.; Moss, Neil; Pav, Susan; Regan, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">268-272</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">The p38 MAP kinase plays a crucial role in regulating the prodn. of proinflammatory cytokines, such as tumor necrosis factor and interleukin-1.  Blocking this kinase may offer an effective therapy for treating many inflammatory diseases.  Here we report a new allosteric binding site for a diaryl urea class of highly potent and selective inhibitors against human p38 MAP kinase.  The formation of this binding site requires a large conformational change not obsd. previously for any of the protein Ser/Thr kinases.  This change is in the highly conserved Asp-Phe-Gly motif within the active site of the kinase.  Soln. studies demonstrate that this class of compds. has slow binding kinetics, consistent with the requirement for conformational change.  Improving interactions in this allosteric pocket, as well as establishing binding interactions in the ATP pocket, enhanced the affinity of the inhibitors by 12,000-fold.  One of the most potent compds. in this series, BIRB 796, has picomolar affinity for the kinase and low nanomolar inhibitory activity in cell culture.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqT8_QT7_pbYrVg90H21EOLACvtfcHk0lgv5hFZnNTAZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xis1Kisbk%253D&md5=68850ecdbe08df4285ba38cfa2c93ea4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnsb770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb770%26sid%3Dliteratum%253Aachs%26aulast%3DPargellis%26aufirst%3DC.%26aulast%3DTong%26aufirst%3DL.%26aulast%3DChurchill%26aufirst%3DL.%26aulast%3DCirillo%26aufirst%3DP.%2BF.%26aulast%3DGilmore%26aufirst%3DT.%26aulast%3DGraham%26aufirst%3DA.%2BG.%26aulast%3DGrob%26aufirst%3DP.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DMoss%26aufirst%3DN.%26aulast%3DPav%26aufirst%3DS.%26aulast%3DRegan%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520p38%2520map%2520kinase%2520by%2520utilizing%2520a%2520novel%2520allosteric%2520binding%2520site%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D268%26epage%3D272%26doi%3D10.1038%2Fnsb770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Millan, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrows, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butcher, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairman, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilty, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemaitre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewthwaite, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahnke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathias, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philip, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefaniak, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeadon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, C.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of inhaled p38 inhibitors for the treatment of chronic obstructive pulmonary disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7797</span>– <span class="NLM_lpage">7814</span>, <span class="refDoi"> DOI: 10.1021/jm200677b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200677b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlOqu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7797-7814&author=D.+S.+Millanauthor=M.+E.+Bunnageauthor=J.+L.+Burrowsauthor=K.+J.+Butcherauthor=P.+G.+Doddauthor=T.+J.+Evansauthor=D.+A.+Fairmanauthor=S.+J.+Hughesauthor=I.+C.+Kiltyauthor=A.+Lemaitreauthor=R.+A.+Lewthwaiteauthor=A.+Mahnkeauthor=J.+P.+Mathiasauthor=J.+Philipauthor=R.+T.+Smithauthor=M.+H.+Stefaniakauthor=M.+Yeadonauthor=C.+Phillips&title=Design+and+synthesis+of+inhaled+p38+inhibitors+for+the+treatment+of+chronic+obstructive+pulmonary+disease&doi=10.1021%2Fjm200677b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Inhaled p38 Inhibitors for the Treatment of Chronic Obstructive Pulmonary Disease</span></div><div class="casAuthors">Millan, David S.; Bunnage, Mark E.; Burrows, Jane L.; Butcher, Kenneth J.; Dodd, Peter G.; Evans, Timothy J.; Fairman, David A.; Hughes, Samantha J.; Kilty, Iain C.; Lemaitre, Arnaud; Lewthwaite, Russell A.; Mahnke, Axel; Mathias, John P.; Philip, James; Smith, Robert T.; Stefaniak, Mark H.; Yeadon, Michael; Phillips, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7797-7814</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper describes the identification and optimization of a novel series of DFG-out binding p38 inhibitors as inhaled agents for the treatment of chronic obstructive pulmonary disease.  Structure based drug design and "inhalation by design" principles have been applied to the optimization of the lead series exemplified by compd. I.  Analogs have been designed to be potent and selective for p38, with an emphasis on slow enzyme dissocn. kinetics to deliver prolonged lung p38 inhibition.  Pharmacokinetic properties were tuned with high intrinsic clearance and low oral bioavailability in mind, to minimize systemic exposure and reduce systemically driven adverse events.  High CYP mediated clearance and glucuronidation were targeted to achieve high intrinsic clearance coupled with multiple routes of clearance to minimize drug-drug interactions.  Furthermore, pharmaceutical properties such as stability, crystallinity, and soly. were considered to ensure compatibility with a dry powder inhaler.  P38 Inhibitor II (PF-03715455) was subsequently identified as a clin. candidate from this series with efficacy and safety profiles confirming its potential as an inhaled agent for the treatment of COPD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzy5ivwDBL6rVg90H21EOLACvtfcHk0lgv5hFZnNTAZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlOqu73E&md5=9bc6cccf04a78831f50fb402bf7cb4b1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fjm200677b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200677b%26sid%3Dliteratum%253Aachs%26aulast%3DMillan%26aufirst%3DD.%2BS.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBurrows%26aufirst%3DJ.%2BL.%26aulast%3DButcher%26aufirst%3DK.%2BJ.%26aulast%3DDodd%26aufirst%3DP.%2BG.%26aulast%3DEvans%26aufirst%3DT.%2BJ.%26aulast%3DFairman%26aufirst%3DD.%2BA.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DKilty%26aufirst%3DI.%2BC.%26aulast%3DLemaitre%26aufirst%3DA.%26aulast%3DLewthwaite%26aufirst%3DR.%2BA.%26aulast%3DMahnke%26aufirst%3DA.%26aulast%3DMathias%26aufirst%3DJ.%2BP.%26aulast%3DPhilip%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DR.%2BT.%26aulast%3DStefaniak%26aufirst%3DM.%2BH.%26aulast%3DYeadon%26aufirst%3DM.%26aulast%3DPhillips%26aufirst%3DC.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520inhaled%2520p38%2520inhibitors%2520for%2520the%2520treatment%2520of%2520chronic%2520obstructive%2520pulmonary%2520disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7797%26epage%3D7814%26doi%3D10.1021%2Fjm200677b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Norman, P.</span></span> <span> </span><span class="NLM_article-title">Investigational p38 inhibitors for the treatment of chronic obstructive pulmonary disease</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">392</span>, <span class="refDoi"> DOI: 10.1517/13543784.2015.1006358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1517%2F13543784.2015.1006358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25599809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisVKrtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2015&pages=383-392&author=P.+Norman&title=Investigational+p38+inhibitors+for+the+treatment+of+chronic+obstructive+pulmonary+disease&doi=10.1517%2F13543784.2015.1006358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Investigational p38 inhibitors for the treatment of chronic obstructive pulmonary disease</span></div><div class="casAuthors">Norman, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">383-392</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Introduction: The p38 protein kinases, in particular p38α and p38β, regulate the prodn. of multiple inflammatory mediators.  Consequentially, considerable effort has been focused on trying to develop p38 inhibitors for the treatment of inflammatory diseases.  Some 20 p38 inhibitors have progressed to clin. development, mostly for the treatment of rheumatoid arthritis, but with little success.  Increasingly, interest has turned to their use in other indications and notably chronic obstructive pulmonary disease (COPD).  Areas covered: In this review, the author discusses the eight p38 inhibitors that have been clin. evaluated.  Acumapimod is the only one of four orally delivered inhibitors that remains in active development while Phase II results of PH-797804 and losmapimod are compared.  The activity of two inhibitors designed for inhaled delivery, RV-568 and PF-03715455, is compared but little is known about AZD-7624 or the discontinued GSK-610677.  Expert opinion: Results from animal models provide a clear rationale for developing p38 inhibitors for COPD, and appear to be (partially) validated by the efficacy seen with PH-797804 and losmapimod.  Inhaled delivery provides the opportunity to enhance p38 inhibition in the lung while reducing unwanted systemic effects of p38 inhibition.  Validation of this hypothesis should come from the results of the recently completed Phase II study with RV-568.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKUbGhXQAMzrVg90H21EOLACvtfcHk0lifhIwpxF-WxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisVKrtLk%253D&md5=97485678de1e2d15bf1ace5139cba51e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1517%2F13543784.2015.1006358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.2015.1006358%26sid%3Dliteratum%253Aachs%26aulast%3DNorman%26aufirst%3DP.%26atitle%3DInvestigational%2520p38%2520inhibitors%2520for%2520the%2520treatment%2520of%2520chronic%2520obstructive%2520pulmonary%2520disease%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2015%26volume%3D24%26spage%3D383%26epage%3D392%26doi%3D10.1517%2F13543784.2015.1006358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrick, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moult, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyck, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakser, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodak, S. J.</span></span> <span> </span><span class="NLM_article-title">CAPRI: A critical assessment of predicted interactions</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1002/prot.10381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1002%2Fprot.10381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=12784359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkslahsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=2-9&author=J.+Janinauthor=K.+Henrickauthor=J.+Moultauthor=L.+T.+Eyckauthor=M.+J.+Sternbergauthor=S.+Vajdaauthor=I.+Vakserauthor=S.+J.+Wodak&title=CAPRI%3A+A+critical+assessment+of+predicted+interactions&doi=10.1002%2Fprot.10381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">CAPRI: A critical assessment of predicted interactions</span></div><div class="casAuthors">Janin, Joel; Henrick, Kim; Moult, John; Ten Eyck, Lynn; Sternberg, Michael J. E.; Vajda, Sandor; Vakser, Ilya; Wodak, Shoshana J.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-9</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  CAPRI is a community wide expt. to assess the capacity of protein-docking methods to predict protein-protein interactions.  Nineteen groups participated in rounds 1 and 2 of CAPRI and submitted blind structure predictions for seven protein-protein complexes based on the known structure of the component proteins.  The predictions were compared to the unpublished X-ray structures of the complexes.  We describe here the motivations for launching CAPRI, the rules that we applied to select targets and run the expt., and some conclusions that can already be drawn.  The results stress the need for new scoring functions and for methods handling the conformation changes that were obsd. in some of the target systems.  CAPRI has already been a powerful drive for the community of computational biologists who development docking algorithms.  We hope that this issue of Proteins will also be of interest to the community of structural biologists, which we call upon to provide new targets for future rounds of CAPRI, and to all mol. biologists who view protein-protein recognition as an essential process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyYaxW6AQ0N7Vg90H21EOLACvtfcHk0lifhIwpxF-WxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkslahsr4%253D&md5=5ae9cf6222afcfebf272141679659289</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fprot.10381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10381%26sid%3Dliteratum%253Aachs%26aulast%3DJanin%26aufirst%3DJ.%26aulast%3DHenrick%26aufirst%3DK.%26aulast%3DMoult%26aufirst%3DJ.%26aulast%3DEyck%26aufirst%3DL.%2BT.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DVakser%26aufirst%3DI.%26aulast%3DWodak%26aufirst%3DS.%2BJ.%26atitle%3DCAPRI%253A%2520A%2520critical%2520assessment%2520of%2520predicted%2520interactions%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2003%26volume%3D52%26spage%3D2%26epage%3D9%26doi%3D10.1002%2Fprot.10381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishima, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freedberg, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torchia, D. A.</span></span> <span> </span><span class="NLM_article-title">Flap opening and dimer-interface flexibility in the free and inhibitor-bound HIV protease, and their implications for function</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1047</span>– <span class="NLM_lpage">1055</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(99)80172-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2FS0969-2126%2899%2980172-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10508781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADyaK1MXmtlartL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1999&pages=1047-1055&author=R.+Ishimaauthor=D.+I.+Freedbergauthor=Y.+X.+Wangauthor=J.+M.+Louisauthor=D.+A.+Torchia&title=Flap+opening+and+dimer-interface+flexibility+in+the+free+and+inhibitor-bound+HIV+protease%2C+and+their+implications+for+function&doi=10.1016%2FS0969-2126%2899%2980172-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Flap opening and dimer-interface flexibility in the free and inhibitor-bound HIV protease, and their implications for function</span></div><div class="casAuthors">Ishima, Rieko; Freedberg, Daron I.; Wang, Yun-Xing; Louis, John M.; Torchia, Dennis A.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1047-1055</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Publications</span>)
        </div><div class="casAbstract">Background: 1H and 15N transverse relaxation measurements on perdeuterated proteins are ideally suited for detecting backbone conformational fluctuations on the millisecond-microsecond timescale.  The identification of conformational exchange on this timescale by measuring the relaxation of both 1H and 15N holds great promise for the elucidation of functionally relevant conformational changes in proteins.  Results: We measured the transverse 1H and 15N relaxation rates of backbone amides of HIV-1 protease in its free and inhibitor-bound forms.  An anal. of these rates, obtained as a function of the effective rotating frame field, provided information about the timescale of structural fluctuations in several regions of the protein.  The flaps that cover the active site of the inhibitor-bound protein undergo significant changes of backbone (Φψ) angles, on the 100 μs timescale, in the free protein.  In addn., the intermonomer β-sheet interface of the bound form, which from protease structure studies appears to be rigid, was found to fluctuate on the millisecond timescale.  Conclusions: We present a working model of the flap-opening mechanism in free HIV-1 protease which involves a transition from a semi-open to an open conformation that is facilitated by interaction of the Phe53 ring with the substrate.  We also identify a surprising fluctuation of the β-sheet intermonomer interface that suggests a structural requirement for maturation of the protease.  Thus, slow conformational fluctuations identified by 1H and 15N transverse relaxation measurements can be related to the biol. functions of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFrYx5b8IAdrVg90H21EOLACvtfcHk0lifhIwpxF-WxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmtlartL8%253D&md5=1ab515e23efcd5eb310db489d0e78351</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2899%2980172-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252899%252980172-5%26sid%3Dliteratum%253Aachs%26aulast%3DIshima%26aufirst%3DR.%26aulast%3DFreedberg%26aufirst%3DD.%2BI.%26aulast%3DWang%26aufirst%3DY.%2BX.%26aulast%3DLouis%26aufirst%3DJ.%2BM.%26aulast%3DTorchia%26aufirst%3DD.%2BA.%26atitle%3DFlap%2520opening%2520and%2520dimer-interface%2520flexibility%2520in%2520the%2520free%2520and%2520inhibitor-bound%2520HIV%2520protease%252C%2520and%2520their%2520implications%2520for%2520function%26jtitle%3DStructure%26date%3D1999%26volume%3D7%26spage%3D1047%26epage%3D1055%26doi%3D10.1016%2FS0969-2126%2899%2980172-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seo, Y. H.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors to disrupt protein-protein interactions of heat shock protein 90 chaperone machinery</span>. <i>J. Cancer Prev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.15430/JCP.2015.20.1.5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.15430%2FJCP.2015.20.1.5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25853099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BC2Mjit1WqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=5-11&author=Y.+H.+Seo&title=Small+molecule+inhibitors+to+disrupt+protein-protein+interactions+of+heat+shock+protein+90+chaperone+machinery&doi=10.15430%2FJCP.2015.20.1.5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors to Disrupt Protein-protein Interactions of Heat Shock Protein 90 Chaperone Machinery</span></div><div class="casAuthors">Seo Young Ho</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cancer prevention</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-11</span>
        ISSN:<span class="NLM_cas:issn">2288-3649</span>.
    </div><div class="casAbstract">Heat shock protein 90 (Hsp90) is an adenosine triphosphate dependent molecular chaperone in eukaryotic cells that regulates the activation and maintenance of numerous regulatory and signaling proteins including epidermal growth factor receptor, human epidermal growth factor receptor 2, mesenchymal-epithelial transition factor, cyclin-dependent kinase-4, protein kinase B, hypoxia-inducible factor 1α, and matrix metalloproteinase-2.  Since many of Hsp90 clients are oncogenic proteins, Hsp90 has become an attractive therapeutic target for treatment of cancer.  To discover small molecule inhibitors targeting Hsp90 chaperone machinery, several strategies have been employed, which results in three classes of inhibitors such as N-terminal inhibitors, C-terminal inhibitors, and inhibitors disrupting protein-protein interactions of Hsp90 chaperone machinery.  Developing small molecule inhibitors that modulate protein-protein interactions of Hsp90 is a challenging task, although it offers many alternative opportunities for therapeutic intervention.  The lack of well-defined binding pocket and starting points for drug design challenges medicinal chemists to discover small molecule inhibitors disrupting protein-protein interactions of Hsp90.  The present review will focus on the current studies on small molecule inhibitors disrupting protein-protein interactions of Hsp90 chaperone machinery, provide biological background on the structure, function and mechanism of Hsp90's protein-protein interactions, and discuss the challenges and promise of its small molecule modulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZvHVQAn27dy86Js-BCLiifW6udTcc2ebJcyuIpjn4Tbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mjit1WqsA%253D%253D&md5=6df1b2f3b2fee148861243d8ff6c8fef</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.15430%2FJCP.2015.20.1.5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15430%252FJCP.2015.20.1.5%26sid%3Dliteratum%253Aachs%26aulast%3DSeo%26aufirst%3DY.%2BH.%26atitle%3DSmall%2520molecule%2520inhibitors%2520to%2520disrupt%2520protein-protein%2520interactions%2520of%2520heat%2520shock%2520protein%252090%2520chaperone%2520machinery%26jtitle%3DJ.%2520Cancer%2520Prev.%26date%3D2015%26volume%3D20%26spage%3D5%26epage%3D11%26doi%3D10.15430%2FJCP.2015.20.1.5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stebbins, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartl, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavletich, N. P.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of an Hsp90-geldanamycin complex: Targeting of a protein chaperone by an antitumor agent</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">239</span>– <span class="NLM_lpage">250</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)80203-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2FS0092-8674%2800%2980203-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=9108479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADyaK2sXislCitLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1997&pages=239-250&author=C.+E.+Stebbinsauthor=A.+A.+Russoauthor=C.+Schneiderauthor=N.+Rosenauthor=F.+U.+Hartlauthor=N.+P.+Pavletich&title=Crystal+structure+of+an+Hsp90-geldanamycin+complex%3A+Targeting+of+a+protein+chaperone+by+an+antitumor+agent&doi=10.1016%2FS0092-8674%2800%2980203-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of an Hsp90-geldanamycin complex: targeting of a protein chaperone by an antitumor agent</span></div><div class="casAuthors">Stebbins, Charles E.; Russo, Alicia A.; Schneider, Christine; Rosen, Neal; Hartl, F. Ulrich; Pavletich, Nikola P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-250</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Hsp90 chaperone is required for the activation of several families of eukaryotic protein kinases and nuclear hormone receptors, many of which are proto-oncogenic and play a prominent role in cancer.  The geldanamycin antibiotic has antiproliferative and anti-tumor effects, as it binds to Hsp90, inhibits the Hsp90-mediated conformational maturation/refolding reaction, and results in the degrdn. of Hsp90 substrates.  The structure of the geldanamycin-binding domain of Hsp90 (residues 9-232) reveals a pronounced pocket, 15 Å deep. that is highly conserved across species.  Geldanamycin binds inside this pocket, adopting a compact structure similar to that of a polypeptide chain in a turn conformation.  This, and the pocket's similarity to substrate-binding sites, suggest that the pocket binds a portion of the polypeptide substrate and participates in the conformational maturation/refolding reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvnmleAOFITbVg90H21EOLACvtfcHk0lhH25v48NXCbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXislCitLo%253D&md5=54a56450c509727e070f4db20a9d7b1a</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2980203-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252980203-2%26sid%3Dliteratum%253Aachs%26aulast%3DStebbins%26aufirst%3DC.%2BE.%26aulast%3DRusso%26aufirst%3DA.%2BA.%26aulast%3DSchneider%26aufirst%3DC.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DHartl%26aufirst%3DF.%2BU.%26aulast%3DPavletich%26aufirst%3DN.%2BP.%26atitle%3DCrystal%2520structure%2520of%2520an%2520Hsp90-geldanamycin%2520complex%253A%2520Targeting%2520of%2520a%2520protein%2520chaperone%2520by%2520an%2520antitumor%2520agent%26jtitle%3DCell%26date%3D1997%26volume%3D89%26spage%3D239%26epage%3D250%26doi%3D10.1016%2FS0092-8674%2800%2980203-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trendowski, M.</span></span> <span> </span><span class="NLM_article-title">Pu-h71: An improvement on nature’s solutions to oncogenic Hsp90 addiction</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">216</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.phrs.2015.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=26117427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFSjt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=202-216&author=M.+Trendowski&title=Pu-h71%3A+An+improvement+on+nature%E2%80%99s+solutions+to+oncogenic+Hsp90+addiction&doi=10.1016%2Fj.phrs.2015.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">PU-H71: An improvement on nature's solutions to oncogenic Hsp90 addiction</span></div><div class="casAuthors">Trendowski, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">202-216</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Despite recent advances in precision medicine, many mol.-based antineoplastic agents do not potentiate sustainable long term remissions, warranting the investigation of novel therapeutic strategies.  Heat shock protein 90 (Hsp90) is a mol. chaperone that not only has oncogenic properties, but also has distinct expression profiles in malignant and normal cells, providing a rational strategy to attain preferential damage.  Prior attempts to target Hsp90 with natural product-based compds. have been hampered by their assocd. off target toxicities, suggesting that novel, fully synthetic inhibitors may be required to achieve the specificity necessary for therapeutic efficacy.  Therefore, this review highlights the antineoplastic potential of PU-H71 (8-[(6-iodo-1,3-benzodioxol-5-yl)sulfanyl]-9-[3-(propan-2-ylamino)propyl]purin-6-amine), a novel purine based analog that has shown efficacy in many preclin. models of malignancy, and is now under clin. examn.  In addn., the review suggests potential concomitant therapeutic approaches that may be particularly beneficial to mol.-based, as well as traditional cytotoxic cancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ-CFQcXC_6bVg90H21EOLACvtfcHk0lhjS4zkZ0Jtvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFSjt7fK&md5=bc9029f7ca82f2fff0fed0ee772bcf10</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DTrendowski%26aufirst%3DM.%26atitle%3DPu-h71%253A%2520An%2520improvement%2520on%2520nature%25E2%2580%2599s%2520solutions%2520to%2520oncogenic%2520Hsp90%2520addiction%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D202%26epage%3D216%26doi%3D10.1016%2Fj.phrs.2015.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lhjS4zkZ0Jtvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasater, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+Mullerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+kinase-bromodomain+inhibitors+for+rationally+designed+polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0lhjS4zkZ0Jtvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520for%2520rationally%2520designed%2520polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of bivalent BET inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1096</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnchembio.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=27775715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1089-1096&author=M.+Tanakaauthor=J.+M.+Robertsauthor=H.+S.+Seoauthor=A.+Souzaauthor=J.+Paulkauthor=T.+G.+Scottauthor=S.+L.+DeAngeloauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Design+and+characterization+of+bivalent+BET+inhibitors&doi=10.1038%2Fnchembio.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Design and characterization of bivalent BET inhibitors</span></div><div class="casAuthors">Tanaka, Minoru; Roberts, Justin M.; Seo, Hyuk-Soo; Souza, Amanda; Paulk, Joshiawa; Scott, Thomas G.; DeAngelo, Stephen L.; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1089-1096</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cellular signaling is often propagated by multivalent interactions.  Multivalency creates avidity, allowing stable biophys. recognition.  Multivalency is an attractive strategy for achieving potent binding to protein targets, as the affinity of bivalent ligands is often greater than the sum of monovalent affinities.  The bromodomain and extraterminal domain (BET) family of transcriptional coactivators features tandem bromodomains through which BET proteins bind acetylated histones and transcription factors.  All reported antagonists of the BET protein BRD4 bind in a monovalent fashion.  Here the authors describe, to the knowledge for the first time, a bivalent BET bromodomain inhibitor-MT1-which has unprecedented potency.  Biophys. and biochem. studies suggest MT1 is an intramol. bivalent BRD4 binder that is more than 100-fold more potent, in cellular assays, than the corresponding monovalent antagonist, JQ1.  MT1 significantly (P < 0.05) delayed leukemia progression in mice, as compared to JQ1.  These data qualify a powerful chem. probe for BET bromodomains and a rationale for further development of multidomain inhibitors of epigenetic reader proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQwc6X0VYYbVg90H21EOLACvtfcHk0lgi1CW50JsrCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurrM&md5=7319203dac409d5dee3c580b49fcf185</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2209%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSeo%26aufirst%3DH.%2BS.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDesign%2520and%2520characterization%2520of%2520bivalent%2520BET%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D1089%26epage%3D1096%26doi%3D10.1038%2Fnchembio.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frkic, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blayo, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pukala, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamenecka, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruning, J. B.</span></span> <span> </span><span class="NLM_article-title">PPARγ in complex with an antagonist and inverse agonist: A tumble and trap mechanism of the activation helix</span>. <i>iScience</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1016/j.isci.2018.06.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.isci.2018.06.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=30123887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVajt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2018&pages=69-79&author=R.+L.+Frkicauthor=A.+C.+Marshallauthor=A.+L.+Blayoauthor=T.+L.+Pukalaauthor=T.+M.+Kameneckaauthor=P.+R.+Griffinauthor=J.+B.+Bruning&title=PPAR%CE%B3+in+complex+with+an+antagonist+and+inverse+agonist%3A+A+tumble+and+trap+mechanism+of+the+activation+helix&doi=10.1016%2Fj.isci.2018.06.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PPARγ in Complex with an Antagonist and Inverse Agonist: a Tumble and Trap Mechanism of the Activation Helix</span></div><div class="casAuthors">Frkic, Rebecca L.; Marshall, Andrew C.; Blayo, Anne-Laure; Pukala, Tara L.; Kamenecka, Theodore M.; Griffin, Patrick R.; Bruning, John B.</div><div class="citationInfo"><span class="NLM_cas:title">iScience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">69-79</span>CODEN:
                <span class="NLM_cas:coden">ISCICE</span>;
        ISSN:<span class="NLM_cas:issn">2589-0042</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Peroxisome proliferator activated receptor γ (PPARγ) is a nuclear receptor and target for antidiabetics that increase insulin sensitivity.  Owing to the side effects of PPARγ full agonists, research has recently focused on non-activating ligands of PPARγ, which increase insulin sensitivity with decreased side effects.  Here, we present the crystal structures of inverse agonist SR10171 and a chem. related antagonist SR11023 bound to the PPARγ ligand-binding domain, revealing an allosteric switch in the activation helix, helix 12 (H12), forming an antagonist conformation in the receptor.  H12 interacts with the antagonists to become fixed in an alternative location.  Native mass spectrometry indicates that this prevents contacts with coactivator peptides and allows binding of corepressor peptides.  Antagonists of related nuclear receptors act to sterically prevent the active configuration of H12, whereas these antagonists of PPARγ alternatively trap H12 in an inactive configuration, which we have termed the tumble and trap mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ-nTSIgus9bVg90H21EOLACvtfcHk0ljYGKoihMwQmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVajt70%253D&md5=69568acac5f75ae39807ab8460b31f5a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.isci.2018.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.isci.2018.06.012%26sid%3Dliteratum%253Aachs%26aulast%3DFrkic%26aufirst%3DR.%2BL.%26aulast%3DMarshall%26aufirst%3DA.%2BC.%26aulast%3DBlayo%26aufirst%3DA.%2BL.%26aulast%3DPukala%26aufirst%3DT.%2BL.%26aulast%3DKamenecka%26aufirst%3DT.%2BM.%26aulast%3DGriffin%26aufirst%3DP.%2BR.%26aulast%3DBruning%26aufirst%3DJ.%2BB.%26atitle%3DPPAR%25CE%25B3%2520in%2520complex%2520with%2520an%2520antagonist%2520and%2520inverse%2520agonist%253A%2520A%2520tumble%2520and%2520trap%2520mechanism%2520of%2520the%2520activation%2520helix%26jtitle%3DiScience%26date%3D2018%26volume%3D5%26spage%3D69%26epage%3D79%26doi%3D10.1016%2Fj.isci.2018.06.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taygerly, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGee, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubenstein, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houze, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cushing, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankmoelle, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Learned, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timmermans, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoolen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kearney, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiszmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biermann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aramaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinkai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furukawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of INT131: A selective PPARγ modulator that enhances insulin sensitivity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">979</span>– <span class="NLM_lpage">992</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2012.11.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.bmc.2012.11.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23294830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkt1Witg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=979-992&author=J.+P.+Taygerlyauthor=L.+R.+McGeeauthor=S.+M.+Rubensteinauthor=J.+B.+Houzeauthor=T.+D.+Cushingauthor=Y.+Liauthor=A.+Motaniauthor=J.+L.+Chenauthor=W.+Frankmoelleauthor=G.+Yeauthor=M.+R.+Learnedauthor=J.+Jaenauthor=S.+Miaoauthor=P.+B.+Timmermansauthor=M.+Thoolenauthor=P.+Kearneyauthor=J.+Flygareauthor=H.+Beckmannauthor=J.+Weiszmannauthor=M.+Lindstromauthor=N.+Walkerauthor=J.+Liuauthor=D.+Biermannauthor=Z.+Wangauthor=A.+Hagiwaraauthor=T.+Iidaauthor=H.+Aramakiauthor=Y.+Kitaoauthor=H.+Shinkaiauthor=N.+Furukawaauthor=J.+Nishiuauthor=M.+Nakamura&title=Discovery+of+INT131%3A+A+selective+PPAR%CE%B3+modulator+that+enhances+insulin+sensitivity&doi=10.1016%2Fj.bmc.2012.11.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of INT131: A selective PPARγ modulator that enhances insulin sensitivity</span></div><div class="casAuthors">Taygerly, Joshua P.; McGee, Lawrence R.; Rubenstein, Steven M.; Houze, Jonathan B.; Cushing, Timothy D.; Li, Yang; Motani, Alykhan; Chen, Jin-Long; Frankmoelle, Walter; Ye, Guosen; Learned, Marc R.; Jaen, Juan; Miao, Shichang; Timmermans, Pieter B.; Thoolen, Martin; Kearney, Patrick; Flygare, John; Beckmann, Holger; Weiszmann, Jennifer; Lindstrom, Michelle; Walker, Nigel; Liu, Jinsong; Biermann, Donna; Wang, Zhulun; Hagiwara, Atsushi; Iida, Tetsuya; Aramaki, Hisateru; Kitao, Yuki; Shinkai, Hisashi; Furukawa, Noboru; Nishiu, Jun; Nakamura, Motonao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">979-992</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">PPARγ is a member of the nuclear hormone receptor family and plays a key role in the regulation of glucose homeostasis.  This study describes the discovery of a novel chem. class of diarylsulfonamide partial agonists that act as selective PPARγ modulators (SPPARγMs) and display a unique pharmacol. profile compared to the thiazolidinedione (TZD) class of PPARγ full agonists.  Herein we report the initial discovery of partial agonist (I) and the structure-activity relation studies that led to the selection of clin. compd. INT131 (II), a potent PPARγ partial agonist that displays robust glucose-lowering activity in rodent models of diabetes while exhibiting a reduced side-effects profile compared to marketed TZDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMnHgzkPStAbVg90H21EOLACvtfcHk0ljYGKoihMwQmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkt1Witg%253D%253D&md5=7056be6ef3ae46eb8a3076dab838da20</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.11.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.11.058%26sid%3Dliteratum%253Aachs%26aulast%3DTaygerly%26aufirst%3DJ.%2BP.%26aulast%3DMcGee%26aufirst%3DL.%2BR.%26aulast%3DRubenstein%26aufirst%3DS.%2BM.%26aulast%3DHouze%26aufirst%3DJ.%2BB.%26aulast%3DCushing%26aufirst%3DT.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMotani%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DFrankmoelle%26aufirst%3DW.%26aulast%3DYe%26aufirst%3DG.%26aulast%3DLearned%26aufirst%3DM.%2BR.%26aulast%3DJaen%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DS.%26aulast%3DTimmermans%26aufirst%3DP.%2BB.%26aulast%3DThoolen%26aufirst%3DM.%26aulast%3DKearney%26aufirst%3DP.%26aulast%3DFlygare%26aufirst%3DJ.%26aulast%3DBeckmann%26aufirst%3DH.%26aulast%3DWeiszmann%26aufirst%3DJ.%26aulast%3DLindstrom%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DBiermann%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DHagiwara%26aufirst%3DA.%26aulast%3DIida%26aufirst%3DT.%26aulast%3DAramaki%26aufirst%3DH.%26aulast%3DKitao%26aufirst%3DY.%26aulast%3DShinkai%26aufirst%3DH.%26aulast%3DFurukawa%26aufirst%3DN.%26aulast%3DNishiu%26aufirst%3DJ.%26aulast%3DNakamura%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520INT131%253A%2520A%2520selective%2520PPAR%25CE%25B3%2520modulator%2520that%2520enhances%2520insulin%2520sensitivity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D979%26epage%3D992%26doi%3D10.1016%2Fj.bmc.2012.11.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Badrinarayan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. N.</span></span> <span> </span><span class="NLM_article-title">Rational approaches towards lead optimization of kinase inhibitors: The issue of specificity</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4714</span>– <span class="NLM_lpage">4738</span>, <span class="refDoi"> DOI: 10.2174/1381612811319260005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.2174%2F1381612811319260005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23260022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wns7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=4714-4738&author=P.+Badrinarayanauthor=G.+N.+Sastry&title=Rational+approaches+towards+lead+optimization+of+kinase+inhibitors%3A+The+issue+of+specificity&doi=10.2174%2F1381612811319260005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Approaches Towards Lead Optimization of Kinase Inhibitors: The Issue of Specificity</span></div><div class="casAuthors">Badrinarayan, Preethi; Sastry, G. Narahari</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">4714-4738</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases are one of the most popular classes of drug targets as they are involved in signal transduction pathways, which are wired through a phosphotransfer cascade and elicit a no. of important and essential physiol. responses.  Kinase specificity has emerged as one of the major issues to be addressed in drug discovery approaches.  In most kinases the active site is the ATP binding site and finding suitable hits which maximize the affinity of binding has been traditionally important to obtain the type I inhibitors.  While type I inhibitors have effective binding affinity more often than not they encounter side-effects usually assocd. with specificity.  Therefore in recent times it has become indispensable to optimize specificity for developing effective kinase inhibitors.  The review presents an overview of kinase drug discovery and the different strategies used to date for the design of kinase leads accounting for their success and failure.  A no. of strategies exploiting different aspects of kinases like allosteric site, size of the gatekeeper residue, DFG-loop, chemotype selectivity, non-covalent interactions, salt-bridge, solvation, etc. have been explored to circumvent the specificity problem in kinases.  The probable hot-spots in kinases having a propensity to bring in specificity have been delineated with special emphasis on the design of type II inhibitors with increased specificity from existing type I using fragment tailoring approach.  In this review we illustrate the current strategies by taking p38 MAP kinase as a model and expect that such strategies are general and can be extended to the other members of the kinase family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmsC1PsUKeGLVg90H21EOLACvtfcHk0ljYGKoihMwQmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wns7rM&md5=ff2b5c5f0fd16af8c72763db036fd3a3</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.2174%2F1381612811319260005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319260005%26sid%3Dliteratum%253Aachs%26aulast%3DBadrinarayan%26aufirst%3DP.%26aulast%3DSastry%26aufirst%3DG.%2BN.%26atitle%3DRational%2520approaches%2520towards%2520lead%2520optimization%2520of%2520kinase%2520inhibitors%253A%2520The%2520issue%2520of%2520specificity%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D4714%26epage%3D4738%26doi%3D10.2174%2F1381612811319260005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochhaus, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derendorf, H.</span></span> <span> </span><span class="NLM_article-title">Comparative pharmacology, bioavailability, pharmacokinetics, and pharmacodynamics of inhaled glucocorticosteroids</span>. <i>Immunol. Allergy Clin. North Am.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1016/j.iac.2005.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.iac.2005.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=16054538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BD2MzpsFymsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=469-488&author=M.+Hubnerauthor=G.+Hochhausauthor=H.+Derendorf&title=Comparative+pharmacology%2C+bioavailability%2C+pharmacokinetics%2C+and+pharmacodynamics+of+inhaled+glucocorticosteroids&doi=10.1016%2Fj.iac.2005.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative pharmacology, bioavailability, pharmacokinetics, and pharmacodynamics of inhaled glucocorticosteroids</span></div><div class="casAuthors">Hubner Melanie; Hochhaus Gunther; Derendorf Hartmut</div><div class="citationInfo"><span class="NLM_cas:title">Immunology and allergy clinics of North America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">469-88</span>
        ISSN:<span class="NLM_cas:issn">0889-8561</span>.
    </div><div class="casAbstract">A comparison of the pharmacodynamics and pharmacokinetics of inhaled corticosteroids is necessary for their assessment.  A good knowledge of these two aspects allows the optimization of efficacy and safety.The currently available inhaled corticosteroids already show some of the desired PK/PD parameters.  The local adverse effects are decreased as soon as the inhaled corticosteroid is administered as an inactive prodrug or shows a bet-ter lung deposition.  HFA-MDI beclomethasone dipropionate (BDP) and ciclesonide are two agents that illustrate this.  Low oral bioavailability, rapid systemic clearance, and high plasma protein binding can minimize systemic adverse effects.  Mometasone furoate, ciclesonide, and fluticasone propionate possess those characteristics.  The pulmonary efficacy is maximized by high lung deposition and long pulmonary residence times.  This effect can be achieved by slow dissolution in the lungs, as is the case for fluticasone propionate or lipid conjugation and has been shown for budesonide and ciclesonide.  Furthermore, the lung deposition depends on the inhalation device, the particle size, and the inhalation technique.  Therefore,improvement in the design of MDIs, DPIs, and nebulizers, and the development of more effective drug particles will lead to an optimized pulmonary targeting.  Much progress has been made in the treatment of asthma.  The available inhaled corticosteroids show a high safety profile and a good pulmonary selectivity.  Development of newer compounds showed that improvement is possible as the result of a complete understanding of the PK/PD concepts.  However,the introduction of further improved formulations with a better efficacy/safety profile will be difficult and protracted because the existing drugs are already highly efficient.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSix-AbkCPCpuqrn-c4Of3KfW6udTcc2eYwxmKU2z1Z7Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MzpsFymsA%253D%253D&md5=867e4f0660f96aba8b7724afb594a39b</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.iac.2005.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.iac.2005.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DHubner%26aufirst%3DM.%26aulast%3DHochhaus%26aufirst%3DG.%26aulast%3DDerendorf%26aufirst%3DH.%26atitle%3DComparative%2520pharmacology%252C%2520bioavailability%252C%2520pharmacokinetics%252C%2520and%2520pharmacodynamics%2520of%2520inhaled%2520glucocorticosteroids%26jtitle%3DImmunol.%2520Allergy%2520Clin.%2520North%2520Am.%26date%3D2005%26volume%3D25%26spage%3D469%26epage%3D488%26doi%3D10.1016%2Fj.iac.2005.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, S. C.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of human calcineurin complexed with cyclosporin A and human cyclophilin</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">13522</span>– <span class="NLM_lpage">13526</span>, <span class="refDoi"> DOI: 10.1073/pnas.212504399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1073%2Fpnas.212504399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=12357034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVKms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=13522-13526&author=L.+Jinauthor=S.+C.+Harrison&title=Crystal+structure+of+human+calcineurin+complexed+with+cyclosporin+A+and+human+cyclophilin&doi=10.1073%2Fpnas.212504399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of human calcineurin complexed with cyclosporin A and human cyclophilin</span></div><div class="casAuthors">Jin, Lei; Harrison, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">13522-13526</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Calcineurin (Cn; protein phosphatase 2B), is an important participant in signaling pathways that activate T cells.  It is the target of the immunosuppressive drugs, cyclosporin A (CsA) and FK 506.  These drugs bind proteins known as cyclophilin (Cyp) and FK 506-binding protein, resp., and the drug-protein complexes in turn inhibit Cn.  Here, the authors report the crystal structure of a Cyp/CsA/Cn ternary complex, detd. to a resoln. of 3.1 Å.  Residues 3-9 of CsA, particularly N-methylleucine (Mle)-4 and Mle-6, and Trp-121 of Cyp were found to form a composite surface for interaction with Cn.  The hydrophobic interface buried 2 H-bonds.  The structure accounted clearly for the effects of mutations in Cn on CsA-resistance and for the way modifications of CsA alter immunosuppressive activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqspopSI-iZLLVg90H21EOLACvtfcHk0likzlYp5vSdEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVKms7c%253D&md5=cff5598bce80f7a372582156f9e8619b</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1073%2Fpnas.212504399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.212504399%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHarrison%26aufirst%3DS.%2BC.%26atitle%3DCrystal%2520structure%2520of%2520human%2520calcineurin%2520complexed%2520with%2520cyclosporin%2520A%2520and%2520human%2520cyclophilin%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D13522%26epage%3D13526%26doi%3D10.1073%2Fpnas.212504399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, J. D.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Calcineurin is a common target of cyclophilin-cyclosporin A and FKBP-FK506 complexes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(91)90124-H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2F0092-8674%2891%2990124-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1715244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADyaK3MXlsV2qu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=1991&pages=807-815&author=J.+Liuauthor=J.+D.+Farmerauthor=W.+S.+Laneauthor=J.+Friedmanauthor=I.+Weissmanauthor=S.+L.+Schreiber&title=Calcineurin+is+a+common+target+of+cyclophilin-cyclosporin+A+and+FKBP-FK506+complexes&doi=10.1016%2F0092-8674%2891%2990124-H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Calcineurin is a common target of cyclophilin-cyclosporin A and FKBP-FK506 complexes</span></div><div class="casAuthors">Liu, Jun; Farmer, Jesse D., Jr.; Lane, Willam S.; Friedman, Jeff; Weissman, Irving; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">807-15</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Although the immediate receptors (immunophilins) of the immunosuppressants cyclosporin A (CsA) and FK506 are distinct, their similar mechanisms of inhibition of cell signaling suggest that their assocd. immunophilin complexes interact with a common target.  The authors report that the complexes cyclophilin-CsA and FKBP-FK506 (but not cyclophilin, FKBP, FKBP-rapamycin, or FKBP-506BD) competitively bind to and inhibit the Ca2+- and calmodulin-dependent phosphatase calcineurin, although the binding and inhibition of calcineurin do not require calmodulin.  These results suggest that calcineurin is involved in a common step assocd. with T cell receptor and IgE receptor signaling pathways and that cyclophilin and FKBP mediate the actions of CsA and FK506, resp., by forming drug-dependent complexes with and altering the activity of calcineurin-calmodulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxfN8mg-hNhLVg90H21EOLACvtfcHk0ljibAcJrU2OgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlsV2qu7w%253D&md5=00c4883ea86e6b2c554f8f72ed717ac9</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990124-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990124-H%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFarmer%26aufirst%3DJ.%2BD.%26aulast%3DLane%26aufirst%3DW.%2BS.%26aulast%3DFriedman%26aufirst%3DJ.%26aulast%3DWeissman%26aufirst%3DI.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DCalcineurin%2520is%2520a%2520common%2520target%2520of%2520cyclophilin-cyclosporin%2520A%2520and%2520FKBP-FK506%2520complexes%26jtitle%3DCell%26date%3D1991%26volume%3D66%26spage%3D807%26epage%3D815%26doi%3D10.1016%2F0092-8674%2891%2990124-H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daigle, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therkelsen, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basavapathruni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span> <span> </span><span class="NLM_article-title">Potent inhibition of DOT1L as treatment of MLL-fusion leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">1017</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1182/blood-2013-04-497644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1182%2Fblood-2013-04-497644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23801631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlWltLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2013&pages=1017-1025&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=A.+Basavapathruniauthor=L.+Jinauthor=P.+A.+Boriack-Sjodinauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Potent+inhibition+of+DOT1L+as+treatment+of+MLL-fusion+leukemia&doi=10.1182%2Fblood-2013-04-497644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Potent inhibition of DOT1L as treatment of MLL-fusion leukemia</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Basavapathruni, Aravind; Jin, Lei; Boriack-Sjodin, P. Ann; Allain, Christina J.; Klaus, Christine R.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1017-1025</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Rearrangements of the MLL gene define a genetically distinct subset of acute leukemias with poor prognosis.  Current treatment options are of limited effectiveness; thus, there is a pressing need for new therapies for this disease.  Genetic and small mol. inhibitor studies have demonstrated that the histone methyltransferase DOT1L is required for the development and maintenance of MLL-rearranged leukemia in model systems.  Here we describe the characterization of EPZ-5676, a potent and selective aminonucleoside inhibitor of DOT1L histone methyltransferase activity.  The compd. has an inhibition const. value of 80 pM, and demonstrates 37000-fold selectivity over all other methyltransferases tested.  In cellular studies, EPZ-5676 inhibited H3K79 methylation and MLL-fusion target gene expression and demonstrated potent cell killing that was selective for acute leukemia lines bearing MLL translocations.  Continuous IV infusion of EPZ-5676 in a rat xenograft model of MLL-rearranged leukemia caused complete tumor regressions that were sustained well beyond the compd. infusion period with no significant wt. loss or signs of toxicity.  EPZ-5676 is therefore a potential treatment of MLL-rearranged leukemia and is under clin. investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT-fqlDzoK07Vg90H21EOLACvtfcHk0ljibAcJrU2OgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlWltLfE&md5=bad23c37a5fe886668e25d10b7d9b43a</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-04-497644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-04-497644%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DBasavapathruni%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DPotent%2520inhibition%2520of%2520DOT1L%2520as%2520treatment%2520of%2520MLL-fusion%2520leukemia%26jtitle%3DBlood%26date%3D2013%26volume%3D122%26spage%3D1017%26epage%3D1025%26doi%3D10.1182%2Fblood-2013-04-497644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monk, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomasiak, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keniya, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huschmann, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyndall, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, J. D.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finer-Moore, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, R. M.</span></span> <span> </span><span class="NLM_article-title">Architecture of a single membrane spanning cytochrome P450 suggests constraints that orient the catalytic domain relative to a bilayer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">3865</span>– <span class="NLM_lpage">3870</span>, <span class="refDoi"> DOI: 10.1073/pnas.1324245111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1073%2Fpnas.1324245111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=24613931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlyht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=3865-3870&author=B.+C.+Monkauthor=T.+M.+Tomasiakauthor=M.+V.+Keniyaauthor=F.+U.+Huschmannauthor=J.+D.+Tyndallauthor=J.+D.+O%E2%80%99Connellauthor=R.+D.+Cannonauthor=J.+G.+McDonaldauthor=A.+Rodriguezauthor=J.+S.+Finer-Mooreauthor=R.+M.+Stroud&title=Architecture+of+a+single+membrane+spanning+cytochrome+P450+suggests+constraints+that+orient+the+catalytic+domain+relative+to+a+bilayer&doi=10.1073%2Fpnas.1324245111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Architecture of a single membrane spanning cytochrome P 450 suggests constraints that orient the catalytic domain relative to a bilayer</span></div><div class="casAuthors">Monk, Brian C.; Tomasiak, Thomas M.; Keniya, Mikhail V.; Huschmann, Franziska U.; Tyndall, Joel D. A.; O'Connell, Joseph D., III; Cannon, Richard D.; McDonald, Jeffrey G.; Rodriguez, Andrew; Finer-Moore, Janet S.; Stroud, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3865-3870</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bitopic integral membrane proteins with a single transmembrane helix play diverse roles in catalysis, cell signaling, and morphogenesis.  Complete monospanning protein structures are needed to show how interaction between the transmembrane helix and catalytic domain might influence assocn. with the membrane and function.  Here, the authors report the crystal structures of full-length Saccharomyces cerevisiae lanosterol 14α-demethylase, a membrane-monospanning cytochrome P 450 of the CYP51 family that catalyzes the 1st post-cyclization step in ergosterol biosynthesis and is inhibited by triazole drugs.  The structures revealed a well-ordered N-terminal amphipathic helix preceding a putative transmembrane helix that would constrain the catalytic domain orientation to lie partly in the lipid bilayer.  The structures located the substrate, lanosterol, identified putative substrate and product channels, and revealed constrained interactions with triazole antifungal drugs that are important for drug design and understanding drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWnbj5vZw9lLVg90H21EOLACvtfcHk0ljibAcJrU2OgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlyht7k%253D&md5=e3e0d0f9577509cc8fc21a8b32f0bb55</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1324245111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1324245111%26sid%3Dliteratum%253Aachs%26aulast%3DMonk%26aufirst%3DB.%2BC.%26aulast%3DTomasiak%26aufirst%3DT.%2BM.%26aulast%3DKeniya%26aufirst%3DM.%2BV.%26aulast%3DHuschmann%26aufirst%3DF.%2BU.%26aulast%3DTyndall%26aufirst%3DJ.%2BD.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DJ.%2BD.%26aulast%3DCannon%26aufirst%3DR.%2BD.%26aulast%3DMcDonald%26aufirst%3DJ.%2BG.%26aulast%3DRodriguez%26aufirst%3DA.%26aulast%3DFiner-Moore%26aufirst%3DJ.%2BS.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26atitle%3DArchitecture%2520of%2520a%2520single%2520membrane%2520spanning%2520cytochrome%2520P450%2520suggests%2520constraints%2520that%2520orient%2520the%2520catalytic%2520domain%2520relative%2520to%2520a%2520bilayer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D3865%26epage%3D3870%26doi%3D10.1073%2Fpnas.1324245111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winston, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maziarz, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekar, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarus, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumer, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitz, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Territo, M. C.</span></span> <span> </span><span class="NLM_article-title">Intravenous and oral itraconazole versus intravenous and oral fluconazole for long-term antifungal prophylaxis in allogeneic hematopoietic stem-cell transplant recipients. A multicenter, randomized trial</span>. <i>Ann. Intern. Med.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">705</span>– <span class="NLM_lpage">713</span>, <span class="refDoi"> DOI: 10.7326/0003-4819-138-9-200305060-00006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.7326%2F0003-4819-138-9-200305060-00006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=12729424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktVyhsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2003&pages=705-713&author=D.+J.+Winstonauthor=R.+T.+Maziarzauthor=P.+H.+Chandrasekarauthor=H.+M.+Lazarusauthor=M.+Goldmanauthor=J.+L.+Blumerauthor=G.+J.+Leitzauthor=M.+C.+Territo&title=Intravenous+and+oral+itraconazole+versus+intravenous+and+oral+fluconazole+for+long-term+antifungal+prophylaxis+in+allogeneic+hematopoietic+stem-cell+transplant+recipients.+A+multicenter%2C+randomized+trial&doi=10.7326%2F0003-4819-138-9-200305060-00006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Intravenous and oral itraconazole versus intravenous and oral fluconazole for long-term antifungal prophylaxis in allogeneic hematopoietic stem-cell transplant recipients: a multicenter, randomized trial</span></div><div class="casAuthors">Winston, Drew J.; Maziarz, Richard T.; Chandrasekar, Pranatharthi H.; Lazarus, Hillard M.; Goldman, Mitchell; Blumer, Jeffrey L.; Leitz, Gerhard J.; Territo, Mary C.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Internal Medicine</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">705-713</span>CODEN:
                <span class="NLM_cas:coden">AIMEAS</span>;
        ISSN:<span class="NLM_cas:issn">0003-4819</span>.
    
            (<span class="NLM_cas:orgname">American College of Physicians-American Society of Internal Medicine</span>)
        </div><div class="casAbstract">Background: Allogeneic hematopoietic stem-cell transplant recipients often receive fluconazole or an amphotericin B prepn. for antifungal prophylaxis.  Because of concerns about fungal resistance with fluconazole and toxicity with amphotericin B, alternative prophylactic regimens have become necessary.  Objective: To compare the efficacy and safety of i.v. and oral itraconazole with the efficacy and safety of i.v. and oral fluconazole for long-term prophylaxis of fungal infections.  Design: open-label, multicenter, randomized trial.  Setting: Five transplantation centers in the United States.  Patients: 140 patients undergoing allogeneic hematopoietic stem-cell transplantation.  Intervention: Itraconazole (200 mg i.v. every 12 h for 2 days followed by 200 mg i.v. every 24 h or a 200-mg oral soln. every 12 h) or fluconazole (400 mg i.v. or orally every 24 h) from day 1 until day 100 after transplantation.  Measurements: Proven invasive or superficial fungal infection, drug-related side effects, mortality from fungal infection, and overall mortality.  Results: Proven invasive fungal infections occurred in 6 of 71 itraconazole recipients (9%) and in 17 of 67 fluconazole recipients (25%) during the first 180 days after transplantation (difference, -16 percentage points [95% CI, -29.2 to -4.7 percentage points]; P = 0.01).  Superficial fungal infections occurred in 3 of 71 itraconazole recipients (4%) and in 2 of 67 fluconazole recipients (3%).  In a multivariable anal. using factors known to affect the risk for invasive fungal infection after hematopoietic stem-cell transplantation, prophylaxis with itraconazole was still assocd. with fewer invasive fungal infections (odds ratio, 0.300 [CI, 0.111 to 0.814]; P = 0.02) caused by either yeasts or molds.  More fungal pathogens were found to be resistant to fluconazole than to itraconazole.  Except for more frequent gastrointestinal side effects (nausea, vomiting, diarrhea, or abdominal pain) in patients given itraconazole (24% vs. 9%; difference, 15 percentage points [CI, 2.9 to 27.0 percentage points]; P = 0.02), both itraconazole and fluconazole were well tolerated.  The overall mortality rate was similar in each group (32 of 71 patients in the itraconazole group [45%] vs. 28 of 67 patients in the fluconazole group [42%]; difference, 3 percentage points [CI, -13.2 to 19.8 percentage points]; P > 0.2), but fewer deaths were related to fungal infection in patients given itraconazole (6 of 71 [9%]) than in patients given fluconazole (12 of 67 [18%]) (difference, 9 percentage points [CI, -20.6 to 1.8 percentage points]; P = 0.13).  Conclusion: Itraconazole is more effective than fluconazole for long-term prophylaxis of invasive fungal infections after allogeneic hematopoietic stem-cell transplantation.  Except for gastrointestinal side effects, itraconazole is well tolerated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRJ3IIYdiGfLVg90H21EOLACvtfcHk0ljiBBs9q4khhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktVyhsLg%253D&md5=eb39b1e6ccf742c6b2f3937780a1ebb9</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.7326%2F0003-4819-138-9-200305060-00006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7326%252F0003-4819-138-9-200305060-00006%26sid%3Dliteratum%253Aachs%26aulast%3DWinston%26aufirst%3DD.%2BJ.%26aulast%3DMaziarz%26aufirst%3DR.%2BT.%26aulast%3DChandrasekar%26aufirst%3DP.%2BH.%26aulast%3DLazarus%26aufirst%3DH.%2BM.%26aulast%3DGoldman%26aufirst%3DM.%26aulast%3DBlumer%26aufirst%3DJ.%2BL.%26aulast%3DLeitz%26aufirst%3DG.%2BJ.%26aulast%3DTerrito%26aufirst%3DM.%2BC.%26atitle%3DIntravenous%2520and%2520oral%2520itraconazole%2520versus%2520intravenous%2520and%2520oral%2520fluconazole%2520for%2520long-term%2520antifungal%2520prophylaxis%2520in%2520allogeneic%2520hematopoietic%2520stem-cell%2520transplant%2520recipients.%2520A%2520multicenter%252C%2520randomized%2520trial%26jtitle%3DAnn.%2520Intern.%2520Med.%26date%3D2003%26volume%3D138%26spage%3D705%26epage%3D713%26doi%3D10.7326%2F0003-4819-138-9-200305060-00006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. V.</span></span> <span> </span><span class="NLM_article-title">The use of fluconazole and itraconazole in the treatment of <i>Candida albicans</i> infections: A review</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1093/jac/44.4.429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fjac%2F44.4.429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10588302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADyaK1MXntV2lurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=1999&pages=429-437&author=M.+V.+Martin&title=The+use+of+fluconazole+and+itraconazole+in+the+treatment+of+Candida+albicans+infections%3A+A+review&doi=10.1093%2Fjac%2F44.4.429"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">The use of fluconazole and itraconazole in the treatment of Candida albicans infections: a review</span></div><div class="casAuthors">Martin, Michael V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">429-437</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review with 93 refs.  Candida albicans is responsible for most fungal infections in humans.  Fluconazole is well established as a first-line management option for the treatment and prophylaxis of localized and systemic C. albicans infections.  Fluconazole exhibits predictable pharmacokinetics and is effective, well tolerated and suitable for use in most patients with C. albicans infections, including children, the elderly and those with impaired immunity.  Prophylactic administration of fluconazole can help to prevent fungal infections in patients receiving cytotoxic cancer therapy.  The increasing use of fluconazole for the long-term prophylaxis and treatment of recurrent oral candidosis in AIDS patients has led to the emergence of C. albicans infections that are not responsive to conventional doses.  Second-line therapy with a wider spectrum anti-fungal, such as itraconazole, should be sought if treatment with fluconazole fails.  A soln. formulation of itraconazole has recently been introduced to overcome the poor and variable absorption of its original capsule formulation.  Efficacy and tolerability studies in HIV-pos. or immunocompromised patients with C. albicans infections have shown that, although itraconazole soln. is as effective as fluconazole, it is less well tolerated as first-line therapy.  Itraconazole soln. can be effective in AIDS patients with C. albicans infections that are non-responsive to fluconazole.  No efficacy or tolerability data are available on the use of itraconazole soln. in children or the elderly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfmtvdjIvc1LVg90H21EOLACvtfcHk0ljiBBs9q4khhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntV2lurs%253D&md5=e3f831937fab31bb98569375c7f4f1f2</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1093%2Fjac%2F44.4.429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252F44.4.429%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%2BV.%26atitle%3DThe%2520use%2520of%2520fluconazole%2520and%2520itraconazole%2520in%2520the%2520treatment%2520of%2520Candida%2520albicans%2520infections%253A%2520A%2520review%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D1999%26volume%3D44%26spage%3D429%26epage%3D437%26doi%3D10.1093%2Fjac%2F44.4.429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hurdle, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, I.</span></span> <span> </span><span class="NLM_article-title">Prospects for aminoacyl-tRNA synthetase inhibitors as new antimicrobial agents</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">4821</span>– <span class="NLM_lpage">4833</span>, <span class="refDoi"> DOI: 10.1128/AAC.49.12.4821-4833.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1128%2FAAC.49.12.4821-4833.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=16304142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSru7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2005&pages=4821-4833&author=J.+G.+Hurdleauthor=A.+J.+O%E2%80%99Neillauthor=I.+Chopra&title=Prospects+for+aminoacyl-tRNA+synthetase+inhibitors+as+new+antimicrobial+agents&doi=10.1128%2FAAC.49.12.4821-4833.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Prospects for aminoacyl-tRNA synthetase inhibitors as new antimicrobial agents</span></div><div class="casAuthors">Hurdle, Julian Gregston; O'Neill, Alexander John; Chopra, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4821-4833</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review with refs.  The authors discuss the prospects for the development of several recently reported AaRS inhibitors as chemotherapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL2y1GuMeel7Vg90H21EOLACvtfcHk0lhurIFiZGUeqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSru7fL&md5=fd6edfc986fd02a9540a8f567c059bf7</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1128%2FAAC.49.12.4821-4833.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.49.12.4821-4833.2005%26sid%3Dliteratum%253Aachs%26aulast%3DHurdle%26aufirst%3DJ.%2BG.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DA.%2BJ.%26aulast%3DChopra%26aufirst%3DI.%26atitle%3DProspects%2520for%2520aminoacyl-tRNA%2520synthetase%2520inhibitors%2520as%2520new%2520antimicrobial%2520agents%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2005%26volume%3D49%26spage%3D4821%26epage%3D4833%26doi%3D10.1128%2FAAC.49.12.4821-4833.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Large FK506-binding proteins shape the pharmacology of rapamycin</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1357</span>– <span class="NLM_lpage">1367</span>, <span class="refDoi"> DOI: 10.1128/MCB.00678-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1128%2FMCB.00678-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23358420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BC3szjt12mtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=1357-1367&author=A.+M.+Marzauthor=A.+K.+Fabianauthor=C.+Kozanyauthor=A.+Bracherauthor=F.+Hausch&title=Large+FK506-binding+proteins+shape+the+pharmacology+of+rapamycin&doi=10.1128%2FMCB.00678-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Large FK506-binding proteins shape the pharmacology of rapamycin</span></div><div class="casAuthors">Marz Andreas M; Fabian Anne-Katrin; Kozany Christian; Bracher Andreas; Hausch Felix</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and cellular biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1357-67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The immunosuppressant and anticancer drug rapamycin works by inducing inhibitory protein complexes with the kinase mTOR, an important regulator of growth and proliferation.  The obligatory accessory partner of rapamycin is believed to be FK506-binding protein 12 (FKBP12).  Here we show that rapamycin complexes of larger FKBP family members can tightly bind to mTOR and potently inhibit its kinase activity.  Cocrystal structures with FKBP51 and FKBP52 reveal the modified molecular binding mode of these alternative ternary complexes in detail.  In cellular model systems, FKBP12 can be functionally replaced by larger FKBPs.  When the rapamycin dosage is limiting, mTOR inhibition of S6K phosphorylation can be enhanced by FKBP51 overexpression in mammalian cells, whereas FKBP12 is dispensable.  FKBP51 could also enable the rapamycin-induced hyperphosphorylation of Akt, which depended on higher FKBP levels than rapamycin-induced inhibition of S6K phosphorylation.  These insights provide a mechanistic rationale for preferential mTOR inhibition in specific cell or tissue types by engaging specific FKBP homologs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRPwiOr3vbNw3RxY89Cm6SyfW6udTcc2eb_b3MKl7ocTrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szjt12mtg%253D%253D&md5=7422f9a3e2dd75a25dae65e7d16213bd</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1128%2FMCB.00678-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00678-12%26sid%3Dliteratum%253Aachs%26aulast%3DMarz%26aufirst%3DA.%2BM.%26aulast%3DFabian%26aufirst%3DA.%2BK.%26aulast%3DKozany%26aufirst%3DC.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DLarge%2520FK506-binding%2520proteins%2520shape%2520the%2520pharmacology%2520of%2520rapamycin%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2013%26volume%3D33%26spage%3D1357%26epage%3D1367%26doi%3D10.1128%2FMCB.00678-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Opal, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laterre, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francois, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaRosa, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angus, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mira, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittebole, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dugernier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrotin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidswell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jauregui, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peckelsen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordasco, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeyen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maruyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalil, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Nuffelen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossignol, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogate, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, J. L.</span></span>; <span class="NLM_contrib-group">ACCESS Study Group.</span> <span> </span><span class="NLM_article-title">Effect of eritoran, an antagonist of MD2-TLR4, on mortality in patients with severe sepsis: The access randomized trial</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>309</i></span>,  <span class="NLM_fpage">1154</span>– <span class="NLM_lpage">1162</span>, <span class="refDoi"> DOI: 10.1001/jama.2013.2194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1001%2Fjama.2013.2194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23512062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltlyqt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=309&publication_year=2013&pages=1154-1162&author=S.+M.+Opalauthor=P.+F.+Laterreauthor=B.+Francoisauthor=S.+P.+LaRosaauthor=D.+C.+Angusauthor=J.+P.+Miraauthor=X.+Witteboleauthor=T.+Dugernierauthor=D.+Perrotinauthor=M.+Tidswellauthor=L.+Jaureguiauthor=K.+Krellauthor=J.+Pachlauthor=T.+Takahashiauthor=C.+Peckelsenauthor=E.+Cordascoauthor=C.+S.+Changauthor=S.+Oeyenauthor=N.+Aikawaauthor=T.+Maruyamaauthor=R.+Scheinauthor=A.+C.+Kalilauthor=M.+Van+Nuffelenauthor=M.+Lynnauthor=D.+P.+Rossignolauthor=J.+Gogateauthor=M.+B.+Robertsauthor=J.+L.+Wheelerauthor=J.+L.+Vincentauthor=ACCESS+Study+Group.&title=Effect+of+eritoran%2C+an+antagonist+of+MD2-TLR4%2C+on+mortality+in+patients+with+severe+sepsis%3A+The+access+randomized+trial&doi=10.1001%2Fjama.2013.2194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of eritoran, an antagonist of MD2-TLR4, on mortality in patients with severe sepsis. The ACCESS randomized trial</span></div><div class="casAuthors">Opal, Steven M.; Laterre, Pierre-Francois; Francois, Bruno; LaRosa, Steven P.; Angus, Derek C.; Mira, Jean-Paul; Wittebole, Xavier; Dugernier, Thierry; Perrotin, Dominique; Tidswell, Mark; Jauregui, Luis; Krell, Kenneth; Pachl, Jan; Takahashi, Takeshi; Peckelsen, Claus; Cordasco, Edward; Chang, Chia-Sheng; Oeyen, Sandra; Aikawa, Naoki; Maruyama, Tatsuya; Schein, Roland; Kalil, Andre C.; Van Nuffelen, Marc; Lynn, Melvyn; Rossignol, Daniel P.; Gogate, Jogadish; Roberts, Mary B.; Wheeler, Janice L.; Vincent, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">309</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1154-1162</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">0098-7484</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">Importance: Eritoran is a synthetic lipid A antagonist that blocks Iipopolysaccharide (LPS) from binding at the cell surface MD2-TLR4 receptor.  LPS is a major component of the outer membrane of gram-neg. bacteria and is a potent activator of the acute inflammatory response.  Objective: To det. if eritoran, a TLR4 antagonist, would significantly reduce sepsis-induced mortality.  Design, Setting, and Participants: We performed a randomized, double-blind, placebo-controlled, multinational phase 3 trial in 197 intensive care units.  Patients were enrolled from June 2006 to Sept. 2010 and final follow-up was completed in Sept. 2011.  Interventions: Patients with severe sepsis (n = 1961) were randomized and treated within 12 h of onset of first organ dysfunction in a 2:1 ratio with a 6-day course of either eritoran tetrasodium (105 mg total) or placebo, with n = 1304 and n = 657 patients, resp.  Main Outcome Measures: The primary end point was 28-day all-cause mortality.  The secondary end points were all-cause mortality at 3, 6, and 12 mo after beginning treatment.  Results: Baseline characteristics of the 2 study groups were similar.  In the modified intent-to-treat anal. (randomized patients who received at least 1 dose) there was no significant difference in the primary end point of 28-day all-cause mortality with 28.1% (366/1304) in the eritoran group vs 26.9% (177/657) in the placebo group (P = .59; hazard ratio, 1.05; 95% CI, 0.88-1.26; difference in mortality rate, -1.1; 95% CI, -5.3 to 3.1) or in the key secondary end point of 1-yr all-cause mortality with 44.1% (290/657) in the eritoran group vs 43.3% (565/1304) in the placebo group, Kaplan-Meier anal. of time to death by 1 yr, P = .79 (hazard ratio, 0.98; 0.85-1.13).  No significant differences were obsd. in any of the prespecified subgroups.  Adverse events, including secondary infection rates, did not differ between study groups.  Conclusions and Relevance: Among patients with severe sepsis, the use of eritoran, compared with placebo, did not result in reduced 28-day mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrbHqr8OtY8bVg90H21EOLACvtfcHk0ljK6I-bpaq8oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltlyqt7o%253D&md5=adf1d02bc28953d8e6b30e9d627fcfa9</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1001%2Fjama.2013.2194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.2013.2194%26sid%3Dliteratum%253Aachs%26aulast%3DOpal%26aufirst%3DS.%2BM.%26aulast%3DLaterre%26aufirst%3DP.%2BF.%26aulast%3DFrancois%26aufirst%3DB.%26aulast%3DLaRosa%26aufirst%3DS.%2BP.%26aulast%3DAngus%26aufirst%3DD.%2BC.%26aulast%3DMira%26aufirst%3DJ.%2BP.%26aulast%3DWittebole%26aufirst%3DX.%26aulast%3DDugernier%26aufirst%3DT.%26aulast%3DPerrotin%26aufirst%3DD.%26aulast%3DTidswell%26aufirst%3DM.%26aulast%3DJauregui%26aufirst%3DL.%26aulast%3DKrell%26aufirst%3DK.%26aulast%3DPachl%26aufirst%3DJ.%26aulast%3DTakahashi%26aufirst%3DT.%26aulast%3DPeckelsen%26aufirst%3DC.%26aulast%3DCordasco%26aufirst%3DE.%26aulast%3DChang%26aufirst%3DC.%2BS.%26aulast%3DOeyen%26aufirst%3DS.%26aulast%3DAikawa%26aufirst%3DN.%26aulast%3DMaruyama%26aufirst%3DT.%26aulast%3DSchein%26aufirst%3DR.%26aulast%3DKalil%26aufirst%3DA.%2BC.%26aulast%3DVan%2BNuffelen%26aufirst%3DM.%26aulast%3DLynn%26aufirst%3DM.%26aulast%3DRossignol%26aufirst%3DD.%2BP.%26aulast%3DGogate%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DM.%2BB.%26aulast%3DWheeler%26aufirst%3DJ.%2BL.%26aulast%3DVincent%26aufirst%3DJ.%2BL.%26aulast%3D%26atitle%3DEffect%2520of%2520eritoran%252C%2520an%2520antagonist%2520of%2520MD2-TLR4%252C%2520on%2520mortality%2520in%2520patients%2520with%2520severe%2520sepsis%253A%2520The%2520access%2520randomized%2520trial%26jtitle%3DJAMA%26date%3D2013%26volume%3D309%26spage%3D1154%26epage%3D1162%26doi%3D10.1001%2Fjama.2013.2194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dieci, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnedos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andre, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor receptor inhibitors as a cancer treatment: From a biologic rationale to medical perspectives</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-12-0362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1158%2F2159-8290.CD-12-0362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23418312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVOktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=264-279&author=M.+V.+Dieciauthor=M.+Arnedosauthor=F.+Andreauthor=J.+C.+Soria&title=Fibroblast+growth+factor+receptor+inhibitors+as+a+cancer+treatment%3A+From+a+biologic+rationale+to+medical+perspectives&doi=10.1158%2F2159-8290.CD-12-0362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast Growth Factor Receptor Inhibitors as a Cancer Treatment: From a Biologic Rationale to Medical Perspectives</span></div><div class="casAuthors">Dieci, Maria Vittoria; Arnedos, Monica; Andre, Fabrice; Soria, Jean Charles</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">264-279</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The fibroblast growth factor/fibroblast growth factor receptor (FGF/FGFR) signaling pathway plays a fundamental role in many physiol. processes, including embryogenesis, adult tissue homeostasis, and wound healing, by orchestrating angiogenesis.  Ligand-independent and ligand-dependent activation have been implicated in a broad range of human malignancies and promote cancer progression in tumors driven by FGF/FGFR oncogenic mutations or amplifications, tumor neoangiogenesis, and targeted treatment resistance, thereby supporting a strong rationale for anti-FGF/FGFR agent development.  Efforts are being pursued to develop selective approaches for use against this pathway by optimizing the management of emerging, class-specific toxicity profiles and correctly designing clin. trials to address these different issues.  Significance: FGF/FGFR pathway deregulations are increasingly recognized across different human cancers.  Understanding the mechanisms at the basis of these alterations and their multiple roles in cancer promotion and drug resistance is a fundamental step for further implementation of targeted therapies and research strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6bibZKwamTLVg90H21EOLACvtfcHk0ljK6I-bpaq8oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVOktL4%253D&md5=0e694a7641aa83c1540f745576f04bfa</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-12-0362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-12-0362%26sid%3Dliteratum%253Aachs%26aulast%3DDieci%26aufirst%3DM.%2BV.%26aulast%3DArnedos%26aufirst%3DM.%26aulast%3DAndre%26aufirst%3DF.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26atitle%3DFibroblast%2520growth%2520factor%2520receptor%2520inhibitors%2520as%2520a%2520cancer%2520treatment%253A%2520From%2520a%2520biologic%2520rationale%2520to%2520medical%2520perspectives%26jtitle%3DCancer%2520Discovery%26date%3D2013%26volume%3D3%26spage%3D264%26epage%3D279%26doi%3D10.1158%2F2159-8290.CD-12-0362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guagnano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Douget, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghausen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussiere, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus Porta, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of 3-(2,6-dichloro-3,5-dimethoxy-phenyl)-1-{6-[4-(4-ethyl-piperazin-1-yl)-phenylamin o]-pyrimidin-4-yl}-1-methyl-urea (NVP-BGJ398), a potent and selective inhibitor of the fibroblast growth factor receptor family of receptor tyrosine kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7066</span>– <span class="NLM_lpage">7083</span>, <span class="refDoi"> DOI: 10.1021/jm2006222</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2006222" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7066-7083&author=V.+Guagnanoauthor=P.+Furetauthor=C.+Spankaauthor=V.+Bordasauthor=M.+Le+Dougetauthor=C.+Stammauthor=J.+Brueggenauthor=M.+R.+Jensenauthor=C.+Schnellauthor=H.+Schmidauthor=M.+Wartmannauthor=J.+Berghausenauthor=P.+Drueckesauthor=A.+Zimmerlinauthor=D.+Bussiereauthor=J.+Murrayauthor=D.+Graus+Porta&title=Discovery+of+3-%282%2C6-dichloro-3%2C5-dimethoxy-phenyl%29-1-%7B6-%5B4-%284-ethyl-piperazin-1-yl%29-phenylamin+o%5D-pyrimidin-4-yl%7D-1-methyl-urea+%28NVP-BGJ398%29%2C+a+potent+and+selective+inhibitor+of+the+fibroblast+growth+factor+receptor+family+of+receptor+tyrosine+kinase&doi=10.1021%2Fjm2006222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-(2,6-Dichloro-3,5-dimethoxy-phenyl)-1-{6-[4-(4-ethyl-piperazin-1-yl)-phenylamino]-pyrimidin-4-yl}-1-methyl-urea (NVP-BGJ398), A Potent and Selective Inhibitor of the Fibroblast Growth Factor Receptor Family of Receptor Tyrosine Kinase</span></div><div class="casAuthors">Guagnano, Vito; Furet, Pascal; Spanka, Carsten; Bordas, Vincent; Le Douget, Mickael; Stamm, Christelle; Brueggen, Josef; Jensen, Michael R.; Schnell, Christian; Schmid, Herbert; Wartmann, Markus; Berghausen, Joerg; Drueckes, Peter; Zimmerlin, Alfred; Bussiere, Dirksen; Murray, Jeremy; Graus Porta, Diana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7066-7083</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of N-aryl-N'-pyrimidin-4-yl ureas has been optimized to afford potent and selective inhibitors of the fibroblast growth factor receptor tyrosine kinases 1, 2, and 3 by rationally designing the substitution pattern of the aryl ring.  On the basis of its in vitro profile, compd. 1h (NVP-BGJ398) was selected for in vivo evaluation and showed significant antitumor activity in RT112 bladder cancer xenografts models overexpressing wild-type FGFR3.  These results support the potential therapeutic use of 1h as a new anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSC1v0A_6I_bVg90H21EOLACvtfcHk0ljfdjVoRC0yFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK&md5=f9a2d742294cd3814e872bdd8add1ffa</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fjm2006222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2006222%26sid%3Dliteratum%253Aachs%26aulast%3DGuagnano%26aufirst%3DV.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DSpanka%26aufirst%3DC.%26aulast%3DBordas%26aufirst%3DV.%26aulast%3DLe%2BDouget%26aufirst%3DM.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DSchnell%26aufirst%3DC.%26aulast%3DSchmid%26aufirst%3DH.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DBerghausen%26aufirst%3DJ.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DZimmerlin%26aufirst%3DA.%26aulast%3DBussiere%26aufirst%3DD.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DGraus%2BPorta%26aufirst%3DD.%26atitle%3DDiscovery%2520of%25203-%25282%252C6-dichloro-3%252C5-dimethoxy-phenyl%2529-1-%257B6-%255B4-%25284-ethyl-piperazin-1-yl%2529-phenylamin%2520o%255D-pyrimidin-4-yl%257D-1-methyl-urea%2520%2528NVP-BGJ398%2529%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520the%2520fibroblast%2520growth%2520factor%2520receptor%2520family%2520of%2520receptor%2520tyrosine%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7066%26epage%3D7083%26doi%3D10.1021%2Fjm2006222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olziersky, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labidi-Galy, S. I.</span></span> <span> </span><span class="NLM_article-title">Clinical development of anti-mitotic drugs in cancer</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1002</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1007/978-3-319-57127-0_6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1007%2F978-3-319-57127-0_6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=28600785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFCns7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1002&publication_year=2017&pages=125-152&author=A.+M.+Olzierskyauthor=S.+I.+Labidi-Galy&title=Clinical+development+of+anti-mitotic+drugs+in+cancer&doi=10.1007%2F978-3-319-57127-0_6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical development of anti-mitotic drugs in cancer</span></div><div class="casAuthors">Olziersky, Anna-Maria; Labidi-Galy, S. Intidhar</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1002</span>
        (<span class="NLM_cas:issue">Cell Division Machinery and Disease</span>),
    <span class="NLM_cas:pages">125-152</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">2214-8019</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Mitosis is one of the most fundamental processes of life by which a mammalian cell divides into two daughter cells.  Mitosis has been an attractive target for anticancer therapies since fast proliferation was identified as one of the hallmarks of cancer cells.  Despite efforts into developing specific inhibitors for mitotic kinases and kinesins, very few drugs have shown the efficiency of microtubule targeting-agents in cancer cells with paclitaxel being the most successful.  A deeper translational research accompanying clin. trials of anti-mitotic drugs will help in identifying potent biomarkers predictive for response.  Here, we review the current knowledge of mitosis targeting agents that have been tested so far in the clinics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp80dIlYTJrsrVg90H21EOLACvtfcHk0ljfdjVoRC0yFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFCns7nL&md5=ecf3ecdb4b8fd3e0ce5e5f0e20ca7654</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-57127-0_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-3-319-57127-0_6%26sid%3Dliteratum%253Aachs%26aulast%3DOlziersky%26aufirst%3DA.%2BM.%26aulast%3DLabidi-Galy%26aufirst%3DS.%2BI.%26atitle%3DClinical%2520development%2520of%2520anti-mitotic%2520drugs%2520in%2520cancer%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D2017%26volume%3D1002%26spage%3D125%26epage%3D152%26doi%3D10.1007%2F978-3-319-57127-0_6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Purcell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samayoa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bean, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziyad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feiler, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cases, S.</span></span> <span> </span><span class="NLM_article-title">Activity of the kinesin spindle protein inhibitor ispinesib (SB-715992) in models of breast cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">566</span>– <span class="NLM_lpage">576</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-09-1498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1158%2F1078-0432.CCR-09-1498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=20068098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktF2qtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=566-576&author=J.+W.+Purcellauthor=J.+Davisauthor=M.+Reddyauthor=S.+Martinauthor=K.+Samayoaauthor=H.+Voauthor=K.+Thomsenauthor=P.+Beanauthor=W.+L.+Kuoauthor=S.+Ziyadauthor=J.+Billigauthor=H.+S.+Feilerauthor=J.+W.+Grayauthor=K.+W.+Woodauthor=S.+Cases&title=Activity+of+the+kinesin+spindle+protein+inhibitor+ispinesib+%28SB-715992%29+in+models+of+breast+cancer&doi=10.1158%2F1078-0432.CCR-09-1498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Activity of the kinesin spindle protein inhibitor ispinesib (SB-715992) in models of breast cancer</span></div><div class="casAuthors">Purcell, James W.; Davis, Jefferson; Reddy, Mamatha; Martin, Shamra; Samayoa, Kimberly; Vo, Hung; Thomsen, Karen; Bean, Peter; Kuo, Wen Lin; Ziyad, Safiyyah; Billig, Jessica; Feiler, Heidi S.; Gray, Joe W.; Wood, Kenneth W.; Cases, Sylvaine</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">566-576</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Ispinesib (SB-715992) is a potent inhibitor of kinesin spindle protein, a kinesin motor protein essential for the formation of a bipolar mitotic spindle and cell cycle progression through mitosis.  Clin. studies of ispinesib have shown a 9% response rate in patients with locally advanced or metastatic breast cancer and a favorable safety profile without significant neurotoxicities, gastrointestinal toxicities, or hair loss.  To better understand the potential of ispinesib in the treatment of breast cancer, we explored the activity of ispinesib alone and in combination with several therapies approved for the treatment of breast cancer.  Exptl. Design: We measured the ispinesib sensitivity and pharmacodynamic response of breast cancer cell lines representative of various subtypes in vitro and as xenografts in vivo and tested the ability of ispinesib to enhance the antitumor activity of approved therapies.  Results: In vitro, ispinesib displayed broad antiproliferative activity against a panel of 53 breast cell lines.  In vivo, ispinesib produced regressions in each of five breast cancer models and tumor-free survivors in three of these models.  The effects of ispinesib treatment on pharmacodynamic markers of mitosis and apoptosis were examd. in vitro and in vivo, revealing a greater increase in both mitotic and apoptotic markers in the MDA-MB-468 model than in the less sensitive BT-474 model.  In vivo, ispinesib enhanced the antitumor activity of trastuzumab, lapatinib, doxorubicin, and capecitabine and exhibited activity comparable with paclitaxel and ixabepilone.  Conclusions: These findings support further clin. exploration of kinesin spindle protein inhibitors for the treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJseCrHGemNrVg90H21EOLACvtfcHk0liCSR6cpn1DWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktF2qtrc%253D&md5=2868435e2d157dd55cd7c68f9530c1df</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-1498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-1498%26sid%3Dliteratum%253Aachs%26aulast%3DPurcell%26aufirst%3DJ.%2BW.%26aulast%3DDavis%26aufirst%3DJ.%26aulast%3DReddy%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSamayoa%26aufirst%3DK.%26aulast%3DVo%26aufirst%3DH.%26aulast%3DThomsen%26aufirst%3DK.%26aulast%3DBean%26aufirst%3DP.%26aulast%3DKuo%26aufirst%3DW.%2BL.%26aulast%3DZiyad%26aufirst%3DS.%26aulast%3DBillig%26aufirst%3DJ.%26aulast%3DFeiler%26aufirst%3DH.%2BS.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26aulast%3DWood%26aufirst%3DK.%2BW.%26aulast%3DCases%26aufirst%3DS.%26atitle%3DActivity%2520of%2520the%2520kinesin%2520spindle%2520protein%2520inhibitor%2520ispinesib%2520%2528SB-715992%2529%2520in%2520models%2520of%2520breast%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2010%26volume%3D16%26spage%3D566%26epage%3D576%26doi%3D10.1158%2F1078-0432.CCR-09-1498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Istvan, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deisenhofer, J.</span></span> <span> </span><span class="NLM_article-title">Structural mechanism for statin inhibition of HMG-CoA reductase</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1164</span>, <span class="refDoi"> DOI: 10.1126/science.1059344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1126%2Fscience.1059344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11349148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjs1Cgtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2001&pages=1160-1164&author=E.+S.+Istvanauthor=J.+Deisenhofer&title=Structural+mechanism+for+statin+inhibition+of+HMG-CoA+reductase&doi=10.1126%2Fscience.1059344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism for statin inhibition of HMG-CoA reductase</span></div><div class="casAuthors">Istvan, Eva S.; Deisenhofer, Johann</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">5519</span>),
    <span class="NLM_cas:pages">1160-1164</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">HMG-CoA (3-hydroxy-3-methylglutaryl-CoA) reductase (HMGR) catalyzes the committed step in cholesterol biosynthesis.  Statins are HMGR inhibitors with inhibition const. values in the nanomolar range that effectively lower serum cholesterol levels and are widely prescribed in the treatment of hypercholesterolemia.  We have detd. structures of the catalytic portion of human HMGR complexed with six different statins.  The statins occupy a portion of the binding site of HMG-CoA, thus blocking access of this substrate to the active site.  Near the carboxyl terminus of HMGR, several catalytically relevant residues are disordered in the enzyme-statin complexes.  If these residues were not flexible, they would sterically hinder statin binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDsSOnK9675bVg90H21EOLACvtfcHk0liCSR6cpn1DWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjs1Cgtbo%253D&md5=235644ee8cca3d2e061093e93bd1d702</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1126%2Fscience.1059344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1059344%26sid%3Dliteratum%253Aachs%26aulast%3DIstvan%26aufirst%3DE.%2BS.%26aulast%3DDeisenhofer%26aufirst%3DJ.%26atitle%3DStructural%2520mechanism%2520for%2520statin%2520inhibition%2520of%2520HMG-CoA%2520reductase%26jtitle%3DScience%26date%3D2001%26volume%3D292%26spage%3D1160%26epage%3D1164%26doi%3D10.1126%2Fscience.1059344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fouty, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodman, D. M.</span></span> <span> </span><span class="NLM_article-title">Mevastatin can cause G1 arrest and induce apoptosis in pulmonary artery smooth muscle cells through a p27<sup>KIP1</sup>-independent pathway</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1161/01.RES.0000061180.03813.0F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1161%2F01.RES.0000061180.03813.0F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=12600884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitVSjsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2003&pages=501-509&author=B.+W.+Foutyauthor=D.+M.+Rodman&title=Mevastatin+can+cause+G1+arrest+and+induce+apoptosis+in+pulmonary+artery+smooth+muscle+cells+through+a+p27KIP1-independent+pathway&doi=10.1161%2F01.RES.0000061180.03813.0F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Mevastatin Can Cause G1 Arrest and Induce Apoptosis in Pulmonary Artery Smooth Muscle Cells Through a p27Kip1-Independent Pathway</span></div><div class="casAuthors">Fouty, Brian W.; Rodman, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Circulation Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">501-509</span>CODEN:
                <span class="NLM_cas:coden">CIRUAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-7330</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Advanced pulmonary arterial hypertension is characterized by extensive vascular remodeling that is usually resistant to vasodilator therapy.  Mevastatin is an inhibitor of 3-hydroxy-3-methylglutaryl-CoA (HMG-CoA) reductase, the rate-limiting step for cholesterol synthesis.  HMG-CoA reductase inhibitors have been shown to upregulate the cyclin-dependent kinase inhibitor p27Kip1 and to block cell proliferation through cholesterol-independent pathways.  The aim of this study was to det. the effect of mevastatin on DNA synthesis, cell cycle progression, and cell proliferation in rat pulmonary artery smooth muscle cells (PASMCs).  We found that mevastatin induced G1 arrest and decreased DNA synthesis in rat PASMCs and did so in assocn. with an increase in both total and cyclin E-bound p27Kip1.  This caused a marked decrease in cyclin E kinase activity, which suggests an important role for p27Kip1 in the ability of mevastatin to induce G1 arrest.  However, in PASMCs lacking functional p27Kip1, mevastatin still decreased cyclin E kinase activity, caused G1 arrest, and decreased DNA synthesis.  In p27Kip1-deficient PASMCs, mevastatin induced a greater redn. of cyclin E protein levels (to 35% of control) than in wild-type cells (to 70% of control) and also reduced the phosphorylation of cdk2 on threonine 160.  Mevastatin also caused apoptosis in both wild-type and p27Kip1-deficient PASMCs and was able to do so at a dose that did not induce cell cycle arrest.  These data suggest that HMG-CoA reductase inhibitors can both inhibit cell proliferation and induce apoptosis in PASMCs through p27Kip1-independent pathways and may be important therapeutic agents in pulmonary arterial hypertension.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodDTLdxOMx-LVg90H21EOLACvtfcHk0liCSR6cpn1DWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitVSjsbY%253D&md5=b99b21f5120251ca1ff421613db6ef6e</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1161%2F01.RES.0000061180.03813.0F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.RES.0000061180.03813.0F%26sid%3Dliteratum%253Aachs%26aulast%3DFouty%26aufirst%3DB.%2BW.%26aulast%3DRodman%26aufirst%3DD.%2BM.%26atitle%3DMevastatin%2520can%2520cause%2520G1%2520arrest%2520and%2520induce%2520apoptosis%2520in%2520pulmonary%2520artery%2520smooth%2520muscle%2520cells%2520through%2520a%2520p27KIP1-independent%2520pathway%26jtitle%3DCirc.%2520Res.%26date%3D2003%26volume%3D92%26spage%3D501%26epage%3D509%26doi%3D10.1161%2F01.RES.0000061180.03813.0F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strickland, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windsor, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanese, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesarz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Rosario, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deskus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallams, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Njoroge, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piwinski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remiszewski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossman, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taveras, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vibulbhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doll, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girijavallabhan, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganguly, A. K.</span></span> <span> </span><span class="NLM_article-title">Tricyclic farnesyl protein transferase inhibitors: Crystallographic and calorimetric studies of structure-activity relationships</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">2125</span>– <span class="NLM_lpage">2135</span>, <span class="refDoi"> DOI: 10.1021/jm990030g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm990030g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADyaK1MXjtlOltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=2125-2135&author=C.+L.+Stricklandauthor=P.+C.+Weberauthor=W.+T.+Windsorauthor=Z.+Wuauthor=H.+V.+Leauthor=M.+M.+Albaneseauthor=C.+S.+Alvarezauthor=D.+Cesarzauthor=J.+del+Rosarioauthor=J.+Deskusauthor=A.+K.+Mallamsauthor=F.+G.+Njorogeauthor=J.+J.+Piwinskiauthor=S.+Remiszewskiauthor=R.+R.+Rossmanauthor=A.+G.+Taverasauthor=B.+Vibulbhanauthor=R.+J.+Dollauthor=V.+M.+Girijavallabhanauthor=A.+K.+Ganguly&title=Tricyclic+farnesyl+protein+transferase+inhibitors%3A+Crystallographic+and+calorimetric+studies+of+structure-activity+relationships&doi=10.1021%2Fjm990030g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Tricyclic Farnesyl Protein Transferase Inhibitors: Crystallographic and Calorimetric Studies of Structure-Activity Relationships</span></div><div class="casAuthors">Strickland, Corey L.; Weber, Patricia C.; Windsor, William T.; Wu, Zhen; Le, Hung V.; Albanese, Margaret M.; Alvarez, Carmen S.; Cesarz, David; del Rosario, Joycelyn; Deskus, Jeffrey; Mallams, Alan K.; Njoroge, F. George; Piwinski, John J.; Remiszewski, Stacy; Rossman, Randall R.; Taveras, Arthur G.; Vibulbhan, Bancha; Doll, Ronald J.; Girijavallabhan, Viyyoor M.; Ganguly, Ashit K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2125-2135</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Crystallog. and thermodn. studies of farnesyl protein transferase (FPT) complexed with novel tricyclic inhibitors provide insights into the obsd. SAR for this unique class of nonpeptidic FPT inhibitors.  The crystallog. structures reveal a binding pattern conserved across the mono-, di-, and trihalogen series.  In the complexes, the tricycle spans the FPT active site cavity and interacts with both protein atoms and the isoprenoid portion of bound farnesyl diphosphate.  An amide carbonyl, common to the tricyclic compds. described here, participates in a water-mediated hydrogen bond to the protein backbone.  Ten high-resoln. crystal structures of inhibitors complexed with FPT are reported.  Included are crystallog. data for FPT complexed with SCH 66336, a compd. currently undergoing clin. trials as an anticancer agent (SCH 66336, 4-[2-[4-(3,10-dibromo-8-chloro-6,11-dihydro-5H-benzo[5,6]cyclohepta[1,2-b]pyridin-11-yl)-1-piperidinyl]-2-oxoethyl]-1-piperidinecarboxamide).  Thermodn. binding parameters show favorable enthalpies of complex formation and small net entropic contributions as obsd. for 4-[2-[4-(3,10-dibromo-8-chloro-6,11-dihydro-11H-benzo[5,6]cyclohepta[1,2-b]pyridin-11-ylidene)-1-piperidinyl]-2-oxoethyl]pyridine N-oxide where ΔH°bind = -12.5 kcal/mol and TΔS°bind = -1.5 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0d_n_7Jl-B7Vg90H21EOLACvtfcHk0lj5RUvxZXB8GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtlOltr8%253D&md5=c49d51c4c82ff82f248091c4d374374c</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fjm990030g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990030g%26sid%3Dliteratum%253Aachs%26aulast%3DStrickland%26aufirst%3DC.%2BL.%26aulast%3DWeber%26aufirst%3DP.%2BC.%26aulast%3DWindsor%26aufirst%3DW.%2BT.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DLe%26aufirst%3DH.%2BV.%26aulast%3DAlbanese%26aufirst%3DM.%2BM.%26aulast%3DAlvarez%26aufirst%3DC.%2BS.%26aulast%3DCesarz%26aufirst%3DD.%26aulast%3Ddel%2BRosario%26aufirst%3DJ.%26aulast%3DDeskus%26aufirst%3DJ.%26aulast%3DMallams%26aufirst%3DA.%2BK.%26aulast%3DNjoroge%26aufirst%3DF.%2BG.%26aulast%3DPiwinski%26aufirst%3DJ.%2BJ.%26aulast%3DRemiszewski%26aufirst%3DS.%26aulast%3DRossman%26aufirst%3DR.%2BR.%26aulast%3DTaveras%26aufirst%3DA.%2BG.%26aulast%3DVibulbhan%26aufirst%3DB.%26aulast%3DDoll%26aufirst%3DR.%2BJ.%26aulast%3DGirijavallabhan%26aufirst%3DV.%2BM.%26aulast%3DGanguly%26aufirst%3DA.%2BK.%26atitle%3DTricyclic%2520farnesyl%2520protein%2520transferase%2520inhibitors%253A%2520Crystallographic%2520and%2520calorimetric%2520studies%2520of%2520structure-activity%2520relationships%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D2125%26epage%3D2135%26doi%3D10.1021%2Fjm990030g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walker, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacold, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perisic, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wymann, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R. L.</span></span> <span> </span><span class="NLM_article-title">Structural determinants of phosphoinositide 3-kinase inhibition by wortmannin, LY294002, quercetin, myricetin, and staurosporine</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">909</span>– <span class="NLM_lpage">919</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(05)00089-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2FS1097-2765%2805%2900089-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11090628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVegurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=909-919&author=E.+H.+Walkerauthor=M.+E.+Pacoldauthor=O.+Perisicauthor=L.+Stephensauthor=P.+T.+Hawkinsauthor=M.+P.+Wymannauthor=R.+L.+Williams&title=Structural+determinants+of+phosphoinositide+3-kinase+inhibition+by+wortmannin%2C+LY294002%2C+quercetin%2C+myricetin%2C+and+staurosporine&doi=10.1016%2FS1097-2765%2805%2900089-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants of phosphoinositide 3-kinase inhibition by wortmannin, LY294002, quercetin, myricetin, and staurosporine</span></div><div class="casAuthors">Walker, Edward H.; Pacold, Michael E.; Perisic, Olga; Stephens, Len; Hawkins, Philip T.; Wymann, Matthias P.; Williams, Roger L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">909-919</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The specific phosphoinositide 3-kinase (P13K) inhibitors wortmannin and LY294002 have been invaluable tools for elucidating the roles of these enzymes in signal transduction pathways.  The X-ray crystallog. structures of P13K bound to these lipid kinase inhibitors and to the broad-spectrum protein kinase inhibitors quercetin, myricetin, and staurosporine reveal how these compds. fit into the ATP binding pocket.  With a nanomolar IC50, wortmannin most closely fits and fills the active site and induces a conformational change in the catalytic domain.  Surprisingly, LY294002 and the lead compd. on which it was designed, quercetin, as well as the closely related flavonoid myricetin bind P13K in remarkably different orientations that are related to each other by 1800 rotations.  Staurosporine/P13K interactions are reminiscent of low-affinity protein kinase/staurosporine complexes.  These results provide a rich basis for development of isoform-specific P13K inhibitors with therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgIbTky2xQmbVg90H21EOLACvtfcHk0lj5RUvxZXB8GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVegurY%253D&md5=5dd92f9b48c437e43c0023fdff10c1ac</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2805%2900089-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252805%252900089-4%26sid%3Dliteratum%253Aachs%26aulast%3DWalker%26aufirst%3DE.%2BH.%26aulast%3DPacold%26aufirst%3DM.%2BE.%26aulast%3DPerisic%26aufirst%3DO.%26aulast%3DStephens%26aufirst%3DL.%26aulast%3DHawkins%26aufirst%3DP.%2BT.%26aulast%3DWymann%26aufirst%3DM.%2BP.%26aulast%3DWilliams%26aufirst%3DR.%2BL.%26atitle%3DStructural%2520determinants%2520of%2520phosphoinositide%25203-kinase%2520inhibition%2520by%2520wortmannin%252C%2520LY294002%252C%2520quercetin%252C%2520myricetin%252C%2520and%2520staurosporine%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D6%26spage%3D909%26epage%3D919%26doi%3D10.1016%2FS1097-2765%2805%2900089-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Folkes, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmadi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alderton, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alix, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuckowree, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depledge, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eccles, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancox, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kugendradas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lensun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivero, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pergl-Wilson, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynaud, F. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saghir, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salphati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallweber, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhyvoloup, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zvelebil, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuttleworth, S. J.</span></span> <span> </span><span class="NLM_article-title">The identification of 2-(1h-indazol-4-yl)-6-(4-methanesulfonyl-piperazin-1-ylmethyl)-4-morpholin-4-yl-t hieno[3,2-d]pyrimidine (GDC-0941) as a potent, selective, orally bioavailable inhibitor of class I PI3 kinase for the treatment of cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5522</span>– <span class="NLM_lpage">5532</span>, <span class="refDoi"> DOI: 10.1021/jm800295d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800295d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVCmsr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5522-5532&author=A.+J.+Folkesauthor=K.+Ahmadiauthor=W.+K.+Aldertonauthor=S.+Alixauthor=S.+J.+Bakerauthor=G.+Boxauthor=I.+S.+Chuckowreeauthor=P.+A.+Clarkeauthor=P.+Depledgeauthor=S.+A.+Ecclesauthor=L.+S.+Friedmanauthor=A.+Hayesauthor=T.+C.+Hancoxauthor=A.+Kugendradasauthor=L.+Lensunauthor=P.+Mooreauthor=A.+G.+Oliveroauthor=J.+Pangauthor=S.+Patelauthor=G.+H.+Pergl-Wilsonauthor=F.+I.+Raynaudauthor=A.+Robsonauthor=N.+Saghirauthor=L.+Salphatiauthor=S.+Sohalauthor=M.+H.+Ultschauthor=M.+Valentiauthor=H.+J.+Wallweberauthor=N.+C.+Wanauthor=C.+Wiesmannauthor=P.+Workmanauthor=A.+Zhyvoloupauthor=M.+J.+Zvelebilauthor=S.+J.+Shuttleworth&title=The+identification+of+2-%281h-indazol-4-yl%29-6-%284-methanesulfonyl-piperazin-1-ylmethyl%29-4-morpholin-4-yl-t+hieno%5B3%2C2-d%5Dpyrimidine+%28GDC-0941%29+as+a+potent%2C+selective%2C+orally+bioavailable+inhibitor+of+class+I+PI3+kinase+for+the+treatment+of+cancer&doi=10.1021%2Fjm800295d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">The identification of 2-(1H-indazol-4-yl)-6-(4-methanesulfonyl-piperazin-1-ylmethyl)-4-morpholin-4-yl-thieno[3,2-d]pyrimidine (GDC-0941) as a potent, selective, orally bioavailable inhibitor of class I PI3 kinase for the treatment of cancer</span></div><div class="casAuthors">Folkes, Adrian J.; Ahmadi, Khatereh; Alderton, Wendy K.; Alix, Sonia; Baker, Stewart J.; Box, Gary; Chuckowree, Irina S.; Clarke, Paul A.; Depledge, Paul; Eccles, Suzanne A.; Friedman, Lori S.; Hayes, Angela; Hancox, Timothy C.; Kugendradas, Arumugam; Lensun, Letitia; Moore, Pauline; Olivero, Alan G.; Pang, Jodie; Patel, Sonal; Pergl-Wilson, Giles H.; Raynaud, Florence I.; Robson, Anthony; Saghir, Nahid; Salphati, Laurent; Sohal, Sukhjit; Ultsch, Mark H.; Valenti, Melanie; Wallweber, Heidi J. A.; Wan, Nan Chi; Wiesmann, Christian; Workman, Paul; Zhyvoloup, Alexander; Zvelebil, Marketa J.; Shuttleworth, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5522-5532</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphatidylinositol-3-kinase (PI3K) is an important target in cancer due to the deregulation of the PI3K/Akt signaling pathway in a wide variety of tumors.  A series of thieno[3,2-d]pyrimidine derivs. were prepd. and evaluated as inhibitors of PI3 kinase p110α.  The synthesis, biol. activity, and further profiling of these compds. are described.  This work resulted in the discovery of GDC-0941 (I) which is a potent, selective, orally bioavailable inhibitor of PI3K and is currently being evaluated in human clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojkFv29FXqO7Vg90H21EOLACvtfcHk0lhwG3EMdvoq4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVCmsr7E&md5=742084954685353ceaf18660ab3a0703</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Fjm800295d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800295d%26sid%3Dliteratum%253Aachs%26aulast%3DFolkes%26aufirst%3DA.%2BJ.%26aulast%3DAhmadi%26aufirst%3DK.%26aulast%3DAlderton%26aufirst%3DW.%2BK.%26aulast%3DAlix%26aufirst%3DS.%26aulast%3DBaker%26aufirst%3DS.%2BJ.%26aulast%3DBox%26aufirst%3DG.%26aulast%3DChuckowree%26aufirst%3DI.%2BS.%26aulast%3DClarke%26aufirst%3DP.%2BA.%26aulast%3DDepledge%26aufirst%3DP.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DHayes%26aufirst%3DA.%26aulast%3DHancox%26aufirst%3DT.%2BC.%26aulast%3DKugendradas%26aufirst%3DA.%26aulast%3DLensun%26aufirst%3DL.%26aulast%3DMoore%26aufirst%3DP.%26aulast%3DOlivero%26aufirst%3DA.%2BG.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DPergl-Wilson%26aufirst%3DG.%2BH.%26aulast%3DRaynaud%26aufirst%3DF.%2BI.%26aulast%3DRobson%26aufirst%3DA.%26aulast%3DSaghir%26aufirst%3DN.%26aulast%3DSalphati%26aufirst%3DL.%26aulast%3DSohal%26aufirst%3DS.%26aulast%3DUltsch%26aufirst%3DM.%2BH.%26aulast%3DValenti%26aufirst%3DM.%26aulast%3DWallweber%26aufirst%3DH.%2BJ.%26aulast%3DWan%26aufirst%3DN.%2BC.%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DZhyvoloup%26aufirst%3DA.%26aulast%3DZvelebil%26aufirst%3DM.%2BJ.%26aulast%3DShuttleworth%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520identification%2520of%25202-%25281h-indazol-4-yl%2529-6-%25284-methanesulfonyl-piperazin-1-ylmethyl%2529-4-morpholin-4-yl-t%2520hieno%255B3%252C2-d%255Dpyrimidine%2520%2528GDC-0941%2529%2520as%2520a%2520potent%252C%2520selective%252C%2520orally%2520bioavailable%2520inhibitor%2520of%2520class%2520I%2520PI3%2520kinase%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5522%26epage%3D5532%26doi%3D10.1021%2Fjm800295d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vukovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huggins, D. J.</span></span> <span> </span><span class="NLM_article-title">Quantitative metrics for drug-target ligandability</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1258</span>– <span class="NLM_lpage">1266</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.02.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.drudis.2018.02.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=29522887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVGrtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1258-1266&author=S.+Vukovicauthor=D.+J.+Huggins&title=Quantitative+metrics+for+drug-target+ligandability&doi=10.1016%2Fj.drudis.2018.02.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative metrics for drug-target ligandability</span></div><div class="casAuthors">Vukovic, Sinisa; Huggins, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1258-1266</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Ligandability is a prerequisite for druggability and is a much easier concept to understand, model and predict because it does not depend on the complex pharmacodynamic and pharmacokinetic mechanisms in the human body.  In this review, we consider a metric for quantifying ligandability from exptl. data.  We discuss ligandability in terms of the balance between effort and reward.  The metric is evaluated for a std. set of well-studied drug targets - some traditionally considered to be ligandable and some regarded as difficult.  We suggest that this metric should be used to systematically improve computational predictions of ligandability, which can then be applied to novel drug targets to predict their tractability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh-DRJxKcFEbVg90H21EOLACvtfcHk0lhwG3EMdvoq4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVGrtbk%253D&md5=3f7d1b09c5dda9e0fa73c83faa655b41</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.02.015%26sid%3Dliteratum%253Aachs%26aulast%3DVukovic%26aufirst%3DS.%26aulast%3DHuggins%26aufirst%3DD.%2BJ.%26atitle%3DQuantitative%2520metrics%2520for%2520drug-target%2520ligandability%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D1258%26epage%3D1266%26doi%3D10.1016%2Fj.drudis.2018.02.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. H.</span></span> <span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">121</span>, <span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keseruauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lh28pQWV8nQSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wehling, M.</span></span> <span> </span><span class="NLM_article-title">Assessing the translatability of drug projects: What needs to be scored to predict success?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">541</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1038/nrd2898</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrd2898" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=19543224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVylu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=541-546&author=M.+Wehling&title=Assessing+the+translatability+of+drug+projects%3A+What+needs+to+be+scored+to+predict+success%3F&doi=10.1038%2Fnrd2898"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the translatability of drug projects: What needs to be scored to predict success?</span></div><div class="casAuthors">Wehling, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">541-546</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drug development projects have high attrition rates, often because efficacy and safety issues have not been foreseen.  More effective prediction of 'translational success' could therefore have a key role in addressing the widely acknowledged problems with weak drug development pipelines.  Here, I discuss how a scoring system to systematically assess key determinants of translational success, such as biomarkers and animal and human data, could help identify deficiencies and potential improvements, and increase the reliability of portfolio risk ests.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWMac7ZHMqTrVg90H21EOLACvtfcHk0lh28pQWV8nQSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVylu74%253D&md5=41553f723f029a6d7077cffd70c3fe8b</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1038%2Fnrd2898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2898%26sid%3Dliteratum%253Aachs%26aulast%3DWehling%26aufirst%3DM.%26atitle%3DAssessing%2520the%2520translatability%2520of%2520drug%2520projects%253A%2520What%2520needs%2520to%2520be%2520scored%2520to%2520predict%2520success%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D541%26epage%3D546%26doi%3D10.1038%2Fnrd2898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexandrov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I.</span></span> <span> </span><span class="NLM_article-title">PDP: Protein domain parser</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">430</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btg006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fbioinformatics%2Fbtg006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=12584135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3sXit1ygtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2003&pages=429-430&author=N.+Alexandrovauthor=I.+Shindyalov&title=PDP%3A+Protein+domain+parser&doi=10.1093%2Fbioinformatics%2Fbtg006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">PDP: protein domain parser</span></div><div class="casAuthors">Alexandrov, Nickolai; Shindyalov, Ilya</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">429-430</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The program PDP (Protein Domain Parser) has been developed for the automatic identification of domains in protein three-dimensional structures.  The performance of the program was assessed by three different benchmarks: (i) comparison with the expert-curated SCOP database of structural domains; (ii) by comparison with a collection of manual domain assignments; and (iii) by comparison with a set of 55 proteins, frequently used as a benchmark for automatic domain assignment.  In all of these benchmarks, PDP identified domains correctly in more than 80% of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq0_EOU1X10LVg90H21EOLACvtfcHk0lh28pQWV8nQSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXit1ygtb8%253D&md5=c1cdc1177c5a24694f4ba0b99e84d7e8</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtg006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtg006%26sid%3Dliteratum%253Aachs%26aulast%3DAlexandrov%26aufirst%3DN.%26aulast%3DShindyalov%26aufirst%3DI.%26atitle%3DPDP%253A%2520Protein%2520domain%2520parser%26jtitle%3DBioinformatics%26date%3D2003%26volume%3D19%26spage%3D429%26epage%3D430%26doi%3D10.1093%2Fbioinformatics%2Fbtg006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span> <i>The PyMOL Molecular
Graphics System</i>, version <span class="NLM_edition">1.8</span>; <span class="NLM_publisher-name">Schrodinger, LLC</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular%0AGraphics+System%2C+version+1.8%3B+Schrodinger%2C+LLC%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%250AGraphics%2520System%26pub%3DSchrodinger%252C%2520LLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 28 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Edward Price, J. Cory Kalvass, David DeGoey, Balakrishna Hosmane, Stella Doktor, <span class="NLM_string-name hlFld-ContribAuthor">Kelly Desino</span>. </span><span class="cited-content_cbyCitation_article-title">Global Analysis of Models for Predicting Human Absorption: QSAR, In Vitro, and Preclinical Models. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 9389-9403. <a href="https://doi.org/10.1021/acs.jmedchem.1c00669" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00669</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00669%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DGlobal%252BAnalysis%252Bof%252BModels%252Bfor%252BPredicting%252BHuman%252BAbsorption%25253A%252BQSAR%25252C%252BIn%252BVitro%25252C%252Band%252BPreclinical%252BModels%26aulast%3DPrice%26aufirst%3DEdward%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D12042021%26date%3D21062021%26volume%3D64%26issue%3D13%26spage%3D9389%26epage%3D9403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">SahaIshika</span><span class="NLM_role">Graduate Student Researcher</span><a class="email" href="/cdn-cgi/l/email-protection#3b52485352505a485a535a7b5c154e58575a155e5f4e"><span class="nobrWithWbr"><span class="__cf_email__" data-cfemail="cba2b8a3a2a0aab8aaa3aa8bace5">[email protected]</span><wbr></wbr>ucla.<wbr></wbr>edu</span></a><span class="hlFld-ContribAuthor ">HarranPatrick G.</span><span class="NLM_role">D.J. & J.M. Cram Chair in Organic Chemistry</span><a class="email" href="/cdn-cgi/l/email-protection#731b120101121d33101b161e5d06101f125d161706"><span class="nobrWithWbr"><span class="__cf_email__" data-cfemail="ea828b98988b84aa89828f87c4">[email protected]</span><wbr></wbr>ucla.<wbr></wbr>edu</span></a>Dr. Jonathan Bohmann, Department of Pharmaceuticals and Bioengineering, Southwest Research Institute, <div class="author-comment"><p class="first last">A great introduction to the topic of virtual screening, which is often seen as a black box.  It does a good job weaving a lot of material together. The molecular simulation community has needed a book like this for a long time. It is honest about the pitfalls of VS while pointing to reasons why VS is needed and the positive impacts on early drug discovery programs, where resources are generally scarce.</p></div><span class="NLM_string-name hlFld-ContribAuthor">Ryan Gumpper, Postdoctoral Researcher, University of North Carolina at Chapel Hill</span><div class="author-comment"><p class="first last">I would recommend this work to young graduate students starting in the field and anyone who is interested in expanding their knowledge in the field.</p></div>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening for Chemists. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1021/acsinfocus.7e5001" title="DOI URL">https://doi.org/10.1021/acsinfocus.7e5001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfocus.7e5001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfocus.7e5001%26sid%3Dliteratum%253Aachs%26atitle%3DVirtual%252BScreening%252Bfor%252BChemists%26aulast%3DSaha%26aufirst%3DIshika%26date%3D2021%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul D. Leeson, A. Patricia Bento, Anna Gaulton, Anne Hersey, Emma J. Manners, Chris J. Radoux, <span class="NLM_string-name hlFld-ContribAuthor">Andrew R. Leach</span>. </span><span class="cited-content_cbyCitation_article-title">Target-Based Evaluation of “Drug-Like” Properties and Ligand Efficiencies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7210-7230. <a href="https://doi.org/10.1021/acs.jmedchem.1c00416" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00416</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00416%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTarget-Based%252BEvaluation%252Bof%252B%2525E2%252580%25259CDrug-Like%2525E2%252580%25259D%252BProperties%252Band%252BLigand%252BEfficiencies%26aulast%3DLeeson%26aufirst%3DPaul%2BD.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08032021%26date%3D13052021%26volume%3D64%26issue%3D11%26spage%3D7210%26epage%3D7230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrew W. Robertson, Jorge Sandoval, Osama G. Mohamed, Yihao Zhuang, Erin E. Gallagher, Jennifer Schmidt, Lisa Caratelli, Arya Menon, Pamela J. Schultz, Rachel M. Torrez, Catherine L. Hay, Bailey A. Bell, Paul A. Price, Amanda L. Garner, <span class="NLM_string-name hlFld-ContribAuthor">Ashootosh Tripathi</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Surfactins as Inhibitors of MicroRNA Processing Using Cat-ELCCA. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (6)
                                     , 878-886. <a href="https://doi.org/10.1021/acsmedchemlett.1c00046" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00046</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00046%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BSurfactins%252Bas%252BInhibitors%252Bof%252BMicroRNA%252BProcessing%252BUsing%252BCat-ELCCA%26aulast%3DRobertson%26aufirst%3DAndrew%2BW.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D22012021%26date%3D22032021%26date%3D02042021%26volume%3D12%26issue%3D6%26spage%3D878%26epage%3D886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Begnini, Vasanthanathan Poongavanam, Yoseph Atilaw, Mate Erdelyi, Stefan Schiesser, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Cell Permeability of Isomeric Macrocycles: Predictions and NMR Studies. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (6)
                                     , 983-990. <a href="https://doi.org/10.1021/acsmedchemlett.1c00126" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00126</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00126%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCell%252BPermeability%252Bof%252BIsomeric%252BMacrocycles%25253A%252BPredictions%252Band%252BNMR%252BStudies%26aulast%3DBegnini%26aufirst%3DFabio%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D02032021%26date%3D13052021%26date%3D18052021%26volume%3D12%26issue%3D6%26spage%3D983%26epage%3D990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dean G. Brown, <span class="NLM_string-name hlFld-ContribAuthor">Heike J. Wobst</span>. </span><span class="cited-content_cbyCitation_article-title">A Decade of FDA-Approved Drugs (2010–2019): Trends and Future Directions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2312-2338. <a href="https://doi.org/10.1021/acs.jmedchem.0c01516" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01516</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01516%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BDecade%252Bof%252BFDA-Approved%252BDrugs%252B%2525282010%2525E2%252580%2525932019%252529%25253A%252BTrends%252Band%252BFuture%252BDirections%26aulast%3DBrown%26aufirst%3DDean%2BG.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D30082020%26date%3D22022021%26volume%3D64%26issue%3D5%26spage%3D2312%26epage%3D2338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Begnini, Vasanthanathan Poongavanam, Björn Over, Marie Castaldo, Stefan Geschwindner, Patrik Johansson, Mohit Tyagi, Christian Tyrchan, Lisa Wissler, Peter Sjö, Stefan Schiesser, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Mining Natural Products for Macrocycles to Drug Difficult Targets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (2)
                                     , 1054-1072. <a href="https://doi.org/10.1021/acs.jmedchem.0c01569" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01569</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01569%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMining%252BNatural%252BProducts%252Bfor%252BMacrocycles%252Bto%252BDrug%252BDifficult%252BTargets%26aulast%3DBegnini%26aufirst%3DFabio%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D08092020%26date%3D18122020%26volume%3D64%26issue%3D2%26spage%3D1054%26epage%3D1072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Giulia Caron, Jan Kihlberg, Gilles Goetz, Ekaterina Ratkova, Vasanthanathan Poongavanam, <span class="NLM_string-name hlFld-ContribAuthor">Giuseppe Ermondi</span>. </span><span class="cited-content_cbyCitation_article-title">Steering New Drug Discovery Campaigns: Permeability, Solubility, and Physicochemical Properties in the bRo5 Chemical Space. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     , 13-23. <a href="https://doi.org/10.1021/acsmedchemlett.0c00581" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00581%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSteering%252BNew%252BDrug%252BDiscovery%252BCampaigns%25253A%252BPermeability%25252C%252BSolubility%25252C%252Band%252BPhysicochemical%252BProperties%252Bin%252Bthe%252BbRo5%252BChemical%252BSpace%26aulast%3DCaron%26aufirst%3DGiulia%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D04112020%26date%3D15122020%26date%3D05012021%26volume%3D12%26issue%3D1%26spage%3D13%26epage%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marta Serafini, Sarah Cargnin, Alberto Massarotti, Tracey Pirali, <span class="NLM_string-name hlFld-ContribAuthor">Armando A. Genazzani</span>. </span><span class="cited-content_cbyCitation_article-title">Essential Medicinal Chemistry of Essential Medicines. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (18)
                                     , 10170-10187. <a href="https://doi.org/10.1021/acs.jmedchem.0c00415" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00415</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00415%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEssential%252BMedicinal%252BChemistry%252Bof%252BEssential%252BMedicines%26aulast%3DSerafini%26aufirst%3DMarta%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11032020%26date%3D12052020%26date%3D30042020%26volume%3D63%26issue%3D18%26spage%3D10170%26epage%3D10187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher P. Tinworth, <span class="NLM_string-name hlFld-ContribAuthor">Robert J. Young</span>. </span><span class="cited-content_cbyCitation_article-title">Facts, Patterns, and Principles in Drug Discovery: Appraising the Rule of 5 with Measured Physicochemical Data. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (18)
                                     , 10091-10108. <a href="https://doi.org/10.1021/acs.jmedchem.9b01596" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01596</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01596%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFacts%25252C%252BPatterns%25252C%252Band%252BPrinciples%252Bin%252BDrug%252BDiscovery%25253A%252BAppraising%252Bthe%252BRule%252Bof%252B5%252Bwith%252BMeasured%252BPhysicochemical%252BData%26aulast%3DTinworth%26aufirst%3DChristopher%2BP.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D25092019%26date%3D11052020%26date%3D23042020%26volume%3D63%26issue%3D18%26spage%3D10091%26epage%3D10108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maria-Jesus Blanco, <span class="NLM_string-name hlFld-ContribAuthor">Kevin M. Gardinier</span>. </span><span class="cited-content_cbyCitation_article-title">New Chemical Modalities and Strategic Thinking in Early Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (3)
                                     , 228-231. <a href="https://doi.org/10.1021/acsmedchemlett.9b00582" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00582</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00582&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00582%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DNew%252BChemical%252BModalities%252Band%252BStrategic%252BThinking%252Bin%252BEarly%252BDrug%252BDiscovery%26aulast%3DBlanco%26aufirst%3DMaria-Jesus%26date%3D2020%26date%3D2020%26date%3D14012020%26volume%3D11%26issue%3D3%26spage%3D228%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lea  Seep</span>, <span class="hlFld-ContribAuthor ">Anne  Bonin</span>, <span class="hlFld-ContribAuthor ">Katharina  Meier</span>, <span class="hlFld-ContribAuthor ">Holger  Diedam</span>, <span class="hlFld-ContribAuthor ">Andreas H.  Göller</span>. </span><span class="cited-content_cbyCitation_article-title">Ensemble completeness in conformer sampling: the case of small macrocycles. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2021,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-021-00524-0" title="DOI URL">https://doi.org/10.1186/s13321-021-00524-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-021-00524-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-021-00524-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DEnsemble%252Bcompleteness%252Bin%252Bconformer%252Bsampling%25253A%252Bthe%252Bcase%252Bof%252Bsmall%252Bmacrocycles%26aulast%3DSeep%26aufirst%3DLea%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiamin  Zheng</span>, <span class="hlFld-ContribAuthor ">Dongdong  Chen</span>, <span class="hlFld-ContribAuthor ">Jie  Xu</span>, <span class="hlFld-ContribAuthor ">Xiao  Ding</span>, <span class="hlFld-ContribAuthor ">Yao  Wu</span>, <span class="hlFld-ContribAuthor ">Hong C.  Shen</span>, <span class="hlFld-ContribAuthor ">Xuefei  Tan</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule approaches to treat autoimmune and inflammatory diseases (Part III): Targeting cytokines and cytokine receptor complexes. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>48 </em>, 128229. <a href="https://doi.org/10.1016/j.bmcl.2021.128229" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128229</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128229%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSmall%252Bmolecule%252Bapproaches%252Bto%252Btreat%252Bautoimmune%252Band%252Binflammatory%252Bdiseases%252B%252528Part%252BIII%252529%25253A%252BTargeting%252Bcytokines%252Band%252Bcytokine%252Breceptor%252Bcomplexes%26aulast%3DZheng%26aufirst%3DJiamin%26date%3D2021%26volume%3D48%26spage%3D128229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isaline F.S.F.  Castan</span>, <span class="hlFld-ContribAuthor ">Jessica S.  Graham</span>, <span class="hlFld-ContribAuthor ">Catherine L.A.  Salvini</span>, <span class="hlFld-ContribAuthor ">Harriet A.  Stanway-Gordon</span>, <span class="hlFld-ContribAuthor ">Michael J.  Waring</span>. </span><span class="cited-content_cbyCitation_article-title">On the design of lead-like DNA-encoded chemical libraries. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>43 </em>, 116273. <a href="https://doi.org/10.1016/j.bmc.2021.116273" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116273</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116273%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DOn%252Bthe%252Bdesign%252Bof%252Blead-like%252BDNA-encoded%252Bchemical%252Blibraries%26aulast%3DCastan%26aufirst%3DIsaline%2BF.S.F.%26date%3D2021%26volume%3D43%26spage%3D116273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dirk  Menche</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Synthesis of Simplified Polyketide Analogs: New Modalities beyond the Rule of 5. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (13)
                                     , 2068-2074. <a href="https://doi.org/10.1002/cmdc.202100150" title="DOI URL">https://doi.org/10.1002/cmdc.202100150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202100150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202100150%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDesign%252Band%252BSynthesis%252Bof%252BSimplified%252BPolyketide%252BAnalogs%25253A%252BNew%252BModalities%252Bbeyond%252Bthe%252BRule%252Bof%252B5%26aulast%3DMenche%26aufirst%3DDirk%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D13%26spage%3D2068%26epage%3D2074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yvonne A.  Nagel</span>, <span class="hlFld-ContribAuthor ">Adrian  Britschgi</span>, <span class="hlFld-ContribAuthor ">Antonio  Ricci</span>. </span><span class="cited-content_cbyCitation_article-title">From Degraders to Molecular Glues: New Ways of Breaking Down Disease‐Associated Proteins. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 47-85. <a href="https://doi.org/10.1002/9783527826872.ch2" title="DOI URL">https://doi.org/10.1002/9783527826872.ch2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527826872.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527826872.ch2%26sid%3Dliteratum%253Aachs%26atitle%3DFrom%252BDegraders%252Bto%252BMolecular%252BGlues%25253A%252BNew%252BWays%252Bof%252BBreaking%252BDown%252BDisease%2525E2%252580%252590Associated%252BProteins%26aulast%3DNagel%26aufirst%3DYvonne%2BA.%26date%3D2021%26date%3D2021%26spage%3D47%26epage%3D85%26pub%3DWiley%26atitle%3DSuccessful%252BDrug%252BDiscovery%26aulast%3DFischer%26aufirst%3DJ%25C3%25A1nos%26date%3D2021%26date%3D2021%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abhishek  Srivastava</span>, <span class="hlFld-ContribAuthor ">Andy  Pike</span>, <span class="hlFld-ContribAuthor ">Beth  Williamson</span>, <span class="hlFld-ContribAuthor ">Katherine  Fenner</span>. </span><span class="cited-content_cbyCitation_article-title">A Novel Method for Preventing Non-specific Binding in Equilibrium Dialysis Assays Using Solutol® as an Additive. </span><span class="cited-content_cbyCitation_journal-name">Journal of Pharmaceutical Sciences</span><span> <strong>2021,</strong> <em>110 </em>
                                    (3)
                                     , 1412-1417. <a href="https://doi.org/10.1016/j.xphs.2020.11.018" title="DOI URL">https://doi.org/10.1016/j.xphs.2020.11.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.xphs.2020.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.xphs.2020.11.018%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DA%252BNovel%252BMethod%252Bfor%252BPreventing%252BNon-specific%252BBinding%252Bin%252BEquilibrium%252BDialysis%252BAssays%252BUsing%252BSolutol%2525C2%2525AE%252Bas%252Ban%252BAdditive%26aulast%3DSrivastava%26aufirst%3DAbhishek%26date%3D2021%26volume%3D110%26issue%3D3%26spage%3D1412%26epage%3D1417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pedro  Brandão</span>, <span class="hlFld-ContribAuthor ">Carolina  Marques</span>, <span class="hlFld-ContribAuthor ">Anthony J.  Burke</span>, <span class="hlFld-ContribAuthor ">Marta  Pineiro</span>. </span><span class="cited-content_cbyCitation_article-title">The application of isatin-based multicomponent-reactions in the quest for new bioactive and druglike molecules. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>211 </em>, 113102. <a href="https://doi.org/10.1016/j.ejmech.2020.113102" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.113102</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.113102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.113102%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bapplication%252Bof%252Bisatin-based%252Bmulticomponent-reactions%252Bin%252Bthe%252Bquest%252Bfor%252Bnew%252Bbioactive%252Band%252Bdruglike%252Bmolecules%26aulast%3DBrand%25C3%25A3o%26aufirst%3DPedro%26date%3D2021%26volume%3D211%26spage%3D113102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohammed Abu  Tayab</span>, <span class="hlFld-ContribAuthor ">Kazi Ashfak Ahmed  Chowdhury</span>, <span class="hlFld-ContribAuthor ">Md.  Jabed</span>, <span class="hlFld-ContribAuthor ">Syed  Mohammed Tareq</span>, <span class="hlFld-ContribAuthor ">A. T. M. Mostafa  Kamal</span>, <span class="hlFld-ContribAuthor ">Mohammad Nazmul  Islam</span>, <span class="hlFld-ContribAuthor ">A. M. Kafil  Uddin</span>, <span class="hlFld-ContribAuthor ">Mohammad Adil  Hossain</span>, <span class="hlFld-ContribAuthor ">Talha Bin  Emran</span>, <span class="hlFld-ContribAuthor ">Jesus  Simal-Gandara</span>. </span><span class="cited-content_cbyCitation_article-title">Antioxidant-Rich Woodfordia fruticosa Leaf Extract Alleviates Depressive-Like Behaviors and Impede Hyperglycemia. </span><span class="cited-content_cbyCitation_journal-name">Plants</span><span> <strong>2021,</strong> <em>10 </em>
                                    (2)
                                     , 287. <a href="https://doi.org/10.3390/plants10020287" title="DOI URL">https://doi.org/10.3390/plants10020287</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/plants10020287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fplants10020287%26sid%3Dliteratum%253Aachs%26jtitle%3DPlants%26atitle%3DAntioxidant-Rich%252BWoodfordia%252Bfruticosa%252BLeaf%252BExtract%252BAlleviates%252BDepressive-Like%252BBehaviors%252Band%252BImpede%252BHyperglycemia%26aulast%3DTayab%26aufirst%3DMohammed%2BAbu%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D2%26spage%3D287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter 't  Hart</span>, <span class="hlFld-ContribAuthor ">Pascal  Hommen</span>, <span class="hlFld-ContribAuthor ">Anaïs  Noisier</span>, <span class="hlFld-ContribAuthor ">Adrian  Krzyzanowski</span>, <span class="hlFld-ContribAuthor ">Darijan  Schüler</span>, <span class="hlFld-ContribAuthor ">Arthur T.  Porfetye</span>, <span class="hlFld-ContribAuthor ">Mohammad  Akbarzadeh</span>, <span class="hlFld-ContribAuthor ">Ingrid R.  Vetter</span>, <span class="hlFld-ContribAuthor ">Hélène  Adihou</span>, <span class="hlFld-ContribAuthor ">Herbert  Waldmann</span>. </span><span class="cited-content_cbyCitation_article-title">Structure Based Design of Bicyclic Peptide Inhibitors of RbAp48. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (4)
                                     , 1841-1848. <a href="https://doi.org/10.1002/ange.202009749" title="DOI URL">https://doi.org/10.1002/ange.202009749</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202009749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202009749%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DStructure%252BBased%252BDesign%252Bof%252BBicyclic%252BPeptide%252BInhibitors%252Bof%252BRbAp48%26aulast%3DHart%26aufirst%3DPeter%2B%2527t%26date%3D2021%26date%3D2020%26volume%3D133%26issue%3D4%26spage%3D1841%26epage%3D1848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter 't  Hart</span>, <span class="hlFld-ContribAuthor ">Pascal  Hommen</span>, <span class="hlFld-ContribAuthor ">Anaïs  Noisier</span>, <span class="hlFld-ContribAuthor ">Adrian  Krzyzanowski</span>, <span class="hlFld-ContribAuthor ">Darijan  Schüler</span>, <span class="hlFld-ContribAuthor ">Arthur T.  Porfetye</span>, <span class="hlFld-ContribAuthor ">Mohammad  Akbarzadeh</span>, <span class="hlFld-ContribAuthor ">Ingrid R.  Vetter</span>, <span class="hlFld-ContribAuthor ">Hélène  Adihou</span>, <span class="hlFld-ContribAuthor ">Herbert  Waldmann</span>. </span><span class="cited-content_cbyCitation_article-title">Structure Based Design of Bicyclic Peptide Inhibitors of RbAp48. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (4)
                                     , 1813-1820. <a href="https://doi.org/10.1002/anie.202009749" title="DOI URL">https://doi.org/10.1002/anie.202009749</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202009749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202009749%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DStructure%252BBased%252BDesign%252Bof%252BBicyclic%252BPeptide%252BInhibitors%252Bof%252BRbAp48%26aulast%3DHart%26aufirst%3DPeter%2B%2527t%26date%3D2021%26date%3D2020%26volume%3D60%26issue%3D4%26spage%3D1813%26epage%3D1820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alan  Talevi</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico ADME: Rule-Based Systems. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-7. <a href="https://doi.org/10.1007/978-3-030-51519-5_148-1" title="DOI URL">https://doi.org/10.1007/978-3-030-51519-5_148-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-51519-5_148-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-51519-5_148-1%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BADME%25253A%252BRule-Based%252BSystems%26aulast%3DTalevi%26aufirst%3DAlan%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D7%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DThe%252BADME%252BEncyclopedia%26date%3D2021%26volume%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alice  Capecchi</span>, <span class="hlFld-ContribAuthor ">Daniel  Probst</span>, <span class="hlFld-ContribAuthor ">Jean-Louis  Reymond</span>. </span><span class="cited-content_cbyCitation_article-title">One molecular fingerprint to rule them all: drugs, biomolecules, and the metabolome. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2020,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-020-00445-4" title="DOI URL">https://doi.org/10.1186/s13321-020-00445-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-020-00445-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-020-00445-4%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DOne%252Bmolecular%252Bfingerprint%252Bto%252Brule%252Bthem%252Ball%25253A%252Bdrugs%25252C%252Bbiomolecules%25252C%252Band%252Bthe%252Bmetabolome%26aulast%3DCapecchi%26aufirst%3DAlice%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert I.  Troup</span>, <span class="hlFld-ContribAuthor ">Charlene  Fallan</span>, <span class="hlFld-ContribAuthor ">Matthias G. J.  Baud</span>. </span><span class="cited-content_cbyCitation_article-title">Current strategies for the design of PROTAC linkers: a critical review. </span><span class="cited-content_cbyCitation_journal-name">Exploration of Targeted Anti-tumor Therapy</span><span> <strong>2020,</strong> <em>1 </em>
                                    (5)
                                     <a href="https://doi.org/10.37349/etat.2020.00018" title="DOI URL">https://doi.org/10.37349/etat.2020.00018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.37349/etat.2020.00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.37349%2Fetat.2020.00018%26sid%3Dliteratum%253Aachs%26jtitle%3DExploration%2520of%2520Targeted%2520Anti-tumor%2520Therapy%26atitle%3DCurrent%252Bstrategies%252Bfor%252Bthe%252Bdesign%252Bof%252BPROTAC%252Blinkers%25253A%252Ba%252Bcritical%252Breview%26aulast%3DTroup%26aufirst%3DRobert%2BI.%26date%3D2020%26date%3D2020%26volume%3D1%26issue%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas H.  Göller</span>, <span class="hlFld-ContribAuthor ">Lara  Kuhnke</span>, <span class="hlFld-ContribAuthor ">Floriane  Montanari</span>, <span class="hlFld-ContribAuthor ">Anne  Bonin</span>, <span class="hlFld-ContribAuthor ">Sebastian  Schneckener</span>, <span class="hlFld-ContribAuthor ">Antonius  ter Laak</span>, <span class="hlFld-ContribAuthor ">Jörg  Wichard</span>, <span class="hlFld-ContribAuthor ">Mario  Lobell</span>, <span class="hlFld-ContribAuthor ">Alexander  Hillisch</span>. </span><span class="cited-content_cbyCitation_article-title">Bayer’s in silico ADMET platform: a journey of machine learning over the past two decades. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (9)
                                     , 1702-1709. <a href="https://doi.org/10.1016/j.drudis.2020.07.001" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.07.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.07.001%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DBayer%2525E2%252580%252599s%252Bin%252Bsilico%252BADMET%252Bplatform%25253A%252Ba%252Bjourney%252Bof%252Bmachine%252Blearning%252Bover%252Bthe%252Bpast%252Btwo%252Bdecades%26aulast%3DG%25C3%25B6ller%26aufirst%3DAndreas%2BH.%26date%3D2020%26volume%3D25%26issue%3D9%26spage%3D1702%26epage%3D1709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jintong  Du</span>, <span class="hlFld-ContribAuthor ">Lulu  Liu</span>, <span class="hlFld-ContribAuthor ">Bo  Liu</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Xuben  Hou</span>, <span class="hlFld-ContribAuthor ">Jinming  Yu</span>, <span class="hlFld-ContribAuthor ">Hao  Fang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based virtual screening, biological evaluation and biophysical study of novel Mcl-1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (14)
                                     , 1293-1304. <a href="https://doi.org/10.4155/fmc-2020-0114" title="DOI URL">https://doi.org/10.4155/fmc-2020-0114</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0114%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bvirtual%252Bscreening%25252C%252Bbiological%252Bevaluation%252Band%252Bbiophysical%252Bstudy%252Bof%252Bnovel%252BMcl-1%252Binhibitors%26aulast%3DDu%26aufirst%3DJintong%26date%3D2020%26volume%3D12%26issue%3D14%26spage%3D1293%26epage%3D1304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kumiko  Sakai-Kato</span>, <span class="hlFld-ContribAuthor ">Kohki  Yoshida</span>. </span><span class="cited-content_cbyCitation_article-title">The elution behavior of cyclosporine congeners in a developed HPLC system reflects the lipophilicity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Pharmaceutical and Biomedical Analysis</span><span> <strong>2020,</strong> <em>180 </em>, 113064. <a href="https://doi.org/10.1016/j.jpba.2019.113064" title="DOI URL">https://doi.org/10.1016/j.jpba.2019.113064</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jpba.2019.113064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jpba.2019.113064%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Pharmaceutical%2520and%2520Biomedical%2520Analysis%26atitle%3DThe%252Belution%252Bbehavior%252Bof%252Bcyclosporine%252Bcongeners%252Bin%252Ba%252Bdeveloped%252BHPLC%252Bsystem%252Breflects%252Bthe%252Blipophilicity%26aulast%3DSakai-Kato%26aufirst%3DKumiko%26date%3D2020%26volume%3D180%26spage%3D113064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gjorgji  Atanasov</span>, <span class="hlFld-ContribAuthor ">Iliyan N.  Kolev</span>, <span class="hlFld-ContribAuthor ">Ognyan  Petrov</span>, <span class="hlFld-ContribAuthor ">Margarita D.  Apostolova</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of PLGA-PEG-PLGA Polymer Nano-Micelles – Carriers of Combretastatin-Like Antitumor Agent 16Z. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 449-458. <a href="https://doi.org/10.1007/978-94-024-2018-0_36" title="DOI URL">https://doi.org/10.1007/978-94-024-2018-0_36</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-94-024-2018-0_36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-94-024-2018-0_36%26sid%3Dliteratum%253Aachs%26atitle%3DSynthesis%252Bof%252BPLGA-PEG-PLGA%252BPolymer%252BNano-Micelles%252B%2525E2%252580%252593%252BCarriers%252Bof%252BCombretastatin-Like%252BAntitumor%252BAgent%252B16Z%26aulast%3DAtanasov%26aufirst%3DGjorgji%26date%3D2020%26date%3D2020%26spage%3D449%26epage%3D458%26pub%3DSpringer%2520Netherlands%26atitle%3DNanoscience%252Band%252BNanotechnology%252Bin%252BSecurity%252Band%252BProtection%252Bagainst%252BCBRN%252BThreats%26aulast%3DPetkov%26aufirst%3DPlamen%26date%3D2020%26volume%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0009.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Complex versus simple binding site hot spot structure, determined by FTMap. The hot spots are labeled according to rank (starting at 0) and colored according to the standard FTMap output (<a href="http://ftmap.bu.edu" class="extLink">http://ftmap.bu.edu</a>). The color of each hot spot ranked 0–7 is shown explicitly at the bottom of the figure. (a) A complex hot spot structure consists of four or more hot spots. Heat shock protein 90 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FWZ">2FWZ</a>) has the hot spots 0(25), 1(21), 2(16), 4(6), 5(6), and 6(5). The hot spots are labeled on the left, and the compound PU-H71 is overlaid on the right. b) A simple hot spot structure has three or fewer hot spots. HMG-CoA reductase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1HWK">1HWK</a>) has the hot spots 0(13), 1(13), and 6(6). The hot spots are labeled on the left, and atorvastatin is overlaid on the right.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Examples of characteristic structure-based ligand profiles for complex I, complex II, and simple protein targets that bind eRo5/bRo5 drugs and clinical candidates. Examples of complex I targets shown are the heat shock protein 90, thrombin, renin, and PPAR-γ. Examples of complex II targets shown are EGFR, HGFR, ABL1, and ALK. Examples of targets with simple hot spot structure are kinesin Eg5, soluble acetylcholine receptor, HMG-CoA reductase, and HCV NS5b. Only ligands that bind to the above proteins with known structure and binding affinity are shown (see <a class="ref internalNav" href="#sec9" aria-label="Methods">Methods</a>). Plots with ligands in blue (left) show the pBA [pBA = −log(<i>K</i><sub>D</sub> or <i>K</i><sub>I</sub> or IC<sub>50</sub>)] versus molecular weight, and plots with ligands in green (right) show pBA versus total number of probe clusters.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. FTMap determined hot spot structure of complex I targets with example ligands bound. Blue ligands represent the eRo5/bRo5 drug/clinical candidate identified by Doak et al., whereas pink ligands represent smaller compounds. FTMap hot spots are colored according to standard output, shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. (a) Hot spot structure of thrombin (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DWC">1DWC</a>). Argatroban (1DWC:MIT, MW = 509.64 Da, <i>K</i><sub>I</sub> = 39 nM, pBA = 7.41) and GR157368 (1QHR:157, MW = 226.31 Da, IC<sub>50</sub> = 130 nM, pBA = 6.89) are shown in blue and pink, respectively. (b) Hot spot structure of renin (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V0Z">2V0Z</a>). Aliskiren (2V0Z:C41, MW = 551.76 Da, <i>K</i><sub>I</sub> = 0.6 nM, pBA = 9.22) and <i>N</i>-piperidin-3-ylpyrimidine-5-carboxamide (5SZ9:74Y, MW = 313.39 Da, IC<sub>50</sub> = 38 μM, pBA = 4.42) are shown in blue and pink, respectively. (c) Hot spot structure of E3 ubiquitin-protein ligase XIAP (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KMP">4KMP</a>). Birinapant (4KMP:GT6, MW = 806.94 Da, <i>K</i><sub>D</sub> = 45 nM, pBA = 7.35) and (2JK7:BI6, MW = 486.61 Da, <i>K</i><sub>I</sub> = 67 nM, pBA = 7.17) are shown in blue and pink, respectively. (d) Hot spot structure of MAP kinase p38 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YIS">2YIS</a>). PF-03715455 (2YIS:YIS MW = 700.27 Da, IC<sub>50</sub> = 1.7 nM, pBA = 8.77) and 3P7B:P7B (MW = 464.58 Da, IC<sub>50</sub> = 18 nM, pBA = 7.74) are shown in blue and pink, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. HIV-1 protease binding hot spots. (a) FTMap hot spots of HIV-1 protease (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OXC">3OXC</a>): 0(21), 1(18), 2(18), 3(13), 2(111) 5(8), 6(5). (b) Hot spot structure of HIV-1 protease (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OXC">3OXC</a>). Saquinavir (3EL4:ROC, MW = 670.84 Da, <i>K</i><sub>D</sub> = 67.4 nM, pBA = 7.17) and XK216 (1HWR:216, MW = 406.52, <i>K</i><sub>I</sub> = 4.6 nM, pBA = 8.34) are shown in blue and pink, respectively. FTMap hot spots are colored according to standard output, shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. (c) Three plots showing data on ligands that bind HIV-1 protease with known structure and binding affinity. Ligands are distinguished as Ro5, eRo5, and bRo5 by the colors red, blue, and green, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. FTMap hot spot structure of kinase DFG-in versus DFG-out conformations. Hot spots in pink represent the FTMap results for the DFG-in conformation, whereas blue represents DFG-out hot spots. The protein cartoon shows the DFG-out conformation. (a) Hot spots of tyrosine protein kinase ABL1 in DFG-in (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TWP">4TWP</a>) and DFG-out (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS9">3CS9</a>) conformations. (b) Hot spots of anaplastic lymphoma kinase in DFG-in (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MKC">4MKC</a>) and DFG-out (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IUH">5IUH</a>) conformations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. FTMap determined hot spot structure of complex II targets with example ligands bound. Pink ligands represent smaller compounds, whereas blue ligands represent the eRo5/bRo5 drug/clinical candidate identified by Doak et al. White ligands (where applicable) show alternative compounds. FTMap hot spots are colored according to the standard output, as shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. (a) Hot spot structure of tyrosine protein kinase ABL1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS9">3CS9</a>). Nilotinib (3CS9:NIL MW = 529.52 Da, <i>K</i><sub>D</sub> = 4.9 nM, pBA = 8.31) and PPY-A (2Z60:P3Y, MW = 372.42, <i>K</i><sub>I</sub> = 20.0 nM, pBA = 7.70) are shown in blue and pink, respectively. (b) Hot spot structure of epidermal growth factor receptor (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK">1XKK</a>). Lapatinib (1XKK:FMM MW = 581.06 Da, <i>K</i><sub>I</sub> = 3 nM, pBA = 8.52) and 5EDR:5N4 (MW = 361.4 Da, <i>K</i><sub>I</sub> = 34.3 nM, pBA = 7.46) are shown in blue and pink, respectively. (c) Hot spot structure of anaplastic lymphoma kinase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MKC">4MKC</a>). Certinib (4MKC:4MK, MW = 558.13 Da, <i>K</i><sub>I</sub> = 3.7 μM, pBA = 5.43) and 7-azaindole based inhibitor (4JOA:3DK, MW = 390.39 Da, IC<sub>50</sub> = 29 nM, pBA = 7.54) are shown in blue and pink, respectively. (d) Hot spot structure of glucocorticoid receptor (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P6W">4P6W</a>). Mometasone furoate (4P6W:MOF, MW = 521.43 Da, <i>K</i><sub>I</sub> = 0.7 nM, pBA = 9.15), 1P93:DEX (MW = 392.46 Da, <i>K</i><sub>D</sub> = 19 nM, pBA = 7.72), and 3K23:JZN (MW = 655.68 Da, IC<sub>50</sub> = 6.3 nM, pBA = 8.2) are shown in blue, pink, and white, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. FTMap determined hot spot structure of complex III targets with example ligands bound. Pink ligands represent smaller compounds, whereas blue ligands represent the eRo5/bRo5 drug/clinical candidateidentified by Doak et al. FTMap hot spots are colored according to standard output as shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. (a) Hot spot structure of cyclophilin A (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CWA">1CWA</a>). Cyclosporine A (1CWA:PRD_000142, MW = 1202.61 Da, <i>K</i><sub>D</sub> = 36.8 nM, pBA = 7.43) and 3RDD:EA4 (MW = 251.28 Da, <i>K</i><sub>I</sub> = 16800 nM, pBA = 4.77) are shown in blue and pink, respectively. (b) Hot spot structure of DOT1-like histone H3 methyltransferase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HRA">4HRA</a>). EPZ-5676 (4HRA:EP6, MW = 562.71 Da, <i>K</i><sub>I</sub> = 0.08 nM, pBA = 10.1) and 4EK9:EP4 (MW = 294.31 Da, <i>K</i><sub>I</sub> = 9 μM, pBA = 5.04) are shown in blue and pink, respectively. (c) Hot spot structure of lanosterol 14a-demethylase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EQB">5EQB</a>). Itraconazole (5EQB:1YN, MW = 705.63 Da, IC<sub>50</sub> = 19.4 nM, pBA = 7.71) and voriconazole (5HS1:VOR, MW = 349.31 Da, <i>K</i><sub>I</sub> = 174 μM, pBA = 3.76) are shown in blue and pink, respectively. (d) Hot spot structure of Bcl-2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LVT">4LVT</a>). Navitoclax (4LVT:1XJ, MW = 974.61 Da, <i>K</i><sub>I</sub> = 0.044 nM, pBA = 10.36) and 2W3L:DRO (MW = 576.09 Da, IC<sub>50</sub> = 30 nM, pBA = 7.52) are shown in blue and pink, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/medium/jm-2018-01732u_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. FTMap determined hot spot structure of type III targets with example ligands bound. Pink ligands represent smaller compounds, whereas blue ligands represent the eRo5/bRo5 drug/clinical candidate identified by Doak et al. White ligands (where applicable) show compounds larger than the Doak et al. identified ligand. FTMap hot spots are colored according to standard output as shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. (a) Hot spot structure of kinesin Eg5 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4A5Y">4A5Y</a>). Ispinesib (4A5Y:G7X, MW = 517.06 Da, <i>K</i><sub>I</sub> = 2.3 nM, pBA = 8.64) and (<i>S</i>)-enastron (2X7C:KZ9, MW = 274.34 Da, IC<sub>50</sub> = 2 μM, pBA = 5.7) are shown in blue and pink, respectively. (b) Hot spot structure of soluble acetylcholine receptor (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PMZ">3PMZ</a>). Tubocurarine (3PMZ:TUB MW = 609.73 Da, <i>K</i><sub>I</sub> = 3481.0 nM, pBA = 5.46), 2WNL:AN5 (MW = 178.23 Da, <i>K</i><sub>I</sub> = 120 μM, pBA = 6), and pinnatoxin A (4XHE:40P, MW = 711.92 Da, <i>K</i><sub>D</sub> = 0.05 nM, pBA = 10.3) are shown in blue, pink, and white, respectively. (c) Hot spot structure of HMG-CoA reductase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1HWK">1HWK</a>). Atorvastatin (1HWK:117, MW = 558.64 Da, <i>K</i><sub>I</sub> = 6.2 nM, pBA = 8.21) and mevastatin (1HW8:114, MW = 408.53 Da, IC<sub>50</sub> = 23 nM, pBA = 7.64) are shown in blue and pink, respectively. (d) Hot spot structure of hepatitis C virus NS5b subunit (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NLD">4NLD</a>). Beclabuvir (4NLD:2N7, MW = 659.84 Da, IC<sub>50</sub> = 0.02 μM, pBA = 7.7) and 2WCX:VGC (MW = 339.45 Da, IC<sub>50</sub> = 0.025 μM, pBA = 7.6) are shown in blue and pink, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.8b01732/20191122/images/large/jm-2018-01732u_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01732&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i57">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34732" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34732" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 85 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0ljP2-VWl0uBpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Two decades under the influence of the rule of five and the changing properties of approved oral drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslait73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1701-1714&author=M.+D.+Shultz&title=Two+decades+under+the+influence+of+the+rule+of+five+and+the+changing+properties+of+approved+oral+drugs&doi=10.1021%2Facs.jmedchem.8b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1701-1714</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two decades have passed since the rule of five ushered in the concept of "drug-like" properties.  Attempts to quantify, correlate, and categorize mols. based on Ro5 parameters evolved into the introduction of efficiency metrics with far reaching consequences in decision making by industry leaders and scientists seeking to discover new medicines.  Examn. of oral drug parameters approved before and after the original Ro5 anal. demonstrates that some parameters such as clogP and HBD remained const. while the cutoffs for parameters such as mol. wt. and HBA have increased substantially over the past 20 years.  The time dependent increase in the mol. wt. of oral drugs during the past 20 years provides compelling evidence to disprove the hypothesis that mol. wt. is a "drug-like" property.  This anal. does not validate parameters that have not changed as being "drug-like" but instead calls into question the entire hypothesis that "drug-like" properties exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnoxpQsv2kbVg90H21EOLACvtfcHk0lgOViEiVmiSFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslait73K&md5=d3c8b1107133dc0b7b277d22fa029597</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DTwo%2520decades%2520under%2520the%2520influence%2520of%2520the%2520rule%2520of%2520five%2520and%2520the%2520changing%2520properties%2520of%2520approved%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1701%26epage%3D1714%26doi%3D10.1021%2Facs.jmedchem.8b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Oral druggable space beyond the rule of 5: Insights from drugs and clinical candidates</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1115</span>– <span class="NLM_lpage">1142</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.chembiol.2014.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25237858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1115-1142&author=B.+C.+Doakauthor=B.+Overauthor=F.+Giordanettoauthor=J.+Kihlberg&title=Oral+druggable+space+beyond+the+rule+of+5%3A+Insights+from+drugs+and+clinical+candidates&doi=10.1016%2Fj.chembiol.2014.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Druggable Space beyond the Rule of 5: Insights from Drugs and Clinical Candidates</span></div><div class="casAuthors">Doak, Bradley Croy; Over, Bjorn; Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1115-1142</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The rule of 5 (Ro5) is a set of in silico guidelines applied to drug discovery to prioritize compds. with an increased likelihood of high oral absorption.  It has been influential in reducing attrition due to poor pharmacokinetics over the last 15 years.  However, strict reliance on the Ro5 may have resulted in lost opportunities, particularly for difficult targets.  To identify opportunities for oral drug discovery beyond the Ro5 (bRo5), we have comprehensively analyzed drugs and clin. candidates with mol. wt. (MW) > 500 Da.  We conclude that oral drugs are found far bRo5 and properties such as intramol. hydrogen bonding, macrocyclization, dosage, and formulations can be used to improve bRo5 bioavailability.  Natural products and structure-based design, often from peptidic leads, are key sources for oral bRo5 drugs.  These insights should help guide the design of oral drugs in bRo5 space, which is of particular interest for difficult targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvYqpeA0dh6rVg90H21EOLACvtfcHk0lgOViEiVmiSFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K&md5=b8a88c793572ad48de3809807a4699d8</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DOral%2520druggable%2520space%2520beyond%2520the%2520rule%2520of%25205%253A%2520Insights%2520from%2520drugs%2520and%2520clinical%2520candidates%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1115%26epage%3D1142%26doi%3D10.1016%2Fj.chembiol.2014.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeGoey, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span> <span> </span><span class="NLM_article-title">Beyond the rule of 5: Lessons learned from AbbVie’s drugs and compound collection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00717</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00717" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2636-2651&author=D.+A.+DeGoeyauthor=H.+J.+Chenauthor=P.+B.+Coxauthor=M.+D.+Wendt&title=Beyond+the+rule+of+5%3A+Lessons+learned+from+AbbVie%E2%80%99s+drugs+and+compound+collection&doi=10.1021%2Facs.jmedchem.7b00717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the Rule of 5: Lessons Learned from AbbVie's Drugs and Compound Collection</span></div><div class="casAuthors">DeGoey, David A.; Chen, Hui-Ju; Cox, Philip B.; Wendt, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2636-2651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing focus on the pursuit of targets considered to be less druggable that offer potential for development of promising new therapeutic agents for the treatment of diseases with large unmet medical need, particularly in the areas of oncol. and virol.  However, conducting drug discovery campaigns in "beyond rule of 5" (bRo5) chem. space presents a significant drug design and development challenge to medicinal chemists to achieve acceptable oral pharmacokinetics.  Retrospective anal. of past successes and failures in drug discovery bRo5 may shed light on the key principles that contribute to the oral bioavailability of successful bRo5 compds. and improve the efficiency of drug design for future projects.  We present here highlights and case studies of lessons learned from discovery of bRo5 compds.  A simple multiparametric scoring function (AB-MPS) was devised that correlated preclin. PK results with cLogD, no. of rotatable bonds, and no. of arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHuaXbtjlALVg90H21EOLACvtfcHk0lgOViEiVmiSFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ&md5=48984dc65410d83d4ac8ddaca030b024</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26aulast%3DDeGoey%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DH.%2BJ.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26atitle%3DBeyond%2520the%2520rule%2520of%25205%253A%2520Lessons%2520learned%2520from%2520AbbVie%25E2%2580%2599s%2520drugs%2520and%2520compound%2520collection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2636%26epage%3D2651%26doi%3D10.1021%2Facs.jmedchem.7b00717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0liEOmnJACNX5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viarengo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Quantifying the chameleonic properties of macrocycles and other high-molecular-weight drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">712</span>– <span class="NLM_lpage">717</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.drudis.2016.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=26891978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=712-717&author=A.+Whittyauthor=M.+Zhongauthor=L.+Viarengoauthor=D.+Beglovauthor=D.+R.+Hallauthor=S.+Vajda&title=Quantifying+the+chameleonic+properties+of+macrocycles+and+other+high-molecular-weight+drugs&doi=10.1016%2Fj.drudis.2016.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying the chameleonic properties of macrocycles and other high-molecular-weight drugs</span></div><div class="casAuthors">Whitty, Adrian; Zhong, Mengqi; Viarengo, Lauren; Beglov, Dmitri; Hall, David R.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">712-717</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Key to the pharmaceutical utility of certain macrocyclic drugs is a 'chameleonic' ability to change their conformation to expose polar groups in aq. soln., but bury them when traversing lipid membranes.  Based on anal. of the structures of 20 macrocyclic compds. that are approved oral drugs, we propose that good soly. requires a topol. polar surface area (TPSA, in Å2) of ≥0.2 × mol. wt. (MW).  Meanwhile, good passive membrane permeability requires a mol. (i.e., 3D) PSA in nonpolar environments of ≤140 Å2.  We show that one or other of these limits is almost invariably violated for compds. with MW > 600 Da, suggesting that some degree of chameleonic behavior is required for most high MW oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlN7rCqts5wLVg90H21EOLACvtfcHk0liEOmnJACNX5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D&md5=024476298f7b72efa3f0a9303bed5900</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DZhong%26aufirst%3DM.%26aulast%3DViarengo%26aufirst%3DL.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DQuantifying%2520the%2520chameleonic%2520properties%2520of%2520macrocycles%2520and%2520other%2520high-molecular-weight%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D712%26epage%3D717%26doi%3D10.1016%2Fj.drudis.2016.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The exploration of macrocycles for drug discovery--an underexploited structural class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+exploration+of+macrocycles+for+drug+discovery%2D%2Dan+underexploited+structural+class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0lifRfc-bNdmrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520exploration%2520of%2520macrocycles%2520for%2520drug%2520discovery--an%2520underexploited%2520structural%2520class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lifRfc-bNdmrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles are great cycles: Applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+Applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lifRfc-bNdmrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520Applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery beyond the rule of 5 - opportunities and challenges</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1080/17460441.2017.1264385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1080%2F17460441.2017.1264385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=27883294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BC2sjgtVejtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=115-119&author=B.+C.+Doakauthor=J.+Kihlberg&title=Drug+discovery+beyond+the+rule+of+5+-+opportunities+and+challenges&doi=10.1080%2F17460441.2017.1264385"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery beyond the rule of 5 - Opportunities and challenges</span></div><div class="casAuthors">Doak Bradley C; Kihlberg Jan</div><div class="citationInfo"><span class="NLM_cas:title">Expert opinion on drug discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-119</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRw3tmClaGOihOk4JA3soTqfW6udTcc2ebGpwd_IWtOg7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjgtVejtw%253D%253D&md5=bccc3e6cb938479044d18a2b25df9c1f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1080%2F17460441.2017.1264385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2017.1264385%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520beyond%2520the%2520rule%2520of%25205%2520-%2520opportunities%2520and%2520challenges%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2017%26volume%3D12%26spage%3D115%26epage%3D119%26doi%3D10.1080%2F17460441.2017.1264385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santos, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emery, F. S.</span></span> <span> </span><span class="NLM_article-title">Oral administration of peptide-based drugs: Beyond Lipinski’s rule</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2245</span>– <span class="NLM_lpage">2251</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1002%2Fcmdc.201600288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=27596610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVGgu7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2245-2251&author=G.+B.+Santosauthor=A.+Ganesanauthor=F.+S.+Emery&title=Oral+administration+of+peptide-based+drugs%3A+Beyond+Lipinski%E2%80%99s+rule&doi=10.1002%2Fcmdc.201600288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Administration of Peptide-Based Drugs: Beyond Lipinski's Rule</span></div><div class="casAuthors">Santos, Gabriela B.; Ganesan, A.; Emery, Flavio S.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2245-2251</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The use of peptides in therapy presents several limitations, from physicochem. characteristics to inadequate pharmacokinetic profiles for oral absorption.  As peptides are gaining importance in the therapeutic arsenal, there is an increasing need to rationalize the main characteristics of this compd. class in the market.  Therefore, we performed an extensive anal. of all known peptide drugs and clin. candidates based on their peptide features, physicochem. and structural properties, and correlated these with their administration route and therapeutic classes.  Peptide drugs are widely distributed across drug and pharmacol. space, covering several therapeutic areas with structural diversity and complexity, distributed between groups of cyclic and linear compds.  Although structural and physicochem. properties are clear within these groups, we counter the consensus that cyclic peptides have better oral availability than linear peptides, as most of the orally administrated peptides have linear structures.  This study and review furnishes information that could support peptide drug design, with a new cutoff of known descriptors that go beyond the Rule of Five.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr88nw1qqAJuLVg90H21EOLACvtfcHk0lh9lnXl6GK5yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVGgu7rI&md5=b2e5b6a1dff54c2c1f85671ab7314031</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600288%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DG.%2BB.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DEmery%26aufirst%3DF.%2BS.%26atitle%3DOral%2520administration%2520of%2520peptide-based%2520drugs%253A%2520Beyond%2520Lipinski%25E2%2580%2599s%2520rule%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D2245%26epage%3D2251%26doi%3D10.1002%2Fcmdc.201600288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">Small-molecule kinase inhibitors: An analysis of FDA-approved drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.drudis.2015.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=26210956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=5-10&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=Small-molecule+kinase+inhibitors%3A+An+analysis+of+FDA-approved+drugs&doi=10.1016%2Fj.drudis.2015.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors (SMKIs), 28 of which are approved by the US Food and Drug Administration (FDA), have been actively pursued as promising targeted therapeutics.  Here, we assess the key structural and physicochem. properties, target selectivity and mechanism of function, and therapeutic indications of these approved inhibitors.  Our anal. showed that >30% of approved SMKIs have a mol. wt. (MW) exceeding 500 and all have a total ring count of between three and five.  The assumption that type II inhibitors tend to be more selective than type I inhibitors has been proved to be unreliable.  Although previous SMKI research was concd. on tyrosine kinase inhibitors for cancer treatment, recent progress indicates diversification of SMKI research in terms of new targets, mechanistic types, and therapeutic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZrY68ZOI2xLVg90H21EOLACvtfcHk0lh9lnXl6GK5yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P&md5=228524fa464393053ecbfac19e994929</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DSmall-molecule%2520kinase%2520inhibitors%253A%2520An%2520analysis%2520of%2520FDA-approved%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.drudis.2015.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of protein-protein interactions: Progressing toward the reality</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1102</span>– <span class="NLM_lpage">1114</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.chembiol.2014.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25237857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1102-1114&author=M.+R.+Arkinauthor=Y.+Tangauthor=J.+A.+Wells&title=Small-molecule+inhibitors+of+protein-protein+interactions%3A+Progressing+toward+the+reality&doi=10.1016%2Fj.chembiol.2014.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing toward the Reality</span></div><div class="casAuthors">Arkin, Michelle R.; Tang, Yinyan; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1102-1114</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The past 20 years have seen many advances in our understanding of protein-protein interactions (PPIs) and how to target them with small-mol. therapeutics.  In 2004, we reviewed some early successes; since then, potent inhibitors have been developed for diverse protein complexes, and compds. are now in clin. trials for six targets.  Surprisingly, many of these PPI clin. candidates have efficiency metrics typical of "lead-like" or "drug-like" mols. and are orally available.  Successful discovery efforts have integrated multiple disciplines and make use of all the modern tools of target-based discovery-structure, computation, screening, and biomarkers.  PPIs become progressively more challenging as the interfaces become more complex, i.e., as binding epitopes are displayed on primary, secondary, or tertiary structures.  Here, we review the last 10 years of progress, focusing on the properties of PPI inhibitors that have advanced to clin. trials and prospects for the future of PPI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP01wt-4lfE7Vg90H21EOLACvtfcHk0liB1fpaz-wnXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J&md5=aec3897dc9bc66656723bc56c8d69155</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520Progressing%2520toward%2520the%2520reality%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1102%26epage%3D1114%26doi%3D10.1016%2Fj.chembiol.2014.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Opinion - how many drug targets are there?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">993</span>– <span class="NLM_lpage">996</span>, <span class="refDoi"> DOI: 10.1038/nrd2199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrd2199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=17139284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=993-996&author=J.+P.+Overingtonauthor=B.+Al-Lazikaniauthor=A.+L.+Hopkins&title=Opinion+-+how+many+drug+targets+are+there%3F&doi=10.1038%2Fnrd2199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">How many drug targets are there?</span></div><div class="casAuthors">Overington, John P.; Al-Lazikani, Bissan; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">993-996</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  For the past decade, the no. of mol. targets for approved drugs has been debated.  Here, we reconcile apparently contradictory previous reports into a comprehensive survey, and propose a consensus no. of current drug targets for all classes of approved therapeutic drugs.  One striking feature is the relatively const. historical rate of target innovation (the rate at which drugs against new targets are launched); however, the rate of developing drugs against new families is significantly lower.  The recent approval of drugs that target protein kinases highlights two addnl. trends: an emerging realization of the importance of polypharmacol., and also the power of a gene-family-led approach in generating novel and important therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDoZu-pkUsAbVg90H21EOLACvtfcHk0liB1fpaz-wnXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kju7jM&md5=14b5716f4e10b57ae71bf67e52a21929</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd2199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2199%26sid%3Dliteratum%253Aachs%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DOpinion%2520-%2520how%2520many%2520drug%2520targets%2520are%2520there%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D993%26epage%3D996%26doi%3D10.1038%2Fnrd2199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basse, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morelli, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, P.</span></span> <span> </span><span class="NLM_article-title">2P2IDB V2: Update of a structural database dedicated to orthosteric modulation of protein-protein interactions</span>. <i>Database</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2016</i></span>,  <span class="NLM_fpage">baw007</span>, <span class="refDoi"> DOI: 10.1093/database/baw007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fdatabase%2Fbaw007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=26980515" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2016&publication_year=2016&pages=baw007&author=M.+J.+Basseauthor=S.+Betziauthor=X.+Morelliauthor=P.+Roche&title=2P2IDB+V2%3A+Update+of+a+structural+database+dedicated+to+orthosteric+modulation+of+protein-protein+interactions&doi=10.1093%2Fdatabase%2Fbaw007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fdatabase%2Fbaw007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fdatabase%252Fbaw007%26sid%3Dliteratum%253Aachs%26aulast%3DBasse%26aufirst%3DM.%2BJ.%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DMorelli%26aufirst%3DX.%26aulast%3DRoche%26aufirst%3DP.%26atitle%3D2P2IDB%2520V2%253A%2520Update%2520of%2520a%2520structural%2520database%2520dedicated%2520to%2520orthosteric%2520modulation%2520of%2520protein-protein%2520interactions%26jtitle%3DDatabase%26date%3D2016%26volume%3D2016%26spage%3Dbaw007%26doi%3D10.1093%2Fdatabase%2Fbaw007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. C.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of protein-protein interactions: How to mimic a protein partner</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4679</span>– <span class="NLM_lpage">4684</span>, <span class="refDoi"> DOI: 10.2174/138161212802651634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.2174%2F138161212802651634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=22650256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2gu7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=4679-4684&author=D.+C.+Fry&title=Small-molecule+inhibitors+of+protein-protein+interactions%3A+How+to+mimic+a+protein+partner&doi=10.2174%2F138161212802651634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of protein-protein interactions: how to mimic a protein partner</span></div><div class="casAuthors">Fry, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">4679-4684</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This systematic review describes successful examples of small-mol. inhibitors of protein-protein interactions, and compares their binding strategies to those employed by the natural protein partners.  It extends and updates an earlier survey of this type.  From anal. of these systems, common themes and lessons are presented that may assist future drug discovery efforts involving targets in this class.  One encouraging finding is that a wide scope appears to be allowed at these sites in terms of binding strategies and chemotypes, which suggests that the outlook for finding small-mol. protein-protein inhibitors is favorable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTu1bO0sottbVg90H21EOLACvtfcHk0liB1fpaz-wnXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2gu7fL&md5=1d093305caf318924fdf72b239326671</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.2174%2F138161212802651634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212802651634%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DD.%2BC.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520How%2520to%2520mimic%2520a%2520protein%2520partner%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D4679%26epage%3D4684%26doi%3D10.2174%2F138161212802651634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClendon, C. L.</span></span> <span> </span><span class="NLM_article-title">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">1001</span>– <span class="NLM_lpage">1009</span>, <span class="refDoi"> DOI: 10.1038/nature06526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnature06526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=18075579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=1001-1009&author=J.+A.+Wellsauthor=C.+L.+McClendon&title=Reaching+for+high-hanging+fruit+in+drug+discovery+at+protein-protein+interfaces&doi=10.1038%2Fnature06526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span></div><div class="casAuthors">Wells, James A.; McClendon, Christopher L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">1001-1009</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting the interfaces between proteins has huge therapeutic potential, but discovering small-mol. drugs that disrupt protein-protein interactions is an enormous challenge.  Several recent success stories, however, indicate that protein-protein interfaces might be more tractable than has been thought.  These studies discovered small mols. that bind with drug-like potencies to 'hotspots' on the contact surfaces involved in protein-protein interactions.  Remarkably, these small mols. bind deeper within the contact surface of the target protein, and bind with much higher efficiencies, than do the contact atoms of the natural protein partner.  Some of these small mols. are now making their way through clin. trials, so this high-hanging fruit might not be far out of reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-nGR3BqqkrVg90H21EOLACvtfcHk0ljh8psoQKFR4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F&md5=e7ed69fd8c362a71c4b99f029c16a6fc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature06526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06526%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26atitle%3DReaching%2520for%2520high-hanging%2520fruit%2520in%2520drug%2520discovery%2520at%2520protein-protein%2520interfaces%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D1001%26epage%3D1009%26doi%3D10.1038%2Fnature06526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of protein-protein interactions: Progressing towards the dream</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd1343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrd1343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=15060526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1GktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=301-317&author=M.+R.+Arkinauthor=J.+A.+Wells&title=Small-molecule+inhibitors+of+protein-protein+interactions%3A+Progressing+towards+the+dream&doi=10.1038%2Fnrd1343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of protein-protein interactions: progressing towards the dream</span></div><div class="casAuthors">Arkin, Michelle R.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions have a key role in most biol. processes, and offer attractive opportunities for therapeutic intervention.  Developing small mols. that modulate protein-protein interactions is difficult, owing to issues such as the lack of well-defined binding pockets.  Nevertheless, there was important progress in this endeavor in recent years.  Here, the authors use illustrative examples to discuss general strategies for addressing the challenges inherent in the discovery and characterization of small-mol. inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7MiFDdLJu27Vg90H21EOLACvtfcHk0ljh8psoQKFR4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1GktrY%253D&md5=3e102c6207be420b4232056b4acdea12</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd1343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1343%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520Progressing%2520towards%2520the%2520dream%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D301%26epage%3D317%26doi%3D10.1038%2Fnrd1343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, B.</span></span> <span> </span><span class="NLM_article-title">Drug discovery beyond the ’rule-of-five’</span>. <i>Curr. Opin. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">478</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1016/j.copbio.2007.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.copbio.2007.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=18035532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislOkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2007&pages=478-488&author=M.+Q.+Zhangauthor=B.+Wilkinson&title=Drug+discovery+beyond+the+%E2%80%99rule-of-five%E2%80%99&doi=10.1016%2Fj.copbio.2007.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery beyond the 'rule-of-five'</span></div><div class="casAuthors">Zhang, Ming-Qiang; Wilkinson, Barrie</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">478-488</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although a very useful guideline for orally bioavailable small-mol. drug design, the 'rule-of-five' (also known as 'Lipinski's rule of drug-likeness') has to some extent been overemphasized.  Firstly, only 51% of all FDA-approved small-mol. drugs are both used orally and comply with the 'rule-of-five'.  This does not even include the increasing no. of biologicals of which several have reached blockbuster' status.  Secondly, it does not cover natural product and semisynthetic natural product drugs, which constitute over one-third of all marketed small-mol. drugs.  A more balanced and programmatic approach to drug discovery should be more productive than to rely on an overemphasis of 'rule-of-five' compliance.  Rather it should consider proactively the development of parenteral drugs in parallel to oral drugs and to consider the development of therapeutic antibodies in parallel to small-mol. drugs.  These are particularly relevant for efforts against 'first-in-class' and/or particularly challenging targets such as proteases and those involving protein-protein interactions.  In addn., more effort should be invested in natural product research.  Emerging novel technologies such as synthetic biol. (genetic engineering of living organisms to produce small-mol. therapeutics) may address several challenging issues of natural product-based drug discovery including synthetic feasibility and ligand efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGollnOuY9kGrbVg90H21EOLACvtfcHk0ljh8psoQKFR4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislOkug%253D%253D&md5=4121024557295c4f241255e43b43ea44</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2007.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2007.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%2BQ.%26aulast%3DWilkinson%26aufirst%3DB.%26atitle%3DDrug%2520discovery%2520beyond%2520the%2520%25E2%2580%2599rule-of-five%25E2%2580%2599%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2007%26volume%3D18%26spage%3D478%26epage%3D488%26doi%3D10.1016%2Fj.copbio.2007.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tonddast-Navaei, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skolnick, J.</span></span> <span> </span><span class="NLM_article-title">On the importance of composite protein multiple ligand interactions in protein pockets</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1252</span>– <span class="NLM_lpage">1259</span>, <span class="refDoi"> DOI: 10.1002/jcc.24523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1002%2Fjcc.24523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=27864975" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVygt7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=1252-1259&author=S.+Tonddast-Navaeiauthor=B.+Srinivasanauthor=J.+Skolnick&title=On+the+importance+of+composite+protein+multiple+ligand+interactions+in+protein+pockets&doi=10.1002%2Fjcc.24523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">On the importance of composite protein multiple ligand interactions in protein pockets</span></div><div class="casAuthors">Tonddast-Navaei, Sam; Srinivasan, Bharath; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1252-1259</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Conventional small mol. drug-discovery approaches target protein pockets.  However, the limited no. of geometrically distinct pockets leads to widespread promiscuity and deleterious side-effects.  Here, the idea of COmposite protein LIGands (COLIG) that interact with each other as well as the protein within a single ligand binding pocket is examd.  As a practical illustration, exptl. evidence that E. coli Dihydrofolate reductase inhibitors are COLIGs is presented.  Then, anal. of a nonredundant set of all holo PDB structures indicates that almost 47-76% of proteins (based on different sequence identity thresholds) can simultaneously bind multiple, interacting ligands in the same pocket.  Moreover, most ligands that are either Singletons and COLIGs bind at the bottom of ligand binding pocket and occupy 30% and 43% of the vol. of the bottom of the pocket.  This suggests the use of COLIGs as a potential new class of small mol. drugs.  © 2016 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjL7YEfq49pLVg90H21EOLACvtfcHk0li-_MSEtAeuhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVygt7vL&md5=0b3bc3ceb75d89e6314b1f5fec740061</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fjcc.24523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.24523%26sid%3Dliteratum%253Aachs%26aulast%3DTonddast-Navaei%26aufirst%3DS.%26aulast%3DSrinivasan%26aufirst%3DB.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DOn%2520the%2520importance%2520of%2520composite%2520protein%2520multiple%2520ligand%2520interactions%2520in%2520protein%2520pockets%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2017%26volume%3D38%26spage%3D1252%26epage%3D1259%26doi%3D10.1002%2Fjcc.24523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">How beyond rule of 5 drugs and clinical candidates bind to their targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2312-2327&author=B.+C.+Doakauthor=J.+Zhengauthor=D.+Dobritzschauthor=J.+Kihlberg&title=How+beyond+rule+of+5+drugs+and+clinical+candidates+bind+to+their+targets&doi=10.1021%2Facs.jmedchem.5b01286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></div><div class="casAuthors">Doak, Bradley C.; Zheng, Jie; Dobritzsch, Doreen; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2327</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve discovery of drugs for difficult targets, the opportunities of chem. space beyond the rule of 5 (bRo5) were examd. by retrospective anal. of a comprehensive set of structures for complexes between drugs and clin. candidates and their targets.  The anal. illustrates the potential of compds. far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites.  However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required.  Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility.  However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites.  These insights should contribute to altering perceptions of what targets are considered "druggable" and provide support for drug design in beyond rule of 5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZD-h9VodiPLVg90H21EOLACvtfcHk0li-_MSEtAeuhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F&md5=a417b605d1a3ffbc7ee84706401f01f4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520beyond%2520rule%2520of%25205%2520drugs%2520and%2520clinical%2520candidates%2520bind%2520to%2520their%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2312%26epage%3D2327%26doi%3D10.1021%2Facs.jmedchem.5b01286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohnuud, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mottarella, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">755</span>, <span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-755&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+family+of+web+servers+for+determining+and+characterizing+ligand-binding+hot+spots+of+proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0li-_MSEtAeuhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520family%2520of%2520web%2520servers%2520for%2520determining%2520and%2520characterizing%2520ligand-binding%2520hot%2520spots%2520of%2520proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D733%26epage%3D755%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0ljXdFOrP9HLsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">The PDBbind database: Collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2977</span>– <span class="NLM_lpage">2980</span>, <span class="refDoi"> DOI: 10.1021/jm030580l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030580l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2977-2980&author=R.+Wangauthor=X.+Fangauthor=Y.+Luauthor=S.+Wang&title=The+PDBbind+database%3A+Collection+of+binding+affinities+for+protein-ligand+complexes+with+known+three-dimensional+structures&doi=10.1021%2Fjm030580l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The PDBbind database: Collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span></div><div class="casAuthors">Wang, Renxiao; Fang, Xueliang; Lu, Yipin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2977-2980</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have screened the entire Protein Data Bank (Release No. 103, Jan. 2003) and identified 5671 protein-ligand complexes out of 19 621 exptl. structures.  A systematic examn. of the primary refs. of these entries has led to a collection of binding affinity data (Kd, Ki, and IC50) for a total of 1359 complexes.  The outcomes of this project have been organized into a Web-accessible database named the PDBbind database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1eUdeWioYBbVg90H21EOLACvtfcHk0ljXdFOrP9HLsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D&md5=86e609172307402d8b0d4589b1270a2f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm030580l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030580l%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520PDBbind%2520database%253A%2520Collection%2520of%2520binding%2520affinities%2520for%2520protein-ligand%2520complexes%2520with%2520known%2520three-dimensional%2520structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2977%26epage%3D2980%26doi%3D10.1021%2Fjm030580l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span> <span> </span><span class="NLM_article-title">Binding MOAD (mother of all databases)</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">340</span>, <span class="refDoi"> DOI: 10.1002/prot.20512</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1002%2Fprot.20512" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=15971202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVyrtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2005&pages=333-340&author=L.+Huauthor=M.+L.+Bensonauthor=R.+D.+Smithauthor=M.+G.+Lernerauthor=H.+A.+Carlson&title=Binding+MOAD+%28mother+of+all+databases%29&doi=10.1002%2Fprot.20512"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Binding MOAD (Mother of All Databases)</span></div><div class="casAuthors">Hu, Liegi; Benson, Mark L.; Smith, Richard D.; Lerner, Michael G.; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">333-340</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Binding MOAD (Mother of All Databases) is the largest collection of high-quality, protein-ligand complexes available from the Protein Data Bank.  At this time, Binding MOAD contains 5331 protein-ligand complexes comprised of 1780 unique protein families and 2630 unique ligands.  We have searched the crystallog. papers for all 5000 + structures and compiled binding data for 1375 (26%) of the protein-ligand complexes.  The binding-affinity data ranges 13 orders of magnitude.  This is the largest collection of binding data reported to date in the literature.  We have also addressed the issue of redundancy in the data.  To create a nonredundant dataset, one protein from each of the 1780 protein families was chosen as a representative.  Representatives were chosen by tightest binding, best resoln., etc.  For the 1780 "best" complexes that comprise the nonredundant version of Binding MOAD, 475 (27%) have binding data.  This significant collection of protein-ligand complexes will be very useful in elucidating the biophys. patterns of mol. recognition and enzymic regulation.  The complexes with binding-affinity data will help in the development of improved scoring functions and structure-based drug discovery techniques.  The dataset can be accessed at http://www.BindingMOAD.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGronvDiEFLeLbVg90H21EOLACvtfcHk0likXSK_QcuhAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVyrtr4%253D&md5=146632a2f30fda98cb987e984beb7506</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fprot.20512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20512%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%26aulast%3DBenson%26aufirst%3DM.%2BL.%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DLerner%26aufirst%3DM.%2BG.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DBinding%2520MOAD%2520%2528mother%2520of%2520all%2520databases%2529%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D60%26spage%3D333%26epage%3D340%26doi%3D10.1002%2Fprot.20512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorissen, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">BindingDB: A web-accessible database of experimentally determined protein–ligand binding affinities</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">D198</span>– <span class="NLM_lpage">D201</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fnar%2Fgkl999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=17145705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=D198-D201&author=T.+Liuauthor=Y.+Linauthor=X.+Wenauthor=R.+N.+Jorissenauthor=M.+K.+Gilson&title=BindingDB%3A+A+web-accessible+database+of+experimentally+determined+protein%E2%80%93ligand+binding+affinities&doi=10.1093%2Fnar%2Fgkl999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB: a web-accessible database of experimentally determined protein-ligand binding affinities</span></div><div class="casAuthors">Liu, Tiqing; Lin, Yuhmei; Wen, Xin; Jorissen, Robert N.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D198-D201</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">BindingDB is a publicly accessible database currently contg. ∼20 000 exptl. detd. binding affinities of protein-ligand complexes, for 110 protein targets including isoforms and mutational variants, and ∼11 000 small mol. ligands.  The data are extd. from the scientific literature, data collection focusing on proteins that are drug-targets or candidate drug-targets and for which structural data are present in the Protein Data Bank.  The BindingDB website supports a range of query types, including searches by chem. structure, substructure and similarity; protein sequence; ligand and protein names; affinity ranges and mol. wt.  Data sets generated by BindingDB queries can be downloaded in the form of annotated SDfiles for further anal., or used as the basis for virtual screening of a compd. database uploaded by the user.  The data in BindingDB are linked both to structural data in the PDB via PDB IDs and chem. and sequence searches, and to the literature in PubMed via PubMed IDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoymy3G_LuG-rVg90H21EOLACvtfcHk0likXSK_QcuhAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivFKktg%253D%253D&md5=0ccb20d9b9178a624d4829b5909e7ff8</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl999%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DJorissen%26aufirst%3DR.%2BN.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520A%2520web-accessible%2520database%2520of%2520experimentally%2520determined%2520protein%25E2%2580%2593ligand%2520binding%2520affinities%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DD198%26epage%3DD201%26doi%3D10.1093%2Fnar%2Fgkl999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalovich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: A large-scale bioactivity database for drug discovery</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">D1100</span>– <span class="NLM_lpage">1107</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+A+large-scale+bioactivity+database+for+drug+discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0likXSK_QcuhAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520A%2520large-scale%2520bioactivity%2520database%2520for%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3D1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span> <span> </span><span class="NLM_article-title">Unraveling hot spots in binding interfaces: Progress and challenges</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">14</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1016/S0959-440X(02)00283-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2FS0959-440X%2802%2900283-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11839484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=14-20&author=W.+L.+DeLano&title=Unraveling+hot+spots+in+binding+interfaces%3A+Progress+and+challenges&doi=10.1016%2FS0959-440X%2802%2900283-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Unraveling hot spots in binding interfaces: progress and challenges</span></div><div class="casAuthors">DeLano, Warren L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14-20</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 49 refs.  Protein interface hot spots, as revealed by alanine scanning mutagenesis, continue to stimulate interest in the biophys. basis of mol. recognition.  Although these regions apparently constitute fertile grounds for intermol. interactions, no general algorithm has yet been developed that can predict hot spots based solely on their shape or compn.  The discovery of structural plasticity in hot spot regions indicates that dynamic simulation techniques may be essential for achieving a predictive understanding of binding interface energetics.  Future progress will depend as much on the application of new computational approaches for dissecting protein interfaces as on expanding the empirical databank of mutagenic substitutions and their effects.  Despite current theor. shortcomings, recent methodol. advances provide efficient exptl. means of probing hot spots and enable immediate applications for hot spots in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk1HNI-4QfrrVg90H21EOLACvtfcHk0ljODWA3ES70tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D&md5=a297ae443e69c94b7cf2e342337f29ea</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2802%2900283-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252802%252900283-X%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26atitle%3DUnraveling%2520hot%2520spots%2520in%2520binding%2520interfaces%253A%2520Progress%2520and%2520challenges%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D12%26spage%3D14%26epage%3D20%26doi%3D10.1016%2FS0959-440X%2802%2900283-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Probing hot spots at protein-ligand binding sites: A fragment-based approach using biophysical methods</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">4992</span>– <span class="NLM_lpage">5000</span>, <span class="refDoi"> DOI: 10.1021/jm060490r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060490r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4992-5000&author=A.+Ciulliauthor=G.+Williamsauthor=A.+G.+Smithauthor=T.+L.+Blundellauthor=C.+Abell&title=Probing+hot+spots+at+protein-ligand+binding+sites%3A+A+fragment-based+approach+using+biophysical+methods&doi=10.1021%2Fjm060490r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Probing Hot Spots at Protein-Ligand Binding Sites: A Fragment-Based Approach Using Biophysical Methods</span></div><div class="casAuthors">Ciulli, Alessio; Williams, Glyn; Smith, Alison G.; Blundell, Tom L.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4992-5000</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mapping interactions at protein-ligand binding sites is an important aspect of understanding many biol. reactions and a key part of drug design.  In this paper, we have used a fragment-based approach to probe "hot spots" at the cofactor-binding site of a model dehydrogenase, Escherichia coli ketopantoate reductase.  Our strategy involved the breaking down of NADPH (Kd = 300 nM) into smaller fragments and the biophys. characterization of their binding using WaterLOGSY NMR spectroscopy, isothermal titrn. calorimetry (ITC), and inhibition studies.  The weak binding affinities of fragments were measured by direct ITC titrns. under low c value conditions.  The 2'-phosphate and the reduced nicotinamide groups were found to contribute a large part of the binding energy.  A combination of ITC and site-directed mutagenesis enabled us to locate the fragments at sep. hot spots on opposite ends of the cofactor-binding site.  This study has identified structural determinants for cofactor recognition that represent a blueprint for future inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAWWHBHKxvxbVg90H21EOLACvtfcHk0ljODWA3ES70tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D&md5=b6788ad16d6009d7cacd3e429b13523b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm060490r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060490r%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%2BG.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DProbing%2520hot%2520spots%2520at%2520protein-ligand%2520binding%2520sites%253A%2520A%2520fragment-based%2520approach%2520using%2520biophysical%2520methods%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D4992%26epage%3D5000%26doi%3D10.1021%2Fjm060490r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfleger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopitz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeiffer-Marek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baringhaus, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span> <span> </span><span class="NLM_article-title">Hot spots and transient pockets: Predicting the determinants of small-molecule binding to a protein-protein interface</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.1021/ci200322s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200322s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=120-133&author=A.+Metzauthor=C.+Pflegerauthor=H.+Kopitzauthor=S.+Pfeiffer-Marekauthor=K.+H.+Baringhausauthor=H.+Gohlke&title=Hot+spots+and+transient+pockets%3A+Predicting+the+determinants+of+small-molecule+binding+to+a+protein-protein+interface&doi=10.1021%2Fci200322s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Hot Spots and Transient Pockets: Predicting the Determinants of Small-Molecule Binding to a Protein-Protein Interface</span></div><div class="casAuthors">Metz, Alexander; Pfleger, Christopher; Kopitz, Hannes; Pfeiffer-Marek, Stefania; Baringhaus, Karl-Heinz; Gohlke, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">120-133</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interfaces are considered difficult targets for small-mol. protein-protein interaction modulators (PPIMs).  Here, the authors present for the first time a computational strategy that simultaneously considers aspects of energetics and plasticity in the context of PPIM binding to a protein interface.  The strategy aims at identifying the determinants of small-mol. binding, hot spots, and transient pockets, in a protein-protein interface to make use of this knowledge for predicting binding modes of and ranking PPIMs with respect to their affinity.  When applied to interleukin-2 (IL-2), the computationally inexpensive constrained geometric simulation method FRODA outperforms mol. dynamics simulations in sampling hydrophobic transient pockets.  The authors introduce the PPIAnalyzer approach for identifying transient pockets on the basis of geometrical criteria only.  A sequence of docking to identified transient pockets, starting structure selection based on hot spot information, RMSD clustering and intermol. docking energies, and MM-PBSA calcns. allows one to enrich IL-2 PPIMs from a set of decoys and to discriminate between subgroups of IL-2 PPIMs with low and high affinity.  The authors' strategy will be applicable in a prospective manner where nothing else than a protein-protein complex structure is known; hence, it can well be the first step in a structure-based endeavor to identify PPIMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol37VBT1jptbVg90H21EOLACvtfcHk0ljODWA3ES70tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt73I&md5=27aa41d22851e357d7edd40c7b7cbf28</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci200322s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200322s%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DA.%26aulast%3DPfleger%26aufirst%3DC.%26aulast%3DKopitz%26aufirst%3DH.%26aulast%3DPfeiffer-Marek%26aufirst%3DS.%26aulast%3DBaringhaus%26aufirst%3DK.%2BH.%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DHot%2520spots%2520and%2520transient%2520pockets%253A%2520Predicting%2520the%2520determinants%2520of%2520small-molecule%2520binding%2520to%2520a%2520protein-protein%2520interface%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D120%26epage%3D133%26doi%3D10.1021%2Fci200322s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Lessons from hot spot analysis for fragment-based drug discovery</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.tips.2015.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=26538314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=724-736&author=D.+R.+Hallauthor=D.+Kozakovauthor=A.+Whittyauthor=S.+Vajda&title=Lessons+from+hot+spot+analysis+for+fragment-based+drug+discovery&doi=10.1016%2Fj.tips.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons from hot spot analysis for fragment-based drug discovery</span></div><div class="casAuthors">Hall, David R.; Kozakov, Dima; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">724-736</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Anal. of binding energy hot spots at protein surfaces can provide crucial insights into the prospects for successful application of fragment-based drug discovery (FBDD), and whether a fragment hit can be advanced into a high-affinity, drug-like ligand.  The key factor is the strength of the top ranking hot spot, and how well a given fragment complements it.  We show that published data are sufficient to provide a sophisticated and quant. understanding of how hot spots derive from a protein 3D structure, and how their strength, no., and spatial arrangement govern the potential for a surface site to bind to fragment-sized and larger ligands.  This improved understanding provides important guidance for the effective application of FBDD in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdQmPulmhPx7Vg90H21EOLACvtfcHk0lixl7hGC1OdmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF&md5=3e682f28482d162590826f8f2e321b96</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLessons%2520from%2520hot%2520spot%2520analysis%2520for%2520fragment-based%2520drug%2520discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D724%26epage%3D736%26doi%3D10.1016%2Fj.tips.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kortvelyesi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Computational mapping identifies the binding sites of organic solvents on proteins</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">4290</span>– <span class="NLM_lpage">4295</span>, <span class="refDoi"> DOI: 10.1073/pnas.062398499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1073%2Fpnas.062398499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11904374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=4290-4295&author=S.+Dennisauthor=T.+Kortvelyesiauthor=S.+Vajda&title=Computational+mapping+identifies+the+binding+sites+of+organic+solvents+on+proteins&doi=10.1073%2Fpnas.062398499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Computational mapping identifies the binding sites of organic solvents on proteins</span></div><div class="casAuthors">Dennis, Sheldon; Kortvelyesi, Tamas; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4290-4295</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Computational mapping places mol. probes-small mols. or functional groups-on a protein surface to identify the most favorable binding positions.  Although x-ray crystallog. and NMR show that org. solvents bind to a limited no. of sites on a protein, current mapping methods result in hundreds of energy min. and do not reveal why some sites bind mols. with different sizes and polarities.  We describe a mapping algorithm that explains the origin of this phenomenon.  The algorithm has been applied to hen egg-white lysozyme and to thermolysin, interacting with eight and four different ligands, resp.  In both cases the search finds the consensus site to which all mols. bind, whereas other positions that bind only certain ligands are not necessarily found.  The consensus sites are pockets of the active site, lined with partially exposed hydrophobic residues and with a no. of polar residues toward the edge.  These sites can accommodate each ligand in a no. of rotational states, some with a hydrogen bond to one of the nearby donor/acceptor groups.  Specific substrates and/or inhibitors of hen egg-white lysozyme and thermolysin interact with the same side chains identified by the mapping, but form several hydrogen bonds and bind in unique orientations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4sEo2VxwF7Vg90H21EOLACvtfcHk0lixl7hGC1OdmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D&md5=90d71ffcc8860e87aefec3c5809b1127</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.062398499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.062398499%26sid%3Dliteratum%253Aachs%26aulast%3DDennis%26aufirst%3DS.%26aulast%3DKortvelyesi%26aufirst%3DT.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DComputational%2520mapping%2520identifies%2520the%2520binding%2520sites%2520of%2520organic%2520solvents%2520on%2520proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D4290%26epage%3D4295%26doi%3D10.1073%2Fpnas.062398499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landon, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancia, D. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Identification of hot spots within druggable binding regions by computational solvent mapping of proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1231</span>– <span class="NLM_lpage">1240</span>, <span class="refDoi"> DOI: 10.1021/jm061134b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061134b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1231-1240&author=M.+R.+Landonauthor=D.+R.+Lanciaauthor=J.+Yuauthor=S.+C.+Thielauthor=S.+Vajda&title=Identification+of+hot+spots+within+druggable+binding+regions+by+computational+solvent+mapping+of+proteins&doi=10.1021%2Fjm061134b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Hot Spots within Druggable Binding Regions by Computational Solvent Mapping of Proteins</span></div><div class="casAuthors">Landon, Melissa R.; Lancia, David R., Jr.; Yu, Jessamin; Thiel, Spencer C.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1231-1240</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here the authors apply the computational solvent mapping (CS-Map) algorithm toward the in silico identification of hot spots, i.e., regions of protein binding sites that are major contributors to the binding energy and, hence, are prime targets in drug design.  The CS-Map algorithm, developed for binding site characterization, moves small org. functional groups around the protein surface and dets. their most energetically favorable binding positions.  The utility of CS-Map algorithm toward the prediction of hot spot regions in druggable binding pockets is illustrated by three test systems: (1) renin aspartic protease, (2) a set of previously characterized druggable proteins, and (3) E. coli ketopantoate reductase.  In each of the three studies, existing literature was used to verify the results.  Based on the authors analyses, the authors conclude that the information provided by CS-Map can contribute substantially to the identification of hot spots, a necessary predecessor of fragment-based drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTVim9shdtBbVg90H21EOLACvtfcHk0lgcdIdtGcAuXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWisrw%253D&md5=2d6a0678acdb2dd45fed73b05229bd03</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm061134b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061134b%26sid%3Dliteratum%253Aachs%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DLancia%26aufirst%3DD.%2BR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DThiel%26aufirst%3DS.%2BC.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520hot%2520spots%2520within%2520druggable%2520binding%2520regions%2520by%2520computational%2520solvent%2520mapping%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1231%26epage%3D1240%26doi%3D10.1021%2Fjm061134b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landon, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, Q. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caaveiro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orwig, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petsko, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringe, D.</span></span> <span> </span><span class="NLM_article-title">Detection of ligand binding hot spots on protein surfaces via fragment-based methods: Application to DJ-1 and glucocerebrosidase</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1007/s10822-009-9283-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1007%2Fs10822-009-9283-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=19521672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=491-500&author=M.+R.+Landonauthor=R.+L.+Liebermanauthor=Q.+Q.+Hoangauthor=S.+Juauthor=J.+M.+Caaveiroauthor=S.+D.+Orwigauthor=D.+Kozakovauthor=R.+Brenkeauthor=G.+Y.+Chuangauthor=D.+Beglovauthor=S.+Vajdaauthor=G.+A.+Petskoauthor=D.+Ringe&title=Detection+of+ligand+binding+hot+spots+on+protein+surfaces+via+fragment-based+methods%3A+Application+to+DJ-1+and+glucocerebrosidase&doi=10.1007%2Fs10822-009-9283-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of ligand binding hot spots on protein surfaces via fragment-based methods: application to DJ-1 and glucocerebrosidase</span></div><div class="casAuthors">Landon, Melissa R.; Lieberman, Raquel L.; Hoang, Quyen Q.; Ju, Shulin; Caaveiro, Jose M. M.; Orwig, Susan D.; Kozakov, Dima; Brenke, Ryan; Chuang, Gwo-Yu; Beglov, Dmitry; Vajda, Sandor; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">491-500</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The identification of hot spots, i.e., binding regions that contribute substantially to the free energy of ligand binding, is a crit. step for structure-based drug design.  Here we present the application of two fragment-based methods to the detection of hot spots for DJ-1 and glucocerebrosidase (GCase), targets for the development of therapeutics for Parkinson's and Gaucher's diseases, resp.  While the structures of these two proteins are known, binding information is lacking.  In this study we employ the exptl. multiple solvent crystal structures (MSCS) method and computational fragment mapping (FTMap) to identify regions suitable for the development of pharmacol. chaperones for DJ-1 and GCase.  Comparison of data derived via MSCS and FTMap also shows that FTMap, a computational method for the identification of fragment binding hot spots, is an accurate and robust alternative to the performance of expensive and difficult crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVrKBMCEfe6LVg90H21EOLACvtfcHk0liDFS012mQCHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtb8%253D&md5=00a35c8ecaaa3b9f3f9aeff7c7e61d27</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9283-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9283-2%26sid%3Dliteratum%253Aachs%26aulast%3DLandon%26aufirst%3DM.%2BR.%26aulast%3DLieberman%26aufirst%3DR.%2BL.%26aulast%3DHoang%26aufirst%3DQ.%2BQ.%26aulast%3DJu%26aufirst%3DS.%26aulast%3DCaaveiro%26aufirst%3DJ.%2BM.%26aulast%3DOrwig%26aufirst%3DS.%2BD.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DDetection%2520of%2520ligand%2520binding%2520hot%2520spots%2520on%2520protein%2520surfaces%2520via%2520fragment-based%2520methods%253A%2520Application%2520to%2520DJ-1%2520and%2520glucocerebrosidase%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D491%26epage%3D500%26doi%3D10.1007%2Fs10822-009-9283-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarnieri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Binding hot spots and amantadine orientation in the influenza a virus M2 proton channel</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">2846</span>– <span class="NLM_lpage">2853</span>, <span class="refDoi"> DOI: 10.1016/j.bpj.2009.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.bpj.2009.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=19917240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhslartbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2009&pages=2846-2853&author=G.+Y.+Chuangauthor=D.+Kozakovauthor=R.+Brenkeauthor=D.+Beglovauthor=F.+Guarnieriauthor=S.+Vajda&title=Binding+hot+spots+and+amantadine+orientation+in+the+influenza+a+virus+M2+proton+channel&doi=10.1016%2Fj.bpj.2009.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Binding hot spots and amantadine orientation in the influenza A virus M2 proton channel</span></div><div class="casAuthors">Chuang, Gwo-Yu; Kozakov, Dima; Brenke, Ryan; Beglov, Dmitri; Guarnieri, Frank; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2846-2853</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Structures of truncated versions of the influenza A virus M2 proton channel have been detd. recently by x-ray crystallog. in the open conformation of the channel, and by NMR in the closed state.  The structures differ in the position of the bound inhibitors.  The x-ray structure shows a single amantadine mol. in the middle of the channel, whereas in the NMR structure four drug mols. bind at the channel's outer surface.  To study this controversy we applied computational solvent mapping, a technique developed for the identification of the most favorable drug binding hot spots of proteins.  The method moves mol. probes - small org. mols. contg. various functional groups - around the protein surface, finds favorable positions using empirical free energy functions, clusters the conformations, and ranks the clusters on the basis of the av. free energy.  The results of the mapping show that in both structures the primary hot spot is an internal cavity overlapping the amantadine binding site seen in the x-ray structure.  However, both structures also have weaker hot spots at the exterior locations that bind rimantadine in the NMR structure, although these sites are partially due to the favorable interactions with the interfacial region of the lipid bilayer.  As confirmed by docking calcns., the open channel binds amantadine at the more favorable internal site, in good agreement with the x-ray structure.  In contrast, the NMR structure is based on a peptide/micelle construct that is able to accommodate the small mol. probes used for the mapping, but has a too narrow pore for the rimantadine to access the internal hot spot, and hence the drug can bind only at the exterior sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC0NckQ50aUbVg90H21EOLACvtfcHk0ljurHPmQsGx4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhslartbs%253D&md5=004c2022981283d2d9f85218e49d898a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2009.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2009.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DGuarnieri%26aufirst%3DF.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DBinding%2520hot%2520spots%2520and%2520amantadine%2520orientation%2520in%2520the%2520influenza%2520a%2520virus%2520M2%2520proton%2520channel%26jtitle%3DBiophys.%2520J.%26date%3D2009%26volume%3D97%26spage%3D2846%26epage%3D2853%26doi%3D10.1016%2Fj.bpj.2009.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buhrman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kearney, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoleon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovrigina, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovrigin, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattos, C.</span></span> <span> </span><span class="NLM_article-title">Analysis of binding site hot spots on the surface of Ras GTPase</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>413</i></span>,  <span class="NLM_fpage">773</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2011.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.jmb.2011.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=21945529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKisbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=413&publication_year=2011&pages=773-789&author=G.+Buhrmanauthor=C.+O%E2%80%99Connorauthor=B.+Zerbeauthor=B.+M.+Kearneyauthor=R.+Napoleonauthor=E.+A.+Kovriginaauthor=S.+Vajdaauthor=D.+Kozakovauthor=E.+L.+Kovriginauthor=C.+Mattos&title=Analysis+of+binding+site+hot+spots+on+the+surface+of+Ras+GTPase&doi=10.1016%2Fj.jmb.2011.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Binding Site Hot Spots on the Surface of Ras GTPase</span></div><div class="casAuthors">Buhrman, Greg; O'Connor, Casey; Zerbe, Brandon; Kearney, Bradley M.; Napoleon, Raeanne; Kovrigina, Elizaveta A.; Vajda, Sandor; Kozakov, Dima; Kovrigin, Evgenii L.; Mattos, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">413</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">773-789</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We have recently discovered an allosteric switch in Ras, bringing an addnl. level of complexity to this GTPase whose mutants are involved in nearly 30% of cancers.  Upon activation of the allosteric switch, there is a shift in helix 3/loop 7 assocd. with a disorder to order transition in the active site.  Here, we use a combination of multiple solvent crystal structures and computational solvent mapping (FTMap) to det. binding site hot spots in the "off" and "on" allosteric states of the GTP-bound form of H-Ras.  Thirteen sites are revealed, expanding possible target sites for ligand binding well beyond the active site.  Comparison of FTMaps for the H and K isoforms reveals essentially identical hot spots.  Furthermore, using NMR measurements of spin relaxation, we detd. that K-Ras exhibits global conformational dynamics very similar to those we previously reported for H-Ras.  We thus hypothesize that the global conformational rearrangement serves as a mechanism for allosteric coupling between the effector interface and remote hot spots in all Ras isoforms.  At least with respect to the binding sites involving the G domain, H-Ras is an excellent model for K-Ras and probably N-Ras as well.  Ras has so far been elusive as a target for drug design.  The present work identifies various unexplored hot spots throughout the entire surface of Ras, extending the focus from the disordered active site to well-ordered locations that should be easier to target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAmYA3G3UBirVg90H21EOLACvtfcHk0lj_2p7pB53oMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKisbrN&md5=d54dee9a6ac58d27f87d85a6d25415a3</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DBuhrman%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DC.%26aulast%3DZerbe%26aufirst%3DB.%26aulast%3DKearney%26aufirst%3DB.%2BM.%26aulast%3DNapoleon%26aufirst%3DR.%26aulast%3DKovrigina%26aufirst%3DE.%2BA.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DKovrigin%26aufirst%3DE.%2BL.%26aulast%3DMattos%26aufirst%3DC.%26atitle%3DAnalysis%2520of%2520binding%2520site%2520hot%2520spots%2520on%2520the%2520surface%2520of%2520Ras%2520GTPase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D413%26spage%3D773%26epage%3D789%26doi%3D10.1016%2Fj.jmb.2011.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span> <span> </span><span class="NLM_article-title">Relationship between hot spot residues and ligand binding hot spots in protein-protein interfaces</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2236</span>– <span class="NLM_lpage">2244</span>, <span class="refDoi"> DOI: 10.1021/ci300175u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300175u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpsl2js7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2236-2244&author=B.+S.+Zerbeauthor=D.+R.+Hallauthor=S.+Vajdaauthor=A.+Whittyauthor=D.+Kozakov&title=Relationship+between+hot+spot+residues+and+ligand+binding+hot+spots+in+protein-protein+interfaces&doi=10.1021%2Fci300175u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Relationship between Hot Spot Residues and Ligand Binding Hot Spots in Protein-Protein Interfaces</span></div><div class="casAuthors">Zerbe, Brandon S.; Hall, David R.; Vajda, Sandor; Whitty, Adrian; Kozakov, Dima</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2236-2244</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the context of protein-protein interactions, the term "hot spot" refers to a residue or cluster of residues that makes a major contribution to the binding free energy, as detd. by alanine scanning mutagenesis.  In contrast, in pharmaceutical research, a hot spot is a site on a target protein that has high propensity for ligand binding and hence is potentially important for drug discovery.  Here we examine the relationship between these two hot spot concepts by comparing alanine scanning data for a set of 15 proteins with results from mapping the protein surfaces for sites that can bind fragment-sized small mols.  We find the two types of hot spots are largely complementary; the residues protruding into hot spot regions identified by computational mapping or exptl. fragment screening are almost always themselves hot spot residues as defined by alanine scanning expts.  Conversely, a residue that is found by alanine scanning to contribute little to binding rarely interacts with hot spot regions on the partner protein identified by fragment mapping.  In spite of the strong correlation between the two hot spot concepts, they fundamentally differ, however.  In particular, while identification of a hot spot by alanine scanning establishes the potential to generate substantial interaction energy with a binding partner, there are addnl. topol. requirements to be a hot spot for small mol. binding.  Hence, only a minority of hot spots identified by alanine scanning represent sites that are potentially useful for small inhibitor binding, and it is this subset that is identified by exptl. or computational fragment screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEczZ5jWRflrVg90H21EOLACvtfcHk0liPQqoQQ3mfCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpsl2js7o%253D&md5=4b5e10c1dd8e2db2ed4ffda8a175de32</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci300175u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300175u%26sid%3Dliteratum%253Aachs%26aulast%3DZerbe%26aufirst%3DB.%2BS.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DKozakov%26aufirst%3DD.%26atitle%3DRelationship%2520between%2520hot%2520spot%2520residues%2520and%2520ligand%2520binding%2520hot%2520spots%2520in%2520protein-protein%2520interfaces%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2236%26epage%3D2244%26doi%3D10.1021%2Fci300175u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Golden, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayeg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sazinsky, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">Comprehensive experimental and computational analysis of binding energy hot spots at the NF-κB essential modulator/IKKβ protein-protein interface</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">6242</span>– <span class="NLM_lpage">6256</span>, <span class="refDoi"> DOI: 10.1021/ja400914z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja400914z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFKnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6242-6256&author=M.+S.+Goldenauthor=S.+M.+Coteauthor=M.+Sayegauthor=B.+S.+Zerbeauthor=E.+A.+Villarauthor=D.+Beglovauthor=S.+L.+Sazinskyauthor=R.+M.+Georgiadisauthor=S.+Vajdaauthor=D.+Kozakovauthor=A.+Whitty&title=Comprehensive+experimental+and+computational+analysis+of+binding+energy+hot+spots+at+the+NF-%CE%BAB+essential+modulator%2FIKK%CE%B2+protein-protein+interface&doi=10.1021%2Fja400914z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Experimental and Computational Analysis of Binding Energy Hot Spots at the NF-κB Essential Modulator/IKKβ Protein-Protein Interface</span></div><div class="casAuthors">Golden, Mary S.; Cote, Shaun M.; Sayeg, Marianna; Zerbe, Brandon S.; Villar, Elizabeth A.; Beglov, Dmitri; Sazinsky, Stephen L.; Georgiadis, Rosina M.; Vajda, Sandor; Kozakov, Dima; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6242-6256</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a comprehensive anal. of binding energy hot spots at the protein-protein interaction (PPI) interface between nuclear factor kappa B (NF-κB) essential modulator (NEMO) and IκB kinase subunit β (IKKβ), an interaction that is crit. for NF-κB pathway signaling, using exptl. alanine scanning mutagenesis and also the FTMap method for computational fragment screening.  The exptl. results confirm that the previously identified NEMO binding domain (NBD) region of IKKβ contains the highest concn. of hot-spot residues, the strongest of which are W739, W741, and L742 (ΔΔG = 4.3, 3.5, and 3.2 kcal/mol, resp.).  The region occupied by these residues defines a potentially druggable binding site on NEMO that extends for ∼16 Å to addnl. include the regions that bind IKKβ L737 and F734.  NBD residues D738 and S740 are also important for binding but do not make direct contact with NEMO, instead likely acting to stabilize the active conformation of surrounding residues.  We addnl. found two previously unknown hot-spot regions centered on IKKβ residues L708/V709 and L719/I723.  The computational approach successfully identified all three hot-spot regions on IKKβ.  Moreover, the method was able to accurately quantify the energetic importance of all hot-spot residues involving direct contact with NEMO.  Our results provide new information to guide the discovery of small-mol. inhibitors that target the NEMO/IKKβ interaction.  They addnl. clarify the structural and energetic complementarity between "pocket-forming" and "pocket-occupying" hot-spot residues, and further validate computational fragment mapping as a method for identifying hot spots at PPI interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLvGilMwRJGrVg90H21EOLACvtfcHk0liPQqoQQ3mfCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFKnsL0%253D&md5=d996a1aef7668ff03fe9bf3445c7d700</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja400914z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja400914z%26sid%3Dliteratum%253Aachs%26aulast%3DGolden%26aufirst%3DM.%2BS.%26aulast%3DCote%26aufirst%3DS.%2BM.%26aulast%3DSayeg%26aufirst%3DM.%26aulast%3DZerbe%26aufirst%3DB.%2BS.%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DSazinsky%26aufirst%3DS.%2BL.%26aulast%3DGeorgiadis%26aufirst%3DR.%2BM.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DComprehensive%2520experimental%2520and%2520computational%2520analysis%2520of%2520binding%2520energy%2520hot%2520spots%2520at%2520the%2520NF-%25CE%25BAB%2520essential%2520modulator%252FIKK%25CE%25B2%2520protein-protein%2520interface%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6242%26epage%3D6256%26doi%3D10.1021%2Fja400914z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoleon, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yueh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">New frontiers in druggability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9063</span>– <span class="NLM_lpage">9088</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9063-9088&author=D.+Kozakovauthor=D.+R.+Hallauthor=R.+L.+Napoleonauthor=C.+Yuehauthor=A.+Whittyauthor=S.+Vajda&title=New+frontiers+in+druggability&doi=10.1021%2Facs.jmedchem.5b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">New Frontiers in Druggability</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Napoleon, Raeanne L.; Yueh, Christine; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9063-9088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A powerful early approach to evaluating the druggability of proteins involved detg. the hit rate in NMR-based screening of a library of small compds.  Here, we show that a computational analog of this method, based on mapping proteins using small mols. as probes, can reliably reproduce druggability results from NMR-based screening and can provide a more meaningful assessment in cases where the two approaches disagree.  We apply the method to a large set of proteins.  The results show that, because the method is based on the biophysics of binding rather than on empirical parametrization, meaningful information can be gained about classes of proteins and classes of compds. beyond those resembling validated targets and conventionally druglike ligands.  In particular, the method identifies targets that, while not druggable by druglike compds., may become druggable using compd. classes such as macrocycles or other large mols. beyond the rule-of-five limit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCWoMwMpuM2bVg90H21EOLACvtfcHk0lihfEo_9hS52Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF&md5=77fd532a1f0129d096eb7a1fbfb1d422</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00586%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DNapoleon%26aufirst%3DR.%2BL.%26aulast%3DYueh%26aufirst%3DC.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DNew%2520frontiers%2520in%2520druggability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9063%26epage%3D9088%26doi%3D10.1021%2Facs.jmedchem.5b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cencic, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelletier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Structural conservation of druggable hot spots in protein-protein interfaces</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">13528</span>– <span class="NLM_lpage">13533</span>, <span class="refDoi"> DOI: 10.1073/pnas.1101835108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1073%2Fpnas.1101835108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=21808046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2ms7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13528-13533&author=D.+Kozakovauthor=D.+R.+Hallauthor=G.+Y.+Chuangauthor=R.+Cencicauthor=R.+Brenkeauthor=L.+E.+Groveauthor=D.+Beglovauthor=J.+Pelletierauthor=A.+Whittyauthor=S.+Vajda&title=Structural+conservation+of+druggable+hot+spots+in+protein-protein+interfaces&doi=10.1073%2Fpnas.1101835108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structural conservation of druggable hot spots in protein-protein interfaces</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Chuang, Gwo-Yu; Cencic, Regina; Brenke, Ryan; Grove, Laurie E.; Beglov, Dmitri; Pelletier, Jerry; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">13528-13533, S13528/1-S13528/26</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Despite the growing no. of examples of small-mol. inhibitors that disrupt protein-protein interactions (PPIs), the origin of druggability of such targets is poorly understood.  To identify druggable sites in protein-protein interfaces we combine computational solvent mapping, which explores the protein surface using a variety of small "probe" mols., with a conformer generator to account for side-chain flexibility.  Applications to unliganded structures of 15 PPI target proteins show that the druggable sites comprise a cluster of binding hot spots, distinguishable from other regions of the protein due to their concave topol. combined with a pattern of hydrophobic and polar functionality.  This combination of properties confers on the hot spots a tendency to bind org. species possessing some polar groups decorating largely hydrophobic scaffolds.  Thus, druggable sites at PPI are not simply sites that are complementary to particular org. functionality, but rather possess a general tendency to bind org. compds. with a variety of structures, including key side chains of the partner protein.  Results also highlight the importance of conformational adaptivity at the binding site to allow the hot spots to expand to accommodate a ligand of drug-like dimensions.  The crit. components of this adaptivity are largely local, involving primarily low energy side-chain motions within 6 Å of a hot spot.  The structural and physicochem. signature of druggable sites at PPI interfaces is sufficiently robust to be detectable from the structure of the unliganded protein, even when substantial conformational adaptation is required for optimal ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpibu1osEQcZrVg90H21EOLACvtfcHk0lihfEo_9hS52Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2ms7rL&md5=fc9b8a30f1b30c97bba5a843fb9dc03b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1101835108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1101835108%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DChuang%26aufirst%3DG.%2BY.%26aulast%3DCencic%26aufirst%3DR.%26aulast%3DBrenke%26aufirst%3DR.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DPelletier%26aufirst%3DJ.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DStructural%2520conservation%2520of%2520druggable%2520hot%2520spots%2520in%2520protein-protein%2520interfaces%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D13528%26epage%3D13533%26doi%3D10.1073%2Fpnas.1101835108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cully, M.</span></span> <span> </span><span class="NLM_article-title">Milestone 9: Ximelagatran sets the stage for NOACs</span>. <i>Nat. Rev. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>, <span class="refDoi"> DOI: 10.1038/nrcardio.2017.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrcardio.2017.179" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=M.+Cully&title=Milestone+9%3A+Ximelagatran+sets+the+stage+for+NOACs&doi=10.1038%2Fnrcardio.2017.179"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnrcardio.2017.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrcardio.2017.179%26sid%3Dliteratum%253Aachs%26aulast%3DCully%26aufirst%3DM.%26atitle%3DMilestone%25209%253A%2520Ximelagatran%2520sets%2520the%2520stage%2520for%2520NOACs%26jtitle%3DNat.%2520Rev.%2520Cardiol.%26date%3D2017%26doi%3D10.1038%2Fnrcardio.2017.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keisu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, T. B.</span></span> <span> </span><span class="NLM_article-title">Drug-induced liver injury in humans: The case of ximelagatran</span>. <i>Handb. Exp. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>196</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1007/978-3-642-00663-0_13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1007%2F978-3-642-00663-0_13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvVymsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=196&publication_year=2010&pages=407-418&author=M.+Keisuauthor=T.+B.+Andersson&title=Drug-induced+liver+injury+in+humans%3A+The+case+of+ximelagatran&doi=10.1007%2F978-3-642-00663-0_13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-induced liver injury in humans: the case of ximelagatran</span></div><div class="casAuthors">Keisu, M.; Andersson, T. B.</div><div class="citationInfo"><span class="NLM_cas:title">Handbook of Experimental Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">196</span>
        (<span class="NLM_cas:issue">Adverse Drug Reactions</span>),
    <span class="NLM_cas:pages">407-418</span>CODEN:
                <span class="NLM_cas:coden">HEPHD2</span>;
        ISSN:<span class="NLM_cas:issn">0171-2004</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Ximelagatran was the first orally available direct thrombin inhibitor under clin. development that also reached the market.  Ximelagatran was tested in an extensive clin. program.  Short-term use (< 12 days) in humans including the phase III clin. trials did not indicate any hepatotoxic potential.  Increased hepatic enzyme levels were first obsd. at a higher frequency when evaluating the long-term (>35 days) use of ximelagatran (incidence of >3 × upper limit of normal (ULN) plasma ALT was 7.9%).  The frequency of elevated total bilirubin levels was similar in the ximelagatran and the comparator groups.  However, the combination of ALT > 3 × ULN and total bilirubin > 2 × ULN was 0.5% among patients treated with ximelagatran and 0.1% among patients in the comparator group.  Symptoms such as fever and rash potentially indicating hypersensitivity (immunol. type of reaction) were low and did not differ between ximelagatran and the comparators.  The withdrawal of ximelagatran from the market and termination of the ximelagatran development program was triggered by safety data from a 35-day study, indicating that severe hepatic injury in a patient could develop after exposure to the drug has been completed and that regular liver function monitoring may not mitigate the possible risk of severe hepatic injury.  As for many drugs causing liver injury, the std. preclin. toxicol. studies provided no indication that ximelagatran affected hepatic functions.  In addn., extensive investigations using human-based in vitro models have not been able to define mechanisms explaining the pattern of hepatic injury obsd. in long-term clin. trials.  A pharmacogenomic study provided evidence that the ALT increases were assocd. with major histocompatibility complex (MHC) alleles DRB1'07 and DQA1*02 suggesting a possible immunogenic pathogenesis.  This example provides important clues to the mechanism of idiosyncratic drug-induced liver toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8MnjfyasU1bVg90H21EOLACvtfcHk0lhi8g4QVIFyUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvVymsr8%253D&md5=8713782d6f002abd9d8f8ca6aa41d7e0</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-00663-0_13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-3-642-00663-0_13%26sid%3Dliteratum%253Aachs%26aulast%3DKeisu%26aufirst%3DM.%26aulast%3DAndersson%26aufirst%3DT.%2BB.%26atitle%3DDrug-induced%2520liver%2520injury%2520in%2520humans%253A%2520The%2520case%2520of%2520ximelagatran%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2010%26volume%3D196%26spage%3D407%26epage%3D418%26doi%3D10.1007%2F978-3-642-00663-0_13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riester, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirsching, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salinas, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebinoga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamphausen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkwirth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesenfeldt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturzebecher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedrich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thurk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwienhorst, A.</span></span> <span> </span><span class="NLM_article-title">Thrombin inhibitors identified by computer-assisted multiparameter design</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">8597</span>– <span class="NLM_lpage">8602</span>, <span class="refDoi"> DOI: 10.1073/pnas.0501983102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1073%2Fpnas.0501983102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=15937115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsFCgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=8597-8602&author=D.+Riesterauthor=F.+Wirschingauthor=G.+Salinasauthor=M.+Kellerauthor=M.+Gebinogaauthor=S.+Kamphausenauthor=C.+Merkwirthauthor=R.+Goetzauthor=M.+Wiesenfeldtauthor=J.+Sturzebecherauthor=W.+Bodeauthor=R.+Friedrichauthor=M.+Thurkauthor=A.+Schwienhorst&title=Thrombin+inhibitors+identified+by+computer-assisted+multiparameter+design&doi=10.1073%2Fpnas.0501983102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Thrombin inhibitors identified by computer-assisted multiparameter design</span></div><div class="casAuthors">Riester, Daniel; Wirsching, Frank; Salinas, Gabriela; Keller, Martina; Gebinoga, Michael; Kamphausen, Stefan; Merkwirth, Christian; Goetz, Ruediger; Wiesenfeldt, Martin; Stuerzebecher, Joerg; Bode, Wolfram; Friedrich, Rainer; Thuerk, Marcel; Schwienhorst, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8597-8602</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Here, the authors present a series of thrombin inhibitors that were generated by using powerful computer-assisted multiparameter optimization process.  The process was organized in design cycles, starting with a set of randomly chosen mols.  Each cycle combined combinatorial synthesis, multiparameter characterization of compds. in a variety of bioassays, and algorithmic processing of the data to devise a set of compds. to be synthesized in the next cycle.  The identified lead compds. exhibited thrombin inhibitory consts. in the lower nanomolar range.  They are by far the most selective synthetic thrombin inhibitors, with selectivities of >100,000-fold toward other proteases such as Factor Xa, Factor XIIa, urokinase, plasmin, and Plasma kallikrein.  Furthermore, these compds. exhibit a favorable profile, comprising nontoxicity, high metabolic stability, low serum protein binding, good soly., high anticoagulant activity, and a slow and exclusively renal elimination from the circulation in a rat model.  Finally, x-ray crystallog. anal. of a thrombin-inhibitor complex revealed a binding mode with a neutral moiety in the S1 pocket of thrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomQF-Oe6iPbbVg90H21EOLACvtfcHk0lj5ST0Afox3oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsFCgtrc%253D&md5=f9f43256de6ddd302b4fe4591572a255</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0501983102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0501983102%26sid%3Dliteratum%253Aachs%26aulast%3DRiester%26aufirst%3DD.%26aulast%3DWirsching%26aufirst%3DF.%26aulast%3DSalinas%26aufirst%3DG.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DGebinoga%26aufirst%3DM.%26aulast%3DKamphausen%26aufirst%3DS.%26aulast%3DMerkwirth%26aufirst%3DC.%26aulast%3DGoetz%26aufirst%3DR.%26aulast%3DWiesenfeldt%26aufirst%3DM.%26aulast%3DSturzebecher%26aufirst%3DJ.%26aulast%3DBode%26aufirst%3DW.%26aulast%3DFriedrich%26aufirst%3DR.%26aulast%3DThurk%26aufirst%3DM.%26aulast%3DSchwienhorst%26aufirst%3DA.%26atitle%3DThrombin%2520inhibitors%2520identified%2520by%2520computer-assisted%2520multiparameter%2520design%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D8597%26epage%3D8602%26doi%3D10.1073%2Fpnas.0501983102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pool, J. L.</span></span> <span> </span><span class="NLM_article-title">Direct renin inhibition: Focus on aliskiren</span>. <i>J. Manag. Care Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.18553/jmcp.2007.13.s8-b.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.18553%2Fjmcp.2007.13.s8-b.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=17970614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BD2snptFKqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=21-33&author=J.+L.+Pool&title=Direct+renin+inhibition%3A+Focus+on+aliskiren&doi=10.18553%2Fjmcp.2007.13.s8-b.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Direct renin inhibition: focus on aliskiren</span></div><div class="casAuthors">Pool James L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of managed care pharmacy : JMCP</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8 Suppl B</span>),
    <span class="NLM_cas:pages">21-33</span>
        ISSN:<span class="NLM_cas:issn">1083-4087</span>.
    </div><div class="casAbstract">BACKGROUND:  Despite the availability of many effective, well-tolerated drugs, a significant proportion of treated hypertensive patients still have uncontrolled high blood pressure (BP) and thus face serious morbidity and mortality.  The renin-angiotensin aldosterone system (RAAS) is a key target for BP control and for cardiovascular and renal protection.  Renin controls the rate-limiting step in the RAAS cascade and hence is the optimal target for RAAS suppression.  Aliskiren is the first direct renin inhibitor (DRI) to be approved by the U.S.  Food and Drug Administration and the European Medicines Agency for treating hypertension.  OBJECTIVE:  To provide an overview of the pharmacology, pharmacokinetics, preclinical, and clinical efficacy and safety data on the DRI aliskiren.  RESULTS:  Approximately 70% of essential hypertension is associated with elevated renin levels.  Aliskiren is a potent and highly specific inhibitor of renin, with oral bioavailability of 2.6% and an elimination half-life of 40 hours, making it suitable for once-daily oral administration.  Aliskiren dose-dependently reduced BP, inhibited plasma renin activity (PRA), attenuated renal damage in animal models, and showed efficient and longer- lasting blockade of the RAAS in normotensive human subjects compared with other RAAS inhibitors.  The clinical efficacy and safety of aliskiren have been evaluated both as monotherapy and in combination with other antihypertensive agents in phase II and phase III trials of patients with mild to severe hypertension.  When used as monotherapy, aliskiren led to significant dose-dependent reductions in BP from baseline that were greater than those obtained with placebo and comparable with those achieved with an angiotensin II receptor blocker (ARB).  The combination of aliskiren with a diuretic, a calcium channel blocker (CCB), an angiotensin-converting enzyme inhibitor (ACEI), or an ARB generally had greater and longer-lasting BP-lowering efficacy than did single agents alone.  Aliskiren also countered the reactive increase in PRA caused by diuretic, CC B, ACEI, and ARB therapy.  Once-daily treatment with aliskiren was well tolerated.  CONCLUSIONS:  As a DRI, aliskiren blocks the RAAS more completely than do other current downstream RAAS inhibitors.  When used once daily, aliskiren is a safe and effective antihypertensive agent that can be used as monotherapy or in combination with other agents to provide additional options to improve BP control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS71W2Wu8cAkEg6wdEZmT2rfW6udTcc2eZN91zHVUBnOLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2snptFKqtA%253D%253D&md5=7f59662d64fbd7bbe4b5be1acdd40a5e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.18553%2Fjmcp.2007.13.s8-b.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18553%252Fjmcp.2007.13.s8-b.21%26sid%3Dliteratum%253Aachs%26aulast%3DPool%26aufirst%3DJ.%2BL.%26atitle%3DDirect%2520renin%2520inhibition%253A%2520Focus%2520on%2520aliskiren%26jtitle%3DJ.%2520Manag.%2520Care%2520Pharm.%26date%3D2007%26volume%3D13%26spage%3D21%26epage%3D33%26doi%3D10.18553%2Fjmcp.2007.13.s8-b.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sepehrdad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frishman, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stier, C. T.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sica, D. A.</span></span> <span> </span><span class="NLM_article-title">Direct inhibition of renin as a cardiovascular pharmacotherapy: Focus on aliskiren</span>. <i>Cardiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">242</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1097/CRD.0b013e318093e43a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1097%2FCRD.0b013e318093e43a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=17700383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BD2svovFensg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=242-256&author=R.+Sepehrdadauthor=W.+H.+Frishmanauthor=C.+T.+Stierauthor=D.+A.+Sica&title=Direct+inhibition+of+renin+as+a+cardiovascular+pharmacotherapy%3A+Focus+on+aliskiren&doi=10.1097%2FCRD.0b013e318093e43a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Direct inhibition of renin as a cardiovascular pharmacotherapy: focus on aliskiren</span></div><div class="casAuthors">Sepehrdad Reza; Frishman William H; Stier Charles T Jr; Sica Domenic A</div><div class="citationInfo"><span class="NLM_cas:title">Cardiology in review</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">242-56</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The important role of renin-angiotensin-aldosterone system blockade in the treatment of systemic hypertension, heart failure, diabetic kidney disease, and atherogenesis has been clearly established.  The theoretical therapeutic advantages for inhibiting the detrimental effects of the renin-angiotensin system at its most upstream point have served as the impetus for the development of renin inhibitors.  The advent of aliskiren, the first in a novel class of orally active, nonpeptide, highly specific, human renin inhibitors, provides a new modality in the armamentarium of renin-angiotensin system antagonists.  Studies in marmosets and rats demonstrated that aliskiren reduced blood pressure in a dose-dependent manner and is highly efficacious in blocking plasma renin activity with parallel reductions in the levels of the other downstream constituents of the renin-angiotensin system.  Clinical trials in hypertensive patients have confirmed these benefits with aliskiren whose blood pressure-lowering efficacy is similar to or better than those of standard therapeutic doses of enalapril, losartan, irbesartan, and hydrochlorothiazide.  Aliskiren is well tolerated, with few reported adverse effects even at the highest doses tested.  Given the established beneficial effects of angiotensin-converting enzyme inhibitors and angiotensin receptor blockers in the treatment of cardiovascular and renovascular diseases, future studies may further elucidate a similar protective role for aliskiren both as a monotherapy and as part of a combination therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQE6VB-Pf2cTkBuFdjgVNKSfW6udTcc2ea0t2MdGviTdbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svovFensg%253D%253D&md5=add28099fe5ccc184b21059b89a4f42a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1097%2FCRD.0b013e318093e43a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCRD.0b013e318093e43a%26sid%3Dliteratum%253Aachs%26aulast%3DSepehrdad%26aufirst%3DR.%26aulast%3DFrishman%26aufirst%3DW.%2BH.%26aulast%3DStier%26aufirst%3DC.%2BT.%26aulast%3DSica%26aufirst%3DD.%2BA.%26atitle%3DDirect%2520inhibition%2520of%2520renin%2520as%2520a%2520cardiovascular%2520pharmacotherapy%253A%2520Focus%2520on%2520aliskiren%26jtitle%3DCardiol.%2520Rev.%26date%3D2007%26volume%3D15%26spage%3D242%26epage%3D256%26doi%3D10.1097%2FCRD.0b013e318093e43a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belunis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowther, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danho, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goggin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remiszewski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuglstatter, A.</span></span> <span> </span><span class="NLM_article-title">The structure of XIAP BIR2: Understanding the selectivity of the BIR domains</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1717</span>– <span class="NLM_lpage">1725</span>, <span class="refDoi"> DOI: 10.1107/S0907444913016284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1107%2FS0907444913016284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23999295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlKrsLnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1717-1725&author=C.+Lukacsauthor=C.+Belunisauthor=R.+Crowtherauthor=W.+Danhoauthor=L.+Gaoauthor=B.+Gogginauthor=C.+A.+Jansonauthor=S.+Liauthor=S.+Remiszewskiauthor=A.+Schuttauthor=M.+K.+Thakurauthor=S.+K.+Singhauthor=S.+Swaminathanauthor=R.+Pandeyauthor=R.+Tyagiauthor=R.+Gosuauthor=A.+V.+Kamathauthor=A.+Kuglstatter&title=The+structure+of+XIAP+BIR2%3A+Understanding+the+selectivity+of+the+BIR+domains&doi=10.1107%2FS0907444913016284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of XIAP BIR2: understanding the selectivity of the BIR domains</span></div><div class="casAuthors">Lukacs, Christine; Belunis, Charles; Crowther, Robert; Danho, Waleed; Gao, Lin; Goggin, Barry; Janson, Cheryl A.; Li, Shirley; Remiszewski, Stacy; Schutt, Andrew; Thakur, Manish K.; Singh, Saroj K.; Swaminathan, Srinivasan; Pandey, Rajat; Tyagi, Rajiv; Gosu, Ramachandraiah; Kamath, Ajith V.; Kuglstatter, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1717-1725</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">XIAP, a member of the inhibitor of apoptosis family of proteins, is a crit. regulator of apoptosis.  Inhibition of the BIR domain-caspase interaction is a promising approach towards treating cancer.  Previous work has been directed towards inhibiting the BIR3-caspase-9 interaction, which blocks the intrinsic apoptotic pathway; selectively inhibiting the BIR2-caspase-3 interaction would also block the extrinsic pathway.  The BIR2 domain of XIAP has successfully been crystd.; peptides and small-mol. inhibitors can be soaked into these crystals, which diffract to high resoln.  Here, the BIR2 apo crystal structure and the structures of five BIR2-tetrapeptide complexes are described.  The structural flexibility obsd. on comparing these structures, along with a comparison with XIAP BIR3, affords an understanding of the structural elements that drive selectivity between BIR2 and BIR3 and which can be used to design BIR2-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6-upk5-drzbVg90H21EOLACvtfcHk0liFa8l4wuSuaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlKrsLnK&md5=deebf6694f2e53929d7c6ec4e763cf14</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444913016284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913016284%26sid%3Dliteratum%253Aachs%26aulast%3DLukacs%26aufirst%3DC.%26aulast%3DBelunis%26aufirst%3DC.%26aulast%3DCrowther%26aufirst%3DR.%26aulast%3DDanho%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DGoggin%26aufirst%3DB.%26aulast%3DJanson%26aufirst%3DC.%2BA.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DRemiszewski%26aufirst%3DS.%26aulast%3DSchutt%26aufirst%3DA.%26aulast%3DThakur%26aufirst%3DM.%2BK.%26aulast%3DSingh%26aufirst%3DS.%2BK.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DPandey%26aufirst%3DR.%26aulast%3DTyagi%26aufirst%3DR.%26aulast%3DGosu%26aufirst%3DR.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DKuglstatter%26aufirst%3DA.%26atitle%3DThe%2520structure%2520of%2520XIAP%2520BIR2%253A%2520Understanding%2520the%2520selectivity%2520of%2520the%2520BIR%2520domains%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1717%26epage%3D1725%26doi%3D10.1107%2FS0907444913016284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span> <span> </span><span class="NLM_article-title">Inhibitor of apoptosis protein (IAP) antagonists in anticancer agent discovery: Current status and perspectives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01668</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01668" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=H.+Congauthor=L.+Xuauthor=Y.+Wuauthor=Z.+Quauthor=T.+Bianauthor=W.+Zhangauthor=C.+Xingauthor=C.+Zhuang&title=Inhibitor+of+apoptosis+protein+%28IAP%29+antagonists+in+anticancer+agent+discovery%3A+Current+status+and+perspectives&doi=10.1021%2Facs.jmedchem.8b01668"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01668%26sid%3Dliteratum%253Aachs%26aulast%3DCong%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DQu%26aufirst%3DZ.%26aulast%3DBian%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXing%26aufirst%3DC.%26aulast%3DZhuang%26aufirst%3DC.%26atitle%3DInhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529%2520antagonists%2520in%2520anticancer%2520agent%2520discovery%253A%2520Current%2520status%2520and%2520perspectives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26doi%3D10.1021%2Facs.jmedchem.8b01668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pargellis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churchill, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirillo, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grob, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickey, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pav, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regan, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of p38 map kinase by utilizing a novel allosteric binding site</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1038/nsb770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnsb770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11896401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD38Xis1Kisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=268-272&author=C.+Pargellisauthor=L.+Tongauthor=L.+Churchillauthor=P.+F.+Cirilloauthor=T.+Gilmoreauthor=A.+G.+Grahamauthor=P.+M.+Grobauthor=E.+R.+Hickeyauthor=N.+Mossauthor=S.+Pavauthor=J.+Regan&title=Inhibition+of+p38+map+kinase+by+utilizing+a+novel+allosteric+binding+site&doi=10.1038%2Fnsb770"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of p38 MAP kinase by utilizing a novel allosteric binding site</span></div><div class="casAuthors">Pargellis, Christopher; Tong, Liang; Churchill, Laurie; Cirillo, Pier F.; Gilmore, Thomas; Graham, Anne G.; Grob, Peter M.; Hickey, Eugene R.; Moss, Neil; Pav, Susan; Regan, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">268-272</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">The p38 MAP kinase plays a crucial role in regulating the prodn. of proinflammatory cytokines, such as tumor necrosis factor and interleukin-1.  Blocking this kinase may offer an effective therapy for treating many inflammatory diseases.  Here we report a new allosteric binding site for a diaryl urea class of highly potent and selective inhibitors against human p38 MAP kinase.  The formation of this binding site requires a large conformational change not obsd. previously for any of the protein Ser/Thr kinases.  This change is in the highly conserved Asp-Phe-Gly motif within the active site of the kinase.  Soln. studies demonstrate that this class of compds. has slow binding kinetics, consistent with the requirement for conformational change.  Improving interactions in this allosteric pocket, as well as establishing binding interactions in the ATP pocket, enhanced the affinity of the inhibitors by 12,000-fold.  One of the most potent compds. in this series, BIRB 796, has picomolar affinity for the kinase and low nanomolar inhibitory activity in cell culture.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqT8_QT7_pbYrVg90H21EOLACvtfcHk0lhh_GR5D8Y40Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xis1Kisbk%253D&md5=68850ecdbe08df4285ba38cfa2c93ea4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnsb770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb770%26sid%3Dliteratum%253Aachs%26aulast%3DPargellis%26aufirst%3DC.%26aulast%3DTong%26aufirst%3DL.%26aulast%3DChurchill%26aufirst%3DL.%26aulast%3DCirillo%26aufirst%3DP.%2BF.%26aulast%3DGilmore%26aufirst%3DT.%26aulast%3DGraham%26aufirst%3DA.%2BG.%26aulast%3DGrob%26aufirst%3DP.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DMoss%26aufirst%3DN.%26aulast%3DPav%26aufirst%3DS.%26aulast%3DRegan%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520p38%2520map%2520kinase%2520by%2520utilizing%2520a%2520novel%2520allosteric%2520binding%2520site%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D268%26epage%3D272%26doi%3D10.1038%2Fnsb770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Millan, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrows, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butcher, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairman, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilty, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemaitre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewthwaite, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahnke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathias, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philip, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefaniak, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeadon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, C.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of inhaled p38 inhibitors for the treatment of chronic obstructive pulmonary disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7797</span>– <span class="NLM_lpage">7814</span>, <span class="refDoi"> DOI: 10.1021/jm200677b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200677b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlOqu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7797-7814&author=D.+S.+Millanauthor=M.+E.+Bunnageauthor=J.+L.+Burrowsauthor=K.+J.+Butcherauthor=P.+G.+Doddauthor=T.+J.+Evansauthor=D.+A.+Fairmanauthor=S.+J.+Hughesauthor=I.+C.+Kiltyauthor=A.+Lemaitreauthor=R.+A.+Lewthwaiteauthor=A.+Mahnkeauthor=J.+P.+Mathiasauthor=J.+Philipauthor=R.+T.+Smithauthor=M.+H.+Stefaniakauthor=M.+Yeadonauthor=C.+Phillips&title=Design+and+synthesis+of+inhaled+p38+inhibitors+for+the+treatment+of+chronic+obstructive+pulmonary+disease&doi=10.1021%2Fjm200677b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Inhaled p38 Inhibitors for the Treatment of Chronic Obstructive Pulmonary Disease</span></div><div class="casAuthors">Millan, David S.; Bunnage, Mark E.; Burrows, Jane L.; Butcher, Kenneth J.; Dodd, Peter G.; Evans, Timothy J.; Fairman, David A.; Hughes, Samantha J.; Kilty, Iain C.; Lemaitre, Arnaud; Lewthwaite, Russell A.; Mahnke, Axel; Mathias, John P.; Philip, James; Smith, Robert T.; Stefaniak, Mark H.; Yeadon, Michael; Phillips, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7797-7814</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper describes the identification and optimization of a novel series of DFG-out binding p38 inhibitors as inhaled agents for the treatment of chronic obstructive pulmonary disease.  Structure based drug design and "inhalation by design" principles have been applied to the optimization of the lead series exemplified by compd. I.  Analogs have been designed to be potent and selective for p38, with an emphasis on slow enzyme dissocn. kinetics to deliver prolonged lung p38 inhibition.  Pharmacokinetic properties were tuned with high intrinsic clearance and low oral bioavailability in mind, to minimize systemic exposure and reduce systemically driven adverse events.  High CYP mediated clearance and glucuronidation were targeted to achieve high intrinsic clearance coupled with multiple routes of clearance to minimize drug-drug interactions.  Furthermore, pharmaceutical properties such as stability, crystallinity, and soly. were considered to ensure compatibility with a dry powder inhaler.  P38 Inhibitor II (PF-03715455) was subsequently identified as a clin. candidate from this series with efficacy and safety profiles confirming its potential as an inhaled agent for the treatment of COPD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzy5ivwDBL6rVg90H21EOLACvtfcHk0lghBu82BHGAvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlOqu73E&md5=9bc6cccf04a78831f50fb402bf7cb4b1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fjm200677b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200677b%26sid%3Dliteratum%253Aachs%26aulast%3DMillan%26aufirst%3DD.%2BS.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBurrows%26aufirst%3DJ.%2BL.%26aulast%3DButcher%26aufirst%3DK.%2BJ.%26aulast%3DDodd%26aufirst%3DP.%2BG.%26aulast%3DEvans%26aufirst%3DT.%2BJ.%26aulast%3DFairman%26aufirst%3DD.%2BA.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DKilty%26aufirst%3DI.%2BC.%26aulast%3DLemaitre%26aufirst%3DA.%26aulast%3DLewthwaite%26aufirst%3DR.%2BA.%26aulast%3DMahnke%26aufirst%3DA.%26aulast%3DMathias%26aufirst%3DJ.%2BP.%26aulast%3DPhilip%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DR.%2BT.%26aulast%3DStefaniak%26aufirst%3DM.%2BH.%26aulast%3DYeadon%26aufirst%3DM.%26aulast%3DPhillips%26aufirst%3DC.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520inhaled%2520p38%2520inhibitors%2520for%2520the%2520treatment%2520of%2520chronic%2520obstructive%2520pulmonary%2520disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7797%26epage%3D7814%26doi%3D10.1021%2Fjm200677b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Norman, P.</span></span> <span> </span><span class="NLM_article-title">Investigational p38 inhibitors for the treatment of chronic obstructive pulmonary disease</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">392</span>, <span class="refDoi"> DOI: 10.1517/13543784.2015.1006358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1517%2F13543784.2015.1006358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25599809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisVKrtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2015&pages=383-392&author=P.+Norman&title=Investigational+p38+inhibitors+for+the+treatment+of+chronic+obstructive+pulmonary+disease&doi=10.1517%2F13543784.2015.1006358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Investigational p38 inhibitors for the treatment of chronic obstructive pulmonary disease</span></div><div class="casAuthors">Norman, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">383-392</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Introduction: The p38 protein kinases, in particular p38α and p38β, regulate the prodn. of multiple inflammatory mediators.  Consequentially, considerable effort has been focused on trying to develop p38 inhibitors for the treatment of inflammatory diseases.  Some 20 p38 inhibitors have progressed to clin. development, mostly for the treatment of rheumatoid arthritis, but with little success.  Increasingly, interest has turned to their use in other indications and notably chronic obstructive pulmonary disease (COPD).  Areas covered: In this review, the author discusses the eight p38 inhibitors that have been clin. evaluated.  Acumapimod is the only one of four orally delivered inhibitors that remains in active development while Phase II results of PH-797804 and losmapimod are compared.  The activity of two inhibitors designed for inhaled delivery, RV-568 and PF-03715455, is compared but little is known about AZD-7624 or the discontinued GSK-610677.  Expert opinion: Results from animal models provide a clear rationale for developing p38 inhibitors for COPD, and appear to be (partially) validated by the efficacy seen with PH-797804 and losmapimod.  Inhaled delivery provides the opportunity to enhance p38 inhibition in the lung while reducing unwanted systemic effects of p38 inhibition.  Validation of this hypothesis should come from the results of the recently completed Phase II study with RV-568.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKUbGhXQAMzrVg90H21EOLACvtfcHk0lghBu82BHGAvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisVKrtLk%253D&md5=97485678de1e2d15bf1ace5139cba51e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1517%2F13543784.2015.1006358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.2015.1006358%26sid%3Dliteratum%253Aachs%26aulast%3DNorman%26aufirst%3DP.%26atitle%3DInvestigational%2520p38%2520inhibitors%2520for%2520the%2520treatment%2520of%2520chronic%2520obstructive%2520pulmonary%2520disease%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2015%26volume%3D24%26spage%3D383%26epage%3D392%26doi%3D10.1517%2F13543784.2015.1006358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrick, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moult, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyck, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakser, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodak, S. J.</span></span> <span> </span><span class="NLM_article-title">CAPRI: A critical assessment of predicted interactions</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1002/prot.10381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1002%2Fprot.10381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=12784359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkslahsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=2-9&author=J.+Janinauthor=K.+Henrickauthor=J.+Moultauthor=L.+T.+Eyckauthor=M.+J.+Sternbergauthor=S.+Vajdaauthor=I.+Vakserauthor=S.+J.+Wodak&title=CAPRI%3A+A+critical+assessment+of+predicted+interactions&doi=10.1002%2Fprot.10381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">CAPRI: A critical assessment of predicted interactions</span></div><div class="casAuthors">Janin, Joel; Henrick, Kim; Moult, John; Ten Eyck, Lynn; Sternberg, Michael J. E.; Vajda, Sandor; Vakser, Ilya; Wodak, Shoshana J.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-9</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  CAPRI is a community wide expt. to assess the capacity of protein-docking methods to predict protein-protein interactions.  Nineteen groups participated in rounds 1 and 2 of CAPRI and submitted blind structure predictions for seven protein-protein complexes based on the known structure of the component proteins.  The predictions were compared to the unpublished X-ray structures of the complexes.  We describe here the motivations for launching CAPRI, the rules that we applied to select targets and run the expt., and some conclusions that can already be drawn.  The results stress the need for new scoring functions and for methods handling the conformation changes that were obsd. in some of the target systems.  CAPRI has already been a powerful drive for the community of computational biologists who development docking algorithms.  We hope that this issue of Proteins will also be of interest to the community of structural biologists, which we call upon to provide new targets for future rounds of CAPRI, and to all mol. biologists who view protein-protein recognition as an essential process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyYaxW6AQ0N7Vg90H21EOLACvtfcHk0lh0KOssALX_eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkslahsr4%253D&md5=5ae9cf6222afcfebf272141679659289</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fprot.10381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10381%26sid%3Dliteratum%253Aachs%26aulast%3DJanin%26aufirst%3DJ.%26aulast%3DHenrick%26aufirst%3DK.%26aulast%3DMoult%26aufirst%3DJ.%26aulast%3DEyck%26aufirst%3DL.%2BT.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DVakser%26aufirst%3DI.%26aulast%3DWodak%26aufirst%3DS.%2BJ.%26atitle%3DCAPRI%253A%2520A%2520critical%2520assessment%2520of%2520predicted%2520interactions%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2003%26volume%3D52%26spage%3D2%26epage%3D9%26doi%3D10.1002%2Fprot.10381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishima, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freedberg, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torchia, D. A.</span></span> <span> </span><span class="NLM_article-title">Flap opening and dimer-interface flexibility in the free and inhibitor-bound HIV protease, and their implications for function</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1047</span>– <span class="NLM_lpage">1055</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(99)80172-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2FS0969-2126%2899%2980172-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10508781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADyaK1MXmtlartL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1999&pages=1047-1055&author=R.+Ishimaauthor=D.+I.+Freedbergauthor=Y.+X.+Wangauthor=J.+M.+Louisauthor=D.+A.+Torchia&title=Flap+opening+and+dimer-interface+flexibility+in+the+free+and+inhibitor-bound+HIV+protease%2C+and+their+implications+for+function&doi=10.1016%2FS0969-2126%2899%2980172-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Flap opening and dimer-interface flexibility in the free and inhibitor-bound HIV protease, and their implications for function</span></div><div class="casAuthors">Ishima, Rieko; Freedberg, Daron I.; Wang, Yun-Xing; Louis, John M.; Torchia, Dennis A.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1047-1055</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Publications</span>)
        </div><div class="casAbstract">Background: 1H and 15N transverse relaxation measurements on perdeuterated proteins are ideally suited for detecting backbone conformational fluctuations on the millisecond-microsecond timescale.  The identification of conformational exchange on this timescale by measuring the relaxation of both 1H and 15N holds great promise for the elucidation of functionally relevant conformational changes in proteins.  Results: We measured the transverse 1H and 15N relaxation rates of backbone amides of HIV-1 protease in its free and inhibitor-bound forms.  An anal. of these rates, obtained as a function of the effective rotating frame field, provided information about the timescale of structural fluctuations in several regions of the protein.  The flaps that cover the active site of the inhibitor-bound protein undergo significant changes of backbone (Φψ) angles, on the 100 μs timescale, in the free protein.  In addn., the intermonomer β-sheet interface of the bound form, which from protease structure studies appears to be rigid, was found to fluctuate on the millisecond timescale.  Conclusions: We present a working model of the flap-opening mechanism in free HIV-1 protease which involves a transition from a semi-open to an open conformation that is facilitated by interaction of the Phe53 ring with the substrate.  We also identify a surprising fluctuation of the β-sheet intermonomer interface that suggests a structural requirement for maturation of the protease.  Thus, slow conformational fluctuations identified by 1H and 15N transverse relaxation measurements can be related to the biol. functions of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFrYx5b8IAdrVg90H21EOLACvtfcHk0lijbgkR3U0Prg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmtlartL8%253D&md5=1ab515e23efcd5eb310db489d0e78351</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2899%2980172-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252899%252980172-5%26sid%3Dliteratum%253Aachs%26aulast%3DIshima%26aufirst%3DR.%26aulast%3DFreedberg%26aufirst%3DD.%2BI.%26aulast%3DWang%26aufirst%3DY.%2BX.%26aulast%3DLouis%26aufirst%3DJ.%2BM.%26aulast%3DTorchia%26aufirst%3DD.%2BA.%26atitle%3DFlap%2520opening%2520and%2520dimer-interface%2520flexibility%2520in%2520the%2520free%2520and%2520inhibitor-bound%2520HIV%2520protease%252C%2520and%2520their%2520implications%2520for%2520function%26jtitle%3DStructure%26date%3D1999%26volume%3D7%26spage%3D1047%26epage%3D1055%26doi%3D10.1016%2FS0969-2126%2899%2980172-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seo, Y. H.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors to disrupt protein-protein interactions of heat shock protein 90 chaperone machinery</span>. <i>J. Cancer Prev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.15430/JCP.2015.20.1.5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.15430%2FJCP.2015.20.1.5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=25853099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BC2Mjit1WqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=5-11&author=Y.+H.+Seo&title=Small+molecule+inhibitors+to+disrupt+protein-protein+interactions+of+heat+shock+protein+90+chaperone+machinery&doi=10.15430%2FJCP.2015.20.1.5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors to Disrupt Protein-protein Interactions of Heat Shock Protein 90 Chaperone Machinery</span></div><div class="casAuthors">Seo Young Ho</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cancer prevention</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-11</span>
        ISSN:<span class="NLM_cas:issn">2288-3649</span>.
    </div><div class="casAbstract">Heat shock protein 90 (Hsp90) is an adenosine triphosphate dependent molecular chaperone in eukaryotic cells that regulates the activation and maintenance of numerous regulatory and signaling proteins including epidermal growth factor receptor, human epidermal growth factor receptor 2, mesenchymal-epithelial transition factor, cyclin-dependent kinase-4, protein kinase B, hypoxia-inducible factor 1α, and matrix metalloproteinase-2.  Since many of Hsp90 clients are oncogenic proteins, Hsp90 has become an attractive therapeutic target for treatment of cancer.  To discover small molecule inhibitors targeting Hsp90 chaperone machinery, several strategies have been employed, which results in three classes of inhibitors such as N-terminal inhibitors, C-terminal inhibitors, and inhibitors disrupting protein-protein interactions of Hsp90 chaperone machinery.  Developing small molecule inhibitors that modulate protein-protein interactions of Hsp90 is a challenging task, although it offers many alternative opportunities for therapeutic intervention.  The lack of well-defined binding pocket and starting points for drug design challenges medicinal chemists to discover small molecule inhibitors disrupting protein-protein interactions of Hsp90.  The present review will focus on the current studies on small molecule inhibitors disrupting protein-protein interactions of Hsp90 chaperone machinery, provide biological background on the structure, function and mechanism of Hsp90's protein-protein interactions, and discuss the challenges and promise of its small molecule modulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZvHVQAn27dy86Js-BCLiifW6udTcc2eblZtDV0JzzWbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mjit1WqsA%253D%253D&md5=6df1b2f3b2fee148861243d8ff6c8fef</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.15430%2FJCP.2015.20.1.5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15430%252FJCP.2015.20.1.5%26sid%3Dliteratum%253Aachs%26aulast%3DSeo%26aufirst%3DY.%2BH.%26atitle%3DSmall%2520molecule%2520inhibitors%2520to%2520disrupt%2520protein-protein%2520interactions%2520of%2520heat%2520shock%2520protein%252090%2520chaperone%2520machinery%26jtitle%3DJ.%2520Cancer%2520Prev.%26date%3D2015%26volume%3D20%26spage%3D5%26epage%3D11%26doi%3D10.15430%2FJCP.2015.20.1.5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stebbins, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartl, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavletich, N. P.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of an Hsp90-geldanamycin complex: Targeting of a protein chaperone by an antitumor agent</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">239</span>– <span class="NLM_lpage">250</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)80203-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2FS0092-8674%2800%2980203-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=9108479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADyaK2sXislCitLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1997&pages=239-250&author=C.+E.+Stebbinsauthor=A.+A.+Russoauthor=C.+Schneiderauthor=N.+Rosenauthor=F.+U.+Hartlauthor=N.+P.+Pavletich&title=Crystal+structure+of+an+Hsp90-geldanamycin+complex%3A+Targeting+of+a+protein+chaperone+by+an+antitumor+agent&doi=10.1016%2FS0092-8674%2800%2980203-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of an Hsp90-geldanamycin complex: targeting of a protein chaperone by an antitumor agent</span></div><div class="casAuthors">Stebbins, Charles E.; Russo, Alicia A.; Schneider, Christine; Rosen, Neal; Hartl, F. Ulrich; Pavletich, Nikola P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-250</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Hsp90 chaperone is required for the activation of several families of eukaryotic protein kinases and nuclear hormone receptors, many of which are proto-oncogenic and play a prominent role in cancer.  The geldanamycin antibiotic has antiproliferative and anti-tumor effects, as it binds to Hsp90, inhibits the Hsp90-mediated conformational maturation/refolding reaction, and results in the degrdn. of Hsp90 substrates.  The structure of the geldanamycin-binding domain of Hsp90 (residues 9-232) reveals a pronounced pocket, 15 Å deep. that is highly conserved across species.  Geldanamycin binds inside this pocket, adopting a compact structure similar to that of a polypeptide chain in a turn conformation.  This, and the pocket's similarity to substrate-binding sites, suggest that the pocket binds a portion of the polypeptide substrate and participates in the conformational maturation/refolding reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvnmleAOFITbVg90H21EOLACvtfcHk0lj4AtkQNg48MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXislCitLo%253D&md5=54a56450c509727e070f4db20a9d7b1a</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2980203-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252980203-2%26sid%3Dliteratum%253Aachs%26aulast%3DStebbins%26aufirst%3DC.%2BE.%26aulast%3DRusso%26aufirst%3DA.%2BA.%26aulast%3DSchneider%26aufirst%3DC.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DHartl%26aufirst%3DF.%2BU.%26aulast%3DPavletich%26aufirst%3DN.%2BP.%26atitle%3DCrystal%2520structure%2520of%2520an%2520Hsp90-geldanamycin%2520complex%253A%2520Targeting%2520of%2520a%2520protein%2520chaperone%2520by%2520an%2520antitumor%2520agent%26jtitle%3DCell%26date%3D1997%26volume%3D89%26spage%3D239%26epage%3D250%26doi%3D10.1016%2FS0092-8674%2800%2980203-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trendowski, M.</span></span> <span> </span><span class="NLM_article-title">Pu-h71: An improvement on nature’s solutions to oncogenic Hsp90 addiction</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">216</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.phrs.2015.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=26117427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFSjt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=202-216&author=M.+Trendowski&title=Pu-h71%3A+An+improvement+on+nature%E2%80%99s+solutions+to+oncogenic+Hsp90+addiction&doi=10.1016%2Fj.phrs.2015.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">PU-H71: An improvement on nature's solutions to oncogenic Hsp90 addiction</span></div><div class="casAuthors">Trendowski, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">202-216</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Despite recent advances in precision medicine, many mol.-based antineoplastic agents do not potentiate sustainable long term remissions, warranting the investigation of novel therapeutic strategies.  Heat shock protein 90 (Hsp90) is a mol. chaperone that not only has oncogenic properties, but also has distinct expression profiles in malignant and normal cells, providing a rational strategy to attain preferential damage.  Prior attempts to target Hsp90 with natural product-based compds. have been hampered by their assocd. off target toxicities, suggesting that novel, fully synthetic inhibitors may be required to achieve the specificity necessary for therapeutic efficacy.  Therefore, this review highlights the antineoplastic potential of PU-H71 (8-[(6-iodo-1,3-benzodioxol-5-yl)sulfanyl]-9-[3-(propan-2-ylamino)propyl]purin-6-amine), a novel purine based analog that has shown efficacy in many preclin. models of malignancy, and is now under clin. examn.  In addn., the review suggests potential concomitant therapeutic approaches that may be particularly beneficial to mol.-based, as well as traditional cytotoxic cancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ-CFQcXC_6bVg90H21EOLACvtfcHk0lhWcQrbjWq1uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFSjt7fK&md5=bc9029f7ca82f2fff0fed0ee772bcf10</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DTrendowski%26aufirst%3DM.%26atitle%3DPu-h71%253A%2520An%2520improvement%2520on%2520nature%25E2%2580%2599s%2520solutions%2520to%2520oncogenic%2520Hsp90%2520addiction%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D202%26epage%3D216%26doi%3D10.1016%2Fj.phrs.2015.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lhWcQrbjWq1uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasater, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+Mullerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+kinase-bromodomain+inhibitors+for+rationally+designed+polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0lhCVkEYDNhNEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520for%2520rationally%2520designed%2520polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of bivalent BET inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1096</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnchembio.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=27775715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1089-1096&author=M.+Tanakaauthor=J.+M.+Robertsauthor=H.+S.+Seoauthor=A.+Souzaauthor=J.+Paulkauthor=T.+G.+Scottauthor=S.+L.+DeAngeloauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Design+and+characterization+of+bivalent+BET+inhibitors&doi=10.1038%2Fnchembio.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Design and characterization of bivalent BET inhibitors</span></div><div class="casAuthors">Tanaka, Minoru; Roberts, Justin M.; Seo, Hyuk-Soo; Souza, Amanda; Paulk, Joshiawa; Scott, Thomas G.; DeAngelo, Stephen L.; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1089-1096</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cellular signaling is often propagated by multivalent interactions.  Multivalency creates avidity, allowing stable biophys. recognition.  Multivalency is an attractive strategy for achieving potent binding to protein targets, as the affinity of bivalent ligands is often greater than the sum of monovalent affinities.  The bromodomain and extraterminal domain (BET) family of transcriptional coactivators features tandem bromodomains through which BET proteins bind acetylated histones and transcription factors.  All reported antagonists of the BET protein BRD4 bind in a monovalent fashion.  Here the authors describe, to the knowledge for the first time, a bivalent BET bromodomain inhibitor-MT1-which has unprecedented potency.  Biophys. and biochem. studies suggest MT1 is an intramol. bivalent BRD4 binder that is more than 100-fold more potent, in cellular assays, than the corresponding monovalent antagonist, JQ1.  MT1 significantly (P < 0.05) delayed leukemia progression in mice, as compared to JQ1.  These data qualify a powerful chem. probe for BET bromodomains and a rationale for further development of multidomain inhibitors of epigenetic reader proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQwc6X0VYYbVg90H21EOLACvtfcHk0ljMbXbGJR7HAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurrM&md5=7319203dac409d5dee3c580b49fcf185</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2209%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSeo%26aufirst%3DH.%2BS.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDesign%2520and%2520characterization%2520of%2520bivalent%2520BET%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D1089%26epage%3D1096%26doi%3D10.1038%2Fnchembio.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frkic, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blayo, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pukala, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamenecka, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruning, J. B.</span></span> <span> </span><span class="NLM_article-title">PPARγ in complex with an antagonist and inverse agonist: A tumble and trap mechanism of the activation helix</span>. <i>iScience</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1016/j.isci.2018.06.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.isci.2018.06.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=30123887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVajt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2018&pages=69-79&author=R.+L.+Frkicauthor=A.+C.+Marshallauthor=A.+L.+Blayoauthor=T.+L.+Pukalaauthor=T.+M.+Kameneckaauthor=P.+R.+Griffinauthor=J.+B.+Bruning&title=PPAR%CE%B3+in+complex+with+an+antagonist+and+inverse+agonist%3A+A+tumble+and+trap+mechanism+of+the+activation+helix&doi=10.1016%2Fj.isci.2018.06.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PPARγ in Complex with an Antagonist and Inverse Agonist: a Tumble and Trap Mechanism of the Activation Helix</span></div><div class="casAuthors">Frkic, Rebecca L.; Marshall, Andrew C.; Blayo, Anne-Laure; Pukala, Tara L.; Kamenecka, Theodore M.; Griffin, Patrick R.; Bruning, John B.</div><div class="citationInfo"><span class="NLM_cas:title">iScience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">69-79</span>CODEN:
                <span class="NLM_cas:coden">ISCICE</span>;
        ISSN:<span class="NLM_cas:issn">2589-0042</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Peroxisome proliferator activated receptor γ (PPARγ) is a nuclear receptor and target for antidiabetics that increase insulin sensitivity.  Owing to the side effects of PPARγ full agonists, research has recently focused on non-activating ligands of PPARγ, which increase insulin sensitivity with decreased side effects.  Here, we present the crystal structures of inverse agonist SR10171 and a chem. related antagonist SR11023 bound to the PPARγ ligand-binding domain, revealing an allosteric switch in the activation helix, helix 12 (H12), forming an antagonist conformation in the receptor.  H12 interacts with the antagonists to become fixed in an alternative location.  Native mass spectrometry indicates that this prevents contacts with coactivator peptides and allows binding of corepressor peptides.  Antagonists of related nuclear receptors act to sterically prevent the active configuration of H12, whereas these antagonists of PPARγ alternatively trap H12 in an inactive configuration, which we have termed the tumble and trap mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ-nTSIgus9bVg90H21EOLACvtfcHk0ljbm4LsTk044g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVajt70%253D&md5=69568acac5f75ae39807ab8460b31f5a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.isci.2018.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.isci.2018.06.012%26sid%3Dliteratum%253Aachs%26aulast%3DFrkic%26aufirst%3DR.%2BL.%26aulast%3DMarshall%26aufirst%3DA.%2BC.%26aulast%3DBlayo%26aufirst%3DA.%2BL.%26aulast%3DPukala%26aufirst%3DT.%2BL.%26aulast%3DKamenecka%26aufirst%3DT.%2BM.%26aulast%3DGriffin%26aufirst%3DP.%2BR.%26aulast%3DBruning%26aufirst%3DJ.%2BB.%26atitle%3DPPAR%25CE%25B3%2520in%2520complex%2520with%2520an%2520antagonist%2520and%2520inverse%2520agonist%253A%2520A%2520tumble%2520and%2520trap%2520mechanism%2520of%2520the%2520activation%2520helix%26jtitle%3DiScience%26date%3D2018%26volume%3D5%26spage%3D69%26epage%3D79%26doi%3D10.1016%2Fj.isci.2018.06.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taygerly, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGee, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubenstein, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houze, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cushing, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankmoelle, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Learned, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timmermans, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoolen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kearney, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiszmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biermann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aramaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinkai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furukawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of INT131: A selective PPARγ modulator that enhances insulin sensitivity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">979</span>– <span class="NLM_lpage">992</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2012.11.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.bmc.2012.11.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23294830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkt1Witg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=979-992&author=J.+P.+Taygerlyauthor=L.+R.+McGeeauthor=S.+M.+Rubensteinauthor=J.+B.+Houzeauthor=T.+D.+Cushingauthor=Y.+Liauthor=A.+Motaniauthor=J.+L.+Chenauthor=W.+Frankmoelleauthor=G.+Yeauthor=M.+R.+Learnedauthor=J.+Jaenauthor=S.+Miaoauthor=P.+B.+Timmermansauthor=M.+Thoolenauthor=P.+Kearneyauthor=J.+Flygareauthor=H.+Beckmannauthor=J.+Weiszmannauthor=M.+Lindstromauthor=N.+Walkerauthor=J.+Liuauthor=D.+Biermannauthor=Z.+Wangauthor=A.+Hagiwaraauthor=T.+Iidaauthor=H.+Aramakiauthor=Y.+Kitaoauthor=H.+Shinkaiauthor=N.+Furukawaauthor=J.+Nishiuauthor=M.+Nakamura&title=Discovery+of+INT131%3A+A+selective+PPAR%CE%B3+modulator+that+enhances+insulin+sensitivity&doi=10.1016%2Fj.bmc.2012.11.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of INT131: A selective PPARγ modulator that enhances insulin sensitivity</span></div><div class="casAuthors">Taygerly, Joshua P.; McGee, Lawrence R.; Rubenstein, Steven M.; Houze, Jonathan B.; Cushing, Timothy D.; Li, Yang; Motani, Alykhan; Chen, Jin-Long; Frankmoelle, Walter; Ye, Guosen; Learned, Marc R.; Jaen, Juan; Miao, Shichang; Timmermans, Pieter B.; Thoolen, Martin; Kearney, Patrick; Flygare, John; Beckmann, Holger; Weiszmann, Jennifer; Lindstrom, Michelle; Walker, Nigel; Liu, Jinsong; Biermann, Donna; Wang, Zhulun; Hagiwara, Atsushi; Iida, Tetsuya; Aramaki, Hisateru; Kitao, Yuki; Shinkai, Hisashi; Furukawa, Noboru; Nishiu, Jun; Nakamura, Motonao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">979-992</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">PPARγ is a member of the nuclear hormone receptor family and plays a key role in the regulation of glucose homeostasis.  This study describes the discovery of a novel chem. class of diarylsulfonamide partial agonists that act as selective PPARγ modulators (SPPARγMs) and display a unique pharmacol. profile compared to the thiazolidinedione (TZD) class of PPARγ full agonists.  Herein we report the initial discovery of partial agonist (I) and the structure-activity relation studies that led to the selection of clin. compd. INT131 (II), a potent PPARγ partial agonist that displays robust glucose-lowering activity in rodent models of diabetes while exhibiting a reduced side-effects profile compared to marketed TZDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMnHgzkPStAbVg90H21EOLACvtfcHk0ljXOKMGMiwSIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkt1Witg%253D%253D&md5=7056be6ef3ae46eb8a3076dab838da20</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.11.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.11.058%26sid%3Dliteratum%253Aachs%26aulast%3DTaygerly%26aufirst%3DJ.%2BP.%26aulast%3DMcGee%26aufirst%3DL.%2BR.%26aulast%3DRubenstein%26aufirst%3DS.%2BM.%26aulast%3DHouze%26aufirst%3DJ.%2BB.%26aulast%3DCushing%26aufirst%3DT.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMotani%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DFrankmoelle%26aufirst%3DW.%26aulast%3DYe%26aufirst%3DG.%26aulast%3DLearned%26aufirst%3DM.%2BR.%26aulast%3DJaen%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DS.%26aulast%3DTimmermans%26aufirst%3DP.%2BB.%26aulast%3DThoolen%26aufirst%3DM.%26aulast%3DKearney%26aufirst%3DP.%26aulast%3DFlygare%26aufirst%3DJ.%26aulast%3DBeckmann%26aufirst%3DH.%26aulast%3DWeiszmann%26aufirst%3DJ.%26aulast%3DLindstrom%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DBiermann%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DHagiwara%26aufirst%3DA.%26aulast%3DIida%26aufirst%3DT.%26aulast%3DAramaki%26aufirst%3DH.%26aulast%3DKitao%26aufirst%3DY.%26aulast%3DShinkai%26aufirst%3DH.%26aulast%3DFurukawa%26aufirst%3DN.%26aulast%3DNishiu%26aufirst%3DJ.%26aulast%3DNakamura%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520INT131%253A%2520A%2520selective%2520PPAR%25CE%25B3%2520modulator%2520that%2520enhances%2520insulin%2520sensitivity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D979%26epage%3D992%26doi%3D10.1016%2Fj.bmc.2012.11.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Badrinarayan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. N.</span></span> <span> </span><span class="NLM_article-title">Rational approaches towards lead optimization of kinase inhibitors: The issue of specificity</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4714</span>– <span class="NLM_lpage">4738</span>, <span class="refDoi"> DOI: 10.2174/1381612811319260005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.2174%2F1381612811319260005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23260022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wns7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=4714-4738&author=P.+Badrinarayanauthor=G.+N.+Sastry&title=Rational+approaches+towards+lead+optimization+of+kinase+inhibitors%3A+The+issue+of+specificity&doi=10.2174%2F1381612811319260005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Approaches Towards Lead Optimization of Kinase Inhibitors: The Issue of Specificity</span></div><div class="casAuthors">Badrinarayan, Preethi; Sastry, G. Narahari</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">4714-4738</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases are one of the most popular classes of drug targets as they are involved in signal transduction pathways, which are wired through a phosphotransfer cascade and elicit a no. of important and essential physiol. responses.  Kinase specificity has emerged as one of the major issues to be addressed in drug discovery approaches.  In most kinases the active site is the ATP binding site and finding suitable hits which maximize the affinity of binding has been traditionally important to obtain the type I inhibitors.  While type I inhibitors have effective binding affinity more often than not they encounter side-effects usually assocd. with specificity.  Therefore in recent times it has become indispensable to optimize specificity for developing effective kinase inhibitors.  The review presents an overview of kinase drug discovery and the different strategies used to date for the design of kinase leads accounting for their success and failure.  A no. of strategies exploiting different aspects of kinases like allosteric site, size of the gatekeeper residue, DFG-loop, chemotype selectivity, non-covalent interactions, salt-bridge, solvation, etc. have been explored to circumvent the specificity problem in kinases.  The probable hot-spots in kinases having a propensity to bring in specificity have been delineated with special emphasis on the design of type II inhibitors with increased specificity from existing type I using fragment tailoring approach.  In this review we illustrate the current strategies by taking p38 MAP kinase as a model and expect that such strategies are general and can be extended to the other members of the kinase family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmsC1PsUKeGLVg90H21EOLACvtfcHk0lijQJtPG0B9EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wns7rM&md5=ff2b5c5f0fd16af8c72763db036fd3a3</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.2174%2F1381612811319260005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319260005%26sid%3Dliteratum%253Aachs%26aulast%3DBadrinarayan%26aufirst%3DP.%26aulast%3DSastry%26aufirst%3DG.%2BN.%26atitle%3DRational%2520approaches%2520towards%2520lead%2520optimization%2520of%2520kinase%2520inhibitors%253A%2520The%2520issue%2520of%2520specificity%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D4714%26epage%3D4738%26doi%3D10.2174%2F1381612811319260005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochhaus, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derendorf, H.</span></span> <span> </span><span class="NLM_article-title">Comparative pharmacology, bioavailability, pharmacokinetics, and pharmacodynamics of inhaled glucocorticosteroids</span>. <i>Immunol. Allergy Clin. North Am.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1016/j.iac.2005.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.iac.2005.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=16054538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BD2MzpsFymsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=469-488&author=M.+Hubnerauthor=G.+Hochhausauthor=H.+Derendorf&title=Comparative+pharmacology%2C+bioavailability%2C+pharmacokinetics%2C+and+pharmacodynamics+of+inhaled+glucocorticosteroids&doi=10.1016%2Fj.iac.2005.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative pharmacology, bioavailability, pharmacokinetics, and pharmacodynamics of inhaled glucocorticosteroids</span></div><div class="casAuthors">Hubner Melanie; Hochhaus Gunther; Derendorf Hartmut</div><div class="citationInfo"><span class="NLM_cas:title">Immunology and allergy clinics of North America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">469-88</span>
        ISSN:<span class="NLM_cas:issn">0889-8561</span>.
    </div><div class="casAbstract">A comparison of the pharmacodynamics and pharmacokinetics of inhaled corticosteroids is necessary for their assessment.  A good knowledge of these two aspects allows the optimization of efficacy and safety.The currently available inhaled corticosteroids already show some of the desired PK/PD parameters.  The local adverse effects are decreased as soon as the inhaled corticosteroid is administered as an inactive prodrug or shows a bet-ter lung deposition.  HFA-MDI beclomethasone dipropionate (BDP) and ciclesonide are two agents that illustrate this.  Low oral bioavailability, rapid systemic clearance, and high plasma protein binding can minimize systemic adverse effects.  Mometasone furoate, ciclesonide, and fluticasone propionate possess those characteristics.  The pulmonary efficacy is maximized by high lung deposition and long pulmonary residence times.  This effect can be achieved by slow dissolution in the lungs, as is the case for fluticasone propionate or lipid conjugation and has been shown for budesonide and ciclesonide.  Furthermore, the lung deposition depends on the inhalation device, the particle size, and the inhalation technique.  Therefore,improvement in the design of MDIs, DPIs, and nebulizers, and the development of more effective drug particles will lead to an optimized pulmonary targeting.  Much progress has been made in the treatment of asthma.  The available inhaled corticosteroids show a high safety profile and a good pulmonary selectivity.  Development of newer compounds showed that improvement is possible as the result of a complete understanding of the PK/PD concepts.  However,the introduction of further improved formulations with a better efficacy/safety profile will be difficult and protracted because the existing drugs are already highly efficient.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSix-AbkCPCpuqrn-c4Of3KfW6udTcc2eaGK8RtrfFAh7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MzpsFymsA%253D%253D&md5=867e4f0660f96aba8b7724afb594a39b</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.iac.2005.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.iac.2005.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DHubner%26aufirst%3DM.%26aulast%3DHochhaus%26aufirst%3DG.%26aulast%3DDerendorf%26aufirst%3DH.%26atitle%3DComparative%2520pharmacology%252C%2520bioavailability%252C%2520pharmacokinetics%252C%2520and%2520pharmacodynamics%2520of%2520inhaled%2520glucocorticosteroids%26jtitle%3DImmunol.%2520Allergy%2520Clin.%2520North%2520Am.%26date%3D2005%26volume%3D25%26spage%3D469%26epage%3D488%26doi%3D10.1016%2Fj.iac.2005.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, S. C.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of human calcineurin complexed with cyclosporin A and human cyclophilin</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">13522</span>– <span class="NLM_lpage">13526</span>, <span class="refDoi"> DOI: 10.1073/pnas.212504399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1073%2Fpnas.212504399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=12357034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVKms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=13522-13526&author=L.+Jinauthor=S.+C.+Harrison&title=Crystal+structure+of+human+calcineurin+complexed+with+cyclosporin+A+and+human+cyclophilin&doi=10.1073%2Fpnas.212504399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of human calcineurin complexed with cyclosporin A and human cyclophilin</span></div><div class="casAuthors">Jin, Lei; Harrison, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">13522-13526</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Calcineurin (Cn; protein phosphatase 2B), is an important participant in signaling pathways that activate T cells.  It is the target of the immunosuppressive drugs, cyclosporin A (CsA) and FK 506.  These drugs bind proteins known as cyclophilin (Cyp) and FK 506-binding protein, resp., and the drug-protein complexes in turn inhibit Cn.  Here, the authors report the crystal structure of a Cyp/CsA/Cn ternary complex, detd. to a resoln. of 3.1 Å.  Residues 3-9 of CsA, particularly N-methylleucine (Mle)-4 and Mle-6, and Trp-121 of Cyp were found to form a composite surface for interaction with Cn.  The hydrophobic interface buried 2 H-bonds.  The structure accounted clearly for the effects of mutations in Cn on CsA-resistance and for the way modifications of CsA alter immunosuppressive activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqspopSI-iZLLVg90H21EOLACvtfcHk0lgRghtd-XBzEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVKms7c%253D&md5=cff5598bce80f7a372582156f9e8619b</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1073%2Fpnas.212504399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.212504399%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHarrison%26aufirst%3DS.%2BC.%26atitle%3DCrystal%2520structure%2520of%2520human%2520calcineurin%2520complexed%2520with%2520cyclosporin%2520A%2520and%2520human%2520cyclophilin%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D13522%26epage%3D13526%26doi%3D10.1073%2Fpnas.212504399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, J. D.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Calcineurin is a common target of cyclophilin-cyclosporin A and FKBP-FK506 complexes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(91)90124-H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2F0092-8674%2891%2990124-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1715244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADyaK3MXlsV2qu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=1991&pages=807-815&author=J.+Liuauthor=J.+D.+Farmerauthor=W.+S.+Laneauthor=J.+Friedmanauthor=I.+Weissmanauthor=S.+L.+Schreiber&title=Calcineurin+is+a+common+target+of+cyclophilin-cyclosporin+A+and+FKBP-FK506+complexes&doi=10.1016%2F0092-8674%2891%2990124-H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Calcineurin is a common target of cyclophilin-cyclosporin A and FKBP-FK506 complexes</span></div><div class="casAuthors">Liu, Jun; Farmer, Jesse D., Jr.; Lane, Willam S.; Friedman, Jeff; Weissman, Irving; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">807-15</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Although the immediate receptors (immunophilins) of the immunosuppressants cyclosporin A (CsA) and FK506 are distinct, their similar mechanisms of inhibition of cell signaling suggest that their assocd. immunophilin complexes interact with a common target.  The authors report that the complexes cyclophilin-CsA and FKBP-FK506 (but not cyclophilin, FKBP, FKBP-rapamycin, or FKBP-506BD) competitively bind to and inhibit the Ca2+- and calmodulin-dependent phosphatase calcineurin, although the binding and inhibition of calcineurin do not require calmodulin.  These results suggest that calcineurin is involved in a common step assocd. with T cell receptor and IgE receptor signaling pathways and that cyclophilin and FKBP mediate the actions of CsA and FK506, resp., by forming drug-dependent complexes with and altering the activity of calcineurin-calmodulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxfN8mg-hNhLVg90H21EOLACvtfcHk0lhfXdKycMlfsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlsV2qu7w%253D&md5=00c4883ea86e6b2c554f8f72ed717ac9</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990124-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990124-H%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFarmer%26aufirst%3DJ.%2BD.%26aulast%3DLane%26aufirst%3DW.%2BS.%26aulast%3DFriedman%26aufirst%3DJ.%26aulast%3DWeissman%26aufirst%3DI.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DCalcineurin%2520is%2520a%2520common%2520target%2520of%2520cyclophilin-cyclosporin%2520A%2520and%2520FKBP-FK506%2520complexes%26jtitle%3DCell%26date%3D1991%26volume%3D66%26spage%3D807%26epage%3D815%26doi%3D10.1016%2F0092-8674%2891%2990124-H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daigle, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therkelsen, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basavapathruni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span> <span> </span><span class="NLM_article-title">Potent inhibition of DOT1L as treatment of MLL-fusion leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">1017</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1182/blood-2013-04-497644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1182%2Fblood-2013-04-497644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23801631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlWltLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2013&pages=1017-1025&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=A.+Basavapathruniauthor=L.+Jinauthor=P.+A.+Boriack-Sjodinauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Potent+inhibition+of+DOT1L+as+treatment+of+MLL-fusion+leukemia&doi=10.1182%2Fblood-2013-04-497644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Potent inhibition of DOT1L as treatment of MLL-fusion leukemia</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Basavapathruni, Aravind; Jin, Lei; Boriack-Sjodin, P. Ann; Allain, Christina J.; Klaus, Christine R.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1017-1025</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Rearrangements of the MLL gene define a genetically distinct subset of acute leukemias with poor prognosis.  Current treatment options are of limited effectiveness; thus, there is a pressing need for new therapies for this disease.  Genetic and small mol. inhibitor studies have demonstrated that the histone methyltransferase DOT1L is required for the development and maintenance of MLL-rearranged leukemia in model systems.  Here we describe the characterization of EPZ-5676, a potent and selective aminonucleoside inhibitor of DOT1L histone methyltransferase activity.  The compd. has an inhibition const. value of 80 pM, and demonstrates 37000-fold selectivity over all other methyltransferases tested.  In cellular studies, EPZ-5676 inhibited H3K79 methylation and MLL-fusion target gene expression and demonstrated potent cell killing that was selective for acute leukemia lines bearing MLL translocations.  Continuous IV infusion of EPZ-5676 in a rat xenograft model of MLL-rearranged leukemia caused complete tumor regressions that were sustained well beyond the compd. infusion period with no significant wt. loss or signs of toxicity.  EPZ-5676 is therefore a potential treatment of MLL-rearranged leukemia and is under clin. investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT-fqlDzoK07Vg90H21EOLACvtfcHk0lhfXdKycMlfsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlWltLfE&md5=bad23c37a5fe886668e25d10b7d9b43a</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-04-497644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-04-497644%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DBasavapathruni%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DPotent%2520inhibition%2520of%2520DOT1L%2520as%2520treatment%2520of%2520MLL-fusion%2520leukemia%26jtitle%3DBlood%26date%3D2013%26volume%3D122%26spage%3D1017%26epage%3D1025%26doi%3D10.1182%2Fblood-2013-04-497644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monk, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomasiak, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keniya, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huschmann, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyndall, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, J. D.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finer-Moore, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, R. M.</span></span> <span> </span><span class="NLM_article-title">Architecture of a single membrane spanning cytochrome P450 suggests constraints that orient the catalytic domain relative to a bilayer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">3865</span>– <span class="NLM_lpage">3870</span>, <span class="refDoi"> DOI: 10.1073/pnas.1324245111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1073%2Fpnas.1324245111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=24613931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlyht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=3865-3870&author=B.+C.+Monkauthor=T.+M.+Tomasiakauthor=M.+V.+Keniyaauthor=F.+U.+Huschmannauthor=J.+D.+Tyndallauthor=J.+D.+O%E2%80%99Connellauthor=R.+D.+Cannonauthor=J.+G.+McDonaldauthor=A.+Rodriguezauthor=J.+S.+Finer-Mooreauthor=R.+M.+Stroud&title=Architecture+of+a+single+membrane+spanning+cytochrome+P450+suggests+constraints+that+orient+the+catalytic+domain+relative+to+a+bilayer&doi=10.1073%2Fpnas.1324245111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Architecture of a single membrane spanning cytochrome P 450 suggests constraints that orient the catalytic domain relative to a bilayer</span></div><div class="casAuthors">Monk, Brian C.; Tomasiak, Thomas M.; Keniya, Mikhail V.; Huschmann, Franziska U.; Tyndall, Joel D. A.; O'Connell, Joseph D., III; Cannon, Richard D.; McDonald, Jeffrey G.; Rodriguez, Andrew; Finer-Moore, Janet S.; Stroud, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3865-3870</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bitopic integral membrane proteins with a single transmembrane helix play diverse roles in catalysis, cell signaling, and morphogenesis.  Complete monospanning protein structures are needed to show how interaction between the transmembrane helix and catalytic domain might influence assocn. with the membrane and function.  Here, the authors report the crystal structures of full-length Saccharomyces cerevisiae lanosterol 14α-demethylase, a membrane-monospanning cytochrome P 450 of the CYP51 family that catalyzes the 1st post-cyclization step in ergosterol biosynthesis and is inhibited by triazole drugs.  The structures revealed a well-ordered N-terminal amphipathic helix preceding a putative transmembrane helix that would constrain the catalytic domain orientation to lie partly in the lipid bilayer.  The structures located the substrate, lanosterol, identified putative substrate and product channels, and revealed constrained interactions with triazole antifungal drugs that are important for drug design and understanding drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWnbj5vZw9lLVg90H21EOLACvtfcHk0lib3O3mqAAouQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlyht7k%253D&md5=e3e0d0f9577509cc8fc21a8b32f0bb55</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1324245111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1324245111%26sid%3Dliteratum%253Aachs%26aulast%3DMonk%26aufirst%3DB.%2BC.%26aulast%3DTomasiak%26aufirst%3DT.%2BM.%26aulast%3DKeniya%26aufirst%3DM.%2BV.%26aulast%3DHuschmann%26aufirst%3DF.%2BU.%26aulast%3DTyndall%26aufirst%3DJ.%2BD.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DJ.%2BD.%26aulast%3DCannon%26aufirst%3DR.%2BD.%26aulast%3DMcDonald%26aufirst%3DJ.%2BG.%26aulast%3DRodriguez%26aufirst%3DA.%26aulast%3DFiner-Moore%26aufirst%3DJ.%2BS.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26atitle%3DArchitecture%2520of%2520a%2520single%2520membrane%2520spanning%2520cytochrome%2520P450%2520suggests%2520constraints%2520that%2520orient%2520the%2520catalytic%2520domain%2520relative%2520to%2520a%2520bilayer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D3865%26epage%3D3870%26doi%3D10.1073%2Fpnas.1324245111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winston, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maziarz, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekar, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarus, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumer, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitz, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Territo, M. C.</span></span> <span> </span><span class="NLM_article-title">Intravenous and oral itraconazole versus intravenous and oral fluconazole for long-term antifungal prophylaxis in allogeneic hematopoietic stem-cell transplant recipients. A multicenter, randomized trial</span>. <i>Ann. Intern. Med.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">705</span>– <span class="NLM_lpage">713</span>, <span class="refDoi"> DOI: 10.7326/0003-4819-138-9-200305060-00006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.7326%2F0003-4819-138-9-200305060-00006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=12729424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktVyhsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2003&pages=705-713&author=D.+J.+Winstonauthor=R.+T.+Maziarzauthor=P.+H.+Chandrasekarauthor=H.+M.+Lazarusauthor=M.+Goldmanauthor=J.+L.+Blumerauthor=G.+J.+Leitzauthor=M.+C.+Territo&title=Intravenous+and+oral+itraconazole+versus+intravenous+and+oral+fluconazole+for+long-term+antifungal+prophylaxis+in+allogeneic+hematopoietic+stem-cell+transplant+recipients.+A+multicenter%2C+randomized+trial&doi=10.7326%2F0003-4819-138-9-200305060-00006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Intravenous and oral itraconazole versus intravenous and oral fluconazole for long-term antifungal prophylaxis in allogeneic hematopoietic stem-cell transplant recipients: a multicenter, randomized trial</span></div><div class="casAuthors">Winston, Drew J.; Maziarz, Richard T.; Chandrasekar, Pranatharthi H.; Lazarus, Hillard M.; Goldman, Mitchell; Blumer, Jeffrey L.; Leitz, Gerhard J.; Territo, Mary C.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Internal Medicine</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">705-713</span>CODEN:
                <span class="NLM_cas:coden">AIMEAS</span>;
        ISSN:<span class="NLM_cas:issn">0003-4819</span>.
    
            (<span class="NLM_cas:orgname">American College of Physicians-American Society of Internal Medicine</span>)
        </div><div class="casAbstract">Background: Allogeneic hematopoietic stem-cell transplant recipients often receive fluconazole or an amphotericin B prepn. for antifungal prophylaxis.  Because of concerns about fungal resistance with fluconazole and toxicity with amphotericin B, alternative prophylactic regimens have become necessary.  Objective: To compare the efficacy and safety of i.v. and oral itraconazole with the efficacy and safety of i.v. and oral fluconazole for long-term prophylaxis of fungal infections.  Design: open-label, multicenter, randomized trial.  Setting: Five transplantation centers in the United States.  Patients: 140 patients undergoing allogeneic hematopoietic stem-cell transplantation.  Intervention: Itraconazole (200 mg i.v. every 12 h for 2 days followed by 200 mg i.v. every 24 h or a 200-mg oral soln. every 12 h) or fluconazole (400 mg i.v. or orally every 24 h) from day 1 until day 100 after transplantation.  Measurements: Proven invasive or superficial fungal infection, drug-related side effects, mortality from fungal infection, and overall mortality.  Results: Proven invasive fungal infections occurred in 6 of 71 itraconazole recipients (9%) and in 17 of 67 fluconazole recipients (25%) during the first 180 days after transplantation (difference, -16 percentage points [95% CI, -29.2 to -4.7 percentage points]; P = 0.01).  Superficial fungal infections occurred in 3 of 71 itraconazole recipients (4%) and in 2 of 67 fluconazole recipients (3%).  In a multivariable anal. using factors known to affect the risk for invasive fungal infection after hematopoietic stem-cell transplantation, prophylaxis with itraconazole was still assocd. with fewer invasive fungal infections (odds ratio, 0.300 [CI, 0.111 to 0.814]; P = 0.02) caused by either yeasts or molds.  More fungal pathogens were found to be resistant to fluconazole than to itraconazole.  Except for more frequent gastrointestinal side effects (nausea, vomiting, diarrhea, or abdominal pain) in patients given itraconazole (24% vs. 9%; difference, 15 percentage points [CI, 2.9 to 27.0 percentage points]; P = 0.02), both itraconazole and fluconazole were well tolerated.  The overall mortality rate was similar in each group (32 of 71 patients in the itraconazole group [45%] vs. 28 of 67 patients in the fluconazole group [42%]; difference, 3 percentage points [CI, -13.2 to 19.8 percentage points]; P > 0.2), but fewer deaths were related to fungal infection in patients given itraconazole (6 of 71 [9%]) than in patients given fluconazole (12 of 67 [18%]) (difference, 9 percentage points [CI, -20.6 to 1.8 percentage points]; P = 0.13).  Conclusion: Itraconazole is more effective than fluconazole for long-term prophylaxis of invasive fungal infections after allogeneic hematopoietic stem-cell transplantation.  Except for gastrointestinal side effects, itraconazole is well tolerated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRJ3IIYdiGfLVg90H21EOLACvtfcHk0ljkefN9TFbu7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktVyhsLg%253D&md5=eb39b1e6ccf742c6b2f3937780a1ebb9</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.7326%2F0003-4819-138-9-200305060-00006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7326%252F0003-4819-138-9-200305060-00006%26sid%3Dliteratum%253Aachs%26aulast%3DWinston%26aufirst%3DD.%2BJ.%26aulast%3DMaziarz%26aufirst%3DR.%2BT.%26aulast%3DChandrasekar%26aufirst%3DP.%2BH.%26aulast%3DLazarus%26aufirst%3DH.%2BM.%26aulast%3DGoldman%26aufirst%3DM.%26aulast%3DBlumer%26aufirst%3DJ.%2BL.%26aulast%3DLeitz%26aufirst%3DG.%2BJ.%26aulast%3DTerrito%26aufirst%3DM.%2BC.%26atitle%3DIntravenous%2520and%2520oral%2520itraconazole%2520versus%2520intravenous%2520and%2520oral%2520fluconazole%2520for%2520long-term%2520antifungal%2520prophylaxis%2520in%2520allogeneic%2520hematopoietic%2520stem-cell%2520transplant%2520recipients.%2520A%2520multicenter%252C%2520randomized%2520trial%26jtitle%3DAnn.%2520Intern.%2520Med.%26date%3D2003%26volume%3D138%26spage%3D705%26epage%3D713%26doi%3D10.7326%2F0003-4819-138-9-200305060-00006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. V.</span></span> <span> </span><span class="NLM_article-title">The use of fluconazole and itraconazole in the treatment of <i>Candida albicans</i> infections: A review</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1093/jac/44.4.429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fjac%2F44.4.429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10588302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADyaK1MXntV2lurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=1999&pages=429-437&author=M.+V.+Martin&title=The+use+of+fluconazole+and+itraconazole+in+the+treatment+of+Candida+albicans+infections%3A+A+review&doi=10.1093%2Fjac%2F44.4.429"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">The use of fluconazole and itraconazole in the treatment of Candida albicans infections: a review</span></div><div class="casAuthors">Martin, Michael V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">429-437</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review with 93 refs.  Candida albicans is responsible for most fungal infections in humans.  Fluconazole is well established as a first-line management option for the treatment and prophylaxis of localized and systemic C. albicans infections.  Fluconazole exhibits predictable pharmacokinetics and is effective, well tolerated and suitable for use in most patients with C. albicans infections, including children, the elderly and those with impaired immunity.  Prophylactic administration of fluconazole can help to prevent fungal infections in patients receiving cytotoxic cancer therapy.  The increasing use of fluconazole for the long-term prophylaxis and treatment of recurrent oral candidosis in AIDS patients has led to the emergence of C. albicans infections that are not responsive to conventional doses.  Second-line therapy with a wider spectrum anti-fungal, such as itraconazole, should be sought if treatment with fluconazole fails.  A soln. formulation of itraconazole has recently been introduced to overcome the poor and variable absorption of its original capsule formulation.  Efficacy and tolerability studies in HIV-pos. or immunocompromised patients with C. albicans infections have shown that, although itraconazole soln. is as effective as fluconazole, it is less well tolerated as first-line therapy.  Itraconazole soln. can be effective in AIDS patients with C. albicans infections that are non-responsive to fluconazole.  No efficacy or tolerability data are available on the use of itraconazole soln. in children or the elderly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfmtvdjIvc1LVg90H21EOLACvtfcHk0ljkefN9TFbu7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntV2lurs%253D&md5=e3f831937fab31bb98569375c7f4f1f2</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1093%2Fjac%2F44.4.429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252F44.4.429%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%2BV.%26atitle%3DThe%2520use%2520of%2520fluconazole%2520and%2520itraconazole%2520in%2520the%2520treatment%2520of%2520Candida%2520albicans%2520infections%253A%2520A%2520review%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D1999%26volume%3D44%26spage%3D429%26epage%3D437%26doi%3D10.1093%2Fjac%2F44.4.429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hurdle, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, I.</span></span> <span> </span><span class="NLM_article-title">Prospects for aminoacyl-tRNA synthetase inhibitors as new antimicrobial agents</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">4821</span>– <span class="NLM_lpage">4833</span>, <span class="refDoi"> DOI: 10.1128/AAC.49.12.4821-4833.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1128%2FAAC.49.12.4821-4833.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=16304142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSru7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2005&pages=4821-4833&author=J.+G.+Hurdleauthor=A.+J.+O%E2%80%99Neillauthor=I.+Chopra&title=Prospects+for+aminoacyl-tRNA+synthetase+inhibitors+as+new+antimicrobial+agents&doi=10.1128%2FAAC.49.12.4821-4833.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Prospects for aminoacyl-tRNA synthetase inhibitors as new antimicrobial agents</span></div><div class="casAuthors">Hurdle, Julian Gregston; O'Neill, Alexander John; Chopra, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4821-4833</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review with refs.  The authors discuss the prospects for the development of several recently reported AaRS inhibitors as chemotherapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL2y1GuMeel7Vg90H21EOLACvtfcHk0lg6n7_x7Z75vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSru7fL&md5=fd6edfc986fd02a9540a8f567c059bf7</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1128%2FAAC.49.12.4821-4833.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.49.12.4821-4833.2005%26sid%3Dliteratum%253Aachs%26aulast%3DHurdle%26aufirst%3DJ.%2BG.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DA.%2BJ.%26aulast%3DChopra%26aufirst%3DI.%26atitle%3DProspects%2520for%2520aminoacyl-tRNA%2520synthetase%2520inhibitors%2520as%2520new%2520antimicrobial%2520agents%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2005%26volume%3D49%26spage%3D4821%26epage%3D4833%26doi%3D10.1128%2FAAC.49.12.4821-4833.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Large FK506-binding proteins shape the pharmacology of rapamycin</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1357</span>– <span class="NLM_lpage">1367</span>, <span class="refDoi"> DOI: 10.1128/MCB.00678-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1128%2FMCB.00678-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23358420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A280%3ADC%252BC3szjt12mtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=1357-1367&author=A.+M.+Marzauthor=A.+K.+Fabianauthor=C.+Kozanyauthor=A.+Bracherauthor=F.+Hausch&title=Large+FK506-binding+proteins+shape+the+pharmacology+of+rapamycin&doi=10.1128%2FMCB.00678-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Large FK506-binding proteins shape the pharmacology of rapamycin</span></div><div class="casAuthors">Marz Andreas M; Fabian Anne-Katrin; Kozany Christian; Bracher Andreas; Hausch Felix</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and cellular biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1357-67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The immunosuppressant and anticancer drug rapamycin works by inducing inhibitory protein complexes with the kinase mTOR, an important regulator of growth and proliferation.  The obligatory accessory partner of rapamycin is believed to be FK506-binding protein 12 (FKBP12).  Here we show that rapamycin complexes of larger FKBP family members can tightly bind to mTOR and potently inhibit its kinase activity.  Cocrystal structures with FKBP51 and FKBP52 reveal the modified molecular binding mode of these alternative ternary complexes in detail.  In cellular model systems, FKBP12 can be functionally replaced by larger FKBPs.  When the rapamycin dosage is limiting, mTOR inhibition of S6K phosphorylation can be enhanced by FKBP51 overexpression in mammalian cells, whereas FKBP12 is dispensable.  FKBP51 could also enable the rapamycin-induced hyperphosphorylation of Akt, which depended on higher FKBP levels than rapamycin-induced inhibition of S6K phosphorylation.  These insights provide a mechanistic rationale for preferential mTOR inhibition in specific cell or tissue types by engaging specific FKBP homologs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRPwiOr3vbNw3RxY89Cm6SyfW6udTcc2eZKCajLwz5GJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szjt12mtg%253D%253D&md5=7422f9a3e2dd75a25dae65e7d16213bd</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1128%2FMCB.00678-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00678-12%26sid%3Dliteratum%253Aachs%26aulast%3DMarz%26aufirst%3DA.%2BM.%26aulast%3DFabian%26aufirst%3DA.%2BK.%26aulast%3DKozany%26aufirst%3DC.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DLarge%2520FK506-binding%2520proteins%2520shape%2520the%2520pharmacology%2520of%2520rapamycin%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2013%26volume%3D33%26spage%3D1357%26epage%3D1367%26doi%3D10.1128%2FMCB.00678-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Opal, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laterre, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francois, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaRosa, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angus, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mira, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittebole, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dugernier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrotin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidswell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jauregui, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peckelsen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordasco, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeyen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maruyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalil, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Nuffelen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossignol, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogate, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, J. L.</span></span>; <span class="NLM_contrib-group">ACCESS Study Group.</span> <span> </span><span class="NLM_article-title">Effect of eritoran, an antagonist of MD2-TLR4, on mortality in patients with severe sepsis: The access randomized trial</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>309</i></span>,  <span class="NLM_fpage">1154</span>– <span class="NLM_lpage">1162</span>, <span class="refDoi"> DOI: 10.1001/jama.2013.2194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1001%2Fjama.2013.2194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23512062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltlyqt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=309&publication_year=2013&pages=1154-1162&author=S.+M.+Opalauthor=P.+F.+Laterreauthor=B.+Francoisauthor=S.+P.+LaRosaauthor=D.+C.+Angusauthor=J.+P.+Miraauthor=X.+Witteboleauthor=T.+Dugernierauthor=D.+Perrotinauthor=M.+Tidswellauthor=L.+Jaureguiauthor=K.+Krellauthor=J.+Pachlauthor=T.+Takahashiauthor=C.+Peckelsenauthor=E.+Cordascoauthor=C.+S.+Changauthor=S.+Oeyenauthor=N.+Aikawaauthor=T.+Maruyamaauthor=R.+Scheinauthor=A.+C.+Kalilauthor=M.+Van+Nuffelenauthor=M.+Lynnauthor=D.+P.+Rossignolauthor=J.+Gogateauthor=M.+B.+Robertsauthor=J.+L.+Wheelerauthor=J.+L.+Vincentauthor=ACCESS+Study+Group.&title=Effect+of+eritoran%2C+an+antagonist+of+MD2-TLR4%2C+on+mortality+in+patients+with+severe+sepsis%3A+The+access+randomized+trial&doi=10.1001%2Fjama.2013.2194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of eritoran, an antagonist of MD2-TLR4, on mortality in patients with severe sepsis. The ACCESS randomized trial</span></div><div class="casAuthors">Opal, Steven M.; Laterre, Pierre-Francois; Francois, Bruno; LaRosa, Steven P.; Angus, Derek C.; Mira, Jean-Paul; Wittebole, Xavier; Dugernier, Thierry; Perrotin, Dominique; Tidswell, Mark; Jauregui, Luis; Krell, Kenneth; Pachl, Jan; Takahashi, Takeshi; Peckelsen, Claus; Cordasco, Edward; Chang, Chia-Sheng; Oeyen, Sandra; Aikawa, Naoki; Maruyama, Tatsuya; Schein, Roland; Kalil, Andre C.; Van Nuffelen, Marc; Lynn, Melvyn; Rossignol, Daniel P.; Gogate, Jogadish; Roberts, Mary B.; Wheeler, Janice L.; Vincent, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">309</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1154-1162</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">0098-7484</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">Importance: Eritoran is a synthetic lipid A antagonist that blocks Iipopolysaccharide (LPS) from binding at the cell surface MD2-TLR4 receptor.  LPS is a major component of the outer membrane of gram-neg. bacteria and is a potent activator of the acute inflammatory response.  Objective: To det. if eritoran, a TLR4 antagonist, would significantly reduce sepsis-induced mortality.  Design, Setting, and Participants: We performed a randomized, double-blind, placebo-controlled, multinational phase 3 trial in 197 intensive care units.  Patients were enrolled from June 2006 to Sept. 2010 and final follow-up was completed in Sept. 2011.  Interventions: Patients with severe sepsis (n = 1961) were randomized and treated within 12 h of onset of first organ dysfunction in a 2:1 ratio with a 6-day course of either eritoran tetrasodium (105 mg total) or placebo, with n = 1304 and n = 657 patients, resp.  Main Outcome Measures: The primary end point was 28-day all-cause mortality.  The secondary end points were all-cause mortality at 3, 6, and 12 mo after beginning treatment.  Results: Baseline characteristics of the 2 study groups were similar.  In the modified intent-to-treat anal. (randomized patients who received at least 1 dose) there was no significant difference in the primary end point of 28-day all-cause mortality with 28.1% (366/1304) in the eritoran group vs 26.9% (177/657) in the placebo group (P = .59; hazard ratio, 1.05; 95% CI, 0.88-1.26; difference in mortality rate, -1.1; 95% CI, -5.3 to 3.1) or in the key secondary end point of 1-yr all-cause mortality with 44.1% (290/657) in the eritoran group vs 43.3% (565/1304) in the placebo group, Kaplan-Meier anal. of time to death by 1 yr, P = .79 (hazard ratio, 0.98; 0.85-1.13).  No significant differences were obsd. in any of the prespecified subgroups.  Adverse events, including secondary infection rates, did not differ between study groups.  Conclusions and Relevance: Among patients with severe sepsis, the use of eritoran, compared with placebo, did not result in reduced 28-day mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrbHqr8OtY8bVg90H21EOLACvtfcHk0lg6n7_x7Z75vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltlyqt7o%253D&md5=adf1d02bc28953d8e6b30e9d627fcfa9</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1001%2Fjama.2013.2194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.2013.2194%26sid%3Dliteratum%253Aachs%26aulast%3DOpal%26aufirst%3DS.%2BM.%26aulast%3DLaterre%26aufirst%3DP.%2BF.%26aulast%3DFrancois%26aufirst%3DB.%26aulast%3DLaRosa%26aufirst%3DS.%2BP.%26aulast%3DAngus%26aufirst%3DD.%2BC.%26aulast%3DMira%26aufirst%3DJ.%2BP.%26aulast%3DWittebole%26aufirst%3DX.%26aulast%3DDugernier%26aufirst%3DT.%26aulast%3DPerrotin%26aufirst%3DD.%26aulast%3DTidswell%26aufirst%3DM.%26aulast%3DJauregui%26aufirst%3DL.%26aulast%3DKrell%26aufirst%3DK.%26aulast%3DPachl%26aufirst%3DJ.%26aulast%3DTakahashi%26aufirst%3DT.%26aulast%3DPeckelsen%26aufirst%3DC.%26aulast%3DCordasco%26aufirst%3DE.%26aulast%3DChang%26aufirst%3DC.%2BS.%26aulast%3DOeyen%26aufirst%3DS.%26aulast%3DAikawa%26aufirst%3DN.%26aulast%3DMaruyama%26aufirst%3DT.%26aulast%3DSchein%26aufirst%3DR.%26aulast%3DKalil%26aufirst%3DA.%2BC.%26aulast%3DVan%2BNuffelen%26aufirst%3DM.%26aulast%3DLynn%26aufirst%3DM.%26aulast%3DRossignol%26aufirst%3DD.%2BP.%26aulast%3DGogate%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DM.%2BB.%26aulast%3DWheeler%26aufirst%3DJ.%2BL.%26aulast%3DVincent%26aufirst%3DJ.%2BL.%26aulast%3D%26atitle%3DEffect%2520of%2520eritoran%252C%2520an%2520antagonist%2520of%2520MD2-TLR4%252C%2520on%2520mortality%2520in%2520patients%2520with%2520severe%2520sepsis%253A%2520The%2520access%2520randomized%2520trial%26jtitle%3DJAMA%26date%3D2013%26volume%3D309%26spage%3D1154%26epage%3D1162%26doi%3D10.1001%2Fjama.2013.2194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dieci, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnedos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andre, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor receptor inhibitors as a cancer treatment: From a biologic rationale to medical perspectives</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-12-0362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1158%2F2159-8290.CD-12-0362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=23418312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVOktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=264-279&author=M.+V.+Dieciauthor=M.+Arnedosauthor=F.+Andreauthor=J.+C.+Soria&title=Fibroblast+growth+factor+receptor+inhibitors+as+a+cancer+treatment%3A+From+a+biologic+rationale+to+medical+perspectives&doi=10.1158%2F2159-8290.CD-12-0362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast Growth Factor Receptor Inhibitors as a Cancer Treatment: From a Biologic Rationale to Medical Perspectives</span></div><div class="casAuthors">Dieci, Maria Vittoria; Arnedos, Monica; Andre, Fabrice; Soria, Jean Charles</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">264-279</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The fibroblast growth factor/fibroblast growth factor receptor (FGF/FGFR) signaling pathway plays a fundamental role in many physiol. processes, including embryogenesis, adult tissue homeostasis, and wound healing, by orchestrating angiogenesis.  Ligand-independent and ligand-dependent activation have been implicated in a broad range of human malignancies and promote cancer progression in tumors driven by FGF/FGFR oncogenic mutations or amplifications, tumor neoangiogenesis, and targeted treatment resistance, thereby supporting a strong rationale for anti-FGF/FGFR agent development.  Efforts are being pursued to develop selective approaches for use against this pathway by optimizing the management of emerging, class-specific toxicity profiles and correctly designing clin. trials to address these different issues.  Significance: FGF/FGFR pathway deregulations are increasingly recognized across different human cancers.  Understanding the mechanisms at the basis of these alterations and their multiple roles in cancer promotion and drug resistance is a fundamental step for further implementation of targeted therapies and research strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6bibZKwamTLVg90H21EOLACvtfcHk0ljB784J2AkdJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVOktL4%253D&md5=0e694a7641aa83c1540f745576f04bfa</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-12-0362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-12-0362%26sid%3Dliteratum%253Aachs%26aulast%3DDieci%26aufirst%3DM.%2BV.%26aulast%3DArnedos%26aufirst%3DM.%26aulast%3DAndre%26aufirst%3DF.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26atitle%3DFibroblast%2520growth%2520factor%2520receptor%2520inhibitors%2520as%2520a%2520cancer%2520treatment%253A%2520From%2520a%2520biologic%2520rationale%2520to%2520medical%2520perspectives%26jtitle%3DCancer%2520Discovery%26date%3D2013%26volume%3D3%26spage%3D264%26epage%3D279%26doi%3D10.1158%2F2159-8290.CD-12-0362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guagnano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Douget, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghausen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussiere, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus Porta, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of 3-(2,6-dichloro-3,5-dimethoxy-phenyl)-1-{6-[4-(4-ethyl-piperazin-1-yl)-phenylamin o]-pyrimidin-4-yl}-1-methyl-urea (NVP-BGJ398), a potent and selective inhibitor of the fibroblast growth factor receptor family of receptor tyrosine kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7066</span>– <span class="NLM_lpage">7083</span>, <span class="refDoi"> DOI: 10.1021/jm2006222</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2006222" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7066-7083&author=V.+Guagnanoauthor=P.+Furetauthor=C.+Spankaauthor=V.+Bordasauthor=M.+Le+Dougetauthor=C.+Stammauthor=J.+Brueggenauthor=M.+R.+Jensenauthor=C.+Schnellauthor=H.+Schmidauthor=M.+Wartmannauthor=J.+Berghausenauthor=P.+Drueckesauthor=A.+Zimmerlinauthor=D.+Bussiereauthor=J.+Murrayauthor=D.+Graus+Porta&title=Discovery+of+3-%282%2C6-dichloro-3%2C5-dimethoxy-phenyl%29-1-%7B6-%5B4-%284-ethyl-piperazin-1-yl%29-phenylamin+o%5D-pyrimidin-4-yl%7D-1-methyl-urea+%28NVP-BGJ398%29%2C+a+potent+and+selective+inhibitor+of+the+fibroblast+growth+factor+receptor+family+of+receptor+tyrosine+kinase&doi=10.1021%2Fjm2006222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-(2,6-Dichloro-3,5-dimethoxy-phenyl)-1-{6-[4-(4-ethyl-piperazin-1-yl)-phenylamino]-pyrimidin-4-yl}-1-methyl-urea (NVP-BGJ398), A Potent and Selective Inhibitor of the Fibroblast Growth Factor Receptor Family of Receptor Tyrosine Kinase</span></div><div class="casAuthors">Guagnano, Vito; Furet, Pascal; Spanka, Carsten; Bordas, Vincent; Le Douget, Mickael; Stamm, Christelle; Brueggen, Josef; Jensen, Michael R.; Schnell, Christian; Schmid, Herbert; Wartmann, Markus; Berghausen, Joerg; Drueckes, Peter; Zimmerlin, Alfred; Bussiere, Dirksen; Murray, Jeremy; Graus Porta, Diana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7066-7083</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of N-aryl-N'-pyrimidin-4-yl ureas has been optimized to afford potent and selective inhibitors of the fibroblast growth factor receptor tyrosine kinases 1, 2, and 3 by rationally designing the substitution pattern of the aryl ring.  On the basis of its in vitro profile, compd. 1h (NVP-BGJ398) was selected for in vivo evaluation and showed significant antitumor activity in RT112 bladder cancer xenografts models overexpressing wild-type FGFR3.  These results support the potential therapeutic use of 1h as a new anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSC1v0A_6I_bVg90H21EOLACvtfcHk0ljB784J2AkdJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK&md5=f9a2d742294cd3814e872bdd8add1ffa</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fjm2006222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2006222%26sid%3Dliteratum%253Aachs%26aulast%3DGuagnano%26aufirst%3DV.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DSpanka%26aufirst%3DC.%26aulast%3DBordas%26aufirst%3DV.%26aulast%3DLe%2BDouget%26aufirst%3DM.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DSchnell%26aufirst%3DC.%26aulast%3DSchmid%26aufirst%3DH.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DBerghausen%26aufirst%3DJ.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DZimmerlin%26aufirst%3DA.%26aulast%3DBussiere%26aufirst%3DD.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DGraus%2BPorta%26aufirst%3DD.%26atitle%3DDiscovery%2520of%25203-%25282%252C6-dichloro-3%252C5-dimethoxy-phenyl%2529-1-%257B6-%255B4-%25284-ethyl-piperazin-1-yl%2529-phenylamin%2520o%255D-pyrimidin-4-yl%257D-1-methyl-urea%2520%2528NVP-BGJ398%2529%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520the%2520fibroblast%2520growth%2520factor%2520receptor%2520family%2520of%2520receptor%2520tyrosine%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7066%26epage%3D7083%26doi%3D10.1021%2Fjm2006222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olziersky, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labidi-Galy, S. I.</span></span> <span> </span><span class="NLM_article-title">Clinical development of anti-mitotic drugs in cancer</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1002</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1007/978-3-319-57127-0_6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1007%2F978-3-319-57127-0_6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=28600785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFCns7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1002&publication_year=2017&pages=125-152&author=A.+M.+Olzierskyauthor=S.+I.+Labidi-Galy&title=Clinical+development+of+anti-mitotic+drugs+in+cancer&doi=10.1007%2F978-3-319-57127-0_6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical development of anti-mitotic drugs in cancer</span></div><div class="casAuthors">Olziersky, Anna-Maria; Labidi-Galy, S. Intidhar</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1002</span>
        (<span class="NLM_cas:issue">Cell Division Machinery and Disease</span>),
    <span class="NLM_cas:pages">125-152</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">2214-8019</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Mitosis is one of the most fundamental processes of life by which a mammalian cell divides into two daughter cells.  Mitosis has been an attractive target for anticancer therapies since fast proliferation was identified as one of the hallmarks of cancer cells.  Despite efforts into developing specific inhibitors for mitotic kinases and kinesins, very few drugs have shown the efficiency of microtubule targeting-agents in cancer cells with paclitaxel being the most successful.  A deeper translational research accompanying clin. trials of anti-mitotic drugs will help in identifying potent biomarkers predictive for response.  Here, we review the current knowledge of mitosis targeting agents that have been tested so far in the clinics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp80dIlYTJrsrVg90H21EOLACvtfcHk0lhNyINxO8pccA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFCns7nL&md5=ecf3ecdb4b8fd3e0ce5e5f0e20ca7654</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-57127-0_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-3-319-57127-0_6%26sid%3Dliteratum%253Aachs%26aulast%3DOlziersky%26aufirst%3DA.%2BM.%26aulast%3DLabidi-Galy%26aufirst%3DS.%2BI.%26atitle%3DClinical%2520development%2520of%2520anti-mitotic%2520drugs%2520in%2520cancer%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D2017%26volume%3D1002%26spage%3D125%26epage%3D152%26doi%3D10.1007%2F978-3-319-57127-0_6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Purcell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samayoa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bean, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziyad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feiler, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cases, S.</span></span> <span> </span><span class="NLM_article-title">Activity of the kinesin spindle protein inhibitor ispinesib (SB-715992) in models of breast cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">566</span>– <span class="NLM_lpage">576</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-09-1498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1158%2F1078-0432.CCR-09-1498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=20068098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktF2qtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=566-576&author=J.+W.+Purcellauthor=J.+Davisauthor=M.+Reddyauthor=S.+Martinauthor=K.+Samayoaauthor=H.+Voauthor=K.+Thomsenauthor=P.+Beanauthor=W.+L.+Kuoauthor=S.+Ziyadauthor=J.+Billigauthor=H.+S.+Feilerauthor=J.+W.+Grayauthor=K.+W.+Woodauthor=S.+Cases&title=Activity+of+the+kinesin+spindle+protein+inhibitor+ispinesib+%28SB-715992%29+in+models+of+breast+cancer&doi=10.1158%2F1078-0432.CCR-09-1498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Activity of the kinesin spindle protein inhibitor ispinesib (SB-715992) in models of breast cancer</span></div><div class="casAuthors">Purcell, James W.; Davis, Jefferson; Reddy, Mamatha; Martin, Shamra; Samayoa, Kimberly; Vo, Hung; Thomsen, Karen; Bean, Peter; Kuo, Wen Lin; Ziyad, Safiyyah; Billig, Jessica; Feiler, Heidi S.; Gray, Joe W.; Wood, Kenneth W.; Cases, Sylvaine</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">566-576</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Ispinesib (SB-715992) is a potent inhibitor of kinesin spindle protein, a kinesin motor protein essential for the formation of a bipolar mitotic spindle and cell cycle progression through mitosis.  Clin. studies of ispinesib have shown a 9% response rate in patients with locally advanced or metastatic breast cancer and a favorable safety profile without significant neurotoxicities, gastrointestinal toxicities, or hair loss.  To better understand the potential of ispinesib in the treatment of breast cancer, we explored the activity of ispinesib alone and in combination with several therapies approved for the treatment of breast cancer.  Exptl. Design: We measured the ispinesib sensitivity and pharmacodynamic response of breast cancer cell lines representative of various subtypes in vitro and as xenografts in vivo and tested the ability of ispinesib to enhance the antitumor activity of approved therapies.  Results: In vitro, ispinesib displayed broad antiproliferative activity against a panel of 53 breast cell lines.  In vivo, ispinesib produced regressions in each of five breast cancer models and tumor-free survivors in three of these models.  The effects of ispinesib treatment on pharmacodynamic markers of mitosis and apoptosis were examd. in vitro and in vivo, revealing a greater increase in both mitotic and apoptotic markers in the MDA-MB-468 model than in the less sensitive BT-474 model.  In vivo, ispinesib enhanced the antitumor activity of trastuzumab, lapatinib, doxorubicin, and capecitabine and exhibited activity comparable with paclitaxel and ixabepilone.  Conclusions: These findings support further clin. exploration of kinesin spindle protein inhibitors for the treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJseCrHGemNrVg90H21EOLACvtfcHk0lhNyINxO8pccA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktF2qtrc%253D&md5=2868435e2d157dd55cd7c68f9530c1df</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-1498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-1498%26sid%3Dliteratum%253Aachs%26aulast%3DPurcell%26aufirst%3DJ.%2BW.%26aulast%3DDavis%26aufirst%3DJ.%26aulast%3DReddy%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSamayoa%26aufirst%3DK.%26aulast%3DVo%26aufirst%3DH.%26aulast%3DThomsen%26aufirst%3DK.%26aulast%3DBean%26aufirst%3DP.%26aulast%3DKuo%26aufirst%3DW.%2BL.%26aulast%3DZiyad%26aufirst%3DS.%26aulast%3DBillig%26aufirst%3DJ.%26aulast%3DFeiler%26aufirst%3DH.%2BS.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26aulast%3DWood%26aufirst%3DK.%2BW.%26aulast%3DCases%26aufirst%3DS.%26atitle%3DActivity%2520of%2520the%2520kinesin%2520spindle%2520protein%2520inhibitor%2520ispinesib%2520%2528SB-715992%2529%2520in%2520models%2520of%2520breast%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2010%26volume%3D16%26spage%3D566%26epage%3D576%26doi%3D10.1158%2F1078-0432.CCR-09-1498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Istvan, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deisenhofer, J.</span></span> <span> </span><span class="NLM_article-title">Structural mechanism for statin inhibition of HMG-CoA reductase</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1164</span>, <span class="refDoi"> DOI: 10.1126/science.1059344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1126%2Fscience.1059344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11349148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjs1Cgtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2001&pages=1160-1164&author=E.+S.+Istvanauthor=J.+Deisenhofer&title=Structural+mechanism+for+statin+inhibition+of+HMG-CoA+reductase&doi=10.1126%2Fscience.1059344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism for statin inhibition of HMG-CoA reductase</span></div><div class="casAuthors">Istvan, Eva S.; Deisenhofer, Johann</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">5519</span>),
    <span class="NLM_cas:pages">1160-1164</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">HMG-CoA (3-hydroxy-3-methylglutaryl-CoA) reductase (HMGR) catalyzes the committed step in cholesterol biosynthesis.  Statins are HMGR inhibitors with inhibition const. values in the nanomolar range that effectively lower serum cholesterol levels and are widely prescribed in the treatment of hypercholesterolemia.  We have detd. structures of the catalytic portion of human HMGR complexed with six different statins.  The statins occupy a portion of the binding site of HMG-CoA, thus blocking access of this substrate to the active site.  Near the carboxyl terminus of HMGR, several catalytically relevant residues are disordered in the enzyme-statin complexes.  If these residues were not flexible, they would sterically hinder statin binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDsSOnK9675bVg90H21EOLACvtfcHk0lhNyINxO8pccA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjs1Cgtbo%253D&md5=235644ee8cca3d2e061093e93bd1d702</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1126%2Fscience.1059344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1059344%26sid%3Dliteratum%253Aachs%26aulast%3DIstvan%26aufirst%3DE.%2BS.%26aulast%3DDeisenhofer%26aufirst%3DJ.%26atitle%3DStructural%2520mechanism%2520for%2520statin%2520inhibition%2520of%2520HMG-CoA%2520reductase%26jtitle%3DScience%26date%3D2001%26volume%3D292%26spage%3D1160%26epage%3D1164%26doi%3D10.1126%2Fscience.1059344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fouty, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodman, D. M.</span></span> <span> </span><span class="NLM_article-title">Mevastatin can cause G1 arrest and induce apoptosis in pulmonary artery smooth muscle cells through a p27<sup>KIP1</sup>-independent pathway</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1161/01.RES.0000061180.03813.0F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1161%2F01.RES.0000061180.03813.0F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=12600884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitVSjsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2003&pages=501-509&author=B.+W.+Foutyauthor=D.+M.+Rodman&title=Mevastatin+can+cause+G1+arrest+and+induce+apoptosis+in+pulmonary+artery+smooth+muscle+cells+through+a+p27KIP1-independent+pathway&doi=10.1161%2F01.RES.0000061180.03813.0F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Mevastatin Can Cause G1 Arrest and Induce Apoptosis in Pulmonary Artery Smooth Muscle Cells Through a p27Kip1-Independent Pathway</span></div><div class="casAuthors">Fouty, Brian W.; Rodman, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Circulation Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">501-509</span>CODEN:
                <span class="NLM_cas:coden">CIRUAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-7330</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Advanced pulmonary arterial hypertension is characterized by extensive vascular remodeling that is usually resistant to vasodilator therapy.  Mevastatin is an inhibitor of 3-hydroxy-3-methylglutaryl-CoA (HMG-CoA) reductase, the rate-limiting step for cholesterol synthesis.  HMG-CoA reductase inhibitors have been shown to upregulate the cyclin-dependent kinase inhibitor p27Kip1 and to block cell proliferation through cholesterol-independent pathways.  The aim of this study was to det. the effect of mevastatin on DNA synthesis, cell cycle progression, and cell proliferation in rat pulmonary artery smooth muscle cells (PASMCs).  We found that mevastatin induced G1 arrest and decreased DNA synthesis in rat PASMCs and did so in assocn. with an increase in both total and cyclin E-bound p27Kip1.  This caused a marked decrease in cyclin E kinase activity, which suggests an important role for p27Kip1 in the ability of mevastatin to induce G1 arrest.  However, in PASMCs lacking functional p27Kip1, mevastatin still decreased cyclin E kinase activity, caused G1 arrest, and decreased DNA synthesis.  In p27Kip1-deficient PASMCs, mevastatin induced a greater redn. of cyclin E protein levels (to 35% of control) than in wild-type cells (to 70% of control) and also reduced the phosphorylation of cdk2 on threonine 160.  Mevastatin also caused apoptosis in both wild-type and p27Kip1-deficient PASMCs and was able to do so at a dose that did not induce cell cycle arrest.  These data suggest that HMG-CoA reductase inhibitors can both inhibit cell proliferation and induce apoptosis in PASMCs through p27Kip1-independent pathways and may be important therapeutic agents in pulmonary arterial hypertension.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodDTLdxOMx-LVg90H21EOLACvtfcHk0lg9mSB45XS0cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitVSjsbY%253D&md5=b99b21f5120251ca1ff421613db6ef6e</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1161%2F01.RES.0000061180.03813.0F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.RES.0000061180.03813.0F%26sid%3Dliteratum%253Aachs%26aulast%3DFouty%26aufirst%3DB.%2BW.%26aulast%3DRodman%26aufirst%3DD.%2BM.%26atitle%3DMevastatin%2520can%2520cause%2520G1%2520arrest%2520and%2520induce%2520apoptosis%2520in%2520pulmonary%2520artery%2520smooth%2520muscle%2520cells%2520through%2520a%2520p27KIP1-independent%2520pathway%26jtitle%3DCirc.%2520Res.%26date%3D2003%26volume%3D92%26spage%3D501%26epage%3D509%26doi%3D10.1161%2F01.RES.0000061180.03813.0F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strickland, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windsor, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanese, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesarz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Rosario, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deskus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallams, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Njoroge, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piwinski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remiszewski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossman, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taveras, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vibulbhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doll, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girijavallabhan, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganguly, A. K.</span></span> <span> </span><span class="NLM_article-title">Tricyclic farnesyl protein transferase inhibitors: Crystallographic and calorimetric studies of structure-activity relationships</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">2125</span>– <span class="NLM_lpage">2135</span>, <span class="refDoi"> DOI: 10.1021/jm990030g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm990030g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADyaK1MXjtlOltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=2125-2135&author=C.+L.+Stricklandauthor=P.+C.+Weberauthor=W.+T.+Windsorauthor=Z.+Wuauthor=H.+V.+Leauthor=M.+M.+Albaneseauthor=C.+S.+Alvarezauthor=D.+Cesarzauthor=J.+del+Rosarioauthor=J.+Deskusauthor=A.+K.+Mallamsauthor=F.+G.+Njorogeauthor=J.+J.+Piwinskiauthor=S.+Remiszewskiauthor=R.+R.+Rossmanauthor=A.+G.+Taverasauthor=B.+Vibulbhanauthor=R.+J.+Dollauthor=V.+M.+Girijavallabhanauthor=A.+K.+Ganguly&title=Tricyclic+farnesyl+protein+transferase+inhibitors%3A+Crystallographic+and+calorimetric+studies+of+structure-activity+relationships&doi=10.1021%2Fjm990030g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Tricyclic Farnesyl Protein Transferase Inhibitors: Crystallographic and Calorimetric Studies of Structure-Activity Relationships</span></div><div class="casAuthors">Strickland, Corey L.; Weber, Patricia C.; Windsor, William T.; Wu, Zhen; Le, Hung V.; Albanese, Margaret M.; Alvarez, Carmen S.; Cesarz, David; del Rosario, Joycelyn; Deskus, Jeffrey; Mallams, Alan K.; Njoroge, F. George; Piwinski, John J.; Remiszewski, Stacy; Rossman, Randall R.; Taveras, Arthur G.; Vibulbhan, Bancha; Doll, Ronald J.; Girijavallabhan, Viyyoor M.; Ganguly, Ashit K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2125-2135</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Crystallog. and thermodn. studies of farnesyl protein transferase (FPT) complexed with novel tricyclic inhibitors provide insights into the obsd. SAR for this unique class of nonpeptidic FPT inhibitors.  The crystallog. structures reveal a binding pattern conserved across the mono-, di-, and trihalogen series.  In the complexes, the tricycle spans the FPT active site cavity and interacts with both protein atoms and the isoprenoid portion of bound farnesyl diphosphate.  An amide carbonyl, common to the tricyclic compds. described here, participates in a water-mediated hydrogen bond to the protein backbone.  Ten high-resoln. crystal structures of inhibitors complexed with FPT are reported.  Included are crystallog. data for FPT complexed with SCH 66336, a compd. currently undergoing clin. trials as an anticancer agent (SCH 66336, 4-[2-[4-(3,10-dibromo-8-chloro-6,11-dihydro-5H-benzo[5,6]cyclohepta[1,2-b]pyridin-11-yl)-1-piperidinyl]-2-oxoethyl]-1-piperidinecarboxamide).  Thermodn. binding parameters show favorable enthalpies of complex formation and small net entropic contributions as obsd. for 4-[2-[4-(3,10-dibromo-8-chloro-6,11-dihydro-11H-benzo[5,6]cyclohepta[1,2-b]pyridin-11-ylidene)-1-piperidinyl]-2-oxoethyl]pyridine N-oxide where ΔH°bind = -12.5 kcal/mol and TΔS°bind = -1.5 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0d_n_7Jl-B7Vg90H21EOLACvtfcHk0lg9mSB45XS0cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtlOltr8%253D&md5=c49d51c4c82ff82f248091c4d374374c</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fjm990030g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990030g%26sid%3Dliteratum%253Aachs%26aulast%3DStrickland%26aufirst%3DC.%2BL.%26aulast%3DWeber%26aufirst%3DP.%2BC.%26aulast%3DWindsor%26aufirst%3DW.%2BT.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DLe%26aufirst%3DH.%2BV.%26aulast%3DAlbanese%26aufirst%3DM.%2BM.%26aulast%3DAlvarez%26aufirst%3DC.%2BS.%26aulast%3DCesarz%26aufirst%3DD.%26aulast%3Ddel%2BRosario%26aufirst%3DJ.%26aulast%3DDeskus%26aufirst%3DJ.%26aulast%3DMallams%26aufirst%3DA.%2BK.%26aulast%3DNjoroge%26aufirst%3DF.%2BG.%26aulast%3DPiwinski%26aufirst%3DJ.%2BJ.%26aulast%3DRemiszewski%26aufirst%3DS.%26aulast%3DRossman%26aufirst%3DR.%2BR.%26aulast%3DTaveras%26aufirst%3DA.%2BG.%26aulast%3DVibulbhan%26aufirst%3DB.%26aulast%3DDoll%26aufirst%3DR.%2BJ.%26aulast%3DGirijavallabhan%26aufirst%3DV.%2BM.%26aulast%3DGanguly%26aufirst%3DA.%2BK.%26atitle%3DTricyclic%2520farnesyl%2520protein%2520transferase%2520inhibitors%253A%2520Crystallographic%2520and%2520calorimetric%2520studies%2520of%2520structure-activity%2520relationships%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D2125%26epage%3D2135%26doi%3D10.1021%2Fjm990030g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walker, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacold, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perisic, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wymann, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R. L.</span></span> <span> </span><span class="NLM_article-title">Structural determinants of phosphoinositide 3-kinase inhibition by wortmannin, LY294002, quercetin, myricetin, and staurosporine</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">909</span>– <span class="NLM_lpage">919</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(05)00089-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2FS1097-2765%2805%2900089-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=11090628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVegurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=909-919&author=E.+H.+Walkerauthor=M.+E.+Pacoldauthor=O.+Perisicauthor=L.+Stephensauthor=P.+T.+Hawkinsauthor=M.+P.+Wymannauthor=R.+L.+Williams&title=Structural+determinants+of+phosphoinositide+3-kinase+inhibition+by+wortmannin%2C+LY294002%2C+quercetin%2C+myricetin%2C+and+staurosporine&doi=10.1016%2FS1097-2765%2805%2900089-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants of phosphoinositide 3-kinase inhibition by wortmannin, LY294002, quercetin, myricetin, and staurosporine</span></div><div class="casAuthors">Walker, Edward H.; Pacold, Michael E.; Perisic, Olga; Stephens, Len; Hawkins, Philip T.; Wymann, Matthias P.; Williams, Roger L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">909-919</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The specific phosphoinositide 3-kinase (P13K) inhibitors wortmannin and LY294002 have been invaluable tools for elucidating the roles of these enzymes in signal transduction pathways.  The X-ray crystallog. structures of P13K bound to these lipid kinase inhibitors and to the broad-spectrum protein kinase inhibitors quercetin, myricetin, and staurosporine reveal how these compds. fit into the ATP binding pocket.  With a nanomolar IC50, wortmannin most closely fits and fills the active site and induces a conformational change in the catalytic domain.  Surprisingly, LY294002 and the lead compd. on which it was designed, quercetin, as well as the closely related flavonoid myricetin bind P13K in remarkably different orientations that are related to each other by 1800 rotations.  Staurosporine/P13K interactions are reminiscent of low-affinity protein kinase/staurosporine complexes.  These results provide a rich basis for development of isoform-specific P13K inhibitors with therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgIbTky2xQmbVg90H21EOLACvtfcHk0lj2ZsW7eQFqpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVegurY%253D&md5=5dd92f9b48c437e43c0023fdff10c1ac</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2805%2900089-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252805%252900089-4%26sid%3Dliteratum%253Aachs%26aulast%3DWalker%26aufirst%3DE.%2BH.%26aulast%3DPacold%26aufirst%3DM.%2BE.%26aulast%3DPerisic%26aufirst%3DO.%26aulast%3DStephens%26aufirst%3DL.%26aulast%3DHawkins%26aufirst%3DP.%2BT.%26aulast%3DWymann%26aufirst%3DM.%2BP.%26aulast%3DWilliams%26aufirst%3DR.%2BL.%26atitle%3DStructural%2520determinants%2520of%2520phosphoinositide%25203-kinase%2520inhibition%2520by%2520wortmannin%252C%2520LY294002%252C%2520quercetin%252C%2520myricetin%252C%2520and%2520staurosporine%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D6%26spage%3D909%26epage%3D919%26doi%3D10.1016%2FS1097-2765%2805%2900089-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Folkes, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmadi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alderton, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alix, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuckowree, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depledge, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eccles, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancox, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kugendradas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lensun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivero, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pergl-Wilson, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynaud, F. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saghir, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salphati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallweber, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhyvoloup, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zvelebil, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuttleworth, S. J.</span></span> <span> </span><span class="NLM_article-title">The identification of 2-(1h-indazol-4-yl)-6-(4-methanesulfonyl-piperazin-1-ylmethyl)-4-morpholin-4-yl-t hieno[3,2-d]pyrimidine (GDC-0941) as a potent, selective, orally bioavailable inhibitor of class I PI3 kinase for the treatment of cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5522</span>– <span class="NLM_lpage">5532</span>, <span class="refDoi"> DOI: 10.1021/jm800295d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800295d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVCmsr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5522-5532&author=A.+J.+Folkesauthor=K.+Ahmadiauthor=W.+K.+Aldertonauthor=S.+Alixauthor=S.+J.+Bakerauthor=G.+Boxauthor=I.+S.+Chuckowreeauthor=P.+A.+Clarkeauthor=P.+Depledgeauthor=S.+A.+Ecclesauthor=L.+S.+Friedmanauthor=A.+Hayesauthor=T.+C.+Hancoxauthor=A.+Kugendradasauthor=L.+Lensunauthor=P.+Mooreauthor=A.+G.+Oliveroauthor=J.+Pangauthor=S.+Patelauthor=G.+H.+Pergl-Wilsonauthor=F.+I.+Raynaudauthor=A.+Robsonauthor=N.+Saghirauthor=L.+Salphatiauthor=S.+Sohalauthor=M.+H.+Ultschauthor=M.+Valentiauthor=H.+J.+Wallweberauthor=N.+C.+Wanauthor=C.+Wiesmannauthor=P.+Workmanauthor=A.+Zhyvoloupauthor=M.+J.+Zvelebilauthor=S.+J.+Shuttleworth&title=The+identification+of+2-%281h-indazol-4-yl%29-6-%284-methanesulfonyl-piperazin-1-ylmethyl%29-4-morpholin-4-yl-t+hieno%5B3%2C2-d%5Dpyrimidine+%28GDC-0941%29+as+a+potent%2C+selective%2C+orally+bioavailable+inhibitor+of+class+I+PI3+kinase+for+the+treatment+of+cancer&doi=10.1021%2Fjm800295d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">The identification of 2-(1H-indazol-4-yl)-6-(4-methanesulfonyl-piperazin-1-ylmethyl)-4-morpholin-4-yl-thieno[3,2-d]pyrimidine (GDC-0941) as a potent, selective, orally bioavailable inhibitor of class I PI3 kinase for the treatment of cancer</span></div><div class="casAuthors">Folkes, Adrian J.; Ahmadi, Khatereh; Alderton, Wendy K.; Alix, Sonia; Baker, Stewart J.; Box, Gary; Chuckowree, Irina S.; Clarke, Paul A.; Depledge, Paul; Eccles, Suzanne A.; Friedman, Lori S.; Hayes, Angela; Hancox, Timothy C.; Kugendradas, Arumugam; Lensun, Letitia; Moore, Pauline; Olivero, Alan G.; Pang, Jodie; Patel, Sonal; Pergl-Wilson, Giles H.; Raynaud, Florence I.; Robson, Anthony; Saghir, Nahid; Salphati, Laurent; Sohal, Sukhjit; Ultsch, Mark H.; Valenti, Melanie; Wallweber, Heidi J. A.; Wan, Nan Chi; Wiesmann, Christian; Workman, Paul; Zhyvoloup, Alexander; Zvelebil, Marketa J.; Shuttleworth, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5522-5532</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphatidylinositol-3-kinase (PI3K) is an important target in cancer due to the deregulation of the PI3K/Akt signaling pathway in a wide variety of tumors.  A series of thieno[3,2-d]pyrimidine derivs. were prepd. and evaluated as inhibitors of PI3 kinase p110α.  The synthesis, biol. activity, and further profiling of these compds. are described.  This work resulted in the discovery of GDC-0941 (I) which is a potent, selective, orally bioavailable inhibitor of PI3K and is currently being evaluated in human clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojkFv29FXqO7Vg90H21EOLACvtfcHk0lj2ZsW7eQFqpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVCmsr7E&md5=742084954685353ceaf18660ab3a0703</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Fjm800295d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800295d%26sid%3Dliteratum%253Aachs%26aulast%3DFolkes%26aufirst%3DA.%2BJ.%26aulast%3DAhmadi%26aufirst%3DK.%26aulast%3DAlderton%26aufirst%3DW.%2BK.%26aulast%3DAlix%26aufirst%3DS.%26aulast%3DBaker%26aufirst%3DS.%2BJ.%26aulast%3DBox%26aufirst%3DG.%26aulast%3DChuckowree%26aufirst%3DI.%2BS.%26aulast%3DClarke%26aufirst%3DP.%2BA.%26aulast%3DDepledge%26aufirst%3DP.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DHayes%26aufirst%3DA.%26aulast%3DHancox%26aufirst%3DT.%2BC.%26aulast%3DKugendradas%26aufirst%3DA.%26aulast%3DLensun%26aufirst%3DL.%26aulast%3DMoore%26aufirst%3DP.%26aulast%3DOlivero%26aufirst%3DA.%2BG.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DPergl-Wilson%26aufirst%3DG.%2BH.%26aulast%3DRaynaud%26aufirst%3DF.%2BI.%26aulast%3DRobson%26aufirst%3DA.%26aulast%3DSaghir%26aufirst%3DN.%26aulast%3DSalphati%26aufirst%3DL.%26aulast%3DSohal%26aufirst%3DS.%26aulast%3DUltsch%26aufirst%3DM.%2BH.%26aulast%3DValenti%26aufirst%3DM.%26aulast%3DWallweber%26aufirst%3DH.%2BJ.%26aulast%3DWan%26aufirst%3DN.%2BC.%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DZhyvoloup%26aufirst%3DA.%26aulast%3DZvelebil%26aufirst%3DM.%2BJ.%26aulast%3DShuttleworth%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520identification%2520of%25202-%25281h-indazol-4-yl%2529-6-%25284-methanesulfonyl-piperazin-1-ylmethyl%2529-4-morpholin-4-yl-t%2520hieno%255B3%252C2-d%255Dpyrimidine%2520%2528GDC-0941%2529%2520as%2520a%2520potent%252C%2520selective%252C%2520orally%2520bioavailable%2520inhibitor%2520of%2520class%2520I%2520PI3%2520kinase%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5522%26epage%3D5532%26doi%3D10.1021%2Fjm800295d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vukovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huggins, D. J.</span></span> <span> </span><span class="NLM_article-title">Quantitative metrics for drug-target ligandability</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1258</span>– <span class="NLM_lpage">1266</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.02.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1016%2Fj.drudis.2018.02.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=29522887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVGrtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1258-1266&author=S.+Vukovicauthor=D.+J.+Huggins&title=Quantitative+metrics+for+drug-target+ligandability&doi=10.1016%2Fj.drudis.2018.02.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative metrics for drug-target ligandability</span></div><div class="casAuthors">Vukovic, Sinisa; Huggins, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1258-1266</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Ligandability is a prerequisite for druggability and is a much easier concept to understand, model and predict because it does not depend on the complex pharmacodynamic and pharmacokinetic mechanisms in the human body.  In this review, we consider a metric for quantifying ligandability from exptl. data.  We discuss ligandability in terms of the balance between effort and reward.  The metric is evaluated for a std. set of well-studied drug targets - some traditionally considered to be ligandable and some regarded as difficult.  We suggest that this metric should be used to systematically improve computational predictions of ligandability, which can then be applied to novel drug targets to predict their tractability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh-DRJxKcFEbVg90H21EOLACvtfcHk0lj2ZsW7eQFqpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVGrtbk%253D&md5=3f7d1b09c5dda9e0fa73c83faa655b41</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.02.015%26sid%3Dliteratum%253Aachs%26aulast%3DVukovic%26aufirst%3DS.%26aulast%3DHuggins%26aufirst%3DD.%2BJ.%26atitle%3DQuantitative%2520metrics%2520for%2520drug-target%2520ligandability%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D1258%26epage%3D1266%26doi%3D10.1016%2Fj.drudis.2018.02.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. H.</span></span> <span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">121</span>, <span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keseruauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lgU5y27C6XODQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wehling, M.</span></span> <span> </span><span class="NLM_article-title">Assessing the translatability of drug projects: What needs to be scored to predict success?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">541</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1038/nrd2898</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1038%2Fnrd2898" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=19543224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVylu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=541-546&author=M.+Wehling&title=Assessing+the+translatability+of+drug+projects%3A+What+needs+to+be+scored+to+predict+success%3F&doi=10.1038%2Fnrd2898"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the translatability of drug projects: What needs to be scored to predict success?</span></div><div class="casAuthors">Wehling, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">541-546</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drug development projects have high attrition rates, often because efficacy and safety issues have not been foreseen.  More effective prediction of 'translational success' could therefore have a key role in addressing the widely acknowledged problems with weak drug development pipelines.  Here, I discuss how a scoring system to systematically assess key determinants of translational success, such as biomarkers and animal and human data, could help identify deficiencies and potential improvements, and increase the reliability of portfolio risk ests.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWMac7ZHMqTrVg90H21EOLACvtfcHk0lgU5y27C6XODQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVylu74%253D&md5=41553f723f029a6d7077cffd70c3fe8b</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1038%2Fnrd2898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2898%26sid%3Dliteratum%253Aachs%26aulast%3DWehling%26aufirst%3DM.%26atitle%3DAssessing%2520the%2520translatability%2520of%2520drug%2520projects%253A%2520What%2520needs%2520to%2520be%2520scored%2520to%2520predict%2520success%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D541%26epage%3D546%26doi%3D10.1038%2Fnrd2898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexandrov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I.</span></span> <span> </span><span class="NLM_article-title">PDP: Protein domain parser</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">430</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btg006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=10.1093%2Fbioinformatics%2Fbtg006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=12584135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;key=1%3ACAS%3A528%3ADC%252BD3sXit1ygtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2003&pages=429-430&author=N.+Alexandrovauthor=I.+Shindyalov&title=PDP%3A+Protein+domain+parser&doi=10.1093%2Fbioinformatics%2Fbtg006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">PDP: protein domain parser</span></div><div class="casAuthors">Alexandrov, Nickolai; Shindyalov, Ilya</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">429-430</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The program PDP (Protein Domain Parser) has been developed for the automatic identification of domains in protein three-dimensional structures.  The performance of the program was assessed by three different benchmarks: (i) comparison with the expert-curated SCOP database of structural domains; (ii) by comparison with a collection of manual domain assignments; and (iii) by comparison with a set of 55 proteins, frequently used as a benchmark for automatic domain assignment.  In all of these benchmarks, PDP identified domains correctly in more than 80% of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq0_EOU1X10LVg90H21EOLACvtfcHk0lgU5y27C6XODQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXit1ygtb8%253D&md5=c1cdc1177c5a24694f4ba0b99e84d7e8</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtg006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtg006%26sid%3Dliteratum%253Aachs%26aulast%3DAlexandrov%26aufirst%3DN.%26aulast%3DShindyalov%26aufirst%3DI.%26atitle%3DPDP%253A%2520Protein%2520domain%2520parser%26jtitle%3DBioinformatics%26date%3D2003%26volume%3D19%26spage%3D429%26epage%3D430%26doi%3D10.1093%2Fbioinformatics%2Fbtg006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span> <i>The PyMOL Molecular
Graphics System</i>, version <span class="NLM_edition">1.8</span>; <span class="NLM_publisher-name">Schrodinger, LLC</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular%0AGraphics+System%2C+version+1.8%3B+Schrodinger%2C+LLC%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%250AGraphics%2520System%26pub%3DSchrodinger%252C%2520LLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FWZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FWZ','PDB','2FWZ'); return false;">PDB: 2FWZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1HWK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1HWK','PDB','1HWK'); return false;">PDB: 1HWK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DWC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DWC','PDB','1DWC'); return false;">PDB: 1DWC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V0Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V0Z','PDB','2V0Z'); return false;">PDB: 2V0Z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KMP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KMP','PDB','4KMP'); return false;">PDB: 4KMP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YIS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YIS','PDB','2YIS'); return false;">PDB: 2YIS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OXC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OXC','PDB','3OXC'); return false;">PDB: 3OXC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HWR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HWR','PDB','2HWR'); return false;">PDB: 2HWR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TWP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TWP','PDB','4TWP'); return false;">PDB: 4TWP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS9','PDB','3CS9'); return false;">PDB: 3CS9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MKC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MKC','PDB','4MKC'); return false;">PDB: 4MKC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IUH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IUH','PDB','5IUH'); return false;">PDB: 5IUH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK','PDB','1XKK'); return false;">PDB: 1XKK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P6W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P6W','PDB','4P6W'); return false;">PDB: 4P6W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HG5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HG5','PDB','5HG5'); return false;">PDB: 5HG5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CWA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CWA','PDB','1CWA'); return false;">PDB: 1CWA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HRA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HRA','PDB','4HRA'); return false;">PDB: 4HRA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EQB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EQB','PDB','5EQB'); return false;">PDB: 5EQB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LVT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LVT','PDB','4LVT'); return false;">PDB: 4LVT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4A5Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4A5Y','PDB','4A5Y'); return false;">PDB: 4A5Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PMZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PMZ','PDB','3PMZ'); return false;">PDB: 3PMZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NLD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NLD','PDB','4NLD'); return false;">PDB: 4NLD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1E8Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1E8Y','PDB','1E8Y'); return false;">PDB: 1E8Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KMU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KMU','PDB','4KMU'); return false;">PDB: 4KMU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VDN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VDN','PDB','2VDN'); return false;">PDB: 2VDN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JKV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JKV','PDB','4JKV'); return false;">PDB: 4JKV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QU2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QU2','PDB','1QU2'); return false;">PDB: 1QU2</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i49"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01732">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_25507"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01732">10.1021/acs.jmedchem.8b01732</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Graphical representation and statistical analysis of structure-based ligand binding profiles for all targets with more than five ligands (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01732/suppl_file/jm8b01732_si_001.pdf">jm8b01732_si_001.pdf (1.43 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01732%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-22" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01732" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67990546c867d19b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
