
<html lang="en"     class="pb-page"  data-request-id="0eba93ee-f2c4-43f9-8e06-1431a789b91a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.9b01094;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2019.62.issue-18;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Discovery and Development of a Series of Potent and Selective Bromodomain and Extra-Terminal Protein Inhibitors" /></meta><meta name="dc.Creator" content="Jianping  Hu" /></meta><meta name="dc.Creator" content="Chang-Qing  Tian" /></meta><meta name="dc.Creator" content="Mohammadali Soleimani  Damaneh" /></meta><meta name="dc.Creator" content="Yanlian  Li" /></meta><meta name="dc.Creator" content="Danyan  Cao" /></meta><meta name="dc.Creator" content="Kaikai  Lv" /></meta><meta name="dc.Creator" content="Ting  Yu" /></meta><meta name="dc.Creator" content="Tao  Meng" /></meta><meta name="dc.Creator" content="Danqi  Chen" /></meta><meta name="dc.Creator" content="Xin  Wang" /></meta><meta name="dc.Creator" content="Lin  Chen" /></meta><meta name="dc.Creator" content="Jian  Li" /></meta><meta name="dc.Creator" content="Shan-Shan  Song" /></meta><meta name="dc.Creator" content="Xia-Juan  Huan" /></meta><meta name="dc.Creator" content="Lihuai  Qin" /></meta><meta name="dc.Creator" content="Jingkang  Shen" /></meta><meta name="dc.Creator" content="Ying-Qing  Wang" /></meta><meta name="dc.Creator" content="Ze-Hong  Miao" /></meta><meta name="dc.Creator" content="Bing  Xiong" /></meta><meta name="dc.Description" content="BRD4 has recently emerged as a promising drug target. Therefore, identifying novel inhibitors with distinct properties could enrich their use in anticancer treatment. Guided by the cocrystal struct..." /></meta><meta name="Description" content="BRD4 has recently emerged as a promising drug target. Therefore, identifying novel inhibitors with distinct properties could enrich their use in anticancer treatment. Guided by the cocrystal struct..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 6, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01094" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01094" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01094" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01094" /></link>
        
    
    

<title>Structure-Based Discovery and Development of a Series of Potent and Selective Bromodomain and Extra-Terminal Protein Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01094" /></meta><meta property="og:title" content="Structure-Based Discovery and Development of a Series of Potent and Selective Bromodomain and Extra-Terminal Protein Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0002.jpeg" /></meta><meta property="og:description" content="BRD4 has recently emerged as a promising drug target. Therefore, identifying novel inhibitors with distinct properties could enrich their use in anticancer treatment. Guided by the cocrystal structure of hit compound 4 harboring a five-membered-ring linker motif, we quickly identified lead compound 7, which exhibited good antitumor effects in an MM.1S xenograft model by oral administration. Encouraged by its high potency and interesting scaffold, we performed further lead optimization to generate a novel potent series of bromodomain and extra-terminal (BET) inhibitors with a (1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1H)-one structure. Among them, compound 19 was found to have the best balance of activity, stability, and antitumor efficacy. After confirming its low brain penetration, we conducted comprehensive preclinical studies, including a multiple-species pharmacokinetics profile, extensive cellular mechanism studies, hERG assay, and in vivo antitumor growth effect testing, and we found that compound 19 is a potential BET protein drug candidate for the treatment of cancer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01094"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01094">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01094&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01094&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01094&amp;href=/doi/10.1021/acs.jmedchem.9b01094" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 18</span><span class="cit-fg-pageRange">, 8642-8663</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/18" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01003" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01384" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Discovery and Development of a Series of Potent and Selective Bromodomain and Extra-Terminal Protein Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jianping Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianping Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianping++Hu">Jianping Hu</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Chang-Qing Tian</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chang-Qing Tian</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Anti-tumor Pharmacology, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chang-Qing++Tian">Chang-Qing Tian</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Mohammadali Soleimani Damaneh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mohammadali Soleimani Damaneh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Anti-tumor Pharmacology, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mohammadali+Soleimani++Damaneh">Mohammadali Soleimani Damaneh</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Yanlian Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanlian Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanlian++Li">Yanlian Li</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Danyan Cao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Danyan Cao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Danyan++Cao">Danyan Cao</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Kaikai Lv</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kaikai Lv</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kaikai++Lv">Kaikai Lv</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ting Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ting Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ting++Yu">Ting Yu</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Tao Meng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tao Meng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tao++Meng">Tao Meng</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Danqi Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Danqi Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Danqi++Chen">Danqi Chen</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xin Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Wang">Xin Wang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Lin Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lin Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lin++Chen">Lin Chen</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jian Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Li">Jian Li</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Shan-Shan Song</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shan-Shan Song</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Anti-tumor Pharmacology, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shan-Shan++Song">Shan-Shan Song</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xia-Juan Huan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xia-Juan Huan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Anti-tumor Pharmacology, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xia-Juan++Huan">Xia-Juan Huan</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Lihuai Qin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lihuai Qin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Chemical Biology and Drug Discovery, Department of Pharmacological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lihuai++Qin">Lihuai Qin</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jingkang Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jingkang Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jingkang++Shen">Jingkang Shen</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ying-Qing Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying-Qing Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Anti-tumor Pharmacology, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#fd848c8a9c939abd8e949090d39c9ed39e93"><span class="__cf_email__" data-cfemail="80f9f1f7e1eee7c0f3e9ededaee1e3aee3ee">[emailÂ protected]</span></a> (Y.W.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying-Qing++Wang">Ying-Qing Wang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ze-Hong Miao</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ze-Hong Miao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Anti-tumor Pharmacology, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#ef958782868e80af9c868282c18e8cc18c81"><span class="__cf_email__" data-cfemail="28524045414947685b41454506494b064b46">[emailÂ protected]</span></a> (Z.M.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ze-Hong++Miao">Ze-Hong Miao</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Bing Xiong</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bing Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#33514b5a5c5d5473405a5e5e1d52501d505d"><span class="__cf_email__" data-cfemail="87e5ffeee8e9e0c7f4eeeaeaa9e6e4a9e4e9">[emailÂ protected]</span></a>. Tel: +86 21 50806600. ext. 5412. Fax: +86 21 50807088 (B.X.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bing++Xiong">Bing Xiong</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9776-8136" title="Orcid link">http://orcid.org/0000-0001-9776-8136</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01094&amp;href=/doi/10.1021%2Facs.jmedchem.9b01094" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 18</span><span class="cit-pageRange">, 8642â8663</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 6, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 July 2019</li><li><span class="item_label"><b>Published</b> online</span>6 September 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 September 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01094" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01094</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8642%26pageCount%3D22%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJianping%2BHu%252C%2BChang-Qing%2BTian%252C%2BMohammadali%2BSoleimani%2BDamaneh%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D18%26contentID%3Dacs.jmedchem.9b01094%26title%3DStructure-Based%2BDiscovery%2Band%2BDevelopment%2Bof%2Ba%2BSeries%2Bof%2BPotent%2Band%2BSelective%2BBromodomain%2Band%2BExtra-Terminal%2BProtein%2BInhibitors%26numPages%3D22%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8663%26publicationDate%3DSeptember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01094"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2348</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">9</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01094" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Discovery and Development of a Series of Potent and Selective Bromodomain and Extra-Terminal Protein Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jianping&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Chang-Qing&quot;,&quot;last_name&quot;:&quot;Tian&quot;},{&quot;first_name&quot;:&quot;Mohammadali&quot;,&quot;last_name&quot;:&quot;Soleimani Damaneh&quot;},{&quot;first_name&quot;:&quot;Yanlian&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Danyan&quot;,&quot;last_name&quot;:&quot;Cao&quot;},{&quot;first_name&quot;:&quot;Kaikai&quot;,&quot;last_name&quot;:&quot;Lv&quot;},{&quot;first_name&quot;:&quot;Ting&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Tao&quot;,&quot;last_name&quot;:&quot;Meng&quot;},{&quot;first_name&quot;:&quot;Danqi&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Lin&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Shan-Shan&quot;,&quot;last_name&quot;:&quot;Song&quot;},{&quot;first_name&quot;:&quot;Xia-Juan&quot;,&quot;last_name&quot;:&quot;Huan&quot;},{&quot;first_name&quot;:&quot;Lihuai&quot;,&quot;last_name&quot;:&quot;Qin&quot;},{&quot;first_name&quot;:&quot;Jingkang&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Ying-Qing&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Ze-Hong&quot;,&quot;last_name&quot;:&quot;Miao&quot;},{&quot;first_name&quot;:&quot;Bing&quot;,&quot;last_name&quot;:&quot;Xiong&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;06&quot;,&quot;issue&quot;:&quot;18&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;8642-8663&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01094&quot;},&quot;abstract&quot;:&quot;BRD4 has recently emerged as a promising drug target. Therefore, identifying novel inhibitors with distinct properties could enrich their use in anticancer treatment. Guided by the cocrystal structure of hit compound 4 harboring a five-membered-ring linker motif, we quickly identified lead compound 7, which exhibited good antitumor effects in an MM.1S xenograft model by oral administration. Encouraged by its high potency and interesting scaffold, we performed further lead optimization to generate a novel potent series of bromodomain and extra-terminal (BET) inhibitors with a (1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1H)-one structure. Among them, compound 19 was found to have the best balance of activity, stability, and antitumor efficacy. After confirming its low brain penetration, we conducted comprehensive preclinical studies, including a multiple-species pharmacokinetics profile, extensive cellular mechanism studies, hERG assay, and in vivo antitumor growth effect testing, and we found that compound 19&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01094&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01094" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01094&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01094" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01094&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01094" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01094&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01094&amp;href=/doi/10.1021/acs.jmedchem.9b01094" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01094" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01094" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01094%26sid%3Dliteratum%253Aachs%26pmid%3D31490070%26genre%3Darticle%26aulast%3DHu%26date%3D2019%26atitle%3DStructure-Based%2BDiscovery%2Band%2BDevelopment%2Bof%2Ba%2BSeries%2Bof%2BPotent%2Band%2BSelective%2BBromodomain%2Band%2BExtra-Terminal%2BProtein%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D18%26spage%3D8642%26epage%3D8663%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291149" title="Apoptosis">Apoptosis</a>,</li><li><a href="/action/doSearch?ConceptID=292586" title="Protein dynamics">Protein dynamics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/18" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/jmcmar.2019.62.issue-18/20190926/jmcmar.2019.62.issue-18.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0002.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">BRD4 has recently emerged as a promising drug target. Therefore, identifying novel inhibitors with distinct properties could enrich their use in anticancer treatment. Guided by the cocrystal structure of hit compound <b>4</b> harboring a five-membered-ring linker motif, we quickly identified lead compound <b>7</b>, which exhibited good antitumor effects in an MM.1S xenograft model by oral administration. Encouraged by its high potency and interesting scaffold, we performed further lead optimization to generate a novel potent series of bromodomain and extra-terminal (BET) inhibitors with a (1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one structure. Among them, compound <b>19</b> was found to have the best balance of activity, stability, and antitumor efficacy. After confirming its low brain penetration, we conducted comprehensive preclinical studies, including a multiple-species pharmacokinetics profile, extensive cellular mechanism studies, hERG assay, and in vivo antitumor growth effect testing, and we found that compound <b>19</b> is a potential BET protein drug candidate for the treatment of cancer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86447" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86447" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Bromodomain-containing proteins (BCPs) can decipher the acetylation code of proteins via binding to their acetylated lysine (KAc) residues, which further modulates acetylation signal transduction and produces diverse physiological functions.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1â3)</a> In many diseases, and best studied in cancers, BCPs are found to be aberrantly expressed, and they further enhanced the expression of oncogenes and anti-apoptotic proteins.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Thus, BCPs have emerged as attractive epigenetic therapeutic targets for cancer.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a></div><div class="NLM_p">There are 61 bromodomains distributed in 46 distinct proteins encoded in the human genome, and these bromodomains can be divided into eight subfamilies.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Among them, the most studied is the bromodomain and extra-terminal domain (BET) subfamily, which includes the following four members: BRD2, BRD3, BRD4, and BRDT.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Since the first BET inhibitor, (+)-JQ-1 (<b>1</b>),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> was discovered, diverse BET inhibitors have arisen, with some of them entering into clinical trials, including OTX-015 (<b>2</b>),<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> I-BET762 (<b>3</b>),<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> CPI0610,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and ABBV-075,<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> to name a few.</div><div class="NLM_p">However, the cellular functions of BET proteins, specifically BRD4, are more complex than previously thought.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> As an epigenetic reader, BRD4 was initially believed to recognize the acetylated lysine in H4 with its two N-terminal bromodomains, further recruiting the transcription elongation factor b (p-TEFb) cyclin T1/CDK9 complex to phosphorylate RNA polymerase II (RNA pol II) to activate gene transcription.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The profound downstream effect of this event was initially attributed to an increased expression of the oncogene <i>Myc</i>, which encodes the multipurpose Myc protein related to cell growth and differentiation.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Since directly targeting Myc protein was very difficult, targeting BRD4 to interrupt p-TEFb binding to histones was considered as a surrogate for Myc-related cancer treatment.</div><div class="NLM_p">Recently, with the advance of sequencing technologies, many other cellular roles of BRD4 have been revealed from transcriptome and epigenome studies. In addition to binding to acetylated H4, BRD4 can also act as a critical component of the superenhancer complex by binding to many transcription factors, including Myc, TWIST,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> CEBPA, CEBPB,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and the transcription-related protein RelA.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Meanwhile, new downstream effects have been discovered; for example, BET protein inhibition can promote antitumor immunity by suppressing PD-L1 expression.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Playing nontranscriptional roles, BRD4 has also been shown to control DNA damage checkpoint activation and repair as well as telomere maintenance.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Using (+)-JQ1 as a chemical probe, BRD4 was identified to have an essential cognitive function in the brain, and BRD4 inhibition in the brain could impair memory abilities.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> These recent studies on the different functions of BRD4 reinforced the argument of Andrieu et al. that clinical trials for BET inhibitors should design with caution as currently, we are still in the primary phase of understanding the roles of BRD4.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_p">Nevertheless, the important role of BRD4 in cancer biology and partial successes of clinical trials with BET inhibitors<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> have spurred researchers to pursue new inhibitors with novel scaffolds as they may bring distinct values with different pharmacological effects or tissue distribution. Researchers recently discovered previous patents that described a series of dihydroquinoxalin-2(1<i>H</i>)-ones with substituted or unsubstituted nitrogen or oxygen linkers as BET inhibitors.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> Moreover, researchers such as Conway et al. revealed a series of potent CREBBP bromodomain inhibitors with a dihydroquinoxalinone scaffold, and they emphasized the importance of the internal hydrogen bond, which is formed from the scaffold amine NH with the side-chain carbonyl.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Meanwhile, we previously incrementally mapped out some novel BRD4-selective inhibitors with a dihydroquinoxalin-2(1<i>H</i>)-one scaffold that were derived from the dual PLK1-BRD4 inhibitor BI-2536.<a onclick="showRef(event, 'ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35">(33â35)</a> However, they did not possess comprehensive drug-like properties. Furthermore, we found that others had also tried to tune the selectivity and potency of BI-2536; for example, Watts et al. designed a series of anaplastic lymphoma kinase (ALK) and BRD4 dual inhibitors from BI-2536,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and Liu et al. described a series of compounds with different selectivity between PLK1 and BRD4 using BI-2536 as the starting point.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div><div class="NLM_p">Now, in this research, by preserving the core dihydroquinoxalin-2(1<i>H</i>)-ones modified from BI2536, we explored new side chains and discovered novel hit compound <b>4</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Guided by crystal structures, we were able to discover a potent and selective inhibitor (<b>19</b>) as a potential drug candidate. The extensive profile of this compound showed that it displayed high selectivity for the BET subfamily over other bromodomains and more than twenty kinases from different families, possessed favorable metabolic stability in multiple species, showed a low bloodâbrain barrier penetration ratio and good PK properties in rodents and dogs, and demonstrated excellent in vivo antitumor efficacy. Hereinafter, we report on the optimization of this potential drug candidate as well as comprehensive profiles of its in vitro and in vivo properties.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0023.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of typical BET inhibitors including (+)-JQ1 (<b>1</b>), OTX-015 (<b>2</b>), I-BET762 (<b>3</b>), and hit compound <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03824" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03824" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Hit to Lead</h3><div class="NLM_p">Based on the BET-selective inhibitors reported in our previous work,<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> we wanted to further improve their in vitro and in vivo properties to develop a drug candidate for clinical research. We initially preserved the core dihydroquinoxalin-2(1<i>H</i>)-one and screened for new side chains, finding that compound <b>4</b> with a pyrrole ring had moderate activity (IC<sub>50</sub> = 303.0 nM) and a low molecular weight that could be used as a starting point. Superimposition of the cocrystal structures of a reported BET inhibitor (<b>5</b>)<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> in BRD4-BD2 to our crystal structure of <b>4</b> bound to BRD4-BD1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) revealed that there was a large space for modification around the pyrrole ring. From our previous study,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> we believed that the <i>ortho</i> position could be extended to reach the WPF subpocket to improve the potency and selectivity. Hence, we added a hydrophobic group (e.g., phenyl or <i>p</i>-methyl phenyl) to this position and synthesized different five-membered ring linkers to generate a series of dihydroquinoxalin-2(1<i>H</i>)-one compounds (<b>6</b>â<b>12</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The FP binding assay showed that they all had excellent potency, indicating this hydrophobic group may enter the WPF subpocket.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0024.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Superimposition of the cocrystal structure of BRD4 BD1 bound with <b>4</b> (PDB entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI3">6JI3</a>, green color) and BRD4-BD2 bound with compound <b>5</b> (PDB entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z93">4Z93</a>, gray color).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibitory Activity and MM.1S Cell Proliferation Inhibitory Activity Results of Compounds <b>6</b>â<b>12</b> on BRD4 (I)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0007.gif" alt="" id="gr17" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0008.gif" alt="" id="gr18" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are the mean of two independent experiments performed in duplicate with standard error of the mean (SEM) values.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values are the mean of three independent experiments performed in triplicate with standard error of the mean (SEM) values.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">N.D. means not determined.</p></div></div><div></div></div><div class="NLM_p">As listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, we kept R<sup>1</sup> as racemic methyl or (<i>R</i>)-methyl, R<sup>2</sup> as cyclopropyl or cyclopentyl, and R<sup>3</sup> as methyl or hydrogen to explore the five-membered ring linker. A comparison of compound <b>6</b> with <b>7</b> found that the addition of a methyl group to the triazole could enhance the molecular and cellular activity by 1.5- and 2.0-fold, respectively. By substituting the triazole group with a tetrazole ring, we found that the molecular binding activities were only slightly decreased (<b>6</b> versus <b>8</b> and <b>10</b> versus <b>11</b>); however, the cellular antiproliferative activities were dramatically reduced, indicating that the more hydrophilic tetrazole may decrease the membrane permeability of the compounds. Replacement of the triazole with thiazolidinone or methylisoxazole resulted in a significant loss of activity (<b>6</b> versus <b>9</b> or <b>10</b> versus <b>12</b>). These results indicated that the methyl position of the five-membered ring may be surrounded by small hydrophobic residues and could not tolerate the polar nitrogen or carbonyl group. Only by changing R<sup>2</sup> cyclopropyl group to a cyclopentyl group could the molecular activity be maintained; in contrast, the cellular activity was dependent on the overall molecule properties and could be retained (<b>7</b> and <b>10</b>) or weakened (<b>8</b> and <b>11</b>). These results indicated that the nature of the five-membered ring linker and R<sup>2</sup> group significantly affected their activities; in the future, perhaps we can change the linker to a pyrazole, pyrrole, or six-membered linker such as phenyl or pyridine to explore more potent compounds. In short, compounds <b>6</b>, <b>7</b>, and <b>10</b> were found to have good molecular and cellular activities with IC<sub>50</sub> values below 10 nM, which were worth further exploration.</div><div class="NLM_p">According to the molecular and cellular activities, compounds <b>6</b>, <b>7</b>, and <b>10</b> were selected for in vitro metabolic stability evaluation (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Table S2</a>). We found that compounds <b>6</b> and <b>7</b> had better metabolic stabilities than compound <b>10</b> in both human and mouse liver microsomes. At 10 Î¼M, compound <b>6</b> showed evident inhibition of most of the tested CYP450 enzymes; in contrast, compounds <b>7</b> and <b>10</b> exhibited a weak inhibitory effect on the CYP450 enzymes, except that compound <b>10</b> showed a more than 50% inhibition ratio for the 2C9 enzyme. Additionally, none of the compounds exhibited time-dependent inhibition of CYP450 enzymes, indicating less vulnerability of these compounds during drugâdrug interactions.</div><div class="NLM_p">Considering the liver microsomal metabolic stability, we chose compounds <b>6</b> and <b>7</b> for in vivo pharmacokinetics (PK) study (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Pharmacokinetic tests demonstrated that compound <b>7</b> indeed possessed better plasma exposure after oral administration at a 3 mg/kg dosage than compound <b>6</b> at a 10 mg/kg dosage, suggesting that compound <b>7</b> could be advanced to an in vivo pharmacological study.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. In Vivo PK Data for Compounds <b>6</b> and <b>7</b> in Male ICR Mice<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd</th><th class="colsep0 rowsep0" align="center">route of administration</th><th class="colsep0 rowsep0" align="center" char=".">dose (mg/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>max</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub> (ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0-t</sub> (ngÂ·h/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0-inf</sub> (ngÂ·h/mL)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>6</b></td><td class="colsep0 rowsep0" align="left">p.o.</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">163</td><td class="colsep0 rowsep0" align="char" char=".">161</td><td class="colsep0 rowsep0" align="char" char=".">164</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>7</b></td><td class="colsep0 rowsep0" align="left">p.o.</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">265</td><td class="colsep0 rowsep0" align="char" char=".">243</td><td class="colsep0 rowsep0" align="char" char=".">244</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">1% DMSO/0.5% HPMC was used as the vehicle.</p></div></div></div><div class="NLM_p last">The in vivo therapeutic effects of compound <b>7</b> were evaluated in a human MM.1S xenograft model. Tumor-bearing SCID mice were treated with <b>7</b> or the phase II BET inhibitor I-BET762 by oral administration at a dosage of 20 mg/kg daily. The results showed that treating with compound <b>7</b> demonstrated good efficacy on tumor growth inhibition with a <i>T</i>/<i>C</i> value of 67.4% (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Figure S1</a>). These encouraging results demonstrated that this new series of compounds bearing a novel triazole linker deserved further optimization.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Lead to Candidate</h3><div class="NLM_p">The good in vivo antitumor efficacy of lead compound <b>7</b> indicated that this class of compounds should be advanced for further exploitation. However, the moderate plasma exposure of compound <b>7</b> and its instability in mouse liver microsomes limited its subsequent development. To confirm the binding mode and guide further optimization, we solved the cocrystal structure of compound <b>7</b> bound to BRD4-BD1. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, the methyl-benzene group indeed stretched into the WPF shelf and contributed favorable VDW interactions to increase the binding affinity. Meanwhile, it was also found that the position adjacent to the methyl group on the triazole ring pointed to the solvent-accessible part, which could be utilized for improving the drug-like properties. Considering its fast metabolic rate and bad solubility (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Table S3</a>), we intended to introduce a hydrophilic group into this position to reduce the binding potential to metabolism enzymes and improve the solubility.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0022.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of compound <b>7</b> bound to BRD4-BD1 (PDB entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI4">6JI4</a>) is shown in cartoon mode and surface mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We preserved the core structure of compound <b>7</b> or compound <b>10</b>, which also displayed excellent molecular and cellular potency, and added a hydrophilic piperazine group to the five-membered ring linker to generate compounds <b>13</b> and <b>14</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), respectively, finding that their molecular and cellular activities could be retained. Hence, we evaluated the drug-like properties of compounds <b>13</b> and <b>14</b>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Inhibitory Activity and MM.1S Cell Proliferation Inhibitory Activity Results of Compounds <b>13</b> and <b>14</b> on BRD4 (I)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0006.gif" alt="" id="gr19" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0001.gif" alt="" id="gr20" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are the mean of two independent experiments performed in duplicate with standard error of the mean (SEM) values.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values are the mean of three independent experiments performed in triplicate with standard error of the mean (SEM) values.</p></div></div><div></div></div><div class="NLM_p">We first evaluated the in vitro metabolic stabilities of compounds <b>13</b> and <b>14</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Comparison of compound <b>13</b> or <b>14</b> with compound <b>7</b> demonstrated that compounds <b>13</b> and <b>14</b> had remarkably improved metabolic stability in mouse liver microsomes, indicating that the addition of a hydrophilic group could decrease the hydrophobic properties of the compounds to reduce their binding probability with metabolic enzymes. Moreover, they did not show evident direct or time-dependent inhibition on five selected CYP450 enzymes, revealing the potential defect in the drugâdrug interaction.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. In Vitro Liver Microsome Intrinsic Clearance Data and Cytochrome P450 Enzyme Inhibition Data</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="2" align="center">HLM<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="2" align="center">mLM<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="rowsep1 colsep0" colspan="5" align="center">DIÂ (%)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a>/TDI (<i>k</i><sub>obs</sub>Â ÃÂ 10<sup>â4</sup>)<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd</th><th class="colsep0 rowsep0" align="center">clearance (Î¼L/min/mg)</th><th class="colsep0 rowsep0" align="center"><i>T</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">clearance (Î¼L/min/mg)</th><th class="colsep0 rowsep0" align="center"><i>T</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">3A4</th><th class="colsep0 rowsep0" align="center">2D6</th><th class="colsep0 rowsep0" align="center">2C9</th><th class="colsep0 rowsep0" align="center">1A2</th><th class="colsep0 rowsep0" align="center">2C19</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>7</b></td><td class="colsep0 rowsep0" align="center">44</td><td class="colsep0 rowsep0" align="center">31.5</td><td class="colsep0 rowsep0" align="center">460</td><td class="colsep0 rowsep0" align="center">3.0</td><td class="colsep0 rowsep0" align="center">3/no</td><td class="colsep0 rowsep0" align="center">no/no</td><td class="colsep0 rowsep0" align="center">10/no</td><td class="colsep0 rowsep0" align="center">no/no</td><td class="colsep0 rowsep0" align="center">41/no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13</b></td><td class="colsep0 rowsep0" align="center">22</td><td class="colsep0 rowsep0" align="center">62</td><td class="colsep0 rowsep0" align="center">13</td><td class="colsep0 rowsep0" align="center">107</td><td class="colsep0 rowsep0" align="center">12/19</td><td class="colsep0 rowsep0" align="center">no/14</td><td class="colsep0 rowsep0" align="center">6/19</td><td class="colsep0 rowsep0" align="center">2/41</td><td class="colsep0 rowsep0" align="center">17/no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>14</b></td><td class="colsep0 rowsep0" align="center">61</td><td class="colsep0 rowsep0" align="center">23</td><td class="colsep0 rowsep0" align="center">34</td><td class="colsep0 rowsep0" align="center">41</td><td class="colsep0 rowsep0" align="center">31/55</td><td class="colsep0 rowsep0" align="center">3/no</td><td class="colsep0 rowsep0" align="center">30/74</td><td class="colsep0 rowsep0" align="center">no/51</td><td class="colsep0 rowsep0" align="center">19/141</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">HLM represents the human liver microsome.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">mLM represents the mouse liver microsome.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">DI, which indicated direct inhibition and is sometimes referred to as reversible inhibition, is assessed by measuring enzyme (CYP) activity in the presence of increasing concentrations of inhibitor without a preincubation step. DI < 20% means no direct inhibition, 20% < DI < 50% means weak direct inhibition, 50% < DI < 70% means moderate direct inhibition, and DI > 70% means strong direct inhibition.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">TDI stands for the time-dependent inhibition, which refers to a change in enzyme inhibition during in vitro incubation and means irreversible inactivation of CYPs. The unit of TDI is 10<sup>â4</sup>/min, and TDI < 200 means no time-dependent inhibition. When the TDI value was close to zero, it was considered as none.</p></div></div></div><div class="NLM_p">Encouraged by this result, we further investigated the in vivo pharmacokinetics properties of compounds <b>13</b> and <b>14</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Pharmacokinetic tests showed that compound <b>14</b> had much better plasma exposure than compound <b>13</b> at the same 10 mg/kg dosage; when compared to compound <b>7</b>, both compounds improved the plasma exposure considerably, indicating that the addition of a hydrophilic group to the triazole ring could enhance the solubility to increase oral absorption.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. In Vivo PK Data for Compounds <b>13</b> and <b>14</b> in Male ICR Mice<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd</th><th class="colsep0 rowsep0" align="center">route of administration</th><th class="colsep0 rowsep0" align="center" char=".">dose (mg/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>max</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub> (ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0-t</sub> (ngÂ·h/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0-inf</sub> (ngÂ·h/mL)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13</b></td><td class="colsep0 rowsep0" align="left">p.o.</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">1231</td><td class="colsep0 rowsep0" align="char" char=".">2742</td><td class="colsep0 rowsep0" align="char" char=".">2764</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>14</b></td><td class="colsep0 rowsep0" align="left">p.o.</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">42,904</td><td class="colsep0 rowsep0" align="char" char=".">58,132</td><td class="colsep0 rowsep0" align="char" char=".">58,146</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">1% DMSO/0.5% HPMC was used as the vehicle.</p></div></div></div><div class="NLM_p">Hence, we further explored the structureâactivity relationships of the hydrophilic group based on the core structure of <b>14</b>. Initially, we explored the SAR of the hydrophilic piperazine position. As elucidated in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>, opening the <i>N</i>-methylpiperazine to an <i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>,<i>N</i><sup>2</sup>-trimethylethane-1,2-diamine group (<b>15</b>), replacing the <i>N</i>-methylpiperazine methyl group with ethyl, isopropyl, or cyclopropyl (<b>16</b>â<b>18</b>), or substituting the <i>N</i>-methylpiperazine with (2<i>S</i>,6<i>R</i>)-2,6-dimethylpiperazine (<b>19</b>) or larger groups, such as 1-methyl-1,4-diazepane (<b>20</b>) or a bridged ring, spiro ring, or fused ring (<b>21</b>â<b>27</b>), could maintain the molecular activity, suggesting that the hydrophilic <i>N</i>-methylpiperazine group might be exposed to the solvent moiety of the protein and did not make direct interactions with the protein. We attempted to resolve the crystal structures of compounds <b>14</b>, <b>15</b>, <b>19</b>, and <b>22</b> with the BRD4-BD1 protein to verify this conjecture but had no success. However, a later crystal structure study did confirm this hypothesis (discussed below). Although their molecular activities were similar, compounds <b>15</b>, <b>16</b>, and <b>19</b>â<b>23</b> had better activities in the MM.1S cell viability test, which may be due to the subtle structural changes that affect compound permeability.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Inhibitory Activity and MM.1S Cell Proliferation Inhibitory Activity Results of Compounds <b>14</b>â<b>27</b> on BRD4 (I)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0004.gif" alt="" id="gr21" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0005.gif" alt="" id="gr22" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are the mean of two independent experiments performed in duplicate with standard error of the mean (SEM) values.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values are the mean of three independent experiments performed in triplicate with standard error of the mean (SEM) values.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">N.D. means not determined.</p></div></div><div></div></div><div class="NLM_p">We also explored the SAR of the side-chain benzene ring (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). By comparing compounds <b>28</b> and <b>31</b> with compound <b>14</b>, we found that removal of the <i>para</i>-methyl group or replacement of the methyl with a chlorine substituent slightly reduced the molecular activity. However, the introduction of a methyl group at the <i>ortho</i> position or a fluorine substituent at the <i>meta</i> position could slightly improve the molecular and cellular activities, indicating that the addition of another small hydrophobic group such as a methyl or fluoro group could enhance the VDW interaction of the side chain with the WPF pocket.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Inhibitory Activity and MM.1S Cell Proliferation Inhibitory Activity Results of Compounds <b>28</b>â<b>31</b> on BRD4(I)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0003.gif" alt="" id="gr23" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="left" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd</th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center" char="Â±">BRD4(I) IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="Â±">MM.1S IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">cpd</th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center" char="Â±">BRD4(I) IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="Â±">MM.1S IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">(+)-JQ-1</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">28.5Â Â±Â 1.5</td><td class="colsep0 rowsep0" align="char" char="Â±">29.8Â Â±Â 16.6</td><td class="colsep0 rowsep0" align="char" char="."><b>29</b></td><td class="colsep0 rowsep0" align="left">2,4-dimethyl</td><td class="colsep0 rowsep0" align="char" char="Â±">4.5Â Â±Â 0.4</td><td class="colsep0 rowsep0" align="char" char="Â±">17.8Â Â±Â 7.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">OTX-015</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">34.3Â Â±Â 6.5</td><td class="colsep0 rowsep0" align="char" char="Â±">6.3Â Â±Â 13.5</td><td class="colsep0 rowsep0" align="char" char="."><b>30</b></td><td class="colsep0 rowsep0" align="left">3-fluoro-4Â -methyl</td><td class="colsep0 rowsep0" align="char" char="Â±">4.2Â Â±Â 0.2</td><td class="colsep0 rowsep0" align="char" char="Â±">13.0Â Â±Â 1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>14</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char="Â±">5.4Â Â±Â 1.2</td><td class="colsep0 rowsep0" align="char" char="Â±">26.1Â Â±Â 12.3</td><td class="colsep0 rowsep0" align="char" char="."><b>31</b></td><td class="colsep0 rowsep0" align="left">4-chloro</td><td class="colsep0 rowsep0" align="char" char="Â±">6.8Â Â±Â 0.6</td><td class="colsep0 rowsep0" align="char" char="Â±">48.5Â Â±Â 10.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>28</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="Â±">7.1Â Â±Â 0.6</td><td class="colsep0 rowsep0" align="char" char="Â±">60.1Â Â±Â 18.0</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">Â </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are the mean of two independent experiments performed in duplicate with standard error of the mean (SEM) values.</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values are the mean of three independent experiments performed in triplicate with standard error of the mean (SEM) values.</p></div><div class="footnote" id="t7fn3"><sup><sup>c</sup></sup><p class="last">N.D. means not determined.</p></div></div></div><div class="NLM_p">We attempted to solve the crystal complex structures of these compounds with the BRD4-BD1 protein and successfully obtained the crystal structure of compound <b>29</b> with the BRD4-BD1 protein (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, the <i>N</i>-methyl piperazine group reached the solvent region around the protein, and the amide group oxygen formed a hydrogen bond with the residue Lys91. This binding mode was consistent with our presumption that this <i>N</i>-methyl piperazine position could be utilized for drug-like optimization and will not significantly affect the inhibitorsâ binding affinity.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0021.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystal structure of BRD4-BD1 bound with compound <b>29</b> (PDB entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI5">6JI5</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Based on the molecular and cellular activities, compounds <b>14</b>, <b>15</b>, <b>19</b>â<b>23</b>, and <b>30</b> were selected to assess their in vivo pharmacokinetics properties in mice (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a> lists the PK properties for compounds <b>14</b>, <b>19</b>, and <b>30</b>; less promising compounds <b>15</b> and <b>20</b>â<b>23</b> are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Table S4</a>). In addition to compound <b>14</b>, compounds <b>19</b> and <b>30</b> were also found to have good PK properties. When compared to compound <b>14</b>, compounds <b>19</b> and <b>30</b> had lower plasma exposure but a longer half-life; in particular, compound <b>19</b> had a half-life of 2.1 h. Moreover, compound <b>19</b> had superior cellular activity compared with compounds <b>14</b> and <b>30</b>. Considering the cellular activities and PK properties, we selected compound <b>19</b> as a preclinical candidate for subsequent comprehensive evaluation.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. In Vivo PK Data for Compounds <b>14</b>, <b>19</b>, and <b>30</b> in Male ICR mice<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd</th><th class="colsep0 rowsep0" align="center">route of administration</th><th class="colsep0 rowsep0" align="center" char=".">dose (mg/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>max</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub> (ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0-t</sub> (ngÂ·h/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0-inf</sub> (ngÂ·h/mL)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>14</b></td><td class="colsep0 rowsep0" align="left">p.o.</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">42,904</td><td class="colsep0 rowsep0" align="char" char=".">58,132</td><td class="colsep0 rowsep0" align="char" char=".">58,146</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>19</b></td><td class="colsep0 rowsep0" align="left">p.o.</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">1898</td><td class="colsep0 rowsep0" align="char" char=".">7383</td><td class="colsep0 rowsep0" align="char" char=".">7386</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>30</b></td><td class="colsep0 rowsep0" align="left">p.o.</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">4967</td><td class="colsep0 rowsep0" align="char" char=".">8154</td><td class="colsep0 rowsep0" align="char" char=".">8365</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">1% DMSO/0.5% HPMC was used as the vehicle.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Comprehensive Profile of Compound <b>19</b></h3><div class="NLM_p last">To comprehensively understand the drug-like properties of compound <b>19</b>, we performed a series of evaluations on compound <b>19</b>, including examining its selectivity and cellular activities, cellular mechanism studies, an hERG study, in vitro metabolic stability studies, in vivo pharmacokinetics studies, and an in vivo antitumor study.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Bromodomain Selectivity</h3><div class="NLM_p">Bromodomains are evolutionarily conserved proteinâprotein interaction modules that are found in a wide range of proteins with diverse catalytic and scaffolding functions. They recognize acetylated lysine by forming several hydrogen bonds with the conserved asparagine (ASN140 in BRD4-BD1) residue at the top of bromodomains. Given that selectivity is critical for the success of drug discovery, we screened the binding specificities of compound <b>19</b> across 32 bromodomains using the DiscoveRx BROMOScan platform of Eurofins Co. at 1 Î¼M concentration. As depicted in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, <b>19</b> exhibited inherent selectivity for BET family members over most of the tested bromodomains, except for a moderate inhibitory effect on TAF1(2) (details are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Table S5</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0020.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selectivity assessment of compound <b>19</b> against the BRD family members using a BROMOscan panel. The 32 screened targets are labeled in black on the BRD family phylogenetic tree. BRDs that were not part of the screening panel are in gray. Compound <b>19</b> was screened at 1 Î¼M concentration, and the results for primary screen binding interactions are reported as the âPercent Controlâ, with lower numbers indicating stronger hits in the matrix.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We also examined the inhibition of compound <b>19</b> as well as other potent compounds, including <b>7</b>, <b>10</b>, <b>14</b>â<b>15</b>, <b>20</b>â<b>23</b>, and <b>30</b> on BRD4-BD2, BRD2-BD1, BRD2-BD2, and EP300, to examine the impact of the core structure on selectivity (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Table S6</a>). The results showed that these compounds also demonstrated good potency on other BET members (BRD4-BD2, BRD2-BD1, and BRD2-BD2), although not as good as on BRD4-BD1. Importantly, none of them displayed any potency on EP300, indicating that the scaffold may be the dominant factor for the BET selectivity.</div><div class="NLM_p last">Moreover, we utilized isothermal titration calorimetry (ITC) as another binding assay method to test the binding of compound <b>19</b> to BRD4-BD1, and the results confirmed that compound <b>19</b> had excellent binding affinity (<i>K</i><sub>d</sub> = 15.2 nM) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Figure S2 and Table S7</a>).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Kinase Selectivity</h3><div class="NLM_p last">As the original core structure stems from the PLK1-BRD4 dual inhibitor BI-2536, we also tested the inhibition of compound <b>19</b> on the PLK1 kinase and 24 other kinases distributed in different subfamilies in the kinome (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Table S8</a>). We found that BI-2536 displayed a significant inhibitory effect on PLK1 kinase at 1 Î¼M; however, compound <b>19</b> only demonstrated negligible inhibitory activity for PLK1 kinase at 10 and 1 Î¼M (less than 10% inhibition ratio). This is consistent with our previous findings that changing the pyrimidine nitrogen atoms in BI-2536 to carbon atoms will eliminate the essential hydrogen bonding with the PLK1 kinase hinge part. Moreover, compound <b>19</b> did not show obvious inhibition of other tested kinases, although it displayed moderate inhibition of Src, Abl, and IGF1R at 10 Î¼M. Given the nanomolar activity of BET proteins, these results reinforced that compound <b>19</b> is a selective BET inhibitor.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Cellular Activities and Mechanism Studies</h3><div class="NLM_p">To further evaluate the therapeutic potential of compound <b>19</b>, we tested its antiproliferative activities against 17 different cell lines, including leukemia, multiple myeloma, thymic carcinoma, hepatocellular carcinoma, lung cancer, breast cancer, colorectal cancer, gastric adenocarcinoma, ovarian cancer, and so on (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Table S9</a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, compound <b>19</b> was found to exhibit better inhibitory activities on all of these cell lines than (+)-JQ-1 and show better inhibitory activities than OTX-015 in several sensitive cell lines, such as MV4-11 and Ty-82. Overall, the MV4-11, MM.1S, Ty-82, KG-1, and NCI-H1299 cell lines were sensitive to compound <b>19</b> treatment, implying that besides leukemia, it may also provide potential therapeutic opportunities for treating multiple myeloma, thymic carcinoma, and lung cancer, which is worth further exploration.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0019.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibitory activities of compound <b>19</b> against 17 cell lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">According to the above results, among the tested cell lines, MV4-11 is the most sensitive to compound <b>19</b>. Therefore, we selected it to explore the antiproliferative activity mechanism. As we know, <i>Myc</i> is an oncogene, and its deregulation is frequently involved in human oncogenesis.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> It was reported that c-Myc expression is reduced after BRD4 inhibition, which leads to the inhibition of cell proliferation.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Hence, we performed western blotting and real-time quantitative PCR (RT-qPCR) assays to study the inhibitory effects of compound <b>19</b> on c-Myc mRNA and protein expression. <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A shows that compound <b>19</b> displayed profound inhibitory effects on c-Myc protein expression, even at 10 nM, while OTX-015 did not show obvious inhibition at that concentration. Moreover, <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B showed that compound <b>19</b> demonstrated a better inhibitory effect on c-<i>Myc</i> mRNA expression at 10 and 100 nM than OTX-015, which was consistent with changes in protein levels (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). Therefore, we concluded that the inhibitory effects of compound <b>19</b> on MV4-11 cell proliferation are partially related to the downregulation of c-Myc expression.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0018.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) MV4-11 cells were treated with compound <b>19</b> or OTX-015 at the indicated doses for 24 h and examined for c-Myc and GAPDH protein levels by western blotting. (B) Similar to (A) but examining <i>c-Myc</i> mRNA levels by RT-qPCR. *<i>p</i> < 0.05. Error bars represent the SD from three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Previous studies suggest that BET inhibitors can induce apoptosis in tumor cells.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Thus, we examined the effect of compound <b>19</b> on apoptosis in MV4-11 cells. The results showed that compound <b>19</b> could induce apoptosis in a dose-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). At the same dose, compound <b>19</b> could induce more apoptosis than OTX-015 (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). The changes in cleaved PARP, an apoptotic marker, were consistent with the above apoptotic phenomena (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B). Next, we found that compound <b>19</b> and OTX-015 caused a dose-dependent increase in apoptotic initiators (cleaved caspase-8 and cleaved caspase-9), accompanied by a decrease in full-length caspase-8 (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C). Additionally, apoptotic executors (cleaved caspase-3 and cleaved caspase-7) also showed a dose-dependent increase (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0017.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) MV4-11 cells were treated with compound <b>19</b> or OTX-015 at the indicated doses for 24 h, and apoptosis was measured by the Annexin V apoptosis assay. *<i>p</i> < 0.05; **<i>p</i> < 0.01. (B) Similar to (A) but examined for cleaved PARP and GAPDH protein expression by western blotting. (C) Similar to (A) but examined for caspase-3, cleaved caspase-3, caspase-7, cleaved caspase-7, caspase-8, cleaved caspase-8, caspase-9, cleaved caspase-9, and GAPDH protein expression by western blotting. (D) Similar to (A) but examined for BCL-2, p-BCL-2(Thr56), p-BCL-2(Ser70), BCL-xL, MCL-1, XIAP, c-IAP1, c-IAP2, BAK, BAX, PUMA, BID, BIM, and GAPDH protein expression by western blotting. (E) Similar to (A) but examined for <i>BCL-2</i> mRNA expression by RT-qPCR. *<i>p</i> < 0.05; **<i>p</i> < 0.01. Error bars represent the SD of three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Overexpression of pro-survival BCL-2 family regulator proteins inhibits apoptosis, so we further investigated whether compound <b>19</b> induced apoptosis by reducing BCL-2 family levels. The results showed that compound <b>19</b> and OTX-015 reduced the levels of BCL-2, p-BCL2 (Thr56), p-BCL2 (Ser70), and BCL-xL in a dose-dependent manner, as well as the pro-apoptotic proteins PUMA, BID, and BIM (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D). Moreover, compound <b>19</b> and OTX-015 reduced the protein levels of the inhibitors of apoptosis family proteins (IAPs) XIAP, c-IAP1, and c-IAP2 in a dose-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D). RT-qPCR results showed that changes in BCL-2 mRNA expression were in line with those of BCL-2 at the protein level (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>D,</a>E). Based on the above results, we concluded that compound <b>19</b> reduced the level of many anti-apoptotic proteins and apoptotic inhibitors, thereby activating caspase family proteins and inducing more apoptosis than OTX-015 at the same doses.</div><div class="NLM_p last">BRD4 is also necessary for cell cycle progression, and inhibition of BRD4 leads to G1 arrest.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Our results showed that at the same dose, compound <b>19</b> caused more G1 phase arrest than OTX-015 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Figure S3</a>), which was consistent with the increased proliferation inhibitory activity of compound <b>19</b>.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> In Vitro Metabolic Stability Studies</h3><div class="NLM_p">Next, we examined the metabolic stability of <b>19</b> in liver microsomes from four different species, including human, mouse, rat, and dog. As depicted in <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>, compound <b>19</b> showed good stability in these liver microsome assays. We also evaluated the effects of compound <b>19</b> on five common cytochrome P450 enzymes, finding that compound <b>19</b> displayed negligible direct or time-dependent inhibition on these enzymes, which indicated that it has less risk of drugâdrug interactions (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>).</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. In Vitro Liver Microsome Intrinsic Clearance Data and Cytochrome P450 Enzyme Inhibition Data</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">liver microsome</th><th class="colsep0 rowsep0" align="center" char=".">clearance (Î¼L/min/mg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">CYP450</th><th class="colsep0 rowsep0" align="center">direct inhibition (%)<a class="ref internalNav" href="#t9fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">TDI <i>k</i><sub>obs</sub><a class="ref internalNav" href="#t9fn6" aria-label="f">f</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">HLM<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="left">3A4</td><td class="colsep0 rowsep0" align="center">23/28</td><td class="colsep0 rowsep0" align="center">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">mLM<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">193</td><td class="colsep0 rowsep0" align="left">2D6</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">RLM<a class="ref internalNav" href="#t9fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">49</td><td class="colsep0 rowsep0" align="left">2C9</td><td class="colsep0 rowsep0" align="center">28</td><td class="colsep0 rowsep0" align="center">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">DLM<a class="ref internalNav" href="#t9fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char=".">77</td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="left">1A2</td><td class="colsep0 rowsep0" align="center">no</td><td class="colsep0 rowsep0" align="center">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="left">2C19</td><td class="colsep0 rowsep0" align="center">12</td><td class="colsep0 rowsep0" align="center">no</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup><sup>a</sup></sup><p class="last">HLM represents the human liver microsome.</p></div><div class="footnote" id="t9fn2"><sup><sup>b</sup></sup><p class="last">mLM represents the mouse liver microsome.</p></div><div class="footnote" id="t9fn3"><sup><sup>c</sup></sup><p class="last">RLM represents the rat liver microsome.</p></div><div class="footnote" id="t9fn4"><sup><sup>d</sup></sup><p class="last">DLM represents the dog liver microsome.</p></div><div class="footnote" id="t9fn5"><sup><sup>e</sup></sup><p class="last">CYP direct inhibition (DI), which is sometimes referred to as reversible inhibition, is assessed by measuring enzyme (CYP) activity in the presence of increasing concentrations of inhibitor without a preincubation step. DI < 20% means no direct inhibition, 20% < DI < 50% means weak direct inhibition, 50% < DI < 70% means moderate direct inhibition, and DI > 70% means strong direct inhibition;</p></div><div class="footnote" id="t9fn6"><sup><sup>f</sup></sup><p class="last">TDI stands for time-dependent inhibition, which refers to a change in enzyme inhibition during in vitro incubation and indicates irreversible CYP inactivation. The unit of TDI is 10<sup>â4</sup>/min. TDI < 200 means no time-dependent inhibition. When the TDI value was close to zero, it was considered as none.</p></div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> hERG Study</h3><div class="NLM_p last">In cardiomyocytes, human ether-a-go-go related gene (hERG) encodes a potassium channel that mediates a delayed rectifier potassium current (IKr), and the inhibition of IKr is the most important mechanism for drug-induced prolongation of QT.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Thus, we wanted to examine the effects of compound <b>19</b> and the positive control cisapride on hERG before we advance the compound into in vivo studies. As illustrated in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Table S10</a>, compound <b>19</b> had no evident effects on hERG, even at 40 Î¼M, indicating that it has a low potential for side effects on the heart.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> In Vivo PK Study</h3><div class="NLM_p">(+)-JQ1 was reported to have a high rate of brain penetration, which could cause neurological side effects.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> To investigate the distribution of compound <b>19</b> in the brain, we evaluated its concentration in the brain and plasma. As illustrated in <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>, the ratio of the brain and plasma concentrations of compound <b>19</b> in mice or rats at 50 mg/kg dosage was 13.9 or 8.7%, respectively. As reported, (+)-JQ1 has a nearly 1:1 ratio of brain and plasma concentrations. Therefore, we thought compound <b>19</b> may have fewer potential neurological toxicity issues than (+)-JQ1.</div><div class="NLM_table-wrap" id="tbl10"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 10. In Vivo Pharmacokinetics Data of Compound <b>19</b> in Multiple Species<a class="ref internalNav" href="#t10fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="left" /></col><col align="char" char="(" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center">tissue</th><th class="colsep0 rowsep0" align="center" char="(">dosage (mg/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>max</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub> (ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0-t</sub> (ngÂ·h/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0-inf</sub> (ngÂ·h/mL)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">mouse</td><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="char" char="(">50Â (p.o.)</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">4019</td><td class="colsep0 rowsep0" align="char" char=".">16,716</td><td class="colsep0 rowsep0" align="char" char=".">16,790</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">mouse</td><td class="colsep0 rowsep0" align="left">brain</td><td class="colsep0 rowsep0" align="char" char="(">50Â (p.o.)</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">377</td><td class="colsep0 rowsep0" align="char" char=".">2321</td><td class="colsep0 rowsep0" align="char" char=".">2335</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">rat</td><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="char" char="(">50Â (p.o.)</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">1251</td><td class="colsep0 rowsep0" align="char" char=".">3820</td><td class="colsep0 rowsep0" align="char" char=".">4309</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">rat</td><td class="colsep0 rowsep0" align="left">brain</td><td class="colsep0 rowsep0" align="char" char="(">50Â (p.o.)</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">87.6</td><td class="colsep0 rowsep0" align="char" char=".">331</td><td class="colsep0 rowsep0" align="char" char=".">399</td><td class="colsep0 rowsep0" align="char" char=".">9.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">mice</td><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="char" char="(">10Â (p.o.)</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">1898</td><td class="colsep0 rowsep0" align="char" char=".">7383</td><td class="colsep0 rowsep0" align="char" char=".">7386</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">mice</td><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="char" char="(">50Â (p.o.)</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">4019</td><td class="colsep0 rowsep0" align="char" char=".">16,716</td><td class="colsep0 rowsep0" align="char" char=".">16,790</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">mice</td><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="char" char="(">100Â (p.o.)</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">13,773</td><td class="colsep0 rowsep0" align="char" char=".">80,595</td><td class="colsep0 rowsep0" align="char" char=".">82,440</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">rat</td><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="char" char="(">10Â (p.o.)</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">182</td><td class="colsep0 rowsep0" align="char" char=".">1215</td><td class="colsep0 rowsep0" align="char" char=".">1268</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">rat</td><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="char" char="(">50Â (p.o.)</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">1251</td><td class="colsep0 rowsep0" align="char" char=".">3820</td><td class="colsep0 rowsep0" align="char" char=".">4309</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">rat</td><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="char" char="(">3Â (i.v.)</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">2008</td><td class="colsep0 rowsep0" align="char" char=".">2085</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">dog</td><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="char" char="(">3Â (p.o.)</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">216</td><td class="colsep0 rowsep0" align="char" char=".">747</td><td class="colsep0 rowsep0" align="char" char=".">798</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">dog</td><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="char" char="(">1Â (i.v.)</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">726</td><td class="colsep0 rowsep0" align="char" char=".">856</td><td class="colsep0 rowsep0" align="char" char=".">9.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t10fn1"><div class="footnote" id="t10fn1"><sup><sup>a</sup></sup><p class="last">1% DMSO/0.5% HPMC was used as the vehicle.</p></div></div></div><div class="NLM_p last">Furthermore, we determined compound <b>19</b>âs dose-dependent PK properties in different species. <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a> demonstrates that the plasma exposure of compound <b>19</b> continuously improved with increasing doses (oral administration of 10, 50, or 100 mg/kg in mice or oral administration of 10 or 50 mg/kg in rats). To assess the PK properties in large animals, we further examined its PK properties in dogs (<a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>). In these three species, <b>19</b> displayed a reasonable plasma exposure and half-life. Additionally, intravenous injection of compound <b>19</b> at a 3 mg/kg dose in rats or 1 mg/kg dose in dogs showed that it had moderate oral bioavailability at 18.2% in rats and 34.3% in dogs.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> In Vivo PD Study</h3><div class="NLM_p">To evaluate the in vivo therapeutic effects of compound <b>19</b>, we established a human MV4-11 xenograft model. Tumor-bearing BALB/c nude mice were treated with different doses of compound <b>19</b> using OTX-015 as the positive control (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A). Treatment with compound <b>19</b> and OTX-015 by oral administration at the same dosage of 50 mg/kg daily showed that compound <b>19</b> had superior inhibitory effects on the tumor growth with a growth inhibition (GI) of 95.6% (OTX-015 had a GI of 71.5%). Compound <b>19</b> reduced tumor growth in a dose-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A). Furthermore, as depicted in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A, compound <b>19</b> even showed slightly better antitumor effects at one fourth the dosage compared with OTX-015 (25 mg/kg daily of <b>19</b> with a GI of 89.7% vs 100 mg/kg of OTX-015 daily with a GI of 87.5%). Moreover, compound <b>19</b> did not promote any obvious loss in body weight at multiple dosages, and no animal death occurred during the experiment (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B), indicating that it has a good safety window.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0016.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) Relative tumor volume (RTV) of human MV4-11 xenografts in Balb/c nude mice after treatments. OTX-015 or compound <b>19</b> group versus the vehicle group; **<i>p</i> < 0.01; ***<i>p</i> < 0.001. Formulation: <b>19</b>, 0.5% Tween80, and 0.5% methylcellulose aqueous solution and OTX-015, 5% dimethylacetamide, and 0.5% hydroxypropyl methyl cellulose. (B) Body weight changes in mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We also examined the pharmakineticsâpharmacodynamics relationship by assessing the drug concentration in plasma and tumor tissue after administration of compound <b>19</b> at the above dosage for 1 and 8 h. The data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Table S11</a>) showed that plasma exposure was increased in a dose-dependent manner at 1 and 8 h but decreased with time at the three different concentrations, which is consistent with the previous PK study. However, the tumor concentration of compound <b>19</b> after oral administration remains higher than plasma, especially at the 8 h time point, indicating that <b>19</b> has a longer half-life in tumor tissue. This effect is more evident in the high-dose groups (25 and 50 mg/kg), which correlated well with the excellent antitumor efficacy in vivo at these two different dosages.</div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Chemistry</h3><div class="NLM_p">The syntheses of novel dihydroquinoxalin-2(1<i>H</i>)-one derivatives were concluded in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>â<named-content content-type="anchor" rid="sch2" type="simple"></named-content><named-content content-type="anchor" rid="sch3" type="simple"></named-content><named-content content-type="anchor" rid="sch4" type="simple"></named-content><named-content content-type="anchor" rid="sch5" type="simple"></named-content><named-content content-type="anchor" rid="sch6" type="simple"></named-content><a class="ref internalNav" href="#sch7" aria-label="7">7</a>.</div><figure id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0015.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route of Compound <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) cyclopropylamine, ClCH<sub>2</sub>CH<sub>2</sub>Cl, 80 Â°C, 12 h; (b) Fe, NH<sub>4</sub>Cl, EtOH, 80 Â°C, 1 h; (c) 1. 2-bromopropanoyl bromide, DIPEA, DCM, 0 Â°C to rt, 2 h; 2. CH<sub>3</sub>CN, DIPEA, 80 Â°C, overnight; (d) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h; and (e) <i>N</i>-Boc-2-pyrroleboronic acid, Pd(dppf)Cl<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, DMF, 120 Â°C, 4 h.</p></p></figure><figure id="sch2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0014.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Routes of Compounds <b>6</b>, <b>8</b>, and <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>p</i>-toluidine, HATU, DIPEA, DMF, rt, overnight; (b) R<sup>2</sup>NH<sub>2</sub>, ClCH<sub>2</sub>CH<sub>2</sub>Cl, 80 Â°C, 12 h; (c) Lawessonâs reagent, toluene, 110 Â°C, overnight; (d) <b>43</b>: 1. N<sub>2</sub>H<sub>4</sub>Â·H<sub>2</sub>O, MeOH, rt, 2 h; 2. trimethylorthoformate, DMF, AcOH, rt, overnight; <b>44</b> and <b>45</b>: Hg(OAc)<sub>2</sub>, C<sub>3</sub>H<sub>9</sub>N<sub>3</sub>Si, THF, 0 Â°C, 3 h; (e) tin(II) chloride dihydrate, conc. HCl, rt, 5 h; (f) 1. 2-bromopropanoyl bromide, DIPEA, DCM, 0 Â°C to rt, 2 h; 2. CH<sub>3</sub>CN, DIPEA, 80 Â°C, overnight; and (g) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h.</p></p></figure><figure id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route of Compound <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) SOCl<sub>2</sub>, MeOH, 60 Â°C, 24 h; (b) cyclopropylamine, ClCH<sub>2</sub>CH<sub>2</sub>Cl, 80 Â°C, 12 h; (c) Fe, NH<sub>4</sub>Cl, EtOH, 80 Â°C, 1 h; (d) 1. 2-bromopropanoyl bromide, DIPEA, DCM, 0 Â°C to rt, 2 h; 2. CH<sub>3</sub>CN, DIPEA, 80 Â°C, overnight; (e) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h; (f) N<sub>2</sub>H<sub>4</sub>Â·H<sub>2</sub>O, EtOH, 80 Â°C, 12 h; and (g) <i>N</i>-(<i>p</i>-tolyl)ethanethioamide, Hg(OAc)<sub>2</sub>, AcOH, THF, 0 Â°C to rt, 24 h.</p></p></figure><figure id="sch4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0012.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthetic Route of Compound <b>9</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>p</i>-toluidine, THF, 0 Â°C to rt, 12 h; (b) cyclopropylamine, ClCH<sub>2</sub>CH<sub>2</sub>Cl, 80 Â°C, 12 h; (c) tin(II) chloride dihydrate, conc. HCl, rt, 5 h; (d) 1. 2-bromopropanoyl bromide, DIPEA, DCM, 0 Â°C to rt, 2 h; 2. CH<sub>3</sub>CN, DIPEA, 80 Â°C, overnight; and (e) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h.</p></p></figure><figure id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0011.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthetic Route of Compound <b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <span class="smallcaps smallerCapital">d</span>-alanine, K<sub>2</sub>CO<sub>3</sub>, EtOH, H<sub>2</sub>O, 80 Â°C, 8 h; (b) K<sub>2</sub>CO<sub>3</sub>, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, H<sub>2</sub>O, 60 Â°C, overnight; (c) phenylsilane, cyclopentanone, dibutyltindichloride, THF, rt, 10 h; (d) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h; (e) N<sub>2</sub>H<sub>4</sub>Â·H<sub>2</sub>O, EtOH, 80 Â°C, 12 h; and (f) <i>N</i>-(<i>p</i>-tolyl)ethanethioamide, Hg(OAc)<sub>2</sub>, AcOH, THF, 0 Â°C to rt, 24 h.</p></p></figure><figure id="sch6" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0010.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthetic Routes of Compounds <b>12</b> and <b>14</b>â<b>31</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <span class="smallcaps smallerCapital">d</span>-alanine, K<sub>2</sub>CO<sub>3</sub>, EtOH, H<sub>2</sub>O, 80 Â°C, 8 h; (b) K<sub>2</sub>CO<sub>3</sub>, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, H<sub>2</sub>O, 60 Â°C, overnight; (c) phenylsilane, cyclopentanone, dibutyltin dichloride, THF, rt, 10 h; (d) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h; (e) Pd(dppf)Cl<sub>2</sub>, bis(pinacolato)diboron, AcOK, DMSO, 80 Â°C, overnight; (f) Pd(dppf)Cl<sub>2</sub>, NaHCO<sub>3</sub>, THF/H<sub>2</sub>O, 80 Â°C, overnight; (g) Pd(dppf)Cl<sub>2</sub>, NaHCO<sub>3</sub>, THF/H<sub>2</sub>O, 80 Â°C, overnight; (h) LiOH, THF/H<sub>2</sub>O, rt, overnight; and (i) HATU, DIPEA, DMF, rt, overnight.</p></p></figure><figure id="sch7" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0009.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthetic Route of Compound <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(dppf)Cl<sub>2</sub>, bis(pinacolato)diboron, AcOK, DMSO, 80 Â°C, overnight; (b) Pd(dppf)Cl<sub>2</sub>, NaHCO<sub>3</sub>, THF/H<sub>2</sub>O, 80 Â°C, overnight; (c) LiOH, THF/H<sub>2</sub>O, rt, overnight; and (d) HATU, DIPEA, DMF, rt, overnight.</p></p></figure><div class="NLM_p">As depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, the synthetic route of compound <b>36</b> was described in our previous work,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> which was coupled with <i>N</i>-Boc-2-pyrroleboronic acid to generate compound <b>4</b>.</div><div class="NLM_p">In <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, condensation of commercially available 3-fluoro-4-nitrobenzoic acid (compound <b>37</b>) with <i>p</i>-toluidine furnished compound <b>38</b>, which was then reacted with cyclopropylamine or cyclopentylamine to afford corresponding compound <b>39</b> or <b>40</b>. The reaction of compound <b>39</b> or <b>40</b> with Lawessonâs reagent acquired compound <b>41</b> or <b>42</b>. Treatment of <b>41</b> with N<sub>2</sub>H<sub>4</sub>Â·H<sub>2</sub>O followed by adding trimethyl orthoformate gained compound <b>43</b>. Cyclization of compound <b>41</b> or <b>42</b> with C<sub>3</sub>H<sub>9</sub>N<sub>3</sub>Si obtained compound <b>44</b> or <b>45</b>. Reduction of compounds <b>43</b>â<b>45</b> with tin(II) chloride dihydrate and conc. HCl yielded compounds <b>46</b>â<b>48</b>, which was converted to compounds <b>6</b>, <b>8</b>, and <b>11</b> through a similar reaction of <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>c,d.</div><div class="NLM_p">In <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, the reaction of compound <b>37</b> with SOCl<sub>2</sub> and methanol afforded compound <b>52</b>, which was then converted to compound <b>56</b> through a similar reaction of <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>aâd. The reaction of compound <b>56</b> with hydrazine hydrate acquired compound <b>57</b>, which was further transformed into compound <b>7</b> by reacting with <i>N</i>-(<i>p</i>-tolyl)ethanethioamide.</div><div class="NLM_p">In <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, the reaction of compound <b>58</b> with <i>p</i>-toluidine and thioglycolic acid received compound <b>59</b>, which was then transformed into compound <b>9</b> through a similar reaction of <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>aâd except using tin(II) chloride dehydrate as a reduction reagent instead of Fe powder.</div><div class="NLM_p">In <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>, the reaction of compound <b>52</b> with <span class="smallcaps smallerCapital">d</span>-alanine in the presence of K<sub>2</sub>CO<sub>3</sub> obtained compound <b>63</b>, which was then converted to compound <b>64</b> under the condition of K<sub>2</sub>CO<sub>3</sub> and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>. Treatment of compound <b>64</b> with cyclopentanone via reductive amination provided compound <b>65</b>, which was switched to compound <b>10</b> through a similar reaction of <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>eâg.</div><div class="NLM_p">In <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>, compound <b>71</b> was prepared in the same manner as compound <b>66</b>. Treatment of compound <b>71</b> with bis(pinacolato)diboron obtained compound <b>72</b>. Suzuki coupling of compound <b>72</b> with 4-bromo-5-methyl-3-phenylisoxazole (the synthetic route was described by Dong et. al<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a>) resulted in compound <b>12</b>. Similarly, Suzuki coupling of compound <b>72</b> with ethyl 5-bromo-1-phenyl-1<i>H</i>-1,2,4-triazole-3-carboxylate derivatives (the synthetic route was described by Zibinsky et. al<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a>) and subsequent hydrolysis yielded corresponding compounds <b>78</b>â<b>82</b>, which was further converted to the desired derivatives <b>14</b>â<b>31</b> by condensation with different amines.</div><div class="NLM_p last">In <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>, compound <b>13</b> was obtained through a similar reaction of <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>e,gâi from compound <b>36</b>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33536" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33536" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Based on our previously reported novel BET-selective inhibitors, we wanted to discover a candidate for further clinical research. With the help of the cocrystal structure of hit compound <b>4</b> harboring a new rigid five-membered ring, we found a lead compound, <b>7</b>, that showed moderate antitumor activity in a human MM.1S xenograft model in SCID mice by oral administration. Inspired by this result, we resolved the cocrystal structure of compound <b>7</b> with BRD4-BD1 for further optimization. Introduction of a hydrophilic group to the five-membered ring linker yielded a novel series of (1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one compounds with excellent molecular and cellular activities. By evaluating the molecular and cellular activities, in vitro metabolic stability, and in vivo PK properties, compound <b>19</b> was discovered as a potential candidate for preclinical research. After a comprehensive evaluation, we confirmed that compound <b>19</b> displayed the following properties: high selectivity for BET over other bromodomains and tested kinases distributed in different subfamilies; excellent inhibitory activities on the MV4-11, MM.1S, Ty-82, KG-1, and NCI-H1299 cell lines; increased G1 cell cycle arrest and apoptosis compared to OTX-015; stable metabolic stability in multiple species; a low rate of bloodâbrain barrier penetration; good PK properties in mice, rats, and dogs; and good antitumor efficacy in vivo. Overall, we demonstrated that compound <b>19</b> is a potential drug candidate for BET-relevant cancer treatment.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06556" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06556" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Chemistry General</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz) spectra were recorded by using a Varian Mercury-400 high-performance digital FT-NMR spectrometer with tetramethylsilane (TMS) as an internal standard. <sup>13</sup>C NMR (126 MHz) spectra were recorded by using a Varian Mercury-500 high-performance digital FT-NMR spectrometer. Abbreviations for peak patterns in NMR spectra: br = broadened, s = singlet, d = doublet, t = triplet, dd = doublet of doublets, and m = multiplet. Low-resolution mass spectra were obtained with a Finnigan LCQ Deca XP mass spectrometer using a CAPCELL PAK C18 (50 mm Ã 2.0 mm, 5 Î¼M) or an Agilent ZORBAX Eclipse XDB C18 (50 mm Ã 2.1 m, 5 Î¼M) in positive or negative electrospray mode. High-resolution mass spectra were recorded by using a Finnigan MAT-95 mass spectrometer. Purity of all compounds was determined using an analytical Gilson high-performance liquid chromatograph (HPLC) with a YMC ODS3 column (50 mm Ã 4.6 mm, 5 Î¼M) and confirmed to be more than 95%. Conditions were as follows: CH<sub>3</sub>CN/H<sub>2</sub>O eluent at 2.5 mLmin<sup>â1</sup> flow [containing 0.1% trifluoroacetic acid (TFA)] at 35 Â°C; 8 min; gradient, 5% CH<sub>3</sub>CN to 95% CH<sub>3</sub>CN; monitored by UV absorption at 214 and 254 nm. TLC analysis was carried out with glass precoated silica gel GF254 plates. TLC spots were visualized under UV light. Flash column chromatography was performed with a Teledyne ISCO CombiFlash <i>R</i><sub>f</sub> system. All solvents and reagents were used directly as obtained commercially unless otherwise noted. All air- and moisture-sensitive reactions were carried out under an atmosphere of dry argon with heat-dried glassware and standard syringe techniques. Melting points were determined using a SGW X-4 hot stage microscope and are uncorrected (spectra data of the synthesized compounds are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">Supporting Information</a>).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Synthetic Procedures</h3><div class="NLM_p">Compounds <b>32</b>, <b>37</b>, and <b>58</b> were purchased. Other compounds were prepared by one of seven schemes.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 4-Cyclopropyl-1,3-dimethyl-6-(1<i>H</i>-pyrrol-2-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>4</b>)</h4><div class="NLM_p last">To a solution of compound <b>36</b> (0.2 g, 0.7 mmol) in anhydrous DMF (10 mL) was added K<sub>2</sub>CO<sub>3</sub> (0.5 g, 3.4 mmol). The mixture was purged with nitrogen for 2 min. Then, Pd(dppf)Cl<sub>2</sub>Â·CH<sub>2</sub>Cl<sub>2</sub> (0.1 g, 0.1 mmol) was added, and the mixture was sealed in a microwave tube and heated to 120 Â°C for 4 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was acidified with 1 N aq. HCl to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 50 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 25% EtOAc/60â90 Â°C petroleum ether) gave compound <b>4</b> as a white solid (0.09 g, 0.3 mmol, 43% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.63 (s, 1H), 7.25 (d, <i>J</i> = 1.9 Hz, 1H), 7.00 (dd, <i>J</i> = 8.2, 2.0 Hz, 1H), 6.91 (d, <i>J</i> = 8.3 Hz, 1H), 6.86 (td, <i>J</i> = 2.6, 1.5 Hz, 1H), 6.51â6.47 (m, 1H), 6.31 (dd, <i>J</i> = 6.0, 2.7 Hz, 1H), 4.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.35 (s, 3H), 2.49â2.39 (m, 1H), 1.19 (d, <i>J</i> = 6.9 Hz, 3H), 1.03â0.96 (m, 1H), 0.84â0.76 (m, 1H), 0.68â0.61 (m, 1H), 0.60â0.54 (m, 1H); MS (ESI) [M + H]<sup>+</sup>: 282.17.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-Cyclopropyl-1,3-dimethyl-6-(4-(<i>p</i>-tolyl)-4<i>H</i>-1,2,4-triazol-3-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>6</b>)</h4><div class="NLM_p last">To a solution of compound <b>49</b> (0.8 g, 2.2 mmol) in anhydrous DMF (2 mL) was added NaH (0.2 g, 6.7 mmol) at 0 Â°C, the mixture was stirred at 0 Â°C for 30 min, then iodomethane (0.2 mL, 3.4 mmol) was added, and the mixture was stirred at room temperature for another 2 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was acidified with 1 N aq. HCl to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 50 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 10% EtOAc/60â90 Â°C petroleum ether) gave compound <b>6</b> as a white solid (0.2 g, 0.6 mmol, 27% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.28 (s, 1H), 7.29â7.24 (m, 2H), 7.17â7.14 (m, 2H), 7.14â7.12 (m, 1H), 7.04 (dd, <i>J</i> = 8.3, 1.9 Hz, 1H), 6.85 (d, <i>J</i> = 8.3 Hz, 1H), 4.05 (q, <i>J</i> = 6.9 Hz, 1H), 3.31 (s, 3H), 2.42 (s, 3H), 2.22â2.14 (m, 1H), 1.10 (d, <i>J</i> = 6.8 Hz, 3H), 0.72â0.64 (m, 1H), 0.58â0.47 (m, 2H), 0.21â0.13 (m, 1H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 168.80, 153.33, 144.87, 139.76, 135.83, 132.42, 130.86, 130.57 (2 Ã C), 125.73 (2 Ã C), 121.66, 120.07, 114.31, 114.30, 58.15, 29.08, 27.84, 21.27, 12.27, 9.10, 6.65; MS (ESI) [M + H]<sup>+</sup>: 374.31; retention time 2.93 min, >96% pure.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 4-Cyclopropyl-1,3-dimethyl-6-(5-methyl-4-(<i>p</i>-tolyl)-4<i>H</i>-1,2,4-triazol-3-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>7</b>)</h4><div class="NLM_p last">To a solution of compound <b>57</b> (0.6 g, 2.2 mmol) in THF (10 mL) and acetic acid (4 mL) were added <i>N</i>-(<i>p</i>-tolyl)ethanethioamide (0.4 g, 2.2 mmol) and Hg(OAc)<sub>2</sub> (1.1 g, 3.3 mmol) at 0 Â°C, and then the mixture was stirred at 0 Â°C for 3 h and further stirred at rt for 24 h. The reaction was monitored by TLC. Upon completion, the residue was basified with saturated NaHCO<sub>3</sub> to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 40 mL). Combined organic layers were washed with brine, and purification by silica gel column chromatography (gradient elution, gradient 0 to 10% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) gave compound <b>7</b> as a red solid (0.3 g, 0.8 mmol, 35% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30 (d, <i>J</i> = 8.0 Hz, 2H), 7.15 (d, <i>J</i> = 1.9 Hz, 1H), 7.09 (d, <i>J</i> = 8.3 Hz, 2H), 6.98 (dd, <i>J</i> = 8.3, 1.9 Hz, 1H), 6.81 (d, <i>J</i> = 8.4 Hz, 1H), 4.04 (q, <i>J</i> = 6.8 Hz, 1H), 3.29 (s, 3H), 2.43 (s, 3H), 2.34 (s, 3H), 2.20â2.15 (m, 1H), 1.09 (d, <i>J</i> = 6.8 Hz, 3H), 0.73â0.65 (m, 1H), 0.64â0.56 (m, 1H), 0.55â0.48 (m, 1H), 0.23â0.14 (m, 1H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 168.87, 154.08, 152.80, 139.96, 135.79, 132.76, 130.78 (2 Ã C), 130.62, 127.14 (2 Ã C), 122.47, 119.81, 114.24, 114.04, 58.21, 29.08, 27.87, 21.34, 12.23, 11.55, 9.23, 6.68; HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for (C<sub>23</sub>H<sub>26</sub>N<sub>5</sub>O<sup>+</sup>) 388.2132, found 388.2123; retention time 2.90 min, >99% pure.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-Cyclopropyl-1,3-dimethyl-6-(1-(<i>p</i>-tolyl)-1<i>H</i>-tetrazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>8</b>)</h4><div class="NLM_p last">To a solution of compound <b>50</b> (0.6 g, 1.7 mmol) in anhydrous DMF (2 mL) was added NaH (0.1 g, 5.0 mmol) at 0 Â°C, the mixture was stirred at 0 Â°C for 30 min, then iodomethane (0.2 mL, 3.0 mmol) was added, and the mixture was stirred at room temperature for another 2 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was acidified with 1 N aq. HCl to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 50 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 10% EtOAc/60â90 Â°C petroleum ether) gave compound <b>8</b> (0.2 g, 0.5 mmol, 29% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.36â7.28 (m, 4H), 7.23 (d, <i>J</i> = 1.7 Hz, 1H), 7.15 (dd, <i>J</i> = 8.3, 1.8 Hz, 1H), 6.91 (d, <i>J</i> = 8.3 Hz, 1H), 4.08 (q, <i>J</i> = 6.9 Hz, 1H), 3.33 (s, 3H), 2.45 (s, 3H), 2.25â2.15 (m, 1H), 1.13 (d, <i>J</i> = 6.8 Hz, 3H), 0.76â0.65 (m, 1H), 0.58â0.48 (m, 2H), 0.26â0.15 (m, 1H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´ 168.71, 153.72, 140.85, 136.16, 132.47, 131.94, 130.52 (2 Ã C), 125.46 (2 Ã C), 120.45, 118.66, 114.53, 114.29, 58.08, 29.13, 27.88, 21.42, 12.49, 9.12, 6.74; MS (ESI) [M + H]<sup>+</sup>: 375.20; retention time 3.63 min, >95% pure.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-(4-Cyclopropyl-1,3-dimethyl-2-oxo-1,2,3,4-tetrahydroquinoxalin-6-yl)-3-(<i>p</i>-tolyl)thiazolidin-4-one (<b>9</b>)</h4><div class="NLM_p last">To a solution of compound <b>62</b> (0.1 g, 0.3 mmol) in anhydrous DMF (5 mL) was added NaH (0.02 g, 0.8 mmol) at 0 Â°C, the mixture was stirred at 0 Â°C for 30 min, and then iodomethane (0.03 mL, 0.5 mmol) was added and stirred at room temperature for another 2 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was acidified with 1 N aq. HCl to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 20 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 25% EtOAc/60â90 Â°C petroleum ether) gave compound <b>9</b> (15 mg, 0.04 mmol, 15% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.11â7.04 (m, 3H), 7.02 (s, 1H), 7.01â6.97 (m, 1H), 6.86â6.79 (m, 1H), 6.78â6.75 (m, 1H), 6.04 (d, <i>J</i> = 34.1 Hz, 1H), 4.06 (q, <i>J</i> = 6.9 Hz, 1H), 3.96 (dd, <i>J</i> = 15.8, 9.5 Hz, 1H), 3.87 (d, <i>J</i> = 15.8 Hz, 1H), 3.27 (d, <i>J</i> = 4.1 Hz, 3H), 2.38â2.30 (m, 1H), 2.25 (d, <i>J</i> = 3.0 Hz, 3H), 1.11 (dd, <i>J</i> = 8.7, 7.1 Hz, 3H), 0.94â0.85 (m, 1H), 0.82â0.71 (m, 1H), 0.63â0.54 (m, 1H), 0.41â0.29 (m, 1H); MS (ESI) [M + H]<sup>+</sup>: 408.21; retention time 3.49 min, >99% pure.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (<i>R</i>)-4-Cyclopentyl-1,3-dimethyl-6-(5-methyl-4-(<i>p</i>-tolyl)-4<i>H</i>-1,2,4-triazol-3-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>10</b>)</h4><div class="NLM_p">To a solution of compound <b>67</b> (0.6 g, 2.0 mmol) in THF (10 mL) and acetic acid (4 mL) were added <i>N</i>-(<i>p</i>-tolyl)ethanethioamide (0.3 g, 2.0 mmol) and Hg(OAc)<sub>2</sub> (1.0 g, 3.0 mmol) at 0 Â°C, then the mixture was stirred at 0 Â°C for 3 h and further stirred at rt for 24 h. The reaction was monitored by TLC. Upon completion, the residue was basified with saturated NaHCO<sub>3</sub> to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 40 mL). Combined organic layers were washed with brine, and purification by silica gel column chromatography (gradient elution, gradient 0 to 10% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) gave compound <b>10</b> as a white solid (0.3 g, 0.8 mmol, 40% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30 (d, <i>J</i> = 8.1 Hz, 2H), 7.10 (d, <i>J</i> = 8.1 Hz, 2H), 7.00 (dd, <i>J</i> = 8.4, 1.5 Hz, 1H), 6.83 (d, <i>J</i> = 8.1 Hz, 2H), 4.10 (q, <i>J</i> = 6.8 Hz, 1H), 3.43 (dt, <i>J</i> = 14.6, 7.4 Hz, 1H), 3.30 (s, 3H), 2.42 (s, 3H), 2.33 (s, 3H), 1.87â1.76 (m, 1H), 1.75â1.46 (m, 7H), 0.96 (d, <i>J</i> = 6.8 Hz, 3H); HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for (C<sub>25</sub>H<sub>30</sub>N<sub>5</sub>O<sup>+</sup>) 416.2445, found 416.244; retention time 3.11 min, >99% pure.</div><div class="NLM_p last">Following similar procedures to that for compound <b>8</b> gave compound <b>11</b>.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 4-Cyclopentyl-1,3-dimethyl-6-(1-(<i>p</i>-tolyl)-1<i>H</i>-tetrazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>11</b>)</h4><div class="NLM_p last">White solid, 35% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30 (q, <i>J</i> = 8.4 Hz, 4H), 7.13 (d, <i>J</i> = 8.3 Hz, 1H), 6.93 (d, <i>J</i> = 8.1 Hz, 2H), 4.15 (q, <i>J</i> = 6.7 Hz, 1H), 3.48 (dd, <i>J</i> = 14.9, 7.4 Hz, 1H), 3.34 (s, 3H), 2.43 (s, 3H), 1.83 (dd, <i>J</i> = 14.2, 8.8 Hz, 2H), 1.74â1.66 (m, 1H), 1.64â1.48 (m, 5H), 0.99 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´ 168.95, 153.70, 140.85, 135.87, 132.89, 132.44, 130.53 (2 Ã C), 125.52 (2 Ã C), 119.91, 118.43, 115.60, 114.74, 58.97, 54.47, 30.76, 30.36, 29.22, 23.99, 23.50, 21.38, 14.22; MS (ESI) [M + H]<sup>+</sup>: 403.13; retention time 3.92 min, >99% pure.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>R</i>)-4-Cyclopentyl-1,3-dimethyl-6-(5-methyl-3-phenylisoxazol-4-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>12</b>)</h4><div class="NLM_p last">To a solution of compound <b>72</b> (1 g, 2.7 mmol) and NaHCO<sub>3</sub> (0.5 g, 5.4 mmol) in THF (10 mL) and H<sub>2</sub>O (3 mL) was added 4-bromo-5-methyl-3-phenylisoxazole (1.3 g, 5.4 mmol), the mixture was bubbled with N<sub>2</sub> for 5 min, then Pd(dppf)<sub>2</sub>Cl<sub>2</sub>Â·CH<sub>2</sub>Cl<sub>2</sub> (0.2 g, 0.3 mmol) was added, and the mixture was further bubbled with N<sub>2</sub> for 5 min. Then, the mixture was heated to 80 Â°C overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with EtOAc (3 Ã 50 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 20% EtOAc/60â90 Â°C petroleum ether) gave compound <b>12</b> (0.7 g, 1.7 mmol, 56% yield) as a light yellow liquid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.58â7.55 (m, 2H), 7.36â7.30 (m, 3H), 7.00 (d, <i>J</i> = 8.2 Hz, 1H), 6.82 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 6.66 (d, <i>J</i> = 1.8 Hz, 1H), 4.20 (q, <i>J</i> = 6.8 Hz, 1H), 3.66â3.56 (m, 1H), 3.41 (s, 3H), 2.29 (s, 3H), 1.90â1.82 (m, 1H), 1.81â1.67 (m, 2H), 1.62â1.46 (m, 5H), 1.08 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 169.21, 164.42, 160.19, 136.06, 130.61, 129.71, 128.70 (2 Ã C), 128.10, 127.08 (2 Ã C), 125.59, 120.31, 117.29, 116.33, 114.97, 58.99, 55.01, 30.87, 30.79, 29.19, 24.21, 23.74, 13.91, 10.87; MS (ESI) [M + H]<sup>+</sup>: 402.19; retention time 4.23 min, >99% pure.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 4-Cyclopropyl-1,3-dimethyl-6-(3-(4-methylpiperazine-1-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>13</b>)</h4><div class="NLM_p last">To a solution of compound 85 (0.7 g, 1.7 mmol) in DMF (10 mL) was added HATU (0.7 g, 1.9 mmol), and the mixture was stirred at rt for 30 min. Then, <i>N</i>-methylpiperazine (0.2 g, 1.9 mmol) and DIPEA (0.3 mL, 1.9 mmol) were added, and the mixture was stirred at rt overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with saturated NaHCO<sub>3</sub> and extracted with EtOAc (3 Ã 50 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 20% EtOAc/60â90 Â°C petroleum ether) gave compound <b>13</b> (0.5 g, 1.0 mmol, 59% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30â7.27 (m, 2H), 7.25â7.22 (m, 2H), 7.17 (dd, J = 8.2, 1.9 Hz, 1H), 7.14 (d, J = 1.8 Hz, 1H), 6.89 (d, J = 8.2 Hz, 1H), 4.06 (q, J = 6.8 Hz, 1H), 4.01â3.96 (m, 2H), 3.90â3.85 (m, 2H), 3.33 (s, 3H), 2.52 (dt, J = 14.2, 5.0 Hz, 4H), 2.41 (s, 3H), 2.34 (s, 3H), 2.20â2.12 (m, 1H), 1.11 (d, J = 6.8 Hz, 3H), 0.73â0.63 (m, 1H), 0.57â0.42 (m, 2H), 0.21â0.11 (m, 1H); MS (ESI) [M + H]<sup>+</sup>: 500.41; retention time 3.04 min, >99% pure.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>R</i>)-4-Cyclopentyl-1,3-dimethyl-6-(3-(4-methylpiperazine-1-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>14</b>)</h4><div class="NLM_p">To a solution of compound <b>78</b> (0.7 g, 1.6 mmol) in DMF (10 mL) was added HATU (0.6 g, 1.7 mmol), and the mixture was stirred at rt for 30 min. Then, <i>N</i>-methylpiperazine (0.2 g, 1.7 mmol) and DIPEA (0.3 mL, 1.7 mmol) were added, and the mixture was stirred at rt overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with saturated NaHCO<sub>3</sub> and extracted with EtOAc (3 Ã 50 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 20% EtOAc/60â90 Â°C petroleum ether) gave compound <b>14</b> (0.5 g, 1.0 mmol, 63% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.29â7.26 (m, 2H), 7.25â7.22 (m, 2H), 7.14 (dd, <i>J</i> = 8.3, 1.9 Hz, 1H), 6.90 (d, <i>J</i> = 8.4 Hz, 1H), 6.84 (d, <i>J</i> = 1.9 Hz, 1H), 4.12 (q, <i>J</i> = 6.8 Hz, 1H), 4.01â3.92 (m, 2H), 3.90â3.83 (m, 2H), 3.48â3.40 (m, 1H), 3.34 (s, 3H), 2.54â2.49 (m, 2H), 2.49â2.45 (m, 2H), 2.39 (s, 3H), 2.32 (s, 3H), 1.84â1.74 (m, 1H), 1.72â1.63 (m, 1H), 1.61â1.43 (m, 5H), 1.35â1.27 (m, 1H), 0.97 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 169.08, 160.70, 156.86, 154.45, 139.61, 135.72, 135.52, 132.26, 130.15 (2 Ã C), 125.67 (2 Ã C), 122.18, 120.21, 116.16, 114.56, 59.05, 55.48, 54.71, 54.56, 47.10, 46.10, 42.43, 30.78, 30.35, 29.24, 24.01, 23.55, 21.30, 14.14; HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for (C<sub>30</sub>H<sub>38</sub>N<sub>7</sub>O<sub>2</sub><sup>+</sup>) 528.3081, found 528.3081; retention time 2.88 min, >99% pure.</div><div class="NLM_p last">Following similar procedures to that for compound <b>14</b> gave compounds <b>15</b>â<b>31</b>.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>R</i>)-5-(4-Cyclopentyl-1,3-dimethyl-2-oxo-1,2,3,4-tetrahydroquinoxalin-6-yl)-<i>N</i>-(2-(dimethylamino)ethyl)-<i>N</i>-methyl-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazole-3-carboxamide (<b>15</b>)</h4><div class="NLM_p last">White solid, 64% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.26â7.19 (m, 4H), 7.13 (dt, <i>J</i> = 8.3, 2.1 Hz, 1H), 6.90 (dd, <i>J</i> = 8.4, 4.1 Hz, 1H), 6.77 (dd, <i>J</i> = 19.2, 1.6 Hz, 1H), 4.10 (q, <i>J</i> = 6.7 Hz, 1H), 3.85â3.79 (m, 1H), 3.78â3.69 (m, 2H), 3.45â3.35 (m, 1H), 3.34 (s, 2H), 3.32 (s, 3H), 2.84 (t, <i>J</i> = 6.5 Hz, 1H), 2.74â2.66 (m, 1H), 2.49 (s, 3H), 2.37 (s, 3H), 2.24 (s, 3H), 1.81â1.70 (m, 1H), 1.69â1.59 (m, 1H), 1.60â1.40 (m, 5H), 1.31â1.24 (m, 1H), 0.95 (dd, <i>J</i> = 6.8, 2.7 Hz, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 168.97 (d, <i>J</i> = 3.3 Hz), 162.25 (d, <i>J</i> = 9.6 Hz), 156.83 (d, <i>J</i> = 12.6 Hz), 154.40 (d, <i>J</i> = 3.1 Hz), 139.64 (d, <i>J</i> = 14.7 Hz), 135.56 (d, <i>J</i> = 15.1 Hz), 135.46, 132.25 (d, <i>J</i> = 10.4 Hz), 130.15, 130.11, 125.57 (2 Ã C), 121.97 (d, <i>J</i> = 19.9 Hz), 120.14, 116.02 (d, <i>J</i> = 9.6 Hz), 114.60 (d, <i>J</i> = 10.3 Hz), 58.93, 57.63, 56.15, 54.45, 45.63, 45.16, 37.62, 30.71, 30.25, 29.16, 23.94, 23.48, 21.23, 14.07; HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for (C<sub>30</sub>H<sub>40</sub>N<sub>7</sub>O<sub>2</sub><sup>+</sup>) 530.3238, found 530.3244; retention time 3.01 min, >99% pure.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (<i>R</i>)-4-Cyclopentyl-6-(3-(4-ethylpiperazine-1-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>16</b>)</h4><div class="NLM_p last">White solid, 58% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30â7.26 (m, 2H), 7.25â7.22 (m, 2H), 7.15 (dd, <i>J</i> = 8.3, 1.8 Hz, 1H), 6.90 (d, <i>J</i> = 8.4 Hz, 1H), 6.86 (d, <i>J</i> = 1.8 Hz, 1H), 4.13 (q, <i>J</i> = 6.8 Hz, 1H), 4.05â3.95 (m, 2H), 3.94â3.84 (m, 2H), 3.50â3.41 (m, 1H), 3.35 (s, 3H), 2.59â2.51 (m, 4H), 2.47 (q, <i>J</i> = 7.2 Hz, 2H), 2.40 (s, 3H), 1.84â1.75 (m, 1H), 1.73â1.64 (m, 1H), 1.62â1.44 (m, 5H), 1.36â1.28 (m, 1H), 1.11 (t, <i>J</i> = 7.2 Hz, 3H), 0.98 (d, <i>J</i> = 6.8 Hz, 3H); MS (ESI) [M + H]<sup>+</sup>: 542.42; retention time 2.98 min, >99% pure.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (<i>R</i>)-4-Cyclopentyl-6-(3-(4-isopropylpiperazine-1-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>17</b>)</h4><div class="NLM_p last">White solid, 62% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30â7.26 (m, 2H), 7.26â7.22 (m, 2H), 7.15 (dd, <i>J</i> = 8.3, 1.9 Hz, 1H), 6.91 (d, <i>J</i> = 8.4 Hz, 1H), 6.84 (d, <i>J</i> = 1.8 Hz, 1H), 4.13 (q, <i>J</i> = 6.8 Hz, 1H), 4.01â3.94 (m, 2H), 3.91â3.83 (m, 2H), 3.48â3.40 (m, 1H), 3.34 (s, 3H), 2.83â2.70 (m, 1H), 2.69â2.57 (m, 4H), 2.39 (s, 3H), 1.84â1.75 (m, 1H), 1.72â1.64 (m, 1H), 1.63â1.44 (m, 5H), 1.34â1.27 (m, 1H), 1.06 (d, <i>J</i> = 6.6 Hz, 6H), 0.98 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´ 169.10, 160.58, 156.87, 154.47, 139.62, 135.70, 135.53, 132.26, 130.17 (2 Ã C), 125.69 (2 Ã C), 122.18, 120.22, 116.16, 114.58, 59.01, 54.84, 54.58, 49.31, 48.32, 47.45, 42.75, 30.80, 30.35, 29.25, 24.03, 23.56, 21.32, 18.45 (2 Ã C), 14.11; MS (ESI) [M + H]<sup>+</sup>: 556.44; retention time 3.04 min, >99% pure.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (<i>R</i>)-4-Cyclopentyl-6-(3-(4-cyclopropylpiperazine-1-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>18</b>)</h4><div class="NLM_p last">White solid, 63% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30â7.21 (m, 4H), 7.15 (d, <i>J</i> = 7.2 Hz, 1H), 6.90 (d, <i>J</i> = 8.4 Hz, 1H), 6.85 (s, 1H), 4.21â4.04 (m, 1H), 3.93â3.86 (m, 2H), 3.84â3.76 (m, 2H), 3.50â3.39 (m, 1H), 3.34 (s, 3H), 2.75â2.62 (m, 4H), 2.39 (s, 3H), 2.28â2.15 (m, 1H), 1.82â1.74 (m, 1H), 1.69â1.63 (m, 2H), 1.58â1.46 (m, 5H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H), 0.87â0.82 (m, 2H), 0.49â0.43 (m, 2H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 169.05, 160.68, 156.92, 154.41, 139.56, 135.72, 135.50, 132.24, 130.13 (2 Ã C), 125.66 (2 Ã C), 122.19, 120.20, 116.16, 114.53, 59.03, 54.54, 53.77, 52.97, 47.21, 42.53, 38.41, 30.77, 30.34, 29.77, 29.21, 24.00, 23.53, 21.28, 14.12, 5.99; MS (ESI) [M + H]<sup>+</sup>: 554.44; retention time 3.00 min, >99% pure.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>R</i>)-4-Cyclopentyl-6-(3-((3<i>S</i>,5<i>R</i>)-3,5-dimethylpiperazine-1-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>19</b>)</h4><div class="NLM_p last">White solid, 59% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30â7.21 (m, 4H), 7.16â7.11 (m, 1H), 6.90 (dd, <i>J</i> = 8.3, 2.7 Hz, 1H), 6.85 (d, <i>J</i> = 7.3 Hz, 1H), 4.68 (d, <i>J</i> = 12.9 Hz, 1H), 4.44 (d, <i>J</i> = 11.9 Hz, 1H), 4.12 (q, <i>J</i> = 6.8 Hz, 1H), 3.49â3.39 (m, 1H), 3.33 (s, 3H), 3.04â2.89 (m, 2H), 2.89â2.78 (m, 1H), 2.48â2.40 (m, 1H), 2.38 (s, 3H), 2.34 (s, 1H), 1.82â1.74 (m, 1H), 1.72â1.63 (m, 1H), 1.62â1.42 (m, 6H), 1.14 (d, <i>J</i> = 6.2 Hz, 3H), 1.07 (d, <i>J</i> = 5.8 Hz, 3H), 0.97 (d, <i>J</i> = 6.6 Hz, 3H);<sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´ 169.04, 160.50(d, <i>J</i> = 5.1 Hz), 156.88, 154.42, 139.60, 135.63, 135.50, 132.22, 130.13 (2 Ã C), 125.63 (2 Ã C), 122.10, 120.15, 116.07, 114.55 (d, <i>J</i> = 4.2 Hz), 58.95, 54.52, 53.49 (d, <i>J</i> = 4.0 Hz), 51.62, 50.79, 48.70, 30.77, 30.32(d, <i>J</i> = 4.3 Hz), 29.21, 23.98, 23.51, 21.28, 19.38, 19.19, 14.07(d, <i>J</i> = 5.5 Hz) ; MS (ESI) [M + H]<sup>+</sup>: 542.42; retention time 2.99 min, >98% pure.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (<i>R</i>)-4-Cyclopentyl-1,3-dimethyl-6-(3-(4-methyl-1,4-diazepane-1-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>20</b>)</h4><div class="NLM_p last">White solid, 62% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30â7.22 (m, 4H), 7.14 (ddd, <i>J</i> = 8.1, 6.2, 1.8 Hz, 1H), 6.91 (d, <i>J</i> = 8.4 Hz, 1H), 6.86 (dd, <i>J</i> = 3.9, 1.8 Hz, 1H), 4.18â4.10 (m, 1H), 4.00â3.95 (m, 1H), 3.91 (t, <i>J</i> = 6.4 Hz, 1H), 3.89â3.86 (m, 1H), 3.83 (t, <i>J</i> = 6.4 Hz, 1H), 3.51â3.41 (m, 1H), 3.35 (s, 3H), 2.86â2.78 (m, 2H), 2.73â2.64 (m, 2H), 2.44â2.38 (m, 6H), 2.09â1.98 (m, 2H), 1.85â1.76 (m, 1H), 1.73â1.65 (m, 1H), 1.62â1.46 (m, 5H), 1.34â1.28 (m, 1H), 0.99 (dd, <i>J</i> = 6.8, 1.3 Hz, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 169.08, 162.31, 157.40, 154.38 (d, <i>J</i> = 6.4 Hz), 139.56, 135.80, 135.57, 132.28, 130.17 (2 Ã C), 125.68 (2 Ã C), 122.29, 120.21 (d, <i>J</i> = 4.5 Hz), 116.19 (d, <i>J</i> = 3.3 Hz), 114.56, 59.98, 59.08, 57.53 (d, <i>J</i> = 22.2 Hz), 56.64, 54.59, 48.48, 46.69 (d, <i>J</i> = 13.2 Hz), 46.15, 30.84, 30.41, 29.26, 24.06, 23.58, 21.33, 14.19; HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for (C<sub>31</sub>H<sub>40</sub>N<sub>7</sub>O<sub>2</sub><sup>+</sup>) 542.3238, found 542.323; retention time 2.97 min, >99% pure.</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (3<i>R</i>)-4-Cyclopentyl-1,3-dimethyl-6-(3-(8-methyl-3,8-diazabicyclo[3.2.1]octane-3-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>21</b>)</h4><div class="NLM_p last">White solid, 67% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30â7.20 (m, 4H), 7.14 (d, <i>J</i> = 8.2 Hz, 1H), 6.89 (d, <i>J</i> = 8.4 Hz, 1H), 6.85 (s, 1H), 5.15 (s, 1H), 4.87 (d, <i>J</i> = 5.9 Hz, 1H), 4.12 (q, <i>J</i> = 6.7 Hz, 1H), 3.51â3.39 (m, 1H), 3.33 (s, 3H), 2.74 (d, <i>J</i> = 10.1 Hz, 2H), 2.50â2.42 (m, 2H), 2.38 (s, 3H), 2.25 (s, 3H), 2.03â1.91 (m, 4H), 1.83â1.74 (m, 1H), 1.71â1.63 (m, 1H), 1.61â1.42 (m, 6H), 0.97 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 169.03, 157.24, 157.01, 154.41, 139.52, 135.80, 135.44, 132.18, 130.08 (2 Ã C), 125.67 (2 Ã C), 122.27, 120.20, 116.18, 114.49, 61.95, 60.28, 59.03, 56.44, 54.57, 52.85, 45.18, 30.74, 30.32, 29.20, 28.66, 26.78, 23.97, 23.52, 21.28, 14.11; MS (ESI) [M + H]<sup>+</sup>: 554.44; retention time 2.98 min, >99% pure.</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (3<i>R</i>)-4-Cyclopentyl-1,3-dimethyl-6-(3-(3-methyl-3,8-diazabicyclo[3.2.1]octane-8-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>22</b>)</h4><div class="NLM_p last">White solid, 60% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.29â7.21 (m, 4H), 7.15 (d, <i>J</i> = 8.3 Hz, 1H), 6.90 (d, <i>J</i> = 8.4 Hz, 1H), 6.83 (t, <i>J</i> = 2.0 Hz, 1H), 5.21 (s, 1H), 4.88 (d, <i>J</i> = 6.2 Hz, 1H), 4.12 (q, <i>J</i> = 6.8 Hz, 1H), 3.47â3.38 (m, 1H), 3.33 (s, 3H), 2.85â2.77 (m, 2H), 2.50 (dd, <i>J</i> = 15.8, 11.1 Hz, 2H), 2.38 (s, 3H), 2.29 (d, <i>J</i> = 2.2 Hz, 3H), 2.06â1.86 (m, 4H), 1.82â1.73 (m, 1H), 1.73â1.63 (m, 1H), 1.60â1.42 (m, 6H), 0.96 (d, <i>J</i> = 6.8 Hz, 3H);<sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´ 169.03, 157.06, 156.97, 154.50, 139.59, 135.69, 135.43, 132.20, 130.11 (2 Ã C), 125.65 (2 Ã C), 122.14, 120.19, 116.12, 114.54, 61.83, 60.20, 58.96, 56.33, 54.56, 52.81, 45.12, 30.73, 30.28, 29.19, 28.52, 26.59, 23.96, 23.50, 21.27, 14.06 (d, <i>J</i> = 2.9 Hz); MS (ESI) [M + H]<sup>+</sup>: 554.42; retention time 3.01 min, >99% pure.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (3<i>R</i>)-4-Cyclopentyl-1,3-dimethyl-6-(3-(octahydropyrrolo[1,2-a]pyrazine-2-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>23</b>)</h4><div class="NLM_p last">White solid, 62% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.31â7.20 (m, 4H), 7.14 (d, <i>J</i> = 8.1 Hz, 1H), 6.90 (d, <i>J</i> = 8.4 Hz, 1H), 6.88â6.83 (m, 1H), 4.84 (dd, <i>J</i> = 54.8, 12.9 Hz, 1H), 4.56 (dd, <i>J</i> = 26.1, 13.0 Hz, 1H), 4.12 (q, <i>J</i> = 6.9 Hz, 1H), 3.49â3.39 (m, 1H), 3.34 (s, 3H), 3.11 (t, <i>J</i> = 8.5 Hz, 2H), 3.07â2.98 (m, 1H), 2.70â2.61 (m, 1H), 2.39 (s, 3H), 2.30 (t, <i>J</i> = 11.1 Hz, 1H), 2.23â2.14 (m, 1H), 2.12â2.01 (m, 1H), 1.95â1.63 (m, 6H), 1.62â1.41 (m, 6H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H); MS (ESI) [M + H]<sup>+</sup>: 554.46; retention time 3.09 min, >99% pure.</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (<i>R</i>)-6-(3-(8-Azaspiro[4.5]decane-8-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-4-cyclopentyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (24)</h4><div class="NLM_p last">White solid, 63% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.28â7.24 (m, 2H), 7.21 (d, <i>J</i> = 8.3 Hz, 2H), 7.13 (dd, <i>J</i> = 8.3, 1.9 Hz, 1H), 6.88 (d, <i>J</i> = 8.4 Hz, 1H), 6.84 (d, <i>J</i> = 1.8 Hz, 1H), 4.11 (q, <i>J</i> = 6.8 Hz, 1H), 3.80â3.70 (m, 4H), 3.48â3.38 (m, 1H), 3.32 (s, 3H), 2.37 (s, 3H), 1.83â1.73 (m, 1H), 1.69â1.58 (m, 5H), 1.56â1.43 (m, 13H), 1.33â1.24 (m, 1H), 0.96 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 168.99, 160.82, 157.27, 154.26, 139.39, 135.73, 135.42, 132.12, 130.04 (2 Ã C), 125.60 (2 Ã C), 122.27, 120.15, 116.13, 114.46, 58.98, 54.49, 45.36, 41.48, 40.54, 38.21, 37.78, 37.73, 37.01, 30.72, 30.29, 29.16, 24.34 (2 Ã C), 23.95, 23.48, 21.23, 14.07; MS (ESI) [M + H]<sup>+</sup>: 567.70; retention time 4.75 min, >95% pure.</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (<i>R</i>)-6-(3-(2-Oxa-8-azaspiro[4.5]decane-8-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-4-cyclopentyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>25</b>)</h4><div class="NLM_p last">White solid, 65% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.31â7.26 (m, 2H), 7.23 (d, <i>J</i> = 8.3 Hz, 2H), 7.14 (dd, <i>J</i> = 8.3, 1.5 Hz, 1H), 6.90 (d, <i>J</i> = 8.4 Hz, 1H), 6.85 (s, 1H), 4.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.89 (t, <i>J</i> = 7.1 Hz, 3H), 3.80â3.69 (m, 2H), 3.60 (d, <i>J</i> = 1.0 Hz, 2H), 3.47â3.39 (m, 1H), 3.34 (s, 3H), 2.39 (s, 3H), 1.86â1.76 (m, 4H), 1.72â1.65 (m, 5H), 1.60â1.45 (m, 6H), 0.98 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 169.06, 160.90, 157.08, 154.43, 139.56, 135.74, 135.52, 132.26, 130.14 (2 Ã C), 125.66 (2 Ã C), 122.23, 120.20, 116.16, 114.54, 67.42, 59.05, 54.55, 45.37, 42.38, 40.73, 37.33, 37.29, 35.78, 34.60, 30.79, 30.36, 29.23, 24.02, 23.55, 21.30, 14.15; MS (ESI) [M + H]<sup>+</sup>: 569.49; retention time 3.68 min, >98% pure.</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>tert</i>-Butyl((<i>S</i>)-5-(5-((<i>R</i>)-4-cyclopentyl-1,3-dimethyl-2-oxo-1,2,3,4-tetrahydroquinoxalin-6-yl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazole-3-carbonyl)-5-azaspiro[2.4]heptan-7-yl)carbamate (<b>26</b>)</h4><div class="NLM_p last">White solid, 58% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.27â7.19 (m, 5H), 7.16â7.09 (m, 1H), 6.87 (dd, <i>J</i> = 8.4, 3.2 Hz, 1H), 6.82 (d, <i>J</i> = 3.5 Hz, 1H), 4.89â4.79 (m, 1H), 4.29 (dd, <i>J</i> = 12.1, 5.3 Hz, 1H), 4.18â4.07 (m, 2H), 4.00 (dd, <i>J</i> = 12.9, 5.7 Hz, 1H), 3.92â3.85 (m, 1H), 3.83â3.69 (m, 2H), 3.53â3.38 (m, 2H), 3.31 (d, <i>J</i> = 2.7 Hz, 3H), 2.37 (s, 3H), 1.81â1.72 (m, 1H), 1.69â1.61 (m, 1H), 1.59â1.44 (m, 6H), 1.42â1.36 (m, 10H), 0.95 (dd, <i>J</i> = 6.8, 1.4 Hz, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 168.99 (d, <i>J</i> = 5.5 Hz), 159.83 (d, <i>J</i> = 33.6 Hz), 157.03 (d, <i>J</i> = 11.9 Hz), 155.48 (d, <i>J</i> = 9.6 Hz), 154.48, 139.53 (d, <i>J</i> = 9.1 Hz), 135.71, 135.41, 132.16, 130.07, 130.05, 125.59 (2 Ã C), 122.13 (d, <i>J</i> = 6.2 Hz), 120.20 (d, <i>J</i> = 12.2 Hz), 116.14 (d, <i>J</i> = 7.5 Hz), 114.46, 58.96, 56.70, 55.71, 54.75, 54.47, 52.71, 49.46, 30.73, 30.72, 30.29, 29.17, 28.36 (3 Ã C), 27.01, 23.95, 23.48, 21.24, 14.07 (d, <i>J</i> = 3.0 Hz); MS (ESI) [M + H]<sup>+</sup>: 640.31; retention time 4.10 min, >99% pure.</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (<i>R</i>)-6-(3-((<i>S</i>)-7-Amino-5-azaspiro[2.4]heptane-5-carbonyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazol-5-yl)-4-cyclopentyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>27</b>)</h4><div class="NLM_p last">White solid, 67% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.30â7.22 (m, 4H), 7.20â7.11 (m, 1H), 6.93â6.88 (m, 1H), 6.87â6.84 (m, 1H), 4.36â4.19 (m, 1H), 4.13 (q, <i>J</i> = 6.8 Hz, 1H), 4.04â3.94 (m, 1H), 3.87 (dd, <i>J</i> = 37.4, 12.0 Hz, 1H), 3.66 (dd, <i>J</i> = 56.3, 13.2 Hz, 1H), 3.50â3.40 (m, 1H), 3.34 (d, <i>J</i> = 2.3 Hz, 3H), 3.18â3.11 (m, 1H), 2.40 (s, 3H), 2.00 (s, 2H), 1.85â1.74 (m, 1H), 1.73â1.64 (m, 1H), 1.61â1.43 (m, 6H), 0.98 (d, <i>J</i> = 6.8 Hz, 3H), 0.83â0.76 (m, 1H), 0.70â0.54 (m, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 169.11, 159.90, 157.29 (d, <i>J</i> = 6.4 Hz), 154.51 (d, <i>J</i> = 6.1 Hz), 139.59, 135.86, 135.52, 132.25, 130.14 (2 Ã C), 125.71 (2 Ã C), 122.30, 120.30 (d, <i>J</i> = 10.3 Hz), 116.23, 114.56, 59.06, 57.23, 56.68, 55.29, 54.86, 54.55, 52.90, 30.82, 30.38, 29.81, 29.25, 24.03, 23.57, 21.33, 14.17; MS (ESI) [M + H]<sup>+</sup>: 540.55; retention time 2.66 min, >99% pure.</div></div><div id="sec4_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (<i>R</i>)-4-Cyclopentyl-1,3-dimethyl-6-(3-(4-methylpiperazine-1-carbonyl)-1-phenyl-1<i>H</i>-1,2,4-triazol-5-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>28</b>)</h4><div class="NLM_p last">White solid, 63% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.47â7.36 (m, 5H), 7.11 (dd, <i>J</i> = 8.3, 1.5 Hz, 1H), 6.89 (d, <i>J</i> = 8.3 Hz, 1H), 6.84 (s, 1H), 4.11 (q, <i>J</i> = 6.7 Hz, 1H), 3.99â3.92 (m, 2H), 3.89â3.82 (m, 2H), 3.47â3.37 (m, 1H), 3.33 (s, 3H), 2.54â2.44 (m, 4H), 2.31 (s, 3H), 1.82â1.72 (m, 1H), 1.70â1.61 (m, 1H), 1.60â1.42 (m, 5H), 1.34â1.25 (m, 1H), 0.96 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 169.01, 160.59, 156.96, 154.52, 138.14, 135.53, 132.29, 129.57(2 Ã C), 129.29, 125.78(2 Ã C), 122.03, 120.14, 116.13, 114.54, 59.02, 55.43, 54.65, 54.49, 47.06, 46.05, 42.40, 30.72, 30.32, 29.20, 23.96, 23.50, 14.16; MS (ESI)[M + H]<sup>+</sup>: 514.44; retention time 2.83 min, >99% pure.</div></div><div id="sec4_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (<i>R</i>)-4-Cyclopentyl-6-(1-(2,4-dimethylphenyl)-3-(4-methylpiperazine-1-carbonyl)-1<i>H</i>-1,2,4-triazol-5-yl)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>29</b>)</h4><div class="NLM_p last">White solid, 59% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.20â7.15 (m, 2H), 7.15â7.13 (m, 1H), 7.13â7.09 (m, 1H), 6.89 (d, <i>J</i> = 1.9 Hz, 1H), 6.86 (d, <i>J</i> = 8.5 Hz, 1H), 4.11 (q, <i>J</i> = 6.8 Hz, 1H), 4.00â3.94 (m, 2H), 3.89â3.82 (m, 2H), 3.40â3.33 (m, 1H), 3.31 (s, 3H), 2.54â2.50 (m, 2H), 2.50â2.46 (m, 2H), 2.38 (s, 3H), 2.32 (s, 3H), 1.98 (s, 3H), 1.83â1.75 (m, 1H), 1.74â1.66 (m, 1H), 1.66â1.55 (m, 2H), 1.55â1.44 (m, 3H), 1.36â1.27 (m, 1H), 0.95 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´ 169.04, 160.67, 156.77, 155.01, 140.48, 135.57, 135.13, 134.96, 132.22, 132.15, 127.98, 127.45, 122.09, 119.66, 115.09, 114.58, 58.95, 55.44, 54.69, 54.23, 47.08, 46.06, 42.43, 30.79, 30.35, 29.19, 23.94, 23.43, 21.33, 17.64, 14.17; MS (ESI) [M + H]<sup>+</sup>: 542.45; retention time 2.98 min, >99% pure.</div></div><div id="sec4_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (<i>R</i>)-4-Cyclopentyl-6-(1-(3-fluoro-4-methylphenyl)-3-(4-methylpiperazine-1-carbonyl)-1<i>H</i>-1,2,4-triazol-5-yl)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>30</b>)</h4><div class="NLM_p last">White solid, 56% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.22 (t, <i>J</i> = 8.0 Hz, 1H), 7.11â7.03 (m, 3H), 6.89 (d, <i>J</i> = 8.4 Hz, 1H), 6.83 (d, <i>J</i> = 1.7 Hz, 1H), 4.11 (q, <i>J</i> = 6.8 Hz, 1H), 3.94â3.88 (m, 2H), 3.87â3.79 (m, 2H), 3.53â3.43(m, 1H), 3.32 (s, 3H), 2.52â2.42 (m, 4H), 2.29 (s, 3H), 2.28 (d, <i>J</i> = 1.5 Hz, 3H), 1.84â1.75 (m, 1H), 1.72â1.62 (m, 1H), 1.60â1.43 (m, 5H), 1.37â1.24 (m, 1H), 0.96 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´ 168.98, 160.39, 156.94, 154.52, 136.69 (d, <i>J</i> = 9.8 Hz), 135.54, 132.40, 132.05 (d, <i>J</i> = 5.8 Hz), 126.48 (d, <i>J</i> = 17.1 Hz), 121.73, 121.13 (d, <i>J</i> = 3.3 Hz), 120.11, 116.03, 114.58, 112.93, 112.76, 58.95, 55.37, 54.59, 54.54, 47.01, 46.01, 42.36, 30.68, 30.29, 29.17, 23.95, 23.46, 14.43 (d, <i>J</i> = 2.9 Hz), 14.02; HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for (C<sub>30</sub>H<sub>37</sub>FN<sub>7</sub>O<sub>2</sub><sup>+</sup>) 546.2987, found 546.2984; retention time 2.97 min, >99% pure.</div></div><div id="sec4_2_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (<i>R</i>)-6-(1-(4-Chlorophenyl)-3-(4-methylpiperazine-1-carbonyl)-1<i>H</i>-1,2,4-triazol-5-yl)-4-cyclopentyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>31</b>)</h4><div class="NLM_p last">White solid, 66% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.44â7.39 (m, 2H), 7.37â7.32 (m, 2H), 7.08 (dd, <i>J</i> = 8.3, 1.7 Hz, 1H), 6.91 (d, <i>J</i> = 8.3 Hz, 1H), 6.82 (d, <i>J</i> = 1.7 Hz, 1H), 4.14 (q, <i>J</i> = 6.8 Hz, 1H), 4.00â3.90 (m, 2H), 3.90â3.79 (m, 2H), 3.55â3.43 (m, 1H), 3.34 (s, 3H), 2.55â2.50 (m, 2H), 2.50â2.45 (m, 2H), 2.32 (s, 3H), 1.87â1.78 (m, 1H), 1.74â1.66 (m, 1H), 1.63â1.46 (m, 5H), 1.37â1.28 (m, 1H), 0.99 (d, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 169.00, 160.42, 157.21, 154.67, 136.56, 135.71, 135.35, 132.52, 129.80 (2 Ã C), 127.00 (2 Ã C), 121.74, 120.14, 116.06, 114.67, 59.11, 55.46, 54.68, 54.55, 47.07, 46.08, 42.45, 30.81, 30.43, 29.25, 24.05, 23.57, 14.22; MS (ESI) [M + H]<sup>+</sup>: 548.38; retention time 3.01 min, >99% pure.</div></div><div id="sec4_2_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 3-Fluoro-4-nitro-<i>N</i>-(<i>p</i>-tolyl)benzamide (<b>38</b>)</h4><div class="NLM_p last">To a solution of compound <b>37</b> (2 g, 10.8 mmol) in anhydrous DMF (10 mL) was added HATU (4.1 g, 10.8 mmol), the mixture was stirred at rt for 30 min, then <i>p</i>-methylaniline (1.2 g, 10.8 mmol) and DIPEA (1.9 mL, 10.8 mmol) were added, and the mixture was stirred at room temperature overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with saturated NaHCO<sub>3</sub> and extracted with EtOAc (3 Ã 40 mL). The combined organic extracts were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 25% EtOAc/60â90 Â°C petroleum ether) gave compound <b>38</b> as a light yellow solid (2.2 g, 8.0 mmol, 74% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d<sub>6</sub></i>) Î´ 10.49 (s, 1H), 8.31 (t, <i>J</i> = 8.1 Hz, 1H), 8.10 (d, <i>J</i> = 12.0 Hz, 1H), 7.96 (d, <i>J</i> = 8.5 Hz, 1H), 7.65 (d, <i>J</i> = 8.4 Hz, 2H), 7.19 (d, <i>J</i> = 8.3 Hz, 2H), 2.29 (s, 3H); retention time 3.29 min, >95% pure.</div></div><div id="sec4_2_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 3-(Cyclopropylamino)-4-nitro-<i>N</i>-(<i>p</i>-tolyl)benzamide (<b>39</b>)</h4><div class="NLM_p last">To a solution of compound <b>38</b> (2 g, 7.3 mmol) in 1,2-dichloroethane (10 mL) was added cyclopropylamine (1.0 mL, 14.6 mmol), and then the mixture was heated to 80 Â°C for 12 h. The reaction was monitored by TLC. Upon completion, the solvent was evaporated, diluted with water, and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 Ã 30 mL). The combined organic extracts were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure to give compound <b>39</b> as a red solid (2.1 g, 6.7 mmol, 92% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.23 (d, <i>J</i> = 8.8 Hz, 1H), 8.12 (s, 1H), 7.86 (s, 1H), 7.80 (s, 1H), 7.53 (d, <i>J</i> = 8.1 Hz, 2H), 7.19 (d, <i>J</i> = 8.3 Hz, 2H), 7.01 (d, <i>J</i> = 8.7 Hz, 1H), 2.69â2.62 (m, 1H), 2.35 (s, 3H), 1.00â0.94 (m, 2H), 0.71â0.66 (m, 2H); retention time 3.45 min, >95% pure.</div></div><div id="sec4_2_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 3-(Cyclopropylamino)-4-nitro-<i>N</i>-(<i>p</i>-tolyl)benzothioamide (<b>41</b>)</h4><div class="NLM_p last">To a solution of compound <b>39</b> (2 g, 6.4 mmol) in toluene (10 mL) was added Lawessonâs reagent (1.4 g, 3.4 mmol), and then the mixture was heated to 110 Â°C overnight. The reaction was monitored by TLC. Upon completion, the solvent was evaporated, and the residue was dissolved with CH<sub>2</sub>Cl<sub>2</sub>. Purification by silica gel column chromatography (gradient elution, gradient 0 to 15% EtOAc/60â90 Â°C petroleum ether) gave compound <b>41</b> as a red solid (1.8 g, 5.5 mmol, 86% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d<sub>6</sub></i>) Î´ 11.97 (s, 1H), 8.12 (d, <i>J</i> = 8.8 Hz, 2H), 7.73 (d, <i>J</i> = 8.2 Hz, 2H), 7.65 (s, 1H), 7.26 (d, <i>J</i> = 8.2 Hz, 2H), 7.07 (d, <i>J</i> = 8.8 Hz, 1H), 2.71 (s, 1H), 2.33 (s, 3H), 0.89 (d, <i>J</i> = 5.0 Hz, 2H), 0.68 (s, 2H). MS (ESI) [M + H]<sup>+</sup>: 328.11; LCMS <i>m</i>/<i>z</i> (ESI, positive) found [M + H]<sup>+</sup> 328.09; retention time 3.68 min, >95% pure.</div></div><div id="sec4_2_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>N</i>-Cyclopropyl-2-nitro-5-(4-(<i>p</i>-tolyl)-4<i>H</i>-1,2,4-triazol-3-yl)aniline (<b>43</b>)</h4><div class="NLM_p last">To a solution of compound <b>41</b> (1.8 g, 5.5 mmol) in methanol (10 mL) and THF (20 mL) was added hydrazine hydrate (2.7 mL, 55.0 mmol), and then the mixture was stirred at rt for 4 h. The reaction was monitored by TLC. Upon completion, the solvent was evaporated, the residue was dissolved with DMF (5 mL), and then acetic acid (4 mL) and trimethyl orthoformate (1.6 mL, 14.8 mmol) were added. The reaction mixture was stirred at rt overnight. The reaction was monitored by TLC. Upon completion, the residue was basified with saturated NaHCO<sub>3</sub> to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 40 mL). Combined organic layers were washed with brine, and purification by silica gel column chromatography (gradient elution, gradient 0 to 30% EtOAc/60â90 Â°C petroleum ether) gave compound <b>43</b> as a red solid (1.5 g, 4.5 mmol, 82% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.34 (s, 1H), 8.10 (d, <i>J</i> = 8.8 Hz, 1H), 8.03 (s, 1H), 7.45 (s, 1H), 7.31 (d, <i>J</i> = 7.9 Hz, 2H), 7.17 (d, <i>J</i> = 8.2 Hz, 2H), 6.81 (d, <i>J</i> = 8.8 Hz, 1H), 2.44 (s, 3H), 2.36â2.29 (m, 1H), 0.74â0.68 (m, 2H), 0.51â0.46 (m, 2H); retention time 3.41 min, >95% pure.</div></div><div id="sec4_2_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> <i>N</i>-Cyclopropyl-2-nitro-5-(1-(<i>p</i>-tolyl)-1<i>H</i>-tetrazol-5-yl)aniline (<b>44</b>)</h4><div class="NLM_p">To a solution of compound <b>41</b> (2 g, 6.1 mmol) in THF (10 mL) were added Hg(OAc)<sub>2</sub> (3.9 g, 12.2 mmol) and azidotrimethylsilane (8.0 mL, 61.2 mmol) at 0 Â°C, and then the mixture was stirred at 0 Â°C for 4 h. The reaction was monitored by TLC. Upon completion, the mixture was diluted with water and extracted with EtOAc (3 Ã 40 mL). Combined organic layers were washed with brine, and purification by silica gel column chromatography (gradient elution, gradient 0 to 10% EtOAc/60â90 Â°C petroleum ether) gave compound <b>44</b> as a red solid (1.8 g, 5.4 mmol, 88% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.14 (d, <i>J</i> = 8.9 Hz, 1H), 8.03 (s, 1H), 7.53 (d, <i>J</i> = 1.8 Hz, 1H), 7.35 (d, <i>J</i> = 8.2 Hz, 2H), 7.32â7.27 (m, 2H), 6.86 (dd, <i>J</i> = 8.9, 1.9 Hz, 1H), 2.46 (s, 3H), 2.39â2.31 (m, 1H), 0.76â0.68 (m, 2H), 0.55â0.44 (m, 2H); MS (ESI) [M + H]<sup>+</sup>: 337.09; retention time 3.43 min, >95% pure.</div><div class="NLM_p last">Compound <b>45</b> was prepared from compound <b>40</b> in the same manner as that of compound <b>44</b>.</div></div><div id="sec4_2_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>N</i><sup>1</sup>-Cyclopropyl-5-(4-(<i>p</i>-tolyl)-4<i>H</i>-1,2,4-triazol-3-yl)benzene-1,2-diamine (<b>46</b>)</h4><div class="NLM_p">To a solution of compound <b>43</b> (1.5 g, 4.5 mmol) in conc. HCl (8 mL) was added tin chloride dihydrate (5.0 g, 22.3 mmol), and then the mixture was stirred at rt for 3 h. The reaction was monitored by TLC. Upon completion, the solvent was evaporated, and the residue was basified with saturated NaHCO<sub>3</sub> to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 20 mL). The combined organic extracts were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure to give compound <b>46</b> as a yellow powder (1.2 g, 3.9 mmol, 87% yield); retention time 3.44 min, >95% pure.</div><div class="NLM_p last">Compound <b>47</b> was prepared from compound <b>44</b> in the same manner as that of compound <b>46</b>, and compound <b>48</b> was prepared from compound <b>45</b> in the same manner as that of compound <b>46</b>.</div></div><div id="sec4_2_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 4-Cyclopropyl-3-methyl-6-(4-(<i>p</i>-tolyl)-4<i>H</i>-1,2,4-triazol-3-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>49</b>)</h4><div class="NLM_p">To a stirred solution of compound <b>46</b> (1.2 g, 3.9 mmol) in anhydrous dichloromethane (8 mL) at 0 Â°C were slowly added DIPEA (1.4 mL, 7.9 mmol) and 2-bromopropanoyl bromide (0.6 mL, 5.9 mmol). After the addition was completed, the cooling bath was removed, and the reaction was stirred for 2 h at rt. The reaction mixture was cooled again to 0 Â°C, diluted with water, and extracted with EtOAc (3 Ã 50 mL). The combined organic fractions were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated by evaporation under reduced pressure. The residue and DIPEA (8 mL) in acetonitrile (20 mL) were heated to 80 Â°C overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with EtOAc (3 Ã 40 mL). The combined organic extracts were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 25% EtOAc/60â90 Â°C petroleum ether) gave compound <b>49</b> as a white solid (0.8 g, 2.2 mmol, 56% yield, two steps); retention time 3.39 min, >95% pure.</div><div class="NLM_p last">Compound <b>50</b> was prepared from compound <b>47</b> in the same manner as compound <b>49</b>, and compound <b>51</b> was prepared from compound <b>48</b> in the same manner as compound <b>49</b>.</div></div><div id="sec4_2_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> Methyl 3-Fluoro-4-nitrobenzoate (<b>52</b>)</h4><div class="NLM_p last">To a solution of compound <b>37</b> (5.0 g, 27.0 mmol) in MeOH (12.9 mL) was added SOCl<sub>2</sub> (4.1 mL, 56.7 mmol) at 0 Â°C. The mixture was heated to 60 Â°C for 24 h and monitored by TLC. Upon completion, the solvent was evaporated. The residue was basified with saturated NaHCO<sub>3</sub> to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 40 mL). Combined organic layers were washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, and evaporated to afford compound <b>52</b> (5.1 g, 25.7 mmol, 95% yield) as a light yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.11 (t, <i>J</i> = 7.8 Hz, 1H), 7.97 (s, 1H), 7.94 (d, <i>J</i> = 3.0 Hz, 1H), 3.98 (s, 3H); MS <i>m/z</i> (EI) found [M]<sup>+</sup> 199; retention time 3.28 min, >95% pure.</div></div><div id="sec4_2_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> Methyl 3-(Cyclopropylamino)-4-nitrobenzoate (<b>53</b>)</h4><div class="NLM_p last">A solution of compound <b>52</b> (5.0 g, 25.1 mmol) and cyclopropanamine (3.5 mL, 50.3 mmol) in 1,2-dichloroethane (20 mL) was heated to 80 Â°C for 12 h. The reaction was monitored by TLC. Upon completion, the solvent was evaporated, diluted with water, and extracted with dichloromethane (3 Ã 50 mL). The combined organic layers were washed with brine, dried over sodium sulfate, and filtered, and excess solvent was removed via a rotary evaporator to give compound <b>53</b> (5.4 g, 22.9 mmol, 91% yield) as a red solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d<sub>6</sub></i>) Î´ 8.15 (d, <i>J</i> = 8.8 Hz, 1H), 8.03 (s, 1H), 7.92 (d, <i>J</i> = 1.7 Hz, 1H), 7.22 (dd, <i>J</i> = 8.8, 1.8 Hz, 1H), 3.89 (s, 3H), 2.72â2.65 (m, 1H), 0.94â0.84 (m, 2H), 0.70â0.60 (m, 2H); MS <i>m/z</i> (EI) found [M]<sup>+</sup> 236; retention time 3.80 min, >99% pure.</div></div><div id="sec4_2_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> Methyl 4-Amino-3-(cyclopropylamino)benzoate (<b>54</b>)</h4><div class="NLM_p last">To a solution of compound <b>53</b> (5.4 g, 22.9 mmol) and iron (5.1 g, 91.8 mmol) in EtOH (20 mL) was added ammonium chloride (4.9 g, 91.8 mmol) in 5 mL of water at 50â55 Â°C. The reaction mixture was heated to 80 Â°C for 1 h, then cooled to room temperature, and filtered through Celite. The filtrate was basified with saturated NaHCO<sub>3</sub> solution to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 40 mL). The combined organic extracts were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure. The crude product was purified by flash column chromatography (gradient elution, gradient 0 to 20% EtOAc/60â90 Â°C petroleum ether) to give compound <b>54</b> (3.5 g, 17.0 mmol, 74% yield) as a brown liquid; LCMS <i>m</i>/<i>z</i> (ESI, positive) found [M + H]<sup>+</sup> 207.61; retention time 2.36 min, >95% pure.</div></div><div id="sec4_2_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> Methyl 4-Cyclopropyl-3-methyl-2-oxo-1,2,3,4-tetrahydroquinoxaline-6-carboxylate (<b>55</b>)</h4><div class="NLM_p last">To a stirred solution of compound <b>54</b> (3.5 g, 17.0 mmol) in anhydrous dichloromethane (15 mL) at 0 Â°C were slowly added DIPEA (5.9 mL, 33.9 mmol) and 2-bromopropanoyl bromide (2.1 mL, 20.4 mmol). After the addition was completed, the cooling bath was removed, and the reaction was stirred for 2 h at rt. The reaction mixture was cooled again to 0 Â°C, diluted with water, and extracted with EtOAc (3 Ã 30 mL). The combined organic fractions were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated by evaporation under reduced pressure to give the intermediate. The intermediate and DIPEA (5.9 mL, 33.9 mmol) in acetonitrile (15 mL) were heated to 80 Â°C overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with EtOAc (3 Ã 40 mL). The combined organic extracts were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 25% EtOAc/60â90 Â°C petroleum ether) gave compound <b>55</b> (1.8 g, 6.9 mmol, 41% yield, two steps) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.61 (s, 1H), 7.75 (d, <i>J</i> = 1.6 Hz, 1H), 7.53 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 6.78 (d, <i>J</i> = 8.1 Hz, 1H), 4.13â4.05 (m, 1H), 3.90 (s, 3H), 2.52â2.45 (m, 1H), 1.24 (d, <i>J</i> = 6.9 Hz, 3H), 1.10â1.02 (m, 1H), 0.86â0.78 (m, 1H), 0.68â0.60 (m, 1H), 0.60â0.51 (m, 1H); MS <i>m/z</i> (ESI, positive) found [M + H]<sup>+</sup> 261.20; retention time 3.01 min, >99% pure.</div></div><div id="sec4_2_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Methyl 4-Cyclopropyl-1,3-dimethyl-2-oxo-1,2,3,4-tetrahydroquinoxaline-6-carboxylate (<b>56</b>)</h4><div class="NLM_p last">To a solution of compound <b>55</b> (1.8 g, 6.9 mmol) in anhydrous DMF (10 mL) was added NaH (0.8 g, 20.8 mmol) at 0 Â°C, the mixture was stirred at 0 Â°C for 30 min, and then iodomethane (0.7 mL, 10.4 mmol) was added and stirred at room temperature for another 2 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was acidified with 1 N aq. HCl to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 20 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 25% EtOAc/60â90 Â°C petroleum ether) gave compound <b>56</b> (1.5 g, 5.5 mmol, 79% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.75 (d, <i>J</i> = 1.9 Hz, 1H), 7.59 (dd, <i>J</i> = 8.3, 1.9 Hz, 1H), 6.93 (d, <i>J</i> = 8.4 Hz, 1H), 4.12 (q, <i>J</i> = 6.9 Hz, 1H), 3.89 (s, 3H), 3.35 (s, 3H), 2.48â2.42 (m, 1H), 1.15 (d, <i>J</i> = 6.9 Hz, 3H), 1.07â0.99 (m, 1H), 0.84â0.76 (m, 1H), 0.66â0.58 (m, 1H), 0.56â0.48 (m, 1H); LCMS <i>m</i>/<i>z</i> (ESI, positive) found [M + H]<sup>+</sup> 275.18; retention time 3.38 min, >99% pure.</div></div><div id="sec4_2_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 4-Cyclopropyl-1,3-dimethyl-2-oxo-1,2,3,4-tetrahydroquinoxaline-6-carbohydrazide (<b>57</b>)</h4><div class="NLM_p last">To a solution of compound <b>56</b> (1.5 g, 5.5 mmol) in EtOH (10 mL) was added hydrazine hydrate (1.3 mL, 27.4 mmol) at 0 Â°C, and then the mixture was heated to 80 Â°C for 24 h. The reaction was monitored by TLC. Upon completion, the solvent was evaporated to give compound <b>57</b> (1.3 g, 4.7 mmol, 87% yield) as a white solid. Retention time 3.32 min, >95% pure.</div></div><div id="sec4_2_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 2-(3-Fluoro-4-nitrophenyl)-3-(<i>p</i>-tolyl)thiazolidin-4-one (<b>59</b>)</h4><div class="NLM_p last">To a solution of 3-fluoro-4-nitrobenzaldehyde (6.3 g, 37.4 mmol) in THF (20 mL) was added <i>p</i>-toluidine (2 g, 18.7 mmol) at 0 Â°C. The reaction mixture was stirred at 0 Â°C for 10 min, then thioglycolic acid (3.9 mL, 56.1 mmol) was added, and the mixture further stirred at 0 Â°C for 10 min. Then, <i>N</i>,<i>N</i>â²-dicyclohexylcarbodiimide (4.6 g, 22.4 mmol) was added, and the mixture was stirred at rt overnight. The reaction was monitored by TLC. Upon completion, the mixture was filtered through Celite. The filtrate was concentrated under reduced pressure. The crude product was purified by flash column chromatography (gradient elution, gradient 0 to 10% EtOAc/60â90 Â°C petroleum ether) to give compound <b>59</b> (3.5 g, 10.5 mmol, 56% yield) as a light yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d<sub>6</sub></i>) Î´ 8.06 (t, <i>J</i> = 8.1 Hz, 1H), 7.70 (d, <i>J</i> = 12.0 Hz, 1H), 7.47 (d, <i>J</i> = 8.5 Hz, 1H), 7.24 (d, <i>J</i> = 8.2 Hz, 2H), 7.12 (d, <i>J</i> = 8.2 Hz, 2H), 6.61 (s, 1H), 4.11 (d, <i>J</i> = 15.6 Hz, 1H), 3.89 (d, <i>J</i> = 15.6 Hz, 1H), 2.21 (s, 3H); retention time 3.24 min, >95% pure.</div></div><div id="sec4_2_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 2-(3-(Cyclopropylamino)-4-nitrophenyl)-3-(<i>p</i>-tolyl)thiazolidin-4-one (<b>60</b>)</h4><div class="NLM_p last">A solution of compound <b>59</b> (2 g, 6.0 mmol) and cyclopropanamine (0.8 mL, 12.0 mmol) in 1,2-dichloroethane (10 mL) was heated to 80 Â°C for 12 h. The reaction was monitored by TLC. Upon completion, the solvent was evaporated, diluted with water, and extracted with dichloromethane (3 Ã 50 mL). The combined organic layers were washed with brine, dried over sodium sulfate, and filtered, and excess solvent was removed via a rotary evaporator to give compound <b>60</b> (1.8 g, 4.9 mmol, 81% yield) as a red solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.11 (d, <i>J</i> = 8.9 Hz, 1H), 8.07 (s, 1H), 7.13â7.11 (m, 4H), 6.66 (dd, <i>J</i> = 8.9, 1.9 Hz, 1H), 6.05 (d, <i>J</i> = 1.0 Hz, 1H), 3.99 (dd, <i>J</i> = 15.8, 1.6 Hz, 1H), 3.87 (d, <i>J</i> = 15.8 Hz, 1H), 2.54â2.47(m, 1H), 2.28 (s, 3H), 0.92â0.84 (m, 2H), 0.60â0.54 (m, 2H). Retention time 3.31 min, >95% pure.</div></div><div id="sec4_2_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 2-(4-Amino-3-(cyclopropylamino)phenyl)-3-(<i>p</i>-tolyl)thiazolidin-4-one (<b>61</b>)</h4><div class="NLM_p last">To a solution of compound <b>60</b> (1.5 g, 4.1 mmol) in conc. HCl (4 mL) was added tin chloride dihydrate (4.4 g, 19.3 mmol), and then the mixture was stirred at rt for 3 h. The reaction was monitored by TLC. Upon completion, the solvent was evaporated, and the residue was basified with saturated NaHCO<sub>3</sub> to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 20 mL). The combined organic extracts were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure to give compound <b>61</b> as a yellow powder (1.1 g, 3.2 mmol, 80% yield); retention time 3.21 min, >95% pure.</div></div><div id="sec4_2_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 2-(4-Cyclopropyl-3-methyl-2-oxo-1,2,3,4-tetrahydroquinoxalin-6-yl)-3-(<i>p</i>-tolyl)thiazolidin-4-one (<b>62</b>)</h4><div class="NLM_p last">To a stirred solution of compound <b>61</b> (1.1 g, 3.2 mmol) in anhydrous dichloromethane (10 mL) at 0 Â°C were slowly added DIPEA (1.1 mL, 6.5 mmol) and 2-bromopropanoyl bromide (0.4 mL, 3.9 mmol). After the addition was completed, the cooling bath was removed, and the reaction was stirred for 2 h at rt. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with EtOAc (3 Ã 20 mL). The combined organic fractions were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated by evaporation under reduced pressure to give the intermediate. The intermediate and DIPEA (1.1 mL, 6.5 mmol) in acetonitrile (10 mL) were heated to 80 Â°C overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with EtOAc (3 Ã 30 mL). The combined organic extracts were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 25% EtOAc/60â90 Â°C petroleum ether) gave compound <b>62</b> (0.1 g, 0.3 mmol, 9% yield, 2 steps) as a white solid. Retention time 3.20 min, >95% pure.</div></div><div id="sec4_2_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> (5-Bromo-2-nitrophenyl)-<span class="smallcaps smallerCapital">d</span>-alanine (<b>68</b>)</h4><div class="NLM_p last">To a solution of compound <b>32</b> (25 g, 113.6 mmol) in EtOH (150 mL) and H<sub>2</sub>O (50 mL) were added <span class="smallcaps smallerCapital">d</span>-alanine (11.1 g, 125.0 mmol) and K<sub>2</sub>CO<sub>3</sub> (17.3 g, 125.0 mmol). The reaction mixture was heated to 80 Â°C for 8 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was acidified with 1 N aq. HCl to pH 2â3. Then, the mixture was filtered, and the solid was dried to give compound <b>68</b> (28.7 g, 99.3 mmol, 87% yield) as a yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.35 (d, <i>J</i> = 6.9 Hz, 1H), 8.06 (d, <i>J</i> = 9.1 Hz, 1H), 6.90 (s, 1H), 6.85 (d, <i>J</i> = 9.2 Hz, 1H), 4.33 (p, <i>J</i> = 7.0 Hz, 1H), 1.67 (d, <i>J</i> = 7.0 Hz, 3H); retention time 3.08 min, >95% pure.</div></div><div id="sec4_2_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> (<i>R</i>)-6-Bromo-3-methyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>69</b>)</h4><div class="NLM_p last">To a solution of compound <b>68</b> (28.7 g, 99.3 mmol) and K<sub>2</sub>CO<sub>3</sub> (27.4 g, 198.6 mmol) in H<sub>2</sub>O (500 mL) was added Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (86.5 g, 496.6 mmol). The reaction mixture was heated to 60 Â°C overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was acidified with 1 N aq. HCl to pH 7â8. Then, the mixture was filtered, and the solid was dried to give compound <b>69</b> (9 g, 37.3 mmol, 38% yield) as a yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 9.42 (s, 1H), 6.89â6.83 (dd, <i>J</i> = 8.2, 2.0 Hz, 1H), 6.81 (s, 1H), 6.68â6.59 (dd, <i>J</i> = 13.1, 7.6 Hz, 1H), 4.02 (q, <i>J</i> = 6.7 Hz, 1H), 3.92 (s, 1H), 1.45 (d, <i>J</i> = 6.7 Hz, 3H); retention time 3.01 min, >95% pure.</div></div><div id="sec4_2_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> (<i>R</i>)-6-Bromo-4-cyclopentyl-3-methyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>70</b>)</h4><div class="NLM_p last">A solution of compound <b>69</b> (9 g, 37.3 mmol), phenylsilane (11.9 g, 113.1 mmol), cyclopentanone (10.0 mL, 113.1 mmol), and dibutyltin dichloride (17.0 g, 56.0 mmol) in THF (100 mL) were stirred at rt for 10 h. The reaction was monitored by TLC. Upon completion, the solvent was evaporated, diluted with water, and extracted with dichloromethane (3 Ã 100 mL). The combined organic layers were washed with brine, dried over sodium sulfate, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 25% EtOAc/60â90 Â°C petroleum ether) gave compound <b>70</b> (10.4 g, 33.6 mmol, 90% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 9.68 (s, 1H), 6.92 (d, <i>J</i> = 1.9 Hz, 1H), 6.88 (dd, <i>J</i> = 8.2, 2.0 Hz, 1H), 6.69 (d, <i>J</i> = 8.2 Hz, 1H), 4.10 (q, <i>J</i> = 6.8 Hz, 1H), 3.88â3.75 (m, 1H), 2.08â1.94 (m, 2H), 1.78â1.55 (m, 6H), 1.14 (d, <i>J</i> = 6.8 Hz, 3H); MS (ESI) [M + H]<sup>+</sup>: 309.03; retention time 3.05 min, >95% pure.</div></div><div id="sec4_2_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> (<i>R</i>)-6-Bromo-4-cyclopentyl-1,3-dimethyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>71</b>)</h4><div class="NLM_p last">To a solution of compound <b>70</b> (10.4 g, 33.6 mmol) in anhydrous DMF (50 mL) was added NaH (1.6 g, 67.3 mmol) at 0 Â°C, the mixture was stirred at 0 Â°C for 30 min, and then iodomethane (3.1 mL, 50.4 mmol) was added and stirred at room temperature for another 2 h. The reaction was monitored by TLC. Upon completion, the reaction mixture was acidified with 1 N aq. HCl to pH 7â8, diluted with water, and extracted with EtOAc (3 Ã 200 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 25% EtOAc/60â90 Â°C petroleum ether) gave compound <b>71</b> (10 g, 30.9 mmol, 92% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 6.97 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 6.92 (d, <i>J</i> = 2.1 Hz, 1H), 6.78 (d, <i>J</i> = 8.5 Hz, 1H), 4.17 (q, <i>J</i> = 6.8 Hz, 1H), 3.81â3.72 (m, 1H), 3.33 (s, 3H), 2.08â1.96 (m, 2H), 1.84â1.74 (m, 1H), 1.72â1.58 (m, 5H), 1.05 (d, <i>J</i> = 6.8 Hz, 3H); retention time 3.06 min, >95% pure.</div></div><div id="sec4_2_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> (<i>R</i>)-4-Cyclopentyl-1,3-dimethyl-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>72</b>)</h4><div class="NLM_p last">To a solution of compound <b>71</b> (10 g, 31.0 mmol) and AcOK (6.1 g, 61.9 mmol) in DMSO (20 mL) was added bis(pinacolato)diboron (8.7 g, 34.1 mmol), the mixture was bubbled with N<sub>2</sub> for 5 min, then Pd(dppf)<sub>2</sub>Cl<sub>2</sub>Â·CH<sub>2</sub>Cl<sub>2</sub> (1.3 g, 1.6 mmol) was added, and the mixture was further bubbled with N<sub>2</sub> for 5 min. Then, the mixture was heated to 80 Â°C overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with EtOAc (3 Ã 100 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 15% EtOAc/60â90 Â°C petroleum ether) gave compound <b>72</b> (9.5 g, 25.7 mmol, 83% yield) as a light yellow liquid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.36 (d, <i>J</i> = 8.3 Hz, 1H), 7.29 (s, 1H), 6.95 (d, <i>J</i> = 7.9 Hz, 1H), 4.17 (q, <i>J</i> = 6.8 Hz, 1H), 3.94â3.85 (m, 1H), 3.37 (s, 3H), 2.09â1.98 (m, 2H), 1.83â1.74 (m, 1H), 1.73â1.58 (m, 5H), 1.35 (s, 12H), 1.03 (d, <i>J</i> = 6.9 Hz, 3H); MS (ESI) [M + H]<sup>+</sup>: 371.14; retention time 3.07 min, >95% pure.</div></div><div id="sec4_2_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Ethyl(<i>R</i>)-5-(4-cyclopentyl-1,3-dimethyl-2-oxo-1,2,3,4-tetrahydroquinoxalin-6-yl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazole-3-carboxylate (<b>73</b>)</h4><div class="NLM_p last">To a solution of compound <b>72</b> (1 g, 2.7 mmol) and NaHCO<sub>3</sub> (0.5 g, 5.4 mmol) in THF (10 mL) and H<sub>2</sub>O (3 mL) was added ethyl 5-bromo-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazole-3-carboxylate (1.2 g, 3.8 mmol), the mixture was bubbled with N<sub>2</sub> for 5 min, then Pd(dppf)<sub>2</sub>Cl<sub>2</sub>Â·CH<sub>2</sub>Cl<sub>2</sub> (0.2 g, 0.3 mmol) was added, and the mixture was further bubbled with N<sub>2</sub> for 5 min. Then, the mixture was heated to 80 Â°C overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was diluted with water and extracted with EtOAc (3 Ã 50 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure. Purification by silica gel column chromatography (gradient elution, gradient 0 to 20% EtOAc/60â90 Â°C petroleum ether) gave compound <b>73</b> (0.8 g, 1.7 mmol, 63% yield) as a light yellow liquid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.29 (d, <i>J</i> = 8.4 Hz, 2H), 7.24 (d, <i>J</i> = 8.3 Hz, 2H), 7.20â7.16 (m, 1H), 6.90 (d, <i>J</i> = 8.4 Hz, 1H), 6.85 (s, 1H), 4.53 (q, <i>J</i> = 7.3 Hz, 2H), 3.52â3.42 (m, 1H), 3.35 (s, 3H), 2.40 (s, 3H), 1.84â1.77 (m, 1H), 1.74â1.64 (m, 3H), 1.61â1.48 (m, 5H), 1.45 (t, <i>J</i> = 7.1 Hz, 3H), 0.97 (d, <i>J</i> = 6.8 Hz, 3H); MS (ESI) [M + H]<sup>+</sup>: 474.35; retention time 3.88 min, >99% pure.</div></div><div id="sec4_2_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> (<i>R</i>)-5-(4-Cyclopentyl-1,3-dimethyl-2-oxo-1,2,3,4-tetrahydroquinoxalin-6-yl)-1-(<i>p</i>-tolyl)-1<i>H</i>-1,2,4-triazole-3-carboxylic acid (<b>78</b>)</h4><div class="NLM_p last">To a solution of compound <b>73</b> (0.8 g, 1.7 mmol) in THF (10 mL) and H<sub>2</sub>O (5 mL) was added LiOH (0.3 g, 6.8 mmol), and the mixture was stirred at rt overnight. The reaction was monitored by TLC. Upon completion, the reaction mixture was acidified with 1 N aq. HCl to pH 2â3, diluted with water, and extracted with EtOAc (3 Ã 20 mL). The combined organic fractions were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated by evaporation under reduced pressure to give compound <b>78</b> (0.7 g, 1.6 mmol, 94% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.81 (s, 1H), 7.30 (d, <i>J</i> = 8.2 Hz, 2H), 7.26â7.19 (m, 3H), 6.93 (d, <i>J</i> = 8.2 Hz, 1H), 6.84 (s, 1H), 4.19 (q, <i>J</i> = 6.7 Hz, 1H), 3.50â3.40 (m, 1H), 3.35 (s, 3H), 2.39 (s, 3H), 1.84â1.74 (m, 1H), 1.71â1.62 (m, 1H), 1.61â1.42 (m, 5H), 1.36â1.24 (m, 1H), 0.98 (d, <i>J</i> = 6.8 Hz, 3H); MS (ESI) [M + H]<sup>+</sup>: 446.30; retention time 3.27 min, >99% pure.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Protein Expression</h3><div class="NLM_p last">The BRD4 (I) protein expression followed the protocol of Filippakopoulos et al.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Colonies from freshly transformed plasmid DNA in <i>E. coli</i> BL21(DE3)-codon plus-RIL cells were grown overnight at 37 Â°C in 50 mL of Terrific Broth medium with 50 Î¼g/mL kanamycin and 34 Î¼g/mL chloramphenicol (start-up culture). Then, the start-up culture was diluted 100-fold in 1 L of fresh TB medium, and cells were grown at 37 Â°C to an optical density of â¼0.8 at OD600 before the temperature was decreased to 16 Â°C. When the system equilibrated at 16 Â°C, the optical density was â¼1.2 at OD600, and protein expression was induced overnight at 16 Â°C with 0.2 mmol isopropyl-Î²-<span class="smallcaps smallerCapital">d</span>-thiogalactopyranoside (IPTG). The bacteria were harvested by centrifugation (4000 Ã <i>g</i> for 20 min at 4 Â°C) and were frozen at â80 Â°C as pellets for storage. Cells expressing His6-tagged proteins were re-suspended in lysis buffer (50 mmol of HEPES, pH 7.5 at 25 Â°C, 500 mmol of NaCl, 10 mmol of imidazole, 5% glycerol with freshly added 0.5 mmol of TCEP (Tris(2-carboxyethyl)phosphine hydrochloride), and 1 mmol of PMSF (phenylmethanesulfonyl fluoride)) and lysed with a JN 3000 PLUS high pressure homogenizer (JNBIO, Guangzhou, China) at 4 Â°C. The lysate was cleared by centrifugation (12,000 Ã <i>g</i> for 1 h at 4 Â°C) and was applied to a nickelânitrilotriacetic acid agarose column. The column was washed once with 50 mL of wash buffer containing 30 mmol of imidazole. The protein was eluted using a step elution of imidazole in elution buffer (100â250 mmol of imidazole in 50 mmol of HEPES, pH 7.5 at 25 Â°C, 500 mmol of NaCl, 5% glycerol). All fractions were collected and monitored by SDS-polyacrylamide gel electrophoresis (Bio-Rad Criterion Precast Gels, 4â12% BisâTris, 1.0 mm, from Bio-Rad, CA). After the addition of 1 mmol of dithiothreitol (DTT), the eluted protein was treated overnight at 4 Â°C with tobacco etch virus (TEV) protease to remove the His6 tag. The protein was concentrated and further purified with size exclusion chromatography on a Superdex 75 16/60 HiLoad gel filtration column. Samples were monitored by SDS-polyacrylamide gel electrophoresis and concentrated to 8â10 mg/mL in the gel filtration buffer (10 mmol of HEPES pH 7.5, 500 mmol of NaCl, 1 mmol of DTT) and were used for protein binding assay and crystallization.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Crystallization and Data Collection</h3><div class="NLM_p">Aliquots of the purified proteins were set up for crystallization using the vapor diffusion method. The complex structure of BRD4 (I), and the ligand was grown at 18 Â°C in 1 Î¼L of protein with an equal volume of reservoir solution containing 1.8â3.8 M sodium formate and 10% glycerol. Crystals grew to diffracting quality within 1â3 weeks in all cases.</div><div class="NLM_p last">Data were collected at 100 K on beam line BL17U at the Shanghai Synchrotron Radiation Facility (SSRF) (Shanghai, China) for the co-crystallized structures. The data were processed with the HKL2000<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> software packages, and the structures were then solved by molecular replacement using the CCP4 program MOLREP.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The search model used for the crystals was the BRD4-JQ1 complex structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a>). The structures were refined using the CCP4 program REFMAC5 combined with the simulated annealing protocol implemented in the program PHENIX.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> With the aid of the program Coot,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> compound, water molecules, and others were fitted into the initial <i>F</i><sub>o</sub>â<i>F</i><sub>c</sub> maps.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Fluorescence Anisotropy Binding Assay</h3><div class="NLM_p">The binding of compounds to BRD4 was assessed using a fluorescence anisotropy binding assay. The fluorescent ligand was prepared by attaching a fluorescent fragment (fluorescein isothiocyanate isomer I, 5-FITC) to (+)-JQ1. Generally, the method involves incubating the bromodomain protein BRD4, fluorescent ligand, and a variable amount of the test compound together to reach thermodynamic equilibrium under conditions such that in the absence of the test compound, the fluorescent ligand is significantly (> 50%) bound and in the presence of a sufficient concentration of a potent inhibitor, the anisotropy of the unbound fluorescent ligand is measurably different from the bound value.</div><div class="NLM_p last">In detail, all components were dissolved in buffer of a composition of 50 mmol of HEPES pH 7.4, 150 mmol of NaCl, and 0.5 mmol of CHAPS with final concentrations of BRD4 (I) of 10 nM and fluorescent ligand of 5 nM. Various amounts of the test compound or DMSO vehicle (2 â° final) in a Corning 384 well Black low volume plate (CLS3575) were added to this reaction mixture and equilibrated in the dark for 4 h at room temperature. Fluorescence anisotropy was read on a BioTek Synergy2 multimode microplate reader (Î»<sub>ex</sub> = 485 nm, Î»<sub>em</sub> = 530 nm; dichroism, â505 nM).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> In Vitro Metabolic Stability Study</h3><div class="NLM_p">Microsomes (human microsome, Xenotech, Lot No. H0610; mouse microsome, Xenotech, Lot No. M1000) (0.5 mg/mL) were preincubated with 1 Î¼M test compounds for 5 min at 37 Â°C in 0.1 M phosphate buffer (pH 7.4) with 1 mmol of EDTA and 5 mmol of MgCl<sub>2</sub>. The reactions were initiated by adding prewarmed cofactors (1 mmol of NADPH). After 0, 5, 10, and 30 min incubations at 37 Â°C, the reactions were stopped by adding an equal volume of cold acetonitrile. The samples were vortexed for 10 min and then centrifuged at 15,000 rpm for 10 min. Supernatants were analyzed by LC/MS/MS for the amount of parent compound remaining, and the corresponding loss of parent compound was also determined by LC/MS/MS.</div><div class="NLM_p">The CYP enzymatic activities were characterized based on their probe reactions: CYP3A4 (midazolam or testosterone), CYP2D6 (dextromethorphan), CYP2C9 (diclofenac), CYP1A2 (phenacetin), and CYP2C19 (mephenytoin). Incubation mixtures were prepared in a total volume of 100 Î¼L as follows: 0.2 mg/mL microsomes (human microsome, Xenotech, Lot No. H0610), 1 mmol of NADPH, 100 mmol of phosphate buffer (pH 7.4), a probe substrate cocktail (10 Î¼M midazolam, 100 Î¼M testosterone, 10 Î¼M dextromethophan, 20 Î¼M diclofenac, 100 Î¼M phenacetin, and 100 Î¼M Mephenytoin), and 10 Î¼M test compounds, a positive control cocktail (10 Î¼M ketoconazole, 10 Î¼M quinidine, 100 Î¼M sulfaphenazole, 10 Î¼M naphthoflavone, and 1000 Î¼M tranylcypromine), or negative control (PBS). The final concentration of organic reagent in incubation mixtures was less than 1% v/v. There was a 5 min preincubation period at 37 Â°C before the reaction was initiated by adding an NADPH generating system. Reactions were conducted for 20 min for CYPs. For each probe drug, the percentage of metabolite conversion was less than 20% of the substrate added. The inhibition rate was calculated as (the formation of the metabolite of probe substrates with 10 Î¼M test compound)/(the formation of the metabolite of probe substrates with PBS) Ã 100%.</div><div class="NLM_p last">The CYP enzymatic activities were characterized based on their probe reactions: CYP3A4 (midazolam or testosterone), CYP2D6 (dextromethorphan), CYP2C9 (diclofenac), CYP1A2 (phenacetin), and CYP2C19 (<i>S</i>-(+)-mephenytoin). Incubation mixtures were prepared in a total volume of 200 Î¼L as follows: 0.2 mg/mL microsomes (human microsome, Xenotech, Lot No. H0610), 1 mmol of NADPH, 100 mmol of phosphate buffer (pH 7.4), 10 Î¼M test compounds, a positive control cocktail (10 Î¼M troleandomycin, 10 Î¼M paroxetine, 10 Î¼M tienilic acid, 10 Î¼M furafylline), or negative control (PRO). The final concentration of organic reagent in incubation mixtures was less than 1% v/v. There was a 0, 5, 10, and 30 min preincubation period at 37 Â°C before the reaction was initiated by adding an NADPH and probe substrate cocktail (5 Î¼M midazolam, 50 Î¼M testosterone, 5 Î¼M dextromethophan, 10 Î¼M diclofenac, 50 Î¼M phenacetin, and 50 Î¼M <i>S</i>-(+)-mephenytoin) generating a system. Reactions were conducted for 10 min for CYPs. The CYP2C19 enzyme was made alone, and the concentration of positive control cocktail <i>S</i>-(+)-fluoxetine was 100 Î¼M. For each probe drug, the percentage of metabolite conversion was less than 20% of the substrate added. The inhibition rate was calculated as (the formation of the metabolite of probe substrates with 10 Î¼M test compound)/(the formation of the metabolite of probe substrates with PRO) Ã 100%.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> BROMOscan Assay</h3><div class="NLM_p last">T7 phage strains displaying bromodomains were grown in parallel in 24-well blocks in an <i>E. coli</i> host derived from the BL21 strain. <i>E. coli</i> were grown to the log phase, infected with the T7 phage from a frozen stock (multiplicity of infection = 0.4), and incubated with shaking at 32 Â°C until lysis (90â150 min). The lysates were centrifuged (5000 Ã <i>g</i>) and filtered (0.2 Î¼m) to remove cell debris. Streptavidin-coated magnetic beads were treated with biotinylated small molecule or acetylated peptide ligands for 30 min at room temperature to generate affinity resins for bromodomain assays. The ligand-attached beads were blocked with excess biotin and washed with blocking buffer (SeaBlock (Pierce), 1% BSA, 0.05% Tween 20, 1 mM DTT) to remove unbound ligands and to reduce non-specific phage binding. Binding reactions were assembled by combining bromodomains, ligand-attached affinity beads, and test compounds in 1Ã binding buffer (16% SeaBlock, 0.32Ã PBS, 0.02%BSA, 0.04% Tween 20, 0.004% sodium azide, 7.9 mM DTT). Compound <b>19</b> was prepared as 1000Ã stocks in 100% DMSO and subsequently diluted to 1:25 in monoethylene glycol (MEG). Compound <b>19</b> was then diluted directly into the assays such that the final concentrations of DMSO and MEG were 0.1 and 2.4%, respectively. All reactions were performed in polypropylene 384-well plates in a final volume of 0.02 mL. The assay plates were incubated at room temperature with shaking for 1 h, and the affinity beads were washed with wash buffer (1Ã PBS, 0.05% Tween20). The beads were then re-suspended in elution buffer (1Ã PBS, 0.05% Tween 20, 2 Î¼M non-biotinylated affinity ligand) and incubated at room temperature with shaking for 30 min. The bromodomain concentration in the eluates was measured by qPCR. Compound <b>19</b> was screened at the 1 Î¼M concentration, and results for primary screen binding interactions are reported as â% Ctrlâ where lower numbers indicate stronger hits in the matrix.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> PLK1 Kinase Assay: Zâ²-LYTE Kinase Assay</h3><div class="NLM_p last">Kinase reactions were performed with assay buffer (50 mM HEPES pH 7.5, 1 mM EGTA, 10 mM MgCl<sub>2</sub>, 0.01% Brij-35); the PLK1 protein concentration was set to 10 nM, the substrate peptide and ATP concentration were set to 2 mM Ser/Thr 16 Peptide and 13 mM ATP, and the reaction was carried out for â¼60 min at 23 Â°C. The detection equipment is Envision (PerkinElmer # 2104).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> Cell Culture</h3><div class="NLM_p last">The Ty-82 cell line was purchased from the Japanese Collection of Research Bioresources Cell Bank (Osaka, Japan). MV4-11, MM.1S, KG-1, RKO, Hep3B, NCI-H522, MDA-MB-468, HT-29, AGS, BT-549, SKOV3, MDA-MB-436, HuTu-80 and A549 cell lines were obtained from the American Type Culture Collection (ATCC; Manassas, VA, USA). NCI-H1299 and SW620 cell lines were purchased from the Cell Bank of the Chinese Academy of Sciences Type Culture Collection (Shanghai, China). All cell culture conditions follow the supplierâs recommendation.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> Cell Viability Assays</h3><div class="NLM_p last">Different cells were cultured in the corresponding medium conditions recommended by the suppliers. Different cells were seeded onto 96-well plates at a suitable density in a volume of 100 Î¼L medium. After incubation overnight, compounds dissolved in DMSO stock solutions were thawed at room temperature and diluted to the desired concentrations with normal saline. The compounds were added to the assay plate and after 72 h of incubation, the IC<sub>50</sub> was measured with the Cell Counting Kit-8 (CCK-8) assay (Dojindo, Japan) for suspended cells or sulforhodamine B (SRB) (Sigma; MO, USA) for adherent cells.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> Western Blotting</h3><div class="NLM_p last">MV4-11 and RKO cells were plated into 6-well plates and incubated for 24 h. Then, cells were harvested after 24 h of treatment with compounds at the indicated doses. Western blotting was performed using a previously described method.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> Real-Time Quantitative PCR</h3><div class="NLM_p last">MV4-11 and RKO cells were seeded into 6-well plates and incubated for 24 h. Compounds were added at the indicated doses and treated for 24 h. Then, total RNA was extracted with TRIzol (Life Technologies; CA, USA) and turned into cDNA with PrimeScript RTMaster mix (Perfect Real Time) (RR036A) (TaKaRa; Kusatsu, Shiga, Japan). Real-time quantitative PCR were performed with a SYBR Premix Ex Taq (Tli RNaseH Plus) (Takara). The primers used were as follows <i>c-Myc</i>: forward 5â²-GCTGCCAGGACCCGCTTCTC-3â², reverse 5â²-ACGTTGAGGGGCATCGTCGC-3â²; <i>BCL-2</i>: forward 5â²-CCTGTGGATGACTGAGTACCTG-3â², reverse 5â²-CAGAGGCCGCATGCTGGG-3â²; Î²<i>-actin</i>: forward 5â²-ATCGTGCGTGACATTAAGGAGAAG-3â², reverse 5â²-AGGAAGGAAGGCTGGAA GAGTG-3â².</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> Cell Cycle Assay</h3><div class="NLM_p last">RKO cells were seeded into 6-well plates and incubated for 24 h. Then, treated cells with compounds at the indicated concentrations for 24 h. Cells were digested, collected, and washed. Next, cells were fixed with 70% ethanol overnight. Samples were digested with RNAase (Beyotime; Shanghai, China) and then treated with propidium iodide (Beyotime; Shanghai, China). Finally, samples were detected by flow cytometry and analyzed with ModFit LT software.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> Apoptosis Assay</h3><div class="NLM_p last">MV4-11 cells were plated into 96-well plates at 10,000 cells/well and incubated for 24 h. Compounds were added and treated for 24 h. Then, apoptosis was measured by a RealTime-Glo Annexin V apoptosis and necrosis assay (Promega; Madison, WI, USA) according to the manufacturerâs protocol. The apoptotic signal refers to the Annexin V signal corrected by the number of cells.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> hERG Assay</h3><div class="NLM_p last">The Qpatch automatic patch clamp method was used to measure the hERG inhibition of compound <b>19</b>. To prepare the compound, we diluted the compound mother liquor (10 Î¼L) with extracellular fluid (4990 Î¼L). Then, we continuously diluted it three times in an extracellular fluid containing 0.2% DMSO to give the final concentration to be tested. The highest concentration of the compound was 40 Î¼M, followed by 40, 13.33, 4.44, 1.48, 0.49, and 0.16 Î¼M, respectively. The highest concentration of positive compound cisapride was 1 Î¼M, followed by 0.333, 0.111, 0.037, 0.012, and 0.004 Î¼M, respectively. The final test concentration of DMSO was not more than 0.2%, and this concentration has no effect on the hERG potassium channel.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> In Vivo Study</h3><div class="NLM_p last">All the animal experiments were performed according to the Institutional Ethical Guidelines of Animal Care.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> In Vivo PK Study</h3><div class="NLM_p last">Compounds were dissolved in 1% DMSO/0.5% HPMC to a concentration of 1 mg/mL and were given to ICR mice (male, 18â22 g, n = 3) by gavage administration. Blood samples were collected 0.25, 0.5, 1, 2, 4, 8, and 24 h after administration (anticoagulant: EDTA-Na<sub>2</sub>). Solvent (100 Î¼L) of methanol/acetonitrile (1:1, v/v) with an internal standard was added to 10 Î¼L of plasma and vortexed thoroughly. It was centrifuged for 5 min, then 20 Î¼L of the supernatant was mixed with 20 Î¼L of water for analysis. Samples were analyzed using a Xevo TQ-S triple quadrupole mass spectrometer (Waters, USA). An ACQUITY UPLC BEH C18 (1.7 Î¼m, 2.0 mm Ã 50 mm, Waters, USA) was used for the analysis. Gradient elution was applied consisting of 5 mM ammonium acetate aqueous solution containing 0.1% formic acid and acetonitrile containing 0.1% formic acid. After analyzing the concentrations of these compounds, the value of AUClast, AUCINF_obs, and MRTINF_obs was calculated from timeâconcentration curves in each animal using Phoenix WinNonlin (CERTARA, USA). <i>C</i><sub>max</sub> was determined as the maximum plasma concentration, and <i>T</i><sub>max</sub> was the time to reach the maximum concentration.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> In Vivo Pharmacological Study</h3><div class="NLM_p">Four to five week-old female SCID nude mice were obtained from Shanghai Lingchang Biotechnology Co., Ltd. (Shanghai, China) and were acclimated 1 week prior to tumor cell inoculation. A total of 5 Ã 10<sup>6</sup> human multiple myeloma cells (MM.1S) were injected subcutaneously. Thirteen days after inoculation, mice with established xenografts were randomized into different groups for daily treatments of I-BET762 (From Pharmacodia) at 20 mg/kg PO and the vehicle and compound <b>7</b> at 20 mg/kg PO, respectively (vehicle, <i>n</i> = 12; other groups, <i>n</i> = 6).</div><div class="NLM_p last">Six week-old female Balb/c nude mice were obtained from Beijing HFK Bioscience Co., Ltd. (Beijing, China) and were acclimated 1 week prior to tumor cell inoculation. A total of 8 Ã 10<sup>6</sup> human multiple myeloma cells (MV4-11) were injected subcutaneously. Thirteen days after inoculation, mice with established xenografts were randomized into different groups for daily treatments of OTX-015 (From Pharmacodia) at 50 and 100 mg/kg PO and the vehicle and compound <b>19</b> at 10, 25, and 50 mg/kg PO, respectively (vehicle, <i>n</i> = 12; other groups, <i>n</i> = 6). The maximum width (<i>X</i>) and length (<i>Y</i>) of the subcutaneous xenograft were measured with a caliper twice a week, and the volume (<i>V</i>) was calculated using the formula: <i>V</i> = (<i>X</i><sup>2</sup><i>Y</i>)/2. Then, relative tumor volume (RTV) was calculated as RTV = <i>V</i><sub>t</sub>/<i>V</i><sub>0</sub>; the growth inhibition rate (GI) was calculated as GI = [1 â (<i>V</i><sub>t,T</sub> â <i>V</i><sub>0,T</sub>)/(<i>V</i><sub>t,C</sub> â <i>V</i><sub>0,C</sub>)] Ã 100% where <i>V</i><sub>0,T</sub> and <i>V</i><sub>0,C</sub> are the tumor volume at the beginning of the treatment and that of the control group and <i>V</i><sub>t,T</sub> and <i>V</i><sub>t,C</sub> are the tumor volume at the end of the treatment and that of the control group. The animal body weight was also measured at the same time.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i103"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01094" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b01094" class="ext-link">10.1021/acs.jmedchem.9b01094</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Crystallography information for the complex structures, in vitro PK profiles, in vivo data of compound <b>7</b>, solubility data of compounds <b>7</b> and <b>19</b>, in vivo PK properties of compounds <b>15</b> and <b>20â23</b>, bromodomain and kinase selectivity data, ITC binding affinity data of <b>19</b>, hERG inhibition data of <b>19,</b> and spectra data of representative compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Table of molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf">jm9b01094_si_001.pdf (1.3 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_002.csv">jm9b01094_si_002.csv (2.42 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01094" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88870" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88870" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying-Qing Wang</span> - <span class="hlFld-Affiliation affiliation">Division of Anti-tumor Pharmacology,
State Key Laboratory of Drug Research,Â 
     and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#eb929a9c8a858cab98828686c58a88c58885"><span class="__cf_email__" data-cfemail="90e9e1e7f1fef7d0e3f9fdfdbef1f3bef3fe">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ze-Hong Miao</span> - <span class="hlFld-Affiliation affiliation">Division of Anti-tumor Pharmacology,
State Key Laboratory of Drug Research,Â 
     and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7e041613171f113e0d171313501f1d501d10"><span class="__cf_email__" data-cfemail="611b090c08000e2112080c0c4f00024f020f">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bing Xiong</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry,Â 
     and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9776-8136" title="Orcid link">http://orcid.org/0000-0001-9776-8136</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#22405a4b4d4c4562514b4f4f0c43410c414c"><span class="__cf_email__" data-cfemail="b5d7cddcdadbd2f5c6dcd8d89bd4d69bd6db">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianping Hu</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry,Â 
     and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chang-Qing Tian</span> - <span class="hlFld-Affiliation affiliation">Division of Anti-tumor Pharmacology,
State Key Laboratory of Drug Research,Â 
     and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohammadali Soleimani Damaneh</span> - <span class="hlFld-Affiliation affiliation">Division of Anti-tumor Pharmacology,
State Key Laboratory of Drug Research,Â 
     and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanlian Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danyan Cao</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaikai Lv</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry,Â 
     and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, NO.19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ting Yu</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tao Meng</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danqi Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lin Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shan-Shan Song</span> - <span class="hlFld-Affiliation affiliation">Division of Anti-tumor Pharmacology,
State Key Laboratory of Drug Research and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xia-Juan Huan</span> - <span class="hlFld-Affiliation affiliation">Division of Anti-tumor Pharmacology,
State Key Laboratory of Drug Research and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lihuai Qin</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Department of Pharmacological
Sciences, Tisch Cancer Institute, Icahn
School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jingkang Shen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and , Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.H. and C.T. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i105">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37332" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37332" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful for financial supports from the National Science & Technology Major Project âKey New Drug Creation and Manufacturing Programâ of China (grant no. 2018ZX09711002-011-018) and the National Natural Science Foundation of China (grant no. 81330076).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS USED</td></tr><tr><td class="NLM_term">ALK</td><td class="NLM_def"><p class="first last">anaplastic lymphoma kinase</p></td></tr><tr><td class="NLM_term">BCPs</td><td class="NLM_def"><p class="first last">bromodomain-containing proteins</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal</p></td></tr><tr><td class="NLM_term">BETi</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal inhibitor</p></td></tr><tr><td class="NLM_term">BRD4-BD1</td><td class="NLM_def"><p class="first last">first bromodomain of BRD4</p></td></tr><tr><td class="NLM_term">FA</td><td class="NLM_def"><p class="first last">fluorescence anisotropy</p></td></tr><tr><td class="NLM_term">GI</td><td class="NLM_def"><p class="first last">growth inhibition</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetylated lysine</p></td></tr><tr><td class="NLM_term">PLK1</td><td class="NLM_def"><p class="first last">polo-like kinase 1</p></td></tr><tr><td class="NLM_term">P-TEFb</td><td class="NLM_def"><p class="first last">positive transcription elongation factor B</p></td></tr><tr><td class="NLM_term">RT-qPCR</td><td class="NLM_def"><p class="first last">real-time quantitative PCR</p></td></tr><tr><td class="NLM_term">SARs</td><td class="NLM_def"><p class="first last">structureâactivity relationships</p></td></tr><tr><td class="NLM_term">SSRF</td><td class="NLM_def"><p class="first last">Shanghai Synchrotron Radiation Facility</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i107">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62627" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62627" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 50 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">384</span>â <span class="NLM_lpage">400</span>, <span class="refDoi">Â DOI: 10.1038/nrd3674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery&doi=10.1038%2Fnrd3674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lgCH32qpD4TlA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400%26doi%3D10.1038%2Fnrd3674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">465</span>â <span class="NLM_lpage">477</span>, <span class="refDoi">Â DOI: 10.1038/nrc3256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer&doi=10.1038%2Fnrc3256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel Î±-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0lgBYM5cO0HvDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477%26doi%3D10.1038%2Fnrc3256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>â <span class="NLM_lpage">231</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize Îµ-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lgBYM5cO0HvDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, N. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alsarraj, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukes, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Officewala, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, K. W.</span></span> <span> </span><span class="NLM_article-title">Bromodomain 4 activation predicts breast cancer survival</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">6380</span>â <span class="NLM_lpage">6385</span>, <span class="refDoi">Â DOI: 10.1073/pnas.0710331105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1073%2Fpnas.0710331105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=18427120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsFyhtb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=6380-6385&author=N.+P.+S.+Crawfordauthor=J.+Alsarrajauthor=L.+Lukesauthor=R.+C.+Walkerauthor=J.+S.+Officewalaauthor=H.+H.+Yangauthor=M.+P.+Leeauthor=K.+Ozatoauthor=K.+W.+Hunter&title=Bromodomain+4+activation+predicts+breast+cancer+survival&doi=10.1073%2Fpnas.0710331105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain 4 activation predicts breast cancer survival</span></div><div class="casAuthors">Crawford, Nigel P. S.; Alsarraj, Jude; Lukes, Luanne; Walker, Renard C.; Officewala, Jennifer S.; Yang, Howard H.; Lee, Maxwell P.; Ozato, Keiko; Hunter, Kent W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6380-6385</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Previous work identified the Rap1 GTPase-activating protein Sipa1 as a germ-line-encoded metastasis modifier.  The bromodomain protein Brd4 phys. interacts with and modulates the enzymic activity of Sipa1.  In vitro anal. of a highly metastatic mouse mammary tumor cell line ectopically expressing Brd4 demonstrates significant redn. of invasiveness without altering intrinsic growth rate.  However, a dramatic redn. of tumor growth and pulmonary metastasis was obsd. after s.c. implantation into mice, implying that activation of Brd4 may somehow be manipulating response to tumor microenvironment in the in vivo setting.  Further in vitro anal. shows that Brd4 modulates extracellular matrix gene expression, a class of genes frequently present in metastasis-predictive gene signatures.  Microarray anal. of the mammary tumor cell lines identified a Brd4 activation signature that robustly predicted progression and/or survival in multiple human breast cancer datasets analyzed on different microarray platforms.  Intriguingly, the Brd4 signature also almost perfectly matches a mol. classifier of low-grade tumors.  Taken together, these data suggest that dysregulation of Brd4-assocd. pathways may play an important role in breast cancer progression and underlies multiple common prognostic signatures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIT8F-snY4rrVg90H21EOLACvtfcHk0lhEVb4XRjFj8g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsFyhtb0%253D&md5=bdd4b3676ebbe0a39da15d020f2994e9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0710331105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0710331105%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DN.%2BP.%2BS.%26aulast%3DAlsarraj%26aufirst%3DJ.%26aulast%3DLukes%26aufirst%3DL.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26aulast%3DOfficewala%26aufirst%3DJ.%2BS.%26aulast%3DYang%26aufirst%3DH.%2BH.%26aulast%3DLee%26aufirst%3DM.%2BP.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DHunter%26aufirst%3DK.%2BW.%26atitle%3DBromodomain%25204%2520activation%2520predicts%2520breast%2520cancer%2520survival%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D6380%26epage%3D6385%26doi%3D10.1073%2Fpnas.0710331105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyoshi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubonishi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grier, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Atayde, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, J. A.</span></span> <span> </span><span class="NLM_article-title">BRD4-NUT fusion oncogene: a novel mechanism in aggressive carcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">304</span>â <span class="NLM_lpage">307</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=12543779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1Klug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=304-307&author=C.+A.+Frenchauthor=I.+Miyoshiauthor=I.+Kubonishiauthor=H.+E.+Grierauthor=A.+R.+Perez-Ataydeauthor=J.+A.+Fletcher&title=BRD4-NUT+fusion+oncogene%3A+a+novel+mechanism+in+aggressive+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4-NUT Fusion Oncogene: A Novel Mechanism in Aggressive Carcinoma</span></div><div class="casAuthors">French, Christopher A.; Miyoshi, Isao; Kubonishi, Ichiro; Grier, Holcombe E.; Perez-Atayde, Antonio R.; Fletcher, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">304-307</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The poorly differentiated carcinoma with t(15;19)(q13,p13.1) is characterized by its highly aggressive, invariably lethal clin. course.  The chromosome 19 translocation breakpoint targets the BRD4 double bromodomain-contg. gene, which functions in regulation of cell cycle progression.  Herein the authors demonstrate that BRD4 is fused with nearly the entire transcript of the novel 15q13 gene, NUT (nuclear protein in testis), forming a 6.4-kb fusion oncogene, BRD4-NUT.  NUT, like BRD4, is predicted to encode a nuclear protein but, unlike the ubiquitous BRD4 transcript, is expressed only in testis.  These findings establish a model to elucidate the oncogenic consequences of unscheduled NUT expression and altered BRD4 function.  Very few fusion oncogenes have been identified in epithelial tumors, and BRD4-NUT is the first fusion oncogene mechanism identified in a highly lethal form of carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTa0K_kBu2pLVg90H21EOLACvtfcHk0lhEVb4XRjFj8g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt1Klug%253D%253D&md5=e927ea5874b2013c7467f981e10d7efd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DMiyoshi%26aufirst%3DI.%26aulast%3DKubonishi%26aufirst%3DI.%26aulast%3DGrier%26aufirst%3DH.%2BE.%26aulast%3DPerez-Atayde%26aufirst%3DA.%2BR.%26aulast%3DFletcher%26aufirst%3DJ.%2BA.%26atitle%3DBRD4-NUT%2520fusion%2520oncogene%253A%2520a%2520novel%2520mechanism%2520in%2520aggressive%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D304%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Bromodomains as therapeutic targets in cancer</span>. <i>Briefings Funct. Genomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">219</span>â <span class="NLM_lpage">230</span>, <span class="refDoi">Â DOI: 10.1093/bfgp/elt007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1093%2Fbfgp%2Felt007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=23543289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXos12hu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=219-230&author=I.+Barbieriauthor=E.+Cannizzaroauthor=M.+A.+Dawson&title=Bromodomains+as+therapeutic+targets+in+cancer&doi=10.1093%2Fbfgp%2Felt007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains as therapeutic targets in cancer</span></div><div class="casAuthors">Barbieri, Isaia; Cannizzaro, Ester; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Functional Genomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">219-230</span>CODEN:
                <span class="NLM_cas:coden">BFGRAR</span>;
        ISSN:<span class="NLM_cas:issn">2041-2649</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The malleability of the epigenome has long been recognized as a unique opportunity for therapeutic intervention.  Interest in targeting components of the epigenetic machinery for therapeutic gain had initially been aimed at chromatin modifying enzymes.  However, advances in medicinal chem. have now made it possible to exploit protein-protein interactions at the chromatin interface.  Bromodomains (BRD) are a conserved motif used by a large no. of chromatin-assocd. proteins to recognize and bind acetylated histone tails.  Small mols. with high specificity for the Bromodomain and Extra Terminal family of proteins (BRD2, BRD3, BRD4 and BRDT) have recently been shown to have remarkable pre-clin. efficacy in various malignancies.  These findings have provided the impetus for exploring other BRD proteins as novel targets in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnjEcNXUf4I7Vg90H21EOLACvtfcHk0lgjP6PtbwlEag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXos12hu74%253D&md5=0a41c0491042217395ffafbc28a25ed5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fbfgp%2Felt007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbfgp%252Felt007%26sid%3Dliteratum%253Aachs%26aulast%3DBarbieri%26aufirst%3DI.%26aulast%3DCannizzaro%26aufirst%3DE.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DBromodomains%2520as%2520therapeutic%2520targets%2520in%2520cancer%26jtitle%3DBriefings%2520Funct.%2520Genomics%26date%3D2013%26volume%3D12%26spage%3D219%26epage%3D230%26doi%3D10.1093%2Fbfgp%2Felt007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">PÃ©rez-Salvia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteller, M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibitors and cancer therapy: from structures to applications</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">323</span>â <span class="NLM_lpage">339</span>, <span class="refDoi">Â DOI: 10.1080/15592294.2016.1265710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1080%2F15592294.2016.1265710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=27911230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A280%3ADC%252BC2sjivVaksQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=323-339&author=M.+P%C3%A9rez-Salviaauthor=M.+Esteller&title=Bromodomain+inhibitors+and+cancer+therapy%3A+from+structures+to+applications&doi=10.1080%2F15592294.2016.1265710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain inhibitors and cancer therapy: From structures to applications</span></div><div class="casAuthors">Perez-Salvia Montserrat; Esteller Manel; Esteller Manel; Esteller Manel</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">323-339</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Aberrations in the epigenetic landscape are a hallmark of cancer.  Alterations in enzymes that are "writers," "erasers," or "readers" of histone modification marks are common.  Bromodomains are "readers" that bind acetylated lysines in histone tails.  Their most important function is the regulation of gene transcription by the recruitment of different molecular partners.  Moreover, proteins containing bromodomains are also epigenetic regulators, although little is known about the specific function of these domains.  In recent years, there has been increasing interest in developing small molecules that can target specific bromodomains.  First, this has helped clarify biological functions of bromodomain-containing proteins.  Secondly, it opens a new front for combatting cancer.  In this review we will describe the structures and mechanisms associated with Bromodomain and Extra-Terminal motif (BET) inhibitors and non-BET inhibitors, their current status of development, and their promising role as anti-cancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCPfARVC-uxp6KlWarZTAzfW6udTcc2ebhmQHvgHOGobntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjivVaksQ%253D%253D&md5=3ce4e19d8177ddc1705fd0d58e5fb020</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1080%2F15592294.2016.1265710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15592294.2016.1265710%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Salvia%26aufirst%3DM.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DBromodomain%2520inhibitors%2520and%2520cancer%2520therapy%253A%2520from%2520structures%2520to%2520applications%26jtitle%3DEpigenetics%26date%3D2017%26volume%3D12%26spage%3D323%26epage%3D339%26doi%3D10.1080%2F15592294.2016.1265710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meslamani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">The bromodomain: from epigenome reader to druggable target</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1839</i></span>,  <span class="NLM_fpage">676</span>â <span class="NLM_lpage">685</span>, <span class="refDoi">Â DOI: 10.1016/j.bbagrm.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.bbagrm.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=24686119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=676-685&author=R.+Sanchezauthor=J.+Meslamaniauthor=M.+M.+Zhou&title=The+bromodomain%3A+from+epigenome+reader+to+druggable+target&doi=10.1016%2Fj.bbagrm.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain: From epigenome reader to druggable target</span></div><div class="casAuthors">Sanchez, Roberto; Meslamani, Jamel; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">676-685</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a fundamental post-translational modification that plays an important role in the control of gene transcription in chromatin in an ordered fashion.  The bromodomain, the conserved structural module present in transcription-assocd. proteins, functions exclusively to recognize acetyl-lysine on histones and non-histone proteins.  The structural analyses of bromodomains' recognition of lysine-acetylated peptides derived from histones and cellular proteins provide detailed insights into the differences and unifying features of biol. ligand binding selectivity by the bromodomains.  Newly developed small-mol. inhibitors targeting bromodomain proteins further highlight the functional importance of bromodomain/acetyl-lysine binding as a key mechanism in orchestrating mol. interactions and regulation in chromatin biol. and gene transcription.  These new studies argue that modulating bromodomain/acetyl-lysine interactions with small-mol. chems. offer new opportunities to control gene expression in a wide array of human diseases including cancer and inflammation.  This article is part of a Special Issue entitled: Mol. mechanisms of histone modification function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO3Au7jfK0ibVg90H21EOLACvtfcHk0lgGtdI8nH4j7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D&md5=95ec21cf13231d9e63bce9b3ccdd6d6f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DMeslamani%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DThe%2520bromodomain%253A%2520from%2520epigenome%2520reader%2520to%2520druggable%2520target%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2014%26volume%3D1839%26spage%3D676%26epage%3D685%26doi%3D10.1016%2Fj.bbagrm.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor Brd4 and transcriptional regulation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">13141</span>â <span class="NLM_lpage">13145</span>, <span class="refDoi">Â DOI: 10.1074/jbc.R700001200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.+Y.+Wuauthor=C.+M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+Brd4+and+transcriptional+regulation&doi=10.1074%2Fjbc.R700001200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0lgGtdI8nH4j7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520Brd4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145%26doi%3D10.1074%2Fjbc.R700001200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>â <span class="NLM_lpage">1073</span>, <span class="refDoi">Â DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lhDcI1nyNDeDA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noel, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ooike, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugahara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daibata, M.</span></span> <span> </span><span class="NLM_article-title">Development of the BET bromodomain inhibitor OTX015</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_fpage">C244</span>â <span class="NLM_lpage">C244</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.TARG-13-C244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1158%2F1535-7163.TARG-13-C244" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=C244-C244&author=J.+K.+Noelauthor=K.+Iwataauthor=S.+Ooikeauthor=K.+Sugaharaauthor=H.+Nakamuraauthor=M.+Daibata&title=Development+of+the+BET+bromodomain+inhibitor+OTX015&doi=10.1158%2F1535-7163.TARG-13-C244"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-13-C244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-13-C244%26sid%3Dliteratum%253Aachs%26aulast%3DNoel%26aufirst%3DJ.%2BK.%26aulast%3DIwata%26aufirst%3DK.%26aulast%3DOoike%26aufirst%3DS.%26aulast%3DSugahara%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DH.%26aulast%3DDaibata%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520the%2520BET%2520bromodomain%2520inhibitor%2520OTX015%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2013%26spage%3DC244%26epage%3DC244%26doi%3D10.1158%2F1535-7163.TARG-13-C244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Roca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de La Motte Rouge, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uro-Coste, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Braud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelosi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span> <span> </span><span class="NLM_article-title">Clinical response of carcinomas harboring the BRD4-NUT oncoprotein to the targeted bromodomain inhibitor OTX015/MK-8628</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">492</span>â <span class="NLM_lpage">500</span>, <span class="refDoi">Â DOI: 10.1158/2159-8290.CD-15-1335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1158%2F2159-8290.CD-15-1335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=26976114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlOqsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=492-500&author=A.+Stathisauthor=E.+Zuccaauthor=M.+Bekraddaauthor=C.+Gomez-Rocaauthor=J.-P.+Delordauthor=T.+de+La+Motte+Rougeauthor=E.+Uro-Costeauthor=F.+de+Braudauthor=G.+Pelosiauthor=C.+A.+French&title=Clinical+response+of+carcinomas+harboring+the+BRD4-NUT+oncoprotein+to+the+targeted+bromodomain+inhibitor+OTX015%2FMK-8628&doi=10.1158%2F2159-8290.CD-15-1335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Response of Carcinomas Harboring the BRD4-NUT Oncoprotein to the Targeted Bromodomain Inhibitor OTX015/MK-8628</span></div><div class="casAuthors">Stathis, Anastasios; Zucca, Emanuele; Bekradda, Mohamed; Gomez-Roca, Carlos; Delord, Jean-Pierre; de La Motte Rouge, Thibault; Uro-Coste, Emmanuelle; de Braud, Filippo; Pelosi, Giuseppe; French, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">492-500</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The antineoplastic, prodifferentiative effects of bromodomain and extra-terminal (BET) bromodomain (BRD) inhibitors were initially discovered in NUT midline carcinoma (NMC), an aggressive subtype of squamous cancer driven by the BRD4-NUT fusion oncoprotein.  BRD4-NUT blocks differentiation and maintains tumor growth through a potent chromatin-modifying mechanism.  OTX015/MK-8628, a novel oral BET inhibitor, targets BRD2/3/4/T with preclin. activity in NMC and several other tumor types and is currently in clin. development.  Antitumor activity was evaluated in four patients with advanced-stage NMC with confirmed BRD4-NUT fusions who were treated with 80 mg OTX015/MK-8628 once daily in a compassionate-use context.  Two patients responded rapidly with tumor regression and symptomatic relief, and a third had meaningful disease stabilization with a minor metabolic response.  The main side effects were mild to moderate gastrointestinal toxicity and fatigue, and reversible grade 3 thrombocytopenia.  This is the first proof-of-concept evidence of clin. activity of a BRD inhibitor in targeting BRD4-NUT.  Significance: We present the first clin. proof-of-concept that targeting BRD4-NUT with a BET inhibitor results in impressive and rapid antitumor activity in NMC.  It offers strong potential for future clin. application in this rare patient population as either a single agent or in combination with other agents.  Cancer Discov; 6(5); 492-500. Â©2016 AACR.  This article is highlighted in the In This Issue feature, p.  461.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGqXhyj-x5pLVg90H21EOLACvtfcHk0lhDcI1nyNDeDA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlOqsrs%253D&md5=73f6d61463bc6db72178d70771544787</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1335%26sid%3Dliteratum%253Aachs%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DGomez-Roca%26aufirst%3DC.%26aulast%3DDelord%26aufirst%3DJ.-P.%26aulast%3Dde%2BLa%2BMotte%2BRouge%26aufirst%3DT.%26aulast%3DUro-Coste%26aufirst%3DE.%26aulast%3Dde%2BBraud%26aufirst%3DF.%26aulast%3DPelosi%26aufirst%3DG.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26atitle%3DClinical%2520response%2520of%2520carcinomas%2520harboring%2520the%2520BRD4-NUT%2520oncoprotein%2520to%2520the%2520targeted%2520bromodomain%2520inhibitor%2520OTX015%252FMK-8628%26jtitle%3DCancer%2520Discovery%26date%3D2016%26volume%3D6%26spage%3D492%26epage%3D500%26doi%3D10.1158%2F2159-8290.CD-15-1335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>â <span class="NLM_lpage">1123</span>, <span class="refDoi">Â DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0liK2Ek8fXVqOg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ClÃ©ment, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnathan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">NicodÃ©me, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>â <span class="NLM_lpage">7515</span>, <span class="refDoi">Â DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=M.+Barnathanauthor=J.-M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.-W.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A9me&title=Discovery+of+epigenetic+regulator+I-BET762%3A+lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0lgyNmyCFZkQag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A9me%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CÃ´tÃ©, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâMeara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span> <span> </span><span class="NLM_article-title">Identification of a benzoisoxazoloazepine inhibitor (CPI-0610) of the bromodomain and extra-terminal (BET) family as a candidate for human clinical trials</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1330</span>â <span class="NLM_lpage">1339</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01882</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01882" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1330-1339&author=B.+K.+Albrechtauthor=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=R.+G.+Vaswaniauthor=A.+C%C3%B4t%C3%A9author=Y.+Leblancauthor=C.+G.+Nasveschukauthor=S.+Bellonauthor=L.+Bergeronauthor=R.+Campbellauthor=N.+Cantoneauthor=M.+R.+Cooperauthor=R.+T.+Cummingsauthor=H.+Jayaramauthor=S.+Joshiauthor=J.+A.+Mertzauthor=A.+Neissauthor=E.+Normantauthor=M.+O%E2%80%99Mearaauthor=E.+Pardoauthor=F.+Poyauthor=P.+Sandyauthor=J.+Supkoauthor=R.+J.+Simsauthor=J.-C.+Harmangeauthor=A.+M.+Taylorauthor=J.+E.+Audia&title=Identification+of+a+benzoisoxazoloazepine+inhibitor+%28CPI-0610%29+of+the+bromodomain+and+extra-terminal+%28BET%29+family+as+a+candidate+for+human+clinical+trials&doi=10.1021%2Facs.jmedchem.5b01882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Benzoisoxazoloazepine Inhibitor (CPI-0610) of the Bromodomain and Extra-Terminal (BET) Family as a Candidate for Human Clinical Trials</span></div><div class="casAuthors">Albrecht, Brian K.; Gehling, Victor S.; Hewitt, Michael C.; Vaswani, Rishi G.; Cote, Alexandre; Leblanc, Yves; Nasveschuk, Christopher G.; Bellon, Steve; Bergeron, Louise; Campbell, Robert; Cantone, Nico; Cooper, Michael R.; Cummings, Richard T.; Jayaram, Hariharan; Joshi, Shivangi; Mertz, Jennifer A.; Neiss, Adrianne; Normant, Emmanuel; O'Meara, Michael; Pardo, Eneida; Poy, Florence; Sandy, Peter; Supko, Jeffrey; Sims, Robert J.; Harmange, Jean-Christophe; Taylor, Alexander M.; Audia, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1330-1339</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, inhibition of the interaction between the bromodomain and extra-terminal domain (BET) family of chromatin adaptors and acetyl-lysine residues on chromatin has emerged as a promising approach to regulate the expression of important disease-relevant genes, including MYC, BCL-2, and NF-ÎºB.  Here we describe the identification and characterization of a potent and selective benzoisoxazoloazepine BET bromodomain inhibitor that attenuates BET-dependent gene expression in vivo, demonstrates antitumor efficacy in an MV-4-11 mouse xenograft model, and is currently undergoing human clin. trials for hematol. malignancies (CPI-0610).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp40qjnYl4EZLVg90H21EOLACvtfcHk0lgyNmyCFZkQag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D&md5=5e1c5aab26aa58844a9f6670a7bbd78b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01882%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DCooper%26aufirst%3DM.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DNormant%26aufirst%3DE.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSupko%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26atitle%3DIdentification%2520of%2520a%2520benzoisoxazoloazepine%2520inhibitor%2520%2528CPI-0610%2529%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520family%2520as%2520a%2520candidate%2520for%2520human%2520clinical%2520trials%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1330%26epage%3D1339%26doi%3D10.1021%2Facs.jmedchem.5b01882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donawho, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlhapp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Preclinical characterization of BET family bromodomain inhibitor ABBV-075 suggests combination therapeutic strategies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2976</span>â <span class="NLM_lpage">2989</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-16-1793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1158%2F0008-5472.CAN-16-1793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=28416490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Siurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2976-2989&author=M.+H.+Buiauthor=X.+Linauthor=D.+H.+Albertauthor=L.+Liauthor=L.+T.+Lamauthor=E.+J.+Faivreauthor=S.+E.+Warderauthor=X.+Huangauthor=D.+Wilcoxauthor=C.+K.+Donawhoauthor=G.+S.+Sheppardauthor=L.+Wangauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+Hasvoldauthor=T.+Uzielauthor=X.+Luauthor=F.+Kohlhappauthor=G.+Fangauthor=S.+W.+Elmoreauthor=S.+H.+Rosenbergauthor=K.+F.+McDanielauthor=W.+M.+Katiauthor=Y.+Shen&title=Preclinical+characterization+of+BET+family+bromodomain+inhibitor+ABBV-075+suggests+combination+therapeutic+strategies&doi=10.1158%2F0008-5472.CAN-16-1793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical Characterization of BET Family Bromodomain Inhibitor ABBV-075 Suggests Combination Therapeutic Strategies</span></div><div class="casAuthors">Bui, Mai H.; Lin, Xiaoyu; Albert, Daniel H.; Li, Leiming; Lam, Lloyd T.; Faivre, Emily J.; Warder, Scott E.; Huang, Xiaoli; Wilcox, Denise; Donawho, Cherrie K.; Sheppard, George S.; Wang, Le; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa; Uziel, Tamar; Lu, Xin; Kohlhapp, Fred; Fang, Guowei; Elmore, Steven W.; Rosenberg, Saul H.; McDaniel, Keith F.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2976-2989</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">ABBV-075 is a potent and selective BET family bromodomain inhibitor that recently entered phase I clin. trials.  Comprehensive preclin. characterization of ABBV-075 demonstrated broad activity across cell lines and tumor models, representing a variety of hematol. malignancies and solid tumor indications.  In most cancer cell lines derived from solid tumors, ABBV-075 triggers prominent G1 cell-cycle arrest without extensive apoptosis.  In this study, we show that ABBV-075 efficiently triggers apoptosis in acute myeloid leukemia (AML), non-Hodgkin lymphoma, and multiple myeloma cells.  Apoptosis induced by ABBV-075 was mediated in part by modulation of the intrinsic apoptotic pathway, exhibiting synergy with the BCL-2 inhibitor venetoclax in preclin. models of AML.  In germinal center diffuse large B-cell lymphoma, BCL-2 levels or venetoclax sensitivity predicted the apoptotic response to ABBV-075 treatment.  In vivo combination studies uncovered surprising benefits of low doses of ABBV-075 coupled with bortezomib and azacitidine treatment, despite the lack of in vitro synergy between ABBV-075 and these agents.  The in vitro/in vivo activities of ABBV-075 described here may serve as a useful ref. to guide the development of ABBV-075 and other BET family inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6P1blFwkIvrVg90H21EOLACvtfcHk0lhp9O0_-Fbv1Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Siurw%253D&md5=d9cc4abc44cfd9c0e07383f908592c2e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-1793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-1793%26sid%3Dliteratum%253Aachs%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DDonawho%26aufirst%3DC.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DKohlhapp%26aufirst%3DF.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DPreclinical%2520characterization%2520of%2520BET%2520family%2520bromodomain%2520inhibitor%2520ABBV-075%2520suggests%2520combination%2520therapeutic%2520strategies%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2976%26epage%3D2989%26doi%3D10.1158%2F0008-5472.CAN-16-1793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-(4-(2,4-difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfon amide (ABBV-075/Mivebresib), a potent and orally available bromodomain and extraterminal domain (BET) family bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8369</span>â <span class="NLM_lpage">8384</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00746</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00746" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8369-8384&author=K.+F.+McDanielauthor=L.+Wangauthor=T.+Soltwedelauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=R.+A.+Manteiauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=M.+H.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=L.+Liauthor=X.+Linauthor=R.+Wangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=J.+J.+Shenauthor=R.+P.+Edaljiauthor=C.+C.+Sunauthor=R.+Martinauthor=W.+Gaoauthor=S.+Wongauthor=G.+Fangauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Kati&title=Discovery+of+N-%284-%282%2C4-difluorophenoxy%29-3-%286-methyl-7-oxo-6%2C7-dihydro-1H-pyrrolo%5B2%2C3-c%5Dpyridin-4-yl%29phenyl%29ethanesulfon+amide+%28ABBV-075%2FMivebresib%29%2C+a+potent+and+orally+available+bromodomain+and+extraterminal+domain+%28BET%29+family+bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.7b00746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span></div><div class="casAuthors">McDaniel, Keith F.; Wang, Le; Soltwedel, Todd; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Mantei, Robert A.; Pratt, John K.; Sheppard, George S.; Bui, Mai H.; Faivre, Emily J.; Huang, Xiaoli; Li, Leiming; Lin, Xiaoyu; Wang, Rongqi; Warder, Scott E.; Wilcox, Denise; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Park, Chang H.; Hutchins, Charles W.; Shen, Jianwei J.; Edalji, Rohinton P.; Sun, Chaohong C.; Martin, Ruth; Gao, Wenqing; Wong, Shekman; Fang, Guowei; Elmore, Steven W.; Shen, Yu; Kati, Warren M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8369-8384</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of bromodomain and extraterminal domain (BET) bromodomain inhibitors and their examn. in clin. studies, particularly in oncol. settings, has garnered substantial recent interest.  An effort to generate novel BET bromodomain inhibitors with excellent potency and drug metab. and pharmacokinetics (DMPK) properties was initiated based upon elaboration of a simple pyridone core.  Efforts to develop a bidentate interaction with a crit. asparagine residue resulted in the incorporation of a pyrrolopyridone core, which improved potency by 9-19-fold.  Addnl. structure-activity relationship (SAR) efforts aimed both at increasing potency and improving pharmacokinetic properties led to the discovery of the clin. candidate I (ABBV-075/mivebresib), which demonstrates excellent potency in biochem. and cellular assays, advantageous exposures and half-life both in animal models and in humans, and in vivo efficacy in mouse models of cancer progression and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom-8BkipNknbVg90H21EOLACvtfcHk0lipeJl8G4F6aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF&md5=16ea9e9ade8a22a6e99aeae16fd1bf9c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DShen%26aufirst%3DJ.%2BJ.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25284-%25282%252C4-difluorophenoxy%2529-3-%25286-methyl-7-oxo-6%252C7-dihydro-1H-pyrrolo%255B2%252C3-c%255Dpyridin-4-yl%2529phenyl%2529ethanesulfon%2520amide%2520%2528ABBV-075%252FMivebresib%2529%252C%2520a%2520potent%2520and%2520orally%2520available%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520family%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8369%26epage%3D8384%26doi%3D10.1021%2Facs.jmedchem.7b00746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, Y.</span></span> <span> </span><span class="NLM_article-title">The bromodomain and extra-terminal domain (BET) family: functional anatomy of BET paralogous proteins</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1849</span>, <span class="refDoi">Â DOI: 10.3390/ijms17111849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.3390%2Fijms17111849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXot1Ohsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=1849&author=Y.+Taniguchi&title=The+bromodomain+and+extra-terminal+domain+%28BET%29+family%3A+functional+anatomy+of+BET+paralogous+proteins&doi=10.3390%2Fijms17111849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain and extra-terminal domain (BET) family: functional anatomy of BET paralogous proteins</span></div><div class="casAuthors">Taniguchi, Yasushi</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1849/1-1849/24</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The Bromodomain and Extra-Terminal Domain (BET) family of proteins is characterized by the presence of two tandem bromodomains and an extra-terminal domain.  The mammalian BET family of proteins comprises BRD2, BRD3, BRD4, and BRDT, which are encoded by paralogous genes that may have been generated by repeated duplication of an ancestral gene during evolution.  Bromodomains that can specifically bind acetylated lysine residues in histones serve as chromatin-targeting modules that decipher the histone acetylation code.  BET proteins play a crucial role in regulating gene transcription through epigenetic interactions between bromodomains and acetylated histones during cellular proliferation and differentiation processes.  On the other hand, BET proteins have been reported to mediate latent viral infection in host cells and be involved in oncogenesis.  Human BRD4 is involved in multiple processes of the DNA virus life cycle, including viral replication, genome maintenance, and gene transcription through interaction with viral proteins.  Aberrant BRD4 expression contributes to carcinogenesis by mediating hyperacetylation of the chromatin contg. the cell proliferation-promoting genes.  BET bromodomain blockade using small-mol. inhibitors gives rise to selective repression of the transcriptional network driven by c-MYC These inhibitors are expected to be potential therapeutic drugs for a wide range of cancers.  This review presents an overview of the basic roles of BET proteins and highlights the pathol. functions of BET and the recent developments in cancer therapy targeting BET proteins in animal models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKHOsfPnlUXbVg90H21EOLACvtfcHk0lipeJl8G4F6aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXot1Ohsg%253D%253D&md5=fe82c03d3e5ffba8cbb0ad51e5fc19e1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3390%2Fijms17111849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms17111849%26sid%3Dliteratum%253Aachs%26aulast%3DTaniguchi%26aufirst%3DY.%26atitle%3DThe%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520family%253A%2520functional%2520anatomy%2520of%2520BET%2520paralogous%2520proteins%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2016%26volume%3D17%26spage%3D1849%26doi%3D10.3390%2Fijms17111849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchrÃ¶der, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaehlcke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisgrove, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SchnÃ¶lzer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, M.</span></span> <span> </span><span class="NLM_article-title">Two-pronged binding with bromodomain-containing protein 4 liberates positive transcription elongation factor b from inactive ribonucleoprotein complexes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">1090</span>â <span class="NLM_lpage">1099</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M111.282855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1074%2Fjbc.M111.282855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=22084242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A280%3ADC%252BC387is1answ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=1090-1099&author=S.+Schr%C3%B6derauthor=S.+Choauthor=L.+Zengauthor=Q.+Zhangauthor=K.+Kaehlckeauthor=L.+Makauthor=J.+Lauauthor=D.+Bisgroveauthor=M.+Schn%C3%B6lzerauthor=E.+Verdinauthor=M.-M.+Zhouauthor=M.+Ott&title=Two-pronged+binding+with+bromodomain-containing+protein+4+liberates+positive+transcription+elongation+factor+b+from+inactive+ribonucleoprotein+complexes&doi=10.1074%2Fjbc.M111.282855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Two-pronged binding with bromodomain-containing protein 4 liberates positive transcription elongation factor b from inactive ribonucleoprotein complexes</span></div><div class="casAuthors">Schroder Sebastian; Cho Sungyoo; Zeng Lei; Zhang Qiang; Kaehlcke Katrin; Mak Lily; Lau Joann; Bisgrove Dwayne; Schnolzer Martina; Verdin Eric; Zhou Ming-Ming; Ott Melanie</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of biological chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1090-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The positive transcription elongation factor b (P-TEFb) exists in two forms in cells as follows: an inactive form where the core components cyclin T1 and CDK9 are incorporated in the 7SK small nuclear ribonucleoprotein complex containing the inhibitory molecule HEXIM1, and an active form, part of which associates with the bromodomain-containing protein BRD4.  Here, we define a novel interaction between P-TEFb and BRD4 involving tri-acetylated cyclin T1 (acK380, acK386, and acK309) and the second bromodomain in BRD4.  This interaction is observed with the short splice variant of BRD4 (amino acids 1-722) lacking a previously defined C-terminal P-TEFb-interacting domain (PID).  Notably, P-TEFb complexes associated with short BRD4 contain HEXIM1 and 7SK snRNA, implicating the PID in the liberation of P-TEFb from the 7SK small nuclear ribonucleoprotein complex (7SK snPNP).  Overexpression of the PID alone in cells dissociates HEXIM1 and 7SK snRNA from P-TEFb, but it is not sufficient to activate P-TEFb-dependent transcription of the HIV LTR.  Our data support a model where two BRD4 domains, the second bromodomain and the PID, bind P-TEFb and are required for full transcriptional activation of P-TEFb response genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSTv2tSJrzcEK69lAbTj9rGfW6udTcc2eY67V3DqoPvDLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC387is1answ%253D%253D&md5=85209cc1f667ed47b97e626dcb2156ec</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.282855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.282855%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6der%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DS.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKaehlcke%26aufirst%3DK.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DBisgrove%26aufirst%3DD.%26aulast%3DSchn%25C3%25B6lzer%26aufirst%3DM.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DTwo-pronged%2520binding%2520with%2520bromodomain-containing%2520protein%25204%2520liberates%2520positive%2520transcription%2520elongation%2520factor%2520b%2520from%2520inactive%2520ribonucleoprotein%2520complexes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D1090%26epage%3D1099%26doi%3D10.1074%2Fjbc.M111.282855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delmore, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issa, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilpatrick, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinzel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffernan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsagel, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">904</span>â <span class="NLM_lpage">917</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+C.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0ljVAr6DCtFexw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%2BC.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B. P.</span></span> <span> </span><span class="NLM_article-title">Disrupting the interaction of BRD4 with diacetylated Twist suppresses tumorigenesis in basal-like breast cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">210</span>â <span class="NLM_lpage">225</span>, <span class="refDoi">Â DOI: 10.1016/j.ccr.2014.01.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.ccr.2014.01.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=24525235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=210-225&author=J.+Shiauthor=Y.+Wangauthor=L.+Zengauthor=Y.+Wuauthor=J.+Dengauthor=Q.+Zhangauthor=Y.+Linauthor=J.+Liauthor=T.+Kangauthor=M.+Taoauthor=E.+Rusinovaauthor=G.+Zhangauthor=C.+Wangauthor=H.+Zhuauthor=J.+Yaoauthor=Y.+X.+Zengauthor=B.+M.+Eversauthor=M.+M.+Zhouauthor=B.+P.+Zhou&title=Disrupting+the+interaction+of+BRD4+with+diacetylated+Twist+suppresses+tumorigenesis+in+basal-like+breast+cancer&doi=10.1016%2Fj.ccr.2014.01.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting the Interaction of BRD4 with Diacetylated Twist Suppresses Tumorigenesis in Basal-like Breast Cancer</span></div><div class="casAuthors">Shi, Jian; Wang, Yifan; Zeng, Lei; Wu, Yadi; Deng, Jiong; Zhang, Qiang; Lin, Yiwei; Li, Junlin; Kang, Tiebang; Tao, Min; Rusinova, Elena; Zhang, Guangtao; Wang, Chi; Zhu, Haining; Yao, Jun; Zeng, Yi-Xin; Evers, B. Mark; Zhou, Ming-Ming; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">210-225</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Twist is a key transcription activator of epithelial-mesenchymal transition (EMT).  It remains unclear how Twist induces gene expression.  Here we report a mechanism by which Twist recruits BRD4 to direct WNT5A expression in basal-like breast cancer (BLBC).  Twist contains a "histone H4-mimic" GK-X-GK motif that is diacetylated by Tip60.  The diacetylated Twist binds the second bromodomain of BRD4, whose first bromodomain interacts with acetylated H4, thereby constructing an activated Twist/BRD4/P-TEFb/RNA-Pol II complex at the WNT5A promoter and enhancer.  Pharmacol. inhibition of the Twist-BRD4 assocn. reduced WNT5A expression and suppressed invasion, cancer stem cell (CSC)-like properties, and tumorigenicity of BLBC cells.  Our study indicates that the interaction with BRD4 is crit. for the oncogenic function of Twist in BLBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovIzriI8A6ZrVg90H21EOLACvtfcHk0lihtGqZSWuhwQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D&md5=d207cbc1abb187a53d807452b4f10d8f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.01.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.01.028%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DTao%26aufirst%3DM.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DY.%2BX.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DDisrupting%2520the%2520interaction%2520of%2520BRD4%2520with%2520diacetylated%2520Twist%2520suppresses%2520tumorigenesis%2520in%2520basal-like%2520breast%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D210%26epage%3D225%26doi%3D10.1016%2Fj.ccr.2014.01.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, K.</span></span> <span> </span><span class="NLM_article-title">Brd4 binds to active enhancers to control cell identity gene induction in adipogenesis and myogenesis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2217</span>, <span class="refDoi">Â DOI: 10.1038/s41467-017-02403-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fs41467-017-02403-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=29263365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A280%3ADC%252BC1MzjslSlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2217&author=J.-E.+Leeauthor=Y.-K.+Parkauthor=S.+Parkauthor=Y.+Jangauthor=N.+Waringauthor=A.+Deyauthor=K.+Ozatoauthor=B.+Laiauthor=W.+Pengauthor=K.+Ge&title=Brd4+binds+to+active+enhancers+to+control+cell+identity+gene+induction+in+adipogenesis+and+myogenesis&doi=10.1038%2Fs41467-017-02403-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 binds to active enhancers to control cell identity gene induction in adipogenesis and myogenesis</span></div><div class="casAuthors">Lee Ji-Eun; Park Young-Kwon; Park Sarah; Jang Younghoon; Waring Nicholas; Lai Binbin; Ge Kai; Waring Nicholas; Lai Binbin; Peng Weiqun; Dey Anup; Ozato Keiko</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2217</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The epigenomic reader Brd4 is an important drug target for cancers.  However, its role in cell differentiation and animal development remains largely unclear.  Using two conditional knockout mouse strains and derived cells, we demonstrate that Brd4 controls cell identity gene induction and is essential for adipogenesis and myogenesis.  Brd4 co-localizes with lineage-determining transcription factors (LDTFs) on active enhancers during differentiation.  LDTFs coordinate with H3K4 mono-methyltransferases MLL3/MLL4 (KMT2C/KMT2D) and H3K27 acetyltransferases CBP/p300 to recruit Brd4 to enhancers activated during differentiation.  Brd4 deletion prevents the enrichment of Mediator and RNA polymerase II transcription machinery, but not that of LDTFs, MLL3/MLL4-mediated H3K4me1, and CBP/p300-mediated H3K27ac, on enhancers.  Consequently, Brd4 deletion prevents enhancer RNA production, cell identity gene induction and cell differentiation.  Interestingly, Brd4 is dispensable for maintaining cell identity genes in differentiated cells.  These findings identify Brd4 as an enhancer epigenomic reader that links active enhancers with cell identity gene induction in differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREMjSSm_naSGZyxfu64m4ffW6udTcc2eaMX43HqANekLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzjslSlsQ%253D%253D&md5=35aa3eb2ed0f9c1f38a6a2d09b438957</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02403-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02403-5%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.-E.%26aulast%3DPark%26aufirst%3DY.-K.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DY.%26aulast%3DWaring%26aufirst%3DN.%26aulast%3DDey%26aufirst%3DA.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DB.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DGe%26aufirst%3DK.%26atitle%3DBrd4%2520binds%2520to%2520active%2520enhancers%2520to%2520control%2520cell%2520identity%2520gene%2520induction%2520in%2520adipogenesis%2520and%2520myogenesis%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D2217%26doi%3D10.1038%2Fs41467-017-02403-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span>; <span class="NLM_string-name">Yang, J.</span>; <span class="NLM_string-name">Zhao, Y.</span>; <span class="NLM_string-name">Ivanciuc, T.</span>; <span class="NLM_string-name">Sun, H.</span>; <span class="NLM_string-name">Garofalo, R. P.</span>; <span class="NLM_string-name">Brasier, A. R.</span></span> <span> </span><span class="NLM_article-title">BRD4 couples NF-ÎºB/RelA with airway inflammation and the IRF-RIG-I amplification loop in respiratory syncytial virus infection</span>.  <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span><b>,</b>  <span class="NLM_volume">91</span>, <span class="refDoi">Â DOI: 10.1128/JVI.00007-17</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1128%2FJVI.00007-17" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2017&author=B.+Tian&author=J.+Yang&author=Y.+Zhao&author=T.+Ivanciuc&author=H.+Sun&author=R.+P.+Garofalo&author=A.+R.+Brasier&title=BRD4+couples+NF-%CE%BAB%2FRelA+with+airway+inflammation+and+the+IRF-RIG-I+amplification+loop+in+respiratory+syncytial+virus+infection&doi=10.1128%2FJVI.00007-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FJVI.00007-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00007-17%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DB.%26atitle%3DBRD4%2520couples%2520NF-%25CE%25BAB%252FRelA%2520with%2520airway%2520inflammation%2520and%2520the%2520IRF-RIG-I%2520amplification%2520loop%2520in%2520respiratory%2520syncytial%2520virus%2520infection%26jtitle%3DJ.%2520Virol.%26date%3D2017%26volume%3D91%26doi%3D10.1128%2FJVI.00007-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hogg, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vervoort, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deswal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cluse, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beavis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darcy, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traunbauer, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadovnik, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shortt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">BET-Bromodomain inhibitors engage the host immune system and regulate expression of the immune checkpoint ligand PD-L1</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2162</span>â <span class="NLM_lpage">2174</span>, <span class="refDoi">Â DOI: 10.1016/j.celrep.2017.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.celrep.2017.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=28249162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Ontb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=2162-2174&author=S.+J.+Hoggauthor=S.+J.+Vervoortauthor=S.+Deswalauthor=C.+J.+Ottauthor=J.+Liauthor=L.+A.+Cluseauthor=P.+A.+Beavisauthor=P.+K.+Darcyauthor=B.+P.+Martinauthor=A.+Spencerauthor=A.+K.+Traunbauerauthor=I.+Sadovnikauthor=K.+Bauerauthor=P.+Valentauthor=J.+E.+Bradnerauthor=J.+Zuberauthor=J.+Shorttauthor=R.+W.+Johnstone&title=BET-Bromodomain+inhibitors+engage+the+host+immune+system+and+regulate+expression+of+the+immune+checkpoint+ligand+PD-L1&doi=10.1016%2Fj.celrep.2017.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">BET-Bromodomain Inhibitors Engage the Host Immune System and Regulate Expression of the Immune Checkpoint Ligand PD-L1</span></div><div class="casAuthors">Hogg, Simon J.; Vervoort, Stephin J.; Deswal, Sumit; Ott, Christopher J.; Li, Jason; Cluse, Leonie A.; Beavis, Paul A.; Darcy, Phillip K.; Martin, Benjamin P.; Spencer, Andrew; Traunbauer, Anna K.; Sadovnik, Irina; Bauer, Karin; Valent, Peter; Bradner, James E.; Zuber, Johannes; Shortt, Jake; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2162-2174</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">BET inhibitors (BETi) target bromodomain-contg. proteins and are currently being evaluated as anti-cancer agents.  We find that maximal therapeutic effects of BETi in a Myc-driven B cell lymphoma model required an intact host immune system.  Genome-wide anal. of the BETi-induced transcriptional response identified the immune checkpoint ligand Cd274 (Pd-l1) as a Myc-independent, BETi target-gene.  BETi directly repressed constitutively expressed and interferon-gamma (IFN-Î³) induced CD274 expression across different human and mouse tumor cell lines and primary patient samples.  Mechanistically, BETi decreased Brd4 occupancy at the Cd274 locus without any change in Myc occupancy, resulting in transcriptional pausing and rapid loss of Cd274 mRNA prodn.  Finally, targeted inhibition of the PD-1/PD-L1 axis by combining anti-PD-1 antibodies and the BETi JQ1 caused synergistic responses in mice bearing Myc-driven lymphomas.  Our data uncover an interaction between BETi and the PD-1/PD-L1 immune-checkpoint and provide mechanistic insight into the transcriptional regulation of CD274.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo06NwFejZMmbVg90H21EOLACvtfcHk0lghAujgS3tn9A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Ontb0%253D&md5=62eac948a07e94c39234ddd7015fd7e8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DHogg%26aufirst%3DS.%2BJ.%26aulast%3DVervoort%26aufirst%3DS.%2BJ.%26aulast%3DDeswal%26aufirst%3DS.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DCluse%26aufirst%3DL.%2BA.%26aulast%3DBeavis%26aufirst%3DP.%2BA.%26aulast%3DDarcy%26aufirst%3DP.%2BK.%26aulast%3DMartin%26aufirst%3DB.%2BP.%26aulast%3DSpencer%26aufirst%3DA.%26aulast%3DTraunbauer%26aufirst%3DA.%2BK.%26aulast%3DSadovnik%26aufirst%3DI.%26aulast%3DBauer%26aufirst%3DK.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShortt%26aufirst%3DJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DBET-Bromodomain%2520inhibitors%2520engage%2520the%2520host%2520immune%2520system%2520and%2520regulate%2520expression%2520of%2520the%2520immune%2520checkpoint%2520ligand%2520PD-L1%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D18%26spage%3D2162%26epage%3D2174%26doi%3D10.1016%2Fj.celrep.2017.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciarrocchi, A.</span></span> <span> </span><span class="NLM_article-title">BRD4 and cancer: going beyond transcriptional regulation</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">164</span>, <span class="refDoi">Â DOI: 10.1186/s12943-018-0915-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1186%2Fs12943-018-0915-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=30466442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlWgu78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=164&author=B.+Donatiauthor=E.+Lorenziniauthor=A.+Ciarrocchi&title=BRD4+and+cancer%3A+going+beyond+transcriptional+regulation&doi=10.1186%2Fs12943-018-0915-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 and Cancer: going beyond transcriptional regulation</span></div><div class="casAuthors">Donati, Benedetta; Lorenzini, Eugenia; Ciarrocchi, Alessia</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">164</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  BRD4, member of the Bromodomain and Extraterminal (BET) protein family, is largely acknowledged in cancer for its role in super-enhancers (SEs) organization and oncogenes expression regulation.  Inhibition of BRD4 shortcuts the communication between SEs and target promoters with a subsequent cell-specific repression of oncogenes to which cancer cells are addicted and cell death.  To date, this is the most credited mechanism of action of BET inhibitors, a class of small mols. targeting BET proteins which are currently in clin. trials in several cancer settings.  However, recent evidence indicates that BRD4 relevance in cancer goes beyond its role in transcription regulation and identifies this protein as a keeper of genome stability.  Indeed, a non-transcriptional role of BRD4 in controlling DNA damage checkpoint activation and repair as well as telomere maintenance has been proposed, throwing new lights into the multiple functions of this protein and opening new perspectives on the use of BETi in cancer.  Here we discuss the current available information on non-canonical, non-transcriptional functions of BRD4 and on their implications in cancer biol.  Integrating this information with the already known BRD4 role in gene expression regulation, we propose a "common" model to explain BRD4 genomic function.  Furthermore, in light of the transversal function of BRD4, we provide new interpretation for the cytotoxic activity of BETi and we discuss new possibilities for a wide and focused employment of these drugs in clin. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbYn0mnQW7ELVg90H21EOLACvtfcHk0lghAujgS3tn9A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlWgu78%253D&md5=2ddfde1363d430aa67e15a52fdc2335e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1186%2Fs12943-018-0915-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-018-0915-9%26sid%3Dliteratum%253Aachs%26aulast%3DDonati%26aufirst%3DB.%26aulast%3DLorenzini%26aufirst%3DE.%26aulast%3DCiarrocchi%26aufirst%3DA.%26atitle%3DBRD4%2520and%2520cancer%253A%2520going%2520beyond%2520transcriptional%2520regulation%26jtitle%3DMol.%2520Cancer%26date%3D2018%26volume%3D17%26spage%3D164%26doi%3D10.1186%2Fs12943-018-0915-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Korb, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucker-Scharff, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">BET protein brd4 activates transcription in neurons and BET inhibitor Jq1 blocks memory in mice</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1464</span>â <span class="NLM_lpage">1473</span>, <span class="refDoi">Â DOI: 10.1038/nn.4095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnn.4095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=26301327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlynsL%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=1464-1473&author=E.+Korbauthor=M.+Herreauthor=I.+Zucker-Scharffauthor=R.+B.+Darnellauthor=C.+D.+Allis&title=BET+protein+brd4+activates+transcription+in+neurons+and+BET+inhibitor+Jq1+blocks+memory+in+mice&doi=10.1038%2Fnn.4095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">BET protein Brd4 activates transcription in neurons and BET inhibitor Jq1 blocks memory in mice</span></div><div class="casAuthors">Korb, Erica; Herre, Margo; Zucker-Scharff, Ilana; Darnell, Robert B.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1464-1473</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Precise regulation of transcription is crucial for the cellular mechanisms underlying memory formation.  However, the link between neuronal stimulation and the proteins that directly interact with histone modifications to activate transcription in neurons remains unclear.  Brd4 is a member of the bromodomain and extra-terminal domain (BET) protein family, which binds acetylated histones and is a crit. regulator of transcription in many cell types, including transcription in response to external cues.  Small mol. BET inhibitors are in clin. trials, yet almost nothing is known about Brd4 function in the brain.  Here we show that Brd4 mediates the transcriptional regulation underlying learning and memory.  The loss of Brd4 function affects crit. synaptic proteins, which results in memory deficits in mice but also decreases seizure susceptibility.  Thus Brd4 provides a crit. link between neuronal activation and the transcriptional responses that occur during memory formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbuMwCm6NDBLVg90H21EOLACvtfcHk0lghAujgS3tn9A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlynsL%252FN&md5=0a09dea39378424cfc003a1e9e40ab9a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnn.4095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.4095%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DE.%26aulast%3DHerre%26aufirst%3DM.%26aulast%3DZucker-Scharff%26aufirst%3DI.%26aulast%3DDarnell%26aufirst%3DR.%2BB.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DBET%2520protein%2520brd4%2520activates%2520transcription%2520in%2520neurons%2520and%2520BET%2520inhibitor%2520Jq1%2520blocks%2520memory%2520in%2520mice%26jtitle%3DNat.%2520Neurosci.%26date%3D2015%26volume%3D18%26spage%3D1464%26epage%3D1473%26doi%3D10.1038%2Fnn.4095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrieu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">Clinical trials for BET inhibitors run ahead of the science</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">45</span>â <span class="NLM_lpage">50</span>, <span class="refDoi">Â DOI: 10.1016/j.ddtec.2016.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.ddtec.2016.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=27769357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=45-50&author=G.+Andrieuauthor=A.+C.+Belkinaauthor=G.+V.+Denis&title=Clinical+trials+for+BET+inhibitors+run+ahead+of+the+science&doi=10.1016%2Fj.ddtec.2016.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical trials for BET inhibitors run ahead of the science</span></div><div class="casAuthors">Andrieu Guillaume; Belkina Anna C; Denis Gerald V</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Several cancer clinical trials for small molecule inhibitors of BET bromodomain proteins have been initiated.  There is enthusiasm for the anti-proliferative effect of inhibiting BRD4, one of the targets of these inhibitors, which is thought to cooperate with MYC, a long-desired target for cancer therapeutics.  However, no current inhibitor is selective for BRD4 among the three somatic BET proteins, which include BRD2 and BRD3; their respective functions are partially overlapping and none are functionally redundant with BRD4.  Each BET protein controls distinct transcriptional pathways that are important for functions beyond cancer cell proliferation, including insulin production, cytokine gene transcription, T cell differentiation, adipogenesis and most seriously, active repression of dangerous latent viruses like HIV.  BET inhibitors have been shown to reactivate HIV in human cells.  Failure to appreciate that at concentrations used, no available BET inhibitor is member-selective, or to develop a sound biological basis to understand the diverse functions of BET proteins before undertaking for these clinical trials is reckless and likely to lead to adverse events.  More mechanistic information from new basic science studies should enable proper focus on the most relevant cancers and define the expected side effect profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEothR-V9Y-Lt7lRqEFUJVfW6udTcc2eZvs-5DXR6M_7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D&md5=5fcc9e96fa590221fb07c5e98ae91d59</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2016.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2016.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DAndrieu%26aufirst%3DG.%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DClinical%2520trials%2520for%2520BET%2520inhibitors%2520run%2520ahead%2520of%2520the%2520science%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2016%26volume%3D19%26spage%3D45%26epage%3D50%26doi%3D10.1016%2Fj.ddtec.2016.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, L.</span></span> <span> </span><span class="NLM_article-title">A phase Ib trial with MK-8628/OTX015, a small molecule inhibitor of bromodomain (BRD) and extra-terminal (BET) proteins, in patients with selected advanced solid tumors</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">S2</span>â <span class="NLM_lpage">S3</span>, <span class="refDoi">Â DOI: 10.1016/S0959-8049(16)32609-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2FS0959-8049%2816%2932609-0" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2016&pages=S2-S3&author=C.+Massardauthor=J.+C.+Soriaauthor=A.+Stathisauthor=J.+P.+Delordauthor=A.+Awadaauthor=S.+Petersauthor=J.+Lewinauthor=M.+Bekraddaauthor=K.+Rezaiauthor=Z.+Zengauthor=H.+Azherauthor=S.+Perezauthor=L.+Siu&title=A+phase+Ib+trial+with+MK-8628%2FOTX015%2C+a+small+molecule+inhibitor+of+bromodomain+%28BRD%29+and+extra-terminal+%28BET%29+proteins%2C+in+patients+with+selected+advanced+solid+tumors&doi=10.1016%2FS0959-8049%2816%2932609-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0959-8049%2816%2932609-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-8049%252816%252932609-0%26sid%3Dliteratum%253Aachs%26aulast%3DMassard%26aufirst%3DC.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DDelord%26aufirst%3DJ.%2BP.%26aulast%3DAwada%26aufirst%3DA.%26aulast%3DPeters%26aufirst%3DS.%26aulast%3DLewin%26aufirst%3DJ.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DAzher%26aufirst%3DH.%26aulast%3DPerez%26aufirst%3DS.%26aulast%3DSiu%26aufirst%3DL.%26atitle%3DA%2520phase%2520Ib%2520trial%2520with%2520MK-8628%252FOTX015%252C%2520a%2520small%2520molecule%2520inhibitor%2520of%2520bromodomain%2520%2528BRD%2529%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%252C%2520in%2520patients%2520with%2520selected%2520advanced%2520solid%2520tumors%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2016%26volume%3D69%26spage%3DS2%26epage%3DS3%26doi%3D10.1016%2FS0959-8049%2816%2932609-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abramson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flinn, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maris, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâMeara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span> <span> </span><span class="NLM_article-title">BET inhibitor CPI-0610 is well tolerated and induces responses in diffuse large B-Cell lymphoma and follicular lymphoma: preliminary analysis of an ongoing phase 1 study</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1491</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1182%2Fblood.V126.23.1491.1491" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2015&pages=1491&author=J.+S.+Abramsonauthor=K.+A.+Blumauthor=I.+W.+Flinnauthor=M.+Gutierrezauthor=A.+Goyauthor=M.+Marisauthor=M.+Cooperauthor=M.+O%E2%80%99Mearaauthor=D.+Borgerauthor=J.+Mertzauthor=R.+J.+Simsauthor=S.+Jeffreyauthor=A.+Younes&title=BET+inhibitor+CPI-0610+is+well+tolerated+and+induces+responses+in+diffuse+large+B-Cell+lymphoma+and+follicular+lymphoma%3A+preliminary+analysis+of+an+ongoing+phase+1+study"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1182%2Fblood.V126.23.1491.1491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V126.23.1491.1491%26sid%3Dliteratum%253Aachs%26aulast%3DAbramson%26aufirst%3DJ.%2BS.%26aulast%3DBlum%26aufirst%3DK.%2BA.%26aulast%3DFlinn%26aufirst%3DI.%2BW.%26aulast%3DGutierrez%26aufirst%3DM.%26aulast%3DGoy%26aufirst%3DA.%26aulast%3DMaris%26aufirst%3DM.%26aulast%3DCooper%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DBorger%26aufirst%3DD.%26aulast%3DMertz%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DJeffrey%26aufirst%3DS.%26aulast%3DYounes%26aufirst%3DA.%26atitle%3DBET%2520inhibitor%2520CPI-0610%2520is%2520well%2520tolerated%2520and%2520induces%2520responses%2520in%2520diffuse%2520large%2520B-Cell%2520lymphoma%2520and%2520follicular%2520lymphoma%253A%2520preliminary%2520analysis%2520of%2520an%2520ongoing%2520phase%25201%2520study%26jtitle%3DBlood%26date%3D2015%26volume%3D126%26spage%3D1491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span>D;, S.
N. H. B. S. <span> </span><span class="NLM_article-title">New Dihydroquinoxalinone Compounds are
Bromodomain-containing Protein 4 Inhibitors</span>. <span class="NLM_patent">2014095775</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=D%3B%2C+S.%0AN.+H.+B.+S.+New+Dihydroquinoxalinone+Compounds+are%0ABromodomain-containing+Protein+4+Inhibitors.+2014095775%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26atitle%3DNew%2520Dihydroquinoxalinone%2520Compounds%2520are%250ABromodomain-containing%2520Protein%25204%2520Inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span>D, S.
N. H. B. S. <span> </span><span class="NLM_article-title">New Dihydropyridopyrazinone Derivatives
Useful as Drug for Treatment of Tumor Diseases</span>. <span class="NLM_patent">2015004075</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=D%2C+S.%0AN.+H.+B.+S.+New+Dihydropyridopyrazinone+Derivatives%0AUseful+as+Drug+for+Treatment+of+Tumor+Diseases.+2015004075%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26atitle%3DNew%2520Dihydropyridopyrazinone%2520Derivatives%250AUseful%2520as%2520Drug%2520for%2520Treatment%2520of%2520Tumor%2520Diseases%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortopassi, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pryde, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">A series of potent CREBBP bromodomain ligands reveals an induced-fit pocket stabilized by a Cation-Ï interaction</span>. <i>Angew. Chem. Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6126</span>â <span class="NLM_lpage">6130</span>, <span class="refDoi">Â DOI: 10.1002/anie.201402750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1002%2Fanie.201402750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6126-6130&author=T.+P.+C.+Rooneyauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=W.+A.+Cortopassiauthor=D.+A.+Hayauthor=S.+Martinauthor=A.+Tumberauthor=C.+M.+Rogersauthor=M.+Philpottauthor=M.+Wangauthor=A.+L.+Thompsonauthor=T.+D.+Heightmanauthor=D.+C.+Prydeauthor=A.+Cookauthor=R.+S.+Patonauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=A+series+of+potent+CREBBP+bromodomain+ligands+reveals+an+induced-fit+pocket+stabilized+by+a+Cation-%CF%80+interaction&doi=10.1002%2Fanie.201402750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A Series of Potent CREBBP Bromodomain Ligands Reveals an Induced-Fit Pocket Stabilized by a Cation-Ï Interaction</span></div><div class="casAuthors">Rooney, Timothy P. C.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Cortopassi, Wilian A.; Hay, Duncan A.; Martin, Sarah; Tumber, Anthony; Rogers, Catherine M.; Philpott, Martin; Wang, Minghua; Thompson, Amber L.; Heightman, Tom D.; Pryde, David C.; Cook, Andrew; Paton, Robert S.; Mueller, Susanne; Knapp, Stefan; Brennan, Paul E.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6126-6130</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Benzoxazinone and dihydroquinoxalinone fragments were employed as novel acetyl lysine mimics in the development of CREBBP bromodomain ligands.  While the benzoxazinone series showed low affinity for the CREBBP bromodomain, expansion of the dihydroquinoxalinone series resulted in the first potent inhibitors, e.g., I, of a bromodomain outside the BET family.  Structural and computational studies revealed that an internal hydrogen bond stabilized the protein-bound conformation of the dihydroquinoxalinone series.  The side chain of this series binds in an induced-fit pocket forming a cation-Ï interaction with R1173 of CREBBP.  Compd. I, the most potent compd., inhibited binding of CREBBP to chromatin in U2OS cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHFgTVUKe7lrVg90H21EOLACvtfcHk0lgs8LIW6RCGmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D&md5=42b12e1fe855d2af02c49f8e1031d39e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201402750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201402750%26sid%3Dliteratum%253Aachs%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DA.%2BL.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DCook%26aufirst%3DA.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DA%2520series%2520of%2520potent%2520CREBBP%2520bromodomain%2520ligands%2520reveals%2520an%2520induced-fit%2520pocket%2520stabilized%2520by%2520a%2520Cation-%25CF%2580%2520interaction%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D6126%26epage%3D6130%26doi%3D10.1002%2Fanie.201402750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damaneh, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Structure-based optimization of a series of selective BET inhibitors containing aniline or indoline groups</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">156</span>â <span class="NLM_lpage">175</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2018.02.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.ejmech.2018.02.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=29525435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktlylsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2018&pages=156-175&author=J.+Huauthor=Y.+Wangauthor=Y.+Liauthor=D.+Caoauthor=L.+Xuauthor=S.+Songauthor=M.+S.+Damanehauthor=J.+Liauthor=Y.+Chenauthor=X.+Wangauthor=L.+Chenauthor=J.+Shenauthor=Z.+Miaoauthor=B.+Xiong&title=Structure-based+optimization+of+a+series+of+selective+BET+inhibitors+containing+aniline+or+indoline+groups&doi=10.1016%2Fj.ejmech.2018.02.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based optimization of a series of selective BET inhibitors containing aniline or indoline groups</span></div><div class="casAuthors">Hu, Jianping; Wang, Yingqing; Li, Yanlian; Cao, Danyan; Xu, Lin; Song, ShanShan; Damaneh, Mohammadali Soleimani; Li, Jian; Chen, Yuelei; Wang, Xin; Chen, Lin; Shen, Jingkang; Miao, Zehong; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">156-175</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recently, several kinase inhibitors were found to also inhibit bromodomains, providing a new strategy for the discovery of bromodomain inhibitors.  Along this line, starting from PLK1-BRD4 dual inhibitor BI-2536, we discovered a new series of dihydroquinoxalin-2(1H)-one with aniline and indoline WPF binders as selective BRD4 inhibitors.  They showed better BRD4-BD1 potency and negligible PLK1 kinase activity comparing with BI-2536.  Addnl., dihydroquinoxalin-2(1H)-ones contg. indoline group showed profound activities in mol. and cellular based assays.  Throughout the study, compds. 9, 28 and 37 showed significant inhibitory activity for c-Myc or PD-L1 protein expression and mRNA transcription both at concn. of 0.2 and 1 Î¼M.  Compd. 9 was found possessing the best balance of binding affinity, in vitro metabolic stability and in vivo pharmacokinetic properties.  Therefore, it was selected for in vivo pharmacol. study.  By using MM.1S cell derived xenograft model, we confirmed compd. 9 showed comparable in vivo tumor inhibition to phase II investigation drug I-BET762, which, together with the novel WPF binder, further indicated the utility of this series of BRD4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovKjmtiqOa9rVg90H21EOLACvtfcHk0lhBJeVHyu-lxw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktlylsLw%253D&md5=5d3ad7e741a140306d42ae2463dd667e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.02.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.02.070%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DDamaneh%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DStructure-based%2520optimization%2520of%2520a%2520series%2520of%2520selective%2520BET%2520inhibitors%2520containing%2520aniline%2520or%2520indoline%2520groups%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D150%26spage%3D156%26epage%3D175%26doi%3D10.1016%2Fj.ejmech.2018.02.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damaneh, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a series of dihydroquinoxalin-2(1H)-ones as selective BET inhibitors from a dual PLK1-BRD4 inhibitor</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">176</span>â <span class="NLM_lpage">195</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2017.05.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.ejmech.2017.05.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=28586718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWmt73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2017&pages=176-195&author=J.+Huauthor=Y.+Wangauthor=Y.+Liauthor=L.+Xuauthor=D.+Caoauthor=S.+Songauthor=M.+S.+Damanehauthor=X.+Wangauthor=T.+Mengauthor=Y.+L.+Chenauthor=J.+Shenauthor=Z.+Miaoauthor=B.+Xiong&title=Discovery+of+a+series+of+dihydroquinoxalin-2%281H%29-ones+as+selective+BET+inhibitors+from+a+dual+PLK1-BRD4+inhibitor&doi=10.1016%2Fj.ejmech.2017.05.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a series of dihydroquinoxalin-2(1H)-ones as selective BET inhibitors from a dual PLK1-BRD4 inhibitor</span></div><div class="casAuthors">Hu, Jianping; Wang, Yingqing; Li, Yanlian; Xu, Lin; Cao, Danyan; Song, Shan Shan; Damaneh, Mohammadali Soleimani; Wang, Xin; Meng, Tao; Chen, Yue-Lei; Shen, Jingkang; Miao, Zehong; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">176-195</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recent years have seen much effort to discover new chemotypes of BRD4 inhibitors.  Some kinase inhibitors are potent bromodomain inhibitors, esp. the PLK1 inhibitor BI-2536 and the JAK2 inhibitor TG101209, which can bind to BRD4 with IC50 values of 0.025 Î¼M and 0.13 Î¼M, resp.  Although the concept of dual inhibition is intriguing, selective BRD4 inhibitors are preferred as they may diminish off-target effects and provide more flexibility in anticancer drug combination therapy.  Inspired by BI-2536, the authors designed and prepd. a series of dihydroquinoxalin-2(1H)-one derivs. as selective bromodomain inhibitors.  The authors found compd. (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one had slightly higher activity than (+)-JQ1 in the fluorescence anisotropy assay and potent antiproliferative cellular activity in the MM.1S cell line.  The authors have successfully solved the cocrystal structure of (R)-4-cyclopropyl-1,3-dimethyl-6-(((S)-1-(p-tolyl)ethyl) amino)-3,4-dihydroquinoxalin-2(1H)-one in complex with BRD4-BD1, providing a solid structural basis for the binding mode of compds. of this series.  Compd. (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one exhibited high selectivity over most non-BET subfamily members and did not show bioactivity towards the PLK1 kinase at 10 or 1 Î¼M.  From in vivo studies, compd. (R)-4-Cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one demonstrated a good PK profile, and the results from in vivo pharmacol. studies clearly showed the efficacy of (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one in the mouse MM.1S xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8GQjGV-VP7LVg90H21EOLACvtfcHk0lhBJeVHyu-lxw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWmt73I&md5=af59e5bd57adeede512b022e3641f4f2</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.05.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.05.049%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DDamaneh%26aufirst%3DM.%2BS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMeng%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%2BL.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520series%2520of%2520dihydroquinoxalin-2%25281H%2529-ones%2520as%2520selective%2520BET%2520inhibitors%2520from%2520a%2520dual%2520PLK1-BRD4%2520inhibitor%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D137%26spage%3D176%26epage%3D195%26doi%3D10.1016%2Fj.ejmech.2017.05.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshioka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fountain, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheenstra, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strovel, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span> <span> </span><span class="NLM_article-title">BRD4 structure-activity relationships of dual PLK1 kinase/BRD4 bromodomain inhibitor BI-2536</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">764</span>â <span class="NLM_lpage">769</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00084</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00084" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=764-769&author=L.+Chenauthor=J.+L.+Yapauthor=M.+Yoshiokaauthor=M.+E.+Lanningauthor=R.+N.+Fountainauthor=M.+Rajeauthor=J.+A.+Scheenstraauthor=J.+W.+Strovelauthor=S.+Fletcher&title=BRD4+structure-activity+relationships+of+dual+PLK1+kinase%2FBRD4+bromodomain+inhibitor+BI-2536&doi=10.1021%2Facsmedchemlett.5b00084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Structure-Activity Relationships of Dual PLK1 Kinase/BRD4 Bromodomain Inhibitor BI-2536</span></div><div class="casAuthors">Chen, Lijia; Yap, Jeremy L.; Yoshioka, Makoto; Lanning, Maryanna E.; Fountain, Rachel N.; Raje, Mithun; Scheenstra, Jacob A.; Strovel, Jeffrey W.; Fletcher, Steven</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">764-769</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused library of analogs of the dual PLK1 kinase/BRD4 bromodomain inhibitor BI-2536 was prepd. and then analyzed for BRD4 and PLK1 inhibitory activities.  Particularly, replacement of the cyclopentyl group with a 3-bromobenzyl moiety afforded the most potent BRD4 inhibitor of the series (39j) with a Ki = 8.7 nM, which was equipotent against PLK1.  The superior affinity of 39j over the parental compd. to BRD4 possibly derives from improved interactions with the WPF shelf.  Meanwhile, substitution of the pyrimidine NH with an oxygen atom reversed the PLK1/BRD4 selectivity to convert BI-2536 into a BRD4-selective inhibitor, likely owing to the loss of a crit. hydrogen bond in PLK1.  We believe further fine-tuning will furnish a BRD4 "magic bullet" or an even more potent PLK1/BRD4 dual inhibitor toward the expansion and improved efficacy of the chemotherapy arsenal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5UnNSmf1L17Vg90H21EOLACvtfcHk0lhBJeVHyu-lxw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D&md5=f39070e652bca20c03a3bc5fe070a7fc</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00084%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYap%26aufirst%3DJ.%2BL.%26aulast%3DYoshioka%26aufirst%3DM.%26aulast%3DLanning%26aufirst%3DM.%2BE.%26aulast%3DFountain%26aufirst%3DR.%2BN.%26aulast%3DRaje%26aufirst%3DM.%26aulast%3DScheenstra%26aufirst%3DJ.%2BA.%26aulast%3DStrovel%26aufirst%3DJ.%2BW.%26aulast%3DFletcher%26aufirst%3DS.%26atitle%3DBRD4%2520structure-activity%2520relationships%2520of%2520dual%2520PLK1%2520kinase%252FBRD4%2520bromodomain%2520inhibitor%2520BI-2536%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D764%26epage%3D769%26doi%3D10.1021%2Facsmedchemlett.5b00084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watts, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strebhardt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellenie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Designing dual inhibitors of anaplastic lymphoma kinase (ALK) and bromodomain-4 (BRD4) by tuning kinase selectivity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2618</span>â <span class="NLM_lpage">2637</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01947</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01947" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2618-2637&author=E.+Wattsauthor=D.+Heidenreichauthor=E.+Tuckerauthor=M.+Raabauthor=K.+Strebhardtauthor=L.+Cheslerauthor=S.+Knappauthor=B.+Bellenieauthor=S.+Hoelder&title=Designing+dual+inhibitors+of+anaplastic+lymphoma+kinase+%28ALK%29+and+bromodomain-4+%28BRD4%29+by+tuning+kinase+selectivity&doi=10.1021%2Facs.jmedchem.8b01947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Designing Dual Inhibitors of Anaplastic Lymphoma Kinase (ALK) and Bromodomain-4 (BRD4) by Tuning Kinase Selectivity</span></div><div class="casAuthors">Watts, Ellen; Heidenreich, David; Tucker, Elizabeth; Raab, Monika; Strebhardt, Klaus; Chesler, Louis; Knapp, Stefan; Bellenie, Benjamin; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2618-2637</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Concomitant inhibition of anaplastic lymphoma kinase (ALK) and bromodomain-4 (BRD4) is a potential therapeutic strategy for targeting two key oncogenic drivers that co-segregate in a significant fraction of high-risk neuroblastoma patients, mutation of ALK and amplification of MYCN.  Starting from known dual polo-like kinase (PLK)-1-BRD4 inhibitor BI-2536, we employed structure-based design to redesign this series toward compds. with a dual ALK-BRD4 profile.  These efforts led to compd. (R)-2-((2-ethoxy-4-(1-methylpiperidin-4-yl)phenyl)amino)-7-ethyl-5-methyl-8-((4-methylthiophen-2-yl)methyl)-7,8-dihydropteridin-6(5H)-one (16k) demonstrating improved ALK activity and significantly reduced PLK-1 activity, while maintaining BRD4 activity and overall kinome selectivity.  We demonstrate the compds.' on-target engagement with ALK and BRD4 in cells as well as favorable broad kinase and bromodomain selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop6qx6PSj2KLVg90H21EOLACvtfcHk0lgVn2TCwy4M4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D&md5=e00fdf8eb20acc2ef1c8777b2afb750d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01947%26sid%3Dliteratum%253Aachs%26aulast%3DWatts%26aufirst%3DE.%26aulast%3DHeidenreich%26aufirst%3DD.%26aulast%3DTucker%26aufirst%3DE.%26aulast%3DRaab%26aufirst%3DM.%26aulast%3DStrebhardt%26aufirst%3DK.%26aulast%3DChesler%26aufirst%3DL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBellenie%26aufirst%3DB.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDesigning%2520dual%2520inhibitors%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%2520and%2520bromodomain-4%2520%2528BRD4%2529%2520by%2520tuning%2520kinase%2520selectivity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2618%26epage%3D2637%26doi%3D10.1021%2Facs.jmedchem.8b01947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosief, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhawan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthengi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and development of potent and selective dual bromodomain 4 (BRD4)/polo-like kinase 1 (PLK1) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7785</span>â <span class="NLM_lpage">7795</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00765</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00765" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7785-7795&author=S.+Liuauthor=H.+O.+Yosiefauthor=L.+Daiauthor=H.+Huangauthor=G.+Dhawanauthor=X.+Zhangauthor=A.+M.+Muthengiauthor=J.+Robertsauthor=D.+L.+Buckleyauthor=J.+A.+Perryauthor=L.+Wuauthor=J.+E.+Bradnerauthor=J.+Qiauthor=W.+Zhang&title=Structure-guided+design+and+development+of+potent+and+selective+dual+bromodomain+4+%28BRD4%29%2Fpolo-like+kinase+1+%28PLK1%29+inhibitors&doi=10.1021%2Facs.jmedchem.8b00765"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors</span></div><div class="casAuthors">Liu, Shuai; Yosief, Hailemichael O.; Dai, Lingling; Huang, He; Dhawan, Gagan; Zhang, Xiaofeng; Muthengi, Alex M.; Roberts, Justin; Buckley, Dennis L.; Perry, Jennifer A.; Wu, Lei; Bradner, James E.; Qi, Jun; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7785-7795</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single mol. could lead to the development of an effective therapeutic strategy for a variety of diseases in which PLK1 and BRD4 are implicated.  Compd. 23 has been found to be a potent dual kinase-bromodomain inhibitor (BRD4-BD1 IC50 = 28 nM, PLK1 IC50 = 40 nM).  Compd. 6 was found to be the most selective PLK1 inhibitor over BRD4 in our series (BRD4-BD1 IC50 = 2579 nM, PLK1 IC50 = 9.9 nM).  Mol. docking studies with 23 and BRD4-BD1/PLK1 as well as with 6 corroborate the biochem. assay results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUL2G-DnFzU7Vg90H21EOLACvtfcHk0lgVn2TCwy4M4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L&md5=e34a58694f0827d33d31516200e2c0d8</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00765%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYosief%26aufirst%3DH.%2BO.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DDhawan%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMuthengi%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DJ.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DStructure-guided%2520design%2520and%2520development%2520of%2520potent%2520and%2520selective%2520dual%2520bromodomain%25204%2520%2528BRD4%2529%252Fpolo-like%2520kinase%25201%2520%2528PLK1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7785%26epage%3D7795%26doi%3D10.1021%2Facs.jmedchem.8b00765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of Î³-Carboline analogues as potent and specific BET bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4927</span>â <span class="NLM_lpage">4939</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4927-4939&author=X.+Ranauthor=Y.+Zhaoauthor=L.+Liuauthor=L.+Baiauthor=C.+Y.+Yangauthor=B.+Zhouauthor=J.+L.+Meagherauthor=K.+Chinnaswamyauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-based+design+of+%CE%B3-Carboline+analogues+as+potent+and+specific+BET+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Î³-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span></div><div class="casAuthors">Ran, Xu; Zhao, Yujun; Liu, Liu; Bai, Longchuan; Yang, Chao-Yie; Zhou, Bing; Meagher, Jennifer L.; Chinnaswamy, Krishnapriya; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4927-4939</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors of bromodomain and extra terminal proteins (BET), including BRD2, BRD3, and BRD4 proteins have therapeutic potential for the treatment of human cancers and other diseases and conditions.  In this paper, the authors report the design, synthesis, and evaluation of Î³-carboline-contg. compds. as a new class of small-mol. BET inhibitors.  The most potent inhibitor I (RX-37) obtained from this study binds to BET bromodomain proteins (BRD2, BRD3, and BRD4) with Ki values of 3.2-24.7 nM and demonstrates high selectivity over other non-BET bromodomain-contg. proteins.  Compd. I potently and selectively inhibits cell growth in human acute leukemia cell lines harboring the rearranged mixed lineage leukemia 1 gene.  The authors have detd. a cocrystal structure of I in complex with BRD4 BD2 at 1.4 Ã resoln., which provides a solid structural basis for the compd.'s high binding affinity and for its further structure-based optimization.  Compd. I represents a promising lead compd. for the development of a new class of therapeutics for the treatment of human cancer and other conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv-VCfWLBWYLVg90H21EOLACvtfcHk0lgVn2TCwy4M4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM&md5=e6e150169812e439c1d7d91416eb3ee9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00613%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520design%2520of%2520%25CE%25B3-Carboline%2520analogues%2520as%2520potent%2520and%2520specific%2520BET%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4927%26epage%3D4939%26doi%3D10.1021%2Facs.jmedchem.5b00613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span> <span> </span><span class="NLM_article-title">Fragment-based drug discovery of 2-thiazolidinones as inhibitors of the histone reader BRD4 bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3833</span>â <span class="NLM_lpage">3851</span>, <span class="refDoi">Â DOI: 10.1021/jm301793a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301793a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3833-3851&author=L.+Zhaoauthor=D.+Caoauthor=T.+Chenauthor=Y.+Wangauthor=Z.+Miaoauthor=Y.+Xuauthor=W.+Chenauthor=X.+Wangauthor=Y.+Liauthor=Z.+Duauthor=B.+Xiongauthor=J.+Liauthor=C.+Xuauthor=N.+Zhangauthor=J.+Heauthor=J.+Shen&title=Fragment-based+drug+discovery+of+2-thiazolidinones+as+inhibitors+of+the+histone+reader+BRD4+bromodomain&doi=10.1021%2Fjm301793a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery of 2-Thiazolidinones as Inhibitors of the Histone Reader BRD4 Bromodomain</span></div><div class="casAuthors">Zhao, Lele; Cao, Danyan; Chen, Tiantian; Wang, Yingqing; Miao, Zehong; Xu, Yechun; Chen, Wuyan; Wang, Xin; Li, Yanlian; Du, Zhiyan; Xiong, Bing; Li, Jian; Xu, Chunyan; Zhang, Naixia; He, Jianhua; Shen, Jingkang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3833-3851</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recognizing acetyllysine of histone is a vital process of epigenetic regulation that is mediated by a protein module called bromodomain.  To contribute novel scaffolds for developing into bromodomain inhibitors, the authors utilize a fragment-based drug discovery approach.  By successively applying docking and x-ray crystallog., they were able to identify 9 fragment hits from diffracting more than 60 crystals.  In the present work, they described four of them and carried out the integrated lead optimization for fragment I, which bears a 2-thiazolidinone core.  After several rounds of structure guided modifications, they assessed the druggability of 2-thiazolidinone by modulating in vitro pharmacokinetic studies and cellular activity assay.  The results showed that two potent of 2-thiazolidinone compds. II and III have good metabolic stability.  Also, the cellular assay confirmed the activities of 2-thiazolidinones.  Together, they hope the identified 2-thiazolidinone chemotype and other fragment hits described herein can stimulate researchers to develop more diversified bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfi2pyYq6c4LVg90H21EOLACvtfcHk0ljEhZQYRmjOKA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D&md5=8308db5d83cb72b29f261cc134514ece</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm301793a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301793a%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DFragment-based%2520drug%2520discovery%2520of%25202-thiazolidinones%2520as%2520inhibitors%2520of%2520the%2520histone%2520reader%2520BRD4%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3833%26epage%3D3851%26doi%3D10.1021%2Fjm301793a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resar, L. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emison, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prescott, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wonsey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeller, K.</span></span> <span> </span><span class="NLM_article-title">Function of the c-Myc oncogenic transcription factor</span>. <i>Exp. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>253</i></span>,  <span class="NLM_fpage">63</span>â <span class="NLM_lpage">77</span>, <span class="refDoi">Â DOI: 10.1006/excr.1999.4686</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1006%2Fexcr.1999.4686" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10579912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADyaK1MXnsF2it7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1999&pages=63-77&author=C.+V.+Dangauthor=L.+M.+S.+Resarauthor=E.+Emisonauthor=S.+Kimauthor=Q.+Liauthor=J.+E.+Prescottauthor=D.+Wonseyauthor=K.+Zeller&title=Function+of+the+c-Myc+oncogenic+transcription+factor&doi=10.1006%2Fexcr.1999.4686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Function of the c-Myc Oncogenic Transcription Factor</span></div><div class="casAuthors">Dang, Chi V.; Resar, Linda M. S.; Emison, Eileen; Kim, Sunkyu; Li, Qing; Prescott, Julia E.; Wonsey, Diane; Zeller, Karen</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Cell Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-77</span>CODEN:
                <span class="NLM_cas:coden">ECREAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-4827</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review with 238 refs.  The c-myc gene and the expression of the c-Myc protein are frequently altered in human cancers.  The c-myc gene encodes the transcription factor c-Myc, which heterodimerizes with a partner protein, termed Max, to regulate gene expression.  Max also heterodimerizes with the Mad family of proteins to repress transcription, antagonize c-Myc, and promote cellular differentiation.  The constitutive activation of c-myc expression is key to the genesis of many cancers, and hence the understanding of c-Myc function depends on our understanding of its target genes.  In this review, the authors attempt to place the putative target genes of c-Myc in the context of c-Myc-mediated phenotypes.  From this perspective, c-Myc emerges as an oncogenic transcription factor that integrates the cell cycle machinery with cell adhesion, cellular metab., and the apoptotic pathways.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjNm2Xvnxf9LVg90H21EOLACvtfcHk0ljEhZQYRmjOKA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsF2it7w%253D&md5=34262c18507e4da66ee47be7c08d55d8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1006%2Fexcr.1999.4686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fexcr.1999.4686%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DResar%26aufirst%3DL.%2BM.%2BS.%26aulast%3DEmison%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DPrescott%26aufirst%3DJ.%2BE.%26aulast%3DWonsey%26aufirst%3DD.%26aulast%3DZeller%26aufirst%3DK.%26atitle%3DFunction%2520of%2520the%2520c-Myc%2520oncogenic%2520transcription%2520factor%26jtitle%3DExp.%2520Cell%2520Res.%26date%3D1999%26volume%3D253%26spage%3D63%26epage%3D77%26doi%3D10.1006%2Fexcr.1999.4686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappaport, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chicas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">524</span>â <span class="NLM_lpage">528</span>, <span class="refDoi">Â DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0ljEhZQYRmjOKA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26epage%3D528%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huthmacher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gudgin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doehner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">529</span>â <span class="NLM_lpage">533</span>, <span class="refDoi">Â DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.+I.+Chanauthor=S.+C.+Robsonauthor=C.+W.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+P.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0lgmaQYYjIs56g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span> <span> </span><span class="NLM_article-title">Brd4 recruits P-TEFb to chromosomes at late mitosis to promote G1 gene expression and cell cycle progression</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">967</span>â <span class="NLM_lpage">976</span>, <span class="refDoi">Â DOI: 10.1128/MCB.01020-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1128%2FMCB.01020-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=18039861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWqur4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=967-976&author=Z.+Yangauthor=N.+Heauthor=Q.+Zhou&title=Brd4+recruits+P-TEFb+to+chromosomes+at+late+mitosis+to+promote+G1+gene+expression+and+cell+cycle+progression&doi=10.1128%2FMCB.01020-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 recruits P-TEFb to chromosomes at late mitosis to promote G1 gene expression and cell cycle progression</span></div><div class="casAuthors">Yang, Zhiyuan; He, Nanhai; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">967-976</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Brd4, a bromodomain protein capable of interacting with acetylated histones, is implicated in transmitting epigenetic memory through mitosis.  It also functions as an assocd. factor and pos. regulator of P-TEFb, a Cdk9-cyclin T1 heterodimer that stimulates transcriptional elongation by phosphorylating RNA polymerase II.  In the present study, expts. were performed to det. whether these two functions of Brd4 are interrelated and, if so, how they may impact cell cycle progression.  Our data demonstrate that while the P-TEFb level remains const., the Brd4-P-TEFb interaction increases dramatically in cells progressing from late mitosis to early G1.  Concurrently, P-TEFb is recruited to chromosomes, beginning around mid- to late anaphase and before nuclear envelope/lamina formation and nuclear import of other general transcription factors.  Importantly, the recruitment of P-TEFb depends on Brd4.  Abrogation of this process through Brd4 knockdown reduces the binding of P-TEFb to and expression of key G1 and growth-assocd. genes, leading to G1 cell cycle arrest and apoptosis.  Because P-TEFb is synonymous with productive elongation, its recruitment by Brd4 to chromosomes at late mitosis may indicate those genes whose active transcription status must be preserved across cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_qFlZcTzQiLVg90H21EOLACvtfcHk0lgmaQYYjIs56g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWqur4%253D&md5=99b71de9dec48e64ecbcb59fda3b6d1d</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1128%2FMCB.01020-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01020-07%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DHe%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DBrd4%2520recruits%2520P-TEFb%2520to%2520chromosomes%2520at%2520late%2520mitosis%2520to%2520promote%2520G1%2520gene%2520expression%2520and%2520cell%2520cycle%2520progression%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2008%26volume%3D28%26spage%3D967%26epage%3D976%26doi%3D10.1128%2FMCB.01020-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamothe, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span> <span> </span><span class="NLM_article-title">The human ether-a-go-go-related Gene (hERG) potassium channel represents an unusual target for protease-mediated damage</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">20387</span>â <span class="NLM_lpage">20401</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M116.743138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1074%2Fjbc.M116.743138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=27502273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFGnurvM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=20387-20401&author=S.+M.+Lamotheauthor=J.+Guoauthor=W.+Liauthor=T.+Yangauthor=S.+Zhang&title=The+human+ether-a-go-go-related+Gene+%28hERG%29+potassium+channel+represents+an+unusual+target+for+protease-mediated+damage&doi=10.1074%2Fjbc.M116.743138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The human ether-a-go-go-related gene (hERG) potassium channel represents an unusual target for protease-mediated damage</span></div><div class="casAuthors">Lamothe, Shawn M.; Guo, Jun; Li, Wentao; Yang, Tonghua; Zhang, Shetuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">20387-20401</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The human ether-a-go-go-related gene (hERG) encodes the pore-forming subunit of the rapidly activating delayed rectifier potassium channel (IKr), which is important for cardiac repolarization.  Dysfunction of hERG causes long QT syndrome and sudden death, which occur in patients with cardiac ischemia.  Cardiac ischemia is also assocd. with activation, up-regulation, and secretion of various proteolytic enzymes.  Here, using whole-cell patch clamp and Western blotting anal., we demonstrate that the hERG/IKr channel was selectively cleaved by the serine protease, proteinase K (PK).  Using mol. biol. techniques including making a chimeric channel between protease-sensitive hERG and insensitive human ether-a-go-go (hEAG), as well as application of the scorpion toxin BeKm-1, we identified that the S5-pore linker of hERG is the target domain for proteinase K cleavage.  To investigate the physiol. relevance of the unique susceptibility of hERG to proteases, we show that cardiac ischemia in a rabbit model was assocd. with a redn. in mature ERG expression and an increase in the expression of several proteases, including calpain.  Using cell biol. approaches, we found that calpain-1 was actively released into the extracellular milieu and cleaved hERG at the S5-pore linker.  Using protease cleavage-predicting software and site-directed mutagenesis, we identified that calpain-1 cleaves hERG at position Gly-603 in the S5-pore linker of hERG.  Clarification of protease-mediated damage of hERG extends our understanding of hERG regulation.  Damage of hERG mediated by proteases such as calpain may contribute to ischemia-assocd. QT prolongation and sudden cardiac death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCd-cfWdEAfLVg90H21EOLACvtfcHk0lgmaQYYjIs56g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFGnurvM&md5=1c5d685ed5fc9161cc33d6a028289026</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.743138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.743138%26sid%3Dliteratum%253Aachs%26aulast%3DLamothe%26aufirst%3DS.%2BM.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DS.%26atitle%3DThe%2520human%2520ether-a-go-go-related%2520Gene%2520%2528hERG%2529%2520potassium%2520channel%2520represents%2520an%2520unusual%2520target%2520for%2520protease-mediated%2520damage%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D20387%26epage%3D20401%26doi%3D10.1074%2Fjbc.M116.743138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span> <span> </span><span class="NLM_article-title">Oxime-mediated facile access to 5-methylisoxazoles and applications in the synthesis of valdecoxib and oxacillin</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">5266</span>â <span class="NLM_lpage">5268</span>, <span class="refDoi">Â DOI: 10.1021/ol502246t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol502246t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlu7%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=5266-5268&author=K.+Y.+Dongauthor=H.+T.+Qinauthor=X.+X.+Baoauthor=F.+Liuauthor=C.+Zhu&title=Oxime-mediated+facile+access+to+5-methylisoxazoles+and+applications+in+the+synthesis+of+valdecoxib+and+oxacillin&doi=10.1021%2Fol502246t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Oxime-Mediated Facile Access to 5-Methylisoxazoles and Applications in the Synthesis of Valdecoxib and Oxacillin</span></div><div class="casAuthors">Dong, Kui-Yong; Qin, Hai-Tao; Bao, Xing-Xing; Liu, Feng; Zhu, Chen</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5266-5268</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A palladium-catalyzed efficient synthesis of 5-methylisoxazoles via oxime-mediated functionalization of unactivated olefins is described.  The reaction affords a variety of 5-methylisoxazoles in moderate to good yields.  To further demonstrate the utility of the method, the rapid synthesis of valdecoxib (I) and oxacillin (II) is reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP58UkfSzID7Vg90H21EOLACvtfcHk0lhc2vIXAhlvkQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlu7%252FO&md5=96e7e292e10e1b208d81f92c3ccc54cc</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fol502246t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol502246t%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DK.%2BY.%26aulast%3DQin%26aufirst%3DH.%2BT.%26aulast%3DBao%26aufirst%3DX.%2BX.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DZhu%26aufirst%3DC.%26atitle%3DOxime-mediated%2520facile%2520access%2520to%25205-methylisoxazoles%2520and%2520applications%2520in%2520the%2520synthesis%2520of%2520valdecoxib%2520and%2520oxacillin%26jtitle%3DOrg.%2520Lett.%26date%3D2014%26volume%3D16%26spage%3D5266%26epage%3D5268%26doi%3D10.1021%2Fol502246t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zibinsky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokin, V. V.</span></span> <span> </span><span class="NLM_article-title">Reactivity of N-(1,2,4-triazolyl)-substituted 1,2,3-triazoles</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4870</span>â <span class="NLM_lpage">4872</span>, <span class="refDoi">Â DOI: 10.1021/ol201949h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol201949h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVegt7vL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=4870-4872&author=M.+Zibinskyauthor=V.+V.+Fokin&title=Reactivity+of+N-%281%2C2%2C4-triazolyl%29-substituted+1%2C2%2C3-triazoles&doi=10.1021%2Fol201949h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Reactivity of N-(1,2,4-Triazolyl)-Substituted 1,2,3-Triazoles</span></div><div class="casAuthors">Zibinsky, Mikhail; Fokin, Valery V.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4870-4872</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetically useful rhodium(II) carbenes were obtained from N-(1,2,4-triazolyl)-substituted 1,2,3-triazoles, e.g., I, and Rh(II) carboxylates.  The electron-withdrawing 1,2,4-triazolyl group reveals the heretofore unknown reactivity of nonsulfonyl 1,2,3-triazoles, which exhibit the reactivity of diazo compds.  The resulting carbenes provide ready asym. access to secondary homoaminocyclopropanes, e.g., II, (80-95% ee, dr >20:1) via reactions with olefins and also engage in efficient transannulation reactions with nitriles to give imidazoles, e.g., III.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxp8JdKAoqqrVg90H21EOLACvtfcHk0lhc2vIXAhlvkQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVegt7vL&md5=178803d69a9ab09c91a8d24ea5fff2c0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fol201949h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol201949h%26sid%3Dliteratum%253Aachs%26aulast%3DZibinsky%26aufirst%3DM.%26aulast%3DFokin%26aufirst%3DV.%2BV.%26atitle%3DReactivity%2520of%2520N-%25281%252C2%252C4-triazolyl%2529-substituted%25201%252C2%252C3-triazoles%26jtitle%3DOrg.%2520Lett.%26date%3D2011%26volume%3D13%26spage%3D4870%26epage%3D4872%26doi%3D10.1021%2Fol201949h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=%5B20%5D+Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (Ï2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Ã unit cell to rotavirus of 700 Ã diam. crystd. in 700 Ã 1000 Ã 1400 Ã cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhc2vIXAhlvkQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Cryst.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>â <span class="NLM_lpage">255</span>, <span class="refDoi">Â DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgPlaUJQxeOMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Cryst.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <i>Acta Cryst.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>â <span class="NLM_lpage">1954</span>, <span class="refDoi">Â DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.+W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0lgPlaUJQxeOMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Cryst.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Cryst.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>â <span class="NLM_lpage">501</span>, <span class="refDoi">Â DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgPlaUJQxeOMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Cryst.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 9 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Kaikai Lv, Weicong Chen, Danqi Chen, Jie Mou, Huijie Zhang, Tiantian Fan, Yanlian Li, Danyan Cao, Xin Wang, Lin Chen, Jingkang Shen, Dongsheng Pei, <span class="NLM_string-name hlFld-ContribAuthor">Bing Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design and Evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as Polypharmacological Inhibitors of BET and Kinases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9787-9802. <a href="https://doi.org/10.1021/acs.jmedchem.0c00962" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00962</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00962%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRational%252BDesign%252Band%252BEvaluation%252Bof%252B6-%252528Pyrimidin-2-ylamino%252529-3%25252C4-dihydroquinoxalin-2%2525281H%252529-ones%252Bas%252BPolypharmacological%252BInhibitors%252Bof%252BBET%252Band%252BKinases%26aulast%3DLv%26aufirst%3DKaikai%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05062020%26date%3D26082020%26date%3D11082020%26volume%3D63%26issue%3D17%26spage%3D9787%26epage%3D9802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhangxu  He</span>, <span class="hlFld-ContribAuthor ">Haomiao  Jiao</span>, <span class="hlFld-ContribAuthor ">Qi  An</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Dan  Zengyangzong</span>, <span class="hlFld-ContribAuthor ">Jiale  Xu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>, <span class="hlFld-ContribAuthor ">Liying  Ma</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel 4-phenylquinazoline-based BRD4 inhibitors for cardiac fibrosis. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>399 </em><a href="https://doi.org/10.1016/j.apsb.2021.07.018" title="DOI URL">https://doi.org/10.1016/j.apsb.2021.07.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2021.07.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2021.07.018%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Bnovel%252B4-phenylquinazoline-based%252BBRD4%252Binhibitors%252Bfor%252Bcardiac%252Bfibrosis%26aulast%3DHe%26aufirst%3DZhangxu%26date%3D2021%26volume%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qian  Wu</span>, <span class="hlFld-ContribAuthor ">Dan-Qi  Chen</span>, <span class="hlFld-ContribAuthor ">Lin  Sun</span>, <span class="hlFld-ContribAuthor ">Xia-Juan  Huan</span>, <span class="hlFld-ContribAuthor ">Xu-Bin  Bao</span>, <span class="hlFld-ContribAuthor ">Chang-Qing  Tian</span>, <span class="hlFld-ContribAuthor ">Jianping  Hu</span>, <span class="hlFld-ContribAuthor ">Kai-Kai  Lv</span>, <span class="hlFld-ContribAuthor ">Ying-Qing  Wang</span>, <span class="hlFld-ContribAuthor ">Bing  Xiong</span>, <span class="hlFld-ContribAuthor ">Ze-Hong  Miao</span>. </span><span class="cited-content_cbyCitation_article-title">Novel bivalent BET inhibitor N2817 exhibits potent anticancer activity and inhibits TAF1. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2021,</strong> <em>185 </em>, 114435. <a href="https://doi.org/10.1016/j.bcp.2021.114435" title="DOI URL">https://doi.org/10.1016/j.bcp.2021.114435</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2021.114435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2021.114435%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DNovel%252Bbivalent%252BBET%252Binhibitor%252BN2817%252Bexhibits%252Bpotent%252Banticancer%252Bactivity%252Band%252Binhibits%252BTAF1%26aulast%3DWu%26aufirst%3DQian%26date%3D2021%26volume%3D185%26spage%3D114435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Morgan  Mann</span>, <span class="hlFld-ContribAuthor ">David S.  Roberts</span>, <span class="hlFld-ContribAuthor ">Yanlong  Zhu</span>, <span class="hlFld-ContribAuthor ">Yi  Li</span>, <span class="hlFld-ContribAuthor ">Jia  Zhou</span>, <span class="hlFld-ContribAuthor ">Ying  Ge</span>, <span class="hlFld-ContribAuthor ">Allan R.  Brasier</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of RSV-Induced BRD4 Protein Interactions Using Native Immunoprecipitation and Parallel AccumulationâSerial Fragmentation (PASEF) Mass Spectrometry. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2021,</strong> <em>13 </em>
                                    (3)
                                     , 454. <a href="https://doi.org/10.3390/v13030454" title="DOI URL">https://doi.org/10.3390/v13030454</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v13030454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv13030454%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DDiscovery%252Bof%252BRSV-Induced%252BBRD4%252BProtein%252BInteractions%252BUsing%252BNative%252BImmunoprecipitation%252Band%252BParallel%252BAccumulation%2525E2%252580%252594Serial%252BFragmentation%252B%252528PASEF%252529%252BMass%252BSpectrometry%26aulast%3DMann%26aufirst%3DMorgan%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D3%26spage%3D454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peng  Fan</span>, <span class="hlFld-ContribAuthor ">Tian-Tian  Liu</span>, <span class="hlFld-ContribAuthor ">Hong-Yu  Qu</span>, <span class="hlFld-ContribAuthor ">Peng  Tao</span>, <span class="hlFld-ContribAuthor ">Chun-Xia  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaoqian  Liu</span>, <span class="hlFld-ContribAuthor ">Mei-Hua  Shen</span>, <span class="hlFld-ContribAuthor ">Xiaoguang  Bao</span>, <span class="hlFld-ContribAuthor ">Hua-Dong  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Intramolecular Alder-Ene Cycloisomerization of Cyclopropene with Alkenes to Access Spirocycles. </span><span class="cited-content_cbyCitation_journal-name">Organic Chemistry Frontiers</span><span> <strong>2021,</strong> <em> </em><a href="https://doi.org/10.1039/D1QO00299F" title="DOI URL">https://doi.org/10.1039/D1QO00299F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1QO00299F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1QO00299F%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Chemistry%2520Frontiers%26atitle%3DIntramolecular%252BAlder-Ene%252BCycloisomerization%252Bof%252BCyclopropene%252Bwith%252BAlkenes%252Bto%252BAccess%252BSpirocycles%26aulast%3DFan%26aufirst%3DPeng%26date%3D2021%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu  Xu</span>, <span class="hlFld-ContribAuthor ">Xiu-Juan  Zhang</span>, <span class="hlFld-ContribAuthor ">Wen-Bo  Li</span>, <span class="hlFld-ContribAuthor ">Xing-Rong  Wang</span>, <span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Xue-Peng  Qiao</span>, <span class="hlFld-ContribAuthor ">Shi-Wu  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of indole-2-one derivatives as potent BRD4 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>208 </em>, 112780. <a href="https://doi.org/10.1016/j.ejmech.2020.112780" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112780</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112780%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bindole-2-one%252Bderivatives%252Bas%252Bpotent%252BBRD4%252Binhibitors%26aulast%3DXu%26aufirst%3DYu%26date%3D2020%26volume%3D208%26spage%3D112780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qifei  Li</span>, <span class="hlFld-ContribAuthor ">Jieming  Li</span>, <span class="hlFld-ContribAuthor ">Yan  Cai</span>, <span class="hlFld-ContribAuthor ">Yuxing  Zou</span>, <span class="hlFld-ContribAuthor ">Bin  Chen</span>, <span class="hlFld-ContribAuthor ">Feng  Zou</span>, <span class="hlFld-ContribAuthor ">Jiaxian  Mo</span>, <span class="hlFld-ContribAuthor ">Ting  Han</span>, <span class="hlFld-ContribAuthor ">Weiwei  Guo</span>, <span class="hlFld-ContribAuthor ">Wenlong  Huang</span>, <span class="hlFld-ContribAuthor ">Qianqian  Qiu</span>, <span class="hlFld-ContribAuthor ">Hai  Qian</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel 6-phenyl-1,3a,4,10b-tetrahydro-2H-benzo[c]thiazolo[4,5-e]azepin-2-one derivatives as potential BRD4 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (15)
                                     , 115601. <a href="https://doi.org/10.1016/j.bmc.2020.115601" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115601</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115601%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252B6-phenyl-1%25252C3a%25252C4%25252C10b-tetrahydro-2H-benzo%25255Bc%25255Dthiazolo%25255B4%25252C5-e%25255Dazepin-2-one%252Bderivatives%252Bas%252Bpotential%252BBRD4%252Binhibitors%26aulast%3DLi%26aufirst%3DQifei%26date%3D2020%26volume%3D28%26issue%3D15%26spage%3D115601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandra  Cipriano</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic reader domains by chemical biology. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 82-94. <a href="https://doi.org/10.1016/j.cbpa.2020.05.006" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.05.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.05.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DTargeting%252Bepigenetic%252Breader%252Bdomains%252Bby%252Bchemical%252Bbiology%26aulast%3DCipriano%26aufirst%3DAlessandra%26date%3D2020%26volume%3D57%26spage%3D82%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fernando D.  Prieto-MartÃ­nez</span>, <span class="hlFld-ContribAuthor ">JosÃ© L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Current advances on the development of BET inhibitors: insights from computational methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 127-180. <a href="https://doi.org/10.1016/bs.apcsb.2020.06.002" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2020.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2020.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2020.06.002%26sid%3Dliteratum%253Aachs%26atitle%3DCurrent%252Badvances%252Bon%252Bthe%252Bdevelopment%252Bof%252BBET%252Binhibitors%25253A%252Binsights%252Bfrom%252Bcomputational%252Bmethods%26aulast%3DPrieto-Mart%25C3%25ADnez%26aufirst%3DFernando%2BD.%26date%3D2020%26spage%3D127%26epage%3D180%26pub%3DElsevier%26date%3D2020%26volume%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0002.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0023.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of typical BET inhibitors including (+)-JQ1 (<b>1</b>), OTX-015 (<b>2</b>), I-BET762 (<b>3</b>), and hit compound <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0024.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Superimposition of the cocrystal structure of BRD4 BD1 bound with <b>4</b> (PDB entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI3">6JI3</a>, green color) and BRD4-BD2 bound with compound <b>5</b> (PDB entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z93">4Z93</a>, gray color).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0022.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of compound <b>7</b> bound to BRD4-BD1 (PDB entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI4">6JI4</a>) is shown in cartoon mode and surface mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0021.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystal structure of BRD4-BD1 bound with compound <b>29</b> (PDB entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI5">6JI5</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0020.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selectivity assessment of compound <b>19</b> against the BRD family members using a BROMOscan panel. The 32 screened targets are labeled in black on the BRD family phylogenetic tree. BRDs that were not part of the screening panel are in gray. Compound <b>19</b> was screened at 1 Î¼M concentration, and the results for primary screen binding interactions are reported as the âPercent Controlâ, with lower numbers indicating stronger hits in the matrix.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0019.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibitory activities of compound <b>19</b> against 17 cell lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0018.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) MV4-11 cells were treated with compound <b>19</b> or OTX-015 at the indicated doses for 24 h and examined for c-Myc and GAPDH protein levels by western blotting. (B) Similar to (A) but examining <i>c-Myc</i> mRNA levels by RT-qPCR. *<i>p</i> < 0.05. Error bars represent the SD from three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0017.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) MV4-11 cells were treated with compound <b>19</b> or OTX-015 at the indicated doses for 24 h, and apoptosis was measured by the Annexin V apoptosis assay. *<i>p</i> < 0.05; **<i>p</i> < 0.01. (B) Similar to (A) but examined for cleaved PARP and GAPDH protein expression by western blotting. (C) Similar to (A) but examined for caspase-3, cleaved caspase-3, caspase-7, cleaved caspase-7, caspase-8, cleaved caspase-8, caspase-9, cleaved caspase-9, and GAPDH protein expression by western blotting. (D) Similar to (A) but examined for BCL-2, p-BCL-2(Thr56), p-BCL-2(Ser70), BCL-xL, MCL-1, XIAP, c-IAP1, c-IAP2, BAK, BAX, PUMA, BID, BIM, and GAPDH protein expression by western blotting. (E) Similar to (A) but examined for <i>BCL-2</i> mRNA expression by RT-qPCR. *<i>p</i> < 0.05; **<i>p</i> < 0.01. Error bars represent the SD of three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0016.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) Relative tumor volume (RTV) of human MV4-11 xenografts in Balb/c nude mice after treatments. OTX-015 or compound <b>19</b> group versus the vehicle group; **<i>p</i> < 0.01; ***<i>p</i> < 0.001. Formulation: <b>19</b>, 0.5% Tween80, and 0.5% methylcellulose aqueous solution and OTX-015, 5% dimethylacetamide, and 0.5% hydroxypropyl methyl cellulose. (B) Body weight changes in mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0015.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route of Compound <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) cyclopropylamine, ClCH<sub>2</sub>CH<sub>2</sub>Cl, 80 Â°C, 12 h; (b) Fe, NH<sub>4</sub>Cl, EtOH, 80 Â°C, 1 h; (c) 1. 2-bromopropanoyl bromide, DIPEA, DCM, 0 Â°C to rt, 2 h; 2. CH<sub>3</sub>CN, DIPEA, 80 Â°C, overnight; (d) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h; and (e) <i>N</i>-Boc-2-pyrroleboronic acid, Pd(dppf)Cl<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, DMF, 120 Â°C, 4 h.</p></p></figure><figure data-id="sch2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0014.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Routes of Compounds <b>6</b>, <b>8</b>, and <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>p</i>-toluidine, HATU, DIPEA, DMF, rt, overnight; (b) R<sup>2</sup>NH<sub>2</sub>, ClCH<sub>2</sub>CH<sub>2</sub>Cl, 80 Â°C, 12 h; (c) Lawessonâs reagent, toluene, 110 Â°C, overnight; (d) <b>43</b>: 1. N<sub>2</sub>H<sub>4</sub>Â·H<sub>2</sub>O, MeOH, rt, 2 h; 2. trimethylorthoformate, DMF, AcOH, rt, overnight; <b>44</b> and <b>45</b>: Hg(OAc)<sub>2</sub>, C<sub>3</sub>H<sub>9</sub>N<sub>3</sub>Si, THF, 0 Â°C, 3 h; (e) tin(II) chloride dihydrate, conc. HCl, rt, 5 h; (f) 1. 2-bromopropanoyl bromide, DIPEA, DCM, 0 Â°C to rt, 2 h; 2. CH<sub>3</sub>CN, DIPEA, 80 Â°C, overnight; and (g) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h.</p></p></figure><figure data-id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route of Compound <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) SOCl<sub>2</sub>, MeOH, 60 Â°C, 24 h; (b) cyclopropylamine, ClCH<sub>2</sub>CH<sub>2</sub>Cl, 80 Â°C, 12 h; (c) Fe, NH<sub>4</sub>Cl, EtOH, 80 Â°C, 1 h; (d) 1. 2-bromopropanoyl bromide, DIPEA, DCM, 0 Â°C to rt, 2 h; 2. CH<sub>3</sub>CN, DIPEA, 80 Â°C, overnight; (e) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h; (f) N<sub>2</sub>H<sub>4</sub>Â·H<sub>2</sub>O, EtOH, 80 Â°C, 12 h; and (g) <i>N</i>-(<i>p</i>-tolyl)ethanethioamide, Hg(OAc)<sub>2</sub>, AcOH, THF, 0 Â°C to rt, 24 h.</p></p></figure><figure data-id="sch4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0012.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthetic Route of Compound <b>9</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>p</i>-toluidine, THF, 0 Â°C to rt, 12 h; (b) cyclopropylamine, ClCH<sub>2</sub>CH<sub>2</sub>Cl, 80 Â°C, 12 h; (c) tin(II) chloride dihydrate, conc. HCl, rt, 5 h; (d) 1. 2-bromopropanoyl bromide, DIPEA, DCM, 0 Â°C to rt, 2 h; 2. CH<sub>3</sub>CN, DIPEA, 80 Â°C, overnight; and (e) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h.</p></p></figure><figure data-id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0011.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthetic Route of Compound <b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <span class="smallcaps smallerCapital">d</span>-alanine, K<sub>2</sub>CO<sub>3</sub>, EtOH, H<sub>2</sub>O, 80 Â°C, 8 h; (b) K<sub>2</sub>CO<sub>3</sub>, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, H<sub>2</sub>O, 60 Â°C, overnight; (c) phenylsilane, cyclopentanone, dibutyltindichloride, THF, rt, 10 h; (d) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h; (e) N<sub>2</sub>H<sub>4</sub>Â·H<sub>2</sub>O, EtOH, 80 Â°C, 12 h; and (f) <i>N</i>-(<i>p</i>-tolyl)ethanethioamide, Hg(OAc)<sub>2</sub>, AcOH, THF, 0 Â°C to rt, 24 h.</p></p></figure><figure data-id="sch6" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0010.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthetic Routes of Compounds <b>12</b> and <b>14</b>â<b>31</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <span class="smallcaps smallerCapital">d</span>-alanine, K<sub>2</sub>CO<sub>3</sub>, EtOH, H<sub>2</sub>O, 80 Â°C, 8 h; (b) K<sub>2</sub>CO<sub>3</sub>, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, H<sub>2</sub>O, 60 Â°C, overnight; (c) phenylsilane, cyclopentanone, dibutyltin dichloride, THF, rt, 10 h; (d) NaH, 0 Â°C, 30 min, iodomethane, rt, 2 h; (e) Pd(dppf)Cl<sub>2</sub>, bis(pinacolato)diboron, AcOK, DMSO, 80 Â°C, overnight; (f) Pd(dppf)Cl<sub>2</sub>, NaHCO<sub>3</sub>, THF/H<sub>2</sub>O, 80 Â°C, overnight; (g) Pd(dppf)Cl<sub>2</sub>, NaHCO<sub>3</sub>, THF/H<sub>2</sub>O, 80 Â°C, overnight; (h) LiOH, THF/H<sub>2</sub>O, rt, overnight; and (i) HATU, DIPEA, DMF, rt, overnight.</p></p></figure><figure data-id="sch7" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/medium/jm9b01094_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0009.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthetic Route of Compound <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b01094/20190920/images/large/jm9b01094_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01094&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(dppf)Cl<sub>2</sub>, bis(pinacolato)diboron, AcOK, DMSO, 80 Â°C, overnight; (b) Pd(dppf)Cl<sub>2</sub>, NaHCO<sub>3</sub>, THF/H<sub>2</sub>O, 80 Â°C, overnight; (c) LiOH, THF/H<sub>2</sub>O, rt, overnight; and (d) HATU, DIPEA, DMF, rt, overnight.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i107">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14624" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14624" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 50 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">384</span>â <span class="NLM_lpage">400</span>, <span class="refDoi">Â DOI: 10.1038/nrd3674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery&doi=10.1038%2Fnrd3674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lhevErrWYMxwA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400%26doi%3D10.1038%2Fnrd3674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">465</span>â <span class="NLM_lpage">477</span>, <span class="refDoi">Â DOI: 10.1038/nrc3256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer&doi=10.1038%2Fnrc3256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel Î±-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0lhxvY2ra5_Hbw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477%26doi%3D10.1038%2Fnrc3256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>â <span class="NLM_lpage">231</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize Îµ-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lhxvY2ra5_Hbw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, N. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alsarraj, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukes, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Officewala, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, K. W.</span></span> <span> </span><span class="NLM_article-title">Bromodomain 4 activation predicts breast cancer survival</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">6380</span>â <span class="NLM_lpage">6385</span>, <span class="refDoi">Â DOI: 10.1073/pnas.0710331105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1073%2Fpnas.0710331105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=18427120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsFyhtb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=6380-6385&author=N.+P.+S.+Crawfordauthor=J.+Alsarrajauthor=L.+Lukesauthor=R.+C.+Walkerauthor=J.+S.+Officewalaauthor=H.+H.+Yangauthor=M.+P.+Leeauthor=K.+Ozatoauthor=K.+W.+Hunter&title=Bromodomain+4+activation+predicts+breast+cancer+survival&doi=10.1073%2Fpnas.0710331105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain 4 activation predicts breast cancer survival</span></div><div class="casAuthors">Crawford, Nigel P. S.; Alsarraj, Jude; Lukes, Luanne; Walker, Renard C.; Officewala, Jennifer S.; Yang, Howard H.; Lee, Maxwell P.; Ozato, Keiko; Hunter, Kent W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6380-6385</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Previous work identified the Rap1 GTPase-activating protein Sipa1 as a germ-line-encoded metastasis modifier.  The bromodomain protein Brd4 phys. interacts with and modulates the enzymic activity of Sipa1.  In vitro anal. of a highly metastatic mouse mammary tumor cell line ectopically expressing Brd4 demonstrates significant redn. of invasiveness without altering intrinsic growth rate.  However, a dramatic redn. of tumor growth and pulmonary metastasis was obsd. after s.c. implantation into mice, implying that activation of Brd4 may somehow be manipulating response to tumor microenvironment in the in vivo setting.  Further in vitro anal. shows that Brd4 modulates extracellular matrix gene expression, a class of genes frequently present in metastasis-predictive gene signatures.  Microarray anal. of the mammary tumor cell lines identified a Brd4 activation signature that robustly predicted progression and/or survival in multiple human breast cancer datasets analyzed on different microarray platforms.  Intriguingly, the Brd4 signature also almost perfectly matches a mol. classifier of low-grade tumors.  Taken together, these data suggest that dysregulation of Brd4-assocd. pathways may play an important role in breast cancer progression and underlies multiple common prognostic signatures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIT8F-snY4rrVg90H21EOLACvtfcHk0lhxvY2ra5_Hbw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsFyhtb0%253D&md5=bdd4b3676ebbe0a39da15d020f2994e9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0710331105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0710331105%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DN.%2BP.%2BS.%26aulast%3DAlsarraj%26aufirst%3DJ.%26aulast%3DLukes%26aufirst%3DL.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26aulast%3DOfficewala%26aufirst%3DJ.%2BS.%26aulast%3DYang%26aufirst%3DH.%2BH.%26aulast%3DLee%26aufirst%3DM.%2BP.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DHunter%26aufirst%3DK.%2BW.%26atitle%3DBromodomain%25204%2520activation%2520predicts%2520breast%2520cancer%2520survival%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D6380%26epage%3D6385%26doi%3D10.1073%2Fpnas.0710331105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyoshi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubonishi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grier, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Atayde, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, J. A.</span></span> <span> </span><span class="NLM_article-title">BRD4-NUT fusion oncogene: a novel mechanism in aggressive carcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">304</span>â <span class="NLM_lpage">307</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=12543779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1Klug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=304-307&author=C.+A.+Frenchauthor=I.+Miyoshiauthor=I.+Kubonishiauthor=H.+E.+Grierauthor=A.+R.+Perez-Ataydeauthor=J.+A.+Fletcher&title=BRD4-NUT+fusion+oncogene%3A+a+novel+mechanism+in+aggressive+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4-NUT Fusion Oncogene: A Novel Mechanism in Aggressive Carcinoma</span></div><div class="casAuthors">French, Christopher A.; Miyoshi, Isao; Kubonishi, Ichiro; Grier, Holcombe E.; Perez-Atayde, Antonio R.; Fletcher, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">304-307</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The poorly differentiated carcinoma with t(15;19)(q13,p13.1) is characterized by its highly aggressive, invariably lethal clin. course.  The chromosome 19 translocation breakpoint targets the BRD4 double bromodomain-contg. gene, which functions in regulation of cell cycle progression.  Herein the authors demonstrate that BRD4 is fused with nearly the entire transcript of the novel 15q13 gene, NUT (nuclear protein in testis), forming a 6.4-kb fusion oncogene, BRD4-NUT.  NUT, like BRD4, is predicted to encode a nuclear protein but, unlike the ubiquitous BRD4 transcript, is expressed only in testis.  These findings establish a model to elucidate the oncogenic consequences of unscheduled NUT expression and altered BRD4 function.  Very few fusion oncogenes have been identified in epithelial tumors, and BRD4-NUT is the first fusion oncogene mechanism identified in a highly lethal form of carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTa0K_kBu2pLVg90H21EOLACvtfcHk0liGLMyuZRJlnA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt1Klug%253D%253D&md5=e927ea5874b2013c7467f981e10d7efd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DMiyoshi%26aufirst%3DI.%26aulast%3DKubonishi%26aufirst%3DI.%26aulast%3DGrier%26aufirst%3DH.%2BE.%26aulast%3DPerez-Atayde%26aufirst%3DA.%2BR.%26aulast%3DFletcher%26aufirst%3DJ.%2BA.%26atitle%3DBRD4-NUT%2520fusion%2520oncogene%253A%2520a%2520novel%2520mechanism%2520in%2520aggressive%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D304%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Bromodomains as therapeutic targets in cancer</span>. <i>Briefings Funct. Genomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">219</span>â <span class="NLM_lpage">230</span>, <span class="refDoi">Â DOI: 10.1093/bfgp/elt007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1093%2Fbfgp%2Felt007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=23543289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXos12hu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=219-230&author=I.+Barbieriauthor=E.+Cannizzaroauthor=M.+A.+Dawson&title=Bromodomains+as+therapeutic+targets+in+cancer&doi=10.1093%2Fbfgp%2Felt007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains as therapeutic targets in cancer</span></div><div class="casAuthors">Barbieri, Isaia; Cannizzaro, Ester; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Functional Genomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">219-230</span>CODEN:
                <span class="NLM_cas:coden">BFGRAR</span>;
        ISSN:<span class="NLM_cas:issn">2041-2649</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The malleability of the epigenome has long been recognized as a unique opportunity for therapeutic intervention.  Interest in targeting components of the epigenetic machinery for therapeutic gain had initially been aimed at chromatin modifying enzymes.  However, advances in medicinal chem. have now made it possible to exploit protein-protein interactions at the chromatin interface.  Bromodomains (BRD) are a conserved motif used by a large no. of chromatin-assocd. proteins to recognize and bind acetylated histone tails.  Small mols. with high specificity for the Bromodomain and Extra Terminal family of proteins (BRD2, BRD3, BRD4 and BRDT) have recently been shown to have remarkable pre-clin. efficacy in various malignancies.  These findings have provided the impetus for exploring other BRD proteins as novel targets in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnjEcNXUf4I7Vg90H21EOLACvtfcHk0liGLMyuZRJlnA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXos12hu74%253D&md5=0a41c0491042217395ffafbc28a25ed5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fbfgp%2Felt007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbfgp%252Felt007%26sid%3Dliteratum%253Aachs%26aulast%3DBarbieri%26aufirst%3DI.%26aulast%3DCannizzaro%26aufirst%3DE.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DBromodomains%2520as%2520therapeutic%2520targets%2520in%2520cancer%26jtitle%3DBriefings%2520Funct.%2520Genomics%26date%3D2013%26volume%3D12%26spage%3D219%26epage%3D230%26doi%3D10.1093%2Fbfgp%2Felt007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">PÃ©rez-Salvia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteller, M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibitors and cancer therapy: from structures to applications</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">323</span>â <span class="NLM_lpage">339</span>, <span class="refDoi">Â DOI: 10.1080/15592294.2016.1265710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1080%2F15592294.2016.1265710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=27911230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A280%3ADC%252BC2sjivVaksQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=323-339&author=M.+P%C3%A9rez-Salviaauthor=M.+Esteller&title=Bromodomain+inhibitors+and+cancer+therapy%3A+from+structures+to+applications&doi=10.1080%2F15592294.2016.1265710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain inhibitors and cancer therapy: From structures to applications</span></div><div class="casAuthors">Perez-Salvia Montserrat; Esteller Manel; Esteller Manel; Esteller Manel</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">323-339</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Aberrations in the epigenetic landscape are a hallmark of cancer.  Alterations in enzymes that are "writers," "erasers," or "readers" of histone modification marks are common.  Bromodomains are "readers" that bind acetylated lysines in histone tails.  Their most important function is the regulation of gene transcription by the recruitment of different molecular partners.  Moreover, proteins containing bromodomains are also epigenetic regulators, although little is known about the specific function of these domains.  In recent years, there has been increasing interest in developing small molecules that can target specific bromodomains.  First, this has helped clarify biological functions of bromodomain-containing proteins.  Secondly, it opens a new front for combatting cancer.  In this review we will describe the structures and mechanisms associated with Bromodomain and Extra-Terminal motif (BET) inhibitors and non-BET inhibitors, their current status of development, and their promising role as anti-cancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCPfARVC-uxp6KlWarZTAzfW6udTcc2eaqsaq7Loindbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjivVaksQ%253D%253D&md5=3ce4e19d8177ddc1705fd0d58e5fb020</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1080%2F15592294.2016.1265710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15592294.2016.1265710%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Salvia%26aufirst%3DM.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DBromodomain%2520inhibitors%2520and%2520cancer%2520therapy%253A%2520from%2520structures%2520to%2520applications%26jtitle%3DEpigenetics%26date%3D2017%26volume%3D12%26spage%3D323%26epage%3D339%26doi%3D10.1080%2F15592294.2016.1265710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meslamani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">The bromodomain: from epigenome reader to druggable target</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1839</i></span>,  <span class="NLM_fpage">676</span>â <span class="NLM_lpage">685</span>, <span class="refDoi">Â DOI: 10.1016/j.bbagrm.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.bbagrm.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=24686119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=676-685&author=R.+Sanchezauthor=J.+Meslamaniauthor=M.+M.+Zhou&title=The+bromodomain%3A+from+epigenome+reader+to+druggable+target&doi=10.1016%2Fj.bbagrm.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain: From epigenome reader to druggable target</span></div><div class="casAuthors">Sanchez, Roberto; Meslamani, Jamel; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">676-685</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a fundamental post-translational modification that plays an important role in the control of gene transcription in chromatin in an ordered fashion.  The bromodomain, the conserved structural module present in transcription-assocd. proteins, functions exclusively to recognize acetyl-lysine on histones and non-histone proteins.  The structural analyses of bromodomains' recognition of lysine-acetylated peptides derived from histones and cellular proteins provide detailed insights into the differences and unifying features of biol. ligand binding selectivity by the bromodomains.  Newly developed small-mol. inhibitors targeting bromodomain proteins further highlight the functional importance of bromodomain/acetyl-lysine binding as a key mechanism in orchestrating mol. interactions and regulation in chromatin biol. and gene transcription.  These new studies argue that modulating bromodomain/acetyl-lysine interactions with small-mol. chems. offer new opportunities to control gene expression in a wide array of human diseases including cancer and inflammation.  This article is part of a Special Issue entitled: Mol. mechanisms of histone modification function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO3Au7jfK0ibVg90H21EOLACvtfcHk0lgTkpBPcaYUsw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D&md5=95ec21cf13231d9e63bce9b3ccdd6d6f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DMeslamani%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DThe%2520bromodomain%253A%2520from%2520epigenome%2520reader%2520to%2520druggable%2520target%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2014%26volume%3D1839%26spage%3D676%26epage%3D685%26doi%3D10.1016%2Fj.bbagrm.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor Brd4 and transcriptional regulation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">13141</span>â <span class="NLM_lpage">13145</span>, <span class="refDoi">Â DOI: 10.1074/jbc.R700001200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.+Y.+Wuauthor=C.+M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+Brd4+and+transcriptional+regulation&doi=10.1074%2Fjbc.R700001200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0lgTkpBPcaYUsw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520Brd4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145%26doi%3D10.1074%2Fjbc.R700001200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>â <span class="NLM_lpage">1073</span>, <span class="refDoi">Â DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0ljNjQGtc0Sm0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noel, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ooike, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugahara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daibata, M.</span></span> <span> </span><span class="NLM_article-title">Development of the BET bromodomain inhibitor OTX015</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_fpage">C244</span>â <span class="NLM_lpage">C244</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.TARG-13-C244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1158%2F1535-7163.TARG-13-C244" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=C244-C244&author=J.+K.+Noelauthor=K.+Iwataauthor=S.+Ooikeauthor=K.+Sugaharaauthor=H.+Nakamuraauthor=M.+Daibata&title=Development+of+the+BET+bromodomain+inhibitor+OTX015&doi=10.1158%2F1535-7163.TARG-13-C244"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-13-C244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-13-C244%26sid%3Dliteratum%253Aachs%26aulast%3DNoel%26aufirst%3DJ.%2BK.%26aulast%3DIwata%26aufirst%3DK.%26aulast%3DOoike%26aufirst%3DS.%26aulast%3DSugahara%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DH.%26aulast%3DDaibata%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520the%2520BET%2520bromodomain%2520inhibitor%2520OTX015%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2013%26spage%3DC244%26epage%3DC244%26doi%3D10.1158%2F1535-7163.TARG-13-C244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Roca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de La Motte Rouge, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uro-Coste, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Braud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelosi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span> <span> </span><span class="NLM_article-title">Clinical response of carcinomas harboring the BRD4-NUT oncoprotein to the targeted bromodomain inhibitor OTX015/MK-8628</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">492</span>â <span class="NLM_lpage">500</span>, <span class="refDoi">Â DOI: 10.1158/2159-8290.CD-15-1335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1158%2F2159-8290.CD-15-1335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=26976114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlOqsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=492-500&author=A.+Stathisauthor=E.+Zuccaauthor=M.+Bekraddaauthor=C.+Gomez-Rocaauthor=J.-P.+Delordauthor=T.+de+La+Motte+Rougeauthor=E.+Uro-Costeauthor=F.+de+Braudauthor=G.+Pelosiauthor=C.+A.+French&title=Clinical+response+of+carcinomas+harboring+the+BRD4-NUT+oncoprotein+to+the+targeted+bromodomain+inhibitor+OTX015%2FMK-8628&doi=10.1158%2F2159-8290.CD-15-1335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Response of Carcinomas Harboring the BRD4-NUT Oncoprotein to the Targeted Bromodomain Inhibitor OTX015/MK-8628</span></div><div class="casAuthors">Stathis, Anastasios; Zucca, Emanuele; Bekradda, Mohamed; Gomez-Roca, Carlos; Delord, Jean-Pierre; de La Motte Rouge, Thibault; Uro-Coste, Emmanuelle; de Braud, Filippo; Pelosi, Giuseppe; French, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">492-500</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The antineoplastic, prodifferentiative effects of bromodomain and extra-terminal (BET) bromodomain (BRD) inhibitors were initially discovered in NUT midline carcinoma (NMC), an aggressive subtype of squamous cancer driven by the BRD4-NUT fusion oncoprotein.  BRD4-NUT blocks differentiation and maintains tumor growth through a potent chromatin-modifying mechanism.  OTX015/MK-8628, a novel oral BET inhibitor, targets BRD2/3/4/T with preclin. activity in NMC and several other tumor types and is currently in clin. development.  Antitumor activity was evaluated in four patients with advanced-stage NMC with confirmed BRD4-NUT fusions who were treated with 80 mg OTX015/MK-8628 once daily in a compassionate-use context.  Two patients responded rapidly with tumor regression and symptomatic relief, and a third had meaningful disease stabilization with a minor metabolic response.  The main side effects were mild to moderate gastrointestinal toxicity and fatigue, and reversible grade 3 thrombocytopenia.  This is the first proof-of-concept evidence of clin. activity of a BRD inhibitor in targeting BRD4-NUT.  Significance: We present the first clin. proof-of-concept that targeting BRD4-NUT with a BET inhibitor results in impressive and rapid antitumor activity in NMC.  It offers strong potential for future clin. application in this rare patient population as either a single agent or in combination with other agents.  Cancer Discov; 6(5); 492-500. Â©2016 AACR.  This article is highlighted in the In This Issue feature, p.  461.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGqXhyj-x5pLVg90H21EOLACvtfcHk0lg_uD7EiFA5Aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlOqsrs%253D&md5=73f6d61463bc6db72178d70771544787</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1335%26sid%3Dliteratum%253Aachs%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DGomez-Roca%26aufirst%3DC.%26aulast%3DDelord%26aufirst%3DJ.-P.%26aulast%3Dde%2BLa%2BMotte%2BRouge%26aufirst%3DT.%26aulast%3DUro-Coste%26aufirst%3DE.%26aulast%3Dde%2BBraud%26aufirst%3DF.%26aulast%3DPelosi%26aufirst%3DG.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26atitle%3DClinical%2520response%2520of%2520carcinomas%2520harboring%2520the%2520BRD4-NUT%2520oncoprotein%2520to%2520the%2520targeted%2520bromodomain%2520inhibitor%2520OTX015%252FMK-8628%26jtitle%3DCancer%2520Discovery%26date%3D2016%26volume%3D6%26spage%3D492%26epage%3D500%26doi%3D10.1158%2F2159-8290.CD-15-1335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>â <span class="NLM_lpage">1123</span>, <span class="refDoi">Â DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lg_uD7EiFA5Aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ClÃ©ment, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnathan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">NicodÃ©me, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>â <span class="NLM_lpage">7515</span>, <span class="refDoi">Â DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=M.+Barnathanauthor=J.-M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.-W.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A9me&title=Discovery+of+epigenetic+regulator+I-BET762%3A+lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0lg_uD7EiFA5Aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A9me%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CÃ´tÃ©, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâMeara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span> <span> </span><span class="NLM_article-title">Identification of a benzoisoxazoloazepine inhibitor (CPI-0610) of the bromodomain and extra-terminal (BET) family as a candidate for human clinical trials</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1330</span>â <span class="NLM_lpage">1339</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01882</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01882" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1330-1339&author=B.+K.+Albrechtauthor=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=R.+G.+Vaswaniauthor=A.+C%C3%B4t%C3%A9author=Y.+Leblancauthor=C.+G.+Nasveschukauthor=S.+Bellonauthor=L.+Bergeronauthor=R.+Campbellauthor=N.+Cantoneauthor=M.+R.+Cooperauthor=R.+T.+Cummingsauthor=H.+Jayaramauthor=S.+Joshiauthor=J.+A.+Mertzauthor=A.+Neissauthor=E.+Normantauthor=M.+O%E2%80%99Mearaauthor=E.+Pardoauthor=F.+Poyauthor=P.+Sandyauthor=J.+Supkoauthor=R.+J.+Simsauthor=J.-C.+Harmangeauthor=A.+M.+Taylorauthor=J.+E.+Audia&title=Identification+of+a+benzoisoxazoloazepine+inhibitor+%28CPI-0610%29+of+the+bromodomain+and+extra-terminal+%28BET%29+family+as+a+candidate+for+human+clinical+trials&doi=10.1021%2Facs.jmedchem.5b01882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Benzoisoxazoloazepine Inhibitor (CPI-0610) of the Bromodomain and Extra-Terminal (BET) Family as a Candidate for Human Clinical Trials</span></div><div class="casAuthors">Albrecht, Brian K.; Gehling, Victor S.; Hewitt, Michael C.; Vaswani, Rishi G.; Cote, Alexandre; Leblanc, Yves; Nasveschuk, Christopher G.; Bellon, Steve; Bergeron, Louise; Campbell, Robert; Cantone, Nico; Cooper, Michael R.; Cummings, Richard T.; Jayaram, Hariharan; Joshi, Shivangi; Mertz, Jennifer A.; Neiss, Adrianne; Normant, Emmanuel; O'Meara, Michael; Pardo, Eneida; Poy, Florence; Sandy, Peter; Supko, Jeffrey; Sims, Robert J.; Harmange, Jean-Christophe; Taylor, Alexander M.; Audia, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1330-1339</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, inhibition of the interaction between the bromodomain and extra-terminal domain (BET) family of chromatin adaptors and acetyl-lysine residues on chromatin has emerged as a promising approach to regulate the expression of important disease-relevant genes, including MYC, BCL-2, and NF-ÎºB.  Here we describe the identification and characterization of a potent and selective benzoisoxazoloazepine BET bromodomain inhibitor that attenuates BET-dependent gene expression in vivo, demonstrates antitumor efficacy in an MV-4-11 mouse xenograft model, and is currently undergoing human clin. trials for hematol. malignancies (CPI-0610).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp40qjnYl4EZLVg90H21EOLACvtfcHk0ljhfBcKg8nZEg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D&md5=5e1c5aab26aa58844a9f6670a7bbd78b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01882%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DCooper%26aufirst%3DM.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DNormant%26aufirst%3DE.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSupko%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26atitle%3DIdentification%2520of%2520a%2520benzoisoxazoloazepine%2520inhibitor%2520%2528CPI-0610%2529%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520family%2520as%2520a%2520candidate%2520for%2520human%2520clinical%2520trials%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1330%26epage%3D1339%26doi%3D10.1021%2Facs.jmedchem.5b01882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donawho, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlhapp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Preclinical characterization of BET family bromodomain inhibitor ABBV-075 suggests combination therapeutic strategies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2976</span>â <span class="NLM_lpage">2989</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-16-1793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1158%2F0008-5472.CAN-16-1793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=28416490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Siurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2976-2989&author=M.+H.+Buiauthor=X.+Linauthor=D.+H.+Albertauthor=L.+Liauthor=L.+T.+Lamauthor=E.+J.+Faivreauthor=S.+E.+Warderauthor=X.+Huangauthor=D.+Wilcoxauthor=C.+K.+Donawhoauthor=G.+S.+Sheppardauthor=L.+Wangauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+Hasvoldauthor=T.+Uzielauthor=X.+Luauthor=F.+Kohlhappauthor=G.+Fangauthor=S.+W.+Elmoreauthor=S.+H.+Rosenbergauthor=K.+F.+McDanielauthor=W.+M.+Katiauthor=Y.+Shen&title=Preclinical+characterization+of+BET+family+bromodomain+inhibitor+ABBV-075+suggests+combination+therapeutic+strategies&doi=10.1158%2F0008-5472.CAN-16-1793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical Characterization of BET Family Bromodomain Inhibitor ABBV-075 Suggests Combination Therapeutic Strategies</span></div><div class="casAuthors">Bui, Mai H.; Lin, Xiaoyu; Albert, Daniel H.; Li, Leiming; Lam, Lloyd T.; Faivre, Emily J.; Warder, Scott E.; Huang, Xiaoli; Wilcox, Denise; Donawho, Cherrie K.; Sheppard, George S.; Wang, Le; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa; Uziel, Tamar; Lu, Xin; Kohlhapp, Fred; Fang, Guowei; Elmore, Steven W.; Rosenberg, Saul H.; McDaniel, Keith F.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2976-2989</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">ABBV-075 is a potent and selective BET family bromodomain inhibitor that recently entered phase I clin. trials.  Comprehensive preclin. characterization of ABBV-075 demonstrated broad activity across cell lines and tumor models, representing a variety of hematol. malignancies and solid tumor indications.  In most cancer cell lines derived from solid tumors, ABBV-075 triggers prominent G1 cell-cycle arrest without extensive apoptosis.  In this study, we show that ABBV-075 efficiently triggers apoptosis in acute myeloid leukemia (AML), non-Hodgkin lymphoma, and multiple myeloma cells.  Apoptosis induced by ABBV-075 was mediated in part by modulation of the intrinsic apoptotic pathway, exhibiting synergy with the BCL-2 inhibitor venetoclax in preclin. models of AML.  In germinal center diffuse large B-cell lymphoma, BCL-2 levels or venetoclax sensitivity predicted the apoptotic response to ABBV-075 treatment.  In vivo combination studies uncovered surprising benefits of low doses of ABBV-075 coupled with bortezomib and azacitidine treatment, despite the lack of in vitro synergy between ABBV-075 and these agents.  The in vitro/in vivo activities of ABBV-075 described here may serve as a useful ref. to guide the development of ABBV-075 and other BET family inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6P1blFwkIvrVg90H21EOLACvtfcHk0ljhfBcKg8nZEg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Siurw%253D&md5=d9cc4abc44cfd9c0e07383f908592c2e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-1793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-1793%26sid%3Dliteratum%253Aachs%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DDonawho%26aufirst%3DC.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DKohlhapp%26aufirst%3DF.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DPreclinical%2520characterization%2520of%2520BET%2520family%2520bromodomain%2520inhibitor%2520ABBV-075%2520suggests%2520combination%2520therapeutic%2520strategies%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2976%26epage%3D2989%26doi%3D10.1158%2F0008-5472.CAN-16-1793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-(4-(2,4-difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfon amide (ABBV-075/Mivebresib), a potent and orally available bromodomain and extraterminal domain (BET) family bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8369</span>â <span class="NLM_lpage">8384</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00746</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00746" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8369-8384&author=K.+F.+McDanielauthor=L.+Wangauthor=T.+Soltwedelauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=R.+A.+Manteiauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=M.+H.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=L.+Liauthor=X.+Linauthor=R.+Wangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=J.+J.+Shenauthor=R.+P.+Edaljiauthor=C.+C.+Sunauthor=R.+Martinauthor=W.+Gaoauthor=S.+Wongauthor=G.+Fangauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Kati&title=Discovery+of+N-%284-%282%2C4-difluorophenoxy%29-3-%286-methyl-7-oxo-6%2C7-dihydro-1H-pyrrolo%5B2%2C3-c%5Dpyridin-4-yl%29phenyl%29ethanesulfon+amide+%28ABBV-075%2FMivebresib%29%2C+a+potent+and+orally+available+bromodomain+and+extraterminal+domain+%28BET%29+family+bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.7b00746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span></div><div class="casAuthors">McDaniel, Keith F.; Wang, Le; Soltwedel, Todd; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Mantei, Robert A.; Pratt, John K.; Sheppard, George S.; Bui, Mai H.; Faivre, Emily J.; Huang, Xiaoli; Li, Leiming; Lin, Xiaoyu; Wang, Rongqi; Warder, Scott E.; Wilcox, Denise; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Park, Chang H.; Hutchins, Charles W.; Shen, Jianwei J.; Edalji, Rohinton P.; Sun, Chaohong C.; Martin, Ruth; Gao, Wenqing; Wong, Shekman; Fang, Guowei; Elmore, Steven W.; Shen, Yu; Kati, Warren M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8369-8384</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of bromodomain and extraterminal domain (BET) bromodomain inhibitors and their examn. in clin. studies, particularly in oncol. settings, has garnered substantial recent interest.  An effort to generate novel BET bromodomain inhibitors with excellent potency and drug metab. and pharmacokinetics (DMPK) properties was initiated based upon elaboration of a simple pyridone core.  Efforts to develop a bidentate interaction with a crit. asparagine residue resulted in the incorporation of a pyrrolopyridone core, which improved potency by 9-19-fold.  Addnl. structure-activity relationship (SAR) efforts aimed both at increasing potency and improving pharmacokinetic properties led to the discovery of the clin. candidate I (ABBV-075/mivebresib), which demonstrates excellent potency in biochem. and cellular assays, advantageous exposures and half-life both in animal models and in humans, and in vivo efficacy in mouse models of cancer progression and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom-8BkipNknbVg90H21EOLACvtfcHk0lgUn83Gq2-KqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF&md5=16ea9e9ade8a22a6e99aeae16fd1bf9c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DShen%26aufirst%3DJ.%2BJ.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25284-%25282%252C4-difluorophenoxy%2529-3-%25286-methyl-7-oxo-6%252C7-dihydro-1H-pyrrolo%255B2%252C3-c%255Dpyridin-4-yl%2529phenyl%2529ethanesulfon%2520amide%2520%2528ABBV-075%252FMivebresib%2529%252C%2520a%2520potent%2520and%2520orally%2520available%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520family%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8369%26epage%3D8384%26doi%3D10.1021%2Facs.jmedchem.7b00746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, Y.</span></span> <span> </span><span class="NLM_article-title">The bromodomain and extra-terminal domain (BET) family: functional anatomy of BET paralogous proteins</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1849</span>, <span class="refDoi">Â DOI: 10.3390/ijms17111849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.3390%2Fijms17111849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXot1Ohsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=1849&author=Y.+Taniguchi&title=The+bromodomain+and+extra-terminal+domain+%28BET%29+family%3A+functional+anatomy+of+BET+paralogous+proteins&doi=10.3390%2Fijms17111849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain and extra-terminal domain (BET) family: functional anatomy of BET paralogous proteins</span></div><div class="casAuthors">Taniguchi, Yasushi</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1849/1-1849/24</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The Bromodomain and Extra-Terminal Domain (BET) family of proteins is characterized by the presence of two tandem bromodomains and an extra-terminal domain.  The mammalian BET family of proteins comprises BRD2, BRD3, BRD4, and BRDT, which are encoded by paralogous genes that may have been generated by repeated duplication of an ancestral gene during evolution.  Bromodomains that can specifically bind acetylated lysine residues in histones serve as chromatin-targeting modules that decipher the histone acetylation code.  BET proteins play a crucial role in regulating gene transcription through epigenetic interactions between bromodomains and acetylated histones during cellular proliferation and differentiation processes.  On the other hand, BET proteins have been reported to mediate latent viral infection in host cells and be involved in oncogenesis.  Human BRD4 is involved in multiple processes of the DNA virus life cycle, including viral replication, genome maintenance, and gene transcription through interaction with viral proteins.  Aberrant BRD4 expression contributes to carcinogenesis by mediating hyperacetylation of the chromatin contg. the cell proliferation-promoting genes.  BET bromodomain blockade using small-mol. inhibitors gives rise to selective repression of the transcriptional network driven by c-MYC These inhibitors are expected to be potential therapeutic drugs for a wide range of cancers.  This review presents an overview of the basic roles of BET proteins and highlights the pathol. functions of BET and the recent developments in cancer therapy targeting BET proteins in animal models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKHOsfPnlUXbVg90H21EOLACvtfcHk0lgUn83Gq2-KqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXot1Ohsg%253D%253D&md5=fe82c03d3e5ffba8cbb0ad51e5fc19e1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3390%2Fijms17111849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms17111849%26sid%3Dliteratum%253Aachs%26aulast%3DTaniguchi%26aufirst%3DY.%26atitle%3DThe%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520family%253A%2520functional%2520anatomy%2520of%2520BET%2520paralogous%2520proteins%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2016%26volume%3D17%26spage%3D1849%26doi%3D10.3390%2Fijms17111849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchrÃ¶der, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaehlcke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisgrove, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SchnÃ¶lzer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, M.</span></span> <span> </span><span class="NLM_article-title">Two-pronged binding with bromodomain-containing protein 4 liberates positive transcription elongation factor b from inactive ribonucleoprotein complexes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">1090</span>â <span class="NLM_lpage">1099</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M111.282855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1074%2Fjbc.M111.282855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=22084242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A280%3ADC%252BC387is1answ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=1090-1099&author=S.+Schr%C3%B6derauthor=S.+Choauthor=L.+Zengauthor=Q.+Zhangauthor=K.+Kaehlckeauthor=L.+Makauthor=J.+Lauauthor=D.+Bisgroveauthor=M.+Schn%C3%B6lzerauthor=E.+Verdinauthor=M.-M.+Zhouauthor=M.+Ott&title=Two-pronged+binding+with+bromodomain-containing+protein+4+liberates+positive+transcription+elongation+factor+b+from+inactive+ribonucleoprotein+complexes&doi=10.1074%2Fjbc.M111.282855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Two-pronged binding with bromodomain-containing protein 4 liberates positive transcription elongation factor b from inactive ribonucleoprotein complexes</span></div><div class="casAuthors">Schroder Sebastian; Cho Sungyoo; Zeng Lei; Zhang Qiang; Kaehlcke Katrin; Mak Lily; Lau Joann; Bisgrove Dwayne; Schnolzer Martina; Verdin Eric; Zhou Ming-Ming; Ott Melanie</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of biological chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1090-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The positive transcription elongation factor b (P-TEFb) exists in two forms in cells as follows: an inactive form where the core components cyclin T1 and CDK9 are incorporated in the 7SK small nuclear ribonucleoprotein complex containing the inhibitory molecule HEXIM1, and an active form, part of which associates with the bromodomain-containing protein BRD4.  Here, we define a novel interaction between P-TEFb and BRD4 involving tri-acetylated cyclin T1 (acK380, acK386, and acK309) and the second bromodomain in BRD4.  This interaction is observed with the short splice variant of BRD4 (amino acids 1-722) lacking a previously defined C-terminal P-TEFb-interacting domain (PID).  Notably, P-TEFb complexes associated with short BRD4 contain HEXIM1 and 7SK snRNA, implicating the PID in the liberation of P-TEFb from the 7SK small nuclear ribonucleoprotein complex (7SK snPNP).  Overexpression of the PID alone in cells dissociates HEXIM1 and 7SK snRNA from P-TEFb, but it is not sufficient to activate P-TEFb-dependent transcription of the HIV LTR.  Our data support a model where two BRD4 domains, the second bromodomain and the PID, bind P-TEFb and are required for full transcriptional activation of P-TEFb response genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSTv2tSJrzcEK69lAbTj9rGfW6udTcc2eZjEP1KYNqwibntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC387is1answ%253D%253D&md5=85209cc1f667ed47b97e626dcb2156ec</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.282855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.282855%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6der%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DS.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKaehlcke%26aufirst%3DK.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DBisgrove%26aufirst%3DD.%26aulast%3DSchn%25C3%25B6lzer%26aufirst%3DM.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DTwo-pronged%2520binding%2520with%2520bromodomain-containing%2520protein%25204%2520liberates%2520positive%2520transcription%2520elongation%2520factor%2520b%2520from%2520inactive%2520ribonucleoprotein%2520complexes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D1090%26epage%3D1099%26doi%3D10.1074%2Fjbc.M111.282855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delmore, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issa, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilpatrick, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinzel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffernan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsagel, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">904</span>â <span class="NLM_lpage">917</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+C.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0ljBNRlOte7_MA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%2BC.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B. P.</span></span> <span> </span><span class="NLM_article-title">Disrupting the interaction of BRD4 with diacetylated Twist suppresses tumorigenesis in basal-like breast cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">210</span>â <span class="NLM_lpage">225</span>, <span class="refDoi">Â DOI: 10.1016/j.ccr.2014.01.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.ccr.2014.01.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=24525235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=210-225&author=J.+Shiauthor=Y.+Wangauthor=L.+Zengauthor=Y.+Wuauthor=J.+Dengauthor=Q.+Zhangauthor=Y.+Linauthor=J.+Liauthor=T.+Kangauthor=M.+Taoauthor=E.+Rusinovaauthor=G.+Zhangauthor=C.+Wangauthor=H.+Zhuauthor=J.+Yaoauthor=Y.+X.+Zengauthor=B.+M.+Eversauthor=M.+M.+Zhouauthor=B.+P.+Zhou&title=Disrupting+the+interaction+of+BRD4+with+diacetylated+Twist+suppresses+tumorigenesis+in+basal-like+breast+cancer&doi=10.1016%2Fj.ccr.2014.01.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting the Interaction of BRD4 with Diacetylated Twist Suppresses Tumorigenesis in Basal-like Breast Cancer</span></div><div class="casAuthors">Shi, Jian; Wang, Yifan; Zeng, Lei; Wu, Yadi; Deng, Jiong; Zhang, Qiang; Lin, Yiwei; Li, Junlin; Kang, Tiebang; Tao, Min; Rusinova, Elena; Zhang, Guangtao; Wang, Chi; Zhu, Haining; Yao, Jun; Zeng, Yi-Xin; Evers, B. Mark; Zhou, Ming-Ming; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">210-225</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Twist is a key transcription activator of epithelial-mesenchymal transition (EMT).  It remains unclear how Twist induces gene expression.  Here we report a mechanism by which Twist recruits BRD4 to direct WNT5A expression in basal-like breast cancer (BLBC).  Twist contains a "histone H4-mimic" GK-X-GK motif that is diacetylated by Tip60.  The diacetylated Twist binds the second bromodomain of BRD4, whose first bromodomain interacts with acetylated H4, thereby constructing an activated Twist/BRD4/P-TEFb/RNA-Pol II complex at the WNT5A promoter and enhancer.  Pharmacol. inhibition of the Twist-BRD4 assocn. reduced WNT5A expression and suppressed invasion, cancer stem cell (CSC)-like properties, and tumorigenicity of BLBC cells.  Our study indicates that the interaction with BRD4 is crit. for the oncogenic function of Twist in BLBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovIzriI8A6ZrVg90H21EOLACvtfcHk0ljBNRlOte7_MA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D&md5=d207cbc1abb187a53d807452b4f10d8f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.01.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.01.028%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DTao%26aufirst%3DM.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DY.%2BX.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DDisrupting%2520the%2520interaction%2520of%2520BRD4%2520with%2520diacetylated%2520Twist%2520suppresses%2520tumorigenesis%2520in%2520basal-like%2520breast%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D210%26epage%3D225%26doi%3D10.1016%2Fj.ccr.2014.01.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, K.</span></span> <span> </span><span class="NLM_article-title">Brd4 binds to active enhancers to control cell identity gene induction in adipogenesis and myogenesis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2217</span>, <span class="refDoi">Â DOI: 10.1038/s41467-017-02403-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fs41467-017-02403-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=29263365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A280%3ADC%252BC1MzjslSlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2217&author=J.-E.+Leeauthor=Y.-K.+Parkauthor=S.+Parkauthor=Y.+Jangauthor=N.+Waringauthor=A.+Deyauthor=K.+Ozatoauthor=B.+Laiauthor=W.+Pengauthor=K.+Ge&title=Brd4+binds+to+active+enhancers+to+control+cell+identity+gene+induction+in+adipogenesis+and+myogenesis&doi=10.1038%2Fs41467-017-02403-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 binds to active enhancers to control cell identity gene induction in adipogenesis and myogenesis</span></div><div class="casAuthors">Lee Ji-Eun; Park Young-Kwon; Park Sarah; Jang Younghoon; Waring Nicholas; Lai Binbin; Ge Kai; Waring Nicholas; Lai Binbin; Peng Weiqun; Dey Anup; Ozato Keiko</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2217</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The epigenomic reader Brd4 is an important drug target for cancers.  However, its role in cell differentiation and animal development remains largely unclear.  Using two conditional knockout mouse strains and derived cells, we demonstrate that Brd4 controls cell identity gene induction and is essential for adipogenesis and myogenesis.  Brd4 co-localizes with lineage-determining transcription factors (LDTFs) on active enhancers during differentiation.  LDTFs coordinate with H3K4 mono-methyltransferases MLL3/MLL4 (KMT2C/KMT2D) and H3K27 acetyltransferases CBP/p300 to recruit Brd4 to enhancers activated during differentiation.  Brd4 deletion prevents the enrichment of Mediator and RNA polymerase II transcription machinery, but not that of LDTFs, MLL3/MLL4-mediated H3K4me1, and CBP/p300-mediated H3K27ac, on enhancers.  Consequently, Brd4 deletion prevents enhancer RNA production, cell identity gene induction and cell differentiation.  Interestingly, Brd4 is dispensable for maintaining cell identity genes in differentiated cells.  These findings identify Brd4 as an enhancer epigenomic reader that links active enhancers with cell identity gene induction in differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREMjSSm_naSGZyxfu64m4ffW6udTcc2eZ_eRXkntA8d7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzjslSlsQ%253D%253D&md5=35aa3eb2ed0f9c1f38a6a2d09b438957</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02403-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02403-5%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.-E.%26aulast%3DPark%26aufirst%3DY.-K.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DY.%26aulast%3DWaring%26aufirst%3DN.%26aulast%3DDey%26aufirst%3DA.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DB.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DGe%26aufirst%3DK.%26atitle%3DBrd4%2520binds%2520to%2520active%2520enhancers%2520to%2520control%2520cell%2520identity%2520gene%2520induction%2520in%2520adipogenesis%2520and%2520myogenesis%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D2217%26doi%3D10.1038%2Fs41467-017-02403-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span>; <span class="NLM_string-name">Yang, J.</span>; <span class="NLM_string-name">Zhao, Y.</span>; <span class="NLM_string-name">Ivanciuc, T.</span>; <span class="NLM_string-name">Sun, H.</span>; <span class="NLM_string-name">Garofalo, R. P.</span>; <span class="NLM_string-name">Brasier, A. R.</span></span> <span> </span><span class="NLM_article-title">BRD4 couples NF-ÎºB/RelA with airway inflammation and the IRF-RIG-I amplification loop in respiratory syncytial virus infection</span>.  <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span><b>,</b>  <span class="NLM_volume">91</span>, <span class="refDoi">Â DOI: 10.1128/JVI.00007-17</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1128%2FJVI.00007-17" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2017&author=B.+Tian&author=J.+Yang&author=Y.+Zhao&author=T.+Ivanciuc&author=H.+Sun&author=R.+P.+Garofalo&author=A.+R.+Brasier&title=BRD4+couples+NF-%CE%BAB%2FRelA+with+airway+inflammation+and+the+IRF-RIG-I+amplification+loop+in+respiratory+syncytial+virus+infection&doi=10.1128%2FJVI.00007-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FJVI.00007-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00007-17%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DB.%26atitle%3DBRD4%2520couples%2520NF-%25CE%25BAB%252FRelA%2520with%2520airway%2520inflammation%2520and%2520the%2520IRF-RIG-I%2520amplification%2520loop%2520in%2520respiratory%2520syncytial%2520virus%2520infection%26jtitle%3DJ.%2520Virol.%26date%3D2017%26volume%3D91%26doi%3D10.1128%2FJVI.00007-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hogg, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vervoort, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deswal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cluse, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beavis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darcy, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traunbauer, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadovnik, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shortt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">BET-Bromodomain inhibitors engage the host immune system and regulate expression of the immune checkpoint ligand PD-L1</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2162</span>â <span class="NLM_lpage">2174</span>, <span class="refDoi">Â DOI: 10.1016/j.celrep.2017.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.celrep.2017.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=28249162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Ontb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=2162-2174&author=S.+J.+Hoggauthor=S.+J.+Vervoortauthor=S.+Deswalauthor=C.+J.+Ottauthor=J.+Liauthor=L.+A.+Cluseauthor=P.+A.+Beavisauthor=P.+K.+Darcyauthor=B.+P.+Martinauthor=A.+Spencerauthor=A.+K.+Traunbauerauthor=I.+Sadovnikauthor=K.+Bauerauthor=P.+Valentauthor=J.+E.+Bradnerauthor=J.+Zuberauthor=J.+Shorttauthor=R.+W.+Johnstone&title=BET-Bromodomain+inhibitors+engage+the+host+immune+system+and+regulate+expression+of+the+immune+checkpoint+ligand+PD-L1&doi=10.1016%2Fj.celrep.2017.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">BET-Bromodomain Inhibitors Engage the Host Immune System and Regulate Expression of the Immune Checkpoint Ligand PD-L1</span></div><div class="casAuthors">Hogg, Simon J.; Vervoort, Stephin J.; Deswal, Sumit; Ott, Christopher J.; Li, Jason; Cluse, Leonie A.; Beavis, Paul A.; Darcy, Phillip K.; Martin, Benjamin P.; Spencer, Andrew; Traunbauer, Anna K.; Sadovnik, Irina; Bauer, Karin; Valent, Peter; Bradner, James E.; Zuber, Johannes; Shortt, Jake; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2162-2174</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">BET inhibitors (BETi) target bromodomain-contg. proteins and are currently being evaluated as anti-cancer agents.  We find that maximal therapeutic effects of BETi in a Myc-driven B cell lymphoma model required an intact host immune system.  Genome-wide anal. of the BETi-induced transcriptional response identified the immune checkpoint ligand Cd274 (Pd-l1) as a Myc-independent, BETi target-gene.  BETi directly repressed constitutively expressed and interferon-gamma (IFN-Î³) induced CD274 expression across different human and mouse tumor cell lines and primary patient samples.  Mechanistically, BETi decreased Brd4 occupancy at the Cd274 locus without any change in Myc occupancy, resulting in transcriptional pausing and rapid loss of Cd274 mRNA prodn.  Finally, targeted inhibition of the PD-1/PD-L1 axis by combining anti-PD-1 antibodies and the BETi JQ1 caused synergistic responses in mice bearing Myc-driven lymphomas.  Our data uncover an interaction between BETi and the PD-1/PD-L1 immune-checkpoint and provide mechanistic insight into the transcriptional regulation of CD274.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo06NwFejZMmbVg90H21EOLACvtfcHk0lhF8u3rE8EMKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Ontb0%253D&md5=62eac948a07e94c39234ddd7015fd7e8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DHogg%26aufirst%3DS.%2BJ.%26aulast%3DVervoort%26aufirst%3DS.%2BJ.%26aulast%3DDeswal%26aufirst%3DS.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DCluse%26aufirst%3DL.%2BA.%26aulast%3DBeavis%26aufirst%3DP.%2BA.%26aulast%3DDarcy%26aufirst%3DP.%2BK.%26aulast%3DMartin%26aufirst%3DB.%2BP.%26aulast%3DSpencer%26aufirst%3DA.%26aulast%3DTraunbauer%26aufirst%3DA.%2BK.%26aulast%3DSadovnik%26aufirst%3DI.%26aulast%3DBauer%26aufirst%3DK.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShortt%26aufirst%3DJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DBET-Bromodomain%2520inhibitors%2520engage%2520the%2520host%2520immune%2520system%2520and%2520regulate%2520expression%2520of%2520the%2520immune%2520checkpoint%2520ligand%2520PD-L1%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D18%26spage%3D2162%26epage%3D2174%26doi%3D10.1016%2Fj.celrep.2017.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciarrocchi, A.</span></span> <span> </span><span class="NLM_article-title">BRD4 and cancer: going beyond transcriptional regulation</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">164</span>, <span class="refDoi">Â DOI: 10.1186/s12943-018-0915-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1186%2Fs12943-018-0915-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=30466442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlWgu78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=164&author=B.+Donatiauthor=E.+Lorenziniauthor=A.+Ciarrocchi&title=BRD4+and+cancer%3A+going+beyond+transcriptional+regulation&doi=10.1186%2Fs12943-018-0915-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 and Cancer: going beyond transcriptional regulation</span></div><div class="casAuthors">Donati, Benedetta; Lorenzini, Eugenia; Ciarrocchi, Alessia</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">164</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  BRD4, member of the Bromodomain and Extraterminal (BET) protein family, is largely acknowledged in cancer for its role in super-enhancers (SEs) organization and oncogenes expression regulation.  Inhibition of BRD4 shortcuts the communication between SEs and target promoters with a subsequent cell-specific repression of oncogenes to which cancer cells are addicted and cell death.  To date, this is the most credited mechanism of action of BET inhibitors, a class of small mols. targeting BET proteins which are currently in clin. trials in several cancer settings.  However, recent evidence indicates that BRD4 relevance in cancer goes beyond its role in transcription regulation and identifies this protein as a keeper of genome stability.  Indeed, a non-transcriptional role of BRD4 in controlling DNA damage checkpoint activation and repair as well as telomere maintenance has been proposed, throwing new lights into the multiple functions of this protein and opening new perspectives on the use of BETi in cancer.  Here we discuss the current available information on non-canonical, non-transcriptional functions of BRD4 and on their implications in cancer biol.  Integrating this information with the already known BRD4 role in gene expression regulation, we propose a "common" model to explain BRD4 genomic function.  Furthermore, in light of the transversal function of BRD4, we provide new interpretation for the cytotoxic activity of BETi and we discuss new possibilities for a wide and focused employment of these drugs in clin. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbYn0mnQW7ELVg90H21EOLACvtfcHk0lhF8u3rE8EMKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlWgu78%253D&md5=2ddfde1363d430aa67e15a52fdc2335e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1186%2Fs12943-018-0915-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-018-0915-9%26sid%3Dliteratum%253Aachs%26aulast%3DDonati%26aufirst%3DB.%26aulast%3DLorenzini%26aufirst%3DE.%26aulast%3DCiarrocchi%26aufirst%3DA.%26atitle%3DBRD4%2520and%2520cancer%253A%2520going%2520beyond%2520transcriptional%2520regulation%26jtitle%3DMol.%2520Cancer%26date%3D2018%26volume%3D17%26spage%3D164%26doi%3D10.1186%2Fs12943-018-0915-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Korb, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucker-Scharff, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">BET protein brd4 activates transcription in neurons and BET inhibitor Jq1 blocks memory in mice</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1464</span>â <span class="NLM_lpage">1473</span>, <span class="refDoi">Â DOI: 10.1038/nn.4095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnn.4095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=26301327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlynsL%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=1464-1473&author=E.+Korbauthor=M.+Herreauthor=I.+Zucker-Scharffauthor=R.+B.+Darnellauthor=C.+D.+Allis&title=BET+protein+brd4+activates+transcription+in+neurons+and+BET+inhibitor+Jq1+blocks+memory+in+mice&doi=10.1038%2Fnn.4095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">BET protein Brd4 activates transcription in neurons and BET inhibitor Jq1 blocks memory in mice</span></div><div class="casAuthors">Korb, Erica; Herre, Margo; Zucker-Scharff, Ilana; Darnell, Robert B.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1464-1473</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Precise regulation of transcription is crucial for the cellular mechanisms underlying memory formation.  However, the link between neuronal stimulation and the proteins that directly interact with histone modifications to activate transcription in neurons remains unclear.  Brd4 is a member of the bromodomain and extra-terminal domain (BET) protein family, which binds acetylated histones and is a crit. regulator of transcription in many cell types, including transcription in response to external cues.  Small mol. BET inhibitors are in clin. trials, yet almost nothing is known about Brd4 function in the brain.  Here we show that Brd4 mediates the transcriptional regulation underlying learning and memory.  The loss of Brd4 function affects crit. synaptic proteins, which results in memory deficits in mice but also decreases seizure susceptibility.  Thus Brd4 provides a crit. link between neuronal activation and the transcriptional responses that occur during memory formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbuMwCm6NDBLVg90H21EOLACvtfcHk0lji2KD-c37ldw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlynsL%252FN&md5=0a09dea39378424cfc003a1e9e40ab9a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnn.4095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.4095%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DE.%26aulast%3DHerre%26aufirst%3DM.%26aulast%3DZucker-Scharff%26aufirst%3DI.%26aulast%3DDarnell%26aufirst%3DR.%2BB.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DBET%2520protein%2520brd4%2520activates%2520transcription%2520in%2520neurons%2520and%2520BET%2520inhibitor%2520Jq1%2520blocks%2520memory%2520in%2520mice%26jtitle%3DNat.%2520Neurosci.%26date%3D2015%26volume%3D18%26spage%3D1464%26epage%3D1473%26doi%3D10.1038%2Fnn.4095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrieu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">Clinical trials for BET inhibitors run ahead of the science</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">45</span>â <span class="NLM_lpage">50</span>, <span class="refDoi">Â DOI: 10.1016/j.ddtec.2016.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.ddtec.2016.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=27769357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=45-50&author=G.+Andrieuauthor=A.+C.+Belkinaauthor=G.+V.+Denis&title=Clinical+trials+for+BET+inhibitors+run+ahead+of+the+science&doi=10.1016%2Fj.ddtec.2016.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical trials for BET inhibitors run ahead of the science</span></div><div class="casAuthors">Andrieu Guillaume; Belkina Anna C; Denis Gerald V</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Several cancer clinical trials for small molecule inhibitors of BET bromodomain proteins have been initiated.  There is enthusiasm for the anti-proliferative effect of inhibiting BRD4, one of the targets of these inhibitors, which is thought to cooperate with MYC, a long-desired target for cancer therapeutics.  However, no current inhibitor is selective for BRD4 among the three somatic BET proteins, which include BRD2 and BRD3; their respective functions are partially overlapping and none are functionally redundant with BRD4.  Each BET protein controls distinct transcriptional pathways that are important for functions beyond cancer cell proliferation, including insulin production, cytokine gene transcription, T cell differentiation, adipogenesis and most seriously, active repression of dangerous latent viruses like HIV.  BET inhibitors have been shown to reactivate HIV in human cells.  Failure to appreciate that at concentrations used, no available BET inhibitor is member-selective, or to develop a sound biological basis to understand the diverse functions of BET proteins before undertaking for these clinical trials is reckless and likely to lead to adverse events.  More mechanistic information from new basic science studies should enable proper focus on the most relevant cancers and define the expected side effect profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEothR-V9Y-Lt7lRqEFUJVfW6udTcc2eYme7qrodU7Obntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D&md5=5fcc9e96fa590221fb07c5e98ae91d59</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2016.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2016.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DAndrieu%26aufirst%3DG.%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DClinical%2520trials%2520for%2520BET%2520inhibitors%2520run%2520ahead%2520of%2520the%2520science%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2016%26volume%3D19%26spage%3D45%26epage%3D50%26doi%3D10.1016%2Fj.ddtec.2016.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, L.</span></span> <span> </span><span class="NLM_article-title">A phase Ib trial with MK-8628/OTX015, a small molecule inhibitor of bromodomain (BRD) and extra-terminal (BET) proteins, in patients with selected advanced solid tumors</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">S2</span>â <span class="NLM_lpage">S3</span>, <span class="refDoi">Â DOI: 10.1016/S0959-8049(16)32609-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2FS0959-8049%2816%2932609-0" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2016&pages=S2-S3&author=C.+Massardauthor=J.+C.+Soriaauthor=A.+Stathisauthor=J.+P.+Delordauthor=A.+Awadaauthor=S.+Petersauthor=J.+Lewinauthor=M.+Bekraddaauthor=K.+Rezaiauthor=Z.+Zengauthor=H.+Azherauthor=S.+Perezauthor=L.+Siu&title=A+phase+Ib+trial+with+MK-8628%2FOTX015%2C+a+small+molecule+inhibitor+of+bromodomain+%28BRD%29+and+extra-terminal+%28BET%29+proteins%2C+in+patients+with+selected+advanced+solid+tumors&doi=10.1016%2FS0959-8049%2816%2932609-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0959-8049%2816%2932609-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-8049%252816%252932609-0%26sid%3Dliteratum%253Aachs%26aulast%3DMassard%26aufirst%3DC.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DDelord%26aufirst%3DJ.%2BP.%26aulast%3DAwada%26aufirst%3DA.%26aulast%3DPeters%26aufirst%3DS.%26aulast%3DLewin%26aufirst%3DJ.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DAzher%26aufirst%3DH.%26aulast%3DPerez%26aufirst%3DS.%26aulast%3DSiu%26aufirst%3DL.%26atitle%3DA%2520phase%2520Ib%2520trial%2520with%2520MK-8628%252FOTX015%252C%2520a%2520small%2520molecule%2520inhibitor%2520of%2520bromodomain%2520%2528BRD%2529%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%252C%2520in%2520patients%2520with%2520selected%2520advanced%2520solid%2520tumors%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2016%26volume%3D69%26spage%3DS2%26epage%3DS3%26doi%3D10.1016%2FS0959-8049%2816%2932609-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abramson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flinn, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maris, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâMeara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span> <span> </span><span class="NLM_article-title">BET inhibitor CPI-0610 is well tolerated and induces responses in diffuse large B-Cell lymphoma and follicular lymphoma: preliminary analysis of an ongoing phase 1 study</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1491</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1182%2Fblood.V126.23.1491.1491" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2015&pages=1491&author=J.+S.+Abramsonauthor=K.+A.+Blumauthor=I.+W.+Flinnauthor=M.+Gutierrezauthor=A.+Goyauthor=M.+Marisauthor=M.+Cooperauthor=M.+O%E2%80%99Mearaauthor=D.+Borgerauthor=J.+Mertzauthor=R.+J.+Simsauthor=S.+Jeffreyauthor=A.+Younes&title=BET+inhibitor+CPI-0610+is+well+tolerated+and+induces+responses+in+diffuse+large+B-Cell+lymphoma+and+follicular+lymphoma%3A+preliminary+analysis+of+an+ongoing+phase+1+study"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1182%2Fblood.V126.23.1491.1491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V126.23.1491.1491%26sid%3Dliteratum%253Aachs%26aulast%3DAbramson%26aufirst%3DJ.%2BS.%26aulast%3DBlum%26aufirst%3DK.%2BA.%26aulast%3DFlinn%26aufirst%3DI.%2BW.%26aulast%3DGutierrez%26aufirst%3DM.%26aulast%3DGoy%26aufirst%3DA.%26aulast%3DMaris%26aufirst%3DM.%26aulast%3DCooper%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DBorger%26aufirst%3DD.%26aulast%3DMertz%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DJeffrey%26aufirst%3DS.%26aulast%3DYounes%26aufirst%3DA.%26atitle%3DBET%2520inhibitor%2520CPI-0610%2520is%2520well%2520tolerated%2520and%2520induces%2520responses%2520in%2520diffuse%2520large%2520B-Cell%2520lymphoma%2520and%2520follicular%2520lymphoma%253A%2520preliminary%2520analysis%2520of%2520an%2520ongoing%2520phase%25201%2520study%26jtitle%3DBlood%26date%3D2015%26volume%3D126%26spage%3D1491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span>D;, S.
N. H. B. S. <span> </span><span class="NLM_article-title">New Dihydroquinoxalinone Compounds are
Bromodomain-containing Protein 4 Inhibitors</span>. <span class="NLM_patent">2014095775</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=D%3B%2C+S.%0AN.+H.+B.+S.+New+Dihydroquinoxalinone+Compounds+are%0ABromodomain-containing+Protein+4+Inhibitors.+2014095775%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26atitle%3DNew%2520Dihydroquinoxalinone%2520Compounds%2520are%250ABromodomain-containing%2520Protein%25204%2520Inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span>D, S.
N. H. B. S. <span> </span><span class="NLM_article-title">New Dihydropyridopyrazinone Derivatives
Useful as Drug for Treatment of Tumor Diseases</span>. <span class="NLM_patent">2015004075</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=D%2C+S.%0AN.+H.+B.+S.+New+Dihydropyridopyrazinone+Derivatives%0AUseful+as+Drug+for+Treatment+of+Tumor+Diseases.+2015004075%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26atitle%3DNew%2520Dihydropyridopyrazinone%2520Derivatives%250AUseful%2520as%2520Drug%2520for%2520Treatment%2520of%2520Tumor%2520Diseases%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortopassi, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pryde, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">A series of potent CREBBP bromodomain ligands reveals an induced-fit pocket stabilized by a Cation-Ï interaction</span>. <i>Angew. Chem. Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6126</span>â <span class="NLM_lpage">6130</span>, <span class="refDoi">Â DOI: 10.1002/anie.201402750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1002%2Fanie.201402750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6126-6130&author=T.+P.+C.+Rooneyauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=W.+A.+Cortopassiauthor=D.+A.+Hayauthor=S.+Martinauthor=A.+Tumberauthor=C.+M.+Rogersauthor=M.+Philpottauthor=M.+Wangauthor=A.+L.+Thompsonauthor=T.+D.+Heightmanauthor=D.+C.+Prydeauthor=A.+Cookauthor=R.+S.+Patonauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=A+series+of+potent+CREBBP+bromodomain+ligands+reveals+an+induced-fit+pocket+stabilized+by+a+Cation-%CF%80+interaction&doi=10.1002%2Fanie.201402750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A Series of Potent CREBBP Bromodomain Ligands Reveals an Induced-Fit Pocket Stabilized by a Cation-Ï Interaction</span></div><div class="casAuthors">Rooney, Timothy P. C.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Cortopassi, Wilian A.; Hay, Duncan A.; Martin, Sarah; Tumber, Anthony; Rogers, Catherine M.; Philpott, Martin; Wang, Minghua; Thompson, Amber L.; Heightman, Tom D.; Pryde, David C.; Cook, Andrew; Paton, Robert S.; Mueller, Susanne; Knapp, Stefan; Brennan, Paul E.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6126-6130</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Benzoxazinone and dihydroquinoxalinone fragments were employed as novel acetyl lysine mimics in the development of CREBBP bromodomain ligands.  While the benzoxazinone series showed low affinity for the CREBBP bromodomain, expansion of the dihydroquinoxalinone series resulted in the first potent inhibitors, e.g., I, of a bromodomain outside the BET family.  Structural and computational studies revealed that an internal hydrogen bond stabilized the protein-bound conformation of the dihydroquinoxalinone series.  The side chain of this series binds in an induced-fit pocket forming a cation-Ï interaction with R1173 of CREBBP.  Compd. I, the most potent compd., inhibited binding of CREBBP to chromatin in U2OS cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHFgTVUKe7lrVg90H21EOLACvtfcHk0ljcKO9vjQpJwg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D&md5=42b12e1fe855d2af02c49f8e1031d39e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201402750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201402750%26sid%3Dliteratum%253Aachs%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DA.%2BL.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DCook%26aufirst%3DA.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DA%2520series%2520of%2520potent%2520CREBBP%2520bromodomain%2520ligands%2520reveals%2520an%2520induced-fit%2520pocket%2520stabilized%2520by%2520a%2520Cation-%25CF%2580%2520interaction%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D6126%26epage%3D6130%26doi%3D10.1002%2Fanie.201402750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damaneh, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Structure-based optimization of a series of selective BET inhibitors containing aniline or indoline groups</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">156</span>â <span class="NLM_lpage">175</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2018.02.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.ejmech.2018.02.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=29525435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktlylsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2018&pages=156-175&author=J.+Huauthor=Y.+Wangauthor=Y.+Liauthor=D.+Caoauthor=L.+Xuauthor=S.+Songauthor=M.+S.+Damanehauthor=J.+Liauthor=Y.+Chenauthor=X.+Wangauthor=L.+Chenauthor=J.+Shenauthor=Z.+Miaoauthor=B.+Xiong&title=Structure-based+optimization+of+a+series+of+selective+BET+inhibitors+containing+aniline+or+indoline+groups&doi=10.1016%2Fj.ejmech.2018.02.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based optimization of a series of selective BET inhibitors containing aniline or indoline groups</span></div><div class="casAuthors">Hu, Jianping; Wang, Yingqing; Li, Yanlian; Cao, Danyan; Xu, Lin; Song, ShanShan; Damaneh, Mohammadali Soleimani; Li, Jian; Chen, Yuelei; Wang, Xin; Chen, Lin; Shen, Jingkang; Miao, Zehong; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">156-175</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recently, several kinase inhibitors were found to also inhibit bromodomains, providing a new strategy for the discovery of bromodomain inhibitors.  Along this line, starting from PLK1-BRD4 dual inhibitor BI-2536, we discovered a new series of dihydroquinoxalin-2(1H)-one with aniline and indoline WPF binders as selective BRD4 inhibitors.  They showed better BRD4-BD1 potency and negligible PLK1 kinase activity comparing with BI-2536.  Addnl., dihydroquinoxalin-2(1H)-ones contg. indoline group showed profound activities in mol. and cellular based assays.  Throughout the study, compds. 9, 28 and 37 showed significant inhibitory activity for c-Myc or PD-L1 protein expression and mRNA transcription both at concn. of 0.2 and 1 Î¼M.  Compd. 9 was found possessing the best balance of binding affinity, in vitro metabolic stability and in vivo pharmacokinetic properties.  Therefore, it was selected for in vivo pharmacol. study.  By using MM.1S cell derived xenograft model, we confirmed compd. 9 showed comparable in vivo tumor inhibition to phase II investigation drug I-BET762, which, together with the novel WPF binder, further indicated the utility of this series of BRD4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovKjmtiqOa9rVg90H21EOLACvtfcHk0ljcKO9vjQpJwg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktlylsLw%253D&md5=5d3ad7e741a140306d42ae2463dd667e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.02.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.02.070%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DDamaneh%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DStructure-based%2520optimization%2520of%2520a%2520series%2520of%2520selective%2520BET%2520inhibitors%2520containing%2520aniline%2520or%2520indoline%2520groups%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D150%26spage%3D156%26epage%3D175%26doi%3D10.1016%2Fj.ejmech.2018.02.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damaneh, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a series of dihydroquinoxalin-2(1H)-ones as selective BET inhibitors from a dual PLK1-BRD4 inhibitor</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">176</span>â <span class="NLM_lpage">195</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2017.05.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2Fj.ejmech.2017.05.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=28586718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWmt73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2017&pages=176-195&author=J.+Huauthor=Y.+Wangauthor=Y.+Liauthor=L.+Xuauthor=D.+Caoauthor=S.+Songauthor=M.+S.+Damanehauthor=X.+Wangauthor=T.+Mengauthor=Y.+L.+Chenauthor=J.+Shenauthor=Z.+Miaoauthor=B.+Xiong&title=Discovery+of+a+series+of+dihydroquinoxalin-2%281H%29-ones+as+selective+BET+inhibitors+from+a+dual+PLK1-BRD4+inhibitor&doi=10.1016%2Fj.ejmech.2017.05.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a series of dihydroquinoxalin-2(1H)-ones as selective BET inhibitors from a dual PLK1-BRD4 inhibitor</span></div><div class="casAuthors">Hu, Jianping; Wang, Yingqing; Li, Yanlian; Xu, Lin; Cao, Danyan; Song, Shan Shan; Damaneh, Mohammadali Soleimani; Wang, Xin; Meng, Tao; Chen, Yue-Lei; Shen, Jingkang; Miao, Zehong; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">176-195</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recent years have seen much effort to discover new chemotypes of BRD4 inhibitors.  Some kinase inhibitors are potent bromodomain inhibitors, esp. the PLK1 inhibitor BI-2536 and the JAK2 inhibitor TG101209, which can bind to BRD4 with IC50 values of 0.025 Î¼M and 0.13 Î¼M, resp.  Although the concept of dual inhibition is intriguing, selective BRD4 inhibitors are preferred as they may diminish off-target effects and provide more flexibility in anticancer drug combination therapy.  Inspired by BI-2536, the authors designed and prepd. a series of dihydroquinoxalin-2(1H)-one derivs. as selective bromodomain inhibitors.  The authors found compd. (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one had slightly higher activity than (+)-JQ1 in the fluorescence anisotropy assay and potent antiproliferative cellular activity in the MM.1S cell line.  The authors have successfully solved the cocrystal structure of (R)-4-cyclopropyl-1,3-dimethyl-6-(((S)-1-(p-tolyl)ethyl) amino)-3,4-dihydroquinoxalin-2(1H)-one in complex with BRD4-BD1, providing a solid structural basis for the binding mode of compds. of this series.  Compd. (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one exhibited high selectivity over most non-BET subfamily members and did not show bioactivity towards the PLK1 kinase at 10 or 1 Î¼M.  From in vivo studies, compd. (R)-4-Cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one demonstrated a good PK profile, and the results from in vivo pharmacol. studies clearly showed the efficacy of (R)-4-cyclopropyl-6-(((S)-1-(2,4-dimethylphenyl)ethyl)amino)-1,3-dimethyl-3,4-dihydroquinoxalin-2(1H)-one in the mouse MM.1S xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8GQjGV-VP7LVg90H21EOLACvtfcHk0lggqipv_VwPMg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWmt73I&md5=af59e5bd57adeede512b022e3641f4f2</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.05.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.05.049%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DDamaneh%26aufirst%3DM.%2BS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMeng%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%2BL.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520series%2520of%2520dihydroquinoxalin-2%25281H%2529-ones%2520as%2520selective%2520BET%2520inhibitors%2520from%2520a%2520dual%2520PLK1-BRD4%2520inhibitor%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D137%26spage%3D176%26epage%3D195%26doi%3D10.1016%2Fj.ejmech.2017.05.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshioka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fountain, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheenstra, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strovel, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span> <span> </span><span class="NLM_article-title">BRD4 structure-activity relationships of dual PLK1 kinase/BRD4 bromodomain inhibitor BI-2536</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">764</span>â <span class="NLM_lpage">769</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00084</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00084" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=764-769&author=L.+Chenauthor=J.+L.+Yapauthor=M.+Yoshiokaauthor=M.+E.+Lanningauthor=R.+N.+Fountainauthor=M.+Rajeauthor=J.+A.+Scheenstraauthor=J.+W.+Strovelauthor=S.+Fletcher&title=BRD4+structure-activity+relationships+of+dual+PLK1+kinase%2FBRD4+bromodomain+inhibitor+BI-2536&doi=10.1021%2Facsmedchemlett.5b00084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Structure-Activity Relationships of Dual PLK1 Kinase/BRD4 Bromodomain Inhibitor BI-2536</span></div><div class="casAuthors">Chen, Lijia; Yap, Jeremy L.; Yoshioka, Makoto; Lanning, Maryanna E.; Fountain, Rachel N.; Raje, Mithun; Scheenstra, Jacob A.; Strovel, Jeffrey W.; Fletcher, Steven</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">764-769</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused library of analogs of the dual PLK1 kinase/BRD4 bromodomain inhibitor BI-2536 was prepd. and then analyzed for BRD4 and PLK1 inhibitory activities.  Particularly, replacement of the cyclopentyl group with a 3-bromobenzyl moiety afforded the most potent BRD4 inhibitor of the series (39j) with a Ki = 8.7 nM, which was equipotent against PLK1.  The superior affinity of 39j over the parental compd. to BRD4 possibly derives from improved interactions with the WPF shelf.  Meanwhile, substitution of the pyrimidine NH with an oxygen atom reversed the PLK1/BRD4 selectivity to convert BI-2536 into a BRD4-selective inhibitor, likely owing to the loss of a crit. hydrogen bond in PLK1.  We believe further fine-tuning will furnish a BRD4 "magic bullet" or an even more potent PLK1/BRD4 dual inhibitor toward the expansion and improved efficacy of the chemotherapy arsenal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5UnNSmf1L17Vg90H21EOLACvtfcHk0lggqipv_VwPMg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D&md5=f39070e652bca20c03a3bc5fe070a7fc</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00084%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYap%26aufirst%3DJ.%2BL.%26aulast%3DYoshioka%26aufirst%3DM.%26aulast%3DLanning%26aufirst%3DM.%2BE.%26aulast%3DFountain%26aufirst%3DR.%2BN.%26aulast%3DRaje%26aufirst%3DM.%26aulast%3DScheenstra%26aufirst%3DJ.%2BA.%26aulast%3DStrovel%26aufirst%3DJ.%2BW.%26aulast%3DFletcher%26aufirst%3DS.%26atitle%3DBRD4%2520structure-activity%2520relationships%2520of%2520dual%2520PLK1%2520kinase%252FBRD4%2520bromodomain%2520inhibitor%2520BI-2536%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D764%26epage%3D769%26doi%3D10.1021%2Facsmedchemlett.5b00084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watts, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strebhardt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellenie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Designing dual inhibitors of anaplastic lymphoma kinase (ALK) and bromodomain-4 (BRD4) by tuning kinase selectivity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2618</span>â <span class="NLM_lpage">2637</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01947</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01947" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2618-2637&author=E.+Wattsauthor=D.+Heidenreichauthor=E.+Tuckerauthor=M.+Raabauthor=K.+Strebhardtauthor=L.+Cheslerauthor=S.+Knappauthor=B.+Bellenieauthor=S.+Hoelder&title=Designing+dual+inhibitors+of+anaplastic+lymphoma+kinase+%28ALK%29+and+bromodomain-4+%28BRD4%29+by+tuning+kinase+selectivity&doi=10.1021%2Facs.jmedchem.8b01947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Designing Dual Inhibitors of Anaplastic Lymphoma Kinase (ALK) and Bromodomain-4 (BRD4) by Tuning Kinase Selectivity</span></div><div class="casAuthors">Watts, Ellen; Heidenreich, David; Tucker, Elizabeth; Raab, Monika; Strebhardt, Klaus; Chesler, Louis; Knapp, Stefan; Bellenie, Benjamin; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2618-2637</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Concomitant inhibition of anaplastic lymphoma kinase (ALK) and bromodomain-4 (BRD4) is a potential therapeutic strategy for targeting two key oncogenic drivers that co-segregate in a significant fraction of high-risk neuroblastoma patients, mutation of ALK and amplification of MYCN.  Starting from known dual polo-like kinase (PLK)-1-BRD4 inhibitor BI-2536, we employed structure-based design to redesign this series toward compds. with a dual ALK-BRD4 profile.  These efforts led to compd. (R)-2-((2-ethoxy-4-(1-methylpiperidin-4-yl)phenyl)amino)-7-ethyl-5-methyl-8-((4-methylthiophen-2-yl)methyl)-7,8-dihydropteridin-6(5H)-one (16k) demonstrating improved ALK activity and significantly reduced PLK-1 activity, while maintaining BRD4 activity and overall kinome selectivity.  We demonstrate the compds.' on-target engagement with ALK and BRD4 in cells as well as favorable broad kinase and bromodomain selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop6qx6PSj2KLVg90H21EOLACvtfcHk0lggqipv_VwPMg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D&md5=e00fdf8eb20acc2ef1c8777b2afb750d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01947%26sid%3Dliteratum%253Aachs%26aulast%3DWatts%26aufirst%3DE.%26aulast%3DHeidenreich%26aufirst%3DD.%26aulast%3DTucker%26aufirst%3DE.%26aulast%3DRaab%26aufirst%3DM.%26aulast%3DStrebhardt%26aufirst%3DK.%26aulast%3DChesler%26aufirst%3DL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBellenie%26aufirst%3DB.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDesigning%2520dual%2520inhibitors%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%2520and%2520bromodomain-4%2520%2528BRD4%2529%2520by%2520tuning%2520kinase%2520selectivity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2618%26epage%3D2637%26doi%3D10.1021%2Facs.jmedchem.8b01947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosief, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhawan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthengi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and development of potent and selective dual bromodomain 4 (BRD4)/polo-like kinase 1 (PLK1) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7785</span>â <span class="NLM_lpage">7795</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00765</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00765" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7785-7795&author=S.+Liuauthor=H.+O.+Yosiefauthor=L.+Daiauthor=H.+Huangauthor=G.+Dhawanauthor=X.+Zhangauthor=A.+M.+Muthengiauthor=J.+Robertsauthor=D.+L.+Buckleyauthor=J.+A.+Perryauthor=L.+Wuauthor=J.+E.+Bradnerauthor=J.+Qiauthor=W.+Zhang&title=Structure-guided+design+and+development+of+potent+and+selective+dual+bromodomain+4+%28BRD4%29%2Fpolo-like+kinase+1+%28PLK1%29+inhibitors&doi=10.1021%2Facs.jmedchem.8b00765"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors</span></div><div class="casAuthors">Liu, Shuai; Yosief, Hailemichael O.; Dai, Lingling; Huang, He; Dhawan, Gagan; Zhang, Xiaofeng; Muthengi, Alex M.; Roberts, Justin; Buckley, Dennis L.; Perry, Jennifer A.; Wu, Lei; Bradner, James E.; Qi, Jun; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7785-7795</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single mol. could lead to the development of an effective therapeutic strategy for a variety of diseases in which PLK1 and BRD4 are implicated.  Compd. 23 has been found to be a potent dual kinase-bromodomain inhibitor (BRD4-BD1 IC50 = 28 nM, PLK1 IC50 = 40 nM).  Compd. 6 was found to be the most selective PLK1 inhibitor over BRD4 in our series (BRD4-BD1 IC50 = 2579 nM, PLK1 IC50 = 9.9 nM).  Mol. docking studies with 23 and BRD4-BD1/PLK1 as well as with 6 corroborate the biochem. assay results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUL2G-DnFzU7Vg90H21EOLACvtfcHk0lh-sulW-21ZeQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L&md5=e34a58694f0827d33d31516200e2c0d8</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00765%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYosief%26aufirst%3DH.%2BO.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DDhawan%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMuthengi%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DJ.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DStructure-guided%2520design%2520and%2520development%2520of%2520potent%2520and%2520selective%2520dual%2520bromodomain%25204%2520%2528BRD4%2529%252Fpolo-like%2520kinase%25201%2520%2528PLK1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7785%26epage%3D7795%26doi%3D10.1021%2Facs.jmedchem.8b00765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of Î³-Carboline analogues as potent and specific BET bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4927</span>â <span class="NLM_lpage">4939</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4927-4939&author=X.+Ranauthor=Y.+Zhaoauthor=L.+Liuauthor=L.+Baiauthor=C.+Y.+Yangauthor=B.+Zhouauthor=J.+L.+Meagherauthor=K.+Chinnaswamyauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-based+design+of+%CE%B3-Carboline+analogues+as+potent+and+specific+BET+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Î³-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span></div><div class="casAuthors">Ran, Xu; Zhao, Yujun; Liu, Liu; Bai, Longchuan; Yang, Chao-Yie; Zhou, Bing; Meagher, Jennifer L.; Chinnaswamy, Krishnapriya; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4927-4939</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors of bromodomain and extra terminal proteins (BET), including BRD2, BRD3, and BRD4 proteins have therapeutic potential for the treatment of human cancers and other diseases and conditions.  In this paper, the authors report the design, synthesis, and evaluation of Î³-carboline-contg. compds. as a new class of small-mol. BET inhibitors.  The most potent inhibitor I (RX-37) obtained from this study binds to BET bromodomain proteins (BRD2, BRD3, and BRD4) with Ki values of 3.2-24.7 nM and demonstrates high selectivity over other non-BET bromodomain-contg. proteins.  Compd. I potently and selectively inhibits cell growth in human acute leukemia cell lines harboring the rearranged mixed lineage leukemia 1 gene.  The authors have detd. a cocrystal structure of I in complex with BRD4 BD2 at 1.4 Ã resoln., which provides a solid structural basis for the compd.'s high binding affinity and for its further structure-based optimization.  Compd. I represents a promising lead compd. for the development of a new class of therapeutics for the treatment of human cancer and other conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv-VCfWLBWYLVg90H21EOLACvtfcHk0lh-sulW-21ZeQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM&md5=e6e150169812e439c1d7d91416eb3ee9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00613%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520design%2520of%2520%25CE%25B3-Carboline%2520analogues%2520as%2520potent%2520and%2520specific%2520BET%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4927%26epage%3D4939%26doi%3D10.1021%2Facs.jmedchem.5b00613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span> <span> </span><span class="NLM_article-title">Fragment-based drug discovery of 2-thiazolidinones as inhibitors of the histone reader BRD4 bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3833</span>â <span class="NLM_lpage">3851</span>, <span class="refDoi">Â DOI: 10.1021/jm301793a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301793a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3833-3851&author=L.+Zhaoauthor=D.+Caoauthor=T.+Chenauthor=Y.+Wangauthor=Z.+Miaoauthor=Y.+Xuauthor=W.+Chenauthor=X.+Wangauthor=Y.+Liauthor=Z.+Duauthor=B.+Xiongauthor=J.+Liauthor=C.+Xuauthor=N.+Zhangauthor=J.+Heauthor=J.+Shen&title=Fragment-based+drug+discovery+of+2-thiazolidinones+as+inhibitors+of+the+histone+reader+BRD4+bromodomain&doi=10.1021%2Fjm301793a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery of 2-Thiazolidinones as Inhibitors of the Histone Reader BRD4 Bromodomain</span></div><div class="casAuthors">Zhao, Lele; Cao, Danyan; Chen, Tiantian; Wang, Yingqing; Miao, Zehong; Xu, Yechun; Chen, Wuyan; Wang, Xin; Li, Yanlian; Du, Zhiyan; Xiong, Bing; Li, Jian; Xu, Chunyan; Zhang, Naixia; He, Jianhua; Shen, Jingkang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3833-3851</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recognizing acetyllysine of histone is a vital process of epigenetic regulation that is mediated by a protein module called bromodomain.  To contribute novel scaffolds for developing into bromodomain inhibitors, the authors utilize a fragment-based drug discovery approach.  By successively applying docking and x-ray crystallog., they were able to identify 9 fragment hits from diffracting more than 60 crystals.  In the present work, they described four of them and carried out the integrated lead optimization for fragment I, which bears a 2-thiazolidinone core.  After several rounds of structure guided modifications, they assessed the druggability of 2-thiazolidinone by modulating in vitro pharmacokinetic studies and cellular activity assay.  The results showed that two potent of 2-thiazolidinone compds. II and III have good metabolic stability.  Also, the cellular assay confirmed the activities of 2-thiazolidinones.  Together, they hope the identified 2-thiazolidinone chemotype and other fragment hits described herein can stimulate researchers to develop more diversified bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfi2pyYq6c4LVg90H21EOLACvtfcHk0lh-sulW-21ZeQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D&md5=8308db5d83cb72b29f261cc134514ece</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm301793a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301793a%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DFragment-based%2520drug%2520discovery%2520of%25202-thiazolidinones%2520as%2520inhibitors%2520of%2520the%2520histone%2520reader%2520BRD4%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3833%26epage%3D3851%26doi%3D10.1021%2Fjm301793a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resar, L. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emison, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prescott, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wonsey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeller, K.</span></span> <span> </span><span class="NLM_article-title">Function of the c-Myc oncogenic transcription factor</span>. <i>Exp. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>253</i></span>,  <span class="NLM_fpage">63</span>â <span class="NLM_lpage">77</span>, <span class="refDoi">Â DOI: 10.1006/excr.1999.4686</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1006%2Fexcr.1999.4686" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10579912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADyaK1MXnsF2it7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1999&pages=63-77&author=C.+V.+Dangauthor=L.+M.+S.+Resarauthor=E.+Emisonauthor=S.+Kimauthor=Q.+Liauthor=J.+E.+Prescottauthor=D.+Wonseyauthor=K.+Zeller&title=Function+of+the+c-Myc+oncogenic+transcription+factor&doi=10.1006%2Fexcr.1999.4686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Function of the c-Myc Oncogenic Transcription Factor</span></div><div class="casAuthors">Dang, Chi V.; Resar, Linda M. S.; Emison, Eileen; Kim, Sunkyu; Li, Qing; Prescott, Julia E.; Wonsey, Diane; Zeller, Karen</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Cell Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-77</span>CODEN:
                <span class="NLM_cas:coden">ECREAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-4827</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review with 238 refs.  The c-myc gene and the expression of the c-Myc protein are frequently altered in human cancers.  The c-myc gene encodes the transcription factor c-Myc, which heterodimerizes with a partner protein, termed Max, to regulate gene expression.  Max also heterodimerizes with the Mad family of proteins to repress transcription, antagonize c-Myc, and promote cellular differentiation.  The constitutive activation of c-myc expression is key to the genesis of many cancers, and hence the understanding of c-Myc function depends on our understanding of its target genes.  In this review, the authors attempt to place the putative target genes of c-Myc in the context of c-Myc-mediated phenotypes.  From this perspective, c-Myc emerges as an oncogenic transcription factor that integrates the cell cycle machinery with cell adhesion, cellular metab., and the apoptotic pathways.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjNm2Xvnxf9LVg90H21EOLACvtfcHk0lg0BiShA_HQkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsF2it7w%253D&md5=34262c18507e4da66ee47be7c08d55d8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1006%2Fexcr.1999.4686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fexcr.1999.4686%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DResar%26aufirst%3DL.%2BM.%2BS.%26aulast%3DEmison%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DPrescott%26aufirst%3DJ.%2BE.%26aulast%3DWonsey%26aufirst%3DD.%26aulast%3DZeller%26aufirst%3DK.%26atitle%3DFunction%2520of%2520the%2520c-Myc%2520oncogenic%2520transcription%2520factor%26jtitle%3DExp.%2520Cell%2520Res.%26date%3D1999%26volume%3D253%26spage%3D63%26epage%3D77%26doi%3D10.1006%2Fexcr.1999.4686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappaport, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chicas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">524</span>â <span class="NLM_lpage">528</span>, <span class="refDoi">Â DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0lg0BiShA_HQkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26epage%3D528%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huthmacher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gudgin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doehner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">529</span>â <span class="NLM_lpage">533</span>, <span class="refDoi">Â DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.+I.+Chanauthor=S.+C.+Robsonauthor=C.+W.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+P.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0lh1Zi0glz5fGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span> <span> </span><span class="NLM_article-title">Brd4 recruits P-TEFb to chromosomes at late mitosis to promote G1 gene expression and cell cycle progression</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">967</span>â <span class="NLM_lpage">976</span>, <span class="refDoi">Â DOI: 10.1128/MCB.01020-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1128%2FMCB.01020-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=18039861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWqur4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=967-976&author=Z.+Yangauthor=N.+Heauthor=Q.+Zhou&title=Brd4+recruits+P-TEFb+to+chromosomes+at+late+mitosis+to+promote+G1+gene+expression+and+cell+cycle+progression&doi=10.1128%2FMCB.01020-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 recruits P-TEFb to chromosomes at late mitosis to promote G1 gene expression and cell cycle progression</span></div><div class="casAuthors">Yang, Zhiyuan; He, Nanhai; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">967-976</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Brd4, a bromodomain protein capable of interacting with acetylated histones, is implicated in transmitting epigenetic memory through mitosis.  It also functions as an assocd. factor and pos. regulator of P-TEFb, a Cdk9-cyclin T1 heterodimer that stimulates transcriptional elongation by phosphorylating RNA polymerase II.  In the present study, expts. were performed to det. whether these two functions of Brd4 are interrelated and, if so, how they may impact cell cycle progression.  Our data demonstrate that while the P-TEFb level remains const., the Brd4-P-TEFb interaction increases dramatically in cells progressing from late mitosis to early G1.  Concurrently, P-TEFb is recruited to chromosomes, beginning around mid- to late anaphase and before nuclear envelope/lamina formation and nuclear import of other general transcription factors.  Importantly, the recruitment of P-TEFb depends on Brd4.  Abrogation of this process through Brd4 knockdown reduces the binding of P-TEFb to and expression of key G1 and growth-assocd. genes, leading to G1 cell cycle arrest and apoptosis.  Because P-TEFb is synonymous with productive elongation, its recruitment by Brd4 to chromosomes at late mitosis may indicate those genes whose active transcription status must be preserved across cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_qFlZcTzQiLVg90H21EOLACvtfcHk0lh1Zi0glz5fGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWqur4%253D&md5=99b71de9dec48e64ecbcb59fda3b6d1d</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1128%2FMCB.01020-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01020-07%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DHe%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DBrd4%2520recruits%2520P-TEFb%2520to%2520chromosomes%2520at%2520late%2520mitosis%2520to%2520promote%2520G1%2520gene%2520expression%2520and%2520cell%2520cycle%2520progression%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2008%26volume%3D28%26spage%3D967%26epage%3D976%26doi%3D10.1128%2FMCB.01020-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamothe, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span> <span> </span><span class="NLM_article-title">The human ether-a-go-go-related Gene (hERG) potassium channel represents an unusual target for protease-mediated damage</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">20387</span>â <span class="NLM_lpage">20401</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M116.743138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1074%2Fjbc.M116.743138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=27502273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFGnurvM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=20387-20401&author=S.+M.+Lamotheauthor=J.+Guoauthor=W.+Liauthor=T.+Yangauthor=S.+Zhang&title=The+human+ether-a-go-go-related+Gene+%28hERG%29+potassium+channel+represents+an+unusual+target+for+protease-mediated+damage&doi=10.1074%2Fjbc.M116.743138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The human ether-a-go-go-related gene (hERG) potassium channel represents an unusual target for protease-mediated damage</span></div><div class="casAuthors">Lamothe, Shawn M.; Guo, Jun; Li, Wentao; Yang, Tonghua; Zhang, Shetuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">20387-20401</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The human ether-a-go-go-related gene (hERG) encodes the pore-forming subunit of the rapidly activating delayed rectifier potassium channel (IKr), which is important for cardiac repolarization.  Dysfunction of hERG causes long QT syndrome and sudden death, which occur in patients with cardiac ischemia.  Cardiac ischemia is also assocd. with activation, up-regulation, and secretion of various proteolytic enzymes.  Here, using whole-cell patch clamp and Western blotting anal., we demonstrate that the hERG/IKr channel was selectively cleaved by the serine protease, proteinase K (PK).  Using mol. biol. techniques including making a chimeric channel between protease-sensitive hERG and insensitive human ether-a-go-go (hEAG), as well as application of the scorpion toxin BeKm-1, we identified that the S5-pore linker of hERG is the target domain for proteinase K cleavage.  To investigate the physiol. relevance of the unique susceptibility of hERG to proteases, we show that cardiac ischemia in a rabbit model was assocd. with a redn. in mature ERG expression and an increase in the expression of several proteases, including calpain.  Using cell biol. approaches, we found that calpain-1 was actively released into the extracellular milieu and cleaved hERG at the S5-pore linker.  Using protease cleavage-predicting software and site-directed mutagenesis, we identified that calpain-1 cleaves hERG at position Gly-603 in the S5-pore linker of hERG.  Clarification of protease-mediated damage of hERG extends our understanding of hERG regulation.  Damage of hERG mediated by proteases such as calpain may contribute to ischemia-assocd. QT prolongation and sudden cardiac death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCd-cfWdEAfLVg90H21EOLACvtfcHk0lh1Zi0glz5fGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFGnurvM&md5=1c5d685ed5fc9161cc33d6a028289026</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.743138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.743138%26sid%3Dliteratum%253Aachs%26aulast%3DLamothe%26aufirst%3DS.%2BM.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DS.%26atitle%3DThe%2520human%2520ether-a-go-go-related%2520Gene%2520%2528hERG%2529%2520potassium%2520channel%2520represents%2520an%2520unusual%2520target%2520for%2520protease-mediated%2520damage%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D20387%26epage%3D20401%26doi%3D10.1074%2Fjbc.M116.743138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span> <span> </span><span class="NLM_article-title">Oxime-mediated facile access to 5-methylisoxazoles and applications in the synthesis of valdecoxib and oxacillin</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">5266</span>â <span class="NLM_lpage">5268</span>, <span class="refDoi">Â DOI: 10.1021/ol502246t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol502246t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlu7%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=5266-5268&author=K.+Y.+Dongauthor=H.+T.+Qinauthor=X.+X.+Baoauthor=F.+Liuauthor=C.+Zhu&title=Oxime-mediated+facile+access+to+5-methylisoxazoles+and+applications+in+the+synthesis+of+valdecoxib+and+oxacillin&doi=10.1021%2Fol502246t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Oxime-Mediated Facile Access to 5-Methylisoxazoles and Applications in the Synthesis of Valdecoxib and Oxacillin</span></div><div class="casAuthors">Dong, Kui-Yong; Qin, Hai-Tao; Bao, Xing-Xing; Liu, Feng; Zhu, Chen</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5266-5268</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A palladium-catalyzed efficient synthesis of 5-methylisoxazoles via oxime-mediated functionalization of unactivated olefins is described.  The reaction affords a variety of 5-methylisoxazoles in moderate to good yields.  To further demonstrate the utility of the method, the rapid synthesis of valdecoxib (I) and oxacillin (II) is reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP58UkfSzID7Vg90H21EOLACvtfcHk0lgiY2sWMjS5zQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlu7%252FO&md5=96e7e292e10e1b208d81f92c3ccc54cc</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fol502246t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol502246t%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DK.%2BY.%26aulast%3DQin%26aufirst%3DH.%2BT.%26aulast%3DBao%26aufirst%3DX.%2BX.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DZhu%26aufirst%3DC.%26atitle%3DOxime-mediated%2520facile%2520access%2520to%25205-methylisoxazoles%2520and%2520applications%2520in%2520the%2520synthesis%2520of%2520valdecoxib%2520and%2520oxacillin%26jtitle%3DOrg.%2520Lett.%26date%3D2014%26volume%3D16%26spage%3D5266%26epage%3D5268%26doi%3D10.1021%2Fol502246t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zibinsky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokin, V. V.</span></span> <span> </span><span class="NLM_article-title">Reactivity of N-(1,2,4-triazolyl)-substituted 1,2,3-triazoles</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4870</span>â <span class="NLM_lpage">4872</span>, <span class="refDoi">Â DOI: 10.1021/ol201949h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol201949h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVegt7vL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=4870-4872&author=M.+Zibinskyauthor=V.+V.+Fokin&title=Reactivity+of+N-%281%2C2%2C4-triazolyl%29-substituted+1%2C2%2C3-triazoles&doi=10.1021%2Fol201949h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Reactivity of N-(1,2,4-Triazolyl)-Substituted 1,2,3-Triazoles</span></div><div class="casAuthors">Zibinsky, Mikhail; Fokin, Valery V.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4870-4872</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetically useful rhodium(II) carbenes were obtained from N-(1,2,4-triazolyl)-substituted 1,2,3-triazoles, e.g., I, and Rh(II) carboxylates.  The electron-withdrawing 1,2,4-triazolyl group reveals the heretofore unknown reactivity of nonsulfonyl 1,2,3-triazoles, which exhibit the reactivity of diazo compds.  The resulting carbenes provide ready asym. access to secondary homoaminocyclopropanes, e.g., II, (80-95% ee, dr >20:1) via reactions with olefins and also engage in efficient transannulation reactions with nitriles to give imidazoles, e.g., III.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxp8JdKAoqqrVg90H21EOLACvtfcHk0lgiY2sWMjS5zQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVegt7vL&md5=178803d69a9ab09c91a8d24ea5fff2c0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fol201949h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol201949h%26sid%3Dliteratum%253Aachs%26aulast%3DZibinsky%26aufirst%3DM.%26aulast%3DFokin%26aufirst%3DV.%2BV.%26atitle%3DReactivity%2520of%2520N-%25281%252C2%252C4-triazolyl%2529-substituted%25201%252C2%252C3-triazoles%26jtitle%3DOrg.%2520Lett.%26date%3D2011%26volume%3D13%26spage%3D4870%26epage%3D4872%26doi%3D10.1021%2Fol201949h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=%5B20%5D+Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (Ï2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Ã unit cell to rotavirus of 700 Ã diam. crystd. in 700 Ã 1000 Ã 1400 Ã cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lh7HUlm5Z5gyg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Cryst.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>â <span class="NLM_lpage">255</span>, <span class="refDoi">Â DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lh7HUlm5Z5gyg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Cryst.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <i>Acta Cryst.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>â <span class="NLM_lpage">1954</span>, <span class="refDoi">Â DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.+W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0lh7HUlm5Z5gyg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Cryst.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Cryst.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>â <span class="NLM_lpage">501</span>, <span class="refDoi">Â DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liWdOQoIbm4cQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Cryst.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI3','PDB','6JI3'); return false;">PDB: 6JI3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z93" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z93','PDB','4Z93'); return false;">PDB: 4Z93</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI4','PDB','6JI4'); return false;">PDB: 6JI4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JI5','PDB','6JI5'); return false;">PDB: 6JI5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF','PDB','3MXF'); return false;">PDB: 3MXF</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i103"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01094">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_51102"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b01094">10.1021/acs.jmedchem.9b01094</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Crystallography information for the complex structures, in vitro PK profiles, in vivo data of compound <b>7</b>, solubility data of compounds <b>7</b> and <b>19</b>, in vivo PK properties of compounds <b>15</b> and <b>20â23</b>, bromodomain and kinase selectivity data, ITC binding affinity data of <b>19</b>, hERG inhibition data of <b>19,</b> and spectra data of representative compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Table of molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_001.pdf">jm9b01094_si_001.pdf (1.3 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01094/suppl_file/jm9b01094_si_002.csv">jm9b01094_si_002.csv (2.42 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01094&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01094%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-18%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01094" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679957ceb9ce3d1e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
