
<html lang="en"     class="pb-page"  data-request-id="e32b0333-8082-4519-986c-544f3c4d1cf9"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.8b01826;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-5;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions" /></meta><meta name="dc.Creator" content="Johannes  Popow" /></meta><meta name="dc.Creator" content="Heribert  Arnhof" /></meta><meta name="dc.Creator" content="Gerd  Bader" /></meta><meta name="dc.Creator" content="Helmut  Berger" /></meta><meta name="dc.Creator" content="Alessio  Ciulli" /></meta><meta name="dc.Creator" content="David  Covini" /></meta><meta name="dc.Creator" content="Christian  Dank" /></meta><meta name="dc.Creator" content="Teresa  Gmaschitz" /></meta><meta name="dc.Creator" content="Peter  Greb" /></meta><meta name="dc.Creator" content="Jale  Karolyi-Özguer" /></meta><meta name="dc.Creator" content="Manfred  Koegl" /></meta><meta name="dc.Creator" content="Darryl B.  McConnell" /></meta><meta name="dc.Creator" content="Mark  Pearson" /></meta><meta name="dc.Creator" content="Maria  Rieger" /></meta><meta name="dc.Creator" content="Joerg  Rinnenthal" /></meta><meta name="dc.Creator" content="Vanessa  Roessler" /></meta><meta name="dc.Creator" content="Andreas  Schrenk" /></meta><meta name="dc.Creator" content="Markus  Spina" /></meta><meta name="dc.Creator" content="Steffen  Steurer" /></meta><meta name="dc.Creator" content="Nicole  Trainor" /></meta><meta name="dc.Creator" content="Elisabeth  Traxler" /></meta><meta name="dc.Creator" content="Corinna  Wieshofer" /></meta><meta name="dc.Creator" content="Andreas  Zoephel" /></meta><meta name="dc.Creator" content="Peter  Ettmayer" /></meta><meta name="dc.Description" content="Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacological inhibition to reduced tum..." /></meta><meta name="Description" content="Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacological inhibition to reduced tum..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 9, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01826" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01826" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01826" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01826" /></link>
        
    
    

<title>Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01826" /></meta><meta property="og:title" content="Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0009.jpeg" /></meta><meta property="og:description" content="Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacological inhibition to reduced tumorigenicity. However, the clinical relevance of targeting PTK2 still remains to be proven. Here, we present two highly selective and functional PTK2 proteolysis-targeting chimeras utilizing von Hippel–Lindau and cereblon ligands to hijack E3 ligases for PTK2 degradation. BI-3663 (cereblon-based) degrades PTK2 with a median DC50 of 30 nM to &gt;80% across a panel of 11 HCC cell lines. Despite effective PTK2 degradation, these compounds did not phenocopy the reported antiproliferative effects of PTK2 depletion in any of the cell lines tested. By disclosing these compounds, we hope to provide valuable tools for the study of PTK2 degradation across different biological systems." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01826"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01826">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01826&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01826&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01826&amp;href=/doi/10.1021/acs.jmedchem.8b01826" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2508-2520</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01806" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01854" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Johannes Popow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Johannes Popow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Johannes++Popow">Johannes Popow</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Heribert Arnhof</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heribert Arnhof</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heribert++Arnhof">Heribert Arnhof</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gerd Bader</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gerd Bader</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gerd++Bader">Gerd Bader</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Helmut Berger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Helmut Berger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Helmut++Berger">Helmut Berger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alessio Ciulli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alessio Ciulli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Biological Chemistry and Drug Discovery, School of Life Sciences, James Black Centre, University of Dundee, Dow Street, DD1 5EH Dundee, Scotland, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alessio++Ciulli">Alessio Ciulli</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8654-1670" title="Orcid link">http://orcid.org/0000-0002-8654-1670</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Covini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Covini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Covini">David Covini</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christian Dank</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Dank</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Dank">Christian Dank</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Teresa Gmaschitz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Teresa Gmaschitz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Teresa++Gmaschitz">Teresa Gmaschitz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter Greb</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Greb</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Greb">Peter Greb</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jale Karolyi-Özguer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jale Karolyi-Özguer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jale++Karolyi-%C3%96zguer">Jale Karolyi-Özguer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Manfred Koegl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Manfred Koegl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Manfred++Koegl">Manfred Koegl</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Darryl B. McConnell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darryl B. McConnell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darryl+B.++McConnell">Darryl B. McConnell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark Pearson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark Pearson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark++Pearson">Mark Pearson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Maria Rieger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maria Rieger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maria++Rieger">Maria Rieger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joerg Rinnenthal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joerg Rinnenthal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joerg++Rinnenthal">Joerg Rinnenthal</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vanessa Roessler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vanessa Roessler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vanessa++Roessler">Vanessa Roessler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Schrenk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Schrenk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Schrenk">Andreas Schrenk</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Markus Spina</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Markus Spina</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Markus++Spina">Markus Spina</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Steffen Steurer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steffen Steurer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steffen++Steurer">Steffen Steurer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicole Trainor</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicole Trainor</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Biological Chemistry and Drug Discovery, School of Life Sciences, James Black Centre, University of Dundee, Dow Street, DD1 5EH Dundee, Scotland, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicole++Trainor">Nicole Trainor</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elisabeth Traxler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elisabeth Traxler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elisabeth++Traxler">Elisabeth Traxler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Corinna Wieshofer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Corinna Wieshofer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Corinna++Wieshofer">Corinna Wieshofer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Zoephel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Zoephel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Zoephel">Andreas Zoephel</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Peter Ettmayer</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Ettmayer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co KG, 1221 Vienna, Austria</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#a0d0c5d4c5d28ec5d4d4cdc1d9c5d2e0c2cfc5c8d2c9cec7c5d28dc9cec7c5ccc8c5c9cd8ec3cfcd"><span class="__cf_email__" data-cfemail="bfcfdacbdacd91dacbcbd2dec6dacdffddd0dad7cdd6d1d8dacd92d6d1d8dad3d7dad6d291dcd0d2">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Ettmayer">Peter Ettmayer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8422-2625" title="Orcid link">http://orcid.org/0000-0002-8422-2625</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01826&amp;href=/doi/10.1021%2Facs.jmedchem.8b01826" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2508–2520</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 9, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 November 2018</li><li><span class="item_label"><b>Published</b> online</span>9 February 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 March 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01826" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01826</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2508%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJohannes%2BPopow%252C%2BHeribert%2BArnhof%252C%2BGerd%2BBader%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D5%26contentID%3Dacs.jmedchem.8b01826%26title%3DHighly%2BSelective%2BPTK2%2BProteolysis%2BTargeting%2BChimeras%2Bto%2BProbe%2BFocal%2BAdhesion%2BKinase%2BScaffolding%2BFunctions%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2520%26publicationDate%3DMarch%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01826"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4126</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">39</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01826" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Johannes&quot;,&quot;last_name&quot;:&quot;Popow&quot;},{&quot;first_name&quot;:&quot;Heribert&quot;,&quot;last_name&quot;:&quot;Arnhof&quot;},{&quot;first_name&quot;:&quot;Gerd&quot;,&quot;last_name&quot;:&quot;Bader&quot;},{&quot;first_name&quot;:&quot;Helmut&quot;,&quot;last_name&quot;:&quot;Berger&quot;},{&quot;first_name&quot;:&quot;Alessio&quot;,&quot;last_name&quot;:&quot;Ciulli&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Covini&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Dank&quot;},{&quot;first_name&quot;:&quot;Teresa&quot;,&quot;last_name&quot;:&quot;Gmaschitz&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Greb&quot;},{&quot;first_name&quot;:&quot;Jale&quot;,&quot;last_name&quot;:&quot;Karolyi-Özguer&quot;},{&quot;first_name&quot;:&quot;Manfred&quot;,&quot;last_name&quot;:&quot;Koegl&quot;},{&quot;first_name&quot;:&quot;Darryl&quot;,&quot;last_name&quot;:&quot;B. McConnell&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;Pearson&quot;},{&quot;first_name&quot;:&quot;Maria&quot;,&quot;last_name&quot;:&quot;Rieger&quot;},{&quot;first_name&quot;:&quot;Joerg&quot;,&quot;last_name&quot;:&quot;Rinnenthal&quot;},{&quot;first_name&quot;:&quot;Vanessa&quot;,&quot;last_name&quot;:&quot;Roessler&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Schrenk&quot;},{&quot;first_name&quot;:&quot;Markus&quot;,&quot;last_name&quot;:&quot;Spina&quot;},{&quot;first_name&quot;:&quot;Steffen&quot;,&quot;last_name&quot;:&quot;Steurer&quot;},{&quot;first_name&quot;:&quot;Nicole&quot;,&quot;last_name&quot;:&quot;Trainor&quot;},{&quot;first_name&quot;:&quot;Elisabeth&quot;,&quot;last_name&quot;:&quot;Traxler&quot;},{&quot;first_name&quot;:&quot;Corinna&quot;,&quot;last_name&quot;:&quot;Wieshofer&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Zoephel&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Ettmayer&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;2508-2520&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01826&quot;},&quot;abstract&quot;:&quot;Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacological inhibition to reduced tumorigenicity. However, the clinical relevance of targeting PTK2 still remains to be proven. Here, we present two highly selective and functional PTK2 proteolysis-targeting chimeras utilizing von Hippel–Lindau and cereblon ligands to hijack E3 ligases for PTK2 degradation. BI-3663 (cereblon-based) degrades PTK2 with a median DC50 of 30 nM to &gt;80% across a panel of 11 HCC cell lines. Despite effective PTK2 degradation, these compounds did not phenocopy the reported antiproliferative effects of PTK2 depletion in any of the cell lines tested. By disclosing these compounds, we hope to provide valuable tools for the study of PTK2 degradation across different biological systems.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01826&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01826" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01826&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01826" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01826&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01826" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01826&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01826&amp;href=/doi/10.1021/acs.jmedchem.8b01826" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01826" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01826" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (1 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01826%26sid%3Dliteratum%253Aachs%26pmid%3D30739444%26genre%3Darticle%26aulast%3DPopow%26date%3D2019%26atitle%3DHighly%2BSelective%2BPTK2%2BProteolysis%2BTargeting%2BChimeras%2Bto%2BProbe%2BFocal%2BAdhesion%2BKinase%2BScaffolding%2BFunctions%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D5%26spage%3D2508%26epage%3D2520%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/jmcmar.2019.62.issue-5/20190314/jmcmar.2019.62.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacological inhibition to reduced tumorigenicity. However, the clinical relevance of targeting PTK2 still remains to be proven. Here, we present two highly selective and functional PTK2 proteolysis-targeting chimeras utilizing von Hippel–Lindau and cereblon ligands to hijack E3 ligases for PTK2 degradation. BI-3663 (cereblon-based) degrades PTK2 with a median DC<sub>50</sub> of 30 nM to >80% across a panel of 11 HCC cell lines. Despite effective PTK2 degradation, these compounds did not phenocopy the reported antiproliferative effects of PTK2 depletion in any of the cell lines tested. By disclosing these compounds, we hope to provide valuable tools for the study of PTK2 degradation across different biological systems.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78203" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78203" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Focal adhesion tyrosine kinase (PTK2) is a cytoplasmic protein tyrosine kinase that is overexpressed and activated in many types of advanced-stage solid cancers. PTK2 is reported to play an important role in adhesion, spreading, motility, invasion, metastasis, survival, angiogenesis, epithelial to mesenchymal transition, cancer stem cells, and the tumor microenvironment.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Overexpression and activation of PTK2 is associated with several human malignant diseases, including colorectal,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> ovarian,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> esophageal,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and hepatocellular carcinoma (HCC)<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and is correlated with poor overall patient survival.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> HCC is the most common type of liver cancer in humans, accounting for 70–85% of primary liver malignancies, and is the third leading cause of cancer-related deaths worldwide.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> HCC is associated with an extremely poor prognosis because it is often diagnosed at advanced stages, restricting the currently available therapeutic options to either surgical resection or liver transplantation.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Several reports link PTK2 depletion or pharmacological inhibition of its kinase activity to reduction of in vitro and in vivo tumorigenicity in HCC models.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> However, the disconnect between the modulation of intracellular PTK2 autophosphorylation and growth inhibition as well as the often suboptimal selectivity profile of the inhibitors used makes it difficult to link the reported blockade of HCC tumor initiation and maintenance to PTK2 inhibition.</div><div class="NLM_p">We previously described BI-853520, which is a novel ATP-competitive inhibitor distinguished by high potency and selectivity.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> BI-853520 inhibits PTK2 autophosphorylation in cancer cell lines and blocks anchorage-independent proliferation with single digit nmol/L potency. In contrast, cells grown in a conventional surface (two-dimensional) culture were found to be 1000-fold less sensitive to BI-853520. These findings are in keeping with the described role of PTK2 in integrin-mediated signaling.<a onclick="showRef(event, 'ref2 ref17'); return false;" href="javascript:void(0);" class="ref ref2 ref17">(2,17)</a> We reasoned that ligands derived from BI-853520 would be suitable to develop PROTACs (proteolysis-targeting chimeras) and compare the potential antioncogenic effects of PTK2 depletion as compared to inhibition of its kinase activity.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><div class="NLM_p">PROTACs are bifunctional degrader molecules composed of a ligand for the target protein linked to a module that recruits an E3 ligase.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> PROTACs represent an emerging therapeutic strategy to use small molecules to deplete a protein by repurposing the ubiquitin–proteasome system.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> Upon formation of a ternary complex target:PROTAC:E3 ligase,<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> the protein of interest is ubiquitinated and subsequently degraded by the proteasome. Compared to pharmacological inhibition of the kinase function, protein degradation more closely resembles genetic approaches interfering with PTK2 expression in HCC models. Moreover, PROTAC molecules can add a layer of target selectivity beyond that expected from the constitutive binding ligands, thus providing highly selective degraders with potentially reduced off-target effects.<a onclick="showRef(event, 'ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27 ref28">(25−28)</a></div><div class="NLM_p">The most common E3 ligases currently recruited using PROTACs are the von Hippel–Lindau (VHL) protein complex, CRL2<sup>VHL</sup> and the cereblon (CRBN) complex, CRL4<sup>CRBN</sup>. PROTACs consisting of the same target ligand but employing either VHL or CRBN ligands can exhibit different degradation selectivity and efficacy.<a onclick="showRef(event, 'ref26 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref26 ref29 ref30 ref31">(26,29−31)</a> Emerging evidence suggests that it might be beneficial to develop parallel chemical series that hijack different E3 ligases. First, expression of the recruited E3 ligase and intrinsic activity may be context-dependent and vary widely among cell and tissue types.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Second, resistance mechanisms could potentially arise from loss of the recruited E3 ligase, as demonstrated by the correlation between the level of CRBN and response to CRBN-recruiting drugs in multiple myeloma.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Flexible choice of the recruited E3 ligase may therefore aid targeted protein degradation.</div><div class="NLM_p">Recently, the first PTK2 PROTACs have been reported. Conjugating the known pyrimidine-based ALK inhibitors, TAE684 and LDK378, to the CRBN ligand, pomalidomide, led to the discovery of the first small molecule degraders of ALK that concomitantly degrade, PTK2, aurora A, FER, and RPS6KA1.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> A PROTAC based on a highly promiscuous kinase inhibitor was also reported to degrade PTK2 among 27 additional kinases.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Although both studies demonstrate that PTK2 can be targeted by a PROTAC approach, they also clearly highlight the need for more selective PTK2 PROTACs to address its role in cancer. Motivated by the potential of targeting kinase-independent functions of PTK2 with a low molecular weight modality, we initiated a PROTAC medicinal chemistry campaign to artificially recruit PTK2 to two different E3 ligases.</div><div class="NLM_p last">Here, we show the synthesis and characterization of the first probe-quality PROTACs targeting PTK2. Structure-guided conjugation of a highly selective PTK2 inhibitor to either a CRBN or VHL ligand led to selective PTK2 degraders. We profiled both PROTACs for degradation efficacy and antiproliferative activity in a panel of HCC, tongue squamous cell carcinoma, melanoma, pancreatic ductal adenocarcinoma, and nonsmall cell lung cancer cell lines, several of which were recently shown to depend on the expression of PTK2 by functional RNA interference screening.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Despite potent and complete degradation of PTK2, both degraders failed to show the anticipated growth reduction beyond the effect of a PTK2 kinase inhibitor in these models.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25174" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25174" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We designed a set of PTK2 PROTACs recruiting the two E3 ubiquitin ligases, VHL and CRBN, to test whether PTK2 can be targeted by targeted proteolysis. Making use of the available E3 ligase ligands, we aimed at maximizing the chance of developing an active PTK2 degrader tool compound. We selected the highly selective ATP competitive inhibitor BI-4464 (<b>1</b>, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), a close analogue of the 2-aminophenyl-4-phenoxypyrimidine, BI-853520,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> currently being tested in clinical trials as a PTK2 ligand. The choice of BI-4464 as a target ligand was driven by its high binding affinity and exquisite selectivity across a large kinase panel (see also <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). We believe that this exceptional selectivity originates from a unique binding mode, which seems to be less tolerated by other kinases. To design a first generation of degraders, we inspected the crystal structure of BI-4464 bound to the PTK2 kinase domain to identify suitable attachment points and vectors for linker conjugation (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The structure shows binding of the trifluoro-aminopyrimidine moiety to the backbone nitrogen and carbonyl of cysteine 502 in the hinge region via two hydrogen bonds. In addition, the oxygen of the dihydroindenone moiety forms another hydrogen bond with the backbone nitrogen of aspartic acid 564. The <i>N</i>-methyl piperidine group of BI-4464 was identified as solvent-exposed and not involved in any interactions with the protein. We further selected <b>2</b><a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and pomalidomide <b>3</b>,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) as the VHL and CRBN ligands, respectively. Their terminal amino groups (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) were conjugated via an amide bond to the linker consisting of up to five PEG units without perturbing the interaction with the E3 ligases, as previously demonstrated.<a onclick="showRef(event, 'ref28 ref31 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref28 ref31 ref39 ref40">(28,31,39,40)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of BI-4464 bound to PTK2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Z">6I8Z</a>). Hydrogen bonds to cysteine 502 and aspartic acid 564 are depicted in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Chemical Structures of BI-4464 Ligand <b>1</b>, VHL Ligand <b>2</b>,<a onclick="showRef(event, 'ref37 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref37 ref41 ref42">(37,41,42)</a> and CRBN Ligand <b>3</b><a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binary Affinities of Compounds <b>1</b>, <b>6</b>, and <b>8</b> for PTK2 and the Respective PTK2 Degradation Data in A549 Cells</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±"> </th><th class="rowsep1 colsep0" colspan="2" align="center">A549 cells, 18 h<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">code</th><th class="colsep0 rowsep0" align="center"><i>N</i></th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">E3 ligase ligand</th><th class="colsep0 rowsep0" align="center" char="±">PTK2<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">pDC<sub>50</sub></th><th class="colsep0 rowsep0" align="center"><i>D</i><sub>max</sub> [%]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b> (BI-4464)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">7.8 ± 0.1</td><td class="colsep0 rowsep0" align="left">>10 000</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b> (BI-3663)</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">CONH</td><td class="colsep0 rowsep0" align="left">POMA</td><td class="colsep0 rowsep0" align="char" char="±">7.7 ± 0.1</td><td class="colsep0 rowsep0" align="left">7.6 ± 0.1</td><td class="colsep0 rowsep0" align="left">95 ± 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b> (BI-0319)</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">O–CH<sub>2</sub>–CONH</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char="±">7.7 ± 0.1</td><td class="colsep0 rowsep0" align="left">6.7 ± 0.4</td><td class="colsep0 rowsep0" align="left">80 ± 9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Thermo Fisher SelectScreen Kinase Profiling Services, Z′-Light, <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="febfaaaebeb593">[email protected]</a>, pIC<sub>50</sub> ± STDEV.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Degradation activity is reported as concentration needed to achieve 50% PTK2 protein degradation (pDC<sub>50</sub> ± STDEV) and maximal achievable protein degradation (<i>D</i><sub>max</sub>) relative to DMSO. PTK2 levels were determined by protein capillary electrophoresis and normalized to GAPDH (<i>N</i> = 3).</p></div></div></div><div class="NLM_p">To obtain the described compounds, <b>6</b> and <b>8</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), a convergent synthesis was developed to connect <b>1a</b> to each E3 ligase ligand, <b>2</b> and <b>3</b>. PEG linkers, 1–5 (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> shows an example of linker length three) were synthesized and profiled. The synthesis of ligand <b>1a</b> was carried out as outlined in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Aromatic nucleophilic substitution of 4-chloropyrimidine, <b>4</b>, with 7-hydroxy-2,3-dihydroindan-1-one produced the carboxylic acid, <b>1a</b>. Amide coupling of azido-PEG-3-propanoic acid and azido-PEG-3-acetic acid to the E3 ligase ligands, <b>2</b> and <b>3,</b> followed by catalytic hydrogenation produced the amines, <b>5b</b>, <b>7b</b>, and <b>10b</b> in good yields. The final PROTACs, <b>6</b>, <b>8</b>, and the <i>cis</i>-VHL analogue, <b>11</b>, were synthesized following amide coupling of the ligase linker conjugates, <b>5b</b>, <b>7b</b>, and <b>10b</b> with <b>1a</b>.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the PTK2 Degraders<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) Cs<sub>2</sub>CO<sub>3</sub>, MgSO<sub>4</sub>, dioxane, 80 °C, 16 h, 78%; (b) HATU, DIPEA, DMF, rt, 2 h, 75%; (c) T3P, pyridine, DMF, 80 °C, 3 h, 76%; (d) H<sub>2</sub> (6 bar), Pd/C (10%), MeOH, rt, 3 h, 90%; (e) <b>1a</b>, HATU, DIPEA, DMF, rt, 18 h, 10%; (f) HATU, DIPEA, DMF, rt, 1 h, 63%; (g) H<sub>2</sub> (5 bar), Pd/C (10%), MeOH, rt, 2.5 h, 82%; (h) <b>1a</b>, HATU, DIPEA, DMF, rt, 24 h, 34%; (i) HATU, DIPEA, DMF, rt, 2 h, 47%; (j) H<sub>2</sub> (6 bar), Pd/C (10%), MeOH, rt, 3 h, 98%; and (k) <b>1a</b>, HATU, DIPEA, DMF, rt, 16 h, 69%.</p></p></figure><div class="NLM_p">We next tested PTK2 degradation by all synthesized PROTACs (sampling linker lengths of up to five ethylene glycol units) after 16 h treatment of the human lung adenocarcinoma cell line, A549. PROTACs <b>6</b> (BI-3663) and <b>8</b> (BI-0319) were identified as the best degraders in the CRBN and VHL series and characterized in greater detail. The CRBN-based PROTAC <b>6</b> (BI-3663) degraded PTK2 potently (DC<sub>50</sub> = 27 nM) with a maximally obtainable degradation at the chosen experimental conditions of 95% (<i>D</i><sub>max</sub>), whereas the best VHL-based PROTAC <b>8</b> (BI-0319) only achieved partial degradation (<i>D</i><sub>max</sub> = 80%) and was less potent (DC<sub>50</sub> = 243 nM) in the A549 cells (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Furthermore, both PROTACs show a small apparent hook effect in the A549 cells at 25 μM. Consistent with VHL dependency, which requires the substituents of the pyrrolidine in the VHL ligand in a trans configuration,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> degradation of PTK2 by <b>8</b> (BI-0319) was abolished by switching the stereochemistry of the pyrrolidine substituents to the cis configuration (<b>11</b>, BI-4206) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Moreover, as predicted for neddylation-dependent E3 ligases such as VHL and CRBN, the NEDD8 inhibitor, MLN4924<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> (MLN) inhibited the degradation of PTK2 by <b>6</b> (BI-3663) and <b>8</b> (BI-0319) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B). Importantly, the PTK2 ligand <b>1</b> itself did not affect the PTK2 levels.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Characterization of BI-0319 and BI-3663 in A549 cells. (A) A549 cells were treated with the indicated concentrations of BI-0319 in the presence or absence of the NEDD8 inhibitor, MLN4924 (MLN, 3 μM) or an inactive stereoisomer (BI-4206) for 18 h. PTK2 levels were determined by protein capillary electrophoresis and normalized to GAPDH. The values are stated as percentages of dimethyl sulfoxide (DMSO) controls (POC). (B) A549 cells were treated with the indicated concentrations of BI-3663 in the presence or absence of the NEDD8 inhibitor, MLN4924 (3 μM), or with the PTK2 inhibitor, BI-4664, for 18 h. PTK2 levels were measured and are stated as in (A).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We assessed PTK2 target engagement in vitro to understand the effect of the linker and exit vector on PTK2 inhibition of PROTACs <b>6</b> and <b>8</b>. As predicted from the structure-based design, both PROTACs and the PTK2 ligand inhibited PTK2 in vitro with virtually identical potencies (IC<sub>50</sub> = 18 and 19 vs 17 nM for the PTK2 inhibitor <b>1</b>).</div><div class="NLM_p">In contrast to the previously published less selective PTK2 degraders,<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> we believe that a defining aspect of the PTK2 degraders described here is their high selectivity, which should enable validation of the relevance of PTK2 scaffolding functions in biologically relevant contexts. Indeed, of the 397 kinases tested in a kinase panel screening, only 2 were inhibited by more than 90% at 1 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Interestingly, BI-0319 (<b>8</b>) is more selective than the already highly selective PTK2 TKI <b>1</b> (BI-4464). We assume that the kinase selectivity panel for BI-3663 (<b>6</b>) might be comparable to BI-0319 because both the exit vector and linker are identical for a distance of nine atoms from the piperidine moiety of the PTK2 ligand.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Kinase selectivity panels (Thermo Fischer SelectScreen Kinase Profiling Services) of the PTK2 ligand, BI-4464 and VHL PROTAC, BI-0319. The bars indicate percent inhibition at 1 μM of either compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We employed multiplexed isobaric tagging mass spectrometry to assess the cellular selectivity of BI-3663 (<b>6</b>) and BI-0319 (<b>8</b>) for PTK2 degradation and identify potential degradation off-targets in a quantitative and unbiased manner. Among the 6008 proteins quantified in this analysis in A549 cells, PTK2 showed a distinct and significant change in abundance upon treatment with either PROTAC (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Table S3</a>). Neither BI-3663 (<b>6</b>) nor BI-0319 (<b>8</b>) induced any significant changes in abundance of other detectable kinases, thus confirming the high selectivity of both degraders within the kinase family. Of note, the two most prominent kinase off-targets of the inhibitor were not detected in this dataset. Interestingly, BI-0319 (<b>8</b>)—but not BI-3663 (<b>6</b>)—also induced a significant change of PDE6D levels (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), a finding corroborated by an immunoblot in the A549 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Figure S1</a>).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Total proteome analysis of A549 cells treated with 3 μM of BI-0319 (A) or BI-3663 (B) for 18 h and compared to DMSO controls. The samples were run in biological triplicates and analyzed by mass spectrometry. Volcano plot displays log<sub>2</sub> of fold-change in abundance versus −log<sub>10</sub> of adjusted <i>p</i> value (<i>N</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Having established the high selectivity of both PROTACs, in particular of the CRBN-based BI-3663, we tested their potential to degrade PTK2 in a panel of 11 HCC cell lines (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Figures S2 and S3</a>). Both PROTACs show comparable degradation potencies and efficacies in the 11 HCC cell lines tested (BI-3663 mean pDC<sub>50</sub> = 7.45 ± 0.60 versus BI-0319 mean pDC50 = 7.08 ± 0.52). Although, the PTK2 levels in cell lines such as SNU-398 were equally sensitive to treatment with both PROTACs, other cell lines, such as Hep3B2.1-7, exhibited a >10 fold difference of the potencies of <b>6</b> and <b>8</b> in cell-based PTK2 degradation assays, as observed for A549 cells during the profiling experiments (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cumulative analysis of half-maximal effective concentrations [DC<sub>50</sub>, (A)] and maximal degree of PTK2 degradation [<i>D</i><sub>max</sub> [%], (B)] achieved by BI-0319 or BI-3663. pDC<sub>50</sub> and <i>D</i><sub>max</sub> at 18 h were determined as indicated in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Figures S2 and S3</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. PTK2 Degradation Data of BI-3663 and BI-0319 in A549 and Hep3B2.1-7 Cells</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">A549, 18 h<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">Hep3B2.1-7, 18 h<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">Hep3B2.1-7, 18 h<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a>+ 10 μM CsnA</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±">pDC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char="±"><i>D</i><sub>max</sub> [%]</th><th class="colsep0 rowsep0" align="center" char=".">pDC<sub>50</sub></th><th class="colsep0 rowsep0" align="center"><i>D</i><sub>max</sub> [%]</th><th class="colsep0 rowsep0" align="center" char=".">pDC<sub>50</sub></th><th class="colsep0 rowsep0" align="center"><i>D</i><sub>max</sub> [%]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b> (BI-3663)</td><td class="colsep0 rowsep0" align="char" char="±">7.6 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">95 ± 4</td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="char" char=".">9.0</td><td class="colsep0 rowsep0" align="left">94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b> (BI-0319)</td><td class="colsep0 rowsep0" align="char" char="±">6.7 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">80 ± 9</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="left">59</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="left">88</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Degradation activity is reported as concentration needed to achieve 50% PTK2 protein degradation (pDC<sub>50</sub> ± STDEV) and maximal achievable protein degradation (<i>D</i><sub>max</sub>) relative to DMSO.</p></div></div></div><div class="NLM_p">It is not unexpected that heterobifunctional molecules unavoidably result in high molecular weight compounds, making it difficult to balance their physicochemical properties in a manner consistent with acceptable permeability and solubility. Although both PROTACs have a comparable low aqueous solubility and high tPSA (241, 243 Å<sup>2</sup>), PROTAC <b>8</b> has one hydrogen-bond donor more than PROTAC <b>6</b>. The negative impact on the permeability for <b>8</b> might be compensated by the higher lipophilicity (clog <i>P</i> 6.9 vs 3.8); however, the higher lipophilicity of <b>8</b> also translates in higher PPB and lower free fraction (<b>6</b>: 11.3%, <b>8</b>: 4.0; <a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Table S1</a>).</div><div class="NLM_p">As assessed by the Caco2 assay, both PROTACs exhibit low permeability and significant efflux (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Table S1</a>). Therefore, high efflux might compromise cellular exposures achieved by both PROTACs. We reasoned that blocking multidrug efflux pumps (such as the multidrug-resistance-protein 1, PGP) might boost the potency and efficacy of both PROTACs in the cell lines with a large discrepancy between biochemical and intracellular activity. Cyclosporine A (CsnA) has been described as a substrate of several drug transporters including PGP<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and can therefore be used to saturate these transporters at high concentrations. Consequently, we assessed PTK2 degradation by both PROTACs upon addition of CsnA to saturate drug transport in Hep3B2.1-7 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). Indeed, we observed a comparable shift in the potency of PTK2 degradation for both PROTACs (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), consistent with efflux contributing to their reduced degradation efficacy in this cell line. Of note, CsnA only had a minor effect on the PTK2 levels (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Analysis of PTK2 degradation in Hep3B2.1-7 cells. (A) Cells were treated with the indicated concentrations of BI-0319 for 18 h in the presence (red triangles) or absence (blue circles) of 10 μM CsnA. (B) Cells were treated for 18 h with the indicated concentrations of BI-3663 in the presence (red triangles) or absence (green circles) of 10 μM CsnA. (C) Cells were treated with 10 μM CsnA for 18 h. (D) Time course analysis of PTK2 levels in cells treated with 3 μM BI-0319. (E) Time course analysis of PTK2 levels in cells treated with 3 μM BI-3663. The values state PTK2 protein levels normalized to GAPDH relative to DMSO (or CsnA only) controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">PROTACs <b>6</b> and <b>8</b> differ considerably with respect to their cellular degradation rate in Hep3B2.1-7 cells. The CRBN-based PROTAC <b>6</b> achieved complete degradation of PTK2 after 5 h, whereas the VHL-based PROTAC <b>8</b> achieved maximal degradation only after 20 h (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D,E). Of note, the CRBN-based PROTAC <b>6</b> was considerably less stable in cell assay buffer containing 10% FCS (M + 18 and +32 observed) than the VHL-based PROTAC <b>8</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Table S1</a>). PROTAC 6 was found to be stable as a solid and in DMSO stock solution (>3 month, data not shown). Despite this previously reported instability of CRBN-based PROTACs,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> PROATC <b>6</b> and PROTAC <b>8</b> showed comparable maximal degradation of PTK2 after 18 and 72 h incubation, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Figure S6</a>). Taken together, this indicates that both PROTACs, <b>6</b> and <b>8</b>, are suitable probes to further study the role of PTK2 in a broad range of tumor cell lines.</div><div class="NLM_p">Reasoning that additional targeting of the scaffolding functions of PTK2 by PROTAC-mediated degradation might result in enhanced efficacy as compared to inhibition of its kinase activity, we next tested the effect of PROTAC-mediated PTK2 degradation on proliferation of the same HCC cell lines. Surprisingly, efficient depletion of PTK2 by both PROTACs (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Figures S2 and S3</a>) did not affect proliferation of the tested cell lines more severely than the PTK2 inhibitor BI-4464 in standard culture conditions or in anchorage-independent growth assays (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Figures S4 and S5</a>). Notably, sensitivity to genetic depletion of PTK2 has been described for several of the tested cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Table S4</a>). One explanation may be that effects on proliferation may require long-term depletion of PTK2. To further address this, we treated cell lines with reported high sensitivity to genetic depletion of PTK2<a onclick="showRef(event, 'ref36 ref46'); return false;" href="javascript:void(0);" class="ref ref36 ref46">(36,46)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Table S4</a>) for 21 days with either PROTAC <b>6</b> (BI-3663) or <b>8</b> (BI-0319) or the PTK2 inhibitor <b>1</b> (BI-4464). Despite the effective depletion of PTK2 even after 21 days, we did not observe any pronounced PROTAC-specific antiproliferative effects for any of the tested cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Figure S6</a>). Notably, CRISPR-based genetic validation in cultured HuH-1 cells also resulted in a weak dependency on PTK2 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Figure S7</a>). Taken together, these data question the scaffolding function of PTK2 being required for in vitro proliferation of the tested cell lines beyond the effect of inhibition of its kinase activity. Nevertheless, our observations indicate that both probes can be used to effectively reduce PTK2 protein levels in long term experiments in cultured cells.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Degradation Characteristics and Effect on Proliferation of BI-3663 (<b>6</b>) and BI-0319 (<b>8</b>) in HCC Lines</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">BI-3663</th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">BI-0319</th><th class="rowsep1 colsep0" align="center" char=".">BI-4464</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center" char=".">pDC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>D</i><sub>max</sub> [%]</th><th class="colsep0 rowsep0" align="center">pIC<sub>50</sub> (proliferation)</th><th class="colsep0 rowsep0" align="center" char=".">pDC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>D</i><sub>max</sub> [%]</th><th class="colsep0 rowsep0" align="center">pIC<sub>50</sub> (proliferation)</th><th class="colsep0 rowsep0" align="center" char=".">pIC<sub>50</sub> (proliferation)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SNU-387</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">90.0</td><td class="colsep0 rowsep0" align="left"><4.6</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td><td class="colsep0 rowsep0" align="char" char=".">98.0</td><td class="colsep0 rowsep0" align="left">5.3</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HUH-1</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="char" char=".">50.0</td><td class="colsep0 rowsep0" align="left">4.6</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="char" char=".">79.0</td><td class="colsep0 rowsep0" align="left">4.7</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Hep3B2.1-7</td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">96.0</td><td class="colsep0 rowsep0" align="left">4.6</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">59.0</td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HepG2</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td><td class="colsep0 rowsep0" align="char" char=".">89.0</td><td class="colsep0 rowsep0" align="left"><4.6</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">65.0</td><td class="colsep0 rowsep0" align="left"><4.6</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SK-Hep-1</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td><td class="colsep0 rowsep0" align="char" char=".">89.0</td><td class="colsep0 rowsep0" align="left">5.8</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">87.0</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HLF</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td><td class="colsep0 rowsep0" align="char" char=".">30.0</td><td class="colsep0 rowsep0" align="left"><4.6</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">66.0</td><td class="colsep0 rowsep0" align="left"><4.6</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SNU-398</td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">95.0</td><td class="colsep0 rowsep0" align="left"><4.6</td><td class="colsep0 rowsep0" align="char" char=".">8.0</td><td class="colsep0 rowsep0" align="char" char=".">99.0</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HUCCT1</td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">90.0</td><td class="colsep0 rowsep0" align="left"><4.6</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">86.0</td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HLE</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="char" char=".">79.0</td><td class="colsep0 rowsep0" align="left"><4.6</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">85.0</td><td class="colsep0 rowsep0" align="left">5.0</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HuH-7</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">93.0</td><td class="colsep0 rowsep0" align="left"><4.6</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="char" char=".">79.0</td><td class="colsep0 rowsep0" align="left">5.4</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SNU-423</td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">93.0</td><td class="colsep0 rowsep0" align="left">4.7</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="char" char=".">87.0</td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td></tr></tbody></table></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42821" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42821" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We describe the development of two highly selective and functional PROTACs to degrade the PTK2 protein, a kinase of significant relevance to cancer research, in particular, in the area of HCC. Structure-guided conjugation of a highly selective PTK2 inhibitor BI-4464 to either a CRBN or VHL ligand via polyethylene glycol linkers led to the selective PTK2 degraders BI-0319 and BI-3663, respectively, using orthogonal E3 ligase binders. Both PROTACs were characterized with respect to in vitro PTK2 engagement, ligase dependence selectivity (kinase inhibition and whole cell proteome), and degradation efficacy (DC<sub>50</sub> and <i>D</i><sub>max</sub>) in 12 cell lines (one lung cancer and 11 HCC cell lines). Both PROTACs are highly selective E3 ligase-dependent PTK2 degraders and show overall comparable potencies and efficacies of PTK2 degradation in the 12 cell lines tested. Although PTK2 levels in cell lines such as SNU-398 were equally sensitive to treatment with both PROTACs, other cell lines, such as Hep3B2.1-7, exhibited a more than ten-fold difference in cellular potency. This difference in degradation efficacy in Hep3B2.1-7 cells was associated with a higher degradation rate of BI-3663. Despite the efficient depletion of PTK2, treatment with either PROTAC did not affect proliferation of the tested cell lines in either short or long term assays in vitro (6 vs 21 days) beyond the effect achieved by a PTK2 kinase inhibitor. Taken together, these data suggest that PROTACs <b>6</b> (BI-3663)- and <b>8</b> (BI-0319)-mediated PTK2 depletion is likely insufficient to affect proliferation in vitro under the conditions tested beyond the effect of inhibition of its kinase activity. Nevertheless, both probes provide valuable tools to effectively reduce PTK2 protein levels in experimental conditions that might be better suited to reveal and differentiate between kinase-dependent and -independent functions of PTK2. BI-3663, BI-0319, and its inactive control BI-4206 will be made available to all scientists via our open innovation portal opnMe.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Synthesis</h4><div class="NLM_p">Unless otherwise indicated, all reactions were carried out in standard commercially available glassware using standard synthetic chemistry methods. Air-sensitive and moisture-sensitive reactions were performed under an atmosphere of dry nitrogen or argon with dried glassware. Commercial starting materials were used without further purification. Solvents used for reactions were of commercial “dry” or “extra-dry” or “analytical” grade. All other solvents used were reagent grade.</div><div class="NLM_p">Thin layer chromatography is carried out on ready-made silica gel 60 TLC plates on glass (with fluorescence indicator F-254) made by Merck. The preparative high pressure chromatography (RP-HPLC) is carried out on Gilson systems with columns made by Waters (names: Sunfire Prep C18, OBD 10 μm, 50 × 150 mm or XBridge Prep C18, OBD 10 μm, 50 × 150 mm) and YMC (names: Actus-Triart Prep C18, 5 μm, 30 × 50 mm). Different gradients of MeCN/H<sub>2</sub>O are used to elute the compounds; for acidic conditions, 0.1% HCOOH is added to the water. For the chromatography under basic conditions, the water is made alkaline as follows: 5 mL NH<sub>4</sub>HCO<sub>3</sub> solution (158 g in 1 L H<sub>2</sub>O) and 2 mL NH<sub>3</sub> (28% in H<sub>2</sub>O) are replenished to 1 L with H<sub>2</sub>O.</div><div class="NLM_p">All compounds had a purity >95% according to the HPLC and NMR analysis acquired with the systems and parameters stated as follows. HPLC samples were analyzed on an Agilent 1200 series LC system coupled with an Agilent 6140 mass spectrometer. Purity was determined via UV detection with a bandwidth of 170 nm in the range from 230 to 400 nm. LC parameters were as follows: Waters Xbridge C18 column, 2.5 μm particle size, 2.1 × 20 mm. Total run time 3.1 min, flow 1 mL/min, column temperature 60 °C, and 5 μL injections. Solvent A (20 mM NH<sub>4</sub>HCO<sub>3</sub>/NH<sub>3</sub> pH 9), solvent B (MS grade acetonitrile). Start 10% B, gradient 10–95% B from 0.0 to 1.5 min, 95% B from 1.5 to 2.0 min, gradient 95–10% B from 2.0 to 2.1 min.</div><div class="NLM_p">NMR experiments were recorded on a Bruker AVANCE HD 500 MHz spectrometer equipped with a TCI cryoprobe at 298 K. The samples were dissolved in 600 μL DMSO-<i>d</i><sub>6,</sub> and TMS was added as an internal standard. 1D <sup>1</sup>H spectra were acquired with 30° excitation pulses and an interpulse delay of 4.2 s with 64k data points and 20 ppm sweep width. 1D <sup>13</sup>C spectra were acquired with broadband composite pulse decoupling (WALTZ16) and an interpulse delay of 3.3 s with 64k data points and a sweep width of 240 ppm. Processing and analysis of 1D spectra was performed with Bruker Topspin 3.2 software. No zero filling was performed, and the spectra were manually integrated after automatic baseline correction. Chemical shifts are reported in ppm on the δ scale.</div><div class="NLM_p">HSQC spectra were recorded on all samples to aid the interpretation of the data and to identify signals hidden underneath solvent peaks. Spectra were acquired with sweep widths obtained by automatic sweep width detection from 1D reference spectra in the direct dimension with 1k datapoints and with 210 ppm and 256 datapoints in the indirect dimension.</div><div class="NLM_p last">For HRMS, an LTQ Orbitrap XL (Thermo Scientific) coupled with a Triversa Nanomate Nanospray ion source (ADVION Bioscience Inc.) was used.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> 3-Methoxy-4-({4-[(3-oxo-2,3-dihydro-1H-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)benzoic Acid (<b>1a</b>)</h4><div class="NLM_p last">4-{[4-Chloro-5-(trifluoromethyl)pyrimidin-2-yl]amino}-3-methoxybenzoic acid (<b>4</b>,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> 1.00 g, 1 equiv) and 7-hydroxy-2,3-dihydro-1<i>H</i>-inden-1-one (879 mg, 2 equiv) were taken up in dioxane (10 mL), and then Cs<sub>2</sub>CO<sub>3</sub> (4.686 g, 5 equiv) was added. The reaction mixture was stirred at 80 °C for 16 h. The reaction mixture was diluted with H<sub>2</sub>O and acetonitrile and concentrated under reduced pressure. The residue was purified by preparative RP-HPLC under basic conditions using MeCN/H<sub>2</sub>O as eluents in a gradient from 20:80 to 65:35 over 12 min. (column: XBridge Prep C18, OBD 10 μm, 50 × 150 mm; flow: 110 mL/min). Product containing fractions were freeze-dried to give 3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)benzoic acid (<b>1a</b>, 1.03 g, 78% yield) as a brown solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.85 (br s, 1H), 8.86 (s, 1H), 8.74 (s, 1H), 7.80 (t, <i>J</i> = 7.88 Hz, 1H), 7.60 (d, <i>J</i> = 7.88 Hz, 1H), 7.41 (d, <i>J</i> = 0.95 Hz, 1H), 7.32 (br d, <i>J</i> = 1.00 Hz, 1H), 7.24 (d, <i>J</i> = 7.88 Hz, 1H), 7.15 (br d, <i>J</i> = 1.00 Hz, 1H), 3.80 (s, 3H), 3.05–3.17 (m, 2H), and two protons under DMSO. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 203.4, 167.3, 166.4, 161.0, 158.4, 157.8, 149.7, 148.0, 137.0, 131.7, 128.9, 126.5, 125.5, 121.8, 121.1, 123.9, 120.4, 111.7, 101.5, 56.3, 36.7, and 25.8. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>16</sub>F<sub>3</sub>N<sub>3</sub>O<sub>5</sub>, 459.10421; found, 459.10411. HPLC–MS <sup>t</sup>R = 1.00 min.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> 3-Methoxy-<i>N</i>-(1-methylpiperidin-4-yl)-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)benzamide (<b>1</b>, BI-4464)</h4><div class="NLM_p last">In a glass vial, 3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)benzoic acid (<b>1a</b>, 100 mg, 1 equiv) was dissolved in DMF (1 mL). DIPEA (86 μL, 3 equiv) and HATU (114 mg, 1.5 equiv) were added. After the reaction mixture was stirred at rt for 5 min, 1-methylpiperidin-4-amine (46 mg, 52 μL, 2 equiv) was added. The mixture was stirred at room temperature (rt) for 2 h. The reaction mixture was diluted with MeCN and H<sub>2</sub>O and filtered through a syringe filter prior to purification via RP-HPLC under basic conditions using MeCN/H<sub>2</sub>O as eluents in a gradient from 30:70 to 98:2 over 8 min (column: YMC Actus-Triart Prep C18, 5 μm, 30 × 50 mm; flow: 50 mL/min). The product-containing fractions were freeze-dried to give 3-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)benzamide [<b>1</b>, (BI-4464), 83 mg, 75% yield] as an off-white lyophilizate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.83 (br s, 1H), 8.69 (s, 1H), 8.13 (d, <i>J</i> = 7.57 Hz, 1H), 7.77 (dd, <i>J</i> = 7.57, 7.88 Hz, 1H), 7.55 (d, <i>J</i> = 7.57 Hz, 1H), 7.38 (d, <i>J</i> = 0.95 Hz, 1H), 7.22 (d, <i>J</i> = 7.88 Hz, 1H), 7.27 (br s, 1H), 7.12 (br s, 1H), 3.78 (s, 3H), 3.66–3.75 (m, 1H), 3.04–3.15 (m, 2H), 2.78 (br d, <i>J</i> = 11.35 Hz, 2H), 2.16 (s, 3H), 1.85–1.98 (m, 2H), 1.69–1.80 (m, 2H), 1.49–1.65 (m, 2H), and two protons under DMSO. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 201.3, 164.2, 163.2, 159.1, 156.2, 155.7, 148.0, 145.8, 134.8, 128.8, 127.8, 126.8, 123.3, 118.8, 121.9, 117.5, 108.1, 99.0, 54.2, 53.0, 45.0, 44.4, 34.6, 29.9, 23.7, and one carbon not detected. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>28</sub>F<sub>3</sub>N<sub>5</sub>O<sub>4</sub>, 555.20934; found, 555.20824. HPLC–MS <sup>t</sup>R = 1.39 min.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 3-{2-[2-(2-Azidoethoxy)ethoxy]ethoxy}-<i>N</i>-[2-(2,6-dioxopiperidin-3-yl)-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl]propanamide (<b>5a</b>)</h4><div class="NLM_p last">In a 50 mL round-bottom flask, 4-amino-2-(2,6-dioxo-3-piperidyl)isoindoline-1,3-dione (500 mg, 1 equiv) was dissolved in DMF (5 mL) and cooled to 0 °C in an ice bath. 3-{2-[2-(2-Azidoethoxy)ethoxy]ethoxy}propanoic acid (905 mg, 2 equiv) was added dropwise. Then, <i>N</i>-propylphosphonic acid anhydride, cyclic trimer (7.0 g, 6.5 mL, 6 equiv) and pyridine (1.5 mL, 10 equiv) were added. The mixture was stirred at 80 °C for 3 h. The reaction mixture was diluted with ACN/H<sub>2</sub>O. The solution was filtered through a syringe filter and purified by preparative RP-HPLC under basic conditions using MeCN/H<sub>2</sub>O as eluents in a gradient from 10:90 to 60:40 over 9 min (column: XBridge Prep C18, OBD 10 μm, 50 × 150 mm; flow: 150 mL/min). The product-containing fractions were freeze-dried to give 3-{2-[2-(2-azidoethoxy)ethoxy]ethoxy}-<i>N</i>-[2-(2,6-dioxopiperidin-3-yl)-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl]propanamide (<b>5a</b>, 697 mg, 76% yield) as a yellow lyophylizate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.15 (s, 1H), 9.88 (s, 1H), 8.54 (d, <i>J</i> = 8.31 Hz, 1H), 7.83 (dd, <i>J</i> = 7.26, 8.31 Hz, 1H), 7.62 (d, <i>J</i> = 7.26 Hz, 1H), 5.14 (dd, <i>J</i> = 5.52, 12.77 Hz, 1H), 3.74 (t, <i>J</i> = 5.83 Hz, 2H), 3.52–3.61 (m, 6H), 3.47–3.52 (m, 4H), 3.34–3.37 (m, 2H), 2.84–2.96 (m, 1H), 2.70 (t, <i>J</i> = 5.99 Hz, 2H), 2.53–2.66 (m, 2H), and 2.00–2.12 (m, 1H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.2, 170.9, 170.2, 168.1, 167.1, 136.9, 136.6, 131.9, 126.5, 118.7, 117.1, 70.2, 70.2, 70.1, 70.1, 69.7, 66.6, 50.4, 49.4, 38.0, 31.4, and 22.4. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>N<sub>6</sub>O<sub>8</sub>, 502.18121; found, 502.18114. HPLC–MS <sup>t</sup>R = 1.05 min.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 3-{2-[2-(2-Aminoethoxy)ethoxy]ethoxy}-<i>N</i>-[2-(2,6-dioxopiperidin-3-yl)-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl]propanamide (<b>5b</b>)</h4><div class="NLM_p last">In a Parr-reactor, 3-{2-[2-(2-azidoethoxy)ethoxy]ethoxy}-<i>N</i>-[2-(2,6-dioxopiperidin-3-yl)-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl]propanamide (<b>5a</b>, 190 mg, 1 equiv) was dissolved in methanol (5 mL) and Pd/C (10%, 40 mg, 0.1 equiv) was added. The reactor was flushed with N<sub>2</sub> and filled with H<sub>2</sub> (6 bar). The reaction mixture was stirred at rt for 3 h. A spoon of celite was added to the reaction mixture, the catalyst was filtered off through a celite pad, and rinsed with DCM and MeOH. The filtrate was concentrated under reduced pressure. 3-{2-[2-(2-Aminoethoxy)ethoxy]ethoxy}-<i>N</i>-[2-(2,6-dioxopiperidin-3-yl)-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl]propanamide (<b>5b</b>, 162 mg, 90% yield) was obtained as a yellow resin. The product was taken to the next step without further purification. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 9.89 (br s, 1H), 8.54 (d, <i>J</i> = 8.20 Hz, 1H), 8.36 (s, 1H), 7.84 (dd, <i>J</i> = 8.20, 7.25 Hz, 1H), 7.62 (br d, <i>J</i> = 7.25 Hz, 1H), 5.14 (dd, <i>J</i> = 12.93, 5.36 Hz, 1H), 3.71–3.77 (m, 2H), 3.45–3.60 (m, overlapped), 2.85–2.96 (m, 2H), 2.82 (br t, <i>J</i> = 5.36 Hz, 2H), 2.71 (br t, <i>J</i> = 5.83 Hz, 2H), and 2.02–2.11 (m, 1H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.3, 170.9, 170.3, 168.1, 167.1, 136.9, 136.7, 131.9, 126.5, 118.8, 117.2, 70.2, 70.1, 70.1, 70.0, 69.2, 66.6, 49.4, 39.8, 38.0, 31.4, and 22.4. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>4</sub>O<sub>8</sub>, 476.19071; found, 476.19041. HPLC–MS <sup>t</sup>R = 0.81 min.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> <i>N</i>-[2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl]-3-{2-[2-(2-{[3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)phenyl]formamido}ethoxy)ethoxy]ethoxy}propanamide (<b>6</b>, BI-3663)</h4><div class="NLM_p last">In a glass vial, 3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)benzoic acid (<b>2</b>, 170 mg, 1 equiv) was dissolved in DMF (2 mL). DIPEA (150 μL, 3 equiv) and HATU (155 mg, 1.2 equiv) were added. The reaction mixture was stirred at rt for 5 min, then 3-{2-[2-(2-aminoethoxy)ethoxy]ethoxy}-<i>N</i>-[2-(2,6-dioxopiperidin-3-yl)-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl]propanamide (<b>5b</b>, 162 mg, 1 equiv) was added. The mixture was stirred at rt overnight. The reaction mixture was diluted with MeCN and H<sub>2</sub>O and filtered through a syringe filter prior to purification via RP-HPLC under acidic conditions using MeCN/H<sub>2</sub>O as eluents in a gradient from 30:70 to 98:2 over 8 min (column: YMC Actus-Triart Prep C18, 5 μm, 30 × 50 mm; flow: 50 mL/min). The product-containing fractions were freeze-dried to give <i>N</i>-[2-(2,6-dioxopiperidin-3-yl)-1,3-dioxo-2,3-dihydro-1<i>H</i>-isoindol-4-yl]-3-{2-[2-(2-{[3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)phenyl]formamido}ethoxy)ethoxy]ethoxy}propanamide [<b>6</b> (BI-3663), 32 mg, 10% yield] as an off-white lyophilizate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.13 (br s, 1H), 9.86 (s, 1H), 8.79 (br s, 1H), 8.70 (s, 1H), 8.53 (d, <i>J</i> = 8.51 Hz, 1H), 8.43 (t, <i>J</i> = 5.52 Hz, 1H), 7.81 (dd, <i>J</i> = 7.25, 8.51 Hz, 1H), 7.77 (t, <i>J</i> = 7.88 Hz, 1H), 7.60 (d, <i>J</i> = 7.25 Hz, 1H), 7.56 (d, <i>J</i> = 7.88 Hz, 1H), 7.40 (s, 1H), 7.21 (d, <i>J</i> = 7.88 Hz, 1H), 7.25 (br s, 1H), 7.10 (br s, 1H), 5.14 (dd, <i>J</i> = 5.52, 12.77 Hz, 1H), 3.78 (s, 3H), 3.71 (t, <i>J</i> = 5.99 Hz, 2H), 3.43–3.61 (m, 10H), 3.36–3.41 (m, 2H), 3.07–3.15 (m, 2H), 2.84–2.95 (m, 1H), 2.68 (t, <i>J</i> = 5.99 Hz, 2H), 2.52–2.65 (m, 2H), 2.02–2.11 (m, 1H), and two protons under DMSO. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 203.4, 173.2, 170.9, 170.3, 168.1, 167.1, 166.4, 165.9, 161.1, 158.3, 157.8, 149.9, 147.9, 136.9, 136.9, 136.6, 131.9, 130.4, 130.0, 128.9, 126.4, 125.5, 120.9, 124.0, 119.5, 118.7, 117.1, 110.2, 101.2, 70.2, 70.2, 70.1, 70.0, 69.5, 66.6, 56.3, 49.4, 39.6, 38.0, 36.8, 31.4, 25.8, 22.4, and one carbon not detected. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>44</sub>H<sub>42</sub>F<sub>3</sub>N<sub>7</sub>O<sub>12</sub>, 917.28435; found, 917.28471. HPLC–MS <sup>t</sup>R = 1.35 min.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (2<i>S</i>,4<i>R</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-Azidoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethyl-butanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>7a</b>)</h4><div class="NLM_p last">(2<i>S</i>,4<i>R</i>)-1-[(2<i>S</i>)-2-Amino-3,3-dimethyl-butanoyl]-4-hydroxy-<i>N</i>-[[4-(4-methylthiazol-5-yl)phenyl]methyl]pyrrolidine-2-carboxamide hydrochloride (<b>2</b>, 1.000 g, 1 equiv) was dissolved in DMF (5 mL). HATU (855 mg, 1.05 equiv) and DIPEA (1.28 mL, 3.5 equiv) were added. The reaction mixture was stirred for 15 min, then 2-[2-[2-(2-azidoethoxy)ethoxy]ethoxy]acetic acid (524 mg, 1.05 equiv) was added. The reaction mixture was stirred at rt for 1 h. The reaction mixture was diluted with EtOAc and washed with a saturated solution of NaHCO<sub>3</sub> and then with brine. The organic layer was dried over MgSO<sub>4</sub> and evaporated. The residue was purified via RP-HPLC under acidic conditions using MeCN/H<sub>2</sub>O as eluents in a gradient from 10:90 to 98:2 over 10 min (column: Sunfire Prep C18, OBD 10 μm, 50 × 150 mm; flow: 120 mL/min). The product-containing fractions were freeze-dried to give (2<i>S</i>,4<i>R</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-azidoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>7a</b>, 870 mg, 63% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.98 (s, 1H), 8.60 (t, <i>J</i> = 5.99 Hz, 1H), 7.35–7.47 (m, 5H), 4.67–5.61 (m, 1H), 4.57 (d, <i>J</i> = 9.77 Hz, 1H), 4.32–4.48 (m, 3H), 4.21–4.29 (m, 1H), 3.97 (s, 2H), 3.64–3.69 (m, 1H), 3.53–3.64 (m, 12H), 2.44 (s, 3H), 2.02–2.09 (m, 1H), 1.85–1.94 (m, 1H), 0.93–0.96 (m, 8H), 0.94 (s, 1H), and one proton under water. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.2, 169.6, 169.1, 152.0, 148.2, 139.9, 131.6, 130.2, 129.2, 127.9, 70.9, 70.4, 70.2, 70.1, 70.1, 69.7, 69.3, 59.2, 57.0, 56.1, 50.4, 42.1, 38.4, 36.2, 26.6, and 16.4. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>43</sub>N<sub>7</sub>O<sub>7</sub>S, 645.29447; found, 645.29442. HPLC–MS <sup>t</sup>R = 1.16 min.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (2<i>S</i>,4<i>R</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-Aminoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>7b</b>)</h4><div class="NLM_p last">(2<i>S</i>,4<i>R</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-Azidoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>7a</b>, 1.475 g, 1 equiv) was dissolved in MeOH (10 mL), Pd/C (250 mg, 10%) was added, and the mixture was hydrogenated (5 bar H<sub>2</sub> pressure) at rt for 2.5 h. The catalyst was filtered off, and the filtrate was purified via RP-HPLC under basic conditions using MeCN/H<sub>2</sub>O as eluents in a gradient from 10:90 to 90:10 over 10 min (column: XBridge Prep C18, OBD 10 μm, 50 × 150 mm; flow: 140 mL/min). The product-containing fractions were freeze-dried to give (2<i>S</i>,4<i>R</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-aminoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>7b</b>, 1.166 g, 82% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.00 (s, 1H), 8.64 (t, <i>J</i> = 6.00 Hz, 1H), 7.37–7.49 (m, 5H), 5.17 (br s, 1H), 4.58 (br d, <i>J</i> = 9.46 Hz, 1H), 4.21–4.48 (m, 4H), 3.99 (s, 2H), 3.65–3.71 (m, 1H), 3.51–3.65 (m, 9H), 3.46 (t, <i>J</i> = 5.50 Hz, 1H), 2.79 (t, <i>J</i> = 5.67 Hz, 1H), 2.45 (s, 3H), 2.02–2.11 (m, 1H), 1.86–1.96 (m, 1H), 0.95 (s, 9H), and two protons not detected (NH<sub>2</sub>). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.2, 169.6, 169.1, 152.0, 148.2, 139.9, 131.6, 130.2, 129.2, 127.9, 70.9, 70.4, 70.2, 70.1, 70.1, 70.0, 69.3, 59.2, 57.1, 56.2, 42.1, 38.4, 36.2, 26.7, 16.4, and one carbon not detected. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>45</sub>N<sub>5</sub>O<sub>7</sub>S, 619.30397; found, 619.30333. HPLC–MS <sup>t</sup>R = 1.00 min.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-[(2<i>S</i>)-2-(2-{2-[2-(2-{[3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)phenyl]formamido}ethoxy) ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>8</b>, BI-0319)</h4><div class="NLM_p last">In a 50 mL round-bottom flask, 3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)benzoic acid (<b>2</b>, 111 mg, 1 equiv)was dissolved in DMF (3 mL). DIPEA (96 μL, 3 equiv) and HATU (101 mg; 266 μmol; 1.2 equiv) were added. The reaction mixture was stirred at rt for 5 min, and then (2<i>S</i>,4<i>R</i>)-1-[(2<i>S</i>)-2-[[2-[2-[2-(2-aminoethoxy)ethoxy] ethoxy]acetyl]amino]-3,3-dimethyl-butanoyl]-4-hydroxy-<i>N</i>-[[4-(4-methylthiazol-5-yl)phenyl]methyl]pyrrolidine-2-carboxamide (<b>7b</b>, 150 mg, 1 equiv) was added. The mixture was stirred at rt for 24 h. The reaction mixture was diluted with MeCN and H<sub>2</sub>O and filtered through a syringe filter prior to purification via RP-HPLC under basic conditions using MeCN/H<sub>2</sub>O as eluents in a gradient from 25:75 to 90:10 over 9 min (column: XBridge Prep C18, OBD 10 μm, 50 × 150 mm; flow: 150 mL/min). The product-containing fractions were pooled and freeze-dried to give (2<i>S</i>,4<i>R</i>)-4-hydroxy-1-[(2<i>S</i>)-2-(2-{2-[2-(2-{[3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)phenyl]formamido}ethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide [<b>8</b> (BI-0319), 81 mg, 34% yield] as a colorless lyophilizate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.97 (s, 1H), 8.80 (br s, 1H), 8.70 (s, 1H), 8.59 (t, <i>J</i> = 5.99 Hz, 1H), 8.46 (t, <i>J</i> = 5.67 Hz, 1H), 7.77 (dd, <i>J</i> = 7.57, 7.88 Hz, 1H), 7.56 (d, <i>J</i> = 7.57 Hz, 1H), 7.35–7.46 (m, 6H), 7.21 (d, <i>J</i> = 7.88 Hz, 1H), 7.23 (br s, 1H), 7.11 (br s, 1H), 5.15 (d, <i>J</i> = 3.47 Hz, 1H), 4.56 (d, <i>J</i> = 9.77 Hz, 1H), 4.20–4.48 (m, 4H), 3.96 (s, 2H), 3.77 (s, 3H), 3.36–3.71 (m, 16H), 3.06–3.15 (m, 2H), 2.43 (s, 3H), 2.00–2.10 (m, 1H), 1.85–1.94 (m, 1H), and 0.93 (s, 9H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 203.4, 172.2, 169.6, 169.1, 166.4, 166.0, 161.1, 158.3, 157.8, 151.9, 150.0, 148.2, 147.9, 139.9, 136.9, 131.6, 130.4, 130.2, 130.0, 129.1, 128.9, 127.9, 125.5, 121.0, 124.0, 119.5, 110.2, 101.3, 101.1, 70.9, 70.3, 70.2, 70.1, 70.0, 69.5, 69.3, 59.2, 57.0, 56.3, 56.2, 42.1, 38.4, 36.8, 36.2, 26.6, 25.8, and 16.4. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>52</sub>H<sub>59</sub>F<sub>3</sub>N<sub>8</sub>O<sub>11</sub>S, 1060.39761; found, 1060.39604. HPLC–MS <sup>t</sup>R = 1.35 min.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (2<i>S</i>,4<i>S</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-Azidoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>10a</b>)</h4><div class="NLM_p last">In a glass vial, 2-[2-[2-(2-azidoethoxy) ethoxy]ethoxy]acetic acid (<b>9</b>, 569 mg, 1.05 equiv) was dissolved in DMF (1 mL) and DIPEA (768 μL, 2 equiv). HATU (1.324 g 1.5 equiv) and (2<i>S</i>,4<i>S</i>)-1-[(2<i>S</i>)-2-amino-3,3-dimethylbutanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (1.000 g, 1 equiv) were added. The mixture was stirred at rt for 2 h. The reaction mixture was diluted with EtOAc and washed with a saturated solution of NaHCO<sub>3</sub> then with brine. The organic layer was evaporated, and the residue was dissolved in MeOH and purified via RP-HPLC under acidic conditions using MeCN/H<sub>2</sub>O as eluents in a gradient from 15:85 to 98:2 over 10 min (column: Sunfire Prep C18, OBD 10 μm, 50 × 150 mm; flow: 120 mL/min). The product-containing fractions were freeze-dried to give (2<i>S</i>,4<i>S</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-azidoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>10a</b>, 700 mg, 47% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.99 (s, 1H), 8.67 (t, <i>J</i> = 5.99 Hz, 1H), 7.36–7.43 (m, 5H), 5.36 (br s, 1H), 4.52 (d, <i>J</i> = 9.14 Hz, 1H), 4.18–4.47 (m, 4H), 3.95 (s, 2H), 3.84–3.91 (m, 1H), 3.51–3.64 (m, 11H), 3.43–3.48 (m, 1H), 2.44 (s, 3H), 2.29–2.39 (m, 1H), 1.68–1.79 (m, 1H), 0.96 (s, 9H), and one proton under water. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.7, 169.8, 169.3, 152.0, 148.2, 139.6, 131.6, 130.2, 129.2, 127.9, 70.9, 70.3, 70.2, 70.1, 70.0, 69.7, 69.5, 59.0, 56.3, 56.1, 50.4, 42.3, 37.4, 35.6, 26.6, and 16.4. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>43</sub>N<sub>7</sub>O<sub>7</sub>S, 645.29447; found, 645.29368. HPLC–MS <sup>t</sup>R = 1.18 min.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (2<i>S</i>,4<i>S</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-Aminoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethyl butanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>10b</b>)</h4><div class="NLM_p last">In a Parr-reactor, (2<i>S</i>,4<i>S</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-azidoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>10a</b>, 153 mg, 1 equiv) was dissolved in methanol (5 mL) and Pd/C (10%, 25 mg, 0.1 equiv) was added. The reactor was flushed with N<sub>2</sub> and filled with H<sub>2</sub> (6 bar). The reaction mixture was allowed to be at rt for 3 h. A spoon of celite was added to the reaction mixture, the catalyst was filtered off through a pad of celite, and rinsed with DCM and MeOH. The filtrate was concentrated under reduced pressure. (2<i>S</i>,4<i>S</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-Aminoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>10b</b>, 144 mg, 98% yield) was obtained as a colorless resin. The product was taken to the next step without further purification. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.99 (s, 1H), 8.72 (br t, <i>J</i> = 5.83 Hz, 1H), 8.36 (br s, 1H), 7.40 (s, 4H), 5.32 (br s, 1H), 4.52 (d, <i>J</i> = 9.14 Hz, 1H), 4.18–4.45 (m, 4H), 3.96 (s, 2H), 3.86–3.91 (m, 1H), 3.42–3.62 (m, 11H), 2.82 (br s, 2H), 2.44 (s, 3H), 2.31–2.38 (m, 1H), 1.68–1.79 (m, 1H), 0.96 (s, 9H), and two protons not detected (NH<sub>2</sub>). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.7, 169.8, 169.4, 152.0, 148.2, 139.7, 131.6, 130.2, 129.2, 127.9, 70.8, 70.2, 70.1, 70.1, 70.0, 69.5, 59.0, 56.3, 56.1, 42.3, 37.4, 35.7, 26.6, 16.4, and two carbons not detected. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>45</sub>N<sub>5</sub>O<sub>7</sub>S, 619.30397; found, 619.30308. HPLC–MS <sup>t</sup>R = 1.01 min.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (2<i>S</i>,4<i>S</i>)-4-Hydroxy-1-[(2<i>S</i>)-2-(2-{2-[2-(2-{[3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)phenyl]formamido}ethoxy) ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>11</b>, BI-4206)</h4><div class="NLM_p last">In a glass vial, 3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)benzoic acid (<b>2</b>, 116 mg, 1 equiv) was dissolved in DMF (2 mL). DIPEA (100 μL, 3 equiv) and HATU (106 mg, 1.2 equiv) were added. The reaction mixture was stirred at rt for 5 min, and then (2<i>S</i>,4<i>S</i>)-1-[(2<i>S</i>)-2-(2-{2-[2-(2-aminoethoxy)ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-4-hydroxy-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide (<b>10b</b>, 144 mg, 1 equiv) was added. The mixture was stirred at rt for 16 h. The reaction mixture was diluted with MeCN and H<sub>2</sub>O and filtered through a syringe filter prior to purification via RP-HPLC under basic conditions using MeCN/H<sub>2</sub>O as the eluents in a gradient from 30:70 to 98:2 over 8 min (column: YMC Actus-Triart Prep C18, 5 μm, 30 × 50 mm; flow: 50 mL/min). The product-containing fractions were freeze-dried to give (2<i>S</i>,4<i>S</i>)-4-hydroxy-1-[(2<i>S</i>)-2-(2-{2-[2-(2-{[3-methoxy-4-({4-[(3-oxo-2,3-dihydro-1<i>H</i>-inden-4-yl)oxy]-5-(trifluoromethyl)pyrimidin-2-yl}amino)phenyl]formamido}ethoxy) ethoxy]ethoxy}acetamido)-3,3-dimethylbutanoyl]-<i>N</i>-{[4-(4-methyl-1,3-thiazol-5-yl)phenyl]methyl}pyrrolidine-2-carboxamide [<b>11</b>, (BI-4206), 169 mg, 69% yield] as an off-white lyophilizate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.97 (s, 1H), 8.81 (br s, 1H), 8.70 (s, 1H), 8.66 (t, <i>J</i> = 5.99 Hz, 1H), 8.45 (t, <i>J</i> = 5.52 Hz, 1H), 7.77 (dd, <i>J</i> = 7.57, 7.88 Hz, 1H), 7.56 (d, <i>J</i> = 7.57 Hz, 1H), 7.35–7.45 (m, 6H), 7.21 (d, <i>J</i> = 7.88 Hz, 1H), 7.25 (br s, 1H), 7.11 (br s, 1H), 5.45 (br d, <i>J</i> = 6.62 Hz, 1H), 4.51 (d, <i>J</i> = 9.14 Hz, 1H), 4.17–4.45 (m, 4H), 3.94 (s, 2H), 3.84–3.91 (m, 1H), 3.77 (s, 3H), 3.36–3.62 (m, 15H), 3.07–3.15 (m, 2H), 2.43 (s, 3H), 2.29–2.36 (m, 1H), 1.69–1.79 (m, 1H), and 0.95 (s, 9H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 203.4, 172.7, 169.8, 169.4, 166.4, 166.0, 161.1, 158.3, 157.8, 151.9, 150.0, 148.2, 147.9, 139.6, 136.9, 131.6, 130.4, 130.2, 130.0, 129.2, 128.9, 127.9, 125.5, 121.0, 124.0, 119.5, 110.2, 101.3, 101.1, 70.9, 70.2, 70.1, 70.1, 70.0, 69.5, 69.5, 59.0, 56.3, 56.1, 42.3, 37.4, 36.8, 35.6, 26.6, 25.8, 16.4, and two carbons not detected. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calculated for C<sub>52</sub>H<sub>59</sub>F<sub>3</sub>N<sub>8</sub>O<sub>11</sub>S, 1060.39761; found, 1060.39665. HPLC–MS <sup>t</sup>R = 1.71 min.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Protein Expression and Purification</h3><div class="NLM_p last">Human PTK2 (residues 411–689, UniProt accession number <a href="https://Q05397" class="ext-link">Q05397</a>) with an N-terminal TEV cleavage site was cloned into the pDEST20 vector (Invitrogen). Expression in this vector results in a fusion protein with a cleavable GST tag. Viral stocks were generated in the Bac-to-Bac system (Invitrogen) and used to infect Trichoplusia ni (Hi5) cells (Invitrogen). Recombinant protein was isolated from cell extracts by affinity chromatography over glutathione sepharose (GE Healthcare) in a batch mode. The GST-tag was removed by incubation with Ac-TEV protease (Thermo Fisher) on the resin, overnight at 4 °C. The cleaved protein was recovered, concentrated (Biomax-10, Merck), and finally purified by size exclusion chromatography on HiLoad Superdex S75 (GE Healthcare). The protein was concentrated (Biomax-10, Merck) to 5 mg/mL and stored in 20 mM HEPES, 250 mM NaCl, 1 mM DTT, 1 mM EDTA, pH 6.7 at −80 °C.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Crystallization of the PTK2 Kinasedomain BI 4464 Binary Complex</h3><div class="NLM_p last">Protein crystallization was done using the sitting drop method by incubating the protein with 1 mM BI 4464 (as 50 mM stock solution) and mixing 0.2 μL of PTK2 (4.5 mg/mL in 25 mM HEPES pH 6.7, 250 mM NaCl, 1 mM DTT, 1 mM EDTA) with 0.2 μL of reservoir solution (7% PEG 1500, 100 mM SPG buffer pH 6.0) at 4 °C. The crystals grew as thin plates within a few days to a final size of 150–200 μm. The crystals were transferred to a cryo buffer (reservoir solution with 15% ethylene glycol) and frozen in liquid nitrogen. Data were collected at the SLS beam line X06DA (Swiss Light Source, Paul Scherrer Institute) at a wavelength of 0.91 Å using the PILATUS 2M detector. The crystals belonged to space group <i>P</i>2<sub>1</sub>2<sub>1</sub>2 with 1 monomer per asymmetric unit. Images were processed with XDS.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The structures were solved by molecular replacement using the PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ETM">2ETM</a> as a search model. Subsequent model building and refinement was done using standard protocols using CCP4,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> COOT,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and autoBUSTER.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The unit cell parameters were <i>a</i> = 48.05 Å, <i>b</i> = 77.09 Å, <i>c</i> = 82.71 Å, and α, β, γ = 90° data, and the structure was refined to <i>R</i> and <i>R</i><sub>free</sub> values of 18.3 and 22.2%, respectively, with 98.1% of the residues in Ramachandran favored regions as validated with Molprobity.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Statistics for data collection and refinement can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Table S2</a>. The coordinates and structure factors of the structures have been deposited at the Protein Data Bank with the accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Z">6I8Z</a>. The authors will release the atomic coordinates and experimental data upon article publication.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Cell Culture</h3><div class="NLM_p last">The cell lines were obtained through ATCC or JCRB, verified for identity by satellite repeat analysis, tested for mycoplasm contamination at regular intervals, and cultured in the specified media in a humidified cell incubator at 37 °C and 5% CO<sub>2</sub>. Dulbecco’s modified Eagle medium (DMEM) was obtained from Lonza (product code BE12-604F), Eagle’s minimal essential medium (EMEM) from ATCC (product code 30–2003), Iscove’s modified Dulbecco’s medium (IMDM) from Thermo Fisher (product code 12440053), and RPMI-1640 from Thermo Fisher (product code A1049101). The following cell lines (product codes and culture media in parentheses) were used for the described experiments: SNU-387 (CRL-2237, RPMI-1640 plus 10% heat inactivated FCS), HUH-1 (JCRB0199, DMEM plus 10% FCS), Hep3B2.1-7 (HB-8064, EMEM plus 10% FCS), HepG2 (HB-8065, EMEM plus 10% FCS plus Glutamax), SK-Hep-1 HLF (JCRB0404 DMEM plus 10% FCS), SNU-398 (CRL-2233, RPMI1640 plus 10% FCS), HUCCT1 (JCRB0425, RPMI-1640 plus 10% FCS), HLE (JCRB0405, DMEM plus 10% FCS), HuH-7 (JCRB0403 DMEM plus 10% FCS), SNU-423 (CRL-2238, RPMI-1640 plus 10% FCS), A549 (CCL-185, F-12K plus 10% heat inactivated FCS), HSC-3 (JCRB0623, EMEM plus 10% FCS), C32 (CRL-1585, EMEM plus 10% FCS), and CFPAC-1 (CRL-1918, IMDM plus 10% FCS).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Protein Degradation Assays</h3><div class="NLM_p last">To quantify the effects of compounds on the PTK2 protein levels, cells were seeded at a density of 125 000 cells in 0.75 mL of culture medium in 24-well plates and allowed to attach for 6–8 h. Subsequently, compounds were added to the cells at logarithmic dose series using the HP Digital Dispenser D300 (Tecan), normalizing for the added DMSO. After the compound addition, cells were incubated for 18 h or the specified time intervals at 37 °C. Cells were washed with cold PBS and lysed by immediate freezing in 100 μL RIPA buffer (Sigma product code R0278) supplemented with 1:100 HALT phosphatase-protease inhibitors (Thermo product code 1861281) at −80 °C. After thawing, samples were transferred into V-bottom 96-well plates, and cellular debris was sedimented at 4000 rpm (Eppendorf centrifuge 5810R, rotor A-4-81). The supernatant (90 μL) was transferred into a new 96-well plate and analyzed using a Wes capillary electrophoresis instrument (Proteinsimple) using the PTK2 antibody (Cell Signaling product code 13009) at a dilution of 1:50 or the PDE6D antibody (Abcam product code ab5665) at a dilution of 1:25 and the GAPDH antibody (Abcam product code ab9485) at a dilution of 1:2000 for normalization.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Proliferation Assays</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Standard Proliferation Assay</h4><div class="NLM_p last">To test the effect of PTK2 PROTACs on proliferation, cells were seeded at 1000 cells per well (2500 cells per well for suspension cell lines) in 100 μL growth medium in a white bottom opaque 96-well plate and allowed to grow overnight. To obtain starting densities, a set of cells seeded in parallel were lysed and measured using 100 μL CellTiter-Glo luminescent cell viability reagent (Promega product code G7570) per well as per the manufacturer’s recommendation. The compounds were added to the cells at logarithmic dose series using the HP Digital Dispenser D300 (Tecan), normalizing for added DMSO. Doxorubicin (Sigma product code D1515) was used as a positive control. After compound addition, the cells were incubated for six days and the viability measured using the CellTiter-Glo reagent as described above. The results are stated as mean and standard deviation of triplicate experiments.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Anchorage-Independent Growth Assays</h4><div class="NLM_p last">Cells were seeded in 0.3% agarose (Gibco product code 18300-012) containing cultivation medium (60 μL) on top of a bottom layer composed of 1.2% agarose in cultivation medium (90 μL) in 96-well plates. Upon solidification of the cell layer, the culture was overlaid with 50 μL cultivation medium and the compounds were added as indicated above. The cultures were allowed to grow for 7–14 days, stained with alamarBlue reagent (Thermo Fisher product code DAL1025), and measured using a fluorescence plate reader (Wallac Victor 1420, 544 nm excitation, 590 nm emission, 0.2 s).</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Long-Term Proliferation Assay</h4><div class="NLM_p last">Cells were inoculated at a density of 250 000 cells in 1.5 mL culture medium in 6-well plates. The compounds or DMSO were added, and every 3–4 days, the cells were split into 250 000 cells. Upon splitting, fresh compound was added to keep the concentration constant. Split rates were recorded and multiplied to derive cumulative cell counts, which were converted into population doublings (<i>n</i>) using the formula <i>n</i> = (log(<i>N</i><sub><i>x</i></sub>) – log(<i>N</i><sub>0</sub>))/log<sub>2</sub> with <i>N</i><sub><i>x</i></sub> indicating the cumulative cell count at time-point <i>x</i> and <i>N</i><sub>0</sub> the initial seeding cell count.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> MS Proteomics</h3><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Sample Preparation</h4><div class="NLM_p last">A549 cells were seeded at 5 × 10<sup>6</sup> cells/mL in a 10 cm plate 18 h before treatment. The cells were treated with 0.1% DMSO as the vehicle control, 3 μM of either active PROTAC <b>6</b> or <b>8</b>, and 3 μM of <i>cis</i>VHL <b>11</b> as the negative control. The cells were incubated at 37 °C and 5% CO<sub>2</sub> for 18 h before harvesting. The cells were washed twice with DPBS (Gibco) and lysed with 0.5 mL of 100 mM Tris pH 8.0, 4% (w/v) SDS, supplemented with cOmplete Mini EDTA-free Protease Inhibitor Cocktail (Roche). The lysates were sonicated (2 × 10 s) and centrifuged at 14 000 rpm for 20 min at 4 °C. The supernatant fraction of the cell extract was collected, and the protein concentration was quantified by BCA assay (Thermo Fisher Scientific). Further sample processing, digestion, desalting, and TMT 10-plex isobaric labeling were performed as previously described.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> After labeling, the peptides from the nine samples were pooled together in equal proportion. The pooled sample was fractionated using high pH reverse-phase chromatography on an XBridge peptide BEH column (130 Å, 3.5 μm, 2.1 × 150 mm, Waters) on an Ultimate 3000 HPLC system (Thermo Scientific/Dionex). A mixture of buffer A (10 mM ammonium formate in water, pH 9) and B (10 mM ammonium formate in 90% CH<sub>3</sub>CN, pH 9) was used over a linear gradient of 5–60% buffer B over 60 min at a flow rate of 200 μL/min. The peptides eluted from the column were collected in 80 fractions before concatenation into 20 fractions based on the UV signal of each fraction. All fractions were dried in a Genevac EZ-2 concentrator and resuspended in 1% formic acid for MS analysis.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> nLC–MS/MS analysis</h4><div class="NLM_p last">The fractions were analyzed sequentially on a Q Exactive HF-X Hybrid Quadrupole-Orbitrap Mass Spectrometer (Thermo Scientific) coupled to an UltiMate 3000 RSLCnano UHPLC system (Thermo Scientific) and EasySpray column (75 μm × 50 cm, PepMap RSLC C18 column, 2 μm, 100 Å, Thermo Scientific). A mix of buffer A (0.1% formic acid in H<sub>2</sub>O) and B (0.08% formic acid in 80% CH<sub>3</sub>CN) was used over a linear gradient from 5 to 35% buffer B over 125 min using a flow rate of 300 nL/min. The column temperature was set at 50 °C. The mass spectrometer was operated in a data-dependent mode with a single MS survey scan from 335 to 1600 <i>m</i>/<i>z</i> followed by 15 sequential <i>m</i>/<i>z-</i>dependent MS2 scans. The 15 most intense precursor ions were sequentially fragmented by higher energy collision dissociation. The MS1 isolation window was set to 0.7 <i>m</i>/<i>z</i> and the resolution set at 120 000. The MS2 resolution was set at 45 000. The AGC targets for MS1 and MS2 were set at 3 × 10<sup>6</sup> ions and 1 × 10<sup>5</sup> ions, respectively. The normalized collision energy was set at 32%. The maximum ion injection times for MS1 and MS2 were set at 50 and 200 ms, respectively.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Peptide and Protein Identification and Quantification</h4><div class="NLM_p last">The raw MS data files for all 20 fractions were merged and searched against the Uniprot-sprot-Human-Canonical database by Maxquant software 1.6.0.16 for protein identification and TMT reporter ion quantitation. The identifications were based on the following criteria: enzyme used trypsin/P; maximum number of missed cleavages equal to two; precursor mass tolerance equal to 10 ppm; fragment mass tolerance equal to 20 ppm; variable modifications: oxidation (M), acetyl (N-term), deamidation (NQ), Gln → pyro-Glu (Q N-term); fixed modifications: carbamidomethyl (C). The data was filtered by applying a 1% false discovery rate followed by exclusion of proteins with less than two unique peptides. Quantified proteins were filtered if the absolute fold-change difference between the three DMSO replicates was ≥1.5.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01826" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35646" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35646" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01826" class="ext-link">10.1021/acs.jmedchem.8b01826</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional PTK2 degradation data for BI-0319 and BI-3663 in HCC cell lines (SNU387, HUH-1, Hep3B2.1-7, HepG2, SK-Hep1, HLF, SNU-398, HUCCT1, HLE, HuH7, and SNU-423); alamarBlue cell viability assays in above HCC cell lines with BI-3663, BI-0319, BI-4464, and doxorubicine (positive control); long-term proliferation assays in HSC-3, C32, CFPAC-1, and A549 with 3 μM of BI-3663 or BI-0319; CRISPR validation of PTK2 dependency, solubility, plasma protein binding, and Caco2 permeability assay data for PROTACs BI-0319 and BI-3663; X-ray data collection and refinement statistics; supplementary chemistry information: MS parameters, HPLC chromatograms and NMR spectrum of BI-3663; supplemental methods: solubility, PPB, CACO2 permeability, stability measurement, and CRISPR depletion of PTK2 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Proteomics raw data (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_003.xlsx" class="ext-link">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf">jm8b01826_si_001.pdf (1.04 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_002.csv">jm8b01826_si_002.csv (0.58 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_003.xlsx">jm8b01826_si_003.xlsx (13.22 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for PTK2: BI-4464 have been deposited to the Protein Data Bank (PDB) under accession number <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Z">6I8Z</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b01826" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37562" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37562" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Ettmayer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8422-2625" title="Orcid link">http://orcid.org/0000-0002-8422-2625</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3b4b5e4f5e49155e4f4f565a425e497b59545e534952555c5e491652555c5e57535e525615585456"><span class="__cf_email__" data-cfemail="671702130215490213130a061e0215270508020f150e090002154a0e0900020b0f020e0a4904080a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johannes Popow</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heribert Arnhof</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gerd Bader</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helmut Berger</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessio Ciulli</span> - <span class="hlFld-Affiliation affiliation">Division of Biological
Chemistry and Drug Discovery, School of Life Sciences, James Black
Centre, University of Dundee, Dow Street, DD1 5EH Dundee, Scotland, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8654-1670" title="Orcid link">http://orcid.org/0000-0002-8654-1670</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Covini</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Dank</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Teresa Gmaschitz</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Greb</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jale Karolyi-Özguer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manfred Koegl</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darryl B. McConnell</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark Pearson</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maria Rieger</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joerg Rinnenthal</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vanessa Roessler</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Schrenk</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Markus Spina</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steffen Steurer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicole Trainor</span> - <span class="hlFld-Affiliation affiliation">Division of Biological
Chemistry and Drug Discovery, School of Life Sciences, James Black
Centre, University of Dundee, Dow Street, DD1 5EH Dundee, Scotland, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisabeth Traxler</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Corinna Wieshofer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Zoephel</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co KG, 1221 Vienna, Austria</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The authors were full-time employees of Boehringer Ingelheim (1) or the University of Dundee (2) at the time when this study was performed.<br /></br><p class="inlineNote">While this manuscript was in advanced preparation, selective VHL-based PTK2 PROTACs were published by P. M. Cromm et al. J. Am. Chem. Soc., DOI: 10.1021/jacs.8b08008 as a manuscript just accepted, describing a VHL-based degrader (PROTAC-3) with chemical structure different from BI-0319.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31783" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31783" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We would like to thank Will Farnaby for coordination of the proteomics study, Moriz Mayer for coordinating the analytics, Gerlinde Flotzinger for PTK2 protein expression and purification, and Gabriele Glenndining and Susanne Mayer for their excellent logistics work for the collaboration between Boehringer Ingelheim and University of Dundee.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">NH<sub>4</sub>OAc</td><td class="NLM_def"><p class="first last">ammonium acetate</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal</p></td></tr><tr><td class="NLM_term">BRD2/3/4/7/9</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 2/3/4/7/9</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">DLBCL</td><td class="NLM_def"><p class="first last">diffuse large B cell lymphoma</p></td></tr><tr><td class="NLM_term">HCl</td><td class="NLM_def"><p class="first last">hydrochloric acid</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">H<sub>2</sub>O</td><td class="NLM_def"><p class="first last">water</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HOAt</td><td class="NLM_def"><p class="first last">1-hydroxy-7-azabenzotriazole solution</p></td></tr><tr><td class="NLM_term">MgSO<sub>4</sub></td><td class="NLM_def"><p class="first last">magnesium sulfate</p></td></tr><tr><td class="NLM_term">MsCl</td><td class="NLM_def"><p class="first last">methanesolfunyl chloride</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis-targeting chimera</p></td></tr><tr><td class="NLM_term">NaHCO<sub>3</sub></td><td class="NLM_def"><p class="first last">sodium bicarbonate</p></td></tr><tr><td class="NLM_term">NaHDMS</td><td class="NLM_def"><p class="first last">sodium bis(trimethylsilyl)amide</p></td></tr><tr><td class="NLM_term">NaH</td><td class="NLM_def"><p class="first last">sodium hydride</p></td></tr><tr><td class="NLM_term">NaOH</td><td class="NLM_def"><p class="first last">sodium hydroxide</p></td></tr><tr><td class="NLM_term">NaIO<sub>4</sub></td><td class="NLM_def"><p class="first last">sodium periodate</p></td></tr><tr><td class="NLM_term">NaBH(OAc)<sub>3</sub></td><td class="NLM_def"><p class="first last">sodium triacetoxyborohydride</p></td></tr><tr><td class="NLM_term">OsO<sub>4</sub></td><td class="NLM_def"><p class="first last">osmium tetroxide</p></td></tr><tr><td class="NLM_term">TEMPO</td><td class="NLM_def"><p class="first last">2,2,6,6-tetramethyl-1-piperidinyloxy</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72260" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72260" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 52 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Golubovskaya, V.</span></span> <span> </span><span class="NLM_article-title">Targeting FAK in human cancer: from finding to first clinical trials</span>. <i>Front. Biosci., Landmark Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">706</span>, <span class="refDoi"> DOI: 10.2741/4236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.2741%2F4236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=24389213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtl2ktb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=687-706&author=V.+Golubovskaya&title=Targeting+FAK+in+human+cancer%3A+from+finding+to+first+clinical+trials&doi=10.2741%2F4236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting FAK in human cancer: from finding to first clinical trials</span></div><div class="casAuthors">Golubovskaya, Vita M.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Landmark Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">687-706, 20</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  It is twenty years since Focal Adhesion Kinase (FAK) was found to be overexpressed in many types of human cancer.  FAK plays an important role in adhesion, spreading, motility, invasion, metastasis, survival, angiogenesis, and recently has been found to play an important role as well in epithelial to mesenchymal transition (EMT), cancer stem cells and tumor microenvironment.  FAK has kinase-dependent and kinase independent scaffolding, cytoplasmic and nuclear functions.  Several years ago FAK was proposed as a potential therapeutic target; the first clin. trials were just reported, and they supported further studies of FAK as a promising therapeutic target.  This review discusses the main functions of FAK in cancer, and specifically focuses on recent novel findings on the role of FAK in cancer stem cells, microenvironment, epithelial-to-mesenchymal transition, invasion, metastasis, and also highlight new approaches of targeting FAK and critically discuss challenges that lie ahead for its targeted therapeutics.  The review provides a summary of translational approaches of FAK-targeted and combination therapies and outline perspectives and future directions of FAK research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5EGzIEc6UMbVg90H21EOLACvtfcHk0lhBxjSO0g7yeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtl2ktb%252FI&md5=977f2530b2d4ed073e84ec64e498c3a3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2741%2F4236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F4236%26sid%3Dliteratum%253Aachs%26aulast%3DGolubovskaya%26aufirst%3DV.%26atitle%3DTargeting%2520FAK%2520in%2520human%2520cancer%253A%2520from%2520finding%2520to%2520first%2520clinical%2520trials%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2014%26volume%3D19%26spage%3D687%26epage%3D706%26doi%3D10.2741%2F4236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sulzmaier, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlaepfer, D. D.</span></span> <span> </span><span class="NLM_article-title">FAK in cancer: mechanistic findings and clinical applications</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1038/nrc3792</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnrc3792" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=25098269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12nu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=598-610&author=F.+J.+Sulzmaierauthor=C.+Jeanauthor=D.+D.+Schlaepfer&title=FAK+in+cancer%3A+mechanistic+findings+and+clinical+applications&doi=10.1038%2Fnrc3792"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">FAK in cancer: mechanistic findings and clinical applications</span></div><div class="casAuthors">Sulzmaier, Florian J.; Jean, Christine; Schlaepfer, David D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">598-610</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Focal adhesion kinase (FAK) is a cytoplasmic protein tyrosine kinase that is overexpressed and activated in several advanced-stage solid cancers.  FAK promotes tumor progression and metastasis through effects on cancer cells, as well as stromal cells of the tumor microenvironment.  The kinase-dependent and kinase-independent functions of FAK control cell movement, invasion, survival, gene expression and cancer stem cell self-renewal.  Small mol. FAK inhibitors decrease tumor growth and metastasis in several preclin. models and have initial clin. activity in patients with limited adverse events.  In this Review, we discuss FAK signalling effects on both tumor and stromal cell biol. that provide rationale and support for future therapeutic opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmKc7m7dsY67Vg90H21EOLACvtfcHk0lhBxjSO0g7yeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12nu7vJ&md5=f498bcf7af890d2f92eda68e42f979d1</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc3792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3792%26sid%3Dliteratum%253Aachs%26aulast%3DSulzmaier%26aufirst%3DF.%2BJ.%26aulast%3DJean%26aufirst%3DC.%26aulast%3DSchlaepfer%26aufirst%3DD.%2BD.%26atitle%3DFAK%2520in%2520cancer%253A%2520mechanistic%2520findings%2520and%2520clinical%2520applications%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D598%26epage%3D610%26doi%3D10.1038%2Fnrc3792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lark, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livasy, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caskey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cance, W. G.</span></span> <span> </span><span class="NLM_article-title">Overexpression of focal adhesion kinase in primary colorectal carcinomas and colorectal liver metastases: immunohistochemistry and real-time PCR analyses</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">222</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=12538472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1OksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2003&pages=215-222&author=A.+L.+Larkauthor=C.+A.+Livasyauthor=B.+Calvoauthor=L.+Caskeyauthor=D.+T.+Mooreauthor=X.+Yangauthor=W.+G.+Cance&title=Overexpression+of+focal+adhesion+kinase+in+primary+colorectal+carcinomas+and+colorectal+liver+metastases%3A+immunohistochemistry+and+real-time+PCR+analyses"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of Focal Adhesion Kinase in Primary Colorectal Carcinomas and Colorectal Liver Metastases: Immunohistochemistry and Real-Time PCR Analyses</span></div><div class="casAuthors">Lark, Amy L.; Livasy, Chad A.; Calvo, Benjamin; Caskey, Laura; Moore, Dominic T.; Yang, XiHui; Cance, William G.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">215-222</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Focal adhesion kinase (FAK), a protein tyrosine kinase that functions in signaling events between cells and their extracellular matrix, is overexpressed in a variety of human solid tumors.  To det. whether FAK expression is up-regulated in colorectal cancer, we analyzed FAK mRNA and protein levels in primary colorectal tumors and colorectal liver metastases.  Exptl. Design: p125FAK expression in formalin-fixed paraffin-embedded (FFPE) tissue was studied using immunohistochem. assays on 24 matched primary colorectal carcinomas and colorectal liver metastases as well as 18 different colorectal liver metastases using monoclonal anti-FAK 4.47.  FAK mRNA expression was quantitated by real-time PCR on 39 matched normal colorectal mucosa and primary colorectal carcinomas as well as on 17 sep. liver metastases.  RESULTS: Elevated levels of p125FAK expression were demonstrated in both primary colorectal tumors and colorectal liver metastases compared with normal colorectal mucosa.  Immunohistochem. expts. demonstrated equiv. FAK expression in matched samples of colorectal primary tumors and liver metastases.  Using real-time PCR in 39 matched samples, FAK mRNA copy no. was significantly higher in primary colorectal tumors compared with normal colorectal mucosa.  FAK expression was analyzed by both real-time PCR and immunohistochem. in a sep. set of colorectal liver metastases.  Immunohistochem. demonstrated high levels of FAK expression in 89% of samples.  Furthermore, FAK mRNA copies in these unmatched liver metastases were significantly higher than the primary tumor FAK mRNA copies.  CONCLUSION: These expts. have shown that both primary colorectal cancers and colorectal liver metastases express high levels of FAK mRNA and p125FAK protein.  Furthermore, the majority of colorectal liver metastases demonstrated robust FAK expression equiv. to or greater than that in the primary colorectal tumor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1OpdCDZYyq7Vg90H21EOLACvtfcHk0lhc758WrRcFUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt1OksA%253D%253D&md5=39a8af2050e11b9e73959f9700daa3fc</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLark%26aufirst%3DA.%2BL.%26aulast%3DLivasy%26aufirst%3DC.%2BA.%26aulast%3DCalvo%26aufirst%3DB.%26aulast%3DCaskey%26aufirst%3DL.%26aulast%3DMoore%26aufirst%3DD.%2BT.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DCance%26aufirst%3DW.%2BG.%26atitle%3DOverexpression%2520of%2520focal%2520adhesion%2520kinase%2520in%2520primary%2520colorectal%2520carcinomas%2520and%2520colorectal%2520liver%2520metastases%253A%2520immunohistochemistry%2520and%2520real-time%2520PCR%2520analyses%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2003%26volume%3D9%26spage%3D215%26epage%3D222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Judson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cance, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Le, L.</span></span> <span> </span><span class="NLM_article-title">Overexpression of focal adhesion kinase, a protein tyrosine kinase, in ovarian carcinoma</span>. <i>Cancer</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">1551</span>– <span class="NLM_lpage">1556</span>, <span class="refDoi"> DOI: 10.1002/(sici)1097-0142(19991015)86:6<1551::aid-cncr23>3.0.co;2-p</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1002%2F%28sici%291097-0142%2819991015%2986%3A6%3C1551%3A%3Aaid-cncr23%3E3.0.co%3B2-p" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10526262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADyaK1MXntlSkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1999&pages=1551-1556&author=P.+L.+Judsonauthor=X.+Heauthor=W.+G.+Canceauthor=L.+Van+Le&title=Overexpression+of+focal+adhesion+kinase%2C+a+protein+tyrosine+kinase%2C+in+ovarian+carcinoma&doi=10.1002%2F%28sici%291097-0142%2819991015%2986%3A6%3C1551%3A%3Aaid-cncr23%3E3.0.co%3B2-p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of focal adhesion kinase, a protein tyrosine kinase, in ovarian carcinoma</span></div><div class="casAuthors">Judson, Patricia L.; He, Xiaping; Cance, William G.; Van Le, Linda</div><div class="citationInfo"><span class="NLM_cas:title">Cancer (New York)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1551-1556</span>CODEN:
                <span class="NLM_cas:coden">CANCAR</span>;
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Focal adhesion kinase (FAK) is a tyrosine kinase that is important to such key functions such as cell adhesion, motility, and invasion.  A MEDLINE search of the years 1980-1998 found no previous reports of FAK expression in human ovarian carcinoma.  The authors performed expts. to det. whether FAK expression is elevated in this disease.  Ten normal human ovarian tissue samples and 26 cancer samples from patients with Stage I-IV ovarian carcinoma were obtained.  Two ovarian carcinoma cell lines were also analyzed.  FAK expression was detd. by Western blot anal. with the V39 anti-human FAK polyclonal antibody.  The level of FAK protein expression was detd. using densitometric scanning of the 125-kDa band on autoradiographs of Western immunoblots.  Serous cancers expressed 4-fold-increased values of FAK relative to normal ovarian tissue, and nonserous adenocarcinomas expressed 3-4-fold-increased values of FAK.  Ovarian carcinoma cell lines also expressed increased values of FAK.  With a cutoff of 40, an elevated FAK level was assocd. with a sensitivity of 93% and specificity of 100%.  There was no significant difference in FAK expression with regard to grade or stage of tumor.  Thus, FAK is overexpressed in ovarian carcinoma, implying that FAK may play an important role in ovarian carcinogenesis.  FAK expression may be useful as a screening tool to identify newly developed disease or as a tumor marker in confirmed cases of epithelial ovarian carcinoma.  FAK may also serve as a potential target for therapeutic disruption of ovarian carcinoma progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk4vVqxQX-V7Vg90H21EOLACvtfcHk0lhc758WrRcFUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntlSkurg%253D&md5=b68496778ced297f8b4d6e80fa8d0629</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2F%28sici%291097-0142%2819991015%2986%3A6%3C1551%3A%3Aaid-cncr23%3E3.0.co%3B2-p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528sici%25291097-0142%252819991015%252986%253A6%253C1551%253A%253Aaid-cncr23%253E3.0.co%253B2-p%26sid%3Dliteratum%253Aachs%26aulast%3DJudson%26aufirst%3DP.%2BL.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DCance%26aufirst%3DW.%2BG.%26aulast%3DVan%2BLe%26aufirst%3DL.%26atitle%3DOverexpression%2520of%2520focal%2520adhesion%2520kinase%252C%2520a%2520protein%2520tyrosine%2520kinase%252C%2520in%2520ovarian%2520carcinoma%26jtitle%3DCancer%26date%3D1999%26volume%3D86%26spage%3D1551%26epage%3D1556%26doi%3D10.1002%2F%28sici%291097-0142%2819991015%2986%3A6%3C1551%3A%3Aaid-cncr23%3E3.0.co%3B2-p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyazaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwano, H.</span></span> <span> </span><span class="NLM_article-title">FAK overexpression is correlated with tumour invasiveness and lymph node metastasis in oesophageal squamous cell carcinoma</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6601050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fsj.bjc.6601050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=12838315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVWmsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2003&pages=140-145&author=T.+Miyazakiauthor=H.+Katoauthor=M.+Nakajimaauthor=M.+Sohdaauthor=Y.+Fukaiauthor=N.+Masudaauthor=R.+Mandaauthor=M.+Fukuchiauthor=K.+Tsukadaauthor=H.+Kuwano&title=FAK+overexpression+is+correlated+with+tumour+invasiveness+and+lymph+node+metastasis+in+oesophageal+squamous+cell+carcinoma&doi=10.1038%2Fsj.bjc.6601050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">FAK overexpression is correlated with tumor invasiveness and lymph node metastasis in esophageal squamous cell carcinoma</span></div><div class="casAuthors">Miyazaki, T.; Kato, H.; Nakajima, M.; Sohda, M.; Fukai, Y.; Masuda, N.; Manda, R.; Fukuchi, M.; Tsukada, K.; Kuwano, H.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">140-145</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Focal adhesion kinase (p125FAK; FAK') is a tyrosine kinase that is localized to cellular focal adhesions and is assocd. with a no. of other proteins, such as integrin adhesion receptors.  We performed an immunohistochem. anal. of FAK protein expression to det. the relationship between FAK overexpression and clinicopathol. factors in esophageal squamous cell carcinoma (ESCC).  We examd. tissue specimens that had been removed from 91 patients with thoracic esophageal cancer who had undergone surgery between 1983 and 2001.  Immunohistochem. staining was performed by the std. streptavidin-biotin method.  Seven human ESCC cell lines-TE-1, TE-2, TE-8, TE-13, TE-15, TT, and TTn-and one immortalized human keratinocyte cell line-HaCaT-were used in Western blot anal.  Immunostaining of FAK was seen in the cytoplasm of cancer cells, particularly in cells located in the invasive fronts of cancer nests.  FAK overexpression was detected in 54 of the 91 patients (59.3%).  Significant correlations were obsd. between FAK overexpression and cell differentiation (P=0.0057), depth of tumor invasion (P=0.0023), presence of regional lymph node metastasis (P=0.0097), no. of lymph node metastases (P=0.0026), and disease stage (P=0.012).  The survival rates of patients with FAK-overexpressing cancer were significantly lower than those of patients without FAK-overexpression cancer (P=0.006).  The 5-yr survival rate of patients without FAK overexpression was 69%, whereas that of patients with FAK overexpression was 38%.  On Western blot anal., FAK was expressed at a high level in TE-1, TE-8, TE-15, and TT cells, at a moderate level in TE-2 and TTn cells, and at a low level in TE13 and HaCaT cells.  FAK phosphorylation at tyrosine 397 was demonstrated in proportion to the intensity of FAK in all cell lines except TE15 and HaCaT.  In conclusion, FAK overexpression of ESCC was related to cell differentiation, tumor invasiveness, and lymph node metastasis.  Consequently, patients with ESCC who had FAK overexpression had a poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJyEVSutRHkrVg90H21EOLACvtfcHk0lhc758WrRcFUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVWmsbw%253D&md5=e0c5df0c3db976836f2d51b35baec1aa</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6601050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6601050%26sid%3Dliteratum%253Aachs%26aulast%3DMiyazaki%26aufirst%3DT.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DNakajima%26aufirst%3DM.%26aulast%3DSohda%26aufirst%3DM.%26aulast%3DFukai%26aufirst%3DY.%26aulast%3DMasuda%26aufirst%3DN.%26aulast%3DManda%26aufirst%3DR.%26aulast%3DFukuchi%26aufirst%3DM.%26aulast%3DTsukada%26aufirst%3DK.%26aulast%3DKuwano%26aufirst%3DH.%26atitle%3DFAK%2520overexpression%2520is%2520correlated%2520with%2520tumour%2520invasiveness%2520and%2520lymph%2520node%2520metastasis%2520in%2520oesophageal%2520squamous%2520cell%2520carcinoma%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2003%26volume%3D89%26spage%3D140%26epage%3D145%26doi%3D10.1038%2Fsj.bjc.6601050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aishima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirabe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maehara, Y.</span></span> <span> </span><span class="NLM_article-title">Role of expression of focal adhesion kinase in progression of hepatocellular carcinoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2812</span>– <span class="NLM_lpage">2817</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-1046-03</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1158%2F1078-0432.ccr-1046-03" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=15102689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1CltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=2812-2817&author=S.+Itohauthor=T.+Maedaauthor=M.+Shimadaauthor=S.+Aishimaauthor=K.+Shirabeauthor=S.+Tanakaauthor=Y.+Maehara&title=Role+of+expression+of+focal+adhesion+kinase+in+progression+of+hepatocellular+carcinoma&doi=10.1158%2F1078-0432.ccr-1046-03"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Role of expression of focal adhesion kinase in progression of hepatocellular carcinoma</span></div><div class="casAuthors">Itoh, Shinji; Maeda, Takashi; Shimada, Mitsuo; Aishima, Shin-ichi; Shirabe, Ken; Tanaka, Shinji; Maehara, Yoshihiko</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2812-2817</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Although hepatocellular carcinoma (HCC) is the most common cancer of the human liver, the mechanisms that regulate HCC development and progression remain unclear.  The aim of this study was to investigate whether focal adhesion kinase (FAK) is involved in the progression of human HCC.  Western blot anal. for FAK was performed on 3 HCC cell lines.  The authors reviewed 64 consecutive patients who had undergone initial liver resection for HCC without preoperative treatment.  Immunohistochem. anal. for FAK was performed on paraffin-embedded tissues.  FAK expression was confirmed by Western blot anal. in several clin. samples.  The authors investigated the correlation between FAK expression and clin. outcome.  FAK proteins were detected in all HCC cell lines.  Hepatocytes in the normal liver and chronic hepatitis with or without cirrhosis were neg. for immunohistochem. staining for FAK expression.  Cytoplasmic FAK expression was obsd. in 18 of 64 patients (28.1%), and this pos. staining was correlated with gender (P < 0.05), a lower level of serum albumin (P < 0.05), and portal venous invasion (P < 0.01).  Pos. staining for FAK was assocd. with significantly poorer survival (P < 0.05).  In multivariate anal., FAK overexpression was an independent factor in detg. the prognosis of patients.  These data suggest that FAK plays an important role in promoting tumor progression, esp. vascular invasion, in HCC.  FAK could play an important role in HCC progression and would be a novel target for HCC therapeutics as well as a prognostic marker.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUUZBKRASEuLVg90H21EOLACvtfcHk0lh-wgYOW0eiVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjt1CltbY%253D&md5=d16eb2f787952aa3a47b15a0730fc6b1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-1046-03&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-1046-03%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DS.%26aulast%3DMaeda%26aufirst%3DT.%26aulast%3DShimada%26aufirst%3DM.%26aulast%3DAishima%26aufirst%3DS.%26aulast%3DShirabe%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DMaehara%26aufirst%3DY.%26atitle%3DRole%2520of%2520expression%2520of%2520focal%2520adhesion%2520kinase%2520in%2520progression%2520of%2520hepatocellular%2520carcinoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2004%26volume%3D10%26spage%3D2812%26epage%3D2817%26doi%3D10.1158%2F1078-0432.ccr-1046-03" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armaiz-Pena, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nick, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spannuth, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deavers, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahzad, M. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Montero, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGeest, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivas-Mejia, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Berestein, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lutgendorf, S. K.</span></span> <span> </span><span class="NLM_article-title">Adrenergic modulation of focal adhesion kinase protects human ovarian cancer cells from anoikis</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">1515</span>– <span class="NLM_lpage">1523</span>, <span class="refDoi"> DOI: 10.1172/jci40802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1172%2Fjci40802" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2010&pages=1515-1523&author=A.+K.+Soodauthor=G.+N.+Armaiz-Penaauthor=J.+Halderauthor=A.+M.+Nickauthor=R.+L.+Stoneauthor=W.+Huauthor=A.+R.+Carrollauthor=W.+A.+Spannuthauthor=M.+T.+Deaversauthor=J.+K.+Allenauthor=L.+Y.+Hanauthor=A.+A.+Kamatauthor=M.+M.+K.+Shahzadauthor=B.+W.+McIntyreauthor=C.+M.+Diaz-Monteroauthor=N.+B.+Jenningsauthor=Y.+G.+Linauthor=W.+M.+Merrittauthor=K.+DeGeestauthor=P.+E.+Vivas-Mejiaauthor=G.+Lopez-Beresteinauthor=M.+D.+Schallerauthor=S.+W.+Coleauthor=S.+K.+Lutgendorf&title=Adrenergic+modulation+of+focal+adhesion+kinase+protects+human+ovarian+cancer+cells+from+anoikis&doi=10.1172%2Fjci40802"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1172%2Fjci40802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci40802%26sid%3Dliteratum%253Aachs%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DArmaiz-Pena%26aufirst%3DG.%2BN.%26aulast%3DHalder%26aufirst%3DJ.%26aulast%3DNick%26aufirst%3DA.%2BM.%26aulast%3DStone%26aufirst%3DR.%2BL.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DCarroll%26aufirst%3DA.%2BR.%26aulast%3DSpannuth%26aufirst%3DW.%2BA.%26aulast%3DDeavers%26aufirst%3DM.%2BT.%26aulast%3DAllen%26aufirst%3DJ.%2BK.%26aulast%3DHan%26aufirst%3DL.%2BY.%26aulast%3DKamat%26aufirst%3DA.%2BA.%26aulast%3DShahzad%26aufirst%3DM.%2BM.%2BK.%26aulast%3DMcIntyre%26aufirst%3DB.%2BW.%26aulast%3DDiaz-Montero%26aufirst%3DC.%2BM.%26aulast%3DJennings%26aufirst%3DN.%2BB.%26aulast%3DLin%26aufirst%3DY.%2BG.%26aulast%3DMerritt%26aufirst%3DW.%2BM.%26aulast%3DDeGeest%26aufirst%3DK.%26aulast%3DVivas-Mejia%26aufirst%3DP.%2BE.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26aulast%3DSchaller%26aufirst%3DM.%2BD.%26aulast%3DCole%26aufirst%3DS.%2BW.%26aulast%3DLutgendorf%26aufirst%3DS.%2BK.%26atitle%3DAdrenergic%2520modulation%2520of%2520focal%2520adhesion%2520kinase%2520protects%2520human%2520ovarian%2520cancer%2520cells%2520from%2520anoikis%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2010%26volume%3D120%26spage%3D1515%26epage%3D1523%26doi%3D10.1172%2Fjci40802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tancioni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N. L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uryu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupack, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plaxe, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlaepfer, D. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of focal adhesion kinase (FAK) activity prevents anchorage-independent ovarian carcinoma cell growth and tumor progression</span>. <i>Clin. Exp. Metastasis</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">579</span>– <span class="NLM_lpage">594</span>, <span class="refDoi"> DOI: 10.1007/s10585-012-9562-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1007%2Fs10585-012-9562-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=23275034" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=579-594&author=K.+K.+Wardauthor=I.+Tancioniauthor=C.+Lawsonauthor=N.+L.+G.+Millerauthor=C.+Jeanauthor=X.+L.+Chenauthor=S.+Uryuauthor=J.+Kimauthor=D.+Tarinauthor=D.+G.+Stupackauthor=S.+C.+Plaxeauthor=D.+D.+Schlaepfer&title=Inhibition+of+focal+adhesion+kinase+%28FAK%29+activity+prevents+anchorage-independent+ovarian+carcinoma+cell+growth+and+tumor+progression&doi=10.1007%2Fs10585-012-9562-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs10585-012-9562-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10585-012-9562-5%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DK.%2BK.%26aulast%3DTancioni%26aufirst%3DI.%26aulast%3DLawson%26aufirst%3DC.%26aulast%3DMiller%26aufirst%3DN.%2BL.%2BG.%26aulast%3DJean%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DX.%2BL.%26aulast%3DUryu%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DTarin%26aufirst%3DD.%26aulast%3DStupack%26aufirst%3DD.%2BG.%26aulast%3DPlaxe%26aufirst%3DS.%2BC.%26aulast%3DSchlaepfer%26aufirst%3DD.%2BD.%26atitle%3DInhibition%2520of%2520focal%2520adhesion%2520kinase%2520%2528FAK%2529%2520activity%2520prevents%2520anchorage-independent%2520ovarian%2520carcinoma%2520cell%2520growth%2520and%2520tumor%2520progression%26jtitle%3DClin.%2520Exp.%2520Metastasis%26date%3D2012%26volume%3D30%26spage%3D579%26epage%3D594%26doi%3D10.1007%2Fs10585-012-9562-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balogh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Victor, D.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asham, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burroughs, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boktour, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saharia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monsour, H.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Hepatocellular carcinoma: a review</span>. <i>J. Hepatocell. Carcinoma</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.2147/jhc.s61146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.2147%2Fjhc.s61146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=27785449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A280%3ADC%252BC2sngtV2qtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2016&pages=41-53&author=J.+Baloghauthor=D.+Victorauthor=E.+H.+Ashamauthor=S.+G.+Burroughsauthor=M.+Boktourauthor=A.+Sahariaauthor=X.+Liauthor=M.+Ghobrialauthor=H.+Monsour&title=Hepatocellular+carcinoma%3A+a+review&doi=10.2147%2Fjhc.s61146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatocellular carcinoma: a review</span></div><div class="casAuthors">Balogh Julius; Asham Emad H; Burroughs Sherilyn Gordon; Boktour Maha; Saharia Ashish; Li Xian; Ghobrial R Mark; Victor David 3rd; Monsour Howard P Jr</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hepatocellular carcinoma</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-53</span>
        ISSN:<span class="NLM_cas:issn">2253-5969</span>.
    </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is the most common primary liver malignancy and is a leading cause of cancer-related death worldwide.  In the United States, HCC is the ninth leading cause of cancer deaths.  Despite advances in prevention techniques, screening, and new technologies in both diagnosis and treatment, incidence and mortality continue to rise.  Cirrhosis remains the most important risk factor for the development of HCC regardless of etiology.  Hepatitis B and C are independent risk factors for the development of cirrhosis.  Alcohol consumption remains an important additional risk factor in the United States as alcohol abuse is five times higher than hepatitis C.  Diagnosis is confirmed without pathologic confirmation.  Screening includes both radiologic tests, such as ultrasound, computerized tomography, and magnetic resonance imaging, and serological markers such as α-fetoprotein at 6-month intervals.  Multiple treatment modalities exist; however, only orthotopic liver transplantation (OLT) or surgical resection is curative.  OLT is available for patients who meet or are downstaged into the Milan or University of San Francisco criteria.  Additional treatment modalities include transarterial chemoembolization, radiofrequency ablation, microwave ablation, percutaneous ethanol injection, cryoablation, radiation therapy, systemic chemotherapy, and molecularly targeted therapies.  Selection of a treatment modality is based on tumor size, location, extrahepatic spread, and underlying liver function.  HCC is an aggressive cancer that occurs in the setting of cirrhosis and commonly presents in advanced stages.  HCC can be prevented if there are appropriate measures taken, including hepatitis B virus vaccination, universal screening of blood products, use of safe injection practices, treatment and education of alcoholics and intravenous drug users, and initiation of antiviral therapy.  Continued improvement in both surgical and nonsurgical approaches has demonstrated significant benefits in overall survival.  While OLT remains the only curative surgical procedure, the shortage of available organs precludes this therapy for many patients with HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTUVKKRQ-ThgJPq5LRczm8ffW6udTcc2ea48AaH5KnuALntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sngtV2qtw%253D%253D&md5=f746bbeacacdd9a8fa881afa307f8f7f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2147%2Fjhc.s61146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252Fjhc.s61146%26sid%3Dliteratum%253Aachs%26aulast%3DBalogh%26aufirst%3DJ.%26aulast%3DVictor%26aufirst%3DD.%26aulast%3DAsham%26aufirst%3DE.%2BH.%26aulast%3DBurroughs%26aufirst%3DS.%2BG.%26aulast%3DBoktour%26aufirst%3DM.%26aulast%3DSaharia%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DGhobrial%26aufirst%3DM.%26aulast%3DMonsour%26aufirst%3DH.%26atitle%3DHepatocellular%2520carcinoma%253A%2520a%2520review%26jtitle%3DJ.%2520Hepatocell.%2520Carcinoma%26date%3D2016%26volume%3D3%26spage%3D41%26epage%3D53%26doi%3D10.2147%2Fjhc.s61146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">El-Serag, H. B.</span></span> <span> </span><span class="NLM_article-title">Hepatocellular carcinoma</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>365</i></span>,  <span class="NLM_fpage">1118</span>– <span class="NLM_lpage">1127</span>, <span class="refDoi"> DOI: 10.1056/nejmra1001683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1056%2Fnejmra1001683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=21992124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1WqtrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2011&pages=1118-1127&author=H.+B.+El-Serag&title=Hepatocellular+carcinoma&doi=10.1056%2Fnejmra1001683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatocellular carcinoma</span></div><div class="casAuthors">El-Serag, Hashem B.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">365</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1118-1127</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ9eGOw4kyT7Vg90H21EOLACvtfcHk0ljsDe3d8wWTvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1WqtrnL&md5=3fb4250042385c5e29a189df86aea635</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1056%2Fnejmra1001683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252Fnejmra1001683%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Serag%26aufirst%3DH.%2BB.%26atitle%3DHepatocellular%2520carcinoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2011%26volume%3D365%26spage%3D1118%26epage%3D1127%26doi%3D10.1056%2Fnejmra1001683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Llovet, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzaferro, V.</span></span> <span> </span><span class="NLM_article-title">Resection and liver transplantation for hepatocellular carcinoma</span>. <i>Semin. Liver Dis.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1055/s-2005-871198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1055%2Fs-2005-871198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=15918147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A280%3ADC%252BD2M3nsFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=181-200&author=J.+M.+Llovetauthor=M.+Schwartzauthor=V.+Mazzaferro&title=Resection+and+liver+transplantation+for+hepatocellular+carcinoma&doi=10.1055%2Fs-2005-871198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Resection and liver transplantation for hepatocellular carcinoma</span></div><div class="casAuthors">Llovet Josep M; Schwartz Myron; Mazzaferro Vincenzo</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in liver disease</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-200</span>
        ISSN:<span class="NLM_cas:issn">0272-8087</span>.
    </div><div class="casAbstract">Surveillance programs in cirrhotic patients enable the detection of hepatocellular carcinoma (HCC) at early stages, when the tumor is amenable to curative treatments (60% of cases in Japan; 25 to 40% in Europe and the United States).  Resection is the mainstay of treatment in noncirrhotic patients and in cirrhotics with well-preserved liver function.  In modern series, a perioperative mortality < or = 3% and 5-year survival rates above 50% are expected.  Tumor recurrence complicates half of the cases at 3 years, but there is no unquestionable preventive treatment.  Liver transplantation provides excellent outcomes applying the Milan criteria (single nodule < or = 5 cm or two or three nodules < or = 3 cm), with 5-year survival rates of 70% and low recurrence rates.  Although expansion of selection criteria is appealing, it should be assessed in the setting of prospective well-designed studies.  Intention-to-treat analysis has shown that wide extended indications lead to 25% 5-year survival rates.  Living donor liver transplantation is having a minor impact in HCC management.  Molecular markers are needed to better select the candidates for surgery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQxJWeDn2fsrwHqQV89nF-2fW6udTcc2eZYabUhdNO0jbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M3nsFWmsA%253D%253D&md5=19be350d7cc28a7d1ac66ae6d07ae4d2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1055%2Fs-2005-871198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-2005-871198%26sid%3Dliteratum%253Aachs%26aulast%3DLlovet%26aufirst%3DJ.%2BM.%26aulast%3DSchwartz%26aufirst%3DM.%26aulast%3DMazzaferro%26aufirst%3DV.%26atitle%3DResection%2520and%2520liver%2520transplantation%2520for%2520hepatocellular%2520carcinoma%26jtitle%3DSemin.%2520Liver%2520Dis.%26date%3D2005%26volume%3D25%26spage%3D181%26epage%3D200%26doi%3D10.1055%2Fs-2005-871198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span> <span> </span><span class="NLM_article-title">Comparison of hepatic resection and radiofrequency ablation for small hepatocellular carcinoma: a meta-analysis of 16,103 patients</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">7252</span>, <span class="refDoi"> DOI: 10.1038/srep07252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fsrep07252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=25429732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A280%3ADC%252BC2Mzhtlyiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=7252&author=Q.+Xuauthor=S.+Kobayashiauthor=X.+Yeauthor=X.+Meng&title=Comparison+of+hepatic+resection+and+radiofrequency+ablation+for+small+hepatocellular+carcinoma%3A+a+meta-analysis+of+16%2C103+patients&doi=10.1038%2Fsrep07252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of hepatic resection and radiofrequency ablation for small hepatocellular carcinoma: a meta-analysis of 16,103 patients</span></div><div class="casAuthors">Xu Qinghua; Ye Xun; Meng Xia; Kobayashi Seijin</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7252</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We performed a meta-analysis to evaluate the therapeutic effects of radiofrequency ablation (RFA) and surgical hepatic resection (HR) in the treatment of small hepatocellular carcinoma (HCC).  Thirty-one studies were included in the analysis.  A total of 16,103 patients were involved: 8,252 treated with RFA and 7,851 with HR.  Compared to the RFA group, the 3, 5-year overall and disease-free survival rates in the HR group were significantly higher.  On the other hand, complications were significantly fewer and hospital-stay was significantly shorter in the RFA group than in the HR group.  In subgroup analyses, the overall and disease-free survival in the HR group were also significantly higher than those in the RFA group for HCCs ≤ 3 cm, whereas there were no significant differences between the two groups for HCCs ≤ 2 cm.  Our analysis showed that although HR was associated with higher complication rate and longer hospital-stay, HR is proposed as the first-line treatment rather than RFA for patients with HCCs larger than 2 cm.  For patients with HCCs of 2 cm or less, RFA may be an alternative to HR because of their comparable long-term efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTduceYDnsZa_6AExegM5LLfW6udTcc2eZYabUhdNO0jbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mzhtlyiuw%253D%253D&md5=43905d4f504f17316dd594e4a0dd1d0c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsrep07252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep07252%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DKobayashi%26aufirst%3DS.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DMeng%26aufirst%3DX.%26atitle%3DComparison%2520of%2520hepatic%2520resection%2520and%2520radiofrequency%2520ablation%2520for%2520small%2520hepatocellular%2520carcinoma%253A%2520a%2520meta-analysis%2520of%252016%252C103%2520patients%26jtitle%3DSci.%2520Rep.%26date%3D2014%26volume%3D4%26spage%3D7252%26doi%3D10.1038%2Fsrep07252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span> <span> </span><span class="NLM_article-title">3’3-Diindolylmethane inhibits migration, invasion and metastasis of hepatocellular carcinoma by suppressing FAK signaling</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">23776</span>– <span class="NLM_lpage">23792</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.4196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.18632%2Foncotarget.4196" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=23776-23792&author=W.-X.+Liauthor=L.-P.+Chenauthor=M.-Y.+Sunauthor=J.-T.+Liauthor=H.-Z.+Liuauthor=W.+Zhu&title=3%E2%80%993-Diindolylmethane+inhibits+migration%2C+invasion+and+metastasis+of+hepatocellular+carcinoma+by+suppressing+FAK+signaling&doi=10.18632%2Foncotarget.4196"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4196%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.-X.%26aulast%3DChen%26aufirst%3DL.-P.%26aulast%3DSun%26aufirst%3DM.-Y.%26aulast%3DLi%26aufirst%3DJ.-T.%26aulast%3DLiu%26aufirst%3DH.-Z.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3D3%25E2%2580%25993-Diindolylmethane%2520inhibits%2520migration%252C%2520invasion%2520and%2520metastasis%2520of%2520hepatocellular%2520carcinoma%2520by%2520suppressing%2520FAK%2520signaling%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D23776%26epage%3D23792%26doi%3D10.18632%2Foncotarget.4196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gnani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romito, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artuso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chierici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Stefanis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panera, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crudele, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccarelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carcarino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Oria, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porru, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giorda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrari, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miele, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furlanello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locatelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nobili, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rota, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonetti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alisi, A.</span></span> <span> </span><span class="NLM_article-title">Focal adhesion kinase depletion reduces human hepatocellular carcinoma growth by repressing enhancer of zeste homolog 2</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">902</span>, <span class="refDoi"> DOI: 10.1038/cdd.2017.34</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fcdd.2017.34" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=28338656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFGqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=889-902&author=D.+Gnaniauthor=I.+Romitoauthor=S.+Artusoauthor=M.+Chiericiauthor=C.+De+Stefanisauthor=N.+Paneraauthor=A.+Crudeleauthor=S.+Ceccarelliauthor=E.+Carcarinoauthor=V.+D%E2%80%99Oriaauthor=M.+Porruauthor=E.+Giordaauthor=K.+Ferrariauthor=L.+Mieleauthor=E.+Villaauthor=C.+Balsanoauthor=D.+Pasiniauthor=C.+Furlanelloauthor=F.+Locatelliauthor=V.+Nobiliauthor=R.+Rotaauthor=C.+Leonettiauthor=A.+Alisi&title=Focal+adhesion+kinase+depletion+reduces+human+hepatocellular+carcinoma+growth+by+repressing+enhancer+of+zeste+homolog+2&doi=10.1038%2Fcdd.2017.34"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Focal adhesion kinase depletion reduces human hepatocellular carcinoma growth by repressing enhancer of zeste homolog 2</span></div><div class="casAuthors">Gnani, Daniela; Romito, Ilaria; Artuso, Simona; Chierici, Marco; De Stefanis, Cristiano; Panera, Nadia; Crudele, Annalisa; Ceccarelli, Sara; Carcarino, Elena; D'Oria, Valentina; Porru, Manuela; Giorda, Ezio; Ferrari, Karin; Miele, Luca; Villa, Erica; Balsano, Clara; Pasini, Diego; Furlanello, Cesare; Locatelli, Franco; Nobili, Valerio; Rota, Rossella; Leonetti, Carlo; Alisi, Anna</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Differentiation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">889-902</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is the most common type of liver cancer in humans.  The focal adhesion tyrosine kinase (FAK) is often over-expressed in human HCC and FAK inhibition may reduce HCC cell invasiveness.  However, the anti-oncogenic effect of FAK knockdown in HCC cells remains to be clarified.  We found that FAK depletion in HCC cells reduced in vitro and in vivo tumorigenicity, by inducing G2/M arrest and apoptosis, decreasing anchorage-independent growth, and modulating the expression of several cancer-related genes.  Among these genes, we showed that FAK silencing decreased transcription and nuclear localization of enhancer of zeste homolog 2 (EZH2) and its tri-methylation activity on lysine 27 of histone H3 (H3K27me3).  Accordingly, FAK, EZH2 and H3K27me3 were concomitantly upregulated in human HCCs compared to non-tumor livers.  In vitro expts. demonstrated that FAK affected EZH2 expression and function by modulating, at least in part, p53 and E2F2/3 transcriptional activity.  Moreover, FAK silencing downregulated both EZH2 binding and histone H3K27me3 levels at the promoter of its target gene NOTCH2.  Finally, we found that pharmacol. inhibition of FAK activity resembled these effects although milder.  In summary, we demonstrate that FAK depletion reduces HCC cell growth by affecting cancer-promoting genes including the pro-oncogene EZH2.  Furthermore, we unveil a novel unprecedented FAK/EZH2 crosstalk in HCC cells, thus identifying a targetable network paving the way for new anticancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR0le0fu-lMrVg90H21EOLACvtfcHk0lj4CX9P6H1Lfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFGqt7g%253D&md5=d9cd4ec78e71f3ade7f8e748172199ad</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fcdd.2017.34&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcdd.2017.34%26sid%3Dliteratum%253Aachs%26aulast%3DGnani%26aufirst%3DD.%26aulast%3DRomito%26aufirst%3DI.%26aulast%3DArtuso%26aufirst%3DS.%26aulast%3DChierici%26aufirst%3DM.%26aulast%3DDe%2BStefanis%26aufirst%3DC.%26aulast%3DPanera%26aufirst%3DN.%26aulast%3DCrudele%26aufirst%3DA.%26aulast%3DCeccarelli%26aufirst%3DS.%26aulast%3DCarcarino%26aufirst%3DE.%26aulast%3DD%25E2%2580%2599Oria%26aufirst%3DV.%26aulast%3DPorru%26aufirst%3DM.%26aulast%3DGiorda%26aufirst%3DE.%26aulast%3DFerrari%26aufirst%3DK.%26aulast%3DMiele%26aufirst%3DL.%26aulast%3DVilla%26aufirst%3DE.%26aulast%3DBalsano%26aufirst%3DC.%26aulast%3DPasini%26aufirst%3DD.%26aulast%3DFurlanello%26aufirst%3DC.%26aulast%3DLocatelli%26aufirst%3DF.%26aulast%3DNobili%26aufirst%3DV.%26aulast%3DRota%26aufirst%3DR.%26aulast%3DLeonetti%26aufirst%3DC.%26aulast%3DAlisi%26aufirst%3DA.%26atitle%3DFocal%2520adhesion%2520kinase%2520depletion%2520reduces%2520human%2520hepatocellular%2520carcinoma%2520growth%2520by%2520repressing%2520enhancer%2520of%2520zeste%2520homolog%25202%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2017%26volume%3D24%26spage%3D889%26epage%3D902%26doi%3D10.1038%2Fcdd.2017.34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirt, U. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waizenegger, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslinger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braunger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weyer-Czernilofsky, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadtmuller, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapountzis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bister, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quant, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garin-Chesa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adolf, G. R.</span></span> <span> </span><span class="NLM_article-title">Efficacy of the highly selective focal adhesion kinase inhibitor BI 853520 in adenocarcinoma xenograft models is linked to a mesenchymal tumor phenotype</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">21</span>, <span class="refDoi"> DOI: 10.1038/s41389-018-0032-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fs41389-018-0032-z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=21&author=U.+A.+Hirtauthor=I.+C.+Waizeneggerauthor=N.+Schweiferauthor=C.+Haslingerauthor=D.+Gerlachauthor=J.+Braungerauthor=U.+Weyer-Czernilofskyauthor=H.+Stadtmullerauthor=I.+Sapountzisauthor=G.+Baderauthor=A.+Zoephelauthor=B.+Bisterauthor=A.+Baumauthor=J.+Quantauthor=N.+Krautauthor=P.+Garin-Chesaauthor=G.+R.+Adolf&title=Efficacy+of+the+highly+selective+focal+adhesion+kinase+inhibitor+BI+853520+in+adenocarcinoma+xenograft+models+is+linked+to+a+mesenchymal+tumor+phenotype&doi=10.1038%2Fs41389-018-0032-z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fs41389-018-0032-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41389-018-0032-z%26sid%3Dliteratum%253Aachs%26aulast%3DHirt%26aufirst%3DU.%2BA.%26aulast%3DWaizenegger%26aufirst%3DI.%2BC.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DHaslinger%26aufirst%3DC.%26aulast%3DGerlach%26aufirst%3DD.%26aulast%3DBraunger%26aufirst%3DJ.%26aulast%3DWeyer-Czernilofsky%26aufirst%3DU.%26aulast%3DStadtmuller%26aufirst%3DH.%26aulast%3DSapountzis%26aufirst%3DI.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DBister%26aufirst%3DB.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DQuant%26aufirst%3DJ.%26aulast%3DKraut%26aufirst%3DN.%26aulast%3DGarin-Chesa%26aufirst%3DP.%26aulast%3DAdolf%26aufirst%3DG.%2BR.%26atitle%3DEfficacy%2520of%2520the%2520highly%2520selective%2520focal%2520adhesion%2520kinase%2520inhibitor%2520BI%2520853520%2520in%2520adenocarcinoma%2520xenograft%2520models%2520is%2520linked%2520to%2520a%2520mesenchymal%2520tumor%2520phenotype%26jtitle%3DOncogenesis%26date%3D2018%26volume%3D7%26spage%3D21%26doi%3D10.1038%2Fs41389-018-0032-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tiede, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer-Schaller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalathur, R. K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagiani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmassmann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stillhard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafliger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waizenegger, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bill, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christofori, G.</span></span> <span> </span><span class="NLM_article-title">The FAK inhibitor BI 853520 exerts anti-tumor effects in breast cancer</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">73</span>, <span class="refDoi"> DOI: 10.1038/s41389-018-0083-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fs41389-018-0083-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=30237500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A280%3ADC%252BB3czhtVamtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=73&author=S.+Tiedeauthor=N.+Meyer-Schallerauthor=R.+K.+R.+Kalathurauthor=R.+Ivanekauthor=E.+Fagianiauthor=P.+Schmassmannauthor=P.+Stillhardauthor=S.+Hafligerauthor=N.+Krautauthor=N.+Schweiferauthor=I.+C.+Waizeneggerauthor=R.+Billauthor=G.+Christofori&title=The+FAK+inhibitor+BI+853520+exerts+anti-tumor+effects+in+breast+cancer&doi=10.1038%2Fs41389-018-0083-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The FAK inhibitor BI 853520 exerts anti-tumor effects in breast cancer</span></div><div class="casAuthors">Tiede Stefanie; Meyer-Schaller Nathalie; Kalathur Ravi Kiran Reddy; Ivanek Robert; Fagiani Ernesta; Schmassmann Philip; Stillhard Patrick; Hafliger Simon; Bill Ruben; Christofori Gerhard; Kalathur Ravi Kiran Reddy; Ivanek Robert; Kraut Norbert; Schweifer Norbert; Waizenegger Irene C</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">73</span>
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    </div><div class="casAbstract">Focal adhesion kinase (FAK) is a cytoplasmic tyrosine kinase that regulates a plethora of downstream signaling pathways essential for cell migration, proliferation and death, processes that are exploited by cancer cells during malignant progression.  These well-established tumorigenic activities, together with its high expression and activity in different cancer types, highlight FAK as an attractive target for cancer therapy.  We have assessed and characterized the therapeutic potential and the biological effects of BI 853520, a novel small chemical inhibitor of FAK, in several preclinical mouse models of breast cancer.  Treatment with BI 853520 elicits a significant reduction in primary tumor growth caused by an anti-proliferative activity by BI 853520.  In contrast, BI 853520 exerts effects with varying degrees of robustness on the different stages of the metastatic cascade.  Together, the data demonstrate that the repression of FAK activity by the specific FAK inhibitor BI 853520 offers a promising anti-proliferative approach for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSPh9ctMIKK5LxanA7vODO6fW6udTcc2eaeZPsqSVUMcrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czhtVamtA%253D%253D&md5=9875efe4b07431bf3399fd19a3f7f289</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41389-018-0083-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41389-018-0083-1%26sid%3Dliteratum%253Aachs%26aulast%3DTiede%26aufirst%3DS.%26aulast%3DMeyer-Schaller%26aufirst%3DN.%26aulast%3DKalathur%26aufirst%3DR.%2BK.%2BR.%26aulast%3DIvanek%26aufirst%3DR.%26aulast%3DFagiani%26aufirst%3DE.%26aulast%3DSchmassmann%26aufirst%3DP.%26aulast%3DStillhard%26aufirst%3DP.%26aulast%3DHafliger%26aufirst%3DS.%26aulast%3DKraut%26aufirst%3DN.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DWaizenegger%26aufirst%3DI.%2BC.%26aulast%3DBill%26aufirst%3DR.%26aulast%3DChristofori%26aufirst%3DG.%26atitle%3DThe%2520FAK%2520inhibitor%2520BI%2520853520%2520exerts%2520anti-tumor%2520effects%2520in%2520breast%2520cancer%26jtitle%3DOncogenesis%26date%3D2018%26volume%3D7%26spage%3D73%26doi%3D10.1038%2Fs41389-018-0083-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanjoni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uryu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, J.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mielgo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupack, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlaepfer, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlaepfer, D. D.</span></span> <span> </span><span class="NLM_article-title">PND-1186 FAK inhibitor selectively promotes tumor cell apoptosis in three-dimensional environments</span>. <i>Cancer Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">764</span>– <span class="NLM_lpage">777</span>, <span class="refDoi"> DOI: 10.4161/cbt.9.10.11434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.4161%2Fcbt.9.10.11434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=20234191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKrtrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=764-777&author=I.+Tanjoniauthor=C.+Walshauthor=S.+Uryuauthor=A.+Tomarauthor=J.-O.+Namauthor=A.+Mielgoauthor=S.-T.+Limauthor=C.+Liangauthor=M.+Koenigauthor=N.+Patelauthor=C.+Kwokauthor=G.+McMahonauthor=D.+G.+Stupackauthor=D.+D.+Schlaepferauthor=D.+D.+Schlaepfer&title=PND-1186+FAK+inhibitor+selectively+promotes+tumor+cell+apoptosis+in+three-dimensional+environments&doi=10.4161%2Fcbt.9.10.11434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">PND-1186 FAK inhibitor selectively promotes tumor cell apoptosis in three-dimensional environments</span></div><div class="casAuthors">Tanjoni, Isabelle; Walsh, Colin; Uryu, Sean; Tomar, Alok; Nam, Ju-Ock; Mielgo, Ainhoa; Lim, Ssang-Taek; Liang, Congxin; Koenig, Marcel; Sun, Connie; Patel, Neela; Kwok, Cheni; McMahon, Gerald; Stupack, Dwayne G.; Schlaepfer, David D.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Therapy</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">764-777</span>CODEN:
                <span class="NLM_cas:coden">CBTAAO</span>;
        ISSN:<span class="NLM_cas:issn">1538-4047</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Tumor cells can grow in an anchorage-independent manner.  This is mediated in part through survival signals that bypass normal growth restraints controlled by integrin cell surface receptors.  Focal adhesion kinase (FAK) is a cytoplasmic protein-tyrosine kinase that assocs. with integrins and modulates various cellular processes including growth, survival and migration.  As increased FAK expression and tyrosine phosphorylation are assocd. with tumor progression, inhibitors of FAK are being tested for anti-tumor effects.  Here, we analyze PND-1186, a substituted pyridine reversible inhibitor of FAK activity with a 50% inhibitory concn. (IC50) of 1.5 nM in vitro.  PND-1186 has an IC50 of ∼100 nM in breast carcinoma cells as detd. by anti-phospho-specific immunoblotting to FAK Tyr-397.  PND-1186 did not alter c-Src or p130Cas tyrosine phosphorylation in adherent cells, yet functioned to restrain cell movement.  Notably, 1.0 μM PND-1186 (>5-fold above IC50) had limited effects on cell proliferation.  However, under non-adherent conditions as spheroids and as colonies in soft agar, 0.1 μM PND-1186 blocked FAK and p130Cas tyrosine phosphorylation, promoted caspase-3 activation, and triggered cell apoptosis.  PND-1186 inhibited 4T1 breast carcinoma s.c. tumor growth correlated with elevated tumor cell apoptosis and caspase 3 activation.  Addn. of PND-1186 to the drinking water of mice was well tolerated and inhibited ascites- and peritoneal membrane-assocd. ovarian carcinoma tumor growth assocd. with the inhibition of FAK Tyr-397 phosphorylation.  Our results with low-level PND-1186 treatment support the conclusion that FAK activity selectively promotes tumor cell survival in three-dimensional environments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsLnD2EGHjs7Vg90H21EOLACvtfcHk0lj4CX9P6H1Lfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKrtrnI&md5=5cb6370e769d6e4f88ef5e27d70a7996</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4161%2Fcbt.9.10.11434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcbt.9.10.11434%26sid%3Dliteratum%253Aachs%26aulast%3DTanjoni%26aufirst%3DI.%26aulast%3DWalsh%26aufirst%3DC.%26aulast%3DUryu%26aufirst%3DS.%26aulast%3DTomar%26aufirst%3DA.%26aulast%3DNam%26aufirst%3DJ.-O.%26aulast%3DMielgo%26aufirst%3DA.%26aulast%3DLim%26aufirst%3DS.-T.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DKoenig%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DKwok%26aufirst%3DC.%26aulast%3DMcMahon%26aufirst%3DG.%26aulast%3DStupack%26aufirst%3DD.%2BG.%26aulast%3DSchlaepfer%26aufirst%3DD.%2BD.%26aulast%3DSchlaepfer%26aufirst%3DD.%2BD.%26atitle%3DPND-1186%2520FAK%2520inhibitor%2520selectively%2520promotes%2520tumor%2520cell%2520apoptosis%2520in%2520three-dimensional%2520environments%26jtitle%3DCancer%2520Biol.%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D764%26epage%3D777%26doi%3D10.4161%2Fcbt.9.10.11434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cance, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golubovskaya, V. M.</span></span> <span> </span><span class="NLM_article-title">Focal adhesion kinase versus p53: apoptosis or survival?</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">pe22</span>, <span class="refDoi"> DOI: 10.1126/stke.120pe22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1126%2Fstke.120pe22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=18493017" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2008&pages=pe22&author=W.+G.+Canceauthor=V.+M.+Golubovskaya&title=Focal+adhesion+kinase+versus+p53%3A+apoptosis+or+survival%3F&doi=10.1126%2Fstke.120pe22"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fstke.120pe22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fstke.120pe22%26sid%3Dliteratum%253Aachs%26aulast%3DCance%26aufirst%3DW.%2BG.%26aulast%3DGolubovskaya%26aufirst%3DV.%2BM.%26atitle%3DFocal%2520adhesion%2520kinase%2520versus%2520p53%253A%2520apoptosis%2520or%2520survival%253F%26jtitle%3DSci.%2520Signaling%26date%3D2008%26volume%3D1%26spage%3Dpe22%26doi%3D10.1126%2Fstke.120pe22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lj9hx8KNFNacA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras: induced protein degradation as a therapeutic strategy</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=892-898&author=P.+Ottisauthor=C.+M.+Crews&title=Proteolysis-targeting+chimeras%3A+induced+protein+degradation+as+a+therapeutic+strategy&doi=10.1021%2Facschembio.6b01068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span></div><div class="casAuthors">Ottis, Philipp; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-898</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Until recently, the only ways to reduce specific protein signaling were to either knock down the target by RNAi or to interfere with the signaling by inhibiting an enzyme or receptor within the signal transduction cascade.  Herein, we review an emerging class of small mol. pharmacol. agents, called PROTACs, that present a novel approach to specifically target proteins and their resp. signaling pathways.  These heterobifunctional mols. utilize endogenous cellular quality control machinery by recruiting it to target proteins in order to induce their degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohB6xwkQMTprVg90H21EOLACvtfcHk0lj9hx8KNFNacA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D&md5=eada521c10faa1e456e0885fd390c554</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01068%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-targeting%2520chimeras%253A%2520induced%2520protein%2520degradation%2520as%2520a%2520therapeutic%2520strategy%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D892%26epage%3D898%26doi%3D10.1021%2Facschembio.6b01068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span> <span> </span><span class="NLM_article-title">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>474</i></span>,  <span class="NLM_fpage">1127</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi"> DOI: 10.1042/bcj20160762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1042%2Fbcj20160762" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2017&pages=1127-1147&author=I.+Collinsauthor=H.+Wangauthor=J.+J.+Caldwellauthor=R.+Chopra&title=Chemical+approaches+to+targeted+protein+degradation+through+modulation+of+the+ubiquitin-proteasome+pathway&doi=10.1042%2Fbcj20160762"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1042%2Fbcj20160762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbcj20160762%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DChemical%2520approaches%2520to%2520targeted%2520protein%2520degradation%2520through%2520modulation%2520of%2520the%2520ubiquitin-proteasome%2520pathway%26jtitle%3DBiochem.%2520J.%26date%3D2017%26volume%3D474%26spage%3D1127%26epage%3D1147%26doi%3D10.1042%2Fbcj20160762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1042/ebc20170041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1042%2Febc20170041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=29118097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=505-516&author=S.+J.+Hughesauthor=A.+Ciulli&title=Molecular+recognition+of+ternary+complexes%3A+a+new+dimension+in+the+structure-guided+design+of+chemical+degraders&doi=10.1042%2Febc20170041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders</span></div><div class="casAuthors">Hughes Scott J; Ciulli Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">505-516</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular glues and bivalent inducers of protein degradation (also known as PROTACs) represent a fascinating new modality in pharmacotherapeutics: the potential to knockdown previously thought 'undruggable' targets at sub-stoichiometric concentrations in ways not possible using conventional inhibitors.  Mounting evidence suggests these chemical agents, in concert with their target proteins, can be modelled as three-body binding equilibria that can exhibit significant cooperativity as a result of specific ligand-induced molecular recognition.  Despite this, many existing drug design and optimization regimens still fixate on binary target engagement, in part due to limited structural data on ternary complexes.  Recent crystal structures of protein complexes mediated by degrader molecules, including the first PROTAC ternary complex, underscore the importance of protein-protein interactions and intramolecular contacts to the mode of action of this class of compounds.  These discoveries have opened the door to a new paradigm for structure-guided drug design: borrowing surface area and molecular recognition from nature to elicit cellular signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0b5XHngOXiV_fW6udTcc2eZrElmZgS_wM7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D&md5=7470e559372f05d34c5924524a6d04bc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1042%2Febc20170041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Febc20170041%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DMolecular%2520recognition%2520of%2520ternary%2520complexes%253A%2520a%2520new%2520dimension%2520in%2520the%2520structure-guided%2520design%2520of%2520chemical%2520degraders%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D505%26epage%3D516%26doi%3D10.1042%2Febc20170041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safaee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponthier, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancias, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Plasticity in binding confers selectivity in ligand-induced protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0055-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fs41589-018-0055-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=29892083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=706-714&author=R.+P.+Nowakauthor=S.+L.+DeAngeloauthor=D.+Buckleyauthor=Z.+Heauthor=K.+A.+Donovanauthor=J.+Anauthor=N.+Safaeeauthor=M.+P.+Jedrychowskiauthor=C.+M.+Ponthierauthor=M.+Ishoeyauthor=T.+Zhangauthor=J.+D.+Manciasauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=E.+S.+Fischer&title=Plasticity+in+binding+confers+selectivity+in+ligand-induced+protein+degradation&doi=10.1038%2Fs41589-018-0055-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Plasticity in binding confers selectivity in ligand-induced protein degradation</span></div><div class="casAuthors">Nowak, Radoslaw P.; De Angelo, Stephen L.; Buckley, Dennis; He, Zhixiang; Donovan, Katherine A.; An, Jian; Safaee, Nozhat; Jedrychowski, Mark P.; Ponthier, Charles M.; Ishoey, Mette; Zhang, Tinghu; Mancias, Joseph D.; Gray, Nathanael S.; Bradner, James E.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">706-714</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Heterobifunctional small-mol. degraders that induce protein degrdn. through ligase-mediated ubiquitination have shown considerable promise as a new pharmacol. modality.  However, we currently lack a detailed understanding of the mol. basis for target recruitment and selectivity, which is critically required to enable rational design of degraders.  Here we utilized a comprehensive characterization of the ligand-dependent CRBN-BRD4 interaction to demonstrate that binding between proteins that have not evolved to interact is plastic.  Multiple x-ray crystal structures showed that plasticity resulted in several distinct low-energy binding conformations that were selectively bound by ligands.  We demonstrated that computational protein-protein docking can reveal the underlying interprotein contacts and inform the design of a BRD4 selective degrader that can discriminate between highly homologous BET bromodomains.  These findings that plastic interprotein contacts confer selectivity for ligand-induced protein dimerization provide a conceptual framework for the development of heterobifunctional ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreQasCW8bz2LVg90H21EOLACvtfcHk0lifvfQEWWWbUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK&md5=e4876c43b1eeb4558c23121b7a9fac27</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0055-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0055-y%26sid%3Dliteratum%253Aachs%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DBuckley%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DSafaee%26aufirst%3DN.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DPonthier%26aufirst%3DC.%2BM.%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DMancias%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DPlasticity%2520in%2520binding%2520confers%2520selectivity%2520in%2520ligand-induced%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D706%26epage%3D714%26doi%3D10.1038%2Fs41589-018-0055-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.-H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lgYnsXMUpb1rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Impact of target warhead and linkage vector on inducing protein degradation: comparison of bromodomain and extra-terminal (BET) degraders derived from triazolodiazepine (JQ1) and tetrahydroquinoline (I-BET726) BET inhibitor scaffolds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01912</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=504-513&author=K.-H.+Chanauthor=M.+Zengerleauthor=A.+Testaauthor=A.+Ciulli&title=Impact+of+target+warhead+and+linkage+vector+on+inducing+protein+degradation%3A+comparison+of+bromodomain+and+extra-terminal+%28BET%29+degraders+derived+from+triazolodiazepine+%28JQ1%29+and+tetrahydroquinoline+%28I-BET726%29+BET+inhibitor+scaffolds&doi=10.1021%2Facs.jmedchem.6b01912"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DImpact%2520of%2520target%2520warhead%2520and%2520linkage%2520vector%2520on%2520inducing%2520protein%2520degradation%253A%2520comparison%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520degraders%2520derived%2520from%2520triazolodiazepine%2520%2528JQ1%2529%2520and%2520tetrahydroquinoline%2520%2528I-BET726%2529%2520BET%2520inhibitor%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D61%26spage%3D504%26epage%3D513%26doi%3D10.1021%2Facs.jmedchem.6b01912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lgYnsXMUpb1rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0liQZiS6Z-cL8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wühr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the BAF complex factor BRD9 by heterobifunctional ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.-S.+Seoauthor=S.+Dastjerdiauthor=M.+W%C3%BChrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+BAF+complex+factor+BRD9+by+heterobifunctional+ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0liQZiS6Z-cL8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DW%25C3%25BChr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520BAF%2520complex%2520factor%2520BRD9%2520by%2520heterobifunctional%2520ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0liQZiS6Z-cL8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerschmied, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Modular PROTAC design for the degradation of oncogenic BCR-ABL</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+design+for+the+degradation+of+oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520design%2520for%2520the%2520degradation%2520of%2520oncogenic%2520BCR-ABL%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petroski, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Function and regulation of cullin-RING ubiquitin ligases</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1038/nrm1547</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnrm1547" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=15688063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVWg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=9-20&author=M.+D.+Petroskiauthor=R.+J.+Deshaies&title=Function+and+regulation+of+cullin-RING+ubiquitin+ligases&doi=10.1038%2Fnrm1547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Function and regulation of cullin-RING ubiquitin ligases</span></div><div class="casAuthors">Petroski, Matthew D.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-20</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cullin-RING complexes comprise the largest known class of ubiquitin ligases.  Owing to the great diversity of their substrate-receptor subunits, it is possible that there are hundreds of distinct cullin-RING ubiquitin ligases in eukaryotic cells, which establishes these enzymes as key mediators of post-translational protein regulation.  Here, the authors focus on the compn., regulation, and function of cullin-RING ligases, and describe how these enzymes can be characterized by a set of general principles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFw5LuMMHZoLVg90H21EOLACvtfcHk0lj-bExT_Dh6eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVWg&md5=297f142f44f971c6858306b1adb36254</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrm1547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1547%26sid%3Dliteratum%253Aachs%26aulast%3DPetroski%26aufirst%3DM.%2BD.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DFunction%2520and%2520regulation%2520of%2520cullin-RING%2520ubiquitin%2520ligases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D9%26epage%3D20%26doi%3D10.1038%2Fnrm1547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braggio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, C.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruins, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Wier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, X.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjorklund, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonseca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsagel, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlowski, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, A. K.</span></span> <span> </span><span class="NLM_article-title">Cereblon expression is required for the antimyeloma activity of lenalidomide and pomalidomide</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">4771</span>– <span class="NLM_lpage">4779</span>, <span class="refDoi"> DOI: 10.1182/blood-2011-05-356063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1182%2Fblood-2011-05-356063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=21860026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOgsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2011&pages=4771-4779&author=Y.+X.+Zhuauthor=E.+Braggioauthor=C.-X.+Shiauthor=L.+A.+Bruinsauthor=J.+E.+Schmidtauthor=S.+Van+Wierauthor=X.-B.+Changauthor=C.+C.+Bjorklundauthor=R.+Fonsecaauthor=P.+L.+Bergsagelauthor=R.+Z.+Orlowskiauthor=A.+K.+Stewart&title=Cereblon+expression+is+required+for+the+antimyeloma+activity+of+lenalidomide+and+pomalidomide&doi=10.1182%2Fblood-2011-05-356063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Cereblon expression is required for the antimyeloma activity of lenalidomide and pomalidomide</span></div><div class="casAuthors">Zhu, Yuan Xiao; Braggio, Esteban; Shi, Chang-Xin; Bruins, Laura A.; Schmidt, Jessica E.; Van Wier, Scott; Chang, Xiu-Bao; Bjorklund, Chad C.; Fonseca, Rafael; Bergsagel, P. Leif; Orlowski, Robert Z.; Stewart, A. Keith</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4771-4779</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The precise mol. mechanism of action and targets through which thalidomide and related immunomodulatory drugs (IMiDs) exert their antitumor effects remains unclear.  We investigated the role of cereblon (CRBN), a primary teratogenic target of thalidomide, in the antimyeloma activity of IMiDs.  CRBN depletion is initially cytotoxic to human myeloma cells, but surviving cells with stable CRBN depletion become highly resistant to both lenalidomide and pomalidomide, but not to the unrelated drugs bortezomib, dexamethasone, and melphalan.  Acquired deletion of CRBN was found to be the primary genetic event differentiating isogenic MM1.S cell lines cultured to be sensitive or resistant to lenalidomide and pomalidomide.  Gene expression changes induced by lenalidomide were dramatically suppressed in the presence of CRBN depletion, further demonstrating that CRBN is required for lenalidomide activity.  Downstream targets of CRBN include interferon regulatory factor 4 (IRF4) previously reported to also be a target of lenalidomide.  Patients exposed to, and putatively resistant to, lenalidomide had lower CRBN levels in paired samples before and after therapy.  In summary, CRBN is an essential requirement for IMiD activity and a possible biomarker for the clin. assessment of antimyeloma efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJCuFi6UHfKrVg90H21EOLACvtfcHk0lj-bExT_Dh6eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOgsrjO&md5=2302c491fec7729d6c2c08735836a585</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1182%2Fblood-2011-05-356063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2011-05-356063%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DY.%2BX.%26aulast%3DBraggio%26aufirst%3DE.%26aulast%3DShi%26aufirst%3DC.-X.%26aulast%3DBruins%26aufirst%3DL.%2BA.%26aulast%3DSchmidt%26aufirst%3DJ.%2BE.%26aulast%3DVan%2BWier%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DX.-B.%26aulast%3DBjorklund%26aufirst%3DC.%2BC.%26aulast%3DFonseca%26aufirst%3DR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DOrlowski%26aufirst%3DR.%2BZ.%26aulast%3DStewart%26aufirst%3DA.%2BK.%26atitle%3DCereblon%2520expression%2520is%2520required%2520for%2520the%2520antimyeloma%2520activity%2520of%2520lenalidomide%2520and%2520pomalidomide%26jtitle%3DBlood%26date%3D2011%26volume%3D118%26spage%3D4771%26epage%3D4779%26doi%3D10.1182%2Fblood-2011-05-356063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loehr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahcall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of anaplastic lymphoma kinase (ALK)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4249</span>– <span class="NLM_lpage">4255</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4249-4255&author=C.+E.+Powellauthor=Y.+Gaoauthor=L.+Tanauthor=K.+A.+Donovanauthor=R.+P.+Nowakauthor=A.+Loehrauthor=M.+Bahcallauthor=E.+S.+Fischerauthor=P.+A.+J%C3%A4nneauthor=R.+E.+Georgeauthor=N.+S.+Gray&title=Chemically+induced+degradation+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1021%2Facs.jmedchem.7b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Powell, Chelsea E.; Gao, Yang; Tan, Li; Donovan, Katherine A.; Nowak, Radoslaw P.; Loehr, Amanda; Bahcall, Magda; Fischer, Eric S.; Janne, Pasi A.; George, Rani E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4249-4255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the development of the first small mol. degraders that can induce anaplastic lymphoma kinase (ALK) degrdn., including in non-small-cell lung cancer (NSCLC), anaplastic large-cell lymphoma (ALCL), and neuroblastoma (NB) cell lines.  These degraders were developed through conjugation of known pyrimidine-based ALK inhibitors, TAE684 or LDK378, and the cereblon ligand pomalidomide.  We demonstrate that in some cell types degrader potency is compromised by expression of drug transporter ABCB1.  In addn., proteomic profiling demonstrated that these compds. also promote the degrdn. of addnl. kinases including PTK2 (FAK), Aurora A, FER, and RPS6KA1 (RSK1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mDH7Sj8EqbVg90H21EOLACvtfcHk0lj-bExT_Dh6eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D&md5=486db11b3fbc2283700f38b09cc39bd7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01655%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DC.%2BE.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DLoehr%26aufirst%3DA.%26aulast%3DBahcall%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGeorge%26aufirst%3DR.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4249%26epage%3D4255%26doi%3D10.1021%2Facs.jmedchem.7b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A chemoproteomic approach to query the degradable kinome using a multi-kinase degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1016%2Fj.chembiol.2017.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=29129717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=88-99&author=H.-T.+Huangauthor=D.+Dobrovolskyauthor=J.+Paulkauthor=G.+Yangauthor=E.+L.+Weisbergauthor=Z.+M.+Doctorauthor=D.+L.+Buckleyauthor=J.-H.+Choauthor=E.+Koauthor=J.+Jangauthor=K.+Shiauthor=H.+G.+Choiauthor=J.+D.+Griffinauthor=Y.+Liauthor=S.+P.+Treonauthor=E.+S.+Fischerauthor=J.+E.+Bradnerauthor=L.+Tanauthor=N.+S.+Gray&title=A+chemoproteomic+approach+to+query+the+degradable+kinome+using+a+multi-kinase+degrader&doi=10.1016%2Fj.chembiol.2017.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span></div><div class="casAuthors">Huang, Hai-Tsang; Dobrovolsky, Dennis; Paulk, Joshiawa; Yang, Guang; Weisberg, Ellen L.; Doctor, Zainab M.; Buckley, Dennis L.; Cho, Joong-Heui; Ko, Eunhwa; Jang, Jaebong; Shi, Kun; Choi, Hwan Geun; Griffin, James D.; Li, Ying; Treon, Steven P.; Fischer, Eric S.; Bradner, James E.; Tan, Li; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heterobifunctional mols. that recruit E3 ubiquitin ligases, such as cereblon, for targeted protein degrdn. represent an emerging pharmacol. strategy.  A major unanswered question is how generally applicable this strategy is to all protein targets.  In this study, we designed a multi-kinase degrader by conjugating a highly promiscuous kinase inhibitor with a cereblon-binding ligand, and used quant. proteomics to discover 28 kinases, including BTK, PTK2, PTK2B, FLT3, AURKA, AURKB, TEC, ULK1, ITK, and nine members of the CDK family, as degradable.  This set of kinases is only a fraction of the intracellular targets bound by the degrader, demonstrating that successful degrdn. requires more than target engagement.  The results guided us to develop selective degraders for FLT3 and BTK, with potentials to improve disease treatment.  Together, this study demonstrates an efficient approach to triage a gene family of interest to identify readily degradable targets for further studies and pre-clin. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYNxhoPV3127Vg90H21EOLACvtfcHk0lhXKr4FR60bBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO&md5=703cfbaca3edd24176472158563f895f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.-T.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.-H.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520chemoproteomic%2520approach%2520to%2520query%2520the%2520degradable%2520kinome%2520using%2520a%2520multi-kinase%2520degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D88%26epage%3D99%26doi%3D10.1016%2Fj.chembiol.2017.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Weck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castelletti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gampa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sovath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caothien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caushi, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortés-Cros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaunay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desplat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwoske, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldridge, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farsidjani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flemming, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattenberger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapoor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollé, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phadke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repko, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schouwey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucke, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiedt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varadarajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Project DRIVE: a compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1016%2Fj.cell.2017.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=28753431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=577-592&author=E.+R.+McDonaldauthor=A.+de%0AWeckauthor=M.+R.+Schlabachauthor=E.+Billyauthor=K.+J.+Mavrakisauthor=G.+R.+Hoffmanauthor=D.+Belurauthor=D.+Castellettiauthor=E.+Friasauthor=K.+Gampaauthor=J.+Goljiauthor=I.+Kaoauthor=L.+Liauthor=P.+Megelauthor=T.+A.+Perkinsauthor=N.+Ramadanauthor=D.+A.+Ruddyauthor=S.+J.+Silverauthor=S.+Sovathauthor=M.+Stumpauthor=O.+Weberauthor=R.+Widmerauthor=J.+Yuauthor=K.+Yuauthor=Y.+Yueauthor=D.+Abramowskiauthor=E.+Ackleyauthor=R.+Barrettauthor=J.+Bergerauthor=J.+L.+Bernardauthor=R.+Billigauthor=S.+M.+Brachmannauthor=F.+Buxtonauthor=R.+Caothienauthor=J.+X.+Caushiauthor=F.+S.+Chungauthor=M.+Cort%C3%A9s-Crosauthor=R.+S.+deBeaumontauthor=C.+Delaunayauthor=A.+Desplatauthor=W.+Duongauthor=D.+A.+Dwoskeauthor=R.+S.+Eldridgeauthor=A.+Farsidjaniauthor=F.+Fengauthor=J.+Fengauthor=D.+Flemmingauthor=W.+Forresterauthor=G.+G.+Galliauthor=Z.+Gaoauthor=F.+Gauterauthor=V.+Gibajaauthor=K.+Haasauthor=M.+Hattenbergerauthor=T.+Hoodauthor=K.+E.+Hurovauthor=Z.+Jaganiauthor=M.+Jenalauthor=J.+A.+Johnsonauthor=M.+D.+Jonesauthor=A.+Kapoorauthor=J.+Kornauthor=J.+Liuauthor=Q.+Liuauthor=S.+Liuauthor=Y.+Liuauthor=A.+T.+Looauthor=K.+J.+Macchiauthor=T.+Martinauthor=G.+McAllisterauthor=A.+Meyerauthor=S.+Molleauthor=R.+A.+Moll%C3%A9author=T.+Phadkeauthor=B.+Repkoauthor=T.+Schouweyauthor=F.+Shanahanauthor=Q.+Shenauthor=C.+Stammauthor=C.+Stephanauthor=V.+M.+Stuckeauthor=R.+Tiedtauthor=M.+Varadarajanauthor=K.+Venkatesanauthor=A.+C.+Vitariauthor=M.+Wallrothauthor=J.+Weilerauthor=J.+Zhangauthor=C.+Mickaninauthor=V.+E.+Myerauthor=J.+A.+Porterauthor=A.+Laiauthor=H.+Bitterauthor=E.+Leesauthor=N.+Keenauthor=A.+Kauffmannauthor=F.+Stegmeierauthor=F.+Hofmannauthor=T.+Schmelzleauthor=W.+R.+Sellers&title=Project+DRIVE%3A+a+compendium+of+cancer+dependencies+and+synthetic+lethal+relationships+uncovered+by+large-scale%2C+deep+RNAi+screening&doi=10.1016%2Fj.cell.2017.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Project DRIVE: A compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span></div><div class="casAuthors">McDonald, E. Robert, III; de Weck, Antoine; Schlabach, Michael R.; Billy, Eric; Mavrakis, Konstantinos J.; Hoffman, Gregory R.; Belur, Dhiren; Castelletti, Deborah; Frias, Elizabeth; Gampa, Kalyani; Golji, Javad; Kao, Iris; Li, Li; Megel, Philippe; Perkins, Thomas A.; Ramadan, Nadire; Ruddy, David A.; Silver, Serena J.; Sovath, Sosathya; Stump, Mark; Weber, Odile; Widmer, Roland; Yu, Jianjun; Yu, Kristine; Yue, Yingzi; Abramowski, Dorothee; Ackley, Elizabeth; Barrett, Rosemary; Berger, Joel; Bernard, Julie L.; Billig, Rebecca; Brachmann, Saskia M.; Buxton, Frank; Caothien, Roger; Caushi, Justina X.; Chung, Franklin S.; Cortes-Cros, Marta; de Beaumont, Rosalie S.; Delaunay, Clara; Desplat, Aurore; Duong, William; Dwoske, Donald A.; Eldridge, Richard S.; Farsidjani, Ali; Feng, Fei; Feng, Jia Jia; Flemming, Daisy; Forrester, William; Galli, Giorgio G.; Gao, Zhenhai; Gauter, Francois; Gibaja, Veronica; Haas, Kristy; Hattenberger, Marc; Hood, Tami; Hurov, Kristen E.; Jagani, Zainab; Jenal, Mathias; Johnson, Jennifer A.; Jones, Michael D.; Kapoor, Avnish; Korn, Joshua; Liu, Jilin; Liu, Qiumei; Liu, Shumei; Liu, Yue; Loo, Alice T.; Macchi, Kaitlin J.; Martin, Typhaine; McAllister, Gregory; Meyer, Amandine; Molle, Sandra; Pagliarini, Raymond A.; Phadke, Tanushree; Repko, Brian; Schouwey, Tanja; Shanahan, Frances; Shen, Qiong; Stamm, Christelle; Stephan, Christine; Stucke, Volker M.; Tiedt, Ralph; Varadarajan, Malini; Venkatesan, Kavitha; Vitari, Alberto C.; Wallroth, Marco; Weiler, Jan; Zhang, Jing; Mickanin, Craig; Myer, Vic E.; Porter, Jeffery A.; Lai, Albert; Bitter, Hans; Lees, Emma; Keen, Nicholas; Kauffmann, Audrey; Stegmeier, Frank; Hofmann, Francesco; Schmelzle, Tobias; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">577-592.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elucidation of the mutational landscape of human cancer has progressed rapidly and been accompanied by the development of therapeutics targeting mutant oncogenes.  However, a comprehensive mapping of cancer dependencies has lagged behind and the discovery of therapeutic targets for counteracting tumor suppressor gene loss is needed.  To identify vulnerabilities relevant to specific cancer subtypes, we conducted a large-scale RNAi screen in which viability effects of mRNA knockdown were assessed for 7837 genes using an av. of 20 shRNAs per gene in 398 cancer cell lines.  We describe findings of this screen, outlining the classes of cancer dependency genes and their relationships to genetic, expression, and lineage features.  In addn., we describe robust gene-interaction networks recapitulating both protein complexes and functional cooperation among complexes and pathways.  This dataset along with a web portal is provided to the community to assist in the discovery and translation of new therapeutic approaches for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokUs33_s2nC7Vg90H21EOLACvtfcHk0lhXKr4FR60bBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI&md5=41592e684cc8f2cd28117cb3303f6120</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3Dde%2BWeck%26aufirst%3DA.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DMavrakis%26aufirst%3DK.%2BJ.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DBelur%26aufirst%3DD.%26aulast%3DCastelletti%26aufirst%3DD.%26aulast%3DFrias%26aufirst%3DE.%26aulast%3DGampa%26aufirst%3DK.%26aulast%3DGolji%26aufirst%3DJ.%26aulast%3DKao%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMegel%26aufirst%3DP.%26aulast%3DPerkins%26aufirst%3DT.%2BA.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DSilver%26aufirst%3DS.%2BJ.%26aulast%3DSovath%26aufirst%3DS.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DWidmer%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DAckley%26aufirst%3DE.%26aulast%3DBarrett%26aufirst%3DR.%26aulast%3DBerger%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DJ.%2BL.%26aulast%3DBillig%26aufirst%3DR.%26aulast%3DBrachmann%26aufirst%3DS.%2BM.%26aulast%3DBuxton%26aufirst%3DF.%26aulast%3DCaothien%26aufirst%3DR.%26aulast%3DCaushi%26aufirst%3DJ.%2BX.%26aulast%3DChung%26aufirst%3DF.%2BS.%26aulast%3DCort%25C3%25A9s-Cros%26aufirst%3DM.%26aulast%3DdeBeaumont%26aufirst%3DR.%2BS.%26aulast%3DDelaunay%26aufirst%3DC.%26aulast%3DDesplat%26aufirst%3DA.%26aulast%3DDuong%26aufirst%3DW.%26aulast%3DDwoske%26aufirst%3DD.%2BA.%26aulast%3DEldridge%26aufirst%3DR.%2BS.%26aulast%3DFarsidjani%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFlemming%26aufirst%3DD.%26aulast%3DForrester%26aufirst%3DW.%26aulast%3DGalli%26aufirst%3DG.%2BG.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DGauter%26aufirst%3DF.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DHattenberger%26aufirst%3DM.%26aulast%3DHood%26aufirst%3DT.%26aulast%3DHurov%26aufirst%3DK.%2BE.%26aulast%3DJagani%26aufirst%3DZ.%26aulast%3DJenal%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DKapoor%26aufirst%3DA.%26aulast%3DKorn%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLoo%26aufirst%3DA.%2BT.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DMeyer%26aufirst%3DA.%26aulast%3DMolle%26aufirst%3DS.%26aulast%3DMoll%25C3%25A9%26aufirst%3DR.%2BA.%26aulast%3DPhadke%26aufirst%3DT.%26aulast%3DRepko%26aufirst%3DB.%26aulast%3DSchouwey%26aufirst%3DT.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DStephan%26aufirst%3DC.%26aulast%3DStucke%26aufirst%3DV.%2BM.%26aulast%3DTiedt%26aufirst%3DR.%26aulast%3DVaradarajan%26aufirst%3DM.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DWallroth%26aufirst%3DM.%26aulast%3DWeiler%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DKauffmann%26aufirst%3DA.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DProject%2520DRIVE%253A%2520a%2520compendium%2520of%2520cancer%2520dependencies%2520and%2520synthetic%2520lethal%2520relationships%2520uncovered%2520by%2520large-scale%252C%2520deep%2520RNAi%2520screening%26jtitle%3DCell%26date%3D2017%26volume%3D170%26spage%3D577%26epage%3D592%26doi%3D10.1016%2Fj.cell.2017.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lgTTmrCyGcVGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&author=E.+S.+Fischerauthor=K.+B%C3%B6hmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thom%C3%A4&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0lgTTmrCyGcVGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DB%25C3%25B6hm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lhWT-NIi_Sx5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lhWT-NIi_Sx5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Group-Based Optimization of Potent and Cell-Active Inhibitors of the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase: Structure-Activity Relationships Leading to the Chemical Probe (2S,4R)-1-((S)-2-(1-Cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00675</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00675" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=599-618&author=P.+Soaresauthor=M.+S.+Gaddauthor=J.+Frostauthor=C.+Galdeanoauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Rochaauthor=K.+D.+Readauthor=A.+Ciulli&title=Group-Based+Optimization+of+Potent+and+Cell-Active+Inhibitors+of+the+von+Hippel-Lindau+%28VHL%29+E3+Ubiquitin+Ligase%3A+Structure-Activity+Relationships+Leading+to+the+Chemical+Probe+%282S%2C4R%29-1-%28%28S%29-2-%281-Cyanocyclopropanecarboxamido%29-3%2C3-dimethylbutanoyl%29-4-hydroxy-N-%284-%284-methylthiazol-5-yl%29benzyl%29pyrrolidine-2-carboxamide+%28VH298%29&doi=10.1021%2Facs.jmedchem.7b00675"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00675%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DGroup-Based%2520Optimization%2520of%2520Potent%2520and%2520Cell-Active%2520Inhibitors%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520Ubiquitin%2520Ligase%253A%2520Structure-Activity%2520Relationships%2520Leading%2520to%2520the%2520Chemical%2520Probe%2520%25282S%252C4R%2529-1-%2528%2528S%2529-2-%25281-Cyanocyclopropanecarboxamido%2529-3%252C3-dimethylbutanoyl%2529-4-hydroxy-N-%25284-%25284-methylthiazol-5-yl%2529benzyl%2529pyrrolidine-2-carboxamide%2520%2528VH298%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D61%26spage%3D599%26epage%3D618%26doi%3D10.1021%2Facs.jmedchem.7b00675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzes, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Potent and selective chemical probe of hypoxic signalling downstream of HIF-alpha hydroxylation via VHL inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13312</span>, <span class="refDoi"> DOI: 10.1038/ncomms13312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fncomms13312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=27811928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13312&author=J.+Frostauthor=C.+Galdeanoauthor=P.+Soaresauthor=M.+S.+Gaddauthor=K.+M.+Grzesauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Shimamuraauthor=M.+Bantscheffauthor=P.+Grandiauthor=K.+D.+Readauthor=D.+A.+Cantrellauthor=S.+Rochaauthor=A.+Ciulli&title=Potent+and+selective+chemical+probe+of+hypoxic+signalling+downstream+of+HIF-alpha+hydroxylation+via+VHL+inhibition&doi=10.1038%2Fncomms13312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span></div><div class="casAuthors">Frost, Julianty; Galdeano, Carles; Soares, Pedro; Gadd, Morgan S.; Grzes, Katarzyna M.; Ellis, Lucy; Epemolu, Ola; Shimamura, Satoko; Bantscheff, Marcus; Grandi, Paola; Read, Kevin D.; Cantrell, Doreen A.; Rocha, Sonia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13312</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. strategies to using small mols. to stimulate hypoxia inducible factors (HIFs) activity and trigger a hypoxic response under normoxic conditions, such as iron chelators and inhibitors of prolyl hydroxylase domain (PHD) enzymes, have broad-spectrum activities and off-target effects.  Here we disclose VH298, a potent VHL inhibitor that stabilizes HIF-α and elicits a hypoxic response via a different mechanism, that is the blockade of the VHL:HIF-α protein-protein interaction downstream of HIF-α hydroxylation by PHD enzymes.  We show that VH298 engages with high affinity and specificity with VHL as its only major cellular target, leading to selective on-target accumulation of hydroxylated HIF-α in a concn.- and time-dependent fashion in different cell lines, with subsequent upregulation of HIF-target genes at both mRNA and protein levels.  VH298 represents a high-quality chem. probe of the HIF signalling cascade and an attractive starting point to the development of potential new therapeutics targeting hypoxia signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLB1WvVF4EFLVg90H21EOLACvtfcHk0liuXAYvh4WcBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ&md5=7a884b6193b3ca5e119fecea2eb31d3e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fncomms13312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13312%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DGrzes%26aufirst%3DK.%2BM.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCantrell%26aufirst%3DD.%2BA.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DPotent%2520and%2520selective%2520chemical%2520probe%2520of%2520hypoxic%2520signalling%2520downstream%2520of%2520HIF-alpha%2520hydroxylation%2520via%2520VHL%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13312%26doi%3D10.1038%2Fncomms13312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+NEDD8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0liuXAYvh4WcBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520NEDD8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saeki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanigawara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komano, T.</span></span> <span> </span><span class="NLM_article-title">Human P-glycoprotein transports cyclosporin A and FK506</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">6077</span>– <span class="NLM_lpage">6080</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1993&pages=6077-6080&author=T.+Saekiauthor=K.+Uedaauthor=Y.+Tanigawaraauthor=R.+Horiauthor=T.+Komano&title=Human+P-glycoprotein+transports+cyclosporin+A+and+FK506"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSaeki%26aufirst%3DT.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DTanigawara%26aufirst%3DY.%26aulast%3DHori%26aufirst%3DR.%26aulast%3DKomano%26aufirst%3DT.%26atitle%3DHuman%2520P-glycoprotein%2520transports%2520cyclosporin%2520A%2520and%2520FK506%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1993%26volume%3D268%26spage%3D6077%26epage%3D6080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chessum, N. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqua, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilding, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span> <span> </span><span class="NLM_article-title">Demonstrating In-cell target engagement using a pirin protein degradation probe (CCT367766)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">918</span>– <span class="NLM_lpage">933</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOqsrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=918-933&author=N.+E.+A.+Chessumauthor=S.+Y.+Sharpauthor=J.+J.+Caldwellauthor=A.+E.+Pasquaauthor=B.+Wildingauthor=G.+Colombanoauthor=I.+Collinsauthor=B.+Ozerauthor=M.+Richardsauthor=M.+Rowlandsauthor=M.+Stubbsauthor=R.+Burkeauthor=P.+C.+McAndrewauthor=P.+A.+Clarkeauthor=P.+Workmanauthor=M.+D.+Cheesemanauthor=K.+Jones&title=Demonstrating+In-cell+target+engagement+using+a+pirin+protein+degradation+probe+%28CCT367766%29&doi=10.1021%2Facs.jmedchem.7b01406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Demonstrating In-Cell Target Engagement Using a Pirin Protein Degradation Probe (CCT367766)</span></div><div class="casAuthors">Chessum, Nicola E. A.; Sharp, Swee Y.; Caldwell, John J.; Pasqua, A. Elisa; Wilding, Birgit; Colombano, Giampiero; Collins, Ian; Ozer, Bugra; Richards, Meirion; Rowlands, Martin; Stubbs, Mark; Burke, Rosemary; McAndrew, P. Craig; Clarke, Paul A.; Workman, Paul; Cheeseman, Matthew D.; Jones, Keith</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">918-933</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Demonstrating intracellular protein target engagement is an essential step in the development and progression of new chem. probes and potential small mol. therapeutics.  However, this can be particularly challenging for poorly studied and noncatalytic proteins, as robust proximal biomarkers are rarely known.  To confirm that the recently discovered chem. probe I (CCT251236) binds the putative transcription factor regulator pirin in living cells, the authors developed a heterobifunctional protein degrdn. probe.  Focusing on linker design and physicochem. properties, the authors generated a highly active probe II (CCT367766) in only three iterations, validating the efficient strategy for degrdn. probe design against nonvalidated protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofmMArlN0aWrVg90H21EOLACvtfcHk0liz4RX80jEb_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOqsrzK&md5=928d24741c26d0b66ab7ce1ef2004bbc</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01406%26sid%3Dliteratum%253Aachs%26aulast%3DChessum%26aufirst%3DN.%2BE.%2BA.%26aulast%3DSharp%26aufirst%3DS.%2BY.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DPasqua%26aufirst%3DA.%2BE.%26aulast%3DWilding%26aufirst%3DB.%26aulast%3DColombano%26aufirst%3DG.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DOzer%26aufirst%3DB.%26aulast%3DRichards%26aufirst%3DM.%26aulast%3DRowlands%26aufirst%3DM.%26aulast%3DStubbs%26aufirst%3DM.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DMcAndrew%26aufirst%3DP.%2BC.%26aulast%3DClarke%26aufirst%3DP.%2BA.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DDemonstrating%2520In-cell%2520target%2520engagement%2520using%2520a%2520pirin%2520protein%2520degradation%2520probe%2520%2528CCT367766%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D918%26epage%3D933%26doi%3D10.1021%2Facs.jmedchem.7b01406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sizemore, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharia, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubonja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strickland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawes, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhivich, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyatt, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span> <span> </span><span class="NLM_article-title">Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1779</span>– <span class="NLM_lpage">1784</span>, <span class="refDoi"> DOI: 10.1038/ng.3984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fng.3984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=29083409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslehtrjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=1779-1784&author=R.+M.+Meyersauthor=J.+G.+Bryanauthor=J.+M.+McFarlandauthor=B.+A.+Weirauthor=A.+E.+Sizemoreauthor=H.+Xuauthor=N.+V.+Dhariaauthor=P.+G.+Montgomeryauthor=G.+S.+Cowleyauthor=S.+Pantelauthor=A.+Goodaleauthor=Y.+Leeauthor=L.+D.+Aliauthor=G.+Jiangauthor=R.+Lubonjaauthor=W.+F.+Harringtonauthor=M.+Stricklandauthor=T.+Wuauthor=D.+C.+Hawesauthor=V.+A.+Zhivichauthor=M.+R.+Wyattauthor=Z.+Kalaniauthor=J.+J.+Changauthor=M.+Okamotoauthor=K.+Stegmaierauthor=T.+R.+Golubauthor=J.+S.+Boehmauthor=F.+Vazquezauthor=D.+E.+Rootauthor=W.+C.+Hahnauthor=A.+Tsherniak&title=Computational+correction+of+copy+number+effect+improves+specificity+of+CRISPR-Cas9+essentiality+screens+in+cancer+cells&doi=10.1038%2Fng.3984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells</span></div><div class="casAuthors">Meyers, Robin M.; Bryan, Jordan G.; McFarland, James M.; Weir, Barbara A.; Sizemore, Ann E.; Xu, Han; Dharia, Neekesh V.; Montgomery, Phillip G.; Cowley, Glenn S.; Pantel, Sasha; Goodale, Amy; Ali, Levi D.; Jiang, Guozhi; Lubonja, Rakela; Harrington, William F.; Strickland, Matthew; Wu, Ting; Hawes, Derek C.; Zhivich, Victor A.; Wyatt, Meghan R.; Kalani, Zohra; Chang, Jaime J.; Okamoto, Michael; Stegmaier, Kimberly; Golub, Todd R.; Boehm, Jesse S.; Vazquez, Francisca; Root, David E.; Hahn, William C.; Tsherniak, Aviad</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1779-1784</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The CRISPR-Cas9 system has revolutionized gene editing both at single genes and in multiplexed loss-of-function screens, thus enabling precise genome-scale identification of genes essential for proliferation and survival of cancer cells.  However, previous studies have reported that a gene-independent antiproliferative effect of Cas9-mediated DNA cleavage confounds such measurement of genetic dependency, thereby leading to false-pos. results in copy no.-amplified regions.  We developed CERES, a computational method to est. gene-dependency levels from CRISPR-Cas9 essentiality screens while accounting for the copy no.-specific effect.  In our efforts to define a cancer dependency map, we performed genome-scale CRISPR-Cas9 essentiality screens across 342 cancer cell lines and applied CERES to this data set.  We found that CERES decreased false-pos. results and estd. sgRNA activity for both this data set and previously published screens performed with different sgRNA libraries.  We further demonstrate the utility of this collection of screens, after CERES correction, for identifying cancer-type-specific vulnerabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8iODR9NtDbVg90H21EOLACvtfcHk0liz4RX80jEb_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslehtrjI&md5=539e8fb16958480ce2d430a7f8e6cddf</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fng.3984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3984%26sid%3Dliteratum%253Aachs%26aulast%3DMeyers%26aufirst%3DR.%2BM.%26aulast%3DBryan%26aufirst%3DJ.%2BG.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DSizemore%26aufirst%3DA.%2BE.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DDharia%26aufirst%3DN.%2BV.%26aulast%3DMontgomery%26aufirst%3DP.%2BG.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DPantel%26aufirst%3DS.%26aulast%3DGoodale%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DAli%26aufirst%3DL.%2BD.%26aulast%3DJiang%26aufirst%3DG.%26aulast%3DLubonja%26aufirst%3DR.%26aulast%3DHarrington%26aufirst%3DW.%2BF.%26aulast%3DStrickland%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DHawes%26aufirst%3DD.%2BC.%26aulast%3DZhivich%26aufirst%3DV.%2BA.%26aulast%3DWyatt%26aufirst%3DM.%2BR.%26aulast%3DKalani%26aufirst%3DZ.%26aulast%3DChang%26aufirst%3DJ.%2BJ.%26aulast%3DOkamoto%26aufirst%3DM.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DTsherniak%26aufirst%3DA.%26atitle%3DComputational%2520correction%2520of%2520copy%2520number%2520effect%2520improves%2520specificity%2520of%2520CRISPR-Cas9%2520essentiality%2520screens%2520in%2520cancer%2520cells%26jtitle%3DNat.%2520Genet.%26date%3D2017%26volume%3D49%26spage%3D1779%26epage%3D1784%26doi%3D10.1038%2Fng.3984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stadtmueller, H.</span>; <span class="NLM_string-name">Sapountzis, I.</span></span> <span> </span><span class="NLM_article-title">Preparation of Pyrimidinamine Derivatives for Treating Diseases Characterized by Excessive or Abnormal Cell Proliferation</span>. Int Pat Appl WO <span class="NLM_patent">2010/058032</span>, May 27, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=H.+Stadtmueller&author=I.+Sapountzis&title=Preparation+of+Pyrimidinamine+Derivatives+for+Treating+Diseases+Characterized+by+Excessive+or+Abnormal+Cell+Proliferation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DStadtmueller%26aufirst%3DH.%26atitle%3DPreparation%2520of%2520Pyrimidinamine%2520Derivatives%2520for%2520Treating%2520Diseases%2520Characterized%2520by%2520Excessive%2520or%2520Abnormal%2520Cell%2520Proliferation%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, scaling, space-group assignment and post-refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+scaling%2C+space-group+assignment+and+post-refinement&doi=10.1107%2Fs0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0lhSS5kUBNswkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520scaling%252C%2520space-group%2520assignment%2520and%2520post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2Fs0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span>; <span class="NLM_string-name">Winn, M.</span>; <span class="NLM_string-name">Ralph, A.</span></span> <span> </span><span class="NLM_article-title">Collaborative Computational Project, Number 4: Providing Programs for Protein Crystallography</span>.  <i>Methods in Enzymology</i>; <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_year">1997</span>; Vol.  <span class="NLM_volume">277</span>, pp  <span class="NLM_fpage">620</span>– <span class="NLM_lpage">633</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=620-633&author=E.+J.+Dodson&author=M.+Winn&author=A.+Ralph&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DCollaborative%2520Computational%2520Project%252C%2520Number%25204%253A%2520Providing%2520Programs%2520for%2520Protein%2520Crystallography%26btitle%3DMethods%2520in%2520Enzymology%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D277%26spage%3D620%26epage%3D633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhSS5kUBNswkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>Buster</i>, version 2.11.5; <span class="NLM_publisher-name">Global Phasing Ltd</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+S.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=Buster"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBuster%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2009&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 39 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Chaowei Ren, Ning Sun, Haixia Liu, Ying Kong, Renhong Sun, Xing Qiu, Jinju Chen, Yan Li, Jianshui Zhang, Yuedong Zhou, Hui Zhong, Qianqian Yin, Xiaoling Song, Xiaobao Yang, <span class="NLM_string-name hlFld-ContribAuthor">Biao Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Brigatinib Degrader SIAIS164018 with Destroying Metastasis-Related Oncoproteins and a Reshuffling Kinome Profile. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 9152-9165. <a href="https://doi.org/10.1021/acs.jmedchem.1c00373" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00373</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00373%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BBrigatinib%252BDegrader%252BSIAIS164018%252Bwith%252BDestroying%252BMetastasis-Related%252BOncoproteins%252Band%252Ba%252BReshuffling%252BKinome%252BProfile%26aulast%3DRen%26aufirst%3DChaowei%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01032021%26date%3D17062021%26volume%3D64%26issue%3D13%26spage%3D9152%26epage%3D9165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter S. Dragovich, Thomas H. Pillow, Robert A. Blake, Jack D. Sadowsky, Emel Adaligil, Pragya Adhikari, Sunil Bhakta, Nicole Blaquiere, Jinhua Chen, Josefa dela Cruz-Chuh, Karen E. Gascoigne, Steven J. Hartman, Mingtao He, Susan Kaufman, Tracy Kleinheinz, Katherine R. Kozak, Liang Liu, Liling Liu, Qi Liu, Ying Lu, Fanwei Meng, Melinda M. Mulvihill, Aimee O’Donohue, Rebecca K. Rowntree, Leanna R. Staben, Steven T. Staben, John Wai, Jian Wang, BinQing Wei, Catherine Wilson, Jianfeng Xin, Zijin Xu, Hui Yao, Donglu Zhang, Hongyan Zhang, Hao Zhou, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoyu Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 1: Exploration of Antibody Linker, Payload Loading, and Payload Molecular Properties. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2534-2575. <a href="https://doi.org/10.1021/acs.jmedchem.0c01845" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01845</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01845%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAntibody-Mediated%252BDelivery%252Bof%252BChimeric%252BBRD4%252BDegraders.%252BPart%252B1%25253A%252BExploration%252Bof%252BAntibody%252BLinker%25252C%252BPayload%252BLoading%25252C%252Band%252BPayload%252BMolecular%252BProperties%26aulast%3DDragovich%26aufirst%3DPeter%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D21102020%26date%3D17022021%26volume%3D64%26issue%3D5%26spage%3D2534%26epage%3D2575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter S. Dragovich, Thomas H. Pillow, Robert A. Blake, Jack D. Sadowsky, Emel Adaligil, Pragya Adhikari, Jinhua Chen, Nicholas Corr, Josefa dela Cruz-Chuh, Geoffrey Del Rosario, Aaron Fullerton, Steven J. Hartman, Fan Jiang, Susan Kaufman, Tracy Kleinheinz, Katherine R. Kozak, Liling Liu, Ying Lu, Melinda M. Mulvihill, Jeremy M. Murray, Aimee O’Donohue, Rebecca K. Rowntree, William S. Sawyer, Leanna R. Staben, John Wai, Jian Wang, BinQing Wei, Wentao Wei, Zijin Xu, Hui Yao, Shang-Fan Yu, Donglu Zhang, Hongyan Zhang, Shenhua Zhang, Yongxin Zhao, Hao Zhou, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoyu Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 2: Improvement of In Vitro Antiproliferation Activity and In Vivo Antitumor Efficacy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2576-2607. <a href="https://doi.org/10.1021/acs.jmedchem.0c01846" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01846</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01846%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAntibody-Mediated%252BDelivery%252Bof%252BChimeric%252BBRD4%252BDegraders.%252BPart%252B2%25253A%252BImprovement%252Bof%252BIn%252BVitro%252BAntiproliferation%252BActivity%252Band%252BIn%252BVivo%252BAntitumor%252BEfficacy%26aulast%3DDragovich%26aufirst%3DPeter%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D21102020%26date%3D17022021%26volume%3D64%26issue%3D5%26spage%3D2576%26epage%3D2607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Cyrille S. Kounde, <span class="NLM_string-name hlFld-ContribAuthor">Edward W. Tate</span>. </span><span class="cited-content_cbyCitation_article-title">Photoactive Bifunctional Degraders: Precision Tools To Regulate Protein Stability. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15483-15493. <a href="https://doi.org/10.1021/acs.jmedchem.0c01542" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01542</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01542%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPhotoactive%252BBifunctional%252BDegraders%25253A%252BPrecision%252BTools%252BTo%252BRegulate%252BProtein%252BStability%26aulast%3DKounde%26aufirst%3DCyrille%2BS.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03092020%26date%3D23112020%26volume%3D63%26issue%3D24%26spage%3D15483%26epage%3D15493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yang Lu, <span class="NLM_string-name hlFld-ContribAuthor">Haiying Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Progress in the Development of Small Molecular Inhibitors of Focal Adhesion Kinase (FAK). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14382-14403. <a href="https://doi.org/10.1021/acs.jmedchem.0c01248" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01248</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01248%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProgress%252Bin%252Bthe%252BDevelopment%252Bof%252BSmall%252BMolecular%252BInhibitors%252Bof%252BFocal%252BAdhesion%252BKinase%252B%252528FAK%252529%26aulast%3DLu%26aufirst%3DYang%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D17072020%26date%3D15102020%26volume%3D63%26issue%3D23%26spage%3D14382%26epage%3D14403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hongying Gao, Yue Wu, Yonghui Sun, Yiqing Yang, Guangbiao Zhou, <span class="NLM_string-name hlFld-ContribAuthor">Yu Rao</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Evaluation of Highly Potent FAK-Targeting PROTACs. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (10)
                                     , 1855-1862. <a href="https://doi.org/10.1021/acsmedchemlett.9b00372" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00372</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00372%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BEvaluation%252Bof%252BHighly%252BPotent%252BFAK-Targeting%252BPROTACs%26aulast%3DGao%26aufirst%3DHongying%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D09082019%26date%3D22102019%26date%3D29102019%26date%3D22102019%26volume%3D11%26issue%3D10%26spage%3D1855%26epage%3D1862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hongying Gao, Xiuyun Sun, <span class="NLM_string-name hlFld-ContribAuthor">Yu Rao</span>. </span><span class="cited-content_cbyCitation_article-title">PROTAC Technology: Opportunities and Challenges. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (3)
                                     , 237-240. <a href="https://doi.org/10.1021/acsmedchemlett.9b00597" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00597</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00597%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPROTAC%252BTechnology%25253A%252BOpportunities%252Band%252BChallenges%26aulast%3DGao%26aufirst%3DHongying%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D3%26spage%3D237%26epage%3D240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hannah Tovell, Andrea Testa, Houjiang Zhou, Natalia Shpiro, Claire Crafter, Alessio Ciulli, <span class="NLM_string-name hlFld-ContribAuthor">Dario R. Alessi</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Characterization of SGK3-PROTAC1, an Isoform Specific SGK3 Kinase PROTAC Degrader. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2019,</strong> <em>14 </em>
                                    (9)
                                     , 2024-2034. <a href="https://doi.org/10.1021/acschembio.9b00505" title="DOI URL">https://doi.org/10.1021/acschembio.9b00505</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00505%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDesign%252Band%252BCharacterization%252Bof%252BSGK3-PROTAC1%25252C%252Ban%252BIsoform%252BSpecific%252BSGK3%252BKinase%252BPROTAC%252BDegrader%26aulast%3DTovell%26aufirst%3DHannah%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D23062019%26date%3D28082019%26date%3D05092019%26date%3D28082019%26volume%3D14%26issue%3D9%26spage%3D2024%26epage%3D2034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Muhamad  Mustafa</span>, <span class="hlFld-ContribAuthor ">Amer Ali  Abd El-Hafeez</span>, <span class="hlFld-ContribAuthor ">Dalia A  Abdelhafeez</span>, <span class="hlFld-ContribAuthor ">Dalia  Abdelhamid</span>, <span class="hlFld-ContribAuthor ">Yaser A  Mostafa</span>, <span class="hlFld-ContribAuthor ">Pradipta  Ghosh</span>, <span class="hlFld-ContribAuthor ">Alaa M  Hayallah</span>, <span class="hlFld-ContribAuthor ">Gamal El-Din  A Abuo-Rahma</span>. </span><span class="cited-content_cbyCitation_article-title">FAK inhibitors as promising anticancer targets: present and future directions. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>7 </em><a href="https://doi.org/10.4155/fmc-2021-0015" title="DOI URL">https://doi.org/10.4155/fmc-2021-0015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2021-0015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2021-0015%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DFAK%252Binhibitors%252Bas%252Bpromising%252Banticancer%252Btargets%25253A%252Bpresent%252Band%252Bfuture%252Bdirections%26aulast%3DMustafa%26aufirst%3DMuhamad%26date%3D2021%26date%3D2021%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhangping  Xiao</span>, <span class="hlFld-ContribAuthor ">Shanshan  Song</span>, <span class="hlFld-ContribAuthor ">Deng  Chen</span>, <span class="hlFld-ContribAuthor ">Ronald  Merkerk</span>, <span class="hlFld-ContribAuthor ">Petra E.  Wouden</span>, <span class="hlFld-ContribAuthor ">Robbert H.  Cool</span>, <span class="hlFld-ContribAuthor ">Wim J.  Quax</span>, <span class="hlFld-ContribAuthor ">Gerrit J.  Poelarends</span>, <span class="hlFld-ContribAuthor ">Barbro N.  Melgert</span>, <span class="hlFld-ContribAuthor ">Frank J.  Dekker</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis Targeting Chimera (PROTAC) for Macrophage Migration Inhibitory Factor (MIF) Has Anti‐Proliferative Activity in Lung Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (32)
                                     , 17655-17662. <a href="https://doi.org/10.1002/ange.202101864" title="DOI URL">https://doi.org/10.1002/ange.202101864</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202101864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202101864%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DProteolysis%252BTargeting%252BChimera%252B%252528PROTAC%252529%252Bfor%252BMacrophage%252BMigration%252BInhibitory%252BFactor%252B%252528MIF%252529%252BHas%252BAnti%2525E2%252580%252590Proliferative%252BActivity%252Bin%252BLung%252BCancer%252BCells%26aulast%3DXiao%26aufirst%3DZhangping%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D32%26spage%3D17655%26epage%3D17662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guangyan  Du</span>, <span class="hlFld-ContribAuthor ">Jie  Jiang</span>, <span class="hlFld-ContribAuthor ">Qibiao  Wu</span>, <span class="hlFld-ContribAuthor ">Nathaniel J.  Henning</span>, <span class="hlFld-ContribAuthor ">Katherine A.  Donovan</span>, <span class="hlFld-ContribAuthor ">Hong  Yue</span>, <span class="hlFld-ContribAuthor ">Jianwei  Che</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Eric S.  Fischer</span>, <span class="hlFld-ContribAuthor ">Nabeel  Bardeesy</span>, <span class="hlFld-ContribAuthor ">Tinghu  Zhang</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Potent Degrader for Fibroblast Growth Factor Receptor 1/2. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (29)
                                     , 16041-16047. <a href="https://doi.org/10.1002/ange.202101328" title="DOI URL">https://doi.org/10.1002/ange.202101328</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202101328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202101328%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DDiscovery%252Bof%252Ba%252BPotent%252BDegrader%252Bfor%252BFibroblast%252BGrowth%252BFactor%252BReceptor%252B1%25252F2%26aulast%3DDu%26aufirst%3DGuangyan%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D29%26spage%3D16041%26epage%3D16047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guangyan  Du</span>, <span class="hlFld-ContribAuthor ">Jie  Jiang</span>, <span class="hlFld-ContribAuthor ">Qibiao  Wu</span>, <span class="hlFld-ContribAuthor ">Nathaniel J.  Henning</span>, <span class="hlFld-ContribAuthor ">Katherine A.  Donovan</span>, <span class="hlFld-ContribAuthor ">Hong  Yue</span>, <span class="hlFld-ContribAuthor ">Jianwei  Che</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Eric S.  Fischer</span>, <span class="hlFld-ContribAuthor ">Nabeel  Bardeesy</span>, <span class="hlFld-ContribAuthor ">Tinghu  Zhang</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Potent Degrader for Fibroblast Growth Factor Receptor 1/2. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (29)
                                     , 15905-15911. <a href="https://doi.org/10.1002/anie.202101328" title="DOI URL">https://doi.org/10.1002/anie.202101328</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202101328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202101328%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DDiscovery%252Bof%252Ba%252BPotent%252BDegrader%252Bfor%252BFibroblast%252BGrowth%252BFactor%252BReceptor%252B1%25252F2%26aulast%3DDu%26aufirst%3DGuangyan%26date%3D2021%26date%3D2021%26volume%3D60%26issue%3D29%26spage%3D15905%26epage%3D15911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao-Jing  Pang</span>, <span class="hlFld-ContribAuthor ">Xiu-Juan  Liu</span>, <span class="hlFld-ContribAuthor ">Yuan  Liu</span>, <span class="hlFld-ContribAuthor ">Wen-Bo  Liu</span>, <span class="hlFld-ContribAuthor ">Yin-Ru  Li</span>, <span class="hlFld-ContribAuthor ">Guang-Xi  Yu</span>, <span class="hlFld-ContribAuthor ">Xin-Yi  Tian</span>, <span class="hlFld-ContribAuthor ">Yan-Bing  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Song</span>, <span class="hlFld-ContribAuthor ">Cheng-Yun  Jin</span>, <span class="hlFld-ContribAuthor ">Sai-Yang  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Discovery Targeting Focal Adhesion Kinase (FAK) as a Promising Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (14)
                                     , 4250. <a href="https://doi.org/10.3390/molecules26144250" title="DOI URL">https://doi.org/10.3390/molecules26144250</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26144250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26144250%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DDrug%252BDiscovery%252BTargeting%252BFocal%252BAdhesion%252BKinase%252B%252528FAK%252529%252Bas%252Ba%252BPromising%252BCancer%252BTherapy%26aulast%3DPang%26aufirst%3DXiao-Jing%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D14%26spage%3D4250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hannah  Kiely-Collins</span>, <span class="hlFld-ContribAuthor ">Georg E.  Winter</span>, <span class="hlFld-ContribAuthor ">Gonçalo J.L.  Bernardes</span>. </span><span class="cited-content_cbyCitation_article-title">The role of reversible and irreversible covalent chemistry in targeted protein degradation. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 952-968. <a href="https://doi.org/10.1016/j.chembiol.2021.03.005" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.03.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.03.005%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DThe%252Brole%252Bof%252Breversible%252Band%252Birreversible%252Bcovalent%252Bchemistry%252Bin%252Btargeted%252Bprotein%252Bdegradation%26aulast%3DKiely-Collins%26aufirst%3DHannah%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D952%26epage%3D968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eclair Venturini  Filho</span>, <span class="hlFld-ContribAuthor ">Erick M.C.  Pinheiro</span>, <span class="hlFld-ContribAuthor ">Sergio  Pinheiro</span>, <span class="hlFld-ContribAuthor ">Sandro J.  Greco</span>. </span><span class="cited-content_cbyCitation_article-title">Aminopyrimidines: Recent synthetic procedures and anticancer activities. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2021,</strong> <em>92 </em>, 132256. <a href="https://doi.org/10.1016/j.tet.2021.132256" title="DOI URL">https://doi.org/10.1016/j.tet.2021.132256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2021.132256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2021.132256%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DAminopyrimidines%25253A%252BRecent%252Bsynthetic%252Bprocedures%252Band%252Banticancer%252Bactivities%26aulast%3DFilho%26aufirst%3DEclair%2BVenturini%26date%3D2021%26volume%3D92%26spage%3D132256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei  Yu</span>, <span class="hlFld-ContribAuthor ">Ming  Cai</span>, <span class="hlFld-ContribAuthor ">Liang  Shao</span>, <span class="hlFld-ContribAuthor ">Jihong  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Protein Kinases Degradation by PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.679120" title="DOI URL">https://doi.org/10.3389/fchem.2021.679120</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.679120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.679120%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DTargeting%252BProtein%252BKinases%252BDegradation%252Bby%252BPROTACs%26aulast%3DYu%26aufirst%3DFei%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhangping  Xiao</span>, <span class="hlFld-ContribAuthor ">Shanshan  Song</span>, <span class="hlFld-ContribAuthor ">Deng  Chen</span>, <span class="hlFld-ContribAuthor ">Ronald  Merkerk</span>, <span class="hlFld-ContribAuthor ">Petra E.  Wouden</span>, <span class="hlFld-ContribAuthor ">Robbert H.  Cool</span>, <span class="hlFld-ContribAuthor ">Wim J.  Quax</span>, <span class="hlFld-ContribAuthor ">Gerrit J.  Poelarends</span>, <span class="hlFld-ContribAuthor ">Barbro N.  Melgert</span>, <span class="hlFld-ContribAuthor ">Frank J.  Dekker</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis Targeting Chimera (PROTAC) for Macrophage Migration Inhibitory Factor (MIF) Has Anti‐Proliferative Activity in Lung Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>15 </em><a href="https://doi.org/10.1002/anie.202101864" title="DOI URL">https://doi.org/10.1002/anie.202101864</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202101864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202101864%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DProteolysis%252BTargeting%252BChimera%252B%252528PROTAC%252529%252Bfor%252BMacrophage%252BMigration%252BInhibitory%252BFactor%252B%252528MIF%252529%252BHas%252BAnti%2525E2%252580%252590Proliferative%252BActivity%252Bin%252BLung%252BCancer%252BCells%26aulast%3DXiao%26aufirst%3DZhangping%26date%3D2021%26date%3D2021%26volume%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ming  He</span>, <span class="hlFld-ContribAuthor ">Wenxing  Lv</span>, <span class="hlFld-ContribAuthor ">Yu  Rao</span>. </span><span class="cited-content_cbyCitation_article-title">Opportunities and Challenges of Small Molecule Induced Targeted Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fcell.2021.685106" title="DOI URL">https://doi.org/10.3389/fcell.2021.685106</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2021.685106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2021.685106%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DOpportunities%252Band%252BChallenges%252Bof%252BSmall%252BMolecule%252BInduced%252BTargeted%252BProtein%252BDegradation%26aulast%3DHe%26aufirst%3DMing%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John C.  Dawson</span>, <span class="hlFld-ContribAuthor ">Alan  Serrels</span>, <span class="hlFld-ContribAuthor ">Dwayne G.  Stupack</span>, <span class="hlFld-ContribAuthor ">David D.  Schlaepfer</span>, <span class="hlFld-ContribAuthor ">Margaret C.  Frame</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting FAK in anticancer combination therapies. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2021,</strong> <em>21 </em>
                                    (5)
                                     , 313-324. <a href="https://doi.org/10.1038/s41568-021-00340-6" title="DOI URL">https://doi.org/10.1038/s41568-021-00340-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41568-021-00340-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41568-021-00340-6%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DTargeting%252BFAK%252Bin%252Banticancer%252Bcombination%252Btherapies%26aulast%3DDawson%26aufirst%3DJohn%2BC.%26date%3D2021%26date%3D2021%26volume%3D21%26issue%3D5%26spage%3D313%26epage%3D324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Radosław P.  Nowak</span>, <span class="hlFld-ContribAuthor ">Lyn H.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Target Validation Using PROTACs: Applying the Four Pillars Framework. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 474-483. <a href="https://doi.org/10.1177/2472555220979584" title="DOI URL">https://doi.org/10.1177/2472555220979584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555220979584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555220979584%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DTarget%252BValidation%252BUsing%252BPROTACs%25253A%252BApplying%252Bthe%252BFour%252BPillars%252BFramework%26aulast%3DNowak%26aufirst%3DRados%25C5%2582aw%2BP.%26date%3D2021%26date%3D2020%26volume%3D26%26issue%3D4%26spage%3D474%26epage%3D483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">William  Farnaby</span>, <span class="hlFld-ContribAuthor ">Manfred  Koegl</span>, <span class="hlFld-ContribAuthor ">Darryl B.  McConnell</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Transforming targeted cancer therapy with PROTACs: A forward-looking perspective. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Pharmacology</span><span> <strong>2021,</strong> <em>57 </em>, 175-183. <a href="https://doi.org/10.1016/j.coph.2021.02.009" title="DOI URL">https://doi.org/10.1016/j.coph.2021.02.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coph.2021.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coph.2021.02.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Pharmacology%26atitle%3DTransforming%252Btargeted%252Bcancer%252Btherapy%252Bwith%252BPROTACs%25253A%252BA%252Bforward-looking%252Bperspective%26aulast%3DFarnaby%26aufirst%3DWilliam%26date%3D2021%26volume%3D57%26spage%3D175%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yvonne A.  Nagel</span>, <span class="hlFld-ContribAuthor ">Adrian  Britschgi</span>, <span class="hlFld-ContribAuthor ">Antonio  Ricci</span>. </span><span class="cited-content_cbyCitation_article-title">From Degraders to Molecular Glues: New Ways of Breaking Down Disease‐Associated Proteins. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 47-85. <a href="https://doi.org/10.1002/9783527826872.ch2" title="DOI URL">https://doi.org/10.1002/9783527826872.ch2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527826872.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527826872.ch2%26sid%3Dliteratum%253Aachs%26atitle%3DFrom%252BDegraders%252Bto%252BMolecular%252BGlues%25253A%252BNew%252BWays%252Bof%252BBreaking%252BDown%252BDisease%2525E2%252580%252590Associated%252BProteins%26aulast%3DNagel%26aufirst%3DYvonne%2BA.%26date%3D2021%26date%3D2021%26spage%3D47%26epage%3D85%26pub%3DWiley%26atitle%3DSuccessful%252BDrug%252BDiscovery%26aulast%3DFischer%26aufirst%3DJ%25C3%25A1nos%26date%3D2021%26date%3D2021%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Liyan cheng</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera (PROTAC) in drug discovery paradigm: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112981. <a href="https://doi.org/10.1016/j.ejmech.2020.112981" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112981</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112981%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%252Bin%252Bdrug%252Bdiscovery%252Bparadigm%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26volume%3D210%26spage%3D112981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M.  Maneiro</span>, <span class="hlFld-ContribAuthor ">E.  De Vita</span>, <span class="hlFld-ContribAuthor ">D.  Conole</span>, <span class="hlFld-ContribAuthor ">C.S.  Kounde</span>, <span class="hlFld-ContribAuthor ">Q.  Zhang</span>, <span class="hlFld-ContribAuthor ">E.W.  Tate</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs, molecular glues and bifunctionals from bench to bedside: Unlocking the clinical potential of catalytic drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/bs.pmch.2021.01.002" title="DOI URL">https://doi.org/10.1016/bs.pmch.2021.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.pmch.2021.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.pmch.2021.01.002%26sid%3Dliteratum%253Aachs%26atitle%3DPROTACs%25252C%252Bmolecular%252Bglues%252Band%252Bbifunctionals%252Bfrom%252Bbench%252Bto%252Bbedside%25253A%252BUnlocking%252Bthe%252Bclinical%252Bpotential%252Bof%252Bcatalytic%252Bdrugs%26aulast%3DManeiro%26aufirst%3DM.%26date%3D2021%26pub%3DElsevier%26date%3D2021%26volume%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuqing  Liang</span>, <span class="hlFld-ContribAuthor ">Kutty Selva  Nandakumar</span>, <span class="hlFld-ContribAuthor ">Kui  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design and pharmaceutical applications of proteolysis-targeting chimeric molecules. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2020,</strong> <em>182 </em>, 114211. <a href="https://doi.org/10.1016/j.bcp.2020.114211" title="DOI URL">https://doi.org/10.1016/j.bcp.2020.114211</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2020.114211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2020.114211%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DDesign%252Band%252Bpharmaceutical%252Bapplications%252Bof%252Bproteolysis-targeting%252Bchimeric%252Bmolecules%26aulast%3DLiang%26aufirst%3DYuqing%26date%3D2020%26volume%3D182%26spage%3D114211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alberto  Ocaña</span>, <span class="hlFld-ContribAuthor ">Atanasio  Pandiella</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimeras (PROTACs) in cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Experimental & Clinical Cancer Research</span><span> <strong>2020,</strong> <em>39 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13046-020-01672-1" title="DOI URL">https://doi.org/10.1186/s13046-020-01672-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13046-020-01672-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13046-020-01672-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Experimental%2520%2526%2520Clinical%2520Cancer%2520Research%26atitle%3DProteolysis%252Btargeting%252Bchimeras%252B%252528PROTACs%252529%252Bin%252Bcancer%252Btherapy%26aulast%3DOca%25C3%25B1a%26aufirst%3DAlberto%26date%3D2020%26date%3D2020%26volume%3D39%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony  Feral</span>, <span class="hlFld-ContribAuthor ">Guillaume  Laconde</span>, <span class="hlFld-ContribAuthor ">Muriel  Amblard</span>, <span class="hlFld-ContribAuthor ">Nicolas  Masurier</span>. </span><span class="cited-content_cbyCitation_article-title">PROteolysis TArgetting Chimeras (PROTACs) Strategy Applied to Kinases: Recent Advances. </span><span class="cited-content_cbyCitation_journal-name">Advanced Therapeutics</span><span> <strong>2020,</strong> <em>3 </em>
                                    (11)
                                     , 2000148. <a href="https://doi.org/10.1002/adtp.202000148" title="DOI URL">https://doi.org/10.1002/adtp.202000148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adtp.202000148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadtp.202000148%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Therapeutics%26atitle%3DPROteolysis%252BTArgetting%252BChimeras%252B%252528PROTACs%252529%252BStrategy%252BApplied%252Bto%252BKinases%25253A%252BRecent%252BAdvances%26aulast%3DFeral%26aufirst%3DAnthony%26date%3D2020%26date%3D2020%26volume%3D3%26issue%3D11%26spage%3D2000148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuseppe  Ermondi</span>, <span class="hlFld-ContribAuthor ">Maura  Vallaro</span>, <span class="hlFld-ContribAuthor ">Giulia  Caron</span>. </span><span class="cited-content_cbyCitation_article-title">Degraders early developability assessment: face-to-face with molecular properties. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (9)
                                     , 1585-1591. <a href="https://doi.org/10.1016/j.drudis.2020.06.015" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.06.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.06.015%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DDegraders%252Bearly%252Bdevelopability%252Bassessment%25253A%252Bface-to-face%252Bwith%252Bmolecular%252Bproperties%26aulast%3DErmondi%26aufirst%3DGiuseppe%26date%3D2020%26volume%3D25%26issue%3D9%26spage%3D1585%26epage%3D1591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dhanusha A.  Nalawansha</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs: An Emerging Therapeutic Modality in Precision Medicine. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (8)
                                     , 998-1014. <a href="https://doi.org/10.1016/j.chembiol.2020.07.020" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.07.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.07.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.07.020%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DPROTACs%25253A%252BAn%252BEmerging%252BTherapeutic%252BModality%252Bin%252BPrecision%252BMedicine%26aulast%3DNalawansha%26aufirst%3DDhanusha%2BA.%26date%3D2020%26volume%3D27%26issue%3D8%26spage%3D998%26epage%3D1014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Li  Zhang</span>, <span class="hlFld-ContribAuthor ">Fuwei  Liu</span>, <span class="hlFld-ContribAuthor ">Yajie  Fu</span>, <span class="hlFld-ContribAuthor ">Xiaoyun  Chen</span>, <span class="hlFld-ContribAuthor ">Dongdong  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">MiR-520d-5p functions as a tumor-suppressor gene in cervical cancer through targeting PTK2. </span><span class="cited-content_cbyCitation_journal-name">Life Sciences</span><span> <strong>2020,</strong> <em>254 </em>, 117558. <a href="https://doi.org/10.1016/j.lfs.2020.117558" title="DOI URL">https://doi.org/10.1016/j.lfs.2020.117558</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lfs.2020.117558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lfs.2020.117558%26sid%3Dliteratum%253Aachs%26jtitle%3DLife%2520Sciences%26atitle%3DMiR-520d-5p%252Bfunctions%252Bas%252Ba%252Btumor-suppressor%252Bgene%252Bin%252Bcervical%252Bcancer%252Bthrough%252Btargeting%252BPTK2%26aulast%3DZhang%26aufirst%3DLi%26date%3D2020%26volume%3D254%26spage%3D117558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">George M.  Burslem</span>, <span class="hlFld-ContribAuthor ">Daniel P.  Bondeson</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">Scaffold hopping enables direct access to more potent PROTACs with
              in vivo
              activity. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2020,</strong> <em>56 </em>
                                    (50)
                                     , 6890-6892. <a href="https://doi.org/10.1039/D0CC02201B" title="DOI URL">https://doi.org/10.1039/D0CC02201B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CC02201B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CC02201B%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DScaffold%252Bhopping%252Benables%252Bdirect%252Baccess%252Bto%252Bmore%252Bpotent%252BPROTACs%252Bwith%252Bin%252Bvivo%252Bactivity%26aulast%3DBurslem%26aufirst%3DGeorge%2BM.%26date%3D2020%26date%3D2020%26volume%3D56%26issue%3D50%26spage%3D6890%26epage%3D6892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carina  Cantrill</span>, <span class="hlFld-ContribAuthor ">Prasoon  Chaturvedi</span>, <span class="hlFld-ContribAuthor ">Caroline  Rynn</span>, <span class="hlFld-ContribAuthor ">Jeannine  Petrig Schaffland</span>, <span class="hlFld-ContribAuthor ">Isabelle  Walter</span>, <span class="hlFld-ContribAuthor ">Matthias B.  Wittwer</span>. </span><span class="cited-content_cbyCitation_article-title">Fundamental aspects of DMPK optimization of targeted protein degraders. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (6)
                                     , 969-982. <a href="https://doi.org/10.1016/j.drudis.2020.03.012" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.03.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.03.012%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DFundamental%252Baspects%252Bof%252BDMPK%252Boptimization%252Bof%252Btargeted%252Bprotein%252Bdegraders%26aulast%3DCantrill%26aufirst%3DCarina%26date%3D2020%26volume%3D25%26issue%3D6%26spage%3D969%26epage%3D982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ivana  Gasic</span>, <span class="hlFld-ContribAuthor ">Brian J.  Groendyke</span>, <span class="hlFld-ContribAuthor ">Radosław P.  Nowak</span>, <span class="hlFld-ContribAuthor ">J. Christine  Yuan</span>, <span class="hlFld-ContribAuthor ">Joann  Kalabathula</span>, <span class="hlFld-ContribAuthor ">Eric S.  Fischer</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>, <span class="hlFld-ContribAuthor ">Timothy J.  Mitchison</span>. </span><span class="cited-content_cbyCitation_article-title">Tubulin Resists Degradation by Cereblon-Recruiting PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2020,</strong> <em>9 </em>
                                    (5)
                                     , 1083. <a href="https://doi.org/10.3390/cells9051083" title="DOI URL">https://doi.org/10.3390/cells9051083</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells9051083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells9051083%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DTubulin%252BResists%252BDegradation%252Bby%252BCereblon-Recruiting%252BPROTACs%26aulast%3DGasic%26aufirst%3DIvana%26date%3D2020%26date%3D2020%26volume%3D9%26issue%3D5%26spage%3D1083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">George M.  Burslem</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery. </span><span class="cited-content_cbyCitation_journal-name">Cell</span><span> <strong>2020,</strong> <em>181 </em>
                                    (1)
                                     , 102-114. <a href="https://doi.org/10.1016/j.cell.2019.11.031" title="DOI URL">https://doi.org/10.1016/j.cell.2019.11.031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%26atitle%3DProteolysis-Targeting%252BChimeras%252Bas%252BTherapeutics%252Band%252BTools%252Bfor%252BBiological%252BDiscovery%26aulast%3DBurslem%26aufirst%3DGeorge%2BM.%26date%3D2020%26volume%3D181%26issue%3D1%26spage%3D102%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rati  Verma</span>, <span class="hlFld-ContribAuthor ">Dane  Mohl</span>, <span class="hlFld-ContribAuthor ">Raymond J.  Deshaies</span>. </span><span class="cited-content_cbyCitation_article-title">Harnessing the Power of Proteolysis for Targeted Protein Inactivation. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cell</span><span> <strong>2020,</strong> <em>77 </em>
                                    (3)
                                     , 446-460. <a href="https://doi.org/10.1016/j.molcel.2020.01.010" title="DOI URL">https://doi.org/10.1016/j.molcel.2020.01.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molcel.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molcel.2020.01.010%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cell%26atitle%3DHarnessing%252Bthe%252BPower%252Bof%252BProteolysis%252Bfor%252BTargeted%252BProtein%252BInactivation%26aulast%3DVerma%26aufirst%3DRati%26date%3D2020%26volume%3D77%26issue%3D3%26spage%3D446%26epage%3D460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiuyun  Sun</span>, <span class="hlFld-ContribAuthor ">Hongying  Gao</span>, <span class="hlFld-ContribAuthor ">Yiqing  Yang</span>, <span class="hlFld-ContribAuthor ">Ming  He</span>, <span class="hlFld-ContribAuthor ">Yue  Wu</span>, <span class="hlFld-ContribAuthor ">Yugang  Song</span>, <span class="hlFld-ContribAuthor ">Yan  Tong</span>, <span class="hlFld-ContribAuthor ">Yu  Rao</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs: great opportunities for academia and industry. </span><span class="cited-content_cbyCitation_journal-name">Signal Transduction and Targeted Therapy</span><span> <strong>2019,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41392-019-0101-6" title="DOI URL">https://doi.org/10.1038/s41392-019-0101-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26jtitle%3DSignal%2520Transduction%2520and%2520Targeted%2520Therapy%26atitle%3DPROTACs%25253A%252Bgreat%252Bopportunities%252Bfor%252Bacademia%252Band%252Bindustry%26aulast%3DSun%26aufirst%3DXiuyun%26date%3D2019%26date%3D2019%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Matthew F.  Calabrese</span>, <span class="hlFld-ContribAuthor ">Alex N.  Bullock</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted protein degradation: expanding the toolbox. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2019,</strong> <em>18 </em>
                                    (12)
                                     , 949-963. <a href="https://doi.org/10.1038/s41573-019-0047-y" title="DOI URL">https://doi.org/10.1038/s41573-019-0047-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DTargeted%252Bprotein%252Bdegradation%25253A%252Bexpanding%252Bthe%252Btoolbox%26aulast%3DSchapira%26aufirst%3DMatthieu%26date%3D2019%26date%3D2019%26volume%3D18%26issue%3D12%26spage%3D949%26epage%3D963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hannah J.  Maple</span>, <span class="hlFld-ContribAuthor ">Nat  Clayden</span>, <span class="hlFld-ContribAuthor ">Anne  Baron</span>, <span class="hlFld-ContribAuthor ">Callum  Stacey</span>, <span class="hlFld-ContribAuthor ">Robert  Felix</span>. </span><span class="cited-content_cbyCitation_article-title">Developing degraders: principles and perspectives on design and chemical space. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (10)
                                     , 1755-1764. <a href="https://doi.org/10.1039/C9MD00272C" title="DOI URL">https://doi.org/10.1039/C9MD00272C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00272C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00272C%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DDeveloping%252Bdegraders%25253A%252Bprinciples%252Band%252Bperspectives%252Bon%252Bdesign%252Band%252Bchemical%252Bspace%26aulast%3DMaple%26aufirst%3DHannah%2BJ.%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D10%26spage%3D1755%26epage%3D1764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Valeur</span>, <span class="hlFld-ContribAuthor ">Frank  Narjes</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>, <span class="hlFld-ContribAuthor ">Alleyn T.  Plowright</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging modes-of-action in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 1550-1568. <a href="https://doi.org/10.1039/C9MD00263D" title="DOI URL">https://doi.org/10.1039/C9MD00263D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00263D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00263D%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DEmerging%252Bmodes-of-action%252Bin%252Bdrug%252Bdiscovery%26aulast%3DValeur%26aufirst%3DEric%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D9%26spage%3D1550%26epage%3D1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of BI-4464 bound to PTK2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Z">6I8Z</a>). Hydrogen bonds to cysteine 502 and aspartic acid 564 are depicted in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Chemical Structures of BI-4464 Ligand <b>1</b>, VHL Ligand <b>2</b>,<a onclick="showRef(event, 'ref37 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref37 ref41 ref42">(37,41,42)</a> and CRBN Ligand <b>3</b><a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the PTK2 Degraders<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) Cs<sub>2</sub>CO<sub>3</sub>, MgSO<sub>4</sub>, dioxane, 80 °C, 16 h, 78%; (b) HATU, DIPEA, DMF, rt, 2 h, 75%; (c) T3P, pyridine, DMF, 80 °C, 3 h, 76%; (d) H<sub>2</sub> (6 bar), Pd/C (10%), MeOH, rt, 3 h, 90%; (e) <b>1a</b>, HATU, DIPEA, DMF, rt, 18 h, 10%; (f) HATU, DIPEA, DMF, rt, 1 h, 63%; (g) H<sub>2</sub> (5 bar), Pd/C (10%), MeOH, rt, 2.5 h, 82%; (h) <b>1a</b>, HATU, DIPEA, DMF, rt, 24 h, 34%; (i) HATU, DIPEA, DMF, rt, 2 h, 47%; (j) H<sub>2</sub> (6 bar), Pd/C (10%), MeOH, rt, 3 h, 98%; and (k) <b>1a</b>, HATU, DIPEA, DMF, rt, 16 h, 69%.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Characterization of BI-0319 and BI-3663 in A549 cells. (A) A549 cells were treated with the indicated concentrations of BI-0319 in the presence or absence of the NEDD8 inhibitor, MLN4924 (MLN, 3 μM) or an inactive stereoisomer (BI-4206) for 18 h. PTK2 levels were determined by protein capillary electrophoresis and normalized to GAPDH. The values are stated as percentages of dimethyl sulfoxide (DMSO) controls (POC). (B) A549 cells were treated with the indicated concentrations of BI-3663 in the presence or absence of the NEDD8 inhibitor, MLN4924 (3 μM), or with the PTK2 inhibitor, BI-4664, for 18 h. PTK2 levels were measured and are stated as in (A).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Kinase selectivity panels (Thermo Fischer SelectScreen Kinase Profiling Services) of the PTK2 ligand, BI-4464 and VHL PROTAC, BI-0319. The bars indicate percent inhibition at 1 μM of either compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Total proteome analysis of A549 cells treated with 3 μM of BI-0319 (A) or BI-3663 (B) for 18 h and compared to DMSO controls. The samples were run in biological triplicates and analyzed by mass spectrometry. Volcano plot displays log<sub>2</sub> of fold-change in abundance versus −log<sub>10</sub> of adjusted <i>p</i> value (<i>N</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cumulative analysis of half-maximal effective concentrations [DC<sub>50</sub>, (A)] and maximal degree of PTK2 degradation [<i>D</i><sub>max</sub> [%], (B)] achieved by BI-0319 or BI-3663. pDC<sub>50</sub> and <i>D</i><sub>max</sub> at 18 h were determined as indicated in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf" class="ext-link">Figures S2 and S3</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/medium/jm-2018-01826c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Analysis of PTK2 degradation in Hep3B2.1-7 cells. (A) Cells were treated with the indicated concentrations of BI-0319 for 18 h in the presence (red triangles) or absence (blue circles) of 10 μM CsnA. (B) Cells were treated for 18 h with the indicated concentrations of BI-3663 in the presence (red triangles) or absence (green circles) of 10 μM CsnA. (C) Cells were treated with 10 μM CsnA for 18 h. (D) Time course analysis of PTK2 levels in cells treated with 3 μM BI-0319. (E) Time course analysis of PTK2 levels in cells treated with 3 μM BI-3663. The values state PTK2 protein levels normalized to GAPDH relative to DMSO (or CsnA only) controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01826/20190308/images/large/jm-2018-01826c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01826&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i42">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42428" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42428" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 52 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Golubovskaya, V.</span></span> <span> </span><span class="NLM_article-title">Targeting FAK in human cancer: from finding to first clinical trials</span>. <i>Front. Biosci., Landmark Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">706</span>, <span class="refDoi"> DOI: 10.2741/4236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.2741%2F4236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=24389213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtl2ktb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=687-706&author=V.+Golubovskaya&title=Targeting+FAK+in+human+cancer%3A+from+finding+to+first+clinical+trials&doi=10.2741%2F4236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting FAK in human cancer: from finding to first clinical trials</span></div><div class="casAuthors">Golubovskaya, Vita M.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Landmark Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">687-706, 20</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  It is twenty years since Focal Adhesion Kinase (FAK) was found to be overexpressed in many types of human cancer.  FAK plays an important role in adhesion, spreading, motility, invasion, metastasis, survival, angiogenesis, and recently has been found to play an important role as well in epithelial to mesenchymal transition (EMT), cancer stem cells and tumor microenvironment.  FAK has kinase-dependent and kinase independent scaffolding, cytoplasmic and nuclear functions.  Several years ago FAK was proposed as a potential therapeutic target; the first clin. trials were just reported, and they supported further studies of FAK as a promising therapeutic target.  This review discusses the main functions of FAK in cancer, and specifically focuses on recent novel findings on the role of FAK in cancer stem cells, microenvironment, epithelial-to-mesenchymal transition, invasion, metastasis, and also highlight new approaches of targeting FAK and critically discuss challenges that lie ahead for its targeted therapeutics.  The review provides a summary of translational approaches of FAK-targeted and combination therapies and outline perspectives and future directions of FAK research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5EGzIEc6UMbVg90H21EOLACvtfcHk0ljGfjsZyjcZng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtl2ktb%252FI&md5=977f2530b2d4ed073e84ec64e498c3a3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2741%2F4236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F4236%26sid%3Dliteratum%253Aachs%26aulast%3DGolubovskaya%26aufirst%3DV.%26atitle%3DTargeting%2520FAK%2520in%2520human%2520cancer%253A%2520from%2520finding%2520to%2520first%2520clinical%2520trials%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2014%26volume%3D19%26spage%3D687%26epage%3D706%26doi%3D10.2741%2F4236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sulzmaier, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlaepfer, D. D.</span></span> <span> </span><span class="NLM_article-title">FAK in cancer: mechanistic findings and clinical applications</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1038/nrc3792</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnrc3792" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=25098269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12nu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=598-610&author=F.+J.+Sulzmaierauthor=C.+Jeanauthor=D.+D.+Schlaepfer&title=FAK+in+cancer%3A+mechanistic+findings+and+clinical+applications&doi=10.1038%2Fnrc3792"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">FAK in cancer: mechanistic findings and clinical applications</span></div><div class="casAuthors">Sulzmaier, Florian J.; Jean, Christine; Schlaepfer, David D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">598-610</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Focal adhesion kinase (FAK) is a cytoplasmic protein tyrosine kinase that is overexpressed and activated in several advanced-stage solid cancers.  FAK promotes tumor progression and metastasis through effects on cancer cells, as well as stromal cells of the tumor microenvironment.  The kinase-dependent and kinase-independent functions of FAK control cell movement, invasion, survival, gene expression and cancer stem cell self-renewal.  Small mol. FAK inhibitors decrease tumor growth and metastasis in several preclin. models and have initial clin. activity in patients with limited adverse events.  In this Review, we discuss FAK signalling effects on both tumor and stromal cell biol. that provide rationale and support for future therapeutic opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmKc7m7dsY67Vg90H21EOLACvtfcHk0lgHPrPwPAefVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12nu7vJ&md5=f498bcf7af890d2f92eda68e42f979d1</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc3792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3792%26sid%3Dliteratum%253Aachs%26aulast%3DSulzmaier%26aufirst%3DF.%2BJ.%26aulast%3DJean%26aufirst%3DC.%26aulast%3DSchlaepfer%26aufirst%3DD.%2BD.%26atitle%3DFAK%2520in%2520cancer%253A%2520mechanistic%2520findings%2520and%2520clinical%2520applications%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D598%26epage%3D610%26doi%3D10.1038%2Fnrc3792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lark, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livasy, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caskey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cance, W. G.</span></span> <span> </span><span class="NLM_article-title">Overexpression of focal adhesion kinase in primary colorectal carcinomas and colorectal liver metastases: immunohistochemistry and real-time PCR analyses</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">222</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=12538472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1OksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2003&pages=215-222&author=A.+L.+Larkauthor=C.+A.+Livasyauthor=B.+Calvoauthor=L.+Caskeyauthor=D.+T.+Mooreauthor=X.+Yangauthor=W.+G.+Cance&title=Overexpression+of+focal+adhesion+kinase+in+primary+colorectal+carcinomas+and+colorectal+liver+metastases%3A+immunohistochemistry+and+real-time+PCR+analyses"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of Focal Adhesion Kinase in Primary Colorectal Carcinomas and Colorectal Liver Metastases: Immunohistochemistry and Real-Time PCR Analyses</span></div><div class="casAuthors">Lark, Amy L.; Livasy, Chad A.; Calvo, Benjamin; Caskey, Laura; Moore, Dominic T.; Yang, XiHui; Cance, William G.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">215-222</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Focal adhesion kinase (FAK), a protein tyrosine kinase that functions in signaling events between cells and their extracellular matrix, is overexpressed in a variety of human solid tumors.  To det. whether FAK expression is up-regulated in colorectal cancer, we analyzed FAK mRNA and protein levels in primary colorectal tumors and colorectal liver metastases.  Exptl. Design: p125FAK expression in formalin-fixed paraffin-embedded (FFPE) tissue was studied using immunohistochem. assays on 24 matched primary colorectal carcinomas and colorectal liver metastases as well as 18 different colorectal liver metastases using monoclonal anti-FAK 4.47.  FAK mRNA expression was quantitated by real-time PCR on 39 matched normal colorectal mucosa and primary colorectal carcinomas as well as on 17 sep. liver metastases.  RESULTS: Elevated levels of p125FAK expression were demonstrated in both primary colorectal tumors and colorectal liver metastases compared with normal colorectal mucosa.  Immunohistochem. expts. demonstrated equiv. FAK expression in matched samples of colorectal primary tumors and liver metastases.  Using real-time PCR in 39 matched samples, FAK mRNA copy no. was significantly higher in primary colorectal tumors compared with normal colorectal mucosa.  FAK expression was analyzed by both real-time PCR and immunohistochem. in a sep. set of colorectal liver metastases.  Immunohistochem. demonstrated high levels of FAK expression in 89% of samples.  Furthermore, FAK mRNA copies in these unmatched liver metastases were significantly higher than the primary tumor FAK mRNA copies.  CONCLUSION: These expts. have shown that both primary colorectal cancers and colorectal liver metastases express high levels of FAK mRNA and p125FAK protein.  Furthermore, the majority of colorectal liver metastases demonstrated robust FAK expression equiv. to or greater than that in the primary colorectal tumor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1OpdCDZYyq7Vg90H21EOLACvtfcHk0lgHPrPwPAefVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt1OksA%253D%253D&md5=39a8af2050e11b9e73959f9700daa3fc</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLark%26aufirst%3DA.%2BL.%26aulast%3DLivasy%26aufirst%3DC.%2BA.%26aulast%3DCalvo%26aufirst%3DB.%26aulast%3DCaskey%26aufirst%3DL.%26aulast%3DMoore%26aufirst%3DD.%2BT.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DCance%26aufirst%3DW.%2BG.%26atitle%3DOverexpression%2520of%2520focal%2520adhesion%2520kinase%2520in%2520primary%2520colorectal%2520carcinomas%2520and%2520colorectal%2520liver%2520metastases%253A%2520immunohistochemistry%2520and%2520real-time%2520PCR%2520analyses%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2003%26volume%3D9%26spage%3D215%26epage%3D222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Judson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cance, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Le, L.</span></span> <span> </span><span class="NLM_article-title">Overexpression of focal adhesion kinase, a protein tyrosine kinase, in ovarian carcinoma</span>. <i>Cancer</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">1551</span>– <span class="NLM_lpage">1556</span>, <span class="refDoi"> DOI: 10.1002/(sici)1097-0142(19991015)86:6<1551::aid-cncr23>3.0.co;2-p</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1002%2F%28sici%291097-0142%2819991015%2986%3A6%3C1551%3A%3Aaid-cncr23%3E3.0.co%3B2-p" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10526262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADyaK1MXntlSkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1999&pages=1551-1556&author=P.+L.+Judsonauthor=X.+Heauthor=W.+G.+Canceauthor=L.+Van+Le&title=Overexpression+of+focal+adhesion+kinase%2C+a+protein+tyrosine+kinase%2C+in+ovarian+carcinoma&doi=10.1002%2F%28sici%291097-0142%2819991015%2986%3A6%3C1551%3A%3Aaid-cncr23%3E3.0.co%3B2-p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of focal adhesion kinase, a protein tyrosine kinase, in ovarian carcinoma</span></div><div class="casAuthors">Judson, Patricia L.; He, Xiaping; Cance, William G.; Van Le, Linda</div><div class="citationInfo"><span class="NLM_cas:title">Cancer (New York)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1551-1556</span>CODEN:
                <span class="NLM_cas:coden">CANCAR</span>;
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Focal adhesion kinase (FAK) is a tyrosine kinase that is important to such key functions such as cell adhesion, motility, and invasion.  A MEDLINE search of the years 1980-1998 found no previous reports of FAK expression in human ovarian carcinoma.  The authors performed expts. to det. whether FAK expression is elevated in this disease.  Ten normal human ovarian tissue samples and 26 cancer samples from patients with Stage I-IV ovarian carcinoma were obtained.  Two ovarian carcinoma cell lines were also analyzed.  FAK expression was detd. by Western blot anal. with the V39 anti-human FAK polyclonal antibody.  The level of FAK protein expression was detd. using densitometric scanning of the 125-kDa band on autoradiographs of Western immunoblots.  Serous cancers expressed 4-fold-increased values of FAK relative to normal ovarian tissue, and nonserous adenocarcinomas expressed 3-4-fold-increased values of FAK.  Ovarian carcinoma cell lines also expressed increased values of FAK.  With a cutoff of 40, an elevated FAK level was assocd. with a sensitivity of 93% and specificity of 100%.  There was no significant difference in FAK expression with regard to grade or stage of tumor.  Thus, FAK is overexpressed in ovarian carcinoma, implying that FAK may play an important role in ovarian carcinogenesis.  FAK expression may be useful as a screening tool to identify newly developed disease or as a tumor marker in confirmed cases of epithelial ovarian carcinoma.  FAK may also serve as a potential target for therapeutic disruption of ovarian carcinoma progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk4vVqxQX-V7Vg90H21EOLACvtfcHk0lgHPrPwPAefVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntlSkurg%253D&md5=b68496778ced297f8b4d6e80fa8d0629</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2F%28sici%291097-0142%2819991015%2986%3A6%3C1551%3A%3Aaid-cncr23%3E3.0.co%3B2-p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528sici%25291097-0142%252819991015%252986%253A6%253C1551%253A%253Aaid-cncr23%253E3.0.co%253B2-p%26sid%3Dliteratum%253Aachs%26aulast%3DJudson%26aufirst%3DP.%2BL.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DCance%26aufirst%3DW.%2BG.%26aulast%3DVan%2BLe%26aufirst%3DL.%26atitle%3DOverexpression%2520of%2520focal%2520adhesion%2520kinase%252C%2520a%2520protein%2520tyrosine%2520kinase%252C%2520in%2520ovarian%2520carcinoma%26jtitle%3DCancer%26date%3D1999%26volume%3D86%26spage%3D1551%26epage%3D1556%26doi%3D10.1002%2F%28sici%291097-0142%2819991015%2986%3A6%3C1551%3A%3Aaid-cncr23%3E3.0.co%3B2-p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyazaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwano, H.</span></span> <span> </span><span class="NLM_article-title">FAK overexpression is correlated with tumour invasiveness and lymph node metastasis in oesophageal squamous cell carcinoma</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6601050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fsj.bjc.6601050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=12838315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVWmsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2003&pages=140-145&author=T.+Miyazakiauthor=H.+Katoauthor=M.+Nakajimaauthor=M.+Sohdaauthor=Y.+Fukaiauthor=N.+Masudaauthor=R.+Mandaauthor=M.+Fukuchiauthor=K.+Tsukadaauthor=H.+Kuwano&title=FAK+overexpression+is+correlated+with+tumour+invasiveness+and+lymph+node+metastasis+in+oesophageal+squamous+cell+carcinoma&doi=10.1038%2Fsj.bjc.6601050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">FAK overexpression is correlated with tumor invasiveness and lymph node metastasis in esophageal squamous cell carcinoma</span></div><div class="casAuthors">Miyazaki, T.; Kato, H.; Nakajima, M.; Sohda, M.; Fukai, Y.; Masuda, N.; Manda, R.; Fukuchi, M.; Tsukada, K.; Kuwano, H.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">140-145</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Focal adhesion kinase (p125FAK; FAK') is a tyrosine kinase that is localized to cellular focal adhesions and is assocd. with a no. of other proteins, such as integrin adhesion receptors.  We performed an immunohistochem. anal. of FAK protein expression to det. the relationship between FAK overexpression and clinicopathol. factors in esophageal squamous cell carcinoma (ESCC).  We examd. tissue specimens that had been removed from 91 patients with thoracic esophageal cancer who had undergone surgery between 1983 and 2001.  Immunohistochem. staining was performed by the std. streptavidin-biotin method.  Seven human ESCC cell lines-TE-1, TE-2, TE-8, TE-13, TE-15, TT, and TTn-and one immortalized human keratinocyte cell line-HaCaT-were used in Western blot anal.  Immunostaining of FAK was seen in the cytoplasm of cancer cells, particularly in cells located in the invasive fronts of cancer nests.  FAK overexpression was detected in 54 of the 91 patients (59.3%).  Significant correlations were obsd. between FAK overexpression and cell differentiation (P=0.0057), depth of tumor invasion (P=0.0023), presence of regional lymph node metastasis (P=0.0097), no. of lymph node metastases (P=0.0026), and disease stage (P=0.012).  The survival rates of patients with FAK-overexpressing cancer were significantly lower than those of patients without FAK-overexpression cancer (P=0.006).  The 5-yr survival rate of patients without FAK overexpression was 69%, whereas that of patients with FAK overexpression was 38%.  On Western blot anal., FAK was expressed at a high level in TE-1, TE-8, TE-15, and TT cells, at a moderate level in TE-2 and TTn cells, and at a low level in TE13 and HaCaT cells.  FAK phosphorylation at tyrosine 397 was demonstrated in proportion to the intensity of FAK in all cell lines except TE15 and HaCaT.  In conclusion, FAK overexpression of ESCC was related to cell differentiation, tumor invasiveness, and lymph node metastasis.  Consequently, patients with ESCC who had FAK overexpression had a poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJyEVSutRHkrVg90H21EOLACvtfcHk0lj8_6zvdrxOqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVWmsbw%253D&md5=e0c5df0c3db976836f2d51b35baec1aa</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6601050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6601050%26sid%3Dliteratum%253Aachs%26aulast%3DMiyazaki%26aufirst%3DT.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DNakajima%26aufirst%3DM.%26aulast%3DSohda%26aufirst%3DM.%26aulast%3DFukai%26aufirst%3DY.%26aulast%3DMasuda%26aufirst%3DN.%26aulast%3DManda%26aufirst%3DR.%26aulast%3DFukuchi%26aufirst%3DM.%26aulast%3DTsukada%26aufirst%3DK.%26aulast%3DKuwano%26aufirst%3DH.%26atitle%3DFAK%2520overexpression%2520is%2520correlated%2520with%2520tumour%2520invasiveness%2520and%2520lymph%2520node%2520metastasis%2520in%2520oesophageal%2520squamous%2520cell%2520carcinoma%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2003%26volume%3D89%26spage%3D140%26epage%3D145%26doi%3D10.1038%2Fsj.bjc.6601050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aishima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirabe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maehara, Y.</span></span> <span> </span><span class="NLM_article-title">Role of expression of focal adhesion kinase in progression of hepatocellular carcinoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2812</span>– <span class="NLM_lpage">2817</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-1046-03</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1158%2F1078-0432.ccr-1046-03" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=15102689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1CltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=2812-2817&author=S.+Itohauthor=T.+Maedaauthor=M.+Shimadaauthor=S.+Aishimaauthor=K.+Shirabeauthor=S.+Tanakaauthor=Y.+Maehara&title=Role+of+expression+of+focal+adhesion+kinase+in+progression+of+hepatocellular+carcinoma&doi=10.1158%2F1078-0432.ccr-1046-03"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Role of expression of focal adhesion kinase in progression of hepatocellular carcinoma</span></div><div class="casAuthors">Itoh, Shinji; Maeda, Takashi; Shimada, Mitsuo; Aishima, Shin-ichi; Shirabe, Ken; Tanaka, Shinji; Maehara, Yoshihiko</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2812-2817</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Although hepatocellular carcinoma (HCC) is the most common cancer of the human liver, the mechanisms that regulate HCC development and progression remain unclear.  The aim of this study was to investigate whether focal adhesion kinase (FAK) is involved in the progression of human HCC.  Western blot anal. for FAK was performed on 3 HCC cell lines.  The authors reviewed 64 consecutive patients who had undergone initial liver resection for HCC without preoperative treatment.  Immunohistochem. anal. for FAK was performed on paraffin-embedded tissues.  FAK expression was confirmed by Western blot anal. in several clin. samples.  The authors investigated the correlation between FAK expression and clin. outcome.  FAK proteins were detected in all HCC cell lines.  Hepatocytes in the normal liver and chronic hepatitis with or without cirrhosis were neg. for immunohistochem. staining for FAK expression.  Cytoplasmic FAK expression was obsd. in 18 of 64 patients (28.1%), and this pos. staining was correlated with gender (P < 0.05), a lower level of serum albumin (P < 0.05), and portal venous invasion (P < 0.01).  Pos. staining for FAK was assocd. with significantly poorer survival (P < 0.05).  In multivariate anal., FAK overexpression was an independent factor in detg. the prognosis of patients.  These data suggest that FAK plays an important role in promoting tumor progression, esp. vascular invasion, in HCC.  FAK could play an important role in HCC progression and would be a novel target for HCC therapeutics as well as a prognostic marker.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUUZBKRASEuLVg90H21EOLACvtfcHk0lj8_6zvdrxOqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjt1CltbY%253D&md5=d16eb2f787952aa3a47b15a0730fc6b1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-1046-03&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-1046-03%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DS.%26aulast%3DMaeda%26aufirst%3DT.%26aulast%3DShimada%26aufirst%3DM.%26aulast%3DAishima%26aufirst%3DS.%26aulast%3DShirabe%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DMaehara%26aufirst%3DY.%26atitle%3DRole%2520of%2520expression%2520of%2520focal%2520adhesion%2520kinase%2520in%2520progression%2520of%2520hepatocellular%2520carcinoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2004%26volume%3D10%26spage%3D2812%26epage%3D2817%26doi%3D10.1158%2F1078-0432.ccr-1046-03" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armaiz-Pena, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nick, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spannuth, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deavers, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahzad, M. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Montero, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGeest, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivas-Mejia, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Berestein, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lutgendorf, S. K.</span></span> <span> </span><span class="NLM_article-title">Adrenergic modulation of focal adhesion kinase protects human ovarian cancer cells from anoikis</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">1515</span>– <span class="NLM_lpage">1523</span>, <span class="refDoi"> DOI: 10.1172/jci40802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1172%2Fjci40802" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2010&pages=1515-1523&author=A.+K.+Soodauthor=G.+N.+Armaiz-Penaauthor=J.+Halderauthor=A.+M.+Nickauthor=R.+L.+Stoneauthor=W.+Huauthor=A.+R.+Carrollauthor=W.+A.+Spannuthauthor=M.+T.+Deaversauthor=J.+K.+Allenauthor=L.+Y.+Hanauthor=A.+A.+Kamatauthor=M.+M.+K.+Shahzadauthor=B.+W.+McIntyreauthor=C.+M.+Diaz-Monteroauthor=N.+B.+Jenningsauthor=Y.+G.+Linauthor=W.+M.+Merrittauthor=K.+DeGeestauthor=P.+E.+Vivas-Mejiaauthor=G.+Lopez-Beresteinauthor=M.+D.+Schallerauthor=S.+W.+Coleauthor=S.+K.+Lutgendorf&title=Adrenergic+modulation+of+focal+adhesion+kinase+protects+human+ovarian+cancer+cells+from+anoikis&doi=10.1172%2Fjci40802"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1172%2Fjci40802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci40802%26sid%3Dliteratum%253Aachs%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DArmaiz-Pena%26aufirst%3DG.%2BN.%26aulast%3DHalder%26aufirst%3DJ.%26aulast%3DNick%26aufirst%3DA.%2BM.%26aulast%3DStone%26aufirst%3DR.%2BL.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DCarroll%26aufirst%3DA.%2BR.%26aulast%3DSpannuth%26aufirst%3DW.%2BA.%26aulast%3DDeavers%26aufirst%3DM.%2BT.%26aulast%3DAllen%26aufirst%3DJ.%2BK.%26aulast%3DHan%26aufirst%3DL.%2BY.%26aulast%3DKamat%26aufirst%3DA.%2BA.%26aulast%3DShahzad%26aufirst%3DM.%2BM.%2BK.%26aulast%3DMcIntyre%26aufirst%3DB.%2BW.%26aulast%3DDiaz-Montero%26aufirst%3DC.%2BM.%26aulast%3DJennings%26aufirst%3DN.%2BB.%26aulast%3DLin%26aufirst%3DY.%2BG.%26aulast%3DMerritt%26aufirst%3DW.%2BM.%26aulast%3DDeGeest%26aufirst%3DK.%26aulast%3DVivas-Mejia%26aufirst%3DP.%2BE.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26aulast%3DSchaller%26aufirst%3DM.%2BD.%26aulast%3DCole%26aufirst%3DS.%2BW.%26aulast%3DLutgendorf%26aufirst%3DS.%2BK.%26atitle%3DAdrenergic%2520modulation%2520of%2520focal%2520adhesion%2520kinase%2520protects%2520human%2520ovarian%2520cancer%2520cells%2520from%2520anoikis%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2010%26volume%3D120%26spage%3D1515%26epage%3D1523%26doi%3D10.1172%2Fjci40802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tancioni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N. L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uryu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupack, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plaxe, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlaepfer, D. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of focal adhesion kinase (FAK) activity prevents anchorage-independent ovarian carcinoma cell growth and tumor progression</span>. <i>Clin. Exp. Metastasis</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">579</span>– <span class="NLM_lpage">594</span>, <span class="refDoi"> DOI: 10.1007/s10585-012-9562-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1007%2Fs10585-012-9562-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=23275034" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=579-594&author=K.+K.+Wardauthor=I.+Tancioniauthor=C.+Lawsonauthor=N.+L.+G.+Millerauthor=C.+Jeanauthor=X.+L.+Chenauthor=S.+Uryuauthor=J.+Kimauthor=D.+Tarinauthor=D.+G.+Stupackauthor=S.+C.+Plaxeauthor=D.+D.+Schlaepfer&title=Inhibition+of+focal+adhesion+kinase+%28FAK%29+activity+prevents+anchorage-independent+ovarian+carcinoma+cell+growth+and+tumor+progression&doi=10.1007%2Fs10585-012-9562-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs10585-012-9562-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10585-012-9562-5%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DK.%2BK.%26aulast%3DTancioni%26aufirst%3DI.%26aulast%3DLawson%26aufirst%3DC.%26aulast%3DMiller%26aufirst%3DN.%2BL.%2BG.%26aulast%3DJean%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DX.%2BL.%26aulast%3DUryu%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DTarin%26aufirst%3DD.%26aulast%3DStupack%26aufirst%3DD.%2BG.%26aulast%3DPlaxe%26aufirst%3DS.%2BC.%26aulast%3DSchlaepfer%26aufirst%3DD.%2BD.%26atitle%3DInhibition%2520of%2520focal%2520adhesion%2520kinase%2520%2528FAK%2529%2520activity%2520prevents%2520anchorage-independent%2520ovarian%2520carcinoma%2520cell%2520growth%2520and%2520tumor%2520progression%26jtitle%3DClin.%2520Exp.%2520Metastasis%26date%3D2012%26volume%3D30%26spage%3D579%26epage%3D594%26doi%3D10.1007%2Fs10585-012-9562-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balogh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Victor, D.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asham, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burroughs, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boktour, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saharia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monsour, H.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Hepatocellular carcinoma: a review</span>. <i>J. Hepatocell. Carcinoma</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.2147/jhc.s61146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.2147%2Fjhc.s61146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=27785449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A280%3ADC%252BC2sngtV2qtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2016&pages=41-53&author=J.+Baloghauthor=D.+Victorauthor=E.+H.+Ashamauthor=S.+G.+Burroughsauthor=M.+Boktourauthor=A.+Sahariaauthor=X.+Liauthor=M.+Ghobrialauthor=H.+Monsour&title=Hepatocellular+carcinoma%3A+a+review&doi=10.2147%2Fjhc.s61146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatocellular carcinoma: a review</span></div><div class="casAuthors">Balogh Julius; Asham Emad H; Burroughs Sherilyn Gordon; Boktour Maha; Saharia Ashish; Li Xian; Ghobrial R Mark; Victor David 3rd; Monsour Howard P Jr</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hepatocellular carcinoma</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-53</span>
        ISSN:<span class="NLM_cas:issn">2253-5969</span>.
    </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is the most common primary liver malignancy and is a leading cause of cancer-related death worldwide.  In the United States, HCC is the ninth leading cause of cancer deaths.  Despite advances in prevention techniques, screening, and new technologies in both diagnosis and treatment, incidence and mortality continue to rise.  Cirrhosis remains the most important risk factor for the development of HCC regardless of etiology.  Hepatitis B and C are independent risk factors for the development of cirrhosis.  Alcohol consumption remains an important additional risk factor in the United States as alcohol abuse is five times higher than hepatitis C.  Diagnosis is confirmed without pathologic confirmation.  Screening includes both radiologic tests, such as ultrasound, computerized tomography, and magnetic resonance imaging, and serological markers such as α-fetoprotein at 6-month intervals.  Multiple treatment modalities exist; however, only orthotopic liver transplantation (OLT) or surgical resection is curative.  OLT is available for patients who meet or are downstaged into the Milan or University of San Francisco criteria.  Additional treatment modalities include transarterial chemoembolization, radiofrequency ablation, microwave ablation, percutaneous ethanol injection, cryoablation, radiation therapy, systemic chemotherapy, and molecularly targeted therapies.  Selection of a treatment modality is based on tumor size, location, extrahepatic spread, and underlying liver function.  HCC is an aggressive cancer that occurs in the setting of cirrhosis and commonly presents in advanced stages.  HCC can be prevented if there are appropriate measures taken, including hepatitis B virus vaccination, universal screening of blood products, use of safe injection practices, treatment and education of alcoholics and intravenous drug users, and initiation of antiviral therapy.  Continued improvement in both surgical and nonsurgical approaches has demonstrated significant benefits in overall survival.  While OLT remains the only curative surgical procedure, the shortage of available organs precludes this therapy for many patients with HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTUVKKRQ-ThgJPq5LRczm8ffW6udTcc2eZgOVBtNt8mM7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sngtV2qtw%253D%253D&md5=f746bbeacacdd9a8fa881afa307f8f7f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2147%2Fjhc.s61146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252Fjhc.s61146%26sid%3Dliteratum%253Aachs%26aulast%3DBalogh%26aufirst%3DJ.%26aulast%3DVictor%26aufirst%3DD.%26aulast%3DAsham%26aufirst%3DE.%2BH.%26aulast%3DBurroughs%26aufirst%3DS.%2BG.%26aulast%3DBoktour%26aufirst%3DM.%26aulast%3DSaharia%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DGhobrial%26aufirst%3DM.%26aulast%3DMonsour%26aufirst%3DH.%26atitle%3DHepatocellular%2520carcinoma%253A%2520a%2520review%26jtitle%3DJ.%2520Hepatocell.%2520Carcinoma%26date%3D2016%26volume%3D3%26spage%3D41%26epage%3D53%26doi%3D10.2147%2Fjhc.s61146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">El-Serag, H. B.</span></span> <span> </span><span class="NLM_article-title">Hepatocellular carcinoma</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>365</i></span>,  <span class="NLM_fpage">1118</span>– <span class="NLM_lpage">1127</span>, <span class="refDoi"> DOI: 10.1056/nejmra1001683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1056%2Fnejmra1001683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=21992124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1WqtrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2011&pages=1118-1127&author=H.+B.+El-Serag&title=Hepatocellular+carcinoma&doi=10.1056%2Fnejmra1001683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatocellular carcinoma</span></div><div class="casAuthors">El-Serag, Hashem B.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">365</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1118-1127</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ9eGOw4kyT7Vg90H21EOLACvtfcHk0lhZmb4ix0UJDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1WqtrnL&md5=3fb4250042385c5e29a189df86aea635</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1056%2Fnejmra1001683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252Fnejmra1001683%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Serag%26aufirst%3DH.%2BB.%26atitle%3DHepatocellular%2520carcinoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2011%26volume%3D365%26spage%3D1118%26epage%3D1127%26doi%3D10.1056%2Fnejmra1001683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Llovet, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzaferro, V.</span></span> <span> </span><span class="NLM_article-title">Resection and liver transplantation for hepatocellular carcinoma</span>. <i>Semin. Liver Dis.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1055/s-2005-871198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1055%2Fs-2005-871198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=15918147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A280%3ADC%252BD2M3nsFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=181-200&author=J.+M.+Llovetauthor=M.+Schwartzauthor=V.+Mazzaferro&title=Resection+and+liver+transplantation+for+hepatocellular+carcinoma&doi=10.1055%2Fs-2005-871198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Resection and liver transplantation for hepatocellular carcinoma</span></div><div class="casAuthors">Llovet Josep M; Schwartz Myron; Mazzaferro Vincenzo</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in liver disease</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-200</span>
        ISSN:<span class="NLM_cas:issn">0272-8087</span>.
    </div><div class="casAbstract">Surveillance programs in cirrhotic patients enable the detection of hepatocellular carcinoma (HCC) at early stages, when the tumor is amenable to curative treatments (60% of cases in Japan; 25 to 40% in Europe and the United States).  Resection is the mainstay of treatment in noncirrhotic patients and in cirrhotics with well-preserved liver function.  In modern series, a perioperative mortality < or = 3% and 5-year survival rates above 50% are expected.  Tumor recurrence complicates half of the cases at 3 years, but there is no unquestionable preventive treatment.  Liver transplantation provides excellent outcomes applying the Milan criteria (single nodule < or = 5 cm or two or three nodules < or = 3 cm), with 5-year survival rates of 70% and low recurrence rates.  Although expansion of selection criteria is appealing, it should be assessed in the setting of prospective well-designed studies.  Intention-to-treat analysis has shown that wide extended indications lead to 25% 5-year survival rates.  Living donor liver transplantation is having a minor impact in HCC management.  Molecular markers are needed to better select the candidates for surgery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQxJWeDn2fsrwHqQV89nF-2fW6udTcc2eZqzmo2TyFJ97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M3nsFWmsA%253D%253D&md5=19be350d7cc28a7d1ac66ae6d07ae4d2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1055%2Fs-2005-871198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-2005-871198%26sid%3Dliteratum%253Aachs%26aulast%3DLlovet%26aufirst%3DJ.%2BM.%26aulast%3DSchwartz%26aufirst%3DM.%26aulast%3DMazzaferro%26aufirst%3DV.%26atitle%3DResection%2520and%2520liver%2520transplantation%2520for%2520hepatocellular%2520carcinoma%26jtitle%3DSemin.%2520Liver%2520Dis.%26date%3D2005%26volume%3D25%26spage%3D181%26epage%3D200%26doi%3D10.1055%2Fs-2005-871198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span> <span> </span><span class="NLM_article-title">Comparison of hepatic resection and radiofrequency ablation for small hepatocellular carcinoma: a meta-analysis of 16,103 patients</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">7252</span>, <span class="refDoi"> DOI: 10.1038/srep07252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fsrep07252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=25429732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A280%3ADC%252BC2Mzhtlyiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=7252&author=Q.+Xuauthor=S.+Kobayashiauthor=X.+Yeauthor=X.+Meng&title=Comparison+of+hepatic+resection+and+radiofrequency+ablation+for+small+hepatocellular+carcinoma%3A+a+meta-analysis+of+16%2C103+patients&doi=10.1038%2Fsrep07252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of hepatic resection and radiofrequency ablation for small hepatocellular carcinoma: a meta-analysis of 16,103 patients</span></div><div class="casAuthors">Xu Qinghua; Ye Xun; Meng Xia; Kobayashi Seijin</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7252</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We performed a meta-analysis to evaluate the therapeutic effects of radiofrequency ablation (RFA) and surgical hepatic resection (HR) in the treatment of small hepatocellular carcinoma (HCC).  Thirty-one studies were included in the analysis.  A total of 16,103 patients were involved: 8,252 treated with RFA and 7,851 with HR.  Compared to the RFA group, the 3, 5-year overall and disease-free survival rates in the HR group were significantly higher.  On the other hand, complications were significantly fewer and hospital-stay was significantly shorter in the RFA group than in the HR group.  In subgroup analyses, the overall and disease-free survival in the HR group were also significantly higher than those in the RFA group for HCCs ≤ 3 cm, whereas there were no significant differences between the two groups for HCCs ≤ 2 cm.  Our analysis showed that although HR was associated with higher complication rate and longer hospital-stay, HR is proposed as the first-line treatment rather than RFA for patients with HCCs larger than 2 cm.  For patients with HCCs of 2 cm or less, RFA may be an alternative to HR because of their comparable long-term efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTduceYDnsZa_6AExegM5LLfW6udTcc2eZqzmo2TyFJ97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mzhtlyiuw%253D%253D&md5=43905d4f504f17316dd594e4a0dd1d0c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsrep07252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep07252%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DKobayashi%26aufirst%3DS.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DMeng%26aufirst%3DX.%26atitle%3DComparison%2520of%2520hepatic%2520resection%2520and%2520radiofrequency%2520ablation%2520for%2520small%2520hepatocellular%2520carcinoma%253A%2520a%2520meta-analysis%2520of%252016%252C103%2520patients%26jtitle%3DSci.%2520Rep.%26date%3D2014%26volume%3D4%26spage%3D7252%26doi%3D10.1038%2Fsrep07252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span> <span> </span><span class="NLM_article-title">3’3-Diindolylmethane inhibits migration, invasion and metastasis of hepatocellular carcinoma by suppressing FAK signaling</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">23776</span>– <span class="NLM_lpage">23792</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.4196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.18632%2Foncotarget.4196" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=23776-23792&author=W.-X.+Liauthor=L.-P.+Chenauthor=M.-Y.+Sunauthor=J.-T.+Liauthor=H.-Z.+Liuauthor=W.+Zhu&title=3%E2%80%993-Diindolylmethane+inhibits+migration%2C+invasion+and+metastasis+of+hepatocellular+carcinoma+by+suppressing+FAK+signaling&doi=10.18632%2Foncotarget.4196"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4196%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.-X.%26aulast%3DChen%26aufirst%3DL.-P.%26aulast%3DSun%26aufirst%3DM.-Y.%26aulast%3DLi%26aufirst%3DJ.-T.%26aulast%3DLiu%26aufirst%3DH.-Z.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3D3%25E2%2580%25993-Diindolylmethane%2520inhibits%2520migration%252C%2520invasion%2520and%2520metastasis%2520of%2520hepatocellular%2520carcinoma%2520by%2520suppressing%2520FAK%2520signaling%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D23776%26epage%3D23792%26doi%3D10.18632%2Foncotarget.4196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gnani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romito, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artuso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chierici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Stefanis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panera, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crudele, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccarelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carcarino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Oria, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porru, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giorda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrari, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miele, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furlanello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locatelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nobili, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rota, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonetti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alisi, A.</span></span> <span> </span><span class="NLM_article-title">Focal adhesion kinase depletion reduces human hepatocellular carcinoma growth by repressing enhancer of zeste homolog 2</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">902</span>, <span class="refDoi"> DOI: 10.1038/cdd.2017.34</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fcdd.2017.34" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=28338656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFGqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=889-902&author=D.+Gnaniauthor=I.+Romitoauthor=S.+Artusoauthor=M.+Chiericiauthor=C.+De+Stefanisauthor=N.+Paneraauthor=A.+Crudeleauthor=S.+Ceccarelliauthor=E.+Carcarinoauthor=V.+D%E2%80%99Oriaauthor=M.+Porruauthor=E.+Giordaauthor=K.+Ferrariauthor=L.+Mieleauthor=E.+Villaauthor=C.+Balsanoauthor=D.+Pasiniauthor=C.+Furlanelloauthor=F.+Locatelliauthor=V.+Nobiliauthor=R.+Rotaauthor=C.+Leonettiauthor=A.+Alisi&title=Focal+adhesion+kinase+depletion+reduces+human+hepatocellular+carcinoma+growth+by+repressing+enhancer+of+zeste+homolog+2&doi=10.1038%2Fcdd.2017.34"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Focal adhesion kinase depletion reduces human hepatocellular carcinoma growth by repressing enhancer of zeste homolog 2</span></div><div class="casAuthors">Gnani, Daniela; Romito, Ilaria; Artuso, Simona; Chierici, Marco; De Stefanis, Cristiano; Panera, Nadia; Crudele, Annalisa; Ceccarelli, Sara; Carcarino, Elena; D'Oria, Valentina; Porru, Manuela; Giorda, Ezio; Ferrari, Karin; Miele, Luca; Villa, Erica; Balsano, Clara; Pasini, Diego; Furlanello, Cesare; Locatelli, Franco; Nobili, Valerio; Rota, Rossella; Leonetti, Carlo; Alisi, Anna</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Differentiation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">889-902</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is the most common type of liver cancer in humans.  The focal adhesion tyrosine kinase (FAK) is often over-expressed in human HCC and FAK inhibition may reduce HCC cell invasiveness.  However, the anti-oncogenic effect of FAK knockdown in HCC cells remains to be clarified.  We found that FAK depletion in HCC cells reduced in vitro and in vivo tumorigenicity, by inducing G2/M arrest and apoptosis, decreasing anchorage-independent growth, and modulating the expression of several cancer-related genes.  Among these genes, we showed that FAK silencing decreased transcription and nuclear localization of enhancer of zeste homolog 2 (EZH2) and its tri-methylation activity on lysine 27 of histone H3 (H3K27me3).  Accordingly, FAK, EZH2 and H3K27me3 were concomitantly upregulated in human HCCs compared to non-tumor livers.  In vitro expts. demonstrated that FAK affected EZH2 expression and function by modulating, at least in part, p53 and E2F2/3 transcriptional activity.  Moreover, FAK silencing downregulated both EZH2 binding and histone H3K27me3 levels at the promoter of its target gene NOTCH2.  Finally, we found that pharmacol. inhibition of FAK activity resembled these effects although milder.  In summary, we demonstrate that FAK depletion reduces HCC cell growth by affecting cancer-promoting genes including the pro-oncogene EZH2.  Furthermore, we unveil a novel unprecedented FAK/EZH2 crosstalk in HCC cells, thus identifying a targetable network paving the way for new anticancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR0le0fu-lMrVg90H21EOLACvtfcHk0lhRe7ZpeySLvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFGqt7g%253D&md5=d9cd4ec78e71f3ade7f8e748172199ad</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fcdd.2017.34&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcdd.2017.34%26sid%3Dliteratum%253Aachs%26aulast%3DGnani%26aufirst%3DD.%26aulast%3DRomito%26aufirst%3DI.%26aulast%3DArtuso%26aufirst%3DS.%26aulast%3DChierici%26aufirst%3DM.%26aulast%3DDe%2BStefanis%26aufirst%3DC.%26aulast%3DPanera%26aufirst%3DN.%26aulast%3DCrudele%26aufirst%3DA.%26aulast%3DCeccarelli%26aufirst%3DS.%26aulast%3DCarcarino%26aufirst%3DE.%26aulast%3DD%25E2%2580%2599Oria%26aufirst%3DV.%26aulast%3DPorru%26aufirst%3DM.%26aulast%3DGiorda%26aufirst%3DE.%26aulast%3DFerrari%26aufirst%3DK.%26aulast%3DMiele%26aufirst%3DL.%26aulast%3DVilla%26aufirst%3DE.%26aulast%3DBalsano%26aufirst%3DC.%26aulast%3DPasini%26aufirst%3DD.%26aulast%3DFurlanello%26aufirst%3DC.%26aulast%3DLocatelli%26aufirst%3DF.%26aulast%3DNobili%26aufirst%3DV.%26aulast%3DRota%26aufirst%3DR.%26aulast%3DLeonetti%26aufirst%3DC.%26aulast%3DAlisi%26aufirst%3DA.%26atitle%3DFocal%2520adhesion%2520kinase%2520depletion%2520reduces%2520human%2520hepatocellular%2520carcinoma%2520growth%2520by%2520repressing%2520enhancer%2520of%2520zeste%2520homolog%25202%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2017%26volume%3D24%26spage%3D889%26epage%3D902%26doi%3D10.1038%2Fcdd.2017.34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirt, U. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waizenegger, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslinger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braunger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weyer-Czernilofsky, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadtmuller, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapountzis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bister, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quant, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garin-Chesa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adolf, G. R.</span></span> <span> </span><span class="NLM_article-title">Efficacy of the highly selective focal adhesion kinase inhibitor BI 853520 in adenocarcinoma xenograft models is linked to a mesenchymal tumor phenotype</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">21</span>, <span class="refDoi"> DOI: 10.1038/s41389-018-0032-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fs41389-018-0032-z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=21&author=U.+A.+Hirtauthor=I.+C.+Waizeneggerauthor=N.+Schweiferauthor=C.+Haslingerauthor=D.+Gerlachauthor=J.+Braungerauthor=U.+Weyer-Czernilofskyauthor=H.+Stadtmullerauthor=I.+Sapountzisauthor=G.+Baderauthor=A.+Zoephelauthor=B.+Bisterauthor=A.+Baumauthor=J.+Quantauthor=N.+Krautauthor=P.+Garin-Chesaauthor=G.+R.+Adolf&title=Efficacy+of+the+highly+selective+focal+adhesion+kinase+inhibitor+BI+853520+in+adenocarcinoma+xenograft+models+is+linked+to+a+mesenchymal+tumor+phenotype&doi=10.1038%2Fs41389-018-0032-z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fs41389-018-0032-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41389-018-0032-z%26sid%3Dliteratum%253Aachs%26aulast%3DHirt%26aufirst%3DU.%2BA.%26aulast%3DWaizenegger%26aufirst%3DI.%2BC.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DHaslinger%26aufirst%3DC.%26aulast%3DGerlach%26aufirst%3DD.%26aulast%3DBraunger%26aufirst%3DJ.%26aulast%3DWeyer-Czernilofsky%26aufirst%3DU.%26aulast%3DStadtmuller%26aufirst%3DH.%26aulast%3DSapountzis%26aufirst%3DI.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DBister%26aufirst%3DB.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DQuant%26aufirst%3DJ.%26aulast%3DKraut%26aufirst%3DN.%26aulast%3DGarin-Chesa%26aufirst%3DP.%26aulast%3DAdolf%26aufirst%3DG.%2BR.%26atitle%3DEfficacy%2520of%2520the%2520highly%2520selective%2520focal%2520adhesion%2520kinase%2520inhibitor%2520BI%2520853520%2520in%2520adenocarcinoma%2520xenograft%2520models%2520is%2520linked%2520to%2520a%2520mesenchymal%2520tumor%2520phenotype%26jtitle%3DOncogenesis%26date%3D2018%26volume%3D7%26spage%3D21%26doi%3D10.1038%2Fs41389-018-0032-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tiede, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer-Schaller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalathur, R. K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagiani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmassmann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stillhard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafliger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waizenegger, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bill, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christofori, G.</span></span> <span> </span><span class="NLM_article-title">The FAK inhibitor BI 853520 exerts anti-tumor effects in breast cancer</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">73</span>, <span class="refDoi"> DOI: 10.1038/s41389-018-0083-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fs41389-018-0083-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=30237500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A280%3ADC%252BB3czhtVamtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=73&author=S.+Tiedeauthor=N.+Meyer-Schallerauthor=R.+K.+R.+Kalathurauthor=R.+Ivanekauthor=E.+Fagianiauthor=P.+Schmassmannauthor=P.+Stillhardauthor=S.+Hafligerauthor=N.+Krautauthor=N.+Schweiferauthor=I.+C.+Waizeneggerauthor=R.+Billauthor=G.+Christofori&title=The+FAK+inhibitor+BI+853520+exerts+anti-tumor+effects+in+breast+cancer&doi=10.1038%2Fs41389-018-0083-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The FAK inhibitor BI 853520 exerts anti-tumor effects in breast cancer</span></div><div class="casAuthors">Tiede Stefanie; Meyer-Schaller Nathalie; Kalathur Ravi Kiran Reddy; Ivanek Robert; Fagiani Ernesta; Schmassmann Philip; Stillhard Patrick; Hafliger Simon; Bill Ruben; Christofori Gerhard; Kalathur Ravi Kiran Reddy; Ivanek Robert; Kraut Norbert; Schweifer Norbert; Waizenegger Irene C</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">73</span>
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    </div><div class="casAbstract">Focal adhesion kinase (FAK) is a cytoplasmic tyrosine kinase that regulates a plethora of downstream signaling pathways essential for cell migration, proliferation and death, processes that are exploited by cancer cells during malignant progression.  These well-established tumorigenic activities, together with its high expression and activity in different cancer types, highlight FAK as an attractive target for cancer therapy.  We have assessed and characterized the therapeutic potential and the biological effects of BI 853520, a novel small chemical inhibitor of FAK, in several preclinical mouse models of breast cancer.  Treatment with BI 853520 elicits a significant reduction in primary tumor growth caused by an anti-proliferative activity by BI 853520.  In contrast, BI 853520 exerts effects with varying degrees of robustness on the different stages of the metastatic cascade.  Together, the data demonstrate that the repression of FAK activity by the specific FAK inhibitor BI 853520 offers a promising anti-proliferative approach for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSPh9ctMIKK5LxanA7vODO6fW6udTcc2ebbytNxfGqueLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czhtVamtA%253D%253D&md5=9875efe4b07431bf3399fd19a3f7f289</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41389-018-0083-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41389-018-0083-1%26sid%3Dliteratum%253Aachs%26aulast%3DTiede%26aufirst%3DS.%26aulast%3DMeyer-Schaller%26aufirst%3DN.%26aulast%3DKalathur%26aufirst%3DR.%2BK.%2BR.%26aulast%3DIvanek%26aufirst%3DR.%26aulast%3DFagiani%26aufirst%3DE.%26aulast%3DSchmassmann%26aufirst%3DP.%26aulast%3DStillhard%26aufirst%3DP.%26aulast%3DHafliger%26aufirst%3DS.%26aulast%3DKraut%26aufirst%3DN.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DWaizenegger%26aufirst%3DI.%2BC.%26aulast%3DBill%26aufirst%3DR.%26aulast%3DChristofori%26aufirst%3DG.%26atitle%3DThe%2520FAK%2520inhibitor%2520BI%2520853520%2520exerts%2520anti-tumor%2520effects%2520in%2520breast%2520cancer%26jtitle%3DOncogenesis%26date%3D2018%26volume%3D7%26spage%3D73%26doi%3D10.1038%2Fs41389-018-0083-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanjoni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uryu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, J.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mielgo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupack, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlaepfer, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlaepfer, D. D.</span></span> <span> </span><span class="NLM_article-title">PND-1186 FAK inhibitor selectively promotes tumor cell apoptosis in three-dimensional environments</span>. <i>Cancer Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">764</span>– <span class="NLM_lpage">777</span>, <span class="refDoi"> DOI: 10.4161/cbt.9.10.11434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.4161%2Fcbt.9.10.11434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=20234191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKrtrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=764-777&author=I.+Tanjoniauthor=C.+Walshauthor=S.+Uryuauthor=A.+Tomarauthor=J.-O.+Namauthor=A.+Mielgoauthor=S.-T.+Limauthor=C.+Liangauthor=M.+Koenigauthor=N.+Patelauthor=C.+Kwokauthor=G.+McMahonauthor=D.+G.+Stupackauthor=D.+D.+Schlaepferauthor=D.+D.+Schlaepfer&title=PND-1186+FAK+inhibitor+selectively+promotes+tumor+cell+apoptosis+in+three-dimensional+environments&doi=10.4161%2Fcbt.9.10.11434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">PND-1186 FAK inhibitor selectively promotes tumor cell apoptosis in three-dimensional environments</span></div><div class="casAuthors">Tanjoni, Isabelle; Walsh, Colin; Uryu, Sean; Tomar, Alok; Nam, Ju-Ock; Mielgo, Ainhoa; Lim, Ssang-Taek; Liang, Congxin; Koenig, Marcel; Sun, Connie; Patel, Neela; Kwok, Cheni; McMahon, Gerald; Stupack, Dwayne G.; Schlaepfer, David D.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Therapy</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">764-777</span>CODEN:
                <span class="NLM_cas:coden">CBTAAO</span>;
        ISSN:<span class="NLM_cas:issn">1538-4047</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Tumor cells can grow in an anchorage-independent manner.  This is mediated in part through survival signals that bypass normal growth restraints controlled by integrin cell surface receptors.  Focal adhesion kinase (FAK) is a cytoplasmic protein-tyrosine kinase that assocs. with integrins and modulates various cellular processes including growth, survival and migration.  As increased FAK expression and tyrosine phosphorylation are assocd. with tumor progression, inhibitors of FAK are being tested for anti-tumor effects.  Here, we analyze PND-1186, a substituted pyridine reversible inhibitor of FAK activity with a 50% inhibitory concn. (IC50) of 1.5 nM in vitro.  PND-1186 has an IC50 of ∼100 nM in breast carcinoma cells as detd. by anti-phospho-specific immunoblotting to FAK Tyr-397.  PND-1186 did not alter c-Src or p130Cas tyrosine phosphorylation in adherent cells, yet functioned to restrain cell movement.  Notably, 1.0 μM PND-1186 (>5-fold above IC50) had limited effects on cell proliferation.  However, under non-adherent conditions as spheroids and as colonies in soft agar, 0.1 μM PND-1186 blocked FAK and p130Cas tyrosine phosphorylation, promoted caspase-3 activation, and triggered cell apoptosis.  PND-1186 inhibited 4T1 breast carcinoma s.c. tumor growth correlated with elevated tumor cell apoptosis and caspase 3 activation.  Addn. of PND-1186 to the drinking water of mice was well tolerated and inhibited ascites- and peritoneal membrane-assocd. ovarian carcinoma tumor growth assocd. with the inhibition of FAK Tyr-397 phosphorylation.  Our results with low-level PND-1186 treatment support the conclusion that FAK activity selectively promotes tumor cell survival in three-dimensional environments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsLnD2EGHjs7Vg90H21EOLACvtfcHk0lgcmGpLgxQ_qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKrtrnI&md5=5cb6370e769d6e4f88ef5e27d70a7996</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4161%2Fcbt.9.10.11434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcbt.9.10.11434%26sid%3Dliteratum%253Aachs%26aulast%3DTanjoni%26aufirst%3DI.%26aulast%3DWalsh%26aufirst%3DC.%26aulast%3DUryu%26aufirst%3DS.%26aulast%3DTomar%26aufirst%3DA.%26aulast%3DNam%26aufirst%3DJ.-O.%26aulast%3DMielgo%26aufirst%3DA.%26aulast%3DLim%26aufirst%3DS.-T.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DKoenig%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DKwok%26aufirst%3DC.%26aulast%3DMcMahon%26aufirst%3DG.%26aulast%3DStupack%26aufirst%3DD.%2BG.%26aulast%3DSchlaepfer%26aufirst%3DD.%2BD.%26aulast%3DSchlaepfer%26aufirst%3DD.%2BD.%26atitle%3DPND-1186%2520FAK%2520inhibitor%2520selectively%2520promotes%2520tumor%2520cell%2520apoptosis%2520in%2520three-dimensional%2520environments%26jtitle%3DCancer%2520Biol.%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D764%26epage%3D777%26doi%3D10.4161%2Fcbt.9.10.11434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cance, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golubovskaya, V. M.</span></span> <span> </span><span class="NLM_article-title">Focal adhesion kinase versus p53: apoptosis or survival?</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">pe22</span>, <span class="refDoi"> DOI: 10.1126/stke.120pe22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1126%2Fstke.120pe22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=18493017" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2008&pages=pe22&author=W.+G.+Canceauthor=V.+M.+Golubovskaya&title=Focal+adhesion+kinase+versus+p53%3A+apoptosis+or+survival%3F&doi=10.1126%2Fstke.120pe22"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fstke.120pe22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fstke.120pe22%26sid%3Dliteratum%253Aachs%26aulast%3DCance%26aufirst%3DW.%2BG.%26aulast%3DGolubovskaya%26aufirst%3DV.%2BM.%26atitle%3DFocal%2520adhesion%2520kinase%2520versus%2520p53%253A%2520apoptosis%2520or%2520survival%253F%26jtitle%3DSci.%2520Signaling%26date%3D2008%26volume%3D1%26spage%3Dpe22%26doi%3D10.1126%2Fstke.120pe22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lgcmGpLgxQ_qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras: induced protein degradation as a therapeutic strategy</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=892-898&author=P.+Ottisauthor=C.+M.+Crews&title=Proteolysis-targeting+chimeras%3A+induced+protein+degradation+as+a+therapeutic+strategy&doi=10.1021%2Facschembio.6b01068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span></div><div class="casAuthors">Ottis, Philipp; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-898</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Until recently, the only ways to reduce specific protein signaling were to either knock down the target by RNAi or to interfere with the signaling by inhibiting an enzyme or receptor within the signal transduction cascade.  Herein, we review an emerging class of small mol. pharmacol. agents, called PROTACs, that present a novel approach to specifically target proteins and their resp. signaling pathways.  These heterobifunctional mols. utilize endogenous cellular quality control machinery by recruiting it to target proteins in order to induce their degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohB6xwkQMTprVg90H21EOLACvtfcHk0lgcmGpLgxQ_qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D&md5=eada521c10faa1e456e0885fd390c554</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01068%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-targeting%2520chimeras%253A%2520induced%2520protein%2520degradation%2520as%2520a%2520therapeutic%2520strategy%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D892%26epage%3D898%26doi%3D10.1021%2Facschembio.6b01068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span> <span> </span><span class="NLM_article-title">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>474</i></span>,  <span class="NLM_fpage">1127</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi"> DOI: 10.1042/bcj20160762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1042%2Fbcj20160762" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2017&pages=1127-1147&author=I.+Collinsauthor=H.+Wangauthor=J.+J.+Caldwellauthor=R.+Chopra&title=Chemical+approaches+to+targeted+protein+degradation+through+modulation+of+the+ubiquitin-proteasome+pathway&doi=10.1042%2Fbcj20160762"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1042%2Fbcj20160762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbcj20160762%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DChemical%2520approaches%2520to%2520targeted%2520protein%2520degradation%2520through%2520modulation%2520of%2520the%2520ubiquitin-proteasome%2520pathway%26jtitle%3DBiochem.%2520J.%26date%3D2017%26volume%3D474%26spage%3D1127%26epage%3D1147%26doi%3D10.1042%2Fbcj20160762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1042/ebc20170041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1042%2Febc20170041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=29118097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=505-516&author=S.+J.+Hughesauthor=A.+Ciulli&title=Molecular+recognition+of+ternary+complexes%3A+a+new+dimension+in+the+structure-guided+design+of+chemical+degraders&doi=10.1042%2Febc20170041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders</span></div><div class="casAuthors">Hughes Scott J; Ciulli Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">505-516</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular glues and bivalent inducers of protein degradation (also known as PROTACs) represent a fascinating new modality in pharmacotherapeutics: the potential to knockdown previously thought 'undruggable' targets at sub-stoichiometric concentrations in ways not possible using conventional inhibitors.  Mounting evidence suggests these chemical agents, in concert with their target proteins, can be modelled as three-body binding equilibria that can exhibit significant cooperativity as a result of specific ligand-induced molecular recognition.  Despite this, many existing drug design and optimization regimens still fixate on binary target engagement, in part due to limited structural data on ternary complexes.  Recent crystal structures of protein complexes mediated by degrader molecules, including the first PROTAC ternary complex, underscore the importance of protein-protein interactions and intramolecular contacts to the mode of action of this class of compounds.  These discoveries have opened the door to a new paradigm for structure-guided drug design: borrowing surface area and molecular recognition from nature to elicit cellular signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0b5XHngOXiV_fW6udTcc2eYwoT3l1Uife7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D&md5=7470e559372f05d34c5924524a6d04bc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1042%2Febc20170041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Febc20170041%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DMolecular%2520recognition%2520of%2520ternary%2520complexes%253A%2520a%2520new%2520dimension%2520in%2520the%2520structure-guided%2520design%2520of%2520chemical%2520degraders%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D505%26epage%3D516%26doi%3D10.1042%2Febc20170041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safaee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponthier, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancias, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Plasticity in binding confers selectivity in ligand-induced protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0055-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fs41589-018-0055-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=29892083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=706-714&author=R.+P.+Nowakauthor=S.+L.+DeAngeloauthor=D.+Buckleyauthor=Z.+Heauthor=K.+A.+Donovanauthor=J.+Anauthor=N.+Safaeeauthor=M.+P.+Jedrychowskiauthor=C.+M.+Ponthierauthor=M.+Ishoeyauthor=T.+Zhangauthor=J.+D.+Manciasauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=E.+S.+Fischer&title=Plasticity+in+binding+confers+selectivity+in+ligand-induced+protein+degradation&doi=10.1038%2Fs41589-018-0055-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Plasticity in binding confers selectivity in ligand-induced protein degradation</span></div><div class="casAuthors">Nowak, Radoslaw P.; De Angelo, Stephen L.; Buckley, Dennis; He, Zhixiang; Donovan, Katherine A.; An, Jian; Safaee, Nozhat; Jedrychowski, Mark P.; Ponthier, Charles M.; Ishoey, Mette; Zhang, Tinghu; Mancias, Joseph D.; Gray, Nathanael S.; Bradner, James E.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">706-714</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Heterobifunctional small-mol. degraders that induce protein degrdn. through ligase-mediated ubiquitination have shown considerable promise as a new pharmacol. modality.  However, we currently lack a detailed understanding of the mol. basis for target recruitment and selectivity, which is critically required to enable rational design of degraders.  Here we utilized a comprehensive characterization of the ligand-dependent CRBN-BRD4 interaction to demonstrate that binding between proteins that have not evolved to interact is plastic.  Multiple x-ray crystal structures showed that plasticity resulted in several distinct low-energy binding conformations that were selectively bound by ligands.  We demonstrated that computational protein-protein docking can reveal the underlying interprotein contacts and inform the design of a BRD4 selective degrader that can discriminate between highly homologous BET bromodomains.  These findings that plastic interprotein contacts confer selectivity for ligand-induced protein dimerization provide a conceptual framework for the development of heterobifunctional ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreQasCW8bz2LVg90H21EOLACvtfcHk0lgOTspuQrQcYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK&md5=e4876c43b1eeb4558c23121b7a9fac27</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0055-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0055-y%26sid%3Dliteratum%253Aachs%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DBuckley%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DSafaee%26aufirst%3DN.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DPonthier%26aufirst%3DC.%2BM.%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DMancias%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DPlasticity%2520in%2520binding%2520confers%2520selectivity%2520in%2520ligand-induced%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D706%26epage%3D714%26doi%3D10.1038%2Fs41589-018-0055-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.-H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0ljK1A5bbJwD0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Impact of target warhead and linkage vector on inducing protein degradation: comparison of bromodomain and extra-terminal (BET) degraders derived from triazolodiazepine (JQ1) and tetrahydroquinoline (I-BET726) BET inhibitor scaffolds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01912</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=504-513&author=K.-H.+Chanauthor=M.+Zengerleauthor=A.+Testaauthor=A.+Ciulli&title=Impact+of+target+warhead+and+linkage+vector+on+inducing+protein+degradation%3A+comparison+of+bromodomain+and+extra-terminal+%28BET%29+degraders+derived+from+triazolodiazepine+%28JQ1%29+and+tetrahydroquinoline+%28I-BET726%29+BET+inhibitor+scaffolds&doi=10.1021%2Facs.jmedchem.6b01912"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DImpact%2520of%2520target%2520warhead%2520and%2520linkage%2520vector%2520on%2520inducing%2520protein%2520degradation%253A%2520comparison%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520degraders%2520derived%2520from%2520triazolodiazepine%2520%2528JQ1%2529%2520and%2520tetrahydroquinoline%2520%2528I-BET726%2529%2520BET%2520inhibitor%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D61%26spage%3D504%26epage%3D513%26doi%3D10.1021%2Facs.jmedchem.6b01912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0ljK1A5bbJwD0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0ljK1A5bbJwD0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wühr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the BAF complex factor BRD9 by heterobifunctional ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.-S.+Seoauthor=S.+Dastjerdiauthor=M.+W%C3%BChrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+BAF+complex+factor+BRD9+by+heterobifunctional+ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0lh7Qj6H-Q6v5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DW%25C3%25BChr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520BAF%2520complex%2520factor%2520BRD9%2520by%2520heterobifunctional%2520ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lh7Qj6H-Q6v5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerschmied, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Modular PROTAC design for the degradation of oncogenic BCR-ABL</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+design+for+the+degradation+of+oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520design%2520for%2520the%2520degradation%2520of%2520oncogenic%2520BCR-ABL%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petroski, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Function and regulation of cullin-RING ubiquitin ligases</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1038/nrm1547</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnrm1547" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=15688063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVWg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=9-20&author=M.+D.+Petroskiauthor=R.+J.+Deshaies&title=Function+and+regulation+of+cullin-RING+ubiquitin+ligases&doi=10.1038%2Fnrm1547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Function and regulation of cullin-RING ubiquitin ligases</span></div><div class="casAuthors">Petroski, Matthew D.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-20</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cullin-RING complexes comprise the largest known class of ubiquitin ligases.  Owing to the great diversity of their substrate-receptor subunits, it is possible that there are hundreds of distinct cullin-RING ubiquitin ligases in eukaryotic cells, which establishes these enzymes as key mediators of post-translational protein regulation.  Here, the authors focus on the compn., regulation, and function of cullin-RING ligases, and describe how these enzymes can be characterized by a set of general principles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFw5LuMMHZoLVg90H21EOLACvtfcHk0lh7Qj6H-Q6v5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVWg&md5=297f142f44f971c6858306b1adb36254</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrm1547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1547%26sid%3Dliteratum%253Aachs%26aulast%3DPetroski%26aufirst%3DM.%2BD.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DFunction%2520and%2520regulation%2520of%2520cullin-RING%2520ubiquitin%2520ligases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D9%26epage%3D20%26doi%3D10.1038%2Fnrm1547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braggio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, C.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruins, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Wier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, X.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjorklund, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonseca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsagel, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlowski, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, A. K.</span></span> <span> </span><span class="NLM_article-title">Cereblon expression is required for the antimyeloma activity of lenalidomide and pomalidomide</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">4771</span>– <span class="NLM_lpage">4779</span>, <span class="refDoi"> DOI: 10.1182/blood-2011-05-356063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1182%2Fblood-2011-05-356063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=21860026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOgsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2011&pages=4771-4779&author=Y.+X.+Zhuauthor=E.+Braggioauthor=C.-X.+Shiauthor=L.+A.+Bruinsauthor=J.+E.+Schmidtauthor=S.+Van+Wierauthor=X.-B.+Changauthor=C.+C.+Bjorklundauthor=R.+Fonsecaauthor=P.+L.+Bergsagelauthor=R.+Z.+Orlowskiauthor=A.+K.+Stewart&title=Cereblon+expression+is+required+for+the+antimyeloma+activity+of+lenalidomide+and+pomalidomide&doi=10.1182%2Fblood-2011-05-356063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Cereblon expression is required for the antimyeloma activity of lenalidomide and pomalidomide</span></div><div class="casAuthors">Zhu, Yuan Xiao; Braggio, Esteban; Shi, Chang-Xin; Bruins, Laura A.; Schmidt, Jessica E.; Van Wier, Scott; Chang, Xiu-Bao; Bjorklund, Chad C.; Fonseca, Rafael; Bergsagel, P. Leif; Orlowski, Robert Z.; Stewart, A. Keith</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4771-4779</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The precise mol. mechanism of action and targets through which thalidomide and related immunomodulatory drugs (IMiDs) exert their antitumor effects remains unclear.  We investigated the role of cereblon (CRBN), a primary teratogenic target of thalidomide, in the antimyeloma activity of IMiDs.  CRBN depletion is initially cytotoxic to human myeloma cells, but surviving cells with stable CRBN depletion become highly resistant to both lenalidomide and pomalidomide, but not to the unrelated drugs bortezomib, dexamethasone, and melphalan.  Acquired deletion of CRBN was found to be the primary genetic event differentiating isogenic MM1.S cell lines cultured to be sensitive or resistant to lenalidomide and pomalidomide.  Gene expression changes induced by lenalidomide were dramatically suppressed in the presence of CRBN depletion, further demonstrating that CRBN is required for lenalidomide activity.  Downstream targets of CRBN include interferon regulatory factor 4 (IRF4) previously reported to also be a target of lenalidomide.  Patients exposed to, and putatively resistant to, lenalidomide had lower CRBN levels in paired samples before and after therapy.  In summary, CRBN is an essential requirement for IMiD activity and a possible biomarker for the clin. assessment of antimyeloma efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJCuFi6UHfKrVg90H21EOLACvtfcHk0liLCx8FGAOQIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOgsrjO&md5=2302c491fec7729d6c2c08735836a585</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1182%2Fblood-2011-05-356063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2011-05-356063%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DY.%2BX.%26aulast%3DBraggio%26aufirst%3DE.%26aulast%3DShi%26aufirst%3DC.-X.%26aulast%3DBruins%26aufirst%3DL.%2BA.%26aulast%3DSchmidt%26aufirst%3DJ.%2BE.%26aulast%3DVan%2BWier%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DX.-B.%26aulast%3DBjorklund%26aufirst%3DC.%2BC.%26aulast%3DFonseca%26aufirst%3DR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DOrlowski%26aufirst%3DR.%2BZ.%26aulast%3DStewart%26aufirst%3DA.%2BK.%26atitle%3DCereblon%2520expression%2520is%2520required%2520for%2520the%2520antimyeloma%2520activity%2520of%2520lenalidomide%2520and%2520pomalidomide%26jtitle%3DBlood%26date%3D2011%26volume%3D118%26spage%3D4771%26epage%3D4779%26doi%3D10.1182%2Fblood-2011-05-356063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loehr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahcall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of anaplastic lymphoma kinase (ALK)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4249</span>– <span class="NLM_lpage">4255</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4249-4255&author=C.+E.+Powellauthor=Y.+Gaoauthor=L.+Tanauthor=K.+A.+Donovanauthor=R.+P.+Nowakauthor=A.+Loehrauthor=M.+Bahcallauthor=E.+S.+Fischerauthor=P.+A.+J%C3%A4nneauthor=R.+E.+Georgeauthor=N.+S.+Gray&title=Chemically+induced+degradation+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1021%2Facs.jmedchem.7b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Powell, Chelsea E.; Gao, Yang; Tan, Li; Donovan, Katherine A.; Nowak, Radoslaw P.; Loehr, Amanda; Bahcall, Magda; Fischer, Eric S.; Janne, Pasi A.; George, Rani E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4249-4255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the development of the first small mol. degraders that can induce anaplastic lymphoma kinase (ALK) degrdn., including in non-small-cell lung cancer (NSCLC), anaplastic large-cell lymphoma (ALCL), and neuroblastoma (NB) cell lines.  These degraders were developed through conjugation of known pyrimidine-based ALK inhibitors, TAE684 or LDK378, and the cereblon ligand pomalidomide.  We demonstrate that in some cell types degrader potency is compromised by expression of drug transporter ABCB1.  In addn., proteomic profiling demonstrated that these compds. also promote the degrdn. of addnl. kinases including PTK2 (FAK), Aurora A, FER, and RPS6KA1 (RSK1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mDH7Sj8EqbVg90H21EOLACvtfcHk0liLCx8FGAOQIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D&md5=486db11b3fbc2283700f38b09cc39bd7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01655%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DC.%2BE.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DLoehr%26aufirst%3DA.%26aulast%3DBahcall%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGeorge%26aufirst%3DR.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4249%26epage%3D4255%26doi%3D10.1021%2Facs.jmedchem.7b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A chemoproteomic approach to query the degradable kinome using a multi-kinase degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1016%2Fj.chembiol.2017.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=29129717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=88-99&author=H.-T.+Huangauthor=D.+Dobrovolskyauthor=J.+Paulkauthor=G.+Yangauthor=E.+L.+Weisbergauthor=Z.+M.+Doctorauthor=D.+L.+Buckleyauthor=J.-H.+Choauthor=E.+Koauthor=J.+Jangauthor=K.+Shiauthor=H.+G.+Choiauthor=J.+D.+Griffinauthor=Y.+Liauthor=S.+P.+Treonauthor=E.+S.+Fischerauthor=J.+E.+Bradnerauthor=L.+Tanauthor=N.+S.+Gray&title=A+chemoproteomic+approach+to+query+the+degradable+kinome+using+a+multi-kinase+degrader&doi=10.1016%2Fj.chembiol.2017.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span></div><div class="casAuthors">Huang, Hai-Tsang; Dobrovolsky, Dennis; Paulk, Joshiawa; Yang, Guang; Weisberg, Ellen L.; Doctor, Zainab M.; Buckley, Dennis L.; Cho, Joong-Heui; Ko, Eunhwa; Jang, Jaebong; Shi, Kun; Choi, Hwan Geun; Griffin, James D.; Li, Ying; Treon, Steven P.; Fischer, Eric S.; Bradner, James E.; Tan, Li; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heterobifunctional mols. that recruit E3 ubiquitin ligases, such as cereblon, for targeted protein degrdn. represent an emerging pharmacol. strategy.  A major unanswered question is how generally applicable this strategy is to all protein targets.  In this study, we designed a multi-kinase degrader by conjugating a highly promiscuous kinase inhibitor with a cereblon-binding ligand, and used quant. proteomics to discover 28 kinases, including BTK, PTK2, PTK2B, FLT3, AURKA, AURKB, TEC, ULK1, ITK, and nine members of the CDK family, as degradable.  This set of kinases is only a fraction of the intracellular targets bound by the degrader, demonstrating that successful degrdn. requires more than target engagement.  The results guided us to develop selective degraders for FLT3 and BTK, with potentials to improve disease treatment.  Together, this study demonstrates an efficient approach to triage a gene family of interest to identify readily degradable targets for further studies and pre-clin. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYNxhoPV3127Vg90H21EOLACvtfcHk0liLCx8FGAOQIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO&md5=703cfbaca3edd24176472158563f895f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.-T.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.-H.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520chemoproteomic%2520approach%2520to%2520query%2520the%2520degradable%2520kinome%2520using%2520a%2520multi-kinase%2520degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D88%26epage%3D99%26doi%3D10.1016%2Fj.chembiol.2017.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Weck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castelletti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gampa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sovath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caothien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caushi, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortés-Cros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaunay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desplat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwoske, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldridge, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farsidjani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flemming, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattenberger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapoor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollé, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phadke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repko, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schouwey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucke, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiedt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varadarajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Project DRIVE: a compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1016%2Fj.cell.2017.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=28753431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=577-592&author=E.+R.+McDonaldauthor=A.+de%0AWeckauthor=M.+R.+Schlabachauthor=E.+Billyauthor=K.+J.+Mavrakisauthor=G.+R.+Hoffmanauthor=D.+Belurauthor=D.+Castellettiauthor=E.+Friasauthor=K.+Gampaauthor=J.+Goljiauthor=I.+Kaoauthor=L.+Liauthor=P.+Megelauthor=T.+A.+Perkinsauthor=N.+Ramadanauthor=D.+A.+Ruddyauthor=S.+J.+Silverauthor=S.+Sovathauthor=M.+Stumpauthor=O.+Weberauthor=R.+Widmerauthor=J.+Yuauthor=K.+Yuauthor=Y.+Yueauthor=D.+Abramowskiauthor=E.+Ackleyauthor=R.+Barrettauthor=J.+Bergerauthor=J.+L.+Bernardauthor=R.+Billigauthor=S.+M.+Brachmannauthor=F.+Buxtonauthor=R.+Caothienauthor=J.+X.+Caushiauthor=F.+S.+Chungauthor=M.+Cort%C3%A9s-Crosauthor=R.+S.+deBeaumontauthor=C.+Delaunayauthor=A.+Desplatauthor=W.+Duongauthor=D.+A.+Dwoskeauthor=R.+S.+Eldridgeauthor=A.+Farsidjaniauthor=F.+Fengauthor=J.+Fengauthor=D.+Flemmingauthor=W.+Forresterauthor=G.+G.+Galliauthor=Z.+Gaoauthor=F.+Gauterauthor=V.+Gibajaauthor=K.+Haasauthor=M.+Hattenbergerauthor=T.+Hoodauthor=K.+E.+Hurovauthor=Z.+Jaganiauthor=M.+Jenalauthor=J.+A.+Johnsonauthor=M.+D.+Jonesauthor=A.+Kapoorauthor=J.+Kornauthor=J.+Liuauthor=Q.+Liuauthor=S.+Liuauthor=Y.+Liuauthor=A.+T.+Looauthor=K.+J.+Macchiauthor=T.+Martinauthor=G.+McAllisterauthor=A.+Meyerauthor=S.+Molleauthor=R.+A.+Moll%C3%A9author=T.+Phadkeauthor=B.+Repkoauthor=T.+Schouweyauthor=F.+Shanahanauthor=Q.+Shenauthor=C.+Stammauthor=C.+Stephanauthor=V.+M.+Stuckeauthor=R.+Tiedtauthor=M.+Varadarajanauthor=K.+Venkatesanauthor=A.+C.+Vitariauthor=M.+Wallrothauthor=J.+Weilerauthor=J.+Zhangauthor=C.+Mickaninauthor=V.+E.+Myerauthor=J.+A.+Porterauthor=A.+Laiauthor=H.+Bitterauthor=E.+Leesauthor=N.+Keenauthor=A.+Kauffmannauthor=F.+Stegmeierauthor=F.+Hofmannauthor=T.+Schmelzleauthor=W.+R.+Sellers&title=Project+DRIVE%3A+a+compendium+of+cancer+dependencies+and+synthetic+lethal+relationships+uncovered+by+large-scale%2C+deep+RNAi+screening&doi=10.1016%2Fj.cell.2017.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Project DRIVE: A compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span></div><div class="casAuthors">McDonald, E. Robert, III; de Weck, Antoine; Schlabach, Michael R.; Billy, Eric; Mavrakis, Konstantinos J.; Hoffman, Gregory R.; Belur, Dhiren; Castelletti, Deborah; Frias, Elizabeth; Gampa, Kalyani; Golji, Javad; Kao, Iris; Li, Li; Megel, Philippe; Perkins, Thomas A.; Ramadan, Nadire; Ruddy, David A.; Silver, Serena J.; Sovath, Sosathya; Stump, Mark; Weber, Odile; Widmer, Roland; Yu, Jianjun; Yu, Kristine; Yue, Yingzi; Abramowski, Dorothee; Ackley, Elizabeth; Barrett, Rosemary; Berger, Joel; Bernard, Julie L.; Billig, Rebecca; Brachmann, Saskia M.; Buxton, Frank; Caothien, Roger; Caushi, Justina X.; Chung, Franklin S.; Cortes-Cros, Marta; de Beaumont, Rosalie S.; Delaunay, Clara; Desplat, Aurore; Duong, William; Dwoske, Donald A.; Eldridge, Richard S.; Farsidjani, Ali; Feng, Fei; Feng, Jia Jia; Flemming, Daisy; Forrester, William; Galli, Giorgio G.; Gao, Zhenhai; Gauter, Francois; Gibaja, Veronica; Haas, Kristy; Hattenberger, Marc; Hood, Tami; Hurov, Kristen E.; Jagani, Zainab; Jenal, Mathias; Johnson, Jennifer A.; Jones, Michael D.; Kapoor, Avnish; Korn, Joshua; Liu, Jilin; Liu, Qiumei; Liu, Shumei; Liu, Yue; Loo, Alice T.; Macchi, Kaitlin J.; Martin, Typhaine; McAllister, Gregory; Meyer, Amandine; Molle, Sandra; Pagliarini, Raymond A.; Phadke, Tanushree; Repko, Brian; Schouwey, Tanja; Shanahan, Frances; Shen, Qiong; Stamm, Christelle; Stephan, Christine; Stucke, Volker M.; Tiedt, Ralph; Varadarajan, Malini; Venkatesan, Kavitha; Vitari, Alberto C.; Wallroth, Marco; Weiler, Jan; Zhang, Jing; Mickanin, Craig; Myer, Vic E.; Porter, Jeffery A.; Lai, Albert; Bitter, Hans; Lees, Emma; Keen, Nicholas; Kauffmann, Audrey; Stegmeier, Frank; Hofmann, Francesco; Schmelzle, Tobias; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">577-592.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elucidation of the mutational landscape of human cancer has progressed rapidly and been accompanied by the development of therapeutics targeting mutant oncogenes.  However, a comprehensive mapping of cancer dependencies has lagged behind and the discovery of therapeutic targets for counteracting tumor suppressor gene loss is needed.  To identify vulnerabilities relevant to specific cancer subtypes, we conducted a large-scale RNAi screen in which viability effects of mRNA knockdown were assessed for 7837 genes using an av. of 20 shRNAs per gene in 398 cancer cell lines.  We describe findings of this screen, outlining the classes of cancer dependency genes and their relationships to genetic, expression, and lineage features.  In addn., we describe robust gene-interaction networks recapitulating both protein complexes and functional cooperation among complexes and pathways.  This dataset along with a web portal is provided to the community to assist in the discovery and translation of new therapeutic approaches for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokUs33_s2nC7Vg90H21EOLACvtfcHk0liF4ViQ5I6Iew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI&md5=41592e684cc8f2cd28117cb3303f6120</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3Dde%2BWeck%26aufirst%3DA.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DMavrakis%26aufirst%3DK.%2BJ.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DBelur%26aufirst%3DD.%26aulast%3DCastelletti%26aufirst%3DD.%26aulast%3DFrias%26aufirst%3DE.%26aulast%3DGampa%26aufirst%3DK.%26aulast%3DGolji%26aufirst%3DJ.%26aulast%3DKao%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMegel%26aufirst%3DP.%26aulast%3DPerkins%26aufirst%3DT.%2BA.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DSilver%26aufirst%3DS.%2BJ.%26aulast%3DSovath%26aufirst%3DS.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DWidmer%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DAckley%26aufirst%3DE.%26aulast%3DBarrett%26aufirst%3DR.%26aulast%3DBerger%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DJ.%2BL.%26aulast%3DBillig%26aufirst%3DR.%26aulast%3DBrachmann%26aufirst%3DS.%2BM.%26aulast%3DBuxton%26aufirst%3DF.%26aulast%3DCaothien%26aufirst%3DR.%26aulast%3DCaushi%26aufirst%3DJ.%2BX.%26aulast%3DChung%26aufirst%3DF.%2BS.%26aulast%3DCort%25C3%25A9s-Cros%26aufirst%3DM.%26aulast%3DdeBeaumont%26aufirst%3DR.%2BS.%26aulast%3DDelaunay%26aufirst%3DC.%26aulast%3DDesplat%26aufirst%3DA.%26aulast%3DDuong%26aufirst%3DW.%26aulast%3DDwoske%26aufirst%3DD.%2BA.%26aulast%3DEldridge%26aufirst%3DR.%2BS.%26aulast%3DFarsidjani%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFlemming%26aufirst%3DD.%26aulast%3DForrester%26aufirst%3DW.%26aulast%3DGalli%26aufirst%3DG.%2BG.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DGauter%26aufirst%3DF.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DHattenberger%26aufirst%3DM.%26aulast%3DHood%26aufirst%3DT.%26aulast%3DHurov%26aufirst%3DK.%2BE.%26aulast%3DJagani%26aufirst%3DZ.%26aulast%3DJenal%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DKapoor%26aufirst%3DA.%26aulast%3DKorn%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLoo%26aufirst%3DA.%2BT.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DMeyer%26aufirst%3DA.%26aulast%3DMolle%26aufirst%3DS.%26aulast%3DMoll%25C3%25A9%26aufirst%3DR.%2BA.%26aulast%3DPhadke%26aufirst%3DT.%26aulast%3DRepko%26aufirst%3DB.%26aulast%3DSchouwey%26aufirst%3DT.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DStephan%26aufirst%3DC.%26aulast%3DStucke%26aufirst%3DV.%2BM.%26aulast%3DTiedt%26aufirst%3DR.%26aulast%3DVaradarajan%26aufirst%3DM.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DWallroth%26aufirst%3DM.%26aulast%3DWeiler%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DKauffmann%26aufirst%3DA.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DProject%2520DRIVE%253A%2520a%2520compendium%2520of%2520cancer%2520dependencies%2520and%2520synthetic%2520lethal%2520relationships%2520uncovered%2520by%2520large-scale%252C%2520deep%2520RNAi%2520screening%26jtitle%3DCell%26date%3D2017%26volume%3D170%26spage%3D577%26epage%3D592%26doi%3D10.1016%2Fj.cell.2017.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lgvZbNXZjLVlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&author=E.+S.+Fischerauthor=K.+B%C3%B6hmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thom%C3%A4&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0lgvZbNXZjLVlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DB%25C3%25B6hm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0liEGV5wmbmkAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0liEGV5wmbmkAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Group-Based Optimization of Potent and Cell-Active Inhibitors of the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase: Structure-Activity Relationships Leading to the Chemical Probe (2S,4R)-1-((S)-2-(1-Cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00675</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00675" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=599-618&author=P.+Soaresauthor=M.+S.+Gaddauthor=J.+Frostauthor=C.+Galdeanoauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Rochaauthor=K.+D.+Readauthor=A.+Ciulli&title=Group-Based+Optimization+of+Potent+and+Cell-Active+Inhibitors+of+the+von+Hippel-Lindau+%28VHL%29+E3+Ubiquitin+Ligase%3A+Structure-Activity+Relationships+Leading+to+the+Chemical+Probe+%282S%2C4R%29-1-%28%28S%29-2-%281-Cyanocyclopropanecarboxamido%29-3%2C3-dimethylbutanoyl%29-4-hydroxy-N-%284-%284-methylthiazol-5-yl%29benzyl%29pyrrolidine-2-carboxamide+%28VH298%29&doi=10.1021%2Facs.jmedchem.7b00675"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00675%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DGroup-Based%2520Optimization%2520of%2520Potent%2520and%2520Cell-Active%2520Inhibitors%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520Ubiquitin%2520Ligase%253A%2520Structure-Activity%2520Relationships%2520Leading%2520to%2520the%2520Chemical%2520Probe%2520%25282S%252C4R%2529-1-%2528%2528S%2529-2-%25281-Cyanocyclopropanecarboxamido%2529-3%252C3-dimethylbutanoyl%2529-4-hydroxy-N-%25284-%25284-methylthiazol-5-yl%2529benzyl%2529pyrrolidine-2-carboxamide%2520%2528VH298%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D61%26spage%3D599%26epage%3D618%26doi%3D10.1021%2Facs.jmedchem.7b00675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzes, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Potent and selective chemical probe of hypoxic signalling downstream of HIF-alpha hydroxylation via VHL inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13312</span>, <span class="refDoi"> DOI: 10.1038/ncomms13312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fncomms13312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=27811928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13312&author=J.+Frostauthor=C.+Galdeanoauthor=P.+Soaresauthor=M.+S.+Gaddauthor=K.+M.+Grzesauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Shimamuraauthor=M.+Bantscheffauthor=P.+Grandiauthor=K.+D.+Readauthor=D.+A.+Cantrellauthor=S.+Rochaauthor=A.+Ciulli&title=Potent+and+selective+chemical+probe+of+hypoxic+signalling+downstream+of+HIF-alpha+hydroxylation+via+VHL+inhibition&doi=10.1038%2Fncomms13312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span></div><div class="casAuthors">Frost, Julianty; Galdeano, Carles; Soares, Pedro; Gadd, Morgan S.; Grzes, Katarzyna M.; Ellis, Lucy; Epemolu, Ola; Shimamura, Satoko; Bantscheff, Marcus; Grandi, Paola; Read, Kevin D.; Cantrell, Doreen A.; Rocha, Sonia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13312</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. strategies to using small mols. to stimulate hypoxia inducible factors (HIFs) activity and trigger a hypoxic response under normoxic conditions, such as iron chelators and inhibitors of prolyl hydroxylase domain (PHD) enzymes, have broad-spectrum activities and off-target effects.  Here we disclose VH298, a potent VHL inhibitor that stabilizes HIF-α and elicits a hypoxic response via a different mechanism, that is the blockade of the VHL:HIF-α protein-protein interaction downstream of HIF-α hydroxylation by PHD enzymes.  We show that VH298 engages with high affinity and specificity with VHL as its only major cellular target, leading to selective on-target accumulation of hydroxylated HIF-α in a concn.- and time-dependent fashion in different cell lines, with subsequent upregulation of HIF-target genes at both mRNA and protein levels.  VH298 represents a high-quality chem. probe of the HIF signalling cascade and an attractive starting point to the development of potential new therapeutics targeting hypoxia signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLB1WvVF4EFLVg90H21EOLACvtfcHk0lg1gVt0cjHTHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ&md5=7a884b6193b3ca5e119fecea2eb31d3e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fncomms13312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13312%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DGrzes%26aufirst%3DK.%2BM.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCantrell%26aufirst%3DD.%2BA.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DPotent%2520and%2520selective%2520chemical%2520probe%2520of%2520hypoxic%2520signalling%2520downstream%2520of%2520HIF-alpha%2520hydroxylation%2520via%2520VHL%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13312%26doi%3D10.1038%2Fncomms13312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+NEDD8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0lg1gVt0cjHTHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520NEDD8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saeki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanigawara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komano, T.</span></span> <span> </span><span class="NLM_article-title">Human P-glycoprotein transports cyclosporin A and FK506</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">6077</span>– <span class="NLM_lpage">6080</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1993&pages=6077-6080&author=T.+Saekiauthor=K.+Uedaauthor=Y.+Tanigawaraauthor=R.+Horiauthor=T.+Komano&title=Human+P-glycoprotein+transports+cyclosporin+A+and+FK506"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSaeki%26aufirst%3DT.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DTanigawara%26aufirst%3DY.%26aulast%3DHori%26aufirst%3DR.%26aulast%3DKomano%26aufirst%3DT.%26atitle%3DHuman%2520P-glycoprotein%2520transports%2520cyclosporin%2520A%2520and%2520FK506%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1993%26volume%3D268%26spage%3D6077%26epage%3D6080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chessum, N. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqua, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilding, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span> <span> </span><span class="NLM_article-title">Demonstrating In-cell target engagement using a pirin protein degradation probe (CCT367766)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">918</span>– <span class="NLM_lpage">933</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOqsrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=918-933&author=N.+E.+A.+Chessumauthor=S.+Y.+Sharpauthor=J.+J.+Caldwellauthor=A.+E.+Pasquaauthor=B.+Wildingauthor=G.+Colombanoauthor=I.+Collinsauthor=B.+Ozerauthor=M.+Richardsauthor=M.+Rowlandsauthor=M.+Stubbsauthor=R.+Burkeauthor=P.+C.+McAndrewauthor=P.+A.+Clarkeauthor=P.+Workmanauthor=M.+D.+Cheesemanauthor=K.+Jones&title=Demonstrating+In-cell+target+engagement+using+a+pirin+protein+degradation+probe+%28CCT367766%29&doi=10.1021%2Facs.jmedchem.7b01406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Demonstrating In-Cell Target Engagement Using a Pirin Protein Degradation Probe (CCT367766)</span></div><div class="casAuthors">Chessum, Nicola E. A.; Sharp, Swee Y.; Caldwell, John J.; Pasqua, A. Elisa; Wilding, Birgit; Colombano, Giampiero; Collins, Ian; Ozer, Bugra; Richards, Meirion; Rowlands, Martin; Stubbs, Mark; Burke, Rosemary; McAndrew, P. Craig; Clarke, Paul A.; Workman, Paul; Cheeseman, Matthew D.; Jones, Keith</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">918-933</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Demonstrating intracellular protein target engagement is an essential step in the development and progression of new chem. probes and potential small mol. therapeutics.  However, this can be particularly challenging for poorly studied and noncatalytic proteins, as robust proximal biomarkers are rarely known.  To confirm that the recently discovered chem. probe I (CCT251236) binds the putative transcription factor regulator pirin in living cells, the authors developed a heterobifunctional protein degrdn. probe.  Focusing on linker design and physicochem. properties, the authors generated a highly active probe II (CCT367766) in only three iterations, validating the efficient strategy for degrdn. probe design against nonvalidated protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofmMArlN0aWrVg90H21EOLACvtfcHk0lg1gVt0cjHTHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOqsrzK&md5=928d24741c26d0b66ab7ce1ef2004bbc</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01406%26sid%3Dliteratum%253Aachs%26aulast%3DChessum%26aufirst%3DN.%2BE.%2BA.%26aulast%3DSharp%26aufirst%3DS.%2BY.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DPasqua%26aufirst%3DA.%2BE.%26aulast%3DWilding%26aufirst%3DB.%26aulast%3DColombano%26aufirst%3DG.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DOzer%26aufirst%3DB.%26aulast%3DRichards%26aufirst%3DM.%26aulast%3DRowlands%26aufirst%3DM.%26aulast%3DStubbs%26aufirst%3DM.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DMcAndrew%26aufirst%3DP.%2BC.%26aulast%3DClarke%26aufirst%3DP.%2BA.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DDemonstrating%2520In-cell%2520target%2520engagement%2520using%2520a%2520pirin%2520protein%2520degradation%2520probe%2520%2528CCT367766%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D918%26epage%3D933%26doi%3D10.1021%2Facs.jmedchem.7b01406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sizemore, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharia, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubonja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strickland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawes, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhivich, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyatt, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span> <span> </span><span class="NLM_article-title">Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1779</span>– <span class="NLM_lpage">1784</span>, <span class="refDoi"> DOI: 10.1038/ng.3984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1038%2Fng.3984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=29083409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslehtrjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=1779-1784&author=R.+M.+Meyersauthor=J.+G.+Bryanauthor=J.+M.+McFarlandauthor=B.+A.+Weirauthor=A.+E.+Sizemoreauthor=H.+Xuauthor=N.+V.+Dhariaauthor=P.+G.+Montgomeryauthor=G.+S.+Cowleyauthor=S.+Pantelauthor=A.+Goodaleauthor=Y.+Leeauthor=L.+D.+Aliauthor=G.+Jiangauthor=R.+Lubonjaauthor=W.+F.+Harringtonauthor=M.+Stricklandauthor=T.+Wuauthor=D.+C.+Hawesauthor=V.+A.+Zhivichauthor=M.+R.+Wyattauthor=Z.+Kalaniauthor=J.+J.+Changauthor=M.+Okamotoauthor=K.+Stegmaierauthor=T.+R.+Golubauthor=J.+S.+Boehmauthor=F.+Vazquezauthor=D.+E.+Rootauthor=W.+C.+Hahnauthor=A.+Tsherniak&title=Computational+correction+of+copy+number+effect+improves+specificity+of+CRISPR-Cas9+essentiality+screens+in+cancer+cells&doi=10.1038%2Fng.3984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells</span></div><div class="casAuthors">Meyers, Robin M.; Bryan, Jordan G.; McFarland, James M.; Weir, Barbara A.; Sizemore, Ann E.; Xu, Han; Dharia, Neekesh V.; Montgomery, Phillip G.; Cowley, Glenn S.; Pantel, Sasha; Goodale, Amy; Ali, Levi D.; Jiang, Guozhi; Lubonja, Rakela; Harrington, William F.; Strickland, Matthew; Wu, Ting; Hawes, Derek C.; Zhivich, Victor A.; Wyatt, Meghan R.; Kalani, Zohra; Chang, Jaime J.; Okamoto, Michael; Stegmaier, Kimberly; Golub, Todd R.; Boehm, Jesse S.; Vazquez, Francisca; Root, David E.; Hahn, William C.; Tsherniak, Aviad</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1779-1784</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The CRISPR-Cas9 system has revolutionized gene editing both at single genes and in multiplexed loss-of-function screens, thus enabling precise genome-scale identification of genes essential for proliferation and survival of cancer cells.  However, previous studies have reported that a gene-independent antiproliferative effect of Cas9-mediated DNA cleavage confounds such measurement of genetic dependency, thereby leading to false-pos. results in copy no.-amplified regions.  We developed CERES, a computational method to est. gene-dependency levels from CRISPR-Cas9 essentiality screens while accounting for the copy no.-specific effect.  In our efforts to define a cancer dependency map, we performed genome-scale CRISPR-Cas9 essentiality screens across 342 cancer cell lines and applied CERES to this data set.  We found that CERES decreased false-pos. results and estd. sgRNA activity for both this data set and previously published screens performed with different sgRNA libraries.  We further demonstrate the utility of this collection of screens, after CERES correction, for identifying cancer-type-specific vulnerabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8iODR9NtDbVg90H21EOLACvtfcHk0liTbqTgvs_b9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslehtrjI&md5=539e8fb16958480ce2d430a7f8e6cddf</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fng.3984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3984%26sid%3Dliteratum%253Aachs%26aulast%3DMeyers%26aufirst%3DR.%2BM.%26aulast%3DBryan%26aufirst%3DJ.%2BG.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DSizemore%26aufirst%3DA.%2BE.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DDharia%26aufirst%3DN.%2BV.%26aulast%3DMontgomery%26aufirst%3DP.%2BG.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DPantel%26aufirst%3DS.%26aulast%3DGoodale%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DAli%26aufirst%3DL.%2BD.%26aulast%3DJiang%26aufirst%3DG.%26aulast%3DLubonja%26aufirst%3DR.%26aulast%3DHarrington%26aufirst%3DW.%2BF.%26aulast%3DStrickland%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DHawes%26aufirst%3DD.%2BC.%26aulast%3DZhivich%26aufirst%3DV.%2BA.%26aulast%3DWyatt%26aufirst%3DM.%2BR.%26aulast%3DKalani%26aufirst%3DZ.%26aulast%3DChang%26aufirst%3DJ.%2BJ.%26aulast%3DOkamoto%26aufirst%3DM.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DTsherniak%26aufirst%3DA.%26atitle%3DComputational%2520correction%2520of%2520copy%2520number%2520effect%2520improves%2520specificity%2520of%2520CRISPR-Cas9%2520essentiality%2520screens%2520in%2520cancer%2520cells%26jtitle%3DNat.%2520Genet.%26date%3D2017%26volume%3D49%26spage%3D1779%26epage%3D1784%26doi%3D10.1038%2Fng.3984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stadtmueller, H.</span>; <span class="NLM_string-name">Sapountzis, I.</span></span> <span> </span><span class="NLM_article-title">Preparation of Pyrimidinamine Derivatives for Treating Diseases Characterized by Excessive or Abnormal Cell Proliferation</span>. Int Pat Appl WO <span class="NLM_patent">2010/058032</span>, May 27, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=H.+Stadtmueller&author=I.+Sapountzis&title=Preparation+of+Pyrimidinamine+Derivatives+for+Treating+Diseases+Characterized+by+Excessive+or+Abnormal+Cell+Proliferation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DStadtmueller%26aufirst%3DH.%26atitle%3DPreparation%2520of%2520Pyrimidinamine%2520Derivatives%2520for%2520Treating%2520Diseases%2520Characterized%2520by%2520Excessive%2520or%2520Abnormal%2520Cell%2520Proliferation%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, scaling, space-group assignment and post-refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+scaling%2C+space-group+assignment+and+post-refinement&doi=10.1107%2Fs0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0liTbqTgvs_b9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520scaling%252C%2520space-group%2520assignment%2520and%2520post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2Fs0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span>; <span class="NLM_string-name">Winn, M.</span>; <span class="NLM_string-name">Ralph, A.</span></span> <span> </span><span class="NLM_article-title">Collaborative Computational Project, Number 4: Providing Programs for Protein Crystallography</span>.  <i>Methods in Enzymology</i>; <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_year">1997</span>; Vol.  <span class="NLM_volume">277</span>, pp  <span class="NLM_fpage">620</span>– <span class="NLM_lpage">633</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=620-633&author=E.+J.+Dodson&author=M.+Winn&author=A.+Ralph&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DCollaborative%2520Computational%2520Project%252C%2520Number%25204%253A%2520Providing%2520Programs%2520for%2520Protein%2520Crystallography%26btitle%3DMethods%2520in%2520Enzymology%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D277%26spage%3D620%26epage%3D633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lh_Y_y9T7zgPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>Buster</i>, version 2.11.5; <span class="NLM_publisher-name">Global Phasing Ltd</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+S.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=Buster"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBuster%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2009&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Z','PDB','6I8Z'); return false;">PDB: 6I8Z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ETM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ETM','PDB','2ETM'); return false;">PDB: 2ETM</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i36"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01826">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_14506"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01826">10.1021/acs.jmedchem.8b01826</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional PTK2 degradation data for BI-0319 and BI-3663 in HCC cell lines (SNU387, HUH-1, Hep3B2.1-7, HepG2, SK-Hep1, HLF, SNU-398, HUCCT1, HLE, HuH7, and SNU-423); alamarBlue cell viability assays in above HCC cell lines with BI-3663, BI-0319, BI-4464, and doxorubicine (positive control); long-term proliferation assays in HSC-3, C32, CFPAC-1, and A549 with 3 μM of BI-3663 or BI-0319; CRISPR validation of PTK2 dependency, solubility, plasma protein binding, and Caco2 permeability assay data for PROTACs BI-0319 and BI-3663; X-ray data collection and refinement statistics; supplementary chemistry information: MS parameters, HPLC chromatograms and NMR spectrum of BI-3663; supplemental methods: solubility, PPB, CACO2 permeability, stability measurement, and CRISPR depletion of PTK2 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_002.csv">CSV</a>)</p></li><li><p class="inline">Proteomics raw data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_003.xlsx">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_001.pdf">jm8b01826_si_001.pdf (1.04 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_002.csv">jm8b01826_si_002.csv (0.58 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01826/suppl_file/jm8b01826_si_003.xlsx">jm8b01826_si_003.xlsx (13.22 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for PTK2: BI-4464 have been deposited to the Protein Data Bank (PDB) under accession number <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Z">6I8Z</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01826%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-5%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01826" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                1MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67992d643cc43c8e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
