
<html lang="en"     class="pb-page"  data-request-id="d530cabf-a466-439a-a59f-3646621ef6fd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.1c00958;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Guided Design of a “Bump-and-Hole” Bromodomain-Based Degradation Tag" /></meta><meta name="dc.Creator" content="Radosław P.  Nowak" /></meta><meta name="dc.Creator" content="Yuan  Xiong" /></meta><meta name="dc.Creator" content="Nadia  Kirmani" /></meta><meta name="dc.Creator" content="Joann  Kalabathula" /></meta><meta name="dc.Creator" content="Katherine A.  Donovan" /></meta><meta name="dc.Creator" content="Nicholas A.  Eleuteri" /></meta><meta name="dc.Creator" content="J. Christine  Yuan" /></meta><meta name="dc.Creator" content="Eric S.  Fischer" /></meta><meta name="dc.Description" content="Chemical biology tools to modulate protein levels in cells are critical to decipher complex biology. Targeted protein degradation offers the potential for rapid and dose-dependent protein depletion..." /></meta><meta name="Description" content="Chemical biology tools to modulate protein levels in cells are critical to decipher complex biology. Targeted protein degradation offers the potential for rapid and dose-dependent protein depletion..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 19, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00958" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00958" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00958" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00958" /></link>
        
    
    

<title>Structure-Guided Design of a “Bump-and-Hole” Bromodomain-Based Degradation Tag | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00958" /></meta><meta property="og:title" content="Structure-Guided Design of a “Bump-and-Hole” Bromodomain-Based Degradation Tag" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0007.jpeg" /></meta><meta property="og:description" content="Chemical biology tools to modulate protein levels in cells are critical to decipher complex biology. Targeted protein degradation offers the potential for rapid and dose-dependent protein depletion through the use of protein fusion tags toward which protein degraders have been established. Here, we present a newly developed protein degradation tag BRD4BD1L94V along with the corresponding cereblon (CRBN)-based heterobifunctional degrader based on a “bump-and-hole” approach. The resulting compound XY-06-007 shows a half-degradation concentration (DC50, 6 h) of 10 nM against BRD4BD1L94V with no degradation of off-targets, as assessed by whole proteome mass spectrometry, and demonstrates suitable pharmacokinetics for in vivo studies. We demonstrate that BRD4BD1L94V can be combined with the dTAG approach to achieve simultaneous degrader-mediated depletion of their respective protein fusions. This orthogonal system complements currently available protein degradation tags and enables investigation into the consequences resulting from rapid degradation of previously undruggable disease codependencies." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00958"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00958">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00958&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00958&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00958&amp;href=/doi/10.1021/acs.jmedchem.1c00958" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00890" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00219" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Guided Design of a “Bump-and-Hole” Bromodomain-Based Degradation Tag</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Radosław P. Nowak</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Radosław P. Nowak</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rados%C5%82aw+P.++Nowak">Radosław P. Nowak</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuan Xiong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuan Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuan++Xiong">Yuan Xiong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nadia Kirmani</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nadia Kirmani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nadia++Kirmani">Nadia Kirmani</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joann Kalabathula</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joann Kalabathula</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joann++Kalabathula">Joann Kalabathula</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katherine A. Donovan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katherine A. Donovan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katherine+A.++Donovan">Katherine A. Donovan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-8539-5106" title="Orcid link">https://orcid.org/0000-0002-8539-5106</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicholas A. Eleuteri</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas A. Eleuteri</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas+A.++Eleuteri">Nicholas A. Eleuteri</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">J. Christine Yuan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">J. Christine Yuan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J.+Christine++Yuan">J. Christine Yuan</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Eric S. Fischer</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eric S. Fischer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#bffacdd6dce0f9d6ccdcd7dacdffdbd9dcd691d7decdc9decddb91dadbca"><span class="__cf_email__" data-cfemail="490c3b202a160f203a2a212c3b092d2f2a206721283b3f283b2d672c2d3c">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Eric+S.++Fischer">Eric S. Fischer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7337-6306" title="Orcid link">https://orcid.org/0000-0001-7337-6306</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00958&amp;href=/doi/10.1021%2Facs.jmedchem.1c00958" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 19, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 May 2021</li><li><span class="item_label"><b>Published</b> online</span>19 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00958" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00958</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRados%25C5%2582aw%2BP.%2BNowak%252C%2BYuan%2BXiong%252C%2BNadia%2BKirmani%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c00958%26title%3DStructure-Guided%2BDesign%2Bof%2Ba%2B%25E2%2580%259CBump-and-Hole%25E2%2580%259D%2BBromodomain-Based%2BDegradation%2BTag%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D14%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00958"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">889</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00958" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Guided Design of a “Bump-and-Hole” Bromodomain-Based Degradation Tag&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Radosław&quot;,&quot;last_name&quot;:&quot;P. Nowak&quot;},{&quot;first_name&quot;:&quot;Yuan&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Nadia&quot;,&quot;last_name&quot;:&quot;Kirmani&quot;},{&quot;first_name&quot;:&quot;Joann&quot;,&quot;last_name&quot;:&quot;Kalabathula&quot;},{&quot;first_name&quot;:&quot;Katherine&quot;,&quot;last_name&quot;:&quot;A. Donovan&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;A. Eleuteri&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Christine Yuan&quot;},{&quot;first_name&quot;:&quot;Eric&quot;,&quot;last_name&quot;:&quot;S. Fischer&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00958&quot;},&quot;abstract&quot;:&quot;Chemical biology tools to modulate protein levels in cells are critical to decipher complex biology. Targeted protein degradation offers the potential for rapid and dose-dependent protein depletion through the use of protein fusion tags toward which protein degraders have been established. Here, we present a newly developed protein degradation tag BRD4BD1L94V along with the corresponding cereblon (CRBN)-based heterobifunctional degrader based on a “bump-and-hole” approach. The resulting compound XY-06-007 shows a half-degradation concentration (DC50, 6 h) of 10 nM against BRD4BD1L94V with no degradation of off-targets, as assessed by whole proteome mass spectrometry, and demonstrates suitable pharmacokinetics for in vivo studies. We demonstrate that BRD4BD1L94V can be combined with the dTAG approach to achieve simultaneous degrader-mediated depletion of their respective protein fusions. This orthogonal system complements currently available protein degradation tags and enables investigation into the consequen&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00958&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00958" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00958&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00958" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00958&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00958" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00958&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00958&amp;href=/doi/10.1021/acs.jmedchem.1c00958" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00958" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00958" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00958%26sid%3Dliteratum%253Aachs%26pmid%3D34279939%26genre%3Darticle%26aulast%3DNowak%26date%3D2021%26atitle%3DStructure-Guided%2BDesign%2Bof%2Ba%2B%25E2%2580%259CBump-and-Hole%25E2%2580%259D%2BBromodomain-Based%2BDegradation%2BTag%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Chemical biology tools to modulate protein levels in cells are critical to decipher complex biology. Targeted protein degradation offers the potential for rapid and dose-dependent protein depletion through the use of protein fusion tags toward which protein degraders have been established. Here, we present a newly developed protein degradation tag BRD4<sub>BD1</sub>L94V along with the corresponding cereblon (CRBN)-based heterobifunctional degrader based on a “bump-and-hole” approach. The resulting compound <b>XY-06-007</b> shows a half-degradation concentration (DC<sub>50, 6 h</sub>) of 10 nM against BRD4<sub>BD1</sub>L94V with no degradation of off-targets, as assessed by whole proteome mass spectrometry, and demonstrates suitable pharmacokinetics for <i>in vivo</i> studies. We demonstrate that BRD4<sub>BD1</sub>L94V can be combined with the dTAG approach to achieve simultaneous degrader-mediated depletion of their respective protein fusions. This orthogonal system complements currently available protein degradation tags and enables investigation into the consequences resulting from rapid degradation of previously undruggable disease codependencies.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86515" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86515" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Targeted protein degradation has emerged as an exciting avenue in chemical biology and presents promising opportunities for clinical development. While selective degradation of desired proteins can be achieved through the development of small molecule degraders, this approach may not always be possible or feasible as some proteins lack binding pockets or established ligands to enable facile development of heterobifunctional degraders (also referred to as PROTAC for PRoteolysis TArgeting Chimera)<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> or fall far from the degron space accessible by monovalent degraders such as immunomodulatory drug (IMiD) analogues.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Even for targets with known active-site binders, developing degrader<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> tool compounds can still be a lengthy process, causing researchers to often resort to validating biological hypotheses by modulation of protein levels through methods such as small interfering RNA (siRNA) or CRISPR-Cas9. While these approaches are highly valuable, they have their limitations. Genetic approaches typically take 24–72 h to reach the full protein depletion effect, allowing cells sufficient time to adapt to changing conditions, and therefore the end result captures the steady state of the biological system rather than an immediate effect of the target protein depletion. Ligand-induced protein degradation based on the genetic fusion of degron tags<a onclick="showRef(event, 'ref3 ref4 ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6 ref7 ref8 ref9">(3−9)</a> to a protein of interest (POI), such as the recently described dTAG system,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> presents an alternative strategy to address these challenges. The dTAG system allows for rapid depletion of a protein of interest in a matter of minutes to hours upon treatment with a degrader such as the cereblon (CRBN)-based dTAG13 or VHL-based dTAG<sup>V</sup>-1,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> making interrogation of immediate biological consequence of target depletion possible. dTAG13 is a degrader molecule that couples to a selective binder of the FKBP12<sup>F36V</sup> protein, AP1867,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> to thalidomide, a binder of the E3 ligase adapter CRBN. The resulting heterobifunctional molecule is capable of recruiting the E3 ligase CRL4<sup>CRBN</sup> into close proximity of the FKBP12<sup>F36V</sup>, which can be present as a genetic fusion to a POI, resulting in its catalytic ubiquitination and subsequent proteasomal degradation. The dTAG system has successfully been applied to decipher complex biology and demonstrated its utility to validate degradation of the protein ENL encoded by the <i>MLLT</i>1 gene as a potential therapeutic target in acute myeloid leukemia<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> as well as to invalidate the kinase MELK as a dependency in breast cancer cells.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Another successful protein degradation tag system utilizes HaloPROTACs, heterobifunctional small molecules that covalently bind to the HaloTag protein and allow for selective targeted degradation of HaloTag fusion proteins.<a onclick="showRef(event, 'ref6 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref8">(6,8)</a></div><div class="NLM_p">To assess the effect of degradation of two proteins, for example, to assess synergy of target degradation, we set out to develop a novel degradation tag/degrader pair. The small molecule inhibitor bump-and-hole approach<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> allows for interrogation of the function of a specific protein by making a minimal change to its active site, a “hole”, which is matched by similar modification to its inhibitor, a “bump”, generating a “bump-and-hole” pair, which results in a substantial gain in binding selectivity over otherwise closely related binding sites. The bump-and-hole inhibitor system has been used in many studies ranging from kinases to epigenetic regulators,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> including BET family bromodomain BRD4.<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a> In this approach, the bromodomain of BRD4 was mutated to create a hole in the ligand binding site, while the corresponding I-BET-based BRD4 inhibitor was modified to include a steric “bump”, allowing for isoform selectivity across other BET bromodomains (BRD2, BRD3, and BRDT). Here, we have utilized this “bump-and-hole” strategy and successfully converted bumped inhibitors of mutant BRD4 into selective CRBN-based degraders, establishing a BRD4<sub>BD1</sub>L94V degron tag with matched degraders.</div><div class="NLM_p last">Using cells stably expressing the BRD4<sub>BD1</sub>L94V degron fused to EGFP and a dTAG-KRAS G12C, we have shown a proof-of-principle degradation of two orthogonal degradation tag systems in the same cells. We anticipate that existing protein degradation tags such as HaloTag-HaloPROTAC,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> auxin-inducible degron,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> or dTAG system<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> could be used in conjunction with the orthogonal CRBN-based BRD4<sub>BD1</sub>L94V degron tag for concurrent removal of multiple targets, thus expanding the use of degradation tags to study the consequences of simultaneous depletion of multiple proteins on cellular function <i>in vitro</i> and <i>in vivo</i>.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88145" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88145" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Based on previous work to engineer a bump-and-hole strategy for BRD4,<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a> we selected <b>2-Me-I-BET762</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) as the binder for hole-modified BRD4 mutant because of its selectivity over wild-type BRD4. To maximize our chances of generating an active degrader, we based our designs on the existing structure–activity relationship (SAR) of CRBN-based BET bromodomain degraders <b>dBET6</b>, <b>dBET57</b>, and <b>ZXH-3-26</b>.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(21)</a> These three degraders utilized the same target binder for BET bromodomain and CRBN, with different linker and exit vector designs. We aligned the crystal structure of BRD4<sub>BD1</sub>L94V in complex with the ester analogue compound <b>3 (XY-06-005)</b><a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> to the corresponding bromodomain in the ternary complex structure of DDB1ΔB-CRBN-<b>dBET6</b>-BRD4<sub>BD1</sub>, in which <b>dBET6</b> and <b>XY-06-005</b> show a close overlap (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). In addition, L94V, conserved across the BET family of bromodomains, was positioned far from the potential dimerization interface (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 2A</a>). Degrader <b>dBET57</b> recruits BRD4 with a distinct binding mode and structural alignment of BRD4<sub>BD1</sub>L94V, indicating that L94V would place itself in close proximity to the CRBN Y355 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). We also aligned the putative model for the conformation of the selective BRD4<sub>BD1</sub> degrader <b>ZXH-3-26</b>, in which no dimerization interface was shown to interfere with BRD4<sub>BD1</sub>L94V (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D, PDB coordinates are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Information</a>). We have therefore determined that designs based on dBET6 analogues would be the most promising starting points. Based on the structural alignments and synthetic tractability, we synthesized a series of molecules with linker lengths ranging from C5 to C9, as well as PEG1 and PEG2 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 1B,C</a>) as the candidate degraders for the BRD4<sub>BD1</sub>L94V degradation tag.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of selective degrader molecules using the bump-and-hole strategy. (A) Bumped BET bromodomain binders and the corresponding thalidomide-based degraders. (B) Structural alignment of bromodomains BRD4<sub>BD1</sub>L94V (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O3C">5O3C</a>) and BRD4<sub>BD1</sub> in complex with <b>dBET6</b> and DDB1–CRBN (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BOY">6BOY</a>). BRD4<sub>BD1</sub>L94V “hole” residue is highlighted in red. (C) Structural alignment of bromodomains BRD4<sub>BD1</sub>L94V (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O3C">5O3C</a>) and BRD4<sub>BD1</sub> in complex with <b>dBET57</b> and DDB1–CRBN (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BNB">6BNB</a>). BRD4<sub>BD1</sub>L94V “hole” residue is highlighted in red and CRBN Y355 is highlighted in cyan. Thalidomide is modeled in by aligning the CRBN C-terminal domain in the crystal structure of DDB1–CRBN–thalidomide (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CI1">4CI1</a>) to the C-terminal domain of CRBN in the dBET57 complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BNB">6BNB</a>). (D) Structural alignment of bromodomain BRD4<sub>BD1</sub>L94V (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O3C">5O3C</a>) and Rosetta docking model<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(21)</a> of BRD4<sub>BD1</sub> complex with <b>ZXH-3-26</b> and CRBN. BRD4<sub>BD1</sub>L94V “hole” residue is highlighted in red. Lenalidomide is modeled in by aligning CRBN (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FQD">5FQD</a>) to CRBN in the <b>ZXH-3-26</b> model. (E) Fluorescence polarization. Dose–response displacement of the <b>FITC–JQ1</b> probe by compounds in BRD4<sub>BD1</sub> and BRD4<sub>BD1</sub>L94V assays. Data is shown as means ± st. dev. of <i>N</i> = 3 independent replicates. (F) Fluorescence polarization. Comparison of dose–responses in (C) between BRD4<sub>BD1</sub> and BRD4<sub>BD1</sub>L94V for the two degrader molecules. Data is shown as means ± st. dev. of <i>N</i> = 3 independent replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To assess the ability of parental compounds and their degrader analogues to bind to the wild-type BRD4 as well as the BRD4 mutants, we established a competitive fluorescence polarization binding assay based on displacement of a <b>JQ1-FITC</b> probe. BRD4<sub>BD2</sub> and their corresponding hole-modified mutants BRD4<sub>BD1</sub>L94V, BRD4<sub>BD2</sub>L387V, and BRD2<sub>BD2</sub>L383V were titrated into 10 nM <b>JQ1-FITC</b> fluorescent probe. The binding assay for BRD4<sub>BD1</sub> required lowering the <b>JQ1-FITC</b> probe concentration to 2.5 nM to increase sensitivity range. These binding assays resulted in apparent dissociation constants (<i>K</i><sub>d, app</sub>) of 5.5 ± 0.2 nM for BRD4<sub>BD1</sub>, 428 ± 126 nM for BRD4<sub>BD1</sub>L94V mutant, 39 ± 4.4 nM for BRD4<sub>BD2</sub>, and 238 ± 30 nM for BRD4<sub>BD2</sub>L387V, indicating that mutant versions of the BRD4 bromodomain have approximately a 70-fold loss in affinity to the <b>JQ1-FITC</b> probe as compared to BRD4 wild type (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 2B</a>). We next performed competitive titration of parental inhibitors as well as degraders to preformed complexes of bromodomains and <b>JQ1-FITC</b> probe (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E,F, and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 2C</a>). The results show potent binding of <b>JQ1</b> and bumped analogues (inhibitors <b>XY-06-005</b> and <b>XY-06-006</b>), as well as bifunctional degraders (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 2C</a>). Notably, <b>XY-06-005</b> was 3-fold more selective in binding BRD2<sub>BD2</sub>L383V than <b>JQ1</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 2C</a>). Degraders derived from inhibitors <b>XY-06-005</b> and <b>XY-06-006</b> showed more potent binding to BRD2<sub>BD2</sub>L383V than from the <b>JQ1</b>. Both inhibitors <b>XY-06-005</b> and <b>XY-06-006</b> have significantly reduced affinity to wild-type BRD4<sub>BD2</sub>, with IC<sub>50</sub> values ranging from 0.8 to 4.5 μM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 2C</a>). The same was observed for the degrader molecules, with IC<sub>50</sub> values ranging from 2.0 μM for <b>XY-06-008</b> to 7.0 μM for <b>XY-06-010</b>. As expected, <b>JQ1</b> was able to potently bind BRD4<sub>BD2</sub> with an IC<sub>50</sub> value of about 123 nM. In conclusion, we have observed significantly weaker binding of bumped ligands to the wild-type bromodomains in line with the previously described SAR.<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> Values in nM from the Competitive Binding Assay<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="colsep0 rowsep0" align="center" char="±">JQ1</th><th class="colsep0 rowsep0" align="center" char="±">XY-06-5</th><th class="colsep0 rowsep0" align="center" char="±">XY-06-6</th><th class="colsep0 rowsep0" align="center" char="±">XY-06-7</th><th class="colsep0 rowsep0" align="center" char="±">XY-06-8</th><th class="colsep0 rowsep0" align="center" char="±">XY-06-9</th><th class="colsep0 rowsep0" align="center" char="±">XY-06-10</th><th class="colsep0 rowsep0" align="center" char="±">XY-06-11</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD1</td><td class="colsep0 rowsep0" align="char" char="±">49 ± 4</td><td class="colsep0 rowsep0" align="char" char="±">39 ± 24</td><td class="colsep0 rowsep0" align="char" char="±">2870 ± 140</td><td class="colsep0 rowsep0" align="char" char="±">1260 ± 60</td><td class="colsep0 rowsep0" align="char" char="±">525 ± 39</td><td class="colsep0 rowsep0" align="char" char="±">260 ± 32</td><td class="colsep0 rowsep0" align="char" char="±">3130 ± 160</td><td class="colsep0 rowsep0" align="char" char="±">2080 ± 140</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD1 L94V</td><td class="colsep0 rowsep0" align="char" char="±">149 ± 13</td><td class="colsep0 rowsep0" align="char" char="±">62 ± 4</td><td class="colsep0 rowsep0" align="char" char="±">187 ± 20</td><td class="colsep0 rowsep0" align="char" char="±">132 ± 10</td><td class="colsep0 rowsep0" align="char" char="±">98 ± 6</td><td class="colsep0 rowsep0" align="char" char="±">63 ± 4</td><td class="colsep0 rowsep0" align="char" char="±">218 ± 19</td><td class="colsep0 rowsep0" align="char" char="±">168 ± 18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2 L387V</td><td class="colsep0 rowsep0" align="char" char="±">523 ± 15</td><td class="colsep0 rowsep0" align="char" char="±">231 ± 4</td><td class="colsep0 rowsep0" align="char" char="±">495 ± 11</td><td class="colsep0 rowsep0" align="char" char="±">438 ± 8</td><td class="colsep0 rowsep0" align="char" char="±">354 ± 9</td><td class="colsep0 rowsep0" align="char" char="±">464 ± 26</td><td class="colsep0 rowsep0" align="char" char="±">686 ± 12</td><td class="colsep0 rowsep0" align="char" char="±">464 ± 12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD2 BD2 L383V</td><td class="colsep0 rowsep0" align="char" char="±">246 ± 15</td><td class="colsep0 rowsep0" align="char" char="±">74 ± 3</td><td class="colsep0 rowsep0" align="char" char="±">160 ± 9</td><td class="colsep0 rowsep0" align="char" char="±">145 ± 7</td><td class="colsep0 rowsep0" align="char" char="±">103 ± 6</td><td class="colsep0 rowsep0" align="char" char="±">130 ± 6</td><td class="colsep0 rowsep0" align="char" char="±">196 ± 9</td><td class="colsep0 rowsep0" align="char" char="±">143 ± 6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2</td><td class="colsep0 rowsep0" align="char" char="±">123 ± 7</td><td class="colsep0 rowsep0" align="char" char="±">863 ± 19</td><td class="colsep0 rowsep0" align="char" char="±">4650 ± 180</td><td class="colsep0 rowsep0" align="char" char="±">3505 ± 110</td><td class="colsep0 rowsep0" align="char" char="±">1990 ± 86</td><td class="colsep0 rowsep0" align="char" char="±">2240 ± 200</td><td class="colsep0 rowsep0" align="char" char="±">6970 ± 300</td><td class="colsep0 rowsep0" align="char" char="±">3040 ± 89</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Compounds were titrated into the FITC–JQ1 probe and wild-type or mutant protein, <i>N</i> = 3. Values are shown as IC<sub>50</sub> ± S.E. (standard error).</p></div></div></div><div class="NLM_p">We next established the degradation efficiency and selectivity of degraders in an EGFP/mCherry reporter system, where the POI is fused to EGFP separated from mCherry by a P2A peptide cleavage site allowing quantitative, mRNA expression normalized readout of protein levels. We established EGFP/mCherry reporter cell lines for BRD4<sub>BD1</sub>L94V and used previously developed BRD4<sub>BD1</sub> and BRD4<sub>BD2</sub> as selectivity controls.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(21)</a> Cells stably expressing BRD4<sub>BD1</sub>–EGFP or BRD4<sub>BD1</sub>L94V–EGFP and mCherry were treated with increasing concentrations of degraders, bumped inhibitors or inhibitor <b>JQ1</b>, and the EGFP and mCherry signals were monitored using laser scanning fluorimetry (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). <b>JQ1</b> as well as inhibitors <b>XY-06-005</b> and <b>XY-06-006</b> did not result in any measurable degradation of either BRD4<sub>BD1</sub> or BRD4<sub>BD1</sub>L94V, while all degraders were able to degrade BRD4<sub>BD1</sub>L94V with varying levels of selectivity against the BRD4<sub>BD1</sub> wild type (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Notably, <b>XY-06-007</b> and <b>XY-06-011</b> showed a selectivity window of 3 orders of magnitude favoring degradation of BRD4<sub>BD1</sub>L94V over the BRD4<sub>BD1</sub> wild type. For example, no degradation of wild-type BRD4<sub>BD1</sub> was observed at a 1 μM concentration, while ∼75% of the mutant BRD4<sub>BD1</sub>L94V was degraded. As a comparison to <b>XY-06-007</b> and <b>XY-06-011</b>, we have tested a known degrader of BRD4<sub>BD1</sub>, <b>dBET6</b>, and observed potent degradation with a DC<sub>50, 6h</sub> of 5.1 ± 0.6 nM (mean ± S.E.). These results confirmed the selective degradation of the mutant BRD4<sub>BD1</sub>L94V over the BRD4<sub>BD1</sub> wild type with these bifunctional compounds. Interestingly, only one molecule, <b>XY-06-008</b>, showed approximately 10% degradation of BRD4<sub>BD2</sub> at the highest tested concentration of 10 μM, and other degraders were unable to degrade the wild-type BRD4<sub>BD2</sub> demonstrating exceptional selectivity of BRD4<sub>BD1</sub>L94V over BRD4<sub>BD2</sub>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Degradation selectivity profile. (A) Quantitative assessment of cellular degradation using the BRD4<sub>BD1</sub>–EGFP reporter assay. Cells stably expressing BRD4<sub>BD1</sub>, BRD4<sub>BD2</sub>, or BRD4<sub>BD1</sub>L94V were treated with increasing concentrations of degraders <b>XY-06-007</b> to <b>XY-06-011</b> or inhibitors <b>JQ1</b>, <b>XY-06-005</b>, <b>XY-06-006</b> for 5 h, and the EGFP-to-mCherry ratio relative to dimethyl sulfoxide (DMSO) was quantified using laser scanning cytometer (Acumen) (B) Cellular degradation selectivity between the BRD4<sub>BD1</sub>L94V mutant and BRD4<sub>BD1</sub> wild-type reporter. Data as in (A). Data for independent replicates is presented as means ± st. dev. of <i>N</i> = 3 replicates for BRD4<sub>BD1</sub>L94V and BRD4<sub>BD1</sub> and <i>N</i> = 2 for BRD4<sub>BD2</sub>. (C) Scatter plot depicting selectivity of <b>XY-06-007</b>. BRD4 levels are unaffected, and the location of BRD4 is highlighted on the volcano plot. MOLT4 cells were treated with 1 μM <b>XY-06-007</b> or DMSO control for 5 h, and protein abundance was analyzed using TMT quantification mass spectrometry. Significant changes were assessed by a moderated t-test, as implemented in the limma package,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(22)</a> with the log<sub>2</sub> fold change (log<sub>2</sub>FC) shown on the <i>y</i>-axis and log<sub>10</sub><i>p</i> values shown on the <i>x</i>-axis (one biological replicate for <b>XY-06-007</b> or three independent biological replicates for DMSO). (D) As in (C) but for <b>XY-06-011</b>. All Acumen data in this figure was analyzed using CellProfiler.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(23)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Calculated DC<sub>50, 6h</sub> Values in nM for BRD4<sub>BD1</sub>, BRD4<sub>BD1</sub>L94V, and BRD4<sub>BD2</sub> Reporter Cell Lines as well as Calculated Fold-Degradation Selectivity for BRD4<sub>BD1</sub> and the Corresponding BRD4<sub>BD1</sub>L94V Mutant<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="colsep0 rowsep0" align="center"><b>XY-06-007</b></th><th class="colsep0 rowsep0" align="center"><b>XY-06-008</b></th><th class="colsep0 rowsep0" align="center"><b>XY-06-009</b></th><th class="colsep0 rowsep0" align="center"><b>XY-06-010</b></th><th class="colsep0 rowsep0" align="center"><b>XY-06-011</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD1</td><td class="colsep0 rowsep0" align="left">>10 000</td><td class="colsep0 rowsep0" align="left">>10 000</td><td class="colsep0 rowsep0" align="left">1840 ± 160</td><td class="colsep0 rowsep0" align="left">>10 000</td><td class="colsep0 rowsep0" align="left">>10 000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD1 L94V</td><td class="colsep0 rowsep0" align="left">10.2 ± 1.8</td><td class="colsep0 rowsep0" align="left">6.1 ± 0.9</td><td class="colsep0 rowsep0" align="left">4 ± 1</td><td class="colsep0 rowsep0" align="left">112 ± 20</td><td class="colsep0 rowsep0" align="left">40 ± 10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2</td><td class="colsep0 rowsep0" align="left">>20 000</td><td class="colsep0 rowsep0" align="left">>20 000</td><td class="colsep0 rowsep0" align="left">>20 000</td><td class="colsep0 rowsep0" align="left">>20 000</td><td class="colsep0 rowsep0" align="left">>20 000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">fold selectivity BRD4 BD1/BD1 L94V</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1600</td><td class="colsep0 rowsep0" align="left">460</td><td class="colsep0 rowsep0" align="left">>89</td><td class="colsep0 rowsep0" align="left">>250</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Values are shown as DC<sub>50, 6h</sub> ± S.E. (standard error) of <i>N</i> = 3 replicates for BRD4<sub>BD1</sub> and BRD4<sub>BD1</sub>L94V and <i>N</i> = 2 for BRD4<sub>BD2</sub>.</p></div></div></div><div class="NLM_p">We next assessed the selectivity of the candidate degraders at the whole proteome level. While both <b>XY-06-007</b> and <b>XY-06-011</b> were able to potently degrade BRD4<sub>BD1</sub>L94V in an EGFP reporter cell line, no off-target degradation of endogenous proteins was detected in a whole proteome mass spectrometry experiment across ∼7800 quantified proteins including the BET family of bromodomains after a 6 h incubation with 1 μM of <b>XY-06-007</b> or <b>XY-06-011</b>, confirming the selectivity of these molecules (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D). LIMK2 appeared to be the only protein upregulated in response to treatment with <b>XY-06-011</b>, but its expression was unchanged in response to <b>XY-06-007</b>. We have seen similar data in MOLT4 cells and anticipate that this may be a result of experimental variability rather than a definitive biological stabilization effect.</div><div class="NLM_p">Synthetic lethality, often discovered using CRISPR/Cas9-based functional genomic screens, can be a very powerful strategy to target cancer.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(24)</a> As many of the discovered codependencies lack available chemical tools for target validation, we sought to establish a degradation method enabling two degrader/tag pairs to simultaneously modulate protein levels in cells. To establish proof-of-principle, we generated a dual reporter line by combining the BRD4<sub>BD1</sub>L94V degron with the dTAG system (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). We have used the previously characterized BRD4<sub>BD1</sub>L94V–EGFP/mCherry reporter cell line and stably expressed a HiBit–FKBP12 F36V–KRAS G12C fusion protein using the lentiviral-mediated stable integration. The abundance of each protein was measured by quantification of EGFP/mCherry ratio for BRD4<sub>BD1</sub>L94V and a HiBit lytic assay for the HiBit-tagged FKBP12 F36V–KRAS G12C. To validate the simultaneous degradation of proteins  in the dual reporter line, we performed cotreatment of <b>XY-06-007</b> with the CRBN-based degrader <b>dTAG13</b> or the VHL-based degrader <b>dTAG</b><sup><b>V</b></sup><b>-1</b> and read the same plate using laser scanning fluorimetry (Acumen) at 5 h, followed by the NanoLuc HiBit lytic assay at 7 h (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B–E and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figures 4 and 5</a>). To our surprise, despite the potential competition for the same binding site on CRBN, a combination of <b>XY-06-007</b> in the presence of <b>dTAG13</b> at a 1 μM concentration was only marginally affecting the degradation of BRD4<sub>BD1</sub>L94V–EGFP. Similarly, the presence of 1 μM of <b>XY-06-007</b> had no effect on degradation of the HiBit–FKBP12 F36V–KRAS G12C fusion. In both cases, treatment with <b>XY-06-007</b> or <b>dTAG13</b> did not show cross-degradation of the reporters (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B,C). We next tested the combination of <b>XY-06-007</b> with the VHL-based degrader <b>dTAG</b><sup><b>V</b></sup><b>-1</b>. Degradation of the HiBit–FKBP12 F36V–KRAS G12C by <b>dTAG</b><sup><b>V</b></sup><b>-1</b> was not affected by the presence of 1 μM of <b>XY-06-007</b>. Interestingly, in a reverse experiment, we observed that degradation of BRD4<sub>BD1</sub>L94V–EGFP by <b>XY-06-007</b> was enhanced by the addition of 100 nM <b>dTAG</b><sup><b>V</b></sup><b>-1</b>, and this enhancement was more apparent with the addition of 1 μM of <b>dTAG</b><sup><b>V</b></sup><b>-1</b>. The same effect was also observed with degradation-inactive negative control <b>dTAG</b><sup><b>V</b></sup><b>-1-neg</b>, suggesting that it is not mediated by binding to VHL or by CUL2<sup>VHL</sup> E3 ligase activity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D,F and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 5</a>). To identify how the enhanced degradation of BRD4<sub>BD1</sub>L94V–EGFP by <b>XY-06-007</b> is modulated by <b>dTAG</b><sup><b>V</b></sup><b>-1</b>, we performed a dose–response titration of <b>dTAG</b><sup><b>V</b></sup><b>-1</b> in the presence of 0.1 μM or 1 μM of <b>XY-06-007</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). We found that 100 nM of <b>dTAG</b><sup><b>V</b></sup><b>-1</b> is sufficient to enhance the degradation of BRD4<sub>BD1</sub>L94V–EGFP by <b>XY-06-007</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). Intrigued by these findings, we noted that in the case of CRBN-based ALK degraders, high cellular levels of the drug efflux pump P-glycoprotein 1 (P-gp), also known as multidrug resistance protein 1 (MDR1), lead to significantly reduced degradation of ALK, and, in turn, pharmacological inhibition of P-gp resulted in increased degradation and improved antiproliferative activity.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(25)</a> We hypothesize that <b>dTAG</b><sup><b>V</b></sup><b>-1</b> may saturate P-gp and therefore act as a P-gp inhibitor, resulting in an increase of the cellular levels of <b>XY-06-007</b> and hence enhanced degradation. This is supported by the observation that similar increases in the degradation potency of BRD4<sub>BD1</sub>L94V–EGFP was observed upon coincubation of <b>XY-06-007</b> with 125 nM or 1 μM of the P-gp inhibitor tariquidar<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(26)</a> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G). Although drug–drug interactions may not be desirable in clinical situations, in the case of <b>dTAG</b><sup><b>V</b></sup><b>-1</b> and <b>XY-06-007</b>, enhanced degradation can at times be beneficial as it lowers the effective concentration of the molecules required for target degradation, thus improving degradation selectivity. It is also important to note that no such drug–drug interaction was observed as a result of cotreatment with two CRBN-based degraders <b>XY-06-007</b> and <b>dTAG13</b>, and target degradation was not affected by this cotreatment. We anticipate that cotreatment with multiple CRBN-based degraders would need to be evaluated further in other cellular settings, as CRBN levels could affect the equilibrium of the competition for the CRBN binding site.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Proof-of-principle simultaneous use of dTAG and BRD4<sub>BD1</sub>L94V degron systems. (A) Schematic of the degradation system. Flp293T cells stably expressing BRD4<sub>BD1</sub>L94V–EGFP with mCherry reporter were engineered to stably express the HiBit–FKBP12 F36V–KRAS G12C fusion. The use of the EGFP/mCherry system as well as the orthogonal HiBit system allowed for sequential read of the protein abundance by Acumen followed by the HiBit lytic assay. (B) Acumen EGFP/mCherry assay. Degradation of BRD4<sub>BD1</sub>L94V–EGFP by CRBN-based degrader <b>XY-06-007</b> at a 5 h time point in the presence or absence of CRBN-based degrader <b>dTAG13</b>. (C) HiBit lytic assay. Degradation of FKBP12 F36V–KRAS G12C by <b>dTAG13</b> in the presence or absence of <b>XY-06-007</b>. The readout was performed on the same assay plate as in (C) at a 7 h time point. (D) Degradation of BRD4<sub>BD1</sub>L94V–EGFP. Dose–response of <b>dTAG</b><sup><b>V</b></sup><b>-1</b> in the presence of 1 or 0.1 μM of <b>XY-06-007</b>. (E) HiBit lytic assay. Degradation of FKBP12 F36V–KRAS G12C by <b>dTAG</b><sup><b>V</b></sup><b>-1</b> in the presence or absence of <b>XY-06-007</b>. The readout was performed on the same assay plate as in (D) at a 7 h time point. (F) Degradation of BRD4<sub>BD1</sub>L94V–EGFP by <b>XY-06-007</b> in the presence of <b>dTAG</b><sup><b>V</b></sup><b>-1</b>. (G) Degradation of BRD4<sub>BD1</sub>L94V–EGFP by <b>XY-06-007</b> in the presence of the P-gp inhibitor Tariquidar. All data in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> is presented as means ± st. dev. of <i>N</i> = 3 replicates. All Acumen data in this figure was analyzed using Cellista software (TTP Labtech).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We further investigated the pharmacokinetic properties of selective BRD4<sub>BD1</sub>L94V degrader compounds <b>XY-06-007</b> and <b>XY-06-011</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 5</a>). Both molecules show acceptable <i>in vitro</i> mouse hepatic microsome stability, with half-lives of 18.4 and 23.9 min, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Subsequent pharmacokinetic study in mice demonstrated that compounds <b>XY-06-007</b> and <b>XY-06-011</b> have favorable pharmacokinetic profiles, including good plasma concentration, area under the curve (AUC), and bioavailability, which are comparable to degraders that have been shown to have good <i>in vivo</i> profiles,<a onclick="showRef(event, 'ref10 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref10 ref28 ref29">(10,27,28)</a> making them good candidates for subsequent animal studies (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Tables <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Mouse Hepatic Microsome Stability</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="colsep0 rowsep0" align="center" char=".">half-life in 1 mg/mL hepatic microsomes mouse <i>T</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="left" char=".">intrinsic clearance mouse Cl<sub>int</sub> (μL/(min·mg))</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>XY-06-007</b></td><td class="colsep0 rowsep0" align="char" char=".">18.4</td><td class="colsep0 rowsep0" align="char" char=".">38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>XY-06-011</b></td><td class="colsep0 rowsep0" align="char" char=".">23.9</td><td class="colsep0 rowsep0" align="char" char=".">29</td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Pharmacokinetic Study with IV Dosing</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="rowsep1 colsep0" colspan="8" align="center" char=".">IV (2 mg/kg)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="colsep0 rowsep0" align="center" char=".">CL(mL/(min·kg))</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>ss</sub>(L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub>(h)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>last</sub>(μM·h)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>INF</sub>(μM·h)</th><th class="colsep0 rowsep0" align="center" char=".">MRT<sub>INF</sub>(h)</th><th class="colsep0 rowsep0" align="center" char=".">MRT<sub>last</sub>(h)</th><th class="colsep0 rowsep0" align="center" char=".">C<sub>0</sub>(μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>XY-06-007</b></td><td class="colsep0 rowsep0" align="char" char=".">21.9</td><td class="colsep0 rowsep0" align="char" char=".">0.378</td><td class="colsep0 rowsep0" align="char" char=".">0.515</td><td class="colsep0 rowsep0" align="char" char=".">1.88</td><td class="colsep0 rowsep0" align="char" char=".">1.88</td><td class="colsep0 rowsep0" align="char" char=".">0.288</td><td class="colsep0 rowsep0" align="char" char=".">0.283</td><td class="colsep0 rowsep0" align="char" char=".">8.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>XY-06-011</b></td><td class="colsep0 rowsep0" align="char" char=".">24.8</td><td class="colsep0 rowsep0" align="char" char=".">0.313</td><td class="colsep0 rowsep0" align="char" char=".">0.367</td><td class="colsep0 rowsep0" align="char" char=".">1.57</td><td class="colsep0 rowsep0" align="char" char=".">1.57</td><td class="colsep0 rowsep0" align="char" char=".">0.210</td><td class="colsep0 rowsep0" align="char" char=".">0.208</td><td class="colsep0 rowsep0" align="char" char=".">9.59</td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Pharmacokinetic Study with IP Dosing</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="rowsep1 colsep0" colspan="8" align="center" char=".">IP (10 mg/kg)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>max</sub>(h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub>(μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub>(h)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>last</sub>(μM·h)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>INF</sub>(μM·h)</th><th class="colsep0 rowsep0" align="center" char=".">MRT<sub>INF</sub>(h)</th><th class="colsep0 rowsep0" align="center" char=".">MRT<sub>last</sub>(h)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i>(%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>XY-06-007</b></td><td class="colsep0 rowsep0" align="char" char=".">0.250</td><td class="colsep0 rowsep0" align="char" char=".">6.10</td><td class="colsep0 rowsep0" align="char" char=".">0.721</td><td class="colsep0 rowsep0" align="char" char=".">7.31</td><td class="colsep0 rowsep0" align="char" char=".">7.21</td><td class="colsep0 rowsep0" align="char" char=".">0.943</td><td class="colsep0 rowsep0" align="char" char=".">0.937</td><td class="colsep0 rowsep0" align="char" char=".">77.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>XY-06-011</b></td><td class="colsep0 rowsep0" align="char" char=".">0.250</td><td class="colsep0 rowsep0" align="char" char=".">5.13</td><td class="colsep0 rowsep0" align="char" char=".">0.623</td><td class="colsep0 rowsep0" align="char" char=".">4.30</td><td class="colsep0 rowsep0" align="char" char=".">4.30</td><td class="colsep0 rowsep0" align="char" char=".">0.690</td><td class="colsep0 rowsep0" align="char" char=".">0.688</td><td class="colsep0 rowsep0" align="char" char=".">54.8</td></tr></tbody></table></div></div><div class="NLM_p last">While the assessment of degradability of the BRD4<sub>BD1</sub>L94V degron tag would require establishing either a genetically engineered mouse model or a xenograft study with the engineered cell line, we note that the plasma concentration level of <b>XY-06-007</b> is maintained above its DC<sub>50, 6h</sub> of 10 nM for approximately 6 h when dosed at 10 mg/kg via intraperitoneal injection (IP) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 5</a>), indicating that such <i>in vivo</i> degradation experiment would result in a favorable outcome.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43306" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43306" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We describe the rational development of a BRD4<sub>BD1</sub>L94V degron tag using a “bump-and-hole” approach combined with structural modeling to develop the corresponding degraders <b>XY-06-007</b> and <b>XY-06-011</b>. We have shown that these compounds bind to BRD4<sub>BD1</sub>L94V with increased selectivity over the wild-type bromodomain, which translates into potent and preferential degradation of the degron tag. We observe exquisite proteome-wide selectivity of <b>XY-06-007</b> and <b>XY-06-011</b> and acceptable pharmacokinetics profiles, which makes these molecules suitable candidates for future <i>in vivo</i> studies. We further demonstrate that the dTAG system can be used together with the BRD4<sub>BD1</sub>L94V degron tag for simultaneous modulation of the expression levels of two protein-tag fusions in cells. Other BET bromodomains should be amenable to a similar “bump-and-hole” strategy (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">Supporting Figure 6</a>), and as we and others have previously shown, additional selectivity between the individual bromodomains 1 and 2 can be achieved with degrader molecules.<a onclick="showRef(event, 'ref22 ref30'); return false;" href="javascript:void(0);" class="ref ref22 ref30">(21,29)</a> We anticipate that the BRD4<sub>BD1</sub>L94V degradation system can be used in tandem with other ligand-induced degradation tags to assess the synergistic effect of degrading multiple targets in a chemically controlled manner.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60758" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60758" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">For the synthesis of 2-methyl-substituted intermediate <b>8</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), <i>tert</i>-butyl protection of <b>1</b> followed by diastereoselective methylation leads to <b>2</b>, which after <i>tert</i>-butyl deprotection was coupled with aniline to yield intermediate <b>3</b> and further deprotected to obtain intermediate <b>4</b>. Ammonolysis of <b>4</b> furnished the 1,4-benzodiazepine core <b>5</b>. Mild thionation condition using phosphorus pentasulfide in the presence of sodium bicarbonate led to the formation of desired product <b>6</b> without epimerization.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(30)</a> Formation of the triazole ring using acetohydrazide and mercury acetate followed by saponification gave rise to intermediate <b>8</b>.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Parental Compound <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) 2-bromo-2-methylpropane, K<sub>2</sub>CO<sub>3</sub>, MeCN, 50 °C; (b) LHMDS, MeI, THF, −78 °C; (c) TFA, DCM, rt; (d) (2-amino-5-methoxyphenyl)(4-chlorophenyl)methanone, HATU, DIPEA, DCM, 0 °C to rt; (e) TMSI, CH<sub>3</sub>CN, rt; (f) HOAc, DCE, rt; (g) P<sub>4</sub>S<sub>10</sub>, NaHCO<sub>3</sub>, DCE, 70 °C; and (h) acetohydrazide, Hg(OAc)<sub>2</sub>, THF/HOAc, 0 °C to rt; (i) NaOH, THF, rt.</p></p></figure><div class="NLM_p">The synthesis of heterobifunctional compounds starts with the peptide coupling of intermediate <b>8</b> with the Boc-protected linker (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>), and the resulting intermediate <b>9</b> was deprotected and coupled with acid <b>11</b> to yield heterobifunctional compound <b>XY-06-011</b>.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Heterobifunctional Degrader <b>XY-06-011</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) <i>tert</i>-butyl (2-(2-(2-aminoethoxy)ethoxy)ethyl)carbamate, HOBt, EDCI, DCM, 0 °C to rt; (b) TFA, DCM, rt; and (c) 2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetic acid <b>11</b>, HOBt, EDCI, DMF, 0 °C to rt.</p></p></figure></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15700" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15700" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Unless otherwise noted, reagents and solvents were used as received from commercial suppliers. All reactions were monitored using <b>LC-MS</b>: Agilent LC/MS system (Agilent 1200 LC/G6130A MS) using a SunFire C18 column (4.6 mm × 50 mm, 3.5 μm particle size). Column temperature 50 °C. LC method: solvent gradient = 5% B-95% B; solvent A = 0.01% TFA in water; solvent B = 0.01% TFA in ACN; flow rate: 2.0 mL/min. Purification of the final products was carried out by the Gilson 281 (PHG008) HPLC system using a Welch C18 column (21.2 mm × 250 mm, 10 μm particle size): solvent gradient 10–95% acetonitrile in H<sub>2</sub>O (0.05% TFA as additive); flow rate: 30 mL/min. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were obtained using Bruker Avance III spectrometers (400 MHz for <sup>1</sup>H). Chemical shifts are reported relative to deuterated methanol (δ = 3.31) or dimethyl sulfoxide (δ = 2.50) for <sup>1</sup>H NMR. Spectra are given in ppm (δ) and as br = broad, s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, and coupling constants J are reported in Hertz. The purity of the final products was determined by UPLC-MS. All final products had a purity of ≥95% unless noted otherwise.</div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemical Synthesis Methods</h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 1-(<i>tert</i>-Butyl) 4-Methyl ((Benzyloxy)carbonyl)-<span class="smallcaps smallerCapital">l</span>-aspartate</h4><div class="NLM_p last">To a solution of (<i>S</i>)-2-(((benzyloxy)carbonyl)amino)-4-methoxy-4-oxobutanoic acid (15 g, 53.4 mmol, 1 equiv) in acetonitrile (180 mL) were added potassium carbonate (47.97 g, 347.1 mmol, 6.5 equiv) and BnEt<sub>3</sub>NCl (12.4 g, 54.5 mmol, 1.02 equiv). After stirring at room temperature for 5 h, 2-bromo-2-methylpropane (73.17 g, 534 mmol, 10 equiv) was added to the reaction mixture. The resulting mixture was stirred at 50 °C for 26 h, cooled down to room temperature, diluted with water (200 mL), and extracted with ethyl acetate (200 mL × 3). The combined organic layers were washed with brine (150 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, concentrated, and purified by flash column chromatography on silica gel (ethyl acetate in petroleum ether, 20% v/v) to afford the title compound as a colorless oil (12 g, yield 66.5%). LC-MS (ESI) <i>m</i>/<i>z</i>: 238 [M – Boc + H]<sup>+</sup>. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ (ppm) 1.45 (s, 9H), 2.82 (dd, <i>J</i> = 16.6 Hz, 1H), 2.98 (dd, <i>J</i> = 16.6 Hz, 1H), 3.66 (s, 3H), 4.50–4.54 (m, 1H), 5.12 (s, 2H), 5.73 (d, <i>J</i> = 8.0 Hz, 1H), 7.27–7.36 (m, 5H).</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 1-(<i>tert</i>-Butyl) 4-Methyl (2<i>S</i>,3<i>R</i>)-2-(((Benzyloxy)carbonyl)amino)-3-methylsuccinate (Compound <b>2</b>)</h4><div class="NLM_p last">The solution of 1-(<i>tert</i>-butyl) 4-methyl ((benzyloxy)carbonyl)-<span class="smallcaps smallerCapital">l</span>-aspartate (8.8 g, 26.1 mmol, 1 equiv) in anhydrous THF (105 mL) was cooled down to −78 °C, and then LiHMDS solution (1 M in THF, 54.8 mL, 54.8 mmol, 2.1 equiv) was added dropwise. The mixture was stirred at −78 °C under nitrogen for 45 minutes, and iodomethane (11.1 g, 78.3 mmol, 3 equiv) was added dropwise. The resulting mixture was stirred at −78 °C for 3 h, quenched with saturated NH<sub>4</sub>Cl solution (50 mL), and extracted with ethyl acetate (100 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, concentrated, and purified by flash column chromatography on silica gel (ethyl acetate in petroleum ether, 20% v/v) to afford compound <b>2</b> as a colorless oil (8.8 g, yield 96%). LC-MS (ESI) <i>m</i>/<i>z</i>: 252 [M – Boc + H]<sup>+</sup>. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ (ppm) 1.21 (d, <i>J</i> = 7.6 Hz 3H), 1.46 (s, 9H), 3.17–3.23 (m, 1H), 3.69 (s, 3H), 4.57 (dd, <i>J</i> = 9.0 Hz, 1H), 5.13 (s, 2H), 5.64 (d, <i>J</i> = 8.4 Hz, 1H), 7.27–7.37 (m, 5H).</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> (2<i>S</i>,3<i>R</i>)-2-(((Benzyloxy)carbonyl)amino)-4-methoxy-3-methyl-4-oxobutanoic Acid</h4><div class="NLM_p last">To a solution of compound <b>2</b> (8.8 g, 25.07 mmol, 1 equiv) in dichloromethane (120 mL) was added TFA (40 mL). The reaction mixture was stirred at room temperature overnight and concentrated to afford the title compound as an oil (9 g, crude), which was used directly in the next step without further purification. LC-MS (ESI) <i>m</i>/<i>z</i>: 296 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Methyl (2<i>R</i>,3<i>S</i>)-3-(((Benzyloxy)carbonyl)amino)-4-((2-(4-chlorobenzoyl)-4-methoxyphenyl)amino)-2-methyl-4-oxobutanoate (Compound <b>3</b>)</h4><div class="NLM_p last">To a solution of (2<i>S</i>,3<i>R</i>)-2-(((benzyloxy)carbonyl)amino)-4-methoxy-3-methyl-4-oxobutanoic acid (9 g crude, 25.07 mmol, 1 equiv) in dichloromethane (180 mL) were added HATU (19.06 g, 50.14 mmol, 2 equiv), DIPEA (12.96 g, 100.28 mmol, 4 equiv), and (2-amino-5-methoxyphenyl)(4-chlorophenyl)methanone (7.44 g, 25.07 mmol, 1 equiv) at 0 °C. The reaction mixture was stirred at room temperature for 2 h, diluted with dichloromethane (120 mL), washed with brine (100 mL × 3), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography on silica gel (ethyl acetate in petroleum ether, 30% v/v) to give compound <b>3</b> as a light yellow solid (10 g, yield 74.1%). LC-MS (ESI) <i>m</i>/<i>z</i>: 539 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Methyl (2<i>R</i>,3<i>S</i>)-3-Amino-4-((2-(4-chlorobenzoyl)-4-methoxyphenyl)amino)-2-methyl-4-oxobutanoate (Compound <b>4</b>)</h4><div class="NLM_p last">To a solution of compound <b>3</b> (10 g, 18.6 mmol, 1 equiv) in acetonitrile (200 mL) was added iodotrimethylsilane (8.23 g, 41.1 mmol, 2.2 equiv) at room temperature. The reaction mixture was stirred at room temperature for 2 h, diluted with ethyl acetate (100 mL), washed with saturated NaHCO<sub>3</sub> solution (100 mL × 2) and brine (100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford crude compound <b>4</b> as an oil (12 g, crude), which was used directly in the next step without further purification. LC-MS (ESI) <i>m</i>/<i>z</i>: 405 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Methyl (<i>R</i>)-2-((<i>S</i>)-5-(4-Chlorophenyl)-7-methoxy-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>e</i>][1,4]diazepin-3-yl)propanoate (Compound <b>5</b>)</h4><div class="NLM_p last">To a solution of compound <b>4</b> (10 g, 15.5 mmol, 1 equiv) in 1,2-dichloroethane (116 mL) was added acetic acid (10.5 g, 173 mmol, 11 equiv) at room temperature. The reaction mixture was stirred at 60 °C for 2 h and then concentrated <i>in vacuo</i>. The residue was dissolved in dichloromethane (150 mL), washed with brine (50 mL × 2), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, concentrated, and purified by flash column chromatography on silica gel (ethyl acetate in petroleum ether, 50%v/v) to yield compound <b>5</b> as a solid (3.5 g, yield 36.6%). LC-MS (ESI) <i>m</i>/<i>z</i>: 387 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ (ppm) 1.44 (d, <i>J</i> = 7.2 Hz, 3H), 3.53–3.58 (m, 1H), 3.74 (s, 3H), 3.76 (s, 3H), 3.84 (d, <i>J</i> = 10.4 Hz, 1H), 6.74 (s, 1H), 7.08–7.11 (m, 2H), 7.35 (d, <i>J</i> = 8.4 Hz, 2H), 7.52 (d, <i>J</i> = 8.8 Hz, 2H), 9.23 (s, 1H).</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Methyl (<i>R</i>)-2-((<i>S</i>)-5-(4-Chlorophenyl)-7-methoxy-2-thioxo-2,3-dihydro-1<i>H</i>-benzo[<i>e</i>][1,4]diazepin-3-yl)propanoate (Compound <b>6</b>)</h4><div class="NLM_p last">A mixture of P<sub>4</sub>S<sub>10</sub> (2.07 g, 4.66 mmol, 1.8 equiv) and NaHCO<sub>3</sub> (0.786 g, 9.36 mmol) in 1,2-dichloroethane (40 mL) was stirred at room temperature for 2 h, and then compound <b>5</b> (1 g, 2.6 mmol, 1 equiv) was added. The resulting mixture was stirred at 70 °C for 2 h and concentrated <i>in vacuo</i>; the residue was partitioned between dichloromethane (50 mL) and water (50 mL). The organic layer was separated, washed with brine (50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, concentrated, and purified by flash column chromatography on silica gel (ethyl acetate in petroleum ether, 30%v/v) to give compound <b>6</b> as a light yellow solid (0.8 g, yield 76.8%). LC-MS (ESI) <i>m</i>/<i>z</i>: 403 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ (ppm) 1.38 (d, <i>J</i> = 6.8 Hz, 3H), 3.76 (s, 3H), 3.80 (s, 3H), 3.90 (d, <i>J</i> = 9.6 Hz, 1H), 4.05–4.15 (m, 1H), 6.78 (s, 1H), 7.08–7.14 (m, 2H), 7.35 (d, <i>J</i> = 8.8 Hz, 2H), 7.52 (d, <i>J</i> = 8.8 Hz, 2H), 9.23 (s, 1H).</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Methyl (<i>R</i>)-2-((<i>S</i>)-6-(4-Chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl)propanoate (Compound <b>7</b>)</h4><div class="NLM_p last">The suspension of compound <b>6</b> (2.5 g, 6.21 mmol, 1 equiv) and acetyl hydrazide (1.38 g, 18.6 mmol, 3 equiv) in THF (37.5 mL) and acetic acid (25 mL) was cooled down to 0 °C, and mercury acetate (2.97 g, 9.32 mmol, 1.5 equiv) was added portionwise to the reaction mixture at such a rate that the temperature was maintained below 5 °C. After addition, the reaction mixture was stirred at 0 °C for 2 h and then stirred at room temperature overnight. The resulting mixture was partitioned between ethyl acetate (50 mL) and saturated NaHCO<sub>3</sub> solution (50 mL), the organic layer was separated, and the aqueous layer was re-extracted with ethyl acetate (50 mL × 3). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography on silica gel (methanol in dichloromethane, 10% v/v) to give compound <b>7</b> as a light yellow solid (1.5 g, yield 56.9%). LC-MS (ESI) <i>m</i>/<i>z</i>: 425 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ (ppm) 1.49 (d, <i>J</i> = 6.8 Hz, 3H), 2.61 (s, 3H), 3.82 (s, 3H), 3.83 (s, 3H), 4.06–4.15 (m, 1H), 4.25 (d, <i>J</i> = 10.8 Hz, 1H), 6.88 (s, 1H), 7.23 (dd, <i>J</i> = 9.0 Hz, 1H), 7.32 (d, <i>J</i> = 6.8 Hz, 2H), 7.42 (q, <i>J</i> = 8.4 Hz, 3H).</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (<i>R</i>)-2-((<i>S</i>)-6-(4-Chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl)propanoic Acid (Compound <b>8</b>)</h4><div class="NLM_p last">To a solution of compound <b>7</b> (0.2 g, 0.471 mmol, 1 equiv) in THF (3 mL) was added NaOH solution (1 M, 0.943 mL, 0.943 mmol, 2 equiv). The reaction mixture was stirred at room temperature for 48 h. The mixture was adjusted to pH 3–4 with a 1 M HCl solution and extracted with ethyl acetate (10 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford crude compound <b>8</b> as a solid (0.15 g, yield 78%), which was used directly in the next step without further purification. LC-MS (ESI) <i>m</i>/<i>z</i>: 411 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Procedure I for Synthesis of Heterobifunctional Compound</h3><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>tert</i>-Butyl (2-(2-(2-((2<i>R</i>)-2-((4<i>S</i>)-6-(4-Chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl)propanamido)ethoxy)ethoxy)ethyl)carbamate (Compound <b>9</b>)</h4><div class="NLM_p last">To a solution of compound <b>8</b> (0.1 g, 0.244 mmol, 1 equiv) and <i>N-</i>Boc-3,6-dioxaoctane-1,8-diamine (119 mg, 0.488 mmol, 2 equiv) in dichloromethane (5 mL) were added HOBt (39 mg, 0.293 mmol, 1.2 equiv) and EDCI·HCl (56.2 mg, 0.293 mmol, 1.2 equiv) at 0 °C. The reaction mixture was stirred at room temperature under nitrogen for 2 h, diluted with dichloromethane (30 mL), washed with brine (10 mL × 2), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to yield a crude product, which was purified by flash column chromatography on silica gel (methanol in dichloromethane, 15% v/v) to furnish compound <b>9</b> as a white solid (0.140 g, 89.6% yield). LC-MS (ESI) <i>m</i>/<i>z</i>: 641 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (2<i>R</i>)-<i>N-</i>(2-(2-(2-Aminoethoxy)ethoxy)ethyl)-2-((4<i>S</i>)-6-(4-chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl)propanamide TFA Salt (Compound <b>10</b>)</h4><div class="NLM_p last">To a solution of compound <b>9</b> (0.17 g, 0.265 mmol, 1 equiv) in dichloromethane (3 mL) was added TFA (1 mL). The reaction mixture was stirred at room temperature for 30 minutes and concentrated <i>in vacuo</i>. The residue was diluted with dichloromethane (30 mL) and washed with saturated NaHCO<sub>3</sub> solution (10 mL × 3) and brine (10 mL). The aqueous layers were combined and re-extracted with a mixture of methanol and dichloromethane (30 mL × 2, 10% methanol in dichloromethane). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to yield crude compound <b>10</b> as an oil (0.14 g crude, 80.6% crude yield), which was used directly for the next step without further purification. LC-MS (ESI) <i>m</i>/<i>z</i>: 541 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (2<i>R</i>)-2-((4<i>S</i>)-6-(4-Chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl)-<i>N-</i>(2-(2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)ethoxy)ethyl) Propanamide TFA Salt (Compound <b>XY-06-011</b>)</h4><div class="NLM_p last">To a solution of compound <b>10</b> (0.12 g crude, 0.183 mmol, 1 equiv) and 2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetic acid (88 mg, 0.266 mmol, 1.45 equiv) in anhydrous DMF (6 mL) were added HOBt (36 mg, 0.266 mmol, 1.45 equiv) and EDCI·HCl (51 mg, 0.266 mmol, 1.45 equiv) at 0 °C. The reaction mixture was stirred at room temperature under nitrogen for 2 h, diluted with water (30 mL), and extracted with ethyl acetate (30 mL × 3). The combined organic layers were washed with brine (30 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>, and the residue was purified by preparative HPLC (MeCN–H<sub>2</sub>O–TFA) to afford compound <b>XY-06-011</b> as a white solid (120 mg, 67.6% yield). Purity is >95%. UPLC-MS (ESI) <i>m</i>/<i>z</i>: 854.62 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.11 (s, 1H), 8.36 (t, <i>J</i> = 5.7 Hz, 1H), 8.02 (t, <i>J</i> = 5.7 Hz, 1H), 7.81 (d, <i>J</i> = 9.0 Hz, 1H), 7.80 (dd, <i>J</i> = 8.4, 7.3 Hz, 1H), 7.50–7.44 (m, 5H), 7.40 (dd, <i>J</i> = 9.0, 2.9 Hz, 1H), 7.39 (d, <i>J</i> = 8.8 Hz, 1H), 6.86 (d, <i>J</i> = 2.9 Hz, 1H), 5.11 (ddd, <i>J</i> = 12.7, 5.5, 1.7 Hz, 1H), 4.78 (s, 2H), 4.11 (d, <i>J</i> = 10.7 Hz, 1H), 3.79 (s, 3H), 3.69 (dq, <i>J</i> = 10.7, 6.7 Hz, 1H), 3.57 (td, <i>J</i> = 4.6, 2.2 Hz, 4H), 3.50 (dt, <i>J</i> = 14.7, 5.9 Hz, 4H), 3.41–3.21 (m, 4H), 2.95–2.82 (m, 1H), 2.65–2.48 (m, 2H), 2.54 (s, 3H), 2.09–1.98 (m, 1H), 1.21 (d, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 174.0, 172.8, 169.9, 166.9, 166.7, 165.4, 164.7, 158.3, 158.0, 157.6, 155.0, 154.8, 150.8, 137.0, 136.9, 135.4, 133.0, 131.1, 129.2, 128.2, 126.1, 125.8, 120.3, 117.9, 116.8, 116.0, 114.9, 69.6, 69.6, 69.3, 68.9, 67.5, 59.3, 55.8, 48.8, 41.8, 38.5, 38.4, 30.9, 22.0, 15.5, 11.4.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (2<i>R</i>)-2-((4<i>S</i>)-6-(4-Chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl)-<i>N-</i>(5-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)pentyl)propanamide TFA Salt (Compound <b>XY-06-007</b>)</h4><div class="NLM_p last">Compound <b>XY-06-007</b> was obtained using similar route as a white solid. Purity is >95%. UPLC-MS (ESI) <i>m</i>/<i>z</i>: 808.61 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.10 (s, 1H), 8.31–8.19 (m, 1H), 7.97 (t, <i>J</i> = 5.7 Hz, 1H), 7.81 (d, <i>J</i> = 8.9 Hz, 1H), 7.81 (t, <i>J</i> = 8.4 Hz, 1H), 7.48 (d, 1H), 7.41–7.36 (m, 2H), 6.87 (dd, <i>J</i> = 2.9, 1.0 Hz, 1H), 5.12 (ddd, <i>J</i> = 12.8, 5.4, 1.7 Hz, 1H), 4.77 (s, 2H), 4.11 (d, <i>J</i> = 10.8 Hz, 1H), 3.79 (s, 3H), 3.65 (dq, <i>J</i> = 10.7, 6.7 Hz, 1H), 3.19 (dq, <i>J</i> = 17.6, 6.7 Hz, 3H), 3.07 (dq, <i>J</i> = 12.8, 6.6 Hz, 1H), 2.88 (ddd, <i>J</i> = 17.4, 14.1, 5.5 Hz, 1H), 2.64–2.46 (m, 2H), 2.54 (s, 3H), 2.04 (dt, <i>J</i> = 12.9, 3.1 Hz, 1H), 1.59–1.45 (m, 4H), 1.42–1.30 (m, 2H), 1.21 (d, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 173.6, 172.7, 169.9, 166.7, 166.7, 165.5, 164.7, 158.3, 158.0, 157.6, 155.1, 154.8, 150.8, 137.0, 136.9, 135.5, 133.0, 131.0, 129.3, 128.2, 126.0, 125.8, 120.4, 117.9, 116.8, 116.0, 114.9, 67.7, 59.3, 55.8, 48.8, 41.9, 38.4, 38.3, 30.9, 29.1, 28.7, 23.7, 22.0, 15.6, 11.4.</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (2<i>R</i>)-2-((4<i>S</i>)-6-(4-Chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl)-<i>N-</i>(7-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)heptyl)propanamide TFA Salt (Compound <b>XY-06-008</b>)</h4><div class="NLM_p last">Compound <b>XY-06-008</b> was obtained using similar route as a white solid. Purity is >95%. UPLC-MS (ESI) <i>m</i>/<i>z</i>: 836.71 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.11 (s, 1H), 8.24 (t, <i>J</i> = 5.6 Hz, 1H), 7.93 (t, <i>J</i> = 5.8 Hz, 1H), 7.81 (d, <i>J</i> = 9.1 Hz, 1H), 7.80 (t, <i>J</i> = 8.5 Hz, 1H), 7.51–7.43 (m, 5H), 7.39 (dd, <i>J</i> = 8.8, 3.3 Hz, 2H), 6.86 (d, <i>J</i> = 2.9 Hz, 1H), 5.11 (dd, <i>J</i> = 12.7, 5.4 Hz, 1H), 4.76 (s, 2H), 4.11 (d, <i>J</i> = 10.7 Hz, 1H), 3.79 (s, 3H), 3.65 (dq, <i>J</i> = 10.8, 6.7 Hz, 1H), 3.25–3.01 (m, 4H), 2.89 (ddd, <i>J</i> = 16.7, 13.7, 5.4 Hz, 1H), 2.63–2.51 (m, 2H), 2.54 (s, 3H), 2.04 (ddd, <i>J</i> = 13.1, 5.8, 3.5 Hz, 1H), 1.56–1.38 (m, 4H), 1.31 (d, <i>J</i> = 5.2 Hz, 6H), 1.21 (d, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 174.1, 173.2, 170.3, 167.2, 167.1, 166.0, 165.1, 158.8, 158.5, 158.1, 155.5, 155.3, 151.2, 137.5, 137.4, 135.9, 133.5, 131.5, 129.7, 128.6, 126.5, 126.3, 120.9, 118.3, 117.3, 116.5, 115.4, 68.1, 59.7, 56.3, 49.3, 42.4, 38.9, 38.8, 31.4, 29.8, 29.5, 29.0, 26.9, 26.8, 22.5, 16.1, 11.9.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (2<i>R</i>)-2-((4<i>S</i>)-6-(4-Chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl)-<i>N</i>-(9-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)nonyl)propanamide TFA Salt (Compound <b>XY-06-009</b>)</h4><div class="NLM_p last">Compound <b>XY-06-009</b> was obtained using similar route as a white solid. Purity is > 95%. UPLC-MS (ESI) <i>m</i>/<i>z</i>: 864.62 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.11 (s, 1H), 8.23 (t, <i>J</i> = 5.7 Hz, 1H), 7.92 (t, <i>J</i> = 5.7 Hz, 1H), 7.82 (d, <i>J</i> = 9.0 Hz, 1H), 7.80 (dd, <i>J</i> = 8.6, 7.2 Hz, 1H), 7.51–7.42 (m, 5H), 7.39 (dd, <i>J</i> = 9.0, 2.9 Hz, 1H), 7.39 (d, <i>J</i> = 8.3 Hz, 1H), 6.85 (d, <i>J</i> = 2.9 Hz, 1H), 5.11 (ddd, <i>J</i> = 12.7, 5.5, 3.7 Hz, 1H), 4.76 (s, 2H), 4.11 (d, <i>J</i> = 10.8 Hz, 1H), 3.79 (s, 3H), 3.64 (dq, <i>J</i> = 10.8, 6.8 Hz, 1H), 3.13 (tp, <i>J</i> = 25.6, 6.4 Hz, 4H), 2.89 (ddd, <i>J</i> = 16.8, 13.7, 5.4 Hz, 1H), 2.61–2.49 (m, 2H), 2.54 (s, 3H), 2.09–1.98 (m, 1H), 1.45 (dt, <i>J</i> = 30.1, 7.3 Hz, 4H), 1.35–1.23 (m, 10H), 1.21 (d, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 173.6, 172.7, 169.8, 166.7, 166.6, 165.5, 164.6, 158.3, 158.0, 157.6, 155.1, 154.8, 150.8, 137.0, 136.9, 135.4, 133.0, 131.0, 129.2, 128.1, 126.1, 125.9, 120.4, 117.8, 116.8, 116.0, 114.9, 67.6, 59.3, 55.8, 48.8, 42.0, 38.5, 38.3, 30.9, 29.4, 29.1, 29.0, 28.9, 28.8, 26.5, 26.4, 22.0, 15.6, 11.4.</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (2<i>R</i>)-2-((4<i>S</i>)-6-(4-Chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl)-<i>N-</i>(2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)ethyl)propanamide TFA Salt (Compound <b>XY-06-010</b>)</h4><div class="NLM_p last">Compound <b>XY-06-010</b> was obtained using similar route as a white solid. Purity is > 95%. UPLC-MS (ESI) <i>m</i>/<i>z</i>: 810.71 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.10 (s, 1H), 8.34 (q, <i>J</i> = 5.3 Hz, 1H), 8.05 (t, <i>J</i> = 5.6 Hz, 1H), 7.81 (d, <i>J</i> = 9.0 Hz, 1H), 7.78 (dd, <i>J</i> = 8.5, 7.3 Hz, 1H), 7.49–7.42 (m, 5H), 7.39 (ddd, <i>J</i> = 9.0, 2.9, 0.9 Hz, 1H), 7.35 (dd, <i>J</i> = 8.6, 1.9 Hz, 1H), 6.86 (d, <i>J</i> = 2.9 Hz, 1H), 5.12 (dt, <i>J</i> = 13.1, 4.9 Hz, 1H), 4.79 (s, 2H), 4.11 (dd, <i>J</i> = 10.8, 1.1 Hz, 1H), 3.79 (s, 3H), 3.69 (dq, <i>J</i> = 10.7, 6.7 Hz, 1H), 3.59–3.48 (m, 4H), 3.46–3.22 (m, 4H), 3.10 (qd, <i>J</i> = 7.3, 4.8 Hz, 1H), 2.88 (ddd, <i>J</i> = 18.0, 14.2, 5.6 Hz, 1H), 2.61–2.46 (m, 1H), 2.54 (s, 3H), 2.04 (dd, <i>J</i> = 13.1, 7.2 Hz, 1H), 1.25–1.20 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 174.5, 173.2, 170.3, 167.4, 167.2, 165.9, 165.3, 158.8, 158.5, 158.1, 155.4, 155.3, 151.3, 137.5, 137.4, 135.9, 133.5, 131.5, 129.7, 128.6, 126.5, 126.3, 120.8, 118.4, 117.2, 116.5, 115.4, 69.5, 69.2, 68.0, 59.8, 56.3, 49.3, 42.3, 38.9, 38.8, 31.4, 22.5, 16.0, 11.9.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Protein Purification</h3><div class="NLM_p last">Wild-type and mutant BRD4<sub>BD1</sub>, BRD4<sub>BD2</sub>, and BRD2<sub>BD1</sub>L383V were obtained from Twist Bioscience in the pNIC-Bio2 vector with N-term TEV-cleavable His<sub>6</sub> tag and C-term Avi tag, and all were expressed in BL21-DE3 Rosetta cells using standard protocols. For purification, His<sub>6</sub>-tagged protein cells were resuspended in a buffer containing 50 mM Tris(hydroxymethyl)aminomethane hydrochloride (Tris–HCl) pH 8.0, 200 mM NaCl, 1 mM Tris(2-carboxyethyl)phosphine (TCEP), 1 mM phenylmethylsulfonyl fluoride (PMSF), 1x protease inhibitor cocktail (Sigma-Aldrich) and lysed by sonication. Following ultracentrifugation, the soluble fraction was passed over a Ni Sepharose 6 Fast Flow affinity resin (GE Healthcare) and eluted with a wash buffer (50 mM Tris–HCl pH 8.0, 200 mM NaCl, 1 mM TCEP) supplemented with 100 mM imidazole (Fischer Chemical). The affinity-purified protein was subjected to size exclusion chromatography in 50 mM HEPES pH 7.4, 200 mM NaCl, and 1 mM TCEP. Proteins were biotinylated overnight as previously described.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(31)</a> The protein-containing fractions were concentrated using ultrafiltration (Millipore) and flash-frozen in liquid nitrogen at ∼25 to 100 μM concentration and stored at −80 °C.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Fluorescence Polarization Assays</h3><div class="NLM_p">FITC-conjugated JQ1 (10 nM final concentration) in assay buffer containing 50 mM Tris pH 7.5, 200 mM NaCl, 0.1% Pluronic F-68 solution (Sigma-Aldrich) was mixed with an increasing concentration of purified his<sub>6</sub>-BRD4<sub>BD2</sub>, his<sub>6</sub>-BRD4<sub>BD1</sub>L94V, BRD4<sub>BD2</sub>L387V, or BRD2<sub>BD2</sub>L383V (2-fold, 23-point dilution, and DMSO control) in 384-well microplates (Corning, 4514) and incubated for 15 min at RT. Titration for his<sub>6</sub>-BRD4<sub>BD1</sub> was performed as described above but using a buffer containing a 2.5 nM final concentration of the FITC-conjugated JQ1 probe. The change in fluorescence polarization was monitored using a PHERAstar FS microplate reader (BMG Labtech). The <i>K</i><sub>d</sub> was obtained from a fit of the “one-site total” model in GraphPad Prism 9 from three independent replicates (<i>N</i> = 3).</div><div class="NLM_p last">Compounds in the FITC-conjugated JQ1 displacement assay were dispensed in a 384-well microplate (Corning, 4514) using a D300e Digital Dispenser (HP) normalized to 1% DMSO into 10 nM FITC–JQ1, 50 mM Tris pH 7.5, 200 mM NaCl, 0.1% Pluronic F-68 solution (Sigma-Aldrich) with 400 nM BRD4<sub>BD1</sub>L94V or 250 nM BRD4<sub>BD2</sub>L387V or 150 nM BRD2<sub>BD2</sub>L383V or 50 nM BRD4<sub>BD2</sub>. Assay for his<sub>6</sub>-BRD4<sub>BD1</sub> was performed as described above but with 10 nM his<sub>6</sub>-BRD4<sub>BD1</sub> protein and 2.5 nM FITC–JQ1 probe. The change in fluorescence polarization was monitored using a PHERAstar FS microplate reader (BMG Labtech) until a stable signal was observed. Dose–response data (<i>N</i> = 3) was plotted, and IC<sub>50</sub> values were estimated using the variable slope equation in GraphPad Prism 9.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Generation of the BRD4<sub>BD1</sub>L94V: KRAS G12C Stable Line</h3><div class="NLM_p last">HEK293T cells were seeded at 700 000 cells/well in a 6-well plate (Falcon, 353046) with DMEM media (Gibco, Life Technologies) containing 10% FBS with a final volume of 3 mL per well. The lentivirus was constructed by cotransfection of the expression plasmid HiBit–FKBP12 F36V–KRAS G12C based on Cilantro2 backbone (Addgene, #74450) but with fluorescence markers removed at 1 μg quantity together with 1 μg psPax2 (Addgene, #12260) and 0.1 μg pVSV-G (Addgene, #36399) and a 4:1 ratio of FuGene to DNA in 100 μL of Opti-MEM media (Gibco, Life Technologies) the following day. The lentivirus was harvested 48 h after transfection by centrifugation at 1000<i>g</i> for 5 min and frozen at −80 °C. For transductions, Flp293T cells stably expressing BRD4<sub>BD1</sub>L94V–EGFP and mCherry reporter separated by the P2A site were seeded in a 6-well plate at 600 000 per well. Transduction was performed the next day at 40%: 800 μL of lentiviral soup was added to 1.2 mL of DMEM + 10% FBS and 1.6 μL of 10 mg/mL polybrene. After 2–3 days, cells with a stable integration of the HiBit–FKBP12 F36V–KRAS G12C were selected using 1 μg/mL of puromycin. The final Flp293T cell line stably expressing both BRD4<sub>BD1</sub>L94V–EGFP–P2A–mCherry under CMV promoter and HiBit–FKBP12 F36V–KRAS G12C under PGK promoter was used for degradation experiments.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Cellular Degradation Assays Using Acumen</h3><div class="NLM_p last">Flp293T cells stably expressing the BRD4<sub>BD1</sub>–EGFP, BRD4<sub>BD2</sub>–EGFP, or BRD4<sub>BD1</sub>L94V–EGFP with mCherry reporter established using the FlipIn system as described before<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(21)</a> or Flp293T with simultaneous stable expression of BRD4<sub>BD1</sub>L94V–EGFP with mCherry reporter and HiBit–FKBP12 F36V–KRAS G12C were seeded at a 30–50% confluency in 384-well plates with 50 μL FluoroBrite DMEM media (Thermo Fisher Scientific, A18967) containing 10% FBS per well a day before compound treatment. Compounds were dispensed using a D300e Digital Dispenser (HP), normalized to 0.5% DMSO, and incubated with cells for 5 h. The assay plate was imaged immediately using an Acumen High Content Imager (TTP Labtech) with 488 nm and 561 nm lasers in a 2 μm × 1 μm grid per well format. The resulting images were analyzed using CellProfiler<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(23)</a> or Cellista Acumen software (TTP Labtech). For Cellista analysis, the signal was quantified as a ratio of 488 nm (EGFP): total (FL-2 Peak Intensity) and 561 nm (mCherry): total (FL-3 Peak Intensity) and normalized to DMSO. For CellProfiler analysis, a series of image analysis steps (“image analysis pipeline”) were constructed. First, the red and green channels were aligned and cropped to target the middle of each well (to avoid analysis of heavily clumped cells at the edges), and a background illumination function was calculated for both red and green channels of each well individually and subtracted to correct for illumination variations across the 384-well plate from various sources of errors. An additional step was then applied to the green channel to suppress the analysis of large autofluorescent artifacts and enhance the analysis of cell-specific fluorescence by way of selecting for objects under a given size, 30 A.U., and with a given shape, speckles. mCherry-positive cells were then identified in the red channel filtering for objects between 8 and 60 pixels in diameter and using intensity to distinguish between clumped objects. The green channel was then segmented into EGFP-positive and -negative areas, and objects were labeled as EGFP-positive if at least 40% of it overlapped with an EGFP-positive area. The fraction of EGFP-positive cells/mCherry-positive cells in each well was then calculated, and the green and red images were rescaled for visualization. The values for the concentrations that lead to a 50% degradation (DC<sub>50</sub>) were calculated using the nonlinear fit variable slope model in GraphPad Prism software.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Cellular Degradation Assays Using HiBit</h3><div class="NLM_p last">Flp293T cells with simultaneous stable expression of BRD4<sub>BD1</sub>L94V–EGFP–P2A–mCherry under CMV promoter and HiBit–FKBP12 F36V–KRAS G12C under PGK promoter were seeded at a 40% confluency in a 384-well plate (Corning, 3573) at 50 μL per well in FluoroBrite DMEM media (Thermo Fisher Scientific, A18967) containing 10% FBS. Compounds were dispensed using a D300e Digital Dispenser (HP) and normalized to 0.5% DMSO. For the simultaneous degradation study, the assay plate was read using Acumen as described above at a 5 h time point and then using the NanoGlo HiBit lytic assay at a 7 h time point. HiBit assay was performed as described in the manufacturer’s instruction (Promega, N3030), and 15 μL of the premixed detection reagent was added to the assay plate, incubated for 15 min, and the luminescence signal was quantified using a PHERAstar FS microplate reader (BMG Labtech). Data was analyzed using GraphPad Prism software.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Sample Preparation TMT LC-MS3 Mass Spectrometry</h3><div class="NLM_p">MOLT4 cells were treated with DMSO (biological triplicates) or 1 μM of compound <b>XY-06-011</b> or 1 μM of <b>XY-06-007</b> (in biological singlicate) for 5 h, and cells were harvested by centrifugation. A lysis buffer (8 M urea, 50 mM NaCl, 50 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (EPPS) pH 8.5, protease and phosphatase inhibitors from Roche) was added to the cell pellets and homogenized by 20 passes through a 21-gauge (1.25 in. long) needle to achieve a cell lysate with a protein concentration between 1 and 4 mg/mL. A micro-BCA assay (Pierce) was used to determine the final protein concentration in the cell lysate. Two hundred micrograms of protein for each sample was reduced and alkylated as previously described.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(32)</a></div><div class="NLM_p">Proteins were precipitated using methanol/chloroform. In brief, four volumes of methanol were added to the cell lysate, followed by one volume of chloroform, and finally three volumes of water. The mixture was vortexed and centrifuged to separate the chloroform phase from the aqueous phase. The precipitated protein was washed with three volumes of methanol, centrifuged, and the resulting washed precipitated protein was allowed to air-dry. The precipitated protein was resuspended in 4 M urea, 50 mM HEPES pH 7.4, followed by dilution to 1 M urea with the addition of 200 mM EPPS, pH 8. Proteins were first digested with LysC (1:50; enzyme/protein) for 12 h at room temperature. The LysC digestion was diluted to 0.5 M urea with 200 mM EPPS pH 8 followed by digestion with trypsin (1:50; enzyme/protein) for 6 h at 37 °C. Tandem mass tag (TMT) reagents (Thermo Fisher Scientific) were dissolved in anhydrous acetonitrile (ACN) according to the manufacturer’s instructions. Anhydrous ACN was added to each peptide sample to a final concentration of 30% v/v, and labeling was induced with the addition of TMT reagent to each sample at a ratio of 1:4 peptide/TMT label. The 10-plex labeling reactions were performed for 1.5 h at room temperature, and the reaction was quenched by the addition of hydroxylamine to a final concentration of 0.3% for 15 minutes at room temperature. The sample channels were combined at a 1:1:1:1:1:1:1:1:1:1:1 ratio, desalted using C<sub>18</sub> solid-phase extraction cartridges (Waters), and analyzed by LC-MS for channel ratio comparison. Samples were then combined using the adjusted volumes determined in the channel ratio analysis and dried down in a speed vacuum. The combined sample was then resuspended in 1% formic acid and acidified (pH 2–3) before being subjected to desalting with C18 SPE (Sep-Pak, Waters). Samples were then offline-fractionated into 96 fractions by high pH reverse-phase HPLC (Agilent LC1260) through an aeris peptide xb-c18 column (phenomenex) with mobile phase A containing 5% acetonitrile and 10 mM NH<sub>4</sub>HCO<sub>3</sub> in LC-MS-grade H<sub>2</sub>O and mobile phase B containing 90% acetonitrile and 10 mM NH<sub>4</sub>HCO<sub>3</sub> in LC-MS-grade H<sub>2</sub>O (both pH 8.0). The 96 resulting fractions were then pooled in a noncontinuous manner into 24 fractions, and these fractions were used for subsequent mass spectrometry analysis.</div><div class="NLM_p">Data was collected using an Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific, San Jose, CA, USA) coupled with a Proxeon EASY-nLC 1200 LC pump (Thermo Fisher Scientific). Peptides were separated on an EasySpray ES803 75 μm inner diameter microcapillary column (Thermo Fisher Scientific). Peptides were separated using a 190 min gradient of 6–27% acetonitrile in 1.0% formic acid with a flow rate of 300 nL/min.</div><div class="NLM_p last">Each analysis used an MS3-based TMT method as described previously.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(33)</a> The data was acquired using a mass range of <i>m</i>/<i>z</i> 340–1350, a resolution of 120 000, an AGC target of 5 × 10<sup>5</sup>, a maximum injection time of 100 ms, and a dynamic exclusion of 120 s for the peptide measurements in the Orbitrap. Data-dependent MS2 spectra were acquired in the ion trap with a normalized collision energy (NCE) set at 35%, an AGC target set to 1.8 × 10<sup>4</sup>, and a maximum injection time of 120 ms. MS3 scans were acquired in the Orbitrap with a HCD collision energy set to 55%, an AGC target set to 2 × 10<sup>5</sup>, a maximum injection time of 150 ms, a resolution at 50 000, and with a maximum synchronous precursor selection (SPS) precursors set to 10.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> LC-MS Data Analysis</h3><div class="NLM_p last">Proteome Discoverer 2.2 (Thermo Fisher Scientific) was used for .RAW file processing and controlling peptide and protein level false discovery rates, assembling proteins from peptides and protein quantification from peptides. MS/MS spectra were searched against a Uniprot human database (September 2016) with both the forward and reverse sequences. Database search criteria are as follows: a tryptic with two missed cleavages, a precursor mass tolerance of 20 ppm, a fragment ion mass tolerance of 0.6 Da, a static alkylation of cysteine (57.02146 Da), a static TMT labeling of lysine residues and N-termini of peptides (229.16293 Da), and a variable oxidation of methionine (15.99491 Da). TMT reporter ion intensities were measured using a 0.003 Da window around the theoretical <i>m</i>/<i>z</i> for each reporter ion in the MS3 scan. Peptide spectral matches with poor-quality MS3 spectra were excluded from quantitation (summed signal-to-noise across 11 channels < 100 and precursor isolation specificity < 0.5), and the resulting data was filtered to only include proteins that had a minimum of two unique peptides identified. Reporter ion intensities were normalized and scaled using in-house scripts in the R framework.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(34)</a> Statistical analysis was carried out using the limma package within the R framework.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(22)</a></div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Microsome Stability Assay</h3><div class="NLM_p last">In vitro metabolic study was conducted at the DMPK core of the Scripps Research Institute, FL. In brief, microsome stability was evaluated by incubating a 1 μM compound with 1 mg/mL mouse hepatic microsomes in 100 mM KPi, pH 7.4. The reaction was initiated by adding NADPH (1 mM final concentration). Aliquots are removed at 0, 5, 10, 20, 40, and 60 minutes and added to acetonitrile (5X v:v) to stop the reaction and precipitate the protein. NADPH dependence of the reaction was evaluated with the addition of no-NADPH control samples. At the end of the assay, the samples were centrifuged through a Millipore Multiscreen Solvinter 0.45 μm low binding PTFE hydrophilic filter plate and analyzed by LC-MS/MS. Data was log-transformed and represented as half-life.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> PK Profiling</h3><div class="NLM_p last">Pharmacokinetic assessment was performed at Shanghai ChemPartners Co., Ltd. All experiments were conducted under the protocols approved by the Animal Care and Use Committee of the Dana-Farber Cancer Institute. Six to eight weeks old male C57BL/6 mice (Shanghai SLAC Laboratory Animal Co. Ltd.) were used. In brief, compounds <b>XY-06-007</b> and <b>XY-06-011</b> were administered via tail vein injection (iv) (formulation: 0.4 mg/mL 5% DMSO + 5% solutol HS15 + 90% saline) at a dose of 2 mg/kg or via intraperitoneal injection (ip) (formulation: 1.0 mg/mL 5% DMSO + 5% solutol HS15 + 90% saline) at a dose of 10 mg/kg to C57BL/6 male mice. Approximately 110 μL of blood/time point was collected into the K<sub>2</sub>EDTA tube via facial vein for bleeding for the time points: 0.083, 0.25, 0.5, 1, 2, 4, 8, and 24 h. Three mice per time point, and a total of 18 mice, were used. Blood samples were put on wet ice and centrifuged to obtain plasma samples (2000 g, 5 min under 4 °C) within 15 minutes. Plasma samples were separated by centrifugation of whole blood and stored below −70 °C until bioanalysis. For plasma samples, an aliquot of 30 μL sample was added with 200 μL of IS (Diclofenac, 300 ng/mL in ACN). The mixture was vortexed for 10 min at 750 rpm and centrifuged at 6000 rpm for 10 min. An aliquot of 0.5 μL supernatant was injected for LC-MS/MS analysis. For 10-fold diluted plasma samples, an aliquot of 3 μL plasma sample was diluted with a 27 μL blank plasma and mixed well to achieve a factor of 10. Then, 200 μL of IS (Diclofenac, 300 ng/mL in ACN) was added for protein precipitation. The following processing procedure was the same as that of the undiluted plasma samples. The mixture was vortexed for 10 min at 750 rpm and centrifuged at 6000 rpm for 10 min. An aliquot of 0.5 μL supernatant was injected for LC-MS/MS analysis. MS analysis was performed using an SCIEX Triple Quad 6500+ LC-MS/MS system. WinNonlin V 6.4 statistics software (Pharsight Corporation, CA, USA) will be used to generate pharmacokinetics parameters (CL, <i>V</i><sub>ss</sub>, <i>T</i><sub>max</sub>, <i>C</i><sub>max</sub>, AUC, <i>T</i><sub>1/2</sub>, etc.) using noncompartmental model.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00958" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78183" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78183" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00958?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00958</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplementary material containing supplementary figures and characterization of key compounds including NMR and HPLC traces for final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings with the binding and degradation assay data (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">PDB coordinates of the Rosetta protein docking model of the ternary complex of BRD4<sub>BD1</sub>–ZXH-3-26 and CRBN (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf">jm1c00958_si_001.pdf (2.78 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_002.csv">jm1c00958_si_002.csv (1.59 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_003.pdb">jm1c00958_si_003.pdb (329.17 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00958" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18474" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18474" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eric S. Fischer</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7337-6306" title="Orcid link">https://orcid.org/0000-0001-7337-6306</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#86c3f4efe5d9c0eff5e5eee3f4c6e2e0e5efa8eee7f4f0e7f4e2a8e3e2f3"><span class="__cf_email__" data-cfemail="eaaf988389b5ac839989828f98aa8e8c8983c4828b989c8b988ec48f8e9f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Radosław P. Nowak</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuan Xiong</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nadia Kirmani</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joann Kalabathula</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katherine A. Donovan</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-8539-5106" title="Orcid link">https://orcid.org/0000-0002-8539-5106</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas A. Eleuteri</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Christine Yuan</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02215, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>R.P.N. and Y.X. contributed equally. R.P.N., Y.X., and E.S.F. conceived and initiated the study. N.K., J.C.Y., J.K., and R.P.N. expressed protein, constructed cell lines, and performed binding and degradation assays. Y.X. synthesized the molecules. K.A.D. and N.E. performed the proteomics experiments. R.P.N., Y.X., and E.S.F. wrote the manuscript with input from all authors, and all authors approved the final version of the manuscript. E.S.F. supervised the study and acquired funding.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was, in part, supported by a sponsored research agreement between Dana-Farber Cancer Institute and Astellas Pharma. E.S.F. is a Damon Runyon-Rachleff Innovator, in part, supported by the Damon Runyon Cancer Research Foundation (DR50-13).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): E.S.F. is a founder, scientific advisory board (SAB) member and equity holder of Civetta Therapeutics, Jengu Therapeutics (board member), and Neomorph, Inc. E.S.F. is an equity holder of C4 Therapeutics. E.S.F. is or has consulted for to Novartis, AbbVie, Astellas, Deerfield, EcoR1 and Pfizer. The Fischer lab receives or has received research funding from Novartis, Deerfield, AJAX and Astellas. E.S.F, R.P.N and Y.X. are inventors of a patent application related to this work.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3566-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i43">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02274" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02274" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank the members of Fischer Lab for their useful discussions. JQ1-FITC probe was a kind gift from Jun Qi at Dana-Farber Cancer Institute. The authors would also like to thank SBGrid Consortium<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(35)</a> for providing access to Pymol Molecular Graphics software.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the curve</p></td></tr><tr><td class="NLM_term">BRD2</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 2</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">Cas9</td><td class="NLM_def"><p class="first last">CRISPR-associated endonuclease Cas9/Csn1</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">CRISPR</td><td class="NLM_def"><p class="first last">clustered regularly interspaced short palindromic repeats</p></td></tr><tr><td class="NLM_term">IMiDs</td><td class="NLM_def"><p class="first last">immunomodulatory imide drugs</p></td></tr><tr><td class="NLM_term">POI</td><td class="NLM_def"><p class="first last">protein of interest</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">siRNA</td><td class="NLM_def"><p class="first last">small interfering RNA</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau disease tumor suppressor</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04118" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04118" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 35 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: Chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+Chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0ljqFq_VN0EZsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520Chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sievers, Q. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunker, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renneville, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slabicki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liddicoat, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulrahman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikkelsen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, N. H.</span></span> <span> </span><span class="NLM_article-title">Defining the human C2H2 zinc finger degrome targeted by thalidomide analogs through CRBN</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>362</i></span>, <span class="NLM_elocation-id">eaat0572</span> <span class="refDoi"> DOI: 10.1126/science.aat0572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1126%2Fscience.aat0572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=30385546" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=2018&author=Q.+L.+Sieversauthor=G.+Petzoldauthor=R.+D.+Bunkerauthor=A.+Rennevilleauthor=M.+Slabickiauthor=B.+J.+Liddicoatauthor=W.+Abdulrahmanauthor=T.+Mikkelsenauthor=B.+L.+Ebertauthor=N.+H.+Thoma&title=Defining+the+human+C2H2+zinc+finger+degrome+targeted+by+thalidomide+analogs+through+CRBN&doi=10.1126%2Fscience.aat0572"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.aat0572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aat0572%26sid%3Dliteratum%253Aachs%26aulast%3DSievers%26aufirst%3DQ.%2BL.%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DBunker%26aufirst%3DR.%2BD.%26aulast%3DRenneville%26aufirst%3DA.%26aulast%3DSlabicki%26aufirst%3DM.%26aulast%3DLiddicoat%26aufirst%3DB.%2BJ.%26aulast%3DAbdulrahman%26aufirst%3DW.%26aulast%3DMikkelsen%26aufirst%3DT.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DThoma%26aufirst%3DN.%2BH.%26atitle%3DDefining%2520the%2520human%2520C2H2%2520zinc%2520finger%2520degrome%2520targeted%2520by%2520thalidomide%2520analogs%2520through%2520CRBN%26jtitle%3DScience%26date%3D2018%26volume%3D362%26doi%3D10.1126%2Fscience.aat0572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukagawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takisawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanemaki, M.</span></span> <span> </span><span class="NLM_article-title">An auxin-based degron system for the rapid depletion of proteins in nonplant cells</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">917</span>– <span class="NLM_lpage">922</span>, <span class="refDoi"> DOI: 10.1038/nmeth.1401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnmeth.1401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=19915560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVWlsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=917-922&author=K.+Nishimuraauthor=T.+Fukagawaauthor=H.+Takisawaauthor=T.+Kakimotoauthor=M.+Kanemaki&title=An+auxin-based+degron+system+for+the+rapid+depletion+of+proteins+in+nonplant+cells&doi=10.1038%2Fnmeth.1401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">An auxin-based degron system for the rapid depletion of proteins in nonplant cells</span></div><div class="casAuthors">Nishimura, Kohei; Fukagawa, Tatsuo; Takisawa, Haruhiko; Kakimoto, Tatsuo; Kanemaki, Masato</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">917-922</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Plants have evolved a unique system in which the plant hormone auxin directly induces rapid degrdn. of the AUX/IAA family of transcription repressors by a specific form of the SCF E3 ubiquitin ligase.  Other eukaryotes lack the auxin response but share the SCF degrdn. pathway, allowing one to transplant the auxin-inducible degron (AID) system into non-plant cells and use a small mol. to conditionally control protein stability.  The AID system allowed rapid and reversible degrdn. of target proteins in response to auxin and enabled the authors to generate efficient conditional mutants of essential proteins in yeast as well as cell lines derived from chicken, mouse, hamster, monkey and human cells, thus offering a powerful tool to control protein expression and study protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzZNy-0P1i8bVg90H21EOLACvtfcHk0liHfk0fYgh_OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVWlsbzM&md5=298f94ba9750fdbfdcaf6f10c62cd408</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1401%26sid%3Dliteratum%253Aachs%26aulast%3DNishimura%26aufirst%3DK.%26aulast%3DFukagawa%26aufirst%3DT.%26aulast%3DTakisawa%26aufirst%3DH.%26aulast%3DKakimoto%26aufirst%3DT.%26aulast%3DKanemaki%26aufirst%3DM.%26atitle%3DAn%2520auxin-based%2520degron%2520system%2520for%2520the%2520rapid%2520depletion%2520of%2520proteins%2520in%2520nonplant%2520cells%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26spage%3D917%26epage%3D922%26doi%3D10.1038%2Fnmeth.1401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasanna, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salo, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vattulainen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikonen, E.</span></span> <span> </span><span class="NLM_article-title">An efficient auxin-inducible degron system with low basal degradation in human cells</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">866</span>– <span class="NLM_lpage">869</span>, <span class="refDoi"> DOI: 10.1038/s41592-019-0512-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fs41592-019-0512-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=31451765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Citb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=866-869&author=S.+Liauthor=X.+Prasannaauthor=V.+T.+Saloauthor=I.+Vattulainenauthor=E.+Ikonen&title=An+efficient+auxin-inducible+degron+system+with+low+basal+degradation+in+human+cells&doi=10.1038%2Fs41592-019-0512-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient auxin-inducible degron system with low basal degradation in human cells</span></div><div class="casAuthors">Li, Shiqian; Prasanna, Xavier; Salo, Veijo T.; Vattulainen, Ilpo; Ikonen, Elina</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">866-869</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Auxin-inducible degron technol. allows rapid and controlled protein depletion.  However, basal degrdn. without auxin and inefficient auxin-inducible depletion have limited its utility.  We have identified a potent auxin-inducible degron system composed of auxin receptor F-box protein AtAFB2 and short degron miniIAA7.  The system showed minimal basal degrdn. and enabled rapid auxin-inducible depletion of endogenous human transmembrane, cytoplasmic and nuclear proteins in 1 h with robust functional phenotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWMKDdzbT70rVg90H21EOLACvtfcHk0liHfk0fYgh_OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Citb7J&md5=1633f47ab052538a72e3d637b8e60f13</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41592-019-0512-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41592-019-0512-x%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DPrasanna%26aufirst%3DX.%26aulast%3DSalo%26aufirst%3DV.%2BT.%26aulast%3DVattulainen%26aufirst%3DI.%26aulast%3DIkonen%26aufirst%3DE.%26atitle%3DAn%2520efficient%2520auxin-inducible%2520degron%2520system%2520with%2520low%2520basal%2520degradation%2520in%2520human%2520cells%26jtitle%3DNat.%2520Methods%26date%3D2019%26volume%3D16%26spage%3D866%26epage%3D869%26doi%3D10.1038%2Fs41592-019-0512-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneekloth, J. S.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonseca, F. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koldobskiy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Chemical genetic control of protein levels: Selective in vivo targeted degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">3748</span>– <span class="NLM_lpage">3754</span>, <span class="refDoi"> DOI: 10.1021/ja039025z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja039025z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhvVegur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=3748-3754&author=J.+S.+Schneeklothauthor=F.+N.+Fonsecaauthor=M.+Koldobskiyauthor=A.+Mandalauthor=R.+Deshaiesauthor=K.+Sakamotoauthor=C.+M.+Crews&title=Chemical+genetic+control+of+protein+levels%3A+Selective+in+vivo+targeted+degradation&doi=10.1021%2Fja039025z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical genetic control of protein levels: selective in vivo targeted degradation</span></div><div class="casAuthors">Schneekloth, John S., Jr.; Fonseca, Fabiana N.; Koldobskiy, Michael; Mandal, Amit; Deshaies, Raymond; Sakamoto, Kathleen; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3748-3754</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Genetic loss of function anal. is a powerful method for the study of protein function.  However, some cell biol. questions are difficult to address using traditional genetic strategies often due to the lack of appropriate genetic model systems.  Here, we present a general strategy for the design and syntheses of mols. capable of inducing the degrdn. of selected proteins in vivo via the ubiquitin-proteasome pathway.  Western blot and fluorometric analyses indicated the loss of two different targets: green fluorescent protein (GFP) fused with FK506 binding protein (FKBP12) and GFP fused with the androgen receptor (AR), after treatment with PROteolysis TArgeting Chimeric mols. (PROTACS) incorporating a FKBP12 ligand and dihydrotestosterone, resp.  These are the first in vivo examples of direct small mol.-induced recruitment of target proteins to the proteasome for degrdn. upon addn. to cultured cells.  Moreover, PROTAC-mediated protein degrdn. offers a general strategy to create "chem. knockouts," thus opening new possibilities for the control of protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvHYtPWyuke7Vg90H21EOLACvtfcHk0lgM5lanQM4PMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhvVegur0%253D&md5=f0855157e4daf32080a23e1728c4b9cc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fja039025z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja039025z%26sid%3Dliteratum%253Aachs%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26aulast%3DFonseca%26aufirst%3DF.%2BN.%26aulast%3DKoldobskiy%26aufirst%3DM.%26aulast%3DMandal%26aufirst%3DA.%26aulast%3DDeshaies%26aufirst%3DR.%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DChemical%2520genetic%2520control%2520of%2520protein%2520levels%253A%2520Selective%2520in%2520vivo%2520targeted%2520degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D3748%26epage%3D3754%26doi%3D10.1021%2Fja039025z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darricarrere, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">HaloPROTACS: Use of small molecule PROTACs to induce degradation of HaloTag fusion proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1831</span>– <span class="NLM_lpage">1837</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00442</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00442" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1831-1837&author=D.+L.+Buckleyauthor=K.+Rainaauthor=N.+Darricarrereauthor=J.+Hinesauthor=J.+L.+Gustafsonauthor=I.+E.+Smithauthor=A.+H.+Miahauthor=J.+D.+Harlingauthor=C.+M.+Crews&title=HaloPROTACS%3A+Use+of+small+molecule+PROTACs+to+induce+degradation+of+HaloTag+fusion+proteins&doi=10.1021%2Facschembio.5b00442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">HaloPROTACS: Use of Small Molecule PROTACs to Induce Degradation of HaloTag Fusion Proteins</span></div><div class="casAuthors">Buckley, Dennis L.; Raina, Kanak; Darricarrere, Nicole; Hines, John; Gustafson, Jeffrey L.; Smith, Ian E.; Miah, Afjal H.; Harling, John D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1831-1837</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol.-induced protein degrdn. is an attractive strategy for the development of chem. probes.  One method for inducing targeted protein degrdn. involves the use of PROTACs, heterobifunctional mols. that can recruit specific E3 ligases to a desired protein of interest.  PROTACs have been successfully used to degrade numerous proteins in cells, but the peptidic E3 ligase ligands used in previous PROTACs have hindered their development into more mature chem. probes or therapeutics.  The authors report the design of a novel class of PROTACs that incorporate small mol. VHL ligands to successfully degrade HaloTag7 fusion proteins.  These HaloPROTACs will inspire the development of future PROTACs with more drug-like properties.  Addnl., these HaloPROTACs are useful chem. genetic tools, due to their ability to chem. knock down widely used HaloTag7 fusion proteins in a general fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotj-Sgj4TyKrVg90H21EOLACvtfcHk0lgM5lanQM4PMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN&md5=b074c58e983f15c384818777b25e248d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00442%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDarricarrere%26aufirst%3DN.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHaloPROTACS%253A%2520Use%2520of%2520small%2520molecule%2520PROTACs%2520to%2520induce%2520degradation%2520of%2520HaloTag%2520fusion%2520proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1831%26epage%3D1837%26doi%3D10.1021%2Facschembio.5b00442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomoshige, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, M.</span></span> <span> </span><span class="NLM_article-title">Degradation of HaloTag-fused nuclear proteins using bestatin-HaloTag ligand hybrid molecules</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">9746</span>– <span class="NLM_lpage">9750</span>, <span class="refDoi"> DOI: 10.1039/C5OB01395J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1039%2FC5OB01395J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=26338696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOrtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=9746-9750&author=S.+Tomoshigeauthor=M.+Naitoauthor=Y.+Hashimotoauthor=M.+Ishikawa&title=Degradation+of+HaloTag-fused+nuclear+proteins+using+bestatin-HaloTag+ligand+hybrid+molecules&doi=10.1039%2FC5OB01395J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of HaloTag-fused nuclear proteins using bestatin-HaloTag ligand hybrid molecules</span></div><div class="casAuthors">Tomoshige, Shusuke; Naito, Mikihiko; Hashimoto, Yuichi; Ishikawa, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">9746-9750</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors have developed a protein knockdown technol. using hybrid small mols. designed as conjugates of a ligand for the target protein and a ligand for ubiquitin ligase cellular inhibitor of apoptosis protein 1 (cIAP1).  However, this technol. has several limitations.  Here, the authors report the development of a novel protein knockdown system to address these limitations.  In this system, target proteins are fused with HaloTag to provide a common binding site for a degrdn. inducer.  The authors designed and synthesized small mols. consisting of alkyl chloride as the HaloTag-binding degrdn. inducer, which binds to HaloTag, linked to BE04(2), which binds to cIAP1.  Using this system, the authors successfully knocked down HaloTag-fused cAMP responsive element binding protein 1 (HaloTag-CREB1) and HaloTag-fused c-jun (HaloTag-c-jun), which are ligand-unknown nuclear proteins, in living cells.  HaloTag-binding degrdn. inducers can be synthesized easily,and are expected to be useful as biol. tools for pan-degrdn. of HaloTag-fused proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHj9MV1-n3NrVg90H21EOLACvtfcHk0lhEfqZLRSTbUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOrtb3I&md5=9c546a5949aa33937a7fef2bac8bde1a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1039%2FC5OB01395J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB01395J%26sid%3Dliteratum%253Aachs%26aulast%3DTomoshige%26aufirst%3DS.%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DHashimoto%26aufirst%3DY.%26aulast%3DIshikawa%26aufirst%3DM.%26atitle%3DDegradation%2520of%2520HaloTag-fused%2520nuclear%2520proteins%2520using%2520bestatin-HaloTag%2520ligand%2520hybrid%2520molecules%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D9746%26epage%3D9750%26doi%3D10.1039%2FC5OB01395J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tovell, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prescott, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macartney, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span> <span> </span><span class="NLM_article-title">Rapid and reversible knockdown of endogenously tagged endosomal proteins via an optimized HaloPROTAC degrader</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">882</span>– <span class="NLM_lpage">892</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b01016</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b01016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnt1emsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=882-892&author=H.+Tovellauthor=A.+Testaauthor=C.+Maniaciauthor=H.+Zhouauthor=A.+R.+Prescottauthor=T.+Macartneyauthor=A.+Ciulliauthor=D.+R.+Alessi&title=Rapid+and+reversible+knockdown+of+endogenously+tagged+endosomal+proteins+via+an+optimized+HaloPROTAC+degrader&doi=10.1021%2Facschembio.8b01016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid and Reversible Knockdown of Endogenously Tagged Endosomal Proteins via an Optimized HaloPROTAC Degrader</span></div><div class="casAuthors">Tovell, Hannah; Testa, Andrea; Maniaci, Chiara; Zhou, Houjiang; Prescott, Alan R.; Macartney, Thomas; Ciulli, Alessio; Alessi, Dario R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">882-892</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inducing posttranslational protein knockdown is an important approach to probe biol. and validate drug targets.  An efficient strategy to achieve this involves expression of a protein of interest fused to an exogenous tag, allowing tag-directed chem. degraders to mediate protein ubiquitylation and proteasomal degrdn.  Here, the authors combine improved HaloPROTAC degrader probes with CRISPR/Cas9 genome editing technol. to trigger rapid degrdn. of endogenous target proteins.  The optimized probe, HaloPROTAC-E, a chloroalkane conjugate of high-affinity VHL binder VH298, induced reversible degrdn. of two endosomally localized proteins, SGK3 and VPS34, with a DC50 of 3-10 nM.  HaloPROTAC-E induced rapid (∼50% degrdn. after 30 min) and complete (Dmax of ∼95% at 48 h) depletion of Halo-tagged SGK3, blocking downstream phosphorylation of the SGK3 substrate NDRG1.  HaloPROTAC-E more potently induced greater steady state degrdn. of Halo tagged endogenous VPS34 than the previously reported HaloPROTAC3 compd.  Quant. global proteomics revealed that HaloPROTAC-E is remarkably selective inducing only degrdn. of the Halo tagged endogenous VPS34 complex (VPS34, VPS15, Beclin1, and ATG14) and no other proteins were significantly degraded.  This study exemplifies the combination of HaloPROTACs with CRISPR/Cas9 endogenous protein tagging as a useful method to induce rapid and reversible degrdn. of endogenous proteins to interrogate their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpJ5B8is89J7Vg90H21EOLACvtfcHk0lhEfqZLRSTbUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnt1emsrg%253D&md5=ea08079224e09d19c1e9f4ee8147909c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b01016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b01016%26sid%3Dliteratum%253Aachs%26aulast%3DTovell%26aufirst%3DH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DPrescott%26aufirst%3DA.%2BR.%26aulast%3DMacartney%26aufirst%3DT.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DRapid%2520and%2520reversible%2520knockdown%2520of%2520endogenously%2520tagged%2520endosomal%2520proteins%2520via%2520an%2520optimized%2520HaloPROTAC%2520degrader%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D882%26epage%3D892%26doi%3D10.1021%2Facschembio.8b01016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon-Clarke, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation as a powerful research tool in basic biology and drug target discovery</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1038/s41594-020-0438-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fs41594-020-0438-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=32541897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFKgtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=605-614&author=T.+Wuauthor=H.+Yoonauthor=Y.+Xiongauthor=S.+E.+Dixon-Clarkeauthor=R.+P.+Nowakauthor=E.+S.+Fischer&title=Targeted+protein+degradation+as+a+powerful+research+tool+in+basic+biology+and+drug+target+discovery&doi=10.1038%2Fs41594-020-0438-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation as a powerful research tool in basic biology and drug target discovery</span></div><div class="casAuthors">Wu, Tao; Yoon, Hojong; Xiong, Yuan; Dixon-Clarke, Sarah E.; Nowak, Radoslaw P.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">605-614</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Controlled perturbation of protein activity is essential to study protein function in cells and living organisms.  Small mols. that hijack the cellular protein ubiquitination machinery to selectively degrade proteins of interest, so-called degraders, have recently emerged as alternatives to selective chem. inhibitors, both as therapeutic modalities and as powerful research tools.  These systems offer unprecedented temporal and spatial control over protein function.  Here, we review recent developments in this field, with a particular focus on the use of degraders as research tools to interrogate complex biol. problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_qHFQVv3fV7Vg90H21EOLACvtfcHk0lgs7ar6_7V-9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFKgtrzI&md5=b9221cbb6d0146234f24ead09d569736</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fs41594-020-0438-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-020-0438-0%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DT.%26aulast%3DYoon%26aufirst%3DH.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DDixon-Clarke%26aufirst%3DS.%2BE.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DTargeted%2520protein%2520degradation%2520as%2520a%2520powerful%2520research%2520tool%2520in%2520basic%2520biology%2520and%2520drug%2520target%2520discovery%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D605%26epage%3D614%26doi%3D10.1038%2Fs41594-020-0438-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">The dTAG system for immediate and target-specific protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0021-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fs41589-018-0021-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=29581585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=431-441&author=B.+Nabetauthor=J.+M.+Robertsauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=S.+Dastjerdiauthor=A.+Yangauthor=A.+L.+Leggettauthor=M.+A.+Erbauthor=M.+A.+Lawlorauthor=A.+Souzaauthor=T.+G.+Scottauthor=S.+Vittoriauthor=J.+A.+Perryauthor=J.+Qiauthor=G.+E.+Winterauthor=K.+K.+Wongauthor=N.+S.+Grayauthor=J.+E.+Bradner&title=The+dTAG+system+for+immediate+and+target-specific+protein+degradation&doi=10.1038%2Fs41589-018-0021-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The dTAG system for immediate and target-specific protein degradation</span></div><div class="casAuthors">Nabet, Behnam; Roberts, Justin M.; Buckley, Dennis L.; Paulk, Joshiawa; Dastjerdi, Shiva; Yang, Annan; Leggett, Alan L.; Erb, Michael A.; Lawlor, Matthew A.; Souza, Amanda; Scott, Thomas G.; Vittori, Sarah; Perry, Jennifer A.; Qi, Jun; Winter, Georg E.; Wong, Kwok-Kin; Gray, Nathanael S.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">431-441</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Dissection of complex biol. systems requires target-specific control of the function or abundance of proteins.  Genetic perturbations are limited by off-target effects, multicomponent complexity, and irreversibility.  Most limiting is the requisite delay between modulation to exptl. measurement.  To enable the immediate and selective control of single protein abundance, we created a chem. biol. system that leverages the potency of cell-permeable heterobifunctional degraders.  The dTAG system pairs a novel degrader of FKBP12F36V with expression of FKBP12F36V in-frame with a protein of interest.  By transgene expression or CRISPR-mediated locus-specific knock-in, we exemplify a generalizable strategy to study the immediate consequence of protein loss.  Using dTAG, we observe an unexpected superior antiproliferative effect of pan-BET bromodomain degrdn. over selective BRD4 degrdn., characterize immediate effects of KRASG12V loss on proteomic signaling, and demonstrate rapid degrdn. in vivo.  This technol. platform will confer kinetic resoln. to biol. investigation and provide target validation in the context of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqwQvCDe-2I7Vg90H21EOLACvtfcHk0liaeSPdWI0zBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D&md5=d7d9c111c68c84bc3aa9c2aad3f591a6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0021-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0021-8%26sid%3Dliteratum%253Aachs%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DA.%26aulast%3DLeggett%26aufirst%3DA.%2BL.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLawlor%26aufirst%3DM.%2BA.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520dTAG%2520system%2520for%2520immediate%2520and%2520target-specific%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D431%26epage%3D441%26doi%3D10.1038%2Fs41589-018-0021-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seong, B. K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuljanin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohardt, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robichaud, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancias, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Rapid and direct control of target protein levels with VHL-recruiting dTAG molecules</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">4687</span> <span class="refDoi"> DOI: 10.1038/s41467-020-18377-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fs41467-020-18377-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=32948771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFSisrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&author=B.+Nabetauthor=F.+M.+Fergusonauthor=B.+K.+A.+Seongauthor=M.+Kuljaninauthor=A.+L.+Leggettauthor=M.+L.+Mohardtauthor=A.+Robichaudauthor=A.+S.+Conwayauthor=D.+L.+Buckleyauthor=J.+D.+Manciasauthor=J.+E.+Bradnerauthor=K.+Stegmaierauthor=N.+S.+Gray&title=Rapid+and+direct+control+of+target+protein+levels+with+VHL-recruiting+dTAG+molecules&doi=10.1038%2Fs41467-020-18377-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid and direct control of target protein levels with VHL-recruiting dTAG molecules</span></div><div class="casAuthors">Nabet, Behnam; Ferguson, Fleur M.; Seong, Bo Kyung A.; Kuljanin, Miljan; Leggett, Alan L.; Mohardt, Mikaela L.; Robichaud, Amanda; Conway, Amy S.; Buckley, Dennis L.; Mancias, Joseph D.; Bradner, James E.; Stegmaier, Kimberly; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4687</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Chem. biol. strategies for directly perturbing protein homeostasis including the degrdn. tag (dTAG) system provide temporal advantages over genetic approaches and improved selectivity over small mol. inhibitors.  We describe dTAGV-1, an exclusively selective VHL-recruiting dTAG mol., to rapidly degrade FKBP12F36V-tagged proteins. dTAGV-1 overcomes a limitation of previously reported CRBN-recruiting dTAG mols. to degrade recalcitrant oncogenes, supports combination degrader studies and facilitates investigations of protein function in cells and mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpguq1r4d5UyrVg90H21EOLACvtfcHk0liaeSPdWI0zBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFSisrnO&md5=aea2209d730ec0379eb966a27817fdb5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18377-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18377-w%26sid%3Dliteratum%253Aachs%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DSeong%26aufirst%3DB.%2BK.%2BA.%26aulast%3DKuljanin%26aufirst%3DM.%26aulast%3DLeggett%26aufirst%3DA.%2BL.%26aulast%3DMohardt%26aufirst%3DM.%2BL.%26aulast%3DRobichaud%26aufirst%3DA.%26aulast%3DConway%26aufirst%3DA.%2BS.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DMancias%26aufirst%3DJ.%2BD.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DRapid%2520and%2520direct%2520control%2520of%2520target%2520protein%2520levels%2520with%2520VHL-recruiting%2520dTAG%2520molecules%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26doi%3D10.1038%2Fs41467-020-18377-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozamus, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amara, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rollins, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magari, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courage, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerasoli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, D. A.</span></span> <span> </span><span class="NLM_article-title">Redesigning an FKBP–ligand interface to generate chemical dimerizers with novel specificity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">10437</span>– <span class="NLM_lpage">10442</span>, <span class="refDoi"> DOI: 10.1073/pnas.95.18.10437</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1073%2Fpnas.95.18.10437" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=9724721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADyaK1cXlvFWgsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=10437-10442&author=T.+Clacksonauthor=W.+Yangauthor=L.+W.+Rozamusauthor=M.+Hatadaauthor=J.+F.+Amaraauthor=C.+T.+Rollinsauthor=L.+F.+Stevensonauthor=S.+R.+Magariauthor=S.+A.+Woodauthor=N.+L.+Courageauthor=X.+Luauthor=F.+Cerasoliauthor=M.+Gilmanauthor=D.+A.+Holt&title=Redesigning+an+FKBP%E2%80%93ligand+interface+to+generate+chemical+dimerizers+with+novel+specificity&doi=10.1073%2Fpnas.95.18.10437"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Redesigning an FKBP-ligand interface to generate chemical dimerizers with novel specificity</span></div><div class="casAuthors">Clackson, Tim; Yang, Wu; Rozamus, Leonard W.; Hatada, Marcos; Amara, Jane F.; Rollins, Carl T.; Stevenson, Lauren F.; Magari, Shannon R.; Wood, Susan A.; Courage, Nancy L.; Lu, Xiaode; Cerasoli, Franklin, Jr.; Gilman, Michael; Holt, Dennis A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10437-10442</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">FKBP ligand homodimers can be used to activate signaling events inside cells and animals that have been engineered to express fusions between appropriate signaling domains and FKBP.  However, use of these dimerizers in vivo is potentially limited by ligand binding to endogenous FKBP.  We have designed ligands that bind specifically to a mutated FKBP over the wild-type protein by remodeling an FKBP-ligand interface to introduce a specificity binding pocket.  A compd. bearing an Et substituent in place of a carbonyl group exhibited sub-nanomolar affinity and 1000-fold selectivity for a mutant FKBP with a compensating truncation of a phenylalanine residue.  Structural and functional anal. of the new pocket showed that recognition is surprisingly relaxed, with the modified ligand only partially filling the engineered cavity.  We incorporated the specificity pocket into a fusion protein contg. FKBP and the intracellular domain of the Fas receptor.  Cells expressing this modified chimeric protein potently underwent apoptosis in response to AP1903, a homodimer of the modified ligand, both in culture and when implanted into mice.  Remodeled dimerizers such as AP1903 are ideal reagents for controlling the activities of cells that have been modified by gene therapy procedures, without interference from endogenous FKBP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBY_8A9vo2YbVg90H21EOLACvtfcHk0liVKYNtxURM0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlvFWgsb4%253D&md5=20fde72ec4c39f5dadf11b32f416b341</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.18.10437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.18.10437%26sid%3Dliteratum%253Aachs%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DRozamus%26aufirst%3DL.%2BW.%26aulast%3DHatada%26aufirst%3DM.%26aulast%3DAmara%26aufirst%3DJ.%2BF.%26aulast%3DRollins%26aufirst%3DC.%2BT.%26aulast%3DStevenson%26aufirst%3DL.%2BF.%26aulast%3DMagari%26aufirst%3DS.%2BR.%26aulast%3DWood%26aufirst%3DS.%2BA.%26aulast%3DCourage%26aufirst%3DN.%2BL.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DCerasoli%26aufirst%3DF.%26aulast%3DGilman%26aufirst%3DM.%26aulast%3DHolt%26aufirst%3DD.%2BA.%26atitle%3DRedesigning%2520an%2520FKBP%25E2%2580%2593ligand%2520interface%2520to%2520generate%2520chemical%2520dimerizers%2520with%2520novel%2520specificity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1998%26volume%3D95%26spage%3D10437%26epage%3D10442%26doi%3D10.1073%2Fpnas.95.18.10437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanjana, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalem, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offei-Addo, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Transcription control by the ENL YEATS domain in acute leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>543</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">274</span>, <span class="refDoi"> DOI: 10.1038/nature21688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnature21688" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=28241139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2sXks1Gls70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=543&publication_year=2017&pages=270-274&author=M.+A.+Erbauthor=T.+G.+Scottauthor=B.+E.+Liauthor=H.+Xieauthor=J.+Paulkauthor=H.+S.+Seoauthor=A.+Souzaauthor=J.+M.+Robertsauthor=S.+Dastjerdiauthor=D.+L.+Buckleyauthor=N.+E.+Sanjanaauthor=O.+Shalemauthor=B.+Nabetauthor=R.+Zeidauthor=N.+K.+Offei-Addoauthor=S.+Dhe-Paganonauthor=F.+Zhangauthor=S.+H.+Orkinauthor=G.+E.+Winterauthor=J.+E.+Bradner&title=Transcription+control+by+the+ENL+YEATS+domain+in+acute+leukaemia&doi=10.1038%2Fnature21688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription control by the ENL YEATS domain in acute leukemia</span></div><div class="casAuthors">Erb, Michael A.; Scott, Thomas G.; Li, Bin E.; Xie, Huafeng; Paulk, Joshiawa; Seo, Hyuk-Soo; Souza, Amanda; Roberts, Justin M.; Dastjerdi, Shiva; Buckley, Dennis L.; Sanjana, Neville E.; Shalem, Ophir; Nabet, Behnam; Zeid, Rhamy; Offei-Addo, Nana K.; Dhe-Paganon, Sirano; Zhang, Feng; Orkin, Stuart H.; Winter, Georg E.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">543</span>
        (<span class="NLM_cas:issue">7644</span>),
    <span class="NLM_cas:pages">270-274</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations producing a chimeric MLL oncogene give rise to a highly aggressive acute leukemia assocd. with poor clin. outcome.  The preferential involvement of chromatin-assocd. factors as MLL fusion partners belies a dependency on transcription control.  Despite recent progress made in targeting chromatin regulators in cancer, available therapies for this well-characterized disease remain inadequate, prompting the need to identify new targets for therapeutic intervention.  Here, using unbiased CRISPR-Cas9 technol. to perform a genome-scale loss-of-function screen in an MLL-AF4-pos. acute leukemia cell line, we identify ENL as an unrecognized gene that is specifically required for proliferation in vitro and in vivo.  To explain the mechanistic role of ENL in leukemia pathogenesis and dynamic transcription control, a chem. genetic strategy was developed to achieve targeted protein degrdn.  Acute loss of ENL suppressed the initiation and elongation of RNA polymerase II at active genes genome-wide, with pronounced effects at genes featuring a disproportionate ENL load.  Notably, an intact YEATS chromatin-reader domain was essential for ENL-dependent leukemic growth.  Overall, these findings identify a dependency factor in acute leukemia and suggest a mechanistic rationale for disrupting the YEATS domain in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8JlG7P9bHArVg90H21EOLACvtfcHk0liVKYNtxURM0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXks1Gls70%253D&md5=5b889ab323bcf91f55b6c0db02878d04</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature21688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature21688%26sid%3Dliteratum%253Aachs%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DLi%26aufirst%3DB.%2BE.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DSeo%26aufirst%3DH.%2BS.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DSanjana%26aufirst%3DN.%2BE.%26aulast%3DShalem%26aufirst%3DO.%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DZeid%26aufirst%3DR.%26aulast%3DOffei-Addo%26aufirst%3DN.%2BK.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DTranscription%2520control%2520by%2520the%2520ENL%2520YEATS%2520domain%2520in%2520acute%2520leukaemia%26jtitle%3DNature%26date%3D2017%26volume%3D543%26spage%3D270%26epage%3D274%26doi%3D10.1038%2Fnature21688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLauchlan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">MELK is not necessary for the proliferation of basal-like breast cancer cells</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e26693</span> <span class="refDoi"> DOI: 10.7554/eLife.26693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.7554%2FeLife.26693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=28926338" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlOit7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&author=H.+T.+Huangauthor=H.+S.+Seoauthor=T.+Zhangauthor=Y.+Wangauthor=B.+Jiangauthor=Q.+Liauthor=D.+L.+Buckleyauthor=B.+Nabetauthor=J.+M.+Robertsauthor=J.+Paulkauthor=S.+Dastjerdiauthor=G.+E.+Winterauthor=H.+McLauchlanauthor=J.+Moranauthor=J.+E.+Bradnerauthor=M.+J.+Eckauthor=S.+Dhe-Paganonauthor=J.+J.+Zhaoauthor=N.+S.+Gray&title=MELK+is+not+necessary+for+the+proliferation+of+basal-like+breast+cancer+cells&doi=10.7554%2FeLife.26693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MELK is not necessary for the proliferation of basal-like breast cancer cells</span></div><div class="casAuthors">Huang, Hai-Tsang; Seo, Hyuk-Soo; Zhang, Tinghu; Wang, Yubao; Jiang, Baishan; Li, Qing; Buckley, Dennis L.; Nabet, Behnam; Roberts, Justin M.; Paulk, Joshiawa; Dastjerdi, Shiva; Winter, Georg E.; McLauchlan, Hilary; Moran, Jennifer; Bradner, James E.; Eck, Michael J.; Dhe-Paganon, Sirano; Zhao, Jean J.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e26693/1-e26693/29</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Thorough preclin. target validation is essential for the success of drug discovery efforts.  In this study, we combined chem. and genetic perturbants, including the development of a novel selective maternal embryonic leucine zipper kinase (MELK) inhibitor HTH-01-091, CRISPR/ Cas9-mediated MELK knockout, a novel chem.-induced protein degrdn. strategy, RNA interference and CRISPR interference to validate MELK as a therapeutic target in basal-like breast cancers (BBC).  In common culture conditions, we found that small mol. inhibition, genetic deletion, or acute depletion of MELK did not significantly affect cellular growth.  This discrepancy to previous findings illuminated selectivity issues of the widely used MELK inhibitor OTSSP167, and potential off-target effects of MELK-targeting short hairpins.  The different genetic and chem. tools developed here allow for the identification and validation of any causal roles MELK may play in cancer biol., which will be required to guide future MELK drug discovery efforts.  Furthermore, our study provides a general framework for preclin. target validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW6AnVwhH8j7Vg90H21EOLACvtfcHk0lgI2cOz3DwpYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlOit7jE&md5=f5abb6df4e61df2b0cc24beb3005b176</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.7554%2FeLife.26693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.26693%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%2BT.%26aulast%3DSeo%26aufirst%3DH.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DMcLauchlan%26aufirst%3DH.%26aulast%3DMoran%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DMELK%2520is%2520not%2520necessary%2520for%2520the%2520proliferation%2520of%2520basal-like%2520breast%2520cancer%2520cells%26jtitle%3DeLife%26date%3D2017%26volume%3D6%26doi%3D10.7554%2FeLife.26693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buzko, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Magic bullets for protein kinases</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">172</span>, <span class="refDoi"> DOI: 10.1016/S0962-8924(01)01928-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1016%2FS0962-8924%2801%2901928-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=11306297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD3MXisF2msrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=167-172&author=A.+C.+Bishopauthor=O.+Buzkoauthor=K.+M.+Shokat&title=Magic+bullets+for+protein+kinases&doi=10.1016%2FS0962-8924%2801%2901928-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Magic bullets for protein kinases</span></div><div class="casAuthors">Bishop, A. C.; Buzko, O.; Shokat, K. M.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">167-172</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review and discussion with 52 refs.  A chem.-genetic method for the generation of target-specific protein kinase inhibitors has been developed recently.  This strategy utilizes a functionally silent active-site mutation to sensitize a target kinase to inhibition by a small mol. that does not inhibit wild-type kinases.  Tyrosine and serine/threonine kinases are equally amenable to the drug-sensitization approach, which has been used to generate selective inhibitors of mutant Src-family kinases, Abl-family kinases, cyclin-dependent kinases, mitogen-activated kinases, p21-activated kinases, and Ca2+/calmodulin-dependent kinases.  The designed inhibitors are specific for the sensitized kinase in a cellular background where the wild-type kinase has been inactivated.  By these means, kinase-sensitization has been used systematically to generate and analyze conditional alleles of several yeast protein kinases in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN2pRnErAok7Vg90H21EOLACvtfcHk0ljiaLOQpAPiyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXisF2msrY%253D&md5=e4eea34d0dcea9c1470b7bb90dbd3f37</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS0962-8924%2801%2901928-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0962-8924%252801%252901928-6%26sid%3Dliteratum%253Aachs%26aulast%3DBishop%26aufirst%3DA.%2BC.%26aulast%3DBuzko%26aufirst%3DO.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DMagic%2520bullets%2520for%2520protein%2520kinases%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2001%26volume%3D11%26spage%3D167%26epage%3D172%26doi%3D10.1016%2FS0962-8924%2801%2901928-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Islam, K.</span></span> <span> </span><span class="NLM_article-title">The bump-and-hole tactic: Expanding the scope of chemical genetics</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1171</span>– <span class="NLM_lpage">1184</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1016%2Fj.chembiol.2018.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=30078633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVersL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1171-1184&author=K.+Islam&title=The+bump-and-hole+tactic%3A+Expanding+the+scope+of+chemical+genetics&doi=10.1016%2Fj.chembiol.2018.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The Bump-and-Hole Tactic: Expanding the Scope of Chemical Genetics</span></div><div class="casAuthors">Islam, Kabirul</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1171-1184</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Successful mapping of the human genome has sparked a widespread interest in deciphering functional information encoded in gene sequences.  However, because of the high degree of conservation in sequences along with topol. and biochem. similarities among members of a protein superfamily, uncovering physiol. role of a particular protein has been a challenging task.  Chem. genetic approaches have made significant contributions toward understanding protein function.  One such effort, dubbed the bump-and-hole approach, has convincingly demonstrated that engineering at the protein-small mol. interface constitutes a powerful method for elucidating the function of a specific gene product.  By manipulating the steric component of protein-ligand interactions in a complementary manner, an orthogonal system is developed to probe a specific enzyme-cofactor pair without interference from related members.  This article outlines current efforts to expand the approach for diverse protein classes and their applications.  Potential future innovations to address contemporary biol. problems are highlighted as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZJbGz0HI7EbVg90H21EOLACvtfcHk0ljiaLOQpAPiyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVersL3P&md5=e2224810004276552f3564040f9957f6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DIslam%26aufirst%3DK.%26atitle%3DThe%2520bump-and-hole%2520tactic%253A%2520Expanding%2520the%2520scope%2520of%2520chemical%2520genetics%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D1171%26epage%3D1184%26doi%3D10.1016%2Fj.chembiol.2018.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">New synthetic routes to triazolo-benzodiazepine analogues: Expanding the scope of the bump-and-hole approach for selective bromo and extra-terminal (BET) bromodomain inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1492</span>– <span class="NLM_lpage">1500</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01135</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01135" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1492-1500&author=M.+G.+Baudauthor=E.+Lin-Shiaoauthor=M.+Zengerleauthor=C.+Tallantauthor=A.+Ciulli&title=New+synthetic+routes+to+triazolo-benzodiazepine+analogues%3A+Expanding+the+scope+of+the+bump-and-hole+approach+for+selective+bromo+and+extra-terminal+%28BET%29+bromodomain+inhibition&doi=10.1021%2Facs.jmedchem.5b01135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Zengerle, Michael; Tallant, Cynthia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1492-1500</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe new synthetic routes developed toward a range of substituted analogs of bromo and extra-terminal (BET) bromodomain inhibitors I-BET762/JQ1 based on the triazolo-benzodiazepine scaffold.  These new routes allow for the derivatization of the methoxyphenyl and chlorophenyl rings, in addn. to the diazepine ternary center and the side chain methylene moiety.  Substitution at the level of the side chain methylene afforded compds. targeting specifically and potently engineered BET bromodomains designed as part of a bump and hole approach.  We further demonstrate that marked selectivity for the second over the first bromodomain can be achieved with an indole deriv. that exploits differential interaction with an aspartate/histidine conservative substitution on the BC loop of BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUpYVun8F94bVg90H21EOLACvtfcHk0ljiFKx2iX6EQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE&md5=65dd39e3f7e0bc52169ffb1466218c43</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01135%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DNew%2520synthetic%2520routes%2520to%2520triazolo-benzodiazepine%2520analogues%253A%2520Expanding%2520the%2520scope%2520of%2520the%2520bump-and-hole%2520approach%2520for%2520selective%2520bromo%2520and%2520extra-terminal%2520%2528BET%2529%2520bromodomain%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1492%26epage%3D1500%26doi%3D10.1021%2Facs.jmedchem.5b01135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardote, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pschibul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, T. T.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>346</i></span>,  <span class="NLM_fpage">638</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1126/science.1249830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1126%2Fscience.1249830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=25323695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=638-641&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=T.+Cardoteauthor=C.+Tallantauthor=A.+Pschibulauthor=K.-H.+Chanauthor=M.+Zengerleauthor=J.+R.+Garciaauthor=T.+T.-L.+Kwanauthor=F.+M.+Fergusonauthor=A.+Ciulli&title=A+bump-and-hole+approach+to+engineer+controlled+selectivity+of+BET+bromodomain+chemical+probes&doi=10.1126%2Fscience.1249830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Cardote, Teresa; Tallant, Cynthia; Pschibul, Annica; Chan, Kwok-Ho; Zengerle, Michael; Garcia, Jordi R.; Kwan, Terence T.-L.; Ferguson, Fleur M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6209</span>),
    <span class="NLM_cas:pages">638-641</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Small mols. are useful tools for probing the biol. function and therapeutic potential of individual proteins, but achieving selectivity is challenging when the target protein shares structural domains with other proteins.  The bromodomain and extra-terminal (BET) proteins have attracted interest because of their roles in transcriptional regulation, epigenetics, and cancer.  The BET bromodomains (protein interaction modules that bind acetyl-lysine) have been targeted by potent small-mol. inhibitors, but these inhibitors lack selectivity for individual family members.  We developed an Et deriv. of an existing small-mol. inhibitor, I-BET/JQ1, and showed that it binds leucine/alanine mutant bromodomains with nanomolar affinity and achieves up to 540-fold selectivity relative to wild-type bromodomains.  Cell culture studies showed that blockade of the first bromodomain alone is sufficient to displace a specific BET protein, Brd4, from chromatin.  Expansion of this approach could help identify the individual roles of single BET proteins in human physiol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9EnzLtvUKwbVg90H21EOLACvtfcHk0ljiFKx2iX6EQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL&md5=04fe967e4681fb4b2b3168f4d5f8ff81</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fscience.1249830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1249830%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DCardote%26aufirst%3DT.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DPschibul%26aufirst%3DA.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DJ.%2BR.%26aulast%3DKwan%26aufirst%3DT.%2BT.-L.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DA%2520bump-and-hole%2520approach%2520to%2520engineer%2520controlled%2520selectivity%2520of%2520BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DScience%26date%3D2014%26volume%3D346%26spage%3D638%26epage%3D641%26doi%3D10.1126%2Fscience.1249830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Runcie, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Beurden, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L. C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulthard, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Optimization of a “bump-and-hole” approach to allele-selective BET bromodomain inhibition</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2452</span>– <span class="NLM_lpage">2468</span>, <span class="refDoi"> DOI: 10.1039/C7SC02536J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1039%2FC7SC02536J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=29732121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=2452-2468&author=A.+C.+Runcieauthor=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Testaauthor=L.+van+Beurdenauthor=M.+G.+J.+Baudauthor=O.+Epemoluauthor=L.+C.+J.+Ellisauthor=K.+D.+Readauthor=V.+Coulthardauthor=A.+Brienauthor=A.+Ciulli&title=Optimization+of+a+%E2%80%9Cbump-and-hole%E2%80%9D+approach+to+allele-selective+BET+bromodomain+inhibition&doi=10.1039%2FC7SC02536J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a "bump-and-hole" approach to allele-selective BET bromodomain inhibition</span></div><div class="casAuthors">Runcie, A. C.; Zengerle, M.; Chan, K.-H.; Testa, A.; van Beurden, L.; Baud, M. G. J.; Epemolu, O.; Ellis, L. C. J.; Read, K. D.; Coulthard, V.; Brien, A.; Ciulli, A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2452-2468</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Allele-specific chem. genetics enables selective inhibition within families of highly-conserved proteins.  The four BET (bromodomain & extra-terminal domain) proteins - BRD2, BRD3, BRD4 and BRDT bind acetylated chromatin via their bromodomains and regulate processes such as cell proliferation and inflammation.  BET bromodomains are of particular interest, as they are attractive therapeutic targets but existing inhibitors are pan-selective.  We previously established a bump-&-hole system for the BET bromodomains, pairing a leucine/alanine mutation with an ethyl-derived analog of an established benzodiazepine scaffold.  Here we optimize upon this system with the introduction of a more conservative and less disruptive leucine/valine mutation.  Extensive structure-activity-relationships of diverse benzodiazepine analogs guided the development of potent, mutant-selective inhibitors with desirable physiochem. properties.  The active enantiomer of our best compd. - 9-ME-1 - shows ~ 200 nM potency, >100-fold selectivity for the L/V mutant over wild-type and excellent DMPK properties.  Through a variety of in vitro and cellular assays we validate the capabilities of our optimized system, and then utilize it to compare the relative importance of the first and second bromodomains to chromatin binding.  These expts. confirm the primacy of the first bromodomain in all BET proteins, but also significant variation in the importance of the second bromodomain.  We also show that, despite having a minor role in chromatin recognition, BRD4 BD2 is still essential for gene expression, likely through the recruitment of non-histone proteins.  The disclosed inhibitor:mutant pair provides a powerful tool for future cellular and in vivo target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq47qHG4-zFurVg90H21EOLACvtfcHk0ljQxgV-KuWBug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D&md5=9a44bdccabca9f1d4dfdc983250d9aa3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC7SC02536J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC02536J%26sid%3Dliteratum%253Aachs%26aulast%3DRuncie%26aufirst%3DA.%2BC.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3Dvan%2BBeurden%26aufirst%3DL.%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DEllis%26aufirst%3DL.%2BC.%2BJ.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCoulthard%26aufirst%3DV.%26aulast%3DBrien%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DOptimization%2520of%2520a%2520%25E2%2580%259Cbump-and-hole%25E2%2580%259D%2520approach%2520to%2520allele-selective%2520BET%2520bromodomain%2520inhibition%26jtitle%3DChem.%2520Sci.%26date%3D2018%26volume%3D9%26spage%3D2452%26epage%3D2468%26doi%3D10.1039%2FC7SC02536J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukagawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takisawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanemaki, M.</span></span> <span> </span><span class="NLM_article-title">An auxin-based degron system for the rapid depletion of proteins in nonplant cells</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">917</span>– <span class="NLM_lpage">922</span>, <span class="refDoi"> DOI: 10.1038/nmeth.1401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnmeth.1401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=19915560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVWlsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=917-922&author=K.+Nishimuraauthor=T.+Fukagawaauthor=H.+Takisawaauthor=T.+Kakimotoauthor=M.+Kanemaki&title=An+auxin-based+degron+system+for+the+rapid+depletion+of+proteins+in+nonplant+cells&doi=10.1038%2Fnmeth.1401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">An auxin-based degron system for the rapid depletion of proteins in nonplant cells</span></div><div class="casAuthors">Nishimura, Kohei; Fukagawa, Tatsuo; Takisawa, Haruhiko; Kakimoto, Tatsuo; Kanemaki, Masato</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">917-922</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Plants have evolved a unique system in which the plant hormone auxin directly induces rapid degrdn. of the AUX/IAA family of transcription repressors by a specific form of the SCF E3 ubiquitin ligase.  Other eukaryotes lack the auxin response but share the SCF degrdn. pathway, allowing one to transplant the auxin-inducible degron (AID) system into non-plant cells and use a small mol. to conditionally control protein stability.  The AID system allowed rapid and reversible degrdn. of target proteins in response to auxin and enabled the authors to generate efficient conditional mutants of essential proteins in yeast as well as cell lines derived from chicken, mouse, hamster, monkey and human cells, thus offering a powerful tool to control protein expression and study protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzZNy-0P1i8bVg90H21EOLACvtfcHk0ljQxgV-KuWBug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVWlsbzM&md5=298f94ba9750fdbfdcaf6f10c62cd408</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1401%26sid%3Dliteratum%253Aachs%26aulast%3DNishimura%26aufirst%3DK.%26aulast%3DFukagawa%26aufirst%3DT.%26aulast%3DTakisawa%26aufirst%3DH.%26aulast%3DKakimoto%26aufirst%3DT.%26aulast%3DKanemaki%26aufirst%3DM.%26atitle%3DAn%2520auxin-based%2520degron%2520system%2520for%2520the%2520rapid%2520depletion%2520of%2520proteins%2520in%2520nonplant%2520cells%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26spage%3D917%26epage%3D922%26doi%3D10.1038%2Fnmeth.1401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safaee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponthier, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancias, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Plasticity in binding confers selectivity in ligand-induced protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0055-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fs41589-018-0055-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=29892083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=706-714&author=R.+P.+Nowakauthor=S.+L.+DeAngeloauthor=D.+Buckleyauthor=Z.+Heauthor=K.+A.+Donovanauthor=J.+Anauthor=N.+Safaeeauthor=M.+P.+Jedrychowskiauthor=C.+M.+Ponthierauthor=M.+Ishoeyauthor=T.+Zhangauthor=J.+D.+Manciasauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=E.+S.+Fischer&title=Plasticity+in+binding+confers+selectivity+in+ligand-induced+protein+degradation&doi=10.1038%2Fs41589-018-0055-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Plasticity in binding confers selectivity in ligand-induced protein degradation</span></div><div class="casAuthors">Nowak, Radoslaw P.; De Angelo, Stephen L.; Buckley, Dennis; He, Zhixiang; Donovan, Katherine A.; An, Jian; Safaee, Nozhat; Jedrychowski, Mark P.; Ponthier, Charles M.; Ishoey, Mette; Zhang, Tinghu; Mancias, Joseph D.; Gray, Nathanael S.; Bradner, James E.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">706-714</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Heterobifunctional small-mol. degraders that induce protein degrdn. through ligase-mediated ubiquitination have shown considerable promise as a new pharmacol. modality.  However, we currently lack a detailed understanding of the mol. basis for target recruitment and selectivity, which is critically required to enable rational design of degraders.  Here we utilized a comprehensive characterization of the ligand-dependent CRBN-BRD4 interaction to demonstrate that binding between proteins that have not evolved to interact is plastic.  Multiple x-ray crystal structures showed that plasticity resulted in several distinct low-energy binding conformations that were selectively bound by ligands.  We demonstrated that computational protein-protein docking can reveal the underlying interprotein contacts and inform the design of a BRD4 selective degrader that can discriminate between highly homologous BET bromodomains.  These findings that plastic interprotein contacts confer selectivity for ligand-induced protein dimerization provide a conceptual framework for the development of heterobifunctional ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreQasCW8bz2LVg90H21EOLACvtfcHk0lhseJ5yLiN5cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK&md5=e4876c43b1eeb4558c23121b7a9fac27</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0055-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0055-y%26sid%3Dliteratum%253Aachs%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DBuckley%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DSafaee%26aufirst%3DN.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DPonthier%26aufirst%3DC.%2BM.%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DMancias%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DPlasticity%2520in%2520binding%2520confers%2520selectivity%2520in%2520ligand-induced%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D706%26epage%3D714%26doi%3D10.1038%2Fs41589-018-0055-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phipson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, G. K.</span></span> <span> </span><span class="NLM_article-title">Limma powers differential expression analyses for RNA-sequencing and microarray studies</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>, <span class="NLM_elocation-id">e47</span> <span class="refDoi"> DOI: 10.1093/nar/gkv007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1093%2Fnar%2Fgkv007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=25605792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFaiu7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&author=M.+E.+Ritchieauthor=B.+Phipsonauthor=D.+Wuauthor=Y.+Huauthor=C.+W.+Lawauthor=W.+Shiauthor=G.+K.+Smyth&title=Limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies&doi=10.1093%2Fnar%2Fgkv007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">limma powers differential expression analyses for RNA-sequencing and microarray studies</span></div><div class="casAuthors">Ritchie, Matthew E.; Phipson, Belinda; Wu, Di; Hu, Yifang; Law, Charity W.; Shi, Wei; Smyth, Gordon K.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e47/1-e47/13</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Limma is an R/Bioconductor software package that provides an integrated soln. for analyzing data from gene expression expts.  It contains rich features for handling complex exptl. designs and for information borrowing to overcome the problem of small sample sizes.  Over the past decade, limma has been a popular choice for gene discovery through differential expression analyses of microarray and high-throughput PCR data.  The package contains particularly strong facilities for reading, normalizing and exploring such data.  Recently, the capabilities of limma have been significantly expanded in two important directions.  First, the package can now perform both differential expression and differential splicing analyses of RNA sequencing (RNA-seq) data.  All the downstream anal. tools previously restricted to microarray data are now available for RNA-seq as well.  These capabilities allow users to analyze both RNA-seq and microarray data with very similar pipelines.  Second, the package is now able to go past the traditional gene-wise expression analyses in a variety of ways, analyzing expression profiles in terms of co-regulated sets of genes or in terms of higher-order expression signatures.  This provides enhanced possibilities for biol. interpretation of gene expression differences.  This article reviews the philosophy and design of the limma package, summarizing both new and historical features, with an emphasis on recent enhancements and features that have not been previously described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJykfsLP7n9bVg90H21EOLACvtfcHk0lj5LezJEMWDcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFaiu7%252FN&md5=e8a5f44160c650ad89adc48a9912bd96</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv007%26sid%3Dliteratum%253Aachs%26aulast%3DRitchie%26aufirst%3DM.%2BE.%26aulast%3DPhipson%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLaw%26aufirst%3DC.%2BW.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DSmyth%26aufirst%3DG.%2BK.%26atitle%3DLimma%2520powers%2520differential%2520expression%2520analyses%2520for%2520RNA-sequencing%2520and%2520microarray%2520studies%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26doi%3D10.1093%2Fnar%2Fgkv007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamprecht, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friman, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guertin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindquist, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golland, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">CellProfiler: Image analysis software for identifying and quantifying cell phenotypes</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">R100</span> <span class="refDoi"> DOI: 10.1186/gb-2006-7-10-r100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1186%2Fgb-2006-7-10-r100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=17076895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252Fjt1emsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&author=A.+E.+Carpenterauthor=T.+R.+Jonesauthor=M.+R.+Lamprechtauthor=C.+Clarkeauthor=I.+H.+Kangauthor=O.+Frimanauthor=D.+A.+Guertinauthor=J.+H.+Changauthor=R.+A.+Lindquistauthor=J.+Moffatauthor=P.+Gollandauthor=D.+M.+Sabatini&title=CellProfiler%3A+Image+analysis+software+for+identifying+and+quantifying+cell+phenotypes&doi=10.1186%2Fgb-2006-7-10-r100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">CellProfiler: image analysis software for identifying and quantifying cell phenotypes</span></div><div class="casAuthors">Carpenter Anne E; Jones Thouis R; Lamprecht Michael R; Clarke Colin; Kang In Han; Friman Ola; Guertin David A; Chang Joo Han; Lindquist Robert A; Moffat Jason; Golland Polina; Sabatini David M</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">R100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Biologists can now prepare and image thousands of samples per day using automation, enabling chemical screens and functional genomics (for example, using RNA interference).  Here we describe the first free, open-source system designed for flexible, high-throughput cell image analysis, CellProfiler.  CellProfiler can address a variety of biological questions quantitatively, including standard assays (for example, cell count, size, per-cell protein levels) and complex morphological assays (for example, cell/organelle shape or subcellular patterns of DNA or protein staining).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkZz-phUrfiVm8bhe6roZYfW6udTcc2eZARFLnET-L8bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252Fjt1emsA%253D%253D&md5=79d0e947559e12792f864d764657d3f4</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2Fgb-2006-7-10-r100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2006-7-10-r100%26sid%3Dliteratum%253Aachs%26aulast%3DCarpenter%26aufirst%3DA.%2BE.%26aulast%3DJones%26aufirst%3DT.%2BR.%26aulast%3DLamprecht%26aufirst%3DM.%2BR.%26aulast%3DClarke%26aufirst%3DC.%26aulast%3DKang%26aufirst%3DI.%2BH.%26aulast%3DFriman%26aufirst%3DO.%26aulast%3DGuertin%26aufirst%3DD.%2BA.%26aulast%3DChang%26aufirst%3DJ.%2BH.%26aulast%3DLindquist%26aufirst%3DR.%2BA.%26aulast%3DMoffat%26aufirst%3DJ.%26aulast%3DGolland%26aufirst%3DP.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DCellProfiler%253A%2520Image%2520analysis%2520software%2520for%2520identifying%2520and%2520quantifying%2520cell%2520phenotypes%26jtitle%3DGenome%2520Biol.%26date%3D2006%26volume%3D7%26doi%3D10.1186%2Fgb-2006-7-10-r100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tutt, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santarosa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knights, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>434</i></span>,  <span class="NLM_fpage">917</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1038/nature03445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnature03445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=15829967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=434&publication_year=2005&pages=917-921&author=H.+Farmerauthor=N.+McCabeauthor=C.+J.+Lordauthor=A.+N.+Tuttauthor=D.+A.+Johnsonauthor=T.+B.+Richardsonauthor=M.+Santarosaauthor=K.+J.+Dillonauthor=I.+Hicksonauthor=C.+Knightsauthor=N.+M.+Martinauthor=S.+P.+Jacksonauthor=G.+C.+Smithauthor=A.+Ashworth&title=Targeting+the+DNA+repair+defect+in+BRCA+mutant+cells+as+a+therapeutic+strategy&doi=10.1038%2Fnature03445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy</span></div><div class="casAuthors">Farmer, Hannah; McCabe, Nuala; Lord, Christopher J.; Tutt, Andrew N. J.; Johnson, Damian A.; Richardson, Tobias B.; Santarosa, Manuela; Dillon, Krystyna J.; Hickson, Ian; Knights, Charlotte; Martin, Niall M. B.; Jackson, Stephen P.; Smith, Graeme C. M.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">434</span>
        (<span class="NLM_cas:issue">7035</span>),
    <span class="NLM_cas:pages">917-921</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">BRCA1 and BRCA2 are important for DNA double-strand break repair by homologous recombination, and mutations in these genes predispose to breast and other cancers.  Poly(ADP-ribose) polymerase (PARP) is an enzyme involved in base excision repair, a key pathway in the repair of DNA single-strand breaks.  The authors show here that BRCA1 or BRCA2 dysfunction unexpectedly and profoundly sensitizes cells to the inhibition of PARP enzymic activity, resulting in chromosomal instability, cell cycle arrest and subsequent apoptosis.  This seems to be because the inhibition of PARP leads to the persistence of DNA lesions normally repaired by homologous recombination.  These results illustrate how different pathways cooperate to repair damage, and suggest that the targeted inhibition of particular DNA repair pathways may allow the design of specific and less toxic therapies for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomVTKE4mFwfrVg90H21EOLACvtfcHk0lj2bh_0TNUVDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrc%253D&md5=03b5ba0adc21a5627fe17ae535f45078</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature03445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03445%26sid%3Dliteratum%253Aachs%26aulast%3DFarmer%26aufirst%3DH.%26aulast%3DMcCabe%26aufirst%3DN.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DTutt%26aufirst%3DA.%2BN.%26aulast%3DJohnson%26aufirst%3DD.%2BA.%26aulast%3DRichardson%26aufirst%3DT.%2BB.%26aulast%3DSantarosa%26aufirst%3DM.%26aulast%3DDillon%26aufirst%3DK.%2BJ.%26aulast%3DHickson%26aufirst%3DI.%26aulast%3DKnights%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DN.%2BM.%26aulast%3DJackson%26aufirst%3DS.%2BP.%26aulast%3DSmith%26aufirst%3DG.%2BC.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DTargeting%2520the%2520DNA%2520repair%2520defect%2520in%2520BRCA%2520mutant%2520cells%2520as%2520a%2520therapeutic%2520strategy%26jtitle%3DNature%26date%3D2005%26volume%3D434%26spage%3D917%26epage%3D921%26doi%3D10.1038%2Fnature03445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loehr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahcall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of anaplastic lymphoma kinase (ALK)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4249</span>– <span class="NLM_lpage">4255</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4249-4255&author=C.+E.+Powellauthor=Y.+Gaoauthor=L.+Tanauthor=K.+A.+Donovanauthor=R.+P.+Nowakauthor=A.+Loehrauthor=M.+Bahcallauthor=E.+S.+Fischerauthor=P.+A.+Janneauthor=R.+E.+Georgeauthor=N.+S.+Gray&title=Chemically+induced+degradation+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1021%2Facs.jmedchem.7b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Powell, Chelsea E.; Gao, Yang; Tan, Li; Donovan, Katherine A.; Nowak, Radoslaw P.; Loehr, Amanda; Bahcall, Magda; Fischer, Eric S.; Janne, Pasi A.; George, Rani E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4249-4255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the development of the first small mol. degraders that can induce anaplastic lymphoma kinase (ALK) degrdn., including in non-small-cell lung cancer (NSCLC), anaplastic large-cell lymphoma (ALCL), and neuroblastoma (NB) cell lines.  These degraders were developed through conjugation of known pyrimidine-based ALK inhibitors, TAE684 or LDK378, and the cereblon ligand pomalidomide.  We demonstrate that in some cell types degrader potency is compromised by expression of drug transporter ABCB1.  In addn., proteomic profiling demonstrated that these compds. also promote the degrdn. of addnl. kinases including PTK2 (FAK), Aurora A, FER, and RPS6KA1 (RSK1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mDH7Sj8EqbVg90H21EOLACvtfcHk0lgSpRY1wTRC9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D&md5=486db11b3fbc2283700f38b09cc39bd7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01655%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DC.%2BE.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DLoehr%26aufirst%3DA.%26aulast%3DBahcall%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DGeorge%26aufirst%3DR.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4249%26epage%3D4255%26doi%3D10.1021%2Facs.jmedchem.7b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folkes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brumwell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chima, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunjan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pretswell, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dangerfield, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlton, P.</span></span> <span> </span><span class="NLM_article-title">Reversal of P-glycoprotein mediated multidrug resistance by novel anthranilamide derivatives</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">595</span>– <span class="NLM_lpage">600</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(99)00030-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1016%2FS0960-894X%2899%2900030-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10098671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADyaK1MXhvVylsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1999&pages=595-600&author=M.+Roeauthor=A.+Folkesauthor=P.+Ashworthauthor=J.+Brumwellauthor=L.+Chimaauthor=S.+Hunjanauthor=I.+Pretswellauthor=W.+Dangerfieldauthor=H.+Ryderauthor=P.+Charlton&title=Reversal+of+P-glycoprotein+mediated+multidrug+resistance+by+novel+anthranilamide+derivatives&doi=10.1016%2FS0960-894X%2899%2900030-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of P-glycoprotein mediated multidrug resistance by novel anthranilamide derivatives</span></div><div class="casAuthors">Roe, Michael; Folkes, Adrian; Ashworth, Philip; Brumwell, Julie; Chima, Lal; Hunjan, Sukhjit; Pretswell, Ian; Dangerfield, Wendy; Ryder, Hamish; Charlton, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">595-600</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">We have synthesized and evaluated a series of anthranilamide based modulators of P-glycoprotein.  These studies have identified XR9576, a potent inhibitor of P-glycoprotein in vitro and in vivo.  The general synthesis and the SAR of these compds. are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdQO2OdePDQrVg90H21EOLACvtfcHk0lhiw81Nw4Fthw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhvVylsrs%253D&md5=d1e096ea9c3a30d3574fbe9b92c657cb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2899%2900030-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252899%252900030-X%26sid%3Dliteratum%253Aachs%26aulast%3DRoe%26aufirst%3DM.%26aulast%3DFolkes%26aufirst%3DA.%26aulast%3DAshworth%26aufirst%3DP.%26aulast%3DBrumwell%26aufirst%3DJ.%26aulast%3DChima%26aufirst%3DL.%26aulast%3DHunjan%26aufirst%3DS.%26aulast%3DPretswell%26aufirst%3DI.%26aulast%3DDangerfield%26aufirst%3DW.%26aulast%3DRyder%26aufirst%3DH.%26aulast%3DCharlton%26aufirst%3DP.%26atitle%3DReversal%2520of%2520P-glycoprotein%2520mediated%2520multidrug%2520resistance%2520by%2520novel%2520anthranilamide%2520derivatives%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1999%26volume%3D9%26spage%3D595%26epage%3D600%26doi%3D10.1016%2FS0960-894X%2899%2900030-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lhiw81Nw4Fthw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faust, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstock, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Bruton tyrosine kinase degradation as a therapeutic strategy for cancer</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">952</span>– <span class="NLM_lpage">961</span>, <span class="refDoi"> DOI: 10.1182/blood-2018-07-862953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1182%2Fblood-2018-07-862953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=30545835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1MXotlGjtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2019&pages=952-961&author=D.+Dobrovolskyauthor=E.+S.+Wangauthor=S.+Morrowauthor=C.+Leahyauthor=T.+Faustauthor=R.+P.+Nowakauthor=K.+A.+Donovanauthor=G.+Yangauthor=Z.+Liauthor=E.+S.+Fischerauthor=S.+P.+Treonauthor=D.+M.+Weinstockauthor=N.+S.+Gray&title=Bruton+tyrosine+kinase+degradation+as+a+therapeutic+strategy+for+cancer&doi=10.1182%2Fblood-2018-07-862953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Bruton tyrosine kinase degradation as a therapeutic strategy for cancer</span></div><div class="casAuthors">Dobrovolsky, Dennis; Wang, Eric S.; Morrow, Sara; Leahy, Catharine; Faust, Tyler; Nowak, Radoslaw P.; Donovan, Katherine A.; Yang, Guang; Li, Zhengnian; Fischer, Eric S.; Treon, Steven P.; Weinstock, David M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">952-961</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The covalent Bruton tyrosine kinase (BTK) inhibitor ibrutinib is highly efficacious against multiple B-cell malignancies.  However, it is not selective for BTK, and multiple mechanisms of resistance, including the C481S-BTK mutation, can compromise its efficacy.  We hypothesized that small-mol.-induced BTK degrdn. may overcome some of the limitations of traditional enzymic inhibitors.  Here, we demonstrate that BTK degrdn. results in potent suppression of signaling and proliferation in cancer cells and that BTK degraders efficiently degrade C481S-BTK.  Moreover, we discovered DD-03-171, an optimized lead compd. that exhibits enhanced antiproliferative effects on mantle cell lymphoma (MCL) cells in vitro by degrading BTK, IKFZ1, and IKFZ3 as well as efficacy against patient-derived xenografts in vivo.  Thus, "triple degrdn." may be an effective therapeutic approach for treating MCL and overcoming ibrutinib resistance, thereby addressing a major unmet need in the treatment of MCL and other B-cell lymphomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN4GLppunGArVg90H21EOLACvtfcHk0li1aZMa38IH8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXotlGjtrY%253D&md5=ce7e48897378c2821d728daf6d9935d5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1182%2Fblood-2018-07-862953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2018-07-862953%26sid%3Dliteratum%253Aachs%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DMorrow%26aufirst%3DS.%26aulast%3DLeahy%26aufirst%3DC.%26aulast%3DFaust%26aufirst%3DT.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DWeinstock%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DBruton%2520tyrosine%2520kinase%2520degradation%2520as%2520a%2520therapeutic%2520strategy%2520for%2520cancer%26jtitle%3DBlood%26date%3D2019%26volume%3D133%26spage%3D952%26epage%3D961%26doi%3D10.1182%2Fblood-2018-07-862953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0li1aZMa38IH8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnathan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: Lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Clementauthor=M.+Barnathanauthor=J.+M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.+W.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicodeme&title=Discovery+of+epigenetic+regulator+I-BET762%3A+Lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0lhlbWW5T7vQrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DClement%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520Lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunker, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldie, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faty, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulrahman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantelic, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugasawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahlberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Cullin–RING ubiquitin E3 ligase regulation by the COP9 signalosome</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>531</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">603</span>, <span class="refDoi"> DOI: 10.1038/nature17416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnature17416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=27029275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlWjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=531&publication_year=2016&pages=598-603&author=S.+Cavadiniauthor=E.+S.+Fischerauthor=R.+D.+Bunkerauthor=A.+Potenzaauthor=G.+M.+Lingarajuauthor=K.+N.+Goldieauthor=W.+I.+Mohamedauthor=M.+Fatyauthor=G.+Petzoldauthor=R.+E.+J.+Beckwithauthor=R.+B.+Tichkuleauthor=U.+Hassiepenauthor=W.+Abdulrahmanauthor=R.+S.+Pantelicauthor=S.+Matsumotoauthor=K.+Sugasawaauthor=H.+Stahlbergauthor=N.+H.+Thom%C3%A4&title=Cullin%E2%80%93RING+ubiquitin+E3+ligase+regulation+by+the+COP9+signalosome&doi=10.1038%2Fnature17416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin-RING ubiquitin E3 ligase regulation by the COP9 signalosome</span></div><div class="casAuthors">Cavadini, Simone; Fischer, Eric S.; Bunker, Richard D.; Potenza, Alessandro; Lingaraju, Gondichatnahalli M.; Goldie, Kenneth N.; Mohamed, Weaam I.; Faty, Mahamadou; Petzold, Georg; Beckwith, Rohan E. J.; Tichkule, Ritesh B.; Hassiepen, Ulrich; Abdulrahman, Wassim; Pantelic, Radosav S.; Matsumoto, Syota; Sugasawa, Kaoru; Stahlberg, Henning; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">531</span>
        (<span class="NLM_cas:issue">7596</span>),
    <span class="NLM_cas:pages">598-603</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The cullin-RING ubiquitin E3 ligase (CRL) family comprises >200 members in humans.  The COP9 signalosome complex (CSN) regulates CRLs by removing their ubiquitin-like activator NEDD8.  The CUL4A-RBX1-DDB1-DDB2 complex (CRL4ADDB2) monitors the genome for UV-light-induced DNA damage.  CRL4ADBB2 is inactive in the absence of damaged DNA and requires CSN to regulate the repair process.  The structural basis of CSN binding to CRL4ADDB2 and the principles of CSN activation are poorly understood.  Here, the authors present cryo-electron microscopy structures for CSN in complex with neddylated CRL4A ligases to 6.4 Å resoln.  The CSN conformers defined by cryo-electron microscopy and a novel apo-CSN crystal structure indicated an induced-fit mechanism that drives CSN activation by neddylated CRLs.  The authors found that CSN and a substrate could not bind simultaneously to CRL4A, favoring a deneddylated, inactive state for substrate-free CRL4 complexes.  These architectural and regulatory principles appeared to be conserved across CRL families, allowing global regulation by CSN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf2nrPiPIJq7Vg90H21EOLACvtfcHk0litMhRk8ZGWzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlWjsbc%253D&md5=9f537d713513d360d605320b6d7d7df1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnature17416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17416%26sid%3Dliteratum%253Aachs%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBunker%26aufirst%3DR.%2BD.%26aulast%3DPotenza%26aufirst%3DA.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DGoldie%26aufirst%3DK.%2BN.%26aulast%3DMohamed%26aufirst%3DW.%2BI.%26aulast%3DFaty%26aufirst%3DM.%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DAbdulrahman%26aufirst%3DW.%26aulast%3DPantelic%26aufirst%3DR.%2BS.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DSugasawa%26aufirst%3DK.%26aulast%3DStahlberg%26aufirst%3DH.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DCullin%25E2%2580%2593RING%2520ubiquitin%2520E3%2520ligase%2520regulation%2520by%2520the%2520COP9%2520signalosome%26jtitle%3DNature%26date%3D2016%26volume%3D531%26spage%3D598%26epage%3D603%26doi%3D10.1038%2Fnature17416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fink, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Thalidomide promotes degradation of SALL4, a transcription factor implicated in Duane Radial Ray syndrome</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e38430</span> <span class="refDoi"> DOI: 10.7554/eLife.38430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.7554%2FeLife.38430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=30067223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyqt7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=K.+A.+Donovanauthor=J.+Anauthor=R.+P.+Nowakauthor=J.+C.+Yuanauthor=E.+C.+Finkauthor=B.+C.+Berryauthor=B.+L.+Ebertauthor=E.+S.+Fischer&title=Thalidomide+promotes+degradation+of+SALL4%2C+a+transcription+factor+implicated+in+Duane+Radial+Ray+syndrome&doi=10.7554%2FeLife.38430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Thalidomide promotes degradation of SALL4, a transcription factor implicated in Duane Radial Ray syndrome</span></div><div class="casAuthors">Donovan, Katherine A.; An, Jian; Nowak, Radoslaw P.; Yuan, Jingting C.; Fink, Emma C.; Berry, Bethany C.; Ebert, Benjamin L.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e38430/1-e38430/25</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">In historical attempts to treat morning sickness, use of the drug thalidomide led to the birth of thousands of children with severe birth defects.  Despite their teratogenicity, thalidomide and related IMiD drugs are now a mainstay of cancer treatment; however, the mol. basis underlying the pleiotropic biol. and characteristic birth defects remains unknown.  Here we show that IMiDs disrupt a broad transcriptional network through induced degrdn. of several C2H2 zinc finger transcription factors, including SALL4, a member of the spalt-like family of developmental transcription factors.  Strikingly, heterozygous loss of function mutations in SALL4 result in a human developmental condition that phenocopies thalidomide-induced birth defects such as absence of thumbs, phocomelia, defects in ear and eye development, and congenital heart disease.  We find that thalidomide induces degrdn. of SALL4 exclusively in humans, primates, and rabbits, but not in rodents or fish, providing a mechanistic link for the species-specific pathogenesis of thalidomide syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogVJ6pTS4Us7Vg90H21EOLACvtfcHk0litMhRk8ZGWzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyqt7nM&md5=a21beb45bc2dd7701a0916d1dfc331b0</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.7554%2FeLife.38430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.38430%26sid%3Dliteratum%253Aachs%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DYuan%26aufirst%3DJ.%2BC.%26aulast%3DFink%26aufirst%3DE.%2BC.%26aulast%3DBerry%26aufirst%3DB.%2BC.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DThalidomide%2520promotes%2520degradation%2520of%2520SALL4%252C%2520a%2520transcription%2520factor%2520implicated%2520in%2520Duane%2520Radial%2520Ray%2520syndrome%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2FeLife.38430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McAlister, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nusinow, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wuhr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huttlin, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gygi, S. P.</span></span> <span> </span><span class="NLM_article-title">MultiNotch MS3 enables accurate, sensitive, and multiplexed detection of differential expression across cancer cell line proteomes</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">7150</span>– <span class="NLM_lpage">7158</span>, <span class="refDoi"> DOI: 10.1021/ac502040v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac502040v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1Kmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=7150-7158&author=G.+C.+McAlisterauthor=D.+P.+Nusinowauthor=M.+P.+Jedrychowskiauthor=M.+Wuhrauthor=E.+L.+Huttlinauthor=B.+K.+Ericksonauthor=R.+Radauthor=W.+Haasauthor=S.+P.+Gygi&title=MultiNotch+MS3+enables+accurate%2C+sensitive%2C+and+multiplexed+detection+of+differential+expression+across+cancer+cell+line+proteomes&doi=10.1021%2Fac502040v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">MultiNotch MS3 Enables Accurate, Sensitive, and Multiplexed Detection of Differential Expression across Cancer Cell Line Proteomes</span></div><div class="casAuthors">McAlister, Graeme C.; Nusinow, David P.; Jedrychowski, Mark P.; Wuhr, Martin; Huttlin, Edward L.; Erickson, Brian K.; Rad, Ramin; Haas, Wilhelm; Gygi, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7150-7158</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multiplexed quantitation via isobaric chem. tags (e.g., tandem mass tags (TMT) and isobaric tags for relative and abs. quantitation (iTRAQ)) has the potential to revolutionize quant. proteomics.  However, until recently the utility of these tags was questionable due to reporter ion ratio distortion resulting from fragmentation of coisolated interfering species.  These interfering signals can be negated through addnl. gas-phase manipulations (e.g., MS/MS/MS (MS3) and proton-transfer reactions (PTR)).  These methods, however, have a significant sensitivity penalty.  Using isolation waveforms with multiple frequency notches (i.e., synchronous precursor selection, SPS), the authors coisolated and cofragmented multiple MS2 fragment ions, thereby increasing the no. of reporter ions in the MS3 spectrum 10-fold over the std. MS3 method (i.e., MultiNotch MS3).  By increasing the reporter ion signals, this method improves the dynamic range of reporter ion quantitation, reduces reporter ion signal variance, and ultimately produces more high-quality quant. measurements.  To demonstrate utility, the authors analyzed biol. triplicates of eight colon cancer cell lines using the MultiNotch MS3 method.  Across all the replicates the authors quantified 8378 proteins in union and 6168 proteins in common.  Taking into account that each of these quantified proteins contains eight distinct cell-line measurements, this data set encompasses 174,704 quant. ratios each measured in triplicate across the biol. replicates.  Herein, the MultiNotch MS3 method uniquely combines multiplexing capacity with quant. sensitivity and accuracy, drastically increasing the informational value obtainable from proteomic expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonIBGSGUtKzbVg90H21EOLACvtfcHk0lhrSD8VXVAMFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1Kmur8%253D&md5=bda847612d4a16030b7921bc8ae6f5c6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fac502040v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac502040v%26sid%3Dliteratum%253Aachs%26aulast%3DMcAlister%26aufirst%3DG.%2BC.%26aulast%3DNusinow%26aufirst%3DD.%2BP.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DWuhr%26aufirst%3DM.%26aulast%3DHuttlin%26aufirst%3DE.%2BL.%26aulast%3DErickson%26aufirst%3DB.%2BK.%26aulast%3DRad%26aufirst%3DR.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26atitle%3DMultiNotch%2520MS3%2520enables%2520accurate%252C%2520sensitive%252C%2520and%2520multiplexed%2520detection%2520of%2520differential%2520expression%2520across%2520cancer%2520cell%2520line%2520proteomes%26jtitle%3DAnal.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D7150%26epage%3D7158%26doi%3D10.1021%2Fac502040v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">R Development Core Team</span>.  <i>R: A
Language and Environment
for Statistical Computing</i>; <span class="NLM_publisher-name">R Foundation for
Statistical Computing</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=R+Development+Core+Team&title=R%3A+A%0ALanguage+and+Environment%0Afor+Statistical+Computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DR%253A%2520A%250ALanguage%2520and%2520Environment%250Afor%2520Statistical%2520Computing%26pub%3DR%2520Foundation%2520for%250AStatistical%2520Computing%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenbraun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Key, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timony, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottaviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliz, P.</span></span> <span> </span><span class="NLM_article-title">Collaboration gets the most out of software</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e01456</span> <span class="refDoi"> DOI: 10.7554/eLife.01456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.7554%2FeLife.01456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=24040512" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&author=A.+Morinauthor=B.+Eisenbraunauthor=J.+Keyauthor=P.+C.+Sanschagrinauthor=M.+A.+Timonyauthor=M.+Ottavianoauthor=P.+Sliz&title=Collaboration+gets+the+most+out+of+software&doi=10.7554%2FeLife.01456"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.7554%2FeLife.01456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.01456%26sid%3Dliteratum%253Aachs%26aulast%3DMorin%26aufirst%3DA.%26aulast%3DEisenbraun%26aufirst%3DB.%26aulast%3DKey%26aufirst%3DJ.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DTimony%26aufirst%3DM.%2BA.%26aulast%3DOttaviano%26aufirst%3DM.%26aulast%3DSliz%26aufirst%3DP.%26atitle%3DCollaboration%2520gets%2520the%2520most%2520out%2520of%2520software%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26doi%3D10.7554%2FeLife.01456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of selective degrader molecules using the bump-and-hole strategy. (A) Bumped BET bromodomain binders and the corresponding thalidomide-based degraders. (B) Structural alignment of bromodomains BRD4<sub>BD1</sub>L94V (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O3C">5O3C</a>) and BRD4<sub>BD1</sub> in complex with <b>dBET6</b> and DDB1–CRBN (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BOY">6BOY</a>). BRD4<sub>BD1</sub>L94V “hole” residue is highlighted in red. (C) Structural alignment of bromodomains BRD4<sub>BD1</sub>L94V (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O3C">5O3C</a>) and BRD4<sub>BD1</sub> in complex with <b>dBET57</b> and DDB1–CRBN (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BNB">6BNB</a>). BRD4<sub>BD1</sub>L94V “hole” residue is highlighted in red and CRBN Y355 is highlighted in cyan. Thalidomide is modeled in by aligning the CRBN C-terminal domain in the crystal structure of DDB1–CRBN–thalidomide (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CI1">4CI1</a>) to the C-terminal domain of CRBN in the dBET57 complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BNB">6BNB</a>). (D) Structural alignment of bromodomain BRD4<sub>BD1</sub>L94V (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O3C">5O3C</a>) and Rosetta docking model<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(21)</a> of BRD4<sub>BD1</sub> complex with <b>ZXH-3-26</b> and CRBN. BRD4<sub>BD1</sub>L94V “hole” residue is highlighted in red. Lenalidomide is modeled in by aligning CRBN (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FQD">5FQD</a>) to CRBN in the <b>ZXH-3-26</b> model. (E) Fluorescence polarization. Dose–response displacement of the <b>FITC–JQ1</b> probe by compounds in BRD4<sub>BD1</sub> and BRD4<sub>BD1</sub>L94V assays. Data is shown as means ± st. dev. of <i>N</i> = 3 independent replicates. (F) Fluorescence polarization. Comparison of dose–responses in (C) between BRD4<sub>BD1</sub> and BRD4<sub>BD1</sub>L94V for the two degrader molecules. Data is shown as means ± st. dev. of <i>N</i> = 3 independent replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Degradation selectivity profile. (A) Quantitative assessment of cellular degradation using the BRD4<sub>BD1</sub>–EGFP reporter assay. Cells stably expressing BRD4<sub>BD1</sub>, BRD4<sub>BD2</sub>, or BRD4<sub>BD1</sub>L94V were treated with increasing concentrations of degraders <b>XY-06-007</b> to <b>XY-06-011</b> or inhibitors <b>JQ1</b>, <b>XY-06-005</b>, <b>XY-06-006</b> for 5 h, and the EGFP-to-mCherry ratio relative to dimethyl sulfoxide (DMSO) was quantified using laser scanning cytometer (Acumen) (B) Cellular degradation selectivity between the BRD4<sub>BD1</sub>L94V mutant and BRD4<sub>BD1</sub> wild-type reporter. Data as in (A). Data for independent replicates is presented as means ± st. dev. of <i>N</i> = 3 replicates for BRD4<sub>BD1</sub>L94V and BRD4<sub>BD1</sub> and <i>N</i> = 2 for BRD4<sub>BD2</sub>. (C) Scatter plot depicting selectivity of <b>XY-06-007</b>. BRD4 levels are unaffected, and the location of BRD4 is highlighted on the volcano plot. MOLT4 cells were treated with 1 μM <b>XY-06-007</b> or DMSO control for 5 h, and protein abundance was analyzed using TMT quantification mass spectrometry. Significant changes were assessed by a moderated t-test, as implemented in the limma package,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(22)</a> with the log<sub>2</sub> fold change (log<sub>2</sub>FC) shown on the <i>y</i>-axis and log<sub>10</sub><i>p</i> values shown on the <i>x</i>-axis (one biological replicate for <b>XY-06-007</b> or three independent biological replicates for DMSO). (D) As in (C) but for <b>XY-06-011</b>. All Acumen data in this figure was analyzed using CellProfiler.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(23)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Proof-of-principle simultaneous use of dTAG and BRD4<sub>BD1</sub>L94V degron systems. (A) Schematic of the degradation system. Flp293T cells stably expressing BRD4<sub>BD1</sub>L94V–EGFP with mCherry reporter were engineered to stably express the HiBit–FKBP12 F36V–KRAS G12C fusion. The use of the EGFP/mCherry system as well as the orthogonal HiBit system allowed for sequential read of the protein abundance by Acumen followed by the HiBit lytic assay. (B) Acumen EGFP/mCherry assay. Degradation of BRD4<sub>BD1</sub>L94V–EGFP by CRBN-based degrader <b>XY-06-007</b> at a 5 h time point in the presence or absence of CRBN-based degrader <b>dTAG13</b>. (C) HiBit lytic assay. Degradation of FKBP12 F36V–KRAS G12C by <b>dTAG13</b> in the presence or absence of <b>XY-06-007</b>. The readout was performed on the same assay plate as in (C) at a 7 h time point. (D) Degradation of BRD4<sub>BD1</sub>L94V–EGFP. Dose–response of <b>dTAG</b><sup><b>V</b></sup><b>-1</b> in the presence of 1 or 0.1 μM of <b>XY-06-007</b>. (E) HiBit lytic assay. Degradation of FKBP12 F36V–KRAS G12C by <b>dTAG</b><sup><b>V</b></sup><b>-1</b> in the presence or absence of <b>XY-06-007</b>. The readout was performed on the same assay plate as in (D) at a 7 h time point. (F) Degradation of BRD4<sub>BD1</sub>L94V–EGFP by <b>XY-06-007</b> in the presence of <b>dTAG</b><sup><b>V</b></sup><b>-1</b>. (G) Degradation of BRD4<sub>BD1</sub>L94V–EGFP by <b>XY-06-007</b> in the presence of the P-gp inhibitor Tariquidar. All data in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> is presented as means ± st. dev. of <i>N</i> = 3 replicates. All Acumen data in this figure was analyzed using Cellista software (TTP Labtech).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Parental Compound <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) 2-bromo-2-methylpropane, K<sub>2</sub>CO<sub>3</sub>, MeCN, 50 °C; (b) LHMDS, MeI, THF, −78 °C; (c) TFA, DCM, rt; (d) (2-amino-5-methoxyphenyl)(4-chlorophenyl)methanone, HATU, DIPEA, DCM, 0 °C to rt; (e) TMSI, CH<sub>3</sub>CN, rt; (f) HOAc, DCE, rt; (g) P<sub>4</sub>S<sub>10</sub>, NaHCO<sub>3</sub>, DCE, 70 °C; and (h) acetohydrazide, Hg(OAc)<sub>2</sub>, THF/HOAc, 0 °C to rt; (i) NaOH, THF, rt.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/medium/jm1c00958_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Heterobifunctional Degrader <b>XY-06-011</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00958/20210719/images/large/jm1c00958_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00958&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) <i>tert</i>-butyl (2-(2-(2-aminoethoxy)ethoxy)ethyl)carbamate, HOBt, EDCI, DCM, 0 °C to rt; (b) TFA, DCM, rt; and (c) 2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetic acid <b>11</b>, HOBt, EDCI, DMF, 0 °C to rt.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i45">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24806" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24806" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 35 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: Chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+Chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0liNDS-8T0ycpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520Chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sievers, Q. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunker, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renneville, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slabicki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liddicoat, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulrahman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikkelsen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, N. H.</span></span> <span> </span><span class="NLM_article-title">Defining the human C2H2 zinc finger degrome targeted by thalidomide analogs through CRBN</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>362</i></span>, <span class="NLM_elocation-id">eaat0572</span> <span class="refDoi"> DOI: 10.1126/science.aat0572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1126%2Fscience.aat0572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=30385546" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=2018&author=Q.+L.+Sieversauthor=G.+Petzoldauthor=R.+D.+Bunkerauthor=A.+Rennevilleauthor=M.+Slabickiauthor=B.+J.+Liddicoatauthor=W.+Abdulrahmanauthor=T.+Mikkelsenauthor=B.+L.+Ebertauthor=N.+H.+Thoma&title=Defining+the+human+C2H2+zinc+finger+degrome+targeted+by+thalidomide+analogs+through+CRBN&doi=10.1126%2Fscience.aat0572"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.aat0572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aat0572%26sid%3Dliteratum%253Aachs%26aulast%3DSievers%26aufirst%3DQ.%2BL.%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DBunker%26aufirst%3DR.%2BD.%26aulast%3DRenneville%26aufirst%3DA.%26aulast%3DSlabicki%26aufirst%3DM.%26aulast%3DLiddicoat%26aufirst%3DB.%2BJ.%26aulast%3DAbdulrahman%26aufirst%3DW.%26aulast%3DMikkelsen%26aufirst%3DT.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DThoma%26aufirst%3DN.%2BH.%26atitle%3DDefining%2520the%2520human%2520C2H2%2520zinc%2520finger%2520degrome%2520targeted%2520by%2520thalidomide%2520analogs%2520through%2520CRBN%26jtitle%3DScience%26date%3D2018%26volume%3D362%26doi%3D10.1126%2Fscience.aat0572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukagawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takisawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanemaki, M.</span></span> <span> </span><span class="NLM_article-title">An auxin-based degron system for the rapid depletion of proteins in nonplant cells</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">917</span>– <span class="NLM_lpage">922</span>, <span class="refDoi"> DOI: 10.1038/nmeth.1401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnmeth.1401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=19915560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVWlsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=917-922&author=K.+Nishimuraauthor=T.+Fukagawaauthor=H.+Takisawaauthor=T.+Kakimotoauthor=M.+Kanemaki&title=An+auxin-based+degron+system+for+the+rapid+depletion+of+proteins+in+nonplant+cells&doi=10.1038%2Fnmeth.1401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">An auxin-based degron system for the rapid depletion of proteins in nonplant cells</span></div><div class="casAuthors">Nishimura, Kohei; Fukagawa, Tatsuo; Takisawa, Haruhiko; Kakimoto, Tatsuo; Kanemaki, Masato</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">917-922</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Plants have evolved a unique system in which the plant hormone auxin directly induces rapid degrdn. of the AUX/IAA family of transcription repressors by a specific form of the SCF E3 ubiquitin ligase.  Other eukaryotes lack the auxin response but share the SCF degrdn. pathway, allowing one to transplant the auxin-inducible degron (AID) system into non-plant cells and use a small mol. to conditionally control protein stability.  The AID system allowed rapid and reversible degrdn. of target proteins in response to auxin and enabled the authors to generate efficient conditional mutants of essential proteins in yeast as well as cell lines derived from chicken, mouse, hamster, monkey and human cells, thus offering a powerful tool to control protein expression and study protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzZNy-0P1i8bVg90H21EOLACvtfcHk0liLaJzZJx1hbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVWlsbzM&md5=298f94ba9750fdbfdcaf6f10c62cd408</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1401%26sid%3Dliteratum%253Aachs%26aulast%3DNishimura%26aufirst%3DK.%26aulast%3DFukagawa%26aufirst%3DT.%26aulast%3DTakisawa%26aufirst%3DH.%26aulast%3DKakimoto%26aufirst%3DT.%26aulast%3DKanemaki%26aufirst%3DM.%26atitle%3DAn%2520auxin-based%2520degron%2520system%2520for%2520the%2520rapid%2520depletion%2520of%2520proteins%2520in%2520nonplant%2520cells%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26spage%3D917%26epage%3D922%26doi%3D10.1038%2Fnmeth.1401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasanna, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salo, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vattulainen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikonen, E.</span></span> <span> </span><span class="NLM_article-title">An efficient auxin-inducible degron system with low basal degradation in human cells</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">866</span>– <span class="NLM_lpage">869</span>, <span class="refDoi"> DOI: 10.1038/s41592-019-0512-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fs41592-019-0512-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=31451765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Citb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=866-869&author=S.+Liauthor=X.+Prasannaauthor=V.+T.+Saloauthor=I.+Vattulainenauthor=E.+Ikonen&title=An+efficient+auxin-inducible+degron+system+with+low+basal+degradation+in+human+cells&doi=10.1038%2Fs41592-019-0512-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient auxin-inducible degron system with low basal degradation in human cells</span></div><div class="casAuthors">Li, Shiqian; Prasanna, Xavier; Salo, Veijo T.; Vattulainen, Ilpo; Ikonen, Elina</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">866-869</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Auxin-inducible degron technol. allows rapid and controlled protein depletion.  However, basal degrdn. without auxin and inefficient auxin-inducible depletion have limited its utility.  We have identified a potent auxin-inducible degron system composed of auxin receptor F-box protein AtAFB2 and short degron miniIAA7.  The system showed minimal basal degrdn. and enabled rapid auxin-inducible depletion of endogenous human transmembrane, cytoplasmic and nuclear proteins in 1 h with robust functional phenotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWMKDdzbT70rVg90H21EOLACvtfcHk0ljtTrbh77s33g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Citb7J&md5=1633f47ab052538a72e3d637b8e60f13</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41592-019-0512-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41592-019-0512-x%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DPrasanna%26aufirst%3DX.%26aulast%3DSalo%26aufirst%3DV.%2BT.%26aulast%3DVattulainen%26aufirst%3DI.%26aulast%3DIkonen%26aufirst%3DE.%26atitle%3DAn%2520efficient%2520auxin-inducible%2520degron%2520system%2520with%2520low%2520basal%2520degradation%2520in%2520human%2520cells%26jtitle%3DNat.%2520Methods%26date%3D2019%26volume%3D16%26spage%3D866%26epage%3D869%26doi%3D10.1038%2Fs41592-019-0512-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneekloth, J. S.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonseca, F. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koldobskiy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Chemical genetic control of protein levels: Selective in vivo targeted degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">3748</span>– <span class="NLM_lpage">3754</span>, <span class="refDoi"> DOI: 10.1021/ja039025z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja039025z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhvVegur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=3748-3754&author=J.+S.+Schneeklothauthor=F.+N.+Fonsecaauthor=M.+Koldobskiyauthor=A.+Mandalauthor=R.+Deshaiesauthor=K.+Sakamotoauthor=C.+M.+Crews&title=Chemical+genetic+control+of+protein+levels%3A+Selective+in+vivo+targeted+degradation&doi=10.1021%2Fja039025z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical genetic control of protein levels: selective in vivo targeted degradation</span></div><div class="casAuthors">Schneekloth, John S., Jr.; Fonseca, Fabiana N.; Koldobskiy, Michael; Mandal, Amit; Deshaies, Raymond; Sakamoto, Kathleen; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3748-3754</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Genetic loss of function anal. is a powerful method for the study of protein function.  However, some cell biol. questions are difficult to address using traditional genetic strategies often due to the lack of appropriate genetic model systems.  Here, we present a general strategy for the design and syntheses of mols. capable of inducing the degrdn. of selected proteins in vivo via the ubiquitin-proteasome pathway.  Western blot and fluorometric analyses indicated the loss of two different targets: green fluorescent protein (GFP) fused with FK506 binding protein (FKBP12) and GFP fused with the androgen receptor (AR), after treatment with PROteolysis TArgeting Chimeric mols. (PROTACS) incorporating a FKBP12 ligand and dihydrotestosterone, resp.  These are the first in vivo examples of direct small mol.-induced recruitment of target proteins to the proteasome for degrdn. upon addn. to cultured cells.  Moreover, PROTAC-mediated protein degrdn. offers a general strategy to create "chem. knockouts," thus opening new possibilities for the control of protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvHYtPWyuke7Vg90H21EOLACvtfcHk0ljtTrbh77s33g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhvVegur0%253D&md5=f0855157e4daf32080a23e1728c4b9cc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fja039025z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja039025z%26sid%3Dliteratum%253Aachs%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26aulast%3DFonseca%26aufirst%3DF.%2BN.%26aulast%3DKoldobskiy%26aufirst%3DM.%26aulast%3DMandal%26aufirst%3DA.%26aulast%3DDeshaies%26aufirst%3DR.%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DChemical%2520genetic%2520control%2520of%2520protein%2520levels%253A%2520Selective%2520in%2520vivo%2520targeted%2520degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D3748%26epage%3D3754%26doi%3D10.1021%2Fja039025z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darricarrere, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">HaloPROTACS: Use of small molecule PROTACs to induce degradation of HaloTag fusion proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1831</span>– <span class="NLM_lpage">1837</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00442</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00442" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1831-1837&author=D.+L.+Buckleyauthor=K.+Rainaauthor=N.+Darricarrereauthor=J.+Hinesauthor=J.+L.+Gustafsonauthor=I.+E.+Smithauthor=A.+H.+Miahauthor=J.+D.+Harlingauthor=C.+M.+Crews&title=HaloPROTACS%3A+Use+of+small+molecule+PROTACs+to+induce+degradation+of+HaloTag+fusion+proteins&doi=10.1021%2Facschembio.5b00442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">HaloPROTACS: Use of Small Molecule PROTACs to Induce Degradation of HaloTag Fusion Proteins</span></div><div class="casAuthors">Buckley, Dennis L.; Raina, Kanak; Darricarrere, Nicole; Hines, John; Gustafson, Jeffrey L.; Smith, Ian E.; Miah, Afjal H.; Harling, John D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1831-1837</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol.-induced protein degrdn. is an attractive strategy for the development of chem. probes.  One method for inducing targeted protein degrdn. involves the use of PROTACs, heterobifunctional mols. that can recruit specific E3 ligases to a desired protein of interest.  PROTACs have been successfully used to degrade numerous proteins in cells, but the peptidic E3 ligase ligands used in previous PROTACs have hindered their development into more mature chem. probes or therapeutics.  The authors report the design of a novel class of PROTACs that incorporate small mol. VHL ligands to successfully degrade HaloTag7 fusion proteins.  These HaloPROTACs will inspire the development of future PROTACs with more drug-like properties.  Addnl., these HaloPROTACs are useful chem. genetic tools, due to their ability to chem. knock down widely used HaloTag7 fusion proteins in a general fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotj-Sgj4TyKrVg90H21EOLACvtfcHk0lh4diUAvhhmaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN&md5=b074c58e983f15c384818777b25e248d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00442%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDarricarrere%26aufirst%3DN.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHaloPROTACS%253A%2520Use%2520of%2520small%2520molecule%2520PROTACs%2520to%2520induce%2520degradation%2520of%2520HaloTag%2520fusion%2520proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1831%26epage%3D1837%26doi%3D10.1021%2Facschembio.5b00442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomoshige, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, M.</span></span> <span> </span><span class="NLM_article-title">Degradation of HaloTag-fused nuclear proteins using bestatin-HaloTag ligand hybrid molecules</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">9746</span>– <span class="NLM_lpage">9750</span>, <span class="refDoi"> DOI: 10.1039/C5OB01395J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1039%2FC5OB01395J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=26338696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOrtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=9746-9750&author=S.+Tomoshigeauthor=M.+Naitoauthor=Y.+Hashimotoauthor=M.+Ishikawa&title=Degradation+of+HaloTag-fused+nuclear+proteins+using+bestatin-HaloTag+ligand+hybrid+molecules&doi=10.1039%2FC5OB01395J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of HaloTag-fused nuclear proteins using bestatin-HaloTag ligand hybrid molecules</span></div><div class="casAuthors">Tomoshige, Shusuke; Naito, Mikihiko; Hashimoto, Yuichi; Ishikawa, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">9746-9750</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors have developed a protein knockdown technol. using hybrid small mols. designed as conjugates of a ligand for the target protein and a ligand for ubiquitin ligase cellular inhibitor of apoptosis protein 1 (cIAP1).  However, this technol. has several limitations.  Here, the authors report the development of a novel protein knockdown system to address these limitations.  In this system, target proteins are fused with HaloTag to provide a common binding site for a degrdn. inducer.  The authors designed and synthesized small mols. consisting of alkyl chloride as the HaloTag-binding degrdn. inducer, which binds to HaloTag, linked to BE04(2), which binds to cIAP1.  Using this system, the authors successfully knocked down HaloTag-fused cAMP responsive element binding protein 1 (HaloTag-CREB1) and HaloTag-fused c-jun (HaloTag-c-jun), which are ligand-unknown nuclear proteins, in living cells.  HaloTag-binding degrdn. inducers can be synthesized easily,and are expected to be useful as biol. tools for pan-degrdn. of HaloTag-fused proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHj9MV1-n3NrVg90H21EOLACvtfcHk0lh4diUAvhhmaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOrtb3I&md5=9c546a5949aa33937a7fef2bac8bde1a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1039%2FC5OB01395J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB01395J%26sid%3Dliteratum%253Aachs%26aulast%3DTomoshige%26aufirst%3DS.%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DHashimoto%26aufirst%3DY.%26aulast%3DIshikawa%26aufirst%3DM.%26atitle%3DDegradation%2520of%2520HaloTag-fused%2520nuclear%2520proteins%2520using%2520bestatin-HaloTag%2520ligand%2520hybrid%2520molecules%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D9746%26epage%3D9750%26doi%3D10.1039%2FC5OB01395J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tovell, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prescott, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macartney, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span> <span> </span><span class="NLM_article-title">Rapid and reversible knockdown of endogenously tagged endosomal proteins via an optimized HaloPROTAC degrader</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">882</span>– <span class="NLM_lpage">892</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b01016</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b01016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnt1emsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=882-892&author=H.+Tovellauthor=A.+Testaauthor=C.+Maniaciauthor=H.+Zhouauthor=A.+R.+Prescottauthor=T.+Macartneyauthor=A.+Ciulliauthor=D.+R.+Alessi&title=Rapid+and+reversible+knockdown+of+endogenously+tagged+endosomal+proteins+via+an+optimized+HaloPROTAC+degrader&doi=10.1021%2Facschembio.8b01016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid and Reversible Knockdown of Endogenously Tagged Endosomal Proteins via an Optimized HaloPROTAC Degrader</span></div><div class="casAuthors">Tovell, Hannah; Testa, Andrea; Maniaci, Chiara; Zhou, Houjiang; Prescott, Alan R.; Macartney, Thomas; Ciulli, Alessio; Alessi, Dario R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">882-892</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inducing posttranslational protein knockdown is an important approach to probe biol. and validate drug targets.  An efficient strategy to achieve this involves expression of a protein of interest fused to an exogenous tag, allowing tag-directed chem. degraders to mediate protein ubiquitylation and proteasomal degrdn.  Here, the authors combine improved HaloPROTAC degrader probes with CRISPR/Cas9 genome editing technol. to trigger rapid degrdn. of endogenous target proteins.  The optimized probe, HaloPROTAC-E, a chloroalkane conjugate of high-affinity VHL binder VH298, induced reversible degrdn. of two endosomally localized proteins, SGK3 and VPS34, with a DC50 of 3-10 nM.  HaloPROTAC-E induced rapid (∼50% degrdn. after 30 min) and complete (Dmax of ∼95% at 48 h) depletion of Halo-tagged SGK3, blocking downstream phosphorylation of the SGK3 substrate NDRG1.  HaloPROTAC-E more potently induced greater steady state degrdn. of Halo tagged endogenous VPS34 than the previously reported HaloPROTAC3 compd.  Quant. global proteomics revealed that HaloPROTAC-E is remarkably selective inducing only degrdn. of the Halo tagged endogenous VPS34 complex (VPS34, VPS15, Beclin1, and ATG14) and no other proteins were significantly degraded.  This study exemplifies the combination of HaloPROTACs with CRISPR/Cas9 endogenous protein tagging as a useful method to induce rapid and reversible degrdn. of endogenous proteins to interrogate their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpJ5B8is89J7Vg90H21EOLACvtfcHk0ljVQBxZfIpDrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnt1emsrg%253D&md5=ea08079224e09d19c1e9f4ee8147909c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b01016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b01016%26sid%3Dliteratum%253Aachs%26aulast%3DTovell%26aufirst%3DH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DPrescott%26aufirst%3DA.%2BR.%26aulast%3DMacartney%26aufirst%3DT.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DRapid%2520and%2520reversible%2520knockdown%2520of%2520endogenously%2520tagged%2520endosomal%2520proteins%2520via%2520an%2520optimized%2520HaloPROTAC%2520degrader%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D882%26epage%3D892%26doi%3D10.1021%2Facschembio.8b01016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon-Clarke, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation as a powerful research tool in basic biology and drug target discovery</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1038/s41594-020-0438-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fs41594-020-0438-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=32541897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFKgtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=605-614&author=T.+Wuauthor=H.+Yoonauthor=Y.+Xiongauthor=S.+E.+Dixon-Clarkeauthor=R.+P.+Nowakauthor=E.+S.+Fischer&title=Targeted+protein+degradation+as+a+powerful+research+tool+in+basic+biology+and+drug+target+discovery&doi=10.1038%2Fs41594-020-0438-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation as a powerful research tool in basic biology and drug target discovery</span></div><div class="casAuthors">Wu, Tao; Yoon, Hojong; Xiong, Yuan; Dixon-Clarke, Sarah E.; Nowak, Radoslaw P.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">605-614</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Controlled perturbation of protein activity is essential to study protein function in cells and living organisms.  Small mols. that hijack the cellular protein ubiquitination machinery to selectively degrade proteins of interest, so-called degraders, have recently emerged as alternatives to selective chem. inhibitors, both as therapeutic modalities and as powerful research tools.  These systems offer unprecedented temporal and spatial control over protein function.  Here, we review recent developments in this field, with a particular focus on the use of degraders as research tools to interrogate complex biol. problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_qHFQVv3fV7Vg90H21EOLACvtfcHk0ljVQBxZfIpDrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFKgtrzI&md5=b9221cbb6d0146234f24ead09d569736</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fs41594-020-0438-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-020-0438-0%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DT.%26aulast%3DYoon%26aufirst%3DH.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DDixon-Clarke%26aufirst%3DS.%2BE.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DTargeted%2520protein%2520degradation%2520as%2520a%2520powerful%2520research%2520tool%2520in%2520basic%2520biology%2520and%2520drug%2520target%2520discovery%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D605%26epage%3D614%26doi%3D10.1038%2Fs41594-020-0438-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">The dTAG system for immediate and target-specific protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0021-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fs41589-018-0021-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=29581585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=431-441&author=B.+Nabetauthor=J.+M.+Robertsauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=S.+Dastjerdiauthor=A.+Yangauthor=A.+L.+Leggettauthor=M.+A.+Erbauthor=M.+A.+Lawlorauthor=A.+Souzaauthor=T.+G.+Scottauthor=S.+Vittoriauthor=J.+A.+Perryauthor=J.+Qiauthor=G.+E.+Winterauthor=K.+K.+Wongauthor=N.+S.+Grayauthor=J.+E.+Bradner&title=The+dTAG+system+for+immediate+and+target-specific+protein+degradation&doi=10.1038%2Fs41589-018-0021-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The dTAG system for immediate and target-specific protein degradation</span></div><div class="casAuthors">Nabet, Behnam; Roberts, Justin M.; Buckley, Dennis L.; Paulk, Joshiawa; Dastjerdi, Shiva; Yang, Annan; Leggett, Alan L.; Erb, Michael A.; Lawlor, Matthew A.; Souza, Amanda; Scott, Thomas G.; Vittori, Sarah; Perry, Jennifer A.; Qi, Jun; Winter, Georg E.; Wong, Kwok-Kin; Gray, Nathanael S.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">431-441</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Dissection of complex biol. systems requires target-specific control of the function or abundance of proteins.  Genetic perturbations are limited by off-target effects, multicomponent complexity, and irreversibility.  Most limiting is the requisite delay between modulation to exptl. measurement.  To enable the immediate and selective control of single protein abundance, we created a chem. biol. system that leverages the potency of cell-permeable heterobifunctional degraders.  The dTAG system pairs a novel degrader of FKBP12F36V with expression of FKBP12F36V in-frame with a protein of interest.  By transgene expression or CRISPR-mediated locus-specific knock-in, we exemplify a generalizable strategy to study the immediate consequence of protein loss.  Using dTAG, we observe an unexpected superior antiproliferative effect of pan-BET bromodomain degrdn. over selective BRD4 degrdn., characterize immediate effects of KRASG12V loss on proteomic signaling, and demonstrate rapid degrdn. in vivo.  This technol. platform will confer kinetic resoln. to biol. investigation and provide target validation in the context of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqwQvCDe-2I7Vg90H21EOLACvtfcHk0liVyxRwgXedug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D&md5=d7d9c111c68c84bc3aa9c2aad3f591a6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0021-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0021-8%26sid%3Dliteratum%253Aachs%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DA.%26aulast%3DLeggett%26aufirst%3DA.%2BL.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLawlor%26aufirst%3DM.%2BA.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520dTAG%2520system%2520for%2520immediate%2520and%2520target-specific%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D431%26epage%3D441%26doi%3D10.1038%2Fs41589-018-0021-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seong, B. K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuljanin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohardt, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robichaud, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancias, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Rapid and direct control of target protein levels with VHL-recruiting dTAG molecules</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">4687</span> <span class="refDoi"> DOI: 10.1038/s41467-020-18377-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fs41467-020-18377-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=32948771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFSisrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&author=B.+Nabetauthor=F.+M.+Fergusonauthor=B.+K.+A.+Seongauthor=M.+Kuljaninauthor=A.+L.+Leggettauthor=M.+L.+Mohardtauthor=A.+Robichaudauthor=A.+S.+Conwayauthor=D.+L.+Buckleyauthor=J.+D.+Manciasauthor=J.+E.+Bradnerauthor=K.+Stegmaierauthor=N.+S.+Gray&title=Rapid+and+direct+control+of+target+protein+levels+with+VHL-recruiting+dTAG+molecules&doi=10.1038%2Fs41467-020-18377-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid and direct control of target protein levels with VHL-recruiting dTAG molecules</span></div><div class="casAuthors">Nabet, Behnam; Ferguson, Fleur M.; Seong, Bo Kyung A.; Kuljanin, Miljan; Leggett, Alan L.; Mohardt, Mikaela L.; Robichaud, Amanda; Conway, Amy S.; Buckley, Dennis L.; Mancias, Joseph D.; Bradner, James E.; Stegmaier, Kimberly; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4687</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Chem. biol. strategies for directly perturbing protein homeostasis including the degrdn. tag (dTAG) system provide temporal advantages over genetic approaches and improved selectivity over small mol. inhibitors.  We describe dTAGV-1, an exclusively selective VHL-recruiting dTAG mol., to rapidly degrade FKBP12F36V-tagged proteins. dTAGV-1 overcomes a limitation of previously reported CRBN-recruiting dTAG mols. to degrade recalcitrant oncogenes, supports combination degrader studies and facilitates investigations of protein function in cells and mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpguq1r4d5UyrVg90H21EOLACvtfcHk0liVyxRwgXedug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFSisrnO&md5=aea2209d730ec0379eb966a27817fdb5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18377-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18377-w%26sid%3Dliteratum%253Aachs%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DSeong%26aufirst%3DB.%2BK.%2BA.%26aulast%3DKuljanin%26aufirst%3DM.%26aulast%3DLeggett%26aufirst%3DA.%2BL.%26aulast%3DMohardt%26aufirst%3DM.%2BL.%26aulast%3DRobichaud%26aufirst%3DA.%26aulast%3DConway%26aufirst%3DA.%2BS.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DMancias%26aufirst%3DJ.%2BD.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DRapid%2520and%2520direct%2520control%2520of%2520target%2520protein%2520levels%2520with%2520VHL-recruiting%2520dTAG%2520molecules%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26doi%3D10.1038%2Fs41467-020-18377-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozamus, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amara, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rollins, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magari, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courage, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerasoli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, D. A.</span></span> <span> </span><span class="NLM_article-title">Redesigning an FKBP–ligand interface to generate chemical dimerizers with novel specificity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">10437</span>– <span class="NLM_lpage">10442</span>, <span class="refDoi"> DOI: 10.1073/pnas.95.18.10437</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1073%2Fpnas.95.18.10437" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=9724721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADyaK1cXlvFWgsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=10437-10442&author=T.+Clacksonauthor=W.+Yangauthor=L.+W.+Rozamusauthor=M.+Hatadaauthor=J.+F.+Amaraauthor=C.+T.+Rollinsauthor=L.+F.+Stevensonauthor=S.+R.+Magariauthor=S.+A.+Woodauthor=N.+L.+Courageauthor=X.+Luauthor=F.+Cerasoliauthor=M.+Gilmanauthor=D.+A.+Holt&title=Redesigning+an+FKBP%E2%80%93ligand+interface+to+generate+chemical+dimerizers+with+novel+specificity&doi=10.1073%2Fpnas.95.18.10437"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Redesigning an FKBP-ligand interface to generate chemical dimerizers with novel specificity</span></div><div class="casAuthors">Clackson, Tim; Yang, Wu; Rozamus, Leonard W.; Hatada, Marcos; Amara, Jane F.; Rollins, Carl T.; Stevenson, Lauren F.; Magari, Shannon R.; Wood, Susan A.; Courage, Nancy L.; Lu, Xiaode; Cerasoli, Franklin, Jr.; Gilman, Michael; Holt, Dennis A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10437-10442</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">FKBP ligand homodimers can be used to activate signaling events inside cells and animals that have been engineered to express fusions between appropriate signaling domains and FKBP.  However, use of these dimerizers in vivo is potentially limited by ligand binding to endogenous FKBP.  We have designed ligands that bind specifically to a mutated FKBP over the wild-type protein by remodeling an FKBP-ligand interface to introduce a specificity binding pocket.  A compd. bearing an Et substituent in place of a carbonyl group exhibited sub-nanomolar affinity and 1000-fold selectivity for a mutant FKBP with a compensating truncation of a phenylalanine residue.  Structural and functional anal. of the new pocket showed that recognition is surprisingly relaxed, with the modified ligand only partially filling the engineered cavity.  We incorporated the specificity pocket into a fusion protein contg. FKBP and the intracellular domain of the Fas receptor.  Cells expressing this modified chimeric protein potently underwent apoptosis in response to AP1903, a homodimer of the modified ligand, both in culture and when implanted into mice.  Remodeled dimerizers such as AP1903 are ideal reagents for controlling the activities of cells that have been modified by gene therapy procedures, without interference from endogenous FKBP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBY_8A9vo2YbVg90H21EOLACvtfcHk0lhp9gXymfWiAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlvFWgsb4%253D&md5=20fde72ec4c39f5dadf11b32f416b341</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.18.10437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.18.10437%26sid%3Dliteratum%253Aachs%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DRozamus%26aufirst%3DL.%2BW.%26aulast%3DHatada%26aufirst%3DM.%26aulast%3DAmara%26aufirst%3DJ.%2BF.%26aulast%3DRollins%26aufirst%3DC.%2BT.%26aulast%3DStevenson%26aufirst%3DL.%2BF.%26aulast%3DMagari%26aufirst%3DS.%2BR.%26aulast%3DWood%26aufirst%3DS.%2BA.%26aulast%3DCourage%26aufirst%3DN.%2BL.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DCerasoli%26aufirst%3DF.%26aulast%3DGilman%26aufirst%3DM.%26aulast%3DHolt%26aufirst%3DD.%2BA.%26atitle%3DRedesigning%2520an%2520FKBP%25E2%2580%2593ligand%2520interface%2520to%2520generate%2520chemical%2520dimerizers%2520with%2520novel%2520specificity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1998%26volume%3D95%26spage%3D10437%26epage%3D10442%26doi%3D10.1073%2Fpnas.95.18.10437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanjana, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalem, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offei-Addo, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Transcription control by the ENL YEATS domain in acute leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>543</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">274</span>, <span class="refDoi"> DOI: 10.1038/nature21688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnature21688" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=28241139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2sXks1Gls70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=543&publication_year=2017&pages=270-274&author=M.+A.+Erbauthor=T.+G.+Scottauthor=B.+E.+Liauthor=H.+Xieauthor=J.+Paulkauthor=H.+S.+Seoauthor=A.+Souzaauthor=J.+M.+Robertsauthor=S.+Dastjerdiauthor=D.+L.+Buckleyauthor=N.+E.+Sanjanaauthor=O.+Shalemauthor=B.+Nabetauthor=R.+Zeidauthor=N.+K.+Offei-Addoauthor=S.+Dhe-Paganonauthor=F.+Zhangauthor=S.+H.+Orkinauthor=G.+E.+Winterauthor=J.+E.+Bradner&title=Transcription+control+by+the+ENL+YEATS+domain+in+acute+leukaemia&doi=10.1038%2Fnature21688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription control by the ENL YEATS domain in acute leukemia</span></div><div class="casAuthors">Erb, Michael A.; Scott, Thomas G.; Li, Bin E.; Xie, Huafeng; Paulk, Joshiawa; Seo, Hyuk-Soo; Souza, Amanda; Roberts, Justin M.; Dastjerdi, Shiva; Buckley, Dennis L.; Sanjana, Neville E.; Shalem, Ophir; Nabet, Behnam; Zeid, Rhamy; Offei-Addo, Nana K.; Dhe-Paganon, Sirano; Zhang, Feng; Orkin, Stuart H.; Winter, Georg E.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">543</span>
        (<span class="NLM_cas:issue">7644</span>),
    <span class="NLM_cas:pages">270-274</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations producing a chimeric MLL oncogene give rise to a highly aggressive acute leukemia assocd. with poor clin. outcome.  The preferential involvement of chromatin-assocd. factors as MLL fusion partners belies a dependency on transcription control.  Despite recent progress made in targeting chromatin regulators in cancer, available therapies for this well-characterized disease remain inadequate, prompting the need to identify new targets for therapeutic intervention.  Here, using unbiased CRISPR-Cas9 technol. to perform a genome-scale loss-of-function screen in an MLL-AF4-pos. acute leukemia cell line, we identify ENL as an unrecognized gene that is specifically required for proliferation in vitro and in vivo.  To explain the mechanistic role of ENL in leukemia pathogenesis and dynamic transcription control, a chem. genetic strategy was developed to achieve targeted protein degrdn.  Acute loss of ENL suppressed the initiation and elongation of RNA polymerase II at active genes genome-wide, with pronounced effects at genes featuring a disproportionate ENL load.  Notably, an intact YEATS chromatin-reader domain was essential for ENL-dependent leukemic growth.  Overall, these findings identify a dependency factor in acute leukemia and suggest a mechanistic rationale for disrupting the YEATS domain in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8JlG7P9bHArVg90H21EOLACvtfcHk0lhp9gXymfWiAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXks1Gls70%253D&md5=5b889ab323bcf91f55b6c0db02878d04</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature21688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature21688%26sid%3Dliteratum%253Aachs%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DLi%26aufirst%3DB.%2BE.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DSeo%26aufirst%3DH.%2BS.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DSanjana%26aufirst%3DN.%2BE.%26aulast%3DShalem%26aufirst%3DO.%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DZeid%26aufirst%3DR.%26aulast%3DOffei-Addo%26aufirst%3DN.%2BK.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DTranscription%2520control%2520by%2520the%2520ENL%2520YEATS%2520domain%2520in%2520acute%2520leukaemia%26jtitle%3DNature%26date%3D2017%26volume%3D543%26spage%3D270%26epage%3D274%26doi%3D10.1038%2Fnature21688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLauchlan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">MELK is not necessary for the proliferation of basal-like breast cancer cells</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e26693</span> <span class="refDoi"> DOI: 10.7554/eLife.26693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.7554%2FeLife.26693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=28926338" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlOit7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&author=H.+T.+Huangauthor=H.+S.+Seoauthor=T.+Zhangauthor=Y.+Wangauthor=B.+Jiangauthor=Q.+Liauthor=D.+L.+Buckleyauthor=B.+Nabetauthor=J.+M.+Robertsauthor=J.+Paulkauthor=S.+Dastjerdiauthor=G.+E.+Winterauthor=H.+McLauchlanauthor=J.+Moranauthor=J.+E.+Bradnerauthor=M.+J.+Eckauthor=S.+Dhe-Paganonauthor=J.+J.+Zhaoauthor=N.+S.+Gray&title=MELK+is+not+necessary+for+the+proliferation+of+basal-like+breast+cancer+cells&doi=10.7554%2FeLife.26693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MELK is not necessary for the proliferation of basal-like breast cancer cells</span></div><div class="casAuthors">Huang, Hai-Tsang; Seo, Hyuk-Soo; Zhang, Tinghu; Wang, Yubao; Jiang, Baishan; Li, Qing; Buckley, Dennis L.; Nabet, Behnam; Roberts, Justin M.; Paulk, Joshiawa; Dastjerdi, Shiva; Winter, Georg E.; McLauchlan, Hilary; Moran, Jennifer; Bradner, James E.; Eck, Michael J.; Dhe-Paganon, Sirano; Zhao, Jean J.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e26693/1-e26693/29</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Thorough preclin. target validation is essential for the success of drug discovery efforts.  In this study, we combined chem. and genetic perturbants, including the development of a novel selective maternal embryonic leucine zipper kinase (MELK) inhibitor HTH-01-091, CRISPR/ Cas9-mediated MELK knockout, a novel chem.-induced protein degrdn. strategy, RNA interference and CRISPR interference to validate MELK as a therapeutic target in basal-like breast cancers (BBC).  In common culture conditions, we found that small mol. inhibition, genetic deletion, or acute depletion of MELK did not significantly affect cellular growth.  This discrepancy to previous findings illuminated selectivity issues of the widely used MELK inhibitor OTSSP167, and potential off-target effects of MELK-targeting short hairpins.  The different genetic and chem. tools developed here allow for the identification and validation of any causal roles MELK may play in cancer biol., which will be required to guide future MELK drug discovery efforts.  Furthermore, our study provides a general framework for preclin. target validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW6AnVwhH8j7Vg90H21EOLACvtfcHk0lggM5s5l2Yf2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlOit7jE&md5=f5abb6df4e61df2b0cc24beb3005b176</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.7554%2FeLife.26693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.26693%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%2BT.%26aulast%3DSeo%26aufirst%3DH.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DMcLauchlan%26aufirst%3DH.%26aulast%3DMoran%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DMELK%2520is%2520not%2520necessary%2520for%2520the%2520proliferation%2520of%2520basal-like%2520breast%2520cancer%2520cells%26jtitle%3DeLife%26date%3D2017%26volume%3D6%26doi%3D10.7554%2FeLife.26693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buzko, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Magic bullets for protein kinases</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">172</span>, <span class="refDoi"> DOI: 10.1016/S0962-8924(01)01928-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1016%2FS0962-8924%2801%2901928-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=11306297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD3MXisF2msrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=167-172&author=A.+C.+Bishopauthor=O.+Buzkoauthor=K.+M.+Shokat&title=Magic+bullets+for+protein+kinases&doi=10.1016%2FS0962-8924%2801%2901928-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Magic bullets for protein kinases</span></div><div class="casAuthors">Bishop, A. C.; Buzko, O.; Shokat, K. M.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">167-172</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review and discussion with 52 refs.  A chem.-genetic method for the generation of target-specific protein kinase inhibitors has been developed recently.  This strategy utilizes a functionally silent active-site mutation to sensitize a target kinase to inhibition by a small mol. that does not inhibit wild-type kinases.  Tyrosine and serine/threonine kinases are equally amenable to the drug-sensitization approach, which has been used to generate selective inhibitors of mutant Src-family kinases, Abl-family kinases, cyclin-dependent kinases, mitogen-activated kinases, p21-activated kinases, and Ca2+/calmodulin-dependent kinases.  The designed inhibitors are specific for the sensitized kinase in a cellular background where the wild-type kinase has been inactivated.  By these means, kinase-sensitization has been used systematically to generate and analyze conditional alleles of several yeast protein kinases in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN2pRnErAok7Vg90H21EOLACvtfcHk0lggM5s5l2Yf2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXisF2msrY%253D&md5=e4eea34d0dcea9c1470b7bb90dbd3f37</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS0962-8924%2801%2901928-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0962-8924%252801%252901928-6%26sid%3Dliteratum%253Aachs%26aulast%3DBishop%26aufirst%3DA.%2BC.%26aulast%3DBuzko%26aufirst%3DO.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DMagic%2520bullets%2520for%2520protein%2520kinases%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2001%26volume%3D11%26spage%3D167%26epage%3D172%26doi%3D10.1016%2FS0962-8924%2801%2901928-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Islam, K.</span></span> <span> </span><span class="NLM_article-title">The bump-and-hole tactic: Expanding the scope of chemical genetics</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1171</span>– <span class="NLM_lpage">1184</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1016%2Fj.chembiol.2018.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=30078633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVersL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1171-1184&author=K.+Islam&title=The+bump-and-hole+tactic%3A+Expanding+the+scope+of+chemical+genetics&doi=10.1016%2Fj.chembiol.2018.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The Bump-and-Hole Tactic: Expanding the Scope of Chemical Genetics</span></div><div class="casAuthors">Islam, Kabirul</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1171-1184</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Successful mapping of the human genome has sparked a widespread interest in deciphering functional information encoded in gene sequences.  However, because of the high degree of conservation in sequences along with topol. and biochem. similarities among members of a protein superfamily, uncovering physiol. role of a particular protein has been a challenging task.  Chem. genetic approaches have made significant contributions toward understanding protein function.  One such effort, dubbed the bump-and-hole approach, has convincingly demonstrated that engineering at the protein-small mol. interface constitutes a powerful method for elucidating the function of a specific gene product.  By manipulating the steric component of protein-ligand interactions in a complementary manner, an orthogonal system is developed to probe a specific enzyme-cofactor pair without interference from related members.  This article outlines current efforts to expand the approach for diverse protein classes and their applications.  Potential future innovations to address contemporary biol. problems are highlighted as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZJbGz0HI7EbVg90H21EOLACvtfcHk0ljNQVgDM02eTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVersL3P&md5=e2224810004276552f3564040f9957f6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DIslam%26aufirst%3DK.%26atitle%3DThe%2520bump-and-hole%2520tactic%253A%2520Expanding%2520the%2520scope%2520of%2520chemical%2520genetics%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D1171%26epage%3D1184%26doi%3D10.1016%2Fj.chembiol.2018.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">New synthetic routes to triazolo-benzodiazepine analogues: Expanding the scope of the bump-and-hole approach for selective bromo and extra-terminal (BET) bromodomain inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1492</span>– <span class="NLM_lpage">1500</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01135</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01135" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1492-1500&author=M.+G.+Baudauthor=E.+Lin-Shiaoauthor=M.+Zengerleauthor=C.+Tallantauthor=A.+Ciulli&title=New+synthetic+routes+to+triazolo-benzodiazepine+analogues%3A+Expanding+the+scope+of+the+bump-and-hole+approach+for+selective+bromo+and+extra-terminal+%28BET%29+bromodomain+inhibition&doi=10.1021%2Facs.jmedchem.5b01135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Zengerle, Michael; Tallant, Cynthia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1492-1500</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe new synthetic routes developed toward a range of substituted analogs of bromo and extra-terminal (BET) bromodomain inhibitors I-BET762/JQ1 based on the triazolo-benzodiazepine scaffold.  These new routes allow for the derivatization of the methoxyphenyl and chlorophenyl rings, in addn. to the diazepine ternary center and the side chain methylene moiety.  Substitution at the level of the side chain methylene afforded compds. targeting specifically and potently engineered BET bromodomains designed as part of a bump and hole approach.  We further demonstrate that marked selectivity for the second over the first bromodomain can be achieved with an indole deriv. that exploits differential interaction with an aspartate/histidine conservative substitution on the BC loop of BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUpYVun8F94bVg90H21EOLACvtfcHk0ljNQVgDM02eTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE&md5=65dd39e3f7e0bc52169ffb1466218c43</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01135%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DNew%2520synthetic%2520routes%2520to%2520triazolo-benzodiazepine%2520analogues%253A%2520Expanding%2520the%2520scope%2520of%2520the%2520bump-and-hole%2520approach%2520for%2520selective%2520bromo%2520and%2520extra-terminal%2520%2528BET%2529%2520bromodomain%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1492%26epage%3D1500%26doi%3D10.1021%2Facs.jmedchem.5b01135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardote, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pschibul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, T. T.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>346</i></span>,  <span class="NLM_fpage">638</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1126/science.1249830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1126%2Fscience.1249830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=25323695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=638-641&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=T.+Cardoteauthor=C.+Tallantauthor=A.+Pschibulauthor=K.-H.+Chanauthor=M.+Zengerleauthor=J.+R.+Garciaauthor=T.+T.-L.+Kwanauthor=F.+M.+Fergusonauthor=A.+Ciulli&title=A+bump-and-hole+approach+to+engineer+controlled+selectivity+of+BET+bromodomain+chemical+probes&doi=10.1126%2Fscience.1249830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Cardote, Teresa; Tallant, Cynthia; Pschibul, Annica; Chan, Kwok-Ho; Zengerle, Michael; Garcia, Jordi R.; Kwan, Terence T.-L.; Ferguson, Fleur M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6209</span>),
    <span class="NLM_cas:pages">638-641</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Small mols. are useful tools for probing the biol. function and therapeutic potential of individual proteins, but achieving selectivity is challenging when the target protein shares structural domains with other proteins.  The bromodomain and extra-terminal (BET) proteins have attracted interest because of their roles in transcriptional regulation, epigenetics, and cancer.  The BET bromodomains (protein interaction modules that bind acetyl-lysine) have been targeted by potent small-mol. inhibitors, but these inhibitors lack selectivity for individual family members.  We developed an Et deriv. of an existing small-mol. inhibitor, I-BET/JQ1, and showed that it binds leucine/alanine mutant bromodomains with nanomolar affinity and achieves up to 540-fold selectivity relative to wild-type bromodomains.  Cell culture studies showed that blockade of the first bromodomain alone is sufficient to displace a specific BET protein, Brd4, from chromatin.  Expansion of this approach could help identify the individual roles of single BET proteins in human physiol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9EnzLtvUKwbVg90H21EOLACvtfcHk0ljU_dsCofFTKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL&md5=04fe967e4681fb4b2b3168f4d5f8ff81</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fscience.1249830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1249830%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DCardote%26aufirst%3DT.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DPschibul%26aufirst%3DA.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DJ.%2BR.%26aulast%3DKwan%26aufirst%3DT.%2BT.-L.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DA%2520bump-and-hole%2520approach%2520to%2520engineer%2520controlled%2520selectivity%2520of%2520BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DScience%26date%3D2014%26volume%3D346%26spage%3D638%26epage%3D641%26doi%3D10.1126%2Fscience.1249830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Runcie, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Beurden, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L. C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulthard, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Optimization of a “bump-and-hole” approach to allele-selective BET bromodomain inhibition</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2452</span>– <span class="NLM_lpage">2468</span>, <span class="refDoi"> DOI: 10.1039/C7SC02536J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1039%2FC7SC02536J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=29732121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=2452-2468&author=A.+C.+Runcieauthor=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Testaauthor=L.+van+Beurdenauthor=M.+G.+J.+Baudauthor=O.+Epemoluauthor=L.+C.+J.+Ellisauthor=K.+D.+Readauthor=V.+Coulthardauthor=A.+Brienauthor=A.+Ciulli&title=Optimization+of+a+%E2%80%9Cbump-and-hole%E2%80%9D+approach+to+allele-selective+BET+bromodomain+inhibition&doi=10.1039%2FC7SC02536J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a "bump-and-hole" approach to allele-selective BET bromodomain inhibition</span></div><div class="casAuthors">Runcie, A. C.; Zengerle, M.; Chan, K.-H.; Testa, A.; van Beurden, L.; Baud, M. G. J.; Epemolu, O.; Ellis, L. C. J.; Read, K. D.; Coulthard, V.; Brien, A.; Ciulli, A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2452-2468</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Allele-specific chem. genetics enables selective inhibition within families of highly-conserved proteins.  The four BET (bromodomain & extra-terminal domain) proteins - BRD2, BRD3, BRD4 and BRDT bind acetylated chromatin via their bromodomains and regulate processes such as cell proliferation and inflammation.  BET bromodomains are of particular interest, as they are attractive therapeutic targets but existing inhibitors are pan-selective.  We previously established a bump-&-hole system for the BET bromodomains, pairing a leucine/alanine mutation with an ethyl-derived analog of an established benzodiazepine scaffold.  Here we optimize upon this system with the introduction of a more conservative and less disruptive leucine/valine mutation.  Extensive structure-activity-relationships of diverse benzodiazepine analogs guided the development of potent, mutant-selective inhibitors with desirable physiochem. properties.  The active enantiomer of our best compd. - 9-ME-1 - shows ~ 200 nM potency, >100-fold selectivity for the L/V mutant over wild-type and excellent DMPK properties.  Through a variety of in vitro and cellular assays we validate the capabilities of our optimized system, and then utilize it to compare the relative importance of the first and second bromodomains to chromatin binding.  These expts. confirm the primacy of the first bromodomain in all BET proteins, but also significant variation in the importance of the second bromodomain.  We also show that, despite having a minor role in chromatin recognition, BRD4 BD2 is still essential for gene expression, likely through the recruitment of non-histone proteins.  The disclosed inhibitor:mutant pair provides a powerful tool for future cellular and in vivo target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq47qHG4-zFurVg90H21EOLACvtfcHk0ljU_dsCofFTKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D&md5=9a44bdccabca9f1d4dfdc983250d9aa3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC7SC02536J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC02536J%26sid%3Dliteratum%253Aachs%26aulast%3DRuncie%26aufirst%3DA.%2BC.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3Dvan%2BBeurden%26aufirst%3DL.%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DEllis%26aufirst%3DL.%2BC.%2BJ.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCoulthard%26aufirst%3DV.%26aulast%3DBrien%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DOptimization%2520of%2520a%2520%25E2%2580%259Cbump-and-hole%25E2%2580%259D%2520approach%2520to%2520allele-selective%2520BET%2520bromodomain%2520inhibition%26jtitle%3DChem.%2520Sci.%26date%3D2018%26volume%3D9%26spage%3D2452%26epage%3D2468%26doi%3D10.1039%2FC7SC02536J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukagawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takisawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanemaki, M.</span></span> <span> </span><span class="NLM_article-title">An auxin-based degron system for the rapid depletion of proteins in nonplant cells</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">917</span>– <span class="NLM_lpage">922</span>, <span class="refDoi"> DOI: 10.1038/nmeth.1401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnmeth.1401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=19915560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVWlsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=917-922&author=K.+Nishimuraauthor=T.+Fukagawaauthor=H.+Takisawaauthor=T.+Kakimotoauthor=M.+Kanemaki&title=An+auxin-based+degron+system+for+the+rapid+depletion+of+proteins+in+nonplant+cells&doi=10.1038%2Fnmeth.1401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">An auxin-based degron system for the rapid depletion of proteins in nonplant cells</span></div><div class="casAuthors">Nishimura, Kohei; Fukagawa, Tatsuo; Takisawa, Haruhiko; Kakimoto, Tatsuo; Kanemaki, Masato</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">917-922</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Plants have evolved a unique system in which the plant hormone auxin directly induces rapid degrdn. of the AUX/IAA family of transcription repressors by a specific form of the SCF E3 ubiquitin ligase.  Other eukaryotes lack the auxin response but share the SCF degrdn. pathway, allowing one to transplant the auxin-inducible degron (AID) system into non-plant cells and use a small mol. to conditionally control protein stability.  The AID system allowed rapid and reversible degrdn. of target proteins in response to auxin and enabled the authors to generate efficient conditional mutants of essential proteins in yeast as well as cell lines derived from chicken, mouse, hamster, monkey and human cells, thus offering a powerful tool to control protein expression and study protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzZNy-0P1i8bVg90H21EOLACvtfcHk0ljU_dsCofFTKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVWlsbzM&md5=298f94ba9750fdbfdcaf6f10c62cd408</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1401%26sid%3Dliteratum%253Aachs%26aulast%3DNishimura%26aufirst%3DK.%26aulast%3DFukagawa%26aufirst%3DT.%26aulast%3DTakisawa%26aufirst%3DH.%26aulast%3DKakimoto%26aufirst%3DT.%26aulast%3DKanemaki%26aufirst%3DM.%26atitle%3DAn%2520auxin-based%2520degron%2520system%2520for%2520the%2520rapid%2520depletion%2520of%2520proteins%2520in%2520nonplant%2520cells%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26spage%3D917%26epage%3D922%26doi%3D10.1038%2Fnmeth.1401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safaee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponthier, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancias, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Plasticity in binding confers selectivity in ligand-induced protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0055-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fs41589-018-0055-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=29892083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=706-714&author=R.+P.+Nowakauthor=S.+L.+DeAngeloauthor=D.+Buckleyauthor=Z.+Heauthor=K.+A.+Donovanauthor=J.+Anauthor=N.+Safaeeauthor=M.+P.+Jedrychowskiauthor=C.+M.+Ponthierauthor=M.+Ishoeyauthor=T.+Zhangauthor=J.+D.+Manciasauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=E.+S.+Fischer&title=Plasticity+in+binding+confers+selectivity+in+ligand-induced+protein+degradation&doi=10.1038%2Fs41589-018-0055-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Plasticity in binding confers selectivity in ligand-induced protein degradation</span></div><div class="casAuthors">Nowak, Radoslaw P.; De Angelo, Stephen L.; Buckley, Dennis; He, Zhixiang; Donovan, Katherine A.; An, Jian; Safaee, Nozhat; Jedrychowski, Mark P.; Ponthier, Charles M.; Ishoey, Mette; Zhang, Tinghu; Mancias, Joseph D.; Gray, Nathanael S.; Bradner, James E.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">706-714</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Heterobifunctional small-mol. degraders that induce protein degrdn. through ligase-mediated ubiquitination have shown considerable promise as a new pharmacol. modality.  However, we currently lack a detailed understanding of the mol. basis for target recruitment and selectivity, which is critically required to enable rational design of degraders.  Here we utilized a comprehensive characterization of the ligand-dependent CRBN-BRD4 interaction to demonstrate that binding between proteins that have not evolved to interact is plastic.  Multiple x-ray crystal structures showed that plasticity resulted in several distinct low-energy binding conformations that were selectively bound by ligands.  We demonstrated that computational protein-protein docking can reveal the underlying interprotein contacts and inform the design of a BRD4 selective degrader that can discriminate between highly homologous BET bromodomains.  These findings that plastic interprotein contacts confer selectivity for ligand-induced protein dimerization provide a conceptual framework for the development of heterobifunctional ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreQasCW8bz2LVg90H21EOLACvtfcHk0lhSKaUhqESUiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK&md5=e4876c43b1eeb4558c23121b7a9fac27</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0055-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0055-y%26sid%3Dliteratum%253Aachs%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DBuckley%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DSafaee%26aufirst%3DN.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DPonthier%26aufirst%3DC.%2BM.%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DMancias%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DPlasticity%2520in%2520binding%2520confers%2520selectivity%2520in%2520ligand-induced%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D706%26epage%3D714%26doi%3D10.1038%2Fs41589-018-0055-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phipson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, G. K.</span></span> <span> </span><span class="NLM_article-title">Limma powers differential expression analyses for RNA-sequencing and microarray studies</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>, <span class="NLM_elocation-id">e47</span> <span class="refDoi"> DOI: 10.1093/nar/gkv007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1093%2Fnar%2Fgkv007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=25605792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFaiu7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&author=M.+E.+Ritchieauthor=B.+Phipsonauthor=D.+Wuauthor=Y.+Huauthor=C.+W.+Lawauthor=W.+Shiauthor=G.+K.+Smyth&title=Limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies&doi=10.1093%2Fnar%2Fgkv007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">limma powers differential expression analyses for RNA-sequencing and microarray studies</span></div><div class="casAuthors">Ritchie, Matthew E.; Phipson, Belinda; Wu, Di; Hu, Yifang; Law, Charity W.; Shi, Wei; Smyth, Gordon K.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e47/1-e47/13</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Limma is an R/Bioconductor software package that provides an integrated soln. for analyzing data from gene expression expts.  It contains rich features for handling complex exptl. designs and for information borrowing to overcome the problem of small sample sizes.  Over the past decade, limma has been a popular choice for gene discovery through differential expression analyses of microarray and high-throughput PCR data.  The package contains particularly strong facilities for reading, normalizing and exploring such data.  Recently, the capabilities of limma have been significantly expanded in two important directions.  First, the package can now perform both differential expression and differential splicing analyses of RNA sequencing (RNA-seq) data.  All the downstream anal. tools previously restricted to microarray data are now available for RNA-seq as well.  These capabilities allow users to analyze both RNA-seq and microarray data with very similar pipelines.  Second, the package is now able to go past the traditional gene-wise expression analyses in a variety of ways, analyzing expression profiles in terms of co-regulated sets of genes or in terms of higher-order expression signatures.  This provides enhanced possibilities for biol. interpretation of gene expression differences.  This article reviews the philosophy and design of the limma package, summarizing both new and historical features, with an emphasis on recent enhancements and features that have not been previously described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJykfsLP7n9bVg90H21EOLACvtfcHk0lhSKaUhqESUiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFaiu7%252FN&md5=e8a5f44160c650ad89adc48a9912bd96</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv007%26sid%3Dliteratum%253Aachs%26aulast%3DRitchie%26aufirst%3DM.%2BE.%26aulast%3DPhipson%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLaw%26aufirst%3DC.%2BW.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DSmyth%26aufirst%3DG.%2BK.%26atitle%3DLimma%2520powers%2520differential%2520expression%2520analyses%2520for%2520RNA-sequencing%2520and%2520microarray%2520studies%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26doi%3D10.1093%2Fnar%2Fgkv007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamprecht, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friman, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guertin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindquist, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golland, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">CellProfiler: Image analysis software for identifying and quantifying cell phenotypes</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">R100</span> <span class="refDoi"> DOI: 10.1186/gb-2006-7-10-r100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1186%2Fgb-2006-7-10-r100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=17076895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252Fjt1emsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&author=A.+E.+Carpenterauthor=T.+R.+Jonesauthor=M.+R.+Lamprechtauthor=C.+Clarkeauthor=I.+H.+Kangauthor=O.+Frimanauthor=D.+A.+Guertinauthor=J.+H.+Changauthor=R.+A.+Lindquistauthor=J.+Moffatauthor=P.+Gollandauthor=D.+M.+Sabatini&title=CellProfiler%3A+Image+analysis+software+for+identifying+and+quantifying+cell+phenotypes&doi=10.1186%2Fgb-2006-7-10-r100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">CellProfiler: image analysis software for identifying and quantifying cell phenotypes</span></div><div class="casAuthors">Carpenter Anne E; Jones Thouis R; Lamprecht Michael R; Clarke Colin; Kang In Han; Friman Ola; Guertin David A; Chang Joo Han; Lindquist Robert A; Moffat Jason; Golland Polina; Sabatini David M</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">R100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Biologists can now prepare and image thousands of samples per day using automation, enabling chemical screens and functional genomics (for example, using RNA interference).  Here we describe the first free, open-source system designed for flexible, high-throughput cell image analysis, CellProfiler.  CellProfiler can address a variety of biological questions quantitatively, including standard assays (for example, cell count, size, per-cell protein levels) and complex morphological assays (for example, cell/organelle shape or subcellular patterns of DNA or protein staining).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkZz-phUrfiVm8bhe6roZYfW6udTcc2eYLhqnL63kwMbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252Fjt1emsA%253D%253D&md5=79d0e947559e12792f864d764657d3f4</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2Fgb-2006-7-10-r100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2006-7-10-r100%26sid%3Dliteratum%253Aachs%26aulast%3DCarpenter%26aufirst%3DA.%2BE.%26aulast%3DJones%26aufirst%3DT.%2BR.%26aulast%3DLamprecht%26aufirst%3DM.%2BR.%26aulast%3DClarke%26aufirst%3DC.%26aulast%3DKang%26aufirst%3DI.%2BH.%26aulast%3DFriman%26aufirst%3DO.%26aulast%3DGuertin%26aufirst%3DD.%2BA.%26aulast%3DChang%26aufirst%3DJ.%2BH.%26aulast%3DLindquist%26aufirst%3DR.%2BA.%26aulast%3DMoffat%26aufirst%3DJ.%26aulast%3DGolland%26aufirst%3DP.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DCellProfiler%253A%2520Image%2520analysis%2520software%2520for%2520identifying%2520and%2520quantifying%2520cell%2520phenotypes%26jtitle%3DGenome%2520Biol.%26date%3D2006%26volume%3D7%26doi%3D10.1186%2Fgb-2006-7-10-r100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tutt, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santarosa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knights, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>434</i></span>,  <span class="NLM_fpage">917</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1038/nature03445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnature03445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=15829967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=434&publication_year=2005&pages=917-921&author=H.+Farmerauthor=N.+McCabeauthor=C.+J.+Lordauthor=A.+N.+Tuttauthor=D.+A.+Johnsonauthor=T.+B.+Richardsonauthor=M.+Santarosaauthor=K.+J.+Dillonauthor=I.+Hicksonauthor=C.+Knightsauthor=N.+M.+Martinauthor=S.+P.+Jacksonauthor=G.+C.+Smithauthor=A.+Ashworth&title=Targeting+the+DNA+repair+defect+in+BRCA+mutant+cells+as+a+therapeutic+strategy&doi=10.1038%2Fnature03445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy</span></div><div class="casAuthors">Farmer, Hannah; McCabe, Nuala; Lord, Christopher J.; Tutt, Andrew N. J.; Johnson, Damian A.; Richardson, Tobias B.; Santarosa, Manuela; Dillon, Krystyna J.; Hickson, Ian; Knights, Charlotte; Martin, Niall M. B.; Jackson, Stephen P.; Smith, Graeme C. M.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">434</span>
        (<span class="NLM_cas:issue">7035</span>),
    <span class="NLM_cas:pages">917-921</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">BRCA1 and BRCA2 are important for DNA double-strand break repair by homologous recombination, and mutations in these genes predispose to breast and other cancers.  Poly(ADP-ribose) polymerase (PARP) is an enzyme involved in base excision repair, a key pathway in the repair of DNA single-strand breaks.  The authors show here that BRCA1 or BRCA2 dysfunction unexpectedly and profoundly sensitizes cells to the inhibition of PARP enzymic activity, resulting in chromosomal instability, cell cycle arrest and subsequent apoptosis.  This seems to be because the inhibition of PARP leads to the persistence of DNA lesions normally repaired by homologous recombination.  These results illustrate how different pathways cooperate to repair damage, and suggest that the targeted inhibition of particular DNA repair pathways may allow the design of specific and less toxic therapies for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomVTKE4mFwfrVg90H21EOLACvtfcHk0ljiLteCpyiRmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFOmsrc%253D&md5=03b5ba0adc21a5627fe17ae535f45078</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature03445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03445%26sid%3Dliteratum%253Aachs%26aulast%3DFarmer%26aufirst%3DH.%26aulast%3DMcCabe%26aufirst%3DN.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DTutt%26aufirst%3DA.%2BN.%26aulast%3DJohnson%26aufirst%3DD.%2BA.%26aulast%3DRichardson%26aufirst%3DT.%2BB.%26aulast%3DSantarosa%26aufirst%3DM.%26aulast%3DDillon%26aufirst%3DK.%2BJ.%26aulast%3DHickson%26aufirst%3DI.%26aulast%3DKnights%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DN.%2BM.%26aulast%3DJackson%26aufirst%3DS.%2BP.%26aulast%3DSmith%26aufirst%3DG.%2BC.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DTargeting%2520the%2520DNA%2520repair%2520defect%2520in%2520BRCA%2520mutant%2520cells%2520as%2520a%2520therapeutic%2520strategy%26jtitle%3DNature%26date%3D2005%26volume%3D434%26spage%3D917%26epage%3D921%26doi%3D10.1038%2Fnature03445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loehr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahcall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of anaplastic lymphoma kinase (ALK)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4249</span>– <span class="NLM_lpage">4255</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4249-4255&author=C.+E.+Powellauthor=Y.+Gaoauthor=L.+Tanauthor=K.+A.+Donovanauthor=R.+P.+Nowakauthor=A.+Loehrauthor=M.+Bahcallauthor=E.+S.+Fischerauthor=P.+A.+Janneauthor=R.+E.+Georgeauthor=N.+S.+Gray&title=Chemically+induced+degradation+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1021%2Facs.jmedchem.7b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Powell, Chelsea E.; Gao, Yang; Tan, Li; Donovan, Katherine A.; Nowak, Radoslaw P.; Loehr, Amanda; Bahcall, Magda; Fischer, Eric S.; Janne, Pasi A.; George, Rani E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4249-4255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the development of the first small mol. degraders that can induce anaplastic lymphoma kinase (ALK) degrdn., including in non-small-cell lung cancer (NSCLC), anaplastic large-cell lymphoma (ALCL), and neuroblastoma (NB) cell lines.  These degraders were developed through conjugation of known pyrimidine-based ALK inhibitors, TAE684 or LDK378, and the cereblon ligand pomalidomide.  We demonstrate that in some cell types degrader potency is compromised by expression of drug transporter ABCB1.  In addn., proteomic profiling demonstrated that these compds. also promote the degrdn. of addnl. kinases including PTK2 (FAK), Aurora A, FER, and RPS6KA1 (RSK1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mDH7Sj8EqbVg90H21EOLACvtfcHk0lhjpmLLQEZfzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D&md5=486db11b3fbc2283700f38b09cc39bd7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01655%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DC.%2BE.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DLoehr%26aufirst%3DA.%26aulast%3DBahcall%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DGeorge%26aufirst%3DR.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4249%26epage%3D4255%26doi%3D10.1021%2Facs.jmedchem.7b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folkes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brumwell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chima, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunjan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pretswell, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dangerfield, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlton, P.</span></span> <span> </span><span class="NLM_article-title">Reversal of P-glycoprotein mediated multidrug resistance by novel anthranilamide derivatives</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">595</span>– <span class="NLM_lpage">600</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(99)00030-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1016%2FS0960-894X%2899%2900030-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10098671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADyaK1MXhvVylsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1999&pages=595-600&author=M.+Roeauthor=A.+Folkesauthor=P.+Ashworthauthor=J.+Brumwellauthor=L.+Chimaauthor=S.+Hunjanauthor=I.+Pretswellauthor=W.+Dangerfieldauthor=H.+Ryderauthor=P.+Charlton&title=Reversal+of+P-glycoprotein+mediated+multidrug+resistance+by+novel+anthranilamide+derivatives&doi=10.1016%2FS0960-894X%2899%2900030-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of P-glycoprotein mediated multidrug resistance by novel anthranilamide derivatives</span></div><div class="casAuthors">Roe, Michael; Folkes, Adrian; Ashworth, Philip; Brumwell, Julie; Chima, Lal; Hunjan, Sukhjit; Pretswell, Ian; Dangerfield, Wendy; Ryder, Hamish; Charlton, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">595-600</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">We have synthesized and evaluated a series of anthranilamide based modulators of P-glycoprotein.  These studies have identified XR9576, a potent inhibitor of P-glycoprotein in vitro and in vivo.  The general synthesis and the SAR of these compds. are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdQO2OdePDQrVg90H21EOLACvtfcHk0lhjpmLLQEZfzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhvVylsrs%253D&md5=d1e096ea9c3a30d3574fbe9b92c657cb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2899%2900030-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252899%252900030-X%26sid%3Dliteratum%253Aachs%26aulast%3DRoe%26aufirst%3DM.%26aulast%3DFolkes%26aufirst%3DA.%26aulast%3DAshworth%26aufirst%3DP.%26aulast%3DBrumwell%26aufirst%3DJ.%26aulast%3DChima%26aufirst%3DL.%26aulast%3DHunjan%26aufirst%3DS.%26aulast%3DPretswell%26aufirst%3DI.%26aulast%3DDangerfield%26aufirst%3DW.%26aulast%3DRyder%26aufirst%3DH.%26aulast%3DCharlton%26aufirst%3DP.%26atitle%3DReversal%2520of%2520P-glycoprotein%2520mediated%2520multidrug%2520resistance%2520by%2520novel%2520anthranilamide%2520derivatives%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1999%26volume%3D9%26spage%3D595%26epage%3D600%26doi%3D10.1016%2FS0960-894X%2899%2900030-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lgMOA69BPfhcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faust, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstock, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Bruton tyrosine kinase degradation as a therapeutic strategy for cancer</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">952</span>– <span class="NLM_lpage">961</span>, <span class="refDoi"> DOI: 10.1182/blood-2018-07-862953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1182%2Fblood-2018-07-862953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=30545835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1MXotlGjtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2019&pages=952-961&author=D.+Dobrovolskyauthor=E.+S.+Wangauthor=S.+Morrowauthor=C.+Leahyauthor=T.+Faustauthor=R.+P.+Nowakauthor=K.+A.+Donovanauthor=G.+Yangauthor=Z.+Liauthor=E.+S.+Fischerauthor=S.+P.+Treonauthor=D.+M.+Weinstockauthor=N.+S.+Gray&title=Bruton+tyrosine+kinase+degradation+as+a+therapeutic+strategy+for+cancer&doi=10.1182%2Fblood-2018-07-862953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Bruton tyrosine kinase degradation as a therapeutic strategy for cancer</span></div><div class="casAuthors">Dobrovolsky, Dennis; Wang, Eric S.; Morrow, Sara; Leahy, Catharine; Faust, Tyler; Nowak, Radoslaw P.; Donovan, Katherine A.; Yang, Guang; Li, Zhengnian; Fischer, Eric S.; Treon, Steven P.; Weinstock, David M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">952-961</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The covalent Bruton tyrosine kinase (BTK) inhibitor ibrutinib is highly efficacious against multiple B-cell malignancies.  However, it is not selective for BTK, and multiple mechanisms of resistance, including the C481S-BTK mutation, can compromise its efficacy.  We hypothesized that small-mol.-induced BTK degrdn. may overcome some of the limitations of traditional enzymic inhibitors.  Here, we demonstrate that BTK degrdn. results in potent suppression of signaling and proliferation in cancer cells and that BTK degraders efficiently degrade C481S-BTK.  Moreover, we discovered DD-03-171, an optimized lead compd. that exhibits enhanced antiproliferative effects on mantle cell lymphoma (MCL) cells in vitro by degrading BTK, IKFZ1, and IKFZ3 as well as efficacy against patient-derived xenografts in vivo.  Thus, "triple degrdn." may be an effective therapeutic approach for treating MCL and overcoming ibrutinib resistance, thereby addressing a major unmet need in the treatment of MCL and other B-cell lymphomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN4GLppunGArVg90H21EOLACvtfcHk0lgMOA69BPfhcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXotlGjtrY%253D&md5=ce7e48897378c2821d728daf6d9935d5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1182%2Fblood-2018-07-862953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2018-07-862953%26sid%3Dliteratum%253Aachs%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DMorrow%26aufirst%3DS.%26aulast%3DLeahy%26aufirst%3DC.%26aulast%3DFaust%26aufirst%3DT.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DWeinstock%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DBruton%2520tyrosine%2520kinase%2520degradation%2520as%2520a%2520therapeutic%2520strategy%2520for%2520cancer%26jtitle%3DBlood%26date%3D2019%26volume%3D133%26spage%3D952%26epage%3D961%26doi%3D10.1182%2Fblood-2018-07-862953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0liVxAsyqwMnXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnathan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: Lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Clementauthor=M.+Barnathanauthor=J.+M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.+W.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicodeme&title=Discovery+of+epigenetic+regulator+I-BET762%3A+Lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0liVxAsyqwMnXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DClement%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520Lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunker, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldie, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faty, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulrahman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantelic, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugasawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahlberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Cullin–RING ubiquitin E3 ligase regulation by the COP9 signalosome</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>531</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">603</span>, <span class="refDoi"> DOI: 10.1038/nature17416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.1038%2Fnature17416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=27029275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlWjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=531&publication_year=2016&pages=598-603&author=S.+Cavadiniauthor=E.+S.+Fischerauthor=R.+D.+Bunkerauthor=A.+Potenzaauthor=G.+M.+Lingarajuauthor=K.+N.+Goldieauthor=W.+I.+Mohamedauthor=M.+Fatyauthor=G.+Petzoldauthor=R.+E.+J.+Beckwithauthor=R.+B.+Tichkuleauthor=U.+Hassiepenauthor=W.+Abdulrahmanauthor=R.+S.+Pantelicauthor=S.+Matsumotoauthor=K.+Sugasawaauthor=H.+Stahlbergauthor=N.+H.+Thom%C3%A4&title=Cullin%E2%80%93RING+ubiquitin+E3+ligase+regulation+by+the+COP9+signalosome&doi=10.1038%2Fnature17416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin-RING ubiquitin E3 ligase regulation by the COP9 signalosome</span></div><div class="casAuthors">Cavadini, Simone; Fischer, Eric S.; Bunker, Richard D.; Potenza, Alessandro; Lingaraju, Gondichatnahalli M.; Goldie, Kenneth N.; Mohamed, Weaam I.; Faty, Mahamadou; Petzold, Georg; Beckwith, Rohan E. J.; Tichkule, Ritesh B.; Hassiepen, Ulrich; Abdulrahman, Wassim; Pantelic, Radosav S.; Matsumoto, Syota; Sugasawa, Kaoru; Stahlberg, Henning; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">531</span>
        (<span class="NLM_cas:issue">7596</span>),
    <span class="NLM_cas:pages">598-603</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The cullin-RING ubiquitin E3 ligase (CRL) family comprises >200 members in humans.  The COP9 signalosome complex (CSN) regulates CRLs by removing their ubiquitin-like activator NEDD8.  The CUL4A-RBX1-DDB1-DDB2 complex (CRL4ADDB2) monitors the genome for UV-light-induced DNA damage.  CRL4ADBB2 is inactive in the absence of damaged DNA and requires CSN to regulate the repair process.  The structural basis of CSN binding to CRL4ADDB2 and the principles of CSN activation are poorly understood.  Here, the authors present cryo-electron microscopy structures for CSN in complex with neddylated CRL4A ligases to 6.4 Å resoln.  The CSN conformers defined by cryo-electron microscopy and a novel apo-CSN crystal structure indicated an induced-fit mechanism that drives CSN activation by neddylated CRLs.  The authors found that CSN and a substrate could not bind simultaneously to CRL4A, favoring a deneddylated, inactive state for substrate-free CRL4 complexes.  These architectural and regulatory principles appeared to be conserved across CRL families, allowing global regulation by CSN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf2nrPiPIJq7Vg90H21EOLACvtfcHk0lgLNXDkah8l9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlWjsbc%253D&md5=9f537d713513d360d605320b6d7d7df1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnature17416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17416%26sid%3Dliteratum%253Aachs%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBunker%26aufirst%3DR.%2BD.%26aulast%3DPotenza%26aufirst%3DA.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DGoldie%26aufirst%3DK.%2BN.%26aulast%3DMohamed%26aufirst%3DW.%2BI.%26aulast%3DFaty%26aufirst%3DM.%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DAbdulrahman%26aufirst%3DW.%26aulast%3DPantelic%26aufirst%3DR.%2BS.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DSugasawa%26aufirst%3DK.%26aulast%3DStahlberg%26aufirst%3DH.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DCullin%25E2%2580%2593RING%2520ubiquitin%2520E3%2520ligase%2520regulation%2520by%2520the%2520COP9%2520signalosome%26jtitle%3DNature%26date%3D2016%26volume%3D531%26spage%3D598%26epage%3D603%26doi%3D10.1038%2Fnature17416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fink, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Thalidomide promotes degradation of SALL4, a transcription factor implicated in Duane Radial Ray syndrome</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e38430</span> <span class="refDoi"> DOI: 10.7554/eLife.38430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.7554%2FeLife.38430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=30067223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyqt7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=K.+A.+Donovanauthor=J.+Anauthor=R.+P.+Nowakauthor=J.+C.+Yuanauthor=E.+C.+Finkauthor=B.+C.+Berryauthor=B.+L.+Ebertauthor=E.+S.+Fischer&title=Thalidomide+promotes+degradation+of+SALL4%2C+a+transcription+factor+implicated+in+Duane+Radial+Ray+syndrome&doi=10.7554%2FeLife.38430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Thalidomide promotes degradation of SALL4, a transcription factor implicated in Duane Radial Ray syndrome</span></div><div class="casAuthors">Donovan, Katherine A.; An, Jian; Nowak, Radoslaw P.; Yuan, Jingting C.; Fink, Emma C.; Berry, Bethany C.; Ebert, Benjamin L.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e38430/1-e38430/25</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">In historical attempts to treat morning sickness, use of the drug thalidomide led to the birth of thousands of children with severe birth defects.  Despite their teratogenicity, thalidomide and related IMiD drugs are now a mainstay of cancer treatment; however, the mol. basis underlying the pleiotropic biol. and characteristic birth defects remains unknown.  Here we show that IMiDs disrupt a broad transcriptional network through induced degrdn. of several C2H2 zinc finger transcription factors, including SALL4, a member of the spalt-like family of developmental transcription factors.  Strikingly, heterozygous loss of function mutations in SALL4 result in a human developmental condition that phenocopies thalidomide-induced birth defects such as absence of thumbs, phocomelia, defects in ear and eye development, and congenital heart disease.  We find that thalidomide induces degrdn. of SALL4 exclusively in humans, primates, and rabbits, but not in rodents or fish, providing a mechanistic link for the species-specific pathogenesis of thalidomide syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogVJ6pTS4Us7Vg90H21EOLACvtfcHk0lh479B3asB_nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyqt7nM&md5=a21beb45bc2dd7701a0916d1dfc331b0</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.7554%2FeLife.38430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.38430%26sid%3Dliteratum%253Aachs%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DYuan%26aufirst%3DJ.%2BC.%26aulast%3DFink%26aufirst%3DE.%2BC.%26aulast%3DBerry%26aufirst%3DB.%2BC.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DThalidomide%2520promotes%2520degradation%2520of%2520SALL4%252C%2520a%2520transcription%2520factor%2520implicated%2520in%2520Duane%2520Radial%2520Ray%2520syndrome%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2FeLife.38430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McAlister, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nusinow, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wuhr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huttlin, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gygi, S. P.</span></span> <span> </span><span class="NLM_article-title">MultiNotch MS3 enables accurate, sensitive, and multiplexed detection of differential expression across cancer cell line proteomes</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">7150</span>– <span class="NLM_lpage">7158</span>, <span class="refDoi"> DOI: 10.1021/ac502040v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac502040v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1Kmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=7150-7158&author=G.+C.+McAlisterauthor=D.+P.+Nusinowauthor=M.+P.+Jedrychowskiauthor=M.+Wuhrauthor=E.+L.+Huttlinauthor=B.+K.+Ericksonauthor=R.+Radauthor=W.+Haasauthor=S.+P.+Gygi&title=MultiNotch+MS3+enables+accurate%2C+sensitive%2C+and+multiplexed+detection+of+differential+expression+across+cancer+cell+line+proteomes&doi=10.1021%2Fac502040v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">MultiNotch MS3 Enables Accurate, Sensitive, and Multiplexed Detection of Differential Expression across Cancer Cell Line Proteomes</span></div><div class="casAuthors">McAlister, Graeme C.; Nusinow, David P.; Jedrychowski, Mark P.; Wuhr, Martin; Huttlin, Edward L.; Erickson, Brian K.; Rad, Ramin; Haas, Wilhelm; Gygi, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7150-7158</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multiplexed quantitation via isobaric chem. tags (e.g., tandem mass tags (TMT) and isobaric tags for relative and abs. quantitation (iTRAQ)) has the potential to revolutionize quant. proteomics.  However, until recently the utility of these tags was questionable due to reporter ion ratio distortion resulting from fragmentation of coisolated interfering species.  These interfering signals can be negated through addnl. gas-phase manipulations (e.g., MS/MS/MS (MS3) and proton-transfer reactions (PTR)).  These methods, however, have a significant sensitivity penalty.  Using isolation waveforms with multiple frequency notches (i.e., synchronous precursor selection, SPS), the authors coisolated and cofragmented multiple MS2 fragment ions, thereby increasing the no. of reporter ions in the MS3 spectrum 10-fold over the std. MS3 method (i.e., MultiNotch MS3).  By increasing the reporter ion signals, this method improves the dynamic range of reporter ion quantitation, reduces reporter ion signal variance, and ultimately produces more high-quality quant. measurements.  To demonstrate utility, the authors analyzed biol. triplicates of eight colon cancer cell lines using the MultiNotch MS3 method.  Across all the replicates the authors quantified 8378 proteins in union and 6168 proteins in common.  Taking into account that each of these quantified proteins contains eight distinct cell-line measurements, this data set encompasses 174,704 quant. ratios each measured in triplicate across the biol. replicates.  Herein, the MultiNotch MS3 method uniquely combines multiplexing capacity with quant. sensitivity and accuracy, drastically increasing the informational value obtainable from proteomic expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonIBGSGUtKzbVg90H21EOLACvtfcHk0lgs5aNvPLuPPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1Kmur8%253D&md5=bda847612d4a16030b7921bc8ae6f5c6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fac502040v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac502040v%26sid%3Dliteratum%253Aachs%26aulast%3DMcAlister%26aufirst%3DG.%2BC.%26aulast%3DNusinow%26aufirst%3DD.%2BP.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DWuhr%26aufirst%3DM.%26aulast%3DHuttlin%26aufirst%3DE.%2BL.%26aulast%3DErickson%26aufirst%3DB.%2BK.%26aulast%3DRad%26aufirst%3DR.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26atitle%3DMultiNotch%2520MS3%2520enables%2520accurate%252C%2520sensitive%252C%2520and%2520multiplexed%2520detection%2520of%2520differential%2520expression%2520across%2520cancer%2520cell%2520line%2520proteomes%26jtitle%3DAnal.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D7150%26epage%3D7158%26doi%3D10.1021%2Fac502040v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">R Development Core Team</span>.  <i>R: A
Language and Environment
for Statistical Computing</i>; <span class="NLM_publisher-name">R Foundation for
Statistical Computing</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=R+Development+Core+Team&title=R%3A+A%0ALanguage+and+Environment%0Afor+Statistical+Computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DR%253A%2520A%250ALanguage%2520and%2520Environment%250Afor%2520Statistical%2520Computing%26pub%3DR%2520Foundation%2520for%250AStatistical%2520Computing%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenbraun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Key, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timony, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottaviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliz, P.</span></span> <span> </span><span class="NLM_article-title">Collaboration gets the most out of software</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e01456</span> <span class="refDoi"> DOI: 10.7554/eLife.01456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=10.7554%2FeLife.01456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;key=24040512" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&author=A.+Morinauthor=B.+Eisenbraunauthor=J.+Keyauthor=P.+C.+Sanschagrinauthor=M.+A.+Timonyauthor=M.+Ottavianoauthor=P.+Sliz&title=Collaboration+gets+the+most+out+of+software&doi=10.7554%2FeLife.01456"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.7554%2FeLife.01456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.01456%26sid%3Dliteratum%253Aachs%26aulast%3DMorin%26aufirst%3DA.%26aulast%3DEisenbraun%26aufirst%3DB.%26aulast%3DKey%26aufirst%3DJ.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DTimony%26aufirst%3DM.%2BA.%26aulast%3DOttaviano%26aufirst%3DM.%26aulast%3DSliz%26aufirst%3DP.%26atitle%3DCollaboration%2520gets%2520the%2520most%2520out%2520of%2520software%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26doi%3D10.7554%2FeLife.01456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O3C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O3C','PDB','5O3C'); return false;">PDB: 5O3C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BOY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BOY','PDB','6BOY'); return false;">PDB: 6BOY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BNB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BNB','PDB','6BNB'); return false;">PDB: 6BNB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CI1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CI1','PDB','4CI1'); return false;">PDB: 4CI1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FQD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FQD','PDB','5FQD'); return false;">PDB: 5FQD</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i41"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00958">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_71368"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00958?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00958</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplementary material containing supplementary figures and characterization of key compounds including NMR and HPLC traces for final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings with the binding and degradation assay data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_002.csv">CSV</a>)</p></li><li><p class="inline">PDB coordinates of the Rosetta protein docking model of the ternary complex of BRD4<sub>BD1</sub>–ZXH-3-26 and CRBN (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_001.pdf">jm1c00958_si_001.pdf (2.78 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_002.csv">jm1c00958_si_002.csv (1.59 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00958/suppl_file/jm1c00958_si_003.pdb">jm1c00958_si_003.pdb (329.17 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00958&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00958%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00958" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993041beea24eb","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
