
<html lang="en"     class="pb-page"  data-request-id="103ba445-2114-4f01-82a0-2b8f955ce8b7"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-24;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.0c01542;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Photoactive Bifunctional Degraders: Precision Tools To Regulate Protein Stability" /></meta><meta name="dc.Creator" content="Cyrille S.  Kounde" /></meta><meta name="dc.Creator" content="Edward W.  Tate" /></meta><meta name="dc.Description" content="Targeted protein degradation with bifunctional degraders is positioned as a remarkable game-changing strategy to control cellular protein levels and promises a new therapeutic modality in drug disc..." /></meta><meta name="Description" content="Targeted protein degradation with bifunctional degraders is positioned as a remarkable game-changing strategy to control cellular protein levels and promises a new therapeutic modality in drug disc..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 23, 2020" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01542" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01542" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01542" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01542" /></link>
        
    
    

<title>Photoactive Bifunctional Degraders: Precision Tools To Regulate Protein Stability | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01542" /></meta><meta property="og:title" content="Photoactive Bifunctional Degraders: Precision Tools To Regulate Protein Stability" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0007.jpeg" /></meta><meta property="og:description" content="Targeted protein degradation with bifunctional degraders is positioned as a remarkable game-changing strategy to control cellular protein levels and promises a new therapeutic modality in drug discovery. Light activation of a degrader to achieve exquisite spatiotemporal control over protein stability in cells has attracted the interest of multiple research groups, with recent reports demonstrating optical control of proteolysis with chimeric molecules bearing photolabile or photoswitchable motifs. In this context of targeted proteolysis research spurring the emergence of innovative tools, we examine the design, synthesis, and properties of light-activated degraders. The significant impact of this approach in regulating disease-relevant protein levels in a light-dependent manner is highlighted with key examples, and future developments to fully harness the potential of light-induced protein degradation with photoactive bifunctional molecules are discussed." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01542"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01542">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01542&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01542&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01542&amp;href=/doi/10.1021/acs.jmedchem.0c01542" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 15483-15493</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01502" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.0c01608" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Photoactive Bifunctional Degraders: Precision Tools To Regulate Protein Stability</span></h1><h2 class="article_header-subtitle"><span class="hlFld-SubTitle">Miniperspective</span></h2><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Cyrille S. Kounde</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cyrille S. Kounde</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Imperial College London, Molecular Sciences Research Hub, White City Campus, Wood Lane, London W12 0BZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cyrille+S.++Kounde">Cyrille S. Kounde</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6944-8717" title="Orcid link">http://orcid.org/0000-0001-6944-8717</a></div></div></span></li><span class="comma-separator"> and </span><li><span><span class="hlFld-ContribAuthor">Edward W. Tate</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Edward W. Tate</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Imperial College London, Molecular Sciences Research Hub, White City Campus, Wood Lane, London W12 0BZ, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ef8ac19b8e9b8aaf86829f8a9d868e83c18e8cc19a84"><span class="__cf_email__" data-cfemail="ea8fc49e8b9e8faa83879a8f98838b86c48b89c49f81">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Edward+W.++Tate">Edward W. Tate</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2213-5814" title="Orcid link">http://orcid.org/0000-0003-2213-5814</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01542&amp;href=/doi/10.1021%2Facs.jmedchem.0c01542" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 15483–15493</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 23, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>3 September 2020</li><li><span class="item_label"><b>Published</b> online</span>23 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 December 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01542" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01542</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D15483%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DCyrille%2BS.%2BKounde%252C%2BEdward%2BW.%2BTate%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D24%26contentID%3Dacs.jmedchem.0c01542%26title%3DPhotoactive%2BBifunctional%2BDegraders%253A%2BPrecision%2BTools%2BTo%2BRegulate%2BProtein%2BStability%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D15493%26publicationDate%3DDecember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01542"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1488</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01542" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Photoactive Bifunctional Degraders: Precision Tools To Regulate Protein Stability&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Cyrille&quot;,&quot;last_name&quot;:&quot;S. Kounde&quot;},{&quot;first_name&quot;:&quot;Edward&quot;,&quot;last_name&quot;:&quot;W. Tate&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;15483-15493&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01542&quot;},&quot;abstract&quot;:&quot;Targeted protein degradation with bifunctional degraders is positioned as a remarkable game-changing strategy to control cellular protein levels and promises a new therapeutic modality in drug discovery. Light activation of a degrader to achieve exquisite spatiotemporal control over protein stability in cells has attracted the interest of multiple research groups, with recent reports demonstrating optical control of proteolysis with chimeric molecules bearing photolabile or photoswitchable motifs. In this context of targeted proteolysis research spurring the emergence of innovative tools, we examine the design, synthesis, and properties of light-activated degraders. The significant impact of this approach in regulating disease-relevant protein levels in a light-dependent manner is highlighted with key examples, and future developments to fully harness the potential of light-induced protein degradation with photoactive bifunctional molecules are discussed.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01542&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01542" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01542&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01542" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01542&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01542" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01542&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01542&amp;href=/doi/10.1021/acs.jmedchem.0c01542" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01542" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01542" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01542%26sid%3Dliteratum%253Aachs%26pmid%3D33226810%26genre%3Darticle%26aulast%3DKounde%26date%3D2020%26atitle%3DPhotoactive%2BBifunctional%2BDegraders%253A%2BPrecision%2BTools%2BTo%2BRegulate%2BProtein%2BStability%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D24%26spage%3D15483%26epage%3D15493%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292444" title="Molecular structure">Molecular structure</a>,</li><li><a href="/action/doSearch?ConceptID=290619" title="Irradiation">Irradiation</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/jmcmar.2020.63.issue-24/20201224/jmcmar.2020.63.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Targeted protein degradation with bifunctional degraders is positioned as a remarkable game-changing strategy to control cellular protein levels and promises a new therapeutic modality in drug discovery. Light activation of a degrader to achieve exquisite spatiotemporal control over protein stability in cells has attracted the interest of multiple research groups, with recent reports demonstrating optical control of proteolysis with chimeric molecules bearing photolabile or photoswitchable motifs. In this context of targeted proteolysis research spurring the emergence of innovative tools, we examine the design, synthesis, and properties of light-activated degraders. The significant impact of this approach in regulating disease-relevant protein levels in a light-dependent manner is highlighted with key examples, and future developments to fully harness the potential of light-induced protein degradation with photoactive bifunctional molecules are discussed.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37575" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37575" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein expression levels within a cell are regulated by both pre- and post-translational mechanisms which maintain a fine balance between protein synthesis and protein degradation.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The ubiquitin–proteasome system (UPS), a key regulator of this balance, operates by tagging proteins with a polyubiquitin chain marking them for destruction at the proteasome.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Due to its involvement in several cellular processes such as cell cycle control, DNA repair, gene transcription, and apoptosis, dysregulation of the UPS is associated with multiple human pathologies including neurodegenerative diseases and cancers,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and pharmacological modulation of the UPS has been pursued extensively in drug discovery.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">In the past decade, targeted protein degradation (TPD) has emerged as an innovative approach to exploit the central ubiquitin-tagging mechanism of the UPS to deplete intracellular protein levels.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> TPD relies on the ability of molecular glues or heterobifunctional molecules to bring a target protein and an E3 ligase into close proximity in a ternary complex, leading to small molecule-directed ubiquitination and proteasomal degradation of the target protein.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> While molecular glues are single agents  for which the mode of action involves a conformational change of the E3 ligase resulting in the recruitment of neosubstrates,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> bifunctional molecules require two distinct ligands joined together by a linker, whereby the first ligand binds to the target protein and the second ligand recruits the E3 ubiquitin ligase (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). Because of their double-headed chemical structure, these compounds were first named proteolysis targeting chimeras (PROTACs).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> They are now established as powerful tools to regulate intracellular protein levels, and the first two PROTACs targeting the androgen receptor (ARV-110) and the estrogen receptor (ARV-471) recently entered clinical investigation for the treatment of metastatic castration-resistant prostate cancer and breast cancer, respectively.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. PROTAC mode of action and caging concept. (A) The PROTAC mediates the formation of a ternary complex between the protein of interest (POI) and an E3 ligase leading to post-translational modification of the POI with ubiquitin (Ub) and degradation within the proteasome catalytic unit. A caging group can be attached to the ligand recruiting the E3 ligase, to the warhead targeting the POI, and potentially to the linker, with the purpose of generating an inactive degrader. Degradation should occur only after removal of the caging group in response to irradiation with the appropriate wavelength of light. (B) Crystal structure of thalidomide bound to DDB1-CRBN E3 ubiquitin ligase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CI1">4CI1</a>)<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (left) and crystal structure of VHL ligand bound to pVHL:EloB:EloC complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W9H">4W9H</a>)<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (right). Caging group attachment points and exit vectors for the linker appendage are identified by red and black arrows, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The simplicity and versatility of small molecule TPD over biological strategies (e.g., genetic or RNAi protein silencing) have attracted the interest of chemical biologists and drug discovery scientists alike.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> Key benefits of PROTAC-mediated proteolysis include action at the protein level independent of regulation at the RNA level,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> a reversible and catalytic mode of action,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> an enhanced pharmacodynamic effect linked to the removal of all functions of the target including scaffolding capabilities,<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> and the opportunity to degrade proteins lacking a classical ligand binding or enzyme active site.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> The latest research efforts around bifunctional degraders have mainly been directed toward expansion of the substrate scope and have provided novel degrading tools for a wide variety of biological targets.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Likewise, ternary complex formation has been investigated in an attempt to identify rules that will better guide PROTAC design and optimization.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a></div><div class="NLM_p">In view of the great opportunity that TPD offers in controlling protein abundance, we and others have envisioned to expand the breadth of PROTAC applications by turning them into light-activated precision tools. Since biological processes regulated by proteins in living organisms are temporally and spatially defined, harnessing light as a tool which also operates in time and space has become a reliable strategy to achieve precise control over many cellular functions.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In the case of bifunctional degraders, light would allow the noninvasive spatiotemporal control of proteolysis, greatly refining the study of protein degradation. Furthermore, with degraders now progressing into the clinic, it may become important to develop strategies to address their potential on- and off-target liabilities.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Tissue-selective delivery of PROTACs has been reported by conjugation to antibodies,<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> and similarly the local application of light to deplete a disease-relevant protein in a tumor or other organ while sparing healthy tissue could improve PROTAC selectivity in vivo and reduce potential toxicity.</div><div class="NLM_p">Light-induced proteolysis was first explored using optogenetic methods<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> and was later accomplished with small molecules in genetically modified mammalian cells, as reported by Delacour et al. in 2015.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The authors developed a photoactivatable version of the plant hormone auxin  whichpromoted the degradation of an expressed target protein fused to an auxin-inducible degron (AID) sequence. Upon irradiation, the released auxin engaged the AID and recruited an E3 ligase for polyubiquitination and final proteasomal lysis of the fused construct. Although conditional degradation with light was achieved, the system required heterologous expression of a plant-derived protein in a mammalian cell to generate an artificial E3 ligase complex. In contrast, several recent reports have described light-induced protein degradation in mammalian cells without engineering the UPS system, by means of bifunctional degraders bearing photolabile or photoswitchable groups.<a onclick="showRef(event, 'ref31 ref32 ref33 ref34 ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33 ref34 ref35 ref36 ref37">(31−37)</a> Combining the attributes of light and PROTAC degradation has delivered promising results which have been highlighted in concurrent reports.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> In this Miniperspective, we carefully review the design, properties, and applications of these novel precision tools as well as the associated challenges and photopharmacology prospects they bring to the table.</div><div class="NLM_p last">While photoactive bifunctional degraders have been given diverse names and acronyms in the literature, the general terms “caged degrader” or “caged PROTAC” and “(photo)switchable [degrader/PROTAC]” will be used in this report for consistency and clarity.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Caged Degraders: Irreversible Activation</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25361" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25361" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The first strategy investigated to control PROTAC activity with light uses a photosensitive appendage on one side of the bifunctional molecule to generate an inactive degrader (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). Upon irradiation with UV light, an active degrader is irreversibly released to trigger the disposal of the studied protein. To qualify as suitable chemical tools for biological investigation, the designed caged molecules should be cell permeable, primarily inert as degraders, and stable intracellularly before photolysis. Moreover, release of the active degrader should occur in high yield after a minimal irradiation time.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Caging the Chimera: Heads or Tails?</h3><div class="NLM_p">The caged PROTACs reviewed here consist of a ligand for a specific target connected to one of the two most frequently used E3 ligase binders for cereblon (CRBN) or von Hippel–Lindau (VHL) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Since the binding modes of these E3 ligands for the respective ubiquitin ligase are well understood, notably through the −NH of the glutarimide group of thalidomide<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and the hydroxyl group of the VHL ligand,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> chemical functionalization with caging groups was mostly conducted on the E3 ligase recruiting part of the PROTAC (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33 ref34">(31−34)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0002.jpeg" id="Gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of the reported caged PROTACs. Each caged degrader is made of a warhead (depicted in blue), a linker (black), an E3 ligase recruiter (green), and a photocleavable group (red). Compounds <b>1</b>, <b>2</b>, <b>5</b>, <b>6</b>, and <b>8</b> were designed to degrade bromodomain-containing protein 4 (BRD4) upon irradiation. Compound <b>4</b> was designed to degrade the estrogen-related receptor α (ERRα) upon irradiation. Compounds <b>3</b> and <b>7</b> were designed to degrade Bruton tyrosine kinase (BTK) and anaplastic lymphoma kinase (ALK), respectively, upon irradiation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">This design relies on the ability of the caged moiety to prevent crucial binding interactions with the corresponding E3 ligase, thus precluding formation of the key ternary complex needed for ubiquitin transfer. Compounds <b>2</b>–<b>8</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) demonstrated the versatility of the approach, and the range of targets degraded by these molecules following uncaging suggests that caging the E3 binder may deliver optical control of degradation for any functional PROTAC. However, such caged PROTACs may still interact with the target of interest via their warhead and may display activity in a binding, biochemical, or cellular assay (vide infra). A caged degrader with attenuated or abolished pharmacological activity may be more advantageous, as it may leave the biological system unperturbed prior to irradiation. However, since degradation of a target protein often induces a greater pharmacodynamic response than its inhibition,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> a caged degrader with fully retained inhibitory activity against its target still represents a useful chemical tool provided its basal activity prior to irradiation is known or evaluated concurrently.</div><div class="NLM_p">A second approach to generate a photoactivatable PROTAC is to install the caging group on the target-binding moiety of the degrader, in which case, ternary complex formation is impeded due to lack of target engagement (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). The only reported example of this kind is caged molecule <b>1</b> which showed more than 100-fold loss in binding affinity against its target bromodomain-containing protein 4 (BRD4) compared to parent inhibitor JQ1.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Unlike the first approach, this design leaves the E3 ligase recruiter of the PROTAC accessible and compound <b>1</b> bears a thalidomide ligand that can interact with the E3 ligase cereblon.</div><div class="NLM_p">With regard to E3 ligase-mediated off-target effects due to PROTAC binding, Winter et al. have shown that the uncaged PROTAC for compound <b>1</b>, dBET1, recruits cereblon for degradation of BRD2–4 without inducing knockdown of the nuclear factor IKZF3.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Although cereblon-induced effects were not expected for caged PROTAC <b>1</b>, it is important to note that modulation of the E3 ligase partner can sometimes lead to unintentional degradation of novel protein substrates as demonstrated by the work of the Winter lab on phthalimide-based degraders.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> In contrast, indirect modulation of secondary proteins with VHL E3 ligase recruiting PROTACs has not been reported to date, and direct interference with VHL HIF binding or hypoxia regulation generally requires compound concentrations far in excess of those required for degradation by a PROTAC.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div><div class="NLM_p last">Considering the importance of the linker in PROTAC mode of action,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> a final option to generate a caged degrader would be to insert the photolabile group on the linker itself (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), particularly if chemical functionalization is impractical on the ligands of the degrader. Such an approach would rely on an unfavorable conformation adopted by the caged linker via steric or stereoelectronic effects to adversely impact ternary complex formation and could be enabled by structural data defining the linker orientation within the ternary complex formed by the parent PROTAC. Otherwise, the challenges of iterating caged linkers to identify a suitable inactive caged degrader suggest that functionalizing the warhead or the E3 ligase ligand with a caging group remains the most practical approach to obtain a caged PROTAC.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.2.  Caging Chemistry and Photocleavage</h3><div class="NLM_p">With most caged PROTACs bearing the photoactivatable group on the E3 ligase recruiting moiety, successful synthesis of the caged molecule is dependent on the chemistry that can be performed on the E3 ligase ligand. With the exception of compound <b>4</b>, all reported caged degraders incorporated a 6-nitroveratryl derived group, also known as 4,5-dimethoxy-2-nitrobenzyl cage (DMNB) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). DMNB is a frequently used photoprotecting group that can be activated using UV light with a maximum absorption at 365 nm <a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and has been installed as a carbamate (compounds <b>2, 3</b>, <b>6</b>, and <b>7</b>), an ether (compounds <b>5</b> and <b>8</b>), or a N-alkylated functionality (compound <b>1</b>). All final molecules were reported as stable intracellularly before irradiation as assessed by Western blots.</div><div class="NLM_p">While thalidomide and its derivatives could be efficiently alkylated on their imide portion (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, caged PROTAC <b>5</b>), acylation to obtain a carbamate usually required a strong base and was achieved in moderate yields (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, caged PROTACs <b>2</b>, <b>6</b>, <b>7</b>). Late stage chemistry was also attempted to insert the DMNB group directly on the parent PROTAC of <b>3</b>, but it delivered the caged molecule in low yield. A final caging step is advantageous, since in theory, a small library of PROTACs could be turned into a set of caged degraders readily, provided the warhead for the POI does not interfere with the chemistry to be performed on the E3 ligand moiety. Comparable to the DMNB group, the diethylamino coumarin (DEACM) of <b>4</b> could be activated at 365 nm and also at the longer 405 nm wavelength.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Chemical functionalization of E3 ligase ligands with photolabile groups. Varied strategies have been used to introduce the caging groups via acylation or alkylation with most groups installed relatively early in the synthesis as direct chemical modification of the parent PROTAC in the final step was not always feasible. The yield reported represents the yield of the caging step. “Early” caging refers to a synthetic route in which the caging group was not directly attached to the parent PROTAC as opposed to “late” caging. DBU = 1,8-diazabicyclo[5.4.0]undec-7-ene; DEACM = diethylamino coumarin; DIPEA = <i>N</i>,<i>N</i>-diisopropylethylamine; DMNB = 4,5-dimethoxy-2-nitrobenzyl; NaHMDS = sodium bis(trimethylsilyl)amide; NPOM = 6-nitropiperonyloxymethyl.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The photouncaging properties of the final molecules were evaluated in solution before initiating cell work. Upon irradiation for less than 180 s, greater than 50% uncaged material was observed in solution for most compounds (caged PROTACs <b>1</b>, <b>3</b>, <b>4</b>, <b>5</b>, and <b>8</b>). Interestingly, compound <b>2</b> failed to release any significant amount of active PROTAC after irradiation for 10 min in phosphate buffered saline (PBS) solution and was eventually not tested in cells. On the contrary, compound <b>6</b> which was made from the same precursor PROTAC as <b>2</b> (dBET1) and carried an identical caging group underwent photolysis to release the active degrader, although a prolonged irradiation time was required to reach significant uncaging in solution (<i>t</i><sub>1/2</sub> = 58 min, reported time to get 50% uncaging from a 1 mM solution of <b>6</b> as measured by UV–vis). High efficiency in releasing the parent molecule upon irradiation (i.e., good quantum yield) is often a desired feature of a photocage; however, the catalytic mode of action of PROTACs may compensate for this limitation. Despite much slower cleavage kinetics than other caged PROTACs, compounds <b>6</b> and <b>7</b> did degrade their respective target upon irradiation and within a concentration range close to their parent PROTAC.</div><div class="NLM_p last">For VHL-based caged degraders <b>4</b> and <b>8</b>, the chemistry was performed on the hydroxyl group of the VHL ligand. Naro et al. synthesized a DEACM caging group before O-acylating the parent PROTAC at the final stage with a reasonable yield, whereas our group used phase transfer catalysis to successfully install the DMNB group early on via O-alkylation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.3.  Caged Degraders in Action</h3><div id="sec2_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7">2.3.1.  Bromodomain-Containing Protein 4 (BRD4)</h4><div class="NLM_p">Degradation of bromodomain and extra terminal (BET) proteins with PROTACs has been explored extensively,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and BETs are often used as targets to validate new assays and methodologies related to targeted protein degradation. BRD4 is also an important target in oncology due to its role in regulation of several genes associated with cell proliferation and maintenance of cell cycle.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Therefore, controlling BRD4 stability with light-activated degraders would provide a compelling way to modulate cancer cell proliferation in a spatiotemporal manner. BRD4 degraders such as dBET1, MZ1, or ARV-825 are well-validated chemical tools with potent cellular activity<a onclick="showRef(event, 'ref42 ref44 ref49'); return false;" href="javascript:void(0);" class="ref ref42 ref44 ref49">(42,44,49)</a> and have shown promising pharmacodynamic effects in preclinical mouse models.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a></div><div class="NLM_p">The BRD4-targeting caged PROTACs reviewed here are directly or closely related to dBET1 (compounds <b>1</b>, <b>2</b>, <b>5</b>, <b>6</b>) or MZ1 (compound <b>8</b>). Consistent with their fast uncaging kinetics, caged compounds <b>1</b>, <b>5</b>, and <b>8</b> tested at 1 μM achieved complete degradation of BRD4 in various cell lines following irradiation at 365 nm for 180 s or less. Depletion of BRD4 was generally more pronounced with longer irradiation times, and the kinetics of degradation were very similar to the parent PROTAC using equal concentrations of caged material. The observed knockdown was also confirmed to be E3 ligase- and proteasome-dependent for all compounds, and the caged PROTACs <b>1</b>, <b>5</b>, and <b>6</b> negatively impacted cell survival upon irradiation (up to 39% reduction in cell viability for <b>5</b>). Work in our laboratory demonstrated that treatment of HeLa cells with caged PROTAC <b>8</b> had a cytostatic effect prior to irradiation (an effect attributed to BRD4 inhibition by the free JQ1 warhead) but led to strong cytotoxicity after uncaging (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). The consistent increase in toxicity observed upon photolysis of the caged degraders was not related to irradiation alone,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> nor did the byproduct generated during photoactivation affect BRD4 levels.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of caged PROTACs on cell proliferation and in zebrafish. (A) HeLa cells were treated with DMSO, caged PROTAC <b>8</b> (1 μM), or the uncaged parent PROTAC of <b>8</b> (1 μM) either without initial irradiation or with a 60 s irradiation time at 365 nm. Cell count was followed over 6 days. (B) Phenotype of zebrafish embryos treated with dBET1 and caged PROTAC <b>1</b> at 24 and 36 h postfertilization (hpf). (C) Expression of Brd4 protein in treated embryos examined by Western blot. Images adapted with permission from ref <a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a>. Copyright 2019 American Chemical Society.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To take light-induced degradation with caged molecules one step further, Xue et al. have investigated the knockdown of BRD4 with compound <b>1</b> in a zebrafish model, a widely used species in drug discovery and developmental biology.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The high BRD4 expression levels during fish embryogenesis combined with the transparency of the embryos allowed the authors to look at light-induced phenotypes originating from BRD4 depletion (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). While nonirradiated embryos treated with <b>1</b> developed a full yolk per the DMSO control, embryos irradiated at 365 nm for 10 min fail to grow a yolk extension in the presence of <b>1</b>, analogous to treatment with dBET1.</div><div class="NLM_p last">This result represents the first use of light-activated degraders in a complex multicellular organism and further highlights the important distinction between the pharmacological consequences of inhibition versus degradation.</div></div><div id="sec2_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8">2.3.2.  Estrogen-Related Receptor α (ERRα)</h4><div class="NLM_p last">In addition to BRD4, Naro et al. applied the photocaging strategy to a PROTAC targeting estrogen-related receptor α (ERRα), an orphan receptor with roles in metabolism, homeostasis, and breast cancer.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The parent PROTAC of <b>4</b> was reported by Bondeson et al. in 2015 and achieved significant reduction of ERRα receptor levels both in MCF-7 cells and in mice.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The caged degrader <b>4</b> was unable to degrade ERRα in the same cell line until it was uncaged by UV light. The relevance of the approach was also highlighted by triggering estrogen receptor α (ERα)-dependent transcription (normally repressed in the presence of ERRα) in MCF-7 cells transfected with a luciferase reporter gene. ERα-mediated transcription (monitored via the luciferase signal) was triggered by degradation of ERRα, which was in turn initiated upon uncaging of <b>4</b>.</div></div><div id="sec2_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9">2.3.3.  Bruton Tyrosine Kinase (BTK) and Anaplastic Lymphoma Kinase (ALK)</h4><div class="NLM_p last">Unsurprisingly, given their attractiveness as therapeutic targets in cancer, several kinases have been targeted successfully with PROTACs. This includes cyclic dependent kinases (CDKs),<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> focal-adhesion kinase (FAK),<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> TANK-binding kinase (TBK),<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and receptor tyrosine kinases (RTKs).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Caged PROTAC <b>3</b>, which is derived from MT-802, a potent Bruton tyrosine kinase (BTK) degrader developed by the Crews lab,<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> substantially decreased BTK levels in a dose-dependent manner only following irradiation at 365 nm. Likewise, <b>7</b> was designed by caging the anaplastic lymphoma kinase (ALK) degrader MS4078.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Photoirradiation of <b>7</b> strongly reduced cellular levels of oncogenic active ALK fusion proteins, EML4-ALK and NPM-ALK, in NCI-H2228 lung cancer cells and SU-DHL-1 lymphoma cells, respectively. To achieve an antiproliferative effect to the same extent as MS4078 on SU-DHL-1 cells, caged PROTAC <b>7</b> required activation by UV light.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">3.  Photoswitchable Degraders: Reversible Activation</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22574" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22574" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The examples above clearly demonstrate the potential of caged degraders as chemical tools to trigger target protein knockdown with timely precision. However, the caging strategy activates proteolysis in an irreversible manner, without the possibility to switch off the degradation event. Conversely, being able to trigger and halt protein degradation on demand would provide a subtle way to control cellular protein levels and associated biological functions. Astute structural modifications of bifunctional degraders with photoswitchable motifs have been devised to accomplish such reversible control.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">3.1.  Photoswitch Design: Azobenzene at the Forefront</h3><div class="NLM_p">Using geometrical isomerism to reversibly control biological processes has been exemplified extensively with azobenzene photoswitches,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and their chemical tractability and tunable properties also make them an appropriate choice for photoswitchable PROTACs.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> It was hypothesized that incorporation of an azobenzene motif into a bifunctional molecule would allow for major changes in binary and ternary binding interactions to take place upon irradiation due to <i>cis</i>/<i>trans</i> isomerism. This change in configuration would then translate into a distinct degradation profile for each isomer (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). As a consequence, turning proteolysis on or off would be feasible with a single molecule, simply by switching between the two geometrical isomers.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Photoswitchable degrader design, structures, and activity profiles. (A) The azobenzene motif delivers two geometrical isomers: the <i>cis</i> isomer can be obtained by irradiation of the <i>trans</i> isomer with a specific wavelength λ<sub>1</sub>, and inversely, the <i>trans</i> isomer can be generated from the <i>cis</i> either with another wavelength λ<sub>2</sub> or by thermal relaxation. The difference in conformation between the two isomers affects binding interactions and allows degradation to occur with only one isomer (<i>cis</i> or <i>trans</i>) but not the other. (B) Structure of the reported photoswitchable PROTAC <b>9</b> (left). Each degrader is made of a warhead (depicted in blue), an E3 ligase recruiter (green), and a photoswitch (red) as part of the linker (black). Immunoblot analysis after treatment of Ramos cells with compound <b>9</b> (right). The <i>trans</i> isomer of <b>9</b> degraded BRD2, while the <i>cis</i> isomer did not affect BRD2 levels. Image adapted with permission from ref <a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a>. Copyright 2019 American Chemical Society. (C) Structure of the reported photoswitchable PROTAC <b>10</b> (left) and immunoblot analysis after treatment of RS4;11 cells with compound <b>10</b> (right). The <i>cis</i> isomer of <b>10</b> knocked down BRD2, BRD3, and BRD4, while the <i>trans</i> isomer did not degrade any BRD. Image adapted with permission from ref <a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>. Copyright The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. Distributed under a Creative Commons Attribution License 4.0 (CC BY) <a href="https://creativecommons.org/licenses/by/4.0/" class="extLink">https://creativecommons.org/licenses/by/4.0/</a>. (D) Structure of the reported photoswitchable PROTAC <b>11</b> (left). Immunoblot analysis following treatment of K562 cells with compound <b>11</b> (right). The <i>trans</i> isomer of <b>11</b> was efficient in reducing levels of BCR-ABL and ABL while its <i>cis</i> counterpart showed no degradation activity. Image adapted with permission from ref <a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a>. Copyright 2020 American Chemical Society.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As previously mentioned, the linker that connects the two ligands of a PROTAC plays a critical role in a degrader’s efficacy.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Parameters such as composition and length of the linker can have a profound impact on a degrader’s physicochemical properties as well as its conformation.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Taking this into account, the Carreira and Jian labs engineered the azobenzene switch as a central linker positioned between the E3 ligand and the warhead against the target protein (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D, compounds <b>9</b> and <b>11</b>).<a onclick="showRef(event, 'ref35 ref37'); return false;" href="javascript:void(0);" class="ref ref35 ref37">(35,37)</a> Moreover, in their search for a bistable PROTAC which does not require continuous irradiation (i.e., thermodynamically stable as a <i>trans</i> isomer with a long <i>cis</i> thermal half-life), the Carreira lab incorporated into compound <b>9</b> an <i>ortho</i>-F<sub>4</sub>-azobenzene motif reported to possess prolonged half-life.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Bistable <i>cis</i>/<i>trans</i> isomers may prove beneficial to outcompete fast protein resynthesis (i.e., in the case of proteins with a high turnover), as higher concentrations of the desired isomer would be maintained without light stimulation contrary to fast relaxing switches.</div><div class="NLM_p">In parallel, the Trauner lab investigated multiple positions for the azobenzene group, and their most promising molecule adapted the phenyl ring of the E3 ligase ligand lenalidomide as an integral part of the photoswitch (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C, compound <b>10</b>).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Synthesis of the highlighted tool compounds <b>9</b>–<b>11</b> was mainly achieved via conventional azo coupling to obtain the photoswitch followed by additional amide bond formation steps to reach several final molecules in a modular fashion. Key  characteristics pertaining to those compounds are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and underscore the diversity in optical properties and degradation profiles that can be obtained using an azobenzene photoswitch.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Diverse Characteristics and Degradation Profiles (<i>cis</i> vs <i>trans</i>) of Photoswitchable Degraders <b>9</b>, <b>10</b>, and <b>11</b><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">characteristic</th><th class="colsep0 rowsep0" align="center"><b>9</b></th><th class="colsep0 rowsep0" align="center"><b>10</b></th><th class="colsep0 rowsep0" align="center"><b>11</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">photoswitch type</td><td class="colsep0 rowsep0" align="left"><i>ortho</i>-F4-azobenzene</td><td class="colsep0 rowsep0" align="left">azobenzene</td><td class="colsep0 rowsep0" align="left">azobenzene</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">activity profile obtained</td><td class="colsep0 rowsep0" align="left"><i>trans</i> active/<i>cis</i> inactive</td><td class="colsep0 rowsep0" align="left"><i>cis</i> active/<i>trans</i> inactive</td><td class="colsep0 rowsep0" align="left"><i>trans</i> active/<i>cis</i> inactive</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">E3 ligand/E3 recruited</td><td class="colsep0 rowsep0" align="left">VH032/VHL</td><td class="colsep0 rowsep0" align="left">lenalidomide/cereblon</td><td class="colsep0 rowsep0" align="left">lenalidomide/cereblon</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">warhead/protein degraded</td><td class="colsep0 rowsep0" align="left">JQ1/BRD2</td><td class="colsep0 rowsep0" align="left">JQ1/BRD4</td><td class="colsep0 rowsep0" align="left">dasatinib/BCR-ABL</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">isoform degradation profile</td><td class="colsep0 rowsep0" align="left">BRD4 not degraded</td><td class="colsep0 rowsep0" align="left">BRD4, BRD3 > BRD2</td><td class="colsep0 rowsep0" align="left">ABL degraded</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">main irradiation mode<a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">single initial irradiation</td><td class="colsep0 rowsep0" align="left">continuous pulse irradiation</td><td class="colsep0 rowsep0" align="left">single initial irradiation</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">λ <i>trans</i> to <i>cis</i>/PSS<a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">530 nm/68% <i>cis</i></td><td class="colsep0 rowsep0" align="left">390 nm/>90% <i>cis</i></td><td class="colsep0 rowsep0" align="left">UVC<a class="ref internalNav" href="#t1fn3" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">λ <i>cis</i> to <i>trans</i>/PSS<a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">415 nm/95% <i>trans</i></td><td class="colsep0 rowsep0" align="left">>450 nm/>70% <i>trans</i></td><td class="colsep0 rowsep0" align="left">white light<a class="ref internalNav" href="#t1fn3" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>cis</i> half-life thermal relaxation<a class="ref internalNav" href="#t1fn4" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">several days (37 °C, DMSO)</td><td class="colsep0 rowsep0" align="left">8.8 h (37 °C, DMSO)</td><td class="colsep0 rowsep0" align="left">10.3 h (25 °C)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">photoswitch resistance</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">multiple cycles tested</td><td class="colsep0 rowsep0" align="left">5 cycles tested</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">in-cell activation and deactivation of degradation</td><td class="colsep0 rowsep0" align="left">achieved</td><td class="colsep0 rowsep0" align="left">achieved</td><td class="colsep0 rowsep0" align="left">achieved</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">stability toward GSH reduction</td><td class="colsep0 rowsep0" align="left">stable</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">stable</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup><sup>a</sup></sup><p class="last">The type of photoswitch, E3 ligase ligand, and warhead used can all affect degradation.</p></div><div class="footnote" id="t1fn1"><sup><sup>b</sup></sup><p class="last">Varied light sources used in these studies; refer to original publications for details.</p></div><div class="footnote" id="t1fn2"><sup><sup>c</sup></sup><p class="last">PSS = photostationary state, reported as percentage of <i>cis</i> or <i>trans</i> at equilibrium after irradiation at λ.</p></div><div class="footnote" id="t1fn3"><sup><sup>d</sup></sup><p class="last">Wavelength not specified, no PSS data.</p></div><div class="footnote" id="t1fn4"><sup><sup>e</sup></sup><p class="last">Measured in the dark.</p></div></div></div><div class="NLM_p last">Overall, the thermodynamically more stable <i>trans</i> isomer could be switched to its <i>cis</i> counterpart using UV or visible light and conversion was achieved with satisfactory to excellent photostationary state (PSS) ratios (68% <i>cis</i> for <b>9</b>, above 90% <i>cis</i> for <b>10</b>). Compound <b>9</b> was reported as bistable since thermal back isomerization from <i>cis</i> to <i>trans</i> was not observed after incubation at 37 °C for several days in solution. Although reasonable <i>cis</i> thermal half-lives were obtained for <b>10</b> and <b>11</b> (8.8 and 10.3 h, respectively), the active <i>cis</i> isomer of <b>10</b> was mainly tested under continuous pulse irradiation to maximize the intracellular concentration of the active isomer over time. Apart from thermal relaxation, switching back to the <i>trans</i> isomer could also be realized for <b>9</b> and <b>10</b> with good PSS using another wavelength of light (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Lastly, in addition to favorable photochemical properties, these novel photoswitchable degraders demonstrated good stability after multiple irradiation cycles and could withstand glutathione without detectable reduction, suggesting  a satisfactory intracellular stability.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">3.2.  Taming the Chimera: On/Off Switching</h3><div class="NLM_p">With regard to degradation activity, the azobenzene photoswitch was generally well tolerated as an atypical linker. The two isomers from compound <b>9</b>, based on pan-BET degrader ARV-771,<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> displayed a different degradation profile toward BRD2 (<i>trans</i> active, <i>cis</i> inactive) while leaving BRD4 unaffected (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). This selectivity toward BRD2 was partly attributed to the unusual inverse amide bond connecting JQ1 to the <i>ortho</i>-F<sub>4</sub>-azobenzene moiety. In contrast, compound <b>10</b> showed an opposite activity profile with the <i>cis</i> isomer able to degrade BRD3, BRD4, and to a lesser extent BRD2, whereas the <i>trans</i> isomer was completely inactive toward all BRDs (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). With the photoswitch embedded in the E3 ligase ligand, considerable changes in binary interactions (with cereblon E3 ligase) but also in ternary complex formation would be expected upon isomerization of <b>10</b> and presumably lead to this distinctive degradation outcome. As for compound <b>11</b>, its <i>trans</i> isomer degraded BCR-ABL fusion protein and ABL while its <i>cis</i> isomer did not affect the levels of those proteins (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). It is worth noting that although the pairs of isomers of <b>9</b>, <b>10</b>, and <b>11</b> showed remarkably unique degradation profiles between <i>cis</i> and <i>trans</i>, many of the related compounds synthesized in the course of those studies did not exhibit a clearly differentiated effect. The extensive investigation from Trauner’s lab which reported a total of 19 photoswitchable degraders including 6 targeting the prolyl <i>cis</i>-<i>trans</i> isomerase FKBP12 suggests that linker length is one critical factor among several influencing the activity profile of the two geometrical isomers (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Taken together, those results highlight the need for additional tools to guide the design of reversible light-activated degraders and to better understand their mode of action.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Influence of the linker length on FKBP12 photoswitchable PROTAC degradation profile. The warhead is depicted in blue, the E3 ligase recruiter is in green, and the photoswitch forming part of the black linker is depicted in red. RS4;11 cells were treated with various FKBP12 targeting photoswitchable compounds for 4 h at different concentrations. Cells were either irradiated with pulses of 390 nm light (<i>cis</i>) or kept the dark (<i>trans</i>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Finally, reversible initiation or arrest of proteolysis upon photoisomerization was an important proof of concept result demonstrated with compounds <b>9</b>, <b>10</b>, and <b>11</b>. In-cell bidirectional switching was achieved using two separate wavelengths and could also be accomplished via thermal relaxation in the case of <b>10</b>. This two-way on/off mechanism, which is strictly dependent on the photoswitchable PROTAC configuration, undoubtedly represents an important milestone in the spatiotemporal control of protein stability with a small molecule. To further illustrate the benefit of this approach, the effect of photoswitchable degrader <b>10</b> on cell viability was correlated with degradation of BRDs by the <i>cis</i> isomer. <b>10</b> was 7 times more toxic when irradiated at 390 nm (<i>cis</i> isomer) than when incubated in the dark (<i>trans</i> isomer).</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">4.  Summary and Outlook</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76154" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76154" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Bifunctional degraders can be engineered into light-activated chemical tools with photocaging or photoswitchable functionalities, providing a compelling method for the spatiotemporal control of protein stability. Inherently, caged degraders can only switch on proteolysis via a relatively fast uncaging mechanism, mainly triggered by UV light and which generates a byproduct. As for photoswitchable degraders, their azobenzene motif allows isomerization within the visible wavelength range and offers a bidirectional on/off handle with a superior level of control, even though they may not deliver quantitative switching. Indeed, residual activity from small quantities of active isomer present in the PSS may compromise the pharmacological applications of switchable PROTACs and the stringency of switching required is likely to depend on the specific application. Furthermore, we note that PROTACs already present difficult profiles for oral delivery and balancing the physicochemical properties of two PROTAC isomers may present additional challenges of optimization.</div><div class="NLM_p">Although bistable switchable degraders (such as compound <b>9)</b> avoid the need for continuous irradiation, fast relaxing switches may find useful applications, particularly if sustained degradation can be attained after transient generation of the bioactive <i>cis</i> isomer and on- or off-target toxicities minimized by subsequent fast inactivation. A tight-binding PROTAC could be retained by its target proteins for a prolonged period in the cell, and thermal reversion may assist in taming extended pharmacodynamics. Decisions on optimal half-lives needed for the active isomer may ultimately depend on the target protein and specific applications pursued, and it remains unclear whether this automatic “switch off” characteristic would offer any benefit over tuning half-life through inactivation by metabolism or excretion.</div><div class="NLM_p">We also note that caging of the E3 ligase component of a PROTAC (compounds <b>3</b>–<b>8</b>) is straightforward and highly modular and offers a mechanistically predictable outcome in switching on degradation upon photoirradiation. In comparison, the reports to date on photoswitchable degraders reveal that incorporation of a photoswitch into a bifunctional molecule can be explorative and identification of geometrical isomers with distinct degradation potency may require multiple synthetic iterations. More importantly, explanation of the observed structure-degradation profiles (which  is also  target-dependent) has yet to be provided at a molecular level for photoswitchable degraders and will certainly  involve a combination of biophysical and computational methods not only to understand the data a posteriori but also to guide future design.</div><div class="NLM_p">Selective and differential degradation of closely related proteins (i.e., proteins within a subfamily or isoforms) is another aspect worthy of investigation with photoswitchable degraders. This singular characteristic of PROTACs has mainly been described for kinase-targeting degraders,<a onclick="showRef(event, 'ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref63 ref64">(63,64)</a> and the data from Pfaff et al. (BRD2 selective degradation with <i>trans</i> isomer of <b>9</b>) encourage further exploitation of this feature with a photoswitch design. Differential degradation of structurally related proteins upon configurational change of the degrader could permit more precise targeting and study of protein function.</div><div class="NLM_p">Equipped with the capacity to degrade proteins via a light-dependent mechanism and consequently modulate cellular functions, photoactive degraders have strong potential for development into the field of photopharmacology provided constraints associated with their physicochemical properties can be overcome.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> The  inclusion of lipophilic photolabile groups and  azobenzene units into degraders place these compounds into an undesired drug property space. Despite their high molecular weight and high lipophilicity, the caged and photoswitchable PROTACs reviewed here displayed sufficient cell permeability and stability to achieve controlled intracellular protein knockdown. Nonetheless, we anticipate that the progression of such tool compounds in vivo (in animal models or in the clinic) will necessitate careful assessment and optimization of their pharmacokinetic properties while preserving their favorable photochemical features.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a></div><div class="NLM_p last">Finally, due to activating wavelengths within the UV–visible range, in vivo applications of photoactive PROTACs may be limited by light-induced cellular damage (if long irradiation times with UV light are needed) and poor tissue penetration of the incident light. Alongside sophisticated light-delivery methods,<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> strategies to develop photocaging groups and photoswitches activated in the red and near-infrared regions are rapidly emerging.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a>  Integration of these innovations in the design of novel photoactive bifunctional degraders will allow deeper in vivo exploration and manipulation of biological pathways,  ultimately enabling advances in precision medicine.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01542" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74211" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74211" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edward W. Tate</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Imperial
College London, Molecular Sciences Research Hub, White City Campus,
Wood Lane, London W12 0BZ, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2213-5814" title="Orcid link">http://orcid.org/0000-0003-2213-5814</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4d2863392c39280d24203d283f242c21632c2e633826"><span class="__cf_email__" data-cfemail="187d366c796c7d587175687d6a71797436797b366d73">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cyrille S. Kounde</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Imperial
College London, Molecular Sciences Research Hub, White City Campus,
Wood Lane, London W12 0BZ, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6944-8717" title="Orcid link">http://orcid.org/0000-0001-6944-8717</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Edward W. Tate is a founder, Director, and shareholder of Myricx Pharma, Ltd.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85108" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85108" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Cyrille S. Kounde</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=BIO-d7e1536-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Cyrille S. Kounde</b> completed his B.Sc. degree in Biochemistry at Paris Diderot University and his M.Sc. in Organic Chemistry at Paris Descartes University. In 2006, he started his career in industry in the United Kingdom as a drug discovery scientist at Evotec Ltd. and then joined Novartis. He moved to Singapore in 2011 and worked on various medicinal chemistry projects related to neglected infectious diseases at the Novartis Institute for Tropical Diseases (NITD). He is currently a Ph.D. student in the group of Prof. Ed Tate at Imperial College London where he is investigating conditional activation of bifunctional degraders.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Edward W. Tate</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=BIO-d7e1541-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Edward W. Tate</b> is Professor of Chemical Biology in the Department of Chemistry at Imperial College London, a Group Leader at the Francis Crick Institute, and Chief Scientific Officer of Myricx Pharma Ltd. He completed his B.Sc. degree in Chemistry at the University of Durham and undertook his Ph.D. in Organic Chemistry at the University of Cambridge under the guidance of Prof. Steve Ley. Following postdoctoral research at Ecole Polytechnique and the Pasteur Institute in Paris, he was awarded a BBSRC David Phillips Research Fellowship in 2006 at Imperial College London and was promoted to full Professor in 2014. His research encompasses the fields of chemical biology and chemical proteomics, with a focus on drug target discovery and validation.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36246" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36246" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Maria Maneiro, Dr. Marcel Muelbaier, and Dr. John D. Harling for their critical evaluation of the draft manuscript. The authors acknowledge support from the Engineering and Physical Sciences Research Council (EPSRC) Grant EP/R512540/1 and GlaxoSmithKline Research and Development Ltd.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AID</td><td class="NLM_def"><p class="first last">auxin-inducible degron</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra terminal</p></td></tr><tr><td class="NLM_term">BRD</td><td class="NLM_def"><p class="first last">bromodomain containing protein</p></td></tr><tr><td class="NLM_term">BTK</td><td class="NLM_def"><p class="first last">Bruton tyrosine kinase</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">DDB1</td><td class="NLM_def"><p class="first last">DNA damage-binding protein 1</p></td></tr><tr><td class="NLM_term">DEACM</td><td class="NLM_def"><p class="first last">diethylamino coumarin</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMNB</td><td class="NLM_def"><p class="first last">4,5-dimethoxy-2-nitrobenzyl</p></td></tr><tr><td class="NLM_term">EloB</td><td class="NLM_def"><p class="first last">elongin-B</p></td></tr><tr><td class="NLM_term">EloC</td><td class="NLM_def"><p class="first last">elongin-C</p></td></tr><tr><td class="NLM_term">FAK</td><td class="NLM_def"><p class="first last">focal adhesion kinase</p></td></tr><tr><td class="NLM_term">IKZF3</td><td class="NLM_def"><p class="first last">zinc finger protein Aiolos</p></td></tr><tr><td class="NLM_term">LED</td><td class="NLM_def"><p class="first last">light-emitting diode</p></td></tr><tr><td class="NLM_term">NaHMDS</td><td class="NLM_def"><p class="first last">sodium bis(trimethylsilyl)amide</p></td></tr><tr><td class="NLM_term">NPOM</td><td class="NLM_def"><p class="first last">6-nitropiperonyloxymethyl</p></td></tr><tr><td class="NLM_term">POI</td><td class="NLM_def"><p class="first last">protein of interest</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">PSS</td><td class="NLM_def"><p class="first last">photostationary state</p></td></tr><tr><td class="NLM_term">RNAi</td><td class="NLM_def"><p class="first last">ribonucleic acid interference</p></td></tr><tr><td class="NLM_term">RTK</td><td class="NLM_def"><p class="first last">receptor tyrosine kinase</p></td></tr><tr><td class="NLM_term">TBK</td><td class="NLM_def"><p class="first last">TANK-binding kinase</p></td></tr><tr><td class="NLM_term">TPD</td><td class="NLM_def"><p class="first last">targeted protein degradation</p></td></tr><tr><td class="NLM_term">Ub</td><td class="NLM_def"><p class="first last">ubiquitin</p></td></tr><tr><td class="NLM_term">UPS</td><td class="NLM_def"><p class="first last">ubiquitin–proteasome system</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06864" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06864" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 68 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alber, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suter, D. M.</span></span> <span> </span><span class="NLM_article-title">Dynamics of Protein Synthesis and Degradation through the Cell Cycle</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">784</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.1080/15384101.2019.1598725</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1080%2F15384101.2019.1598725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=30907235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsVSqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=784-794&issue=8&author=A.+B.+Alberauthor=D.+M.+Suter&title=Dynamics+of+Protein+Synthesis+and+Degradation+through+the+Cell+Cycle&doi=10.1080%2F15384101.2019.1598725"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics of protein synthesis and degradation through the cell cycle</span></div><div class="casAuthors">Alber, Andrea Brigitta; Suter, David Michael</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">784-794</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1551-4005</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein expression levels depend on the balance between their synthesis and degrdn. rates.  Even quiescent (G0) cells display a continuous turnover of proteins, despite protein levels remaining largely const. over time.  In cycling cells, global protein levels need to be precisely doubled at each cell division in order to maintain cellular homeostasis, but we still lack a quant. understanding of how this is achieved.  Recent studies have shed light on cell cycle-dependent changes in protein synthesis and degrdn. rates.  Here we discuss current population-based and single cell approaches used to assess protein synthesis and degrdn., and review the insights they have provided into the dynamics of protein turnover in different cell cycle phases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4nVSPIQg-v7Vg90H21EOLACvtfcHk0lgPb9fJc5KgUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsVSqtLw%253D&md5=98a155677bc6f1c56e5d01ed1b4bcd04</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1080%2F15384101.2019.1598725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15384101.2019.1598725%26sid%3Dliteratum%253Aachs%26aulast%3DAlber%26aufirst%3DA.%2BB.%26aulast%3DSuter%26aufirst%3DD.%2BM.%26atitle%3DDynamics%2520of%2520Protein%2520Synthesis%2520and%2520Degradation%2520through%2520the%2520Cell%2520Cycle%26jtitle%3DCell%2520Cycle%26date%3D2019%26volume%3D18%26issue%3D8%26spage%3D784%26epage%3D794%26doi%3D10.1080%2F15384101.2019.1598725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hershko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciechanover, A.</span></span> <span> </span><span class="NLM_article-title">The Ubiquitin System</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">425</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.67.1.425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1146%2Fannurev.biochem.67.1.425" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=9759494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADyaK1cXlsFOmsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1998&pages=425-479&issue=1&author=A.+Hershkoauthor=A.+Ciechanover&title=The+Ubiquitin+System&doi=10.1146%2Fannurev.biochem.67.1.425"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin system</span></div><div class="casAuthors">Hershko, Avram; Ciechanover, Aaron</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">425-479</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review, with ∼298 refs.  The selective degrdn. of many short-lived proteins in eukaryotic cells is carried out by the ubiquitin system.  In this pathway, proteins are targeted for degrdn. by covalent ligation to ubiquitin, a highly conserved small protein.  Ubiquitin-mediated degrdn. of regulatory proteins plays important roles in the control of numerous processes, including cell-cycle progression, signal transduction, transcriptional regulation, receptor down-regulation, and endocytosis.  The ubiquitin system has been implicated in the immune response, development, and programmed cell death.  Abnormalities in ubiquitin-mediated processes have been shown to cause pathol. conditions, including malignant transformation.  In this review we discuss recent information on functions and mechanisms of the ubiquitin system.  Since the selectivity of protein degrdn. is detd. mainly at the stage of ligation to ubiquitin, special attention is focused on what we know, and would like to know, about the mode of action of ubiquitin-protein ligation systems and about signals in proteins recognized by these systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqo5VhDx0kSLVg90H21EOLACvtfcHk0lgPb9fJc5KgUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlsFOmsLc%253D&md5=7c3c0a6df9c07baf55b5000eae13ef49</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.67.1.425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.67.1.425%26sid%3Dliteratum%253Aachs%26aulast%3DHershko%26aufirst%3DA.%26aulast%3DCiechanover%26aufirst%3DA.%26atitle%3DThe%2520Ubiquitin%2520System%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D1998%26volume%3D67%26issue%3D1%26spage%3D425%26epage%3D479%26doi%3D10.1146%2Fannurev.biochem.67.1.425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciechanover, A.</span></span> <span> </span><span class="NLM_article-title">Intracellular Protein Degradation: From a Vague Idea Thru the Lysosome and the Ubiquitin-Proteasome System and onto Human Diseases and Drug Targeting</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1178</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1038/sj.cdd.4401692</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fsj.cdd.4401692" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=16094394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnt1Shtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=1178-1190&issue=9&author=A.+Ciechanover&title=Intracellular+Protein+Degradation%3A+From+a+Vague+Idea+Thru+the+Lysosome+and+the+Ubiquitin-Proteasome+System+and+onto+Human+Diseases+and+Drug+Targeting&doi=10.1038%2Fsj.cdd.4401692"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular protein degradation: from a vague idea thru the lysosome and the ubiquitin-proteasome system and onto human diseases and drug targeting</span></div><div class="casAuthors">Ciechanover, A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death and Differentiation</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1178-1190</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Between the 1950s and 1980s, scientists were focusing mostly on how the genetic code is transcribed to RNA and translated to proteins, but how proteins were degraded had remained a neglected research area.  With the discovery of the lysosome by Christian de Duve, it was assumed that cellular proteins were degraded within this organelle.  However, several independent lines of exptl. evidence strongly suggested that intracellular proteolysis was largely non-lysosomal, but the mechanisms involved remained obscure.  The discovery of the ubiquitin-proteasome system resolved the enigma.  It is now recognized that degrdn. of intracellular proteins is involved in the regulation of a broad array of cellular processes, such as the cell cycle and division, regulation of transcription factors, and assurance of the cellular quality control.  Not surprisingly, aberrations in the system have been implicated in the pathogenesis of human disease, such as malignancies and neurodegenerative disorders, which have subsequently led to an increasing effort to develop mechanism-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM8QTfF_nT07Vg90H21EOLACvtfcHk0ljrjEtXfNqbEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnt1Shtro%253D&md5=70d3202030b388727cf3d77d9145810a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fsj.cdd.4401692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.cdd.4401692%26sid%3Dliteratum%253Aachs%26aulast%3DCiechanover%26aufirst%3DA.%26atitle%3DIntracellular%2520Protein%2520Degradation%253A%2520From%2520a%2520Vague%2520Idea%2520Thru%2520the%2520Lysosome%2520and%2520the%2520Ubiquitin-Proteasome%2520System%2520and%2520onto%2520Human%2520Diseases%2520and%2520Drug%2520Targeting%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2005%26volume%3D12%26issue%3D9%26spage%3D1178%26epage%3D1190%26doi%3D10.1038%2Fsj.cdd.4401692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nalepa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery in the Ubiquitin - Proteasome System</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">596</span>– <span class="NLM_lpage">613</span>, <span class="refDoi"> DOI: 10.1038/nrd2056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fnrd2056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=16816840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVagsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=596-613&issue=7&author=G.+Nalepaauthor=M.+Rolfeauthor=J.+W.+Harper&title=Drug+Discovery+in+the+Ubiquitin+-+Proteasome+System&doi=10.1038%2Fnrd2056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery in the ubiquitin-proteasome system</span></div><div class="casAuthors">Nalepa, Grzegorz; Rolfe, Mark; Harper, J. Wade</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">596-613</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Regulated protein turnover via the ubiquitin-proteasome system (UPS) underlies a wide variety of signaling pathways, from cell-cycle control and transcription to development.  Recent evidence that pharmacol. inhibition of the proteasome can be efficacious in the treatment of human cancers has set the stage for attempts to selectively inhibit the activities of disease-specific components of the UPS.  Here, we review recent advances linking UPS components with specific human diseases, most prominently cancer and neurodegenerative disorders, and emphasize potential sites of therapeutic intervention along the regulated protein-degrdn. pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAhoqSVFxu-bVg90H21EOLACvtfcHk0ljrjEtXfNqbEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVagsro%253D&md5=41af26cfd515d10504e2d97f161a9626</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd2056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2056%26sid%3Dliteratum%253Aachs%26aulast%3DNalepa%26aufirst%3DG.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26atitle%3DDrug%2520Discovery%2520in%2520the%2520Ubiquitin%2520-%2520Proteasome%2520System%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26issue%3D7%26spage%3D596%26epage%3D613%26doi%3D10.1038%2Fnrd2056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROteolysis TArgeting Chimeras (PROTACs) — Past, Present and Future</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=PROteolysis+TArgeting+Chimeras+%28PROTACs%29+%E2%80%94+Past%2C+Present+and+Future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eaaNcx4fat3N7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROteolysis%2520TArgeting%2520Chimeras%2520%2528PROTACs%2529%2520%25E2%2580%2594%2520Past%252C%2520Present%2520and%2520Future%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span> <span> </span><span class="NLM_article-title">Inducing Protein-Protein Interactions with Molecular Glues</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">2585</span>– <span class="NLM_lpage">2592</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.04.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.bmcl.2018.04.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=29980357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2585-2592&issue=15&author=Y.+Cheauthor=A.+M.+Gilbertauthor=V.+Shanmugasundaramauthor=M.+C.+Noe&title=Inducing+Protein-Protein+Interactions+with+Molecular+Glues&doi=10.1016%2Fj.bmcl.2018.04.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Inducing protein-protein interactions with molecular glues</span></div><div class="casAuthors">Che, Ye; Gilbert, Adam M.; Shanmugasundaram, Veerabahu; Noe, Mark C.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2585-2592</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The drugable proteome is limited by the no. of functional binding sites that can bind small mols. and respond with a therapeutic effect.  Orthosteric and allosteric modulators of enzyme function or receptor signaling are well-established mechanisms of drug action.  Drugs that perturb protein-protein interactions have only recently been launched.  This approach is more difficult due to the extensive contact surfaces that must be perturbed antagonistically.  Compds. that promote novel protein-protein interactions promise to dramatically expand opportunities for therapeutic intervention.  This approach is precedented with natural products (rapamycin, FK506, sanglifehrin A), synthetic small mols. (thalidomide and IMiD derivs.) and indisulam analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5IlWCdwTjKrVg90H21EOLACvtfcHk0lienRnd2OS78Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjur%252FL&md5=c9fc1cf5e0f60c869a3bb1bf041fa2be</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.04.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.04.046%26sid%3Dliteratum%253Aachs%26aulast%3DChe%26aufirst%3DY.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26atitle%3DInducing%2520Protein-Protein%2520Interactions%2520with%2520Molecular%2520Glues%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26issue%3D15%26spage%3D2585%26epage%3D2592%26doi%3D10.1016%2Fj.bmcl.2018.04.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainolfi, N.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation Mechanisms</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.ddtec.2019.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31200860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=53-60&author=Y.+Zhangauthor=C.+Lohauthor=J.+Chenauthor=N.+Mainolfi&title=Targeted+Protein+Degradation+Mechanisms&doi=10.1016%2Fj.ddtec.2019.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation mechanisms</span></div><div class="casAuthors">Zhang Yi; Loh Christine; Chen Jesse; Mainolfi Nello</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">53-60</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Targeted protein degradation mediated by small molecule degraders represents an exciting new therapeutic opportunity to eliminate disease-causing proteins.  These molecules recruit E3 ubiquitin ligases to the protein of interest and mediate its ubiquitination and subsequent proteolysis by the proteasome.  Significant advancements have been made in the discovery and development of clinically relevant degraders.  In this review we will focus on the recent progress in understanding ternary complex formation and structures, ubiquitination, and other critical factors that govern the efficiency of degraders both in vitro and in vivo.  With deeper knowledges of these areas, the field is building guiding principles to reduce the level of empiricism and to identify therapeutically relevant degraders more rationally and efficiently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRBrI9DTGjC0fW6udTcc2eZ4T0nzZ5-LJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtg%253D%253D&md5=bff167555f8bcab6b672cad3c2946a7b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLoh%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMainolfi%26aufirst%3DN.%26atitle%3DTargeted%2520Protein%2520Degradation%2520Mechanisms%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D53%26epage%3D60%26doi%3D10.1016%2Fj.ddtec.2019.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fink, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span> <span> </span><span class="NLM_article-title">The Novel Mechanism of Lenalidomide Activity</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>126</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">2366</span>– <span class="NLM_lpage">2369</span>, <span class="refDoi"> DOI: 10.1182/blood-2015-07-567958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1182%2Fblood-2015-07-567958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=26438514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvFSrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2015&pages=2366-2369&issue=21&author=E.+C.+Finkauthor=B.+L.+Ebert&title=The+Novel+Mechanism+of+Lenalidomide+Activity&doi=10.1182%2Fblood-2015-07-567958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The novel mechanism of lenalidomide activity</span></div><div class="casAuthors">Fink, Emma C.; Ebert, Benjamin L.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2366-2369</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Lenalidomide acts by a novel drug mechanism-modulation of the substrate specificity of the CRL4CRBN E3 ubiquitin ligase.  In multiple myeloma, lenalidomide induces the ubiquitination of IKZF1 and IKZF3 by CRL4CRBN.  Subsequent proteasomal degrdn. of these transcription factors kills multiple myeloma cells.  In del(5q) myelodysplastic syndrome, lenalidomide induces the degrdn. of CK1α, which preferentially affects del(5q) cells because they express this gene at haploin sufficient levels.  In the future, modulation of ubiquitin ligase function may enable us to target previously ''undruggable'' proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh0d_VTHctuLVg90H21EOLACvtfcHk0lienRnd2OS78Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvFSrtbo%253D&md5=81696cccbf1dda9d96cf970a4bcb10a6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1182%2Fblood-2015-07-567958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2015-07-567958%26sid%3Dliteratum%253Aachs%26aulast%3DFink%26aufirst%3DE.%2BC.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26atitle%3DThe%2520Novel%2520Mechanism%2520of%2520Lenalidomide%2520Activity%26jtitle%3DBlood%26date%3D2015%26volume%3D126%26issue%3D21%26spage%3D2366%26epage%3D2369%26doi%3D10.1182%2Fblood-2015-07-567958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: Chimeric Molecules That Target Proteins to the Skp1-Cullin-F Box Complex for Ubiquitination and Degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&issue=15&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+Chimeric+Molecules+That+Target+Proteins+to+the+Skp1-Cullin-F+Box+Complex+for+Ubiquitination+and+Degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lienRnd2OS78Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520Chimeric%2520Molecules%2520That%2520Target%2520Proteins%2520to%2520the%2520Skp1-Cullin-F%2520Box%2520Complex%2520for%2520Ubiquitination%2520and%2520Degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26issue%3D15%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">First Targeted Protein Degrader Hits the Clinic</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1038/d41573-019-00043-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fd41573-019-00043-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=237-239&author=A.+Mullard&title=First+Targeted+Protein+Degrader+Hits+the+Clinic&doi=10.1038%2Fd41573-019-00043-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fd41573-019-00043-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-019-00043-6%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DFirst%2520Targeted%2520Protein%2520Degrader%2520Hits%2520the%2520Clinic%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D237%26epage%3D239%26doi%3D10.1038%2Fd41573-019-00043-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 Ubiquitin Ligase in Complex with Thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&issue=1&author=E.+S.+Fischerauthor=K.+B%C3%B6hmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thom%C3%A4&title=Structure+of+the+DDB1-CRBN+E3+Ubiquitin+Ligase+in+Complex+with+Thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0liSYd3gqhQIAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DB%25C3%25B6hm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520Ubiquitin%2520Ligase%2520in%2520Complex%2520with%2520Thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26issue%3D1%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&issue=20&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-Guided+Design+and+Optimization+of+Small+Molecules+Targeting+the+Protein-Protein+Interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+Ubiquitin+Ligase+and+the+Hypoxia+Inducible+Factor+%28HIF%29+Alpha+Subunit+with+in+Vitro+Nanomolar+Affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lhEEingJnMhgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-Guided%2520Design%2520and%2520Optimization%2520of%2520Small%2520Molecules%2520Targeting%2520the%2520Protein-Protein%2520Interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520Ubiquitin%2520Ligase%2520and%2520the%2520Hypoxia%2520Inducible%2520Factor%2520%2528HIF%2529%2520Alpha%2520Subunit%2520with%2520in%2520Vitro%2520Nanomolar%2520Affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D20%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&issue=2&author=I.+Churcher&title=Protac-Induced+Protein+Degradation+in+Drug+Discovery%3A+Breaking+the+Rules+or+Just+Making+New+Ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0lhEEingJnMhgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-Induced%2520Protein%2520Degradation%2520in%2520Drug%2520Discovery%253A%2520Breaking%2520the%2520Rules%2520or%2520Just%2520Making%2520New%2520Ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D2%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation: From Chemical Biology to Drug Discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&issue=9&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+Protein+Degradation%3A+From+Chemical+Biology+to+Drug+Discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0lhEEingJnMhgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%253A%2520From%2520Chemical%2520Biology%2520to%2520Drug%2520Discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26issue%3D9%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Röth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulcher, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapkota, G. P.</span></span> <span> </span><span class="NLM_article-title">Advances in Targeted Degradation of Endogenous Proteins</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">2761</span>– <span class="NLM_lpage">2777</span>, <span class="refDoi"> DOI: 10.1007/s00018-019-03112-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1007%2Fs00018-019-03112-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31030225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosFKlur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=2761-2777&issue=14&author=S.+R%C3%B6thauthor=L.+J.+Fulcherauthor=G.+P.+Sapkota&title=Advances+in+Targeted+Degradation+of+Endogenous+Proteins&doi=10.1007%2Fs00018-019-03112-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in targeted degradation of endogenous proteins</span></div><div class="casAuthors">Roth, Sascha; Fulcher, Luke J.; Sapkota, Gopal P.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2761-2777</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">A review.  Protein silencing is often employed as a means to aid investigations in protein function and is increasingly desired as a therapeutic approach.  Several types of protein silencing methodologies have been developed, including targeting the encoding genes, transcripts, the process of translation or the protein directly.  Despite these advances, most silencing systems suffer from limitations.  Silencing protein expression through genetic ablation, for example by CRISPR/Cas9 genome editing, is irreversible, time consuming and not always feasible.  Similarly, RNA interference approaches warrant prolonged treatments, can lead to incomplete protein depletion and are often assocd. with off-target effects.  Targeted proteolysis has the potential to overcome some of these limitations.  The field of targeted proteolysis has witnessed the emergence of many methodologies aimed at targeting specific proteins for degrdn. in a spatio-temporal manner.  In this review, we provide an appraisal of the different targeted proteolytic systems and discuss their applications in understanding protein function, as well as their potential in therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplQoWtY4BlG7Vg90H21EOLACvtfcHk0lhjWRl67SgoLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosFKlur4%253D&md5=e69ce7541082c46b5fd536cf5c93c7dd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs00018-019-03112-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-019-03112-6%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25B6th%26aufirst%3DS.%26aulast%3DFulcher%26aufirst%3DL.%2BJ.%26aulast%3DSapkota%26aufirst%3DG.%2BP.%26atitle%3DAdvances%2520in%2520Targeted%2520Degradation%2520of%2520Endogenous%2520Proteins%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2019%26volume%3D76%26issue%3D14%26spage%3D2761%26epage%3D2777%26doi%3D10.1007%2Fs00018-019-03112-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in Vivo Protein Knockdown by Small-Molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&issue=8&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+Den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+Vivo+Protein+Knockdown+by+Small-Molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lhjWRl67SgoLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3DDen%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520Vivo%2520Protein%2520Knockdown%2520by%2520Small-Molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D8%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuaid, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2017.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=29129716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=67-77&issue=1&author=G.+M.+Burslemauthor=B.+E.+Smithauthor=A.+C.+Laiauthor=S.+Jaime-Figueroaauthor=D.+C.+McQuaidauthor=D.+P.+Bondesonauthor=M.+Toureauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=A.+P.+Crewauthor=J.+Hinesauthor=C.+M.+Crews&title=The+Advantages+of+Targeted+Protein+Degradation+Over+Inhibition%3A+An+RTK+Case+Study&doi=10.1016%2Fj.chembiol.2017.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span></div><div class="casAuthors">Burslem, George M.; Smith, Blake E.; Lai, Ashton C.; Jaime-Figueroa, Saul; McQuaid, Daniel C.; Bondeson, Daniel P.; Toure, Momar; Dong, Hanqing; Qian, Yimin; Wang, Jing; Crew, Andrew P.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-77.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proteolysis targeting chimera (PROTAC) technol. has emerged over the last two decades as a powerful tool for targeted degrdn. of endogenous proteins.  Herein we describe the development of PROTACs for receptor tyrosine kinases, a protein family yet to be targeted for induced protein degrdn.  The use of VHL-recruiting PROTACs against this protein family reveals several advantages of degrdn. over inhibition alone: direct comparisons of fully functional, target-degrading PROTACs with target-inhibiting variants that contain an inactivated E3 ligase-recruiting ligand show that degrdn. leads to more potent inhibition of cell proliferation and a more durable and sustained downstream signaling response, and thus addresses the kinome rewiring challenge seen with many receptor tyrosine kinase inhibitors.  Combined, these findings demonstrate the ability to target receptor tyrosine kinases for degrdn. using the PROTAC technol. and outline the advantages of this degrdn.-based approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4yDhRDWqNv7Vg90H21EOLACvtfcHk0lhr48yBRzadCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM&md5=0ac39a6f5836ea3772a0396356efb630</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DMcQuaid%26aufirst%3DD.%2BC.%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520Advantages%2520of%2520Targeted%2520Protein%2520Degradation%2520Over%2520Inhibition%253A%2520An%2520RTK%2520Case%2520Study%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26issue%3D1%26spage%3D67%26epage%3D77%26doi%3D10.1016%2Fj.chembiol.2017.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span> (<span class="NLM_issue">49</span>),  <span class="NLM_fpage">17019</span>– <span class="NLM_lpage">17026</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=17019-17026&issue=49&author=P.+M.+Crommauthor=K.+T.+G.+Samarasingheauthor=J.+Hinesauthor=C.+M.+Crews&title=Addressing+Kinase-Independent+Functions+of+Fak+via+PROTAC-Mediated+Degradation&doi=10.1021%2Fjacs.8b08008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span></div><div class="casAuthors">Cromm, Philipp M.; Samarasinghe, Kusal T. G.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17019-17026</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic inhibition proved to be a successful modality for the development of many small-mol. drugs.  In recent years, small-mol.-induced protein degrdn. has emerged as an orthogonal therapeutic strategy that has the potential to expand the druggable target space.  Focal adhesion kinase (Fak) is a key player in tumor invasion and metastasis, acting simultaneously as a kinase and a scaffold for several signaling proteins.  While previous efforts to modulate Fak activity were limited to kinase inhibitors with low success in clin. studies, protein degrdn. offers a possibility to simultaneously block Fak's kinase signaling and scaffolding capabilities.  Here, the authors report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clin. candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion.  These results underline the potential that PROTACs offer in expanding the druggable space and controlling protein functions that are not easily addressed by traditional small-mol. therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTn8r4NmDDObVg90H21EOLACvtfcHk0lhr48yBRzadCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF&md5=b97b1a646a8f9dbf7a2bed640d0f001d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAddressing%2520Kinase-Independent%2520Functions%2520of%2520Fak%2520via%2520PROTAC-Mediated%2520Degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26issue%3D49%26spage%3D17019%26epage%3D17026%26doi%3D10.1021%2Fjacs.8b08008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomoshige, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohgane, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of Small Molecules That Induce the Degradation of Huntingtin</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">11530</span>– <span class="NLM_lpage">11533</span>, <span class="refDoi"> DOI: 10.1002/anie.201706529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1002%2Fanie.201706529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSmurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=11530-11533&issue=38&author=S.+Tomoshigeauthor=S.+Nomuraauthor=K.+Ohganeauthor=Y.+Hashimotoauthor=M.+Ishikawa&title=Discovery+of+Small+Molecules+That+Induce+the+Degradation+of+Huntingtin&doi=10.1002%2Fanie.201706529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecules that Induce the Degradation of Huntingtin</span></div><div class="casAuthors">Tomoshige, Shusuke; Nomura, Sayaka; Ohgane, Kenji; Hashimoto, Yuichi; Ishikawa, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11530-11533</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is an autosomal dominant neurodegenerative disorder caused by the aggregation of mutant huntingtin (mHtt), and removal of toxic mHtt is expected to be an effective therapeutic approach.  The authors designed two small hybrid mols. by linking a ligand for ubiquitin ligase (cellular inhibitor of apoptosis protein 1; cIAP1) with probes for mHtt aggregates, anticipating that these compds. would recruit cIAP1 to mHtt and induce selective degrdn. by the ubiquitin-proteasome system.  The synthesized compds. reduced mHtt levels in HD patient fibroblasts and appear to be promising candidates for the development of a treatment for HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSFiHw5mJy2rVg90H21EOLACvtfcHk0lhr48yBRzadCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSmurzL&md5=fb68cd4e5577d195a4a26f42c29428c3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fanie.201706529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201706529%26sid%3Dliteratum%253Aachs%26aulast%3DTomoshige%26aufirst%3DS.%26aulast%3DNomura%26aufirst%3DS.%26aulast%3DOhgane%26aufirst%3DK.%26aulast%3DHashimoto%26aufirst%3DY.%26aulast%3DIshikawa%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520Small%2520Molecules%2520That%2520Induce%2520the%2520Degradation%2520of%2520Huntingtin%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26issue%3D38%26spage%3D11530%26epage%3D11533%26doi%3D10.1002%2Fanie.201706529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">11280</span>– <span class="NLM_lpage">11300</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01530</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01530" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11280-11300&issue=24&author=H.+Zhouauthor=L.+Baiauthor=R.+Xuauthor=Y.+Zhaoauthor=J.+Chenauthor=D.+McEachernauthor=K.+Chinnaswamyauthor=B.+Wenauthor=L.+Daiauthor=P.+Kumarauthor=C.+Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=J.+L.+Meagherauthor=H.+Yiauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-Based+Discovery+of+SD-36+as+a+Potent%2C+Selective%2C+and+Efficacious+PROTAC+Degrader+of+STAT3+Protein&doi=10.1021%2Facs.jmedchem.9b01530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span></div><div class="casAuthors">Zhou, Haibin; Bai, Longchuan; Xu, Renqi; Zhao, Yujun; Chen, Jianyong; McEachern, Donna; Chinnaswamy, Krishnapriya; Wen, Bo; Dai, Lipeng; Kumar, Praveen; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Meagher, Jennifer L.; Yi, Han; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11280-11300</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases.  Despite twenty-years of persistent research efforts, targeting STAT3 has been very challenging.  The authors report herein the structure-based discovery of potent small-mol. STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept.  The authors first designed SI-109 as a potent, small-mol. inhibitor of the STAT3 SH2 domain.  Employing ligands for cereblon/Cullin 4A E3 ligase and SI-109, the authors obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36.  SD-36 induces rapid STAT3 degrdn. at low nanomolar concns. in cells and fails to degrader other STAT proteins.  SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3.  A single dose of SD-36 results in complete STAT3 protein degrdn. in xenograft tumor tissue and normal mouse tissues.  SD-36 achieves complete and long-lasting tumor regression in the Molm-16 xenograft tumor model at well-tolerated dose-schedules.  SD-36 is a potent, selective and efficacious STAT3 degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIAWd1hiuqy7Vg90H21EOLACvtfcHk0lizSC91aTzFRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J&md5=7d0c656a2d96196a3b71829c9efe715a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01530%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-Based%2520Discovery%2520of%2520SD-36%2520as%2520a%2520Potent%252C%2520Selective%252C%2520and%2520Efficacious%2520PROTAC%2520Degrader%2520of%2520STAT3%2520Protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D24%26spage%3D11280%26epage%3D11300%26doi%3D10.1021%2Facs.jmedchem.9b01530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Harnessing the Power of Proteolysis for Targeted Protein Inactivation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>77</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.molcel.2020.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32004468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFCjsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2020&pages=446-460&issue=3&author=R.+Vermaauthor=D.+Mohlauthor=R.+J.+Deshaies&title=Harnessing+the+Power+of+Proteolysis+for+Targeted+Protein+Inactivation&doi=10.1016%2Fj.molcel.2020.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing the Power of Proteolysis for Targeted Protein Inactivation</span></div><div class="casAuthors">Verma, Rati; Mohl, Dane; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">446-460</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Two decades into the twenty-first century, a confluence of breakthrough technologies wielded at the mol. level is presenting biologists with unique opportunities to unravel the complexities of the cellular world.  CRISPR/Cas9 allows gene knock-outs, knock-ins, and single-base editing at chromosomal loci.  RNA-based tools such as siRNA, antisense oligos, and morpholinos can be used to silence expression of specific genes.  Meanwhile, protein knockdown tools that draw inspiration from natural regulatory mechanisms and facilitate elimination of native or degron-tagged proteins from cells are rapidly emerging.  The acute and reversible redn. in protein levels enabled by these methods allows for precise detn. of loss-of-function phenotypes free from secondary effects or compensatory adaptation that can confound nucleic-acid-based methods that involve slow depletion or permanent loss of a protein.  In this Review, we summarize the ingenious ways biologists have exploited natural mechanisms for protein degrdn. to direct the elimination of specific proteins at will.  This has led to advancements not only in basic research but also in the therapeutic space with the introduction of PROTACs into clin. trials for cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-2OyHAw-uMLVg90H21EOLACvtfcHk0lizSC91aTzFRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFCjsr4%253D&md5=8d74ed00d55184be33786d0ee44ad787</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DR.%26aulast%3DMohl%26aufirst%3DD.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DHarnessing%2520the%2520Power%2520of%2520Proteolysis%2520for%2520Targeted%2520Protein%2520Inactivation%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D77%26issue%3D3%26spage%3D446%26epage%3D460%26doi%3D10.1016%2Fj.molcel.2020.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riching, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDougall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span> <span> </span><span class="NLM_article-title">Quantitative Live-Cell Kinetic Degradation and Mechanistic Profiling of PROTAC Mode of Action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2758</span>– <span class="NLM_lpage">2770</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00692</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00692" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFOrtbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2758-2770&issue=9&author=K.+M.+Richingauthor=S.+Mahanauthor=C.+R.+Coronaauthor=M.+McDougallauthor=J.+D.+Vastaauthor=M.+B.+Robersauthor=M.+Urhauthor=D.+L.+Daniels&title=Quantitative+Live-Cell+Kinetic+Degradation+and+Mechanistic+Profiling+of+PROTAC+Mode+of+Action&doi=10.1021%2Facschembio.8b00692"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative Live-Cell Kinetic Degradation and Mechanistic Profiling of PROTAC Mode of Action</span></div><div class="casAuthors">Riching, Kristin M.; Mahan, Sarah; Corona, Cesear R.; McDougall, Mark; Vasta, James D.; Robers, Matthew B.; Urh, Marjeta; Daniels, Danette L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2758-2770</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new generation of heterobifunctional small mols., termed proteolysis targeting chimeras (PROTACs), targets proteins for degrdn. through recruitment to E3 ligases and holds significant therapeutic potential.  Despite numerous successful examples, PROTAC small mol. development remains laborious and unpredictable, involving testing compds. for end-point degrdn. activity at fixed times and concns. without resolving or optimizing for the important biol. steps required for the process.  Given the complexity of the ubiquitin proteasomal pathway, technologies that enable real-time characterization of PROTAC efficacy and mechanism of action are crit. for accelerating compd. development, profiling, and improving guidance of chem. structure-activity relationship.  Here, the authors present an innovative, modular live-cell platform utilizing endogenous tagging technologies and apply it to monitoring PROTAC-mediated degrdn. of the bromodomain and extra-terminal family members.  The authors show comprehensive real-time degrdn. and recovery profiles for each target, precisely quantifying degrdn. rates, maximal levels of degrdn. (Dmax), and time frame at Dmax.  These degrdn. metrics show specific PROTAC and family member-dependent responses that are closely assocd. with the key cellular protein interactions required for the process.  Kinetic studies show cellular ternary complex stability influences potency and degrdn. efficacy.  Meanwhile, the level of ubiquitination is highly correlated to degrdn. rate, indicating ubiquitination stemming from productive ternary complex formation is the main driver of the degrdn. rate.  The approaches applied here highlight the steps at which the choice of E3 ligase handle can elicit different outcomes and discern individual parameters required for degrdn., ultimately enabling chem. design strategies and rank ordering of potential therapeutic compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnkETO-K622bVg90H21EOLACvtfcHk0lizSC91aTzFRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFOrtbbM&md5=9a2a3162dd930ac04622db6990c0d748</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00692%26sid%3Dliteratum%253Aachs%26aulast%3DRiching%26aufirst%3DK.%2BM.%26aulast%3DMahan%26aufirst%3DS.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DMcDougall%26aufirst%3DM.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26atitle%3DQuantitative%2520Live-Cell%2520Kinetic%2520Degradation%2520and%2520Mechanistic%2520Profiling%2520of%2520PROTAC%2520Mode%2520of%2520Action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D9%26spage%3D2758%26epage%3D2770%26doi%3D10.1021%2Facschembio.8b00692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=361-368&issue=3&author=M.+J.+Royauthor=S.+Winklerauthor=S.+J.+Hughesauthor=C.+Whitworthauthor=M.+Galantauthor=W.+Farnabyauthor=K.+Rumpelauthor=A.+Ciulli&title=SPR-Measured+Dissociation+Kinetics+of+PROTAC+Ternary+Complexes+Influence+Target+Degradation+Rate&doi=10.1021%2Facschembio.9b00092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span></div><div class="casAuthors">Roy, Michael J.; Winkler, Sandra; Hughes, Scott J.; Whitworth, Claire; Galant, Michael; Farnaby, William; Rumpel, Klaus; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-368</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bifunctional degrader mols., known as proteolysis-targeting chimeras (PROTACs), function by recruiting a target to an E3 ligase, forming a target/PROTAC/ligase ternary complex.  Despite the importance of this key intermediate species, no detailed validation of a method to directly det. binding parameters for ternary complex kinetics has been reported, and it remains to be addressed whether tuning the kinetics of PROTAC ternary complexes may be an effective strategy to improve the efficiency of targeted protein degrdn.  Here, we develop an SPR-based assay to quantify the stability of PROTAC-induced ternary complexes by measuring for the first time the kinetics of their formation and dissocn. in vitro using purified proteins.  We benchmark our assay using four PROTACs that target the bromodomains (BDs) of bromodomain and extraterminal domain proteins Brd2, Brd3, and Brd4 to the von Hippel-Lindau E3 ligase (VHL).  We reveal marked differences in ternary complex off-rates for different PROTACs that exhibit either pos. or neg. cooperativity for ternary complex formation relative to binary binding.  The pos. cooperative degrader MZ1 forms comparatively stable and long-lived ternary complexes with either Brd4BD2 or Brd2BD2 and VHL.  Equivalent complexes with Brd3BD2 are destabilized due to a single amino acid difference (Glu/Gly swap) present in the bromodomain.  We observe that this difference in ternary complex dissociative half-life correlates to a greater initial rate of intracellular degrdn. of Brd2 and Brd4 relative to Brd3.  These findings establish a novel assay to measure the kinetics of PROTAC ternary complexes and elucidate the important kinetic parameters that drive effective target degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFhvuKYztcbVg90H21EOLACvtfcHk0lgXyqoAKAyXMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D&md5=629d1acbd218b668a2696b727946962b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00092%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSPR-Measured%2520Dissociation%2520Kinetics%2520of%2520PROTAC%2520Ternary%2520Complexes%2520Influence%2520Target%2520Degradation%2520Rate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26issue%3D3%26spage%3D361%26epage%3D368%26doi%3D10.1021%2Facschembio.9b00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ankenbruck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courtney, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Optochemical Control of Biological Processes in Cells and Animals</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2768</span>– <span class="NLM_lpage">2798</span>, <span class="refDoi"> DOI: 10.1002/anie.201700171</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1002%2Fanie.201700171" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFGjur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=2768-2798&issue=11&author=N.+Ankenbruckauthor=T.+Courtneyauthor=Y.+Naroauthor=A.+Deiters&title=Optochemical+Control+of+Biological+Processes+in+Cells+and+Animals&doi=10.1002%2Fanie.201700171"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Optochemical Control of Biological Processes in Cells and Animals</span></div><div class="casAuthors">Ankenbruck, Nicholas; Courtney, Taylor; Naro, Yuta; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2768-2798</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Biol. processes are naturally regulated with high spatial and temporal control, as is perhaps most evident in metazoan embryogenesis.  Chem. tools have been extensively utilized in cell and developmental biol. to investigate cellular processes, and conditional control methods have expanded applications of these technologies toward resolving complex biol. questions.  Light represents an excellent external trigger since it can be controlled with very high spatial and temporal precision.  To this end, several optically regulated tools have been developed and applied to living systems.  In this review we discuss recent developments of optochem. tools, including small mols., peptides, proteins, and nucleic acids that can be irreversibly or reversibly controlled through light irradn., with a focus on applications in cells and animals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDYWdcoWgIZ7Vg90H21EOLACvtfcHk0lgXyqoAKAyXMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFGjur4%253D&md5=2b36ff9c84a2611458386526b16adcc7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201700171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201700171%26sid%3Dliteratum%253Aachs%26aulast%3DAnkenbruck%26aufirst%3DN.%26aulast%3DCourtney%26aufirst%3DT.%26aulast%3DNaro%26aufirst%3DY.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DOptochemical%2520Control%2520of%2520Biological%2520Processes%2520in%2520Cells%2520and%2520Animals%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26issue%3D11%26spage%3D2768%26epage%3D2798%26doi%3D10.1002%2Fanie.201700171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachl, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newham, P.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-Targeting Chimeras in Drug Development: A Safety Perspective</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>177</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1709</span>– <span class="NLM_lpage">1718</span>, <span class="refDoi"> DOI: 10.1111/bph.15014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1111%2Fbph.15014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32022252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjslSms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2020&pages=1709-1718&issue=8&author=K.+Moreauauthor=M.+Coenauthor=A.+X.+Zhangauthor=F.+Pachlauthor=M.+P.+Castaldiauthor=G.+Dahlauthor=H.+Boydauthor=C.+Scottauthor=P.+Newham&title=Proteolysis-Targeting+Chimeras+in+Drug+Development%3A+A+Safety+Perspective&doi=10.1111%2Fbph.15014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-targeting chimeras in drug development: A safety perspective</span></div><div class="casAuthors">Moreau, Kevin; Coen, Muireann; Zhang, Andrew X.; Pachl, Fiona; Castaldi, M. Paola; Dahl, Goran; Boyd, Helen; Scott, Clay; Newham, Pete</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1709-1718</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Proteolysis-targeting chimeras are a new drug modality that exploits the endogenous ubiquitin proteasome system to degrade a protein of interest for therapeutic benefit.  As the first-generation of proteolysis-targeting chimeras have now entered clin. trials for oncol. indications, it is timely to consider the theor. safety risks inherent with this modality which include off-target degrdn., intracellular accumulation of natural substrates for the E3 ligases used in the ubiquitin proteasome system, proteasome satn. by ubiquitinated proteins, and liabilities assocd. with the "hook effect" of proteolysis-targeting chimeras This review describes in vitro and non-clin. in vivo data that provide mechanistic insight of these safety risks and approaches being used to mitigate these risks in the next generation of proteolysis-targeting chimera mols. to extend therapeutic applications beyond life-threatening diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreTK_TldLxHbVg90H21EOLACvtfcHk0lgXyqoAKAyXMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjslSms7o%253D&md5=1e26bbe1d623f2f8f4218c2e3fad9dd6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1111%2Fbph.15014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.15014%26sid%3Dliteratum%253Aachs%26aulast%3DMoreau%26aufirst%3DK.%26aulast%3DCoen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DA.%2BX.%26aulast%3DPachl%26aufirst%3DF.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DBoyd%26aufirst%3DH.%26aulast%3DScott%26aufirst%3DC.%26aulast%3DNewham%26aufirst%3DP.%26atitle%3DProteolysis-Targeting%2520Chimeras%2520in%2520Drug%2520Development%253A%2520A%2520Safety%2520Perspective%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2020%26volume%3D177%26issue%3D8%26spage%3D1709%26epage%3D1718%26doi%3D10.1111%2Fbph.15014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shchepinova, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kounde, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chudasama, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, E. W.</span></span> <span> </span><span class="NLM_article-title">Antibody-PROTAC Conjugates Enable HER2-Dependent Targeted Protein Degradation of BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1306</span>– <span class="NLM_lpage">1312</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00285</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00285" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1306-1312&issue=6&author=M.+Maneiroauthor=N.+Forteauthor=M.+M.+Shchepinovaauthor=C.+S.+Koundeauthor=V.+Chudasamaauthor=J.+R.+Bakerauthor=E.+W.+Tate&title=Antibody-PROTAC+Conjugates+Enable+HER2-Dependent+Targeted+Protein+Degradation+of+BRD4&doi=10.1021%2Facschembio.0c00285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-PROTAC Conjugates Enable HER2-Dependent Targeted Protein Degradation of BRD4</span></div><div class="casAuthors">Maneiro, Maria; Forte, Nafsika; Shchepinova, Maria M.; Kounde, Cyrille S.; Chudasama, Vijay; Baker, James Richard; Tate, Edward W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1306-1312</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting protein degrdn. with Proteolysis-Targeting Chimeras (PROTACs) is an area of great current interest in drug discovery.  Nevertheless, although the high effectiveness of PROTACs against a wide variety of targets has been established, most degraders reported to date display limited intrinsic tissue selectivity and do not discriminate between cells of different types.  Here, we describe a strategy for selective protein degrdn. in a specific cell type.  We report the design and synthesis of a trastuzumab-PROTAC conjugate (Ab-PROTAC 3) in which E3 ligase-directed degrader activity is caged with an antibody linker which can be hydrolyzed following antibody-PROTAC internalization, releasing the active PROTAC and inducing catalytic protein degrdn.  We show that 3 selectively targets bromodomain-contg. protein 4 (BRD4) for degrdn. only in HER2 pos. breast cancer cell lines, while sparing HER2 neg. cells.  Using live cell confocal microscopy, we show internalization and lysosomal trafficking of the conjugate specifically in HER2 pos. cells, leading to the release of active PROTAC in quantities sufficient to induce potent BRD4 degrdn.  These studies demonstrate proof-of-concept for tissue-specific BRD4 degrdn., overcoming limitations of PROTAC selectivity, with significant potential for application to novel targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPwGNLaGmRbbVg90H21EOLACvtfcHk0liFzVbNVSIIzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D&md5=544b53e54d3f0696f0b425c3f6e48e46</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00285%26sid%3Dliteratum%253Aachs%26aulast%3DManeiro%26aufirst%3DM.%26aulast%3DForte%26aufirst%3DN.%26aulast%3DShchepinova%26aufirst%3DM.%2BM.%26aulast%3DKounde%26aufirst%3DC.%2BS.%26aulast%3DChudasama%26aufirst%3DV.%26aulast%3DBaker%26aufirst%3DJ.%2BR.%26aulast%3DTate%26aufirst%3DE.%2BW.%26atitle%3DAntibody-PROTAC%2520Conjugates%2520Enable%2520HER2-Dependent%2520Targeted%2520Protein%2520Degradation%2520of%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26issue%3D6%26spage%3D1306%26epage%3D1312%26doi%3D10.1021%2Facschembio.0c00285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaquiere, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rei Ingalla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis Phillips, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowntree, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertz, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Antibody-Mediated Delivery of Chimeric Protein Degraders Which Target Estrogen Receptor Alpha (ERα)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">126907</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.bmcl.2019.126907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31902710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFWgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=126907&issue=4&author=P.+S.+Dragovichauthor=P.+Adhikariauthor=R.+A.+Blakeauthor=N.+Blaquiereauthor=J.+Chenauthor=Y.+X.+Chengauthor=W.+den+Bestenauthor=J.+Hanauthor=S.+J.+Hartmanauthor=J.+Heauthor=M.+Heauthor=E.+Rei+Ingallaauthor=A.+V.+Kamathauthor=T.+Kleinheinzauthor=T.+Laiauthor=D.+D.+Leipoldauthor=C.+S.+Liauthor=Q.+Liuauthor=J.+Luauthor=Y.+Luauthor=F.+Mengauthor=L.+Mengauthor=C.+Ngauthor=K.+Pengauthor=G.+Lewis+Phillipsauthor=T.+H.+Pillowauthor=R.+K.+Rowntreeauthor=J.+D.+Sadowskyauthor=D.+Sampathauthor=L.+Stabenauthor=S.+T.+Stabenauthor=J.+Waiauthor=K.+Wanauthor=X.+Wangauthor=B.+Weiauthor=I.+E.+Wertzauthor=J.+Xinauthor=K.+Xuauthor=H.+Yaoauthor=R.+Zangauthor=D.+Zhangauthor=H.+Zhouauthor=Y.+Zhao&title=Antibody-Mediated+Delivery+of+Chimeric+Protein+Degraders+Which+Target+Estrogen+Receptor+Alpha+%28ER%CE%B1%29&doi=10.1016%2Fj.bmcl.2019.126907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-mediated delivery of chimeric protein degraders which target estrogen receptor alpha (ERα)</span></div><div class="casAuthors">Dragovich, Peter S.; Adhikari, Pragya; Blake, Robert A.; Blaquiere, Nicole; Chen, Jinhua; Cheng, Yun-Xing; den Besten, Willem; Han, Jinping; Hartman, Steven J.; He, Jintang; He, Mingtao; Rei Ingalla, Ellen; Kamath, Amrita V.; Kleinheinz, Tracy; Lai, Tommy; Leipold, Douglas D.; Li, Chun Sing; Liu, Qi; Lu, Jiawei; Lu, Ying; Meng, Fanwei; Meng, Lingyao; Ng, Carl; Peng, Kaishan; Lewis Phillips, Gail; Pillow, Thomas H.; Rowntree, Rebecca K.; Sadowsky, Jack D.; Sampath, Deepak; Staben, Leanna; Staben, Steven T.; Wai, John; Wan, Kunpeng; Wang, Xinxin; Wei, Bin Qing; Wertz, Ingrid E.; Xin, Jianfeng; Xu, Keyang; Yao, Hui; Zang, Richard; Zhang, Donglu; Zhou, Hao; Zhao, Yongxin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">126907</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Chimeric mols. which effect intracellular degrdn. of target proteins via E3 ligase-mediated ubiquitination (e.g., PROTACs) are currently of high interest in medicinal chem.  However, these entities are relatively large compds. that often possess mol. characteristics which may compromise oral bioavailability, soly., and/or in vivo pharmacokinetic properties.  Accordingly, we explored whether conjugation of chimeric degraders to monoclonal antibodies using technologies originally developed for cytotoxic payloads might provide alternate delivery options for these novel agents.  In this report we describe the construction of several degrader-antibody conjugates comprised of two distinct ERα-targeting degrader entities and three independent ADC linker modalities.  We subsequently demonstrate the antigen-dependent delivery to MCF7-neo/HER2 cells of the degrader payloads that are incorporated into these conjugates.  We also provide evidence for efficient intracellular degrader release from one of the employed linkers.  In addn., preliminary data are described which suggest that reasonably favorable in vivo stability properties are assocd. with the linkers utilized to construct the degrader conjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdPWcGkXC85LVg90H21EOLACvtfcHk0liFzVbNVSIIzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFWgtw%253D%253D&md5=5f196b194df7f4622825c4b901130f7c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126907%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DAdhikari%26aufirst%3DP.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DBlaquiere%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DY.%2BX.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DHartman%26aufirst%3DS.%2BJ.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DRei%2BIngalla%26aufirst%3DE.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLeipold%26aufirst%3DD.%2BD.%26aulast%3DLi%26aufirst%3DC.%2BS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DPeng%26aufirst%3DK.%26aulast%3DLewis%2BPhillips%26aufirst%3DG.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DRowntree%26aufirst%3DR.%2BK.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DStaben%26aufirst%3DL.%26aulast%3DStaben%26aufirst%3DS.%2BT.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWan%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26aulast%3DXin%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DZang%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DAntibody-Mediated%2520Delivery%2520of%2520Chimeric%2520Protein%2520Degraders%2520Which%2520Target%2520Estrogen%2520Receptor%2520Alpha%2520%2528ER%25CE%25B1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26issue%3D4%26spage%3D126907%26doi%3D10.1016%2Fj.bmcl.2019.126907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonger, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakhit, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payumo, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wandless, T. J.</span></span> <span> </span><span class="NLM_article-title">General Method for Regulating Protein Stability with Light</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1021/cb400755b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400755b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWiu7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=111-115&issue=1&author=K.+M.+Bongerauthor=R.+Rakhitauthor=A.+Y.+Payumoauthor=J.+K.+Chenauthor=T.+J.+Wandless&title=General+Method+for+Regulating+Protein+Stability+with+Light&doi=10.1021%2Fcb400755b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">General Method for Regulating Protein Stability with Light</span></div><div class="casAuthors">Bonger, Kimberly M.; Rakhit, Rishi; Payumo, Alexander Y.; Chen, James K.; Wandless, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111-115</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational regulation of protein abundance in cells is a powerful tool for studying protein function.  Here, we describe a novel genetically encoded protein domain that is degraded upon exposure to nontoxic blue light.  We demonstrate that fusion proteins contg. this domain are rapidly degraded in cultured cells and in zebrafish upon illumination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplL0DRj0csdLVg90H21EOLACvtfcHk0liFzVbNVSIIzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWiu7rO&md5=4a69db08c6128a6b0e48d53e0a6e99f4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fcb400755b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400755b%26sid%3Dliteratum%253Aachs%26aulast%3DBonger%26aufirst%3DK.%2BM.%26aulast%3DRakhit%26aufirst%3DR.%26aulast%3DPayumo%26aufirst%3DA.%2BY.%26aulast%3DChen%26aufirst%3DJ.%2BK.%26aulast%3DWandless%26aufirst%3DT.%2BJ.%26atitle%3DGeneral%2520Method%2520for%2520Regulating%2520Protein%2520Stability%2520with%2520Light%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26issue%3D1%26spage%3D111%26epage%3D115%26doi%3D10.1021%2Fcb400755b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Renicke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usherenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essen, L. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taxis, C.</span></span> <span> </span><span class="NLM_article-title">A LOV2 Domain-Based Optogenetic Tool to Control Protein Degradation and Cellular Function</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2013.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=23601651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1Cmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=619-626&issue=4&author=C.+Renickeauthor=D.+Schusterauthor=S.+Usherenkoauthor=L.+O.+Essenauthor=C.+Taxis&title=A+LOV2+Domain-Based+Optogenetic+Tool+to+Control+Protein+Degradation+and+Cellular+Function&doi=10.1016%2Fj.chembiol.2013.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A LOV2 Domain-Based Optogenetic Tool to Control Protein Degradation and Cellular Function</span></div><div class="casAuthors">Renicke, Christian; Schuster, Daniel; Usherenko, Svetlana; Essen, Lars-Oliver; Taxis, Christof</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">619-626</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Light perception is indispensable for plants to respond adequately to external cues and is linked to proteolysis of key transcriptional regulators.  To provide synthetic light control of protein stability, we developed a generic photosensitive degron (psd) module combining the light-reactive LOV2 domain of Arabidopsis thaliana phot1 with the murine ornithine decarboxylase-like degrdn. sequence cODC1.  Functionality of the psd module was demonstrated in the model organism Saccharomyces cerevisiae.  Generation of conditional mutants, light regulation of cyclin-dependent kinase activity, light-based patterning of cell growth, and yeast photog. exemplified its versatility.  In silico modeling of psd module behavior increased understanding of its characteristics.  This engineered degron module transfers the principle of light-regulated degrdn. to nonplant organisms.  It will be highly beneficial to control protein levels in biotechnol. or biomedical applications and offers the potential to render a plethora of biol. processes light-switchable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXrMBGmwMVSrVg90H21EOLACvtfcHk0lhZhxAI4PsDcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1Cmt7k%253D&md5=5e9dbad466b44431f060b30a83a904cd</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DRenicke%26aufirst%3DC.%26aulast%3DSchuster%26aufirst%3DD.%26aulast%3DUsherenko%26aufirst%3DS.%26aulast%3DEssen%26aufirst%3DL.%2BO.%26aulast%3DTaxis%26aufirst%3DC.%26atitle%3DA%2520LOV2%2520Domain-Based%2520Optogenetic%2520Tool%2520to%2520Control%2520Protein%2520Degradation%2520and%2520Cellular%2520Function%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26issue%3D4%26spage%3D619%26epage%3D626%26doi%3D10.1016%2Fj.chembiol.2013.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delacour, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plamont, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billon-Denis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aujard, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Saux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jullien, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautier, A.</span></span> <span> </span><span class="NLM_article-title">Light-Activated Proteolysis for the Spatiotemporal Control of Proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1643</span>– <span class="NLM_lpage">1647</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00069</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00069" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1Ght7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1643-1647&issue=7&author=Q.+Delacourauthor=C.+Liauthor=M.+A.+Plamontauthor=E.+Billon-Denisauthor=I.+Aujardauthor=T.+Le+Sauxauthor=L.+Jullienauthor=A.+Gautier&title=Light-Activated+Proteolysis+for+the+Spatiotemporal+Control+of+Proteins&doi=10.1021%2Facschembio.5b00069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Light-Activated Proteolysis for the Spatiotemporal Control of Proteins</span></div><div class="casAuthors">Delacour, Quentin; Li, Chenge; Plamont, Marie-Aude; Billon-Denis, Emmanuelle; Aujard, Isabelle; Le Saux, Thomas; Jullien, Ludovic; Gautier, Arnaud</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1643-1647</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The regulation of proteolysis is an efficient way to control protein function in cells.  Here, we present a general strategy enabling to increase the spatiotemporal resoln. of conditional proteolysis by using light activation as trigger.  Our approach relies on the auxin-inducible degrdn. system obtained by transposing components of the plant auxin-dependent degrdn. pathway in mammalian cells.  We developed a photoactivatable auxin that acts as a photoactivatable inducer of degrdn.  Upon local and short light illumination, auxin is released in cells and triggers the degrdn. of a protein of interest with spatiotemporal control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorsagFMqI1b7Vg90H21EOLACvtfcHk0lhZhxAI4PsDcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1Ght7s%253D&md5=5da59aa63d7b7365d8a53d929070e9cc</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00069%26sid%3Dliteratum%253Aachs%26aulast%3DDelacour%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DPlamont%26aufirst%3DM.%2BA.%26aulast%3DBillon-Denis%26aufirst%3DE.%26aulast%3DAujard%26aufirst%3DI.%26aulast%3DLe%2BSaux%26aufirst%3DT.%26aulast%3DJullien%26aufirst%3DL.%26aulast%3DGautier%26aufirst%3DA.%26atitle%3DLight-Activated%2520Proteolysis%2520for%2520the%2520Spatiotemporal%2520Control%2520of%2520Proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D7%26spage%3D1643%26epage%3D1647%26doi%3D10.1021%2Facschembio.5b00069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xue, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z.</span></span> <span> </span><span class="NLM_article-title">Light-Induced Protein Degradation with Photocaged PROTACs</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span> (<span class="NLM_issue">46</span>),  <span class="NLM_fpage">18370</span>– <span class="NLM_lpage">18374</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b06422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b06422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVOmurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=18370-18374&issue=46&author=G.+Xueauthor=K.+Wangauthor=D.+Zhouauthor=H.+Zhongauthor=Z.+Pan&title=Light-Induced+Protein+Degradation+with+Photocaged+PROTACs&doi=10.1021%2Fjacs.9b06422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Light-Induced Protein Degradation with Photocaged PROTACs</span></div><div class="casAuthors">Xue, Gang; Wang, Kun; Zhou, Danli; Zhong, Hanbing; Pan, Zhengying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">18370-18374</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Induction of protein degrdn. is emerging as a powerful strategy to modulate protein functions and alter cellular signaling pathways.  Proteolysis-targeting chimeras (PROTACs) have been used to degrade a range of diverse proteins in vitro and in vivo.  Here we present a type of photo-caged PROTACs (pc-PROTACs) to induce degrdn. activity with light.  Photo-removable blocking groups were added to a degrader of Brd4, and the resulting mol. pc-PROTAC1 showed potent degrdn. activity in live cells only after light irradn.  Furthermore, this mol. efficiently degraded Brd4 and induced expected phenotypic changes in zebrafish.  Addnl., this approach was successfully applied to construct pc-PROTAC3 of BTK.  Thus, a general strategy to induce protein degrdn. with light was established to augment the chemists' toolbox to study disease-relevant protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYGnlMSsz2ubVg90H21EOLACvtfcHk0lhZhxAI4PsDcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVOmurvJ&md5=75920669e7228755cfefb5650e4af6b0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b06422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b06422%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DZ.%26atitle%3DLight-Induced%2520Protein%2520Degradation%2520with%2520Photocaged%2520PROTACs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26issue%3D46%26spage%3D18370%26epage%3D18374%26doi%3D10.1021%2Fjacs.9b06422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darrah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Optical Control of Small Molecule-Induced Protein Degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">2193</span>– <span class="NLM_lpage">2197</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b12718</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b12718" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslOhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=2193-2197&issue=5&author=Y.+Naroauthor=K.+Darrahauthor=A.+Deiters&title=Optical+Control+of+Small+Molecule-Induced+Protein+Degradation&doi=10.1021%2Fjacs.9b12718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Optical Control of Small Molecule-Induced Protein Degradation</span></div><div class="casAuthors">Naro, Yuta; Darrah, Kristie; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2193-2197</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As an emerging approach to protein perturbation, small mol.-induced protein degrdn. has gained significant attention as both a chem. tool and a potential therapeutic.  To enable discrete control over its function, we have developed a broadly applicable approach for the optical activation of small mol.-induced protein degrdn.  By installing two different photolabile protecting groups, so-called caging groups, onto two different ligands recruiting Von Hippel-Lindau (VHL) and cereblon (CRBN) E3 ubiquitin ligases, our strategy enables light-triggered protein degrdn. for any small mol. warhead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7kp9dSOD7EbVg90H21EOLACvtfcHk0li_-eJTNt01VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslOhsw%253D%253D&md5=ea8a4bfe78a8acd5fcb8d0bba85b2c02</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b12718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b12718%26sid%3Dliteratum%253Aachs%26aulast%3DNaro%26aufirst%3DY.%26aulast%3DDarrah%26aufirst%3DK.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DOptical%2520Control%2520of%2520Small%2520Molecule-Induced%2520Protein%2520Degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26issue%3D5%26spage%3D2193%26epage%3D2197%26doi%3D10.1021%2Fjacs.9b12718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kounde, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shchepinova, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muelbaier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rackham, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, E. W.</span></span> <span> </span><span class="NLM_article-title">A Caged E3 Ligase Ligand for PROTAC-Mediated Protein Degradation with Light</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">5532</span>– <span class="NLM_lpage">5535</span>, <span class="refDoi"> DOI: 10.1039/D0CC00523A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1039%2FD0CC00523A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32297626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmt1KitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2020&pages=5532-5535&issue=41&author=C.+S.+Koundeauthor=M.+M.+Shchepinovaauthor=C.+N.+Saundersauthor=M.+Muelbaierauthor=M.+D.+Rackhamauthor=J.+D.+Harlingauthor=E.+W.+Tate&title=A+Caged+E3+Ligase+Ligand+for+PROTAC-Mediated+Protein+Degradation+with+Light&doi=10.1039%2FD0CC00523A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A caged E3 ligase ligand for PROTAC-mediated protein degradation with light</span></div><div class="casAuthors">Kounde, Cyrille S.; Shchepinova, Maria M.; Saunders, Charlie N.; Muelbaier, Marcel; Rackham, Mark D.; Harling, John D.; Tate, Edward W.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">5532-5535</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">With the intent of achieving greater spatiotemporal control of PROTAC-induced protein degrdn., a light-activated degrader was designed by photocaging an essential E3 ligase binding motif in a BRD4 targeting PROTAC.  Proteolysis was triggered only after a short irradn. time, the kinetics of which could be monitored by live-cell video microscopy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8qIQehviok7Vg90H21EOLACvtfcHk0li_-eJTNt01VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmt1KitrY%253D&md5=dc4bdede41675bf7f451a71e8aa4ee14</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1039%2FD0CC00523A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FD0CC00523A%26sid%3Dliteratum%253Aachs%26aulast%3DKounde%26aufirst%3DC.%2BS.%26aulast%3DShchepinova%26aufirst%3DM.%2BM.%26aulast%3DSaunders%26aufirst%3DC.%2BN.%26aulast%3DMuelbaier%26aufirst%3DM.%26aulast%3DRackham%26aufirst%3DM.%2BD.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DTate%26aufirst%3DE.%2BW.%26atitle%3DA%2520Caged%2520E3%2520Ligase%2520Ligand%2520for%2520PROTAC-Mediated%2520Protein%2520Degradation%2520with%2520Light%26jtitle%3DChem.%2520Commun.%26date%3D2020%26volume%3D56%26issue%3D41%26spage%3D5532%26epage%3D5535%26doi%3D10.1039%2FD0CC00523A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span> <span> </span><span class="NLM_article-title">Light-Induced Control of Protein Destruction by Opto-PROTAC</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">eaay5154</span>, <span class="refDoi"> DOI: 10.1126/sciadv.aay5154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1126%2Fsciadv.aay5154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32128407" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=eaay5154&issue=8&author=J.+Liuauthor=H.+Chenauthor=L.+Maauthor=Z.+Heauthor=D.+Wangauthor=Y.+Liuauthor=Q.+Linauthor=T.+Zhangauthor=N.+Grayauthor=H.+%C3%9C.+Kaniskanauthor=J.+Jinauthor=W.+Wei&title=Light-Induced+Control+of+Protein+Destruction+by+Opto-PROTAC&doi=10.1126%2Fsciadv.aay5154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aay5154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aay5154%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DW.%26atitle%3DLight-Induced%2520Control%2520of%2520Protein%2520Destruction%2520by%2520Opto-PROTAC%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26issue%3D8%26spage%3Deaay5154%26doi%3D10.1126%2Fsciadv.aay5154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pfaff, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carreira, E. M.</span></span> <span> </span><span class="NLM_article-title">Reversible Spatiotemporal Control of Induced Protein Degradation by Bistable PhotoPROTACs</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1682</span>– <span class="NLM_lpage">1690</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.9b00713</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.9b00713" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslOmsb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=1682-1690&issue=10&author=P.+Pfaffauthor=K.+T.+G.+Samarasingheauthor=C.+M.+Crewsauthor=E.+M.+Carreira&title=Reversible+Spatiotemporal+Control+of+Induced+Protein+Degradation+by+Bistable+PhotoPROTACs&doi=10.1021%2Facscentsci.9b00713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Spatiotemporal Control of Induced Protein Degradation by Bistable PhotoPROTACs</span></div><div class="casAuthors">Pfaff, Patrick; Samarasinghe, Kusal T. G.; Crews, Craig M.; Carreira, Erick M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1682-1690</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Off-tissue effects are persistent issues of modern inhibition-based therapies.  By merging the strategies of photopharmacol. and small-mol. degraders, we introduce a novel concept for persistent spatiotemporal control of induced protein degrdn. that potentially prevents off-tissue toxicity.  Building on the successful principle of bifunctional all-small-mol. Proteolysis Targeting Chimeras (PROTACs), we designed photoswitchable PROTACs (photoPROTACs) by including ortho-F4-azobenzene linkers between both warhead ligands.  This highly bistable yet photoswitchable structural component leads to reversible control over the topol. distance between both ligands.  The azo-cis-isomer is obsd. to be inactive because the distance defined by the linker is prohibitively short to permit complex formation between the protein binding partners.  By contrast, the azo-trans-isomer is active since it can engage both protein partners to form the necessary and productive ternary complex.  Importantly, due to the bistable nature of the ortho-F4-azobenzene moiety employed, the photostationary state of the photoPROTAC is persistent, with no need for continuous irradn.  This technique offers reversible on/off switching of protein degrdn. that is compatible with an intracellular environment and, therefore, could be useful in exptl. exploration of biol. signaling pathways-such as those crucial for oncogenic signal transduction.  Addnl., this strategy may be suitable for therapeutic intervention to address a variety of diseases.  By enabling reversible activation and deactivation of protein degrdn., photoPROTACs offer advantages over conventional photocaging strategies that irreversibly release active agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKaC13qAq9JrVg90H21EOLACvtfcHk0lgp-WQEi-VgGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslOmsb%252FP&md5=13d737b69314b763f9dc75e231b7fe70</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.9b00713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.9b00713%26sid%3Dliteratum%253Aachs%26aulast%3DPfaff%26aufirst%3DP.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DCarreira%26aufirst%3DE.%2BM.%26atitle%3DReversible%2520Spatiotemporal%2520Control%2520of%2520Induced%2520Protein%2520Degradation%2520by%2520Bistable%2520PhotoPROTACs%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2019%26volume%3D5%26issue%3D10%26spage%3D1682%26epage%3D1690%26doi%3D10.1021%2Facscentsci.9b00713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reynders, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuura, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bérouti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simoneschi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauner, D.</span></span> <span> </span><span class="NLM_article-title">PHOTACs Enable Optical Control of Protein Degradation</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">8</span>), <span class="NLM_elocation-id">eaay5064</span> <span class="refDoi"> DOI: 10.1126/sciadv.aay5064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1126%2Fsciadv.aay5064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32128406" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&issue=8&author=M.+Reyndersauthor=B.+S.+Matsuuraauthor=M.+B%C3%A9routiauthor=D.+Simoneschiauthor=A.+Marzioauthor=M.+Paganoauthor=D.+Trauner&title=PHOTACs+Enable+Optical+Control+of+Protein+Degradation&doi=10.1126%2Fsciadv.aay5064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aay5064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aay5064%26sid%3Dliteratum%253Aachs%26aulast%3DReynders%26aufirst%3DM.%26aulast%3DMatsuura%26aufirst%3DB.%2BS.%26aulast%3DB%25C3%25A9routi%26aufirst%3DM.%26aulast%3DSimoneschi%26aufirst%3DD.%26aulast%3DMarzio%26aufirst%3DA.%26aulast%3DPagano%26aufirst%3DM.%26aulast%3DTrauner%26aufirst%3DD.%26atitle%3DPHOTACs%2520Enable%2520Optical%2520Control%2520of%2520Protein%2520Degradation%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26issue%3D8%26doi%3D10.1126%2Fsciadv.aay5064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, W. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Azo-Protac: Novel Light-Controlled Small-Molecule Tool for Protein Knockdown</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4644</span>– <span class="NLM_lpage">4654</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b02058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b02058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFaju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4644-4654&issue=9&author=Y.+H.+Jinauthor=M.+C.+Luauthor=Y.+Wangauthor=W.+X.+Shanauthor=X.+Y.+Wangauthor=Q.+D.+Youauthor=Z.+Y.+Jiang&title=Azo-Protac%3A+Novel+Light-Controlled+Small-Molecule+Tool+for+Protein+Knockdown&doi=10.1021%2Facs.jmedchem.9b02058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Azo-PROTAC: Novel Light-Controlled Small-Molecule Tool for Protein Knockdown</span></div><div class="casAuthors">Jin, Yu-Hui; Lu, Meng-Chen; Wang, Yan; Shan, Wen-Xin; Wang, Xuan-Yu; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4644-4654</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Reversibly altering endogenous protein levels are persistent issues.  Herein, we designed photoswitchable azobenzene-proteolysis targeting chimeras (Azo-PROTACs) by including azobenzene moieties between ligands for the E3 ligase and the protein of interest.  Azo-PROTACs are light-controlled small-mol. tools for protein knockdown in cells.  The light-induced configuration change can switch the active state to induce protein degrdn. activity, which can be reversely controlled by light exposure in intact cells.  We compared the protein degrdn. abilities of Azo-PROTACs with different configurations and linker lengths.  Using the stable form with the best degrdn. ability against the BCR-ABL fusion and ABL proteins in myelogenous leukemia K562 cells, we showed that Azo-PROTAC combines the potent protein knockdown and facile cell uptake properties of the small-mol. PROTAC with a reversible photoswitchability, offering a promising chem. knockdown strategy based on the light-induced reversible on/off properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp1TuuF8Q3g7Vg90H21EOLACvtfcHk0lgp-WQEi-VgGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFaju7Y%253D&md5=0850ca7c222c5837b425b06487993563</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b02058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b02058%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DY.%2BH.%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DShan%26aufirst%3DW.%2BX.%26aulast%3DWang%26aufirst%3DX.%2BY.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26atitle%3DAzo-Protac%253A%2520Novel%2520Light-Controlled%2520Small-Molecule%2520Tool%2520for%2520Protein%2520Knockdown%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D9%26spage%3D4644%26epage%3D4654%26doi%3D10.1021%2Facs.jmedchem.9b02058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manna, D.</span></span> <span> </span><span class="NLM_article-title">Controlling PROTACs With Light</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1258</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1002/cmdc.202000249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1002%2Fcmdc.202000249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32558301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1yqsbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1258-1261&author=S.+Vermaauthor=D.+Manna&title=Controlling+PROTACs+With+Light&doi=10.1002%2Fcmdc.202000249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Controlling PROTACs with Light</span></div><div class="casAuthors">Verma, Shreya; Manna, Debasish</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1258-1261</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras, PROTACs, are emerging as a powerful strategy for exerting exogenous control over protein levels, allowing small mols. to exploit the ubiquitin-proteasome pathway for targeted protein degrdn.  This highlight focuses on the fusion of photochem. with these bifunctional compds., which has provided a novel pathway for spatiotemporally tuning the activation of PROTACs in the form of their photocaged and photoswitchable versions.  Photocaged PROTACs consist of a hindered optolabile group that detaches only upon irradn. at a specific wavelength, releasing the active PROTAC.  These modified PROTACs are inactive in the dark.  Photoswitchable PROTACs are photoisomerizable mols. with azobenzene linkages that are active in either the cis or trans form and inactive in the other.  The isomers interconvert upon irradn. with an appropriate wavelength of light and relax to the thermodynamically stable isomer in the dark or with another wavelength of light.  Although photocaged PROTACs only permit activation control for protein degrdn., photoswitching PROTACs offer reversible activation and deactivation by using suitable wavelengths of light.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9qDTGxuQEFLVg90H21EOLACvtfcHk0lgp-WQEi-VgGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1yqsbrL&md5=560aa3946270ddeca6ea34692a41a7e8</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.202000249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.202000249%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DS.%26aulast%3DManna%26aufirst%3DD.%26atitle%3DControlling%2520PROTACs%2520With%2520Light%26jtitle%3DChemMedChem%26date%3D2020%26volume%3D15%26spage%3D1258%26epage%3D1261%26doi%3D10.1002%2Fcmdc.202000249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nalawansha, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROTACs: An Emerging Therapeutic Modality in Precision Medicine</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">998</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.07.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2020.07.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32795419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1Wrs7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=998-1014&author=D.+A.+Nalawanshaauthor=C.+M.+Crews&title=PROTACs%3A+An+Emerging+Therapeutic+Modality+in+Precision+Medicine&doi=10.1016%2Fj.chembiol.2020.07.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: An Emerging Therapeutic Modality in Precision Medicine</span></div><div class="casAuthors">Nalawansha, Dhanusha A.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">998-1014</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.,.  Targeted protein degrdn. (TPD) has emerged as an exciting new era in chem. biol. and drug discovery.  PROteolysis TArgeting Chimera (PROTAC) technol. targets cellular proteins for degrdn. by co-opting the ubiquitin-proteasome system.  Over the last 5 years, numerous studies have expanded our understanding of the unique mode of action and advantages of PROTACs, which has in turn spurred interest in both academia and industry to explore PROTACs as a novel therapeutic strategy.  In this review, we first highlight the key advantages of PROTACs and then discuss the spatiotemporal regulation of protein degrdn.  Next, we explore current chem. tractable E3 ligases focusing on expanding the existing repertoire with novel E3 ligases to uncover the full potential of TPD.  Collectively, these studies are guiding the development of the PROTAC technol. as it emerges as a new modality in precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4GkJzvvYnp7Vg90H21EOLACvtfcHk0lhF4nuzEuliRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Wrs7vM&md5=63c4eb78ee4fc9272b9ae2409889861e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.07.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.07.020%26sid%3Dliteratum%253Aachs%26aulast%3DNalawansha%26aufirst%3DD.%2BA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROTACs%253A%2520An%2520Emerging%2520Therapeutic%2520Modality%2520in%2520Precision%2520Medicine%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D998%26epage%3D1014%26doi%3D10.1016%2Fj.chembiol.2020.07.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckel, A.</span></span> <span> </span><span class="NLM_article-title">Biologically Active Molecules with a “Light Switch”</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">4900</span>– <span class="NLM_lpage">4921</span>, <span class="refDoi"> DOI: 10.1002/anie.200600387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1002%2Fanie.200600387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvVaqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=4900-4921&issue=30&author=G.+Mayerauthor=A.+Heckel&title=Biologically+Active+Molecules+with+a+%E2%80%9CLight+Switch%E2%80%9D&doi=10.1002%2Fanie.200600387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Biologically active molecules with a "Light switch"</span></div><div class="casAuthors">Mayer, Guenter; Heckel, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">4900-4921</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Biol. active compds. which are light-responsive offer exptl. possibilities which are otherwise very difficult to achieve.  Since light can be manipulated very precisely, for example, with lasers and microscopes rapid jumps in concn. of the active form of mols. are possible with exact control of the area, time, and dosage.  The development of such strategies started in the 1970s.  This review summarizes new developments of the last five years and deals with "small mols.", proteins, and nucleic acids which can either be irreversibly activated with light (these compds. are referred to as "caged compds.") or reversibly switched between an active and an inactive state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA75s7az7307Vg90H21EOLACvtfcHk0lhF4nuzEuliRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvVaqsrg%253D&md5=bf95a5ccf1a6d75509b6d2ae5f75e0c8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fanie.200600387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200600387%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DHeckel%26aufirst%3DA.%26atitle%3DBiologically%2520Active%2520Molecules%2520with%2520a%2520%25E2%2580%259CLight%2520Switch%25E2%2580%259D%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26issue%3D30%26spage%3D4900%26epage%3D4921%26doi%3D10.1002%2Fanie.200600387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faust, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstock, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Bruton Tyrosine Kinase Degradation as a Therapeutic Strategy for Cancer</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>133</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">952</span>– <span class="NLM_lpage">961</span>, <span class="refDoi"> DOI: 10.1182/blood-2018-07-862953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1182%2Fblood-2018-07-862953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=30545835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXotlGjtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2019&pages=952-961&issue=9&author=D.+Dobrovolskyauthor=E.+S.+Wangauthor=S.+Morrowauthor=C.+Leahyauthor=T.+Faustauthor=R.+P.+Nowakauthor=K.+A.+Donovanauthor=G.+Yangauthor=Z.+Liauthor=E.+S.+Fischerauthor=S.+P.+Treonauthor=D.+M.+Weinstockauthor=N.+S.+Gray&title=Bruton+Tyrosine+Kinase+Degradation+as+a+Therapeutic+Strategy+for+Cancer&doi=10.1182%2Fblood-2018-07-862953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Bruton tyrosine kinase degradation as a therapeutic strategy for cancer</span></div><div class="casAuthors">Dobrovolsky, Dennis; Wang, Eric S.; Morrow, Sara; Leahy, Catharine; Faust, Tyler; Nowak, Radoslaw P.; Donovan, Katherine A.; Yang, Guang; Li, Zhengnian; Fischer, Eric S.; Treon, Steven P.; Weinstock, David M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">952-961</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The covalent Bruton tyrosine kinase (BTK) inhibitor ibrutinib is highly efficacious against multiple B-cell malignancies.  However, it is not selective for BTK, and multiple mechanisms of resistance, including the C481S-BTK mutation, can compromise its efficacy.  We hypothesized that small-mol.-induced BTK degrdn. may overcome some of the limitations of traditional enzymic inhibitors.  Here, we demonstrate that BTK degrdn. results in potent suppression of signaling and proliferation in cancer cells and that BTK degraders efficiently degrade C481S-BTK.  Moreover, we discovered DD-03-171, an optimized lead compd. that exhibits enhanced antiproliferative effects on mantle cell lymphoma (MCL) cells in vitro by degrading BTK, IKFZ1, and IKFZ3 as well as efficacy against patient-derived xenografts in vivo.  Thus, "triple degrdn." may be an effective therapeutic approach for treating MCL and overcoming ibrutinib resistance, thereby addressing a major unmet need in the treatment of MCL and other B-cell lymphomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN4GLppunGArVg90H21EOLACvtfcHk0lhF4nuzEuliRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXotlGjtrY%253D&md5=ce7e48897378c2821d728daf6d9935d5</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1182%2Fblood-2018-07-862953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2018-07-862953%26sid%3Dliteratum%253Aachs%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DMorrow%26aufirst%3DS.%26aulast%3DLeahy%26aufirst%3DC.%26aulast%3DFaust%26aufirst%3DT.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DWeinstock%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DBruton%2520Tyrosine%2520Kinase%2520Degradation%2520as%2520a%2520Therapeutic%2520Strategy%2520for%2520Cancer%26jtitle%3DBlood%26date%3D2019%26volume%3D133%26issue%3D9%26spage%3D952%26epage%3D961%26doi%3D10.1182%2Fblood-2018-07-862953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide Conjugation as a Strategy for in Vivo Target Protein Degradation</span>. <i>Science (Washington, DC, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span> (<span class="NLM_issue">6241</span>),  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&issue=6241&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+Conjugation+as+a+Strategy+for+in+Vivo+Target+Protein+Degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lg0-smI71aMsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520Conjugation%2520as%2520a%2520Strategy%2520for%2520in%2520Vivo%2520Target%2520Protein%2520Degradation%26jtitle%3DScience%2520%2528Washington%252C%2520DC%252C%2520U.%2520S.%2529%26date%3D2015%26volume%3D348%26issue%3D6241%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parapatics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecker, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00969</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00969" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=553-560&issue=3&author=M.+Ishoeyauthor=S.+Chornauthor=N.+Singhauthor=M.+G.+Jaegerauthor=M.+Brandauthor=J.+Paulkauthor=S.+Bauerauthor=M.+A.+Erbauthor=K.+Parapaticsauthor=A.+C.+M%C3%BCllerauthor=K.+L.+Bennettauthor=G.+F.+Eckerauthor=J.+E.+Bradnerauthor=G.+E.+Winter&title=Translation+Termination+Factor+GSPT1+Is+a+Phenotypically+Relevant+Off-Target+of+Heterobifunctional+Phthalimide+Degraders&doi=10.1021%2Facschembio.7b00969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span></div><div class="casAuthors">Ishoey, Mette; Chorn, Someth; Singh, Natesh; Jaeger, Martin G.; Brand, Matthias; Paulk, Joshiawa; Bauer, Sophie; Erb, Michael A.; Parapatics, Katja; Muller, Andre C.; Bennett, Keiryn L.; Ecker, Gerhard F.; Bradner, James E.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">553-560</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein degrdn. is an emerging therapeutic strategy with a unique mol. pharmacol. that enables the disruption of all functions assocd. with a target.  This is particularly relevant for proteins depending on mol. scaffolding, such as transcription factors or receptor tyrosine kinases (RTKs).  To address tractability of multiple RTKs for chem. degrdn. by the E3 ligase CUL4-RBX1-DDB1-CRBN (CRL4CRBN), we synthesized a series of phthalimide degraders based on the promiscuous kinase inhibitors sunitinib and PHA665752.  While both series failed to induce degrdn. of their consensus targets, individual mols. displayed pronounced efficacy in leukemia cell lines.  Orthogonal target identification supported by mol. docking led us to identify the translation termination factor G1 to S phase transition 1 (GSPT1) as a converging off-target, resulting from inadvertent E3 ligase modulation.  This research highlights the importance of monitoring degrdn. events that are independent of the resp. targeting ligand as a unique feature of small-mol. degraders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_ZxnB5rKvvLVg90H21EOLACvtfcHk0lg0-smI71aMsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D&md5=02a70959e8a8929a3c70846c1ed0200f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00969%26sid%3Dliteratum%253Aachs%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DChorn%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DJaeger%26aufirst%3DM.%2BG.%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DParapatics%26aufirst%3DK.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DTranslation%2520Termination%2520Factor%2520GSPT1%2520Is%2520a%2520Phenotypically%2520Relevant%2520Off-Target%2520of%2520Heterobifunctional%2520Phthalimide%2520Degraders%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D3%26spage%3D553%26epage%3D560%26doi%3D10.1021%2Facschembio.7b00969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&issue=8&author=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Ciulli&title=Selective+Small+Molecule+Induced+Degradation+of+the+BET+Bromodomain+Protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lg0-smI71aMsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520Small%2520Molecule%2520Induced%2520Degradation%2520of%2520the%2520BET%2520Bromodomain%2520Protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D8%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donoghue, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cubillos-Rojas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez-Prat, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Zarzalejo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdaguer, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nebreda, A. R.</span></span> <span> </span><span class="NLM_article-title">Optimal Linker Length for Small Molecule PROTACs That Selectively Target P38α and P38β for Degradation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>201</i></span>,  <span class="NLM_fpage">112451</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.ejmech.2020.112451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32634680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlais73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=2020&pages=112451&author=C.+Donoghueauthor=M.+Cubillos-Rojasauthor=N.+Gutierrez-Pratauthor=C.+Sanchez-Zarzalejoauthor=X.+Verdaguerauthor=A.+Rieraauthor=A.+R.+Nebreda&title=Optimal+Linker+Length+for+Small+Molecule+PROTACs+That+Selectively+Target+P38%CE%B1+and+P38%CE%B2+for+Degradation&doi=10.1016%2Fj.ejmech.2020.112451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Optimal linker length for small molecule PROTACs that selectively target p38α and p38β for degradation</span></div><div class="casAuthors">Donoghue, Craig; Cubillos-Rojas, Monica; Gutierrez-Prat, Nuria; Sanchez-Zarzalejo, Carolina; Verdaguer, Xavier; Riera, Antoni; Nebreda, Angel R.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112451</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">We report the design of hetero-bifunctional small mols. that selectively target p38α and p38β for degrdn.  These proteolysis targeted chimeras (PROTACs) are based on an ATP competitive inhibitor of p38α and p38β, which is linked to thalidomide analogs to recruit the Cereblon E3 ubiquitin ligase complex.  Compd. synthesis was facilitated by the use of a copper catalyzed "click" reaction.  We show that optimization of the linker length and compn. is crucial for the degrdn.-inducing activity of these PROTACs.  We provide evidence that these chem. compds. can induce degrdn. of p38α and p38β but no other related kinases at nanomolar concns. in several mammalian cell lines.  Accordingly, the PROTACs inhibit stress and cytokine-induced p38α signaling.  Our compds. contribute to understanding the development of PROTACs, and provide a useful tool to investigate functions of the p38 MAPK pathway and its involvement in diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU67xl-ZkKDbVg90H21EOLACvtfcHk0lg-c-fv9Vodgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlais73E&md5=612294e7cc0af1d535e24219bc03de2c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112451%26sid%3Dliteratum%253Aachs%26aulast%3DDonoghue%26aufirst%3DC.%26aulast%3DCubillos-Rojas%26aufirst%3DM.%26aulast%3DGutierrez-Prat%26aufirst%3DN.%26aulast%3DSanchez-Zarzalejo%26aufirst%3DC.%26aulast%3DVerdaguer%26aufirst%3DX.%26aulast%3DRiera%26aufirst%3DA.%26aulast%3DNebreda%26aufirst%3DA.%2BR.%26atitle%3DOptimal%2520Linker%2520Length%2520for%2520Small%2520Molecule%2520PROTACs%2520That%2520Selectively%2520Target%2520P38%25CE%25B1%2520and%2520P38%25CE%25B2%2520for%2520Degradation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D201%26spage%3D112451%26doi%3D10.1016%2Fj.ejmech.2020.112451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klán, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">šolomek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bochet, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Givens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popik, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostikov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirz, J.</span></span> <span> </span><span class="NLM_article-title">Photoremovable Protecting Groups in Chemistry and Biology: Reaction Mechanisms and Efficacy</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1021/cr300177k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr300177k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVyiurvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2013&pages=119-191&issue=1&author=P.+Kl%C3%A1nauthor=T.+%C5%A1olomekauthor=C.+G.+Bochetauthor=A.+Blancauthor=R.+Givensauthor=M.+Rubinaauthor=V.+Popikauthor=A.+Kostikovauthor=J.+Wirz&title=Photoremovable+Protecting+Groups+in+Chemistry+and+Biology%3A+Reaction+Mechanisms+and+Efficacy&doi=10.1021%2Fcr300177k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Photoremovable Protecting Groups in Chemistry and Biology: Reaction Mechanisms and Efficacy</span></div><div class="casAuthors">Klan, Petr; Solomek, Tomas; Bochet, Christian G.; Blanc, Aurelien; Givens, Richard; Rubina, Marina; Popik, Vladimir; Kostikov, Alexey; Wirz, Jakob</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">119-191</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Photoremovable (sometimes called photoreleasable, photocleavable or photoactivatable) protecting groups (PPGs) provide spatial and temporal control over the release of various chems. such as bioagents (neurotransmitters and cell-signaling mols.), acids, bases, Ca2+ ions, oxidants, insecticides, phermones, fragrances, etc.  The present review covers recent developments in the field, focusing on the scope, limitations, and applications of PPGs, which are used to release org. mols.  Simplified basic structures of PPGs discussed in this review are provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXac8Ccb-fmLVg90H21EOLACvtfcHk0lg-c-fv9Vodgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVyiurvL&md5=b8defe84c6c629e44d429f82fa4eb5f0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fcr300177k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr300177k%26sid%3Dliteratum%253Aachs%26aulast%3DKl%25C3%25A1n%26aufirst%3DP.%26aulast%3D%25C5%25A1olomek%26aufirst%3DT.%26aulast%3DBochet%26aufirst%3DC.%2BG.%26aulast%3DBlanc%26aufirst%3DA.%26aulast%3DGivens%26aufirst%3DR.%26aulast%3DRubina%26aufirst%3DM.%26aulast%3DPopik%26aufirst%3DV.%26aulast%3DKostikov%26aufirst%3DA.%26aulast%3DWirz%26aufirst%3DJ.%26atitle%3DPhotoremovable%2520Protecting%2520Groups%2520in%2520Chemistry%2520and%2520Biology%253A%2520Reaction%2520Mechanisms%2520and%2520Efficacy%26jtitle%3DChem.%2520Rev.%26date%3D2013%26volume%3D113%26issue%3D1%26spage%3D119%26epage%3D191%26doi%3D10.1021%2Fcr300177k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTAC Degraders of the Bromodomain and Extra Terminal (BET) Proteins — A Review</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.ddtec.2019.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31200858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=43-51&author=C.+Y.+Yangauthor=C.+Qinauthor=L.+Baiauthor=S.+Wang&title=Small-Molecule+PROTAC+Degraders+of+the+Bromodomain+and+Extra+Terminal+%28BET%29+Proteins+%E2%80%94+A+Review&doi=10.1016%2Fj.ddtec.2019.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule PROTAC degraders of the Bromodomain and Extra Terminal (BET) proteins - A review</span></div><div class="casAuthors">Yang Chao-Yie; Qin Chong; Bai Longchuan; Wang Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43-51</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The PROteolysis TArgeting Chimeric (PROTAC) concept has provided an opportunity for the discovery and development of a completely new type of therapy involving induction of protein degradation.  The BET proteins, comprised of BRD2, BRD3, BRD4 and the testis-specific BRDT protein, are epigenetic readers and master transcription coactivators.  Extremely potent and efficacious small-molecule PROTAC degraders of the BET proteins, based on available, potent and selective BET inhibitors, have been reported.  BET degraders differ from BET inhibitors in their cellular potency, phenotypic effects, pharmacokinetic properties and toxicity profiles.  Herein, we provide a review of BET degraders and the differential outcome observed in the cellular and animal models for BET degraders in comparison to BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoQvupmecNw_lfW6udTcc2ebvdnr-HcsVpbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D&md5=de5790c1db28d56717ff76374072ad81</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSmall-Molecule%2520PROTAC%2520Degraders%2520of%2520the%2520Bromodomain%2520and%2520Extra%2520Terminal%2520%2528BET%2529%2520Proteins%2520%25E2%2580%2594%2520A%2520Review%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D43%26epage%3D51%26doi%3D10.1016%2Fj.ddtec.2019.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappaport, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chicas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">RNAi Screen Identifies Brd4 as a Therapeutic Target in Acute Myeloid Leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span> (<span class="NLM_issue">7370</span>),  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&issue=7370&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+Screen+Identifies+Brd4+as+a+Therapeutic+Target+in+Acute+Myeloid+Leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0liFxWIN3MjK0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520Screen%2520Identifies%2520Brd4%2520as%2520a%2520Therapeutic%2520Target%2520in%2520Acute%2520Myeloid%2520Leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26issue%3D7370%26spage%3D524%26epage%3D528%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&issue=6&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+Ubiquitin+Ligase+Cereblon+to+Efficiently+Target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0liFxWIN3MjK0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520Ubiquitin%2520Ligase%2520Cereblon%2520to%2520Efficiently%2520Target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26issue%3D6%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noblejas-López, M. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieto-Jimenez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Juárez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montero, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esparís-Ogando, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandiella, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galán-Moya, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ocaña, A.</span></span> <span> </span><span class="NLM_article-title">Activity of BET-Proteolysis Targeting Chimeric (PROTAC) Compounds in Triple Negative Breast Cancer</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">383</span>, <span class="refDoi"> DOI: 10.1186/s13046-019-1387-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1186%2Fs13046-019-1387-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31470872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A280%3ADC%252BB3MrjslKksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=383&issue=1&author=M.+D.+M.+Noblejas-L%C3%B3pezauthor=C.+Nieto-Jimenezauthor=M.+Burgosauthor=M.+G%C3%B3mez-Ju%C3%A1rezauthor=J.+C.+Monteroauthor=A.+Espar%C3%ADs-Ogandoauthor=A.+Pandiellaauthor=E.+M.+Gal%C3%A1n-Moyaauthor=A.+Oca%C3%B1a&title=Activity+of+BET-Proteolysis+Targeting+Chimeric+%28PROTAC%29+Compounds+in+Triple+Negative+Breast+Cancer&doi=10.1186%2Fs13046-019-1387-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Activity of BET-proteolysis targeting chimeric (PROTAC) compounds in triple negative breast cancer</span></div><div class="casAuthors">Noblejas-Lopez Maria Del Mar; Nieto-Jimenez Cristina; Burgos Miguel; Gomez-Juarez Monica; Ocana Alberto; Noblejas-Lopez Maria Del Mar; Nieto-Jimenez Cristina; Galan-Moya Eva M; Ocana Alberto; Montero Juan Carlos; Esparis-Ogando Azucena; Pandiella Atanasio; Montero Juan Carlos; Esparis-Ogando Azucena; Pandiella Atanasio; Montero Juan Carlos; Esparis-Ogando Azucena; Pandiella Atanasio; Pandiella Atanasio; Ocana Alberto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of experimental & clinical cancer research : CR</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">383</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Triple negative breast cancer (TNBC) is an incurable disease where novel therapeutic strategies are needed.  Proteolysis targeting chimeric (PROTAC) are novel compounds that promote protein degradation by binding to an ubiquitin ligase.  In this work, we explored the antitumoral activity of two novel BET-PROTACs, MZ1 and ARV-825, in TNBC, ovarian cancer and in a BET inhibitor resistant model.  METHODS:  OVCAR3, SKOV3, BT549, MDA-MB-231 cell lines and the JQ1 resistant cell line MDA-MB-231R were evaluated.  MTTs, colony-forming assay, three-dimensional cultures in matrigel, flow cytometry, and western blots were performed to explore the anti-proliferative effect and biochemical mechanism of action of MZ1 and ARV-825.  In vivo studies included BALB/c nu/nu mice engrafted with MDA-MB-231R cells.  RESULTS:  The BET-PROTACs MZ1 and ARV-825 efficiently downregulated the protein expression levels of the BET protein BRD4, in MDA-MB-231 and MDA-MB-231R.  MZ1 and ARV-825 also showed an antiproliferative effect on sensitive and resistant cells.  This effect was corroborated in other triple negative (BT549) and ovarian cancer (SKOV3, OVCAR3) cell lines.  MZ1 provoked G2/M arrest in MDA-MB-231.  In addition, a profound effect on caspase-dependent apoptosis was observed in both sensitive and resistant cells.  No synergistic activity was observed when it was combined with docetaxel, cisplatin or olaparib.  Finally, in vivo administration of MZ1 rescued tumor growth in a JQ1-resistant xenograft model, reducing the expression levels of BRD4.  CONCLUSIONS:  Using both in vitro and in vivo approaches, we describe the profound activity of BET-PROTACs in parental and BETi-resistant TNBC models.  This data provides options for further clinical development of these agents in TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQfKRCiECTUNcrRqEXsZufAfW6udTcc2eazmZbFnz5dT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MrjslKksw%253D%253D&md5=b6ded6e32e4d1820dcb3e51c75de3f5c</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1186%2Fs13046-019-1387-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-019-1387-5%26sid%3Dliteratum%253Aachs%26aulast%3DNoblejas-L%25C3%25B3pez%26aufirst%3DM.%2BD.%2BM.%26aulast%3DNieto-Jimenez%26aufirst%3DC.%26aulast%3DBurgos%26aufirst%3DM.%26aulast%3DG%25C3%25B3mez-Ju%25C3%25A1rez%26aufirst%3DM.%26aulast%3DMontero%26aufirst%3DJ.%2BC.%26aulast%3DEspar%25C3%25ADs-Ogando%26aufirst%3DA.%26aulast%3DPandiella%26aufirst%3DA.%26aulast%3DGal%25C3%25A1n-Moya%26aufirst%3DE.%2BM.%26aulast%3DOca%25C3%25B1a%26aufirst%3DA.%26atitle%3DActivity%2520of%2520BET-Proteolysis%2520Targeting%2520Chimeric%2520%2528PROTAC%2529%2520Compounds%2520in%2520Triple%2520Negative%2520Breast%2520Cancer%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2019%26volume%3D38%26issue%3D1%26spage%3D383%26doi%3D10.1186%2Fs13046-019-1387-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiskus, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapakshe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saenz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mill, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoury, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhalla, K. N.</span></span> <span> </span><span class="NLM_article-title">BET Protein Proteolysis Targeting Chimera (PROTAC) Exerts Potent Lethal Activity against Mantle Cell Lymphoma Cells</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fleu.2017.207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=28663582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1eqt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=343-352&issue=2&author=B.+Sunauthor=W.+Fiskusauthor=Y.+Qianauthor=K.+Rajapaksheauthor=K.+Rainaauthor=K.+G.+Colemanauthor=A.+P.+Crewauthor=A.+Shenauthor=D.+T.+Saenzauthor=C.+P.+Millauthor=A.+J.+Nowakauthor=N.+Jainauthor=L.+Zhangauthor=M.+Wangauthor=J.+D.+Khouryauthor=C.+Coarfaauthor=C.+M.+Crewsauthor=K.+N.+Bhalla&title=BET+Protein+Proteolysis+Targeting+Chimera+%28PROTAC%29+Exerts+Potent+Lethal+Activity+against+Mantle+Cell+Lymphoma+Cells&doi=10.1038%2Fleu.2017.207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Bet protein proteolysis targeting chimera (protac) exerts potent lethal activity against mantle cell lymphoma cells</span></div><div class="casAuthors">Sun, B.; Fiskus, W.; Qian, Y.; Rajapakshe, K.; Raina, K.; Coleman, K. G.; Crew, A. P.; Shen, A.; Saenz, D. T.; Mill, C. P.; Nowak, A. J.; Jain, N.; Zhang, L.; Wang, M.; Khoury, J. D.; Coarfa, C.; Crews, C. M.; Bhalla, K. N.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">343-352</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Bromodomain extraterminal protein (BETP) inhibitors transcriptionally repress oncoproteins and nuclear factor-κB (NF-κB) target genes that undermines the growth and survival of mantle cell lymphoma (MCL) cells.  However, BET bromodomain inhibitor (BETi) treatment causes accumulation of BETPs, assocd. with reversible binding and incomplete inhibition of BRD4 that potentially compromises the activity of BETi in MCL cells.  Unlike BETi, BET-PROTACs (proteolysis-targeting chimera) ARV-825 and ARV-771 (Arvinas, Inc.) recruit and utilize an E3-ubiquitin ligase to effectively degrade BETPs in MCL cells.  BET-PROTACs induce more apoptosis than BETi of MCL cells, including those resistant to ibrutinib.  BET-PROTAC treatment induced more perturbations in the mRNA and protein expressions than BETi, with depletion of c-Myc, CDK4, cyclin D1 and the NF-κB transcriptional targets Bcl-xL, XIAP and BTK, while inducing the levels of HEXIM1, NOXA and CDKN1A/p21.  Treatment with ARV-771, which possesses superior pharmacol. properties compared with ARV-825, inhibited the in vivo growth and induced greater survival improvement than the BETi OTX015 of immune-depleted mice engrafted with MCL cells.  Cotreatment of ARV-771 with ibrutinib or the BCL2 antagonist venetoclax or CDK4/6 inhibitor palbociclib synergistically induced apoptosis of MCL cells.  These studies highlight promising and superior preclin. activity of BET-PROTAC than BETi, requiring further in vivo evaluation of BET-PROTAC as a therapy for ibrutinib-sensitive or -resistant MCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAZxihoDZILrVg90H21EOLACvtfcHk0ljPdUsMRHNo9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1eqt77J&md5=f771eee574daf11546b93866ea314381</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.207%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DB.%26aulast%3DFiskus%26aufirst%3DW.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DRajapakshe%26aufirst%3DK.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DShen%26aufirst%3DA.%26aulast%3DSaenz%26aufirst%3DD.%2BT.%26aulast%3DMill%26aufirst%3DC.%2BP.%26aulast%3DNowak%26aufirst%3DA.%2BJ.%26aulast%3DJain%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DKhoury%26aufirst%3DJ.%2BD.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DBhalla%26aufirst%3DK.%2BN.%26atitle%3DBET%2520Protein%2520Proteolysis%2520Targeting%2520Chimera%2520%2528PROTAC%2529%2520Exerts%2520Potent%2520Lethal%2520Activity%2520against%2520Mantle%2520Cell%2520Lymphoma%2520Cells%26jtitle%3DLeukemia%26date%3D2018%26volume%3D32%26issue%3D2%26spage%3D343%26epage%3D352%26doi%3D10.1038%2Fleu.2017.207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cassar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adatto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamse, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iturria, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muriana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Cruchten, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zon, L. I.</span></span> <span> </span><span class="NLM_article-title">Use of Zebrafish in Drug Discovery Toxicology</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">118</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrestox.9b00335</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrestox.9b00335" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFOrurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2020&pages=95-118&issue=1&author=S.+Cassarauthor=I.+Adattoauthor=J.+L.+Freemanauthor=J.+T.+Gamseauthor=I.+Iturriaauthor=C.+Lawrenceauthor=A.+Murianaauthor=R.+T.+Petersonauthor=S.+Van+Cruchtenauthor=L.+I.+Zon&title=Use+of+Zebrafish+in+Drug+Discovery+Toxicology&doi=10.1021%2Facs.chemrestox.9b00335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Zebrafish in Drug Discovery Toxicology</span></div><div class="casAuthors">Cassar, Steven; Adatto, Isaac; Freeman, Jennifer L.; Gamse, Joshua T.; Iturria, Inaki; Lawrence, Christian; Muriana, Arantza; Peterson, Randall T.; Van Cruchten, Steven; Zon, Leonard I.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-118</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unpredicted human safety events in clin. trials for new drugs are costly in terms of human health and money.  The drug discovery industry attempts to minimize those events with diligent preclin. safety testing.  Current std. practices are good at preventing toxic compds. from being tested in the clinic; however, false neg. preclin. toxicity results are still a reality.  Continual improvement must be pursued in the preclin. realm.  Higher quality therapies can be brought forward with more information about potential toxicities and assocd. mechanisms.  The zebrafish model is a bridge between in vitro assays and mammalian in vivo studies.  This model is powerful in its breadth of application and tractability for research.  In the past two decades our understanding of disease biol. and drug toxicity has grown significantly owing to thousands of studies on this tiny vertebrate.  This review summarizes challenges and strengths of the model, discusses the 3Rs value that it can deliver, highlights translatable and untranslatable biol., and brings together reports from recent studies with zebrafish focusing on new drug discovery toxicol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7bTRPKrZR37Vg90H21EOLACvtfcHk0ljPdUsMRHNo9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFOrurjI&md5=cca6ff69b1d1feb25bc639a3523118a2</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrestox.9b00335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrestox.9b00335%26sid%3Dliteratum%253Aachs%26aulast%3DCassar%26aufirst%3DS.%26aulast%3DAdatto%26aufirst%3DI.%26aulast%3DFreeman%26aufirst%3DJ.%2BL.%26aulast%3DGamse%26aufirst%3DJ.%2BT.%26aulast%3DIturria%26aufirst%3DI.%26aulast%3DLawrence%26aufirst%3DC.%26aulast%3DMuriana%26aufirst%3DA.%26aulast%3DPeterson%26aufirst%3DR.%2BT.%26aulast%3DVan%2BCruchten%26aufirst%3DS.%26aulast%3DZon%26aufirst%3DL.%2BI.%26atitle%3DUse%2520of%2520Zebrafish%2520in%2520Drug%2520Discovery%2520Toxicology%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2020%26volume%3D33%26issue%3D1%26spage%3D95%26epage%3D118%26doi%3D10.1021%2Facs.chemrestox.9b00335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katz, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behrmann, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plas, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alayev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holz, M. K.</span></span> <span> </span><span class="NLM_article-title">ERRα Regulates the Growth of Triple-Negative Breast Cancer Cells via S6K1-Dependent Mechanism</span>. <i>Signal Transduct. Target. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">17035</span>, <span class="refDoi"> DOI: 10.1038/sigtrans.2017.35</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fsigtrans.2017.35" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=17035&author=A.+Y.+Bermanauthor=S.+Mannaauthor=N.+S.+Schwartzauthor=Y.+E.+Katzauthor=Y.+Sunauthor=C.+A.+Behrmannauthor=J.+J.+Yuauthor=D.+R.+Plasauthor=A.+Alayevauthor=M.+K.+Holz&title=ERR%CE%B1+Regulates+the+Growth+of+Triple-Negative+Breast+Cancer+Cells+via+S6K1-Dependent+Mechanism&doi=10.1038%2Fsigtrans.2017.35"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fsigtrans.2017.35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsigtrans.2017.35%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DA.%2BY.%26aulast%3DManna%26aufirst%3DS.%26aulast%3DSchwartz%26aufirst%3DN.%2BS.%26aulast%3DKatz%26aufirst%3DY.%2BE.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DBehrmann%26aufirst%3DC.%2BA.%26aulast%3DYu%26aufirst%3DJ.%2BJ.%26aulast%3DPlas%26aufirst%3DD.%2BR.%26aulast%3DAlayev%26aufirst%3DA.%26aulast%3DHolz%26aufirst%3DM.%2BK.%26atitle%3DERR%25CE%25B1%2520Regulates%2520the%2520Growth%2520of%2520Triple-Negative%2520Breast%2520Cancer%2520Cells%2520via%2520S6K1-Dependent%2520Mechanism%26jtitle%3DSignal%2520Transduct.%2520Target.%2520Ther.%26date%3D2017%26volume%3D2%26spage%3D17035%26doi%3D10.1038%2Fsigtrans.2017.35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendjennat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kizhake, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, A.</span></span> <span> </span><span class="NLM_article-title">Selective Degradation of CDK6 by a Palbociclib Based PROTAC</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1375</span>– <span class="NLM_lpage">1379</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.03.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.bmcl.2019.03.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=30935795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtlOrt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1375-1379&issue=11&author=S.+Ranaauthor=M.+Bendjennatauthor=S.+Kourauthor=H.+M.+Kingauthor=S.+Kizhakeauthor=M.+Zahidauthor=A.+Natarajan&title=Selective+Degradation+of+CDK6+by+a+Palbociclib+Based+PROTAC&doi=10.1016%2Fj.bmcl.2019.03.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Selective degradation of CDK6 by a palbociclib based PROTAC</span></div><div class="casAuthors">Rana, Sandeep; Bendjennat, Mourad; Kour, Smit; King, Hannah M.; Kizhake, Smitha; Zahid, Muhammad; Natarajan, Amarnath</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1375-1379</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Development of selective kinase inhibitors that target the ATP binding site continues to be a challenge largely due to similar binding pockets.  Palbociclib is a cyclin-dependent kinase inhibitor that targets the ATP binding site of CDK4 and CDK6 with similar potency.  The enzymic function assocd. with the kinase can be effectively probed using kinase inhibitors however the kinase-independent functions cannot.  Herein, we report a palbociclib based PROTAC that selectively degrades CDK6 while sparing the homolog CDK4.  We used competition studies to characterize the binding and mechanism of CDK6 degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrboIMRvD268rVg90H21EOLACvtfcHk0lhyTpi9a0LtWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtlOrt7o%253D&md5=c9c561340e6c80ce51b747b84837ce46</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.03.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.03.035%26sid%3Dliteratum%253Aachs%26aulast%3DRana%26aufirst%3DS.%26aulast%3DBendjennat%26aufirst%3DM.%26aulast%3DKour%26aufirst%3DS.%26aulast%3DKing%26aufirst%3DH.%2BM.%26aulast%3DKizhake%26aufirst%3DS.%26aulast%3DZahid%26aufirst%3DM.%26aulast%3DNatarajan%26aufirst%3DA.%26atitle%3DSelective%2520Degradation%2520of%2520CDK6%2520by%2520a%2520Palbociclib%2520Based%2520PROTAC%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26issue%3D11%26spage%3D1375%26epage%3D1379%26doi%3D10.1016%2Fj.bmcl.2019.03.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Özguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rieger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span> <span> </span><span class="NLM_article-title">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">2508</span>– <span class="NLM_lpage">2520</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01826</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01826" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2508-2520&issue=5&author=J.+Popowauthor=H.+Arnhofauthor=G.+Baderauthor=H.+Bergerauthor=A.+Ciulliauthor=D.+Coviniauthor=C.+Dankauthor=T.+Gmaschitzauthor=P.+Grebauthor=J.+Karolyi-%C3%96zguerauthor=M.+Koeglauthor=D.+B.+McConnellauthor=M.+Pearsonauthor=M.+Riegerauthor=J.+Rinnenthalauthor=V.+Roesslerauthor=A.+Schrenkauthor=M.+Spinaauthor=S.+Steurerauthor=N.+Trainorauthor=E.+Traxlerauthor=C.+Wieshoferauthor=A.+Zoephelauthor=P.+Ettmayer&title=Highly+Selective+PTK2+Proteolysis+Targeting+Chimeras+to+Probe+Focal+Adhesion+Kinase+Scaffolding+Functions&doi=10.1021%2Facs.jmedchem.8b01826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span></div><div class="casAuthors">Popow, Johannes; Arnhof, Heribert; Bader, Gerd; Berger, Helmut; Ciulli, Alessio; Covini, David; Dank, Christian; Gmaschitz, Teresa; Greb, Peter; Karolyi-Oezguer, Jale; Koegl, Manfred; McConnell, Darryl B.; Pearson, Mark; Rieger, Maria; Rinnenthal, Joerg; Roessler, Vanessa; Schrenk, Andreas; Spina, Markus; Steurer, Steffen; Trainor, Nicole; Traxler, Elisabeth; Wieshofer, Corinna; Zoephel, Andreas; Ettmayer, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2508-2520</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacol. inhibition to reduced tumorigenicity.  However, the clin. relevance of targeting PTK2 still remains to be proven.  Here, we present two highly selective and functional PTK2 proteolysis-targeting chimeras utilizing von Hippel-Lindau and cereblon ligands to hijack E3 ligases for PTK2 degrdn.  BI-3663 (cereblon-based) degrades PTK2 with a median DC50 of 30 nM to >80% across a panel of 11 HCC cell lines.  Despite effective PTK2 degrdn., these compds. did not phenocopy the reported antiproliferative effects of PTK2 depletion in any of the cell lines tested.  By disclosing these compds., we hope to provide valuable tools for the study of PTK2 degrdn. across different biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsL2R_QBrfk7Vg90H21EOLACvtfcHk0lhyTpi9a0LtWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D&md5=f0ad27d3ff46ab12e8a90e8d40a6f3c6</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01826%26sid%3Dliteratum%253Aachs%26aulast%3DPopow%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DBerger%26aufirst%3DH.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCovini%26aufirst%3DD.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DKarolyi-%25C3%2596zguer%26aufirst%3DJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DRieger%26aufirst%3DM.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DRoessler%26aufirst%3DV.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSpina%26aufirst%3DM.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DEttmayer%26aufirst%3DP.%26atitle%3DHighly%2520Selective%2520PTK2%2520Proteolysis%2520Targeting%2520Chimeras%2520to%2520Probe%2520Focal%2520Adhesion%2520Kinase%2520Scaffolding%2520Functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D5%26spage%3D2508%26epage%3D2520%26doi%3D10.1021%2Facs.jmedchem.8b01826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serebrenik, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=583-598&issue=2&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+Characterization+of+von+Hippel-Lindau-Recruiting+Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+TANK-Binding+Kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span></div><div class="casAuthors">Crew, Andrew P.; Raina, Kanak; Dong, Hanqing; Qian, Yimin; Wang, Jing; Vigil, Dominico; Serebrenik, Yevgeniy V.; Hamman, Brian D.; Morgan, Alicia; Ferraro, Caterina; Siu, Kam; Neklesa, Taavi K.; Winkler, James D.; Coleman, Kevin G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">583-598</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are bifunctional mols. that recruit an E3 ligase to a target protein to facilitate ubiquitination and subsequent degrdn. of that protein.  While the field of targeted degraders is still relatively young, the potential for this modality to become a differentiated and therapeutic reality is strong, such that both academic and pharmaceutical institutions are now entering this interesting area of research.  In this paper, the authors describe a broadly applicable process for identifying degrader hits based around the serine/threonine kinase TANK-binding kinase 1 (TBK1), and have generalized the key structural elements assocd. with degrdn. activities.  Compd. I is a potent hit (TBK1 DC50 = 12 nM, Dmax = 96%) with excellent selectivity against a related kinase IKKε, which was further used as a chem. tool to assess TBK1 as a target in mutant K-Ras cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIJWCnFncQRLVg90H21EOLACvtfcHk0li4goStj672eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK&md5=41c3627e05ca31aee3ba85815685fcb2</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520von%2520Hippel-Lindau-Recruiting%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520TANK-Binding%2520Kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D2%26spage%3D583%26epage%3D598%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woyach, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the C481S Ibrutinib-Resistance Mutation in Bruton’s Tyrosine Kinase Using PROTAC-Mediated Degradation</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">26</span>),  <span class="NLM_fpage">3564</span>– <span class="NLM_lpage">3575</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00391</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00391" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVCgt7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=3564-3575&issue=26&author=A.+D.+Buhimschiauthor=H.+A.+Armstrongauthor=M.+Toureauthor=S.+Jaime-Figueroaauthor=T.+L.+Chenauthor=A.+M.+Lehmanauthor=J.+A.+Woyachauthor=A.+J.+Johnsonauthor=J.+C.+Byrdauthor=C.+M.+Crews&title=Targeting+the+C481S+Ibrutinib-Resistance+Mutation+in+Bruton%E2%80%99s+Tyrosine+Kinase+Using+PROTAC-Mediated+Degradation&doi=10.1021%2Facs.biochem.8b00391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the C481S Ibrutinib-Resistance Mutation in Bruton's Tyrosine Kinase Using PROTAC-Mediated Degradation</span></div><div class="casAuthors">Buhimschi, Alexandru D.; Armstrong, Haley A.; Toure, Momar; Jaime-Figueroa, Saul; Chen, Timothy L.; Lehman, Amy M.; Woyach, Jennifer A.; Johnson, Amy J.; Byrd, John C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">3564-3575</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of Bruton's tyrosine kinase (BTK) with the irreversible inhibitor ibrutinib has emerged as a transformative treatment option for patients with chronic lymphocytic leukemia (CLL) and other B-cell malignancies.  Yet, more than 80% of CLL patients develop resistance due to a cysteine to serine mutation at the site covalently bound by ibrutinib (C481S).  Currently, an effective treatment option for C481S patients exhibiting relapse to ibrutinib does not exist and these patients have poor outcomes.  To address this, we have developed a PROteolysis TArgeting Chimera (PROTAC) that induces degrdn. of both wild-type and C481S mutant BTK.  We selected a lead PROTAC, MT-802, from several candidates based on its potency to induce BTK knockdown.  MT-802 recruits BTK to the cereblon (CRBN) E3 ubiquitin ligase complex to trigger BTK ubiquitination and degrdn. via the proteasome.  MT-802 binds fewer off-target kinases than ibrutinib and retains equiv. potency (>99% degrdn. at nanomolar concns.) against wild-type and C481S BTK.  In cells isolated from CLL patients with the C481S mutation, MT-802 is able to reduce the pool of active, phosphorylated BTK whereas ibrutinib cannot.  Collectively, these data provide basis for further preclin. study of BTK PROTACs as a novel strategy for treatment of C481S mutant CLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh-ls5ps9FaLVg90H21EOLACvtfcHk0li4goStj672eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVCgt7zO&md5=bd27f09d041413c7bacb6d61fcb9a327</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00391%26sid%3Dliteratum%253Aachs%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DArmstrong%26aufirst%3DH.%2BA.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DT.%2BL.%26aulast%3DLehman%26aufirst%3DA.%2BM.%26aulast%3DWoyach%26aufirst%3DJ.%2BA.%26aulast%3DJohnson%26aufirst%3DA.%2BJ.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520the%2520C481S%2520Ibrutinib-Resistance%2520Mutation%2520in%2520Bruton%25E2%2580%2599s%2520Tyrosine%2520Kinase%2520Using%2520PROTAC-Mediated%2520Degradation%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26issue%3D26%26spage%3D3564%26epage%3D3575%26doi%3D10.1021%2Facs.biochem.8b00391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.+R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+Anaplastic+Lymphoma+Kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0li4goStj672eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520Anaplastic%2520Lymphoma%2520Kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beharry, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolley, G. A.</span></span> <span> </span><span class="NLM_article-title">Azobenzene Photoswitches for Biomolecules</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">4422</span>– <span class="NLM_lpage">4437</span>, <span class="refDoi"> DOI: 10.1039/c1cs15023e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1039%2Fc1cs15023e" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=21483974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVemurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=4422-4437&issue=8&author=A.+A.+Beharryauthor=G.+A.+Woolley&title=Azobenzene+Photoswitches+for+Biomolecules&doi=10.1039%2Fc1cs15023e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Azobenzene photoswitches for biomolecules</span></div><div class="casAuthors">Beharry, Andrew A.; Woolley, G. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4422-4437</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The photoisomerization of azobenzene has been known for almost 75 years but only recently has this process been widely applied to biol. systems.  The central challenge of how to productively couple the isomerization process to a large functional change in a biomol. has been met in a no. of instances and it appears that effective photocontrol of a large variety of biomols. may be possible.  This crit. review summarizes key properties of azobenzene that enable its use as a photoswitch in biol. systems and describes strategies for using azobenzene photoswitches to drive functional changes in peptides, proteins, nucleic acids, lipids, and carbohydrates (192 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU0_QvNuRJzrVg90H21EOLACvtfcHk0ljmvZUli6EbWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVemurg%253D&md5=49407115a7f1bc77edbdba90a79c1e58</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2Fc1cs15023e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cs15023e%26sid%3Dliteratum%253Aachs%26aulast%3DBeharry%26aufirst%3DA.%2BA.%26aulast%3DWoolley%26aufirst%3DG.%2BA.%26atitle%3DAzobenzene%2520Photoswitches%2520for%2520Biomolecules%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26issue%3D8%26spage%3D4422%26epage%3D4437%26doi%3D10.1039%2Fc1cs15023e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of PROTAC Cooperative Recognition for Selective Protein Degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&issue=5&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+Basis+of+PROTAC+Cooperative+Recognition+for+Selective+Protein+Degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0ljmvZUli6EbWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520Basis%2520of%2520PROTAC%2520Cooperative%2520Recognition%2520for%2520Selective%2520Protein%2520Degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D5%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bléger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brouwer, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, S.</span></span> <span> </span><span class="NLM_article-title"><i>o</i>-Fluoroazobenzenes as Readily Synthesized Photoswitches Offering Nearly Quantitative Two-Way Isomerization with Visible Light</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span> (<span class="NLM_issue">51</span>),  <span class="NLM_fpage">20597</span>– <span class="NLM_lpage">20600</span>, <span class="refDoi"> DOI: 10.1021/ja310323y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja310323y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCjur7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=20597-20600&issue=51&author=D.+Bl%C3%A9gerauthor=J.+Schwarzauthor=A.+M.+Brouwerauthor=S.+Hecht&title=o-Fluoroazobenzenes+as+Readily+Synthesized+Photoswitches+Offering+Nearly+Quantitative+Two-Way+Isomerization+with+Visible+Light&doi=10.1021%2Fja310323y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">o-Fluoroazobenzenes as Readily Synthesized Photoswitches Offering Nearly Quantitative Two-Way Isomerization with Visible Light</span></div><div class="casAuthors">Bleger, David; Schwarz, Jutta; Brouwer, Albert M.; Hecht, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">20597-20600</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Azobenzene functionalized with ortho-fluorine atoms has a lower energy of the n-orbital of the Z-isomer, resulting in a sepn. of the E and Z isomers' n→π* absorption bands.  Introducing para-substituents allows for further tuning of the absorption spectra of o-fluoroazobenzenes.  In particular, electron-withdrawing ester groups give rise to a 50 nm sepn. of the n→π* transitions.  Green and blue light can therefore be used to induce E→Z and Z→E isomerizations, resp.  The o-fluoroazobenzene scaffold is readily synthesized and can be inserted into larger structures via its aryl termini.  These new azobenzene derivs. can be switched in both ways with high photoconversions, and their Z-isomers display a remarkably long thermal half-life.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4ku9tBFTq4rVg90H21EOLACvtfcHk0ljmvZUli6EbWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCjur7O&md5=c3e0e90eb46e5d6196f79909e8f2ba51</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fja310323y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja310323y%26sid%3Dliteratum%253Aachs%26aulast%3DBl%25C3%25A9ger%26aufirst%3DD.%26aulast%3DSchwarz%26aufirst%3DJ.%26aulast%3DBrouwer%26aufirst%3DA.%2BM.%26aulast%3DHecht%26aufirst%3DS.%26atitle%3Do-Fluoroazobenzenes%2520as%2520Readily%2520Synthesized%2520Photoswitches%2520Offering%2520Nearly%2520Quantitative%2520Two-Way%2520Isomerization%2520with%2520Visible%2520Light%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26issue%3D51%26spage%3D20597%26epage%3D20600%26doi%3D10.1021%2Fja310323y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-Induced BET Protein Degradation as a Therapy for Castration-Resistant Prostate Cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">26</span>),  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&issue=26&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-Induced+BET+Protein+Degradation+as+a+Therapy+for+Castration-Resistant+Prostate+Cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lgawmmdGAg6tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-Induced%2520BET%2520Protein%2520Degradation%2520as%2520a%2520Therapy%2520for%2520Castration-Resistant%2520Prostate%2520Cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26issue%3D26%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-Kinase Degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2017.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=29129717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=88-99&issue=1&author=H.+T.+Huangauthor=D.+Dobrovolskyauthor=J.+Paulkauthor=G.+Yangauthor=E.+L.+Weisbergauthor=Z.+M.+Doctorauthor=D.+L.+Buckleyauthor=J.+H.+Choauthor=E.+Koauthor=J.+Jangauthor=K.+Shiauthor=H.+G.+Choiauthor=J.+D.+Griffinauthor=Y.+Liauthor=S.+P.+Treonauthor=E.+S.+Fischerauthor=J.+E.+Bradnerauthor=L.+Tanauthor=N.+S.+Gray&title=A+Chemoproteomic+Approach+to+Query+the+Degradable+Kinome+Using+a+Multi-Kinase+Degrader&doi=10.1016%2Fj.chembiol.2017.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span></div><div class="casAuthors">Huang, Hai-Tsang; Dobrovolsky, Dennis; Paulk, Joshiawa; Yang, Guang; Weisberg, Ellen L.; Doctor, Zainab M.; Buckley, Dennis L.; Cho, Joong-Heui; Ko, Eunhwa; Jang, Jaebong; Shi, Kun; Choi, Hwan Geun; Griffin, James D.; Li, Ying; Treon, Steven P.; Fischer, Eric S.; Bradner, James E.; Tan, Li; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heterobifunctional mols. that recruit E3 ubiquitin ligases, such as cereblon, for targeted protein degrdn. represent an emerging pharmacol. strategy.  A major unanswered question is how generally applicable this strategy is to all protein targets.  In this study, we designed a multi-kinase degrader by conjugating a highly promiscuous kinase inhibitor with a cereblon-binding ligand, and used quant. proteomics to discover 28 kinases, including BTK, PTK2, PTK2B, FLT3, AURKA, AURKB, TEC, ULK1, ITK, and nine members of the CDK family, as degradable.  This set of kinases is only a fraction of the intracellular targets bound by the degrader, demonstrating that successful degrdn. requires more than target engagement.  The results guided us to develop selective degraders for FLT3 and BTK, with potentials to improve disease treatment.  Together, this study demonstrates an efficient approach to triage a gene family of interest to identify readily degradable targets for further studies and pre-clin. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYNxhoPV3127Vg90H21EOLACvtfcHk0lgawmmdGAg6tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO&md5=703cfbaca3edd24176472158563f895f</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%2BT.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520Chemoproteomic%2520Approach%2520to%2520Query%2520the%2520Degradable%2520Kinome%2520Using%2520a%2520Multi-Kinase%2520Degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26issue%3D1%26spage%3D88%26epage%3D99%26doi%3D10.1016%2Fj.chembiol.2017.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&issue=1&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+Design+from+Selective+Degradation+with+a+Promiscuous+Warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lgawmmdGAg6tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520Design%2520from%2520Selective%2520Degradation%2520with%2520a%2520Promiscuous%2520Warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26issue%3D1%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span> <span> </span><span class="NLM_article-title">Development of Targeted Protein Degradation Therapeutics</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">937</span>– <span class="NLM_lpage">944</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0362-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fs41589-019-0362-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31527835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=937-944&issue=10&author=P.+P.+Chamberlainauthor=L.+G.+Hamann&title=Development+of+Targeted+Protein+Degradation+Therapeutics&doi=10.1038%2Fs41589-019-0362-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Development of targeted protein degradation therapeutics</span></div><div class="casAuthors">Chamberlain, Philip P.; Hamann, Lawrence G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">937-944</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeted protein degrdn. as a therapeutic modality has seen dramatic progress and massive investment in recent years because of the convergence of two key scientific breakthroughs: optimization of first-generation peptidic proteolysis-targeted chimeras (PROTACs) into more drug-like mols. able to support in vivo proof of concept and the discovery that clin. mols. function as degraders by binding and repurposing the proteins cereblon and DCAF15.  This provided clin. validation for the general approach through the cereblon modulator class of drugs and provided highly drug-like and ligand-efficient E3 ligase binders upon which to tether target-binding moieties.  Increasingly rational and systematic approaches including biophys. and structural studies on ternary complexes are being leveraged as the field advances.  In this Perspective we summarize the discoveries that have laid the foundation for future degrdn. therapeutics, focusing on those classes of small mols. that redirect E3 ubiquitin ligases to non-native substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfcIEwVR-Xq7Vg90H21EOLACvtfcHk0lhyfR6Bais5yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF&md5=468e3fd87ba2b21607dac8b495e5ecf6</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0362-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0362-y%26sid%3Dliteratum%253Aachs%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26atitle%3DDevelopment%2520of%2520Targeted%2520Protein%2520Degradation%2520Therapeutics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26issue%3D10%26spage%3D937%26epage%3D944%26doi%3D10.1038%2Fs41589-019-0362-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuchter, M. J.</span></span> <span> </span><span class="NLM_article-title">On the Promise of Photopharmacology Using Photoswitches: A Medicinal Chemist’s Perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">11436</span>– <span class="NLM_lpage">11447</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00629</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00629" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=11436-11447&issue=20&author=M.+J.+Fuchter&title=On+the+Promise+of+Photopharmacology+Using+Photoswitches%3A+A+Medicinal+Chemist%E2%80%99s+Perspective&doi=10.1021%2Facs.jmedchem.0c00629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">On the Promise of Photopharmacology Using Photoswitches: A Medicinal Chemist's Perspective</span></div><div class="casAuthors">Fuchter, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">11436-11447</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Photopharmacol. is a growing area of endeavor that employs photoswitchable ligands to allow for light-dependent pharmacol. activity.  By coupling light to therapeutic action, improved spatial and temporal selectivity can be achieved and subsequently harnessed for new concepts in therapy.  Tremendous progress has already been made, with photopharmacol. agents now reported against a wide array of target classes and light-dependent results demonstrated in a range of live cell and animal models.  Several challenges remain, however, esp. in order for photopharmacol. to truly impact the clin. management of disease.  This Perspective aims to summarize these challenges, particularly with attention to the medicinal chem. that will be unavoidably required for the further translation of these agents/approaches.  By clearly defining challenges for drug hunters, it is hoped that further research into the medicinal chem. of photopharmacol. agents will be stimulated, ultimately enabling full realization of the huge potential for this exciting field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ_HhYiq4_z7Vg90H21EOLACvtfcHk0lhyfR6Bais5yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3E&md5=ab802ee3cb69c87058b34b56b1175a8b</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00629%26sid%3Dliteratum%253Aachs%26aulast%3DFuchter%26aufirst%3DM.%2BJ.%26atitle%3DOn%2520the%2520Promise%2520of%2520Photopharmacology%2520Using%2520Photoswitches%253A%2520A%2520Medicinal%2520Chemist%25E2%2580%2599s%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D20%26spage%3D11436%26epage%3D11447%26doi%3D10.1021%2Facs.jmedchem.0c00629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morstein, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauner, D.</span></span> <span> </span><span class="NLM_article-title">New Players in Phototherapy: Photopharmacology and Bio-Integrated Optoelectronics</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2019.03.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.cbpa.2019.03.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31108317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFOgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2019&pages=145-151&author=J.+Morsteinauthor=D.+Trauner&title=New+Players+in+Phototherapy%3A+Photopharmacology+and+Bio-Integrated+Optoelectronics&doi=10.1016%2Fj.cbpa.2019.03.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">New players in phototherapy: photopharmacology and bio-integrated optoelectronics</span></div><div class="casAuthors">Morstein, Johannes; Trauner, Dirk</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">145-151</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Photodynamic therapy and phototherapy are used in the clinic to treat dermatol. conditions, cancer, macular degeneration, and a variety of other diseases.  Despite their long history and widespread application, the scope of these therapeutic approaches has been limited by a lack of specificity and challenges with light delivery.  In recent years, much progress has been made in these regards.  Photopharmacol. has provided drug-like mols. that change their efficacy upon irradn. and allow for the optical control of a wide range of defined biol. targets.  Many photopharmaceuticals are now used in vivo and some show promising results in preclin. development.  At the same time, new bioelectronics for subdermal light delivery have been engineered that could enable phototherapy deep in tissue, for example within the human brain.  These developments could increase the impact of photodynamic therapy in human precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEDSiiVMgoY7Vg90H21EOLACvtfcHk0lhyfR6Bais5yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFOgsb8%253D&md5=23ee3fbfdf33ec9c2cb5e5292f0bbc4a</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2019.03.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2019.03.013%26sid%3Dliteratum%253Aachs%26aulast%3DMorstein%26aufirst%3DJ.%26aulast%3DTrauner%26aufirst%3DD.%26atitle%3DNew%2520Players%2520in%2520Phototherapy%253A%2520Photopharmacology%2520and%2520Bio-Integrated%2520Optoelectronics%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D50%26spage%3D145%26epage%3D151%26doi%3D10.1016%2Fj.cbpa.2019.03.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konrad, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savasci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allmendinger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochsenfeld, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A. M.</span></span> <span> </span><span class="NLM_article-title">Computational Design and Synthesis of a Deeply Red-Shifted and Bistable Azobenzene</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">6538</span>– <span class="NLM_lpage">6547</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b10430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b10430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXltlylsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=6538-6547&issue=14&author=D.+B.+Konradauthor=G.+Savasciauthor=L.+Allmendingerauthor=D.+Traunerauthor=C.+Ochsenfeldauthor=A.+M.+Ali&title=Computational+Design+and+Synthesis+of+a+Deeply+Red-Shifted+and+Bistable+Azobenzene&doi=10.1021%2Fjacs.9b10430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Design and Synthesis of a Deeply Red-Shifted and Bistable Azobenzene</span></div><div class="casAuthors">Konrad, David B.; Savasci, Goekcen; Allmendinger, Lars; Trauner, Dirk; Ochsenfeld, Christian; Ali, Ahmed M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6538-6547</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We computationally dissected the electronic and geometrical influences of ortho-chlorinated azobenzenes on their photophys. properties.  X-ray anal. provided the insight that trans-tetra-ortho-chloro azobenzene is conformationally flexible and thus subject to mol. motions.  This allows the photoswitch to adopt a range of red-shifted geometries which account for the extended n → π* band tails.  Based on our results, we designed the di-ortho-fluoro di-ortho-chloro (dfdc) azobenzene and provided computational evidence for the superiority of this substitution pattern to tetra-ortho-chloro azobenzene.  Thereafter, we synthesized dfdc azobenzene by ortho-chlorination via 2-fold C-H activation and exptl. confirmed its structural and photophys. properties through UV-vis, NMR and X-ray analyses.  The advantages include near-bistable isomers and an increased sepn. of the n → π* bands between the trans- and cis-conformations which allows for the generation of unusually high levels of the cis-isomer by irradn. with green/yellow light as well as red light within the biooptical window.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopyLeuf7pYA7Vg90H21EOLACvtfcHk0lhAKgSp-6QdJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXltlylsrs%253D&md5=72f3c84a0357af9f90a2f9b99c066460</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b10430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b10430%26sid%3Dliteratum%253Aachs%26aulast%3DKonrad%26aufirst%3DD.%2BB.%26aulast%3DSavasci%26aufirst%3DG.%26aulast%3DAllmendinger%26aufirst%3DL.%26aulast%3DTrauner%26aufirst%3DD.%26aulast%3DOchsenfeld%26aufirst%3DC.%26aulast%3DAli%26aufirst%3DA.%2BM.%26atitle%3DComputational%2520Design%2520and%2520Synthesis%2520of%2520a%2520Deeply%2520Red-Shifted%2520and%2520Bistable%2520Azobenzene%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26issue%3D14%26spage%3D6538%26epage%3D6547%26doi%3D10.1021%2Fjacs.9b10430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Petra  Dunkel</span>, <span class="hlFld-ContribAuthor ">Janez  Ilaš</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted Cancer Therapy Using Compounds Activated by Light. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (13)
                                     , 3237. <a href="https://doi.org/10.3390/cancers13133237" title="DOI URL">https://doi.org/10.3390/cancers13133237</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13133237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13133237%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DTargeted%252BCancer%252BTherapy%252BUsing%252BCompounds%252BActivated%252Bby%252BLight%26aulast%3DDunkel%26aufirst%3DPetra%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D13%26spage%3D3237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Hongjie  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Photopharmacology of Proteolysis-Targeting Chimeras: A New Frontier for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.639176" title="DOI URL">https://doi.org/10.3389/fchem.2021.639176</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.639176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.639176%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DPhotopharmacology%252Bof%252BProteolysis-Targeting%252BChimeras%25253A%252BA%252BNew%252BFrontier%252Bfor%252BDrug%252BDiscovery%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M.  Maneiro</span>, <span class="hlFld-ContribAuthor ">E.  De Vita</span>, <span class="hlFld-ContribAuthor ">D.  Conole</span>, <span class="hlFld-ContribAuthor ">C.S.  Kounde</span>, <span class="hlFld-ContribAuthor ">Q.  Zhang</span>, <span class="hlFld-ContribAuthor ">E.W.  Tate</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs, molecular glues and bifunctionals from bench to bedside: Unlocking the clinical potential of catalytic drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/bs.pmch.2021.01.002" title="DOI URL">https://doi.org/10.1016/bs.pmch.2021.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.pmch.2021.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.pmch.2021.01.002%26sid%3Dliteratum%253Aachs%26atitle%3DPROTACs%25252C%252Bmolecular%252Bglues%252Band%252Bbifunctionals%252Bfrom%252Bbench%252Bto%252Bbedside%25253A%252BUnlocking%252Bthe%252Bclinical%252Bpotential%252Bof%252Bcatalytic%252Bdrugs%26aulast%3DManeiro%26aufirst%3DM.%26date%3D2021%26pub%3DElsevier%26date%3D2021%26volume%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. PROTAC mode of action and caging concept. (A) The PROTAC mediates the formation of a ternary complex between the protein of interest (POI) and an E3 ligase leading to post-translational modification of the POI with ubiquitin (Ub) and degradation within the proteasome catalytic unit. A caging group can be attached to the ligand recruiting the E3 ligase, to the warhead targeting the POI, and potentially to the linker, with the purpose of generating an inactive degrader. Degradation should occur only after removal of the caging group in response to irradiation with the appropriate wavelength of light. (B) Crystal structure of thalidomide bound to DDB1-CRBN E3 ubiquitin ligase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CI1">4CI1</a>)<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (left) and crystal structure of VHL ligand bound to pVHL:EloB:EloC complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W9H">4W9H</a>)<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (right). Caging group attachment points and exit vectors for the linker appendage are identified by red and black arrows, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0002.jpeg" id="rightTab-Gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of the reported caged PROTACs. Each caged degrader is made of a warhead (depicted in blue), a linker (black), an E3 ligase recruiter (green), and a photocleavable group (red). Compounds <b>1</b>, <b>2</b>, <b>5</b>, <b>6</b>, and <b>8</b> were designed to degrade bromodomain-containing protein 4 (BRD4) upon irradiation. Compound <b>4</b> was designed to degrade the estrogen-related receptor α (ERRα) upon irradiation. Compounds <b>3</b> and <b>7</b> were designed to degrade Bruton tyrosine kinase (BTK) and anaplastic lymphoma kinase (ALK), respectively, upon irradiation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Chemical functionalization of E3 ligase ligands with photolabile groups. Varied strategies have been used to introduce the caging groups via acylation or alkylation with most groups installed relatively early in the synthesis as direct chemical modification of the parent PROTAC in the final step was not always feasible. The yield reported represents the yield of the caging step. “Early” caging refers to a synthetic route in which the caging group was not directly attached to the parent PROTAC as opposed to “late” caging. DBU = 1,8-diazabicyclo[5.4.0]undec-7-ene; DEACM = diethylamino coumarin; DIPEA = <i>N</i>,<i>N</i>-diisopropylethylamine; DMNB = 4,5-dimethoxy-2-nitrobenzyl; NaHMDS = sodium bis(trimethylsilyl)amide; NPOM = 6-nitropiperonyloxymethyl.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of caged PROTACs on cell proliferation and in zebrafish. (A) HeLa cells were treated with DMSO, caged PROTAC <b>8</b> (1 μM), or the uncaged parent PROTAC of <b>8</b> (1 μM) either without initial irradiation or with a 60 s irradiation time at 365 nm. Cell count was followed over 6 days. (B) Phenotype of zebrafish embryos treated with dBET1 and caged PROTAC <b>1</b> at 24 and 36 h postfertilization (hpf). (C) Expression of Brd4 protein in treated embryos examined by Western blot. Images adapted with permission from ref <a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a>. Copyright 2019 American Chemical Society.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Photoswitchable degrader design, structures, and activity profiles. (A) The azobenzene motif delivers two geometrical isomers: the <i>cis</i> isomer can be obtained by irradiation of the <i>trans</i> isomer with a specific wavelength λ<sub>1</sub>, and inversely, the <i>trans</i> isomer can be generated from the <i>cis</i> either with another wavelength λ<sub>2</sub> or by thermal relaxation. The difference in conformation between the two isomers affects binding interactions and allows degradation to occur with only one isomer (<i>cis</i> or <i>trans</i>) but not the other. (B) Structure of the reported photoswitchable PROTAC <b>9</b> (left). Each degrader is made of a warhead (depicted in blue), an E3 ligase recruiter (green), and a photoswitch (red) as part of the linker (black). Immunoblot analysis after treatment of Ramos cells with compound <b>9</b> (right). The <i>trans</i> isomer of <b>9</b> degraded BRD2, while the <i>cis</i> isomer did not affect BRD2 levels. Image adapted with permission from ref <a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a>. Copyright 2019 American Chemical Society. (C) Structure of the reported photoswitchable PROTAC <b>10</b> (left) and immunoblot analysis after treatment of RS4;11 cells with compound <b>10</b> (right). The <i>cis</i> isomer of <b>10</b> knocked down BRD2, BRD3, and BRD4, while the <i>trans</i> isomer did not degrade any BRD. Image adapted with permission from ref <a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>. Copyright The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. Distributed under a Creative Commons Attribution License 4.0 (CC BY) <a href="https://creativecommons.org/licenses/by/4.0/" class="extLink">https://creativecommons.org/licenses/by/4.0/</a>. (D) Structure of the reported photoswitchable PROTAC <b>11</b> (left). Immunoblot analysis following treatment of K562 cells with compound <b>11</b> (right). The <i>trans</i> isomer of <b>11</b> was efficient in reducing levels of BCR-ABL and ABL while its <i>cis</i> counterpart showed no degradation activity. Image adapted with permission from ref <a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a>. Copyright 2020 American Chemical Society.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/medium/jm0c01542_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Influence of the linker length on FKBP12 photoswitchable PROTAC degradation profile. The warhead is depicted in blue, the E3 ligase recruiter is in green, and the photoswitch forming part of the black linker is depicted in red. RS4;11 cells were treated with various FKBP12 targeting photoswitchable compounds for 4 h at different concentrations. Cells were either irradiated with pulses of 390 nm light (<i>cis</i>) or kept the dark (<i>trans</i>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01542/20201218/images/large/jm0c01542_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01542&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i19">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75404" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75404" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 68 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alber, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suter, D. M.</span></span> <span> </span><span class="NLM_article-title">Dynamics of Protein Synthesis and Degradation through the Cell Cycle</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">784</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.1080/15384101.2019.1598725</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1080%2F15384101.2019.1598725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=30907235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsVSqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=784-794&issue=8&author=A.+B.+Alberauthor=D.+M.+Suter&title=Dynamics+of+Protein+Synthesis+and+Degradation+through+the+Cell+Cycle&doi=10.1080%2F15384101.2019.1598725"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics of protein synthesis and degradation through the cell cycle</span></div><div class="casAuthors">Alber, Andrea Brigitta; Suter, David Michael</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">784-794</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1551-4005</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein expression levels depend on the balance between their synthesis and degrdn. rates.  Even quiescent (G0) cells display a continuous turnover of proteins, despite protein levels remaining largely const. over time.  In cycling cells, global protein levels need to be precisely doubled at each cell division in order to maintain cellular homeostasis, but we still lack a quant. understanding of how this is achieved.  Recent studies have shed light on cell cycle-dependent changes in protein synthesis and degrdn. rates.  Here we discuss current population-based and single cell approaches used to assess protein synthesis and degrdn., and review the insights they have provided into the dynamics of protein turnover in different cell cycle phases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4nVSPIQg-v7Vg90H21EOLACvtfcHk0lisP55SqsuliQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsVSqtLw%253D&md5=98a155677bc6f1c56e5d01ed1b4bcd04</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1080%2F15384101.2019.1598725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15384101.2019.1598725%26sid%3Dliteratum%253Aachs%26aulast%3DAlber%26aufirst%3DA.%2BB.%26aulast%3DSuter%26aufirst%3DD.%2BM.%26atitle%3DDynamics%2520of%2520Protein%2520Synthesis%2520and%2520Degradation%2520through%2520the%2520Cell%2520Cycle%26jtitle%3DCell%2520Cycle%26date%3D2019%26volume%3D18%26issue%3D8%26spage%3D784%26epage%3D794%26doi%3D10.1080%2F15384101.2019.1598725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hershko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciechanover, A.</span></span> <span> </span><span class="NLM_article-title">The Ubiquitin System</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">425</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.67.1.425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1146%2Fannurev.biochem.67.1.425" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=9759494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADyaK1cXlsFOmsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1998&pages=425-479&issue=1&author=A.+Hershkoauthor=A.+Ciechanover&title=The+Ubiquitin+System&doi=10.1146%2Fannurev.biochem.67.1.425"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin system</span></div><div class="casAuthors">Hershko, Avram; Ciechanover, Aaron</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">425-479</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review, with ∼298 refs.  The selective degrdn. of many short-lived proteins in eukaryotic cells is carried out by the ubiquitin system.  In this pathway, proteins are targeted for degrdn. by covalent ligation to ubiquitin, a highly conserved small protein.  Ubiquitin-mediated degrdn. of regulatory proteins plays important roles in the control of numerous processes, including cell-cycle progression, signal transduction, transcriptional regulation, receptor down-regulation, and endocytosis.  The ubiquitin system has been implicated in the immune response, development, and programmed cell death.  Abnormalities in ubiquitin-mediated processes have been shown to cause pathol. conditions, including malignant transformation.  In this review we discuss recent information on functions and mechanisms of the ubiquitin system.  Since the selectivity of protein degrdn. is detd. mainly at the stage of ligation to ubiquitin, special attention is focused on what we know, and would like to know, about the mode of action of ubiquitin-protein ligation systems and about signals in proteins recognized by these systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqo5VhDx0kSLVg90H21EOLACvtfcHk0lisP55SqsuliQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlsFOmsLc%253D&md5=7c3c0a6df9c07baf55b5000eae13ef49</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.67.1.425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.67.1.425%26sid%3Dliteratum%253Aachs%26aulast%3DHershko%26aufirst%3DA.%26aulast%3DCiechanover%26aufirst%3DA.%26atitle%3DThe%2520Ubiquitin%2520System%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D1998%26volume%3D67%26issue%3D1%26spage%3D425%26epage%3D479%26doi%3D10.1146%2Fannurev.biochem.67.1.425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciechanover, A.</span></span> <span> </span><span class="NLM_article-title">Intracellular Protein Degradation: From a Vague Idea Thru the Lysosome and the Ubiquitin-Proteasome System and onto Human Diseases and Drug Targeting</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1178</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1038/sj.cdd.4401692</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fsj.cdd.4401692" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=16094394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnt1Shtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=1178-1190&issue=9&author=A.+Ciechanover&title=Intracellular+Protein+Degradation%3A+From+a+Vague+Idea+Thru+the+Lysosome+and+the+Ubiquitin-Proteasome+System+and+onto+Human+Diseases+and+Drug+Targeting&doi=10.1038%2Fsj.cdd.4401692"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular protein degradation: from a vague idea thru the lysosome and the ubiquitin-proteasome system and onto human diseases and drug targeting</span></div><div class="casAuthors">Ciechanover, A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death and Differentiation</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1178-1190</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Between the 1950s and 1980s, scientists were focusing mostly on how the genetic code is transcribed to RNA and translated to proteins, but how proteins were degraded had remained a neglected research area.  With the discovery of the lysosome by Christian de Duve, it was assumed that cellular proteins were degraded within this organelle.  However, several independent lines of exptl. evidence strongly suggested that intracellular proteolysis was largely non-lysosomal, but the mechanisms involved remained obscure.  The discovery of the ubiquitin-proteasome system resolved the enigma.  It is now recognized that degrdn. of intracellular proteins is involved in the regulation of a broad array of cellular processes, such as the cell cycle and division, regulation of transcription factors, and assurance of the cellular quality control.  Not surprisingly, aberrations in the system have been implicated in the pathogenesis of human disease, such as malignancies and neurodegenerative disorders, which have subsequently led to an increasing effort to develop mechanism-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM8QTfF_nT07Vg90H21EOLACvtfcHk0ljiVgIzyTJg3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnt1Shtro%253D&md5=70d3202030b388727cf3d77d9145810a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fsj.cdd.4401692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.cdd.4401692%26sid%3Dliteratum%253Aachs%26aulast%3DCiechanover%26aufirst%3DA.%26atitle%3DIntracellular%2520Protein%2520Degradation%253A%2520From%2520a%2520Vague%2520Idea%2520Thru%2520the%2520Lysosome%2520and%2520the%2520Ubiquitin-Proteasome%2520System%2520and%2520onto%2520Human%2520Diseases%2520and%2520Drug%2520Targeting%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2005%26volume%3D12%26issue%3D9%26spage%3D1178%26epage%3D1190%26doi%3D10.1038%2Fsj.cdd.4401692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nalepa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery in the Ubiquitin - Proteasome System</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">596</span>– <span class="NLM_lpage">613</span>, <span class="refDoi"> DOI: 10.1038/nrd2056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fnrd2056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=16816840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVagsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=596-613&issue=7&author=G.+Nalepaauthor=M.+Rolfeauthor=J.+W.+Harper&title=Drug+Discovery+in+the+Ubiquitin+-+Proteasome+System&doi=10.1038%2Fnrd2056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery in the ubiquitin-proteasome system</span></div><div class="casAuthors">Nalepa, Grzegorz; Rolfe, Mark; Harper, J. Wade</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">596-613</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Regulated protein turnover via the ubiquitin-proteasome system (UPS) underlies a wide variety of signaling pathways, from cell-cycle control and transcription to development.  Recent evidence that pharmacol. inhibition of the proteasome can be efficacious in the treatment of human cancers has set the stage for attempts to selectively inhibit the activities of disease-specific components of the UPS.  Here, we review recent advances linking UPS components with specific human diseases, most prominently cancer and neurodegenerative disorders, and emphasize potential sites of therapeutic intervention along the regulated protein-degrdn. pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAhoqSVFxu-bVg90H21EOLACvtfcHk0ljiVgIzyTJg3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVagsro%253D&md5=41af26cfd515d10504e2d97f161a9626</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd2056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2056%26sid%3Dliteratum%253Aachs%26aulast%3DNalepa%26aufirst%3DG.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26atitle%3DDrug%2520Discovery%2520in%2520the%2520Ubiquitin%2520-%2520Proteasome%2520System%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26issue%3D7%26spage%3D596%26epage%3D613%26doi%3D10.1038%2Fnrd2056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROteolysis TArgeting Chimeras (PROTACs) — Past, Present and Future</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=PROteolysis+TArgeting+Chimeras+%28PROTACs%29+%E2%80%94+Past%2C+Present+and+Future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eaU90xZ0YQJLrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROteolysis%2520TArgeting%2520Chimeras%2520%2528PROTACs%2529%2520%25E2%2580%2594%2520Past%252C%2520Present%2520and%2520Future%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span> <span> </span><span class="NLM_article-title">Inducing Protein-Protein Interactions with Molecular Glues</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">2585</span>– <span class="NLM_lpage">2592</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.04.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.bmcl.2018.04.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=29980357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2585-2592&issue=15&author=Y.+Cheauthor=A.+M.+Gilbertauthor=V.+Shanmugasundaramauthor=M.+C.+Noe&title=Inducing+Protein-Protein+Interactions+with+Molecular+Glues&doi=10.1016%2Fj.bmcl.2018.04.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Inducing protein-protein interactions with molecular glues</span></div><div class="casAuthors">Che, Ye; Gilbert, Adam M.; Shanmugasundaram, Veerabahu; Noe, Mark C.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2585-2592</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The drugable proteome is limited by the no. of functional binding sites that can bind small mols. and respond with a therapeutic effect.  Orthosteric and allosteric modulators of enzyme function or receptor signaling are well-established mechanisms of drug action.  Drugs that perturb protein-protein interactions have only recently been launched.  This approach is more difficult due to the extensive contact surfaces that must be perturbed antagonistically.  Compds. that promote novel protein-protein interactions promise to dramatically expand opportunities for therapeutic intervention.  This approach is precedented with natural products (rapamycin, FK506, sanglifehrin A), synthetic small mols. (thalidomide and IMiD derivs.) and indisulam analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5IlWCdwTjKrVg90H21EOLACvtfcHk0lg8ChyJ0t4M_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjur%252FL&md5=c9fc1cf5e0f60c869a3bb1bf041fa2be</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.04.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.04.046%26sid%3Dliteratum%253Aachs%26aulast%3DChe%26aufirst%3DY.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26atitle%3DInducing%2520Protein-Protein%2520Interactions%2520with%2520Molecular%2520Glues%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26issue%3D15%26spage%3D2585%26epage%3D2592%26doi%3D10.1016%2Fj.bmcl.2018.04.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainolfi, N.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation Mechanisms</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.ddtec.2019.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31200860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=53-60&author=Y.+Zhangauthor=C.+Lohauthor=J.+Chenauthor=N.+Mainolfi&title=Targeted+Protein+Degradation+Mechanisms&doi=10.1016%2Fj.ddtec.2019.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation mechanisms</span></div><div class="casAuthors">Zhang Yi; Loh Christine; Chen Jesse; Mainolfi Nello</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">53-60</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Targeted protein degradation mediated by small molecule degraders represents an exciting new therapeutic opportunity to eliminate disease-causing proteins.  These molecules recruit E3 ubiquitin ligases to the protein of interest and mediate its ubiquitination and subsequent proteolysis by the proteasome.  Significant advancements have been made in the discovery and development of clinically relevant degraders.  In this review we will focus on the recent progress in understanding ternary complex formation and structures, ubiquitination, and other critical factors that govern the efficiency of degraders both in vitro and in vivo.  With deeper knowledges of these areas, the field is building guiding principles to reduce the level of empiricism and to identify therapeutically relevant degraders more rationally and efficiently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRBrI9DTGjC0fW6udTcc2eaU90xZ0YQJLrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtg%253D%253D&md5=bff167555f8bcab6b672cad3c2946a7b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLoh%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMainolfi%26aufirst%3DN.%26atitle%3DTargeted%2520Protein%2520Degradation%2520Mechanisms%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D53%26epage%3D60%26doi%3D10.1016%2Fj.ddtec.2019.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fink, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span> <span> </span><span class="NLM_article-title">The Novel Mechanism of Lenalidomide Activity</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>126</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">2366</span>– <span class="NLM_lpage">2369</span>, <span class="refDoi"> DOI: 10.1182/blood-2015-07-567958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1182%2Fblood-2015-07-567958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=26438514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvFSrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2015&pages=2366-2369&issue=21&author=E.+C.+Finkauthor=B.+L.+Ebert&title=The+Novel+Mechanism+of+Lenalidomide+Activity&doi=10.1182%2Fblood-2015-07-567958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The novel mechanism of lenalidomide activity</span></div><div class="casAuthors">Fink, Emma C.; Ebert, Benjamin L.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2366-2369</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Lenalidomide acts by a novel drug mechanism-modulation of the substrate specificity of the CRL4CRBN E3 ubiquitin ligase.  In multiple myeloma, lenalidomide induces the ubiquitination of IKZF1 and IKZF3 by CRL4CRBN.  Subsequent proteasomal degrdn. of these transcription factors kills multiple myeloma cells.  In del(5q) myelodysplastic syndrome, lenalidomide induces the degrdn. of CK1α, which preferentially affects del(5q) cells because they express this gene at haploin sufficient levels.  In the future, modulation of ubiquitin ligase function may enable us to target previously ''undruggable'' proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh0d_VTHctuLVg90H21EOLACvtfcHk0lg8ChyJ0t4M_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvFSrtbo%253D&md5=81696cccbf1dda9d96cf970a4bcb10a6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1182%2Fblood-2015-07-567958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2015-07-567958%26sid%3Dliteratum%253Aachs%26aulast%3DFink%26aufirst%3DE.%2BC.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26atitle%3DThe%2520Novel%2520Mechanism%2520of%2520Lenalidomide%2520Activity%26jtitle%3DBlood%26date%3D2015%26volume%3D126%26issue%3D21%26spage%3D2366%26epage%3D2369%26doi%3D10.1182%2Fblood-2015-07-567958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: Chimeric Molecules That Target Proteins to the Skp1-Cullin-F Box Complex for Ubiquitination and Degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&issue=15&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+Chimeric+Molecules+That+Target+Proteins+to+the+Skp1-Cullin-F+Box+Complex+for+Ubiquitination+and+Degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0liixCeSjmWYEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520Chimeric%2520Molecules%2520That%2520Target%2520Proteins%2520to%2520the%2520Skp1-Cullin-F%2520Box%2520Complex%2520for%2520Ubiquitination%2520and%2520Degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26issue%3D15%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">First Targeted Protein Degrader Hits the Clinic</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1038/d41573-019-00043-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fd41573-019-00043-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=237-239&author=A.+Mullard&title=First+Targeted+Protein+Degrader+Hits+the+Clinic&doi=10.1038%2Fd41573-019-00043-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fd41573-019-00043-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-019-00043-6%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DFirst%2520Targeted%2520Protein%2520Degrader%2520Hits%2520the%2520Clinic%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D237%26epage%3D239%26doi%3D10.1038%2Fd41573-019-00043-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 Ubiquitin Ligase in Complex with Thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&issue=1&author=E.+S.+Fischerauthor=K.+B%C3%B6hmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thom%C3%A4&title=Structure+of+the+DDB1-CRBN+E3+Ubiquitin+Ligase+in+Complex+with+Thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0liixCeSjmWYEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DB%25C3%25B6hm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520Ubiquitin%2520Ligase%2520in%2520Complex%2520with%2520Thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26issue%3D1%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&issue=20&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-Guided+Design+and+Optimization+of+Small+Molecules+Targeting+the+Protein-Protein+Interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+Ubiquitin+Ligase+and+the+Hypoxia+Inducible+Factor+%28HIF%29+Alpha+Subunit+with+in+Vitro+Nanomolar+Affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0ljMXeMVw69YPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-Guided%2520Design%2520and%2520Optimization%2520of%2520Small%2520Molecules%2520Targeting%2520the%2520Protein-Protein%2520Interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520Ubiquitin%2520Ligase%2520and%2520the%2520Hypoxia%2520Inducible%2520Factor%2520%2528HIF%2529%2520Alpha%2520Subunit%2520with%2520in%2520Vitro%2520Nanomolar%2520Affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D20%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&issue=2&author=I.+Churcher&title=Protac-Induced+Protein+Degradation+in+Drug+Discovery%3A+Breaking+the+Rules+or+Just+Making+New+Ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0lghRURi4GF5lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-Induced%2520Protein%2520Degradation%2520in%2520Drug%2520Discovery%253A%2520Breaking%2520the%2520Rules%2520or%2520Just%2520Making%2520New%2520Ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D2%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation: From Chemical Biology to Drug Discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&issue=9&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+Protein+Degradation%3A+From+Chemical+Biology+to+Drug+Discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0lghRURi4GF5lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%253A%2520From%2520Chemical%2520Biology%2520to%2520Drug%2520Discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26issue%3D9%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Röth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulcher, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapkota, G. P.</span></span> <span> </span><span class="NLM_article-title">Advances in Targeted Degradation of Endogenous Proteins</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">2761</span>– <span class="NLM_lpage">2777</span>, <span class="refDoi"> DOI: 10.1007/s00018-019-03112-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1007%2Fs00018-019-03112-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31030225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosFKlur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=2761-2777&issue=14&author=S.+R%C3%B6thauthor=L.+J.+Fulcherauthor=G.+P.+Sapkota&title=Advances+in+Targeted+Degradation+of+Endogenous+Proteins&doi=10.1007%2Fs00018-019-03112-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in targeted degradation of endogenous proteins</span></div><div class="casAuthors">Roth, Sascha; Fulcher, Luke J.; Sapkota, Gopal P.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2761-2777</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">A review.  Protein silencing is often employed as a means to aid investigations in protein function and is increasingly desired as a therapeutic approach.  Several types of protein silencing methodologies have been developed, including targeting the encoding genes, transcripts, the process of translation or the protein directly.  Despite these advances, most silencing systems suffer from limitations.  Silencing protein expression through genetic ablation, for example by CRISPR/Cas9 genome editing, is irreversible, time consuming and not always feasible.  Similarly, RNA interference approaches warrant prolonged treatments, can lead to incomplete protein depletion and are often assocd. with off-target effects.  Targeted proteolysis has the potential to overcome some of these limitations.  The field of targeted proteolysis has witnessed the emergence of many methodologies aimed at targeting specific proteins for degrdn. in a spatio-temporal manner.  In this review, we provide an appraisal of the different targeted proteolytic systems and discuss their applications in understanding protein function, as well as their potential in therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplQoWtY4BlG7Vg90H21EOLACvtfcHk0lghRURi4GF5lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosFKlur4%253D&md5=e69ce7541082c46b5fd536cf5c93c7dd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs00018-019-03112-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-019-03112-6%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25B6th%26aufirst%3DS.%26aulast%3DFulcher%26aufirst%3DL.%2BJ.%26aulast%3DSapkota%26aufirst%3DG.%2BP.%26atitle%3DAdvances%2520in%2520Targeted%2520Degradation%2520of%2520Endogenous%2520Proteins%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2019%26volume%3D76%26issue%3D14%26spage%3D2761%26epage%3D2777%26doi%3D10.1007%2Fs00018-019-03112-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in Vivo Protein Knockdown by Small-Molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&issue=8&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+Den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+Vivo+Protein+Knockdown+by+Small-Molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lj7CmdHFapyYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3DDen%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520Vivo%2520Protein%2520Knockdown%2520by%2520Small-Molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D8%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuaid, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2017.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=29129716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=67-77&issue=1&author=G.+M.+Burslemauthor=B.+E.+Smithauthor=A.+C.+Laiauthor=S.+Jaime-Figueroaauthor=D.+C.+McQuaidauthor=D.+P.+Bondesonauthor=M.+Toureauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=A.+P.+Crewauthor=J.+Hinesauthor=C.+M.+Crews&title=The+Advantages+of+Targeted+Protein+Degradation+Over+Inhibition%3A+An+RTK+Case+Study&doi=10.1016%2Fj.chembiol.2017.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span></div><div class="casAuthors">Burslem, George M.; Smith, Blake E.; Lai, Ashton C.; Jaime-Figueroa, Saul; McQuaid, Daniel C.; Bondeson, Daniel P.; Toure, Momar; Dong, Hanqing; Qian, Yimin; Wang, Jing; Crew, Andrew P.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-77.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proteolysis targeting chimera (PROTAC) technol. has emerged over the last two decades as a powerful tool for targeted degrdn. of endogenous proteins.  Herein we describe the development of PROTACs for receptor tyrosine kinases, a protein family yet to be targeted for induced protein degrdn.  The use of VHL-recruiting PROTACs against this protein family reveals several advantages of degrdn. over inhibition alone: direct comparisons of fully functional, target-degrading PROTACs with target-inhibiting variants that contain an inactivated E3 ligase-recruiting ligand show that degrdn. leads to more potent inhibition of cell proliferation and a more durable and sustained downstream signaling response, and thus addresses the kinome rewiring challenge seen with many receptor tyrosine kinase inhibitors.  Combined, these findings demonstrate the ability to target receptor tyrosine kinases for degrdn. using the PROTAC technol. and outline the advantages of this degrdn.-based approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4yDhRDWqNv7Vg90H21EOLACvtfcHk0lj7CmdHFapyYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM&md5=0ac39a6f5836ea3772a0396356efb630</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DMcQuaid%26aufirst%3DD.%2BC.%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520Advantages%2520of%2520Targeted%2520Protein%2520Degradation%2520Over%2520Inhibition%253A%2520An%2520RTK%2520Case%2520Study%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26issue%3D1%26spage%3D67%26epage%3D77%26doi%3D10.1016%2Fj.chembiol.2017.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span> (<span class="NLM_issue">49</span>),  <span class="NLM_fpage">17019</span>– <span class="NLM_lpage">17026</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=17019-17026&issue=49&author=P.+M.+Crommauthor=K.+T.+G.+Samarasingheauthor=J.+Hinesauthor=C.+M.+Crews&title=Addressing+Kinase-Independent+Functions+of+Fak+via+PROTAC-Mediated+Degradation&doi=10.1021%2Fjacs.8b08008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span></div><div class="casAuthors">Cromm, Philipp M.; Samarasinghe, Kusal T. G.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17019-17026</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic inhibition proved to be a successful modality for the development of many small-mol. drugs.  In recent years, small-mol.-induced protein degrdn. has emerged as an orthogonal therapeutic strategy that has the potential to expand the druggable target space.  Focal adhesion kinase (Fak) is a key player in tumor invasion and metastasis, acting simultaneously as a kinase and a scaffold for several signaling proteins.  While previous efforts to modulate Fak activity were limited to kinase inhibitors with low success in clin. studies, protein degrdn. offers a possibility to simultaneously block Fak's kinase signaling and scaffolding capabilities.  Here, the authors report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clin. candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion.  These results underline the potential that PROTACs offer in expanding the druggable space and controlling protein functions that are not easily addressed by traditional small-mol. therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTn8r4NmDDObVg90H21EOLACvtfcHk0lj7CmdHFapyYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF&md5=b97b1a646a8f9dbf7a2bed640d0f001d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAddressing%2520Kinase-Independent%2520Functions%2520of%2520Fak%2520via%2520PROTAC-Mediated%2520Degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26issue%3D49%26spage%3D17019%26epage%3D17026%26doi%3D10.1021%2Fjacs.8b08008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomoshige, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohgane, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of Small Molecules That Induce the Degradation of Huntingtin</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">11530</span>– <span class="NLM_lpage">11533</span>, <span class="refDoi"> DOI: 10.1002/anie.201706529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1002%2Fanie.201706529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSmurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=11530-11533&issue=38&author=S.+Tomoshigeauthor=S.+Nomuraauthor=K.+Ohganeauthor=Y.+Hashimotoauthor=M.+Ishikawa&title=Discovery+of+Small+Molecules+That+Induce+the+Degradation+of+Huntingtin&doi=10.1002%2Fanie.201706529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecules that Induce the Degradation of Huntingtin</span></div><div class="casAuthors">Tomoshige, Shusuke; Nomura, Sayaka; Ohgane, Kenji; Hashimoto, Yuichi; Ishikawa, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11530-11533</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is an autosomal dominant neurodegenerative disorder caused by the aggregation of mutant huntingtin (mHtt), and removal of toxic mHtt is expected to be an effective therapeutic approach.  The authors designed two small hybrid mols. by linking a ligand for ubiquitin ligase (cellular inhibitor of apoptosis protein 1; cIAP1) with probes for mHtt aggregates, anticipating that these compds. would recruit cIAP1 to mHtt and induce selective degrdn. by the ubiquitin-proteasome system.  The synthesized compds. reduced mHtt levels in HD patient fibroblasts and appear to be promising candidates for the development of a treatment for HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSFiHw5mJy2rVg90H21EOLACvtfcHk0lhMmQvLvpHSOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSmurzL&md5=fb68cd4e5577d195a4a26f42c29428c3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fanie.201706529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201706529%26sid%3Dliteratum%253Aachs%26aulast%3DTomoshige%26aufirst%3DS.%26aulast%3DNomura%26aufirst%3DS.%26aulast%3DOhgane%26aufirst%3DK.%26aulast%3DHashimoto%26aufirst%3DY.%26aulast%3DIshikawa%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520Small%2520Molecules%2520That%2520Induce%2520the%2520Degradation%2520of%2520Huntingtin%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26issue%3D38%26spage%3D11530%26epage%3D11533%26doi%3D10.1002%2Fanie.201706529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">11280</span>– <span class="NLM_lpage">11300</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01530</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01530" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11280-11300&issue=24&author=H.+Zhouauthor=L.+Baiauthor=R.+Xuauthor=Y.+Zhaoauthor=J.+Chenauthor=D.+McEachernauthor=K.+Chinnaswamyauthor=B.+Wenauthor=L.+Daiauthor=P.+Kumarauthor=C.+Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=J.+L.+Meagherauthor=H.+Yiauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-Based+Discovery+of+SD-36+as+a+Potent%2C+Selective%2C+and+Efficacious+PROTAC+Degrader+of+STAT3+Protein&doi=10.1021%2Facs.jmedchem.9b01530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span></div><div class="casAuthors">Zhou, Haibin; Bai, Longchuan; Xu, Renqi; Zhao, Yujun; Chen, Jianyong; McEachern, Donna; Chinnaswamy, Krishnapriya; Wen, Bo; Dai, Lipeng; Kumar, Praveen; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Meagher, Jennifer L.; Yi, Han; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11280-11300</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases.  Despite twenty-years of persistent research efforts, targeting STAT3 has been very challenging.  The authors report herein the structure-based discovery of potent small-mol. STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept.  The authors first designed SI-109 as a potent, small-mol. inhibitor of the STAT3 SH2 domain.  Employing ligands for cereblon/Cullin 4A E3 ligase and SI-109, the authors obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36.  SD-36 induces rapid STAT3 degrdn. at low nanomolar concns. in cells and fails to degrader other STAT proteins.  SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3.  A single dose of SD-36 results in complete STAT3 protein degrdn. in xenograft tumor tissue and normal mouse tissues.  SD-36 achieves complete and long-lasting tumor regression in the Molm-16 xenograft tumor model at well-tolerated dose-schedules.  SD-36 is a potent, selective and efficacious STAT3 degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIAWd1hiuqy7Vg90H21EOLACvtfcHk0lhMmQvLvpHSOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J&md5=7d0c656a2d96196a3b71829c9efe715a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01530%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-Based%2520Discovery%2520of%2520SD-36%2520as%2520a%2520Potent%252C%2520Selective%252C%2520and%2520Efficacious%2520PROTAC%2520Degrader%2520of%2520STAT3%2520Protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D24%26spage%3D11280%26epage%3D11300%26doi%3D10.1021%2Facs.jmedchem.9b01530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Harnessing the Power of Proteolysis for Targeted Protein Inactivation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>77</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.molcel.2020.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32004468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFCjsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2020&pages=446-460&issue=3&author=R.+Vermaauthor=D.+Mohlauthor=R.+J.+Deshaies&title=Harnessing+the+Power+of+Proteolysis+for+Targeted+Protein+Inactivation&doi=10.1016%2Fj.molcel.2020.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing the Power of Proteolysis for Targeted Protein Inactivation</span></div><div class="casAuthors">Verma, Rati; Mohl, Dane; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">446-460</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Two decades into the twenty-first century, a confluence of breakthrough technologies wielded at the mol. level is presenting biologists with unique opportunities to unravel the complexities of the cellular world.  CRISPR/Cas9 allows gene knock-outs, knock-ins, and single-base editing at chromosomal loci.  RNA-based tools such as siRNA, antisense oligos, and morpholinos can be used to silence expression of specific genes.  Meanwhile, protein knockdown tools that draw inspiration from natural regulatory mechanisms and facilitate elimination of native or degron-tagged proteins from cells are rapidly emerging.  The acute and reversible redn. in protein levels enabled by these methods allows for precise detn. of loss-of-function phenotypes free from secondary effects or compensatory adaptation that can confound nucleic-acid-based methods that involve slow depletion or permanent loss of a protein.  In this Review, we summarize the ingenious ways biologists have exploited natural mechanisms for protein degrdn. to direct the elimination of specific proteins at will.  This has led to advancements not only in basic research but also in the therapeutic space with the introduction of PROTACs into clin. trials for cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-2OyHAw-uMLVg90H21EOLACvtfcHk0lhMmQvLvpHSOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFCjsr4%253D&md5=8d74ed00d55184be33786d0ee44ad787</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DR.%26aulast%3DMohl%26aufirst%3DD.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DHarnessing%2520the%2520Power%2520of%2520Proteolysis%2520for%2520Targeted%2520Protein%2520Inactivation%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D77%26issue%3D3%26spage%3D446%26epage%3D460%26doi%3D10.1016%2Fj.molcel.2020.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riching, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDougall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span> <span> </span><span class="NLM_article-title">Quantitative Live-Cell Kinetic Degradation and Mechanistic Profiling of PROTAC Mode of Action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2758</span>– <span class="NLM_lpage">2770</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00692</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00692" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFOrtbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2758-2770&issue=9&author=K.+M.+Richingauthor=S.+Mahanauthor=C.+R.+Coronaauthor=M.+McDougallauthor=J.+D.+Vastaauthor=M.+B.+Robersauthor=M.+Urhauthor=D.+L.+Daniels&title=Quantitative+Live-Cell+Kinetic+Degradation+and+Mechanistic+Profiling+of+PROTAC+Mode+of+Action&doi=10.1021%2Facschembio.8b00692"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative Live-Cell Kinetic Degradation and Mechanistic Profiling of PROTAC Mode of Action</span></div><div class="casAuthors">Riching, Kristin M.; Mahan, Sarah; Corona, Cesear R.; McDougall, Mark; Vasta, James D.; Robers, Matthew B.; Urh, Marjeta; Daniels, Danette L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2758-2770</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new generation of heterobifunctional small mols., termed proteolysis targeting chimeras (PROTACs), targets proteins for degrdn. through recruitment to E3 ligases and holds significant therapeutic potential.  Despite numerous successful examples, PROTAC small mol. development remains laborious and unpredictable, involving testing compds. for end-point degrdn. activity at fixed times and concns. without resolving or optimizing for the important biol. steps required for the process.  Given the complexity of the ubiquitin proteasomal pathway, technologies that enable real-time characterization of PROTAC efficacy and mechanism of action are crit. for accelerating compd. development, profiling, and improving guidance of chem. structure-activity relationship.  Here, the authors present an innovative, modular live-cell platform utilizing endogenous tagging technologies and apply it to monitoring PROTAC-mediated degrdn. of the bromodomain and extra-terminal family members.  The authors show comprehensive real-time degrdn. and recovery profiles for each target, precisely quantifying degrdn. rates, maximal levels of degrdn. (Dmax), and time frame at Dmax.  These degrdn. metrics show specific PROTAC and family member-dependent responses that are closely assocd. with the key cellular protein interactions required for the process.  Kinetic studies show cellular ternary complex stability influences potency and degrdn. efficacy.  Meanwhile, the level of ubiquitination is highly correlated to degrdn. rate, indicating ubiquitination stemming from productive ternary complex formation is the main driver of the degrdn. rate.  The approaches applied here highlight the steps at which the choice of E3 ligase handle can elicit different outcomes and discern individual parameters required for degrdn., ultimately enabling chem. design strategies and rank ordering of potential therapeutic compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnkETO-K622bVg90H21EOLACvtfcHk0lhxp3Hq4Px5QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFOrtbbM&md5=9a2a3162dd930ac04622db6990c0d748</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00692%26sid%3Dliteratum%253Aachs%26aulast%3DRiching%26aufirst%3DK.%2BM.%26aulast%3DMahan%26aufirst%3DS.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DMcDougall%26aufirst%3DM.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26atitle%3DQuantitative%2520Live-Cell%2520Kinetic%2520Degradation%2520and%2520Mechanistic%2520Profiling%2520of%2520PROTAC%2520Mode%2520of%2520Action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D9%26spage%3D2758%26epage%3D2770%26doi%3D10.1021%2Facschembio.8b00692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=361-368&issue=3&author=M.+J.+Royauthor=S.+Winklerauthor=S.+J.+Hughesauthor=C.+Whitworthauthor=M.+Galantauthor=W.+Farnabyauthor=K.+Rumpelauthor=A.+Ciulli&title=SPR-Measured+Dissociation+Kinetics+of+PROTAC+Ternary+Complexes+Influence+Target+Degradation+Rate&doi=10.1021%2Facschembio.9b00092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span></div><div class="casAuthors">Roy, Michael J.; Winkler, Sandra; Hughes, Scott J.; Whitworth, Claire; Galant, Michael; Farnaby, William; Rumpel, Klaus; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-368</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bifunctional degrader mols., known as proteolysis-targeting chimeras (PROTACs), function by recruiting a target to an E3 ligase, forming a target/PROTAC/ligase ternary complex.  Despite the importance of this key intermediate species, no detailed validation of a method to directly det. binding parameters for ternary complex kinetics has been reported, and it remains to be addressed whether tuning the kinetics of PROTAC ternary complexes may be an effective strategy to improve the efficiency of targeted protein degrdn.  Here, we develop an SPR-based assay to quantify the stability of PROTAC-induced ternary complexes by measuring for the first time the kinetics of their formation and dissocn. in vitro using purified proteins.  We benchmark our assay using four PROTACs that target the bromodomains (BDs) of bromodomain and extraterminal domain proteins Brd2, Brd3, and Brd4 to the von Hippel-Lindau E3 ligase (VHL).  We reveal marked differences in ternary complex off-rates for different PROTACs that exhibit either pos. or neg. cooperativity for ternary complex formation relative to binary binding.  The pos. cooperative degrader MZ1 forms comparatively stable and long-lived ternary complexes with either Brd4BD2 or Brd2BD2 and VHL.  Equivalent complexes with Brd3BD2 are destabilized due to a single amino acid difference (Glu/Gly swap) present in the bromodomain.  We observe that this difference in ternary complex dissociative half-life correlates to a greater initial rate of intracellular degrdn. of Brd2 and Brd4 relative to Brd3.  These findings establish a novel assay to measure the kinetics of PROTAC ternary complexes and elucidate the important kinetic parameters that drive effective target degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFhvuKYztcbVg90H21EOLACvtfcHk0lhxp3Hq4Px5QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D&md5=629d1acbd218b668a2696b727946962b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00092%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSPR-Measured%2520Dissociation%2520Kinetics%2520of%2520PROTAC%2520Ternary%2520Complexes%2520Influence%2520Target%2520Degradation%2520Rate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26issue%3D3%26spage%3D361%26epage%3D368%26doi%3D10.1021%2Facschembio.9b00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ankenbruck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courtney, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Optochemical Control of Biological Processes in Cells and Animals</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2768</span>– <span class="NLM_lpage">2798</span>, <span class="refDoi"> DOI: 10.1002/anie.201700171</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1002%2Fanie.201700171" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFGjur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=2768-2798&issue=11&author=N.+Ankenbruckauthor=T.+Courtneyauthor=Y.+Naroauthor=A.+Deiters&title=Optochemical+Control+of+Biological+Processes+in+Cells+and+Animals&doi=10.1002%2Fanie.201700171"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Optochemical Control of Biological Processes in Cells and Animals</span></div><div class="casAuthors">Ankenbruck, Nicholas; Courtney, Taylor; Naro, Yuta; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2768-2798</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Biol. processes are naturally regulated with high spatial and temporal control, as is perhaps most evident in metazoan embryogenesis.  Chem. tools have been extensively utilized in cell and developmental biol. to investigate cellular processes, and conditional control methods have expanded applications of these technologies toward resolving complex biol. questions.  Light represents an excellent external trigger since it can be controlled with very high spatial and temporal precision.  To this end, several optically regulated tools have been developed and applied to living systems.  In this review we discuss recent developments of optochem. tools, including small mols., peptides, proteins, and nucleic acids that can be irreversibly or reversibly controlled through light irradn., with a focus on applications in cells and animals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDYWdcoWgIZ7Vg90H21EOLACvtfcHk0lhxp3Hq4Px5QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFGjur4%253D&md5=2b36ff9c84a2611458386526b16adcc7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201700171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201700171%26sid%3Dliteratum%253Aachs%26aulast%3DAnkenbruck%26aufirst%3DN.%26aulast%3DCourtney%26aufirst%3DT.%26aulast%3DNaro%26aufirst%3DY.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DOptochemical%2520Control%2520of%2520Biological%2520Processes%2520in%2520Cells%2520and%2520Animals%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26issue%3D11%26spage%3D2768%26epage%3D2798%26doi%3D10.1002%2Fanie.201700171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachl, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newham, P.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-Targeting Chimeras in Drug Development: A Safety Perspective</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>177</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1709</span>– <span class="NLM_lpage">1718</span>, <span class="refDoi"> DOI: 10.1111/bph.15014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1111%2Fbph.15014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32022252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjslSms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2020&pages=1709-1718&issue=8&author=K.+Moreauauthor=M.+Coenauthor=A.+X.+Zhangauthor=F.+Pachlauthor=M.+P.+Castaldiauthor=G.+Dahlauthor=H.+Boydauthor=C.+Scottauthor=P.+Newham&title=Proteolysis-Targeting+Chimeras+in+Drug+Development%3A+A+Safety+Perspective&doi=10.1111%2Fbph.15014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-targeting chimeras in drug development: A safety perspective</span></div><div class="casAuthors">Moreau, Kevin; Coen, Muireann; Zhang, Andrew X.; Pachl, Fiona; Castaldi, M. Paola; Dahl, Goran; Boyd, Helen; Scott, Clay; Newham, Pete</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1709-1718</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Proteolysis-targeting chimeras are a new drug modality that exploits the endogenous ubiquitin proteasome system to degrade a protein of interest for therapeutic benefit.  As the first-generation of proteolysis-targeting chimeras have now entered clin. trials for oncol. indications, it is timely to consider the theor. safety risks inherent with this modality which include off-target degrdn., intracellular accumulation of natural substrates for the E3 ligases used in the ubiquitin proteasome system, proteasome satn. by ubiquitinated proteins, and liabilities assocd. with the "hook effect" of proteolysis-targeting chimeras This review describes in vitro and non-clin. in vivo data that provide mechanistic insight of these safety risks and approaches being used to mitigate these risks in the next generation of proteolysis-targeting chimera mols. to extend therapeutic applications beyond life-threatening diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreTK_TldLxHbVg90H21EOLACvtfcHk0lhaE-9DK5B15w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjslSms7o%253D&md5=1e26bbe1d623f2f8f4218c2e3fad9dd6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1111%2Fbph.15014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.15014%26sid%3Dliteratum%253Aachs%26aulast%3DMoreau%26aufirst%3DK.%26aulast%3DCoen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DA.%2BX.%26aulast%3DPachl%26aufirst%3DF.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DBoyd%26aufirst%3DH.%26aulast%3DScott%26aufirst%3DC.%26aulast%3DNewham%26aufirst%3DP.%26atitle%3DProteolysis-Targeting%2520Chimeras%2520in%2520Drug%2520Development%253A%2520A%2520Safety%2520Perspective%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2020%26volume%3D177%26issue%3D8%26spage%3D1709%26epage%3D1718%26doi%3D10.1111%2Fbph.15014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shchepinova, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kounde, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chudasama, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, E. W.</span></span> <span> </span><span class="NLM_article-title">Antibody-PROTAC Conjugates Enable HER2-Dependent Targeted Protein Degradation of BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1306</span>– <span class="NLM_lpage">1312</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00285</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00285" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1306-1312&issue=6&author=M.+Maneiroauthor=N.+Forteauthor=M.+M.+Shchepinovaauthor=C.+S.+Koundeauthor=V.+Chudasamaauthor=J.+R.+Bakerauthor=E.+W.+Tate&title=Antibody-PROTAC+Conjugates+Enable+HER2-Dependent+Targeted+Protein+Degradation+of+BRD4&doi=10.1021%2Facschembio.0c00285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-PROTAC Conjugates Enable HER2-Dependent Targeted Protein Degradation of BRD4</span></div><div class="casAuthors">Maneiro, Maria; Forte, Nafsika; Shchepinova, Maria M.; Kounde, Cyrille S.; Chudasama, Vijay; Baker, James Richard; Tate, Edward W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1306-1312</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting protein degrdn. with Proteolysis-Targeting Chimeras (PROTACs) is an area of great current interest in drug discovery.  Nevertheless, although the high effectiveness of PROTACs against a wide variety of targets has been established, most degraders reported to date display limited intrinsic tissue selectivity and do not discriminate between cells of different types.  Here, we describe a strategy for selective protein degrdn. in a specific cell type.  We report the design and synthesis of a trastuzumab-PROTAC conjugate (Ab-PROTAC 3) in which E3 ligase-directed degrader activity is caged with an antibody linker which can be hydrolyzed following antibody-PROTAC internalization, releasing the active PROTAC and inducing catalytic protein degrdn.  We show that 3 selectively targets bromodomain-contg. protein 4 (BRD4) for degrdn. only in HER2 pos. breast cancer cell lines, while sparing HER2 neg. cells.  Using live cell confocal microscopy, we show internalization and lysosomal trafficking of the conjugate specifically in HER2 pos. cells, leading to the release of active PROTAC in quantities sufficient to induce potent BRD4 degrdn.  These studies demonstrate proof-of-concept for tissue-specific BRD4 degrdn., overcoming limitations of PROTAC selectivity, with significant potential for application to novel targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPwGNLaGmRbbVg90H21EOLACvtfcHk0lhaE-9DK5B15w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D&md5=544b53e54d3f0696f0b425c3f6e48e46</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00285%26sid%3Dliteratum%253Aachs%26aulast%3DManeiro%26aufirst%3DM.%26aulast%3DForte%26aufirst%3DN.%26aulast%3DShchepinova%26aufirst%3DM.%2BM.%26aulast%3DKounde%26aufirst%3DC.%2BS.%26aulast%3DChudasama%26aufirst%3DV.%26aulast%3DBaker%26aufirst%3DJ.%2BR.%26aulast%3DTate%26aufirst%3DE.%2BW.%26atitle%3DAntibody-PROTAC%2520Conjugates%2520Enable%2520HER2-Dependent%2520Targeted%2520Protein%2520Degradation%2520of%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26issue%3D6%26spage%3D1306%26epage%3D1312%26doi%3D10.1021%2Facschembio.0c00285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaquiere, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rei Ingalla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis Phillips, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowntree, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertz, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Antibody-Mediated Delivery of Chimeric Protein Degraders Which Target Estrogen Receptor Alpha (ERα)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">126907</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.bmcl.2019.126907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31902710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFWgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=126907&issue=4&author=P.+S.+Dragovichauthor=P.+Adhikariauthor=R.+A.+Blakeauthor=N.+Blaquiereauthor=J.+Chenauthor=Y.+X.+Chengauthor=W.+den+Bestenauthor=J.+Hanauthor=S.+J.+Hartmanauthor=J.+Heauthor=M.+Heauthor=E.+Rei+Ingallaauthor=A.+V.+Kamathauthor=T.+Kleinheinzauthor=T.+Laiauthor=D.+D.+Leipoldauthor=C.+S.+Liauthor=Q.+Liuauthor=J.+Luauthor=Y.+Luauthor=F.+Mengauthor=L.+Mengauthor=C.+Ngauthor=K.+Pengauthor=G.+Lewis+Phillipsauthor=T.+H.+Pillowauthor=R.+K.+Rowntreeauthor=J.+D.+Sadowskyauthor=D.+Sampathauthor=L.+Stabenauthor=S.+T.+Stabenauthor=J.+Waiauthor=K.+Wanauthor=X.+Wangauthor=B.+Weiauthor=I.+E.+Wertzauthor=J.+Xinauthor=K.+Xuauthor=H.+Yaoauthor=R.+Zangauthor=D.+Zhangauthor=H.+Zhouauthor=Y.+Zhao&title=Antibody-Mediated+Delivery+of+Chimeric+Protein+Degraders+Which+Target+Estrogen+Receptor+Alpha+%28ER%CE%B1%29&doi=10.1016%2Fj.bmcl.2019.126907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-mediated delivery of chimeric protein degraders which target estrogen receptor alpha (ERα)</span></div><div class="casAuthors">Dragovich, Peter S.; Adhikari, Pragya; Blake, Robert A.; Blaquiere, Nicole; Chen, Jinhua; Cheng, Yun-Xing; den Besten, Willem; Han, Jinping; Hartman, Steven J.; He, Jintang; He, Mingtao; Rei Ingalla, Ellen; Kamath, Amrita V.; Kleinheinz, Tracy; Lai, Tommy; Leipold, Douglas D.; Li, Chun Sing; Liu, Qi; Lu, Jiawei; Lu, Ying; Meng, Fanwei; Meng, Lingyao; Ng, Carl; Peng, Kaishan; Lewis Phillips, Gail; Pillow, Thomas H.; Rowntree, Rebecca K.; Sadowsky, Jack D.; Sampath, Deepak; Staben, Leanna; Staben, Steven T.; Wai, John; Wan, Kunpeng; Wang, Xinxin; Wei, Bin Qing; Wertz, Ingrid E.; Xin, Jianfeng; Xu, Keyang; Yao, Hui; Zang, Richard; Zhang, Donglu; Zhou, Hao; Zhao, Yongxin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">126907</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Chimeric mols. which effect intracellular degrdn. of target proteins via E3 ligase-mediated ubiquitination (e.g., PROTACs) are currently of high interest in medicinal chem.  However, these entities are relatively large compds. that often possess mol. characteristics which may compromise oral bioavailability, soly., and/or in vivo pharmacokinetic properties.  Accordingly, we explored whether conjugation of chimeric degraders to monoclonal antibodies using technologies originally developed for cytotoxic payloads might provide alternate delivery options for these novel agents.  In this report we describe the construction of several degrader-antibody conjugates comprised of two distinct ERα-targeting degrader entities and three independent ADC linker modalities.  We subsequently demonstrate the antigen-dependent delivery to MCF7-neo/HER2 cells of the degrader payloads that are incorporated into these conjugates.  We also provide evidence for efficient intracellular degrader release from one of the employed linkers.  In addn., preliminary data are described which suggest that reasonably favorable in vivo stability properties are assocd. with the linkers utilized to construct the degrader conjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdPWcGkXC85LVg90H21EOLACvtfcHk0lhaE-9DK5B15w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFWgtw%253D%253D&md5=5f196b194df7f4622825c4b901130f7c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126907%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DAdhikari%26aufirst%3DP.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DBlaquiere%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DY.%2BX.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DHartman%26aufirst%3DS.%2BJ.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DRei%2BIngalla%26aufirst%3DE.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLeipold%26aufirst%3DD.%2BD.%26aulast%3DLi%26aufirst%3DC.%2BS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DPeng%26aufirst%3DK.%26aulast%3DLewis%2BPhillips%26aufirst%3DG.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DRowntree%26aufirst%3DR.%2BK.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DStaben%26aufirst%3DL.%26aulast%3DStaben%26aufirst%3DS.%2BT.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWan%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26aulast%3DXin%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DZang%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DAntibody-Mediated%2520Delivery%2520of%2520Chimeric%2520Protein%2520Degraders%2520Which%2520Target%2520Estrogen%2520Receptor%2520Alpha%2520%2528ER%25CE%25B1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26issue%3D4%26spage%3D126907%26doi%3D10.1016%2Fj.bmcl.2019.126907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonger, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakhit, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payumo, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wandless, T. J.</span></span> <span> </span><span class="NLM_article-title">General Method for Regulating Protein Stability with Light</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1021/cb400755b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400755b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWiu7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=111-115&issue=1&author=K.+M.+Bongerauthor=R.+Rakhitauthor=A.+Y.+Payumoauthor=J.+K.+Chenauthor=T.+J.+Wandless&title=General+Method+for+Regulating+Protein+Stability+with+Light&doi=10.1021%2Fcb400755b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">General Method for Regulating Protein Stability with Light</span></div><div class="casAuthors">Bonger, Kimberly M.; Rakhit, Rishi; Payumo, Alexander Y.; Chen, James K.; Wandless, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111-115</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational regulation of protein abundance in cells is a powerful tool for studying protein function.  Here, we describe a novel genetically encoded protein domain that is degraded upon exposure to nontoxic blue light.  We demonstrate that fusion proteins contg. this domain are rapidly degraded in cultured cells and in zebrafish upon illumination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplL0DRj0csdLVg90H21EOLACvtfcHk0lid9gd55_idvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWiu7rO&md5=4a69db08c6128a6b0e48d53e0a6e99f4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fcb400755b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400755b%26sid%3Dliteratum%253Aachs%26aulast%3DBonger%26aufirst%3DK.%2BM.%26aulast%3DRakhit%26aufirst%3DR.%26aulast%3DPayumo%26aufirst%3DA.%2BY.%26aulast%3DChen%26aufirst%3DJ.%2BK.%26aulast%3DWandless%26aufirst%3DT.%2BJ.%26atitle%3DGeneral%2520Method%2520for%2520Regulating%2520Protein%2520Stability%2520with%2520Light%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26issue%3D1%26spage%3D111%26epage%3D115%26doi%3D10.1021%2Fcb400755b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Renicke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usherenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essen, L. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taxis, C.</span></span> <span> </span><span class="NLM_article-title">A LOV2 Domain-Based Optogenetic Tool to Control Protein Degradation and Cellular Function</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2013.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=23601651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1Cmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=619-626&issue=4&author=C.+Renickeauthor=D.+Schusterauthor=S.+Usherenkoauthor=L.+O.+Essenauthor=C.+Taxis&title=A+LOV2+Domain-Based+Optogenetic+Tool+to+Control+Protein+Degradation+and+Cellular+Function&doi=10.1016%2Fj.chembiol.2013.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A LOV2 Domain-Based Optogenetic Tool to Control Protein Degradation and Cellular Function</span></div><div class="casAuthors">Renicke, Christian; Schuster, Daniel; Usherenko, Svetlana; Essen, Lars-Oliver; Taxis, Christof</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">619-626</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Light perception is indispensable for plants to respond adequately to external cues and is linked to proteolysis of key transcriptional regulators.  To provide synthetic light control of protein stability, we developed a generic photosensitive degron (psd) module combining the light-reactive LOV2 domain of Arabidopsis thaliana phot1 with the murine ornithine decarboxylase-like degrdn. sequence cODC1.  Functionality of the psd module was demonstrated in the model organism Saccharomyces cerevisiae.  Generation of conditional mutants, light regulation of cyclin-dependent kinase activity, light-based patterning of cell growth, and yeast photog. exemplified its versatility.  In silico modeling of psd module behavior increased understanding of its characteristics.  This engineered degron module transfers the principle of light-regulated degrdn. to nonplant organisms.  It will be highly beneficial to control protein levels in biotechnol. or biomedical applications and offers the potential to render a plethora of biol. processes light-switchable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXrMBGmwMVSrVg90H21EOLACvtfcHk0lid9gd55_idvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1Cmt7k%253D&md5=5e9dbad466b44431f060b30a83a904cd</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DRenicke%26aufirst%3DC.%26aulast%3DSchuster%26aufirst%3DD.%26aulast%3DUsherenko%26aufirst%3DS.%26aulast%3DEssen%26aufirst%3DL.%2BO.%26aulast%3DTaxis%26aufirst%3DC.%26atitle%3DA%2520LOV2%2520Domain-Based%2520Optogenetic%2520Tool%2520to%2520Control%2520Protein%2520Degradation%2520and%2520Cellular%2520Function%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26issue%3D4%26spage%3D619%26epage%3D626%26doi%3D10.1016%2Fj.chembiol.2013.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delacour, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plamont, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billon-Denis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aujard, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Saux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jullien, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautier, A.</span></span> <span> </span><span class="NLM_article-title">Light-Activated Proteolysis for the Spatiotemporal Control of Proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1643</span>– <span class="NLM_lpage">1647</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00069</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00069" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1Ght7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1643-1647&issue=7&author=Q.+Delacourauthor=C.+Liauthor=M.+A.+Plamontauthor=E.+Billon-Denisauthor=I.+Aujardauthor=T.+Le+Sauxauthor=L.+Jullienauthor=A.+Gautier&title=Light-Activated+Proteolysis+for+the+Spatiotemporal+Control+of+Proteins&doi=10.1021%2Facschembio.5b00069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Light-Activated Proteolysis for the Spatiotemporal Control of Proteins</span></div><div class="casAuthors">Delacour, Quentin; Li, Chenge; Plamont, Marie-Aude; Billon-Denis, Emmanuelle; Aujard, Isabelle; Le Saux, Thomas; Jullien, Ludovic; Gautier, Arnaud</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1643-1647</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The regulation of proteolysis is an efficient way to control protein function in cells.  Here, we present a general strategy enabling to increase the spatiotemporal resoln. of conditional proteolysis by using light activation as trigger.  Our approach relies on the auxin-inducible degrdn. system obtained by transposing components of the plant auxin-dependent degrdn. pathway in mammalian cells.  We developed a photoactivatable auxin that acts as a photoactivatable inducer of degrdn.  Upon local and short light illumination, auxin is released in cells and triggers the degrdn. of a protein of interest with spatiotemporal control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorsagFMqI1b7Vg90H21EOLACvtfcHk0lgON0VGXeUC4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1Ght7s%253D&md5=5da59aa63d7b7365d8a53d929070e9cc</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00069%26sid%3Dliteratum%253Aachs%26aulast%3DDelacour%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DPlamont%26aufirst%3DM.%2BA.%26aulast%3DBillon-Denis%26aufirst%3DE.%26aulast%3DAujard%26aufirst%3DI.%26aulast%3DLe%2BSaux%26aufirst%3DT.%26aulast%3DJullien%26aufirst%3DL.%26aulast%3DGautier%26aufirst%3DA.%26atitle%3DLight-Activated%2520Proteolysis%2520for%2520the%2520Spatiotemporal%2520Control%2520of%2520Proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D7%26spage%3D1643%26epage%3D1647%26doi%3D10.1021%2Facschembio.5b00069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xue, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z.</span></span> <span> </span><span class="NLM_article-title">Light-Induced Protein Degradation with Photocaged PROTACs</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span> (<span class="NLM_issue">46</span>),  <span class="NLM_fpage">18370</span>– <span class="NLM_lpage">18374</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b06422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b06422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVOmurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=18370-18374&issue=46&author=G.+Xueauthor=K.+Wangauthor=D.+Zhouauthor=H.+Zhongauthor=Z.+Pan&title=Light-Induced+Protein+Degradation+with+Photocaged+PROTACs&doi=10.1021%2Fjacs.9b06422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Light-Induced Protein Degradation with Photocaged PROTACs</span></div><div class="casAuthors">Xue, Gang; Wang, Kun; Zhou, Danli; Zhong, Hanbing; Pan, Zhengying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">18370-18374</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Induction of protein degrdn. is emerging as a powerful strategy to modulate protein functions and alter cellular signaling pathways.  Proteolysis-targeting chimeras (PROTACs) have been used to degrade a range of diverse proteins in vitro and in vivo.  Here we present a type of photo-caged PROTACs (pc-PROTACs) to induce degrdn. activity with light.  Photo-removable blocking groups were added to a degrader of Brd4, and the resulting mol. pc-PROTAC1 showed potent degrdn. activity in live cells only after light irradn.  Furthermore, this mol. efficiently degraded Brd4 and induced expected phenotypic changes in zebrafish.  Addnl., this approach was successfully applied to construct pc-PROTAC3 of BTK.  Thus, a general strategy to induce protein degrdn. with light was established to augment the chemists' toolbox to study disease-relevant protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYGnlMSsz2ubVg90H21EOLACvtfcHk0lgON0VGXeUC4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVOmurvJ&md5=75920669e7228755cfefb5650e4af6b0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b06422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b06422%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DZ.%26atitle%3DLight-Induced%2520Protein%2520Degradation%2520with%2520Photocaged%2520PROTACs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26issue%3D46%26spage%3D18370%26epage%3D18374%26doi%3D10.1021%2Fjacs.9b06422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darrah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Optical Control of Small Molecule-Induced Protein Degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">2193</span>– <span class="NLM_lpage">2197</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b12718</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b12718" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslOhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=2193-2197&issue=5&author=Y.+Naroauthor=K.+Darrahauthor=A.+Deiters&title=Optical+Control+of+Small+Molecule-Induced+Protein+Degradation&doi=10.1021%2Fjacs.9b12718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Optical Control of Small Molecule-Induced Protein Degradation</span></div><div class="casAuthors">Naro, Yuta; Darrah, Kristie; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2193-2197</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As an emerging approach to protein perturbation, small mol.-induced protein degrdn. has gained significant attention as both a chem. tool and a potential therapeutic.  To enable discrete control over its function, we have developed a broadly applicable approach for the optical activation of small mol.-induced protein degrdn.  By installing two different photolabile protecting groups, so-called caging groups, onto two different ligands recruiting Von Hippel-Lindau (VHL) and cereblon (CRBN) E3 ubiquitin ligases, our strategy enables light-triggered protein degrdn. for any small mol. warhead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7kp9dSOD7EbVg90H21EOLACvtfcHk0lgON0VGXeUC4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslOhsw%253D%253D&md5=ea8a4bfe78a8acd5fcb8d0bba85b2c02</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b12718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b12718%26sid%3Dliteratum%253Aachs%26aulast%3DNaro%26aufirst%3DY.%26aulast%3DDarrah%26aufirst%3DK.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DOptical%2520Control%2520of%2520Small%2520Molecule-Induced%2520Protein%2520Degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26issue%3D5%26spage%3D2193%26epage%3D2197%26doi%3D10.1021%2Fjacs.9b12718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kounde, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shchepinova, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muelbaier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rackham, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, E. W.</span></span> <span> </span><span class="NLM_article-title">A Caged E3 Ligase Ligand for PROTAC-Mediated Protein Degradation with Light</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">5532</span>– <span class="NLM_lpage">5535</span>, <span class="refDoi"> DOI: 10.1039/D0CC00523A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1039%2FD0CC00523A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32297626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmt1KitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2020&pages=5532-5535&issue=41&author=C.+S.+Koundeauthor=M.+M.+Shchepinovaauthor=C.+N.+Saundersauthor=M.+Muelbaierauthor=M.+D.+Rackhamauthor=J.+D.+Harlingauthor=E.+W.+Tate&title=A+Caged+E3+Ligase+Ligand+for+PROTAC-Mediated+Protein+Degradation+with+Light&doi=10.1039%2FD0CC00523A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A caged E3 ligase ligand for PROTAC-mediated protein degradation with light</span></div><div class="casAuthors">Kounde, Cyrille S.; Shchepinova, Maria M.; Saunders, Charlie N.; Muelbaier, Marcel; Rackham, Mark D.; Harling, John D.; Tate, Edward W.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">5532-5535</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">With the intent of achieving greater spatiotemporal control of PROTAC-induced protein degrdn., a light-activated degrader was designed by photocaging an essential E3 ligase binding motif in a BRD4 targeting PROTAC.  Proteolysis was triggered only after a short irradn. time, the kinetics of which could be monitored by live-cell video microscopy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8qIQehviok7Vg90H21EOLACvtfcHk0lgON0VGXeUC4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmt1KitrY%253D&md5=dc4bdede41675bf7f451a71e8aa4ee14</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1039%2FD0CC00523A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FD0CC00523A%26sid%3Dliteratum%253Aachs%26aulast%3DKounde%26aufirst%3DC.%2BS.%26aulast%3DShchepinova%26aufirst%3DM.%2BM.%26aulast%3DSaunders%26aufirst%3DC.%2BN.%26aulast%3DMuelbaier%26aufirst%3DM.%26aulast%3DRackham%26aufirst%3DM.%2BD.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DTate%26aufirst%3DE.%2BW.%26atitle%3DA%2520Caged%2520E3%2520Ligase%2520Ligand%2520for%2520PROTAC-Mediated%2520Protein%2520Degradation%2520with%2520Light%26jtitle%3DChem.%2520Commun.%26date%3D2020%26volume%3D56%26issue%3D41%26spage%3D5532%26epage%3D5535%26doi%3D10.1039%2FD0CC00523A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span> <span> </span><span class="NLM_article-title">Light-Induced Control of Protein Destruction by Opto-PROTAC</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">eaay5154</span>, <span class="refDoi"> DOI: 10.1126/sciadv.aay5154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1126%2Fsciadv.aay5154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32128407" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=eaay5154&issue=8&author=J.+Liuauthor=H.+Chenauthor=L.+Maauthor=Z.+Heauthor=D.+Wangauthor=Y.+Liuauthor=Q.+Linauthor=T.+Zhangauthor=N.+Grayauthor=H.+%C3%9C.+Kaniskanauthor=J.+Jinauthor=W.+Wei&title=Light-Induced+Control+of+Protein+Destruction+by+Opto-PROTAC&doi=10.1126%2Fsciadv.aay5154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aay5154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aay5154%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DW.%26atitle%3DLight-Induced%2520Control%2520of%2520Protein%2520Destruction%2520by%2520Opto-PROTAC%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26issue%3D8%26spage%3Deaay5154%26doi%3D10.1126%2Fsciadv.aay5154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pfaff, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carreira, E. M.</span></span> <span> </span><span class="NLM_article-title">Reversible Spatiotemporal Control of Induced Protein Degradation by Bistable PhotoPROTACs</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1682</span>– <span class="NLM_lpage">1690</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.9b00713</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.9b00713" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslOmsb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=1682-1690&issue=10&author=P.+Pfaffauthor=K.+T.+G.+Samarasingheauthor=C.+M.+Crewsauthor=E.+M.+Carreira&title=Reversible+Spatiotemporal+Control+of+Induced+Protein+Degradation+by+Bistable+PhotoPROTACs&doi=10.1021%2Facscentsci.9b00713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Spatiotemporal Control of Induced Protein Degradation by Bistable PhotoPROTACs</span></div><div class="casAuthors">Pfaff, Patrick; Samarasinghe, Kusal T. G.; Crews, Craig M.; Carreira, Erick M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1682-1690</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Off-tissue effects are persistent issues of modern inhibition-based therapies.  By merging the strategies of photopharmacol. and small-mol. degraders, we introduce a novel concept for persistent spatiotemporal control of induced protein degrdn. that potentially prevents off-tissue toxicity.  Building on the successful principle of bifunctional all-small-mol. Proteolysis Targeting Chimeras (PROTACs), we designed photoswitchable PROTACs (photoPROTACs) by including ortho-F4-azobenzene linkers between both warhead ligands.  This highly bistable yet photoswitchable structural component leads to reversible control over the topol. distance between both ligands.  The azo-cis-isomer is obsd. to be inactive because the distance defined by the linker is prohibitively short to permit complex formation between the protein binding partners.  By contrast, the azo-trans-isomer is active since it can engage both protein partners to form the necessary and productive ternary complex.  Importantly, due to the bistable nature of the ortho-F4-azobenzene moiety employed, the photostationary state of the photoPROTAC is persistent, with no need for continuous irradn.  This technique offers reversible on/off switching of protein degrdn. that is compatible with an intracellular environment and, therefore, could be useful in exptl. exploration of biol. signaling pathways-such as those crucial for oncogenic signal transduction.  Addnl., this strategy may be suitable for therapeutic intervention to address a variety of diseases.  By enabling reversible activation and deactivation of protein degrdn., photoPROTACs offer advantages over conventional photocaging strategies that irreversibly release active agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKaC13qAq9JrVg90H21EOLACvtfcHk0lh-b_qdDTWMuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslOmsb%252FP&md5=13d737b69314b763f9dc75e231b7fe70</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.9b00713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.9b00713%26sid%3Dliteratum%253Aachs%26aulast%3DPfaff%26aufirst%3DP.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DCarreira%26aufirst%3DE.%2BM.%26atitle%3DReversible%2520Spatiotemporal%2520Control%2520of%2520Induced%2520Protein%2520Degradation%2520by%2520Bistable%2520PhotoPROTACs%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2019%26volume%3D5%26issue%3D10%26spage%3D1682%26epage%3D1690%26doi%3D10.1021%2Facscentsci.9b00713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reynders, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuura, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bérouti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simoneschi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauner, D.</span></span> <span> </span><span class="NLM_article-title">PHOTACs Enable Optical Control of Protein Degradation</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">8</span>), <span class="NLM_elocation-id">eaay5064</span> <span class="refDoi"> DOI: 10.1126/sciadv.aay5064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1126%2Fsciadv.aay5064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32128406" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&issue=8&author=M.+Reyndersauthor=B.+S.+Matsuuraauthor=M.+B%C3%A9routiauthor=D.+Simoneschiauthor=A.+Marzioauthor=M.+Paganoauthor=D.+Trauner&title=PHOTACs+Enable+Optical+Control+of+Protein+Degradation&doi=10.1126%2Fsciadv.aay5064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aay5064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aay5064%26sid%3Dliteratum%253Aachs%26aulast%3DReynders%26aufirst%3DM.%26aulast%3DMatsuura%26aufirst%3DB.%2BS.%26aulast%3DB%25C3%25A9routi%26aufirst%3DM.%26aulast%3DSimoneschi%26aufirst%3DD.%26aulast%3DMarzio%26aufirst%3DA.%26aulast%3DPagano%26aufirst%3DM.%26aulast%3DTrauner%26aufirst%3DD.%26atitle%3DPHOTACs%2520Enable%2520Optical%2520Control%2520of%2520Protein%2520Degradation%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26issue%3D8%26doi%3D10.1126%2Fsciadv.aay5064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, W. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Azo-Protac: Novel Light-Controlled Small-Molecule Tool for Protein Knockdown</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4644</span>– <span class="NLM_lpage">4654</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b02058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b02058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFaju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4644-4654&issue=9&author=Y.+H.+Jinauthor=M.+C.+Luauthor=Y.+Wangauthor=W.+X.+Shanauthor=X.+Y.+Wangauthor=Q.+D.+Youauthor=Z.+Y.+Jiang&title=Azo-Protac%3A+Novel+Light-Controlled+Small-Molecule+Tool+for+Protein+Knockdown&doi=10.1021%2Facs.jmedchem.9b02058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Azo-PROTAC: Novel Light-Controlled Small-Molecule Tool for Protein Knockdown</span></div><div class="casAuthors">Jin, Yu-Hui; Lu, Meng-Chen; Wang, Yan; Shan, Wen-Xin; Wang, Xuan-Yu; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4644-4654</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Reversibly altering endogenous protein levels are persistent issues.  Herein, we designed photoswitchable azobenzene-proteolysis targeting chimeras (Azo-PROTACs) by including azobenzene moieties between ligands for the E3 ligase and the protein of interest.  Azo-PROTACs are light-controlled small-mol. tools for protein knockdown in cells.  The light-induced configuration change can switch the active state to induce protein degrdn. activity, which can be reversely controlled by light exposure in intact cells.  We compared the protein degrdn. abilities of Azo-PROTACs with different configurations and linker lengths.  Using the stable form with the best degrdn. ability against the BCR-ABL fusion and ABL proteins in myelogenous leukemia K562 cells, we showed that Azo-PROTAC combines the potent protein knockdown and facile cell uptake properties of the small-mol. PROTAC with a reversible photoswitchability, offering a promising chem. knockdown strategy based on the light-induced reversible on/off properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp1TuuF8Q3g7Vg90H21EOLACvtfcHk0lh-b_qdDTWMuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFaju7Y%253D&md5=0850ca7c222c5837b425b06487993563</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b02058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b02058%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DY.%2BH.%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DShan%26aufirst%3DW.%2BX.%26aulast%3DWang%26aufirst%3DX.%2BY.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26atitle%3DAzo-Protac%253A%2520Novel%2520Light-Controlled%2520Small-Molecule%2520Tool%2520for%2520Protein%2520Knockdown%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D9%26spage%3D4644%26epage%3D4654%26doi%3D10.1021%2Facs.jmedchem.9b02058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manna, D.</span></span> <span> </span><span class="NLM_article-title">Controlling PROTACs With Light</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1258</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1002/cmdc.202000249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1002%2Fcmdc.202000249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32558301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1yqsbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1258-1261&author=S.+Vermaauthor=D.+Manna&title=Controlling+PROTACs+With+Light&doi=10.1002%2Fcmdc.202000249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Controlling PROTACs with Light</span></div><div class="casAuthors">Verma, Shreya; Manna, Debasish</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1258-1261</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras, PROTACs, are emerging as a powerful strategy for exerting exogenous control over protein levels, allowing small mols. to exploit the ubiquitin-proteasome pathway for targeted protein degrdn.  This highlight focuses on the fusion of photochem. with these bifunctional compds., which has provided a novel pathway for spatiotemporally tuning the activation of PROTACs in the form of their photocaged and photoswitchable versions.  Photocaged PROTACs consist of a hindered optolabile group that detaches only upon irradn. at a specific wavelength, releasing the active PROTAC.  These modified PROTACs are inactive in the dark.  Photoswitchable PROTACs are photoisomerizable mols. with azobenzene linkages that are active in either the cis or trans form and inactive in the other.  The isomers interconvert upon irradn. with an appropriate wavelength of light and relax to the thermodynamically stable isomer in the dark or with another wavelength of light.  Although photocaged PROTACs only permit activation control for protein degrdn., photoswitching PROTACs offer reversible activation and deactivation by using suitable wavelengths of light.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9qDTGxuQEFLVg90H21EOLACvtfcHk0ljVf2dMzFQHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1yqsbrL&md5=560aa3946270ddeca6ea34692a41a7e8</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.202000249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.202000249%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DS.%26aulast%3DManna%26aufirst%3DD.%26atitle%3DControlling%2520PROTACs%2520With%2520Light%26jtitle%3DChemMedChem%26date%3D2020%26volume%3D15%26spage%3D1258%26epage%3D1261%26doi%3D10.1002%2Fcmdc.202000249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nalawansha, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROTACs: An Emerging Therapeutic Modality in Precision Medicine</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">998</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.07.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2020.07.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32795419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1Wrs7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=998-1014&author=D.+A.+Nalawanshaauthor=C.+M.+Crews&title=PROTACs%3A+An+Emerging+Therapeutic+Modality+in+Precision+Medicine&doi=10.1016%2Fj.chembiol.2020.07.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: An Emerging Therapeutic Modality in Precision Medicine</span></div><div class="casAuthors">Nalawansha, Dhanusha A.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">998-1014</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.,.  Targeted protein degrdn. (TPD) has emerged as an exciting new era in chem. biol. and drug discovery.  PROteolysis TArgeting Chimera (PROTAC) technol. targets cellular proteins for degrdn. by co-opting the ubiquitin-proteasome system.  Over the last 5 years, numerous studies have expanded our understanding of the unique mode of action and advantages of PROTACs, which has in turn spurred interest in both academia and industry to explore PROTACs as a novel therapeutic strategy.  In this review, we first highlight the key advantages of PROTACs and then discuss the spatiotemporal regulation of protein degrdn.  Next, we explore current chem. tractable E3 ligases focusing on expanding the existing repertoire with novel E3 ligases to uncover the full potential of TPD.  Collectively, these studies are guiding the development of the PROTAC technol. as it emerges as a new modality in precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4GkJzvvYnp7Vg90H21EOLACvtfcHk0ljVf2dMzFQHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Wrs7vM&md5=63c4eb78ee4fc9272b9ae2409889861e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.07.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.07.020%26sid%3Dliteratum%253Aachs%26aulast%3DNalawansha%26aufirst%3DD.%2BA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROTACs%253A%2520An%2520Emerging%2520Therapeutic%2520Modality%2520in%2520Precision%2520Medicine%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D998%26epage%3D1014%26doi%3D10.1016%2Fj.chembiol.2020.07.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckel, A.</span></span> <span> </span><span class="NLM_article-title">Biologically Active Molecules with a “Light Switch”</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">4900</span>– <span class="NLM_lpage">4921</span>, <span class="refDoi"> DOI: 10.1002/anie.200600387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1002%2Fanie.200600387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvVaqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=4900-4921&issue=30&author=G.+Mayerauthor=A.+Heckel&title=Biologically+Active+Molecules+with+a+%E2%80%9CLight+Switch%E2%80%9D&doi=10.1002%2Fanie.200600387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Biologically active molecules with a "Light switch"</span></div><div class="casAuthors">Mayer, Guenter; Heckel, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">4900-4921</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Biol. active compds. which are light-responsive offer exptl. possibilities which are otherwise very difficult to achieve.  Since light can be manipulated very precisely, for example, with lasers and microscopes rapid jumps in concn. of the active form of mols. are possible with exact control of the area, time, and dosage.  The development of such strategies started in the 1970s.  This review summarizes new developments of the last five years and deals with "small mols.", proteins, and nucleic acids which can either be irreversibly activated with light (these compds. are referred to as "caged compds.") or reversibly switched between an active and an inactive state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA75s7az7307Vg90H21EOLACvtfcHk0ljVf2dMzFQHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvVaqsrg%253D&md5=bf95a5ccf1a6d75509b6d2ae5f75e0c8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fanie.200600387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200600387%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DHeckel%26aufirst%3DA.%26atitle%3DBiologically%2520Active%2520Molecules%2520with%2520a%2520%25E2%2580%259CLight%2520Switch%25E2%2580%259D%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26issue%3D30%26spage%3D4900%26epage%3D4921%26doi%3D10.1002%2Fanie.200600387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faust, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstock, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Bruton Tyrosine Kinase Degradation as a Therapeutic Strategy for Cancer</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>133</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">952</span>– <span class="NLM_lpage">961</span>, <span class="refDoi"> DOI: 10.1182/blood-2018-07-862953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1182%2Fblood-2018-07-862953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=30545835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXotlGjtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2019&pages=952-961&issue=9&author=D.+Dobrovolskyauthor=E.+S.+Wangauthor=S.+Morrowauthor=C.+Leahyauthor=T.+Faustauthor=R.+P.+Nowakauthor=K.+A.+Donovanauthor=G.+Yangauthor=Z.+Liauthor=E.+S.+Fischerauthor=S.+P.+Treonauthor=D.+M.+Weinstockauthor=N.+S.+Gray&title=Bruton+Tyrosine+Kinase+Degradation+as+a+Therapeutic+Strategy+for+Cancer&doi=10.1182%2Fblood-2018-07-862953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Bruton tyrosine kinase degradation as a therapeutic strategy for cancer</span></div><div class="casAuthors">Dobrovolsky, Dennis; Wang, Eric S.; Morrow, Sara; Leahy, Catharine; Faust, Tyler; Nowak, Radoslaw P.; Donovan, Katherine A.; Yang, Guang; Li, Zhengnian; Fischer, Eric S.; Treon, Steven P.; Weinstock, David M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">952-961</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The covalent Bruton tyrosine kinase (BTK) inhibitor ibrutinib is highly efficacious against multiple B-cell malignancies.  However, it is not selective for BTK, and multiple mechanisms of resistance, including the C481S-BTK mutation, can compromise its efficacy.  We hypothesized that small-mol.-induced BTK degrdn. may overcome some of the limitations of traditional enzymic inhibitors.  Here, we demonstrate that BTK degrdn. results in potent suppression of signaling and proliferation in cancer cells and that BTK degraders efficiently degrade C481S-BTK.  Moreover, we discovered DD-03-171, an optimized lead compd. that exhibits enhanced antiproliferative effects on mantle cell lymphoma (MCL) cells in vitro by degrading BTK, IKFZ1, and IKFZ3 as well as efficacy against patient-derived xenografts in vivo.  Thus, "triple degrdn." may be an effective therapeutic approach for treating MCL and overcoming ibrutinib resistance, thereby addressing a major unmet need in the treatment of MCL and other B-cell lymphomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN4GLppunGArVg90H21EOLACvtfcHk0ljVf2dMzFQHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXotlGjtrY%253D&md5=ce7e48897378c2821d728daf6d9935d5</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1182%2Fblood-2018-07-862953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2018-07-862953%26sid%3Dliteratum%253Aachs%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DMorrow%26aufirst%3DS.%26aulast%3DLeahy%26aufirst%3DC.%26aulast%3DFaust%26aufirst%3DT.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DWeinstock%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DBruton%2520Tyrosine%2520Kinase%2520Degradation%2520as%2520a%2520Therapeutic%2520Strategy%2520for%2520Cancer%26jtitle%3DBlood%26date%3D2019%26volume%3D133%26issue%3D9%26spage%3D952%26epage%3D961%26doi%3D10.1182%2Fblood-2018-07-862953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide Conjugation as a Strategy for in Vivo Target Protein Degradation</span>. <i>Science (Washington, DC, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span> (<span class="NLM_issue">6241</span>),  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&issue=6241&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+Conjugation+as+a+Strategy+for+in+Vivo+Target+Protein+Degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lheuSaxreOKjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520Conjugation%2520as%2520a%2520Strategy%2520for%2520in%2520Vivo%2520Target%2520Protein%2520Degradation%26jtitle%3DScience%2520%2528Washington%252C%2520DC%252C%2520U.%2520S.%2529%26date%3D2015%26volume%3D348%26issue%3D6241%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parapatics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecker, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00969</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00969" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=553-560&issue=3&author=M.+Ishoeyauthor=S.+Chornauthor=N.+Singhauthor=M.+G.+Jaegerauthor=M.+Brandauthor=J.+Paulkauthor=S.+Bauerauthor=M.+A.+Erbauthor=K.+Parapaticsauthor=A.+C.+M%C3%BCllerauthor=K.+L.+Bennettauthor=G.+F.+Eckerauthor=J.+E.+Bradnerauthor=G.+E.+Winter&title=Translation+Termination+Factor+GSPT1+Is+a+Phenotypically+Relevant+Off-Target+of+Heterobifunctional+Phthalimide+Degraders&doi=10.1021%2Facschembio.7b00969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span></div><div class="casAuthors">Ishoey, Mette; Chorn, Someth; Singh, Natesh; Jaeger, Martin G.; Brand, Matthias; Paulk, Joshiawa; Bauer, Sophie; Erb, Michael A.; Parapatics, Katja; Muller, Andre C.; Bennett, Keiryn L.; Ecker, Gerhard F.; Bradner, James E.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">553-560</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein degrdn. is an emerging therapeutic strategy with a unique mol. pharmacol. that enables the disruption of all functions assocd. with a target.  This is particularly relevant for proteins depending on mol. scaffolding, such as transcription factors or receptor tyrosine kinases (RTKs).  To address tractability of multiple RTKs for chem. degrdn. by the E3 ligase CUL4-RBX1-DDB1-CRBN (CRL4CRBN), we synthesized a series of phthalimide degraders based on the promiscuous kinase inhibitors sunitinib and PHA665752.  While both series failed to induce degrdn. of their consensus targets, individual mols. displayed pronounced efficacy in leukemia cell lines.  Orthogonal target identification supported by mol. docking led us to identify the translation termination factor G1 to S phase transition 1 (GSPT1) as a converging off-target, resulting from inadvertent E3 ligase modulation.  This research highlights the importance of monitoring degrdn. events that are independent of the resp. targeting ligand as a unique feature of small-mol. degraders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_ZxnB5rKvvLVg90H21EOLACvtfcHk0lheuSaxreOKjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D&md5=02a70959e8a8929a3c70846c1ed0200f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00969%26sid%3Dliteratum%253Aachs%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DChorn%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DJaeger%26aufirst%3DM.%2BG.%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DParapatics%26aufirst%3DK.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DTranslation%2520Termination%2520Factor%2520GSPT1%2520Is%2520a%2520Phenotypically%2520Relevant%2520Off-Target%2520of%2520Heterobifunctional%2520Phthalimide%2520Degraders%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D3%26spage%3D553%26epage%3D560%26doi%3D10.1021%2Facschembio.7b00969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&issue=8&author=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Ciulli&title=Selective+Small+Molecule+Induced+Degradation+of+the+BET+Bromodomain+Protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lheuSaxreOKjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520Small%2520Molecule%2520Induced%2520Degradation%2520of%2520the%2520BET%2520Bromodomain%2520Protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D8%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donoghue, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cubillos-Rojas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez-Prat, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Zarzalejo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdaguer, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nebreda, A. R.</span></span> <span> </span><span class="NLM_article-title">Optimal Linker Length for Small Molecule PROTACs That Selectively Target P38α and P38β for Degradation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>201</i></span>,  <span class="NLM_fpage">112451</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.ejmech.2020.112451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=32634680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlais73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=2020&pages=112451&author=C.+Donoghueauthor=M.+Cubillos-Rojasauthor=N.+Gutierrez-Pratauthor=C.+Sanchez-Zarzalejoauthor=X.+Verdaguerauthor=A.+Rieraauthor=A.+R.+Nebreda&title=Optimal+Linker+Length+for+Small+Molecule+PROTACs+That+Selectively+Target+P38%CE%B1+and+P38%CE%B2+for+Degradation&doi=10.1016%2Fj.ejmech.2020.112451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Optimal linker length for small molecule PROTACs that selectively target p38α and p38β for degradation</span></div><div class="casAuthors">Donoghue, Craig; Cubillos-Rojas, Monica; Gutierrez-Prat, Nuria; Sanchez-Zarzalejo, Carolina; Verdaguer, Xavier; Riera, Antoni; Nebreda, Angel R.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112451</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">We report the design of hetero-bifunctional small mols. that selectively target p38α and p38β for degrdn.  These proteolysis targeted chimeras (PROTACs) are based on an ATP competitive inhibitor of p38α and p38β, which is linked to thalidomide analogs to recruit the Cereblon E3 ubiquitin ligase complex.  Compd. synthesis was facilitated by the use of a copper catalyzed "click" reaction.  We show that optimization of the linker length and compn. is crucial for the degrdn.-inducing activity of these PROTACs.  We provide evidence that these chem. compds. can induce degrdn. of p38α and p38β but no other related kinases at nanomolar concns. in several mammalian cell lines.  Accordingly, the PROTACs inhibit stress and cytokine-induced p38α signaling.  Our compds. contribute to understanding the development of PROTACs, and provide a useful tool to investigate functions of the p38 MAPK pathway and its involvement in diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU67xl-ZkKDbVg90H21EOLACvtfcHk0liu5vDhAIWCrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlais73E&md5=612294e7cc0af1d535e24219bc03de2c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112451%26sid%3Dliteratum%253Aachs%26aulast%3DDonoghue%26aufirst%3DC.%26aulast%3DCubillos-Rojas%26aufirst%3DM.%26aulast%3DGutierrez-Prat%26aufirst%3DN.%26aulast%3DSanchez-Zarzalejo%26aufirst%3DC.%26aulast%3DVerdaguer%26aufirst%3DX.%26aulast%3DRiera%26aufirst%3DA.%26aulast%3DNebreda%26aufirst%3DA.%2BR.%26atitle%3DOptimal%2520Linker%2520Length%2520for%2520Small%2520Molecule%2520PROTACs%2520That%2520Selectively%2520Target%2520P38%25CE%25B1%2520and%2520P38%25CE%25B2%2520for%2520Degradation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D201%26spage%3D112451%26doi%3D10.1016%2Fj.ejmech.2020.112451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klán, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">šolomek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bochet, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Givens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popik, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostikov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirz, J.</span></span> <span> </span><span class="NLM_article-title">Photoremovable Protecting Groups in Chemistry and Biology: Reaction Mechanisms and Efficacy</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1021/cr300177k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr300177k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVyiurvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2013&pages=119-191&issue=1&author=P.+Kl%C3%A1nauthor=T.+%C5%A1olomekauthor=C.+G.+Bochetauthor=A.+Blancauthor=R.+Givensauthor=M.+Rubinaauthor=V.+Popikauthor=A.+Kostikovauthor=J.+Wirz&title=Photoremovable+Protecting+Groups+in+Chemistry+and+Biology%3A+Reaction+Mechanisms+and+Efficacy&doi=10.1021%2Fcr300177k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Photoremovable Protecting Groups in Chemistry and Biology: Reaction Mechanisms and Efficacy</span></div><div class="casAuthors">Klan, Petr; Solomek, Tomas; Bochet, Christian G.; Blanc, Aurelien; Givens, Richard; Rubina, Marina; Popik, Vladimir; Kostikov, Alexey; Wirz, Jakob</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">119-191</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Photoremovable (sometimes called photoreleasable, photocleavable or photoactivatable) protecting groups (PPGs) provide spatial and temporal control over the release of various chems. such as bioagents (neurotransmitters and cell-signaling mols.), acids, bases, Ca2+ ions, oxidants, insecticides, phermones, fragrances, etc.  The present review covers recent developments in the field, focusing on the scope, limitations, and applications of PPGs, which are used to release org. mols.  Simplified basic structures of PPGs discussed in this review are provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXac8Ccb-fmLVg90H21EOLACvtfcHk0liu5vDhAIWCrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVyiurvL&md5=b8defe84c6c629e44d429f82fa4eb5f0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fcr300177k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr300177k%26sid%3Dliteratum%253Aachs%26aulast%3DKl%25C3%25A1n%26aufirst%3DP.%26aulast%3D%25C5%25A1olomek%26aufirst%3DT.%26aulast%3DBochet%26aufirst%3DC.%2BG.%26aulast%3DBlanc%26aufirst%3DA.%26aulast%3DGivens%26aufirst%3DR.%26aulast%3DRubina%26aufirst%3DM.%26aulast%3DPopik%26aufirst%3DV.%26aulast%3DKostikov%26aufirst%3DA.%26aulast%3DWirz%26aufirst%3DJ.%26atitle%3DPhotoremovable%2520Protecting%2520Groups%2520in%2520Chemistry%2520and%2520Biology%253A%2520Reaction%2520Mechanisms%2520and%2520Efficacy%26jtitle%3DChem.%2520Rev.%26date%3D2013%26volume%3D113%26issue%3D1%26spage%3D119%26epage%3D191%26doi%3D10.1021%2Fcr300177k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTAC Degraders of the Bromodomain and Extra Terminal (BET) Proteins — A Review</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.ddtec.2019.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31200858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=43-51&author=C.+Y.+Yangauthor=C.+Qinauthor=L.+Baiauthor=S.+Wang&title=Small-Molecule+PROTAC+Degraders+of+the+Bromodomain+and+Extra+Terminal+%28BET%29+Proteins+%E2%80%94+A+Review&doi=10.1016%2Fj.ddtec.2019.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule PROTAC degraders of the Bromodomain and Extra Terminal (BET) proteins - A review</span></div><div class="casAuthors">Yang Chao-Yie; Qin Chong; Bai Longchuan; Wang Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43-51</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The PROteolysis TArgeting Chimeric (PROTAC) concept has provided an opportunity for the discovery and development of a completely new type of therapy involving induction of protein degradation.  The BET proteins, comprised of BRD2, BRD3, BRD4 and the testis-specific BRDT protein, are epigenetic readers and master transcription coactivators.  Extremely potent and efficacious small-molecule PROTAC degraders of the BET proteins, based on available, potent and selective BET inhibitors, have been reported.  BET degraders differ from BET inhibitors in their cellular potency, phenotypic effects, pharmacokinetic properties and toxicity profiles.  Herein, we provide a review of BET degraders and the differential outcome observed in the cellular and animal models for BET degraders in comparison to BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoQvupmecNw_lfW6udTcc2ea3WaQJSuIBjbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D&md5=de5790c1db28d56717ff76374072ad81</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSmall-Molecule%2520PROTAC%2520Degraders%2520of%2520the%2520Bromodomain%2520and%2520Extra%2520Terminal%2520%2528BET%2529%2520Proteins%2520%25E2%2580%2594%2520A%2520Review%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D43%26epage%3D51%26doi%3D10.1016%2Fj.ddtec.2019.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappaport, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chicas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">RNAi Screen Identifies Brd4 as a Therapeutic Target in Acute Myeloid Leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span> (<span class="NLM_issue">7370</span>),  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&issue=7370&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+Screen+Identifies+Brd4+as+a+Therapeutic+Target+in+Acute+Myeloid+Leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0lg7_pzOC785Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520Screen%2520Identifies%2520Brd4%2520as%2520a%2520Therapeutic%2520Target%2520in%2520Acute%2520Myeloid%2520Leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26issue%3D7370%26spage%3D524%26epage%3D528%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&issue=6&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+Ubiquitin+Ligase+Cereblon+to+Efficiently+Target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lg7_pzOC785Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520Ubiquitin%2520Ligase%2520Cereblon%2520to%2520Efficiently%2520Target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26issue%3D6%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noblejas-López, M. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieto-Jimenez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Juárez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montero, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esparís-Ogando, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandiella, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galán-Moya, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ocaña, A.</span></span> <span> </span><span class="NLM_article-title">Activity of BET-Proteolysis Targeting Chimeric (PROTAC) Compounds in Triple Negative Breast Cancer</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">383</span>, <span class="refDoi"> DOI: 10.1186/s13046-019-1387-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1186%2Fs13046-019-1387-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31470872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A280%3ADC%252BB3MrjslKksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=383&issue=1&author=M.+D.+M.+Noblejas-L%C3%B3pezauthor=C.+Nieto-Jimenezauthor=M.+Burgosauthor=M.+G%C3%B3mez-Ju%C3%A1rezauthor=J.+C.+Monteroauthor=A.+Espar%C3%ADs-Ogandoauthor=A.+Pandiellaauthor=E.+M.+Gal%C3%A1n-Moyaauthor=A.+Oca%C3%B1a&title=Activity+of+BET-Proteolysis+Targeting+Chimeric+%28PROTAC%29+Compounds+in+Triple+Negative+Breast+Cancer&doi=10.1186%2Fs13046-019-1387-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Activity of BET-proteolysis targeting chimeric (PROTAC) compounds in triple negative breast cancer</span></div><div class="casAuthors">Noblejas-Lopez Maria Del Mar; Nieto-Jimenez Cristina; Burgos Miguel; Gomez-Juarez Monica; Ocana Alberto; Noblejas-Lopez Maria Del Mar; Nieto-Jimenez Cristina; Galan-Moya Eva M; Ocana Alberto; Montero Juan Carlos; Esparis-Ogando Azucena; Pandiella Atanasio; Montero Juan Carlos; Esparis-Ogando Azucena; Pandiella Atanasio; Montero Juan Carlos; Esparis-Ogando Azucena; Pandiella Atanasio; Pandiella Atanasio; Ocana Alberto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of experimental & clinical cancer research : CR</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">383</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Triple negative breast cancer (TNBC) is an incurable disease where novel therapeutic strategies are needed.  Proteolysis targeting chimeric (PROTAC) are novel compounds that promote protein degradation by binding to an ubiquitin ligase.  In this work, we explored the antitumoral activity of two novel BET-PROTACs, MZ1 and ARV-825, in TNBC, ovarian cancer and in a BET inhibitor resistant model.  METHODS:  OVCAR3, SKOV3, BT549, MDA-MB-231 cell lines and the JQ1 resistant cell line MDA-MB-231R were evaluated.  MTTs, colony-forming assay, three-dimensional cultures in matrigel, flow cytometry, and western blots were performed to explore the anti-proliferative effect and biochemical mechanism of action of MZ1 and ARV-825.  In vivo studies included BALB/c nu/nu mice engrafted with MDA-MB-231R cells.  RESULTS:  The BET-PROTACs MZ1 and ARV-825 efficiently downregulated the protein expression levels of the BET protein BRD4, in MDA-MB-231 and MDA-MB-231R.  MZ1 and ARV-825 also showed an antiproliferative effect on sensitive and resistant cells.  This effect was corroborated in other triple negative (BT549) and ovarian cancer (SKOV3, OVCAR3) cell lines.  MZ1 provoked G2/M arrest in MDA-MB-231.  In addition, a profound effect on caspase-dependent apoptosis was observed in both sensitive and resistant cells.  No synergistic activity was observed when it was combined with docetaxel, cisplatin or olaparib.  Finally, in vivo administration of MZ1 rescued tumor growth in a JQ1-resistant xenograft model, reducing the expression levels of BRD4.  CONCLUSIONS:  Using both in vitro and in vivo approaches, we describe the profound activity of BET-PROTACs in parental and BETi-resistant TNBC models.  This data provides options for further clinical development of these agents in TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQfKRCiECTUNcrRqEXsZufAfW6udTcc2eb9v39CYGOQ8bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MrjslKksw%253D%253D&md5=b6ded6e32e4d1820dcb3e51c75de3f5c</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1186%2Fs13046-019-1387-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-019-1387-5%26sid%3Dliteratum%253Aachs%26aulast%3DNoblejas-L%25C3%25B3pez%26aufirst%3DM.%2BD.%2BM.%26aulast%3DNieto-Jimenez%26aufirst%3DC.%26aulast%3DBurgos%26aufirst%3DM.%26aulast%3DG%25C3%25B3mez-Ju%25C3%25A1rez%26aufirst%3DM.%26aulast%3DMontero%26aufirst%3DJ.%2BC.%26aulast%3DEspar%25C3%25ADs-Ogando%26aufirst%3DA.%26aulast%3DPandiella%26aufirst%3DA.%26aulast%3DGal%25C3%25A1n-Moya%26aufirst%3DE.%2BM.%26aulast%3DOca%25C3%25B1a%26aufirst%3DA.%26atitle%3DActivity%2520of%2520BET-Proteolysis%2520Targeting%2520Chimeric%2520%2528PROTAC%2529%2520Compounds%2520in%2520Triple%2520Negative%2520Breast%2520Cancer%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2019%26volume%3D38%26issue%3D1%26spage%3D383%26doi%3D10.1186%2Fs13046-019-1387-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiskus, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapakshe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saenz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mill, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoury, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhalla, K. N.</span></span> <span> </span><span class="NLM_article-title">BET Protein Proteolysis Targeting Chimera (PROTAC) Exerts Potent Lethal Activity against Mantle Cell Lymphoma Cells</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fleu.2017.207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=28663582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1eqt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=343-352&issue=2&author=B.+Sunauthor=W.+Fiskusauthor=Y.+Qianauthor=K.+Rajapaksheauthor=K.+Rainaauthor=K.+G.+Colemanauthor=A.+P.+Crewauthor=A.+Shenauthor=D.+T.+Saenzauthor=C.+P.+Millauthor=A.+J.+Nowakauthor=N.+Jainauthor=L.+Zhangauthor=M.+Wangauthor=J.+D.+Khouryauthor=C.+Coarfaauthor=C.+M.+Crewsauthor=K.+N.+Bhalla&title=BET+Protein+Proteolysis+Targeting+Chimera+%28PROTAC%29+Exerts+Potent+Lethal+Activity+against+Mantle+Cell+Lymphoma+Cells&doi=10.1038%2Fleu.2017.207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Bet protein proteolysis targeting chimera (protac) exerts potent lethal activity against mantle cell lymphoma cells</span></div><div class="casAuthors">Sun, B.; Fiskus, W.; Qian, Y.; Rajapakshe, K.; Raina, K.; Coleman, K. G.; Crew, A. P.; Shen, A.; Saenz, D. T.; Mill, C. P.; Nowak, A. J.; Jain, N.; Zhang, L.; Wang, M.; Khoury, J. D.; Coarfa, C.; Crews, C. M.; Bhalla, K. N.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">343-352</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Bromodomain extraterminal protein (BETP) inhibitors transcriptionally repress oncoproteins and nuclear factor-κB (NF-κB) target genes that undermines the growth and survival of mantle cell lymphoma (MCL) cells.  However, BET bromodomain inhibitor (BETi) treatment causes accumulation of BETPs, assocd. with reversible binding and incomplete inhibition of BRD4 that potentially compromises the activity of BETi in MCL cells.  Unlike BETi, BET-PROTACs (proteolysis-targeting chimera) ARV-825 and ARV-771 (Arvinas, Inc.) recruit and utilize an E3-ubiquitin ligase to effectively degrade BETPs in MCL cells.  BET-PROTACs induce more apoptosis than BETi of MCL cells, including those resistant to ibrutinib.  BET-PROTAC treatment induced more perturbations in the mRNA and protein expressions than BETi, with depletion of c-Myc, CDK4, cyclin D1 and the NF-κB transcriptional targets Bcl-xL, XIAP and BTK, while inducing the levels of HEXIM1, NOXA and CDKN1A/p21.  Treatment with ARV-771, which possesses superior pharmacol. properties compared with ARV-825, inhibited the in vivo growth and induced greater survival improvement than the BETi OTX015 of immune-depleted mice engrafted with MCL cells.  Cotreatment of ARV-771 with ibrutinib or the BCL2 antagonist venetoclax or CDK4/6 inhibitor palbociclib synergistically induced apoptosis of MCL cells.  These studies highlight promising and superior preclin. activity of BET-PROTAC than BETi, requiring further in vivo evaluation of BET-PROTAC as a therapy for ibrutinib-sensitive or -resistant MCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAZxihoDZILrVg90H21EOLACvtfcHk0lipboo0tMoYFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1eqt77J&md5=f771eee574daf11546b93866ea314381</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.207%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DB.%26aulast%3DFiskus%26aufirst%3DW.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DRajapakshe%26aufirst%3DK.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DShen%26aufirst%3DA.%26aulast%3DSaenz%26aufirst%3DD.%2BT.%26aulast%3DMill%26aufirst%3DC.%2BP.%26aulast%3DNowak%26aufirst%3DA.%2BJ.%26aulast%3DJain%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DKhoury%26aufirst%3DJ.%2BD.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DBhalla%26aufirst%3DK.%2BN.%26atitle%3DBET%2520Protein%2520Proteolysis%2520Targeting%2520Chimera%2520%2528PROTAC%2529%2520Exerts%2520Potent%2520Lethal%2520Activity%2520against%2520Mantle%2520Cell%2520Lymphoma%2520Cells%26jtitle%3DLeukemia%26date%3D2018%26volume%3D32%26issue%3D2%26spage%3D343%26epage%3D352%26doi%3D10.1038%2Fleu.2017.207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cassar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adatto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamse, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iturria, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muriana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Cruchten, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zon, L. I.</span></span> <span> </span><span class="NLM_article-title">Use of Zebrafish in Drug Discovery Toxicology</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">118</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrestox.9b00335</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrestox.9b00335" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFOrurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2020&pages=95-118&issue=1&author=S.+Cassarauthor=I.+Adattoauthor=J.+L.+Freemanauthor=J.+T.+Gamseauthor=I.+Iturriaauthor=C.+Lawrenceauthor=A.+Murianaauthor=R.+T.+Petersonauthor=S.+Van+Cruchtenauthor=L.+I.+Zon&title=Use+of+Zebrafish+in+Drug+Discovery+Toxicology&doi=10.1021%2Facs.chemrestox.9b00335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Zebrafish in Drug Discovery Toxicology</span></div><div class="casAuthors">Cassar, Steven; Adatto, Isaac; Freeman, Jennifer L.; Gamse, Joshua T.; Iturria, Inaki; Lawrence, Christian; Muriana, Arantza; Peterson, Randall T.; Van Cruchten, Steven; Zon, Leonard I.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-118</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unpredicted human safety events in clin. trials for new drugs are costly in terms of human health and money.  The drug discovery industry attempts to minimize those events with diligent preclin. safety testing.  Current std. practices are good at preventing toxic compds. from being tested in the clinic; however, false neg. preclin. toxicity results are still a reality.  Continual improvement must be pursued in the preclin. realm.  Higher quality therapies can be brought forward with more information about potential toxicities and assocd. mechanisms.  The zebrafish model is a bridge between in vitro assays and mammalian in vivo studies.  This model is powerful in its breadth of application and tractability for research.  In the past two decades our understanding of disease biol. and drug toxicity has grown significantly owing to thousands of studies on this tiny vertebrate.  This review summarizes challenges and strengths of the model, discusses the 3Rs value that it can deliver, highlights translatable and untranslatable biol., and brings together reports from recent studies with zebrafish focusing on new drug discovery toxicol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7bTRPKrZR37Vg90H21EOLACvtfcHk0lipboo0tMoYFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFOrurjI&md5=cca6ff69b1d1feb25bc639a3523118a2</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrestox.9b00335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrestox.9b00335%26sid%3Dliteratum%253Aachs%26aulast%3DCassar%26aufirst%3DS.%26aulast%3DAdatto%26aufirst%3DI.%26aulast%3DFreeman%26aufirst%3DJ.%2BL.%26aulast%3DGamse%26aufirst%3DJ.%2BT.%26aulast%3DIturria%26aufirst%3DI.%26aulast%3DLawrence%26aufirst%3DC.%26aulast%3DMuriana%26aufirst%3DA.%26aulast%3DPeterson%26aufirst%3DR.%2BT.%26aulast%3DVan%2BCruchten%26aufirst%3DS.%26aulast%3DZon%26aufirst%3DL.%2BI.%26atitle%3DUse%2520of%2520Zebrafish%2520in%2520Drug%2520Discovery%2520Toxicology%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2020%26volume%3D33%26issue%3D1%26spage%3D95%26epage%3D118%26doi%3D10.1021%2Facs.chemrestox.9b00335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katz, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behrmann, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plas, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alayev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holz, M. K.</span></span> <span> </span><span class="NLM_article-title">ERRα Regulates the Growth of Triple-Negative Breast Cancer Cells via S6K1-Dependent Mechanism</span>. <i>Signal Transduct. Target. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">17035</span>, <span class="refDoi"> DOI: 10.1038/sigtrans.2017.35</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fsigtrans.2017.35" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=17035&author=A.+Y.+Bermanauthor=S.+Mannaauthor=N.+S.+Schwartzauthor=Y.+E.+Katzauthor=Y.+Sunauthor=C.+A.+Behrmannauthor=J.+J.+Yuauthor=D.+R.+Plasauthor=A.+Alayevauthor=M.+K.+Holz&title=ERR%CE%B1+Regulates+the+Growth+of+Triple-Negative+Breast+Cancer+Cells+via+S6K1-Dependent+Mechanism&doi=10.1038%2Fsigtrans.2017.35"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fsigtrans.2017.35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsigtrans.2017.35%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DA.%2BY.%26aulast%3DManna%26aufirst%3DS.%26aulast%3DSchwartz%26aufirst%3DN.%2BS.%26aulast%3DKatz%26aufirst%3DY.%2BE.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DBehrmann%26aufirst%3DC.%2BA.%26aulast%3DYu%26aufirst%3DJ.%2BJ.%26aulast%3DPlas%26aufirst%3DD.%2BR.%26aulast%3DAlayev%26aufirst%3DA.%26aulast%3DHolz%26aufirst%3DM.%2BK.%26atitle%3DERR%25CE%25B1%2520Regulates%2520the%2520Growth%2520of%2520Triple-Negative%2520Breast%2520Cancer%2520Cells%2520via%2520S6K1-Dependent%2520Mechanism%26jtitle%3DSignal%2520Transduct.%2520Target.%2520Ther.%26date%3D2017%26volume%3D2%26spage%3D17035%26doi%3D10.1038%2Fsigtrans.2017.35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendjennat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kizhake, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, A.</span></span> <span> </span><span class="NLM_article-title">Selective Degradation of CDK6 by a Palbociclib Based PROTAC</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1375</span>– <span class="NLM_lpage">1379</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.03.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.bmcl.2019.03.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=30935795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtlOrt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1375-1379&issue=11&author=S.+Ranaauthor=M.+Bendjennatauthor=S.+Kourauthor=H.+M.+Kingauthor=S.+Kizhakeauthor=M.+Zahidauthor=A.+Natarajan&title=Selective+Degradation+of+CDK6+by+a+Palbociclib+Based+PROTAC&doi=10.1016%2Fj.bmcl.2019.03.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Selective degradation of CDK6 by a palbociclib based PROTAC</span></div><div class="casAuthors">Rana, Sandeep; Bendjennat, Mourad; Kour, Smit; King, Hannah M.; Kizhake, Smitha; Zahid, Muhammad; Natarajan, Amarnath</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1375-1379</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Development of selective kinase inhibitors that target the ATP binding site continues to be a challenge largely due to similar binding pockets.  Palbociclib is a cyclin-dependent kinase inhibitor that targets the ATP binding site of CDK4 and CDK6 with similar potency.  The enzymic function assocd. with the kinase can be effectively probed using kinase inhibitors however the kinase-independent functions cannot.  Herein, we report a palbociclib based PROTAC that selectively degrades CDK6 while sparing the homolog CDK4.  We used competition studies to characterize the binding and mechanism of CDK6 degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrboIMRvD268rVg90H21EOLACvtfcHk0lhNwORJwtzJjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtlOrt7o%253D&md5=c9c561340e6c80ce51b747b84837ce46</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.03.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.03.035%26sid%3Dliteratum%253Aachs%26aulast%3DRana%26aufirst%3DS.%26aulast%3DBendjennat%26aufirst%3DM.%26aulast%3DKour%26aufirst%3DS.%26aulast%3DKing%26aufirst%3DH.%2BM.%26aulast%3DKizhake%26aufirst%3DS.%26aulast%3DZahid%26aufirst%3DM.%26aulast%3DNatarajan%26aufirst%3DA.%26atitle%3DSelective%2520Degradation%2520of%2520CDK6%2520by%2520a%2520Palbociclib%2520Based%2520PROTAC%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26issue%3D11%26spage%3D1375%26epage%3D1379%26doi%3D10.1016%2Fj.bmcl.2019.03.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Özguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rieger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span> <span> </span><span class="NLM_article-title">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">2508</span>– <span class="NLM_lpage">2520</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01826</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01826" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2508-2520&issue=5&author=J.+Popowauthor=H.+Arnhofauthor=G.+Baderauthor=H.+Bergerauthor=A.+Ciulliauthor=D.+Coviniauthor=C.+Dankauthor=T.+Gmaschitzauthor=P.+Grebauthor=J.+Karolyi-%C3%96zguerauthor=M.+Koeglauthor=D.+B.+McConnellauthor=M.+Pearsonauthor=M.+Riegerauthor=J.+Rinnenthalauthor=V.+Roesslerauthor=A.+Schrenkauthor=M.+Spinaauthor=S.+Steurerauthor=N.+Trainorauthor=E.+Traxlerauthor=C.+Wieshoferauthor=A.+Zoephelauthor=P.+Ettmayer&title=Highly+Selective+PTK2+Proteolysis+Targeting+Chimeras+to+Probe+Focal+Adhesion+Kinase+Scaffolding+Functions&doi=10.1021%2Facs.jmedchem.8b01826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span></div><div class="casAuthors">Popow, Johannes; Arnhof, Heribert; Bader, Gerd; Berger, Helmut; Ciulli, Alessio; Covini, David; Dank, Christian; Gmaschitz, Teresa; Greb, Peter; Karolyi-Oezguer, Jale; Koegl, Manfred; McConnell, Darryl B.; Pearson, Mark; Rieger, Maria; Rinnenthal, Joerg; Roessler, Vanessa; Schrenk, Andreas; Spina, Markus; Steurer, Steffen; Trainor, Nicole; Traxler, Elisabeth; Wieshofer, Corinna; Zoephel, Andreas; Ettmayer, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2508-2520</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacol. inhibition to reduced tumorigenicity.  However, the clin. relevance of targeting PTK2 still remains to be proven.  Here, we present two highly selective and functional PTK2 proteolysis-targeting chimeras utilizing von Hippel-Lindau and cereblon ligands to hijack E3 ligases for PTK2 degrdn.  BI-3663 (cereblon-based) degrades PTK2 with a median DC50 of 30 nM to >80% across a panel of 11 HCC cell lines.  Despite effective PTK2 degrdn., these compds. did not phenocopy the reported antiproliferative effects of PTK2 depletion in any of the cell lines tested.  By disclosing these compds., we hope to provide valuable tools for the study of PTK2 degrdn. across different biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsL2R_QBrfk7Vg90H21EOLACvtfcHk0lhNwORJwtzJjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D&md5=f0ad27d3ff46ab12e8a90e8d40a6f3c6</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01826%26sid%3Dliteratum%253Aachs%26aulast%3DPopow%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DBerger%26aufirst%3DH.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCovini%26aufirst%3DD.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DKarolyi-%25C3%2596zguer%26aufirst%3DJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DRieger%26aufirst%3DM.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DRoessler%26aufirst%3DV.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSpina%26aufirst%3DM.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DEttmayer%26aufirst%3DP.%26atitle%3DHighly%2520Selective%2520PTK2%2520Proteolysis%2520Targeting%2520Chimeras%2520to%2520Probe%2520Focal%2520Adhesion%2520Kinase%2520Scaffolding%2520Functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D5%26spage%3D2508%26epage%3D2520%26doi%3D10.1021%2Facs.jmedchem.8b01826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serebrenik, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=583-598&issue=2&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+Characterization+of+von+Hippel-Lindau-Recruiting+Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+TANK-Binding+Kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span></div><div class="casAuthors">Crew, Andrew P.; Raina, Kanak; Dong, Hanqing; Qian, Yimin; Wang, Jing; Vigil, Dominico; Serebrenik, Yevgeniy V.; Hamman, Brian D.; Morgan, Alicia; Ferraro, Caterina; Siu, Kam; Neklesa, Taavi K.; Winkler, James D.; Coleman, Kevin G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">583-598</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are bifunctional mols. that recruit an E3 ligase to a target protein to facilitate ubiquitination and subsequent degrdn. of that protein.  While the field of targeted degraders is still relatively young, the potential for this modality to become a differentiated and therapeutic reality is strong, such that both academic and pharmaceutical institutions are now entering this interesting area of research.  In this paper, the authors describe a broadly applicable process for identifying degrader hits based around the serine/threonine kinase TANK-binding kinase 1 (TBK1), and have generalized the key structural elements assocd. with degrdn. activities.  Compd. I is a potent hit (TBK1 DC50 = 12 nM, Dmax = 96%) with excellent selectivity against a related kinase IKKε, which was further used as a chem. tool to assess TBK1 as a target in mutant K-Ras cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIJWCnFncQRLVg90H21EOLACvtfcHk0lhPnObiFkJ6cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK&md5=41c3627e05ca31aee3ba85815685fcb2</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520von%2520Hippel-Lindau-Recruiting%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520TANK-Binding%2520Kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D2%26spage%3D583%26epage%3D598%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woyach, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the C481S Ibrutinib-Resistance Mutation in Bruton’s Tyrosine Kinase Using PROTAC-Mediated Degradation</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">26</span>),  <span class="NLM_fpage">3564</span>– <span class="NLM_lpage">3575</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00391</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00391" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVCgt7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=3564-3575&issue=26&author=A.+D.+Buhimschiauthor=H.+A.+Armstrongauthor=M.+Toureauthor=S.+Jaime-Figueroaauthor=T.+L.+Chenauthor=A.+M.+Lehmanauthor=J.+A.+Woyachauthor=A.+J.+Johnsonauthor=J.+C.+Byrdauthor=C.+M.+Crews&title=Targeting+the+C481S+Ibrutinib-Resistance+Mutation+in+Bruton%E2%80%99s+Tyrosine+Kinase+Using+PROTAC-Mediated+Degradation&doi=10.1021%2Facs.biochem.8b00391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the C481S Ibrutinib-Resistance Mutation in Bruton's Tyrosine Kinase Using PROTAC-Mediated Degradation</span></div><div class="casAuthors">Buhimschi, Alexandru D.; Armstrong, Haley A.; Toure, Momar; Jaime-Figueroa, Saul; Chen, Timothy L.; Lehman, Amy M.; Woyach, Jennifer A.; Johnson, Amy J.; Byrd, John C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">3564-3575</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of Bruton's tyrosine kinase (BTK) with the irreversible inhibitor ibrutinib has emerged as a transformative treatment option for patients with chronic lymphocytic leukemia (CLL) and other B-cell malignancies.  Yet, more than 80% of CLL patients develop resistance due to a cysteine to serine mutation at the site covalently bound by ibrutinib (C481S).  Currently, an effective treatment option for C481S patients exhibiting relapse to ibrutinib does not exist and these patients have poor outcomes.  To address this, we have developed a PROteolysis TArgeting Chimera (PROTAC) that induces degrdn. of both wild-type and C481S mutant BTK.  We selected a lead PROTAC, MT-802, from several candidates based on its potency to induce BTK knockdown.  MT-802 recruits BTK to the cereblon (CRBN) E3 ubiquitin ligase complex to trigger BTK ubiquitination and degrdn. via the proteasome.  MT-802 binds fewer off-target kinases than ibrutinib and retains equiv. potency (>99% degrdn. at nanomolar concns.) against wild-type and C481S BTK.  In cells isolated from CLL patients with the C481S mutation, MT-802 is able to reduce the pool of active, phosphorylated BTK whereas ibrutinib cannot.  Collectively, these data provide basis for further preclin. study of BTK PROTACs as a novel strategy for treatment of C481S mutant CLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh-ls5ps9FaLVg90H21EOLACvtfcHk0lhPnObiFkJ6cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVCgt7zO&md5=bd27f09d041413c7bacb6d61fcb9a327</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00391%26sid%3Dliteratum%253Aachs%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DArmstrong%26aufirst%3DH.%2BA.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DT.%2BL.%26aulast%3DLehman%26aufirst%3DA.%2BM.%26aulast%3DWoyach%26aufirst%3DJ.%2BA.%26aulast%3DJohnson%26aufirst%3DA.%2BJ.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520the%2520C481S%2520Ibrutinib-Resistance%2520Mutation%2520in%2520Bruton%25E2%2580%2599s%2520Tyrosine%2520Kinase%2520Using%2520PROTAC-Mediated%2520Degradation%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26issue%3D26%26spage%3D3564%26epage%3D3575%26doi%3D10.1021%2Facs.biochem.8b00391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.+R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+Anaplastic+Lymphoma+Kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lhPnObiFkJ6cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520Anaplastic%2520Lymphoma%2520Kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beharry, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolley, G. A.</span></span> <span> </span><span class="NLM_article-title">Azobenzene Photoswitches for Biomolecules</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">4422</span>– <span class="NLM_lpage">4437</span>, <span class="refDoi"> DOI: 10.1039/c1cs15023e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1039%2Fc1cs15023e" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=21483974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVemurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=4422-4437&issue=8&author=A.+A.+Beharryauthor=G.+A.+Woolley&title=Azobenzene+Photoswitches+for+Biomolecules&doi=10.1039%2Fc1cs15023e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Azobenzene photoswitches for biomolecules</span></div><div class="casAuthors">Beharry, Andrew A.; Woolley, G. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4422-4437</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The photoisomerization of azobenzene has been known for almost 75 years but only recently has this process been widely applied to biol. systems.  The central challenge of how to productively couple the isomerization process to a large functional change in a biomol. has been met in a no. of instances and it appears that effective photocontrol of a large variety of biomols. may be possible.  This crit. review summarizes key properties of azobenzene that enable its use as a photoswitch in biol. systems and describes strategies for using azobenzene photoswitches to drive functional changes in peptides, proteins, nucleic acids, lipids, and carbohydrates (192 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU0_QvNuRJzrVg90H21EOLACvtfcHk0lghLYErlWTyLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVemurg%253D&md5=49407115a7f1bc77edbdba90a79c1e58</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2Fc1cs15023e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cs15023e%26sid%3Dliteratum%253Aachs%26aulast%3DBeharry%26aufirst%3DA.%2BA.%26aulast%3DWoolley%26aufirst%3DG.%2BA.%26atitle%3DAzobenzene%2520Photoswitches%2520for%2520Biomolecules%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26issue%3D8%26spage%3D4422%26epage%3D4437%26doi%3D10.1039%2Fc1cs15023e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of PROTAC Cooperative Recognition for Selective Protein Degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&issue=5&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+Basis+of+PROTAC+Cooperative+Recognition+for+Selective+Protein+Degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lghLYErlWTyLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520Basis%2520of%2520PROTAC%2520Cooperative%2520Recognition%2520for%2520Selective%2520Protein%2520Degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D5%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bléger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brouwer, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, S.</span></span> <span> </span><span class="NLM_article-title"><i>o</i>-Fluoroazobenzenes as Readily Synthesized Photoswitches Offering Nearly Quantitative Two-Way Isomerization with Visible Light</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span> (<span class="NLM_issue">51</span>),  <span class="NLM_fpage">20597</span>– <span class="NLM_lpage">20600</span>, <span class="refDoi"> DOI: 10.1021/ja310323y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja310323y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCjur7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=20597-20600&issue=51&author=D.+Bl%C3%A9gerauthor=J.+Schwarzauthor=A.+M.+Brouwerauthor=S.+Hecht&title=o-Fluoroazobenzenes+as+Readily+Synthesized+Photoswitches+Offering+Nearly+Quantitative+Two-Way+Isomerization+with+Visible+Light&doi=10.1021%2Fja310323y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">o-Fluoroazobenzenes as Readily Synthesized Photoswitches Offering Nearly Quantitative Two-Way Isomerization with Visible Light</span></div><div class="casAuthors">Bleger, David; Schwarz, Jutta; Brouwer, Albert M.; Hecht, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">20597-20600</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Azobenzene functionalized with ortho-fluorine atoms has a lower energy of the n-orbital of the Z-isomer, resulting in a sepn. of the E and Z isomers' n→π* absorption bands.  Introducing para-substituents allows for further tuning of the absorption spectra of o-fluoroazobenzenes.  In particular, electron-withdrawing ester groups give rise to a 50 nm sepn. of the n→π* transitions.  Green and blue light can therefore be used to induce E→Z and Z→E isomerizations, resp.  The o-fluoroazobenzene scaffold is readily synthesized and can be inserted into larger structures via its aryl termini.  These new azobenzene derivs. can be switched in both ways with high photoconversions, and their Z-isomers display a remarkably long thermal half-life.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4ku9tBFTq4rVg90H21EOLACvtfcHk0lghLYErlWTyLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCjur7O&md5=c3e0e90eb46e5d6196f79909e8f2ba51</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fja310323y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja310323y%26sid%3Dliteratum%253Aachs%26aulast%3DBl%25C3%25A9ger%26aufirst%3DD.%26aulast%3DSchwarz%26aufirst%3DJ.%26aulast%3DBrouwer%26aufirst%3DA.%2BM.%26aulast%3DHecht%26aufirst%3DS.%26atitle%3Do-Fluoroazobenzenes%2520as%2520Readily%2520Synthesized%2520Photoswitches%2520Offering%2520Nearly%2520Quantitative%2520Two-Way%2520Isomerization%2520with%2520Visible%2520Light%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26issue%3D51%26spage%3D20597%26epage%3D20600%26doi%3D10.1021%2Fja310323y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-Induced BET Protein Degradation as a Therapy for Castration-Resistant Prostate Cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">26</span>),  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&issue=26&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-Induced+BET+Protein+Degradation+as+a+Therapy+for+Castration-Resistant+Prostate+Cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0liUtkDY5X21iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-Induced%2520BET%2520Protein%2520Degradation%2520as%2520a%2520Therapy%2520for%2520Castration-Resistant%2520Prostate%2520Cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26issue%3D26%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-Kinase Degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2017.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=29129717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=88-99&issue=1&author=H.+T.+Huangauthor=D.+Dobrovolskyauthor=J.+Paulkauthor=G.+Yangauthor=E.+L.+Weisbergauthor=Z.+M.+Doctorauthor=D.+L.+Buckleyauthor=J.+H.+Choauthor=E.+Koauthor=J.+Jangauthor=K.+Shiauthor=H.+G.+Choiauthor=J.+D.+Griffinauthor=Y.+Liauthor=S.+P.+Treonauthor=E.+S.+Fischerauthor=J.+E.+Bradnerauthor=L.+Tanauthor=N.+S.+Gray&title=A+Chemoproteomic+Approach+to+Query+the+Degradable+Kinome+Using+a+Multi-Kinase+Degrader&doi=10.1016%2Fj.chembiol.2017.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span></div><div class="casAuthors">Huang, Hai-Tsang; Dobrovolsky, Dennis; Paulk, Joshiawa; Yang, Guang; Weisberg, Ellen L.; Doctor, Zainab M.; Buckley, Dennis L.; Cho, Joong-Heui; Ko, Eunhwa; Jang, Jaebong; Shi, Kun; Choi, Hwan Geun; Griffin, James D.; Li, Ying; Treon, Steven P.; Fischer, Eric S.; Bradner, James E.; Tan, Li; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heterobifunctional mols. that recruit E3 ubiquitin ligases, such as cereblon, for targeted protein degrdn. represent an emerging pharmacol. strategy.  A major unanswered question is how generally applicable this strategy is to all protein targets.  In this study, we designed a multi-kinase degrader by conjugating a highly promiscuous kinase inhibitor with a cereblon-binding ligand, and used quant. proteomics to discover 28 kinases, including BTK, PTK2, PTK2B, FLT3, AURKA, AURKB, TEC, ULK1, ITK, and nine members of the CDK family, as degradable.  This set of kinases is only a fraction of the intracellular targets bound by the degrader, demonstrating that successful degrdn. requires more than target engagement.  The results guided us to develop selective degraders for FLT3 and BTK, with potentials to improve disease treatment.  Together, this study demonstrates an efficient approach to triage a gene family of interest to identify readily degradable targets for further studies and pre-clin. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYNxhoPV3127Vg90H21EOLACvtfcHk0liUtkDY5X21iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO&md5=703cfbaca3edd24176472158563f895f</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%2BT.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520Chemoproteomic%2520Approach%2520to%2520Query%2520the%2520Degradable%2520Kinome%2520Using%2520a%2520Multi-Kinase%2520Degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26issue%3D1%26spage%3D88%26epage%3D99%26doi%3D10.1016%2Fj.chembiol.2017.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&issue=1&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+Design+from+Selective+Degradation+with+a+Promiscuous+Warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0liUtkDY5X21iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520Design%2520from%2520Selective%2520Degradation%2520with%2520a%2520Promiscuous%2520Warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26issue%3D1%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span> <span> </span><span class="NLM_article-title">Development of Targeted Protein Degradation Therapeutics</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">937</span>– <span class="NLM_lpage">944</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0362-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1038%2Fs41589-019-0362-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31527835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=937-944&issue=10&author=P.+P.+Chamberlainauthor=L.+G.+Hamann&title=Development+of+Targeted+Protein+Degradation+Therapeutics&doi=10.1038%2Fs41589-019-0362-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Development of targeted protein degradation therapeutics</span></div><div class="casAuthors">Chamberlain, Philip P.; Hamann, Lawrence G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">937-944</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeted protein degrdn. as a therapeutic modality has seen dramatic progress and massive investment in recent years because of the convergence of two key scientific breakthroughs: optimization of first-generation peptidic proteolysis-targeted chimeras (PROTACs) into more drug-like mols. able to support in vivo proof of concept and the discovery that clin. mols. function as degraders by binding and repurposing the proteins cereblon and DCAF15.  This provided clin. validation for the general approach through the cereblon modulator class of drugs and provided highly drug-like and ligand-efficient E3 ligase binders upon which to tether target-binding moieties.  Increasingly rational and systematic approaches including biophys. and structural studies on ternary complexes are being leveraged as the field advances.  In this Perspective we summarize the discoveries that have laid the foundation for future degrdn. therapeutics, focusing on those classes of small mols. that redirect E3 ubiquitin ligases to non-native substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfcIEwVR-Xq7Vg90H21EOLACvtfcHk0lhn__ZqwSoMqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF&md5=468e3fd87ba2b21607dac8b495e5ecf6</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0362-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0362-y%26sid%3Dliteratum%253Aachs%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26atitle%3DDevelopment%2520of%2520Targeted%2520Protein%2520Degradation%2520Therapeutics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26issue%3D10%26spage%3D937%26epage%3D944%26doi%3D10.1038%2Fs41589-019-0362-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuchter, M. J.</span></span> <span> </span><span class="NLM_article-title">On the Promise of Photopharmacology Using Photoswitches: A Medicinal Chemist’s Perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">11436</span>– <span class="NLM_lpage">11447</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00629</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00629" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=11436-11447&issue=20&author=M.+J.+Fuchter&title=On+the+Promise+of+Photopharmacology+Using+Photoswitches%3A+A+Medicinal+Chemist%E2%80%99s+Perspective&doi=10.1021%2Facs.jmedchem.0c00629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">On the Promise of Photopharmacology Using Photoswitches: A Medicinal Chemist's Perspective</span></div><div class="casAuthors">Fuchter, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">11436-11447</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Photopharmacol. is a growing area of endeavor that employs photoswitchable ligands to allow for light-dependent pharmacol. activity.  By coupling light to therapeutic action, improved spatial and temporal selectivity can be achieved and subsequently harnessed for new concepts in therapy.  Tremendous progress has already been made, with photopharmacol. agents now reported against a wide array of target classes and light-dependent results demonstrated in a range of live cell and animal models.  Several challenges remain, however, esp. in order for photopharmacol. to truly impact the clin. management of disease.  This Perspective aims to summarize these challenges, particularly with attention to the medicinal chem. that will be unavoidably required for the further translation of these agents/approaches.  By clearly defining challenges for drug hunters, it is hoped that further research into the medicinal chem. of photopharmacol. agents will be stimulated, ultimately enabling full realization of the huge potential for this exciting field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ_HhYiq4_z7Vg90H21EOLACvtfcHk0lhn__ZqwSoMqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3E&md5=ab802ee3cb69c87058b34b56b1175a8b</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00629%26sid%3Dliteratum%253Aachs%26aulast%3DFuchter%26aufirst%3DM.%2BJ.%26atitle%3DOn%2520the%2520Promise%2520of%2520Photopharmacology%2520Using%2520Photoswitches%253A%2520A%2520Medicinal%2520Chemist%25E2%2580%2599s%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D20%26spage%3D11436%26epage%3D11447%26doi%3D10.1021%2Facs.jmedchem.0c00629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morstein, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauner, D.</span></span> <span> </span><span class="NLM_article-title">New Players in Phototherapy: Photopharmacology and Bio-Integrated Optoelectronics</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2019.03.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=10.1016%2Fj.cbpa.2019.03.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=31108317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFOgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2019&pages=145-151&author=J.+Morsteinauthor=D.+Trauner&title=New+Players+in+Phototherapy%3A+Photopharmacology+and+Bio-Integrated+Optoelectronics&doi=10.1016%2Fj.cbpa.2019.03.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">New players in phototherapy: photopharmacology and bio-integrated optoelectronics</span></div><div class="casAuthors">Morstein, Johannes; Trauner, Dirk</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">145-151</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Photodynamic therapy and phototherapy are used in the clinic to treat dermatol. conditions, cancer, macular degeneration, and a variety of other diseases.  Despite their long history and widespread application, the scope of these therapeutic approaches has been limited by a lack of specificity and challenges with light delivery.  In recent years, much progress has been made in these regards.  Photopharmacol. has provided drug-like mols. that change their efficacy upon irradn. and allow for the optical control of a wide range of defined biol. targets.  Many photopharmaceuticals are now used in vivo and some show promising results in preclin. development.  At the same time, new bioelectronics for subdermal light delivery have been engineered that could enable phototherapy deep in tissue, for example within the human brain.  These developments could increase the impact of photodynamic therapy in human precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEDSiiVMgoY7Vg90H21EOLACvtfcHk0lhn__ZqwSoMqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFOgsb8%253D&md5=23ee3fbfdf33ec9c2cb5e5292f0bbc4a</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2019.03.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2019.03.013%26sid%3Dliteratum%253Aachs%26aulast%3DMorstein%26aufirst%3DJ.%26aulast%3DTrauner%26aufirst%3DD.%26atitle%3DNew%2520Players%2520in%2520Phototherapy%253A%2520Photopharmacology%2520and%2520Bio-Integrated%2520Optoelectronics%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D50%26spage%3D145%26epage%3D151%26doi%3D10.1016%2Fj.cbpa.2019.03.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konrad, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savasci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allmendinger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochsenfeld, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A. M.</span></span> <span> </span><span class="NLM_article-title">Computational Design and Synthesis of a Deeply Red-Shifted and Bistable Azobenzene</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">6538</span>– <span class="NLM_lpage">6547</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b10430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b10430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;key=1%3ACAS%3A528%3ADC%252BB3cXltlylsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=6538-6547&issue=14&author=D.+B.+Konradauthor=G.+Savasciauthor=L.+Allmendingerauthor=D.+Traunerauthor=C.+Ochsenfeldauthor=A.+M.+Ali&title=Computational+Design+and+Synthesis+of+a+Deeply+Red-Shifted+and+Bistable+Azobenzene&doi=10.1021%2Fjacs.9b10430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Design and Synthesis of a Deeply Red-Shifted and Bistable Azobenzene</span></div><div class="casAuthors">Konrad, David B.; Savasci, Goekcen; Allmendinger, Lars; Trauner, Dirk; Ochsenfeld, Christian; Ali, Ahmed M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6538-6547</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We computationally dissected the electronic and geometrical influences of ortho-chlorinated azobenzenes on their photophys. properties.  X-ray anal. provided the insight that trans-tetra-ortho-chloro azobenzene is conformationally flexible and thus subject to mol. motions.  This allows the photoswitch to adopt a range of red-shifted geometries which account for the extended n → π* band tails.  Based on our results, we designed the di-ortho-fluoro di-ortho-chloro (dfdc) azobenzene and provided computational evidence for the superiority of this substitution pattern to tetra-ortho-chloro azobenzene.  Thereafter, we synthesized dfdc azobenzene by ortho-chlorination via 2-fold C-H activation and exptl. confirmed its structural and photophys. properties through UV-vis, NMR and X-ray analyses.  The advantages include near-bistable isomers and an increased sepn. of the n → π* bands between the trans- and cis-conformations which allows for the generation of unusually high levels of the cis-isomer by irradn. with green/yellow light as well as red light within the biooptical window.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopyLeuf7pYA7Vg90H21EOLACvtfcHk0liahlsIkljA5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXltlylsrs%253D&md5=72f3c84a0357af9f90a2f9b99c066460</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b10430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b10430%26sid%3Dliteratum%253Aachs%26aulast%3DKonrad%26aufirst%3DD.%2BB.%26aulast%3DSavasci%26aufirst%3DG.%26aulast%3DAllmendinger%26aufirst%3DL.%26aulast%3DTrauner%26aufirst%3DD.%26aulast%3DOchsenfeld%26aufirst%3DC.%26aulast%3DAli%26aufirst%3DA.%2BM.%26atitle%3DComputational%2520Design%2520and%2520Synthesis%2520of%2520a%2520Deeply%2520Red-Shifted%2520and%2520Bistable%2520Azobenzene%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26issue%3D14%26spage%3D6538%26epage%3D6547%26doi%3D10.1021%2Fjacs.9b10430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CI1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CI1','PDB','4CI1'); return false;">PDB: 4CI1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W9H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W9H','PDB','4W9H'); return false;">PDB: 4W9H</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01542&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-24%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01542%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01542" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799027aaac51228","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
